<!doctype html>
<style>
@charset "utf-8";
body {
	height: 100%;
	margin: 0;
	padding: 0;
}

.banner {
	position: fixed;
	top: 0;
	width: 100%;
	height: 65px;
	background: gray;
	line-height: 65px;
}
.spacer {
	height: 10px;
}

.main-content {
	padding: 5%;
}

.collapse_ACT {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-style: none solid none none;
	border-color: #B4045F;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_ACT + input{
	display: none;
}
.collapse_ACT + input + div{
	display: none;
}
.collapse_ACT + input:checked + div{
	display: block;
}
.ACT {
	text-align: justify;
	text-justify: inter-word;
}
.ACT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B4045F;
}
.ACT_collapse_similarity:hover{
	background: #e5d3dc;
}
.ACT_collapse_similarity + input {
	display: none;
}
.ACT_collapse_similarity + input + div{
	display: none;
}

.ACT_collapse_similarity + input:checked + div{
	display: block;
}
.ACT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.ACT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B4045F;
	padding-left: 5px;
}
.ACT_collapse_similarity:hover{
	background: #e5d3dc;
}
.ACT_collapse_similarity + input {
	display: none;
}
.ACT_collapse_similarity + input + div{
	display: none;
}
.ACT_collapse_similarity + input:checked + div{
	display: block;
}
.ACT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.ACT_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #B4045F;
}
.ACT_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_DOM {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-style: none solid none none;
	border-color: #B404AE;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_DOM + input{
	display: none;
}
.collapse_DOM + input + div{
	display: none;
}
.collapse_DOM + input:checked + div{
	display: block;
}
.DOM {
	text-align: justify;
	text-justify: inter-word;
}
.DOM_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B404AE;
	padding-left: 5px;
}
.DOM_collapse_similarity:hover{
	background: #e8d7e7;
}
.DOM_collapse_similarity + input {
	display: none;
}
.DOM_collapse_similarity + input + div{
	display: none;
}
.DOM_collapse_similarity + input:checked + div{
	display: block;
}
.DOM_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}.DOM_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #B404AE;
}
.DOM_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_EVO {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #7401DF;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_EVO + input{
	display: none;
}
.collapse_EVO + input + div{
	display: none;
}
.collapse_EVO + input:checked + div{
	display: block;
}
.EVO {
	text-align: justify;
	text-justify: inter-word;
}
.EVO_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #7401DF;
	padding-left: 5px;
}
.EVO_collapse_similarity:hover{
	background: #e2d7ed;
}
.EVO_collapse_similarity + input {
	display: none;
}
.EVO_collapse_similarity + input + div{
	display: none;
}
.EVO_collapse_similarity + input:checked + div{
	display: block;
}
.EVO_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.EVO_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #7401DF;
}
.EVO_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_RP {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #0174DF;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_RP + input{
	display: none;
}
.collapse_RP + input + div{
	display: none;
}
.collapse_RP + input:checked + div{
	display: block;
}
.RP {
	text-align: justify;
	text-justify: inter-word;
}
.RP_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #0174DF;
	padding-left: 5px;
}
.RP_collapse_similarity:hover{
	background: #c9d7e2;
}
.RP_collapse_similarity + input {
	display: none;
}
.RP_collapse_similarity + input + div{
	display: none;
}
.RP_collapse_similarity + input:checked + div{
	display: block;
}
.RP_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.RP_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #0174DF;
}
.RP_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_SIT {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #0489B1;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_SIT + input{
	display: none;
}
.collapse_SIT + input + div{
	display: none;
}
.collapse_SIT + input:checked + div{
	display: block;
}
.SIT {
	text-align: justify;
	text-justify: inter-word;
}
.SIT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #0489B1;
	padding-left: 5px;
}
.SIT_collapse_similarity:hover{
	background: #d0dfe2;
}
.SIT_collapse_similarity + input {
	display: none;
}
.SIT_collapse_similarity + input + div{
	display: none;
}
.SIT_collapse_similarity + input:checked + div{
	display: block;
}
.SIT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.SIT_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #0489B1;
}
.SIT_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_TU {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #04B486;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_TU + input{
	display: none;
}
.collapse_TU + input + div{
	display: none;
}
.collapse_TU + input:checked + div{
	display: block;
}
.TU {
	text-align: justify;
	text-justify: inter-word;
}
.TU_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #04B486;
	padding-left: 5px;
}
.TU_collapse_similarity:hover{
	background: #d0e2de;
}
.TU_collapse_similarity + input {
	display: none;
}
.TU_collapse_similarity + input + div{
	display: none;
}
.TU_collapse_similarity + input:checked + div{
	display: block;
}
.TU_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.TU_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #04B486;
}
.TU_similar > a {
	text-decoration: none;
	color: gray;
}

ul { 
	list-style-type: none;
	padding: 0;
	margin: 0;
}
ul li > a {
	text-decoration: none;
	color: gray;
}
ul li:hover, ul li:focus {
	background: #F0F0F0;
}

ul li:first-child span {
  border-left:none;  
}

</style>
<html>
<head>
<meta charset="utf-8">
<title>TF properties - Lrp</title>
<meta name="theme-color" content="#ffffff">
</head>
<body>
	<div class="banner">
		<center><font color="white" size="16px" face="arial">Lrp</font></center>
	</div>
	<div align="center" class="main-content">
		<div class="spacer">
	</div><br>			<label class="collapse_ACT" for="_1"><font size="5px">Lrp transcriptional activity</font></label>
				<input id="_1" type="checkbox">
				<div class="ACT"><br><ul><li>Leucine decreased the hydrodyn-amic radius of 6HisLrp from approximately 6.3 nm to 5.6 nm .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11575919" target="_blank">11575919</a>]<br></li>
<li>As observed for 6HisLrp in the absence of leucine , smaller species such as dimer and tetramer made little contribution to the equilibrium protein concentration gradient of 1-80 mM ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11575919" target="_blank">11575919</a>]<br></li>
<li>Protein molecular markers are shown in lane 5 of A and B and lane 1 of C. Sedimentation velocity and total intensity light scattering support an octameric structure in solutions containing leucine Sedimentation velocity measurements performed on 6HisLrp in the presence of 10 mM leucine were analyzed by the time-derivative method to obtain pro ® les of the distribution of sedimenting species .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11575919" target="_blank">11575919</a>]<br></li>
<li>This RH value is signi ® - cantly lower than the one for 6HisLrp in the absence of leucine ( 6.3 nm ) and in the presence of 10 mM leucine ( 5.6 nm ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11575919" target="_blank">11575919</a>]<br></li>
<li>175 , No. 24 Complex Transcriptional Control of the us-Dependent Stationary-Phase-Induced and Osmotically Regulated osmY ( csi-5 ) Gene Suggests Novel Roles for Lrp , Cyclic AMP ( cAMP ) Receptor Protein-cAMP Complex , and Integration Host Factor in the Stationary-Phase Response of Escherichia coli ROLAND LANGE , MECHTHILD BARTH , AND REGINE HENGGE-ARONIS * Department of Biology , University of Konstanz , P.O. Box 5560 , 78434 Konstanz , Germany Received 15 June 1993/Accepted 18 October 1993 osmY ( csi-5 ) is a representative of a large group of cj-dependent genes in Escherichia coli that exhibit both stationary-phase induction and osmotic regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8253679" target="_blank">8253679</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_6">
[ 21 ] Lange R. , Barth M. , Hengge-Aronis N. , Complex transcriptional control of the sigma s-dependent stationary-phase-induced and osmotically regulated osmY ( csi-5 ) gene suggests novel roles for Lrp , cyclic AMP ( cAMP ) receptor protein-cAMP complex , and integration host factor in the stationary-phase response of Escheri-chia coli , J. Bacteriol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9924977" target="_blank">9924977</a>]<br></label>
<input id="act_similarity_6", type="checkbox">
<div class="ACT_similar">
<li>Complex transcriptional control of the a ' - dependent stationary-phase-in-duced and osmotically regulated osmY ( csi-5 ) gene suggests novel roles for Lrp , cyclic AMP ( cAMP ) receptor protein-cAMP complex , and integration host factor in the stationary-phase response of Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8071194" target="_blank">8071194</a>]<br></li>
<li>Lange , R. , Barth , M. , and Hengge-Aronis , R. ( 1993 ) Complex control of the ss-dependent stationary phase-induced and osmotically regulated osmY ( csi-5 ) gene suggests novel roles for Lrp , cyclic AMP ( cAMP ) , receptor protein -- cAMP complex , and integration host factor in the stationary phase response of Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9663683" target="_blank">9663683</a>]<br></li>
<li>Lange , R. , Barth , M. , and Hengge-Aronis , R. ( 1993 ) Complex transcriptional control of the os-dependent stationary-phase-induced and osmotically regulated osmY ( csi-5 ) gene suggests novel roles for Lrp , cyclic AMP ( CAMP ) - receptor protein-CAMP complex , and integration host factor in the stationary-phase response of Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8809748" target="_blank">8809748</a>]<br></li>
<li>Lange , R. , Barth , M. , and Hengge-Aronis , R. ( 1993 ) Complex transcriptional control of stationary-phase-induced and osmo-tically regulated osmY ( csi-5 ) gene suggests novel roles for Lrp , cyclic AMP ( cAMP ) receptor protein ± cAMP complex , and integration host factor in the stationary-phase response of Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10692161" target="_blank">10692161</a>]<br></li>
<li>Lange , R. , Barth , M. , and Henge-Aronis , R. ( 1993 ) Complex transcriptional control of the sigma S-dependent station-ary-phase-induced and osmotically regulated osmY ( csi-5 ) gene suggests novel roles for Lrp , cyclic AMP ( cAMP ) receptor protein ± cAMP complex , and integration host factor in the stationary-phase response of Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10361282" target="_blank">10361282</a>]<br></li>
<li>Lange , R. , Barth , M. , and Hengge-Aronis , R. ( 1993 ) Complex transcriptional control of the sigmaS-dependent stationary-phase-induced and osmotically regulated osmY ( csi-5 ) gene suggests novel roles for Lrp , cyclic AMP ( cAMP ) receptor protein-cAMP complex , and integration host factor in the stationary-phase response of Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17224607" target="_blank">17224607</a>]<br></li>
<li>Lange , R , , Barth , M , , and Hengge-Aronis , R. ( 1993 ) Complex transcriptional control of the a ^ - dependent stationary phase-induced and osmotically regulated osmY -LCB- csi-5 ) gene suggests novel roles for Lrp , cyclic AMP ( cAMP ) receptor protein-cAMP complex , and integration host factor in the stationary-phase response of Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7984106" target="_blank">7984106</a>]<br></li></div>
<li>BMC Microbiology BioMed Central ss Open Acce Research article Integration of regulatory signals through involvement of multiple global regulators : control of the Escherichia coli gltBDF operon by Lrp , IHF , Crp , and ArgR Ligi Paul1 ,6 , Pankaj K Mishra2 , Robert M Rowena G Matthews1 ,4,5 Blumenthal * 2,3 and Address : 1Life Sciences Institute , University of Michigan , Ann Arbor , Michigan 48109 , USA , 2Department of Medical Microbiology and Immunology , University of Toledo Health Science Campus , Toledo , Ohio 43614-5806 , USA , 3Program in Bioinformatics and Proteomics / Genomics , University of Toledo Health Science Campus , Toledo , Ohio 43614-5806 , USA , 4Biophysics Research Division , University of Michigan , Ann Arbor , Michigan 48109 , USA , 5Department of Biological Chemistry , University of Michigan , Ann Arbor , Michigan 48109 , USA and 6Vitamin Metabolism and Aging Laboratory , Jean Mayer USDA Human Nutrition Research Center on Aging , Tufts University , Boston , Massachusetts 02111 , USA Email : Ligi Paul - Pankaj K Mishra - Pankaj.Mishra@utoledo.edu; Robert M Blumenthal * - Robert.Blumenthal@utoledo.edu; Rowena G Matthews - RMatthew@umich.edu * Corresponding author Received : 22 September 2006 Accepted : 18 January 2007 Published : 18 January 2007 BMC Microbiology 2007 , 7:2 doi :10.1186 / 1471-2180-7-2 This article is available from : http://www.biomedcentral.com/1471-2180/7/2 © 2007 Paul et al ; licensee BioMed Central Ltd. .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17233899" target="_blank">17233899</a>]<br></li>
<li>BMC Microbiology BioMed Central ss Open Acce Research article Limited functional conservation of a global regulator am ong related bacterial genera : Lrp in Escherichia , Proteus and Vibrio Robert E Lintner1 ,5 , Pankaj K Mishra1 ,6 , Poonam Srivastava2 , Betsy M Martinez-Vaz2 ,7 , Arkady B Khodursky2 ,3 and Robert M Blumenthal * 1,4 Address : 1Department of Medical Microbiology and Immunology , University of Toledo Health Sciences Center , Toledo , OH 43614-2598 , USA , 2Department of Biochemistry , Molecular Biology , and Biophysics , University of Minnesota , St. Paul , MN 55108-1022 , USA , 3Biotechnology Institute , University of Minnesota , St. Paul , MN 55108-1022 , USA , 4Program in Bioinformatics and Proteomics/Genomics , University of Toledo Health Sciences Center , Toledo , OH 43614-2598 , USA , 5Current Address : Whitehead Institute for Biomedical Research , 9 Cambridge Center , Cambridge , MA 02142 , USA , 6Current Address : Department of Pediatrics , Wayne State University , Detroit , MI 48201 , USA and 7Current Address : Hamline University , St. Paul , MN 55104 , USA Email : Robert E Lintner - lintner@wi.mit.edu; Pankaj K Mishra - pmishra@med.wayne.edu; Poonam Srivastava - Sriva036@umn.edu; Betsy M Martinez-Vaz - BMartinezVaz01@gw.hamline.edu; Arkady B Khodursky - Khodu001@umn.edu; Robert M Blumenthal * - Robert.Blumenthal@utoledo.edu * Corresponding author Received : 16 November 2007 Accepted : 11 April 2008 Published : 11 April 2008 BMC Microbiology 2008 , 8:60 doi :10.1186 / 1471-2180-8-60 This article is available from : http://www.biomedcentral.com/1471-2180/8/60 © 2008 Lintner et al ; licensee BioMed Central Ltd. .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18405378" target="_blank">18405378</a>]<br></li>
<li>174 , No. 1 Lrp , a Leucine-Responsive Protein , Regulates Branched-Chain Amino Acid Transport Genes in Escherichia coli STEVEN A. HANEY , ' t JILL V. PLATKO ,2 t DALE L. OXENDER ,1 ' 3 AND JOSEPH M. CALVO2 ,4 * Department of Biological Chemistry , University of Michigan Medical School , Ann Arbor , Michigan 48109-06061 ; Sections of Genetics and Development2 and Biochemistry , Molecular and Cell Biology ,4 * Cornell University , Ithaca , New York 14853 ; and Warner Lambert/Parke Davis , Ann Arbor , Michigan 480153 Received 15 August 1991/Accepted 28 October 1991 We investigated the relationship between two regulatory genes , livR and lip , that map near min 20 on the Escherichia coli chromosome .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1729203" target="_blank">1729203</a>]<br></li>
<li>Identiﬁcation of ArgP and Lrp as Transcriptional Regulators of lysP , the Gene Encoding the Speciﬁc Lysine Permease of Escherichia coli † Jimena Ruiz , ‡ Ina Haneburger , and Kirsten Jung * Ludwig-Maximilians-Universita ̈t München , Munich Center for integrated Protein Science ( CiPSM ) at the Department of Biology I , Microbiology , Grosshaderner Strasse 2-4 , 82152 Martinsried , Germany Received 12 July 2010/Accepted 14 March 2011 Expression of lysP , which encodes the lysine-speciﬁc transporter LysP in Escherichia coli , is regulated by the concentration of exogenous available lysine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21441513" target="_blank">21441513</a>]<br></li>
<li>The Cyclic AMP Receptor Protein Is Dependent on GcvA for Regulation of the gcv Operon LAURA D. WONDERLING AND GEORGE V. STAUFFER * Department of Microbiology , The University of Iowa , Iowa City , Iowa 52242 Received 17 August 1998/Accepted 8 January 1999 The Escherichia coli gcv operon is transcriptionally regulated by the GcvA , GcvR , Lrp , and PurR proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10074087" target="_blank">10074087</a>]<br></li>
<li>JOURNAL OF BACTERIOLOGY , Dec. 1996 , p. 7234 -- / 96 / $ 04.0010 Copyright q 1996 , American Society for Microbiology Lrp Is a Direct Repressor of the dad Operon in Escherichia coli ELIZABETH MATHEW , † JIZU ZHI , AND MARTIN FREUNDLICH * Department of Biochemistry and Cell Biology , State University of New York , Stony Brook , New York 11794 Received 7 August 1996/Accepted 8 October 1996 Expression of the degradative D-amino acid dehydrogenase ( dad ) operon is known to be increased when Escherichia coli is grown in the presence of D - or L-alanine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955407" target="_blank">8955407</a>]<br></li>
<li>Crp , GalS MglAB Crp , RbsR RbsAB Crp , GalS MglA Crp , GntR GntK Crp , EbgR EbgA Crp , LacI LacZ Crp , RhaS RhaBD Crp , AraC AraBD Crp , FucR FucK Crp , YiaJ LyxK Crp , FucR FucA Crp , YiaJ SgbE RcsB WcaB CpxR LpxA RcsB WcaB CpxR LpxD RcsB B2060 PhoB PhnCKLN RcsB B2060 PhoB PstB RcsB B2060 PhoB PhoH MhpR MhpF Fur EntA FecI , Fur FecI UidR , UxuR UidR Crp , FucR FucR Crp , Fur Fur Homologous pairs of transcription factors with autoregulation IdnR PurR AsnC Lrp Fnr Crp Homologous triplets of transcription factors with autoregulation TorR PhoB CpxR EmrR PdhR ExuR Eight homologous transcription factors with autoregulation DsdC GcvA CynR Hcar IlvY LysR OxyR CysB Figure 6 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12582243" target="_blank">12582243</a>]<br></li>
<li>aerobic to anaerobic anaerobic to aerobic ArgR CysB BirA DgsA MalT GadW MetJ NikR OxyR SoxR SoxS DcuR CueR FadR HcaR NtrC PhoB PurR SgrR TdcA TrpR ZraR ArcA CpxR CRP CusR FhlA Fis FNR Fur H-NS IHF IscR Lrp ModE NarL NrdR PdhR FlhDC GlcC EvgA GadX LeuO PspF RbsR steady states Figure 3 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22870390" target="_blank">22870390</a>]<br></li>
<li>sitesa % CGb Count % Non-codinge Ada 3 38.7 AraC 6 40.3 2 3 100 100 47 76 100 28 33 32 73 38 100 52 100 86 79 89 100 67 26 100 28 100 70 37 67 28 100 82 100 47 50 51 100 100 38 33 64 3 6 100 100 38 54 100 23 40 18 43 29 100 38 100 39 27 53 58 71 19 100 23 33 47 100 40 65 21 23 60 25 100 41 100 26 43 36 100 35 35 30 64 36 23 22 26 14 100 83 28 ArcA 12 20.0 ArgR 17 22.9 CarP 2 46.0 Crp 49 37.0 CspA 4 41.3 CynR 2 33.3 CysB 3 34.2 CytR 5 37.8 DeoR 3 22.9 DnaA 8 36.7 FadR 7 47.0 FarR 4 20.0 FhlA 3 42.0 Fis 19 38.2 FlhCD 3 53.8 Fnr 9 25.8 FruR 12 42.7 Fur 9 23.4 GalR 7 43.8 GcvA 4 22.5 GlpR 13 39.6 HipB 4 40.9 Hns 15 31.5 IclR 2 26.7 Ihf 26 32.6 IlvY 2 32.4 LacI 3 44.4 LexA 18 33.9 Lrp 18 28.9 MalT 10 61.0 MarR 2 37.5 MelR 2 33.3 MetJ 15 40.1 MetR 8 33.3 NagC 6 22.5 NarL 9 34.6 NtrC 6 51.9 OmpR 9 26.7 OxyR 4 42.3 PdhR 3 33.3 PhoB 15 29.7 PurR 23 45.3 RhaS 2 50.0 RpoD15 27 37.4 RpoD16 48 34.9 RpoD17 116 37.3 RpoD18 34 38.0 RpoD19 25 38.2 RpoE 3 38.9 RpoH13 8 42.1 RpoH14 7 44.3 RpoN 7 40.6 RpoS17 15 43.0 RpoS18 7 45.2 SoxS 14 40.8 TorR 4 37.5 TrpR 4 38.5 Tus 6 26.8 TyrR 17 33.2 a The number of input sites used to construct the matrix .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9813115" target="_blank">9813115</a>]<br></li>
<li>of additional probable sites in predictionsd 3 3 7e 7 -- 4 22e 0 1 0 2 -- -- 5 0 1 -- 13 7 18 1 0 6e 1 1 1 3 Ada AraC ArcA ArgR CarP CpxR Crp CspA CynR CysB CytR DeoR DnaA FadR FhlA Fis FlhCD Fnr FruR Fur GalR GcvA GlpR IciA ( ArgP ) IclR Ihf IlvY LacI LexA Lrp MalT MarR MelR MetJ MetR Mlc ( DgsA ) ModE NagC NarL NarP NtrC OmpR OxyR PdhR PhoB PurR RhaR RhaS SoxR SoxS TorR TrpR TyrR Stem -- loopsf 6 2 0 5 2 9 1 3 4 5e 5 -- 5 5 4 6 3 4 2 -- 1 3 5 1 3 -- aThe number of genes in our study set with documented binding sites for each TF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11160901" target="_blank">11160901</a>]<br></li>
<li>Genes induced at the onset of stationary phase with s - dependent expression as identiﬁed by microarray analyses Functional Mean WT d Ratio classa valueb ( WT/rpoS ) Regulation SDc Gene Gene product Starvation and stress response genes dpsS DNA protection protein , bacterial ferritin IHFS , OxyRS , rssBTC , MMC , PQ , NaSal , Lrp , BG , DHCP , OS FisS , NaSal , Lrp , OS CRPS , LrpS , IHFS , H-NSS , arcBTC , MMC , Lrp , Rcs , BG , DHCP ompR/envZTC , CD uvrYTC , arcBTC SSR 1,597 2.5 0.4 osmES osmYS Stress-inducible outer membrane lipoprotein Hyperosmotically inducible periplasmic protein Induced by phosphate-starvation Superoxide dismulase precursor ( Cu-Zn ) SSR SSR 1,728 424 9.6 4.4 2.1 1.3 656 459 psiF sodCS 4.8 2.7 0.4 0.5 SSR Protein synthesis rpsVS infA Amino acid transport and metabolism artP artI 5.2 2.8 FISS , IHFS , ppGppS 2.7 0.8 30S ribosomal subunit protein S22 Protein chain initiation factor IF-1 3,466 1,011 PS PS Arginine transport , ATP-binding component Arginine transport , periplasmic binding protein L-Asparagine permease Dihydroxyacid dehydratase Tryptophanase AATM AATM 714 346 2.6 0.7 PQ , NaSal , Lrp 3.8 1.9 PQ , Lrp ansP ilvD tnaA AATM AATM AATM 572 364 DHCP ompR/envZTC 2.5 4.8 0.4 2.1 4.3 phoQTC , ompR/envZTC , CRP , uvrYTC , arcATC , arcBTC , rcsBTC , citABTC , MMC , PQ ( ilvG : Lrp ) NaSal , Lrp , BG , DHCP , OS 13.7 7,808 wrbAS Flavodoxin-like protein ; binds the trp repressor SSR 364 5.0 2.0 Iron uptake and storage bfr entD Carbohydrate metabolism crr Bacterioferrin , iron storage homoprotein Enterochelin synthetase , component D IUS IUS 905 508 5.1 2.1 DHCP , ryhB RNA 4.2 0.5 acrBTC Phosphotransferase system , glucose-speciﬁc IIA component Fructose 1,6-bisphosphate aldolase ( b2097 ) Glyoxylate-induced protein Putative oxidase , glycolate utilization ( b2977 ) Glucosamine-6-phosphate deaminase Phosphofructokinase-2 Transaldolase A CHM 443 2.7 0.6 CD 1,264 1,097 CHM 4.6 3.6 3.0 2.6 2.5 2.9 1.4 1.0 1.7 0.6 2 1.8 MMC , Lrp kdpABCDETC fbaB gip uvrYTC , arcATC , arcBTC ompR/envZTC , arcBTC MMC , CD , DHCP rssBTC , MMC , DHCP , CreBC CHM CHM CHM CHM glcG nagB pfkB talA Miscellaneous function or unknown function deoD moeB msyB yafN yahOS ybgA 3.0 1.2 4.0 0.1 2.2 1.7 4.8 3.4 6.1 2.6 4.5 4.5 349 Purine nucleoside phosphorylase Molybdopterin biosynthesis NM CB 1,100 2,038 CD , Lrp , DHCP MMC , GadX Lrp Membrane protein , protein export ( b1051 ) Hypothetical ( adherence ) protein ( b0232 ) Hypothetical protein ( b0329 ) Hypothetical ( inner membrane ) protein ( b0707 ) Putative enzyme ( b0865 ) Hypothetical protein , bioﬁlm dependent ( b1481 ) Hypothetical protein ( b1724 ) Hypothetical protein ( b2427 ) Putative dehydrogenase protein ( b2660 ) Putative immunogenic protein ( b2922 ) Putative transport periplasmic protein ( b3020 ) Hypothetical ( outer membrane ) protein ( b3024 ) Hypothetical ( regulatory ) protein ( b3555 ) Hypothetical protein ( b4045 ) Hypothetical ( membrane ) protein ( b1582 ) Hypothetical ( periplasmic ) protein ( b3097 ) Hypothetical protein ( b4217 ) MF ( MF ) rssBTC , baeSRTC , atoSCTC , Lrp ybjP yddX 2.5 1.4 4.2 4.2 1,317 376 ( EF ) rssBTC , OS ompR/envZTC ydiZ yfeT ygaF yggE ygiS 1.7 0.9 0.5 1.7 0.8 3.1 3.5 2.8 4.8 2.7 436 rssBTC , NarXLTC rssBTC , MMC ; Rcs , phoQP ( EF ) ( MF ) rssBTC , citABTC , arcATC , NaSal ( MF ) ygiW 1.9 4.0 892 rssBTC , ntrBCTC , NaSal yiaG yjbJ ynfA yqjC ytfK 0.5 1.2 1.1 1.3 3.0 2.7 4.5 3.5 5.9 3.9 1,227 601 3,334 1,121 556 ( MF ) PQ , Fur , BG a SSR , starvation and stress response ; PS , protein synthesis ; AATM , amino acid transport and metabolism ; IUS , iron uptake and storage ; CHM , carbohydrate metabolism ; NM , nucleoside metabolism ; CB , cofactor biosynthesis ; MF , membrane function ; EF , extracellular function .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15489429" target="_blank">15489429</a>]<br></li>
<li>More than 30 proteins are altered in the hfq null mutant ; some of these proteins are Sdependent genes Involved in fourth step of Lhistidine from 5-phospho-alpha-D-ribose 1-diphosphate P0ABC3 Protein HﬂC 6.3 / 37,649.88 179 Hfq 204 P0A6X3 Protein 7.20 / 11,035.19 HisA P10371 4.94 / 26,032.65 1 - ( 5-Phosphoribosyl ) -5 - ( 5-phosphoribosylamino ) methylideneamino imidazole-4-carboxamide isomerase Histidine biosynthesis bifunctional protein Histidinol-phosphate aminotransferase Histidinol dehydrogenase 4.99 / 24,831 ( DIGE 4.5 -- 6.5 ) 5.46 / 23,581 ( DIGE 4.5 -- 6.5 ) 179 , 321 5.76 / 40,277.96 5.43 / 35,112 See HisA : steps 6 and 8 286 P06987 HisB See HisA : step 7 ; induced by high pH during anaerobic-growth See HisA : ﬁnal step ; catalyzes the sequential NAD-dependent oxidations of L-histidinol to Lhistidinaldehyde and then to L-histidine ; induced by high pH during anaerobic-growth See HisA : step 5 5.01 / 39,360.14 P06986 HisC 323 HisD P06988 5.06 / 45,979.15 5.07 / 48,837 ( DIGE 4.5 -- 6.5 ) 179 , 321 , 323 P60664 HisF 5.03 / 28,454.45 179 Imidazole glycerol phosphate synthase subunit ATP phosphoribosyltransferase Has a crucial role in the pathway because the rate of histidine biosynthesis seems to be controlled primarily by regulation of hisG enzymatic activity ; feedback inhibited by histidine or inhibited by AMP and N -5 - phosphoribosyl-ATP See HisA : step 1 HisG 5.47 / 33,366.72 P60757 179 , 321 E 4.5 -- 6.5 ) IG 5.40 / 32,959 ( D HisH 287 P60595 5.33 / 21,652.86 5.24 / 27,057 ( 5 -- 6 ) Imidazole glycerol phosphate synthase subunit Histidine biosynthesis bifunctional protein HisIE Histidine-binding periplasmic protein HisI P06989 See HisA : steps 2 and 3 179 5.24 / 22,755.82 Component of the high-afﬁnity histidine permease , a binding-protein-dependent transport system ( HisJ , HisQ , HisM , and HisP ) HisJ 179 , 286 , 287 , 321 5.17 / 26,232.61 5.05 / 28,668 P0AEU0 5.28 / 27,969 ( DIGE 4.5 -- 6.5 ) 5.14 / 27,701 ( DIGE 4.5 -- 6.5 ) 4.99 / 10,461 ( 4 -- 5 ) 5.14 / 28,391 ( 4.5 -- 5.5 ) 5.59 / 50,642 HisS HldE 5.65 / 46,898.25 5.29 / 51,050.62 29 , 179 , 228 , 294 179 Histidyl-tRNA synthetase Bifunctional protein P60906 P76658 Catalyzes the phosphorylation of D-glycero-D-manno-heptose 7-phosphate at the C-1 position to form d,D-heptose-1 ,7 - bisphosphate ; also catalyzes the ADP transfer to D-glycero-D-manno-heptose 1-phosphate , yielding ADP-d , D-heptose Involved in NO detoxiﬁcation in an aerobic process , termed nitric oxide dioxygenase reaction , that utilizes O2 and NAD ( P ) H to convert NO to nitrate , which protects the bacterium from various noxious nitrogen compounds ; induced by nitric oxide NO ( under aerobic conditions ) , nitrite , nitrate ( under anaerobic conditions ) , nitroso compounds , and paraquat ; induced by low pH during anaerobic-growth HisS HldE 5.65 / 46,898.25 5.29 / 51,050.62 29 , 179 , 228 , 294 179 Histidyl-tRNA synthetase Bifunctional protein P60906 P76658 Hmp 5.48 / 43,867.66 Flavohemoprotein P24232 323 Continued on facing pag Has possible histone-like function and may be a global transcriptional-regulator through its ability to-bind to curved DNA sequences , which are found in regions upstream of a certain subset of promoters ; subject to transcriptional autorepression by binding preferentially to the upstream region of its own gene recognizing two segments of DNA on both sides of a bend centered around position 150 ; the hns mutant exhibit derepression not only of csgBA , hdeAB , mcc , and osmY but also of the expression of at least 22 S-controlled genes , including S itself ; increases during phosphate limitation and phosphonate growth Involved in purine salvage pathway and acts exclusively on hypoxanthine Involved in the maturation of iron-sulfur cluster-containing proteins ; seems to help targeting proteins to be folded toward HscA Redox regulated molecular chaperone ; protects both thermally unfolding and oxidatively damaged proteins from irreversible aggregation ; plays an important role in the bacterial defense system toward oxidative-stress ; induced by heat-shock Chaperone subunit of a proteasome-like degradation complex ; increases following exposure to the uncoupler of oxidative phosphorylation 2,4-dinitrophenol Protease subunit of a proteasome-like degradation complex ; increases following exposure to the uncoupler of oxidative phosphorylation 2,4-dinitrophenol Molecular chaperone ; has ATPase activity ; increases following exposure to the uncoupler of oxidative phosphorylation 2,4-dinitrophenol but decreases during phosphate limitation late Participates in the periplasmic electron-transferring activity of hydrogenase 2 during its catalytic turnover Required for the maturation of the three NiFe hydrogenases ; exhibits a low intrinsic GTPase activity ; affects some aspect of the processing of hydrogenases 1 and 2 , perhaps nickel incorporation into the apoenzymes , since hypB gene lesions can be complemented by high nickel ion concn in the medium TABLE 2 -- Continued Hns P0ACF8 DNA-binding protein H-NS 5.44 / 15,408.44 5.45 / 15,612 5.00 / 12,390 5.38 / 15,657 5.02 / 9,935 Hpt 5.09 / 20,115.24 5.38 / 12,316 ( 4.5 -- 5.5 ) 179 , 287 P0A9M2 Hypoxanthine phosphoribosyltransferase P0A6L9 Cochaperone protein 5.05 / 20,137.71 HscB 79 1 4.35 / 32,534.48 P0A6Y5 33-kDa chaperonin HslO 79 1 HslU 5.24 / 49,593.80 5.30 / 48,579 P0A6H5 ATP-dependent hsl protease ATP-binding subunit 71 , 179 , 228 , 294 5.95 / 18,961.66 5.93 / 22,161 P0A7B8 ATP-dependent protease HslV 71 , 228 , 294 HtpG 5.09 / 71,422.53 5.06 / 65,639 P0A6Z3 Chaperone protein 71 , 179 , 228 , 293 , 294 HybA 79 P0AAJ8 Hydrogenase-2 operon protein 6.38 / 33,263.79 6.38 / 33,263.79 1 P0AAN3 Hydrogenase isoenzymes nickel 5.42 / 31,565.04 6.32 / 32,224 incorporation protein HypB 286 Continued on following pag Required for the formation of all three hydrogenase isoenzymes Associates with aggregated proteins , together with IbpB , to stabilize and protect them from irreversible denaturation and extensive proteolysis during heat-shock and oxidative-stress ; aggregated proteins bound to the IbpAB complex are more efﬁciently refolded and reactivated by the ATP-dependent chaperone systems ClpB and DnaK/DnaJ/GrpE ; its activity is ATP independent ; IbpAB increase in overexpression of heterologous proteins and biopolymers See IbpA ; decreases during phosphate limitation early Inhibition of this enzyme by phosphorylation regulates the branch point between the Krebs cycle and the glyoxylate bypass , which is an alternate route that accumulates carbon for biosynthesis when acetate is the sole carbon source for growth ; increases during high-cell-density cultivation but decreases during phosphate limitation Fermentative lactate dehydrogenase ; increases during phosphate limitation and at low pH under anaerobic conditions Catalyzes the formation of alpha-ketobutyrate from threonine in isoleucine biosynthesis ; isoleucine allosterically inhibits whereas valine allosterically activates this enzyme ; increases during phosphate limitation and phosphonate growth Involved in valine and isoleucine biosynthesis : step 1 See IlvB : step 2 ; induced in the presence of acetohydroxybutyrate and acetolactate ; decreases after benzoic acid treatment See IlvB : step 3 ; increases after benzoic acid treatment See IlvB : step 4 ( ﬁnal ) ; acts on leucine , isoleucine , and valine ; decreases during phosphate limitation See IlvB : step 1 ; sensitive to valine inhibition Determines N-hexane tolerance ; involved in outer membrane permeability ; essential for envelope biogenesis ; could be part of a targeting/usher system for outer membrane components TABLE 2 -- Continued HypD P24192 5.80 / 41,363.38 5.81 / 41,711 Hydrogenase isoenzymes formation protein P0C054 Small heat-shock protein IbpA 5.57 / 15,773.71 5.35 / 15,600 5.68 / 15,600 4 , 97 , 98 , 228 , 287 , 296 5.63 / 15,600 5.63 / 15,384 ( 5 -- 6 ) 5.63 / 14,237 ( 5.5 -- 6.7 ) 5.19 / 16,093.20 P0C058 Small heat-shock protein IbpB 5.02 / 15,600 5.20 / 15,600 4 , 97 , 98 , 295 Icd 5.06 / 46,200 5.15 / 46,200 5.02 / 46,100 5.11 / 44,335 ( DIGE 4.5 -- 6.5 ) 5.15 / 44,053 ( DIGE 4.5 -- 6.5 ) 4.70 / 22,351 ( 4 -- 5 ) 5.15 / 43,951 ( 4.5 -- 5.5 ) 5.00 / 74,518 ( 5 -- 6 ) 5.15 / 45,756.71 P08200 Isocitrate dehydrogenase ( NADP ) 179 , 228 , 236 , 287 , 293 , 294 , 321 , 325 P52643 IdhA 5.29 / 36,534.79 5.59 / 34,100 D-Lactate dehydrogenase 128 , 293 5.57 / 56,195.25 6.16 / 54,999 Threonine dehydratase biosynthetic P04968 IlvA 293 287 5.22 / 56,152 ( 4.5 -- 5.5 ) P08142 5.30 / 60,440.57 I e Acetolactate synthase isozym large subunit Ketol-acid reductoisomerase IlvB 5.26 / 52,066 5.20 / 53,937.83 P05793 IlvC 286 , 321 5.22 / 52,046 ( DIGE 4.5 -- 6.5 ) 5.29 / 51,264 ( DIGE 4.5 -- 6.5 ) 321 5.54 / 62,065 ( DIGE 4.5 -- 6.5 ) 5.59 / 65,400.37 P05791 Dihydroxy-acid dehydratase IlvD 5.43 / 35,112 5.54 / 33,962.46 P0AB80 Branched-chain amino acid aminotransferase IlvE 228 , 236 , 293 , 294 287 8.11 / 15,587 ( 6 -- 11 ) 8.01 / 17,976.76 e Acetolactate synthase isozym III small subunit Organic solvent tolerance protein P00894 IlvH Imp 4.91 / 88,368 ( DIGE 4.5 -- 6.5 ) 4.85 / 87,068.95 P31554 179 , 321 Continued on facing pag IscA P0AAC8 Iron-binding protein ( YfhF ) 4.75 / 11,556.05 IscU 179 Involved in Fe-S biosynthesis 4.82 / 13,848.59 IspA IspG Geranyltranstransferase 5.27 / 32,159.63 5.37 / 26,625 4-Hydroxy-3-methylbut-2-en-1-yl 5.87 / 40,683.61 diphosphate synthase P22939 P62620 179 , 286 179 5.2 / 34,774.56 4-Hydroxy-3-methylbut-2-enyl diphosphate reductase P62623 IspH 179 5.46 / 15,065 5.41 / 14,304 ( 5 -- 6 ) 5.51 / 14,102.90 Inhibitor of vertebrate lysozyme P0AD59 Ivy 228 , 287 5.33 / 12,548 ( 5 -- 6 ) 5.47 / 86,507 ( DIGE 4.5 -- 6.5 ) 103 , 321 5.54 / 84,162.61 Catalase HPII P21179 KatE Peroxidase/catalase HPI P13029 KatG 103 , 287 , 321 5.18 / 77,774 ( DIGE 4.5 -- 6.5 ) 5.16 / 74,539 ( 4.5 -- 5.5 ) 5.14 / 80,023.82 5.37 / 31,293.68 5.98 / 31,100 P0AB74 Tagatose-1 ,6 - bisphosphate aldolase AgaY Kba 99 2-Amino-3-ketobutyrate CoA ligase 2-Dehydro-3-deoxygluconokinase 2-Dehydro-3-deoxyphosphooctonate aldolase Could transfer iron-sulfur clusters to apo-ferredoxin ; multiple cycles of 2Fe2S cluster formation and transfer are observed ; acts catalytically ; recruits intracellular free iron so as to provide iron for the assembly of transient ironsulfur cluster in IscU in the presence of IscS , L-cysteine , and the thioredoxin reductase system TrxA/TrxB Not found in SWISS-PROT / TrEMBL database but available in NCBI database ( GI : 16130454 ) Converts 2C-methyl-D-erythritol 2,4-cyclodiphosphate into 1-hydroxy-2-methyl-2 - ( E ) - butenyl 4-diphosphate Converts 1-hydroxy-2-methyl-2 - ( E ) - butenyl 4-diphosphate into isopentenyl diphosphate and dimethylallyl diphosphate ; also involved in penicillin tolerance , control of the stringent response , and isoprenoid biosynthesis Strong inhibitor of lysozyme C Decomposes hydrogen peroxide into water and oxygen ; serves to protect cells from the toxic effects of hydrogen peroxide ; induced by entry into stationary phase and after benzoic acid treatment Bifunctional , exhibiting both catalase and broad-spectrum peroxidase activities ; induced by hydrogen peroxide and by entry into stationary phase but decreases after benzoic acid treatment Catalyzes the reversible condensation of dihydroxyacetone phosphate with glyceraldehyde 3-phosphate to produce tagatose-1 ,6 - bisphosphate ; repressed in the presence of glucose Negatively regulated by Lrp Kba 99 - / 42,200 5.64 / 43,117.04 P0AB77 61 Kbl 4.92 / 33,962.36 6.32 / 30,832.69 P37647 P0A715 179 KdgK KdsA Synthesis of 2-keto-3 - 179 , 287 6.10 / 33,261 ( 6 -- 11 ) deoxyoctulosonic acid 8-phosphate , which is required for lipid A maturation and cellular growth Activates 2-keto-3-deoxyoctulosonic acid for incorporation into bacterial lipopolysaccharide in gramnegative bacteria Involved in the transport of maltose and maltodextrins ; indispensable for translocation of dextrins containing more than three glucosyl moieties ; acts as a receptor for several bacteriophages including lambda ; induced by maltose 5.15 / 27,483.25 5.06 / 33,695 3-Deoxy-manno-octulosonate cytidylyltransferase P04951 KdsB 228 , 294 LamB 79 4.72 / 47,385.03 Maltoporin P02943 1 Continued on following pag Fermentative lactate dehydrogenase ; increases during phosphate limitation and at low pH under anaerobic conditions Catalyzes the condensation of the acetyl group of acetyl-CoA with 3-methyl-2-oxobutanoate ( 2-oxoisovalerate ) to form 3-carboxy-3-hydroxy-4-methylpentanoate ( 2-isopropylmalate ) ; may be directly degraded by ClpAP and ClpXP , respectively , or be modulated by a protease-dependent mechanism Catalyzes the isomerization between 2-isopropylmalate and 3-isopropylmalate via the formation of 2-isopropylmaleate LdhA P52643 D-Lactate dehydrogenase 5.29 / 36,534.79 5.59 / 34,100 P09151 2-Isopropylmalate synthase LeuA 308 5.47 / 57,166.71 LeuC 3-Isopropylmalate dehydratase large subunit P0A6A6 5.90 / 49,750.64 5.95 / 51,656 5.77 / 51,451 179 , 286 , 321 5.42 / 44,439 5.93 / 50,340 ( DIGE 4.5 -- 6.5 ) See LeuC ; increases after benzoic acid treatment Increases late during phosphate limitation Changes very little throughout the normal temp ( 23 -- 37 ° C ) and increases in level with increasing growth rate Represses a no .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16760308" target="_blank">16760308</a>]<br></li>
<li>of T-even like phages and also acts as a porin with low permeability that allows slow penetration of small solutes ; induced by low pH during aerobic or anaerobic-growth but decreases by phosphate limitation and SOS induction Forms passive diffusion pores which allow small-molecular-wt hydrophilic materials across the outer membrane ; negatively regulated by Lrp Forms passive diffusion pores which allow low-mol wt hydrophilic materials across the outer membrane ; receptor for the bacteriophage T2 ; positively regulated by Lrp ; decreases during phosphate limitation and after benzoic acid treatment and treatment with superoxidegenerating agents such as menadione P33602 NuoG NusA 4.53 / 54,870.92 4.55 / 61,127 P0AFF6 133 , 179 , 212 , 228 , 294 P0A780 N utilization substance protein B 6.6 / 15,689.06 NusB 179 NusG 6.33 / 20,400.32 5.93 / 22,161 5.44 / 14,684 ( 5 -- 6 ) 6.40 / 21,860 ( 6 -- 11 ) P0AFG0 Transcription antitermination protein 179 , 286 , 287 OmpA 5.55 / 34,419 5.36 / 32,707 5.60 / 35,172.28 P0A910 Outer membrane protein A 75 , 179 , 228 , 274 , 287 , 293 , 294 , 321 , 323 5.34 / 28,591 5.50 / 33,875 ( DIGE 4.5 -- 6.5 ) 5.38 / 33,875 ( DIGE 4.5 -- 6.5 ) 5.23 / 14,346 ( 5 -- 6 ) 5.25 / 13,108 ( 5 -- 6 ) 4.48 / 38,307.50 -- / 36,000 Outer membrane protein C P06996 OmpC 61 OmpF 4.61 / 36,170 4.64 / 37,084.49 Outer membrane protein F P02931 61 , 88 , 179 , 228 , 293 , 294 , 321 4.56 / 37,316 ( DIGE 4.5 -- 6.5 ) 4.64 / 37,000 ( DIGE 4.5 -- 6.5 ) 4.71 / 37,000 ( DIGE 4.5 -- 6.5 ) Continued on facing pag The N terminus of this protein is required for the transcriptional expression of both major outer membrane protein genes ompF and ompC ; its carboxyl-terminal moiety mediates the multimerization of the ompR protein Protease that can cleave T7 RNA polymerase , ferric enterobactin receptor protein ( FEP ) , antimicrobial peptide protamine , and other proteins Induced by low or high pH 79 OmpR P0AA16 Transcriptional regulatory protein 6.04 / 27,353.62 1 OmpT Protease VII P09169 179 , 321 5.38 / 33,477.72 5.36 / 37,000 ( DIGE 4.5 -- 6.5 ) OmpX Outer membrane protein X P0A917 5.30 / 16,382.89 5.25 / 16,126 ( DIGE 4.5 -- 6.5 ) 5.15 / 16,000 ( DIGE 4.5 -- 6.5 ) 5.93 / 56,137 5.70 / 57,487 5.64 / 56,808 5.44 / 54,493 5.93 / 55,635 ( DIGE 4.5 -- 6.5 ) 5.82 / 55,131 ( DIGE 4.5 -- 6.5 ) 5.71 / 52,681 ( DIGE 4.5 -- 6.5 ) 5.13 / 50,973 ( 5 -- 6 ) 5.47 / 50,921 ( 5 -- 6 ) 5.63 / 50,921 ( 5 -- 6 ) 5.65 / 50,915 ( 5.5 -- 6.7 ) 179 , 274 , 321 Component of the oligopeptide permease and binding-protein-dependent transport system ; binds peptides up to 5 amino acids long with high afﬁnity ; increases by high pH during anaerobic-growth and after benzoic acid treatment OppA 5.85 / 58,359.84 Periplasmic oligopeptide-binding protein P23843 179 , 228 , 286 , 287 , 294 , 321 , 323 Part of the binding-protein-dependent transport system for oligopeptides ; probably responsible for energy coupling to the transport system Provides resistance to osmotic stress ; may be important for stationary-phase survival ; induced by elevated osmotic pressure in the growth medium Preferentially metabolizes organic hydroperoxides over inorganic hydrogen peroxide ; induced by elevated osmotic pressure and during the decelerating phase , before entry into stationary phase ; regulated by a complex network of several global regulators , including at least S , Lrp , and H-NS Induced by elevated osmotic pressure Osmotically and stationary phase inducible and controlled by several global regulators , including S , cAMP-CRP , IHF , and Lrp Induced by growth in glucose-mineral medium of elevated osmotic strength ; controlled by S Induced by growth in glucose-mineral medium of elevated osmotic strength ; controlled by S May play a role in bacterial envelope integrity ; strongly associated with the peptidoglycan Involved in pantothenate biosynthesis : ﬁrst step See PanB : ﬁnal step .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16760308" target="_blank">16760308</a>]<br></li>
<li>Acknowledgements This work was supported by Public Health Service Grant GM39496 from the National Institutes of Health to J.M.C. and by the CNR ( Target projects ` Biotechnology and Bioinstrumentation ' and ` Genetic Engineering ' ) and Ministero Agricoltura e Foreste ( Piano Nazionale di Receroa ` Sviluppo di tecnologie avanzate applicate alle piante ' ) to M.D.F , We thank D , A. Willins for purified Lrp , and P , Dervan for a gift of MPE , References Barrera-Saldana , H , , Takahashi , K , , Vigneron , M. , Wildeman , A , , Davidson , I. , and Chambon , P , ( 1985 ) All six GC-motifs of the SV40 early upstream element contribute to promoter activity in vivo and in vitro , EiVlBOJA : 3839-3849 , DeFelice , M. , Lago , CT , , Squires , C , H , , and Calvo , J.M. ( 1982 ) Acetohydroxy acid synthase isoenzymes of Escherichia coii K12 and Saimoneiia typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8483418" target="_blank">8483418</a>]<br></li>
<li>Nonstandard reaction conditions differed from standard reaction conditions in containing 2 nM PstI-cut pCV7 , 295 nM Lrp , 0.08 U of RNAP per pl , and 110 mM NaCl instead of potassium glutamate in the transcription buffer ; also , preincubation of Lrp with the DNA template was omitted .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1447135" target="_blank">1447135</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1447135" target="_blank">1447135</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1447135" target="_blank">1447135</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_29">
Zhi , J. , Mathew , E. , and Freundlich , M. ( 1999 ) Lrp binds to two regions in the dadAX promoter region of Escherichia coli to repress and activate transcription directly .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19906180" target="_blank">19906180</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11985712" target="_blank">11985712</a>]<br></label>
<input id="act_similarity_29", type="checkbox">
<div class="ACT_similar">
<li>Zhi , J. ( 1997 ) Lrp binds to two regions in the dadAX promoter region of Escherichia coli to directly repress and activate transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10216857" target="_blank">10216857</a>]<br></li>
<li>Lrp binds to two regions in the dadAX promoter region of Escherichia coli to repress and activate transcription directly .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2162465" target="_blank">2162465</a>]<br></li></div>
<li>This model can be applied to all Lrp-regulated promoters in that Lrp can bind to the promoter DNA and either repress or activate transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1917830" target="_blank">1917830</a>]<br></li>
<li>Platko and Calvo ( 1993 ) have isolated several mutations affecting the ability of Lrp to-bind DNA , activate transcription , or respond to leucine , on the basis of their effects on the ilvIH gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7603430" target="_blank">7603430</a>]<br></li>
<li>Annu Rev Biochem 53:293 -321 Platko JV , Calvo JM ( 1993 ) Mutations affecting the ability of Escherichia coli Lrp to-bind DNA , activate transcription , or respond to leucine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7603430" target="_blank">7603430</a>]<br></li>
<li>Mol Microbiol 7:523 -536 Platko J , Calvo J ( 1993 ) Mutations affecting the ability of Escherichia coli Lrp to-bind DNA , activate transcription , or respond to leucine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8152423" target="_blank">8152423</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_36">
Mutations affecting the ability of Eschenichia coli Lrp to-bind DNA , activate transcription , or respond to leucine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8144448" target="_blank">8144448</a>]<br></label>
<input id="act_similarity_36", type="checkbox">
<div class="ACT_similar">
<li>Mutations affecting the ability of Escherichia coli Lrp to-bind DNA , activate transcription , or respond to leucine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8071194" target="_blank">8071194</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2162465" target="_blank">2162465</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11575919" target="_blank">11575919</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19074390" target="_blank">19074390</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9236118" target="_blank">9236118</a>]<br></li></div>
<li>The Leucine-Responsive Regulatory Protein , a Global Regulator of Metabolism in Escherichia coli TRODUCTION ... 4 VOLVEMENT OF LRP AND LEUCINE IN REGULATING OPERON EXPRESSION ... 4 Properties of Lrp ... 4 Interaction of Lrp with DNA ... 4 Consensus sequence ... 4 Stoichiometry and cooperativity of binding ... 4 Bending caused by binding ... 4 Lrp binding can be affected by DNA methylation and vice versa ... 4 Organization and Regulation of the lrp Operon ... 4 Organization of the lrp operon ... 4 Autogenous regulation ... 4 Effect of growth conditions on lrp expression ... 4 Effects of Lrp on Operon Expression ... 4 Direct versus indirect effects ... 4 Can Lrp act from a distance ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li>
<li>Lrp stimulates phase variation of type 1fimbriation in Escherichia coil K-12 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8402918" target="_blank">8402918</a>]<br></li>
<li>Mol Gen Genet 233:184 -192 BlomfieldI , CalieP , Eberhardt K , McClain M , EisensteinB ( 1993 '' Lrp stimulates phase variation of type 1 fimbriation in cherichia coli K-12 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8152423" target="_blank">8152423</a>]<br></li>
<li>J Bacteriol 176:5537 ± 5540 Blom ® eld IC , Calie PJ , Eberhardt KJ , McClain MS , Eisenstein BI ( 1993 ) Lrp stimulates the phase variation of type 1 ® mbriae in Escherichia coli K-12 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9749677" target="_blank">9749677</a>]<br></li>
<li>Bacteriol 173:5298 -5307 Blomfield IC , Calie PJ , Eberhardt KJ , McClain MS , Eisenstein BI ( 1993 ) Lrp stimulates phase variation of type 1 fimbriation in Escherichia coli K-12 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8121417" target="_blank">8121417</a>]<br></li>
<li>Nou , X. , Skinner , B. , Braaten , B. , Blyn , L. , Hirsch , D. , and Low , D.A. ( 1993 ) Regulation of pyelonephritis-associated pili phase-variation in Escherichia coli : binding of the PapI and the Lrp regulatory proteins is controlled by DNA methylation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17238917" target="_blank">17238917</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_44">
Regulation of pyelonephritis-associated pili phase-varia-tion in Escherichia coli : binding of the PapI and the Lrp regulatory proteins is controlled by DNA methylation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></label>
<input id="act_similarity_44", type="checkbox">
<div class="ACT_similar">
<li>Regulation of pyelonephritis-associ-ated pili phase-variation in Escherichia coli : binding of the PapI and the Lrp regulatory proteins is controlled by DNA methylation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9512707" target="_blank">9512707</a>]<br></li>
<li>Regulation of pyelonephritis-associated pili phase-variation in Escherichia coli : binding of the PapI and the Lrp regulatory proteins is controlled by DNA methylation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7961507" target="_blank">7961507</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8002608" target="_blank">8002608</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8144448" target="_blank">8144448</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8195106" target="_blank">8195106</a>]<br></li></div>
				<label class="ACT_collapse_similarity" for="act_similarity_47">
Nou , X. , Braaten , B. , Kaltenbach , L. , and Low , D. ( 1995 ) Differential binding of Lrp to two sets of pap DNA binding sites mediated by PapI regulates Pap phase variation in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10692164" target="_blank">10692164</a>]<br></label>
<input id="act_similarity_47", type="checkbox">
<div class="ACT_similar">
<li>Nou , X. , Braaten , B. , Kaltenbach , L. , and Low , D.A. ( 1995 ) Differential binding of Lrp to two sets of pap DNA binding sites mediated by PapI regulates phase variation in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10216857" target="_blank">10216857</a>]<br></li></div>
				<label class="ACT_collapse_similarity" for="act_similarity_49">
Differential binding of Lrp to two sets of Pap DNA binding sites mediated by Pap I regulates Pap phase variation in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11872702" target="_blank">11872702</a>]<br></label>
<input id="act_similarity_49", type="checkbox">
<div class="ACT_similar">
<li>Differential binding of Lrp to two sets of pap DNA binding sites mediated by PapI regulates Pap phase variation in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955407" target="_blank">8955407</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9811649" target="_blank">9811649</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9236118" target="_blank">9236118</a>]<br></li></div>
<li>For ilvIH , one of the operons positively regulated by Lrp , Lrp binds to multiple sites upstream of the transcriptional start site and activates transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7665463" target="_blank">7665463</a>]<br></li>
<li>Wang , Q. and Calvo , J.M , ( 1993 ) Lrp , a global regulatory protein of Escherichia coii , binds co-operatively to multiple sites and activates transcription of iiviH .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8483418" target="_blank">8483418</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_53">
Wang , Q. , and Calvo , J.M. ( 1993 ) Lrp , a global regulatory protein of E. coli binds cooperatively to multiple sites and activates transcription of ilvIH .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9663683" target="_blank">9663683</a>]<br></label>
<input id="act_similarity_53", type="checkbox">
<div class="ACT_similar">
<li>Wang , Q. , and Calvo , J.M. ( 1993 ) Lrp , a global regulatory protein of Escherichia coli , binds co-operatively to multiple sites and activates transcription of ilvIH .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10216857" target="_blank">10216857</a>]<br></li>
<li>Wang , Q. , and Calvo , J.M. ( 1993a ) Lrp , a global regulatory protein of Escherichia coli , binds co-operatively to multiple sites and activates transcription of ilvIH .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11985712" target="_blank">11985712</a>]<br></li>
<li>Wang , Q. , and Calvo , J.M. ( 1993b ) Lrp , a global regulatory protein of Escherichia coli , binds co-operatively to multiple sites and activates transcription of ilvIH .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9515701" target="_blank">9515701</a>]<br></li>
<li>Lrp , a global regulatory protein of Escherichia coli , binds co-operatively to multiple sites and activates transcription of ilvZH .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9611815" target="_blank">9611815</a>]<br></li>
<li>Lrp , a global regulatory protein of Escherichia coli , binds co-operatively to multipl ; sites and activates transcription of ilvZH .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9802028" target="_blank">9802028</a>]<br></li></div>
				<label class="ACT_collapse_similarity" for="act_similarity_59">
Lrp , a global regulat-ory protein of E. coli , binds cooperatively to multiple sites and activates transcription of ilvIH .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9236118" target="_blank">9236118</a>]<br></label>
<input id="act_similarity_59", type="checkbox">
<div class="ACT_similar">
<li>Lrp , a global regulatory protein of Esch-erichia coli , binds cooperatively to multiple sites and activates transcription of ilvIH .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7592387" target="_blank">7592387</a>]<br></li>
<li>Lrp , a global regulatory protein of E. coli , binds cooperatively to multiple sites and activates transcription of ilvIH .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8144448" target="_blank">8144448</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11575919" target="_blank">11575919</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7901196" target="_blank">7901196</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12218014" target="_blank">12218014</a>]<br></li></div>
				<label class="ACT_collapse_similarity" for="act_similarity_62">
Lrp , a global regulat-ory protein of Escherichia coli , binds co-operatively to multiple sites and activates transcription of ilvIH .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9512707" target="_blank">9512707</a>]<br></label>
<input id="act_similarity_62", type="checkbox">
<div class="ACT_similar">
<li>Lrp , a global regulatory protein of Escherichia col , binds co-operatively to multiple sites and activates transcription of ilvIH .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7928983" target="_blank">7928983</a>]<br></li>
<li>Lrp , a global regulatory protein of Escherichia coli , binds co-operatively to multiple sites and activates transcription of ilvIH .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8071194" target="_blank">8071194</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8253679" target="_blank">8253679</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7916011" target="_blank">7916011</a>]<br></li>
<li>Lrp , a global regulatory protein of Escbericbia coli , binds co-operatively to multiple sites and activates transcription of ilvIH .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7881552" target="_blank">7881552</a>]<br></li>
<li>Lrp , a global regulatory protein of Esch-erichia coli , binds co-operatively to multiple sites and activates transcription of ilvIH .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955407" target="_blank">8955407</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8002608" target="_blank">8002608</a>]<br></li></div>
				<label class="ACT_collapse_similarity" for="act_similarity_67">
Man , T.K. , Pease , A.J. , and Winkler , M.E. ( 1997 ) Maximi-zation of transcription of the serC ( pdxF ) -- aroA multifunctional operon by antagonistic effects of the cyclic AMP ( cAMP ) receptor protein -- cAMP complex and Lrp Global regulators of Escherichia coli K-12 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11985712" target="_blank">11985712</a>]<br></label>
<input id="act_similarity_67", type="checkbox">
<div class="ACT_similar">
<li>Maximization of transcription of the serC ( pdxF ) - aroA multifunctional operon by antagonistic effects of the cyclic AMP ( cAMP ) receptor protein-cAMP complex and Lrp global regulators of Escherichia coli K-12 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11844765" target="_blank">11844765</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9393714" target="_blank">9393714</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9696782" target="_blank">9696782</a>]<br></li></div>
<li>Summary In vivo and in vitro analyses indicate that transcription of the argO gene coding for an arginine exporter is regulated by the global transcriptional-regulator Lrp , an effect that went by unnoticed in previous genome-scale screenings of the Lrp regulatory network in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19906180" target="_blank">19906180</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_70">
Zhi , J. , Mathew , E. , and Freundlich , M. ( 1998 ) In vitro and in vivo characterization of three major dadAX promoters in Escherichia coli that are regulated by cyclic AMP-CRP and Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10216857" target="_blank">10216857</a>]<br></label>
<input id="act_similarity_70", type="checkbox">
<div class="ACT_similar">
<li>In vitro and in vivo characterization of three major dadAX promoters in Escherichia coli that are regulated by cyclic AMP-CRP and Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2162465" target="_blank">2162465</a>]<br></li></div>
<li>Regulated expression of 21 small proteins Comparison of growth under the two indicated conditions Minimal glucose vs minimal glycerol Proteina M b ap c LB vs Aerobic vs Mock treatment vs pH 7.5 vs 30 ° C vs 37 ° C vs Mock Mock low oxygen SDS EDTA pH 5.5 45 ° C 10 ° C treatment treatment vs vs H2O2 diamide RegulonDB minimal glucose YpfM YccB Monocistronic None d appAB-yccB - 38 , 70 , AppY , appC ArcA Monocistronic None Monocistronic None cydAB-ybgTE 70 , ArcA , Fnr , FruR , HNS ykgMO 70 , Zur yqgB-speAB None Monocistronic None yjhBC-ythA None Monocistronic NL yqeKL None Monocistronic NL Monocistronic NL ilvXGMEDA 70 , Lrp , IHF yeaP-yoaKJ None Monocistronic NL yeaP-yoaKJ None yoeI-yeeF None Monocistronic None In IsrB sRNA NL yobF-cspC None YncL YneM - - e YbgT - ¬ YkgO YqgB YnhF YthA YobI YqeL YohP YkgR IlvX YoaJ YpdK YoaK YoeI YbhT AzuC YobF - - - - - ¬ - ¬ - ¬ - - - ¬ - - ¬ - - - ¬ - ¬ - -- ¬ -- - ¬ - ¬ - ¬ - - ¬ a All small proteins that had at least a 4-fold change in protein levels in at least one experiment are listed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19734316" target="_blank">19734316</a>]<br></li>
<li>Lrp is at a maximum cellular concentration in bacteria growing exponentially in MMGlu and decreases about 4 - to 10-fold in bacteria in amino acid-rich media and LB ( 4 , 32 , 44 , 45 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9171388" target="_blank">9171388</a>]<br></li>
<li>For example , expression of the serC ( pdxF ) - encoded transaminase , which acts between PdxB and PdxA in the de novo PLP pathway ( 40 , 43 ) , is minimal in rich LB medium and maximal in minimal salts-glucose medium due to opposing positive and negative controls exerted by the Lrp and cyclic AMP receptor proteins , respectively ( T.-K .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11844765" target="_blank">11844765</a>]<br></li>
<li>As cellular amounts of Lrp or CRP-cAMP drop or increase , respectively , in minimal salts medium containing glycerol instead of glucose or glucose plus amino acids or in rich LB ( 4 , 26 , 27 , 32 , 44 , 45 , 47 ) , serC ( pdxF ) - aroA transcript and SerC ( PdxF ) transaminase amounts drop to the minimal level detected in LB ( Fig. 5 to 8 ; Table 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9171388" target="_blank">9171388</a>]<br></li>
<li>One might question whether the three - to fourfold increase in Lrp levels that occurs as cells slow their growth rate could account for a major shift in some aspects of metabolism .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955316" target="_blank">8955316</a>]<br></li>
<li>This is likely a consequence of both the fourfold-lower levels of Lrp in rich medium and the fact that Lrp binds only relatively weakly to sites upstream of the lrp gene ( 36 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955316" target="_blank">8955316</a>]<br></li>
<li>In contrast , Lrp is present in cells at high concentrations ( 60 ) , and decreased concentrations of Lrp on entrance into a rich medium probably require either dilution by cell division or degradation of existing protein ( 12 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8002608" target="_blank">8002608</a>]<br></li>
<li>Lrp is required for glutamate synthase expression : lrp strains can not grow on media containing low levels of NH 4 and expression of a gltB : : lacZ fusion is increased 44-fold in an lrp strain as compared to an isogenic lrp strain when both strains are grown in glucose minimal medium ( Ernsting et al. , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9236118" target="_blank">9236118</a>]<br></li>
<li>© 2002 Blackwell Scienc Effects of Lrp in the regulation of the two serA promoters In the lrp mutant , serA expression in glucose-minimal medium is decreased to < 20 % , and to about 50 % in LB ( Lin et al. , 1990 ) , and an increase in Lrp synthesis causes a fourfold stimulation ( Table 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11985712" target="_blank">11985712</a>]<br></li>
<li>The reduced levels of gltBDF expression seen in rich media and in lrp strains may be causally connected if growth in rich media leads to reduced concentrations of Lrp ; evidence on this point will be presented in the Discussion section .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7901196" target="_blank">7901196</a>]<br></li>
<li>The rate offimE-promoted switching was the same in both the supplemented minimal medium an of lysU by Lrp showed that elevated expression of a lysU-lacZ translational fusion after a temperature shift was seen in both lrp + and lrp strains ( 69 ) , so Lrp does not appear to be involved in induction of lysU by heat-shock .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li>
<li>The small protein IlvX , which is encoded in the ilvXGMEDA isoleucine biosynthesis operon regulated by Lrp ( 35 ) , shows the expected increase in protein levels in cells grown in minimal medium ( 8-fold ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19734316" target="_blank">19734316</a>]<br></li>
<li>The serAP2 promoter showed low expression in the presence of Lrp in all the growth conditions tested , accounting for < 5 % of b-galactosidase activity in glucose-minimal medium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11985712" target="_blank">11985712</a>]<br></li>
<li>The Lrp mutant can grow in minimal medium , but its enzyme content is not optimized for this culture condition .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1532173" target="_blank">1532173</a>]<br></li>
<li>The enhanced expression of stpA in minimal medium is more readily explicable , being dependent on trans activation by the Lrp protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9006049" target="_blank">9006049</a>]<br></li>
<li>Since the levels of Lrp in LB are known to be decreased compared with those in glucose minimal media ( 6 , 12 , 27 ) , it will be interesting to determine whether the effect of LB on micF expression requires a functional Lrp protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8002608" target="_blank">8002608</a>]<br></li>
<li>Continued growth in rich medium would lead to further reductions in gltBDF expression as the concentration of Lrp in the cells decreased as a result of protein degradation and/or dilution during cell division .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li>
<li>Another global regulator is the Lrp protein , which controls gene expression upon transition from rich to depleted medium and upon the reverse transition [ 27 , 44 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17025177" target="_blank">17025177</a>]<br></li>
<li>In LB-grown cells , lrp expression was very much decreased , suggesting that autoregulation by Lrp must be overridden by other controls .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1532173" target="_blank">1532173</a>]<br></li>
<li>Transcription from serA : : lacZ is also decreased in the wild type grown in LB , which could in fact result from decreased synthesis of Lrp in LB-grown cells , as our earlier experiments showed that expression of the lrp gene is repressed during growth in rich medium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11985712" target="_blank">11985712</a>]<br></li>
<li>Their work showed that the expression of Lrp is more than 10-fold lower when cells are grown in Luria broth ( LB ) than when they are grown in a minimal medium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7901196" target="_blank">7901196</a>]<br></li>
<li>To evaluate the capacity of Lrp to-bind to the PcadBA pro-grown in glucose minimal medium , and after 2.5 h 0.2 % arabinose ( wt/vol ) was added to induce the expression of the lrp gene cloned in pBAD24 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21441513" target="_blank">21441513</a>]<br></li>
<li>These considerations , together with the fact that lrp expression is repressed during growth in a rich medium , suggest a model for the regulation of genes in the Lrp regulon during shifts from minimal to rich media .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li>
<li>Lrp is known to repress its own synthesis ( 19 , 36 ) , and it was previously shown that the low lrp expression level seen in rich medium is independent of Lrp ( 19 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955316" target="_blank">8955316</a>]<br></li>
<li>However , in rich medium , when levels of Lrp are reduced , we have found that the lower level and more transient expression of stpA is independent of Lrp ( 15a ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9006049" target="_blank">9006049</a>]<br></li>
<li>An absence of Lrp results in an increase in the amount of RpoS during exponential-growth in minimal medium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9663683" target="_blank">9663683</a>]<br></li>
<li>In the absence of rifampin , there was a marked increase in the rate of product formation , suggestin thetase isozyme that is normally cryptic or expressed at very low levels during growth-on-glucose minimal medium and that the expression of this isozyme is normally repressed by Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li>
<li>Although we still know little about the factors that regulate Lrp expression , it is likely that Lrp levels are higher in glucose minimal medium than in media containing nitrogen bases and amino acids .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li>
<li>This work demonstrates that Lrp and osmolarity exert their effects independently on ompC and ompF expression during growth in glucose minimal MOPS [ 3 - ( N-morpholino ) propanesulfonic acid ] medium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8002608" target="_blank">8002608</a>]<br></li>
<li>Although much remains to be learned about the role of Lrp in the regulation of one-carbon metabolism , the preliminary evidence suggests that this protein plays a major role in ensuring an adequate flux of carbon from glucose into serine biosynthesis during growth in glucose minimal medium and in ensuring that the catabolic pathway for the conversion of serine to pyruvate is activated during growth in a defined glucose-rich medium ( 163 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li>
<li>For strain CV1445 , although the b-galac-tosidase values were higher because of relief of Lrp repression , the stationary-phase effects were relatively unchanged ( 1.6-and 3.3-fold for minimal and rich medium , respectively ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955316" target="_blank">8955316</a>]<br></li>
<li>Stationary-phase induction of osmY in minimal medium can be explained by stimulation by cs combined with a relief of Lrp repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8253679" target="_blank">8253679</a>]<br></li>
<li>This indicates that in the rpoS mutant , stationary-phase induction in minimal medium consists in a relief of repression by Lrp that is maintained during exponential phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8253679" target="_blank">8253679</a>]<br></li>
<li>In wild-type cells grown in minimal medium , stationary-phase induction of osmY is only threefold ( Fig. 2 ) and as our analysis of osmY regulation in various mutants indicated , is brought about by a combination of stimulation by as and a relief of inhibition by Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8253679" target="_blank">8253679</a>]<br></li>
<li>In an rpoS background , a small induction of osmY-lacZ was still seen on entry into stationary phase during growth in M9 glucose medium , and this induction required a functional Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li>
<li>In another rich medium ( LB ) , activation of gltB by Lrp is reduced relative to minimal glucose , but is still about triple the level in a lrp disruptant [ 67 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18405378" target="_blank">18405378</a>]<br></li>
<li>We therefore asked whether Lrp contributes to the activation of stpA in minimal medium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9006049" target="_blank">9006049</a>]<br></li>
<li>Nevertheless , it is important to note that because of the extremely transient activation of stpA during growth in rich medium , we had to perform the latter experiment in minimal medium , when stpA is subject to trans activation by the Lrp protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9006049" target="_blank">9006049</a>]<br></li>
<li>Lrp was measured directly by Western blotting ( immunoblotting ) in cells grown in a chemically deﬁned rich medium or in a minimal medium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955316" target="_blank">8955316</a>]<br></li>
<li>We measured both b-galactosidase activities and the amounts of Lrp in extracts prepared from cells grown in minimal medium and in a chemically deﬁned rich medium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955316" target="_blank">8955316</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_112">
When cells in the log phase were compared , Lrp levels were 3 - to 4-fold higher in cells growing in a minimal medium than those in a rich medium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955316" target="_blank">8955316</a>]<br></label>
<input id="act_similarity_112", type="checkbox">
<div class="ACT_similar">
<li>For cells in log phase , Lrp levels were 3 - to 4-fold higher when cells were grown in minimal medium than those in cells grown in rich medium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955316" target="_blank">8955316</a>]<br></li></div>
<li>For cells in stationary phase , on the other hand , Lrp levels were only ap proximately 1.3-fold higher when cells were grown in minimal medium versus rich medium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955316" target="_blank">8955316</a>]<br></li>
<li>For cells in a minimal medium , Lrp levels were consistently lowest during the logarithmic phase of growth , but overall , there was not much variation in levels as a function of growth-phase ( 1.3-fold difference between highest and lowest values ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955316" target="_blank">8955316</a>]<br></li>
<li>We found that the Lrp levels vary signiﬁcantly as a function of growth-phase for cells grown in rich medium , but much less so for cells grown in minimal medium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955316" target="_blank">8955316</a>]<br></li>
<li>Lrp protein levels in all three species were reduced in rich medium relative to glucose minimal medium ( Fig. 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18405378" target="_blank">18405378</a>]<br></li>
<li>Cells in the logarithmic phase of growth show the largest differences in Lrp levels , which are three - to fourfold lower in rich than in minimal medium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955316" target="_blank">8955316</a>]<br></li>
<li>Furthermore , all three species showed severalfold lower Lrp levels in rich than in minimal medium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18405378" target="_blank">18405378</a>]<br></li>
<li>Transcription from lrp is decreased by Lrp and by growth in rich medium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1532173" target="_blank">1532173</a>]<br></li>
<li>One difference between growth in minimal medium and growth in rich medium is the presence of elevated levels of Lrp in the former case ( 23 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9006049" target="_blank">9006049</a>]<br></li>
<li>Measurement of Abundance of Lrp in E. coli-Strains CSH26 and CV1009 were grown in minimal medium at 37 `` C to an ODsso of 0.9 and 0.6 , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2040596" target="_blank">2040596</a>]<br></li>
<li>Our results confirm the earlier studies of E. coli , in that we saw several-fold higher Lrp levels when cells were grown in MOPS glucose than when they were grown in MOPS rich medium ( Fig. 8 , compare panels D and J ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18405378" target="_blank">18405378</a>]<br></li>
<li>For cells grown in minimal medium , both reporter gene activity and Lrp protein concentration were nearly constant throughout growth , being about 30 % lower during log phase than in stationary phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955316" target="_blank">8955316</a>]<br></li>
<li>Both reporter gene activity and Lrp protein concentration in log-phase cells were three - to fourfold lower than those for cells grown in minimal medium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955316" target="_blank">8955316</a>]<br></li>
<li>In minimal medium , both b-galactosidase speciﬁc activities ( per microgram of total protein ) and Lrp amounts ( in nano-grams per microgram of total protein ) decreased somewhat during the initial phase of growth before increasing again as the culture entered stationary phase ( Fig. 1A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955316" target="_blank">8955316</a>]<br></li>
<li>In E. coli , lrp mRNA levels are profoundly lower in rich than in minimal medium , irrespective of growth-phase , and similar to what was seen for Lrp protein ( compare black bars to Fig. 8 panels D and J ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18405378" target="_blank">18405378</a>]<br></li>
<li>The fact that a unique Lrp concentration yields the proper leucine sensitivities for both gltBDF and ilvIH suggests that this is the effective intracellular concentration of Lrp during steady-state growth in glucose minimal MOPS medium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7901196" target="_blank">7901196</a>]<br></li>
<li>The effective intracellular concentration of Lrp during exponential-growth in glucose minimal MOPS medium is 5.5 nM .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7901196" target="_blank">7901196</a>]<br></li>
<li>It was estimated that the effective intracellular Lrp concentration is approximately 5.5 nM in minimal medium ( 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7928983" target="_blank">7928983</a>]<br></li>
<li>During steady-state growth in LB , the level of gltBDF expression was about 16-fold lower than during growth in glucose minimal MOPS medium ( 44 ) , suggesting that a shift to LB resulted in a 3.5-fold decrease in the steady-state concentration of Lrp dimer .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li>
<li>Repression by Lrp was greater in cells grown in minimal medium than in LB medium ( 78 ) , consistent with the higher concentration of Lrp present in cells during exponential-growth in glucose minimal medium compared with LB ( 44 , 84 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li>
<li>Continued growth in a rich medium should lead to a decrease in the steady-state concentration of Lrp and thus to a continued decline in the transcription of positively regulated genes in the Lrp regulon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li>
<li>Consistent with such a proposed role of Lrp is the observation that Lrp is down regulated during rapid growth in LB medium compared with slower growth in minimal medium ( 25 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8253679" target="_blank">8253679</a>]<br></li>
<li>We conclude that the concentration of Lrp in LB-grown cells is considerably lower than that of cells grown in minimal medium and that genes requiring elevated concentrations of Lrp may be regulated by it only in minimal medium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1532173" target="_blank">1532173</a>]<br></li>
<li>However , we wondered whether the fact that serAP1P2 is transcribed less in LB ( 1550 units ) where there is some Lrp than serAP1P2 in the absence of Lrp in glucose-minimal medium ( 3600 ) might indicate that there is a second regulator that activates P2 in some conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11985712" target="_blank">11985712</a>]<br></li>
<li>It is clear that P1 is strongly activated by Lrp , and that most transcription in minimal medium comes from P1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11985712" target="_blank">11985712</a>]<br></li>
<li>These putative proteins could enhance repression of ompC and micF transcription by Lrp during growth in glucose minimal MOPS medium at 378C .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8002608" target="_blank">8002608</a>]<br></li>
<li>Binding of Lrp modifies the pattern of DNasel cleavage , particularly around 25-30 base pairs downstream of the transcriptional start site ; however , DNaselhypersensitive and protected bands can be detected also in the -35 / - 10 area and upstream , becoming more evident at higher Lrp concentrations ( Fig. 4 , even-numbered lanes ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8809748" target="_blank">8809748</a>]<br></li>
<li>Among the other 12 amino acids tested , comprising two previously reported potential effector-molecules of Lrp binding , L-alanine and L-methionine ( Mathew et al. , 1996 ; Zhi et al. , 1999 ; Berthiaume et al. , 2004 ) , and the effector-molecules of ArgP ( L-arginine and L-lysine ) , only methionine exerts a minor , but repeatedly observed , negative effect ( Fig. 3D ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19906180" target="_blank">19906180</a>]<br></li>
<li>In vivo and in vitro experiments have shown that these amino acids acted as inducers to antagonize Lrp repression of the multiple dad promoters ( Mathew et al. , 1996 ; Zhi et al. , 1998 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10216857" target="_blank">10216857</a>]<br></li>
<li>In place of Leu , Ala has been indicated to act as an effector of Lrp ( Berthiaume et al. , 2004 ; Kim et al. , 2010 ; Martin , 1996 ; Zhi et al. , 1998 , 1999 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2162465" target="_blank">2162465</a>]<br></li>
<li>Both modes of regulation by GcvA require at least two additional protein factors : ( i ) GcvR is a negative-acting protein that is hypothesized to interact with GcvA , antagonizing activation of gcvTHP by GcvA and promoting repression ( Ghrist and Stauffer , 1995a ) ; ( ii ) Lrp is a global regulatory protein affecting the control of many genes , including gcvTHP ( Lin et al. , 1992 ; Calvo and Matthews , 1994 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10972807" target="_blank">10972807</a>]<br></li>
<li>Transcription of the gcv operon is under the control of three global regulatory proteins [ Lrp , PurR and cAMP-receptor protein ( CRP ) ] and two gcv-speciﬁc transcriptional-regulatory proteins ( GcvA and GcvR ) ( Ghrist & Stauffer , 1995 ; Lin et al. , 1992 ; Stauffer & Stauffer , 1994 ; Wilson et al. , 1993a ; Wilson & Stauffer , 1994 ; Wonderling & Stauffer , 1999 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12101307" target="_blank">12101307</a>]<br></li>
<li>A certain proportion of the regulated genes are involved , as originally proposed , in transport and metabolism of amino acids , but Lrp has also been suggested to regulate genes involved in biosynthesis and degradation of various metabolites other than amino acids ( Brinkman et al. , 2003 ; Calvo & Matthews , 1994 ; Newman & Lin , 1995 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2162465" target="_blank">2162465</a>]<br></li>
<li>Lrp , a global regulator of numerous genes relating to amino acid metabolism ( Calvo & Matthews , 1994 ) , is required for induction of the gcv operon ( Lin et al. , 1992 ; Stauffer & Stauffer , 1994 ) , but is insensitive to the co-regulator leucine ( Ghrist & Stauffer , 1995b ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9802028" target="_blank">9802028</a>]<br></li>
<li>Lrp , a global transcriptional-regulator of genes involved in amino acid metabolism ( Lin et al. , 1992 ; Calvo & Matthews , 1994 ) , is required for both activation and repression of gcv expression ( Stauffer & Stauffer , 1994 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11495998" target="_blank">11495998</a>]<br></li>
<li>When leucine , which in the control of some genes acts as a positive or negative co-factor for regulation by Lrp ( Calvo & Matthews , 1994 ; Newman et al. , 1996 ) , was present in the growth medium , stationary phase induction of csiD was slightly stimulated , indicating that in the case of csiD regulation , leucine may act as a weak positive co-regu-lator ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9512707" target="_blank">9512707</a>]<br></li>
<li>Conversely , methylation of these sequences affects translocation of Lrp , and thus pap regulation ( Braaten et al. , 1994 ; Hale et al. , 1998 ; Weyand and Low , 1999 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10692164" target="_blank">10692164</a>]<br></li>
<li>Furthermore , such operons have been shown to be regulated by the global regulator Lrp , which is involved in the regulation of at least 30 different proteins in E.coli ( Ernsting et al. , 1992 ; reviewed in Newman et al. , 1992 ; D'Ari et al. , 1993 ; Calvo and Matthews , 1994 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8890170" target="_blank">8890170</a>]<br></li>
<li>For each gene in Table III the mode of regulation by Lrp ís known ( D'Ari el al. , 1993 ; Calvo and Matthews , 1994 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8521047" target="_blank">8521047</a>]<br></li>
<li>The activity of sS is ﬁne tuned by a number of global regulators , including CRP , Lrp , IHF , H-NS and Crl ( Hengge-Aronis , 1996b ; Loewen et al. , 1998 ; Pratt and Silhavy , 1998 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10594827" target="_blank">10594827</a>]<br></li>
<li>The requirements for Lrp and IHF ( especially in the absence of FNR ) are consistent with their greater abundance in slow-growing or stationary-phase organisms ( Landgraf et al. , 1996 ; Goosen & van de Putte , 1995 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11101674" target="_blank">11101674</a>]<br></li>
<li>The target DNA sites in different operons shown to be bound by Lrp or H-NS have features in common and are often naturally bent DNA regions ; several protein molecules may bind to such curved DNA ( Yamada et al. , 1990 , 1991 ; Newman et al. , 1992 ; Owen-Hughes et al. , 1992 ; Calvo and Matthews , 1994 ; Zuber et al. , 1994 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8890170" target="_blank">8890170</a>]<br></li>
<li>The regulatory function of Lrp was ﬁrst recognized under the control of Leu ( Chen & Calvo , 2002 ; Chen et al. , 2001a ; Haney et al. , 1992 ; Platko & Calvo , 1993 ; Roesch & Blomﬁeld , 1998 ; Willins et al. , 1991 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2162465" target="_blank">2162465</a>]<br></li>
<li>The effector Leu modulates multimerization of Lrp and thereby controls the transcription of certain target genes ( Chen & Calvo , 2002 ; Chen et al. , 2001a , b ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2162465" target="_blank">2162465</a>]<br></li>
<li>Thus , neither Lrp nor IHF has a signiﬁcant effect on ﬁmB expression under the conditions described in this paper , and H-NS represses the expression of the recombinase ( Blomﬁeld et al. , 1993 ; 1997 ; Donato et al. , 1997 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12890032" target="_blank">12890032</a>]<br></li>
<li>This was also seen , although not understood , in an experiment that we described earlier ( Chen et al. , 1997 ) , in which expression of serA was measured as a function of in vivo levels of Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11985712" target="_blank">11985712</a>]<br></li>
<li>Many genes where expression is affected by Lrp have actually been shown to be subjected to the effects , either directly or indirectly , of H-NS , e.g. papB , fin7B , ilvIH and lvsU ( Kawula and Omdorff , 1991 ; Van der Woude et al. , 1992 ; Ito et al. , 1994 ; Levinthal et al. , 1994 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8890170" target="_blank">8890170</a>]<br></li>
<li>Moreover , lrp expression is regulated autogenously by Lrp , as has been suggested previously ( Lin et al. 1992b ; Nakamura and Ito 1993 ; Wang et al. 1994 ) , and is further controlled by H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7603430" target="_blank">7603430</a>]<br></li>
<li>Since ArgP was previously shown not to exert autoregulation ( Nandineni and Gowrishankar , 2004 ; Caldara et al. , 2006 ) , in contrast to many other LysR-family members , we may conclude that argP expression is regulated neither by Lrp or ArgR , nor by ArgP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19906180" target="_blank">19906180</a>]<br></li>
<li>The Both Lrp and GcvA are required for expression of the GS852 +2 l - , , , lysogen showed about a 2-fold higher gcv operon ( Stauffer & Stauffer , 1994 ; Wilson et al. , basal level of expression in G M medium than the 1993b ; Lin et al. , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9802028" target="_blank">9802028</a>]<br></li>
<li>The branched-chain amino acids and alanine affect the interaction of Lrp with ﬁmS , whereas Neu5Ac and GlcNAc inhibit ﬁmB expression and hence inhibit OFF-to-ON recombination selectively ( Roesch and Blomﬁeld , 1998 ; El-Labany et al. , 2003 ; Sohanpal et al. , 2004 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17238917" target="_blank">17238917</a>]<br></li>
<li>Among the genes regulated by Lrp is osmY , a gene encoding a periplasmic protein of unknown function , which is osmotically and stationary phase inducible and ss dependent ( Yim and Villarejo , 1992 ; Lange et al. , 1993 ; Yim et al. , 1994 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9663683" target="_blank">9663683</a>]<br></li>
<li>d. Also regulated by Crp , IHF , Lrp , CpxR , RstA , Fur as well as sE ( Rhodius et al. , 2006 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18710431" target="_blank">18710431</a>]<br></li>
<li>Ernsting et al. ( 1992 ) developed this idea further by observing a correlation between genes positively regulated by Lrp and the required physiology for nitrogen limitation , cold , and low osmolarity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8152423" target="_blank">8152423</a>]<br></li>
<li>The artPIQM operon is , besides being downregulated by Lrp ( Hung et al. , 2002 ) , also repressed by ArgR ( Caldara et al. , 2007 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19906180" target="_blank">19906180</a>]<br></li>
<li>The second gene of serine biosynthesis , serC , is also activated strongly by Lrp ( Man et al. , 1997 ) and is also affected by Crp , negatively according to these authors , and positively according to others ( Lim et al. , 1994 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11985712" target="_blank">11985712</a>]<br></li>
<li>Thirteen of these genes ( adhP , cfa , dps , gadC , gadW , mlrA , poxB , otsB , otsA , yciG , yeaG , yjgB , and yohF ) were also described as Lrp repressed by Tani et al. ( 64 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15716429" target="_blank">15716429</a>]<br></li>
<li>Interestingly , Lrp and Nac also act antagonistically at the gltBD promoter , which , like serA , is activated by Lrp and repressed by Nac ( Ernsting et al. , 1993 ; R. A. Bender and R. A. Muse , personal communication ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12123449" target="_blank">12123449</a>]<br></li>
<li>Leucine and alanine were previously believed to act positively in ® m phase variation by enhancing Lrp activation ( Bloom ® eld et al. , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10216857" target="_blank">10216857</a>]<br></li>
<li>Leucine normally diminishes the Lrp activation of the ilvIH operon ( Platko et al. 1990 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8152423" target="_blank">8152423</a>]<br></li>
<li>There is evidence that cells modulate Lrp levels as much as tenfold in response to alterations in the external environment ( Landgraf et al. , 1996 ; Lin et al. , 1992a ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9236118" target="_blank">9236118</a>]<br></li>
<li>However , Lrp synthesis is much decreased in cells grown in LB , glycerol and succinate ( Chen et al. , 1997 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11985712" target="_blank">11985712</a>]<br></li>
<li>pLrp of Lrp protein was constructed essentially according to the standard procedure in this laboratory ( Shimada et al. , 2005 ; Yamamoto et al. , 2005 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2162465" target="_blank">2162465</a>]<br></li>
<li>It is unlikely that this cooperativity is solely attributable to specific protein interactions between adjacently bound Lrp protomers since Willins et al. ( 39 ) have demonstrated that , in solution , Lrp exists exclusively in a homodimeric form at con-26 centrations up to and exceeding 1 3 10 M .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8900118" target="_blank">8900118</a>]<br></li>
<li>Because the product of the ilvlH gene , AHAS III , is specific for isoleucine biosynthes ; s and thus competes for the threonine pool ( see fig i ) , we wanted to test if H-NS would also regulate the action of the kblitdh operon for which Lrp acts as a repressor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7748928" target="_blank">7748928</a>]<br></li>
<li>Table 2 : Statistical analysis of permuted microarray data Pair Size of overlap # of overlaps with smaller size P-value 16 21 28 0.04 0.052 0.07 Eco and Pmi Eco and Vch Pmi and Vch Microarray analysis of gene regulation by heterologous Lrp proteins -- effects on magnitude of regulation Predictions of regulatory connections between a regulator and a target gene are useful in themselves , but substantial understanding of a cell 's gene regulation also requires knowing the sign and strength of the regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18405378" target="_blank">18405378</a>]<br></li>
<li>At present , we can not decide whether cAMP-CRP , Lrp , and IHF directly bind to the osmY regulatory region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8253679" target="_blank">8253679</a>]<br></li>
<li>Under these conditions , cAMP-CRP and IHF do not contribute to the specific regulatory pattern of starvation induction ( which is no longer found in a strain that is deficient for as and Lrp but has wild-type cya , crp , himA , and hip alleles ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8253679" target="_blank">8253679</a>]<br></li>
<li>None of the several regulatory genes mentioned above seems to be essential for hyperosmotic induction of osmY , which could still be observed in mutants deficient in as , Lrp , cAMP-CRP , or IHF and even in strains carrying rpoS lrp or rpoS hip double mutations .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8253679" target="_blank">8253679</a>]<br></li>
<li>Two hypotheses could account for these results : ( i ) osmoregulation of osmY requires yet another unidentified regulatory mechanism that is totally independent of as , Lrp , cAMP-CRP , and IHF ; or ( ii ) osmotic induction could be mediated by several independently and additively acting components , some of which could be those identified in the present study as being involved in the regulation of osmY .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8253679" target="_blank">8253679</a>]<br></li>
<li>Therefore , osmY remains repressed by the concerted action of CRP , Lrp , and IHF until the growth rate is further reduced below a certain limit .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8253679" target="_blank">8253679</a>]<br></li>
<li>However , CRP , Lrp , and IHF are not essential for osmotic induction of osmY ( 33 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7814331" target="_blank">7814331</a>]<br></li>
<li>Stationary-phase induction of osmY in rich medium is mediated by the combined action of as , Lrp , cAMP-CRP , and IHF , with the latter three proteins acting as transition state regulators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8253679" target="_blank">8253679</a>]<br></li>
<li>Whereas rs , Lrp , CRP , and IHF are clearly involved in stationary-phase induction , none of these regulators is essential for osmotic regulation of osmY .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8253679" target="_blank">8253679</a>]<br></li>
<li>In contrast to station-ary-phase induction of osmY , which now can be explained by the concerted action of us and the negatively acting transitio state regulators Lrp , cAMP-CRP , and IHF , the problem of osmotic regulation of osmY still awaits its solution .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8253679" target="_blank">8253679</a>]<br></li>
<li>Thus , ArcA and FNR regulate respiration in response to oxygen availability ( the variable in these experiments ) , CRP and Lrp sense nutritional state via cAMP and L-leucine concentrations , and Fis , H-NS and IHF modify DNA topology , which is itself dependent on energy status [ 7,25 -- 27 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22870390" target="_blank">22870390</a>]<br></li>
<li>By these criteria , ArcA , CRP , Fis , FNR , H-NS , IHF and Lrp were designated as global TFs in E. coli K-12 [ 24 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22870390" target="_blank">22870390</a>]<br></li>
<li>Thus , it is concluded that ArcA , CRP , Fis , FNR , H-NS , IHF and Lrp are truly global regulators , at least as far as the response to oxygen availability is concerned , in the sense that their outputs create the transcriptional framework upon which local regulators operate in response to specific unstable environments Table 1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22870390" target="_blank">22870390</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_191">
Regulatory patterns III ( anaerobically repressed , not affected by FNR ) and IV ( anaerobically activated , not affected by FNR ) are most easily explained as genes affected by the ArcAB system or as yet unidentified global regulators such as Lrp , IHF , FIS , or H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12754220" target="_blank">12754220</a>]<br></label>
<input id="act_similarity_191", type="checkbox">
<div class="ACT_similar">
<li>Regulatory pattern III ( anaerobically repressed , but not affected by ArcA ) and pattern IV ( anaerobically activated , but not affected by ArcA ) are most easily explained as genes controlled by the FNR protein or by an as yet unidentified global regulator such as Lrp , IHF , FIS , or H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15699038" target="_blank">15699038</a>]<br></li></div>
<li>Additionally , these genes are regulated by transcription factors speciﬁc for certain stress responses ( e.g. , OxyR , which is involved in the oxidative-stress response ) or more-global regulators , such as H-NS , IHF , cyclic AMP ( cAMP ) - cAMP receptor protein ( CRP ) , and Lrp ( 104 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16760308" target="_blank">16760308</a>]<br></li>
<li>Indeed , the more stress-responsive genes are studied , the more it appears that their regulation involves a network of global regulators , such as H-NS or Lrp , but also CRP , IHF or Fis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9663683" target="_blank">9663683</a>]<br></li>
<li>A number of the rsdependent genes are also regulated through the action of one or several additional global regulators like CRP-cAMP , Fis , IHF , Lrp or H-NS ( see for instance Xu and Johnson 1995a , 1995b , 1995c ; Bouvier et al. 1998 ; Colland et al. 2000 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12395194" target="_blank">12395194</a>]<br></li>
<li>The near equality of F ( serC [ pdxF ] 9-lacZ ) expression in the crp lrp and crp1 lrp1 strains grown in MMGlu is expected if the decrease and increase due to loss of Lrp and CRP-cAMP function , respectively , offset each other .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9171388" target="_blank">9171388</a>]<br></li>
<li>Moreover , the activation by Lrp and repression by CRP-cAMP seem to be independent , because the serC ( pdxF ) transcription level in the crp lrp double mutant was intermediate to those in the crp and lrp single mutants grown in MMGlu ( Table 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9171388" target="_blank">9171388</a>]<br></li>
<li>We also found that serC ( pdxF ) - aroA cotranscript levels are regulated by Lrp activation and CRP-cAMP repression ( Fig. 6 ; Table 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9171388" target="_blank">9171388</a>]<br></li>
<li>The antagonism of the Lrp and CRP-cAMP global regulators also provides a mechanism for reduction of serC ( pdxF ) - aroA transcription and SerC ( PdxF ) transaminase and AroA synthase amounts in cells growing in amino acid-rich media , which provide serine and decrease the need for B6 vitamers PLP and pyridoxamine 59-phosphate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9171388" target="_blank">9171388</a>]<br></li>
<li>During growth in MMGlu , cellular Lrp or CRP-cAMP amounts are high or low , respectively ( 4 , 26 , 27 , 32 , 44 , 45 , 47 ) , which maximizes serC ( pdxF ) - aroA transcription and SerC ( PdxF ) transaminase amount .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9171388" target="_blank">9171388</a>]<br></li>
<li>During growth in LB , repression of serC ( pdxF ) - aroA transcription by CRP-cAMP is maximal and stimulation by Lrp is negligible .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9171388" target="_blank">9171388</a>]<br></li>
<li>These results led to the hypotheses that CRP-cAMP represses and Lrp activates serC ( pdxF ) - aroA operon transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9171388" target="_blank">9171388</a>]<br></li>
<li>The foregoing results strongly supported the hypothesis that Lrp and CRP-cAMP act as positive and negative effectors , respectively , of serC ( pdxF ) - aroA operon transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9171388" target="_blank">9171388</a>]<br></li>
<li>This regulation is exerted at the transcriptional level ( Fig. 5 and 8 ; Table 2 ) and is mediated positively by Lrp ( 2.2-fold in MMGlu ) and negatively by CRP-cAMP ( twofold in MMGlu and ﬁvefold in LB ) ( Fig. 5 ; Tables 2 and 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9171388" target="_blank">9171388</a>]<br></li>
<li>However , dad expression is directly repressed by Lrp and directly activated CRP-cAMP ( 37 ) , daa expression is activated by both Lrp and CRP-cAMP ( 2 ) , and osmY expression is repressed by both Lrp and CRP-cAMP ( 33 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9171388" target="_blank">9171388</a>]<br></li>
<li>Other operons regulated by both Lrp and CRP-cAMP include dad ( degradative D-amino acid dehydrogenase [ 37 ] ) , daa ( F1845 ﬁmbrial adhesin [ 2 ] ) , and osmY ( osmotically induced periplasmic protein [ 33 ] ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9171388" target="_blank">9171388</a>]<br></li>
<li>To our knowledge , this is the ﬁrst example of an aromatic biosynthetic gene regulated by Lrp or CRP-cAMP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9171388" target="_blank">9171388</a>]<br></li>
<li>As the lrp mutation had 70 no effect when the CRP box was located at the nonproductive positions between 272.5 and 276.5 , Lrp seems to act as a co-activator for cAMP-CRP without affecting the ` topological ' rules for activation by cAMP-CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11532138" target="_blank">11532138</a>]<br></li>
<li>Here , we used our series of strains with the CRP box located at different positions in the csiD promoter region in order to test whether this modulation by H-NS and Lrp is coupled to activation by CRP or proceeds by CRP-independent mechanisms .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11532138" target="_blank">11532138</a>]<br></li>
<li>The histone-like proteins H-NS and Lrp positively modulate csiD expression by differentially affecting activation by cAMP-CRP Table 1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11532138" target="_blank">11532138</a>]<br></li>
<li>In summary , both histone-like proteins , H-NS and Lrp , modulate csiD expression by inﬂuencing activation by cAMP-CRP , but they obviously do so by different mechanisms .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11532138" target="_blank">11532138</a>]<br></li>
<li>Finally , the constructs with S the relocated CRP box were used to test whether modulation by the histone-like proteins H-NS and Lrp is connected to activation by cAMP-CRP or proceeds independently .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11532138" target="_blank">11532138</a>]<br></li>
<li>Although both H-NS and Lrp stimulate csiD activation by cAMP-CRP , they obviously do so by different mechan-isms .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11532138" target="_blank">11532138</a>]<br></li>
<li>Thus , Lrp somehow improves the efﬁciency but does not alter the ` geometry ' of activation by cAMP-CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11532138" target="_blank">11532138</a>]<br></li>
<li>Moreover , since Lrp can positively modulate csiD expression under certain conditions in vivo ( Figure 2 ) , effects of Lrp were also tested besides those of cAMP-CRP ( Figure 10 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9512707" target="_blank">9512707</a>]<br></li>
<li>This is also suggested by the ® nding that in vitro , even low concentrations of Lrp interfere with cAMP-CRP binding ( Figure 7B ) , which can not be relevant in vivo where cAMP-CRP is clearly the main activator of csiD expression and Lrp seems to play a modulatory , perhaps `` back-up '' role .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9512707" target="_blank">9512707</a>]<br></li>
<li>Beyond this point , osmYis finally strongly induced , which also implies that the repressing activity of cAMP-CRP , Lrp , and IHF is at least partially relieved .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8253679" target="_blank">8253679</a>]<br></li>
<li>In summary , we conclude that the csiD promoter has an intrinsic preference for EsS , but that also protein factors such as cAMP-CRP , Lrp and probably H-NS as well as DNA conformation contribute to its strong EsS selectivity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9512707" target="_blank">9512707</a>]<br></li>
<li>We observed that not only cAMP-CRP , but also Lrp could stimulate csiD transcription by EsS ( on linear as well as on supercoiled templates ) , and that the effects of cAMP-CRP and Lrp were additive .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9512707" target="_blank">9512707</a>]<br></li>
<li>These results suggest that both activation and repression of the gcvT : : lac2 fusion are face-of-the-helix dependent and require a precise stereospecific orientation of either GcvA , Lrp or both , relative to the gcu promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9611815" target="_blank">9611815</a>]<br></li>
<li>Thus , although it is possible that part of the effects of the a mutations could be mediated through Lrp and CRP , the results are more consistent with a GcvA Æ aCTD interaction likely being required for GcvA-mediated activation at the PgcvT promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15621456" target="_blank">15621456</a>]<br></li>
<li>Results with a series of deletions upstream of the PserC ( pdxF ) promoter revealed that activation by Lrp was likely direct , whereas repression by the cyclic AMP ( cAMP ) receptor protein-cAMP complex ( CRP-cAMP ) was likely indirect , possibly via a repressor whose amount or activity was stimulated by CRP-cAMP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9171388" target="_blank">9171388</a>]<br></li>
<li>Alternatively , some of the negative effects of Lrp may be the result of direct effects on cyclic AMP-CRP , an activator of dad that binds in proximity to Lrp ( Fig. 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10216857" target="_blank">10216857</a>]<br></li>
<li>These mechanisms could involve interactions of Lrp with other proteins , such as cyclic AMP-CRP , that bind in the dad promoter region , as well as direct effects of Lrp on RNA polymerase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955407" target="_blank">8955407</a>]<br></li>
<li>As the speciﬁcity of target recognition of Lrp is known to change toward different directions , depending on the species of interacting amino acid effector ( Hart & Blumenthal , 2011 ) , in this study we carried out SELEX screening using 0.1 mM Lrp alone in the absence of effectors .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2162465" target="_blank">2162465</a>]<br></li>
<li>Looking only at the subset of genes that were significantly affected by all three Lrps , we found that whether we examined genes significantly repressed by Lrp , activated by Lrp ( in both cases including both direct and indirect effects ) , or directly controlled by Lrp in E. coli , the results were consistent .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18405378" target="_blank">18405378</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_227">
Column C shows the set of 57 genes recognized in RegulonDB [ 71,72 ] as being directly controlled by Lrp , whether the control is positive or negative .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18405378" target="_blank">18405378</a>]<br></label>
<input id="act_similarity_227", type="checkbox">
<div class="ACT_similar">
<li>This column shows the set of 57 genes recognized in RegulonDB [ 71 , 72 ] as being directly controlled by Lrp , whether the control is positive or negative .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18405378" target="_blank">18405378</a>]<br></li></div>
<li>Examples of transcriptional control in which the action of ArgR occurs in combination with ( an ) other regulatory protein ( s ) and co-factors have been documented previously for CRP , NtrC , Lrp , and IHF .22,23,29,50 Furthermore , artP mRNA levels were found to be induced in the stationary phase by EσS , the general stress-specific RNA polymerase ,51 upregulated ( 1.9-fold ) by the superoxide generating agent paraquat , an inducer of the soxRS system , and by sodium salicylate ( NaSal ) , an inducer of the marRA system ( 2.5-fold ) ,52 and down-regulated by Lrp ( about sixfold ) .53 Genome-wide expression profiling studies in E. coli indicate that argT and hisJ transcription are activated by NtrC ( GlnG ) ,45 and that in anaerobiosis hisJ expression is strongly increased in fnr ( 25-fold ) and arcA ( 60-fold ) mutants .54,55 None of the underlying molecular mechanisms has been unraveled yet , but the induction of E. coli artP in the stationary phase appears to depend on several secondary promoters ,51 upstream of PartP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17850814" target="_blank">17850814</a>]<br></li>
<li>We found that , relative to the vector control , activation of gltB transcription by P. mirabilis or V. cholerae Lrp was essentially indistinguishable from that of the E. coli lrp positive control ( Fig. 4C ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18405378" target="_blank">18405378</a>]<br></li>
<li>Compared to the vector control , V. cholerae Lrp repressed Plrp-lacZ to the same extent as E. coli Lrp ( ~ 2 fold ) , while P. mirabilis Lrp ( 98 % identical to E. coli Lrp ) repressed about twice that much ( Fig 4B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18405378" target="_blank">18405378</a>]<br></li>
<li>We found that in E. coli gltB is activated ~ 25 fold by Lrp during mid-log , and about half as much in early stationary phase ( Fig. 10E , circles ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18405378" target="_blank">18405378</a>]<br></li>
<li>Bearing in mind that this is the ortholog showing only 43 % identity to E. coli gltB , it is nevertheless the case that while Lrp activates gltB in log-phase E. coli and P. mirabilis , under the same conditions it slightly represses Vc2373 in V. cholerae .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18405378" target="_blank">18405378</a>]<br></li>
<li>PgltB from P. mirabilis was barely expressed in V. cholerae irrespective of Lrp , was expressed weakly in its native host ( under the conditions used ) and with statistically insignificant effects of Lrp , while in E. coli this promoter was expressed at a moderate level and was activated by Lrp ( 5-fold ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18405378" target="_blank">18405378</a>]<br></li>
<li>Results : Heterologous Lrp regulated gltB , livK and lrp transcriptional fusions in an E. coli background in the same general way as the native Lrp , though with significant differences in extent .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18405378" target="_blank">18405378</a>]<br></li>
<li>Strikingly , while Lrp negatively regulates its own gene in E. coli , and was shown to do so even more strongly in P. mirabilis , Lrp appears to activate its own gene in V. cholerae .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18405378" target="_blank">18405378</a>]<br></li>
<li>This , and the fact that the E. coli and P. mirabilis Plrp fusions were both well expressed in the Vibrio background , suggests that V. chol-erae negatively regulates its Plrp via some Vibrio-specific factor , and that Lrp may in this case act as an anti-repres-sor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18405378" target="_blank">18405378</a>]<br></li>
<li>Lrp directly and strongly activates gltBD transcription in E. coli [ 74,92 ] , in a process that also requires the global regulator IHF [ 93 ] and is antagonized by Crp and ArgR [ 94 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18405378" target="_blank">18405378</a>]<br></li>
<li>In E. coli gltB is activated 30 -- 40 fold by Lrp when grown in MOPS glucose [ 67,69,74,92 ] , with the activation codependent on another global regulator , IHF [ 93,94 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18405378" target="_blank">18405378</a>]<br></li>
<li>In E. coli , PgltB is activated by Lrp [ 74,92-94 ,112 ] , but the results in Fig. 11 are more varied than for Plrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18405378" target="_blank">18405378</a>]<br></li>
<li>In E. coli , PgltB is controlled by Lrp , Crp , IHF , and ArgR [ 67,74,93,94,112 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18405378" target="_blank">18405378</a>]<br></li>
<li>We report here that gltBDF is directly and negatively controlled by Crp and ArgR , in addition to the direct positive control by Lrp and IHF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17233899" target="_blank">17233899</a>]<br></li>
<li>In addition to the previously-demon-strated direct positive roles of Lrp and IHF , there are direct negative roles for Crp and ArgR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17233899" target="_blank">17233899</a>]<br></li>
<li>Expression and purification of Lrp , ArgR , Crp and IHF In all cases , protein purity was assessed from single ( or , in the case of IHF , double ) Coomassie-stained bands when μg amounts were loaded onto SDS polyacrylamide gels .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17233899" target="_blank">17233899</a>]<br></li>
<li>In relation to leuE , the Lrp protein may be considered as a negative regulator whose functions could , in principle , be fulﬁlled by a specialized LysRtype repressor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17025177" target="_blank">17025177</a>]<br></li>
<li>Results of the DNasel-protection experiment shown in Fig. 16 are consistent with the in vitro transcription assay : in the absence of meAda , Ea7 ' ( 0.2 pM ) forms a binary complex that is inactive in transcription initiation ( Landini and Volkert , 1995b ) , and strongly protects the 01996 Blackwell Science Ltd , Mo/ecu/ar Microbiology , 20 , 947-95 Repression of os-dependent transcription by Lrp 949 aidB promoter region upstream of -40 ( lane 2 , Fig. 1B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8809748" target="_blank">8809748</a>]<br></li>
<li>shown in Fig. 6 , in the presence of 10pmol of Lrp , aid8 transcription by both Ea '' and EoS is strongly inhibited , while transcription from the control promoter Placuv5 is either slightly inhibited or unaffected ( Fig. 6 , lanes 3 and 7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8809748" target="_blank">8809748</a>]<br></li>
<li>Lanes : a , primer PE-1 alone ; b , primer extension products of RNA made in vivo ; remaining lanes , primer extension products of RNA made in vitro in the absence of cyclic AMP-CRP ( c ) , in the presence of 50 mM cyclic AMP and 40 nM CRP ( d ) , and in the presence of cyclic AMP-CRP and 50 nM Lrp ( e ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955407" target="_blank">8955407</a>]<br></li>
<li>H-NS does not directly a ¡ ect switching frequency In order to test whether H-NS plays a direct role in inversion of the phase switch , for example by blocking access of proteins required for switching such as IHF or Lrp , in vitro switching experiments were carried out in the absence or presence of H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9595659" target="_blank">9595659</a>]<br></li>
<li>In vitro experiments of the type shown in Fig. 1B were performed in the presence of increasing amounts of Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1447135" target="_blank">1447135</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1447135" target="_blank">1447135</a>]<br></li>
<li>Autoradiogram of in vitro transcription reactions , performed with a DNA template concentration of 8 nM , either in the absence of presence of increasing amounts of purified Lrp as described under `` Materials and Methods .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8900118" target="_blank">8900118</a>]<br></li>
<li>In Vitro Transcription Reactions -- In vitro transcription reactions were performed using the closed-circular supercoiled plasmid pDHWT ( described above ) , in the absence and presence of purified Lrp , according to the procedures of Hauser et al. ( 32 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8900118" target="_blank">8900118</a>]<br></li>
<li>In vitro transcription in the presence of Lrp and methylated Ada .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8809748" target="_blank">8809748</a>]<br></li>
<li>In vitro transcription reactions of the ilvPG promoter in the absence and presence of increasing amounts of purified Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8900118" target="_blank">8900118</a>]<br></li>
<li>The data indicated that plasmid-borne mutant Lrp proteins reduced ilvIH expression to the basal level , which is equivalent to that in an lrp null , even in the presence of chromosomal Irp + ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7603430" target="_blank">7603430</a>]<br></li>
<li>It should be noted that these plasmid-borne mutant Lrp proteins exerted dominant negative effects since the Irp expression level was enhanced to that seen in Irp null strains even in the presence of chromosomal wild-type Irp ( Table 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7603430" target="_blank">7603430</a>]<br></li>
<li>However , the exception is the V. chol-erae Plrp in its native host , which shows tenfold higher expression in the presence of Lrp , suggesting activation rather than repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18405378" target="_blank">18405378</a>]<br></li>
<li>We found that maximum operon expression in bacteria growing in MMGlu depended on the presence of the upstream Lrp box and that decreased operon expression caused by an lrp mutation was lost when the Lrp box was deleted ( Results and Table 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9171388" target="_blank">9171388</a>]<br></li>
<li>When levels of expression of this fusion in isogenic lrp1 and lrp strains were compared , Lrp was found to decrease micF-lacZ1 expression 2.0-fold in the absence of sucrose and 2.2-fold in the presence of sucrose .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8002608" target="_blank">8002608</a>]<br></li>
<li>To further test this , we measured the expression of yojI in the presence and in the absence of Lrp by using a chromosomal yojI-lacZ transcriptional fusion constructed as follows .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19074390" target="_blank">19074390</a>]<br></li>
<li>In contrast , while expression of the ompF-lacZ1 transcriptional fusion was decreased 1.3-fold by Lrp during growth in the absence of sucrose , expression of the ompF9-lacZ1 translational fusion was increased by Lrp 3.7-fold in the absence of sucrose and 3.2-fold in the presence of 0.2 M sucrose .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8002608" target="_blank">8002608</a>]<br></li>
<li>When 2 ng of Lrp protein was used in DNase I footprinting assays , no obvious protection was observed ( data not shown ) , whereas the presence of 4 -- 64 ng of Lrp protein showed similar patterns of protection ( Fig. 5 , lanes Fig. 2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11985712" target="_blank">11985712</a>]<br></li>
<li>Also as expected from previous studies [ 74,92 ] , activation of PgltB by E. coli Lrp was moderately reduced in the presence of ILV ; the two heterologous Lrp orthologs gave patterns essentially indistinguishable from that of E. coli Lrp ( Fig 4F ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18405378" target="_blank">18405378</a>]<br></li>
<li>The mode of regulation of polypeptide synthesis by Lrp ( positive [ + ] or negative [ - ] ) was determined by comparing the net rates of synthesis in the presence and absence of a functional Lrp protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1346534" target="_blank">1346534</a>]<br></li>
<li>When a glnL deletion ( 7 ) was crossed into strains W3110 , BE1 , and BE2 , the level of glutamine synthetase was no longer sensitive to the presence or absence of a functional Lrp protein ( Table 6 , strains WS7014 , WS7026 , and WS7038 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1346534" target="_blank">1346534</a>]<br></li>
<li>The level of glutamine synthetase in strains carrying the g1nL2302 allele ( 9 ) , which produces a GlnL protein that directs constitutive synthesis of glutamine synthetase , was also insensitive to the presence or absence of a functional Lrp protein ( Table 6 , strains WS7017 , WS7029 , and WS7041 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1346534" target="_blank">1346534</a>]<br></li>
<li>Extracts prepared from strains W3110 ( lrp + ) and BE3 ( lrp-1 ) showed mobility shifts characteristic of the presence of a functional Lrp protein ( 32 ) when 1 , ul of extract was used .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1346534" target="_blank">1346534</a>]<br></li>
<li>Two-dimensional gel electrophoretic analysis of net protein synthesis in strains containing or lacking a functional lrp gene and grown in the presence and absence of leucine detected a number of known and unknown proteins regulated by Lrp in a leucine-sensitive manner .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7901196" target="_blank">7901196</a>]<br></li>
<li>We show that the presence of either of two different mutant alleles of hns results in increased activity of Lrp and that the presence of a pBR322 plasmid containing only the hns + gene blocks the action of Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8152423" target="_blank">8152423</a>]<br></li>
<li>We also show that RNAP binds stably to the gltBDF promoter in the presence of Lrp and IHF , or in the presence of Crp-cAMP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17233899" target="_blank">17233899</a>]<br></li>
<li>A. DNase I protection assay of gltBDF promoter region in the presence of both Lrp and IHF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17233899" target="_blank">17233899</a>]<br></li>
<li>One particularly striking difference , as an example , is the DNase cleavage at position -19 , that shows hypersensitivity in the presence of Lrp + IHF but not in the presence of Crp ( compare lane 7 of Fig. 3B to lane 7 of Fig. 7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17233899" target="_blank">17233899</a>]<br></li>
<li>RNAP only yields a defined DNAse I footprint under two tested conditions : in the presence of both Lrp and IHF , or in the presence of Crp-cAMP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17233899" target="_blank">17233899</a>]<br></li>
<li>The DNAse I footprint of RNAP in the presence of Lrp and IHF is altered by ArgR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17233899" target="_blank">17233899</a>]<br></li>
<li>ArgR alters , but does not prevent , footprint formation by RNAP + Lrp + IHF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17233899" target="_blank">17233899</a>]<br></li>
<li>The activation of the gltBDF operon by Lrp is totally dependent on the presence of IHF , since introduction of an Lrp mutation in strain LP1048 did not lead to a further decrease in b-galactosidase level .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11395454" target="_blank">11395454</a>]<br></li>
<li>The A G sequencing ladder is indicated by an asterisk tightly even in the presence of a ® vefold excess of cAMP-CRP , i.e. the footprint pattern obtained with Lrp was identical in the absence and presence of cAMP-CRP ( Figure 7B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9512707" target="_blank">9512707</a>]<br></li>
<li>The b-galactosidase activity from each chimeric strain in the presence or absence of Lrp is presented on the left side of the construct diagrams .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11395454" target="_blank">11395454</a>]<br></li>
<li>The top strand of the fragment used in Fig. 3 was subjected to hydroxyl radical cleavage in the presence ( lanes 3 and 6 ) or absence ( lanes 2 and 5 ) of 10 nM Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955407" target="_blank">8955407</a>]<br></li>
<li>Transcription from plasmid pCV7 in the presence of purified RNAP was examined in the presence or absence of Lrp ( 298 % purity ) and the presence or absence of a crude cell extract from Lrp-deficient strain CV1008 ( lrp-35 : : TnlO ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1447135" target="_blank">1447135</a>]<br></li>
<li>One exception was the culture in the presence of both NH4Cl and Leu , in which growth of the lrp mutant was signiﬁcantly reduced , indicating that excess of Leu speciﬁcally interferes with cell growth in the absence of Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2162465" target="_blank">2162465</a>]<br></li>
<li>In vivo experiments with lrp and lrp strains indicated that P2 , ÿ but not P1 or P3 , was activated by Lrp in the presence of alanine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10216857" target="_blank">10216857</a>]<br></li>
<li>In the presence of Lrp , this region showed alternately increased and decreased sensitivity to DNase I cleavage with a periodicity of about 10 nucleotides .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11985712" target="_blank">11985712</a>]<br></li>
<li>lacZ transcription is also increased in the presence of Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7751276" target="_blank">7751276</a>]<br></li>
<li>Expression from pAF330 in the presence of the second hns allele hns-203 with or without functional Lrp was also examined .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9006049" target="_blank">9006049</a>]<br></li>
<li>DNasel-protectionassay in the presence of Lrp protein , aid6 template strand .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8809748" target="_blank">8809748</a>]<br></li>
<li>Lane 1 , control , no protein added ; lanes 2-9 , amount of Lrp and presence of leucine are indicated in the figure .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8809748" target="_blank">8809748</a>]<br></li>
<li>However , in the presence of Lrp , the effect of Crp is negligible .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11985712" target="_blank">11985712</a>]<br></li>
<li>Further experiments examining the effect of Lrp on the sucrose response of these fusions were performed in the presence of 0.2 M sucrose .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8002608" target="_blank">8002608</a>]<br></li>
<li>Fractions ( 3 ml ) were collected , and the fractions with peak A280 readings were run on a denaturing polyacrylamide gel to verify the presence of Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9515701" target="_blank">9515701</a>]<br></li>
<li>Circles have been added to represent the degree of cleavage seen in the presence of Lrp relative to that seen in its absence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9236118" target="_blank">9236118</a>]<br></li>
<li>Even in the presence of Lrp , some increase could be seen .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11985712" target="_blank">11985712</a>]<br></li>
<li>When either 134 or 284 bp separate the sites , the processivity is increased with the presence of Lrp ( Figure 7C ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25785689" target="_blank">25785689</a>]<br></li>
<li>Thus , the af ® nity of Lrp for site 5 in relation to site 9 was increased from almost twofold in the absence of alanine to almost fourfold in the presence of alanine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10216857" target="_blank">10216857</a>]<br></li>
<li>Department of Molecular Biology and Genetics , Cornell University , Ithaca NY 14853 , USA 2 Protein Chemistry , P ® zer Inc Groton , CT 06340 , USA # 2001 Academic Press Keywords : Lrp ; Escherichia coli ; leucine ; oligomerization ; global regulation * Corresponding author Introduction Lrp is a global regulator of metabolism in Escher-ichia coli that helps cells respond to changes in environmental conditions .1,2 Many Lrp-regulated operons are also subject to control by leucine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11575919" target="_blank">11575919</a>]<br></li>
<li>Identified genes known to be regulated by Lrp ' Activity encod of : Leucine ed Lrp Position ( min ) on genome Gene Lrp + leucine ilvIH 1.85 Acetolactate synthetase III 2 Leucine biosynthesis lrp 20 Leucine-responsive regulatory protein oppA-D 27 Oligopeptide permease sdaA 40.2 L-Serine deaminase I 60.5 Gene regulating sdaB serA 63 Phosphoglycerate dehydrogenase gcv 63 Glycine cleavage enzymes livJIK 76 Branched-chain amino acid transport tdh 91 Threonine dehydrogenase lysU 94 Lysyl-tRNA synthetase II a + , increased activity ; - , decreased activity ; None , no effect ; NT , not testable .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1532173" target="_blank">1532173</a>]<br></li>
<li>As this work was in progress , Lrp ( leucine-regulatory protein ) was shown to be involved in the regulation of several leucine-regulated genes ( 31 , 40a , 43 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1917830" target="_blank">1917830</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_298">
However , it is interesting that these newly discovered genes that are regulated by FNR under anaerobic conditions fall into to the same functional classes as the genes regulated by the leucine-responsive regulatory protein ( Lrp ) under aerobic conditions ( 11 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12754220" target="_blank">12754220</a>]<br></label>
<input id="act_similarity_298", type="checkbox">
<div class="ACT_similar">
<li>These newly discovered genes fall into the same functional classes as the genes regulated by the leucine-responsive regulatory protein Lrp under aerobic conditions ( 25 ) and FNR under anaerobic conditions ( 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15699038" target="_blank">15699038</a>]<br></li></div>
<li>, 1993 ) ; for aid6 , the Lrp/oS mechanism of regulationcan be explained by the putative role of the Aids protein in leucine breakdown.At low intracellular concentrations of leucine , Lrp is bound to the aid6 promoter , preventing the production of the aidBencoded leucine-degradation enzyme .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8809748" target="_blank">8809748</a>]<br></li>
<li>At high concentrations of leucine , Lrp dissociates from the promoter ; under physiological conditions that allow production of os , such as starvation or stationary phase , when leucine is not used for protein synthesis , aid5 transcription is activated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8809748" target="_blank">8809748</a>]<br></li>
<li>The inversion of ﬁmS requires a site-speciﬁc recombination , catalysed by recombinases FimB ( inversion in either direction ) and FimE ( inversion from ON-to-OFF predominately ) and is controlled by DNA bending proteins IHF ( integration host factor ) and Lrp ( leucine-responsive regulatory protein ) ( Klemm , 1986 ; Dorman and Higgins , 1987 ; Eisenstein et al. , 1987 ; McClain et al. , 1991 ; 1993 ; Blomﬁeld et al. , 1993 ; Gally et al. , 1994 ; 1996 ; Blomﬁeld et al. , 1997 ; Roesch and Blomﬁeld , 1998 ; Lahooti et al. , 2005 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17238917" target="_blank">17238917</a>]<br></li>
<li>Expression of both serA and serC is activated by the leucine-responsive regulatory protein , Lrp , a global regulator ﬁrst recognized as Rbl ( regulation by leucine ; Lin et al. , 1990 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11985712" target="_blank">11985712</a>]<br></li>
<li>In addition , dad was shown to be induced by leucine in addition to alanine , and both of these compounds appear to act by antagonizing the repression of all the dad transcripts by leucine-responsive regulatory protein ( Lrp ) ( Mathew et al. 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9648752" target="_blank">9648752</a>]<br></li>
<li>The Lrp protein ( 18.8 kDa ) is a global transcriptional-regulator of several E. coligenes and operons ; Lrp can either activate or inhibit transcription by binding its target promoters , and leucine often plays a role in the modulation of Lrp activity ( Calvo and Matthews , 1994 ; Newman eta / .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8809748" target="_blank">8809748</a>]<br></li>
<li>Wiliins , D.A. , Ryan , C.W , , Piatko , J.V , , and Calvo , J.M , ( 1991 ) Characterization of Lrp , an Escherichia coii regulatory protein that mediates a global response to leucine , J Bioi Chem 266:10768 -10774 , Xiao , H , , Perisic , O. , and Us , J.T , ( 1991 ) Cooperative binding of Drosophiia heat-shock factor to arrays of a conserved 5 bp unit .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8483418" target="_blank">8483418</a>]<br></li>
<li>The leucine-responsive regulatory protein ( Lrp ) has been shown to both positively and negatively regulate transcription of a number of genes in response to exogenous leucine ( 215 ) ; Lrp action is signiﬁcantly activated by the absence of L-leucine in the growth medium , whereas it is repressed in the presence of L-leucine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16760308" target="_blank">16760308</a>]<br></li>
<li>The sdaA gene is expressed in glucose-minimal medium and in LB , repressed by the leucine-responsive regulatory protein ( Lrp ) [ 21 ] , and induced by the amino acids , glycine and leucine [ 18,32 ] , which partially reverse Lrp repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12706508" target="_blank">12706508</a>]<br></li>
<li>Abbreviations used : cAMP , 30,50-cyclic adenosine monophosphate ; CRP , cAMP receptor protein ; Lrp , leucine-responsive regulatory protein ; E , RNA polymerase core enzyme ; SDS-PAGE , sodium dodecyl sulfate-polyacrylamide gel electrophoresis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9512707" target="_blank">9512707</a>]<br></li>
<li>The dadAX operon is expressed by multiple promoters that are repressed by leucine-responsive regulatory protein ( Lrp ) and activated by cyclic AMP-CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10216857" target="_blank">10216857</a>]<br></li>
<li>We also show that the serC ( pdxF ) - aroA operon is in fact repressed by CRP-cAMP and activated by the leucine-responsive protein ( Lrp ) at the transcriptional level .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9171388" target="_blank">9171388</a>]<br></li>
<li>Lrp Leucine-responsive regulatory protein ( Lrp ) is a 15kDa protein that functionally responds to changes in leucine concentration .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16879983" target="_blank">16879983</a>]<br></li>
<li>Abbreviations : GCV , glycine cleavage ; GM , glucose minimal ; GMS , gel mobility shift ; LM , lactose minimal ; Lrp , leucine-responsive regulatory protein ; TPEG , phenylethyl-B-D-thiogalactoside ; wt , wild-type .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10463182" target="_blank">10463182</a>]<br></li>
<li>The leucine-responsive protein Lrp represses osmCp1 and activates osmCp2 , the nucleoid-associated protein H-NS represses both promoters , and the stationary-phase sigma factor s speciﬁcally recognizes osmCp2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11566985" target="_blank">11566985</a>]<br></li>
<li>Abbreviations : Ap , ampicillin ; H-NS , histone-like nucleoid-structuring protein ; IHF , integration host factor ; IRL , left inverted repeat ; Km , kanamycin ; Lrp , leucine-responsive regulatory protein ; Sm , spectinomycin ; UPEC , uropathogenic E. coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16207912" target="_blank">16207912</a>]<br></li>
<li>Lrp , leucine-regulatory protein ; L-SD1 and L-SD2 , L-serine deaminases 1 and 2 ; orf , open-reading frame ; Crp , CAMP-regulatory protein ; IHF , integration host factor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8026499" target="_blank">8026499</a>]<br></li>
<li>Abbreviations used : OmpF , outer membrane protein F ; OmpC , outer membrane protein C ; Lrp , leucine-responsive protein ; NO , nitric oxide ; IHF , integration host factor ; HTH , helix-turn-helix .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11601842" target="_blank">11601842</a>]<br></li>
<li>Abbreviations : HNS , nucleoid-associated protein ; IHF , integration host factor ; Lrp , leucine-responsive protein ; Fis , factor for inversion stimulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16760308" target="_blank">16760308</a>]<br></li>
<li>ABBREVIATIONS ■ ENase , restriction endonuclease ; EcoRI , restriction endonulease EcoRI ; EcoRV , restriction endonuclease EcoRV ; bp , base pair ; Dam , DNA adenine methyltransferase ; EDTA , ethylenediami-netetraacetic acid ; BSA , bovine serum albumin ; DTT , dithiothreitol ; PAGE , polyacrylamide gel electrophoresis ; NAPs , nucleoid-associated proteins ; Lrp , leucine-responsive regulatory protein ; AAG , alkyl adenine glycosylase ; AdoMet , S-adenosyl methionine REFERENCES ■ ( 1 ) Halford , S. E. ( 2009 ) An end to 40 years of mistakes in DNA-protein association kinetics ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25785689" target="_blank">25785689</a>]<br></li>
<li>This view of Lrp is consistent with a postulated primary function of Lrp as a DNA-organizing protein13 ,14 but is at var-iance with the view of Lrp as a regulatory protein that helps E. coli to make transitions between growth in rich and lean media .1,9 The work presented here concerns the effects of leucine on Lrp self-association , effects that may have relevance to the different regulatory patterns exhibited by Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11575919" target="_blank">11575919</a>]<br></li>
<li>The histone-like leucine-responsive regulatory protein ( Lrp ) binds DNA nonspeciﬁcally as an octamer , and also increases Dam 's processivity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25785689" target="_blank">25785689</a>]<br></li>
<li>Pap phase variation is known to involve the formation of differential DNA methylation patterns and to require the presence of the leucine-responsive regulatory protein ( Lrp ) , a global transcriptional-regulator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11972787" target="_blank">11972787</a>]<br></li>
<li>174 , No. 6 X placMu Insertions in Genes of the Leucine Regulon : Extension the Regulon to Genes Not Regulated by Leucine of RONGTUAN LIN , ' R. D'ARI ,2 AND E. B. NEWMAN ' * Department of Biological Sciences , Concordia University , Montreal , Quebec H3G IM8 , Canada , 1 and Institut Jacques Monod , Centre National de la Recherche Scientifique , Universite de Paris , 75251 Paris Cedex 05 , France Received 28 October 1991/Accepted 16 January 1992 The leucine regulon coordinates the expression of several Escherichia coli genes according to the presence of exogenous leucine , which interacts with the bp gene product , Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1532173" target="_blank">1532173</a>]<br></li>
<li>Kutukova , E.A. , Livshits , V.A. , Altman , I.P. , Ptitsyn , L.R. , Zyiatdinov , M.H. , Tokmakova , I.L. , and Zakataeva , N.P. , The yeaS ( leuE ) Gene of Escherichia coli Encodes an Exporter of Leucine , and the Lrp Protein Regulates Its Expression , FEBS Lett .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17025177" target="_blank">17025177</a>]<br></li>
<li>FEBS 29827 The yeaS ( leuE ) gene of Escherichia coli encodes an exporter of leucine , and the Lrp protein regulates its expression Ekaterina A. Kutukova , Vitaliy A. Livshits * , Irina P. Altman , Leonid R. Ptitsyn , Michael H. Zyiatdinov , Irina L. Tokmakova , Natalia P. Zakataeva Ajinomoto-Genetika Research Institute , Laboratory N2 , 1-st Dorozhny Proezd , b. 1 , Moscow 117545 , Russia Received 29 April 2005 ; revised 1 July 2005 ; accepted 12 July 2005 Available online 10 August 2005 Edited by Felix Wieland Abstract Overexpression of the yeaS gene encoding a protein belonging to the RhtB transporter family conferred upon cells resistance to glycyl-L-leucine , leucine analogues , several amino acids and their analogues .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16098526" target="_blank">16098526</a>]<br></li>
<li>Kutukova , E.A. , Livshits , V.A. , Altman , I.A. , Ptitsyn , L.R. , Zyiatdinov , M.H. , Tokmakova , I.L. , and Zakataeva , N.P. ( 2005 ) The yeaS ( leuE ) gene of Escherichia coli encodes an exporter of leucine , and the Lrp protein regulates its expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19906180" target="_blank">19906180</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_327">
The yeaS ( leuE ) gene of Escherichia coli encodes an exporter of leucine , and the Lrp protein regulates its expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17504942" target="_blank">17504942</a>]<br></label>
<input id="act_similarity_327", type="checkbox">
<div class="ACT_similar">
<li>The yeaS ( leuE ) gene of Esche-richia coli encodes an exporter of leucine , and the Lrp protein regulates its expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25845847" target="_blank">25845847</a>]<br></li></div>
<li>175 , No. 4 Copyright © 1993 , American Society for Microbiology Mutations Affecting the Ability of Escherichia coli Lrp To B DNA , Activate Transcription , or Respond to Leucine ind JILL V. PLATKO1 AND JOSEPH M. CALVO2 * Sections of Genetics and Development ' and Biochemistry , Molecular and Cell Biology ,2 Cornell University , Ithaca , New York 14853 Received 2 September 1992/Accepted 13 December 1992 Lrp is a regulatory protein in Escherichia coli that increases expression of some operons and decreases expression of others .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8432705" target="_blank">8432705</a>]<br></li>
<li>The Leucine-responsive Escherichia coli Regulatory Protein ( Lrp ) from STOICHIOMETRY AND MINIMAL REQUIREMENTS FOR BINDING TO DNA * ( Received for publication , August 28 , 1995 , and in revised form , January 5 , 1996 ) Yuhai Cui ‡ , Michael A. Midkiff , Qing Wang § , and Joseph M. Calvo ¶ From the Section of Biochemistry , Molecular and Cell Biology , Cornell University , Ithaca , New York 14853 Lrp ( Leucine-responsive regulatory protein ) regulates the expression of a number of operons in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8636076" target="_blank">8636076</a>]<br></li>
<li>JOURNAL OF BACTERIOLOGY , Dec. 1996 , p. 6930 -- / 96 / $ 04.0010 Copyright q 1996 , American Society for Microbiology Effects of Nutrition and Growth Rate on Lrp Levels in Escherichia coli JEFFREY R. LANDGRAF , JINGCAI WU , AND JOSEPH M. CALVO * Section of Biochemistry , Molecular and Cell Biology , Cornell University , Ithaca , New York 14853 Received 12 June 1996/Accepted 2 October 1996 Lrp ( leucine-responsive regulatory protein ) activates some Escherichia coli operons that function in anab-olism and represses others involved in catabolism ( for a review , see J. M. Calvo and R. G. Matthews , Microbiol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955316" target="_blank">8955316</a>]<br></li>
<li>176 , No. 7 Regulation of the Escherichia coli lrp Gene QING WANG , ' t JINGCAI WU ,2 DEVORAH FRIEDBERG ,3 JILL PLAKTO ,1 t AND JOSEPH M. CALVO2 * Sections of Genetics and Development ' and Biochemistry , Molecular and Cell Biology ,2 Comell University , Ithaca , New York 14853 , and Department ofApplied Microbiology , The Hebrew University-Hadassah Medical School , Jerusalem , Israel 910103 Received 18 October 1993/Accepted 15 January 1994 Lrp ( leucine-responsive regulatory protein ) is a major Escherichia coli regulatory protein which regulates expression of a number of operons , some negatively and some positively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8144448" target="_blank">8144448</a>]<br></li>
<li>174 , No. 23 In Vitro Transcription from the Escherichia coli ilvIH Promoter DEBRA AKER WILLINS AND JOSEPH M. CALVO * Section of Biochemistry , Molecular and Cell Biology , Comnell University , Ithaca , New York 14853 Received 24 June 1992/Accepted 21 September 1992 Lrp ( leucine-responsive regulatory protein ) activates the expression of the Escherichia coli ilvIH operon in vivo and mediates the repression of the operon by exogenous leucine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1447135" target="_blank">1447135</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_334">
Roesch , P.L. , and Blom ® eld , I.C. ( 1998 ) Leucine alters the interaction of the leucine-responsive regulatory protein ( Lrp ) with the ® m switch to stimulate site-speci ® c recombination in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10216857" target="_blank">10216857</a>]<br></label>
<input id="act_similarity_334", type="checkbox">
<div class="ACT_similar">
<li>Roesch , P.L. , and Blomﬁeld , I.C. ( 1998 ) Leucine alters the interaction of the leucine-responsive regulatory protein ( Lrp ) with the ﬁm switch to stimulate site-speciﬁc recombination in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11985712" target="_blank">11985712</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17238917" target="_blank">17238917</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20132443" target="_blank">20132443</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10096084" target="_blank">10096084</a>]<br></li></div>
<li>Research Expanded roles of leucine-responsive regulatory protein i transcription regulation of the Escherichia coli genome : Genomic SELEX screening of the regulation targets n Tomohiro Shimada ,1,2,3 Natsumi Saito ,4,5 Michihisa Maeda ,6 Kan Tanaka1 and Akira Ishihama2 ,3 1 Chemical Resources Laboratory , Tokyo Institute of Technology , Nagatsuta , Yokohama , Japan 2 Department of Frontier Bioscience , Hosei University , Koganei , Tokyo , Japan 3 Research Center for Micro-Nano Technology , Hosei University , Koganei , Tokyo , Japan 4 Department of Chemistry and Material Engineering , Tsuruoka National College of Technology , Yamagata , Japan 5 Institute for Advanced Biosciences , Keio University , Yamagata , Japan 6 School of Agriculture , Meiji University , Kawasaki , Kanagawa , Japan Correspondence : Tomohiro Shimada ( tshimada@res.titech.ac.jp ) DOI : 10.1099 / mgen .0.000001 Leucine-responsive regulatory protein ( Lrp ) is a transcriptional-regulator for the genes involved in transport , biosynthesis and catabolism of amino acids in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2162465" target="_blank">2162465</a>]<br></li>
<li>The diverse set of genes regulated by Lrp suggested a regulon controlled by Lrp , with the effect of Lrp on expres ¬ The leucine-responsive regulatory protein ( Lrp ) has been shown to regulate , either positively or negatively , the transcription of several Escherichia coli genes in response to leucine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1346534" target="_blank">1346534</a>]<br></li>
<li>ow Is BRIAN R. ERNSTING , `` 2 JOHN W. DENNINGER ,12 ROBERT M. BLUMENTHAL ,3 AND ROWENA G. MATTHEWS 1,2 * Biophysics Research Division1 and Department of Biological Chemistry ,2 University of Michigan , Ann Arbor , Michigan 48109 , and Department of Microbiology , Medical College of Ohio , Toledo , Ohio 436993 Received 26 July 1993/Accepted 2 September 1993 The regulon controlled by the leucine-responsive regulatory protein ( Lrp ) of Escherichia coli consists of over 40 genes and proteins whose expression is regulated , either positively or negatively , by Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7901196" target="_blank">7901196</a>]<br></li>
<li>ctivation from a Distance : Roles of Lrp and Integration A Host Factor in Transcriptional Activation of gltBDF LIGI PAUL ,1 ROBERT M. BLUMENTHAL ,2 AND ROWENA G. MATTHEWS1 ,3 * Biophysics Research Division1 and Department of Biological Chemistry ,3 University of Michigan , Ann Arbor , Michigan 48109 , and Department of Microbiology and Immunology , Medical College of Ohio , Toledo , Ohio 436142 Received 23 January 2001/Accepted 13 April 2001 The leucine-responsive regulatory protein ( Lrp ) binds to three sites centered 252 , 216 , and 152 bp upstream of the transcription start site of the Escherichia coli glutamate synthase operon ( gltBDF ) and activates transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11395454" target="_blank">11395454</a>]<br></li>
<li>A om between the L o n herichia col a d DNA f the U DF sc i Don E. Wiese II1 , Brian R. Ernsting1 , Robert M. Blumenthal2 and Rowena G. Matthews1 * 1 Department of Biological Chemistry and Biophysics Research Division , University of Michigan , Ann Arbor , MI 48109-1055 , USA 2 Department of Microbiology and Immunology , Medical College of Ohio , Toledo , OH 43699-0008 , USA The global regulator Lrp ( leucine-responsive regulatory protein ) , in some cases modulated by its co-regulator leucine , has been shown to regulate more than 40 genes and operons in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9236118" target="_blank">9236118</a>]<br></li>
<li>JOURNAL OF BACTERIOLOGY , Feb. 1992 , p. / 92/041109 -10 $ 02.00 / 0 Copyright © 1992 , American Society for Microbiology Characterization of the Regulon Controlled by the Leucine - Responsive Regulatory Protein in Escherichia coli BRIAN R. ERNSTING , ' MARIETTE R. ATKINSON ,2 ALEXANDER J. NINFA ,2 AND ROWENA G. MATTHEWS ' * Biophysics Research Division and Department of Biological Chemistry , The University of Michigan , Ann Arbor , Michigan 48109,1 and Department of Biochemistry , Wayne State University School of Medicine , Detroit , Michigan 482012 Received 7 October 1991/Accepted 10 December 1991 The leucine-responsive regulatory protein ( Lrp ) controls the transcription of a number of genes in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1346534" target="_blank">1346534</a>]<br></li>
<li>This is the case for TyrR ,23 MalT ,24 BirA ,25 and NtrC .26 For eukaryotic nuclear receptors , including the vitamin D and retinoic acid receptors , regulatory ligands determine not only the oligomeric state of the receptor but als in ¯ uence the assembly of heterodimers of mixed receptor type , processes that have consequences for target site selection .27,28 In general , these effectorlinked assembly processes provide an indirect means of regulating DNA target-site occupancy , and ultimately transcriptional activity , via small changes in ligand concentration .25 One of the most interesting aspects of the Lrp regulon is the different regulatory patterns that relate to Lrp-leucine interactions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11575919" target="_blank">11575919</a>]<br></li>
<li>Therefore a simple model , in which the af ® nity of Lrp for its target DNA is the sole determinant of the degree of transcriptional activation of the target , is not suf - ® cient to describe in vivo regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9236118" target="_blank">9236118</a>]<br></li>
<li>Leucine affects the DNase I footprinting pattern Previous work in this laboratory has demonstrated that addition of leucine to the medium decreases the transcriptional activation of gltBDF by Lrp 2.2-fold , and that at least some of thi decrease in activation may be due to decreased af - ® nity of Lrp for gltBDF DNA ( Ernsting et al. , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9236118" target="_blank">9236118</a>]<br></li>
<li>The comparison shown in Figure 7C suggests that for a given Lrp/DNA af ® nity , the transcriptional activation by an Lrp leucine complex is roughly 3.6-fold greater than activation by Lrp alone .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9236118" target="_blank">9236118</a>]<br></li>
<li>In the case of gltBDF , this model and our data suggest that the 2.2-fold decrease in activation observed in the presence of leucine is the result of an 8.8-fold decrease in the af ® nity of Lrp for gltBDF DNA ( Figure 5 ) coupled with an ~ 3.6-fold increase in the activation ef ® - ciency by Lrp that is bound ( Figure 7C ) ( 8.8-fold decrease/3 .6 - fold increase 2.4-fold decrease ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9236118" target="_blank">9236118</a>]<br></li>
<li>Thus , independent data from two DNA binding studies predict an identical effective intracellular concentration of Lrp during exponential-growth in glucose minimal MOPS medium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7901196" target="_blank">7901196</a>]<br></li>
<li>We think that sduB is so little expressed in minimal medium , due to its poor ribosome binding site ( Shao and Newman , 1993 ) , that an effect of Lrp would not be noticed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8026499" target="_blank">8026499</a>]<br></li>
<li>In Fig. 5 , the predicted immediate effect of a rich medium containing leucine on transcription of gltBDF is a twofold decrease in expression ( inferred from the change in DNA-binding effected by leucine at 5.5 nM Lrp dimer , which is estimated to be the effective in vivo concentration during growth in glucose minimal MOPS medium [ 44 ] ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li>
<li>We predict that a shift from growth in a minimal medium to growth in a rich medium containing leucine would have an immediate effect , as the level of Lrp binding to the gltBD promoter is reduced twofold by leucine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7901196" target="_blank">7901196</a>]<br></li>
<li>Comparison of these concentrations of leucine with plots of the effect of leucine on mobility shift assays of Lrp binding togltBDF ( 44 ) suggest that intracellular leucine should have very little effect on binding during growth in glucose MOPS minimal medium and a nearly maximal effect on binding shortly after addition of leucine to the growth medium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li>
<li>In those studies , very little effect of leucine on OmpC and OmpF expression was seen during exponential-growth at 378C in glucose minimal MOPS medium , and these proteins were classiﬁed as leucine-insensitive members of the Lrp regulon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8002608" target="_blank">8002608</a>]<br></li>
<li>Furthera as more , that repression was independent of the presence of Lrp caused by the of leucine in LB , and was not presence excess because leucine had little effect on lrp expression in a minimal medium ( 10 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8144448" target="_blank">8144448</a>]<br></li>
<li>This model is consistent with the observed effect of exponential-growth in a rich medium on the expression of gltBDF : : lacZ fusions , which is predicted to arise in part from the presence of leucine and in part from decreased levels of Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7901196" target="_blank">7901196</a>]<br></li>
<li>To test the idea that the leucine effect on MccJ25 susceptibility could be mediated by Lrp , we compared the responses of the isogenic strains MC4100 and RO64 ( MC4100 lrp : : Tn10 ) to the antibiotic in M9 minimal medium and in the same medium containing either 12.5 or 25 g/ml of leucine by using a twofold serial dilution assay .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19074390" target="_blank">19074390</a>]<br></li>
<li>When CSH50 was grown in M9-glucose medium supplemented with 100 mg of leucine per ml , signiﬁcant repression of stpA relative to the medium without leucine was observed ( Fig. 4C ) , but the effect was not as dramatic as that of mutating Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9006049" target="_blank">9006049</a>]<br></li>
<li>The negative effect of Lrp on ArgP-mediated activation is further illustrated by the activities on arginine - , and arginine plus leucine-supplemented minimal medium which are even slightly higher in the lrp : : Tn10 mutant than in the Fig. 5 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19906180" target="_blank">19906180</a>]<br></li>
<li>This would be even more evident in LB medium , where there would be a considerable concentration of L-leucine and , as shown here , a very great decrease in Lrp concentration , so that degradative functions would be even more favored in LB than in minimal medium with L-leucine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1532173" target="_blank">1532173</a>]<br></li>
<li>For the third pair ( patterns 3 and 6 ) , Lrp either activates or represses but the Lrp-related regulation is relatively insensitive to leucine , at least during growth of cells in minimal medium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li>
<li>During growth in minimal media , the level of Lrp in cells is high and intracellular concentrations of leucine are relatively low .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li>
<li>The level of free Lrp is decreased by about two to threefold and 16-fold for cells grown in the medium supplemented with leucine and in rich-de ® ned med-ium , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11575919" target="_blank">11575919</a>]<br></li>
<li>Furthermore , we measured the amount of free Lrp and Lrp bound to DNA in cells grown in minimal medium and in minimal medium containing leucine ( 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12218014" target="_blank">12218014</a>]<br></li>
<li>Moreover , when cells are grown in minimal medium , we observe a more sustained induction of stpA which is dependent on the leucine-responsive regulatory protein ( Lrp ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9006049" target="_blank">9006049</a>]<br></li>
<li>These measurements allowed us to calculate that in cells grown in a minimal medium , about 40 % of the total Lrp is free and that free protein is distributed as 187 hexadecamer mole-cules , 99 octamers , and 65 leucine-bound octamers ( with almost no leucine-bound hexadecamers ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12218014" target="_blank">12218014</a>]<br></li>
<li>However , in the presence of leucine or in the absence of Lrp , a residual growth-phase-dependent regulation of stpA can still be observed , and the Lrp-dependent activation of stpA in minimal medium is still affected by growth-phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9006049" target="_blank">9006049</a>]<br></li>
<li>In contrast , the shortest promoter construct ( pAF327 ) was found if anything to be repressed in minimal medium relative to the situation in L broth and was unaffected by Lrp or leucine ( Table 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9006049" target="_blank">9006049</a>]<br></li>
<li>In E. coli , adhE is preferentially expressed in stationary phase [ 107-109 ] , and repressed by Lrp in a leucine-independent manner during exponential-growth in minimal glucose medium [ 69 ] and ABK , unpublished data ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18405378" target="_blank">18405378</a>]<br></li>
<li>Non-ideality inside the cell due to macromolecular crowding is expected to promote the formation of higher-order oligomers .30 Even if in vivo conditions favor Lrp dissociation leading to levels of dimer high enough to-bind DNA , the dimer will contribute to the thermodyn amic linkage between leucine-regulated Lrp selfassociation and DNA binding and may undergo DNA-dependent assembly to the hexadecamer as reported for transcription regulators such as Lac repressor31 ,32 and l repressor .33 The linkage can still lead to six different regulatory patterns if the nucleoprotein complexes assembled through dimer , octamer , or hexadecamer serve as active forms for some operons and those assembled through leucine-bound octamer are active forms for other operons .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11575919" target="_blank">11575919</a>]<br></li>
<li>In contrast , several transporters for aromatic amino acids ( brnQ , ilvKHMGF and ilvJHMGF ) appear to be regulated by the concerted mode , their expression is downregulated by Lrp and leucine potentiates this effect ( Cho et al. , 2008 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19906180" target="_blank">19906180</a>]<br></li>
<li>For many operons regulated by Lrp , leucine also affects expression , and it has been suggested that Lrp and leucine together constitute a global response mechanism in E.coli ( Ernsting et al. , 1992 ; Newman et al. , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8508774" target="_blank">8508774</a>]<br></li>
<li>According to the classiﬁcation of Cho et al. ( 2008 ) , argO belongs to the category of genes that are regulated in a reciprocal manner by Lrp and leucine , i.e. Lrp and leucine have opposite effects .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19906180" target="_blank">19906180</a>]<br></li>
<li>Transcription of the ilvIH operon is positively regulated by Lrp , the activator function of which is abolished by leucine ( Platko et al. 1990 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8152423" target="_blank">8152423</a>]<br></li>
<li>For example , leucine antagonized the repressive activity of Lrp in the sdaA operon ( Lin et al. , 1990 ) , whereas it was required for Lrp inhibition of the livJ operon ( Haney et al. , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10216857" target="_blank">10216857</a>]<br></li>
<li>Additional in vitro analysis will be necessary to determine whether alanine/leucine are absolutely required for activation or , as suggested for the gltBDF operon ( Wiese et al. , 1997 ) , they increase the ef ® ciency of Lrp activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10216857" target="_blank">10216857</a>]<br></li>
<li>Some polypeptides identified by Ernsting et al. by two-dimen-sional gel electrophoresis , including OmpC , show the opposite pattern : Lrp represses , and the repression is insensitive to leucine ( 43 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li>
<li>Thus , Lrp has been considered to be responsible for greatly increased sensitivity toward growth inhibition by leucine , valine and serine , when it was mutated ( Quay et al. , 1977 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16164569" target="_blank">16164569</a>]<br></li>
<li>Recent studies have shown that lysU is repressed by Lrp and is induced by addition of leucine or gene-knockout mutations ( Linet al. 1992a ; Gazeau et al. 1992 ; Ito et al. 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7603430" target="_blank">7603430</a>]<br></li>
<li>According to this model , the systems most responsive to exogenous leucine are those that bind Lrp relatively weakly ( Ernsting et al. , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9515701" target="_blank">9515701</a>]<br></li>
<li>For lrp strains growing in minimal media , the effect of leucine on the ability of Lrp to activate or repress the expression of members of the Lrp regulon varies greatly , ranging from overcoming the effect of Lrp ( e.g. ilvIH ) , to having nearly no effect ( e.g. gltBDF , papBA , gcv ) , to potentiating the effect of Lrp ( e.g. livJ , livK , ® mB-promoted switching ) .1 Leucine-regulated self-association of Lrp may have relevance to promoter selection and by extension to the different regulatory patterns exhibited by Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11575919" target="_blank">11575919</a>]<br></li>
<li>It can also be seen that for strains growing in minimal media , the ability of leucine to antagonize the effect of Lrp on transcription varied greatly , with the effect of exogenous leucine ranging from substantial antagonism of the effect of Lrp ( tdh , sdaA ) to nearly no effect ( gltBDF , papBA , gcv , ompC ) to potentiation of the effect of Lrp ( livJ , livK ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li>
<li>Another possibility is that , in cells containing high levels of Lrp , leucine prevents degradation of liv mRNAs or of proteins involved in transport , thus leading to the accumulation of transport capacity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1729203" target="_blank">1729203</a>]<br></li>
<li>Two operons involved in high-affinity transport of branched-chain amino acids , the livJ and livKHMGF operons , are negatively regulated by Lrp , and that negative regulation is potentiated by leucine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li>
<li>For the proteins involved in high-affinity branched-chain amino acid transport , regulation by Lrp is negative and is stimulated in the presence of leucine ( 43 , 62 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li>
<li>The livK gene is repressed by Lrp when exogenous leucine is present [ 70,96 ] , and activated when leucine is not in the medium [ 66 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18405378" target="_blank">18405378</a>]<br></li>
<li>The final Lrp concentration in the dialysis chambers was 0.74 , uM , while the leucine concentration ranged from 0.5 to 70 jiM in the different chambers , with the same specific activity of 2.43 x 105 cpm/nmol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8071194" target="_blank">8071194</a>]<br></li>
<li>The Lrp leucine response-mutations correspond to the following residues of AsnC : Y100 , S106 , I116 , Q128 , L139 , I140 and V141 ( Figure 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16528101" target="_blank">16528101</a>]<br></li>
<li>The differing affinities of the target genes for Lrp create a hierarchy of response to changes in Lrp and/or leucine concentration .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li>
<li>The leucine effect seems to be mediated by the global regulator Lrp , which acts as a positive regulator of an endog-enous chromosome-encoded MccJ25 efﬂux pump , YojI .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19074390" target="_blank">19074390</a>]<br></li>
<li>The phase variation of type 1 ﬁmbria in E. coli provides a unique system in which to investigate the effects of leucine on Lrp , as it is the only known example in which Lrp is a positive regulator and leucine potentiates this effect .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9515701" target="_blank">9515701</a>]<br></li>
<li>The ﬁm system is unique , as it provides the only known example in which Lrp is a positive regulator and leucine potentiates this effect ( Blomﬁeld et al. , 1993 ; Gally et al. , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9515701" target="_blank">9515701</a>]<br></li>
<li>The model assumes that leucine promotes the selective dissociation of Lrp from site 3 in vivo in the same way that it does in vitro and that this effect explains why leucine stimulates the ﬁm switch .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9515701" target="_blank">9515701</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_392">
The hexadecamer , but not the octamer , binds with weak or moderate af ® nity and leucine overcomes the effect of Lrp by decreasing the amount of hexadecamer .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11575919" target="_blank">11575919</a>]<br></label>
<input id="act_similarity_392", type="checkbox">
<div class="ACT_similar">
<li>The leucine-bound octamer , but not the hexadecamer , binds with weak or moderate af ® nity and leucine potentiates the effect of Lrp by increasing the amount of octamer .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11575919" target="_blank">11575919</a>]<br></li></div>
<li>The effect of Lrp in any particular case may depend on the design of the promoter , the affinity of Lrp for L-leucine , and the relative affinity of free Lrp and leucine-Lrp for particular sequences .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1532173" target="_blank">1532173</a>]<br></li>
<li>The Lrp-DNA interactions studied in this work suggest that the primary effect of leucine is to modulate the affinity of Lrp for DNA over a relatively narrow range .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7901196" target="_blank">7901196</a>]<br></li>
<li>Taken together , these data suggest that a major function of alanine/leucine in activation is to convert bound Lrp into an activator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10216857" target="_blank">10216857</a>]<br></li>
<li>Leucine can antagonize the action of Lrp , when it acts as either an activator or a repressor ; however , in some cases Lrp requires leucine to act as a repressor and in other cases the Lrp activation function is unaffected by leucine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8152423" target="_blank">8152423</a>]<br></li>
<li>Similarly , strain CP8 , in which activation by Lrp is lessened by leucine ( class 1 ) , was unable to use several carbon sources , D-xylose , D-ribose , and rhamnose , that both the lrp mutant and its parent are able to use .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1532173" target="_blank">1532173</a>]<br></li>
<li>Among the operons activated by Lrp , in some cases that activation is abolished by leucine ( as is the case for the ilvIH operon ) , in some cases the activation requires leucine , and in yet other cases the activation is independent of leucine ( 8 , 13 , 18 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8432705" target="_blank">8432705</a>]<br></li>
<li>For some of the operons activated by Lrp , that activation is negated by leucine ( as is the case for the ilvIH operon ) , requires leucine , or is independent of leucine ( 3 , 11 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1447135" target="_blank">1447135</a>]<br></li>
<li>Among operons activated by Lrp , in some cases that activation is overcome by leucine , in other cases the activation requires leucine , and in yet other cases the activation is independent of leucine ( 10 , 13 , 20 , 25 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7665463" target="_blank">7665463</a>]<br></li>
<li>Among the operons activated by Lrp , in some cases that activation is overcome by leucine , in other cases the activation is potentiated by leucine , and in yet other cases the activation is independent of leucine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11575919" target="_blank">11575919</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_403">
Of the operons activated by Lrp , in some cases the activation requires leucine , in some cases the activation is negated by leucine , and in other cases the activation is independent of leucine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8144448" target="_blank">8144448</a>]<br></label>
<input id="act_similarity_403", type="checkbox">
<div class="ACT_similar">
<li>Of the operons activated by Lrp , in some cases that activation is negated by leucine , in some cases the activation requires leucine , and in yet other cases the activation is independent of leucine ( Ernsting etal .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8483418" target="_blank">8483418</a>]<br></li></div>
<li>The hexadecamer is postulated to be the active form for those operons that are activated or repressed by Lrp and for which activation or repression is overcome by leucine ( Figure 8 ( b ) ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11575919" target="_blank">11575919</a>]<br></li>
<li>How is it that leucine can overcome the effect of Lrp in one case ( for example , the activation of ilvIH ) , and potentiate the effect of Lrp in another case ( for example , the repression of liv operons by Lrp ) ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li>
<li>However , it does not explain how leucine can potentiate the repression of liv operons by Lrp or potentiate the activation of an operon by Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li>
<li>Strain CP5 is of interest because the regulatory patterns of CP5 and CP36 are quite different : the insert in strain CP36 is the only one known which requires both Lrp and leucine to repress it , while the inserts in strains CP5 and CP28 are strongly activated by Lrp and strongly repressed by leucine ( 13 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7751276" target="_blank">7751276</a>]<br></li>
<li>It is proposed that leucine regulates the equilibrium between Lrp oligomers and thus Lrp occupancy of sites within different operons , leading to diverse regulatory patterns .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11575919" target="_blank">11575919</a>]<br></li>
<li>In most cases , Lrp regulation of gene activity either requires or is antagonized by leucine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955407" target="_blank">8955407</a>]<br></li>
<li>In class 3 mutants ( 4 isolates ) , Lrp and leucine together activate 3-galactosidase activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1532173" target="_blank">1532173</a>]<br></li>
<li>In others ( class 2 ) , Lrp repressed and L-leucine increased activity , as in lysU and sdaA ( 11 , 12 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1532173" target="_blank">1532173</a>]<br></li>
<li>In summary , the results of three independent methods , sedimentation equilibrium , sedimentation velocity , and total-intensity light scattering , all indicated that Lrp at mM concentrations and in the presence of 10 mM leucine has an octameric structure .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11575919" target="_blank">11575919</a>]<br></li>
<li>In the absence of leucine , sites 1 , 2 and 3 are occupied even at the lowest Lrp concentrations tested .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9515701" target="_blank">9515701</a>]<br></li>
<li>Both the GcvA and leucine-responsive regulatory ( Lrp ) proteins are required for regulation of the gcv operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10217490" target="_blank">10217490</a>]<br></li>
<li>As mentioned in the Introduction , one of the interesting features of the Lrp regulon is the number of different patterns of regulation that result from the interaction of Lrp , leucine , and perhaps other proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_417">
One of the most striking aspects of the Lrp regulon is the number of different patterns of regulation fhat are caused by the interaction of Lrp and leucine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8483418" target="_blank">8483418</a>]<br></label>
<input id="act_similarity_417", type="checkbox">
<div class="ACT_similar">
<li>One of the most striking aspects of the Lrp regulon is the number of different patterns of regulation that are caused by the interaction of Lrp and leucine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8432705" target="_blank">8432705</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1447135" target="_blank">1447135</a>]<br></li></div>
<li>Thus , via small changes in its level , leucine might regulate Lrp occupancy to different operons through its regulation of Lrp self-association state , leading to promoter selection .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11575919" target="_blank">11575919</a>]<br></li>
<li>These patterns of regulation are those expected if Lrp were to act as an activator or repressor only in the absence of exogenous leucine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1346534" target="_blank">1346534</a>]<br></li>
<li>Thus , as leucine also promotes dissociation of Lrp from ﬁm , the regulation of the ﬁm switch appears paradoxical .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9515701" target="_blank">9515701</a>]<br></li>
<li>This is the first operon identified in which Lrp regulation is unaffected by leucine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1532173" target="_blank">1532173</a>]<br></li>
<li>This regulation depends on Lrp only and not on L-leucine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1532173" target="_blank">1532173</a>]<br></li>
<li>Leucine modulates Lrp regulation of leucine-responsive operons .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9236118" target="_blank">9236118</a>]<br></li>
<li>First , gltBDF is one of the operons for which Lrp regulation is relatively insensitive to the coregulator leucine [ 5,28 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17233899" target="_blank">17233899</a>]<br></li>
<li>We suggested ( 6 ) that the regulation of leucine-insensitive genes and operons by Lrp might be modulated in response to altered physiological conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7901196" target="_blank">7901196</a>]<br></li>
<li>To probe the regulation of leucine-insensitive genes by Lrp , we investigated the regulation of glutamate synthase by Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7901196" target="_blank">7901196</a>]<br></li>
<li>According to this model , hexadecamer and leucine-bound octamer are active forms of Lrp that regulate transcription of different operons .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11575919" target="_blank">11575919</a>]<br></li>
<li>Leucine-bound octamer , by contrast , is the active form for those operons that are activated or repressed by Lrp and for which the effects of Lrp are potentiated by leucine ( Figure 8 ( c ) ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11575919" target="_blank">11575919</a>]<br></li>
<li>If the bound Lrp leucine complex is a superior activator for gltBDF relative to bound Lrp alone , then a re ® nement is needed for the model explaining the variability in sensitivity of Lrp-regulated operons to leucine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9236118" target="_blank">9236118</a>]<br></li>
<li>The attenuation of the effect of Lrp by exogenous leucine has also been shown for some operons regulated by Lrp ( 5 , 26 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19074390" target="_blank">19074390</a>]<br></li>
<li>Like many of the operons regulated by Lrp , ilvIH is also regulated by leucine ( 3 , 11 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1447135" target="_blank">1447135</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_433">
Many operons under control of Lrp are also subject to control by leucine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8144448" target="_blank">8144448</a>]<br></label>
<input id="act_similarity_433", type="checkbox">
<div class="ACT_similar">
<li>Many operons that are regulated by Lrp are also subject to control by leucine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8483418" target="_blank">8483418</a>]<br></li></div>
<li>If Lrp acts directly to activate or repress all of these operons , then leucine must interfere with Lrp action in some cases and promote Lrp action in other cases .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1729203" target="_blank">1729203</a>]<br></li>
<li>Lrp directly represses its own promoter in E. coli , and this occurs whether or not leucine is present [ 89 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18405378" target="_blank">18405378</a>]<br></li>
<li>The EMSA was performed with 5.5 nM Lrp and increasing concentrations of leucine ( Fig. 4A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9515701" target="_blank">9515701</a>]<br></li>
<li>A. Electrophoretic mobility shift assay of radiolabelled PCR12 incubated with 5.5 nM Lrp and increasing concentrations of leucine ( 0 -- 32 mM ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9515701" target="_blank">9515701</a>]<br></li>
<li>This possibility is interesting in view of recent studies from our laboratory demonstrating that Lrp exists at micromolar concentrations as a hexadecamer and/or an octamer and that leucine shifts the equilibrium in favor of the octamer ( 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12218014" target="_blank">12218014</a>]<br></li>
<li>This latter result can now be understood in terms of Casamino Acids leading to lower concentrations of a leucine-insensitive Lrp , as discussed above .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li>
<li>At lower concentrations of Lrp , transcription of the target gene should be sensitive to leucine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7901196" target="_blank">7901196</a>]<br></li>
<li>At low concentrations of Lrp , the leucine sensitivity is infinite .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7901196" target="_blank">7901196</a>]<br></li>
<li>At that concentration in vitro , Lrp exists mainly as an octamer or hexadecamer , and millimolar concentrations of leucine promote conversion of hexadecamer to octamer .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12218014" target="_blank">12218014</a>]<br></li>
<li>At low levels of Lrp , leucine reduced expression much less than expected from its effects at high Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11985712" target="_blank">11985712</a>]<br></li>
<li>where data from several experiments -LCB- High levels of leucine are needed to antagonize Lrp effects both in vivo and in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9236118" target="_blank">9236118</a>]<br></li>
<li>Effects of Leucine on Lrp-Mediated Regulation At least some effects of leucine in vivo are mediated by Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li>
<li>Conceivably , leucine regulates Lrp activities through its effects on the C-terminal domain interactions that lead to Lrp self-association .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11575919" target="_blank">11575919</a>]<br></li>
<li>A model that accounts for the diverse effects of leucine on Lrp function .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11575919" target="_blank">11575919</a>]<br></li>
<li>Thus , it is possible that PapI interacts directly with Lrp and that this interaction is responsible for the observed leucine insensitivity of the pap operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li>
<li>The ability of leucine to control the action of Lrp varies from one operon to another .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8152423" target="_blank">8152423</a>]<br></li>
<li>Depending on the operon , Lrp can function independently , or its action can be enhanced or antagonized by leucine or alanine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10216857" target="_blank">10216857</a>]<br></li>
<li>Effects of Lrp and leucine on gltBDF a.nd ilvIH operon fusions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7901196" target="_blank">7901196</a>]<br></li>
<li>This is surprising given our in vivo data showing that leucine antagonizes activation of gltBDF by Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9236118" target="_blank">9236118</a>]<br></li>
<li>This now seems to be explained by the fact that , at low Lrp , most of the activation is by Crp and unaffected by leucine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11985712" target="_blank">11985712</a>]<br></li>
<li>This , then , is the first representative we have identified of the sixth class of Lrp-leucine interactions , those promoters which are repressed by Lrp but unaffected by leucine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1532173" target="_blank">1532173</a>]<br></li>
<li>This means that livJ is still the only gene which requires both Lrp and leucine to repress it .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7751276" target="_blank">7751276</a>]<br></li>
<li>Furthermore , as the concentration of Lrp drops , Lrp-activated promoters should become more sensitive to leucine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7901196" target="_blank">7901196</a>]<br></li>
<li>Many of these pathways are normally repressed by Lrp in a leucine-sensitive fashion and would remain constitutively repressed in strain BE3 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li>
<li>Observations from this laboratory ( R. Gibbs & R. G. M. , unpublished data ) indicate that leucine increases the hydrodynamic radius of Lrp as assessed by gel ® ltration .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9236118" target="_blank">9236118</a>]<br></li>
<li>Lrp without leucine activates synthesis of GOGAT and pyridine nucleotide transhydrogenase , which together provide two of the three substrates for ammonia assimilation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12730324" target="_blank">12730324</a>]<br></li>
<li>Lrp activity without L-leucine effector : autoregulation as an of Irp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1532173" target="_blank">1532173</a>]<br></li>
<li>Lrp activates gcv but is insensitive to the coregulator leucine ( 7 , 10 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7928983" target="_blank">7928983</a>]<br></li>
<li>Berthiaume , F. , Crost , C. , Labrie , V. , Martin , C. , Newman , E.B. , and Harel , J. ( 2004 ) Inﬂuence of L-leucine and L-alanine on Lrp regulator of foo , coding for F1651 , a Pap homologue .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19906180" target="_blank">19906180</a>]<br></li>
<li>Inﬂuence of l-leucine and l-alanine on Lrp regulation of foo , coding for F1651 , a Pap homologue .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2162465" target="_blank">2162465</a>]<br></li>
<li>We suggest that the conformation of Lrp may change according to whether L-leucine is bound .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1532173" target="_blank">1532173</a>]<br></li>
<li>We conclude that Lrp mediates repression of both avtA and yfbQ by alanine and leucine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20729367" target="_blank">20729367</a>]<br></li>
<li>We showed that alanine or leucine via Lrp modestly represses YfbQ/AlaA and AvtA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20729367" target="_blank">20729367</a>]<br></li>
<li>A paradoxical result noted in Fig. 4 related to the fact that leucine caused an increase in leucine transport in a strain overproducing Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1729203" target="_blank">1729203</a>]<br></li>
<li>One possible explanation for stimulation of transport by leucine is that there is another system capable of transporting leucine that is active when leucine is added to a culture of a strain overproducing Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1729203" target="_blank">1729203</a>]<br></li>
<li>One could summarize this with the idea that Lrp increases biosynthetic functions and decreases degradative ones , a tendency that is partly reversed with exogenous L-leucine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1532173" target="_blank">1532173</a>]<br></li>
<li>Leucine stimulation of fim inversion could result from direct interaction of leucine with Lrp at the fim switch .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7916011" target="_blank">7916011</a>]<br></li>
<li>These leucine-Lrp operons have been classified into six subclasses by defining three subcategories each of positively and negatively controlled operons in which leucine negates the effect , leucine is required for the effect , or leucine has no effect ( Lin et al. 1992b ; Ernsting et al. 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7603430" target="_blank">7603430</a>]<br></li>
<li>Similarly , for operons which are negatively regulated by Lrp , the same three subcategories have been observed : leucine negates the effect , leucine is required for the effect , and leucine has no effect .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8144448" target="_blank">8144448</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_474">
Similarly , for those operons whose expression is repressed by Lrp , the same three subcategories have been recognized : leucine overcomes the effect , leucine potentiates the effect , or leucine has no effect .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11575919" target="_blank">11575919</a>]<br></label>
<input id="act_similarity_474", type="checkbox">
<div class="ACT_similar">
<li>Similarly , for operons whose expression is repressed by Lrp , the same three subcategories have been recognized ; leucine negates the effect , leucine is required for the effect , or leucine has no effect ( 3 , 11 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1447135" target="_blank">1447135</a>]<br></li>
<li>Similarly , for those operons whose expression is repressed by Lrp , the same three subcategories have been recognized ; leucine overcomes the effect , leucine is required for the effect , or leucine has no effect .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7665463" target="_blank">7665463</a>]<br></li>
<li>Similarly , for those operons whose expression is repressed by Lrp , the same three subcategories have been recognized ; leucine abolishes the effect , leucine is required for the effect , and leucine has no effect .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8432705" target="_blank">8432705</a>]<br></li>
<li>Similarly , for those operons whose expression is repressed by Lrp , the same three subcategories have been recognized : leucine neg-ates the effect , leucine is required for the effect , and leucine has no effect .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8483418" target="_blank">8483418</a>]<br></li></div>
<li>Thus , the model explains a minimal effect of leu-cine on the fractional saturation of operators by predicting that the in vivo concentration of Lrp is well above its Kpromoter d in cases where the promo-ters show little leucine effect .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9236118" target="_blank">9236118</a>]<br></li>
<li>Since the amplitude of leucine regulation of serA is relatively small , it is possible that the observed effect of leucine-Lrp is indirect or that this promoter has a target for Lrp that can not be identified by the computer algorithm that was employed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1917830" target="_blank">1917830</a>]<br></li>
<li>Genes may either be activated or repressed by Lrp , and leucine may in each case antagonize , potentiate , or have no effect on this regulation ( 2 , 14 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12218014" target="_blank">12218014</a>]<br></li>
<li>Exogenous leucine reduces transcription of the ilvIH operon 5 - to 10-fold in vivo , an effect that is mediated by Lrp ( 6 , 13 , 15 , 19 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1447135" target="_blank">1447135</a>]<br></li>
<li>Thus , for these operons , the effect of leucine on transcription appears to be mediated by Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li>
<li>Similarly , for cases in which Lrp acts negatively , there are examples in which leucine overcomes the effect , is required for the effect , or has no effect upon Lrp-mediated repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19074390" target="_blank">19074390</a>]<br></li>
<li>Lrp can act as either an activator or a repressor , and for each of these situations , examples are known in which leucine overcomes the effect of Lrp , leucine is required for the effect of Lrp , or leucine has no effect upon Lrp action ( 25 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7665463" target="_blank">7665463</a>]<br></li>
<li>Lrp can function as an activator or as a repressor , and in both cases leucine can be required for or can abolish the Lrp effect ( 1 , 3 , 4 , 9-12 , 14 , 22 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8071194" target="_blank">8071194</a>]<br></li>
<li>Lrp activates the argO promoter fourfold ; exogenous leucine antagonizes , but does not completely eliminate this effect .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19906180" target="_blank">19906180</a>]<br></li>
<li>However , in these instances Lrp has a negative effect on promoter activity and leucine antagonizes this effect .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19906180" target="_blank">19906180</a>]<br></li>
<li>Therefore , Lrp appears to act as an activator for PargO and leucine , as a cofactor , reduces this stimulatory effect about twofold .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19906180" target="_blank">19906180</a>]<br></li>
<li>Furthermore , under at least some conditions , leucine reversed the stimulatory effect of Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1447135" target="_blank">1447135</a>]<br></li>
<li>( c ) Operon III ; leucine potentiates the effect of Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11575919" target="_blank">11575919</a>]<br></li>
<li>( A ) Repression by Lrp and effect of alanine and leucine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955407" target="_blank">8955407</a>]<br></li>
<li>In two cases , leucine has little or no effect ; Lrp is absolutely required for transcription of gcv and represses its own expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1532173" target="_blank">1532173</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_494">
b In class 1 mutants ( 11 isolates ) , Lrp activates and leucine decreases its effect .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1532173" target="_blank">1532173</a>]<br></label>
<input id="act_similarity_494", type="checkbox">
<div class="ACT_similar">
<li>In class 2 mutants ( 6 isolates ) , Lrp represses and leucine decreases its effect .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1532173" target="_blank">1532173</a>]<br></li></div>
<li>In some cases , leucine potentiates the effect of Lrp ; in other cases it overcomes the effect , and in yet other cases it has no effect on the action of Lrp ( for reviews , see references 5 and 24 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955316" target="_blank">8955316</a>]<br></li>
<li>In some cases , leucine overcomes the effect of Lrp , in other cases it is required for the effect , and in yet other cases it does not seem to influence the effect of Lrp ( 1 , 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8636076" target="_blank">8636076</a>]<br></li>
<li>Also , for some operons controlled by Lrp , the effect of Lrp in vivo is overcome by exogenous leucine , whereas in others , the effect of Lrp is seen only in the presence of exogenous leucine ( 18 , 25 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7665463" target="_blank">7665463</a>]<br></li>
<li>For some operons , exogenous leucine reduces the extent of Lrp action , for others it potentiates the effect of Lrp , and for yet other operons it has no effect .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11575919" target="_blank">11575919</a>]<br></li>
<li>Leucine did not inhibit the growth of strain BE3 in the presence of exogenous isoleucine and valine , suggesting that the toxic effect of leucine on growth of lrp + strains is mediated by the effect of leucine on Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li>
<li>These differences were magnified in the presence of leucine , where P. mirabilis Lrp was unique in showing virtually no effect ( Fig. 4E ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18405378" target="_blank">18405378</a>]<br></li>
<li>Expression of the kbl/tdh operon is induced by the presence of leucine , an effect mediated through Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7748928" target="_blank">7748928</a>]<br></li>
<li>Lrp may act positively or negatively or have no effect , and Lrp in the presence of exogenous leucine may act in any of these ways .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1346534" target="_blank">1346534</a>]<br></li>
<li>Leucine is known to modulate the effect of Lrp as an activator ( 12 , 27 , 44 ) or repressor ( 11 , 26 -- 28 ) of many genes in the Lrp regulon , and so we examined the effect of leucine on expression of the ompC-lacZ1 operon fusion in lrp1 and lrp backgrounds .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8002608" target="_blank">8002608</a>]<br></li>
<li>Lrp acts negatively or positively and in either mode the effect of Lrp may be enhanced , reduced or left unchanged by exogenous leucine , depending on the target gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19906180" target="_blank">19906180</a>]<br></li>
<li>Also , for each target gene , there will be an Lrp concentration at which the effect of leucine on expression is maximal , i.e. , a concentration of Lrp that balances the diminishing expression of a positively regulated gene as the Lrp concentration is decreased with the increasing leucine sensitivity as the Lrp concentration is decreased .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li>
<li>We suggest that a reduced expression of LIV-I/LS system and an increased expression of YeaS , both mediated by Lrp , constitute a mechanism that regulates intracellular leucine concentration and protects the cell against the toxic effect of this amino acid and its analogues .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16098526" target="_blank">16098526</a>]<br></li>
<li>Thus , it was thought that a sufficient level of Lrp is required to activate the expression of L-valine biosynthesis genes in the presence of exogenous L-leucine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17463081" target="_blank">17463081</a>]<br></li>
<li>For these leucine-sensitive genes and operons , Lrp can be a positive or a negative regulator of expression , and the effect of Lrp is antagonized in the presence of exogenous leucine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7901196" target="_blank">7901196</a>]<br></li>
<li>Thus , for example , Lrp acts negatively on livJ and livKHMGF expression , and this negative effect requires the presence of leucine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li>
<li>Lrp has several modes of action : It can either activate or repress gene expression ; leucine can reverse , enhance , or have no effect on this regulation ( 68 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12730324" target="_blank">12730324</a>]<br></li>
<li>Leucine weakly antagonized the effect of Lrp on ompC expression , and , as in other cases , the effect of leucine required a functional Lrp protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li>
<li>While the effects of leucine on ompC expression were small , this result strengthens the conclusion that ompC is regulated by Lrp , in that the effect of leucine depends on the presence of a functional Lrp protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8002608" target="_blank">8002608</a>]<br></li>
<li>For those cases in which Lrp acts positively as an activator , leucine sometimes overcomes the effect of Lrp ( thus causing reduced expression ) , sometimes intensiﬁes the effect of Lrp , and sometimes has no effect on Lrp-mediated activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19074390" target="_blank">19074390</a>]<br></li>
<li>For the first pair ( patterns 1 and 4 ) , Lrp activates or represses expression and leucine antagonizes the effect of Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li>
<li>For the second pair ( patterns 2 and 5 ) , Lrp again activates or represses expression but the action of Lrp is potentiated by leucine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li>
<li>For example , the pap operon of E. coli requires Lrp for expression , but this effect of Lrp is independent of leucine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li>
<li>Table 3 lists the effects of leucine on expression of lacZ operon fusions to a number of members of the Lrp regulon and compares the effect of leucine in lrp + and lrp backgrounds .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li>
<li>In an effort to understand how leucine affects Lrp-mediated expression , we examined Lrp self-association and the effect of leucine on self-association using light scattering , chemical cross-link-ing , and analytical ultracentrifugation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11575919" target="_blank">11575919</a>]<br></li>
<li>Electrophoretic analysis of the Lrp regulon established that target proteins differ greatly in the degree to which the effect of Lrp on their expression is antagonized by leucine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7901196" target="_blank">7901196</a>]<br></li>
<li>Exogenous leucine antagonizes the effect of Lrp , leading to a diminished expression of the pump and an increased susceptibility to microcin J25 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19074390" target="_blank">19074390</a>]<br></li>
<li>Exog-enous leucine would antagonize the effect of Lrp , leading to a diminished expression of the pump and to the accumulation of microcin in AB1133 cells .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19074390" target="_blank">19074390</a>]<br></li>
<li>In addition to controlling a large number of genes , Lrp generates a wide variety of regulatory patterns : examples exist in which the co-regulator leucine inhibits , is required for , or has little effect on repression or activation by Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9236118" target="_blank">9236118</a>]<br></li>
<li>In summary , this study demonstrates that the presence of Lrp is required for expression of YojI and that addition of leucine to the culture medium reduces this activation , presumably by antagonizing the effect of Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19074390" target="_blank">19074390</a>]<br></li>
<li>In addition , the presence of leucine can enhance , reduce or have no effect on activation or repression by Lrp ( Calvo and Matthews , 1994 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9663683" target="_blank">9663683</a>]<br></li>
<li>For genes whose expression is affected by Lrp only in the presence of leucine ( Haney et al. , 1992 ; Lin et al. , 1992a ) , the effect of leucine on transcriptional activation ( repression ) may dominate the effect of leucine on the af ® nity of Lrp for promoter DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9236118" target="_blank">9236118</a>]<br></li>
<li>Lrp is a transcriptional-activator of ilvIH expression ( 29 ) , and the presence of leucine in the medium reduces this activation 5 - to 10-fold ( 9 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7901196" target="_blank">7901196</a>]<br></li>
<li>Taken together , it was assumed that the L-leucine requirement in our system may cause insufficient activation of ilvIH expression by Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17463081" target="_blank">17463081</a>]<br></li>
<li>Evidence that Lrp directly activates expression from the ilvIH promoter and that leucine reduces the extent of activation was summarized above .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li>
<li>The expression ratios are completely reversed in the pDIA510 strains , which could have an excess of H-NS protein , and the biggest difference is seen at high osmolarity in the absence of leucine where we believe that Lrp activator is rendered limiting by a high concentration of active H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8152423" target="_blank">8152423</a>]<br></li>
<li>The regulation of the gcv operon by Lrp has not been extensively studied , although it has been noted that the expression of a gcv-lacZ operon fusion was almost insensitive to leucine ( 84 ) ( Table 3 ) but highly sensitive to the presence or absence of Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li>
<li>The absence of Lrp decreased transcription of the lacZ insert to a very low level , 4 % of the parent strain , regardless of whether L-leucine was present .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1532173" target="_blank">1532173</a>]<br></li>
<li>The coregulator for Lrp is leucine , and exogenous leucine reduces the expression of gltBDF about twofold , although this level is still substantially above baseline transcription in the absence of Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11395454" target="_blank">11395454</a>]<br></li>
<li>The regulation of gltBDF expression by Lrp is not absolutely independent of leucine but is better described as insensitive to leucine , relative to the regulation of ilvIH transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7901196" target="_blank">7901196</a>]<br></li>
<li>Altogether , six different patterns of regulation by Lrp have been recognized , depending upon whether Lrp acts negatively or positively and upon the way in which leucine affects expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19074390" target="_blank">19074390</a>]<br></li>
<li>In this work , we have provided evidence consistent with the direct regulation of gltBDF expression by Lrp and demonstrating that 10 mM exogenous leucine decreases the expression of this operon approximately twofold .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7901196" target="_blank">7901196</a>]<br></li>
<li>In one class of operons including ilvIH and serA , Lrp acts positively to increase operon expression and leucine seems to interfere with Lrp activity , resulting in lower expression ( 11 , 16 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1729203" target="_blank">1729203</a>]<br></li>
<li>In a second class of operons , Lrp acts negatively and leucine induces operon expression , presumably by interfering with Lrp action .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1729203" target="_blank">1729203</a>]<br></li>
<li>The ability of alanine or leucine to strongly antagonize Lrp repression of dad is consistent with the positive role of these amino acids in the expression of numerous operons that are negatively regulated by Lrp ( 3 , 21 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955407" target="_blank">8955407</a>]<br></li>
<li>According to our results , Lrp potentiates the ArgP-mediated lysP expression when cells are cultivated in the absence of lysine , irrespective of L-leucine availability .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21441513" target="_blank">21441513</a>]<br></li>
<li>Taken together , these data suggest that Lrp represses dad and that alanine and leucine act positively on dad expression by blocking Lrp action .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955407" target="_blank">8955407</a>]<br></li>
<li>These results indicate that Lrp represses yeaS , and that leucine and a number of other amino acids act positively on yeaS expression , presumably by blocking Lrp repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16098526" target="_blank">16098526</a>]<br></li>
<li>For livJ and livKHMGF , Lrp represses expression and leucine is required for this repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1729203" target="_blank">1729203</a>]<br></li>
<li>Taken together , these results suggest that Lrp acts directly or indirectly to repress livJ and livK expression and that leucine is required for this repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1729203" target="_blank">1729203</a>]<br></li>
<li>Together , these results support the idea that Lrp serves to repress livJ and livK expression and that leucine acts as a corepressor for repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1729203" target="_blank">1729203</a>]<br></li>
<li>The Lrp stimulation of stpA expression was -2 - fold , and exogenous leucine caused a 2-fold repression ( Table I ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8890170" target="_blank">8890170</a>]<br></li>
<li>The fact that Lrp is required for expression of at least one gene and decreases expression at another suggests , by extension , that it may be used to activate and silence genes and that Lrp is independent of L-leucine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1532173" target="_blank">1532173</a>]<br></li>
<li>This confirms that induction of CP41 and CP52 insert-gene expression by L-leucine is mediated by Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8026499" target="_blank">8026499</a>]<br></li>
<li>A striking feature of the Lrp regulon is the variety of ways that leucine and Lrp interact to regulate gene expression ( 16 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8144448" target="_blank">8144448</a>]<br></li>
<li>The first part is concerned with the broad question of how Lrp and leucine interact to regulate gene expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li>
<li>Regulation of the concentration of Lrp in response to changing growth conditions might provide a way to regulate Lrp-dependent genes that appear to be insensitive to leucine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7901196" target="_blank">7901196</a>]<br></li>
<li>In contrast to these `` switch-like '' regulatory proteins that turn transcription of the target gene off or on in response to the concentration of the regulatory ligand , Lrp modulates rather than abolishes transcription of target genes in response to leucine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li>
<li>Lrp is a bifunctional transcription factor that activates or represses many genes , including genes involved in leucine biosynthesis and transport , as well as many that are not directly involved in leucine metabolism ( leucine-Lrp regulon ) ; ( for review , see Newman et al. 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7603430" target="_blank">7603430</a>]<br></li>
<li>One of its most severe problems comes from the fact that Lrp is needed to activate genes involved in L-leucine biosynthesis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1532173" target="_blank">1532173</a>]<br></li>
<li>However , L-leucine binds to Lrp and prevents it from activating L-valine biosynthesis genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17463081" target="_blank">17463081</a>]<br></li>
<li>Subsequently , leucine regulates the expression of numerous genes involved in amino acid metabolism through Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17289841" target="_blank">17289841</a>]<br></li>
<li>Last , we tested leucine , because Lrp is known to activate the expression of serA and other genes involved in one-carbon metabolism ( 4 , 35 , 52 ) , and leucine addition reduces serA transcription ( 35 , 52 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9171388" target="_blank">9171388</a>]<br></li>
<li>The amino acid L-leucine might act as a coeffector of Lrp and potentiates , overcomes , or has no effect on the function of Lrp upon its target genes ( 6 , 33 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21441513" target="_blank">21441513</a>]<br></li>
<li>The presence of leucine in rich medium is predicted to produce an immediate decrease in the effect of Lrp on positively regulated target genes , with a hierarchy of responsiveness of these target genes that depends primarily on the affinity of leucine-liganded Lrp for the target .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li>
<li>... 4 Effects of Leucine on Lrp-Mediated Regulation ... 4 At least some effects of leucine in vivo are mediated by Lrp ... 4 Specificity of the leucine effect ... 4 Leucine modulates , but does not abolish , the binding of Lrp to target operons ... 4 The effects of leucine on positively regulated genes probably depend on the effective in vivo concentration of Lrp ... .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li>
<li>For some of the operons known to be affected by leucine in vivo , alanine has been show hierarchy of positively regulated target genes , ranging from those with the lowest affinity for liganded Lrp , which are most sensitive to the effect of leucine , to those with the highest affinity for liganded Lrp , which are least sensitive to the effect of leucine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li>
<li>Conversely , at sufficiently low concentrations of Lrp , all positively regulated genes and operons will be sensitive to leucine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7901196" target="_blank">7901196</a>]<br></li>
<li>Those positively regulated genes with the lowest affinity for Lrp will be the first to be repressed as the leucine concentration rises or the Lrp concentration drops .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li>
<li>The observation that the gltBDF promoter , which is relatively insensitive to leucine , has a higher affinity for Lrp than does the leucine-sensitive ilvIH promoter suggests a model : positively regulated genes and operons that bind Lrp with a high affinity are intrinsically less sensitive to the presence or absence of exogenous leucine than are positively regulated genes and operons that bind Lrp with a lower affinity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7901196" target="_blank">7901196</a>]<br></li>
<li>This model predicts that at sufficiently high concentrations of Lrp , promoters will form complexes with an activator regardless of the presence or absence of leucine and that all positively regulated genes and operons will be insensitive to leucine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7901196" target="_blank">7901196</a>]<br></li>
<li>The first genes and operons of the Lrp regulon to be recognized were regulated in response to the presence or absence of exogenous leucine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7901196" target="_blank">7901196</a>]<br></li>
<li>DISCUSSION The identification of genes and proteins whose expression is regulated by Lrp , but not apparently in response to the presence or absence of leucine in the growth medium , was th most surprising result to emerge from an analysis of the Lrp regulon ( 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7901196" target="_blank">7901196</a>]<br></li>
<li>The polypeptide whose expression is altered in comparisons of strains BE3 and W3110 is a polypeptide ( LivJ ) that is normally negatively regulated by Lrp only in the presence of leucine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1346534" target="_blank">1346534</a>]<br></li>
<li>How is the expression of these proteins regulated by Lrp if not in response to leucine ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7901196" target="_blank">7901196</a>]<br></li>
<li>The leucine-insensitive Lrp protein in strain BE3 retains the ability to regulate all but one of the polypeptides regulated by the wild-type Lrp protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1346534" target="_blank">1346534</a>]<br></li>
<li>The gltBDF operon , encoding glutamate synthase , was among the group of genes and proteins positively regulated by Lrp but insensitive to leucine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7901196" target="_blank">7901196</a>]<br></li>
<li>Major effects on the expression of genes regulated by Lrp may occur through a two-stage process involving immediate effects of leucine following a shift to a rich medium and then slowly decreasing Lrp concentrations in response to continued growth in a rich medium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7901196" target="_blank">7901196</a>]<br></li>
<li>Thus , growth in a rich medium , especially one containing leucine , may result in increased expression of genes that are negatively regulated by Lrp and in decreased expression of genes that are positively regulated by Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li>
<li>Ten of the 30 polypeptides are positively or negatively regulated by Lrp in the absence of added leucine , and the addition of 10 mM leucine to the medium abolishes regulation by Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1346534" target="_blank">1346534</a>]<br></li>
<li>Most ( 19 of 30 ) of the polypeptides listed in Table 3 regulated by Lrp in the are same manner whether or not 10 mM leucine is added the to growth medium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1346534" target="_blank">1346534</a>]<br></li>
<li>This activation by alanine or leucine was not simply relief of repression , as P2 transcription from a constitutive template was increased ® vefold compared with the basal level of transcription found in the absence of Lrp and the co-activator cyclic AMP-CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10216857" target="_blank">10216857</a>]<br></li>
<li>This activation of Tr2 by Lrp plus alanine or leucine was best seen when transcription was performed in the absence of cyclic AMP-CRP ( Fig. 5B ) , a known activator of this transcript ( Zhi et al. , 1998 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10216857" target="_blank">10216857</a>]<br></li>
<li>Lrp stimulates transcription , mutations in a critical Lrp binding site reduce transcription , and leucine reduces transcription ( albeit only under certain conditions in vitro ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1447135" target="_blank">1447135</a>]<br></li>
<li>Our results showed that , both in vivo and in vitro , transcription initiated at very similar sites , the levels of Lrp repression and alanine or leucine induction were similar , and a mutation in an Lrp binding site prevented Lrp repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955407" target="_blank">8955407</a>]<br></li>
<li>In addition , Lrp strongly blocked dad transcription in a puriﬁed in vitro system and this inhibition was reversed by alanine or leucine , the in vivo inducers of the dad operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955407" target="_blank">8955407</a>]<br></li>
<li>In addition to having effects on Lrp binding , alanine and leucine were found to very strongly reverse Lrp repression of dad transcription in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955407" target="_blank">8955407</a>]<br></li>
<li>For example , when site 9 was mutated , the expression of P2 in vitro was essentially unaffected by Lrp , but P2 transcription was increased ® vefold by Lrp plus alanine/leucine ( Fig. 5B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10216857" target="_blank">10216857</a>]<br></li>
<li>( iii ) Lrp blocked dad expression in a puriﬁed in vitro transcription system , and this repression was markedly reduced by alanine or leucine , compounds that induce dad in vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955407" target="_blank">8955407</a>]<br></li>
<li>Activation of P2 by Lrp and alanine ( or leucine ) was also found using a puri ® ed in vitro transcription system , strongly suggesting that Lrp , as has been found for repression ( Mathew et al. , 1996 ) , activated dad transcription directly .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10216857" target="_blank">10216857</a>]<br></li>
<li>Taken together , these results strongly suggest that Lrp and alanine or leucine act directly to repress and induce , respectively , transcription of the dad operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955407" target="_blank">8955407</a>]<br></li>
<li>Thus , for example , Lrp activates transcription from the ilvIH promoter and leucine causes repression by interfering with the action of Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li>
<li>Second , leucine-mediated repression of ilvIH expression can be understood in simple terms : Lrp activates transcription from the ilvIH promoter by binding to specific sites , and leucine reduces such binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li>
<li>Since leucine blocks the transcription of the ilvIH operon by interfering with the activator function of Lrp , the addition of non-saturating leucine concentrations should reduce the cellular concentration of active Lrp activator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8152423" target="_blank">8152423</a>]<br></li>
<li>Lrp must have at least three functions with respect to the regulation of the ilvIH operon : it must bind DNA , activate transcription , and respond to leucine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8432705" target="_blank">8432705</a>]<br></li>
<li>Although it is thought that leucine and Lrp form a complex which is not able to activate ilvIH transcription , such interaction is so far only hypothetical .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8071194" target="_blank">8071194</a>]<br></li>
<li>We conclude that the relatively slight effects of leucine are mediated by Lrp , directly or indirectly , but that other effectors cause major changes in the regulation of transcription of sdaB in LB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1532173" target="_blank">1532173</a>]<br></li>
<li>Furthermore , the transcription of stpA was reduced to the same degree by the presence of leucine in the growth medium , indicating that leucine repression is mediated via Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8890170" target="_blank">8890170</a>]<br></li>
<li>Under these nonstandard conditions , ilvIH transcription in the presence of Lrp was reduced twofold by leucine ( Fig. 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1447135" target="_blank">1447135</a>]<br></li>
<li>Lrp can act as a repressor or as an activator of transcription with its effects being potentiated , repressed or unaffected by the presence of exogenous leucine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9515701" target="_blank">9515701</a>]<br></li>
<li>Depending on the system , Lrp acts as an activator or repressor of transcription , and its effects can be repressed , potentiated or unaffected by the presence of leucine and alanine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9515701" target="_blank">9515701</a>]<br></li>
<li>Lrp can act as a repressor or as an activator of transcription , and its effects can be inhibited , potentiated or unaffected by the presence of leucine and alanine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9515701" target="_blank">9515701</a>]<br></li>
<li>( iv ) Does leucine prevent the effects of Lrp on transcription in vitro ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1447135" target="_blank">1447135</a>]<br></li>
<li>Radioactive Lrp was prepared by in vitro transcription and transla-3 tion in the presence of [ H ] leucine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8636076" target="_blank">8636076</a>]<br></li>
<li>E.coli Lrp utilizes the binding of L-leucine to trigger either activation or repression of some target promoters , although in the majority of cases control by Lrp is leucine-independent , e.g. negative autoregulation of its own lrp gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16528101" target="_blank">16528101</a>]<br></li>
<li>Leucine , by binding directly to Lrp , triggers the activation or repression of the Lrp regulon genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17025177" target="_blank">17025177</a>]<br></li>
<li>It is unlikely that a cryptic repressor site is a factor in the role of alanine and leucine in Lrp activation of dad , as the mutation in the major negative site ( site 9 ) fully prevented repression by Lrp ( Fig. 5A and B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10216857" target="_blank">10216857</a>]<br></li>
<li>Second , Lrp can generate diverse regulatory patterns that include both activation and repression , and differing sensitivities to the coregulators L-leucine and L-alanine [ 73,74 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18405378" target="_blank">18405378</a>]<br></li>
<li>In previous work , we found that alanine or leucine acted as inducers to antagonize Lrp repression of the three major promoters directly .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10216857" target="_blank">10216857</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_604">
In contrast , promoter-distal sites ( upstream sites ) were not required for repression or induction , but they were necessary for the activation of one of the dad promoters by Lrp plus alanine/leucine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10216857" target="_blank">10216857</a>]<br></label>
<input id="act_similarity_604", type="checkbox">
<div class="ACT_similar">
<li>In contrast , sites upstream of the promoters did not appear to be necessary for repression , but were required for activation by Lrp plus alanine or leucine of one of the major dad promoters , P2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10216857" target="_blank">10216857</a>]<br></li></div>
<li>, 1994 ) , under the same conditions , there is no evidence for accumulationof leucine , which is necessaryfor overcomingLrp repression : and ( ii ) in an lrp - strain , induction by anoxia results in significantlyhigher levelsof expressionthan those attained in aerobic conditions , suggesting that a mechanism other than Lrp derepression can take place ( Fig. 3B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8809748" target="_blank">8809748</a>]<br></li>
<li>They showed that repression of pyridine nucleotide transhydrogenase by leucine , alanine , or methionine was abolished in a strain containing a livR mutation which , as discussed above , is an allele of lrp encoding a leucine-insensitive but active Lrp protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li>
<li>This is consistent with the view that changes in growth rate , together with availability of exogenous leucine , determine whether a set of biosynthetic reactions or a set of degradative reactions are stimulated .1,9 However , further analysis of the results points up a fundamental dilemma : free Lrp in cells is in the mM range but Lrp binds to speci ® c DNA sites in vitro with af ® nities in the nM range .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11575919" target="_blank">11575919</a>]<br></li>
<li>Since expression of ilvIH in vivo is affected by leucine , the chemical activity of Lrp in cells may be low , either because most of it is bound nonspecifically to DNA or because Lrp binds more poorly to specific DNA sites in vivo than expected from the results of in vitro studies .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li>
<li>Activation of stpA by Lrp can be largely eliminated by growth of the cells in the presence of leucine , thus placing stpA in the class of genes identiﬁed by Calvo and Matthews ( 3 ) which exhibit leucine-dependent regulatory pattern 1 , along with the ilvIH operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9006049" target="_blank">9006049</a>]<br></li>
<li>Together with the results from RTPCR and β-galactosidase activity analyses shown in Fig. 6 , the ﬁnding that EIIANtr did not interact with puriﬁed Lrp regardless of the presence of leucine ( data not shown ) excluded the possibility for the involvement of EIIANtr in the regulation of Lrp activity and ilvIH derepression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16164569" target="_blank">16164569</a>]<br></li>
<li>In the absence of ArgP , both the affinity of Lrp for the argO control region and its concentration in the cell would be sufficiently high to ensure Lrp-mediated activation of PargO , even in the presence of excess leucine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19906180" target="_blank">19906180</a>]<br></li>
<li>In general the Lrp proteins from P. mirabilis and V. cholerae yielded livK regulatory patterns similar to that of the E. coli control , showing activation in the absence of leucine and repression in its presence ( Fig 4D and 4G ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18405378" target="_blank">18405378</a>]<br></li>
<li>Lrp binds to this fragment in the presence and absence of leucine , and saturating leucine ( 30 mM ) -LCB- decreases the af ® nity of Lrp for this region of gltBDF while increasing the cooperativity of binding ( Ernsting et al. , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9236118" target="_blank">9236118</a>]<br></li>
<li>Apparent K and a values for Lrp binding to the low-affinity complex ( the more retarded band ) , required for activation of transcription of the ilvIH promoter ( B ) , are K = 6.9 nM and a = 1.1 in the absence of leucine and K = 14.1 nM and a = 2.6 in the presence of 30 mM leucine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7901196" target="_blank">7901196</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7901196" target="_blank">7901196</a>]<br></li>
<li>The shown curves were constructed by dividing the percent DNA bound at each Lrp concentration in the absence of leucine by the percent DNA bound at the corresponding Lrp concentration in the presence of 30 mM leucine , using the data for gltBDF and ilvIH shown in Fig. 6 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7901196" target="_blank">7901196</a>]<br></li>
<li>In the presence of 30 mM leucine , the affinity of Lrp for DNA is reduced , but as Lrp concentrations are increased the DNA can nonetheless be driven into the bound complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7901196" target="_blank">7901196</a>]<br></li>
<li>Data from gel mobility shift assays carried out in the absence or presence of 30 mM leucine were quantified by Phosphor-imager scanning and plotted as percent DNA bound at various concentrations of Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7901196" target="_blank">7901196</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7901196" target="_blank">7901196</a>]<br></li>
<li>RNA was made in vitro from plasmid pCV156-derived DNA templates in the absence or presence of Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1447135" target="_blank">1447135</a>]<br></li>
<li>RNA was made in vitro with plasmid pCV7 DNA as a template in the absence of Lrp , in the presence of Lrp , or in the presence of Lrp and 37 mM leucine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1447135" target="_blank">1447135</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1447135" target="_blank">1447135</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1447135" target="_blank">1447135</a>]<br></li>
<li>Thus , conditions of selection that would favor DNA sequences that bind more tightly to Lrp in the presence of leucine yielded sequences that closely resembled those that were selected in the absence of leucine , and none of the selected sequences bound more tightly to Lrp in the presence of leucine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7665463" target="_blank">7665463</a>]<br></li>
<li>At high concentrations of Lrp , at which the DNA is fully bound in the presence and absence of leucine , leucine has no effect , and the value on they axis is 1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7901196" target="_blank">7901196</a>]<br></li>
<li>According to this model , the binding of leucine by Lrp would affect only the affinity of the DNA-Lrp interaction ; Lrp would retain its ability to activate transcription regardless of the presence or absence of bound leucine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7901196" target="_blank">7901196</a>]<br></li>
<li>We have demonstrated that the binding of Lrp to specific DNA sequences occurs in the presence and absence of leucine and that the activation of transcription occurs under both conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7901196" target="_blank">7901196</a>]<br></li>
<li>Lanes 9 through 16 show Lrp binding to the same DNA fragment over the same range of Lrp concentrations as were used for lanes 1 through 8 but in the presence of 20 mM leucine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8002608" target="_blank">8002608</a>]<br></li>
<li>To test the idea that Lrp in the presence of leucine might bind to a different subset of DNA sequences , we carried out a second selection experiment with leucine present during the binding reactions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7665463" target="_blank">7665463</a>]<br></li>
<li>Selection of DNA fragments that bind Lrp in the presence of leucine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7665463" target="_blank">7665463</a>]<br></li>
<li>To investigate further the interactions between Lrp , leucine and the ﬁm switch in vitro , EMSA and MPE-Fe ( II ) footprinting studies using PCR12 were performed in the presence of leucine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9515701" target="_blank">9515701</a>]<br></li>
<li>EMSAs using PCR12 were also performed in the presence of each of the other amino acids , and it was determined that alanine and , to a lesser degree , valine and glycine also decreased the binding afﬁnity of Lrp for ﬁm , but not to the extent of leucine ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9515701" target="_blank">9515701</a>]<br></li>
<li>Lrp undergoes a leucine-induced dissociation of hexadecamer to an octamer The effect of leucine on the hydrodynamic radius of Lrp Dynamic light-scattering experiments were performed on 6HisLrp in the presence of 10 mM leucine ( Table 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11575919" target="_blank">11575919</a>]<br></li>
<li>Effect of leucine on the molecular mass distribution of Lrp To examine the effect of leucine on Lrp molecular mass in a way independent of shape , sedimentation equilibrium experiments were conducted in the presence of 10 mM leucine ( Figure 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11575919" target="_blank">11575919</a>]<br></li>
<li>B. Lanes 1 -- 5 show retardation with 0 , 50 , 100 , ,200 and 400 ng of Lrp , and lanes 6 -- 17 represent retardation by 400 ng of Lrp in the presence of L-leucine , glycine , alanine and isoleucine in the order and concentrations used in Fig. 3A .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11985712" target="_blank">11985712</a>]<br></li>
<li>For cells overproducing Lrp , growth in the presence of leucine gave an unexpected transport phenotype : leucine increased the rate of leucine transport ( Fig. 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1729203" target="_blank">1729203</a>]<br></li>
<li>Furthermore , we wished to test the that possibility when concentrations of Lrp are low , target genes may be more sensitive to the presence of leucine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7901196" target="_blank">7901196</a>]<br></li>
<li>If at nanomolar Lrp concentrations , leucine shifts the monomer-dimer equilibrium toward the monomeric form , a larger fraction of Lrp dimers will be dissociated in the presence of leucine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7901196" target="_blank">7901196</a>]<br></li>
<li>While leucine did not lead to dissociation of Lrp from this fragment , its presence did lead to subtle alterations in the mobilities of the shifted fragments .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8002608" target="_blank">8002608</a>]<br></li>
<li>One hypothesis that can explain how leucine might overcome an effect of Lrp in some systems and be required for an effect in others is that Lrp binds to different sequences in the presence and absence of leucine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7665463" target="_blank">7665463</a>]<br></li>
<li>Strains BEl and BE2 grew more slowly than strain W3110 , even in the presence of leucine , isoleucine , and valine , and so the absence of Lrp must create some limitation other than for these amino acids .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_639">
Lrp binds to the gltBDF and ilvIH promoters in the presence and absence of leucine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7901196" target="_blank">7901196</a>]<br></label>
<input id="act_similarity_639", type="checkbox">
<div class="ACT_similar">
<li>Lrp binds to the gltBDF promoter in the presence and absence of leucine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7901196" target="_blank">7901196</a>]<br></li></div>
<li>Lrp binds to the gltBDF promoter ( A ) with an apparent K value of 2.0 nM and a value for ot of 1.6 in the absence of leucine and an apparent K value of 6.6 nM and a value for a of 2.6 in the presence of 30 mM leucine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7901196" target="_blank">7901196</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7901196" target="_blank">7901196</a>]<br></li>
<li>To elucidate the leucine-speci ® c changes on the interaction of Lrp with gltBDF , footprinting experiments were repeated in the presence and absence of leucine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9236118" target="_blank">9236118</a>]<br></li>
<li>If Lrp is a repressor in the absence of leucine , it remains a repressor in the presence of leucine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li>
<li>Thus , a shorter promoter construct which is not repressed by H-NS is not signiﬁcantly repressed by the absence of Lrp or the presence of leucine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9006049" target="_blank">9006049</a>]<br></li>
<li>Lower panel , leucine sensitivity ( percent promoter occupancy in the presence of leucine/percent promoter occupancy in the absence of leucine ) is plotted versus the ratio [ Lrp ] / Kd as Kd is varied at ® xed Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9236118" target="_blank">9236118</a>]<br></li>
<li>The apparent affinity of Lrp for occupancy of the lower-affinity sites was decreased twofold in the presence of leucine , while the apparent cooperativity was increased from ax = 1.1 to a = 2.6 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7901196" target="_blank">7901196</a>]<br></li>
<li>The addition of alanine or leucine partially overcame Lrp inhibition of Tr1 and Tr3 and increased Tr2 above the levels made in the absence of Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10216857" target="_blank">10216857</a>]<br></li>
<li>The results presented here suggest that this is not the case : sequences selected for their ability to-bind Lrp were the same or very similar irrespective of whether the selection was performed in the presence or absence of leucine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7665463" target="_blank">7665463</a>]<br></li>
<li>The Lrp -- ﬁmS 21 complex is shown in the absence ( lane 2 ) and in the presence of leucine ( lane 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9515701" target="_blank">9515701</a>]<br></li>
<li>When the serA promoter within this DNA fragment was used to drive lacZ , the effect of leucine and of the lrp : : TnJO mutation was unaltered , indicating that the site of action of Lrp , or of a factor that responds to Lrp , is still present in this construct .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1917830" target="_blank">1917830</a>]<br></li>
<li>For instance , oligo-merization of l cI repressor appears to have unfavorable contributions to cooperativity ,20,29 and oligomerization of the GAGA transcription factor allows strong cooperative binding to multiple sites but inhibits binding to single sites .22 Lrp self-associates predominantly to a hexadeca-mer at mM concentrations and dissociates to an octamer upon binding leucine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11575919" target="_blank">11575919</a>]<br></li>
<li>Insensitivity to leucine would be expected when the effective intracellular concentration of Lrp is high relative to the affinity of Lrp binding sites required for transcription of the target gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7901196" target="_blank">7901196</a>]<br></li>
<li>Ernsting et al. ( 4 ) proposed a model for Lrp regulation , suggesting that operons insensitive to leucine would be expected if the effective intracellular concentration of Lrp was high relative to the affinity of Lrp for the binding sites for transcription of the target operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7928983" target="_blank">7928983</a>]<br></li>
<li>However , while leucine decreases the af ® nity of Lrp for gltBDF DNA , it also appears to increase the amount of transcription from Lrp DNA complexes that do form .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9236118" target="_blank">9236118</a>]<br></li>
<li>However , at certain concentrations of Lrp and leucine , the relative amounts of some DNA-Lrp complexes seemed to be enhanced by leucine ( 41a ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7665463" target="_blank">7665463</a>]<br></li>
<li>Modulation of Lrp action in Escherichia coli by leucine : effects on non-speciﬁc binding of Lrp to DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12218014" target="_blank">12218014</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19074390" target="_blank">19074390</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2162465" target="_blank">2162465</a>]<br></li>
<li>The positions of DNA binding , activation and leucine response mutants identified in E.coli Lrp are indicated by the symbols plus , asterisk and hash , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16528101" target="_blank">16528101</a>]<br></li>
<li>The complex regulatory patterns summarized in Table 2 may ultimately be understood in terms of effects of leucine upon the specificity of DNA binding or in terms of interactions of Lrp and leucine with other proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li>
<li>In summary , Lrps from the leucine response mutants form the same complexes with ilvIH DNA as does wild-type Lrp , but at least in some cases , one or more binding constants are altered .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8432705" target="_blank">8432705</a>]<br></li>
<li>The fact that in vitro , leucine reduces binding of wild-type Lrp , but not of Lrp-1 ( 22 ) Lrps from or other leucine response mutants , to ilvIH DNA is consistent with the idea that leucine acts by reducing the extent of binding of Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8432705" target="_blank">8432705</a>]<br></li>
<li>This study suggests that Lrp has separate domains responsible for binding DNA , activating transcription , and responding to leucine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8432705" target="_blank">8432705</a>]<br></li>
<li>In summary , the work reported here suggests that Lrp has domains responsible for binding DNA , activating transcription , and responding to leucine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8432705" target="_blank">8432705</a>]<br></li>
<li>In the presence of leucine , the two shifts were not quite as clearly separated , and it was possible to measure only the apparent afﬁnity of binding of Lrp to either site or both sites ( the total percentage of DNA shifted ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8002608" target="_blank">8002608</a>]<br></li>
<li>In all the systems studied to date , leucine increases the dissociation constant for Lrp binding to DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9515701" target="_blank">9515701</a>]<br></li>
<li>In addition , the free amino acid , L-leucine , acts as an effector ligand of Lrp ; and , at some DNA target sites , L-leucine is required for Lrp binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8900118" target="_blank">8900118</a>]<br></li>
<li>Small molecule effectors of mobility shifts As L-leucine decreases the expression of serA , we thought it might affect the binding of Lrp to serA DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11985712" target="_blank">11985712</a>]<br></li>
<li>C , Comparison of the effect of leucine on the binding of Lrp to gltBDF DNA and on b-galactosidase reporter gene activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9236118" target="_blank">9236118</a>]<br></li>
<li>Furthermore , they indicate that , not only leucine , but also alanine and , to a lesser extent , methionine are able to displace Lrp from binding to osmC DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9663683" target="_blank">9663683</a>]<br></li>
<li>Addifion of leucine to fhe incubation mixture reduced fhe extent of binding of S. typhimurium Lrp to S typhimurium DNA ( Fig. 1 ; compare the second and third lanes ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8483418" target="_blank">8483418</a>]<br></li>
<li>In vitro experiments have demonstrated that leucine decreases the overall binding affinity of Lrp , but also changes the nature of the bound complexes ( Platko and Calvo 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8152423" target="_blank">8152423</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_671">
Here , it is shown that , even though leucine stimulates the ﬁm switch in vivo , it nevertheless causes a slight decrease in Lrp binding to the ﬁm switch in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9515701" target="_blank">9515701</a>]<br></label>
<input id="act_similarity_671", type="checkbox">
<div class="ACT_similar">
<li>In this work , it is shown that , even though leucine stimulates the ﬁm switch in vivo , it nevertheless causes a decrease in Lrp binding to the ﬁm switch in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9515701" target="_blank">9515701</a>]<br></li></div>
<li>We suggest that the observed range in leucine sensitivity shown by various operons results from two competing effects : leucine has a negative effect on Lrp binding af ® nity for speci ® c sites , and has a positive effect on transcriptional activation by whatever Lrp is bound .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9236118" target="_blank">9236118</a>]<br></li>
<li>Therefore , this may lessen the ability of alanine/leucine to reduce Lrp binding to the dad activation sites relative to repressor sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10216857" target="_blank">10216857</a>]<br></li>
<li>Conversely , the effect of leucine on the fractional saturation of promoter activation sites should be large when the in vivo concentration of Lrp is near or below its Kpromoter d , even if leucine has only a modest effect on binding af ® nity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9236118" target="_blank">9236118</a>]<br></li>
<li>Alanine or leucine decreased the af ® nity of Lrp to repressor sites , while having little or no effect on the binding of Lrp to activator sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10216857" target="_blank">10216857</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_677">
According to this model , leucine stimulates recombination by selectively disrupting Lrp binding to this newly characterized region , while having little or no effect on Lrp binding to ﬁm sites 1 and 2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9515701" target="_blank">9515701</a>]<br></label>
<input id="act_similarity_677", type="checkbox">
<div class="ACT_similar">
<li>According to this model , leucine stimulates recombination by selectively disrupting Lrp binding to this newly characterized region while having little or no effect on Lrp binding to ﬁm sites 1 and 2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9515701" target="_blank">9515701</a>]<br></li></div>
<li>In contrast to the on or off binding of the switch-like Lac repressor and catabolite activator protein , leucine has a relatively subtle effect on the binding of Lrp to target genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7901196" target="_blank">7901196</a>]<br></li>
<li>In some cases , L-leucine lessens the effect of Lrp binding ; in others , regulation requires both Lrp and L-leucine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1532173" target="_blank">1532173</a>]<br></li>
<li>In all the cases that have been studied , the effect of leucine is merely to modulate the binding of Lrp to the target gene and not to alter the effect of Lrp on transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li>
<li>This differential effect of alanine and leucine on Lrp binding helps to explain how these modi ® ers in ¯ uence both repression and activa ¬ tion of the dad operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10216857" target="_blank">10216857</a>]<br></li>
<li>However , a recent report indicates that this effect by modi ® ers in ® m is indirect and is probably caused by the ability of leucine and alanine to antagonize Lrp binding to a newly discovered negative site ( Roesch and Blom ® eld , 1998 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10216857" target="_blank">10216857</a>]<br></li>
<li>Quantitative measurements of the effect of leucine on binding of Lrp to the gltBDF and ilvIH promoter regions , as assessed by mobility shift assays , indicated that relatively high levels of leucine were required to alter binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li>
<li>Although the effect of leucine on binding of Lrp to the gltBDF promoter region appeared to be specific ( similar concentrations of glutamate had no effect ) , one wonders whether such high levels of leucine occur under physiological conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li>
<li>The very minor effect of leucine on binding of Lrp to the ompC-micF intergenic region is also in agreement with the results of earlier studies using two-dimensional gel electro-phoretic analysis ( 11 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8002608" target="_blank">8002608</a>]<br></li>
<li>The substitution of residues Y82 , Y100 and Q128 in AsnC by a phenylalanine and two leucines , respect-ively , in Lrp , changes the size and nature of the pocket such that it would be ideally suited to-binding the aliphatic side chain of leucine as opposed to the carboxyamide group of asparagine ( Figure 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16528101" target="_blank">16528101</a>]<br></li>
<li>The in vitro stimulatory effects of leucine that we and others have observed ( 10 ) are most readily explained in terms of effects of leucine upon cooperative binding of Lrp to multiple sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7665463" target="_blank">7665463</a>]<br></li>
<li>The effects of leucine on a particular system are , therefore , expected to depend on the Kd for Lrp binding and on co-operative interactions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9515701" target="_blank">9515701</a>]<br></li>
<li>Previous studies have shown that Lrp has a high binding affinity for site 2 and that the presence of leucine does not completely abolish binding of Lrp to upstream sites in vitro ( 10 , 28 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8071194" target="_blank">8071194</a>]<br></li>
<li>At these concentrations in vitro , , all of the binding sites upstream of ilvIH would contain bound Lrp , even in the presence of leucine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li>
<li>Although in the presence of leucine some in vivo interaction of Lrp with the ilvIH regulatory region may still occur , binding to sites 1 , 3 , 4 , and 5 is impaired .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8071194" target="_blank">8071194</a>]<br></li>
<li>They also reported mobility shift assay results indicating that Lrp binding to ilvIH promoter-distal sites is maintained in the presence of leucine ( 10 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8071194" target="_blank">8071194</a>]<br></li>
<li>Hill binding curves fitted to the plotted data showed that , in the presence of leucine , the affinity of Lrp for gltBDF promoter DNA was reduced 3.3-fold and that the apparent cooperativity of binding was increased from a ( = 1.6 to a - = 2.6 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7901196" target="_blank">7901196</a>]<br></li>
<li>Lrp binds specifically to DNA fragments containing the gltBDF promoter , and this binding occurs even in the presence of 30 mM leucine in the binding reaction mixture , although with a reduced affinity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7901196" target="_blank">7901196</a>]<br></li>
<li>The presence of leucine in the binding buffers has been shown to modulate Lrp binding to a number of its target sites ( Calvo and Matthews , 1994 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9663683" target="_blank">9663683</a>]<br></li>
<li>The labeled DNA , present at 240 pM in the binding reaction mixture , was electrophoresed in the presence of Lrp with or without were Phos-leucine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7901196" target="_blank">7901196</a>]<br></li>
<li>The increased cooperativity in the presence of leucine suggests that much of the cooperativity may be due to interactions between Lrp monomers associated with binding events .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7901196" target="_blank">7901196</a>]<br></li>
<li>A similar pattern was obtained in the presence of leucine , although Lrp binding occurred with a reduced affinity in the presence of the effector-molecule ( Fig. 4A ) , as also observed in the EMSAs ( Fig. 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19906180" target="_blank">19906180</a>]<br></li>
<li>However the subsequent comparison of the Ka of Lrp and transcriptional activity , measured in the presence and absence of leucine , suggests that leucine effects are due to cumulat-ive effects both on binding and activation ( see next section ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9236118" target="_blank">9236118</a>]<br></li>
<li>When Lrp-mediated activation was impaired by the presence of exogenous leucine , only one promoter-distal site ( site 2 ) was partially protected by Lrp binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8071194" target="_blank">8071194</a>]<br></li>
<li>Figure 5 shows Lrp binding to the gltBDF promoter in the presence and absence of 30 mM leucine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7901196" target="_blank">7901196</a>]<br></li>
<li>Thus , the Kd measured for Lrp binding in the presence of 20 mM leucine was only 1.2-fold higher than the Kd measured in the absence of leucine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8002608" target="_blank">8002608</a>]<br></li>
<li>This is the reduction in the Lrp concentration required to reduce binding from 84 % in the absence of leucine to 5.3 % in the presence of leucine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7901196" target="_blank">7901196</a>]<br></li>
<li>Lrp binding to an isolated site in the presence and absence of leucine will be studied to gain further insight into the nature of the observed cooperativity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7901196" target="_blank">7901196</a>]<br></li>
<li>Equally important , leucine did not stimulate binding of Lrp to any of the sequences that were selected in the presence of leucine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7665463" target="_blank">7665463</a>]<br></li>
<li>Selection for Lrp binding in the presence of leucine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7665463" target="_blank">7665463</a>]<br></li>
<li>Lrp binding to PCR12 in the presence of leucine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9515701" target="_blank">9515701</a>]<br></li>
<li>These in combination data , with the methylation pattern analyses , suggest that binding of Lrp to the GATC site of clones 4-1A / 4-3E is inhibited by leucine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8195106" target="_blank">8195106</a>]<br></li>
<li>Notably , fully to the methylated upon addition of leucine growth medium , suggesting that binding of Lrp to this site was inhibited by leucine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li>
<li>In this regard , it is interesting to cite a comparative study of Lrp binding to the gltBDF and ilvIH operons which correlates the binding affinity constants ( K ) of Lrp with the sensitivity of the operons to repression by leucine ( 10 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8071194" target="_blank">8071194</a>]<br></li>
<li>In the pre-sence of 10 mM leucine , the Kd for Lrp binding was increased to 3.2 6 0.4 nM ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9515701" target="_blank">9515701</a>]<br></li>
<li>In a control experiment with RNA polymerase instead of Lrp , no difference in the concentration of leucine in the two chambers was detected , showing the specificity of leucine binding to Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8071194" target="_blank">8071194</a>]<br></li>
<li>Similar results were seen in experiments with Lrp binding to the ilvIH promoter : at a fixed concentration of Lrp , increasing the leucine concentration beyond 20 mM does not lead to further decreases in Lrp binding ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7901196" target="_blank">7901196</a>]<br></li>
<li>We estimate that at an effective Lrp concentration of 5.5 nM , binding to the gltBDF promoter would be reduced 2.2-fold in the presence of leucine , while binding to the ilvIH promoter would be reduced 5.5-fold .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7901196" target="_blank">7901196</a>]<br></li>
<li>To compare Lrp binding to these promoters , we needed to choose a concentration of leucine that would be saturating for both ilvIH and gltBDF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7901196" target="_blank">7901196</a>]<br></li>
<li>Effect of leucine on Lrp binding to the gltBDF and ilvIH promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7901196" target="_blank">7901196</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7901196" target="_blank">7901196</a>]<br></li>
<li>At this concentration of Lrp , leucine decreases the binding of Lrp to the ilvIH promoter by a factor of approximately 5.5 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7901196" target="_blank">7901196</a>]<br></li>
<li>At this concentration of Lrp , leucine has a 5.5-fold effect on the binding of Lrp to the ilvIH promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7901196" target="_blank">7901196</a>]<br></li>
<li>Thermodynamics of hexadecamer dissociation and leucine binding of leucine to Lrp octamer ( G2 ) and to Lrp hexadecamer ( G3 ) was thus estimated to be in the range of ÿ5 .12 to ÿ5 .25 kcal/mol and ÿ6 .92 to ÿ7 .18 kcal/mol , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11575919" target="_blank">11575919</a>]<br></li>
<li>Increasing concentrations of leucine , at a fixed Lrp concentration , decrease binding , but this effect is saturable .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7901196" target="_blank">7901196</a>]<br></li>
<li>At concentrations of leucine higher than 20 mM , no further decrease in Lrp binding is seen .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7901196" target="_blank">7901196</a>]<br></li>
<li>Saturating levels of leucine do not abolish binding of Lrp upstream of glBDF but merely increase its apparent dissociation constant from 2.0 to 6.9 nM .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7901196" target="_blank">7901196</a>]<br></li>
<li>Thus most Lrp sites will be occupied even if leucine signi ® cantly decreases binding af ® nity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9236118" target="_blank">9236118</a>]<br></li>
<li>' Zn uitro , leucine reduces the extent of binding of Lrp to these binding sites ( 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2040596" target="_blank">2040596</a>]<br></li>
<li>Thus , although L-leucine significantly destabilizes binding of Lrp in the ilvG-MEDA leader region , it does not abolish it .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8900118" target="_blank">8900118</a>]<br></li>
<li>Linkage of leucine binding and Lrp dissociation from hexadecamer to octamer .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11575919" target="_blank">11575919</a>]<br></li>
<li>Therefore , it is unlikely that leucine changes the speciﬁcity of Lrp binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7665463" target="_blank">7665463</a>]<br></li>
<li>Given these results , it seems unlikely that the binding speciﬁcity of Lrp changes upon binding leucine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7665463" target="_blank">7665463</a>]<br></li>
<li>Quantitative analysis of Lrp binding to the gltBDF and ilvIH operons in the presence and absence of leucine suggested the following model : leucine sensitivity of a positively regulated operon is a function both of the intracellular concentration of Lrp and of the af ® nity of Lrp for a given promo-ter ; effects on expression are directly proportional to the occupancy of speci ® c promoter DNA binding sites by Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9236118" target="_blank">9236118</a>]<br></li>
<li>From the in vitro studies of Lrp binding to gltBDF shown in Fig. 5 , a maximal effect of leucine on expression should be seen at an Lrp dimer concentration of about 3 nM .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li>
<li>The relative insensitivity of glutamate synthase expression to the presence or absence of leucine in the growth medium could be explained by leucine-insensitive binding of Lrp to the gltBDF promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7901196" target="_blank">7901196</a>]<br></li>
<li>The saturability of this system allowed us to investigate Lrp binding to the ilvIH and gltBDF promoters in the absence of leucine and under conditions in which the leucine effect is maximal .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7901196" target="_blank">7901196</a>]<br></li>
<li>The in vitro effect of leucine on Lrp binding to the gltBDF and ilvIH promoters explains the leucine sensitivity of these operons in vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7901196" target="_blank">7901196</a>]<br></li>
<li>On the basis of a quantitative analysis of the in vitro binding affinities of Lrp for these two promoters in the presence and absence of leucine , we propose a model to explain the differing effects of leucine on the in vivo expression of ilvIH and gltBDF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7901196" target="_blank">7901196</a>]<br></li>
<li>The binding of Lrp to the gltBDF and ilvIH promoters in the presence and absence of leucine is in accord with this model .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7901196" target="_blank">7901196</a>]<br></li>
<li>Thus , leucine seems to decrease the overall affinity with which Lrp binds , but it seems to favor the formation of particular complexes over others .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8432705" target="_blank">8432705</a>]<br></li>
<li>One possible explanation is that Lrp binds to two different consensus sequences and that high concentrations of leucine decrease binding to sequences of one type and increase binding to the other .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li>
<li>Thus , the results presented here , together with the studies mentioned above , lead to the hypothesis that interaction of leucine with Lrp may cause a conformational change of the protein and may in turn reduce its affinity for the promoter region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8071194" target="_blank">8071194</a>]<br></li>
<li>Lrp binds to the argO control region in a leucine-sensitive manner , forming at least three distinct complexes EMSAs were performed over a range of Lrp protein concentrations to assess the binding of puriﬁed Lrp to the argO control region ( Fig. 3A -- C ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19906180" target="_blank">19906180</a>]<br></li>
<li>However , as-dependenttranscription of aidB is inhibited both in witro and in wivo by binding of the transcriptional-regulator Lrp ( leucine responsive protein ) to the aid6 promoter region ( P&B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8809748" target="_blank">8809748</a>]<br></li>
<li>Lrp-mediated repression of aidB is abolished by high concentrations of leucine , which inhibits Lrp binding to the promoter region ; binding of the transcriptional-activator '' `` Ada to Pa , ds also results in dissociation of Lrp from the promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8809748" target="_blank">8809748</a>]<br></li>
<li>theseresults indicate that Lrp stimulatestranscription of ilvZH by binding to thepromoter/regulatory region and that leucine represses expression of the ilvZH operon by preventing the binding of Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2040596" target="_blank">2040596</a>]<br></li>
<li>Lrp from this strain binds in vitro to the ilvZH promoter/regulatory region , but in a leucine-independent manner ( 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2040596" target="_blank">2040596</a>]<br></li>
<li>Lrp isolated from such strains ( hereafter referred to as Lrp-1 ) binds to theiluZH promoter/regulatory region in uitro but the pattern of binding and the effect of leucine upon binding are different than for wild-type Lrp ( 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2040596" target="_blank">2040596</a>]<br></li>
<li>Therefore , the absence of trans activation by Lrp in rich medium may make the stpA promoter much more sensitive to the changes in DNA topology induced by growth-phase , osmotic shock , and temperature shift .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9006049" target="_blank">9006049</a>]<br></li>
<li>Continued growth in this medium would result in slowly falling concentrations of Lrp , as preexisting Lrp molecules are diluted through degradation , doubling , and growth , leading to progressively less binding and activation by Lrp at the gltBDF promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7901196" target="_blank">7901196</a>]<br></li>
<li>It appears that in conditions of maximal ArgPmediated stimulation ( high arginine , low lysine ) the lack of Lrp-mediated activation of PargO is largely compensated for and even surpassed by the increased efficiency of ArgP , which is no longer hindered by Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19906180" target="_blank">19906180</a>]<br></li>
<li>Combined , the effector modulated activation of argO transcription by ArgP and Lrp must ensure an adapted and ﬁne-tuned synthesis of the transporter in response to environmental conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19906180" target="_blank">19906180</a>]<br></li>
<li>In spite of this vast repertoire of regulatory strategies , a competitive activation mechanism as the one described here , with Lrp and ArgP acting as competitive activators on a single promoter , each regulator being more potent in the absence of the other , appears to be rare and has to the best of our knowledge not been reported previously .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19906180" target="_blank">19906180</a>]<br></li>
<li>Activation by Lrp interferes with the previously dem-onstrated activation of the argO promoter by ArgP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19906180" target="_blank">19906180</a>]<br></li>
<li>Therefore , we may conclude that activation by ArgP is stronger in the absence of Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19906180" target="_blank">19906180</a>]<br></li>
<li>Furthermore , it is shown that activation by Lrp and ArgP are intertwined , each activator being more potent in the absence of the other .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19906180" target="_blank">19906180</a>]<br></li>
<li>Each activator appears thus to operate more efficiently in the absence of the other , and ArgP ( maximum activation coefficient of 17.8 ) is a more potent activator than is Lrp ( maximum activation coefficient of 6.7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19906180" target="_blank">19906180</a>]<br></li>
<li>Paul , L. , Blumenthal , R.M. , and Matthews , R.G. ( 2001 ) Activation from a distance : roles of Lrp and integration host factor in transcriptional activation of Gltbdf .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16238633" target="_blank">16238633</a>]<br></li>
<li>Competitive activation of the Escherichia coli argO gene coding for an arginine exporter by the transcriptional-regulators Lrp and ArgP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22904281" target="_blank">22904281</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21890697" target="_blank">21890697</a>]<br></li>
<li>Lrp plays a major role in transcriptional repression of micF when cells are in nutrient poor media .29 This repression enables the cell to maximize the number of OmpF porins in order to compete for scarce nutrients .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11601842" target="_blank">11601842</a>]<br></li>
<li>In rich media , Lrp and StpA are in low concentrations .29,93 This results in an abundance of micF RNA and the concomitant decrease in OmpF porins ( Figure 5 ( a ) ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11601842" target="_blank">11601842</a>]<br></li>
<li>[ 20 ] A.D. Hernday , B.A. Braaten , G. Broitman-Maduro , P. Engelberts , D.A. Low , Regulation of the pap epigenetic switch by CpxAR : Phosphorylated CpxR inhibits transition to the phase ON state by competition with Lrp , Mol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16487683" target="_blank">16487683</a>]<br></li>
<li>Regulation of the pap epigenetic switch by CpxAR : phosphorylated CpxR inhibits transition to the phase ON state by competition with Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16077119" target="_blank">16077119</a>]<br></li>
<li>A similar experiment with the lrp rpoS double mutant ( Fig. 3 ) clearly demonstrated that in contrast to stationary-phase induction , osmotic regulation occurs in the absence of both Lrp and u ' and therefore yet another mechanism must be involved .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8253679" target="_blank">8253679</a>]<br></li>
<li>Here , we first demonstrate a role for Lrp in the regulation of a gene that is specifically induced in stationary phase , i.e. , under conditions of total starvation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8253679" target="_blank">8253679</a>]<br></li>
<li>We demonstrate that H-NS can decrease the activity of Lrp in stationary phase and under conditions of high osmolarity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8152423" target="_blank">8152423</a>]<br></li>
<li>Moreover , repression of hdeAB by MarA was greater in the absence of GadX or Lrp in exponential phase at pH 7.0 and in the absence of GadW or RpoS in stationary phase at pH 5.5 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18083817" target="_blank">18083817</a>]<br></li>
<li>In turn , MarA enhanced repression of hdeAB by H-NS and hindered activation by GadE in stationary phase and also reduced the activity of GadX , GadW , RpoS , and Lrp on hdeAB under some conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18083817" target="_blank">18083817</a>]<br></li>
<li>Polypeptides that are positively regulated by Lrp show decreased rates of synthesis in strains lacking a functional Lrp protein , while polypeptides that are negatively regulated by Lrp show increased rates of synthesis in the absence of a functional Lrp protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1346534" target="_blank">1346534</a>]<br></li>
<li>Previously , we reported that the serC ( pdxF ) gene is positively regulated by the Lrp protein and negatively regulated by the cyclic AMP receptor protein ( 43 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11844765" target="_blank">11844765</a>]<br></li>
<li>The formation of these transcripts was differentially regulated by cyclic AMP-cyclic AMP receptor protein complex , and each was strongly repressed by Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955407" target="_blank">8955407</a>]<br></li>
<li>Leucine-regulated self-association of Lrp It is dif ® cult to determine the molecular mass of native Lrp by size-exclusion chromatography , in part because of interactions of the protein with the column matrix and in part because of Lrp selfassociation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11575919" target="_blank">11575919</a>]<br></li>
<li>These results indicated that the lrp gene is not only feedback regulated by Lrp but is also controlled by H-NS protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7603430" target="_blank">7603430</a>]<br></li>
<li>However , the expression of lrp is not regulated solely by Lrp but is also under the control of H-NS protein , as revealed in this study .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7603430" target="_blank">7603430</a>]<br></li>
<li>The lysU gene encodes one of the two lysyl-tRNA synthetases ( the other being lysS ) , reported previously to be regulated by Lrp under aerobic conditions ( 48 ) and induced under anaerobic conditions but not by FIG. 9 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12754220" target="_blank">12754220</a>]<br></li>
<li>The evidence that glutamate synthase is directly regulated by Lrp ( 44 ) provides insight into the strategy for regulation of nitrogen metabolism in response to Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li>
<li>The genes currently known to be regulated by Lrp are listed in Table 6 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1532173" target="_blank">1532173</a>]<br></li>
<li>The list of regulated proteins shown in Table 3 does not include all proteins whose expression is known to be regulated by Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1346534" target="_blank">1346534</a>]<br></li>
<li>Thus , although the activity of serine hydroxymethyltransferase is said to be negatively regulated by Lrp ( 31 ) , sufficient activity remains to supply the cell with glycine and one-carbon units derived from serine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li>
<li>Many of the proteins that are regulated by Lrp , either directly or indirectly , are members of other regulons , such as those for heat-shock , osmolarity , nitrogen assimilation , and pilus formation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7603430" target="_blank">7603430</a>]<br></li>
<li>Identification of the Lrp-regulated proteins was undertaken in order to understand which areas of metabolism are controlled by Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1346534" target="_blank">1346534</a>]<br></li>
<li>Direct regulation of LysU by Lrp is consistent with the observation that other heat-shock proteins are not regulated by Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1346534" target="_blank">1346534</a>]<br></li>
<li>We tested whether Lrp regulated avtA or yfbQ .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20729367" target="_blank">20729367</a>]<br></li>
<li>Each of these operons is positively regulated by Lrp , although as described below , the effects on glnALG are indirect .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li>
<li>Transport Six operons involved in nutrient transport are known to be regulated by Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li>
<li>The gcv operon is negatively regulated by PurR ( 169 ) and positively regulated by Lrp ( 84 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li>
<li>Whereas the sdaA and tdh operons are negatively regulated by Lrp , the gcv operon is positively regulated by Lrp , like the amino acid biosynthetic operons described above .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li>
<li>In agreement with these ﬁndings , we now present evidence that transcriptional ( operon ) fusions of lacZ1 to ompC and micF are negatively regulated by Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8002608" target="_blank">8002608</a>]<br></li>
<li>ompF-lacZ1 transcriptional fusions are not positively regulated by Lrp , while ompF9-lacZ1 translational fusions are .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8002608" target="_blank">8002608</a>]<br></li>
<li>These include two genes which were not previously known to be regulated by Lrp , malF and ORF 489 , an as yet unidentiﬁed gene which has also been identiﬁed as both Lrp and methylase regulated by Low and his group ( 11 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7751276" target="_blank">7751276</a>]<br></li>
<li>At the same time , it is noteworthy that two of the genes that are most prominently regulated by ArgP , argO and lysP , are both also regulated by the leucine-responsive general transcriptional-regulator Lrp ( 37 , 41 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21890697" target="_blank">21890697</a>]<br></li>
<li>Finally , it is noteworthy that many of the genes regulated by ArgP are controlled also by other mechanisms , such as by LysR for lysA , osmolarity for gdhA , the riboswitch for lysC , and Lrp for argO and lysP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21890697" target="_blank">21890697</a>]<br></li>
<li>The Venn diagram shows subsets of E. coli genes that were differentially regulated in response to Lrp orthologs from the indicated species ( but not to the vector control ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18405378" target="_blank">18405378</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18405378" target="_blank">18405378</a>]<br></li>
<li>Fourth , are the orthologs of what , in E. coli K-12 , are Lrp-controlled genes regulated by Lrp in the same manner in the two other spe-cies ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18405378" target="_blank">18405378</a>]<br></li>
<li>Roughly half of the genes regulated by E. coli Lrp under these conditions , directly or indirectly , were regulated by either of the other two orthologs ( 51 % by P. mirabilis Lrp , and 42 % by V. cholerae Lrp ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18405378" target="_blank">18405378</a>]<br></li>
<li>To illustrate the overlap between genes regulated by ArcA , FNR , and Lrp , a Venn diagram was constructed ( Fig. 7B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15699038" target="_blank">15699038</a>]<br></li>
<li>Only 16 % of the genes differentially regulated by one or more of the three Lrp orthologs were regulated in common by all three of them ( Fig. 5 ) , but this group included recognized Lrp-controlled genes such as gltBD , ilvG1 , lysU , and osmC [ 67,74,99-101 ] ( these results are available in spreadsheet form [ see Additional file 2 ] ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18405378" target="_blank">18405378</a>]<br></li>
<li>When levels of Lrp and hence of glutamate synthase are high , genes of the nitrogen regulon are regulated by NRI in response to the availability of ammonia , which is thought to be sensed by the ratio of glutamine to a-ketoglutarate in the cell .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li>
<li>When levels of Lrp are low ( for instance during growth in rich media ) , levels of glutamate synthase are also low and ammonia limitation will no longer be coupled to induction of nitrogen-regulated operons .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li>
<li>They demonstrated a similar correlation with genes that are negatively regulated by Lrp and the physiology needed for conditions of higher temperature , high osmolarity , and nitrogen sufficiency .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8152423" target="_blank">8152423</a>]<br></li>
<li>A similar correlation is apparent between genes that are negatively regulated by Lrp and that needed during in genes are growth an environment that is warm and of high osmolarity and in which nitrogen is provided by peptides and amino acids , i.e. , conditions encountered in the intestinal tract of the host .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1346534" target="_blank">1346534</a>]<br></li>
<li>Genes that are positively regulated by Lrp , either directly or indirectly , encode proteins involved in the assimilation of nitrogen under nitrogenlimiting conditions ( GltD and GlnA ) , proteins involved in the biosynthesis of amino acids ( IlvIH and SerA ) , and proteins involved in the adaptation of the cell to media of low osmolarity and low temperature ( OmpF ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1346534" target="_blank">1346534</a>]<br></li>
<li>A number of genes that are negatively regulated by Lrp are involved in the transport and catabolism of amino acids and peptides ( the proteins of the oligopeptide permease operon , Tdh , Kbl , and SdaA ) and in the adaptation of the cell to media of high osmolarity and high temperature ( OmpC and LysU ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1346534" target="_blank">1346534</a>]<br></li>
<li>Several genes required for amino acid and peptide transport and catabolism are negatively regulated by Lrp , and other genes required for amino acid biosynthesis and ammonia assimilation in a nitrogen-poor environment are positively regulated by Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1346534" target="_blank">1346534</a>]<br></li>
<li>Genes and proteins that are involved in catabolism of amino acids and oligopeptide transport are negatively regulated by Lrp , while genes and proteins involved in amino acid biosynthesis and the assimilation of ammonia are positively regulated by Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8002608" target="_blank">8002608</a>]<br></li>
<li>Both authors have proposed that Lrp is involved in the adaptation of E. coIi to major shifts in environmental conditions ; Lrp decreases synthesis from transport and degra-dative genes needed primarily in rich media and activates synthesis from biosynthetic genes needed in poor media , in response to `` feast and famine '' in the environment .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7603430" target="_blank">7603430</a>]<br></li>
<li>This was expressed strongly only in minimal medium with glycerol and maltose ; in these conditions , the Lrp deﬁciency had no effect .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7751276" target="_blank">7751276</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_805">
This set includes genes that are only controlled by Lrp under growth conditions that differ from those used by us [ 66,69 ] , so the cluster of genes showing little or no effect of Lrp is not surprising .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18405378" target="_blank">18405378</a>]<br></label>
<input id="act_similarity_805", type="checkbox">
<div class="ACT_similar">
<li>This set includes genes that are controlled by Lrp , but not under the growth conditions used by us , so the cluster of genes showing little or no effect of Lrp is not surprising [ 66 , 69 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18405378" target="_blank">18405378</a>]<br></li></div>
<li>It is tempting to speculate that these results reveal common functions between FNR and Lrp under aerobic or anaerobic-growth conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12754220" target="_blank">12754220</a>]<br></li>
<li>In fact , of the 31 remaining genes of this expression group regulated by FNR under anaerobic-growth conditions , 10 are regulated by Lrp under aerobic conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12754220" target="_blank">12754220</a>]<br></li>
<li>In agreement with the suggestion that there is a significant interaction between the Lrp and FNR regulatory networks , 23 of the 58 genes of this group are regulated by Lrp under aerobic-growth conditions ( 11 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12754220" target="_blank">12754220</a>]<br></li>
<li>However , it is striking that the genes of these patterns fall into the same functional classes as the functional classes of genes regulated by Lrp under aerobic conditions ( 11 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12754220" target="_blank">12754220</a>]<br></li>
<li>This gene is also regulated by Lrp under aerobic-growth conditions ( 53 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12754220" target="_blank">12754220</a>]<br></li>
<li>Three of the genes of this cluster ( gcvT , lysU , and dadX ) are known to be regulated by Lrp under aerobic-growth conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12754220" target="_blank">12754220</a>]<br></li>
<li>Both artI and potF are regulated by Lrp under aerobic-growth conditions ( 11 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12754220" target="_blank">12754220</a>]<br></li>
<li>These pathways are known to be regulated by Lrp under aerobic-growth conditions ( 11 , 54 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12754220" target="_blank">12754220</a>]<br></li>
<li>During a growth shift , for example , from outside-the-host to inside-the-host conditions , the roles of OmpR and Lrp may be temporally different as well as functionally different .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8002608" target="_blank">8002608</a>]<br></li>
<li>Low concentrations of Lrp resulting from continued growth in LB lead to might levels of Lrp insufficient to regulate some genes and operons .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7901196" target="_blank">7901196</a>]<br></li>
<li>Low growth rates associated with lean nutritional conditions lead to elevated ppGpp levels , and , as a result , relatively high Lrp levels .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955316" target="_blank">8955316</a>]<br></li>
<li>Even under conditions of rapid growth leading to minimal free Lrp levels ,9 those levels are still in the mM range .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11575919" target="_blank">11575919</a>]<br></li>
<li>Distance from Center ( cm ) 3 G 2.5 D - lrp + Pmi lrp + A - Wild 2 1.5 1 B - lrp E - lrp + Eco lrp + 0.5 0 Pmi Eco Vch lrp + lrp + lrp + WT C - lrp + vector F - lrp + Vch lrp + EFfifgeuctreof3heterologous Lrp proteins on the swarming phenotype of P. mirabilis Effect of heterologous Lrp proteins on the swarming phenotype of P. mirabilis.Proteus mirabilis wild-type ( A ) or lrp null strains ( B ) were grown in LB medium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18405378" target="_blank">18405378</a>]<br></li>
<li>Thus the significant differences in swarming ( e.g. , comparing the outer growth rings of strains complemented by lrp from Vibrio and Proteus in Fig. 3G ) indicate functional differences in the Lrp proteins , some of which may be amplified by indirect effects .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18405378" target="_blank">18405378</a>]<br></li>
<li>Because of the fourfold repression of lysU by increased Irp gene dosage in RM460 ( lrp ÷ ) cells , Lrp repression may not be complete under normal conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7603430" target="_blank">7603430</a>]<br></li>
<li>mRNA of lysU , the known target of Lrp , was virtually undetectable in the WT strain under the culture conditions employed , but a high level of lysU mRNA was detected in the lrp mutant strain , indicating strong repression of the lysU gene by Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2162465" target="_blank">2162465</a>]<br></li>
<li>To see whether this increase is truly due to the availability of Lrp , the Vlrp strain , which is the lrp-deleted Val strain , was transformed with pKBRilvBNCED and cultured .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17463081" target="_blank">17463081</a>]<br></li>
<li>gltBD is positively regulated by Lrp , and the levels of GltBD are reduced about 40-fold in lrp strains ( 43 , 44 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li>
<li>Since nothing is known about the stability of lrp mRNA or of Lrp in cells grown under different conditions or whether Lrp synthesis is regulated at the level of translation , the assumption that reporter gene expression reﬂects Lrp levels needed to be tested directly .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955316" target="_blank">8955316</a>]<br></li>
<li>To examine whether dad is regulated by Lrp , we transduced a deletion of lrp into strain EC1051 and measured b-galactosidase levels in the mutan and parent grown under various conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955407" target="_blank">8955407</a>]<br></li>
<li>Lrp would repress genes involved in the degradation of exogenous compounds and activate those involved in the biosynthesis of compounds starting from simple intermediates such as pyruvate Because Lrp is apparently made at lower concentration in LB , it is likely to affect transcription of only genes with a higher affinity for it .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1532173" target="_blank">1532173</a>]<br></li>
<li>For example , Lrp regulates the serA and sdaA genes involved in the production of pyruvate and the tdh and kbl genes , which affect the in vivo levels of L-threonine and a-ketobutyrate ( 5 , 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8900118" target="_blank">8900118</a>]<br></li>
<li>Full induction of gcv in response to glycine also requires Lrp ( 12 ) , a global regulator necessary for either activation or repression of many genes involved in amino acid metabolism ( 1 , * Corresponding author .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7665470" target="_blank">7665470</a>]<br></li>
<li>As Lrp is a global transcriptional-regulator that controls the expression of numerous genes in response to the availability of amino acids and nitrogen bases ( 6 , 9 , 33 ) , the role of this protein in lysP expression was analyzed in more detail .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21441513" target="_blank">21441513</a>]<br></li>
<li>In a nitrogen-limited environment , ppGpp via Lrp is required for glutamate synthesis , induction of the Ntr response , and ammonia assimilation , and therefore the synthesis of all the amino acids .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12730324" target="_blank">12730324</a>]<br></li>
<li>Lrp activates gltBD , which means that Lrp is required for the Ntr response ( 29 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12730324" target="_blank">12730324</a>]<br></li>
<li>For example , carbon limitation through growth rate control increases ppGpp , which via Lrp is required for induction of the Ntr response .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12730324" target="_blank">12730324</a>]<br></li>
<li>In addition to these amino acid-related transcription factors , Lrp was found to regulate a total of 15 transcription factors ( Fig. 6 , grey symbols ) , which are involved in the stress-response and life-style selection of E. coli , such as GadW for acid response , YedW for copper and peroxide response , EnvR for response to drugs , QseB for quorum sensing , BluR for bioﬁlm formation , and MatA and SlyA for planktonic growth .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2162465" target="_blank">2162465</a>]<br></li>
<li>Leucine , the most abundant amino acid in E. coli and the major effector-molecule for Lrp , is used as a reference for the richness of the medium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19906180" target="_blank">19906180</a>]<br></li>
<li>Lrp functions to regulate metabolic pathways in E. coli in response to the availability of amino acids and nitrogen bases in the external environment .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li>
<li>The results are consistent with a previously proposed model for gcv regulation where Lrp binds to the gcv control region , bending the DNA to position other regulatory proteins into proximity to RNAP so possible pro-tein-protein interactions can occur to either activate or repress gcv operon expression ( 25 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9733689" target="_blank">9733689</a>]<br></li>
<li>Whether GcvA and Lrp bind directly to the gcv control region and where these proteins bind are unknown .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7928983" target="_blank">7928983</a>]<br></li>
<li>e-mail : george-stauffer@uiowa.edu Department of Microbiology , University of Iowa , Iowa City , IA 52242 , USA Keywords : glycine cleavage , gcvT : : ZacZ , GcvA protein , Lrp protein , regulation INTRODUCTION Both GcvA and Lrp are requiredfor normal regulation of the gcv operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9611815" target="_blank">9611815</a>]<br></li>
<li>It will be interesting to determine whether positive regulation of the gcv operon by Lrp requires a functional GcvA protein and whether gcvA is regulated by Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li>
<li>e-mail : george-stauffer@uiowa.edu Department of UniversityOf GcvA and Lrp are both necessary for activation of the gcv operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9802028" target="_blank">9802028</a>]<br></li>
<li>Since repression by GcvA , GcvR , and Lrp is poorly understood , further elucidation of the roles of these proteins in the regulatory mechanism is necessary for further investigation into CRP 's role in controlling the gcv operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10074087" target="_blank">10074087</a>]<br></li>
<li>This protein , UspA ( universai stress protein A ) , is thus a generai , non-specific responder to growth arrest ( this paper ; Nystrom and Neidhardt , 1992 ) , and its induction is independent of RpoS as weli as of aii other giobal reguiators so far tested , inciuding PhoB , ReiA , SpoT , AppY , OmpR , H-ns , Lrp , and RpoH ( Nystrom and Neidhardt , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8152377" target="_blank">8152377</a>]<br></li>
<li>Thus , the apparent difference in repression between the experiments might be due to a difference in the level of Lrp protein produced in the test strains , or to the structure of Irp fusion vectors used in the assay .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7603430" target="_blank">7603430</a>]<br></li>
<li>Thus , the GcvA super-repression in the GS852 + 15-213 and +25 -213 lysogens appears to require not only that sites 3 and 2 are on the opposite side of the DNA helix relative to the wild-type gcuT : : lac2 fusion , but also that a functional Lrp protein be present .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9802028" target="_blank">9802028</a>]<br></li>
<li>We postulate that the group of genes controlled jointly by H-NS and Lrp serves to modulate the supply of methylene-THE We believe that manipulation of the one carbon pool can regulate the rate of protein synthesis by managing the concentration of N-formyl methioninyl tRNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7748928" target="_blank">7748928</a>]<br></li>
<li>Since Lrp appears to affect glutamine synthetase through controls common to the Ntr regulon , we expected that other proteins of this regulon would also fail to be induced in strains lacking a functional Lrp protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1346534" target="_blank">1346534</a>]<br></li>
<li>Here , we demonstrate that expression of the E. coli argO gene is affected not only by the internal concentrations of arginine and lysine through the action of ArgP , but also by the general metabolic state of the cell , through regulation by the global transcriptional-regulator Lrp ( leucine-responsive regulatory protein ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19906180" target="_blank">19906180</a>]<br></li>
<li>Unlike other global regulators , the physiological function of Lrp is unclear , and the action of the ligands that regulate the protein are complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9515701" target="_blank">9515701</a>]<br></li>
<li>As with both the ompC-lacZ1 and ompF-lacZ1 transcriptional fusions , the absence of functional Lrp protein did not signiﬁcantly affect regulation of micF-lacZ1 expression by sucrose .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8002608" target="_blank">8002608</a>]<br></li>
<li>These results indicate that the Lrp regulation of glutamine synthetase is not direct ; rather , it is dependent on a functional GInL protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1346534" target="_blank">1346534</a>]<br></li>
<li>This same functional role can be argued for Lrp regulation ( see introduction ) , perhaps implying that the H-NS protein is involved in fine tuning this regulatory network in some way .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7748928" target="_blank">7748928</a>]<br></li>
<li>Target gene regulation is , not surprisingly , affected by the level of the regulatory protein ; this is specifically true for Lrp [ 66,92,116 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18405378" target="_blank">18405378</a>]<br></li>
<li>After fractionation of the crude extracts and pure proteS inDsS-opnolyacryl-amide gels , protein was transferred toa membrane and anti-body-reactivepolypeptides were visualized after successive treatment with rabbit antibodiesraised against Lrp and goat-anti-rabbit antibodiesconjugated to alkaline phosphataseA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2040596" target="_blank">2040596</a>]<br></li>
<li>This result conﬁrms both the integrity of the nonspeciﬁcally bound Lrp roadblock ( demonstrating that it can disrupt protein movement ) and that EcoRI 's sliding shows the expected response to the large Lrp roadblock .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25785689" target="_blank">25785689</a>]<br></li>
<li>Samples were removed 550 throughout growth of the culture and assayed for Lrp , total protein , or b-galac-tosidase activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955316" target="_blank">8955316</a>]<br></li>
<li>However , this seems unlikely since under the growth conditions used in our experiments , the Lrp protein is not expected to occupy Lrp site III ( 48 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16321930" target="_blank">16321930</a>]<br></li>
<li>Results The hns-118 mutation affects both TDH enzyme activity and gcv-lacZ expression Under certain growth conditions , the H-NS protein modulates the activation of iivlH expression by Lrp [ 9 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7748928" target="_blank">7748928</a>]<br></li>
<li>Alternatively , the activity of Lrp might be modulated , either by an unidentified metabolite or metabolites or by interaction with another regulatory protein whose expression or activity is modulated in response to changing physiological conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1346534" target="_blank">1346534</a>]<br></li>
<li>These results demonstrate that Lrp represses the expression of the ilv : : lacZ transcriptional fusion construct described above and that this repression is dependent on a site-specific interaction between Lrp and its consensus-like site within the ilvGMEDA leader region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8900118" target="_blank">8900118</a>]<br></li>
<li>Taken together , these in vitro results show that these amino acids act directly to induce dad expression and strongly suggest that they are not converted to other compounds ( 21 ) in order to alleviate Lrp repression of dad in vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955407" target="_blank">8955407</a>]<br></li>
<li>This view derived primarily from the fact that Lrp tends to activate some operons involved in biosynthesis and repress others involved in catabolism .1 Another view advanced by D'Ari et al. 13 is that Lrp is primarily a DNA-organizing protein , much like HNS , IHF , and HU , which also affect gene expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11575919" target="_blank">11575919</a>]<br></li>
<li>Despite the ability of Lrp to-bind DNA semispecifically [ 76 ] , expression of the great majority of genes is unaffected by deletion of lrp ( see Fig. 1A in [ 69 ] and Fig. 3 in [ 68 ] ) , so its generalized effects as a histonelike protein are limited .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18405378" target="_blank">18405378</a>]<br></li>
<li>Results Lrp stimulates argO expression in a leucine-sensitive manner The fact that Lrp affects the expression of many genes involved in the metabolism and transport of amino acids , including the artPIQM operon involved in arginine uptake ( Hung et al. , 2002 ; Tani et al. , 2002 ; Cho et al. , 2008 ) , and our interest in establishing a coherent global picture of the regulation of arginine metabolism in E. coli ( Caldara et al. , 2006 ; 2008 ) , urged us to analyse argO promoter ( P ) argO activity in isogenic lrp + and lrp : : Tn10 strains .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19906180" target="_blank">19906180</a>]<br></li>
<li>The apparent discrepancy between the effect of Lrp shown here ( leading to a 1.1 - to 1.3-fold decrease in ompF-lacZ1 expression ) and the effect of Lrp on OmpF expression as analyzed by two-dimensional gel electrophoresis ( an ; 2-fold increase in expression ) led us to investigate other factors involved in regulation of ompF expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8002608" target="_blank">8002608</a>]<br></li>
<li>The GcvA ( 31 ) and Lrp ( 10 ) proteins are required for expression of the gcv operon , and the GcvA and PurR proteins ( 30 ) are involved in negative regulation of the gcv operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7928983" target="_blank">7928983</a>]<br></li>
<li>The expression of most of the genes encoding these proteins is not induced by the substrates they transport but is controlled by the global regulation systems , such as the Lrp protein , and activated by the signal compounds involved in the intracellular communication .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17025177" target="_blank">17025177</a>]<br></li>
<li>Regulation of gcvB expression GcvA-mediated regulation of the gcvTHP operon requires the GcvR and Lrp proteins and is responsive to the addition of glycine and purines to the growth medium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10972807" target="_blank">10972807</a>]<br></li>
<li>Therefore , the regulation of the gcvB gene was examined in vivo with respect to the effects of glycine and purine supplementation of the growth medium and with respect to the GcvR and Lrp proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10972807" target="_blank">10972807</a>]<br></li>
<li>Lrp , and H-NS iii coiitrolling f i , ~ expression and the in \ ~ olveineiit of these tliree proteins and RpoS in cross-control of inembers of each of tlie regulons , ule were interested in exploriiig tlie possibility that tlie pliase-variable Ji17iA gene niiglit be iiifluenced by RpoS aild , hence , by growlli pliase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9108285" target="_blank">9108285</a>]<br></li>
<li>Oshima , T. , Ito , K. , Kabayama , H. , and Nakamura , Y. ( 1995 ) Regulation of lrp gene expression by H-NS and Lrp proteins in Escherichia coli : dominant negative mutations in lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16238633" target="_blank">16238633</a>]<br></li>
<li>Microarray analysis of gene regulation by heterologous Lrp proteins -- effects on regulon membership The functional conservation of Lrp orthologs was more broadly assessed by using microarrays to analyze gene regulation , when lrp alleles from E. coli , P. mirabilis , or V. chol-erae ( all fused to the same expression sequences ) were used to complement the lrp null mutation in E. coli K-12 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18405378" target="_blank">18405378</a>]<br></li>
<li>These are Lrp , integration host factor ( IHF ) , and the cAMP receptor protein-cAMP complex ( cAMP-CRP ) , all of which negatively modulate osmY expression in response to growth-phase and medium composition .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8253679" target="_blank">8253679</a>]<br></li>
<li>During exponential phase , repression by Lrp , CRP , or IHF can not account for the low level of expression because similar low expression was found in mutants deficient for these regulators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8253679" target="_blank">8253679</a>]<br></li>
<li>Thus , in the rpoS background , repression of osmY-lacZ expression by Lrp appears to be relieved during entry into stationary phase , although in a wild-type background Lrp clearly continues to repress osmY-lacZ expression in stationary phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li>
<li>V. cholerae gave the most divergent expression pattern : gltB mRNA levels were halved by Lrp in log phase , but increased about fivefold by Lrp in early stationary phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18405378" target="_blank">18405378</a>]<br></li>
<li>Together , the best explanation for the present data is that leuO expression is induced indirectly via the increase of cellular Lrp level in response to ppGpp level increase caused by the slow growth rate when cells are entering stationary phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11006083" target="_blank">11006083</a>]<br></li>
<li>The level of osmY-lacZ expression was repressed by Lrp both during exponential-growth and after entrance into stationary phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li>
<li>Taken together , these results suggest that both H-NS and LRP may require DNA sequences located several hundred base pairs upstream of the primary stpA promoter in order to regulate its expression correctly and that Lrp may in part act to overcome the negative effect of H-NS at the stpA promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9006049" target="_blank">9006049</a>]<br></li>
<li>These authors also examined the expression of stpA in response to H-NS , Lrp , and temperature and the requirement for promoter-distal DNA for regulation , and they obtained results in good agreement with our own .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9006049" target="_blank">9006049</a>]<br></li>
<li>The following work demonstrates that expression of the gItBDF operon is regulated by Lrp at the level of transcription and that Lrp plays a role in the repression of glutamate synthase expression in a rich medium in which nitrogen is present in the form of amino acids and nucleic acid bases as well as ammonia .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7901196" target="_blank">7901196</a>]<br></li>
<li>Thus , we concluded that Lrp participates in transcription regulation of at least eight aminoacyl-tRNA synthetase genes , of which expression of ﬁve aminoacyl-tRNA synthe-tase , including AlaRS , LysRS , SerRS , ThrRS and TyrRS , is repressed by Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2162465" target="_blank">2162465</a>]<br></li>
<li>Further , in addition to inhibiting ompF expression , MicF negatively regulates the transcription factor Lrp , which in turn is responsible for the regulation of genes involved in a number of functions that we also identiﬁed in our microarray data , including amino acid metabolism and transport ( 78 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23564175" target="_blank">23564175</a>]<br></li>
<li>For instance , the highly accumulated aromatic amino acids are all under the control of a single transcription factor , TyrR Lrp ( Tables 3 and S1 ) and thus the expression of a number of genes involved in the metabolism of amino acids should be indirectly regulated in the absence of Lrp , leading to inﬂuence in the intracellular pool of respective amino acids .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2162465" target="_blank">2162465</a>]<br></li>
<li>A yeiL -- lacZ transcriptional fusion showed that aerobic yeiL expression increases at least sixfold during stationary phase , requires RpoS , and is positively autoregulated by YeiL , positively activated by Lrp ( and IHF in the absence of FNR ) and negatively regulated by FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11101674" target="_blank">11101674</a>]<br></li>
<li>First , the lrp gene , which encodes a global regulator Lrp , was selected as a target ; this was because Lrp positively regulates the expression of genes involved in L-valine biosynthesis , such as ilvIH .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17463081" target="_blank">17463081</a>]<br></li>
<li>Lrp regulates the expression of many genes , either positively or negatively ( Calvo and Matthews , 1994 ) , but it is unusual for Lrp to be both an activator and a repressor in the same operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10216857" target="_blank">10216857</a>]<br></li>
<li>Thus , although the high-affinity branched-chain amino acid transport proteins are negatively regulated by Lrp obic conditions also served to elevate expression from the opp operon promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li>
<li>It will be interesting to learn whether the expression of other genes involved in serine biosynthesis or one-carbon metabolism are regulated by Lrp activation and CRP-cAMP repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9171388" target="_blank">9171388</a>]<br></li>
<li>Lrp , by regulating the expression and activity of glutamate synthase , can override induction of nitrogen-regulated operons by ammonia limitation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li>
<li>Interaction between Lrp and Nitrogen Metabolism Deficiencies in glutamate synthase expression are associated with the inability to induce nitrogen-regulated genes in response to nitrogen limitation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li>
<li>When levels of Lrp are low , glutamate synthase expression will also be low , and the levels of nitrogen-regulated genes will remain low even under nitrogen-limiting conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li>
<li>Furthermore , stpA expression is temperature regulated , activated during exponential-growth , regulated by Lrp , and induced by osmotic stress ( 16 , 37 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16980475" target="_blank">16980475</a>]<br></li>
<li>These data suggest that sdaB expression is regulated by Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1532173" target="_blank">1532173</a>]<br></li>
<li>Many of the polypeptides whose expression is regulated by Lrp are also members of other regulons .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1346534" target="_blank">1346534</a>]<br></li>
<li>In this expression pattern , only five genes are known to be regulated by Lrp ( lamB , malM , yabI , pheA , and hupB ) ; however , 17 genes of this cluster are of unclassified function .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12754220" target="_blank">12754220</a>]<br></li>
<li>We have used this method to con-ﬁrm that Lrp regulates the expression of and leuDBCA livJK and to demonstrate for the ﬁrst time that expression of the malF gene , and other genes of the maltose operon , is regulated by Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7751276" target="_blank">7751276</a>]<br></li>
<li>Lrp does not affect the synthesis of ArgP , the arginine-dependent transcriptional-activator of argO A major question concerning the effect of Lrp on argO expression is whether Lrp acts directly or indirectly on PargO .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19906180" target="_blank">19906180</a>]<br></li>
<li>Lrp and ArgP act as competitive activators on PargO Since both Lrp and ArgP stimulate argO expression and bind to overlapping targets in the argO control region , we were interested in analysing potential interferences in the action of these transcriptional-activators on argO expression , in response to environmental conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19906180" target="_blank">19906180</a>]<br></li>
<li>It was shown that Lrp and ArgP behave as competitive activators able to activate argO expression under different conditions ( 35 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21441513" target="_blank">21441513</a>]<br></li>
<li>We believe that the effect of Lrp on the level and activity of glutamine synthetase is secondary to the effect of Lrp on expression of glutamate synthase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1346534" target="_blank">1346534</a>]<br></li>
<li>Since very low levels of glutamate synthase are expressed inlrp strains , the possibility was examined that the effect of Lrp on glutamine synthetase expression was indirect and was secondary to its effect on glutamate synthase expression ( 43 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li>
<li>The regulation of glutamine synthetase by Lrp can be explained as an effect secondary to the regulation of glutamate synthase activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1346534" target="_blank">1346534</a>]<br></li>
<li>The effect of Lrp on GlnA expression and activity is indirect , probably involving the direct Lrp effect on GltBD expression that , in turn , modulates the intracellular ratios of α-ketoglutarate , glutamate , and glutamine [ 49 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17233899" target="_blank">17233899</a>]<br></li>
<li>The effect of Lrp on glutamine synthetase expression was shown to require a functional NRI , protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li>
<li>In strains carrying a glnL deletion and in strains carrying the glnL2302 allele , which directs the synthesis of a GInL protein that is constitutively active , expression of glutamine synthetase is no longer regulated by Lrp , demonstrating that the effect of Lrp on glutamine synthetase levels is indirect and requires an intact glnL gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1346534" target="_blank">1346534</a>]<br></li>
<li>In the absence of a functional Lrp protein , glutamate synthase activity is undetectable , and we postulate that this leads to the observed decrease in the expression and activity of glutamine synthetase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1346534" target="_blank">1346534</a>]<br></li>
<li>Like some other important global regulators , for example , cyclic AMP response protein ( 15 ) , Lrp activates expression of some operons and represses expression of others .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8432705" target="_blank">8432705</a>]<br></li>
<li>It is thought that Lrp acts differently on these operons in that it represses expression of sdaA , oppA , and tdh and induces expression of ilvIH and serA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1917830" target="_blank">1917830</a>]<br></li>
<li>example Crp ( Mallick and Herrlich , 1979 ) , Lrp activates expression of some operons and represses expression of others .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8483418" target="_blank">8483418</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_911">
Lrp activates expression of some operons and represses expression of others .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7665463" target="_blank">7665463</a>]<br></label>
<input id="act_similarity_911", type="checkbox">
<div class="ACT_similar">
<li>Lrp activates the expression of some operons and represses the expression of others .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8508774" target="_blank">8508774</a>]<br></li></div>
<li>It is focused mainly on Lrp and its interaction with DNA , on the organization and regulation of the lrp operon , and on the way in which Lrp regulates expression of several operons .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li>
<li>In this scenario , Lrp would impose another level of regulation , adjusting the expression levels of lysP and cadBA in response to the physiological status of the cell .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21441513" target="_blank">21441513</a>]<br></li>
<li>This sta-tionary-phase induction of lrp expression is independent of Lrp autogenous regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955316" target="_blank">8955316</a>]<br></li>
<li>Lrp was also found to be involved in the regulation of cadBA expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21441513" target="_blank">21441513</a>]<br></li>
<li>These observations led us to investigate the relationship between regulation of porin expression by osmolarity , by micF , and by Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8002608" target="_blank">8002608</a>]<br></li>
<li>This technique revealed that malT , malEFG , and malB-lamB-malK are all activated two-to ﬁvefold by Lrp and conﬁrmed that Lrp regulates expression of the leuDBCA and livHJKG operons .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7751276" target="_blank">7751276</a>]<br></li>
<li>This is probably because of the decreased synthesis of Lrp in LB , as production of Lrp from a plasmid increased expression even in LB ( Table 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11985712" target="_blank">11985712</a>]<br></li>
<li>This increase of gene expression is caused by alleviation o f a H-NS-mediated blockage of Lrp activator function .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7748928" target="_blank">7748928</a>]<br></li>
<li>This might involve , for instance , an increase in expression of an alternative reductase gene repressed by Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12706508" target="_blank">12706508</a>]<br></li>
<li>These results suggest that the induction of the ygaZH expression by - valine is likely to be mediated by Lrp , which functions as an L activator of ygaZH expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17463081" target="_blank">17463081</a>]<br></li>
<li>These results suggest that Lrp , the product of lrp , negatively regulates livJ and livK expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1729203" target="_blank">1729203</a>]<br></li>
<li>In addition , stpA expression is stimulated by the global regulator Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8890170" target="_blank">8890170</a>]<br></li>
<li>While Lrp stimulated lysP expression in the absence of lysine , this global regulator did not alter the lysine-dependent repression of lysP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21441513" target="_blank">21441513</a>]<br></li>
<li>These activity measurements are consistent with repression of lysU by Lrp and with our observations of LysU expression by electrophoretic analysis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1346534" target="_blank">1346534</a>]<br></li>
<li>It should be noted that the micF deletion also abolishes the repression of OmpC expression exerted by Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8002608" target="_blank">8002608</a>]<br></li>
<li>Lrp , through its control of glutamate synthase , induces the Ntr response , which stimulates expression of polyamine catabolic enzymes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12730324" target="_blank">12730324</a>]<br></li>
<li>These mutants were called activation mutants since they affected the ability of Lrp to activate expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8432705" target="_blank">8432705</a>]<br></li>
<li>Leucine causes the Lrp activator function to be come limiting for ilvIH expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8152423" target="_blank">8152423</a>]<br></li>
<li>As demonstrated here , lysP expression is induced under lysine limitation and requires ArgP and Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21441513" target="_blank">21441513</a>]<br></li>
<li>Lrp has major effects on GlnA expression and adenylylation status in response to ammonia levels [ 2,5,7 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17233899" target="_blank">17233899</a>]<br></li>
<li>Leucine and Lrp together stimulated expression two - to fivefold in class 3 mutants .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1532173" target="_blank">1532173</a>]<br></li>
<li>723 RESULTS Lrp inhibits dad expression in vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955407" target="_blank">8955407</a>]<br></li>
<li>Effect of Lrp and micF on porin expression in outer membrane preparations .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8002608" target="_blank">8002608</a>]<br></li>
<li>Crp and Lrp both activate serA expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11985712" target="_blank">11985712</a>]<br></li>
<li>Lrp stimulates cadBA expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21441513" target="_blank">21441513</a>]<br></li>
<li>As shown in Fig. 1 , eightfold-elevated Lrp levels led to only a relatively slight increase in expression from the ilvIH promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8432705" target="_blank">8432705</a>]<br></li>
<li>Since the levels of Lrp in strain CV1069 are maintained relatively low because of autogenous regulation , it was of interest to determine the effect of higher levels of Lrp on expression from the lrp promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8144448" target="_blank">8144448</a>]<br></li>
<li>Thus , haploid levels of Lrp repress expression from the lip promoter only a few fold , but higher levels of Lrp have a much more profound effect .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8144448" target="_blank">8144448</a>]<br></li>
<li>The V. cholerae PgltB that is less similar to E. coli gltB ( Vc2373 ) was strongly expressed in its native host , with no significant effect of Lrp , while showing lower expression levels and activation ( 3-fold ) in E. coli , and yielding low expression but repression by Lrp ( 3-fold ) in the P. mirabilis background .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18405378" target="_blank">18405378</a>]<br></li>
<li>There is no effect of Lrp on recA expression , at least in E. coli and V. cholerae under our conditions [ 69 ] and N. Dolganov , pers .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18405378" target="_blank">18405378</a>]<br></li>
<li>Lrp had insignificant effects on Vc2376 in the native host and in E. coli , while in P. mirabilis there was very low expression irrespective of Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18405378" target="_blank">18405378</a>]<br></li>
<li>To understand the reason for this positive effect of coexpressing the lrp and ygaZH genes , the regulation of ygaZH expression by Lrp was investigated by real-time RT-PCR analysis by using the W3110 and Wlrp strains in response to 5 M - valine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17463081" target="_blank">17463081</a>]<br></li>
<li>In addition , Lrp had a positive effect on the expression of the cadBA operon when cells were exposed to moderate acidic stress in the presence of lysine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21441513" target="_blank">21441513</a>]<br></li>
<li>P. mirabilis , in contrast ( triangles ) , showed a strong repressive effect of Lrp , though no real growth-phase dependent change in expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18405378" target="_blank">18405378</a>]<br></li>
<li>Lrp had little effect on hdeAB expression in LB medium , in contrast with the strong repression of hdeAB by Lrp that occurs in minimal media ( 42 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18083817" target="_blank">18083817</a>]<br></li>
<li>Lrp had very little effect on sucrosemediated repression of ompF-lacZ1 expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8002608" target="_blank">8002608</a>]<br></li>
<li>Recent studies have established that Lrp exerts its effect on ompF and ompC expression independently of OmpR and EnvZ and that OmpF and OmpC expression is normally regulated by osmolarity in an Irp strain ( 46 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li>
<li>Thus , ompF expression is positively regulated by Lrp at the level of translation , and this effect of Lrp is exerted independently of the effect of sucrose .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8002608" target="_blank">8002608</a>]<br></li>
<li>To clarify the mode of Lrp control , we examined the feedback effect of dominant negative lrpmutations on lrp expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7603430" target="_blank">7603430</a>]<br></li>
<li>Our studies suggest that the positive effect of Lrp on OmpF expression stems from a negative effect of Lrp on the expression of micF , an antisense RNA that inhibits ompF translation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8002608" target="_blank">8002608</a>]<br></li>
<li>The positive effect on OmpF expression exerted by Lrp is probably due to the negative effect of Lrp on the expression of micF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8002608" target="_blank">8002608</a>]<br></li>
<li>Lrp must be bound to all three sites for maximal reporter gene expression To determine the extent to which each of the three sites contributes to the transcriptional activation of gltBDF by Lrp , several additional mutants were generated , footprinted , moved into lacZ operon fusions , and introduced into the chromosome of strain PS2209 ( lac ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9236118" target="_blank">9236118</a>]<br></li>
<li>However , in contrast to ilvIH , mutagenesis of the ÿ70 site in dad had little effect on the increase in dad expression by alanine , suggesting that this mechan-ism may not have a signi ® cant role in Lrp activation in this operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10216857" target="_blank">10216857</a>]<br></li>
<li>Unlike the GcvA-inducible gcvTHP operon , activation of the gcvB promoter is not dependent on the Lrp protein , as the levels of gcvB ± lacZ expression in the lrp mutant lysogen GS997lgcvB ± lacZ are as high or higher than in the wild-type lysogen [ an lrp mutant is functionally Gcv ( Stauffer and Stauffer , 1994 ) , 2 and the higher gcvB ± lacZ expression in GM media may reflect the accumulation of intracellular glycine in the absence of Gcv activity ( Ghrist and Stauffer , 1995b ) ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10972807" target="_blank">10972807</a>]<br></li>
<li>et al. ( 31 ) have shown that Lrp binds to the ilvIH promoter and activates the expression of the ilvIH operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1917830" target="_blank">1917830</a>]<br></li>
<li>Furthermore , because the expression of the ilvGM operon is repressed by Lrp ( 18 ) , the native promoter of the ilvGM operon , to which Lrp binds , was replaced with the tac promoter to abolish the repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17463081" target="_blank">17463081</a>]<br></li>
<li>Lrp is thought to be evolutionarily related to AsnC , a regulatory protein that can positively stimulate the expression of one operon and negatively affect its own promoter ( 47 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1917830" target="_blank">1917830</a>]<br></li>
<li>If yojI is indeed regulated by Lrp , it is to be expected that cloning of yojI devoid of its regulatory sequences and under the control of another promoter should lead to Lrp-independent expression of the gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19074390" target="_blank">19074390</a>]<br></li>
<li>We identified a region just upstream from the lrp promoter that binds Lrp in vitro and showed that upon deletion of this region , autogenous regulation by Lrp in vivo is lost .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8144448" target="_blank">8144448</a>]<br></li>
<li>Thus , the region between -88 and - 40 , which binds Lrp in vitro , makes the lrp operon sensitive to repression by Lrp in vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8144448" target="_blank">8144448</a>]<br></li>
<li>Titration of Lrp with PCR9 showed that the more slowly migrating complexes were favored with increasing concentrations of Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7916011" target="_blank">7916011</a>]<br></li>
<li>Radiolabelled wild-type PCR12 was incubated with increasing concentrations of puriﬁed Lrp as described in Experimental procedures .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9515701" target="_blank">9515701</a>]<br></li>
<li>Figure 2 shows the results of titration experiments performed with the wild-type and 10 constructs and increasing concentrations of puriﬁed Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12218014" target="_blank">12218014</a>]<br></li>
<li>Binding of Lrp to wild-type and 10 constructs in vitro , as measured by gel mobility shift experiments .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12218014" target="_blank">12218014</a>]<br></li>
<li>Here we showed that Lrp binds in vitro to the wild-type and 2 constructs with slightly higher afﬁnity than to the 4 , 6 , 8 , and 10 constructs and that the increased afﬁnity is due to increased cooperativity of binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12218014" target="_blank">12218014</a>]<br></li>
<li>Higher concentrations of Lrp in the gel-shift assay resulted in multiple bands of altered mobility and it was difficult to predict with certainty which bands resulted from various multimers of the Lrp protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9802028" target="_blank">9802028</a>]<br></li>
<li>Considering that different concentrations of Lrp gave similar patterns in both gel retardation and DNase I protection , it seems clear that , once Lrp binds to any of this region , it binds to all of it .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11985712" target="_blank">11985712</a>]<br></li>
<li>Lrp binds speciﬁcally to the intergenic region between micF and ompC , as indicated by mobility shift assays and by DNase I footprinting .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8002608" target="_blank">8002608</a>]<br></li>
<li>Acknowledgments -- We thank Debra A. Willins for purified Lrp , T. Murphy for help in preparing plasmid pCV225 , J. Landgraf for help in preparing figures , and G. W. Feigenson , P. C. Hinkle , and P. A. Karplus , for helpful discussions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8636076" target="_blank">8636076</a>]<br></li>
<li>Lrp Lrp -200 GACCGTGTCCTTGCAATAGGCCTGCAAGTCAAGGTGGTCTGGCGCTCGTAATTAAGAAC -140 -80 TGTTCTTAAAACATTCACTTTTGCTTATGTTTTCGCTGATATCCCGAGCGGTTTCAAAAT ACAAGAATTTTGTAAGTGAAAACGAATACAAAAGCGACTATAGGGCTCGCCAAAGTTTTA bent DNA -10 +1 CRP IHF -20 TGTGATCTATATTTAACAAAGTGATGACATTTCTGACGGCGTTAAATACCGTTCAATGCG CRP +41 ATCTATAGTCATAGATTTCGGCAGCTAACAGTAAGATGGCTATAATTATTGACTAAGTCT +101 GGCTGTAATGGTCGTTATTCATCACTCATCGCTTTTGTGATGGCGACCATTGACTTCTGT CCGACATTACCAGCAATAAGTAGTGAGTAGCGAAAACACTACCGCTGGTAACTGAAGACA .161 AGAGGGTGAAGTCTCTCCCTATTCAGCAATGCAACCTCGTGTTCGCAGGCTCAAATTACG TCTCCCACTTCAGAOAGGGATAAGTCGTTACGTTGGAGCACAAGCGTCCGAGTTTAATGC ACKNOWLEDGMENTS We thank Erhard Bremer and his coworkers for providing bacterial strains and Merna Villarejo for sharing unpublished data .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8253679" target="_blank">8253679</a>]<br></li>
<li>Acknowledgements We thank J. Krakow for puri ® ed CRP , J. Calvo for puri ® ed Lrp , and J. Fuchs and M. Lobocka for bacterial strains and plasmids .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10216857" target="_blank">10216857</a>]<br></li>
<li>ACKNOWLEDGMENTS We are grateful to J. Krakow for puriﬁed CRP , J. Calvo for puriﬁed Lrp , and J. Fuchs and M. Lobocka for bacterial strains and plasmids .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955407" target="_blank">8955407</a>]<br></li>
<li>Althoughthe Lrp inhibitory effect is detectablewith both forms of RNA polymerases in vitro ( Fig. 6 ) , only repression of EoSis physiologically significant , because the levels of E ~ ~ O-dependetnra t nscription of aidB in vivo appear to be negligible in the absence of ada-dependent induction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8809748" target="_blank">8809748</a>]<br></li>
<li>In a lrp background , alanine repressed neither avtA nor yfbQ , which implies that Lrp mediates alanine repression ( Fig. 2B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20729367" target="_blank">20729367</a>]<br></li>
<li>Addition of ILV interfered to a moderate extent with the repression of Plrp by all three Lrp orthologs ( Fig 4E ) , but the differences observed in the MOPS glucose cultures were maintained , with substantially greater repression by P. mirabilis Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18405378" target="_blank">18405378</a>]<br></li>
<li>Plrp in E. coli is autogenously repressed [ 89 ] , and all three lrp promoters in all three hosts ( with one exception ) show lower expression in the presence of Lrp than in its absence ( Fig. 11 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18405378" target="_blank">18405378</a>]<br></li>
<li>PlivK , which in E. coli is activated by Lrp in the absence of leucine and repressed in its presence , was regulated equally by all three Lrp orthologs with one exception : in the presence of leucine , the E. coli Lrp represses less than the others ( Fig. 4D , G ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18405378" target="_blank">18405378</a>]<br></li>
<li>A similar caveat may apply to Plrp , which in addition to being autogenously repressed by Lrp [ 89 ] may be activated by GadE in E. coli [ 113 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18405378" target="_blank">18405378</a>]<br></li>
<li>Once again the assumption is supported in general -- most Plrp combinations are unaffected or repressed by Lrp ( Fig. 11D ) , while all but one of the PgltB combinations are unaffected or activated by Lrp ( Fig. 11E ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18405378" target="_blank">18405378</a>]<br></li>
<li>However DNAse I footprint analyses suggest that Lrp and ArgR bind independently of one another ( Fig. 9A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17233899" target="_blank">17233899</a>]<br></li>
<li>It is interesting that the Pasteurellaceae appear to form an outgroup with respect to Lrp , as the Lrp differences are far more pronounced than their overall relationship to neighboring bacterial genera would suggest ( see Fig. 2 in [ 81 ] , and Fig .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18405378" target="_blank">18405378</a>]<br></li>
<li>Expression from the livJ promoter was very sensitive to the level of Lrp ; it was repressed about 8-fold when Lrp was expressed from a moderate-copy-number plasmid and about 30-fold when Lrp was expressed from a high-copy-number plasmid ( compare data for SH-51 in Fig. 5 with data for SH183 and SH184 in Fig. 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1729203" target="_blank">1729203</a>]<br></li>
<li>The amount of Lrp decreased 2.7-fold during the initial phase of growth , was constant during log phase , and then increased to the beginning level as cell growth ceased ( Fig. 1B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955316" target="_blank">8955316</a>]<br></li>
<li>When P1 transcription was normalized to amp , P1 basal-level transcription decreased by at least a factor of 5 , but the Lrp stimulation factor increased from 1.5 to 3.8 ( Fig. 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1447135" target="_blank">1447135</a>]<br></li>
<li>of Lrp , but this number increases by setting the cut-off level of SELEX pattern v10 ( Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2162465" target="_blank">2162465</a>]<br></li>
<li>b-Galactosidase activity in cells grown with cAMP ( mM ) Host strain lrp + Plasmid 0 1.0 2.0 3.0 pserAP1P2 pserAP2 pserAP1 pserAP1P2 pserAP2 pserAP1 19 195 3 600 1 475 2 725 17 425 4 075 3 175 3 025 23 875 4 025 2 900 3 300 19 925 1 000 20 225 6 400 3 425 2 950 lrp -- Binding of Lrp to the serA promoter : activation by Lrp at P1 How Lrp stimulates transcription at serA ( Lin et al. , 1990 ) is unknown .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11985712" target="_blank">11985712</a>]<br></li>
<li>An increase in Lrp synthesis in LB stimulates expression from a chromosomal serA : : lacZ fusion As suggested by the long Lrp binding site , we showed earlier that transcription from serA : : lacZ is strongly affected by Lrp , and is thus much lower in an lrp mutant ( Lin et al. , 1992a ; Chen et al. , 1997 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11985712" target="_blank">11985712</a>]<br></li>
<li>A similar situation was also described for the gene csiD , in which Lrp activates E-ss-dependent transcription ( Marschall et al. , 1998 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9663683" target="_blank">9663683</a>]<br></li>
<li>However , according to the model presented by Paul et al. 29 ArgR is supposed to exerts its effect on the gltB promoter by interfering with the Lrp-mediated activation of transcription initiation , but should have no effect by itself ( in the absence of Lrp ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17850814" target="_blank">17850814</a>]<br></li>
<li>In the absence of Lrp , transcription of P2 is activated by cAMP/Crp , a regulator of serB expression as well ( Man et al. , 1997 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11985712" target="_blank">11985712</a>]<br></li>
<li>Upon addition of methylated Ada protein ( IOpmol ) , aid6 transcription is restored almost to the same extent as meAda-activatedtranscription in the absence of Lrp ( Fig. 6 , lanes 2 and 4 for En '' , and lanes 6 and 8 for EoS ) , suggesting that activation by meAdais dominant over repression by Lrp in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8809748" target="_blank">8809748</a>]<br></li>
<li>Results of the in vitro transcription experiments demonstrate that EDs-dependenttranscrip-tion can be stimulated by methylatedAda protein.Another activator might sharewith Ada the ability of stimulatingED ' as well as the mechanismfor bypassing Lrp repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8809748" target="_blank">8809748</a>]<br></li>
<li>Since the insert in malF was positively regulated by Lrp , we studied the regulation of transcription from inserts in several genes involved in maltose metabolism , aided by gifts of strains from several investigators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7751276" target="_blank">7751276</a>]<br></li>
<li>Since transcription of the ilvZH operon is positively regulated by the Lrp protein [ 12 ] , we investigated the possibility that the down-regulation of iluZH transcription during amino acid downshift is directly related to inhibition of transcription of the lrp gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7557316" target="_blank">7557316</a>]<br></li>
<li>Many RpoS-dependent genes are regulated by othe transcription factors ( Table 3 ) ; for example , the promoters of genes ygaU and yhjD are known to be activated by Lrp ( 77 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15576800" target="_blank">15576800</a>]<br></li>
<li>Although the gltBDF operon of E. coli is positively regulated by Lrp ( 13 ) , in vitro transcription with puriﬁed Lrp produced minimal activation of transcription from the gltBDF promoter ( B. R. Ernsting , unpublished results ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11395454" target="_blank">11395454</a>]<br></li>
<li>While transcription of sduC is regulated by Lrp , it is curious that sduB , in the same operon , is not regulated noticeably by Lrp ( Shao and Newman , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8026499" target="_blank">8026499</a>]<br></li>
<li>Therefore , one possible explanation for the above-described results is that Lrp could be acting as a positive regulator of yojI transcription , which would lead to an increase in YojI levels in the membrane .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19074390" target="_blank">19074390</a>]<br></li>
<li>Several lines of evidence , including the evidence presented here , indicate that Lrp acts directly to activate the transcription of the ilvIH operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1447135" target="_blank">1447135</a>]<br></li>
<li>These results , considered altogether , fulfil the predictions that follow from the proposal that H-NS reduces the ability of Lrp to activate ilvIH transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8152423" target="_blank">8152423</a>]<br></li>
<li>For operons controlled by Lrp , does Lrp act directly to activate or repress the transcription of target operons , or does it act indirectly by affecting the expression of some other regulatory gene which , in turn , affects the expression of a target operon ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1447135" target="_blank">1447135</a>]<br></li>
<li>Although Lrp alters the expression of numerous ope-rons , precisely how Lrp acts to repress or activate transcription is not known ( Newman and Lin , 1995 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10216857" target="_blank">10216857</a>]<br></li>
<li>After SELEX-chip screening , a total of 23 transcription factors were indicated to be under the direct control of Lrp , altogether forming a big network , in which Lrp is located on the top of the hierarchy .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2162465" target="_blank">2162465</a>]<br></li>
<li>Thus , the life style of E. coli under these transcription factors is also under the control of Lrp , which monitors the nutritional conditions in the environment .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2162465" target="_blank">2162465</a>]<br></li>
<li>Together , these results suggest that Lrp potentiates the activator effect of ArgP on lysP transcription in the absence of lysine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21441513" target="_blank">21441513</a>]<br></li>
<li>Therefore , it would be reasonable to assume that in the absence of Lrp , the residual transcription of yojI in pCLO7 should lead to some degree of microcin resistance .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19074390" target="_blank">19074390</a>]<br></li>
<li>Therefore , Lrp seems to be an activator of lysP transcription in the absence of lysine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21441513" target="_blank">21441513</a>]<br></li>
<li>In the absence of Lrp , there was a dramatic increase in the transcription of both promoters , P2 from 1475 to 10 500 and P1 from 2725 to 6200 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11985712" target="_blank">11985712</a>]<br></li>
<li>In particular , we studied three transcription factors ( Lrp , PdhR , and FecI ) with substantial connectivity in the network mapped by CLR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17214507" target="_blank">17214507</a>]<br></li>
<li>In this communication , we provide evidence that Lrp negatively regulates the transcription of ompC and micF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8002608" target="_blank">8002608</a>]<br></li>
<li>Nonetheless transcription from this chimeric promoter was activated over 20-fold by Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11395454" target="_blank">11395454</a>]<br></li>
<li>This suggests that Lrp activates transcription from the P1 promoter and represses transcription from P2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11985712" target="_blank">11985712</a>]<br></li>
<li>Is Lrp sufficient for the activation of transcription in vitro , or are additional factors necessary ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1447135" target="_blank">1447135</a>]<br></li>
<li>Does Lrp stimulate transcription in vitro from P1 , and does it repress transcription from other start points ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1447135" target="_blank">1447135</a>]<br></li>
<li>Lrp represses dad transcription in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955407" target="_blank">8955407</a>]<br></li>
<li>Effect of Lrp on transcription of other carbohydrate utili-zation operons .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7751276" target="_blank">7751276</a>]<br></li>
<li>Like Lrp , Crp activates transcription of some operons and represses transcription of others .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li>
<li>It is also clear that Lrp activates malF transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7751276" target="_blank">7751276</a>]<br></li>
<li>Thus , Lrp activates transcription from fan genes , as it does for pap , daa , and sfa genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li>
<li>, Wurm , R. , Lux , B. , Meltzer , M. , Menzel , A. , and Wagner , R. ( 2005 ) Lrp and H-NS -- cooperative partners for transcription regulation at Escherichia coli rRNA promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19906180" target="_blank">19906180</a>]<br></li>
<li>This regulation is due to a twofold activation by Lrp of transcription of , the gene coding for the activator of the malT maltose operons .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7751276" target="_blank">7751276</a>]<br></li>
<li>These results demonstrate that autogenous regulation of lrp involves Lrp acting directly to cause repression of urp transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8144448" target="_blank">8144448</a>]<br></li>
<li>Even though lrp transcription is autoregulated , some other regulation must override this in order to result in this decrease in Lrp in LB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1532173" target="_blank">1532173</a>]<br></li>
<li>As discussed above , there is evidence that the concentration of Lrp is itself regulated , in part , by autogenous regulation of lrp transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li>
<li>The stimulation or repression exerted by Lrp on different operons in E.coli varies considerably , and effects between 2 - and 50-fold have been found ( Calvo and Matthews , 1994 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8890170" target="_blank">8890170</a>]<br></li>
<li>However , it is unclear why Lrp is also an activator in dad , as Lrp is usually a positive effector of biosynthetic operons ( Calvo and Matthews , 1994 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10216857" target="_blank">10216857</a>]<br></li>
<li>The apparent cooperativity of Lrp binding to the low-affinity sites ( 3 + 4 + 5 ) upstream of ilvIH was also increased in th Genes That Are Evolutionarily Related to Lrp Affinity-purified antibodies against Lrp reacted with Lrp-like proteins in a variety of members of the family Enterobacteriaceae , namely Serratia strain 8011 , Enterobacter aerogenes , Kiebsiella aerogenes , and S. typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li>
<li>The data in Fig. 3 show that the removal of these sites did not alter Lrp repression , but it strongly reduced alanine induction to about the same extent as did the mutations in site 5 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10216857" target="_blank">10216857</a>]<br></li>
<li>The increased cooperativity is most likely the result of an interaction between Lrp bound at upstream and downstream sites , as represented diagrammatically in Fig. 4 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12218014" target="_blank">12218014</a>]<br></li>
<li>The absence of Lrp decreased transcription to 3600 , consistent with the fact that the lrp mutant has Cells grown in the medium stated were subcultured in the same medium , and their b-galactosidase activity was assayed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11985712" target="_blank">11985712</a>]<br></li>
<li>Its activity was increased in the lrp mutant , consistent with the idea that P2 is actually repressed by Lrp ( Table 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11985712" target="_blank">11985712</a>]<br></li>
<li>The induction was independent of the RelA/SpoT , RpoH , KatF , OmpR , AppY , Lrp , PhoB and H-NS proteins during stress conditions that are known to induce or activate these global regulators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1453957" target="_blank">1453957</a>]<br></li>
<li>The abundence of the small nucleoid-asso-ciated proteins , HU , IHF , Fis , Lrp , in the cell makes it likely that many more examples of their participation in transcription regulation of complex promoters will be found .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15066032" target="_blank">15066032</a>]<br></li>
<li>The ompC gene , which was underexpressed in this condition ( Table 3 ) , has a complex regulation including protein regulators ( OmpR , CpxR , Lrp , IHF and EnvY ) and small regulatory RNAs ( micC , rseX and rybB ) that respond to different signals [ 42-46 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22513097" target="_blank">22513097</a>]<br></li>
<li>The direction and level of the inﬂuence on Lrp activity by each amino acid effector appears variable depending on the target genes and under the culture conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2162465" target="_blank">2162465</a>]<br></li>
<li>The whole set of regulation targets herein identiﬁed indicates that Lrp senses the presence of nutritional conditions and regulates not only the transport and metabolism ( synthesis and degradation ) of amino acids , but also the utilization amino acids up to protein synthesis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2162465" target="_blank">2162465</a>]<br></li>
<li>A role for cAMP in the regulation of serA The data in Table 2 are consistent with the model of two promoters for serA , P1 activated by Lrp and P2 repressed , P2 being a quantitatively very minor promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11985712" target="_blank">11985712</a>]<br></li>
<li>The reaction mixtures were pre-incubated for 5 min at +21 -213 insertions were used to lysogenize the purR 37 O C and then 2 pl purified Lrp protein was added to the strain GS852 , in which inosine repression is due solely to reaction mixtures to give final concentrations of approxi - the GcvA/GcvR system .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9802028" target="_blank">9802028</a>]<br></li>
<li>The reaction mixtures preincubated at 370C for 5 min , were 2 pul of purified Lrp protein was added to the reaction mixtures as indicated in Fig. 3 , and incubation was continued for 15 min .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7928983" target="_blank">7928983</a>]<br></li>
<li>The results in Fig. 6B show that in contrast to the strong inhibition by Lrp of transcription from the wild-type plasmid , Lrp had little or no inhibitory effect on Tr1 , Tr2 , and Tr3 when the mutant plasmid was used as the template .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955407" target="_blank">8955407</a>]<br></li>
<li>The net effect of these interactions is that , in intact cells , serA transcription was regulated about twofold by Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12123449" target="_blank">12123449</a>]<br></li>
<li>Binding of Lrp protein to the upstream region of the serA promoter The profound effect of Lrp on serA transcription suggests that Lrp must bind to promoter sequences .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11985712" target="_blank">11985712</a>]<br></li>
<li>The mechanism by which methylation of GATC-II stimulates pap transcription is not known but may involve a reduction in the afﬁnity of Lrp for pap sites 1 , 2 , and 3 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9811649" target="_blank">9811649</a>]<br></li>
<li>The positioning of these regions is also consistent with repression of micF transcription being mediated by Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8002608" target="_blank">8002608</a>]<br></li>
<li>The addition of Lrp increased transcription from P1 but decreased transcription from P2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1447135" target="_blank">1447135</a>]<br></li>
<li>The mechanisms employed by Lrp to regulate transcription vary and often involve interaction with other regulator proteins and NAPs .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21441513" target="_blank">21441513</a>]<br></li>
<li>The experiments reported here provide deﬁnitive evidence that Lrp dimers , bound more than 110 bp upstream of the 235 hexamer , can activate transcription of a s70-dependent promoter , since the lac operon is controlled by what is probably the best-categorized such promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11395454" target="_blank">11395454</a>]<br></li>
<li>The global regulator Lrp also bound to the lysP promoter/control region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21441513" target="_blank">21441513</a>]<br></li>
<li>The in vivo dad transcript seen between Tr2 and Tr3 was not found in vitro Lrp could only be inferred from these studies since the negative effects of Lrp on operon expression were observed only in vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955407" target="_blank">8955407</a>]<br></li>
<li>Lin , W. , Kovacikova , G. , and Skorupski , K. ( 2007 ) The quorum sensing regulator HapR downregulates the expression of the virulence gene transcription factor AphA in Vibrio cholerae by antagonizing Lrp - and VpsR-mediated activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19906180" target="_blank">19906180</a>]<br></li>
<li>DISCUSSION The data in this report strongly suggest that Lrp acts as a repressor to directly inhibit dad expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955407" target="_blank">8955407</a>]<br></li>
<li>The expression of some target operons is activated by Lrp , while the expression of others is repressed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8900118" target="_blank">8900118</a>]<br></li>
<li>The specific activity of , - galactosidase was about twofold higher in strain CV1070 than in strain CV1069 , indicating that Lrp represses its own expression ( Table 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8144448" target="_blank">8144448</a>]<br></li>
<li>The expression of a plasmid-borne micF-lacZ1 operon fusion is inhibited by Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8002608" target="_blank">8002608</a>]<br></li>
<li>The effector switch Like Lrp , AsnC binds DNA and regulates gene expression in the absence or presence of effector .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16528101" target="_blank">16528101</a>]<br></li>
<li>The results reported in Table I show that in the absence of a functional Lrp gene product , the expression of the reporter gene was increased nearly 3-fold ( compare strains IH-G2490 and IH-G2491 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8900118" target="_blank">8900118</a>]<br></li>
<li>The absence of Lrp results in a complete modiﬁcation of this expression pattern : transcription at osmCp2 is higher during exponential phase , and its decelerating phase induction is abolished .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9663683" target="_blank">9663683</a>]<br></li>
<li>The requirements for YeiL , RpoS and Lrp for optimal yeiL expression were retained in the absence of FNR ( Fig. 6b ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11101674" target="_blank">11101674</a>]<br></li>
<li>The 306 bp XbaI-to-KpnI fragment from pGU5 , labelled at the XbaI end with [ a-32P ] - dATP , was incubated with increasing amounts of puriﬁed Lrp protein : lane 0 , no Lrp ; 1 , 4 ng ; 2 , 8 ng ; 3 , 16 ng ; 4 , 32 ng ; 5 , 64 ng .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11985712" target="_blank">11985712</a>]<br></li>
<li>The indicated radiolabelled PCR product was incubated with increasing amounts of puriﬁed Lrp as described in Experimental procedures .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9515701" target="_blank">9515701</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9515701" target="_blank">9515701</a>]<br></li>
<li>The ® t o the data was slightly better when a 32mer was included in the model , although the amount of 32mer is small for concentrations of Lrp lower than 50 mM ( Kd for dissociation of 32mer to hexadeca-mer , 5.48 10ÿ4 M ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11575919" target="_blank">11575919</a>]<br></li>
<li>The fact that Lrp and CRP are required for full activation of PgcvT [ 14,15 ] complicates interpretation of the results .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15621456" target="_blank">15621456</a>]<br></li>
<li>The stimulating effect of Lrp is not sigma factor speciﬁc , and it can only be observed when the CRP box is optimally located ( Fig. 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11532138" target="_blank">11532138</a>]<br></li>
<li>The line indicates the position for data if no effect of Lrp is seen ( ratio of 1 ) ; points above the line are consistent with repression , while those below the line are consistent with activation by Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18405378" target="_blank">18405378</a>]<br></li>
<li>The positive regulation of glycine cleavage by Lrp may be rationalized because this coordination is most important when cells are actively synthesizing purines , pyrimidines , and amino acids .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li>
<li>The effects of changing the Lrp concentration in vivo predict two-stage regulation of gltBDF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7901196" target="_blank">7901196</a>]<br></li>
<li>The interplay between Lrp and other NAPs , in particular H-NS , in the regulation of several genes is well documented ( 10 , 25 , 38 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21441513" target="_blank">21441513</a>]<br></li>
<li>The next step in the research is to identify the whole set of regulation targets of Lrp in the presence of each effector ligand .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2162465" target="_blank">2162465</a>]<br></li>
<li>The fact that Lrp regulates malT is surprising , and the physiological rationale for this regulation is not clear .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7751276" target="_blank">7751276</a>]<br></li>
<li>The pattern of regulation of ompC and ompF by Lrp is in general accord with this model .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8002608" target="_blank">8002608</a>]<br></li>
<li>The influence of CRP on the transcriptome progressively increased with increased aerobiosis , whereas the activities of ArcA , FNR , IHF and Lrp decreased as aerobiosis increased .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22870390" target="_blank">22870390</a>]<br></li>
<li>The simplest interpretation of these results is that Lrp acts negatively on the LIV-I and LS transport systems and that transport through these systems is repressed when Lrp is overproduced .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1729203" target="_blank">1729203</a>]<br></li>
<li>The physiological function of modulation by Lrp is not yet apparent , but it seems to be more important when cells are starved for glycerol instead of glucose .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9512707" target="_blank">9512707</a>]<br></li>
<li>The intensities of bands were quantified with either a densitometer or a fl-scope Blot Analyser and increased or decreased were the intensities of those that either plotted against the concentration of Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8508774" target="_blank">8508774</a>]<br></li>
<li>The apparent dissociation constant was taken as the concentration of Lrp that caused a half-maximal change .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8508774" target="_blank">8508774</a>]<br></li>
<li>The vertical lines in Fig. 6 and 7 are drawn at this concentration of Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7901196" target="_blank">7901196</a>]<br></li>
<li>The global regulator Lrp mediated this induction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16098526" target="_blank">16098526</a>]<br></li>
<li>The 2-fold repression by Lrp may not seem signiﬁcant .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20729367" target="_blank">20729367</a>]<br></li>
<li>The total number of Lrp targets increased * 2.3-fold from 130 up to 296 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2162465" target="_blank">2162465</a>]<br></li>
<li>The lrp allele and Lrp in strain RL125 are hereafter referred to as lrp-5 ,6 and Lrp-5 ,6 , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1729203" target="_blank">1729203</a>]<br></li>
<li>Newman , E.B. , Lin , R.T. , and D'Ari , R. ( 1996 ) The Leucine / Lrp regulon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767577" target="_blank">9767577</a>]<br></li>
<li>The concentrations of both Lrp and ArgR were 25 nM .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17233899" target="_blank">17233899</a>]<br></li>
<li>The final yield of homogenous Lrp from 3 liters of induced culture was 38.5 mg .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7901196" target="_blank">7901196</a>]<br></li>
<li>The functional equivalence of Lrp from these related species was assessed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18405378" target="_blank">18405378</a>]<br></li>
<li>The goal of this study was to isolate mutants altered in specific functions of Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8432705" target="_blank">8432705</a>]<br></li>
<li>The specific activity of Lrp was calculated as the product : specific activity [ 3H ] Leu ( dpm/pmol ) 3 23 pmol of Leu/pmol of Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8636076" target="_blank">8636076</a>]<br></li>
<li>The lrp mutant grows more slowly on maltose than its parent , but this is due to general effects of Lrp and not to a speciﬁc action on the maltose regulon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7751276" target="_blank">7751276</a>]<br></li>
<li>The negative control to which aid8 is subject raises the question of how Lrp repressionis bypassedwhen the AidB protein is needed to counteract the effects of alkylating agents .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8809748" target="_blank">8809748</a>]<br></li>
<li>In this report , we show that Lrp does , in fact , bind to a high affinity , consensus-like site located between the attenuator and the first structural gene of the ilvGMEDA operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8900118" target="_blank">8900118</a>]<br></li>
<li>In most cases , the physiological role of Lrp is to activate biosynthetic operons and to repress degradative ones .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8900118" target="_blank">8900118</a>]<br></li>
<li>In general , Lrp appears to stimulate expression of operons that function in biosynthetic pathways and to repress expression of those that function in catabolic pathways .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8636076" target="_blank">8636076</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_1094">
In general , among Lrp-related operons , those which function in biosynthetic pathways are stimulated by Lrp while those involved in catabolic pathways are repressed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955316" target="_blank">8955316</a>]<br></label>
<input id="act_similarity_1094", type="checkbox">
<div class="ACT_similar">
<li>In general , Lrp-related operons functioning in biosynthetic pathways are stimulated by Lrp , whereas those involved in catabolic pathways are repressed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17289841" target="_blank">17289841</a>]<br></li></div>
<li>In strain LP1008 , lacZ was still under the control of Lrp and showed a nearly 30-fold reduction in b-galactosi-dase levels in an Lrp2 background relative to the wild-type strain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11395454" target="_blank">11395454</a>]<br></li>
<li>In this respect , Lrp is unique because its function is considered to be regulated at various levels by not only Leu , but also Ala , His , Ile , Met and Thr ( Hart & Blumenthal , 2011 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2162465" target="_blank">2162465</a>]<br></li>
<li>In contrast to Lrp , AsnC has only been shown to exert speciﬁc control of its own gene and that of asnA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16528101" target="_blank">16528101</a>]<br></li>
<li>In contrast , the three Lrp orthologs gave equivalent activation of PgltB ( Figs. 4C , F ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18405378" target="_blank">18405378</a>]<br></li>
<li>In light of these ﬁndings , an interpretation of the present data is that yojI is induced indirectly by ppGpp via the increase in cellular Lrp levels .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19074390" target="_blank">19074390</a>]<br></li>
<li>In minimal media , Lrp and StpA levels are high .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11601842" target="_blank">11601842</a>]<br></li>
<li>In minimal medium , Lrp might act on those and other genes , repressing degradative genes and activating biosynthesis genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1532173" target="_blank">1532173</a>]<br></li>
<li>When Lrp was ® rst bound to the promoter , the addition of CRP followed by an incubation of 20 minutes had no effect .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9512707" target="_blank">9512707</a>]<br></li>
<li>In contrast , when CRP was ® rst bound , the addition of Lrp followed by a 20 minute incubation displaced CRP from its site , resulting in full occupancy by Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9512707" target="_blank">9512707</a>]<br></li>
<li>We have determined that this is a genuine osmotic induction rather than a salt effect , as it can also be mediated by 0.6 M sucrose , which causes an osmotic shock identical to that caused by 0.4 M NaCl , and that as expected in rich medium , it is independent of the Lrp protein ( 15a ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9006049" target="_blank">9006049</a>]<br></li>
<li>A second regulator , cAMP/CRP , activates P2 , an effect that is largely inhibited by Lrp , such that catabolite repressor protein ( Crp ) and Lrp are rival activators of serA transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11985712" target="_blank">11985712</a>]<br></li>
<li>Binding of Lrp to aid6 in vitro To determine if the negative effect of the Irpgene on aidB transcriptionis mediated by direct Lrp/PaidB interaction , we performedin vffrobinding assays with pure Lrp protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8809748" target="_blank">8809748</a>]<br></li>
<li>Presence of a functional Irp gene , in fact , does not have any detectable effect on MMS induction in vivo ( Fig. 3 ) , nor does Lrp protein inhibit meAdaactivation of aid5 transcription in vitro ( Fig. 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8809748" target="_blank">8809748</a>]<br></li>
<li>This degree of sensitivity to leucine defines a unique effective intracellular Lrp concentration of 5.5 nM under these growth conditions , assuming that our mobility shift assays reflect the effect of Lrp on the transcription of gltBDF : : lacZ fusions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7901196" target="_blank">7901196</a>]<br></li>
<li>This effect was not seen in glucose-grown cells , but the total transcription was low even in the presence of Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7751276" target="_blank">7751276</a>]<br></li>
<li>If the LB effect was a result of the lowered Lrp concentration , introducing the lrp gene on a multicopy plasmid should increase the Lrp concentration and thus increase transcription from the serA gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11985712" target="_blank">11985712</a>]<br></li>
<li>When lrp + and lrp : : TnJO strains are grown in Luria broth , the effect of Lrp on the level and adenylylation state of glutamate synthase is almost nonexistent .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1346534" target="_blank">1346534</a>]<br></li>
<li>In summary , it is probable , although not yet proven , that the effect of Lrp on glutamine synthetase levels is entirely due to the effect of Lrp on gltBDF transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li>
<li>In class 1 mutants , the effect of the absence of Lrp on gen a Results are expressed as described in footnote a of Table 2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1532173" target="_blank">1532173</a>]<br></li>
<li>In some cases , plasmid pCV180 was transferred to strains so that the effect of elevated Lrp levels could be assessed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8144448" target="_blank">8144448</a>]<br></li>
<li>In contrast , it is not known how HapR counteracts the stimulatory effect of upstream bound Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19906180" target="_blank">19906180</a>]<br></li>
<li>As judged by the effect of an insertion in the lip gene , Lrp activates synthesis from serA , ilvIH , and gcv .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1532173" target="_blank">1532173</a>]<br></li>
<li>Regulatory interactions at Lrp target genes Action Prototype gene Lrp activates Leucine reduces effect ... .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1532173" target="_blank">1532173</a>]<br></li>
<li>Another example is the effect of impaired protein synthesis by uncharged tRNA , which increases ppGpp and Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12730324" target="_blank">12730324</a>]<br></li>
<li>Of the three positive regulators , Crp , MalT , and Lrp , the effect of Lrp is by far the weakest and may not be direct .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7751276" target="_blank">7751276</a>]<br></li>
<li>Thus , the effect of Lrp and YgaZH is more than additive , and the positive effect of Lrp seems to be further enhanced when accumulated L-valine is more rapidly exported by the overproduced YgaZH .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17463081" target="_blank">17463081</a>]<br></li>
<li>Leucine may inhibit the Lrp effect by removing Lrp from the DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1917830" target="_blank">1917830</a>]<br></li>
<li>c - + , positive effect of Lrp ; negative effect of Lrp ; 0 , no apparent effect of Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1346534" target="_blank">1346534</a>]<br></li>
<li>0 , no effect of Lrp was noted .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1346534" target="_blank">1346534</a>]<br></li>
<li>gcv Lrp represses Leucine reduces effect ... .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1532173" target="_blank">1532173</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_1126">
ilvIH Leucine is needed for Lrp effect ... .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1532173" target="_blank">1532173</a>]<br></label>
<input id="act_similarity_1126", type="checkbox">
<div class="ACT_similar">
<li>Leucine is needed for Lrp effect ... .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1532173" target="_blank">1532173</a>]<br></li></div>
<li>D035 .8 D038 .3 RpoS B015 .5 C013 .5 OxyR Cd , QN , HP , uspA PS , NS , Ile , Cd , QN , HP , DNP , 42 NS DNP fumC glpK dnaN ompC DNP , 42 RpoS OmpR , Lrp Lrp , RpoS FIG. 4 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7559363" target="_blank">7559363</a>]<br></li>
<li>This activation is brought about in an RpoS-depen-dent manner in concert with a combination of one or more global regulators such as Lrp , H-NS , and IHF ( 11 , 14 , 37 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12081976" target="_blank">12081976</a>]<br></li>
<li>This differentiates it from tlie many stereotypically regulated geiies controlled by IHF , Lrp , H-NS aiid RpoS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9108285" target="_blank">9108285</a>]<br></li>
<li>Leu was not used alone as it can lead to starvation for Ile , via feedback inhibition of L-threonine deaminase FRe l igguraetio4n of selected target genes by heterologous Lrp proteins Regulation of selected target genes by heterologous Lrp proteins.E .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18405378" target="_blank">18405378</a>]<br></li>
<li>Further analysis of these dominant negative mutations and proteins should greatly facilitate the structural and functional study of Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7603430" target="_blank">7603430</a>]<br></li>
<li>if the typical significantly-affected gene varies twofold , but this change is only counted as being significant in half of the genes that truly change twofold , then 1/4 of genes would appear to be unaffected by a Lrp ortholog , even if there were no real functional differences between the Lrp proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18405378" target="_blank">18405378</a>]<br></li>
<li>Finally , evidence pieced together from different sources suggests that Lrp and Ntr regulators might independently activate ompF , oppABCDF , ydcSTUVW , and yeaGH ( 68 , 90 , 100 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12730324" target="_blank">12730324</a>]<br></li>
<li>In the majority of these six cases , including Lrp ( rank 1/4 15 ) ( Figure S3B ) , this was due to the presence of combinatorial or conditional regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17214507" target="_blank">17214507</a>]<br></li>
<li>In the case of LysU , Lrp regulation be appears to direct and not through signal transduction pathways mediating the heat-shock response ( 19 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1346534" target="_blank">1346534</a>]<br></li>
<li>Furthermore , Lrp , via its potent regulation of gltBDF and control of intracellular glutamine concentration , plays a significant role in the regulation of nitrogen assimilation genes ( 30 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17289841" target="_blank">17289841</a>]<br></li>
<li>This model suggests that regulation of the concentration of active Lrp is critical to control of the Lrp regulon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7901196" target="_blank">7901196</a>]<br></li>
<li>Thus Lrp appears to affect the level of glutamine synthetase by altering the regulation of kinase activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1346534" target="_blank">1346534</a>]<br></li>
<li>We are currently examining possible interactions between Lrp and CRP as part of an analysis of the mechanism of Lrp regulation of the major dad promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9648752" target="_blank">9648752</a>]<br></li>
<li>Alanine may also contribute to this regulation because alanine also binds Lrp [ ( 59 ) and references cited therein ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12730324" target="_blank">12730324</a>]<br></li>
<li>We then addressed the question whether Lrp is also involved in osmotic regulation of osmY .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8253679" target="_blank">8253679</a>]<br></li>
<li>We show that H-NS is involved in the transcriptional regulation of the kbl/tdh operon , which is also Lrp regulated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7748928" target="_blank">7748928</a>]<br></li>
<li>Further study of these Lrp mutants should prove useful in understanding the regulation of the ilvIH operon and other operons controlled by Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8432705" target="_blank">8432705</a>]<br></li>
<li>This pattern of regulation is unusual for operons that are controlled by Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1729203" target="_blank">1729203</a>]<br></li>
<li>Regulatory roles of Lrp After SELEX-chip screening , at least 296 regulation targets were identiﬁed for Lrp , resulting in an increase of * 2.3-fold .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2162465" target="_blank">2162465</a>]<br></li>
<li>Also , Lrp , RpoS , and OxyR have been reported previously ( 8 , 21 , 28 ) to be involved in the regulation of some Gsps reported here to be IHF dependent ( Table 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7559363" target="_blank">7559363</a>]<br></li>
<li>It should be noted , however , that the selectivity of regulation targets by Lrp should be altered after interaction of an effector ligand .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2162465" target="_blank">2162465</a>]<br></li>
<li>Hence , regulation by Lrp intersects with several channels of regulation , and H-NS participates , at least in part , in this intersection .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7603430" target="_blank">7603430</a>]<br></li>
<li>However , it remains to be clarified whether regulation by Lrp always involves H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7603430" target="_blank">7603430</a>]<br></li>
<li>Sites of regulation by Lrp and CRP-cAMP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9171388" target="_blank">9171388</a>]<br></li>
<li>A discussion of the functio of Lrp and Lrp-dependent regulation , which does not require 54 , is beyond the scope of the review .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11528004" target="_blank">11528004</a>]<br></li>
<li>Relationship between S and Lrp in global regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15716429" target="_blank">15716429</a>]<br></li>
<li>Lrp , a global regulator involved in regulation of ... .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11065369" target="_blank">11065369</a>]<br></li>
<li>Lrp is involved in the regulation of osmY .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8253679" target="_blank">8253679</a>]<br></li>
<li>Conﬁrmation of regulation of leuDBCA , livHJKG , and sdaC by Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7751276" target="_blank">7751276</a>]<br></li>
<li>Concerted regulation by RpoS and Lrp has already been reported for a number of genes ( osmC , osmY , aidB , and csiD [ 13 , 48 , 50 , 59 ] ) , suggesting tight integration of Lrp - and RpoS-dependent regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15489429" target="_blank">15489429</a>]<br></li>
<li>By studying regulation of lacZ insertions in target genes of Lrp , we showed that binding of Lrp can either activate or repress these genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1532173" target="_blank">1532173</a>]<br></li>
<li>A group of regulation target genes are , however , activated by Lrp independent of Leu .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2162465" target="_blank">2162465</a>]<br></li>
<li>In contrast with genetic methods previously used to identify genes in the Lrp regulon , this method does not require any knowledge of the phenotypic consequences of regulation by Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1346534" target="_blank">1346534</a>]<br></li>
<li>In this study , an attempt was made to identify the regulation target genes that are recognized by Lrp alone in the absence of any effectors .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2162465" target="_blank">2162465</a>]<br></li>
<li>'' Lane 1 , no Lrp ; lanes 2 -- 5 : 1.6 nM ; 8.0 nM ; 16 nM ; 80 nM Lrp , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8900118" target="_blank">8900118</a>]<br></li>
<li>Lanes : 1 , no Lrp ; 2 to 10 , 0.27 , 0.54 , 1.1 , 2.2 , 4.3 , 8.6 , 17.2 , 34.5 , and 69 nM Lrp dimer , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7928983" target="_blank">7928983</a>]<br></li>
<li>Lanes : 1 , no Lrp ; 2 to 10 , 55.2 , 27.6 , 13.8 , 6.9 , 3.45 , 1.72 , 0.86 , 0.43 , and 0.22 nM Lrp , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7928983" target="_blank">7928983</a>]<br></li>
<li>Lrp ( 25 ng ) was added to lanes 2 , 5 , 8 , 11 , 14 and 17 ; 250 ng to lanes 3 , 6 , 9 , 12 , 15 and 18 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11985712" target="_blank">11985712</a>]<br></li>
<li>Lanes contained the following Lrp concentrations : 1 and 12 , no Lrp ; 2 , 0.0625 nM ; 3 , 0.125 nM ; 4 , 0.25 nM ; 5 , 0.5 nM ; 6 , 1 nM ; 7 , 2 nM ; 8 , 3 nM ; 9 , 4 nM ; 10 , 5 nM ; 11 , 10 nM Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7901196" target="_blank">7901196</a>]<br></li>
<li>Lrp concentrations were as follows : lanes 1 and 9 , no Lrp dimer ; lanes 2 and 10 , 25 nM Lrp dimer ; lanes 3 and 11 , 38 nM Lrp dimer ; lanes 4 and 12 , 44 nM Lrp dimer ; lanes 5 and 13 , 50 nM Lrp dimer ; lanes 6 and 14 , 63 nM Lrp dimer ; lanes 7 and 15 , 69 nM Lrp dimer ; and lanes 8 and 16 , 75 nM Lrp dimer .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8002608" target="_blank">8002608</a>]<br></li>
<li>As the Lrp dimer concentration was increased from 0.22 to 27.6 nM , Lrp bound to and protected sequences spanning about a 137-bp region from position -92 to position -229 ( Fig. 4 , bars to the right of the gels ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7928983" target="_blank">7928983</a>]<br></li>
<li>When present , the concentration of RNAP was 30 nM , Lrp 100 nM and IHF 100 nM .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17233899" target="_blank">17233899</a>]<br></li>
<li>Again , site 1 appears before site 2 as the Lrp concentration is increased .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8900118" target="_blank">8900118</a>]<br></li>
<li>As in panel B of Figure 2 except that the Lrp concentration in the reaction was 16 nM .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8508774" target="_blank">8508774</a>]<br></li>
<li>As the Lrp concentration was increased , a greater proportion of complexes 2 and 3 were formed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8900118" target="_blank">8900118</a>]<br></li>
<li>As the concentration of Lrp was increased , band A remained in small amounts , while band B increased to become the dominant species at 2 nM Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7928983" target="_blank">7928983</a>]<br></li>
<li>Ten cycles of the Selex procedure were required to select sequences that bound strongly to Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7665463" target="_blank">7665463</a>]<br></li>
<li>As described in Materials and Methods , the conditions for this Selex experiment were not optimal and 10 cycles were required for selection of sequences that bound Lrp tightly .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7665463" target="_blank">7665463</a>]<br></li>
<li>Since PargO is known to be stimulated by ArgP , we determined whether or not the argP promoter activity ( PargP ) is affected by Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19906180" target="_blank">19906180</a>]<br></li>
<li>It is particularly interesting that , unique to the V. cholerae Plrp in the V. cholerae background , Lrp activated rather than repressing the promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18405378" target="_blank">18405378</a>]<br></li>
<li>This conclusion is supported by results with lysogen GS1003XTlac ( A-466 +8 C +10 A ) , where both Lrp and GcvA are nonfunctional and no significant purine-mediated repression is observed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7928983" target="_blank">7928983</a>]<br></li>
<li>They achieved tenfold repression of Irp under conditions where Lrp is overproduced eightfold from an Lrp-over-producing plasmid .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7603430" target="_blank">7603430</a>]<br></li>
<li>This repression is dependent on Lrp and is partially dependent on GcvR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9802028" target="_blank">9802028</a>]<br></li>
<li>Activation and repression in the + 1 and +21 , , , , lysogens are dependent on GcvA , Lrp and GcvR Table2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9802028" target="_blank">9802028</a>]<br></li>
<li>Approximately half of the 140 core S-dependent genes identiﬁed here are also under repression by Lrp ( 64 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15716429" target="_blank">15716429</a>]<br></li>
<li>For almost half of these genes , this correlates with repression by Lrp ( 64 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15716429" target="_blank">15716429</a>]<br></li>
<li>As shown in Table 2 , expression of the fusion was almost four times lower in the absence of Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19074390" target="_blank">19074390</a>]<br></li>
<li>As shown above , when active Lrp is reduce Table 3 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8152423" target="_blank">8152423</a>]<br></li>
<li>Under the conditions used here , about 70 % of the pap GATC sites were methylated in the absence of Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9811649" target="_blank">9811649</a>]<br></li>
<li>Primer extension assays showed that P1 induction by hydrogen peroxide occurs in the absence of Lrp , IHF and H-NS ( Fig. 4A and B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10760142" target="_blank">10760142</a>]<br></li>
<li>Some 30 proteins were clearly affected up or down by the absence of Lrp ( 61 , 66 , 172 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16760308" target="_blank">16760308</a>]<br></li>
<li>Lrp was found to-bind tion was observed in the absence of cAMP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9512707" target="_blank">9512707</a>]<br></li>
<li>Osmotic induction of osmY in the absence of as and Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8253679" target="_blank">8253679</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8253679" target="_blank">8253679</a>]<br></li>
<li>Lrp is not synthesized when cells are grown in broth ( 19 ) , which predicts that alanine will not repress for cells grown in broth .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20729367" target="_blank">20729367</a>]<br></li>
<li>Moreover , under certain conditions , Lrp acts as a positive modulator , e.g. when cells grow in glycerol , a non-optimal carbon source .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9512707" target="_blank">9512707</a>]<br></li>
<li>However , an alternate explanation is that such starvation results in accumulation of guanosine tetraphosphate , which induces Lrp ( 19 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20729367" target="_blank">20729367</a>]<br></li>
<li>Our results linking H-NS to Lrp lend further support to the notion that both regulators are involved in responses to stress conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8152423" target="_blank">8152423</a>]<br></li>
<li>According to the results described here , Lrp participates in the activation of cadBA under inducing conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21441513" target="_blank">21441513</a>]<br></li>
<li>It is clear , therefore , that Lrp is a major physiological effector of synthesis of glycine cleavage enzymes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1532173" target="_blank">1532173</a>]<br></li>
<li>Lrp positively controls the synthesis of GOGAT , which affects the level of glutamine , the most important effector of the Ntr response .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12730324" target="_blank">12730324</a>]<br></li>
<li>This includes DnaA , FarR , GlpR , MalT , Lrp , MetR , NarL , TyrR , TorR and TreR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9813115" target="_blank">9813115</a>]<br></li>
<li>This is most likely Pot-box 3 , which is located outside of the ArgP footprint and which shows the highest affinity for Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19906180" target="_blank">19906180</a>]<br></li>
<li>This scenario is plausible , because ArgP and Lrp competitively activate argO ( 35 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21441513" target="_blank">21441513</a>]<br></li>
<li>Expression of livJ ' - lacZ or livK ' - lacZ operon fusions in strains overexpressing Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1729203" target="_blank">1729203</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_1202">
Reactions were started by addition of Lrp to the reaction mixture and incubated for 15 min .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7916011" target="_blank">7916011</a>]<br></label>
<input id="act_similarity_1202", type="checkbox">
<div class="ACT_similar">
<li>Reactions were initiated by the addition of Lrp to the reaction mixture and were incubated for 15 min at room temperature .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9515701" target="_blank">9515701</a>]<br></li></div>
<li>Lanes 1 and 5 , RNA polymerase only ; lanes 2 and 6 , 10pmol of methylated Ada ; lanes 3 and 7 , lopmol of Lrp ; lanes 4 and 8 , 10pmol of methylated Ada and 10pmol of Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8809748" target="_blank">8809748</a>]<br></li>
<li>When required , 10pmol of Lrp , unmethylated Ada , or methylatedAda were added and the mixture was incubated for 15min at 37 ° C .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8809748" target="_blank">8809748</a>]<br></li>
<li>Lrp positively regulates YojI , an inner-membrane MccJ25 efﬂux pump .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19074390" target="_blank">19074390</a>]<br></li>
<li>Lrp positively regulates genes that function during famine and negatively regulates genes that function during a feast .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li>
<li>models that might explain this result is the possibility that Lrp is regulating an intermediate effector that in turn regulates serA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1917830" target="_blank">1917830</a>]<br></li>
<li>Lrp regulates glutamine synthetase levels indirectly .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1346534" target="_blank">1346534</a>]<br></li>
<li>After single - GcvA and Lrp bind co-operatively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9802028" target="_blank">9802028</a>]<br></li>
<li>Fragments that bind Lrp protein with high afﬁnity are labelled * ; those with low afﬁnity ** .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11985712" target="_blank">11985712</a>]<br></li>
<li>Individual colonies that either were nonmucoid or appeared light blue , phenotypes indicating a decrease in Lrp activity , were chosen .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8432705" target="_blank">8432705</a>]<br></li>
<li>Lrp acts as a sensor of this key signal , leading to modulation of its activity and speci-ﬁcity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2162465" target="_blank">2162465</a>]<br></li>
<li>There is experimental evidence that Lrp functions as a transcriptional-regulator for glyA , metK , and lysU .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1917830" target="_blank">1917830</a>]<br></li>
<li>We suggest that Lrp is similar to RAP-1 and ABF-1 ( 4 , 7 ) as a general regulator , in spite of its drastic effects specific on areas .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1532173" target="_blank">1532173</a>]<br></li>
<li>More general effects of Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1532173" target="_blank">1532173</a>]<br></li>
<li>S1 ; see below ) , was detected with an equivalent reporter fusion , indicating that the effects of Lrp on PargO are speciﬁc .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19906180" target="_blank">19906180</a>]<br></li>
<li>One unique characteristic of Lrp is its functional modulation after interaction with multiple effectors .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2162465" target="_blank">2162465</a>]<br></li>
<li>Lrp binds to multiple ... , ... , .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9802028" target="_blank">9802028</a>]<br></li>
<li>4-i ` i7 2852-1 RESULTS Lrp + + Lrp binds to the fim switch in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7916011" target="_blank">7916011</a>]<br></li>
<li>Expression of an ompF * - lacZ1 translational fusion is increased by Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8002608" target="_blank">8002608</a>]<br></li>
<li>Expression of sduC , but not sdaB , is also induced 2-3-fold by Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8026499" target="_blank">8026499</a>]<br></li>
<li>Results from this study demonstrate that Lrp interferes negatively with the action of ArgP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19906180" target="_blank">19906180</a>]<br></li>
<li>F , free Pa , dBDNA ; CL , Lrp / complex ; CA , meAda/Pa , dBcomplex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8809748" target="_blank">8809748</a>]<br></li>
<li>Leucine sensitivity reflects the affinity of Lrp for target promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7901196" target="_blank">7901196</a>]<br></li>
<li>Leucine is known only as an effector of Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8195106" target="_blank">8195106</a>]<br></li>
<li>Is Leucine an Lrp Antagonist at Physiologically Significant Concentrations ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li>
<li>( OP ) 2Cu2 + footprinting of Lrp bound to PCR6 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7916011" target="_blank">7916011</a>]<br></li>
<li>Genes activated by Lrp are underlined .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8900118" target="_blank">8900118</a>]<br></li>
<li>Use of this buffer minimized adsorption of Lrp to the ® lter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11575919" target="_blank">11575919</a>]<br></li>
<li>Puriﬁcation of Lrp and ArgP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21441513" target="_blank">21441513</a>]<br></li>
<li>IHF bound to the phage ; II1A site Downloaded from www.microbiologyresearch.org by IP : 132.248.34.24 On : Fri , 05 Aug 2016 22:42:4 Lrp-binding analysis Role of Lrp in gcvT : : lacZexpression in the IHF-C , IHF-D and IHF-G mutants We used a gel mobility-shift assay to determine whether Lrp could still bind to the gcv control region carrying the Since all but the upstream 22 bp of the Lrp-binding insertion sequences , and this binding could be responregion was replaced in the IHF lysogens , we predicted sible for the negative effect of Lrp on g c v T : : l a c Z that Lrp would no longer be involved in regulation of regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10217490" target="_blank">10217490</a>]<br></li>
<li>For example , six Lrp-binding sites exist upstream of the ilvIH operon promoter ,4,5 and Lrp binding to these sites is highly cooperative , leading to the formation of a large nucleoprotein complex .5 At nM concentrations ( 1-50 nM ; unless otherwise stated , concentration is in terms of the monomer ) , the stoichiometry of Lrp binding to DNA containing a single site is one dimer per site and the consensus for binding is aAGbAdATTdTgCTy , where a C or T , b not G , d A or T , g not C , y A or G. 6,7 Leucine has two effects upon binding in vitro : the af ® nity of Lrp is reduced but the cooperativity of binding to multiple sites is increased .6 ± 8 Lrp is moderately abundant in E. coli ; 9,10 for cells grown in glucose Mops minimal medium , the concentration is about 15 mM , assuming a total protein per E. coli cell of 0.156 pg and a cellular volume of 6.7 10ÿ13 ml .11 Applying a correction for cytoplasmic volume ,12 the estimated intracellular concentration is 17 mM for cells grown in glucose Mops minimal medium ( osmolarity , 0.19 OsM ) and 27 mM for the same medium supplemented with 0.5 M NaCl ( osmolarity , 1.0 OsM ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11575919" target="_blank">11575919</a>]<br></li>
<li>Half-maximal effects were concentration of 3 10-9 M , which is observed at an Lrp x about the same as the dissociation constant for Lrp bound to sites 1 and 2 , measured in gel retardation experiments ( Wang and Calvo , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8508774" target="_blank">8508774</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_1235">
It is this second complex , containing Lrp dimers bound at six sites , that is required for transcriptional activation ( 29 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7901196" target="_blank">7901196</a>]<br></label>
<input id="act_similarity_1235", type="checkbox">
<div class="ACT_similar">
<li>It is this second complex , containing Lrp dimers bound at six sites , that is required for the transcriptional activation of ilvIH transcription ( 29 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7901196" target="_blank">7901196</a>]<br></li></div>
<li>Binding of Lrp to these sites is both necessary and sufficient for transcriptional activation of the ilvIH operon ( 31 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8144448" target="_blank">8144448</a>]<br></li>
<li>Binding of Lrp dimers to all six sites is required for activation of the ilvIH promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7928983" target="_blank">7928983</a>]<br></li>
<li>Since transcriptional activation is associated with the occupancy of the lower-affinity sites ( 29 ) , we have plotted the percent DNA bound in the second complex , in which Lrp is bound to both low - and high-affinity sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7901196" target="_blank">7901196</a>]<br></li>
<li>Bending of the DNA by Lrp and GcvA allows GcvA bound to sites 32 to interact with the α-carboxy-terminal domain of RNA polymerase , permitting stimulation of RNAP , and activation of transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12101307" target="_blank">12101307</a>]<br></li>
<li>Here , we found that Lrp binds to several sites at the lysP control region and is able to potentiate the transcriptional activation mediated by ArgP when lysine becomes limiting .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21441513" target="_blank">21441513</a>]<br></li>
<li>Lrp also appears to be a canonical transcriptional-regulator , although it binds nearly 140 sites in the genome ( 11 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21890697" target="_blank">21890697</a>]<br></li>
<li>In vitro , Lrp binds specifically and wifh relatively high affinity to six sites upsfream of fhe iivlH promoter -LCB- Ricca et ai , 1989 ; Wang and Calvo , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8483418" target="_blank">8483418</a>]<br></li>
<li>Lrp acted as a repressor in this system when bound to sites close to or within the promoter and as an activator when bound to sites further upstream ( Nou et al. , 1995 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10216857" target="_blank">10216857</a>]<br></li>
<li>The upstream P2 promoter , overlapping the Lrp binding sites , was repressed up to 10-fold by Lrp , and the downstream P1 promoter was activated up to 10-fold by Lrp ( Yang et al. , 2002 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12123449" target="_blank">12123449</a>]<br></li>
<li>Binding of Lrp to sites 1 and 2 , centered at -250 and -219 , respectively , was postulated to increase the effectiveness of site 5 through formation of a large nucleoprotein complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li>
<li>However , when Lrp dimers are bound to two sites , bending angles of about 81 ' and 89 '' were calculated for the XhoI and BamHI fragments relative to the MluI fragment .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9802028" target="_blank">9802028</a>]<br></li>
<li>Lrp induced a bend of 520 uponbinding to a single binding site , and the angle of bending is increased to at least 135 ° when Lrp binds to two adjacent sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8508774" target="_blank">8508774</a>]<br></li>
<li>By using vectors described by Kim et al. to generate circularly permuted fragments ( 73 ) , Lrp induced a bend of about 52 ° upon binding to a single site , and the angle of bending was increased to at least 1350 when Lrp bound to two adjacent sites ( 160 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li>
<li>Whereas Lrp bound to the site containing GATC ' 130 whether or not this sequence was methylated , the DNase I footprint for Lrp binding to GATC1130 was clearly different for the methylated and nonmethylated sites ( 107 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li>
<li>Wang and Calvo ( 159 ) argued that Lrp bound to site 5 probably activated transcription and that cooperative binding to sites 3 and 4 helped to deliver Lrp to site 5 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li>
<li>PCR fragments LP-2 and LP-7 , but not fragment LP-3 or LP-6 , bound purified Lrp , localizing binding sites to the region -88 to -12 ( Fig. 4A and C ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8144448" target="_blank">8144448</a>]<br></li>
<li>In view of this model , it seems most likely that Lrp binding to sites 1 and 2 alone produces a structure that is more favourable for recombination than that produced by Lrp bound to all three sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9515701" target="_blank">9515701</a>]<br></li>
<li>If cooperative interactions between Lrp bound at upstream and downstream sites are required for promoter activity ( and not just correlated with it ) , then this would provide some support for the idea that hexadecamers but not octamers activate the ilvIH promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12218014" target="_blank">12218014</a>]<br></li>
<li>Of course , the correlation between cooperative interactions between Lrp bound at upstream and downstream sites and promoter activity may have no signiﬁcance , and a conformer related to the one on the left of Fig. 4 may be active in transcription initiation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12218014" target="_blank">12218014</a>]<br></li>
<li>In this model , Lrp bound to upstream sites directly stimulates transcription , whereas Lrp bound to downstream sites is necessary to bend the DNA so that the upstream Lrp is positioned properly with respect to the promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12218014" target="_blank">12218014</a>]<br></li>
<li>Another likely exception is gltBDF , in which Lrp bound at sites upstream of 128 stimulates transcription from a downstream promoter ( 15 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12218014" target="_blank">12218014</a>]<br></li>
<li>Binding of Lrp to upstream sites 1 and 2 seemed to enhance promoter expression but was not absolutely required ( Q. Wang and J. M. Calvo , J. Mol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12218014" target="_blank">12218014</a>]<br></li>
<li>In this model , Lrp bound at upstream sites was not essential for promoter expression but contributed to more efﬁcient expression , perhaps by interacting with Lrp bound at downstream sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12218014" target="_blank">12218014</a>]<br></li>
<li>In a model for activation of ilvIH expression , Wang and Calvo postulated that Lrp bound to downstream sites 3 , 4 , and 5 was required for RNA polymerase action at the promoter and that binding of Lrp to sites further upstream improved but was not absolutely required for promoter function ( 20 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12218014" target="_blank">12218014</a>]<br></li>
<li>In vitro , Lrp bound to multiple sites in the dad promoter region and mutagenesis showed that this binding was necessary for the negative effect of Lrp on dad expression in vivo and in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955407" target="_blank">8955407</a>]<br></li>
<li>Insertions of 4 or more base pairs resulted in a further decrease in ilvIH expression , and for the 6 and 8 constructs , the promoter activity was as low as that for a construct lacking all six Lrp binding sites ( Fig. 1 , compare lines 1 , 2 , 6 , 7 , 8 , 9 , and 10 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12218014" target="_blank">12218014</a>]<br></li>
<li>Thus , Dam could affect fimB expression by inhibiting NanR and NagC binding to their cognate sites as it can do for Lrp and OxyR ( 49 , 51 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15534208" target="_blank">15534208</a>]<br></li>
<li>Lrp binds to six sites upstream of iivlH , and Lrp binding is required for / / WH expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8483418" target="_blank">8483418</a>]<br></li>
<li>lrp -- Binding of Lrp to the serA promoter : repression by Lrp The action of Lrp in repressing the transcription of P2 is much simpler to understand as the binding site for Lrp covers the P2 -- 10 and -- 35 regions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11985712" target="_blank">11985712</a>]<br></li>
<li>The activation of ilvIH transcription probably requires the binding of Lrp to the site centered at position -72 , at which Lrp can presumably contact RNAP bound at promoter P1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1447135" target="_blank">1447135</a>]<br></li>
<li>The effect of mutations in these regions on ilvIH promoter expression in vivo led to the proposal that activation of transcription required Lrp binding to downstream sites 3 , 4 , 5 , and 6 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12218014" target="_blank">12218014</a>]<br></li>
<li>Similar footprinting data led Wang and Calvo to propose the invol-vement of half-site interactions for the binding of Lrp to ilvIH ( Wang & Calvo , 1993a ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9236118" target="_blank">9236118</a>]<br></li>
<li>Alternatively , as suggested for the effect of Lrp in ilvIH ( Wang and Calvo , 1993 ) , positive co-operative binding of Lrp among upstream sites and a site close to the promoter could allow direct interaction between Lrp and polymerase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10216857" target="_blank">10216857</a>]<br></li>
<li>ArgR also binds to the gltBDF promoter region , upstream of the Lrp binding sites , and decreases transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17233899" target="_blank">17233899</a>]<br></li>
<li>We show that Lrp binds specifically to a site or sites upstream of the gltBDF operon and compare this binding to the binding of Lrp to the ilvIH promoter region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7901196" target="_blank">7901196</a>]<br></li>
<li>Lrp binding is necessary ( Wang and Calvo , 1993 ) and sufficient ( Willins and Calvo , 1992 ) for the activation of transcription from the ilvIH It will be important to establish whether bending promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8508774" target="_blank">8508774</a>]<br></li>
<li>Lrp regulates ilvIH transcription by binding to DNA in a sequence-specific manner at six sites within the ilvIH promoter-control region and induces bending ( Wang and Calvo 1993b ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8152423" target="_blank">8152423</a>]<br></li>
<li>By studying ilvIH , one of the operons regulated by Lrp , we have shown that Lrp binds directly to multiple sites over the ilvIH promoter/regulatory region , and that the binding of Lrp to DNA is required for the transcription of the operon ( Wang and Calvo , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8508774" target="_blank">8508774</a>]<br></li>
<li>Both positive and negative supercoiling increase the aYnity of Lrp for DNA , presumably because bridging and wrapping is promoted by the presence of loops ( Beloin et al. , 2003 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16879983" target="_blank">16879983</a>]<br></li>
<li>Falconi et al. ( 1991 ) suggested that H-NS potentiates the activity of transcriptional-repressors , Our results are in accordance with these hypotheses and suggest that H-NS binding might stabilise or mould bent DNA and thereby reduce the ability of Lrp to activate R N A polymerase , perhaps by using the elastic energy stored in the induced bend , as suggested by Wang and Calvo ( 1993b ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8152423" target="_blank">8152423</a>]<br></li>
<li>Furthermore , the amino acid appears to increase co-operativity between Lrp dimers bound to DNA ( Ernsting et al. , 1993 ; Wang and Calvo , 1993b ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9515701" target="_blank">9515701</a>]<br></li>
<li>For example , binding of Lrp to the isolated strong binding site 2 of the ilvIH control region occurred in the nM range ( Cui et al. , 1996 ) but , in contrast , binding to site 2 of the pap operon could not be detected by EMSA , although this site was successfully used for the growth of co-crystals ( de los Rios and Perona , 2007 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19906180" target="_blank">19906180</a>]<br></li>
<li>Our results are consistent with aspects of the model previously proposed ( Ernsting et al. , 1993 ) : speci ® cally , mutations that reduce the Kd for Lrp binding to the gltBDF promoter lead to reduced transcriptional activation in vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9236118" target="_blank">9236118</a>]<br></li>
<li>It is interesting that , in both systems , when negative co-operativity was absent because of mutation , the ability of modi ® ers , i.e. alanine , to reduce Lrp binding to negative sites was decreased ( Fig. 6D ; Nou et al. , 1995 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10216857" target="_blank">10216857</a>]<br></li>
<li>Whereas Lrp binding to sites 1 and 2 stimulates switching ( Gally et al. , 1994 ) , it was hypothesized that the interaction of Lrp with sites 1 , 2 and the third region inhibits recombination .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9515701" target="_blank">9515701</a>]<br></li>
<li>Activation required the pre-sence of PapI , a protein that binds to and alters the DNA binding of Lrp ( Nou et al. , 1995 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10216857" target="_blank">10216857</a>]<br></li>
<li>Dam-methylation can in ¯ uence Lrp binding ( Huisman & de Graaf , 1995 ; Nou et al. , 1993 ) , whereas CRP binding is hardly affected by methylation ( Gunasekara et al. , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9512707" target="_blank">9512707</a>]<br></li>
<li>However , unlike Lrp and OxyR binding to their operator sequences at pap and ag43 respectively ( Nou et al. , 1993 ; Haagmans and van der Woude , 2000 ) , methylation does not seem to affect the binding of NagC to O .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17238917" target="_blank">17238917</a>]<br></li>
<li>Leucine alters Lrp binding to the ﬁm switch in vitro The addition of exogenous leucine to minimal media stimulated switching by as much as 15-fold , and this stimulation required lrp ( Gally et al. , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9515701" target="_blank">9515701</a>]<br></li>
<li>Loss of Lrp binding to site 3 stimulates the ﬁm switch in vivo and diminishes leucine stimulation of recombination To determine if binding to region three represses the ﬁm recombination , the ﬁmS 21 mutation was transferred into the chromosome at ﬁm via allelic exchange ( Blomﬁeld et al. , 1991 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9515701" target="_blank">9515701</a>]<br></li>
<li>This corresponds to previous observations in which leucine was shown to exert several effects on the Lrp protein and its binding to DNA : ( i ) it promotes the dissociation of a hexa-decamer to leucine-bound octamers , ( ii ) it increases the binding to non-speciﬁc targets , ( iii ) it decreases the intrinsic affinity of Lrp for binding to a single site , and ( iv ) it increases the cooperativity with which dimers bind to adjacent sites ( reviewed in Chen et al. , 2005 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19906180" target="_blank">19906180</a>]<br></li>
<li>As has been found in other Lrp-regulated operons ( Calvo and Matthews , 1994 ; Newman and Lin , 1995 ) , Lrp bound to an extensive area in the dad promoter region , and alanine/leucine reduced , but did not eliminate , this binding ( Mathew et al. , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10216857" target="_blank">10216857</a>]<br></li>
<li>This possibility was examined in vitro ( Ernsting et al. , 1993 ; Wang & Calvo , 1993a ) and in vivo ( Borst et al. , 1996 ) , and revealed signi ® cant effects of leucine on the af ® nity and cooperativity of Lrp binding to target genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9236118" target="_blank">9236118</a>]<br></li>
<li>In the systems tested , leucine decreased Lrp binding to DNA ( Ricca et al. , 1989 ; Willins et al. , 1991 ; Lin et al. , 1992 ; Ernsting et al. , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9515701" target="_blank">9515701</a>]<br></li>
<li>Lrp binding in promoter regions appears to be co-operative ( Calvo and Matthews , 1994 ) , and it has been reported that leucine increases the positive co-operativity between Lrp binding sites ( Ernsting et al. , 1993 ; Wang and Calvo , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10216857" target="_blank">10216857</a>]<br></li>
<li>Analysis of the effects of leucine on Lrp binding in vitro reveal that , whereas the amino acid decreases the binding afﬁnity of Lrp for its cognate DNA binding sites , it apparently increases co-operativity between Lrp dimers ( Ernsting et al. , 1993 ; Wang and Calvo , 1993b ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9515701" target="_blank">9515701</a>]<br></li>
<li>Nucleotides identical to the CRP binding site consensus are indicated by vertical lines In vitro analysis of binding sites for cAMP-CRP and Lrp in the csiD promoter region In gel retardation assays , puri ® ed CRP protein ( in the presence of cAMP ) was found to reduce mobility of a digoxigenin-labeled PCR fragment containing 475 nucleotides upstream and 111 nucleotides downstream of the csiD transcriptional start site ( data not shown ) , suggesting that CRP regulation of csiD is probably due to direct binding of CRP in the regulatory region of csiD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9512707" target="_blank">9512707</a>]<br></li>
<li>Transcription of hdeAB depends on complex circuits of regulation involving RpoD ( 70 ) , RpoS ( 38 ) , H-NS , cyclic AMP receptor protein , GadE , GadX , GadW , GadY , EvgAS , YdeO , Lrp , MarA , SoxRS , TorRS , and TrmE , among other regulators ( see Fig. 1 and references in the legend ; see Fig. 2 for the DNA binding sites of some of these regulators in the hdeAB promoter ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18083817" target="_blank">18083817</a>]<br></li>
<li>This notion is supported by previous biochemical studies on E.coli Lrp and P.putida BkdR , which noted a decrease in the intrinsic ﬂuorescence upon their binding of their amino acid effectors to high afﬁnity sites , suggesting a conformational change in both proteins ( 49,52 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16528101" target="_blank">16528101</a>]<br></li>
<li>Leucine has also been shown to antagonize the binding of Lrp to the promoter regions of lysU ( 85 ) , which is negatively regulated by Lrp , and gltBDF , which is positively regulated by Lrp ( 44 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li>
<li>Considering the in vivo and in vitro data presented in this work , the mechanism by which ArgP regulates argO transcription ( 24 ) , and the known capacity of Lrp to alter the shape of the DNA by inducing bending and wrapping ( 57 ) , a model for lysP regulation in which binding of Lrp to the lysP control region may favor and/or stabilize the ArgP-RNA polymerase-DNA complexes or introduce DNA conformational changes is conceivable .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21441513" target="_blank">21441513</a>]<br></li>
<li>This organization of binding sites strongly suggests potential interferences in the binding of Lrp and ArgP to the argO operator and in their modulation of PargO activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19906180" target="_blank">19906180</a>]<br></li>
<li>In summary , we may conclude that binding of the transcriptional-regulators Lrp and ArgP to the argO control region is mutually inhibitory .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19906180" target="_blank">19906180</a>]<br></li>
<li>The molecular basis of this interference resides in the reciprocal inhibition of Lrp and ArgP binding to overlapping targets in the argO control region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19906180" target="_blank">19906180</a>]<br></li>
<li>These negative interferences result from competitive binding of Lrp and ArgP to partially overlapping targets in the argO control region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19906180" target="_blank">19906180</a>]<br></li>
<li>Autoradiographs of EMSAs with ArgP and Lrp binding to the argO control region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19906180" target="_blank">19906180</a>]<br></li>
<li>This might be explained by the lack of competition in the binding of Lrp and ArgP to the argO control region in the mutant .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19906180" target="_blank">19906180</a>]<br></li>
<li>Binding of ArgP and Lrp to the argO control region are mutually inhibitory EMSAs with various combinations of the two regulatory proteins were performed to analyse the effect of prebound ArgP on the binding of Lrp and vice versa ( Fig. 5 and Fig .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19906180" target="_blank">19906180</a>]<br></li>
<li>Leucine reduces the binding of Lrp to the argO control region about twofold ( Fig. 3A -- C ) and also reduces the activating effect of Lrp on PargO in vivo , at least in the WT strain ( Fig. 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19906180" target="_blank">19906180</a>]<br></li>
<li>Although alanine altered Lrp binding to the dad repressor and activator sites differentially , this difference in af ® - nity was only approximately 3.5-fold .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10216857" target="_blank">10216857</a>]<br></li>
<li>Differential binding of Lrp to activator and repressor sites To gain additional information on the Lrp binding sites involved in repression and activation , we analysed the binding of Lrp to these sites by DNase I footprinting in the presence and absence of L-alanine ( Fig. 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10216857" target="_blank">10216857</a>]<br></li>
<li>It was suggested that Lrp was a repressor in the absence of PapI and an activator in the presence of PapI , because PapI shifted Lrp binding from repressor to activator sites ( Nou et al. , 1995 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10216857" target="_blank">10216857</a>]<br></li>
<li>Our data show that Crp-cAMP binding in vitro ( Fig. 5B , E ) is correlated with reduced transcription in vivo ( Fig. 6 ) , and that RNAP generates a protection pattern at PgltB in the presence of Crp-cAMP ( Fig. 7 , lane 7 ) different from the pattern yielded by RNAP in the presence of IHF + Lrp ( Fig. 3B , lane 7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17233899" target="_blank">17233899</a>]<br></li>
<li>We tested this by gel retardation , subcloning different regions of the serA upstream sequence into the Bluescript vector and testing for Lrp binding in the presence of 25 or 250 ng of Lrp ( Fig. 3A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11985712" target="_blank">11985712</a>]<br></li>
<li>Lrp binding ( lane 5 ) is barely affected by 0.25 pM unmethylated Ada , while at the same concentration of methylated Ada the Lrp / PaldB complex is hardly detectable and PardB is almost completely shifted as the meAda/P , , dB complex ( Fig. 5 , lane 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8809748" target="_blank">8809748</a>]<br></li>
<li>As shown by Western blotting experiments , the poor binding was not due to the absence of Lrp ( Fig. 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8432705" target="_blank">8432705</a>]<br></li>
<li>As shown in Fig. 5 , in the presence of an equimolar concentration of meAda , Lrp binding to PaidB is almost completely abolished .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8809748" target="_blank">8809748</a>]<br></li>
<li>Since the in vivo experiments demonstrated that the presence of lysine affected lysP expression levels ( Fig. 8A ) , we evaluated whether lysine affects the binding of Lrp to the lysP control region in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21441513" target="_blank">21441513</a>]<br></li>
<li>Dissociation of Lrp from the aid5 promoter is observed even in the presence of a twofold excess of Lrp over meAda ( data not shown ) , suggestingthat the displacement of Lrp is mediated by binding of meAdato PaidSrather than protein : protein interaction and titration of Lrp by the transcriptional-activator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8809748" target="_blank">8809748</a>]<br></li>
<li>This could indicate that Lrp itself binds the osmC promoter region or that some protein expressed only in the presence of Lrp is responsible for this binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9663683" target="_blank">9663683</a>]<br></li>
<li>This implies that the binding of Lrp to its site in the promoter region would be inhibitory for the functioning of E-ss , but not for that of E-s70 , as the presence or absence of Lrp is neutral in the absence of ss .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9663683" target="_blank">9663683</a>]<br></li>
<li>Fig. 3A shows that the Lrp-binding region yields a subtly altered Lrp-dependent DNAse I footprint in the presence of IHF , though the IHF footprint is not obviously changed by the presence of Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17233899" target="_blank">17233899</a>]<br></li>
<li>D. EMSA of Lrp binding to the argO control region in the presence of various amino acids ( 7 mM ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19906180" target="_blank">19906180</a>]<br></li>
<li>Thus , EMSAs were performed in the presence of an excess of nonspeciﬁc competitor to assess the binding of puri-ﬁed His6-Lrp to the lysP promoter/control region ( Fig. 9 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21441513" target="_blank">21441513</a>]<br></li>
<li>Phase variation of pap and agn are both Dam dependent , and the regulation of each operon involves the DNA methylation-sensitive binding of a transcriptional-regulator , Lrp and OxyR respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10692164" target="_blank">10692164</a>]<br></li>
<li>The definition of the DNA sequences important for the Lrp regulation of gltBDF expression will be useful in the continuing effort to define DNA sequences recognized by Lrp in specific binding events and is currently under investigation in our laboratory .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7901196" target="_blank">7901196</a>]<br></li>
<li>( 18 ) Peterson , S. N. , Dahlquist , F. W. , and Reich , N. O. ( 2007 ) The role of high affinity non-specific DNA binding by Lrp in transcriptional regulation and DNA organization .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25785689" target="_blank">25785689</a>]<br></li>
<li>The role of high afﬁnity non-speciﬁc DNA binding by Lrp in transcriptional regulation and DNA organization .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21441513" target="_blank">21441513</a>]<br></li>
<li>Leucine had little effect upon the binding of Lrp to lrp DNA , as seen from DNase I footprinting ( Fig. 6A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8144448" target="_blank">8144448</a>]<br></li>
<li>Preparation of fragments used to measure DNA bending The following two complementary oligonucleotides containing Lrp binding site 2 ( Wang and Calvo , 1993 ) were chemically synthesized : 5 ' - CTAGATTGAATGTAGAATTTTATTCTGAATG-3 ' ; 5 ' - TCGACATTCAGAATAAAATTCTACATTCAAT-3 ' .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8508774" target="_blank">8508774</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_1327">
[ 18 ] Cui , Y. , Wang , Q. , Stormo , G.D. and Calvo , J.M. ( 1995 ) A consensus sequence for binding of Lrp to DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16098526" target="_blank">16098526</a>]<br></label>
<input id="act_similarity_1327", type="checkbox">
<div class="ACT_similar">
<li>Cui , Y. , Wang , Q. , Stormo , G.D. , and Calvo , J.M. ( 1995 ) A consensus sequence for binding of Lrp to DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16238633" target="_blank">16238633</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9515701" target="_blank">9515701</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10216857" target="_blank">10216857</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8809748" target="_blank">8809748</a>]<br></li>
<li>Cui , Y. , Wang , Q. , Stormo , J.D. , and Calvo , J.M. ( 1995 ) A consensus sequence for binding of Lrp to DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19906180" target="_blank">19906180</a>]<br></li></div>
<li>Binding of Lrp to pap regulatory DNA inhibited methylation of the pap GATC-I and GATC-II sites , whereas binding of GutR blocked methylation of GATC-44 .5 in the gutABD operon ( Fig. 1A and C ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9811649" target="_blank">9811649</a>]<br></li>
<li>One can then suppose that if the cell is deﬁcient in any one of the proteins , e.g. , Lrp , its DNA conformation will change slightly at all Lrp binding sites , and this may alter promoter strength more or less , depending on the individual gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7751276" target="_blank">7751276</a>]<br></li>
<li>DNA sites bound by Lrp seem to be naturally bent and this bending may play a crucial role in the regulation of ilvlH ; binding of Lrp further induces bending and contributes to loop formation ( Wang and Calvo 1993b ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7603430" target="_blank">7603430</a>]<br></li>
<li>The relatively stable complexes formed from Lrp binding to ilvIH DNA are here called complex 1 ( Lrp bound to sites 1 and 2 ) and complex 2 ( Lrp bound to sites 1 , 2 , 3 , 4 , 5 , and 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12218014" target="_blank">12218014</a>]<br></li>
<li>The relative sizes of DNA and Lrp , the positions of binding sites , and the 2 - , 4 - , 6 - , 8 - , and 10-bp addition site ( triangle ) are drawn roughly to scale , but the path of DNA relative to Lrp is imaginary .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12218014" target="_blank">12218014</a>]<br></li>
<li>In the cell , Lrp may interact with a complex mixture of solutes and proteins , some of which may be bound to DNA near Lrp binding sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7901196" target="_blank">7901196</a>]<br></li>
<li>Arrows denote the positions of free DNA , complex 1 ( Lrp bound to sites 1 and 2 ) , and complex 2 ( Lrp bound to sites 1 , 2 , 3 , 4 , 5 , and 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12218014" target="_blank">12218014</a>]<br></li>
<li>Leucine responsiveness of heterologous Lrp proteins Conserved function among regulators depends not only on DNA-binding ( and RNA poymerase-contacting ) properties , but in some cases also on responses to small mole-cule coregulators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18405378" target="_blank">18405378</a>]<br></li>
<li>In addition , we measured the effects of competitor DNA on binding of Lrp to the two constructs shown in Fig. 8A .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8636076" target="_blank">8636076</a>]<br></li>
<li>In order to study further the interaction between Lrp and DNA , we investigated whether binding of Lrp induces changes in the structure of DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8508774" target="_blank">8508774</a>]<br></li>
<li>In these analyses it was assumed that the binding of Lrp to gltBDF DNA is both necessary and sufficient for transcriptional activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li>
<li>In general , stimulation of transcription was observed over the same Lrp concentration range as that required for Lrp binding to ilvIH DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1447135" target="_blank">1447135</a>]<br></li>
<li>In some cases , the amount of free DNA and LrpzDNA complex were meas-ured over a range of Lrp concentrations and binding constants were calculated as the concentration of Lrp at which half of the DNA existed as complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8636076" target="_blank">8636076</a>]<br></li>
<li>Lrp bends DNA when it binds to a single binding site To examine whether Lrp can induce a bend in DNA upon binding , we employed a circular permutation assay developed by Wu and Crothers ( 1984 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8508774" target="_blank">8508774</a>]<br></li>
<li>Thus , binding of Lrp to the consensus-like site appears to involve symmetrical half-site contacts and interactions with adjacent major grooves of the DNA helix .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8900118" target="_blank">8900118</a>]<br></li>
<li>Although the functional oligomeric state of Lrp is an octamer or hexadecamer , the interaction with DNA is med-iated by individual homo-dimers and thus multiple DNA binding domains are present .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16879983" target="_blank">16879983</a>]<br></li>
<li>Our results for Lrp binding to gltBDF DNA bear striking resemblance to earlier studies on binding of l repressor to the right operator of bacteriophage lambda ( Hochschild & Ptashne , 1986 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9236118" target="_blank">9236118</a>]<br></li>
<li>Radiolabeled DNA fragments used in these experiments were present at a final concentration of 1 3 10211 M to ensure that the final free concentration of Lrp present in the binding reaction was essentially equivalent to the total Lrp concentration .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8900118" target="_blank">8900118</a>]<br></li>
<li>The abscissa is the concentration of Lrp dimer , assuming that it is fully active in binding DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8144448" target="_blank">8144448</a>]<br></li>
<li>The results in Figs. 6 and 7 demonstrating that both doubleand single-stranded flanking DNA can contribute to Lrp binding were obtained in experiments employing only a single Lrp concentration .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8636076" target="_blank">8636076</a>]<br></li>
<li>Lrp also interacts with a ligand , but the ligand has effects that can not be explained solely by a single mode of action , such as reducing the DNA-binding ability of Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li>
<li>Thus , it is likely that one of the two subunits of Lrp is bound more tightly than the other , and this might contribute to the asymmetry observed in the binding of flanking single-stranded DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8636076" target="_blank">8636076</a>]<br></li>
<li>This possibility seems ruled out by the data showing that single-stranded competitor DNA reduces rather than increases binding to Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8636076" target="_blank">8636076</a>]<br></li>
<li>Binding of Lrp to double-stranded DNA is expected to release more sodium ions than binding to an equivalent length of single-stranded DNA and thus higher sodium ion concentrations should reduce binding to double-stranded DNA more than binding to single-stranded DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8636076" target="_blank">8636076</a>]<br></li>
<li>However , as demonstrated here , single-stranded DNA can stimulate Lrp binding when it flanks a double-stranded consensus sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8636076" target="_blank">8636076</a>]<br></li>
<li>Effect of NaCl concentration on binding of Lrp to constructs having flanking double - or single-stranded DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8636076" target="_blank">8636076</a>]<br></li>
<li>We identiﬁed 63 related DNA sequences that bound Lrp and estimated their relative binding afﬁnities for Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7665463" target="_blank">7665463</a>]<br></li>
<li>This positive interaction ( and not Lrp binding ) is partially abolished by ArgR-Arg bound to glt DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17233899" target="_blank">17233899</a>]<br></li>
<li>Lrp binding to the gcv control region induces DNA bending .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9802028" target="_blank">9802028</a>]<br></li>
<li>Kaltenbach , L.S. , Braaten , B.A. , and Low , D.A. ( 1995 ) Spe-ciﬁc binding of PapI to Lrp -- pap DNA complexes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19906180" target="_blank">19906180</a>]<br></li>
<li>Determination of association constants for binding of Lrp to ilvIH DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12218014" target="_blank">12218014</a>]<br></li>
<li>Using this procedure , the angle of the DNA bend induced by Lrp binding was calculated to be 520 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8508774" target="_blank">8508774</a>]<br></li>
<li>Poly ( dI z dC ) was added to prevent nonspeciﬁc binding of Lrp to the DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955407" target="_blank">8955407</a>]<br></li>
<li>Part of the binding of Lrp to DNA is likely driven by an increase in entropy due to the release of Na ions from the DNA ( 24 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8636076" target="_blank">8636076</a>]<br></li>
<li>Therefore , it is very likely that Lrp itself is present in the retarded band A , and this band is probably the result of direct binding of Lrp on the osmC DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9663683" target="_blank">9663683</a>]<br></li>
<li>Slight differences in the mobilities of the free DNA fragments were observed , suggesting that the DNA itself is slightly bent in the absence of any protein , but this degree of bending is small in comparison with that induced by Lrp binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8508774" target="_blank">8508774</a>]<br></li>
<li>This last characteristic , together with the high abundance of Lrp in the cell and the fact that binding of Lrp to the DNA causes major conformational changes ( 13 , 57 ) , classiﬁes Lrp as a nucleoid-associated protein ( NAP ) ( 14 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21441513" target="_blank">21441513</a>]<br></li>
<li>One possibility , given the proximity of the Crp and IHF binding sites ( Fig. 2A ) , and the requirement of IHF for activation by Lrp [ 14 ] , is that Crp interferes with IHF binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17233899" target="_blank">17233899</a>]<br></li>
<li>Since the proximal Lrp binding site in gltBDF operon is positioned 110 bp upstream of the 235 hexamer , an intermediate IHF binding site appears to be essential to Lrpmediated activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11395454" target="_blank">11395454</a>]<br></li>
<li>Studies of binding of Lrp and CRP-cAMP to the PserC ( pdxF ) promoter region are currently in progress .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9171388" target="_blank">9171388</a>]<br></li>
<li>CpxR-P most likely inhibits transition to the ON position by competing with Lrp for binding to the pap promoter [ 20 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16487683" target="_blank">16487683</a>]<br></li>
<li>Lrp and IHF appear to stabilize RNAP binding to the promoter , as no RNAP-dependent footprint appears unless both Lrp and IHF are present .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17233899" target="_blank">17233899</a>]<br></li>
<li>Mutations affecting the Lrp binding site reduce autogenous regulation by Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8144448" target="_blank">8144448</a>]<br></li>
<li>Most of the autogenous regulation by Lrp could be accounted for by binding of Lrp to a site located immediately upstream of the lip promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li>
<li>Effect of Lrp dimer concentration on binding of Lrp to the ompC-micF intergenic region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8002608" target="_blank">8002608</a>]<br></li>
<li>Thus , it is likely that bands A through D are the result of specific binding of Lrp to the labeled gcv fragment .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7928983" target="_blank">7928983</a>]<br></li>
<li>Lrp does respond to exogenous alanine and this could clearly be incorporated into the binding pocket .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16528101" target="_blank">16528101</a>]<br></li>
<li>However , it was difficult to determine from this experiment which Lrp binding regions had to be occupied for stimulation to occur .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1447135" target="_blank">1447135</a>]<br></li>
<li>Leucine in the binding reaction reduced Lrp binding to each of the 12 clones which bound Lrp and did not cause binding of Lrp to the other 41 clones ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7665463" target="_blank">7665463</a>]<br></li>
<li>Collectively , the data suggest that either the binding of Lrp to the ﬁm switch to form complexes 1 and 2 is co-operative or the stability of intermediate complexes is too low to survive electrophoresis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9515701" target="_blank">9515701</a>]<br></li>
<li>Therefore , the absence of binding activity in vitro was not simply due to an absence of Lrp protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8432705" target="_blank">8432705</a>]<br></li>
<li>No transcriptional effects are observed , in vivo or in vitro , in the absence of an intact Lrp primary binding site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8900118" target="_blank">8900118</a>]<br></li>
<li>In part as a control to conﬁrm the binding of the Lrp and its potential as an effective roadblock , we carried out a similar experiment probing the EcoRI 's processivity response to nonspeciﬁcally bound Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25785689" target="_blank">25785689</a>]<br></li>
<li>Complementary sequences F and R comprise a double-stranded 50 mer that bound Lrp most strongly from a collection of 63 sequences that were selected in vitro for binding to Lrp ( Leu-19 in Ref .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8636076" target="_blank">8636076</a>]<br></li>
<li>In parallel , we investigated the effect of salt on binding of Lrp to the double-stranded 50 mer .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8636076" target="_blank">8636076</a>]<br></li>
<li>In the presence of in vitro Lrp binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9236118" target="_blank">9236118</a>]<br></li>
<li>The gel-purified Lrp protein was checked for binding to the palp intercistronic region ( known previously to-bind the Lrp protein ; Braaten et of .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8890170" target="_blank">8890170</a>]<br></li>
<li>Analysis of Lrp and Ada binding by gel retardation assay Lane 1 , PardBDNA , no protein added ; lane 2 , 5 prnol of unmethyl-ated Ada protein ; lane 3 , 5 pmol of methylated Ada protein ; lane 4 , 5 prnol of Lrp ; lane 5 , 5 pmol of Lrp and unmethylated Ada : lane 6 , 5pmol of Lrp and methylated Ada .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8809748" target="_blank">8809748</a>]<br></li>
<li>The present data were interpreted as indicating that the species active in binding for repression of lysU is a dimer , as has been suggested for activation of ilvlH ( Platko and Calvo 1993 ) , and that heterodimers are inactive because they have only a single wild-type Lrp subunit .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7603430" target="_blank">7603430</a>]<br></li>
<li>The magnitude of the effect of these mutations on switching frequencies shows a good correlation with their effect on in vitro binding of Lrp to the respective fimS alleles .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7916011" target="_blank">7916011</a>]<br></li>
<li>The gltBDF binding data plotted in Fig. 6 predict that the effective concentration of Lrp will have decreased approximately 3.5-fold under these conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7901196" target="_blank">7901196</a>]<br></li>
<li>The extent of binding was linear with the amount of Lrp under these conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2040596" target="_blank">2040596</a>]<br></li>
<li>The T 's in boldface lettering at 110 and 111 were changed by mutagenesis to C and G , respectively , resulting in altered Lrp binding ( Fig. 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955407" target="_blank">8955407</a>]<br></li>
<li>To investigate structure-function relationships of Lrp protein in terms of its DNA binding and repression of lysU , we isolated dominant-negative Lrp mutations by selecting lys U-constitutive mutants through targeted mutagenesis of plasmid-encoded lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7603430" target="_blank">7603430</a>]<br></li>
<li>These observations clearly suggest that the mode of DNA binding of Lrp in both repression and activation of transcription may be similar , although it is not presently known whether all lrp mutations isolated by Platko and Calvo ( 1993 ) also exert dominant negative effects on lysU repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7603430" target="_blank">7603430</a>]<br></li>
<li>Leucine was added to in vitro transcription reaction mixtures at a concentration ( 37 mM ) that was found by Ricca et al. ( 15 ) to reduce Lrp binding to ilvIH DNA by 2.5-fold .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1447135" target="_blank">1447135</a>]<br></li>
<li>Since Lrp was able to-bind to ilvIH DNA in vitro but in vivo expression was reduced , these strains were presumed to have mutations that affect the ability of Lrp to activate transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8432705" target="_blank">8432705</a>]<br></li>
<li>For at least the ilvIH operon , Lrp regulates gene expression directly by binding to speciﬁc DNA sequences ( 42 , 44 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7665463" target="_blank">7665463</a>]<br></li>
<li>Mutations in lrp that reduced ilvIH expression in vivo 3 - to 12-fold , but that did not markedly affect the pattern of Lrp binding to DNA in vitro , were termed activation mutants .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8432705" target="_blank">8432705</a>]<br></li>
<li>These mutations affected a predicted DNA-binding domain of Lrp and mutants were defective both in activation of ilvIH expression and in repression of lysU expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7603430" target="_blank">7603430</a>]<br></li>
<li>These results are consistent with those obtained using the dad ± lacZ fusion strains and from our previous in vitro analysis ( Mathew et al. , 1996 ) and strongly support the contention that binding of Lrp to site 9 is required for repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10216857" target="_blank">10216857</a>]<br></li>
<li>Taken together , the results indicate that Lrp binding to the downstream region , primarily binding to site 9 , is required for repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10216857" target="_blank">10216857</a>]<br></li>
<li>Since the PurR-binding site was inactivated by the 2-bp change ( 31 ) and Lrp was inactivated by the TnJO insertion , the results suggest that GcvA could be responsible for the purine-mediated repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7928983" target="_blank">7928983</a>]<br></li>
<li>Additionally , the concentration of Lrp required for half-maximal repression is comparable to the concentration of Lrp required for halfmaximal saturation of the Lrp binding sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8900118" target="_blank">8900118</a>]<br></li>
<li>Effect of mutagenesis of the Lrp binding sites on repression We used site-directed mutagenesis to evaluate the role of individual Lrp binding sites on Lrp-mediated repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10216857" target="_blank">10216857</a>]<br></li>
<li>For example , in lysU , cooperative binding of Lrp to multiple sites has been suggested to enhance Lrp binding to a critical site for repression that encompasses the 235 region ( 9 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955407" target="_blank">8955407</a>]<br></li>
<li>Thus , a cluster of single-base substitutions within the region -60 to - 53 prevents binding of Lrp in vitro and significantly reduces repression of lip caused by Lrp in vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8144448" target="_blank">8144448</a>]<br></li>
<li>A mutation in that site significantly decreased repression caused by Lrp and reduced binding in vitro ( 162 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li>
<li>( B ) Lrp repression is strongly reduced by the Lrp binding site mutation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955407" target="_blank">8955407</a>]<br></li>
<li>Site-directed mutagenesis of an Lrp binding site strongly reduced Lrp binding and prevented Lrp repression of dad transcription in vivo and in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955407" target="_blank">8955407</a>]<br></li>
<li>Site-directed mutagenesis alters Lrp binding in vitro and strongly reduces Lrp repression in vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955407" target="_blank">8955407</a>]<br></li>
<li>( ii ) A mutation that strongly reduced Lrp binding in vitro prevented Lrp repression of dad in vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955407" target="_blank">8955407</a>]<br></li>
<li>Mutagenesis in a region close to the dad transcription start sites reduced Lrp binding and prevented Lrp repression in vivo and in vitro ( Mathew et al. , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10216857" target="_blank">10216857</a>]<br></li>
<li>Our binding and transcription analysis showed that dad also contains Lrp binding sites for repression and activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10216857" target="_blank">10216857</a>]<br></li>
<li>As shown in Fig. 8C , the nine protected regions show minimal overlap with the known binding sites for integration host factor ( 21 ) , and OmpR ( 56 ) , in agreement with the absence of an effect of Lrp on the increase in ompC transcription caused by sucrose .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8002608" target="_blank">8002608</a>]<br></li>
<li>Thus , there is a possibility that binding of Lrp to the intergenic region could negatively regulate the transcription of both of these genes , leading to a direct negative effect on ompC transcription and an indirect positive effect on ompF translation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8002608" target="_blank">8002608</a>]<br></li>
<li>Transcriptional level CpxR-P not only modulates the action of different regulators , such as BaeR , Lrp and OmpR , by binding to the same regulatory sequences ( see above ) , but it also represses the gene transcription of other signaling systems , such as σE and Aer .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16487683" target="_blank">16487683</a>]<br></li>
<li>To compare the concentration of Lrp needed for the stimulation of in vitro transcription with that needed for binding , we performed in vitro transcription under standard conditions with different concentrations of Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1447135" target="_blank">1447135</a>]<br></li>
<li>This result may be due to differences in Lrp binding constants under in vitro transcription conditions versus gel retardation assay conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1447135" target="_blank">1447135</a>]<br></li>
<li>Our results are consistent with the assumption that Lrp and OmpR bind independently and that binding of Lrp to a promoter containing bound OmpR represses transcription of ompC even in the presence of OmpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8002608" target="_blank">8002608</a>]<br></li>
<li>Thus , it seems likely that Lrp regulated serC ( pdxF ) - aroA transcription by directly binding to the upstream Lrp box .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9171388" target="_blank">9171388</a>]<br></li>
<li>Lrp acts directly to regulate its own synthesis by binding to a region just upstream of the l1p promoter and repressing transcription from that promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8144448" target="_blank">8144448</a>]<br></li>
<li>Lrp stimulates transcription of lysP by direct binding to its control region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21441513" target="_blank">21441513</a>]<br></li>
<li>Xbel Mlul Xhol I f 1 Xhol I Lrp bindingregion t Xbel Mkrl Xhol I I F I BemHl I Hypothesis for Lrp and GcvA involvement in gcv expression We propose that the role of Lrp in gcv expression is probably one of bending the DNA such that GcvA bound at the upstream sites 3 and 2 can interact either with RNAP to activategcu or possibly with GcvA bound at site 1 to repress gcv ( Fig. 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9802028" target="_blank">9802028</a>]<br></li>
<li>Since the P-galactosidase levels observed in the GS852 + 1l - , , , and +21 - , , , lysogens were reduced in the f r p mutant , the results suggest that Lrp is still required for gcvT : : lac2 expression when GcvA-binding sites 3 and 2 are shifted by helical turns of DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9802028" target="_blank">9802028</a>]<br></li>
<li>elevated levels of / I-galactosidase compared to the GS852 We showed previously that moving the entire Lrp - +15 - , , , and +25 -21 , lysogens ( Table 2 ) , these levels binding region and GcvA-binding sites 3 and 2 by half were still significantly lower than the levels found in the integral turns of DNA did not result in super-repression , gcuA GS998 +15 -21 , and +25 -21 , lysogens ( Table 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9802028" target="_blank">9802028</a>]<br></li>
<li>Effects of insertion of integral or half-integral turns of DNA between base -213 and -214 on gcvT : : lacZ expression To demonstrate whether the glycine-mediated activation was GcvA and Lrp dependent and the inosinemediated repression was GcvA and GcvR dependent , we lysogenized the gcuA strain GS998 , the f r p strain GS997 and the gcuR strain GS1053 with the + ll - , , , and +2 l - , , , insertion phage .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9802028" target="_blank">9802028</a>]<br></li>
<li>Nevertheless , the results suggest that GcvA could be responsible for the purine-mediated repression in lysogen GS997XT lac ( A-466 +8 C + 1OA ) and that GcvA might be able to-bind to DNA in the absence of Lrp to repress gcvT-lacZ expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7928983" target="_blank">7928983</a>]<br></li>
<li>la ) , as well as the strict requirement that GcvA and Lrp proteins bind on the correct surface of the DNA relative to RNAP for both activation and repression , make it likely that DNA looping is part of the mechanism of gcu operon regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9611815" target="_blank">9611815</a>]<br></li>
<li>However , evidence suggest that Lrp s role is structural , bending the PgcvT DNA to allow a GcvA Æ RNAP interaction [ 14 ] and CRP s role is to antagonize GcvA-mediated repression [ 15 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15621456" target="_blank">15621456</a>]<br></li>
<li>Activation and repression by GcvA requires Lrp , presumably to bend the DNA appropriately ( Stauffer & Stauffer , 1998a , b , 1999 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11065369" target="_blank">11065369</a>]<br></li>
<li>One model proposed for gcv regulation is that Lrp plays a structural role , bending the DNA to allow GcvA to function as either an activator or a repressor in response to environmental signals .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10217490" target="_blank">10217490</a>]<br></li>
<li>Microbiology , The University of Iowa , Iowa City , IA 52242 , USA Keywords : gcvTHP , repressor , activator , repression , antiactivation INTRODUCTION proposed that Lrp plays a structural role in gcv regulation , binding to and bending the DNA between GcvA-binding sites 1 and 2 to facilitate the formation of appropriate regulatory complexes ( Stauffer & Stauffer , 1999 ) ( Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11495998" target="_blank">11495998</a>]<br></li>
<li>The repression observed in the GS852 IHF-C and IHF-D results are also consistent with a model where Lrp lysogens are due to IHF binding and bending the gcv normally plays primarily a structural role , and GcvA is DNA , this activation and repression should not be the gcv-specific regulatory protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10217490" target="_blank">10217490</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_1434">
A. ArgP was used at 4.5 nM ( monomer equivalents ) and allowed to-bind to the argO operator DNA prior to the addition of different concentrations of Lrp ( as indicated in nM , monomer equivalents ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19906180" target="_blank">19906180</a>]<br></label>
<input id="act_similarity_1434", type="checkbox">
<div class="ACT_similar">
<li>B. Lrp was used at 425 nM ( monomer equivalents ) and allowed to-bind prior to the addition of various concentrations of ArgP ( as indicated in nM , monomer equivalents ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19906180" target="_blank">19906180</a>]<br></li></div>
<li>All 12 DNA-binding proteins , CbpA , CbpB ( or Rob ) , DnaA , Dps , Fis , Hfq , H-NS , HU , IciA , IHF , Lrp and StpA , were over-expressed using pCU60 , pMK19 , pSY567 , pDPS1 , pRJ1077 , pHFQ607 , pHOP11 , pLhupAhupB , pICS1 , pSA5hiphimA , pMWD1 and pT7stpA , respectively , and puri ® ed to apparent homogeneity as previously described ( Talukder & Ishihama 1999 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10947847" target="_blank">10947847</a>]<br></li>
<li>In the ﬁrst model , wild-type levels of H-NS could bind to DNA sequences within the IE , to sequences on either side of the IE , or to 9-bp inverted repeats ﬂanking the IE to hinder ﬂipping or preclude the binding of activators such as integration host factor ( 11 , 14 ) and Lrp ( 4 , 19 ) or the recombinases ( FimB and FimE ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9352908" target="_blank">9352908</a>]<br></li>
<li>Some of these proteins bind nonspeci ® cally to DNA , e.g. HU and H-NS , while others are site-speci ® c , e.g. FIS , integration host factor ( IHF ) , and Lrp ( Schmid 1990 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9613586" target="_blank">9613586</a>]<br></li>
<li>In contrast , when a DNA fragment incorporating Lrp site III ( Fig. 4 ) was used to drive lacZ in plasmid pLrp3 , - galactosidase activity increased to values that paralleled those seen in pLrp4 , indicating that sequence FIG. 3 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19074390" target="_blank">19074390</a>]<br></li>
<li>DNA fragments containing the intergenic region were labeled at their 59 ends , and then they were incubated for 30 min in 200 ml of assay buffer in the presence of 0 , 25 , 50 , 100 nM puriﬁed Lrp ( lanes 1 , 2 , 3 , and 4 , respectively ) before being digested by DNase I. Lanes labeled G , A , T , and C indicate chain termination sequencing ladders generated with the indicated dideoxynucleotides .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8002608" target="_blank">8002608</a>]<br></li>
<li>Binding of His6-Lrp to DNA was performed with 30 fmol of a DNA fragment and increasing concentrations of His6-Lrp in 25 l Lrp binding buffer supplemented with 20 g ml 1 salmon sperm DNA and 0.1 mM lysine where indicated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21441513" target="_blank">21441513</a>]<br></li>
<li>A 6 cadBA fragment encompassing the P promoter/control region was incucadBA bated with increasing concentrations of puriﬁed His6-Lrp in the presence of salmon sperm DNA as a nonspeciﬁc competitor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21441513" target="_blank">21441513</a>]<br></li>
<li>Alternatively , if Lrp were monomeric in solution at the concentrations used for these mobility shift assays , but it bound to DNA as a dimer , Hill coefﬁcients of up to 2 could be observed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8002608" target="_blank">8002608</a>]<br></li>
<li>Data from gel mobility shift assays carried out in the presence of a ramp of Lrp dimer concentrations from 0 to 125 nM were quantiﬁed by PhosphorImager scanning and plotted as the percent DNA bound versus the concentration of Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8002608" target="_blank">8002608</a>]<br></li>
<li>As shown in Figure 1B , in the presence of increasing amounts of Lrp , the amount of Lrp-DNA complex increased and the amount of double-stranded , but not single-stranded , DNA decreased .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8508774" target="_blank">8508774</a>]<br></li>
<li>Densitometric tracing of Figure lB indicated that the amount of single-stranded DNA increased - 1.5-fold at the higher concentrations of Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8508774" target="_blank">8508774</a>]<br></li>
<li>Autoradiogram of DNase I protection patterns obtained in the absence and presence of increasing amounts of purified Lrp bound to the nontranscribed ( ilv bp 110 -- 278 ( A ) ) or transcribed ( ilv bp 6 -- 278 ( B ) ) DNA strand of the ilvGMEDA leader-regulatory region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8900118" target="_blank">8900118</a>]<br></li>
<li>Autoradiogram of DMS interference patterns obtained in the absence and presence of a minimally saturating amount of purified Lrp ( 2.6 3 1028 M ) bound to the nontranscribed ( ilv bp 1110 -- 1278 ) DNA strand of the ilvGMEDA leader-regulatory region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8900118" target="_blank">8900118</a>]<br></li>
<li>The autoradiogram in Fig. 6A shows that in the presence of increasing concentrations of Lrp ( 5.0 3 10210 to 3.25 3 1028 M ) , three regions of DNase I protection and two intervening regions of enhancement on the nontranscribed DNA strand are observed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8900118" target="_blank">8900118</a>]<br></li>
<li>In the presence of Lrp a fraction of PaldBdoes not enter the gel , and appears as a separate complex in Fig. 5 , lanes 4-9 ; it seems likely that this complex is the result of non-specificbinding of Lrp to DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8809748" target="_blank">8809748</a>]<br></li>
<li>In the presence of saturating amounts of Lrp and IHF , most of the DNA was bound to both IHF and Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11395454" target="_blank">11395454</a>]<br></li>
<li>In the presence of both Lrp and IHF in the reaction mixture , we observed an additional band corresponding to the ternary complex , which was not present when either Lrp or IHF alone was used in the reactions ; at saturating concentrations of Lrp and IHF , most of the DNA migrated in this band .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11395454" target="_blank">11395454</a>]<br></li>
<li>Analysis of Lrp distribution between cytoplasm and nucleoid suggests that free Lrp in cells ( i.e. Lrp not bound to DNA either speci ® cally or non-speci ® cally ) is about 7 mM in cells grown in Mops minimal med-ium ( S.C. , Z. Hao , E. Bieniek & J.M.C. , unpublished results ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11575919" target="_blank">11575919</a>]<br></li>
<li>For E. coli grown in a minimal me-dium , about 60 % of the total Lrp is bound nonspeciﬁcally to DNA , but the concentration of free Lrp is high enough that some mixture of octamers and hexadecamers is expected in vivo ( 4 , 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12218014" target="_blank">12218014</a>]<br></li>
<li>Lrp ( formerly called IHB ) binds ilvIH DNA in this region and activates transcription of the ilvIH operon ( 13 , 15 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1447135" target="_blank">1447135</a>]<br></li>
<li>Lrp strongly and cooperatively binds to multiple sites located between positions ÿ14 and ÿ102 ( in a way that suggests DNA wrapping around multiple Lrp molecules ) and moderately stimulates in vitro transcription , especially with EsS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9512707" target="_blank">9512707</a>]<br></li>
<li>In Vitro Transcriptions in the Absence and Presence of Lrp -- A purified in vitro transcription system was employed to examine the effects of Lrp on the production of attenuated RNA transcripts from the promoter-attenuator region of the ilvG-MEDA operon on a negatively supercoiled DNA template .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8900118" target="_blank">8900118</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_1458">
Lrp is sufﬁcient for the establishment of the phase off pap DNA methylation and repression of pap transcription in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11972787" target="_blank">11972787</a>]<br></label>
<input id="act_similarity_1458", type="checkbox">
<div class="ACT_similar">
<li>Lrp is sufﬁcient for the establishment of the phase OFF pap DNA methylation pattern and repression of pap transcription in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12029051" target="_blank">12029051</a>]<br></li></div>
<li>No proteins RNAP NA G C T A Discussion IHF R P+C rp Role of IHF in mediating activation of gltBD by Lrp IHF , a DNA-bending protein [ 30 ] , is required for activation of gltBDF transcription by Lrp [ 14 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17233899" target="_blank">17233899</a>]<br></li>
<li>( 12950 ; A streptavidin ) ArgR , 16863 Lrp , 18756 IhfB , IhfA , 10520 11223 Lrp , 18756 B Crp , 23509 A second possibility , which we did not test , is that the Crp-induced bend in the DNA [ 23 ] opposes that generated by IHF , preventing proper activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17233899" target="_blank">17233899</a>]<br></li>
<li>We propose that IHF-induced bending of DNA positions bound Lrp close to RNA polymer-ase for transcriptional activation of the gltBDF operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11395454" target="_blank">11395454</a>]<br></li>
<li>ACKNOWLEDGMENTS We thank the Protein-DNA Core Facility , Cancer Center , University of Utah , Salt Lake City , for the production of the oligonucleotides used in this study ; Julie Tucker for technical assistance ; Felix Vajdos for puriﬁed Lrp ; and Bruce Braaten for helpful discussion .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9811649" target="_blank">9811649</a>]<br></li>
<li>Thus , Lrp appears to form a complex with the ﬁm switch in which multiple Lrp dimers are bound to the DNA with the latter wrapped or looped around a protein core .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9515701" target="_blank">9515701</a>]<br></li>
<li>As the sole regulator , Lrp can directly repress or stimulate promoter activity by steric exclusion of the polymerase , DNA remodelling or protein -- protein contacts .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19906180" target="_blank">19906180</a>]<br></li>
<li>If Lrp is behaving like a typical dimeric helix-turn-helix protein when it binds to the palindromic site 2 , we can also infer which way Lrp bends the DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9236118" target="_blank">9236118</a>]<br></li>
<li>Lrp-1 in crude extracts also binds to the iluZH promoter/regulatory region , but themobility of the major DNA-protein complex formed is faster than that formed by Lrp ( 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2040596" target="_blank">2040596</a>]<br></li>
<li>, Lrp is a relatively small protein ( 19 kDa ) and binds to DNA as a dimer ( Wang and Calvo , 1993a ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11985712" target="_blank">11985712</a>]<br></li>
<li>Alternatively , it is possible that Lrp forms protein -- protein contacts in complex 2L to produce the same DNA to protein stoichiometry as complex 1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9515701" target="_blank">9515701</a>]<br></li>
<li>This protein can bind both E. coli and S. typhimurium iivlH DNA , as can E. coli Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8483418" target="_blank">8483418</a>]<br></li>
<li>Our results suggest that Lrp binds a central palindromic site , interacting predominantly with the major groove of its DNA target , and that additional dimers bind to ¯ anking sites to form a nucleoprotein activation complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9236118" target="_blank">9236118</a>]<br></li>
<li>An extension of the model can be made in which two octamers form a hexadecamer and bind multiple DNA sites , as observed for E.coli Lrp ( 51 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16528101" target="_blank">16528101</a>]<br></li>
<li>The presence of numerous Lrp-DNA complexes suggests that Lrp binds to multiple sites on this fragment .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955407" target="_blank">8955407</a>]<br></li>
<li>PapI binds to the Lrp moiety of Lrp-pap DNA complexes and increases the afﬁnity of Lrp for pap sites 4 and 5 ( 14 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9811649" target="_blank">9811649</a>]<br></li>
<li>Contributing to DNA wrapping is a propensity of Lrp to bend DNA at sites to which it binds speciﬁcally ( 21 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12218014" target="_blank">12218014</a>]<br></li>
<li>Lrp appeared to act directly to inhibit methylation since Lrp specifically bound to a DNA fragment this site became containing this GATC site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li>
<li>This is perhaps not surprising given reports that members of the family , such as Lrp , LrpA , LrpC and PutR bind to and induce bending in DNA ( 6,7,11,25,26 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16528101" target="_blank">16528101</a>]<br></li>
<li>Visual inspection and densitometric analyses of the various bands indicated that Lrp binds more readily to free DNA molecules than to ArgP complexed operator DNA ( Fig. 5A and Fig .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19906180" target="_blank">19906180</a>]<br></li>
<li>Protein was in excess over DNA in these experiments , and the apparent dissociation constant was taken as the concentration of Lrp that drove half of the DNA into complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7665463" target="_blank">7665463</a>]<br></li>
<li>Consequently , increasing the concentration of Lrp favors further Lrp-Lrp and/or Lrp-DNA interactions that extend the area of protection .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7916011" target="_blank">7916011</a>]<br></li>
<li>We conﬁrmed conditions where Lrp , most likely as an octamer , binds DNA nonspeci 18 ﬁcally by electrophoresis ( Figure 6A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25785689" target="_blank">25785689</a>]<br></li>
<li>Single-stranded flanking DNA might bind to the same site on Lrp that normally binds double-stranded flanking DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8636076" target="_blank">8636076</a>]<br></li>
<li>This result suggests that Lrp binds to double-stranded , but not to single-stranded DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8508774" target="_blank">8508774</a>]<br></li>
<li>Binding of Lrp to DNA from the serA upstream region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11985712" target="_blank">11985712</a>]<br></li>
<li>Lrp binds to DNA containing the ompC-micF intergenic region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8002608" target="_blank">8002608</a>]<br></li>
<li>Free DNA in the absence of Lrp is shown in lane 1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9515701" target="_blank">9515701</a>]<br></li>
<li>Alternatively , a nucleoprotein structure may be formed in which DNA is wrapped around multiple Lrp molecules .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8508774" target="_blank">8508774</a>]<br></li>
<li>It is more likely that Lrp causes DNA to be looped or to be wrapped around a core of Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li>
<li>From left to right the lanes include 0 , 0.5 , 1 , and 2 Lrp octamers per DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25785689" target="_blank">25785689</a>]<br></li>
<li>RESULTS Selection of DNA molecules that bind Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7665463" target="_blank">7665463</a>]<br></li>
<li>Binding of Lrp to differentially phased DNA constructs .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12218014" target="_blank">12218014</a>]<br></li>
<li>The latter migrated slightly above the ArgP -- DNA complex formed at lower ArgP concentrations , suggesting that ArgP might bind to argO fragments having a single target bound by Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19906180" target="_blank">19906180</a>]<br></li>
<li>The Effect of NaCl and Competitor DNA on Binding of Lrp to Constructs Having Flanking Double - or Single-stranded DNA -- We measured the relative stability of LrpzDNA complexes as a function of NaCl concentration .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8636076" target="_blank">8636076</a>]<br></li>
<li>The effects of these mutations appeared to be due to inhibition of GATC methylation , since the afﬁnities of Lrp and Lrp-PapI for pap mutant DNA were not signiﬁcantly altered ( 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9811649" target="_blank">9811649</a>]<br></li>
<li>The palindromic nature of the consensus sequence derived here suggests that Lrp binds to DNA as a homodimer in such a way that each monomer binds to a half-site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7665463" target="_blank">7665463</a>]<br></li>
<li>The direction of the bend induced in gltBDF DNA by Lrp can be inferred from our data .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9236118" target="_blank">9236118</a>]<br></li>
<li>The ` disk-like ' structure of Lrp enables it to wrap DNA , which occurs in a right handed superhelix constraining positively supercoils and eVectively compacting DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16879983" target="_blank">16879983</a>]<br></li>
				</div></ul><br>
			<label class="collapse_DOM" for="_2"><font size="5px">Lrp protein domains</font></label>
				<input id="_2" type="checkbox">
				<div class="DOM"><br><ul><li>Materials and Methods Plasmid construction and protein purification E. coli strain JM105 containing plasmid pJWD28 and strain CV1306 containing plasmid pCV2947 were used to prepare Lrp and an Lrp derivative having six histidine residues at the N terminus ( 6HisLrp ) , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11575919" target="_blank">11575919</a>]<br></li>
<li>To prepare 6HisLrp containing C-terminal deletions , lrp gene fragments lacking DNA coding for 11 , 31 , and 61 amino acid residues at the C terminus were ampli ® ed by the polymerase chain reaction ( PCR ) using a set of 30 terminal primers containing a stop codon and were ligated into a 6His fusion protein expression vector pQE30 ( Qiagen ) , producing plasmids pCV400 , pCV401 , and pCV402 , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11575919" target="_blank">11575919</a>]<br></li>
<li>Our results , when viewed together with the crystallographic results from a distantly related Lrp family member , suggest that a basic octomer structure , with dimerization and oligomerization promoted by C-terminal sequences , will be a common feature of most members of this family .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11575919" target="_blank">11575919</a>]<br></li>
<li>Among operons activated by Lrp , in some cases that activation is overcome by leucine , in other cases the activation requires leucine , and in yet other cases the activation is independent of leucine ( 10 , 13 , 20 , 25 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7665463" target="_blank">7665463</a>]<br></li>
				<label class="DOM_collapse_similarity" for="dom_similarity_5">
Of the operons activated by Lrp , in some cases the activation requires leucine , in some cases the activation is negated by leucine , and in other cases the activation is independent of leucine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8144448" target="_blank">8144448</a>]<br></label>
<input id="dom_similarity_5", type="checkbox">
<div class="DOM_similar">
<li>Of the operons activated by Lrp , in some cases that activation is negated by leucine , in some cases the activation requires leucine , and in yet other cases the activation is independent of leucine ( Ernsting etal .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8483418" target="_blank">8483418</a>]<br></li></div>
<li>Lrp appears to pliiy primarily a structural role in gcu reg : ulntion , bending the D N A ( Stauffer & Stauffer , 1999 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10463182" target="_blank">10463182</a>]<br></li>
<li>Lrp does not violate any of these constraints except that residue 15 is a cysteine , which is polar rather than hydrophobic .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8432705" target="_blank">8432705</a>]<br></li>
<li>Although E137 in AsnC becomes R145 in Lrp , there is a compensatory change in an interacting residue of an H for a D at a position equivalent to residue 104 of AsnC ( Figure 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16528101" target="_blank">16528101</a>]<br></li>
<li>Furthermore , residue G103 in the turn between b3 and b4 ( equivalent to G111 in Lrp ) that is critical to the interaction with the generic amino group of the effector is totally conserved across all known sequences ( 29 ) and is the only glycine in the structure to possess a positive phi angle ( 90 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16528101" target="_blank">16528101</a>]<br></li>
<li>Plasmids pLRP11-pLRP81 were isolated from pfRP22 by plasmid mutagenesis and encoded mutant Lrp proteins ( see below ) , pRP77 carries a lysUlacZ gene fusion containing the N-terminal 110 amino acids of lysU ( Ito et al. 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7603430" target="_blank">7603430</a>]<br></li>
<li>A systematic survey of effector function for all amino acids indicated that His , Ile , Met and Thr inﬂuence , besides Leu and Ala , Lrp activity ( Hart & Blumenthal , 2011 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2162465" target="_blank">2162465</a>]<br></li>
<li>Two alleles were m a p p e d in helix-1 of a putative helix-turn-helix motif ( Willins et al. 1991 ) of Lrp : leucine to phenylalanine and serine to phenylalanine changes at amino acid positions 34 and 35 , respectively ( lrp-703 , -706 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7603430" target="_blank">7603430</a>]<br></li>
<li>This suggests that the strains are deficient in reactions that transfer ammonia and/or amino groups and that Lrp regulates synthesis of one or more of the enzymes for handling nitrogen obtained at a low level from these amino acids .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1532173" target="_blank">1532173</a>]<br></li>
<li>NtrC and Lrp are identical in the two strains , and catabolite gene activator protein and the a subunit of RNA polymerase differ by only one and two amino acids in these bacteria .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9495755" target="_blank">9495755</a>]<br></li>
<li>The Lrp used in the experiments reported here is a 6-His-tagged derivative of Lrp containing 12 extra amino acids at the NH2 terminus and was purified to near homogeneity in this laboratory according to the methods of Cui et al. ( 12 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8900118" target="_blank">8900118</a>]<br></li>
<li>Heterodimer Formation in Vitro -- An Lrp derivative having 12 extra amino acids at the NH2 terminus , including 6 His residues ( 6XHis-Lrp ) was made by using the QIAexpress system ( Qiagen , Inc. ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8636076" target="_blank">8636076</a>]<br></li>
<li>An Lrp derivative having 12 extra amino acids at the NH2 terminus , including 6 His residues ( 6XHis-Lrp ) , shows DNA-binding characteristics that are almost identical to the wild type ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8636076" target="_blank">8636076</a>]<br></li>
				<label class="DOM_collapse_similarity" for="dom_similarity_19">
( iii ) A mutant Lrp lacking 11 amino acid residues at the C terminus does not form higher-order oligomers , suggesting that the C terminus is involved in subunit association .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11575919" target="_blank">11575919</a>]<br></label>
<input id="dom_similarity_19", type="checkbox">
<div class="DOM_similar">
<li>A mutant Lrp lacking 11 amino acid residues at the C terminus does not form higher-order oligomers , suggesting that the C terminus is involved in subunit association .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11575919" target="_blank">11575919</a>]<br></li></div>
<li>The C-terminal domain of Lrp is involved in self-association A set of deletion mutants of 6HisLrp which lack 11 , 31 , or 61 C-terminal amino acid residues was constructed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11575919" target="_blank">11575919</a>]<br></li>
<li>The C-terminal domain of Lrp Using a mutant with a deletion of 11 C-terminal amino acid residues , we found that the C-terminal domain is required for Lrp to form higher-order oligomers at mM concentrations .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11575919" target="_blank">11575919</a>]<br></li>
<li>Similarly , the Lrp orthologs of these bacteria are completely conserved for amino acids implicated in coregulator recognition ( boxed amino acids in lower panel of Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18405378" target="_blank">18405378</a>]<br></li>
<li>According to this supposition , the amino acid leu-cine stimulates recombination by selectively disrupting the interaction of Lrp with the third region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9515701" target="_blank">9515701</a>]<br></li>
<li>The amino acid sequence of Lrp was also examined for 338 different patterns by using Prosite ; no motifs were uncovered by this analysis other than the HTH motif described later .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8432705" target="_blank">8432705</a>]<br></li>
<li>The amino acid sequence of Lrp was searched for an HTH motif .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8432705" target="_blank">8432705</a>]<br></li>
<li>The Lrp amino acid sequence predicted from it differs from thaf expecfed of the E. coli protein by only two amino acids , and those amino acids are not within fhe DNA-binding domain of Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8483418" target="_blank">8483418</a>]<br></li>
<li>The amino acid residues previously demonstrated to be involved in leucine binding in E. coli Lrp [ 64,97 ] are completely conserved in the P. mirabilis and V. cholerae orthologs ( boxes in lower half of Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18405378" target="_blank">18405378</a>]<br></li>
<li>Thus , it appears that Lrp plays an important role in the maintenance of appropriate in vivo levels of the branched-chain amino acids .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8900118" target="_blank">8900118</a>]<br></li>
<li>Lrp appears to be one factor that coordinates ammonia assimilation and amino acid catabolism in an amino acid -- poor environment .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12730324" target="_blank">12730324</a>]<br></li>
<li>These mutants had changes in the Lrp amino acid sequence between amino acid residues 108 and 149 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8432705" target="_blank">8432705</a>]<br></li>
<li>Lrp has a monomer molecular mass of 18.8 kDa ( 164 amino acid residues ) and a pI of 9.1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11575919" target="_blank">11575919</a>]<br></li>
<li>Lrp transduces metabolic signals in the form of amino acid pool levels , in particular the amino acids L-leucine and L-alanine [ 64 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18405378" target="_blank">18405378</a>]<br></li>
<li>Lrp from these strains had changes in amino acids between residues 76 and 125 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8432705" target="_blank">8432705</a>]<br></li>
				<label class="DOM_collapse_similarity" for="dom_similarity_35">
Lrp has 164 amino acids .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8432705" target="_blank">8432705</a>]<br></label>
<input id="dom_similarity_35", type="checkbox">
<div class="DOM_similar">
<li>Lrp contains 164 amino acids .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1729203" target="_blank">1729203</a>]<br></li></div>
<li>Escherichia coli Lrp of 164 aa in size consists of three functional domains : an N-terminal 40 % domain Received 15 April 2015 ; Accepted 26 May 2015 Downloaded from www.microbiologyresearch.org by G 2015 The Authors .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2162465" target="_blank">2162465</a>]<br></li>
<li>The gray-shaded regions indicate N - and C-terminal sequences that are conserved among enterobacterial Lrp orthologs , and the black-shaded regions indicate substitutions relative to E. coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18405378" target="_blank">18405378</a>]<br></li>
<li>To deﬁne such sites would require biochemical analysis of the interaction of Lrp with the stpA promoter ; we are currently undertaking such studies .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9006049" target="_blank">9006049</a>]<br></li>
<li>Lrp , a dimer containing two identical subunits of mol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8508774" target="_blank">8508774</a>]<br></li>
<li>At the minimally saturating Lrp concentration used in this experiment , only the Lrp-DNA complex containing two Lrp protomers contained enough radioactivity to be suitable for further analysis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8900118" target="_blank">8900118</a>]<br></li>
<li>The substitution of residues Y82 , Y100 and Q128 in AsnC by a phenylalanine and two leucines , respect-ively , in Lrp , changes the size and nature of the pocket such that it would be ideally suited to binding the aliphatic side chain of leucine as opposed to the carboxyamide group of asparagine ( Figure 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16528101" target="_blank">16528101</a>]<br></li>
<li>DNA-binding region Mapping of residues identiﬁed in E.coli Lrp as being involved in DNA-binding [ residues D13 , L34 , L40 , S41 , P44 , L46 , R48 , Y61 , L65 and L70 , ( 50 ) ] onto the structures of AsnC and LrpC reveals two main clusters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16528101" target="_blank">16528101</a>]<br></li>
<li>A structure-based multiple sequence alignment of the AsnC/Lrp family is shown and the residues forming the ligand-binding site are identified .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16528101" target="_blank">16528101</a>]<br></li>
<li>Using a structure-based sequence alignment , mapping of equivalent Lrp residues onto the AsnC structure results in one end of the binding pocket becoming predominantly hydrophobic .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16528101" target="_blank">16528101</a>]<br></li>
<li>The importance of such a small shift in structure is supported by work on an E.coli Lrp variant lacking the ﬁnal 11 C-terminal residues which does not bind to ilvIH operator DNA , despite having a CD trace comparable to that of the wild-type protein ( 49 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16528101" target="_blank">16528101</a>]<br></li>
<li>The remainder of the pocket is largely unchanged with the AsnC TET motif ( residues 136 -- 138 ) mimicked by TRT ( residues 144 -- 146 ) in Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16528101" target="_blank">16528101</a>]<br></li>
<li>Arrows indicate clusters of residues hypersensitive to cleavage by Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9236118" target="_blank">9236118</a>]<br></li>
<li>The similarity between the residues of the first putative helix and turn of Lrp ( residues 29 to 41 ) and the comparison data set was relatively good ( score of 0.76 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8432705" target="_blank">8432705</a>]<br></li>
<li>The algorithm developed by Gamier et al. ( 9 ) predicted that residues 87 to 106 of Lrp form an a helix .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8432705" target="_blank">8432705</a>]<br></li>
<li>The analysis was focused on G residues protected by Lrp from DMS attack .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8071194" target="_blank">8071194</a>]<br></li>
<li>The fact that the S. typhimurium downstream region contains only half motifs but binds Lrp strongly indicates that this is not the case .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8483418" target="_blank">8483418</a>]<br></li>
<li>Furthermore , one or more half motifs spaced about 30 bp from a palindromic motif showed cooperative and strong binding to Lrp , with binding constants in the range 4-20 nanomolar .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8483418" target="_blank">8483418</a>]<br></li>
<li>In summary , a computer analysis indicated that one region of Lrp shows similarity , albeit weak , with regions of proteins known to contain H1TH motifs .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8432705" target="_blank">8432705</a>]<br></li>
<li>For Lrp , two motifs in the database ( 6 leucine ) were incorrectly collapsed into one by motif analysis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17214507" target="_blank">17214507</a>]<br></li>
<li>Thus , it is possible that PapI interacts directly with Lrp and that this interaction is responsible for the observed leucine insensitivity of the pap operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li>
<li>In Lrp , variants found to be insensitive to leucine carried mutations in one of seven positions in the C-terminal domain ( residues L107 , D114 , M124 , L136 , Y147 , V148 and V149 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16528101" target="_blank">16528101</a>]<br></li>
<li>Conceivably , leucine regulates Lrp activities through its effects on the C-terminal domain interactions that lead to Lrp self-association .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11575919" target="_blank">11575919</a>]<br></li>
<li>A previous mutation analysis of Lrp suggested that the Cterminal domain is required for the response to leucine .3 Thus , the C-terminal domain is involved in both Lrp oligomerization and the response to leucine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11575919" target="_blank">11575919</a>]<br></li>
<li>A mutational analysis showed that the N-terminal third of Lrp is responsible for DNA-binding , the middle third mediates transcriptional activation , and the C-terminal third determines the response of Lrp to leucine ( 19 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8144448" target="_blank">8144448</a>]<br></li>
<li>A similar mutational analysis also suggested that the middle third of Lrp is responsible for transcriptional activation and that the C-terminal third determines the response of this protein to leucine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8508774" target="_blank">8508774</a>]<br></li>
<li>A mutafional analysis suggests that Lrp has three functional domains : a DNA-binding domain within the W-terminal third of the profein , a transcription-activation domain within the middle part , and a leucine response domain within the C-terminal third of fhe protein ( Platko , 1991 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8483418" target="_blank">8483418</a>]<br></li>
<li>A mutational analysis suggested that Lrp contains three functional domains of about equivalent sizes : an N-terminal domain containing the site for DNA-binding , a middle domain responsible for transcription-activation , and a C-terminal domain that is required for the response to leucine ( 29 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7665463" target="_blank">7665463</a>]<br></li>
<li>A mutation analysis suggests that Lrp contains three functional domains of about equal sizes : an Nterminal domain containing the site for DNA-binding , a middle domain responsible for transcription-activation , and a C-terminal domain that is required for the response to leucine .3 In members of the Lrp regulon that have been studied extensively , there are multiple Lrp-binding sites near the respective promoters and Lrp binds cooperatively to these sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11575919" target="_blank">11575919</a>]<br></li>
<li>The positions of DNA-binding , activation and leucine response mutants identified in E.coli Lrp are indicated by the symbols plus , asterisk and hash , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16528101" target="_blank">16528101</a>]<br></li>
<li>A recent work by Platko and Calvo ( 20 ) suggests that Lrp has distinct domains for binding to DNA in a presumptive helix-turn-helix motif ( 30 ) , for activating transcription , and for responding to leucine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8071194" target="_blank">8071194</a>]<br></li>
<li>This study suggests that Lrp has separate domains responsible for binding DNA , activating transcription , and responding to leucine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8432705" target="_blank">8432705</a>]<br></li>
<li>In summary , the work reported here suggests that Lrp has domains responsible for binding DNA , activating transcription , and responding to leucine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8432705" target="_blank">8432705</a>]<br></li>
<li>The possibility that Lrp protein-binding to Lrp binding site III inﬂuences ﬁmE transcription termination remains an open question .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16321930" target="_blank">16321930</a>]<br></li>
<li>The topology and tertiary organization of the NikR C domain also matches the regulatory domain of LrpA17 , thought to be representative of the leucine-responsive regulatory protein ( Lrp ) family of transcription factors .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12970756" target="_blank">12970756</a>]<br></li>
<li>The autoradiogram in Fig. 6A shows that in the presence of increasing concentrations of Lrp ( 5.0 3 10210 to 3.25 3 1028 M ) , three regions of DNase I protection and two intervening regions of enhancement on the nontranscribed DNA strand are observed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8900118" target="_blank">8900118</a>]<br></li>
<li>The autoradiogram in Fig. 8B shows that in the presence of a minimally saturating concentration of Lrp ( 2.6 3 1028 M ) , 2 guanine residues separated by 9 base pairs ( nearly one complete turn of the DNA helix ) at base pair positions 229 and 238 on the transcribed DNA strand were not alkylated by DMS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8900118" target="_blank">8900118</a>]<br></li>
<li>It also shows a 50 offset of six nucleotides on each strand characteristic of dimeric proteins binding in the major groove and is consistent with the model of Lrp as a putative helix-turn-helix binding protein ( Platko & Calvo , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9236118" target="_blank">9236118</a>]<br></li>
<li>The results of DNase I footprinting experiments suggest that Lrp binding to this region facilitates the formation of a higher-order nucleoprotein structure .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8508774" target="_blank">8508774</a>]<br></li>
<li>To date , DNase I footprinting experiments have not proven very useful in defining binding sites because these footprints extend over many dozens of base pairs without defining regions that contact Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li>
<li>Accordingly , studies were carried out to investigate Lrp interactions with the ﬁm switch further and to determine the function of the uncharacterized Lrp binding region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9515701" target="_blank">9515701</a>]<br></li>
<li>D. EMSA of Lrp binding to the argO control region in the presence of various amino acids ( 7 mM ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19906180" target="_blank">19906180</a>]<br></li>
<li>DNase I protection studies further showed that Lrp binding in the dad promoter-regulatory region was altered by these amino acids .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955407" target="_blank">8955407</a>]<br></li>
<li>Using MPE?Fe ( II ) footprinting , we identi ® ed 11 Lrp binding sites located from ÿ275 to 90 in the dad pro-moter-regulatory region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10216857" target="_blank">10216857</a>]<br></li>
<li>coli , Lrp interacts more strongly with an upsfream region containing two binding sites than with a downstream region containing four binding sites ( Wang and Calvo , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8483418" target="_blank">8483418</a>]<br></li>
<li>Through both in vitro and in vivo studies we show that all three Lrp binding sites , and their proper phasing with one another , are required for optimal DNA-binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9236118" target="_blank">9236118</a>]<br></li>
<li>Two other possibilities that emphasize phasing and potential interaction between the upstream and downstream Lrp binding sites are discussed below .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12218014" target="_blank">12218014</a>]<br></li>
<li>Although in the presence of leucine some in vivo interaction of Lrp with the ilvIH regulatory region may still occur , binding to sites 1 , 3 , 4 , and 5 is impaired .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8071194" target="_blank">8071194</a>]<br></li>
<li>All of these mutations were within the recognition helix portion of the putative HTH , suggesting that the specific binding of Lrp to ilvIH DNA may require particular amino acids which are not normally present at those positions within the HTH regions of other DNA-binding proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8432705" target="_blank">8432705</a>]<br></li>
<li>These mutations could be in regions that are important for stabilizing the putative HTH region , or they could define residues that are important to a structure , other than an HTH domain , that is involved in the binding of Lrp to ilvIH DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8432705" target="_blank">8432705</a>]<br></li>
<li>These mutations affected a predicted DNA-binding domain of Lrp and mutants were defective both in activation of ilvIH expression and in repression of lysU expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7603430" target="_blank">7603430</a>]<br></li>
<li>Lrp binds DNA and has a helix-turn-helix motif at the N-terminal region ; it exists in solution as a dimer .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7603430" target="_blank">7603430</a>]<br></li>
<li>The Nterminal part of Lrp contains a helix-turn-helix motif ( Willins et al. , 1991 ) and a mutational analysis suggested that this region is responsible for DNA-binding ( Platko and Calvo , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8508774" target="_blank">8508774</a>]<br></li>
<li>Together , these results support the sequence-derived inference that Lrp is a helix-turn-helix DNA-binding protein ( 38 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8900118" target="_blank">8900118</a>]<br></li>
<li>HTH motifs consist of a short a-helix Interaction of Lrp with DNA Consensus sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li>
<li>The three sites , de ® ned in this way , are centered on the same face of the DNA helix allowing for facile Lrp/Lrp interactions , as might be expected with cooperative binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9236118" target="_blank">9236118</a>]<br></li>
<li>The regions of Lrp-induced DNase I sensitivity suggest that Lrp alters the structure of the DNA helix in and around its primary and secondary binding sites ( Fig. 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8900118" target="_blank">8900118</a>]<br></li>
<li>The fact the mutations that markedly reduce DNA-binding are within or near the putative HTH region is at least consistent with the idea that Lrp has a functional H1TH motif .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8432705" target="_blank">8432705</a>]<br></li>
<li>The binding in vivo of Lrp should be interfered by competitive binding by other DNA-binding proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2162465" target="_blank">2162465</a>]<br></li>
<li>It is , of course , possible that other DNA-binding proteins in the cell interact with Lrp and/or with the ompC-micF intergenic region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8002608" target="_blank">8002608</a>]<br></li>
<li>Although the functional oligomeric state of Lrp is an octamer or hexadecamer , the interaction-with-DNA is med-iated by individual homo-dimers and thus multiple DNA-binding domains are present .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16879983" target="_blank">16879983</a>]<br></li>
<li>Although the GATC box regions each encompass Lrp binding domains , methylation protection of clone 3-1 was not dependent on Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8195106" target="_blank">8195106</a>]<br></li>
<li>Next , we carried out a more detailed analysis of Lrp binding to these two regions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11985712" target="_blank">11985712</a>]<br></li>
<li>Alternatively , it is possible that Lrp forms protein -- protein contacts in complex 2L to produce the same DNA to protein stoichiometry as complex 1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9515701" target="_blank">9515701</a>]<br></li>
<li>The observation that symmetrically spaced guanine residues on opposite DNA strands in the consensus-like site are protected from alkylation by DMS demonstrates that the homodimeric Lrp molecule forms symmetrical half-site contacts in this palindromic , con-sensus-like site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8900118" target="_blank">8900118</a>]<br></li>
<li>Since DMS alkylates the N-7 position of guanine residues exposed in the major groove of the DNA helix , this result shows that the homodimeric Lrp molecule binds to adjacent major grooves positioned on the same face of the DNA helix .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8900118" target="_blank">8900118</a>]<br></li>
<li>Since DMS alkylates the N7 position of guanines in the major groove of the DNA helix the DMS protection patterns also demonstrate that , like other homodimeric helix-turn-helix DNA-binding proteins , the Lrp protomer binds in adjacent major grooves on the same face of the DNA helix .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8900118" target="_blank">8900118</a>]<br></li>
<li>Chemical , enzymatic , and alkylation protection and interference footprinting experiments demonstrate that the Lrp homodimer contacts the DNA helix at symmetrical half-sites present in adjacent major grooves and that the primary and secondary binding sites are separated by one helical turn and aligned along the same face of the DNA helix .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8900118" target="_blank">8900118</a>]<br></li>
<li>Thus , binding of Lrp to the consensus-like site appears to involve symmetrical half-site contacts and interactions with adjacent major grooves of the DNA helix .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8900118" target="_blank">8900118</a>]<br></li>
<li>The interference observed with the modification of the adenine residue at nucleotide position 235 can be interpreted either as a site of direct contact with Lrp in the minor groove of the DNA helix or as a residue the modification of which alters the local conformation of the DNA helix .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8900118" target="_blank">8900118</a>]<br></li>
<li>Since the alkylation protection experiment identified the guanine at position 228 as a protected residue , it is likely that direct contacts between Lrp and this guanine residue are important for high affinity binding of Lrp to site 1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8900118" target="_blank">8900118</a>]<br></li>
<li>In the presence of Lrp , a 30-bp region of DNase I protection from base pair position 220 -- 250 covering the Lrp , consensus-like DNA-binding site ( base pair position 226 -- 240 ) , was observed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8900118" target="_blank">8900118</a>]<br></li>
				<label class="DOM_collapse_similarity" for="dom_similarity_108">
... 4 Does Lrp interact with other proteins ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></label>
<input id="dom_similarity_108", type="checkbox">
<div class="DOM_similar">
<li>Does Lrp interact with other proteins ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li></div>
<li>Abbreviations used : OmpF , outer-membrane-protein F ; OmpC , outer-membrane-protein C ; Lrp , leucine-responsive protein ; NO , nitric oxide ; IHF , integration host factor ; HTH , helix-turn-helix .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11601842" target="_blank">11601842</a>]<br></li>
<li>We suggest that Lrp may also interact in this way , perhaps with CAP , integration host factor , and others .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1532173" target="_blank">1532173</a>]<br></li>
<li>Activation from a distance : roles of Lrp and integration host factor in transcriptional activation of gltBDF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12218014" target="_blank">12218014</a>]<br></li>
<li>In contrast , the three Lrp orthologs gave equivalent activation of PgltB ( Figs. 4C , F ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18405378" target="_blank">18405378</a>]<br></li>
<li>In general terms , we found that Lrp behaves in similar ways in the three tested species .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18405378" target="_blank">18405378</a>]<br></li>
<li>Is Lrp sufficient for the activation of transcription in vitro , or are additional factors necessary ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1447135" target="_blank">1447135</a>]<br></li>
<li>However , further studies are required to determine if Lrp 's role in the activation of gcv is structural or if interactions with RNA polymerase ( RNAP ) or other regulatory proteins occur .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9733689" target="_blank">9733689</a>]<br></li>
<li>Therefore , S we conclude that , in contrast to csiD activation by H-NS , activation by Lrp is not speciﬁc for either s - or S s - containing RNA polymerase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11532138" target="_blank">11532138</a>]<br></li>
<li>Bending of the DNA by Lrp and GcvA allows GcvA bound to sites 32 to interact with the α-carboxy-terminal domain of RNA polymerase , permitting stimulation of RNAP , and activation of transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12101307" target="_blank">12101307</a>]<br></li>
<li>On the other hand , the Lrp binding sites in dad and pap that are required for activation are located too far upstream to allow direct contact of Lrp with RNA polymerase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10216857" target="_blank">10216857</a>]<br></li>
<li>So far , activation of transcription by direct contacts between Lrp and RNA polymerase has not been documented for any promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11395454" target="_blank">11395454</a>]<br></li>
				</div></ul><br>
			<label class="collapse_EVO" for="_3"><font size="5px">Lrp evolution</font></label>
				<input id="_3" type="checkbox">
				<div class="EVO"><br><ul><li>FEBS 29827 The yeaS ( leuE ) gene of Escherichia coli encodes an exporter of leucine , and the Lrp protein regulates its expression Ekaterina A. Kutukova , Vitaliy A. Livshits * , Irina P. Altman , Leonid R. Ptitsyn , Michael H. Zyiatdinov , Irina L. Tokmakova , Natalia P. Zakataeva Ajinomoto-Genetika Research Institute , Laboratory N2 , 1-st Dorozhny Proezd , b. 1 , Moscow 117545 , Russia Received 29 April 2005 ; revised 1 July 2005 ; accepted 12 July 2005 Available online 10 August 2005 Edited by Felix Wieland Abstract Overexpression of the yeaS gene encoding a protein belonging to the RhtB transporter family conferred upon cells resistance to glycyl-L-leucine , leucine analogues , several amino acids and their analogues .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16098526" target="_blank">16098526</a>]<br></li>
<li>Interestingly , it has 36 % identity to Lrp of Sinorhizobium meliloti , which is a leucine-responsive regulatory protein used for lipopolysaccharide ( LPS ) synthesis ( Lagares et al. 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14727089" target="_blank">14727089</a>]<br></li>
<li>The operon belongs to the Lrp ( leucine-respon-sive regulatory protein ) regulon [ 111 , and is positively regulated by the Lrp protein 1121 , apparently by means of a mechanism involving Lrp-RNA polymerase interaction [ 13 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7557316" target="_blank">7557316</a>]<br></li>
<li>The topology and tertiary organization of the NikR C domain also matches the regulatory domain of LrpA17 , thought to be representative of the leucine-responsive regulatory protein ( Lrp ) family of transcription factors .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12970756" target="_blank">12970756</a>]<br></li>
<li>As Lrp-regu - Introduction Leucine-responsive regulatory protein ( Lrp ) belongs to the widely distributed Lrp -- AsnC family of small , basic transcription factors .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2162465" target="_blank">2162465</a>]<br></li>
<li>Members of the family influence cellular metabolism in both a global ( Lrp ) and specific ( AsnC ) manner , often in response to exogenous amino acid effectors .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16528101" target="_blank">16528101</a>]<br></li>
<li>Lrp has some amino acid sequence similarity to AsnC , a protein that regulates asnA expression ( Kolling , R. , and Lother , H. ( 1985 ) J. Bacteriol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2040596" target="_blank">2040596</a>]<br></li>
<li>The amino acid sequences of Lrp and AsnC are 25 % identical , and at least another 25 % show close similarity ( 168 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li>
<li>E.coli AsnC shows notable sequence identity ( 25 % ) with Lrp ( Figure 1 ) , which has resulted in them being classiﬁed as a distinct evolutionary protein family ( 21 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16528101" target="_blank">16528101</a>]<br></li>
<li>It shares 34 and 25 % identity with E.coli Lrp and AsnC , respectively , and has reported functionality in both sporulation and amino acid metabolism ( 24 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16528101" target="_blank">16528101</a>]<br></li>
<li>The ybaO locus encodes a predicted protein of 181 amino acids with strong similarity to E. coli leucine-responsive regulatory protein ( Lrp ; 33 % identity over 150 residues ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10850996" target="_blank">10850996</a>]<br></li>
<li>The ybaO-encoded protein is most closely related to another Lrp homolog : the glutamate uptake regulatory protein ( Grp ) of Zymomonas mobilis ( 44 % identity over 150 residues ) ( 26 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10850996" target="_blank">10850996</a>]<br></li>
<li>cReported TBLASTN identity to Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21350637" target="_blank">21350637</a>]<br></li>
<li>The Lrp ortholog in P. mirabilis differs from that in E. coli by only 4/164 amino acids ( 98 % identity ) , while Lrp from V. cholerae shows 92 % identity to E. coli Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18405378" target="_blank">18405378</a>]<br></li>
<li>Members of the Lrp / AsnC family typically have a molecular mass of 15 kDa but populate a range of multimeric species in solution , which include dimers , tetramers , octamers and hexadecamers ( 4 -- 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16528101" target="_blank">16528101</a>]<br></li>
<li>In contrast to most other family members , which are speciﬁc regulators , E. coli Lrp is a global regulator that affects the expression of numerous genes and operons , many of which belong to amino acid metabolic pathways ( Newman and Lin , 1995 ; Hung et al. , 2002 ; Tani et al. , 2002 ; Cho et al. , 2008 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19906180" target="_blank">19906180</a>]<br></li>
<li>This is perhaps not surprising given reports that members of the family , such as Lrp , LrpA , LrpC and PutR bind to and induce bending in DNA ( 6,7,11,25,26 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16528101" target="_blank">16528101</a>]<br></li>
<li>The apparent cooperativity of Lrp binding to the low-affinity sites ( 3 + 4 + 5 ) upstream of ilvIH was also increased in th Genes That Are Evolutionarily Related to Lrp Affinity-purified antibodies against Lrp reacted with Lrp-like proteins in a variety of members of the family Enterobacteriaceae , namely Serratia strain 8011 , Enterobacter aerogenes , Kiebsiella aerogenes , and S. typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li>
<li>Perhaps the requirement for Lrp , with binding regions from 292 to 2229 ( 30 ) , necessitates the atypical location and may represent a novel mode of regulation for this LysR family member .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7665470" target="_blank">7665470</a>]<br></li>
<li>The best-characterized member of the family is Escherichia coli Lrp , which controls a global regulon encompassing at least 10 % of all E.coli genes ( 18 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16528101" target="_blank">16528101</a>]<br></li>
<li>The Lrp family of transcriptional regulators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2162465" target="_blank">2162465</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21685294" target="_blank">21685294</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16879983" target="_blank">16879983</a>]<br></li>
				<label class="EVO_collapse_similarity" for="evo_similarity_22">
Brinkman , A.B. , Ettema , T.J.G. , de Vos , W.M. , and van der Oost , J. ( 2003 ) The Lrp family of transcription regulators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19906180" target="_blank">19906180</a>]<br></label>
<input id="evo_similarity_22", type="checkbox">
<div class="EVO_similar">
<li>Brinkman , A.B. , Ettema , T.J. , De Vos , W.M. , and van der Oost , J. ( 2003 ) The Lrp family of transcriptional regulators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16238633" target="_blank">16238633</a>]<br></li></div>
<li>Lrp is not related to the large families of bacterial regulatory proteins such as the LysR family or the two-component family of bacterial regulatory proteins , nor does it show relatedness to other well-known regulatory proteins such as Crp or Fnr or to abundant DNA-binding proteins such as integration host factor or the histone-like regulatory proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li>
<li>Examples are regulatory proteins in the LysR family ( 34 , 48 ) , H-NS ( 29 , 61 ) , and Lrp ( 41 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21685294" target="_blank">21685294</a>]<br></li>
<li>The size of such regulatory proteins is relatively small ( LysR family , 30 kDa ; H-NS , 16 kDa ; Lrp , 15 kDa ) compared to the length of their protected DNA fragments .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21685294" target="_blank">21685294</a>]<br></li>
<li>Most of the other Lrp sites in dad show little similarity to this consensus or to each other ( Fig. 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10216857" target="_blank">10216857</a>]<br></li>
<li>The consensus binding sequences we used were the following : 5 WYAGGMWWWWDYTW YWWW for binding of GadE , GadX , and GadW ( gadbox [ 26 , 43 ] ) ; 5 YAGHA WATTWTDCTR for Lrp ( 11 ) ; and 5 CTGTTCATAT for TorR ( 2 , 40 ) ; where D is A , T , or G ; H is A , C , or T ; M is A or C ; R is A or G ; W is A or T ; and Y is C or T. Only sequences with 70 % or higher identity to the consensus sequences were selected .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18083817" target="_blank">18083817</a>]<br></li>
				</div></ul><br>
			<label class="collapse_RP" for="_4"><font size="5px">Processes regulated by Lrp</font></label>
				<input id="_4" type="checkbox">
				<div class="RP"><br><ul><li>sgm P1 : IK H REITZER Lrp ... 1 Cyclic AMP ... 1 EINtr-NPr-EIINtr ... 1 UANOSINE TETRAPHOSPHATE , AMINO ACID SYNTHESIS , AND AMMONIA ASSIMILATION ... 1 ONCLUDING REMARKS : INTERACTIONS BETWEEN GLOBAL REGULATORS ... 1 67 68 68 G 70 C 71 PHYSIOLOGICAL CONTEXT Nitrogen Sources monia Assimilation and Intracellular Nitrogen Donors Am If GS assimilates ammonia , then glutamate synthase ( GOGAT ) catalyzes glutamate-formation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12730324" target="_blank">12730324</a>]<br></li>
<li>Genes induced at the onset of stationary phase with s - dependent expression as identiﬁed by microarray analyses Functional Mean WT d Ratio classa valueb ( WT/rpoS ) Regulation SDc Gene Gene product Starvation and stress response genes dpsS DNA-protection protein , bacterial ferritin IHFS , OxyRS , rssBTC , MMC , PQ , NaSal , Lrp , BG , DHCP , OS FisS , NaSal , Lrp , OS CRPS , LrpS , IHFS , H-NSS , arcBTC , MMC , Lrp , Rcs , BG , DHCP ompR/envZTC , CD uvrYTC , arcBTC SSR 1,597 2.5 0.4 osmES osmYS Stress-inducible outer membrane lipoprotein Hyperosmotically inducible periplasmic protein Induced by phosphate starvation Superoxide dismulase precursor ( Cu-Zn ) SSR SSR 1,728 424 9.6 4.4 2.1 1.3 656 459 psiF sodCS 4.8 2.7 0.4 0.5 SSR Protein synthesis rpsVS infA Amino acid transport and metabolism artP artI 5.2 2.8 FISS , IHFS , ppGppS 2.7 0.8 30S ribosomal subunit protein S22 Protein chain initiation factor IF-1 3,466 1,011 PS PS Arginine transport , ATP-binding component Arginine transport , periplasmic binding protein L-Asparagine permease Dihydroxyacid dehydratase Tryptophanase AATM AATM 714 346 2.6 0.7 PQ , NaSal , Lrp 3.8 1.9 PQ , Lrp ansP ilvD tnaA AATM AATM AATM 572 364 DHCP ompR/envZTC 2.5 4.8 0.4 2.1 4.3 phoQTC , ompR/envZTC , CRP , uvrYTC , arcATC , arcBTC , rcsBTC , citABTC , MMC , PQ ( ilvG : Lrp ) NaSal , Lrp , BG , DHCP , OS 13.7 7,808 wrbAS Flavodoxin-like protein ; binds the trp repressor SSR 364 5.0 2.0 Iron uptake and storage bfr entD Carbohydrate metabolism crr Bacterioferrin , iron storage homoprotein Enterochelin synthetase , component D IUS IUS 905 508 5.1 2.1 DHCP , ryhB RNA 4.2 0.5 acrBTC Phosphotransferase system , glucose-speciﬁc IIA component Fructose 1,6-bisphosphate aldolase ( b2097 ) Glyoxylate-induced protein Putative oxidase , glycolate utilization ( b2977 ) Glucosamine-6-phosphate deaminase Phosphofructokinase-2 Transaldolase A CHM 443 2.7 0.6 CD 1,264 1,097 CHM 4.6 3.6 3.0 2.6 2.5 2.9 1.4 1.0 1.7 0.6 2 1.8 MMC , Lrp kdpABCDETC fbaB gip uvrYTC , arcATC , arcBTC ompR/envZTC , arcBTC MMC , CD , DHCP rssBTC , MMC , DHCP , CreBC CHM CHM CHM CHM glcG nagB pfkB talA Miscellaneous function or unknown function deoD moeB msyB yafN yahOS ybgA 3.0 1.2 4.0 0.1 2.2 1.7 4.8 3.4 6.1 2.6 4.5 4.5 349 Purine nucleoside phosphorylase Molybdopterin-biosynthesis NM CB 1,100 2,038 CD , Lrp , DHCP MMC , GadX Lrp Membrane protein , protein export ( b1051 ) Hypothetical ( adherence ) protein ( b0232 ) Hypothetical protein ( b0329 ) Hypothetical ( inner membrane ) protein ( b0707 ) Putative enzyme ( b0865 ) Hypothetical protein , bioﬁlm dependent ( b1481 ) Hypothetical protein ( b1724 ) Hypothetical protein ( b2427 ) Putative dehydrogenase protein ( b2660 ) Putative immunogenic protein ( b2922 ) Putative transport periplasmic protein ( b3020 ) Hypothetical ( outer membrane ) protein ( b3024 ) Hypothetical ( regulatory ) protein ( b3555 ) Hypothetical protein ( b4045 ) Hypothetical ( membrane ) protein ( b1582 ) Hypothetical ( periplasmic ) protein ( b3097 ) Hypothetical protein ( b4217 ) MF ( MF ) rssBTC , baeSRTC , atoSCTC , Lrp ybjP yddX 2.5 1.4 4.2 4.2 1,317 376 ( EF ) rssBTC , OS ompR/envZTC ydiZ yfeT ygaF yggE ygiS 1.7 0.9 0.5 1.7 0.8 3.1 3.5 2.8 4.8 2.7 436 rssBTC , NarXLTC rssBTC , MMC ; Rcs , phoQP ( EF ) ( MF ) rssBTC , citABTC , arcATC , NaSal ( MF ) ygiW 1.9 4.0 892 rssBTC , ntrBCTC , NaSal yiaG yjbJ ynfA yqjC ytfK 0.5 1.2 1.1 1.3 3.0 2.7 4.5 3.5 5.9 3.9 1,227 601 3,334 1,121 556 ( MF ) PQ , Fur , BG a SSR , starvation and stress response ; PS , protein-synthesis ; AATM , amino-acid-transport and metabolism ; IUS , iron uptake and storage ; CHM , carbohydrate-metabolism ; NM , nucleoside-metabolism ; CB , cofactor-biosynthesis ; MF , membrane function ; EF , extracellular function .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15489429" target="_blank">15489429</a>]<br></li>
<li>Functionb Other regulator ( s ) Reference ( s ) Rich Minimal Medium Medium No 15 12 Yes 24 18 No 6 6 ybaY gabP ugpE Unknown GABA-transporter protein b0453 b2663 b3451 GabC ( ) , Nac ( ) PhoBR ( ) 55 , 66 , 93 15 G-3-P transporter membrane bound component ATP binding protein of G-3-P transporter Putative binding protein of ABC transporter Hypothetical ATP-binding protein of ABC transporter Beta subunit of nitrate reductase-II Subunit of cytochrome bd-II oxidase Lysine decarboxylase Induced by alkylating agents ; protects DNA from alkylation damage Hydroperoxidase II Universal stress protein B ; required for ethanol resistance Periplasmic protein induced by high osmolarity ; function unknown Multidrug efﬂux pump No 20 9 Yes 6 2 No 28 21 ugpC ydcS yehX b3450 b1440 b2129 PhoBR ( ) Nac ( ) 15 93 narY appB IdcC aidB b1467 b0979 b0186 b4187 13 23 Yes Yes Yes Yes 10 25 7 36 2 7 AppR ( ) 47 , 79 45 , 46 11 33 Lrp ( ) Yes 40 66 Yes 4 8 katE uspB b1732 b3494 74 26 Yes 35 64 osmY Lrp ( ) , Crp ( ) , IHF ( ) EvgA ( ) , Lrp ( ) b4376 20 , 90 , 91 No 31 yhiV b3514 56 , 57 , 74 , 78 56 , 57 , 78 25 72 , 74 12 77 24 4 yhiU mltB otsA ecnB argH aroM Multidrug efﬂux pump EvgA ( ) b3513 b2701 b1896 8 2 Yes No Yes Yes No No 39 2 31 23 2 15 Membrane-bound lytic transglycosylase B Trehalose phosphate synthase Bacteriolytic lipoprotein entericidin B Argino-succinate lyase Unknown ; may play a role in aromatic-amino-acid biosynthesis Putative lipase 2,5-diketo-D-gluconic acid reductase A ; catalyzes the conversion of 2,5-KGDR to 2-KLG Aldehyde dehydrogenase B ; converts lactaldehyde and NAD to lactate and NADH Succinate-semialdehyde dehydrogenase Membrane-bound subunit of carbonphosphorus lyase complex Transaldolase A ; converts Sed-7-P and Gly-3-P to Fru-6-P and Ery-4-P Putative glutathione S-transferase Putative dehydrogenase Unknown Unknown Unknown Unknown Unknown Unknown Unknown Unknown Unknown Unknown Unknown Unknown Unknown Unknown Unknown Unknown Unknown Lrp ( ) 57 74 5 78 ( ) EnvZ/OmpR ArgR ( ) TyrR ( ) b3960 b0390 No 15 No 8 yhjY yqhE b3548 b3012 9 92 11 aldB 22 Yes 21 b3588 74 , 87 gabD phnP B2661 B4092 16 8 No No GabC ( ) , Nac ( ) PhnF and PhnO ( ) 55 , 93 51 13 2 talA B2464 26 20 No CreBC ( ) 7 , 70 yfcG yliI yjbJ yjbE ygaU ygdI ygaF yjgR ydaM ydcK yhiN yebF yjiN yeaG yodC ybiO yhjG yphA yhjD B2302 B0837 B4045 B4026 B2665 B2809 B2660 B4263 B1341 B1428 B3492 B1847 B4336 B1783 B1957 B0808 B3524 B2543 B3522 52 37 37 17 22 22 4 7 3 6 16 15 5 41 4 13 41 13 17 24 5 No Yes No No No No No No No No No No No Yes No No No No Yes SdiA ( ) 84 66 179 58 12 5 7 3 23 9 7 25 6 10 17 26 15 18 34 Lrp ( ) 74 63 , 74 Lrp ( ) Lrp ( ) 63 a Genes were identiﬁed using the BLAST algorithm ( 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15576800" target="_blank">15576800</a>]<br></li>
<li>91.5 78 N/N glpTQ 0.46 0.28 0.30 0.64 0.35 0.68 0.34 GlpR-repressed , CRP-activated 82.5 5 Y/Y 79 N/N Y/Y ArcA - and NarL-activated ArcA-activated 80 ndh 81 Y/Y 0.30 0.99 6.60 1.21 6.16 1.18 5.80 ArcA-regulated acnA nrdAB hmpA Aconitate hydrase 1 Y/Y N/N Y/Y CRP-activated , ArcA-repressed Proposed anti-activation in stationary phase 2.58 4.16 75.91 1.43 2.23 20.42 2.51 3.79 3.02 1.36 5.99 1.68 2.02 6.32 2.04 1.18 1.73 29.89 2.44 3.04 149.7 40.5 82 75 Ribonucleoside diphosphate reductase 1 Flavohemoglobin H alf-site 5.5 37 , 85 Half-sites Fur-repressed 86 N/N IHF - and ArcA-activated ModE-activated ArcA-repressed N/N Y/Y N/N 57 , 87 83 88 83 , 84 Y/Y 89 Y/N IHF - , Lrp - , and YeiL-activated 22 N/N NorR-activated , NarL/NarP-repressed 73 N/Y IHF-repressed nitrite ) and strain ( wild type or fnr ) compared with transcript TABLE 3 Operons that display FNR regulation and that have putative FNR-binding sites upstream Ratiosa O2 O Ø 2 2 NO3 NO3 ] NO2 NO2 WT fnr fnr WT fnr WT fnr Location of putative TS FNR FNR site ( s ) b determined site IDsc Operon Function of operon Putative FNR-activated operons caiTABCDE Carnitine metabolism fixABCX Carnitine metabolism allB Allantoinase ybcW Hypothetical protein fes-entF-fepE Ferric enterobactin-synthesis and transport Citrate lyase Hypothetical protein Hypothetical protein Putative peptidase T Hypothetical proteins Outer membrane protein ; colicin S4 receptor ydcA Hypothetical protein ydfZ Hypothetical protein ynfEFGHI Me2SO reductase paralogue ynfK Hypothetical protein ydhYVWXU Putative oxidoreductase ydjXYZynjABCDE Hypothetical proteins pykA Pyruvate kinase II tar tap Chemotaxis cheRBYZ yecR Hypothetical protein yedM Hypothetical protein ackA Acetate kinase yfcC Putative S-transferase upp-uraA Uracil salvage yqjI Hypothetical protein aer Aerotaxis sensor receptor uxaCA Glucuronate degradation tdcABCDEFG Threonine degradation garPLRK Glucarate degradation yhbTS Hypothetical proteins yhbUV Hypothetical protein and putative collagenase Hypothetical protein Maltose degradation Low affinity phosphate-transport Putative 2-component regulator PTS-system Catalase Hypothetical proteins Hypothetical protein Transcriptional activator of cad operon fimBEAICDFGH Type 1 fimbriae yjiD Hypothetical protein tsr Chemotaxis protein yjjIW Hypothetical protein Putative FNR-repressed operons pdhR-aceEF-lpdA Pyruvate dehydrogenase and regulator Glucose dehydrogenase Hypothetical protein Cytochrome o ubiquinol oxidase Rhs element protein / hypotheticals Hypothetical protein Outer membrane protein X Putative cysteine hydrolase 41.5 and 90.5 197.5 185.5 142.5 and 21.5 143.5 Mapped Mapped Predicted Predicted Mapped Y/N Y/N Y/N Y/N Y/N 0.79 1.35 1.16 0.29 1.39 0.94 1.31 0.77 0.29 1.32 0.51 0.40 0.57 0.44 0.30 0.82 0.82 1.20 0.66 0.21 0.39 0.56 0.56 0.28 0.16 0.74 0.81 0.84 0.89 0.29 0.60 0.64 0.41 0.33 0.14 32.5 30.5 Predicted Predicted Y/Y Y/N Y/Y Y/N Y/Y Y/Y citCDEFXG ycbJ yccM pepT ymfED ompW 0.19 0.23 0.27 0.33 0.46 0.09 0.19 0.30 0.42 0.59 0.21 0.06 0.37 0.41 0.43 0.39 0.69 0.27 0.49 0.39 1.64 0.66 1.49 0.03 0.30 0.33 0.44 0.35 0.60 0.02 0.48 0.74 2.80 0.85 0.91 0.53 0.34 0.36 0.36 0.36 0.68 0.02 25.5 83.5 124.5 2 putative sites Predicted Predicted Predicted No predicted ts 0.46 0.01 0.06 0.21 0.42 0.49 0.02 0.05 0.15 0.14 43.5 35.5 64.5 29.5 167.5 Predicted Predicted Predicted Predicted Predicted Y/N N/Y Y/Y N/Y Y/Y 0.40 0.01 0.03 0.11 0.21 0.66 0.52 0.03 0.50 0.23 0.39 0.01 0.02 0.09 0.25 0.84 0.73 0.25 0.77 0.31 0.45 0.01 0.03 0.09 0.16 2.81 0.82 0.96 1.23 1.19 0.54 0.48 0.48 0.51 1.78 0.48 0.39 0.18 0.42 0.39 0.33 0.25 0.78 0.39 0.44 0.57 0.42 0.91 1.63 0.55 0.24 0.29 0.10 0.61 1.15 0.88 0.26 0.80 1.39 0.80 0.31 0.23 0.34 0.71 1.91 79.5 117.5 Predicted Predicted Y/N Y/N Y/N Y/Y Y/N Y/N Y/N Y/Y Y/N Y/N 0.53 0.42 0.28 0.55 0.42 0.28 0.16 0.42 0.40 0.62 0.63 1.01 0.53 0.67 0.91 0.73 0.84 0.93 1.38 0.64 0.46 0.43 0.05 0.17 0.29 0.27 0.23 0.31 0.47 0.27 160.5 113.5 14.5 1 putative site 18.5 61.5 150.5 and 200.5 51.5 Predicted Predicted Predicted No predicted ts Predicted Mapped Predicted Mapped 0.57 0.44 0.60 0.37 0.24 1.12 0.81 1.30 0.80 0.97 0.42 0.28 0.46 0.35 0.34 0.64 0.60 0.59 0.58 0.88 0.47 0.45 0.49 0.34 0.56 1.14 1.04 1.67 0.63 0.71 0.85 0.75 0.99 0.53 0.46 58.5 Predicted Mapped Predicted Predicted Predicted Y/N Y/Y Y/N Y/N Y/N 10.5 and 148.5 154.5 76.5 45.5 yhcC 0.22 0.62 0.37 0.62 0.84 0.85 0.93 0.82 0.21 0.54 0.34 0.71 0.68 0.73 0.57 0.75 0.32 0.49 0.48 0.45 0.38 4.29 0.70 0.67 0.28 1.54 0.75 0.50 Predicted Mapped Predicted Predicted 70.5 127.5 66.5 37.5 Y/N Y/N Y/Y Y/N malPQ pitA yieNM frvABXR katG yjdAZ yjdK cadC 0.34 0.27 0.04 0.20 0.15 0.43 0.94 0.50 0.65 0.25 0.37 0.23 0.22 0.16 0.11 0.39 0.73 1.36 0.78 0.29 0.32 0.21 0.15 0.31 0.14 1.45 0.32 0.76 0.08 1.54 1.82 0.22 1.19 0.09 1.98 Predicted Mapped Predicted Predicted Mapped 11.5 Y/N Y/Y Y/N Y/Y Y/Y 96.5 and 112.5 50.5 and 56.5 14.5 156.5 0.40 0.59 0.05 0.06 0.86 1.06 0.62 0.70 0.70 0.37 0.19 0.04 1.06 0.67 1.22 0.62 0.93 0.50 0.16 0.06 0.35 0.71 0.78 0.11 0.50 0.44 1.55 0.11 Y/Y Y/Y Y/N Y/N Mapped Predicted Predicted Predicted 71.5 148.5 80.5 56.5 3.30 0.81 2.97 1.05 2.64 0.74 0.63 Y/Y Mapped 50.5 Y/N Y/N Y/Y gcd yaiB 1.52 1.96 43.04 0.75 1.09 1.12 1.85 2.04 61.33 0.91 1.04 13.07 1.07 2.30 60.68 5.27 2.48 33.75 10.29 1.54 74.26 Mapped Predicted Mapped 31.5 wrt p1 , 15.5 wrt p2 62.5 192.5 E cyoABCD rhsDybbCylbH-ybbDb0501 ybiJ ompX 2.05 2.17 1.98 0.90 0.42 1.44 1.13 2.46 1.94 0.55 1.05 1.61 1.52 1.44 2.15 0.72 1.68 1.70 0.49 2.84 1.81 Predicted Predicted Predicted 61.5 and 128.5 2.5 and 62.5 3.5 Y/Y Y/N Y/N ycaC Y/Y 134.5 Predicted 0.79 3.01 1.73 2.54 1.41 2.99 1.17 2 , part B , be confirmed ; these were cydDC , hlyE , adhE , glpTQ , and arcA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16377617" target="_blank">16377617</a>]<br></li>
<li>of adenyl dinucleotides , which have been proposed to act as modulators of the heat shock response and stress response ; induced by temp and ethanol ; negatively regulated by Lrp but positively regulated by htpR Acetylates maltose and other sugars Efﬂux transporter for macrolide antibiotics Involved in the high-afﬁnity maltose membrane transport system malEFGK and is the initial receptor for the active transport of chemotaxis toward maltooligosaccharides ; induced by high pH during aerobic or anaerobic growth ; increases in the physiological short-term adaptation to glucose limitation but reduces at the acnA or/and acnB mutants TABLE 2 -- Continued Theoreticalc LldD P33232 LolA P61316 L-Lactate dehydrogenase Outer-membrane lipoprotein carrier protein 6.33 / 42,728.19 5.55 / 20,322.37 5.52 / 21,066 ( 5 -- 6 ) Outer-membrane lipoprotein P61320 LolB 8.58 / 21,234.84 179 P0A9M0 ATP-dependent protease La Lon 6.01 / 87,438.12 -- / 94,000 237 , 297 LpdA 5.79 / 50,557.30 P0A9P0 Dihydrolipoyl dehydrogenase 5.81 / 53,637 5.82 / 50,800 133 , 179 , 228 , 270 , 294 , 321 , 323 5.90 / 28,059 ( DIGE 4.5 -- 6.5 ) 8.12 / 6,385.04 Major outer membrane lipoprotein P69776 Lpp 179 5.18 / 19,285.00 5.20 / 18,715 ( 4.5 -- 5.5 ) 5.07 / 18,656 ( 4.5 -- 5.5 ) S-Ribosylhomocysteine lyase LuxS P45578 ( YgaG ) 179 , 274 , 287 P00861 Diaminopimelate decarboxylase 5.63 / 46,177.26 5.76 / 44,970 P0A8N3 Lysyl-tRNA synthetase 5.11 / 57,472.23 5.14 / 58,637 LysA LysS 228 , 294 29 , 109 , 228 , 230 , 294 LysU 5.10 / 57,695.38 -- / 60,500 P0A8N5 Lysyl-tRNA synthetase , heat inducible 61 , 297 286 6.04 / 22,694 6.19 / 19,964.80 Maltose O-acetyltransferase P77791 Maa 179 5.49 / 37,032.35 Macrolide-speciﬁc efﬂux protei P75830 n MacA MalE 5.08 / 41,137 5.22 / 40,707.32 Maltose-binding periplasmic protein P0AEX 9 27 , 179 , 274 , 278 , 286 , 287 , 311 , 321 , 323 5.20 / 38,691 ( DIGE 4.5 -- 6.5 ) 5.23 / 38,521 ( 4.5 -- 5.5 ) 5.21 / 39,072 ( 4.5 -- 5.5 ) 5.09 / 50,544 ( 5 -- 6 ) 5.00 / 50,085 ( 5 -- 6 ) Continued on following pag Part of the ABC transporter complex MalEFGK , involved in maltose/maltodextrin import ; responsible for energy coupling to the transport system Involved in the conversion of glucose to GDP-L-fucose , which can be converted to L-fucose , a capsular polysaccharide The phosphoenolpyruvate-dependent sugar PTS , a major carbohydrate active-transport system , catalyzes the phosphorylation of incoming sugar substrates concomitant with their translocation across the cell membrane ; involved in mannose-transport ; induced by low pH Removes the amino-terminal methionine from nascent proteins Catalyzes the reversible oxidation of malate to oxaloacetate MalK P68187 Maltose/maltodextrin import ATP-binding protein 6.23 / 40,990.45 179 Mannose-6-phosphate isomerase 5.29 / 42,849.95 5.16 / 50,629 ( 5 -- 6 ) P00946 ManA 87 2 ManX PTS-system mannose-speciﬁc IIAB component P69797 5.74 / 34,916.36 5.17 / 26,187 ( 4.5 -- 5.5 ) 27 , 179 , 287 Map P0AE18 Methionine aminopeptidase 179 , 228 , 287 , 294 5.64 / 29,330.80 5.66 / 33,838 5.71 / 38,478 ( 5 -- 6 ) 5.81 / 37,872 ( 5.5 -- 6.7 ) 5.55 / 35,532 5.62 / 33,338 5.43 / 35,112 5.56 / 35,156 ( DIGE 4.5 -- 6.5 ) 5.57 / 40,903 ( 5 -- 6 ) 5.49 / 38,144 ( 5 -- 6 ) 5.25 / 37,049 ( 5 -- 6 ) 6.86 / 29,566 ( 6 -- 11 ) 8.04 / 24,966 ( 6 -- 11 ) 6.28 / 87,397 ( 6 -- 9 ) Mdh 5.61 / 32,337.30 Malate dehydrogenase P61889 Involved in the biosynthesis of osmoregulated periplasmic glucans Converts O-succinylbenzoyl-CoA to 1,4-dihydroxy-2-naphthoic acid ; involved in menaquinone-biosynthesis Catalyzes the transfer of a methyl group from 5-methyltetrahydrofolate to homocysteine , resulting in methionine-formation ; increases during growth in the absence of methionine Required not only for elongation of protein-synthesis but also for the initiation of all mRNA translation through initiator tRNA ( fMet ) aminoacylation ; decreases during phosphate limitation Involved in terminal step in the de novo biosynthesis of methionine ; decreases during phosphate limitation Catalyzes the formation of S-adenosylmethionine from methionine and ATP ; may be directly degraded by ClpAP and ClpXP , respectively , or be modulated by a protease-dependent mechanism Component of a D-methionine permease , a binding-protein-dependent , ATP-driven transport system ; decreases after benzoic acid treatment Involved in the active transport of galactose and glucose ; plays a role in the chemotaxis towards the two sugars by interacting with the Trg chemoreceptor ; induced by high pH during anaerobic growth ; increases in the physiological short-term adaptation to glucose limitation Malate dehydrogenase P61889 MdoG P33136 179 , 287 6.26 / 55,365.38 Glucans biosynthesis protein G 5.99 / 31,633.08 P0ABU0 Naphthoate synthase MenB 179 MetE 294 5-Methyltetrahydropteroyl triglutamate-homocysteine methyltransferase P25665 5.61 / 84,542.35 MetG 5.56 / 76,123.54 5.56 / 76,123 Methionyl-tRNA synthetase P00959 179 , 228 , 236 , 293 , 294 4.97 / 135,865.85 4.96 / 134,153 Methionine synthase P13009 MetH 228 , 293 , 294 MetK 5.10 / 41,820.43 5.03 / 44,970 5.13 / 43,069 ( 4.5 -- 5.5 ) 5.00 / 30,257 ( 4.5 -- 5.5 ) P0A817 S-Adenosylmethionine synthetase 29 , 179 , 228 , 287 , 294 , 308 P28635 MetQ 4.90 / 27,000 ( DIGE 4.5 -- 6.5 ) 4.95 / 11,887 ( 4 -- 5 ) 4.95 / 11,491 ( 4 -- 5 ) 4.93 / 27,236.94 D-Methionine-binding lipoprotein 287 , 321 MglB P0AEE5 D-Galactose-binding periplasmic 5.25 / 33,367.70 5.16 / 31,345 protein 5.19 / 31,192 ( 4.5 -- 5.5 ) 179 , 228 , 287 , 294 , 311 , 323 Continued on facing pag Cell division inhibitors MinC and MinD act in concert to form an inhibitor capable of blocking formation of the polar Z ring septa ; rapidly oscillates between the poles of the cell to destabilize ftsZ ﬁlaments that have formed before they mature into polar Z rings Prevents the cell-division inhibition by proteins MinC and MinD at internal division sites while permitting inhibition at polar sites ; ensures cell-division at the proper site by restricting the formation of a division septum at the midpoint of the long axis of the cell May serve as a scaffold protein required for the formation of a complex with MrcB/PonB and MltA ; this complex could play a role in enlargement and septation of the murein sacculus Involved in the biosynthesis of molybdopterin precursor Z from guanosine ; induced by anaerobiosis but repressed by the molybdenum cofactor Involved in the transport of molybdenum into the cell ; binds molybdate with high speciﬁcity and afﬁnity Involved in the biosynthesis of a demolybdo cofactor ( molybdopterin ) necessary for molybdoenzymes ; plays a role in the activation of the small subunit of the molybdopterinconverting factor ( MoaD ) Involved in the biosynthesis of a demolybdo cofactor ( molybdopterin ) necessary for molybdoenzymes Required for the rotation of the ﬂagellar motor ; may be a linker that fastens the torquegenerating machinery to the cell wall Essential for the uptake of the murein peptide L-alanyl-gamma-D-glutamyl-meso - diaminopimelate and also transports some alpha-linked peptides such as Pro-Phe-Lys with low afﬁnity ; affected by the oligopeptide permease system Cell wall formation ; synthesis of cross-linked peptidoglycan from the lipid intermediates ; has a penicillin-insensitive transglycosylase N-terminal domain ( formation of linear glycan strands ) and a penicillin-sensitive transpeptidase C-terminal domain ( cross-linking of the peptide subunits ) Involved in formation of the rod shape of the cell ; may act as a negative regulator of FtsI TABLE 2 -- Continued Protein Accession nameb no.b Theoreticalc MgsA P0A731 Methylglyoxal synthase 6.12 / 16,918.58 179 MinD P0AEZ3 Septum site-determining protein 5.25 / 29,482.84 5.28 / 29,709 ( 4.5 -- 5.5 ) P0A734 Cell division topological speciﬁcity factor MinE 179 5.15 / 10,234.91 5.03 / 25,670.3 MipA 9 MltA-interacting protein P0A908 17 MoaB 5.73 / 18,533.90 5.74 / 18,915 ( 5 -- 6 ) 5.87 / 18,128 ( 5.5 -- 6.7 ) P0AEZ9 Molybdenum cofactor-biosynthesis protein B 179 , 287 ModA Molybdate-binding periplasmic 6.38 / 24,918.32 6.70 / 29,299 ( 6 -- 11 ) protein P37329 179 , 287 4.91 / 26,718.83 Molybdopterin-biosynthesis protein P12282 MoeB 179 4.97 / 21,222.35 4.84 / 9,749 ( 4 -- 5 ) Molybdopterin-biosynthesis protein P28694 Mog 287 9/34 ,186.08 P0AF06 Chemotaxis protein MotB 179 MppA 8.32 / 57,618.48 8.55 / 73,914 ( 6 -- 11 ) Periplasmic murein peptidebinding protein P77348 179 , 287 6.15 / 93,636.17 Penicillin-binding protein 1A MrcA MreB P0A9X4 Rod shape-determining protein 5.19 / 36,952.4 179 Continued on following pag MsbA P60752 Lipid A export ATP-binding / permease protein MscS P0C0S1 Small-conductance ( YggB ) mechanosensitive channel 7.9 / 30,896.02 179 MsrA 179 4.99 / 23,183.84 P0A744 Peptide methionine sulfoxide reductase 6.05 / 67,972.26 PTS-system mannitol-speciﬁc EIICBA component P00550 MtlA 179 MtnN 5.09 / 24,353.97 5.06 / 25,943 P0AF12 Methylthioadenosine/S-adenosylhomocysteine nucleosidase 179 , 228 , 294 MurA 179 P0A749 UDP-N-acetylglucosamine-1-carboxyvinyltransferase 5.81 / 44,817.65 UDP-N-acetylmuramoylalanyl-D-glutamate-2 ,6 - diaminopimelate ligase UDP-N-acetylglucosamine-N-acetylmuramyl - ( pentapeptide ) pyrophosphoryl-undecaprenol N-acetylglucosamine transferase Involved in lipid A export and possibly also in glycerophospholipid export and biogenesis of the outer membrane Participates in the regulation of osmotic pressure changes within the cell , opening in response to stretch forces in the membrane lipid bilayer , without the need for other proteins ; forms an ion channel of 1.0-nanosiemen conductance with a slight preference for anions ; sensitive to voltage : as the membrane is depolarized , less tension is required to open the channel and vice versa ; characterized by short bursts of activity that last for a few seconds Could have an important function as a repair enzyme for proteins that have been inactivated by oxidation ; catalyzes the reversible oxidation-reduction of methionine sulfoxide in proteins to methionine The phosphoenolpyruvate-dependent sugar PTS , a major carbohydrate active-transport system , catalyzes the phosphorylation of incoming sugar substrates concomitant with their translocation across the cell membrane ; involved in mannitol-transport Responsible for cleavage of the glycosidic bond in both 5-methylthioadenosine and S-adenosylhomocysteine Cell wall formation ; adds enolpyruvyl to UDP-N-acetylglucosamine ; target for the antibiotic phosphomycin Involved in peptidoglycan-biosynthesis forming cell wall 179 , 228 , 294 MurA 179 P0A749 UDP-N-acetylglucosamine-1-carboxyvinyltransferase 5.81 / 44,817.65 P22188 MurE 228 , 294 5.43 / 53,212.40 5.44 / 54,493 P17443 MurG Cell wall formation ; catalyzes the transfer of a GlcNAc subunit on undecaprenyl-pyrophosphoryl-MurNAc - pentapeptide ( lipid intermediate I ) to form undecaprenyl-pyrophosphoryl-MurNAc - ( pentapeptide ) GlcNAc ( lipid intermediate II ) Involved in the repair of mismatches in DNA ; possibly carries out the mismatch recognition step Involved in NAD-biosynthesis : step 2 ; catalyzes the condensation of iminoaspartate with dihydroxyacetone phosphate to form quinolinate See NadA : step 3 ( ﬁnal ) 9.74 / 37,683.53 179 5.39 / 95,246.90 5.42 / 100,213 DNA mismatch-repair protein P23909 MutS 228 , 294 NadA 79 5.19 / 38,240.8 Quinolinate synthetase A P11458 1 NadC 179 Nicotinate-nucleotide pyrophosphorylase ( carboxylating ) P30011 5.07 / 32,630.87 Continued on facing pag Catalyzes a key step in NAD-biosynthesis , transforming deamido-NAD into NAD by a two-step reaction The phosphoenolpyruvate-dependent sugar PTS , a major carbohydrate active-transport system , catalyzes the phosphorylation of incoming sugar substrates concomitant with their translocation across the cell membrane ; involved in N-acetylglucosamine-transport Involved in cell-wall-synthesis The nitrate reductase enzyme complex allows E. coli to use nitrate as an electron acceptor during anaerobic growth ; the beta chain is an electron-transfer unit containing four cysteine clusters involved in the formation of iron-sulfur centers ; electrons are transferred from the gamma chain to the molybdenum cofactor of the alpha subunit ; induced by nitrate Required for the assembly of the nitrate reductasecytochrome b-nitrate reductase complex to be fully active in the membrane Plays major role in the synthesis of nucleoside triphosphates other than ATP ; the ATP gamma phosphate is transferred to the NDP beta phosphate via a ping-pong mechanism by use of a phosphorylated active-site intermediate NH3-dependent NAD synthetase 5.41 / 30,636.83 5.43 / 45,820 ( 5 -- 6 ) 5.34 / 42,978 ( 5 -- 6 ) NadE P18843 179 , 287 PTS-system N ¬ P09323 NagE 5.78 / 68,346.89 179 acetylglucosamine-speciﬁc EIICBA component Beta-hexosaminidase 5.86 / 37,594.73 Respiratory nitrate reductase 1 6.36 / 58,066.41 beta chain P75949 P11349 NagZ NarH 179 179 Respiratory nitrate reductase 2 4.45 / 26,160.59 delta chain P19317 NarW 179 Ndk P0A763 Nucleoside diphosphate kinase 5.55 / 15,332.25 5.59 / 15,260 5.19 / 14,514 ( 5 -- 6 ) 5.53 / 14,556 ( 5 -- 6 ) 179 , 228 , 287 , 294 NemA NfnB 5.78 / 39,021 ( DIGE 4.5 -- 6.5 ) 5.83 / 24,964 ( DIGE 4.5 -- 6.5 ) 5.09 / 14,115 ( 4.5 -- 5.5 ) 5.55 / 25,091 ( 4.5 -- 5.5 ) 5.47 / 32,502 ( 5 -- 6 ) 5.32 / 28,018 ( 5 -- 6 ) 5.80 / 39,516.41 5.80 / 23,905.19 N-Ethylmaleimide reductase Oxygen-insensitive NAD ( P ) H nitroreductase P77258 P38489 179 , 321 Involved in reduction of a variety of nitroaromatic compounds using NADH ( and to a lesser extent NADPH ) as a source of reducing equivalents : two electrons are transferred ; capable of reducing nitrofurazone , quinones , and the antitumor agent CB1954 5 - ( aziridin-1-yl ) -2,4 - dinitrobenzamide Plays a role in DNA-repair ; induced by agents which generate superoxide radical anions such as menadione ; controlled by soxR Involved in a nickel transport system , probably represents the nickel binder ; induced by low pH during anaerobic growth 179 , 287 , 321 Nfo 88 5.43 / 31,479.53 P0A6C1 Endonuclease IV 5.51 / 56,302.05 Nickel-binding periplasmic protein NikA NlpA NlpB NlpD 179 , 321 179 , 321 287 5.15 / 17,637 ( DIGE 4.5 -- 6.5 ) 4.88 / 35,000 ( DIGE 4.5 -- 6.5 ) 8.84 / 38,798 ( 6 -- 11 ) 5.29 / 27,069.69 4.96 / 34,371.39 9.26 / 37,534.85 P04846 Lipoprotein 28 P0A903 Lipoprotein 34 P0ADA3 Lipoprotein May be involved in stationary-phase survival 5.29 / 27,069.69 4.96 / 34,371.39 9.26 / 37,534.85 P04846 Lipoprotein 28 P0A903 Lipoprotein 34 P0ADA3 Lipoprotein Continued on following pag NADH-quinone oxidoreductase chain E NADH-quinone oxidoreductase chain G Transcription elongation protein NDH-1 shuttles electrons from NADH , via FMN and ironsulfur ( Fe-S ) centers , to quinones in the respiratory chain ; the immediate electron acceptor for the enzyme in this species is believed to be ubiquinone ; couples the redox reaction to proton translocation ( for every two electrons transferred , four hydrogen ions are translocated across the cytoplasmic membrane ) and thus conserves the redox energy in a proton gradient See NuoB NuoB P0AFC7 NADH-quinone oxidoreductase chain B 5.59 / 25,055.82 5.46 / 25,413 ( DIGE 5.42 / 30,843 ( 5 -- 6 ) 4.5 -- 6.5 ) 179 , 287 , 321 5.4 / 18,590.05 P0AFD1 NuoE See NuoB 179 5.85 / 100,168.01 P33602 NuoG Participates in both the termination and antitermination of transcription ; binds directly to the core enzyme of the DNA-dependent RNA polymerase and also interacts with lambda N protein , RNA , Rho factor , and perhaps NusB ; increases during the low temp at 10 ° C One of the proteins essential for the formation of the RNA polymerase antitermination complex in the presence of lambda phage N protein ; involved in the transcription termination process at certain sites during normal bacterial growth Inﬂuences transcription termination and antitermination ; acts as a component of the transcription complex and interacts with the termination factor Rho and RNA polymerase Required for the action of colicins K and L and for the stabilization of mating aggregates in conjugation ; serves as a receptor for a no .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16760308" target="_blank">16760308</a>]<br></li>
<li>Involved in a glutamine-transport system GlnHPQ ; induced by lack of glutamine P0A9Z1 Nitrogen regulatory protein 5.17 / 12,425.45 P-II 1 GlnB 1 79 GlnH 6.87 / 24,963.41 7.04 / 25,213 6.93 / 24,504 7.32 / 30,106 ( 6 -- 11 ) 7.13 / 29,388 ( 6 -- 11 ) P0AEQ3 Glutamine-binding periplasmic protein 48 , 179 , 286 , 287 See GlnB P0AC55 179 , 287 Nitrogen regulatory protein P-II 2 Glutamine-transport ATP-binding protein 5.84 / 12,259.22 5.78 / 12,732 ( 5 -- 6 ) 5.94 / 11,914 ( 5.5 -- 6.7 ) 6.08 / 26,730.99 GlnK Part of the binding-protein-dependent transport system for glutamine ; probably responsible for energy coupling to the transport system ; induced by lack of glutamine Changes very little throughout the normal temp ( 23 -- 37 ° C ) and increases in level with increasing growth rate Conversion of glycerol-3-phosphate to dihydroxyacetone ; uses fumarate or nitrate as the electron acceptor Electron transfer protein ; may also function as the membrane anchor for the GlpAB dimer Converts glycerol-3-phosphate to dihydroxyacetone by using molecular oxygen or nitrate as the electron acceptor Key enzyme in the regulation of glycerol uptake and metabolism ; the activity of this regulatory enzyme is affected by several metabolites ; the inhibition by fructose 1,6-biphosphate causes alterations in the quaternary structure of the enzyme ; induced by L-alpha-glycerol-3-phosphate ; increases following exposure to the uncoupler of oxidative-phosphorylation 2,4-dinitrophenol Hydrolyzes deacylated phospholipids to glycerol-3-phosphate and the corresponding alcohols Repressor of the glycerol-3-phosphate regulon Involved in tricarboxylic-acid-cycle and is allosterically inhibited by NADH Involved in nitrogen metabolism and glutamate-biosynthesis ; has low activity in the presence of exogenous leucine ; regulated positively by Lrp Part of the binding-protein-dependent transport system for glutamate and aspartate 179 P10346 GlnQ GlnS Glutaminyl-tRNA synthetase P00962 5.89 / 63,346.70 5.88 / 63,346 29 , 109 , 179 , 228 , 230 , 294 5.91 / 74,850 ( DIGE 4.5 -- 6.5 ) Anaerobic glycerol-3-phosphate 5.75 / 45,357.24 dehydrogenase subunit B P13033 GlpB 179 P0A996 Anaerobic glycerol-3-phosphate 8.78 / 44,108.04 dehydrogenase subunit C GlpC 79 1 GlpD 6.96 / 56,750.54 5.38 / 49,649 Aerobic glycerol-3-phosphate dehydrogenase P13035 179 , 228 , 294 GlpK 5.36 / 56,099.57 5.30 / 50,642 P0A6F3 Glycerol kinase 29 , 71 , 179 , 228 , 296 5.22 / 38,200 GlpQ 179 Glycerophosphoryl diester phosphodiesterase P09394 5.82 / 28,047.84 5.82 / 28,724 GlpR P0ACL0 Glycerol-3-phosphate regulon repressor P0ABH7 Citrate synthase 228 , 294 GltA 6.21 / 48,014.99 6.85 / 45,571 179 , 294 Glutamate synthase ( NADPH ) 5.54 / 51,884.08 5.47 / 52,481 small chain 5.40 / 52,585 5.48 / 53,050 ( 5 -- 6 ) P09832 GltD 61 , 212 , 228 , 286 , 287 , 294 GltI 5.67 / 15,568 ( F ) 7.87 / 36,857 ( 6 -- 11 ) 7.82 / 31,229.49 Glutamate/aspartate periplasmic binding protein P37902 179 , 228 , 287 5.59 / 53,815.73 6.03 / 45,316.59 Glutamyl-tRNA synthetase Serine hydroxymethyltransferase P04805 P0A825 179 GltX GlyA Interconversion of serine and glycine and a key enzyme in the biosynthesis of purines , lipids , hormones , and other components ; decreases during overexpression of recombinant protein in large scale 97 , 179 , 228 , 286 , 294 , 321 6.04 / 45,960 5.94 / 46,142 4.5 -- 6.5 ) 4.5 -- 6.5 ) 4.5 -- 6.5 ) 6.02 / 45,274 ( DIGE 6.00 / 45,000 ( DIGE 6.00 / 43,217 ( DIGE 179 5.29 / 76,681.76 P00961 GlyS Glycyl-tRNA synthetase beta chain Phosphoheptose isomerase 179 Catalyzes the isomerization of sedoheptulose 7-phosphate in D-glycero-D-manno-heptose 7-phosphate Glycyl-tRNA synthetase beta chain Phosphoheptose isomerase 5.97 / 20,814.75 P63224 GmhA 179 Continued on following pag Phosphoglycolate phosphatase Gph P32662 4.58 / 27,389.17 4.52 / 30,891 ( DIGE 4.5 -- 6.5 ) 179 , 321 2,3-Bisphosphoglycerate - dependent phosphoglycerate mutase GpmA P62707 5.86 / 28,425 5.86 / 28,425.21 5.86 / 27,000 ( DIGE 4.5 -- 6.5 ) 5.82 / 32,032 ( 5 -- 6 ) 2,3-Bisphosphoglycerate-independent phosphoglycerate mutase Xanthine-guanine phosphoribosyltransferase Involved in the hexose monophosphate shunt ; increases during phosphate limitation Negative regulator for the gluconate utilization system GNT-I , the gntUKR operon Could be involved in gluconate metabolism Maintains high levels of reduced glutathione in the cytosol Speciﬁcally catalyzes the dephosphorylation of 2-phosphoglycolate ; involved in the dissimilation of the intracellular 2-phosphoglycolate formed during the DNA-repair of 3 phosphoglycolate ends , a major class of DNA lesions induced by oxidative stress ; constitutively expressed Catalyzes the interconversion of 2-phosphoglycerate and 3-phosphoglycerate ; strongly inhibited by vanadate and regulated by the Fur protein See GpmA ; insensitive to vanadate Protein GntY GntY ( YhgI ) Gor P63020 4.52 / 20,997.70 4.72 / 19,718 ( 4 -- 5 ) 4.96 / 19,460 ( 4 -- 5 ) 5.65 / 48,676 179 , 287 P06715 Glutathione reductase 5.64 / 48,772.51 29 , 179 , 228 , 294 Gph P32662 4.58 / 27,389.17 4.52 / 30,891 ( DIGE 4.5 -- 6.5 ) 179 , 321 GpmA P62707 5.86 / 28,425 5.86 / 28,425.21 5.86 / 27,000 ( DIGE 4.5 -- 6.5 ) 5.82 / 32,032 ( 5 -- 6 ) P37689 GpmI 5.15 / 52,681 ( DIGE 4.5 -- 6.5 ) 5.11 / 52,363 ( DIGE 4.5 -- 6.5 ) 5.15 / 52,348 ( 4.5 -- 5.5 ) 5.44 / 13,981 ( 4.5 -- 5.5 ) 5.21 / 16,164 ( 5 -- 6 ) 5.14 / 56,193.89 287 , 321 Involved in purine-salvage pathway ; acts on guanine , on xanthine , and to a lesser extent on hypoxanthine Acts as a radical domain for damaged PFL and possibly other radical proteins ; induced by low pH ; a strong candidate for response to internal acidiﬁcation 5.52 / 16,970.59 P0A9M5 Gpt 87 2 GrcA ( YﬁD ) 5.09 / 14,284.19 5.02 / 15,065 5.00 / 12,390 Autonomous glycyl radical cofactor P68066 27 , 179 , 274 , 286 , 287 5.13 / 11,675 5.01 / 11,079 ( 4 -- 5 ) 4.96 / 10,626 ( 4 -- 5 ) 5.05 / 9,834 ( 4.5 -- 5.5 ) 5.21 / 9,497 ( 4.5 -- 5.5 ) 5.09 / 11,523 ( 4.5 -- 5.5 ) 5.23 / 9,710 ( 4.5 -- 5.5 ) 4.68 / 15,984 4.95 / 13,865 ( 4 -- 5 ) Necessary for efﬁcient RNA polymerase transcription elongation past template-encoded arresting sites ; cleavage of the nascent transcript by cleavage factors such as GreA or GreB allows the resumption of elongation from the new 3 terminus Prevents misfolding and promotes the refolding and proper assembly of unfolded polypeptides generated under stress conditions ; increases following exposure to the uncoupler of oxidative-phosphorylation 2,4-dinitrophenol and overexpression of bioproducts ; induced by temp , ethanol , nalidixic acid , and puromycin Binds to GroL in the presence of Mg-ATP and suppresses the ATPase activity of the latter ; increases following exposure to the uncoupler of oxidative-phosphorylation 2,4-dinitrophenol and is induced by temp , ethanol , nalidixic acid , puromycin , and phosphate limitation GreA 179 , 286 , 287 4.71 / 17,640.96 Transcription elongation factor P0A6W5 GroL 4.85 / 57,197.66 P0A6F5 60-kDa chaperonin 71 , 97 , 138 , 179 , 228 , 287 , 294 , 297 , 321 4.85 / 56,695 4.93 / 61,278 ( DIGE 4.5 -- 6.5 ) 4.98 / 60,758 ( DIGE 4.5 -- 6.5 ) 4.96 / 59,987 ( DIGE 4.5 -- 6.5 ) 4.99 / 56,877 ( 4 -- 5 ) 4.95 / 55,185 ( 4 -- 5 ) 4.97 / 61,125 ( 4 -- 5 ) 4.93 / 55,185 ( 4 -- 5 ) 4.95 / 55,185 ( 4 -- 5 ) 4.84 / 52,802 ( 4 -- 5 ) 4.96 / 54,146 ( 4.5 -- 5.5 ) 4.99 / 55,284 ( 4.5 -- 5.5 ) 5.15 / 15,657 5.15 / 10,500 ( DIGE 4.5 -- 6.5 ) 5.22 / 12,998 ( 4.5 -- 5.5 ) GroS P0A6F9 10 -- kDa chaperonin 71 , 179 , 228 , 287 , 293 , 294 , 297 , 321 5.15 / 10,386.95 Continued on facing pag GMP reductase GuaC 179 6.1 / 37,383.67 P60560 5.09 / 96,963.51 5.28 / 100,213 P0AES4 DNA gyrase subunit A GyrA 228 , 294 5.72 / 89,818.72 5.63 / 31,059.27 P0AES6 DNA gyrase subunit B P31658 Chaperone protein GyrB HchA 179 321 5.44 / 33,098 ( DIGE 4.5 -- 6.5 ) HdeA 4.68 / 9,740.91 4.56 / 10,724 4.84 / 9,424 ( 4 -- 5 ) P0AES9 Protein 103 , 179 , 228 , 287 , 294 , 323 HdeB 4.94 / 9,065.24 4.85 / 11,285 5.00 / 9,588 ( 4.5 -- 5.5 ) P0AET2 Protein 103 , 179 , 228 , 278 , 287 , 294 HdhA 5.22 / 26,778.59 5.17 / 25,083 P0AET8 7-Alpha-hydroxysteroid dehydrogenase 179 , 286 321 5.20 / 38,691 ( DIGE 4.5 -- 6.5 ) 5.25 / 35,493.60 HemB P0ACB2 Delta-aminolevulinic acid dehydratase P06983 Porphobilinogen deaminase 5.31 / 33,851.83 HemC 179 Uroporphyrinogen decarboxylase Putative uroporphyrin-III C-methyltransferase Cell division protein FtsH Participates actively in the response to hyperosmotic and heat shock by preventing the aggregation of stress-denatured proteins in association with DnaK ; increases following exposure to the uncoupler of oxidative-phosphorylation 2,4-dinitrophenol ; induced by temp , ethanol , nalidixic acid , CdCl2 , and puromycin Inhibited by 7,8-dihydrofolate , methotrexate , and trimethoprim Conjugates reduced glutathione to a large no .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16760308" target="_blank">16760308</a>]<br></li>
<li>More than 30 proteins are altered in the hfq null mutant ; some of these proteins are Sdependent genes Involved in fourth step of Lhistidine from 5-phospho-alpha-D-ribose 1-diphosphate P0ABC3 Protein HﬂC 6.3 / 37,649.88 179 Hfq 204 P0A6X3 Protein 7.20 / 11,035.19 HisA P10371 4.94 / 26,032.65 1 - ( 5-Phosphoribosyl ) -5 - ( 5-phosphoribosylamino ) methylideneamino imidazole-4-carboxamide isomerase Histidine biosynthesis bifunctional protein Histidinol-phosphate aminotransferase Histidinol dehydrogenase 4.99 / 24,831 ( DIGE 4.5 -- 6.5 ) 5.46 / 23,581 ( DIGE 4.5 -- 6.5 ) 179 , 321 5.76 / 40,277.96 5.43 / 35,112 See HisA : steps 6 and 8 286 P06987 HisB See HisA : step 7 ; induced by high pH during anaerobic growth See HisA : ﬁnal step ; catalyzes the sequential NAD-dependent oxidations of L-histidinol to Lhistidinaldehyde and then to L-histidine ; induced by high pH during anaerobic growth See HisA : step 5 5.01 / 39,360.14 P06986 HisC 323 HisD P06988 5.06 / 45,979.15 5.07 / 48,837 ( DIGE 4.5 -- 6.5 ) 179 , 321 , 323 P60664 HisF 5.03 / 28,454.45 179 Imidazole glycerol phosphate synthase subunit ATP phosphoribosyltransferase Has a crucial role in the pathway because the rate of histidine-biosynthesis seems to be controlled primarily by regulation of hisG enzymatic activity ; feedback inhibited by histidine or inhibited by AMP and N -5 - phosphoribosyl-ATP See HisA : step 1 HisG 5.47 / 33,366.72 P60757 179 , 321 E 4.5 -- 6.5 ) IG 5.40 / 32,959 ( D HisH 287 P60595 5.33 / 21,652.86 5.24 / 27,057 ( 5 -- 6 ) Imidazole glycerol phosphate synthase subunit Histidine biosynthesis bifunctional protein HisIE Histidine-binding periplasmic protein HisI P06989 See HisA : steps 2 and 3 179 5.24 / 22,755.82 Component of the high-afﬁnity histidine permease , a binding-protein-dependent transport system ( HisJ , HisQ , HisM , and HisP ) HisJ 179 , 286 , 287 , 321 5.17 / 26,232.61 5.05 / 28,668 P0AEU0 5.28 / 27,969 ( DIGE 4.5 -- 6.5 ) 5.14 / 27,701 ( DIGE 4.5 -- 6.5 ) 4.99 / 10,461 ( 4 -- 5 ) 5.14 / 28,391 ( 4.5 -- 5.5 ) 5.59 / 50,642 HisS HldE 5.65 / 46,898.25 5.29 / 51,050.62 29 , 179 , 228 , 294 179 Histidyl-tRNA synthetase Bifunctional protein P60906 P76658 Catalyzes the phosphorylation of D-glycero-D-manno-heptose 7-phosphate at the C-1 position to form d,D-heptose-1 ,7 - bisphosphate ; also catalyzes the ADP transfer to D-glycero-D-manno-heptose 1-phosphate , yielding ADP-d , D-heptose Involved in NO detoxiﬁcation in an aerobic process , termed nitric oxide dioxygenase reaction , that utilizes O2 and NAD ( P ) H to convert NO to nitrate , which protects the bacterium from various noxious nitrogen compounds ; induced by nitric oxide NO ( under aerobic conditions ) , nitrite , nitrate ( under anaerobic conditions ) , nitroso compounds , and paraquat ; induced by low pH during anaerobic growth HisS HldE 5.65 / 46,898.25 5.29 / 51,050.62 29 , 179 , 228 , 294 179 Histidyl-tRNA synthetase Bifunctional protein P60906 P76658 Hmp 5.48 / 43,867.66 Flavohemoprotein P24232 323 Continued on facing pag Has possible histone-like function and may be a global transcriptional regulator through its ability to bind to curved DNA sequences , which are found in regions upstream of a certain subset of promoters ; subject to transcriptional autorepression by binding preferentially to the upstream region of its own gene recognizing two segments of DNA on both sides of a bend centered around position 150 ; the hns mutant exhibit derepression not only of csgBA , hdeAB , mcc , and osmY but also of the expression of at least 22 S-controlled genes , including S itself ; increases during phosphate limitation and phosphonate growth Involved in purine-salvage pathway and acts exclusively on hypoxanthine Involved in the maturation of iron-sulfur cluster-containing proteins ; seems to help targeting proteins to be folded toward HscA Redox regulated molecular chaperone ; protects both thermally unfolding and oxidatively damaged proteins from irreversible aggregation ; plays an important role in the bacterial defense system toward oxidative stress ; induced by heat shock Chaperone subunit of a proteasome-like degradation complex ; increases following exposure to the uncoupler of oxidative-phosphorylation 2,4-dinitrophenol Protease subunit of a proteasome-like degradation complex ; increases following exposure to the uncoupler of oxidative-phosphorylation 2,4-dinitrophenol Molecular chaperone ; has ATPase activity ; increases following exposure to the uncoupler of oxidative-phosphorylation 2,4-dinitrophenol but decreases during phosphate limitation late Participates in the periplasmic electron-transferring activity of hydrogenase 2 during its catalytic turnover Required for the maturation of the three NiFe hydrogenases ; exhibits a low intrinsic GTPase activity ; affects some aspect of the processing of hydrogenases 1 and 2 , perhaps nickel incorporation into the apoenzymes , since hypB gene lesions can be complemented by high nickel ion concn in the medium TABLE 2 -- Continued Hns P0ACF8 DNA-binding protein H-NS 5.44 / 15,408.44 5.45 / 15,612 5.00 / 12,390 5.38 / 15,657 5.02 / 9,935 Hpt 5.09 / 20,115.24 5.38 / 12,316 ( 4.5 -- 5.5 ) 179 , 287 P0A9M2 Hypoxanthine phosphoribosyltransferase P0A6L9 Cochaperone protein 5.05 / 20,137.71 HscB 79 1 4.35 / 32,534.48 P0A6Y5 33-kDa chaperonin HslO 79 1 HslU 5.24 / 49,593.80 5.30 / 48,579 P0A6H5 ATP-dependent hsl protease ATP-binding subunit 71 , 179 , 228 , 294 5.95 / 18,961.66 5.93 / 22,161 P0A7B8 ATP-dependent protease HslV 71 , 228 , 294 HtpG 5.09 / 71,422.53 5.06 / 65,639 P0A6Z3 Chaperone protein 71 , 179 , 228 , 293 , 294 HybA 79 P0AAJ8 Hydrogenase-2 operon protein 6.38 / 33,263.79 6.38 / 33,263.79 1 P0AAN3 Hydrogenase isoenzymes nickel 5.42 / 31,565.04 6.32 / 32,224 incorporation protein HypB 286 Continued on following pag Required for the formation of all three hydrogenase isoenzymes Associates with aggregated proteins , together with IbpB , to stabilize and protect them from irreversible denaturation and extensive proteolysis during heat shock and oxidative stress ; aggregated proteins bound to the IbpAB complex are more efﬁciently refolded and reactivated by the ATP-dependent chaperone systems ClpB and DnaK/DnaJ/GrpE ; its activity is ATP independent ; IbpAB increase in overexpression of heterologous proteins and biopolymers See IbpA ; decreases during phosphate limitation early Inhibition of this enzyme by phosphorylation regulates the branch point between the Krebs-cycle and the glyoxylate-bypass , which is an alternate route that accumulates carbon for biosynthesis when acetate is the sole carbon source for growth ; increases during high-cell-density cultivation but decreases during phosphate limitation Fermentative lactate dehydrogenase ; increases during phosphate limitation and at low pH under anaerobic conditions Catalyzes the formation of alpha-ketobutyrate from threonine in isoleucine-biosynthesis ; isoleucine allosterically inhibits whereas valine allosterically activates this enzyme ; increases during phosphate limitation and phosphonate growth Involved in valine and isoleucine-biosynthesis : step 1 See IlvB : step 2 ; induced in the presence of acetohydroxybutyrate and acetolactate ; decreases after benzoic acid treatment See IlvB : step 3 ; increases after benzoic acid treatment See IlvB : step 4 ( ﬁnal ) ; acts on leucine , isoleucine , and valine ; decreases during phosphate limitation See IlvB : step 1 ; sensitive to valine inhibition Determines N-hexane tolerance ; involved in outer membrane permeability ; essential for envelope biogenesis ; could be part of a targeting/usher system for outer membrane components TABLE 2 -- Continued HypD P24192 5.80 / 41,363.38 5.81 / 41,711 Hydrogenase isoenzymes formation protein P0C054 Small heat shock protein IbpA 5.57 / 15,773.71 5.35 / 15,600 5.68 / 15,600 4 , 97 , 98 , 228 , 287 , 296 5.63 / 15,600 5.63 / 15,384 ( 5 -- 6 ) 5.63 / 14,237 ( 5.5 -- 6.7 ) 5.19 / 16,093.20 P0C058 Small heat shock protein IbpB 5.02 / 15,600 5.20 / 15,600 4 , 97 , 98 , 295 Icd 5.06 / 46,200 5.15 / 46,200 5.02 / 46,100 5.11 / 44,335 ( DIGE 4.5 -- 6.5 ) 5.15 / 44,053 ( DIGE 4.5 -- 6.5 ) 4.70 / 22,351 ( 4 -- 5 ) 5.15 / 43,951 ( 4.5 -- 5.5 ) 5.00 / 74,518 ( 5 -- 6 ) 5.15 / 45,756.71 P08200 Isocitrate dehydrogenase ( NADP ) 179 , 228 , 236 , 287 , 293 , 294 , 321 , 325 P52643 IdhA 5.29 / 36,534.79 5.59 / 34,100 D-Lactate dehydrogenase 128 , 293 5.57 / 56,195.25 6.16 / 54,999 Threonine dehydratase biosynthetic P04968 IlvA 293 287 5.22 / 56,152 ( 4.5 -- 5.5 ) P08142 5.30 / 60,440.57 I e Acetolactate synthase isozym large subunit Ketol-acid reductoisomerase IlvB 5.26 / 52,066 5.20 / 53,937.83 P05793 IlvC 286 , 321 5.22 / 52,046 ( DIGE 4.5 -- 6.5 ) 5.29 / 51,264 ( DIGE 4.5 -- 6.5 ) 321 5.54 / 62,065 ( DIGE 4.5 -- 6.5 ) 5.59 / 65,400.37 P05791 Dihydroxy-acid dehydratase IlvD 5.43 / 35,112 5.54 / 33,962.46 P0AB80 Branched-chain amino acid aminotransferase IlvE 228 , 236 , 293 , 294 287 8.11 / 15,587 ( 6 -- 11 ) 8.01 / 17,976.76 e Acetolactate synthase isozym III small subunit Organic solvent tolerance protein P00894 IlvH Imp 4.91 / 88,368 ( DIGE 4.5 -- 6.5 ) 4.85 / 87,068.95 P31554 179 , 321 Continued on facing pag IscA P0AAC8 Iron-binding protein ( YfhF ) 4.75 / 11,556.05 IscU 179 Involved in Fe-S biosynthesis 4.82 / 13,848.59 IspA IspG Geranyltranstransferase 5.27 / 32,159.63 5.37 / 26,625 4-Hydroxy-3-methylbut-2-en-1-yl 5.87 / 40,683.61 diphosphate synthase P22939 P62620 179 , 286 179 5.2 / 34,774.56 4-Hydroxy-3-methylbut-2-enyl diphosphate reductase P62623 IspH 179 5.46 / 15,065 5.41 / 14,304 ( 5 -- 6 ) 5.51 / 14,102.90 Inhibitor of vertebrate lysozyme P0AD59 Ivy 228 , 287 5.33 / 12,548 ( 5 -- 6 ) 5.47 / 86,507 ( DIGE 4.5 -- 6.5 ) 103 , 321 5.54 / 84,162.61 Catalase HPII P21179 KatE Peroxidase/catalase HPI P13029 KatG 103 , 287 , 321 5.18 / 77,774 ( DIGE 4.5 -- 6.5 ) 5.16 / 74,539 ( 4.5 -- 5.5 ) 5.14 / 80,023.82 5.37 / 31,293.68 5.98 / 31,100 P0AB74 Tagatose-1 ,6 - bisphosphate aldolase AgaY Kba 99 2-Amino-3-ketobutyrate CoA ligase 2-Dehydro-3-deoxygluconokinase 2-Dehydro-3-deoxyphosphooctonate aldolase Could transfer iron-sulfur clusters to apo-ferredoxin ; multiple cycles of 2Fe2S cluster formation and transfer are observed ; acts catalytically ; recruits intracellular free iron so as to provide iron for the assembly of transient ironsulfur cluster in IscU in the presence of IscS , L-cysteine , and the thioredoxin reductase system TrxA/TrxB Not found in SWISS-PROT / TrEMBL database but available in NCBI database ( GI : 16130454 ) Converts 2C-methyl-D-erythritol 2,4-cyclodiphosphate into 1-hydroxy-2-methyl-2 - ( E ) - butenyl 4-diphosphate Converts 1-hydroxy-2-methyl-2 - ( E ) - butenyl 4-diphosphate into isopentenyl diphosphate and dimethylallyl diphosphate ; also involved in penicillin tolerance , control of the stringent-response , and isoprenoid-biosynthesis Strong inhibitor of lysozyme C Decomposes hydrogen peroxide into water and oxygen ; serves to protect cells from the toxic effects of hydrogen peroxide ; induced by entry into stationary phase and after benzoic acid treatment Bifunctional , exhibiting both catalase and broad-spectrum peroxidase activities ; induced by hydrogen peroxide and by entry into stationary phase but decreases after benzoic acid treatment Catalyzes the reversible condensation of dihydroxyacetone phosphate with glyceraldehyde 3-phosphate to produce tagatose-1 ,6 - bisphosphate ; repressed in the presence of glucose Negatively regulated by Lrp Kba 99 - / 42,200 5.64 / 43,117.04 P0AB77 61 Kbl 4.92 / 33,962.36 6.32 / 30,832.69 P37647 P0A715 179 KdgK KdsA Synthesis of 2-keto-3 - 179 , 287 6.10 / 33,261 ( 6 -- 11 ) deoxyoctulosonic acid 8-phosphate , which is required for lipid A maturation and cellular-growth Activates 2-keto-3-deoxyoctulosonic acid for incorporation into bacterial lipopolysaccharide in gramnegative bacteria Involved in the transport of maltose and maltodextrins ; indispensable for translocation of dextrins containing more than three glucosyl moieties ; acts as a receptor for several bacteriophages including lambda ; induced by maltose 5.15 / 27,483.25 5.06 / 33,695 3-Deoxy-manno-octulosonate cytidylyltransferase P04951 KdsB 228 , 294 LamB 79 4.72 / 47,385.03 Maltoporin P02943 1 Continued on following pag Fermentative lactate dehydrogenase ; increases during phosphate limitation and at low pH under anaerobic conditions Catalyzes the condensation of the acetyl group of acetyl-CoA with 3-methyl-2-oxobutanoate ( 2-oxoisovalerate ) to form 3-carboxy-3-hydroxy-4-methylpentanoate ( 2-isopropylmalate ) ; may be directly degraded by ClpAP and ClpXP , respectively , or be modulated by a protease-dependent mechanism Catalyzes the isomerization between 2-isopropylmalate and 3-isopropylmalate via the formation of 2-isopropylmaleate LdhA P52643 D-Lactate dehydrogenase 5.29 / 36,534.79 5.59 / 34,100 P09151 2-Isopropylmalate synthase LeuA 308 5.47 / 57,166.71 LeuC 3-Isopropylmalate dehydratase large subunit P0A6A6 5.90 / 49,750.64 5.95 / 51,656 5.77 / 51,451 179 , 286 , 321 5.42 / 44,439 5.93 / 50,340 ( DIGE 4.5 -- 6.5 ) See LeuC ; increases after benzoic acid treatment Increases late during phosphate limitation Changes very little throughout the normal temp ( 23 -- 37 ° C ) and increases in level with increasing growth rate Represses a no .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16760308" target="_blank">16760308</a>]<br></li>
<li>179 , 287 Flagellar hook-associated protein 3 RNA polymerase sigma factor for ﬂagellar operon FlgL P29744 Sigma factors are initiation factors that promote the attachment of RNA polymerase to speciﬁc initiation sites and are then released ; speciﬁc for class 3 ﬂagellar operons Polymerizes to form the ﬁlaments of bacterial ﬂagella 179 P0AEM6 FliA FliC Flagellin P04949 4.58 / 45,401 ( 4 -- 5 ) 4.63 / 45,613 ( 4 -- 5 ) 4.55 / 45,507 ( 4 -- 5 ) 4.98 / 45,932 ( 4 -- 5 ) 4.61 / 45,507 ( 4 -- 5 ) 4.54 / 45,507 ( 4 -- 5 ) 5.00 / 44,877 ( 4 -- 5 ) 4.96 / 42,894 ( 4.5 -- 5.5 ) 179 , 287 4.50 / 51,163.80 Required for the morphogenesis and for the elongation of the ﬂagellar ﬁlament by facilitating polymerization of the ﬂagellin monomers at the tip of growing ﬁlament Three proteins ( FliG , FliN , FliM ) form a switch complex that is proposed to be located at the base of the basal body ; interacts with the CheY and CheZ chemotaxis proteins , in addition to contacting components of the motor that determine the direction of ﬂagellar rotation Needed for ﬂagellar regrowth and assembly See FliG Part of a binding-protein-dependent transport system for cystine ; repressed by sulfate or cysteine FliD 4.82 / 48,325.26 Flagellar hook-associated protein 2 P24216 179 , 287 4.69 / 36,775.98 Flagellar motor switch protein P31067 FliG 179 4.62 / 25,050.16 Flagellar assembly protein P31068 FliH 179 5.47 / 37,849.16 5.29 / 26,068.48 Flagellar motor switch protein Cystine-binding periplasmic protein P06974 P39174 FliM FliY 179 179 , 286 , 287 , 321 5.01 / 26,213 5.11 / 25,809 5.17 / 25,686 ( DIGE 4.5 -- 6.5 ) 5.19 / 26,500 ( 4.5 -- 5.5 ) 5.23 / 28,594 ( 4.5 -- 5.5 ) Controls colony form variation and autoaggregation ; may function as an adhesin Modiﬁes the free amino group of the aminoacyl moiety of methionyl-tRNA ( fMet ) ; the formyl group appears to play a dual role in the initiator identity of N-formylmethionyl-tRNA by promoting its recognition by IF2 and impairing its binding to EF-Tu -- GTP Increases during phosphate limitation and phosphonate growth but decreases after benzoic acid treatment Two distinct membrane-bound FAD-containing enzymes are responsible for the catalysis of fumarate and succinate interconversion ; the fumarate reductase is used in anaerobic growth , and the succinate dehydrogenase is used in aerobic growth See FrdA Flu 5.37 / 101,278.83 Antigen 43 P39180 1 Fmt 5.56 / 34,037.27 5.49 / 35,885 Methionyl-tRNA formyltransferase P23882 179 , 228 , 236 , 294 FolA P0ABQ4 Dihydrofolate reductase 179 , 228 , 293 , 294 , 321 5.01 / 19,989 4.84 / 17,999.38 5.02 / 21,096 ( DIGE 4.5 -- 6.5 ) Fumarate reductase ﬂavoprotein 5.87 / 65,840.41 subunit P00363 FrdA 179 179 P0AC47 Fumarate reductase iron-sulfur 6.09 / 26,991.87 protein P0A805 Ribosome recycling factor 6.44 / 20,638.57 6.16 / 21,725 FrdB Responsible for the release of ribosomes from mRNA at the termination of protein-biosynthesis ; may increase the efﬁciency of translation by recycling ribosomes from one round of translation to another FrdB Frr 179 , 228 , 294 The phosphoenolpyruvate-dependent sugar PTS , a major carbohydrate active-transport system , catalyzes the phosphorylation of incoming sugar substrates concomitant with their translocation across the cell membrane ; involved in fructose-transport ; induced by fructose ; decreases after benzoic acid treatment May be involved in anomalous ﬁlament growth ; may be a component of the septum and interact with FtsZ ; induced at the stationary phase Essential to the cell-division process ; seems to assemble into a dynamic ring on the inner surface of the cytoplasmic membrane at the place where division will occur , with the formation of the ring being the signal for septation to begin ; induced at the stationary phase Involved in second step of fucose-metabolism Involved in fourth step of fucose-metabolism Functions in the generation of fumarate for use as an anaerobic electron acceptor Acts as a global negative controlling element , employing Fe2 as a cofactor to bind the operator of the repressed genes ; regulates the expression of several outer membrane proteins , including the iron-transport operon ; induced during phosphate limitation FruB P69811 M ultiphosphoryl transfer protein 4.77 / 39,647.83 4.85 / 38,938 ( D IG E 4.5 -- 6.5 ) 179 , 321 P0ABH0 Cell division protein FtsA 103 5.84 / 45,329.97 FtsZ 4.65 / 40,323.92 4.63 / 40,491 4.83 / 39,220 ( 4 -- 5 ) 4.77 / 39,129 ( 4 -- 5 ) P0A9A6 Cell division protein 103 , 179 , 228 , 287 , 294 P11553 Fuck L-Fuculokinase 5.38 / 53,235.87 4.73 / 55,145 286 Lactaldehyde reductase 5.1 / 40,644.56 P0A9S1 FucO 179 Fumarate hydratase class 5.88 / 60,105.32 I , anaerobic P14407 FumB 179 Fur 179 , 228 , 287 , 293 , 294 P0A9A9 Ferric uptake regulation 5.68 / 16,794.85 5.79 / 17,433 protein 5.78 / 17,897 ( 5 -- 6 ) 5.78 / 17,233 ( 5 -- 6 ) Promotes the GTP-dependent translocation of the nascent protein chain from the A site to the P site of the ribosome ; decreases during phosphate limitation early GadA converts glutamate to gamma-aminobutyrate , consuming one intracellular proton in the reaction ; the Gad system helps to maintain a near-neutral intracellular pH when cells are exposed to extremely acidic conditions ; the ability to survive transit through the acidic conditions of the stomach is essential for successful colonization of the mammalian host by commensal and pathogenic bacteria ; induced by high pH during anaerobic growth Involved in third step of galactose-metabolism ; increases after benzoic acid treatment Involved in lipopolysaccharide-biosynthesis Converts alpha-aldose to the beta-anomer ; active on D-glucose , L-arabinose , D-xylose , D-galactose , maltose , and lactose May play a role in stationary-phase survival FumB 179 Fur 179 , 228 , 287 , 293 , 294 P0A9A9 Ferric uptake regulation 5.68 / 16,794.85 5.79 / 17,433 protein 5.78 / 17,897 ( 5 -- 6 ) 5.78 / 17,233 ( 5 -- 6 ) FusA P0A6M8 Elongation factor G 179 , 287 , 293 , 321 5.29 / 85,046 ( DIGE 4.5 -- 6.5 ) 5.20 / 84,325 ( DIGE 4.5 -- 6.5 ) 5.51 / 46,525 ( DIGE 4.5 -- 6.5 ) 5.20 / 38,691 ( DIGE 4.5 -- 6.5 ) 5.78 / 32,101 ( DIGE 4.5 -- 6.5 ) 5.21 / 75,514 ( 4.5 -- 5.5 ) 5.23 / 50,424 ( 5 -- 6 ) 5.24 / 77,450.11 5.22 / 52,685.16 Glutamate decarboxylase alpha P69908 GadA 27 GalE P09147 179 , 321 5.91 / 35,787 ( DIGE 4.5 -- 6.5 ) 5.89 / 37,265.11 UDP-glucose 4-epimerase 5.73 / 32,829.23 P0AAB6 UTP-glucose-1-phosphate uridylyltransferase Aldose-1-epimerase 179 GalF GalM 179 , 287 4.84 / 38,190.46 P0A9C3 4.97 / 39,311 ( 4 -- 5 ) 4.83 / 39,220 ( 4 -- 5 ) 1 P0AEP3 UTP-glucose-1-phosphate 5.11 / 32,811.07 uridylyltransferase 79 GalU Continued on following pag TABLE 2 -- Continued GapA 6.58 / 35,401.30 6.93 / 23,085 Involved in the pathway of glycolysis ; 6.28 / 36,748 induced by high pH during anaerobic 6.58 / 36,386 growth 5.93 / 22,606 ( DIGE 4.5 -- 6.5 ) 5.94 / 36,106 ( DIGE 4.5 -- 6.5 ) 5.32 / 9,085 ( 4.5 -- 5.5 ) 5.03 / 10,612 ( 4.5 -- 5.5 ) 5.74 / 40,430 ( 5 -- 6 ) 6.11 / 34,907 ( 6 -- 11 ) 5.87 / 30,811.93 179 , 228 , 286 , 287 , 294 , 321 , 323 P0A9B2 Glyceraldehyde 3-phosphate dehydrogenase A P37192 Catalyzes the reversible condensation of dihydroxyacetone phosphate with 27 , 311 , 323 Tagatose-1 ,6 - bisphosphate aldolase GatY glyceraldehyde 3-phosphate to produce tagatose-1 ,6 - bisphosphate ; induced by low pH during aerobic or anaerobic growth ; increases in the physiological short-term adaptation to glucose limitation Probably involved in energy conservation rather than in sugar metabolism 5.36 / 40,015.52 The glycine-cleavage-system catalyzes the degradation of glycine Gcd 179 Quinoprotein glucose dehydrogenase 5.4 / 86,747.35 P15877 Aminomethyltransferase ; glycine-cleavage-system T protein NADP-speciﬁc glutamate dehydrogenase P27248 GcvT 179 GdhA 5.98 / 48,581.37 5.96 / 45,327 Decreases during phosphate limitation ; 5.81 / 46,385 ( DIGE 4.5 -- 6.5 ) may be directly degraded by ClpAP or 5.94 / 44,524 ( DIGE 4.5 -- 6.5 ) ClpXP , respectively , or be modulated by a protease-dependent mechanism P00370 4.81 / 38,712.2 P0A9S5 Glycerol dehydrogenase GldA Induced by hydroxyacetone and stationary-phase growth conditions 179 Involved in the conversion of UDP-galactose proton symporter ( GalP ) into UDP-GalF through a 2-keto intermediate for lipopolysaccharide-biosynthesis Involved in glycogen-synthesis ; induced strongly at the early stationary phase and also less osmotic induction ; regulated by S and cAMP-CRP Catalyzes the conversion of glucosamine-6-phosphate to glucosamine-1-phosphate Responsible for the acetylation of glucosamine-1-phosphate ( Glc-N-1-P ) to give N-acetylglucosamine-1-phosphate ( GlcNAc-1-P ) and the synthesis of UDP-GlcNAc 5.26 / 51,772.57 5.25 / 53,849 Controlled by adenylation under 5.18 / 53,743 conditions of abundant glutamine ; 5.39 / 36,675 decreases during phosphate limitation 5.30 / 53,162 ( DIGE 4.5 -- 6.5 ) and after benzoic acid treatment ; 5.00 / 26,795 ( 4 -- 5 ) regulated positively by Lrp ; may be directly degraded by ClpAP or ClpXP , respectively , or be modulated by a protease-dependent mechanism 6.62 / 42,965.88 P37747 UDP-galactopyranose mutase Glf 179 Glycogen synthesis protein 5.38 / 7,891.88 P26649 GlgS 103 , 105 P31120 Phosphoglucosamine mutase GlmM 5.71 / 47,412.38 P0ACC7 Bifunctional protein 6.09 / 49,190.08 GlmU 179 GlnA P0A9C5 Glutamine synthetase 61 , 179 , 228 , 286 , 287 , 293 , 294 , 308 , 321 P-II indirectly controls the transcription of the glutamine synthetase gene ( glnA ) ; prevents nitrogen regulatory ( NR ) - II-catalyzed conversion of NR-I to NR-I-phosphate , the transcriptional activator of glnA ; when P-II is uridylylated to P-II-UMP , these events are reversed ; when the ratio of Gln to 2-ketoglutarate decreases , P-II is uridylylated to P-II-UMP , which causes the deadenylylation of glutamine synthetase by GlnE , thus activating the enzyme .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16760308" target="_blank">16760308</a>]<br></li>
<li>of T-even like phages and also acts as a porin with low permeability that allows slow penetration of small solutes ; induced by low pH during aerobic or anaerobic growth but decreases by phosphate limitation and SOS induction Forms passive diffusion pores which allow small-molecular-wt hydrophilic materials across the outer membrane ; negatively regulated by Lrp Forms passive diffusion pores which allow low-mol wt hydrophilic materials across the outer membrane ; receptor for the bacteriophage T2 ; positively regulated by Lrp ; decreases during phosphate limitation and after benzoic acid treatment and treatment with superoxidegenerating agents such as menadione P33602 NuoG NusA 4.53 / 54,870.92 4.55 / 61,127 P0AFF6 133 , 179 , 212 , 228 , 294 P0A780 N utilization substance protein B 6.6 / 15,689.06 NusB 179 NusG 6.33 / 20,400.32 5.93 / 22,161 5.44 / 14,684 ( 5 -- 6 ) 6.40 / 21,860 ( 6 -- 11 ) P0AFG0 Transcription antitermination protein 179 , 286 , 287 OmpA 5.55 / 34,419 5.36 / 32,707 5.60 / 35,172.28 P0A910 Outer membrane protein A 75 , 179 , 228 , 274 , 287 , 293 , 294 , 321 , 323 5.34 / 28,591 5.50 / 33,875 ( DIGE 4.5 -- 6.5 ) 5.38 / 33,875 ( DIGE 4.5 -- 6.5 ) 5.23 / 14,346 ( 5 -- 6 ) 5.25 / 13,108 ( 5 -- 6 ) 4.48 / 38,307.50 -- / 36,000 Outer membrane protein C P06996 OmpC 61 OmpF 4.61 / 36,170 4.64 / 37,084.49 Outer membrane protein F P02931 61 , 88 , 179 , 228 , 293 , 294 , 321 4.56 / 37,316 ( DIGE 4.5 -- 6.5 ) 4.64 / 37,000 ( DIGE 4.5 -- 6.5 ) 4.71 / 37,000 ( DIGE 4.5 -- 6.5 ) Continued on facing pag The N terminus of this protein is required for the transcriptional expression of both major outer membrane protein genes ompF and ompC ; its carboxyl-terminal moiety mediates the multimerization of the ompR protein Protease that can cleave T7 RNA polymerase , ferric enterobactin receptor protein ( FEP ) , antimicrobial peptide protamine , and other proteins Induced by low or high pH 79 OmpR P0AA16 Transcriptional regulatory protein 6.04 / 27,353.62 1 OmpT Protease VII P09169 179 , 321 5.38 / 33,477.72 5.36 / 37,000 ( DIGE 4.5 -- 6.5 ) OmpX Outer membrane protein X P0A917 5.30 / 16,382.89 5.25 / 16,126 ( DIGE 4.5 -- 6.5 ) 5.15 / 16,000 ( DIGE 4.5 -- 6.5 ) 5.93 / 56,137 5.70 / 57,487 5.64 / 56,808 5.44 / 54,493 5.93 / 55,635 ( DIGE 4.5 -- 6.5 ) 5.82 / 55,131 ( DIGE 4.5 -- 6.5 ) 5.71 / 52,681 ( DIGE 4.5 -- 6.5 ) 5.13 / 50,973 ( 5 -- 6 ) 5.47 / 50,921 ( 5 -- 6 ) 5.63 / 50,921 ( 5 -- 6 ) 5.65 / 50,915 ( 5.5 -- 6.7 ) 179 , 274 , 321 Component of the oligopeptide permease and binding-protein-dependent transport system ; binds peptides up to 5 amino acids long with high afﬁnity ; increases by high pH during anaerobic growth and after benzoic acid treatment OppA 5.85 / 58,359.84 Periplasmic oligopeptide-binding protein P23843 179 , 228 , 286 , 287 , 294 , 321 , 323 Part of the binding-protein-dependent transport system for oligopeptides ; probably responsible for energy coupling to the transport system Provides resistance to osmotic stress ; may be important for stationary-phase survival ; induced by elevated osmotic pressure in the growth medium Preferentially metabolizes organic hydroperoxides over inorganic hydrogen peroxide ; induced by elevated osmotic pressure and during the decelerating phase , before entry into stationary phase ; regulated by a complex network of several global regulators , including at least S , Lrp , and H-NS Induced by elevated osmotic pressure Osmotically and stationary phase inducible and controlled by several global regulators , including S , cAMP-CRP , IHF , and Lrp Induced by growth in glucose-mineral medium of elevated osmotic strength ; controlled by S Induced by growth in glucose-mineral medium of elevated osmotic strength ; controlled by S May play a role in bacterial envelope integrity ; strongly associated with the peptidoglycan Involved in pantothenate-biosynthesis : ﬁrst step See PanB : ﬁnal step .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16760308" target="_blank">16760308</a>]<br></li>
<li>Genetic studies have identi ® ed cAMP-CRP , Lrp , integration host factor ( IHF ) and H-NS as such modulators of sS-dependent genes ( see Hengge-Aronis ( 1996b ) for review , and references cited therein ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9512707" target="_blank">9512707</a>]<br></li>
<li>175 , No. 24 Complex Transcriptional Control of the us-Dependent Stationary-Phase-Induced and Osmotically Regulated osmY ( csi-5 ) Gene Suggests Novel Roles for Lrp , Cyclic AMP ( cAMP ) Receptor Protein-cAMP Complex , and Integration Host Factor in the Stationary-Phase Response of Escherichia coli ROLAND LANGE , MECHTHILD BARTH , AND REGINE HENGGE-ARONIS * Department of Biology , University of Konstanz , P.O. Box 5560 , 78434 Konstanz , Germany Received 15 June 1993/Accepted 18 October 1993 osmY ( csi-5 ) is a representative of a large group of cj-dependent genes in Escherichia coli that exhibit both stationary-phase induction and osmotic regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8253679" target="_blank">8253679</a>]<br></li>
<li>R. ( 1993 ) Complex transcriptional-control of the a ^ - dependent stationary phase-induced osmY -LCB- csi-S ) gene suggests novel roles for Lrp , cAMP-CRP and IHF in the stationary phase response of Escherichia coli J Bacterioi MS : 7910-7917 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7997184" target="_blank">7997184</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_13">
Complex transcriptional-control of the sS-depen-dent stationary phase-induced osmY ( csi-5 ) gene suggests novel roles for Lrp , cyclic AMP ( cAMP ) receptor protein-cAMP complex and integration host factor in the stationary phase response of Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9512707" target="_blank">9512707</a>]<br></label>
<input id="rp_similarity_13", type="checkbox">
<div class="RP_similar">
<li>Complex transcriptional S control of the s - dependent stationary phase-induced osmY ( csi-5 ) gene suggests novel roles for Lrp , cyclic AMP ( cAMP ) receptor protein-cAMP complex , and integration host factor in the stationary phase response of Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7814331" target="_blank">7814331</a>]<br></li>
<li>Complex transcriptional-control of the sS-dependent stationary-phase-induced osmY ( csi-5 ) gene suggests novel roles for Lrp , cyclic AMP ( cAMP ) receptor protein-cAMP complex , and integration host factor in the stationary-phase response of Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7768855" target="_blank">7768855</a>]<br></li>
<li>Complex transcriptional-control of the sigma S-dependent stationary-phase-induced and osmotically regulated osmY ( csi-5 ) gene suggests novel roles for Lrp , cyclic AMP ( cAMP ) receptor protein-cAMP complex , and integration host factor in the station-ary-phase response of Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9537375" target="_blank">9537375</a>]<br></li>
<li>Complex transcriptional-control of the sS-dependent stationary-phase-induced and osmotically regulated osmY ( csi-5 ) gene suggests novel roles for Lrp , cyclic AMP ( cAMP ) receptor protein-cAMP complex , and integration host factor in the station-ary-phase response of Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7559363" target="_blank">7559363</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8631665" target="_blank">8631665</a>]<br></li>
<li>Complex transcriptional-control of the ss-dependent stationary-phase-induced and osmotically regulated osmY ( csi-5 ) gene suggests novel roles for Lrp , cyclic AMP ( cAMP ) receptor protein-cAMP complex , and integration host factor in the station-ary-phase response of Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9079929" target="_blank">9079929</a>]<br></li>
<li>Complex transcriptional-control of the S-dependent stationary-phase-induced and osmotically regulated osmY ( csi-5 ) gene suggests novel roles for Lrp , cyclic AMP ( cAMP ) receptor protein-cAMP complex , and integration host factor in the station-ary-phase response of Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15489429" target="_blank">15489429</a>]<br></li>
<li>Complex transcriptional-control of the s-dependent stationary-phase-induced and osmotically regulated osmY ( csi-5 ) gene suggests novel roles for Lrp , cyclic AMP ( cAMP ) receptor protein-cAMP complex , and integration host factor in the station-ary-phase response of Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12730174" target="_blank">12730174</a>]<br></li>
<li>Complex transcriptional S control of the s - dependent stationary-phase-induced and osmotically regulated osmY ( csi-5 ) gene suggests novel roles for Lrp , cyclic AMP ( cAMP ) receptor protein-cAMP complex , and integration host factor in the station-ary-phase response of Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7768815" target="_blank">7768815</a>]<br></li>
<li>Complex transcriptional S control of the - dependent stationary phase-induced and osmotically regulated osmY ( csi-5 ) gene suggests novel roles for Lrp , cyclic AMP ( cAMP ) receptor protein-cAMP complex , and integration host factor in the stationary phase response of Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15716429" target="_blank">15716429</a>]<br></li>
<li>Complex transcriptional-control of the a ' - dependent stationary-phase-in-duced and osmotically regulated osmY ( csi-5 ) gene suggests novel roles for Lrp , cyclic AMP ( cAMP ) receptor protein-cAMP complex , and integration host factor in the stationary-phase response of Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8071194" target="_blank">8071194</a>]<br></li>
<li>J Biol Chem 278:37160 -- 37168 Lange R , Barth M , Hengge-Aronis R ( 1993 ) Complex transcriptional-control of the sigma S-dependent stationary-phase-in-duced and osmotically regulated osmY ( csi-5 ) gene suggests novel roles for Lrp , cyclic AMP ( cAMP ) receptor proteincAMP complex , and integration host factor in the stationary-phase response of Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16404567" target="_blank">16404567</a>]<br></li>
<li>[ 21 ] Lange R. , Barth M. , Hengge-Aronis N. , Complex transcriptional-control of the sigma s-dependent stationary-phase-induced and osmotically regulated osmY ( csi-5 ) gene suggests novel roles for Lrp , cyclic AMP ( cAMP ) receptor protein-cAMP complex , and integration host factor in the stationary-phase response of Escheri-chia coli , J. Bacteriol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9924977" target="_blank">9924977</a>]<br></li></div>
				<label class="RP_collapse_similarity" for="rp_similarity_26">
Lange , R. , Barth , M. , and Hengge-Aronis , R. ( 1993 ) Complex transcriptional-control of the os-dependent stationary phase-induced osmY ( csi-5 ) gene suggests novel roles for Lrp , cyclic AMP ( cAMP ) receptor protein-cAMP complex and integrationhost factor in the stationary phase response of Escherichia coli J Bacteriol175 : 7910-791 7 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596457" target="_blank">8596457</a>]<br></label>
<input id="rp_similarity_26", type="checkbox">
<div class="RP_similar">
<li>Lange , R. , Barth , M. , and Hengge-Aronis , R. ( 1993 ) Complex transcriptional-control of the Ds-dependent stationary-phase-induced osmY ( csi-5 ) gene suggests novel roles for Lrp , cyclic AMP ( CAMP ) receptor protein-CAMP complex , and integration host factor in the stationary-phase response of Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8899708" target="_blank">8899708</a>]<br></li>
<li>b31 Lange , R. , Barth , M. and Hengge-Aronis , R. ( 1993 ) Complex transcriptional-control of the ss-dependent stationary-phase-induced and osmotically regulated osmy ( csi-5 ) gene suggests novel roles of Lrp , cyclic AMP ( CAMP ) receptor protein-CAMP complex , and integration host factor in the stationary-phase response of Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7557316" target="_blank">7557316</a>]<br></li>
<li>Lange , R. , Barth , M. , and Henge-Aronis , R. ( 1993 ) Complex transcriptional-control of the sigma S-dependent station-ary-phase-induced and osmotically regulated osmY ( csi-5 ) gene suggests novel roles for Lrp , cyclic AMP ( cAMP ) receptor protein ± cAMP complex , and integration host factor in the stationary-phase response of Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10361282" target="_blank">10361282</a>]<br></li>
<li>Lange , R. , Barth , M. , and Hengge-Aronis , R. ( 1993 ) Complex transcriptional-control of stationary-phase-induced and osmo-tically regulated osmY ( csi-5 ) gene suggests novel roles for Lrp , cyclic AMP ( cAMP ) receptor protein ± cAMP complex , and integration host factor in the stationary-phase response of Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10692161" target="_blank">10692161</a>]<br></li>
<li>Lange , R. , Barth , M. , and Hengge-Aronis , R. ( 1993 ) Complex control of the ss-dependent stationary phase-induced and osmotically regulated osmY ( csi-5 ) gene suggests novel roles for Lrp , cyclic AMP ( cAMP ) , receptor protein -- cAMP complex , and integration host factor in the stationary phase response of Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9663683" target="_blank">9663683</a>]<br></li>
<li>Lange , R. , Barth , M. , and Hengge-Aronis , R. ( 1993 ) Complex transcriptional-control of the os-dependent stationary-phase-induced and osmotically regulated osmY ( csi-5 ) gene suggests novel roles for Lrp , cyclic AMP ( CAMP ) - receptor protein-CAMP complex , and integration host factor in the stationary-phase response of Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8809748" target="_blank">8809748</a>]<br></li>
<li>Lange , R. , Barth , M. , and Hengge-Aronis , R. ( 1993 ) Complex transcriptional-control of the sigma s-dependent stationary-phase-induced and osmotically regulated osmY ( csi-5 ) gene suggests novel roles for Lrp , cyclic AMP ( cAMP ) receptor protein-cAMP complex , and integration host factor in the stationary-phase response of Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16420371" target="_blank">16420371</a>]<br></li>
<li>Lange , R , , Barth , M , , and Hengge-Aronis , R. ( 1993 ) Complex transcriptional-control of the a ^ - dependent stationary phase-induced and osmotically regulated osmY -LCB- csi-5 ) gene suggests novel roles for Lrp , cyclic AMP ( cAMP ) receptor protein-cAMP complex , and integration host factor in the stationary-phase response of Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7984106" target="_blank">7984106</a>]<br></li>
<li>Lange , R. , Barth , M. , and Hengge-Aronis , R. ( 1993 ) Complex transcriptional-control of the s - dependent stationary S phase-induced and osmotically regulated osmY ( csi-5 ) gene suggests novel roles for Lrp , cyclic AMP ( cAMP ) receptor protein-cAMP complex and integration host factor in the stationary phase response of Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11251833" target="_blank">11251833</a>]<br></li>
<li>Lange , R. , Barth , M. , and Hengge-Aronis , R. ( 1993 ) Complex transcriptional-control of the sigmaS-dependent stationary-phase-induced and osmotically regulated osmY ( csi-5 ) gene suggests novel roles for Lrp , cyclic AMP ( cAMP ) receptor protein-cAMP complex , and integration host factor in the stationary-phase response of Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17224607" target="_blank">17224607</a>]<br></li></div>
<li>These are Lrp , integration host factor ( IHF ) , and the cAMP receptor protein-cAMP complex ( cAMP-CRP ) , all of which negatively modulate osmY expression in response to growth phase and medium composition .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8253679" target="_blank">8253679</a>]<br></li>
<li>The cyclic AMP ( cAMP ) - cAMP receptor protein complex ( 13 , 20 , 21 , 29 , 47 ) , integration host factor ( 1 , 20 ) , and Lrp ( 20 ) have been identiﬁed as modulating factors in the control of various sS-dependent genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7768855" target="_blank">7768855</a>]<br></li>
<li>We show here that in addition to crS , the global regulators Lrp , cyclic AMP ( cAMP ) receptor protein-cAMP complex ( cAMP-CRP ) , and integration host factor ( IHF ) are involved in the control of osmY .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8253679" target="_blank">8253679</a>]<br></li>
<li>In recent studies we have shown that in addition to sS , the cyclic AMP ( cAMP ) - cAMP receptor protein ( cAMP-CRP ) complex , Lrp , and integration host factor ( IHF ) are involved in modulating osmY transcription ( 33 , 56 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7814331" target="_blank">7814331</a>]<br></li>
<li>BMC Microbiology BioMed Central ss Open Acce Research article Limited functional conservation of a global regulator am ong related bacterial genera : Lrp in Escherichia , Proteus and Vibrio Robert E Lintner1 ,5 , Pankaj K Mishra1 ,6 , Poonam Srivastava2 , Betsy M Martinez-Vaz2 ,7 , Arkady B Khodursky2 ,3 and Robert M Blumenthal * 1,4 Address : 1Department of Medical Microbiology and Immunology , University of Toledo Health Sciences Center , Toledo , OH 43614-2598 , USA , 2Department of Biochemistry , Molecular Biology , and Biophysics , University of Minnesota , St. Paul , MN 55108-1022 , USA , 3Biotechnology Institute , University of Minnesota , St. Paul , MN 55108-1022 , USA , 4Program in Bioinformatics and Proteomics/Genomics , University of Toledo Health Sciences Center , Toledo , OH 43614-2598 , USA , 5Current Address : Whitehead Institute for Biomedical Research , 9 Cambridge Center , Cambridge , MA 02142 , USA , 6Current Address : Department of Pediatrics , Wayne State University , Detroit , MI 48201 , USA and 7Current Address : Hamline University , St. Paul , MN 55104 , USA Email : Robert E Lintner - lintner@wi.mit.edu; Pankaj K Mishra - pmishra@med.wayne.edu; Poonam Srivastava - Sriva036@umn.edu; Betsy M Martinez-Vaz - BMartinezVaz01@gw.hamline.edu; Arkady B Khodursky - Khodu001@umn.edu; Robert M Blumenthal * - Robert.Blumenthal@utoledo.edu * Corresponding author Received : 16 November 2007 Accepted : 11 April 2008 Published : 11 April 2008 BMC Microbiology 2008 , 8:60 doi :10.1186 / 1471-2180-8-60 This article is available from : http://www.biomedcentral.com/1471-2180/8/60 © 2008 Lintner et al ; licensee BioMed Central Ltd. .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18405378" target="_blank">18405378</a>]<br></li>
<li>For example , expression of the serC ( pdxF ) - encoded transaminase , which acts between PdxB and PdxA in the de novo PLP pathway ( 40 , 43 ) , is minimal in rich LB medium and maximal in minimal salts-glucose medium due to opposing positive and negative controls exerted by the Lrp and cyclic AMP receptor proteins , respectively ( T.-K .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11844765" target="_blank">11844765</a>]<br></li>
<li>Interestingly , the genes coding for local regulators of the genes for individual amino acids are under the control of Lrp , including AdiY ( a regulator of Arg regulon ) , CysB ( a regulator of the Cys regulon ) , GadW ( a regulator of Glu reg ¬ Lrp Network Y J YhjI AdiY OLm r pR BluR YbaO YedW CadX CadW SdiA PhoP H-NS Cbl CysB TyrR TdcR DeoT TdcA CadX CadW DctR SdiA EnvR TdcA TdcA FabR StpA GadW LeuO GadE GadW GadX DtcR SlyA LeuO BglG StpA QseB Rob LrhA MatA NanR ~ 20 TFs FlhDC FliE LrhA FlhDC ~ 10 TFs FliZ MatA FlhDC QseCB Fig. 6 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2162465" target="_blank">2162465</a>]<br></li>
<li>As an attempt to obtain insights into the regulatory role of Lrp , we performed a PM assay , which allows the detection of cell-growth under a total of 960 culture conditions : the pre-sence of 192 species of carbon source ( PM plates 1 and 2 ) , 96 species of nitrogen source ( PM plate 3 ) , 96 species of phosphorus and sulfur sources ( PM plate 4 ) , 96 species of nutrient supplement ( PM plate 5 ) , 288 chemicals as peptide nitrogen source ( PM plates 6 -- 8 ) , 96 species of osmolyte ( PM plate 9 ) and 96 different pH conditions ( PM plate 10 ) ( Bochner , 2009 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2162465" target="_blank">2162465</a>]<br></li>
<li>Leucine responsiveness of heterologous Lrp proteins Conserved function among regulators depends not only on DNA-binding ( and RNA poymerase-contacting ) properties , but in some cases also on responses to small mole-cule coregulators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18405378" target="_blank">18405378</a>]<br></li>
<li>Many of the proteins that are regulated by Lrp , either directly or indirectly , are members of other regulons , such as those for heat shock , osmolarity , nitrogen assimilation , and pilus-formation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7603430" target="_blank">7603430</a>]<br></li>
<li>Genes that are positively regulated by Lrp , either directly or indirectly , encode proteins involved in the assimilation of nitrogen under nitrogenlimiting conditions ( GltD and GlnA ) , proteins involved in the biosynthesis of amino acids ( IlvIH and SerA ) , and proteins involved in the adaptation of the cell to media of low osmolarity and low temperature ( OmpF ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1346534" target="_blank">1346534</a>]<br></li>
<li>174 , No. 1 Lrp , a Leucine-Responsive Protein , Regulates Branched-Chain Amino Acid Transport Genes in Escherichia coli STEVEN A. HANEY , ' t JILL V. PLATKO ,2 t DALE L. OXENDER ,1 ' 3 AND JOSEPH M. CALVO2 ,4 * Department of Biological Chemistry , University of Michigan Medical School , Ann Arbor , Michigan 48109-06061 ; Sections of Genetics and Development2 and Biochemistry , Molecular and Cell Biology ,4 * Cornell University , Ithaca , New York 14853 ; and Warner Lambert/Parke Davis , Ann Arbor , Michigan 480153 Received 15 August 1991/Accepted 28 October 1991 We investigated the relationship between two regulatory genes , livR and lip , that map near min 20 on the Escherichia coli chromosome .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1729203" target="_blank">1729203</a>]<br></li>
<li>Identiﬁcation of ArgP and Lrp as Transcriptional Regulators of lysP , the Gene Encoding the Speciﬁc Lysine Permease of Escherichia coli † Jimena Ruiz , ‡ Ina Haneburger , and Kirsten Jung * Ludwig-Maximilians-Universita ̈t München , Munich Center for integrated Protein Science ( CiPSM ) at the Department of Biology I , Microbiology , Grosshaderner Strasse 2-4 , 82152 Martinsried , Germany Received 12 July 2010/Accepted 14 March 2011 Expression of lysP , which encodes the lysine-speciﬁc transporter LysP in Escherichia coli , is regulated by the concentration of exogenous available lysine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21441513" target="_blank">21441513</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_50">
Sigma factor selectivity of Escherichia coli RNA polymerase : role for CRP , IHF and Lrp transcription factors .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15576800" target="_blank">15576800</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16621818" target="_blank">16621818</a>]<br></label>
<input id="rp_similarity_50", type="checkbox">
<div class="RP_similar">
<li>Sigma factor selectivity of Escherichia coli RNA polymerase : a role for CRP , IHF and Lrp transcription factors .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15716429" target="_blank">15716429</a>]<br></li></div>
				<label class="RP_collapse_similarity" for="rp_similarity_52">
177 , 3166 -- [ 30 ] Colland , F. , Barth , M. , Hengge-Aronis , R. and Kolb , A. ( 2000 ) r factor selectivity of Escherichia coli RNA polymerase , role for CRP , IHF and Lrp transcription factors .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15251200" target="_blank">15251200</a>]<br></label>
<input id="rp_similarity_52", type="checkbox">
<div class="RP_similar">
<li>Colland , F. , Barth , M. , Hengge-Aronis , R. and Kolb , A. ( 2000 ) sigma factor selectivity of Escherichia coli RNA polymerase : role for CRP , IHF and Lrp transcription factors .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21398637" target="_blank">21398637</a>]<br></li>
<li>Colland , F. , Barth , M. , Hengge-Aronis , R. , and Kolb , A. ( 2000 ) Sigma factor selectivity of Escherichia coli RNA polymerase : a role for CRP , IHF and Lrp transcription factors .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11251833" target="_blank">11251833</a>]<br></li>
<li>Colland , F. , Barth , M. , Hengge-Aronis , R. , and Kolb , A. ( 2000 ) Sigma factor selectivity of Escherichia coli RNA polymer-ase : a role for CRP , IHF and Lrp transcription factors .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11532138" target="_blank">11532138</a>]<br></li></div>
<li>Maximization of Transcription of the serC ( pdxF ) - aroA Multifunctional Operon by Antagonistic Effects of the Cyclic AMP ( cAMP ) Receptor Protein-cAMP Complex and Lrp Global Regulators of Escherichia coli K-12 The arrangement of the Escherichia coli serC ( pdxF ) and aroA genes into a cotranscribed multifunctional operon allows coregulation of two enzymes required for the biosynthesis of L-serine , pyridoxal 5 * - phosphate , chorismate , and the aromatic-amino-acids and vitamins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9171388" target="_blank">9171388</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_57">
Man , T.K. , Pease , A.J. , and Winkler , M.E. ( 1997 ) Maximi-zation of transcription of the serC ( pdxF ) -- aroA multifunctional operon by antagonistic effects of the cyclic AMP ( cAMP ) receptor protein -- cAMP complex and Lrp Global regulators of Escherichia coli K-12 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11985712" target="_blank">11985712</a>]<br></label>
<input id="rp_similarity_57", type="checkbox">
<div class="RP_similar">
<li>Maximization of transcription of the serC ( pdxF ) - aroA multifunctional operon by antagonistic effects of the cyclic AMP ( cAMP ) receptor protein-cAMP complex and Lrp global regulators of Escherichia coli K-12 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11844765" target="_blank">11844765</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9393714" target="_blank">9393714</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9696782" target="_blank">9696782</a>]<br></li></div>
<li>Identiﬁcation of ArgP and Lrp as transcriptional regulators of lysP , the gene encoding the speciﬁc lysine permease of Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21890697" target="_blank">21890697</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22904281" target="_blank">22904281</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22685278" target="_blank">22685278</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23222720" target="_blank">23222720</a>]<br></li>
<li>Discussion In the present work , we have demonstrated that the expression of the gene osmC of E. coli is regulated by a complex s network of several global regulators , including at least s , Lrp and H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9663683" target="_blank">9663683</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_61">
Interplay between global regulators of Escherichia coli : effect of RpoS , Lrp and H-NS on transcription of the gene osmC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15838058" target="_blank">15838058</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15489429" target="_blank">15489429</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10601207" target="_blank">10601207</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12867437" target="_blank">12867437</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11566985" target="_blank">11566985</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11577158" target="_blank">11577158</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12218039" target="_blank">12218039</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11133959" target="_blank">11133959</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14702398" target="_blank">14702398</a>]<br></label>
<input id="rp_similarity_61", type="checkbox">
<div class="RP_similar">
<li>Interplay between global regulators of Escherichia coli : effect of RpoS , H-NS and Lrp on transcription of the gene osmC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15716429" target="_blank">15716429</a>]<br></li></div>
				<label class="RP_collapse_similarity" for="rp_similarity_63">
[ 6 ] J. Bouvier , S. Gordia , G. Kampmann , R. Lange , R. Hengge-Aronis , C. Gutierrez , Interplay between global regulators of Escherichia coli : Effect of RpoS , Lrp and H-NS on transcription of the gene osmC , Mol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16427772" target="_blank">16427772</a>]<br></label>
<input id="rp_similarity_63", type="checkbox">
<div class="RP_similar">
<li>Mol Microbiol 35:845 -- 853 Bouvier J , Gordia S , Kampmann G , Lange R , Hengge-Aronis R , Gutierrez C ( 1998 ) Interplay between global regulators of Escherichia coli : effect of RpoS , Lrp and H-NS on transcription of the gene osmC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12395194" target="_blank">12395194</a>]<br></li>
<li>Bouvier J , Gordia S , Kampmann G , Lange R , Hengge-Aronis R , Guti-errez C : Interplay between global regulators of Escherichia coli : effect of RpoS , Lrp and H-NS on transcription of the gene osmC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18405378" target="_blank">18405378</a>]<br></li>
<li>Bouvier , J. , Gordia , S. , Kampmann , G. , Lange , R. , Hengge-Aronis , R. , and Gutierrez , C. ( 1998 ) Interplay between global regulators of Escherichia coli : effect of RpoS , Lrp and H-NS on transcription of the gene OsmC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17224607" target="_blank">17224607</a>]<br></li>
<li>Bouvier , J. , Gordia , S. , Kampmann , G. , Lange , R. , Hengge-Aronis , R. , and Gutierrez , C. ( 1998 ) Interplay between global regulators of Escherichia coli : effect of RpoS , Lrp , and H-NS on transcription of the gene osmC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10692161" target="_blank">10692161</a>]<br></li>
<li>Bouvier , J. , Gordia , S. , Kampmann , G. , Lange , R. , Hengge-Aronis , R. , and Gutierrez , C. ( 1998 ) Interplay between global regulators of Escherichia coli : effect of RpoS , Lrp and H-NS on transcription of the gene osmC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16238633" target="_blank">16238633</a>]<br></li>
<li>Bouvier , J. , Gordia , S. , Kampmann , G. , Lange , R. , Hengge-Aronis , R. , and Gutierrez , C. ( 1998 ) Interplay between global regulators of Escherichia coli : effect of RpoS , H-NS and Lrp on transcription of the gene osmC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11251833" target="_blank">11251833</a>]<br></li>
<li>Bouvier , J. , Gordia , S. , Kampmann , G. , Lange , R. , Hengge-Aronis , R. and Gutierrez , C. ( 1998 ) Interplay between global regulators of Escherichia coli : effect of RpoS , Lrp and H-NS on transcription of the gene osmC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17881364" target="_blank">17881364</a>]<br></li></div>
<li>The Leucine-Responsive Regulatory Protein , a Global Regulator of Metabolism in Escherichia coli TRODUCTION ... 4 VOLVEMENT OF LRP AND LEUCINE IN REGULATING OPERON EXPRESSION ... 4 Properties of Lrp ... 4 Interaction of Lrp with DNA ... 4 Consensus sequence ... 4 Stoichiometry and cooperativity of binding ... 4 Bending caused by binding ... 4 Lrp binding can be affected by DNA-methylation and vice versa ... 4 Organization and Regulation of the lrp Operon ... 4 Organization of the lrp operon ... 4 Autogenous regulation ... 4 Effect of growth conditions on lrp expression ... 4 Effects of Lrp on Operon Expression ... 4 Direct versus indirect effects ... 4 Can Lrp act from a distance ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li>
<li>The effects of E. coli lrp ( pEcoLrp ) on gene-expression are shown on the x-axis in every panel .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18405378" target="_blank">18405378</a>]<br></li>
<li>The leucine effect seems to be mediated by the global regulator Lrp , which acts as a positive regulator of an endog-enous chromosome-encoded MccJ25 efﬂux pump , YojI .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19074390" target="_blank">19074390</a>]<br></li>
<li>Newman , E.B , , D'Ari , R. , and Lin , R , T , ( 1992 ) The leucine-Lrp regulon in E. coii : a global response in search of a raison d'etre , Ce / / 68 : 617-619 , Piatko , J , V , ( 1991 ) Genetic analysis of Lrp , a Ieucine-respon-sive regulatory protein of Escherichia coii .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8483418" target="_blank">8483418</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_75">
The leucine-Lrp regulon in Escherichia coli : a global response in search of a raison d'etre .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8144448" target="_blank">8144448</a>]<br></label>
<input id="rp_similarity_75", type="checkbox">
<div class="RP_similar">
<li>The leucine-Lrp regulon in E. coli : a global response in search of a raison d' être .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8002608" target="_blank">8002608</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2162465" target="_blank">2162465</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955407" target="_blank">8955407</a>]<br></li>
<li>The leucine-Lrp regulon in E. coli : a global response in search of a raison d'etre .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8071194" target="_blank">8071194</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8188587" target="_blank">8188587</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10463182" target="_blank">10463182</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7901196" target="_blank">7901196</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7665470" target="_blank">7665470</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7768815" target="_blank">7768815</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8195106" target="_blank">8195106</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11575919" target="_blank">11575919</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9733689" target="_blank">9733689</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1447135" target="_blank">1447135</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7751276" target="_blank">7751276</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8253679" target="_blank">8253679</a>]<br></li>
<li>The leucine-Lrp regulon in E. global coli : a response in search of a raison d'etre .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8432705" target="_blank">8432705</a>]<br></li></div>
<li>R.D. , and Lin , R.T. -LCB- 1992 ) The Leucine-Lrp regulon in E. coli : a global response in search of a raison d'etre .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7984106" target="_blank">7984106</a>]<br></li>
<li>Addison-Wesley , Redwood , Calif. , pp 63-103 Newman EB , D'ari R , Lin RT ( 1992 ) The leucine-Lrp regulon in E. coli - a global response in search of a raison d'etre .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8152423" target="_blank">8152423</a>]<br></li>
<li>Mol Microbiol 10:225 -231 Newman EB , D'Ari R , Lin RT ( 1992 ) The leucine-Lrp regulon in E. coli - a global response in search of a raison d'etre .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7603430" target="_blank">7603430</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_82">
Newman , E. B. , D'Ari , R. & Lin , R. T. ( 1992 ) The leucine-Lrp regulon in E. coli : a global response in search of a raison d'etre , Cell 68 , 617-619 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8026499" target="_blank">8026499</a>]<br></label>
<input id="rp_similarity_82", type="checkbox">
<div class="RP_similar">
<li>[ 27 ] E.B. Newman , R. D'Ari , R.T. Lin , The leucine-Lrp regulon in E. coli : A global response in search of a raison d'etre , Cell 68 ( 1992 ) 617 -- 619 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12706508" target="_blank">12706508</a>]<br></li>
<li>Newman , E.B. , D'Ari , R. , and Lin , R.T. ( 1992 ) The leucine-Lrp regulon in E. coli : a global response in search of a raison d'etre .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8809748" target="_blank">8809748</a>]<br></li>
<li>Newman , E.B. , D'Ari , R. , and Lin , R.T. ( 1992 ) The leucine-Lrp regulon in E. coli : a global response in search of a raison d'eÃtre .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10216857" target="_blank">10216857</a>]<br></li>
<li>Newman , E.B. , Lin , R.T. , and D'Ari , R. ( 1992 ) The leucine-Lrp regulon in E. coli : a global response in search of a raison d'eÃtre .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11298273" target="_blank">11298273</a>]<br></li></div>
<li>Genes consistently induced in both data sets are in bold Gene b-number Expression ratio ( set 1 ) Expression ratio ( set 2 ) Description Known genes hslS b3686 soxS b4062 hha b0460 glnA b3870 rhoL b3782 hslT b3687 fimG b4319 rho b3783 tus b1610 metK b2942 16 63 30 1.5 6 4 1.4 5 4 48 35 20 12 5 Heat shock protein Regulation of superoxide response regulon , global regulator Haemolysin expression modulating protein , regulator Glutamine synthetase Leader , RNA synthesis , modification , DNA transcription Heat shock protein Fimbrial morphology Transcription termination factor Rho , polarity suppressor Factor , DNA-replication , repair , restriction/modification Methionine adenosyltransferase 1 ( adomet synthetase ) , methyl and propylamine donor , corepressor of met genes Anthranilate synthase component I 4 3 3 3 3 2 trpE b1264 Unknown genes ybaJ b0461 b2377 b2377 b1112 b1112 b3022 b3022 4 2 37 21 17 8 Unknown Unknown 35 4 Unknown , possible stress response ( Zheng et al. 2001 ) Unknown ( protein 36 % identical to Sinorhizobium melilotiLrp , leucine-responsive regulatory protein , for LPS synthesis ) Unknown Putative ligase Unknown Unknown Putative receptor Unknown Unknow 7 8 b1601 b2912 b1060 b1373 b0836 b1206 b2017 b1601 ygfA b1060 b1373 b0836 ychM yefM 6 5 8 5 5 4 1.3 3 3 3 3 2 3 2 Results E. coli JM109 biofilm-formation on glass wool in batch reactors E. coli JM109 was tested and found to yield good biofilms when grown on glass wool for 7 h ( about 40 μg total RNA was obtained from biofilm cells in each culture ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14727089" target="_blank">14727089</a>]<br></li>
<li>Subsequent screening of factors that parti-cipate in a decreased-motility phenotype identified four transcriptional regulators , LrhA , Lrp , SlyA and P fimbriae protein PapX , that partially restore motility without affecting type 1 fimbriae expression ( Simms & Mobley , 2008a ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22422754" target="_blank">22422754</a>]<br></li>
<li>Continued growth in rich medium would lead to further reductions in gltBDF expression as the concentration of Lrp in the cells decreased as a result of protein-degradation and/or dilution during cell-division .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li>
<li>Growth curves ( A-C , MOPS glucose medium ; G-I , MOPS rich medium ) and Lrp protein levels ( D-F , glucose ; J-L , rich ) are shown .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18405378" target="_blank">18405378</a>]<br></li>
<li>New Biol 3 : 615-625 Gazeau M , Delort F , Dessen P , Blanquet S , Plateau P ( 1992 ) Escherichia-colileucine-responsiveregulatory protein ( Lrp ) controls lysyl-transfer RNA synthetase expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8152423" target="_blank">8152423</a>]<br></li>
<li>Lrp is thought to be evolutionarily related to AsnC , a regulatory protein that can positively stimulate the expression of one operon and negatively affect its own promoter ( 47 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1917830" target="_blank">1917830</a>]<br></li>
<li>For operons controlled by Lrp , does Lrp act directly to activate or repress the transcription of target operons , or does it act indirectly by affecting the expression of some other regulatory gene which , in turn , affects the expression of a target operon ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1447135" target="_blank">1447135</a>]<br></li>
<li>coli , codes for a protein that binds to anti-Lrp antibodies and that functions , albeit poorly , as an activator of the E. coli ilvIH promoter ( 118a ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li>
<li>Lrp , a global regulat-ory protein of E. coli , binds cooperatively to multiple sites and activates transcription of ilvIH .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9236118" target="_blank">9236118</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_96">
Lrp , a global regulatory protein of Escbericbia coli , binds co-operatively to multiple sites and activates transcription of ilvIH .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7881552" target="_blank">7881552</a>]<br></label>
<input id="rp_similarity_96", type="checkbox">
<div class="RP_similar">
<li>Lrp , a global regulatory protein of Esch-erichia coli , binds co-operatively to multiple sites and activates transcription of ilvIH .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955407" target="_blank">8955407</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8002608" target="_blank">8002608</a>]<br></li></div>
				<label class="RP_collapse_similarity" for="rp_similarity_98">
Wang , Q. , and Calvo , J.M. ( 1993 ) Lrp , a global regulatory protein of E. coli binds cooperatively to multiple sites and activates transcription of ilvIH .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9663683" target="_blank">9663683</a>]<br></label>
<input id="rp_similarity_98", type="checkbox">
<div class="RP_similar">
<li>Lrp , a global regulatory protein of Esch-erichia coli , binds cooperatively to multiple sites and activates transcription of ilvIH .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7592387" target="_blank">7592387</a>]<br></li>
<li>Lrp , a global regulatory protein of E. coli , binds cooperatively to multiple sites and activates transcription of ilvIH .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12218014" target="_blank">12218014</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7901196" target="_blank">7901196</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11575919" target="_blank">11575919</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8144448" target="_blank">8144448</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955316" target="_blank">8955316</a>]<br></li></div>
<li>Lrp , a global regulatory protein of Escherichia col , binds co-operatively to multiple sites and activates transcription of ilvIH .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7928983" target="_blank">7928983</a>]<br></li>
<li>Wang , Q. and Calvo , J.M , ( 1993 ) Lrp , a global regulatory protein of Escherichia coii , binds co-operatively to multiple sites and activates transcription of iiviH .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8483418" target="_blank">8483418</a>]<br></li>
<li>Lrp , a global regulat-ory protein of Escherichia coli , binds co-operatively to multiple sites and activates transcription of ilvIH .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9512707" target="_blank">9512707</a>]<br></li>
<li>Nucleic Acids Res 4:2725 -2745 Wang Q , Calvo JM ( 1993a ) Lrp , a global regulatory protein of Escherichia coli , binds co-operatively to multiple sites and activates transcription of ilvlH .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7603430" target="_blank">7603430</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_105">
Lrp , a global regulatory protein of Escherichia coli , binds co-operatively to multiple sites and activates transcription of ilvZH .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9611815" target="_blank">9611815</a>]<br></label>
<input id="rp_similarity_105", type="checkbox">
<div class="RP_similar">
<li>Lrp , a global regulatory protein of Escherichia coli , binds co-operatively to multipl ; sites and activates transcription of ilvZH .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9802028" target="_blank">9802028</a>]<br></li>
<li>Lrp , a global regulatory protein of Escherichia coli , binds co-operatively to multiple sites and activates transcription of ilvIH .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8071194" target="_blank">8071194</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8253679" target="_blank">8253679</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7916011" target="_blank">7916011</a>]<br></li>
<li>Wang , Q. , and Calvo , J.M. ( 1993 ) Lrp , a global regulatory protein of Escherichia coli , binds co-operatively to multiple sites and activates transcription of ilvIH .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10216857" target="_blank">10216857</a>]<br></li>
<li>Wang , Q. , and Calvo , J.M. ( 1993a ) Lrp , a global regulatory protein of Escherichia coli , binds co-operatively to multiple sites and activates transcription of ilvIH .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11985712" target="_blank">11985712</a>]<br></li>
<li>Wang , Q. , and Calvo , J.M. ( 1993b ) Lrp , a global regulatory protein of Escherichia coli , binds co-operatively to multiple sites and activates transcription of ilvIH .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9515701" target="_blank">9515701</a>]<br></li></div>
<li>And lrp gene product , Leucine-responsive regulatory protein ( Lrp ) is known to be the positive regulator required for activating ilvIH transcription activity ( 18 ) which is crucial for initiating a promoter relay mechanism ( 7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11006083" target="_blank">11006083</a>]<br></li>
<li>A second regulator , cAMP/CRP , activates P2 , an effect that is largely inhibited by Lrp , such that catabolite repressor protein ( Crp ) and Lrp are rival activators of serA transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11985712" target="_blank">11985712</a>]<br></li>
<li>Department of Molecular Biology and Genetics , Cornell University , Ithaca NY 14853 , USA 2 Protein Chemistry , P ® zer Inc Groton , CT 06340 , USA # 2001 Academic Press Keywords : Lrp ; Escherichia coli ; leucine ; oligomerization ; global regulation * Corresponding author Introduction Lrp is a global regulator of metabolism in Escher-ichia coli that helps cells respond to changes in environmental conditions .1,2 Many Lrp-regulated operons are also subject to control by leucine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11575919" target="_blank">11575919</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_114">
Use of an in vivo titration method to study a global regulator : effect of varying Lrp levels on expression of gltBDF in Escherichiu coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9802028" target="_blank">9802028</a>]<br></label>
<input id="rp_similarity_114", type="checkbox">
<div class="RP_similar">
<li>Use of an in vivo titration method to study a global regulator : the effect of varying Lrp levels on the expression of gltBDF in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9236118" target="_blank">9236118</a>]<br></li>
<li>Borst , D.W. , Blumenthal , R.M. , and Matthews , R.G. ( 1996 ) Use of an in vivo titration method to study a global regulator : effect of varying Lrp levels on expression of gltBDF in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9515701" target="_blank">9515701</a>]<br></li></div>
<li>Unexpected coregulator range for the global regulator Lrp of Escherichia coli and Proteus mirabilis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2162465" target="_blank">2162465</a>]<br></li>
<li>We have studied regulatory extrapolation by examining a well-conserved global regulator ( Lrp ) , conserved genes that are Lrp regulatory targets in E. coli , and two species of increasing but limited genetic distance from E. coli : Proteus mirabilis and Vibrio cholerae .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18405378" target="_blank">18405378</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_119">
Integration of regulatory signals through involvement of multiple global regulators : control of the Escherichia coli gltBDF operon by Lrp , IHF , CRP and ArgR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17850814" target="_blank">17850814</a>]<br></label>
<input id="rp_similarity_119", type="checkbox">
<div class="RP_similar">
<li>Integration of regulatory signals through involvement of multiple global regulators : control of the Escherichia coli gltBDF operon by Lrp , IHF , Crp , and ArgR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25845847" target="_blank">25845847</a>]<br></li></div>
				<label class="RP_collapse_similarity" for="rp_similarity_121">
Evidence for global regulatory control of pilus expression in Escherichia coli by Lrp and DNA-methylation : model building based on analysis of pap .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9813115" target="_blank">9813115</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9236118" target="_blank">9236118</a>]<br></label>
<input id="rp_similarity_121", type="checkbox">
<div class="RP_similar">
<li>Evidence for a global regulatory control of pilus expression in Escherichia coli by Lrp and DNA-methylation : model building based on analysis of pap .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8195106" target="_blank">8195106</a>]<br></li></div>
				<label class="RP_collapse_similarity" for="rp_similarity_123">
EMBO J 12:1039 -1046 van der Woude M , Braaten B , Low D ( 1992 ) Evidence for global regulatory control of pilus expression in Escherichia coli by Lrp and DNA-methylation : model buildingbased on analysis of pap .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7603430" target="_blank">7603430</a>]<br></label>
<input id="rp_similarity_123", type="checkbox">
<div class="RP_similar">
<li>van der Woude , M.W. , Braaten , B.A. , and Low , D.A. ( 1992 ) Evidence for global regulatory control of pilus expression in Escherichia coli by Lrp and DNA-methylation : model building based on analyses of pap .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19906180" target="_blank">19906180</a>]<br></li>
<li>Mol Gen Genet 236:171 -178 van der Woude M , Braaten B , Low D ( 1992 ) Evidence for global regulatory control of pilus expression in Escherichia coli by Lrp and DNA-methylation : model building based on analysis of pap .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8152423" target="_blank">8152423</a>]<br></li></div>
<li>Lrp , a site-specific DNA-binding protein , controls a regulon of at least 40 genes in Escherichia coli , including fimbrial expression and amino-acid-transport , degradation , and biosyn-e Leucine-Responsive Regulatory Protein Binds to the fi Switch To Control Phase Variation of Type 1 Fimbrial Expression in Escherichia coli K-12 DAVID L. GALLY , THOMAS J. RUCKER , AND IAN C. BLOMFIELD * Department of Microbiology and Immunology , Bowman Gray School of Medicine , Winston-Salem , North Carolina 27157-1064 Received 3 March 1994/Accepted 11 July 1994 h m Phase variation of type 1 fimbriation in Escherichia coli is associated with the site-specific recombination of a 314-bp DNA invertible element .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7916011" target="_blank">7916011</a>]<br></li>
<li>If Lrp coordinates ammonia assimilation with amino acid catabolism , then leucine is an indicator of amino acid sufﬁciency .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12730324" target="_blank">12730324</a>]<br></li>
<li>The structural gene for Lrp was initially identified as a mutation in a regulatory gene , livR , for the branched-chain amino acid ( L-leucine , L-isoleucine , and L-valine ) transport system ( 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8900118" target="_blank">8900118</a>]<br></li>
<li>Mol Microbiol 3:1287 -1294 Haney SA , Platko JV , Oxender DL , Calvo JM ( 1992 ) Lrp , a leu-cine-responsive protein , regulates branched-chain-amino-acid-transport genes in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8152423" target="_blank">8152423</a>]<br></li>
<li>Haney , S.A. , Platko , J.V. , Oxender , D.L. , and Calvo , J.M. ( 1992 ) Lrp , a leucine-responsive protein , regulates branched-chain-amino-acid-transport genes in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10216857" target="_blank">10216857</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_131">
Lrp , a leucine-responsive protein , regulates branchedchain amino-acid-transport genes in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8432705" target="_blank">8432705</a>]<br></label>
<input id="rp_similarity_131", type="checkbox">
<div class="RP_similar">
<li>Lrp , a leucine-responsive protein , regulates branched-chain-amino-acid-transport genes in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8071194" target="_blank">8071194</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2162465" target="_blank">2162465</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7901196" target="_blank">7901196</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1346534" target="_blank">1346534</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11065367" target="_blank">11065367</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9236118" target="_blank">9236118</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8253679" target="_blank">8253679</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8144448" target="_blank">8144448</a>]<br></li></div>
<li>70 The E. coli nucleoid-associated proteins including Fis ( factor for inversion stimulation ) , H-NS , IHF ( integration host factor ) and Lrp ( leucine-responsive regulatory protein ) are known to be involved in DNA compaction as well as in transcriptional-control of gene-expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10760142" target="_blank">10760142</a>]<br></li>
<li>The RpoS and Lrp regulatory protein are not involved in hns expression A number of stationary-phase-induced genes of E. coli are regulated by RpoS ( KatF ) , a putative alternative sigma factor ( Lange and Hengge-Aronis , 1991 ; Siegele and Kolter , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8355613" target="_blank">8355613</a>]<br></li>
<li>The indirect regulation of glutamine synthetase and other proteins of the Ntr regulon by Lrp provides a working model for the way in which a single regulatory protein can control the expression of a large number of gene products .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1346534" target="_blank">1346534</a>]<br></li>
<li>Thus , as in the case of other nucleoid proteins , Lrp is a bifunctional protein , playing a regulatory role in gene-expression and an architectural role in nucleoid organization .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2162465" target="_blank">2162465</a>]<br></li>
<li>Kutukova , E.A. , Livshits , V.A. , Altman , I.P. , Ptitsyn , L.R. , Zyiatdinov , M.H. , Tokmakova , I.L. , and Zakataeva , N.P. , The yeaS ( leuE ) Gene of Escherichia coli Encodes an Exporter of Leucine , and the Lrp Protein Regulates Its Expression , FEBS Lett .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17025177" target="_blank">17025177</a>]<br></li>
<li>The leucine/Lrp regulon inﬂuences expression of a large number of Escherichia coli genes ( 6 , 16 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7751276" target="_blank">7751276</a>]<br></li>
<li>FEBS 29827 The yeaS ( leuE ) gene of Escherichia coli encodes an exporter of leucine , and the Lrp protein regulates its expression Ekaterina A. Kutukova , Vitaliy A. Livshits * , Irina P. Altman , Leonid R. Ptitsyn , Michael H. Zyiatdinov , Irina L. Tokmakova , Natalia P. Zakataeva Ajinomoto-Genetika Research Institute , Laboratory N2 , 1-st Dorozhny Proezd , b. 1 , Moscow 117545 , Russia Received 29 April 2005 ; revised 1 July 2005 ; accepted 12 July 2005 Available online 10 August 2005 Edited by Felix Wieland Abstract Overexpression of the yeaS gene encoding a protein belonging to the RhtB transporter family conferred upon cells resistance to glycyl-L-leucine , leucine analogues , several amino acids and their analogues .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16098526" target="_blank">16098526</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_140">
The yeaS ( leuE ) gene of Esche-richia coli encodes an exporter of leucine , and the Lrp protein regulates its expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25845847" target="_blank">25845847</a>]<br></label>
<input id="rp_similarity_140", type="checkbox">
<div class="RP_similar">
<li>The yeaS ( leuE ) gene of Escherichia coli encodes an exporter of leucine , and the Lrp protein regulates its expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17504942" target="_blank">17504942</a>]<br></li></div>
<li>Kutukova , E.A. , Livshits , V.A. , Altman , I.A. , Ptitsyn , L.R. , Zyiatdinov , M.H. , Tokmakova , I.L. , and Zakataeva , N.P. ( 2005 ) The yeaS ( leuE ) gene of Escherichia coli encodes an exporter of leucine , and the Lrp protein regulates its expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19906180" target="_blank">19906180</a>]<br></li>
<li>( Lrp ) controls the expression of both the Pap and Fan pili operons in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8144448" target="_blank">8144448</a>]<br></li>
<li>Characterization of Lrp , an Escherichia coli Regulatory Protein That Mediates a Global Response to Leucine * Exogenousleucine affects the expression of a number of different operons in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2040596" target="_blank">2040596</a>]<br></li>
<li>Mol Gen Genet ( 1995 ) 247:521 -528 © Springer-Verlag 1995 Taku Oshima • Koichi Ito • Hiroyuki Kabayama Yoshikazu Nakamura Regulation of Irp gene-expression by H-NS and Lrp proteins in Escherichia coli : dominant negative mutations in Irp Received : 3 October 1994 / Accepted : 17 December 1994 A b s t r a c t Lrp ( leucine-responsive regulatory protein ) is a global transcription factor of Escherichia coli and regulates , negatively or positively , many genes including lysU , which encodes lysyl-tRNA synthetase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7603430" target="_blank">7603430</a>]<br></li>
<li>Oshima , T. , Ito , K. , Kabayama , H. , and Nakamura , Y. ( 1995 ) Regulation of lrp gene-expression by H-NS and Lrp proteins in Escherichia coli : dominant negative mutations in lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16238633" target="_blank">16238633</a>]<br></li>
<li>Lrp , leucine-responsive protein ; OxyR , oxidation stress regulator ; RpoS , RNA polymerase sigma factor for stationary phase ; OmpR , outer membrane protein regulator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7559363" target="_blank">7559363</a>]<br></li>
<li>Studies are in progress to determine whether ompA deletion affects known regulators of ﬁmB and ﬁmE such as leucine response regulatory protein ( Lrp ) , integration host factor , and OmpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16988236" target="_blank">16988236</a>]<br></li>
<li>Expression of both serA and serC is activated by the leucine-responsive regulatory protein , Lrp , a global regulator ﬁrst recognized as Rbl ( regulation by leucine ; Lin et al. , 1990 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11985712" target="_blank">11985712</a>]<br></li>
<li>Here , we demonstrate that expression of the E. coli argO gene is affected not only by the internal concentrations of arginine and lysine through the action of ArgP , but also by the general metabolic state of the cell , through regulation by the global transcriptional regulator Lrp ( leucine-responsive regulatory protein ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19906180" target="_blank">19906180</a>]<br></li>
<li>In addition , lysP expression is ﬁne-tuned by the global regulator Lrp ( leucine-responsive protein ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21441513" target="_blank">21441513</a>]<br></li>
<li>Our interest in gltBDF , the operon that includes structural genes for GltBD , stems from its membership in the regulon controlled by Lrp ( leucine-responsive regulatory protein ) [ 5 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17233899" target="_blank">17233899</a>]<br></li>
<li>The leucine-responsive regulatory protein ( Lrp ) has been shown to both positively and negatively regulate transcription of a number of genes in response to exogenous leucine ( 215 ) ; Lrp action is signiﬁcantly activated by the absence of L-leucine in the growth medium , whereas it is repressed in the presence of L-leucine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16760308" target="_blank">16760308</a>]<br></li>
<li>The diverse set of genes regulated by Lrp suggested a regulon controlled by Lrp , with the effect of Lrp on expres ¬ The leucine-responsive regulatory protein ( Lrp ) has been shown to regulate , either positively or negatively , the transcription of several Escherichia coli genes in response to leucine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1346534" target="_blank">1346534</a>]<br></li>
<li>The mutations in these strains were shown to lie within a previously identified gene ( lrp ) that encodes Lrp ( leucine-responsive regulatory protein ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1917830" target="_blank">1917830</a>]<br></li>
<li>The leucine-responsive regulatory protein ( Lrp ) controls the synthesis of glutamate synthase , which controls the Ntr response , presumably through its effect on intracellular glutamine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12730324" target="_blank">12730324</a>]<br></li>
<li>The leucine response regulatory protein ( Lrp ) is known to activate glutamate synthase expression in E. coli ( 20 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955321" target="_blank">8955321</a>]<br></li>
<li>The histone-like leucine-responsive regulatory protein ( Lrp ) binds DNA nonspeciﬁcally as an octamer , and also increases Dam 's processivity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25785689" target="_blank">25785689</a>]<br></li>
<li>The leucine regulatory protein ( Lrp ) controls a wide array of operons both as an activator ( e.g. ilvlH ) and as a repressor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8152423" target="_blank">8152423</a>]<br></li>
<li>Currently , four proteins , the leucine-responsive regulatory protein ( Lrp ) , the purine repressor protein ( PurR ) , the glycine cleavage activator protein ( GcvA ) , and the glycine cleavage repressor protein ( GcvR ) , have been shown to be involved in regulating expression of the gcv operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9537378" target="_blank">9537378</a>]<br></li>
<li>In addition , leucine-responsive regulatory protein ( Lrp ) acts as a positive modulator of csiD expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9512707" target="_blank">9512707</a>]<br></li>
<li>Introduction Lrp ( leucine-responsive reguiatory protein ) has recently attracted attention as an Esaherichia coti master regulatory protein that influences the expression of more than a dozen different operons ( Ernsting et ai .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8483418" target="_blank">8483418</a>]<br></li>
<li>In this study , we present results showing that the leucine-responsive regulatory protein , Lrp , is involved in this phenotype by acting as a positive regulator of YojI , a chromosomally encoded efﬂux pump which expels microcin out of cells .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19074390" target="_blank">19074390</a>]<br></li>
<li>As this work was in progress , Lrp ( leucine-regulatory protein ) was shown to be involved in the regulation of several leucine-regulated genes ( 31 , 40a , 43 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1917830" target="_blank">1917830</a>]<br></li>
<li>In addition to cAMP-CRP , the leucine-responsive regulatory protein ( Lrp ) is involved in the regulation of csiD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9512707" target="_blank">9512707</a>]<br></li>
<li>Also , the leucine-responsive regulatory protein ( Lrp ) is involved in the regulation of certain S-depen-dent genes ( 9 , 13 , 33 , 64 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15716429" target="_blank">15716429</a>]<br></li>
<li>The first protein shown to be involved in the regulation of gcv is Lrp , or leucine-responsive regulatory protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8188587" target="_blank">8188587</a>]<br></li>
<li>Leucine-responsive regulatory protein ( Lrp ) is a transcription regulator that controls the expression of a number of genes , many of them , but not all , involved in biosynthetic and degradative properties of the cells ( Newman et al. , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9663683" target="_blank">9663683</a>]<br></li>
<li>Leucine-responsive Regulatory Protein-DNA Interactions in the Leader Region of the ilvGMEDA Operon of Escherichia coli * ( Received for publication , June 13 , 1996 , and in revised form , August 7 , 1996 ) Kyu Young Rhee , Bhavin S. Parekh , and G. Wesley Hatfield ‡ From the Department of Microbiology and Molecular Genetics , College of Medicine , University of California , Irvine , California 92697 The leucine-responsive regulatory protein ( Lrp ) regulates the expression of many operons in Escherichia coli including several involved in the metabolism of the branched-chain amino acids , L-isoleucine , L-valine , and L-leucine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8900118" target="_blank">8900118</a>]<br></li>
<li>The leucine-responsive regulatory protein ( Lrp ) is a regulatory factor that controls , either positively or negatively , the expression of many genes , including numerous operons involved in amino-acid-metabolism ( reviewed in references 3 and 21 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955407" target="_blank">8955407</a>]<br></li>
<li>The leucine-responsive regulatory protein Lrp was known to control the expression of a large number of genes , including the ilvIH operon , but not ilvBN in E. coli ( Jafri et al. , 2002 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16164569" target="_blank">16164569</a>]<br></li>
<li>The leucine-responsive protein ( Lrp ) is a global regulatory protein that controls the expression of more than 40 genes in E. coli , some of which are activated and other repressed .62,63 Lrp is an import-ant regulator of nitrogen and carbon metabolism and may play a crucial role the cell 's adaptation to nutritionally poor and rich media .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11601842" target="_blank">11601842</a>]<br></li>
<li>Discussion The leucine-responsive regulatory protein ( Lrp ) of E. coli controls a regulon which includes genes involved in amino-acid-metabolism and ﬁmbriae biosynthesis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9515701" target="_blank">9515701</a>]<br></li>
<li>The leucine-responsive regulatory protein Lrp of Escherichia coli controls the expression of a large number of genes , including some involved in amino acid biosynthesis , amino acid deg-radation , transport of metabolites , pilus-formation , and one-carbon-metabolism ( 2 , 14 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12218014" target="_blank">12218014</a>]<br></li>
<li>Introduction The leucine-responsive regulatory protein ( Lrp ) is a global regulatory factor that controls the expression of numerous genes in Escherichia coli including many operons involved in the metabolism of amino acids and in the formation of a Q 1999 Blackwell Science Lt Results Identi ® cation of multiple Lrp binding sites in the dad promoter region We have shown previously that Lrp protected a large area of the dad promoter-regulatory region from cleavage by DNase I or hydroxyl radical ( Mathew et al. , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10216857" target="_blank">10216857</a>]<br></li>
<li>Many of these operons are involved in amino acid biosynthesis , Summary The leucine-responsive regulatory protein ( Lrp ) is a global regulator that controls the expression of nume-rous operons in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9515701" target="_blank">9515701</a>]<br></li>
<li>Lrp is a global regulatory protein involved in the control of transcription of numerous genes involved in amino-acid-metabolism ( 6 ) and is required for normal induction of gcv ( 11 , 21 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9537378" target="_blank">9537378</a>]<br></li>
<li>The leucine-responsive protein ( Lrp ) , a global regulator involved in the control of transcription of several genes involved in amino-acid-metabolism ( Calvo & Matthews , 1994 ) , binds to multiple sites in the gcv control region from bp ® 229 to ® 92 relative to the transcriptional start site ( Fig. 1 ) ( Stauffer & Stauffer , 1994 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12101307" target="_blank">12101307</a>]<br></li>
<li>It is noteworthy that both the artP and the wrbA promoter are controlled by the global regulator Lrp ( leucine responsive protein ) ( Table 2 ) , the main regulator of amino acid biosynthesis and metabolism .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15489429" target="_blank">15489429</a>]<br></li>
<li>The leucine-responsive protein , Lrp , is a global regulator of genes involved in amino-acid-metabolism ( 5 ) and is required for activation of the gcv operon ( 22 , 41 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10074087" target="_blank">10074087</a>]<br></li>
<li>The leucine responsive regulatory protein ( Lrp ) is a global regulator in E. coli and is known to activate or repress many genes involved in amino-acid-metabolism ( 1 , 14 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9733689" target="_blank">9733689</a>]<br></li>
<li>The leucine-responsive regulatory protein ( Lrp ) is another global regulator in E. coli and has been shown to activate or repress many genes involved in amino-acid-metabolism ( Calvo & Matthews , 1994 ; Newman et al. , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10463182" target="_blank">10463182</a>]<br></li>
<li>Further , leucine acts as a signaling molecule regulating the leucine-responsive regulatory protein , Lrp ( 28 ) , a global regulator involved in modulating a variety of metabolic functions including the catabolism and anabolism of amino acids as well as pili synthesis ( 29 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17289841" target="_blank">17289841</a>]<br></li>
<li>Impact Statement Leucine-responsive regulatory protein ( Lrp ) is known as a global regulator of the genes for transport , biosynthesis and catabolism of amino acids to establish their balance needed for protein-synthesis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2162465" target="_blank">2162465</a>]<br></li>
<li>Research Expanded roles of leucine-responsive regulatory protein i transcription regulation of the Escherichia coli genome : Genomic SELEX screening of the regulation targets n Tomohiro Shimada ,1,2,3 Natsumi Saito ,4,5 Michihisa Maeda ,6 Kan Tanaka1 and Akira Ishihama2 ,3 1 Chemical Resources Laboratory , Tokyo Institute of Technology , Nagatsuta , Yokohama , Japan 2 Department of Frontier Bioscience , Hosei University , Koganei , Tokyo , Japan 3 Research Center for Micro-Nano Technology , Hosei University , Koganei , Tokyo , Japan 4 Department of Chemistry and Material Engineering , Tsuruoka National College of Technology , Yamagata , Japan 5 Institute for Advanced Biosciences , Keio University , Yamagata , Japan 6 School of Agriculture , Meiji University , Kawasaki , Kanagawa , Japan Correspondence : Tomohiro Shimada ( tshimada@res.titech.ac.jp ) DOI : 10.1099 / mgen .0.000001 Leucine-responsive regulatory protein ( Lrp ) is a transcriptional regulator for the genes involved in transport , biosynthesis and catabolism of amino acids in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2162465" target="_blank">2162465</a>]<br></li>
<li>Lrp was ﬁrst identiﬁed in E. coli as a regulatory protein involved in the control of the transport of branchedchain amino acids ( Anderson et al. , 1976 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2162465" target="_blank">2162465</a>]<br></li>
<li>Mol Microbiol 6:2429 -2435 Wang Q , Calvo J ( 1993a ) Lrp , a global regulatory protein ofE .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8152423" target="_blank">8152423</a>]<br></li>
<li>J Mol Biol 229:306 -318 Wang Q , Calvo JM ( 1993b ) Lrp , a global regulatory protein in Escherichia coli , bends DNA and can organize the assembly of a higher-order nucleoprotein structure .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7603430" target="_blank">7603430</a>]<br></li>
<li>Lrp , a global regulatory protein in Esch-erichia coli , bends DNA and can organize the assembly of a higher-order nucleoprotein structure .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8002608" target="_blank">8002608</a>]<br></li>
<li>Wiliins , D.A. , Ryan , C.W , , Piatko , J.V , , and Calvo , J.M , ( 1991 ) Characterization of Lrp , an Escherichia coii regulatory protein that mediates a global response to leucine , J Bioi Chem 266:10768 -10774 , Xiao , H , , Perisic , O. , and Us , J.T , ( 1991 ) Cooperative binding of Drosophiia heat shock factor to arrays of a conserved 5 bp unit .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8483418" target="_blank">8483418</a>]<br></li>
<li>J Bacteriol 176:1831 -1839 Willins DA , Ryan CW , Platko JV , Calvo JM ( 1991 ) Characterization of Lrp , an Escherichia coli regulatory protein that mediates a global response to leucine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7603430" target="_blank">7603430</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_192">
EMBO J 12:2495 -2501 Willins D , Ryan C , Platko J , Calvo J ( 1991 ) Characterization of Lrp , an Escherichia coli regulatory protein that mediates a global response to leucine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8152423" target="_blank">8152423</a>]<br></label>
<input id="rp_similarity_192", type="checkbox">
<div class="RP_similar">
<li>Willins , D.A. , Ryan , C.W. , Platko , J.V. and Calvo , J.M. ( 1991 ) Characterization of Lrp , an Escherichia coli regulatory protein that mediates a global response to leucine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16528101" target="_blank">16528101</a>]<br></li>
<li>Willins , A.K. , Ryan , C.W. , Platko , J.V. , and Calvo , J.M. ( 1991 ) Characterization of Lrp , an Escherichia coli regulatory protein that mediates a global response to leucine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9515701" target="_blank">9515701</a>]<br></li>
<li>Characterization of Lrp , an Escherichia coli global regulatory protein that mediates a global response to leucine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8002608" target="_blank">8002608</a>]<br></li>
<li>Characterization of Lrp , an Escherichia coli regulatory protein that mediates a global response to leucine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8071194" target="_blank">8071194</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12218014" target="_blank">12218014</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7901196" target="_blank">7901196</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1346534" target="_blank">1346534</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7916011" target="_blank">7916011</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11575919" target="_blank">11575919</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1729203" target="_blank">1729203</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7592387" target="_blank">7592387</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955316" target="_blank">8955316</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1917830" target="_blank">1917830</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8432705" target="_blank">8432705</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8144448" target="_blank">8144448</a>]<br></li>
<li>Characterization of Lrp , and Escherichia coli regulatory protein that mediates a global response to leucine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2162465" target="_blank">2162465</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9813115" target="_blank">9813115</a>]<br></li>
<li>Characterization of an coli Lrp , Escherichia regulatory protein that mediates a global response to leucine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1447135" target="_blank">1447135</a>]<br></li>
<li>Characterization of Lrp , an Eschenichia coli regulatory protein that mediates a global response to leucine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1532173" target="_blank">1532173</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li></div>
<li>A om between the L o n herichia col a d DNA f the U DF sc i Don E. Wiese II1 , Brian R. Ernsting1 , Robert M. Blumenthal2 and Rowena G. Matthews1 * 1 Department of Biological Chemistry and Biophysics Research Division , University of Michigan , Ann Arbor , MI 48109-1055 , USA 2 Department of Microbiology and Immunology , Medical College of Ohio , Toledo , OH 43699-0008 , USA The global regulator Lrp ( leucine-responsive regulatory protein ) , in some cases modulated by its co-regulator leucine , has been shown to regulate more than 40 genes and operons in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9236118" target="_blank">9236118</a>]<br></li>
<li>Pap phase variation is known to involve the formation of differential DNA-methylation patterns and to require the presence of the leucine-responsive regulatory protein ( Lrp ) , a global transcriptional regulator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11972787" target="_blank">11972787</a>]<br></li>
<li>The glutamate uptake regulatory protein ( Grp ) of Zymomonas mobilis and its relation to the global regulator Lrp of Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10850996" target="_blank">10850996</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_203">
Lrp ( Leucine-responsive regulatory protein ) 1 is a recently recognized global regulator of metabolism in Escherichia coli ( reviewed in Refs .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8636076" target="_blank">8636076</a>]<br></label>
<input id="rp_similarity_203", type="checkbox">
<div class="RP_similar">
<li>Lrp ( leucine-responsive regulatory protein ) is a recently recognized global regulator of metabolism in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955316" target="_blank">8955316</a>]<br></li>
<li>The leucine-responsive regulatory protein ( Lrp ) , a global regulator of metabolism in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12218014" target="_blank">12218014</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955316" target="_blank">8955316</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11575919" target="_blank">11575919</a>]<br></li></div>
<li>( 2001 ) 312 , 625 ± 635 ucine-regulated Self-association f L onsive Regulatory Protein ( Lrp ) from herichia coli Le o eucine ¬ resp Esc Shaolin Chen1 , Michele H. Rosner2 and Joseph M. Calvo1 * 1 Lrp is a global regulatory protein in Escherichia coli that activates expression of more than a dozen operons and represses expression of another dozen .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11575919" target="_blank">11575919</a>]<br></li>
<li>An analysis of operons in E. coli whose expression is affected by Lrp suggested that Lrp is a global regulatory protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11575919" target="_blank">11575919</a>]<br></li>
<li>J Bacteriol 174:1109 -1118 Gazeau M , Delort F , Dessen P , Blanquet S , Plateau P ( 1992 ) Escherichia coIi leucine-responsive regulatory protein ( Lrp ) controls lysyl-tRNA synthetase expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7603430" target="_blank">7603430</a>]<br></li>
<li>Braaten , B.A. , Platko , J.V. , van der Woude , M.W. , Simons , B.H. , de Graaf , F.K. , Calvo , J.M. , et al. ( 1992 ) Leucine-responsive regulatory protein ( Lrp ) controls the expression of both the pap and fan pili operons in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10216857" target="_blank">10216857</a>]<br></li>
<li>Summary Lrp , a major regulatory protein in Escherichia coli , controis the expression of numerous operons , inciuding / / WH .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8483418" target="_blank">8483418</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_211">
Leucine responsive regulatory protein ( Lrp ) controls the expression of both the Pap and Fan pili operons in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9236118" target="_blank">9236118</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></label>
<input id="rp_similarity_211", type="checkbox">
<div class="RP_similar">
<li>Leucine-responsive regulatory protein ( Lrp ) controls the expression of both the pap and fan pili operons in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8195106" target="_blank">8195106</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20118257" target="_blank">20118257</a>]<br></li></div>
<li>d Lrp ( leucine-responsive regulatory protein ) is an Escherichia coli master regulator affecting the expression of at least 18 operons with known functions ( 6 , 18 ) and many others with unknown functions , identified either by two-dimensional electrophoresis ( 9 ) or by the analysis of mutants generated by placMu transposition ( 16 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8071194" target="_blank">8071194</a>]<br></li>
<li>K e y w o r d s Lrp • Dominant negative mutation • H-NS • lysU • lrp autoregulation Introduction Lrp ( leucine-responsive regulatory protein ) is a global regulatory protein of Escherichia coli , and controls expression of a number of operons .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7603430" target="_blank">7603430</a>]<br></li>
<li>Lrp ( leucine-responsive regulatory protein ) , a newly recognized global regulatory protein in Escherichia coli , regulates the expression of at least 12 operons ( 3 , 11 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1447135" target="_blank">1447135</a>]<br></li>
<li>The leucine-responsive regulatory protein , Lrp ,1 is a global regulatory protein of Escherichia coli that affects the expression of many operons ( reviewed in Refs .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8900118" target="_blank">8900118</a>]<br></li>
<li>Consensus Sequence for Binding N Lrp ( leucine-responsive regulatory protein ) is a major regulatory protein involved in the expression of numerous operons in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7665463" target="_blank">7665463</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_218">
Escherichia coli leucine-responsive regulatory protein Lrp controls lysyl-tRNA synthetase expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8144448" target="_blank">8144448</a>]<br></label>
<input id="rp_similarity_218", type="checkbox">
<div class="RP_similar">
<li>Escherichia coli leucine-responsive regulatory protein ( Lrp ) controls lysyl-tRNA synthetase expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2162465" target="_blank">2162465</a>]<br></li></div>
<li>Lrp ( leucine-responsive regulatory protein ) is a major Esch-erichia coli regulatory protein which regulates expression of a number of operons , some negatively and some positively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8144448" target="_blank">8144448</a>]<br></li>
<li>In E. coli K-12 the leucine-responsive regulatory protein ( Lrp ) affects expression of ~ 400 genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18405378" target="_blank">18405378</a>]<br></li>
<li>Molecular Microbiology ( 1996 ) 20 ( 5 ) ,947 -955 The leucine-responsive regulatory protein ( Lrp ) acts as a pecific repressor en n transcription o s for & dep de t f the Escherichia coli aidB gene Paolo Landini , ' Laurel I. Hajec , ' Lam H. Nguyen ,2 Richard R. Burgess2and Michael R. Volkert ' * ` Department of Molecular Genetics and Microbiology , Universityof MassachusettsMedical School , 55 Lake Avenue North , Worcester , Massachusetts 01655 , USA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8809748" target="_blank">8809748</a>]<br></li>
<li>The leucine-responsive protein ( Lrp ) acts as a speciﬁc repressor for sS-dependent transcription of the Escherichia coli aidB gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11073893" target="_blank">11073893</a>]<br></li>
<li>Landini , P. , Hajec , L.I. , Nguyen , L.H. , Burgess , R.R. and Volkert , M.R. ( 1996 ) The leucine-responsive regulatory protein ( Lrp ) acts as a speciﬁc repressor for sigmaS-dependent transcription of the Escherichia coli aidB gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21398637" target="_blank">21398637</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_225">
The leucine-responsive regulatory protein ( Lrp ) acts as a speciﬁc repressor for sigma s-dependent transcription of the Escherichia coli aidB gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20889740" target="_blank">20889740</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15576800" target="_blank">15576800</a>]<br></label>
<input id="rp_similarity_225", type="checkbox">
<div class="RP_similar">
<li>The leucine-responsive regulatory protein ( Lrp ) acts as a speciﬁc S repressor for - dependent transcription of the Escherichia coli aidB gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15716429" target="_blank">15716429</a>]<br></li>
<li>The leucine-responsive regulatory protein ( Lrp ) acts as a speciﬁc repressor for S-dependent transcription of the Escherichia coli aidB gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15489429" target="_blank">15489429</a>]<br></li>
<li>The leucine-responsive regulatory protein ( Lrp ) acts as a speciﬁc repressor for sS-dependent transcription of the Escherichia coli aidB gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10542189" target="_blank">10542189</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10049384" target="_blank">10049384</a>]<br></li></div>
<li>Role for the leucine-responsive regulatory protein ( Lrp ) as a structural protein in regulating the Escherichia coli gcvTHP o The Escherichiacoli glycine-cleavage enzyme system ( gcvTHP and lpd gene products ) provides C , units for cellular methylation reactions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10217490" target="_blank">10217490</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_230">
Role for the leucine-responsive regulatory protein ( Lrp ) as a structural protein in regulating the Escherichia coli gcvTHP operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12101307" target="_blank">12101307</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10463182" target="_blank">10463182</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11065369" target="_blank">11065369</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11395454" target="_blank">11395454</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18177266" target="_blank">18177266</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11495998" target="_blank">11495998</a>]<br></label>
<input id="rp_similarity_230", type="checkbox">
<div class="RP_similar">
<li>[ 14 ] Stauffer , L.T. and Stauffer , G.V. ( 1999 ) Role for the leucine ¬ responsive regulatory protein ( Lrp ) as a structural protein in regulating the Escherichia coli gcvTHP operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15621456" target="_blank">15621456</a>]<br></li>
<li>Stauffer , L.T. , and Stauffer , G.V. ( 1999 ) Role for the leucine-responsive regulatory protein ( Lrp ) as a structural protein in regulating the Escherichia coli gcvTHP operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10972807" target="_blank">10972807</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16238633" target="_blank">16238633</a>]<br></li></div>
<li>This gene codes for a predicted protein that has a strong similarity to leucine-responsive regulatory protein ( Lrp ) from E. coli and to Lrp homologs in several other bacteria ( 25 , 34 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10850996" target="_blank">10850996</a>]<br></li>
<li>Summary Expression of the Escherichia coli serA gene is activated in vivo by the product of the lrp gene , leucine-responsive regulatory protein ( Lrp ) , an effect partially reversed by L-leucine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11985712" target="_blank">11985712</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_235">
Leucine-regulated self association of leucine-responsive regulatory protein ( Lrp ) from Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19074390" target="_blank">19074390</a>]<br></label>
<input id="rp_similarity_235", type="checkbox">
<div class="RP_similar">
<li>Leucine-regulated selfassociation of leucine-responsive regulatory protein ( Lrp ) from Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2162465" target="_blank">2162465</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12218014" target="_blank">12218014</a>]<br></li></div>
<li>JOURNAL OF BACTERIOLOGY , Feb. 1992 , p. / 92/041109 -10 $ 02.00 / 0 Copyright © 1992 , American Society for Microbiology Characterization of the Regulon Controlled by the Leucine - Responsive Regulatory Protein in Escherichia coli BRIAN R. ERNSTING , ' MARIETTE R. ATKINSON ,2 ALEXANDER J. NINFA ,2 AND ROWENA G. MATTHEWS ' * Biophysics Research Division and Department of Biological Chemistry , The University of Michigan , Ann Arbor , Michigan 48109,1 and Department of Biochemistry , Wayne State University School of Medicine , Detroit , Michigan 482012 Received 7 October 1991/Accepted 10 December 1991 The leucine-responsive regulatory protein ( Lrp ) controls the transcription of a number of genes in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1346534" target="_blank">1346534</a>]<br></li>
<li>ow Is BRIAN R. ERNSTING , `` 2 JOHN W. DENNINGER ,12 ROBERT M. BLUMENTHAL ,3 AND ROWENA G. MATTHEWS 1,2 * Biophysics Research Division1 and Department of Biological Chemistry ,2 University of Michigan , Ann Arbor , Michigan 48109 , and Department of Microbiology , Medical College of Ohio , Toledo , Ohio 436993 Received 26 July 1993/Accepted 2 September 1993 The regulon controlled by the leucine-responsive regulatory protein ( Lrp ) of Escherichia coli consists of over 40 genes and proteins whose expression is regulated , either positively or negatively , by Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7901196" target="_blank">7901196</a>]<br></li>
<li>JOURNAL OF BACTERIOLOGY , Dec. 1996 , p. 6930 -- / 96 / $ 04.0010 Copyright q 1996 , American Society for Microbiology Effects of Nutrition and Growth Rate on Lrp Levels in Escherichia coli JEFFREY R. LANDGRAF , JINGCAI WU , AND JOSEPH M. CALVO * Section of Biochemistry , Molecular and Cell Biology , Cornell University , Ithaca , New York 14853 Received 12 June 1996/Accepted 2 October 1996 Lrp ( leucine-responsive regulatory protein ) activates some Escherichia coli operons that function in anab-olism and represses others involved in catabolism ( for a review , see J. M. Calvo and R. G. Matthews , Microbiol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955316" target="_blank">8955316</a>]<br></li>
<li>175 , No. 4 Copyright © 1993 , American Society for Microbiology Mutations Affecting the Ability of Escherichia coli Lrp To B DNA , Activate Transcription , or Respond to Leucine ind JILL V. PLATKO1 AND JOSEPH M. CALVO2 * Sections of Genetics and Development ' and Biochemistry , Molecular and Cell Biology ,2 Cornell University , Ithaca , New York 14853 Received 2 September 1992/Accepted 13 December 1992 Lrp is a regulatory protein in Escherichia coli that increases expression of some operons and decreases expression of others .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8432705" target="_blank">8432705</a>]<br></li>
<li>The EMBO Journal vol .12 no. 6 pp.2495-2501 , 1993 Lrp - a major regulatory protein in Escherichia coli , bends DNA and can organize the assembly of a higher-order nucleoprotein structure Qing Wang ' and Joseph M.CaIvo2 ,3 Section of IGenetics and Development , 2Biochemistry , Molecular and Cell Biology , Cornell University , Ithaca , NY 14853 , USA 3Corresponding author Communicated by M.Iaccarino Lrp ( Leucine-responsive regulatory protein ) is a global regulatory protein that controls the expression of many operons in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8508774" target="_blank">8508774</a>]<br></li>
<li>176 , No. 7 Regulation of the Escherichia coli lrp Gene QING WANG , ' t JINGCAI WU ,2 DEVORAH FRIEDBERG ,3 JILL PLAKTO ,1 t AND JOSEPH M. CALVO2 * Sections of Genetics and Development ' and Biochemistry , Molecular and Cell Biology ,2 Comell University , Ithaca , New York 14853 , and Department ofApplied Microbiology , The Hebrew University-Hadassah Medical School , Jerusalem , Israel 910103 Received 18 October 1993/Accepted 15 January 1994 Lrp ( leucine-responsive regulatory protein ) is a major Escherichia coli regulatory protein which regulates expression of a number of operons , some negatively and some positively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8144448" target="_blank">8144448</a>]<br></li>
<li>The Leucine-responsive Escherichia coli Regulatory Protein ( Lrp ) from STOICHIOMETRY AND MINIMAL REQUIREMENTS FOR BINDING TO DNA * ( Received for publication , August 28 , 1995 , and in revised form , January 5 , 1996 ) Yuhai Cui ‡ , Michael A. Midkiff , Qing Wang § , and Joseph M. Calvo ¶ From the Section of Biochemistry , Molecular and Cell Biology , Cornell University , Ithaca , New York 14853 Lrp ( Leucine-responsive regulatory protein ) regulates the expression of a number of operons in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8636076" target="_blank">8636076</a>]<br></li>
<li>ilvIH Operon Expression in Escherichia coli Requires Lrp Binding to Two Distinct Regions of DNA Samina Jafri , † Shaolin Chen , and Joseph M. Calvo * Department of Molecular Biology and Genetics , Cornell University , Ithaca , New York 14853 Received 12 April 2002/Accepted 10 June 2002 The leucine-responsive regulatory protein Lrp regulates the expression of a number of operons in Escherichia coli , including the ilvIH operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12218014" target="_blank">12218014</a>]<br></li>
<li>All rights reserved First published online as a Review in Advance on May 1 , 2003 NITROGEN ASSIMILATION GLOBAL AND REGULATION IN ESCHERICHIA COLI Larry Reitzer Department of Molecular and Cell Biology , The University of Texas at Dallas , Richardson , Texas 75080-0688 ; email : reitzer@utdallas.edu Key Words Ntr response , σ 54 , Lrp , ppGpp , metabolic integration ■ Abstract Nitrogen limitation in Escherichia coli controls the expression of about 100 genes of the nitrogen regulated ( Ntr ) response , including the ammonia-assimilating glutamine synthetase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12730324" target="_blank">12730324</a>]<br></li>
<li>Lrp , through its control of glutamate synthase , induces the Ntr response , which stimulates expression of polyamine catabolic enzymes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12730324" target="_blank">12730324</a>]<br></li>
<li>In contrast to Lrp , AsnC has only been shown to exert speciﬁc control of its own gene and that of asnA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16528101" target="_blank">16528101</a>]<br></li>
<li>In addition , the corresponding protein was purified from a strain having a mutation within the gene that encodes Lrp ( Zrp ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2040596" target="_blank">2040596</a>]<br></li>
<li>Iowa , Iowa City , IA 52242 , USA Keywords : glycine cleavage , gcvT : : l a d , GcvA , Lrp , DNA bending INTRODUCTION In Escherichia coli , the biosynthesis of glycine and C , units via the serine hydroxymethyltransferase reaction ( glyA gene product ) plays a central role in cell-physiology .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9802028" target="_blank">9802028</a>]<br></li>
<li>These include dps whose product appears to protect DNA from oxidative damage in the stationary phase ( Bridges and Timms , 1998 ) , stpA , which encodes an H-NS paralogue in E. coli ( Zhang et al. , 1996 ) , and the DNA-binding protein , Lrp , structural gene ( Newman et al. , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11298273" target="_blank">11298273</a>]<br></li>
<li>We used microarray analysis to more globally assess the ability of orthologous Lrp proteins to properly control the E. coli K-12 Lrp regulon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18405378" target="_blank">18405378</a>]<br></li>
<li>A. Western blot analysis of Lrp accumulation ( Eco , E. coli Lrp ; Pmi , P. mirabilis Lrp ; Vch , V. cholerae Lrp ; pCC1 , vector control ) using polyclonal antiserum raised against E. coli Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18405378" target="_blank">18405378</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18405378" target="_blank">18405378</a>]<br></li>
<li>Mol Gen Genet 233:184 -192 BlomfieldI , CalieP , Eberhardt K , McClain M , EisensteinB ( 1993 '' Lrp stimulates phase variation of type 1 fimbriation in cherichia coli K-12 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8152423" target="_blank">8152423</a>]<br></li>
<li>E. coli BL21 ( DE3 ) pLysS harboring plasmid pET16lrp or pET21argP ( Table 1 ) was grown to exponential phase at 30 ° C , and expression of genes encoding N-terminally His-tagged Lrp ( His6-Lrp ) or C-ter-minally His-tagged ArgP ( ArgP-His6 ) was induced with 0.5 mM isopropyl - - D-thiogalactopyranoside ( IPTG ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21441513" target="_blank">21441513</a>]<br></li>
<li>In order to identify the whole set of genes under the direct control of Lrp , we performed Genomic SELEX screening and identiﬁed a total of 314 Lrp-binding sites on the E. coli genome .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2162465" target="_blank">2162465</a>]<br></li>
<li>Of the seven most-global regulators , together affecting expression of about half of all E. coli genes , two were previously shown to exert direct , positive control on gltBDF transcription : Lrp and IHF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17233899" target="_blank">17233899</a>]<br></li>
<li>Just seven proteins ( ArcA , Crp , FIS , Fnr , IHF , Lrp , and NarL ) directly control expression of about half of all genes in E. coli [ 1 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17233899" target="_blank">17233899</a>]<br></li>
<li>This column includes the set of genes showing significant decreases in expression associated with lrp mutation in E. coli , indicating direct or indirect activation by E. coli Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18405378" target="_blank">18405378</a>]<br></li>
<li>Column A thus represents genes that are repressed ( directly or indirectly ) by E. coli Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18405378" target="_blank">18405378</a>]<br></li>
<li>This set includes genes that are repressed ( directly or indirectly ) by E. coli Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18405378" target="_blank">18405378</a>]<br></li>
<li>Here we conﬁrm and extend these results by measuring lrp gene-expression in E. coli cells during different phases of growth in either a minimal or a deﬁned rich medium using transcriptional fusions to lacZ and also by measuring Lrp directly by Western blotting ( immunoblotting ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955316" target="_blank">8955316</a>]<br></li>
<li>To construct the pET-lrpEc vector for overexpression of untagged E. coli Lrp , the open reading frame of the lrp gene ( including the stop codon ) was PCR ampliﬁed with the oligo-nucleotides lrp-NdeI and lrp-BamHI as primers .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19906180" target="_blank">19906180</a>]<br></li>
<li>Lrp has been shown to be involved in the adaptation of E.coli to shifts in environmental conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8890170" target="_blank">8890170</a>]<br></li>
<li>The gene that encodes Lrp , lrp , was also shown to lie in the 20-min region of the E. coli chromosome .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1917830" target="_blank">1917830</a>]<br></li>
<li>Strikingly , while Lrp negatively regulates its own gene in E. coli , and was shown to do so even more strongly in P. mirabilis , Lrp appears to activate its own gene in V. cholerae .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18405378" target="_blank">18405378</a>]<br></li>
<li>Column B includes gene showing direct or indirect activation by E. coli Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18405378" target="_blank">18405378</a>]<br></li>
<li>The combination of Lrp and CRP-cAMP regulation described above likely plays a role in regulating transcription of serA , which encodes the dehydrogenase that carries out the ﬁrst step in serine biosynthesis ( Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9171388" target="_blank">9171388</a>]<br></li>
<li>Lrp appears as an important factor involved in this shift through its opposite effects on transcription by E-ss of osmCp1 ( repressor ) and osmCp2 ( activator ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9663683" target="_blank">9663683</a>]<br></li>
<li>Lrp has a dual role in pap regulation and acts as either an activator or a repressor of transcription , depending on which of ﬁve sites it binds in the pap promoter region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11972787" target="_blank">11972787</a>]<br></li>
<li>Although much remains to be learned about the role of Lrp in the regulation of one-carbon-metabolism , the preliminary evidence suggests that this protein plays a major role in ensuring an adequate flux of carbon from glucose into serine biosynthesis during growth in glucose minimal medium and in ensuring that the catabolic pathway for the conversion of serine to pyruvate is activated during growth in a defined glucose-rich medium ( 163 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li>
<li>One model proposed for gcv regulation is that Lrp plays a structural role , bending the DNA to allow GcvA to function as either an activator or a repressor in response to environmental signals .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10217490" target="_blank">10217490</a>]<br></li>
<li>This suggests a functional role for Lrp during anaerobic growth in an FNR-deficient strain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12754220" target="_blank">12754220</a>]<br></li>
<li>Furthermore , Lrp , via its potent regulation of gltBDF and control of intracellular glutamine concentration , plays a significant role in the regulation of nitrogen assimilation genes ( 30 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17289841" target="_blank">17289841</a>]<br></li>
<li>Second , and partially overlapping the first , Lrp plays an important role in regulating nitrogen metab-olism [ 2,5,7 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17233899" target="_blank">17233899</a>]<br></li>
<li>There is experimental evidence that Lrp functions as a transcriptional regulator for glyA , metK , and lysU .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1917830" target="_blank">1917830</a>]<br></li>
<li>In summary , the available evidence suggests that Lrp is a positive regulator of some operons ( serA and ilvIH ) and a negative regulator of others ( oppABCDF , s d d , tdh , and lysU ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2040596" target="_blank">2040596</a>]<br></li>
<li>In a lrp background , alanine repressed neither avtA nor yfbQ , which implies that Lrp mediates alanine repression ( Fig. 2B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20729367" target="_blank">20729367</a>]<br></li>
<li>In general , Lrp appears to stimulate expression of operons that function in biosynthetic pathways and to repress expression of those that function in catabolic pathways .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8636076" target="_blank">8636076</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_279">
In general , among Lrp-related operons , those which function in biosynthetic pathways are stimulated by Lrp while those involved in catabolic pathways are repressed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955316" target="_blank">8955316</a>]<br></label>
<input id="rp_similarity_279", type="checkbox">
<div class="RP_similar">
<li>In general , Lrp-related operons functioning in biosynthetic pathways are stimulated by Lrp , whereas those involved in catabolic pathways are repressed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17289841" target="_blank">17289841</a>]<br></li></div>
<li>Because the product of the ilvlH gene , AHAS III , is specific for isoleucine biosynthes ; s and thus competes for the threonine pool ( see fig i ) , we wanted to test if H-NS would also regulate the action of the kblitdh operon for which Lrp acts as a repressor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7748928" target="_blank">7748928</a>]<br></li>
<li>Moreover , under certain conditions , Lrp acts as a positive modulator , e.g. when cells grow in glycerol , a non-optimal carbon source .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9512707" target="_blank">9512707</a>]<br></li>
<li>In minimal media , Lrp acts as a repressor of osmY expression throughout the growth cycle ( Lange et al. , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9663683" target="_blank">9663683</a>]<br></li>
<li>Lrp acts as a specific repressor for as-dependenttranscription of aidi3 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8809748" target="_blank">8809748</a>]<br></li>
<li>It is evolutionarily related asnC , a gene that positively to asnA Lrp regulates the operon ( 6 , 12 , 30 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8432705" target="_blank">8432705</a>]<br></li>
<li>Lrp positively regulates YojI , an inner-membrane MccJ25 efﬂux pump .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19074390" target="_blank">19074390</a>]<br></li>
<li>Considering the important role of LysP in regulating the expression of cadA ( 55 ) , which encodes the inducible lysine decarboxylase , we analyzed the effect of Lrp on transcription of the cadBA operon , which encodes the induc-ible lysine decarboxylase CadA and the lysine/cadaverine antiporter CadB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21441513" target="_blank">21441513</a>]<br></li>
<li>Expression of livJ ' - lacZ or livK ' - lacZ operon fusions in strains overexpressing Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1729203" target="_blank">1729203</a>]<br></li>
<li>Lrp decreases b-galactosidase expression from an ompC-lacZ1 operon fusion .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8002608" target="_blank">8002608</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_290">
Lrp decreases expression of b-galactosidase from a plasmid-borne micF-lacZ1 operon ( transcriptional ) fusion .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8002608" target="_blank">8002608</a>]<br></label>
<input id="rp_similarity_290", type="checkbox">
<div class="RP_similar">
<li>Lrp decreases expression of b-galactosidase from an ompC-lacZ1 operon ( transcriptional ) fusion .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8002608" target="_blank">8002608</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8002608" target="_blank">8002608</a>]<br></li></div>
<li>It has been known that Lrp activates the expression of the ilvIH operon , which encodes the AHAS III isoenzyme ( 16 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17463081" target="_blank">17463081</a>]<br></li>
<li>It is thought that Lrp acts differently on these operons in that it represses expression of sdaA , oppA , and tdh and induces expression of ilvIH and serA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1917830" target="_blank">1917830</a>]<br></li>
<li>Like some other important global regulators , for example , cyclic AMP response protein ( 15 ) , Lrp activates expression of some operons and represses expression of others .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8432705" target="_blank">8432705</a>]<br></li>
<li>Like Lrp , Crp activates transcription of some operons and represses transcription of others .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li>
<li>Like some other important global regulators , for example , Crp ( 14 ) , Lrp activates the expression of some operons and represses the expression of others .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1447135" target="_blank">1447135</a>]<br></li>
<li>example Crp ( Mallick and Herrlich , 1979 ) , Lrp activates expression of some operons and represses expression of others .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8483418" target="_blank">8483418</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_298">
Lrp activates expression of some operons and represses expression of others .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7665463" target="_blank">7665463</a>]<br></label>
<input id="rp_similarity_298", type="checkbox">
<div class="RP_similar">
<li>Lrp activates the expression of some operons and represses the expression of others .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8508774" target="_blank">8508774</a>]<br></li></div>
<li>Further evidence that Lrp regulates YojI expression came from experiments in which we transformed the strains MC4100 and RO64 ( MC4100 lrp : : Tn10 ) with plasmid pCLO7 to determine the effect of Lrp depletion on the expression of the plasmid-borne yojI gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19074390" target="_blank">19074390</a>]<br></li>
<li>Additionally , by using the Lrp , which upregulates the expression of the ilvIH gene ( encoding AHAS III ) , further amplification of AHAS was possible without any detrimental effect of plasmid-born overexpression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17463081" target="_blank">17463081</a>]<br></li>
<li>To ﬁnd out whether Lrp affects sigma factor selectivity of promoters in a more global way , we again determined ratios of expression in rpoS and rpoS mutant strains , i.e. , S dependence on genomic microarrays but now for strains that were defective in the lrp gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15716429" target="_blank">15716429</a>]<br></li>
<li>One such example is the expression of virulence gene transcription factor AphA in Vibrio cholerae by Lrp and VpsR which both activate aphA expression ( Lin et al. , 2007 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19906180" target="_blank">19906180</a>]<br></li>
<li>Lin , W. , Kovacikova , G. , and Skorupski , K. ( 2007 ) The quorum-sensing regulator HapR downregulates the expression of the virulence gene transcription factor AphA in Vibrio cholerae by antagonizing Lrp - and VpsR-mediated activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19906180" target="_blank">19906180</a>]<br></li>
<li>Our studies suggest that the positive effect of Lrp on OmpF expression stems from a negative effect of Lrp on the expression of micF , an antisense RNA that inhibits ompF translation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8002608" target="_blank">8002608</a>]<br></li>
<li>These results indicate that ArgP is a major regulator of lysP expression but that Lrp modulates lysP transcription under lysine-limiting conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21441513" target="_blank">21441513</a>]<br></li>
<li>These results suggest that Lrp may be a negative regulator of OmpC expression and a positive regulator of OmpF expression , but indirect effects of Lrp on the expression of these genes are also possible .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8002608" target="_blank">8002608</a>]<br></li>
<li>The complex nature of Lrp action may be related to its physiological role to harmonize the expression of Lrp regulon genes to match with the surrounding conditions , such as the composition and availability of nutrients .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2162465" target="_blank">2162465</a>]<br></li>
<li>Thus , for example , Lrp acts negatively on livJ and livKHMGF expression , and this negative effect requires the presence of leucine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li>
<li>Leucine is known to modulate the effect of Lrp as an activator ( 12 , 27 , 44 ) or repressor ( 11 , 26 -- 28 ) of many genes in the Lrp regulon , and so we examined the effect of leucine on expression of the ompC-lacZ1 operon fusion in lrp1 and lrp backgrounds .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8002608" target="_blank">8002608</a>]<br></li>
<li>The results of our preliminary analysis indicate that Lrp controls an extensive regulon , including many polypeptides whose expression is unaffected by exogenous leucine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1346534" target="_blank">1346534</a>]<br></li>
<li>As mentioned in the introduction , leucine affects the expression of a number of operons in the Lrp regulon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8144448" target="_blank">8144448</a>]<br></li>
<li>The expression of many genes and operons within the Lrp regulon has been found to be relatively insensitive to leucine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9236118" target="_blank">9236118</a>]<br></li>
<li>Lrp regulates the expression of ilvIH and possibly ilvGMEDA and aroP ( see below ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1917830" target="_blank">1917830</a>]<br></li>
<li>These results suggest that Lrp , the product of lrp , negatively regulates livJ and livK expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1729203" target="_blank">1729203</a>]<br></li>
<li>The following experiment shows that Lrp , in fact , regulates expression of the gcv genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1532173" target="_blank">1532173</a>]<br></li>
<li>Lrp acts negatively on expression of genes encoding LIV-I / Fig. 2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16098526" target="_blank">16098526</a>]<br></li>
<li>Lrp has subsequently been shown to regulate the expression of other genes and operons .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1346534" target="_blank">1346534</a>]<br></li>
<li>Lrp regulates the expression of many genes , either positively or negatively ( Calvo and Matthews , 1994 ) , but it is unusual for Lrp to be both an activator and a repressor in the same operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10216857" target="_blank">10216857</a>]<br></li>
<li>Lrp positively regulates genes that function during famine and negatively regulates genes that function during a feast .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li>
<li>We have used this method to con-ﬁrm that Lrp regulates the expression of and leuDBCA livJK and to demonstrate for the ﬁrst time that expression of the malF gene , and other genes of the maltose operon , is regulated by Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7751276" target="_blank">7751276</a>]<br></li>
<li>In summary , Lrp controls Ntr gene-expression , and regulators of the Ntr response modulate the expression of some Lrp-regulated genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12730324" target="_blank">12730324</a>]<br></li>
<li>The effector switch Like Lrp , AsnC binds DNA and regulates gene-expression in the absence or presence of effector .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16528101" target="_blank">16528101</a>]<br></li>
<li>This might involve , for instance , an increase in expression of an alternative reductase gene repressed by Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12706508" target="_blank">12706508</a>]<br></li>
<li>In most cases , Lrp has been reported to activate the operons that As a short-cut approach to identify the whole set of regulation target genes of the RNA polymerase ( RNAP ) sigma subunits and a total of * 300 species of transcription factors , we developed the Genomic SELEX screening Methods system in vitro ( Shimada et al. , 2005 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2162465" target="_blank">2162465</a>]<br></li>
<li>Lrp would repress genes involved in the degradation of exogenous compounds and activate those involved in the biosynthesis of compounds starting from simple intermediates such as pyruvate Because Lrp is apparently made at lower concentration in LB , it is likely to affect transcription of only genes with a higher affinity for it .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1532173" target="_blank">1532173</a>]<br></li>
<li>Thus , we concluded that Lrp participates in transcription regulation of at least eight aminoacyl-tRNA synthetase genes , of which expression of ﬁve aminoacyl-tRNA synthe-tase , including AlaRS , LysRS , SerRS , ThrRS and TyrRS , is repressed by Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2162465" target="_blank">2162465</a>]<br></li>
<li>The effector Leu modulates multimerization of Lrp and thereby controls the transcription of certain target genes ( Chen & Calvo , 2002 ; Chen et al. , 2001a , b ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2162465" target="_blank">2162465</a>]<br></li>
<li>Effect of Lrp on transcription of other genes of the maltose regulon ( malE , malK , lamB , and malP ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7751276" target="_blank">7751276</a>]<br></li>
<li>Thus , Lrp activates transcription from fan genes , as it does for pap , daa , and sfa genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li>
<li>Control at the transcriptional level has been demonstrated for both inﬂux and efflux genes , by several speciﬁc ( ArgR , ArgP ) and more general ( NtrC , Lrp ) repressors and activators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19906180" target="_blank">19906180</a>]<br></li>
<li>In addition , the genes for other cellular functions , such as pili synthesis and adhesion-to-host cells , have been indicated to be under the control of Lrp ( Calvo & Matthews , 1994 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2162465" target="_blank">2162465</a>]<br></li>
<li>In a poor medium that is not optimal for growth but nevertheless still allows growth and cell-division to proceed at a reduced rate , Lrp stimulates the expression of biosynthetic genes ( e.g. , ilvIH [ 36 ] , serA [ 26 ] , gltD [ 13 ] , and leu [ 34 ] ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8253679" target="_blank">8253679</a>]<br></li>
<li>In minimal medium , Lrp might act on those and other genes , repressing degradative genes and activating biosynthesis genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1532173" target="_blank">1532173</a>]<br></li>
<li>Here , we first demonstrate a role for Lrp in the regulation of a gene that is specifically induced in stationary phase , i.e. , under conditions of total starvation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8253679" target="_blank">8253679</a>]<br></li>
<li>Recently , the implication of Lrp in the regulation of the gene encoding the arginine exporter ( argO ) was reported .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21441513" target="_blank">21441513</a>]<br></li>
<li>In cases where multiple promoters with different regulation have been recognized , they are distinguished as P1 , P2 etc. - Fold up-regulation relative to the anaerobic steady state at the indicated times following the introduction of air 5 min 10 min 15 min 60 min 4.8 5.2 4.6 3.1 3.1 3.5 2.1 2.0 4.2 4.2 5.1 4.6 Oxidative stress and iron-sulfur-cluster-assembly genes Relevant regulatory proteins ahpC alkyl hydroperoxide reductase small subunit ahpF alkyl hydroperoxide reductase large subunit dadA D-amino acid dehydrogenase dadX alanine racemase OxyR * ( ) MetJ ( ) OxyR * ( ) MetJ ( ) P1 , CRP * ( / ) Lrp * ( ) P2 , CRP * ( ) Lrp * ( / ) P3 , CRP * ( ) Lrp * ( ) P1 , CRP * ( / ) Lrp * ( ) P2 , CRP * ( ) Lrp * ( / ) P3 , CRP * ( ) Lrp * ( ) IscR * ( ) SoxRS * ( ) OxyR * ( ) OxyR * ( ) IscR * ( ) IscR * ( ) IscR * ( ) IscR * ( ) IscR * ( ) IscR * ( ) IscR * ( ) OxyR * ( ) P1 , H-NS ( ) Lrp * ( ) NhaR ( ) RcsB ( P2 , H-NS ( ) Lrp * ( ) IscR * ( ) OxyR ( ) IHF ( ) Fur SoxR ( ) SoxS ( ) OxyR * ( ) IHF * ( ) Fur ( ) IscR * ( ) OxyR * ( ) IHF * ( ) Fur ( ) IscR * ( ) OxyR * ( ) IHF * ( ) Fur ( ) IscR * ( ) 3.3 2.1 3.4 1.9 3.2 1.3 7.2 1.6 2.2 2.8 6.3 6.1 3.4 3.0 2.1 5.3 2.1 2.8 3.0 9.1 5.8 4.1 fdx ferredoxin fur ferric uptake regulator 2.1 3.4 2.4 2.8 3.3 10.1 7.0 5.1 2.0 1.9 1.8 2.7 1.8 1.8 2.8 2.5 11.2 5.7 4.4 1.8 1.1 3.6 grxA glutaredoxin 1 hscA co-chaperone involved in IscU iron-sulfur-cluster-assembly hscB co-chaperone involved in IscU iron-sulfur-cluster-assembly iscA iron-sulfur-cluster-assembly protein iscR transcriptional regulator iscS cysteine desulfurase iscU scaffold protein involved in iron-sulfur-cluster-assembly iscX possible role in iron-sulfur-cluster-biosynthesis katG bifunctional hydroperoxidase I osmC organic hydroperoxidase Within 5 min of air exposure the abundance of the soxS transcript is increased 8-fold ( Table 2 ) , indicating activation of the sensor SoxR , which enhances soxS expression , which in turn controls the SoxRS regulon ( 39 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16857675" target="_blank">16857675</a>]<br></li>
<li>Our data suggest that additional regulatory proteins , i.e. cAMP-CRP and probably Lrp and H-NS , are crucially involved in generating this speci ® c pattern of csiD expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9512707" target="_blank">9512707</a>]<br></li>
<li>This same functional role can be argued for Lrp regulation ( see introduction ) , perhaps implying that the H-NS protein is involved in fine tuning this regulatory network in some way .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7748928" target="_blank">7748928</a>]<br></li>
<li>The above-described results suggested that Lrp was somehow involved in intrinsic resistance to MccJ25 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19074390" target="_blank">19074390</a>]<br></li>
<li>The apparent discrepancy between the effect of Lrp shown here ( leading to a 1.1 - to 1.3-fold decrease in ompF-lacZ1 expression ) and the effect of Lrp on OmpF expression as analyzed by two-dimensional gel electrophoresis ( an ; 2-fold increase in expression ) led us to investigate other factors involved in regulation of ompF expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8002608" target="_blank">8002608</a>]<br></li>
<li>GcvA-mediated regulation of gcv expression requires two additional proteins , Lrp and GcvR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11495998" target="_blank">11495998</a>]<br></li>
<li>The GcvA ( 31 ) and Lrp ( 10 ) proteins are required for expression of the gcv operon , and the GcvA and PurR proteins ( 30 ) are involved in negative regulation of the gcv operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7928983" target="_blank">7928983</a>]<br></li>
<li>Preliminary data indicate that Lrp is also involved in the control of several other stationary-phase-induced genes besides osmY ( 28a ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8253679" target="_blank">8253679</a>]<br></li>
<li>Also , Lrp , RpoS , and OxyR have been reported previously ( 8 , 21 , 28 ) to be involved in the regulation of some Gsps reported here to be IHF dependent ( Table 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7559363" target="_blank">7559363</a>]<br></li>
<li>Similarly , it has been demonstrated that IHF is involved in stationary-phase regulation of the osmY ( 16 ) and dps ( 1 ) genes and that additional factors , including Lrp , Cap , and RpoS ( osmY [ 16 ] and dps [ 1 ] ) participate in their regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7559363" target="_blank">7559363</a>]<br></li>
<li>Whereas rs , Lrp , CRP , and IHF are clearly involved in stationary-phase induction , none of these regulators is essential for osmotic regulation of osmY .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8253679" target="_blank">8253679</a>]<br></li>
<li>Both Lrp and ArgR are also involved in transcriptional regulation of the gltBDF operon for glutamate synthase ( GOGAT ) , which plays an important role in the synthesis of glutamate , a precursor of arginine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19906180" target="_blank">19906180</a>]<br></li>
<li>Lrp was also found to be involved in the regulation of cadBA expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21441513" target="_blank">21441513</a>]<br></li>
<li>In addition to the results reported here for E. coli , a recent report suggests that Lrp is also involved in the regulation of dad expression in Salmonella typhimurium ( 11 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955407" target="_blank">8955407</a>]<br></li>
<li>Lrp is clearly involved in maintaining this balance .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1532173" target="_blank">1532173</a>]<br></li>
<li>We show that H-NS is involved in the transcriptional regulation of the kbl/tdh operon , which is also Lrp regulated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7748928" target="_blank">7748928</a>]<br></li>
<li>Lrp is involved in the regulation of osmY .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8253679" target="_blank">8253679</a>]<br></li>
<li>We then addressed the question whether Lrp is also involved in osmotic regulation of osmY .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8253679" target="_blank">8253679</a>]<br></li>
<li>Analysis of Lrp distribution between cytoplasm and nucleoid using minicells suggests that both leucine level and growth rate are involved in regulating free Lrp concentrations within cells ( S.C. , Z. Hao , E. Bieniek & J.M.C. , unpublished results ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11575919" target="_blank">11575919</a>]<br></li>
<li>For cells overproducing Lrp , growth in the presence of leucine gave an unexpected transport phenotype : leucine increased the rate of leucine-transport ( Fig. 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1729203" target="_blank">1729203</a>]<br></li>
<li>Another possibility is that , in cells containing high levels of Lrp , leucine prevents degradation of liv mRNAs or of proteins involved in transport , thus leading to the accumulation of transport capacity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1729203" target="_blank">1729203</a>]<br></li>
<li>Mutations in genes affected by Lrp and L-leucine were transduced by antibiotic resistance into strain MEWi , and the lIp double mutants were created by transduction as described above .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1532173" target="_blank">1532173</a>]<br></li>
<li>Inﬂuence of l-leucine and l-alanine on Lrp regulation of foo , coding for F1651 , a Pap homologue .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2162465" target="_blank">2162465</a>]<br></li>
<li>In addition to these amino acid-related transcription factors , Lrp was found to regulate a total of 15 transcription factors ( Fig. 6 , grey symbols ) , which are involved in the stress-response and life-style selection of E. coli , such as GadW for acid response , YedW for copper and peroxide response , EnvR for response-to-drugs , QseB for quorum-sensing , BluR for bioﬁlm formation , and MatA and SlyA for planktonic growth .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2162465" target="_blank">2162465</a>]<br></li>
<li>In a nitrogen-limited environment , ppGpp via Lrp is required for glutamate-synthesis , induction of the Ntr response , and ammonia assimilation , and therefore the synthesis of all the amino acids .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12730324" target="_blank">12730324</a>]<br></li>
<li>Lrp functions to regulate metabolic pathways in E. coli in response to the availability of amino acids and nitrogen bases in the external environment .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li>
<li>Our results linking H-NS to Lrp lend further support to the notion that both regulators are involved in responses to stress conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8152423" target="_blank">8152423</a>]<br></li>
<li>The induction was independent of the RelA/SpoT , RpoH , KatF , OmpR , AppY , Lrp , PhoB and H-NS proteins during stress conditions that are known to induce or activate these global regulators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1453957" target="_blank">1453957</a>]<br></li>
<li>The antagonism of the Lrp and CRP-cAMP global regulators also provides a mechanism for reduction of serC ( pdxF ) - aroA transcription and SerC ( PdxF ) transaminase and AroA synthase amounts in cells growing in amino acid-rich media , which provide serine and decrease the need for B6 vitamers PLP and pyridoxamine 59-phosphate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9171388" target="_blank">9171388</a>]<br></li>
<li>No effect of amino acid downshift was observed on the synthesis of lrp mRNA , encoding the positive regulator ( Lrp ) of ilvIH transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7557316" target="_blank">7557316</a>]<br></li>
<li>These results demonstrate that autogenous regulation of lrp involves Lrp acting directly to cause repression of urp transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8144448" target="_blank">8144448</a>]<br></li>
<li>They also participate in diverse roles such as regulation-of-DNA-replication and transcription .51 These proteins include H-NS , HU , Lrp , and IHF which are distributed uniformly throughout the nucleoid of E. coli and are termed group ÿI proteins .52 We discuss here nucleoid proteins that in ¯ uence micF expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11601842" target="_blank">11601842</a>]<br></li>
<li>These authors also examined the expression of stpA in response to H-NS , Lrp , and temperature and the requirement for promoter-distal DNA for regulation , and they obtained results in good agreement with our own .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9006049" target="_blank">9006049</a>]<br></li>
<li>Taken together , our data suggest that the abundant Lrp protein not only acts as a global repressor ( or acti-S 70 vator in some cases ) but also affects E - E selectivity at many stationary-phase-induced promoters , probably by modulating local DNA topology .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15716429" target="_blank">15716429</a>]<br></li>
<li>Additional regulators acting on some of these genes include transcription factors speciﬁc for certain stress response , such as OxyR , and also some more global regulators , such as H-NS or Lrp ( Hengge-Aronis , 1996b and references herein ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9663683" target="_blank">9663683</a>]<br></li>
<li>Additionally , these genes are regulated by transcription factors speciﬁc for certain stress responses ( e.g. , OxyR , which is involved in the oxidative stress response ) or more-global regulators , such as H-NS , IHF , cyclic AMP ( cAMP ) - cAMP receptor protein ( CRP ) , and Lrp ( 104 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16760308" target="_blank">16760308</a>]<br></li>
<li>Transcription of the mal genes is under the control of several factors , including a maltose oper-on-speciﬁc activator protein ( MalT ) , two global regulators ( Crp and now Lrp ) , and perhaps also malK .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7751276" target="_blank">7751276</a>]<br></li>
<li>Indeed , the more stress-responsive genes are studied , the more it appears that their regulation involves a network of global regulators , such as H-NS or Lrp , but also CRP , IHF or Fis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9663683" target="_blank">9663683</a>]<br></li>
<li>We investigated the possible role of other regulators in pga expression , and analysis of the pga promoter region using the BPROM online promoter analysis tool , revealed possible binding sites for the transcription factors Lrp , Crp , and ArgR , which are global regulators in E. coli ( Paul et al. , 2007 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18445167" target="_blank">18445167</a>]<br></li>
<li>Network of transcription factors ( TFs ) involving Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2162465" target="_blank">2162465</a>]<br></li>
<li>Thus , the life style of E. coli under these transcription factors is also under the control of Lrp , which monitors the nutritional conditions in the environment .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2162465" target="_blank">2162465</a>]<br></li>
<li>Lrp is also induced under starvation conditions and plays an enhancing role in transcription of some stress-response genes ( 41 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10368152" target="_blank">10368152</a>]<br></li>
<li>Here , we also identiﬁed another expanded role of Lrp in regulation of a set of transcription factors , each playing a regulatory role in the control of a speciﬁc metabolism pathway or physiological response to a speciﬁc nutritional condition .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2162465" target="_blank">2162465</a>]<br></li>
<li>On the other hand , according to data in the literature concerning the dcrA/sdaC gene [ 22 ] and our preliminary observations for dcrB , both genes are involved in regulation by global regulators such as Lrp and Crp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12558182" target="_blank">12558182</a>]<br></li>
<li>Lrp has a global inﬂuence on transcription regulation and is also involved in microbial virulence [ 8 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21350637" target="_blank">21350637</a>]<br></li>
<li>We also have shown that the global regulator Lrp is involved in repression of this gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16098526" target="_blank">16098526</a>]<br></li>
<li>Lrp , a global regulator involved in regulation of ... .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11065369" target="_blank">11065369</a>]<br></li>
<li>Lrp as a major regulator of C1 metabolism .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1532173" target="_blank">1532173</a>]<br></li>
<li>Among the down-regulated genes , the lrp and ygaZH genes , which encode a global regulator Lrp and L-valine exporter , respectively , were overexpressed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17463081" target="_blank">17463081</a>]<br></li>
<li>S dependence of many genes is altered in the absence of the global regulator Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15716429" target="_blank">15716429</a>]<br></li>
<li>Another module within the S network comprises genes that are also under the control of the global regulator Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15716429" target="_blank">15716429</a>]<br></li>
<li>First , the lrp gene , which encodes a global regulator Lrp , was selected as a target ; this was because Lrp positively regulates the expression of genes involved in L-valine-biosynthesis , such as ilvIH .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17463081" target="_blank">17463081</a>]<br></li>
<li>Finally , Lrp is a global regulator which inﬂuences expression of genes that are mostly involved in adaptation of the cellular-metabolism to changing nutritional environments [ 6 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16209909" target="_blank">16209909</a>]<br></li>
<li>In some of the speculation regarding the role of Lrp as a global regulator , an assumption was made that lacZ reporter gene activity reﬂected Lrp levels in cells .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955316" target="_blank">8955316</a>]<br></li>
<li>Lrp is a global regulator of various operons ( 25 , 34 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10850996" target="_blank">10850996</a>]<br></li>
<li>A mixed double negative feedback loop between the sRNA MicF and the global regulator Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23564175" target="_blank">23564175</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25246476" target="_blank">25246476</a>]<br></li>
<li>Another global regulator is the Lrp protein , which controls gene-expression upon transition from rich to depleted medium and upon the reverse transition [ 27 , 44 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17025177" target="_blank">17025177</a>]<br></li>
<li>Negative control of fae ( K88 ) expression by the `` global '' regulator Lrp is modulated by the local regulator FaeA and affected by DNA-methylation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9512707" target="_blank">9512707</a>]<br></li>
<li>On the other hand , leuE expression is suppressed by the Lrp global regulator ( Fig. 4b ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17025177" target="_blank">17025177</a>]<br></li>
<li>The Lrp protein ( 18.8 kDa ) is a global transcriptional regulator of several E. coligenes and operons ; Lrp can either activate or inhibit transcription by binding its target promoters , and leucine often plays a role in the modulation of Lrp activity ( Calvo and Matthews , 1994 ; Newman eta / .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8809748" target="_blank">8809748</a>]<br></li>
<li>Lrp is considered to be a global regulator in E. coli as it controls a regulon of more than 40 genes ( for reviews , see Calvo and Matthews , 1994 ; Newman and Lin , 1995 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9515701" target="_blank">9515701</a>]<br></li>
<li>It is well known that exogenous leucine affects the expression of some genes regulated by Lrp , a global regulator of metabolism in E. coli ( 4 , 22 , 23 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19074390" target="_blank">19074390</a>]<br></li>
<li>For many operons regulated by Lrp , leucine also affects expression , and it has been suggested that Lrp and leucine together constitute a global response mechanism in E.coli ( Ernsting et al. , 1992 ; Newman et al. , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8508774" target="_blank">8508774</a>]<br></li>
<li>We also show that leucine , its natural analogue L-a-amino-n-butyric acid and a number of other amino acids induce yeaS expression , and the global regulator Lrp mediates this induction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16098526" target="_blank">16098526</a>]<br></li>
<li>As Lrp is a global transcriptional regulator that controls the expression of numerous genes in response to the availability of amino acids and nitrogen bases ( 6 , 9 , 33 ) , the role of this protein in lysP expression was analyzed in more detail .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21441513" target="_blank">21441513</a>]<br></li>
<li>In contrast to most other family members , which are speciﬁc regulators , E. coli Lrp is a global regulator that affects the expression of numerous genes and operons , many of which belong to amino-acid-metabolic pathways ( Newman and Lin , 1995 ; Hung et al. , 2002 ; Tani et al. , 2002 ; Cho et al. , 2008 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19906180" target="_blank">19906180</a>]<br></li>
<li>Results Lrp stimulates argO expression in a leucine-sensitive manner The fact that Lrp affects the expression of many genes involved in the metabolism and transport of amino acids , including the artPIQM operon involved in arginine uptake ( Hung et al. , 2002 ; Tani et al. , 2002 ; Cho et al. , 2008 ) , and our interest in establishing a coherent global picture of the regulation of arginine-metabolism in E. coli ( Caldara et al. , 2006 ; 2008 ) , urged us to analyse argO promoter ( P ) argO activity in isogenic lrp + and lrp : : Tn10 strains .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19906180" target="_blank">19906180</a>]<br></li>
<li>Lrp is a bifunctional transcription factor that activates or represses many genes , including genes involved in leucine-biosynthesis and transport , as well as many that are not directly involved in leucine-metabolism ( leucine-Lrp regulon ) ; ( for review , see Newman et al. 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7603430" target="_blank">7603430</a>]<br></li>
<li>Last , we tested leucine , because Lrp is known to activate the expression of serA and other genes involved in one-carbon-metabolism ( 4 , 35 , 52 ) , and leucine addition reduces serA transcription ( 35 , 52 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9171388" target="_blank">9171388</a>]<br></li>
<li>The global regulator leucine responsive protein ( Lrp ) is involved in the regulation of L-valine-biosynthesis ; it activates the expression of the ilvIH genes encoding the AHAS III isoenzyme ( 16 , 17 ) and represses the expression of the ilvGMEDA operon encoding the AHAS II isoenzyme ( 18 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17463081" target="_blank">17463081</a>]<br></li>
<li>The expression of most of the genes encoding these proteins is not induced by the substrates they transport but is controlled by the global regulation systems , such as the Lrp protein , and activated by the signal compounds involved in the intracellular communication .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17025177" target="_blank">17025177</a>]<br></li>
<li>Furthermore , such operons have been shown to be regulated by the global regulator Lrp , which is involved in the regulation of at least 30 different proteins in E.coli ( Ernsting et al. , 1992 ; reviewed in Newman et al. , 1992 ; D'Ari et al. , 1993 ; Calvo and Matthews , 1994 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8890170" target="_blank">8890170</a>]<br></li>
<li>Lrp is a global regulator that has been implicated in gene regulation during carbon source downshifts ( for recent reviews , see Calvo & Matthews , 1994 ; Newman et al. , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9512707" target="_blank">9512707</a>]<br></li>
<li>Both modes of regulation by GcvA require at least two additional protein factors : ( i ) GcvR is a negative-acting protein that is hypothesized to interact with GcvA , antagonizing activation of gcvTHP by GcvA and promoting repression ( Ghrist and Stauffer , 1995a ) ; ( ii ) Lrp is a global regulatory protein affecting the control of many genes , including gcvTHP ( Lin et al. , 1992 ; Calvo and Matthews , 1994 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10972807" target="_blank">10972807</a>]<br></li>
<li>Lrp , a global transcriptional regulator of genes involved in amino-acid-metabolism ( Lin et al. , 1992 ; Calvo & Matthews , 1994 ) , is required for both activation and repression of gcv expression ( Stauffer & Stauffer , 1994 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11495998" target="_blank">11495998</a>]<br></li>
<li>Lrp , a global regulatory protein of numerous genes relating to amino-acid-metabolism ( Calvo & Matthews , 1994 ) , is required for normal induction of the gcu operon ( Lin et al. , 1992 ; Abbreviations : GM , glucose minimal ; RNAP , RNA polymerase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9611815" target="_blank">9611815</a>]<br></li>
<li>Lrp , a global regulator of numerous genes relating to amino-acid-metabolism ( Calvo & Matthews , 1994 ) , is required for induction of the gcv operon ( Lin et al. , 1992 ; Stauffer & Stauffer , 1994 ) , but is insensitive to the co-regulator leucine ( Ghrist & Stauffer , 1995b ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9802028" target="_blank">9802028</a>]<br></li>
<li>Interestingly , many other transporters for amino acids such as the uptake genes for arginine ( artPIQM ) , serine ( sdaC ) , alanine ( cycA ) and proline ( proY ) , and the export gene for leucine ( leuE ) are also regulated reciprocally by Lrp and leucine ( Kutukova et al. , 2005 ; Cho et al. , 2008 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19906180" target="_blank">19906180</a>]<br></li>
<li>This view derived primarily from the fact that Lrp tends to activate some operons involved in biosynthesis and repress others involved in catabolism .1 Another view advanced by D'Ari et al. 13 is that Lrp is primarily a DNA-organizing protein , much like HNS , IHF , and HU , which also affect gene-expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11575919" target="_blank">11575919</a>]<br></li>
<li>Subsequently , mutations in the lrp gene were found to inﬂuence the expression of operons involved in the biosynthesis and degradation of some more amino acids ( Lin et al. , 1992 ; Platko et al. , 1993 ) , suggesting that Lrp plays a regulatory role in transport and metabolism of not only Leu , but also some other amino acids .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2162465" target="_blank">2162465</a>]<br></li>
<li>Outer membrane porins OmpC and OmpF , glutamine synthetase ( GlnA ) , the small subunit of glutamate synthase ( GltD ) , lysyl-tRNA synthetase form II ( LysU ) , a high-affinity periplasmic binding protein specific for branched-chain amino acids ( LivJ ) , W protein , and the enzymes of the pathway converting threonine to glycine , namely , threonine dehydrogenase ( Tdh ) and 2-amino-3-ketobutyrate coenzyme A ligase ( Kbl ) , were identified as members of the Lrp regulon by electrophoretic analysis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1346534" target="_blank">1346534</a>]<br></li>
<li>Since Lrp controls synthesis of several enzymes in related areas of metabolism ( L-serine deaminase , phosphoglycerate dehydrogenase , and L-threonine dehydrogenase ) , we thought that enzymes of glycine-metabolism might also be affected by Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1532173" target="_blank">1532173</a>]<br></li>
<li>Lrp also controls the metabolism of alanine , serine , and glycine , which are feedback inhibitors of GS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12730324" target="_blank">12730324</a>]<br></li>
<li>Members of the Lrp regulon include genes and operons involved in amino acid biosynthesis ( gltBDF ( Ernsting et al. , 1992 ) , glnALG ( Ernsting et al. , 1992 ) , glyA ( D'Ari et al. , 1993 ) , ilvIH ( Platko et al. , 1990 ) , ilvGMEDA ( Rhee et al. , 1996 ) , leuABCD ( Lin et al. , 1992a ) , serA ( Lin et al. , 1990 ; Rex et al. , 1991 ) ) ; amino acid degradation ( gcv ( Lin et al. , 1992a ) , sdaA ( Lin et al. , 1990 ) , kbl ( Lin et al. , 1990 ; Rex et al. , 1991 ) , tdh ( Lin et al. , 1990 ; Rex et al. , 1991 ) ) ; nutrient transport ( livKHMGF ( Haney et al. , 1992 ) , livJ ( Haney et al. , 1992 ) , micF ( Ferrario et al. , 1995 ) , ompC ( Ernsting et al. , 1992 ) , ompF ( Ernsting et al. , 1992 ) , oppABCDF ( Austin et al. , 1989 ; Hiles et al. , 1987 ) ) ; and pilus-formation ( daa ( Bilge et al. , 1993 ; van der Woude et al. , 1992 ) , fae ( Huisman et al. , 1994 ) , fan ( Braaten et al. , 1992 ) , pap ( Braaten et al. , 1992 ) , ® m ( Blom ® eld et al. , 1993 ) , sfa ( van der Woude & Low , 1994 ) ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9236118" target="_blank">9236118</a>]<br></li>
<li>Both authors have proposed that Lrp is involved in the adaptation of E. coIi to major shifts in environmental conditions ; Lrp decreases synthesis from transport and degra-dative genes needed primarily in rich media and activates synthesis from biosynthetic genes needed in poor media , in response to `` feast and famine '' in the environment .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7603430" target="_blank">7603430</a>]<br></li>
<li>Thus , Lrp positively regulates operons involved in transport of maltose and maltodextrins but not those involved in maltose deg-radation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7751276" target="_blank">7751276</a>]<br></li>
<li>Transport Six operons involved in nutrient transport are known to be regulated by Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li>
<li>This suggests that Lrp target genes code for products involved in a much larger array of metabolic reactions than was previously thought .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1532173" target="_blank">1532173</a>]<br></li>
<li>For example , Lrp regulates the serA and sdaA genes involved in the production of pyruvate and the tdh and kbl genes , which affect the in vivo levels of L-threonine and a-ketobutyrate ( 5 , 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8900118" target="_blank">8900118</a>]<br></li>
<li>After Genomic SELEX screening , however , we ident-iﬁed that Lrp not only controls the production of amino acids , but also the utilization pathway of amino acids by regulating the genes for tRNAs , ami-noacyl-tRNA synthetases and rRNAs .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2162465" target="_blank">2162465</a>]<br></li>
<li>Conceivably , the regulation of pntAB by Lrp reflects a special role that pyridine nucleotide transhydrogenase plays in supplying NADPH for amino-acid-uptake and metabolism .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li>
<li>Regulation of branched-chain-amino-acid-transport by Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1729203" target="_blank">1729203</a>]<br></li>
<li>A possible physiological role for Lrp in the regulation of branched-chain amino-acid-metabolism is discussed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8900118" target="_blank">8900118</a>]<br></li>
<li>It remains to be established whether bkdR has evolved to have a more specialized role limited to branched-chain amino acid catabolism or whether , like Lrp , it plays some more general role in coordinating metabolism .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li>
<li>Lrp of Corynebacterium glutamicum controls expression of the brnFE operon encoding the export system for L-methionine and branched-chain amino acids .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25845847" target="_blank">25845847</a>]<br></li>
<li>Lrp has some amino acid sequence similarity to AsnC , a protein that regulates asnA expression ( Kolling , R. , and Lother , H. ( 1985 ) J. Bacteriol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2040596" target="_blank">2040596</a>]<br></li>
<li>Therefore , since Lrp is involved in the regulation of other operons involved in branched-chain amino-acid-metabolism , it might be expected that it would modulate the expression of this central operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8900118" target="_blank">8900118</a>]<br></li>
<li>Lrp regulates expression of genes whose products are involved in amino acid biosynthesis , amino-acid-transport , amino acid degradation , and amino acyl-tRNA-synthesis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1532173" target="_blank">1532173</a>]<br></li>
<li>Subsequently , leucine regulates the expression of numerous genes involved in amino-acid-metabolism through Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17289841" target="_blank">17289841</a>]<br></li>
<li>Lrp is also known to repress the expression of the livJ and livK genes , encoding the branched-chain-amino-acid-transporter ( 19 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17463081" target="_blank">17463081</a>]<br></li>
<li>Also , Lrp regulates the expression of the leucine biosynthetic operon , leuABCD ( 7 ) , and the expression of the branched-chain-amino-acid-transport genes livJ and livKHMG ( 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8900118" target="_blank">8900118</a>]<br></li>
<li>The ilvIH operon , one of the best-studied members of the Lrp regulon , encodes an acetohydroxy acid synthase that is involved in the biosynthesis of branched-chain amino acids ( 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12218014" target="_blank">12218014</a>]<br></li>
<li>OPERONS REGULATED BY Lrp Amino Acid Biosynthesis Five operons involved in amino acid biosynthesis are regulated by Lrp : the glnALG operon , which includes the coding sequence for glutamine synthetase ; the gltBDF operon , which encodes glutamate synthase ; the ilvIH operon , which encodes acetohydroxyacid synthase III ( an isozyme involved in branched-chain amino acid biosynthesis ) ; the leuABCD operon , which encodes enzymes that catalyze the biosynthesis of leucine ; and serA , which encodes phosphoglycerate dehydrogenase ( the enzyme catalyzing the first step in serine biosynthesis ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li>
<li>Interestingly , many of the other operons regulated by Lrp are involved in either the production or the dissimilation of the carbon substrates , pyruvate and a-ketobutyrate , of the AHAS isozymes required for branched-chain amino acid biosynthesis ( Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8900118" target="_blank">8900118</a>]<br></li>
<li>Two operons involved in high-affinity transport of branched-chain amino acids , the livJ and livKHMGF operons , are negatively regulated by Lrp , and that negative regulation is potentiated by leucine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li>
<li>For the proteins involved in high-affinity branched-chain-amino-acid-transport , regulation by Lrp is negative and is stimulated in the presence of leucine ( 43 , 62 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li>
<li>Since the insert in malF was positively regulated by Lrp , we studied the regulation of transcription from inserts in several genes involved in maltose-metabolism , aided by gifts of strains from several investigators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7751276" target="_blank">7751276</a>]<br></li>
<li>It will be interesting to learn whether the expression of other genes involved in serine biosynthesis or one-carbon-metabolism are regulated by Lrp activation and CRP-cAMP repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9171388" target="_blank">9171388</a>]<br></li>
<li>Further , in addition to inhibiting ompF expression , MicF negatively regulates the transcription factor Lrp , which in turn is responsible for the regulation of genes involved in a number of functions that we also identiﬁed in our microarray data , including amino-acid-metabolism and transport ( 78 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23564175" target="_blank">23564175</a>]<br></li>
<li>For instance , the highly accumulated aromatic-amino-acids are all under the control of a single transcription factor , TyrR Lrp ( Tables 3 and S1 ) and thus the expression of a number of genes involved in the metabolism of amino acids should be indirectly regulated in the absence of Lrp , leading to inﬂuence in the intracellular pool of respective amino acids .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2162465" target="_blank">2162465</a>]<br></li>
<li>A certain proportion of the regulated genes are involved , as originally proposed , in transport and metabolism of amino acids , but Lrp has also been suggested to regulate genes involved in biosynthesis and degradation of various metabolites other than amino acids ( Brinkman et al. , 2003 ; Calvo & Matthews , 1994 ; Newman & Lin , 1995 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2162465" target="_blank">2162465</a>]<br></li>
<li>A number of genes that are negatively regulated by Lrp are involved in the transport and catabolism of amino acids and peptides ( the proteins of the oligopeptide permease operon , Tdh , Kbl , and SdaA ) and in the adaptation of the cell to media of high osmolarity and high temperature ( OmpC and LysU ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1346534" target="_blank">1346534</a>]<br></li>
<li>Most of the genes regulated by Lrp are involved in small-molecule transport and amino-acid-metabolism ( 9 , 53 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21441513" target="_blank">21441513</a>]<br></li>
<li>Several genes required for amino acid and peptide-transport and catabolism are negatively regulated by Lrp , and other genes required for amino acid biosynthesis and ammonia assimilation in a nitrogen-poor environment are positively regulated by Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1346534" target="_blank">1346534</a>]<br></li>
<li>Genes and proteins that are involved in catabolism of amino acids and oligopeptide-transport are negatively regulated by Lrp , while genes and proteins involved in amino acid biosynthesis and the assimilation of ammonia are positively regulated by Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8002608" target="_blank">8002608</a>]<br></li>
<li>Among the regulatory targets of Lrp , there are many transcription factors that participate in the regulation-of-amino-acid-metabolism and molecule transport ( 9 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21441513" target="_blank">21441513</a>]<br></li>
<li>As a result , the regulation target of Lrp was predicted to expand from the hitherto identiﬁed genes for amino-acid-metabolism to a set of novel target genes for utilization of amino acids for protein-synthesis , including tRNAs , aminoacyl-tRNA synthases and rRNAs .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2162465" target="_blank">2162465</a>]<br></li>
<li>The whole set of regulation targets herein identiﬁed indicates that Lrp senses the presence of nutritional conditions and regulates not only the transport and metabolism ( synthesis and degradation ) of amino acids , but also the utilization amino acids up to protein-synthesis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2162465" target="_blank">2162465</a>]<br></li>
<li>The interaction of Lrp and H-NS in the regulation of another amino-acid-biosynthetic pathway , strongly encourages the view of a coupling between H-NS fimction and amino acid biosynthesis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7748928" target="_blank">7748928</a>]<br></li>
<li>The effects identified so far have largely been involved in amino-acid-metabolism , but this bias is probably partly a result of the interests of the persons investigating Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1532173" target="_blank">1532173</a>]<br></li>
<li>The results herein described indicate a novel role of Lrp in the regulation of a large group of genes involved in not only the transport and metabolism of amino acids , but also the polymerization of amino acids into proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2162465" target="_blank">2162465</a>]<br></li>
<li>SELEX-chip screening of Lrp-binding sequences : Lrp regulon genes involved in transport and metabolism of amino acids The Lrp regulon genes involved in transport and metabolism of amino acids , tRNA , tRNA-charging and rRNA are listed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2162465" target="_blank">2162465</a>]<br></li>
<li>The Lrp regulon comprises several genes involved in amino acid synthesis and degradation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21441513" target="_blank">21441513</a>]<br></li>
<li>Members of the family influence cellular-metabolism in both a global ( Lrp ) and specific ( AsnC ) manner , often in response to exogenous amino acid effectors .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16528101" target="_blank">16528101</a>]<br></li>
<li>Lrp seems to be a central regulator for the induction of genes required after nutritional downshift ( e.g. , amino-acid-biosynthetic genes ) as well as for the repression of genes no longer required under these conditions ( e.g. , for amino-acid-uptake ) ( 10 , 34 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8253679" target="_blank">8253679</a>]<br></li>
<li>Lrp , a global regulator for many genes involved in amino-acid-metabolism [ 13 ] , binds to multiple sites upstream of the PgcvT promoter , from bp 92 to 229 ( Fig. 1 ( a ) ) and plays a structural role in gcvTHP regulation , bending the DNA to allow GcvA to function as either an activator or repressor [ 14 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15621456" target="_blank">15621456</a>]<br></li>
<li>Full induction of gcv in response to glycine also requires Lrp ( 12 ) , a global regulator necessary for either activation or repression of many genes involved in amino-acid-metabolism ( 1 , * Corresponding author .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7665470" target="_blank">7665470</a>]<br></li>
<li>Taken together , we propose that Lrp is a global regulator of transcription of a large number of the genes involved in not only amino-acid-transport and metab-olism , but also amino acid utilization .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2162465" target="_blank">2162465</a>]<br></li>
<li>Other members of the Lrp regulon , including ompF , ompC , and gltD , were identiﬁed by a two-dimensional gel electrophoresis analysis ( 13 ) , and yet others were identiﬁed by a placMu transposition analysis ( 20 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7665463" target="_blank">7665463</a>]<br></li>
<li>Several DNA-binding proteins , including H-NS ( Suzuki et al. , 1996 ) , HU ( Painbeni et al. , 1997 ) , Lrp ( Ferrario et al. , 1995 ) , Rob ( Ariza et al. , 1995 ) , SoxS ( Chou et al. , 1993 ) and MarA ( Cohen et al. , 1988 ; Jair et al. , 1995 ) , are known to regulate micF promoter activity and , consequently , modulate ompF mRNA translation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11029695" target="_blank">11029695</a>]<br></li>
<li>Other anonymous operons controlled by Lrp were identified by two-dimensional protein gel electrophoresis and by screening random transcriptional fusions generated by placMu transposition ( Ernsting et al. , 1992 ; Lin et al. , 1992b ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8508774" target="_blank">8508774</a>]<br></li>
<li>In fact , control by Lrp often involves H-NS protein , e.g ompF and ompC ( Graeme-Cook et al. 1989 ; Ernsting et al. 1992 ) , fimB ( Kawula and Orndorff 1991 ; Blomfield et al. 1993 ) , papB ( van der Woude et al. 1992 ) , ilvIH ( Levinthal et al. 1994 ) , and lysU ( Ito et al. 1994 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7603430" target="_blank">7603430</a>]<br></li>
<li>This hypothesis is consistent with findings regarding H-NS control of the transcription of several other operons , which are also under Lrp control : ompFand ompC ( Graeme-Cook et al. 1989 ; Ernsting et al. 1992 ) , fimB ( Kawula and Orndorff 1991 ; Blomfield et al. 1993 ) , and papB ( van der Woude et al. 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8152423" target="_blank">8152423</a>]<br></li>
				</div></ul><br>
			<label class="collapse_SIT" for="_5"><font size="5px">Lrp binding site</font></label>
				<input id="_5" type="checkbox">
				<div class="SIT"><br><ul><li>KGdIaILaDs GIRnGlDVvr miALGAdtVl LGRaFLYALA Vv kaA qGrIPVFlDg GVRRGTDVfK ALALGAaGVf iGRPvvFsLA ... VLAqsVA KtgkPVLIDs GfRRGsDIvK ALALGAeaVl LGRatLYgLA Vv kA sGdtPVLfDs GIRtGaDVvK ALAMGAsaVg iGRPYaWgaA tmPILeqRnl KdkleVFVDg GVRRGTDV1K ALcLGAkGVg LGePFLYAns VmPILkeRgl dqkIdIFVDg GVRRGTDV1K ALcLGAkGVg LGRPFLYAMs V-PILA-RVA KG-IPV-VD - GVRRGTDV-K ALALGA-GV - LGRPFLYALA LCTD 338 GOX 317 MDH 335 LOX 353 B2-S 441 B2-H 500 Consensus taGqaGVanl LnLiekEmkv aMtLtGakSI sEiTqDsLvq glgKelpaAl AeGEaGVkKV LQMMrDEfEL TMALsGCRS1 KEisrshiaa DWdgpssrAV ArGEtGVDeV LtLLkaDIDr TLAqiGCpdI tsLsPDyLqn EgVtNtApvd lgGskGIEhV arsLlaEaDL iMAvdGYRnl KELTiDaLrp tR ... .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8407843" target="_blank">8407843</a>]<br></li>
<li>ctivation from a Distance : Roles of Lrp and Integration A Host Factor in Transcriptional Activation of gltBDF LIGI PAUL ,1 ROBERT M. BLUMENTHAL ,2 AND ROWENA G. MATTHEWS1 ,3 * Biophysics Research Division1 and Department of Biological Chemistry ,3 University of Michigan , Ann Arbor , Michigan 48109 , and Department of Microbiology and Immunology , Medical College of Ohio , Toledo , Ohio 436142 Received 23 January 2001/Accepted 13 April 2001 The leucine-responsive regulatory protein ( Lrp ) binds to three sites centered 252 , 216 , and 152 bp upstream of the transcription start site of the Escherichia coli glutamate synthase operon ( gltBDF ) and activates transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11395454" target="_blank">11395454</a>]<br></li>
<li>Wang , Q. , and Calvo , J.M. ( 1993 ) Lrp , a global regulatory protein of E. coli binds cooperatively to multiple sites and activates transcription of ilvIH .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9663683" target="_blank">9663683</a>]<br></li>
				<label class="SIT_collapse_similarity" for="sit_similarity_4">
Wang , Q. and Calvo , J.M , ( 1993 ) Lrp , a global regulatory protein of Escherichia coii , binds co-operatively to multiple sites and activates transcription of iiviH .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8483418" target="_blank">8483418</a>]<br></label>
<input id="sit_similarity_4", type="checkbox">
<div class="SIT_similar">
<li>Wang , Q. , and Calvo , J.M. ( 1993 ) Lrp , a global regulatory protein of Escherichia coli , binds co-operatively to multiple sites and activates transcription of ilvIH .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10216857" target="_blank">10216857</a>]<br></li>
<li>Wang , Q. , and Calvo , J.M. ( 1993a ) Lrp , a global regulatory protein of Escherichia coli , binds co-operatively to multiple sites and activates transcription of ilvIH .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11985712" target="_blank">11985712</a>]<br></li>
<li>Wang , Q. , and Calvo , J.M. ( 1993b ) Lrp , a global regulatory protein of Escherichia coli , binds co-operatively to multiple sites and activates transcription of ilvIH .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9515701" target="_blank">9515701</a>]<br></li></div>
				<label class="SIT_collapse_similarity" for="sit_similarity_8">
Lrp , a global regulat-ory protein of E. coli , binds cooperatively to multiple sites and activates transcription of ilvIH .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9236118" target="_blank">9236118</a>]<br></label>
<input id="sit_similarity_8", type="checkbox">
<div class="SIT_similar">
<li>Lrp , a global regulatory protein of Esch-erichia coli , binds cooperatively to multiple sites and activates transcription of ilvIH .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7592387" target="_blank">7592387</a>]<br></li>
<li>Lrp , a global regulatory protein of E. coli , binds cooperatively to multiple sites and activates transcription of ilvIH .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12218014" target="_blank">12218014</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7901196" target="_blank">7901196</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11575919" target="_blank">11575919</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8144448" target="_blank">8144448</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955316" target="_blank">8955316</a>]<br></li></div>
				<label class="SIT_collapse_similarity" for="sit_similarity_11">
Lrp , a global regulatory protein of Escherichia coli , binds co-operatively to multiple sites and activates transcription of ilvZH .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9611815" target="_blank">9611815</a>]<br></label>
<input id="sit_similarity_11", type="checkbox">
<div class="SIT_similar">
<li>Lrp , a global regulatory protein of Escherichia coli , binds co-operatively to multipl ; sites and activates transcription of ilvZH .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9802028" target="_blank">9802028</a>]<br></li></div>
				<label class="SIT_collapse_similarity" for="sit_similarity_13">
Lrp , a global regulat-ory protein of Escherichia coli , binds co-operatively to multiple sites and activates transcription of ilvIH .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9512707" target="_blank">9512707</a>]<br></label>
<input id="sit_similarity_13", type="checkbox">
<div class="SIT_similar">
<li>Lrp , a global regulatory protein of Escherichia col , binds co-operatively to multiple sites and activates transcription of ilvIH .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7928983" target="_blank">7928983</a>]<br></li>
<li>Lrp , a global regulatory protein of Esch-erichia coli , binds co-operatively to multiple sites and activates transcription of ilvIH .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955407" target="_blank">8955407</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8002608" target="_blank">8002608</a>]<br></li>
<li>Lrp , a global regulatory protein of Escbericbia coli , binds co-operatively to multiple sites and activates transcription of ilvIH .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7881552" target="_blank">7881552</a>]<br></li>
<li>Lrp , a global regulatory protein of Escherichia coli , binds co-operatively to multiple sites and activates transcription of ilvIH .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7916011" target="_blank">7916011</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8253679" target="_blank">8253679</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8071194" target="_blank">8071194</a>]<br></li></div>
<li>Sequence similarities detected in the 5 ' upstream areas of genes that belong to the leucine regulon Sequence Location Score ' Gene 72.17 77.08 79.17 73.67 74.33 66.08 69.42 68.08 65.33 70.17 70.17 66.67 -61 to-72 -242 to-253 tdh T T T A T C C T G A A T T T T A T T C T G C A Ta T T T A T T C T G A A Ta T T T A T T A T C A A T T T T A T T C T T A T Ta T C T A T T C G A C A T T T T A T T C A T T A Cb T T T A T T A G T G A T T T T A T T C T G G T Gb T T T A T T C T A C G T T T T A T T A A C A A Tb T C T G T T C T T A A T ilvIH -211 to-222 -140 to-129 -66 to-77 Over the initiating ATG 95 bp upstream of the putative -35 hexamer 15 bp upstream of the putative -35 hexamer 15 bp upstream of the putative -35 hexamer 10 bp downstream of the initiating ATG -50 to-61 +1 to +12 sdaA lysU oppA oppBDCF livJ livKHMGF Consensus T T T A T T C t N a A T a Within the regions recognized by the Lrp protein ( 37 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1917830" target="_blank">1917830</a>]<br></li>
<li>Modified DNA fragments ( 1 10210 ) were incubated with a minimally , 3 M saturating amount of Lrp ( 2.6 3 1028 M ) and resolved by electrophoresis through a 5 % nondenaturing acrylamide gel at 25 ° C to separate Lrp bound from unbound fragments .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8900118" target="_blank">8900118</a>]<br></li>
<li>Plots of the relative band intensities in digitized images of autoradiograms of hydroxyl radical protection patterns obtained in the absence ( thin line ) and presence ( thick line ) of a minimally saturating amount of purified Lrp ( 2.6 3 1028 M ) bound to the nontranscribed ( ilv bp 110 -- 278 ( A ) ) or transcribed ( ilv bp 6 -- 278 ( B ) ) DNA strand of the ilvGMEDA leader-regulatory region are shown .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8900118" target="_blank">8900118</a>]<br></li>
				<label class="SIT_collapse_similarity" for="sit_similarity_21">
Autoradiogram of DMS protection patterns obtained in the absence and presence of a minimally saturating amount of purified Lrp ( 2.6 3 1028 M ) bound to the nontranscribed ( ilv bp 110 -- 278 ( A ) ) or transcribed ( ilv bp 6 -- 278 ( B ) ) DNA strand of the ilvGMEDA leader-regulatory region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8900118" target="_blank">8900118</a>]<br></label>
<input id="sit_similarity_21", type="checkbox">
<div class="SIT_similar">
<li>Autoradiogram of DMS interference patterns obtained in the absence and presence of a minimally saturating amount of purified Lrp ( 2.6 3 1028 M ) bound to the nontranscribed ( ilv bp 1110 -- 1278 ) DNA strand of the ilvGMEDA leader-regulatory region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8900118" target="_blank">8900118</a>]<br></li></div>
<li>A T C A K A C A G ~ ~ T A n ; T C A T A C ~ ~ A T G T T G C R A C G C A A A C G ~ C C C T A ~ C A T A A A C C G T A ~ A T C G C T4A48T G A ~ ATCGCCATGATTRACTATATTGATPrrTTCAGTTC ~ CCTACATATATTGCGCGCCCCGGMG ~ GTCAGATGTCGmCG55 ~ 3T8G Lrp binding site G G C A A A T A T T G C C C T T A A A T T C T C ~ A C m T G R m A C A G G T A ~ 62G8 Crp binding site AGATCRAGATCACTGATAGATACATAACTTGTGTGTATC ~ CCGCCCTCAAATTATTACGGCGGTAAATGATTAAGCCATCGCCGATAG 718 M E ACAGAmCATPr?TACGGTCAGGCACCTTCCCGGGCTGRACTGGCT ~ GCTGAATTA ~ GCATTCCTCCA ~ ~ TAGATGG80A8A T T Q T S T I A S K D S R S A W R K T D T M W M L G L Y G T ACGACTCAAACCAGCACGATGCGTCGAAAGACTCTCGTAGTGCCTGGCGCAAGACAGACACCATGTGGATGCTGGGCC ~ ACGGCACG 898 A I G A G V L F L P I N A G V G G W I P L I I M A I L A F P GCAATCGGCGCGGGCGTGCTGTTCCTGCCMTCAACGCCGGTGTGGCGGTATGATCCCGCTGATCATCATGGCTATCCTTGCGTTCCCG 988 M T F F A A R G L T R F V L S G K N P G E D I T E V V E E H ATGACG ~ GCTCACCGCGGCCTGACTCGCTTCGTACTGTCTGGT ~ CCCGGGCGAAGACATCACCGAGGTTGTAG ~ GAACAC1078 F G I G A G K L I T L L Y F F A I Y P I L L V Y S V A I T N mGGTATTGGCGCAGGTAAACTGA ~ ACCCTGCTCTACZTCZTCGCTATCTACCCGATCCTGCTGGTTTATAGCGTGGCAATCACCMT 1168 T V E S F M S H Q L G M T P P P R A I L S L I L I V G M M T ACCGTTGAAAGCTTCATGTCTCACCAGCTGGGTATGACGCCACCGCCGCGTGCGATTCTGTCGCTGATCCTGATCGTGGGTATGATGACC 1258 I V R F G E Q M I V K A M S I L V T P F V G V L M L L A L Y ATCGTTCGCTTCGGTGAGCAGATGATCGTTAAAGCGATGAGTATTCTGGTATTCCCG ~ GTTGGCGTACTGATGCTGCTGGCTCTGTAC 1348 L I P Q W N G A A L E T L S L D T A S A T G N G L W M T L W CTGATCCCGCAGTGGAACGGCGCTGCACTGGAAACGCTGTCTCTGGACACTGCATCTGCAACCGGAAACGGTCTGTGGATGACCCTGTGG 1438 L A I P V M V F S F N A S P I I S S F A V A K R E E Y G D M CTGGCAATTCCGGTAATGGTGTTCTCGTTC ~ CCACTCTCCGATCATCTCTTC ~ CGCCGTTGCGAAGCGTGAAGAGTACGGCGATATG 1528 A E Q K C S K I L A F A A I M M V L T V M F F V F S C V L S GCAGAACAGRAATGCTCGAAGATCCTGGCAZTCGCACACATCATGATGGTGCTGACCGTAATG ~ C ~ CGTCTTCAGCTGTGTACTGAGC 1618 l .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8026499" target="_blank">8026499</a>]<br></li>
<li>Plasmids or strains Relevant characteristic ( s ) Reference or source Plasmids pBdad1 ClaI ± Bst EII dad promoter fragment from ÿ296 to 126 cloned into the SmaI site of pBluescript-II KS ± , the insert orientation is EcoRI ± ( ClaI ± Bst EII dad fragment ) ± BamHI Low-copy-number plasmid EcoRI ± BamHI dad promoter fragment from pBdad1 subcloned into pJEL170 pJ490 with Lrp binding site 1 mutation ( mut # 29 ) pJ490 with Lrp binding site 2 mutation ( mut # 67 ) pJ490 with Lrp binding site 3 mutation ( mut # 116 ) pJ490 with Lrp binding site 4 mutation ( mut # 129 ) pJ490 with Lrp binding site 5 mutation ( mut # 163 ) pJ490 with Lrp binding site 5 mutation ( mut # 170 ) pJ490 with Lrp binding site 6 mutation ( mut # 197 ) pJ490 with Lrp binding site 7 mutation ( mut # 231 ) pJ490 with Lrp binding site 8 mutation , and the ÿ10 region mutation of promoter 1 ( mut # 272 ) pJ490 with Lrp binding site 9 mutation ( mut # 308 ) pJ490 with Lrp binding site 10 mutation ( mut # 357 ) pJ490 with the sequence above ÿ95 deleted ( mut # 205 ) Plasmid used for in vitro and in vivo transcription assay EcoRI ± BamHI dad promoter fragment from pBdad1 subcloned into pLSZ13 pELM490 with Lrp binding site 5 mutation ( mut # 163 ) pELM490 with Lrp binding site 6 mutation ( mut # 197 ) pELM490 with Lrp binding site 8 mutation , and the ÿ10 region mutation of promoter 1 ( mut # 272 ) pELM490 with Lrp binding site 9 mutation ( mut # 308 ) Mathew et al. ( 1996 ) pJEL170 pJ490 pJ29 pJ67 pJ116 pJ129 pJ163 pJ170 pJ197 pJ231 pJ272 pJ308 pJ357 pJ205 pLSZ13 pELM490 pELM163 pELM197 pELM272 Sun and Fuchs ( 1994 ) Mathew et al. ( 1996 ) This study This study This study This study This study This study This study This study This study This study This study This study Sun and Fuchs ( 1994 ) Mathew et al. ( 1996 ) This study This study This study pELM308 Mathew et al. ( 1996 ) Strains CV1008 TG1 ZF1000 ara thi D ( lac-pro ) ilvIH : : Mu dI 1734 lrp-35 : : Tn10 Platko et al. ( 1990 ) Miller ( 1992 ) This study supE hsdD5 thi D ( lac ± proAB ) F8 [ traD36 proAB lacI lacZDM15 ] q TG1 lrp : : Tn10 Primer extension Site-directed mutagenesis Mutagenesis was performed using the polymerase chain reaction ( PCR ) as described by Hemsley et al. ( 1989 ) using two back-to-back primers .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10216857" target="_blank">10216857</a>]<br></li>
<li>To our knowledge , this is the fourth time that methylation protection has been demonstrated for a bacterial regulatory system in vitro , with the interactions of Lrp with the papA-papB intergenic region , OxyR with ag43 and GutR with its operator sequence being the ﬁrst three ( van der Woude et al. , 1998 ; Correnti et al. , 2002 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17238917" target="_blank">17238917</a>]<br></li>
<li>Lrp bound with high af ® nity and in a highly cooperative manner .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9512707" target="_blank">9512707</a>]<br></li>
<li>We suggested that single palindromic motifs , but not single half motifs , bound Lrp strongly enough so that complexes could be observed in gel retardation experiments .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8483418" target="_blank">8483418</a>]<br></li>
<li>Incubation of PCR10 with Lrp protected 77 bp from ( OP ) 2Cu2 + - activated cleavage ( for a summary , see Fig. 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7916011" target="_blank">7916011</a>]<br></li>
<li>As the Lrp dimer concentration was increased from 0.22 to 27.6 nM , Lrp bound to and protected sequences spanning about a 137-bp region from position -92 to position -229 ( Fig. 4 , bars to the right of the gels ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7928983" target="_blank">7928983</a>]<br></li>
<li>Red shaded nucleotide stretches represent zones of the top strand that are protected against DNase I cleavage upon Lrp binding , short vertical red lines and lines with a dot represent nucleotides that remain accessible and become hypersensitive to DNase I upon Lrp binding respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19906180" target="_blank">19906180</a>]<br></li>
<li>SFeigquernec1es of selected Lrp proteins Sequences of selected Lrp proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18405378" target="_blank">18405378</a>]<br></li>
<li>Numbered brackets indicate sequences protected from DNase I digestion by Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8002608" target="_blank">8002608</a>]<br></li>
<li>Sequences enclosed by solid boxes were protected from DNase I cleavage in the presence of Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8002608" target="_blank">8002608</a>]<br></li>
<li>IHF bound to the phage ; II1A site Downloaded from www.microbiologyresearch.org by IP : 132.248.34.24 On : Fri , 05 Aug 2016 22:42:4 Lrp-binding analysis Role of Lrp in gcvT : : lacZexpression in the IHF-C , IHF-D and IHF-G mutants We used a gel mobility-shift assay to determine whether Lrp could still bind to the gcv control region carrying the Since all but the upstream 22 bp of the Lrp-binding insertion sequences , and this binding could be responregion was replaced in the IHF lysogens , we predicted sible for the negative effect of Lrp on g c v T : : l a c Z that Lrp would no longer be involved in regulation of regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10217490" target="_blank">10217490</a>]<br></li>
<li>These data showed that the Rho-dependent terminator within ﬁmS is located in the region of Lrp site III and consists of the boxA-like element and the C-rich , G-poor sequences located immediately 3 of it ( Fig. 1B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16321930" target="_blank">16321930</a>]<br></li>
<li>Also , regions with similarities to the Lrp ( 44 ) and IHF ( 15 ) consensus sites can be found ( Fig. 7 ) , and the location of all these sequences is consistent with the results of a recent deletion analysis of the osmY control region performed by Villarejo and coworkers ( 43a ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8253679" target="_blank">8253679</a>]<br></li>
<li>There is a potential Lrp site ( YAGHAWATTWTDCTR ; Y ¯ T or C ; H ¯ A , C or T ; W ¯ A or T ; D ¯ G , A or T ; R ¯ A or G ; Cui et al. , 1995 ) centred at ® 120 bp just inside the yeiK coding region , and there are two sequences matching the IHF-site consensus ( WATCAANNNNTTR ; Freundlich et al. , 1992 ) , which are consistent with the observed effects of Lrp and IHF on yeiL gene expression ( Fig. 7c ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11101674" target="_blank">11101674</a>]<br></li>
<li>Another possible Arr candidate is the amino-acid-responsive regulator Lrp ( Leucine-responsive regulatory protein ) ; however , the putative Arr-binding site sequences do not closely resemble the Lrp consensus , c/tAG-Aa/tATTa / tT-CTa/g ( Cuiet al. , 1995 ) and Lrp has a subunit molecular mass of 18-8kDa , which is much less than the 35 kDa polypeptide originally designated for Arr ( Green & Guest , 1994 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9308170" target="_blank">9308170</a>]<br></li>
<li>Lrp Binding as a Function of DNA Length : Experiments with Oligonucleotides of Defined Length -- A set of double-stranded DNA molecules were prepared having lengths 15 , 17 , 19 , 21 , 23 , and 25 bp with the consensus occupying the center of molecules ( Fig. 6A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8636076" target="_blank">8636076</a>]<br></li>
<li>We chose 17 bp as the length of the double-stranded consensus region because such a minimal sequence did not bind Lrp ( Fig. 6 ) , and therefore we could expect to see an effect of single-stranded flanking DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8636076" target="_blank">8636076</a>]<br></li>
<li>To identify DNA sequences that are recognized by Lrp , we used a variation of a protocol developed by Tuerk and Gold ( 39 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7665463" target="_blank">7665463</a>]<br></li>
<li>In our studies , we used a 20-bp degenerate sequence to search for DNA sequences that were recognized by Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7665463" target="_blank">7665463</a>]<br></li>
<li>However , the possibility must be entertained that Lrp recognizes not a consensus but some other feature of DNA such as bending or a pattern of repeating A 's and T 's .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li>
<li>A Consensus Sequence for Binding of Lrp to DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9236118" target="_blank">9236118</a>]<br></li>
<li>A sequence ladder ; lane 2 , G 5 A sequence ladder ; lane 3 , unbound modified DNA fragments ; lane 4 , Lrp bound modified DNA fragments .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8900118" target="_blank">8900118</a>]<br></li>
<li>HTH motifs consist of a short a-helix Interaction of Lrp with DNA Consensus sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li>
<li>No Lrp consensus target sequence was present upstream of serA , suggesting that Lrp acts indirectly on the serA promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1917830" target="_blank">1917830</a>]<br></li>
<li>There is no obvious Lrp consensus sequence in the 457-bp promoter fragment .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1917830" target="_blank">1917830</a>]<br></li>
<li>When the Lrp consensus table ( Table 3 ) was used to scan the serA promoter sequence , the highest score for any 12-bp sequence was 57.58 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1917830" target="_blank">1917830</a>]<br></li>
<li>One possibility consistent with these results is that Lrp recognizes one consensus sequence and that Lrp with bound leucine recognizes another .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7665463" target="_blank">7665463</a>]<br></li>
<li>Since the palindromic Lrp consensus sequence is degenerate at several positions , it can be asymmetrical .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8900118" target="_blank">8900118</a>]<br></li>
<li>Identification of Lrp consensus target .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1917830" target="_blank">1917830</a>]<br></li>
<li>A potential Lrp target consensus sequence was found near the LivR-regulated promoters of livJ and livKH-MGF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1917830" target="_blank">1917830</a>]<br></li>
<li>Site 1 ( approximate base pair position , 220 -- 250 ) , which contains the 15-bp Lrp consensus-like binding sequence located between base pair positions 226 and 240 , appears at the lowest Lrp concentration followed by the appearance of site 2 ( approximate base pair position 185 -- 215 ) and then site 3 ( approximate base pair position 155 -- 180 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8900118" target="_blank">8900118</a>]<br></li>
<li>A preliminary consensus sequence was derived from a comparison of 12 sites to which Lrp was shown to bind in MPE footprinting experiments ( 11 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8636076" target="_blank">8636076</a>]<br></li>
<li>Comparison of a total of 12 sites from the two organisms yielded a consensus sequence , AGAATTTTATTCT , and a synthetic DNA molecule containing this sequence exhibited a relatively high afﬁnity for Lrp ( 40 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7665463" target="_blank">7665463</a>]<br></li>
<li>It is likely that in addition to the sequence of binding sites , both the number of binding sites and their organization determine the binding af ® nity of hexadecamer and leucine-bound octamer and these parameters are likely different for each member of the Lrp regulon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11575919" target="_blank">11575919</a>]<br></li>
<li>Lrp Binds Cooperatively to Multiple Sites in the ilvGMEDA Leader Region -- Since no secondary Lrp binding sites with similarities to the consensus-like DNA sequence could be identified on the DNA fragment used in the gel mobility shift experiment described above , the presence of multiple Lrp-DNA complexes observed in Fig. 3 might , alternatively , be explained by protein aggregation at high Lrp concentrations .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8900118" target="_blank">8900118</a>]<br></li>
<li>The results of DNase I footprinting experiments ( Fig. 6 ) showed that these complexes are formed by the binding of Lrp to its primary , consensus-like , site followed by the sequential and directional binding to two adjacent secondary sites that possess no sequence similarity to the Lrp consensus site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8900118" target="_blank">8900118</a>]<br></li>
<li>These results confirm that Lrp binds in a site-specific manner to the consensus-like DNA sequence and suggest that Lrp binding to this site ( complex 1 ) is required for subsequent , cooperative binding to secondary sites which give rise to complexes 2 and 3 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8900118" target="_blank">8900118</a>]<br></li>
<li>The further observations that Lrp does not bind with high affinity to an ilv DNA fragment containing a mutationally altered or deleted primary binding site suggest that the binding of Lrp to these secondary sites requires cooperative interactions associated with the binding of an initial Lrp protomer to the primary consensus-like sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8900118" target="_blank">8900118</a>]<br></li>
<li>A systematic search of the DNA sequence of the ilvGMEDA operon from base pair position 2460 to 1278 did not reveal any other Lrp consensus-like DNA binding sites with predicted values greater than 3.0 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8900118" target="_blank">8900118</a>]<br></li>
<li>p1 p2 Consensus for binding sites of the Lrp protein is indicated above the osmC sequence , and nucleotides matching this consensus are underlined .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9663683" target="_blank">9663683</a>]<br></li>
<li>For this reason , it seemed important to gain further information on a consensus sequence for Lrp binding but by using binding sites that were unrelated to those in the ilvIH operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7665463" target="_blank">7665463</a>]<br></li>
<li>An alignment of 12 Lrp binding sites within ilvIH DNA from two different organisms revealed a tentative consensus sequence AGAAT TTTATTCT ( Q. Wang , M. Sacco , E. Ricca , C. T. Lago , M. DeFelice , and J. M. Calvo , Mol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7665463" target="_blank">7665463</a>]<br></li>
<li>From the results of the quantitative analysis shown in Fig. 7A , the consensus may be formulated as YAGHAWATTWT DCTR , where Y 5 C or T , H 5 not G , W 5 A or T , D 5 not C , and R 5 A or G. Previously , by aligning six Lrp binding sites from E. coli ilvIH DNA and six sites from S. typhimurium ilvIH DNA , we derived a tentative consensus sequence for Lrp binding : AgaATTTTATtcT ( 42 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7665463" target="_blank">7665463</a>]<br></li>
<li>Shultzaberger , R.K. , and Schneider , T.D. ( 1999 ) Using sequence logos and information analysis of Lrp binding sites to investigate discrepancies between natural selection and SELEX .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19906180" target="_blank">19906180</a>]<br></li>
<li>Brackets denote Lrp binding sites 1 , 2 and 3 based on best ﬁt to the consensus .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9515701" target="_blank">9515701</a>]<br></li>
<li>Based on an analysis of the contribution of each of the consensus base pairs to the energetics of binding ( 7 ) , it was predicted that mutant sites 1 and 2 would not be recognized by Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12218014" target="_blank">12218014</a>]<br></li>
<li>A consensus for Lrp binding was formulated by a computer analysis of sequences associated with the Lrp regulon ( 23 ) and by a comparison of naturally occurring Lrp binding sites ( 11 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8636076" target="_blank">8636076</a>]<br></li>
<li>Comparison of the sequences of the three Lrpprotected sites upstream of gltBDF with consensus sequences for Lrp binding to other genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9236118" target="_blank">9236118</a>]<br></li>
<li>A weaker match ( 59-CTTGAAATTATTCTG-39 ) is centered at 2107 with respect to the transcriptional start , and given the degeneracy of the Lrp consensus and the known contribution of ﬂanking DNA to Lrp binding ( 5 ) , it is likely that other binding sites are also present within the intervening sequences .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9006049" target="_blank">9006049</a>]<br></li>
<li>The proposed Lrp-binding sites within the fim switch have the same separation as ilvIH sites 1 and 2 ( 18 bp ) , although the orientations of the asymmetric sequences within the consensus differ between fim and ilvIH GU Lrp has been shown to bend DNA , organizing it into a nucleoprotein complex ( 22 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7916011" target="_blank">7916011</a>]<br></li>
<li>Therefore , although Lrp initiates binding to the core region ( sites 1 and 2 ) , continued Lrp cooperativity permits further Lrp-DNA interactions to occur to the sides flanking these sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7916011" target="_blank">7916011</a>]<br></li>
<li>How useful is the matrix in Fig. 7A for predicting Lrp binding sites within DNA of any given sequence ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7665463" target="_blank">7665463</a>]<br></li>
<li>Here , we identify 11 Lrp binding sites located within 350 bp of dad DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10216857" target="_blank">10216857</a>]<br></li>
<li>Lrp-induced bending within a DNA fragment containing six Lrp binding sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8508774" target="_blank">8508774</a>]<br></li>
<li>The fact that this same sequence bound Lrp tightly when embedded within a 50-mer sequence ( 12 ) suggested that strong binding required , in addition to a 15-mer recognition sequence , DNA that flanked that recognition sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8636076" target="_blank">8636076</a>]<br></li>
<li>Five of the Lrp sites in dad ( sites 3 , 5 , 6 , 8 and 9 ) have a 73 % or better match to a proposed Lrp DNA-binding consensus sequence initially derived from the Lrp binding sites in ilvIH ( Wang and Calvo , 1993 ) and later modi ® ed by examining Lrp binding to randomly synthesized DNA fragments ( Cui et al. , 1995 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10216857" target="_blank">10216857</a>]<br></li>
<li>However , a unique consensus sequence for Lrp-binding sites has never been clearly identified , and researchershave entertainedthe possibility that DNA features other than nucleotide sequence , such as bending and A/T content , might be important for Lrp binding ( Calvo and Matthews , 1994 ; Cui et 1995 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8809748" target="_blank">8809748</a>]<br></li>
<li>Lrp sites 3 , 5 , 6 , 8 and 9 have a 73 % or better match to the Lrp DNA-binding consensus YAGHAWATTWTDCTR , where Y C / T , H not G , W A / T , D not C and R A / G ( Cui et al. , 1995 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10216857" target="_blank">10216857</a>]<br></li>
<li>The DNA binding domain recognizes the consensus sequence ` AGAATTTTATTCT ' ( Cui et al. , 1995 ) , but Lrp is believed to bind cooperatively to DNA with high aYnity via multiple suboptimal binding sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16879983" target="_blank">16879983</a>]<br></li>
<li>Wang and Calvo ( 1993 ) found that the sequence AGAATTTTATTCT is a consensus for the Lrp binding site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9663683" target="_blank">9663683</a>]<br></li>
<li>A recent study of DNA sequences recognized by Lrp established the consensus as a 15-bp sequence , YAGHA-WATTWTDCTR ( Y 5 C/T , H 5 `` not G , '' W 5 A/T , D 5 `` not C , '' R 5 A/G ) ( Cui , Y. , Wang , Q. , Stormo , G. D. , and Calvo , J. M. ( 1995 ) J. Bacteriol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8636076" target="_blank">8636076</a>]<br></li>
				<label class="SIT_collapse_similarity" for="sit_similarity_85">
[ 18 ] Cui , Y. , Wang , Q. , Stormo , G.D. and Calvo , J.M. ( 1995 ) A consensus sequence for binding of Lrp to DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16098526" target="_blank">16098526</a>]<br></label>
<input id="sit_similarity_85", type="checkbox">
<div class="SIT_similar">
<li>Cui , Y. , Wang , Q. , Stormo , G.D. , and Calvo , J.M. ( 1995 ) A consensus sequence for binding of Lrp to DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16238633" target="_blank">16238633</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9515701" target="_blank">9515701</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10216857" target="_blank">10216857</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8809748" target="_blank">8809748</a>]<br></li>
<li>Cui , Y. , Wang , Q. , Stormo , J.D. , and Calvo , J.M. ( 1995 ) A consensus sequence for binding of Lrp to DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19906180" target="_blank">19906180</a>]<br></li></div>
<li>Lrp binds tightly to double-stranded DNA having a sequence related to the consensus , but not to that same DNA after it has been denatured ( 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8636076" target="_blank">8636076</a>]<br></li>
<li>At Lrp concentrations as high as 1 3 1027 M , no detectable Lrp-DNA complexes were observed with a DNA fragment ( ilv base pair positions 6 -- 188 ) that does not contain the consensuslike Lrp binding sequence or with a DNA fragment ( ilv base pair positions 6 -- 278 ) that contains point mutations in the consensus-like sequence ( Fig. 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8900118" target="_blank">8900118</a>]<br></li>
<li>These bases were chosen for alteration because they were protected by Lrp in footprinting experiments ( Fig. 3 ) and they are centered in a group of 13 of 15 nucleotides ( Fig. 4 ) that are identical to a consensus sequence that has been suggested for Lrp binding to DNA ( 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955407" target="_blank">8955407</a>]<br></li>
<li>Lrp Binding as a Function of DNA Length-Primer Extension Analysis -- An analysis of both natural and synthetic DNAs established the following as the consensus sequence for Lrp binding : YAGHAWATTWTDCTR , where Y 5 C/T , H 5 `` not G , '' W 5 A/T , D5 `` not C , '' R 5 A/G ( 12 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8636076" target="_blank">8636076</a>]<br></li>
<li>A consensus sequence for binding Lrp has been proposed ( 7 ) , but it is not always readily discernible in some known members of the regulon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12218014" target="_blank">12218014</a>]<br></li>
<li>To further define the nature of fhe consensus sequence for Lrp binding , we chose to invesfigafe the binding of Lrp fo ilvll-l DNA from a closely related relafive of E. coli , Saimoneila typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8483418" target="_blank">8483418</a>]<br></li>
<li>Another possibility is that single-stranded DNA interacts with Lrp at a second site and that this binding increases the strength with which the consensus sequence interacts with its binding site on Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8636076" target="_blank">8636076</a>]<br></li>
<li>However , as demonstrated here , single-stranded DNA can stimulate Lrp binding when it flanks a double-stranded consensus sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8636076" target="_blank">8636076</a>]<br></li>
<li>A consensus sequence for binding of Lrp to DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9308170" target="_blank">9308170</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18083817" target="_blank">18083817</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955407" target="_blank">8955407</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21441513" target="_blank">21441513</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12218014" target="_blank">12218014</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16879983" target="_blank">16879983</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11101674" target="_blank">11101674</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11575919" target="_blank">11575919</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9813115" target="_blank">9813115</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9006049" target="_blank">9006049</a>]<br></li>
<li>The competitor DNAs were single and double-stranded forms of a 44 mer having a sequence unrelated to the Lrp binding consensus .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8636076" target="_blank">8636076</a>]<br></li>
<li>We demonstrate that a dou-ble-stranded 15 mer having a sequence that closely matches the consensus does not bind Lrp and that flanking DNA sequences are required for strong binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8636076" target="_blank">8636076</a>]<br></li>
<li>A double-stranded 15 mer having a sequence that closely matches the consensus does not show measurable binding to Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8636076" target="_blank">8636076</a>]<br></li>
<li>We synthesized a double-stranded 15 mer having a sequence that closely matched the consensus and observed no measurable binding of this DNA to Lrp ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8636076" target="_blank">8636076</a>]<br></li>
<li>The Lrp protein has a degenerate AT-rich consensus binding sequence which has recently been reﬁned ( 59-YAGHAWATTWTDCTR-39 [ 4 ] ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9006049" target="_blank">9006049</a>]<br></li>
<li>If this model is accurate , the Lrp-responsive promoter sequences may contain a similar consensus sequence for Lrp binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1917830" target="_blank">1917830</a>]<br></li>
<li>The nucleotide sequence encompassed by PCR4 contains two sequences that resemble a consensus for Lrp binding ( 23 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7916011" target="_blank">7916011</a>]<br></li>
<li>the choice of parameters should become part of the and will be determined automaticalIy based on the number and of base pairs in the da ta seL In addition to bp consensus for Lrp binding , was identified ( Rex et a / .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8521047" target="_blank">8521047</a>]<br></li>
<li>in A 13-bp consensus sequence for Lrp binding , AgaA1T ' ` '' ATt cT , was derived from a comparison of Lrp binding sites upstream of the ilvIH operons of E. coli and S. typhimunium ( 28 , 30 ) , and a similar consensus was derived by Rex et al. from a comparison of sequences of Lrp-related genes ( 21 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8144448" target="_blank">8144448</a>]<br></li>
<li>These sites contain sequences ( Fig. 2 ) similar to a proposed 15 bp Lrp consensus binding sequence ( Cui et al. , 1995 ) and to a 7 bp Lrp consensus derived from studies in the papBA operon ( Nou et al. , 1995 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10216857" target="_blank">10216857</a>]<br></li>
<li>One of these sites ( Fig. 4 , site VI ) shows a perfect ﬁt to the 12-bp Lrp consensus binding sequence ( TTT ATTCtNaAT ) proposed by Rex et al. ( 28 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19074390" target="_blank">19074390</a>]<br></li>
<li>These sites show partial sequence conservation with the 15 bp Lrp consensus binding site [ CAG -- N9 ( A+T rich ) -- CTG ] , established on basis of Lrp targets identiﬁed by ChIP-chip experiments ( Cho et al. , 2008 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19906180" target="_blank">19906180</a>]<br></li>
<li>By sequence comparison , using the Lrp consensus sequence of Rex et al. ( 1991 ) , putative Lrp-binding sites could be identified in the stpA control region ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8890170" target="_blank">8890170</a>]<br></li>
<li>The three potential binding sites for Lrp identiﬁed in silico on the basis of sequence homology with the Lrp consensus box ( Cho et al. , 2008 ) are boxed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19906180" target="_blank">19906180</a>]<br></li>
<li>Protection extended only 1 base beyond the end of site 2 , as deﬁned by the consensus for Lrp binding ( 58-YAGHAWATTWT-DCTR ; Cui et al. , 1995 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9515701" target="_blank">9515701</a>]<br></li>
<li>Cui et al. ( 12 ) have shown that double-stranded DNA oligonucleotides containing five base pairs flanking a consensus Lrp binding site are sufficent for site-specific , high affinity , Lrp-DNA interactions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8900118" target="_blank">8900118</a>]<br></li>
<li>This site contains sequences that are a 13/15 match to a 15-bp Lrp consensus sequence recently proposed by Cui et al. ( 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955407" target="_blank">8955407</a>]<br></li>
<li>Furthermore , the site overlapping the 10 region contains 11 of 15 nucleotides that are identical to the consensus sequence for Lrp binding to DNA , described by Cui et al. [ 18 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16098526" target="_blank">16098526</a>]<br></li>
<li>The region of protection encompasses a sequence that shows signiﬁcant homology ( 9/15 bp ) to a consensus ( 58-YAGHAWATTWTDCTR ) derived for Lrp binding ( Cui et al. , 1995 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9515701" target="_blank">9515701</a>]<br></li>
<li>The nine regions that are protected from DNase I cleavage by Lrp in the ompC-micF intergenic region show only weak homology to the reported consensus sequence for Lrp , AGAATTTTATTCT ( 58 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8002608" target="_blank">8002608</a>]<br></li>
<li>The relatively poor matches with the Lrp consensus sequence are consistent with the low afﬁnity of Lrp for the ompC-micF intergenic region , compared with its afﬁn-ities for the promoter regions of other target genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8002608" target="_blank">8002608</a>]<br></li>
<li>The leucine-responsive protein ( Lrp ) was identiﬁed as another protein that speciﬁcally binds to a DNA sequence encompassing the lysP control region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21441513" target="_blank">21441513</a>]<br></li>
<li>Lrp binds cooperatively to the DNA at a degenerate consensu sequence ( 11 ) , but also with high afﬁnity in a nonspeciﬁc manner ( 36 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21441513" target="_blank">21441513</a>]<br></li>
<li>The amino acid sequence of Lrp was searched for an HTH motif .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8432705" target="_blank">8432705</a>]<br></li>
<li>A 12-bp sequence within that region differs from the consensus they defined by only 1 bp and is probably a site at which Lrp binds .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li>
<li>The results provided here indicate that Lrp binds in a sequence-speciﬁc manner to a consensus sequence YAGHAWATTWTDCTR , where Y 5 C or T , H 5 not G , W 5 A or T , D 5 not C , and R 5 A or G .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7665463" target="_blank">7665463</a>]<br></li>
<li>The sequence of site 2 closely matches the consensus sequence for Lrp binding , and Lrp binds to a 31 mer containing this sequence with a K in the nanomolar range ( 12 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8636076" target="_blank">8636076</a>]<br></li>
<li>The palindromic nature of the consensus sequence derived here suggests that Lrp binds to DNA as a homodimer in such a way that each monomer binds to a half-site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7665463" target="_blank">7665463</a>]<br></li>
<li>Also shown in the third line of the comparison in Table 5 is a consensus sequence derived by Rex et al. ( 32 ) from an analysis of sequences near promoters known to be affected by Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7665463" target="_blank">7665463</a>]<br></li>
<li>A computer analysis of 11 Lrp regulon-related sequences by Rex et al. ( 4 from ilvIH , 2 from lysU , and 1 each from tdh , sdaA , oppA , oppBDCF , livJ , and livKHMGF ) suggested ` l' l'ATTCt NaAT as a potential consensus sequence ( 127 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li>
<li>The consensus binding sequences we used were the following : 5 WYAGGMWWWWDYTW YWWW for binding of GadE , GadX , and GadW ( gadbox [ 26 , 43 ] ) ; 5 YAGHA WATTWTDCTR for Lrp ( 11 ) ; and 5 CTGTTCATAT for TorR ( 2 , 40 ) ; where D is A , T , or G ; H is A , C , or T ; M is A or C ; R is A or G ; W is A or T ; and Y is C or T. Only sequences with 70 % or higher identity to the consensus sequences were selected .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18083817" target="_blank">18083817</a>]<br></li>
<li>This consensus was used to identify similar 12 bp sequences in six other genes of the Lrp ( Rex el 1991 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8521047" target="_blank">8521047</a>]<br></li>
<li>Alignment of Lrp consensus-like sequences in the ilvG-MEDA operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8900118" target="_blank">8900118</a>]<br></li>
<li>Second , an earlier study demonstrated that a 5 bp ( half helical turn ) insertion between the proximal Lrp binding site and the promoter reduced gltB-lacZ expression 3-7-fold , while a 10 bp ( full helical turn ) insertion at the same location did not reduce fusion expression ( Fig. 5B of Wiese et al. [ 26 ] ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17233899" target="_blank">17233899</a>]<br></li>
<li>Lrp bound most tightly to sites 1 and 2 ( centers separated by 31 bp [ Fig. 2 ] ) , with an apparent dissociation constant of about 5 x 10-9 M. Binding of Lrp to one of these sites , when measured by the method of Tsai et al. ( 154 ) , was more than 100-fold tighter if the other site was already occupied by Lrp than if it was unoccupied .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li>
<li>An essential element of the Pap phase variation model is that the DNA methylation patterns in pap are a result of competition between Lrp and Dam for GATC sequences .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9811649" target="_blank">9811649</a>]<br></li>
<li>It seems likely that Lrp must dissociate from pap DNA to allow methylation of the GATC sequences .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9811649" target="_blank">9811649</a>]<br></li>
<li>Phase variation of pap requires the translocation of Lrp between sites 4 and 5 and sites 1 , 2 , and 3 , each containing one GATC sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9811649" target="_blank">9811649</a>]<br></li>
<li>Characterization of the 60 clones was based on DNA restriction patterns , analysis of the leucine and Lrp dependency of the GATC methylation patterns , and finally the sequence of the DNA adjacent to the GATC sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8195106" target="_blank">8195106</a>]<br></li>
<li>This translocation is mediated by converse methylation states of two Dam target sequences , one in each Lrp binding site , in conjunction with protein-protein interactions between Lrp and a pap-speciﬁc regulatory protein ( 5 , 7 , 44 , 46 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12029051" target="_blank">12029051</a>]<br></li>
<li>This idea implies not only that methylation affects Lrp binding but also that Lrp binding at a GATC-containing region blocks Dam-dependent methylation of GATC sequences .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9811649" target="_blank">9811649</a>]<br></li>
<li>Together , these results showed that Lrp binding was necessary and sufﬁcient to protect the GATC sequences in the pap regulatory region from methylation by Dam .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9811649" target="_blank">9811649</a>]<br></li>
<li>To determine if Lrp binding was sufﬁcient to protect these GATC sequences from methylation in vitro , Lrp was incubated with nonmethylated pap regulatory DNA containing GATC-I and GATC-II for 20 min , followed by the addition of Dam and the methyl group donor S-adenosyl-L-methionine to methylate accessible GATC sites ( see Materials and Methods ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9811649" target="_blank">9811649</a>]<br></li>
<li>Moreover , the pap GATC-I and GATC-II sequences are present within two different sets of binding sites for Lrp , designated sites 4 and 5 and sites 1 , 2 , and 3 , respectively ( 18 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9811649" target="_blank">9811649</a>]<br></li>
<li>Phase variation of pap expression and the protection against Dam-dependent methylation of the two GATC sequences result directly from the translocation of the leucine-responsive regulatory protein ( Lrp ) between two binding sites in the regulatory region , each containing a GATC sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10692164" target="_blank">10692164</a>]<br></li>
<li>The DNA methylation states of the pap GATC-I and GATC-II sites directly regulate Pap pilus phase variation by modulating the binding of Lrp to pap DNA sites which overlap GATC sequences ( 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9811649" target="_blank">9811649</a>]<br></li>
<li>The data presented in this paper support this latter part of the regulatory model by showing that in vitro , Lrp binding is sufﬁcient to block Dam-dependent methylation of the GATC sequences in the pap Lrp binding sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9811649" target="_blank">9811649</a>]<br></li>
<li>Binding of the regulatory protein Lrp at its target sites in pap resulted in methylation protection of GATC sequences located in the Lrp binding sites ( see above ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9811649" target="_blank">9811649</a>]<br></li>
<li>In turn , it is implied that Lrp binding must block methylation of the GATC sequences in the Lrp binding sites in pap .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9811649" target="_blank">9811649</a>]<br></li>
<li>© Oxford University Press Results and di ^ s ; c-ussion Lrp binds to double-Tstranded but not to single-stranded DNA Two complementary 31 nucleotide synthetic oligonucleotides ( Figure IA ) were synthesized that correspond to Lrp binding site 2 , located at -209 to -233 relative to the transcriptional start site of the ilvIH operon ( Wang and Calvo , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8508774" target="_blank">8508774</a>]<br></li>
<li>) , putative ® 35 and ® 10 hexamers ( underlined or overlined ) ; potential binding sites for FNR , Lrp and IHF ( boxed ) ; regions of partial dyad symmetry ( converging arrows ) ; ribosome-binding sites ( RBS , consensus bases italicized ) , and translation initiators ( fMet ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11101674" target="_blank">11101674</a>]<br></li>
<li>According to the methods of Cui et al. ( 33 ) , this site scores 4.81 arbitrary units of predicted binding energy ( sites with values .3 are predicted to be specific binding sites for Lrp ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8900118" target="_blank">8900118</a>]<br></li>
<li>Cui et al. ( 33 ) suggest that sequences with predicted values less than 3.0 are not likely to be specific Lrp binding sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8900118" target="_blank">8900118</a>]<br></li>
<li>We suspected that the AAA/TTT sequence present at the center of site 2 and at one end of both sites 1 and 3 might be critical for binding , since most Lrp binding sites contain a stretch of three to four adenines ( e.g. , 6/6 Lrp binding sites upstream of pap and 6/6 sites upstream of ilvIH contain AAA ( Nou et al. , 1995 ; Wang & Calvo , 1993a ) ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9236118" target="_blank">9236118</a>]<br></li>
<li>Site 2 shows clear palindromic symmetry ( Figure 6A ) ; its sequence is compared with two experimentally generated consensus sequences determined by Calvo and his colleagues , one derived from the 12 Lrp binding sites in the E. coli and Sal-monella typhimurium ilvIH operons ( Wang et al. , 1993 ) , and the other derived using the Selex meth-od ( Cui et al. , 1995 ; Tuerk & Gold , 1990 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9236118" target="_blank">9236118</a>]<br></li>
<li>The transcriptional start site ( indicated by xxx in the upper part of the figure and by the heavy arrow in the lower part ) and the most likely - 10 region are indicated , as well as regions with homology to the Lrp , CRP , and IHF consensus sites and a motif with the characteristics of curved DNA and palindromic symmetry ( thin arrows ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8253679" target="_blank">8253679</a>]<br></li>
<li>The transcription initiation site ( +1 ) , the promoter 10 and 35 regions [ 17 ] , and the GcvA [ 12 ] , Lrp [ 3 ] , PurR [ 30 ] and CRP [ 15 ] binding sites are indicated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15621456" target="_blank">15621456</a>]<br></li>
				<label class="SIT_collapse_similarity" for="sit_similarity_154">
The relative binding sites for GcvA ( Wilson et al. , 1995 ) , Lrp ( Stauffer & Stauffer , 1994 ) , CRP ( Wonderling & Stauffer , 1999 ) and PurR ( Wilson et al. , 1993a ) are shown with respect to the 1 transcription start site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11065369" target="_blank">11065369</a>]<br></label>
<input id="sit_similarity_154", type="checkbox">
<div class="SIT_similar">
<li>The transcription start site ( 1 ) , the promoter ® 10 and ® 35 regions ( Stauffer et al. , 1993 ) , and the GcvA ( Wilson et al. , 1995 ) , Lrp ( Stauffer & Stauffer , 1994 ) , PurR ( Wilson et al. , 1993a ) and CRP ( Wonderling & Stauffer , 1999 ) binding sites are indicated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12101307" target="_blank">12101307</a>]<br></li></div>
<li>CCG GG '' '' '' '' '' Site 6 # 197 GTTCAGCAGGAGATACTAAAGACG C GG C '' '' '' '' Site 7 # 231 AGGGAGATGTGAGCCAGCTCACCAT CG '' '' Site 8 # 272 TAATATTTTCAACTGAGTTA CG '' '' Site 9 # 308 TGATTAGATTATTATTCTTTTACT GC CGG '' '' '' '' '' Site 10 # 357 ATGCGAGTTGTCATACTGGGAAGT Taken together , these data suggest that the Lrp binding sites in the dad promoter region may be functionally distinct .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10216857" target="_blank">10216857</a>]<br></li>
<li>To determine the binding of His6-Lrp to the cadBA promoter/control region , a fragment comprising positions 150 to 72 upstream of the cadB gene was cloned into the EcoRI and BamHI sites of the pUC19 plasmid ( 60 ) and ampliﬁed with primers 6-FAM uni-24 ( 5 - ACGACGTTGTAAAACGACGGCCAG-3 ) and rev-24 ( 5 - TTCACACAG GAAACAGCTATGACC-3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21441513" target="_blank">21441513</a>]<br></li>
<li>By using vectors described by Kim et al. to generate circularly permuted fragments ( 73 ) , Lrp induced a bend of about 52 ° upon binding to a single site , and the angle of bending was increased to at least 1350 when Lrp bound to two adjacent sites ( 160 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li>
<li>Previously , Lrp has been reported to induce a bend of 52 ° at a single binding site and at least 135 ° at two adjacent sites ( Wang and Calvo , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19906180" target="_blank">19906180</a>]<br></li>
<li>For example , E.coli Lrp was found to induce a bend of 52 upon binding to a single site , progressing to a bend of at least 135 upon binding to two adjacent sites ( 11 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16528101" target="_blank">16528101</a>]<br></li>
<li>Lrp induced a bend of 520 uponbinding to a single binding site , and the angle of bending is increased to at least 135 ° when Lrp binds to two adjacent sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8508774" target="_blank">8508774</a>]<br></li>
<li>Again , it is not clear whether Lrp induces a single bend at position -150 , or whether the results reflect a series of bends induced at each of the six binding sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8508774" target="_blank">8508774</a>]<br></li>
				<label class="SIT_collapse_similarity" for="sit_similarity_163">
A schematic diagram shows the position of the sequence-directed bend ( a vertical arrow ) relative to the six Lrp binding sites and the start site of transcription ( horizontal arrow ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8508774" target="_blank">8508774</a>]<br></label>
<input id="sit_similarity_163", type="checkbox">
<div class="SIT_similar">
<li>A schematic diagram shows the position of the Lrp-induced bend , represented by a vertical arrow , relative to the six Lrp binding sites and the start site of transcription ( horizontal arrow ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8508774" target="_blank">8508774</a>]<br></li></div>
<li>Nucleotides identical to the CRP binding site consensus are indicated by vertical lines In vitro analysis of binding sites for cAMP-CRP and Lrp in the csiD promoter region In gel retardation assays , puri ® ed CRP protein ( in the presence of cAMP ) was found to reduce mobility of a digoxigenin-labeled PCR fragment containing 475 nucleotides upstream and 111 nucleotides downstream of the csiD transcriptional start site ( data not shown ) , suggesting that CRP regulation of csiD is probably due to direct binding of CRP in the regulatory region of csiD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9512707" target="_blank">9512707</a>]<br></li>
<li>As in panel A of Figure 2 except that a head to tail dimer was made from a 455 bp DNA fragment containing six Lrp binding sites spanning a region from -346 to + 110 of the ilvIH operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8508774" target="_blank">8508774</a>]<br></li>
<li>S - ij DNA bending induced upon Lrp binding to six Lrp binding sites within the ilvlH promoter/regulatory region A 455 bp DNA fragment spanning a region from -345 to + 110 of the ilvIH operon and containing six Lrp binding sites was dimerized head to tail ( Figure 4A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8508774" target="_blank">8508774</a>]<br></li>
<li>In vitro , Lrp binds in a sequence-specific fashion to multiple sites upstream of the ilvIH promoters from E. coli and Salmo-nella typhimurium ( 28 , 30 ) and bends DNA upon binding ( 29 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8144448" target="_blank">8144448</a>]<br></li>
<li>In vitro , Lrp binds to six sites within a 200-bp sequence of DNA upstream of the ilvIH promoter ( 20 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12218014" target="_blank">12218014</a>]<br></li>
<li>Wang and Calvo ( 1993a ) have demonstrated that Lrp activates transcription by binding co-operatively to six sites over a 200 bp region upstream from the ilvlH promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7603430" target="_blank">7603430</a>]<br></li>
<li>.1 f l , ... A '' '' DNase I footprinting suggests that a large nucleoprotein structure forms on binding of Lrp to ilvlH DNA The results of MPE ( methidiumpropyl-EDTA ) footprinting studies suggested that Lrp binds to six distinct sites spanning a region of > 200 bp upstream of the ilvIH promoter ( Wang and Calvo , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8508774" target="_blank">8508774</a>]<br></li>
<li>Lrp regulates ilvIH transcription by binding to DNA in a sequence-specific manner at six sites within the ilvIH promoter-control region and induces bending ( Wang and Calvo 1993b ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8152423" target="_blank">8152423</a>]<br></li>
<li>DNA bending induced upon Lrp binding to two adjacent sites In vitro , Lrp binds cooperatively to sites 1 and 2 , two sites separated by 30 bp that span a region from -253 to -212 relative to the start point of transcription of the ilvIH operon ( Wang and Calvo , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8508774" target="_blank">8508774</a>]<br></li>
<li>In the gltBDF operon , IHF binds the region from -75 to-113 , while the three Lrp binding sites are centered at nucleotides -152 , -215 and -246 relative to the transcription start site [ 5,14 ] ( Fig. 2A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17233899" target="_blank">17233899</a>]<br></li>
<li>Discussion Lrp binds far upstream of the gltBDF transcriptional start site This work identi ® es three Lrp binding sites upstream of gltBDF , the most distal of which is located more than 250 bp upstream of the transcription start site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9236118" target="_blank">9236118</a>]<br></li>
<li>There are three Lrp binding sites centered 246 , 215 , and 152 bp upstream of the gltBDF transcription start site ( 35 ) ; the closest of these sites is ; 110 bp from the 235 hexamer .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11395454" target="_blank">11395454</a>]<br></li>
<li>Protection from both phenanthroline copper and DNase I extends well beyond sites 1 and 2 , and here it is shown that Lrp binds to a third region adjacent to site 1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9515701" target="_blank">9515701</a>]<br></li>
<li>Despite the absence of a CpxR-P box in the pap regulatory region , CpxR-P binds to the papI -- papB intergenic region at sites overlapping the Lrp ( leucine-responsive regulatory protein ) binding sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16487683" target="_blank">16487683</a>]<br></li>
<li>Although in the presence of leucine some in vivo interaction of Lrp with the ilvIH regulatory region may still occur , binding to sites 1 , 3 , 4 , and 5 is impaired .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8071194" target="_blank">8071194</a>]<br></li>
<li>Lrp is a negative regulator of micF transcription and binds to multiple sites in the micF promoter as well as in the micF coding sequence and downstream sequences beyond micF .29 A major Lrp binding site in the micF promoter overlaps the binding site for the transcription activators Rob / MarA/SoxS ( see below ) and consists of the sequence Gÿ52ACAGCACTGAATÿ40 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11601842" target="_blank">11601842</a>]<br></li>
<li>Lrp binds to at least two sites within the ilvZH promoter/regulatory region , at positions -40 to -100 and -190 to -260 , relative to the startpointof transcription ( 4 ) and mutationswithin these binding sites reduce expression of the iluIH operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2040596" target="_blank">2040596</a>]<br></li>
<li>Lrp binds to multiple sites within the i!vlH promoter-regula-tory region and this binding is necessary ( Wang and Calvo 1993a ) and sufficient ( Willins et al. 1991 ; Platko and Calvo 1993 ) for the transcription of the operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8152423" target="_blank">8152423</a>]<br></li>
<li>In vitro studies have shown that Lrp binds to multiple sites over a large region upstream of Lrp-regulated promoters and that this binding is modulated by leucine or alanine ( 17 , 30 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955407" target="_blank">8955407</a>]<br></li>
<li>This exact palindrome does not occur in the gltBDF promoter region , even though this promoter region binds Lrp with an affinity comparable to that of Lrp for the high-affinity sites upstream of the ilvIH operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7901196" target="_blank">7901196</a>]<br></li>
<li>ArgR also binds to the gltBDF promoter region , upstream of the Lrp binding sites , and decreases transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17233899" target="_blank">17233899</a>]<br></li>
<li>We show that Lrp binds specifically to a site or sites upstream of the gltBDF operon and compare this binding to the binding of Lrp to the ilvIH promoter region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7901196" target="_blank">7901196</a>]<br></li>
<li>More recently , a model based on the crystal structure of Lrp has been proposed in which a 118 bp DNA duplex containing three binding sites , roughly separated by three helical turns , is smoothly wrapped around octameric Lrp ( de los Rios and Perona , 2007 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19906180" target="_blank">19906180</a>]<br></li>
<li>DNA fragments containing the 174 ( 5 ) mutation ( with a 5 bp insertion between bp ÿ174 and ÿ175 ) showed protection of sites 1 and 2 in the presence of Lrp , but did not show protection at site 3 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9236118" target="_blank">9236118</a>]<br></li>
<li>As in panel A of Figure 2 except that PCRXH , a 171 bp DNA fragment prepared by PCR containing Lrp binding sites 1 and 2 , was cloned into the XbaI site of pBend2 ( Kim et al. , 1989 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8508774" target="_blank">8508774</a>]<br></li>
<li>If this element does indeed deﬁne the binding site for Lrp in region 3 of complex 1 , then the separation between sites 1 and 2 and between sites 1 and 3 are identical ( 16 bp ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9515701" target="_blank">9515701</a>]<br></li>
<li>The binding of Lrp protected gltBDF DNA from cleavage at three sites ( Figure 2 ) : site 1 ( bp ÿ244 to ÿ258 ) , site 2 ( bp ÿ205 to ÿ225 ) , and site 3 ( bp ÿ142 to ÿ161 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9236118" target="_blank">9236118</a>]<br></li>
<li>The relative sizes of DNA and Lrp , the positions of binding sites , and the 2 - , 4 - , 6 - , 8 - , and 10-bp addition site ( triangle ) are drawn roughly to scale , but the path of DNA relative to Lrp is imaginary .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12218014" target="_blank">12218014</a>]<br></li>
<li>For the future , better prediction will likely require an improved correlation between predicted and observed binding energies for Lrp binding to 15-mer sequences and more information about the requirements for cooperative binding of Lrp to multiple sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7665463" target="_blank">7665463</a>]<br></li>
<li>Next we applied the matrix analysis to naturally occurring sequences , some within coding regions of genes ( not expected to have Lrp binding sites ) and some known to bind Lrp in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7665463" target="_blank">7665463</a>]<br></li>
<li>We illustrate the robustness of the metbod on a database that includesan equal number of known CRP-binding sites and random sequences , and del110nstrate its application to the Lrp regulon , for which httle experimental binding data exists .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8521047" target="_blank">8521047</a>]<br></li>
<li>The upper part of the diagram depicts the relative positions of RNAP ( grey ovals ) and Lrp ( black boxes ) binding sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1447135" target="_blank">1447135</a>]<br></li>
<li>The 3 end of the ﬁmE gene is shown on the left side of the diagram , and the positions of the three binding sites for Lrp ( ﬁlled ovals with Roman numerals ) and the boxA sequence ( ﬁlled box ) are indicated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16321930" target="_blank">16321930</a>]<br></li>
<li>Demonstrated Lrp binding sites are in underlined lowercase italics , and the -35 and -10 sequences inferred from the known +1 position ( for E. coli ) are boxed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18405378" target="_blank">18405378</a>]<br></li>
<li>The presumptive 10 and 35 promoter sequences ( open boxes ) and the locations of the six putative Lrp binding sites ( shaded boxes ) are shown .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19074390" target="_blank">19074390</a>]<br></li>
<li>The primer extension sites T1 -- T4 are indicated by arrowheads , the putative -- 35 and -- 10 sequences are underlined , the upstream sequence protected by Lrp is given in bold , and a possible Crp binding site is indicated by a dotted underline .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11985712" target="_blank">11985712</a>]<br></li>
<li>The data in Fig. 6 were fitted to equation 1 , the Hill equation ( 10 ) , assuming n Lrp binding sites : -2591 bp 1370 bp - 4 - 916 bp -677 bp -405 bp 406 bp - 11 ilvlH gItBDF Lrp ( nM ) ° 3 re , c. 3 ° , < ; > OI o I * q , * r , ICI FIG. 3 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7901196" target="_blank">7901196</a>]<br></li>
<li>PCR fragments LP-2 and LP-7 , but not fragment LP-3 or LP-6 , bound purified Lrp , localizing binding sites to the region -88 to -12 ( Fig. 4A and C ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8144448" target="_blank">8144448</a>]<br></li>
<li>The functional importance of the presumptive Lrp binding sites was assessed by PCR ampliﬁcation of four regions containing the putative promoter and progressively greater amounts of DNA from the region upstream of the promoter , using pCLO7 plasmid DNA as a template ( Fig. 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19074390" target="_blank">19074390</a>]<br></li>
<li>The data indicate that 11 Lrp binding sites , from approximately ÿ275 to 90 , are located in this region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10216857" target="_blank">10216857</a>]<br></li>
<li>The cooperative binding of Lrp to the argO control region and the footprinting data ( Figs 3 and 4 ) suggest the binding of at least three Lrp dimers to regularly spaced binding sites ( Fig. 1 ) and wrapping of the operator DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19906180" target="_blank">19906180</a>]<br></li>
<li>E -- G. EMSAs of Lrp binding to 46 bp duplexes bearing the potential Lrp binding sites Pot-box 1 , Pot-box 2 or Pot-box 3 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19906180" target="_blank">19906180</a>]<br></li>
<li>To evaluate Lrp binding to these separate potential boxes we performed EMSAs with 46 bp oligonucleotide duplexes ( see Table S1 ) bearing either one of the three potential Lrp binding sites ﬂanked by identical 15 bp stretches corresponding to the 15 bp stretch upstream of Pot-box 3 and the 15 bp stretch downstream of Pot-box 1 ( Fig. 3E -- G ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19906180" target="_blank">19906180</a>]<br></li>
<li>This protection extended ~ 30 bp upstream of the -35 hexamer , merging with the footprint of IHF , and some protection was visible even farther upstream between the IHF and Lrp binding sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17233899" target="_blank">17233899</a>]<br></li>
<li>To investigate potential phase relationships between upstream and downstream Lrp binding sites , we used PCR mutagenesis to insert up to 10 additional base pairs between the two regions in increments of 2 bp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12218014" target="_blank">12218014</a>]<br></li>
<li>coli , Lrp interacts more strongly with an upsfream region containing two binding sites than with a downstream region containing four binding sites ( Wang and Calvo , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8483418" target="_blank">8483418</a>]<br></li>
<li>identificatíon of binding si tes for Lrp has been quite sites in ilvIH sequence were mapped far upstream from the mRNA start site ( Wang and Calvo , 1993 ) , we decided to use a region spanning 300 nucleotides upstream and 100 nucleotides downstream froI11 ' the transcription start position of each gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8521047" target="_blank">8521047</a>]<br></li>
<li>Wang and Calvo ( 159 ) argued that Lrp bound to site 5 probably activated transcription and that cooperative binding to sites 3 and 4 helped to deliver Lrp to site 5 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li>
<li>Binding was indeed severely reduced , but some residual binding was still evident for these mutated constructs ( 20 ) , highlighting the fact that the rules by which Lrp recognizes binding sites have not yet been established .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12218014" target="_blank">12218014</a>]<br></li>
<li>Taken together , these data suggest that sites 1 and 3 must either be a speci ® c distance from , or on a speci ® c face of the helix with respect to , site 2 in order to be bound by Lrp , and that the binding at the ¯ anking sites ( 1 and 3 ) must be sequencespeci ® c .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9236118" target="_blank">9236118</a>]<br></li>
<li>Note that while the binding of Lrp to the ¯ anking sites is cooperativ with binding to the center site , the occupancy of one ¯ anking site is not cooperative with occupancy of the other .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9236118" target="_blank">9236118</a>]<br></li>
<li>Similar results were obtained with a strain that contained Lrp binding sites 3 , 4 , 5 , and 6 but lacked binding sites 1 and 2 ( Fig. 1 , compare lines 1 , 2 , and 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12218014" target="_blank">12218014</a>]<br></li>
<li>To define more precisely the degree of bending associated with Lrp binding , one or two binding sites were separately cloned into a pBend vector and analyzed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8508774" target="_blank">8508774</a>]<br></li>
<li>Analysis of Lrp binding to speciﬁc sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7665463" target="_blank">7665463</a>]<br></li>
<li>' Zn uitro , leucine reduces the extent of binding of Lrp to these binding sites ( 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2040596" target="_blank">2040596</a>]<br></li>
<li>Phase variation of pap requires translocation of Lrp between two binding sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10692164" target="_blank">10692164</a>]<br></li>
<li>Lrp initiates binding at one of two sites within thefim switch .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7916011" target="_blank">7916011</a>]<br></li>
<li>Second , leucine-mediated repression of ilvIH expression can be understood in simple terms : Lrp activates transcription from the ilvIH promoter by binding to specific sites , and leucine reduces such binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li>
<li>Whether Lrp acts directly , by binding to liv operator sites and repressing transcription , or indirectly remains to be established .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1729203" target="_blank">1729203</a>]<br></li>
<li>Taken together , these results suggest that Lrp activates transcription by binding to one of the downstream sites and directly interacting with a neighboring RNA polymerase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li>
<li>For the case of ilvIH , Lrp binds to six distinct sites , and the binding is highly cooperative to two groups of those sites ( 10 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8636076" target="_blank">8636076</a>]<br></li>
<li>Dimeric Lrp binds cooperatively to at least three regularly spaced semi-palindromic binding sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19906180" target="_blank">19906180</a>]<br></li>
<li>Previous work showed that Lrp binds with highest avidity to the upstream sites ( 20 ) , and we presume that K1 reﬂects binding to upstream sites for all of these constructs .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12218014" target="_blank">12218014</a>]<br></li>
<li>Lrp binds to six sites upstream of iivlH , and Lrp binding is required for / / WH expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8483418" target="_blank">8483418</a>]<br></li>
<li>Except for sites V and VI ( Fig. 4 ) , the rest of the potential Lrp binding sites identiﬁed are located far upstream from the putative promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19074390" target="_blank">19074390</a>]<br></li>
<li>Lrp from a wild-type strain bound specifically to several sites upstream of the ilvIH operon , whereas binding by Lrp from a livRIcontaining strain was barely detectable .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1729203" target="_blank">1729203</a>]<br></li>
<li>At these concentrations in vitro , , all of the binding sites upstream of ilvIH would contain bound Lrp , even in the presence of leucine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li>
<li>One of those operons , ilvIH , contains six Lrp binding sites located within a several hundred base pair region upstream of the .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8508774" target="_blank">8508774</a>]<br></li>
<li>ing of Lrp to sites 1 and 2 ( upstream region ) is highly cooperative , as is binding to sites 3 , 4 , 5 , and 6 ( downstream sites ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12218014" target="_blank">12218014</a>]<br></li>
<li>Organization of Lrp binding sites upstream of ilvIH in Salmonella typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8144448" target="_blank">8144448</a>]<br></li>
<li>Lrp ( cylinders ) is shown binding to upstream ( hatched ) and downstream ( stippled ) sites near the ilvIH promoter ( P3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12218014" target="_blank">12218014</a>]<br></li>
<li>In these experiments , Lrp initially formed a complex involving cooperative binding to two high-affinity sites upstream of the ilvIH promoter ( resulting in the smaller mobility shift ) and then bound cooperatively to lower-affinity sites located closer to the start site for transcription ( resulting in the larger mobility shift ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7901196" target="_blank">7901196</a>]<br></li>
<li>Our observation that transcriptional activation by Lrp is dependent on the phasing between the Lrp binding sites upstream of gltBDF and the start of transcription are also consistent with a direct contact between bound Lrp and RNA polymerase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9236118" target="_blank">9236118</a>]<br></li>
<li>The analysis above calls into question our earlier interpretation of the importance of upstream binding sites and our focus on Lrp bound at downstream sites as the most signiﬁcant factor in transcription activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12218014" target="_blank">12218014</a>]<br></li>
<li>Complementary sequences F and R comprise a double-stranded 50 mer that bound Lrp most strongly from a collection of 63 sequences that were selected in vitro for binding to Lrp ( Leu-19 in Ref .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8636076" target="_blank">8636076</a>]<br></li>
<li>Mutations in these two sequences resulted in a reduction in Lrp binding to fim in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7916011" target="_blank">7916011</a>]<br></li>
<li>One possible explanation is that Lrp binds to two different consensus sequences and that high concentrations of leucine decrease binding to sequences of one type and increase binding to the other .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li>
<li>To study the role of the DNA sequences downstream of the proximal Lrp binding site centered at 2152 in the regulation of the glutamate synthase ( gltBDF ) operon of E. coli , these downstream sequences were systematically replaced with the corresponding regions of the lacZYA operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11395454" target="_blank">11395454</a>]<br></li>
<li>To test the idea that Lrp in the presence of leucine might bind to a different subset of DNA sequences , we carried out a second selection experiment with leucine present during the binding reactions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7665463" target="_blank">7665463</a>]<br></li>
<li>The definition of the DNA sequences important for the Lrp regulation of gltBDF expression will be useful in the continuing effort to define DNA sequences recognized by Lrp in specific binding events and is currently under investigation in our laboratory .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7901196" target="_blank">7901196</a>]<br></li>
<li>For at least the ilvIH operon , Lrp regulates gene expression directly by binding to speciﬁc DNA sequences ( 42 , 44 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7665463" target="_blank">7665463</a>]<br></li>
<li>We identiﬁed 63 related DNA sequences that bound Lrp and estimated their relative binding afﬁnities for Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7665463" target="_blank">7665463</a>]<br></li>
<li>Overlapping DNA fragments that included sequences both adjacent to and within the fim switch were tested for Lrp binding in gel retardation assays ( Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7916011" target="_blank">7916011</a>]<br></li>
<li>CpxR-P most likely inhibits transition to the ON position by competing with Lrp for binding to the pap promoter [ 20 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16487683" target="_blank">16487683</a>]<br></li>
<li>Puriﬁed Lrp , CAP , or GutR was added to binding reaction mixtures and incubated for 20 min at 37 ° C .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9811649" target="_blank">9811649</a>]<br></li>
<li>Since sorne of the Lrp-binding Search for a C01111110/1 motiffor Lrp binding We searched for a 15 bp long motif .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8521047" target="_blank">8521047</a>]<br></li>
<li>Furthermore , one or more half motifs spaced about 30 bp from a palindromic motif showed cooperative and strong binding to Lrp , with binding constants in the range 4-20 nanomolar .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8483418" target="_blank">8483418</a>]<br></li>
<li>Double-stranded299 bp MluI , Xhol binding region , but separates the upstream 16 bp of the and BamHI DNA fragments from pGS281 carrying the Lrp - Lrp-protected region from the rest of the Lrp-binding binding region at different positions were 32P-labelledat the 5 ' region ( Fig .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9802028" target="_blank">9802028</a>]<br></li>
<li>We note that a very similar sequence is present upstream from the osmC promoters ( Fig. 1 ) and is likely to be responsible for Lrp binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9663683" target="_blank">9663683</a>]<br></li>
<li>Lrp and IHF bind and , according to the model , bend gltBDF promoter DNA such that an upstream regulatory sequence or Lrp surface required for stable binding of RNAP is brought in contact with RNAP that is loosely associated with the promoter region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17233899" target="_blank">17233899</a>]<br></li>
<li>B Lrp ArgR ( -- ) + Crp IHF ( + ) ( -- ) RNAP gFlitgBDurFepr2omoter region Our inspection of the gltBDF DNA sequence revealed a potential Crp-binding site between -76 and -55 ( Fig. 5A , top two lines ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17233899" target="_blank">17233899</a>]<br></li>
<li>However , given the unusual position of the primary Lrp binding site 226 bp downstream from the transcriptional initiation site and 40 bp downstream of the transcriptional termination site of the attenuator at base pair position 186 , it is unclear how Lrp affects transcription from the upstream promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8900118" target="_blank">8900118</a>]<br></li>
<li>The DNA used was Lrp binding site 2 that is at position 2232 to 2211 upstream of the E. coli ilvIH promoter ( 10 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8636076" target="_blank">8636076</a>]<br></li>
<li>RESULTS Stoichiometry of Lrp Binding to DNA -- For these studies , we chose site 2 , a strong Lrp binding site located at positions 2232 to 2211 upstream of the E. coli ilvIH promoter ( 10 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8636076" target="_blank">8636076</a>]<br></li>
<li>Based on our measurements of the length of the truncated ﬁmE transcript that is associated with OFF-phase ﬁmS and our knowledge of the ﬁmE transcription initiation site , we estimated that the most likely location for the DNA sequence specifying the terminator would be at or near Lrp binding site III ( 23 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16321930" target="_blank">16321930</a>]<br></li>
<li>Lrp bends DNA when it binds to a single binding site To examine whether Lrp can induce a bend in DNA upon binding , we employed a circular permutation assay developed by Wu and Crothers ( 1984 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8508774" target="_blank">8508774</a>]<br></li>
<li>Here we report the stoichiometry of Lrp binding ( an Lrp dimer binds to a single binding site ) and studies that define the minimal length of DNA required for binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8636076" target="_blank">8636076</a>]<br></li>
<li>In addition , we performed stoichiometry experiments that demonstrate that a single Lrp binding site binds one Lrp dimer .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8636076" target="_blank">8636076</a>]<br></li>
<li>IHF ( dotled circ1es ) binding lo site 11 , and Lrp ( hatched circ1es ) binding wlthin the fim switch , help bend the DNA to align IRL and IRR in the correel orientation lor synapsis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9157242" target="_blank">9157242</a>]<br></li>
<li>Mobilities , measured as cm from the top of the gel to the Lrp - DNA complexes shown in panel B , were plotted as a function of the position ( bp ) of binding site 2 relative to the fragment ends .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8508774" target="_blank">8508774</a>]<br></li>
<li>The shape of the curve relating mobility to position of the binding site , a smooth curve ( Figure 2C ) , strongly suggests that Lrp bends DNA upon binding to it .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8508774" target="_blank">8508774</a>]<br></li>
<li>It should be noted that the final 16 bp of the upstream Lrp-binding region is still present in the insertion mutants , and it is possible that Lrp bound to this site alone is sufficient for cooperative binding with GcvA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9802028" target="_blank">9802028</a>]<br></li>
<li>Bases conserved in all four genera are shown in bold , and the shaded regions indicate the proximal Lrp binding site , the poly ( A ) region upstream of the IHF binding site , the consensus IHF binding site ( 295 to 283 ) , and the 235 and 210 hexamers .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11395454" target="_blank">11395454</a>]<br></li>
<li>The proposed IHF-induced bend in the gltBDF upstream region would bring the proximal Lrp binding site close to the region just upstream of the 235 hexamer , a position where 70 activators of s - dependent promoters usually bind ( 19 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11395454" target="_blank">11395454</a>]<br></li>
<li>DNase I footprinting in the serA upstream region The above experiments show the existence of a highafﬁnity Lrp binding site between -- 168 and -- 52 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11985712" target="_blank">11985712</a>]<br></li>
<li>A short horizontal line represents a site that remains accessible to DNase I in the presence of Lrp , and a short horizontal line with a dot represents a site that becomes hyperreactive for DNase I upon Lrp binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19906180" target="_blank">19906180</a>]<br></li>
<li>This Lrp binding site starts about 10 bp downstream from a putative catabolite repressor protein ( Crp ) binding site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11985712" target="_blank">11985712</a>]<br></li>
<li>Lanes 4 -- 6 and 10 -- 18 also show a large fragment from the Bluescript vector that has at least one Lrp binding site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11985712" target="_blank">11985712</a>]<br></li>
<li>( A ) Nucleotide sequences of synthetic oligonucleotides a and b used to prepare double-stranded Lrp binding site 2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8508774" target="_blank">8508774</a>]<br></li>
<li>At higher Lrp concentrations , binding to a third site is observed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8900118" target="_blank">8900118</a>]<br></li>
<li>Lrp and meAdamay bind the same binding site , and meAdamight be able to out-compete Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8809748" target="_blank">8809748</a>]<br></li>
<li>Lrp induces a bend of about 50 ° in binding to a single site ( 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8636076" target="_blank">8636076</a>]<br></li>
<li>Here we investigate further the requirements for Lrp interaction with a single binding site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8636076" target="_blank">8636076</a>]<br></li>
<li>An analysis of binding site mutations showed that the mutation in site 5 strongly reduced binding to site 5 but increased the af ® nity of Lrp for site 9 ( Fig. 6C and D ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10216857" target="_blank">10216857</a>]<br></li>
<li>Plasmids pCV157 , pCV158 , and pCV159 are identical to pCV156 except that they carry mutations in the -72 Lrp binding site ( X7ho A8 , and 10 mutations , respectively ; sequences are shown in Fig. 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1447135" target="_blank">1447135</a>]<br></li>
<li>Combined , the data from the in vitro binding studies indicate that Lrp contacts sequences overlapping the ArgP binding site and sequences extending further upstream ( Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19906180" target="_blank">19906180</a>]<br></li>
<li>The top line represents part of vector pBend5 ( Kim et al. , 1989 ) containing Lrp binding site 2 flanked by two direct repeats , each 113 bp in length .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8508774" target="_blank">8508774</a>]<br></li>
<li>The 133 bp HindIII-PmeI Results from previous studies led t o the hypothesis that fragments carrying the I1A site were ligated into the HindIIIl Lrp , through its ability to bind t o the gcv-control region PmeI sites of pGS398 , replacing a 133 bp region of pGS398 and all but the 5 ' 22 bp of the Lrp-binding sequence ( Fig .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10217490" target="_blank">10217490</a>]<br></li>
<li>( OP ) 2Cu2 + footprinting confirmed Lrp binding to a core region ( 45 bp ) that stretches from fim site 1 to site 2 ( Fig. 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7916011" target="_blank">7916011</a>]<br></li>
<li>The presence of Lrp protected some regions from DNase and sensitized others over a length of at least 85 bp from -- 158 to -- 82 , corresponding to the high-afﬁnity binding site described above ( Fig. 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11985712" target="_blank">11985712</a>]<br></li>
<li>The binding site is at least partially located within the 25 bp between -40 and -65 because fragments LP-3 and LP-6 do not bind Lrp ( Fig. 4A and C ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8144448" target="_blank">8144448</a>]<br></li>
<li>The distance between the proximal Lrp binding site and the 235 hexamer was 1 bp shorter than in the other constructs ( 111 versus 110 bp ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11395454" target="_blank">11395454</a>]<br></li>
<li>The data also suggest that Lrp binding to a site that includes bp 110 and 111 is critical for this inhibition .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955407" target="_blank">8955407</a>]<br></li>
<li>In the gltBDF operon , the proximal Lrp binding site centered at 2152 and the RNA polymerase recognition sequence at 234 are farther apart ( more than 110 bp ) than is usually seen in other s70-dependent promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11395454" target="_blank">11395454</a>]<br></li>
<li>Selex-D is a double-stranded 50-mer having a central 20-bp sequence that is a known Lrp binding site ( binding site 4 upstream of the ilvIH operon ) ( 40 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7665463" target="_blank">7665463</a>]<br></li>
<li>The apparent center of bending after Lrp binding , -150 , is 20 bp to the left of the sequence-directed center of bending .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8508774" target="_blank">8508774</a>]<br></li>
<li>It should be noted that site 2 is totally within the region shown previously to be protected from DNase I digestion by the Lrp protein ( 41 ) , and we believe this site is probably not accessible to CRP in vivo due to Lrp binding to this sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10074087" target="_blank">10074087</a>]<br></li>
<li>A structure-based multiple sequence alignment of the AsnC/Lrp family is shown and the residues forming the ligand-binding site are identified .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16528101" target="_blank">16528101</a>]<br></li>
<li>DNase I footprinting of Lrp binding in the absence or presence of L-alanine to fragment I containing the wild-type sequence ( A ) or the mutation # 163 in site 5 ( C ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10216857" target="_blank">10216857</a>]<br></li>
<li>In cases when an Lrp-binding site contains a GATC sequence , the nature of Lrp binding can be affected by whether the GATC sequence is methylated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li>
<li>Whereas Lrp bound to the site containing GATC ' 130 whether or not this sequence was methylated , the DNase I footprint for Lrp binding to GATC1130 was clearly different for the methylated and nonmethylated sites ( 107 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li>
<li>The extended protection seen at site 1 in the presence of leucine involves a region with only a poor match ( 4/10 ) to the right consensus half site for Lrp binding shown in Figure 6A .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9236118" target="_blank">9236118</a>]<br></li>
<li>The possible binding site ( s ) for Lrp in the aid8 promoter regioncould not be readily identified : a poor homologywith Lrp consensus sequences can be found in the -80 to -40 region , which also contains the binding sites for meAdaand for the a subunit of RNA polymerase ( Landini and Volkert , 1995b ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8809748" target="_blank">8809748</a>]<br></li>
<li>There are two good matches to the consensus ArgR binding site sequence [ 24,25 ] upstream of the promoter-distal Lrp binding site ( Figs. 2A and 8A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17233899" target="_blank">17233899</a>]<br></li>
<li>The two most promoter proximal Lrp boxes are conﬁned within the ArgP binding site , whereas Pot-box 3 , which shows the most similarities to the consensus , is located slightly further upstream .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19906180" target="_blank">19906180</a>]<br></li>
<li>The lower part of the diagram shows the DNA sequence near the -72 Lrp binding site and promoter P2 ( -35 and -10 regions are indicated ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1447135" target="_blank">1447135</a>]<br></li>
<li>The approximate distance between the proximal Lrp binding site and the IHF site is conserved among the different genera , but the sequence of the intervening region is not highly conserved except for the poly ( A ) region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11395454" target="_blank">11395454</a>]<br></li>
<li>The Lrp binding site in the ilvGMEDA operon is highly conserved in sequence , position , and alignment orientation in the four closely related enterobacteria , S. typhimurium , E. tarda , S. marscescens , and K. aerogenes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8900118" target="_blank">8900118</a>]<br></li>
<li>When an Lrp dimer is bound to a single site , bending angles of about 30 '' and 26 '' were calculated for the XhoI and BamHI fragments relative to the MluI fragment , with the bending centre located near the centre of the XhoI fragment .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9802028" target="_blank">9802028</a>]<br></li>
<li>For the first pattern , characteristic of alleles urp-1 ,34 , lrp-24 , an lrp-27 , we observed a single Lrp-DNA complex that corresponded in mobility to the wild-type complex having Lrp bound at a single upstream site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8432705" target="_blank">8432705</a>]<br></li>
<li>Lrp appeared to act directly to inhibit methylation since Lrp specifically bound to a DNA fragment this site became containing this GATC site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li>
<li>The position of the DMS protection in site 2 at base pair position 199 , three and four integral turns of the DNA helix away from the DMS protection sites in site 1 at base pair positions 229 and 238 , suggests that Lrp molecules bound to sites 1 and 2 are separated by one helical turn and are aligned along the same face of the DNA helix .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8900118" target="_blank">8900118</a>]<br></li>
<li>If Lrp is behaving like a typical dimeric helix-turn-helix protein when it binds to the palindromic site 2 , we can also infer which way Lrp bends the DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9236118" target="_blank">9236118</a>]<br></li>
<li>Our results suggest that Lrp binds a central palindromic site , interacting predominantly with the major groove of its DNA target , and that additional dimers bind to ¯ anking sites to form a nucleoprotein activation complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9236118" target="_blank">9236118</a>]<br></li>
<li>These data are consistent with a model in which the consensus-like palindromic central site acts as an anchor to pull Lrp to the DNA , with protein/protein interactions and/or induced changes in local DNA structure then allowing Lrp dimers to bind cooperatively to the ¯ anking sites to form a nucleoprotein regulatory complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9236118" target="_blank">9236118</a>]<br></li>
<li>After single - GcvA and Lrp bind co-operatively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9802028" target="_blank">9802028</a>]<br></li>
<li>Evidence suggesting that two Lrp monomers bind to a single site on DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8636076" target="_blank">8636076</a>]<br></li>
<li>The conclusion that two Lrp monomers bind to a single site was confirmed by employing the strategy of Hope and Struhl ( 22 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8636076" target="_blank">8636076</a>]<br></li>
<li>At present , we can not decide whether cAMP-CRP , Lrp , and IHF directly bind to the osmY regulatory region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8253679" target="_blank">8253679</a>]<br></li>
<li>Previous work showed that Lrp binds at the pap GATC sequences in vitro ( 5 , 18 , 19 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9811649" target="_blank">9811649</a>]<br></li>
<li>We are currently determining if Lrp binds to the GATC box regions of clone 3-1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8195106" target="_blank">8195106</a>]<br></li>
<li>This analysis indicated that Lrp binds strongly and with about equal avidity to one region of these constructs , at least within a factor of about 2 ( Table 3 , compare K1 values ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12218014" target="_blank">12218014</a>]<br></li>
<li>Lrp binds speciﬁcally to the intergenic region between micF and ompC , as indicated by mobility shift assays and by DNase I footprinting .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8002608" target="_blank">8002608</a>]<br></li>
<li>Mobility shift assays with DNA containing the ompC-micF intergenic region indicated that Lrp bound specifically to this region , forming at least two complexes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li>
<li>Lrp binds to DNA containing the ompC-micF intergenic region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8002608" target="_blank">8002608</a>]<br></li>
<li>The gene coding for Lrp ( lrp ) has been cloned ( 1 , 8 , 9 , 22 , 30 ) and sequenced ( 1 , 45 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7665463" target="_blank">7665463</a>]<br></li>
<li>The results confirm that both GcvA and Lrp are required for expression of gcv and that sequences greater than 313 bp upstream of +1 are required for normal gcv expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7928983" target="_blank">7928983</a>]<br></li>
<li>The promoter sequences for five of the six loci that are regulated by Lrp are known .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1917830" target="_blank">1917830</a>]<br></li>
<li>Binding of Lrp protein to the upstream region of the serA promoter The profound effect of Lrp on serA transcription suggests that Lrp must bind to promoter sequences .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11985712" target="_blank">11985712</a>]<br></li>
<li>The methylation pattern of the 5 cents - GATC-3 cents sequences dictates whether Lrp binds to sites overlapping the promoter , thus blocking transcription by steric hindrance of RNA polymerase , or binds to other distant sites allowing transcription to proceed ( Blyn et al. , 1990 ; Braaten et al. , 1994 ; Van der Woude et al. , 1996 ; Weyand and Low , 2000 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11972787" target="_blank">11972787</a>]<br></li>
<li>-LCB- belo , , > , the sequencel , are shown , Binding sites 1 and 2 for Lrp ( Gally et al. , 1994 ) , and the timA promoter -10 and -35 homologues ( Olsen and Klemm , 1994 ) , are also shown .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9157242" target="_blank">9157242</a>]<br></li>
<li>This compares with a size of 13 ± 21 bp and a spacing of 6 ± 68 bp between the Lrp sites in ilvIH ( Wang and Calvo , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10216857" target="_blank">10216857</a>]<br></li>
<li>In vitro , Lrp binds specifically and wifh relatively high affinity to six sites upsfream of fhe iivlH promoter -LCB- Ricca et ai , 1989 ; Wang and Calvo , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8483418" target="_blank">8483418</a>]<br></li>
<li>As previously shown by Ricca et al. ( 17 ) and Wang and Calvo ( 20 ) , two major bands were observed in these titrations , representing a fastermoving species containing Lrp bound to sites 1 and 2 and a slower-moving species containing Lrp bound to sites 1 , 2 , 3 , 4 , 5 , and 6 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12218014" target="_blank">12218014</a>]<br></li>
<li>Previous MPE foofprinting studies carried out wifh purified Lrp and E. coli DNA established fhat the faster-moving complex had Lrp bound to two upstream sites -LCB- sites 1 and 2 ) whereas fhe slower-moving complex had Lrp bound , in addition , to four downsfream sites ( sifes 3 , 4 , 5 , and 6 ) -LCB- Wang and Calvo , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8483418" target="_blank">8483418</a>]<br></li>
<li>In a model put forward by Wang and Calvo for regulation of ilvIH expression , Lrp bound at downstream sites 3 , 4 , 5 , and 6 was postulated to interact directly with RNA polymerase at the promoter ( 20 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12218014" target="_blank">12218014</a>]<br></li>
				<label class="SIT_collapse_similarity" for="sit_similarity_331">
Together with previous results , we conclude that Lrp bound at downstream sites is necessary but not sufﬁcient for promoter activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12218014" target="_blank">12218014</a>]<br></label>
<input id="sit_similarity_331", type="checkbox">
<div class="SIT_similar">
<li>Together with our previous results , we conclude that Lrp bound at downstream sites is necessary but not sufﬁcient for promoter activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12218014" target="_blank">12218014</a>]<br></li></div>
<li>Another likely exception is gltBDF , in which Lrp bound at sites upstream of 128 stimulates transcription from a downstream promoter ( 15 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12218014" target="_blank">12218014</a>]<br></li>
<li>In this model , Lrp bound at upstream sites was not essential for promoter expression but contributed to more efﬁcient expression , perhaps by interacting with Lrp bound at downstream sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12218014" target="_blank">12218014</a>]<br></li>
<li>In this model , Lrp bound to upstream sites directly stimulates transcription , whereas Lrp bound to downstream sites is necessary to bend the DNA so that the upstream Lrp is positioned properly with respect to the promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12218014" target="_blank">12218014</a>]<br></li>
<li>Trace amounts of other complexes seen in the ﬁgure represent DNAs containing Lrp bound to one , three , four , or ﬁve sites ( 20 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12218014" target="_blank">12218014</a>]<br></li>
<li>Binding of Lrp to ilvIH sites 1 and 2 is highly cooperative ( 28 ) , and therefore a single complex , composed of Lrp bound to both sites 1 and 2 , predominates .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8144448" target="_blank">8144448</a>]<br></li>
<li>The relatively stable complexes formed from Lrp binding to ilvIH DNA are here called complex 1 ( Lrp bound to sites 1 and 2 ) and complex 2 ( Lrp bound to sites 1 , 2 , 3 , 4 , 5 , and 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12218014" target="_blank">12218014</a>]<br></li>
<li>Arrows denote the positions of free DNA , complex 1 ( Lrp bound to sites 1 and 2 ) , and complex 2 ( Lrp bound to sites 1 , 2 , 3 , 4 , 5 , and 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12218014" target="_blank">12218014</a>]<br></li>
<li>The increased cooperativity is most likely the result of an interaction between Lrp bound at upstream and downstream sites , as represented diagrammatically in Fig. 4 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12218014" target="_blank">12218014</a>]<br></li>
<li>Complexes 1A and 1B contain Lrp bound to DNA at one and two upstream sites , respectively , whereas complex 1C contains Lrp bound to those sites as well as to four sites further downstream ( 29 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8432705" target="_blank">8432705</a>]<br></li>
<li>At equilibrium , one expects a mixture of free DNA , DNAs of each type with Lrp bound to upstream sites , and DNAs of each type with Lrp bound at both upstream and downstream sites ( represented in Fig. 3A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12218014" target="_blank">12218014</a>]<br></li>
<li>In contrast , when the sites are separated by 134 bp it becomes possible for the Dam sites to be looped proximally as a consequence of having the DNA looped around the Lrp octamer ( Figure 7B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25785689" target="_blank">25785689</a>]<br></li>
<li>( B ) The same Lrp complex ( from A ) with DNA where the distance between Dam sites is ∼ 134 bp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25785689" target="_blank">25785689</a>]<br></li>
<li>When separated by 284 bp the Dam sites again are unlikely to be imbedded within the Lrp − DNA complex and instead are likely to be positioned proximally along the DNA ﬂanks .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25785689" target="_blank">25785689</a>]<br></li>
<li>Dam 's processivity is largely unaffected by the nonspeciﬁcally bound Lrp roadblock for a substrate with 42 bp between the sites ( Figure 6B , Table 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25785689" target="_blank">25785689</a>]<br></li>
<li>( D ) EcoRI 's processivity on a substrate with 134 bp separating the sites is signiﬁcantly decreased in the presence of Lrp , consistent with EcoRI 's inability to utilize intersegmental hopping .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25785689" target="_blank">25785689</a>]<br></li>
<li>When either 134 or 284 bp separate the sites , the processivity is increased with the presence of Lrp ( Figure 7C ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25785689" target="_blank">25785689</a>]<br></li>
<li>( C ) When the Dam sites are separated by 134 bp and 284 bp , Lrp 's looping increases Dam 's processivity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25785689" target="_blank">25785689</a>]<br></li>
<li>Finally , EcoRI 's processivity with Lrp when the EcoRI sites are separated by 134 bp was explored .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25785689" target="_blank">25785689</a>]<br></li>
<li>Open and closed circles represent sites unoccupied and occupied by Lrp , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8432705" target="_blank">8432705</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8432705" target="_blank">8432705</a>]<br></li>
<li>In contrast to cAMP-CRP , Lrp did not bind to a single region , but protected at multiple sites extending at least between positions ÿ102 and ÿ14 ( Figure 7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9512707" target="_blank">9512707</a>]<br></li>
<li>It is not known in either system how these modi ® ers cause Lrp to bind differentially to these sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10216857" target="_blank">10216857</a>]<br></li>
<li>It may be that half palindromic sites bind Lrp more weakly than do full palindromic sites and that the stronger full palindromic sites were selected for in these in vitro experiments .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7665463" target="_blank">7665463</a>]<br></li>
<li>Empty bars represent a region of extended DNase I protection not seen when either 30 mM L-leucine or glycine are absent Sites 1 and 3 must be in phase with site 2 to be protected from DNase I cleavage To study the nature of the complex formed when Lrp binds to the gltBDF upstream region , a series of site-directed mutations was generated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9236118" target="_blank">9236118</a>]<br></li>
<li>Sites at which Lrp binds upstream of the promoter are represented in the cartoon as solid circles ( dimer assumed to bind to one site ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12218014" target="_blank">12218014</a>]<br></li>
<li>One dimer of Lrp binds to one site ( 160a ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li>
<li>Examination of the region upstream of the PserC ( pdxF ) promoter revealed a potential Lrp box ( CAGTAAAAACATCGG ) at positions 2141 to 2127 relative to the transcription start at 11 ( Fig. 2 ) ( 14 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9171388" target="_blank">9171388</a>]<br></li>
<li>In that intergenic region , the six sites are each centered 31 bp apart , suggesting that the putative Lrp/Lrp contacts may be similar to those present when Lrp binds gltBDF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9236118" target="_blank">9236118</a>]<br></li>
<li>Lrp binds to a long AT-rich sequence from -- 158 to -- 82 from the start of the coding region , i.e. upstream of P1 and overlapping P2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11985712" target="_blank">11985712</a>]<br></li>
<li>Lrp binds to a region upstream of the Irp promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8144448" target="_blank">8144448</a>]<br></li>
<li>Lrp binds specifically to the gltBDF promoter region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7901196" target="_blank">7901196</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7901196" target="_blank">7901196</a>]<br></li>
<li>The work presented here demonstrates that Lrp binds speci ® cally and cooperatively to at least three sites upstream of gltBDF , and that in doing so Lrp bends or wraps the region of DNA ¯ anking these three sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9236118" target="_blank">9236118</a>]<br></li>
<li>The results indicate that Lrp binds cooperatively to three sites quite far upstream , spanning the region from ÿ140 to ÿ260 base-pairs relative to the start of transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9236118" target="_blank">9236118</a>]<br></li>
<li>Lrp strongly and cooperatively binds to multiple sites located between positions ÿ14 and ÿ102 ( in a way that suggests DNA wrapping around multiple Lrp molecules ) and moderately stimulates in vitro transcription , especially with EsS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9512707" target="_blank">9512707</a>]<br></li>
<li>As observed from DNA mobility shift and DNase I footprinting analyses , Lrp binds to one or more sites within the region -80 to -32 relative to the start point of lip transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8144448" target="_blank">8144448</a>]<br></li>
<li>Our results are consistent with the notion that Lrp binds at fim sites 1 and 2 and then interacts with the DNA flanking both sides of this region ( Fig. 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7916011" target="_blank">7916011</a>]<br></li>
<li>Contributing to DNA wrapping is a propensity of Lrp to bend DNA at sites to which it binds speciﬁcally ( 21 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12218014" target="_blank">12218014</a>]<br></li>
<li>Lrp binds to at least six sites upstream of the ilvIH promoter , two `` upstream '' sites located between -253 and -211 and four `` downstream '' promoter-proximal sites located between -141 and -46 ( 20a ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1447135" target="_blank">1447135</a>]<br></li>
<li>Lrp binds to multiple ... , ... , .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9802028" target="_blank">9802028</a>]<br></li>
<li>Gel mobility shift assays and DNase I footprinting showed that Lrp protein binds to multiple sites upstream of the gcv promoter , from about bp -92 to bp -229 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7928983" target="_blank">7928983</a>]<br></li>
<li>Lrp binds to multiple sites upstream of the gcv promoter ( Fig .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9611815" target="_blank">9611815</a>]<br></li>
				<label class="SIT_collapse_similarity" for="sit_similarity_373">
Lrp binds to multiple sites upstream of the gcv promoter , suggesting a direct role for Lrp in gcv expression ( 21 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9537378" target="_blank">9537378</a>]<br></label>
<input id="sit_similarity_373", type="checkbox">
<div class="SIT_similar">
<li>addition , the results show that Lrp binds to multiple sites upstream of the gcv promoter , suggesting a direct role for Lrp in gcv expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7928983" target="_blank">7928983</a>]<br></li></div>
<li>The gel mobility shift assay suggested that Lrp binds to multiple sites in the gcv control region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7928983" target="_blank">7928983</a>]<br></li>
<li>Lrp binds to multiple sites in the dad promoter region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955407" target="_blank">8955407</a>]<br></li>
<li>We have shown that Lrp binds to multiple sites in the dad promoter region and that a site that overlaps or is very close to the start of transcription of each of the major dad transcripts is essential for repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955407" target="_blank">8955407</a>]<br></li>
<li>For ilvIH , one of the operons positively regulated by Lrp , Lrp binds to multiple sites upstream of the transcriptional start site and activates transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7665463" target="_blank">7665463</a>]<br></li>
<li>Results from a DNase I footprint assay showed that Lrp binds to multiple sites in the gcv control region , spanning a region from about -92 to -229 relative to the transcription initiation site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7928983" target="_blank">7928983</a>]<br></li>
<li>The leucine-responsive protein ( Lrp ) , a global regulator involved in the control of transcription of several genes involved in amino acid metabolism ( Calvo & Matthews , 1994 ) , binds to multiple sites in the gcv control region from bp ® 229 to ® 92 relative to the transcriptional start site ( Fig. 1 ) ( Stauffer & Stauffer , 1994 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12101307" target="_blank">12101307</a>]<br></li>
<li>At higher concentrations , Lrp dimers bind cooperatively to four additional sites near the transcription start site , spanning about a 200-bp region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7928983" target="_blank">7928983</a>]<br></li>
				<label class="SIT_collapse_similarity" for="sit_similarity_382">
At low concentrations of Lrp , Lrp dimers bind to two sites more than 200 bp upstream of the transcription start in a single highly cooperative step .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7901196" target="_blank">7901196</a>]<br></label>
<input id="sit_similarity_382", type="checkbox">
<div class="SIT_similar">
<li>At low concentrations , Lrp binds to two sites which are more than 200 bp upstream of the transcription start site in a cooperative step .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7928983" target="_blank">7928983</a>]<br></li></div>
<li>As the Lrp dimer concentration is increased from 0 to 25 nM , the 406-bp fragment , which contains the ilvIH promoter , shifts first into a high-affinity complex containing Lrp dimers bound at two sites located 200 to 250 bp upstream of the start of transcription and then into a lower-affinity complex containin TABLE 2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7901196" target="_blank">7901196</a>]<br></li>
<li>Lrp dimers bound at four additional sites located 50 to 150 bp upstream of the transcription start ( 29 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7901196" target="_blank">7901196</a>]<br></li>
<li>Results Lrp binds to three sites upstream of the gltBDF promoter In previous studies DNA mobility shift assays were used to show that Lrp binds a DNA fragment containing base-pairs ÿ329 to 109 relative to the start of gltBDF transcription and that the af ® nity of Lrp for the DNA is not affected by an additiona 234 bp of upstream sequence ( Ernsting et al. , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9236118" target="_blank">9236118</a>]<br></li>
<li>DNA containing the gltBDF promoter and upstream regions ( bp ÿ335 to 109 relative to the start of transcription ) was isolated from plasmid pBE18 , end-labeled , and digested with DNase I in the presence and absence of Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9236118" target="_blank">9236118</a>]<br></li>
<li>A complicating factor in this analysis is the fact that Lrp is known in some cases to bind cooperatively to multiple sites , for example , to sites upstream of the 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 Y A G H A W A T T W T D C T R This study A g a A T T T T A T t c T 42 T T T A T T C t N a A T 32 a Y 5 C or T , H 5 not G , W 5 A or T , D 5 not C , and R 5 A or G. ilvIH operon of E. coli ( 40 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7665463" target="_blank">7665463</a>]<br></li>
<li>176 , No. 17 In Vivo Footprinting Analysis of Lrp Binding to the ilvIH Promoter Region of Escherichia coli ROSANGELA MARASCO , ' MARIO VARCAMONTI ,2 FRANCO LA CARA ,3 EZIO RICCA , ' MAURILIO DE FELICE , ' AND MARGHERITA SACCO4 * Sezione di Microbiologia , Dipartimento di Fisiologia Generale ed Ambientak , Universita Federico II , 80134 Naples , ' and Istituto di Biochimica delle Proteine ed Enzimologia3 and Istituto Intemazionale di Genetica e Biofisica ,4 Consiglio Nazionale delle Ricerche , 80125 Naples , and Istituto di Scienze dellAlimentazione , Consiglio Nazionale delle Ricerche , 83100 Avellino ,2 Italy Received 22 February 1994/Accepted 15 June 1994 An in vivo footprinting analysis of the ilvIH regulatory region of Escherichia coli showed that the transcription activator Lrp binds to six sites , scattered over 250 bp upstream of the transcriptional start point .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8071194" target="_blank">8071194</a>]<br></li>
<li>11 no 41995 Pages 379-38 ' 7 ------------------------------------------------------ J ~ Ident7fication 01 common motifs in unaligned ~ DNA sequences : Lrp regulon applicatiori to Escherichia coli ~ Yishai M.Eraenke / , Yaé / Mande / , Oevorah Friedberg and Marga/it !	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8521047" target="_blank">8521047</a>]<br></li>
<li>Search for Lrp-binding sequences by Genomic SELEX screening In order to identify the whole set of target promoters , genes and operons under the direct control of Lrp , we performed Genomic SELEX screening ( Shimada et al. , 2005 ) , in which puriﬁed His-tagged Lrp was mixed with a collection of E. coli genome fragments of 200 -- 300 bp in length and Lrp-bound DNA fragments were afﬁnity-isolated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2162465" target="_blank">2162465</a>]<br></li>
<li>As a shortcut approach to identify the whole set of sequences recognized by Lrp , we subjected this isolated SELEX fragment mixture to DNA chip analysis using an E. coli tilling array ( Shimada et al. , 2008 , 2011 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2162465" target="_blank">2162465</a>]<br></li>
<li>At higher Lrp concentrations , regions of protection and regions hypersusceptible to -50 cleavage were visible for sequences from -190 to ( Figure 5A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8508774" target="_blank">8508774</a>]<br></li>
<li>Regions corresponding to possible Lrp recognition sequences , labeled I to VI , are in boldface type and underlined .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19074390" target="_blank">19074390</a>]<br></li>
<li>Ten cycles of the Selex procedure were required to select sequences that bound strongly to Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7665463" target="_blank">7665463</a>]<br></li>
<li>As described in Materials and Methods , the conditions for this Selex experiment were not optimal and 10 cycles were required for selection of sequences that bound Lrp tightly .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7665463" target="_blank">7665463</a>]<br></li>
				</ul></div><br>
			<label class="collapse_TU" for="_6"><font size="5px">Transcription unit regulated by Lrp</font></label>
				<input id="_6" type="checkbox">
				<div class="TU"><br><ul><li>The effects of E. coli lrp ( pEcoLrp ) on gene expression are shown on the x-axis in every panel .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18405378" target="_blank">18405378</a>]<br></li>
<li>In every panel , the x-axis shows the gene expression ratio for E. coli K-12 Δlrp relative to that in the same strain complemented by E. coli lrp ( pEcoLrp ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18405378" target="_blank">18405378</a>]<br></li>
<li>Kutukova , E.A. , Livshits , V.A. , Altman , I.P. , Ptitsyn , L.R. , Zyiatdinov , M.H. , Tokmakova , I.L. , and Zakataeva , N.P. , The yeaS ( leuE ) Gene of Escherichia coli Encodes an Exporter of Leucine , and the Lrp Protein Regulates Its Expression , FEBS Lett .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17025177" target="_blank">17025177</a>]<br></li>
<li>Under these conditions , cAMP-CRP and IHF do not contribute to the specific regulatory pattern of starvation induction ( which is no longer found in a strain that is deficient for as and Lrp but has wild-type cya , crp , himA , and hip alleles ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8253679" target="_blank">8253679</a>]<br></li>
<li>Operons known to be regulated by Lrp include g1nA ( Ernsting et al. , 1992 ) , gltBD ( Ernsting et al. , 1992 ) , ilvIH ( Platko et al. , 1990 ) , livJ and livK ( Haney et al. , 1992 ) , tysU ( Gazeau et al. , 1992 ; Lin et al. , 1992a ) , ompC and ompF ( Ernsting et al. , 1992 ) , oppABCDF ( Austin et al. , 1989 ) , papBA ( Braaten et al. , 1992 ) , sdaA ( Lin et al. , 1990 ) , serA ( Lin et al. , 1990 ; Rex et al. , 1991 ) and tdh ( Lin et al. , 1990 ; Rex et al. , 1991 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8508774" target="_blank">8508774</a>]<br></li>
<li>This is perhaps not surprising given reports that members of the family , such as Lrp , LrpA , LrpC and PutR bind to and induce bending in DNA ( 6,7,11,25,26 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16528101" target="_blank">16528101</a>]<br></li>
<li>Leu was not used alone as it can lead to starvation for Ile , via feedback inhibition of L-threonine deaminase FRe l igguraetio4n of selected target genes by heterologous Lrp proteins Regulation of selected target genes by heterologous Lrp proteins.E .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18405378" target="_blank">18405378</a>]<br></li>
<li>Regulatory patterns III ( anaerobically repressed , not affected by FNR ) and IV ( anaerobically activated , not affected by FNR ) are most easily explained as genes affected by the ArcAB system or as yet unidentified global regulators such as Lrp , IHF , FIS , or H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12754220" target="_blank">12754220</a>]<br></li>
<li>Thirteen of these genes ( adhP , cfa , dps , gadC , gadW , mlrA , poxB , otsB , otsA , yciG , yeaG , yjgB , and yohF ) were also described as Lrp repressed by Tani et al. ( 64 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15716429" target="_blank">15716429</a>]<br></li>
<li>First , Lrp appears to sense the shift between two major E. coli environments , `` the gut and the gutter '' [ 6 ] , activating many amino acid biosynthetic operons ( such as gltBDF ) and repressing many cat-abolic ones .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17233899" target="_blank">17233899</a>]<br></li>
<li>Strikingly , while Lrp negatively regulates its own gene in E. coli , and was shown to do so even more strongly in P. mirabilis , Lrp appears to activate its own gene in V. cholerae .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18405378" target="_blank">18405378</a>]<br></li>
<li>In E. coli , the native Plrp ( fused to lacZ ) was repressed equivalently by E. coli and V. cholerae Lrp , but about twice as much by P. mirabilis Lrp ( Fig. 4B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18405378" target="_blank">18405378</a>]<br></li>
<li>We found that in E. coli gltB is activated ~ 25 fold by Lrp during mid-log , and about half as much in early stationary phase ( Fig. 10E , circles ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18405378" target="_blank">18405378</a>]<br></li>
<li>Bearing in mind that this is the ortholog showing only 43 % identity to E. coli gltB , it is nevertheless the case that while Lrp activates gltB in log-phase E. coli and P. mirabilis , under the same conditions it slightly represses Vc2373 in V. cholerae .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18405378" target="_blank">18405378</a>]<br></li>
<li>Here , we first demonstrate a role for Lrp in the regulation of a gene that is specifically induced in stationary phase , i.e. , under conditions of total starvation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8253679" target="_blank">8253679</a>]<br></li>
<li>Because of the fourfold repression of lysU by increased Irp gene dosage in RM460 ( lrp ÷ ) cells , Lrp repression may not be complete under normal conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7603430" target="_blank">7603430</a>]<br></li>
<li>When leucine , which in the control of some genes acts as a positive or negative co-factor for regulation by Lrp ( Calvo & Matthews , 1994 ; Newman et al. , 1996 ) , was present in the growth medium , stationary phase induction of csiD was slightly stimulated , indicating that in the case of csiD regulation , leucine may act as a weak positive co-regu-lator ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9512707" target="_blank">9512707</a>]<br></li>
<li>It is unlikely that the phenomenon described here is a secondary effect of regulation of Lrp synthesis or stability because : ( i ) transcription of the lrp gene is normal during amino acid downshift ( Fig. 1C ) ; and ( ii ) Lrp is present in the cell during the onset of stationary phase , a condition similar to amino acid deprivation and characterized by the accumulation of ppGpp , and acts as a negative regulator of some genes transcribed by a specific stationary phase sigma factor , us [ 18 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7557316" target="_blank">7557316</a>]<br></li>
<li>Expression from the livJ promoter was very sensitive to the level of Lrp ; it was repressed about 8-fold when Lrp was expressed from a moderate-copy-number plasmid and about 30-fold when Lrp was expressed from a high-copy-number plasmid ( compare data for SH-51 in Fig. 5 with data for SH183 and SH184 in Fig. 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1729203" target="_blank">1729203</a>]<br></li>
<li>Expression from the ilvIH promoter was reduced more than 30-fold in a strain lacking a functional Lrp ( 119 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li>
<li>Expression of an ompF * - lacZ1 translational fusion is increased by Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8002608" target="_blank">8002608</a>]<br></li>
<li>Expression of sduC , but not sdaB , is also induced 2-3-fold by Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8026499" target="_blank">8026499</a>]<br></li>
<li>These observations clearly suggest that the mode of DNA binding of Lrp in both repression and activation of transcription may be similar , although it is not presently known whether all lrp mutations isolated by Platko and Calvo ( 1993 ) also exert dominant negative effects on lysU repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7603430" target="_blank">7603430</a>]<br></li>
<li>Even though lrp transcription is autoregulated , some other regulation must override this in order to result in this decrease in Lrp in LB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1532173" target="_blank">1532173</a>]<br></li>
<li>These results were consistent with the observed properties of the mutant Lrp as well as with the notion that lrp is autoregulated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7603430" target="_blank">7603430</a>]<br></li>
<li>Transcriptional level CpxR-P not only modulates the action of different regulators , such as BaeR , Lrp and OmpR , by binding to the same regulatory sequences ( see above ) , but it also represses the gene transcription of other signaling systems , such as σE and Aer .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16487683" target="_blank">16487683</a>]<br></li>
<li>In this report we show that us-holoenzyme ( E d ) is able to carry out transcription from the aidB promoter ( Pa , & in the absence of any additional transcription factor ; we also show that EDs-dependent transcription of aid8 is repressed by the leucine responsive protein ( Lrp ) both in vitro and in vivo under standard growth conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8809748" target="_blank">8809748</a>]<br></li>
<li>Furthermore , the transcription of stpA was reduced to the same degree by the presence of leucine in the growth medium , indicating that leucine repression is mediated via Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8890170" target="_blank">8890170</a>]<br></li>
<li>Under these nonstandard conditions , ilvIH transcription in the presence of Lrp was reduced twofold by leucine ( Fig. 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1447135" target="_blank">1447135</a>]<br></li>
<li>Taken together , these results strongly suggest that Lrp and alanine or leucine act directly to repress and induce , respectively , transcription of the dad operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955407" target="_blank">8955407</a>]<br></li>
<li>Taken together , these in vitro results show that these amino acids act directly to induce dad expression and strongly suggest that they are not converted to other compounds ( 21 ) in order to alleviate Lrp repression of dad in vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955407" target="_blank">8955407</a>]<br></li>
<li>Lrp acts directly to regulate its own synthesis by binding to a region just upstream of the l1p promoter and repressing transcription from that promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8144448" target="_blank">8144448</a>]<br></li>
<li>Thus , it seems likely that Lrp regulated serC ( pdxF ) - aroA transcription by directly binding to the upstream Lrp box .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9171388" target="_blank">9171388</a>]<br></li>
<li>During growth in LB , repression of serC ( pdxF ) - aroA transcription by CRP-cAMP is maximal and stimulation by Lrp is negligible .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9171388" target="_blank">9171388</a>]<br></li>
<li>The foregoing results strongly supported the hypothesis that Lrp and CRP-cAMP act as positive and negative effectors , respectively , of serC ( pdxF ) - aroA operon transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9171388" target="_blank">9171388</a>]<br></li>
<li>These results led to the hypotheses that CRP-cAMP represses and Lrp activates serC ( pdxF ) - aroA operon transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9171388" target="_blank">9171388</a>]<br></li>
<li>These results strongly support the idea that the Lrp box binds Lrp directly as a positive regulator of serC ( pdxF ) - aroA operon transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9171388" target="_blank">9171388</a>]<br></li>
<li>Lrp is also induced under starvation conditions and plays an enhancing role in transcription of some stress-response genes ( 41 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10368152" target="_blank">10368152</a>]<br></li>
<li>It is also clear that Lrp activates malF transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7751276" target="_blank">7751276</a>]<br></li>
<li>Thus , Lrp activates transcription from fan genes , as it does for pap , daa , and sfa genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li>
<li>Many RpoS-dependent genes are regulated by othe transcription factors ( Table 3 ) ; for example , the promoters of genes ygaU and yhjD are known to be activated by Lrp ( 77 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15576800" target="_blank">15576800</a>]<br></li>
<li>Since ompF translation is regulated by an antisense RNA encoded by the micF gene ( 4 ) and since the micF and ompC genes are divergently transcribed , these observations suggested that Lrp might be regulating the transcription of both micF and ompC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li>
<li>At the same time , it is noteworthy that two of the genes that are most prominently regulated by ArgP , argO and lysP , are both also regulated by the leucine-responsive general transcriptional regulator Lrp ( 37 , 41 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21890697" target="_blank">21890697</a>]<br></li>
<li>Finally , it is noteworthy that many of the genes regulated by ArgP are controlled also by other mechanisms , such as by LysR for lysA , osmolarity for gdhA , the riboswitch for lysC , and Lrp for argO and lysP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21890697" target="_blank">21890697</a>]<br></li>
<li>The lysU gene encodes one of the two lysyl-tRNA synthetases ( the other being lysS ) , reported previously to be regulated by Lrp under aerobic conditions ( 48 ) and induced under anaerobic conditions but not by FIG. 9 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12754220" target="_blank">12754220</a>]<br></li>
<li>These include two genes which were not previously known to be regulated by Lrp , malF and ORF 489 , an as yet unidentiﬁed gene which has also been identiﬁed as both Lrp and methylase regulated by Low and his group ( 11 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7751276" target="_blank">7751276</a>]<br></li>
<li>These newly discovered genes fall into the same functional classes as the genes regulated by the leucine-responsive regulatory protein Lrp under aerobic conditions ( 25 ) and FNR under anaerobic conditions ( 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15699038" target="_blank">15699038</a>]<br></li>
<li>Both artI and potF are regulated by Lrp under aerobic growth conditions ( 11 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12754220" target="_blank">12754220</a>]<br></li>
<li>This gene is also regulated by Lrp under aerobic growth conditions ( 53 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12754220" target="_blank">12754220</a>]<br></li>
<li>Three of the genes of this cluster ( gcvT , lysU , and dadX ) are known to be regulated by Lrp under aerobic growth conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12754220" target="_blank">12754220</a>]<br></li>
<li>However , it is striking that the genes of these patterns fall into the same functional classes as the functional classes of genes regulated by Lrp under aerobic conditions ( 11 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12754220" target="_blank">12754220</a>]<br></li>
<li>Among the genes regulated by Lrp is osmY , a gene encoding a periplasmic protein of unknown function , which is osmotically and stationary phase inducible and ss dependent ( Yim and Villarejo , 1992 ; Lange et al. , 1993 ; Yim et al. , 1994 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9663683" target="_blank">9663683</a>]<br></li>
<li>Other operons regulated by both Lrp and CRP-cAMP include dad ( degradative D-amino acid dehydrogenase [ 37 ] ) , daa ( F1845 ﬁmbrial adhesin [ 2 ] ) , and osmY ( osmotically induced periplasmic protein [ 33 ] ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9171388" target="_blank">9171388</a>]<br></li>
<li>For example , expression of the serC ( pdxF ) - encoded transaminase , which acts between PdxB and PdxA in the de novo PLP pathway ( 40 , 43 ) , is minimal in rich LB medium and maximal in minimal salts-glucose medium due to opposing positive and negative controls exerted by the Lrp and cyclic AMP receptor proteins , respectively ( T.-K .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11844765" target="_blank">11844765</a>]<br></li>
<li>In E. coli , adhE is preferentially expressed in stationary phase [ 107-109 ] , and repressed by Lrp in a leucine-independent manner during exponential growth in minimal glucose medium [ 69 ] and ABK , unpublished data ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18405378" target="_blank">18405378</a>]<br></li>
<li>These putative proteins could enhance repression of ompC and micF transcription by Lrp during growth in glucose minimal MOPS medium at 378C .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8002608" target="_blank">8002608</a>]<br></li>
<li>In a poor medium that is not optimal for growth but nevertheless still allows growth and cell division to proceed at a reduced rate , Lrp stimulates the expression of biosynthetic genes ( e.g. , ilvIH [ 36 ] , serA [ 26 ] , gltD [ 13 ] , and leu [ 34 ] ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8253679" target="_blank">8253679</a>]<br></li>
<li>Although we still know little about the factors that regulate Lrp expression , it is likely that Lrp levels are higher in glucose minimal medium than in media containing nitrogen bases and amino acids .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li>
<li>In the absence of rifampin , there was a marked increase in the rate of product formation , suggestin thetase isozyme that is normally cryptic or expressed at very low levels during growth on glucose minimal medium and that the expression of this isozyme is normally repressed by Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li>
<li>This work demonstrates that Lrp and osmolarity exert their effects independently on ompC and ompF expression during growth in glucose minimal MOPS [ 3 - ( N-morpholino ) propanesulfonic acid ] medium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8002608" target="_blank">8002608</a>]<br></li>
<li>The direction of the bend induced in gltBDF DNA by Lrp can be inferred from our data .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9236118" target="_blank">9236118</a>]<br></li>
<li>The stimulating effect of Lrp is not sigma factor speciﬁc , and it can only be observed when the CRP box is optimally located ( Fig. 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11532138" target="_blank">11532138</a>]<br></li>
<li>The lysU gene of E. coli encodes lysyl-tRNA synthetase ( LysRS ) and is a member of the leucine-Lrp regulon ( reviewed by Nakamura and Ito 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7603430" target="_blank">7603430</a>]<br></li>
<li>FEBS 29827 The yeaS ( leuE ) gene of Escherichia coli encodes an exporter of leucine , and the Lrp protein regulates its expression Ekaterina A. Kutukova , Vitaliy A. Livshits * , Irina P. Altman , Leonid R. Ptitsyn , Michael H. Zyiatdinov , Irina L. Tokmakova , Natalia P. Zakataeva Ajinomoto-Genetika Research Institute , Laboratory N2 , 1-st Dorozhny Proezd , b. 1 , Moscow 117545 , Russia Received 29 April 2005 ; revised 1 July 2005 ; accepted 12 July 2005 Available online 10 August 2005 Edited by Felix Wieland Abstract Overexpression of the yeaS gene encoding a protein belonging to the RhtB transporter family conferred upon cells resistance to glycyl-L-leucine , leucine analogues , several amino acids and their analogues .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16098526" target="_blank">16098526</a>]<br></li>
<li>Kutukova , E.A. , Livshits , V.A. , Altman , I.A. , Ptitsyn , L.R. , Zyiatdinov , M.H. , Tokmakova , I.L. , and Zakataeva , N.P. ( 2005 ) The yeaS ( leuE ) gene of Escherichia coli encodes an exporter of leucine , and the Lrp protein regulates its expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19906180" target="_blank">19906180</a>]<br></li>
				<label class="TU_collapse_similarity" for="tu_similarity_66">
The yeaS ( leuE ) gene of Escherichia coli encodes an exporter of leucine , and the Lrp protein regulates its expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17504942" target="_blank">17504942</a>]<br></label>
<input id="tu_similarity_66", type="checkbox">
<div class="TU_similar">
<li>The yeaS ( leuE ) gene of Esche-richia coli encodes an exporter of leucine , and the Lrp protein regulates its expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25845847" target="_blank">25845847</a>]<br></li></div>
<li>176 , No. 7 Regulation of the Escherichia coli lrp Gene QING WANG , ' t JINGCAI WU ,2 DEVORAH FRIEDBERG ,3 JILL PLAKTO ,1 t AND JOSEPH M. CALVO2 * Sections of Genetics and Development ' and Biochemistry , Molecular and Cell Biology ,2 Comell University , Ithaca , New York 14853 , and Department ofApplied Microbiology , The Hebrew University-Hadassah Medical School , Jerusalem , Israel 910103 Received 18 October 1993/Accepted 15 January 1994 Lrp ( leucine-responsive regulatory protein ) is a major Escherichia coli regulatory protein which regulates expression of a number of operons , some negatively and some positively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8144448" target="_blank">8144448</a>]<br></li>
<li>Lrp ( leucine-responsive regulatory protein ) is a major Esch-erichia coli regulatory protein which regulates expression of a number of operons , some negatively and some positively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8144448" target="_blank">8144448</a>]<br></li>
<li>The leucine-responsive regulatory protein ( Lrp ) is a regulatory factor that controls , either positively or negatively , the expression of many genes , including numerous operons involved in amino acid metabolism ( reviewed in references 3 and 21 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955407" target="_blank">8955407</a>]<br></li>
<li>The RpoS and Lrp regulatory protein are not involved in hns expression A number of stationary-phase-induced genes of E. coli are regulated by RpoS ( KatF ) , a putative alternative sigma factor ( Lange and Hengge-Aronis , 1991 ; Siegele and Kolter , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8355613" target="_blank">8355613</a>]<br></li>
<li>The expression of most of the genes encoding these proteins is not induced by the substrates they transport but is controlled by the global regulation systems , such as the Lrp protein , and activated by the signal compounds involved in the intracellular communication .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17025177" target="_blank">17025177</a>]<br></li>
<li>Recently , it has been observed that Lrp , the leucine-respon-sive protein ( 32 ) , also affects the transcription of malT as well as of some malT-dependent genes ( 268 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9529892" target="_blank">9529892</a>]<br></li>
<li>The leucine-responsive regulatory protein ( Lrp ) has been shown to both positively and negatively regulate transcription of a number of genes in response to exogenous leucine ( 215 ) ; Lrp action is signiﬁcantly activated by the absence of L-leucine in the growth medium , whereas it is repressed in the presence of L-leucine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16760308" target="_blank">16760308</a>]<br></li>
<li>The mechanism by which the leucine-respon-sive regulatory protein ( Lrp ) regulates the genes under its control is not completely understood .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11395454" target="_blank">11395454</a>]<br></li>
<li>The sdaA gene is expressed in glucose-minimal medium and in LB , repressed by the leucine-responsive regulatory protein ( Lrp ) [ 21 ] , and induced by the amino acids , glycine and leucine [ 18,32 ] , which partially reverse Lrp repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12706508" target="_blank">12706508</a>]<br></li>
<li>The dadAX operon is expressed by multiple promoters that are repressed by leucine-responsive regulatory protein ( Lrp ) and activated by cyclic AMP-CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10216857" target="_blank">10216857</a>]<br></li>
<li>We also show that the serC ( pdxF ) - aroA operon is in fact repressed by CRP-cAMP and activated by the leucine-responsive protein ( Lrp ) at the transcriptional level .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9171388" target="_blank">9171388</a>]<br></li>
<li>First , gltBDF is one of the operons for which Lrp regulation is relatively insensitive to the coregulator leucine [ 5,28 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17233899" target="_blank">17233899</a>]<br></li>
<li>Thus , relative to more responsive operons , gltBDF is a leucine-independent member of the Lrp regulon ( 12 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11395454" target="_blank">11395454</a>]<br></li>
<li>The gltBDF operon is positively regulated by the leucine-responsive regulatory protein ( Lrp ) ( 12 , 13 ) and is transcribed at very low levels in the absence of Lrp ( 12 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11395454" target="_blank">11395454</a>]<br></li>
<li>The gltBDF operon , encoding glutamate synthase , was among the group of genes and proteins positively regulated by Lrp but insensitive to leucine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7901196" target="_blank">7901196</a>]<br></li>
<li>One operon controlled by Lrp that is relatively insensitive to leucine is gltBDF , which includes genes specifying the large ( GltB ) and small ( GltD ) subunits of glutamate synthase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9236118" target="_blank">9236118</a>]<br></li>
<li>Our interest in gltBDF , the operon that includes structural genes for GltBD , stems from its membership in the regulon controlled by Lrp ( leucine-responsive regulatory protein ) [ 5 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17233899" target="_blank">17233899</a>]<br></li>
<li>For example , H-NS is known to interfere with ss and Lrp in transcription of the osmC gene ( 8 ) and with FNR and CAP in the regulation of cai and ﬁx operons ( 11 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10601207" target="_blank">10601207</a>]<br></li>
<li>Some of these operons ( e.g. , ilvIH ) are known to be directly controlled by Lrp , whereas others ( e.g. , ginA ) are affected only indirectly by Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8144448" target="_blank">8144448</a>]<br></li>
<li>Further study of these Lrp mutants should prove useful in understanding the regulation of the ilvIH operon and other operons controlled by Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8432705" target="_blank">8432705</a>]<br></li>
<li>This pattern of regulation is unusual for operons that are controlled by Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1729203" target="_blank">1729203</a>]<br></li>
<li>For ilvIH , one of the operons positively regulated by Lrp , Lrp binds to multiple sites upstream of the transcriptional start site and activates transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7665463" target="_blank">7665463</a>]<br></li>
<li>Some Lrp-regulated operons ( e.g. , ilvIH and gltBD ) have been shown to be directly regulated by Lrp , with Lrp binding upstream of these genes to bring about changes in transcription ( 12 , 58 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8002608" target="_blank">8002608</a>]<br></li>
<li>Like many of the operons regulated by Lrp , ilvIH is also regulated by leucine ( 3 , 11 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1447135" target="_blank">1447135</a>]<br></li>
<li>Each of these operons is positively regulated by Lrp , although as described below , the effects on glnALG are indirect .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li>
<li>Since ArgP was previously shown not to exert autoregulation ( Nandineni and Gowrishankar , 2004 ; Caldara et al. , 2006 ) , in contrast to many other LysR-family members , we may conclude that argP expression is regulated neither by Lrp or ArgR , nor by ArgP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19906180" target="_blank">19906180</a>]<br></li>
<li>E. coli BL21 ( DE3 ) pLysS harboring plasmid pET16lrp or pET21argP ( Table 1 ) was grown to exponential phase at 30 ° C , and expression of genes encoding N-terminally His-tagged Lrp ( His6-Lrp ) or C-ter-minally His-tagged ArgP ( ArgP-His6 ) was induced with 0.5 mM isopropyl - - D-thiogalactopyranoside ( IPTG ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21441513" target="_blank">21441513</a>]<br></li>
<li>The ilvIH operon , one of the best-studied members of the Lrp regulon , encodes an acetohydroxy acid synthase that is involved in the biosynthesis of branched-chain amino acids ( 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12218014" target="_blank">12218014</a>]<br></li>
<li>OPERONS REGULATED BY Lrp Amino Acid Biosynthesis Five operons involved in amino acid biosynthesis are regulated by Lrp : the glnALG operon , which includes the coding sequence for glutamine synthetase ; the gltBDF operon , which encodes glutamate synthase ; the ilvIH operon , which encodes acetohydroxyacid synthase III ( an isozyme involved in branched-chain amino acid biosynthesis ) ; the leuABCD operon , which encodes enzymes that catalyze the biosynthesis of leucine ; and serA , which encodes phosphoglycerate dehydrogenase ( the enzyme catalyzing the first step in serine biosynthesis ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li>
<li>Two operons involved in high-affinity transport of branched-chain amino acids , the livJ and livKHMGF operons , are negatively regulated by Lrp , and that negative regulation is potentiated by leucine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li>
<li>The following work demonstrates that expression of the gItBDF operon is regulated by Lrp at the level of transcription and that Lrp plays a role in the repression of glutamate synthase expression in a rich medium in which nitrogen is present in the form of amino acids and nucleic acid bases as well as ammonia .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7901196" target="_blank">7901196</a>]<br></li>
<li>The ability of alanine or leucine to strongly antagonize Lrp repression of dad is consistent with the positive role of these amino acids in the expression of numerous operons that are negatively regulated by Lrp ( 3 , 21 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955407" target="_blank">8955407</a>]<br></li>
<li>Stimulation of brnFE expression encoding a multi-ami-no-acid exporter , by methionine and the branched-chain amino acids in conjunction with an Lrp-like transcription factor ( 45 , 46 ) in C. glutamicum and by Lrp and leucine of the inducible LeuE leucine exporter in E. coli ( 9 ) , is another example , besides those of lysE and argO ( described above ) , where the amino acid substrate exerts control over the expression of its exporter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25845847" target="_blank">25845847</a>]<br></li>
<li>Thus , although the high-affinity branched-chain amino acid transport proteins are negatively regulated by Lrp obic conditions also served to elevate expression from the opp operon promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li>
<li>Lrp of Corynebacterium glutamicum controls expression of the brnFE operon encoding the export system for L-methionine and branched-chain amino acids .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25845847" target="_blank">25845847</a>]<br></li>
<li>Lrp is also known to repress the expression of the livJ and livK genes , encoding the branched-chain amino acid transporter ( 19 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17463081" target="_blank">17463081</a>]<br></li>
<li>Also , Lrp regulates the expression of the leucine biosynthetic operon , leuABCD ( 7 ) , and the expression of the branched-chain amino acid transport genes livJ and livKHMG ( 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8900118" target="_blank">8900118</a>]<br></li>
<li>If yojI is indeed regulated by Lrp , it is to be expected that cloning of yojI devoid of its regulatory sequences and under the control of another promoter should lead to Lrp-independent expression of the gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19074390" target="_blank">19074390</a>]<br></li>
<li>These results indicated that the lrp gene is not only feedback regulated by Lrp but is also controlled by H-NS protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7603430" target="_blank">7603430</a>]<br></li>
<li>However , the expression of lrp is not regulated solely by Lrp but is also under the control of H-NS protein , as revealed in this study .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7603430" target="_blank">7603430</a>]<br></li>
<li>In contrast to Lrp , AsnC has only been shown to exert speciﬁc control of its own gene and that of asnA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16528101" target="_blank">16528101</a>]<br></li>
<li>Thus , it was found that the control of the osmY gene was dependent also on Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7715442" target="_blank">7715442</a>]<br></li>
<li>Genes repressed by Lrp are shown in bold type .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8900118" target="_blank">8900118</a>]<br></li>
<li>Genes activated by Lrp are underlined .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8900118" target="_blank">8900118</a>]<br></li>
<li>Lrp directly represses its own promoter in E. coli , and this occurs whether or not leucine is present [ 89 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18405378" target="_blank">18405378</a>]<br></li>
<li>This means that livJ is still the only gene which requires both Lrp and leucine to repress it .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7751276" target="_blank">7751276</a>]<br></li>
<li>This confirms that induction of CP41 and CP52 insert-gene expression by L-leucine is mediated by Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8026499" target="_blank">8026499</a>]<br></li>
<li>Expression of the L-leucine-biosynthesis gene ( s ) is clearly activated by Lrp and is also controlled independently by L-leucine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1532173" target="_blank">1532173</a>]<br></li>
<li>The livK gene is repressed by Lrp when exogenous leucine is present [ 70,96 ] , and activated when leucine is not in the medium [ 66 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18405378" target="_blank">18405378</a>]<br></li>
<li>They also participate in diverse roles such as regulation of DNA replication and transcription .51 These proteins include H-NS , HU , Lrp , and IHF which are distributed uniformly throughout the nucleoid of E. coli and are termed group ÿI proteins .52 We discuss here nucleoid proteins that in ¯ uence micF expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11601842" target="_blank">11601842</a>]<br></li>
<li>The roles of additional regulatory proteins ( Lrp , H-NS ) in the control of csiD expression It seems that with the elucidation of the crucial role of cAMP-CRP and a perhaps modulatory role of negative DNA supercoiling , our understanding of the regulation of csiD is not yet complete .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9512707" target="_blank">9512707</a>]<br></li>
<li>These results demonstrate that Lrp represses the expression of the ilv : : lacZ transcriptional fusion construct described above and that this repression is dependent on a site-specific interaction between Lrp and its consensus-like site within the ilvGMEDA leader region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8900118" target="_blank">8900118</a>]<br></li>
<li>A yeiL -- lacZ transcriptional fusion showed that aerobic yeiL expression increases at least sixfold during stationary phase , requires RpoS , and is positively autoregulated by YeiL , positively activated by Lrp ( and IHF in the absence of FNR ) and negatively regulated by FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11101674" target="_blank">11101674</a>]<br></li>
<li>10 600 ( a ) â-Galactosidase â-Galactosidase 400 1 wt yeiL rpoS 200 0 · 1 ihfA Studies with monolysogens lacking YeiL or various transcription factors indicated that in the stationary phase aerobic yeiL expression is positively autoregulated by YeiL , and signiﬁcantly dependent on RpoS ( σS ) and Lrp but not IHF ( Fig. 6b ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11101674" target="_blank">11101674</a>]<br></li>
<li>These are Lrp , integration host factor ( IHF ) , and the cAMP receptor protein-cAMP complex ( cAMP-CRP ) , all of which negatively modulate osmY expression in response to growth phase and medium composition .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8253679" target="_blank">8253679</a>]<br></li>
<li>For example , the stationary phase expression of cgsA ( encoding the curlin subunit gene ) is modulated by both a ® and HN-S ( Olsen etai , 1993 ) , and osm / ( encoding a periplasmic protein ) is activated by a ^ and repressed by Lrp , cAMP-CRP and IHF in stationary phase cells ( Lange et ai , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7984106" target="_blank">7984106</a>]<br></li>
<li>However , dad expression is directly repressed by Lrp and directly activated CRP-cAMP ( 37 ) , daa expression is activated by both Lrp and CRP-cAMP ( 2 ) , and osmY expression is repressed by both Lrp and CRP-cAMP ( 33 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9171388" target="_blank">9171388</a>]<br></li>
<li>Together , the best explanation for the present data is that leuO expression is induced indirectly via the increase of cellular Lrp level in response to ppGpp level increase caused by the slow growth rate when cells are entering stationary phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11006083" target="_blank">11006083</a>]<br></li>
<li>V. cholerae gave the most divergent expression pattern : gltB mRNA levels were halved by Lrp in log phase , but increased about fivefold by Lrp in early stationary phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18405378" target="_blank">18405378</a>]<br></li>
<li>The level of osmY-lacZ expression was repressed by Lrp both during exponential growth and after entrance into stationary phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li>
<li>Thus , in the rpoS background , repression of osmY-lacZ expression by Lrp appears to be relieved during entry into stationary phase , although in a wild-type background Lrp clearly continues to repress osmY-lacZ expression in stationary phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li>
<li>Thus , we concluded that Lrp participates in transcription regulation of at least eight aminoacyl-tRNA synthetase genes , of which expression of ﬁve aminoacyl-tRNA synthe-tase , including AlaRS , LysRS , SerRS , ThrRS and TyrRS , is repressed by Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2162465" target="_blank">2162465</a>]<br></li>
<li>For operons controlled by Lrp , does Lrp act directly to activate or repress the transcription of target operons , or does it act indirectly by affecting the expression of some other regulatory gene which , in turn , affects the expression of a target operon ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1447135" target="_blank">1447135</a>]<br></li>
<li>While it is possible that Rho may directly or indirectly inﬂuence Lrp expression at some level , the inhibition of Rho function by bicyclomycin treatment does not alter lrp gene expression , at least at the level of transcription ( our unpublished data ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16321930" target="_blank">16321930</a>]<br></li>
<li>One such example is the expression of virulence gene transcription factor AphA in Vibrio cholerae by Lrp and VpsR which both activate aphA expression ( Lin et al. , 2007 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19906180" target="_blank">19906180</a>]<br></li>
<li>In the absence of Lrp , transcription of P2 is activated by cAMP/Crp , a regulator of serB expression as well ( Man et al. , 1997 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11985712" target="_blank">11985712</a>]<br></li>
<li>Although Lrp alters the expression of numerous ope-rons , precisely how Lrp acts to repress or activate transcription is not known ( Newman and Lin , 1995 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10216857" target="_blank">10216857</a>]<br></li>
<li>Last , we tested leucine , because Lrp is known to activate the expression of serA and other genes involved in one-carbon metabolism ( 4 , 35 , 52 ) , and leucine addition reduces serA transcription ( 35 , 52 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9171388" target="_blank">9171388</a>]<br></li>
<li>It is focused mainly on Lrp and its interaction with DNA , on the organization and regulation of the lrp operon , and on the way in which Lrp regulates expression of several operons .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li>
<li>Although much has been learned recently about expression of operons regulated by Lrp , less is known about the regulation of the lrp operon itself .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8144448" target="_blank">8144448</a>]<br></li>
<li>Although much has been learned recently about expression of operons regulated by Lrp and some of their regulatory DNA elements , less is known about structure-function relationships of Lrp protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7603430" target="_blank">7603430</a>]<br></li>
<li>This mutation ( lrp-1 ) causes high constitutive expression of iluIH , one of the operons controlled by Lrp ( Platko , J. V. , Willins , D. A. , and Calvo , J. M. ( 1990 ) J. Bacteriol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2040596" target="_blank">2040596</a>]<br></li>
<li>This technique revealed that malT , malEFG , and malB-lamB-malK are all activated two-to ﬁvefold by Lrp and conﬁrmed that Lrp regulates expression of the leuDBCA and livHJKG operons .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7751276" target="_blank">7751276</a>]<br></li>
<li>In general , Lrp appears to stimulate expression of operons that function in biosynthetic pathways and to repress expression of those that function in catabolic pathways .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8636076" target="_blank">8636076</a>]<br></li>
<li>Lrp is known to have a negative role in the expression of numerous catabolic operons ( Calvo and Matthews , 1994 ) , and its role as a repressor of dad is consistent with this function .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10216857" target="_blank">10216857</a>]<br></li>
<li>( Lrp ) controls the expression of both the Pap and Fan pili operons in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8144448" target="_blank">8144448</a>]<br></li>
<li>An analysis of operons in E. coli whose expression is affected by Lrp suggested that Lrp is a global regulatory protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11575919" target="_blank">11575919</a>]<br></li>
<li>The expression of some target operons is activated by Lrp , while the expression of others is repressed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8900118" target="_blank">8900118</a>]<br></li>
<li>example Crp ( Mallick and Herrlich , 1979 ) , Lrp activates expression of some operons and represses expression of others .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8483418" target="_blank">8483418</a>]<br></li>
				<label class="TU_collapse_similarity" for="tu_similarity_147">
Lrp activates expression of some operons and represses expression of others .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7665463" target="_blank">7665463</a>]<br></label>
<input id="tu_similarity_147", type="checkbox">
<div class="TU_similar">
<li>Lrp activates the expression of some operons and represses the expression of others .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8508774" target="_blank">8508774</a>]<br></li></div>
<li>Lrp is known to regulate the expression of at least four other operons in E. coli : ilvIH ( 31 ) , oppA ( 40a ) , sdaA , and serA ( 43 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1917830" target="_blank">1917830</a>]<br></li>
<li>It is thought that Lrp acts differently on these operons in that it represses expression of sdaA , oppA , and tdh and induces expression of ilvIH and serA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1917830" target="_blank">1917830</a>]<br></li>
<li>Crp and Lrp both activate serA expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11985712" target="_blank">11985712</a>]<br></li>
<li>Rex and coworkers have demonstrated that Lrp regulates the expression of the tdh and serA operons ( 33a ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1346534" target="_blank">1346534</a>]<br></li>
<li>Lrp , through its control of glutamate synthase , induces the Ntr response , which stimulates expression of polyamine catabolic enzymes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12730324" target="_blank">12730324</a>]<br></li>
<li>Lrp , by regulating the expression and activity of glutamate synthase , can override induction of nitrogen-regulated operons by ammonia limitation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li>
<li>Interaction between Lrp and Nitrogen Metabolism Deficiencies in glutamate synthase expression are associated with the inability to induce nitrogen-regulated genes in response to nitrogen limitation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li>
<li>When levels of Lrp are low , glutamate synthase expression will also be low , and the levels of nitrogen-regulated genes will remain low even under nitrogen-limiting conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li>
<li>In fact , of the 31 remaining genes of this expression group regulated by FNR under anaerobic growth conditions , 10 are regulated by Lrp under aerobic conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12754220" target="_blank">12754220</a>]<br></li>
<li>In this expression pattern , only five genes are known to be regulated by Lrp ( lamB , malM , yabI , pheA , and hupB ) ; however , 17 genes of this cluster are of unclassified function .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12754220" target="_blank">12754220</a>]<br></li>
<li>Furthermore , stpA expression is temperature regulated , activated during exponential growth , regulated by Lrp , and induced by osmotic stress ( 16 , 37 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16980475" target="_blank">16980475</a>]<br></li>
<li>These data suggest that sdaB expression is regulated by Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1532173" target="_blank">1532173</a>]<br></li>
<li>Peeters et al. have recently demonstrated that argP expression was not regulated by Lrp ( 35 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21441513" target="_blank">21441513</a>]<br></li>
<li>The list of regulated proteins shown in Table 3 does not include all proteins whose expression is known to be regulated by Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1346534" target="_blank">1346534</a>]<br></li>
<li>RESULTS The expression of gltBDF : : lacZ operon fusions is regulated by Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7901196" target="_blank">7901196</a>]<br></li>
<li>First , the lrp gene , which encodes a global regulator Lrp , was selected as a target ; this was because Lrp positively regulates the expression of genes involved in L-valine biosynthesis , such as ilvIH .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17463081" target="_blank">17463081</a>]<br></li>
<li>We have used this method to con-ﬁrm that Lrp regulates the expression of and leuDBCA livJK and to demonstrate for the ﬁrst time that expression of the malF gene , and other genes of the maltose operon , is regulated by Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7751276" target="_blank">7751276</a>]<br></li>
<li>Lrp regulates the expression of many genes , either positively or negatively ( Calvo and Matthews , 1994 ) , but it is unusual for Lrp to be both an activator and a repressor in the same operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10216857" target="_blank">10216857</a>]<br></li>
<li>DISCUSSION The identification of genes and proteins whose expression is regulated by Lrp , but not apparently in response to the presence or absence of leucine in the growth medium , was th most surprising result to emerge from an analysis of the Lrp regulon ( 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7901196" target="_blank">7901196</a>]<br></li>
<li>Lrp controls expression of so many genes that it is possible that one of the many effects of Lrp deﬁciency suppresses the SGL − phenotype of the torA strain and does not alter the nuoM phenotype .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12706508" target="_blank">12706508</a>]<br></li>
<li>Of the genes whose expression was significantly changed by E. coli Lrp , over a third were not significantly affected by either of the other Lrp orthologs .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18405378" target="_blank">18405378</a>]<br></li>
<li>Also , we were able to show that the expression of the ygaZH genes is activated by Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17463081" target="_blank">17463081</a>]<br></li>
<li>Lrp has subsequently been shown to regulate the expression of other genes and operons .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1346534" target="_blank">1346534</a>]<br></li>
<li>Study of the glycine cleavage ( gcv ) genes shows that Lrp can also have drastic effects on expression of genes that are not affected by L-leucine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1532173" target="_blank">1532173</a>]<br></li>
<li>The expression of many genes and operons within the Lrp regulon has been found to be relatively insensitive to leucine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9236118" target="_blank">9236118</a>]<br></li>
<li>For these leucine-sensitive genes and operons , Lrp can be a positive or a negative regulator of expression , and the effect of Lrp is antagonized in the presence of exogenous leucine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7901196" target="_blank">7901196</a>]<br></li>
<li>The increased expression of OmpC in strain SM3001 that has been noted both by us and by Matsuyama and Mizushima ( 31 ) may indicate that Lrp no longer binds to the ompC promoter in this strain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8002608" target="_blank">8002608</a>]<br></li>
<li>The specific activity of , - galactosidase was about twofold higher in strain CV1070 than in strain CV1069 , indicating that Lrp represses its own expression ( Table 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8144448" target="_blank">8144448</a>]<br></li>
<li>The requirements for YeiL , RpoS and Lrp for optimal yeiL expression were retained in the absence of FNR ( Fig. 6b ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11101674" target="_blank">11101674</a>]<br></li>
<li>The exact mechanism by which Lrp controls gene expression is not well understood .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12218014" target="_blank">12218014</a>]<br></li>
<li>The positive effect on OmpF expression exerted by Lrp is probably due to the negative effect of Lrp on the expression of micF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8002608" target="_blank">8002608</a>]<br></li>
<li>MarA seems also to compete functionally with GadX , GadW , RpoS , and Lrp under some conditions , which might ensure that hdeAB expression is not repressed more than needed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18083817" target="_blank">18083817</a>]<br></li>
<li>Variations in Lrp might also be important for regulation of gene expression , but this would not be the only role that it plays .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7751276" target="_blank">7751276</a>]<br></li>
<li>It should be noted that the micF deletion also abolishes the repression of OmpC expression exerted by Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8002608" target="_blank">8002608</a>]<br></li>
<li>For example , expression of ldcC , coding for the constitutive lysine decarboxylase of E. coli , is downregulated by Lrp ( 53 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21441513" target="_blank">21441513</a>]<br></li>
<li>There is no effect of Lrp on recA expression , at least in E. coli and V. cholerae under our conditions [ 69 ] and N. Dolganov , pers .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18405378" target="_blank">18405378</a>]<br></li>
<li>It was shown that Lrp and ArgP behave as competitive activators able to activate argO expression under different conditions ( 35 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21441513" target="_blank">21441513</a>]<br></li>
<li>These results indicate that ArgP is a major regulator of lysP expression but that Lrp modulates lysP transcription under lysine-limiting conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21441513" target="_blank">21441513</a>]<br></li>
<li>As demonstrated here , lysP expression is induced under lysine limitation and requires ArgP and Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21441513" target="_blank">21441513</a>]<br></li>
<li>Additionally , by using the Lrp , which upregulates the expression of the ilvIH gene ( encoding AHAS III ) , further amplification of AHAS was possible without any detrimental effect of plasmid-born overexpression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17463081" target="_blank">17463081</a>]<br></li>
<li>Since Lrp was able to bind to ilvIH DNA in vitro but in vivo expression was reduced , these strains were presumed to have mutations that affect the ability of Lrp to activate transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8432705" target="_blank">8432705</a>]<br></li>
<li>( iii ) Lrp blocked dad expression in a puriﬁed in vitro transcription system , and this repression was markedly reduced by alanine or leucine , compounds that induce dad in vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955407" target="_blank">8955407</a>]<br></li>
<li>A simple model that accounts for these results is that Lrp activates transcription from the ilvIH promoter and leucine reduces expression by interfering with the action of Lrp ( 20 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8432705" target="_blank">8432705</a>]<br></li>
<li>This conclusion was strengthened by results which showed that the positive action of Lrp on ilvIH expression in vivo could be duplicated in a puriﬁed in vitro transcription system ( 35 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955407" target="_blank">8955407</a>]<br></li>
<li>We used transcriptional fusions of the lacZ reporte gene to the gltBDF operon to confirm the eff - cts of Lrp and leucine on the expression of this operon anc1 to determine whether the effect of Lrp was exerted at the leviel of transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7901196" target="_blank">7901196</a>]<br></li>
<li>The regulation of the gcv operon by Lrp has not been extensively studied , although it has been noted that the expression of a gcv-lacZ operon fusion was almost insensitive to leucine ( 84 ) ( Table 3 ) but highly sensitive to the presence or absence of Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li>
<li>The coregulator for Lrp is leucine , and exogenous leucine reduces the expression of gltBDF about twofold , although this level is still substantially above baseline transcription in the absence of Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11395454" target="_blank">11395454</a>]<br></li>
<li>The regulation of gltBDF expression by Lrp is not absolutely independent of leucine but is better described as insensitive to leucine , relative to the regulation of ilvIH transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7901196" target="_blank">7901196</a>]<br></li>
<li>In two cases , leucine has little or no effect ; Lrp is absolutely required for transcription of gcv and represses its own expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1532173" target="_blank">1532173</a>]<br></li>
<li>For livJ and livKHMGF , Lrp represses expression and leucine is required for this repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1729203" target="_blank">1729203</a>]<br></li>
<li>Furthera as more , that repression was independent of the presence of Lrp caused by the of leucine in LB , and was not presence excess because leucine had little effect on lrp expression in a minimal medium ( 10 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8144448" target="_blank">8144448</a>]<br></li>
<li>Lrp affects expression of sdaB in glucose-minimal medium with L-leucine ( Table 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1532173" target="_blank">1532173</a>]<br></li>
<li>Taken together , it was assumed that the L-leucine requirement in our system may cause insufficient activation of ilvIH expression by Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17463081" target="_blank">17463081</a>]<br></li>
<li>Note that CV1008 is white on plates containing X-Gal because it does not make the Lrp required for ilvIH expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1729203" target="_blank">1729203</a>]<br></li>
<li>Considering the important role of LysP in regulating the expression of cadA ( 55 ) , which encodes the inducible lysine decarboxylase , we analyzed the effect of Lrp on transcription of the cadBA operon , which encodes the induc-ible lysine decarboxylase CadA and the lysine/cadaverine antiporter CadB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21441513" target="_blank">21441513</a>]<br></li>
<li>In vitro , Lrp binds specifically and with relatively high affinity to six sites upstream of the ilvIH promoter , and mutations in five of these sites reduce expression of the operon in vivo ( 28 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8432705" target="_blank">8432705</a>]<br></li>
<li>For wild-type strains of E. coli , leucine repressed in vivo expression of the ilvIH operon 5 - to 10-fold ( 145 ) and reduced binding of Lrp to sites upstream of the ilvIH promoter in vitro ( 128 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li>
<li>Lrp binds to at least two sites within the ilvZH promoter/regulatory region , at positions -40 to -100 and -190 to -260 , relative to the startpointof transcription ( 4 ) and mutationswithin these binding sites reduce expression of the iluIH operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2040596" target="_blank">2040596</a>]<br></li>
<li>These results indicate that the presence of the two most upstream Lrp binding sites , centered at positions 246 and 221 , is crucial for Lrp-activated ilvIH promoter expression and that the spacing and/or phasing of these sites relative to the remainder of the operon is important .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12218014" target="_blank">12218014</a>]<br></li>
<li>Regulation of gcvB expression GcvA-mediated regulation of the gcvTHP operon requires the GcvR and Lrp proteins and is responsive to the addition of glycine and purines to the growth medium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10972807" target="_blank">10972807</a>]<br></li>
<li>As with both the ompC-lacZ1 and ompF-lacZ1 transcriptional fusions , the absence of functional Lrp protein did not signiﬁcantly affect regulation of micF-lacZ1 expression by sucrose .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8002608" target="_blank">8002608</a>]<br></li>
<li>The expression of a plasmid-borne micF-lacZ1 operon fusion is inhibited by Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8002608" target="_blank">8002608</a>]<br></li>
				<label class="TU_collapse_similarity" for="tu_similarity_211">
Lrp decreases expression of b-galactosidase from an ompC-lacZ1 operon ( transcriptional ) fusion .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8002608" target="_blank">8002608</a>]<br></label>
<input id="tu_similarity_211", type="checkbox">
<div class="TU_similar">
<li>Lrp decreases expression of b-galactosidase from a plasmid-borne micF-lacZ1 operon ( transcriptional ) fusion .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8002608" target="_blank">8002608</a>]<br></li></div>
<li>Lrp is thought to be evolutionarily related to AsnC , a regulatory protein that can positively stimulate the expression of one operon and negatively affect its own promoter ( 47 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1917830" target="_blank">1917830</a>]<br></li>
<li>Landgraf , J.R. , Boxer , J.A. , and Calvo , J.M. ( 1999 ) Escheri-chia coli Lrp ( Leucine-responsive regulatory protein ) does not directly regulate expression of the leu operon promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19906180" target="_blank">19906180</a>]<br></li>
<li>Furthermore , it has been known that exogenous L-leucine antagonizes the positive effect of Lrp on the expression of the ilvIH operon and certain Lrp-regulated genes ( 26 , 27 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17463081" target="_blank">17463081</a>]<br></li>
<li>The global regulator leucine responsive protein ( Lrp ) is involved in the regulation of L-valine biosynthesis ; it activates the expression of the ilvIH genes encoding the AHAS III isoenzyme ( 16 , 17 ) and represses the expression of the ilvGMEDA operon encoding the AHAS II isoenzyme ( 18 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17463081" target="_blank">17463081</a>]<br></li>
<li>The leucine-responsive regulatory protein Lrp was known to control the expression of a large number of genes , including the ilvIH operon , but not ilvBN in E. coli ( Jafri et al. , 2002 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16164569" target="_blank">16164569</a>]<br></li>
<li>Furthermore , because the expression of the ilvGM operon is repressed by Lrp ( 18 ) , the native promoter of the ilvGM operon , to which Lrp binds , was replaced with the tac promoter to abolish the repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17463081" target="_blank">17463081</a>]<br></li>
<li>theseresults indicate that Lrp stimulatestranscription of ilvZH by binding to thepromoter/regulatory region and that leucine represses expression of the ilvZH operon by preventing the binding of Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2040596" target="_blank">2040596</a>]<br></li>
<li>In a second class of operons , Lrp acts negatively and leucine induces operon expression , presumably by interfering with Lrp action .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1729203" target="_blank">1729203</a>]<br></li>
<li>On the other hand , Lrp represses expression from the sdaA promoter and leucine induces expression of this operon , presumably by interfering with the repressive action of Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li>
<li>On the other hand , expression of the artPIQM operon , one of the systems responsible for arginine uptake , is downregulated by ArgR ( 5 ) and Lrp ( 21 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21441513" target="_blank">21441513</a>]<br></li>
<li>L Regulation of the sdaCB operon by Lrp and Crp At least three global regulators may affect sduC expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8026499" target="_blank">8026499</a>]<br></li>
<li>This may be the case for regulation of pap , an operon whose expression is affected by Lrp but not by leucine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li>
<li>This operon 's expression is dependent on Lrp as an activator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7748928" target="_blank">7748928</a>]<br></li>
<li>Lrp modulates expression of the cadBA operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21441513" target="_blank">21441513</a>]<br></li>
<li>The depressed expression in the pDIA510 strain implies that excess copies of the hns gene reduce the ability of Lrp to activate the ilvIH operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8152423" target="_blank">8152423</a>]<br></li>
<li>The magnitude of the effect is reasonable , since we are presumably observing an indirect effect of H-NS on the expression of the ilvIH operon and we do not know what other conditions might affect the synthesis of Lrp and H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8152423" target="_blank">8152423</a>]<br></li>
<li>Strains containing Irp-5 ,6 have some functional Lrp-5 ,6 because the ilvIH operon , which requires Lrp for expression , was expressed in such cells , albeit at a reduced level ( Fig. 2A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1729203" target="_blank">1729203</a>]<br></li>
<li>et al. ( 31 ) have shown that Lrp binds to the ilvIH promoter and activates the expression of the ilvIH operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1917830" target="_blank">1917830</a>]<br></li>
<li>It has been known that Lrp activates the expression of the ilvIH operon , which encodes the AHAS III isoenzyme ( 16 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17463081" target="_blank">17463081</a>]<br></li>
<li>lrp mutations that prevent Lrp from binding ilvIH DNA in vitro prevent ilvIH operon expression in vivo ( 12a ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1447135" target="_blank">1447135</a>]<br></li>
<li>For at least the ilvIH operon , Lrp regulates gene expression directly by binding to speciﬁc DNA sequences ( 42 , 44 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7665463" target="_blank">7665463</a>]<br></li>
<li>Overexpression of Lrp from a high-copy-number plasmid repressed expression from the tdh promoters of both SP1234 ( Xpdrl ) and SP1300 ( Apdrl ) under all growth conditions tested .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1917830" target="_blank">1917830</a>]<br></li>
<li>Plrp in E. coli is autogenously repressed [ 89 ] , and all three lrp promoters in all three hosts ( with one exception ) show lower expression in the presence of Lrp than in its absence ( Fig. 11 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18405378" target="_blank">18405378</a>]<br></li>
<li>Regulation of orthologous target genes in their native backgrounds We next tested an explicit assumption of regulatory extrapolation , by determining if the expression of orthologs of E. coli Lrp target genes are Lrp-responsive in igruartieon9 of lrp mRNA levels with growth phase and medium Variation of lrp mRNA levels with growth phase and medium.A .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18405378" target="_blank">18405378</a>]<br></li>
<li>Deletion of the Lrp box in strain TX3733 reduced fusion expression by half compared to TX3187 ( Table 3 , row 1 ) , and fusion expression in TX3733 was not further reduced in an lrp mutant ( Table 3 , rows 1 and 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9171388" target="_blank">9171388</a>]<br></li>
<li>We found that maximum operon expression in bacteria growing in MMGlu depended on the presence of the upstream Lrp box and that decreased operon expression caused by an lrp mutation was lost when the Lrp box was deleted ( Results and Table 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9171388" target="_blank">9171388</a>]<br></li>
<li>Excess Lrp , conferred by plasmid pCV180 , repressed reporter gene expression about 10-fold for the construct having the lrp region - 157 to +223 but only about 1.4-fold for the construct having the region -40 to +223 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8144448" target="_blank">8144448</a>]<br></li>
<li>These results suggest that the lrp gene product is needed for expression of these genes and that L-leucine modulates expression of the activated gene , probably by interacting with Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1532173" target="_blank">1532173</a>]<br></li>
<li>These results suggest that Lrp , the product of lrp , negatively regulates livJ and livK expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1729203" target="_blank">1729203</a>]<br></li>
<li>In LB-grown cells , lrp expression was very much decreased , suggesting that autoregulation by Lrp must be overridden by other controls .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1532173" target="_blank">1532173</a>]<br></li>
<li>Therefore , it remains to be investigated if Lrp and H-NS bind the same sites on DNA and control lrp expression co-operatively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7603430" target="_blank">7603430</a>]<br></li>
<li>Lrp autorepression does not affect stationary-phase-in-duced expression from the lrp promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955316" target="_blank">8955316</a>]<br></li>
<li>This sta-tionary-phase induction of lrp expression is independent of Lrp autogenous regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955316" target="_blank">8955316</a>]<br></li>
<li>To evaluate the capacity of Lrp to bind to the PcadBA pro-grown in glucose minimal medium , and after 2.5 h 0.2 % arabinose ( wt/vol ) was added to induce the expression of the lrp gene cloned in pBAD24 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21441513" target="_blank">21441513</a>]<br></li>
<li>Transcription from lrp is decreased by Lrp and by growth in rich medium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1532173" target="_blank">1532173</a>]<br></li>
<li>Lrp is known to repress its own synthesis ( 19 , 36 ) , and it was previously shown that the low lrp expression level seen in rich medium is independent of Lrp ( 19 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955316" target="_blank">8955316</a>]<br></li>
<li>To understand the reason for this positive effect of coexpressing the lrp and ygaZH genes , the regulation of ygaZH expression by Lrp was investigated by real-time RT-PCR analysis by using the W3110 and Wlrp strains in response to 5 M - valine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17463081" target="_blank">17463081</a>]<br></li>
<li>Despite the ability of Lrp to bind DNA semispecifically [ 76 ] , expression of the great majority of genes is unaffected by deletion of lrp ( see Fig. 1A in [ 69 ] and Fig. 3 in [ 68 ] ) , so its generalized effects as a histonelike protein are limited .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18405378" target="_blank">18405378</a>]<br></li>
<li>Regulation of P-galactosidase synthesis in strains CP41 and CP52 by Lrp and Crp Strains CP41 and CPS2 were isolated by screening for genes whose expression is affected by L-leucine ( Lin et al. , 1992 ) ; expression of the insert-encoded lacZ is indeed induced about 2-3-fold by exogenous L-leucine ( Table 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8026499" target="_blank">8026499</a>]<br></li>
				<label class="TU_collapse_similarity" for="tu_similarity_252">
Similarly , for operons whose expression is repressed by Lrp , the same three subcategories have been recognized ; leucine negates the effect , leucine is required for the effect , or leucine has no effect ( 3 , 11 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1447135" target="_blank">1447135</a>]<br></label>
<input id="tu_similarity_252", type="checkbox">
<div class="TU_similar">
<li>Similarly , for those operons whose expression is repressed by Lrp , the same three subcategories have been recognized : leucine neg-ates the effect , leucine is required for the effect , and leucine has no effect .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8483418" target="_blank">8483418</a>]<br></li>
<li>Similarly , for those operons whose expression is repressed by Lrp , the same three subcategories have been recognized ; leucine abolishes the effect , leucine is required for the effect , and leucine has no effect .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8432705" target="_blank">8432705</a>]<br></li>
<li>Similarly , for those operons whose expression is repressed by Lrp , the same three subcategories have been recognized ; leucine overcomes the effect , leucine is required for the effect , or leucine has no effect .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7665463" target="_blank">7665463</a>]<br></li></div>
<li>PapI , a regulatory protein encoded by the pap operon , did not bind to pap DNA by itself but affected both the pattern of bands observed in gel mobility experiments and the DNase I footprint pattern that was induced by Lrp ( 107 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li>
<li>Both Lrp and ArgR are also involved in transcriptional regulation of the gltBDF operon for glutamate synthase ( GOGAT ) , which plays an important role in the synthesis of glutamate , a precursor of arginine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19906180" target="_blank">19906180</a>]<br></li>
<li>By using operon fusions to malT , malE , and malK in the genetic background of strain MC4100 , this Lrp dependence could not be observed in the laboratory of W.B. .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9529892" target="_blank">9529892</a>]<br></li>
<li>In this report , we show that Lrp does , in fact , bind to a high affinity , consensus-like site located between the attenuator and the first structural gene of the ilvGMEDA operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8900118" target="_blank">8900118</a>]<br></li>
<li>Previously , we reported that the serC ( pdxF ) gene is positively regulated by the Lrp protein and negatively regulated by the cyclic AMP receptor protein ( 43 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11844765" target="_blank">11844765</a>]<br></li>
<li>gltBD is positively regulated by Lrp , and the levels of GltBD are reduced about 40-fold in lrp strains ( 43 , 44 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li>
<li>It is evolutionarily related asnC , a gene that positively to asnA Lrp regulates the operon ( 6 , 12 , 30 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8432705" target="_blank">8432705</a>]<br></li>
<li>A TnlO insertion within the lrp gene that abolishes Lrp activity reduces transcription of the iluZH operon more than 30-fold ( 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2040596" target="_blank">2040596</a>]<br></li>
<li>However , unlike pap transcription , ecpD-htrE operon transcription is not regulated by Lrp ( the product of the lrp [ = mbf ] gene ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8102362" target="_blank">8102362</a>]<br></li>
<li>Since transcription of the ilvZH operon is positively regulated by the Lrp protein [ 12 ] , we investigated the possibility that the down-regulation of iluZH transcription during amino acid downshift is directly related to inhibition of transcription of the lrp gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7557316" target="_blank">7557316</a>]<br></li>
<li>The net effect of these interactions is that , in intact cells , serA transcription was regulated about twofold by Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12123449" target="_blank">12123449</a>]<br></li>
<li>serA transcription is positively regulated about ﬁvefold by Lrp ( 35 , 45 , 52 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9171388" target="_blank">9171388</a>]<br></li>
<li>Although the gltBDF operon of E. coli is positively regulated by Lrp ( 13 ) , in vitro transcription with puriﬁed Lrp produced minimal activation of transcription from the gltBDF promoter ( B. R. Ernsting , unpublished results ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11395454" target="_blank">11395454</a>]<br></li>
<li>While transcription of sduC is regulated by Lrp , it is curious that sduB , in the same operon , is not regulated noticeably by Lrp ( Shao and Newman , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8026499" target="_blank">8026499</a>]<br></li>
<li>In a recent review ( 31 ) , unpublished data by Ambartsoumian and Newman was cited , indicating that the membrane-bound pyridine nucleotide transhydrogenase encoded by the pntAB operon was positively regulated by Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li>
<li>The gltBDF operon has been shown to be positively regulated by Lrp in studies using a reporter gene inserted in gltB and in a study of the Lrp regulon by two-dimensional gel analysis ( 9 , 10 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7751276" target="_blank">7751276</a>]<br></li>
<li>It will be interesting to determine whether positive regulation of the gcv operon by Lrp requires a functional GcvA protein and whether gcvA is regulated by Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li>
<li>The gcv operon is negatively regulated by PurR ( 169 ) and positively regulated by Lrp ( 84 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li>
<li>Whereas the sdaA and tdh operons are negatively regulated by Lrp , the gcv operon is positively regulated by Lrp , like the amino acid biosynthetic operons described above .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li>
<li>In agreement with these ﬁndings , we now present evidence that transcriptional ( operon ) fusions of lacZ1 to ompC and micF are negatively regulated by Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8002608" target="_blank">8002608</a>]<br></li>
<li>Expression of the lrp operon is autogenously regulated by Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8144448" target="_blank">8144448</a>]<br></li>
<li>Expression from an ompF-lacZ1 operon fusion is not positively regulated by Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8002608" target="_blank">8002608</a>]<br></li>
<li>Strain '' Plasmid 0.3 M NaC1 lacZ specific activityb lrp allele pDIA510 pDIA510 None None pDIA510 pDIA510 O11 O11 O25 O25 O26 O26 DM DM DM DM DM DM 90 30 4 2 6 2 lrp + lrp + ¬ + - + ¬ + lrp-35 : : Tnl0 lrp-35 : : Tnl0 lrp-35 : : TnlO lrp-35 : : TnlO Discussion The ilvlH operon is a member of the leucine regulon because its transcription is repressed by leucine ( Squires et al. 1981 ) and Lrp is the mediator of this leucine effect ( Platko et al. 1990 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8152423" target="_blank">8152423</a>]<br></li>
<li>The artPIQM operon is , besides being downregulated by Lrp ( Hung et al. , 2002 ) , also repressed by ArgR ( Caldara et al. , 2007 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19906180" target="_blank">19906180</a>]<br></li>
<li>Transcription of the ilvIH operon is positively regulated by Lrp , the activator function of which is abolished by leucine ( Platko et al. 1990 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8152423" target="_blank">8152423</a>]<br></li>
<li>Expression of the ilvIH operon is also repressed in cells growing in a medium containing leucine , through the leucine-responsive regulatory protein Lrp ( Platko et al. , 1990 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16164569" target="_blank">16164569</a>]<br></li>
<li>In vivo , Lrp represses transcription through the leader-attenu-ator region of the ilvGMEDA operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8900118" target="_blank">8900118</a>]<br></li>
<li>Lrp ( formerly called IHB ) binds ilvIH DNA in this region and activates transcription of the ilvIH operon ( 13 , 15 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1447135" target="_blank">1447135</a>]<br></li>
<li>Exogenous leucine reduces transcription of the ilvIH operon 5 - to 10-fold in vivo , an effect that is mediated by Lrp ( 6 , 13 , 15 , 19 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1447135" target="_blank">1447135</a>]<br></li>
<li>In the case of the ilvIH operon , it has also been shown that the binding of Lrp induces further bending to the DNA ( Wang and Calvo , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8890170" target="_blank">8890170</a>]<br></li>
<li>Also we do not know the other factors , if any , that might control the ilvIH operon , apart from Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8152423" target="_blank">8152423</a>]<br></li>
<li>Expression of livJ ' - lacZ or livK ' - lacZ operon fusions in strains overexpressing Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1729203" target="_blank">1729203</a>]<br></li>
<li>A role for Lrp in the induction by leucine of the tdh operon has been proposed ( 43 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1917830" target="_blank">1917830</a>]<br></li>
<li>Expression of the kbl/tdh operon is induced by the presence of leucine , an effect mediated through Lrp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7748928" target="_blank">7748928</a>]<br></li>
<li>The clp ( CS31A ) operon is negatively controlled by Lrp , ClpB , and l-alanine at the transcriptional level .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2162465" target="_blank">2162465</a>]<br></li>
<li>It has been demonstrated that Lrp does not alter intracellular levels of CadC , the transcriptional activator of the cadBA operon ( 42 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21441513" target="_blank">21441513</a>]<br></li>
				</div></ul></div><br>
	</div>
</body>
</html>