<!doctype html>
<style>
@charset "utf-8";
body {
	height: 100%;
	margin: 0;
	padding: 0;
}

.banner {
	position: fixed;
	top: 0;
	width: 100%;
	height: 65px;
	background: gray;
	line-height: 65px;
}
.spacer {
	height: 10px;
}

.main-content {
	padding: 5%;
}

.collapse_ACT {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-style: none solid none none;
	border-color: #B4045F;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_ACT + input{
	display: none;
}
.collapse_ACT + input + div{
	display: none;
}
.collapse_ACT + input:checked + div{
	display: block;
}
.ACT {
	text-align: justify;
	text-justify: inter-word;
}
.ACT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B4045F;
}
.ACT_collapse_similarity:hover{
	background: #e5d3dc;
}
.ACT_collapse_similarity + input {
	display: none;
}
.ACT_collapse_similarity + input + div{
	display: none;
}

.ACT_collapse_similarity + input:checked + div{
	display: block;
}
.ACT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.ACT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B4045F;
	padding-left: 5px;
}
.ACT_collapse_similarity:hover{
	background: #e5d3dc;
}
.ACT_collapse_similarity + input {
	display: none;
}
.ACT_collapse_similarity + input + div{
	display: none;
}
.ACT_collapse_similarity + input:checked + div{
	display: block;
}
.ACT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.ACT_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #B4045F;
}
.ACT_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_DOM {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-style: none solid none none;
	border-color: #B404AE;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_DOM + input{
	display: none;
}
.collapse_DOM + input + div{
	display: none;
}
.collapse_DOM + input:checked + div{
	display: block;
}
.DOM {
	text-align: justify;
	text-justify: inter-word;
}
.DOM_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B404AE;
	padding-left: 5px;
}
.DOM_collapse_similarity:hover{
	background: #e8d7e7;
}
.DOM_collapse_similarity + input {
	display: none;
}
.DOM_collapse_similarity + input + div{
	display: none;
}
.DOM_collapse_similarity + input:checked + div{
	display: block;
}
.DOM_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}.DOM_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #B404AE;
}
.DOM_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_EVO {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #7401DF;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_EVO + input{
	display: none;
}
.collapse_EVO + input + div{
	display: none;
}
.collapse_EVO + input:checked + div{
	display: block;
}
.EVO {
	text-align: justify;
	text-justify: inter-word;
}
.EVO_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #7401DF;
	padding-left: 5px;
}
.EVO_collapse_similarity:hover{
	background: #e2d7ed;
}
.EVO_collapse_similarity + input {
	display: none;
}
.EVO_collapse_similarity + input + div{
	display: none;
}
.EVO_collapse_similarity + input:checked + div{
	display: block;
}
.EVO_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.EVO_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #7401DF;
}
.EVO_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_RP {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #0174DF;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_RP + input{
	display: none;
}
.collapse_RP + input + div{
	display: none;
}
.collapse_RP + input:checked + div{
	display: block;
}
.RP {
	text-align: justify;
	text-justify: inter-word;
}
.RP_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #0174DF;
	padding-left: 5px;
}
.RP_collapse_similarity:hover{
	background: #c9d7e2;
}
.RP_collapse_similarity + input {
	display: none;
}
.RP_collapse_similarity + input + div{
	display: none;
}
.RP_collapse_similarity + input:checked + div{
	display: block;
}
.RP_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.RP_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #0174DF;
}
.RP_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_SIT {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #0489B1;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_SIT + input{
	display: none;
}
.collapse_SIT + input + div{
	display: none;
}
.collapse_SIT + input:checked + div{
	display: block;
}
.SIT {
	text-align: justify;
	text-justify: inter-word;
}
.SIT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #0489B1;
	padding-left: 5px;
}
.SIT_collapse_similarity:hover{
	background: #d0dfe2;
}
.SIT_collapse_similarity + input {
	display: none;
}
.SIT_collapse_similarity + input + div{
	display: none;
}
.SIT_collapse_similarity + input:checked + div{
	display: block;
}
.SIT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.SIT_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #0489B1;
}
.SIT_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_TU {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #04B486;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_TU + input{
	display: none;
}
.collapse_TU + input + div{
	display: none;
}
.collapse_TU + input:checked + div{
	display: block;
}
.TU {
	text-align: justify;
	text-justify: inter-word;
}
.TU_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #04B486;
	padding-left: 5px;
}
.TU_collapse_similarity:hover{
	background: #d0e2de;
}
.TU_collapse_similarity + input {
	display: none;
}
.TU_collapse_similarity + input + div{
	display: none;
}
.TU_collapse_similarity + input:checked + div{
	display: block;
}
.TU_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.TU_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #04B486;
}
.TU_similar > a {
	text-decoration: none;
	color: gray;
}

ul { 
	list-style-type: none;
	padding: 0;
	margin: 0;
}
ul li > a {
	text-decoration: none;
	color: gray;
}
ul li:hover, ul li:focus {
	background: #F0F0F0;
}

ul li:first-child span {
  border-left:none;  
}

</style>
<html>
<head>
<meta charset="utf-8">
<title>TF properties - MarA</title>
<meta name="theme-color" content="#ffffff">
</head>
<body>
	<div class="banner">
		<center><font color="white" size="16px" face="arial">MarA</font></center>
	</div>
	<div align="center" class="main-content">
		<div class="spacer">
	</div><br>			<label class="collapse_ACT" for="_1"><font size="5px">MarA transcriptional activity</font></label>
				<input id="_1" type="checkbox">
				<div class="ACT"><br><ul><li>sigma scores boxes evidence 1 glkP1 - P2 k n glk FruR C [ 36 ] , in this work GCGTTGTTGTTATGCCCCCAGGTATTTACAGTGTGAGAA 236 s38 2 s70 : 3.92 glkP3 - P4 n n s70 3 in this work A , C CCCCCAGGTATTTACAGTGTGAGAAAGAATTATTTTGAC 221 s38 : 2.95 pgiP1 - P2 k n [ 37 ] , in this work A , B , E ¬ pgi ACCATCACATTTTTCTGTGACTGGCGCTACAATCTTCCA 236 s70 : 6.9 ; s38 : 7.44 pfkAP1 - P2 k n ¬ A , B FruR pfkA [ 39 ] , in this work ATCAATTCAGCAGGAAGTGATTGTTATACTATTTGCACA 278 s70 : 8.61 ; s38 : 5.7 pfkAP3 - P4 n n s38 2 in this work A , E TCACTTCGATGTGCAAGAAGACTTCCGGCAACAGATTTC 229 s70 : 1.12 ki CTGCCCTGCGGGGCGGCCATCTTCCTTTATTCGCTTATA 219 ¬ tpiA tpiAP [ 40 ] ¬ ¬ tpiAP1 - P2 in this work n n ¬ A , E CAAAGCCTTTGTGCCGATGAATCTCTATACTGTTTCACA 262 s70 : 8.07 ; s38 : 7.4 gapA gapAP1 - P1a [ 41,43 ] , in k n this work gapAP2 [ 41,43 ] k ¬ A , B , D CGCTTGACGCTGCGTAAGGTTTTTGTAATTTTACAGGCA 236 s70 : 3.76 ; s38 : 0.82 ¬ A CCTTTAAAATTCGGGGCGCCGACCCCATGTGGTCTCAAG 2153 ¬ k TCACATTTTTATCGTAATTGCCCTTTAAAATTCGGGGCG 2174 - - ¬ gapAP3 [ 41,43 ] Crp k AACACCAACTGGCAAAATTTTGTCCTAAACTTGATCTCG 2245 - - ¬ gapAP4 [ 41,43 ] ACATTCCTTTTATTCCACGTTTCGCTTATCCTAGCTGAA 2132 no predictions s703 , s382 A , D epdP1 - P2 k FruR , Crp epd [ 45 ] , in this work pgkP1 k A pgk [ 45 ] AAGCAGCACAAGGTGCATTTCATGGTATAGTTGACTATA 2235 - ¬ n TGTTGCTTTCAGGTAAGACGCAAGCAGCGTCTGCAAAAC 226 no predictions ¬ pgkP2 in this work D , E n GCACACCTGATCAAAACGTTGGTCTGGTGCGATAACGAA 2113 no predictions s703 pgkP3 in this work A , D gpmA gpmAP1 - P2 [ 46 ] , in this ki n work gpmAP3 - P4 in this work n n Fur A ¬ AAGCATTGCTGTTGCTTCGTCGCGGCAATATAATGAGAA 237 s70 :7.56 ; s38 : 3.66 ¬ ¬ CCTTACACTGCGCCACTATTTTCGCTATGGTTATGCGTA 275 s70 :5.80 ; s38 : 7.48 eno enoP1 [ 47 ] k ¬ ¬ AATCGTGTCAGCGTCAACATCAAACTGATCGATTCACAA 2731 ¬ k CGAAGCACTGAAACACGGTGGGCTGAAGAATCGTGTCAG 2760 - - ¬ enoP2 [ 47 ] k TAAACGATTCAGCTTAAACTGCCCGGAAGCGAATCTGTC 2903 - - ¬ enoP3 [ 47 ] n GCGTACCCTGGGTACGCGTTGTTTGTCTGGAGTTTCAGT 227 no predictions s703 , s382 A , D , E enoP4 - P5 in this work n GAGTTCCAGAAACGTCAGGCGAAGTAAGTAAAAAAGTTA 277 no predictions - D enoP6 in this work n n GGGGATGATCAGTTGGTCGAGATCATCGAAGTTCCGAAT 2212 no predictions s703 , s382 D enoP7 - P8 in this work pykF pykFPPki [ 49 ] FruR TGTTTTCCTTTTGGATTAATTTCAGCGTATAATGCGCGC 2231 - -- pykFP1 - P2 in this work n n GGATTCGCTTTCCGGCAGTGCGCCCAGAAAGCAAGTTTC 234 no predictions s703 , s382 A , B , D n GATGTCACCTATCCTTAGAGCGAGGCACCACCACTTTCG 280 no predictions - D pykFP3 in this work pykFP4 - P5 n n in this work CCTCTGCACGCTTTTTCGATGTCACCTATCCTTAGAGCG 297 s70 : 4.79 ; s38 : 9.95 ¬ B , D pykA pykAP1 - P2 n n s38 2 in this work AAGTGACGATCGCTAAAAACGACTGTCACTGTCCTAATC 2105 s70 : 0.90 , D B n n GTCAAAGAAGCGCTGAAGGAATCGCGTTTTGATAAGCAG 2153 s70 : 1.76 s382 A , B , D pykAP3 - P4 in this work Crp , Fnr , GTATGGACATAAGGTGAATACTTTGTTACTTTAGCGTCA 258 no predictions s703 , s382 B , D PdhR s38 2 pdhR pdhRP1 - P2 k n [ 51 -- 52 ] , in this work aceE aceEP1 - P2 k n ¬ AACCTGTCTTATTGAGCTTTCCGGCGAGAGTTCAATGGG 248 s70 : 1.90 [ 50 ] , in this work poxB poxBP1 - P2 k n [ 57,58 ] , in his work B , C ¬ MarA , SoxS TCCCTTCCCCCTCCGTCAGATGAACTAAACTTGTTACCG 227 s70 : 5.31 ; s38 : 9.5 acs acsP1 k [ 60 -- 61 ] ¬ Fis , Crp , Ihf TCTTTAATCAATTGTAAGTGCATGTAAAATACCACTTTA 2224 ¬ k n CCCCTACATTTAACGCTTATGCCACATATTATTAACATC 220 s70 : 4.03 ; this work s38 : 4.97 B , C ¬ acsP2 - P3 [ 60 -- 61 ] , in Fis , Crp , Ihf consensus sigma 70 : TGGTATAATG sigma 38 : CTACACTT sequences Proposed promoters in which putative s and s overlapping recognition sequences have been determined , are presented as bold sequences .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19838295" target="_blank">19838295</a>]<br></li>
<li>The Escherichia coli Transcriptional Regulator MarA Directly Represses Transcription of purA and hdeA * Received for publication , December 11 , 2003 , and in revised form , December 22 , 2003 Published , JBC Papers in Press , December 29 , 2003 , DOI 10.1074 / jbc.M313602200 Thamarai Schneiders ‡ , Teresa M. Barbosa ‡ § , Laura M. McMurry ‡ , and Stuart B. Levy ‡ ¶ From the ‡ Center for Adaptation Genetics and Drug Resistance and the Departments of Molecular Biology and Microbiology and of ¶ Medicine , Tufts University School of Medicine , Boston , Massachusetts 02111 The Escherichia coli MarA protein mediates a response to multiple environmental stresses through the activation or repression in vivo of a large number of chromosomal genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14701822" target="_blank">14701822</a>]<br></li>
<li>Printed in the U.S.A. Transcriptional Repression of the rob MarA-mediated Promoter * Received for publication , November 9 , 2005 , and in revised form , February 8 , 2006 Published , JBC Papers in Press , February 14 , 2006 , DOI 10.1074 / jbc.M512097200 Thamarai Schneiders ‡ and Stuart B. Levy ‡ § 1 From the ‡ Center for Adaptation Genetics and Drug Resistance and the Departments of Molecular Biology and Microbiology and of § Medicine , Tufts University School of Medicine , Boston , Massachusetts 02111 The Escherichia coli transcriptional-regulator MarA affects functions that include antibiotic resistance , persistence , and survival .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16478729" target="_blank">16478729</a>]<br></li>
<li>175 , No. 10 JOURNAL OF BACTERIOLOGY , May 1993 , p. / 93/102888 -07 $ 02.00 / 0 Copyright © 1993 , American Society for Microbiology Overexpression of the MarA Positive Regulator Is Sufficient To Confer Multiple Antibiotic Resistance in Escherichia coli LAURA GAMBINO , STEPHEN J. GRACHECK , AND PAUL F. MILLER * Infectious Diseases , Experimental Therapy Department , Parke-Davis Pharmaceutical Research , Division of Warner-Lambert Company , Ann Arbor , Michigan 48106-1047 Received 14 December 1992/Accepted 8 March 1993 A genetic approach was undertaken to identify normal bacterial genes whose products function to limit the effective concentration of antibiotics .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8491710" target="_blank">8491710</a>]<br></li>
<li>JOURNAL OF BACTERIOLOGY , Dec. 1995 , p. 7100 -- / 95 / $ 04.0010 Copyright q 1995 , American Society for Microbiology Puriﬁcation and Regulatory Properties of MarA Protein , a Transcriptional Activator of Escherichia coli Multiple Antibiotic and Superoxide Resistance Promoters KAM-WING JAIR ,1 ROBERT G. MARTIN ,2 JUDAH L. ROSNER ,2 NOBUYUKI FUJITA ,3 AKIRA ISHIHAMA ,3 AND RICHARD E. WOLF , JR. 1 * Department of Biological Sciences , University of Maryland Baltimore County , Baltimore , Maryland 212281 ; Laboratory of Molecular Biology , National Institute of Diabetes and Digestive and Kidney Diseases , Bethesda , Maryland 208922 ; and Department of Molecular Genetics , National Institute of Genetics , Mishima , Shizuoka-ken 411 , Japan3 Received 8 September 1995/Accepted 17 October 1995 Expression of the marA or soxS genes is induced by exposure of Escherichia coli to salicylate or superoxides , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8522515" target="_blank">8522515</a>]<br></li>
<li>JOURNAL OF BACTERIOLOGY , Apr. 1995 , p. 1655 -- / 95 / $ 04.0010 Copyright q 1995 , American Society for Microbiology Activation of Multiple Antibiotic Resistance and Binding of Stress-Inducible Promoters by Escherichia coli Rob Protein RAFAEL R. ARIZA , † ZIYI LI , NIELS RINGSTAD , AND BRUCE DEMPLE * Department of Molecular and Cellular Toxicology , Harvard School of Public Health , Boston , Massachusetts 02115 Received 10 November 1994/Accepted 18 January 1995 Multiple antibiotic resistance in Escherichia coli can be mediated by induction of the SoxS or MarA protein , triggered by oxygen radicals ( in the soxRS regulon ) or certain antibiotics ( in the marRAB regulon ) , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7896685" target="_blank">7896685</a>]<br></li>
<li>JOURNAL OF BACTERIOLOGY , Apr. 1996 , p. 2216 -- / 96 / $ 04.0010 Copyright q 1996 , American Society for Microbiology Autoactivation of the marRAB Multiple Antibiotic Resistance Operon by the MarA Transcriptional Activator in Escherichia coli ROBERT G. MARTIN ,1 * KAM-WING JAIR ,2 RICHARD E. WOLF , JR. ,2 AND JUDAH L. ROSNER1 Laboratory of Molecular Biology , National Institute of Diabetes and Digestive and Kidney Diseases , Bethesda , Maryland 20892-0560 ,1 and Department of Biological Sciences , University of Maryland Baltimore County , Baltimore , Maryland 212282 Received 18 October 1995/Accepted 4 February 1996 Transcriptional activation of the promoters of the mar/soxRS regulons by the sequence-related but independently inducible MarA and SoxS proteins renders Escherichia coli resistant to a broad spectrum of antibiotics and superoxide generators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8636021" target="_blank">8636021</a>]<br></li>
<li>JOURNAL OF BACTERIOLOGY , May 1996 , p. 2507 -- / 96 / $ 04.0010 Copyright q 1996 , American Society for Microbiology Transcriptional Activation of Promoters of the Superoxide and Multiple Antibiotic Resistance Regulons by Rob , a Binding Protein of the Escherichia coli Origin of Chromosomal Replication KAM-WING JAIR ,1 XIN YU ,1 KIRSTEN SKARSTAD ,2 BEAT THÖNY ,3 NOBUYUKI FUJITA ,4 AKIRA ISHIHAMA ,4 AND RICHARD E. WOLF , JR. 1 * Department of Biological Sciences , University of Maryland Baltimore County , Baltimore , Maryland 212281 ; Department of Biophysics , Institute for Cancer Research , The Norwegian Radium Hospital , 0310 Oslo 3 , Norway2 ; Department of Pediatrics , Division of Clinical Chemistry , University of Zürich , CH-8032 Zürich , Switzerland3 ; and Department of Molecular Genetics , National Institute of Genetics , Mishima , Shizuoka-ken 411 , Japan4 Received 14 November 1995/Accepted 25 February 1996 The Rob protein , isolated on the basis of its ability to-bind to the right arm of the Escherichia coli origin of chromosomal replication , is about 50 % identical in amino acid sequence to SoxS and MarA , the direct regulators of the superoxide ( soxRS ) and multiple antibiotic resistance ( mar ) regulons , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626315" target="_blank">8626315</a>]<br></li>
<li>Transcription of hdeAB depends on complex circuits of regulation involving RpoD ( 70 ) , RpoS ( 38 ) , H-NS , cyclic AMP receptor protein , GadE , GadX , GadW , GadY , EvgAS , YdeO , Lrp , MarA , SoxRS , TorRS , and TrmE , among other regulators ( see Fig. 1 and references in the legend ; see Fig. 2 for the DNA binding sites of some of these regulators in the hdeAB promoter ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18083817" target="_blank">18083817</a>]<br></li>
<li>c Results in italics ( P 0.05 ) or underlined ( P 0.01 ) indicate that MarA produced a statistically signiﬁcant change in the activation or repression produced by a regulator under the same conditions ( e.g. , the repression produced by H-NS on hdeAB in stationary phase at pH 5.5 was signiﬁcantly stronger in the presence of MarA than in the absence of MarA ) FIG. 3 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18083817" target="_blank">18083817</a>]<br></li>
<li>Results in bold ( P 0.05 ) or in bold and underlined ( P 0.01 ) indicate when the inactivation of a regulator produced a statistically signiﬁcant change in the repression produced by MarA compared to the repression produced by MarA in the parental strain under the same conditions ( same column ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18083817" target="_blank">18083817</a>]<br></li>
<li>Moreover , repression of hdeAB by MarA was greater in the absence of GadX or Lrp in exponential phase at pH 7.0 and in the absence of GadW or RpoS in stationary phase at pH 5.5 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18083817" target="_blank">18083817</a>]<br></li>
<li>We found that , in wild-type cells grown in LB medium at pH 7.0 or pH 5.5 , repression of hdeAB by MarA occurred only in stationary phase and was reduced in the absence of H-NS and GadE , the main regulators of hdeAB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18083817" target="_blank">18083817</a>]<br></li>
<li>MarA enhanced hdeAB repression by H-NS in stationary phase at both pH 7.0 and pH 5.5 , which indicates that MarA assists or collaborates with H-NS to repress hdeAB in stationary phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18083817" target="_blank">18083817</a>]<br></li>
<li>Therefore , both H-NS and GadE were involved in the increased hdeAB repression by MarA that occurred as cells enter stationary phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18083817" target="_blank">18083817</a>]<br></li>
<li>H-NS and GadE were associated with the increase in hdeAB repression by MarA that occurred in stationary phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18083817" target="_blank">18083817</a>]<br></li>
<li>On the contrary , MarA reduced the ability of GadE to activate hdeAB in stationary phase at pH 5.5 , which suggests that MarA hinders hdeAB activation by GadE .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18083817" target="_blank">18083817</a>]<br></li>
<li>In turn , MarA enhanced repression of hdeAB by H-NS and hindered activation by GadE in stationary phase and also reduced the activity of GadX , GadW , RpoS , and Lrp on hdeAB under some conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18083817" target="_blank">18083817</a>]<br></li>
<li>Conversely , MarA helped H-NS repression and interfered with GadE activation of hdeAB in stationar phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18083817" target="_blank">18083817</a>]<br></li>
<li>We report the effect of pH , growth-phase , and known regulators of hdeAB on hdeAB expression and repression by MarA , as well as the effect of MarA on the activities of these other regulators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18083817" target="_blank">18083817</a>]<br></li>
<li>To know whether repression of hdeAB by MarA was correlated with decreased acid resistance , we studied the effect of pH and growth-phase on the expression of marA and the effect of MarA on acid resistance .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18083817" target="_blank">18083817</a>]<br></li>
<li>Further research is needed to understand the biological meaning of MarA repression of hdeAB at pH 7.0 in response to salicylate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18083817" target="_blank">18083817</a>]<br></li>
<li>It seems unlikely that GadE , the main activator of hdeAB , assists MarA in repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18083817" target="_blank">18083817</a>]<br></li>
<li>Before studying how these regulators affected MarA repression on hdeAB , we examined their own effect on hdeAB under the chosen growth conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18083817" target="_blank">18083817</a>]<br></li>
<li>hdeAB repression by MarA in strains inactivated for selected regulators of hdeAB TABLE 4 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18083817" target="_blank">18083817</a>]<br></li>
<li>Effect of the other regulators of hdeAB on repression by MarA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18083817" target="_blank">18083817</a>]<br></li>
<li>This strong induction made sta-tionary-phase cells sevenfold more sensitive to acid challenge at pH 2.5 in the presence of marA than in its absence ( Fig. 4 ) , consistent with the repression produced by MarA on hdeAB ( and probably other acid resistance genes ) in stationary phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18083817" target="_blank">18083817</a>]<br></li>
<li>When HdeA100 cells ( bearing pMB102 ) were induced by IPTG to overexpress MarA , we found that MarA prevented nearly all of the 1.7 - to 2.5-fold increase in hdeAB expression as cells entered stationary phase ( Fig. 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18083817" target="_blank">18083817</a>]<br></li>
<li>Overexpression of MarA largely prevented the induction of hdeAB expression as cells enter stationary phase , in part because MarA-mediated repression of hdeAB increased in stationary phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18083817" target="_blank">18083817</a>]<br></li>
<li>The wild-type MarA - galactosi-dase activities ( Miller units ) , control and induced , respectively , for each of the promoters were as follows : for acnA , 50 and 140 ; acrAB , 80 and 450 ; fpr , 80 and 180 ; marRAB , 750 and 2000 ; mdtG , 7 and 37 ; poxB , 35 and 85 ; and zwf , 200 and 1.200 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21097628" target="_blank">21097628</a>]<br></li>
<li>The WT MarA - galactosidase activities ( Miller units ) , control and induced , respectively , for the indicated promoters were as follows : for fumC , 75 ( control ) and 220 ( induced ) ; inaA , 100 and 1,400 ; mdaB , 150 and 3,000 ; micF , 80 and 800 ; nfsB , 80 and 1,000 ; pqiA , 25 and 100 ; sodA , 1,200 and 3,000 ; ybjC 150 and 3,800 ; and yhbW , 400 and 1,800 FIG. 2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21097628" target="_blank">21097628</a>]<br></li>
<li>Purified His6-MarA , diluted as needed in Reaction Buffer , was added to the pre-incubation mixture on ice to give 200 nM ; the buffer in which His - MarA was prepared was similarly 6 diluted and added to the control .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14701822" target="_blank">14701822</a>]<br></li>
<li>Lane 1 , DNA alone ( no protein ) ; lanes 2 -- 5 , increasing concentrations of His - MarA , 100 , 6 200 , 400 and 600 nM respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12100559" target="_blank">12100559</a>]<br></li>
<li>The concentrations of MarA used were 0 , 100 , 200 , or 400 nM ( lanes 0 , 1 , 2 , and 4 , respectively ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14701822" target="_blank">14701822</a>]<br></li>
<li>Overproduction of cytoplasmic GadA and GadB and of several unidentiﬁed cytoplasmic membrane proteins as well as reduction in the amount of the outer membrane protein , OmpP , in the mppA null mutant indicate that MppA regulates a number of genes in addition to those already known to be controlled by MarA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10438753" target="_blank">10438753</a>]<br></li>
<li>It is known that MAR is associated with an increase in MarA , which then activates the acrAB operon to produce AcrA and AcrB , which constitute the drug efﬂux pump primarily responsible for MAR ( 1 , 21 , 27 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10438753" target="_blank">10438753</a>]<br></li>
<li>Both base-specific and phosphate contacts contribute to MarA activity and DNA binding Eight amino acid residues of MarA are positioned to contact bases in the major groove of the marbox ; only three of these , Arg46 , Thr93 and Arg96 are predicted to form hydrogen bonds ( Figure 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10873449" target="_blank">10873449</a>]<br></li>
<li>Although the afﬁnity of MarA for this marbox was relatively low in vitro , this ﬁnding is not uncommon ( Martin et al. , 2002 ) and may be related to its unusual sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12100559" target="_blank">12100559</a>]<br></li>
<li>MarA is an AraC family global regulator ( Rhee et al. , 1998 ; Martin and Rosner , 2001 ) , which transcriptionally activates at least 21 promoters , the marA regulon ( Barbosa and Levy , 2000 ; Pomposiello et al. , 2001 ; Martin and Rosner , 2002 ; 2003 ) , by binding to a site ( marbox ) overlapping or upstream of the -35 signal ( Martin et al. , 1996 ; 1999 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15225313" target="_blank">15225313</a>]<br></li>
<li>The surface of MarA , derived by X-ray crystallography [ GRASP PROGRAM ( Nicholls et al. , 1991 ; Rhee et al. , 1998 ) ] , bound to B-form DNA is depicted as it might exist when binding the mar promoter ( left ) or the zwf promoter ( right ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10564485" target="_blank">10564485</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_41">
Our understanding of how AraC-type proteins bind to DNA has been enormously advanced by the determination of the high-resolution structure of MarA bound to its target ( Rhee et al. , 1998 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10760178" target="_blank">10760178</a>]<br></label>
<input id="act_similarity_41", type="checkbox">
<div class="ACT_similar">
<li>Our understanding of how AraC-related proteins bind to DNA has been greatly advanced by the determination of the high-resolution structure of MarA bound to its target ( Rhee et al. , 1998 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12675795" target="_blank">12675795</a>]<br></li></div>
<li>In the parental strain , MarA activation of acrA-lacZ expression obtained by - galactosidase assays ( not shown ) was similar to MarA activation of acrA expression found by RT-qPCR , showing the equivalence of both assays ; however , in the acrB mutant , acrA-lacZ activation by MarA was no longer reduced compared to the level for the parental strain ( not shown ) , in contrast to the RT-qPCR results .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20211899" target="_blank">20211899</a>]<br></li>
<li>To study whether this reduction in acrA activation by MarA was produced by the insertion of the transposome in acrB or by the absence of the AcrB protein , we added a single-copy plasmid bearing an acrA-lacZ transcriptional fusion to the parental and acrB mutant strains .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20211899" target="_blank">20211899</a>]<br></li>
<li>This ﬁnding shows that the reduced acrA activation by MarA in the acrB mutant compared to the level for the parental strain observed by RT-qPCR was produced artifactually by the cis transposome insertion in acrB and not by the absence of the AcrB protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20211899" target="_blank">20211899</a>]<br></li>
<li>After transferring the mutations in the identiﬁed genes to the marR and marRA parental strains , we examined th susceptibilities of the resulting strains to four structurally and functionally unrelated bactericidal ( cefoxitin and norﬂoxacin ) or bacteriostatic ( chloramphenicol and minocycline ) antibiot-ics whose increased MICs were MarA dependent ( their MICs were increased in the marR parental strain compared to the level for the marRA parental strain ) ( Table 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20211899" target="_blank">20211899</a>]<br></li>
<li>MarA mediated low-level resistance to all four antibiotics , as shown by the signiﬁcant increase in MICs in the marR parental strain compared to the levels for the marRA parental and wild-type strains .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20211899" target="_blank">20211899</a>]<br></li>
<li>Interestingly , the DmarO288 deletion responded exceptionally well to induction by salicylate in marRAB1 cells , suggesting that increased production of MarA can overcome the DmarO288 defect .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8636021" target="_blank">8636021</a>]<br></li>
<li>The results in Fig. 7 demonstrate that sodium salicylate also induces the waaY gene as efﬁciently as paraquat does , and the induction is dependent on both the SoxRS and MarA systems .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19376854" target="_blank">19376854</a>]<br></li>
<li>In conclusion , MarA plays an active and complex role in the regulation of hdeAB and acid resistance in E. coli that depends on the growth conditions , on other regulators of hdeAB , and on the degree of induction of marA expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18083817" target="_blank">18083817</a>]<br></li>
<li>Our results agree with those in vitro results and suggest that activation of marA expression by CRP-cAMP does occur in cells grown in rich medium , which is important for full acrA , tolC , and micF activation by MarA and thus for MarA-mediated MDR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20211899" target="_blank">20211899</a>]<br></li>
<li>Moreover , the functional overlapping of these regulators with MarA may explain why the level of marA expression was higher in the tolC mutant than in the parental strain but the level of MarA induction of acrA and micF was not higher .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20211899" target="_blank">20211899</a>]<br></li>
<li>The reason for such a reduction in MarA-mediated MDR in the crp and cyaA mutants seems to be that the expression of marA ( Fig. 2A ) and , in consequence , the MarA induction of acrA , tolC , and micF ( Fig. 2B ) were signiﬁcantly reduced in both mutants .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20211899" target="_blank">20211899</a>]<br></li>
<li>The reason for such a reduction in MarA-mediated MDR in the pcnB mutant seems to be that the levels of marA transcripts ( Fig. 2A ) and , by consequence , MarA induction of acrA , tolC , and micF ( Fig. 2B ) were strongly reduced in this mutant compared to the levels for the parental strain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20211899" target="_blank">20211899</a>]<br></li>
<li>Considering that marA expression was unaffected ( Fig. 2A ) , that no protein-protein interaction between DamX and MarA was found in two-hybrid experiments ( data not shown ) , and that DamX is a predicted membrane protein , the effect of DamX on acrA induction by MarA is likely to be indirect .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20211899" target="_blank">20211899</a>]<br></li>
<li>In the same experiment we also examined the expression of hdeA , which had been observed in another macroarray study to be repressed upon MarA induction in cells ( 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14701822" target="_blank">14701822</a>]<br></li>
<li>Thus both plasmid-mediated expression and salicylate induction of MarA , directly or indirectly , reduced the levels of rob expression 2 -- 4-fold in vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16478729" target="_blank">16478729</a>]<br></li>
<li>Therefore , damX contributes to MarA-mediated MDR by speciﬁcally enhancing , probably indirectly , MarA induction of acrA expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20211899" target="_blank">20211899</a>]<br></li>
<li>A. Kinetics of MarA regulation of nfnB expression and of IPTG induction of marA on pMB102 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12100559" target="_blank">12100559</a>]<br></li>
<li>B. Effect of IPTG induction of MarA on the expression of nfnB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12100559" target="_blank">12100559</a>]<br></li>
<li>However , it dramatically decreased MarA induction of acrA expression ( Fig. 2B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20211899" target="_blank">20211899</a>]<br></li>
<li>( B ) Effect of inactivation of each gene on MarA induction of acrA , tolC , and micF expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20211899" target="_blank">20211899</a>]<br></li>
<li>Those genes activated both by exposure to NaSal and by expression of MarA , 20 in total , are indicated in boldface in Table 2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11395452" target="_blank">11395452</a>]<br></li>
<li>In addition , the PQ - or SoxSresponsive genes gatABD , gltA , nfnB , and ybjC were recently shown to be activated by constitutive expression of MarA ( 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11395452" target="_blank">11395452</a>]<br></li>
<li>MarA activates the expression of diverse genes , including acrAB , micF , mlr-1 , -2 , and -3 , slp , and inaA , which endow cells with resistance ( 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19376854" target="_blank">19376854</a>]<br></li>
<li>The work presented here and previously ( 14 ) indicates that elevated MarA expression also directly activates some oxidative-stress genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8282690" target="_blank">8282690</a>]<br></li>
<li>To conﬁrm that the nfnB activation observed in the array study was speciﬁcally mediated by MarA rather than by a cell stress response to the constitutive expression of the protein from a multicopy number plasmid , Northern blot analysis was performed on cells in which MarA production was inducible .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12100559" target="_blank">12100559</a>]<br></li>
<li>To confirm that the decrease in rob expression was a result of MarA expression , we analyzed the levels of the rob transcript by Northern blot analysis and reverse transcription-PCR under different conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16478729" target="_blank">16478729</a>]<br></li>
<li>Such a speciﬁc contribution by H-NS to MarA induction of acrA , but not the other MarA-regulated genes tested , is of interest because H-NS is known to have no effect on regulation of acrA per se ( 37 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20211899" target="_blank">20211899</a>]<br></li>
<li>An activity of the efﬂux pump is suggested to be regulated by MarA ( 38 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9473050" target="_blank">9473050</a>]<br></li>
<li>Production of OmpF and LamB was regulated negatively by MarA also in MC4100 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9473050" target="_blank">9473050</a>]<br></li>
<li>Moreover , such an effect also occurs on hdeAB , which is regulated by MarA and H-NS both independently and synergistically ( 42 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20211899" target="_blank">20211899</a>]<br></li>
<li>If MppA is involved in the negative regulation of marA , the MarA and AcrAB contents of the mppA mutant should increase relative to its parent .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10438753" target="_blank">10438753</a>]<br></li>
<li>In addition , we report on the PQ regulation of the hmpSt and report that RamA , a homologue of MarA in Salmonella is responsible for most of the hmpSt induction by this redox-cycling agent .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17024490" target="_blank">17024490</a>]<br></li>
<li>In the presence of inducers such as salicylate ( SAL ) , MarR repressor activity is antagonized , resulting in increased levels of MarA and , consequently , increased antibiotic resistance .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8521301" target="_blank">8521301</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8521301" target="_blank">8521301</a>]<br></li>
<li>Therefore , if MarA plays a role in the susceptibility to the nitrofurans , this appears to result in increased susceptibility , which is in contrast with the increased resistance seen with all the other agents that are part of the Mar phenotype .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12100559" target="_blank">12100559</a>]<br></li>
<li>It is likely that the effects of classic mar mutations on outer membrane permeability are the direct result of increased MarA activity in these cells .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8491710" target="_blank">8491710</a>]<br></li>
<li>Their increased expression would place them in the regulon , possibly as responders to MarA , MarB , and/or MarR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8383113" target="_blank">8383113</a>]<br></li>
<li>MarA is speciﬁed by the marRAB operon , whose expression is induced by sodium salicylate , menadione , and other chemicals which inactivate the autorepressor MarR ( 1 , 10 , 33 , 38 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18083817" target="_blank">18083817</a>]<br></li>
<li>A homologous MarR repressor binds salicylate and controls the expression of a global regulator , MarA , in E. coli ( 14 -- 16 , 19 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10438794" target="_blank">10438794</a>]<br></li>
<li>The other unit is an operon comprising marRAB , encoding the MarR repressor , which binds to marO and negatively regulates expression of marRAB ( 24 , 33 ) ; MarA , a transcriptional-activator ( 1 ) , which activates expression of other * Corresponding author .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10438753" target="_blank">10438753</a>]<br></li>
<li>MarA transcription is controlled by the repressor function of MarR ( encoded within the marRAB operon ; Cohen et al. , 1993 ) , whereas SoxS transcription depends on its activation by SoxR ( encoded within the soxRS region ; Amabile-Cuevas & Demple , 1991 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22515487" target="_blank">22515487</a>]<br></li>
<li>In the case of MarA , this is done by the interaction of repressor MarR , which regulates the transcription of the marRAB operon , with chemicals such as salicylate © 2003 Blackwell Publishing Lt ( Cohen et al. , 1993 ; Martin and Rosner , 1995 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12791142" target="_blank">12791142</a>]<br></li>
<li>Thus , the mar operon has the remarkable property that its ﬁrst gene product , MarR , represses its transcription whereas the second gene product , MarA , stimulates its transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8636021" target="_blank">8636021</a>]<br></li>
<li>The multidrug-resistant operon , marRAB , encodes a repressor , MarR , an activator , MarA , and an unknown function protein , MarB , which coordinately regulate the mar transcription to maintain intrinsic resistance to antimicrobial substances ( 8 , 9 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25294881" target="_blank">25294881</a>]<br></li>
<li>According to the hypothesis that proposes a repressor role for MarR ( 6 ) , the increased level of antioxidant enzymes and the enhanced antibiotic resistance observed in the soxQl and the cfxBl mutants would be due to increased transcription of the mar operon and therefore to elevated levels of the putative activator protein MarA in these strains .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8282690" target="_blank">8282690</a>]<br></li>
<li>Active efﬂux of FQ by multidrug resistance pumps like AcrAB , and reduced uptake due to OmpF , both regulated by the transcription factor MarA , are also implicated in resistance ( 5 , 18 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10858345" target="_blank">10858345</a>]<br></li>
<li>Introduction Fluoroquinolone resistance of clinical Escherichia coli isolates was shown to involve target mutations in topoisomerases II ( GyrA/B ) and IV ( ParC/E ) , often in combination with lowered drug accumulation due to multidrug efﬂux pumps .1 -- 5 Efﬂux via the AcrAB efﬂux system is positively regulated by the transcription factor MarA , which itself is under negative control by the repressor MarR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11751765" target="_blank">11751765</a>]<br></li>
<li>The product of marA , the transcription factor MarA , directly activates or represses many chromosomal genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20211899" target="_blank">20211899</a>]<br></li>
<li>MarA , an AraC/XylS transcriptional-regulator of Esche-richia coli , directly activates or represses multiple chromosomal genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18083817" target="_blank">18083817</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_91">
The Esche-richia coli transcriptional-regulator MarA directly represses transcription of purA and hdeA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22210763" target="_blank">22210763</a>]<br></label>
<input id="act_similarity_91", type="checkbox">
<div class="ACT_similar">
<li>The Escherichia coli transcriptional-regulator MarA directly represses transcription of purA and hdeA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18083817" target="_blank">18083817</a>]<br></li></div>
<li>The MarA protein is evidently a transcriptional-regulator whose activity in the cell increases when its concentration is increased ( 4 , 14 , 20 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7896685" target="_blank">7896685</a>]<br></li>
<li>The MarA protein of Escherichia coli activates the transcription of the genes of the mar regulon resulting in increased resistance to antibiotics and super-oxide-generating compounds ( Alekshun & Levy , 1997 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10873449" target="_blank">10873449</a>]<br></li>
<li>MarA is believed to be a transcriptional-activator of genes involved in the Mar phenotype , since alone it causes increased resistance to multiple antibiotics ( 5 , 11 , 31 , 33 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7836283" target="_blank">7836283</a>]<br></li>
<li>The MarA protein is a transcriptional-activator that controls the expression of the genes in the Mar regulon , leading to increased antibiotic resistance [ 8,9 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11094278" target="_blank">11094278</a>]<br></li>
<li>MarA ( E89A ) exhibited increased binding relative to WT MarA for 13 of the 16 promoter marboxes examined although the increase was small for acrAB ( Table 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21097628" target="_blank">21097628</a>]<br></li>
<li>Furthermore , WT MarA binding to a synthetic marbox in vitro was enhanced when the phosphate group between positions 12 and 13 was eliminated on one strand .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21097628" target="_blank">21097628</a>]<br></li>
<li>Consistent with this model , we ﬁnd that , ﬁrst , WT MarA binds more tightly to DNA of the fpr marbox lacking a phosphate between positions 12 and 13 than to uninterrupted dsDNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21097628" target="_blank">21097628</a>]<br></li>
<li>) As outlined in the Discussion , these results , namely , ( i ) that variant E89D is virtually inactive , ( ii ) that E89G is very active although to a lesser extent than E89A , ( iii ) that E89V is only marginally more active than the WT , and ( iv ) that SoxS ( V83E ) has reduced activation although not to the low levels expressed by WT MarA , are consistent with the possibility that that the side chain of E89 sterically inhibits interaction with the DNA for marbox sequences lacking a T at position 12 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21097628" target="_blank">21097628</a>]<br></li>
<li>In spite of this , MarA ( E89A ) activation was greater than that of WT MarA for inaA , sodA , and ybjC ; it was comparable to that of WT MarA for micF and less than that of WT MarA for fumC , mdaB , nfsB , and ybhW ( Table 3 and Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21097628" target="_blank">21097628</a>]<br></li>
<li>For the nine class II promoters , MarA ( Q91A ) signiﬁcantly reduced the activation of fumC , inaA , micF , pqiA , and ybjC but had no signiﬁcant effect on nfsB , mdaB , sodA , or yhbW .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21097628" target="_blank">21097628</a>]<br></li>
<li>His43Ala and Lys49Ala also had severe effects on activation , decreasing MarA activity by at least 80 % on the fpr , fumC and zwf promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10873449" target="_blank">10873449</a>]<br></li>
<li>SoxS ( V83E ) reduced the activation of these promoters relative to WT SoxS , but SoxS ( V83E ) was still more active than WT MarA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21097628" target="_blank">21097628</a>]<br></li>
<li>We have shown here that the MarA glutamic acid residue E89 is responsible for decreasing the binding of WT MarA relative to SoxS for the class I marbox promoters , acnA , mdtG , fpr , and zwf , thereby decreasing the relative activation of these promoters and hence the resistance engendered to superoxides by MarA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21097628" target="_blank">21097628</a>]<br></li>
<li>Among the promoters activated by MarA , the correlation between the strength of binding and the extent of activation is poor ( 23 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21097628" target="_blank">21097628</a>]<br></li>
<li>MarA ( E89A ) showed large increases in binding ( relative to WT MarA ) , concomitant with large increases in relative activation for the acnA , fpr , mdtG , and zwf promoters , and showed modest increases in both binding and activation of the acrAB promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21097628" target="_blank">21097628</a>]<br></li>
<li>Among the class I promoters that exhibited measurable binding to MarA , MarA ( Q91A ) showed no greater binding or activation for acrAB , a modest reduction in activation and binding for mdtG and zwf , and a reduction in binding but not in activation for marRAB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21097628" target="_blank">21097628</a>]<br></li>
<li>E89G was approximately twice as active as WT MarA for fpr and 4-fold more active for mdtG although for both promoters it was considerably less active than E89A .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21097628" target="_blank">21097628</a>]<br></li>
<li>Of the seven class I promoters examined , MarA ( Q91A ) activated the acrAB , marRAB , and zwf promoters to similar extents as WT MarA but activated acnA , fpr , mdtG , and poxB to only 60 % or less of MarA WT levels ( Table 2 ; see also below ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21097628" target="_blank">21097628</a>]<br></li>
<li>Indeed , nfsB and sodA are among the promoters that Q91A activated to the same extent as WT MarA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21097628" target="_blank">21097628</a>]<br></li>
<li>179 , No. 23 Fis , an Accessorial Factor for Transcriptional Activation of the mar ( Multiple Antibiotic Resistance ) Promoter of Escherichia coli in the Presence of the Activator MarA , SoxS , or Rob ROBERT G. MARTIN * AND JUDAH L. ROSNER Laboratory of Molecular Biology , National Institute of Diabetes and Digestive and Kidney Diseases , Bethesda , Maryland 20892-0560 Received 12 June 1997/Accepted 3 September 1997 Transcription of the multiple antibiotic resistance marRAB operon increases when one of the sequence-related activators , MarA , SoxS , or Rob , binds to the `` marbox '' centered at 261.5 relative to the transcriptional start site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9393706" target="_blank">9393706</a>]<br></li>
<li>MICROBIOLOGY LETTERS R E S E A R C H L E T T E R SoxS-dependent coregulation of ompN and ydbK in a multidrug-resistant Escherichia coli strain Anna Fàbrega1 , Judah L. Rosner2 , Robert G. Martin2 , Mar Solé1 & Jordi Vila1 ,3 1 2 Barcelona Centre for International Health Research , CRESIB , Hospital Clınic-Universitat de Barcelona , Barcelona , Spain ; Laboratory of Molecular Biology , National Institute of Diabetes and Digestive and Kidney Diseases , National Institutes of Health , Bethesda , MD , USA ; and Department of 3 Microbiology , Hospital Clınic , School of Medicine , University of Barcelona , Barcelona , Spain Correspondence : Jordi Vila , Servei de Microbiologia , Hospital Clınic , Facultat de Abstract SoxS , MarA , and Rob are homologous transcriptional-activators of numerous superoxide - and antibiotic resistance genes but many of the regulated genes are yet to be characterized .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22515487" target="_blank">22515487</a>]<br></li>
<li>Alternatively , this transcriptional fusion was also tested for induction in the presence of SAL and DIP to evaluate the regulatory role of MarA and Rob , respectively ( Table 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22515487" target="_blank">22515487</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_115">
Martin , R.G. , and Rosner , J.L. ( 1997 ) Fis , an accessorial factor for transcriptional activation of the mar ( multiple anti-biotic resistance ) promoter of Escherichia coli in the presence of activator MarA , SoxS , or Rob .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12791142" target="_blank">12791142</a>]<br></label>
<input id="act_similarity_115", type="checkbox">
<div class="ACT_similar">
<li>Martin , R.G. , and Rosner , J.L. ( 1997 ) Fis , an accessorial factor for transcriptional activation of the mar ( multiple anti-biotic resistance ) promoter of Escherichia coli in the presence of the activator MarA , SoxS , or Rob .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10564485" target="_blank">10564485</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15225313" target="_blank">15225313</a>]<br></li></div>
<li>Fis , an accessor-ial factor for transcriptional activation of the mar ( multiple antibiotic resistance ) promoter of Escheri-chia coli in the presence of the activator MarA , SoxS , or Rob .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11866515" target="_blank">11866515</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_118">
Fis , an accessorial factor for transcriptional activation of the mar ( multiple antibiotic resist-ance ) promoter of Escherichia coli in the presence of the activator MarA , SoxS , or Rob .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11751765" target="_blank">11751765</a>]<br></label>
<input id="act_similarity_118", type="checkbox">
<div class="ACT_similar">
<li>Martin R , Rosner J. Fis , an accessorial factor for transcriptional activation of the mar ( multiple antibiotic resistance ) promoter of Escherichia coli in the presence of the activator MarA , SoxS , or Rob .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22465792" target="_blank">22465792</a>]<br></li>
<li>Fis , an accessorial factor for transcriptional activation of the mar ( multiple antibiotic resistance ) promoter of Escherichia coli in the presence of the activator MarA , SoxS , or Rob .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10873449" target="_blank">10873449</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10368169" target="_blank">10368169</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14679232" target="_blank">14679232</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20211899" target="_blank">20211899</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10850996" target="_blank">10850996</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21097628" target="_blank">21097628</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16885445" target="_blank">16885445</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10572146" target="_blank">10572146</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10894725" target="_blank">10894725</a>]<br></li></div>
<li>Northern analysis conﬁrmed the transcriptional activation of yggX by oxidative-stress or SoxS expression but not in response to the related MarA or Rob proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14594836" target="_blank">14594836</a>]<br></li>
<li>In JA300 , AcrA production was regulated positively by each of the transcriptional-activator proteins , MarA , Rob , and SoxS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9473050" target="_blank">9473050</a>]<br></li>
<li>Characterization of transcriptional-activator proteins SoxS , MarA , and Rob in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22465792" target="_blank">22465792</a>]<br></li>
<li>Common features of DNA binding and transcriptional activation by the Escherichia coli SoxS , Rob , and MarA proteins , p. 116 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8522515" target="_blank">8522515</a>]<br></li>
<li>In the presence of MarA , SoxS , or Rob , Fis further enhanced the activity of the promoter twofold provided the promoter was also capable of binding Fis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9393706" target="_blank">9393706</a>]<br></li>
<li>Thus , basal levels of MarA and Rob and induced levels of SoxS independently contribute to transcriptional activation of mar and require a functional marbox for activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9393706" target="_blank">9393706</a>]<br></li>
<li>Thus , activation by Fis is dependent on the presence of MarA or Rob ( or induced levels of SoxS ; see below ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9393706" target="_blank">9393706</a>]<br></li>
<li>We conclude that transcriptional activation by Fis is dependent on the presence of MarA , SoxS , or Rob at the mar promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9393706" target="_blank">9393706</a>]<br></li>
<li>Nevertheless , Fis stimulated the transcriptional activation of the promoter by Rob , SoxS , and MarA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9393706" target="_blank">9393706</a>]<br></li>
<li>AcrB is regulated by a global transcriptional-activator MarA and a local transcriptional-repressor AcrR [ 12 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17644067" target="_blank">17644067</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_131">
Gillette , W.K. , Martin , R.G. , and Rosner , J.L. ( 2000 ) Probing the Escherichia coli transcriptional-activator MarA using alanine-scanning mutagenesis : residues important for DNA binding and activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12100559" target="_blank">12100559</a>]<br></label>
<input id="act_similarity_131", type="checkbox">
<div class="ACT_similar">
<li>Probing the Escherichia coli transcriptional-activator MarA using alanine-scanning mutagenesis : residues important for DNA binding and activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21097628" target="_blank">21097628</a>]<br></li>
<li>Probing the Escherichia coli transcriptional-activator MarA using alanine-scanning mutagenesis : res-idues important for DNA binding and activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10940041" target="_blank">10940041</a>]<br></li></div>
<li>An additional critical hydrophobic interaction was detected by substitutions for Leu24 and Lys64 which severely reduced MarA activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10873449" target="_blank">10873449</a>]<br></li>
<li>Alternatively , the repression or activation function of marRAB ( through MarA ) might be indirect .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7836283" target="_blank">7836283</a>]<br></li>
<li>Thus MarA is a dual regulator capable of both activation and repression , depending upon the promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14701822" target="_blank">14701822</a>]<br></li>
<li>This suggests that the DmarO281 promoter can not respond to MarA in vivo , and so it may be used to distinguish MarA-dependent activation from other causes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8636021" target="_blank">8636021</a>]<br></li>
<li>This observation is consistent with activation of the glutamate decarboxylase genes by MarA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10438753" target="_blank">10438753</a>]<br></li>
<li>Because helical dependence is a critical factor in activation by MarA ( 19 ) , the transcriptional outcome , repression versus activation , may be dependent on which DNA face the protein binds as is the case for GalR ( 41 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16478729" target="_blank">16478729</a>]<br></li>
<li>Sufﬁciency of MarA for in vitro transcriptional activation of zwf , fpr , fumC , micF , nfo , and sodA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8522515" target="_blank">8522515</a>]<br></li>
<li>Identiﬁcation of the zwf sequences required for transcriptional activation by MarA in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8522515" target="_blank">8522515</a>]<br></li>
<li>In preliminary experiments , transcriptional activation of zwf was found to be maximal when open complexes were formed in the presence of 1 pmol of MarA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8522515" target="_blank">8522515</a>]<br></li>
<li>Model of transcriptional activation by MarA in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21097628" target="_blank">21097628</a>]<br></li>
<li>Ambidextrous transcriptional activation by MarA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8522515" target="_blank">8522515</a>]<br></li>
<li>In the absence of this region ( DmarO280 ) , binding of MarA decreased about 30-fold and transcriptional activation by MarA and MalE-SoxS was reduced about 3-fold ( Table 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8636021" target="_blank">8636021</a>]<br></li>
<li>Marbox orientation and the spacing between marbox and signals for RNP binding are critical for transcriptional activation , presumably to align MarA with RNP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10564485" target="_blank">10564485</a>]<br></li>
<li>Deletion of upstream DNA adjoining the marbox reduced MarA binding efﬁciency 30-fold and transcriptional activation 2 - to 3-fold , providing evidence for an accessory marbox .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8636021" target="_blank">8636021</a>]<br></li>
<li>We chose these substitutions because the co-crystal structure of MarA bound to the marbox of the mar promoter indicates that R90 of MarA makes a van der Waals contact with C15 , a conserved residue in RE2 ( CAAA ; C15 in boldface ) .21,22 further the transcription activation of class II promoters by Rob , in order to determine whether the presence of the CTD has any effect on the underlying mechanism .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22465792" target="_blank">22465792</a>]<br></li>
<li>For example , Rob , SoxS , and MarA function as monomers ,18 bind the same but highly degenerate target sequences,18-23 and activate transcription of a common set of ~ 100 genes,16,24-28 by the same , novel prerecruitment/DNA scanning mechanism,11,29-31 although each paralog differentially activates transcription of the target genes .16,32,33 Rob differs from its paralogs in several ways : it is synthesized constitutively34 ,35 rather than in response to an inducing signal , as are SoxS36 and MarA ,37,38 and it contains a CTD .35,39 The CTD is both necessary and sufficient for sequestration -- dispersal ,40 the novel mechanism for turning off and turning on Rob 's ability to function as a transcription activator in the presence or absence , respectively , of an inducing signal , for example , bile salts .37 In addition , the co-crystal structure of Rob bound to the robbox in the micF promoter39 differs significantly from the co-crystal structure of MarA bound to the marbox in the mar promoter .41 ( We note that even though the binding sites for MarA , Rob , and SoxS are the same in a given promoter , the site is given the name of the protein bound there in a specific situation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22465792" target="_blank">22465792</a>]<br></li>
<li>In turn , this indicates that Rob and MarA do not bind DNA differently and that in vivo Rob 's CTD does not cause it to-bind robbox DNA differently than MarA binds marbox DNA in vitro41 or than SoxS binds soxbox DNA in vivo .42 We also note that the in vitro binding of Rob , SoxS , and MarA to a robbox/soxbox/marbox bends the DNA by about 30 ° ; 50 however , although mar DNA is bent by about 35 ° in the co-crystal structure with MarA ,41 the micF DNA is straight in the co-crystal structure with Rob .39 In addition , the boundaries of the regions protected from DNase I digestion by the binding of purified Rob or MalE -- SoxS to DNA containing the zwf and fumC promoters were the same ; 50 this would not be the case unless Rob 's C-terminal HTH motif was bound to RE2 of these promoters , as occurs with MalE -- SoxS .51 Together , these data suggest that the absence of DNA binding by helix-6 and the linearity of DNA in the Rob -- micF co-crystal structure do not accurately represent the DNA binding properties of Rob .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22465792" target="_blank">22465792</a>]<br></li>
<li>In particular , recognition helix-3 and helix-6 of MarA 's dual HTH motifs bind RE1 and RE2 , respectively , of mar 's robbox , while helix-3 within Rob 's N-terminal HTH motif binds RE1 of micF 's robbox , but helix-6 does not bind RE2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22465792" target="_blank">22465792</a>]<br></li>
<li>Thus , the data in Fig. 3b and Fig. 3c not only show that in vivo Rob binds to robbox DNA in the same orientation as in the co-crystal structure of MarA bound to the marbox of the mar promoter but also , more importantly , show that amino acids of helix-3 and helix-6 of Rob are bound in vivo to RE1 and RE2 , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22465792" target="_blank">22465792</a>]<br></li>
<li>Also shown , at the lower right , are the indirect negative effects of SoxS and MarA on OmpF synthesis , mediated by induction of micF , whose RNA product interacts with ompF mRNA to prevent OmpF production .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8866468" target="_blank">8866468</a>]<br></li>
<li>Presumably , uninduced levels of MarA and/or SoxS mediate a low-level induction of the inaA gene ; the lower basal level of inaA expression in Dmar strains was noted earlier ( Rosner and Slonczewski , 1994 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12791142" target="_blank">12791142</a>]<br></li>
<li>The strong upregulation of nfnB by the plasmidspeciﬁed MarA observed in this study contrasts with the reported weaker upregulation of NfnB activity and nfnB expression by SoxS ( Liochev et al. , 1999 ; Pomposiello et al. , 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12100559" target="_blank">12100559</a>]<br></li>
<li>Although MarA homologue SoxS was also shown to regulate the transcription of acrAB strongly ( Ma et al. , 1996 ; White et al. , 1997 ) , SoxS , also a small protein of 107 residues , exerts its action by altering the level of its own expression , which is regulated via SoxR by oxidizing agents that generate the superoxide radical ( Greenberg et al. , 1990 ; Ding et al. , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12791142" target="_blank">12791142</a>]<br></li>
<li>Transcription of DmarO279 DNA , with a deletion of mar nt 1312 to 1338 ( and nt 1311 of the vector ) , was stimulated by both MarA and MalE-SoxS to the same extent as that of the wild type ( approximately ﬁvefold ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8636021" target="_blank">8636021</a>]<br></li>
<li>Hence , to ensure that MarA was not limiting , activation reaction mixtures contained 2 pmol of MarA ( 67 nM ) ; other parameters of in vitro transcription , e.g. , salt concentration , were not optimized for each test promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8522515" target="_blank">8522515</a>]<br></li>
<li>We report here the puriﬁcation of MarA and its ability to stimulate transcription of six mar/soxRS promoters in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8522515" target="_blank">8522515</a>]<br></li>
<li>The report ( 32 ) that overexpression of SoxS induced mar transcription in vivo led us to test for transcriptional activation of mar by puriﬁed MarA and MalE-SoxS proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8636021" target="_blank">8636021</a>]<br></li>
<li>221 RESULTS Transcriptional activation in vitro of mar by MarA and MalE-SoxS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8636021" target="_blank">8636021</a>]<br></li>
<li>We report here the existence of a `` mar/soxbox '' upstream of the 235 hexamer which binds MarA and enables activation of mar transcription by MarA and SoxS both in vitro and in vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8636021" target="_blank">8636021</a>]<br></li>
<li>MarA activation of in vitro mar transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8636021" target="_blank">8636021</a>]<br></li>
<li>Effects of upstream deletions on MarA activation of mar transcription in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8636021" target="_blank">8636021</a>]<br></li>
<li>Having previously demonstrated that promoters that respond to MalE-SoxS also respond to MarA ( 18 ) , we tested puriﬁed MarA for its ability to stimulate mar transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8636021" target="_blank">8636021</a>]<br></li>
<li>The ﬁnding that puriﬁed MarA activates transcription of these promoters in the absence of other factors conﬁrms the in vivo observations that MarA alone , like SoxS , can transcriptionally activate the mar regulon ( 9 , 24 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8522515" target="_blank">8522515</a>]<br></li>
<li>To determine whether the puriﬁed MarA protein affected gene expression , we tested its ability to activate the in vitro transcription of zwf , sodA , micF , and fumC , genes known to be under mar control in vivo , and of fpr and nfo , genes not known to be regulated by mar in vivo but activated by SoxS in vivo and by MalE-SoxS in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8522515" target="_blank">8522515</a>]<br></li>
<li>Since MarA protein is highly homologous to SoxS , and since a MalE-SoxS fusion protein has recently been shown to activate soxRS regulon transcription , the ability of MarA to activate transcription of these genes was tested .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8522515" target="_blank">8522515</a>]<br></li>
<li>Having previously demonstrated that SoxS ( as a MalE-SoxS fusion protein ) and MarA are essentially identical in their abilities to activate in vitro transcription of genes of the sox-mar regulons , we investigated the properties of Rob as a transcriptional-activator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626315" target="_blank">8626315</a>]<br></li>
<li>Moreover , it would be interesting to purify the MarA , TetD , and Rob proteins and determine their respective abilities to activate in vitro transcription of the fusion constructs , as we have already done with our puriﬁed MalE-SoxS fusion protein ( 19 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7896696" target="_blank">7896696</a>]<br></li>
<li>Moreover , we previously showed that puriﬁed MarA and MalE-SoxS proteins activate zwf transcription by about three - and eightfold , respectively ( 18 , 19 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626315" target="_blank">8626315</a>]<br></li>
<li>PCR-ampliﬁed DNA fragments were assayed for their abilities to act as templates for in vitro transcription and for the stimulation of transcription by MarA and MalE-SoxS ( Fig. 5 ; Table 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8636021" target="_blank">8636021</a>]<br></li>
<li>Puriﬁed MarA and MalE-SoxS proteins stimulated mar transcription about 6 - and 15-fold , respectively , when the RNA polymerase/DNA ratio was 1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8636021" target="_blank">8636021</a>]<br></li>
<li>As with MalE-SoxS , MarA enhanced mar transcription only ; 1.3-fold when RNA polymerase was in 10-fold excess ( Fig. 1A ) , but when RNA polymerase was limiting , MarA stimulated mar transcription by 4.1 - to 4.9-fold , or 5.5 - to 6.3-fold when normalized to gnd ( Fig. 1B ; Table 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8636021" target="_blank">8636021</a>]<br></li>
<li>Thus , in a minimal in vitro system under conditions of both limiting and excess RNA polymerase , both MalE-SoxS and MarA transcriptionally activate the mar promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8636021" target="_blank">8636021</a>]<br></li>
<li>Miller et al. ( 28 ) have found that SoxS activates mar transcription in vivo , and we have recently found a marbox sequence in the promoter region upstream of marRAB which binds MarA and enables either puriﬁed MalE-SoxS or MarA to activate transcription of marRAB in vitro ( 18 , 26 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8522515" target="_blank">8522515</a>]<br></li>
<li>The similarity of MarA and SoxS with respect to amino acid sequence homology , DNA recognition , and transcriptional activation properties suggests that they arose initially by gene duplication and then diverged to accommodate the constraints mandated by their speciﬁc stimulus-responsive regulatory systems : the salicylate-sensitive negative control mediated by MarR and the superoxide-sensitive positive control mediated by SoxR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8522515" target="_blank">8522515</a>]<br></li>
<li>These derivatives were exposed for 1 h at 328C to salicylate ( SAL ) , to induce MarA , or to paraquat ( PQ ) , to induce SoxS , and assayed for b-galactosidase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10564485" target="_blank">10564485</a>]<br></li>
<li>When the predicted soxbox sequence was deleted , induction by both paraquat and sodium salicylate was completely gone , indicating that MarA and SoxS activation was mediated through the same cis-acting site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19376854" target="_blank">19376854</a>]<br></li>
<li>Since MarA activates some promoters more effectively and SoxS activates others more effectively , we wished to determine what structural differences might account for this promoter discrimination ( 23 ) and whether these differences were related to DNA binding at the corresponding marboxes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21097628" target="_blank">21097628</a>]<br></li>
<li>To understand why MarA does not activate certain promoters as strongly as SoxS , we compared MarA , MarA mutants , and SoxS for their abilities to activate 16 promoters and to-bind their cognate marbox binding sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21097628" target="_blank">21097628</a>]<br></li>
<li>However , these activators differ in the extents to which they activate particular promoters , for example , SoxS activates fpr to a much greater extent than MarA does .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22515487" target="_blank">22515487</a>]<br></li>
<li>The plasmids pJLR70 , pRGM9818 , pRGM489 , and pRGM5009 were previously constructed by cloning SoxS , MarA , Rob and MarA E89A , respectively , in the original vector pRGM9817 ( Martin et al. , 2000 ; Rosner et al. , 2002 ; Martin & Rosner , 2011 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22515487" target="_blank">22515487</a>]<br></li>
<li>Moreover , to rule out any possible side effect of PQ treatment , strain M4458b was transformed with the plasmids pRGM9817 , pJLR70 , pRGM9818 , pRGM489 , and pRGM5009 , which respectively correspond to the vector alone or carrying SoxS , MarA , Rob , and MarA E89A ( a modiﬁed MarA protein with the substitution E89A which has been shown to act like SoxS in the preferential activation of the regulon genes ( Martin & Rosner , 2011 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22515487" target="_blank">22515487</a>]<br></li>
<li>When present , the concentration of MarA protein was 67 nM and the concentration of MalE-SoxS fusion protein was 200 nM .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8636021" target="_blank">8636021</a>]<br></li>
<li>Like MarA , the activity of SoxS seems to be controlled by regulation of its synthesis , which results from transcriptional regulation governed by the redox-sensing SoxR protein ( 2 , 21 , 35 , 45 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7896685" target="_blank">7896685</a>]<br></li>
<li>Furthermore , the transformation of this MarA - and Rob-deﬁcient strain with a plasmid that overexpresses MarA ( 28 ) increased mar : : lacZ expression fourfold , and this increase was marbox dependent ( 30 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8636021" target="_blank">8636021</a>]<br></li>
<li>Thus , according to these arguments , it should be possible to determine the degree of coincidence between the zwf soxbox and the DNA binding / transcriptional activation sites of MarA , TetD , and Rob by testing whether the overproduction of the respective proteins elevates b-galactosidase expression in vivo from our set of zwf-lacZ and zwf-gnd-lacZ fusions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7896696" target="_blank">7896696</a>]<br></li>
<li>Hence , it is possible that the divergence of the nfnB marbox , and more particularly of RE 2 , although allowing in vitro binding by MarA , SoxS and Rob , will effect discriminated regulation of gene expression by the three transcriptional proteins in a physiological context , i.e. within the cell .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12100559" target="_blank">12100559</a>]<br></li>
<li>This experiment con-ﬁrmed the transcriptional activity of artiﬁcially expressed MarA and Rob proteins ( Fig. 4B ) and veriﬁed that they do not activate yggX expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14594836" target="_blank">14594836</a>]<br></li>
<li>Genetic elimination of both MarA and Rob resulted in complete loss of activation , equal to that seen in marbox transversion mutants ( Fig. 5C ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9393706" target="_blank">9393706</a>]<br></li>
<li>On the other hand , Rob and MarA are similar with respect to both the extent of activation and the concentration required for maximum activation ( 19 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626315" target="_blank">8626315</a>]<br></li>
<li>Nevertheless , as reported here , MarA , SoxS and Rob showed highest afﬁnity for the marbox closer to or overlapping the -35 sequence , which is the primary cis-acting site if not the only , for activation of these genes ( Fawcett and Wolf , 1995 ; Li and Demple , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12100559" target="_blank">12100559</a>]<br></li>
<li>Despite overlapping regulation by SoxS , MarA , and Rob , the three systems exhibit differential control in terms of both the strength of their effects on individual genes and activation of some promoters uniquely ( e.g. , zwf by SoxS or MarA but not by Rob ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7896685" target="_blank">7896685</a>]<br></li>
<li>Recently , we demonstrated that activation of in vitro zwf transcription by MalE-SoxS ( 18 ) , and by MarA as well ( 19 ) , requires this same DNA binding element and that this soxbox / marbox can bring in vitro gnd transcription under control of these two activators ( 18 , 19 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626315" target="_blank">8626315</a>]<br></li>
<li>Like CAP , MarA , SoxS and Rob are ` ambidextrous ' ( Jair et al. , 1995 , 1996a , 1996b ) : when the marbox is located upstream of the ÿ35 hexamer for RNP binding ( as in the fpr and zwf promoters ) , activation requires interaction with a-CTD ( class I ) ; when the marbox overlaps the ÿ35 hexamer ( as in the fumC , micF and nfo promoters ) , a-CTD is not required for activation ( class II ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10564485" target="_blank">10564485</a>]<br></li>
<li>Sufﬁciency of Rob for in vitro transcriptional activation of zwf , fpr , fumC , micF , nfo , and sodA , promoters known to be activated by MalE-SoxS and MarA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626315" target="_blank">8626315</a>]<br></li>
<li>Thus , like MalE-SoxS ( 16 ) , MarA requires the a CTD for activation of zwf transcription but not for activation of fumC and micF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8522515" target="_blank">8522515</a>]<br></li>
<li>Thus , the increased production of MarA ( caused by some environmental chemicals , such as salicylate , inactivating its cognate repressor , MarR ) or SoxS ( caused by the inactivation of its repressor SoxR via its oxidation ) or the binding of coregulators such as dipyridyl ( 561 ) or some bile salts ( 558 ) to Rob activates the transcription of micF antisense RNA , decreasing OmpF synthesis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14665678" target="_blank">14665678</a>]<br></li>
<li>Despite the absence of functional MarA , MarR , SoxS , Rob , and EmrRAB , salicylate stimulated the wild-type and mutant promoter activities twofold ( Fig. 5D ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9393706" target="_blank">9393706</a>]<br></li>
<li>Surprisingly , salicylate induced a twofold increase in promoter activity from the lacZ fusions that was independent of sequences from 291 to 232 , of MarA , Rob , and SoxS activators and of MarR and EmrR repressors ( Fig. 5D ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9393706" target="_blank">9393706</a>]<br></li>
<li>If MarA and SoxS were the only proteins capable of activating the mar promoter via the marbox , no difference in activity between wild-type and defective marbox promoters should be seen in strains lacking MarA and uninduced for SoxS ( 27 , 38 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9393706" target="_blank">9393706</a>]<br></li>
<li>The b-galactosidase activities show that only when its marbox is in the F-orientation is the zwf promoter activated by MarA or SoxS ( Table 3 , top rows ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10564485" target="_blank">10564485</a>]<br></li>
<li>Average stimulation by Fis of mar : : lacZ in the absence or presence of activatorsa Activator Promoter None Rob SoxS MarA Wild type 0.8 1.5 1.5 1.6 Fis site mutants 0.7 0.8 1.0 1.0 Marbox mutants 0.7 0.7 0.7 0.8 a The Fis1-to-Fis2 b-galactosidase ratios in the presence of the indicated marbox activator for each of the three promoter types were averaged and divided by 0.72 , the average Fis1-to-Fis2 ratio for all the strains in the ﬁrst section of Table 4 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9393706" target="_blank">9393706</a>]<br></li>
<li>Transcription of the wild type and all of the mutant promoters tested was reduced approximately 25 % by the presence of Fis in the absence of the transcriptional-activators MarA , SoxS , and Rob ( Table 4 , no marbox activator ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9393706" target="_blank">9393706</a>]<br></li>
<li>MarA , SoxS , and Rob each activated the mar promoter 1.5-to 2-fold when it had a wild-type marbox but Fis was absent .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9393706" target="_blank">9393706</a>]<br></li>
<li>Both SoxS and MarA activated the promoter by greater than twofold ( Table 5 ) in the absence of Fis ( requiring only a functional marbox to do so ) , and the effects of both were FIG. 2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9393706" target="_blank">9393706</a>]<br></li>
<li>Although we have come a long way in the deﬁnition of the marbox sequence and its relationship with differential regulation by MarA , SoxS and Rob , more work still needs to be done to deﬁne and clarify all the different types of interactions that lead to a coordinated and effective cell response to different environmental stresses .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12100559" target="_blank">12100559</a>]<br></li>
<li>Unlike MarA or SoxS , Rob is present at a high constitutional level ( estimated to be around 5000 per cell ; Skarstad et al. , 1993 ; Kakeda et al. , 1995 ) , which translates to 2 -- 5 mM , and we are not aware of any experimental study showing that the steady-state level of Rob is altered physiologically in response to environmental signals .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12791142" target="_blank">12791142</a>]<br></li>
<li>Regardless of the detailed mechanism , our present study as well as that of Rosner et al. ( 2002 ) show clearly that regulation via Rob , involving the binding of effector ligands to the regulator protein , is mechanistically very different from regulation via the Mar or Sox system , which involves alteration of the levels of regulator MarA or SoxS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12791142" target="_blank">12791142</a>]<br></li>
<li>Different afﬁnities of MarA , SoxS and Rob for the promoters of these genes , probably dictated by differences among these transcription factor recognition sequences , lead to differential regulation of particular promoters by these proteins , therefore allowing the cell to respond differently to various stresses ( Martin et al. , 2000 ; Grifﬁth and Wolf , 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12100559" target="_blank">12100559</a>]<br></li>
<li>For instance , Rob has been shown to-bind and activate the zwf promoter in vitro but whole cell zwf regulation can not be activated by Rob , although the gene responds to SoxS and MarA ( Ariza et al. , 1995 ; Jair et al. , 1995 1996a ; b ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12100559" target="_blank">12100559</a>]<br></li>
<li>This observation is in agreement with reports of others which found that Rob bound in vitro more tightly than MarA and SoxS to most marboxes ( Martin et al. , 2002 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12100559" target="_blank">12100559</a>]<br></li>
<li>MarA , SoxS and Rob bind as monomers to an asymmetrical and degenerate 19 bp DNA sequence known as the marbox ( Martin et al. , 1999 ; Wood et al. , 1999 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18673442" target="_blank">18673442</a>]<br></li>
<li>To test whether either MarA or Rob could activate the transcription of yggX , we measured the relative level of yggX mRNA in cells in which the expression of MarA or Rob could be selectively activated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14594836" target="_blank">14594836</a>]<br></li>
<li>Evidence also suggests that the up-regulationof acrAB expression under general stress conditions is not likely to be mediated by the known global regulators , such as MarA or SoxS , although elevated levels of these proteins were shown to increase the transcription of acrAB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8821940" target="_blank">8821940</a>]<br></li>
<li>Third , unlike the general stress conditions , the constitutive overexpression of MarA or SoxS did not increase the transcription of acrR at all ( D. Ma , unpublished results ) , although it increased the transcription of acrAB , as shown in Table 2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8821940" target="_blank">8821940</a>]<br></li>
<li>Although MarA , SoxS and Rob could increase acrAB transcription when they were overexpressed , these gene products did not appear to be responsible for the acrAB induction by the general stress conditions such as 4 % ethanol , 0.5 M NaCl or the onset of stationary-phase growth in LB medium ( Table 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8821940" target="_blank">8821940</a>]<br></li>
<li>It would be inte-resting to determine under what conditions these acrAB homologues are induced and how they respond to the global stress-induced regulators , including MarA , SoxS and ( possibly ) Rob .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8821940" target="_blank">8821940</a>]<br></li>
<li>Based on the data presented , we assume that general stress conditions result in the generation of global transcription regulators , which are unlikely to be MarA , SoxS , or Rob , but could be homologues of them .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8821940" target="_blank">8821940</a>]<br></li>
<li>Also , the MarA homolog SoxS repressed its own expression in whole cells and bound its own promoter in vitro ( 31 ) ; no further studies of this observation have been reported .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14701822" target="_blank">14701822</a>]<br></li>
<li>First , three independently regulated but homologous proteins , MarA , SoxS and Rob , activate a common set of promoters which results in multiple antibiotic resistance , superoxide resistance and organic solvent tolerance ( for review , see Alekshun and Levy , 1997 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10564485" target="_blank">10564485</a>]<br></li>
<li>tolC is regulated in part by MarA , SoxS and Rob , three paralogous transcriptional-activators which bind a sequence called the marbox and which activate multiple antibiotic and superoxide resistance functions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18673442" target="_blank">18673442</a>]<br></li>
<li>Previously , overexpression of Rob was shown to induce multiple antibiotic and superoxide resistance phenotypes ( 3 ) , to activate expression of fumC , inaA , and sodA in vivo ( 3 ) , and to activate the transcription of six mar/soxRS regulon promoters in vitro by mechanisms that are strikingly similar to that of MarA and SoxS ( 22 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9393706" target="_blank">9393706</a>]<br></li>
<li>Since SoxS and MarA seem able to activate some of the same genes in vitro , we tested whether intact Rob or its isolated N-terminal 133-residue domain could activate phenotypic antibiotic resistance or speciﬁc genes for antibiotic resistance or oxidative-stress .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7896685" target="_blank">7896685</a>]<br></li>
<li>Assays of bgalactosidase due to the expression of micF : : lacZ fusions in strains M3710 and M3944 and inaA : : lacZ fusions in strains M4110 and M4113 showed 19 - , 16 - and 61-fold increases of fusion expression due to the upregulation of MarA , SoxS and Rob respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18673442" target="_blank">18673442</a>]<br></li>
<li>The consequence is that transcription of the global regulators , MarA and SoxS , are modulated by repression of AcrR at the predicted AcrR binding sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25176444" target="_blank">25176444</a>]<br></li>
<li>The concentrations of MarA and SoxS are as described in the legend of Fig. 2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21097628" target="_blank">21097628</a>]<br></li>
<li>The mechanism by which such small ( 13-kDa ) proteins as SoxS and MarA activate gene expression is unknown .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8282690" target="_blank">8282690</a>]<br></li>
<li>The signaling mechanisms that prevail in OmpR , MarA , and SoxS action are known .9,10,14 Activation of micF by Rob in response to cationic peptide antibiotics is not as well formulated .32 Transcriptional activators of micF Transcriptional regulators are known to activate micF expression in response to environmental factors ( Figure 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11601842" target="_blank">11601842</a>]<br></li>
<li>Although Rob , SoxS and MarA activate micF transcription in response to different environmental stimuli , there is some overlap in response .14 soxS is part of the soxRS regulon .66 SoxR is a sensor protein activated by oxidative-stress and in turn it transcriptionally activates soxS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11601842" target="_blank">11601842</a>]<br></li>
<li>11 and 13 ) , bile salts , and reactive oxygen species , MarA , in concert with global regulators Rob and SoxS , binds to the mar box sequence to activate transcription of the mar operon ( 14 -- 17 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25294881" target="_blank">25294881</a>]<br></li>
<li>The Rob homologs MarA and SoxS can activate transcription of a broad range of genes in vivo ( reviewed in references 2 and 14 ) , including sodA , fumC , micF , zwf , and inaA , suggesting broadly overlapping activities of these three regulators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10850996" target="_blank">10850996</a>]<br></li>
<li>Despite their broadly overlapping activities , MarA , SoxS , and Rob differentially activate the transcription of certain genes2 ,5 -- 12 , suggesting that different types of interactions occur with the varied promoter sequences bound by these transcription factors .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10802742" target="_blank">10802742</a>]<br></li>
<li>The related MarA , SoxS and Rob regulators bind to the same promoter element and activate micF transcription in response to various stimuli , including weak acids , oxidative-stress and certain antibiotics .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17007876" target="_blank">17007876</a>]<br></li>
<li>The expression of micF is also induced by toxic agents , such as paraquat and weak acids , by the SoxS , MarA , and Rob transcription factors , which bind to the same sequence element in the micF promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15466019" target="_blank">15466019</a>]<br></li>
<li>For example , the E. coli proteins MarA , Rob and SoxS all target genes involved in the oxidative-stress response and bind to closely related DNA sequences .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9813115" target="_blank">9813115</a>]<br></li>
<li>For instance , the expression of acrAB in E. coli is controlled by repressor protein AcrR , global activator protein MarA , SoxS , Rob and quorum sensing regulator LuxR homologue protein SdiA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15686558" target="_blank">15686558</a>]<br></li>
<li>Many bacteria also possess transcription factors , such as the homologous E. coli regulators MarA , Rob , and SoxS ( 3 , 4 ) , that respond to drugs by activating the expression of genes encoding proteins involved in antibiotic efﬂux and detoxiﬁcation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23010927" target="_blank">23010927</a>]<br></li>
<li>Moreover , expression of acrZ is coregulated with acrAB and tolC by MarA , Rob , and SoxS , a set of transcription factors that regulate cellular responses to antibiotics , detergents , and redox-cycling drugs ( 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23010927" target="_blank">23010927</a>]<br></li>
<li>However , it does not appear that modulation of acrAB expression in response to these environmental stresses operates through AcrR , MarA , or SoxS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8866468" target="_blank">8866468</a>]<br></li>
<li>We utilized microarray technology to identify genetic loci which differed in expression based on the presence or absence of the MarA , Rob , and SoxS regulators in response to conditions within the host .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22210763" target="_blank">22210763</a>]<br></li>
<li>However , the expression of these regulators was not affected by AcrS and AcrR ( data not shown ) , indicating that the acrAB repression by AcrR and AcrS is unlikely to be mediated by MarA , SoxS , or Rob .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18567659" target="_blank">18567659</a>]<br></li>
<li>It is known that acrAB expression is also controlled by the global regulators MarA , SoxS , and Rob ( 4 , 5 , 10 , 21 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18567659" target="_blank">18567659</a>]<br></li>
<li>Its expression is regulated by three XylS/AraC family regulators , MarA , SoxS and Rob .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12791142" target="_blank">12791142</a>]<br></li>
<li>Although MarA and SoxS regulation works by the alteration of their own expression levels , it was not known how Rob , which is constitutively expressed , exerts its regulatory action .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12791142" target="_blank">12791142</a>]<br></li>
<li>Unlike MarA , Rob and SoxS upregulated nfnB expression only when they were encoded on multicopy plasmids , but less efﬁciently than MarA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12100559" target="_blank">12100559</a>]<br></li>
<li>Transcription of nfnB is greatly increased by induced or constitutive chromosomal expression of MarA but is unaltered by constitutive expression of chromosomal SoxS and Rob .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12100559" target="_blank">12100559</a>]<br></li>
<li>When expressed from identical plasmid vectors , Rob and SoxS caused increased nfnB expression but it was considerably less than that produced by MarA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12100559" target="_blank">12100559</a>]<br></li>
<li>The genes encoding these proteins are activated by the homologous proteins MarA and SoxS , although the level of activation can vary for the different transcription factors .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10802742" target="_blank">10802742</a>]<br></li>
<li>Transcriptionof acrAB is increased in MarA - and SoxS-constitutivemutants , and in a strain overproducing Rob The transcription of acrAB is increased in E. coli strain JHC1069 ( Ma et a / .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8821940" target="_blank">8821940</a>]<br></li>
<li>The ﬁrst four were greatly enhanced on overexpression of MarA and SoxS ( growth in IPTG ) or on activation of Rob in strain M4113 by DIP whereas the latter two products were diminished .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18673442" target="_blank">18673442</a>]<br></li>
<li>The SoxS paralogs MarA and Rob are not sufﬁcient for activation of yggX .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14594836" target="_blank">14594836</a>]<br></li>
<li>Although puriﬁed SoxS and Rob proteins , like MarA , activated nfnB transcription in vitro , only constitutive expression of chromosomal marA , but not of soxS and rob genes , affected nfnB expression in whole cells .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12100559" target="_blank">12100559</a>]<br></li>
<li>Of note , Rob and SoxS also increased transcription in vitro and showed preferential binding to the same nfnB promoter sequence as MarA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12100559" target="_blank">12100559</a>]<br></li>
<li>Rob-dependent activation of in vitro transcription of six promoters also activated by SoxS and MarA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626315" target="_blank">8626315</a>]<br></li>
<li>These results indicated that MarA , SoxS and Rob were sufﬁcient for transcription activation in vitro of nfnB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12100559" target="_blank">12100559</a>]<br></li>
<li>MarA , SoxS and Rob activation of nfnB transcription in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12100559" target="_blank">12100559</a>]<br></li>
<li>However , the observation that in vivo zwf transcription can be activated by MarA and SoxS , but not by Rob ( 5 ) , shows that control by these regulators may not overlap completely .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10850996" target="_blank">10850996</a>]<br></li>
<li>Moreover , transcription of marA is activated by MarA and by the MarA homologs SoxS and Rob , mostly in the presence of the helper protein Fis ( 32 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20211899" target="_blank">20211899</a>]<br></li>
<li>However , the observation that SoxS binds to , and seems to negatively regulate , its own promoter ( 33 ) raises the possibility that MarA and Rob could also negatively regulate soxS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8636021" target="_blank">8636021</a>]<br></li>
<li>Binding of MarA , SoxS , and Rob to the rob Promoter -- MarA , SoxS , and Rob bound to the full-length rob promoter ( spanning 234 to 110 ) ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16478729" target="_blank">16478729</a>]<br></li>
<li>Protein-DNA interactions show that SoxS and Rob , like MarA , bind the same marbox in the rob promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16478729" target="_blank">16478729</a>]<br></li>
<li>SDS-PAGE analysis of purified proteins and binding of MarA , SoxS , and Rob to the rob promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16478729" target="_blank">16478729</a>]<br></li>
<li>From an analysis of the promoters that are activated by these proteins , it is clear that the orientation of the marbox and its distance from the -10 hexamer of the promoter are critical in determining whether MarA , SoxS and Rob can activate the promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18673442" target="_blank">18673442</a>]<br></li>
<li>Like MarA , puriﬁed His6-Rob and His6-SoxS bound to marbox 2 in gel mobility assays .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12100559" target="_blank">12100559</a>]<br></li>
<li>Activation by the monomeric proteins MarA , SoxS and Rob requires that they bind to a marbox located in a particular orientation upstream of the TSS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18673442" target="_blank">18673442</a>]<br></li>
<li>MarA and Rob can activate several of the SoxS-activated genes , since both MarA and Rob recognize the same DNA sequence for binding ( 26 , 27 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14594836" target="_blank">14594836</a>]<br></li>
<li>Two previously identiﬁed tolC promoters , p1 and p2 , are not regulated by MarA , SoxS or Rob but p2 is activated by EvgAS and PhoPQ which also regulate other functions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18673442" target="_blank">18673442</a>]<br></li>
<li>As shown here , p3 and p4 are activated by MarA , SoxS and Rob , with each activator being regulated by different environmental signals .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18673442" target="_blank">18673442</a>]<br></li>
<li>The inaA gene is activated by SoxS , MarA , and Rob , with MarA being the strongest activator ( 34 , 43 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14594836" target="_blank">14594836</a>]<br></li>
<li>For example , the nfo gene , coding for endonuclease IV , is activated by SoxS but not by MarA or Rob ( 17 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14594836" target="_blank">14594836</a>]<br></li>
<li>Similarly , the gshB gene , coding for a glutathione synthase , is activated by MarA but not by SoxS or Rob ( 5 , 39 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14594836" target="_blank">14594836</a>]<br></li>
<li>In particular , the synthesis was high in JA300 derivatives in which MarA production was elevated rather than that of Rob and SoxS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9473050" target="_blank">9473050</a>]<br></li>
<li>Nevertheless , when induced by paraquat , the activity of SoxS was not dependent on either MarA or Rob ( Table 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9393706" target="_blank">9393706</a>]<br></li>
<li>Rob is most closely related to the SoxS , MarA , and TetD proteins2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10802742" target="_blank">10802742</a>]<br></li>
<li>Therefore , we chose to pursue znuACB and its relation to MarA , Rob , and SoxS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22210763" target="_blank">22210763</a>]<br></li>
<li>The TolC level was regulated positively by MarA , Rob , or SoxS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9473050" target="_blank">9473050</a>]<br></li>
<li>OmpX synthesis is likely to be regulated positively by MarA , Rob , or SoxS ( Fig. 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9473050" target="_blank">9473050</a>]<br></li>
<li>In addition , both AcrZ -- SPA and AcrB protein levels were elevated when MarA , Rob , or SoxS were overproduced from plasmids ( Fig. 5C ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23010927" target="_blank">23010927</a>]<br></li>
<li>We suggest that rising levels of NADH are the major reasons for the induction of SoxS , MarA and Rob stress responses in the cells lacking TolC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20545840" target="_blank">20545840</a>]<br></li>
<li>‡ Miller units were also determined in the presence of the vector pRGM9817 and fold induction was evaluated in the presence of the vector carrying SoxS , MarA , Rob , and MarA E89A .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22515487" target="_blank">22515487</a>]<br></li>
<li>b Rob presence was due to chromosomal wild-type rob expression , SoxS presence was due to paraquat induction of the otherwise quiescent soxRS , and MarA presence was due to plasmid p37 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9393706" target="_blank">9393706</a>]<br></li>
<li>However , when we compared the induction mediated by plasmid-speciﬁed SoxS and Rob with that of MarA , a striking difference was noticed ( Fig. 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12100559" target="_blank">12100559</a>]<br></li>
<li>However , in contrast to SoxS and MarA , whose syntheses are induced by speciﬁc environmental stimuli and elicit a clear defense response , Rob is expressed constitutively and its normal function is unknown .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626315" target="_blank">8626315</a>]<br></li>
<li>Thus , it would be of great interest to understand the interaction of multiple regulators such as SoxS , MarA , and perhaps Rob at single sites that could respond differentially to various environmental signals .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8282690" target="_blank">8282690</a>]<br></li>
<li>The poorly homologous C-terminal sequence in MarA is structured19 , at least when the protein is bound to DNA and in the lower salt concentration crystallization buffer .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19422057" target="_blank">19422057</a>]<br></li>
<li>This is the ﬁrst report of functional interactions between MarA and other gene regulators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18083817" target="_blank">18083817</a>]<br></li>
<li>This solution structure is discussed in the context of extensive biochemical and physiological data on AraC and with respect to the DNA-bound structures of the MarA and Rob homologs .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19422057" target="_blank">19422057</a>]<br></li>
<li>Models of MarA bound to DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10564485" target="_blank">10564485</a>]<br></li>
<li>Overlay of AraC and MarA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19422057" target="_blank">19422057</a>]<br></li>
<li>Thus , information about MarA may provide insights as to how other proteins of this large family bind DNA and activate transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10873449" target="_blank">10873449</a>]<br></li>
<li>This DNA-binding domain has also been shown to be involved in transcription activation in a number of AraC/XylS family proteins including Ada , RhaS , AraC , MelR , MarA , SoxS , XylS , and UreR ( 1 , 4 , 5 , 15 , 16 , 20 -- 22 , 37 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15342598" target="_blank">15342598</a>]<br></li>
<li>When MarA was induced by treatment with-salicylate , the stimulation was smaller , especially in the micF core promoter constructs .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10564485" target="_blank">10564485</a>]<br></li>
<li>Although MarA and the mar operon repressor , MarR , bound to independent sites , they competed for promoter DNA in band shift experiments .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8636021" target="_blank">8636021</a>]<br></li>
<li>Since MarR complexes with the promoter region are disrupted when MarR binds to salic-ylate ( 29 ) , an inducer of the mar operon ( 7 ) , the effect of salicylate on MarA complexes was examined .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8636021" target="_blank">8636021</a>]<br></li>
<li>DNase I footprint of the mar promoter region in the absence or presence of 50 nM Fis and/or 100 nM MarA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9393706" target="_blank">9393706</a>]<br></li>
<li>DNase I footprint analysis of the mar promoter in the presence of MarA and MarR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8636021" target="_blank">8636021</a>]<br></li>
<li>Optimal activation of the mar promoter by MarA occurred when the spacing was the normal 16 or 15 bp and diminished progressively as the spa-cing was increased or decreased ( Table 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10564485" target="_blank">10564485</a>]<br></li>
<li>Thus , maximum activation of the mar regulon promoter inaA depends on the autostimulatory effect of MarA on the mar operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8636021" target="_blank">8636021</a>]<br></li>
<li>Thus , when MarA is bound to the marbox in these three different promoter con ® gurations , it presents different surfaces to RNAP bound at the promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10873449" target="_blank">10873449</a>]<br></li>
<li>This indicates that the binding of MarA competes with the binding of MarR even though MarA and MarR bind to different sites in the promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8636021" target="_blank">8636021</a>]<br></li>
<li>Thus , MarA and MarR appear to-bind to independent sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8636021" target="_blank">8636021</a>]<br></li>
<li>Computer analysis of MarR , MarA , and MarB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8383113" target="_blank">8383113</a>]<br></li>
<li>Proximity of the bound MarA and Fis proteins is suggested by the disappearance of a MarA hypersensitive site 39 to bp 275 on the template strand when the promoter fragment was digested in the presence of both MarA and Fis ( Fig. 2B ) , consistent with some interaction of Fis with the sequences ﬂanking the binding site ( 34 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9393706" target="_blank">9393706</a>]<br></li>
<li>Hydroxyl-radical footprinting with MarA revealed a hypersensitive site 39 to position 264 of the template strand when MarA was bound to the promoter both in the presence and absence of Fis ( data not presented ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9393706" target="_blank">9393706</a>]<br></li>
<li>Given that binding by MarA in the EMSA studies was detected to marbox 2 but not to marbox 1 , site-directed mutagenesis was carried out in areas of marbox 2 in construct pSP-nfnB1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12100559" target="_blank">12100559</a>]<br></li>
<li>The presence of a MarA binding site upstream of the acrAB promoter strongly suggests that overexpression of MarA in TP985 ( mppA : : Cm ) should lead to increased levels of the AcrAB proteins that constitute the efﬂux pump and hence to the MAR phenotype .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10438753" target="_blank">10438753</a>]<br></li>
<li>In this case the binding coef-ﬁcient of MarA to a DNA fragment should be altered in the presence of Fis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9393706" target="_blank">9393706</a>]<br></li>
<li>The prior presence of either MarA or RNAP did not inhibit binding of the other protein to the rob promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16478729" target="_blank">16478729</a>]<br></li>
<li>The marbox was originally identiﬁed as a sequence whose sensitivity to DNase I was altered by the presence of MarA ( 27 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9393706" target="_blank">9393706</a>]<br></li>
<li>The basis for the activation of these promoters is the presence of a 20-bp binding site for MarA and SoxS ( the marbox or soxbox ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21097628" target="_blank">21097628</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_313">
The binding of MarA to the marbox and that of Fis to its site were independent of each other .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9393706" target="_blank">9393706</a>]<br></label>
<input id="act_similarity_313", type="checkbox">
<div class="ACT_similar">
<li>The binding of Fis to this Fis site was independent of the binding of MarA to the marbox .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9393706" target="_blank">9393706</a>]<br></li></div>
<li>In contrast , we show here that CRP-cAMP contributes to MDR in E. coli in the presence of MarA and the AcrAB-TolC pump .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20211899" target="_blank">20211899</a>]<br></li>
<li>In contrast we found that H-NS increases MDR in E. coli in the presence of MarA and an intact AcrAB-TolC pump .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20211899" target="_blank">20211899</a>]<br></li>
<li>Under each condition , we compared the regulator-inactivated strain/parental strain ratio obtained in the presence of MarA with that obtained without MarA ( Table 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18083817" target="_blank">18083817</a>]<br></li>
<li>Mixtures of the test and gnd reference promoters were transcribed under standard conditions in the absence and the presence of native MarA protein ( 2 pmol ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8522515" target="_blank">8522515</a>]<br></li>
<li>Moreover , the same cleavage patterns were observed in the presence or absence of MarA ( Fig. 7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16478729" target="_blank">16478729</a>]<br></li>
<li>The effect of MarA is expressed as the ratio of purA transcription in the presence of MarA divided by purA transcription in its absence , normalized for any differences in the transcription of gnd .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14701822" target="_blank">14701822</a>]<br></li>
<li>The average ratios among these transcripts in the absence and presence of MarA represented the level of repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16478729" target="_blank">16478729</a>]<br></li>
<li>The assays were performed in the absence of MarA ( without IPTG ) or in the presence of MarA ( with IPTG ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18083817" target="_blank">18083817</a>]<br></li>
<li>However , in the presence of MarA the rob transcript levels decreased over time , but as expected the levels of the gnd transcript ( the control ) remained unaffected ( Fig. 6C ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16478729" target="_blank">16478729</a>]<br></li>
<li>When reactions were carried out in the presence of RNAP alone as compared with 200 nM puriﬁed His6-MarA , we detected MarA upregulation of nfnB transcription ( Fig. 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12100559" target="_blank">12100559</a>]<br></li>
<li>In the presence of 200 nM MarA , there was a 5 -- 6-fold activation in nfnB transcription , as expected ( Fig. 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14701822" target="_blank">14701822</a>]<br></li>
<li>Therefore , gene-speciﬁc alteration of MarA activity by other proteins seems not unusual , which may explain why some genes are differently activated in vivo by the same concentrations of MarA , especially when these differences are poorly correlated with the in vitro afﬁnity of MarA for their promoters ( 34 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20211899" target="_blank">20211899</a>]<br></li>
<li>Given the proximity of iscR to the Fe-S cluster assembly genes iscSUA , the similarity of IscR to MarA , and the possibility that IscR contains a Fe-S cluster , we characterized the properties of IscR and investigated whether this protein regulates the transcription of the isc gene cluster .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11742080" target="_blank">11742080</a>]<br></li>
<li>Although the transcription factor MarA is not directly regulated by FNR or ArcA , four genes that are activated by MarA ( acnA , fumC , sodA , zwf ) are repressed by ArcA and/or FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23980183" target="_blank">23980183</a>]<br></li>
<li>The effect of the a CTD on the ability of puriﬁed MarA to activate transcription from the zwf , fumC , and micF templates was tested ( Fig. 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8522515" target="_blank">8522515</a>]<br></li>
<li>When MarA was preincubated with the truncated rob promoters ( pRobF4 or pRobF5 ) for 5 min at 37 ° C before the addition of RNAP , repression was detected ( average repression ratio , 0.37 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16478729" target="_blank">16478729</a>]<br></li>
<li>In contrast , when RNAP was preincubated with the DNA for 5 min at 37 ° C before the addition of MarA , no repression ( average repression ratio , 1.2 -- 1.4 ) was observed ( Fig. 6A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16478729" target="_blank">16478729</a>]<br></li>
<li>Transversions of this sequence are present in mutants 6 to 9 and result in reduced promoter activity in vivo and reduced afﬁnity for MarA in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9393706" target="_blank">9393706</a>]<br></li>
<li>For example , MarA binds tightly to the marRAB promoter in vitro yet stimulates transcription by only 3-fold ; no signiﬁcant binding to inaA can be demonstrated by gel shift experiments , yet transcription is stimulated 5-fold .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21097628" target="_blank">21097628</a>]<br></li>
<li>For hdeA , even though repression of transcription was seen in vitro , EMSA experiments showed minimal binding of MarA to the hdeA promoter region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14701822" target="_blank">14701822</a>]<br></li>
<li>Therefore , both MarA and Rob are able to repress the rob promoter directly in vitro , and this repression occurs at the level of transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16478729" target="_blank">16478729</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_336">
Effect of mutations in the purA promoter upon repression in vitro by MarA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14701822" target="_blank">14701822</a>]<br></label>
<input id="act_similarity_336", type="checkbox">
<div class="ACT_similar">
<li>Effect of mutations in the hdeA promoter upon repression in vitro by MarA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14701822" target="_blank">14701822</a>]<br></li></div>
<li>We tested the ability of puriﬁed MarA to activate transcription of the nfnB gene in vitro by mixing the nfnB promoter-containing DNA ( 261 bp ) and the control gnd promoter-containing DNA ( 167 bp ) ( unresponsive to MarA regulation ) ( Jair et al. , 1995 ; 1996a ; b ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12100559" target="_blank">12100559</a>]<br></li>
<li>This protein , engineered to facilitate purification and hereafter referred to as MarA , had been shown previously to activate nfnB transcription in vitro ( 20 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14701822" target="_blank">14701822</a>]<br></li>
<li>Effect of MarA on transcription in vitro of purA , hdeA , and nfnB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14701822" target="_blank">14701822</a>]<br></li>
<li>In the case of purA , marbox 2 was defined as critical by its affinity for MarA in EMSA and by the fact that point mutations in RE1 of marbox 2 prevented both the mobility shift and the repression by MarA in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14701822" target="_blank">14701822</a>]<br></li>
<li>In addition the DNA and MarA contacts found to be crucial in activation ( 19 , 26 , 28 , 37 ) are also important for repression ( 20 ) as mutations within RE1 in both the purA and rob marboxes were sufficient to abolish MarA binding and , in the case of purA , to eliminate MarA-mediated repression in vitro ( 20 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16478729" target="_blank">16478729</a>]<br></li>
<li>As for the hdeA and purA promoters , which are repressed by MarA , the rob marbox is also in the `` backward '' orientation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16478729" target="_blank">16478729</a>]<br></li>
<li>Purified MarA Repressed Transcription of purA and hdeA in Vitro -- To determine whether repression was direct or indirect , we tested the effect of purified His - MarA protein on transcrip-6 tion of purA and hdeA in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14701822" target="_blank">14701822</a>]<br></li>
<li>Possibly repression in vivo of hdeA by MarA is aided by a cellular product that is constitutive or is induced by MarA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14701822" target="_blank">14701822</a>]<br></li>
<li>It was not known whether repression by MarA was also direct .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14701822" target="_blank">14701822</a>]<br></li>
<li>The physiological significance of the repression of purA and hdeA by MarA is not known .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14701822" target="_blank">14701822</a>]<br></li>
<li>Both sRNAs regulate the mRNA encoding MarA , which is involved in the response to antimicrobial compounds and oxidative-stress ( 46 ) , and of the mRNA for the β-N-acetylglucosaminidase NagZ , which permits resistance to β-lactams in Pseudomonas aeruginosa ( 48 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23980183" target="_blank">23980183</a>]<br></li>
<li>To begin dissecting the regulatory cascade of the response to NaSal , we expressed MarA in the absence of exogenous toxic agents .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11395452" target="_blank">11395452</a>]<br></li>
<li>At the level of global regulation , acrAB transcription is activated by MarA ( 3 , 29 ) in response to exposure to various chemicals , such as salicylate ( 2 , 7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12107134" target="_blank">12107134</a>]<br></li>
<li>Interruptionof marA by Tn5 insertion results in drug hypersusceptibility.Based on these and other results , Cohen et al. ( 1993a ) have proposed that MarA acts as a global transcription activator which is produced in increased amounts in response to antibiotic stress .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8821940" target="_blank">8821940</a>]<br></li>
<li>~ Induction Ratio of LacZa 4 % stationary 0.5 M Increased transcription of acrAB , caused by the general stress conditions , is not mediated by MarA or soxs acrAB transcription becomes elevated by the general stress conditions ( Ma et al. , 1995 ) , but this response occurs in the absence of functional AcrR , as described above .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8821940" target="_blank">8821940</a>]<br></li>
<li>MarA responds to weak acids such as salicylate67 and aryloxoalcanoic acids ,68 and certain anti-biotics .31 marA is part of the marRAB locus which is controlled by the repressor MarR .14 During the signaling process , salicylate binds to MarR and inactivates its activity .69 This results in the derepression of marA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11601842" target="_blank">11601842</a>]<br></li>
<li>As there are ª250 mole-cules of MarA per cell in the repressed state and 10 times that amount when derepressed by salicylate treatment ( Martin et al. , 2002 ) , and as MarA turns over with a halftime of ª1 min ( Grifﬁth et al. , 2004 ) , we calculate that , if MarR were synthesized at the same rate as MarA , there should be ª5000 molecules of MarR made per generation in the repressed state and 50 000 molecules in the derepressed state .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15225313" target="_blank">15225313</a>]<br></li>
<li>Using gene arrays , Pomposiello et al. ( 33 ) found that overexpression of MarA from an isopropyl - - D-thiogalactopyrano-side ( IPTG ) - inducible plasmid repressed hdeAB and other genes involved in acid resistance ( gadA , gadB , gadC , hdeD , and slp ) , whereas sodium salicylate activated these same genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18083817" target="_blank">18083817</a>]<br></li>
				</div></ul><br>
			<label class="collapse_DOM" for="_2"><font size="5px">MarA protein domains</font></label>
				<input id="_2" type="checkbox">
				<div class="DOM"><br><ul><li>Gene Description b0958 b1004 b0707 b0719 b0850 b1003 b1198 b2597 b2946 b3160 b3516 b3555 b4367 b4378 b3098 sulA wrbA ybgA ybgD ybjC yccJ ycgC yﬁA yggJ yhbW yhiX yiaG yjjS yjjV yqjD Inhibits cell division and ftsZ ring formation Affects association between Trp-repressor and operators in stationary phase Function unknown Putative ﬁmbria-like protein Function unknown Function unknown Function unknown Ribosome-associated factor , stabilizes ribosomes against dissociation Function unknown Putative monooxygenase Putative-transcriptional-regulator ( AraC/XylS-family ) Putative-transcriptional-regulator Ferric hydroxamate transport Putative hydrolase Function unknown b3734 b3731 b3736 b3735 b3739 b0295 b3261 b3340 b2215 b0565 b4201 b3259 b4245 b3984 b3317 b3320 b3319 b3308 b3305 b4203 b3985 b3983 b3986 b3231 b3310 b3301 b3313 b3294 b3304 b3186 b3315 b3318 b3309 b3185 b3637 b3312 b3302 b1717 b3299 b3295 b0169 b3314 b3296 b3303 b3306 b3230 b3321 b3297 b3307 b2609 b3311 NaSal-down-regulated 0.4 0.4 0.3 0.3 0.4 0.4 0.2 0.4 0.5 0.3 0.4 0.3 0.4 0.3 0.3 0.5 0.3 0.4 0.3 0.2 0.3 0.4 0.3 0.4 0.3 0.4 0.2 0.3 0.4 0.3 0.1 0.4 0.3 0.4 0.4 0.3 0.3 0.4 0.3 0.4 0.4 0.4 0.4 0.3 0.3 0.3 0.3 0.5 0.3 0.3 0.3 atpA atpC atpF atpH atpI b0295 ﬁs fusA ompC ompT priB prmA pyrB rplA rplB rplC rplD rplE rplF rplI rplJ rplK rplL rplM rplN rplO rplP rplQ rplR rplU rplV rplW rplX rpmA rpmB rpmC rpmD rpmI rpmJ rpoA rpsB rpsC rpsD rpsE rpsH rpsI rpsJ rpsK rpsN rpsP rpsQ Membrane-bound ATP synthase , F1 sector , alpha subunit Membrane-bound ATP synthase , F1 sector , epsilon subunit Membrane-bound ATP synthase , F0 sector , subunit b Membrane-bound ATP synthase , F1 sector , delta subunit Membrane-bound ATP synthase subunit , F1-F0-type proton-ATPase Function unknown Transcriptional activator for rRNA operons Fusidic acid resistance ; protein chain elongation factor G Outer-membrane-protein , porin Outer-membrane-protein , protease VII Primosomal protein N Methyltransferase for 50S subunit L11 protein modiﬁcation Aspartate transcarbamylase , catalytic subunit 50S ribosomal subunit protein L1 50S ribosomal subunit protein L2 50S ribosomal subunit protein L3 50S ribosomal subunit protein L4 50S ribosomal subunit protein L5 50S ribosomal subunit protein L6 50S ribosomal subunit protein L9 50S ribosomal subunit protein L10 50S ribosomal subunit protein L11 50S ribosomal subunit protein L7/L12 50S ribosomal subunit protein L13 50S ribosomal subunit protein L14 50S ribosomal subunit protein L15 50S ribosomal subunit protein L16 50S ribosomal subunit protein L17 50S ribosomal subunit protein L18 50S ribosomal subunit protein L21 50S ribosomal subunit protein L22 50S ribosomal subunit protein L23 50S ribosomal subunit protein L24 50S ribosomal subunit protein L27 50S ribosomal subunit protein L28 50S ribosomal subunit protein L29 50S ribosomal subunit protein L30 50S ribosomal subunit protein A ( L35 ) 50S ribosomal subunit protein X ( L36 ) RNA polymerase , alpha subunit 30S ribosomal subunit protein S2 30S ribosomal subunit protein S3 30S ribosomal subunit protein S4 30S ribosomal subunit protein S5 30S ribosomal subunit protein S8 30S ribosomal subunit protein S9 30S ribosomal subunit protein S10 30S ribosomal subunit protein S11 30S ribosomal subunit protein S14 30S ribosomal subunit protein S16 30S ribosomal subunit protein S17 Continued on following page For those experiments involving induction of the SoxS and MarA transcription factors , we performed a single experiment .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11395452" target="_blank">11395452</a>]<br></li>
<li>RhaS E181 , however , aligns with a residue on MarA , where alanine substitution resulted in a severe defect both at promoters where MarA binds overlapping the 235 region and at promoters where MarA binds further upstream ( 21 ) , suggesting that this residue may have a role other than interaction with s70 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10940041" target="_blank">10940041</a>]<br></li>
<li>van der Waals contacts are made between Trp-42 and Gln-45 of MarA and bases at consensus positions C-5 , A-6 and C-7 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10564485" target="_blank">10564485</a>]<br></li>
<li>Two conserved thymines within the B-box ( positions 4 ´ and 5 ´ ) have been suggested to contribute to sequence specific interactions with MarA in the form of van der Waals contacts to the C5 methyl groups13 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10802742" target="_blank">10802742</a>]<br></li>
<li>Both Gln45 of MarA and Gln39 of Rob make van der Waals contacts with the complement of C7 in the mar and micF soxboxes ; however , a T is found at this position in three of the four other soxboxes with mismatches at this position ( 28 ) , suggesting that such a change is tolerated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11741843" target="_blank">11741843</a>]<br></li>
<li>The complement of A6 of the mar soxbox forms van der Waals interactions with both Gln45 and Trp42 residues of MarA , while the complement of C7 interacts with Gln45 of MarA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11741843" target="_blank">11741843</a>]<br></li>
<li>MarA has three internal hydrophobic regions that are critical for function Alanine substitutions in residues that participate in the formation of the two hydrophobic cores of the MarA HTH motifs ( Leu44 , Phe48 , Met65 , Ile68 , Leu72 , Ile79 , Leu82 , Leu94 , Phe98 and Phe102 ) displayed greatly decreased activity ( Figure 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10873449" target="_blank">10873449</a>]<br></li>
<li>We therefore took advantage of the two highly conserved marbox `` recognition elements , '' RE1 and RE2 , of 4 bp each which are particularly critical for activation by SoxS ( 27 ) and presumably by MarA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14701822" target="_blank">14701822</a>]<br></li>
<li>An additional critical hydrophobic interaction was detected by substitutions for Leu24 and Lys64 which severely reduced MarA activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10873449" target="_blank">10873449</a>]<br></li>
<li>The involvement of the negatively charged Glu77 and hydrophobic Leu80 in class II activation suggests that the interaction between MarA and RNAP may differ from the AR2-RNAP interaction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10873449" target="_blank">10873449</a>]<br></li>
<li>Given that it lacks its dimerization domain , we predicted that RhaS-CTD would function as a monomer , similar to MarA ( 28 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17513476" target="_blank">17513476</a>]<br></li>
<li>Briefly , each protein ( RNAP or MarA ) was preincubated for 5 min 2 The abbreviations used are : RACE , rapid amplification of cDNA ends ; EMSA , electro-phoretic mobility shift analyses ; RNAP , RNA polymerase ; IPTG , isopropyl 1-thio - - D-galactopyranoside ; RE , recognition elements ; dCTD , - C-terminal domain of RNA polymerase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16478729" target="_blank">16478729</a>]<br></li>
<li>MalESoxS and MarA require the C-terminal domain of the RNA polymerase a subunit for activation of the zwf promoter but not for the fumC and micF promoters ( 18 , 19 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626315" target="_blank">8626315</a>]<br></li>
<li>Like MalE-SoxS , MarA ( i ) activated the transcription of zwf , fpr , fumC , micF , nfo , and sodA ; ( ii ) required a 21-bp `` soxbox '' sequence to activate zwf transcription ; and ( iii ) was `` ambidextrous , '' i.e. , required the C-terminal domain of the a subunit of RNA polymerase for activation of zwf but not fumC or micF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8522515" target="_blank">8522515</a>]<br></li>
				<label class="DOM_collapse_similarity" for="dom_similarity_15">
Versatility of the carboxy-terminal domain of the subunit of RNA polymerase in transcriptional activation : use of the DNA contact site as a protein contact site for MarA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21097628" target="_blank">21097628</a>]<br></label>
<input id="dom_similarity_15", type="checkbox">
<div class="DOM_similar">
<li>Versatility of the carboxy-terminal domain of the alpha subunit of RNA polymerase in transcriptional activation : use of the DNA contact site as a protein contact site for MarA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18366439" target="_blank">18366439</a>]<br></li></div>
<li>We previously demonstrated that in vitro transcriptional activation by MarA at the zwf promoter requires interaction with the carboxy-terminal domain of the a subunit of RNA polymerase ( 21 ) and inferred that MarA activates the mar promoter by enhancing the binding of RNA polymerase ( 27 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9393706" target="_blank">9393706</a>]<br></li>
<li>We surmise , from the distance of this marbox ( centered at 261.5 ) to the putative 235 transcriptional signal ( 17 , 18 ) , that activation of mar by MarA requires interaction with the carboxy-terminal domain of the alpha subunit of RNA polymerase , i.e. , that MarA is a class I activator ( 16 ) of mar as it is for zwf .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8636021" target="_blank">8636021</a>]<br></li>
<li>Puriﬁed near-native MarA and MarR ( each with a single histidine residue N terminal to the native protein ) and the hybrid MalE-SoxS proteins were prepared as previously described ( 9 , 18 , 29 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8636021" target="_blank">8636021</a>]<br></li>
<li>MarA was overexpressed as a histidinetagged fusion protein , puriﬁed , cleaved with thrombin ( leaving one N-terminal histidine residue ) , and renatured .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8522515" target="_blank">8522515</a>]<br></li>
<li>Certain members of the family are small monomeric proteins which lack the N-terminal domain , e.g. MarA and Rob ( although Rob has an unusual Cterminal extension ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15066032" target="_blank">15066032</a>]<br></li>
<li>Most family members contain one or more domains in addition to the AraC/XylS-family domain , but a few family members consist only of this single domain , such as MarA and SoxS ( 13 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17513476" target="_blank">17513476</a>]<br></li>
<li>These small proteins ( SoxS , 107 residues ; MarA , 127 residues ) are homologous to the C terminus of the XylS-AraC-family of proteins and are more closely related to a ; 100-residue segment in the N terminus of Rob protein , which binds the right arm of the replication origin , oriC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7896685" target="_blank">7896685</a>]<br></li>
<li>However , in Rob the 100-residue domain homologous to SoxS and MarA is located at the N terminus of the protein ; the C-terminal 175-residue segment of Rob is not related to XylS-AraC-family members ( 40 ) ( Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7896685" target="_blank">7896685</a>]<br></li>
<li>Given that AarP , MarA , PqrA , RamA , SoxS , and TetD , the shortest members of the XylS family ( 106 to 166 amino acids long ) , consist mainly of the homologous segment , the stretch of conserved residues most probably contains all the domains necessary for these regulators to interact with target DNA sequences and RNA polymerase and thus activate transcription from target promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9409145" target="_blank">9409145</a>]<br></li>
<li>MarA is substantially shorter than other members of this family ( except SoxS ) , whose sizes range from 265 residues ( Rns ) to 321 residues ( XylS ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8383113" target="_blank">8383113</a>]<br></li>
<li>While the A2 is in a position that is not highly conserved and does not interact directly with either the MarA or Rob protein , the T6 and A7 mismatches lie within the GCAC domain at positions that have been shown to be protein contact sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11741843" target="_blank">11741843</a>]<br></li>
<li>Similar antibiotic resistance was conferred by expression of a Rob fragment containing only the N-terminal 123 residues that constitute the SoxS-MarA homology .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7896685" target="_blank">7896685</a>]<br></li>
<li>These residues are highly conserved in the MarA homologs , SoxS and Rob , suggesting a common role in overall protein structure or transcriptional activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10873449" target="_blank">10873449</a>]<br></li>
<li>Furthermore , these residues are poorly conserved in SoxS and Rob , suggesting a unique interaction between MarA and RNAP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10873449" target="_blank">10873449</a>]<br></li>
<li>The full length MarA protein and Rob 's analogous N-terminal domain both function as transcriptional activators6 ,9 , implying that similar features of these structurally homologous proteins are directly involved in gene activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10802742" target="_blank">10802742</a>]<br></li>
<li>The homology to SoxS and MarA is restricted to the N-terminal domain of Rob ( 40 ) , which suggests that a common regulatory activity might reside in this region of the protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7896685" target="_blank">7896685</a>]<br></li>
<li>The N-terminal domain of Rob shares similarity with MarA and SoxS , which is in contrast to the other members of the subgroup that share homology within the C-terminal domain ( 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16478729" target="_blank">16478729</a>]<br></li>
<li>The helix ends in the MarA and Rob structures are at identical positions + / − 1 residue of the indicated positions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19422057" target="_blank">19422057</a>]<br></li>
<li>The complex reveals two helix-turn-helix ( HTH ) motifs within Rob 's N-terminal domain , an arrangement similar to the MarA protein13 , and a unique C-terminal domain that is structurally similar to the E. coli galactose-1-phosphate uridyltransferase enzyme ( GalT ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10802742" target="_blank">10802742</a>]<br></li>
				<label class="DOM_collapse_similarity" for="dom_similarity_36">
Gillette , W.K. , Martin , R.G. , and Rosner , J.L. ( 2000 ) Probing the Escherichia coli transcriptional activator MarA using alanine-scanning mutagenesis : residues important for DNA-binding and activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12100559" target="_blank">12100559</a>]<br></label>
<input id="dom_similarity_36", type="checkbox">
<div class="DOM_similar">
<li>Probing the Escherichia coli transcriptional activator MarA using alanine-scanning mutagenesis : residues important for DNA-binding and activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21097628" target="_blank">21097628</a>]<br></li></div>
				<label class="DOM_collapse_similarity" for="dom_similarity_38">
DNA-binding motif in MarA : the ﬁrst structure for an AraC-family transcription activator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16621812" target="_blank">16621812</a>]<br></label>
<input id="dom_similarity_38", type="checkbox">
<div class="DOM_similar">
<li>A novel DNA-binding motif in MarA : the ﬁrst structure for an AraC-family transcriptional activator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11741843" target="_blank">11741843</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10940041" target="_blank">10940041</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10438753" target="_blank">10438753</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16980449" target="_blank">16980449</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15342598" target="_blank">15342598</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10464186" target="_blank">10464186</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21097628" target="_blank">21097628</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17513476" target="_blank">17513476</a>]<br></li>
<li>A novel DNA-binding motif in MarA : the first structure for an AraC-family transcriptional activator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10802742" target="_blank">10802742</a>]<br></li>
<li>Protein , nucleotide , structure A novel DNA-binding motif in MarA : the ® rst structure for an AraC-family transcriptional activator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11601842" target="_blank">11601842</a>]<br></li>
<li>A novel DNA-binding motif in MarA : the ® rst structure for an AraC-family transcriptional activator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10873449" target="_blank">10873449</a>]<br></li></div>
				<label class="DOM_collapse_similarity" for="dom_similarity_43">
Rhee S , Martin RG , Rosner JL , Davies DR. A novel DNA-binding motif in MarA : The first structure for an AraC-family transcriptional activator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19422057" target="_blank">19422057</a>]<br></label>
<input id="dom_similarity_43", type="checkbox">
<div class="DOM_similar">
<li>Rhee S , Martin RG , Rosner JL , Davies DR. A novel DNA-binding motif in MarA : the first structure for an AraC-family transcriptional activator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22465792" target="_blank">22465792</a>]<br></li>
<li>[ PubMed : 10610768 ] Rhee S , Martin RG , Rosner JL , Davies DR. A novel DNA-binding motif in MarA : the first structure for an AraC-family transcriptional activator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18366439" target="_blank">18366439</a>]<br></li></div>
				<label class="DOM_collapse_similarity" for="dom_similarity_46">
[ 15 ] Rhee , S. , Martin , R.G. , Rosner , J.L. and Davies , D.R. ( 1998 ) A novel DNA-binding motif in MarA : the cents rst structure for an AraC-family transciptional activator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11470373" target="_blank">11470373</a>]<br></label>
<input id="dom_similarity_46", type="checkbox">
<div class="DOM_similar">
<li>Rhee , S. , Martin , R. , Rosner , J. , and Davies , D. ( 1998 ) A novel DNA-binding motif in MarA : The ﬁrst structure for an AraC-family transcriptional activator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15066032" target="_blank">15066032</a>]<br></li>
<li>Rhee , S. , Martin , R.G. , Rosner , J.L. , and Davies , D.R. ( 1998 ) A novel DNA-binding motif in MarA : the ﬁrst structure for an AraC-family transcriptional activator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12791142" target="_blank">12791142</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12100559" target="_blank">12100559</a>]<br></li>
<li>Rhee , S. , Martin , R. , Rosner , J. , and Davies , D. ( 1998 ) A novel DNA-binding motif in MarA : the ﬁrst structure for an AraC-family transcriptional activator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12675795" target="_blank">12675795</a>]<br></li>
<li>Rhee , S.R. , Martin , R.G. , Rosner , J.L. , and Davies , D.R. ( 1998 ) A novel DNA-binding motif in MarA : the ﬁrst structure for an AraC-family transcriptional activator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15225313" target="_blank">15225313</a>]<br></li>
<li>Rhee , S. , Martin , R. , Rosner , J. , and Davies , D. ( 1998 ) A novel DNA-binding motif in MarA : the first structure for an AraC family transcriptional activator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10760178" target="_blank">10760178</a>]<br></li>
<li>Rhee , S. , Martin , R. , Rosner , J. and Davies , D. ( 1998 ) A novel DNA-binding motif in MarA : the first structure for an AraC-family transcriptional activator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12060687" target="_blank">12060687</a>]<br></li>
<li>Rhee , S. , Martin , R.G. , Rosner , J.L. , and Davies , D.R. ( 1998 ) A novel DNA-binding motif in MarA : the ® rst structure for an AraC-family transcriptional activator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10564485" target="_blank">10564485</a>]<br></li>
<li>Rhee , S. , Martin , R.G. , Rosner , J.L. and Davies , D.R. ( 1998 ) A novel DNA-binding motif in MarA : the ® rst structure for an AraC-family transcriptional activator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12582243" target="_blank">12582243</a>]<br></li></div>
<li>This finding suggests that MarA does not alter the RNAP-DNA conformation after open complex formation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16478729" target="_blank">16478729</a>]<br></li>
<li>Second , we considered the possibility that Fis and MarA might form a protein-protein complex ( in addition to their DNA-binding and/or RNA polymerase binding capacities ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9393706" target="_blank">9393706</a>]<br></li>
<li>This result supports the hypothesis that these amino acids might be part of a second DNA-binding motif in RhaS , which we predict to be an H-T-H motif based on alignment with MarA ( 30 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10464186" target="_blank">10464186</a>]<br></li>
<li>We propose that this region is part of a second DNA-binding motif of RhaS that , based on alignment with the crystal structure of MarA ( 30 ) , is likely an H-T-H motif .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10464186" target="_blank">10464186</a>]<br></li>
<li>In particular , there was striking identity in two regions of putative DNA-binding domains , 12 of 12 amino acids ( GYSKWHLQRMFK ) from positions 38 to 49 of MarA and 10 of 14 amino acids from positions 87 to 100 of MarA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8383113" target="_blank">8383113</a>]<br></li>
<li>This is a particularly attractive hypothesis , because the three IscR cysteine residues , which may coordinate the Fe-S cluster , are located in a region that in MarA separates the two helix-turn-helix DNA-binding motifs ( 14 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11742080" target="_blank">11742080</a>]<br></li>
<li>An analysis of MarA by the method of Chou and Fasman ( 4 ) identified two potential helix-turn-helix DNA-binding domains located between amino acid residues 25 and 52 and between residues 79 and 92 , within regions which have conserved residues with the proteins AraC and XylS ( 23 ) ( Fig. 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8383113" target="_blank">8383113</a>]<br></li>
<li>We considered it a reasonable possibility that IscR might bind these sequences , because IscR contains regions similar to the helix-turn-helix DNA-binding domains of MarA ( Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11742080" target="_blank">11742080</a>]<br></li>
<li>Rob , MarA , and SoxS contain similarly positioned helix-turn-helix motifs ( Fig. 1 ) that might mediate speciﬁc DNA-binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7896685" target="_blank">7896685</a>]<br></li>
<li>Alignment with the recently crystallized MarA protein suggest that both RhaS motifs are likely helix-turn-helix DNA-binding motifs .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10464186" target="_blank">10464186</a>]<br></li>
<li>Comparison of the predicted amino acid sequences of SoxS , MarA and Rob ( all homologous regulatory proteins that bind to the promoters of a number of genes , including fumC and acnA ) , with SoxR , showed a similar helix ^ turn ^ helix DNA-binding motif .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11470373" target="_blank">11470373</a>]<br></li>
<li>The similar predicted helix-turn-helix motifs of SoxS and MarA , which differ by a single amino acid residue in the second helix ( 6 ) , might mediate protein-binding to the same or overlapping sites in the responsive promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8282690" target="_blank">8282690</a>]<br></li>
<li>The transcription of the acrB gene is regulated by a global transcriptional activator MarA and a local transcriptional repressor AcrR .21 The acrR gene is located 141 bp upstream of the acrAB operon and transcribed divergently .21 It encodes a 215-aminoacid protein , which shares N-terminal sequence and structural similarities to members of the TetR family of transcriptional repressors .22 The signatures of the TetR family of regulators include a homologous three helix DNA-binding domain , and a diverse C-terminal domain that possesses unique sequences , forming a specific ligand-binding site for inducing compounds .22 Many TetR family members bind as dimers to their ~ 15-bp palindrome ( IR ) operator sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17950313" target="_blank">17950313</a>]<br></li>
<li>Because MarA is a small protein ( 129 amino acid resi-dues ) that contains only the DNA-binding domain ( Rhee et al. , 1998 ) , it is expected to exert its regulatory action solely by the alteration of its intracellular level .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12791142" target="_blank">12791142</a>]<br></li>
<li>MarA and its homolog SoxS are unique among AraC proteins in that they have only the DNA-binding domain ( 129 and 107 amino acid residues , respectively ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10873449" target="_blank">10873449</a>]<br></li>
<li>Importantly , both MarA and SoxS are small proteins with 129 and 107 amino acid residues , respectively , and correspond only to the DNA-binding domains of the typical XylS/AraC regulators ( Gallegos et al. , 1997 ; Rhee et al. , 1998 ; Martin and Rosner , 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12791142" target="_blank">12791142</a>]<br></li>
<li>Additionally , and despite similarities in the structures of their DNA-binding domains and cognate DNA-binding sites , the SoxS homologs MarA and Rob did not activate in vivo transcription of yggX under the conditions tested ( Fig. 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14594836" target="_blank">14594836</a>]<br></li>
<li>The structures of the DNA-binding domains of Rob and MarA are superimposable with a root mean squared deviation of only 0.9 Å for all main chain atoms .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12060687" target="_blank">12060687</a>]<br></li>
<li>Rob , however , is far larger than MarA or SoxS , containing 289 residues , and includes a CTD that is separate from the DNA-binding , N-terminal domain ( Kwon et al. , 2000 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12791142" target="_blank">12791142</a>]<br></li>
<li>The Rob protein consists of an N-terminal-DNA-binding-domain that is homologous to the MarA protein13 , and a unique C-terminal domain of unknown function ( Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10802742" target="_blank">10802742</a>]<br></li>
<li>The SoxS and MarA proteins consist only of the conserved DNA-binding domain ( ∼ 100 residues ) that is present in all AraC/XylS-family members , whereas the Rob protein has an additional C-terminal domain ( ∼ 200 residues ) of unknown function .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10802742" target="_blank">10802742</a>]<br></li>
<li>The MarA protein is shorter than all other known members of this protein family except SoxS and appears to consist primarily of a putative DNA-binding domain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8383113" target="_blank">8383113</a>]<br></li>
<li>The location of the DNA-binding domain , consisting of seven - helix elements and deduced from similarities with MarA and the published MarA structure ( 20 ) , is shown .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16621812" target="_blank">16621812</a>]<br></li>
<li>Since MarA and SoxS are members of the AraC class of DNA-binding proteins and are highly homologous to each other in the putative helix-turn-helix domains ( 1 , 4 , 8 , 9 ) , it was of interest to determine whether MarA is also a transcriptional activator and whether its mode of action is similar to that of SoxS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8522515" target="_blank">8522515</a>]<br></li>
<li>Substitution for residues comprising the hydrophobic core of MarA The homology among the AraC-family of proteins is greatest for residues which comprise the hydrophobic core of the DNA-binding motifs .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10873449" target="_blank">10873449</a>]<br></li>
<li>MarA is one of relatively few AraC-family members that consists of a DNA-binding domain without any dimerization domain , and it activates transcription as a monomer ( 6 , 23 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10464186" target="_blank">10464186</a>]<br></li>
<li>The conserved DNA-binding domains of other AraC-family members are expected to adopt the same tertiary structure as MarA , since they share greater than 20 % similarity ( Gallegos et al. , 1997 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10873449" target="_blank">10873449</a>]<br></li>
<li>The crystal structures of the AraC/XylS-family DNA-binding domains of MarA and Rob have been solved ( MarA consists of only the DNA-binding domain while Rob also has a second domain ) ( Kwon et al. , 2000 ; Rhee et al. , 1998 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18366439" target="_blank">18366439</a>]<br></li>
<li>Detailed molecular structures have been determined for the DNA-binding domains of two AraC/XylS-family members , MarA and Rob ( 20 , 28 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17513476" target="_blank">17513476</a>]<br></li>
<li>There are currently structures available for the DNA-binding domain of two AraC / XylS family proteins , MarA and Rob ( 31 , 46 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10940041" target="_blank">10940041</a>]<br></li>
<li>Of interest in this vein is the recent identification of a protein that binds to the right arm of the replication origin oriC , called Rob ( 30 ) , which contains a predicted helix-turn-helix motif with the second helix identical to that of MarA ( 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8282690" target="_blank">8282690</a>]<br></li>
<li>On the other hand , if the side group of a given wild-type amino acid residue does not make a specific protein -- DNA contact with a functional group of a wild-type base pair , then the effect on transcription-activation of combining a single alanine substitution of that amino acid of Rob and a single base-pair substitution at that position of the robbox would be significantly different from that conferred by the more severe of the two individual mutations ; that is , the effect on activation conferred by one mutation would not be masked by the effect conferred by the other , and thus , the mutations would be non-epistatic to one another .42,48 According to the co-crystal structure of Rob bound to micF DNA ( Fig. 3a ) , residue W36 in helix-3 of Rob 's N-terminal HTH motif makes a van der Waals contact with nucleotide C5 in RE1 ( GCAC ; C5 in boldface ) ,39 the most highly conserved residue in the Rob/SoxS / MarA binding sites.20-22 ,39 Indeed , changing C5 to any of the other three nucleotides severely reduces SoxS-dependent transcription-activation of the fpr and zwf promoters .20 Thus , for our epistasis experiment , we used site-directed mutagenesis to introduce a C5T substitution into the robbox of a micF-lac transcriptional fusion carried on plasmid pMLB1022 and to prepare a derivative of plasmid pBAD33-Rob carrying substitution W36A of Rob .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22465792" target="_blank">22465792</a>]<br></li>
<li>Importantly , we also carried out a genetic epistasis experiment to determine whether the side group of an amino acid within helix-6 of Rob 's C-terminal HTH motif interacts specifically with a functional group of a base pair of RE2 in the robbox of the micF promoter , as do amino acids of helix-6 of MarA and base pair of RE2 in the marbox of the mar promoter .41 We note that the amino acid sequences of helix-6 of Rob and MarA are identical at 10 of 12 positions .16 To conduct the genetic epistasis test , we used site-directed mutagenesis to substitute amino acid R90 of pBAD33-Rob with alanine43 and to introduce a C15G substitution into the robbox of a micF-lac fusion carried on plasmid pMLB1022 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22465792" target="_blank">22465792</a>]<br></li>
<li>Unlike MarA , Rob does not utilize its C-terminal HTH motif to interact sequence specifically with the major groove of the B-box in the crystal structure ( Fig. 1a ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10802742" target="_blank">10802742</a>]<br></li>
<li>This bulky substitution is located in the major groove of the B-box and it would hinder the C-terminal HTH from inserting into the major groove of the bent DNA as seen in the MarA -- DNA crystal structure .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10802742" target="_blank">10802742</a>]<br></li>
<li>) In the MarA -- mar structure , the recognition helices of both the N-terminal HTH motif ( helix-3 ) and the C-terminal HTH motif ( helix-6 ) are bound to successive major grooves containing recognition elements 1 and 2 ( RE1 and RE2 ) , respectively .20,41 In contrast , the structure of the Rob -- micF complex shows that the recognition helix of the N-terminal HTH motif ( helix-3 ) binds RE1 of the robbox while that of the C-terminal HTH motif ( helix-6 ) is not bound to DNA .39 Moreover , the DNA in the MarA -- mar co-crystal structure is bent Results 35 ° ,41 whereas the DNA in the Rob -- micF co-crystal structure is straight ,39 probably because Rob 's second HTH motif does not specifically contact base pairs in RE2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22465792" target="_blank">22465792</a>]<br></li>
<li>In particular , recognition helix-3 and helix-6 of MarA 's dual HTH motifs bind RE1 and RE2 , respectively , of mar 's robbox , while helix-3 within Rob 's N-terminal HTH motif binds RE1 of micF 's robbox , but helix-6 does not bind RE2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22465792" target="_blank">22465792</a>]<br></li>
<li>The crystal structure of MarA with its cognate sequence ( 6 ) shows direct hydrogen bond contacts to two groups of bases 9 bp apart , the equivalent of the A and B boxes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12060687" target="_blank">12060687</a>]<br></li>
<li>The crystal structure of MarA bound to marbox DNA shows the two recognition helices of the protein in contact with the recognition elements of the marbox ( 10 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14701822" target="_blank">14701822</a>]<br></li>
<li>The crystal structure also predicted 15 contacts between MarA and the sugar-phosphate backbone of marbox DNA ( indicated by P in Figure 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10873449" target="_blank">10873449</a>]<br></li>
<li>In the crystal structure of a complex with DNA ( PDB code : 1BL0 ) , MarA appears as a bipartite helix -- turn -- helix protein ( 51 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10318909" target="_blank">10318909</a>]<br></li>
<li>The only major difference between the MarA and Rob structures is that MarA makes base-speciﬁc contacts with DNA using both of its helix-turn-helix motifs , while Rob only makes base-speciﬁc contacts with its N-termi-nal helix-turn-helix motif ( 31 , 46 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10940041" target="_blank">10940041</a>]<br></li>
<li>In brief , two helix ± turn ± helix regions of the MarA monomer were found to contact bases in adjacent segments of the major groove of the marbox DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10564485" target="_blank">10564485</a>]<br></li>
				<label class="DOM_collapse_similarity" for="dom_similarity_98">
The structure clearly shows that MarA , as predicted , contains two helix ± turn ± helix motifs , and that these penetrate two adjacent major grooves on the same side of the DNA duplex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10760178" target="_blank">10760178</a>]<br></label>
<input id="dom_similarity_98", type="checkbox">
<div class="DOM_similar">
<li>The structure shows that MarA , as predicted , contains two HTH motifs , and that these penetrate two adjacent segments of the major groove on the same side of the DNA duplex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12675795" target="_blank">12675795</a>]<br></li></div>
<li>RESULTS Detailed examination of the crystal structure of MarA suggested that the two regions of the molecule containing the two helix-turn-helix motifs were likely to be less ﬂexible than those of SoxS ( see Fig .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21097628" target="_blank">21097628</a>]<br></li>
<li>From X-ray crystallographic analysis of the cocrystal of MarA with the marbox from the marRAB promoter , two helix-turn-helix ( HTH ) motifs were identiﬁed that make 34 contacts with the DNA and bend it by 35 ° ( 36 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21097628" target="_blank">21097628</a>]<br></li>
<li>Sol-ution of the MarA-marbox X-ray crystallographic structure suggested the presence of base-speci ® c and non-speci ® c interactions between the marbox and two helix-turn-helix ( HTH ) motifs on the monomeric MarA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10873449" target="_blank">10873449</a>]<br></li>
<li>While we were in the process of creating and analyzing the alanine substitutions , publication of the MarA crystal structure showed that Arg96 makes speciﬁc contacts with DNA ( 30 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10464186" target="_blank">10464186</a>]<br></li>
<li>Disruption of several predicted contacts had small effects on MarA activity Several contacts predicted from the MarA-mar-box co-crystal structure were not important for MarA activity , as revealed by alanine substitutions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10873449" target="_blank">10873449</a>]<br></li>
<li>In the crystal structure , C-5 makes a van der Waals contact with Trp-42 of MarA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10564485" target="_blank">10564485</a>]<br></li>
<li>Based on the position of residue 191 on the crystal structure of MarA ( 46 ) ( Fig. 4B ) , it seemed very unlikely to contact s70 , so when technical difﬁ-culties were encountered in this construction it was not further pursued .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10940041" target="_blank">10940041</a>]<br></li>
<li>Residues 265 and 294 within the - CTD of RNAP contact MarA when interacting at the activated promoters ( 39 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16478729" target="_blank">16478729</a>]<br></li>
<li>The loss of activity was striking in the case of Tyr39Ala with negligible ( < 10 % of wildtype versus < 25 % of wild-type for Ser40Ala ) activation of all of the test promoters , suggesting that this phosphate contact is extremely important for MarA activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10873449" target="_blank">10873449</a>]<br></li>
<li>To identify such promoterspeci ® c MarA-marbox interactions we determined the activity of mutant MarA proteins with single alanine substitutions at each position in recognition helices 3 and 6 on the ® ve test promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10873449" target="_blank">10873449</a>]<br></li>
<li>MarA activity was sensitive to disruption of these contacts , particularly those involving the N-term-inal HTH motif ( Figure 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10873449" target="_blank">10873449</a>]<br></li>
<li>At least one phosphate contact with the C-term-inal HTH motif was also important for MarA activity ; the disruption of Lys99 resulted in reduced activity on all but the mar promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10873449" target="_blank">10873449</a>]<br></li>
<li>While both N and C-terminal HTH motifs make essential contributions to binding site af ® nity , MarA is more sensitive to alterations in the N-terminal HTH .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10873449" target="_blank">10873449</a>]<br></li>
<li>In a similar way , the activity of MarA is more sensitive to alterations in the hydrophobic core of this HTH .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10873449" target="_blank">10873449</a>]<br></li>
<li>The results of alanine-scanning mutagenesis on MarA show that both HTHs are important for MarA activity ; however , the protein is more sensitive to alterations in the N-terminal HTH than in the C-terminal HTH .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10873449" target="_blank">10873449</a>]<br></li>
<li>Increased binding and activation occurred only at class I promoters when the 12th base of the promoter 's marbox ( a position at which there is no known interaction between the marbox and MarA ) was not a T residue .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21097628" target="_blank">21097628</a>]<br></li>
<li>Furthermore , WT MarA binding to a synthetic marbox in vitro was enhanced when the phosphate group between positions 12 and 13 was eliminated on one strand .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21097628" target="_blank">21097628</a>]<br></li>
<li>Furthermore , the important role of phosphate contacts in marbox af ® nity suggests that indirect readout contrib-utes to binding site recognition by MarA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10873449" target="_blank">10873449</a>]<br></li>
<li>What are the important contacts between MarA and the highly degenerate marbox that allow binding with very different levels of af ® - nity ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10873449" target="_blank">10873449</a>]<br></li>
<li>However , the nearly invariant A-1 ( 15 out of 16 ) makes no contact with MarA and does not appear to be important for MarA recognition .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10564485" target="_blank">10564485</a>]<br></li>
<li>Our results suggest that MarA : marbox interactions will vary slightly for each degenerate marbox ; a given contact may not be important for interaction with all marboxes in the mar regulon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10873449" target="_blank">10873449</a>]<br></li>
<li>However , this idea was contradicted when a single mutation at position 16 ( C to G ) of RE 2 , which has been shown to establish direct contacts with the second of two HTH motifs of MarA ( Rhee et al. , 1998 ; Dangi et al. , 2001 ) , eliminated MarA-mediated activation of the nfnB gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12100559" target="_blank">12100559</a>]<br></li>
<li>The finding that MarA consists of two structurally similar subdomains ( Rhee et al. , 1998 ) , one including HTH # 1 ( involved in the RhaS mechanism ) , and one including HTH # 2 ( involved in the RhaR mechanism ) , suggests the possibility that the underlying mechanisms of the RhaS and RhaR allosteric responses may be more similar than they appear .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18366439" target="_blank">18366439</a>]<br></li>
<li>MarA is composed of seven a-helices which fold to form two HTH subdomains connected by a long a-helix ( Rhee et al. , 1998 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10873449" target="_blank">10873449</a>]<br></li>
<li>The locations of the MarA and Rob residues that , according to structural studies ( Rhee et al. , 1998 ; Kwon et al. , 2000 ) , either make base-speciﬁc contacts or contribute to the hydrophobic core of the domain are indicated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12675795" target="_blank">12675795</a>]<br></li>
<li>For MarA and Rob , residues that participate in formation of the hydrophobic core are shaded grey , whereas residues that make base-speciﬁc contacts are boxed ( according to Rhee et al. , 1998 ; Kwon et al. , 2000 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12675795" target="_blank">12675795</a>]<br></li>
<li>Residue locations are indicated under `` L' ' : S ; surface-exposed residues ; I , internal residues ; residues not visible in the crystal structure reported by Rhee et al. ( 1998 ) , a dash The same amino acid residues are important in recognition of the different marboxes Although a marbox consensus sequence has been determined ( Martin et al. , 1999 ) , the bases positioned to contact MarA in the mar marbox are not highly conserved in the test promoters ( Figure 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10873449" target="_blank">10873449</a>]<br></li>
<li>Hydrogen bonds and van der Waals interactions between the MarA amino acid residues and bases of the marbox indicated by Rhee et al. ( 1998 ) are depicted as continuous and broken lines , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10873449" target="_blank">10873449</a>]<br></li>
<li>In contrast , with RNA polymerase reconstituted with the a-235 mutant , whose a subunits lack the 94 amino acid residues from the C terminus , MarA was completely unable to activate zwf ( 0.7-fold activation ) , although transcription from the fumC and micF promoters was still activated by 1.4 - and 2.4-fold , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8522515" target="_blank">8522515</a>]<br></li>
<li>There are ﬁve different examples in MarA of base pairs contacted by two amino acids , and even one example of a base pair contacted by three amino acids ( 30 ) ( Fig. 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10464186" target="_blank">10464186</a>]<br></li>
<li>Further , two of the amino acids in MarA ( Trp42 and Arg96 ) contact three and four adjacent base pairs , respectively , although with Arg96 there is a water molecule involved in one of the contacts ( 30 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10464186" target="_blank">10464186</a>]<br></li>
<li>As expected , alanine substitution for amino acids that are positioned to make hydrogen bonds to the sugar-phosphate backbone through their a-amino groups , Gly57 , Gln58 , His107 and Lys108 , did not affect MarA activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10873449" target="_blank">10873449</a>]<br></li>
<li>Additionally , experimental structures of three of them have been publicly released : T0056 ( NMR structure of the N-terminal domain of Escherichia coli Dnab helicase ; PDB entry : 1JWE ) , T0061 [ x-ray structure of HDEA , E. coli ( 50 ) ; 89 amino acid residues ; PDB entry : 1BG8 ] and T0079 [ x-ray structure of MarA ( 51 ) ; 129 amino acid residues ; PDB entry : 1BL0 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10318909" target="_blank">10318909</a>]<br></li>
<li>To test the importance of the conserved hydrophobic residues identi ® ed as comprising the cores of the two HTH motifs of MarA , alanine-scanning mutagenesis was performed on 11 of the 17 amino acid residues implicated in hydrophobic core formation and four additional N-terminal hydrophobic residues conserved among MarA , SoxS and Rob .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10873449" target="_blank">10873449</a>]<br></li>
<li>These amino acid residues constitute a third hydrophobic region located between the two HTH motifs that may be important for proper MarA tertiary structure formation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10873449" target="_blank">10873449</a>]<br></li>
<li>As expected , the amino acid residues which comprise the hydrophobic core of the two subdo mains of MarA are sensitive to substitution .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10873449" target="_blank">10873449</a>]<br></li>
<li>Due to the relative unavailability of potentially charged and hydrogen bond-forming groups from Trp19 , Glu21 and Arg110 , it is unclear : ( i ) how these amino acid residues would all interact with RNAP ; and ( ii ) how they would function identically in the alternative orientations MarA adopts at class I and class II promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10873449" target="_blank">10873449</a>]<br></li>
<li>Class II promoter-specific defects localize on the MarA surface Alanine substitutions for amino acid residues Glu77 , Leu80 and Gln91 severely affected activity on the class II promoters , micF and fumC , and to a lesser extent , fpr ( Table 1 and Figure 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10873449" target="_blank">10873449</a>]<br></li>
<li>Mutagenesis of solvent-exposed MarA residues Given the multiple con ® gurations of MarA with respect to RNAP ( Figure 1 ( b ) ) , certain amino acid residues necessary for activity on one set of the promoters may not be necessary for activity on another set e.g. class I and II promoters ( see below ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10873449" target="_blank">10873449</a>]<br></li>
<li>Solvent-exposed amino acid residues located at many positions on the MarA surface are important for activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10873449" target="_blank">10873449</a>]<br></li>
<li>The adjacent amino acid residues Glu77 and Leu80 are on the MarA surface proximal to promoter-bound RNAP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10873449" target="_blank">10873449</a>]<br></li>
<li>The MarA residues implicated in class I promoter activity include charged and polar amino acid residues , suggesting that MarA can make multiple contacts with RNAP at these promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10873449" target="_blank">10873449</a>]<br></li>
<li>To determine which residues contribute to these interactions , alanine substitutions were made at 107 of the 126 non-alanine amino acid residues in MarA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10873449" target="_blank">10873449</a>]<br></li>
<li>Here , we use alanine-scanning mutagenesis and DNA retardation analysis to : ( i ) evaluate the contacts between MarA and the marboxes of ® ve differently con ® gured mar regulon promoters ; ( ii ) assess the role of conserved hydrophobic amino acid residues for MarA activity ; and ( iii ) identify residues required for RNA polymerase activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10873449" target="_blank">10873449</a>]<br></li>
<li>Here , we seek to determine the important contacts between speci ® c MarA amino acid side-chains and the marboxes of the mar regulon promoters as well as to identify MarA residues that interact with RNAP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10873449" target="_blank">10873449</a>]<br></li>
<li>The results show that both HTHs contribute to DNA-binding and that disruption of either non-speci ® c contacts with the sugar-phosphate backbone or speci ® c contacts made by Arg46 and Arg96 to bases within the major groove of the marbox severely reduce MarA activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10873449" target="_blank">10873449</a>]<br></li>
<li>Both base-specific and phosphate contacts contribute to MarA activity and DNA-binding Eight amino acid residues of MarA are positioned to contact bases in the major groove of the marbox ; only three of these , Arg46 , Thr93 and Arg96 are predicted to form hydrogen bonds ( Figure 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10873449" target="_blank">10873449</a>]<br></li>
<li>MarA amino acid residues whose side-chains interact with marbox DNA and are not conserved between Rob and SoxS , all interact with the sugar-phosphate backbone : His43 , Lys49 , Thr97 and Lys99 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10873449" target="_blank">10873449</a>]<br></li>
<li>Some of these residues affect activity on all promoters and thus , are implicated in maintaining MarA structure whereas several solvent-exposed amino acids not involved in DNA-binding were important for MarA activity on speci ® c promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10873449" target="_blank">10873449</a>]<br></li>
<li>Five MarA solvent-exposed residues are important for activity on all promoters Apart from the DNA-binding domains , only ® ve alanine substitutions in solvent-exposed amino acids of MarA resulted in severely decreased activity on all promoters tested : Trp19 , Glu21 , Leu28 , Pro78 and Arg110 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10873449" target="_blank">10873449</a>]<br></li>
<li>The pocket harboring Lys64 is located o the DNA-binding face of MarA and is lined with six conserved hydrophobic amino acid residues .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10873449" target="_blank">10873449</a>]<br></li>
<li>To identify amino acid residues that interact with RNAP , alanine-scanning mutagenesis was performed on solvent-exposed amino acid residues of MarA outside of the DNA-binding region described above .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10873449" target="_blank">10873449</a>]<br></li>
<li>In previous work ,42 we found that epistatic and non-epistatic interactions between amino acid residues of SoxS and base pairs of soxbox DNA agreed with the predictions based on the co-crystal structure of MarA bound to marbox DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22465792" target="_blank">22465792</a>]<br></li>
<li>A subset of these residues ( Asp18 , Thr52 , Leu121 ) is located on the DNA-distal surface of MarA ; however , this region is likely interrupted by the C-terminal ® ve amino acid residues not represented in the crystal structure .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10873449" target="_blank">10873449</a>]<br></li>
<li>The structure of MarA , a 129 amino acid activator , which only contains the C-terminal homologous region of the protein , has been determined bound to DNA ( 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12060687" target="_blank">12060687</a>]<br></li>
<li>Therefore , MarA may recognize these marboxes with a different combination of amino acid contacts four those predicted from the crystal structure .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10873449" target="_blank">10873449</a>]<br></li>
<li>Inspection on the MarA structure of the amino acids that aligns with each of these nonspe-ciﬁc RhaS amino acids ( by using RasMol version 2.6 ) indicates that all of them except Phe254 are signiﬁcantly surface exposed and near the DNA backbone .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10464186" target="_blank">10464186</a>]<br></li>
<li>In fact , most of the MarA residues positioned to make DNA contacts are conserved between the three proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10873449" target="_blank">10873449</a>]<br></li>
<li>It is not surprising that MarA , which binds to DNA as a monomer ( 23 ) , has more amino acid-DNA contacts per monomer than RhaS , which is predicted to bind as a dimer .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10464186" target="_blank">10464186</a>]<br></li>
<li>MarA has a third amino acid in H-T-H 1 that contacts DNA ( Gln45 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10464186" target="_blank">10464186</a>]<br></li>
<li>Summary of speciﬁc amino acid-base pair contacts by RhaS and comparison with MarA DNA contacts .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10464186" target="_blank">10464186</a>]<br></li>
<li>Our current analysis shows an equal number of amino acids in the MarA monomer and the RhaS dimer that make DNA contacts ; however , the total number of base-speciﬁc contacts made by the MarA monomer ( 20 contacts ) is greater than the number ( 12 to 14 ) made by the RhaS dimer .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10464186" target="_blank">10464186</a>]<br></li>
<li>There are no DNA contacts ( neither speciﬁc nor phosphate backbone ) by amino acids in this turn of MarA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10464186" target="_blank">10464186</a>]<br></li>
<li>Eleven different amino acids in MarA make only phosphate backbone contacts with the DNA ( 30 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10464186" target="_blank">10464186</a>]<br></li>
<li>Both of the amino acids in H-T-H 1 of RhaS that make sequence-speciﬁc DNA contacts align with MarA residues also predicted to make speciﬁc DNA contacts .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10464186" target="_blank">10464186</a>]<br></li>
<li>There are four amino acids in MarA H-T-H 2 which make base-speciﬁc DNA contacts and align with RhaS amino acids that do not make contacts ( Fig. 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10464186" target="_blank">10464186</a>]<br></li>
<li>Most amino acids implicated in promoter-specific activation are variously located on the MarA surface Substitutions that conferred promoter-speci ® c defects were found at many positions on the protein surface ( Table 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10873449" target="_blank">10873449</a>]<br></li>
<li>Interestingly , IscR contains a stretch of amino acids not found in MarA , which includes cysteine residues that may coordinate a Fe-S cluster ( Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11742080" target="_blank">11742080</a>]<br></li>
<li>Interactions between the mar marbox and MarA amino acids .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10873449" target="_blank">10873449</a>]<br></li>
<li>These amino acids are located opposite one another on the MarA surface .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10873449" target="_blank">10873449</a>]<br></li>
<li>( The amino acids chosen for this study were known not to involve the interaction of MarA with RNAP [ 8 ] ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21097628" target="_blank">21097628</a>]<br></li>
<li>The crystal structure suggests that substitutions for these amino acids may affect MarA stability rather than RNAP interactions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10873449" target="_blank">10873449</a>]<br></li>
<li>Except as noted below , these residues are scattered over the surface of MarA and no single region is clearly implicated in contacting RNAP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10873449" target="_blank">10873449</a>]<br></li>
<li>Although these residues are all located on the surface of MarA ( His126 is not visible in the crystal structure ) , they do not form a discrete activating region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10873449" target="_blank">10873449</a>]<br></li>
<li>When modeled on the structure of MarA ( 46 ) , the positions of these residues of RhaS suggest a possible face of RhaS that could interact with s70 ( Fig. 4A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10940041" target="_blank">10940041</a>]<br></li>
<li>Second , considering that D241 is located within the ﬁrst helix of H-T-H 2 of RhaS ( helix-5 of the MarA structure ) ( 4 ) and that H-T-H 2 binds to a major groove that overlaps the 235 region of the promoter ( 12 ) , D241 appears to be ideally positioned on the surface of RhaS to make contact with s70 ( Fig. 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10940041" target="_blank">10940041</a>]<br></li>
<li>Whereas it may be the case that different contacts with RNP are used for the two situations , it is also possible that the different orientations and spacings of the marboxes permits the ¯ exible a-CTD to contact the same residues on the back surface of MarA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10564485" target="_blank">10564485</a>]<br></li>
<li>We also used a molecular model of the RhaR - 70 domain 4 interaction in which the structure of MarA ( 38 ) represented RhaR ( Fig. 4 ) to identify the only two negatively charged RhaR residues , E284 and D285 , that were near 70 K593 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15342598" target="_blank">15342598</a>]<br></li>
				</div></ul><br>
			<label class="collapse_EVO" for="_3"><font size="5px">MarA evolution</font></label>
				<input id="_3" type="checkbox">
				<div class="EVO"><br><ul><li>Considerable ( 50 % ) residue identity was also noted between MarA ( amino acids 13 to 111 ) and the N-terminal portion ( amino acids 7 to 104 ) of the right-origin-binding protein ( RobA ) of E. coli ( 28 ) ( GenBank accession number M94042 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8383113" target="_blank">8383113</a>]<br></li>
<li>In particular , there was striking identity in two regions of putative DNA-binding domains , 12 of 12 amino acids ( GYSKWHLQRMFK ) from positions 38 to 49 of MarA and 10 of 14 amino acids from positions 87 to 100 of MarA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8383113" target="_blank">8383113</a>]<br></li>
<li>Similarly , the Klebsiella pneumoniae multidrug resistance ramA gene is another regulatory locus which codes for a transcriptional activator homologous to MarA ( 78 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8987357" target="_blank">8987357</a>]<br></li>
				<label class="EVO_collapse_similarity" for="evo_similarity_4">
Rhee S , Martin RG , Rosner JL , Davies DR. A novel DNA-binding motif in MarA : The first structure for an AraC family transcriptional activator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19422057" target="_blank">19422057</a>]<br></label>
<input id="evo_similarity_4", type="checkbox">
<div class="EVO_similar">
<li>Rhee S , Martin RG , Rosner JL , Davies DR. A novel DNA-binding motif in MarA : the first structure for an AraC family transcriptional activator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22465792" target="_blank">22465792</a>]<br></li>
<li>[ PubMed : 10610768 ] Rhee S , Martin RG , Rosner JL , Davies DR. A novel DNA-binding motif in MarA : the first structure for an AraC family transcriptional activator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18366439" target="_blank">18366439</a>]<br></li></div>
				<label class="EVO_collapse_similarity" for="evo_similarity_7">
[ 15 ] Rhee , S. , Martin , R.G. , Rosner , J.L. and Davies , D.R. ( 1998 ) A novel DNA-binding motif in MarA : the cents rst structure for an AraC family transciptional activator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11470373" target="_blank">11470373</a>]<br></label>
<input id="evo_similarity_7", type="checkbox">
<div class="EVO_similar">
<li>Rhee , S. , Martin , R. , Rosner , J. , and Davies , D. ( 1998 ) A novel DNA-binding motif in MarA : the first structure for an AraC family transcriptional activator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10760178" target="_blank">10760178</a>]<br></li>
<li>Rhee , S. , Martin , R. , Rosner , J. and Davies , D. ( 1998 ) A novel DNA-binding motif in MarA : the first structure for an AraC family transcriptional activator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12060687" target="_blank">12060687</a>]<br></li>
<li>Rhee , S. , Martin , R.G. , Rosner , J.L. , and Davies , D.R. ( 1998 ) A novel DNA-binding motif in MarA : the ® rst structure for an AraC family transcriptional activator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10564485" target="_blank">10564485</a>]<br></li>
<li>Rhee , S. , Martin , R.G. , Rosner , J.L. and Davies , D.R. ( 1998 ) A novel DNA-binding motif in MarA : the ® rst structure for an AraC family transcriptional activator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12582243" target="_blank">12582243</a>]<br></li>
<li>Rhee , S. , Martin , R. , Rosner , J. , and Davies , D. ( 1998 ) A novel DNA-binding motif in MarA : The ﬁrst structure for an AraC family transcriptional activator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15066032" target="_blank">15066032</a>]<br></li>
<li>Rhee , S. , Martin , R.G. , Rosner , J.L. , and Davies , D.R. ( 1998 ) A novel DNA-binding motif in MarA : the ﬁrst structure for an AraC family transcriptional activator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12791142" target="_blank">12791142</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12100559" target="_blank">12100559</a>]<br></li>
<li>Rhee , S. , Martin , R. , Rosner , J. , and Davies , D. ( 1998 ) A novel DNA-binding motif in MarA : the ﬁrst structure for an AraC family transcriptional activator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12675795" target="_blank">12675795</a>]<br></li>
<li>Rhee , S.R. , Martin , R.G. , Rosner , J.L. , and Davies , D.R. ( 1998 ) A novel DNA-binding motif in MarA : the ﬁrst structure for an AraC family transcriptional activator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15225313" target="_blank">15225313</a>]<br></li></div>
				<label class="EVO_collapse_similarity" for="evo_similarity_16">
DNA-binding motif in MarA : the ﬁrst structure for an AraC family transcription activator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16621812" target="_blank">16621812</a>]<br></label>
<input id="evo_similarity_16", type="checkbox">
<div class="EVO_similar">
<li>A novel DNA-binding motif in MarA : the ﬁrst structure for an AraC family transcriptional activator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11741843" target="_blank">11741843</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10940041" target="_blank">10940041</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10438753" target="_blank">10438753</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16980449" target="_blank">16980449</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15342598" target="_blank">15342598</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10464186" target="_blank">10464186</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21097628" target="_blank">21097628</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17513476" target="_blank">17513476</a>]<br></li>
<li>A novel DNA-binding motif in MarA : the first structure for an AraC family transcriptional activator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10802742" target="_blank">10802742</a>]<br></li>
<li>Protein , nucleotide , structure A novel DNA-binding motif in MarA : the ® rst structure for an AraC family transcriptional activator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11601842" target="_blank">11601842</a>]<br></li>
<li>A novel DNA-binding motif in MarA : the ® rst structure for an AraC family transcriptional activator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10873449" target="_blank">10873449</a>]<br></li></div>
<li>Some members of this group of proteins are highly homologous to each other , and some of them -- SoxS , MarA , and Rob -- crossregulate certain genes ( 8 , 48 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9409145" target="_blank">9409145</a>]<br></li>
<li>Certain members of the family are small monomeric proteins which lack the N-terminal domain , e.g. MarA and Rob ( although Rob has an unusual Cterminal extension ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15066032" target="_blank">15066032</a>]<br></li>
<li>The homology to SoxS and MarA is restricted to the N-terminal domain of Rob ( 40 ) , which suggests that a common regulatory activity might reside in this region of the protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7896685" target="_blank">7896685</a>]<br></li>
<li>The N-terminal domain of Rob shares similarity with MarA and SoxS , which is in contrast to the other members of the subgroup that share homology within the C-terminal domain ( 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16478729" target="_blank">16478729</a>]<br></li>
<li>Percent identity ( I ) and percent similarity ( I and :) between MarA and SoxS were 42.1 and 64.5 % , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8383113" target="_blank">8383113</a>]<br></li>
<li>Percent identity and percent similarity between MarA and AraC were 20.0 and 44.0 % , respectively , and between MarA and XylS were 24.6 and 38.1 % , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8383113" target="_blank">8383113</a>]<br></li>
<li>However , the N-terminal 100 residues of Rob share about 50 % sequence identity with MarA and soxs .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8821940" target="_blank">8821940</a>]<br></li>
<li>The sequence identity between AraC and the homologs MarA and Rob is only 21 % and 25 % , respectively , and similarity is ~ 45 % for both .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19422057" target="_blank">19422057</a>]<br></li>
<li>The C-terminal domain of RhaS shares 24 % identity and 46 % similarity with MarA and 31 % identity and 45 % similarity with Rob .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10940041" target="_blank">10940041</a>]<br></li>
<li>Furthermore , we have recently recognized a structural similarity between MarA and the integrase family of sitespeci ® c recombinases ( Gillette et al. , 2000 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10873449" target="_blank">10873449</a>]<br></li>
<li>The transcription of the acrB gene is regulated by a global transcriptional activator MarA and a local transcriptional repressor AcrR .21 The acrR gene is located 141 bp upstream of the acrAB operon and transcribed divergently .21 It encodes a 215-aminoacid protein , which shares N-terminal sequence and structural similarities to members of the TetR family of transcriptional repressors .22 The signatures of the TetR family of regulators include a homologous three helix DNA-binding domain , and a diverse C-terminal domain that possesses unique sequences , forming a specific ligand-binding site for inducing compounds .22 Many TetR family members bind as dimers to their ~ 15-bp palindrome ( IR ) operator sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17950313" target="_blank">17950313</a>]<br></li>
<li>MarA and Rob share particularly high sequence similarity , 51 % , and the structures of their DNA-binding domains ( the only domain of MarA ) are nearly identical ( with a root mean square deviation of 0.9 Å ) ( 20 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17513476" target="_blank">17513476</a>]<br></li>
<li>The Rob protein consists of an N-terminal DNA binding domain that is homologous to the MarA protein13 , and a unique C-terminal domain of unknown function ( Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10802742" target="_blank">10802742</a>]<br></li>
<li>The second ORF ( 0RF129 ) , designated marA , would encode a protein , MarA , whose sequence shows strong similarity to those of a family of positive transcriptional regulators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8383113" target="_blank">8383113</a>]<br></li>
<li>Given that AarP , MarA , PqrA , RamA , SoxS , and TetD , the shortest members of the XylS family ( 106 to 166 amino acids long ) , consist mainly of the homologous segment , the stretch of conserved residues most probably contains all the domains necessary for these regulators to interact with target DNA sequences and RNA polymerase and thus activate transcription from target promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9409145" target="_blank">9409145</a>]<br></li>
<li>MarA and SoxS are members of the AraC subfamily of helix-turn-helix transcriptional acti - vators and have more than 45 % sequence homology with each other ( 1 , 6 , 11 , 12 , 43 ) and with a third protein , Rob , which can also activate antibiotic and superoxide resistance via at least some of the same genes ( 3 , 19 , 39 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8636021" target="_blank">8636021</a>]<br></li>
<li>This occurs typically by MarA ( Okusu et al. , 1996 ; Barbosa and Levy , 2000 ) and SoxS ( Ma et al. , 1996 ; White et al. , 1997 ) , both belonging to the XylS/AraC family of transcription regulators ( Gallegos et al. , 1997 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12791142" target="_blank">12791142</a>]<br></li>
<li>Introduction The Escherichia coli MarA belongs to the AraC/XylS family of transcriptional regulators ( Gallegos et al. , 1997 ; Martin and Rosner , 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12100559" target="_blank">12100559</a>]<br></li>
<li>Most of the regulatory proteins of the XylS/AraC family have a similarly large size , except those rare exceptions such as MarA and SoxS ( Gallegos et al. , 1997 ; Martin and Rosner , 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12791142" target="_blank">12791142</a>]<br></li>
<li>Recently Rhee et al. ( 30 ) determined the crystal structure of the ﬁrst AraC family member , MarA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10464186" target="_blank">10464186</a>]<br></li>
<li>The conserved DNA-binding domains of other AraC family members are expected to adopt the same tertiary structure as MarA , since they share greater than 20 % similarity ( Gallegos et al. , 1997 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10873449" target="_blank">10873449</a>]<br></li>
<li>The sequence homology within the AraC family suggests that the MarA fold is structurally conserved ( Gallegos et al. , 1997 ; Rhee et al. , 1998 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10873449" target="_blank">10873449</a>]<br></li>
<li>MarA , a member of the AraC family of Present address : W. K. Gillette , Building 5 , Room 230 , NIH , Bethesda , MD 20892-0560 , USA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10873449" target="_blank">10873449</a>]<br></li>
<li>This regulator can be an activator or a repressor , so that stimulation or derepression of transcription leads to the overexpression of the member of the AraC/XylS family , which in turn regulates transcription from cognate promoters ( i.e. , MarA , SoxS , and TcpN ) ( 4 , 113 , 129 , 190 , 260 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9409145" target="_blank">9409145</a>]<br></li>
<li>We conclude that MarA , a member of the AraC/XylS family , can act directly as a repressor or an activator , depending on the position and orientation of the marbox within a promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14701822" target="_blank">14701822</a>]<br></li>
<li>Structural homology between MarA of the AraC family of transcriptional activators and the integrase family of site-speci ® c recombinases .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10873449" target="_blank">10873449</a>]<br></li>
<li>The Escherichia coli MarA protein , a member of the AraC / XylS family of transcriptional regulators , mediates cellular responses to stress through the differential control of a large number of chromosomal genes comprising the mar regulon ( 1 -- 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14701822" target="_blank">14701822</a>]<br></li>
<li>MarA is a global regulator of genes induced by antibiotics and is a member of a subfamily of AraC-like transcriptional regulators called the Mar Sox Rob family ( 14 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11742080" target="_blank">11742080</a>]<br></li>
<li>The nature of the interactions between the transcription activator and RNAP depends on the position and orientation of the DNA-bound activator with respect to the location of the − 35 promoter hexamer .8 Thus , at class I promoters , the activator binding site lies upstream of the − 35 element , and the predominant interaction of the activator is with the `` 287 '' determinant of the C-terminal domain ( CTD ) of the α subunit of RNAP .9 On the other hand , at class II promoters , the activator binding site partially or completely overlaps the − 35 element such that the DNA-bound activator is able to interact directly with region 4 of the σ70 subunit ( σ70 R4 ) .10 -15 Rob , SoxS , MarA , and TetD form a highly paralogous subset of proteins16 within the AraC / XylS family of transcriptional activators , whose hallmark is the presence of two helix -- turn -- helix ( HTH ) DNA binding motifs per monomer .17 Extensive amino acid sequence identity is shared between Rob ( 55 % ) , MarA ( 42 % ) , and TetD ( 49 % ) over the l07-amino-acid length of SoxS , the shortest member of the subset .16 Accordingly , the paralogs share many characteristics .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22465792" target="_blank">22465792</a>]<br></li>
<li>The size of this binding site , which may actually be less than 21 bp ( 9 ) , is similar to that of the ; 17-bp sequence bound by a monomer of AraC protein ( 7 ) , a founder of the XylS/AraC subfamily of DNA binding proteins to which Rob , SoxS , and MarA belong ( 11 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626315" target="_blank">8626315</a>]<br></li>
<li>The DNA-binding domains of MarA , SoxS and Rob share about 80 % sequence identity , which suggests that these proteins have overlapping specificity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8821940" target="_blank">8821940</a>]<br></li>
<li>Since MarA and SoxS are members of the AraC class of DNA binding proteins and are highly homologous to each other in the putative helix-turn-helix domains ( 1 , 4 , 8 , 9 ) , it was of interest to determine whether MarA is also a transcriptional activator and whether its mode of action is similar to that of SoxS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8522515" target="_blank">8522515</a>]<br></li>
<li>Substitution for residues comprising the hydrophobic core of MarA The homology among the AraC family of proteins is greatest for residues which comprise the hydrophobic core of the DNA-binding motifs .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10873449" target="_blank">10873449</a>]<br></li>
<li>MarA is one of relatively few AraC family members that consists of a DNA-binding domain without any dimerization domain , and it activates transcription as a monomer ( 6 , 23 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10464186" target="_blank">10464186</a>]<br></li>
<li>This DNA-binding domain has also been shown to be involved in transcription activation in a number of AraC/XylS family proteins including Ada , RhaS , AraC , MelR , MarA , SoxS , XylS , and UreR ( 1 , 4 , 5 , 15 , 16 , 20 -- 22 , 37 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15342598" target="_blank">15342598</a>]<br></li>
<li>Third , MarA , SoxS and Rob belong to a subfamily of the AraC family of DNA-binding proteins ( Gallegos et al. , 1997 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10564485" target="_blank">10564485</a>]<br></li>
<li>Detailed molecular structures have been determined for the DNA-binding domains of two AraC/XylS family members , MarA and Rob ( 20 , 28 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17513476" target="_blank">17513476</a>]<br></li>
<li>There are currently structures available for the DNA-binding domain of two AraC / XylS family proteins , MarA and Rob ( 31 , 46 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10940041" target="_blank">10940041</a>]<br></li>
<li>Most family members contain one or more domains in addition to the AraC/XylS family domain , but a few family members consist only of this single domain , such as MarA and SoxS ( 13 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17513476" target="_blank">17513476</a>]<br></li>
<li>The SoxS and MarA proteins consist only of the conserved DNA binding domain ( ∼ 100 residues ) that is present in all AraC/XylS family members , whereas the Rob protein has an additional C-terminal domain ( ∼ 200 residues ) of unknown function .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10802742" target="_blank">10802742</a>]<br></li>
<li>The MarA protein is shorter than all other known members of this protein family except SoxS and appears to consist primarily of a putative DNA-binding domain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8383113" target="_blank">8383113</a>]<br></li>
<li>MarA is substantially shorter than other members of this family ( except SoxS ) , whose sizes range from 265 residues ( Rns ) to 321 residues ( XylS ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8383113" target="_blank">8383113</a>]<br></li>
<li>SoxS and MarA , along with the Tn10-encoded TetD protein of unknown function , are related to the XylS-AraC family of transcription regulators ( 13 ) but correspond only to the Cterminal ; 100-residue domain of this family ( Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7896685" target="_blank">7896685</a>]<br></li>
<li>These small proteins ( SoxS , 107 residues ; MarA , 127 residues ) are homologous to the C-terminus of the XylS-AraC family of proteins and are more closely related to a ; 100-residue segment in the N-terminus of Rob protein , which binds the right arm of the replication origin , oriC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7896685" target="_blank">7896685</a>]<br></li>
<li>However , in Rob the 100-residue domain homologous to SoxS and MarA is located at the N-terminus of the protein ; the C-terminal 175-residue segment of Rob is not related to XylS-AraC family members ( 40 ) ( Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7896685" target="_blank">7896685</a>]<br></li>
<li>In fact , MarA and SoxS , the proximalactivatorsof these systems , are closely related members of the AraC family of bacterial regulatory proteins ( Cohen et a / .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8866468" target="_blank">8866468</a>]<br></li>
<li>Both MarA and SoxS belong to the XylS/AraC family of transcription regulators ( 6 ) , and yet another member of this family , RobA , also increases acrAB expression ( 37 , 41 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12107134" target="_blank">12107134</a>]<br></li>
<li>Both MarA and SoxS belong to the XylS/AraC family of transcription regulators , and they share 42 % overall sequence identity ( Cohen et a / .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8821940" target="_blank">8821940</a>]<br></li>
<li>The SoxS protein is a member of the XylS/AraC family of transcription activators that includes two other proteins , MarA and Rob , that share sequence identity ( 41 and 55 % , respectively ) with SoxS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11741843" target="_blank">11741843</a>]<br></li>
<li>The three paralogous Escherichia coli AraC/XylS family activators MarA , SoxS , and Rob are regulated by three different systems ( marRAB , soxRS , and rob , respectively ) in response to different stresses ( 29 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21097628" target="_blank">21097628</a>]<br></li>
<li>The MarA protein is related to a family of positive transcriptional activators ( 23 ) , including SoxS , XylS , AraC , and others .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8383113" target="_blank">8383113</a>]<br></li>
<li>Rob , SoxS , and MarA comprise a closely related subset of members of the AraC/XylS family of transcription factors that activate transcription of both class I and class II promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22465792" target="_blank">22465792</a>]<br></li>
<li>MarA , SoxS and Rob are members of the AraC/XylS family of transcriptional regulators .65 These are evolutionarily related proteins that share sequence homologies at the N-terminal regions but differ in amino acid sequence at their C-terminal ends .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11601842" target="_blank">11601842</a>]<br></li>
<li>This is now known to be a part of the global regulation of porins mediated by three XylS-AraC family regulatory proteins , MarA , SoxS , and Rob ( 9 , 174 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14665678" target="_blank">14665678</a>]<br></li>
<li>Rob is , however , a member of a subgroup within the AraC/XylS family of transcriptional regulators , which includes the MarA and SoxS proteins that are involved in a wide range of regulatory functions ( 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16478729" target="_blank">16478729</a>]<br></li>
<li>Introduction MarA , SoxS , and Rob of Escherichia coli are highly homologous members of the AraC/XylS family of positive regulators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22515487" target="_blank">22515487</a>]<br></li>
<li>Its expression is regulated by three XylS/AraC family regulators , MarA , SoxS and Rob .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12791142" target="_blank">12791142</a>]<br></li>
<li>Underlining demonstrates the ammo acid identity between AraC and XylS ( which showed as much amino acid identity as did MarA with either AraC or XylS ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8383113" target="_blank">8383113</a>]<br></li>
<li>IscR is the first MarA Rob SoxS family member , to our knowledge , that has been shown to directly function as a repressor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11742080" target="_blank">11742080</a>]<br></li>
<li>IscR shares a similar amino acid sequence with MarA , a member of the Mar/Sox/Rob family of transcription factors ( Martin and Rosner 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12382038" target="_blank">12382038</a>]<br></li>
<li>IscR shares amino acid similarity with MarA , a member of the MarA SoxS Rob family of transcription factors .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11742080" target="_blank">11742080</a>]<br></li>
<li>IscR , a newly recognized member of the AraC family of transcriptional regulators and a `` distant '' MarA relative ( 20 % identity , 40 % amino acid similarity ) , was able to function in vitro as a transcriptional repressor of the iscRSUA operon ( 30 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14701822" target="_blank">14701822</a>]<br></li>
<li>MarA shares significant amino acid sequence similarity to members of the AraC family of transcriptional activators ( 7,8 ) , most notably SoxS , a positive regulator of the superoxide stress response whose induction also leads to a Mar phenotype ( 9 , 10 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8521301" target="_blank">8521301</a>]<br></li>
<li>Also noted was 43 % identity of MarA with OrfR ( of unknown function ) in TnlO ( 27a ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8383113" target="_blank">8383113</a>]<br></li>
<li>Percent identities between MarA and the other proteins are : MarA and SoxS , 39.3 % over a 107-amino-acid overlap ; MarA and Rob , 49.0 % over 104 amino acids ; MarA and TetD , 43.1 % over 109 amino acids .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8491710" target="_blank">8491710</a>]<br></li>
<li>Within this subfamily , the proteins encoded by soxS ( 1 , 41 ) , tetD ( 4 , 31 ) , marA ( 6 , 7 ) , and rob ( 37 ) are highly homologous to one another over the entire 107-amino-acid length of SoxS , with the extent of homology ranging from 42 % identity for the SoxS-MarA pair to 59 % identity for the TetD-Rob pair .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7896696" target="_blank">7896696</a>]<br></li>
<li>Rob , SoxS , MarA , and TetD are 42 to 59 % identical over the entire 107-amino-acid length of SoxS , the smallest of these four proteins ( 1 , 8 , 12 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626315" target="_blank">8626315</a>]<br></li>
<li>A vertical line ( ) above a given protein sequence indicates an amino acid identity between that protein and MarA , while a colon ( :) designates a functionally similar amino acid .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8491710" target="_blank">8491710</a>]<br></li>
<li>MarA shows significant amino acid sequence similarity to other E. coli proteins ( Fig. 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8491710" target="_blank">8491710</a>]<br></li>
				</div></ul><br>
			<label class="collapse_RP" for="_4"><font size="5px">Processes regulated by MarA</font></label>
				<input id="_4" type="checkbox">
				<div class="RP"><br><ul><li>Gene Description b0958 b1004 b0707 b0719 b0850 b1003 b1198 b2597 b2946 b3160 b3516 b3555 b4367 b4378 b3098 sulA wrbA ybgA ybgD ybjC yccJ ycgC yﬁA yggJ yhbW yhiX yiaG yjjS yjjV yqjD Inhibits cell-division and ftsZ ring formation Affects association between Trp repressor and operators in stationary phase Function unknown Putative ﬁmbria-like protein Function unknown Function unknown Function unknown Ribosome-associated factor , stabilizes ribosomes against dissociation Function unknown Putative monooxygenase Putative transcriptional regulator ( AraC/XylS family ) Putative transcriptional regulator Ferric hydroxamate transport Putative hydrolase Function unknown b3734 b3731 b3736 b3735 b3739 b0295 b3261 b3340 b2215 b0565 b4201 b3259 b4245 b3984 b3317 b3320 b3319 b3308 b3305 b4203 b3985 b3983 b3986 b3231 b3310 b3301 b3313 b3294 b3304 b3186 b3315 b3318 b3309 b3185 b3637 b3312 b3302 b1717 b3299 b3295 b0169 b3314 b3296 b3303 b3306 b3230 b3321 b3297 b3307 b2609 b3311 NaSal-down-regulated 0.4 0.4 0.3 0.3 0.4 0.4 0.2 0.4 0.5 0.3 0.4 0.3 0.4 0.3 0.3 0.5 0.3 0.4 0.3 0.2 0.3 0.4 0.3 0.4 0.3 0.4 0.2 0.3 0.4 0.3 0.1 0.4 0.3 0.4 0.4 0.3 0.3 0.4 0.3 0.4 0.4 0.4 0.4 0.3 0.3 0.3 0.3 0.5 0.3 0.3 0.3 atpA atpC atpF atpH atpI b0295 ﬁs fusA ompC ompT priB prmA pyrB rplA rplB rplC rplD rplE rplF rplI rplJ rplK rplL rplM rplN rplO rplP rplQ rplR rplU rplV rplW rplX rpmA rpmB rpmC rpmD rpmI rpmJ rpoA rpsB rpsC rpsD rpsE rpsH rpsI rpsJ rpsK rpsN rpsP rpsQ Membrane-bound ATP synthase , F1 sector , alpha subunit Membrane-bound ATP synthase , F1 sector , epsilon subunit Membrane-bound ATP synthase , F0 sector , subunit b Membrane-bound ATP synthase , F1 sector , delta subunit Membrane-bound ATP synthase subunit , F1-F0-type proton-ATPase Function unknown Transcriptional activator for rRNA operons Fusidic acid resistance ; protein chain elongation factor G Outer membrane protein , porin Outer membrane protein , protease VII Primosomal protein N Methyltransferase for 50S subunit L11 protein modiﬁcation Aspartate transcarbamylase , catalytic subunit 50S ribosomal subunit protein L1 50S ribosomal subunit protein L2 50S ribosomal subunit protein L3 50S ribosomal subunit protein L4 50S ribosomal subunit protein L5 50S ribosomal subunit protein L6 50S ribosomal subunit protein L9 50S ribosomal subunit protein L10 50S ribosomal subunit protein L11 50S ribosomal subunit protein L7/L12 50S ribosomal subunit protein L13 50S ribosomal subunit protein L14 50S ribosomal subunit protein L15 50S ribosomal subunit protein L16 50S ribosomal subunit protein L17 50S ribosomal subunit protein L18 50S ribosomal subunit protein L21 50S ribosomal subunit protein L22 50S ribosomal subunit protein L23 50S ribosomal subunit protein L24 50S ribosomal subunit protein L27 50S ribosomal subunit protein L28 50S ribosomal subunit protein L29 50S ribosomal subunit protein L30 50S ribosomal subunit protein A ( L35 ) 50S ribosomal subunit protein X ( L36 ) RNA polymerase , alpha subunit 30S ribosomal subunit protein S2 30S ribosomal subunit protein S3 30S ribosomal subunit protein S4 30S ribosomal subunit protein S5 30S ribosomal subunit protein S8 30S ribosomal subunit protein S9 30S ribosomal subunit protein S10 30S ribosomal subunit protein S11 30S ribosomal subunit protein S14 30S ribosomal subunit protein S16 30S ribosomal subunit protein S17 Continued on following page For those experiments involving induction of the SoxS and MarA transcription factors , we performed a single experiment .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11395452" target="_blank">11395452</a>]<br></li>
<li>This approach would offer additional advantages , such as the possibility of inhibiting only some MarA functions or the possibility of synergistic effects , since some of these genes also have a role in MDR or virulence We thank Francis Domain for performing the two-hybrid experiments and Laura McMurry for stimulating discussion .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20211899" target="_blank">20211899</a>]<br></li>
<li>This strain produces blue colonies on LB X-Gal ( 5-bromo-4-chloro-3-indolyl - - D-galactopyranoside ) plates and white colonies on LB X-Gal -- IPTG plates because MarA represses hdeAB expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20211899" target="_blank">20211899</a>]<br></li>
<li>175 , No. 10 JOURNAL OF BACTERIOLOGY , May 1993 , p. / 93/102888 -07 $ 02.00 / 0 Copyright © 1993 , American Society for Microbiology Overexpression of the MarA Positive Regulator Is Sufficient To Confer Multiple Antibiotic Resistance in Escherichia coli LAURA GAMBINO , STEPHEN J. GRACHECK , AND PAUL F. MILLER * Infectious Diseases , Experimental Therapy Department , Parke-Davis Pharmaceutical Research , Division of Warner-Lambert Company , Ann Arbor , Michigan 48106-1047 Received 14 December 1992/Accepted 8 March 1993 A genetic approach was undertaken to identify normal bacterial genes whose products function to limit the effective concentration of antibiotics .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8491710" target="_blank">8491710</a>]<br></li>
<li>Overproduction of cytoplasmic GadA and GadB and of several unidentiﬁed cytoplasmic membrane proteins as well as reduction in the amount of the outer membrane protein , OmpP , in the mppA null mutant indicate that MppA regulates a number of genes in addition to those already known to be controlled by MarA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10438753" target="_blank">10438753</a>]<br></li>
<li>Using in vitro assays and mutagenesis of promoter fusions in whole cells , we identiﬁed the cis regulatory element involved in MarA upregulation of the oxygen-insensitive nitroreductase nfnB gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12100559" target="_blank">12100559</a>]<br></li>
<li>MarA also mediated resistance to other antibiotics tested ( ampicillin , cephalothin , nalidixic acid , rifampin , erythromycin , tetracycline , and doxycycline ) but had no effect on susceptibility to aminoglycosides , trimethoprim-sulfamethoxazole , imipenem , or fosfomycin ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20211899" target="_blank">20211899</a>]<br></li>
<li>Several DNA-binding proteins , including H-NS ( Suzuki et al. , 1996 ) , HU ( Painbeni et al. , 1997 ) , Lrp ( Ferrario et al. , 1995 ) , Rob ( Ariza et al. , 1995 ) , SoxS ( Chou et al. , 1993 ) and MarA ( Cohen et al. , 1988 ; Jair et al. , 1995 ) , are known to regulate micF promoter activity and , consequently , modulate ompF mRNA translation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11029695" target="_blank">11029695</a>]<br></li>
<li>Some regulators are involved in the response-to-stressors , e.g. , response to alkylating agents ( Ada from E. coli , S. typhi-murium , and Mycobacterium tuberculosis and AdaA from Ba-cillus subtilis ) ( 57 , 100 , 176 , 177 ) ; response-to-oxidative-stress ( SoxS from E. coli and S. typhimurium ) ( 4 , 260 ) ; tolerance to antibiotics , organic solvents , and heavy metals ( AarP from Providencia stuartii , MarA and Rob from E. coli , PqrA from Proteus vulgaris , and RamA from Klebsiella pneumoniae ) ( 90 , 121 , 161 , 229 , 240 ) ; and transition from exponential growth t the stationary phase ( AppY from E. coli ) ( 12 , 131 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9409145" target="_blank">9409145</a>]<br></li>
<li>Evidence from the analysis of clinical isolates so far indicates a major role in resistance development for the primary targets of ﬂuoroquinolones , the topoisomerases II and IV , and stress response activators , namely MarA , SoxS , Ram and RobA .6,11 -- 13 As reported previously , the contribution of the mar regulon to ﬂuoroquinolone resistance in this genetically highly related pair of clinical E. coli isolates increased the MIC of ciproﬂoxacin from 16 to 256 mg/L .7 Considering the substantial inﬂuence of MarA expression on quinolone resistance ,14 more information about the repressor dysfunction of MarR is needed to understand resistance mechanisms against quinolones and other antibiotics .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11751765" target="_blank">11751765</a>]<br></li>
<li>This DNA-binding domain has also been shown to be involved in transcription activation in a number of AraC/XylS family proteins including Ada , RhaS , AraC , MelR , MarA , SoxS , XylS , and UreR ( 1 , 4 , 5 , 15 , 16 , 20 -- 22 , 37 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15342598" target="_blank">15342598</a>]<br></li>
<li>The analysis revealed a strong similarity between MarA and the family of positive regulators that includes several regulators involved in carbohydrate-metabolism in Escherichia coli ( AraC , RhaR , RhaS , and MelR ) , Erwinia corotovora ( AraC ) , and Pseudomonas putida ( XylS ) , virulence in Yersinia enterocolitica ( VirF ) and E. coli ( Rns ) , and oxidative stress response in E. coli ( SoxS ) ( 1 , 23 , 31 ) ( Fig. 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8383113" target="_blank">8383113</a>]<br></li>
<li>Plasmid MIC MIC Relative Relative ( M ) ( mM ) increase increase in MIC in MIC 12 35 31 22 42 57 35 55 M5390 M5391 M5392 M5393 M5394 M5395 M5396 M5397 Vector 0.5 1.8 1.6 1.2 1.4 2.4 0.9 3.5 WT MarA MarA ( E25R ) MarA ( E77A ) MarA ( L80A ) MarA ( E89A ) MarA ( Q91A ) WT SoxS 1.0 1.0 0.89 0.65 1.2 1.7 1.0 1.6 0.91 0.68 0.76 1.3 0.49 2.0 DISCUSSION The MarA-SoxS-Rob regulon of E. coli consists of a variety of genes that enable cells to adapt to multiple stresses .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21097628" target="_blank">21097628</a>]<br></li>
<li>However , Lon of E. coli also degrades speciﬁc substrates , such as cell-division inhibitor SulA ( Mizusawa and Gottesman , 1983 ) , transcriptional activators MarA , SoxS and Rob ( Griffith et al. , 2004 ; 2009 ) , idling ribosomal proteins ( Kuroda et al. , 2001 ) and antitoxins encoded by toxin-antitoxin modules ( Gerdes and Maisonneuve , 2012 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25491382" target="_blank">25491382</a>]<br></li>
<li>Moreover , expression of acrZ is coregulated with acrAB and tolC by MarA , Rob , and SoxS , a set of transcription factors that regulate cellular responses to antibiotics , detergents , and redox-cycling drugs ( 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23010927" target="_blank">23010927</a>]<br></li>
<li>We utilized microarray technology to identify genetic loci which differed in expression based on the presence or absence of the MarA , Rob , and SoxS regulators in response to conditions within the host .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22210763" target="_blank">22210763</a>]<br></li>
<li>The related MarA , SoxS and Rob regulators bind to the same promoter element and activate micF transcription in response to various stimuli , including weak acids , oxidative stress and certain antibiotics .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17007876" target="_blank">17007876</a>]<br></li>
<li>Expression of MicF is also controlled by the transcription regulators MarA , SoxS and Rob and the RNA mediates OmpF variation in response to various toxic factors ( Delihas and Forst , 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19400782" target="_blank">19400782</a>]<br></li>
<li>Similarly , recent results have reported an indirect effect for SoxS ( not MarA or Rob ) in activating the znuACB genes involved in zinc uptake and growth in the absence or limitation of zinc ( Warner & Levy , 2012 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22515487" target="_blank">22515487</a>]<br></li>
<li>Furthermore , using transposon mutagenesis , we have identiﬁed eight genes under Rob control , including six not previously connected to Rob , MarA , or SoxS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10850996" target="_blank">10850996</a>]<br></li>
<li>The MarA structurally related transcriptional activators , SoxS and Rob , also recognize the marbox sequence and activate a similar set of regulon genes ( Ariza et al. , 1995 ; Demple , 1996 ; Jair et al. , 1996a , b ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15225313" target="_blank">15225313</a>]<br></li>
<li>MarA , SoxS and Rob function as virulence factors in an Escherichia coli murine model of ascending pyelonephritis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20211899" target="_blank">20211899</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22210763" target="_blank">22210763</a>]<br></li>
<li>Two E. coli MarA homologues , SoxS and Rob also control , to different extents , genes in the mar/sox/rob regulon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12100559" target="_blank">12100559</a>]<br></li>
<li>Many bacteria also possess transcription factors , such as the homologous E. coli regulators MarA , Rob , and SoxS ( 3 , 4 ) , that respond to drugs by activating the expression of genes encoding proteins involved in antibiotic efﬂux and detoxiﬁcation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23010927" target="_blank">23010927</a>]<br></li>
<li>For example , the E. coli proteins MarA , Rob and SoxS all target genes involved in the oxidative stress response and bind to closely related DNA sequences .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9813115" target="_blank">9813115</a>]<br></li>
<li>However , the AcrR-mediated increase in reductive capacity in cells of the ΔfadR ΔmarR acrR + strain , was not observable , therefore the AcrR-mediated improvement of Table 2 Summary of the differences of organic solvent tolerances ( OSTs ) by deletion or overexpression ( OverEx ) of each gene by spot assay and the relative expression of candidate genes in background of acrR deletion ( ΔacrR ) or acrR overproduction ( acrR + ) by quantitative real-time PCR Genes Locusa Description OSTsb qRTc Deletiond acrR + OverEx ΔacrR Multidrug efflux pump -- acrR b0464 Acriflavine resistance regulator + -- + -- acrA b0463 Component of AcrAB-TolC multidrug efflux pump acrB b0462 Component of AcrAB-TolC multidrug efflux pump + △ △ -- -- △ △ -- -- △ △ tolC b3035 Outer membrane channel TolC Regulation of multidrug-resistance -- -- marA b1531 Multiple antibiotic resistance global regulator marR b1030 Multidrug-resistance transcriptional repressor + △ + △ + -- soxS b4062 Global regulator of superoxide response rob b4396 Paralogous transcriptional regulator with MarA and SoxS + △ nd nd nd △ △ nd Fatty acid biosynthesis fadR b1187 Fatty acid degradation regulator Fatty acid biosynthesis regulator + -- nd nd nd nd -- fabR b3963 fabA b0954 nd -- Dehydratase in the synthesis of unsaturated fatty acid β-ketoacyl-ACP synthase I nd nd nd nd nd △ nd fabB b2323 Acid stress response -- -- Lysine/cadaverine antiporter cadB b4132 cadC b4133 nd nd nd nd -- -- Metal-sensitive transcriptional activator -- -- nd Putative ATP dependent transporter of the MgtC family yhiD b3508 Cellular osmoprotectant nd △ -- betI b0313 gcvA b2808 Betaine inhibitor Glycine cleavage A nd nd -- -- nd nd Outer membrane protein ompF b0929 △ △ nd nd nd nd Member of the general bacterial porin Small outer membrane protein ompX b0814 micF b4439 + △ △ △ nd nd Small RNA negatively regulates translation of the OmpF Other genes slt △ Soluble lytic murein transglycosylase Potassium-dependent mechanosensitive channel b4392 b0465 nd nd nd mscK nd nd △ △ nd not determined a The locus tags is according to accession number in BioCyc database ( www.biocyc.org ) b Difference of OSTs was indicated as increased ( + ) , decreased ( − ) , or unchanged ( △ ) growth in colony formation of each strain compared with the wildtype strain c Difference of gene-expression was indicated as increased ( + ) , decreased ( − ) , or unchanged ( △ ) in ΔacrR and acrR + strain compared with the wild-type strain d Summary of OSTs was investigated in Oh et al. ( 2012 OST in the ΔfadR ΔmarR acrR + strain was not only due to an increase in the reductive capacity of cells .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25176444" target="_blank">25176444</a>]<br></li>
<li>Interruptionof marA by Tn5 insertion results in drug hypersusceptibility.Based on these and other results , Cohen et al. ( 1993a ) have proposed that MarA acts as a global transcription activator which is produced in increased amounts in response-to-antibiotic stress .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8821940" target="_blank">8821940</a>]<br></li>
<li>SoxS Increases the Expression of the Zinc Uptake System ZnuACB in n Escherichia coli Murine Pyelonephritis Model a Douglas M. Warnera , b and Stuart B. Levya , b , c Center for Adaptation Genetics and Drug Resistancea and Departments of Molecular Biology and Microbiologyb and of Medicine , c Tufts University School of Medicine , Boston , Massachusetts , USA Paralogous transcriptional regulators MarA , Rob , and SoxS act individually and together to control expression of more than 80 Escherichia coli genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22210763" target="_blank">22210763</a>]<br></li>
<li>The Escherichia coli Transcriptional Regulator MarA Directly Represses Transcription of purA and hdeA * Received for publication , December 11 , 2003 , and in revised form , December 22 , 2003 Published , JBC Papers in Press , December 29 , 2003 , DOI 10.1074 / jbc.M313602200 Thamarai Schneiders ‡ , Teresa M. Barbosa ‡ § , Laura M. McMurry ‡ , and Stuart B. Levy ‡ ¶ From the ‡ Center for Adaptation Genetics and Drug Resistance and the Departments of Molecular Biology and Microbiology and of ¶ Medicine , Tufts University School of Medicine , Boston , Massachusetts 02111 The Escherichia coli MarA protein mediates a response to multiple environmental stresses through the activation or repression in vivo of a large number of chromosomal genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14701822" target="_blank">14701822</a>]<br></li>
<li>Printed in the U.S.A. Transcriptional Repression of the rob MarA-mediated Promoter * Received for publication , November 9 , 2005 , and in revised form , February 8 , 2006 Published , JBC Papers in Press , February 14 , 2006 , DOI 10.1074 / jbc.M512097200 Thamarai Schneiders ‡ and Stuart B. Levy ‡ § 1 From the ‡ Center for Adaptation Genetics and Drug Resistance and the Departments of Molecular Biology and Microbiology and of § Medicine , Tufts University School of Medicine , Boston , Massachusetts 02111 The Escherichia coli transcriptional regulator MarA affects functions that include antibiotic resistance , persistence , and survival .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16478729" target="_blank">16478729</a>]<br></li>
<li>Role of the Multidrug Resistance Regulator MarA in Global Regulation of the hdeAB Acid Resistance Operon in Escherichia coli † Cristian Ruiz ,1,2 Laura M. McMurry ,1,2 and Stuart B. Levy1 ,2,3 * Center for Adaptation Genetics and Drug Resistance1 and the Departments of Molecular Biology and Microbiology2 and of Medicine ,3 Tufts University School of Medicine , Boston , Massachusetts 02111 Received 29 October 2007/Accepted 30 November 2007 MarA , a transcriptional regulator in Escherichia coli , affects functions such as multiple-antibiotic resistance ( Mar ) and virulence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18083817" target="_blank">18083817</a>]<br></li>
<li>Periplasmic Protein MppA Requires an Additional Mutated To Repress marA Expression in Escherichia coli The Locus Xiaowen Bina ,1,2 Vincent Perreten ,1,2 † and Stuart B. Levy1 ,2,3 * Center for Adaptation Genetics and Drug Resistance1 and Departments of Molecular Biology and Microbiology2 and of Medicine ,3 Tufts University School of Medicine , Boston , Massachusetts 02111 Received 14 August 2002/Accepted 20 November 2002 Escherichia coli strain TP985 , which has an insertional mutation in the gene for the periplasmic murein tripeptide binding protein MppA , was previously reported to overproduce MarA and exhibit a multiple-antibiotic resistance ( Mar ) phenotype ( H. Li and J. T. Park , J. Bacteriol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12562820" target="_blank">12562820</a>]<br></li>
<li>MarA also upregulates superoxide resistance ( acnA , fumC , fpr , sodA , and zwf ) and DNA-repair ( nfo ) , among other effects ( 5 , 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20211899" target="_blank">20211899</a>]<br></li>
<li>For instance , the expression of acrAB in E. coli is controlled by repressor protein AcrR , global activator protein MarA , SoxS , Rob and quorum-sensing regulator LuxR homologue protein SdiA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15686558" target="_blank">15686558</a>]<br></li>
<li>A homologous MarR repressor binds salicylate and controls the expression of a global regulator , MarA , in E. coli ( 14 -- 16 , 19 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10438794" target="_blank">10438794</a>]<br></li>
<li>The other unit is an operon comprising marRAB , encoding the MarR repressor , which binds to marO and negatively regulates expression of marRAB ( 24 , 33 ) ; MarA , a transcriptional activator ( 1 ) , which activates expression of other * Corresponding author .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10438753" target="_blank">10438753</a>]<br></li>
<li>The multidrug-resistant operon , marRAB , encodes a repressor , MarR , an activator , MarA , and an unknown function protein , MarB , which coordinately regulate the mar transcription to maintain intrinsic resistance to antimicrobial substances ( 8 , 9 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25294881" target="_blank">25294881</a>]<br></li>
<li>Introduction Fluoroquinolone resistance of clinical Escherichia coli isolates was shown to involve target mutations in topoisomerases II ( GyrA/B ) and IV ( ParC/E ) , often in combination with lowered drug accumulation due to multidrug efﬂux pumps .1 -- 5 Efﬂux via the AcrAB efﬂux system is positively regulated by the transcription factor MarA , which itself is under negative control by the repressor MarR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11751765" target="_blank">11751765</a>]<br></li>
<li>The increased chloramphenicol efﬂux in the resistant Mar mutants ( 28 ) indicates a role for MarA in the transcriptional regulation of the chloramphenicol efﬂux gene ( s ) in E. coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8655497" target="_blank">8655497</a>]<br></li>
<li>This obvious similarity to proteins involved in positive regulation suggests that MarA may be a transcriptional activator of genes involved in the Mar phenotype .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8383113" target="_blank">8383113</a>]<br></li>
<li>vesicatoria Xanthomonas oryzae Salmonella typhimurium HrpXc HrpXv HrpXo InvF 502 216 Similar to B. solanacearum HrpB 193 127 TE : Q56831 SP : P39437 Transcriptional activator of the inv operon required for epithelial tissue invasion Transcriptional activator of the virulence regulon ( similar to Y. enterocolitica VirF ) Probable transcriptional regulator Unknown Transcriptional activator of the maoA gene coding a monoamine oxidase Transcriptional activator of the sodA , zwf , micF , slp , fpr , fumC , and nfo genes , which are involved in the multiple antibiotic resistance ( mar ) phenotype Similar to E. coli MarA Transcriptional activator of the melAB operon for transport and catabolism of melibiose Transcriptional activator of the mmsAB operon for valine-catabolism Transcriptional activator of the msm operon ( msmEFGK , aga , dexB , gftA ) required for the transport of melibiose , rafﬁnose , and isomaltotriose and for melibiose , saccharose , and isomaltosaccharide catabolism Transcriptional activator of mxi and spa operons involved in the synthesis and secretion of the Ipa proteins required for the epithelial tissue invasion Similar to S. ﬂexneri MxiE Transcriptional activator of nitA , which codes for a nitrilase Probable transcriptional activator of the orni-thine utilization operon Unknown Unknown Unknown Transcriptional activator of the pyochelin and ferripyochelin receptor Transcriptional activator of the eaeA gene for intimin , a protein required for adherence to the host-cell-membrane , in enterohemorrhagic and enteropathogenic E. coli strains Probable transcriptional activator of pobA , which codes the p-hydroxybenzoate hydroxylase Transcriptional activator of cbiABCDETF-GHJKLMNQOP and cobUST operons , required for the adenosyl-cobalamine ( vitamin B12 ) synthesis , and pduABC and pduF , required for 1,2-propanediol-catabolism .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9409145" target="_blank">9409145</a>]<br></li>
<li>MarA produces intrinsic multiple antibiotic resistance ( Mar ) mainly by upregulating the expression of acrAB and tolC , which encode AcrAB-TolC , the major multidrug efﬂux pump in E. coli ( 28 , 39 ) , and by upregulating micF , a regulatory antisense RNA which downregulates ompF translation and thus the OmpF porin-mediated outer membrane permeability to antibiotics .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20211899" target="_blank">20211899</a>]<br></li>
<li>These and other data reported herein indicate that MppA functions upstream of MarA in a signal-transduction-pathway to negatively regulate the expression of marA and hence of the MarA-driven multiple antibiotic resistance .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10438753" target="_blank">10438753</a>]<br></li>
<li>MarA is a transcriptional activator of the mar regulon , a dozen or more genes whose expression renders Escherichia coli partially resistant to various antibiotics and superoxide-gener-ating agents ( see references in reference 27 ) and tolerant of organic solvents ( 14 , 45a ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9393706" target="_blank">9393706</a>]<br></li>
<li>Barbosa , T. M. , Levy , S. B. : Differential expression of over 60 chromosomal genes in Escherichia coli by constitutive expression of MarA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12195740" target="_blank">12195740</a>]<br></li>
<li>Summary MarA is a global regulator that mediates resistance to multiple environmental hazards such as antibiotics , disinfectants and oxidative stress agents by modulating the expression of a large number of genes in the Escherichia coli chromosome .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12100559" target="_blank">12100559</a>]<br></li>
<li>The MarA global regulatory protein in E. coli is known to control expression of multiple chromosomal genes affecting resistance to antibiotics and other environmental hazards , such as the multidrug efflux genes acrAB and tolC ( Aono , 1998 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11321589" target="_blank">11321589</a>]<br></li>
<li>MarA , an AraC/XylS transcriptional regulator of Esche-richia coli , directly activates or represses multiple chromosomal genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18083817" target="_blank">18083817</a>]<br></li>
<li>The Escherichia coli MarA protein , a member of the AraC / XylS family of transcriptional regulators , mediates cellular responses to stress through the differential control of a large number of chromosomal genes comprising the mar regulon ( 1 -- 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14701822" target="_blank">14701822</a>]<br></li>
<li>The MarA protein of Escherichia coli activates the transcription of the genes of the mar regulon resulting in increased resistance to antibiotics and super-oxide-generating compounds ( Alekshun & Levy , 1997 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10873449" target="_blank">10873449</a>]<br></li>
<li>MarA , produced from the marRAB operon , affects a variety of genes associated with antibiotic resistance ( 11 ) and virulence ( 10 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22210763" target="_blank">22210763</a>]<br></li>
<li>Role of the multidrug-resistance regulator MarA in global regulation of the hdeAB acid resistance operon in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20211899" target="_blank">20211899</a>]<br></li>
<li>Usually an activator , MarA is a repressor of hdeAB and other acid resistance genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18083817" target="_blank">18083817</a>]<br></li>
<li>MICROBIOLOGY LETTERS R E S E A R C H L E T T E R SoxS-dependent coregulation of ompN and ydbK in a multidrug-resistant Escherichia coli strain Anna Fàbrega1 , Judah L. Rosner2 , Robert G. Martin2 , Mar Solé1 & Jordi Vila1 ,3 1 2 Barcelona Centre for International Health Research , CRESIB , Hospital Clınic-Universitat de Barcelona , Barcelona , Spain ; Laboratory of Molecular Biology , National Institute of Diabetes and Digestive and Kidney Diseases , National Institutes of Health , Bethesda , MD , USA ; and Department of 3 Microbiology , Hospital Clınic , School of Medicine , University of Barcelona , Barcelona , Spain Correspondence : Jordi Vila , Servei de Microbiologia , Hospital Clınic , Facultat de Abstract SoxS , MarA , and Rob are homologous transcriptional activators of numerous superoxide - and antibiotic resistance genes but many of the regulated genes are yet to be characterized .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22515487" target="_blank">22515487</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_55">
Grifﬁth , K.L. , Shah , I.M. , and Wolf , R.E. , Jr ( 2004 ) Proteolytic degradation of Escherichia coli transcription activators SaxS and MarA as the mechanism for reversing the induction of the superoxide ( Sox RS ) and multiple antibiotic resistance ( mar ) regulons .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15225313" target="_blank">15225313</a>]<br></label>
<input id="rp_similarity_55", type="checkbox">
<div class="RP_similar">
<li>Griffith , K. L. , Shah , I. M. , and Wolf , R. E. , Jr. ( 2004 ) Proteolytic degradation of Escherichia coli transcription activators SoxS and MarA as the mechanism for reversing the induction of the superoxide ( SoxRS ) and multiple antibiotic resistance ( Mar ) regulons .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23091052" target="_blank">23091052</a>]<br></li>
<li>Griffith , K.L. , I. , Shah , M. , and Wolf , R.E. , Jr ( 2004 ) Proteo-lytic degradation of Escherichia coli transcription activators SoxS and MarA as the mechanism for reversing the induction of the superoxide ( SoxRS ) and multiple antibiotic resistance ( Mar ) regulons .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25491382" target="_blank">25491382</a>]<br></li></div>
				<label class="RP_collapse_similarity" for="rp_similarity_58">
Proteolytic degradation of Esch-erichia coli transcription activators SoxS and MarA as the mechanism for reversing the induction of the superoxide ( SoxRS ) and multiple antibiotic resistance ( Mar ) regulons .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21097628" target="_blank">21097628</a>]<br></label>
<input id="rp_similarity_58", type="checkbox">
<div class="RP_similar">
<li>Proteolytic degradation of Escherichia coli transcription activators SoxS and MarA as the mechanism for reversing the induction of the superoxide ( SoxRS ) and multiple antibiotic resistance ( Mar ) regulons .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20211899" target="_blank">20211899</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18083817" target="_blank">18083817</a>]<br></li></div>
				<label class="RP_collapse_similarity" for="rp_similarity_60">
Puri ® cation and regulatory properties of MarA protein , a transcriptional activator of Escherichia coli multiple antibiotic and superoxide resistance promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9813115" target="_blank">9813115</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10873449" target="_blank">10873449</a>]<br></label>
<input id="rp_similarity_60", type="checkbox">
<div class="RP_similar">
<li>Puriﬁcation and regulatory properties of MarA protein , a transcriptional activator of Escherichia coli multiple antibiotic and superoxide resistance promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8655497" target="_blank">8655497</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10940041" target="_blank">10940041</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15342598" target="_blank">15342598</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14594836" target="_blank">14594836</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626315" target="_blank">8626315</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9393706" target="_blank">9393706</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9473050" target="_blank">9473050</a>]<br></li>
<li>Puriﬁcation and regulatory properties of MarA protein , a transcriptional activator of Escherichia coli multiple antibiotic resistance and superoxide resistance promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8636021" target="_blank">8636021</a>]<br></li>
<li>Purification and regulatory properties of MarA protein , a transcriptional activator of Escherichia coli multiple antibiotic and superoxide resistance promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18366439" target="_blank">18366439</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9019397" target="_blank">9019397</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22465792" target="_blank">22465792</a>]<br></li></div>
				<label class="RP_collapse_similarity" for="rp_similarity_64">
, Martin , R.G. , Rosner , J.L. , Nobuyuki , F. , Ishihama , A. and Wolf , R.E. ( 1995 ) Purification and regulatory properties of MarA , a transcriptional activator of Escherichia coli multiple antibiotic and superoxide resistance properties .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12060687" target="_blank">12060687</a>]<br></label>
<input id="rp_similarity_64", type="checkbox">
<div class="RP_similar">
<li>, Martin , R.G. , Rosner , J.L. , Fujita , N. , Ishihama , A. , and Wolf , Jr , R.E. ( 1995 ) Puri ® cation and regulatory properties of MarA protein , a transcriptional activator of Escherichia coli multiple antibiotic and superoxide resistance promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10564485" target="_blank">10564485</a>]<br></li>
<li>, Martin , R.G. , Rosner , J.L. , Fujita , N. , Ishihama , A. , and Wolf , R.E. , Jr ( 1995 ) Purification and regulatory properties of MarA protein , a transcriptional activator of Escherichia coli multiple antibiotic and superoxide resistance promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11029695" target="_blank">11029695</a>]<br></li>
<li>, Martin , R.G. , Rosner , J.L. , Fujita , N. , Ishihama , A. , and Wolf , R.E. , Jr. ( 1995 ) Puriﬁcation and regulatory properties of MarA protein , a transcriptional activator of Escherichia coli multiple antibiotic resistance and superoxide resistance promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12100559" target="_blank">12100559</a>]<br></li>
<li>, Martin , R.G. , Rosner , J.L. , Fujita , N. , Ishihama , A. , and Wolf , R.E. , Jr. ( 1995 ) Puriﬁcation and regulatory properties of MarA protein , a transcriptional activator of Escherichia coli multiple antibiotic and superoxide resistance promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12791142" target="_blank">12791142</a>]<br></li></div>
<li>A clue that the genes encoding Fe-S cluster assembly proteins might be regulated came from the observation that immediately upstream of iscSUA in E. coli and several other bacteria , there is a gene ( here called iscR ) whose protein product is similar to the well characterized transcriptional regulator MarA ( multiple antibiotic resistance ) from E. coli ( Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11742080" target="_blank">11742080</a>]<br></li>
<li>As is observed in the multiple antibiotic resistance phenotype in E. coli cells , the mppA null mutant overproduces the transcriptional activator , MarA , resulting in expression of the membrane-bound AcrAB proteins that function as a drug efﬂux pump .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10438753" target="_blank">10438753</a>]<br></li>
<li>MarA is believed to be a transcriptional activator of genes involved in the Mar phenotype , since alone it causes increased resistance to multiple antibiotics ( 5 , 11 , 31 , 33 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7836283" target="_blank">7836283</a>]<br></li>
<li>The MarA protein is a transcriptional activator that controls the expression of the genes in the Mar regulon , leading to increased antibiotic resistance [ 8,9 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11094278" target="_blank">11094278</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_73">
Martin R , Rosner J. Fis , an accessorial factor for transcriptional activation of the mar ( multiple antibiotic resistance ) promoter of Escherichia coli in the presence of the activator MarA , SoxS , or Rob .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22465792" target="_blank">22465792</a>]<br></label>
<input id="rp_similarity_73", type="checkbox">
<div class="RP_similar">
<li>Fis , an accessorial factor for transcriptional activation of the mar ( multiple antibiotic resistance ) promoter of Escherichia coli in the presence of the activator MarA , SoxS , or Rob .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10873449" target="_blank">10873449</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10368169" target="_blank">10368169</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14679232" target="_blank">14679232</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20211899" target="_blank">20211899</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10850996" target="_blank">10850996</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21097628" target="_blank">21097628</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16885445" target="_blank">16885445</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10572146" target="_blank">10572146</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10894725" target="_blank">10894725</a>]<br></li></div>
<li>First , three independently regulated but homologous proteins , MarA , SoxS and Rob , activate a common set of promoters which results in multiple antibiotic resistance , superoxide resistance and organic solvent tolerance ( for review , see Alekshun and Levy , 1997 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10564485" target="_blank">10564485</a>]<br></li>
<li>Some of the proteins that regulate stress response genes are SoxS ( oxidative stress , E. coli ) , MarA ( multiple antibiotic resistance , E. coli ) , Ada ( alkylating agents , E. coli ) , and Rob ( originally identiﬁed as binding to oriC but cross-activates SoxS - and MarA-regulated genes , E. coli ) ( 6 , 18 -- 21 , 30 , 39 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10464186" target="_blank">10464186</a>]<br></li>
<li>The closest SoxS paralogs are MarA , a protein involved in multiple-antibiotic resistance ( 1 ) , and Rob , an abundant protein of uncertain physiological relevance ( 6 , 46 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14594836" target="_blank">14594836</a>]<br></li>
<li>Of interest are the overlapping regulons of the MarA , Rob , and SoxS transcriptional regulators , which control many functions , including antibiotic resistance ( 10 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22210763" target="_blank">22210763</a>]<br></li>
<li>Using gene arrays , Pomposiello et al. ( 33 ) found that overexpression of MarA from an isopropyl - - D-thiogalactopyrano-side ( IPTG ) - inducible plasmid repressed hdeAB and other genes involved in acid resistance ( gadA , gadB , gadC , hdeD , and slp ) , whereas sodium salicylate activated these same genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18083817" target="_blank">18083817</a>]<br></li>
<li>Both sRNAs regulate the mRNA encoding MarA , which is involved in the response to antimicrobial compounds and oxidative stress ( 46 ) , and of the mRNA for the β-N-acetylglucosaminidase NagZ , which permits resistance to β-lactams in Pseudomonas aeruginosa ( 48 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23980183" target="_blank">23980183</a>]<br></li>
<li>Interestingly , both MarA and NagZ are not obviously involved in iron-homeostasis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23980183" target="_blank">23980183</a>]<br></li>
<li>They also show an interconnection of the MarA system with global regulation and cell metabolism .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20211899" target="_blank">20211899</a>]<br></li>
<li>Surprisingly , global microarray analysis of an E. coli strain constitutively expressing MarA only affected expression of two other possible efflux systems : yadGH , encoding an ABC efflux system of unknown specificity and ydeA , an MFS sugar and drug efflux system ( Barbosa and Levy , 2000 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11321589" target="_blank">11321589</a>]<br></li>
<li>Modi-ﬁcation of marA expression ( directly or indirectly , e.g. , via crp , cyaA , pcnB , or tolC ) is a mechanism of generally altering the expression of the MarA regulon , which may thus affect other MarA-regulated functions such as virulence ( 1 , 6 , 9 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20211899" target="_blank">20211899</a>]<br></li>
<li>Finally , inhibitors of multidrug efﬂux pumps and inhibitors of MDR and virulence regulators , including MarA , have been proposed to enhance the effectiveness of antibiotics ( 7 , 28 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20211899" target="_blank">20211899</a>]<br></li>
<li>The CpxR/CpxA response induces the expression of a variety of genes encoding MarA , MarR , AroK , and other envelope folding factors DsbA , DegP , etc. .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25294881" target="_blank">25294881</a>]<br></li>
<li>CpxR-induced MarA activates the genes encoding TolC and other components of tripartite multidrug-transporters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25294881" target="_blank">25294881</a>]<br></li>
<li>The product of marA , the transcription factor MarA , directly activates or represses many chromosomal genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20211899" target="_blank">20211899</a>]<br></li>
<li>Among those that are regulated by MarA are many genes coding for proteins that are important in antimicrobial resistance , including the OmpF porin and the AcrAB multiple-drug efﬂux system ( 6 , 9 , 17 , 18 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12562820" target="_blank">12562820</a>]<br></li>
<li>MarA is thought to induce the expression of several genes , including genes for efﬂux pumps ( 14 , 17 , 21 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8655497" target="_blank">8655497</a>]<br></li>
<li>The global transcriptional regulator MarA activates the expression of over 60 genes including those of the multidrug efflux pump AcrAB-TolC ( Ruiz and Levy 2010 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25176444" target="_blank">25176444</a>]<br></li>
<li>The MarA protein controls expression of some of the same genes as SoxS , including many that confer resistance to antibiotics4 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10802742" target="_blank">10802742</a>]<br></li>
<li>MarA activates the expression of diverse genes , including acrAB , micF , mlr-1 , -2 , and -3 , slp , and inaA , which endow cells with resistance ( 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19376854" target="_blank">19376854</a>]<br></li>
<li>Additional effects on the expression of many genes may occur through Robmediated activation of other regulatory proteins ( MarA , AslB , and YbaO ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10850996" target="_blank">10850996</a>]<br></li>
<li>These genes encode unrelated transcriptional regulators , enzymes , transporters , and unknown proteins and can affect MarA response in general ( by altering marA expression or MarA function ) or speciﬁcally only for some antibiotics .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20211899" target="_blank">20211899</a>]<br></li>
<li>Therefore , hns plays an essential role in MarA-mediated MDR by being necessary for full MarA induction of acrA expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20211899" target="_blank">20211899</a>]<br></li>
<li>( B ) Effect of inactivation of each gene on MarA induction of acrA , tolC , and micF expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20211899" target="_blank">20211899</a>]<br></li>
<li>Such genes , identiﬁed just because of the screening method used , were not expected to affect MarA-mediated MDR ; however , some may still be relevant for MarA-mediated MDR if , independently , they were also involved in regulation of marA expression or MarA function ( for example , global regulators and other genes with pleiotropic effects ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20211899" target="_blank">20211899</a>]<br></li>
<li>MarA also controls hdeAB expression by modifying the effect of other regulators of hdeAB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18083817" target="_blank">18083817</a>]<br></li>
<li>At a single-gene level , the function of MarA on at least one operon , hdeAB , which is involved in acid resistance , depends also on growth phase , pH , and other regulators of hdeAB ( 42 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20211899" target="_blank">20211899</a>]<br></li>
<li>In conclusion , MarA plays an active and complex role in the regulation of hdeAB and acid resistance in E. coli that depends on the growth conditions , on other regulators of hdeAB , and on the degree of induction of marA expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18083817" target="_blank">18083817</a>]<br></li>
				</div></ul><br>
			<label class="collapse_SIT" for="_5"><font size="5px">MarA binding site</font></label>
				<input id="_5" type="checkbox">
				<div class="SIT"><br><ul><li>sigma scores boxes evidence 1 glkP1 - P2 k n glk FruR C [ 36 ] , in this work GCGTTGTTGTTATGCCCCCAGGTATTTACAGTGTGAGAA 236 s38 2 s70 : 3.92 glkP3 - P4 n n s70 3 in this work A , C CCCCCAGGTATTTACAGTGTGAGAAAGAATTATTTTGAC 221 s38 : 2.95 pgiP1 - P2 k n [ 37 ] , in this work A , B , E ¬ pgi ACCATCACATTTTTCTGTGACTGGCGCTACAATCTTCCA 236 s70 : 6.9 ; s38 : 7.44 pfkAP1 - P2 k n ¬ A , B FruR pfkA [ 39 ] , in this work ATCAATTCAGCAGGAAGTGATTGTTATACTATTTGCACA 278 s70 : 8.61 ; s38 : 5.7 pfkAP3 - P4 n n s38 2 in this work A , E TCACTTCGATGTGCAAGAAGACTTCCGGCAACAGATTTC 229 s70 : 1.12 ki CTGCCCTGCGGGGCGGCCATCTTCCTTTATTCGCTTATA 219 ¬ tpiA tpiAP [ 40 ] ¬ ¬ tpiAP1 - P2 in this work n n ¬ A , E CAAAGCCTTTGTGCCGATGAATCTCTATACTGTTTCACA 262 s70 : 8.07 ; s38 : 7.4 gapA gapAP1 - P1a [ 41,43 ] , in k n this work gapAP2 [ 41,43 ] k ¬ A , B , D CGCTTGACGCTGCGTAAGGTTTTTGTAATTTTACAGGCA 236 s70 : 3.76 ; s38 : 0.82 ¬ A CCTTTAAAATTCGGGGCGCCGACCCCATGTGGTCTCAAG 2153 ¬ k TCACATTTTTATCGTAATTGCCCTTTAAAATTCGGGGCG 2174 - - ¬ gapAP3 [ 41,43 ] Crp k AACACCAACTGGCAAAATTTTGTCCTAAACTTGATCTCG 2245 - - ¬ gapAP4 [ 41,43 ] ACATTCCTTTTATTCCACGTTTCGCTTATCCTAGCTGAA 2132 no predictions s703 , s382 A , D epdP1 - P2 k FruR , Crp epd [ 45 ] , in this work pgkP1 k A pgk [ 45 ] AAGCAGCACAAGGTGCATTTCATGGTATAGTTGACTATA 2235 - ¬ n TGTTGCTTTCAGGTAAGACGCAAGCAGCGTCTGCAAAAC 226 no predictions ¬ pgkP2 in this work D , E n GCACACCTGATCAAAACGTTGGTCTGGTGCGATAACGAA 2113 no predictions s703 pgkP3 in this work A , D gpmA gpmAP1 - P2 [ 46 ] , in this ki n work gpmAP3 - P4 in this work n n Fur A ¬ AAGCATTGCTGTTGCTTCGTCGCGGCAATATAATGAGAA 237 s70 :7.56 ; s38 : 3.66 ¬ ¬ CCTTACACTGCGCCACTATTTTCGCTATGGTTATGCGTA 275 s70 :5.80 ; s38 : 7.48 eno enoP1 [ 47 ] k ¬ ¬ AATCGTGTCAGCGTCAACATCAAACTGATCGATTCACAA 2731 ¬ k CGAAGCACTGAAACACGGTGGGCTGAAGAATCGTGTCAG 2760 - - ¬ enoP2 [ 47 ] k TAAACGATTCAGCTTAAACTGCCCGGAAGCGAATCTGTC 2903 - - ¬ enoP3 [ 47 ] n GCGTACCCTGGGTACGCGTTGTTTGTCTGGAGTTTCAGT 227 no predictions s703 , s382 A , D , E enoP4 - P5 in this work n GAGTTCCAGAAACGTCAGGCGAAGTAAGTAAAAAAGTTA 277 no predictions - D enoP6 in this work n n GGGGATGATCAGTTGGTCGAGATCATCGAAGTTCCGAAT 2212 no predictions s703 , s382 D enoP7 - P8 in this work pykF pykFPPki [ 49 ] FruR TGTTTTCCTTTTGGATTAATTTCAGCGTATAATGCGCGC 2231 - -- pykFP1 - P2 in this work n n GGATTCGCTTTCCGGCAGTGCGCCCAGAAAGCAAGTTTC 234 no predictions s703 , s382 A , B , D n GATGTCACCTATCCTTAGAGCGAGGCACCACCACTTTCG 280 no predictions - D pykFP3 in this work pykFP4 - P5 n n in this work CCTCTGCACGCTTTTTCGATGTCACCTATCCTTAGAGCG 297 s70 : 4.79 ; s38 : 9.95 ¬ B , D pykA pykAP1 - P2 n n s38 2 in this work AAGTGACGATCGCTAAAAACGACTGTCACTGTCCTAATC 2105 s70 : 0.90 , D B n n GTCAAAGAAGCGCTGAAGGAATCGCGTTTTGATAAGCAG 2153 s70 : 1.76 s382 A , B , D pykAP3 - P4 in this work Crp , Fnr , GTATGGACATAAGGTGAATACTTTGTTACTTTAGCGTCA 258 no predictions s703 , s382 B , D PdhR s38 2 pdhR pdhRP1 - P2 k n [ 51 -- 52 ] , in this work aceE aceEP1 - P2 k n ¬ AACCTGTCTTATTGAGCTTTCCGGCGAGAGTTCAATGGG 248 s70 : 1.90 [ 50 ] , in this work poxB poxBP1 - P2 k n [ 57,58 ] , in his work B , C ¬ MarA , SoxS TCCCTTCCCCCTCCGTCAGATGAACTAAACTTGTTACCG 227 s70 : 5.31 ; s38 : 9.5 acs acsP1 k [ 60 -- 61 ] ¬ Fis , Crp , Ihf TCTTTAATCAATTGTAAGTGCATGTAAAATACCACTTTA 2224 ¬ k n CCCCTACATTTAACGCTTATGCCACATATTATTAACATC 220 s70 : 4.03 ; this work s38 : 4.97 B , C ¬ acsP2 - P3 [ 60 -- 61 ] , in Fis , Crp , Ihf consensus sigma 70 : TGGTATAATG sigma 38 : CTACACTT sequences Proposed promoters in which putative s and s overlapping recognition sequences have been determined , are presented as bold sequences .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19838295" target="_blank">19838295</a>]<br></li>
<li>Note the lack of MarA protection at the bottom of the gel , where the sequences deleted in DmarO280 are clearly visible .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8636021" target="_blank">8636021</a>]<br></li>
<li>Two conserved thymines within the B-box ( positions 4 ´ and 5 ´ ) have been suggested to contribute to sequence specific interactions with MarA in the form of van der Waals contacts to the C5 methyl groups13 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10802742" target="_blank">10802742</a>]<br></li>
<li>From 2 liters of culture , 35 mg of MarA was obtained which appeared as a single homogeneous band of approximately 15,000 Da , the molecular mass of MarA deduced from the marA DNA sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8522515" target="_blank">8522515</a>]<br></li>
<li>( 2000 ) 299 , 1245 ± 1255 Probing the s n E cherichia coli Tra scriptional Activator g Alanine-scanning enesis : Residue N n MarA usin portant s Mutag Im for D A Binding an d Activatio William K. Gillette * , Robert G. Martin and Judah L. Rosner Laboratory of Molecular Biology , National Institute of Diabetes and Digestive and Kidney Diseases , National Institutes of Health , Bethesda MD 20892-0560 , USA # 2000 Academic Press Keywords : marbox ; backbone contacts ; RNA polymerase ; hydrophobic core ; AraC * Corresponding author Introduction The MarA transcriptional activator binds to a 20 bp asymmetric degenerate sequence ( marbox ) located at different positions and orientations within the promoters of the genes of the Escherichia coli mar regulon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10873449" target="_blank">10873449</a>]<br></li>
<li>MarA binds to a marbox that is highly divergent from the previously proposed consensus ( eight differences out of 14 speciﬁed nucleotides ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12100559" target="_blank">12100559</a>]<br></li>
<li>Recently , Barbosa and Levy showed that the nfnB gene is regulated preferentially by MarA , which binds to a site that diverges substantially from the Mar-Sox-Rob consensus , with eight mismatches out of the 14 nonrandom positions ( 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14594836" target="_blank">14594836</a>]<br></li>
<li>In particular , recognition helix-3 and helix-6 of MarA 's dual HTH motifs bind RE1 and RE2 , respectively , of mar 's robbox , while helix-3 within Rob 's N-terminal HTH motif binds RE1 of micF 's robbox , but helix-6 does not bind RE2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22465792" target="_blank">22465792</a>]<br></li>
<li>Thus , MarA and MarR appear to bind to independent sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8636021" target="_blank">8636021</a>]<br></li>
<li>We conclude that the phosphate group between nt 12 and 13 of the consensus sequence inhibits the ability of MarA to bind .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21097628" target="_blank">21097628</a>]<br></li>
<li>Footprinting experiments identiﬁed a sequence at 269 to 254 ( nt 1349 to 1364 ) which contained sites that MarA either protected from or rendered hypersensitive to DNase I ( Fig. 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8636021" target="_blank">8636021</a>]<br></li>
<li>Interestingly , although the region from nt 1327 to 1344 ( 291 to 274 ) contains an imperfect direct repeat ( 12/18 identity ) of the marbox region at nt 1347 to 1364 , this sequence was not protected from DNase I attack by MarA ( Fig. 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8636021" target="_blank">8636021</a>]<br></li>
<li>Although marbox 1 and marbox 4 were closely related to the degenerate consensus sequence ( 3 nt differences out of 14 speciﬁed nt ) ( Fig. 3B ) , their location , downstream from the -10 hexamer and 83 nt upstream from the -35 hexamer , respectively , suggested that these marboxes were probably not functional in the MarA activation of nfnB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12100559" target="_blank">12100559</a>]<br></li>
<li>The only oligomer pair ( RobO3 ) to which MarA specifically bound corresponded to the DNA sequence between the 35 hexamer with a partial overlap of the 10 hexamer ( Fig. 3B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16478729" target="_blank">16478729</a>]<br></li>
<li>The crystal structure of MarA with its cognate sequence ( 6 ) shows direct hydrogen bond contacts to two groups of bases 9 bp apart , the equivalent of the A and B boxes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12060687" target="_blank">12060687</a>]<br></li>
				<label class="SIT_collapse_similarity" for="sit_similarity_16">
Thus , a single MarA subunit recognizes the specific 20 bp ` Mar-box ' sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10760178" target="_blank">10760178</a>]<br></label>
<input id="sit_similarity_16", type="checkbox">
<div class="SIT_similar">
<li>Thus , a single MarA subunit recognizes the speciﬁc 20 bp ` Mar box ' sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12675795" target="_blank">12675795</a>]<br></li></div>
<li>Although a series of 20 bp fragments starting at position ÿ53 showed no binding to MarA by gel retardation ( data not shown ) , we could readily align the inaA sequence from ÿ51 to ÿ32 in the F-orienta-tion with the other optimal 20-mer sequences ( Table 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10564485" target="_blank">10564485</a>]<br></li>
<li>Fragments as short as 20 bp bound MarA with a KD of 130 nM , but this increased sharply for 19 bp or smaller fragments ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10564485" target="_blank">10564485</a>]<br></li>
<li>DNA fragments comprising the individual marboxes showed that MarA did not bind to a `` control '' poor consensus backward `` marbox 3 '' ( bp 42 to 61 ; only 12/20 consensus bp ) but did bind to marboxes 1 and 2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14701822" target="_blank">14701822</a>]<br></li>
<li>Binding of MarA and the indicated 20 bp fragments covering the mar and micF marbox regions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10564485" target="_blank">10564485</a>]<br></li>
<li>Binding of MarA to permuted series of 20 bp fragments from several marboxes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10564485" target="_blank">10564485</a>]<br></li>
<li>None-theless , by aligning the footprint sequences of oriC ( Skar-stad et al. , 1993 ) , sodA , nfo ( Jair et al. , 1996b ; Li and Demple , 1996 ) and pqi ( Koh and Roe , 1996 ) in the F-orientation and fpr in the B-orientation so as to obtain the maximum sequence identity with the optimal 20-mer sequences for mar , micF , fumC and zwf , we tentatively identi ® ed the MarA binding sites in these sequences ( Table 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10564485" target="_blank">10564485</a>]<br></li>
<li>Determination of the orientations of the MarA binding sites in the promoters of the mar regulon As we have discerned binding sites with two possible orientations , we tested whether the orientation affected function by inverting the binding sites within the natural zwf and micF promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10564485" target="_blank">10564485</a>]<br></li>
<li>We therefore prepared a permuted set of 20-mers , corresponding to regions footprinted by MarA ( Martin et al. , 1996 ) , beginning at positions ÿ76 to ÿ70 [ relative to the transcription start site ( TSS ) of the mar promoter ] and analysed their af ® nities for MarA by gel mobility .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10564485" target="_blank">10564485</a>]<br></li>
<li>Second , hydroxyl radical footprinting shows that MelR bound to Site 2 protects three sets of bases , each separated by 10 bp , consistent with the binding of a single MelR subunit penetrating two adjacent major grooves at its DNA target ( as for MarA ; see Rhee et al. , 1998 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10760178" target="_blank">10760178</a>]<br></li>
<li>Recently , Rhee et al. reported the crystal structure of MarA bound to the marO sequence and proposed a consensus sequence for MarA binding based on 10 mar regulon promoters ( 30 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10438753" target="_blank">10438753</a>]<br></li>
<li>Despite the overall degeneracy of the marbox sequence , two areas stand out as being highly conserved : recognition element ( RE ) 1 ( consensus GCAC ) and RE 2 ( consensus YAAA ) ( Martin et al. , 1999 ; Grifﬁth and Wolf , 2001 ) , which establish essential contacts with the helix -- turn -- helix ( HTH ) binding motifs of the MarA , Rob and SoxS proteins ( Li and Demple , 1996 ; Rhee et al. , 1998 ; Gillette et al. , 2000 ; Kwon et al. , 2000 ; Dangi et al. , 2001 ; Grifﬁth and Wolf , 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12100559" target="_blank">12100559</a>]<br></li>
<li>MarA is an AraC family global regulator ( Rhee et al. , 1998 ; Martin and Rosner , 2001 ) , which transcriptionally activates at least 21 promoters , the marA regulon ( Barbosa and Levy , 2000 ; Pomposiello et al. , 2001 ; Martin and Rosner , 2002 ; 2003 ) , by binding to a site ( marbox ) overlapping or upstream of the -35 signal ( Martin et al. , 1996 ; 1999 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15225313" target="_blank">15225313</a>]<br></li>
<li>MarA and Fis bound with normal afﬁnity to the insertions containing 5 or 10 bp between positions 270 and 271 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9393706" target="_blank">9393706</a>]<br></li>
<li>Puriﬁed MarA bound as a monomer to a 16-bp `` marbox '' located 69 to 54 nucleotides upstream of a putative RNA initiation site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8636021" target="_blank">8636021</a>]<br></li>
<li>For the mar promoter , MarA is bound to a marbox in the B-orientation and displaced by 16 bp from the ÿ35 hexamer ( green arrowhead at the bottom of the ® gure ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10564485" target="_blank">10564485</a>]<br></li>
<li>For the zwf promoter , MarA is bound to a marbox in the F-orientation and displaced by 7 bp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10564485" target="_blank">10564485</a>]<br></li>
<li>Proximity of the bound MarA and Fis proteins is suggested by the disappearance of a MarA hypersensitive site 39 to bp 275 on the template strand when the promoter fragment was digested in the presence of both MarA and Fis ( Fig. 2B ) , consistent with some interaction of Fis with the sequences ﬂanking the binding site ( 34 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9393706" target="_blank">9393706</a>]<br></li>
<li>Puriﬁed MarA bound promoter 28 DNA as a monomer ( estimated KD , ; 2 3 10 M ; Fig. 2 ) at a site distinct from those bound by MarR ( Fig. 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8636021" target="_blank">8636021</a>]<br></li>
<li>How the same protein is able both to activate and to repress transcription probably relates to the identity and location of DNA sequence motifs that are recognized by MarA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14701822" target="_blank">14701822</a>]<br></li>
<li>11 and 13 ) , bile salts , and reactive oxygen species , MarA , in concert with global regulators Rob and SoxS , binds to the mar box sequence to activate transcription of the mar operon ( 14 -- 17 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25294881" target="_blank">25294881</a>]<br></li>
<li>The transcription of the acrB gene is regulated by a global transcriptional activator MarA and a local transcriptional repressor AcrR .21 The acrR gene is located 141 bp upstream of the acrAB operon and transcribed divergently .21 It encodes a 215-aminoacid protein , which shares N-terminal sequence and structural similarities to members of the TetR family of transcriptional repressors .22 The signatures of the TetR family of regulators include a homologous three helix DNA-binding domain , and a diverse C-terminal domain that possesses unique sequences , forming a specific ligand-binding site for inducing compounds .22 Many TetR family members bind as dimers to their ~ 15-bp palindrome ( IR ) operator sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17950313" target="_blank">17950313</a>]<br></li>
<li>Both genes were found to have potential , though imperfect , MarA binding sites about 60 nucleotides upstream of the coding sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10438753" target="_blank">10438753</a>]<br></li>
<li>Inspection of the DNA sequence upstream of iscR revealed the presence of DNA sequences similar to those that MarA binds 151 to 106 nucleotides upstream of the iscR initiation codon ( Fig. 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11742080" target="_blank">11742080</a>]<br></li>
<li>From our results we postulate that MarA represses by binding to a region of the promoter that overlaps the RNA polym-erase 35 binding motif .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14701822" target="_blank">14701822</a>]<br></li>
<li>With the knowledge that the binding sites exist in two orientations , we aligned 16 MarA / SoxS/Rob binding sites and sequences implicated in binding by either footprint or activation data to develop a new consensus binding sequence ( Table 1 , bottom row ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10564485" target="_blank">10564485</a>]<br></li>
<li>This , and the inclusion of three other footprinted marboxes ( nfo , pqi-5 and oriC ) , has enabled us to compile an improved consensus sequ-ence for the MarA , SoxS and Rob binding sites : AYnGCACnnWnnRYYAAAYn ( Table 1 , bottom row ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10564485" target="_blank">10564485</a>]<br></li>
<li>tolC is regulated in part by MarA , SoxS and Rob , three paralogous transcriptional activators which bind a sequence called the marbox and which activate multiple antibiotic and superoxide resistance functions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18673442" target="_blank">18673442</a>]<br></li>
<li>To understand why MarA does not activate certain promoters as strongly as SoxS , we compared MarA , MarA mutants , and SoxS for their abilities to activate 16 promoters and to bind their cognate marbox binding sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21097628" target="_blank">21097628</a>]<br></li>
<li>This is due to the close relatedness of the two regulators SoxS and MarA ( 21 ) , which bind to a common set of promoters with a regulatory sequence called either the `` soxbox , '' for SoxS binding , or the `` marbox , '' for MarA binding ( 3 , 21 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19376854" target="_blank">19376854</a>]<br></li>
<li>Rob and MarA bind to the uracil containing promoter sequences with apparent affinities that are indistinguishable from those obtained for the unmodified promoter site ( Fig. 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10802742" target="_blank">10802742</a>]<br></li>
<li>For example , the E. coli proteins MarA , Rob and SoxS all target genes involved in the oxidative stress response and bind to closely related DNA sequences .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9813115" target="_blank">9813115</a>]<br></li>
<li>Rob , SoxS , and MarA bind sequences adjacent to and partly overlapping the ÿ35 region of the micF promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11601842" target="_blank">11601842</a>]<br></li>
<li>DNA recognition by Rob Rob and the related transcription factors SoxS and MarA bind to asymmetric DNA sequences ∼ 20 base pairs in length within the promoter regions of regulated genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10802742" target="_blank">10802742</a>]<br></li>
<li>Within the soxbox consensus sequence , AYNGCACNNWNNRYYAAA YN , two functional domains ( nt 4 to 7 and nt 13 to 19 , underlined ) have been identiﬁed from an analysis of aligned SoxS / Mar/Rob binding sites ( 28 ) by molecular information theory ( 44 , 45 , 55 ) and from the protein-DNA interaction sites identiﬁed from the crystal structure of the MarA ( 41 ) and Rob ( 21 ) proteins complexed with the mar and micF promoter sequences , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11741843" target="_blank">11741843</a>]<br></li>
<li>The size of this binding site , which may actually be less than 21 bp ( 9 ) , is similar to that of the ; 17-bp sequence bound by a monomer of AraC protein ( 7 ) , a founder of the XylS/AraC subfamily of DNA binding proteins to which Rob , SoxS , and MarA belong ( 11 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626315" target="_blank">8626315</a>]<br></li>
<li>The binding of Rob with the micF promoter at the A box is similar to binding of MarA with its cognate promoter82 but differs from binding at the B motif where MarA binds speci ® cally and bends the DNA promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11601842" target="_blank">11601842</a>]<br></li>
<li>The interactions between Rob and the A-box sequence are very analogous to those described for the crystal structure of MarA in complex with the mar promoter site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10802742" target="_blank">10802742</a>]<br></li>
<li>Results Identi ® cation of minimal MarA binding sequences in promoters of the mar/sox/rob regulon and indication of inverted orientations Two highly degenerate consensus sequences have been proposed ( Table 1A ) for the binding site of MarA , SoxS and Rob in the promoters of the mar/sox/rob regulon ( Fawcett and Wolf , 1995 ; Li and Demple , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10564485" target="_blank">10564485</a>]<br></li>
<li>MarA , Rob , and SoxS are highly homologous transcription factors that recognize the same consensus binding site ( AYnGCACnnWnnRYYAAAY ) ( 21 ) and are induced by noxious compounds such as antibiotics , detergents , and oxidizing agents .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23010927" target="_blank">23010927</a>]<br></li>
<li>It is tempting to hypothesize that the sequence divergence from the consensus in the putative SoxS binding site at the yggX promoter and the lack of regulation by the SoxS homologs MarA and Rob are related phenomena .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14594836" target="_blank">14594836</a>]<br></li>
<li>In contrast , the repression of both soxS and rob by SoxS , Rob , or MarA occurs via a yet to be identified binding site ( s ) within the sox and rob promoter regions ( 15 , 16 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16478729" target="_blank">16478729</a>]<br></li>
<li>Binding of MarA , SoxS and Rob to the nfnB promoter In an attempt to deﬁne the MarA binding site within the nfnB responsive promoter region , we asked if puriﬁed His6-MarA protein was able to bind a digoxigenin ( DIG ) - labelled 110 bp DNA fragment comprising marbox 1 and 2 , called nfnB F2/R1 , and an 86 bp fragment that comprised marbox 3 and 4 , termed nfnB F1/R2 in electrophoretic mobility-shift assays ( EMSA ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12100559" target="_blank">12100559</a>]<br></li>
<li>MalE - SoxS and MarA have also been shown to activate transcription of mar/soxRS regulon genes and to recognize a speciﬁc 21-bp zwf promoter `` soxbox '' sequence in vitro ( 10 , 18 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8636021" target="_blank">8636021</a>]<br></li>
<li>Miller et al. ( 28 ) have found that SoxS activates mar transcription in vivo , and we have recently found a marbox sequence in the promoter region upstream of marRAB which binds MarA and enables either puriﬁed MalE-SoxS or MarA to activate transcription of marRAB in vitro ( 18 , 26 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8522515" target="_blank">8522515</a>]<br></li>
<li>MarA , SoxS and Rob bind as monomers to an asymmetrical and degenerate 19 bp DNA sequence known as the marbox ( Martin et al. , 1999 ; Wood et al. , 1999 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18673442" target="_blank">18673442</a>]<br></li>
<li>DNA Sequences Involved in Binding by MarA , EMSA -- Activation of transcription by MarA involves binding of this protein as a monomer to a degenerate 20-bp `` marbox '' located upstream from or overlapping the 35 motif within the promoter ( 6 , 10 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14701822" target="_blank">14701822</a>]<br></li>
<li>MarA activates transcription by binding as a monomer to an asymmetric degenerate 20 bp DNA sequence , the marbox ( with a AYnGCACnnWnnRYYAAAYn consensus where R = A or G ; Y = C or T ; W = A or T ; n = A , T , C or G ) , located in the promoter region of controlled genes ( Martin et al. , 1999 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12100559" target="_blank">12100559</a>]<br></li>
<li>Transcriptional activation of those mar regulon members that have been studied occurs when MarA binds as a monomer to the promoter region at a degenerate asymmetric 20-bp DNA sequence known as the `` marbox '' ( 6 , 7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14701822" target="_blank">14701822</a>]<br></li>
<li>For micF , the MarA binding sequence from ÿ51 to ÿ32 ( Table 1 ) , or its inverted sequence , was followed by the normal micF promoter sequences from ± 31 to 3 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10564485" target="_blank">10564485</a>]<br></li>
<li>Because the binding site for MarA overlaps the ÿ35 hexamer of micF , the inverted site alters the ÿ35 hexamer sequence ( AAAACA becomes ACTGTA ) and reduces the basal level of expression considerably .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10564485" target="_blank">10564485</a>]<br></li>
<li>To analyze the role in transcription of the MarA binding site and of these upstream sequences , a series of deletions in the region from nt 1312 to 1364 ( Fig. 4 ) were introduced into pRGM258 , a plasmid containing the wild-type mar region from nt 1312 to 2769 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8636021" target="_blank">8636021</a>]<br></li>
<li>These transcriptional fusions have lacZ fused downstream of bp -72 to +48 ( relative to the TSS of the marRAB promoter ) and thus extend from the MarA binding site ( Martin and Rosner , 1997 ) starting at bp -72 to the seventh codon of the marR gene ( Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15225313" target="_blank">15225313</a>]<br></li>
<li>Curiously , deletion of the 18 bp ( nt 1329 to 1346 ) lying just upstream of the MarA binding site in DmarO280 diminished the stimulation of transcription by MarA ( to 3.9-fold ) and MalE-SoxS ( to 2.3-fold ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8636021" target="_blank">8636021</a>]<br></li>
<li>MarA binding site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8636021" target="_blank">8636021</a>]<br></li>
<li>We recognized a near-perfect MarA binding site 27 nucleotides upstream of promoter 1 of the acrAB operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10438753" target="_blank">10438753</a>]<br></li>
<li>Remarkably , the mar promoter is itself transcriptionally activated by the binding of MarA or SoxS to a `` marbox '' sequence at bp 269 to 254 ( with respect to the RNA start site [ 44 ] ) ( 27 , 31 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9393706" target="_blank">9393706</a>]<br></li>
<li>MarA also autoactivates marRAB operon transcription by binding to a 20 bp marbox sequence 72 -- 53 bp upstream of the transcription start site ( TSS ) ( Martin et al. , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15225313" target="_blank">15225313</a>]<br></li>
				</ul></div><br>
			<label class="collapse_TU" for="_6"><font size="5px">Transcription units regulated by MarA</font></label>
				<input id="_6" type="checkbox">
				<div class="TU"><br><ul><li>Strains M3710 ( pMarA ) and M3944 ( pSoxS ) were then diluted twofold into LB without ( control ) or with IPTG ( 0.5 mM ﬁnal concentration ) for 1 h to induce plasmid-borne marA or soxS expression respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18673442" target="_blank">18673442</a>]<br></li>
<li>Although this has helped us to understand the basic design of MarA and its homologues , it has not furthered the de ® nition of the marbox appreciably because the majority of the MarA ± marbox DNA contacts are due to van der Waals interactions ( see Discussion ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10564485" target="_blank">10564485</a>]<br></li>
<li>JOURNAL OF BACTERIOLOGY , Dec. 1995 , p. 7100 -- / 95 / $ 04.0010 Copyright q 1995 , American Society for Microbiology Puriﬁcation and Regulatory Properties of MarA Protein , a Transcriptional Activator of Escherichia coli Multiple Antibiotic and Superoxide Resistance Promoters KAM-WING JAIR ,1 ROBERT G. MARTIN ,2 JUDAH L. ROSNER ,2 NOBUYUKI FUJITA ,3 AKIRA ISHIHAMA ,3 AND RICHARD E. WOLF , JR. 1 * Department of Biological Sciences , University of Maryland Baltimore County , Baltimore , Maryland 212281 ; Laboratory of Molecular Biology , National Institute of Diabetes and Digestive and Kidney Diseases , Bethesda , Maryland 208922 ; and Department of Molecular Genetics , National Institute of Genetics , Mishima , Shizuoka-ken 411 , Japan3 Received 8 September 1995/Accepted 17 October 1995 Expression of the marA or soxS genes is induced by exposure of Escherichia coli to salicylate or superoxides , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8522515" target="_blank">8522515</a>]<br></li>
<li>Results MarA upregulates expression of nfnB Both oxygen-insensitive nitroreductases nfsA and nfnB were shown to be upregulated by constitutive expression of MarA from plasmid pAS10 ( Barbosa and Levy , 2000 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12100559" target="_blank">12100559</a>]<br></li>
<li>Recently , Barbosa and Levy [ 10 ] used DNA micro-arrays to determine that over 60 genes were induced or repressed by constitutive expression of MarA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11094278" target="_blank">11094278</a>]<br></li>
				<label class="TU_collapse_similarity" for="tu_similarity_6">
Differential expression of over 60 chromosomal genes in Escherichia coli by constitutive expression of MarA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11741843" target="_blank">11741843</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18083817" target="_blank">18083817</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12670992" target="_blank">12670992</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22210763" target="_blank">22210763</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11321589" target="_blank">11321589</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11395452" target="_blank">11395452</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19376854" target="_blank">19376854</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14594836" target="_blank">14594836</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12107134" target="_blank">12107134</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20211899" target="_blank">20211899</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21097628" target="_blank">21097628</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15691972" target="_blank">15691972</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12218039" target="_blank">12218039</a>]<br></label>
<input id="tu_similarity_6", type="checkbox">
<div class="TU_similar">
<li>Barbosa , T.M. , and Levy , S.B. ( 2000 ) Differential expression of over 60 chromosomal genes in Escherichia coli by constitutive expression of MarA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12791142" target="_blank">12791142</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12100559" target="_blank">12100559</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15225313" target="_blank">15225313</a>]<br></li>
<li>[ 19 ] Barbosa , T.M. and Levy , S.B. ( 2000 ) Di ¡ erential expression of over 60 chromosomal genes in Escherichia coli by constitutive expression of MarA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11287152" target="_blank">11287152</a>]<br></li>
<li>[ 10 ] Barbosa , T.M. and Levy , S.B. ( 2000 ) Di ¡ erential expression of over 60 chromosomal genes in Escherichia coli by constitutive expression of MarA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11094278" target="_blank">11094278</a>]<br></li></div>
<li>The MarA global regulatory protein in E. coli is known to control expression of multiple chromosomal genes affecting resistance to antibiotics and other environmental hazards , such as the multidrug efflux genes acrAB and tolC ( Aono , 1998 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11321589" target="_blank">11321589</a>]<br></li>
<li>Evidence that regulatory protein MarA of Escherichia coli represses rob by steric hindrance .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22210763" target="_blank">22210763</a>]<br></li>
<li>These values were compared with the predicted masses of the proteins encoded by the Rob-regulated genes and operons identiﬁed in this study and with those of proteins encoded by genes known to be induced by Rob in vivo ( 5 ) or in vitro ( 15 , 32 ) , namely , InaA ( 25.3 kDa ) , the mar operon products ( 13.9 kDa [ MarR ] , 15.4 kDa [ MarA ] , and 7.5 kDa [ MarB ] ) , the gal operon products ( 37.3 kDa [ GalE ] , 39.6 kDa [ GalT ] , 41.4 kDa [ GalK ] , and 38.2 kDa [ GalM ] ) , AslB ( 46.6 kDa ) , YbaO ( 20.9 kDa ) , the mdl operon products ( 66.0 kDa [ MdlA ] and 65.2 kDa [ MdlB ] ) , YfhD ( 53.2 kDa ) , YbiS ( 33.3 kDa ) , AcrA ( 42.2 kDa ) , AcrB ( 114 kDa ) , Fpr ( 27.8 kDa ) , FumC ( 50.5 kDa ) , Nfo ( 31.5 kDa ) , SodA ( 23.1 kDa ) , and Zwf ( 55.7 kDa ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10850996" target="_blank">10850996</a>]<br></li>
<li>Mixtures of the test and gnd reference promoters were transcribed under standard conditions in the absence and the presence of native MarA protein ( 2 pmol ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8522515" target="_blank">8522515</a>]<br></li>
<li>Test templates were transcribed in mixtures containing the gnd reference promoter in the presence and absence of puriﬁed MarA protein ( 2 pmol ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8522515" target="_blank">8522515</a>]<br></li>
<li>If MppA is involved in the negative regulation of marA , the MarA and AcrAB contents of the mppA mutant should increase relative to its parent .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10438753" target="_blank">10438753</a>]<br></li>
<li>It is known that the the DegP protein , whose nascent form is unstable and thus marA gene encodes MarA , a regulator that activates transcripbecomes absent in a dsbA null mutant ( 38 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25294881" target="_blank">25294881</a>]<br></li>
<li>Using gene arrays , Pomposiello et al. ( 33 ) found that overexpression of MarA from an isopropyl - - D-thiogalactopyrano-side ( IPTG ) - inducible plasmid repressed hdeAB and other genes involved in acid resistance ( gadA , gadB , gadC , hdeD , and slp ) , whereas sodium salicylate activated these same genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18083817" target="_blank">18083817</a>]<br></li>
<li>Thus , the data in Fig. 3b and Fig. 3c not only show that in vivo Rob binds to robbox DNA in the same orientation as in the co-crystal structure of MarA bound to the marbox of the mar promoter but also , more importantly , show that amino acids of helix-3 and helix-6 of Rob are bound in vivo to RE1 and RE2 , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22465792" target="_blank">22465792</a>]<br></li>
<li>Transcription from all the 5 - deleted promoter fragments tested was repressed by MarA ( data not shown for all fragments except for representative truncated fragments , pRobF5 and pRobF4 ; see Fig. 4 and Fig. 6A , respectively ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16478729" target="_blank">16478729</a>]<br></li>
<li>The product of marA , the transcription factor MarA , directly activates or represses many chromosomal genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20211899" target="_blank">20211899</a>]<br></li>
<li>Although the transcription factor MarA is not directly regulated by FNR or ArcA , four genes that are activated by MarA ( acnA , fumC , sodA , zwf ) are repressed by ArcA and/or FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23980183" target="_blank">23980183</a>]<br></li>
<li>The effect of MarA is expressed as the ratio of purA transcription in the presence of MarA divided by purA transcription in its absence , normalized for any differences in the transcription of gnd .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14701822" target="_blank">14701822</a>]<br></li>
<li>The multiple-round transcription experiments demonstrated that in the absence of MarA , the mRNA transcripts for both the rob and gnd promoters accumulated during the course of the assay ( Fig. 6C ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16478729" target="_blank">16478729</a>]<br></li>
<li>Two relatives of MarA were reported to directly repress transcription of other promoters , but the marbox involved was not determined .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14701822" target="_blank">14701822</a>]<br></li>
<li>Therefore , both MarA and Rob are able to repress the rob promoter directly in vitro , and this repression occurs at the level of transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16478729" target="_blank">16478729</a>]<br></li>
<li>As for the hdeA and purA promoters , which are repressed by MarA , the rob marbox is also in the `` backward '' orientation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16478729" target="_blank">16478729</a>]<br></li>
<li>The same concentration and batch of MarA protein , on the other hand , repressed both purA and hdeA ( Fig. 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14701822" target="_blank">14701822</a>]<br></li>
<li>The physiological significance of the repression of purA and hdeA by MarA is not known .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14701822" target="_blank">14701822</a>]<br></li>
				<label class="TU_collapse_similarity" for="tu_similarity_29">
The Esche-richia coli transcriptional regulator MarA directly represses transcription of purA and hdeA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22210763" target="_blank">22210763</a>]<br></label>
<input id="tu_similarity_29", type="checkbox">
<div class="TU_similar">
<li>The Escherichia coli transcriptional regulator MarA directly represses transcription of purA and hdeA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18083817" target="_blank">18083817</a>]<br></li></div>
<li>We show here that MarA directly represses both purA and hdeA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14701822" target="_blank">14701822</a>]<br></li>
<li>We found that purified MarA was sufficient in vitro to repress transcription of both purA and hdeA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14701822" target="_blank">14701822</a>]<br></li>
<li>These results showed that MarA by itself could directly repress purA and hdeA transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14701822" target="_blank">14701822</a>]<br></li>
<li>These amino acids are located opposite one another on the MarA surface .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10873449" target="_blank">10873449</a>]<br></li>
<li>While it was encouraging to observe similarities across studies , we also noted several genes or operons expressed oppositely in mice than in previous in vitro ﬁndings for soxS , namely , cysB , cysM , pdhR , and nikB for SoxS ( 12 , 41 ) and yadH for MarA ( 11 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22210763" target="_blank">22210763</a>]<br></li>
<li>MarA is a global regulator of genes induced by antibiotics and is a member of a subfamily of AraC-like transcriptional regulators called the Mar Sox Rob family ( 14 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11742080" target="_blank">11742080</a>]<br></li>
<li>The three paralogous Escherichia coli AraC/XylS family activators MarA , SoxS , and Rob are regulated by three different systems ( marRAB , soxRS , and rob , respectively ) in response to different stresses ( 29 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21097628" target="_blank">21097628</a>]<br></li>
<li>In JA300 , AcrA production was regulated positively by each of the transcriptional activator proteins , MarA , Rob , and SoxS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9473050" target="_blank">9473050</a>]<br></li>
<li>First , three independently regulated but homologous proteins , MarA , SoxS and Rob , activate a common set of promoters which results in multiple antibiotic resistance , superoxide resistance and organic solvent tolerance ( for review , see Alekshun and Levy , 1997 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10564485" target="_blank">10564485</a>]<br></li>
<li>tolC is regulated in part by MarA , SoxS and Rob , three paralogous transcriptional activators which bind a sequence called the marbox and which activate multiple antibiotic and superoxide resistance functions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18673442" target="_blank">18673442</a>]<br></li>
<li>in d u locus The soxRS The acrAB locus The acrAB operon , which encodes two subunits of the AcrAB multidrug-efflux pump , is a critical target for both MarA and SoxS in the multiple-antibiotic-resistance response ( Ma et a / .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8866468" target="_blank">8866468</a>]<br></li>
<li>As the YdbK function has been related to superoxide resistance and this two-gene operon is activated by SoxS and not by MarA , the mutants were tested for resistance to oxidative stress .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22515487" target="_blank">22515487</a>]<br></li>
<li>Gel B ( showing only the bound material ) employed a different preparation of MarA from which the His6 tag had not been removed ( the same as in Fig. 2 ) and the comparable preparation of SoxS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21097628" target="_blank">21097628</a>]<br></li>
<li>The acrAB gene is induced by various environmental stresses , such as eth-anol , osmotic shock and oxidizing agents via MarA and SoxS ( Ma et al. , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15686558" target="_blank">15686558</a>]<br></li>
<li>Although MarA homologue SoxS was also shown to regulate the transcription of acrAB strongly ( Ma et al. , 1996 ; White et al. , 1997 ) , SoxS , also a small protein of 107 residues , exerts its action by altering the level of its own expression , which is regulated via SoxR by oxidizing agents that generate the superoxide radical ( Greenberg et al. , 1990 ; Ding et al. , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12791142" target="_blank">12791142</a>]<br></li>
<li>Although MarA , SoxS and Rob could increase acrAB transcription when they were overexpressed , these gene products did not appear to be responsible for the acrAB induction by the general stress conditions such as 4 % ethanol , 0.5 M NaCl or the onset of stationary-phase growth in LB medium ( Table 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8821940" target="_blank">8821940</a>]<br></li>
<li>It would be inte-resting to determine under what conditions these acrAB homologues are induced and how they respond to the global stress-induced regulators , including MarA , SoxS and ( possibly ) Rob .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8821940" target="_blank">8821940</a>]<br></li>
<li>Third , unlike the general stress conditions , the constitutive overexpression of MarA or SoxS did not increase the transcription of acrR at all ( D. Ma , unpublished results ) , although it increased the transcription of acrAB , as shown in Table 2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8821940" target="_blank">8821940</a>]<br></li>
<li>Evidence also suggests that the up-regulationof acrAB expression under general stress conditions is not likely to be mediated by the known global regulators , such as MarA or SoxS , although elevated levels of these proteins were shown to increase the transcription of acrAB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8821940" target="_blank">8821940</a>]<br></li>
<li>Since it is known that acrAB expression is also induced by the ` global ' regulators MarA ( Ma et al. , 1995 ) and SoxS ( see below ) , we examined the possibility that MarA or SoxS might be mediatingthe responseto general stress conditions , by usingstrainsthat are totally lackingor constitutively producing these regulatoryproteins ( Table 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8821940" target="_blank">8821940</a>]<br></li>
<li>, 1994 ) , and inaA , encoding a weak acid-inducible protein ( Rosner and Slonczewski , 1994 ) have been shown to be activated by both MarA and SoxS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8821940" target="_blank">8821940</a>]<br></li>
<li>Presumably , uninduced levels of MarA and/or SoxS mediate a low-level induction of the inaA gene ; the lower basal level of inaA expression in Dmar strains was noted earlier ( Rosner and Slonczewski , 1994 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12791142" target="_blank">12791142</a>]<br></li>
<li>Similar results were obtained when SoxS was induced by para-quat or MarA was provided from a plasmid ( data not presented ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9393706" target="_blank">9393706</a>]<br></li>
<li>Moreover , both MarA and MalE-SoxS , respectively , have been shown recently to bend zwf DNA by 508 and 458 and fumC DNA by 348 and 488 ( 38 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8522515" target="_blank">8522515</a>]<br></li>
<li>The report ( 32 ) that overexpression of SoxS induced mar transcription in vivo led us to test for transcriptional activation of mar by puriﬁed MarA and MalE-SoxS proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8636021" target="_blank">8636021</a>]<br></li>
<li>MalE - SoxS and MarA have also been shown to activate transcription of mar/soxRS regulon genes and to recognize a speciﬁc 21-bp zwf promoter `` soxbox '' sequence in vitro ( 10 , 18 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8636021" target="_blank">8636021</a>]<br></li>
<li>Thus , SoxS and MarA can utilize the same 21-bp sequence to activate zwf transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8522515" target="_blank">8522515</a>]<br></li>
<li>Previously , overexpression of Rob was shown to induce multiple antibiotic and superoxide resistance phenotypes ( 3 ) , to activate expression of fumC , inaA , and sodA in vivo ( 3 ) , and to activate the transcription of six mar/soxRS regulon promoters in vitro by mechanisms that are strikingly similar to that of MarA and SoxS ( 22 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9393706" target="_blank">9393706</a>]<br></li>
<li>Sufﬁciency of Rob for in vitro transcriptional activation of zwf , fpr , fumC , micF , nfo , and sodA , promoters known to be activated by MalE-SoxS and MarA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626315" target="_blank">8626315</a>]<br></li>
<li>To determine whether the puriﬁed MarA protein affected gene expression , we tested its ability to activate the in vitro transcription of zwf , sodA , micF , and fumC , genes known to be under mar control in vivo , and of fpr and nfo , genes not known to be regulated by mar in vivo but activated by SoxS in vivo and by MalE-SoxS in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8522515" target="_blank">8522515</a>]<br></li>
<li>However , in Rob the 100-residue domain homologous to SoxS and MarA is located at the N terminus of the protein ; the C-terminal 175-residue segment of Rob is not related to XylS-AraC family members ( 40 ) ( Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7896685" target="_blank">7896685</a>]<br></li>
<li>To test whether either MarA or Rob could activate the transcription of yggX , we measured the relative level of yggX mRNA in cells in which the expression of MarA or Rob could be selectively activated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14594836" target="_blank">14594836</a>]<br></li>
<li>The signaling mechanisms that prevail in OmpR , MarA , and SoxS action are known .9,10,14 Activation of micF by Rob in response to cationic peptide antibiotics is not as well formulated .32 Transcriptional activators of micF Transcriptional regulators are known to activate micF expression in response to environmental factors ( Figure 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11601842" target="_blank">11601842</a>]<br></li>
<li>The slight increase of MarB observed in the acrR + strain ( Fig. 6b ) might suppress marA transcription , resulting in increased expression of soxS only ( between MarA and SoxS ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25176444" target="_blank">25176444</a>]<br></li>
<li>The mechanism by which such small ( 13-kDa ) proteins as SoxS and MarA activate gene expression is unknown .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8282690" target="_blank">8282690</a>]<br></li>
<li>For instance , the expression of acrAB in E. coli is controlled by repressor protein AcrR , global activator protein MarA , SoxS , Rob and quorum sensing regulator LuxR homologue protein SdiA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15686558" target="_blank">15686558</a>]<br></li>
<li>Both MarA and SoxS belong to the XylS/AraC family of transcription regulators ( 6 ) , and yet another member of this family , RobA , also increases acrAB expression ( 37 , 41 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12107134" target="_blank">12107134</a>]<br></li>
<li>However , it does not appear that modulation of acrAB expression in response to these environmental stresses operates through AcrR , MarA , or SoxS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8866468" target="_blank">8866468</a>]<br></li>
<li>Its expression is regulated by three XylS/AraC family regulators , MarA , SoxS and Rob .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12791142" target="_blank">12791142</a>]<br></li>
<li>It is known that acrAB expression is also controlled by the global regulators MarA , SoxS , and Rob ( 4 , 5 , 10 , 21 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18567659" target="_blank">18567659</a>]<br></li>
<li>However , the expression of these regulators was not affected by AcrS and AcrR ( data not shown ) , indicating that the acrAB repression by AcrR and AcrS is unlikely to be mediated by MarA , SoxS , or Rob .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18567659" target="_blank">18567659</a>]<br></li>
<li>This selective expression of MarA and Rob was achieved by using strains MB102 ( marA ) ( 39 ) and MB101 ( robA ) ( 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14594836" target="_blank">14594836</a>]<br></li>
<li>This experiment con-ﬁrmed the transcriptional activity of artiﬁcially expressed MarA and Rob proteins ( Fig. 4B ) and veriﬁed that they do not activate yggX expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14594836" target="_blank">14594836</a>]<br></li>
<li>Northern analysis conﬁrmed the transcriptional activation of yggX by oxidative stress or SoxS expression but not in response to the related MarA or Rob proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14594836" target="_blank">14594836</a>]<br></li>
<li>In contrast to MarA and Rob , basal expression of SoxS does not contribute signiﬁcantly to mar activation ( compare Fig. 5C and D ) ( 27 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9393706" target="_blank">9393706</a>]<br></li>
<li>In contrast , the level of yggX transcript was unaffected by the expression of MarA or Rob ( Fig. 4A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14594836" target="_blank">14594836</a>]<br></li>
<li>Effect of MarA and Rob on the expression of yggX .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14594836" target="_blank">14594836</a>]<br></li>
<li>The p1 promoter is not responsive to EvgAS , PhoPQ , MarA , SoxS or Rob but seems to have signiﬁcant constitutive expression under laboratory conditions ( Eguchi et al. , 2003 ; Figs 1 and 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18673442" target="_blank">18673442</a>]<br></li>
<li>Although puriﬁed SoxS and Rob proteins , like MarA , activated nfnB transcription in vitro , only constitutive expression of chromosomal marA , but not of soxS and rob genes , affected nfnB expression in whole cells .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12100559" target="_blank">12100559</a>]<br></li>
<li>Transcription of nfnB is greatly increased by induced or constitutive chromosomal expression of MarA but is unaltered by constitutive expression of chromosomal SoxS and Rob .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12100559" target="_blank">12100559</a>]<br></li>
<li>b Rob presence was due to chromosomal wild-type rob expression , SoxS presence was due to paraquat induction of the otherwise quiescent soxRS , and MarA presence was due to plasmid p37 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9393706" target="_blank">9393706</a>]<br></li>
<li>Although MarA and SoxS regulation works by the alteration of their own expression levels , it was not known how Rob , which is constitutively expressed , exerts its regulatory action .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12791142" target="_blank">12791142</a>]<br></li>
<li>When expressed from identical plasmid vectors , Rob and SoxS caused increased nfnB expression but it was considerably less than that produced by MarA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12100559" target="_blank">12100559</a>]<br></li>
<li>Increased expression , but limited as compared with MarA , was only achieved by plasmid-mediated overexpression of SoxS and Rob .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12100559" target="_blank">12100559</a>]<br></li>
<li>Comparison of the regulation of nfnB expression by the plasmid-speciﬁed IPTG-inducible MarA , SoxS and Rob proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12100559" target="_blank">12100559</a>]<br></li>
<li>Effects of Rob on gene expression were also explored with lac reporter genes linked to the promoter of sodA , the gene encoding managanese-containing superoxide dismutase ( 4 , 18 , 42 ) , and to the coding region of inaA , which encodes a protein of unknown function ( 43 ) controlled by MarA and SoxS ( 36 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7896685" target="_blank">7896685</a>]<br></li>
<li>The genes encoding these proteins are activated by the homologous proteins MarA and SoxS , although the level of activation can vary for the different transcription factors .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10802742" target="_blank">10802742</a>]<br></li>
<li>Many bacteria also possess transcription factors , such as the homologous E. coli regulators MarA , Rob , and SoxS ( 3 , 4 ) , that respond to drugs by activating the expression of genes encoding proteins involved in antibiotic efﬂux and detoxiﬁcation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23010927" target="_blank">23010927</a>]<br></li>
<li>Since these transcription activation properties of Rob are virtually identical to those of MalE-SoxS and MarA , it appears as if the E. coli genome encodes three genes with the same functional capacity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626315" target="_blank">8626315</a>]<br></li>
<li>These genes are transcriptionally activated by three paralogous proteins , MarA , SoxS and Rob .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18673442" target="_blank">18673442</a>]<br></li>
<li>Down-regulation of rob mediated by MarA ( and also SoxS and Rob ) adds to the regulatory cross-talk already reported ( 15 -- 17 ) and shows that the rob promoter and protein are controlled at both pre - and posttranscriptional levels in vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16478729" target="_blank">16478729</a>]<br></li>
<li>Surprisingly , salicylate induced a twofold increase in promoter activity from the lacZ fusions that was independent of sequences from 291 to 232 , of MarA , Rob , and SoxS activators and of MarR and EmrR repressors ( Fig. 5D ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9393706" target="_blank">9393706</a>]<br></li>
<li>However , in contrast to SoxS and MarA , whose syntheses are induced by speciﬁc environmental stimuli and elicit a clear defense response , Rob is expressed constitutively and its normal function is unknown .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626315" target="_blank">8626315</a>]<br></li>
<li>Although Rob , SoxS and MarA activate micF transcription in response to different environmental stimuli , there is some overlap in response .14 soxS is part of the soxRS regulon .66 SoxR is a sensor protein activated by oxidative stress and in turn it transcriptionally activates soxS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11601842" target="_blank">11601842</a>]<br></li>
<li>For instance , Rob has been shown to bind and activate the zwf promoter in vitro but whole cell zwf regulation can not be activated by Rob , although the gene responds to SoxS and MarA ( Ariza et al. , 1995 ; Jair et al. , 1995 1996a ; b ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12100559" target="_blank">12100559</a>]<br></li>
<li>The b-galactosidase activities show that only when its marbox is in the F-orientation is the zwf promoter activated by MarA or SoxS ( Table 3 , top rows ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10564485" target="_blank">10564485</a>]<br></li>
<li>Similarly , the micF promoter is activated by MarA or SoxS only when the micF marbox is in the F-orientation ( Table 3 , rows 10 and 11 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10564485" target="_blank">10564485</a>]<br></li>
<li>From an analysis of the promoters that are activated by these proteins , it is clear that the orientation of the marbox and its distance from the -10 hexamer of the promoter are critical in determining whether MarA , SoxS and Rob can activate the promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18673442" target="_blank">18673442</a>]<br></li>
<li>Activation by the monomeric proteins MarA , SoxS and Rob requires that they bind to a marbox located in a particular orientation upstream of the TSS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18673442" target="_blank">18673442</a>]<br></li>
<li>Protein-DNA interactions show that SoxS and Rob , like MarA , bind the same marbox in the rob promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16478729" target="_blank">16478729</a>]<br></li>
<li>( 1991 ) Same as GC4468 ; marR ( cfxB1 ) JHC1069 Discussion Transcription of acrAB is up-regulated in mutants constitutively producing MarA or SoxS and in a strain that overexpresses Rob .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8821940" target="_blank">8821940</a>]<br></li>
<li>Additionally , and despite similarities in the structures of their DNA binding domains and cognate DNA binding sites , the SoxS homologs MarA and Rob did not activate in vivo transcription of yggX under the conditions tested ( Fig. 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14594836" target="_blank">14594836</a>]<br></li>
<li>It is likely that the three gene products ( MarA , Rob , and SoxS ) improve the level of organic solvent tolerance via a TolC function .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9473050" target="_blank">9473050</a>]<br></li>
<li>MarA , Rob , and SoxS are highly homologous transcription factors that recognize the same consensus binding site ( AYnGCACnnWnnRYYAAAY ) ( 21 ) and are induced by noxious compounds such as antibiotics , detergents , and oxidizing agents .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23010927" target="_blank">23010927</a>]<br></li>
<li>Expression of MicF is also controlled by the transcription regulators MarA , SoxS and Rob and the RNA mediates OmpF variation in response to various toxic factors ( Delihas and Forst , 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19400782" target="_blank">19400782</a>]<br></li>
<li>The expression of micF is also induced by toxic agents , such as paraquat and weak acids , by the SoxS , MarA , and Rob transcription factors , which bind to the same sequence element in the micF promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15466019" target="_blank">15466019</a>]<br></li>
<li>The highly conserved acrZ gene is coregulated with acrAB through induction by the MarA , Rob , and SoxS transcription regulators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23010927" target="_blank">23010927</a>]<br></li>
<li>Expression of acrZ is coregulated with acrAB and tolC by the MarA , Rob , and SoxS transcription factors .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23010927" target="_blank">23010927</a>]<br></li>
<li>Moreover , expression of acrZ is coregulated with acrAB and tolC by MarA , Rob , and SoxS , a set of transcription factors that regulate cellular responses to antibiotics , detergents , and redox-cycling drugs ( 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23010927" target="_blank">23010927</a>]<br></li>
<li>However , the observation that SoxS binds to , and seems to negatively regulate , its own promoter ( 33 ) raises the possibility that MarA and Rob could also negatively regulate soxS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8636021" target="_blank">8636021</a>]<br></li>
<li>Results and discussion tolC TSSs Transcription of tolC is activated by MarA , SoxS and Rob ( Aono et al. , 1998 ) but the relevant tolC promoter has not yet been deﬁned .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18673442" target="_blank">18673442</a>]<br></li>
<li>Two previously identiﬁed tolC promoters , p1 and p2 , are not regulated by MarA , SoxS or Rob but p2 is activated by EvgAS and PhoPQ which also regulate other functions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18673442" target="_blank">18673442</a>]<br></li>
<li>On re-investigation , we found two new TSSs for tolC ( corresponding to promoters p3 and p4 ) that lie downstream of the marbox and are activated by MarA , SoxS and Rob .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18673442" target="_blank">18673442</a>]<br></li>
<li>We conclude that the tolC promoter ( s ) activated by MarA , SoxS and Rob must lie downstream of p1 and p2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18673442" target="_blank">18673442</a>]<br></li>
<li>However , the observation that in vivo zwf transcription can be activated by MarA and SoxS , but not by Rob ( 5 ) , shows that control by these regulators may not overlap completely .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10850996" target="_blank">10850996</a>]<br></li>
<li>Moreover , transcription of marA is activated by MarA and by the MarA homologs SoxS and Rob , mostly in the presence of the helper protein Fis ( 32 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20211899" target="_blank">20211899</a>]<br></li>
<li>The inaA gene is activated by SoxS , MarA , and Rob , with MarA being the strongest activator ( 34 , 43 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14594836" target="_blank">14594836</a>]<br></li>
<li>For example , the nfo gene , coding for endonuclease IV , is activated by SoxS but not by MarA or Rob ( 17 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14594836" target="_blank">14594836</a>]<br></li>
<li>Similarly , the gshB gene , coding for a glutathione synthase , is activated by MarA but not by SoxS or Rob ( 5 , 39 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14594836" target="_blank">14594836</a>]<br></li>
<li>Nevertheless , when induced by paraquat , the activity of SoxS was not dependent on either MarA or Rob ( Table 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9393706" target="_blank">9393706</a>]<br></li>
<li>The SoxS paralogs MarA and Rob are not sufﬁcient for activation of yggX .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14594836" target="_blank">14594836</a>]<br></li>
<li>acrZ Is Regulated by MarA , Rob , and SoxS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23010927" target="_blank">23010927</a>]<br></li>
<li>By the criteria examined , Rob is the same as SoxS and MarA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626315" target="_blank">8626315</a>]<br></li>
<li>The results in Fig. 7 demonstrate that sodium salicylate also induces the waaY gene as efﬁciently as paraquat does , and the induction is dependent on both the SoxRS and MarA systems .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19376854" target="_blank">19376854</a>]<br></li>
<li>waaY expression was also regulated by MarA ( multiple-antibiotic resistance regulator ) , which shares a binding site ( soxbox ) with SoxS , and was induced by salicylate , a nonoxidative compound .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19376854" target="_blank">19376854</a>]<br></li>
<li>The waaY promoter is also induced by antibiotics through the MarA system .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19376854" target="_blank">19376854</a>]<br></li>
<li>Therefore , the waaY gene that is induced by superoxide generators ( redox cyclers ) is also induced by nonredox cyclers through the MarA system and confers resistance toward a wide range of chemicals .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19376854" target="_blank">19376854</a>]<br></li>
<li>Production of OmpF and LamB was regulated negatively by MarA also in MC4100 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9473050" target="_blank">9473050</a>]<br></li>
<li>It was not known whether repression by MarA was also direct .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14701822" target="_blank">14701822</a>]<br></li>
<li>Presumably , the MarR-mediated repression of rnarRAB operon could also dampen the expression of the MarA positive regulator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8821940" target="_blank">8821940</a>]<br></li>
<li>A change in expression of LacZ and PhoA was also observed on X-P and X-Gal plates when only MarA , the putative activator protein encoded by the marA gene of the mar operon ( 11 , 31 , 33 ) , was introduced on a high-copy-number plasmid into the ﬁve different strains .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7836283" target="_blank">7836283</a>]<br></li>
<li>These changes were also caused by marA alone , strongly suggesting that MarA is the regulatory protein , although a role for other genes in the mar locus has not been fully excluded .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7836283" target="_blank">7836283</a>]<br></li>
<li>They were unable to detect an in # uence of MarA on slp expression ; however , the cDNA population that they examined was derived from exponentially growing cells that are known to express little slp mRNA [ 7 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11094278" target="_blank">11094278</a>]<br></li>
<li>To confirm that the decrease in rob expression was a result of MarA expression , we analyzed the levels of the rob transcript by Northern blot analysis and reverse transcription-PCR under different conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16478729" target="_blank">16478729</a>]<br></li>
<li>Furthermore , deletion of the marbox severely reduced its ability to bind MarA in vitro and reduced mar : : lacZ expression about fourfold in vivo ( Table 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8636021" target="_blank">8636021</a>]<br></li>
<li>This study shows that MarA can regulate gene expression through a functional marbox that is considerably divergent from the current consensus sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12100559" target="_blank">12100559</a>]<br></li>
<li>DISCUSSION This study demonstrates that MarA can repress the expression of the rob gene both in vivo and in vitro via a marbox sequence located within its promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16478729" target="_blank">16478729</a>]<br></li>
<li>The CpxR/CpxA response induces the expression of a variety of genes encoding MarA , MarR , AroK , and other envelope folding factors DsbA , DegP , etc. .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25294881" target="_blank">25294881</a>]<br></li>
<li>These genes encode unrelated transcriptional regulators , enzymes , transporters , and unknown proteins and can affect MarA response in general ( by altering marA expression or MarA function ) or speciﬁcally only for some antibiotics .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20211899" target="_blank">20211899</a>]<br></li>
<li>The last ﬁve genes affected MarA-mediated MDR by altering marA expression or MarA function speciﬁcally on acrA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20211899" target="_blank">20211899</a>]<br></li>
<li>This ﬁnding suggests that these genes do not affect marA expression or MarA function on genes essential for the Mar phenotype , such as acrAB or tolC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20211899" target="_blank">20211899</a>]<br></li>
<li>It is noteworthy in this connection that overexpression of MarR or MarA , or mutations in the cognate genes , did not affect emr expression ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7730261" target="_blank">7730261</a>]<br></li>
<li>In addition , the PQ - or SoxSresponsive genes gatABD , gltA , nfnB , and ybjC were recently shown to be activated by constitutive expression of MarA ( 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11395452" target="_blank">11395452</a>]<br></li>
<li>In total , 88 genes were modulated by MarA expression ; 67 genes were activated and 21 genes were down-regulated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11395452" target="_blank">11395452</a>]<br></li>
<li>Those genes activated both by exposure to NaSal and by expression of MarA , 20 in total , are indicated in boldface in Table 2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11395452" target="_blank">11395452</a>]<br></li>
<li>MarA is thought to induce the expression of several genes , including genes for efﬂux pumps ( 14 , 17 , 21 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8655497" target="_blank">8655497</a>]<br></li>
<li>The MarA protein controls expression of some of the same genes as SoxS , including many that confer resistance to antibiotics4 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10802742" target="_blank">10802742</a>]<br></li>
<li>MarA activates the expression of diverse genes , including acrAB , micF , mlr-1 , -2 , and -3 , slp , and inaA , which endow cells with resistance ( 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19376854" target="_blank">19376854</a>]<br></li>
<li>The percentage of MarA induction of gene expression for each mutant was calculated by comparing the marR / marRA ratio ( in terms of the number of tran ¬ FIG. 4 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20211899" target="_blank">20211899</a>]<br></li>
<li>Upregulation of this gene was detected in JHC1069 , whose deletion in the beginning of marR , leads to constitutive expression of MarA ( Fig. 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12100559" target="_blank">12100559</a>]<br></li>
<li>inaA is a weak acid-inducible gene whose expression is stimulated by MarA ( 41 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8521301" target="_blank">8521301</a>]<br></li>
<li>MarA seems also to compete functionally with GadX , GadW , RpoS , and Lrp under some conditions , which might ensure that hdeAB expression is not repressed more than needed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18083817" target="_blank">18083817</a>]<br></li>
<li>Expression of hdeA was repressed by induced expression of MarA ( see Ref .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14701822" target="_blank">14701822</a>]<br></li>
<li>In the same experiment we also examined the expression of hdeA , which had been observed in another macroarray study to be repressed upon MarA induction in cells ( 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14701822" target="_blank">14701822</a>]<br></li>
<li>A. Kinetics of MarA regulation of nfnB expression and of IPTG induction of marA on pMB102 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12100559" target="_blank">12100559</a>]<br></li>
<li>MarA expression from plasmid pMB102 was induced by inoculation of cells from the overnight cultures into fresh medium containing 0.5 mM IPTG .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18083817" target="_blank">18083817</a>]<br></li>
<li>Also , the MarA homolog SoxS repressed its own expression in whole cells and bound its own promoter in vitro ( 31 ) ; no further studies of this observation have been reported .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14701822" target="_blank">14701822</a>]<br></li>
<li>As anticipated , MarA did not affect the expression from the control gnd promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14701822" target="_blank">14701822</a>]<br></li>
<li>Intriguingly , we had observed down-regulation of the same promoter by constitutive expression of MarA ( 4 ) ( iscS b2530 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14701822" target="_blank">14701822</a>]<br></li>
<li>Specifically , overexpression of MarA from p9 results in decreased ompF expression that is dependent on micF but which does not require other sequences encoded by the mar locus .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8491710" target="_blank">8491710</a>]<br></li>
<li>Our results agree with those in vitro results and suggest that activation of marA expression by CRP-cAMP does occur in cells grown in rich medium , which is important for full acrA , tolC , and micF activation by MarA and thus for MarA-mediated MDR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20211899" target="_blank">20211899</a>]<br></li>
<li>Therefore , hns plays an essential role in MarA-mediated MDR by being necessary for full MarA induction of acrA expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20211899" target="_blank">20211899</a>]<br></li>
<li>Therefore , damX contributes to MarA-mediated MDR by speciﬁcally enhancing , probably indirectly , MarA induction of acrA expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20211899" target="_blank">20211899</a>]<br></li>
<li>Decreased induction of acrA expression by MarA in the hns mutant ( Fig. 2B ) did not involve alteration of marA expression ( Fig. 2A ) , and no protein-protein interaction between H-NS and MarA was found in two-hybrid experiments ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20211899" target="_blank">20211899</a>]<br></li>
<li>( B ) Effect of inactivation of each gene on MarA induction of acrA , tolC , and micF expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20211899" target="_blank">20211899</a>]<br></li>
<li>The reason for such a reduction in MarA-mediated MDR in the crp and cyaA mutants seems to be that the expression of marA ( Fig. 2A ) and , in consequence , the MarA induction of acrA , tolC , and micF ( Fig. 2B ) were signiﬁcantly reduced in both mutants .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20211899" target="_blank">20211899</a>]<br></li>
<li>However , marA expression ( Fig. 2A ) and tolC and micF activation by MarA ( Fig. 2B ) were not affected .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20211899" target="_blank">20211899</a>]<br></li>
<li>However , it dramatically decreased MarA induction of acrA expression ( Fig. 2B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20211899" target="_blank">20211899</a>]<br></li>
<li>RESULTS Transient Induction of MarA Caused a Decrease of purA and hdeA Expression in Cells -- The earlier E. coli DNA macroarray experiments showing that constitutively produced MarA reduced expression of purA ( 4 ) were confirmed subsequently using Northern assays ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14701822" target="_blank">14701822</a>]<br></li>
<li>Possibly repression in vivo of hdeA by MarA is aided by a cellular product that is constitutive or is induced by MarA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14701822" target="_blank">14701822</a>]<br></li>
<li>This ﬁnding shows that the reduced acrA activation by MarA in the acrB mutant compared to the level for the parental strain observed by RT-qPCR was produced artifactually by the cis transposome insertion in acrB and not by the absence of the AcrB protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20211899" target="_blank">20211899</a>]<br></li>
<li>In fact , when we studied the acrB mutant at the level of gene expression ( RT-qPCR ) , we found that acrA transcriptional activation by MarA was indeed signiﬁcantly reduced ( Fig. 2B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20211899" target="_blank">20211899</a>]<br></li>
<li>In the parental strain , MarA activation of acrA-lacZ expression obtained by - galactosidase assays ( not shown ) was similar to MarA activation of acrA expression found by RT-qPCR , showing the equivalence of both assays ; however , in the acrB mutant , acrA-lacZ activation by MarA was no longer reduced compared to the level for the parental strain ( not shown ) , in contrast to the RT-qPCR results .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20211899" target="_blank">20211899</a>]<br></li>
<li>Effect of different growth conditions on the expression of marRABa with and without the inducer sodium salicylate Repression in indicated growth phasea Expression ratiob Condition Strain pH 7.0 pH 5.5 Exponential Stationary phasec phasec Exponential Stationary Exponential Stationary No salicylate pH 7.0 WTb -- 1.1 -- 1.7 -- 1.1 -- 2.0 hns -- 1.1 -- 1.2 -- 1.1 -- 1.3 gadE -- 1.1 -- 1.0 -- 1.1 -- 1.1 gadX -- 1.3 -- 1.7 -- 1.2 -- 1.9 gadW -- 1.2 -- 1.7 -- 1.2 -- 2.6 rpoS -- 1.2 -- 1.5 -- 1.3 -- 2.5 lrp -- 1.4 -- 1.8 -- 1.1 -- 2.3 torR -- 1.1 -- 1.7 -- 1.1 -- 2.0 a Results are the n-fold repression ( since the values are negative ) of hdeAB expression produced by MarA ( the ratio calculated as hdeAB expression without MarA divided by hdeAB expression with MarA in the same strain and conditions ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18083817" target="_blank">18083817</a>]<br></li>
<li>Statistically signiﬁcant differences ( P 0.01 ) between related conditions ( exponential versus stationary phase at the same pH ; pH 7.0 versus pH 5.5 in the same growth phase ; and presence versus absence of MarA under the same pH and growth phase ) are indicated by asterisks .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18083817" target="_blank">18083817</a>]<br></li>
<li>On the contrary , repression by MarA increased in the gadX and lrp mutants in exponential phase at pH 7.0 and in the gadW , rpoS , and lrp mutants in stationary phase at pH 5.5 pH 5.5 5 mM salicylate for 1 hd pH 7.0 pH 5.5 5 mM salicylate , added to diluted stationary-phase cellse pH 7.0 13.9 27.4 pH 5.5 ND ND a The expression of the marRAB operon in LB medium was studied using the chromosomal marO-lacZ reporter of the strain SPC105 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18083817" target="_blank">18083817</a>]<br></li>
<li>We conclude that changes in the level of MarA are primarily responsible for neither the low level of expression of Slp during exponential phase nor for its high level of accumulation during stationary phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11094278" target="_blank">11094278</a>]<br></li>
<li>RESULTS Effect of stationary phase and pH on hdeAB expression and repression by MarA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18083817" target="_blank">18083817</a>]<br></li>
<li>To know whether repression of hdeAB by MarA was correlated with decreased acid resistance , we studied the effect of pH and growth phase on the expression of marA and the effect of MarA on acid resistance .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18083817" target="_blank">18083817</a>]<br></li>
<li>When HdeA100 cells ( bearing pMB102 ) were induced by IPTG to overexpress MarA , we found that MarA prevented nearly all of the 1.7 - to 2.5-fold increase in hdeAB expression as cells entered stationary phase ( Fig. 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18083817" target="_blank">18083817</a>]<br></li>
<li>b Repression of expression produced by MarA in the parental strain hdeAB ( see Fig. 3 ) was statistically signiﬁcant ( P 0.01 ) only in stationary phase ( values in italics ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18083817" target="_blank">18083817</a>]<br></li>
<li>This strong induction made sta-tionary-phase cells sevenfold more sensitive to acid challenge at pH 2.5 in the presence of marA than in its absence ( Fig. 4 ) , consistent with the repression produced by MarA on hdeAB ( and probably other acid resistance genes ) in stationary phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18083817" target="_blank">18083817</a>]<br></li>
<li>A dramatic decrease in the repression produced by MarA in stationary phase was found in the hns and gadE mutants at both pH 7.0 and pH 5.5 ( Table 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18083817" target="_blank">18083817</a>]<br></li>
<li>MarA did not repress hdeAB in exponential phase but did repress about twofold in stationary phase regardless of the pH ( Fig. 3 ) to nearly the exponential-phase level .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18083817" target="_blank">18083817</a>]<br></li>
<li>MarA enhanced hdeAB repression by H-NS in stationary phase at both pH 7.0 and pH 5.5 , which indicates that MarA assists or collaborates with H-NS to repress hdeAB in stationary phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18083817" target="_blank">18083817</a>]<br></li>
<li>MarA responds to weak acids such as salicylate67 and aryloxoalcanoic acids ,68 and certain anti-biotics .31 marA is part of the marRAB locus which is controlled by the repressor MarR .14 During the signaling process , salicylate binds to MarR and inactivates its activity .69 This results in the derepression of marA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11601842" target="_blank">11601842</a>]<br></li>
<li>Since overexpression of soxS or rob engenders multiple antibiotic and superoxide resistance even in the absence of mar ( 3 , 32 , 35 ) , MarA is clearly not obligatory .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8636021" target="_blank">8636021</a>]<br></li>
<li>The antibiotic resistance of AG102 , which has a point mutation in marR , the repressor gene in the marRAB operon , and consequently overexpresses the transcriptional activator , MarA ( 2 ) , was three - to sixfold greater than that of its parent , AG100 , under our conditions ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10438753" target="_blank">10438753</a>]<br></li>
				<label class="TU_collapse_similarity" for="tu_similarity_189">
Autoactivation of the marRAB multiple antibiotic resistance operon by the MarA transcriptional activator in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11395452" target="_blank">11395452</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10438794" target="_blank">10438794</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18567659" target="_blank">18567659</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10464186" target="_blank">10464186</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626315" target="_blank">8626315</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9393706" target="_blank">9393706</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21097628" target="_blank">21097628</a>]<br></label>
<input id="tu_similarity_189", type="checkbox">
<div class="TU_similar">
<li>, Wolf , Jr , R.E. and Rosner , J.L. ( 1996 ) Autoactivation of the marRAB multiple antibiotic resistance operon by the MarA transcriptional activator in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10564485" target="_blank">10564485</a>]<br></li>
<li>, Wolf , R.E. , Jr , and Rosner , J.L. ( 1996 ) Autoactivation of the marRAB multiple antibiotic resistance operon by the MarA transcriptional activator in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15225313" target="_blank">15225313</a>]<br></li>
<li>Martin , R.G. , Jair , K.W. , Wolf , R.E. , Jr. & Rosner , J.L. Autoactivation of the marRAB multiple antibiotic resistance operon by the MarA transcriptional activator in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10802742" target="_blank">10802742</a>]<br></li>
<li>Martin , R. G. , Jair , K. W. , Wolf , R. E. , Jr. , and Rosner , J. L. ( 1996 ) Autoactivation of the marRAB multiple antibiotic resistance operon by the MarA transcriptional activator in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25294881" target="_blank">25294881</a>]<br></li></div>
<li>Coregulation of nfsA and ybjC from the same SoxSdependent promoter conﬁrms an earlier suggestion that these two genes are part of an operon , consisting of ybjC , nfsA , rimK , and b0853 , that is upregulated by MarA ( 7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11741843" target="_blank">11741843</a>]<br></li>
<li>MarA , produced from the marRAB operon , affects a variety of genes associated with antibiotic resistance ( 11 ) and virulence ( 10 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22210763" target="_blank">22210763</a>]<br></li>
<li>Derepression of the marRAB operon results in an elevated amount of MarA ( 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12562820" target="_blank">12562820</a>]<br></li>
<li>The other unit is an operon comprising marRAB , encoding the MarR repressor , which binds to marO and negatively regulates expression of marRAB ( 24 , 33 ) ; MarA , a transcriptional activator ( 1 ) , which activates expression of other * Corresponding author .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10438753" target="_blank">10438753</a>]<br></li>
<li>MarA is speciﬁed by the marRAB operon , whose expression is induced by sodium salicylate , menadione , and other chemicals which inactivate the autorepressor MarR ( 1 , 10 , 33 , 38 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18083817" target="_blank">18083817</a>]<br></li>
<li>Thus , the mar operon has the remarkable property that its ﬁrst gene product , MarR , represses its transcription whereas the second gene product , MarA , stimulates its transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8636021" target="_blank">8636021</a>]<br></li>
<li>MarA may therefore act as an activator of transcription of genes ( e.g. , micF [ 6 ] ) at a distance from the marRA4B operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8383113" target="_blank">8383113</a>]<br></li>
<li>MarR is a negative autoregulator of the marRAB operon whereas MarA positively regulates transcription of the operon ( 1 , 7 ) and other genes in a mar regulon ( 6 , 26 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12562820" target="_blank">12562820</a>]<br></li>
<li>In the case of MarA , this is done by the interaction of repressor MarR , which regulates the transcription of the marRAB operon , with chemicals such as salicylate © 2003 Blackwell Publishing Lt ( Cohen et al. , 1993 ; Martin and Rosner , 1995 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12791142" target="_blank">12791142</a>]<br></li>
<li>Thus , transcription of the marRAB operon is autorepressed by MarR and autoactivated by MarA at a site that also can be activated by SoxS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8636021" target="_blank">8636021</a>]<br></li>
<li>MarA transcription is controlled by the repressor function of MarR ( encoded within the marRAB operon ; Cohen et al. , 1993 ) , whereas SoxS transcription depends on its activation by SoxR ( encoded within the soxRS region ; Amabile-Cuevas & Demple , 1991 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22515487" target="_blank">22515487</a>]<br></li>
<li>The transcription of the acrB gene is regulated by a global transcriptional activator MarA and a local transcriptional repressor AcrR .21 The acrR gene is located 141 bp upstream of the acrAB operon and transcribed divergently .21 It encodes a 215-aminoacid protein , which shares N-terminal sequence and structural similarities to members of the TetR family of transcriptional repressors .22 The signatures of the TetR family of regulators include a homologous three helix DNA-binding domain , and a diverse C-terminal domain that possesses unique sequences , forming a specific ligand-binding site for inducing compounds .22 Many TetR family members bind as dimers to their ~ 15-bp palindrome ( IR ) operator sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17950313" target="_blank">17950313</a>]<br></li>
<li>It is known that MAR is associated with an increase in MarA , which then activates the acrAB operon to produce AcrA and AcrB , which constitute the drug efﬂux pump primarily responsible for MAR ( 1 , 21 , 27 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10438753" target="_blank">10438753</a>]<br></li>
				</div></ul></div><br>
	</div>
</body>
</html>