<!doctype html>
<style>
@charset "utf-8";
body {
	height: 100%;
	margin: 0;
	padding: 0;
}

.banner {
	position: fixed;
	top: 0;
	width: 100%;
	height: 65px;
	background: gray;
	line-height: 65px;
}
.spacer {
	height: 10px;
}

.main-content {
	padding: 5%;
}

.collapse_ACT {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-style: none solid none none;
	border-color: #B4045F;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_ACT + input{
	display: none;
}
.collapse_ACT + input + div{
	display: none;
}
.collapse_ACT + input:checked + div{
	display: block;
}
.ACT {
	text-align: justify;
	text-justify: inter-word;
}
.ACT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B4045F;
}
.ACT_collapse_similarity:hover{
	background: #e5d3dc;
}
.ACT_collapse_similarity + input {
	display: none;
}
.ACT_collapse_similarity + input + div{
	display: none;
}

.ACT_collapse_similarity + input:checked + div{
	display: block;
}
.ACT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.ACT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B4045F;
	padding-left: 5px;
}
.ACT_collapse_similarity:hover{
	background: #e5d3dc;
}
.ACT_collapse_similarity + input {
	display: none;
}
.ACT_collapse_similarity + input + div{
	display: none;
}
.ACT_collapse_similarity + input:checked + div{
	display: block;
}
.ACT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.ACT_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #B4045F;
}
.ACT_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_DOM {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-style: none solid none none;
	border-color: #B404AE;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_DOM + input{
	display: none;
}
.collapse_DOM + input + div{
	display: none;
}
.collapse_DOM + input:checked + div{
	display: block;
}
.DOM {
	text-align: justify;
	text-justify: inter-word;
}
.DOM_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B404AE;
	padding-left: 5px;
}
.DOM_collapse_similarity:hover{
	background: #e8d7e7;
}
.DOM_collapse_similarity + input {
	display: none;
}
.DOM_collapse_similarity + input + div{
	display: none;
}
.DOM_collapse_similarity + input:checked + div{
	display: block;
}
.DOM_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}.DOM_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #B404AE;
}
.DOM_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_EVO {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #7401DF;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_EVO + input{
	display: none;
}
.collapse_EVO + input + div{
	display: none;
}
.collapse_EVO + input:checked + div{
	display: block;
}
.EVO {
	text-align: justify;
	text-justify: inter-word;
}
.EVO_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #7401DF;
	padding-left: 5px;
}
.EVO_collapse_similarity:hover{
	background: #e2d7ed;
}
.EVO_collapse_similarity + input {
	display: none;
}
.EVO_collapse_similarity + input + div{
	display: none;
}
.EVO_collapse_similarity + input:checked + div{
	display: block;
}
.EVO_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.EVO_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #7401DF;
}
.EVO_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_RP {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #0174DF;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_RP + input{
	display: none;
}
.collapse_RP + input + div{
	display: none;
}
.collapse_RP + input:checked + div{
	display: block;
}
.RP {
	text-align: justify;
	text-justify: inter-word;
}
.RP_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #0174DF;
	padding-left: 5px;
}
.RP_collapse_similarity:hover{
	background: #c9d7e2;
}
.RP_collapse_similarity + input {
	display: none;
}
.RP_collapse_similarity + input + div{
	display: none;
}
.RP_collapse_similarity + input:checked + div{
	display: block;
}
.RP_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.RP_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #0174DF;
}
.RP_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_SIT {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #0489B1;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_SIT + input{
	display: none;
}
.collapse_SIT + input + div{
	display: none;
}
.collapse_SIT + input:checked + div{
	display: block;
}
.SIT {
	text-align: justify;
	text-justify: inter-word;
}
.SIT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #0489B1;
	padding-left: 5px;
}
.SIT_collapse_similarity:hover{
	background: #d0dfe2;
}
.SIT_collapse_similarity + input {
	display: none;
}
.SIT_collapse_similarity + input + div{
	display: none;
}
.SIT_collapse_similarity + input:checked + div{
	display: block;
}
.SIT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.SIT_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #0489B1;
}
.SIT_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_TU {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #04B486;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_TU + input{
	display: none;
}
.collapse_TU + input + div{
	display: none;
}
.collapse_TU + input:checked + div{
	display: block;
}
.TU {
	text-align: justify;
	text-justify: inter-word;
}
.TU_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #04B486;
	padding-left: 5px;
}
.TU_collapse_similarity:hover{
	background: #d0e2de;
}
.TU_collapse_similarity + input {
	display: none;
}
.TU_collapse_similarity + input + div{
	display: none;
}
.TU_collapse_similarity + input:checked + div{
	display: block;
}
.TU_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.TU_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #04B486;
}
.TU_similar > a {
	text-decoration: none;
	color: gray;
}

ul { 
	list-style-type: none;
	padding: 0;
	margin: 0;
}
ul li > a {
	text-decoration: none;
	color: gray;
}
ul li:hover, ul li:focus {
	background: #F0F0F0;
}

ul li:first-child span {
  border-left:none;  
}

</style>
<html>
<head>
<meta charset="utf-8">
<title>TF properties - MlrA</title>
<meta name="theme-color" content="#ffffff">
</head>
<body>
	<div class="banner">
		<center><font color="white" size="16px" face="arial">MlrA</font></center>
	</div>
	<div align="center" class="main-content">
		<div class="spacer">
	</div><br>			<label class="collapse_ACT" for="_1"><font size="5px">MlrA transcriptional activity</font></label>
				<input id="_1" type="checkbox">
				<div class="ACT"><br><ul><li>MlrA represents a newly deﬁned transcriptional-regulator of csgD and agfD that is required for curli production and extracellular matrix formation by wild-type E. coli and S. typhimurium , and it is positively regulated by RpoS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11489123" target="_blank">11489123</a>]<br></li>
<li>Bars 1/4 0.5 mm MlrA expression is positively regulated by RpoS RpoS and MlrA are required for Csg and AgF synthesis and aggregation by wild-type E. coli and S. typhimurium , respectively , but neither of these regulators is necessary for induction of these phenotypes in the constitutive AgFexpressing S. typhimurium x4666 , which contains an altered agfD promoter region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11489123" target="_blank">11489123</a>]<br></li>
<li>If this is the case , the set of genes under the control of the CsgD global regulator must be activated in the presence of MlrA overexpression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20874755" target="_blank">20874755</a>]<br></li>
<li>The S-tag served for affinity chromatography to-bind MlrA to S-protein agarose .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22783906" target="_blank">22783906</a>]<br></li>
<li>Since the expression of the curli regulator CsgD is under the control of the BluR paralogue MlrA ( Brown et al. , 2001 ; Ogasawara et al. , 2010 ) , we hypothesized that BluF , when present at elevated levels , may be able to-bind MlrA and thereby also act as an anti-activator to MlrA and csgD transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22783906" target="_blank">22783906</a>]<br></li>
<li>In summary , BluR may have evolved from a duplicate of MlrA , as is suggested not only by its structural and functional similarity to MlrA , but is also evident from its residual ability to directly bind to the MlrA-dependent csgD promoter , which -- when overproduced -- allows BluR to negatively inﬂuence the expression of curli ﬁbres in E. coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22783906" target="_blank">22783906</a>]<br></li>
<li>In the presence of MlrA overexpression , an increase of csgD mRNA was observed in the mutants lacking Crl ( Fig. 4 , lanes 5 and 6 ) , H-NS ( Fig. 4 , lanes 7 and 8 ) , RstA ( Fig. 4 , lanes 11 and 12 ) , CpxR ( Fig. 4 , lanes 13 and 14 ) and RcsB ( Fig. 4 , lanes 15 and 16 ) , indicating that MlrA functions independent of these regulators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20874755" target="_blank">20874755</a>]<br></li>
<li>In the simultaneous presence of MlrA , OmpR and IHF , all three regulators bind to the same CD6 probe forming MlrA -- OmpR -- IHF -- DNA quaternary complexes ( Fig. 5c ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20874755" target="_blank">20874755</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20874755" target="_blank">20874755</a>]<br></li>
<li>Five factors ( OmpR , RstA , IHF , CpxR and H-NS ) bind to the promoter-proximal hot-spot II , while four factors ( IHF , MlrA , CpxR and H-NS ) bind to the promoter-distal hot-spot I. P1 and P2 represent the transcription initiation site P1 ( Hammar et al. , 1995 ) and P2 promoter ( Ogasawara et al. , 2007a ) , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20874755" target="_blank">20874755</a>]<br></li>
<li>CRP , CpxR , CsgD , H-NS , IHF , MlrA , OmpR , and RstA on the csgD promoter ( Ogasawara et al. 2010a , b ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23233451" target="_blank">23233451</a>]<br></li>
<li>The binding sites of MlrA , OmpR , RstA , CpxR , IHF and H-NS were determined by DNase-1 footprinting assays ( Fig. 2 ; and Ogasawara et al. , 2010 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20874755" target="_blank">20874755</a>]<br></li>
<li>Interplay between MlrA , OmpR and IHF Three positive factors , IHF , OmpR and RstA , can associate simultaneously within the promoter-proximal hot-spot II region of transcription factor binding ( Fig. 6 ) , and cooperate with each other for activation of the csgD promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20874755" target="_blank">20874755</a>]<br></li>
<li>DNA curvature induced by MlrA should inﬂuence not only the activity of neighbouring DNA such as binding afﬁnity to positive factors , IHF , OmpR and RstA , and/or to negative fractors , H-NS and CpxR , but also the mode of the molecular interplay between csgD promoter-bound transcription factors .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20874755" target="_blank">20874755</a>]<br></li>
<li>Besides requiring sS and MlrA , csgD expression is also modulated by IHF and the two-component response-regulators OmpR and CpxR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17010156" target="_blank">17010156</a>]<br></li>
<li>In principle , YdaM-generated c-di-GMP may modulate the action of any of these regulators ( i.e. OmpR , IHF , MlrA , CpxR , H-NS ) at the csgD promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17010156" target="_blank">17010156</a>]<br></li>
<li>Interplay between three activators , MlrA , OmpR and IHF , was also analysed with respect to control of this complex promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20874755" target="_blank">20874755</a>]<br></li>
<li>One explanation for the lack of csgD promoter activation in the mutants lacking OmpR and IHF is that MlrA requires these two activators for function .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20874755" target="_blank">20874755</a>]<br></li>
				</div></ul><br>
			<label class="collapse_DOM" for="_2"><font size="5px">MlrA protein domains</font></label>
				<input id="_2" type="checkbox">
				<div class="DOM"><br><ul><li>For in vitro protein interaction analysis , S-tagged MlrA was generated using pETDuet ( Merck , previously Novagen ) and primers listed in Table S1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22783906" target="_blank">22783906</a>]<br></li>
<li>This suggests that MlrA may interact with an inducer that is not speciﬁc to other known MerR-family members .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11489123" target="_blank">11489123</a>]<br></li>
<li>The C-terminal portion of MlrA from E. coli and S. typhimurium contains three conserved cysteine residues .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11489123" target="_blank">11489123</a>]<br></li>
<li>Comparison of the amino-terminal domains of E. coli MlrA and the most similar proteins in the MerR regulator family .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11489123" target="_blank">11489123</a>]<br></li>
<li>In the N-terminal 70 amino acids of MlrA , 17 residues are identical or similar to the 22 conserved residues identiﬁed in different MerR proteins from diverse sources ( Summers , 1992 ) , and many of these residues are also similar in other MerR-related proteins ( Fig. 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11489123" target="_blank">11489123</a>]<br></li>
<li>The 22 conserved residues present in most known MerR proteins ( Summers , 1992 ) are indicated below the alignment , and those that are identical or similar to MlrA are underlined .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11489123" target="_blank">11489123</a>]<br></li>
<li>The C-terminal domain MlrA , however , exhibits little similarity to the MerR-family regulators , indicating that MlrA interacts with an as yet unidentiﬁed effector using the C-terminal domain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20874755" target="_blank">20874755</a>]<br></li>
				<label class="DOM_collapse_similarity" for="dom_similarity_8">
MlrA contains a conserved Nterminal DNA-binding domain present in members of the MerR-family , implying that MlrA is a DNA-binding regulatory protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20874755" target="_blank">20874755</a>]<br></label>
<input id="dom_similarity_8", type="checkbox">
<div class="DOM_similar">
<li>MlrA contains a conserved Nterminal DNA-binding domain present in members of the MerR-family , implying the mode of MlrA action should be the same with that of other MerR-family transcription factors .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20874755" target="_blank">20874755</a>]<br></li></div>
<li>MlrA and BluR have a characteristic MerR-like DNA-binding N-terminal domain ( NTD ) and also share a spe-ciﬁc C-terminal domain ( CTD ; Fig .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22783906" target="_blank">22783906</a>]<br></li>
<li>Moreover , BluR and MlrA have similar N-terminal DNA-binding domains with only four residues differing in the helix -- turn -- helix motif .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22783906" target="_blank">22783906</a>]<br></li>
<li>The N-terminal portion of MlrA contains a putative helix -- turn -- helix domain , residues 6 -- 25 ( Fig. 2 ) , that is associated with DNA-binding by MerR regulators ( Summers , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11489123" target="_blank">11489123</a>]<br></li>
<li>What drew our attention to YcgE was the finding that not only does it contain a MerR-like N-terminal domain , but also that the complete protein shows clear homology with MlrA , which is an activator essential for the expression of the curli regulator CsgD ( Brown et al. 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19240136" target="_blank">19240136</a>]<br></li>
<li>Moreover , this assay showed that BluF interacted with the C-terminal domain of MlrA , which at ﬁrst glance seemed surprising given the previous result that BluF interacts with the N-terminal domain of the MlrA paralogue BluR ( Tschowri et al. , 2009 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22783906" target="_blank">22783906</a>]<br></li>
<li>All these observations taken together indicate that ( i ) both domains of BluF contribute in a complex way to the interactions with BluR or MlrA ( that an interaction between the puriﬁed BluFNTD and the two BluR domains was not observed in vitro ( Tschowri et al. , 2009 ) , may have been due to non-native protein conformation of BluFNTD alone ) , ( ii ) BluF is not just a defective PDE but has evolved to speciﬁcally bind to BluR , which involves contacts to both domains of BluR and ( iii ) BluF probably has evolved from a protein that originally had PDE activity and bound to the ligand-binding C-terminal domain of MlrA , the activator of csgD expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22783906" target="_blank">22783906</a>]<br></li>
<li>In order to conﬁrm the interaction between BluF and MlrA also in vivo and to assign this interaction to a speciﬁc domain of MlrA , we used the BacterioMatch II Two-Hybrid System .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22783906" target="_blank">22783906</a>]<br></li>
<li>Re-introducing conserved amino acids involved in phosphodiesterase activity of EAL-domains did not restore enzymatic activity or c-di-GMP binding of BluF , but weakened its ability to antagonize BluR and improved a residual interaction with the BluR paralogue MlrA , which controls expression of the bioﬁlm regulator CsgD and curli ﬁbres .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22783906" target="_blank">22783906</a>]<br></li>
<li>Two-hybrid assays with the isolated N-terminal and EAL-domains of BluF showed that both BluF domains contribute to the interactions with MlrA and BluR , and conﬁrmed that only MlrACTD is contacted whereas in BluR , both domains are involved ( Fig. 4B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22783906" target="_blank">22783906</a>]<br></li>
<li>In contrast , the isolated EAL-domains of BluF and BluF-M8 both interacted equally well with all interaction partner domains ( Fig. 4B ) , suggesting that in wild-type BluF the sites of interaction with MlrA are present but conformationally occluded in a way that is relieved in the mutated BluF-M8 variant .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22783906" target="_blank">22783906</a>]<br></li>
<li>Identiﬁcation of the putative effector affecting MlrA activity is also important for detailed understanding of the regulatory role of MlrA in activation of the csgD promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20874755" target="_blank">20874755</a>]<br></li>
<li>Then it may have evolved into a factor that via interaction with MlrA directly and locally antagonized csgD transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22783906" target="_blank">22783906</a>]<br></li>
<li>Currently , we are investigating regulation of mlrA in response to a variety of environmental stresses encountered during entry into stationary phase and the natural course of the disease process , what type of ligand may interact with MlrA and whether the cysteine residues play a role in this interaction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11489123" target="_blank">11489123</a>]<br></li>
				</div></ul><br>
			<label class="collapse_EVO" for="_3"><font size="5px">MlrA evolution</font></label>
				<input id="_3" type="checkbox">
				<div class="EVO"><br><ul><li>However , the C-terminal of MlrA only shares signiﬁcant homology to the putative regulators of unknown function , YcgE , ORF242 , and VCA0056 from E. coli K-12 , Salmonella enterica and Vibrio cholerae , respectively , which exhibit an overall similarity to MlrA of 76 % , 70 % and 50 % respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11489123" target="_blank">11489123</a>]<br></li>
<li>Escherichia coli contains an uncharacterized protein YcgE that exhibits an overall similarity of 76 % to MlrA ( Brown et al. , 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20874755" target="_blank">20874755</a>]<br></li>
<li>The latter protein is annotated as ` MlrA ' , but in fact shows 65 % identity to BluR and 48 % identity only to MlrA from E. coli , whereas another MerR-like protein ( ) in the K. pneumoniae chromosome shows 37 % identity to BluR and 44 % identity to MlrA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22783906" target="_blank">22783906</a>]<br></li>
<li>The 22 conserved residues present in most known MerR proteins ( Summers , 1992 ) are indicated below the alignment , and those that are identical or similar to MlrA are underlined .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11489123" target="_blank">11489123</a>]<br></li>
<li>This suggests that MlrA may interact with an inducer that is not speciﬁc to other known MerR family members .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11489123" target="_blank">11489123</a>]<br></li>
<li>Hence , this suggests the possibility that other regulators belonging to the MerR family , such as MlrA , may also interact directly with RNA polymerase in the preinitiation complex to control gene transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11489123" target="_blank">11489123</a>]<br></li>
				<label class="EVO_collapse_similarity" for="evo_similarity_7">
MlrA contains a conserved Nterminal DNA-binding domain present in members of the MerR family , implying that MlrA is a DNA-binding regulatory protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20874755" target="_blank">20874755</a>]<br></label>
<input id="evo_similarity_7", type="checkbox">
<div class="EVO_similar">
<li>MlrA contains a conserved Nterminal DNA-binding domain present in members of the MerR family , implying the mode of MlrA action should be the same with that of other MerR family transcription factors .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20874755" target="_blank">20874755</a>]<br></li></div>
<li>The C-terminal domain MlrA , however , exhibits little similarity to the MerR family regulators , indicating that MlrA interacts with an as yet unidentiﬁed effector using the C-terminal domain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20874755" target="_blank">20874755</a>]<br></li>
<li>Comparison of the amino-terminal domains of E. coli MlrA and the most similar proteins in the MerR regulator family .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11489123" target="_blank">11489123</a>]<br></li>
<li>MlrA exhibits similarity in the N-terminal portion of the molecule with the MerR family of regulatory proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11489123" target="_blank">11489123</a>]<br></li>
				</div></ul><br>
			<label class="collapse_RP" for="_4"><font size="5px">Processes regulated by MlrA</font></label>
				<input id="_4" type="checkbox">
				<div class="RP"><br><ul><li>Thus , BluR and MlrA have most likely evolved from a common ancestral gene , possibly involving a gene duplication event .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22783906" target="_blank">22783906</a>]<br></li>
<li>Note that besides the csgD promoter , the genes for two additional transcription factors , CadC and YrbA , are also under the control of MlrA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20874755" target="_blank">20874755</a>]<br></li>
<li>MICROBIOLOGY LETTERS R E S E A R C H L E T T E R Regulatory role of MlrA in transcription activation of csgD , the master regulator of bio cents lm formation in Escherichia coli Hiroshi Ogasawara1 , Kaneyoshi Yamamoto1 ,2 & Akira Ishihama1 ,2 1 2 Department of Frontier Bioscience , Hosei University , Tokyo , Japan ; and Research Center for Micro-Nano Technology , Hosei University , Tokyo , Japan Abstract Correspondence : Akira Ishihama , Department of Frontier Bioscience , Hosei The transcription factor CsgD plays a key role in the control of bioﬁlm formation in Escherichia coli by controlling the production of curli ﬁmbriae and other bioﬁlm components .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20874755" target="_blank">20874755</a>]<br></li>
<li>Regulatory role of MlrA in transcription activation of csgD , the master regulator of biofilm-formation in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22442305" target="_blank">22442305</a>]<br></li>
<li>J Bacteriol 189:4791 -- 4799 Ogasawara H , Yamamoto K , Ishihama A ( 2010a ) Regulatory role of MlrA in transcription activation of csgD , the master regulator of bioWlm formation in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21559929" target="_blank">21559929</a>]<br></li>
<li>Microbiology 156:2470 -- 2483 Ogasawara H , Yamamoto K , Ishihama A ( 2010b ) Regulatory role of MlrA in transcription activation of csgD , the master regulator of biofilm-formation in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23824318" target="_blank">23824318</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_7">
Ogasawara , H. , Yamamoto , K. , and Ishihama , A. ( 2010 ) Regulatory role of MlrA in transcription activation of csgD , the master regulator of bioﬁlm formation in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22783906" target="_blank">22783906</a>]<br></label>
<input id="rp_similarity_7", type="checkbox">
<div class="RP_similar">
<li>Ogasawara , H. , Yamamoto , K. , and Ishihama , A. ( 2010b ) Regulatory role of MlrA in transcription activation of csgD , the master regulator of bioﬁlm formation in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22250746" target="_blank">22250746</a>]<br></li></div>
<li>In parallel , S-containing RNAP ( E S ) activates the expression of MlrA ( a MerR-like regulator ) and YdaM ( a GGDEF protein , see below ) , which , together with E S again , are essential to activate transcription of the csgD gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18765794" target="_blank">18765794</a>]<br></li>
<li>YcgE represents a closely related paralogue of MlrA , a MerR-like regulator that directly activates the transcription of the important bioﬁlm regulator CsgD ( Brown et al. , 2001 ; 2003 ; Ogasawara et al. , 2010 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22783906" target="_blank">22783906</a>]<br></li>
<li>( b ) The MlrA box-like sequences on the also examined for a set of E. coli mutants , each lacking one of the transcription factors that are involved in the regulation of the csgD promoter ( Ogasawara et al. , 2010 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20874755" target="_blank">20874755</a>]<br></li>
<li>We have identiﬁed a novel regulator , termed MlrA , that is Introduction Bacterial adherence to host tissues and cells is often mediated by non-ﬂagellar ﬁlamentous cell surface appen-dages , known as ﬁmbriae ( or pili ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11489123" target="_blank">11489123</a>]<br></li>
<li>Here we describe an as yet uncharacterized regulator , MlrA , that has been suggested to participate in control of curli formation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20874755" target="_blank">20874755</a>]<br></li>
<li>In this report , we determine that curli mediate MRHA by strain x7122 , and identify that a novel regulator , MlrA , or RpoS ( s ) encoded by the x7122 DNA-containing cosmids s confers curli expression to E. coli HB101 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11489123" target="_blank">11489123</a>]<br></li>
<li>These results indicate that MlrA is a newly deﬁned transcriptional regulator of csgD/agfD that acts as Summary Production of curli ( AgF ) adhesins by Escherichia coli and Salmonella enterica serovar Typhimurium ( S. typhimurium ) is associated with extracellular matrix production and is optimal at low temperature a positive regulator of RpoS-dependent curli and extracellular matrix production by E. coli and S. typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11489123" target="_blank">11489123</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_16">
MlrA , a novel regulator of curli ( Agf ) and extracellular matrix synthesis by Escher-ichia coli and Salmonella enterica serovar typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19240136" target="_blank">19240136</a>]<br></label>
<input id="rp_similarity_16", type="checkbox">
<div class="RP_similar">
<li>MlrA , a novel regulator of curli ( Agf ) and extracellular matrix synthesis by Escherichia coli and Salmonella enterica serovar Typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19332833" target="_blank">19332833</a>]<br></li>
<li>MlrA , a novel regulator of curli ( Agf ) and extracellular matrix synthesis by Escherichia coli and Salmonella enterica serovar typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18765794" target="_blank">18765794</a>]<br></li>
<li>MlrA , a novel regulator of curli ( AgF ) and extracellular matrix synthesis by Escherichia coli and Salmonella enterica serovar Typhi-murium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16513732" target="_blank">16513732</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17468243" target="_blank">17468243</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12775697" target="_blank">12775697</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17873038" target="_blank">17873038</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15743952" target="_blank">15743952</a>]<br></li>
<li>MlrA , a novel regulator curli ( AgF ) and exracellular matrix synthesis by Escherichia coli and Salmonella enterica serovar Typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20466769" target="_blank">20466769</a>]<br></li>
<li>MlrA , a novel regulator of curli ( AgF ) and extracellular matrix synthesis by Escherichia coli and Salmonella enterica serovar Typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22289118" target="_blank">22289118</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16000739" target="_blank">16000739</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22685278" target="_blank">22685278</a>]<br></li></div>
				<label class="RP_collapse_similarity" for="rp_similarity_22">
Brown , P.K. , Dozois , C.M. , Nickerson , C.A. , Zuppardo , A. , Terlonge , J. , and Curtiss , III , R. ( 2001 ) MlrA , a novel regulator of curli ( Agf ) and extracellular matrix synthesis by Escherichia coli and Salmonella enterica serovar typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22783906" target="_blank">22783906</a>]<br></label>
<input id="rp_similarity_22", type="checkbox">
<div class="RP_similar">
<li>Brown , P.K. , Dozois , C.M. , Nickerson , C.A. , Zuppardo , A. , Terlonge , J. , and Curtiss , R. , III ( 2001 ) MlrA , a novel regulator of curli ( Agf ) and extracellular matrix synthesis by Escherichia coli and Salmonella enterica serovar typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17010156" target="_blank">17010156</a>]<br></li></div>
				<label class="RP_collapse_similarity" for="rp_similarity_24">
FEMS Microbiol Rev 27 : Brown PK , Dozois CM , Nickerson CA , Zupardo A , Terlonge J & Curtiss R III ( 2001 ) MlrA , a novel regulator curli ( AgF ) and exracellular matrix synthesis by Escherichia coli and Salmonella enterica serovar Typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20874755" target="_blank">20874755</a>]<br></label>
<input id="rp_similarity_24", type="checkbox">
<div class="RP_similar">
<li>Annu Rev Microbiol 60:131 -- 147 Brown PK , Dozois CM , Nickerson CA , Zuppardo A , Terlonge J , Curtiss R III ( 2001 ) MlrA , a novel regulator of curli ( AgF ) and extracellular matrix synthesis by Escherichia coli and Salmonella enterica serovar Typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21559929" target="_blank">21559929</a>]<br></li></div>
<li>Reﬂecting such situations , expression of the csgD gene is known to be positively or negatively regulated by a number of transcription factors , including MlrA ( 5 ) , Crl ( 3 ) , RpoS ( 3 ) , integration host factor ( 6 ) , Rcs ( 34 ) , the EnvZ-OmpR TCS ( 13 , 35 ) and the CpxAR TCS ( 13 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17468243" target="_blank">17468243</a>]<br></li>
<li>It is strictly dependent on the general stress sigma factor s , S which activates the expression of MlrA , a transcriptional activator that cooperates with s - containing RNA poly-S merase ( RNAP ) to activate the expression of CsgD , which acts as an essential activator that together with vegetative RNAP turns on the expression of the curli structural operon csgBAC ( Brown et al. , 2001 ; Gerstel et al. , 2003 ; Römling et al. , 2000 ; Weber et al. , 2006 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19332833" target="_blank">19332833</a>]<br></li>
<li>The stationary phase global regulator RpoS ( S ) ( Olsén et al. 1993 ; Römling et al. 1998 ) and other global regulators including H-NS ( Arnqvist et al. 1994 ; Olsén et al. 1993 ) , Crl ( Arnqvist et al. 1992 ) , OmpR ( Römling et al. 1998 ; Vidal et al. 1998 ) , MlrA ( Brown et al. 2001 ) and CRP ( Zheng et al. 2004 ) can aVect curli synthesis by promoting or suppressing expression of the curli-encoding genes by regulating the expression of csgD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21559929" target="_blank">21559929</a>]<br></li>
<li>2010 SGM Printed in Great Britain In concert with the complex mode of regulation for biofilm-formation , the expression of csgD has been proposed to be under the control of various stress-sensing regulatory proteins such as OmpR ( Romling et al. , 1998 ; Vidal et al. , 1998 ; Prigent-Combaret et al. , 1999 , 2000 , 2001 ; Gerstel & Romling , 2001 ; Gerstel et al. , 2003 ) , CpxR ( PrigentCombaret et al. , 2001 ; Jubelin et al. , 2005 ) , Crl ( Bougdour et al. , 2004 ; Robbe-Saule et al. , 2006 ) , CRP ( Zheng et al. , 2004 ) , H-NS ( Arnqvist et al. , 1994 ; Gerstel et al. , 2003 ; Olsen et al. , 1993 ) , IHF ( Gerstel et al. , 2003 ; Gerstel & Romling , 2001 ) , MlrA ( Brown et al. , 2001 ) and RcsB ( Ferrieres & Clarke , 2003 ; Vianney et al. , 2005 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20466769" target="_blank">20466769</a>]<br></li>
<li>Furthermore , a number of transcription factors are involved in the regulation of the csgD promoter , including CpxR ( Jubelin et al. , 2005 ) , Crl ( Bougdour et al. , 2004 ) , H-NS ( Gerstel et al. , 2003 ) , IHF ( Gerstel et al. , 2003 , 2006 ) , OmpR ( Vidal et al. , 1998 ; Gerstel et al. , 2003 , 2006 ) , RstA ( Ogasawara et al. , 2007a ) , MlrA ( Brown et al. , 2001 ) , RcsB ( Ferrieres & Clarke , 2003 ; Vianney et al. , 2005 ) and CRP ( Zheng et al. , 2004 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20874755" target="_blank">20874755</a>]<br></li>
<li>This module exhibits complex feedforward regulation by sS , i.e. the target genes are under multiple direct and indirect control by sS ( summarized in Fig. 10 ) : sS activates the expression of MlrA ( an essential positive regulator for csgD expression ; Brown et al. , 2001 ) ; in addition , sS is probably directly involved in csgD transcription ( Römling et al. , 1998a ) , and , as shown here , sS is required for the expression of + c ydaM / ydaM : : cat csgG csgF csgE csgD csgB csgA csgC ymdA ymfE rpsN hdeB + yciR / yciR : : kan csgG csgB csgA csgC ymdA yddV + / yddV : : Tn5 ygdI dppA tnaA ytfK yedQ + / yedQ : : cat miaA paaJ glmM b1037 b1038 b1039 b1040 b1041 b1042 b1043 b1044 b1138 b3307 b3509 3.215 4.404 4.117 4.050 14.505 29.392 10.037 4.755 7.465 2.243 2.219 b1037 b1041 b1042 b1043 b1044 0.469 0.202 0.205 0.193 0.341 b2809 b3544 b3708 b4217 2.349 2.585 2.161 2.286 2.006 0.449 0.403 b4171 b1397 b3176 YdaM and YciR ( Figs 1 and 2 , and Fig .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17010156" target="_blank">17010156</a>]<br></li>
<li>In wild-type background , overexpression of MlrA led to an approximately 2.5-fold increase in csgD -- lacZ expression ( Fig. 1a ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20874755" target="_blank">20874755</a>]<br></li>
<li>Using single-copy lacZ reporter fusions , we then found that pFliZ reduced the expression of MlrA , YdaM , and YciR ( Fig. 1B ) , which together and highly specifically control csgD transcription ( Weber et al. 2006 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18765794" target="_blank">18765794</a>]<br></li>
<li>If this is the case , the set of genes under the control of the CsgD global regulator must be activated in the presence of MlrA overexpression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20874755" target="_blank">20874755</a>]<br></li>
<li>Two directly S-controlled genes encoding MlrA ( A ) and YdaM ( B ) ( i.e. , the factors essential for the expression of the curli activator CsgD ) , as well as a construct with a synthetic promoter that responds exclusively to S , synp9 ( C ) , were assayed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18765794" target="_blank">18765794</a>]<br></li>
<li>Graphical summary of the roles of the BluF/BluR and YdaM/YciR/MlrA systems in the control of expression of CsgD and curli ﬁbres .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22783906" target="_blank">22783906</a>]<br></li>
<li>W3110 reaches very similar S levels in stationary phase as MC4100 ( data not shown ) and also expresses the curli genes ( assayed using a single-copy csgB lacZ reporter fusion ) under the same control of S , MlrA , YdaM , YciR , and CsgD as MC4100 ( Supplemental Fig .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18765794" target="_blank">18765794</a>]<br></li>
				</div></ul><br>
			<label class="collapse_SIT" for="_5"><font size="5px">MlrA binding site</font></label>
				<input id="_5" type="checkbox">
				<div class="SIT"><br><ul><li>MlrA binds a 33-bp-long palindromic sequence ( see Fig. 3 ) , which is much longer than the hitherto identiﬁed binding sequences by E. coli transcription factors ( Ishihama , 2010 ) and probably induces DNA underwinding as in the case of MerR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20874755" target="_blank">20874755</a>]<br></li>
<li>The MlrA binding site in the csgD promoter region was recently identiﬁed ( AAAGTTGTACA ( 12N ) TGCACAATT TT ) ( Ogasawara et al. , 2010 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22783906" target="_blank">22783906</a>]<br></li>
<li>Interestingly , the GTACA motif recognized by BluR is also present in the MlrA binding site in the csgD promoter region ( see above , highlighted in bold ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22783906" target="_blank">22783906</a>]<br></li>
<li>S2 ) as MlrA and binds to a GTACA sequence motif , which is also present in the MlrA binding site in the csgD promoter region , we wondered whether BluR is able to bind to the csgD promoter region , too .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22783906" target="_blank">22783906</a>]<br></li>
<li>Based on both in vivo and in vitro analyses , we identiﬁed MlrA as a positive factor of the csgD promoter by directly binding to its upstream region ( 113 to 146 ) with a palindromic sequence of AAAATTGTA CA ( 12N ) TGTACAATTTT between the binding sites of two activators , IHF and OmpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20874755" target="_blank">20874755</a>]<br></li>
<li>Boxed sequences represent the binding sites of IHF ( site-I ) and MlrA while the OmpR-binding sequence is shown by a bar below the sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20874755" target="_blank">20874755</a>]<br></li>
<li>For the genes shown in blue , MlrA box-like sequences are located at promoterproximal upstream regions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20874755" target="_blank">20874755</a>]<br></li>
<li>( a ) After searches for the inverted-repeat sequence [ AAAATTTGTACA ( 12N ) TGTACAATTTT ] of predicted MlrA target genes are aligned .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20874755" target="_blank">20874755</a>]<br></li>
				</ul></div><br>
			<label class="collapse_TU" for="_6"><font size="5px">Transcription unit regulated by MlrA</font></label>
				<input id="_6" type="checkbox">
				<div class="TU"><br><ul><li>For the genes shown in blue , MlrA box-like sequences are located at promoterproximal upstream regions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20874755" target="_blank">20874755</a>]<br></li>
<li>Expression and purification of transcription factors For construction of plasmids ( pMlrA and pOmpR ) for expression of His-tagged MlrA and OmpR , DNA fragments corresponding to the coding sequences of their respective transcription factors were ampliﬁed by PCR using E. coli W3110 genome DNA as a template and pairs of primers , which were designed so as to hybridize upstream or downstream of each coding sequence ( Table S2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20874755" target="_blank">20874755</a>]<br></li>
<li>MlrA binds a 33-bp-long palindromic sequence ( see Fig. 3 ) , which is much longer than the hitherto identiﬁed binding sequences by E. coli transcription factors ( Ishihama , 2010 ) and probably induces DNA underwinding as in the case of MerR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20874755" target="_blank">20874755</a>]<br></li>
<li>Expression of the csgBAC operon , which encodes the curli structural proteins , is driven by a regulatory cascade consisting of sS , the MerR-like regulator MlrA and the CsgD regulatory protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17010156" target="_blank">17010156</a>]<br></li>
<li>BluF has residual affinity for the BluR paralogue MlrA which is improved upon restoring of consensus amino acids in the degenerate EAL domain Knocking out bluF in E. coli does not inﬂuence the c-di-GMP-responsive expression of curli ﬁbres as monitored by a csgB : : lacZ reporter gene fusion ( Sommerfeldt et al. , 2009 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22783906" target="_blank">22783906</a>]<br></li>
<li>During entry into stationary phase , sS is induced and exerts multiple activating effects in parallel ( on the expression of the GGDEF/EAL proteins YdaM/YciR and of the serially arranged activators MlrA and CsgD ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17010156" target="_blank">17010156</a>]<br></li>
<li>W3110 reaches very similar S levels in stationary phase as MC4100 ( data not shown ) and also expresses the curli genes ( assayed using a single-copy csgB lacZ reporter fusion ) under the same control of S , MlrA , YdaM , YciR , and CsgD as MC4100 ( Supplemental Fig .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18765794" target="_blank">18765794</a>]<br></li>
<li>Using single-copy lacZ reporter fusions , we then found that pFliZ reduced the expression of MlrA , YdaM , and YciR ( Fig. 1B ) , which together and highly specifically control csgD transcription ( Weber et al. 2006 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18765794" target="_blank">18765794</a>]<br></li>
<li>Using a single copy translational mlrA : : lacZ fusion , we found that expression of MlrA , i.e. the second factor in the sS-MlrA-CsgD transcription factor cascade , is not affected by mutations in ydaM or yciR ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17010156" target="_blank">17010156</a>]<br></li>
<li>As overexpression of MlrA did not affect the promoter of the divergently transcribed csgBC operon , the effect of MlrA on curli production is solely attributable to activation of the csgD promoter , ultimately leading to activation of the csgB promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20874755" target="_blank">20874755</a>]<br></li>
<li>Furthermore , s may be required for both expression of s mlrA and direct co-operative interaction with the MlrA protein to initiate csg gene transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11489123" target="_blank">11489123</a>]<br></li>
<li>Reﬂecting such situations , expression of the csgD gene is known to be positively or negatively regulated by a number of transcription factors , including MlrA ( 5 ) , Crl ( 3 ) , RpoS ( 3 ) , integration host factor ( 6 ) , Rcs ( 34 ) , the EnvZ-OmpR TCS ( 13 , 35 ) and the CpxAR TCS ( 13 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17468243" target="_blank">17468243</a>]<br></li>
<li>Besides requiring sS and MlrA , csgD expression is also modulated by IHF and the two-component response regulators OmpR and CpxR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17010156" target="_blank">17010156</a>]<br></li>
<li>Note that besides the csgD promoter , the genes for two additional transcription factors , CadC and YrbA , are also under the control of MlrA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20874755" target="_blank">20874755</a>]<br></li>
<li>DNA curvature induced by MlrA should inﬂuence not only the activity of neighbouring DNA such as binding afﬁnity to positive factors , IHF , OmpR and RstA , and/or to negative fractors , H-NS and CpxR , but also the mode of the molecular interplay between csgD promoter-bound transcription factors .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20874755" target="_blank">20874755</a>]<br></li>
<li>Together , these observations indicate that MlrA is a DNA-binding positive regulator of csgD transcription , but its homologue YcgE is not involved in csgD regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20874755" target="_blank">20874755</a>]<br></li>
<li>The molecular mechanisms of action of MlrA ( in csgD transcription ) and CsgD ( in csgBAC transcription ) have not been clariﬁed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17010156" target="_blank">17010156</a>]<br></li>
<li>When MlrA was overexpressed using arabinose-inducible expression vector , csgD mRNA markedly increased even in the wild-type E. coli ( Fig. 4 , lanes 1 and 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20874755" target="_blank">20874755</a>]<br></li>
<li>The expression level of MlrA in both wild-type E. coli and the csgD mutant was essentially the same as detected by immunoblot analysis of whole-cell lysates ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20874755" target="_blank">20874755</a>]<br></li>
<li>The induction level of csgD mRNA by MlrA overexpression in both the rpoS ( data not shown ) and the crl ( Ishihama , 2010 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20874755" target="_blank">20874755</a>]<br></li>
<li>In summary , BluR may have evolved from a duplicate of MlrA , as is suggested not only by its structural and functional similarity to MlrA , but is also evident from its residual ability to directly bind to the MlrA-dependent csgD promoter , which -- when overproduced -- allows BluR to negatively inﬂuence the expression of curli ﬁbres in E. coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22783906" target="_blank">22783906</a>]<br></li>
<li>Also at its nonphysiological target , the csgD promoter , overexpressed BluR inactivates expression , although it replaces an activator , i.e. MlrA , in this situation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22783906" target="_blank">22783906</a>]<br></li>
				</div></ul></div><br>
	</div>
</body>
</html>