<!doctype html>
<style>
@charset "utf-8";
body {
	height: 100%;
	margin: 0;
	padding: 0;
}

.banner {
	position: fixed;
	top: 0;
	width: 100%;
	height: 65px;
	background: gray;
	line-height: 65px;
}
.spacer {
	height: 10px;
}

.main-content {
	padding: 5%;
}

.collapse_ACT {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-style: none solid none none;
	border-color: #B4045F;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_ACT + input{
	display: none;
}
.collapse_ACT + input + div{
	display: none;
}
.collapse_ACT + input:checked + div{
	display: block;
}
.ACT {
	text-align: justify;
	text-justify: inter-word;
}
.ACT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B4045F;
}
.ACT_collapse_similarity:hover{
	background: #e5d3dc;
}
.ACT_collapse_similarity + input {
	display: none;
}
.ACT_collapse_similarity + input + div{
	display: none;
}

.ACT_collapse_similarity + input:checked + div{
	display: block;
}
.ACT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.ACT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B4045F;
	padding-left: 5px;
}
.ACT_collapse_similarity:hover{
	background: #e5d3dc;
}
.ACT_collapse_similarity + input {
	display: none;
}
.ACT_collapse_similarity + input + div{
	display: none;
}
.ACT_collapse_similarity + input:checked + div{
	display: block;
}
.ACT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.ACT_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #B4045F;
}
.ACT_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_DOM {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-style: none solid none none;
	border-color: #B404AE;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_DOM + input{
	display: none;
}
.collapse_DOM + input + div{
	display: none;
}
.collapse_DOM + input:checked + div{
	display: block;
}
.DOM {
	text-align: justify;
	text-justify: inter-word;
}
.DOM_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B404AE;
	padding-left: 5px;
}
.DOM_collapse_similarity:hover{
	background: #e8d7e7;
}
.DOM_collapse_similarity + input {
	display: none;
}
.DOM_collapse_similarity + input + div{
	display: none;
}
.DOM_collapse_similarity + input:checked + div{
	display: block;
}
.DOM_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}.DOM_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #B404AE;
}
.DOM_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_EVO {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #7401DF;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_EVO + input{
	display: none;
}
.collapse_EVO + input + div{
	display: none;
}
.collapse_EVO + input:checked + div{
	display: block;
}
.EVO {
	text-align: justify;
	text-justify: inter-word;
}
.EVO_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #7401DF;
	padding-left: 5px;
}
.EVO_collapse_similarity:hover{
	background: #e2d7ed;
}
.EVO_collapse_similarity + input {
	display: none;
}
.EVO_collapse_similarity + input + div{
	display: none;
}
.EVO_collapse_similarity + input:checked + div{
	display: block;
}
.EVO_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.EVO_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #7401DF;
}
.EVO_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_RP {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #0174DF;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_RP + input{
	display: none;
}
.collapse_RP + input + div{
	display: none;
}
.collapse_RP + input:checked + div{
	display: block;
}
.RP {
	text-align: justify;
	text-justify: inter-word;
}
.RP_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #0174DF;
	padding-left: 5px;
}
.RP_collapse_similarity:hover{
	background: #c9d7e2;
}
.RP_collapse_similarity + input {
	display: none;
}
.RP_collapse_similarity + input + div{
	display: none;
}
.RP_collapse_similarity + input:checked + div{
	display: block;
}
.RP_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.RP_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #0174DF;
}
.RP_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_SIT {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #0489B1;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_SIT + input{
	display: none;
}
.collapse_SIT + input + div{
	display: none;
}
.collapse_SIT + input:checked + div{
	display: block;
}
.SIT {
	text-align: justify;
	text-justify: inter-word;
}
.SIT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #0489B1;
	padding-left: 5px;
}
.SIT_collapse_similarity:hover{
	background: #d0dfe2;
}
.SIT_collapse_similarity + input {
	display: none;
}
.SIT_collapse_similarity + input + div{
	display: none;
}
.SIT_collapse_similarity + input:checked + div{
	display: block;
}
.SIT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.SIT_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #0489B1;
}
.SIT_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_TU {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #04B486;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_TU + input{
	display: none;
}
.collapse_TU + input + div{
	display: none;
}
.collapse_TU + input:checked + div{
	display: block;
}
.TU {
	text-align: justify;
	text-justify: inter-word;
}
.TU_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #04B486;
	padding-left: 5px;
}
.TU_collapse_similarity:hover{
	background: #d0e2de;
}
.TU_collapse_similarity + input {
	display: none;
}
.TU_collapse_similarity + input + div{
	display: none;
}
.TU_collapse_similarity + input:checked + div{
	display: block;
}
.TU_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.TU_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #04B486;
}
.TU_similar > a {
	text-decoration: none;
	color: gray;
}

ul { 
	list-style-type: none;
	padding: 0;
	margin: 0;
}
ul li > a {
	text-decoration: none;
	color: gray;
}
ul li:hover, ul li:focus {
	background: #F0F0F0;
}

ul li:first-child span {
  border-left:none;  
}

</style>
<html>
<head>
<meta charset="utf-8">
<title>TF properties - ModE</title>
<meta name="theme-color" content="#ffffff">
</head>
<body>
	<div class="banner">
		<center><font color="white" size="16px" face="arial">ModE</font></center>
	</div>
	<div align="center" class="main-content">
		<div class="spacer">
	</div><br>			<label class="collapse_ACT" for="_1"><font size="5px">ModE transcriptional activity</font></label>
				<input id="_1" type="checkbox">
				<div class="ACT"><br><ul><li>A 7000 6000 2000 1500 1000 0 pSch2 - pSch2 - pSch3 - pSch5 - pSch7 - pSch7-ybhK moaA dmsA ynfE napF eco 1000 ββ-Galactosidase activity ( Miller unit 500 0 pSch5 - pSch6 - pSch9-anti - anti- anti-ynfD cutC yghX B-300 -200 -100 +1 +100 +200 +300 ModE-activating promoter ybhKp moaAp1 moaAp2 ModE FNR moaAp2 ybhKp FNR ybhK moaAp1 moaA ModE dmsA2 dmsAp1 dmsA serS FNR ModE ynfEp1 ynfE FNR ModE napFp2 napFp1 napF napFp3 yojO ecop ModE F napFp3 NR napFp1 napFp2 ecop eco yojO ModE FNR ModE-repressing promoter modAp modA ModE morAp ybhT yghX yghW ModE FIG 2 Promoters activated and repressed by ModE .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23913318" target="_blank">23913318</a>]<br></li>
<li>By genomic systematic evolution of ligands by exponential enrichment ( SELEX ) screening and promoter reporter assays , we have identiﬁed a total of nine operons , including the hitherto identiﬁed modA , moaA , dmsA , and napF operons , of which six were activated by ModE and three were repressed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23913318" target="_blank">23913318</a>]<br></li>
<li>Results indicated that ModE activates six promoters ( dmsAp , napFp , moaAp , ybhKp , ynfEp , and ecop ) and repressed the modA promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23913318" target="_blank">23913318</a>]<br></li>
<li>( 2003 ) 326 , 761 -- 767 mational es Involving B Oxyanion A-binding D M E eveals oth Structure of Activated od R Crystal Confor N Ch g an ains and D om Alexander W. Schüttelkopf , David H. Boxer and William N. Hunter * Division of Biological Chemistry and Molecular Microbiology , The Wellcome Trust Biocentre , School of Life Sciences , University of Dundee Dundee DD1 5EH , UK * Corresponding author Introduction The molybdate-dependent transcriptional-activator/repressor ModE appears to be the principal regulatory component of bacterial molybdenum metabolism , occuring in organisms like Escherichia coli , Yersinia pestis , Ralstonia solanacearum and Rhodobacter capsulatus .1 -- 6 ModE , activated by binding molybdate , acts as both a positive and a negative regulator of transcription of operons involved in various aspects of molybdenum metabolism .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12581638" target="_blank">12581638</a>]<br></li>
<li>Oxyanion Binding Alters Conformation and Quaternary Structure of the C-terminal Domain of the Transcriptional Regulator ModE IMPLICATIONS FOR MOLYBDATE-DEPENDENT REGULATION , SIGNALING , STORAGE , AND TRANSPORT * Received for publication , January 31 , 2001 , and in revised form , March 19 , 2001 Published , JBC Papers in Press , March 20 , 2001 , DOI 10.1074 / jbc.M100919200 David G. Gourley ‡ § , Alexander W. Schüttelkopf ‡ § , Lisa A. Anderson ‡ , Nicholas C. Price ¶ i , David H. Boxer ‡ , and William N. Hunter ‡ ** From ‡ The Wellcome Trust Biocentre , University of Dundee , Dundee , DD1 5EH and the ¶ Department of Biological Sciences , University of Stirling , Stirling , FK9 4LA , United Kingdom The molybdate-dependent transcriptional-regulator ModE of Escherichia coli functions as a sensor of intra - cellular molybdate concentration and a regulator for the transcription of several operons that control the uptake and utilization of molybdenum .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11259434" target="_blank">11259434</a>]<br></li>
<li>Identiﬁcation of the Set of Genes , Including Nonannotated morA , under the Direct Control of ModE in Escherichia coli Tatsuaki Kurata , a Akira Katayama , c Masakazu Hiramatsu , a Yuya Kiguchi , a Masamitsu Takeuchi , a Tomoyuki Watanabe , a Hiroshi Ogasawara , a , d Akira Ishihama , a , b Kaneyoshi Yamamotoa , b Department of Frontier Biosciencea and Research Center for Micro-Nano Technology , b Hosei University , Tokyo , Japan ; Department of Biochemistry and Molecular Biology , Nippon Medical School , Tokyo , Japanc ; Research Center for Human and Environmental Sciences , Shinshu University , Nagano Prefecture , Japand ModE is the molybdate-sensing transcription regulator that controls the expression of genes related to molybdate homeostasis in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23913318" target="_blank">23913318</a>]<br></li>
<li>The Molybdate-Responsive Escherichia coli ModE Transcriptional Regulator Coordinates Periplasmic Nitrate Reductase ( napFDAGHBC ) Operon Expression with Nitrate and Molybdate Availability Paul M. McNicholas † and Robert P. Gunsalus * Department of Microbiology , Immunology , and Molecular Genetics , and Molecular Biology Institute , University of California , Los Angeles , Los Angeles , California 90095-1489 Received 1 June 2001/Accepted 16 March 2002 Expression of the Escherichia coli napFDAGHBC operon ( also known as aeg46 .5 ) , which encodes the periplasmic molybdoenzyme for nitrate reduction , is increased in response to anaerobiosis and further stimulated by the addition of nitrate or to a lesser extent by nitrite to the cell culture medium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12029041" target="_blank">12029041</a>]<br></li>
<li>naerobic regulation of the Escherichia coli dmsABC operon requires the molyb A date-responsive regulator ModE Paul M. McNicholas , Robin C. Chiang and Robert P. Gunsalus * Department of Microbiology and Molecular Genetics and the Molecular Biology Institute , 1602 Molecular Sciences Building , 405 Hilgard Avenue , University of California at Los Angeles , Los Angeles , CA 90095 1489 , USA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466267" target="_blank">9466267</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_9">
McNicholas , P. , Chiang , R. , and Gunsalus , R. ( 1998 ) Anaerobic regulation of the Escherichia coli dmsABC operon requires the molybdate-responsive regulator ModE .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11115116" target="_blank">11115116</a>]<br></label>
<input id="act_similarity_9", type="checkbox">
<div class="ACT_similar">
<li>[ 10 ] McNicholas , P.M. , Chiang , R.C. and Gunsalus , R.P. ( 1998 ) Anaerobic regulation of the Escherichia coli dmsABC operon requires the molybdate-response-regulator ModE .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9851041" target="_blank">9851041</a>]<br></li></div>
<li>Department of Microbiology and Cell Science , University of Florida , Gainesville , FL 32611 , USA 1 Keywords : molybdate , regulation , ModE , formate hydrogenlyase , nay operon INTRODUCTION Under anaerobic-growth conditions , Escherichia coli produces several molybdoenzymes , almost all of which serve as terminal enzymes in anaerobic respiratory pathways ( Gennis & Stewart , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10206709" target="_blank">10206709</a>]<br></li>
<li>An Analysis of the Binding of Repressor Protein ModE to modABCD ( Molybdate Transport ) Operator/Promoter DNA of Escherichia coli * ( Received for publication , October 9 , 1998 , and in revised form , April 5 , 1999 ) Amy M. Grunden ‡ § ¶ , William T. Self ‡ ¶ , Matteo Villaini , J. Edwin Blalocki , and K. T. Shanmugam ‡ ** From the ‡ Department of Microbiology and Cell Science , University of Florida , Gainesville , Florida 32611 and the iDepartment of Physiology and Biophysics , University of Alabama at Birmingham , Birmingham , Alabama 35294 Expression of the modABCD operon in Escherichia coli , which codes for a molybdate-specific transporter , is repressed by ModE in vivo in a molybdate-dependent fashion .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10446207" target="_blank">10446207</a>]<br></li>
<li>Thus , changes in oxygen availability ( sensed directly by FNR , and indirectly by ArcA ) affect the redox state of the system , leading to consequences for metal ion homeostasis ( copper , molybdate , iron ; sensed by CusR , ModE and Fur ) , iron -- sulphur cluster turnover ( sensed by IscR ) , ribonucleotide reductase activity 20 0 31 0 10 aerobiosis ( % ) time ( min ) Figure 1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22870390" target="_blank">22870390</a>]<br></li>
<li>A further difference between apo and ligandbound structure is that helix a4 of chain B is kinked at residue 97 in the apo structure , while this kink is not seen in liganded ModE ( asterisk in Figure 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12581638" target="_blank">12581638</a>]<br></li>
<li>This might prevent a5 of chain A from adopting its apo conformation , which would be expected to be a requirement for ModE achieving its bent conformation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12581638" target="_blank">12581638</a>]<br></li>
<li>Lane 1 , no protein added ; lane 2 , 85 nM ModE ; lane 3 , 130 nM ModE ; lane 4 , 170 nM ModE ; lane 5 , 212nM ModE ; lane 6 , 255nM ModE ; lane 7 , 340 nM modE .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10206709" target="_blank">10206709</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10206709" target="_blank">10206709</a>]<br></li>
<li>Lane 1 , DNA alone ; lanes 2 -- 5 , * ModE ; lanes 6 -- 8 , * ModE ( T125I ) ; lanes 9 -- 11 , * ModE ( A76V ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10446207" target="_blank">10446207</a>]<br></li>
<li>Lane 1 , sequencing ladder ( only the C track is shown ) ; lane 2 DNA alone ( 9.3 nM ) ; lanes 3 and 6 , 1.77 pM ModE ; lanes 4 and 7 , 3.54 pM ModE ; lanes 5 and 8 , 7.08 pM ModE .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10206709" target="_blank">10206709</a>]<br></li>
<li>Lanes 1 and 2 , sequencing ladder ( T and C tracks , respectively ) ; lane 3 , DNA alone ( 5 nM ) ; lanes 4 and 8 , 0.5 pM ModE ; lanes 5 and 9 , 1.0 pM ModE ; lanes 6 and 10 , 1.5 pM ModE ; lanes 7 and 11 , 2.0 pM ModE : lane 12 , 4.0 pM ModE .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10206709" target="_blank">10206709</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10206709" target="_blank">10206709</a>]<br></li>
<li>Furthermore , many of the intergenic matches fell upstream of genes not obviously involved in molybdenum metabolism and therefore probably not subject to regulation by ModE .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14641908" target="_blank">14641908</a>]<br></li>
<li>This suggested that the full-length orthologue of ModE ( CT1543 , Q8KC82 ) functions in regulation of molybdenum transport in C. tepidum in a similar manner to ModE in E. coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14641908" target="_blank">14641908</a>]<br></li>
<li>We tried to isolate modE and hfq double mutants by P1 transduction but failed , implying that small amounts of transcripts under the control of ModE might be stable in the absence of Hfq .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23913318" target="_blank">23913318</a>]<br></li>
<li>By one model , ModE monomers interact and dimerize in the cytoplasm ; by another , the monomers bind the ModE operator independently of one another and then interact to form stable dimers .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9721306" target="_blank">9721306</a>]<br></li>
<li>This region contains a symmetric dyad , and its downstream half-site corresponds to the region in which mutations relieved repression by ModE ( Recli et al. , 1995 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9210473" target="_blank">9210473</a>]<br></li>
<li>The consensus sequence identified here is also found , with the correct inter half-site spacing , in the promoters of A. vinelan-dii inodG and R. capsulutus anfA , whose molybdenum-depen-dent repression is mediated by ModE and in R. capsulutus by the ModE-like proteins MopA and MopB ( Mouncey et a ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9210473" target="_blank">9210473</a>]<br></li>
<li>In molybdenum cofactor-de-ﬁcient strains also lacking ModE activity , anaerobiosis elevates moa expression about eightfold over the aerobic levels but only to a relatively modest level ( ; 800 Miller units ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11092866" target="_blank">11092866</a>]<br></li>
<li>In view of this homology , and the possible labile nature of a bound pterin , we puriﬁed ModE under both aerobic and anaerobic conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9044285" target="_blank">9044285</a>]<br></li>
<li>We propose that these increases are primarily a result of relieving repression from P2 as , when IHF and ModE are bound at dmsA , they would be predicted to occlude the ¹ 35 and ¹ 10 consensus sequences respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466267" target="_blank">9466267</a>]<br></li>
<li>ModE and IHF bind dmsA in the vicinity of the P1 transcript start site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466267" target="_blank">9466267</a>]<br></li>
<li>IHF binds dmsA immediately upstream of ModE The vast majority of repressors bind their cognate promo-ters in a region that spans from position ¹ 70 to þ 20 relative to the start of transcription ( Gralla and Collado-Vides , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466267" target="_blank">9466267</a>]<br></li>
<li>However , this appears unlikely because the ratio of protein to molybdenum in the puriﬁed samples of ModE , as determined by induced coupled plasma mass spectroscopy ( ICP-MS ) , was < 100:1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9044285" target="_blank">9044285</a>]<br></li>
<li>The level of molybdenum in the ModE preparations was assayed using ICP-MS by the Soil and Plant Analysis Laboratory , University of Wisconsin , Madison .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9044285" target="_blank">9044285</a>]<br></li>
<li>Expression of the N terminus of ModE in vivo resulted in weak repression of a modA-lacZ fusion , and in vitro studies conﬁrmed that a similar truncated ModE polypeptide bound the modA operator with low afﬁnity ( unpublished results ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9721306" target="_blank">9721306</a>]<br></li>
<li>This appears to be the case since fusing the dimerization domain from CI to the N terminus of ModE resulted in an 11-fold increase in modA-lacZ repression in vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9721306" target="_blank">9721306</a>]<br></li>
<li>Finally , speciﬁc deletions within the ModE portion of the CI-ModE protein chimera abolished both in vivo repression and the ability to interact with wild-type ModE .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9721306" target="_blank">9721306</a>]<br></li>
<li>Dimerization , as measured by both in vivo repressor activity and negative dominance over wild-type ModE , was also abolished by completely deleting the MopE2 domain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9721306" target="_blank">9721306</a>]<br></li>
<li>These data suggest that , in addition to ModE , other factor ( s ) require molybdate to effect nitrate-dependent repression : this factor is likely to be the NarX/NarQ sensor -- transmitter proteins ( Gunsalus , 1992 ; Stewart , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466267" target="_blank">9466267</a>]<br></li>
<li>These results suggest that the nitrate-dependent activation of the op-erons coding for anaerobic nitrate respiration also requires both ModE and MoeA proteins in addition to the nitrate-speci cents c response-regulators , NarL and its cognate sensor , NarX ( or NarQ ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9851041" target="_blank">9851041</a>]<br></li>
<li>Although molyb-date-ModE has been shown to mediate enhanced expression of dmsA under certain conditions , its mode of action is thought to be indirect , namely that of a transcriptional-repressor of a direct repressor of dmsA ( 21 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11092866" target="_blank">11092866</a>]<br></li>
<li>Both FhlA126 and FhlA132 proteins activated the hyc operon even in the absence of ModE and MoeA , two components of Mo-metabolism which are required for hyc-lac expression in wild-type E. coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10689164" target="_blank">10689164</a>]<br></li>
<li>To explore the potential role of ModE and MoeA in the regulation of other E. coli genes , the global gene expression proﬁle of a wild type and a modE , moeA double mutant grown in glucose-minimal medium under anaerobic conditions were compared .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16205910" target="_blank">16205910</a>]<br></li>
<li>A modest increase in the level of expression of hyf-lac occurred in a modE mutant compared to that of the wild-type parent , suggesting a potential repression by ModE .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14702328" target="_blank">14702328</a>]<br></li>
<li>Nitrate-dependent increase in transcription of nar-lacZ also requires both ModE and MoeA proteins The nitrate-dependent increases in expression of the three nar operons analyzed in this study , narG-lacZ ( nitrate reductase structural genes [ 9 ] ) , narKlacZ ( nitrite extrusion [ 16 ] ) and narX-lacZ ( regulatory proteins [ 9,17 ] ) were also lower in a modE mutant background as compared to the respective pa-rent strains ( Table 3 ; also [ 5 ] ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9851041" target="_blank">9851041</a>]<br></li>
<li>Although the native ModE protein remained in solution upon expression from the T7-ex-pression plasmid , the mutant ModE proteins were localized in inclusion bodies .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10446207" target="_blank">10446207</a>]<br></li>
<li>Since narG expression required the ModE protein , the effect of the modE mutation on narK-lac2 expression was investigated and the results are presented in Table 6 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10206709" target="_blank">10206709</a>]<br></li>
<li>Although the mechanism by which the MoeA-mediated control of nar-lac expression is not known , these results show that it is independent of ModE protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9851041" target="_blank">9851041</a>]<br></li>
<li>Since the Mo-dependent po-sitive control of gene expression also requires MoeA protein for optimal expression of hyc and nar , and moeA mutation had no detectable effect on the negative control of gene expression by ModE , the global gene expression proﬁle of strain SE2138 was compared to that of the double mutant , strain SE2155 , to enhance the sensitivity of the experiment .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16205910" target="_blank">16205910</a>]<br></li>
<li>Mutated FhlA proteins activate transcription of PhyclacZ in a modE , moeA double mutant The molybdate requirement for hyc expression is transduced through two proteins , ModE and MoeA and the hyc operon is not transcribed in a modE , moeA double mutant [ 12 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10689164" target="_blank">10689164</a>]<br></li>
<li>The requirement for ModE and MoeA for hyc expression is in addition to FhlA protein and formate since the modE , moeA double mutant , strain WS172 , is fhlA and produces formate internally .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9851041" target="_blank">9851041</a>]<br></li>
<li>To address the role of ModE in regulating dmsA expression further , we subjected the dmsA promoter fragment , cloned in lPM40 , to site-directed mutagenesis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466267" target="_blank">9466267</a>]<br></li>
<li>Although the functions of ModD and ModF , and therefore the need for their further regulation by ModE , are unclear , f ( modF-lacZ ) expression levels were higher by approximately fourfold when the cells were grown in low-molybdate -- glucose -- minimal medium in comparison with levels for cells grown in molybdate-supple-mented medium , indicating molybdate-induced repression of modF expression ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8550508" target="_blank">8550508</a>]<br></li>
<li>Introduction of the molybdate-independent modE * allele , which encodes a molybdate-independent variant of ModE , into a modE modC strain resulted in an unusually large ( when compared to the wild-type strain ) increase in napF-lacZ expression in response to anaerobiosis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12029041" target="_blank">12029041</a>]<br></li>
<li>In the 146 strain , which retains the ModE protein binding site , ( napF-lacZ ) expression was induced during growth with added molybdate , and this induction required the modE allele ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=13129959" target="_blank">13129959</a>]<br></li>
<li>By contrast , in the 123 strain , in which the ModE protein binding site is deleted , ( napF-lacZ ) expression was independent of both molybdate limitation and the modE allele ( data not shown ; Table 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=13129959" target="_blank">13129959</a>]<br></li>
<li>Results ModE is required for regulation of dmsA -- lacZ expression in response to oxygen availability To determine if ModE plays a role in the molybdate-depen-dent regulation of the dmsABC operon , we constructed a dmsA -- lacZ operon fusion , lPM40 ( Fig. 1 ) , and introduced it into strain PM8 ( modE : : kan ) and its wild-type progenitor , MC4100 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466267" target="_blank">9466267</a>]<br></li>
<li>ModE activates expression of moaA in vivo To further investigate how ModE regulates moaA expression in vivo we introduced a moaA -- lacZ operon fusion into a modC modE strain ( PM6 ) and looked at the effect of expressing modE þ in trans in the presence and absence of molybdate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9044285" target="_blank">9044285</a>]<br></li>
<li>The magnitude of the ModE-depen-dent molybdate activation of moa , the direct binding of ModE to the proximal moa promoter region , and the persistent low levels of moa expression in modE strains point to molybdateModE being a major direct , positive transcriptional-regulator of this operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11092866" target="_blank">11092866</a>]<br></li>
<li>The minor effect of a modE mutation on the expression of narGHJI and frdABCD establishes that the ModE - and NarQ/NarX / NarL/NarP-regulatory pathways detect and respond to molybdate independently of one another .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466267" target="_blank">9466267</a>]<br></li>
<li>However , in the absence of ModE , P2 promoter expression was elevated fourfold under all conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466267" target="_blank">9466267</a>]<br></li>
<li>As P2 also functions under anaerobic-growth conditions and is derepressed in the absence of ModE , it is likely that , under these conditions , a signiﬁcant amount of dmsA expression , measured from the full-length construct ( lPM40 ) , originates from P2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466267" target="_blank">9466267</a>]<br></li>
<li>However , anaerobic-growth of such strains with a functional ModE results in a further fourfold increase in moa expression ( Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11092866" target="_blank">11092866</a>]<br></li>
<li>However , under anaerobic conditions , the presence of ModE * only increased dmsA -- lacZ expression to a level midway between that seen in the wild-type and modE strains .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466267" target="_blank">9466267</a>]<br></li>
<li>325 RESULTS ModE is required for normal napF-lacZ expression under all growth conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12029041" target="_blank">12029041</a>]<br></li>
<li>When cells were grown aerobically , anaerobically , or anaerobically with nitrate present , the ModE recognition site mutations markedly reduced napF-lacZ expression under each condition .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12029041" target="_blank">12029041</a>]<br></li>
<li>In this study , we establish that ModE is required for optimal expression of the dmsABC operon in response to anaerobiosis and for repression in response to nitrate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466267" target="_blank">9466267</a>]<br></li>
<li>Here we show that in the absence of ModE , expression of the napF operon in response to both nitrate and nitrite is dramatically reduced .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12029041" target="_blank">12029041</a>]<br></li>
<li>Haemophilus influenxae ModE can functionally replace E. coli ModE in the repression of E. coli modA-lacZ ( A. M. Grunden , unpublished ) and thus its apparent function in the native organism is also to repress m o d A B C operon expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10206709" target="_blank">10206709</a>]<br></li>
<li>In contract , morA ( ORF2 ) - lacZ fusions directed LacZ expression in the modE mutant ( Fig. 5 ) , indicating that morA ( ORF2 ) is expressed in the absence of repression by ModE .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23913318" target="_blank">23913318</a>]<br></li>
<li>In vivo expression of a moaA -- lacZ operon fusion was stimulated twofold by ModE .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9044285" target="_blank">9044285</a>]<br></li>
<li>In contrast to modA , ModE stimulated moaA -- lacZ expression approx .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9044285" target="_blank">9044285</a>]<br></li>
<li>Binding of ModE downstream of P1 is required for regulation of dms -- lacZ expression The location of the NarL - and Fnr-binding sites at the dmsA-regulatory region suggests that they function to regulate expression from P1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466267" target="_blank">9466267</a>]<br></li>
<li>As ModE binds well downstream of P1 , the question arises as to whether binding of ModE to dmsA is required for regulation in response to anaerobiosis and nitrate addition , or whether ModE plays an independent role .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466267" target="_blank">9466267</a>]<br></li>
<li>ModE is required for nitrate-dependent repression of dmsA -- lacZ expression It has been shown previously that the addition of nitrate to anaerobically growing cells represses dmsA -- lacZ expression 10-fold , and that this repression is relieved in cells deﬁcient in molybdate uptake ( Pascal et al. , 1982 ; Iuchi and Lin , 1987 ; Cotter and Gunsalus , 1989 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466267" target="_blank">9466267</a>]<br></li>
<li>The addition of nitrate reduced anaerobic expression 10-fold , implying that nitrate-dependent repression is independent of either P2 or ModE ( data not shown ) ; this apparent contradiction with the data shown above is addressed below .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466267" target="_blank">9466267</a>]<br></li>
<li>To determine if ModE is required for nitrate-dependent repression , we monitored the effect of nitrate addition on expression from lPM40 in MC4100 and PM8 ( modE : : kan ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466267" target="_blank">9466267</a>]<br></li>
<li>NarL binding at P1 stimulates expression from P2 The ﬁnding that , in a modE strain , addition of nitrate did not reduce dmsA expression from the full-length fusion ( lPM40 ) suggested that ModE is required along with NarL to mediate nitrate-dependent repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466267" target="_blank">9466267</a>]<br></li>
<li>Assuming that ModE * binds the dmsA promoter in the absence of molybdate ( see below ) , these data suggest that the binding of ModE to its operator sequence alone is insufﬁcient to effect repression , and that molybdate is required to activate some other component of the regulatory pathway .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466267" target="_blank">9466267</a>]<br></li>
<li>In the absence of molybdate , binding of the molybdate-indepen-dent protein , ModE * , to the full-length dmsA promoter is insufﬁcient to effect nitrate-dependent repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466267" target="_blank">9466267</a>]<br></li>
<li>Furthermore , in strain PM6 ( modC modE : : kan ) , expression of modE * in the absence of molybdate restored repression , conﬁrming that ModE * is able to-bind the dmsA promoter in the absence of molybdate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466267" target="_blank">9466267</a>]<br></li>
<li>It is less clear why the absence of molybdate and/or ModE alleviates repression of P2 and why nitrate addition stimulates increased expression from P2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466267" target="_blank">9466267</a>]<br></li>
<li>However , deletion of ModE - and IHF-binding sites ( lPM43 ) had no measurable effect on the nitrate-dependent repression of P1 , implying that NarL binding is independent of ModE .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466267" target="_blank">9466267</a>]<br></li>
<li>In the absence of ModE , nitrate-dependent repression was abolished .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466267" target="_blank">9466267</a>]<br></li>
<li>This increase was dependent on ModE .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11357510" target="_blank">11357510</a>]<br></li>
<li>The metal elements ( Na , Mn , Ni , Cu , Zn , Fe , and Mo ) are directly recognized by the DNA-binding TFs ( NhaR , MntR , NikR , CueR , ZntR/Zur , Fur , and ModE , respectively ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26010043" target="_blank">26010043</a>]<br></li>
<li>In the putative promoter ( 200 bp upstream of the pitA start codon ) , no DNA-binding sites for the following transcription factors are evident : MerR , CueR , ZntR ( all MerR family ) , Zur , NikR , CusR , ZraR , CpxR and ModE .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19054109" target="_blank">19054109</a>]<br></li>
<li>At these concentrations of ModE , the DNA -- ModE complex migrated more slowly than the complex in the control lane ; migration was also progressively retarded as the amount of modE was increased .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9044285" target="_blank">9044285</a>]<br></li>
<li>The control of deo operon is complex involving two other repressors , DeoR and CytR as well as CRP-cAMP ( Valentin-Hansen et al. 1978 ; Valentin-Hansen 1982 ; Short and Singer 1984 ; Shin et al. 2001 ) , and the absence of these proteins in the assay could account for the higher than expected amount of ModE required for this DNA mobility shift .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16205910" target="_blank">16205910</a>]<br></li>
<li>ModE regulates the top four operons negatively ( indicated in parentheses ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10206709" target="_blank">10206709</a>]<br></li>
<li>Thus , binding of ModE at the two operons differs fundamentally in location , in response to ligand , and in the net effect on transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9044285" target="_blank">9044285</a>]<br></li>
<li>) indicates that the role of ModE is not clearly defined in these two operons but the presence of sequence motifs suggest that it could negatively regulate transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10206709" target="_blank">10206709</a>]<br></li>
<li>To examine the regulation of seven ModE targets , including three newly identiﬁed promoters , we next carried out the reporter assay in vivo in the presence and absence of ModE .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23913318" target="_blank">23913318</a>]<br></li>
<li>and lane 10 shows the mobility in the absence of ModE .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9210473" target="_blank">9210473</a>]<br></li>
<li>Fluorescence spectra of ModE in the absence and presence of ligands .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9210473" target="_blank">9210473</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9210473" target="_blank">9210473</a>]<br></li>
<li>Tungstate produces similar conformational changes , and the bound and unbound spectra of the His-tagged ModE protein are similar to those of the protein lacking the His tag .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9210473" target="_blank">9210473</a>]<br></li>
<li>We conclude that promoters P1 and P2 are subject to equivalent regulation by the ModE protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=13129959" target="_blank">13129959</a>]<br></li>
<li>Since the chimeric protein lacks the entire C terminus of ModE , these data imply that regulation is not dependent on interactions between the missing domain and other proteins bound at either the dmsA or the moaA promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9721306" target="_blank">9721306</a>]<br></li>
<li>The promoter fragment from PM54 no longer bound ModE ( i.e. , even when a 10-fold-higher level of ModE was used relative to the amount needed to shift the wild-type fragment ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12029041" target="_blank">12029041</a>]<br></li>
<li>The conformation of ModE ( Q216 * ) , based on intrinsic fluorescence emission characteristics , is also significantly different from that of native ModE .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10446207" target="_blank">10446207</a>]<br></li>
<li>The - ga-lactosidase activity directed by this morB promoter also increased in the modE mutant , suggesting the negative regulation of the morB promoter by ModE .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23913318" target="_blank">23913318</a>]<br></li>
<li>The observed negative effect of ModE and MoeA on opp expression was conﬁrmed by quantitative RT-PCR ( Table 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16205910" target="_blank">16205910</a>]<br></li>
<li>The inhibitory effect of cell growth by pSch9-yghW transformation might be due to the expression of this truncated morA transcript in the absence of ModE .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23913318" target="_blank">23913318</a>]<br></li>
<li>The presence or absence of functional ModE did not inﬂuence this tungstate-speciﬁc effect , which was observed only in molybdenum cofactor-sufﬁ-cient strains .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11092866" target="_blank">11092866</a>]<br></li>
<li>The absence of ModE also had a twofold effect on regulation of the narGHJI and frdABCD operons ( unpublished data ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466267" target="_blank">9466267</a>]<br></li>
<li>The ModE-CI fusion protein , encoded by pPM82 , displayed an 11-fold increase in repressor activity in vivo compared to that of the truncated ModE protein ( pPM81 [ Table 2 ] ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9721306" target="_blank">9721306</a>]<br></li>
<li>The ModE protein could bind to this region of DNA and repress transcription of the opp operon or the observed repression could represent a physiological effect of elevated expression of mod gene products on the expression of another transport system such as opp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16205910" target="_blank">16205910</a>]<br></li>
<li>The ModE protein did bind to the deo upstream DNA and shifted its electrophoretic mobility .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16205910" target="_blank">16205910</a>]<br></li>
<li>ModE interacts with deo operator DNA The observed repression of deo operon by ModE suggests that the ModE protein binds to the deo operator DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16205910" target="_blank">16205910</a>]<br></li>
<li>The ModE protein is , therefore , required for the molybdenum-dependent activation of moa .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11092866" target="_blank">11092866</a>]<br></li>
<li>The CI-ModE1 chimeric protein is negatively dominant over wild-type ModE .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9721306" target="_blank">9721306</a>]<br></li>
<li>The concentration of purified monomeric ModE protein was determined by its absorbance ( E = 22 190 1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9210473" target="_blank">9210473</a>]<br></li>
<li>The ModE protein was eluted with 0.3 M imidazole in Tris buffer .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10446207" target="_blank">10446207</a>]<br></li>
<li>The ultraviolet and visible spectrum of purified ModE was typical of that of an unconjugated protein consistent with the absence of bound chromophores .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9210473" target="_blank">9210473</a>]<br></li>
<li>The ultraviolet and fluorescence spectra of ModE show no evidence for the presence of a non-protein chromophore .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9210473" target="_blank">9210473</a>]<br></li>
<li>The lysate was loaded on a HiTrap Mono Q column ( Pharmacia ) and washed with the same buffer ; the majority of the contaminating proteins bound to the column while ModE was eluted .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9044285" target="_blank">9044285</a>]<br></li>
<li>The pattern of protection of either ModE and IHF alone , or together , when bound at dmsA are shown .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466267" target="_blank">9466267</a>]<br></li>
<li>The DNase I protection experiment also showed the presence of two sites which were hypersensitive to DNase 1 digestion when ModE was bound .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9210473" target="_blank">9210473</a>]<br></li>
<li>The pattern of protection when ModE is bound at napF in the presence of 100 M molybdate is shown .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12029041" target="_blank">12029041</a>]<br></li>
<li>Taken together , these results indicate that the amino acids in the TSARNQXXG region , as well as the C-terminal 11 amino acids , are critical for molybdate-dependent conformational TABLE II b-Galactosidase activity Amino acid change Mutation 2Mo 1Mo None D663-end Wild type 220hyb 650 ,10 ,10 580 ,10 ,10 ,10 ,10 ,10 ,10 620 540 754AAT756 to TAA 649TGCGAA654 to CTCGA 687GTGC690 to CGAG 783GTGC786 to CGAG D13 -- 27 383GTAAC387 to CCGGG N252 * C217L C230E C262E D5ILLTL9 RN ( 128,129 ) PG G Oxyanion Specificity of ModE -- The possibility that other oxyanion analogs of molybdate could functionally substitute for molybdate in promoting the association of ModE with modA operator/promoter DNA was investigated under both in vivo and in vitro conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10446207" target="_blank">10446207</a>]<br></li>
<li>To determine accurately where ModE binds , we performed DNase I footprinting .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466267" target="_blank">9466267</a>]<br></li>
<li>DNA mobility-shift assays showed that ModE requires molybdenum , or tungstate , to-bind with high affinity ( approximate K , , of 30 nM ModE ) to the modABCD promoter region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9210473" target="_blank">9210473</a>]<br></li>
<li>ModE also binds tungstate in v i m , and furthermore the tung-state-ModE complex is able to-bind to the ModE promoter region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9210473" target="_blank">9210473</a>]<br></li>
<li>ModE binds tung-state in a virtually identical manner to molybdate , and ModEtungstate can bind to the modA promoter ( 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11092866" target="_blank">11092866</a>]<br></li>
<li>Circular dichroism spectroscopy indicated that when molybdate or tungstate bind to ModE there is little change in its a-helical content , but a major change in the environment of tryptophan and tyrosine residues occurs .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9210473" target="_blank">9210473</a>]<br></li>
<li>Therefore , the ability of ModE to sense different levels of molybdate , and thereby differentially bind its target promoters , would allow the cell to co-ordinate molybdate util-ization with uptake .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9044285" target="_blank">9044285</a>]<br></li>
<li>Consider the Mop2 domain of E. coli ModE ; the residue corresponding to Mop1 Ser126 is Asn198 , which , in the observed conformation , would for steric reasons be unable to-bind molybdate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11259434" target="_blank">11259434</a>]<br></li>
<li>Replacing the first pentamer with a tetramer sequence , 5 ' GTTA 3 ' , apparently reduces the affinity of ModE to DNA , as would be needed for operons whose transcription is only enhanced by ModE-molybdate ( hyc and nar ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10206709" target="_blank">10206709</a>]<br></li>
<li>It is likely that the MoeA product is interacting with another DNA-binding protein , besides the ModE protein , which is also required for expression of the hyc operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10689164" target="_blank">10689164</a>]<br></li>
<li>ModE binds to hyc promoter DNA Since genetic and physiological experiments suggested that ModE is involved in the regulation of both P , , , EacZ and n a r X - l a d , direct binding experiments consisting of ModE protein and promoter DNA were carried out .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10206709" target="_blank">10206709</a>]<br></li>
<li>ModE , the DNA-binding protein of the pair , binds to hyc upstream DNA and enhances transcription mediated by the activator protein , FhlA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16205910" target="_blank">16205910</a>]<br></li>
<li>In both ( A and B ) , lanes 1-9 show the mobility of the labelled DNA fragment in the presence of increasing concentrations of ModE .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9210473" target="_blank">9210473</a>]<br></li>
<li>Three micrograms of plasmid pRM1 DNA and , when appropriate , 40 pmol of ModE were included in the S30 reaction mixtures .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8550508" target="_blank">8550508</a>]<br></li>
<li>Binding of ModE to the DNA under molybdate-sufﬁcient conditions relives this control .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12029041" target="_blank">12029041</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_133">
Binding of ModE to target DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8550508" target="_blank">8550508</a>]<br></label>
<input id="act_similarity_133", type="checkbox">
<div class="ACT_similar">
<li>Binding of ModE to appropriate target mod DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8550508" target="_blank">8550508</a>]<br></li></div>
<li>Figure 4 shows a model of how ModE may bind to an idealized DNA double helix .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12581638" target="_blank">12581638</a>]<br></li>
<li>Stereo view of a model of apo and ligand-bound ModE interacting with DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12581638" target="_blank">12581638</a>]<br></li>
<li>The above data demonstrate that , although the N terminus of ModE encodes all the determinants necessary for DNA recognition and binding , efﬁcient repression is achieved only when ModE is able to dimerize .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9721306" target="_blank">9721306</a>]<br></li>
<li>Taken together , for high-affinity binding of ModE to DNA , as in the case of repression , the two pentamer sequences appear to be critical .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10206709" target="_blank">10206709</a>]<br></li>
<li>Alternatively , binding of ModE and IHF may alter DNA conformation at the dmsA promoter to facilitate binding of Fnr .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466267" target="_blank">9466267</a>]<br></li>
<li>Therefore , the successful activation of ModE relies solely on the binding of an appropriately sized and shaped moiety to affect the change in the conformation of ModE that allows for efficient binding of DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10446207" target="_blank">10446207</a>]<br></li>
<li>Furthermore , addition of modA-42mer DNA ( Fig. 3 ) to the reaction mixtures containing molybdate and ModE reversed the repression of modAB9 by ModE .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8550508" target="_blank">8550508</a>]<br></li>
<li>Among the various oxyanions tested , only tungstate replaced molybdate in the repression of modA by ModE , but the affinity of ModE-tungstate for modABCD operator DNA was 6 times lower than with ModE-molybdate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10446207" target="_blank">10446207</a>]<br></li>
<li>A mutant ModE ( T125I ) protein , which repressed modA-lac even in the absence of molybdate , protected the same region of modA operator DNA in the absence of molybdate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10446207" target="_blank">10446207</a>]<br></li>
<li>However , McNicholas et al. ( 17 ) reported that ModE protected the modA operator DNA even in the absence of added molybdate , which is in contrast to the in vivo observations of strict molybdate dependence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10446207" target="_blank">10446207</a>]<br></li>
<li>However , the purified ModE was reported to-bind to modA operator DNA even in the absence of added molybdate , and it protected the DNA from hydrolysis by DNase I ( 17 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10446207" target="_blank">10446207</a>]<br></li>
<li>The apparent Kd for this interaction between * ModE and modA operator/promoter DNA ( 218 to 125 ) was calculated to be 0.3 nM .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10446207" target="_blank">10446207</a>]<br></li>
<li>In the absence of molybdate , this apparent Kd value for the interaction between ModE and modA operator/promoter DNA increased to 8 nM .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10446207" target="_blank">10446207</a>]<br></li>
<li>The apparent K for the interac-d tion between modA operator DNA and ModE ( T125I ) was 3 nM in the presence of molybdate and 4 nM without molybdate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10446207" target="_blank">10446207</a>]<br></li>
<li>The apparent Kd for the interaction between * ModE ( T125I ) and modA operator DNA was 3 nM in the presence of molybdate and 4 nM in the absence of molybdate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10446207" target="_blank">10446207</a>]<br></li>
<li>We reevaluated the kinetics of interaction between ModE and modA operator DNA , as well as the role of molybdate in this interaction , and the results are presented in this communication .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10446207" target="_blank">10446207</a>]<br></li>
<li>These results suggest that greater than 95 % of the * ModE in this preparation is free of molybdate and that the active form of ModE binding to modA operator DNA is ModEmolybdate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10446207" target="_blank">10446207</a>]<br></li>
<li>Confirmation of the binding of ModE to E. coli modA operator/promoter DNA was provided by electrophoretic mobility shift and by DNase I-footprinting of this DNA in the presence of ModE ( 15 -- 17 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10446207" target="_blank">10446207</a>]<br></li>
<li>To resolve this difference , the interaction between ModE and modA operator/promoter DNA was determined using an eva-nescent wave biosensor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10446207" target="_blank">10446207</a>]<br></li>
<li>In contrast to the modA promoter , ModE binding at moaA appeared to be unaffected by the presence of added molybdate over a range of concentrations from 0.01 nM to 10 mM ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9044285" target="_blank">9044285</a>]<br></li>
<li>Therefore , the data appear to support a model whereby binding of molybdate to ModE induces a conformational change that enhances ModE binding to the modA promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9044285" target="_blank">9044285</a>]<br></li>
<li>Although the addition of molybdate caused a dramatic increase in ModE binding to the modA promoter in vitro , we also observed signiﬁcant DNA-binding activity in the absence of molybdate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9044285" target="_blank">9044285</a>]<br></li>
<li>However , relative to modA , binding of ModE to the moaA promoter appeared to be largely molybdate independent both in vitro and in vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9044285" target="_blank">9044285</a>]<br></li>
<li>ModE binding at modA is regulated by molybdate in vitro As shown above , ModE bound the modA promoter in the absence of added molybdate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9044285" target="_blank">9044285</a>]<br></li>
<li>As the data presented in Fig. 6 indicate , the addition of ModE to both the modA-42mer and the modF-42mer in the presence of 1 mM molybdate caused a distinct shift in the migration of the oligonucleotides ( Fig. 6 , lanes 2 and 4 ) , leading to a unique band containing the DNA-protein complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8550508" target="_blank">8550508</a>]<br></li>
<li>When the mobility shift experiment was performed in the absence of molybdate ( Fig. 6 , lanes 5 to 8 ) , a lesser amount of modA-42mer was retarded by binding with ModE and there was no detectable modF-42mer -- ModE complex formed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8550508" target="_blank">8550508</a>]<br></li>
<li>In the absence of added molybdate , ModE bound to the same DNA at a lower rate , and the total response signifying the amount of protein bound was also lower .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10446207" target="_blank">10446207</a>]<br></li>
<li>When * ModE ( T125I ) mutant protein was used in these experiments , no significant difference was observed either in the rate of association or dissociation of the protein with the DNA both in the presence and absence of molybdate ( Fig. 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10446207" target="_blank">10446207</a>]<br></li>
<li>In vitro DNA mobility-shift experiments showed that molybdate is essential for the ModE protein to be bound with a high affinity to the modA promoter fragment .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9210473" target="_blank">9210473</a>]<br></li>
<li>In the presence of * ModE protein purified using the nickel affinity procedure , protection of modA operator/promoter DNA from DNase I hydrolysis was depend-ent on molybdate ( Fig. 1 , lanes 11 -- 18 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10446207" target="_blank">10446207</a>]<br></li>
<li>DNase Ifootprinting experiments revealed that the ModE protein binds the hyc promoter DNA in the presence of molybdate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10206709" target="_blank">10206709</a>]<br></li>
<li>DNase I hydrolysis of hyc promoter DNA in the presence of ModE with or without molybdate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10206709" target="_blank">10206709</a>]<br></li>
<li>Electrophoretic mobility of a 396 bp h y c promoter fragment was retarded in the presence of ModE , but only in the presence of molybdate ( Fig. 2b ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10206709" target="_blank">10206709</a>]<br></li>
<li>In the presence of molybdate , ModE protected a 30 bp region of DNA corresponding to -156 to -127 in which + 1 is the ` A ' in the ATG for narX ( Fig. 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10206709" target="_blank">10206709</a>]<br></li>
<li>It is yet unclear if ModE interactions with molybdate act primarily to modulate DNA binding or , alternatively , to affect ModE interactions with other proteins involved in napF gene expression , including Fnr , NarP , NarL , and RNAP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12029041" target="_blank">12029041</a>]<br></li>
<li>According to an alternative model , it is possible that ModE binds DNA irrespective of the presence of molybdate , and that , once bound , molybdate is required to effect repression , either by inducing dimerization or inducing a critical conformational change in ModE .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9044285" target="_blank">9044285</a>]<br></li>
<li>So far we have characterized the structural and conformational changes that ModE undergoes during activation as well as the details of oxyanion binding .10 The ﬁnal step would be the determination of the structure of ModE in complex with both molybdate and an appropriate DNA oligo-nucleotide in order to elucidate the details of DNA binding and speciﬁcity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12581638" target="_blank">12581638</a>]<br></li>
<li>ModE is a bacterial transcriptional-regulator that orchestrates many aspects of molybdenum metabolism by binding to speciﬁc DNA sequences in a molybdate-dependent fashion .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12581638" target="_blank">12581638</a>]<br></li>
<li>Size Determines Oxyanion Selectivity in ModE -- ModE functions by binding molybdate or tungstate but not other tetrahedral oxyanions such as phosphate , sulfate , or vanadate ( 9 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11259434" target="_blank">11259434</a>]<br></li>
<li>ModE is activated by binding molybdate and acts as both an activator and a repressor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23913318" target="_blank">23913318</a>]<br></li>
<li>Previous data suggested that the two MopI-like domains in full-length ModE negatively regulate ModE 's DNA binding activity in response to low molybdate availability ( 11 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9721306" target="_blank">9721306</a>]<br></li>
<li>Thus , the mobility shift experiments demonstrated binding of ModE in the presence of molybdate to the 8-bp inverted repeat ( TAAC z GTTA ) , with enhanced binding demonstrated for the DNA containing the ﬂanking CA and CATTAA sequences as well .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8550508" target="_blank">8550508</a>]<br></li>
<li>The presence of a dyad element in the DNA-binding site for ModE suggests that ModE binds as a dimer .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9210473" target="_blank">9210473</a>]<br></li>
<li>The DNA binding activity of ModE , both in vitro and in vivo , is increased in the presence of molybdate ( 1 , 12 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9721306" target="_blank">9721306</a>]<br></li>
<li>The molybdate-inde-pendent DNA binding observed at high concentrations of ModE may not be representative of physiological conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9210473" target="_blank">9210473</a>]<br></li>
<li>Given that molybdate binding enhances DNA binding3 and that our comparison of apo and activated ModE indicates structural changes at the DNA-interacting surface , it is tempting to conclude that it is these changes that alter the DNA afﬁnity of the protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12581638" target="_blank">12581638</a>]<br></li>
<li>ModE binding to DNA was used as the assay for molybdate contamination of ModE , and to increase the sensitivity , DNase I footprinting was used as the analytical tool .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10446207" target="_blank">10446207</a>]<br></li>
<li>Manual docking of an idealized DNA structure suggests that this conformational change should improve DNA binding of the activated molybdate-bound ModE .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12581638" target="_blank">12581638</a>]<br></li>
<li>Another possibility , suggested by the ﬁnding that molybdate binding to ModE induces a conformational change ( 1 ) , is that the molybdate-induced conformational change unmasks a previously occluded DNA binding domain ; experiments are in progress to test this model .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9721306" target="_blank">9721306</a>]<br></li>
<li>Relative fluorescence of ModE decreased linearly between the molybdate concentrations of 0.2 and 2.0 mM , and maximum response was observed at about 10 mM .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10446207" target="_blank">10446207</a>]<br></li>
<li>It is possible that one or more of these cysteines play a role in the conformation change of ModE in response to molybdate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10446207" target="_blank">10446207</a>]<br></li>
<li>Since it is established that ModE binds molybdate , it appear that the internal availability of molybdate , rather than any other form of molybdenum , is the signal for moa activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11092866" target="_blank">11092866</a>]<br></li>
<li>By fusing the P2 promoter region to lacZ ( lPM42 ) , we determined that P2 functions in vivo , is repressed in a molybdate-dependent manner by ModE and independently of molybdate by ModE * .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466267" target="_blank">9466267</a>]<br></li>
<li>Repression of both P1 and P2 by NarL and ModE , respectively , requires molybdate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466267" target="_blank">9466267</a>]<br></li>
<li>There is no evidence for such regulation of moa , since all the molybdate effects appear to be totally dependent upon ModE .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11092866" target="_blank">11092866</a>]<br></li>
<li>ModE * restored regulation under aerobic conditions independently of the molybdate content of the cells .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466267" target="_blank">9466267</a>]<br></li>
<li>Complementation experiments with pPM6 ( modE þ ) demonstrated that ModE required molybdate in order to restore regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466267" target="_blank">9466267</a>]<br></li>
<li>The presence of ModE as a molybdate sensor raises interesting questions about the role of this protein in regulating transcription of molybdoenzyme operons like narGHJZ .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10206709" target="_blank">10206709</a>]<br></li>
<li>The molybdate-response-regulator , ModE , has a role in transcription of P , , , lacZ and narG-lacZ Since the ModE protein is the only known molybdate-responsive DNA-binding protein in E. coli , its role in molybdate-dependent activation of transcription of hyc-lac2 and narG-lac2 was investigated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10206709" target="_blank">10206709</a>]<br></li>
<li>The synthesis of all apomolybdoenzymes in E. coli may also be regulated by the two distinct molybdate-dependent control systems : ModE , a DNA binding regulatory protein , and MoeA , an enzyme which seems to activate molybdenum .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9851041" target="_blank">9851041</a>]<br></li>
<li>By contrast , 30 nM ModE protein resulted in 50 % binding in the presence of 0.1 mM molybdate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9210473" target="_blank">9210473</a>]<br></li>
<li>The molybdate-dependent binding ModE protein was specific .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9210473" target="_blank">9210473</a>]<br></li>
<li>The binding of molybdate to ModE resulted in a decrease in fluorescence emission , indicating a conformational change of the protein upon molybdate binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10446207" target="_blank">10446207</a>]<br></li>
<li>ModE protein with or without molybdate ( 1 mM ) was added to the cuvette , and the rate of increase in refractive index ( response units in arczs ) was recorded .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10446207" target="_blank">10446207</a>]<br></li>
<li>A Kd of this magnitude is consistent with any molybdate originally bound to ModE being rapidly dissociated from the protein early in the preparation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9210473" target="_blank">9210473</a>]<br></li>
<li>It is clear from our results that molybdate binds directly to the ModE protein without a requirement for any other compound .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9210473" target="_blank">9210473</a>]<br></li>
<li>DNase I hydrolysis of the narX-coding strand in the narX-narK intergenic region , in the presence of ModE protein with or without molybdate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10206709" target="_blank">10206709</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10206709" target="_blank">10206709</a>]<br></li>
<li>ModE-molybdate binds to DNA in the narXf-narK intergenic region Sincethe ModE protein is required for nitrate-dependent expression of nar operons and also protects h y c promoter DNA from DNase I , the ability of ModE to-bind to various nar promoter sequences was determined .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10206709" target="_blank">10206709</a>]<br></li>
<li>In this work , the ModE protein was also found to be a required component inthe activation of hyc-lacZ to an optimum level , but only in the presence of molybdate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10206709" target="_blank">10206709</a>]<br></li>
<li>Even in the presence of pWS31 , which encodes native ModE protein , expression of Ph , , lacZ was completely molybdate-dependent ( Table 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10206709" target="_blank">10206709</a>]<br></li>
<li>We also note that the IscR binding site overlaps with that of ModE , which increases napF transcription in the presence of molybdate ( McNicholas and Gunsalus , 2002 ) , although it has not been established which start site ModE regulates .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16677314" target="_blank">16677314</a>]<br></li>
<li>ModE , in the presence of 1 mM molybdate , repressed the production of plasmid-encoded ModA and ModB * proteins in an in vitro transcription-translation system .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8550508" target="_blank">8550508</a>]<br></li>
<li>These mutant proteins repressed modABCD transcription even in the absence of molybdate , which implies that this stretch of amino acids is essential for the binding of molybdate by the ModE protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8550508" target="_blank">8550508</a>]<br></li>
<li>With the addition of molybdate , this repression of modAB9 by ModE became even more pronounced ( greater than 90 % decrease in the production of both ModA and ModB9 ) ( Fig. 5 , lanes 3 and 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8550508" target="_blank">8550508</a>]<br></li>
<li>The in vitro transcriptiontranslation experiments using S30 extract and L - [ 35S ] methionine to monitor the expression of modA and modB9 from plasmid pRM1 in the presence and absence of ModE and 1 mM molybdate were conducted as suggested by the supplier of the S30 extract ( Promega , Madison , Wis. ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8550508" target="_blank">8550508</a>]<br></li>
<li>The molybdate-independent mutant ModE proteins apparently mimic in its conformation the native ModE-molybdate complex , which binds to a DNA sequence motif of TATAT-7bp-TAYAT .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10446207" target="_blank">10446207</a>]<br></li>
<li>These results suggest that the molybdate-independent ModE mutant proteins resemble ModE-molybdate in its conformation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10446207" target="_blank">10446207</a>]<br></li>
<li>ModE mutant proteins that are mo-lybdate-independent for modA repression have similar intrinsic fluorescence characteristics of ModE-molybdate complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10446207" target="_blank">10446207</a>]<br></li>
<li>We have previously described mutant ModE proteins which repressed modA-lac2 expression even in the absence of molybdate ( Grunden et al. , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10206709" target="_blank">10206709</a>]<br></li>
<li>However , in vivo , ModE ( T125I ) repressed modA-lac expression in the absence of molybdate , whereas native ModE failed to repress it in the absence of molybdate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10446207" target="_blank">10446207</a>]<br></li>
<li>( 4 ) Repression of modA-lacZ by ModE only required molybdate or tungstate and was independent of moeA gene product .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9851041" target="_blank">9851041</a>]<br></li>
<li>Mutant ModE proteins which are molybdate-independent for repression of modA-lacZ also restored hyc-lacZ expression to the wild-type level even in the absence of molybdate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10206709" target="_blank">10206709</a>]<br></li>
<li>When plasmids encoding these mutant ModE proteins were introduced into strain WS192 , P , , , lacZ expression was also independent of added molybdate ( Table 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10206709" target="_blank">10206709</a>]<br></li>
<li>Except for the mod and dms operon , molybdate-dependent control of the other ope-rons also requires speciﬁc inducers such as formate , nitrate , etc. or a mutation in moa before the ModE requirement can be demonstrated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16205910" target="_blank">16205910</a>]<br></li>
<li>In the presence of molybdate , ModE repressed transcription of the modABC operon encoding the transporter of molybdate ( 2 ) and activates the synthesis of molybdoenzymes and proteins for Mo metabolism , including dmsABC encoding dimethyl sulfoxide reductase , napFDAGHBC encoding nitrate reductase , and moaABCDE coding for molybdenum cofactor synthase ( 5 -- 7 ) ( see Fig. 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23913318" target="_blank">23913318</a>]<br></li>
<li>' Nitrogen Fixation Laboratory , John Innes Centre , Norwich , UK XER ' and Richard N. PAU ' ( Received 23 December 1996/7 March 1997 ) - EJB Molybdenum-dependent repression of transcription of the Escherichia coli modABCD operon , which encodes the high-affinity molybdate transporter , is mediated by the ModE protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9210473" target="_blank">9210473</a>]<br></li>
<li>The repression is mediated by the ModE protein , which is encoded by a second operon expressed divergently from modABCD ( 7 , 20 , 25 , 38 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11092866" target="_blank">11092866</a>]<br></li>
<li>IA DISCUSSION Mutations in ModE abolish molybdate-dependent repression of modABCD transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9210473" target="_blank">9210473</a>]<br></li>
<li>The higher affinity of * ModE ( T125I ) in the absence of molybdate is in agreement with the observed in vivo results on repression of modABCD operon by the ModE mutant protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10446207" target="_blank">10446207</a>]<br></li>
<li>The ModE protein was used in an in vitro transcription-translation experiment with the objective of determining whether expression of the modABCD operon is repressed in the presence of ModE and molybdate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8550508" target="_blank">8550508</a>]<br></li>
<li>The reduction of modABCD expression in the in vitro transcription-translation reaction mixtures containing ModE shows that ModE is a repressor of the modABCD operon , and this repression is enhanced by molybdate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8550508" target="_blank">8550508</a>]<br></li>
<li>ModE also binds and regulates expression of the moaADCDE and dmsABC operons , which encode enzymes involved in the ﬁrst steps of molybdate assimilation into molybdopterin and the DMSO reductase , respectively ( 1 , 20 , 21 , 23 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12029041" target="_blank">12029041</a>]<br></li>
<li>The molybdate-responsive transcription factor , ModE , regulates the expression of a number of operons in Escherichia coli in response to changes in the intracellular levels of molybdate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9721306" target="_blank">9721306</a>]<br></li>
<li>Conclusions Transcriptional regulation of molybdo-enzymes , and genes involved in molybdenum metabolism and homeos-tasis , is performed by the molybdate-responsive transcription factor ModE in E. coli and related Proteobacteria .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14641908" target="_blank">14641908</a>]<br></li>
<li>These results show that molybdate transport in E. coli is regulated by ModE , which acts as a repressor when bound to molybdate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8550508" target="_blank">8550508</a>]<br></li>
<li>In E. coli , ModE bound to molybdate has been shown to contribute to the regulation of modABCD [ 4 ] , napFDAGHBC [ 5 ] , moaABCDE [ 6 ] , and dmsABC operons [ 7 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14641908" target="_blank">14641908</a>]<br></li>
<li>It is possible that the molybdate-bound form of ModE , the known repressor of the m o d A B C D operon , is also the active form participating in h y c and narG expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10206709" target="_blank">10206709</a>]<br></li>
<li>Transcription activation of hyc-lacZ requires both ModE and MoeA ModE-molybdate , a repressor of modABCD operon transcription was previously shown to activate hyc-lacZ [ 5 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9851041" target="_blank">9851041</a>]<br></li>
<li>ModE also binds to the rnoa operon upstream DNA but a need for molybdate in the control of moa-lac2 could not be demonstrated ( Baker & Boxer , 1991 ; McNicholas et al. , 1997 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10206709" target="_blank">10206709</a>]<br></li>
<li>Expression of the moa operon may respond to molybdate levels in the cell , since ModE was recently shown to-bind to the moa operon upstream region ( McNicholas et al. 1997 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11357510" target="_blank">11357510</a>]<br></li>
<li>Results Mo-sensitive differential expression proﬁle The molybdate sensor protein , ModE , in association with molybdate , binds to unique DNA sequences and regulates the transcription of mod , nar , hyc , fdhF , moa , nap and dms operons in E. coli ( Rosentel et al. 1995 ; Grunden et al. 1996 ; McNicholas et al. 1996 ; McNich-olas et al. 1997 ; Anderson et al. 2000 ; Self et al. 2001a ; McNicholas and Gunsalus 2002 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16205910" target="_blank">16205910</a>]<br></li>
<li>ModE protein repressed the modABC operon under conditions of Mo-excess ( Grunden et al. 1996 ; McNicholas et al. 1996 ) and the interaction between the mod operator DNA and ModE has been established at the biochemical level ( Grunden et al. 1999 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16205910" target="_blank">16205910</a>]<br></li>
<li>Fluorescence Characteristics of ModE Mutant Proteins -- Anderson et al. ( 15 ) reported that the intrinsic fluorescence of ModE protein decreased upon binding molybdate and this could be a result of conformational change of the protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10446207" target="_blank">10446207</a>]<br></li>
<li>The Kdfor molybdate binding to ModE is of the same order as the concentration of molybdate in the medium ( 0.1-0 .5 pM ) required to produce 50 % repression of transcription of the mo-lybdenum transport operons in E. coli and A. viizelandii ( Miller et al. , 1987 ; Mouncey et al. , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9210473" target="_blank">9210473</a>]<br></li>
<li>This allele has a nonsense mutation in the distal portion of the gene and , as a consequence , its product , ModE * , does not require molybdate to effect repression of the modABCD operon ( McNicholas et al. , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466267" target="_blank">9466267</a>]<br></li>
<li>The molybdate-dependent repression of the modABCD operon is mediated through ModE ( Grunden , 1996 ; Grunden et al. , 1996 ; McNicholas et al. , 1996 ; Walkenhorst et al. , 1995 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10206709" target="_blank">10206709</a>]<br></li>
<li>Consistent with in vivo studies ( Rech et al. , 1995 ; Grunden et al. , 1996 ; McNicholas et al. , 1996 ) we found that , in vitro , puriﬁed ModE binds the modA promoter with high afﬁnity and that binding was strongly enhanced by the addition of molybdate over a range of 5 -- 25 nM .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9044285" target="_blank">9044285</a>]<br></li>
<li>This ﬁnding is consistent with the observation that molybdate increased ModE 's afﬁnity for its binding site ( Anderson et al. , 1997 ; McNicholas et al. , 1997 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466267" target="_blank">9466267</a>]<br></li>
<li>The minimal concentration of ModE-molybdate required for the mobility shift was about 250 n M and this was about a 10-fold higher concentration of ModE than was required for binding to the modA operator ( Anderson et al. , 1997 ; McNicholas et ai .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10206709" target="_blank">10206709</a>]<br></li>
<li>Again , this seems unlikely because deleting part , or all , of the putative molybdate-binding domains , which comprise the C-terminal half of ModE , results in constitutive repression of modA -- lacZ expression rather than abolishing repression ( McNicholas et al. , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9044285" target="_blank">9044285</a>]<br></li>
<li>Molybdate-independent Mutant ModE -- Grunden et al. ( 13 ) previously described several ModE mutants that are either inactive ( A76V ) or molybdate-independent ( T125I and Q216 * ) in modA repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10446207" target="_blank">10446207</a>]<br></li>
<li>In Azotobacter vinelandii , the ModE protein is likewise involved in repression of the operon encoding genes for the high-affinity mo-lybdate transporter ( Mouncey et al. , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9210473" target="_blank">9210473</a>]<br></li>
<li>ModE protein is requiredfor expression of narXf operon The narXL operon encodes the two regulatory proteins needed for activation of the narG and narK operons ( Darwin & Stewart , 1996 ; Egan & Stewart , 1990 ; Kalman & Gunsalus , 1990 ; Schroder et al. , 1994 ; Walker & DeMoss , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10206709" target="_blank">10206709</a>]<br></li>
<li>These studies show that the mechanism of control of hyc by ModE and MoeA proteins is different and the observed effect is independent and additive ( Self et al. 2001b ) Besides the nar , fdhF and hyc , the dms operon encoding dimethylsulfoxide reductase and the nap operon encoding the components of periplasmic nitrate reductase are also regulated by the availability of Mo ( McNicholas et al. 1998 ; McNicholas and Gunsalus 2002 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16205910" target="_blank">16205910</a>]<br></li>
<li>With regard to the role that ModE plays in the regulation of P1 of the dmsABC operon , and narGHJI and frdABCD operons , it is perhaps signiﬁcant that the synthesis of NarQ and NarX is itself mildly regulated ( two - to threefold ) in response to molybdate availability ( Darwin and Stewart , 1995 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466267" target="_blank">9466267</a>]<br></li>
<li>Although ModE apparently plays a role in the regulation of these two operons , the mechanism of this regulation is not known and these studies are also complicated by the presence of two promoters in the dms operon ( McNicholaset al. , 1998 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10206709" target="_blank">10206709</a>]<br></li>
<li>Although ModE plays a role in the control of the dmsABC operon , the ModE protein appears to repress transcription from a downstream promoter , P2 ( McNicholas et al. , 1998 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10206709" target="_blank">10206709</a>]<br></li>
<li>In this paper , we present evidence that the ModE protein is a secondary transcriptional-activator for the hyc and n a r X L operons which are primarily activated by the transcriptional-activators , FhlA and NarL-I ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10206709" target="_blank">10206709</a>]<br></li>
<li>It is proposed that the MoeA-catalyzed product , an activated form of molybdate , interacts with a transcriptional-activator/regulator other than ModE and regulates hyc and nar operons .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9851041" target="_blank">9851041</a>]<br></li>
<li>Since ModE is a transcriptional-activator of hyc operon , it is possible that the MoeA-catalyzed product interacts with ModE and that the ModE - ( MoeA product ) is the true active form .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9851041" target="_blank">9851041</a>]<br></li>
<li>These results show that certain mutations in FhlA can replace the need for both ModE and MoeA in the activation of transcription of the hyc operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10689164" target="_blank">10689164</a>]<br></li>
<li>Keywords : Molybdate ; Regulation ; ModE ; MoeA ; FhlA ; Formate hydrogenlyase 1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10689164" target="_blank">10689164</a>]<br></li>
<li>Keywords : Molybdate ; Regulation ; ModE ; MoeA ; Formate hydrogenlyase ; nar operon ; Escherichia coli 1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9851041" target="_blank">9851041</a>]<br></li>
<li>It is also possible that aegA-lacZ is directly regulated by a mo-lybdate-dependent regulator , such as ModE ( 22 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955321" target="_blank">8955321</a>]<br></li>
<li>Control of deo operon is ModE - and MoeA-dependent In order to further evaluate the regulation of the deo operon by ModE and MoeA , as well as the effect of individual mutations , the level of speciﬁc mRNA in the cell for the two operons was determined using quantitative RT-PCR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16205910" target="_blank">16205910</a>]<br></li>
<li>In this study , modA was selected as the control gene in the evaluation of putative control of deo by ModE .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16205910" target="_blank">16205910</a>]<br></li>
<li>It seems highly likely that tungstate repression of modA is mediated by ModE in E. coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9210473" target="_blank">9210473</a>]<br></li>
<li>Is the repression of deo in E. coli by ModE and MoeA analogous to such a physiological role ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16205910" target="_blank">16205910</a>]<br></li>
<li>Tao , H. , Hasona , A. , Do , P. , Ingram , L. , and Shanmugan , K. ( 2005 ) Global gene expression analysis revealed an unsuspected deo operon under the control of molydate sensor , ModE protein , in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19007420" target="_blank">19007420</a>]<br></li>
<li>DISCUSSION It is now well established that the mod operon transcription is repressed by ModE , which is encoded by the first gene in the divergent modEF operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10206709" target="_blank">10206709</a>]<br></li>
<li>It is possible that the ModE binds to this region and alters the regulation of the deo operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16205910" target="_blank">16205910</a>]<br></li>
<li>Therefore it is not surprising that the yecKbisZ operon might be regulated by ModE .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14641908" target="_blank">14641908</a>]<br></li>
<li>This core network was extended to include local regulators ( CpxR , CusR , FhlA , Fur , IscR , ModE , NarL , NrdR and PdhR ) that exhibited responses in the stable environments o steady-state cultures across the aerobiosis scale and in the unstable environments of the transitions ( figure 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22870390" target="_blank">22870390</a>]<br></li>
<li>of additional probable sites in predictionsd 3 3 7e 7 -- 4 22e 0 1 0 2 -- -- 5 0 1 -- 13 7 18 1 0 6e 1 1 1 3 Ada AraC ArcA ArgR CarP CpxR Crp CspA CynR CysB CytR DeoR DnaA FadR FhlA Fis FlhCD Fnr FruR Fur GalR GcvA GlpR IciA ( ArgP ) IclR Ihf IlvY LacI LexA Lrp MalT MarR MelR MetJ MetR Mlc ( DgsA ) ModE NagC NarL NarP NtrC OmpR OxyR PdhR PhoB PurR RhaR RhaS SoxR SoxS TorR TrpR TyrR Stem -- loopsf 6 2 0 5 2 9 1 3 4 5e 5 -- 5 5 4 6 3 4 2 -- 1 3 5 1 3 -- aThe number of genes in our study set with documented binding sites for each TF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11160901" target="_blank">11160901</a>]<br></li>
<li>aerobic to anaerobic anaerobic to aerobic ArgR CysB BirA DgsA MalT GadW MetJ NikR OxyR SoxR SoxS DcuR CueR FadR HcaR NtrC PhoB PurR SgrR TdcA TrpR ZraR ArcA CpxR CRP CusR FhlA Fis FNR Fur H-NS IHF IscR Lrp ModE NarL NrdR PdhR FlhDC GlcC EvgA GadX LeuO PspF RbsR steady states Figure 3 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22870390" target="_blank">22870390</a>]<br></li>
				</div></ul><br>
			<label class="collapse_DOM" for="_2"><font size="5px">ModE protein domains</font></label>
				<input id="_2" type="checkbox">
				<div class="DOM"><br><ul><li>The structure of the DiMop-WO22 4 complex was solved by molecular replacement with the program AMORE ( 25 ) using a polyalanine search model constructed from a monomer of the DiMop domain from apoModE ( 16 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11259434" target="_blank">11259434</a>]<br></li>
<li>A putative helix-turn-helix DNA-binding region in EcModE is underlined .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8550508" target="_blank">8550508</a>]<br></li>
<li>This motif , located at the start of MopE1 ( MopE2 carries a much poorer match ) , is strongly conserved among the various ModE homologs ( Fig. 1 ) and has been identiﬁed in several molybdoenzymes ( 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9721306" target="_blank">9721306</a>]<br></li>
<li>Hydroxylamine , which causes G-to-A transitions , was used to mutate the DNA coding for ModE ( plasmid pAG1 ) in order to identify the critical amino acids responsible for the interaction-with-DNA and/or molyb-a Cells were grown in LBG medium with or without sodium molybdate at a ﬁnal concentration of 1 mM .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8550508" target="_blank">8550508</a>]<br></li>
<li>A further difference between apo and ligandbound structure is that helix a4 of chain B is kinked at residue 97 in the apo structure , while this kink is not seen in liganded ModE ( asterisk in Figure 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12581638" target="_blank">12581638</a>]<br></li>
<li>Given that the SARNQ region is conserved among four of the ﬁve homologs and partially conserved in the ﬁfth and that two of the superrepressor mu tations are situated near the SARNQ region , it is reasonable to conclude that these amino acids are important for the proper functioning of ModE .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8550508" target="_blank">8550508</a>]<br></li>
<li>The partially derepressed ModE mutant ( A76V ) contains a replacement of alanine at amino acid position 76 with valine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8550508" target="_blank">8550508</a>]<br></li>
<li>In the partially derepressed ( A76V ) ModE mutant , the mutation lies immediately after a stretch of three glycines which are identical in three of the four homologs , the fourth having two of the three glycines ( Fig. 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8550508" target="_blank">8550508</a>]<br></li>
<li>Implications for Other Mop Proteins -- Mop domains occur in a variety of proteins other than ModE ; the molbindins consist of either one ( e.g. Mop from Hemeophilus influenzae or Mop I -- III from C. pasteurianum ) or two ( e.g. ModG from A. vinelan-dii ) Mop domains .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11259434" target="_blank">11259434</a>]<br></li>
<li>According to the model proposed by Hall et al. [ 3 ] based on the threedimensional crystal structure of ModE , nine amino acid residues in the HTH directly interact with the target DNA and have a role in sequence-recognition : Ser35 , Gln36 , Lys39 , Ser44 , Tyr45 , Lys46 , Ser47 , Trp49 , and Asp50 ( numbering as in [ 3 ] ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14641908" target="_blank">14641908</a>]<br></li>
<li>Other ModE Mutant Proteins -- The C-terminal 47 amino ac-ids ( 217 -- 262 ) , which contain all three cysteines in the protein ( positions 217 , 230 , and 262 ) ( Fig. 7 ) are apparently essential for normal function of ModE , because ModE ( Q216 * ) , which lacks the amino acids from 216 to the C terminus , is molybdate-independent for repression of modABCD operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10446207" target="_blank">10446207</a>]<br></li>
<li>In ModE ( 220hyb ) , the cysteines at positions 230 and 262 were removed , and in ModE ( N252 * ) , only the C-terminal cysteine was removed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10446207" target="_blank">10446207</a>]<br></li>
<li>ModE proteins purified by this nickel affinity method , contained an extra three amino acids ( Gly , Ser , and His ) in the N termini and are designated * ModE to denote this alteration .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10446207" target="_blank">10446207</a>]<br></li>
<li>This plasmid produced a ModE C-terminal hybrid protein of 227 amino acids in which ModE was 220 amino acids long .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10446207" target="_blank">10446207</a>]<br></li>
<li>Lane 1 , DNA alone ; lanes 2 -- 9 , native ModE protein ; lanes 11 -- 18 , * ModE ( contains three extra amino acids , Gly , Ser , and His in the N-terminal end ; see under `` Experimental Procedures '' for details ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10446207" target="_blank">10446207</a>]<br></li>
<li>We next addressed the role that the C terminus of ModE plays in mediating dimerization .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9721306" target="_blank">9721306</a>]<br></li>
<li>We also addressed the role of the SARNQ motif in mediating ModE dimerization .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9721306" target="_blank">9721306</a>]<br></li>
<li>MATERIALS AND METHODS Expression of the C-terminal DiMop Domain of ModE -- A fragment of the modE gene coding for the C-terminal domain was amplified by polymerase chain reaction using pHW121 as the template plasmid , which carries a fragment covering the modE region in the vector pUC18 ( 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11259434" target="_blank">11259434</a>]<br></li>
<li>The modE gene was ampliﬁed with primers 683 ( 59-GGGCATATG [ CAT ] 6CAGGCCGAAATCCTTCTCA-39 ) and 723 ( 59-ATGGTACCGAACCACTGGTTACGGGCGCTGGTCTGCAGT-39 ) to introduce six histidine residues at the N terminus of ModE and a unique PstI site midway ( the PstI site corresponded to amino acid residues 123 and 124 and required one conservative mutation [ Fig. 1 ] ) between the sequences encoding the N - and C-terminal domains .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9721306" target="_blank">9721306</a>]<br></li>
<li>of conserved residues in the ModE recognition sequence centered at 133.5 relative to the napF promoter ( Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12029041" target="_blank">12029041</a>]<br></li>
<li>To determine the stoichiometry of the ModE-DNA association , ModE was bound to a 43-bp modA operator/promoter DNA that contains all three ModE-protected regions in a standard DNA mobility shift binding reaction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10446207" target="_blank">10446207</a>]<br></li>
<li>Speciﬁc interaction of ModE with the dmsA promoter region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466267" target="_blank">9466267</a>]<br></li>
<li>Speciﬁc interaction of ModE with the modA and moaA promoter regions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9044285" target="_blank">9044285</a>]<br></li>
<li>Consistent with these ﬁndings , a search of the BLOCKS database indicated that the ﬁrst 54 residues of ModE have a distant homology with the same region of the LysR-family of transcriptional regulators ( 1 ) , and inspection of this sequence revealed the presence of a weak helix-turn-helix motif ( Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9721306" target="_blank">9721306</a>]<br></li>
<li>Other experiments revealed that the 28271 Da ModE protein binds to DNA as a dimer through the amino acid sequence in the Nterminal part of the protein , which contains a helix-turn-helix motif .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10206709" target="_blank">10206709</a>]<br></li>
<li>The various ModE domains , N-terminal , MopE1 , and MopE2 domains , are boxed , and the putative helix-turn-helix motif is indicated by a bracket below the sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9721306" target="_blank">9721306</a>]<br></li>
<li>A possible helix-turn-helix motif which is a known prokaryotic DNA-binding protein structure motif ( 31 ) has been identiﬁed for the E. coli ModE , as indicated in Fig. 4 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8550508" target="_blank">8550508</a>]<br></li>
<li>In this study , we employed fusions to the l CI repressor protein to determine the individual roles of the N - and Cterminal domains of ModE .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9721306" target="_blank">9721306</a>]<br></li>
<li>A second fusion protein , CI-ModE2 , which had the SARNQ motif deleted , was unable to mediate repression in vivo or to interact with wild-type ModE .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9721306" target="_blank">9721306</a>]<br></li>
<li>Since the chimeric protein lacks the entire C terminus of ModE , these data imply that regulation is not dependent on interactions between the missing domain and other proteins bound at either the dmsA or the moaA promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9721306" target="_blank">9721306</a>]<br></li>
<li>Following thrombin cleavage the DiMop domain consisted of residues 124 -- 262 of ModE with a Gly-Ser-His extension at the N terminus .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11259434" target="_blank">11259434</a>]<br></li>
<li>To study the pro-tein-oxyanion interactions relevant to the function of ModE we cloned and characterized the C-terminal ( DiMop ) domain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11259434" target="_blank">11259434</a>]<br></li>
<li>We have previously shown that each subdomain corresponds to the structure of a single Mop unit and termed the C-terminal domain of ModE the DiMop domain ( 16 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11259434" target="_blank">11259434</a>]<br></li>
<li>We previously suggested a role for the C-terminal domain of ModE in the recruitment of partner proteins to form the complexes necessary for the regulation of transcription ( 16 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11259434" target="_blank">11259434</a>]<br></li>
<li>C , the dimeric DiMop arrangement in ModE ; and D , a postulated organization for the Mop domain of ModC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11259434" target="_blank">11259434</a>]<br></li>
<li>Molybdate is presumed to bind to the C-terminal domain of ModE because of its sequence similarity with Mop proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9210473" target="_blank">9210473</a>]<br></li>
<li>Consistent with this , ModE and the other proteins described above , the HTH domain occurs at the N terminus .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14641908" target="_blank">14641908</a>]<br></li>
<li>Dimerization , as measured by both in vivo repressor activity and negative dominance over wild-type ModE , was also abolished by completely deleting the MopE2 domain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9721306" target="_blank">9721306</a>]<br></li>
<li>Since mutations in FhlA suppressed the requirement for MoeA and not ModE in the expression of hyc-lacZ , it has been suggested that the MoeA product interacts with the unique N-terminal domain of FhlA protein ( Self and Shanmugam 2000 ; Self et al. 2001b ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16205910" target="_blank">16205910</a>]<br></li>
<li>This appears to be the case since fusing the dimerization domain from CI to the N terminus of ModE resulted in an 11-fold increase in modA-lacZ repression in vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9721306" target="_blank">9721306</a>]<br></li>
<li>Previously , we observed that expression of the N-terminal domain of ModE resulted in a twofold decrease in modA-lacZ expression in vivo ( 11 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9721306" target="_blank">9721306</a>]<br></li>
<li>One explanation for the impaired in vivo repression is that ModE binds the modA operator as a dimer and that removal of the C-terminal domain abolished dimerization .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9721306" target="_blank">9721306</a>]<br></li>
<li>In the reciprocal experiment , we restored repressor activity to a truncated CI protein by addition of the C-terminal domain of ModE , which is comprised of two MopI-like subdomains .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9721306" target="_blank">9721306</a>]<br></li>
<li>Mutational analysis suggests that the C-terminal domain of the ModE protein is involved in repressor activity ( McNicholas et al. , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9210473" target="_blank">9210473</a>]<br></li>
<li>Fusing a protein dimerization domain , encoded by the C terminus of l CI repressor protein , to the truncated ModE protein generated a ModE-CI fusion protein that not only displayed a greatly increased in vivo repressor activity but could also substitute for ModE at the moaA and dmsA promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9721306" target="_blank">9721306</a>]<br></li>
<li>By constructing the reciprocal protein chimera , we show that the C-terminal domain of ModE is able to substitute for the dimerization domain of CI , implying that it plays a similar role in the native protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9721306" target="_blank">9721306</a>]<br></li>
<li>To test this hypothesis , we fused the C terminus of l CI ( residues 117 to 237 ) , which encodes a well-character-ized protein dimerization domain , to the N-terminal domain of ModE ( residues 1 to 124 [ Fig. 1 and 2 ] ) and expressed the resultant protein in vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9721306" target="_blank">9721306</a>]<br></li>
<li>The residues of ModE directly involved in oxyanion binding are marked with triangles , shaded to indicate the use of a side chain functional group and open where a main chain amide is used for binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11259434" target="_blank">11259434</a>]<br></li>
<li>While it is not entirely clear what causes the asymmetry between the two identical polypeptide chains in the apo-structure , one important point might be that the crooked apoprotein is more compact than the ligand-bound form : on ligand binding the surface area of a ModE dimer increases from 23,100 Å2 to 24,100 Å2 despite the decrease in accessible surface area in the molybdate-binding domain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12581638" target="_blank">12581638</a>]<br></li>
<li>A single tandem mop repeat , which is also known as a diMop domain , forms two Greek-key b-barrels , which are structu-rally similar to the oligonucleotide/oligosaccharide binding fold .15 ModE binds two molybdate mole-cules per dimer with high afﬁnity ( < 0.8 mM ) ,3 in binding sites located at the interface between the two chains .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12581638" target="_blank">12581638</a>]<br></li>
<li>Molybdate binding results in extensive conformational changes within this domain .10 ModE requires molybdate or tungstate to bind to the modABCD promoter region with high afﬁnity ( Kd < 30 mM ModE ) .3 The clear delineation of the two functional domains raises the question of how oxyanion binding in one can inﬂuence the biological function carried out by the other .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12581638" target="_blank">12581638</a>]<br></li>
<li>Thus , this region contains a ModE binding site typical of others on the chromosome ( 22 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12029041" target="_blank">12029041</a>]<br></li>
<li>Until now no type 1 site-like ligand bindin has been observed for ModE , most likely because only two thirds of the residues required for binding are present .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12581638" target="_blank">12581638</a>]<br></li>
<li>The highly charged anions like VO32 are disadvantaged because ModE has only a single positively charged residue in the binding site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11259434" target="_blank">11259434</a>]<br></li>
<li>The ﬁnding that CI-ModE1 and ModE are able to form stable heterodimers , despite the fact that they recognize and interact with totally different DNA-binding sites , strongly suggests that dimerization takes in the cytoplasm .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9721306" target="_blank">9721306</a>]<br></li>
<li>Taken together , these data show that the molybdate binding domain ( motif ) contains the amino acid sequence SARNQ and in ModE this domain is extended to include additional amino acids [ ( T / S ) SARNQXXG ) ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8550508" target="_blank">8550508</a>]<br></li>
<li>Results and Discussion Overall structure and molybdate - binding domain The present crystal structure contains two complete ModE dimers in the asymmetric unit .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12581638" target="_blank">12581638</a>]<br></li>
				<label class="DOM_collapse_similarity" for="dom_similarity_58">
Oxyanion binding alters conformation and quaternary structure of the C-terminal domain of th transcriptional regulator ModE .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12581638" target="_blank">12581638</a>]<br></label>
<input id="dom_similarity_58", type="checkbox">
<div class="DOM_similar">
<li>Oxyanion binding alter conformation and quaternary structure of the C-terminal domain of the transcriptional regulator ModE .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12029041" target="_blank">12029041</a>]<br></li></div>
<li>Moreover , it suggests that the third tryptophan ( Trp49 ) of ModE , located in the DNA-binding domain , makes a less important contribution to the molybdate-induced fluorescence quench of ModE .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11259434" target="_blank">11259434</a>]<br></li>
<li>Several other Bacteria and Archaea encode proteins that contain the HTH domain characteristic of ModE , but completely lack a molybdate-binding TOBE-domain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14641908" target="_blank">14641908</a>]<br></li>
<li>( b ) View of the ModE ligand-binding domain ( green and blue ) and the linker region ( yellow ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12581638" target="_blank">12581638</a>]<br></li>
<li>Alignment of the ModE molybdate-binding domain in its apo form ( blue ) and oxyanion-bound form ( green ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11259434" target="_blank">11259434</a>]<br></li>
<li>These data further substantiate the in vivo observation that the CATAA motif is required for speciﬁc and efﬁcient binding of ModE at the modA promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9044285" target="_blank">9044285</a>]<br></li>
<li>These data demonstrate that in vitro ModE binds the moa promoter in the vicinity of the putative RNA polymerase-binding domain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9044285" target="_blank">9044285</a>]<br></li>
<li>Fusing this domain to the DNA-binding domain of CI generated a chimera , CI-ModE1 , that both functioned as an efﬁcient repressor in vivo and retained the ability to interact with wild-type ModE .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9721306" target="_blank">9721306</a>]<br></li>
<li>To determine if the C-terminal domain of ModE is able to mediate dimerization , we fused this domain to the DNA-binding domain of CI ( residues 1 to 115 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9721306" target="_blank">9721306</a>]<br></li>
<li>Together , these data demonstrate that the N-terminal domain of ModE is sufﬁcient to mediate DNA-binding , although efﬁcient binding requires that ModE form a dimer , a function that is supplied by the C-terminal MopI-like subdomains .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9721306" target="_blank">9721306</a>]<br></li>
<li>These ﬁndings suggest that the C-terminal domain of ModE may both negatively regulate DNA-binding and be required for efﬁcient binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9721306" target="_blank">9721306</a>]<br></li>
<li>These data also suggest that the N terminus of ModE may be sufﬁcient to mediate DNA-binding , albeit weakly .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9721306" target="_blank">9721306</a>]<br></li>
<li>The above data demonstrate that , although the N terminus of ModE encodes all the determinants necessary for DNA-recognition and binding , efﬁcient repression is achieved only when ModE is able to dimerize .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9721306" target="_blank">9721306</a>]<br></li>
<li>Whether this differential binding is achieved by molybdate binding to either one or both of the putative molybdopterin-binding domains located in the C-terminus of ModE remains to be established .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9044285" target="_blank">9044285</a>]<br></li>
<li>The possibility that the MopI-like domains in ModE from E. coli are involved in molybdate sensing and regulation of DNA-binding was suggested by the ﬁnding that either partially or completely deleting the last domain ( MopE2 [ Fig. 1 ] ) abolished the requirement for molybdate in effecting repression of modA-lacZ expression in vivo ( 3 , 11 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9721306" target="_blank">9721306</a>]<br></li>
<li>Previous data suggested that the two MopI-like domains in full-length ModE negatively regulate ModE 's DNA-binding activity in response to low molybdate availability ( 11 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9721306" target="_blank">9721306</a>]<br></li>
<li>Conservation of DNA-recognition elements in ModE homologues The HTH domain of ModE is responsible for its sequence-specific DNA-binding activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14641908" target="_blank">14641908</a>]<br></li>
<li>Here we investigated the roles of the N - and C-terminal domains of ModE in mediating DNA-binding and protein dimerization , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9721306" target="_blank">9721306</a>]<br></li>
<li>The ModE polypeptide can be subdivided into four structural domains , from N to C terminus : the N-terminal domain involved in DNA-binding , a linker region and then two so-called mop domains ( referred to as mop1 ( N-terminal ) and mop2 ( C-terminal , Figure 1 ( a ) ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12581638" target="_blank">12581638</a>]<br></li>
<li>The ModE protein functions as a homodimer and consists of two domains , the N-terminal DNA-binding domain containing a winged-helix-turn-helix motif and the C-terminal molybdate-binding domain ( 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23913318" target="_blank">23913318</a>]<br></li>
<li>RESULTS Efﬁcient DNA-binding requires that ModE form a dimer .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9721306" target="_blank">9721306</a>]<br></li>
<li>One future question is the role that molybdate plays in regulating DNA-binding activity of ModE .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9721306" target="_blank">9721306</a>]<br></li>
<li>ModE binding to DNA On a smaller scale than the changes described so far are the relative movement of the two N-termi-nal domains of the dimer with respect to each other .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12581638" target="_blank">12581638</a>]<br></li>
<li>Hall and colleagues recently reported the crystallographic structure of ModE , which appears to possess distinct DNA-and molybdate-binding domains ( 9 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11092866" target="_blank">11092866</a>]<br></li>
<li>Approximate locations of the DNA-binding domains in both ModE and CI are shown .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9721306" target="_blank">9721306</a>]<br></li>
<li>Crystal structure of activated ModE reveals conformational changes involving both oxyanion and DNA-binding domains .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23913318" target="_blank">23913318</a>]<br></li>
<li>The N termini of the ModE proteins produced from these plasmids also contain six histidines , which are separated from the ModE protein by a linker containing thrombin cleavage site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10446207" target="_blank">10446207</a>]<br></li>
<li>The complete genome sequence of H. hepaticus does not encode a full-length ModE homologue ; it does encode a 135 amino acid protein ( HH0653 ) , consisting of a single HTH domain , which is therefore probably able to specifically bind to the identified sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14641908" target="_blank">14641908</a>]<br></li>
<li>A ModE-binding site is found upstream of modA but A. tumefaciens has no full-length ModE homologue , only a 131 amino acid protein ( ATU2654 , Q8UCD4 ) consisting of a HTH domain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14641908" target="_blank">14641908</a>]<br></li>
<li>The C terminus of ModE mediates dimerization .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9721306" target="_blank">9721306</a>]<br></li>
<li>The above data imply that the ModE Cterminal domain is sufﬁcient to mediate dimerization .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9721306" target="_blank">9721306</a>]<br></li>
<li>Taken together , these data argue that the C-terminal domain of ModE alone is sufﬁcient to promote dimerization and that dimerization requires that both MopIlike domains be intact .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9721306" target="_blank">9721306</a>]<br></li>
<li>These data , taken together with the previous observations , strongly suggest that efﬁcient dimerization of ModE requires two intact MopIlike domains ( Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9721306" target="_blank">9721306</a>]<br></li>
<li>The DiMop fragment retains two of the three tryptophan residues of ModE ( Trp131 , Trp186 ) , and the intrinsic protein fluorescence emission spectrum is similar to that reported for intact ModE with lmax at 347 nm .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11259434" target="_blank">11259434</a>]<br></li>
<li>The highly symmetric assembly creates two additional binding sites located on a 3-fold axis and formed by residues from three neighboring Mop domains ( A/C/E and B/D/F ) and therefore do not occur in ModE ( Fig. 7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11259434" target="_blank">11259434</a>]<br></li>
<li>The crystal structure of apo-ModE revealed that the protein forms distinct N - and C-terminal domains ( Fig. 1 , Ref .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11259434" target="_blank">11259434</a>]<br></li>
<li>The final two sequences are the Mop2 domains of E. coli and H. influenzae ModE , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11259434" target="_blank">11259434</a>]<br></li>
<li>The ModE protein has two distinct domains .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9210473" target="_blank">9210473</a>]<br></li>
<li>The ModE protein from E. coli consists of two domains [ 3 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14641908" target="_blank">14641908</a>]<br></li>
<li>The fusion protein consists of the full-length ModE with the N-terminal extension MetGlySerSerHis , SerSerGlyLeuValProArgGlySerHis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9210473" target="_blank">9210473</a>]<br></li>
<li>The C-terminal domain of ModE is comprised of two conserved tandem repeats , designated MopE1 and MopE2 ( Fig. 1 ) , each of which is similar to a small molybdopterin binding protein , MopI , from Clostridium pasteurianum ( 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9721306" target="_blank">9721306</a>]<br></li>
<li>The rel - evance of the structures is indicated by fluorescence measurements , which show that the oxyanion binding properties of the C-terminal domain of ModE are similar to those of the full-length protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11259434" target="_blank">11259434</a>]<br></li>
<li>The Overall Structure of the Ligand-bound DiMop Domain and Comparisons with the Apo Form -- Like intact ModE , the ligand-bound DiMop forms a homodimer with a secondary structure similar to the apoprotein ( Figs. 1 and 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11259434" target="_blank">11259434</a>]<br></li>
<li>We present the crystal structure of Escherichia coli ModE in complex with molybdate , which was determined at 2.75 Å from a merohedrally twinned crystal ( twin fraction < 0.30 ) with space group P43 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12581638" target="_blank">12581638</a>]<br></li>
				<label class="DOM_collapse_similarity" for="dom_similarity_103">
The high-resolution crystal structure of the mo-lybdate-dependent transcriptional regulator ( ModE ) from Escherichia co-li : a novel combination of domain folds .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23913318" target="_blank">23913318</a>]<br></label>
<input id="dom_similarity_103", type="checkbox">
<div class="DOM_similar">
<li>The high-resolution crystal structure of the molybdate-dependent transcriptional regulator ( ModE ) from Escherichia coli : a novel combination of domain folds .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12581638" target="_blank">12581638</a>]<br></li>
<li>The high-resolution crystal structure of the molybdate-dependent transcriptional regulator ( ModE ) from Esche-richia coli : a novel combination of domain folds .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11092866" target="_blank">11092866</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12029041" target="_blank">12029041</a>]<br></li></div>
<li>In a number of cases Ser166 ( numbering for the Mop1 domain of E. coli ModE ) is replaced with an alanine , but from the structural data it seems perfectly conceivable that in those cases a water could take the place of the serine hydroxyl group .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11259434" target="_blank">11259434</a>]<br></li>
<li>Consider the Mop2 domain of E. coli ModE ; the residue corresponding to Mop1 Ser126 is Asn198 , which , in the observed conformation , would for steric reasons be unable to bind molybdate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11259434" target="_blank">11259434</a>]<br></li>
				</div></ul><br>
			<label class="collapse_EVO" for="_3"><font size="5px">ModE evolution</font></label>
				<input id="_3" type="checkbox">
				<div class="EVO"><br><ul><li>These proteins were identiﬁed as ModA from A. vinelandii ( 35 % identity , 45 % similarity ) ( 23 ) ( renamed ModE [ 29 ] ) , ModE from H. inﬂuenzae Rd ( 48 % identity , 58 % similarity ) ( 9 ) , MopA from R. capsulatus ( 34 % identity , 46 % similarity ) ( 45 ) , and MopII from Clostridium pasteurianum ( 29 % identity , 46 % similarity ) ( 17 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8550508" target="_blank">8550508</a>]<br></li>
<li>All identity and similarity designations are in relation to E. coli ModE .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8550508" target="_blank">8550508</a>]<br></li>
<li>Conclusions : Whereas most of the archaeal helix-turn-helix-containing transcription factors belong to families that are Archaea-specific , ModE is unusual in that it is found in both Archaea and Bacteria .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14641908" target="_blank">14641908</a>]<br></li>
<li>ModE may therefore be distantly related to the LysR family of transcriptional regulators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9210473" target="_blank">9210473</a>]<br></li>
<li>Consistent with these ﬁndings , a search of the BLOCKS database indicated that the ﬁrst 54 residues of ModE have a distant homology with the same region of the LysR family of transcriptional regulators ( 1 ) , and inspection of this sequence revealed the presence of a weak helix-turn-helix motif ( Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9721306" target="_blank">9721306</a>]<br></li>
<li>A search of the BLOCKS database to reveal more distant relationships identified the first 53 residues of ModE to be similar to both the AraC and LysR families of regulatory proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9210473" target="_blank">9210473</a>]<br></li>
				</div></ul><br>
			<label class="collapse_RP" for="_4"><font size="5px">Processes regulated by ModE</font></label>
				<input id="_4" type="checkbox">
				<div class="RP"><br><ul><li>This hypothesis is consistent with the notion that a genome evolves from a set of precursor genes to a mature size by gene duplications and increasing modiﬁcations ( Yanai et al. , 2000 ) Table 1 TFs identiﬁed in complete genomes Family DBD Role PFAM Annotations Total 63 328 Function MarR w R 502 Multiple antibiotic resistance response Mercury resistance regulator MerR c R 244 MetJ -- Helmann et al. ( 1989 ) ; Zeng et al. ( 1998 ) Old et al. ( 1993 ) ; Wu et al. ( 1993 ) 48 235 10 R 0 Methionine biosynthesis repressor Molybdenum metabolism regulation R ModE 30 NadRa c 798 w 0 20 12 R 0 Foster and Penfound ( 1993 ) Regulation of NAD-synthesis and nucleotide-transport Regulation of the nag ( N-acetylglucosamine ) operon Biosynthesis of membrane components Probable growth inhibitor regulator NagR/XylRa w 244 A 2 31 1 A 135 OmpR w 722 638 n/d n/d 0 PemK -- PhoU -- Pérez - Rueda and Collado-Vides ( 2000 ) 36 62 0 0 -- -- Phosphate assimilation Ribose phosphate isomerase 137 R R 94 RpiR c Sorensen and Hove-Jensen ( 1996 ) Aramaki et al. ( 1995 ) 157 6 865 12 897 TetR/AcrR m Tetracycline resistance repressors Tryptophan operon repressor 12 29 TrpR WrbA R m Yang et al. ( 1993 ) Yang et al. ( 1993 ) -- -- R -- Enhancer in the formation and/or stability of noncovalent complexes between TrpR and DNA The notation is as follows : family name ( Hyp : hypothetical families ) ; DBD structure ( w , winged ; m , miscellaneous , and c , classic ) ; HLH , helix-loop-helix ; , beta-strand antiparallel ; CSD , cold shock DNA-binding domain-like ; and Zn , zinc-ﬁnger ; ND , not determined .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15556475" target="_blank">15556475</a>]<br></li>
<li>Furthermore , many of the intergenic matches fell upstream of genes not obviously involved in molybdenum metabolism and therefore probably not subject to regulation by ModE .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14641908" target="_blank">14641908</a>]<br></li>
<li>Another newly identiﬁed ModE target , MorA-LacZ ( K-I-G-G-R - ) MorA-LacZ LacZ GlpK ybhK , encodes a homologue of YvcK , which is required for a normal cell shape and is involved in carbon metabolism in Bacillus subtilis , and restores the growth of the B. subtilis yvcK mutant ( 38 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23913318" target="_blank">23913318</a>]<br></li>
<li>Effect of mutant ModE proteins on transcription of Ph , & cz in a rnodf3 mutant All strains carry lWS1 ( P , , , lacZ ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10206709" target="_blank">10206709</a>]<br></li>
<li>C , kon values for * ModE ( T125I ) versus * ModE ( T125I ) concentration .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10446207" target="_blank">10446207</a>]<br></li>
<li>Keywords E. coli Æ deo operon Æ Regulation Æ Molybdenum Æ ModE Æ Macroarray Introduction Molybdenum containing enzymes play a central role in the global nitrogen cycle and in various redox reactions ( Hille 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16205910" target="_blank">16205910</a>]<br></li>
<li>Recently ModE was shown to play a minor role in regulating the hyc and nar operons in E. coli ( 29 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12029041" target="_blank">12029041</a>]<br></li>
<li>Thus , ModE appears to play a global role in regulating molybdenum homeostasis in a number of bacterial species .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12029041" target="_blank">12029041</a>]<br></li>
<li>As P2 also functions under anaerobic growth conditions and is derepressed in the absence of ModE , it is likely that , under these conditions , a signiﬁcant amount of dmsA expression , measured from the full-length construct ( lPM40 ) , originates from P2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466267" target="_blank">9466267</a>]<br></li>
<li>One possible explanation is that the expression of morA in the absence of repressor ModE is toxic for cell-growth as measured by colony formation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23913318" target="_blank">23913318</a>]<br></li>
<li>To explore the potential role of ModE and MoeA in the regulation of other E. coli genes , the global gene-expression proﬁle of a wild type and a modE , moeA double mutant grown in glucose-minimal medium under anaerobic conditions were compared .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16205910" target="_blank">16205910</a>]<br></li>
<li>Since the Mo-dependent po-sitive control of gene-expression also requires MoeA protein for optimal expression of hyc and nar , and moeA mutation had no detectable effect on the negative control of gene-expression by ModE , the global gene-expression proﬁle of strain SE2138 was compared to that of the double mutant , strain SE2155 , to enhance the sensitivity of the experiment .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16205910" target="_blank">16205910</a>]<br></li>
<li>To address the role of ModE in regulating dmsA expression further , we subjected the dmsA promoter fragment , cloned in lPM40 , to site-directed mutagenesis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466267" target="_blank">9466267</a>]<br></li>
<li>Results ModE is required for regulation of dmsA -- lacZ expression in response to oxygen availability To determine if ModE plays a role in the molybdate-depen-dent regulation of the dmsABC operon , we constructed a dmsA -- lacZ operon fusion , lPM40 ( Fig. 1 ) , and introduced it into strain PM8 ( modE : : kan ) and its wild-type progenitor , MC4100 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466267" target="_blank">9466267</a>]<br></li>
<li>To determine if ModE plays a role in the regulated expression from the napF promoter , we measured expression from a napF-lacZ operon fusion , HW2 ( Materials and Methods ) ( Fig. 1 ) in both wild-type ( MC4100 ) and modE ( PM8 ) backgrounds .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12029041" target="_blank">12029041</a>]<br></li>
<li>Identiﬁcation of the Set of Genes , Including Nonannotated morA , under the Direct Control of ModE in Escherichia coli Tatsuaki Kurata , a Akira Katayama , c Masakazu Hiramatsu , a Yuya Kiguchi , a Masamitsu Takeuchi , a Tomoyuki Watanabe , a Hiroshi Ogasawara , a , d Akira Ishihama , a , b Kaneyoshi Yamamotoa , b Department of Frontier Biosciencea and Research Center for Micro-Nano Technology , b Hosei University , Tokyo , Japan ; Department of Biochemistry and Molecular Biology , Nippon Medical School , Tokyo , Japanc ; Research Center for Human and Environmental Sciences , Shinshu University , Nagano Prefecture , Japand ModE is the molybdate-sensing transcription regulator that controls the expression of genes related to molybdate homeostasis in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23913318" target="_blank">23913318</a>]<br></li>
<li>( 2003 ) 326 , 761 -- 767 mational es Involving B Oxyanion A-binding D M E eveals oth Structure of Activated od R Crystal Confor N Ch g an ains and D om Alexander W. Schüttelkopf , David H. Boxer and William N. Hunter * Division of Biological Chemistry and Molecular Microbiology , The Wellcome Trust Biocentre , School of Life Sciences , University of Dundee Dundee DD1 5EH , UK * Corresponding author Introduction The molybdate-dependent transcriptional activator/repressor ModE appears to be the principal regulatory component of bacterial molybdenum metabolism , occuring in organisms like Escherichia coli , Yersinia pestis , Ralstonia solanacearum and Rhodobacter capsulatus .1 -- 6 ModE , activated by binding molybdate , acts as both a positive and a negative regulator of transcription of operons involved in various aspects of molybdenum metabolism .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12581638" target="_blank">12581638</a>]<br></li>
<li>Oxyanion Binding Alters Conformation and Quaternary Structure of the C-terminal Domain of the Transcriptional Regulator ModE IMPLICATIONS FOR MOLYBDATE-DEPENDENT REGULATION , SIGNALING , STORAGE , AND TRANSPORT * Received for publication , January 31 , 2001 , and in revised form , March 19 , 2001 Published , JBC Papers in Press , March 20 , 2001 , DOI 10.1074 / jbc.M100919200 David G. Gourley ‡ § , Alexander W. Schüttelkopf ‡ § , Lisa A. Anderson ‡ , Nicholas C. Price ¶ i , David H. Boxer ‡ , and William N. Hunter ‡ ** From ‡ The Wellcome Trust Biocentre , University of Dundee , Dundee , DD1 5EH and the ¶ Department of Biological Sciences , University of Stirling , Stirling , FK9 4LA , United Kingdom The molybdate-dependent transcriptional regulator ModE of Escherichia coli functions as a sensor of intra - cellular molybdate concentration and a regulator for the transcription of several operons that control the uptake and utilization of molybdenum .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11259434" target="_blank">11259434</a>]<br></li>
<li>These data argue that , in vivo , ModE regulates expression of modABCD by binding the region of DNA that includes the CATAA box and also upstream sequences .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9044285" target="_blank">9044285</a>]<br></li>
<li>The Molybdate-Responsive Escherichia coli ModE Transcriptional Regulator Coordinates Periplasmic Nitrate Reductase ( napFDAGHBC ) Operon Expression with Nitrate and Molybdate Availability Paul M. McNicholas † and Robert P. Gunsalus * Department of Microbiology , Immunology , and Molecular Genetics , and Molecular Biology Institute , University of California , Los Angeles , Los Angeles , California 90095-1489 Received 1 June 2001/Accepted 16 March 2002 Expression of the Escherichia coli napFDAGHBC operon ( also known as aeg46 .5 ) , which encodes the periplasmic molybdoenzyme for nitrate reduction , is increased in response to anaerobiosis and further stimulated by the addition of nitrate or to a lesser extent by nitrite to the cell culture medium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12029041" target="_blank">12029041</a>]<br></li>
<li>Department of Microbiology and Cell Science , University of Florida , Gainesville , FL 32611 , USA 1 Keywords : molybdate , regulation , ModE , formate hydrogenlyase , nay operon INTRODUCTION Under anaerobic growth conditions , Escherichia coli produces several molybdoenzymes , almost all of which serve as terminal enzymes in anaerobic respiratory pathways ( Gennis & Stewart , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10206709" target="_blank">10206709</a>]<br></li>
<li>An Analysis of the Binding of Repressor Protein ModE to modABCD ( Molybdate Transport ) Operator/Promoter DNA of Escherichia coli * ( Received for publication , October 9 , 1998 , and in revised form , April 5 , 1999 ) Amy M. Grunden ‡ § ¶ , William T. Self ‡ ¶ , Matteo Villaini , J. Edwin Blalocki , and K. T. Shanmugam ‡ ** From the ‡ Department of Microbiology and Cell Science , University of Florida , Gainesville , Florida 32611 and the iDepartment of Physiology and Biophysics , University of Alabama at Birmingham , Birmingham , Alabama 35294 Expression of the modABCD operon in Escherichia coli , which codes for a molybdate-specific transporter , is repressed by ModE in vivo in a molybdate-dependent fashion .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10446207" target="_blank">10446207</a>]<br></li>
<li>Grunden , A. M. ( 1996 ) Regulation of an Operon Coding for the Components of the Molybdate Transport System ( modABCD ) by ModE Protein , a Molybdate-dependent Repressor in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10446207" target="_blank">10446207</a>]<br></li>
<li>On the one hand it limits molybdate uptake by repressing the molybdate transporter operon modABCD , on the other hand it enhances the transcription of molybdenum-dependent enzymes like dimethyl sulphoxide reductase5 or nitrate reductase A7 and also of molybdenum cofactor-biosynthesis enzymes ( moaABCDE operon ) .8 Functional ModE is a homodimer of 2 # 28.3 kDa ; the structures of apo-ModE ,9 as well Abbreviations used : HTH , helix-turn-helix ; wHTH , winged helix-turn-helix ; r.m.s.d. , root-mean-square deviation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12581638" target="_blank">12581638</a>]<br></li>
<li>In the presence of molybdate , ModE repressed transcription of the modABC operon encoding the transporter of molybdate ( 2 ) and activates the synthesis of molybdoenzymes and proteins for Mo metabolism , including dmsABC encoding dimethyl sulfoxide reductase , napFDAGHBC encoding nitrate reductase , and moaABCDE coding for molybdenum cofactor synthase ( 5 -- 7 ) ( see Fig. 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23913318" target="_blank">23913318</a>]<br></li>
<li>ModE is a bacterial transcriptional regulator that orchestrates many aspects of molybdenum metabolism by binding to speciﬁc DNA sequences in a molybdate-dependent fashion .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12581638" target="_blank">12581638</a>]<br></li>
<li>The synthesis of all apomolybdoenzymes in E. coli may also be regulated by the two distinct molybdate-dependent control systems : ModE , a DNA binding regulatory protein , and MoeA , an enzyme which seems to activate molybdenum .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9851041" target="_blank">9851041</a>]<br></li>
<li>Identification of the set of genes , including non-annotated morA , under the direct control of ModE in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25406449" target="_blank">25406449</a>]<br></li>
<li>Our results revealed that ModE plays a signiﬁcant role in the control of the deo operon ( nucleoside-catabolism ) and opp op-eron ( oligopeptide-transport ) expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16205910" target="_blank">16205910</a>]<br></li>
<li>The presence and location of the consensus sequence is in agreement with a role of repressor for ModE in the control of deo operon expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16205910" target="_blank">16205910</a>]<br></li>
<li>See the Methods section for detail Discussion A combination of global gene-expression analysis and bioinformatics correctly identiﬁed many genes that are controlled by the molybdate sensor/regulator protein , ModE , as well as a set of new genes under Mo-control in E. coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16205910" target="_blank">16205910</a>]<br></li>
<li>Arch Microbiol ( 2005 ) 184 : 225 -- 233 DOI 10.1007 / s00203-005-0039-7 O R I GI N A L P A P E R Han Tao Æ Adnan Hasona Æ Phi M. Do Æ L.O. Ingram K.T. Shanmugam deo operon Global gene-expression analysis revealed an unsuspected under the control of molybdate sensor , ModE protein , in Escherichia coli Received : 29 March 2005 / Revised : 30 August 2005 / Accepted : 2 September 2005 / Published online : 5 October 2005 Springer-Verlag 2005 Abstract ModE protein , a molybdate sensor/regulator , controls the transcription of genes coding for molybdate uptake ( mod ) , molybdopterin synthesis ( moa ) , molybdoenzymes nitrate reductase ( nap ) and dimethylsulfoxide reductase ( dms ) , as well as fermentative dihydrogen production ( fdhF and hyc ) and respiratory nitrate reductase ( narXL ) in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16205910" target="_blank">16205910</a>]<br></li>
<li>Tao , H. , Hasona , A. , Do , P. , Ingram , L. , and Shanmugan , K. ( 2005 ) Global gene-expression analysis revealed an unsuspected deo operon under the control of molydate sensor , ModE protein , in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19007420" target="_blank">19007420</a>]<br></li>
<li>Global gene-expression analysis revealed an unsuspected deo operon under the control of molybdate sensor , ModE protein , in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23913318" target="_blank">23913318</a>]<br></li>
<li>Results Mo-sensitive differential expression proﬁle The molybdate sensor protein , ModE , in association with molybdate , binds to unique DNA sequences and regulates the transcription of mod , nar , hyc , fdhF , moa , nap and dms operons in E. coli ( Rosentel et al. 1995 ; Grunden et al. 1996 ; McNicholas et al. 1996 ; McNich-olas et al. 1997 ; Anderson et al. 2000 ; Self et al. 2001a ; McNicholas and Gunsalus 2002 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16205910" target="_blank">16205910</a>]<br></li>
<li>ModE is a sensor of intracellular Mo , and a ModE-Mo complex represses the modABCD operon , which speciﬁes components of a Mo transport system .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11528004" target="_blank">11528004</a>]<br></li>
<li>' Nitrogen Fixation Laboratory , John Innes Centre , Norwich , UK XER ' and Richard N. PAU ' ( Received 23 December 1996/7 March 1997 ) - EJB Molybdenum-dependent repression of transcription of the Escherichia coli modABCD operon , which encodes the high-affinity molybdate transporter , is mediated by the ModE protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9210473" target="_blank">9210473</a>]<br></li>
<li>Recently , the product of the modE gene , ModE , was identiﬁed as the molybdate-responsive regulator of the modABCD operon , which encodes a high-afﬁnity molybdate uptake system ( Walkenhorst et al. , 1995 ; Grunden et al. , 1996 ; McNicholas et al. , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466267" target="_blank">9466267</a>]<br></li>
<li>The molybdate enhancement of moa transcription is fully dependent upon the molybdate-binding protein , ModE , which also mediates molybdate repression of the mod operon encoding the high-afﬁnity molybdate uptake system .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11092866" target="_blank">11092866</a>]<br></li>
<li>ModE , initially characterized as a molybdate-dependent repressor of the modABC operon carrying high-afﬁnity molybdate transport genes ( 18 ) , has subsequently been shown to act as a positive transcriptional regulator of the hyc operon * Corresponding author .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14702328" target="_blank">14702328</a>]<br></li>
<li>In this regard , the ModE protein is both a positive regulator of molybdoenzyme synthesis and a repressor of mod ( molybdate transport ) operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9851041" target="_blank">9851041</a>]<br></li>
<li>In E. coli , ModE was previously identified as a repressor controlling transcription of the operon encoding molybdate transport components ( modABCD ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10206709" target="_blank">10206709</a>]<br></li>
<li>Regulation of an operon coding for the components of the molybdate-transport system ( modABCD ) by ModE protein , a molybdate-dependent repressor in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10206709" target="_blank">10206709</a>]<br></li>
<li>To determine if the molybdate-responsive transcription factor ModE is involved in this process , a set of dmsA -- lacZ operon fusions were constructed and analysed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466267" target="_blank">9466267</a>]<br></li>
<li>Conclusions Transcriptional regulation of molybdo-enzymes , and genes involved in molybdenum metabolism and homeos-tasis , is performed by the molybdate-responsive transcription factor ModE in E. coli and related Proteobacteria .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14641908" target="_blank">14641908</a>]<br></li>
<li>The ModE protein has a wider role in mediating molybdate regulation of other operons which include dmsA , the structu-ral operon for the molybdoenzyme dimethyl sulfoxide reductase ( 21 ) , hyc , the hydrogenase 3 structural operon ( 34 ) , and narG , which encodes the molybdoenzyme nitrate reductase A ( 34 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11092866" target="_blank">11092866</a>]<br></li>
<li>The molybdate-responsive transcription factor , ModE , regulates the expression of a number of operons in Escherichia coli in response to changes in the intracellular levels of molybdate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9721306" target="_blank">9721306</a>]<br></li>
<li>Molybdate ( MoO 2 4 ) availability controls anaerobic respiratory gene-expression ( 23 ) through the molybdate-responsive transcriptional regulator ModE ( 28 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=13129959" target="_blank">13129959</a>]<br></li>
<li>Anaerobic regulation of the Escherichia coli dmsABC operon requires the molybdate responsive regulator , ModE .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9721306" target="_blank">9721306</a>]<br></li>
<li>J Bacteriol 145:1110 -- 1112 McNicholas PM , Gunsalus RP ( 2002 ) The molybdate-responsive Eschrichia coli ModE transcriptional regulator coordinates periplasmic nitrate reductase ( napFDAGHBC ) operon expression with nitrate and molybdate availability .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16205910" target="_blank">16205910</a>]<br></li>
<li>McNicholas , P.M. , and Gunsalus , R.P. ( 2002 ) The molyb-date-responsive Escherichia coli ModE transcriptional regulator coordinates periplasmic nitrate reductase ( napFDAGHBC ) operon expression with nitrate and molybdate availability .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16677314" target="_blank">16677314</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_52">
The molybdate-responsive Esche-richia coli ModE transcriptional regulator coordinates periplasmic nitrate reductase ( napFDAGHBC ) operon expression with nitrate and molybdate availability .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23913318" target="_blank">23913318</a>]<br></label>
<input id="rp_similarity_52", type="checkbox">
<div class="RP_similar">
<li>The molybdate-responsive Escherichia coli ModE transcriptional regulator coordinates periplasmic nitrate reductase ( napFDAGHBC ) operon expression with nitrate and molybdate availability .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=13129959" target="_blank">13129959</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19060147" target="_blank">19060147</a>]<br></li></div>
<li>These results show that besides ModE , another factor is involved in Mo-dependent expression of the h y c and narGHJZ operons .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10206709" target="_blank">10206709</a>]<br></li>
<li>Haemophilus influenxae ModE can functionally replace E. coli ModE in the repression of E. coli modA-lacZ ( A. M. Grunden , unpublished ) and thus its apparent function in the native organism is also to repress m o d A B C operon expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10206709" target="_blank">10206709</a>]<br></li>
<li>Even in the presence of pWS31 , which encodes native ModE protein , expression of Ph , , lacZ was completely molybdate-dependent ( Table 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10206709" target="_blank">10206709</a>]<br></li>
<li>ModE is essential for optimum expression of Ph , , / xz and narG-lacZ ... I. .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10206709" target="_blank">10206709</a>]<br></li>
<li>ModE also binds and regulates expression of the moaADCDE and dmsABC operons , which encode enzymes involved in the ﬁrst steps of molybdate assimilation into molybdopterin and the DMSO reductase , respectively ( 1 , 20 , 21 , 23 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12029041" target="_blank">12029041</a>]<br></li>
<li>It is possible that the molybdate-bound form of ModE , the known repressor of the m o d A B C D operon , is also the active form participating in h y c and narG expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10206709" target="_blank">10206709</a>]<br></li>
<li>ModE also positively regulates expression of the moaABCDE operon ( McNicholas et al. , 1997 ) , which encodes enzymes involved in molybdate utilization ( Rivers et al. , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466267" target="_blank">9466267</a>]<br></li>
<li>We also characterize ModE binding at the promoter of the moaABCDE operon , which encodes enzymes involved in molybdate utilization within the cell ( Rivers et al. , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9044285" target="_blank">9044285</a>]<br></li>
<li>It is yet unclear if ModE interactions with molybdate act primarily to modulate DNA binding or , alternatively , to affect ModE interactions with other proteins involved in napF gene-expression , including Fnr , NarP , NarL , and RNAP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12029041" target="_blank">12029041</a>]<br></li>
<li>The upstream ModE regulatory site functions to override nitrate control of napF gene-expression when the essential enzyme component , molybdate , is limiting in the cell environment .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12029041" target="_blank">12029041</a>]<br></li>
<li>Our studies strongly suggest that for expression originating from the napF promoter , the only molybdate-requiring component involved in mediating the response to nitrate addition is ModE .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12029041" target="_blank">12029041</a>]<br></li>
<li>In Azotobacter vinelandii , the ModE protein is likewise involved in repression of the operon encoding genes for the high-affinity mo-lybdate transporter ( Mouncey et al. , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9210473" target="_blank">9210473</a>]<br></li>
<li>The identiﬁcation of ModE homologues in a number of other bacteria ( Wang et al. , 1993 ; Mouncey et al. , 1995 ) , and the report that , in at least one instance , ModE may be involved in the regulation of a related molybdoenzyme in Rhodobacter capsulatus ( Solomon et al. , 1997 ) suggests that information gained in the study of molybdate homeostasis in E. coli may be applicable to a wide number of bacteria , including those capable of nitrogen ﬁxation , photosynthesis and denitriﬁcation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466267" target="_blank">9466267</a>]<br></li>
				</div></ul><br>
			<label class="collapse_SIT" for="_5"><font size="5px">ModE binding site</font></label>
				<input id="_5" type="checkbox">
				<div class="SIT"><br><ul><li>To construct a pModE plasmid for overproduction of His-tagged ModE , the DNA fragments were prepared by PCR using E. coli W3110 genome DNA as the template and a set of primer pairs , modEF and modER ( for sequences , see Table S1 in the supplemental material ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23913318" target="_blank">23913318</a>]<br></li>
<li>E. coli K-12 strains , plasmids , and bacteriophages Strain , plasmid , or phage Origin Source or reference Relevant genotype or phenotype Strains MC4100 PM6 PM8 F ( argF-lac ) U169 modC modE : : Kanr modE : : kanr Laboratory stock 20 20 MC4100 MC4100 Plasmids pR415 lacZ lacY lacA Ampr modE Cmr pBR322 30 20 pACYC184 pACYC184 pRS415 pRS415 pPM58 pPM58 pPM69 pRS415 pPM6 pPM9 pHW2 pPM58 and -59 pPM69 pPM70 pPM71 pPM54 and -55 modE ; encodes a molybdate independent modE allele , Cmr napF-lacZ operon fusion , Ampr napF-lacZ ; operon fusions with upstream deletions in napF promoter region , Ampr napF-lacZ ; pPM58 with upstream region cloned back in correct orientation , Ampr napF-lacZ ; pPM58 with upstream region cloned back in reverse orientation , Ampr napF-lacZ ; pPM69 105-bp insert from the cat gene cloned in the EcoRI site , Ampr napF-lacZ ; operon fusions with mutations in the ModE binding motif , Ampr 20 Laboratory stock This study This study This study This study This study Phages RS45 lacZ lacY lacA ( napF-lacZ ) ( operon fusion ) 30 HW2 PM58 and -59 PM69 PM70 PM71 PM54 and 55 RS45 RS45 PM58 PM58 PM69 RS45 Laboratory stock This study This study This study This study This study ( napF-lacZ ) ( operon fusions with upstream deletions in napF promoter region ) ( napF-lacZ ) ( PM58 with upstream region cloned back in correct orientation ) ( napF-lacZ ) ( PM58 with upstream region cloned back in reverse orientation ) ( napF-lacZ ) ( PM69 with 105-bp insert from the cat gene cloned in the EcoRI site ) ( napF-lacZ ) ( operon fusions with mutations in the ModE binding site ) ModE binds the napF promoter well upstream of the start site of transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12029041" target="_blank">12029041</a>]<br></li>
<li>We note that the spacing between the inverted-repeats , at all three of the experimentally determined ModE sites in E. coli , is an invariant 6 bp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466267" target="_blank">9466267</a>]<br></li>
<li>Thus , ModE appears to recognize and bind a 9 bp inverted-repeat which is separated by 6 bp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9044285" target="_blank">9044285</a>]<br></li>
<li>This revised ModE box is composed of an inverted-repeat of 9 nucleotides , CGNTATATA , with a spacer of 6 nucleotides .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23913318" target="_blank">23913318</a>]<br></li>
<li>A single tandem mop repeat , which is also known as a diMop domain , forms two Greek-key b-barrels , which are structu-rally similar to the oligonucleotide/oligosaccharide binding fold .15 ModE binds two molybdate mole-cules per dimer with high afﬁnity ( < 0.8 mM ) ,3 in binding sites located at the interface between the two chains .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12581638" target="_blank">12581638</a>]<br></li>
<li>Matches to the ModE ( Anderson et al. , 1997 ; McNicholas et al. , 1997 ) and IHF consensus sequences ( Freundlich et al. , 1992 ) are bracketed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466267" target="_blank">9466267</a>]<br></li>
<li>In vitro studies conﬁrmed that ModE bound the dmsA promo-ter with high afﬁnity at a sequence , which closely matches the operator sequences at the modA and moaA promoters ( Anderson et al. , 1997 ; McNicholas et al. , 1997 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466267" target="_blank">9466267</a>]<br></li>
<li>Location of the experimentally determined ModE and IHF recognition sequences at dmsA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466267" target="_blank">9466267</a>]<br></li>
<li>We propose that these increases are primarily a result of relieving repression from P2 as , when IHF and ModE are bound at dmsA , they would be predicted to occlude the ¹ 35 and ¹ 10 consensus sequences respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466267" target="_blank">9466267</a>]<br></li>
<li>Hatched bars represent ModE sequences , and solid bars represent sequences from CI .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9721306" target="_blank">9721306</a>]<br></li>
<li>Comparison of the sequences protected from cleavage with DNase I by ModE at the promoters of the mod and moa operons .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9044285" target="_blank">9044285</a>]<br></li>
<li>Since we identiﬁed the ModE-mo-lybdate binding sequences for six additional targets , we reexamined the ModE box sequence using the W-AlignACE program ( 34 ) , which was successfully employed for identiﬁcation of the MntR box sequence ( 35 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23913318" target="_blank">23913318</a>]<br></li>
<li>Sequences for binding the NarP and NarL , Fnr , and ModE proteins are boxed ; consensus sequences are shown below .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=13129959" target="_blank">13129959</a>]<br></li>
<li>Nucleotides that match the consensus sequences are shaded black ( promoter 10 and 35 elements ) or gray ( NarP , Fnr , and Crp ) or are enclosed within boxes ( NsrR and ModE ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19060147" target="_blank">19060147</a>]<br></li>
<li>In the presence of molybdate , ModE protected a 30 bp region of DNA corresponding to -156 to -127 in which + 1 is the ` A ' in the ATG for narX ( Fig. 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10206709" target="_blank">10206709</a>]<br></li>
<li>ModE protected a 30-bp region ( nucleotides 147 to 118 ) centered at position 133.5 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12029041" target="_blank">12029041</a>]<br></li>
<li>In contrast to previously characterized ModE operator sites , which either overlap or are located immediately upstream of the ModE-regulated promo-ter , the ModE site is centred 52.5 bp downstream of the major dmsA transcript start site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466267" target="_blank">9466267</a>]<br></li>
<li>In vitro studies conﬁrmed that ModE bound dmsA at a site located well downstream of the major Fnr-regulated promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466267" target="_blank">9466267</a>]<br></li>
<li>DNase I footprinting localized ModE binding to a site located well downstream ( centred at þ 52.5 ) of the Fnr / NarL-regulated promoter , P1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466267" target="_blank">9466267</a>]<br></li>
<li>Gel shift and DNase I footprinting analyses establish that ModE binds to the napF promoter with an apparent Kd of about 35 nM at a position centered at 133.5 relative to the start of napF transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12029041" target="_blank">12029041</a>]<br></li>
<li>IHF binds dmsA immediately upstream of ModE The vast majority of repressors bind their cognate promo-ters in a region that spans from position ¹ 70 to þ 20 relative to the start of transcription ( Gralla and Collado-Vides , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466267" target="_blank">9466267</a>]<br></li>
<li>ModE and IHF bind dmsA in the vicinity of the P1 transcript start site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466267" target="_blank">9466267</a>]<br></li>
<li>ModE binds the dmsA promoter downstream of the start site of transcription To conﬁrm that ModE binds the dmsA promoter region , we performed a gel shift assay with a 600 bp promoter fragment isolated from pPM40 ( Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466267" target="_blank">9466267</a>]<br></li>
<li>ModE protected a single 28 bp region extending from position þ 39 to þ 66 ( Fig. 4 ) , relative to the P1 transcription start site ( Fig. 6A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466267" target="_blank">9466267</a>]<br></li>
<li>However , we note that these data do not rule out the possibility that ModE binds a sequence located downstream of the CATAA box .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9044285" target="_blank">9044285</a>]<br></li>
<li>ModE has three tryptophans ( positions 49 , 131 , and 186 ) ( Fig. 7 ) , and the tryptophan at 131 is located within a sequence motif ( 125TSARNQWFG133 ) that is conserved in several ModE homologs from other organisms ( ( T / S ) SARNQXXG ) and also in a molybdopterin-binding protein from C. pasteurianum ( 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10446207" target="_blank">10446207</a>]<br></li>
<li>Q 1997 Blackwell Science Ltd Results ModE binds the modA promoter at the start site of transcription in vivo To determine if sequences upstream of the ¹ 35 consensus sequence were required for regulation of modA -- lacZ expression in response to molybdate availability , we deleted upstream DNA from plasmid pPM24 to within 100 bp of the start site of modA transcription , generating pPM26 ( Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9044285" target="_blank">9044285</a>]<br></li>
<li>Precise localization of the ModE-binding site at the modA promoter To accurately identify which sequences are necessary for binding of ModE we performed DNase I footprinting on the modA promoter fragment derived from pPM26 , in the pre-sence and absence of 100 nM molybdate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9044285" target="_blank">9044285</a>]<br></li>
<li>ModE binds upstream of the putative ¹ 35 consensus sequence at the moaA promoter We performed DNase I footprinting on the moaA promoter fragment derived from pPM29 , in the presence and absence of 100 nM molybdate ( Fig. 5B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9044285" target="_blank">9044285</a>]<br></li>
<li>ModE binds the modA promoter at the start site of transcription in vitro The C-terminus of ModE comprises two highly homologous tandem repeats of MopI , a small pterin-binding protein from Clostridium pasteurianum ( Hinton and Freyer , 1986 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9044285" target="_blank">9044285</a>]<br></li>
<li>In vitro studies have demonstrated that dimeric ModE directly binds two molybdate ions with a Kd of about 0.8 mM and that the molybdate-ModE complex binds to mod DNA at a site which spans the modA transcription start site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11092866" target="_blank">11092866</a>]<br></li>
<li>In this study we demonstrate that ModE binds the modA promoter at a single site that overlaps the ¹ 10 consensus sequence and the transcript start site , and that this binding is enhanced by the addition of molybdate over a range of concentrations equivalent to those found in the cell .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9044285" target="_blank">9044285</a>]<br></li>
<li>ModE binds molybdate , and the complex can function as a repressor by binding at a site that overlaps the transcription start of the modABCD operon ( 10 -- 12 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11259434" target="_blank">11259434</a>]<br></li>
<li>A ModE site was identiﬁed well upstream of the Fnr and NarP binding sites at the napF promoter , and we demonstrate that ModE binds this site in a molybdate-responsive manner to thereby coordinate enzyme synthesis with molybdate availability .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12029041" target="_blank">12029041</a>]<br></li>
<li>Sequence analysis of the napF promoter region identiﬁed a typical ModE binding site , centered at 133.5 bp with respect to the transcript start site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12029041" target="_blank">12029041</a>]<br></li>
<li>Inspection of the napF promoter region revealed a close match to the proposed E. coli ModE consensus recognition sequence reported by Anderson , McNicholas , and McNicholas ( 2 , 20 , 23 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12029041" target="_blank">12029041</a>]<br></li>
<li>of conserved residues in the ModE recognition sequence centered at 133.5 relative to the napF promoter ( Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12029041" target="_blank">12029041</a>]<br></li>
<li>ModE regulates a second promoter , P2 , located downstream of the major dmsA promoter Examination of the DNA sequence in the vicinity of the ModE-binding site revealed that the left half of the ModEbinding site overlapped a putative ¹ 10 consensus sequ-ence for a second promoter , P2 ( Fig. 6A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466267" target="_blank">9466267</a>]<br></li>
<li>Based upon the ModE consensus sequence ( Fig. 4 ) , we searched for other putative ModE binding sites by using the PatScan program ( 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12029041" target="_blank">12029041</a>]<br></li>
<li>To conﬁrm that ModE bound dmsA in the vicinity of this sequence and to rule out the possibility that there were other ModE-binding sites elsewhere in the promoter region , we performed gel shift assays using various truncated dmsA promoter fragments ( Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466267" target="_blank">9466267</a>]<br></li>
<li>Fig. 2 Upstream DNA sequence of the deo operon indicating the binding sites for various regulatory elements Fig. 3 DNA electrophoretic mobility shift of the deoC operator / promoter DNA with puriﬁed ModE protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16205910" target="_blank">16205910</a>]<br></li>
<li>A search for sites in these two organisms ' mod DNAs which are similar to the putative ModE repressor binding site resulted in the detection of only a GATAA sequence upstream of the modA gene in A. vinelandii ( 23 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8550508" target="_blank">8550508</a>]<br></li>
<li>Although the ModE binding site sequence is similar to other E. coli ModE binding sites , the location is atypical , because it is not centered near the start of transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12029041" target="_blank">12029041</a>]<br></li>
<li>As expected , we found good matches to the proposed consensus ModE binding sequence immediately upstream of genes implicated in molybdenum metabolism in the γ-proteobacterial genomes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14641908" target="_blank">14641908</a>]<br></li>
<li>This ModE box sequence includes the previously proposed consensus sequence of ModE binding ( 33 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23913318" target="_blank">23913318</a>]<br></li>
<li>The sequence of the protected region at moaA closely matched that seen with modA and the proposed ModE consensus sequence ( Fig. 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9044285" target="_blank">9044285</a>]<br></li>
<li>Employing DNase I footprinting , we demonstrated that ModE binds a region that overlaps the start site of transcription and that the sequence of the binding site closely matches the proposed consensus sequence ( Kutsche et al. , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9044285" target="_blank">9044285</a>]<br></li>
<li>B. Comparison of the ModE-binding site at dmsA with the sites at modA and moaA ( Anderson et al. , 1997 ; McNicholas et al. , 1997 ) and with a proposed ModE consensus sequence ( Kutsche et al. , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466267" target="_blank">9466267</a>]<br></li>
<li>Alignment of the ModE binding site at the napF , dmsA , modA , and moaA promoters with the proposed ModE consensus sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12029041" target="_blank">12029041</a>]<br></li>
<li>The ModE DNaseI footprinting experiments indicate that this proposed IHF site would overlap the left half of the ModE binding site by 5 bp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12029041" target="_blank">12029041</a>]<br></li>
<li>Each of the three DNase I-cleav-age protected regions contains either a TATAT ( protected region 1 ) or a TACAT ( protected regions 2 and 3 ) sequence , which suggests that it is these sequences that are recognized by ModE as the target binding site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10446207" target="_blank">10446207</a>]<br></li>
<li>The fact that the DNase I footprint presented in Fig. 1 resolves the modA operator / promoter region into discrete ModE-binding regions with a distinct DNase I hypersensitive site delineating protected regions 2 and 3 suggests that ModE binds and protects the three indicated regions in vivo also .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10446207" target="_blank">10446207</a>]<br></li>
<li>The fact that ModE is capable of binding to DNA containing the TAAC z GTTA inverted-repeat which is not ﬂanked by the CAT boxes ( modF-42mer DNA ) , albeit at a much reduced level compared with the binding of the CAT box-ﬂanked inverted-repeat ( modA-42mer DNA ) , recommended the analysis of modABCD DNA for the presence of other similar inverted-repeats which might serve as potential sites of additional regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8550508" target="_blank">8550508</a>]<br></li>
<li>Further experimentation indicated that ModE binding to target DNA minimally requires an 8-bp inverted-repeat sequence , TAAC z GTTA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8550508" target="_blank">8550508</a>]<br></li>
<li>For the purpose of distinguishing whether the inverted-repeat or the ﬂanking CAT sequences are required for binding ModE , a second 42-bp-long oligonucleotide derived from the modF gene containing only the TAAC z GTTA inverted-repeat sequence ( Fig. 3 ) ( modF-42mer DNA ) was also included in the mobility shift experiments .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8550508" target="_blank">8550508</a>]<br></li>
<li>Thus , the mobility shift experiments demonstrated binding of ModE in the presence of molybdate to the 8-bp inverted-repeat ( TAAC z GTTA ) , with enhanced binding demonstrated for the DNA containing the ﬂanking CA and CATTAA sequences as well .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8550508" target="_blank">8550508</a>]<br></li>
<li>An alternative explanation , suggested by the dyad symmetry of the ModE binding site ( 1 , 8 , 12 , 13 ) , is that ModE must dimerize in order to efﬁciently bind DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9721306" target="_blank">9721306</a>]<br></li>
<li>The region of ModE that corresponds to those DNA sequences that were deleted in the cI-modE2 fusion ( i.e. , between the PstI and KpnI sites ) is labeled .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9721306" target="_blank">9721306</a>]<br></li>
<li>After analysis of a set of 200-bp DNA sequences centered on each ModE-molybdate binding peak for six ModE target sequences ( Sch1 , Sch2 , Sch3 , Sch5 , Sch6 , and Sch9 ) , we identiﬁed a conserved sequence of 24 bp in length ( Fig. 1C ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23913318" target="_blank">23913318</a>]<br></li>
<li>ModE is a bacterial transcriptional regulator that orchestrates many aspects of molybdenum metabolism by binding to speciﬁc DNA sequences in a molybdate-dependent fashion .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12581638" target="_blank">12581638</a>]<br></li>
<li>These data argue that , in vivo , ModE regulates expression of modABCD by binding the region of DNA that includes the CATAA box and also upstream sequences .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9044285" target="_blank">9044285</a>]<br></li>
<li>The presence of a dyad element in the DNA-binding site for ModE suggests that ModE binds as a dimer .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9210473" target="_blank">9210473</a>]<br></li>
<li>) Finally , to establish that DNA sequences 5 of the ModE binding site were not required for the molybdate response , PM59 was constructed where the nucleotides from 233 to 157 were deleted .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12029041" target="_blank">12029041</a>]<br></li>
<li>The analysis presented above demonstrated that deletion of upstream DNA sequences containing the ModE binding site alleviated the in vivo requirement for molybdate and ModE for optimal napF-lacZ expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12029041" target="_blank">12029041</a>]<br></li>
<li>NarP ( Darwin and Stewart , 1995 ) , FNR ( Darwin et al. , 1998 ) and ModE ( McNicholas and Gunsalus , 2002 ) binding sites are over - or underlined .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16677314" target="_blank">16677314</a>]<br></li>
<li>It is noteworthy that the ModE-activated promoters are always associated with FNR-binding sites within a distance of less than 100 bp ( see Fig. 2 ) , suggesting a cooperative function between ModE and FNR for induction of promoters by ModE .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23913318" target="_blank">23913318</a>]<br></li>
<li>If this is the case , and given the size of the ModE protein ( 28 , 271 Da ) and the target binding sites ( TATAT and TACAT ) , it is likely that ModE would bind to each of the two ModE-protected regions as a monomer .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10446207" target="_blank">10446207</a>]<br></li>
<li>The difference in the amounts of binding between the two oligonucleotides and ModE must correlate with the differences in the target binding sequences found within the oligonucleotides .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8550508" target="_blank">8550508</a>]<br></li>
<li>Implications of the differential binding of ModE to the two binding sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8550508" target="_blank">8550508</a>]<br></li>
<li>It was proposed that two inverted-repeat sequences located within the ModE-protected region serve as possible ModE binding sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10446207" target="_blank">10446207</a>]<br></li>
<li>The complete genome sequence of H. hepaticus does not encode a full-length ModE homologue ; it does encode a 135 amino acid protein ( HH0653 ) , consisting of a single domain , which is therefore probably able to specifically bind to the identified sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14641908" target="_blank">14641908</a>]<br></li>
<li>The proposed ModE consensus sequence , shown at the bottom of the ﬁgure , is deduced from comparison of the inverted-repeats found in front of the mod operons from various bacterial species ( Kutsche et al. , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9044285" target="_blank">9044285</a>]<br></li>
<li>The regions protected from DNaseI digestion are bracketed and matches to the ModE consensus sequence shown in upper case .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466267" target="_blank">9466267</a>]<br></li>
<li>Nucleotides protected from DNaseI digestion are bracketed , and nucleotide matches to the ModE consensus sequence are shown in uppercase ( 2 , 20 , 23 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12029041" target="_blank">12029041</a>]<br></li>
<li>Binding of molybdate to the C-terminal domain induces a conformational change of ModE so that it recognizes a palindromic sequence of its target promoters ( 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23913318" target="_blank">23913318</a>]<br></li>
<li>The molybdate-independent mutant ModE proteins apparently mimic in its conformation the native ModE-molybdate complex , which binds to a DNA sequence motif of TATAT-7bp-TAYAT .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10446207" target="_blank">10446207</a>]<br></li>
<li>The various ModE domains , N-terminal , MopE1 , and MopE2 domains , are boxed , and the putative helix-turn-helix motif is indicated by a bracket below the sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9721306" target="_blank">9721306</a>]<br></li>
<li>The translation start site for modE was conﬁrmed by the N-terminal amino acid sequence of puriﬁed ModE .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8550508" target="_blank">8550508</a>]<br></li>
				</ul></div><br>
			<label class="collapse_TU" for="_6"><font size="5px">Transcription units regulated by ModE</font></label>
				<input id="_6" type="checkbox">
				<div class="TU"><br><ul><li>ModE has been reported to activate hyc encoding a formate hydrogenlyase , narXL encoding a two-component regulatory system , and deoCABD operons encoding the deoxyribose utilization enzyme by transcriptome analysis of an modE and moeA double mutant ( 1 , 41 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23913318" target="_blank">23913318</a>]<br></li>
<li>The Molybdate-Responsive Escherichia coli ModE Transcriptional Regulator Coordinates Periplasmic Nitrate Reductase ( napFDAGHBC ) Operon Expression with Nitrate and Molybdate Availability Paul M. McNicholas † and Robert P. Gunsalus * Department of Microbiology , Immunology , and Molecular Genetics , and Molecular Biology Institute , University of California , Los Angeles , Los Angeles , California 90095-1489 Received 1 June 2001/Accepted 16 March 2002 Expression of the Escherichia coli napFDAGHBC operon ( also known as aeg46 .5 ) , which encodes the periplasmic molybdoenzyme for nitrate reduction , is increased in response to anaerobiosis and further stimulated by the addition of nitrate or to a lesser extent by nitrite to the cell culture medium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12029041" target="_blank">12029041</a>]<br></li>
<li>Transcription of hyc and narX operons are enhanced by ModE ( + ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10206709" target="_blank">10206709</a>]<br></li>
<li>Recent studies have shown that the hyc and nar operons can be transcribed in the absence of functional ModE when molybdate is present [ 5 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9851041" target="_blank">9851041</a>]<br></li>
<li>Nitrate-dependent increase in transcription of nar-lacZ also requires both ModE and MoeA proteins The nitrate-dependent increases in expression of the three nar operons analyzed in this study , narG-lacZ ( nitrate reductase structural genes [ 9 ] ) , narKlacZ ( nitrite extrusion [ 16 ] ) and narX-lacZ ( regulatory proteins [ 9,17 ] ) were also lower in a modE mutant background as compared to the respective pa-rent strains ( Table 3 ; also [ 5 ] ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9851041" target="_blank">9851041</a>]<br></li>
<li>The second pentamer sequence found in the m o d A operator DNA is also conserved in the two operons ( hyc and nar ; 5 ' CATAT 3 ' and 5 ' TACAT 3 ' , respectively ) whose transcription is positively regulated by ModE ( Fig. 7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10206709" target="_blank">10206709</a>]<br></li>
<li>Since the control of synthesis of three of the six positively regulated operons requires both ModE and MoeA proteins , it is possible that the regulation of transcription of these additional molybdoenzyme gene ( s ) is also controlled by a combination of these two proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16205910" target="_blank">16205910</a>]<br></li>
<li>To explore the potential role of ModE and MoeA in the regulation of other E. coli genes , the global gene expression proﬁle of a wild type and a modE , moeA double mutant grown in glucose-minimal medium under anaerobic conditions were compared .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16205910" target="_blank">16205910</a>]<br></li>
<li>Although ModE apparently plays a role in the regulation of these two operons , the mechanism of this regulation is not known and these studies are also complicated by the presence of two promoters in the dms operon ( McNicholaset al. , 1998 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10206709" target="_blank">10206709</a>]<br></li>
<li>Although ModE plays a role in the control of the dmsABC operon , the ModE protein appears to repress transcription from a downstream promoter , P2 ( McNicholas et al. , 1998 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10206709" target="_blank">10206709</a>]<br></li>
<li>A mutant ModE ( T125I ) protein , which repressed modA-lac even in the absence of molybdate , protected the same region of modA operator DNA in the absence of molybdate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10446207" target="_blank">10446207</a>]<br></li>
<li>DISCUSSION It is now well established that the mod operon transcription is repressed by ModE , which is encoded by the first gene in the divergent modEF operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10206709" target="_blank">10206709</a>]<br></li>
<li>ModE protein repressed the modABC operon under conditions of Mo-excess ( Grunden et al. 1996 ; McNicholas et al. 1996 ) and the interaction between the mod operator DNA and ModE has been established at the biochemical level ( Grunden et al. 1999 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16205910" target="_blank">16205910</a>]<br></li>
<li>On the one hand it limits molybdate uptake by repressing the molybdate transporter operon modABCD , on the other hand it enhances the transcription of molybdenum-dependent enzymes like dimethyl sulphoxide reductase5 or nitrate reductase A7 and also of molybdenum cofactor biosynthesis enzymes ( moaABCDE operon ) .8 Functional ModE is a homodimer of 2 # 28.3 kDa ; the structures of apo-ModE ,9 as well Abbreviations used : HTH , helix-turn-helix ; wHTH , winged helix-turn-helix ; r.m.s.d. , root-mean-square deviation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12581638" target="_blank">12581638</a>]<br></li>
<li>In the presence of molybdate , ModE repressed transcription of the modABC operon encoding the transporter of molybdate ( 2 ) and activates the synthesis of molybdoenzymes and proteins for Mo metabolism , including dmsABC encoding dimethyl sulfoxide reductase , napFDAGHBC encoding nitrate reductase , and moaABCDE coding for molybdenum cofactor synthase ( 5 -- 7 ) ( see Fig. 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23913318" target="_blank">23913318</a>]<br></li>
<li>The possibility that ModE also regulates another promoter that is transcribed in the opposite direction to the dmsA promoter was tested by cloning the promoter regions from lPM42 and lPM44 in the ` reverse orientation ' and measuring expression in MC4100 and PM8 ( modE : : kan ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466267" target="_blank">9466267</a>]<br></li>
<li>( These promoter alterations are in the 146 version of the napF operon control region , which contains the ModE protein binding site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=13129959" target="_blank">13129959</a>]<br></li>
<li>These data argue that , in vivo , ModE regulates expression of modABCD by binding the region of DNA that includes the CATAA box and also upstream sequences .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9044285" target="_blank">9044285</a>]<br></li>
<li>ModE protein is requiredfor expression of narXf operon The narXL operon encodes the two regulatory proteins needed for activation of the narG and narK operons ( Darwin & Stewart , 1996 ; Egan & Stewart , 1990 ; Kalman & Gunsalus , 1990 ; Schroder et al. , 1994 ; Walker & DeMoss , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10206709" target="_blank">10206709</a>]<br></li>
<li>Tao , H. , Hasona , A. , Do , P. , Ingram , L. , and Shanmugan , K. ( 2005 ) Global gene expression analysis revealed an unsuspected deo operon under the control of molydate sensor , ModE protein , in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19007420" target="_blank">19007420</a>]<br></li>
<li>Both FhlA126 and FhlA132 proteins activated the hyc operon even in the absence of ModE and MoeA , two components of Mo-metabolism which are required for hyc-lac expression in wild-type E. coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10689164" target="_blank">10689164</a>]<br></li>
<li>Although the mechanism by which the MoeA-mediated control of nar-lac expression is not known , these results show that it is independent of ModE protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9851041" target="_blank">9851041</a>]<br></li>
<li>These results suggest that the ModE protein is required for optimum expression of both hyc and n a r c operons .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10206709" target="_blank">10206709</a>]<br></li>
<li>These results show that both ModE and MoeA are needed for maximal expression of hyc operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9851041" target="_blank">9851041</a>]<br></li>
<li>The deo operon is another operon that is repressed by ModE and does not code for a molybdoenzyme or participate directly in Mo-metabolism in E. coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16205910" target="_blank">16205910</a>]<br></li>
<li>The E. coli enzyme is predicted to contain a molybdopterin moiety that raised the possibility tha napFDAGHBC operon expression may be ModE dependent .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12029041" target="_blank">12029041</a>]<br></li>
<li>J Bacteriol 145:1110 -- 1112 McNicholas PM , Gunsalus RP ( 2002 ) The molybdate-responsive Eschrichia coli ModE transcriptional regulator coordinates periplasmic nitrate reductase ( napFDAGHBC ) operon expression with nitrate and molybdate availability .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16205910" target="_blank">16205910</a>]<br></li>
<li>McNicholas , P.M. , and Gunsalus , R.P. ( 2002 ) The molyb-date-responsive Escherichia coli ModE transcriptional regulator coordinates periplasmic nitrate reductase ( napFDAGHBC ) operon expression with nitrate and molybdate availability .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16677314" target="_blank">16677314</a>]<br></li>
				<label class="TU_collapse_similarity" for="tu_similarity_29">
The molybdate-responsive Esche-richia coli ModE transcriptional regulator coordinates periplasmic nitrate reductase ( napFDAGHBC ) operon expression with nitrate and molybdate availability .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23913318" target="_blank">23913318</a>]<br></label>
<input id="tu_similarity_29", type="checkbox">
<div class="TU_similar">
<li>The molybdate-responsive Escherichia coli ModE transcriptional regulator coordinates periplasmic nitrate reductase ( napFDAGHBC ) operon expression with nitrate and molybdate availability .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=13129959" target="_blank">13129959</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19060147" target="_blank">19060147</a>]<br></li></div>
<li>ModE also positively regulates expression of the moaABCDE operon ( McNicholas et al. , 1997 ) , which encodes enzymes involved in molybdate utilization ( Rivers et al. , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466267" target="_blank">9466267</a>]<br></li>
<li>ModE also binds and regulates expression of the moaADCDE and dmsABC operons , which encode enzymes involved in the ﬁrst steps of molybdate assimilation into molybdopterin and the DMSO reductase , respectively ( 1 , 20 , 21 , 23 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12029041" target="_blank">12029041</a>]<br></li>
<li>Mutated FhlA proteins activate transcription of PhyclacZ in a modE , moeA double mutant The molybdate requirement for hyc expression is transduced through two proteins , ModE and MoeA and the hyc operon is not transcribed in a modE , moeA double mutant [ 12 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10689164" target="_blank">10689164</a>]<br></li>
<li>Tungstate substituted for molybdate in the activation of transcription of hyc and nar operons by ModE could not replace molybdate for MoeA-dependent expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9851041" target="_blank">9851041</a>]<br></li>
<li>The molybdate enhancement of moa transcription is fully dependent upon the molybdate-binding protein , ModE , which also mediates molybdate repression of the mod operon encoding the high-afﬁnity molybdate uptake system .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11092866" target="_blank">11092866</a>]<br></li>
<li>The ModE protein is also proposed as one of the molybdate sensors in the expression of the narGHJI , narK and n a r X L operons .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10206709" target="_blank">10206709</a>]<br></li>
<li>It is possible that the molybdate-bound form of ModE , the known repressor of the m o d A B C D operon , is also the active form participating in h y c and narG expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10206709" target="_blank">10206709</a>]<br></li>
<li>The mechanism by which the molybdate-responsive ModE protein controls napF operon expression has not been determined .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=13129959" target="_blank">13129959</a>]<br></li>
<li>The reduction of modABCD expression in the in vitro transcription-translation reaction mixtures containing ModE shows that ModE is a repressor of the modABCD operon , and this repression is enhanced by molybdate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8550508" target="_blank">8550508</a>]<br></li>
<li>The ModE protein was used in an in vitro transcription-translation experiment with the objective of determining whether expression of the modABCD operon is repressed in the presence of ModE and molybdate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8550508" target="_blank">8550508</a>]<br></li>
<li>The expression of the modABC operon is repressed by the molybdate-bound ModE transcription factor ( 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23913318" target="_blank">23913318</a>]<br></li>
<li>Finally , napF operon transcription is also controlled by the molybdate-re-sponsive regulator ModE ( 43 , 66 ) and by the nitric oxide-responsive repressor NsrR ( 27 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19060147" target="_blank">19060147</a>]<br></li>
<li>To determine if the molybdate-responsive transcription factor ModE is involved in this process , a set of dmsA -- lacZ operon fusions were constructed and analysed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466267" target="_blank">9466267</a>]<br></li>
<li>Utilizing a napF-lacZ operon fusion , we demonstrate that napF gene expression is impaired in strain defective for the molybdate-responsive ModE transcription factor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12029041" target="_blank">12029041</a>]<br></li>
<li>Haemophilus influenxae ModE can functionally replace E. coli ModE in the repression of E. coli modA-lacZ ( A. M. Grunden , unpublished ) and thus its apparent function in the native organism is also to repress m o d A B C operon expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10206709" target="_blank">10206709</a>]<br></li>
<li>Here we show that in the absence of ModE , expression of the napF operon in response to both nitrate and nitrite is dramatically reduced .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12029041" target="_blank">12029041</a>]<br></li>
<li>In this study , we establish that ModE is required for optimal expression of the dmsABC operon in response to anaerobiosis and for repression in response to nitrate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466267" target="_blank">9466267</a>]<br></li>
<li>In this study , we examine the role of ModE in regulating the expression of the napFDAGHBC operon of E. coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12029041" target="_blank">12029041</a>]<br></li>
<li>ModE activates expression of moaA in vivo To further investigate how ModE regulates moaA expression in vivo we introduced a moaA -- lacZ operon fusion into a modC modE strain ( PM6 ) and looked at the effect of expressing modE þ in trans in the presence and absence of molybdate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9044285" target="_blank">9044285</a>]<br></li>
<li>In vivo expression of a moaA -- lacZ operon fusion was stimulated twofold by ModE .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9044285" target="_blank">9044285</a>]<br></li>
<li>In contrast to its action at the modABCD operon , ModE functions as a modest activator of moaA expression in vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9044285" target="_blank">9044285</a>]<br></li>
<li>The repression is mediated by the ModE protein , which is encoded by a second operon expressed divergently from modABCD ( 7 , 20 , 25 , 38 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11092866" target="_blank">11092866</a>]<br></li>
<li>Therefore it is not surprising that the yecKbisZ operon might be regulated by ModE .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14641908" target="_blank">14641908</a>]<br></li>
<li>( 4 ) Repression of modA-lacZ by ModE only required molybdate or tungstate and was independent of moeA gene product .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9851041" target="_blank">9851041</a>]<br></li>
<li>In the 146 strain , which retains the ModE protein binding site , ( napF-lacZ ) expression was induced during growth with added molybdate , and this induction required the modE allele ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=13129959" target="_blank">13129959</a>]<br></li>
<li>By contrast , in the 123 strain , in which the ModE protein binding site is deleted , ( napF-lacZ ) expression was independent of both molybdate limitation and the modE allele ( data not shown ; Table 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=13129959" target="_blank">13129959</a>]<br></li>
<li>Even in the presence of pWS31 , which encodes native ModE protein , expression of Ph , , lacZ was completely molybdate-dependent ( Table 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10206709" target="_blank">10206709</a>]<br></li>
<li>When plasmids encoding these mutant ModE proteins were introduced into strain WS192 , P , , , lacZ expression was also independent of added molybdate ( Table 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10206709" target="_blank">10206709</a>]<br></li>
<li>In contract , morA ( ORF2 ) - lacZ fusions directed LacZ expression in the modE mutant ( Fig. 5 ) , indicating that morA ( ORF2 ) is expressed in the absence of repression by ModE .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23913318" target="_blank">23913318</a>]<br></li>
<li>Results ModE is required for regulation of dmsA -- lacZ expression in response to oxygen availability To determine if ModE plays a role in the molybdate-depen-dent regulation of the dmsABC operon , we constructed a dmsA -- lacZ operon fusion , lPM40 ( Fig. 1 ) , and introduced it into strain PM8 ( modE : : kan ) and its wild-type progenitor , MC4100 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466267" target="_blank">9466267</a>]<br></li>
<li>To determine if ModE plays a role in the regulated expression from the napF promoter , we measured expression from a napF-lacZ operon fusion , HW2 ( Materials and Methods ) ( Fig. 1 ) in both wild-type ( MC4100 ) and modE ( PM8 ) backgrounds .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12029041" target="_blank">12029041</a>]<br></li>
<li>Although the functions of ModD and ModF , and therefore the need for their further regulation by ModE , are unclear , f ( modF-lacZ ) expression levels were higher by approximately fourfold when the cells were grown in low-molybdate -- glucose -- minimal medium in comparison with levels for cells grown in molybdate-supple-mented medium , indicating molybdate-induced repression of modF expression ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8550508" target="_blank">8550508</a>]<br></li>
<li>ModE is required for nitrate-dependent repression of dmsA -- lacZ expression It has been shown previously that the addition of nitrate to anaerobically growing cells represses dmsA -- lacZ expression 10-fold , and that this repression is relieved in cells deﬁcient in molybdate uptake ( Pascal et al. , 1982 ; Iuchi and Lin , 1987 ; Cotter and Gunsalus , 1989 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466267" target="_blank">9466267</a>]<br></li>
<li>Additional regulators of dmsABC expression have also been described that involve molybdenum dependent gene expression via the ModE transcription factor , and integration host factor ( IHF ) that interacts with DNA at a downstream site called P2 [ 4,6,11,12 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12079504" target="_blank">12079504</a>]<br></li>
<li>As P2 also functions under anaerobic growth conditions and is derepressed in the absence of ModE , it is likely that , under these conditions , a signiﬁcant amount of dmsA expression , measured from the full-length construct ( lPM40 ) , originates from P2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466267" target="_blank">9466267</a>]<br></li>
<li>It seems highly likely that tungstate repression of modA is mediated by ModE in E. coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9210473" target="_blank">9210473</a>]<br></li>
<li>These two promoters , morA and morB , were found to be repressed by ModE , as in the case of the modA promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23913318" target="_blank">23913318</a>]<br></li>
				</div></ul></div><br>
	</div>
</body>
</html>