<!doctype html>
<style>
@charset "utf-8";
body {
	height: 100%;
	margin: 0;
	padding: 0;
}

.banner {
	position: fixed;
	top: 0;
	width: 100%;
	height: 65px;
	background: gray;
	line-height: 65px;
}
.spacer {
	height: 10px;
}

.main-content {
	padding: 5%;
}

.collapse_ACT {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-style: none solid none none;
	border-color: #B4045F;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_ACT + input{
	display: none;
}
.collapse_ACT + input + div{
	display: none;
}
.collapse_ACT + input:checked + div{
	display: block;
}
.ACT {
	text-align: justify;
	text-justify: inter-word;
}
.ACT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B4045F;
}
.ACT_collapse_similarity:hover{
	background: #e5d3dc;
}
.ACT_collapse_similarity + input {
	display: none;
}
.ACT_collapse_similarity + input + div{
	display: none;
}

.ACT_collapse_similarity + input:checked + div{
	display: block;
}
.ACT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.ACT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B4045F;
	padding-left: 5px;
}
.ACT_collapse_similarity:hover{
	background: #e5d3dc;
}
.ACT_collapse_similarity + input {
	display: none;
}
.ACT_collapse_similarity + input + div{
	display: none;
}
.ACT_collapse_similarity + input:checked + div{
	display: block;
}
.ACT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.ACT_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #B4045F;
}
.ACT_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_DOM {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-style: none solid none none;
	border-color: #B404AE;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_DOM + input{
	display: none;
}
.collapse_DOM + input + div{
	display: none;
}
.collapse_DOM + input:checked + div{
	display: block;
}
.DOM {
	text-align: justify;
	text-justify: inter-word;
}
.DOM_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B404AE;
	padding-left: 5px;
}
.DOM_collapse_similarity:hover{
	background: #e8d7e7;
}
.DOM_collapse_similarity + input {
	display: none;
}
.DOM_collapse_similarity + input + div{
	display: none;
}
.DOM_collapse_similarity + input:checked + div{
	display: block;
}
.DOM_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}.DOM_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #B404AE;
}
.DOM_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_EVO {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #7401DF;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_EVO + input{
	display: none;
}
.collapse_EVO + input + div{
	display: none;
}
.collapse_EVO + input:checked + div{
	display: block;
}
.EVO {
	text-align: justify;
	text-justify: inter-word;
}
.EVO_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #7401DF;
	padding-left: 5px;
}
.EVO_collapse_similarity:hover{
	background: #e2d7ed;
}
.EVO_collapse_similarity + input {
	display: none;
}
.EVO_collapse_similarity + input + div{
	display: none;
}
.EVO_collapse_similarity + input:checked + div{
	display: block;
}
.EVO_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.EVO_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #7401DF;
}
.EVO_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_RP {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #0174DF;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_RP + input{
	display: none;
}
.collapse_RP + input + div{
	display: none;
}
.collapse_RP + input:checked + div{
	display: block;
}
.RP {
	text-align: justify;
	text-justify: inter-word;
}
.RP_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #0174DF;
	padding-left: 5px;
}
.RP_collapse_similarity:hover{
	background: #c9d7e2;
}
.RP_collapse_similarity + input {
	display: none;
}
.RP_collapse_similarity + input + div{
	display: none;
}
.RP_collapse_similarity + input:checked + div{
	display: block;
}
.RP_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.RP_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #0174DF;
}
.RP_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_SIT {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #0489B1;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_SIT + input{
	display: none;
}
.collapse_SIT + input + div{
	display: none;
}
.collapse_SIT + input:checked + div{
	display: block;
}
.SIT {
	text-align: justify;
	text-justify: inter-word;
}
.SIT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #0489B1;
	padding-left: 5px;
}
.SIT_collapse_similarity:hover{
	background: #d0dfe2;
}
.SIT_collapse_similarity + input {
	display: none;
}
.SIT_collapse_similarity + input + div{
	display: none;
}
.SIT_collapse_similarity + input:checked + div{
	display: block;
}
.SIT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.SIT_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #0489B1;
}
.SIT_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_TU {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #04B486;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_TU + input{
	display: none;
}
.collapse_TU + input + div{
	display: none;
}
.collapse_TU + input:checked + div{
	display: block;
}
.TU {
	text-align: justify;
	text-justify: inter-word;
}
.TU_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #04B486;
	padding-left: 5px;
}
.TU_collapse_similarity:hover{
	background: #d0e2de;
}
.TU_collapse_similarity + input {
	display: none;
}
.TU_collapse_similarity + input + div{
	display: none;
}
.TU_collapse_similarity + input:checked + div{
	display: block;
}
.TU_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.TU_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #04B486;
}
.TU_similar > a {
	text-decoration: none;
	color: gray;
}

ul { 
	list-style-type: none;
	padding: 0;
	margin: 0;
}
ul li > a {
	text-decoration: none;
	color: gray;
}
ul li:hover, ul li:focus {
	background: #F0F0F0;
}

ul li:first-child span {
  border-left:none;  
}

</style>
<html>
<head>
<meta charset="utf-8">
<title>TF properties - MraZ</title>
<meta name="theme-color" content="#ffffff">
</head>
<body>
	<div class="banner">
		<center><font color="white" size="16px" face="arial">MraZ</font></center>
	</div>
	<div align="center" class="main-content">
		<div class="spacer">
	</div><br>			<label class="collapse_ACT" for="_1"><font size="5px">MraZ transcriptional activity</font></label>
				<input id="_1" type="checkbox">
				<div class="ACT"><br><ul><li>Interestingly , the effect of mutation of the DRs is additive ; the lacZ fusion to the O1 operator , in which the middle DR ( DR2 ) is mutated , was still repressed , but the lacZ fusion to the O4 mutant operator , in which all of the DRs are mutated , was totally derepressed , irrespective of whether MraZ was overproduced or not .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24659771" target="_blank">24659771</a>]<br></li>
<li>As shown in Fig. 6A , increasing concentrations of WT MraZ led to the increased formation of two stable DNA-protein complexes in the presence of the noncompetitive inhibi-tor poly ( dI · dC ) and a concomitant decrease in the amount of free DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24659771" target="_blank">24659771</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_3">
( A ) Binding of increasing concentrations of WT MraZ to biotinylated OWT in EMSAs .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24659771" target="_blank">24659771</a>]<br></label>
<input id="act_similarity_3", type="checkbox">
<div class="ACT_similar">
<li>( B ) Binding of increasing concentrations of WT MraZ to biotinylated mioC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24659771" target="_blank">24659771</a>]<br></li></div>
<li>Cells containing the pDSW208-mraZ plasmid in minimal medium supplemented with glycerol as the sole carbon source ﬁlamented and formed bulges ( data not shown ) even without IPTG induction , suggesting that excess MraZ is more toxic when cells are grown in minimal medium than when grown in LB ( see Fig .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24659771" target="_blank">24659771</a>]<br></li>
<li>We then measured - galactosidase production in response to excess MraZ produced from a compatible plasmid , pDSW208-mraZ , induced with a low , nonlethal level of IPTG ( Fig. 4B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24659771" target="_blank">24659771</a>]<br></li>
<li>The stability of the MraZR15A mutant protein is similar to that of WT MraZ under uninduced and induced conditions ( Fig. 5C ) , suggesting that the mutation inactivates a speciﬁc function of MraZ .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24659771" target="_blank">24659771</a>]<br></li>
<li>We calculate that there was an 10-fold increase in protein concentration in induced versus uninduced cells , with the caveat that MraZ was lethal to cells at this concentration of IPTG .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24659771" target="_blank">24659771</a>]<br></li>
<li>If MraZ indeed binds to DNA as a dodeca-mer , it might wrap the DNA around its toroidal structure similar to the E. coli transcriptional-repressor RcnR ( 65 ) and DNA gyrase ( 66 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24659771" target="_blank">24659771</a>]<br></li>
<li>We used functional enrichment analysis to identify pathways and functions that were biased toward differential regulation by MraZ .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24659771" target="_blank">24659771</a>]<br></li>
<li>As co-overproduction of MraW protects cells from the effects of MraZ overproduction in vivo and mraZ and mraW are usually a tightly linked gene pair , it is reasonable to assume that this effect is direct , although additional data are needed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24659771" target="_blank">24659771</a>]<br></li>
<li>Trimethoprim inhibits DHFR and one-carbon metabo-lism , which might be involved in the function of the MraW meth-ylase , potentially impacting MraZ activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24659771" target="_blank">24659771</a>]<br></li>
<li>Although they do not affect dcw transcription in stationary phase , it is important to emphasize that physiological levels of MraZ do repress the transcription of arginine and polyamine ca-tabolism , the ast and puu genes , respectively , and activate other genes encoding proteins involved in translation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24659771" target="_blank">24659771</a>]<br></li>
<li>This regulation might be direct or indirect , but it is clear that MraZ can activate ( Fig. 7A , white bars ) or repress ( Fig. 7A , black bars ) transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24659771" target="_blank">24659771</a>]<br></li>
<li>This indicates that increases in polyamine concentration and some regulation of translation in stationary phase might be a consequence of MraZ activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24659771" target="_blank">24659771</a>]<br></li>
<li>Our results suggest that cell ﬁlamentation and killing are directly correlated with increased cellular levels of MraZ .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24659771" target="_blank">24659771</a>]<br></li>
<li>Consistent with this , when expressed from the more tightly repressed arabi-nose-inducible promoter in pBAD33 , ﬁlamentation increased as MraZ was overproduced by induction with arabinose , but there was no killing ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24659771" target="_blank">24659771</a>]<br></li>
<li>Here we show that overproduction of Escherichia coli MraZ inhibited cell division and was lethal in rich medium at high induction levels and in minimal medium at low induction levels .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24659771" target="_blank">24659771</a>]<br></li>
<li>Such repression by MraZ might limit the amount of some essential cell division and/or PG biosynthesis proteins , and this is the likely cause of cell division inhibition after the overproduction of MraZ .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24659771" target="_blank">24659771</a>]<br></li>
<li>A total of 69 genes were activated by MraZ , and 31 were repressed ( see Table S4 in the supplemental material ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24659771" target="_blank">24659771</a>]<br></li>
<li>In addition , of the 69 genes activated by MraZ , 13 encode proteins involved in translation ( see Table S4 ; highlighted in yellow ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24659771" target="_blank">24659771</a>]<br></li>
<li>One gene selected this way was mioC , which , according to the RNA-seq data , is negatively regulated by MraZ overproduction by as much as 12-fold during early log phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24659771" target="_blank">24659771</a>]<br></li>
<li>MraZ activates the expression of both fadB and fadE when overexpressed early in the highly aerobic log phase , when ArcA is not phosphorylated and is inactive as a repressor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24659771" target="_blank">24659771</a>]<br></li>
<li>In stationary phase , however , ArcA-P represses both fadB and fadE , which could prevent activation by MraZ .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24659771" target="_blank">24659771</a>]<br></li>
<li>In support of this , puriﬁed MraZ could directly bind to a putative operator site upstream of mioC , one of the repressed genes identiﬁed by RNA-seq .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24659771" target="_blank">24659771</a>]<br></li>
<li>Puriﬁed MraZ binds directly and speciﬁcally to its operator sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24659771" target="_blank">24659771</a>]<br></li>
<li>Thus , puriﬁed WT MraZ binds speciﬁcally to OWT in vitro , and the toxicity observed upon its overproduction is directly related to its ability to-bind to its DNA site ( s ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24659771" target="_blank">24659771</a>]<br></li>
<li>These results strongly suggest that MraZ represses its own promoter , Pmra , and that the MraW negative modulatory effect on MraZ described above leads to derepression of Pmra expression , possibly by preventing DNA binding by MraZ .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24659771" target="_blank">24659771</a>]<br></li>
<li>Finally , several lines of evidence , including MraZ localization throughout the nucleoid , activation or repression of multiple genes by RNA-seq , and in vitro binding to the mioC gene near oriC , suggest that MraZ may also regulate the expression of genes outside the dcw cluster and perhaps inﬂuence other cell cycle functions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24659771" target="_blank">24659771</a>]<br></li>
<li>As MraZ binds to the operator site in front of its own gene and represses expression from the Pmra promoter , we wanted to determine how many dcw cluster-proximal genes are affected in their expression by MraZ binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24659771" target="_blank">24659771</a>]<br></li>
<li>We also provide evidence that MraZ autoregulates its own expression from the Pmra promoter and can potentially repress the ﬁrst 11 genes in the dcw cluster .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24659771" target="_blank">24659771</a>]<br></li>
<li>Notably , the RNA levels of the ﬁrst 11 genes of the dcw cluster are signiﬁcantly decreased , as expected from repression of Pmra by MraZ .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24659771" target="_blank">24659771</a>]<br></li>
<li>MraZ , at physiological levels , has the potential to autoregulate its expression and that of the ﬁrst 11 genes in the dcw cluster , including mraW , by binding to the OWT operator at Pmra .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24659771" target="_blank">24659771</a>]<br></li>
<li>Thus , MraZ represses the expression of Pmra and the mraZ gene and the DRs constitute DNA elements necessary for this repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24659771" target="_blank">24659771</a>]<br></li>
<li>Any of these scenarios might result in more mraW gene expression because of the loss of MraZ repression at Pmra , which would induce a positive feedback loop .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24659771" target="_blank">24659771</a>]<br></li>
<li>MraZ repression of Pmra expression , as inferred from our data ( Fig. 4 to 6 ) , suggests that MraZ can potentially regulate genes for cell division and PG metabolism .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24659771" target="_blank">24659771</a>]<br></li>
<li>MraZ negatively regulates its own expression from Pmra .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24659771" target="_blank">24659771</a>]<br></li>
<li>As both AbrB and MazE negatively regulate their own expression ( 50 , 51 ) , we decided to test whether MraZ does the same by repressing transcription from Pmra .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24659771" target="_blank">24659771</a>]<br></li>
				</div></ul><br>
			<label class="collapse_DOM" for="_2"><font size="5px">MraZ protein domains</font></label>
				<input id="_2" type="checkbox">
				<div class="DOM"><br><ul><li>We deﬁned three MraZ consensus DRs as DR1 , DR2 and DR3 , where DR2 contains one mismatch .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24659771" target="_blank">24659771</a>]<br></li>
<li>Alternatively , MraZ may assemble into polymers of dimers similar to MazE ( 67 ) or ArcA-P , the latter being able to occupy two , three , and even four consecutive DR sequences separated by variable spacer regions , all located on one side of the helix ( 68 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24659771" target="_blank">24659771</a>]<br></li>
<li>AbrB and MraZ lack formal β sheet contacts in the β1 -- β2 hairpin , meaning that they must be considered pseudobarrels .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15939023" target="_blank">15939023</a>]<br></li>
<li>The N-terminal end of MraZ , encompassing approximately 45 resi-dues , is similar to the N-terminal DNA-binding domains of the transition state regulator AbrB from Bacillus subtilis ( 14 , 15 ) and the antidote protein of the MazE/F addiction module , MazE , from E. coli ( 16 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24659771" target="_blank">24659771</a>]<br></li>
<li>Because of the similarity between the N-terminal domain of MraZ and the N-terminal DNA-binding domains of AbrB and MazE , we decided to investigate whether MraZ binds DNA , perhaps acting as a transcriptional regulator of dcw genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24659771" target="_blank">24659771</a>]<br></li>
<li>A point mutation in the putative DNA-binding domain of MraZ eliminates its toxicity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24659771" target="_blank">24659771</a>]<br></li>
<li>AbrB , which integrates environmental and metabolic information to minimize inappropriate gene expression during log phase ( 70 ) , shows similarity to MraZ in its `` looped-hinge helix fold '' DNA-binding domain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24659771" target="_blank">24659771</a>]<br></li>
<li>MraZ contains two tandem homologous copies of the UPF0040 fold , a novel protein fold with no signiﬁcant similarity to any other proteins whose three-dimensional structures are known ( 19 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24659771" target="_blank">24659771</a>]<br></li>
<li>R15 of MraZ forms part of the ﬁrst DXXXR DNA-binding motif of MraZ present in the two tandem homologous copies of the UPF0040 fold ( Fig. 5A ) ( 6 , 18 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24659771" target="_blank">24659771</a>]<br></li>
<li>Structurally , MraZ takes a special place among AbrB homologs in that the two subunits needed to form the fold are fused here into a single chain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15939023" target="_blank">15939023</a>]<br></li>
<li>AbrB shares with MazE the β turn structure over the GD box residues typical of the fold family , while MraZ lacks both of these residues and the β turn .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15939023" target="_blank">15939023</a>]<br></li>
<li>The helical hairpin appears after each half of MraZ , and the two halves are nearest neighbors in the cluster map , suggesting that the protein originated through duplication and fusion of an ancestral homodimeric MraZ , which already contained the helical hairpin at its C terminus .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15939023" target="_blank">15939023</a>]<br></li>
<li>This motif is only present in the first half of MraZ , while , in the second half and in MazE , a large hydrophobic residue replaces the arginine ( I and M , respectively ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15939023" target="_blank">15939023</a>]<br></li>
<li>This is seen particularly clearly in the MraZ clusters , where the After convergence of the sequence searches , the obtained sedomain cluster is much more irregular than the C domain cluster .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15939023" target="_blank">15939023</a>]<br></li>
				</div></ul><br>
			<label class="collapse_EVO" for="_3"><font size="5px">MraZ evolution</font></label>
				<input id="_3" type="checkbox">
				<div class="EVO"><br><ul><li>MraZ from E. coli ( MraZEc ) and Mycoplasma pneumoniae ( MraZMp ) , which share 28 % sequence identity , have both been crystallized ( 6 , 18 ) , and the solved structures are unusual .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24659771" target="_blank">24659771</a>]<br></li>
<li>MraZ contains two tandem homologous copies of the UPF0040 fold , a novel protein fold with no signiﬁcant similarity to any other proteins whose three-dimensional structures are known ( 19 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24659771" target="_blank">24659771</a>]<br></li>
				</div></ul><br>
			<label class="collapse_RP" for="_4"><font size="5px">Processes regulated by MraZ</font></label>
				<input id="_4" type="checkbox">
				<div class="RP"><br><ul><li>'' To understand this discrepancy , we undertook a bio-informatics study of AbrB and its homologs ; these form a large superfamily , which includes SpoVT , PrlF , MraZ , addiction module antidotes ( PemI , MazE ) , plasmid-maintenance proteins ( VagC , VapB ) , and archaeal PhoU homologs .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15939023" target="_blank">15939023</a>]<br></li>
<li>We indeed found new evidence of a role for MraZ as a transcriptional regulator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24659771" target="_blank">24659771</a>]<br></li>
<li>In summary , here we show that MraZ , the product of the highly conserved mraZ gene , acts as a transcriptional regulator in E. coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24659771" target="_blank">24659771</a>]<br></li>
<li>We used functional enrichment analysis to identify pathways and functions that were biased toward differential regulation by MraZ .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24659771" target="_blank">24659771</a>]<br></li>
<li>MraZ from Escherichia coli : cloning , puriﬁcation , crystallization and preliminary X-ray analysis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24659771" target="_blank">24659771</a>]<br></li>
<li>Although they do not affect dcw transcription in stationary phase , it is important to emphasize that physiological levels of MraZ do repress the transcription of arginine and polyamine ca-tabolism , the ast and puu genes , respectively , and activate other genes encoding proteins involved in translation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24659771" target="_blank">24659771</a>]<br></li>
<li>In addition , of the 69 genes activated by MraZ , 13 encode proteins involved in translation ( see Table S4 ; highlighted in yellow ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24659771" target="_blank">24659771</a>]<br></li>
<li>In addition to the synthetic effects of combining ftsZ and min mutants with excess MraZ described above , we found two sets of genes involved in cell-division that were repressed to various extents by MraZ overproduction during growth in early logarithmic phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24659771" target="_blank">24659771</a>]<br></li>
<li>Excess MraZ reduced the expression of an mraZ-lacZ reporter , suggesting that MraZ acts as a repressor of Pmra , whereas a DNA-binding mutant form of MraZ failed to repress expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24659771" target="_blank">24659771</a>]<br></li>
<li>Such repression by MraZ might limit the amount of some essential cell-division and/or PG biosynthesis proteins , and this is the likely cause of cell-division inhibition after the overproduction of MraZ .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24659771" target="_blank">24659771</a>]<br></li>
<li>Transcriptome sequencing ( RNA-seq ) analysis suggested that MraZ also regulates the expression of genes outside the dcw cluster .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24659771" target="_blank">24659771</a>]<br></li>
<li>Finally , several lines of evidence , including MraZ localization throughout the nucleoid , activation or repression of multiple genes by RNA-seq , and in vitro binding to the mioC gene near oriC , suggest that MraZ may also regulate the expression of genes outside the dcw cluster and perhaps inﬂuence other cell-cycle functions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24659771" target="_blank">24659771</a>]<br></li>
<li>MraZ negatively regulates its own expression from Pmra .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24659771" target="_blank">24659771</a>]<br></li>
<li>MraZ repression of Pmra expression , as inferred from our data ( Fig. 4 to 6 ) , suggests that MraZ can potentially regulate genes for cell-division and PG metabolism .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24659771" target="_blank">24659771</a>]<br></li>
<li>Our RNA-seq data showed that MraZ overproduction represses cra transcription by 4.4-fold , either directly or indirectly , and as would be expected , upregulates TCA-cycle genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24659771" target="_blank">24659771</a>]<br></li>
<li>Thus , altered expression of TCA-cycle genes , which has been shown to involve other regulators ( 73 ) , might be partly responsible for the sensitivity to MraZ overproduction under these particular growth conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24659771" target="_blank">24659771</a>]<br></li>
				</div></ul><br>
			<label class="collapse_SIT" for="_5"><font size="5px">MraZ binding site</font></label>
				<input id="_5" type="checkbox">
				<div class="SIT"><br><ul><li>TABLE 1 Bacterial strains and plasmids used in this study Source or reference Strain or plasmid E. coli strains CGSC 8019 CGSC 8021 DH5 DH5 phe JE6478 JE6430 JE6513 JE6517 JE6583 JE6598 JE6599 JE6627 JE6628 JE6647 JE6648 JE6661 JE6707 JE6708 JE6710 JE6711 JE6713 JE6714 JE7023 JE7025 JE7031 JE7032 JE7035 JE7108 MG1655 MG1655 lacU169 Genotype or phenotype F mraZ52 lacL fnr267 rph-1 F mraW54 lacL fnr267 rph-1 40 40 F 80dlacZ M15 ( lacZYA-argF ) U169 recA1 endA1 hsdR17 ( r m K K ) supE44 F 80dlacZ M15 ( lacZYA-argF ) U169 recA1 endA1 hsdR17 ( rK mK ) supE44 MG1655/pJE6448 Apr MG1655/pDSW208 Apr MG1655/pJE6496 Apr MG1655/pJE6500 Apr MG1655/pJE6584 Cmr WM2909/pRK415 Tetr WM2909/pJE6595 Tetr MG1655 lacU169/pDSW208/pJE6618 Apr Tetr MG1655 lacU169/pJE6500/pJE6618 Apr Tetr MG1655 lacU169/pDSW208/pJE6621 Apr Tetr MG1655 lacU169/pJE6500/pJE6621 Apr Tetr MG1655/pJE6653 Apr MG1655 lacU169/pDSW208/pJE6695 Apr Tetr MG1655 lacU169/pJE6500/pJE6695 Apr Tetr MG1655 lacU169/pDSW208/pJE6696 Apr Tetr MG1655 lacU169/pJE6500/pJE6696 Apr Tetr MG1655 lacU169/pDSW208/pJE6697 Apr Tetr MG1655 lacU169/pJE6500/pJE6697 Apr Tetr MG1655 mraZ52 MG1655 leuO : : tet P1 ( CGSC8019 ) Tets prototroph MG1655 mraW54 MG1655 leuO : : tet P1 ( CGSC8021 ) Tets prototroph MG1655 mraZ52/pDSW208 MG1655 mraZ52/pJE6500 MG1655 mraW54/pDSW208 MG1655 lacU169/pJE6640/pJE6618 Apr Tetr Sequenced and F derivative of K-12 lac mutant derivative of MG1655 thi-1 gyrA relA1 76 77 This study This study This study This study This study This study This study This study This study This study This study This study This study This study This study This study This study This study This study This study This study This study This study This study 78 Laboratory strain 79 Laboratory strain thi-1 gyrA relA1 phe : : Tn10dCm W3110 with native ftsZ Ptrc-ftsZ-gfp at att site W3110 ftsL : : kan with Ts plasmid pTSA29-ftsL Kanr Apr WM2724 WM2909 Plasmids pBAD33 pACYC184 vector with P r BAD promoter ; Cm 80 Apr , with T3 and T7 promoters Ptrc promoter vector ; Apr pDSW208 EcoRI-HindIII with mraZ-gfp ; Apr pDSW208 EcoRI-HindIII with mraW-ﬂag ; Apr pDSW208 EcoRI-HindIII with mraZ-gluglu ; Apr pBAD33 KpnI-HindIII with mraZ-gluglu ; Cmr pRK415 BamHI-KpnI with 2,700-bp PCR fragment encompassing 563 bp upstream of mraZ to downstream ftsL pBSII SmaI with same 2,700-bp PCR fragment as pJE6595 but with MraZR15A mutation WT mraZ : : lacZ translational fusion vector ; pUI523A : :593 - bp PCR fragment from mraZ ( 563 bp upstream and 30 bp within gene ) ; Tetr O1 mraZ : : lacZ translational fusion in pJE6618 with TGGGG-to-ATCGG mutation in DR2 ; Tetr pDSW208 EcoRI-HindIII with mraZR15A-gluglu ; Apr O2 mraZ : : lacZ fusion in pJE6618 with TGGGA-to-AGGCA mutation in DR1 and TGGGG to ATCGG mutation in DR2 ; Tetr O3 mraZ : : lacZ fusion in pJE6618 with TGGGG-to-ATCGG mutation in DR2 and TGGGA to ATCGA mutation in DR3 ; Tetr O4 mraZ : : lacZ fusion in pJE6618 with TGGGA-to-AGGCA mutation in DR1 , TGGGG-to-ATCGG mutation in DR2 , and TGGGA-to-ATCGA mutation in DR3 ; Tetr pDSW208 EcoRI-HindIII with mraZ-mraW ; Apr pDSW208 NcoI-HindIII with his-mraZ-gluglu ; Apr pDSW208 NcoI-HindIII with his-mraZR15A-gluglu ; Apr pBAD33 KpnI-HindIII with mioC Low-copy-number IncP plasmid ; Tetr Promoterless lacZ translational fusion vector ; Tetr Stratagene 81 This study This study This study This study This study This study This study pBlueScriptII ( pBSII ) pDSW208 pJE6448 pJE6496 pJE6500 pJE6584 pJE6595 pJE6617 pJE6618 pJE6621 pJE6640 pJE6695 This study This study This study pJE6696 This study pJE6697 This study pJE6653 pJE6761 pJE6762 pJE7145 pRK415 pUI523A This study This study This study This study 82 38 RESULTS Phenotypic analysis of cells after loss of MraZ or MraW .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24659771" target="_blank">24659771</a>]<br></li>
<li>Therefore , MraZ might use a similar strategy for DNA recognition , allowing it to bind unrelated DNA sequences in a speciﬁc manner .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24659771" target="_blank">24659771</a>]<br></li>
<li>Nevertheless , addition of unlabeled competitor mioC DNA at an approximately 10-fold molar excess displaced MraZ from MraZ-mioC DNA complexes , suggesting that MraZ binds to the regulatory region of mioC speciﬁcally .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24659771" target="_blank">24659771</a>]<br></li>
<li>Puriﬁed MraZ binds directly and speciﬁcally to its operator sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24659771" target="_blank">24659771</a>]<br></li>
<li>We then used EMSAs to test whether MraZ binds directly to this region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24659771" target="_blank">24659771</a>]<br></li>
<li>Consistent with this idea , puriﬁed MraZ directly bound a region of DNA containing three direct repeats between Pmra and the mraZ gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24659771" target="_blank">24659771</a>]<br></li>
<li>To demonstrate directly that MraZ binds to the regulatory region near Pmra , we puriﬁed His6-MraZ-GluGlu and His6-MraZR15A-GluGlu to homogeneity and measured their binding to a 240-bp PCR-ampliﬁed DNA fragment containing the OWT DRs by EMSAs .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24659771" target="_blank">24659771</a>]<br></li>
<li>As this repeat region is located 8 nt downstream from the Pmra promoter ( Fig. 4A ) , it is consistent with the idea that these DRs constitute an operator site for MraZ binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24659771" target="_blank">24659771</a>]<br></li>
<li>In support of this , puriﬁed MraZ could directly bind to a putative operator site upstream of mioC , one of the repressed genes identiﬁed by RNA-seq .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24659771" target="_blank">24659771</a>]<br></li>
<li>Finally , we characterize a second MraZ binding site upstream of mioC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24659771" target="_blank">24659771</a>]<br></li>
				</ul></div><br>
			<label class="collapse_TU" for="_6"><font size="5px">Transcription units regulated by MraZ</font></label>
				<input id="_6" type="checkbox">
				<div class="TU"><br><ul><li>Although they do not affect dcw transcription in stationary phase , it is important to emphasize that physiological levels of MraZ do repress the transcription of arginine and polyamine ca-tabolism , the ast and puu genes , respectively , and activate other genes encoding proteins involved in translation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24659771" target="_blank">24659771</a>]<br></li>
<li>Those genes were puuA ( 4.3-fold ) , puuB ( 6.5-fold ) , puuD ( 3.0-fold ) , and puuP ( 2.4-fold ) ; in contrast , puuA ( 6.4-fold ) , puuB ( 7.8-fold ) , puuC ( 3.0-fold ) , puuD ( 8.1-fold ) , puuE ( 11.2-fold ) , and puuR ( 4.4-fold ) were activated by MraZ overproduction at low cell density .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24659771" target="_blank">24659771</a>]<br></li>
<li>In contrast to the activator role suggested for MraZ early in log phase ( see below ) , the fatty acid ( FA ) degradation operon fadBA and the monocistronic genes fadD and fadE were repressed 2-fold by MraZ .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24659771" target="_blank">24659771</a>]<br></li>
<li>The numbers of genes activated ( white columns ) or repressed ( black columns ) by MraZ are shown ; they were binned by the fold changes in their expression ( top ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24659771" target="_blank">24659771</a>]<br></li>
<li>When we compared the expression of genes in the WT strain with that in the mraZ null mutant , both grown to stationary phase , we found that the arginine catabolism pathway was transcriptionally repressed by physiological levels of MraZ ( see Table S4 in the supplemental material ; values in parentheses are fold changes in expression ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24659771" target="_blank">24659771</a>]<br></li>
<li>A total of 69 genes were activated by MraZ , and 31 were repressed ( see Table S4 in the supplemental material ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24659771" target="_blank">24659771</a>]<br></li>
<li>The speciﬁc genes repressed by MraZ were astA ( 3.5-fold ) , astB ( 2.3-fold ) , astC ( 4.7-fold ) , and astD ( 3.2-fold ) ( see Table S4 ; highlighted in purple ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24659771" target="_blank">24659771</a>]<br></li>
<li>MraZ activates the expression of both fadB and fadE when overexpressed early in the highly aerobic log phase , when ArcA is not phosphorylated and is inactive as a repressor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24659771" target="_blank">24659771</a>]<br></li>
<li>In stationary phase , however , ArcA-P represses both fadB and fadE , which could prevent activation by MraZ .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24659771" target="_blank">24659771</a>]<br></li>
<li>If MraZ is a transcriptional regulator , then the killing phenotype observed upon its overproduction is most likely caused by inappropriate regulation of gene expression at the OWT operator located upstream of mraZ and/or at additional genomic targets .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24659771" target="_blank">24659771</a>]<br></li>
<li>MraZ negatively regulates its own expression from Pmra .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24659771" target="_blank">24659771</a>]<br></li>
<li>As both AbrB and MazE negatively regulate their own expression ( 50 , 51 ) , we decided to test whether MraZ does the same by repressing transcription from Pmra .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24659771" target="_blank">24659771</a>]<br></li>
<li>We also provide evidence that MraZ autoregulates its own expression from the Pmra promoter and can potentially repress the ﬁrst 11 genes in the dcw cluster .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24659771" target="_blank">24659771</a>]<br></li>
<li>MraZ , at physiological levels , has the potential to autoregulate its expression and that of the ﬁrst 11 genes in the dcw cluster , including mraW , by binding to the OWT operator at Pmra .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24659771" target="_blank">24659771</a>]<br></li>
<li>As MraZ binds to the operator site in front of its own gene and represses expression from the Pmra promoter , we wanted to determine how many dcw cluster-proximal genes are affected in their expression by MraZ binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24659771" target="_blank">24659771</a>]<br></li>
<li>By binding to a DNA sequence organized with different combinations of TGGGN DRs located immediately downstream from the Pmra promoter , MraZ represses its own expression and that of the 10 subsequent genes in the proximal part of the dcw cluster .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24659771" target="_blank">24659771</a>]<br></li>
<li>These results strongly suggest that MraZ represses its own promoter , Pmra , and that the MraW negative modulatory effect on MraZ described above leads to derepression of Pmra expression , possibly by preventing DNA binding by MraZ .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24659771" target="_blank">24659771</a>]<br></li>
<li>Thus , MraZ represses the expression of Pmra and the mraZ gene and the DRs constitute DNA elements necessary for this repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24659771" target="_blank">24659771</a>]<br></li>
				</div></ul></div><br>
	</div>
</body>
</html>