<!doctype html>
<style>
@charset "utf-8";
body {
	height: 100%;
	margin: 0;
	padding: 0;
}

.banner {
	position: fixed;
	top: 0;
	width: 100%;
	height: 65px;
	background: gray;
	line-height: 65px;
}
.spacer {
	height: 10px;
}

.main-content {
	padding: 5%;
}

.collapse_ACT {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-style: none solid none none;
	border-color: #B4045F;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_ACT + input{
	display: none;
}
.collapse_ACT + input + div{
	display: none;
}
.collapse_ACT + input:checked + div{
	display: block;
}
.ACT {
	text-align: justify;
	text-justify: inter-word;
}
.ACT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B4045F;
}
.ACT_collapse_similarity:hover{
	background: #e5d3dc;
}
.ACT_collapse_similarity + input {
	display: none;
}
.ACT_collapse_similarity + input + div{
	display: none;
}

.ACT_collapse_similarity + input:checked + div{
	display: block;
}
.ACT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.ACT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B4045F;
	padding-left: 5px;
}
.ACT_collapse_similarity:hover{
	background: #e5d3dc;
}
.ACT_collapse_similarity + input {
	display: none;
}
.ACT_collapse_similarity + input + div{
	display: none;
}
.ACT_collapse_similarity + input:checked + div{
	display: block;
}
.ACT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.ACT_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #B4045F;
}
.ACT_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_DOM {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-style: none solid none none;
	border-color: #B404AE;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_DOM + input{
	display: none;
}
.collapse_DOM + input + div{
	display: none;
}
.collapse_DOM + input:checked + div{
	display: block;
}
.DOM {
	text-align: justify;
	text-justify: inter-word;
}
.DOM_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B404AE;
	padding-left: 5px;
}
.DOM_collapse_similarity:hover{
	background: #e8d7e7;
}
.DOM_collapse_similarity + input {
	display: none;
}
.DOM_collapse_similarity + input + div{
	display: none;
}
.DOM_collapse_similarity + input:checked + div{
	display: block;
}
.DOM_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}.DOM_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #B404AE;
}
.DOM_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_EVO {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #7401DF;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_EVO + input{
	display: none;
}
.collapse_EVO + input + div{
	display: none;
}
.collapse_EVO + input:checked + div{
	display: block;
}
.EVO {
	text-align: justify;
	text-justify: inter-word;
}
.EVO_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #7401DF;
	padding-left: 5px;
}
.EVO_collapse_similarity:hover{
	background: #e2d7ed;
}
.EVO_collapse_similarity + input {
	display: none;
}
.EVO_collapse_similarity + input + div{
	display: none;
}
.EVO_collapse_similarity + input:checked + div{
	display: block;
}
.EVO_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.EVO_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #7401DF;
}
.EVO_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_RP {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #0174DF;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_RP + input{
	display: none;
}
.collapse_RP + input + div{
	display: none;
}
.collapse_RP + input:checked + div{
	display: block;
}
.RP {
	text-align: justify;
	text-justify: inter-word;
}
.RP_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #0174DF;
	padding-left: 5px;
}
.RP_collapse_similarity:hover{
	background: #c9d7e2;
}
.RP_collapse_similarity + input {
	display: none;
}
.RP_collapse_similarity + input + div{
	display: none;
}
.RP_collapse_similarity + input:checked + div{
	display: block;
}
.RP_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.RP_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #0174DF;
}
.RP_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_SIT {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #0489B1;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_SIT + input{
	display: none;
}
.collapse_SIT + input + div{
	display: none;
}
.collapse_SIT + input:checked + div{
	display: block;
}
.SIT {
	text-align: justify;
	text-justify: inter-word;
}
.SIT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #0489B1;
	padding-left: 5px;
}
.SIT_collapse_similarity:hover{
	background: #d0dfe2;
}
.SIT_collapse_similarity + input {
	display: none;
}
.SIT_collapse_similarity + input + div{
	display: none;
}
.SIT_collapse_similarity + input:checked + div{
	display: block;
}
.SIT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.SIT_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #0489B1;
}
.SIT_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_TU {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #04B486;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_TU + input{
	display: none;
}
.collapse_TU + input + div{
	display: none;
}
.collapse_TU + input:checked + div{
	display: block;
}
.TU {
	text-align: justify;
	text-justify: inter-word;
}
.TU_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #04B486;
	padding-left: 5px;
}
.TU_collapse_similarity:hover{
	background: #d0e2de;
}
.TU_collapse_similarity + input {
	display: none;
}
.TU_collapse_similarity + input + div{
	display: none;
}
.TU_collapse_similarity + input:checked + div{
	display: block;
}
.TU_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.TU_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #04B486;
}
.TU_similar > a {
	text-decoration: none;
	color: gray;
}

ul { 
	list-style-type: none;
	padding: 0;
	margin: 0;
}
ul li > a {
	text-decoration: none;
	color: gray;
}
ul li:hover, ul li:focus {
	background: #F0F0F0;
}

ul li:first-child span {
  border-left:none;  
}

</style>
<html>
<head>
<meta charset="utf-8">
<title>TF properties - NadR</title>
<meta name="theme-color" content="#ffffff">
</head>
<body>
	<div class="banner">
		<center><font color="white" size="16px" face="arial">NadR</font></center>
	</div>
	<div align="center" class="main-content">
		<div class="spacer">
	</div><br>			<label class="collapse_ACT" for="_1"><font size="5px">NadR transcriptional activity</font></label>
				<input id="_1" type="checkbox">
				<div class="ACT"><br><ul><li>Based on HPLC analysis , no nucleoside phosphorylation was detected after overnight incubation of 0.2 mM concentrations of ribosyl derivatives of nicotinic acid , uracil , and cytosine in the presence of 5 mM ATP with 0.02 mg of stNadR per ml .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12446641" target="_blank">12446641</a>]<br></li>
<li>Due to the very poor afﬁnity of stNadR for its substrate , only rough estimates of NMNAT kinetic parameters could be obtained ( K 1 m , app , 8 2 mM , and kcat , app , 0.26 0.04 s at a saturating concentration of 0.2 mM ATP ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12446641" target="_blank">12446641</a>]<br></li>
<li>Direct veriﬁcation of NMNAT and RNK activities of hiNadR and stNadR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12446641" target="_blank">12446641</a>]<br></li>
<li>All attempts to overexpress the C-terminal RNK domain failed to produce any soluble and active protein , regardless of the source ( hiNadR or stNadR ) , type of fusion ( His6 or thiore-stNadR , full size 0.04 250 1 10,000 25 hiNadR , full size 0.90 250 0.04 10,000 0.04 hiNadR , NMNAT domain 3 None a Speciﬁc activities of stNadR and hiNadR were determined at 1 mM NaMN , NMN , and N-ribosylnicotinamide in the presence of 1 mM ATP by coupled assays .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12446641" target="_blank">12446641</a>]<br></li>
<li>Not only the set of regulated genes can vary , but the autoregulation of the nadR gene by NadR , predicted here for the ﬁrst time , is a feature of several , but not all genomes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16078372" target="_blank">16078372</a>]<br></li>
<li>However , none of these organisms contain the nadA , nadB , or pncB gene , known to be regulated by NadR in Enterobacteriaceae .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12446641" target="_blank">12446641</a>]<br></li>
<li>It is tempting to propose that the nadV gene may provide a point of regulation by NadR in H. ducreyi and similar organisms .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12446641" target="_blank">12446641</a>]<br></li>
<li>Several genes involved in NAD synthesis and recycling are repressed by the NadR protein , and mutations in nadR might increase their expression and the resulting intracellular concentration of NAD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16751270" target="_blank">16751270</a>]<br></li>
<li>In this regard , it has been proposed that NadR behaves as an alloste-ric protein : in the presence of high NAD levels it might assume a conformation which allows both repression of NAD biosynthetic genes transcription and inhibition of NMN transport system ; conversely , when NAD levels are low , NadR might associate with the membrane , allowing full expression of bio-synthetic genes and stimulating NMN uptake ( 5 , 18 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10464228" target="_blank">10464228</a>]<br></li>
<li>The NadR protein was previously predicted to possess ATPase activity ( 35 ) , but no speciﬁc functional role for this activity was proposed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12446641" target="_blank">12446641</a>]<br></li>
<li>Therefore , its presence in NadR prompted us to clone and express the protein to investigate its possible NMN adenylyltransferase activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10464228" target="_blank">10464228</a>]<br></li>
				</div></ul><br>
			<label class="collapse_DOM" for="_2"><font size="5px">NadR protein domains</font></label>
				<input id="_2" type="checkbox">
				<div class="DOM"><br><ul><li>Crystallographic studies of hiNadR ( 51 ) show that the NMNAT and the NRK domains are well separated in the tertiary structure , consistent with their independent action .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12446641" target="_blank">12446641</a>]<br></li>
<li>Directed mutagenesis of this site ( H77A and H80A of the HXGH motif ) in stNadR produced the full-size protein completely lacking NMNAT activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12446641" target="_blank">12446641</a>]<br></li>
<li>Based on metabolic reconstruction from genomic data , we suggested that the last step of the N-ribosylnicotinamide salvage pathway in H. inﬂuenzae is performed by the NMNAT domain of hiNadR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12446641" target="_blank">12446641</a>]<br></li>
<li>All attempts to overexpress the C-terminal RNK domain failed to produce any soluble and active protein , regardless of the source ( hiNadR or stNadR ) , type of fusion ( His6 or thiore-stNadR , full size 0.04 250 1 10,000 25 hiNadR , full size 0.90 250 0.04 10,000 0.04 hiNadR , NMNAT domain 3 None a Speciﬁc activities of stNadR and hiNadR were determined at 1 mM NaMN , NMN , and N-ribosylnicotinamide in the presence of 1 mM ATP by coupled assays .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12446641" target="_blank">12446641</a>]<br></li>
<li>To verify the distribution of NMNAT and RNK activities within the domain boundaries predicted from the multiple alignment , we have attempted overexpressing the corresponding hiNadR domains individually .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12446641" target="_blank">12446641</a>]<br></li>
<li>Full-size NadR contains two putative ATP-binding sites in the NMNAT and the RNK domains ( Fig. 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12446641" target="_blank">12446641</a>]<br></li>
<li>Since the hiNadR protein contains two functional domains involved in the same pathway , our transposon mutagenesis results do not allow us to assert whether only one ( NMNAT domain ) or both are essential .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12446641" target="_blank">12446641</a>]<br></li>
<li>( i ) The N-terminal helix-turn-helix ( HTH ) domain ( corresponding to amino acids [ aa ] 1 to 63 in ecNadR ) is characteristic of the DNA-binding proteins involved in the transcriptional regulation of the early NAD biosynthetic genes nadB , nadA , and pncB ( 42 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12446641" target="_blank">12446641</a>]<br></li>
<li>( ii ) The central NMNAT domain ( aa 64 to 233 in ecNadR ) has a recognizable Rossman fold with a conserved ATP-binding site in the Nterminal part of the sequence , characteristic of the nucleoti-dyltransferase superfamily ( 9 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12446641" target="_blank">12446641</a>]<br></li>
<li>( iii ) The C-terminal RNK domain ( aa 234 to 417 in ecNadR ) has a Rossman-like fold and recognizable Walker A and B motifs typical of members of the P-loop kinase family ( 11 , 26 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12446641" target="_blank">12446641</a>]<br></li>
<li>Based on a multiple alignment of NadR homologs from various sources , this C-terminal domain represents a putative P-loop kinase ( Fig. 3 ) with the predicted characteristic fold and conserved Walker-type ATP-binding motifs A and B ( 11 , 26 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12446641" target="_blank">12446641</a>]<br></li>
<li>We predicted the C-terminal domain of NadR to be a candidate for this functional role based on cumulative genomic evidence , such as ( i ) clustering on the chromosome with the pnuC gene ( Fig. 3 ) , ( ii ) translational fusion with the NMNAT domain , ( iii ) cooccurrence of pnuC and nadR in all Pasteurellaceae and Enterobacteriaceae , and ( iv ) a recognizable Rossman-like fold with Walker A and B sequence motifs , characteristic of the P-loop kinase superfamily .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12446641" target="_blank">12446641</a>]<br></li>
<li>NadR requires divalent metal cations for activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10464228" target="_blank">10464228</a>]<br></li>
<li>DNA-binding activity of the recombinant NadR protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10464228" target="_blank">10464228</a>]<br></li>
<li>NAD-dependent DNA-binding activity of the bifunctional NadR regulator of Salmonella typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12446641" target="_blank">12446641</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10464228" target="_blank">10464228</a>]<br></li>
<li>Indeed , no DNA-binding domains were found in NadR orthologs from genomes outside the Enterobacteriaceae and Pasteurellaceae families .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16078372" target="_blank">16078372</a>]<br></li>
<li>NadR of other Pasteurellaceae have the DNA-binding domain , but these genomes have no nadA , nadB and pncB orthologs , nor do they have candidate sites for the enterobacterial NadR-signal .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16078372" target="_blank">16078372</a>]<br></li>
<li>It is known that in some gamma-proteobacteria , for example in Haemophilus inﬂuenzae , NadR orthologs do not contain the DNA-binding domain31 and thus have only enzymatic , but not regulatory role .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16078372" target="_blank">16078372</a>]<br></li>
<li>Among the latter , Haemophilus inﬂuenzae is the only genome with NadR lacking the DNA-binding domain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16078372" target="_blank">16078372</a>]<br></li>
<li>Nicotinate-phosphoribosyltransferase Salvage II : niacin to NMN No No No No No Yes Yes No Yes Nicotinamide phosphoribosyltransferase 2.4.2.12 nadV Salvage III : NMN / N-ribosylnicotinamide to NAD NMN phosphohydrolase ( periplasmic ) Pyridine nucleoside transporter Tripartite NadR protein : Ribosylnicotinamide kinase ; domain 3 NMN adenylyltransferase ; domain 2 Transcription regulator ; domain 1 Yes Yes No No Yes Yes Yes Yes Yes ushAe pnuC nadN 3.1.3.5 2.7.1.22 2.7.7.1 nadR nadR nadR Universal pathways : NaMN to NAD Yes NaMN adenylyltransferase 2.7.7.18 nadDd NAD synthetase 6.3.5.1 nadE Yes Yes Yes Yes No No No No Universal pathways : NAD to NADP Yes NAD-kinase 2.7.1.23 nadFd Yes Yes Yes Yes Yes Yes Yes Yes a Representative genomes containing NadR homologs are shown .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12446641" target="_blank">12446641</a>]<br></li>
<li>In order to reﬁne the functional assignment for the NadR RNK domain , we analyzed its substrate speciﬁcity towards structurally similar nucleosides .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12446641" target="_blank">12446641</a>]<br></li>
<li>Gene clustering , domain organization , and amino acid sequence conservation in NadR homologs .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12446641" target="_blank">12446641</a>]<br></li>
<li>The HTH domain is not preserved in hiNadR , and the reliable sequence similarity between hiNadR and other NadR homologs starts only within the boundaries of the NMNAT domain ( Fig. 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12446641" target="_blank">12446641</a>]<br></li>
<li>In contrast to hiNadR , all of the sequenced NadR homologs from V factor-independent Pasteurelaceae contain a recognizable HTH domain ( Table 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12446641" target="_blank">12446641</a>]<br></li>
<li>( B ) Domain organization and approximate domain boundaries in ecNadR protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12446641" target="_blank">12446641</a>]<br></li>
<li>Domain organization of NadR protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12446641" target="_blank">12446641</a>]<br></li>
<li>Expression of the NadR domain involved in transport and located in the protein 's C terminus might still occur if the IS150 element provides the necessary transcription and translation signals .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11014799" target="_blank">11014799</a>]<br></li>
<li>While the repression function resides in the NadR N-terminal domain , the C-terminal domain is involved in the transport function ( 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10464228" target="_blank">10464228</a>]<br></li>
<li>The pnuC orthologs in P. aeruginosa ( PA1958 ) and some related species form a tight operon ( Fig. 3A ) with an uncharacterized protein ( PA1957 ) corresponding to the `` stand-alone '' C-terminal domain of the NadR protein ( residues 238 to 416 of ecNadR ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12446641" target="_blank">12446641</a>]<br></li>
<li>DISCUSSION Using a comparative genomics approach , we established that RNK , involved with the N-ribosylnicotinamide salvage pathway of NAD biosynthesis ( Fig. 1 ) , is encoded in the previously uncharacterized C-terminal domain of NadR protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12446641" target="_blank">12446641</a>]<br></li>
<li>A comparative analysis of the domain organizations of various NadR homologs led us to predict and experimentally verify that RNK activity resides within the C-terminal domain of the NadR protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12446641" target="_blank">12446641</a>]<br></li>
<li>In summary , a comparative genome analysis and reconstruction of NAD metabolism from genomic data allowed us to predict and experimentally conﬁrm that the C-terminal domain of the multifunctional NadR protein constitutes the previously uncharacterized RNK .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12446641" target="_blank">12446641</a>]<br></li>
<li>2002 Ribosylnicotinamide kinase domain of NadR protein : identiﬁcation and implications in NAD biosynthesis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15353568" target="_blank">15353568</a>]<br></li>
<li>Based on these results and previous data , the full-size multifunctional NadR protein ( as in Escherichia coli ) is composed of ( i ) an N-terminal DNA-binding domain involved in the transcriptional regulation of NAD biosynthesis , ( ii ) a central nicotinamide mononucleotide adenylyltransferase ( NMNAT ) domain , and ( iii ) a C-terminal RNK domain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12446641" target="_blank">12446641</a>]<br></li>
<li>NadR is a multifunctional protein , consisting of an N-terminal DNA-binding domain which represses NAD biosynthesis , a central nicotinamide mononucleotide adehyltransferase ( NMNAT ) domain and a Cterminal RNK domain .31,32 The NAD biosynthetic pathway and transformations are shown in Fig. 1.31 Genes known to be repressed by NadR in E. coli and S. typhi are marked by rectangles .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16078372" target="_blank">16078372</a>]<br></li>
<li>Therefore , we considered the C-terminal domain of NadR protein to be a likely candidate for the missing RNK enzyme .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12446641" target="_blank">12446641</a>]<br></li>
<li>Multiple protein alignment demonstrated that NadR orthologs in all considered genomes contained DNA-binding domain , NMNAT domain and RNK domain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16078372" target="_blank">16078372</a>]<br></li>
<li>A multiple alignment of NadR proteins from various sources suggests the existence of three distinct functional domains ( Fig. 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12446641" target="_blank">12446641</a>]<br></li>
<li>The purloined letter : bacterial orthologs of archaeal NMN adenylyltransferase are domains within multifunctional transcription regulator NadR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12446641" target="_blank">12446641</a>]<br></li>
<li>When the archaeal Methanococcus jannaschii NMN adenylyltransferase sequence ( 12 ) was used as a probe in a BLAST search for homologous sequences , a region spanning 29 residues at the N terminus of the protein was found to signiﬁcantly align with an internal region of both S. typhimurium and E. coli NadR ( Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10464228" target="_blank">10464228</a>]<br></li>
				</div></ul><br>
			<label class="collapse_EVO" for="_3"><font size="5px">NadR evolution</font></label>
				<input id="_3" type="checkbox">
				<div class="EVO"><br><ul><li>Conserved amino acids are outlined by black ( for identity ) and gray ( for similarity ) backgrounds based on a larger alignment of 15 NadR homologs produced by using ClustalW ( http://dot.imgen.bcm.tmc.edu:9331/multi-align/Options/clustalw .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12446641" target="_blank">12446641</a>]<br></li>
<li>( iii ) The C-terminal RNK domain ( aa 234 to 417 in ecNadR ) has a Rossman-like fold and recognizable Walker A and B motifs typical of members of the P-loop kinase family ( 11 , 26 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12446641" target="_blank">12446641</a>]<br></li>
				</div></ul><br>
			<label class="collapse_RP" for="_4"><font size="5px">Processes regulated by NadR</font></label>
				<input id="_4" type="checkbox">
				<div class="RP"><br><ul><li>The N-terminal extension in hiNadR preceding the alternative translation start ( Met-53 ) is shown in lowercase letters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12446641" target="_blank">12446641</a>]<br></li>
<li>For site-directed mutagenesis of the ATP-binding site in the stNadR protein ( mutations H77A and H80A ) , the primers were CCATTGgcT ACCGGAgcCATCTACTTGATCC ( sense ) and GGATCAAGTAGATGgcTC CGGTAgcCAATGG ( antisense ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12446641" target="_blank">12446641</a>]<br></li>
<li>Many of such predictions were subsequently con-ﬁrmed in experiment ,23,24,12 or even served as a starting point for experimental analysis .18,25 -- 27 There exist several Internet servers for comparative analysis of bacterial regulation , in particular , EnteriX28 and PredictRegulon .29 In an attempt to analyze the evolutionary dynamics of a relatively simple , wellstudied regulon that includes genes from an essential part of the metabolism , we considered the NadR regulon in Enterobacteriaceae .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16078372" target="_blank">16078372</a>]<br></li>
<li>NadR substrate preferencesa Sp act of NMNAT / Sp act RNK / RNK / NMNAT NaMNAT of RNK RNaK NMNAT ( U/mg ) ratio ( U/mg ) ratio ratio Protein , size FIG. 6 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12446641" target="_blank">12446641</a>]<br></li>
<li>The nicotinamide-adenine-dinucleotides ( NAD , NADH , NADP , NADPH ) are essential cofactors in all living systems and function as hydride acceptors ( NAD , NADP ) and donors ( NADH , NADPH ) in biochemical redox reactions .30 At high internal levels of NAD , the transcriptional regulator NadR represses the de novo synthesis and salvage pathways .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16078372" target="_blank">16078372</a>]<br></li>
<li>Besides its regulatory role , NadR is a bifunctional protein involved in transport of nicotinamide mono-nucleotide ( NMN ) across the cell membrane .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11014799" target="_blank">11014799</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_7">
Foster , J. W. , and T. Penfound , 1993 The bifunctional NadR regulator of Salmonella typhimurium : location of regions involved with DNA binding , nucleotide-transport and intramolecular communication .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11014799" target="_blank">11014799</a>]<br></label>
<input id="rp_similarity_7", type="checkbox">
<div class="RP_similar">
<li>The bifunctional NadR regulator of Salmonella typhimurium : location of regions involved with DNA binding , nucleotide-transport and intramolecular communication .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12446641" target="_blank">12446641</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15556475" target="_blank">15556475</a>]<br></li></div>
<li>Expression of the NadR domain involved in transport and located in the protein 's C terminus might still occur if the IS150 element provides the necessary transcription and translation signals .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11014799" target="_blank">11014799</a>]<br></li>
<li>Several genes involved in NAD-synthesis and recycling are repressed by the NadR protein , and mutations in nadR might increase their expression and the resulting intracellular concentration of NAD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16751270" target="_blank">16751270</a>]<br></li>
<li>Keywords : NAD-biosynthesis ; NadR ; transcription factor ; regulation of transcription ; comparative genomics ; phylogenetic footprinting ; evolution .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16078372" target="_blank">16078372</a>]<br></li>
<li>Based on these results and previous data , the full-size multifunctional NadR protein ( as in Escherichia coli ) is composed of ( i ) an N-terminal DNA-binding domain involved in the transcriptional regulation of NAD-biosynthesis , ( ii ) a central nicotinamide mononucleotide adenylyltransferase ( NMNAT ) domain , and ( iii ) a C-terminal RNK domain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12446641" target="_blank">12446641</a>]<br></li>
<li>Kurnasov OV , Polanuyer BM , Ananta S , Sloutsky R , Tam A , Gerdes SY , Osterman AL , Ribosylnicotinamide kinase domain of NadR protein : identiﬁcation and implications in NAD-biosynthesis , J Bacteriol 184:6906 -- 6917 , 2003 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16078372" target="_blank">16078372</a>]<br></li>
<li>2002 Ribosylnicotinamide kinase domain of NadR protein : identiﬁcation and implications in NAD-biosynthesis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15353568" target="_blank">15353568</a>]<br></li>
<li>Gene clustering , domain organization , and amino acid sequence conservation in NadR homologs .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12446641" target="_blank">12446641</a>]<br></li>
<li>Genes from candidate NadR regulons .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16078372" target="_blank">16078372</a>]<br></li>
				</div></ul><br>
			<label class="collapse_SIT" for="_5"><font size="5px">NadR binding site</font></label>
				<input id="_5" type="checkbox">
				<div class="SIT"><br><ul><li>Sequence alignment of homologous regions of NadR proteins from E. coli ( ENadR ) and S. typhimurium ( SNadR ) , NMN adenylyltransferase from M. jannaschii ( MjAT ) , putative NMN adenylyltransferases from Methanobacterium thermoautotrophicum ( MtAT ) , Archaeoglobus fulgidus ( AfAT ) , and Pyrococcus horikoshii ( PhAT ) , and NMN adenylyltransferase from Synechocystis sp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10464228" target="_blank">10464228</a>]<br></li>
<li>In an attempt to ﬁnd new candidate members of the NadR regulon , we identiﬁed candidate sites and considered all genes with candidate sites in at least four genomes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16078372" target="_blank">16078372</a>]<br></li>
<li>A computer-aided search for sequence similarity revealed the presence in NadR of a 29-residue region highly conserved among known nicotinamide mononucleotide adenylyltransferases .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10464228" target="_blank">10464228</a>]<br></li>
<li>The sequence logo of the NadR signal is shown in Fig. 2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16078372" target="_blank">16078372</a>]<br></li>
<li>( ii ) The central NMNAT domain ( aa 64 to 233 in ecNadR ) has a recognizable Rossman fold with a conserved ATP-binding site in the Nterminal part of the sequence , characteristic of the nucleoti-dyltransferase superfamily ( 9 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12446641" target="_blank">12446641</a>]<br></li>
<li>The conserved region coincides with the spacer of the candidate NadR binding site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16078372" target="_blank">16078372</a>]<br></li>
				</ul></div><br>
			<label class="collapse_TU" for="_6"><font size="5px">Transcription units regulated by NadR</font></label>
				<input id="_6" type="checkbox">
				<div class="TU"><br><ul><li>As mentioned before , a stand-alone RNK gene in P. aeruginosa ( encoding a paNadR protein corresponding to the ecNadR Cterminal domain ) occurs in an operon with the pnuC gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12446641" target="_blank">12446641</a>]<br></li>
<li>However , none of these organisms contain the nadA , nadB , or pncB gene , known to be regulated by NadR in Enterobacteriaceae .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12446641" target="_blank">12446641</a>]<br></li>
<li>Several genes involved in NAD synthesis and recycling are repressed by the NadR protein , and mutations in nadR might increase their expression and the resulting intracellular concentration of NAD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16751270" target="_blank">16751270</a>]<br></li>
<li>Identical results were obtained with the nadA control region ( not shown ) , suggesting that the recombinant NadR retained its ability to speciﬁcally bind to NAD biosynthetic genes operator regions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10464228" target="_blank">10464228</a>]<br></li>
				</div></ul></div><br>
	</div>
</body>
</html>