<!doctype html>
<style>
@charset "utf-8";
body {
	height: 100%;
	margin: 0;
	padding: 0;
}

.banner {
	position: fixed;
	top: 0;
	width: 100%;
	height: 65px;
	background: gray;
	line-height: 65px;
}
.spacer {
	height: 10px;
}

.main-content {
	padding: 5%;
}

.collapse_ACT {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-style: none solid none none;
	border-color: #B4045F;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_ACT + input{
	display: none;
}
.collapse_ACT + input + div{
	display: none;
}
.collapse_ACT + input:checked + div{
	display: block;
}
.ACT {
	text-align: justify;
	text-justify: inter-word;
}
.ACT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B4045F;
}
.ACT_collapse_similarity:hover{
	background: #e5d3dc;
}
.ACT_collapse_similarity + input {
	display: none;
}
.ACT_collapse_similarity + input + div{
	display: none;
}

.ACT_collapse_similarity + input:checked + div{
	display: block;
}
.ACT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.ACT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B4045F;
	padding-left: 5px;
}
.ACT_collapse_similarity:hover{
	background: #e5d3dc;
}
.ACT_collapse_similarity + input {
	display: none;
}
.ACT_collapse_similarity + input + div{
	display: none;
}
.ACT_collapse_similarity + input:checked + div{
	display: block;
}
.ACT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.ACT_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #B4045F;
}
.ACT_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_DOM {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-style: none solid none none;
	border-color: #B404AE;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_DOM + input{
	display: none;
}
.collapse_DOM + input + div{
	display: none;
}
.collapse_DOM + input:checked + div{
	display: block;
}
.DOM {
	text-align: justify;
	text-justify: inter-word;
}
.DOM_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B404AE;
	padding-left: 5px;
}
.DOM_collapse_similarity:hover{
	background: #e8d7e7;
}
.DOM_collapse_similarity + input {
	display: none;
}
.DOM_collapse_similarity + input + div{
	display: none;
}
.DOM_collapse_similarity + input:checked + div{
	display: block;
}
.DOM_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}.DOM_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #B404AE;
}
.DOM_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_EVO {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #7401DF;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_EVO + input{
	display: none;
}
.collapse_EVO + input + div{
	display: none;
}
.collapse_EVO + input:checked + div{
	display: block;
}
.EVO {
	text-align: justify;
	text-justify: inter-word;
}
.EVO_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #7401DF;
	padding-left: 5px;
}
.EVO_collapse_similarity:hover{
	background: #e2d7ed;
}
.EVO_collapse_similarity + input {
	display: none;
}
.EVO_collapse_similarity + input + div{
	display: none;
}
.EVO_collapse_similarity + input:checked + div{
	display: block;
}
.EVO_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.EVO_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #7401DF;
}
.EVO_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_RP {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #0174DF;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_RP + input{
	display: none;
}
.collapse_RP + input + div{
	display: none;
}
.collapse_RP + input:checked + div{
	display: block;
}
.RP {
	text-align: justify;
	text-justify: inter-word;
}
.RP_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #0174DF;
	padding-left: 5px;
}
.RP_collapse_similarity:hover{
	background: #c9d7e2;
}
.RP_collapse_similarity + input {
	display: none;
}
.RP_collapse_similarity + input + div{
	display: none;
}
.RP_collapse_similarity + input:checked + div{
	display: block;
}
.RP_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.RP_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #0174DF;
}
.RP_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_SIT {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #0489B1;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_SIT + input{
	display: none;
}
.collapse_SIT + input + div{
	display: none;
}
.collapse_SIT + input:checked + div{
	display: block;
}
.SIT {
	text-align: justify;
	text-justify: inter-word;
}
.SIT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #0489B1;
	padding-left: 5px;
}
.SIT_collapse_similarity:hover{
	background: #d0dfe2;
}
.SIT_collapse_similarity + input {
	display: none;
}
.SIT_collapse_similarity + input + div{
	display: none;
}
.SIT_collapse_similarity + input:checked + div{
	display: block;
}
.SIT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.SIT_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #0489B1;
}
.SIT_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_TU {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #04B486;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_TU + input{
	display: none;
}
.collapse_TU + input + div{
	display: none;
}
.collapse_TU + input:checked + div{
	display: block;
}
.TU {
	text-align: justify;
	text-justify: inter-word;
}
.TU_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #04B486;
	padding-left: 5px;
}
.TU_collapse_similarity:hover{
	background: #d0e2de;
}
.TU_collapse_similarity + input {
	display: none;
}
.TU_collapse_similarity + input + div{
	display: none;
}
.TU_collapse_similarity + input:checked + div{
	display: block;
}
.TU_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.TU_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #04B486;
}
.TU_similar > a {
	text-decoration: none;
	color: gray;
}

ul { 
	list-style-type: none;
	padding: 0;
	margin: 0;
}
ul li > a {
	text-decoration: none;
	color: gray;
}
ul li:hover, ul li:focus {
	background: #F0F0F0;
}

ul li:first-child span {
  border-left:none;  
}

</style>
<html>
<head>
<meta charset="utf-8">
<title>TF properties - NagC</title>
<meta name="theme-color" content="#ffffff">
</head>
<body>
	<div class="banner">
		<center><font color="white" size="16px" face="arial">NagC</font></center>
	</div>
	<div align="center" class="main-content">
		<div class="spacer">
	</div><br>			<label class="collapse_ACT" for="_1"><font size="5px">NagC transcriptional activity</font></label>
				<input id="_1" type="checkbox">
				<div class="ACT"><br><ul><li>As expected , however , loss of methylation protection of GATCNagC occurs in the presence of GlcNAc , and as Neu5Ac catabolism generates GlcNAc-6P ( Plumbridge and Vimr , 1999 ) , both GATC sites become more fully methylated when Neu5Ac is metabolized .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17238917" target="_blank">17238917</a>]<br></li>
<li>The pattern of methylation protection of GATCNanR and GATCNagC in vivo appears to alternate , suggesting that when NanR is bound to the DNA , ONC1 is unoccupied and vice versa ( El-Labany et al. , 2003 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17238917" target="_blank">17238917</a>]<br></li>
<li>1.3 1.9 5.0 2.4 4.7 2.7 2.6 1.4 PspF , IHF PhoB RbsR , Crp 37.9 22.0 2.6 1.2 1.8 2.3 0.8 1.1 Inner membrane Unknown Ribose transport Ribose transport Ribose transport Ribose transport Ribose transport Ribose transport pspG ( yjbO ) ytfK rbsD rbsA rbsC rbsB rbsK rbsR 0.7 3.8 1.8 1.0 1.6 1.4 1.2 0.8 0.9 3.0 8.2 5.4 21.3 6.6 9.0 2.8 manX Mannose permease 4.2 2.1 1.1 5.0 manY Mannose permease 3.4 2.6 1.1 5.2 manZ Mannose permease 6.5 5.7 1.0 16.7 NagC , DgsA , Crp a. Genes and operons which are induced 10-fold in three independent experiments are listed with the average fold induction for the three experiments .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18710431" target="_blank">18710431</a>]<br></li>
<li>23.5 20.0 3.8 16.3 9.1 12.7 Maltose transport MalT , Crp malK lamB malM ibpB ibpA htpG Maltose transport MalT , Crp 18.6 5.3 3.3 39.0 24.4 11.7 32.7 13.3 9.1 24.2 20.2 8.2 2.6 2.1 1.7 4.6 4.1 2.0 48.6 19.2 20.0 52.6 35.9 16.6 s32 Heat shock Heat shock s32 Heat shock chaperone s32 Protease ; chaperone Regulator Regulator ; chaperone Protease Periplasmic clpB 11.5 12.9 17.3 21.0 20.4 14.3 3.3 3.2 1.3 1.8 2.5 2.9 8.0 2.7 2.9 24.5 6.8 5.7 1.7 1.3 yceP/bssS cpxP ( b3913 ) degP ( htrA ) spy CpxR CpxR , H-NS CpxR , BaeSR CpxR CpxR Unknown Unknown ydeH yebE 49.2 44.8 63.1 21.0 30.7 32.2 56.9 33.1 9.5 1.1 1.6 6.1 7.3 7.3 2.6 1.5 1.6 2.0 1.4 1.2 1.3 2.9 Unknown ; signal peptide Putative transporter Regulator ; phage shock protein Regulator ; phage shock protein Regulator ; phage shock protein Inner membrane Sulphurtransferase yncJ ( b1436 ) yhjX pspA pspB pspC pspD pspE 1.6 44.1 1.1 1.6 1.7 1.2 1.2 3.1 165.6 1.8 1.5 2.1 1.2 1.5 PspF , IHF 1.3 1.9 5.0 2.4 4.7 2.7 2.6 1.4 PspF , IHF PhoB RbsR , Crp 37.9 22.0 2.6 1.2 1.8 2.3 0.8 1.1 Inner membrane Unknown Ribose transport Ribose transport Ribose transport Ribose transport Ribose transport Ribose transport pspG ( yjbO ) ytfK rbsD rbsA rbsC rbsB rbsK rbsR 0.7 3.8 1.8 1.0 1.6 1.4 1.2 0.8 0.9 3.0 8.2 5.4 21.3 6.6 9.0 2.8 manX Mannose permease 4.2 2.1 1.1 5.0 manY Mannose permease 3.4 2.6 1.1 5.2 manZ Mannose permease 6.5 5.7 1.0 16.7 NagC , DgsA , Crp a. Genes and operons which are induced 10-fold in three independent experiments are listed with the average fold induction for the three experiments .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18710431" target="_blank">18710431</a>]<br></li>
<li>sitesa % CGb Count % Non-codinge Ada 3 38.7 AraC 6 40.3 2 3 100 100 47 76 100 28 33 32 73 38 100 52 100 86 79 89 100 67 26 100 28 100 70 37 67 28 100 82 100 47 50 51 100 100 38 33 64 3 6 100 100 38 54 100 23 40 18 43 29 100 38 100 39 27 53 58 71 19 100 23 33 47 100 40 65 21 23 60 25 100 41 100 26 43 36 100 35 35 30 64 36 23 22 26 14 100 83 28 ArcA 12 20.0 ArgR 17 22.9 CarP 2 46.0 Crp 49 37.0 CspA 4 41.3 CynR 2 33.3 CysB 3 34.2 CytR 5 37.8 DeoR 3 22.9 DnaA 8 36.7 FadR 7 47.0 FarR 4 20.0 FhlA 3 42.0 Fis 19 38.2 FlhCD 3 53.8 Fnr 9 25.8 FruR 12 42.7 Fur 9 23.4 GalR 7 43.8 GcvA 4 22.5 GlpR 13 39.6 HipB 4 40.9 Hns 15 31.5 IclR 2 26.7 Ihf 26 32.6 IlvY 2 32.4 LacI 3 44.4 LexA 18 33.9 Lrp 18 28.9 MalT 10 61.0 MarR 2 37.5 MelR 2 33.3 MetJ 15 40.1 MetR 8 33.3 NagC 6 22.5 NarL 9 34.6 NtrC 6 51.9 OmpR 9 26.7 OxyR 4 42.3 PdhR 3 33.3 PhoB 15 29.7 PurR 23 45.3 RhaS 2 50.0 RpoD15 27 37.4 RpoD16 48 34.9 RpoD17 116 37.3 RpoD18 34 38.0 RpoD19 25 38.2 RpoE 3 38.9 RpoH13 8 42.1 RpoH14 7 44.3 RpoN 7 40.6 RpoS17 15 43.0 RpoS18 7 45.2 SoxS 14 40.8 TorR 4 37.5 TrpR 4 38.5 Tus 6 26.8 TyrR 17 33.2 a The number of input sites used to construct the matrix .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9813115" target="_blank">9813115</a>]<br></li>
<li>of additional probable sites in predictionsd 3 3 7e 7 -- 4 22e 0 1 0 2 -- -- 5 0 1 -- 13 7 18 1 0 6e 1 1 1 3 Ada AraC ArcA ArgR CarP CpxR Crp CspA CynR CysB CytR DeoR DnaA FadR FhlA Fis FlhCD Fnr FruR Fur GalR GcvA GlpR IciA ( ArgP ) IclR Ihf IlvY LacI LexA Lrp MalT MarR MelR MetJ MetR Mlc ( DgsA ) ModE NagC NarL NarP NtrC OmpR OxyR PdhR PhoB PurR RhaR RhaS SoxR SoxS TorR TrpR TyrR Stem -- loopsf 6 2 0 5 2 9 1 3 4 5e 5 -- 5 5 4 6 3 4 2 -- 1 3 5 1 3 -- aThe number of genes in our study set with documented binding sites for each TF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11160901" target="_blank">11160901</a>]<br></li>
<li>NagE-lacZ NagB-lacZ p -LCB- NagC ) controi repression p ( NagC ) control repression Strain 0.45 ± 0.15 1 ± 0,26 2.2 0.11 ± 0.01 1 ± 0.09 8.9 4.3 ± 0.6 18.3 ± 2.4 4,2 0.8510.22 14.213.2 16.7 4.810.4 23.412.8 4.9 16,0 ± 2.1 16.212.7 1 Wild type +6 BoxB '' IBPC5321R carrying the pJEL122 derived ` long ` prolein fusions was transformed with pBGS ( NagC ) which overproduces NagC from the lac promoter or the parental vector pBGS19 ( control ) Beta-galactosidase activities were measured in MOPS medium -LCB- Neidhardt et al. 1974 -RCB- containing 0.4 % glyceroi , 0.5 % casamino acids , 75 \ xq ml '' ' ampicillin and 25  ig ml '' ' kanamycin .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7934873" target="_blank">7934873</a>]<br></li>
<li>We thank Guy Condemine , Eric Vimr , Marjan van der Woude , Shadi Shafei , and Gary Robinson for discussions ; Kathryn Kalivoda and Eric Vimr ( University of Illinois at Urbana-Champaign , Urbana ) , for purified NanR and Charles Bell and Mitchell Lewis , ( University of Pennsylvania School of Medicine , Philadelphia ) for purified NagC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15534208" target="_blank">15534208</a>]<br></li>
<li>ACKNOWLEDGEMENTS I thank Annie Kolb for many helpful discussions and critical reading of the manuscript and Mitchel Lewis and Winfried Boos for the gifts of purified NagC and Mlc , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11139621" target="_blank">11139621</a>]<br></li>
<li>ACKNOWLEDGMENTS We thank Ian Blomﬁeld and Corinne Dorel for strains and helpful discussions , Charles Bell and Mitchell Lewis for the gift of NagC , Annie Kolb for CAP , Anne Bibonne for technical assistance , and Nicole Cotte-Pattat for reading the manuscript .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15743943" target="_blank">15743943</a>]<br></li>
<li>While glmUS transcription is controlled by NagC protein ,2 which also represses operons involved in amino sugar uptake and degradation , σ54 and GlmY may independently adjust GlmS synthesis in response to the general nitrogen status of the cell .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17854828" target="_blank">17854828</a>]<br></li>
<li>Although the nature of the inhibitory effect exerted by the yjhA proximal sequences is not yet understood , we hypothesize that NagC and NanR somehow interrupt long-range cis-acting repression by forming alternative incompatible nucleoprotein structures .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15534208" target="_blank">15534208</a>]<br></li>
<li>Formation of complexes between this DNA and NagC in the presence of IHF prevented methylation by Dam methylase of GATCNagC , as judged by cleavage with MboI producing a band of 64 bp ( PM-GATCNagC ; Fig. 5 , lane 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17238917" target="_blank">17238917</a>]<br></li>
<li>In contrast to the situation with NagC and IHF , we where unable to demonstrate methylation protection of ONR in the presence of NanR in the assay described above ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17238917" target="_blank">17238917</a>]<br></li>
<li>In control experiments , it was found that prior methylation of the DNA did not affect the affinity of NagC for binding to the operator region in the presence or absence of IHF ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17238917" target="_blank">17238917</a>]<br></li>
<li>Moreover , NagC and IHF binding to the DNA in vitro does not inhibit NanR binding to ONR or vice versa ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17238917" target="_blank">17238917</a>]<br></li>
<li>Binding of NanR to its operator sequence ONR , and NagC to ONC1 , is alternate , as judged by Dam methylation protection studies in vivo ( Sohanpal et al. , 2004 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17238917" target="_blank">17238917</a>]<br></li>
<li>Moreover , the combination of NagC and IHF prevented Dam methylation of GATCNagC in vitro , and IHF was shown additionally to be required to prevent methylation of GATCNagC in vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17238917" target="_blank">17238917</a>]<br></li>
<li>Thus , like NagC , NanR may require an additional cofactor in order to prevent Dam methylation of the GATC site adjacent to its binding site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17238917" target="_blank">17238917</a>]<br></li>
<li>Thus , like NagC at GATCNagC , an additional cofactor could be required for NanR to prevent Dam methylation of GATCNanR in vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17238917" target="_blank">17238917</a>]<br></li>
<li>To determine if IHF binds to the potential site identiﬁed by mutagenesis above , and to investigate the effect of the protein on the interaction between NagC bound to ONC1 and ONC2 , DNase I footprinting was carried out on a 341 bp PCR generated fragment ( PM ; Figs 1 and 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17238917" target="_blank">17238917</a>]<br></li>
<li>Taking these results together with those presented above , it is concluded that the cooperative interaction of NagC bound to ONC1 and ONC2 , with IHF bound to the site identiﬁed in this work , forms a nucleoprotein complex that activates ﬁmB expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17238917" target="_blank">17238917</a>]<br></li>
<li>Considered together , these data show that IHF promotes an interaction between NagC bound to ONC1 and ONC2 , forming a stable nucleoprotein complex that activates ﬁmB expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17238917" target="_blank">17238917</a>]<br></li>
<li>Mutations in either subunit of IHF resulted in the loss of methylation protection of GATCNagC ( Fig. 2B ) , and thus IHF does facilitate the interaction between NagC bound to ONC1 and ONC2 in vivo as it does in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17238917" target="_blank">17238917</a>]<br></li>
<li>However , neither mutation of ONC1 , nor that of ONC2 , seems to enhance the methylation protection of GATCNanR relative to the wild type in vivo as would be expected if the binding of NagC and IHF inhibited NanR binding ( Fig. 2B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17238917" target="_blank">17238917</a>]<br></li>
<li>Examination of this and additional footprints allows us to conclude that IHF stimulates NagC binding to ONC1 at least 10-fold , and that IHF clearly facilitates in vitro the interaction between ONC1 and ONC2 predicted to occur in vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17238917" target="_blank">17238917</a>]<br></li>
<li>Using genetic and biochemical approaches , IHF was shown to-bind to a site midway between ONC1 and ONC2 to facilitate strong cooperative binding of NagC to these elements .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17238917" target="_blank">17238917</a>]<br></li>
<li>Moreover , binding of IHF to this site considerably enhanced binding of NagC to both ONC1 and ONC2 ( compare lanes 2 -- 5 with 6 -- 9 ) , showing that IHF facilitated cooperative binding of NagC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17238917" target="_blank">17238917</a>]<br></li>
<li>The ibs mutation described above eliminated IHF binding ( lanes 26 -- 30 ) , and prevented the enhanced binding of NagC to ONC1 , while mutation of the ONC2 site considerably reduced IHF-stimulated binding of NagC to both ONC1 and ONC2 ( compare lanes 6 -- 9 and 16 -- 19 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17238917" target="_blank">17238917</a>]<br></li>
<li>Thus , although NagC bound to NagC1 at region 2 is likely to control fimB expression , the function of NagC2 is unclear .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15534208" target="_blank">15534208</a>]<br></li>
<li>This raises the possibility that NagC bound to region 2 is only partially inactivated by sialic acid , or that the regulator plays a more complex role in the control of fimB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15534208" target="_blank">15534208</a>]<br></li>
<li>Neither sequences up to 820 bp upstream of ONR , nor those 270 bp downstream of ONC2 , are required for activation by NanR and NagC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17238917" target="_blank">17238917</a>]<br></li>
<li>To test this hypothesis further , and to examine the effect of operator-to-promoter distance on NagC activation , the effect of deleting the 274 bp region ( D7 ) described above on ﬁmB expression was also analysed ( Fig. 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17238917" target="_blank">17238917</a>]<br></li>
<li>NanR and NagC each bind to operator sequences [ ONR ( originally termed region 1 ) and ONC1 ( originally termed region 2/NagC1 ) respectively ] situated far upstream of ﬁmB ( Sohanpal et al. , 2004 ; Fig. 1 ) to stimulate ﬁmB expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17238917" target="_blank">17238917</a>]<br></li>
<li>NanR and NagC activate ﬁmB expression by binding to operator sequences ( ONR and ONC1 respectively ; Fig. 1 ) situated further upstream of the regulated promoter than is usual in bacteria .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17238917" target="_blank">17238917</a>]<br></li>
<li>One interpretation of these results was that NagC and NanR antagonize the inhibitory action of one or more cis-active silencer elements that repress ﬁmB from a distance .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17238917" target="_blank">17238917</a>]<br></li>
<li>If NanR is less effective at enhancing fimB expression than NagC , as it seems to be , then the recombinase genes expression must therefore be controlled by phase variation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15534208" target="_blank">15534208</a>]<br></li>
<li>This observation not only allows us to understand how a modest mutation in ONC2 can affect ﬁmB expression whereas the D3 mutation does not , but it also supports our prior assertion that NanR activates ﬁmB expression without NagC ( Sohanpal et al. , 2004 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17238917" target="_blank">17238917</a>]<br></li>
<li>Taken together , these observations support a model in which ﬁmB expression is activated alternately by NanR and NagC , with GlcNAc inhibiting one , and Neu5Ac both , of these pathways ( Sohanpal et al. , 2004 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17238917" target="_blank">17238917</a>]<br></li>
<li>Discussion We showed in earlier work that ﬁmB expression , and hence the OFF-to-ON phase switching of type 1 ﬁmbriation , is activated by NanR and NagC and hence repressed by Neu5Ac and GlcNAc ( El-Labany et al. , 2003 ; Sohanpal et al. , 2004 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17238917" target="_blank">17238917</a>]<br></li>
<li>The results extend our understanding of how NanR and NagC , and hence Neu5Ac and GlcNAc , control ﬁmB expression and type 1 ﬁmbriation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17238917" target="_blank">17238917</a>]<br></li>
<li>Thus one function of the large segment of DNA that separates the ﬁmB and nanC promoters seems to be to insulate ﬁmB expression from the effects of NanR and NagC , and hence presumably to optimize the control of type 1 ﬁmbriation by Neu5Ac and GlcNAc .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17238917" target="_blank">17238917</a>]<br></li>
<li>On the other hand , the double ibs ONR mutation exhibited a much lower level of ﬁmB expression , as would be expected if it prevents activation by both NagC and NanR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17238917" target="_blank">17238917</a>]<br></li>
<li>Increased efficacy of NanR in the D3 background compensates for loss of NagC function The results described above show that ONC2 activates ﬁmB expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17238917" target="_blank">17238917</a>]<br></li>
<li>Accordingly we reasoned that a loss of ﬁmB expression as a result of decreased NagC activity in the D3 background might be compensated for by enhanced activation by NanR for some reason .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17238917" target="_blank">17238917</a>]<br></li>
<li>Indeed opposing effects of IHF on activation by NanR and NagC could explain why mutations that affect IHF function have less effect on ﬁmB expression than do those affecting NagC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17238917" target="_blank">17238917</a>]<br></li>
<li>Furthermore , deletion of this region increases the efficacy of both NanR and NagC as activators , producing an overall increase in ﬁmB expression of nearly threefold .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17238917" target="_blank">17238917</a>]<br></li>
<li>This regulation occurs via the NanR and NagC regulators , which also control ﬁmB expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15743943" target="_blank">15743943</a>]<br></li>
<li>However , it is clear that they allow coordination of nanC and ﬁmB expression , since NanR and NagC repress one and activate the other of these divergently arranged genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15743943" target="_blank">15743943</a>]<br></li>
<li>In contrast to the situation with NagC , IHF is not required for NanR to activate ﬁmB expression in the wildtype background .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17238917" target="_blank">17238917</a>]<br></li>
<li>Thus , NanR and NagC appear to activate ﬁmB expression independently .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17238917" target="_blank">17238917</a>]<br></li>
<li>Alternatively , the sequence used in the replacement construct might somehow inhibit ﬁmB expression independently of NanR or NagC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17238917" target="_blank">17238917</a>]<br></li>
<li>We suppose that the 274 bp wild-type region adjacent to ONC2 analysed could contain one or more sequences that activate ﬁmB expression even in the absence of NanR or NagC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17238917" target="_blank">17238917</a>]<br></li>
<li>While the basis for this effect is uncertain , it is nevertheless evident that neither NanR nor NagC requires the wild-type segment of D7 DNA in order to be able to activate ﬁmB expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17238917" target="_blank">17238917</a>]<br></li>
<li>By replacing the 274 bp of wild-type DNA from -467 to -194 with heterologous sequences it was shown that neither NanR nor NagC requires this segment of DNA in order to activate ﬁmB expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17238917" target="_blank">17238917</a>]<br></li>
<li>The DNA sequences upstream of the D2 region are not required for NanR or NagC to control ﬁmB expression We previously presented evidence suggesting that the D2 region , and hence NanR and NagC , could activate ﬁmB expression by blocking the inhibitory effect of additional cis-acting element ( s ) positioned closer to nanC ( El-Labany et al. , 2003 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17238917" target="_blank">17238917</a>]<br></li>
<li>Although we have not characterized these fortuitous regulatory sequences further , it now seems unlikely that NanR and NagC overcome the inhibitory effect of cis-acting sequences further upstream of ﬁm to activate ﬁmB expression as we supposed originally .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17238917" target="_blank">17238917</a>]<br></li>
<li>NagC sites characteristically function in pairs ( Plum-bridge , 2001 ) , but although NagC binds to a second element ( ONC2 ; originally termed NagC2 ) situated 212 bp closer to ﬁmB than ONC1 , cooperative binding of NagC to ONC1 and ONC2 was not apparent in vitro , and a deletion that includes ONC2 ( D3 ) did not diminish ﬁmB expression ( El-Labany et al. , 2003 ; Sohanpal et al. , 2004 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17238917" target="_blank">17238917</a>]<br></li>
<li>yjhA encodes an outer membrane channel that can facilitate Neu5Ac uptake ( G. Condemine , personal communication ) , and this gene is also repressed strongly by NanR , an effect which is modulated by NagC binding to the NagC1 site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15534208" target="_blank">15534208</a>]<br></li>
<li>Site-directed mutagenesis of one pair of the conserved thymines ( Plumbridge , 1995 ; Plumbridge and Kolb , 1995 ) diminished NagC binding in vitro , and decreased ﬁmB expression in vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17238917" target="_blank">17238917</a>]<br></li>
<li>The combined effects of mutations in IHF , NagC and NanR binding sites on ﬁmB expression and of IHF on methylation protection in vivo The results presented above show that IHF , bound to an element situated midway between the NagC operators ONC1 and ONC2 , forms a cooperative complex with NagC in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17238917" target="_blank">17238917</a>]<br></li>
<li>Here it is demonstrated that NagC binding to a second site ( ONC2 ) , located 212 bp closer to ﬁmB , also controls ﬁmB expression , and that integration host factor ( IHF ) , which binds midway between ONC1 and ONC2 , facilitates NagC binding to its two operator sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17238917" target="_blank">17238917</a>]<br></li>
<li>Thus , when NagC binding to its operator sites is disrupted , IHF bound to the regulatory site identiﬁed above has no apparent effect on ﬁmB expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17238917" target="_blank">17238917</a>]<br></li>
<li>Thus , NagC could form a more extensive looped DNA structure than that formed by NanR binding to three adjacent sites , and this might at least in part , contribute to NagC being more effective as an antirepressor than NanR for fimB expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15534208" target="_blank">15534208</a>]<br></li>
<li>Thus , Dam could affect fimB expression by inhibiting NanR and NagC binding to their cognate sites as it can do for Lrp and OxyR ( 49 , 51 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15534208" target="_blank">15534208</a>]<br></li>
<li>IHF therefore does seem to have a small activating effect on ﬁmB expression in the absence of both NagC and NanR binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17238917" target="_blank">17238917</a>]<br></li>
<li>In the absence of NagC binding , a residual activating effect of IHF on ﬁmB expression could thus be offset by an inhibi-tory effect of IHF on NanR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17238917" target="_blank">17238917</a>]<br></li>
<li>However , placing the NanR , IHF and NagC binding sites closer to the ﬁmB promoter enhances the ability of the regulators to activate ﬁmB expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17238917" target="_blank">17238917</a>]<br></li>
<li>NanR and NagC are two regulators that control nanC expression by binding to or in the vicinity of its promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15743943" target="_blank">15743943</a>]<br></li>
<li>Thus , in the absence of NagC binding , even in the presence of the lowered cyclic AMP ( cAMP ) / CAP concentration produced by growth on GlcNAc or glucose , cAMP / CAP can bind and activate nanC expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15743943" target="_blank">15743943</a>]<br></li>
<li>This shows that binding of NagC is necessary for expression of the glmU P1 promoter and implies that NagC is playing the role of activator for this promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7545108" target="_blank">7545108</a>]<br></li>
<li>In this case , binding of the NagC protein during growth-on-glucose results in a reduced , i.e. repressed , level of expression from the P2 promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7545108" target="_blank">7545108</a>]<br></li>
<li>However , it is nevertheless clear that loss of NagC function in the D3 mutant background is indeed compensated for by an increased efficacy of NanR as an activator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17238917" target="_blank">17238917</a>]<br></li>
<li>It would not be surprising if , like NanR , NagC , and OmpR , CpxR plays a role in the control of type 1 ﬁmbriation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15743943" target="_blank">15743943</a>]<br></li>
<li>Repression is calculated as the ratio of ` control ' to ` p ( NagC ) ' values .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7934873" target="_blank">7934873</a>]<br></li>
<li>The Interaction of NanR and NagC with Regions 1 and 2 in Vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15534208" target="_blank">15534208</a>]<br></li>
<li>Such differential actions are also observed for NanR and NagC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15743943" target="_blank">15743943</a>]<br></li>
<li>Here , we show that whereas NanR , a sialic acid-response-regulator , binds to region 1 , NagC , a GlcNAc-6P-responsive protein , binds to region 2 instead .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15534208" target="_blank">15534208</a>]<br></li>
<li>Sialic acid catabolism generates GlcNAc-6-P , and whereas GlcNAc disrupts methylation protection by NagC alone , Neu5Ac inhibits the protection mediated by both NanR and NagC as expected .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15534208" target="_blank">15534208</a>]<br></li>
<li>Thus , the loss of methylation protection at both regions 1 and 2 in the presence of Neu Ac is explained by the fact 5 that the Neu5Ac generates the inducing signal for NanR , and its metabolism produces GlcNAc-6-P , the inducing signal for NagC ( 41 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15534208" target="_blank">15534208</a>]<br></li>
<li>In addition the operon is repressed by NagC and full induction requires that sufﬁcient amounts of GlcNAc-6P are generated to displace the NagC repressor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15066032" target="_blank">15066032</a>]<br></li>
<li>It is possible that the ﬂux of chitobiose uptake produced from chitin degradation , although sufﬁcient to convert the ChbR protein into an activator , is not sufﬁcient to generate a sufﬁciently high concentration of GlcNAc-6P , the inducing signal for NagC , to displace NagC from its binding sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15066032" target="_blank">15066032</a>]<br></li>
<li>This is consistent with the hypothesis that growth on chitin under these conditions does not produce a sufﬁciently rapid ﬂux of chitobiose to GlcNAc-6P to produce a concentration sufﬁcient to induce NagC repressed genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15066032" target="_blank">15066032</a>]<br></li>
<li>Genes nagA , B , and E are members of the nag regulon which is regulated by the repressor NagC ( nagC ) and the inducer GlcNAc 6P ( Plumbridge , 1989 ; Vogler and Lengeler , 1989 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10931310" target="_blank">10931310</a>]<br></li>
<li>Preliminary experimental evidence ir ^ dicates that NagC could be a repressor for the nag regulon since disrupting the nagC gene on the chromosome results in activation of both nagE and nagE promoters in the absence of the inducer GlcNAc .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2668691" target="_blank">2668691</a>]<br></li>
<li>On the other hand , the activity of ChbG and ChbF enzymes leads to the production of GlcNAc-6P , the inducer for NagC ( Fig. 6 ) ( Plumbridge , 1991 ; Verma and Mahadevan , 2012 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24593230" target="_blank">24593230</a>]<br></li>
<li>Its subsequent metabolism generates GlcNAc-6P , the inducer for NagC , which displaces NagC from its sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15066032" target="_blank">15066032</a>]<br></li>
<li>GlcNAc-6-P is the inducer for NagC , and releases it from its operator sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17824929" target="_blank">17824929</a>]<br></li>
<li>The inducer for NagC is GlcNAc-6-P ( Plumbridge , 1991 ) , while for XylR from B. subtilis , it is the unphosphorylated sugar , xylose ( Gärtner et al. , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767573" target="_blank">9767573</a>]<br></li>
<li>The inducer , which prevents NagC from binding to its operators , has been shown to be GlcNAc-6-P ( Plumbridge , 1991 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10411743" target="_blank">10411743</a>]<br></li>
<li>The inducer for the NagC repressor is GlcNAc-6-P ( 31 ) , showing that growth on ManNAc produces amounts of GlcNAc-6-P that are sufﬁcient to displace NagC from its operators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9864311" target="_blank">9864311</a>]<br></li>
<li>The inducer for NagC is GlcNAc-6P .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15066032" target="_blank">15066032</a>]<br></li>
<li>The inducer for NagC is GlcNAc-6-P , the product of the transport of GlcNAc by the PTS ( 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11139621" target="_blank">11139621</a>]<br></li>
<li>In order to account for GlcNAc-6-P production , we hypothesize that ChbG removes the acetyl group only from the nonphosphorylated end of chitobiose-6-P , resulting in monoacetylchitobiose-6-P that upon subsequent hydrolysis by the phospho - - glucosidase ChbF generates the inducing signal for NagC , GlcNAc-6-P .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22797760" target="_blank">22797760</a>]<br></li>
<li>The difference in the growth of the wildtype strain on chitobiose and chitosan dimer is most likely due to the difference in the efﬁciency of NagC derepression by GlcNAc-6-P and GlcN-6-P , expected to be generated by the hydrolysis of chitobiose and chitosan dimer , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22797760" target="_blank">22797760</a>]<br></li>
<li>Deacetylation at the same end as the phosphate would result in GlcN-6-P instead of GlcNAc-6-P after hydrolysis by ChbF , which is a poor inducer of NagC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22797760" target="_blank">22797760</a>]<br></li>
<li>Hydrolysis of monoacetylchitobiose-6-P by ChbF produces GlcNAc-6-P , the signal for NagC derepression , and GlcN .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22797760" target="_blank">22797760</a>]<br></li>
<li>Hydrolysis of the dimer by ChbF is not necessary to generate the induction signal for ChbR activation , but it is necessary for the derepression of NagC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22797760" target="_blank">22797760</a>]<br></li>
<li>Chitosan dimer-6-P can act as inducer for ChbR , but its hydrolysis by ChbF does not produce GlcNAc-6-P required for NagC derepression , resulting in poor induction and consequently poor growth .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22797760" target="_blank">22797760</a>]<br></li>
<li>For all three systems , NagC , FadR and FruR , release of the bound regulatory protein is achieved by the presence of the inducing metabolite in the cell , GlcNAc-6-phosphate for NagC , long chain acyl-CoAs in the case of FadR and fructose-l-phosphate for FruR ( Ramseier et al. , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7545108" target="_blank">7545108</a>]<br></li>
<li>NagC coordinates the expression of genes involved in the degradation and biosynthesis of GlcNAc and GlcN ( Plumbridge , 1995 ) and the inducing signal for NagC displacement is GlcNAc-6P ( Plum-bridge , 1991 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15066032" target="_blank">15066032</a>]<br></li>
<li>NagC also functions to activate expression of glmS , encoding a glucosamine-6-phosphate synthase involved in the synthesis of N-acetylglucosamine , thereby coordinately regulating the synthesis and degradation of N-acetylglucosamine ( Plumbridge et al. , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14731281" target="_blank">14731281</a>]<br></li>
<li>Results NagC binds upstream of chiP in E. coli and Salmonella but ChbR only binds in Salmonella DNase I footprinting conﬁrmed that NagC bound to a site upstream of chiP overlapping the − 10 and − 35 regions of the chiP promoter ( site NagC1 ) in S. enterica serovar Typhimurium LT2 ( henceforth referred to as Salmonella ) as well as in E. coli ( Fig. 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24593230" target="_blank">24593230</a>]<br></li>
<li>Although the proximal site , BoxGl , is located at a position typical of prokaryotic activators ( Collado-Vides et al. , 1991 ; Gralla , 1991 ) , it is not sufficient to allow NagC to to activate glmU by itself ; this could be because NagC occupation of this lower affinity site is inadequate in the absence of the higher affinity upstream site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7545108" target="_blank">7545108</a>]<br></li>
<li>In this case the upstream site is needed to increase the local NagC concentration at BoxGl ( a chelation effect ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7545108" target="_blank">7545108</a>]<br></li>
<li>The experiments were carried out in the presence of an excess of non-homologous competitor DNA to eliminate possible problems owing to the presence of Escherichia coii DNA in the crude bacterial extracts from the overproducing strain used as a source of NagC protein DNA toop formation and nagE-B expression 977 Table 2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7934873" target="_blank">7934873</a>]<br></li>
<li>When Mlc binds to the two operators , it also alters the cleavage pattern of the intervening DNA but not to the same extent as NagC ( Fig. 3A and B , lanes 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11139621" target="_blank">11139621</a>]<br></li>
<li>Mlc binds to the mlc regulatory region A DNA fragment containing part of mlc and its upstream regulatory region was labelled , and DNaseI protection assays were performed in the presence of extracts containing Mlc or NagC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9484893" target="_blank">9484893</a>]<br></li>
<li>This stabilization seems to be caused by an effect on the curvature of the DNA rather than any direct protein -- protein contacts between the rep-ressor , NagC , and CAP ( J. A. Plumbridge and A. Kolb , in preparation ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9484892" target="_blank">9484892</a>]<br></li>
<li>However , unlike the nagE-B region , there is no change in the sensitivity of the intervening DNA to DNase I. Adding CAMP to the NagC extracts permits the visualization of the CAP induced pattern of hypersensitive bands and the NagC protection at box M , suggesting that again the two proteins are binding simultaneously ( Fig. 9 , lane 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1848637" target="_blank">1848637</a>]<br></li>
<li>Interestingly NagC and CAP bind together and their simultaneous binding produces a slight modiﬁcation in the pattern of DNAse I cleavages between the two NagC boxes ( Fig. 2 , lanes 3 and 12 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15066032" target="_blank">15066032</a>]<br></li>
<li>ChbR is also capable of binding with NagC and CAP and all three proteins can bind simultaneously ( Fig. 2 , lane 8 and 17 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15066032" target="_blank">15066032</a>]<br></li>
<li>We tested the effect of adding low concentrations of cAMP/CAP ( 20 nM CAP with 200  iM cAMP ) on the binding of NagC to the wild-type and mutated nag DNA fragments ( Fig. 2 , lanes 4 , 5 ; 10-12 ; 16 , 17 ; 22-25 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7934873" target="_blank">7934873</a>]<br></li>
<li>) We note that the binding of CAP , as indicated by the change in the intensity of the three hypersensitive bands ( marked with arrows in Fig. 2 ) , is stronger to the wild-type and +10 mutant DNAs where NagC binds to form a loop .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7934873" target="_blank">7934873</a>]<br></li>
<li>( As we are using bacterial extracts containing an unknown concentration of NagC , we have no measure of the NagC binding constant and it is possible that at the concentrations of extract and DNA employed , binding is stoichiometric to the high afﬁnity super-operators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7791215" target="_blank">7791215</a>]<br></li>
<li>Secondly , we performed a competition experiment for NagC binding to labelled nag DNA in the presence of unlabelled NagC box DNA fragments .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1848637" target="_blank">1848637</a>]<br></li>
<li>It should also be noted that the presence of NagC enhances the binding of CAP to nag DNA : compare the binding of CAP from crude extracts in the presence or absence of NagC ( Fig. 3 ( b ) , lanes 5 and 9 and data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1848637" target="_blank">1848637</a>]<br></li>
<li>These results imply that crude extracts containing NagC form stable complexes with nag DNA only in the presence of CAMP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1848637" target="_blank">1848637</a>]<br></li>
<li>CAP protein or the NagC extracts were diluted into the same buffer and mixed with the labelled DNA in the presence , where indicated , of 200 PM-CAMP ; at room temperature .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1848637" target="_blank">1848637</a>]<br></li>
<li>DNase I protection experiments of complexes formed between nag DNA 5 ' end labelled at the Hind111 site and extracts from a NagC overproducing strain in the presence and absence of CAMP and CAP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1848637" target="_blank">1848637</a>]<br></li>
<li>Binding of CAMP-CAP or NagC to the labelled DNA ( approx .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1848637" target="_blank">1848637</a>]<br></li>
<li>CAP and NagC can bind simultaneously and produce a complex more stable than the binary NagC-DNA complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1848637" target="_blank">1848637</a>]<br></li>
<li>D , free DNA ; Cl , 1st CAP complex ; C2 , 2nd CAP complex ; C3 , NagC complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1848637" target="_blank">1848637</a>]<br></li>
<li>The co-operative binding of CAP and NagC in the complex should allow CAP to-bind at lower CAMP concentrations than would be required on a linear DNA fragment .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1848637" target="_blank">1848637</a>]<br></li>
<li>Either NagC interacts directly with CAMP or the stable binding of NagC to the DNA requires the CAMP-CAP complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1848637" target="_blank">1848637</a>]<br></li>
<li>Effect of CAP and NagC binding on methylation of nag DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1848637" target="_blank">1848637</a>]<br></li>
<li>NagA does not behave as a DNA-binding protein and thus it seems unlikely that NagC and NagA form a dimeric repressor or that they bind co-operatively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1766379" target="_blank">1766379</a>]<br></li>
<li>1989 ) , Both the DNA-binding activity of NagC and the deacetylase activity of NagA are stable through several cycles of freezing and thawing .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1766379" target="_blank">1766379</a>]<br></li>
<li>However , DNaseI footprinting gave no evidence of DNA loop formation in vitro in the presence of the NagC repressor ( Plumbridge and Kolb , 1991 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9484892" target="_blank">9484892</a>]<br></li>
<li>To identify the DNA sequence important for NagC recognition , we have taken advantage of the fact that repression of the nagE and nagB genes requires the formation of a loop of DNA between molecules of the repressor bound to the nagE and nagB operators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7791215" target="_blank">7791215</a>]<br></li>
<li>Normally , DNA loop formation between NagC bound to the nagE and nagB operators is necessary for repression ( 1 ) but the nagE s.o. mutation increased the nagE operator affinity so much that repression in the absence of DNA loop formation was detected ( 16 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11139621" target="_blank">11139621</a>]<br></li>
<li>NagC regulation of the divergent nagE-B promoters depends upon DNA loop formation between NagC bound to operators overlapping the nagE and nagB promoters ( Plumbridge and Kolb , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767573" target="_blank">9767573</a>]<br></li>
<li>Discussion A system to analyse NagC protein -- DNA interactions Previous experiments have shown that repression of the divergent nagE-B promoters by the NagC repressor is dependent on an interaction between the repressor bound to the two operators and formation of a loop of DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7791215" target="_blank">7791215</a>]<br></li>
<li>They are negatively controlled by the NagC protein , which binds to the two operators overlapping the divergent nagE-B promoters , forming a DNA repression loop ( 33 , 34 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626331" target="_blank">8626331</a>]<br></li>
<li>As described in the Introduction , NagC regulates nagE -- B expression via DNA loop formation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9484892" target="_blank">9484892</a>]<br></li>
<li>Binding of NagC to the upstream site , which could produce greater repression via DNA loop formation , has thus been lost .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24593230" target="_blank">24593230</a>]<br></li>
<li>This bend makes an essential contribution to DNA loop formation and hence is necessary for repression by the NagC protein ( 19 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11139621" target="_blank">11139621</a>]<br></li>
<li>In an attempt to identify the intracellular inducer of the nag regulon , these phosphorylated sugars as well as the non-phosphorylated forms were tested for their ability to displace NagC from its DNA target in in vitro binding experiments .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1766379" target="_blank">1766379</a>]<br></li>
<li>Formation of a DNA loop has been demonstrated or deduced to be required for NagC regulation at these sites , since it allowed NagC to-bind to the lower affinity but promoterproximal site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24593230" target="_blank">24593230</a>]<br></li>
<li>Thus , in the absence of any interaction between the repressor proteins bound to two sites on the same template , the binding of NagC and Mlc to DNA is essentially identical , showing it is really the co-operative interaction that produces the distorted inter-operator footprints shown in Figure 3 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11139621" target="_blank">11139621</a>]<br></li>
<li>( d ) Short DNA fragments corresponding to the NagC boxes inhibit NagC binding : evidence for interaction between repressor bound to the NagC boxes In experiments where the nag DNA was titrated with different dilutions of the NagC extracts , we always saw simultaneous binding of repressor to the two sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1848637" target="_blank">1848637</a>]<br></li>
<li>( d ) Short DNA fragments corresponding to the NagC boxes inhibit NagC binding : evidence for interaction between repressor bound to the NagC boxes Figure 6 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1848637" target="_blank">1848637</a>]<br></li>
<li>Either cAMP/CAP enhances the interaction between NagC bound to BoxB and BoxE by stabilising the bend in the DNA between the two sites ( i.e. effect on DNA conformation ) or cAMP/CAP stabilises NagC binding by direct protein -- protein interactions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7791215" target="_blank">7791215</a>]<br></li>
<li>from two promoters present in front of glmU , which are subject to regulation by the DNA-binding protein NagC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17824929" target="_blank">17824929</a>]<br></li>
<li>The superpositioning of the DNase I hypersensitive sites induced by CAP and NagC binding show that the two proteins are binding in phase and suggest that the DNA is wrapped around a NagC-CAP protein core ( see Raibaud et al. , 1989 ; Travers , 1989 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1848637" target="_blank">1848637</a>]<br></li>
<li>The simultaneous binding of repressor and activator type proteins to nag DNA implies two corollaries : ( 1 ) NagC does not repress by competition with CAP and ( 2 ) CAP does not activate by displacing NagC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1848637" target="_blank">1848637</a>]<br></li>
<li>The glmUS operon , which is regulated positively by NagC , also relies upon co-operative binding of NagC to two sites ( 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11139621" target="_blank">11139621</a>]<br></li>
<li>The repressor is still bound to the two sites , but non co-operatively : binding to BoxB occurred at lower NagC concentrations than to BoxE and the regularly spaced hypersensitive DNase I cleavages , characteristic of DNA loop formation , were not observed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7934873" target="_blank">7934873</a>]<br></li>
<li>Repression is dependent upon the co-operative binding of NagC to these two sites resulting in DNA loop formation ( 1 ) , which can be detected in vitro by DNaseI footprinting .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11139621" target="_blank">11139621</a>]<br></li>
<li>We conclude that binding of NagC to its two sites on wild-type nag DNA is a co-operative event resulting in formation of a loop which enhances repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1848637" target="_blank">1848637</a>]<br></li>
<li>One reason why no loop formation is detected in vitro could be because some other factor is necessary to permit NagC protein molecules bound to the two sites to approach .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7545108" target="_blank">7545108</a>]<br></li>
<li>In the repressed state , NagC and ChbR sites are occupied , probably forming a complex structure involving an interaction between the two NagC proteins and formation of a DNA loop .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15066032" target="_blank">15066032</a>]<br></li>
<li>Compared with other operons regulated by NagC , nagE-BACD and glmUS , it is very likely that the repression complex involves an interaction between NagC bound to its two sites , NagC1 and NagC2 , so that a DNA loop is formed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15066032" target="_blank">15066032</a>]<br></li>
<li>- cm NagC NagAC Comrol NagB NagA NagC NagAC Cwtrol NagC binds to the nagE-nagB regutatory region but NagA does not nagC : : cm One possible explanation of the above results is that the nagA gene , as well as encoding the enzyme deacetylase , also codes for a co-repressor of the regulon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1766379" target="_blank">1766379</a>]<br></li>
<li>Binding of NagC to a mutated nagE-B opera ¬ + + I + ¬ NagC + H - + t t Fig. 8 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1848637" target="_blank">1848637</a>]<br></li>
<li>Binding of CAP and NagC upstream from manX .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1848637" target="_blank">1848637</a>]<br></li>
<li>Table 3 shows that an increase in the intracellular NagC concentration represses both nagE and nagB in the - f6 mutation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7934873" target="_blank">7934873</a>]<br></li>
<li>NagC binds best to the nagE s.o. , then to the nagB operator an has lowest affinity for the nagE w.t. ( wild-type ) operator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11139621" target="_blank">11139621</a>]<br></li>
<li>Simultaneous binding of NagC repressor and CAP to the nagE-nagB intergenic region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1848637" target="_blank">1848637</a>]<br></li>
<li>NagC binds to the intergenic nagE-nagB region and upstream of manX .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1766379" target="_blank">1766379</a>]<br></li>
<li>In the case of NagC operator , the minor groove centred at the dyad axis might be more accessible in the presence of the repressor since the central A residue of the nagE and nagB operators is hypermethylated by DMS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7791215" target="_blank">7791215</a>]<br></li>
<li>Binding of NagC repressor to isolated operators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7791215" target="_blank">7791215</a>]<br></li>
<li>While Mlc binds well to both operators , NagC is only a weak repressor for mlc and does not repress malT .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9484893" target="_blank">9484893</a>]<br></li>
<li>However mutagenesis experiments exchanging the Gln to Glu in NagC , and vice versa in Mlc , gave no indication of a specificity change ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11139621" target="_blank">11139621</a>]<br></li>
<li>NagC bound to the NagC selected sites ( C5 , C9 and C42 ) at least 100-fold better than to the Mlc sites ( M1 , M26 ) and at least 10 times better than to M3 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11139621" target="_blank">11139621</a>]<br></li>
<li>Positive interaction between NagC and the C/G base pair at +11 / -- 11 presumably accounts for the higher affinity of NagC for its sites in vitro compared with Mlc for its sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11139621" target="_blank">11139621</a>]<br></li>
<li>Plasmids overproducing either the NagC protein or the Mlc protein repress the expression of manX , but the effect of the Mlc protein is stronger .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9484892" target="_blank">9484892</a>]<br></li>
<li>Effect of overproduced NagC and Mlc on manX -- lacZ and borG -- lacZ expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9484892" target="_blank">9484892</a>]<br></li>
<li>Regulation of PTS gene expression by the homologous transcriptional-regulators , Mlc and NagC , in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15150239" target="_blank">15150239</a>]<br></li>
<li>El Qaidi , S. , and Plumbridge , J. ( 2008 ) Switching control of expression of ptsG from the Mlc regulon to the NagC regulon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19007420" target="_blank">19007420</a>]<br></li>
<li>Mlc and NagC are not the only examples of proteins with overlapping specificity in E.coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11139621" target="_blank">11139621</a>]<br></li>
<li>Even though the helix -- turn -- helix motifs of Mlc and NagC are almost identical and both proteins bind to similar sequences , there are distinctions in their operator selection .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9484893" target="_blank">9484893</a>]<br></li>
<li>On the other hand , NagC did bind to the Mlc regulatory region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9484893" target="_blank">9484893</a>]<br></li>
<li>NagC and Mlc can bind to each others operator , at least in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11139621" target="_blank">11139621</a>]<br></li>
<li>As described above , NagC and Mlc both bind very well to M134 in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11139621" target="_blank">11139621</a>]<br></li>
<li>For both Mlc and NagC sites the affinities in vivo seem to increase with the presence of more bases in the CGCG configuration in positions -- 4 to +4 ( shown in pink in Fig. 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11139621" target="_blank">11139621</a>]<br></li>
<li>This amino acid is Gln in NagC and Glu in Mlc ( Fig. 1A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11139621" target="_blank">11139621</a>]<br></li>
<li>The relative occupancy of each operator by Mlc or NagC was calculated by comparing the intensity of three bands within each operator in the presence of NagC or Mlc with that in their absence ( Fig. 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11139621" target="_blank">11139621</a>]<br></li>
<li>The relative concentrations of Mlc and NagC are shown at the top of each lane .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11139621" target="_blank">11139621</a>]<br></li>
<li>The last compound is the inducer for NagC ( Plum-bridge , 1991 ) , the repressor of the nagE -- BACD operons and a homologue of Mlc .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9484893" target="_blank">9484893</a>]<br></li>
<li>The nagE operator does exhibit a higher affinity for Mlc than NagC , but no regulation of nagE by physiological concentrations of Mlc was detected in vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11139621" target="_blank">11139621</a>]<br></li>
<li>The implications of this experiment were that the nagE operator , the part of the divergent nagE-B operon system that is known to be controlled by NagC , appeared to have the characteristics of a Mlc operator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11139621" target="_blank">11139621</a>]<br></li>
<li>Binding of NagC and Mlc to nagE operators in vivo As Mlc showed a stronger affinity for the nagE operator than NagC , it seemed possible that Mlc might be able to regulate the nagE promoter in the absence of NagC in vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11139621" target="_blank">11139621</a>]<br></li>
<li>To verify that Mlc does bind preferentially to the nagE operator in vivo , the effect of plasmids overproducing either Mlc or NagC was tested in a strain mutated for both mlc and nagC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11139621" target="_blank">11139621</a>]<br></li>
<li>NagC is best characterized as a repressor ( Plumbridge , 2001 ; Plumbridge and Pellegrini , 2004 ) , although the regulator does activate one of the glmUS operon promoters albeit by binding at the more conventional position of -47 relative to the transcriptional start site ( Plumbridge , 1995 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17238917" target="_blank">17238917</a>]<br></li>
<li>NagC ) were made after IPTG induction as described previously ( Plumbridge and Kolb .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1766379" target="_blank">1766379</a>]<br></li>
<li>This demonstrates that both promoters are equally induced during growth on GlcNAc , i.e. even though the two operators have different affinities for the repressor ( Plumbridge and Kolb , 1991 , and see below ) , the presence of the intracellular inducer results in the simultaneous displacement of NagC from both its binding sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7934873" target="_blank">7934873</a>]<br></li>
<li>This fourfold repression of glmS corresponds well to the known fourfold reduction of glmUS promoter activity by GlcNAc that results from the release of NagC binding ( Plumbridge , 1995 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17824929" target="_blank">17824929</a>]<br></li>
<li>In particular , two operators are necessary for repression , and loss of NagC binding produces 40-fold derepression ( Plumbridge and Kolb , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9484893" target="_blank">9484893</a>]<br></li>
<li>Surprisingly , although the NagC1 binding site overlaps the promoter , excess NagC either does not bind efﬁciently alone or its binding is not sufﬁcient to inhibit RNA polymerase access to the promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15066032" target="_blank">15066032</a>]<br></li>
<li>However in the presence of cAMP/CAP , NagC showed the same apparent binding afﬁnity to wild-type and super-operators indicating that one role of cAMP/CAP in the repression complex is to reduce the need for high afﬁnity sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7791215" target="_blank">7791215</a>]<br></li>
<li>The repression observed by excess NagC , expressed from a plasmid , is presumably the result of NagC binding to its weaker site , BoxM8 , which overlaps the transcription start point .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9484892" target="_blank">9484892</a>]<br></li>
<li>Elimination of NagC binding to the strong distal NagC binding site by mutation should prevent binding of NagC to the weaker proximal site which overlaps the promoter and which should be the site actually responsible for repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15066032" target="_blank">15066032</a>]<br></li>
<li>represses weakly by itself and strongly in the presence of nagC + , implying that part of the repression by ChbR involves enhancing the binding of NagC to its promoter proximal site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15066032" target="_blank">15066032</a>]<br></li>
<li>Thus cAMP/CAP replaces the need for a strong NagC site to maintain repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7791215" target="_blank">7791215</a>]<br></li>
<li>The location of its binding site , between galOE1 and nagO1 , raises the question of whether cAMP/CAP , via its DNA bending activity , could also participate in formation of the NagC / GalR/GalS repression complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19007420" target="_blank">19007420</a>]<br></li>
<li>The competition between NagC and CAP for binding to the same site explains the complex catabolite repression pattern of nanC expression ( Table 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15743943" target="_blank">15743943</a>]<br></li>
<li>The intracellular formation of GlcNAc releases NagC from its site and in consequence allows the binding of CAP to its overlapping site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15743943" target="_blank">15743943</a>]<br></li>
<li>The square brackets M and M ' indicate regions strongly and weakly protected against DNase I by binding of NagC Expression of both nagE and nagB was shown to be stimulated by CAP in viva , although nagE was much more dependent than nagB ( Plumbridge , 1990 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1848637" target="_blank">1848637</a>]<br></li>
<li>Keywords : protein -- DNA recognition ; nagE-B operator ; NagC repressor ; cAMP/CAP Physico-chimique ( URA1139 ) 13 , rue Pierre et Marie Curie 75005 Paris , France 2Institut Pasteur ( URA1149 ) 25 , rue de Dr Roux , 75015 Paris , France * Corresponding author Introduction The Nag repressor controls expression of the divergent nagE-nagBACD operons by binding to two operators , called BoxE and BoxB , which overlap the nagE and nagB promoters ( Plumbridge & Kolb , 1991 , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7791215" target="_blank">7791215</a>]<br></li>
<li>NagC regulates expression of the divergent nagE-BACD operons by binding to two operators ( called BoxE and BoxB ) overlapping the nagE-B promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11139621" target="_blank">11139621</a>]<br></li>
<li>An alternative explanation , that NagC requires GalR/S to enhance its binding , as proposed for HU binding at galE ( Kar and Adhya , 2001 ) seems not to be the case .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19007420" target="_blank">19007420</a>]<br></li>
<li>Estimates of the relative binding strengths of NagC to this fragment and the BoxB wild-type fragment show that the mutated BoxE binds NagC about three to four times more strongly than the BoxB operator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7791215" target="_blank">7791215</a>]<br></li>
<li>Its weak NagC binding ability is considerably enhanced by the chelation effect due to the high-affinity nagB operator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11139621" target="_blank">11139621</a>]<br></li>
<li>An unusual feature of three of these repressor binding sites was the observation that an adenine residue in the centre of the region protected against DNAseI cleavage , was hypermethylated by DMS , when the NagC repressor was bound .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7791215" target="_blank">7791215</a>]<br></li>
<li>No G residue contacts were detected by D&IS reactivity in the NagC binding sites , which is perhaps not surprising considering the -4 - t T-rich nature of the NagC boxes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1848637" target="_blank">1848637</a>]<br></li>
<li>The binding of NagC produced no effect on the G methylation pattern , but the reaction with one A residue was strongly enhanced near the centre of both NagC boxes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1848637" target="_blank">1848637</a>]<br></li>
<li>The fusions carrying mutations in galOI and nagO2 behaved essentially as wt , conﬁrming that these sites , which show negligible binding of GalR or NagC , do not contribute to regulation of galP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19007420" target="_blank">19007420</a>]<br></li>
<li>To investigate the importance of the NagC boxes for the regulation , mutations were introduced into each box to prevent NagC binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7545108" target="_blank">7545108</a>]<br></li>
<li>The Mlc protein binds to the NagC operators , binding preferentially to the promoter-proximal operator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9484892" target="_blank">9484892</a>]<br></li>
<li>The site at which Mlc binds shows homology to the previously identiﬁed NagC operators ( Plumbridge and Kolb , 1995 ) , but no NagC binding to it was detected ( Fig. 1 , lane 9 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9484893" target="_blank">9484893</a>]<br></li>
<li>The introduction of the mlc mutation does seem to provoke a small increase ( 1.3-fold ) , which could be explained by some weak Mlc binding in the absence of NagC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11139621" target="_blank">11139621</a>]<br></li>
<li>The presence of -- 11C / +11 G can explain the good NagC binding but the strong binding to Mlc , despite there being a C/G at +11 / -- 11 , must reflect another affinity factor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11139621" target="_blank">11139621</a>]<br></li>
<li>The relative derepression caused by each sequence is a measure of its binding affinity for Mlc or NagC ( Table 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11139621" target="_blank">11139621</a>]<br></li>
<li>The inherent binding of NagC to its specific sites was ~ 10-fold stronger than Mlc to its specific sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11139621" target="_blank">11139621</a>]<br></li>
<li>The stronger binding of NagC to its sites , compared with Mlc binding to its sites , presumably reflects the importance of the C/G at +11 / -- 11 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11139621" target="_blank">11139621</a>]<br></li>
<li>NagC extracts were tested for binding to the cut fragments .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7791215" target="_blank">7791215</a>]<br></li>
<li>One criticism is that other proteins in the extracts can influence NagC or Mlc binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11139621" target="_blank">11139621</a>]<br></li>
<li>On all three templates the DNaseI footprint produced by Mlc binding is qualitatively the same as that of NagC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11139621" target="_blank">11139621</a>]<br></li>
<li>Rather surprisingly , none of the selected NagC binding sites produced > 8-fold induction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11139621" target="_blank">11139621</a>]<br></li>
<li>) RESULTS Selection of random Mlc and NagC binding sites DNA fragments with affinity for NagC or Mlc were selected .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11139621" target="_blank">11139621</a>]<br></li>
<li>However , at least one of the Mlc selected DNA binding sites , M120 , with limited homology to the NagC/Mlc consensus , did not bind to purified Mlc or NagC when tested in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11139621" target="_blank">11139621</a>]<br></li>
<li>To verify the protein-binding specificity of the selected sites , some were tested in vitro for binding to Mlc and NagC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11139621" target="_blank">11139621</a>]<br></li>
<li>Despite the fact that NagC is homologous to Mlc and that binding of NagC to both the ptsG and the ptsH operators can be detected by DNAse I footprinting in vitro , there is no effect of a nagC mutation on the expression of either ptsG or ptsH ( Table 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10411743" target="_blank">10411743</a>]<br></li>
<li>( Note , since the absolute amounts of NagC or Mlc in the extracts are unknown it is only possible to compare the relative binding of the same extract with the different operators and not the binding of NagC and Mlc with the same operator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11139621" target="_blank">11139621</a>]<br></li>
<li>Binding of NagC and Mlc to some ` selected ' operators in vivo To examine the relative binding affinities of some of the selected Mlc and NagC binding sites in vivo , an ` operator titration ' technique was used .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11139621" target="_blank">11139621</a>]<br></li>
<li>Recent experiments on the mlc-regulated genes showed that although NagC binding to Mlc operators could be detected in vitro , there was no or little regulation in vivo ( 6,17 , unpublished results ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11139621" target="_blank">11139621</a>]<br></li>
<li>These experiments show that with the physiological concentrations of Mlc present in the cell , there is little binding of Mlc to the nagE operator , even in the absence of NagC binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11139621" target="_blank">11139621</a>]<br></li>
<li>Footprint of NagC and Mlc binding to the single nagB and nagE operators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11139621" target="_blank">11139621</a>]<br></li>
<li>Although Mlc does bind specifically to both the nagE and nagB operators in vitro , no regulation of nagBE expression in vivo by the natural chromosomal copy of mlc was detected , even in the absence of NagC binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11139621" target="_blank">11139621</a>]<br></li>
<li>Negative regulation of manX expression by NagC and Mlc The DNase I footprint of NagC on the manX regulatory region had shown two NagC boxes with NagC binding more strongly to the upstream , promoter-distal site ( Plumbridge and Kolb , 1991 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9484892" target="_blank">9484892</a>]<br></li>
<li>Both of these operons are controlled by NagC binding to two sites and both NagC binding sites are essential for regulation , but one site shows inherently lower afﬁnity for NagC than the other ( Plumbridge , 1995 ; 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15066032" target="_blank">15066032</a>]<br></li>
<li>It is also signiﬁcant that , of the two binding sites upstream of manXYZ , the site showing higher afﬁnity for Mlc has lower afﬁnity for NagC ( Plumbridge , 1997 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9484893" target="_blank">9484893</a>]<br></li>
<li>This could be a consequence of the low affinity of the repressor for the upstream NagC site so that cooperative binding of NagC to the two operators is unlikely .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24593230" target="_blank">24593230</a>]<br></li>
<li>It is worth emphasising that NagC prefers G/C at +11 / -- 11 even when binding to a single site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11139621" target="_blank">11139621</a>]<br></li>
<li>In this case , the role of the upstream operator is to enhance the binding of NagC to the weaker downstream site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7545108" target="_blank">7545108</a>]<br></li>
<li>Furthermore , as NagC binding to ONC1 prevents CRP-dependent activation of nanC ( Condemine et al. , 2005 ) , this study should enhance our understanding of the regulation of this gene as well .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17238917" target="_blank">17238917</a>]<br></li>
<li>However , unlike Lrp and OxyR binding to their operator sequences at pap and ag43 respectively ( Nou et al. , 1993 ; Haagmans and van der Woude , 2000 ) , methylation does not seem to affect the binding of NagC to O .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17238917" target="_blank">17238917</a>]<br></li>
<li>Binding of NagC to operator sites present in front of glmU enhances glmUS transcription fourfold ( Plumbridge et al. , 1993 ; Plumbridge , 1995 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17824929" target="_blank">17824929</a>]<br></li>
<li>However , the in vivo results with the glmU-lacZ fusions demonstrate that the two NagC boxes are necessary to activate transcription from P1 and it seems reasonable to assume that an interaction between NagC molecules bound to the two sites is necessary for activation in vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7545108" target="_blank">7545108</a>]<br></li>
<li>The interaction of IHF and NagC in vitro Binding of IHF to the element investigated above would be expected to produce a hairpin-like bend in the DNA midway between ONC1 and ONC2 ( see Rice et al. , 1996 for a review ) , and if so , should either facilitate or prevent contact between NagC bound to its two operator sites depending upon the phasing of the sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17238917" target="_blank">17238917</a>]<br></li>
<li>Under uninduced conditions , transcription of the chb operon is repressed by the NagC repressor , which binds to two sites in the chb promoter region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24593230" target="_blank">24593230</a>]<br></li>
<li>In addition the NagC repressor binds to two sites upstream of the manXYZ operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7791215" target="_blank">7791215</a>]<br></li>
<li>In addition it was found to-bind NagC as well as the NagC sites tested ( C5 , C9 and C42 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11139621" target="_blank">11139621</a>]<br></li>
<li>Moreover it binds NagC as well as the other C sites tested .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11139621" target="_blank">11139621</a>]<br></li>
<li>At the moment we do not know how NagC activates the ﬁmB promoter , but two general scenarios seem most plausible .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17238917" target="_blank">17238917</a>]<br></li>
<li>Co-operative binding of NagC to two sites is required for regulation of most NagC controlled operons in E. coli ( nagEBACD , glmUS , chbBCARFG , ﬁmB ) , although the galP gene is controlled via a single operator with high affinity ( El Qaidi et al. , 2009 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24593230" target="_blank">24593230</a>]<br></li>
<li>However , all the other operons known to be controlled by NagC require two sites for NagC to function ( nagE-B , chb , glmU , ﬁmB ) ( El Qaidi and Plum-bridge , 2008 and refs within ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19007420" target="_blank">19007420</a>]<br></li>
<li>However , in both organisms repression of chiP by NagC is inefficient resulting in a relatively high basal level of chiP transcription in the absence of chitobi-ose ( Figueroa-Bossi et al. , 2009 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24593230" target="_blank">24593230</a>]<br></li>
<li>This seemed to imply another link between galactose and amino sugar metabolism as exempliﬁed by NagC participating with GalR and GalS in the repression of galP in E. coli ( El Qaidi et al. , 2009 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24593230" target="_blank">24593230</a>]<br></li>
<li>These genes are , however , only weakly repressed by NagC under normal cellular conditions ( 27 , 41 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626331" target="_blank">8626331</a>]<br></li>
<li>Mutations in nagC , galR and galS increase expression from a galP-lacZ fusion : galR is epistatic to galS To investigate the relative contributions of NagC , GalR and GalS to galP repression , we tested the effect of mutations in nagC , galR and galS in trans on the fusions in the three media ( Fig. 3A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19007420" target="_blank">19007420</a>]<br></li>
<li>We have found that NagC , binds to a single , high-affinity site overlapping the promoter of galP ( galactose permease ) gene and that expression of galP is repressed by a combination of NagC , GalR and GalS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19007420" target="_blank">19007420</a>]<br></li>
<li>If galP were repressed by NagC , then this could logically explain the better growth of the nagC mutant of MG1655 on galactose .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19007420" target="_blank">19007420</a>]<br></li>
<li>These observations are consistent with galP being controlled by NagC as well as GalR/S .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19007420" target="_blank">19007420</a>]<br></li>
<li>Interestingly , this novel mode of coregulation of galP by NagC does not seem to be well conserved within the g-proteobacteria .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19007420" target="_blank">19007420</a>]<br></li>
<li>Regulation of galP by three speciﬁc repressors , NagC , GalR and GalS is unusual in that it involves multiple , speciﬁc regulators from two different areas of metabolism .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19007420" target="_blank">19007420</a>]<br></li>
<li>It is interesting to compare the NagC -- Mlc pair with the isorepressors , GalS and GalR , of the gal regulon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9484892" target="_blank">9484892</a>]<br></li>
<li>B. Fluorescence anisotropy titration curves for Alexa 488 labelled GalR with GalS , GalR or NagC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19007420" target="_blank">19007420</a>]<br></li>
<li>Binding of NagC , ChbR , GalR and GalS to the regulatory regions of E. coli and Salmonella chiP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24593230" target="_blank">24593230</a>]<br></li>
<li>Regions protected by NagC , ChbR , GalR and GalS are indicated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24593230" target="_blank">24593230</a>]<br></li>
<li>Induction was normal in the presence of chitosan dimer in the chbG mutant when NagC repression was abrogated by deleting nagC , whereas chitobiose and chitotriose continued to be ineffective as inducers under the same conditions ( Fig. 4B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22797760" target="_blank">22797760</a>]<br></li>
<li>Further evidence that growth on chitin is limiting for induction of NagC is shown by the analysis of the effect of chitin and chitobiose on induction of a NagC-regulated nagB -- lacZ fusion .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15066032" target="_blank">15066032</a>]<br></li>
<li>Thus under both slow and fast growth conditions ( minimal or enriched glycerol media respectively ) the induction by exogenous chitobiose , produced by the action of the secreted chitinase on extracellular chitin , is more efﬁcient in the absence of NagC repression than in its presence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15066032" target="_blank">15066032</a>]<br></li>
<li>Thus the presence of NagC is limiting the use of the chitobiose produced from chitin plus chitinases .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15066032" target="_blank">15066032</a>]<br></li>
<li>This activation of transcription was lost when the assays were performed in a strain carrying the chromosomal nagC and a disruption of chbR , underscoring the necessity of the loss of NagC repression in acquiring a Cel + phenotype .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18028317" target="_blank">18028317</a>]<br></li>
<li>As described above , NagC repression was abrogated by deleting the nagC locus .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22797760" target="_blank">22797760</a>]<br></li>
<li>These were mutations that resulted in the loss of NagC repression and gain-of-function mutations in the chbR locus .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18028317" target="_blank">18028317</a>]<br></li>
<li>Induction is normally achieved by the abrogation of NagC repression and the subsequent activation of the chb promoter by ChbR and CRP-cAMP ( 45 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22797760" target="_blank">22797760</a>]<br></li>
<li>In the chbR strain the twofold induction corresponding to the loss of ChbR is slightly reduced by the NagC plasmid while the ChbR plasmid again produces greater than 30-fold repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15066032" target="_blank">15066032</a>]<br></li>
<li>So there is no strong repression of chbB -- lacZ by plasmid-encoded NagC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15066032" target="_blank">15066032</a>]<br></li>
<li>Effect of excess ChbR and NagC on repression of the chbB -- lacZ fusion .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15066032" target="_blank">15066032</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15066032" target="_blank">15066032</a>]<br></li>
<li>The subsequent metabolism of chitobiose by the enzymes of the chb operon , ChbFG ( Verma and Mahadevan , 2012 ) generates GlcNAc6P , which inactivates NagC leading to full-scale induction of both the chiP and chb operons .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24593230" target="_blank">24593230</a>]<br></li>
<li>Second , as the ﬂux of chitobiose through the ChbBCA transporter and ChbFG enzymes increases , the level of GlcNAc6P , the inducer for NagC , increases and relieves repression by NagC of both chiP and chb operons .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24593230" target="_blank">24593230</a>]<br></li>
<li>Since this level corresponds to the fully chitobiose-induced level of the nagC + ΔchiX strain , these results strongly suggest that residual chiP induction in ΔchiX bacteria grown on chitobiose results from a relief of NagC repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24593230" target="_blank">24593230</a>]<br></li>
<li>NagC repression is critical in maintaining chiP mRNA levels low enough , relative to ChiX , to allow full silencing by this sRNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24593230" target="_blank">24593230</a>]<br></li>
<li>Further metabolism of chitobiose generates GlcNAc6P , the inducing signal for NagC leading to loss of NagC repression and full expression of chbB and chiP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24593230" target="_blank">24593230</a>]<br></li>
<li>NagC is required for full repression of the chbB operon and it serves to limit induction when the supply of chitobiose is low .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15066032" target="_blank">15066032</a>]<br></li>
<li>NagC can bind alone in vitro ( Fig. 4B ) and repress in the galR , galS strain in vivo ( Fig. 3A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19007420" target="_blank">19007420</a>]<br></li>
<li>First , repression of nanC and fimB divergent transcripts by NagC has been observed both in vivo and in vitro [ 32 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21070640" target="_blank">21070640</a>]<br></li>
<li>Excess NagC permits repression in vivo Despite the fact that both the +6 and BoxB '' mutations seemed to result in almost complete derepression of bot nagEand nagB in vivo , it was possible to detect binding of NagC to both BoxE and BoxB of the - t-6 mutation in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7934873" target="_blank">7934873</a>]<br></li>
<li>This single operator site is very strongly bound by NagC such that the nagC mutation produces a 77-fold derepression ( as compared with 34-fold for the nagBE looping construct ) demonstrating its high affinity for NagC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11139621" target="_blank">11139621</a>]<br></li>
<li>However the much stronger repression of this fusion by overproduced Mlc , compared with overproduced NagC , shows that the isolated nagE operator site has a higher affinity for Mlc than NagC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11139621" target="_blank">11139621</a>]<br></li>
<li>For the single-site nagE -- lacZ fusion the NagC plasmid is surprisingly inefficient at repressing ( < 3-fold repression ) while the Mlc plasmid is 10 times better ( 28-fold repression ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11139621" target="_blank">11139621</a>]<br></li>
<li>As expected , the NagC plasmid is more efficient than the Mlc plasmid for the nagE -- lacZ s.o. fusion ( nearly 100-fold for NagC compared with just 8-fold repression by Mlc ) , which is consistent with the -- 11G , +11 C s.o. mutation creating a better NagC site and a worse Mlc site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11139621" target="_blank">11139621</a>]<br></li>
<li>Plasmids expressing either NagC or Mlc produce considerable repression of the nagBE -- lacZ looped fusion , reducing expression 88 - and 56-fold ( Table 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11139621" target="_blank">11139621</a>]<br></li>
<li>Rep ( repression ) is the relative decrease in expression of the fusion caused by the presence of the NagC or Mlc plasmid compared with the empty vector plasmid .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11139621" target="_blank">11139621</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11139621" target="_blank">11139621</a>]<br></li>
<li>The nagE -- lacZ single operator fusion is highly expressed in the wild-type strain , since NagC can not bind to the wild-type nagE operator in the absence of loop formation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11139621" target="_blank">11139621</a>]<br></li>
<li>The preferential repression by NagC at the looped nagBE -- lacZ fusion , compared with the nagE -- lacZ fusion with a single operator site , emphasises that regulation by NagC is normally dependent upon DNA loop formation involving the higher-affinity nagB operator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11139621" target="_blank">11139621</a>]<br></li>
<li>These super-operators allow a higher level of repression of the nagE promoter compared to the nagB , presumably due to the existence of linear complexes of NagC bound to BoxE .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7791215" target="_blank">7791215</a>]<br></li>
<li>The location of mutations MBE constructed in this work is indicated Results Identiﬁcation of the bases important for NagC binding Mutations which decrease repression A systematic series of mutations were created in the left-hand half of the NagC operator upstream of nagE , BoxE .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7791215" target="_blank">7791215</a>]<br></li>
<li>Binding of NagC to the BoxE super-operator in vitro and in vivo The mutations at +11 in BoxE are a little less effective ( twofold enhanced repression of nagB compared to threefold for changes at − 11 ) implying some kind of asymmetry / directionality in the operator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7791215" target="_blank">7791215</a>]<br></li>
<li>Replacing the A/T at +11 / -- 11 with C/G allows repression by NagC in the absence of the nagB operator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11139621" target="_blank">11139621</a>]<br></li>
<li>N-acetylglucosamine 6-phosphate ( GlcNAc6P ) , the product of the nagE-encoded PTS transporter , and also a downstream product of chitobiose metabolism ( Fig. 1B ) functions as the allosteric-effector of NagC inactivating the protein and leading to derepression of the nag and chb operon ( Plumbridge , 1991 ; Plumbridge and Pellegrini , 2004 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24593230" target="_blank">24593230</a>]<br></li>
<li>Expression of the chb operon is repressed by NagC , which regulates genes involved in amino sugar metabolism .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15066032" target="_blank">15066032</a>]<br></li>
<li>The latter step is critical for the initiation of the entire induction cascade , as the chbBCARFG operon itself is repressed by NagC ( Plumbridge and Pellegrini , 2004 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24593230" target="_blank">24593230</a>]<br></li>
<li>Many Cel + strains carry mutations in nagC The NagC repressor that regulates the nag operon involved in N-acetylglucosamine metabolism was also shown to regulate the chb operon ( Plumbridge and Pellegrini , 2004 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18028317" target="_blank">18028317</a>]<br></li>
<li>In the absence of an environmental supply of amino sugars , NagC represses the divergent nagE-BACD operons , which are necessary for the utilisation of GlcNAc as a carbon source ( 1 ) , and activates the expression of one promoter of the glmUS operon that is part of the biosynthetic pathway for the formation of GlcN-6-phosphate ( GlcN-6-P ) and UDP-GlcNAc ( 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11139621" target="_blank">11139621</a>]<br></li>
<li>In the absence of chitosugars ( uninduced ) the chb , chiP and nag operons are repressed by NagC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24593230" target="_blank">24593230</a>]<br></li>
<li>It has been shown that the three proteins ChbR , CRP and NagC regulate the expression of the chb operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18028317" target="_blank">18028317</a>]<br></li>
<li>The need for two speciﬁc inducing signals , one for ChbR to activate the expression of the operon and a second for NagC to relieve its repression , ensure that the chb operon is only induced when there is sufﬁcient ﬂux through the combined chb-nag metabolic pathway to activate expression of both the chb and nag operons .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15066032" target="_blank">15066032</a>]<br></li>
<li>Degradation of cellobiose does not produce the GlcNAc-6P necessary to relieve NagC repression , but in the absence of NagC repression cellobiose is capable of activating chb operon expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15066032" target="_blank">15066032</a>]<br></li>
<li>It thus appears that NagC plays a key role in setting chiP expression levels under uninduced conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24593230" target="_blank">24593230</a>]<br></li>
<li>In a nagC strain , the threefold increase in chbB -- lacZ expression compared with the wild type is , as expected , eliminated by the NagC plasmid .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15066032" target="_blank">15066032</a>]<br></li>
<li>Inactivation of the NagC repressor causes chiP expression to increase no more than twofold to threefold in ΔchiX background .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24593230" target="_blank">24593230</a>]<br></li>
<li>Effect of an increase in NagC repressor concentration on the expression of the protein fusions canning wiid-type and mutated nag regulatory regions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7934873" target="_blank">7934873</a>]<br></li>
<li>This suggested that the phenotypic effect of the mutations within chbR can be elicited only in the absence of negative regulation by NagC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18028317" target="_blank">18028317</a>]<br></li>
<li>Possibly stronger NagC repression is not useful within the present context of post-transcriptional regulation by the sRNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24593230" target="_blank">24593230</a>]<br></li>
<li>A further level of complexity to the regulation of the chb operon is the involvement of NagC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15066032" target="_blank">15066032</a>]<br></li>
<li>What is the rationale for regulation of the galactose transport by NagC ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19007420" target="_blank">19007420</a>]<br></li>
<li>The chiP gene is under the transcriptional control of the NagC repressor As the next step in our comparative study , we examined the effects of nagC and chbR mutations on regulation of chiP -- lacZ translational fusions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24593230" target="_blank">24593230</a>]<br></li>
<li>These consist of mutations that abrogate negative regulation by the repressor NagC as well as single base-pair changes in the transcriptional-regulator chbR that translate into single-amino-acid substitutions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18028317" target="_blank">18028317</a>]<br></li>
<li>The complete absence of any growth in the case of the permease ( chbA ) mutant and the phospho - - glucosidase ( chbF ) mutant compared to the parent strain when NagC is inactive suggests that chitosan dimer is transported and hydrolyzed by the chb operon ( Table 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22797760" target="_blank">22797760</a>]<br></li>
<li>The CAP ( = CRP = cAMP receptor protein = catabolite activator protein ) binds , in the presence of cAMP , to a site adjacent to the distal NagC site ( Fig. 2 , Lanes 2 and 11 ) centred at -83.5 from the transcriptional start site although protection extends back as far as -100 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15066032" target="_blank">15066032</a>]<br></li>
<li>The observation that NagC is required for methylation protection at region 2 indicated that GlcNAc would stimulate methylation of this site specifically , and this was found to be the case ( Fig. 4C , lanes 5 -- 7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15534208" target="_blank">15534208</a>]<br></li>
<li>The very high derepression produced by the insertions in nagA is , however , the result of the lack of NagA protein itself and not a reduction in the concentration of NagC , since it is only fully complemented by plasmids expressing nagA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1766379" target="_blank">1766379</a>]<br></li>
<li>Effect of the NagC boxes on expression of P2 The primer extension and SI experiments ( Figures 3 and 4 ) suggested that when P1 was activated the transcript from P2 was reduced in intensity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7545108" target="_blank">7545108</a>]<br></li>
<li>The results of this analysis conﬁrm this assertion , and show additionally that NagC is also better able to activate ﬁmB expression when the operator sites involved are situated closer to the promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17238917" target="_blank">17238917</a>]<br></li>
<li>There are two NagC operator sites upstream of gimU The perfect inverse correlation between the induction of nag and repression of glm genes posed the question as to whether the regulation of glmU expression involved the NagC protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7545108" target="_blank">7545108</a>]<br></li>
<li>The effect of NagC on the regulation of munX expression in vivo and the relative importance of the sites M and M ' is currently under investigation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1848637" target="_blank">1848637</a>]<br></li>
<li>The displacement of a second repressing protein , NagC , could serve the equivalent function for ChbR as the occupation of a low afﬁnity site for AraC and MelR , i.e. allow ChbR to switch into activation mode .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15066032" target="_blank">15066032</a>]<br></li>
<li>Discussion The results described here show that three transcription factors , ChbR , NagC and cAMP/CAP , cooperate in the repression and induction of the chbBCARFG operon in E. coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15066032" target="_blank">15066032</a>]<br></li>
<li>The repression by NagC was epistatic over the transcriptional activation by mutant ChbR in the presence of cellobiose .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18028317" target="_blank">18028317</a>]<br></li>
<li>The formation of the loop allows repression at lower NagC concentrations .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7934873" target="_blank">7934873</a>]<br></li>
<li>( g ) Meckaniswb oj induction and repression The relative positions of the NagC protected sequences differ for the three genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1848637" target="_blank">1848637</a>]<br></li>
<li>The N-acetyl glucosamine 6-phosphate that is generated after phosphorylation and subsequent hydrolysis of N , N ′ - diacetylchitobiose acts as an inducing signal for NagC , thereby relieving NagC repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18028317" target="_blank">18028317</a>]<br></li>
<li>The relief of NagC repression by growth on GlcNAc is masked by the catabolite repression generated by GlcNAc ( Fig .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24593230" target="_blank">24593230</a>]<br></li>
<li>The expression of both fusions was increased by growth on galactose and to a lesser extent by growth on GlcNAc , supporting the hypothesis that NagC is involved in repression of galP ( Table 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19007420" target="_blank">19007420</a>]<br></li>
<li>The amino sugars are essential constituents of the peptidoglycan and lipo-polysaccharide of gram-negative bacteria , and the NagC protein is involved in the regulation of their biosynthesis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626331" target="_blank">8626331</a>]<br></li>
<li>The NagC protein is much less easily detected under the same conditions ( Piumbridge .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1766379" target="_blank">1766379</a>]<br></li>
<li>The A residues hypermethylated by dimethyl sulfoxide ( DMS ) in the presence of NagC in boxes E , B and M ' are shown in bold .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7545108" target="_blank">7545108</a>]<br></li>
<li>The presence of NagC bound to BoxGl could impede the passage of polymerase from P2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7545108" target="_blank">7545108</a>]<br></li>
<li>The location of a NagC box at -47 relative to P1 is consistent with it being an activation site for the P1 promoter when NagC is bound .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7545108" target="_blank">7545108</a>]<br></li>
<li>The man operon is controlled by the NagC repressor ( 345 , 510 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8246840" target="_blank">8246840</a>]<br></li>
<li>The NagC repressor inhibits chiP transcription under uninduced conditions through binding to the chiP promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24593230" target="_blank">24593230</a>]<br></li>
<li>This double control , presumably requiring higher levels of GlcNAc6P to derepress NagC than the monodeaceylated Chb6P to activate ChbR , ensures that only when sufficient chitobiose is present are both operons maximally expressed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24593230" target="_blank">24593230</a>]<br></li>
<li>Considering the wide spectrum of substrates for the manXYZ-encoded transporter , it is , in fact , not desirable that manXYZ should be strongly regulated by the NagC repressor and only expressed in the presence of its amino sugar substrates , GlcNAc and GlcN .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9484892" target="_blank">9484892</a>]<br></li>
<li>Both proteins are labelled in maxicells carrying pB34-1 ( lane 2 ) but only NagC is labelled in the presence of pB35-1 ( lane 7 ) which carries just the 3 ' end of the operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2668691" target="_blank">2668691</a>]<br></li>
<li>This is not just due to the presence of G/C at positions +11 / -- 11 in all the NagC selected sequences , since the Mlc selected sequence M134 , which has -- 11C , +11 G , produces a 2-fold derepression of ptsG -- lacZ and M11 , with +11 G , produces a 6-fold derepression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11139621" target="_blank">11139621</a>]<br></li>
<li>This excludes the presence of a third NagC operator controlling nagBE -- lacZ expression ( compare with 23,24 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11139621" target="_blank">11139621</a>]<br></li>
<li>This observation implies that the presence of a G · C base-pair at position + / − 11 increases the afﬁnity of the NagC repressor for the operator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7791215" target="_blank">7791215</a>]<br></li>
<li>This reduction could be attributed to the role of ChbR as a repressor in the presence of NagC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18028317" target="_blank">18028317</a>]<br></li>
<li>In both the nag and cyt cases the activator protein , cAMP/CAP complex , has been recruited to a second role in repression , essential in the CytR case and modulating in the NagC case .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7791215" target="_blank">7791215</a>]<br></li>
<li>DNase I protection experiments were performed with different amounts ot a crude extract of the NagC repressor from an overproducing strain in the presence or absence of 20 nM CAP , 200 \ iM cAMP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7934873" target="_blank">7934873</a>]<br></li>
<li>Incubation with NagC reduces the intensity of the transcript in the presence or absence of cAMP/CAP ( lanes 6 and 7 ) , whereas the presence of ChbR and NagC together eliminates the chbB transcript ( lane 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15066032" target="_blank">15066032</a>]<br></li>
<li>CAP concentration was 50 nM with 200 mM cAMP ; NagC , 100 nM and ChbR , 30 nM .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15066032" target="_blank">15066032</a>]<br></li>
<li>In the presence of the two inducing signals , GlcNAc-6P to displace NagC and the signal generated by growth on chitobiose to activate ChbR , NagC is displaced and RNA polymerase can bind to the promoter in the presence of CAP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15066032" target="_blank">15066032</a>]<br></li>
<li>Model for combined NagC and ChbR regulation of the chbB promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15066032" target="_blank">15066032</a>]<br></li>
<li>RNA polymerase binding in the presence of CAP , ChbR and NagC to the chbB promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15066032" target="_blank">15066032</a>]<br></li>
<li>In the former , binding of NagC to NagC site1 prevents RNA polymerase binding and this is enhanced by the presence of ChbR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15066032" target="_blank">15066032</a>]<br></li>
<li>This shows that whereas both NagC and ChbR appear to act as repressors for the chbB operon in the absence of the inducing sugar chitobiose , ChbR is necessary for induction and so behaves as a dual-function repressor -- activator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15066032" target="_blank">15066032</a>]<br></li>
<li>In the absence of the substrate N , N ′ - diacetylchitobiose , ChbR binds to the direct repeats present within chbOP and represses transcription along with the negative regulator NagC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18028317" target="_blank">18028317</a>]<br></li>
<li>In the absence of chitobiose both NagC and ChbR act as repressors .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15066032" target="_blank">15066032</a>]<br></li>
<li>NagC , along with ChbR represses the operon in the absence of the inducer .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22797760" target="_blank">22797760</a>]<br></li>
<li>These experiments show that in the absence of CAP ChbR behaves as a repressor , preventing RNA polymerase from initiating transcription and that this repression is enhanced by NagC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15066032" target="_blank">15066032</a>]<br></li>
<li>Effect of NagC and ChbR on open complex formation and transcription in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15066032" target="_blank">15066032</a>]<br></li>
<li>GalR-Alexa 488 , at a constant concentration of 20 nM , was mixed with a saturating amount of non-labelled protein , GalS , GalR or NagC , in 20 mM Tris-HCl , pH 7.5 , 300 mM KCl , 2.5 mM MgCl2 , 0.5 mg ml-1 bovine serum albumin and the anisotropy was measured .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19007420" target="_blank">19007420</a>]<br></li>
<li>On the other hand , when the extract overproducing NagA and NagC was tested , all the GICNAC-6-PO4 was converted to G!cN -6 - PO4 by the action of the nagA-encoded deacetylase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1766379" target="_blank">1766379</a>]<br></li>
<li>NagC bound to this fragment but the NagC box ( B ) was filled at lower concentrations than was the NagC box ( E ) and quite clearly there was no change in the pattern of DNask I bands in the intervening region ( Fig. 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1848637" target="_blank">1848637</a>]<br></li>
<li>Using NagC extracts from the Acrp strain , in the presence or absence of CAMP , results in the same pattern of protection and hypersensitive DNase I bands as observed with the JMlOl extract minus CAMP ( Fig. 6 , lanes 5 and 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1848637" target="_blank">1848637</a>]<br></li>
<li>When the labelled fragment crude extracts of a culture overproducing the NagC protein in the presence of CAMP , a strongly retarded complex was detected ( Fig. 2 ( b ) ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1848637" target="_blank">1848637</a>]<br></li>
<li>Extracts of TP2139 pUC ( NagC ) produce no stably retarded complex in the presence of CAMP ( Fig. 3 , lanes 2 to 4 ) , whereas extracts from TP2100 ( m-p + ) do ( Fig. 3 , lanes 8 to 10 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1848637" target="_blank">1848637</a>]<br></li>
<li>However , the NagB NagA NagC NagAC Conlrol NagB NagA NagC NagAC Control Fig. 2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1766379" target="_blank">1766379</a>]<br></li>
<li>NagA extracts ; lanes 4 and 5 and & -12 , NagC extracts ; lanes 6 , 7 and 13 , NagA plus NagC extracts .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1766379" target="_blank">1766379</a>]<br></li>
<li>Moreover , the presence of NagC enhances protection of the galOE3 and OE5 operators by low concentrations of GalR and GalS ( Fig. 3A lanes 13 and 14 compared with lanes 11 and 12 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19007420" target="_blank">19007420</a>]<br></li>
<li>Moreover , no interaction between GalR and NagC was detected by ﬂuorescence anisotropy ( Fig. 5B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19007420" target="_blank">19007420</a>]<br></li>
<li>Reactions included NagC at ﬁnal concentrations of 100 , 50 , 25 , 12.5 nM ( indicated by the horizontal triangle ) in the presence ( + ) or absence ( -- ) of 50 nM IHF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17238917" target="_blank">17238917</a>]<br></li>
<li>When 4 concentrations of NagC , ChbR , GalR or GalS were tested on each DNA ( indicated by a wedge covering 4 lanes ) , the concentrations were 100 , 50 , 25 and 12.5 nM ( calculated for monomeric protein ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24593230" target="_blank">24593230</a>]<br></li>
<li>The Gal11-Gal2 DNA fragment , labelled at either end , was mixed with NagC , GalR or GalS at the following concentrations : A. Lanes 2 -- 6 GalR 50 , 25 , 12.5 , 6.2 and 3.1 nM ; lanes 7 -- 10 , GalS 200 , 100 , 50 , 25 nM ; lane 11 , GalS 25 nM and GalR 25 nM ; lane 12 , GalS 25 nM and GalR 12.5 nM ; lanes 13 and 14 , as lanes 11 and 12 with NagC 25 nM ; Lane 15 NagC 25 nM .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19007420" target="_blank">19007420</a>]<br></li>
<li>B. Lanes 2 -- 6 NagC 50 , 25 , 12.5 , 6.2 and 3.1 nM , lanes 7 -- 11 GalR 50 , 25 , 12.5 , 6.2 and 3.1 nM ; lanes 12 -- 15 GalS 200 , 100 , 50 , 25 nM .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19007420" target="_blank">19007420</a>]<br></li>
<li>DNA 5 ' end labelled at the NcoI site was mixed wit , h CAP and NagC extracts from JMlOl , then treated with DNase I. Lane 1 and 8 , no added protein ; lane 2 , 100 n & r-CAP ; lane 3 , 10 nivr-CAP ; lanes 4 and 6,250 fig NagC extract/ml ( + + ) ; lanes 5 and 7,50 pg NagC extract/ml ( + ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1848637" target="_blank">1848637</a>]<br></li>
<li>Lane 1 , 50 nM-CAP plus @ 2 mi \ l-CAMP ; lane 2 , NagC extract ( JMlOl , 95 , ug/ml ) plus 02 m & r-CAMP ; lane 3 , as lane 2 in the absence of CAMP ; lane 4 , no added protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1848637" target="_blank">1848637</a>]<br></li>
<li>Lanes 1 and 8 , no added protein ; lane 2 , 50 ni % - CAP plus O-2mM-CAMP ; lane 3 , NagC extract from JMlOl , 28 pg/ml ; lane 4 ; as lane 3 plus @ 2mM-CAMP ; lane 5 , NagC extract from TP2139 ( Acrp ) 130pg/ml ; 1ane 6 , as lane 5 plus 0.2 mivr-CAMP ; lane 7 , as lane 6 plus 50 n & r-CAP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1848637" target="_blank">1848637</a>]<br></li>
<li>Assuming an enrichment of NagC or Mlc to ~ 2.5 % of the total protein in the extracts , the amount of NagC or Mlc mixed with the 15 pmol of labelled template was ~ 250 ng .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11139621" target="_blank">11139621</a>]<br></li>
<li>+ corresponds to 10 jig ml '' '' total protein of the NagC extract ; + + .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7934873" target="_blank">7934873</a>]<br></li>
<li>x Expression of the chitobios operon of Escherichia coli e scription is regulated by three tran factors : NagC , ChbR and CAP Jacqueline Plumbridge * and Olivier Pellegrini Institut de Biologie Physico-Chimique ( CNRS UPR9073 ) , 13 , rue Pierre et Marie Curie , 75005 Paris , France .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15066032" target="_blank">15066032</a>]<br></li>
<li>The EMBO Journal vol .14 no. 16 pp.3958-3965 , 1995 Co-ordinated regulation of amino sugar biosynthesis and degradation : the NagC repressor acts as both an activator and a repressor for the transcription of the gimUS operon and requires two separated NagC binding sites Jacqueline Plumbridge Institut de Biologie Physico-chimique ( URA1 139 ) , 13 rue P. et M.Curie , 75005 Paris , France 393 presence of the relevant sugars in the medium ( White , 1968 ; Vogler and Lengeler , 1989 ; Plumbridge , 1990 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7545108" target="_blank">7545108</a>]<br></li>
<li>Pennetier C , Dominguez-Ramirez L , Plumbridge J ( 2008 ) Different regions of Mlc and NagC , homologoustranscriptionalrepressorscontrollingexpressionoftheglucoseand N-acetylglucosamine phosphotransferase systems in Escherichia coli , are required for inducer signal recognition .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18319344" target="_blank">18319344</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_379">
Pennetier , C. , Domínguez-Ramírez , L. , and Plumbridge , J. ( 2008 ) Different regions of Mlc and NagC , homologous transcriptional-repressors controlling expression of the glucose and N-acetylglucosamine phosphotransferase systems in Escherichia coli , are required for inducer signal recognition .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24593230" target="_blank">24593230</a>]<br></label>
<input id="act_similarity_379", type="checkbox">
<div class="ACT_similar">
<li>Pennetier , C. , Dominguez-Ramirez , L. , and Plumbridge , J. ( 2008 ) Different regions of Mlc and NagC , homologous transcriptional-repressors controlling expression of the glucose and N-acetylglucosamine phosphotransferase systems in Escherichia coli , are required for inducer signal recognition .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19007420" target="_blank">19007420</a>]<br></li></div>
<li>It is interesting that NagE ( the N-acetylglucosamine-speciﬁc PTS transporter ) and PtsG , which are closely related EII membrane transporters ( Saier et al. , 1988 ; Postma et al. , 1996 ) , are regulated by homologous repressors , NagC and Mlc .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767573" target="_blank">9767573</a>]<br></li>
<li>506 -- 514 © 2001 Oxford University Press DNA binding sites for the Mlc and NagC proteins : regulation of nagE , encoding the N-acetylglucosamine - specific transporter in Escherichia coli Jacqueline Plumbridge * Received August 15 , 2000 ; Revised and Accepted November 9 , 2000 ABSTRACT The NagC and Mlc proteins are homologous transcriptional-regulators that control the expression of several phosphotransferase system ( PTS ) genes in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11139621" target="_blank">11139621</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_383">
Plumbridge , J. ( 2001 ) DNA binding sites for the Mlc and NagC proteins : regulation of nagE , encoding the N-acetylglucosamine-speciﬁc transporter in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17238917" target="_blank">17238917</a>]<br></label>
<input id="act_similarity_383", type="checkbox">
<div class="ACT_similar">
<li>Plumbridge , J. ( 2001 ) DNA binding sites for the Mlc and NagC proteins : regulation of nagE , encoding the N-acetylglucosamine speciﬁc transporter in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15066032" target="_blank">15066032</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19007420" target="_blank">19007420</a>]<br></li>
<li>DNA binding sites for the Mlc and NagC proteins : regulation of nagE , encoding the N-acetylglucosamine-speciﬁc transporter in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22178967" target="_blank">22178967</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11535784" target="_blank">11535784</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16885453" target="_blank">16885453</a>]<br></li></div>
<li>Mlc encodes a homologue of NagC and is responsible for the repression of other PTS operons involved in glucose uptake ( reviewed in Plumbridge , 2002 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15066032" target="_blank">15066032</a>]<br></li>
<li>NagC represses nagE , encoding the N-acetylglucosamine-specific transporter , while Mlc represses three PTS operons , ptsG , manXYZ and ptsHIcrr , involved in the uptake of glucose .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11139621" target="_blank">11139621</a>]<br></li>
				</div></ul><br>
			<label class="collapse_DOM" for="_2"><font size="5px">NagC protein domains</font></label>
				<input id="_2" type="checkbox">
				<div class="DOM"><br><ul><li>We suggest either that NagC contacts RNA polymerase directly ( or another regulatory protein bound closer to the promoter ) to enhance transcription-activation , or that the nucleoprotein complex that includes NagC and IHF alters the DNA structure nearer the promoter in such a way as to somehow facilitate transcription initiation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17238917" target="_blank">17238917</a>]<br></li>
<li>A hypermethylated A residue is found on the lower strand within NagC box ( E ) residue -31 Fig. 6 t ¬ ¬ ¬ s + ¬ ¬ CAP CAMP i - I -- c ¬ ¬ t ¬ t + ¬ ¬ ¬ t Nogclcrd ) NaqC ( & rpi ¬ ¬ ¬ i t ¬ ¬ ¬ ¬ a striking pattern where two regions are completely protected against DNase I attack ( Fig. 6 , lanes 3 to 7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1848637" target="_blank">1848637</a>]<br></li>
<li>ag Repressor -- Operator Interactions : Protein -- DNA Contacts C More h n T o Turns e N Helix N over T a w of th D A Jacqueline Plumbridge1 * and Annie Kolb2 1Institut de Biologie The NagC repressor binds to two sites in the intergenic nagE-B region overlapping the divergently expressed nagE and nagB promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7791215" target="_blank">7791215</a>]<br></li>
<li>( A ) Alignment of the HTH motifs in the N-terminal regions of NagC and Mlc from E.coli and the XylR repressor from Bacillus subtilis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11139621" target="_blank">11139621</a>]<br></li>
<li>In the repressed state , NagC and ChbR sites are occupied , probably forming a complex structure involving an interaction between the two NagC proteins and formation of a DNA loop .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15066032" target="_blank">15066032</a>]<br></li>
<li>The only candidate amino acids in the HTH of NagC and Mlc that could potentially make a differential interaction with the G/C at positions +11 / -- 11 are at position 10 , at the far end of the recognition helix .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11139621" target="_blank">11139621</a>]<br></li>
<li>This family consists of two classes of proteins : the transcriptional regulatory proteins exempliﬁed by NagC and XylR , and the somewhat smaller proteins encoding sugar kinases which are missing the N-terminal helix-turn-helix DNA-binding domain present in the transcription factors .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9864311" target="_blank">9864311</a>]<br></li>
<li>Comparison of NagC with other prokaryotic DNA-binding proteins In comparison to other prokaryotic operators interacting with helix-turn-helix proteins of known structures , protein -- DNA-recognition contacts at positions 3 to 8 from the centre of symmetry are to be expected ( see Harrison & Aggarwal , 1990 for a review ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7791215" target="_blank">7791215</a>]<br></li>
<li>) nagE-lacZ w.t. w.t. nagC Strain Medium An interaction at + / -- 11 bp from the centre of symmetry of the nagC operator Having chosen to mutate what seemed to be the worst of the two NagC operators in the nagE-B intergenic region , we were expecting to ﬁnd mutations which enhanced binding and repression within the region shown to be important for protein -- DNA contacts in these other systems i.e. in the range of positions 3 to 8 presumed to interact with the helix-turn-helix in the major groove .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7791215" target="_blank">7791215</a>]<br></li>
<li>Even though the helix -- turn -- helix motifs of Mlc and NagC are almost identical and both proteins bind to similar sequences , there are distinctions in their operator selection .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9484893" target="_blank">9484893</a>]<br></li>
<li>This presumably reflects the fact that the amino acids in positions 1 , 2 and 6 in the recognition helix of the HTH , known to be important for DNA-binding in other HTH proteins ( 27 -- 29 ) , are identical in NagC and Mlc .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11139621" target="_blank">11139621</a>]<br></li>
<li>Surprisingly , the Mlc protein has an almost identical amino acid sequence in the recognition helix of the helix -- turn -- helix motif to that of NagC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9484892" target="_blank">9484892</a>]<br></li>
<li>The homology between NagC and Mlc is particularly intriguing , because the amino acid sequence believed to form the helix -- turn -- helix motif for DNA-binding is almost identical in the two proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9484893" target="_blank">9484893</a>]<br></li>
<li>The NagC and XylR repressors show signiﬁcant differences in their protein sequences in the region , which are predicted to form the helix -- turn -- helix DNA-binding motif .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9484892" target="_blank">9484892</a>]<br></li>
<li>Analysis of the protein sequence deduced from the DNA sequence of nagC identified a possible DNA-binding region of the type a helix-turn-a helix described by Pabo and Sauer ( 1985 ) , located in the carboxy terminus of NagC ( amino acids 315-334 , nucleotides 2154-2213 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2668691" target="_blank">2668691</a>]<br></li>
<li>No G residue contacts were detected by D&IS reactivity in the NagC binding sites , which is perhaps not surprising considering the -4 - t T-rich nature of the NagC boxes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1848637" target="_blank">1848637</a>]<br></li>
<li>This is consistent with the idea that formation of the loop permits some synergy in the binding of CAP and NagC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7934873" target="_blank">7934873</a>]<br></li>
<li>Two NagC binding sites have been located in the glmUS upstream region , and both are necessary for activation ( 32 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626331" target="_blank">8626331</a>]<br></li>
<li>The region protected by NagC and the three hypersensitive bands indicative of CAP binding are indicated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15743943" target="_blank">15743943</a>]<br></li>
<li>The C/G base pair provides an additional recognition point and binding interaction for NagC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11139621" target="_blank">11139621</a>]<br></li>
<li>( c ) What constitutes a NagC binding site ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1848637" target="_blank">1848637</a>]<br></li>
<li>Thus , the DMS methylation protection experiments also demonstrated a site-specific interaction between NagC and two regions of the nagE-nagB region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1848637" target="_blank">1848637</a>]<br></li>
<li>Since no protection against A residue methylation was observed with NagC , it seems unlikely that NagC is making specific contacts in the minor groove .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1848637" target="_blank">1848637</a>]<br></li>
<li>As a G or C at positions +11 / -- 11 is equally beneficial to NagC binding , it is likely that NagC recognises the conformation produced by a GC base pair ( e.g. a wider minor groove ) and/or makes a contact with the phosphate backbone rather than making a specific contact with the nucleotide base at positions +11 / -- 11 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11139621" target="_blank">11139621</a>]<br></li>
<li>Recently , a detailed mutagenesis study of the nagE operator has identified the two AT base pairs at positions + 5 and +6 as the most important recognition points for NagC binding and has shown that the base pairs at positions 3 , 4 and 11 also contribute to the binding strength ( Plumbridge and Kolb , 1995 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7545108" target="_blank">7545108</a>]<br></li>
<li>Note that the NagC consensus has a central zero position .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19007420" target="_blank">19007420</a>]<br></li>
<li>Two T residues , at positions 5 and 6 relative to the hypermethylated As at the centre of the protected regions , are conserved in all the four known NagC boxes ( Plumbridge and Koib , 1991 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7934873" target="_blank">7934873</a>]<br></li>
<li>In the case of NagC operator , the minor groove centred at the dyad axis might be more accessible in the presence of the repressor since the central A residue of the nagE and nagB operators is hypermethylated by DMS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7791215" target="_blank">7791215</a>]<br></li>
<li>The eight half sites from the four known NagC operators are aligned about the central A residue hypersensitive to DMS attack which is underlined ( Plumbridge & Kolb , 1991 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7791215" target="_blank">7791215</a>]<br></li>
<li>The ( few ) G residues in the NagC operators were quite insensitive to DMS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7791215" target="_blank">7791215</a>]<br></li>
<li>The A residues hypermethylated by dimethyl sulfoxide ( DMS ) in the presence of NagC in boxes E , B and M ' are shown in bold .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7545108" target="_blank">7545108</a>]<br></li>
				</div></ul><br>
			<label class="collapse_EVO" for="_3"><font size="5px">NagC evolution</font></label>
				<input id="_3" type="checkbox">
				<div class="EVO"><br><ul><li>* Tel : +33 1 58 41 51 52 ; Fax : +33 1 58 41 50 20 ; Email : plumbridge@ibpc.f Mlc is a homologue of NagC ( 40 % identity ) ( 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11139621" target="_blank">11139621</a>]<br></li>
<li>This family consists of two classes of proteins : the transcriptional regulatory proteins exempliﬁed by NagC and XylR , and the somewhat smaller proteins encoding sugar kinases which are missing the N-terminal helix-turn-helix DNA binding domain present in the transcription factors .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9864311" target="_blank">9864311</a>]<br></li>
<li>The first class is the transcriptional regulators , notably the family of various xylose repressors ( XylR ) in Gram-positive bacteria , and NagC and Mlc in E.coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11139621" target="_blank">11139621</a>]<br></li>
<li>NagC is a member of the so-called ROK family of proteins , which includes a family of xylose repressors from Gram-positive bacteria , some glucose and fructose kinases ( Titgemeyer et al. , 1994 ) and the recently identiﬁed Mlc protein ( Hosono et al. , 1995 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9484892" target="_blank">9484892</a>]<br></li>
<li>Mlc is a member of the so-called ROK family of proteins ( Titgemeyer et al. , 1994 ) , which includes NagC , the repressor for the nagE -- BACD operons ( Plumbridge , 1991 ; Plumbridge and Kolb , 1995 ) , and XylR , the repressor of the xylose operons , from a number of Gram-positive bacteria plus certain glucose and fructose kinases .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767573" target="_blank">9767573</a>]<br></li>
<li>The NagC repressor in fact , shows strong homology to the family of xylose repressors from different Gram-positive species ( Kreuzer et al. , 1989 ; Lokman et al. , 1991 ) , and certain other proteins , notably some sugar kinases ( Angell et al. , 1992 ; Titgemeyer et al. , 1994 ) but not signiﬁcantly to the other major families of prokaryotic repressors , the LacI-GalR family ( Weickert & Adhya , 1992 ) or the DeoR-GlpR family ( van Rooijen & de Vos , 1990 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7791215" target="_blank">7791215</a>]<br></li>
<li>The NagC protein shows strong homology to a family of xylose repressors from Gram-positive species ( Titgemeyer et al. , 1994 ) and it would be interesting to know whether these repressors also possess an activator function .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7545108" target="_blank">7545108</a>]<br></li>
<li>Both NagC and Mlc are members of the so-called ROK ( repressors , ORFs , kinases ) family of proteins which contains at least two distinct classes of proteins ( 15 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11139621" target="_blank">11139621</a>]<br></li>
<li>The ﬁrst of these other genes , yhcI , encodes a protein homologous to NagC , Mlc , and other proteins of the so-called ROK ( repressor , ORF , and kinase ) family ( 42 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9864311" target="_blank">9864311</a>]<br></li>
<li>Recently , the Mlc protein has been described , which is homologous to NagC ( 40 % identity and 80 % similarity ; Hosono et al. , 1995 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9484892" target="_blank">9484892</a>]<br></li>
<li>The deduced amino acid sequence of mlc shows 40 % identity ( 80 % similarity ) with NagC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9484893" target="_blank">9484893</a>]<br></li>
<li>The peculiarity of the NagC and Mlc proteins is that the sequence of the amino acids in their recognition helices is almost identical and much more similar to each other than to the equivalent sequences in the XylR family ( Fig. 1A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11139621" target="_blank">11139621</a>]<br></li>
<li>The sequence of Mlc is 40 % identical with NagC , the dual functional repressor and activator of the genes encoding enzymes for amino sugar degradation ( Vogler and Lenge-ler , 1989 ; Plumbridge , 1991 ) and biosynthesis ( Plumbridge , 1995 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9484893" target="_blank">9484893</a>]<br></li>
				</div></ul><br>
			<label class="collapse_RP" for="_4"><font size="5px">Processes regulated by NagC</font></label>
				<input id="_4" type="checkbox">
				<div class="RP"><br><ul><li>The N-acetyl glucosamine 6-phosphate that is generated after phosphorylation and subsequent hydrolysis of N , N ′ - diacetylchitobiose acts as an inducing signal for NagC , thereby relieving NagC repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18028317" target="_blank">18028317</a>]<br></li>
<li>While glmUS transcription is controlled by NagC protein ,2 which also represses operons involved in amino sugar uptake and degradation , σ54 and GlmY may independently adjust GlmS synthesis in response to the general nitrogen status of the cell .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17854828" target="_blank">17854828</a>]<br></li>
<li>The EMBO Journal vol .14 no. 16 pp.3958-3965 , 1995 Co-ordinated regulation of amino-sugar-biosynthesis and degradation : the NagC repressor acts as both an activator and a repressor for the transcription of the gimUS operon and requires two separated NagC binding sites Jacqueline Plumbridge Institut de Biologie Physico-chimique ( URA1 139 ) , 13 rue P. et M.Curie , 75005 Paris , France 393 presence of the relevant sugars in the medium ( White , 1968 ; Vogler and Lengeler , 1989 ; Plumbridge , 1990 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7545108" target="_blank">7545108</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_4">
Plumbridge , J. ( 1995 ) Co-ordinated regulation of amino-sugar-biosynthesis and degradation : the NagC repressor acts as both an activator and a repressor for the transcription of the glmUS operon and requires two separated NagC binding sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9484893" target="_blank">9484893</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19843219" target="_blank">19843219</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17238917" target="_blank">17238917</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17824929" target="_blank">17824929</a>]<br></label>
<input id="rp_similarity_4", type="checkbox">
<div class="RP_similar">
<li>Coordinated regulation of amino-sugar-biosynthesis and degradation : the NagC repressor acts as both an activator and a repressor for the transcription of the glmUS operon and requires two separated NagC binding sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15060041" target="_blank">15060041</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9325090" target="_blank">9325090</a>]<br></li>
<li>Co-ordinated regulation of amino-sugar-biosynthesis and degradation : the NagC repressor acts as both an activator and a repressor for the transcription of the glmUS operon and requires two separated NagC binding sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17854828" target="_blank">17854828</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9529892" target="_blank">9529892</a>]<br></li></div>
<li>NagC coordinates the expression of genes involved in the degradation and biosynthesis of GlcNAc and GlcN ( Plumbridge , 1995 ) and the inducing signal for NagC displacement is GlcNAc-6P ( Plum-bridge , 1991 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15066032" target="_blank">15066032</a>]<br></li>
<li>Since the sequence of Mlc is ~ 40 % identical to NagC , a repressor and activator of the genes encoding enzymes for amino-sugar-degradation in response to N-acetylglucosamine-6-phosphate ( Plumbridge , 1998a ) , the possibility that glucose or one of its metabolic intermediates may act as the inducer of Mlc-regulated genes has been considered .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11157755" target="_blank">11157755</a>]<br></li>
<li>The sequence of Mlc is 40 % identical with NagC , the dual functional repressor and activator of the genes encoding enzymes for amino-sugar-degradation ( Vogler and Lenge-ler , 1989 ; Plumbridge , 1991 ) and biosynthesis ( Plumbridge , 1995 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9484893" target="_blank">9484893</a>]<br></li>
<li>The amino sugars are essential constituents of the peptidoglycan and lipo-polysaccharide of gram-negative bacteria , and the NagC protein is involved in the regulation of their biosynthesis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626331" target="_blank">8626331</a>]<br></li>
<li>In addition to controlling genes involved in amino-sugar-metabolism , NagC is also involved in controlling expression of the ﬁmB gene , required for phase variation of type I pilae ( Sohanpal et al. , 2007 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19007420" target="_blank">19007420</a>]<br></li>
<li>Expression of the chb operon is repressed by NagC , which regulates genes involved in amino-sugar-metabolism .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15066032" target="_blank">15066032</a>]<br></li>
<li>The chb operon is regulated by three transcription factors : NagC , a key regulator of the nag genes involved in amino-sugar-metabolism ; ChbR , a dual function operon-speciﬁc regulator ; and CRP-cyclic AMP ( CRP-cAMP ) ( 45 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22797760" target="_blank">22797760</a>]<br></li>
<li>Mlc is a member of the so-called ROK family of proteins ( Titgemeyer et al. , 1994 ) , which includes NagC , the repressor for the nagE -- BACD operons ( Plumbridge , 1991 ; Plumbridge and Kolb , 1995 ) , and XylR , the repressor of the xylose operons , from a number of Gram-positive bacteria plus certain glucose and fructose kinases .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767573" target="_blank">9767573</a>]<br></li>
<li>Mlc encodes a homologue of NagC and is responsible for the repression of other PTS operons involved in glucose-uptake ( reviewed in Plumbridge , 2002 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15066032" target="_blank">15066032</a>]<br></li>
<li>NagC represses nagE , encoding the N-acetylglucosamine-specific transporter , while Mlc represses three PTS operons , ptsG , manXYZ and ptsHIcrr , involved in the uptake of glucose .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11139621" target="_blank">11139621</a>]<br></li>
<li>) Plasmids expressing individual nag genes Plasmids pUC ( NagA ) and pUC ( NagC ) ( Plumbridge .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1766379" target="_blank">1766379</a>]<br></li>
<li>The NagC repressor together with the cyclic AMP-cyclic AMP receptor protein ( CRP ) complex controls expression of the nag regulon ( Plumbridge , 1991 ; Plumbridge , and Kolb , 1995 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11361063" target="_blank">11361063</a>]<br></li>
<li>N-acetylglucosamine 6-phosphate ( GlcNAc6P ) , the product of the nagE-encoded PTS transporter , and also a downstream product of chitobiose metabolism ( Fig. 1B ) functions as the allosteric effector of NagC inactivating the protein and leading to derepression of the nag and chb operon ( Plumbridge , 1991 ; Plumbridge and Pellegrini , 2004 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24593230" target="_blank">24593230</a>]<br></li>
<li>Another , also involving the NagC repressor , is the chbBCAFG operon , encoding genes for the uptake and utilization of chitobiose ( Plumbridge and Pellegrini , 2004 ) but as chitobiose is the dimer of GlcNAc , inclusion of chb in the nag regulon is not surprising .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19007420" target="_blank">19007420</a>]<br></li>
<li>Many Cel + strains carry mutations in nagC The NagC repressor that regulates the nag operon involved in N-acetylglucosamine-metabolism was also shown to regulate the chb operon ( Plumbridge and Pellegrini , 2004 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18028317" target="_blank">18028317</a>]<br></li>
<li>NagC is the repressor of the divergent nagE -- nagBACD operon , which encodes proteins involved in N-acetylglucosamine ( GlcNAc ) transport and metabolism .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24593230" target="_blank">24593230</a>]<br></li>
<li>expression glmU The NagC repressor controls the expression of the divergently transcribed nagE-nagBACD operons involved in the uptake and degradation of the amino sugars , N-acetyl-D-glucosamine ( GlcNAc ) and glucos-amine ( GlcN ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7545108" target="_blank">7545108</a>]<br></li>
<li>In addition , NagC acts as a repressor of the nag and chb operons that encode proteins involved in uptake and degradation of exogenous amino sugars like GlcNAc which are sequentially converted to N-acetyl-glucosamine-6-phosphate ( GlcNAc-6-P ) and GlcN-6-P ( Plumbridge , 1991 ; Plumbridge and Pellegrini , 2004 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17824929" target="_blank">17824929</a>]<br></li>
<li>NagC acts as a repressor for the two divergent operons , nagE and nagBACD ( uptake and utilization of glucosamine and N-acetyl glucosamine ; Vogler and Lengeler , 1989 ; Plumbridge , 1991 ) , and is also functioning as an activator for the biosynthetic glmUS operon ( synthesis of glucos-amine-6-P and UDP-GlcNAc ; Plumbridge , 1995 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9484893" target="_blank">9484893</a>]<br></li>
<li>Mlc shows homology to NagC , a gene regulator functioning as a repressor for the divergent nag operons ( encoding proteins for the uptake and degradation of N-acetyl-D-glucosamine ) and as an activator as well as a repressor for the glmUS operon encoding enzymes for the biosynthesis of N-acetyl-D-glucosamine ( 188 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9529892" target="_blank">9529892</a>]<br></li>
<li>The proximal site , overlapping the +1 , is predicted to be particularly weak as it has three CG basepairs in the positions 7 -- 10 from the centre of symmetry which are usually expected to be A or T. NagC controls the expression of two other operons , the nagE-BACD operon for the uptake and degradation of GlcNAc and the glmUS operon , which encodes enzymes of the glucosamine biosynthetic pathway to synthesize UDP-GlcNAc for cell wall and outer membranes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15066032" target="_blank">15066032</a>]<br></li>
<li>In the absence of an environmental supply of amino sugars , NagC represses the divergent nagE-BACD operons , which are necessary for the utilisation of GlcNAc as a carbon source ( 1 ) , and activates the expression of one promoter of the glmUS operon that is part of the biosynthetic pathway for the formation of GlcN-6-phosphate ( GlcN-6-P ) and UDP-GlcNAc ( 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11139621" target="_blank">11139621</a>]<br></li>
<li>Probe 622 527 Nt Iu P2 s 404 - o - P1 309 Discussion NagC is an activator for the gimU P1 promoter and a repressor for the P2 promoter The expression of the glmU gene , like that of the downstream gene , glmS , which was studied previously , is inversely correlated with that of the nagE-BA genes necessary for the utilization of GlcNAc as carbon source ( Plumbridge et al. , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7545108" target="_blank">7545108</a>]<br></li>
<li>NagC also functions to activate expression of glmS , encoding a glucosamine-6-phosphate synthase involved in the synthesis of N-acetylglucosamine , thereby coordinately regulating the synthesis and degradation of N-acetylglucosamine ( Plumbridge et al. , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14731281" target="_blank">14731281</a>]<br></li>
<li>Institut de Biologie Physico-Chimique ( UPR9073 ) , 13 rue Pierre et Marie Curie , 75005 Paris , France INTRODUCTION NagC is a transcriptional regulator whose function is to co-ordinate the biosynthesis of the amino sugars , glucosamine ( GlcN ) and N-acetylglucosamine ( GlcNAc ) with their catabolism .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11139621" target="_blank">11139621</a>]<br></li>
<li>JOURNAL OF BACTERIOLOGY , May 1996 , p. 2629 -- / 96 / $ 04.0010 Copyright q 1996 , American Society for Microbiology o Achieve Constitutive Expression of a Gene an Inducible Operon : the Example of the nagC Gene of Escherichia coli How T within JACQUELINE PLUMBRIDGE Institut de Biologie Physico-chimique ( URA1139 ) , 75005 Paris , France Received 20 November 1995/Accepted 13 February 1996 The nagC gene , encoding the NagC repressor/activator of the nag regulon , is part of the nagBACD operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626331" target="_blank">8626331</a>]<br></li>
<li>However , a mutation in the mlc gene , encoding a homologue of nagC , results in a threefold derepression of manX expression , suggesting that this protein is a more important regulator of manX expression than NagC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9484892" target="_blank">9484892</a>]<br></li>
<li>The affected gene was sequenced and identiﬁed as mlc , a known gene encoding a protein of unknown function with homology to the Escherichia coli NagC protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9484893" target="_blank">9484893</a>]<br></li>
<li>The nagBACD operon encodes ( a ) glucosamine-6-P deaminase ( NagB ) , ( b ) N-acetylglucosamine-6-P deacetylase ( NagA ) , ( c ) the nag regulon transcriptional regulator ( NagC ) and ( d ) a gene of unknown function which is , however , homologous to functionally characterized phosphatases ( NagD ) ( Plumbridge , 1989 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11361063" target="_blank">11361063</a>]<br></li>
<li>The transcriptional regulator , NagC , represses expression of the nag regulon , encoding proteins involved in the utilization of N-acetylglucosamine ( Vogler and Lengeler , 1989 ) , also known to be a component of lipopolysaccharides in some E. coli strains .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14731281" target="_blank">14731281</a>]<br></li>
<li>The Escherichia coli K12 MG1655 genome was screened for potential binding sites for the transcriptional regulator NagC involved in N-acetylglucosamine metabo-lism .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21070640" target="_blank">21070640</a>]<br></li>
<li>506 -- 514 © 2001 Oxford University Press DNA binding sites for the Mlc and NagC proteins : regulation of nagE , encoding the N-acetylglucosamine - specific transporter in Escherichia coli Jacqueline Plumbridge * Received August 15 , 2000 ; Revised and Accepted November 9 , 2000 ABSTRACT The NagC and Mlc proteins are homologous transcriptional regulators that control the expression of several phosphotransferase system ( PTS ) genes in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11139621" target="_blank">11139621</a>]<br></li>
<li>The chiP gene is under the transcriptional-control of the NagC repressor As the next step in our comparative study , we examined the effects of nagC and chbR mutations on regulation of chiP -- lacZ translational fusions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24593230" target="_blank">24593230</a>]<br></li>
<li>Discussion We have shown that the chiP gene , encoding a specialized porin for chitobiose and chitotriose , is under the control of the N-acetylglucosamine ( GlcNAc ) operon repressor , NagC , in both E. coli and Salmonella .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24593230" target="_blank">24593230</a>]<br></li>
<li>It thus appears that NagC plays a key role in setting chiP expression levels under uninduced conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24593230" target="_blank">24593230</a>]<br></li>
<li>wild type nagC chbR pTZ ( NagC ) pTZ ( Mlc ) pTZ ( ChbR ) pTZ control .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15066032" target="_blank">15066032</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15066032" target="_blank">15066032</a>]<br></li>
<li>Possibly the unregulated expression of both sets of genes controlled by NagC and Mlc has a detrimental effect on cell-physiology .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11139621" target="_blank">11139621</a>]<br></li>
<li>Thus , NagC is a positive regulator of fimB expression and could participate in the control of the recombinase genes expression in response to sialic acid .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15534208" target="_blank">15534208</a>]<br></li>
<li>Table 2 also shows that neither a mutation in nagC nor the overproduction of plasmid-encoded NagC , the protein most homologous to Mlc , affects malT expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9484893" target="_blank">9484893</a>]<br></li>
<li>This regulation occurs via the NanR and NagC regulators , which also control ﬁmB expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15743943" target="_blank">15743943</a>]<br></li>
<li>Operons which have been experimentally verified as being regulated by CRP Gene function Reference e ROMA Operon nam Other CRP sites regulators b Score KE/wt Position Sites a WT HL HL/wt KE 1.96 0.85 # bglG 1.73 -- 61.5 AACTGCGAGCATGGTCATATTT 11.2 Fis Positive regulation of bgl ( 49 ) operon Uptake of C4-dicarboxylic ( 50 ) acids D-ribose high-affinity ( 51 ) transport system Positive regulator for rhabad ( 52 ) operon Succinate dehydrogenase , ( 53 ) cytochrome b556 Tryptophanase leader peptide ( 54 ) PTS-system enzyme II , ( 55 ) trehalose specific Probable formate-transporter ( 43 ) Mgl repressor , galactose ( 56 ) operon inducer Sn-glycerol-3-phosphate ( 57 ) permease PTS-system , maltose and ( 58 ) glucose-specific II ABC Regulator of melibiose operon ( 59 ) Galactose-binding transport ( 56 ) protein PTS-system , glucose-specific ( 35 ) IIBC component PTS-system protein hpr ( 60 ) Orf , hypothetical protein ( 61 ) Aspartate ammonia-lyase ( 62 ) ( aspartase ) 2.44 1.09 # dctA 1.85 -- 81.5 TTGTGCGAGCCAGCTCAAACTT 14.1 DcuR , ArcA 3.80 0.89 # rbsDACBK R 4.10 -- 61.5 CGTTTCGAGGTTGATCACATTT 9.4 RbsR 1.46 0.86 # rhaSR 1.66 -- 113.5 TGATGTGATGCTCACCGCATTT 12.4 RhaR 2.71 1.11 # sdhCDAB 2.38 -- 83.5 TATCGTGACCTGGATCACTGTT 16.4 ArcA , FNR 1.98 0.95 # 3.09 1.35 # tnaLAB treBC 2.08 -- 4.73 -- 61.5 60.5 GATTGTGATTCGATTCACATTT 19.6 AATTGTGATCTTCGCTGCGTTT 8.8 1.39 # 1.09 # 3.90 1.59 # 2.49 1.22 # galS 41.5 41.5 AGATATGATCTATATCAATTTC 10.1 FNR TGCTGTGACTCGATTCACGAAG 10.4 GalR , GalS 2.21 # 4.51 0.89 # glpTQ ATGTGTGCGGCAATTCACATTT 17.2 GlpR 41.5 2.94 # 6.24 1.71 # malXY TTATGTGACAGATAAAACGTTT 11.2 49.5 1.17 # 1.78 # 2.29 1.20 # 6.78 1.49 # melR AACCGTGCTCCCACTCGCAGTC 5.5 MelR ATCTGTGAGTGATTTCACAGTA 16.6 41.5 41.5 mglBAC 1.13 # 2.11 1.14 # ptsG AAACGTGATAGCCGTCAAACAA 14.3 Mlc 40.5 1.56 # 1.10 # 2.65 # 2.95 1.01 # 1.88 0.97 # 6.29 2.50 # yhfA aspA TTTTATGATTTGGTTCAATTCT 9.5 TAATGTGACGTCCTTTGCATAC 11.7 42.5 44.5 90.5 AGCGGTGATCTATTTCACAAAT 15.5 FNR TAAAGTGATCCAGATTACGGTA 14.2 40.5 94.5 deoCABD TTATTTGAACCAGATCGCATTA 17.1 CytR , DeoR 2.37 -- 3.22 2.37 -- 2-Deoxyribose-5-phosphate ( 21 ) aldolase AATTGTGATGTGTATCGAAGTG 11.9 41.5 90.5 4.38 1.72 # AAATGTGAATTGCCGCACACAT 17.2 FNR , ArcA , ( 57 ) glpACB 3.31 -- Sn-glycerol-3-phosphate dehydrogenase FlhD , GlpR AATGACGCATGAAATCACGTTT 3.3 40.5 92.5 2.17 # 4.26 2.59 # manXYZ PTS enzyme IIAB , mannase-specific GAATGTGACAAGGATATTTTAC 2.4 Mlc , NagC ( 63,36 ) ATTACGGATCTTCATCACATAA 7.6 AAATGTATGACAGATCACTATT 12.6 CytR 40.5 89.5 1.39 # 3.25 1.17 # nupC Permease of transport system ( 64 ) for 3 nucleosides TAGTGTGTGTCAGATCTCGTTT 12.6 40.5 92.5 4.45 1.01 # nupG Transport of nucleosides , permease protein AAATGTTATCCACATCACAATT 20.4 CytR , DeoR 3.25 -- ( 65 ) TTATTTGCCACAGGTAACAAAA 10.6 ATTTCATCTCTATGTCACATTT 7.4 CytR 42.5 93.5 41.5 40.5 1.15 # rpoH Sigma ( 32 ) factor ( 22 ) 2.18 1.75 -- ACTTGTGGATAAAATCACGGTC 2.2 AACTGTGAAACGAAACATATTT 12.9 CytR 2.40 # 3.96 1.19 # tsx Receptor of phage T6 and ( 66 ) colicin K 74.5 AAACGTGAACGCAATCGATTAC 5.8 a The centre of an experimentally verified CRP-binding site relative to corresponding transcription start site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15520470" target="_blank">15520470</a>]<br></li>
<li>23.5 20.0 3.8 16.3 9.1 12.7 Maltose transport MalT , Crp malK lamB malM ibpB ibpA htpG Maltose transport MalT , Crp 18.6 5.3 3.3 39.0 24.4 11.7 32.7 13.3 9.1 24.2 20.2 8.2 2.6 2.1 1.7 4.6 4.1 2.0 48.6 19.2 20.0 52.6 35.9 16.6 s32 Heat shock Heat shock s32 Heat shock chaperone s32 Protease ; chaperone Regulator Regulator ; chaperone Protease Periplasmic clpB 11.5 12.9 17.3 21.0 20.4 14.3 3.3 3.2 1.3 1.8 2.5 2.9 8.0 2.7 2.9 24.5 6.8 5.7 1.7 1.3 yceP/bssS cpxP ( b3913 ) degP ( htrA ) spy CpxR CpxR , H-NS CpxR , BaeSR CpxR CpxR Unknown Unknown ydeH yebE 49.2 44.8 63.1 21.0 30.7 32.2 56.9 33.1 9.5 1.1 1.6 6.1 7.3 7.3 2.6 1.5 1.6 2.0 1.4 1.2 1.3 2.9 Unknown ; signal peptide Putative transporter Regulator ; phage-shock protein Regulator ; phage-shock protein Regulator ; phage-shock protein Inner membrane Sulphurtransferase yncJ ( b1436 ) yhjX pspA pspB pspC pspD pspE 1.6 44.1 1.1 1.6 1.7 1.2 1.2 3.1 165.6 1.8 1.5 2.1 1.2 1.5 PspF , IHF 1.3 1.9 5.0 2.4 4.7 2.7 2.6 1.4 PspF , IHF PhoB RbsR , Crp 37.9 22.0 2.6 1.2 1.8 2.3 0.8 1.1 Inner membrane Unknown Ribose transport Ribose transport Ribose transport Ribose transport Ribose transport Ribose transport pspG ( yjbO ) ytfK rbsD rbsA rbsC rbsB rbsK rbsR 0.7 3.8 1.8 1.0 1.6 1.4 1.2 0.8 0.9 3.0 8.2 5.4 21.3 6.6 9.0 2.8 manX Mannose permease 4.2 2.1 1.1 5.0 manY Mannose permease 3.4 2.6 1.1 5.2 manZ Mannose permease 6.5 5.7 1.0 16.7 NagC , DgsA , Crp a. Genes and operons which are induced 10-fold in three independent experiments are listed with the average fold induction for the three experiments .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18710431" target="_blank">18710431</a>]<br></li>
				</div></ul><br>
			<label class="collapse_SIT" for="_5"><font size="5px">NagC binding site</font></label>
				<input id="_5" type="checkbox">
				<div class="SIT"><br><ul><li>The mutation in BoxGl ( lanes 5 and 6 ) eliminated the NagC footprint at w.t. BoxGl BoxG2-I 1 ~ NAGC OPERATOR SITES 404 1110 9 8 7 6 5 4 3 2 1 0 1 2 3 4 5 6 7 8 91011 TTTAA TTTGCG ATACG AATTAAA BOXE CTTAT TTTATC ATTCA AAAAATC BOXB GATAT TTTACC TTTCG AAATTTC BOXM T T A T T T T A G A T A T C G A A A A A A A T BOX M ' GTTTA TTCATT GATCG AAATAAG BOX G2 TCCAT TTCACG ATGAA AAAAATG BOX G1 309 BoxG2 242 238 g t t a t T T c g c g A A a t a a c CONSENSUS c a a t a t a t t g 217 9n1 190-180 Fig. 6 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7545108" target="_blank">7545108</a>]<br></li>
<li>l b522 cI857 Pam80 nin5 ( mini-Tn10 Kan / Ptac-ATS-transposase ) Plasmids pACYC184 Cmr , Tcr Chang and Cohen ( 1978 ) Bolivar et al. ( 1977 ) Schauder et al. ( 1987 ) Reidl and Boos ( 1991 ) This study Reyes and Shuman ( 1988 ) Peist et al. ( 1997 ) M. Ehrmann Amann et al. ( 1988 ) Plumbridge ( 1989 ) Apr , Tcr pBR322 pJLA502 Apr , cIts857 pTrc99B Ptrc-malY þ pJLA502 l PRPL-mlcþ pACYC184 Ptrc-malK þ pJR115 pKDmlc pMR11 pBR322 aesþ pRP11 pRR1 pMR11 Ptrc-malK þ lacI q Apr lacI q Apr nagCþ pTrc99B pUC ( NagC ) Experimental procedures Construction of the mlc -- lacZ protein fusion A 1220 bp fragment , NruI -- EcoRV , comprising part of the mlc structural gene and 630 bp of upstream sequence , was excised from pTZ18R ( Mlc ) ( Plumbridge , 1997 ) as an EcoRI to EcoRV fragment and inserted into pRS414 digested with EcoRI and SmaI to produce a mlc8 -- 8lacZ protein fusion .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9484893" target="_blank">9484893</a>]<br></li>
<li>) Under these conditions NagC was bound co-opera-tively to the DNA fragments carrying the wiid-type sequence or the +10 mutation , resulting in simuitaneous protection of the two regions previously identified as BoxB and BoxE and causing the appearance of seven and eight hypersensitive DNase i bands in the intervening DNA ( Fig. 2 , lanes 7 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7934873" target="_blank">7934873</a>]<br></li>
<li>Dam methylation protection at regions 1 and 2 is apparently mutually exclusive , suggesting that NanR and NagC each form alternative , stable nucleoprotein complexes with their cognatebinding sites ( 21 , 25 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15534208" target="_blank">15534208</a>]<br></li>
<li>We note that GATCNagC NC1 lies 2 bp outside of the consensus region for NagC binding at ONC1 ( 5 ′ - CATAAGGCTCATCATGAAATATGTCGATC ; consensus region underlined ) , and hence the lack of an effect of Dam methylation of GATCNagC on binding is not particularly surprising .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17238917" target="_blank">17238917</a>]<br></li>
<li>To evaluate the importance of this region for the activation of ﬁmB expression further , the 274 bp portion of the D3 region adjacent to ONC2 was replaced with heterologous sequences of the same length derived from the bla gene ( Fig. 1 ; the D7 region ) , and the effect of this mutation on the ability of both NanR and NagC to activate ﬁmB expression was determined ( Fig. 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17238917" target="_blank">17238917</a>]<br></li>
<li>We suppose that the 274 bp wild-type region adjacent to ONC2 analysed could contain one or more sequences that activate ﬁmB expression even in the absence of NanR or NagC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17238917" target="_blank">17238917</a>]<br></li>
<li>By replacing the 274 bp of wild-type DNA from -467 to -194 with heterologous sequences it was shown that neither NanR nor NagC requires this segment of DNA in order to activate ﬁmB expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17238917" target="_blank">17238917</a>]<br></li>
<li>NagC sites characteristically function in pairs ( Plum-bridge , 2001 ) , but although NagC binds to a second element ( ONC2 ; originally termed NagC2 ) situated 212 bp closer to ﬁmB than ONC1 , cooperative binding of NagC to ONC1 and ONC2 was not apparent in vitro , and a deletion that includes ONC2 ( D3 ) did not diminish ﬁmB expression ( El-Labany et al. , 2003 ; Sohanpal et al. , 2004 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17238917" target="_blank">17238917</a>]<br></li>
<li>The sequence includes the NanR and NagC operator sites ( ONR and ONC1 respectively ) ( shown in bold ) , the -10 and -35 region of the nanC promoter ( underlined ) , and the methylation-protected GATC sites ( GATCNanR and GATCNagC ; shown by a larger font size ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17238917" target="_blank">17238917</a>]<br></li>
<li>ONR and ONC1 lie close to Dam methylation sites ( GATCNanR and GATCNagC respectively ; Fig. 1 ) , which are frequently unmodiﬁed ( El-Labany et al. , 2003 ) , and NanR and NagC produce methylation protection of the GATC sequences adjacent to their binding sites in vivo ( Sohanpal et al. , 2004 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17238917" target="_blank">17238917</a>]<br></li>
<li>Neither sequences up to 820 bp upstream of ONR , nor those 270 bp downstream of ONC2 , are required for activation by NanR and NagC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17238917" target="_blank">17238917</a>]<br></li>
<li>NanR ( Neu5Ac-responsive ) and NagC ( GlcNAc-6P-responsive ) activate ﬁmB expression by binding to operators ( ONR and ONC1 respectively ) located more than 600 bp upstream of the ﬁmB promoter within the large ( 1.4 kb ) nanC-ﬁmB intergenic region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17238917" target="_blank">17238917</a>]<br></li>
<li>NanR and NagC each bind to operator sequences [ ONR ( originally termed region 1 ) and ONC1 ( originally termed region 2/NagC1 ) respectively ] situated far upstream of ﬁmB ( Sohanpal et al. , 2004 ; Fig. 1 ) to stimulate ﬁmB expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17238917" target="_blank">17238917</a>]<br></li>
<li>NanR and NagC activate ﬁmB expression by binding to operator sequences ( ONR and ONC1 respectively ; Fig. 1 ) situated further upstream of the regulated promoter than is usual in bacteria .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17238917" target="_blank">17238917</a>]<br></li>
<li>Even though the helix -- turn -- helix motifs of Mlc and NagC are almost identical and both proteins bind to similar sequences , there are distinctions in their operator selection .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9484893" target="_blank">9484893</a>]<br></li>
<li>In addition , the adjacent DNA sequences , position +7 / -- 7 to +10 / -- 10 , were predominately AT-rich as found for the native NagC and Mlc sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11139621" target="_blank">11139621</a>]<br></li>
<li>A characteristic of both sets of ` selected ' sequences is that the central positions , -- 4 to +4 , are GC-rich , ~ 76 and 69 % for NagC and Mlc sites , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11139621" target="_blank">11139621</a>]<br></li>
<li>NagC bound to the NagC selected sites ( C5 , C9 and C42 ) at least 100-fold better than to the Mlc sites ( M1 , M26 ) and at least 10 times better than to M3 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11139621" target="_blank">11139621</a>]<br></li>
<li>Effect of Mlc and NagC operator sequences on expression of ptsG -- lacZ and nagBE -- lacZ fusions Derepression ptsG -- lacZ Plasmid Strain nagBE -- lacZ w.t w.t. M26 M1 1.3 1.8 13.1 11.5 w.t. w.t. M11 M3 2.6 1.8 6.5 5.8 w.t. w.t. M134 C24 5.4 4.7 2.2 1 w.t. w.t. C17 C5 5.3 5.5 1 1 w.t. w.t. C43 C42 7.5 8.1 1 1 1 w.t. w.t. C9 8.2 15 1.1 1 pTZ ( Nag14E-15B ) w.t. pTZ19 pTZ19 1 35 mlc nagC 13 JM101 ( mlc + , nagC + ) , lysogenised with either λRS / nagBE -- lacZ or λRS / ptsG -- lacZ , was transformed with pTZ19R carrying the NagC and Mlc selected binding sites indicated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11139621" target="_blank">11139621</a>]<br></li>
<li>Results NagC binds upstream of chiP in E. coli and Salmonella but ChbR only binds in Salmonella DNase I footprinting conﬁrmed that NagC bound to a site upstream of chiP overlapping the − 10 and − 35 regions of the chiP promoter ( site NagC1 ) in S. enterica serovar Typhimurium LT2 ( henceforth referred to as Salmonella ) as well as in E. coli ( Fig. 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24593230" target="_blank">24593230</a>]<br></li>
<li>We searched by eye and found a second potential NagC site at -98 ( nagO2 Fig. 1A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19007420" target="_blank">19007420</a>]<br></li>
<li>NagC binds tightly to the nagO1 site but no protection of the putative nagO2 operator ( which overlaps the galOE2 operator ) was detected ( Fig. 3A lane 15 , 3B lanes 2 -- 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19007420" target="_blank">19007420</a>]<br></li>
<li>Binding of NagC to wild-type and mutated glmU upstream The DNA fragment used was made by PCR using plasmids sequences .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7545108" target="_blank">7545108</a>]<br></li>
<li>Complexes between 32P-labelled PM DNA , carrying the wild-type ﬁm sequence or mutated in the ONC2 or ibs sites , and NagC ( 100 nM ) and/or IHF ( 40 nM ) , were formed as for the DNase I footprinting experiments .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17238917" target="_blank">17238917</a>]<br></li>
<li>Moreover , region 2 also contains a sequence that shows homology ( 5 - tgcAaTT ( N9 ) AAATAtG ) to a binding consensus for the GlcNAc-6-P-responsive protein , NagC ( 5-STTATTT ( N9 ) AAATAAS ) ( 43 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15534208" target="_blank">15534208</a>]<br></li>
<li>Analysis of the protein sequence deduced from the DNA sequence of nagC identified a possible DNA-binding region of the type a helix-turn-a helix described by Pabo and Sauer ( 1985 ) , located in the carboxy terminus of NagC ( amino acids 315-334 , nucleotides 2154-2213 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2668691" target="_blank">2668691</a>]<br></li>
<li>Lanes 3 to 5 contain 0.4 , 1.0 and 4.0 pmol NagC box ( B ) DPU`A ; lanes 6 to 8 contain 0.4 , 1.0 and 4.0 pmol NagC box ( E ) DNA ; lanes 9 to 11 contain 04 , 1.0 and 40 pmol of an arbitrary 30mer sequence ( GGGGTGT ~ ~ ~ G ( 2 ~ G ~ CFF ~ - CCTTTTGTAATTACAA ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1848637" target="_blank">1848637</a>]<br></li>
<li>Regions protected by NagC against DNase I attack are shown by diagonally shaded boxes and the bases protected are boxed on the sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1848637" target="_blank">1848637</a>]<br></li>
<li>Moreover , the location of the distal NagC site suggests that this site is behaving like an upstream activating sequence ( UAS ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7545108" target="_blank">7545108</a>]<br></li>
<li>In this work we wanted to investigate the NagC recognition sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7791215" target="_blank">7791215</a>]<br></li>
<li>This sequence is denoted as NagC box ( M ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1848637" target="_blank">1848637</a>]<br></li>
<li>We analysed the three sequences protected by NagC , box ( E ) , box ( B ) and box ( M ) , to determine if a sequence can be deduced as the NagC recognition site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1848637" target="_blank">1848637</a>]<br></li>
<li>The precise limits of the regions protected by NagC are difficult to define since both contain A+T - rich sequences not sensitive to DNase I .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1848637" target="_blank">1848637</a>]<br></li>
<li>( b ) Comparison of the NagC protected sequences in the nagE-nagB and manX regions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1848637" target="_blank">1848637</a>]<br></li>
<li>The sequences of the proposed GalR and NagC operators are given and compared with the GalR consensus ( Weickert and Adhya , 1993a ) and NagC consensus ( Plumbridge , 2001 ; El Qaidi and Plumbridge , 2008 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19007420" target="_blank">19007420</a>]<br></li>
<li>Operator sequences of E. coli and conserved bases in other species are highlighted : GalR/S operators blue ; NagC operator yellow , CAP site green .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19007420" target="_blank">19007420</a>]<br></li>
<li>The NagC and Mlc protein -- DNA interaction The strictly conserved TT/AA at positions -- 6 , -- 5 / +5 , +6 bp from the centre of symmetry , is probably the major recognition point for both NagC and Mlc .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11139621" target="_blank">11139621</a>]<br></li>
<li>The hypermethylated A residues in the nag DNA complex are separated by 94 bp , i.e. by nine turns of the DNA helix , assuming a helical pitch of 195 bp ; so that the NagC molecules should be bound to the same face of the DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1848637" target="_blank">1848637</a>]<br></li>
<li>In the nag case , the CAP site is located in phase with the two NagC operators with the centres of symmetry separated by 30.5 and 63.5 bp ( 2.9 and 6.1 turns of the DNA helix ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9484892" target="_blank">9484892</a>]<br></li>
<li>The centre of symmetry of the CAP site is separated from the centres of the NagC box ( E ) and ( B ) by three and six turns of the double helix , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1848637" target="_blank">1848637</a>]<br></li>
<li>Two other mutations were made by the insertion of 6 and lObp of DNA between the two NagC boxes at position -92 compared to the nagE transcription start site and -42 relative to the nagB transcription start site ( Flg ,1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7934873" target="_blank">7934873</a>]<br></li>
<li>Binding of NagC was tested with a mutated form of the nag-EB operator DNA where 6 bp had been introduced into the DNA between the NagC boxes ( at position -92 on Fig .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1848637" target="_blank">1848637</a>]<br></li>
<li>Introducing 6 bp , i.e. half a helical turn , into the DNA between the two NagC boxes eliminated the production of hypersensitive bands in this region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1848637" target="_blank">1848637</a>]<br></li>
<li>The interaction of IHF and NagC in vitro Binding of IHF to the element investigated above would be expected to produce a hairpin-like bend in the DNA midway between ONC1 and ONC2 ( see Rice et al. , 1996 for a review ) , and if so , should either facilitate or prevent contact between NagC bound to its two operator sites depending upon the phasing of the sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17238917" target="_blank">17238917</a>]<br></li>
<li>Using genetic and biochemical approaches , IHF was shown to bind to a site midway between ONC1 and ONC2 to facilitate strong cooperative binding of NagC to these elements .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17238917" target="_blank">17238917</a>]<br></li>
<li>Despite their comparatively weak binding of NagC , inspection of the DNA sequence in these two regions identified sequences , called BoxGl and BoxG2 , in agreement with the consensus for NagC operators ( Figure 2B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7545108" target="_blank">7545108</a>]<br></li>
<li>As well as producing two completely protected regions , the binding of NagC changes the pattern of DNase I attack in between these two regions producing a series of seven hypersensitive bands separated by 9 to 11 bp ( Fig. 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1848637" target="_blank">1848637</a>]<br></li>
<li>The regions protected by NagC binding , called NagC , BoxB and BoxE referring to the adjacent gene , cover at least 20 bp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7934873" target="_blank">7934873</a>]<br></li>
<li>To try to distinguish these two possibilities we examined the binding of NagC to synthetic double-stranded DNA fragments of 28 and 34 bp corresponding to the regions protected by NagC , box ( B ) and box ( E ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1848637" target="_blank">1848637</a>]<br></li>
<li>Both sites can be recognized on the DNA sequence ( Fig. 1A ) by their similarity with the 23 bp consensus for NagC sites ( -11 SWWWWTTNNNNNNNNNAAWWWWS +11 ) , where W is A or T and S is C or G ( Plumbridge , 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15066032" target="_blank">15066032</a>]<br></li>
<li>This site however , is not optimal because it has a CG base pair within the region expected to be either A or T and is missing CG bp at positions 11 ( Fig. 1B ) , which are characteristic of high-affinity NagC sites ( Plum-bridge and Kolb , 1995 ; El Qaidi and Plumbridge , 2008 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19007420" target="_blank">19007420</a>]<br></li>
<li>NagC binds to two sites separated by 115 bp and the transcription start point of the chb operon lies within the downstream NagC operator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15066032" target="_blank">15066032</a>]<br></li>
<li>In addition the chb operon encodes its own speciﬁc regulator , ChbR , an AraC-type dual repressor -- activator , which binds to two direct repeats of 19 bp located between the two NagC sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15066032" target="_blank">15066032</a>]<br></li>
<li>DNase I footprinting demonstrated that NagC binds to two sites , one site corresponds to region 2 as described above ( Fig. 3B , lanes 7 -- 10 , NagC1 ) and the other to an additional site centered 212 bp downstream ( Fig. 3B , lanes 12 -- 15 , NagC2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15534208" target="_blank">15534208</a>]<br></li>
<li>Thus , the binding sites are out of phase , which probably makes it impossible for NagC to bind simultaneously to the two operators and form a DNA loop .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9484892" target="_blank">9484892</a>]<br></li>
<li>NagC binding produces a series of DNase I hypersensitive sites separated by 9 to 11 bp in the region between the two NagC binding sites , supporting a model where the NagC proteins bind co-operatively to these two sites on the DNA and interact to form a DNA loop .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1848637" target="_blank">1848637</a>]<br></li>
<li>The distance between the two NagC sites upstream of chiP is 223 bp in E. coli and 235 bp in Salmonella .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24593230" target="_blank">24593230</a>]<br></li>
<li>NagC binds to two sites upstream of the chiP gene , one spanning the interval between the − 35 and − 10 hexamers of the promoter , the other more than 200 bp upstream from the promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24593230" target="_blank">24593230</a>]<br></li>
<li>Two binding sites for the NagC repressor are located at 47 -200 and bp upstream of P1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7545108" target="_blank">7545108</a>]<br></li>
<li>However , the introduction of 6 bp between the two NagC binding sites changed the phasing of the two NagC boxes and resulted in preferential binding of NagC to the box ( B ) ( Fig. 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1848637" target="_blank">1848637</a>]<br></li>
<li>Here it is demonstrated that NagC binding to a second site ( ONC2 ) , located 212 bp closer to ﬁmB , also controls ﬁmB expression , and that integration host factor ( IHF ) , which binds midway between ONC1 and ONC2 , facilitates NagC binding to its two operator sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17238917" target="_blank">17238917</a>]<br></li>
<li>It thus seemed possible to us that a cofactor , binding within the 189 bp of DNA separating ONC1 and ONC2 , is required for the cooperative binding of NagC to the two sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17238917" target="_blank">17238917</a>]<br></li>
<li>As a ﬁrst approach to identify a role for these three putative regulatory proteins , ChbR , NagC and CAP , DNaseI footprinting was used to locate binding sites in the chbB promoter region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15066032" target="_blank">15066032</a>]<br></li>
<li>An unusual feature of three of these repressor binding sites was the observation that an adenine residue in the centre of the region protected against DNAseI cleavage , was hypermethylated by DMS , when the NagC repressor was bound .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7791215" target="_blank">7791215</a>]<br></li>
<li>Identical bases in the direct repeats of the two ChbR binding sites are underlined and the bases which do not agree with the NagC consensus ( Plumbridge , 2001 ) are shown with a dot underneath .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15066032" target="_blank">15066032</a>]<br></li>
<li>The four previously characterized NagC binding sites , BoxE , BoxB , BoxM and BoxM ' ( Plumbridge and Kolb , 1991 ) are aligned with the two binding sites BoxGl and G2 upstream of glmU .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7545108" target="_blank">7545108</a>]<br></li>
<li>DNAsel footprinting located two binding sites for the NagC protein overlapping the promoters of the divergent nagE and nagB genes -LCB- Plumbridge and Kolb .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1766379" target="_blank">1766379</a>]<br></li>
<li>Sequences tested for NagC or Mlc binding in vivo by operator titration ( Table 3 ) are indicated by a cross .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11139621" target="_blank">11139621</a>]<br></li>
<li>The query sites are shown in Table 1 and consisted of known NagC operator sequences compiled from [ 31 ] and J. Plumbridge ( pers .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21070640" target="_blank">21070640</a>]<br></li>
<li>Moreover it binds NagC as well as the other C sites tested .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11139621" target="_blank">11139621</a>]<br></li>
<li>Comparison of NagC operator sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7545108" target="_blank">7545108</a>]<br></li>
<li>NagC and Mlc have signiﬁcant sequence homology in the region predicted to form the helix -- turn -- helix DNA binding motif , and some cross-recognition between NagC and Mlc binding sites is observed at the mlc and manX operators ( but not at the malT site ) ( Decker et al. , 1998 ; Plumbridge , 1998 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767573" target="_blank">9767573</a>]<br></li>
<li>However , at least one of the Mlc selected DNA binding sites , M120 , with limited homology to the NagC/Mlc consensus , did not bind to purified Mlc or NagC when tested in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11139621" target="_blank">11139621</a>]<br></li>
<li>] DISCUSSION Consensus sequences for NagC and Mlc binding sites The consensus sequence for a NagC binding site has been deduced by three methods : ( i ) from the comparison of the native sites ; ( ii ) by mutagenesis of one operator to determine which bases alter regulation ( i.e. on the criteria of NagC binding in vivo ) ; and ( iii ) by the selection of DNA fragments binding NagC in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11139621" target="_blank">11139621</a>]<br></li>
<li>The numbers under the line give the location of the CAP and Mlc/NagC binding sites relative to the transcription start points ( þ1 ) of nagE on the nagE -- B sequence ( top ) and of manX on the lower sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9484892" target="_blank">9484892</a>]<br></li>
<li>Both sites conform with the so far identiﬁed consensus for a NagC binding site , namely TT at positions ¹ 5 and ¹ 6 and AA at þ 5 , þ 6 and AT basepairs at positions 7 -- 10 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9484893" target="_blank">9484893</a>]<br></li>
<li>A binding site selection procedure was used to try to distinguish NagC and Mlc sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11139621" target="_blank">11139621</a>]<br></li>
<li>So for both Mlc and NagC sites there is some indication that the CGCG pattern in the centre of the palindromic recognition site does indeed enhance binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11139621" target="_blank">11139621</a>]<br></li>
<li>The sequences were aligned by searching for the ` TT 9 bp AA ' motif shown to be essential for NagC binding ( 16 ) and apparently conserved in the Mlc sites ( Fig. 1B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11139621" target="_blank">11139621</a>]<br></li>
<li>The square brackets locate the CAP binding sites 1 and 2 and the brackets B and E locate the NagC protected sequences .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1848637" target="_blank">1848637</a>]<br></li>
<li>The major difference was that all selected NagC binding sites had a G or a C at positions +11 / -- 11 from the centre of symmetry .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11139621" target="_blank">11139621</a>]<br></li>
<li>Rather surprisingly , none of the selected NagC binding sites produced > 8-fold induction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11139621" target="_blank">11139621</a>]<br></li>
<li>Note that the NagC consensus has a central zero position .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19007420" target="_blank">19007420</a>]<br></li>
<li>The deduced palindromic NagC operator half site , as written in Figures 1 and 3 , has a unique base at the 0 position .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7791215" target="_blank">7791215</a>]<br></li>
<li>To deﬁne the NagC operator binding site and in particular to see if it is palindromic with a central position we undertook a mutagenesis study on the nagE operator , BoxE from positions 0 to − 13 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7791215" target="_blank">7791215</a>]<br></li>
<li>A second very weak CAP site upstream from manX in the region - 80 to - 100 is detected only at very high CAP concentra , tions As in the case of the NagC boxes ( E ) and ( B ) , NagC completely protects a region of at least 20 bp against DNase I attack .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1848637" target="_blank">1848637</a>]<br></li>
<li>NagC binds to the intergenic nagE-nagB region and upstream of manX .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1766379" target="_blank">1766379</a>]<br></li>
<li>The CAP ( = CRP = cAMP receptor protein = catabolite activator protein ) binds , in the presence of cAMP , to a site adjacent to the distal NagC site ( Fig. 2 , Lanes 2 and 11 ) centred at -83.5 from the transcriptional start site although protection extends back as far as -100 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15066032" target="_blank">15066032</a>]<br></li>
<li>The strong CAP tor fragment where 6 bp have been introduced between the NagC boxes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1848637" target="_blank">1848637</a>]<br></li>
<li>Note the lack of hypersensitive DNase I bands in the DEA between the 2 NagC boxes ( a ) The strong CAP site is located at functional distances from nagE and nagB Figure 8 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1848637" target="_blank">1848637</a>]<br></li>
<li>The P2 start site is at -103 and the centres of the NagC boxes are at -47 and -200 on this scale .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7545108" target="_blank">7545108</a>]<br></li>
<li>The location of a NagC box at -47 relative to P1 is consistent with it being an activation site for the P1 promoter when NagC is bound .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7545108" target="_blank">7545108</a>]<br></li>
<li>The 35 and 10 regions of the promoter are underlined , the NanR binding site is boxed in gray , the NagC binding site is boxed in white , and the CAP binding site is underlined with a wavy line .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15743943" target="_blank">15743943</a>]<br></li>
<li>The NagC and Mlc protected regions , BoxM and BoxM8 , are indicated by speckled boxes and the cAMP/CAP-binding site by an open box .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9484892" target="_blank">9484892</a>]<br></li>
<li>The relative positions of the NagC operators BoxE and BoxB are shown by hatched boxes and the CAP site by a shaded box .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7791215" target="_blank">7791215</a>]<br></li>
<li>Distance from transcription start site -- 100 +1 +100 -- 300 -- 200 +200 +300 CTTATTTTTTATATTGA TATTTAATCTATTGTTA ATAAGTA TGTAAGAA AATCCTA AATCCTA ydeP RcsB RcsB NarL/NarP TACCTCT EvgA NarL/NarP NagC PhoP Figure 6 Location of TF-binding sites on the ydeP promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26010043" target="_blank">26010043</a>]<br></li>
<li>In addition , the general silencer H-NS-binding site was identiﬁed in the ydeP promoter region ( Shimada et al. 2011 ) , and NagC - and PhoP-binding sequences are known to exist in the ydeP promoter region ( RegulonDB ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26010043" target="_blank">26010043</a>]<br></li>
<li>As described above , a potential NagC site was identiﬁed upstream of the structural genes ( centred at -112 from the transcription start point , +1 ) and a possible CAP consensus site is located at -83.5 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15066032" target="_blank">15066032</a>]<br></li>
<li>The upstream NagC-binding site , BoxM , which binds NagC strongly is centred at ¹ 80 , compared with the transcriptional start site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9484892" target="_blank">9484892</a>]<br></li>
<li>A NagC binding site is centered 65 nucleotides upstream of the transcription start site , so that it overlaps the CAP site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15743943" target="_blank">15743943</a>]<br></li>
<li>It is worth emphasising that NagC prefers G/C at +11 / -- 11 even when binding to a single site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11139621" target="_blank">11139621</a>]<br></li>
<li>Inspection of the region upstream of the chb structural genes identiﬁed a sequence which res-sembled a NagC consensus binding site ( Plumbridge , 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15066032" target="_blank">15066032</a>]<br></li>
<li>The site at which Mlc binds shows homology to the previously identiﬁed NagC operators ( Plumbridge and Kolb , 1995 ) , but no NagC binding to it was detected ( Fig. 1 , lane 9 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9484893" target="_blank">9484893</a>]<br></li>
<li>However , no binding of ChbR to the chiP upstream region of E. coli was detected ( Fig. 2C and D ) and the sequence protected in Salmonella , unlike the NagC site , is not conserved in E. coli ( Fig. 1C ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24593230" target="_blank">24593230</a>]<br></li>
<li>ChbR binding protects a region of about 40 bp between the CAP site and the proximal NagC site ( Fig. 2 , lane 6 and 15 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15066032" target="_blank">15066032</a>]<br></li>
				</ul></div><br>
			<label class="collapse_TU" for="_6"><font size="5px">Transcription units regulated by NagC</font></label>
				<input id="_6" type="checkbox">
				<div class="TU"><br><ul><li>Additional work is thus required to determine why methylation protection of GATCNanR and GATCNagC alternates as it does , and if and how methylation of these elements affect ﬁmB and nanC expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17238917" target="_blank">17238917</a>]<br></li>
<li>1.3 1.9 5.0 2.4 4.7 2.7 2.6 1.4 PspF , IHF PhoB RbsR , Crp 37.9 22.0 2.6 1.2 1.8 2.3 0.8 1.1 Inner membrane Unknown Ribose transport Ribose transport Ribose transport Ribose transport Ribose transport Ribose transport pspG ( yjbO ) ytfK rbsD rbsA rbsC rbsB rbsK rbsR 0.7 3.8 1.8 1.0 1.6 1.4 1.2 0.8 0.9 3.0 8.2 5.4 21.3 6.6 9.0 2.8 manX Mannose permease 4.2 2.1 1.1 5.0 manY Mannose permease 3.4 2.6 1.1 5.2 manZ Mannose permease 6.5 5.7 1.0 16.7 NagC , DgsA , Crp a. Genes and operons which are induced 10-fold in three independent experiments are listed with the average fold induction for the three experiments .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18710431" target="_blank">18710431</a>]<br></li>
<li>Precultures of JMlOlpUC ( NagC ) were grown in Luria broth containing 500 pg ampicillin/ml and then diluted 20-fold into 10 ml of minimal Mops medium containing 65 % ( v/v ) Casamino acids , 500 pg ampicillin/ml plus 1 mM-IPTG to induce expression of nagC , and grown for 4 to 6 h .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1848637" target="_blank">1848637</a>]<br></li>
<li>The very high derepression produced by the insertions in nagA is , however , the result of the lack of NagA protein itself and not a reduction in the concentration of NagC , since it is only fully complemented by plasmids expressing nagA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1766379" target="_blank">1766379</a>]<br></li>
<li>nagC , is refuted by the observation that the derepression caused by insertions of antibiotic resistance cassettes in nagA is complemented by a plasmid expressing just NagA in trans and not by a plasmid expressing NagC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1766379" target="_blank">1766379</a>]<br></li>
<li>Although the proximal site , BoxGl , is located at a position typical of prokaryotic activators ( Collado-Vides et al. , 1991 ; Gralla , 1991 ) , it is not sufficient to allow NagC to to activate glmU by itself ; this could be because NagC occupation of this lower affinity site is inadequate in the absence of the higher affinity upstream site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7545108" target="_blank">7545108</a>]<br></li>
<li>As described above , a potential NagC site was identiﬁed upstream of the structural genes ( centred at -112 from the transcription start point , +1 ) and a possible CAP consensus site is located at -83.5 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15066032" target="_blank">15066032</a>]<br></li>
<li>We have found that NagC , binds to a single , high-affinity site overlapping the promoter of galP ( galactose permease ) gene and that expression of galP is repressed by a combination of NagC , GalR and GalS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19007420" target="_blank">19007420</a>]<br></li>
<li>Screening the E. coli genome for potential NagC sites using a position-weighted matrice ( J. Oberto , in preparation ) located a high scoring NagC site upstream of the galP gene , encoding the major galactose permease .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19007420" target="_blank">19007420</a>]<br></li>
<li>Induction was normal in the presence of chitosan dimer in the chbG mutant when NagC repression was abrogated by deleting nagC , whereas chitobiose and chitotriose continued to be ineffective as inducers under the same conditions ( Fig. 4B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22797760" target="_blank">22797760</a>]<br></li>
<li>This is consistent with the hypothesis that growth on chitin under these conditions does not produce a sufﬁciently rapid ﬂux of chitobiose to GlcNAc-6P to produce a concentration sufﬁcient to induce NagC repressed genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15066032" target="_blank">15066032</a>]<br></li>
<li>These genes are , however , only weakly repressed by NagC under normal cellular conditions ( 27 , 41 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626331" target="_blank">8626331</a>]<br></li>
<li>Keywords : protein -- DNA recognition ; nagE-B operator ; NagC repressor ; cAMP/CAP Physico-chimique ( URA1139 ) 13 , rue Pierre et Marie Curie 75005 Paris , France 2Institut Pasteur ( URA1149 ) 25 , rue de Dr Roux , 75015 Paris , France * Corresponding author Introduction The Nag repressor controls expression of the divergent nagE-nagBACD operons by binding to two operators , called BoxE and BoxB , which overlap the nagE and nagB promoters ( Plumbridge & Kolb , 1991 , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7791215" target="_blank">7791215</a>]<br></li>
<li>NagC regulates expression of the divergent nagE-BACD operons by binding to two operators ( called BoxE and BoxB ) overlapping the nagE-B promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11139621" target="_blank">11139621</a>]<br></li>
<li>Three nagE -- lacZ fusions were tested : the standard ` loop-forming ' nagBE -- lacZ fusion , whose expression is repressed by NagC binding co-operatively to the nagE and nagB operators ; and two single operator nagE -- lacZ fusions , missing the nagB promoter/operator , but carrying the same nagE -- lacZ junction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11139621" target="_blank">11139621</a>]<br></li>
<li>Although Mlc does bind specifically to both the nagE and nagB operators in vitro , no regulation of nagBE expression in vivo by the natural chromosomal copy of mlc was detected , even in the absence of NagC binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11139621" target="_blank">11139621</a>]<br></li>
<li>If NanR is less effective at enhancing fimB expression than NagC , as it seems to be , then the recombinase genes expression must therefore be controlled by phase variation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15534208" target="_blank">15534208</a>]<br></li>
<li>Thus , NagC is a positive regulator of fimB expression and could participate in the control of the recombinase genes expression in response to sialic acid .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15534208" target="_blank">15534208</a>]<br></li>
<li>Thus the same factors , NanR and NagC , control both fimB and yjhA expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15534208" target="_blank">15534208</a>]<br></li>
<li>Here it is demonstrated that NagC binding to a second site ( ONC2 ) , located 212 bp closer to ﬁmB , also controls ﬁmB expression , and that integration host factor ( IHF ) , which binds midway between ONC1 and ONC2 , facilitates NagC binding to its two operator sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17238917" target="_blank">17238917</a>]<br></li>
<li>Although we have not characterized these fortuitous regulatory sequences further , it now seems unlikely that NanR and NagC overcome the inhibitory effect of cis-acting sequences further upstream of ﬁm to activate ﬁmB expression as we supposed originally .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17238917" target="_blank">17238917</a>]<br></li>
<li>However , it is clear that they allow coordination of nanC and ﬁmB expression , since NanR and NagC repress one and activate the other of these divergently arranged genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15743943" target="_blank">15743943</a>]<br></li>
<li>NanR and NagC are two regulators that control nanC expression by binding to or in the vicinity of its promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15743943" target="_blank">15743943</a>]<br></li>
<li>NanR ( Neu5Ac-responsive ) and NagC ( GlcNAc-6P-responsive ) activate ﬁmB expression by binding to operators ( ONR and ONC1 respectively ) located more than 600 bp upstream of the ﬁmB promoter within the large ( 1.4 kb ) nanC-ﬁmB intergenic region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17238917" target="_blank">17238917</a>]<br></li>
<li>NanR and NagC activate ﬁmB expression by binding to operator sequences ( ONR and ONC1 respectively ; Fig. 1 ) situated further upstream of the regulated promoter than is usual in bacteria .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17238917" target="_blank">17238917</a>]<br></li>
<li>First , repression of nanC and fimB divergent transcripts by NagC has been observed both in vivo and in vitro [ 32 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21070640" target="_blank">21070640</a>]<br></li>
<li>The expression of both fusions was increased by growth on galactose and to a lesser extent by growth on GlcNAc , supporting the hypothesis that NagC is involved in repression of galP ( Table 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19007420" target="_blank">19007420</a>]<br></li>
<li>There are two NagC operator sites upstream of gimU The perfect inverse correlation between the induction of nag and repression of glm genes posed the question as to whether the regulation of glmU expression involved the NagC protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7545108" target="_blank">7545108</a>]<br></li>
<li>However , mutations in neither galR nor galS had any effect on chiP -- lacZ expression ( Fig. 4 ) and did not affect repression by NagC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24593230" target="_blank">24593230</a>]<br></li>
<li>Plasmids expressing either NagC or Mlc produce considerable repression of the nagBE -- lacZ looped fusion , reducing expression 88 - and 56-fold ( Table 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11139621" target="_blank">11139621</a>]<br></li>
<li>It can also be noted that the distances between the NagC sites at chiP are longer than at other NagC controlled genes ( Cournac and Plumbridge , 2013 ) , which will not favour loop formation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24593230" target="_blank">24593230</a>]<br></li>
<li>It thus appears that NagC plays a key role in setting chiP expression levels under uninduced conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24593230" target="_blank">24593230</a>]<br></li>
<li>These observations are consistent with galP being controlled by NagC as well as GalR/S .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19007420" target="_blank">19007420</a>]<br></li>
<li>While Mlc binds well to both operators , NagC is only a weak repressor for mlc and does not repress malT .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9484893" target="_blank">9484893</a>]<br></li>
<li>Chitosan dimer-6-P can act as inducer for ChbR , but its hydrolysis by ChbF does not produce GlcNAc-6-P required for NagC derepression , resulting in poor induction and consequently poor growth .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22797760" target="_blank">22797760</a>]<br></li>
<li>Considering the wide spectrum of substrates for the manXYZ-encoded transporter , it is , in fact , not desirable that manXYZ should be strongly regulated by the NagC repressor and only expressed in the presence of its amino sugar substrates , GlcNAc and GlcN .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9484892" target="_blank">9484892</a>]<br></li>
<li>Genes nagA , B , and E are members of the nag regulon which is regulated by the repressor NagC ( nagC ) and the inducer GlcNAc 6P ( Plumbridge , 1989 ; Vogler and Lengeler , 1989 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10931310" target="_blank">10931310</a>]<br></li>
<li>The NagC box ( M ) on the other hand is much further upstream , being located at -86 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1848637" target="_blank">1848637</a>]<br></li>
<li>Previously , four NagC operators had been characterized , two in the intergenic nagE-B region and two upstream of the manXYZ operon ( Plumbridge and Kolb , 1991 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7545108" target="_blank">7545108</a>]<br></li>
<li>We have recently shown that the NagC protein is capable of binding to the DNA of the nagE-nagB intergenic region and also upstream ot the manX gene ( Plumbridge and Kolb , 1991 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1766379" target="_blank">1766379</a>]<br></li>
<li>( e ) The DNA sequewe upstream from manX contains binding sites for CAP and NagC The results of Vogler & Lengler ( 1989 ) suggested that the manXYZ operon , whose gene product transports GlcN and GlcNAc , is also controlled by the nagC encoded repressor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1848637" target="_blank">1848637</a>]<br></li>
<li>Two binding sites for the NagC protein also exist upstream of the manXYZ operon ( 33 ) , which encodes a second transporter for GlcNAc and is the major transporter for GlcN as well as for other hexose sugars ( 5 , 15 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626331" target="_blank">8626331</a>]<br></li>
<li>Two NagC binding sites were also located upstream of the manXYZ operon , of which one , BoxM , was strongly protected and the second , BoxM ' , more weakly ( Plumbridge & Kolb , 1991 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7791215" target="_blank">7791215</a>]<br></li>
<li>In addition NagC and CAP binding sites have been found upstream from the manXYZ operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1848637" target="_blank">1848637</a>]<br></li>
<li>This also explains why the IS elements isolated by Parker and Hall ( 1990b ) in the distal NagC site ` decryptiﬁed ' the operon and allowed growth on cellobiose .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15066032" target="_blank">15066032</a>]<br></li>
<li>In the absence of chitosugars ( uninduced ) the chb , chiP and nag operons are repressed by NagC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24593230" target="_blank">24593230</a>]<br></li>
<li>The subsequent metabolism of chitobiose by the enzymes of the chb operon , ChbFG ( Verma and Mahadevan , 2012 ) generates GlcNAc6P , which inactivates NagC leading to full-scale induction of both the chiP and chb operons .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24593230" target="_blank">24593230</a>]<br></li>
<li>Second , as the ﬂux of chitobiose through the ChbBCA transporter and ChbFG enzymes increases , the level of GlcNAc6P , the inducer for NagC , increases and relieves repression by NagC of both chiP and chb operons .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24593230" target="_blank">24593230</a>]<br></li>
<li>The complete absence of any growth in the case of the permease ( chbA ) mutant and the phospho - - glucosidase ( chbF ) mutant compared to the parent strain when NagC is inactive suggests that chitosan dimer is transported and hydrolyzed by the chb operon ( Table 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22797760" target="_blank">22797760</a>]<br></li>
<li>This shows that whereas both NagC and ChbR appear to act as repressors for the chbB operon in the absence of the inducing sugar chitobiose , ChbR is necessary for induction and so behaves as a dual-function repressor -- activator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15066032" target="_blank">15066032</a>]<br></li>
<li>NagC is required for full repression of the chbB operon and it serves to limit induction when the supply of chitobiose is low .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15066032" target="_blank">15066032</a>]<br></li>
<li>It thus seemed likely that the chb operon forms part of the NagC controlled amino sugar regulon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15066032" target="_blank">15066032</a>]<br></li>
<li>Expression of the chb operon is repressed by NagC , which regulates genes involved in amino sugar metabolism .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15066032" target="_blank">15066032</a>]<br></li>
<li>The chb operon is regulated by three transcription factors : NagC , a key regulator of the nag genes involved in amino sugar metabolism ; ChbR , a dual function operon-speciﬁc regulator ; and CRP-cyclic AMP ( CRP-cAMP ) ( 45 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22797760" target="_blank">22797760</a>]<br></li>
<li>NagC , along with ChbR represses the operon in the absence of the inducer .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22797760" target="_blank">22797760</a>]<br></li>
<li>NagC is encoded by the third gene of this operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11139621" target="_blank">11139621</a>]<br></li>
<li>Degradation of cellobiose does not produce the GlcNAc-6P necessary to relieve NagC repression , but in the absence of NagC repression cellobiose is capable of activating chb operon expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15066032" target="_blank">15066032</a>]<br></li>
<li>We can deduce that cellobi-ose-6P is capable of generating an inducing signal for ChbR but not an inducing signal for NagC and that induction of both regulators is necessary for derepressed expression of the chb operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15066032" target="_blank">15066032</a>]<br></li>
<li>The need for two speciﬁc inducing signals , one for ChbR to activate the expression of the operon and a second for NagC to relieve its repression , ensure that the chb operon is only induced when there is sufﬁcient ﬂux through the combined chb-nag metabolic pathway to activate expression of both the chb and nag operons .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15066032" target="_blank">15066032</a>]<br></li>
<li>The situation described here for the chb operon is more unusual in the sense that two ` sugar-speciﬁc ' factors , NagC and ChbR , cooperate with CAP to control chb operon expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15066032" target="_blank">15066032</a>]<br></li>
<li>It will require a detailed study to clarify the exact roles of NagC and ChbR in expression of the chb operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15066032" target="_blank">15066032</a>]<br></li>
<li>It has been shown that the three proteins ChbR , CRP and NagC regulate the expression of the chb operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18028317" target="_blank">18028317</a>]<br></li>
<li>The repression of the manXYZ operon by a plasmid expressing NagC is in agreement with the original observations of Vogler and Lengeler ( 1989 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9484892" target="_blank">9484892</a>]<br></li>
<li>The implications of this experiment were that the nagE operator , the part of the divergent nagE-B operon system that is known to be controlled by NagC , appeared to have the characteristics of a Mlc operator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11139621" target="_blank">11139621</a>]<br></li>
<li>Under uninduced conditions , transcription of the chb operon is repressed by the NagC repressor , which binds to two sites in the chb promoter region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24593230" target="_blank">24593230</a>]<br></li>
<li>The glmUS operon , which is regulated positively by NagC , also relies upon co-operative binding of NagC to two sites ( 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11139621" target="_blank">11139621</a>]<br></li>
<li>The man operon is controlled by the NagC repressor ( 345 , 510 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8246840" target="_blank">8246840</a>]<br></li>
<li>The glmUS operon is positively controlled by NagC , which acts as an essential activator for one promoter and a weak repressor for a second , upstream promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626331" target="_blank">8626331</a>]<br></li>
<li>Here , we show that the chiP gene , like the chbBCARFG operon , is also downregulated at the transcriptional level by the NagC repressor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24593230" target="_blank">24593230</a>]<br></li>
<li>This suggested that , like the chbBCARFG operon , the chiPQ operon might be under the negative control of the NagC repressor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24593230" target="_blank">24593230</a>]<br></li>
<li>Discussion We have shown that the chiP gene , encoding a specialized porin for chitobiose and chitotriose , is under the control of the N-acetylglucosamine ( GlcNAc ) operon repressor , NagC , in both E. coli and Salmonella .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24593230" target="_blank">24593230</a>]<br></li>
<li>Another , also involving the NagC repressor , is the chbBCAFG operon , encoding genes for the uptake and utilization of chitobiose ( Plumbridge and Pellegrini , 2004 ) but as chitobiose is the dimer of GlcNAc , inclusion of chb in the nag regulon is not surprising .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19007420" target="_blank">19007420</a>]<br></li>
<li>The latter step is critical for the initiation of the entire induction cascade , as the chbBCARFG operon itself is repressed by NagC ( Plumbridge and Pellegrini , 2004 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24593230" target="_blank">24593230</a>]<br></li>
<li>NagC is best characterized as a repressor ( Plumbridge , 2001 ; Plumbridge and Pellegrini , 2004 ) , although the regulator does activate one of the glmUS operon promoters albeit by binding at the more conventional position of -47 relative to the transcriptional start site ( Plumbridge , 1995 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17238917" target="_blank">17238917</a>]<br></li>
<li>While glmUS transcription is controlled by NagC protein ,2 which also represses operons involved in amino sugar uptake and degradation , σ54 and GlmY may independently adjust GlmS synthesis in response to the general nitrogen status of the cell .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17854828" target="_blank">17854828</a>]<br></li>
<li>NagC represses nagE , encoding the N-acetylglucosamine-specific transporter , while Mlc represses three PTS operons , ptsG , manXYZ and ptsHIcrr , involved in the uptake of glucose .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11139621" target="_blank">11139621</a>]<br></li>
<li>In the absence of an environmental supply of amino sugars , NagC represses the divergent nagE-BACD operons , which are necessary for the utilisation of GlcNAc as a carbon source ( 1 ) , and activates the expression of one promoter of the glmUS operon that is part of the biosynthetic pathway for the formation of GlcN-6-phosphate ( GlcN-6-P ) and UDP-GlcNAc ( 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11139621" target="_blank">11139621</a>]<br></li>
<li>expression glmU The NagC repressor controls the expression of the divergently transcribed nagE-nagBACD operons involved in the uptake and degradation of the amino sugars , N-acetyl-D-glucosamine ( GlcNAc ) and glucos-amine ( GlcN ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7545108" target="_blank">7545108</a>]<br></li>
<li>NagC is the repressor of the divergent nagE -- nagBACD operon , which encodes proteins involved in N-acetylglucosamine ( GlcNAc ) transport and metabolism .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24593230" target="_blank">24593230</a>]<br></li>
<li>NagC acts as a repressor for the two divergent operons , nagE and nagBACD ( uptake and utilization of glucosamine and N-acetyl-glucosamine ; Vogler and Lengeler , 1989 ; Plumbridge , 1991 ) , and is also functioning as an activator for the biosynthetic glmUS operon ( synthesis of glucos-amine-6-P and UDP-GlcNAc ; Plumbridge , 1995 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9484893" target="_blank">9484893</a>]<br></li>
<li>Mlc shows homology to NagC , a gene regulator functioning as a repressor for the divergent nag operons ( encoding proteins for the uptake and degradation of N-acetyl-D-glucosamine ) and as an activator as well as a repressor for the glmUS operon encoding enzymes for the biosynthesis of N-acetyl-D-glucosamine ( 188 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9529892" target="_blank">9529892</a>]<br></li>
<li>JOURNAL OF BACTERIOLOGY , May 1996 , p. 2629 -- / 96 / $ 04.0010 Copyright q 1996 , American Society for Microbiology o Achieve Constitutive Expression of a Gene an Inducible Operon : the Example of the nagC Gene of Escherichia coli How T within JACQUELINE PLUMBRIDGE Institut de Biologie Physico-chimique ( URA1139 ) , 75005 Paris , France Received 20 November 1995/Accepted 13 February 1996 The nagC gene , encoding the NagC repressor/activator of the nag regulon , is part of the nagBACD operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626331" target="_blank">8626331</a>]<br></li>
<li>Discussion The results described here show that three transcription factors , ChbR , NagC and cAMP/CAP , cooperate in the repression and induction of the chbBCARFG operon in E. coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15066032" target="_blank">15066032</a>]<br></li>
				<label class="TU_collapse_similarity" for="tu_similarity_84">
Expression of the chitobiose operon of Escherichia coli is regulated by three transcription factors : NagC , ChbR , and CAP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22797760" target="_blank">22797760</a>]<br></label>
<input id="tu_similarity_84", type="checkbox">
<div class="TU_similar">
<li>Expression of the chitobiose operon of Escherichia coli is regulated by three transcription factors : NagC , ChbR and CAP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15516564" target="_blank">15516564</a>]<br></li>
<li>Plumbridge , J. , and Pellegrini , O. ( 2004 ) Expression of the chitobiose operon of Escherichia coli is regulated by three transcription factors : NagC , ChbR and CAP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18028317" target="_blank">18028317</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17824929" target="_blank">17824929</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17238917" target="_blank">17238917</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24593230" target="_blank">24593230</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19007420" target="_blank">19007420</a>]<br></li>
<li>Plumbridge , J. and Pellegrini , O. ( 2004 ) Expression of the chitobiose operon of Escherichia coli is regulated by three transcription factors : NagC , ChbR and CAP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18346968" target="_blank">18346968</a>]<br></li></div>
				</div></ul></div><br>
	</div>
</body>
</html>