<!doctype html>
<style>
@charset "utf-8";
body {
	height: 100%;
	margin: 0;
	padding: 0;
}

.banner {
	position: fixed;
	top: 0;
	width: 100%;
	height: 65px;
	background: gray;
	line-height: 65px;
}
.spacer {
	height: 10px;
}

.main-content {
	padding: 5%;
}

.collapse_ACT {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-style: none solid none none;
	border-color: #B4045F;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_ACT + input{
	display: none;
}
.collapse_ACT + input + div{
	display: none;
}
.collapse_ACT + input:checked + div{
	display: block;
}
.ACT {
	text-align: justify;
	text-justify: inter-word;
}
.ACT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B4045F;
}
.ACT_collapse_similarity:hover{
	background: #e5d3dc;
}
.ACT_collapse_similarity + input {
	display: none;
}
.ACT_collapse_similarity + input + div{
	display: none;
}

.ACT_collapse_similarity + input:checked + div{
	display: block;
}
.ACT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.ACT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B4045F;
	padding-left: 5px;
}
.ACT_collapse_similarity:hover{
	background: #e5d3dc;
}
.ACT_collapse_similarity + input {
	display: none;
}
.ACT_collapse_similarity + input + div{
	display: none;
}
.ACT_collapse_similarity + input:checked + div{
	display: block;
}
.ACT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.ACT_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #B4045F;
}
.ACT_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_DOM {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-style: none solid none none;
	border-color: #B404AE;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_DOM + input{
	display: none;
}
.collapse_DOM + input + div{
	display: none;
}
.collapse_DOM + input:checked + div{
	display: block;
}
.DOM {
	text-align: justify;
	text-justify: inter-word;
}
.DOM_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B404AE;
	padding-left: 5px;
}
.DOM_collapse_similarity:hover{
	background: #e8d7e7;
}
.DOM_collapse_similarity + input {
	display: none;
}
.DOM_collapse_similarity + input + div{
	display: none;
}
.DOM_collapse_similarity + input:checked + div{
	display: block;
}
.DOM_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}.DOM_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #B404AE;
}
.DOM_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_EVO {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #7401DF;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_EVO + input{
	display: none;
}
.collapse_EVO + input + div{
	display: none;
}
.collapse_EVO + input:checked + div{
	display: block;
}
.EVO {
	text-align: justify;
	text-justify: inter-word;
}
.EVO_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #7401DF;
	padding-left: 5px;
}
.EVO_collapse_similarity:hover{
	background: #e2d7ed;
}
.EVO_collapse_similarity + input {
	display: none;
}
.EVO_collapse_similarity + input + div{
	display: none;
}
.EVO_collapse_similarity + input:checked + div{
	display: block;
}
.EVO_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.EVO_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #7401DF;
}
.EVO_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_RP {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #0174DF;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_RP + input{
	display: none;
}
.collapse_RP + input + div{
	display: none;
}
.collapse_RP + input:checked + div{
	display: block;
}
.RP {
	text-align: justify;
	text-justify: inter-word;
}
.RP_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #0174DF;
	padding-left: 5px;
}
.RP_collapse_similarity:hover{
	background: #c9d7e2;
}
.RP_collapse_similarity + input {
	display: none;
}
.RP_collapse_similarity + input + div{
	display: none;
}
.RP_collapse_similarity + input:checked + div{
	display: block;
}
.RP_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.RP_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #0174DF;
}
.RP_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_SIT {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #0489B1;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_SIT + input{
	display: none;
}
.collapse_SIT + input + div{
	display: none;
}
.collapse_SIT + input:checked + div{
	display: block;
}
.SIT {
	text-align: justify;
	text-justify: inter-word;
}
.SIT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #0489B1;
	padding-left: 5px;
}
.SIT_collapse_similarity:hover{
	background: #d0dfe2;
}
.SIT_collapse_similarity + input {
	display: none;
}
.SIT_collapse_similarity + input + div{
	display: none;
}
.SIT_collapse_similarity + input:checked + div{
	display: block;
}
.SIT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.SIT_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #0489B1;
}
.SIT_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_TU {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #04B486;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_TU + input{
	display: none;
}
.collapse_TU + input + div{
	display: none;
}
.collapse_TU + input:checked + div{
	display: block;
}
.TU {
	text-align: justify;
	text-justify: inter-word;
}
.TU_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #04B486;
	padding-left: 5px;
}
.TU_collapse_similarity:hover{
	background: #d0e2de;
}
.TU_collapse_similarity + input {
	display: none;
}
.TU_collapse_similarity + input + div{
	display: none;
}
.TU_collapse_similarity + input:checked + div{
	display: block;
}
.TU_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.TU_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #04B486;
}
.TU_similar > a {
	text-decoration: none;
	color: gray;
}

ul { 
	list-style-type: none;
	padding: 0;
	margin: 0;
}
ul li > a {
	text-decoration: none;
	color: gray;
}
ul li:hover, ul li:focus {
	background: #F0F0F0;
}

ul li:first-child span {
  border-left:none;  
}

</style>
<html>
<head>
<meta charset="utf-8">
<title>TF properties - NarP</title>
<meta name="theme-color" content="#ffffff">
</head>
<body>
	<div class="banner">
		<center><font color="white" size="16px" face="arial">NarP</font></center>
	</div>
	<div align="center" class="main-content">
		<div class="spacer">
	</div><br>			<label class="collapse_ACT" for="_1"><font size="5px">NarP transcriptional activity</font></label>
				<input id="_1" type="checkbox">
				<div class="ACT"><br><ul><li>The RR-bound 32P was stably retained for the A1 group RRs such as pArcA , pCpxR , pCusR , pKdpE , and pNarL but was rapidly released for the A2 group RRs , including pBaeR , pCheB , pCheY , pCreB , pDcuR , pNarP , pNtrC , pTorR , pUhpA , pYehT , and pYfhA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15522865" target="_blank">15522865</a>]<br></li>
<li>Location and functional classification of 32-regulon members A. Location Periplasm ( 2 ) Regulon members DsbC YceI CreC CutE FtsH FxsA GntYa HflX HflK HflC HtpX LipBa LspA MenA PgpA PhoQ YbeXa YbeZa YceJa YcjFa ClpB ClpP ClpX CreB DnaJ DnaK FkpB GapA GlnS GroELS GrpE HepA Hfq HolC Hsp33 HslU HslV HtpG IbpA IbpB IleS IspH Lon MiaA Mlc MutL MutM NarP NusB PhoP PrlC PyrF RecA RecJ RdgB RpoD rrnB RrmJ SdaA TopA TyrR ValS XerD YciH YfjN YrfH ZntR CreA FolP RibE ThiL YadF YafD YafE YbeD YbeY YceP YciH YciM YciS YcjX YdhQ YeaD YfjN YgaD YgbF YggW YhdN YhiQ YnfK Inner membrane ( 18 ) Cytoplasm ( 48 ) Unknown ( 25 ) B. Functional category Regulon members Metabolism CutE FolP GapA IspH LipB LspA MenA PgpA PyrF RibE SdaA ThiL YadF YafE YceJ YggW Chaperone/folding catalysts ClpB DnaJ DnaK DsbC FkpB GroELS GrpE Hsp33 HtpG IbpA IbpB Protein degradation ClpP ClpX FtsH HflX HflK HflC HslU HslV HtpX Lon PrlC DNA modification HolC MutL MutM RecA RecJ RdgB TopA XerD RNA state GlnS Hfq IleS MiaA rrnB RrmJ ValS YciH YfjN YrfH Transcription regulators CreB CreC HepA Mlc NarP NusB PhoP PhoQ RpoD TyrR ZntR Transporter YbeX YbeZ Miscellaneous FxsA GntY YciH YfjN YhiQ Unknown function CreA YafD YbeD YbeY YceI YceP YciM YciS YcjF YcjX YdhQ YeaD YgaD YgbF YhdN YnfK Only regulon members encoded on the chromosome and with a confirmed 32-dependent promoter are considered .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16818608" target="_blank">16818608</a>]<br></li>
<li>The simplest scenario would be that nitrate-triggered NarUNarP is more active than the nitrite-triggered factors , because nitrate induces a higher level of phosphorylation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7854119" target="_blank">7854119</a>]<br></li>
<li>of additional probable sites in predictionsd 3 3 7e 7 -- 4 22e 0 1 0 2 -- -- 5 0 1 -- 13 7 18 1 0 6e 1 1 1 3 Ada AraC ArcA ArgR CarP CpxR Crp CspA CynR CysB CytR DeoR DnaA FadR FhlA Fis FlhCD Fnr FruR Fur GalR GcvA GlpR IciA ( ArgP ) IclR Ihf IlvY LacI LexA Lrp MalT MarR MelR MetJ MetR Mlc ( DgsA ) ModE NagC NarL NarP NtrC OmpR OxyR PdhR PhoB PurR RhaR RhaS SoxR SoxS TorR TrpR TyrR Stem -- loopsf 6 2 0 5 2 9 1 3 4 5e 5 -- 5 5 4 6 3 4 2 -- 1 3 5 1 3 -- aThe number of genes in our study set with documented binding sites for each TF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11160901" target="_blank">11160901</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_5">
Darwin , A.J. , Tyson , K.L. , Busby , S.J.W. , and Stewart , V. ( 1997 ) Differential regulation by the homologous response-regulators NarL and NarP of Escherichia coli K-12 depends upon DNA binding site arrangement .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9484892" target="_blank">9484892</a>]<br></label>
<input id="act_similarity_5", type="checkbox">
<div class="ACT_similar">
<li>Darwin , A. , Tyson , K. , Busby , S. , and Stewart , V. ( 1997 ) Differential regulation by the homologous response-regulators NarL and NarP of Escherichia coli K-12 depends on DNA-binding site arrangement .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9484902" target="_blank">9484902</a>]<br></li>
<li>Darwin , A.J. , Tyson , K.L. , Busby , S.J.W. , and Stewart , V. ( 1997 ) Differential regulation by the homologous response-regulators NarL and NarP of Escherichia coli K-12 depends on DNA binding site arrangement .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10972841" target="_blank">10972841</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15978080" target="_blank">15978080</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15225315" target="_blank">15225315</a>]<br></li></div>
				<label class="ACT_collapse_similarity" for="act_similarity_8">
Differential regulation by the homologous response-regulators NarL and NarP of Esch-erichia coli K12 depends on DNA binding site arrangement .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9852003" target="_blank">9852003</a>]<br></label>
<input id="act_similarity_8", type="checkbox">
<div class="ACT_similar">
<li>Differential regulation by the homologous response-regulators NarL and NarP of Esch-erichia coli K-12 depends on DNA binding site arrangement .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923080" target="_blank">12923080</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11976297" target="_blank">11976297</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17720788" target="_blank">17720788</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19251855" target="_blank">19251855</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16936015" target="_blank">16936015</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15995204" target="_blank">15995204</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11844760" target="_blank">11844760</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11004182" target="_blank">11004182</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12644479" target="_blank">12644479</a>]<br></li>
<li>Differential regulation by the homologous response-regulators NarL and NarP of Esch-erichia coli K-12 depends on DNA-binding site arrangement .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9696769" target="_blank">9696769</a>]<br></li></div>
<li>Only five operons were activated more than 3-fold by NarP ; these were the previously identified nirB , nap , and nrf operons as well as yeaR and fdnH ( Table 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16377617" target="_blank">16377617</a>]<br></li>
<li>Nitrate-stimulated NarP activated expression of 14 operons ; all of those most strongly activated by NarP were part of the FNR regulon and had been characterized independ-ently .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16377617" target="_blank">16377617</a>]<br></li>
<li>Nine operons were regulated in different directions by NarXL and NarP ( five being NarXL-repressed and NarP activated ) ; three of these were also FNR-dependent .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16377617" target="_blank">16377617</a>]<br></li>
<li>Taken together , these observations suggest that the NarQ protein phosphorylates the NarP protein to a somewhat higher level than does the NarX protein , although the differences are rather subtle .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8501030" target="_blank">8501030</a>]<br></li>
<li>For the napFDAGHBC operon , Fnr pro- tein , bound at position 64.5 , acts synergistically with phos - pho-NarP protein bound at position 44.5 ( 15 , 17 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17720788" target="_blank">17720788</a>]<br></li>
<li>In a deﬁned in vitro transcription system , the T2 transcript is weakly synthesized irrespective of added Fnr protein , but its synthesis is inhibited in reactions containing phospho-NarP or phosphoNarL proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=13129959" target="_blank">13129959</a>]<br></li>
<li>These results establish the unconventional napF operon control region architecture , in which the major promoter P1 is activated by the Fnr protein bound to a site centered at 64.5 with respect to the transcription initiation site , working in conjunction with the phospho-NarP protein bound to a site centered at 44.5 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=13129959" target="_blank">13129959</a>]<br></li>
<li>scription of many other operons is known to be activated by the Fnr protein acting alone ( 12 , 26 ) , to date there are no examples of Fnr-independent transcription activation by the phosphoNarL or - NarP protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17720788" target="_blank">17720788</a>]<br></li>
<li>Target operon transcription is activated by the Fnr protein in response to anaerobiosis , and it is further activated and/or repressed by the phosphoNarL and phospho-NarP proteins , which bind to heptamer DNA sequences .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12644479" target="_blank">12644479</a>]<br></li>
<li>For the nirBDC and nrfABCDEFG operons , Fnr protein , bound near position 41.5 , activates transcription maximally only when phospho-NarL or - NarP protein is bound further upstream to block inhibition by other proteins ( 2 , 8 , 59 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17720788" target="_blank">17720788</a>]<br></li>
<li>As there was no previous convincing evidence that NarP can repress transcription of nitrate-responsive operons , it was both interesting and surprising that 37 operons were repressed by nitrate even in a narXL deletion strain relative to the narXL narP mutant , showing that these operons are repressed by NarP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16417494" target="_blank">16417494</a>]<br></li>
<li>Transcription from promoter P1 is activated by Fnr and phospho-NarP during anerobic growth with nitrate , whereas transcription from the overlapping promoter P2 occurs during aerobic-growth or during anaerobic-growth in the absence of nitrate ( 19 , 22 , 23 , 66 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19060147" target="_blank">19060147</a>]<br></li>
<li>The physiological relevance of these results is uncertain , however , since transcription initiation from promoter P1 in wild-type constructs is only detected during growth with nitrate ( 19 , 66 ) or in the presence of phospho-NarP protein ( 23 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19060147" target="_blank">19060147</a>]<br></li>
<li>Considering also the relatively weak match of this element to the 35 consensus and its proximity to the NarP ¬ FIG. 1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=13129959" target="_blank">13129959</a>]<br></li>
<li>The narP gene was also deleted from the narXL mutant to reveal the extent of regulation by phosphorylated NarP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16377617" target="_blank">16377617</a>]<br></li>
<li>The hypothesis that NsrR mediates regulation of denitriﬁcation genes in Neisseria is further supported by the observation that in N. gonorrhoeae , the norB gene is strongly induced by NO independently of FNR and NarP [ 30 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16261196" target="_blank">16261196</a>]<br></li>
<li>Although unlikely , a second possible explanation to account for the unusual regulation of the hyb operon in a narP mutant is that the NarP protein is somehow involved in the anaerobic induction of hyb expression ( see Table 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10537212" target="_blank">10537212</a>]<br></li>
<li>The che-mostat data ( Fig. 4A ) imply that bound NarP phosphate is not competent for activating fdnG transcription since fdnG-lacZ expression was not affected in the narP narL strain compared to expression in the narP narL strain ( see below ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923080" target="_blank">12923080</a>]<br></li>
<li>Coordinate Regulation of the Escherichia coli Formate Dehydrogenase fdnGHI and fdhF Genes in Response to Nitrate , Nitrite , and Formate : Roles for NarL and NarP Henian Wang and Robert P. Gunsalus * Department of M icrobiology , Immunology , and Molecular Genetics and Molecular Biology Institute , University of California , Los Angeles , California 90095 Received 28 February 2003/Accepted 4 June 2003 Escherichia coli possesses three distinct formate dehydrogenase enzymes encoded by the fdnGHI , fdhF , and fdoGHI operons .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923080" target="_blank">12923080</a>]<br></li>
<li>NarP has been reported to act as a negative regulator of the FNR-dependent class II fdnGHI promoter under low-nitrate conditions ( Wang & Gunsalus , 2003 ) , although this promoter has also been shown to be weakly activated by NarP ( Rabin & Stewart , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18227264" target="_blank">18227264</a>]<br></li>
<li>However , under low-nitrate conditions ( 1 to 4 mM added nitrate or less than 0.4 M nitrate remaining ) sufﬁcient activation of NarP occurs , and the newly formed NarP phosphate then binds at the two NarP recognition heptamers centered at positions 109 and 100 upstream of the fdhG promoter ( 11 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923080" target="_blank">12923080</a>]<br></li>
<li>This shows that the NarP protein is responsible for the residual induction observed in narL null strains .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8501030" target="_blank">8501030</a>]<br></li>
<li>Disruption of narL hardly affects nitrite induction at pnrf as NarP can mediate nitrite induction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7854119" target="_blank">7854119</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7854119" target="_blank">7854119</a>]<br></li>
<li>This contrasts with the situation found with the wild-type promoter in the narL background , where nitrate-triggered NarP is unable to repress , presumably because it occupies heptamers - 7 9 and - 7 0 in preference to the - 5 0 and - 2 2 heptamers .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7854119" target="_blank">7854119</a>]<br></li>
<li>Experiments performed in the narL background show that nitrate-triggered NarP is normally unable to repress pnrf .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7854119" target="_blank">7854119</a>]<br></li>
<li>Nevertheless , this study has provided the first definitive evidence that NarP can exert global negative control .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16377617" target="_blank">16377617</a>]<br></li>
<li>Presumably this is because of the occupation of heptamers - 5 0 and - 2 2 by NarP ; we suppose that this results from reduction in NarP affinity for heptamers - 7 0 and - 79 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7854119" target="_blank">7854119</a>]<br></li>
<li>We suggest that increasing the level of NarP phosphorylation first triggers the occupation of heptamers - 7 0 and - 7 9 , but that , even in the presence of nitrate , insufficient active NarP is generated for occupation of heptamers - 5 0 and - 2 2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7854119" target="_blank">7854119</a>]<br></li>
<li>Note that , in experiments such as this where fnr was present on a multi-copy plasmid , we consistently found higher expression during aerobic-growth than in experiments such as those in Tables 1-5 , where fnr was present in single copy ( K. L. Tyson , J. A. Cole and S. J. W. Busby , unpublished data ) In spite of the complicated differences between pnirand pnrf , it is remarkable that , in both cases , the sequence elements required for activation by NarP or NarL consist of an inverted pair of heptamers separated by two base pairs .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7854119" target="_blank">7854119</a>]<br></li>
<li>( B ) Test promoter RR Test promoter Gene product Protein function EvgA NarL NarP RcsB UhpA UvrY ydeO nirB nirB wza uhpT csrB AraC family transcription factor Nitrite reductase enzyme Regulation of acid resistance genes Reduction of nitrite to ammonia Reduction of nitrite to ammonia Capsular polysaccharide translocation Hexose-6-phosphate : phosphate antiporter Regulatory sRNA for carbon storage Nitrite reductase enzyme Outer membrane lipoprotein Major facilitator superfamily ( MFS ) transporter Small regulatory RNA ( sRNA ) Cross talk in vivo in promoter recognition between six NarL-family RRs The stage 3 cross talk in vivo was examined for a total of 30 combinations between six RRs and ﬁve representative promoters [ note that NarL and NarP recognize the same nirB promoter ] under the induction condition characteristic to each RR or in the presence and absence of over-expressed RRs .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26010043" target="_blank">26010043</a>]<br></li>
<li>Construction of arabinose-inducible expression system of RRs The coding sequences of six species of NarL-family RR ( EvgA , NarL , NarP , RcsB , UhpA , and UvrY ) were PCR-ampliﬁed using 5 - proximal and 3 - proximal primers of 0 0 each open reading frame ( for the sequence , see Table S1 0 in Supporting Information ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26010043" target="_blank">26010043</a>]<br></li>
<li>In the presence of AcP , however , the binding afﬁnity to the cognate promoters signiﬁcantly increased for EvgA , NarL , UhpA , and UvrY , but little enhancement was observed for NarP and RcsB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26010043" target="_blank">26010043</a>]<br></li>
<li>As an initial attempt to identify the stage 3 cross talk at the step of target promoter recognition by RR , we analyzed in this study the cross-recognition of target promoters by six NarL-family RRs , EvgA , NarL , NarP , RcsB , UhpA , and UvrY .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26010043" target="_blank">26010043</a>]<br></li>
<li>Multifactor promoters Among the NarL-family RR-dependent promoters analyzed in this study , the ydeP promoter was found to be activated by ﬁve RRs , EvgA , NarL , NarP , RcsB , and UvrY ( Fig. 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26010043" target="_blank">26010043</a>]<br></li>
<li>These insertions resulted in the H74 .5 , H78 .5 , H84 .5 , H90 .5 , H94 .5 and H100 .5 derivatives of the pnir 7150 fragment , in which the centre of the DNA site for NarL/NarP was moved upstream to positions ¹ 74.5 , ¹ 78.5 , ¹ 84.5 , ¹ 90.5 , ¹ 94.5 and ¹ 100.5 respectively ( illustrated in Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9484902" target="_blank">9484902</a>]<br></li>
<li>Another example is the NarL / NarP - and FNR-stimulated expression of nirB and nrf genes , which are repressed by Fis and IHF ( Browning et al. , 2002 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15066032" target="_blank">15066032</a>]<br></li>
<li>Transcription control of acs -- yjcH -- actP is complex and the intergenic region between acs and nrfA is bound by CRP , FNR , IHF , FIS and NarL or NarP and is folded into an ordered nucleoprotein structure that permits the two divergent promoters to be regulated independently ( Browning et al. 2002 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17273855" target="_blank">17273855</a>]<br></li>
<li>These problems are especially acute for studies of global regulators such as FNR at promoters where FNR regulation is co-de-pendent upon another factor , for example NarL or NarP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16377617" target="_blank">16377617</a>]<br></li>
<li>Genes and regulators responding to O2 and other acceptors FNR and ArcA , as shown earlier for NarL and NarP , control different sets of target genes Žreviewed in w4 ,5,9,11,12,44 x. FNR acts as the major regulator of anaerobic metabolism and ArcA as the regulator of aerobic metabolism .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9230919" target="_blank">9230919</a>]<br></li>
<li>The chemostat studies also revealed new roles for the NarL and NarP proteins in the activation and/or repression of the two operons in contrast to prior conclusions derived from batch culture studies .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11004182" target="_blank">11004182</a>]<br></li>
<li>NarP is not essential for either the activation or repression process .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11004182" target="_blank">11004182</a>]<br></li>
<li>NarP phosphate is unable to perform activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923080" target="_blank">12923080</a>]<br></li>
<li>Unless NarP , when acting as a repressor , can also recognize and occupy individual heptamer sites , as has been demonstrated for NarL , this argues against competition between the repressors as being the explanation for the enhanced repression of hyb in a narP mutant background .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10537212" target="_blank">10537212</a>]<br></li>
<li>A further striking result is that the regulation of six of the operons implicated in the management of reactive nitrogen species , hcp-hcr , ygbA , ygiS , yeaR-yoaG , yibHI and ytfE , is only partially dependent on NarL or NarP : nitrate induction from these promoters was detected even in the narXL narP deletion strain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16417494" target="_blank">16417494</a>]<br></li>
<li>Although the frdA , dmsA , aspA , ansB , and hyb promoters were significantly repressed only by nitrate-activated NarL , the hycA and dcuB promoters were also repressed by nitrate-activated NarP , as predicted in Table 4 , part E. Eight of the 37 significantly repressed operons gave repression ratios greater than 4-fold ( Table 5 , amount of transcript in the narXL narP strain relative to that of the narXL strain ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16377617" target="_blank">16377617</a>]<br></li>
<li>The hyaA promoter has been reported to be repressed by both NarL and NarP ; the other operons in this group possibly share this pattern of regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16377617" target="_blank">16377617</a>]<br></li>
<li>Similar differences were observed in the regulation by nitrate of genes for hydrogenase synthesis : the hyaABCDEF and hyc ¬ ABCDEFGH operons were repressed by both NarL and NarP ; the hyb operon was repressed only by NarL ( Figures 1C and 1D ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16417494" target="_blank">16417494</a>]<br></li>
<li>In contrast with the frdABCD , aspA and ansB operons that are repressed only by NarL , the dcuB-fumB operon was among 37 operons that are repressed by NarP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16417494" target="_blank">16417494</a>]<br></li>
<li>Although the frdABCD , aspA and ansB operons were repressed only by NarL , the dcuB-fumB operon was strongly repressed by both NarL and NarP ( Figures 1A and 1B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16417494" target="_blank">16417494</a>]<br></li>
<li>As Fnr does not affect the expression of NarP or NarL and vice versa ( Darwin and Stewart , 1995 ; K. Tyson and A. Bell , unpublished results ) , the co-depen-dence of the nir promoter on Fnr and NarL/NarP is likely to be caused by some sort of direct or indirect interaction between Fnr and NarL/NarP bound at their cognate targets .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9484902" target="_blank">9484902</a>]<br></li>
<li>Thus , at the nir operon promoter , our present results argue that the major role of NarL and NarP is to counteract proteins that are suppressing Fnr activity , and , although we can not prove the point conclusively , any direct activatory role is likely to be very minor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9484902" target="_blank">9484902</a>]<br></li>
<li>The search for proteins with homology to the putative BglJ protein yielded other members of this family including FimZ , UvrC , MalT , RcsA , NarP , and NarL ( ALTSCHUL et al. 1990 ; STOUTet al. 1991 ; SWENSOaNnd CLEGG1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8725214" target="_blank">8725214</a>]<br></li>
<li>On the other hand , nine species of RR ( AtoC , CheY , CusR , HydG , KdpE , NarL , NarP , NtrC , and YfhA ) were phosphoryl-ated by non-cognate HKs besides their cognate HKs .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15522865" target="_blank">15522865</a>]<br></li>
<li>The level of phosphorylated RRs was less than 1 % for CreB , HydG , YedW , NarL , CheB , OmpR , DcuR , CpxR , YehT , and NarP ( in decreasing order ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15522865" target="_blank">15522865</a>]<br></li>
<li>However , significant levels of the cross-talk in trans-phosphorylation were observed for a group of RRs ( NarP , CpxR , NarL , and HydG ) , which showed the low level of trans-phosphorylation by the cognate HKs .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15522865" target="_blank">15522865</a>]<br></li>
<li>In previous work , we found that NarL or NarP activates the nir promoter by binding to a site centred at position -69.5 and counteracting the repressive effects of IHF and Fis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15225315" target="_blank">15225315</a>]<br></li>
<li>The binding of NarL or NarP to the 7-2-7 element , centred at position -74.5 , displaces IHF and activates the promoter provided that Fis is not present ( see text ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15978080" target="_blank">15978080</a>]<br></li>
<li>The binding of NarL or NarP to the 7-2-7 element , centred at position -69.5 , counteracts the effects of IHF and Fis , and activates the promoter provided that Cra is not present .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15978080" target="_blank">15978080</a>]<br></li>
<li>Our observation that NarL could displace IHF from its site at the nrf promoter suggested that NarL/NarP activates transcription by counteracting IHF-mediated repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15978080" target="_blank">15978080</a>]<br></li>
<li>Once phosphorylated , NarL or NarP binds to the same target site centered at position 69.5 ( Fig. 1A ) and activates transcription by displacing IHF from IHF I , thereby counteracting the repression by IHF and Fis from upstream sites ( 3 , 25 , 28 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18757534" target="_blank">18757534</a>]<br></li>
<li>1 NarL and NarP counteract the negative effects of Fis and IHF but not H-NS Wu et al. ( 1998 ) proposed that NarL and NarP function at pnir , in response to nitrite , by counteracting the repressive effects of upstream factors .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10972841" target="_blank">10972841</a>]<br></li>
<li>Thus , it is possible that NarL and NarP can displace IHF and that the consequence of this is to reposition pnir upstream sequences ( Fig. 8B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10972841" target="_blank">10972841</a>]<br></li>
<li>Thus , our measurements reﬂect the ability of FNR to activate pnir without the aid of NarL or NarP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18757534" target="_blank">18757534</a>]<br></li>
<li>Thus , when repression by both Fis and IHF is disrupted , nitrite-triggered NarL or NarP is no longer required for optimal activation of pnir .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10972841" target="_blank">10972841</a>]<br></li>
<li>Our results clearly show that H-NS represses pnir and that the action of H-NS is independent of the action of Fis , IHF and NarL/NarP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10972841" target="_blank">10972841</a>]<br></li>
<li>Thus , we conclude that IHF and Fis can repress pnir in the absence of H-NS ( Table 2 ) , and that repression by H-NS can not be reversed by the action of NarL/NarP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10972841" target="_blank">10972841</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_77">
NarL and NarP can relieve both IHF - and Fis-mediated repression , but are unable to counteract H-NS - mediated repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10972841" target="_blank">10972841</a>]<br></label>
<input id="act_similarity_77", type="checkbox">
<div class="ACT_similar">
<li>NarL and NarP counteract Fis - and IHF-mediated repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10972841" target="_blank">10972841</a>]<br></li></div>
<li>The resulting plasmids were then transformed into the Dlac narL narP strain , JCB3884 ( Table 1 ) , in order to monitor the effects of IHF in the absence of activation by NarL or NarP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15225315" target="_blank">15225315</a>]<br></li>
<li>The effects of IHF are observed only in the absence of NarL/NarP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16936015" target="_blank">16936015</a>]<br></li>
<li>Binding of NarL/NarP appears to insulate the nrf promoter from the effects of IHF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16936015" target="_blank">16936015</a>]<br></li>
<li>This supports the proposal that NarL and NarP function by counteracting the repressive effects of IHF and Fis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10972841" target="_blank">10972841</a>]<br></li>
<li>However , we propose that , although NarP and NarL bind as dimers , transcription activation requires activity in just one of the subunits of the dimer ( this has been shown for both CRP and FNR : Zhou etal .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7854119" target="_blank">7854119</a>]<br></li>
<li>Thus , NarL and NarP protect FNR-dependent transcription at pnrf from both the negative and positive effects of IHF and appear to function like insulator proteins rather than conventional transcription activators , which stimulate promoter activity by recruitment of RNA polymerase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16936015" target="_blank">16936015</a>]<br></li>
<li>NarL and NarP activate FNR-dependent transcription by counteracting the repressive effects of IHF and Fis ( Fig. 1B ) , and thus the role of IHF and Fis is to set the co-dependence of pnir on both FNR and NarL/NarP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15225315" target="_blank">15225315</a>]<br></li>
<li>It was suggested that FNR-dependent transcription activation at pnir is suppressed by these upstream-bound proteins , and that the role of NarL and NarP is simply to counteract this suppression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10972841" target="_blank">10972841</a>]<br></li>
<li>Thus , at pnir , Wu et al. ( 1998 ) found that FNR-dependent transcription activation is repressed by upstream-bound factors , and that NarL and NarP activate pnir by counteracting these repressing factors .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10972841" target="_blank">10972841</a>]<br></li>
<li>The ratio column indicates the inhibition in Fnr-dependent activity in the absence of NarL and NarP : this inhibition is lifted by removal of the upstream sequences .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9484902" target="_blank">9484902</a>]<br></li>
<li>The ratio column indicates the inhibition in Fnr-dependent activity in the absence of NarL and NarP caused by Fis protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9484902" target="_blank">9484902</a>]<br></li>
<li>Thus , in the absence of NarL and NarP , Fis inhibits Fnr-dependent activation of the nir promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9484902" target="_blank">9484902</a>]<br></li>
<li>Thus , deletions and various point mutations in the upstream sequences lead to an increase in Fnr-dependent nir promoter activity in the absence of NarL and NarP , reducing the co-dependence of the nir promoter on both Fnr and NarL/NarP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9484902" target="_blank">9484902</a>]<br></li>
<li>Note that our observation that NarL can increase FNR-dependent open complex formation in the absence of IHF argues that it , and perhaps NarP , behave as genuine co-activators at the nrf promoter as well as anti-repressors .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15978080" target="_blank">15978080</a>]<br></li>
<li>Hence , Fis protein is a major factor responsible for catabolite repression at the nrf promoter , and Fis can override activation by FNR and NarL or NarP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15978080" target="_blank">15978080</a>]<br></li>
<li>The binding of NarL or NarP to the nir promoter displaces IHF and overrides the repression of FNR-dependent activation by IHF and Fis ( Browning et al. , 2004 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15978080" target="_blank">15978080</a>]<br></li>
<li>Nevertheless , our results strongly suggest that another factor , whose nature remains to be elucidated , is playing the major control , although in competition with FNR and NarL/NarP , and its action is mainly triggered by the presence of nitrite .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12586421" target="_blank">12586421</a>]<br></li>
<li>Thus , the results suggest that NarL and NarP proteins are competing with the main regulator factor of # rd , which is activated by nitrite ( or by a derivative product ) , and may bind to the nucleotide region in the vicinity of the putative narL/narP binding site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12586421" target="_blank">12586421</a>]<br></li>
<li>A plausible explanation for these observations is that mutation of the +6 or +15 heptamers inhibits phospho-NarP binding , allowing nitrite-activated phospho-NarL to-bind at the remaining unaltered heptamer ( either at the +6 heptamer , where NarL acts as an effective repressor , or at the +15 heptamer , where NarL does not contribute significantly to repression ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18227264" target="_blank">18227264</a>]<br></li>
<li>However , mutation of the 216 heptamer enhanced repression in the presence of nitrite , suggesting that NarL binding at 216 antagonizes NarP action at the 7-2-7 site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18227264" target="_blank">18227264</a>]<br></li>
<li>The enhanced repression observed in the presence of nitrite when the 216 heptamer was mutated suggests that the presence of NarL at the 216 site inhibits the action of NarP at the 7-2-7 site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18227264" target="_blank">18227264</a>]<br></li>
<li>These sites , which result in missense substitutions ( NarL , Lys-165 to Glu ; NarP , Lys-125 to Gly and His-163 to Glu ) , were designed for a separate study to analyse properties of NarL -- NarP chimerae .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20634237" target="_blank">20634237</a>]<br></li>
<li>Although phospho-NarL can also bind to heptamers that are deployed in other conﬁgurations ( 2 , 4 , 7 , 17 ) , we focus here on the 7-2-7 heptamer pairs that comprise the sole or principal NarL and NarP protein binding sites in their respective control regions ( Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12644479" target="_blank">12644479</a>]<br></li>
<li>Regardless of the mechanism , the NarL protein is unable to activate the napF promoter when bound to the 244.5 site and can be thought of as a natural positive-control mutant ( relative to NarP function ) in this context .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9696769" target="_blank">9696769</a>]<br></li>
<li>Phosphorylation increases the afﬁnity of the NarL and NarP proteins for their speciﬁc DNA binding sites , from whence they activate and repress target operon expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12644479" target="_blank">12644479</a>]<br></li>
<li>It is yet unclear if ModE interactions with molybdate act primarily to modulate DNA binding or , alternatively , to affect ModE interactions with other proteins involved in napF gene expression , including Fnr , NarP , NarL , and RNAP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12029041" target="_blank">12029041</a>]<br></li>
<li>This competitive binding presumably leads to antagonization of NarP-dependent activation by the NarL protein , since only NarP and Fnr together cause maximum napF expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9696769" target="_blank">9696769</a>]<br></li>
<li>NarL binding to the site results in reduced napF expression relative to NarP binding , including a reduced contribution to activation by the Fnr protein ( related to CRP ) from its DNA site upstream of the NarP/NarL site ( 9 , 10 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19854903" target="_blank">19854903</a>]<br></li>
<li>By contrast , both NarL and NarP are competent to activate transcription activation from the more upstream binding sites of other operon control regions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9696769" target="_blank">9696769</a>]<br></li>
<li>The inability of NarL to activate napF operon expression led us to hypothesize that NarL is deﬁcient in the mechanism by which NarP activates transcription from the 244.5 binding site ( 9 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9696769" target="_blank">9696769</a>]<br></li>
<li>The NarL protein does not signiﬁcantly activate napF operon expression in the presence of Fnr , whereas the NarP protein does .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9696769" target="_blank">9696769</a>]<br></li>
<li>However , phosphorylated NarP protein bound at the same heptamer sites at positions 109 and 100 competes with NarL to antagonize its ability to stimulate transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923080" target="_blank">12923080</a>]<br></li>
<li>When the phosphorylated MBP-NarP and MBP-NarL proteins were spun through a Sephadex G-25 column to remove acetyl phosphate , 20 to 40 % of the protein was lost .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9696769" target="_blank">9696769</a>]<br></li>
<li>This variability meant that slightly different concentrations of phosphorylated MBP-NarP or MBP-NarL protein were used in duplicate experiments .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9696769" target="_blank">9696769</a>]<br></li>
<li>This indicates that phospho-NarL protein is sufﬁcient for normal ( yeaR-lacZ ) induction but that phospho-NarP protein also can contribute , at least in the absence of phosphoNarL protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17720788" target="_blank">17720788</a>]<br></li>
<li>This suggests that low levels of phosphorylated NarP and NarL proteins may be present during growth in the absence of an inducer and are able to stimulate signiﬁcant levels of transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=13129959" target="_blank">13129959</a>]<br></li>
<li>This suggests that the low levels of phospho-NarP ( and phos-pho-NarL ) are present even with no added inducer .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=13129959" target="_blank">13129959</a>]<br></li>
<li>in the case of pnir , single substitutions substantially block NarP - or NarL-dependent activation , whilst at pnrf the effects are smaller .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7854119" target="_blank">7854119</a>]<br></li>
<li>This activation is solely dependent on the NarP protein , unlike all other operons studied , for which activation is dependent on NarL only or on either NarL or NarP ( 8 , 26 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9696769" target="_blank">9696769</a>]<br></li>
<li>We also reproduced the regulation of the napF promoter by Fnr , NarP , and NarL in vitro and conﬁrmed and extended our previous conclusions about napF operon regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9696769" target="_blank">9696769</a>]<br></li>
<li>Phosphorylated NarP protein activates transcription from the napF promoter ; transcription activation is antagonized by nitrate-activated NarL [ 10 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9716493" target="_blank">9716493</a>]<br></li>
<li>Interestingly , NarL and NarP operate differently at the napF promoter , where NarL blocks activation of NarP ( 30 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923080" target="_blank">12923080</a>]<br></li>
<li>These in vitro transcription experiments demonstrate that , in the presence of Fnr , the NarP protein causes efﬁcient transcription activation whereas the NarL protein does not .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9696769" target="_blank">9696769</a>]<br></li>
<li>As observed in vivo , this activation by Fnr and NarP is antagonized by the addition of NarL in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9696769" target="_blank">9696769</a>]<br></li>
<li>Many questions raised in part by this structure remain unanswered , and must be the subject of future investigations : ( 1 ) How are NarL and NarP activated for binding to DNA by phosphorylation ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8780507" target="_blank">8780507</a>]<br></li>
<li>NarP phosphate appears to be a mild antagonist of NarL phosphate binding to DNA when formate levels are low ( Fig. 3A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923080" target="_blank">12923080</a>]<br></li>
<li>This conﬁrms that the phospho-NarL and - NarP protein binding site , identiﬁed by sequence inspection , is critical for regulation by these proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17720788" target="_blank">17720788</a>]<br></li>
<li>Substitutions were designed to eliminate binding to the half-site sequence , to leave the promoter 35 element and the phospho-NarL and - NarP protein binding site intact , and to maintain the overall G C composition .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17720788" target="_blank">17720788</a>]<br></li>
<li>Cells were grown anaerobically in ni-trate-free minimal medium in order to `` sample '' bound proteins in the absence of NarL or NarP binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20472787" target="_blank">20472787</a>]<br></li>
<li>This confirms that the NarP receiver inhibits binding , similar to NarL , and that NarP-CTD binds target sites in vitro as avidly as NarL-CTD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20634237" target="_blank">20634237</a>]<br></li>
<li>Here we show that pnrf activity is stimulated by IHF binding to the IHF III site , but only in the absence of NarL/NarP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16936015" target="_blank">16936015</a>]<br></li>
<li>This indicates that NarL/NarP can compensate for the loss of IHF III and that the effect of IHF III is only observed in the absence of NarL/NarP binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16936015" target="_blank">16936015</a>]<br></li>
<li>However , by competing with NarP for the 244.5 binding site , NarL antagonizes NarP-dependent activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9696769" target="_blank">9696769</a>]<br></li>
<li>This activation is dependent on the face of the DNA helix , independent of IHF binding at other locations , and found only when NarL/NarP are not bound at position 74.5 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16936015" target="_blank">16936015</a>]<br></li>
<li>This suppression is reversed by the binding of NarL or NarP at position 74.5 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16936015" target="_blank">16936015</a>]<br></li>
<li>Acetyl phosphate at 100 mM was 21 added for phosphorylation of His - NarL and His - NarP ; control 6 6 assays established that this concentration did not influence the EMSA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20634237" target="_blank">20634237</a>]<br></li>
<li>In the nar regulatory system , it is ultimately the concentration of active ( phosphorylated ) NarL and NarP which determines the level of expression of target genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7601854" target="_blank">7601854</a>]<br></li>
<li>Thus , it appears that NarL and NarP adopt overlapping mechanisms to inhibit ydhY -- T expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18227264" target="_blank">18227264</a>]<br></li>
<li>Hence , evolution appears to have adjusted the binding of IHF to the two targets in order to set the levels of anaerobic nir operon expression in the absence of NarL or NarP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18757534" target="_blank">18757534</a>]<br></li>
<li>In the absence of NarP , NarL binding is unimpeded and NarL more effectively represses hya expression compared with NarP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10537212" target="_blank">10537212</a>]<br></li>
<li>Therefore , these are good candidates in this group for repression by NarL but activation by NarP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16377617" target="_blank">16377617</a>]<br></li>
<li>Nitrate repression of the hya operon was mediated by both NarL and NarP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10537212" target="_blank">10537212</a>]<br></li>
<li>Clearly , in vitro studies with puriﬁed NarL and NarP proteins will be required to resolve this question .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10537212" target="_blank">10537212</a>]<br></li>
<li>The role of this complex in repressing pnir , and its disruption by NarL and NarP , are discussed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10972841" target="_blank">10972841</a>]<br></li>
<li>This may explain why some NarL-sensitive promoters do not respond to NarP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7854119" target="_blank">7854119</a>]<br></li>
<li>This group includes well characterized promoters that are activated by NarL but not by NarP , such as narK and narG ( 7 , 23 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16377617" target="_blank">16377617</a>]<br></li>
<li>Furthermore , in the narLþ narP þ background , NarL and NarP must counteract this suppression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9484902" target="_blank">9484902</a>]<br></li>
<li>NarL and NarP are represented by circled L and circled P , respectively , while the phosphorylated forms are represented by the same letters boxed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8501030" target="_blank">8501030</a>]<br></li>
<li>FNR , NarL and NarP regulation of # rd / # rd-red transcription 3.6 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12586421" target="_blank">12586421</a>]<br></li>
<li>, ArcA and FNR , or NarL and NarP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9230919" target="_blank">9230919</a>]<br></li>
<li>Reguiation of pnrf by NarL and NarP Figure 2 illustrates the # .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7854119" target="_blank">7854119</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7854119" target="_blank">7854119</a>]<br></li>
<li>Residues underlined in NarL and NarP sequences are identical in both .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9521685" target="_blank">9521685</a>]<br></li>
<li>( c ) Binding of NarL and NarP to the ydhY promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18227264" target="_blank">18227264</a>]<br></li>
<li>This inhibition is counteracted by NarL or NarP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9484902" target="_blank">9484902</a>]<br></li>
<li>NarL and NarP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20634237" target="_blank">20634237</a>]<br></li>
<li>Binding of NarL or NarP counteracts this inhibition .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9484902" target="_blank">9484902</a>]<br></li>
<li>Expression of some operons is controlled by both NarL and NarP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7601854" target="_blank">7601854</a>]<br></li>
<li>is repressed by NarL and activated by NarP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9230919" target="_blank">9230919</a>]<br></li>
<li>( 3 ) How do NarL and NarP bind to DNA : as monomers , normal symmetric dimers , head-to-tail dimers , or even possibly heterodimers under some circumstances ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8780507" target="_blank">8780507</a>]<br></li>
<li>We conciude that NarL and NarP bind to the same hepta-mer sequences , but that the affinities for the two factors vary from site to site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7854119" target="_blank">7854119</a>]<br></li>
<li>In addition NarL , but not NarP , can also bind to heptamers in other arrangements ( 10 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9696769" target="_blank">9696769</a>]<br></li>
<li>, 1993 ) , the simplest explanation being that both NarL and NarP bind to the two NarL heptamers .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7854119" target="_blank">7854119</a>]<br></li>
<li>Both NarL and NarP bind to heptamers organized as inverted repeats with 2-bp spacing .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9696769" target="_blank">9696769</a>]<br></li>
<li>Similarly , the homologous NarL and NarP proteins of the nitrate regulon bind to the same sites but regulate differentially ( Darwin et al. , 1997 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9484892" target="_blank">9484892</a>]<br></li>
<li>NarL is the more proﬁcient activator under low-nitrate conditions , where NarP antagonizes NarL binding ( Fig. 4A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11004182" target="_blank">11004182</a>]<br></li>
<li>Note that NarP is also activated in response to nitrate and can bind to 7-2-7 NarL-binding targets [ 4,24 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19245365" target="_blank">19245365</a>]<br></li>
<li>Under these conditions NarP also antagonizes the superior activator response by NarL-phosphate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11004182" target="_blank">11004182</a>]<br></li>
<li>and a cytoplasmic response-regulator ŽArcA , NarL and NarP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9230919" target="_blank">9230919</a>]<br></li>
<li>Phosphorylated NarL and NarP bind to speciﬁc heptamer sequences within target promoters , and either up - or downregulate transcription initiation depending on the promoter organization .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15225315" target="_blank">15225315</a>]<br></li>
<li>Phosphorylated NarL and NarP then bind to specific heptamer sequences at target promoters , and either up - or downregulate transcription initiation at these promoters ( for examples , see Tyson et al. , 1994 ; Darwin et al. , 1997 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10972841" target="_blank">10972841</a>]<br></li>
<li>Very little is known about how NarL and NarP repress transcription , and it is often assumed that they simply block access of RNAP to target promoters ( 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19251855" target="_blank">19251855</a>]<br></li>
<li>This demonstrated that either NarL or NarP is required to activate transcription from the hcp promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15667305" target="_blank">15667305</a>]<br></li>
<li>Upon activation , NarL and NarP bind to identical heptamer sequences and , at many target promoters , two heptamers are found as a ` tail-to-tail ' inverted repeat ( often referred to as the 7-2-7 arrangement ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20533909" target="_blank">20533909</a>]<br></li>
<li>The NarP regulon overlaps the NarL regulon because both NarL and NarP can bind to a NarP site , which is a 7-2-7 inverted repeat sequence where the seven conserved bases form the NarL heptamer [ 10 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16417494" target="_blank">16417494</a>]<br></li>
<li>The effects of point mutations suggest that NarL and NarP both bind to the same target , which is a pair of heptamer sequences organized as an inverted repeat , centred 69V ^ bp upstream of the transcript startpoint .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7854119" target="_blank">7854119</a>]<br></li>
<li>We suggest that both NarL and NarP recognize the same heptamer sequence , but that there is a set of degenerate sequences that , though still recognized by NarL , are bound very poorly by NarP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7854119" target="_blank">7854119</a>]<br></li>
<li>Taken together , these results suggest that NarL and NarP bind to the same target sequences at the nir promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7854119" target="_blank">7854119</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7854119" target="_blank">7854119</a>]<br></li>
<li>In previous work ( 3 -- 6 ) , we showed that the DNA target for FNR at pnrf is centered at position 41.5 ( i.e. , between 41 and 42 bp upstream from the transcript start point ) , that anaerobically induced FNR-dependent activation of pnrf is downregulated by the binding of the nucleoid-associated protein IHF to a target ( IHF I ) centered at position 54 , and that this repression is relieved by the binding of either NarL or NarP to a DNA target centered at position 74.5 ( Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20472787" target="_blank">20472787</a>]<br></li>
<li>This results from its dependence on FNR that binds to a site at position -41.5 , and NarL or NarP that bind to a 7-2-7 element centred at position -69.5 ( Fig. 10 ; Jayaraman et al. , 1989 ; Tyson et al. , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15978080" target="_blank">15978080</a>]<br></li>
<li>Deletions and mutations in the nir promoter upstream sequences The simplest model to account for the results in Fig. 2 is to suppose that one or more factors other than NarL and NarP bind to DNA elements upstream of the NsiI site at the nir promoter and can up or downregulate Fnr-depen-dent transcription activation , according to their precise location .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9484902" target="_blank">9484902</a>]<br></li>
<li>Activation by NarL or NarP occurs when one or both of these proteins bind upstream of an Fnr-binding site centered at 41.5 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9696769" target="_blank">9696769</a>]<br></li>
<li>Fnr binds to a single DNA site centred between basepairs 41 and 42 upstream from the nir transcript start , whereas NarL and NarP bind to a site upstream , centred between basepairs 69 and 70 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9484902" target="_blank">9484902</a>]<br></li>
<li>In this context , only NarP is competent to activate transcription from this 244.5 binding site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9696769" target="_blank">9696769</a>]<br></li>
<li>It is possible that the binding of NarP at position 244.5 allows Fnr to activate efﬁciently from this nonpermissive position .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9696769" target="_blank">9696769</a>]<br></li>
<li>The explanation for this is dependent on the mechanism by which NarP activates napF transcription , as discussed above .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9696769" target="_blank">9696769</a>]<br></li>
<li>In the presence of MBP-NarP ( 7.5 mM ) , the addition of 0.5 mM Fnr ( D154A ) increased the amount of napF transcript by approximately two - to threefold .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9696769" target="_blank">9696769</a>]<br></li>
<li>In the presence of both Fnr ( D154A ) and MBP-NarP there was a sevenfold increase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9696769" target="_blank">9696769</a>]<br></li>
<li>This suggests that Fnr and NarP may act synergistically to activate napF operon expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9696769" target="_blank">9696769</a>]<br></li>
<li>Phospho-NarL and - NarP activate transcription of the narG and napF operons , respectively , whereas phospho-NarL antagonizes napF operon transcription ( 18 , 22 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19060147" target="_blank">19060147</a>]<br></li>
<li>It is possible that NarP and Fnr may activate napF transcription synergistically whereas Fnr and NarL can not , as discussed below .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9696769" target="_blank">9696769</a>]<br></li>
<li>Do Fnr and NarP activate napF transcription synergistically ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9696769" target="_blank">9696769</a>]<br></li>
<li>A 20 nM concentration of supercoiled pVJS2111 plasmid DNA was preincubated in reaction buffer for 20 min with Fnr ( D154A ) , phosphorylated MBP-NarP , or phosphorylated MBP-NarL proteins in a total volume of 20 ml .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9696769" target="_blank">9696769</a>]<br></li>
<li>( A ) Effect of phosphorylated MBP-NarP concentration on napF transcription in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9696769" target="_blank">9696769</a>]<br></li>
<li>In the presence of either phospho-MBP-NarP or phospho-MBP-NarL fusion proteins alone , there was a slight increase in napF transcription ( Fig. 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9696769" target="_blank">9696769</a>]<br></li>
<li>Interestingly , in the presence of either phospho-MBP-NarP or phospho-MBP-NarL fusion proteins the transcript was slightly shorter than in their absence ( Fig. 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9696769" target="_blank">9696769</a>]<br></li>
<li>It appeared that the in vitro napF transcription start site in the presence of either RNA polymerase alone or Fnr ( D154A ) was approximately three nucleotides upstream relative to the site in the presence of MBP-NarL or MBP-NarP ( Fig. 3 and 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9696769" target="_blank">9696769</a>]<br></li>
<li>This may indicate that the napF transcription start site changes in the presence of MBP-NarP or MBP-NarL .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9696769" target="_blank">9696769</a>]<br></li>
<li>This prediction was tested in vitro by investigating the effect of increasing amounts of MBP-NarL on napF transcription in the presence of Fnr ( D154A ) and MBP-NarP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9696769" target="_blank">9696769</a>]<br></li>
<li>In this class , KdpD/KdpE , NarP and NtrB/NtrC are known to be involved in response to turgor pressure , nitrate and nitrogen availability respectively ( Laimins et al. , 1981 ; Darwin and Stewart , 1995 ; Zimmer et al. , 2000 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12366850" target="_blank">12366850</a>]<br></li>
<li>Whether the NarP phosphate bound to DNA is unable to productively interact with FNR or RNA polymerase to induce fdnG gene expression is not known .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923080" target="_blank">12923080</a>]<br></li>
<li>In contrast to NarP , when NarL phosphate is bound to DNA , it can successfully interact with FNR and/or RNA polymerase to switch on fdnG gene expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923080" target="_blank">12923080</a>]<br></li>
<li>JOURNAL OF BACTERIOLOGY , July 1995 , p. 3647 -- / 95 / $ 04.0010 Copyright q 1995 , American Society for Microbiology Repression of the Escherichia coli pﬂ Operon Is M by the Dual Sensors NarQ and NarX and the Dual Regulators NarL and NarP Nitrate ediated MANUELA KAISER AND GARY SAWERS * Lehrstuhl für Mikrobiologie , Universität München , 80638 Munich , Germany Received 3 October 1994/Accepted 23 January 1995 The pﬂ operon is expressed at high levels anaerobically .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7601827" target="_blank">7601827</a>]<br></li>
<li>175 , No. 11 Dual Response Regulators ( NarL and NarP ) Interact with Dual Sensors ( NarX and NarQ ) To Control Nitrate - and Nitrite-Regulated Gene Expression in Eschenichia coli K-12 ROSS S. RABIN ' AND VALLEY STEWART1 ,2 * Sections of Microbiology ' * and Genetics and Development , 2 Cornell University , Ithaca , New York 14853-8101 Received 26 January 1993/Accepted 17 March 1993 Two sensor proteins , NarX and NarQ , mediate nitrate regulation of anaerobic respiratory gene expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8501030" target="_blank">8501030</a>]<br></li>
<li>Synthetic lac Operator Substitutions for Studying the Nitrate - and Nitrite-Responsive NarX-NarL and NarQ-NarP Two-Component Regulatory Systems of Escherichia coli K-12 Valley Stewart * and Peggy J. Bledsoe Section of M icrobiology , University of California , Davis , California 95616-8665 Received 19 November 2002/Accepted 17 January 2003 The NarX and NarQ sensor-histidine kinases control phosphorylation of the NarL and NarP response-regulators in response to the respiratory oxidants nitrate and nitrite .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12644479" target="_blank">12644479</a>]<br></li>
<li>Microbiology ( 2010 ) , 156 , 2933 -- 2943 DOI 10.1099 / mic.0.040469-0 Functional roles for the GerE-family carboxyl - terminal domains of nitrate response-regulators NarL and NarP of Escherichia coli K-12 Alice V. Lin13 and Valley Stewart1 ,2 Correspondence Valley Stewart 1 Biochemistry and Molecular Biology Graduate Group , University of California , Davis , CA 95616-8665 , USA vjstewart@ucdavis.edu 2 Department of Microbiology , University of California , Davis , CA 95616-8665 , USA NarL and NarP are paralogous response-regulators that control anaerobic gene expression in response to the favoured electron acceptors nitrate and nitrite .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20634237" target="_blank">20634237</a>]<br></li>
<li>Rabin , R. , and Stewart , V. ( 1993 ) Duai response reguiators ( NarL and NarP ) interact with dual sensors ( NarX and NarQ ) to controi nitrate - and nitrite-reguiated gene expression in Escherichia coii K-12 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7854119" target="_blank">7854119</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_205">
Rabin RS & Stewart V ( 1993 ) Dual response-regulators ( NarL and NarP ) interact with dual sensors ( NarX and NarQ ) to control nitrate - and nitrite-regulated gene expression in Escherichia coli K-12 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22029434" target="_blank">22029434</a>]<br></label>
<input id="act_similarity_205", type="checkbox">
<div class="ACT_similar">
<li>Rabin , R.S. , and Stewart , V. ( 1993 ) Dual response-regulators ( NarL and NarP ) interact with dual sensors ( NarX and NarQ ) to control nitrate - and nitrite-regulated gene expression in Escherichia coli K-12 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11967083" target="_blank">11967083</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9593308" target="_blank">9593308</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8809780" target="_blank">8809780</a>]<br></li>
<li>Dual response-regulators ( NarL and NarP ) interact with dual sensors ( NarX and NarQ ) t control nitrate - and nitrite-regulated gene expression in Esche-richia coli K-12 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8071220" target="_blank">8071220</a>]<br></li>
<li>Dual response-regulators ( NarL and NarP ) interact with dual sensors ( NarX and NarQ ) to control nitrate - and nitrite-regulated gene expression in Escherichia coli K-12 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923080" target="_blank">12923080</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17720788" target="_blank">17720788</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12029041" target="_blank">12029041</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7601827" target="_blank">7601827</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9335308" target="_blank">9335308</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17449618" target="_blank">17449618</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10206709" target="_blank">10206709</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626281" target="_blank">8626281</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8808940" target="_blank">8808940</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9696769" target="_blank">9696769</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20233924" target="_blank">20233924</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11739766" target="_blank">11739766</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15995204" target="_blank">15995204</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=13129959" target="_blank">13129959</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11844760" target="_blank">11844760</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16804183" target="_blank">16804183</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12644479" target="_blank">12644479</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7601854" target="_blank">7601854</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11004182" target="_blank">11004182</a>]<br></li></div>
<li>Nitrate and nitrite control is mediated by homologous DNA-binding response-regulators ( NarL and NarP ) , which communicate with homologous sensor proteins ( NarX and NarQ ) ( reviewed in references 8 and 26 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9696769" target="_blank">9696769</a>]<br></li>
<li>The NarL and NarP response-regulators bind speciﬁc DNA sites upon phosphorylation by the NarX and NarQ sensors .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19060147" target="_blank">19060147</a>]<br></li>
<li>` Locked-on ' and ` locked-off ' signal transduction mutations in the periplasmic domain of the Escherichia coli NarQ and NarX sensors affect nitrate - and nitrite-dependent regulation by NarL and NarP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11004182" target="_blank">11004182</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10464201" target="_blank">10464201</a>]<br></li>
<li>Nitrate regulation is mediated exclusively by the dual sensors ( NarX and NarQ ) and the dual regulators ( NarL and NarP ) in rich medium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7601827" target="_blank">7601827</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_213">
Nitrate repression of the Escherichia coli pﬂ operon is mediated by the dual sensors NarQ and NarX and the dual regulators NarL and NarP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15995204" target="_blank">15995204</a>]<br></label>
<input id="act_similarity_213", type="checkbox">
<div class="ACT_similar">
<li>Nitrate regulation of the Escherichia coli pﬂ operon is mediated by the dual sensors NarQ and NarX and the dual regulators NarL and NarP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10537212" target="_blank">10537212</a>]<br></li></div>
<li>Rather , NarP appears to ﬁne-tune gene expression under low-nitrate conditions by delaying the regulatory response to nitrate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11004182" target="_blank">11004182</a>]<br></li>
<li>In E. coli , an alternative two-component regulatory system , NarQ-NarP , activates gene expression in response to nitrate and is especially sensitive to low concentrations of nitrate [ 8 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16417494" target="_blank">16417494</a>]<br></li>
<li>The effects of NO ¹ 3 2 and NO ¹ are independent of the NarL and NarP response-regulators ( Poole et al. , 1996 ) that mediate induction by these ions of anaerobic respiratory gene regulation in E. coli , further suggesting the operation of novel mechanism ( s ) for upregulation of hmp expression by small nitrogen-containing molecules .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767577" target="_blank">9767577</a>]<br></li>
<li>In the context of regulated gene expression , the napF operon control region is unique among known Nar-regulated targets in both its architecture and its response to the transcriptional-regulators NarL and NarP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11844760" target="_blank">11844760</a>]<br></li>
<li>NarX and/or NarQ sensors , both NarL and NarP are phosphorylated where either response-regulator can then activate nrfA gene expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11004182" target="_blank">11004182</a>]<br></li>
<li>Signal output through NarL phosphate and NarP phosphate continuously adjusts fdnG gene expression in response to the environmental signal input .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923080" target="_blank">12923080</a>]<br></li>
<li>Mutations in narP , the gene encoding the alternative nitrate regulator NarP , reduced the nitrate induction only marginally , but in narLnarP double mutants , nitrate induction was oomplelely absent ( not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7565112" target="_blank">7565112</a>]<br></li>
<li>To determine whether NarP is responsible for the nitrate repression of the hya operon , further experiments need to be done .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8071220" target="_blank">8071220</a>]<br></li>
<li>In addition , anaerobic induction of hya expression was partially dependent on AppY , while hybO expression was partially dependent on NarP even in the absence of nitrate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659690" target="_blank">15659690</a>]<br></li>
<li>The reason for this is unclear , but since nitrate repression of dcuB expression was unaffected by a narP null mutation ( data not shown ) , it appears that NarP is not involved in mediating the nitrate-induced repression of dcuB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9852003" target="_blank">9852003</a>]<br></li>
<li>However , NarP could not induce expression ; rather , it functioned as an antagonist of fdnG-lacZ expression under low-nitrate conditions ( i.e. , it was a negative regulator ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923080" target="_blank">12923080</a>]<br></li>
<li>The effect of NarP on nitrate-dependent repression of fdnG-lacZ expression was not observed in batch cultures sinc this control was observed only at low to intermediate levels of the anion .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923080" target="_blank">12923080</a>]<br></li>
<li>The transcriptional-regulators under 32 control mostly sense extracellular conditions : Mlc senses glucose , Pho P/Q senses low Mg + + and is induced in acid stress , ZntR responds to excess Zn + + , and NarP induction requires nitrite/nitrate under anaerobic conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16818608" target="_blank">16818608</a>]<br></li>
<li>Two of the most obvious possible explanations for these results are either that these genes are repressed by the alternative nitrate-responsive transcription factor , NarP , in response to nitrate or that the metabolites required to induce transcription from these promoters are less available during anaerobic-growth in the presence of nitrate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16377617" target="_blank">16377617</a>]<br></li>
<li>Anaerobic induction was totally dependent upon a functional FNR protein , partially dependent on the nitrate-sensing two-component regulatory system NarXL , and independent of the alternative nitrate response protein NarP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17449618" target="_blank">17449618</a>]<br></li>
<li>Operons repressed by NarP in response to nitrate included a subset of about half of the anaerobically induced genes involved in hydrogen and dicarboxylate metabolism .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16377617" target="_blank">16377617</a>]<br></li>
<li>The microarray data revealed 14 operons that are activated by NarP in response to the availability of nitrate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16417494" target="_blank">16417494</a>]<br></li>
<li>Loss of NarP resulted in lower expression of 14 operons in response to nitrate and increased expression of a further 37 operons ( supplemental Table S2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16377617" target="_blank">16377617</a>]<br></li>
<li>However , the effects , if any , of FNR , nitrate , nitrite , NarXL , and NarP on torCAD and frdABCD transcription , and also the responses of the large majority of operons known to be subject to FNR control , were entirely consistent with previous data from other studies , so we conclude that this was not a problem .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16377617" target="_blank">16377617</a>]<br></li>
<li>Only five operons were induced by nitrate , NarXL and NarP , two of which were also activated by FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16377617" target="_blank">16377617</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_235">
B , subgroups of operons activated directly or indirectly by nitrate , NarXL , or NarP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16377617" target="_blank">16377617</a>]<br></label>
<input id="act_similarity_235", type="checkbox">
<div class="ACT_similar">
<li>C , subgroups of operons repressed directly or indirectly by nitrate , NarXL , or NarP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16377617" target="_blank">16377617</a>]<br></li></div>
<li>The napF operon control region exhibits unusual organization of DNA binding sites for the transcription regulators Fnr and NarP , which activate transcription in response to anaerobiosis and nitrate , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=13129959" target="_blank">13129959</a>]<br></li>
<li>The second level of hierarchical control is medi - ated by the NarL and NarP response-regulators , which , when phosphorylated , bind DNA to activate or repress transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17720788" target="_blank">17720788</a>]<br></li>
<li>Partridge , J.D. , Browning , D.F. , Xu , M. , Newnham , L.J. , Scott , C. , Roberts , R.E. , Poole , R.K. and Green , J. ( 2008 ) Characterization of the Escherichia coli K-12 ydhYVWXUT operon : regulation by FNR , NarL and NarP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21398637" target="_blank">21398637</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_240">
Darwin , A.J. , Ziegelhoffer , E.C. , Kiley , P.J. , and Stewart , V. ( 1998 ) Fnr , NarP , and NarL regulation of Escherichia coli K-12 napF ( periplasmic nitrate reductase ) operon transcription in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15978080" target="_blank">15978080</a>]<br></label>
<input id="act_similarity_240", type="checkbox">
<div class="ACT_similar">
<li>Darwin , A. J. , Ziegelhoffer , E. C. , Kiley , P. J. , and Stewart , V. ( 1998 ) Fnr , NarP , and NarL regulation of Escherichia coli K-12 napF ( periplasmic nitrate reductase ) operon transcription in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21169565" target="_blank">21169565</a>]<br></li></div>
<li>The nrf Escherichia coli K-12 NarL and NarP Proteins Insulate the Promoter from the Effects of Integration Host Factor Douglas F. Browning ,1 * David J. Lee ,1 Alan J. Wolfe ,2 Jeffrey A. Cole ,1 and Stephen J. W. Busby1 School of Biosciences , University of Birmingham , Birmingham B15 2TT , United Kingdom ,1 and Department of Microbiology and Immunology , Stritch School of Medicine , Loyola University Chicago , Maywood , Illinois 601532 Received 5 July 2006/Accepted 16 August 2006 The Escherichia coli K-12 nrf operon promoter can be activated fully by the FNR protein ( regulator of fumarate and nitrate reduction ) binding to a site centered at position 41.5 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16936015" target="_blank">16936015</a>]<br></li>
<li>A DNA site for NarL and NarP , located at position − 104.5 , plays only a minor David L. CHISMON , Douglas F. BROWNING , Gregory K. FARRANT and Stephen J. W. BUSBY1 School of Biosciences , University of Birmingham , Edgbaston , Birmingham B15 2TT , U.K. INTRODUCTION The expression of many Escherichia coli transcripts is tightly controlled in response to environmental cues , and , in many instances , this control is exercised at the level of the promoter by the regulation of the initiation of transcript formation [ 1 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20533909" target="_blank">20533909</a>]<br></li>
<li>This argument would necessarily invoke the involvement of the non-phosphorylated form of NarP in this process , since without added nitrate the NarX and NarQ histidine kinases are inactive .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10537212" target="_blank">10537212</a>]<br></li>
<li>Finally , the narX narQ narL triple-null mutant reveals that the high uninduced basal level of CF ( aeg-46 .5 - lacZ ) expression required either the narX + or narQ + gene , indicating that this expression is due to relatively low-level , nitrate - and nitrite-independent phosphorylation of the NarP protein by the NarX and NarQ proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8501030" target="_blank">8501030</a>]<br></li>
<li>In the presence of nitrate or nitrite , the NarP protein is phosphorylated by either NarQ or NarX to approximately the same extent , although phosphorylation by NarQ may be slightly more efficient .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8501030" target="_blank">8501030</a>]<br></li>
<li>This shows that either the NarX or NarQ protein is sufficient to phosphorylate the NarP protein in response to either nitrate or nitrite .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8501030" target="_blank">8501030</a>]<br></li>
<li>In the narL null narP + strain , either narX + or narQ + was sufficient for full induction by either nitrate or nitrite ( Table 6 ) , demonstrating again that either the NarX or NarQ protein can activate the NarP protein in response to either nitrate or nitrite .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8501030" target="_blank">8501030</a>]<br></li>
<li>Furthermore , it is shown that NarP recognizes a 7-2-7 site close to the transcript start to repress ydhY -- T expression in the presence of nitrate and nitrite , providing new evidence in support of earlier transcript-profiling experiments that suggest that nitrateactivated NarP can act as a repressor ( Constantinidou et al. , 2006 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18227264" target="_blank">18227264</a>]<br></li>
<li>To examine the roles of NarL and NarP , this fragment was cloned into pRW50 to give pKT53 , In which / ac expression was controlled by pnrf .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7854119" target="_blank">7854119</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7854119" target="_blank">7854119</a>]<br></li>
<li>To investigate the effects of the global regulators Fnr , Arc , NarL , and NarP on dcuA and dcuB expression , the fnr , arc , narL , and narP deletions were transferred from the corresponding donor strains , JRG1728 , RM315 , RKP3580 , and RKP3655 , by P1vir-mediated transduction to the dcuA - and dcuB-lacZ fusion strains , JRG3834 and JRG3835 ( Table 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9852003" target="_blank">9852003</a>]<br></li>
<li>Studies to resolve the individual contributions of the NarL and NarP proteins at low substrat levels to the activation and repression of napF gene expression are in progress .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10464201" target="_blank">10464201</a>]<br></li>
<li>The simplest interpretation of this gene expression proﬁle is that NarP acts as an antagonist of NarL-dependent repression of fdhF-lacZ expression ( compare the narL narP strain to the narL narP strain ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923080" target="_blank">12923080</a>]<br></li>
<li>Expression in fnr + strain Expression in fnr m utant + Nitrate + Nitrite + Nitrate + Nitrite Example Regulatory subgroup 150 175 0.4 2.5 0.03 60.4 7.8 76 29.9 38.5 8.8 47.8 4.7 3.8 1.7 20.4 51.4 15.5 42.9 0.3 87.5 5.2 hmpA ytfE Repressed by FNR ; induced by both nitrate and nitrite 129 0.2 1.6 0.03 14.6 9.5 Activated by FNR ; induced by both nitrate and nitrite nirB hcp nrfA yeaR-yoaG yibIH Activated by FNR ; repressed by NarL but activated by NarP Induced by nitrate ; strongly induced by nitrite in an fnr mutant In summary , we have exploited at least five critical factors that limited previous attempts to define the FNR regulon of E. coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16417494" target="_blank">16417494</a>]<br></li>
<li>The membrane-bound sensors NarX and NarP perceive the presence of nitrate and nitrite in the environment ( Lee et al. , 1999 ; Rabin & Stewart , 1992 , 1993 ; Williams & Stewart , 1997 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18227264" target="_blank">18227264</a>]<br></li>
<li>The presence of nitrite ( or nitrate ) in the growth medium triggers the activity of the NarL and NarP activator proteins : these are closely related regulators , activated by the sensor kinases NarX and NarQ , that bind to a single 16 bp sequence at the nir promoter , centred at position ¹ 69.5 ( note that NarL and NarP are highly homologous , bind to identical target sequences and appa-rently play identical roles in activating transcription at the nir promoter ) ( Tyson et al. , 1993 ; 1994 ; reviewed by Stewart , 1993 ; Darwin and Stewart , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9484902" target="_blank">9484902</a>]<br></li>
<li>Transcription control by NarL-CTD and NarP-CTD Plasmid Activator Percentage of wild-type regulation * in strainD Repression O1-napF Class II activation W ( napFHi -- lacZ ) W ( yeaR -- lacZ ) None NarL pACYC184 pVJS4095 pVJS4505 pVJS4098 pVJS4506 0 0 0 100 66 99 63 100 21 76 20 NarL-CTD NarP NarP-CTD * Percentage repression or activation relative to wild-type value during growth with nitrate ( Zhang et al. , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20634237" target="_blank">20634237</a>]<br></li>
<li>For other RRs , however , high-level activation was not observed using the known induction conditions : the addition of nitrate for NarL and NarP ( Constantindou et al. 2006 ) ; the addition of ZnCl2 for RcsB ( Hagiwara et al. 2003 ) ; and the addition of formate for UvrY ( Chavez et al. 2010 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26010043" target="_blank">26010043</a>]<br></li>
<li>Microarray analysis of gene regulation by oxygen , nitrate , nitrite , FNR , NarL and NarP during anaerobic-growth of Escherichia coli : new insights into microbial physiology T.W. Overton , L. Grifﬁths , M.D. Patel , J.L. Hobman , C.W. Penn , J.A. Cole1 and C. Constantinidou School of Biosciences , University of Birmingham , Birmingham B15 2TT , U.K. Abstract RNA was isolated from cultures of Escherichia coli strain MG1655 and derivatives defective in fnr , narXL , or narXL with narP , during aerobic-growth , or anaerobic-growth in the presence or absence of nitrate or nitrite , in non-repressing media in which both strain MG1655 and an fnr deletion mutant grew at similar rates .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16417494" target="_blank">16417494</a>]<br></li>
<li>However , open complex formation in the presence of RNA polymerase , which is an indication for active FNR and transcriptional activation , is stimulated by Fe ( II ) Most of the FNR-dependent promoters are regulated by FNR in cooperation with other regulators such as NarL , NarP ( nitrate regulation ) , ArcA , Fur ( ferric uptake regulation ) , or CRP ( cyclic AMP receptor protein , glucose repression ) ( Table 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8588737" target="_blank">8588737</a>]<br></li>
<li>Many FNR-activated promoters are also regulated by NarL and its homologue , NarP , which share the same consensus target sequence and orchestrate adaptation to the presence of nitrate ions ( 6 , 17 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19251855" target="_blank">19251855</a>]<br></li>
<li>On the other hand , the level of # rd / # rd-red mRNA almost doubled in nitrite-grown strains defective in narP and/or narL , showing that under these growth conditions NarL and NarP proteins act as negative regulators of the # rd operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12586421" target="_blank">12586421</a>]<br></li>
<li>To examine the roles of NarL and NarP at pnir , the EcoRI-H/ndlll fragment was cloned into the low-copy-number iac expression vector pRW50 , such that iac expression was controlled by pnir The resulting recombinant , pKT1 , was transferred to different genetic backgrounds and iac expression was rrieasured during growth in different conditions ( Table 1 ) , In the narL * narP * background , pnir expression was induced in anaerobic conditions and was further increased in the presence of nitrate or nitrite .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7854119" target="_blank">7854119</a>]<br></li>
<li>The highest expression level , 6,180 units , was detected during anaerobic-growth of the nsrR mutant in the presence of nitrite , but the activity was slightly lower , 4,880 units , in the absence of nitrite , presumably reﬂecting the smaller effect of NarL and NarP under these conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17449618" target="_blank">17449618</a>]<br></li>
<li>From this , we conclude that the nitrite response element from the nir promoter can be transplanted to the nrf promoter and that it retains its property of being activated strongly in response to nitrite by NarL but not NarP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7854119" target="_blank">7854119</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7854119" target="_blank">7854119</a>]<br></li>
<li>Finally , in contrast to the well characterized effects of nitrate , nitrite , NarL , and NarP at the nrf and nap promoters , which are activated by NarP but antagonized or repressed by NarL , the dcuB-fumB operon was strongly repressed by both nitrate and nitrite , suggesting that it might provide the first example of a promoter that is repressed by NarP as well as by NarL .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16377617" target="_blank">16377617</a>]<br></li>
<li>Enzymes implicated in NO production in the E. coli cytoplasm The NarL-activated narGHJI operon is strongly induced during anaerobic-growth in the presence of high concentrations of nitrate , whereas the nrf operon is repressed by nitrate-activated NarL , but induced by nitrite - or nitrateactivated NarP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22092912" target="_blank">22092912</a>]<br></li>
<li>, 1994 ) , Rabin and Stewart ( 1993 ) showed that the aeg-93 promoter is reguiated by both NarL and NarP : repression by nitrate is dependent on NarL , whilst activation by nitrite can be effected by NarP or NarL , Hence , in this work , we identified a number of potential NarL hepta-mers at pnrf and exploited site-directed mutagenesis to investigate their roie in NarP - and NarL-dependent regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7854119" target="_blank">7854119</a>]<br></li>
<li>NarL functions as the superior activator at high concentrations of nitrite , whereas NarP plays an activation role at low nitrite concentration .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26010043" target="_blank">26010043</a>]<br></li>
<li>In these instances , nitrate-triggered NarL can stimulate pnir activity whilst nitrite-triggered NarL ( or nitrate-triggered NarP ) is ineffective , again presumably because the binding of the latter to the mutated sites is insufficient .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7854119" target="_blank">7854119</a>]<br></li>
<li>In this case , either nitrite-triggered NarP or NarL were sufficient for activation , contrasting with pnir where nitrite-triggered NarP is not active .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7854119" target="_blank">7854119</a>]<br></li>
<li>We show that activation of pnrf in response to nitrite requires an inverted pair of NarL heptamers , and , as at pnir , these heptamers are a target for both NarP and NarL , Nitrate-triggered NarL-dependent repression of pnrf is dependent on occupation of a second pair of heptamers flanking the FNR-binding site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7854119" target="_blank">7854119</a>]<br></li>
<li>The GA85 substitution suppresses nitrite-dependent stimulation of pnrf , suggesting again that nitrite-triggered NarL or NarP Interacts in some way with FNR , and that this interaction can be disrupted by an alteration in FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7854119" target="_blank">7854119</a>]<br></li>
<li>The simplest explanation for our results is that FNR-dependent expression from pnircan be further stimulated by nitrite - or nitrate-triggered NarL or nitrate-triggered NarP binding to the inverted repeat .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7854119" target="_blank">7854119</a>]<br></li>
<li>It is especially intriguing that binding of IHF to IHF I and to IHF II at the nir promoter produces opposing effects , and it is the balance between these effects that sets the level of FNR-dependent expression in the absence of nitrate/nitrite-dependent activation via NarL or NarP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18757534" target="_blank">18757534</a>]<br></li>
<li>The primary role of IHF , mediated by binding at the IHF I target , is also to downregulate FNR-dependent activation , but this can be reversed by NarP or NarL in response to nitrite or nitrate ions in the environment ( 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20472787" target="_blank">20472787</a>]<br></li>
<li>The role of IHF at the nrf promoter is to set its activity under conditions where nitrite/nitrate-dependent activation via NarL/NarP is minimal .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16936015" target="_blank">16936015</a>]<br></li>
<li>The regulation of nap and nrf has been characterized previously ( 6 , 10 , 24 , 29 , 30 ) ; NarL antagonizes transcription activation in the presence of nitrate , but in the narXL mutant this repression is lifted , and NarP activates transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16377617" target="_blank">16377617</a>]<br></li>
<li>The NarP protein , by contrast , mediates activation by both nitrate and nitrite .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8501030" target="_blank">8501030</a>]<br></li>
<li>Maximal expression of the napFDAGHBC operon encoding periplasmic nitrate reductase results from synergistic transcription activation by the Fnr and phospho-NarP proteins , acting in response to anaerobiosis and nitrate or nitrite , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19060147" target="_blank">19060147</a>]<br></li>
<li>Expression of the E. coli napFDAGHBC operon is positively regulated in response to anaerobiosis by Fnr and by the presence of nitrate and/or nitrite by NarP ( 5 , 7 , 26 , 34 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12029041" target="_blank">12029041</a>]<br></li>
<li>2 napF Fnr , NarP , and NarL Regulation of Escherichia coli K-1 ( Periplasmic Nitrate Reductase ) Operon Transcription In Vitro ANDREW J. DARWIN ,1 † EVA C. ZIEGELHOFFER ,2 ‡ PATRICIA J. KILEY ,3 AND VALLEY STEWART1 * Section of Microbiology , Cornell University , Ithaca , New York 14853,1 and Departments of Bacteriology2 and Biomolecular Chemistry ,3 University of Wisconsin , Madison , Wisconsin 53706 Received 16 March 1998/Accepted 17 June 1998 The expression of several Escherichia coli operons is activated by the Fnr protein during anaerobic-growth and is further controlled in response to nitrate and nitrite by the homologous response-regulators , NarL and NarP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9696769" target="_blank">9696769</a>]<br></li>
<li>Effects of Fnr-positive control mutations on F ( napF-lacZ ) expression - Galactosidase sp act b of culture growna : b pFnr derivative 1O2 16 17 16 14 2O2 68 Dfnrc fnr1 fnr ( G85A ) fnr ( S73F ) DISCUSSION The napF promoter is one of several regulated by Fnr in response to anaerobiosis and by NarL or NarP in response to nitrate and nitrite .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9696769" target="_blank">9696769</a>]<br></li>
<li>This assumption is supported by differences in sensitivity of NarX and NarQ to nitrate and nitrite , as well as differences in their ability to interact with NarL , and in the specificity of the DNA target operons bound by NarL and NarP ( Schröder et al. , 1994 ; Darwin & Stewart , 1995 ; and unpublished work ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8780507" target="_blank">8780507</a>]<br></li>
<li>Like the dmsABC operon , these operons are regulated in response to nitrate availability , in a molybdate-dependent manner , by the NarQ/NarX/NarL / NarP two-component system ( reviewed in Gunsalus , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466267" target="_blank">9466267</a>]<br></li>
<li>Future studies to examine the roles of the two sensor-transmitter proteins , NarX and NarQ , and the two response-regulators , NarL and NarP , should be informative concerning differences in the abilities of the two sensortransmitter proteins and the two response-regulator proteins to modulate nitrate-dependent induction of the narGHJI and the napF operons .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10464201" target="_blank">10464201</a>]<br></li>
<li>Enzyme synthesis is controlled at the level of transcription initiation by the transcription activators Fnr , an iron-sulfur protein responsive to anaerobiosis ( 15 , 16 ) , and the NarL and NarP proteins , response-regulators whose phosphorylation is controlled by the sensors NarX and NarQ in response to nitrate and nitrite ( 9 , 33 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=13129959" target="_blank">13129959</a>]<br></li>
<li>Many FNR-dependent promoters are regulated further in the presence of nitrate and nitrite by two similar response-regulator transcription factors , NarL and NarP , that are phosphorylated by the NarX and NarQ sensor kinases ( reviewed by Darwin and Stewart , 1996 ; Stewart , 2003 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15225315" target="_blank">15225315</a>]<br></li>
<li>Figure 7 Kinetics of induction of the hcp-hcr and ogt promoters Kinetics of activation of promoters in response to nitrate The induction of promoter activity by NarL/NarP follows from the direct sensing of external nitrate or nitrite ions by the NarX/NarQ transmembrane sensor kinases [ 5,6 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20533909" target="_blank">20533909</a>]<br></li>
<li>The response elements of this system also are 2-fold ; the DNA-binding activity of the two transcriptional-regulator proteins , NarL and NarP , can be modulated by either NarX or NarQ in response to nitrate and nitrite .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8780507" target="_blank">8780507</a>]<br></li>
<li>Mutational studies have demonstrated that either NarX or NarQ is capable of modulating the DNA-binding activity of the NarL and NarP proteins in response to nitrate and nitrite ( for a detailed review , see reference 33 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7601827" target="_blank">7601827</a>]<br></li>
<li>The nitrate and nitrite anaerobic respiratory pathways are controlled by two distinct histidine kinases , NarX and NarQ , which simultaneously modulate the phosphorylation of two response-regulators , NarL and NarP , in the presence of nitrate and/or nitrite ( Stewart and Rabin , 1995 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8809780" target="_blank">8809780</a>]<br></li>
<li>Activation by nitrite or nitrate is mediated by either NarL or NarP , two homologous response-regulators that are controlled by the NarX and NarQ sensor kinases ( reviewed by Darwin and Stewart , 1996 ; Stewart , 2003 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15978080" target="_blank">15978080</a>]<br></li>
<li>NarL and NarP are homologous response-regulators that are controlled by the NarX and NarP sensor kinases in response to the presence of nitrite or nitrate ( reviewed in references 9 and 24 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16936015" target="_blank">16936015</a>]<br></li>
<li>Activation of both NarL and NarP is mediated by the membrane-bound NarX and NarQ sensor kinases that are triggered by nitrate or nitrite ions in the environment ( reviewed in [ 5,6 ] ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20533909" target="_blank">20533909</a>]<br></li>
<li>In response to nitrite or nitrate , NarL and NarP are phosphorylated by the membrane bound sensor kinase proteins , NarX and NarQ .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10972841" target="_blank">10972841</a>]<br></li>
<li>Sensing of nitrate and nitrite is mediated by the homologous membrane bound sensor proteins NarX and NarQ which control-the-activity of the homologous response-regulators NarL and NarP via phosphoryl transfer w78 -- 82x .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9230919" target="_blank">9230919</a>]<br></li>
<li>The NarX and NarQ sensors control NarL and NarP phosphor - ylation in response to nitrate and nitrite ( 53 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17720788" target="_blank">17720788</a>]<br></li>
<li>By contrast , the NarQ sensor is hypothesized to act as a NarL and NarP kinase in response to both nitrate and nitrite ( 5 , 31 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12644479" target="_blank">12644479</a>]<br></li>
<li>This suggests that the NarQ sensor can partner effectively with either the NarP or the NarL response-regulator in response to either nitrate or nitrite .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12644479" target="_blank">12644479</a>]<br></li>
<li>Strains unable to synthesize any one of the NarL , NarP , NarQ , or NarX proteins , but retaining the capacity to synthesize the remaining three , exhibited essentially normal nitrate regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7601827" target="_blank">7601827</a>]<br></li>
<li>Therefore , in the absence of NarQ , nitrate-dependent repression of aegA can be mediated by the NarX protein via the NarL and/or NarP protein , as seen for other genes in E. coli ( 11 , 34 , 46 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955321" target="_blank">8955321</a>]<br></li>
<li>Regulation of pﬂ expression by nitrate was dependent on the NarL , NarP , NarQ , and NarX proteins but independent of FNR , ArcA , and integration host factor , which are additional regulators of pﬂ expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7601827" target="_blank">7601827</a>]<br></li>
<li>RESULTS Maximal nitrate repression of pﬂ expression is dependent on functional NarL , NarP , NarX , and NarQ proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7601827" target="_blank">7601827</a>]<br></li>
<li>These ﬁndings indicate that nitrate repression of hya expression is inﬂuenced by the NarL -RCB- NarX and NarP -RCB- NarQ nitrate-sensing systems .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10537212" target="_blank">10537212</a>]<br></li>
<li>An even more complex scheme occurs in the system that controls nitrate/nitrite-responsive gene expression , in which two HKs , NarX and NarQ , regulate two RRs , NarL and NarP ( 225 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10966457" target="_blank">10966457</a>]<br></li>
<li>We provide genetic evidence that either the NarX or the NarQ protein can interact with the NarL and NarP proteins to regulate target operon expression in response to nitrate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8501030" target="_blank">8501030</a>]<br></li>
<li>Finally , we found that nitrite ( NO2 - ) , the reduction product of nitrate , also acts to control target operon expression through the NarX , NarQ , NarL , and NarP proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8501030" target="_blank">8501030</a>]<br></li>
<li>In contrast , the transcription of genes for the periplasmic Nap-Nrf pathway is activated by NarQ-NarP in response to low concentrations of nitrate ( < 100 lM ) but are repressed by NarX-NarL when nitrate is abundant ( Page et al. , 1990 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22029434" target="_blank">22029434</a>]<br></li>
<li>These operons are likely to be regulated by NarL and either NarP , as is the case of hcp , or other transcription factors in response to nitrate or other stimuli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16377617" target="_blank">16377617</a>]<br></li>
<li>The role of NarL heptamer sequences at pnrf The results In Table 1 suggest that NarL and NarP are responsible for nitrite and nitrate regulation at pnrf .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7854119" target="_blank">7854119</a>]<br></li>
<li>Conclusions Beta-galactosidase Activities This work has focused on the roles of NarL and NarP in nitrite/nitrate regulation at pn/r and pnrf and the identification of c/s-acting sequences that are essential for this regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7854119" target="_blank">7854119</a>]<br></li>
<li>Nitrite and nitrate regulation of nir and nrf operon expression is mediated by the NarL and NarP proteins ( Tyson et al. 1993 , 1994 ) : whilst transcription from the nir promoter is induced by both nitrite and nitrate , transcription of the nrf operon is induced by nitrite but repressed by nitrate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9382703" target="_blank">9382703</a>]<br></li>
<li>Conversely , synthesis of the NADH-dependent nitrite reductase , NirBD , is induced by both nitrate and nitrite ; transcription of the nir operon is equally activated by both NarL and by NarP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16377617" target="_blank">16377617</a>]<br></li>
<li>as by-products of nitrate and nitrite metabolism this means that nitrate - or nitrite-induced changes in transcription can be mediated by NsrR as well as by NarL and NarP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20533909" target="_blank">20533909</a>]<br></li>
<li>Nitrate - and nitrite-induced transcription from the hcp promoter was activated by the response-regulator proteins NarL and NarP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15667305" target="_blank">15667305</a>]<br></li>
<li>Consequently , some promoters , for example one of the two napF promoters regulating expression of the periplasmic nitrate reductase operon [ 11 ] , and the nrfA promoter , are activated by NarP in response to nitrite or low concentrations of nitrate , but repressed by NarL when nitrate is abundant .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16417494" target="_blank">16417494</a>]<br></li>
<li>Transcription initiation at the nrfA promoter is activated by FNR in the absence of oxygen and induced further by NarL and NarP in response to low concentrations of nitrate or to nitrite ( 35 , 44 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17449618" target="_blank">17449618</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_319">
The nIr promoter is activated by NarL in response to nitrate and nitrite and activated by NarP in response to nitrate but not nitrite .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7854119" target="_blank">7854119</a>]<br></label>
<input id="act_similarity_319", type="checkbox">
<div class="ACT_similar">
<li>The nrf promoter can be activated by either NarP or NarL in response to nitrite but is repressed by NarL in response to nitrate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7854119" target="_blank">7854119</a>]<br></li></div>
<li>Taken together , these results suggest that NarL is largely sufficient for induction of pnir in response to both nitrate and nitrite , NarP is sufficient for limited induction of pnir and can replace NarL in mediating nitrate - but not nitrite-induced effects .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7854119" target="_blank">7854119</a>]<br></li>
<li>Hence , the role of the regulatory systems NarL and NarP , which integrate signals resulting from the presence of nitrate or nitrite , in # rd / # rd-red transcription was investigated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12586421" target="_blank">12586421</a>]<br></li>
<li>Similarly , the presence of nitrate or nitrite ions leads to the activation of NarL and its homologue , NarP , which belong to the response-regulator family of transcription factors .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20533909" target="_blank">20533909</a>]<br></li>
<li>Therefore , the expression of YdeP might be induced in response to varieties of the environmental signal such as an acidic condition ( through EvgSA ) , in the presence of nitrate/nitrite ( through NarXQ-NarL and NarXQ-NarP ) or in the presence of formate and ace-tate ( through BarA-UvrY ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26010043" target="_blank">26010043</a>]<br></li>
<li>Transcription is also induced by NarL and NarP in response to nitrite , but is repressed by NarL in the presence of nitrate [ 11,18 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10548535" target="_blank">10548535</a>]<br></li>
<li>Synthesis of the cytoplasmic nitrite reductase , NirBD , is induced by both NarP and NarL during anaerobic-growth in the presence of nitrate or nitrite .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22092912" target="_blank">22092912</a>]<br></li>
<li>Together , NarL and NarP appear to co-ordinate transcriptional responses to nitrate and nitrite , although , as yet , the precise relationship between NarL and NarP is not clear ( Stewart , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7854119" target="_blank">7854119</a>]<br></li>
<li>Second , we suppose that nitrate and nitrite trigger NarL and NarP to different activities , with nitrate being more effective in both cases .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7854119" target="_blank">7854119</a>]<br></li>
<li>Second , nitrate and nitrite activation is solely dependent on NarP but is antagonized by the NarL protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9696769" target="_blank">9696769</a>]<br></li>
<li>These sensors then act to control the phosphorylation state of the regulators NarL and NarP ( Darwin & Stewart , 1996 ; Yamamoto et al. , 2005 ) , and there is a complex exchange of information between the sensors and the regulators to allow fine control of targetgene expression in response to nitrate and nitrite .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18227264" target="_blank">18227264</a>]<br></li>
<li>Many of these target promoters control the expression of gene products involved in the reduction of nitrate or nitrite and , hence , NarL and NarP are thought to be the principal regulators of nitrate and nitrite metabolism .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20533909" target="_blank">20533909</a>]<br></li>
<li>Recall that NarL and NarP are homologous response-regulators whose activity is triggered by nitrite or nitrate ions ( 10 ) , and this provides a simple mechanism for the induction of nrf operon expression in response to environmental nitrite or nitrate ( 26 , 28 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20472787" target="_blank">20472787</a>]<br></li>
<li>The addition of nitrite , but not nitrate , is reported to further elevate nrfA expression via either the NarL or NarP response-regulators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11004182" target="_blank">11004182</a>]<br></li>
<li>The NarL and NarP response-regulators mediate nitrate and nitrite induction of anaerobic respiratory gene expression in E. coli ( 49 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8808940" target="_blank">8808940</a>]<br></li>
<li>Integration of three signals at the Escherichia coli nrf ole f Summary Expression from the Escherichia coli nrf operon promoter is activated by the anaerobically triggered transcription factor , FNR , and by the nitrate/nitrite ioncontrolled response-regulators , NarL or NarP , but is repressed by the IHF and Fis proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15978080" target="_blank">15978080</a>]<br></li>
<li>Summary Expression from both the Escherichia coli nir and nrf promoters is dependent on anaerobic induction by FNR but is further reguiated by NarL and NarP in response to the presence of nitrite and nitrate in the growth medium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7854119" target="_blank">7854119</a>]<br></li>
<li>Coordinate regulation of the Escherichia coli formate dehydrogenase fdnGHI and fdhF genes in response to nitrate , nitrite , and formate : roles for NarL and NarP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18227264" target="_blank">18227264</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24659770" target="_blank">24659770</a>]<br></li>
<li>Expression can be increased in response to nitrite or nitrate ions by the nitrite/nitrate-triggered NarL or NarP response-regulators that bind to an upstream site at position -74.5 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15978080" target="_blank">15978080</a>]<br></li>
<li>We proposed that the binding of NarL or NarP to a site centred between positions 269 and 270 counteracts this suppression , resulting in increased transcription in response to nitrite or nitrate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10972841" target="_blank">10972841</a>]<br></li>
<li>Either of the response-regulators , NarL or NarP , was sufﬁcient to confer maximal nitrate and nitrite repression in rich medium , while the NarL protein appeared to be marginally more effective when cells were grown in minimal medium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7601827" target="_blank">7601827</a>]<br></li>
<li>Each of the sensors responds to nitrate and nitrite as well , and each is able to phosphorylate both regulators NarL and NarP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9230919" target="_blank">9230919</a>]<br></li>
<li>the transcription activator FNR ( fumarate-nitrate regulation ) in response to anaerobiosis , and by transcription regulators NarL and NarP in response to nitrite and nitrate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15667305" target="_blank">15667305</a>]<br></li>
<li>For some target genes , FNR is the only regulator , whereas most are regulated coordinately by FNR and the regulators ArcA , NarL , NarP ( nitrate/nitrite regulation ) , Fur ( Fe regulator ) , SoxRS and Q ( regulation by O2̇ ¯ ) , LexA ( SOS regulation ) , or CRP ( glucose regulation ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8588737" target="_blank">8588737</a>]<br></li>
<li>In our recent microarray study of the FNR , NarL , and NarP regulons of E. coli , members of the NsrR regulon were up-regulated in cultures growing anaerobically in the presence of nitrate and nitrite , presumably as a consequence of NO formation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17449618" target="_blank">17449618</a>]<br></li>
<li>At the highest levels of nitrate tested , NarL-dependent induction was approximately 34-fold , while NarP induction was about 26-fold .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11004182" target="_blank">11004182</a>]<br></li>
<li>Transcription from most Nar-regulated operons is activated by Fnr , so NarL and NarP impose nitrate-responsive control on top of Fnr-dependent anaerobic induction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20634237" target="_blank">20634237</a>]<br></li>
<li>These results indicate that both the NarL and NarP proteins are involved in induction of the narXL operon by nitrate or nitrite , a pattern similar to that observed for the narG and fdnG operons ( 23 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7601854" target="_blank">7601854</a>]<br></li>
<li>For example , the nitrate reductase genes narGHJI are regulated exclusively by NarL and are not responsive to NarP as observed here for cydD , while nitrate induction of the fdnGHI operon is regulated predominantly by NarL but is also responsive in part to NarP ( 33 , 39 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9335308" target="_blank">9335308</a>]<br></li>
<li>Thus , for example , the narGHJI operon is regulated exclusively by NarL and is not responsive to NarP , while nitrate induction of the fdnGHI operon is con ¬ * Corresponding author .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7601827" target="_blank">7601827</a>]<br></li>
<li>This also shows that the inﬂuence of the narL and narP null alleles on yeaR-yoaG operon expression reﬂects a direct effect of phospho-NarL and - NarP proteins on transcription activation rather than an indirect effect of altered nitrate and nitrite metabolism in narL and narP null strains .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17720788" target="_blank">17720788</a>]<br></li>
<li>Interestingly , the residual nitrate-dependent repression in the narL mutant host is lost in an narL narP double mutant , in which NarP , a NarL homologue , is inactivated ( Fig. 3B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19251855" target="_blank">19251855</a>]<br></li>
<li>In the narL '' narP ^ background both nitrate repression and nitrite induction were suppressed , suggesting that NarP and NarL are responsible for nitrite induction and nitrate repression , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7854119" target="_blank">7854119</a>]<br></li>
<li>Finally , in the narL * narP ~ background , pnrf is subject to nitrate repression but can be activated by nitrite , suggesting that nitrite activation can be effected by both NarL and NarP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7854119" target="_blank">7854119</a>]<br></li>
<li>As additional NarL protein is activated in the presence of higher levels of nitrate ( ca. 7 mM ) , it fully displaces the NarP phosphate from the DNA and maximally induces fdnG expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923080" target="_blank">12923080</a>]<br></li>
<li>At increased levels of nitrate , the NarL phosphate protein binds to the sites at positions 109 and 100 and replaces the NarP protein ; NarL phosphate then maximally induces fdnG expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923080" target="_blank">12923080</a>]<br></li>
<li>In the presence of low levels of nitrate , either NarL ( L ) or NarP ( P ) bind at the 270 and 279 heptamer sites ( hatched boxes ) to activate nrfA gene expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11004182" target="_blank">11004182</a>]<br></li>
<li>Rather , the che-mostat experiments ( Fig. 4A ) revealed that NarP protein acts to antagonize or spoil the ability of the NarL protein to activate gene expression at intermediate nitrate levels ( i.e. , it functions as a negative regulator ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923080" target="_blank">12923080</a>]<br></li>
<li>At low to intermediate levels of nitrate ( 2 to 6 mM ) NarP appeared to antagonize the ability of NarL to activate fdnG operon expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923080" target="_blank">12923080</a>]<br></li>
<li>NarP thus performs a unique role to ﬁne-tune fdnG operon expression by delaying the ability of NarL to activate when nitrate levels are low .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923080" target="_blank">12923080</a>]<br></li>
<li>Furthermore , in the absence of NarL , dcuB is repressed fourfold by nitrate in a manner that appears to be NarP independent , and , in addition , dcuB expression is reduced by glucose in a fashion that is mostly CRP independent .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9852003" target="_blank">9852003</a>]<br></li>
<li>In the absence of nitrate , fdnG expression is minimal since the NarL and NarP regulatory proteins are in their inactivate forms and therefore unable to recognize and bind DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923080" target="_blank">12923080</a>]<br></li>
<li>The mechanism which the cell employs to repress pﬂ expression with nitrate clearly involves NarL and NarP , but a further factor ( s ) which dictates whether repression is strong or weak , perhaps in response to the intracellular redox status , must be involved .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7601827" target="_blank">7601827</a>]<br></li>
<li>NarL and NarP appear to be responsible for repression of hya expression in response to nitrate ( 5 , 31 ) , whereas the anaerobic transcription factors ArcA and AppY are required for the increased expression of the hya operon in response to anaerobiosis ( 5 , 31 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23065979" target="_blank">23065979</a>]<br></li>
<li>In the case of hya operon regulation , both NarL and NarP appear to be equally effective at repressing expression in the presence of nitrate ( see Table 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10537212" target="_blank">10537212</a>]<br></li>
<li>The NarL and NarP proteins were required for full nitrate induction of narXL operon expression , whereas the nitrate repression of narP and narQ expression was mediated solely by the NarL protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7601854" target="_blank">7601854</a>]<br></li>
<li>One possible explanation for the enhanced nitrate-dependent repression of hya gene expression in the narP mutant is that normally NarL and NarP compete to repress hyb operon expression , with NarP binding being favoured .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10537212" target="_blank">10537212</a>]<br></li>
<li>Anaerobic expression required a functional fnr gene , and nitrate repression depended on NarL and NarP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19661178" target="_blank">19661178</a>]<br></li>
<li>Nitrate repression could however be relieved in a narL narP double mutant , indicating that both NarL and NarP mediate nitrate repression of hya expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10537212" target="_blank">10537212</a>]<br></li>
<li>Neither NarL nor NarP signiﬁcantly effects expression of either the fdhF or fdnG genes when nitrite is present ( Figs. 3B and 4B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923080" target="_blank">12923080</a>]<br></li>
<li>Functioning of NarL and NarP in the control of nrfA and nirB operon expression in response to nitrate and nitrite Actiona by the response-regulator Reporter fusion and method Source or reference NarL NarP Nitrite Nitrate Nitrite Nitrate nrfA 1 1 24 Batch culture 1 2 Chemostat culture 1 , 2 1 , 2 1 1 This study nirB Batch culture 1 1 None 1 24 Chemostat culture 1 1 1 1 This study a 1 , activation of transcription by either the NarL or the NarP response-regulatory protein ; 2 , transcription repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11004182" target="_blank">11004182</a>]<br></li>
<li>These ﬁndings therefore require a reexamination of the regulatory models based on batch culture studies wherein NarL or NarP can activate nrfA expression in response to nitrite only ( 24 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11004182" target="_blank">11004182</a>]<br></li>
<li>Also , since NarL and NarP clearly function in more versatile ways in response to either nitrate or nitrite as a regulatory signal , the prior models based on the batch culture studies must be reconsidered ( 6 , 24 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11004182" target="_blank">11004182</a>]<br></li>
<li>From prior batch culture experiments , both NarL and NarP were reported to activate nirB-lacZ in response to nitrate ( 23 , 24 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11004182" target="_blank">11004182</a>]<br></li>
<li>During batch cell culture conditions , nitrite was reported to induce nrfA expression via either the NarL or NarP response-regulator proteins , while only nitrate was reported to cause repression of nrfA expression and only by NarL ( 14 , 16 , 24 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11004182" target="_blank">11004182</a>]<br></li>
<li>In summary , NarP was the less-effective response-regulator for inducing nirB-lacZ expression relative to NarL when either nitrate or nitrite was present as the inducer signal .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11004182" target="_blank">11004182</a>]<br></li>
<li>It is therefore evident that NarL and NarP regulate nrfA-lacZ expression in response to nitrite addition in the same qualitative way that the nitrate anion signal does in contrast to prior proposals .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11004182" target="_blank">11004182</a>]<br></li>
<li>While either NarL or NarP was able to induce nrfA-lacZ expression in response to low levels of nitrate , only NarL could repress at high nitrate levels .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11004182" target="_blank">11004182</a>]<br></li>
<li>First , either NarL or NarP can activate nrfA-lacZ expression in response to added nitrate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11004182" target="_blank">11004182</a>]<br></li>
<li>In contrast , NarL is reported to repress nrfA expression in response to nitrate , whereas NarP can not ( 14 -- 16 , 23 , 24 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11004182" target="_blank">11004182</a>]<br></li>
<li>Expression of the nirB operon differs from nrfA in that NarL and NarP are reported to activate nirB expression in response to nitrate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11004182" target="_blank">11004182</a>]<br></li>
<li>The various target operons regulated by nitrate appear to have different speciﬁcities for NarL and NarP , and this provides a means of ﬁnetuning gene expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7601827" target="_blank">7601827</a>]<br></li>
<li>The expression of dcuC * 9-9lacZ was increased by a factor of about 2.2 compared to that in the wild type , but the responses to oxygen , nitrate , and the regulators FNR , NarL , and NarP were comparable to those in the wild type ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10368146" target="_blank">10368146</a>]<br></li>
<li>Response of ttdA9 -- 9lacZ expression to the presence of O2 , nitrate and the regulators FNR , ArcA and NarL/NarP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19661178" target="_blank">19661178</a>]<br></li>
<li>Although fdhF expression was reported to be activated by formate and repressed by nitrate , neither NarL nor NarP was tested to determine its role in nitrate - or nitrite-dependent fdhF gene expression ( 21 , 29 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923080" target="_blank">12923080</a>]<br></li>
<li>The different expression proﬁle for the alternative nitrite reductase operon encoded by nirBDC under high-nitrate conditions was due to transcriptional activation by either NarL or NarP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11004182" target="_blank">11004182</a>]<br></li>
<li>Regulation of aeg-93 is more complex than that of aeg-46 .5 , since nitrite is an effective activator and it appears that NarL represses expression in the presence of nitrate but activates it in the presence of nitrite , while NarP is an activator with both nitrate and nitrite .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7601827" target="_blank">7601827</a>]<br></li>
<li>Interestingly , when nitrite was present , NarP did not antagonize NarL activation of nirB-lacZ expression to the extent that it did when nitrate was present ( Fig. 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11004182" target="_blank">11004182</a>]<br></li>
<li>These results show that the NarP protein activates F ( aeg-46 .5 - lacZ ) expression in response to nitrate and nitrite and that the NarL protein antagonizes this activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8501030" target="_blank">8501030</a>]<br></li>
<li>Both NarL and NarP activate the expression of nitrate and nitrite catabolic genes and repress other anaerobic respiratory systems like fumarate reductase ŽFrd .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9230919" target="_blank">9230919</a>]<br></li>
<li>Most of the nitrate and nitrite induction of the fdnG operon is mediated by NarL , although NarP can weakly induce expression ( 23 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7601854" target="_blank">7601854</a>]<br></li>
<li>Both the NarL and NarP proteins were required for full nitrate and nitrite induction of F ( narX-lacZ ) expression and for full nitrite induction of F ( narK-9lacZ ) expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7601854" target="_blank">7601854</a>]<br></li>
<li>The chemostat experiments ( Fig. 3A ) also demonstrate that NarL or NarP can function independently of each other to induce nirB expression in response to nitrate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11004182" target="_blank">11004182</a>]<br></li>
<li>The effect of NarL ( and NarP ) on the expression of the dcuB-lacZ fusion was measured in the presence and absence of nitrate ( Fig. 5D ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9852003" target="_blank">9852003</a>]<br></li>
<li>However , when elevated nitrate conditions are encountered , NarL-phosphate overcomes the antagonistic effect of NarP to give optimal induction of nirB expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11004182" target="_blank">11004182</a>]<br></li>
<li>Therefore , NarP provides a regulatory response to delay NarL-dependent induction of fdnG-lacZ expression when the nitrate concentration is low to intermediate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923080" target="_blank">12923080</a>]<br></li>
<li>Either NarL or NarP was able to induce nirB-lacZ expression in response to nitrite ( compare the NarL2 NarP1 and the NarL1 NarP2 strains to the wild-type strain [ Fig. 3B ] ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11004182" target="_blank">11004182</a>]<br></li>
<li>Second , NarL can repress nrfA-lacZ expression when nitrate is present , whereas NarP can not ( compare the NarL1 NarP2 strain to the NarL2 NarP1 strain and to the wild-type strain ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11004182" target="_blank">11004182</a>]<br></li>
<li>Signiﬁcantly , NarP also appears to somehow antagonize the ability of NarL to activate nirB-lacZ expression at low nitrate levels since , in a NarP2 NarL1 strain , nirB-lacZ expression was higher relative to the wild-type strain during low nitrate additions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11004182" target="_blank">11004182</a>]<br></li>
<li>Lastly , NarP appears to weakly antagonize the ability of NarL to activate nrfA-lacZ expression at low nitrate levels ( compare the NarL1 NarP2 strain to the wild-type strain ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11004182" target="_blank">11004182</a>]<br></li>
<li>Finally , either NarL or NarP was able to activate nrfA-lacZ expression independently of the other , while only NarL could repress expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11004182" target="_blank">11004182</a>]<br></li>
<li>One of the more striking points in each model is that the NarP protein is nonessential ( i.e. , not a major player ) and that it can function as an antagonist of NarL in its ability to activate nirB or nrfA operon expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11004182" target="_blank">11004182</a>]<br></li>
<li>Negative regulation of the NarP response-regulator by the NarQ sensor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=13129959" target="_blank">13129959</a>]<br></li>
<li>Recent work by Rabin and Stewart ( 1993 ) has revealed the presence of a NarL homologue in E. coli , NarP , which is also controlled by NarX and NarQ and can modulate FNR-dependent expression at some promo-ters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7854119" target="_blank">7854119</a>]<br></li>
<li>Our current model to describe the Nar-mediated regulation of anaerobic enzyme synthesis implicitly assumes that the levels of the NarX , NarL , NarP , and NarQ proteins are essentially constant under different growth conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7601854" target="_blank">7601854</a>]<br></li>
<li>Nitrate repression could be relieved completely only in strains grown in rich medium and unable to synthesize either functional NarL and NarP proteins or NarX and NarQ proteins ( Table 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7601827" target="_blank">7601827</a>]<br></li>
<li>We conclude that , in wild-type strains , the NarQ protein communicates the presence of nitrite to both the NarP and NarL proteins and that the NarX protein inhibits this communication with the NarL protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8501030" target="_blank">8501030</a>]<br></li>
<li>This is achieved by two homologous transcription factors , NarL and NarP , which are phosphorylated by the sensor kinases NarX and NarQ ( 23 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18757534" target="_blank">18757534</a>]<br></li>
<li>Dephosphorylation of NarP is restricted to NarQ , dephosphorylation of NarL to NarX .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9230919" target="_blank">9230919</a>]<br></li>
<li>This implies that the NarQ protein negatively regulates the NarP protein , much like the NarX protein negatively regulates the NarL protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8501030" target="_blank">8501030</a>]<br></li>
<li>The simplest model would suggest that the lack of NarP-dependent repression is simply a matter of relative affinities , and that under some circumstances it might be possible to fill heptamers - 5 0 and - 2 2 with NarP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7854119" target="_blank">7854119</a>]<br></li>
<li>The concentrations of MBP -- NarP protein , which had been pre-incubated with 50 mM acetyl phosphate for 45 min to generate active MBP -- NarP , in the reactions are shown above each lane .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18227264" target="_blank">18227264</a>]<br></li>
				</div></ul><br>
			<label class="collapse_DOM" for="_2"><font size="5px">NarP protein domains</font></label>
				<input id="_2" type="checkbox">
				<div class="DOM"><br><ul><li>We have succeeded in purifying the NarP protein , and like NarL , it shows no speciﬁc interaction-with-DNA in the ( presumably ) dephosphorylated state .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7601827" target="_blank">7601827</a>]<br></li>
<li>In the Klebsiella and Erwinia examples , the transcription control region is also different and contains an NsrR protein-binding site ( 40 ) but no apparent phospho-NarL or - NarP protein-binding site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17720788" target="_blank">17720788</a>]<br></li>
<li>In particular , the carboxyl termini of the NarP and NarL proteins contain predicted helix-turn-helix motifs of the type that is found in other DNA-binding proteins such as MalT , FixJ , and RcsA ( Fig. 3 ) ( 18 , 42 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8501030" target="_blank">8501030</a>]<br></li>
<li>Similar results were obtained with the corresponding residues in NarP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20634237" target="_blank">20634237</a>]<br></li>
<li>Each in vitro transcription reaction mixture contained 50 nM RNA polymerase , 0.5 mM Fnr ( D154A ) , and approximately 5 mM phosphorylated MBP-NarP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9696769" target="_blank">9696769</a>]<br></li>
<li>By analogy with other studies of synergistic activation , Fnr and NarP may make independent contacts with RNA polymerase ( 4 , 16 , 17 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9696769" target="_blank">9696769</a>]<br></li>
<li>In this case the NarP protein may not make a stimulatory contact with RNA polymerase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9696769" target="_blank">9696769</a>]<br></li>
<li>The chemostat studies also revealed new roles for the NarL and NarP proteins in the activation and/or repression of the two operons in contrast to prior conclusions derived from batch culture studies .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11004182" target="_blank">11004182</a>]<br></li>
<li>NarP phosphate is unable to perform activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923080" target="_blank">12923080</a>]<br></li>
				</div></ul><br>
			<label class="collapse_EVO" for="_3"><font size="5px">NarP evolution</font></label>
				<input id="_3" type="checkbox">
				<div class="EVO"><br><ul><li>The search for proteins with homology to the putative BglJ protein yielded other members of this family including FimZ , UvrC , MalT , RcsA , NarP , and NarL ( ALTSCHUL et al. 1990 ; STOUTet al. 1991 ; SWENSOaNnd CLEGG1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8725214" target="_blank">8725214</a>]<br></li>
<li>The primary sequence of NarP shares 44 % identity with NarL and the putative NarP helix-turn-helix DNA-bind-ing element is identical to NarL at 12 out of 20 positions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7854119" target="_blank">7854119</a>]<br></li>
<li>The predicted NarP sequence shares similar features with members of this family .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8501030" target="_blank">8501030</a>]<br></li>
<li>The system consists of dual-inter-acting two-component regulatory systems ( 23 ) with homologous membrane-bound sensor proteins ( NarX and NarQ ) and homologous DNA-binding response regulators ( NarL and NarP ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7601854" target="_blank">7601854</a>]<br></li>
<li>Superposition of helices R7 , R8 , and R9 of FIGURE 4 : Sequence alignment of NarL with NarP and with other members of the same subfamily of transcriptional regulators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8780507" target="_blank">8780507</a>]<br></li>
<li>It is therefore appealing that the response regulators , NarL and NarP , are members of the cognate subfamily .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8501030" target="_blank">8501030</a>]<br></li>
<li>Sequence comparisons show that UhpA is a member of the famity of response-regulator proteins that inciudes NarL , NarP and RcsB from # coli , DegU and ComA from Bacittus subtilis , FixJ from Rhizobium species , and others [ Pao etai .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7596290" target="_blank">7596290</a>]<br></li>
<li>This is achieved by two very similar response-regulator family transcription factors , NarL and NarP ( reviewed by Darwin and Stewart , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10972841" target="_blank">10972841</a>]<br></li>
<li>Similarly , the presence of nitrate or nitrite ions leads to the activation of NarL and its homologue , NarP , which belong to the response regulator family of transcription factors .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20533909" target="_blank">20533909</a>]<br></li>
				</div></ul><br>
			<label class="collapse_RP" for="_4"><font size="5px">Processes regulated by NarP</font></label>
				<input id="_4" type="checkbox">
				<div class="RP"><br><ul><li>Doubling regulated regulated of altered time Identiﬁed function Class system ( HK/RR ) Relevant functions genes genes genes ( min ) in this experiment Strain name Mutation ntrol o o C N 1 4 5 27 BW25113 WT or little signiﬁcant alteration CreC/CreB 1 1 2 Catabolite repression ( Wanner , 1996 ) Drug resistance ( Kato et al. , 2000 ) BW26983 DE ( creABCD ) 154 29 2 3 5 EvgS/EvgA BW27869 DE ( evgAS ) 1291 29 2 6 8 YgiY/YgiX NarX/NarL BW27551 BW27864 DE ( ygiXY ) 1302 DE ( narXL ) 1316 27 29 5 4 9 Nitrate regulation ( Stewart and Rabin , 1995 ) Hexose phosphate uptake ( Kadner , 1995 ) Phosphate regulation ( Wanner , 1995 ) Copper response ( Munson et al. , 2000 ) Nitrate regulation ( Stewart and Rabin ,1995 ) UhpB/UhpA 6 3 9 BW27871 DE ( uhpAB ) 1322 28 PhoR/PhoB 1 8 9 BW24476 DE ( phoBR ) 580 28 CusS/CusR 2 7 BW28077 DE ( cusRS ) 1204 27 9 / NarP 9 BW27873 DnarP1312 33 12 3 YedV/YedW YfhK / YehU/YehT KdpD/KdpE DE ( yedWV ) 1298 DyfhK1328 DE ( yehUT ) 1324 DE ( kdpABCDE ) 1224 26 28 27 27 13 15 20 20 6 7 7 8 BW BW BW BW 27550 27872 27877 27564 13 10 7 10 Potassium transport ( Walderhaug et al. , 1992 ) Nitrogen regulation ( Porter et al. , 1995 ) NtrB/NtrC BW27880 DE ( ntrBC ) 1318 31 20 8 12 Signiﬁcant alteration BaeS/BaeR HydH/HydG ( Nagasawa et al. , 1993 ) Response to zinc and lead ( Leonhartsberger et al. , 2001 ) Hydrogen peroxide sensitivity ( Pernestig et al. , 2001 ) 14 11 BW27553 BW27867 DE ( baeSR ) 579 DE ( hydHG ) 1336 8 22 24 29 31 13 2 BarA / BW29434 DbarA1358 24 26 30 7 / FimZ BW28078 BW27558 Enterochelin DﬁmZ1214 22 27 29 31 28 27 4 PhoQ/PhoP Response to magnesium ( Kato et al. , 1999 ) ( Nagasawa et al. , 1993 ) Response to cell envelope stress ( De Wulf et al. , 1999 ) DE ( phoPQ ) 1244 20 16 12 16 BasS/BasR CpxA/CpxR BW27848 BW27559 DE ( basRS ) 1287 DE ( cpxRA ) 623 32 32 29 29 Enterochelin 12 20 22 17 RstB/RstA YojN ( Hpt ) BW27552 BW27866 DE ( rstAB ) 1278 DyojN1332 34 37 26 29 Enterochelin Capsular synthesis ( Takeda et al. , 2001 ) 14 Control of TMAO respiratory system ( Simon et al. , 1994 ) 38 24 TorS/TorR BW26423 26 DE ( torSTRCAD ) 518 a .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12366850" target="_blank">12366850</a>]<br></li>
<li>Stage-3 -- AcP + AcP K R Signal S R Promoter NarL ( pmol ) 0 1 5 10 50 100 ydeP NarL ( pmol ) 0 1 5 10 50 100 ydeP ydeP Low pH Na + K + EvgS EvgA NarL nirB NarQ NarX Nitrate Nitrite NarL NarP ( pmol ) NarP ( pmol ) nirB Nitrate Nitrite NarQ NarX 0 1000 ydeP 0 1000 ydeP NarP arP N wza Saturated fatty acid * RcsB RcsD uhpT Glucose UhpB UhpA RcsB ( pmol ) RcsB ( pmol ) 0 1 5 10 50 100 0 1 5 10 50 100 RcsB ydeP ydeP 6-phosphate csrB BarA UvrY Formate Figure 5 Cross talks in the three stages of signal-transduction involving six NarL-family RRs .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26010043" target="_blank">26010043</a>]<br></li>
<li>Although unlikely , a second possible explanation to account for the unusual regulation of the hyb operon in a narP mutant is that the NarP protein is somehow involved in the anaerobic induction of hyb expression ( see Table 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10537212" target="_blank">10537212</a>]<br></li>
<li>To incorporate a role for IscR , we would suggest that for the napF promoter , anaerobic activation of the downstream promoter ( s ) via FNR , a bona ﬁde anaerobic global regulator , and NarP , a nitrate-dependent activator ( Darwin et al. , 1998 ; Stewart et al. , 2003 ) , masks detection of IscR-dependent changes in expression from the upstream promoter ( Fig. 4C ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16677314" target="_blank">16677314</a>]<br></li>
<li>In this class , KdpD/KdpE , NarP and NtrB/NtrC are known to be involved in response to turgor pressure , nitrate and nitrogen availability respectively ( Laimins et al. , 1981 ; Darwin and Stewart , 1995 ; Zimmer et al. , 2000 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12366850" target="_blank">12366850</a>]<br></li>
<li>Operons repressed by NarP in response to nitrate included a subset of about half of the anaerobically induced genes involved in hydrogen and dicarboxylate-metabolism .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16377617" target="_blank">16377617</a>]<br></li>
<li>Recently , a second regulator , called NarP , involved in nitrate regulation has been found ( 27 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8071220" target="_blank">8071220</a>]<br></li>
<li>The lack of oxygen triggers a global regulator , the Fnr protein , Summary Expression from the Escherichia coli nir promoter is co-dependent on Fnr ( a transcription factor triggered by oxygen starvation ) and on NarL or NarP ( transcription factors triggered by nitrite and nitrate ions ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9484902" target="_blank">9484902</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_9">
Fnr - , NarP - and NarL-dependent regulation of transcription initiation from the Haemophilus influenzae Rd napF ( periplasmic nitrate reductase ) promoter in Escherichia coli K-12 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20634237" target="_blank">20634237</a>]<br></label>
<input id="rp_similarity_9", type="checkbox">
<div class="RP_similar">
<li>Fnr - , NarP - , and NarL-dependent regulation of transcription initiation from the Haemophilus inﬂuenzae Rd napF ( periplasmic nitrate reductase ) promoter in Escherichia coli K-12 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19060147" target="_blank">19060147</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17720788" target="_blank">17720788</a>]<br></li></div>
<li>Fnr , NarP , and NarL regulation of Escherichia coli K-12 napF ( periplasmic nitrate reductase ) operon transcription in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10537212" target="_blank">10537212</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17720788" target="_blank">17720788</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17449618" target="_blank">17449618</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19854903" target="_blank">19854903</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19060147" target="_blank">19060147</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=13129959" target="_blank">13129959</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12644479" target="_blank">12644479</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11844760" target="_blank">11844760</a>]<br></li>
<li>J Bacteriol 171:3817 -- 3823 Darwin AJ , Stewart V ( 1998 ) FNR , NarP , and NarL regulation of Escherichia coli K-12 napF ( periplasmic nitrate reductase ) operon transcription in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11357510" target="_blank">11357510</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_13">
Darwin , A. J. , Ziegelhoffer , E. C. , Kiley , P. J. , and Stewart , V. ( 1998 ) Fnr , NarP , and NarL regulation of Escherichia coli K-12 napF ( periplasmic nitrate reductase ) operon transcription in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21169565" target="_blank">21169565</a>]<br></label>
<input id="rp_similarity_13", type="checkbox">
<div class="RP_similar">
<li>Darwin , A.J. , Ziegelhoffer , E.C. , Kiley , P.J. , and Stewart , V. ( 1998 ) Fnr , NarP , and NarL regulation of Escherichia coli K-12 napF ( periplasmic nitrate reductase ) operon transcription in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15978080" target="_blank">15978080</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16677314" target="_blank">16677314</a>]<br></li></div>
<li>Mutations in narP , the gene encoding the alternative nitrate regulator NarP , reduced the nitrate induction only marginally , but in narLnarP double mutants , nitrate induction was oomplelely absent ( not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7565112" target="_blank">7565112</a>]<br></li>
<li>Table 1 NarL-family RRs and their promoters ( A ) Test response regulator SK RR Regulatory roles Regulation target operons ( total number of operons ) EvgA NarL NarP RcsB UhpA UvrY gadE-mdtEF , acrD , evgAS , emrKY , -- ( 8 ) EvgS NarX Regulator for acid and multidrug-resistance Regulator against high-level nitrate and nitrite Regulator against low-level nitrate and nitrite Regulator of colanic acid in bioﬁlm formation citCDEFXG , nirBDC , hyaABCDEF , narGHJI , -- ( 33 ) hyaABCDEF , napFDAGHEC , nirBDC , -- ( 11 ) leuO , bglGFB , yidL , sfsB , ynbABCD , -- ( 35 ) uhpT ( 1 ) NarQ RcsCD Regulator of hexose phosphate uptake Regulator of carbon storage system UhpBC BarA csrB , csrC ( 2 ) Bold indicates the tested RR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26010043" target="_blank">26010043</a>]<br></li>
<li>In E. coli , phospho-NarP protein apparently played a relatively minor role in ( nnrS-lacZ ) expression , similar to the role in ( yeaR-lacZ ) expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17720788" target="_blank">17720788</a>]<br></li>
<li>The NarP regulon in E. coli contains mainly genes from the nitrate / nitrite respiration pathway [ 18 ] , whereas in the above two Pasteurellaceae , the NarP regulon is extended to include the detoxiﬁcation genes hcp-hcr , dnrN , and norB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16261196" target="_blank">16261196</a>]<br></li>
<li>It is yet unclear if ModE interactions with molybdate act primarily to modulate DNA binding or , alternatively , to affect ModE interactions with other proteins involved in napF gene-expression , including Fnr , NarP , NarL , and RNAP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12029041" target="_blank">12029041</a>]<br></li>
<li>The simplest interpretation of this gene-expression proﬁle is that NarP acts as an antagonist of NarL-dependent repression of fdhF-lacZ expression ( compare the narL narP strain to the narL narP strain ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923080" target="_blank">12923080</a>]<br></li>
<li>We do not understand the relative action of the NarL and NarP proteins in controlling F ( aeg-46 .5 - lacZ ) expression , and we do not understand the role of phosphorylation in this control .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8501030" target="_blank">8501030</a>]<br></li>
<li>NarL also caused a higher ﬁnal level of gene-expression than NarP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11004182" target="_blank">11004182</a>]<br></li>
<li>Revised models for NarL and NarP control of nrfA and nirB gene-expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11004182" target="_blank">11004182</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_24">
Revised model for NarL and NarP control of fdnG gene-expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923080" target="_blank">12923080</a>]<br></label>
<input id="rp_similarity_24", type="checkbox">
<div class="RP_similar">
<li>Revised model for NarL and NarP control of fdhF gene-expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923080" target="_blank">12923080</a>]<br></li></div>
<li>2 napF Fnr , NarP , and NarL Regulation of Escherichia coli K-1 ( Periplasmic Nitrate Reductase ) Operon Transcription In Vitro ANDREW J. DARWIN ,1 † EVA C. ZIEGELHOFFER ,2 ‡ PATRICIA J. KILEY ,3 AND VALLEY STEWART1 * Section of Microbiology , Cornell University , Ithaca , New York 14853,1 and Departments of Bacteriology2 and Biomolecular Chemistry ,3 University of Wisconsin , Madison , Wisconsin 53706 Received 16 March 1998/Accepted 17 June 1998 The expression of several Escherichia coli operons is activated by the Fnr protein during anaerobic growth and is further controlled in response to nitrate and nitrite by the homologous response regulators , NarL and NarP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9696769" target="_blank">9696769</a>]<br></li>
<li>175 , No. 11 Dual Response Regulators ( NarL and NarP ) Interact with Dual Sensors ( NarX and NarQ ) To Control Nitrate - and Nitrite-Regulated Gene Expression in Eschenichia coli K-12 ROSS S. RABIN ' AND VALLEY STEWART1 ,2 * Sections of Microbiology ' * and Genetics and Development , 2 Cornell University , Ithaca , New York 14853-8101 Received 26 January 1993/Accepted 17 March 1993 Two sensor proteins , NarX and NarQ , mediate nitrate regulation of anaerobic respiratory gene-expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8501030" target="_blank">8501030</a>]<br></li>
<li>In E. coli , an alternative two-component regulatory system , NarQ-NarP , activates gene-expression in response to nitrate and is especially sensitive to low concentrations of nitrate [ 8 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16417494" target="_blank">16417494</a>]<br></li>
<li>These sensors then act to control the phosphorylation state of the regulators NarL and NarP ( Darwin & Stewart , 1996 ; Yamamoto et al. , 2005 ) , and there is a complex exchange of information between the sensors and the regulators to allow fine control of targetgene-expression in response to nitrate and nitrite .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18227264" target="_blank">18227264</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_30">
Rabin RS & Stewart V ( 1993 ) Dual response regulators ( NarL and NarP ) interact with dual sensors ( NarX and NarQ ) to control nitrate - and nitrite-regulated gene-expression in Escherichia coli K-12 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22029434" target="_blank">22029434</a>]<br></label>
<input id="rp_similarity_30", type="checkbox">
<div class="RP_similar">
<li>4 Rabin RS & Stewart V ( 1993 ) Dual response regulators ( NarL and NarP ) interact with dual sensors ( NarX and NarQ ) to control nitrate - and nitrite-regulated gene expression in Escherichia coli K-12 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24314029" target="_blank">24314029</a>]<br></li>
<li>Rabin , R.S. , and Stewart , V. ( 1993 ) Dual response regulators ( NarL and NarP ) interact with dual sensors ( NarX and NarQ ) to control nitrate - and nitrite-regulated gene-expression in Escherichia coli K-12 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11967083" target="_blank">11967083</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9593308" target="_blank">9593308</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8809780" target="_blank">8809780</a>]<br></li>
<li>Dual response regulators ( NarL and NarP ) interact with dual sensors ( NarX and NarQ ) to control nitrateand nitrite-regulated gene-expression in Escherichia coli K-12 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18227264" target="_blank">18227264</a>]<br></li>
<li>Dual response regulators ( NarL and NarP ) interact with dual sensors ( NarX and NarQ ) t control nitrate - and nitrite-regulated gene-expression in Esche-richia coli K-12 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8071220" target="_blank">8071220</a>]<br></li>
<li>Dual response regulators ( NarL and NarP ) interact with dual sensors ( NarX and NarQ ) to control nitrate - and nitrite-regulated gene-expression in Escherichia coli K-12 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923080" target="_blank">12923080</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17720788" target="_blank">17720788</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12029041" target="_blank">12029041</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7601827" target="_blank">7601827</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9335308" target="_blank">9335308</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17449618" target="_blank">17449618</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10206709" target="_blank">10206709</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626281" target="_blank">8626281</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8808940" target="_blank">8808940</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9696769" target="_blank">9696769</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20233924" target="_blank">20233924</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11739766" target="_blank">11739766</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15995204" target="_blank">15995204</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=13129959" target="_blank">13129959</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11844760" target="_blank">11844760</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16804183" target="_blank">16804183</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12644479" target="_blank">12644479</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7601854" target="_blank">7601854</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11004182" target="_blank">11004182</a>]<br></li></div>
<li>Many of these target promoters control the expression of gene products involved in the reduction of nitrate or nitrite and , hence , NarL and NarP are thought to be the principal regulators of nitrate and nitrite metabolism .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20533909" target="_blank">20533909</a>]<br></li>
<li>Finally , we found that nitrite ( NO2 - ) , the reduction product of nitrate , also acts to control target operon expression through the NarX , NarQ , NarL , and NarP proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8501030" target="_blank">8501030</a>]<br></li>
<li>These results show that the NarP protein activates F ( aeg-46 .5 - lacZ ) expression in response to nitrate and nitrite and that the NarL protein antagonizes this activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8501030" target="_blank">8501030</a>]<br></li>
<li>Both NarL and NarP activate the expression of nitrate and nitrite catabolic genes and repress other anaerobic respiratory systems like fumarate reductase ŽFrd .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9230919" target="_blank">9230919</a>]<br></li>
<li>the transcription activator FNR ( fumarate-nitrate regulation ) in response to anaerobiosis , and by transcription regulators NarL and NarP in response to nitrite and nitrate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15667305" target="_blank">15667305</a>]<br></li>
<li>Coordinate regulation of the Escherichia coli formate dehydrogenase fdnGHI and fdhF genes in response to nitrate , nitrite , and formate : roles for NarL and NarP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18227264" target="_blank">18227264</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24659770" target="_blank">24659770</a>]<br></li>
<li>Loss of NarP resulted in lower expression of 14 operons in response to nitrate and increased expression of a further 37 operons ( supplemental Table S2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16377617" target="_blank">16377617</a>]<br></li>
<li>We have identified the structural gene for a second nitrate-responsive response regulator , the NarP protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8501030" target="_blank">8501030</a>]<br></li>
<li>Rather , the che-mostat experiments ( Fig. 4A ) revealed that NarP protein acts to antagonize or spoil the ability of the NarL protein to activate gene-expression at intermediate nitrate levels ( i.e. , it functions as a negative regulator ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923080" target="_blank">12923080</a>]<br></li>
<li>Genes and regulators responding to O2 and other acceptors FNR and ArcA , as shown earlier for NarL and NarP , control different sets of target genes Žreviewed in w4 ,5,9,11,12,44 x. FNR acts as the major regulator of anaerobic metabolism and ArcA as the regulator of aerobic metabolism .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9230919" target="_blank">9230919</a>]<br></li>
<li>NarL and NarP are transcriptional regulators of a number of anaerobic electron transport and fermentation-related genes ( Stewart 1994 ; Unden & Bongaerts 1997 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26010043" target="_blank">26010043</a>]<br></li>
				</div></ul><br>
			<label class="collapse_SIT" for="_5"><font size="5px">NarP binding site</font></label>
				<input id="_5" type="checkbox">
				<div class="SIT"><br><ul><li>Y , C or T ; M , A or C ; K , G or T ; and W , A or T. Sequence removed between the NarP and 10 elements is indicated by numbers .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19060147" target="_blank">19060147</a>]<br></li>
				<label class="SIT_collapse_similarity" for="sit_similarity_2">
Nucleotides that match the consensus sequences are shaded black ( promoter 10 and 35 elements ) or gray ( NarP , Fnr , and Crp ) or are enclosed within boxes ( NsrR and ModE ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19060147" target="_blank">19060147</a>]<br></label>
<input id="sit_similarity_2", type="checkbox">
<div class="SIT_similar">
<li>Nucleotides that match the consensus sequences are shaded black ( promoter 10 and 35 elements ) or gray ( NarP , Fnr , and Crp ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19060147" target="_blank">19060147</a>]<br></li></div>
<li>Target operon transcription is activated by the Fnr protein in response to anaerobiosis , and it is further activated and/or repressed by the phosphoNarL and phospho-NarP proteins , which bind to heptamer DNA sequences .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12644479" target="_blank">12644479</a>]<br></li>
<li>These results establish the unconventional napF operon control region architecture , in which the major promoter P1 is activated by the Fnr protein bound to a site centered at 64.5 with respect to the transcription initiation site , working in conjunction with the phospho-NarP protein bound to a site centered at 44.5 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=13129959" target="_blank">13129959</a>]<br></li>
<li>One obvious difference between the synthetic promoter and the napF operon promoter is that transcription from the latter is also activated by the NarP protein bound to a site centered at position 44.5 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=13129959" target="_blank">13129959</a>]<br></li>
<li>Upstream control sequences from representative species contain NarP and Fnr sites in all cases examined ( 53 ; data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19060147" target="_blank">19060147</a>]<br></li>
<li>For NarP , the consensus sequence TACYNMTNNAKNRGTA was used .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16377617" target="_blank">16377617</a>]<br></li>
<li>Identification of FNR - and NarP Promoter-binding Sites -- Potential FNR - and NarP-binding sites were located using the sequence motif search facility of Colibri ( genolist.pasteur.fr / Colibri/genome .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16377617" target="_blank">16377617</a>]<br></li>
<li>Eco57 motif # 1 matches the known FNR binding sequence and motif # 3 matches the known NarP binding sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22144874" target="_blank">22144874</a>]<br></li>
<li>Motif # 1 matches the E. coli FNR ( fumarate and nitrate reduction ) binding consensus sequence ( TTGAT N4 ATCAA ) [ 57 ] and eco57 motif # 3 corresponds to the NarP binding sequence [ 58,59 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22144874" target="_blank">22144874</a>]<br></li>
<li>These sites , which result in missense substitutions ( NarL , Lys-165 to Glu ; NarP , Lys-125 to Gly and His-163 to Glu ) , were designed for a separate study to analyse properties of NarL -- NarP chimerae .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20634237" target="_blank">20634237</a>]<br></li>
<li>This implies that more Fis binds to the nrf promoter region p23C p10T p8T p7T p22C / p22A p21A p42T NarL / NarP pnrf FNR TAACTCTAAAGTGGTATTTTACATGCACTTACAATTGATTAAAGACAACATTCACAGTGTGGTTATTTGTTACACATAGGGGcGAGCA WWWWWWnnnnnnnnWATCAAnnnnTTR GntcAaaTttTganC +1 IHF Consensus tg tAa c Fis Consensus Fig. 6 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15978080" target="_blank">15978080</a>]<br></li>
<li>The H74 .5 , H78 .5 , H84 .5 , H90 .5 , H94 .5 and H100 .5 derivatives were made by insertion of 5 , 9 , 15 , 21 , 25 and 31 bp respectively ( see Experimental procedures ) , which move the centre of the DNA site for NarL/NarP to positions ¹ 74.5 , ¹ 78.5 , ¹ 84.5 , ¹ 90.5 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9484902" target="_blank">9484902</a>]<br></li>
<li>The MBP -- NarP footprint covered ~ 33 bp ( 210 to +23 ) punctuated by two hypersensitive sites ( +6 and +7 ) , and thus the NarP footprint at the ydhY -- T promoter is similar to that observed at nirB ( Darwin et al. , 1997 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18227264" target="_blank">18227264</a>]<br></li>
<li>It is worth noting that , like malEp , the E. coli aeg-46 .5 promoter , which is synergistically controlled by NarP and FNR ( a CRP-related activator ) has a FNR-binding site at a non-permiss-ive position that is separated from the RNA polymerase recognition sequence by two binding sites for a second transcription factor ( Darwin & Stewart , 1995 ; Wing et al. , 1995 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9000616" target="_blank">9000616</a>]<br></li>
<li>NarP ( Darwin and Stewart , 1995 ) , FNR ( Darwin et al. , 1998 ) and ModE ( McNicholas and Gunsalus , 2002 ) binding sites are over - or underlined .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16677314" target="_blank">16677314</a>]<br></li>
<li>This is consistent with the known DNA binding properties of NarP ( recognizes 7-2-7 sites ) and NarL ( recognizes both 7-2-7 sites and isolated heptamers ) ( Darwin et al. , 1997 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18227264" target="_blank">18227264</a>]<br></li>
<li>Like NarP , NarL clearly protected the 7-2-7 region , but in contrast to NarP , the NarL footprint extended further upstream to 229 and thus encompassed the 216 heptamer , with hypersensitive sites at +19 and 215 ( Fig. 3b ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18227264" target="_blank">18227264</a>]<br></li>
<li>In contrast , RR of the NarL family ( including UhpA , NarL , NarP , BvgA , and DegU ) may recognize multiple sites with dyad symmetry , suggesting that they bind as tail-to-tail dimers ( 2 , 4 , 11 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11976297" target="_blank">11976297</a>]<br></li>
<li>The primary sequence of NarP shares 44 % identity with NarL and the putative NarP helix-turn-helix DNA-bind-ing element is identical to NarL at 12 out of 20 positions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7854119" target="_blank">7854119</a>]<br></li>
<li>The ﬁndings of that study , together with those derived from detailed mutagenesis experiments , have identiﬁed a sequence motif ( TACYNMT , where Y 5 C or T , M 5 A or C , and N 5 any nucleotide ) that is recognized by the NarL and NarP proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7601827" target="_blank">7601827</a>]<br></li>
<li>Speciﬁc DNA sites for binding phospho-NarL and phosphoNarP proteins are comprised of heptamer sequences ( 10 , 17 , 18 , 33 ) for which a consensus sequence reads 5 - TAC YYMT-3 ( where Y C or T and M A or C [ 7 ] ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12644479" target="_blank">12644479</a>]<br></li>
<li>Although transcriptional regulation of the hcp operon has not yet been studied experimentally , two motifs with similarity to the consensus sequence ` TACYNKT ' for binding of NarL or NarP , the nitrate-response regulatory proteins [ 37 ] , can be assigned around 142 and 133 bp upstream the ATG start codon of HCP , the second of these being inverted .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10651802" target="_blank">10651802</a>]<br></li>
<li>β-Galactosidase activity narL + narP + 14 400 + narL narP 12 500 + narL narP 16 700 narL narP 7300 36 500 36 000 30 700 11 000 32 400 42 500 22 300 10 100 Nitrate - and nitrite-mediated expression of the hcp is dependent on NarL and NarP Sequence analysis of the hcp regulatory region suggested that it contains a binding site for FNR and also binding sites for nitrite - and nitrate-activated regulators , NarL and NarP ( Figure 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15667305" target="_blank">15667305</a>]<br></li>
<li>The binding sites for FNR and NarL/NarP , centred at position -41.5 and position -69.5 , respectively , are shown by inverted arrows .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15225315" target="_blank">15225315</a>]<br></li>
				<label class="SIT_collapse_similarity" for="sit_similarity_27">
FNR and NarL/NarP binding sites are represented by inverted arrows , while IHF sites are depicted by boxes , with the positions of binding sites shown in parentheses .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20472787" target="_blank">20472787</a>]<br></label>
<input id="sit_similarity_27", type="checkbox">
<div class="SIT_similar">
<li>FNR and NarL/NarP binding sites are represented by inverted arrows , while IHF and Fis binding sites are depicted by boxes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18757534" target="_blank">18757534</a>]<br></li>
<li>FNR , CRP , and NarL/NarP binding sites are represented by inverted arrows , while IHF and Fis binding sites are depicted by boxes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16936015" target="_blank">16936015</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16936015" target="_blank">16936015</a>]<br></li>
<li>FNR and NarL/NarP binding sites are represented by inverted arrows , and the IHF I binding site is depicted by a box .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20472787" target="_blank">20472787</a>]<br></li>
<li>The locations of FNR and NarL / NarP binding sites are represented by inverted arrows , and the IHF binding site is shown by a box .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20472787" target="_blank">20472787</a>]<br></li></div>
<li>Although phospho-NarL can also bind to heptamers that are deployed in other conﬁgurations ( 2 , 4 , 7 , 17 ) , we focus here on the 7-2-7 heptamer pairs that comprise the sole or principal NarL and NarP protein binding sites in their respective control regions ( Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12644479" target="_blank">12644479</a>]<br></li>
<li>The unaltered ydhY promoter region ( 2189 to +176 ) contained all four potential NarL/P binding sites : heptamer 242 , heptamer 216 , heptamer +6 and heptamer +15 ; the +6 and +15 heptamers are arranged in the 7-2-7 pattern recognized by NarP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18227264" target="_blank">18227264</a>]<br></li>
<li>The NarP heptameric binding sites possess a 7-2-7 organization , whereas NarL can recognize heptamers in various arrangements ( 7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9852003" target="_blank">9852003</a>]<br></li>
<li>( 2 ) How do NarL and NarP recognize the different base sequences of putative binding sites in their various regulons ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8780507" target="_blank">8780507</a>]<br></li>
<li>Most phospho-NarL and - NarP binding sites consist of inverted heptamer sequences ( consensus sequence , TACYYMT , where Y is C or T and M is A or C ) separated by 2 nt ( 16 , 30 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17720788" target="_blank">17720788</a>]<br></li>
<li>The NarL and NarP proteins recognize heptamer binding sites that resemble the consensus TACYYMT ( where Y 5 C or T and M 5 A or C ) ( 6 , 12 , 20 , 27 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9696769" target="_blank">9696769</a>]<br></li>
<li>The heptamer binding sites for NarL and NarP are indicated by arrows .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10972841" target="_blank">10972841</a>]<br></li>
<li>The NarP - and NarL-binding site ( inverted heptamer sequences ) is indicated by numbered arrows .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9696769" target="_blank">9696769</a>]<br></li>
<li>A ModE site was identiﬁed well upstream of the Fnr and NarP binding sites at the napF promoter , and we demonstrate that ModE binds this site in a molybdate-responsive manner to thereby coordinate enzyme synthesis with molybdate availability .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12029041" target="_blank">12029041</a>]<br></li>
<li>The Figure shows the base sequence from position − 125 to position + 98 with respect to the reported transcript start , and the previously proposed DNA sites for NarL and NarP , FNR and NsrR [ 9 -- 11 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20533909" target="_blank">20533909</a>]<br></li>
<li>Consistent with this , comparison of the promoter sequence in several related bacteria show that core elements , including the DNA sites for FNR and NsrR , are better conserved than the upstream ﬂanking sequences that contain the DNA site for NarL and NarP ( Figure 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20533909" target="_blank">20533909</a>]<br></li>
<li>Effects of changing the spacing between the DNA sites for Fnr and NarL/NarP Extensive mutational analysis of the nir promoter has identiﬁed a single functional 22 bp DNA site for Fnr from positions ¹ 31 to ¹ 52 ( centred at position ¹ 41.5 ) ( Jayaraman et al. , 1988 ; 1989 ; Bell et al. , 1990 ; see also Green et al. , 1996a ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9484902" target="_blank">9484902</a>]<br></li>
<li>This results from its dependence on FNR that binds to a site at position -41.5 , and NarL or NarP that bind to a 7-2-7 element centred at position -69.5 ( Fig. 10 ; Jayaraman et al. , 1989 ; Tyson et al. , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15978080" target="_blank">15978080</a>]<br></li>
<li>The nir promoter ( pnir ) contains an FNR binding site centred at position -41.5 and a NarL/NarP binding site centred at position -69.5 ( Jayaraman et al. , 1989 ; Tyson et al. , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15225315" target="_blank">15225315</a>]<br></li>
<li>Deletions and mutations in the nir promoter upstream sequences The simplest model to account for the results in Fig. 2 is to suppose that one or more factors other than NarL and NarP bind to DNA elements upstream of the NsiI site at the nir promoter and can up or downregulate Fnr-depen-dent transcription activation , according to their precise location .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9484902" target="_blank">9484902</a>]<br></li>
<li>Fnr binds to a single DNA site centred between basepairs 41 and 42 upstream from the nir transcript start , whereas NarL and NarP bind to a site upstream , centred between basepairs 69 and 70 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9484902" target="_blank">9484902</a>]<br></li>
<li>To date , the nir operon promoter is the only case in which sequences far upstream of the DNA site for NarL and NarP have been shown to play a signiﬁcant role , and in which this site is centred at position ¹ 69.5 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9484902" target="_blank">9484902</a>]<br></li>
<li>Prior DNA footprinting studies demonstrate that NarL and NarP bind to and protect several NarL heptamer recognition sites located in the nrfA promoter region ( 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11004182" target="_blank">11004182</a>]<br></li>
<li>The nrfA operon has a promoter element containing an activation site consisting of the 279 and 270 heptamers that can be recognized by either NarL or NarP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11004182" target="_blank">11004182</a>]<br></li>
<li>Footprinting assays showed that NarP recognized the 7-2-7 site centred at +10.5 and that NarL recognized a larger region of the ydhY -- T promoter that included the 7-2-7 site and 216 heptamer .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18227264" target="_blank">18227264</a>]<br></li>
<li>Using mutational analysis and gel retardation assays , we showed that IHF binds to a site centred at position -54 , overlapping the DNA sites for FNR and NarL/NarP , while Fis binds to a site at position -15 , overlapping the promoter -10 element ( Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15978080" target="_blank">15978080</a>]<br></li>
<li>Arrows show the NarL / NarP 7-2-7 element and the DNA site for FNR , while the nrf promoter -10 element is underlined .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15978080" target="_blank">15978080</a>]<br></li>
<li>The NarP regulon overlaps the NarL regulon because both NarL and NarP can bind to a NarP site , which is a 7-2-7 inverted-repeat sequence where the seven conserved bases form the NarL heptamer [ 10 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16417494" target="_blank">16417494</a>]<br></li>
<li>In addition , analysis of the upstream region of # rd revealed the presence of a second cis-acting element , the inverted-repeat heptamer centered at 3115 ( 5P-TACT-CAT-TA-ATGGGCA-3P ) , that shares sequence similarity with the NarL/NarP binding site consensus .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12586421" target="_blank">12586421</a>]<br></li>
<li>In previous work ( 3 -- 6 ) , we showed that the DNA target for FNR at pnrf is centered at position 41.5 ( i.e. , between 41 and 42 bp upstream from the transcript start point ) , that anaerobically induced FNR-dependent activation of pnrf is downregulated by the binding of the nucleoid-associated protein IHF to a target ( IHF I ) centered at position 54 , and that this repression is relieved by the binding of either NarL or NarP to a DNA target centered at position 74.5 ( Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20472787" target="_blank">20472787</a>]<br></li>
<li>The results in Fig. 5 show that IHF alone protected an extended region of 100 bp , encompassing IHF I , IHF II , and IHF III ( lane 5 ) , while NarL alone protected the NarL/NarP binding site centered at position 74.5 ( lane 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16936015" target="_blank">16936015</a>]<br></li>
<li>In previous work , we found that NarL or NarP activates the nir promoter by binding to a site centred at position -69.5 and counteracting the repressive effects of IHF and Fis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15225315" target="_blank">15225315</a>]<br></li>
<li>Once phosphorylated , NarL or NarP binds to the same target site centered at position 69.5 ( Fig. 1A ) and activates transcription by displacing IHF from IHF I , thereby counteracting the repression by IHF and Fis from upstream sites ( 3 , 25 , 28 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18757534" target="_blank">18757534</a>]<br></li>
<li>Distance from transcription start site -- 100 +1 +100 -- 300 -- 200 +200 +300 CTTATTTTTTATATTGA TATTTAATCTATTGTTA ATAAGTA TGTAAGAA AATCCTA AATCCTA ydeP RcsB RcsB NarL/NarP TACCTCT EvgA NarL/NarP NagC PhoP Figure 6 Location of TF-binding sites on the ydeP promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26010043" target="_blank">26010043</a>]<br></li>
<li>Sequences for binding the NarP and NarL , Fnr , and ModE proteins are boxed ; consensus sequences are shown below .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=13129959" target="_blank">13129959</a>]<br></li>
<li>Five sequences with similarity to the NarL or NarP binding site consensus sequence ( 7 ) were identiﬁed by using the score matrix approach .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9852003" target="_blank">9852003</a>]<br></li>
<li>This conﬁrms that the phospho-NarL and - NarP protein binding site , identiﬁed by sequence inspection , is critical for regulation by these proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17720788" target="_blank">17720788</a>]<br></li>
<li>Evidence suggests that the phospho-NarL protein competes with the phospho-NarP protein for binding to the site centered at 44.5 but that only the phospho-NarP protein is able to activate transcription from this location ( 8 , 10 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=13129959" target="_blank">13129959</a>]<br></li>
<li>At napF , the NarL protein competes with the 44 % identical NarP protein for binding to a common DNA site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19854903" target="_blank">19854903</a>]<br></li>
<li>A potential phospho-NarL and - NarP binding site is centered at position 43.5 relative to the transcription initiation site ( Fig. 1A ) , immediately adjacent to the promoter 35 element .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17720788" target="_blank">17720788</a>]<br></li>
<li>Substitutions were designed to eliminate binding to the half-site sequence , to leave the promoter 35 element and the phospho-NarL and - NarP protein binding site intact , and to maintain the overall G C composition .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17720788" target="_blank">17720788</a>]<br></li>
<li>Spacers of 5 and 10 bp between the Fnr binding site and the NarP and NarL binding sites are in lowercase letters and underlined NarL binding site , we therefore draw no conclusions regarding its role in napF operon expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=13129959" target="_blank">13129959</a>]<br></li>
<li>Fig. 1 Organisation of the Escherichia coli nir promoter showing the relative locations of the transcription start ( arrow labelled +1 ) , the -- 10 hexamer , the binding site for the FNR protein centred between bases 41 and 42 upstream of the transcription start ( labelled -- 41.5 ) , the overlapping binding sites for the NarL and NarP proteins centred as an inverted-repeat at -- 69.5 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9382703" target="_blank">9382703</a>]<br></li>
<li>NarL or NarP sites are normally located , at multiple positions , upstream of the 235 site ( 7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9852003" target="_blank">9852003</a>]<br></li>
<li>In pnir7061 , sequences upstream of position 261 are deleted ( including the NarL/NarP binding site ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10972841" target="_blank">10972841</a>]<br></li>
<li>is centered at or near position 41.5 with respect to the transcription initiation site ( 15 ) and the NarL-NarP binding sites are upstream of the Fnr binding site ( 9 , 33 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=13129959" target="_blank">13129959</a>]<br></li>
<li>Activation by NarL or NarP occurs when one or both of these proteins bind upstream of an Fnr-binding site centered at 41.5 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9696769" target="_blank">9696769</a>]<br></li>
<li>The pnrf - 7 0 and - 7 9 heptamers must bind nitrite-activated NarP with greater affinity than the heptamer sites at pnir .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7854119" target="_blank">7854119</a>]<br></li>
<li>The presence of nitrite ( or nitrate ) in the growth medium triggers the activity of the NarL and NarP activator proteins : these are closely related regulators , activated by the sensor kinases NarX and NarQ , that bind to a single 16 bp sequence at the nir promoter , centred at position ¹ 69.5 ( note that NarL and NarP are highly homologous , bind to identical target sequences and appa-rently play identical roles in activating transcription at the nir promoter ) ( Tyson et al. , 1993 ; 1994 ; reviewed by Stewart , 1993 ; Darwin and Stewart , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9484902" target="_blank">9484902</a>]<br></li>
<li>To pinpoint sequences necessary for this regulation , we scanned the pnrf sequence for possible NarL heptamer sequences , because it is possible that these are the targets for both NarL and NarP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7854119" target="_blank">7854119</a>]<br></li>
<li>We suggest that both NarL and NarP recognize the same heptamer sequence , but that there is a set of degenerate sequences that , though still recognized by NarL , are bound very poorly by NarP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7854119" target="_blank">7854119</a>]<br></li>
<li>Upon activation , NarL and NarP bind to identical heptamer sequences and , at many target promoters , two heptamers are found as a ` tail-to-tail ' inverted-repeat ( often referred to as the 7-2-7 arrangement ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20533909" target="_blank">20533909</a>]<br></li>
<li>At the nrf promoter , NarL and NarP bind to two heptamer sequences , organized as an inverted-repeat separated by 2 base pairs , centred at position -74.5 ( referred to as the 7-2-7 element ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15978080" target="_blank">15978080</a>]<br></li>
<li>The effects of point mutations suggest that NarL and NarP both bind to the same target , which is a pair of heptamer sequences organized as an inverted-repeat , centred 69V ^ bp upstream of the transcript startpoint .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7854119" target="_blank">7854119</a>]<br></li>
<li>Optimal activation by NarP or NarL in response to nitrite requires an inverted pair of heptamer sequences , similar to that found at the n/r promoter , but centred 74 ^ ^ bp upstream from the transcript start .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7854119" target="_blank">7854119</a>]<br></li>
<li>Phosphorylated NarL and NarP bind to speciﬁc heptamer sequences within target promoters , and either up - or downregulate transcription initiation depending on the promoter organization .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15225315" target="_blank">15225315</a>]<br></li>
<li>Phosphorylated NarL and NarP then bind to specific heptamer sequences at target promoters , and either up - or downregulate transcription initiation at these promoters ( for examples , see Tyson et al. , 1994 ; Darwin et al. , 1997 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10972841" target="_blank">10972841</a>]<br></li>
<li>We conciude that NarL and NarP bind to the same hepta-mer sequences , but that the affinities for the two factors vary from site to site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7854119" target="_blank">7854119</a>]<br></li>
<li>Our work with the nir and nrf promoters shows that there is an overlap in the target sequences for NarL and NarP binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7854119" target="_blank">7854119</a>]<br></li>
<li>Taken together , these results suggest that NarL and NarP bind to the same target sequences at the nir promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7854119" target="_blank">7854119</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7854119" target="_blank">7854119</a>]<br></li>
<li>The inverted sequence symmetry of 7-2-7 heptamers pairs , coupled with analysis of single - and double-nu-cleotide substitutions , strongly suggests that they are bound by dimers of phospho-NarL or phospho-NarP protein ( 33 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12644479" target="_blank">12644479</a>]<br></li>
<li>The latter heptamers are arranged as a 7-2-7 inverted-repeat , which is required for recognition by NarP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18227264" target="_blank">18227264</a>]<br></li>
<li>The repressive effects of IHF and Fis at pnir are relieved by NarL or NarP binding to two target heptamers , organized as an inverted site centred at position 269.5 ( Tyson et al. , 1994 ; Darwin et al. , 1997 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10972841" target="_blank">10972841</a>]<br></li>
<li>Similarly , the targets for both NarL and NarP have been identiﬁed as two heptamer sequences organized as an inverted-repeat separated by 2 bp : this ` 7 -- 2 -- 7 ' NarL / NarP-binding sequence is located from positions ¹ 77 to ¹ 62 ( centred at ¹ 69.5 ) ( Tyson et al. , 1993 ; Darwin et al. , 1997 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9484902" target="_blank">9484902</a>]<br></li>
<li>Both NarL and NarP bind to heptamers organized as inverted-repeats with 2-bp spacing .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9696769" target="_blank">9696769</a>]<br></li>
<li>In addition , there is differential recognition of NarL/P heptamer patterning , such that NarL recognizes all TACYYMT heptamers , whereas NarP only binds these sequences when they are arranged as an inverted-repeat with a 2 bp spacing , the socalled 7-2-7 arrangement ( Darwin et al. , 1997 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18227264" target="_blank">18227264</a>]<br></li>
<li>In contrast to NarL , the nitrite-responsive NarP protein recognizes heptamers organized as 7-2-7 inverted-repeats ( Darwin et al. , 1997 ) , such as heptamers +6 and +15 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18227264" target="_blank">18227264</a>]<br></li>
				</ul></div><br>
			<label class="collapse_TU" for="_6"><font size="5px">Transcription units regulated by NarP</font></label>
				<input id="_6" type="checkbox">
				<div class="TU"><br><ul><li>TABLE 4 Examples of NarXL - and nitrate-regulated operons TABLE 5 Examples of NarP-regulated operons Ratiob Transcript in each T NO / WT narXL NO / WT subgroupa 3 3 NO3 NO3 A. NarXL activated in response to nitrate ( 44 operons ) narK * 14.3 1.04 narG * 33.9 0.91 ogt 12.8 1.51 cyoA 13.1 1.47 sodA 6.7 1.05 B. Induced by nitrate even in the narXL mutant ( 7 operons ) hcp * 30.0 13.6 fdnG * 15.2 2.0 yeaR 89.9 5.5 yedF 6.94 2.66 hmpA 20.4 9.07 yibI 5.05 2.31 ytfE 50.5 10.2 C. Induced by nitrate but not significantly NarXL-activated ( 11 operons ) nirB * 9.87 11.02 ydjY 3.36 3.10 D. Induced by nitrate only in the narXL mutant ( 3 operons ) napA * 1.01 14.8 nrfA * 0.32 8.81 yjiM 1.62 3.4 E. Possibly repressed by both NarL and NarP ( 4 operons ) hyaA * 0.09 0.48 appC 0.12 0.39 ynfE 0.03 0.13 fumB * 0.03 0.27 F. Repressed by nitrate and NarXL ( 34 operons ) dmsA * 0.10 0.61 ansB 0.25 1.18 frdA * 0.03 0.87 fdoG 0.24 0.84 hybA * 0.17 0.85 a Genes marked with asterisks have been reported previously to be regulated by NarL and/or NarP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16377617" target="_blank">16377617</a>]<br></li>
<li>Ratiob narXL NO3 / WT narXLP NO3 / WT Transcript a W NO3 NO3 Activated by NarP ( total of 14 operons ) fdnH 2.03 0.58 yeaR 5.48 0.91 napF * 12.67 0.90 nirB * 11.0 1.22 nrfA * 8.81 2.02 Repressed by NarP ( total of 37 operons ) yahL 0.36 1.46 prpB 0.20 1.17 yaiV 0.24 1.04 ynfE 0.14 1.08 chbF 0.13 0.72 hycA 0.47 4.65 yhiM 0.86 3.55 dcuB 0.26 1.19 a Genes marked with asterisks have been reported previously to be regulated by NarP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16377617" target="_blank">16377617</a>]<br></li>
<li>Mutations in narP , the gene encoding the alternative nitrate regulator NarP , reduced the nitrate induction only marginally , but in narLnarP double mutants , nitrate induction was oomplelely absent ( not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7565112" target="_blank">7565112</a>]<br></li>
<li>By contrast , the NarQ-NarP system may play a more restricted role in controlling gene expression when respiratory electron acceptors are limiting ( 52 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15995204" target="_blank">15995204</a>]<br></li>
<li>Deletion , mutational , and footprint analyses had indicated that napF op-eron expression in vivo is induced weakly by either Fnr or NarP alone and strongly by Fnr and NarP together .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9696769" target="_blank">9696769</a>]<br></li>
<li>Do Fnr and NarP activate napF transcription synergistically ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9696769" target="_blank">9696769</a>]<br></li>
<li>It appeared that the in vitro napF transcription start site in the presence of either RNA polymerase alone or Fnr ( D154A ) was approximately three nucleotides upstream relative to the site in the presence of MBP-NarL or MBP-NarP ( Fig. 3 and 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9696769" target="_blank">9696769</a>]<br></li>
<li>Both MBP-NarP and Fnr ( D154A ) are required for maximum napF transcription in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9696769" target="_blank">9696769</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9696769" target="_blank">9696769</a>]<br></li>
<li>These results conﬁrm that MBP-NarL decreases MBP-NarP - and Fnr ( D154A ) - depen-dent transcription of napF in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9696769" target="_blank">9696769</a>]<br></li>
<li>Antagonization of MBP-NarP - and Fnr ( D154A ) - dependent napF transcription by MBP-NarL .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9696769" target="_blank">9696769</a>]<br></li>
<li>The che-mostat data ( Fig. 4A ) imply that bound NarP phosphate is not competent for activating fdnG transcription since fdnG-lacZ expression was not affected in the narP narL strain compared to expression in the narP narL strain ( see below ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923080" target="_blank">12923080</a>]<br></li>
<li>Whether the NarP phosphate bound to DNA is unable to productively interact with FNR or RNA polymerase to induce fdnG gene expression is not known .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923080" target="_blank">12923080</a>]<br></li>
<li>Expression in fnr + strain Expression in fnr m utant + Nitrate + Nitrite + Nitrate + Nitrite Example Regulatory subgroup 150 175 0.4 2.5 0.03 60.4 7.8 76 29.9 38.5 8.8 47.8 4.7 3.8 1.7 20.4 51.4 15.5 42.9 0.3 87.5 5.2 hmpA ytfE Repressed by FNR ; induced by both nitrate and nitrite 129 0.2 1.6 0.03 14.6 9.5 Activated by FNR ; induced by both nitrate and nitrite nirB hcp nrfA yeaR-yoaG yibIH Activated by FNR ; repressed by NarL but activated by NarP Induced by nitrate ; strongly induced by nitrite in an fnr mutant In summary , we have exploited at least five critical factors that limited previous attempts to define the FNR regulon of E. coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16417494" target="_blank">16417494</a>]<br></li>
<li>Nitrate-stimulated NarP activated expression of 14 operons ; all of those most strongly activated by NarP were part of the FNR regulon and had been characterized independ-ently .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16377617" target="_blank">16377617</a>]<br></li>
<li>Only five operons were induced by nitrate , NarXL and NarP , two of which were also activated by FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16377617" target="_blank">16377617</a>]<br></li>
<li>Nine operons were regulated in different directions by NarXL and NarP ( five being NarXL-repressed and NarP activated ) ; three of these were also FNR-dependent .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16377617" target="_blank">16377617</a>]<br></li>
<li>Two of the most obvious possible explanations for these results are either that these genes are repressed by the alternative nitrate-responsive transcription factor , NarP , in response to nitrate or that the metabolites required to induce transcription from these promoters are less available during anaerobic growth in the presence of nitrate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16377617" target="_blank">16377617</a>]<br></li>
<li>Operons repressed by NarP in response to nitrate included a subset of about half of the anaerobically induced genes involved in hydrogen and dicarboxylate metabolism .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16377617" target="_blank">16377617</a>]<br></li>
<li>In all such cases examined , the napD and nrfA operon upstream control regions contain NarP , Fnr , and Crp sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19060147" target="_blank">19060147</a>]<br></li>
<li>Thus , we conclude that IHF and Fis can repress pnir in the absence of H-NS ( Table 2 ) , and that repression by H-NS can not be reversed by the action of NarL/NarP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10972841" target="_blank">10972841</a>]<br></li>
<li>Although unlikely , a second possible explanation to account for the unusual regulation of the hyb operon in a narP mutant is that the NarP protein is somehow involved in the anaerobic induction of hyb expression ( see Table 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10537212" target="_blank">10537212</a>]<br></li>
<li>Microbiology ( 2008 ) , 154 , 608 -- 618 DOI 10.1099 / mic .0.2007 / 012146-0 Characterization of the Escherichia coli K-12 ydhYVWXUT operon : regulation by FNR , NarL NarP and Jonathan D. Partridge ,1 Douglas F. Browning ,2 Meng Xu ,2 Louise J. Newnham ,1 Colin Scott ,1 Ruth E. Roberts ,1 Robert K. Poole1 and Jeffrey Green1 Correspondence Jeffrey Green 1 Department of Molecular Biology and Biotechnology , The University of Sheffield , Sheffield S10 2TN , UK jeff.green@sheffield.ac.uk 2 The School of Biosciences , The University of Birmingham , Birmingham B15 2TT , UK In Escherichia coli K-12 the expression of many genes is controlled by the oxygen-responsive transcription factor FNR and the nitrate - and nitrite-responsive two-component systems NarXL and NarPQ .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18227264" target="_blank">18227264</a>]<br></li>
<li>The hyaA promoter has been reported to be repressed by both NarL and NarP ; the other operons in this group possibly share this pattern of regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16377617" target="_blank">16377617</a>]<br></li>
<li>Similar differences were observed in the regulation by nitrate of genes for hydrogenase synthesis : the hyaABCDEF and hyc ¬ ABCDEFGH operons were repressed by both NarL and NarP ; the hyb operon was repressed only by NarL ( Figures 1C and 1D ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16417494" target="_blank">16417494</a>]<br></li>
<li>In contrast with the frdABCD , aspA and ansB operons that are repressed only by NarL , the dcuB-fumB operon was among 37 operons that are repressed by NarP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16417494" target="_blank">16417494</a>]<br></li>
<li>Although the frdABCD , aspA and ansB operons were repressed only by NarL , the dcuB-fumB operon was strongly repressed by both NarL and NarP ( Figures 1A and 1B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16417494" target="_blank">16417494</a>]<br></li>
<li>As there was no previous convincing evidence that NarP can repress transcription of nitrate-responsive operons , it was both interesting and surprising that 37 operons were repressed by nitrate even in a narXL deletion strain relative to the narXL narP mutant , showing that these operons are repressed by NarP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16417494" target="_blank">16417494</a>]<br></li>
<li>Although the frdA , dmsA , aspA , ansB , and hyb promoters were significantly repressed only by nitrate-activated NarL , the hycA and dcuB promoters were also repressed by nitrate-activated NarP , as predicted in Table 4 , part E. Eight of the 37 significantly repressed operons gave repression ratios greater than 4-fold ( Table 5 , amount of transcript in the narXL narP strain relative to that of the narXL strain ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16377617" target="_blank">16377617</a>]<br></li>
<li>Rather , NarP appears to ﬁne-tune gene expression under low-nitrate conditions by delaying the regulatory response to nitrate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11004182" target="_blank">11004182</a>]<br></li>
<li>However , NarP could not induce expression ; rather , it functioned as an antagonist of fdnG-lacZ expression under low-nitrate conditions ( i.e. , it was a negative regulator ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923080" target="_blank">12923080</a>]<br></li>
<li>To examine the roles of NarL and NarP at pnir , the EcoRI-H/ndlll fragment was cloned into the low-copy-number iac expression vector pRW50 , such that iac expression was controlled by pnir The resulting recombinant , pKT1 , was transferred to different genetic backgrounds and iac expression was rrieasured during growth in different conditions ( Table 1 ) , In the narL * narP * background , pnir expression was induced in anaerobic conditions and was further increased in the presence of nitrate or nitrite .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7854119" target="_blank">7854119</a>]<br></li>
<li>The reason for this is unclear , but since nitrate repression of dcuB expression was unaffected by a narP null mutation ( data not shown ) , it appears that NarP is not involved in mediating the nitrate-induced repression of dcuB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9852003" target="_blank">9852003</a>]<br></li>
<li>Certainly , phospho-NarP protein is an effective activator of napF operon transcription ( 6 , 8 , 25 ) , suggesting that it probably has a relatively high afﬁnity for the napF operon 7-2-7 heptamer pair ( Table 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12644479" target="_blank">12644479</a>]<br></li>
<li>Target operon transcription is activated by the Fnr protein in response to anaerobiosis , and it is further activated and/or repressed by the phosphoNarL and phospho-NarP proteins , which bind to heptamer DNA sequences .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12644479" target="_blank">12644479</a>]<br></li>
<li>Maximal expression of the napFDAGHBC operon encoding periplasmic nitrate reductase results from synergistic transcription activation by the Fnr and phospho-NarP proteins , acting in response to anaerobiosis and nitrate or nitrite , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19060147" target="_blank">19060147</a>]<br></li>
<li>Expression of the E. coli napFDAGHBC operon is positively regulated in response to anaerobiosis by Fnr and by the presence of nitrate and/or nitrite by NarP ( 5 , 7 , 26 , 34 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12029041" target="_blank">12029041</a>]<br></li>
<li>One obvious difference between the synthetic promoter and the napF operon promoter is that transcription from the latter is also activated by the NarP protein bound to a site centered at position 44.5 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=13129959" target="_blank">13129959</a>]<br></li>
<li>It is quite likely that NarL and/or NarP can activate transcription directly when bound at other locations ( such as at position ¹ 74.5 at the nrf operon promoter and at position ¹ 44.5 at the nap -- ccm operon promoter ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9484902" target="_blank">9484902</a>]<br></li>
<li>For the nirBDC and nrfABCDEFG operons , Fnr protein , bound near position 41.5 , activates transcription maximally only when phospho-NarL or - NarP protein is bound further upstream to block inhibition by other proteins ( 2 , 8 , 59 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17720788" target="_blank">17720788</a>]<br></li>
<li>Regulated yeaR-yoaG operon expression required no more than 62 nt upstream of the transcription initiation site , which included the phylogenetically conserved regulatory elements : a site for binding phospho-NarL and - NarP activators and an overlapping site for binding the NsrR repressor ( Fig. 1A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17720788" target="_blank">17720788</a>]<br></li>
<li>Transcription from the Haemophilus inﬂuenzae napF ( napFHi ) operon control region is activated by Fnr and phos-pho-NarP or phospho-NarL in E. coli ( 65 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19060147" target="_blank">19060147</a>]<br></li>
<li>The transcription factors NarL and NarP are not essential for flrd expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12586421" target="_blank">12586421</a>]<br></li>
<li>The inability of NarL to activate napF operon expression led us to hypothesize that NarL is deﬁcient in the mechanism by which NarP activates transcription from the 244.5 binding site ( 9 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9696769" target="_blank">9696769</a>]<br></li>
<li>Phospho-NarL and - NarP activate transcription of the narG and napF operons , respectively , whereas phospho-NarL antagonizes napF operon transcription ( 18 , 22 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19060147" target="_blank">19060147</a>]<br></li>
<li>This suggests that Fnr and NarP may act synergistically to activate napF operon expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9696769" target="_blank">9696769</a>]<br></li>
<li>Results indicate that the downstream promoter ( P1 ) was responsible for Fnr - and NarP-regulated napF operon expression , whereas transcription from the upstream promoter ( P2 ) was activated only weakly by the Fnr protein and was inhibited by phospho-NarP and - NarL proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=13129959" target="_blank">13129959</a>]<br></li>
<li>The results support our hypothesis ( 9 ) that the Fnr and NarP proteins act together to cause maximum napF operon expression whereas the Fnr and NarL proteins do not .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9696769" target="_blank">9696769</a>]<br></li>
<li>The NarL protein does not signiﬁcantly activate napF operon expression in the presence of Fnr , whereas the NarP protein does .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9696769" target="_blank">9696769</a>]<br></li>
<li>Expression from a monocopy ( dcuS-lacZ ) operon fusion revealed three - to fourfold repression by nitrate ( Table 2 ) , virtually all of which can be attributed to the NarX-NarL regulatory system rather than the paralogous NarQ-NarP system ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15995204" target="_blank">15995204</a>]<br></li>
<li>The O1-nrfA construct exhibited strong repression of lacZ gene expression by either phos pho-NarL or phospho-NarP protein ( Table 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12644479" target="_blank">12644479</a>]<br></li>
<li>We do not understand the relative action of the NarL and NarP proteins in controlling F ( aeg-46 .5 - lacZ ) expression , and we do not understand the role of phosphorylation in this control .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8501030" target="_blank">8501030</a>]<br></li>
<li>In the context of regulated gene expression , the napF operon control region is unique among known Nar-regulated targets in both its architecture and its response to the transcriptional regulators NarL and NarP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11844760" target="_blank">11844760</a>]<br></li>
<li>Revised models for NarL and NarP control of nrfA and nirB gene expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11004182" target="_blank">11004182</a>]<br></li>
				<label class="TU_collapse_similarity" for="tu_similarity_54">
Revised model for NarL and NarP control of fdnG gene expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923080" target="_blank">12923080</a>]<br></label>
<input id="tu_similarity_54", type="checkbox">
<div class="TU_similar">
<li>Revised model for NarL and NarP control of fdhF gene expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923080" target="_blank">12923080</a>]<br></li></div>
<li>As additional NarL protein is activated in the presence of higher levels of nitrate ( ca. 7 mM ) , it fully displaces the NarP phosphate from the DNA and maximally induces fdnG expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923080" target="_blank">12923080</a>]<br></li>
<li>We do not know how or where the NarP protein acts in controllingfdnG operon expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8501030" target="_blank">8501030</a>]<br></li>
<li>One possible explanation for the enhanced nitrate-dependent repression of hya gene expression in the narP mutant is that normally NarL and NarP compete to repress hyb operon expression , with NarP binding being favoured .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10537212" target="_blank">10537212</a>]<br></li>
<li>In the case of hya operon regulation , both NarL and NarP appear to be equally effective at repressing expression in the presence of nitrate ( see Table 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10537212" target="_blank">10537212</a>]<br></li>
<li>The NarL and NarP proteins were required for full nitrate induction of narXL operon expression , whereas the nitrate repression of narP and narQ expression was mediated solely by the NarL protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7601854" target="_blank">7601854</a>]<br></li>
<li>Most of the nitrate and nitrite induction of the fdnG operon is mediated by NarL , although NarP can weakly induce expression ( 23 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7601854" target="_blank">7601854</a>]<br></li>
<li>The phospho-NarL and phospho-NarP proteins are not direct activators of nirB operon expression , but rather counter the effects of negative regulatory proteins ( 39 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12644479" target="_blank">12644479</a>]<br></li>
<li>During growth with nitrate , phospho-NarL protein ( but not phospho-NarP protein ) binds also to lower-afﬁnity heptamers at positions 50 and 22 ( Fig. 1 ) to repress nrfA operon expression ( 7 , 34 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12644479" target="_blank">12644479</a>]<br></li>
<li>Expression of the nirB operon differs from nrfA in that NarL and NarP are reported to activate nirB expression in response to nitrate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11004182" target="_blank">11004182</a>]<br></li>
<li>One of the more striking points in each model is that the NarP protein is nonessential ( i.e. , not a major player ) and that it can function as an antagonist of NarL in its ability to activate nirB or nrfA operon expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11004182" target="_blank">11004182</a>]<br></li>
<li>Furthermore , in the absence of NarL , dcuB is repressed fourfold by nitrate in a manner that appears to be NarP independent , and , in addition , dcuB expression is reduced by glucose in a fashion that is mostly CRP independent .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9852003" target="_blank">9852003</a>]<br></li>
<li>Second , NarL can repress nrfA-lacZ expression when nitrate is present , whereas NarP can not ( compare the NarL1 NarP2 strain to the NarL2 NarP1 strain and to the wild-type strain ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11004182" target="_blank">11004182</a>]<br></li>
<li>The expression of dcuC * 9-9lacZ was increased by a factor of about 2.2 compared to that in the wild type , but the responses to oxygen , nitrate , and the regulators FNR , NarL , and NarP were comparable to those in the wild type ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10368146" target="_blank">10368146</a>]<br></li>
<li>In the nar regulatory system , it is ultimately the concentration of active ( phosphorylated ) NarL and NarP which determines the level of expression of target genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7601854" target="_blank">7601854</a>]<br></li>
<li>Another example is the NarL / NarP - and FNR-stimulated expression of nirB and nrf genes , which are repressed by Fis and IHF ( Browning et al. , 2002 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15066032" target="_blank">15066032</a>]<br></li>
<li>2 napF Fnr , NarP , and NarL Regulation of Escherichia coli K-1 ( Periplasmic Nitrate Reductase ) Operon Transcription In Vitro ANDREW J. DARWIN ,1 † EVA C. ZIEGELHOFFER ,2 ‡ PATRICIA J. KILEY ,3 AND VALLEY STEWART1 * Section of Microbiology , Cornell University , Ithaca , New York 14853,1 and Departments of Bacteriology2 and Biomolecular Chemistry ,3 University of Wisconsin , Madison , Wisconsin 53706 Received 16 March 1998/Accepted 17 June 1998 The expression of several Escherichia coli operons is activated by the Fnr protein during anaerobic growth and is further controlled in response to nitrate and nitrite by the homologous response regulators , NarL and NarP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9696769" target="_blank">9696769</a>]<br></li>
<li>Transcription of the narG and napF operons is activated during anaerobic growth by the oxygen-responsive Fnr protein ( 36 ) and is controlled further by the nitrate - and nitrite-respon-sive NarX-NarL and NarQ-NarP two-component regulatory systems ( 69 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19060147" target="_blank">19060147</a>]<br></li>
<li>For example , the nitrate reductase genes narGHJI are regulated exclusively by NarL and are not responsive to NarP as observed here for cydD , while nitrate induction of the fdnGHI operon is regulated predominantly by NarL but is also responsive in part to NarP ( 33 , 39 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9335308" target="_blank">9335308</a>]<br></li>
<li>It is clear that E. coii contains a family of anaerobically regulated promoters where expression is dependent on at least two transcription factors , FNR and NarL/NarP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7854119" target="_blank">7854119</a>]<br></li>
<li>Like the dmsABC operon , these operons are regulated in response to nitrate availability , in a molybdate-dependent manner , by the NarQ/NarX/NarL / NarP two-component system ( reviewed in Gunsalus , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466267" target="_blank">9466267</a>]<br></li>
<li>Expression of some operons is controlled by both NarL and NarP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7601854" target="_blank">7601854</a>]<br></li>
<li>These operons are likely to be regulated by NarL and either NarP , as is the case of hcp , or other transcription factors in response to nitrate or other stimuli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16377617" target="_blank">16377617</a>]<br></li>
<li>g001 in denitrifying and ammonifying c-proteobacteria , the nitrate / nitrite signal is processed by the two-component sensorregulator NarX-NarL and its paralog NarQ-NarP in E. coli that control the respiratory nitrate reductase operon nar and the nitrite ammonifying loci nir and nrf [ 19 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16261196" target="_blank">16261196</a>]<br></li>
<li>Both NarL and NarP activate the expression of nitrate and nitrite catabolic genes and repress other anaerobic respiratory systems like fumarate reductase ŽFrd .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9230919" target="_blank">9230919</a>]<br></li>
<li>Nitrate and nitrite regulate anaerobic respiratory gene expression through the NarX-NarL and NarQ-NarP two-component systems .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17720788" target="_blank">17720788</a>]<br></li>
<li>Therefore , the expression of YdeP might be induced in response to varieties of the environmental signal such as an acidic condition ( through EvgSA ) , in the presence of nitrate/nitrite ( through NarXQ-NarL and NarXQ-NarP ) or in the presence of formate and ace-tate ( through BarA-UvrY ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26010043" target="_blank">26010043</a>]<br></li>
<li>Finally , in contrast to the well characterized effects of nitrate , nitrite , NarL , and NarP at the nrf and nap promoters , which are activated by NarP but antagonized or repressed by NarL , the dcuB-fumB operon was strongly repressed by both nitrate and nitrite , suggesting that it might provide the first example of a promoter that is repressed by NarP as well as by NarL .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16377617" target="_blank">16377617</a>]<br></li>
<li>Integration of three signals at the Escherichia coli nrf ole f Summary Expression from the Escherichia coli nrf operon promoter is activated by the anaerobically triggered transcription factor , FNR , and by the nitrate/nitrite ioncontrolled response regulators , NarL or NarP , but is repressed by the IHF and Fis proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15978080" target="_blank">15978080</a>]<br></li>
<li>Conversely , synthesis of the NADH-dependent nitrite reductase , NirBD , is induced by both nitrate and nitrite ; transcription of the nir operon is equally activated by both NarL and by NarP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16377617" target="_blank">16377617</a>]<br></li>
<li>Nitrite and nitrate regulation of nir and nrf operon expression is mediated by the NarL and NarP proteins ( Tyson et al. 1993 , 1994 ) : whilst transcription from the nir promoter is induced by both nitrite and nitrate , transcription of the nrf operon is induced by nitrite but repressed by nitrate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9382703" target="_blank">9382703</a>]<br></li>
<li>Enzymes implicated in NO production in the E. coli cytoplasm The NarL-activated narGHJI operon is strongly induced during anaerobic growth in the presence of high concentrations of nitrate , whereas the nrf operon is repressed by nitrate-activated NarL , but induced by nitrite - or nitrateactivated NarP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22092912" target="_blank">22092912</a>]<br></li>
<li>Consequently , some promoters , for example one of the two napF promoters regulating expression of the periplasmic nitrate reductase operon [ 11 ] , and the nrfA promoter , are activated by NarP in response to nitrite or low concentrations of nitrate , but repressed by NarL when nitrate is abundant .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16417494" target="_blank">16417494</a>]<br></li>
<li>Transcription is also induced by NarL and NarP in response to nitrite , but is repressed by NarL in the presence of nitrate [ 11,18 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10548535" target="_blank">10548535</a>]<br></li>
<li>Transcription initiation at the nrfA promoter is activated by FNR in the absence of oxygen and induced further by NarL and NarP in response to low concentrations of nitrate or to nitrite ( 35 , 44 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17449618" target="_blank">17449618</a>]<br></li>
<li>The yccM gene was induced 2.8-fold by nitrite compared with only 1.6-fold by nitrate ( compare Table 3 , 4th and 6th col-umns ) , suggesting that its promoter might be activated by NarP rather than by NarL .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16377617" target="_blank">16377617</a>]<br></li>
<li>Although fdhF expression was reported to be activated by formate and repressed by nitrate , neither NarL nor NarP was tested to determine its role in nitrate - or nitrite-dependent fdhF gene expression ( 21 , 29 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923080" target="_blank">12923080</a>]<br></li>
				</div></ul></div><br>
	</div>
</body>
</html>