<!doctype html>
<style>
@charset "utf-8";
body {
	height: 100%;
	margin: 0;
	padding: 0;
}

.banner {
	position: fixed;
	top: 0;
	width: 100%;
	height: 65px;
	background: gray;
	line-height: 65px;
}
.spacer {
	height: 10px;
}

.main-content {
	padding: 5%;
}

.collapse_ACT {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-style: none solid none none;
	border-color: #B4045F;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_ACT + input{
	display: none;
}
.collapse_ACT + input + div{
	display: none;
}
.collapse_ACT + input:checked + div{
	display: block;
}
.ACT {
	text-align: justify;
	text-justify: inter-word;
}
.ACT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B4045F;
}
.ACT_collapse_similarity:hover{
	background: #e5d3dc;
}
.ACT_collapse_similarity + input {
	display: none;
}
.ACT_collapse_similarity + input + div{
	display: none;
}

.ACT_collapse_similarity + input:checked + div{
	display: block;
}
.ACT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.ACT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B4045F;
	padding-left: 5px;
}
.ACT_collapse_similarity:hover{
	background: #e5d3dc;
}
.ACT_collapse_similarity + input {
	display: none;
}
.ACT_collapse_similarity + input + div{
	display: none;
}
.ACT_collapse_similarity + input:checked + div{
	display: block;
}
.ACT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.ACT_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #B4045F;
}
.ACT_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_DOM {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-style: none solid none none;
	border-color: #B404AE;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_DOM + input{
	display: none;
}
.collapse_DOM + input + div{
	display: none;
}
.collapse_DOM + input:checked + div{
	display: block;
}
.DOM {
	text-align: justify;
	text-justify: inter-word;
}
.DOM_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B404AE;
	padding-left: 5px;
}
.DOM_collapse_similarity:hover{
	background: #e8d7e7;
}
.DOM_collapse_similarity + input {
	display: none;
}
.DOM_collapse_similarity + input + div{
	display: none;
}
.DOM_collapse_similarity + input:checked + div{
	display: block;
}
.DOM_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}.DOM_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #B404AE;
}
.DOM_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_EVO {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #7401DF;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_EVO + input{
	display: none;
}
.collapse_EVO + input + div{
	display: none;
}
.collapse_EVO + input:checked + div{
	display: block;
}
.EVO {
	text-align: justify;
	text-justify: inter-word;
}
.EVO_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #7401DF;
	padding-left: 5px;
}
.EVO_collapse_similarity:hover{
	background: #e2d7ed;
}
.EVO_collapse_similarity + input {
	display: none;
}
.EVO_collapse_similarity + input + div{
	display: none;
}
.EVO_collapse_similarity + input:checked + div{
	display: block;
}
.EVO_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.EVO_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #7401DF;
}
.EVO_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_RP {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #0174DF;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_RP + input{
	display: none;
}
.collapse_RP + input + div{
	display: none;
}
.collapse_RP + input:checked + div{
	display: block;
}
.RP {
	text-align: justify;
	text-justify: inter-word;
}
.RP_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #0174DF;
	padding-left: 5px;
}
.RP_collapse_similarity:hover{
	background: #c9d7e2;
}
.RP_collapse_similarity + input {
	display: none;
}
.RP_collapse_similarity + input + div{
	display: none;
}
.RP_collapse_similarity + input:checked + div{
	display: block;
}
.RP_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.RP_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #0174DF;
}
.RP_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_SIT {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #0489B1;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_SIT + input{
	display: none;
}
.collapse_SIT + input + div{
	display: none;
}
.collapse_SIT + input:checked + div{
	display: block;
}
.SIT {
	text-align: justify;
	text-justify: inter-word;
}
.SIT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #0489B1;
	padding-left: 5px;
}
.SIT_collapse_similarity:hover{
	background: #d0dfe2;
}
.SIT_collapse_similarity + input {
	display: none;
}
.SIT_collapse_similarity + input + div{
	display: none;
}
.SIT_collapse_similarity + input:checked + div{
	display: block;
}
.SIT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.SIT_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #0489B1;
}
.SIT_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_TU {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #04B486;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_TU + input{
	display: none;
}
.collapse_TU + input + div{
	display: none;
}
.collapse_TU + input:checked + div{
	display: block;
}
.TU {
	text-align: justify;
	text-justify: inter-word;
}
.TU_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #04B486;
	padding-left: 5px;
}
.TU_collapse_similarity:hover{
	background: #d0e2de;
}
.TU_collapse_similarity + input {
	display: none;
}
.TU_collapse_similarity + input + div{
	display: none;
}
.TU_collapse_similarity + input:checked + div{
	display: block;
}
.TU_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.TU_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #04B486;
}
.TU_similar > a {
	text-decoration: none;
	color: gray;
}

ul { 
	list-style-type: none;
	padding: 0;
	margin: 0;
}
ul li > a {
	text-decoration: none;
	color: gray;
}
ul li:hover, ul li:focus {
	background: #F0F0F0;
}

ul li:first-child span {
  border-left:none;  
}

</style>
<html>
<head>
<meta charset="utf-8">
<title>TF properties - NikR</title>
<meta name="theme-color" content="#ffffff">
</head>
<body>
	<div class="banner">
		<center><font color="white" size="16px" face="arial">NikR</font></center>
	</div>
	<div align="center" class="main-content">
		<div class="spacer">
	</div><br>			<label class="collapse_ACT" for="_1"><font size="5px">NikR transcriptional activity</font></label>
				<input id="_1" type="checkbox">
				<div class="ACT"><br><ul>				<label class="ACT_collapse_similarity" for="act_similarity_1">
Bahlawane , C. , Dian , C. , Muller , C. , Round , A. , Fauquant , C. , Schauer , K. , de Reuse , H. , Terradot , L. , and Michaud-Soret , I. ( 2010 ) Structural and mechanistic insights into Helicobacter pylori NikR activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20583753" target="_blank">20583753</a>]<br></label>
<input id="act_similarity_1", type="checkbox">
<div class="ACT_similar">
<li>( 33 ) Bahlawane , C. , Dian , C. , Muller , C. , Round , A. , Fauquant , C. , Schauer , K. , de Reuse , H. , Terradot , L. , and Michaud-Soret , I. ( 2010 ) Structural and mechanistic insights into Helicobacter pylori NikR activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23737072" target="_blank">23737072</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22971172" target="_blank">22971172</a>]<br></li></div>
<li>Dian , C. , Schauer , K. , Kapp , U. , McSweeney , S. M. , Labigne , A. , and Terradot , L. ( 2006 ) Structural Basis of the Nickel Response in Helicobacter pylori : Crystal Structures of HpNikR in Apo and Nickel-bound States , J. Mol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18193897" target="_blank">18193897</a>]<br></li>
<li>Dian , C. , Schauer , K. , Kapp , U. , McSweeney , S. M. , Labigne , A. , and Terradot , L. ( 2006 ) Structural basis of the nickel response in Helico-bacter pylori : Crystal structures of HpNikR in apo and nickel-bound states .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20583753" target="_blank">20583753</a>]<br></li>
<li>Conformation of High -- Affinity Sites in the Presence of Ni2 + or K + Since typical energy minimization routines find local energy minima in the vicinity of the starting structure , we performed molecular dynamics ( MD ) simulations of EcNikR with K + in the high-affinity sites to sample low-energy states over a larger region of conformational space .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19621966" target="_blank">19621966</a>]<br></li>
<li>Moreover , this co-crystal structure raises the question of what metal occupies the second site under physiological conditions : K + , which is present in the crystal structure , or Ni2 + , which has been proposed to-bind to low - as well as highaffinity sites on EcNikR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19621966" target="_blank">19621966</a>]<br></li>
<li>Conformation of High -- Affinity Sites in the Presence of Ni2 + or K + The high-affinity metal-binding sites on EcNikR have been well-characterized as squareplanar Ni2 + - binding sites ( Figure 1a ) .8,16,17 While this site is preferentially a Ni2 + - binding site , in vitro metal-binding studies have shown that the coordinating species are flexible and can rearrange under different conditions .8,17,18 In light of this , we explored whether highaffinity sites in EcNikR could rearrange to accommodate K + , which is present at much higher concentrations in the intracellular space than Ni2 + .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19621966" target="_blank">19621966</a>]<br></li>
<li>The localization of this K + effect to the second metal-binding sites is derived from the structure of the EcNikR-DNA complex , in which a K + ion from the crystallization buffer is modeled in this site .9 Although the binding of ions from the crystallization buffer is often an artifact , in this case the concentration of K + in the crystallization buffer ( 200 mM ) is slightly lower than that in E. coli ( 250 mM ) , indicating that the presence of K + in the second metal-binding sites of this structure could be physiologically relevant .9,14 If these second metal-binding sites are K + sites , then they would not be regulatory sites but rather structural sites in which the binding of abundant cellular monovalent ions affords additional stability to the DNA-bound conformation of EcNikR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19621966" target="_blank">19621966</a>]<br></li>
<li>Conformation of the Second Metal -- Binding Site in the Presence of Ni2 + or K + In the crystal structure of EcNikR bound to DNA , ligands that coordinate the ion in the second metal-binding site are arranged in an octahedral geometry ( Figure 1b ) , and energy minimization of this site does not significantly alter this geometry ( Figure 3a ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19621966" target="_blank">19621966</a>]<br></li>
<li>To explore whether the second metal-binding site can rearrange to adopt conformations that are favorable for Ni2 + , we replaced K + with Ni2 + in the crystal structure of EcNikR bound to DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19621966" target="_blank">19621966</a>]<br></li>
<li>In particular , metal-binding studies have shown that the affinity of EcNikR for DNA increases from nanomolar , when most stable when Ni2 + occupies high-affinity sites and that EcNikR binding to DNA is more favorable when the second site contains K + .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19621966" target="_blank">19621966</a>]<br></li>
<li>Electrostatic binding free energy calculations , however , suggest that EcNikR binding to DNA is more favorable when the second metal-binding site contains K + .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19621966" target="_blank">19621966</a>]<br></li>
<li>In the first reaction , EcNikR binds DNA with K + in the second metal-binding site , and the associated free energy difference is ΔGK .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19621966" target="_blank">19621966</a>]<br></li>
<li>aerobic to anaerobic anaerobic to aerobic ArgR CysB BirA DgsA MalT GadW MetJ NikR OxyR SoxR SoxS DcuR CueR FadR HcaR NtrC PhoB PurR SgrR TdcA TrpR ZraR ArcA CpxR CRP CusR FhlA Fis FNR Fur H-NS IHF IscR Lrp ModE NarL NrdR PdhR FlhDC GlcC EvgA GadX LeuO PspF RbsR steady states Figure 3 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22870390" target="_blank">22870390</a>]<br></li>
<li>Protein Expression and Purification -- E. coli strain BB101 ( F9 lacIq lac9 pro9/ara D ( lac-pro ) nal1 argE r am rif thi-1 slyD lDE3 ) transformed with pNIK103 was grown in LB broth to an A600 of 0.7 to 0.8 , and NikR expression was induced by addition of isopropyl-1-thio-b-D-galactopyr-anoside to a final concentration of 0.5 mM .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10787413" target="_blank">10787413</a>]<br></li>
<li>Delany , I. , Ieva , R. , Soragni , A. , Hilleringmann , M. , Rappuoli , R. , and Scarlato , V. ( 2005 ) In Vitro Analysis of Protein-Operator Interactions of the NikR and Fur Metal-Responsive Regulators of Coregulated Genes in Helicobacter pylori , J. Bacteriol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18193897" target="_blank">18193897</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_17">
Delany I , Ieva R , Soragni A , Hilleringmann M , Rappuoli R , Scarlato V. 2005 In vitro analysis of protein-operator interactions of the NikR and Fur metal-responsive regulators of coregulated genes in Helicobacter pylori .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17120142" target="_blank">17120142</a>]<br></label>
<input id="act_similarity_17", type="checkbox">
<div class="ACT_similar">
<li>Delany , I. , Ieva , R. , Soragni , A. , Hilleringmann , M. , Rappuoli , R. , and Scarlato , V. ( 2005 ) In vitro analysis of protein-operator interactions of the NikR and Fur metal-responsive regulators of coregulated genes in Helicobacter pylori .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20583753" target="_blank">20583753</a>]<br></li></div>
<li>Ernst FD , Kuipers EJ , Heijens A , et al. 2005 The nickelresponsive regulator NikR controls activation and repression of gene transcription in Helicobacter pylori .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17120142" target="_blank">17120142</a>]<br></li>
<li>Ernst , F. D. , Kuipers , E. J. , Heijens , A. , Sarwari , R. , Stoof , J. , Penn , C. W. , Kusters , J. G. , and van Vliet , A. H. M. ( 2005 ) The Nickel-Responsive Regulator NikR controls Activation and Repression of Gene Transcription in Helicobacter pylori , Infect .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18193897" target="_blank">18193897</a>]<br></li>
<li>Ernst , F. D. , Stoof , J. , Horrevoets , W. M. , Kuipers , E. J. , Kusters , J. G. , and van Vliet , A. H. M. ( 2006 ) NikR Mediates NickelResponsive Transcriptional Repression of the Helicobacter pylori Outer Membrane Proteins FecA3 ( HP1400 ) and FrpB4 ( HP1512 ) , Infect .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18193897" target="_blank">18193897</a>]<br></li>
<li>A thorough computer-assisted investigation indicates that the NikR sequence is also functionally related to the ferric ion uptake regulation ( Fur ) proteins from the four Vibrio species , V. cholerae , V. anguillarum , V. vulnificus , V. parahaemolyticus -- although the Smallest Sum Probability is too low to establish sequence relatedness between them ( DePina et al 1999 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10648098" target="_blank">10648098</a>]<br></li>
<li>The Smallest Sum Probability P ( N ) score obtained by sequence comparison between NikR and these Fur proteins ranges from 0.60 to 0.97 , which is too low to establish sequence relatedness between them .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9882686" target="_blank">9882686</a>]<br></li>
<li>Remarkably , a cluster of histidines in Fur around positions 86 -- 90 , which is thought to be involved in metal sensing , is conserved in E. coli NikR and in B. subtilis Zur and PerR ( but not in E. coli Zur which has not been aligned ; Patzer and Hantke 1998 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10648098" target="_blank">10648098</a>]<br></li>
<li>Sequences of the Fur proteins of Staphylococcus epidermis , Campylobacter jejuni , C. uppsaliensis , Neisseria men-ingitidis , N. gonorrhoeae , Bordetella pertussis , Ralstonia eutropha , Pseudomonas putida , P. aeruginosa , Legionella pneumophila , Klebsiella pneumoniae , Escherichia coli , Yersinia pestis , Vibrio anguillarum , V. cholerae , V. vulnificus , V. parahaemolyticus , Haemophilus ducreyi , Helicobacter pylori , Rhizobium leguminosarum , Synechococcus PCC7942 , and Bacillus subtilis , the Zur and PerR proteins of B. subtilis , are compared with the sequence of E. coli NikR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10648098" target="_blank">10648098</a>]<br></li>
<li>[ 13 ] J.S. Iwig , J.L. Rowe , P.T. Chivers , Nickel homeostasis in Escherichia coli e the rcnRercnA efﬂux pathway and its linkage to NikR function , Mol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21040754" target="_blank">21040754</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_27">
Nickel homeostasis in Escherichia coli -- the rcnR -- rcnA efflux pathway and its linkage to NikR function .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19703465" target="_blank">19703465</a>]<br></label>
<input id="act_similarity_27", type="checkbox">
<div class="ACT_similar">
<li>Nickel homeostasis in Escherichia coli -- the rcnR-rcnA efﬂux pathway and its linkage to NikR function .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21097627" target="_blank">21097627</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21665978" target="_blank">21665978</a>]<br></li></div>
<li>Wang , S. C. , Dias , A. V. , and Zamble , D. B. ( 2009 ) The `` metallo-specific '' response of proteins : A perspective based on the Escherichia coli transcriptional-regulator NikR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20583753" target="_blank">20583753</a>]<br></li>
<li>Leitch , S. , Bradley , M. J. , Rowe , J. L. , Chivers , P. T. , and Maroney , M. J. ( 2007 ) Nickel-Specific Response in the Transcriptional Regulator , Escherichia coli NikR , J. Am .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18193897" target="_blank">18193897</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_31">
Leitch , S. , Bradley , M. J. , Rowe , J. L. , Chivers , P. T. , and Maroney , M. J. ( 2007 ) Nickel-specific response in the transcriptional-regulator , Escherichia coli NikR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20583753" target="_blank">20583753</a>]<br></label>
<input id="act_similarity_31", type="checkbox">
<div class="ACT_similar">
<li>( 38 ) Leitch , S. , Bradley , M. J. , Rowe , J. L. , Chivers , P. T. , and Maroney , M. J. ( 2007 ) Nickel-specific response in the transcriptional-regulator , Escherichia coli NikR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23737072" target="_blank">23737072</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22971172" target="_blank">22971172</a>]<br></li></div>
<li>Regulation of High Affinity Nickel Uptake in Bacteria Ni21-DEPENDENT INTERACTION OF NikR WITH WILD-TYPE AND MUTANT OPERATOR SITES * Received for publication , March 16 , 2000 , and in revised form , April 27 , 2000 Published , JBC Papers in Press , April 27 , 2000 , DOI 10.1074 / jbc.M002232200 Peter T. Chivers ‡ and Robert T. Sauer § From the Department of Biology , Massachusetts Institute of Technology , Cambridge , Massachusetts 02139 Escherichia coli actively imports nickel via the ATP-dependent NikABCDE permease .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10787413" target="_blank">10787413</a>]<br></li>
<li>Physical Basis of Metal-Binding Specificity in Escherichia coli NikR Christine M. Phillips † , ⊥ , Paul S. Nerenberg ‡ , ⊥ , Catherine L. Drennan † , § , # , and Collin M. Stultz * ,   , ⊥ , ∇ Departments of Chemistry , Physics , Biology , and Electrical Engineering and Computer Science , Research Laboratory of Electronics , Howard Hughes Medical Institute , and Harvard -- MIT Division of Health Sciences and Technology , Massachusetts Institute of Technology , Cambridge , Massachusetts 02139 Abstract In Escherichia coli and other bacteria , nickel uptake is regulated by the transcription factor NikR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19621966" target="_blank">19621966</a>]<br></li>
<li>Article pubs.acs.org/biochemistry Nonspeciﬁc Interactions Between Escherichia coli NikR and DNA Are Critical for Nickel-Activated DNA Binding Sandra Krecisz , Michael D. Jones , and Deborah B. Zamble * Department of Chemistry , University of Toronto , Toronto , Ontario , Canada M5S 3H6 * S Supporting Information ABSTRACT : The Escherichia coli transcription factor NikR is responsible for nickel-mediated repression of the operon encoding the Nik uptake transporter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23737072" target="_blank">23737072</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22971172" target="_blank">22971172</a>]<br></li>
<li>Indeed , recent characterization of HpNikR revealed that the presence of excess nickel ions does not substantially increase the affinity of that protein for the ureA promoter ( 12 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16945905" target="_blank">16945905</a>]<br></li>
<li>A comparison between E. coli apo-NikR 1 Abbreviations : EXAFS , extended X-ray absorption fine structure ; HEPES , 4 - ( 2-hydroxyethyl ) -1 - piperazineethanesulfonic acid ; GuHCl , guanidine hydrochloride ; MBD , metal-binding domain of NikR , residues 49-133 ; NTA , nitrilotriacetic acid ; PAR , 4 - ( 2-pyridylazo ) - resorcinol ; RHH , ribbon-helix-helix domain of NikR , residues 1-48 ; rmsd , root-mean-square deviation ; Tris , tris ( hydroxymethyl ) ami-nomethane ; XANES , X-ray absorption near-edge spectroscopy .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18193897" target="_blank">18193897</a>]<br></li>
<li>To test this hypothesis , we examined the activities of several E. coli NikR mutants both in vitro and in vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23737072" target="_blank">23737072</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22971172" target="_blank">22971172</a>]<br></li>
<li>To investigate further how the double mutation modulates NikR activity , in vitro characterization of the puriﬁed NikR mutants was also performed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23737072" target="_blank">23737072</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22971172" target="_blank">22971172</a>]<br></li>
<li>The molecular masses of wild-type NikR and mutants were confirmed by electrospray ionization mass spectrometry ( ESI-MS ; Department of Chemistry , University of Toronto ) ( Mr-calc NikR = 15093.7 Da , Mr-obs NikR = 15094.0 Da , Mr-calc H48N = 15070.4 Da , Mr-obs H48N = 15070.0 Da , Mr-calc H92A = 15027.3 Da , Mr-obs H92A = 15028.0 Da , Mr-calc H110N = 15070.4 Da , Mr-obs H110N = 15070.0 Da , Mr-calc H48/110N = 15046.3 Da , Mr-obs H48/110N = 15046.0 Da , Mr-calc H125N = 15070.4 Da , Mr-obs H125N = 15070.0 Da , Mr-calc C128A = 15061.3 Da , Mr-obs C128A = 15062.0 Da ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20583753" target="_blank">20583753</a>]<br></li>
<li>Upon reaction of thiols with iodoacetamide , carboxamidomethylation results in a mass increase of 57 Da / cysteine ( Figure S3 ) , and the extent of modification of NikR under different Ni ( II ) conditions was determined by ESI-MS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20583753" target="_blank">20583753</a>]<br></li>
<li>Only the monomeric form of NikR ( 15094 Da ) is observed by ESI-MS under our experimental conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20583753" target="_blank">20583753</a>]<br></li>
<li>Replacing Q118 with alanine , which would disrupt the putative hydrogen bond interaction but not the potassium site , does result in a small but signi cant reduction in the activity of NikR in vivo , ﬁ suggesting that this residue plays a minor role in the allosteric effect of nickel as previously suggested by molecular dynamics simulations .39 However , Q118A NikR is still able to repress transcription from the nik promoter in response to nickel supplementation of the growth media , indicating that it is not required for the full function of the protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23737072" target="_blank">23737072</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22971172" target="_blank">22971172</a>]<br></li>
<li>Halfmaximal binding was calculated by fitting the data to a Langmuir equation with a variable Hill coefficient n , r = [ NikR ] / ( ( Kd ( app ) ) n n þ [ NikR ] n ) , where r is the fraction of DNA bound to NikR and Kd ( app ) is the concentration of protein required for 50 % binding ( 18 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20583753" target="_blank">20583753</a>]<br></li>
<li>Halfmaximal binding was calculated by ﬁtting the data to the Hill equation with a variable Hill coe cient n , r = [ NikR ] n fi / ( ( K n n d ( app ) ) + [ NikR ] ) , where r is the fraction of DNA bound to NikR and Kd ( app ) is the concentration of protein required for 50 % binding .14 Data were ﬁt using OriginPro8 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23737072" target="_blank">23737072</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22971172" target="_blank">22971172</a>]<br></li>
<li>Nickel-bound selenomethionine-labeled NikR was produced and purified as described in ref .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16945905" target="_blank">16945905</a>]<br></li>
<li>Chivers , P. T. , and Tahirov , T. H. ( 2005 ) Structure of Pyrococcus horikoshii NikR : Nickel sensing and implications for the regulation of DNA recognition .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20583753" target="_blank">20583753</a>]<br></li>
<li>Nickel coordination is regulated by the DNA-bound state of NikR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16159764" target="_blank">16159764</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_49">
( 18 ) Carrington , P. E. , Chivers , P. T. , Al-Mjeni , F. , Sauer , R. T. , and Maroney , M. J. ( 2003 ) Nickel coordination is regulated by the DNA-bound state of NikR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23737072" target="_blank">23737072</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22971172" target="_blank">22971172</a>]<br></label>
<input id="act_similarity_49", type="checkbox">
<div class="ACT_similar">
<li>Carrington , P. E. , Chivers , P. T. , Al-Mjeni , F. , Sauer , R. T. , and Maroney , M. J. ( 2003 ) Nickel coordination is regulated by the DNA-bound state of NikR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20583753" target="_blank">20583753</a>]<br></li>
<li>Carrington , P.E. , Chivers , P.T. , Al-Mjeni , F. , Sauer , R.T. & Maroney , M.J. Nickel coordination is regulated by the DNA-bound state of NikR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12970756" target="_blank">12970756</a>]<br></li></div>
<li>The MBD of NikR was expressed and purified on a Ni2 + - NTA column ( Qiagen , Valencia , CA ) , followed by incubation with EDTA and anion exchange chromatography , as previously described ( 8 , 16 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18193897" target="_blank">18193897</a>]<br></li>
<li>The disruption of this hydrogen bonding network in the apo or Zn2 + - bound form of NikR could play a role in determining the overall flexibility of the R3 helix observed crystallographically and by circular dichroism .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18193897" target="_blank">18193897</a>]<br></li>
<li>Although pNIK103 is constructed in a pET backbone ,15 leaky expression in the absence of chemical induction produces levels of wild-type NikR that are suficient to repress Pnik-lacZ upon exposure of the bacteria to 1 μM nickel ( Figure 2 ) .15,26 C95A NikR , with a mutation in the high-afinity nickel-binding site ,18 serves as a negative control .26 Signiﬁcantly higher β-galactosidase activity ( p < 0.05 ) was detected in extracts of cells expressing C95A NikR grown in the absence of nickel supplementation compared to cells expressing wild-type NikR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23737072" target="_blank">23737072</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22971172" target="_blank">22971172</a>]<br></li>
<li>In support of this hypothesis , preliminary experiments demonstrated that inclusion of the nickel chelator dimethylglyoxime in the growth media of cells expressing wild-type NikR resulted in an increase in β-galactosidase activity ( data not shown ) , although a decrease in cell growth was also noted .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23737072" target="_blank">23737072</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22971172" target="_blank">22971172</a>]<br></li>
<li>Finally , the increase in β-galactosidase activity observed in extracts from cells expressing C95A NikR upon nickel supplementation of the media was also observed in the absence of any pNIK103 plasmid ( data not shown ) , suggesting that this effect is independent of NikR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23737072" target="_blank">23737072</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22971172" target="_blank">22971172</a>]<br></li>
<li>Strains containing pPC181 , the Pnik-lacZ fusion , and pNIK103 ( 9 ) , which provides a low level of NikR expression in the absence of any inducer , were inoculated in the deﬁned media to a starting OD600 of 0.0001 and grown 14 to 16 h at 37 ° C in capped microcentrifuge tubes with no headspace .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16159764" target="_blank">16159764</a>]<br></li>
<li>Cells lacking both NarLX and functional NikR showed constant high levels of Pnik-lacZ , suggesting that NarL and NikR are sufﬁcient to account for the Pnik repression observed under the conditions tested here ( Fig. 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16159764" target="_blank">16159764</a>]<br></li>
<li>Five of six operator mutants also resulted in increased levels of b-galactosidase expression in the absence of added Ni21 , suggesting that these mutations may increase intrinsic promoter strength and/or abolish repression by NikR that occurs in the absence of Ni21 or in the presence of very low levels of Ni21 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10787413" target="_blank">10787413</a>]<br></li>
<li>For example , nickel-dependent stabilization of the loop/helix α3 was also observed by X-ray crystallography in several subunits of NikR from Pyrococcus horikoshii , but this NikR homologue does not have basic residues at the equivalent positions .34 As an alternative , the authors predicted hydrogen bonding between a backbone amide nitrogen of the protein loop and the DNA , as observed in other types of transcription factors .42 Furthermore , in vitro analysis of NikR from H. pylori demonstrated that mutation of the corresponding Q76 and R77 to alanine weakened complex formation with one of its recognition sequences by several orders of magnitude ,33 although DNA binding could still be detected .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23737072" target="_blank">23737072</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22971172" target="_blank">22971172</a>]<br></li>
<li>RcnA inactivation affects hydrogenase-3 function via an increase in NikR activity Hydrogenase-3 activity can be assayed qualitatively on solid media under anaerobic conditions using MacConkey agar base supplemented-with-glucose and formate ( Wu and Mandrand-Berthelot , 1986 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16956381" target="_blank">16956381</a>]<br></li>
<li>The activities of these proteins are controlled in some manner by a small mol-ecule : FNR ( positive ) is inactivated by O2 , NarL ( negative ) is phosphorylated by NarX in the presence of NO3 , NikR ( negative ) is activated by Ni2 , and NikR function is inhibited by one or more formate-inducible hydrogenase assembly components .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16159764" target="_blank">16159764</a>]<br></li>
<li>They also suggest that NikR is active when nitrate is present in the growth media but that its function is somehow inhibited under conditions that favor the expression of hydrogenase isozymes and their corresponding assembly proteins ( Fig. 2B ) , such as the presence of formate in the growth medium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16159764" target="_blank">16159764</a>]<br></li>
<li>Rather , NikR function seems to be inhibited by components of the hydrogenase assembly pathway .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16159764" target="_blank">16159764</a>]<br></li>
<li>Interestingly , RcnA affects NikR function and is necessary for hydrogenase activity in the presence of NikR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16956381" target="_blank">16956381</a>]<br></li>
<li>While the specificity of a nickel-regulatory protein for nickel makes logical sense , if there is no free Cu2 + in the cell there would be no evolutionary pressure for NikR to have evolved such that Cu2 + would not activate it ( 39 , 40 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18193897" target="_blank">18193897</a>]<br></li>
<li>Current research suggests that any free copper in the cell is reduced to Cu + ( 39 , 40 ) , which would negate any evolutionary pressure for NikR to be able to distinguish between Ni2 + and Cu2 + .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18193897" target="_blank">18193897</a>]<br></li>
<li>EXAFS and XANES studies by the Maroney laboratory have indicated that Ni2 + and Cu2 + ions bind to NikR in square planar environments ( 18 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18193897" target="_blank">18193897</a>]<br></li>
<li>RcnA could also be important in the efflux of Ni2 + after the turnover of hydrogenase-3 large subunit , preventing access by NikR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16956381" target="_blank">16956381</a>]<br></li>
<li>Nickel ions in pool B enter the cell through an unidentiﬁed pathway and are available to other Ni-requiring enzymes , such as Glx I , as well as the Ni2 + - efflux protein RcnA , and the regulatory proteins RcnR and NikR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16956381" target="_blank">16956381</a>]<br></li>
<li>It is not possible , based on the data presented here , to differentiate between the two NikR activation mechanisms described above .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16159764" target="_blank">16159764</a>]<br></li>
<li>NikR -- operator complex structure and the mechanism of repressor activation by metal ions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18586269" target="_blank">18586269</a>]<br></li>
<li>( 13 ) Schreiter , E. R. , Wang , S. C. , Zamble , D. B. , and Drennan , C. L. ( 2006 ) NikR − operator complex structure and the mechanism of repressor activation by metal ions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23737072" target="_blank">23737072</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22971172" target="_blank">22971172</a>]<br></li>
<li>This observation is consistent with solution experiments demonstrating that upon loading NikR with stoichiometric nickel K64 and R65 become protected from limited tryptic proteolysis22 and that nickel induces an increase in α-helicity in the MBD that can be observed by circular dichroism ( CD ) spectroscopy .16,17,23 A functional role for the nickel-dependent ordering of this region in the center of the protein was suggested by the structure of Ni ( II ) - NikR bound to DNA .13 In addition to the expected sequence-speciﬁc contacts between the DBDs and bases in the major groove of the operator half sites , there appear to be nonspeciﬁc electrostatic interactions between K64 and R65 of two of the MBDs and the DNA phosphodiester backbone ( Figure 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23737072" target="_blank">23737072</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22971172" target="_blank">22971172</a>]<br></li>
<li>Helix α3 was removed in three apo NikR molecules because of high B-factors and a lack of interpretable electron density in omit maps .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12970756" target="_blank">12970756</a>]<br></li>
<li>Operator Binding -- In electrophoretic mobility shift experiments performed in the presence of 50 mM Ni21 , purified NikR half-maximally bound to a 100-base pair DNA fragment containing the nik promoter-operator region at a protein concentration of roughly 15 pM ( Fig. 3 , a and c ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10787413" target="_blank">10787413</a>]<br></li>
<li>Under these conditions , NikR half-maximally bound to the nik operator at a concentration of roughly 15 pM .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10787413" target="_blank">10787413</a>]<br></li>
<li>The next set of experiments was performed in the presence of an oligonucleotide containing the sequence from the nik promo-ter that is recognized by NikR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20583753" target="_blank">20583753</a>]<br></li>
<li>NikR or the indicated NikR mutant protein ( 10 pM ) was incubated with the 133-bp oligonucleotide containing the nik recognition sequence in the presence of 35 μM NiSO4 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20583753" target="_blank">20583753</a>]<br></li>
<li>Tight control by NikR is achieved through the permanent , basal level synthesis of the repressor from the nikR promoter , enabling the cells to respond immediately to changes in the Ni status .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10648098" target="_blank">10648098</a>]<br></li>
<li>Expression of the nik operon is controlled by the Ni-responsive repressor NikR , which shows functional similarity to the ferric ion uptake regulator Fur .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10648098" target="_blank">10648098</a>]<br></li>
<li>Perspectives The transcription of nik genes is stimulated by the global regulator FNR under anoxic conditions , and negatively controlled by a new repressor protein , NikR , responding specifically to Ni .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10648098" target="_blank">10648098</a>]<br></li>
<li>This observation suggests that enough nickel was present in the unsupplemented media to partially activate wild-type NikR , even though precautions were taken to minimize trace nickel contamination .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23737072" target="_blank">23737072</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22971172" target="_blank">22971172</a>]<br></li>
<li>Ni2 + - dependent interaction of NikR with wild-type and mutant operator sites , J. Biol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18193897" target="_blank">18193897</a>]<br></li>
<li>Chivers , P.T. & Sauer , R.T. Regulation of high affinity nickel uptake in bacteria -- Ni2 + - dependent interaction of NikR with wild-type and mutant operator sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12524532" target="_blank">12524532</a>]<br></li>
<li>This mutant bound NikR with wild-type affinity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10787413" target="_blank">10787413</a>]<br></li>
<li>Transcriptional repression by wild-type and mutant NikR proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23737072" target="_blank">23737072</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22971172" target="_blank">22971172</a>]<br></li>
<li>SUPPORTING INFORMATION AVAILABLE Table S1 containing the sequences of PCR primer for mutagenesis as well as the 54-bp oligonucleotide containing the nik recognition sequence , Figure S1 presenting XAS data of the highaffinity Ni ( II ) site in the presence and absence of DNA , Figure S2 showing a nickel release experiment , Figure S3 showing the chemical modification reactions used and Figure S4 showing the MS spectra following carboxamidomethylation of NikR in the absence of DNA , Figures S5 and S6 presenting the CD , GFC , and EGTA nickel competitions of WT and mutant NikR proteins , Figure S7 presenting the DNase footprinting of holo-C128A NikR , Figure S8 presenting the MS spectra following carboethoxylation of NikR and Figure S9 presenting the representative MS/MS data of peptide 38-65 , and Figures S10 and S11 presenting nickel titration and the mobility shift assay data of H48/110N , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20583753" target="_blank">20583753</a>]<br></li>
<li>It is unclear why the K64A or R65A NikR − DNA complexes were not observed by EMSA , given that DNA binding was observed in the DNase footprinting experiment , although discrepancies between the two types of DNA-binding assays have been observed in the past .14,15,24 It could be due to aggregation of the proteins in the presence of the excess metal used in the EMSA conditions , as 14,19 previously reported , but no evidence of aggregation was observed in these experiments .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23737072" target="_blank">23737072</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22971172" target="_blank">22971172</a>]<br></li>
<li>Another assay that is often used to monitor the DNA-binding activity of NikR is the electrophoretic mobility shift assay ( EMSA ) , which is performed in the presence of excess nickel .15 As expected , DNA binding to a 100-bp DNA probe containing the nik promoter was not observed by K64A/R65A NikR in this assay ( Figure S7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23737072" target="_blank">23737072</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22971172" target="_blank">22971172</a>]<br></li>
<li>In contrast , DNase footprinting assays revealed that up to 1 μM of the double K64A/R65A NikR mutant is unable to-bind to the DNA probe in the presence of stoichiometric nickel ( Figure 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23737072" target="_blank">23737072</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22971172" target="_blank">22971172</a>]<br></li>
<li>Furthermore , DNase footprinting assays were also conducted with K64A / R65A NikR in the presence of excess nickel and no binding to the DNA probe was detected ( Figure S6 ) , suggesting that the DNA-binding ability of the double mutant is completely abrogated in vitro as is the activity of the protein in vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23737072" target="_blank">23737072</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22971172" target="_blank">22971172</a>]<br></li>
<li>This is most probably caused by disturbed Ni ( II ) homeostasis when NikR was missing and the NikABC-transporter is constitutively switched on .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17120142" target="_blank">17120142</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_94">
Experiments with apo - and Ni ( II ) - bound NikR in the absence of DNA were performed under the same conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20583753" target="_blank">20583753</a>]<br></label>
<input id="act_similarity_94", type="checkbox">
<div class="ACT_similar">
<li>Experiments with apo - and Ni ( II ) - bound NikR in the absence of DNA were performed under the same conditions except that half as much DEPC was used .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20583753" target="_blank">20583753</a>]<br></li></div>
<li>Although the number of DEPC modifications on NikR seemed to decrease in the presence of excess Ni ( II ) and DNA , this experiment could not be reliably reproduced because the protein was not consistently observed by MS under these experimental conditions ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20583753" target="_blank">20583753</a>]<br></li>
<li>To confirm that the DNA-binding properties of C128A in the presence of stoichiometric Ni ( II ) are also similar to that of wild-type NikR , DNase footprinting experiments were performed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20583753" target="_blank">20583753</a>]<br></li>
<li>In the presence of stoichiometric Ni ( II ) C128A bound to the DNA and produced a footprint in the same location as wild-type NikR ( Figure S7 ) , around the nik transcription start , with a half-maximal binding affinity of 50 ( 10 nM , a KD comparable to that of wild-type NikR ( Table 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20583753" target="_blank">20583753</a>]<br></li>
<li>NikR binds to DNA in the presence of Ni ( II ) and represses the transcription of the genes encoding the NikABCDE uptake permease .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18505253" target="_blank">18505253</a>]<br></li>
<li>Although other amino acids , such as Ser and Tyr , can also react with exces FIGURE 2 : ESI mass spectra following the carboxamidomethylation of NikR in the presence of DNA and increasing Ni ( II ) concentrations .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20583753" target="_blank">20583753</a>]<br></li>
<li>FIGURE 4 : Carbethoxylation of NikR in the presence of DNA and increasing Ni ( II ) concentrations .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20583753" target="_blank">20583753</a>]<br></li>
<li>Carbethoxylated b11-24 ( which includes His48 only ) and carbethoxylated y4-1 FIGURE 4 : Carbethoxylation of NikR in the presence of DNA and increasing Ni ( II ) concentrations .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20583753" target="_blank">20583753</a>]<br></li>
<li>The concentration of wildtype NikR required for half-maximal DNA binding in the presence of excess Ni ( II ) is roughly 10 pM ( Figures 3 and 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20583753" target="_blank">20583753</a>]<br></li>
<li>FIGURE 6 : DNA binding of NikR and NikR mutants in the presence of excess Ni ( II ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20583753" target="_blank">20583753</a>]<br></li>
<li>Mutation of His48 and His110 to asparagines reduces significantly both NikR 's tendency to precipitate in the presence of excess Ni ( II ) and the affinity of the DNA-bound complex in the presence of excess Ni ( II ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20583753" target="_blank">20583753</a>]<br></li>
<li>Differential labeling of NikR in the presence of DNA and increasing concentrations of Ni ( II ) suggested that His48 and His110 may be involved in low-affinity Ni ( II ) binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20583753" target="_blank">20583753</a>]<br></li>
<li>In response to an increase in intracellular concentrations of Ni ( II ) , NikR binds to the nik operator/promoter and represses transcription of the nik operon , thus turning off Ni ( II ) import ( 5 , 12 , 13 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20583753" target="_blank">20583753</a>]<br></li>
<li>In the presence of nitrate ( II ) , NikABCDE synthesis is repressed by both the NarLX two-component system and NikR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16159764" target="_blank">16159764</a>]<br></li>
<li>NikR was modified with excess iodoacetamide in the presence of an oligonucleotide containing the nik recognition sequence and 0 , 1 , or 5 equiv of Ni ( II ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20583753" target="_blank">20583753</a>]<br></li>
<li>In the presence of stoichiometric Ni ( II ) , NikR binds to the nik promoter with a nanomolar KD ( 15 , 18 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20583753" target="_blank">20583753</a>]<br></li>
<li>Zambelli , B. , Danielli , A. , Romagnoli , S. , Neyroz , P. , Ciurli , S. , and Scarlato , V. ( 2008 ) High-affinity Ni binding selectively promotes 2þ binding of Helicobacter pylori NikR to its target urease promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20583753" target="_blank">20583753</a>]<br></li>
<li>The relative mass difference of three carbethoxy groups between the apo - and Ni ( II ) - loaded forms of NikR is consistent with the roles of His76 , His87 , and His89 as high-affinity Ni ( II ) - binding ligands .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20583753" target="_blank">20583753</a>]<br></li>
<li>The overall extent of derivitization of the whole protein increased in the presence of the higher concentrations of DEPC ( compare Figures S8 and 4A ) ; however , the relative mass difference of three carbethoxy groups between the apo - and Ni ( II ) - loaded forms of NikR was maintained ( Figure 4A ) , consistent with the roles of His76 , His87 , and His89 as high-affinity Ni ( II ) - binding ligands .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20583753" target="_blank">20583753</a>]<br></li>
<li>Bloom , S. L. , and Zamble , D. B. ( 2004 ) Metal-Selective DNA-Binding Response of Escherichia coli NikR , Biochemistry 43 , 10029-10038 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18193897" target="_blank">18193897</a>]<br></li>
<li>Benanti , E. L. , and Chivers , P. T. ( 2007 ) The N-terminal Arm of the Helicobacter pylori Ni2 + - dependent Transcription Factor NikR Is Required for Specific DNA Binding , J. Biol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18193897" target="_blank">18193897</a>]<br></li>
<li>The corresponding values for Arc ,34 MetJ ,35 CopG ,36 NikR ,37 omega ,38 and FitA39 bound to DNA span the ranges 8.0 -- 11.9 Å , 5.8 -- 8.1 Å , and 8.2 -- 11.7 Å , respectively , for these three residue positions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18586269" target="_blank">18586269</a>]<br></li>
<li>Copyright © 1999 The Protein Society NikR is a ribbon-helix-helix DNA-binding protein PETER T. CHIVERS and ROBERT T. SAUER Department of Biology , Massachusetts Institute of Technology , Cambridge , Massachusetts 02139 ~ Received April 15 , 1999 ; Accepted August 13 , 1999 !	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10595554" target="_blank">10595554</a>]<br></li>
<li>The side chain of Arg-119 , in a loop connecting the last - helix and - strand of NikR , extends between the phosphates of the DNA minor groove .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16945905" target="_blank">16945905</a>]<br></li>
<li>The NikR molecule makes initial contacts to DNA , stabilized via hydrogen bonds and electrostatic interactions between residues in the loop of the MBD and the phosphate backbone ( II f III ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18193897" target="_blank">18193897</a>]<br></li>
<li>In contrast , four-coordinate planar complexes undergo ligand exchange via a mechanism involving association of the incoming ligand , and therefore depend strongly on the nature of the nucleophile involved .68 Thus , the NikR high-affinity site , which is not easily accessible in the protein , is not prone to easy nickel release once the metal is bound .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18505253" target="_blank">18505253</a>]<br></li>
<li>Moreover , in all NikR structures where the high-affinity Ni2 + ion is appropriately coordinated by four ligands , this site is buried and not solvent exposed .6,8,9 In total , these observations suggest that K + - containing high-affinity sites are associated with large conformational changes that cause the protein to significantly deviate from the prototypical NikR structure .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19621966" target="_blank">19621966</a>]<br></li>
<li>For NikR , in vitro spectroscopic studies and metal binding experiments seem to contradict this basic principle , as they indicate that NikR binds a number of transition metals , including Ni2 + , Co2 + , Zn2 + , Cd2 + , and Cu2 + with significant affinity ( 16 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18193897" target="_blank">18193897</a>]<br></li>
<li>To investigate how different metals bind NikR and to explore the issue of specificity , we have determined the structures of the E. coli NikR metal-binding domain ( MBD ) 1 with Zn2 + and Cu2 + bound and with no metals bound .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18193897" target="_blank">18193897</a>]<br></li>
<li>Excess nickel enhances the affinity of EcNikR for its operator by at least 250-fold , and evidence supports the existence of a second nickel-binding site in the presence of operator DNA with micromolar to nanomolar affinity for nickel ( 9 , 11 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16945905" target="_blank">16945905</a>]<br></li>
<li>Results stoichiometric Ni2 + ions are bound to the high-affinity nickel-binding sites , to picomolar , when excess nickel is present .11,12 This increased DNA-binding affinity with excess nickel suggests the presence of one or multiple low-affinity Ni2 + - binding sites on EcNikR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19621966" target="_blank">19621966</a>]<br></li>
<li>Furthermore , R65 is in hydrogen-bonding proximity to Q118 , a residue that contributes a backbone amide carbonyl oxygen as a ligand in a potassium-binding site .13 This potassium is bound at the interface of the MBD and DBD and is critical for NikR − DNA complex formation ,24 so Q118 could serve as a link between nickel binding and an activated DNA-binding conformation of E. coli NikR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23737072" target="_blank">23737072</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22971172" target="_blank">22971172</a>]<br></li>
<li>When assayed in parallel , puriﬁed NikR bound 63Ni , as observed i the previous study , whereas puriﬁed RcnB exhibited no binding ( Fig. 3A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21665978" target="_blank">21665978</a>]<br></li>
<li>We demonstrate that high affinity binding of NikR to the operator in vitro is Ni21-dependent , identify the operator bases most important for NikR binding , and then show that NikR binding to this operator site is required for efficient Ni21-de-pendent repression of the nikABCDE operon in vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10787413" target="_blank">10787413</a>]<br></li>
<li>Does binding of NikR to the operator site identified here regulate nikABCDE expression in vivo ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10787413" target="_blank">10787413</a>]<br></li>
<li>Mutations in the GTATGA sequences reduce NikR binding affinity in vitro and reduce repression of a Pnik-lacZ fusion in vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10787413" target="_blank">10787413</a>]<br></li>
<li>Four of the operator mutants that severely reduced NikR binding in vitro were found to reduce Ni21-dependent repression of b-galactosidase expression in vivo ( Fig. 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10787413" target="_blank">10787413</a>]<br></li>
<li>Regulation of nikABCDE expression is positively controlled by FNR ( 44 ) and negatively controlled by NikR ( 9 ) , in both cases by direct protein binding to the nikABCDE promoter ( PnikABCDE or Pnik ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16159764" target="_blank">16159764</a>]<br></li>
<li>Our model implies that NikR becomes active after binding Ni in order to repress the transcription of nik .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10648098" target="_blank">10648098</a>]<br></li>
<li>By binding to an operator within Pnik , NikR represses nikABCDE transcription in the presence of Ni21 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10787413" target="_blank">10787413</a>]<br></li>
<li>The metal elements ( Na , Mn , Ni , Cu , Zn , Fe , and Mo ) are directly recognized by the DNA-binding TFs ( NhaR , MntR , NikR , CueR , ZntR/Zur , Fur , and ModE , respectively ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26010043" target="_blank">26010043</a>]<br></li>
<li>In the putative promoter ( 200 bp upstream of the pitA start codon ) , no DNA-binding sites for the following transcription factors are evident : MerR , CueR , ZntR ( all MerR family ) , Zur , NikR , CusR , ZraR , CpxR and ModE .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19054109" target="_blank">19054109</a>]<br></li>
<li>For example , although the isolated N-terminal domain of NikR protects the operator in footprinting experiments ( 7 ) , this binding is observed at concentrations 10,000-fold higher than those required for binding of intact NikR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10787413" target="_blank">10787413</a>]<br></li>
<li>These experiments support a model for NikR function in which the N-terminal domain binds DNA subsites and the C-terminal domain binds Ni21 and regulates operator binding via specific interactions between adjacently bound NikR dimers .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10787413" target="_blank">10787413</a>]<br></li>
<li>Moreover , removal of one of the operator subsites also reduced NikR binding dramatically , suggesting that some specific interactions between adjacently bound NikR dimers are necessary for high affinity operator binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10787413" target="_blank">10787413</a>]<br></li>
<li>Ni2 + binds to the high-affinity site of NikR , stabilizing the R3 helix and its preceding loop ( I f II ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18193897" target="_blank">18193897</a>]<br></li>
<li>Cu2 + - and Ni2 + - bound NikR have nanomolar affinity for the nik operon , while Zn2 + - NikR has a binding affinity 2 orders of magnitude lower for the operon ( 10 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18193897" target="_blank">18193897</a>]<br></li>
<li>The central high-affinity Ni2 + - binding sites , which are conserved among all NikR homologues , are sensory sites that specifically bind the scarce cellular Ni2 + ion , enabling binding to the nik promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19621966" target="_blank">19621966</a>]<br></li>
<li>The shaded bases correspond to the subsites that are critical for high affinity NikR binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10787413" target="_blank">10787413</a>]<br></li>
<li>The structures of apo - , 2 + 2 + Zn - , and Cu - MBD in combination with previously determined structures and DNA binding data provide further evidence that the ordering of the R3 helix is an important step in NikR 's recognition of DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18193897" target="_blank">18193897</a>]<br></li>
<li>The data presented here allow us to discuss how a metal 's ability to order the R3 helix corresponds to its ability to increase NikR 's affinity for DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18193897" target="_blank">18193897</a>]<br></li>
<li>Thus , this network could be involved in stabilizing both helix R3 and the overall structure of NikR for DNA binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18193897" target="_blank">18193897</a>]<br></li>
<li>Here we illustrate a correlation between the ability of a metal to order the R3 helix and that metal 's ability to induce DNA binding and use this information along with previous results to describe a mechanism for binding of NikR to DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18193897" target="_blank">18193897</a>]<br></li>
<li>Helix R3 , residues 65-78 ( Figure 1b , c ) , has been implicated in the mechanism of binding of NikR to DNA , due to the fact that nickel binding orders this helix and there are contacts made by residues of this helix and the preceding loop to DNA ( Figure 1c ) ( 20 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18193897" target="_blank">18193897</a>]<br></li>
<li>It is our proposal that nickel and copper 's ability to ligate this residue and in turn `` tie down '' the R3 helix and its preceding loop is related to the ability of Ni2 + - and Cu2 + - bound NikR 's ability to-bind DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18193897" target="_blank">18193897</a>]<br></li>
<li>While Zn2 + can bind NikR with some affinity , it does not induce the ordering of the R3 helix , and Zn2 + - NikR 's affinity for DNA is decreased compared to that of Ni2 + - NikR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18193897" target="_blank">18193897</a>]<br></li>
<li>Along with the previously published structure of apo-NikR , these structures allow us to evaluate functional proposals for how metal ions activate NikR , delineate the drastic conformational changes required for operator recognition , and describe the formation of a second metal-binding site in the presence of DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16945905" target="_blank">16945905</a>]<br></li>
<li>These structures allow us to evaluate the possible mechanisms of nickel-induced DNA binding , describe the conformational changes of NikR required for operator recognition , and investigate the formation of additional metal-binding sites in the presence of DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16945905" target="_blank">16945905</a>]<br></li>
<li>Although the existence of this second site can explain the increase in affinity of NikR for DNA in the presence of K ( Fig. 5B ) , it does not seem intuitive that replacing K with Ni2 , a metal less suited to this type of site , would further increase the affinity of NikR for DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16945905" target="_blank">16945905</a>]<br></li>
<li>First we address an alternative explanation for the apparent increase in DNA-binding affinity in the presence of extra metal , which is that there is only one metalbinding site on NikR but the metal site changes upon DNA binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20583753" target="_blank">20583753</a>]<br></li>
<li>The secondary metal-binding site , formed at the interface between the RHH and MBD domains in the DNA-bound conformation of NikR and occupied by potassium in our structure , adds an additional level of complexity to the function of this transcription factor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16945905" target="_blank">16945905</a>]<br></li>
<li>The high-affinity metalbinding sites have been well characterized as nickel-binding sites , which upon nickel binding induce a conformational change allowing NikR to-bind DNA .10,11 Interestingly , nickel binding does not appear to affect the conformation of the RHH DNA-binding domains that make specific contacts to DNA , instead altering the structure of the MBD that makes nonspecific DNA contacts ( Figure 1 ) .9 In particular , Ni2 + binding orders the α-helix 3 through its coordination of helical residue H76 ; conversely , H76 and α-helix 3 are disordered in the apo EcNikR structure .8 Crystallographic and mutagenesis data suggest that , like NikR 's high-affinity site , the second metal-binding sites are also important in modulating the affinity of NikR for DNA .9 The location of these sites at the interface of the MBD and RHH domains suggests that a metal in this site would stabilize the `` down-cis '' or the DNA-binding mode of EcNikR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19621966" target="_blank">19621966</a>]<br></li>
<li>The absence of a biphasic repression curve suggested that in vivo roles for the distinct Ni-NikR DNA complexes observed in vitro as a function of nickel ( 4 , 8 , 9 ) are not observable under these conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16159764" target="_blank">16159764</a>]<br></li>
<li>It is conceivable that the conditions under which we soaked the NikR -- DNA complex crystals somehow prevented exchange of nickel for potassium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16945905" target="_blank">16945905</a>]<br></li>
<li>Moreover , the solvent accessibilities of His48 and His110 do not appear to change in a comparison of the structures of the DNA-bound and DNA-free holo-NikR complexes ; therefore , we would not expect His48/110 to be protected due to the conformational change that occurs upon DNA binding of the protein loaded with stoichiometric nickel .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20583753" target="_blank">20583753</a>]<br></li>
<li>The absorbance increase was initially linear , indicating a binding site of submicromolar af-finity , and then saturated at a Ni21 concentration of roughly 1 nickel ion for each NikR subunit , or 2 ions bound for each NikR dimer ( 7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10787413" target="_blank">10787413</a>]<br></li>
<li>This portion of NikR is disordered in the absence of bound nickel ( Fig. 1 A ) , implying that positioning of these residues upon metal binding is of functional importance .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16945905" target="_blank">16945905</a>]<br></li>
<li>Our crystal structures of NikR in the unliganded ( apo ) , nickel-bound ( activated ) , and operator-bound forms provide insight into allosteric transitions upon ligand binding ( Fig. 1 ) and do not support such a mechanism .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16945905" target="_blank">16945905</a>]<br></li>
<li>However , other biochemical results demonstrated that H. pylori NikR does not seem to respond to more than stoichiometric nickel binding ( 28-33 ) , which suggests that the low-affinity Ni ( II ) - induced DNA-binding response of E. coli NikR is not conserved in all NikR orthologues .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20583753" target="_blank">20583753</a>]<br></li>
<li>This change is what is observed for both the high - and low-affinity Ni ( II ) - binding ligands of NikR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20583753" target="_blank">20583753</a>]<br></li>
<li>Whether this greater affinity is due to a specific low-affinity nickel site in NikR or to a general electrostatic effect is unknown .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18193897" target="_blank">18193897</a>]<br></li>
<li>Purified NikR binds two nickel ions per dimer with high affinity in a manner that gives rise to distinctive changes in both the UV absorbance and circular dichroism spectra , although the function of nickel binding to this site is unknown .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10787413" target="_blank">10787413</a>]<br></li>
<li>Previous experiments to measure the affinity of nickel for the `` low-affinity '' nickelbinding site demonstrated that nanomolar concentrations of excess nickel are sufficient to increase the affinity of NikR for its operator , even in the presence of 100 mM potassium ( 11 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16945905" target="_blank">16945905</a>]<br></li>
<li>Strong operator binding requires the continual presence of 20 -- 50 nickel , inmM dicating the presence of a low affinity nickel-binding site , and NikR dimers also contain two high affinity nickel-binding sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10787413" target="_blank">10787413</a>]<br></li>
<li>A low affinity interaction between NikR and 20 -- 50 mM Ni21 results in tight binding of NikR to the operator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10787413" target="_blank">10787413</a>]<br></li>
<li>Perhaps tetrameric NikR can form in vivo and in vitro without nickel , but is not stable as a tetramer in the presence of DNA without nickel , and thus is ineffective as a repressor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12970756" target="_blank">12970756</a>]<br></li>
<li>A sixcoordinate nickel site composed of O - or N-donor ligands , but lacking cysteine , forms when NikR binds to operator DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12524532" target="_blank">12524532</a>]<br></li>
<li>Wild type NikR binds with 35 nM affinity to operator DNA in the presence of 1:1 nickel .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12524532" target="_blank">12524532</a>]<br></li>
<li>Unfortunately , we were unable to crystallize the NikR -- DNA complex in the presence of excess nickel ions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16945905" target="_blank">16945905</a>]<br></li>
<li>Note that complete protection is not observed with 0.1 nM WT NikR in the presence of excess nickel because the DNA concentration is higher than the protein concentration .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20583753" target="_blank">20583753</a>]<br></li>
<li>As previously reported ( 4 , 11 ) , DNA binding is observed in this assay only if there is excess metal present , so experiments were performed with NikR loaded with stoichiometric amounts of nickel and in the presence of excess Ni2 or K .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16945905" target="_blank">16945905</a>]<br></li>
<li>Furthermore , in subsequent XAS analysis of NikR performed in the presence of nik DNA and excess metal a transformation of the high-affinity nickel site was not observed ( 25 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20583753" target="_blank">20583753</a>]<br></li>
<li>The high-affinity nickel-binding site within the MBD of NikR has the same coordination sphere in the NikR -- DNA complex as in the structures of nickel-activated EcNikR and PhNikR alone ( Fig. 8 , which is published as supporting information on the PNAS web site ) , with His-76 , His-87 , His-89 , and Cys-95 bound to the nickel in a square-planar geometry .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16945905" target="_blank">16945905</a>]<br></li>
<li>To understand how the high-affinity nickel binding site of NikR is able to accommodate these other metal ions , and to improve our understanding of NikR 's mechanism of binding to DNA , we have determined structures of the metalbinding domain ( MBD ) of NikR in the apo form and in complex with Cu2 + and Zn2 + ions and compared them with the previously published structures with Ni2 + .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18193897" target="_blank">18193897</a>]<br></li>
<li>Recently , it was demonstrated that NikR can bind a variety of divalent transition metal ions with high affinity and that stoichiometric amounts of these other ions can activate NikR to-bind operator DNA with a potency comparable to that of nickel ( 10 , 11 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16945905" target="_blank">16945905</a>]<br></li>
<li>Binding of nickel ions to the high-affinity sites of NikR and the subsequent increase in affinity for specific DNA therefore seems to be the primary conserved mechanism of function of the NikR family .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16945905" target="_blank">16945905</a>]<br></li>
<li>In a second model of NikR activation , nickel would exert its influence on gene transcription by formation or stabilization of the tetrameric form of NikR that is required for DNA binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12970756" target="_blank">12970756</a>]<br></li>
<li>These results provide a structural explanation for the nickel-specificity of NikR and illustrate novel aspects of a mechanism by which transcription factors regulate DNA-binding in response to metal binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12524532" target="_blank">12524532</a>]<br></li>
<li>How stoichiometric nickel binding activates DNA binding by NikR is not completely understood , but it is likely that the mechanism involves organization of stable structures in the metal-binding domains ( 19-21 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20583753" target="_blank">20583753</a>]<br></li>
<li>An unanticipated feature of the NikR structure is that the nickel coordination responds to DNA binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12524532" target="_blank">12524532</a>]<br></li>
<li>How nickel activates DNA binding by NikR is a question that remains to be resolved conclusively ( 21 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20583753" target="_blank">20583753</a>]<br></li>
<li>These ions appear to be optimally placed to enhance the DNA-binding activity of NikR because they would reinforce the conformation of the protein with the DNA-binding domains in the cis orientation , and it was possible that they act as placeholders for the low-affinity nickel sites ( 19 , 23 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20583753" target="_blank">20583753</a>]<br></li>
<li>Site-directed mutation of His87 , His89 , Cys95 or Glu97 in NikR to alanine eliminates high-affinity nickel binding and abolishes DNA binding but maintains stable protein folding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12524532" target="_blank">12524532</a>]<br></li>
<li>Taken together , the mutagenesis studies and XAS results strongly suggest that the side chains of His87 , His89 , Cys95 and Glu97 participate in high-affinity nickel binding by NikR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12524532" target="_blank">12524532</a>]<br></li>
<li>Finally , nickel binding could act to increase the affinity of NikR for operator DNA by decreasing the net negative charge of the regulatory C-domain tetramer , making an interaction with DNA more electrostatically favorable .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12970756" target="_blank">12970756</a>]<br></li>
<li>NikR repressor : high-afﬁnity nickel binding to the C-terminal domain regulates binding to operator DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16159764" target="_blank">16159764</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_191">
Chivers , P. T. , and Sauer , R. T. ( 2002 ) NikR repressor : High-affinity nickel binding to the C-terminal domain regulates binding to operator DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20583753" target="_blank">20583753</a>]<br></label>
<input id="act_similarity_191", type="checkbox">
<div class="ACT_similar">
<li>Chivers , P.T. & Sauer , R.T. NikR repressor : high-affinity nickel binding to the Cterminal domain regulates binding to operator DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12524532" target="_blank">12524532</a>]<br></li>
<li>Chivers , P.T. & Sauer , R.T. NikR repressor : high-affinity nickel binding to the C-termi-nal domain regulates binding to operator DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12970756" target="_blank">12970756</a>]<br></li></div>
<li>However , this type of rearrangement was not observed in the structure of NikR loaded with stoichiometric nickel in the absence of DNA ( 19 ) , and more subtle changes induced by the binding of the high-affinity nickel may be important for DNA binding ( 19 , 21 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20583753" target="_blank">20583753</a>]<br></li>
<li>As a result , NikR must also contain a low affinity nickel-binding site that regulates DNA binding affinity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10787413" target="_blank">10787413</a>]<br></li>
<li>Although binding of nickel to the high-affinity site of NikR does not directly alter the conformation of the RHH domains for DNA binding , it does have a very important short-range effect on the conformation of NikR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16945905" target="_blank">16945905</a>]<br></li>
<li>Here we report the structure of the high-affinity nickel-binding site in NikR and show that it responds dramatically to DNA binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12524532" target="_blank">12524532</a>]<br></li>
<li>We report here structural studies of the high-affinity nickel-binding site in NikR and evidence showing that its structure is sensitive to the DNA-bound state of NikR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12524532" target="_blank">12524532</a>]<br></li>
<li>UV-visible and CD spectra , the peaks attributed to nickel binding required the presence of both the metal ion and NikR and were absent following incubation with 50 mM EDTA for 12 h at room temperature .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10787413" target="_blank">10787413</a>]<br></li>
<li>Co-ordination could be achieved by tuning the nickel-binding affinities and/or kinetics of both repressors so that NikR binds Ni2 + preferentially .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16956381" target="_blank">16956381</a>]<br></li>
<li>Picomolar binding of NikR to its operator is observed with excess nickel in the 20 -- 50 µM range .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12524532" target="_blank">12524532</a>]<br></li>
<li>It seems likely that nickel has multiple roles in activating NikR for operator binding , and that a combination of changes in repressor properties is required for function .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12970756" target="_blank">12970756</a>]<br></li>
<li>Moreover , no intermediate species indicative of a dimer-DNA complex was observed for this or any other operator variant , suggesting that complexes between single NikR dimers and the operator are too unstable to be detected in this assay .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10787413" target="_blank">10787413</a>]<br></li>
<li>In contrast , neither metal affords tight DNA binding by D34A-NikR , and no shift indicative of a 4:1 NikR -- DNA complex is observed ( Fig. 5 ) , demonstrating that Asp-34 is critical for the DNA-binding response of NikR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16945905" target="_blank">16945905</a>]<br></li>
<li>A lower mobility band is observed in mobility-shift assay experiments with D34A NikR ( Fig. 5 ) and may be due to multiple NikR tetramers binding to the DNA with lower affinity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16945905" target="_blank">16945905</a>]<br></li>
<li>DTNB reacted with just one thiol in holo-NikR ( presumably Cys128 ) but reacted with both thiols when holo-NikR was bound to the half-site DNA fragment .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12524532" target="_blank">12524532</a>]<br></li>
<li>Binding of a cation such as K + to this site may serve to help lock NikR onto its DNA operator ( 20 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18193897" target="_blank">18193897</a>]<br></li>
<li>Mechanism of Binding of NikR to DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18193897" target="_blank">18193897</a>]<br></li>
<li>95 % binding of NikR to the resin and allowed efficient single step purification .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10787413" target="_blank">10787413</a>]<br></li>
<li>Nickel ion binding by NikR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10787413" target="_blank">10787413</a>]<br></li>
<li>Further research will determine whether the model of E. coli NikR binding to DNA can be extended to the activity of other homologues .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23737072" target="_blank">23737072</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22971172" target="_blank">22971172</a>]<br></li>
<li>Finally , NikR is a transcriptional-repressor and DNA-binding protein and thus shares common functions with other members of the b-a-a superfamily .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10595554" target="_blank">10595554</a>]<br></li>
<li>Abraham , L. O. , Li , Y. , and Zamble , D. B. ( 2006 ) The metal - and DNA-binding activities of Helicobacter pylori NikR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20583753" target="_blank">20583753</a>]<br></li>
<li>Li , Y. , and Zamble , D. B. ( 2009 ) pH-responsive DNA-binding activity of Helicobacter pylori NikR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20583753" target="_blank">20583753</a>]<br></li>
<li>E. coli can not synthesize adenosylcobalamin ( vitamin B12 ) de novo but has a specific transport pathway for this essential cofactor .17 -- 20 Cobalt ions are thought to enter the cell only through transporters with low specificity such as the magnesium transporter , CorA .20 -- 22 These observations suggest that Co ( II ) ions are regarded by the cell as toxic , one manifestation of which is its adverse effects on Fe -- S clusters in enzymes such as aconitase and ferrichrome reductase .23 Physiologically , the rcnR-rcnA module is functionally linked to the other nickel-dependent activities of the cell .9 At low extracellular nickel levels , low or transient expression of RcnA is required to prevent premature nikABCDE repression by NikR .9 This effect likely results from a competition between nickel efflux by RcnA and nickel-binding and transcriptional repression by NikR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18505253" target="_blank">18505253</a>]<br></li>
<li>In contrast to the single mutations , this double mutation completely abrogated the ability of NikR to repress transcription of lacZ either with or without nickel supplementation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23737072" target="_blank">23737072</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22971172" target="_blank">22971172</a>]<br></li>
<li>All three NikR mutants , K64A , R65A , or Q118A , repress lacZ transcription in response to nickel supplementation , although with slightly weaker activity than wild-type NikR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23737072" target="_blank">23737072</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22971172" target="_blank">22971172</a>]<br></li>
<li>For example , the operator site to which NikR binds to control nik transcription has not been rigorously identified , and it is not clear whether NikR senses nickel directly or through mechanisms mediated by other proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10787413" target="_blank">10787413</a>]<br></li>
<li>In the presence of stoichiometric nickel , both K64A NikR and R65A NikR protected the nik transcriptional start site at the same position as wild-type NikR ( Figures S3 and S4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23737072" target="_blank">23737072</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22971172" target="_blank">22971172</a>]<br></li>
<li>■ RESULTS To further our understanding of the mechanism of action of E. coli NikR , we examined the ability of several mutant proteins to act as nickel-responsive transcriptional-repressors in βgalactosidase activity assays .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23737072" target="_blank">23737072</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22971172" target="_blank">22971172</a>]<br></li>
<li>This result indicates that an interaction between Q118 and R65 is not necessary for NikR to act as a nickel-activated transcriptional-repressor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23737072" target="_blank">23737072</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22971172" target="_blank">22971172</a>]<br></li>
<li>The presence of two Ni2 + - responsive regulators in E. coli raises the question of how Ni2 + partitions between NikR and RcnR so that repression of nikABCDE occurs before activation of rcnA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16956381" target="_blank">16956381</a>]<br></li>
<li>These features have been demonstrated for the Ni ( II ) - responsive transcriptional-repressor , NikR , from E. coli , which has been shown to-bind to a number of metals with high affinity7 but responds in vivo only to the presence of Ni ( II ) ions .8 Recently , we identified a nickel - and cobalt-responsive metalloregulator , E. coli RcnR ,9 that represses transcription of rcnA , which encodes a nickel and cobalt efflux protein .10 RcnR exhibits only minimal sequence identity with the previously described families of transcriptional-regulators leading to the proposal of a new metalloregulator family .9 RcnR homologs are found throughout the eubacterial kingdom , emphasizing their widespread role in regulating the cell physiology of microbes from diverse growth niches .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18505253" target="_blank">18505253</a>]<br></li>
<li>NikR and NarLX act independently to repress Pnik-lacZ .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16159764" target="_blank">16159764</a>]<br></li>
<li>This strain lacks a gene for T7 RNA polymerase , but sufficient NikR was still synthesized from the chromosomal and/or pNIK103 copies of the nikR gene to repress Pnik-lacZ expression in the presence of nickel .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10787413" target="_blank">10787413</a>]<br></li>
<li>Without the lacIq gene , Pnik-lacZ expression was low even in the absence of added nickel , suggesting that too much NikR was being synthesized .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10787413" target="_blank">10787413</a>]<br></li>
<li>To determine whether NarL - and NikR-dependent regulation of Pnik expression were independent , LacZ levels were measured in nikR-deﬁcient and nikR narLX-deﬁcient strains containing wild-type NikR or a mutant protein , the Cys95Ala protein , which lacks high-afﬁnity nickel-binding activity and shows no DNA binding in vitro with up to 1 M NiCl2 ( 7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16159764" target="_blank">16159764</a>]<br></li>
<li>The effect of the nikABCDE deletion should mimic the effect of the Cys95Ala high-afﬁnity nickel-binding mutant of NikR on Pnik lacZ activity by reducing the amount of functional NikR in the cell .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16159764" target="_blank">16159764</a>]<br></li>
<li>Deletion of nikABCDE ( Fig. 4 ) or nikA alone ( data not shown ) had no effect on the nickel-dependent repression of Pnik-lacZ expression , leading to the surprising conclusion that NikR repression is independent of nickel transport by NikAB-CDE .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16159764" target="_blank">16159764</a>]<br></li>
<li>Additionally , the nickel-dependent repression at high nickel concentrations was NarLX independent , raising the question of whether NikR - and NarLX-dependent repression rely on the same operator site in Pnik .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16159764" target="_blank">16159764</a>]<br></li>
<li>The difference in Pnik-lacZ repression at low nickel concentrations in the narLX-deﬁcient strain indicates that NikR likely binds at a site distinct from that required for NarL-dependent repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16159764" target="_blank">16159764</a>]<br></li>
<li>The effect of excess added cobalt chloride on NikR function in wildtype or mutant E. coli strains is not consistent with an increase in intracellular nickel under deﬁned minimal medium growth conditions ( J.L. Rowe and P.T. Chivers , unpubl .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16956381" target="_blank">16956381</a>]<br></li>
<li>The two nickel-responsive transcriptional-regulators from E. coli , RcnR and NikR , operate in a sequential fashion in response to increasing intracellular nickel9 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18505253" target="_blank">18505253</a>]<br></li>
<li>The nickel-dependent regulation of yqjH and yqjI by YqjI is physiologically consistent with the nickel homeostasis systems in E. coli that are regulated by the NikR and RcnR transcription factors .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21097627" target="_blank">21097627</a>]<br></li>
<li>Nickel coordination is regulated by the The uptake of nickel in Escherichia coli and other microorganisms is transcriptionally regulated by the NikR repressor or its homologs .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12524532" target="_blank">12524532</a>]<br></li>
<li>Regulation of nickel concentrations in Escherichia coli is mediated by the NikR repressor via nickel-induced transcriptional repression of the nickel ABC-type transporter , NikABCDE .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16945905" target="_blank">16945905</a>]<br></li>
<li>Role of NikR The nikABCDE operon of E. coli encodes proteins that appear to assemble to form an ABC-type periplasmic transporter for nickel ~ Navarro et al. , 1993 ; Tam & Saier , 1993 !	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10595554" target="_blank">10595554</a>]<br></li>
<li>Structural Basis of the Metal Specificity for Nickel Regulatory Protein NikR † , ‡ ABSTRACT : In the presence of excess nickel , Escherichia coli NikR regulates cellular nickel uptake by suppressing the transcription of the nik operon , which encodes the nickel uptake transporter , NikABCDE .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18193897" target="_blank">18193897</a>]<br></li>
<li>Hydrogenase activity in E. coli requires the NikAB-CDE nickel transporter , which is transcriptionally repressed by NikR in the presence of excess nickel .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16956381" target="_blank">16956381</a>]<br></li>
<li>Surprisingly , NikR function does not depend on nickel transport by NikABCDE , suggesting the presence of another nickel transporter in E. coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16159764" target="_blank">16159764</a>]<br></li>
<li>The mechanism of nickel partitioning between these pathways has not been elucidated , but genetic data indicate that NikR function is independent of NikABCDE function ( Rowe et al. , 2005 ) , suggesting the presence of a second nickel-transporter in E. coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16956381" target="_blank">16956381</a>]<br></li>
<li>Deletion of rcnA had a striking effect on Pnik regulation ( Fig. 5 ) , resulting in a 50 % decrease in Pnik expression at low nickel ( 10 nM ) , which corresponds to an increase in NikR activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16956381" target="_blank">16956381</a>]<br></li>
<li>The low-level expression inferred from the effect of the rcnA deletion on NikR activity may be required for buffering effects of transient increases in intracellular nickel ions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16956381" target="_blank">16956381</a>]<br></li>
<li>RcnA likely competes with NikR for nickel ions and is necessary for full hydrogenase-3 activity at low nickel ion concentrations by preventing premature repression of nikABCDE .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16956381" target="_blank">16956381</a>]<br></li>
<li>NikR is functional under all nickel conditions tested , but its activity is modulated by the total nickel concentration present as well as by one or more components of the hydrogenase assembly pathway .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16159764" target="_blank">16159764</a>]<br></li>
<li>These results suggest that th nickel-dependent regulation of Pnik is strongly correlated with the synthesis of active hydrogenase at limiting nickel concentrations and raise the question of whether NikR activity is altered under these conditions as a result of competition for available nickel ions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16159764" target="_blank">16159764</a>]<br></li>
<li>Under nickel-limiting conditions , deletion of rcnA increased NikR activity in vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16956381" target="_blank">16956381</a>]<br></li>
<li>In particular , the results suggest that rcnA deletion would impact NikABCDE-dependent nickel uptake and hydrogenase activity due to an increase in NikR activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16956381" target="_blank">16956381</a>]<br></li>
<li>These comparable affinities raise the question of how NikR specifically responds to nickel within the cell .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18193897" target="_blank">18193897</a>]<br></li>
<li>NikR is found in both bacteria and archaea and seems to regulate a variety of different nickel-uptake systems .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12524532" target="_blank">12524532</a>]<br></li>
<li>NikR orthologs are found in Gram-negative bacteria and archaea , suggesting common mechanisms of nickel regulation across a diverse set of microorganisms .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12970756" target="_blank">12970756</a>]<br></li>
<li>NikR represses expression of the NikACBED ABC-uptake system in the presence of elevated intracellular Ni ( II ) - concentrations ( De Pina et al. 1999 ) and co-expression of this uptake-system with RcnA would lead to an expensive futile cycle of nickel import and export .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17120142" target="_blank">17120142</a>]<br></li>
<li>The nikABCDE nickel import system is repressed by Ni-NikR as nickel concentrations rise above 100 nM in M63 / glucose minimal medium ( 25 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21097627" target="_blank">21097627</a>]<br></li>
<li>Under steady state growth conditions , the NikR repressor responds to lower ambient Ni ( II ) concentrations than RcnR , as nikABCDE expression is completely repressed at nickel levels that exhibit minimal rcnA expression .9 As Ni ( II ) levels increase , RcnR-repression of rcnA transcription is diminished allowing full induction of RcnA expression to prevent excess metal accumulation in the cell .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18505253" target="_blank">18505253</a>]<br></li>
<li>When nickel concentrations in the cell fall below some critical level , the affinity of NikR for the operator decreases , and the operon is derepressed , allowing synthesis of NikABCDE , which can catalyze increased levels of nickel uptake .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10787413" target="_blank">10787413</a>]<br></li>
<li>Transcription of the NikABCDE nickel transporter , which is required for NiFe hydrogenase synthesis , was previously shown to be upregulated by FNR ( fumarate-nit rate regulator ) in the absence of oxygen and repressed by the NikR repressor in the presence of high extracellular nickel levels .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16159764" target="_blank">16159764</a>]<br></li>
<li>Transcription of nikABCDE is up-regulated by the fumarate and nitrate regulatory protein ( FNR ) in response to low O2 levels and down-regulated by NikR in response to high intracellular nickel concentration9 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12524532" target="_blank">12524532</a>]<br></li>
<li>It is activated by fumarate nitrate regulatory protein ( Fnr ) , an oxygen-sensitive transcriptional factor , and repressed by NikR when intracellular nickel concentrations are high ( 5 , 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10787413" target="_blank">10787413</a>]<br></li>
<li>Surprisingly , NikR function does not depend on nickel import by NikABCDE at any nickel concentration or under any growth condition tested here .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16159764" target="_blank">16159764</a>]<br></li>
<li>It should be noted that these models of nickel-dependent regulation of NikR function are not mutually exclusive .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12970756" target="_blank">12970756</a>]<br></li>
<li>At low nickel concentrations in the absence of nitrate ( curves and diagrams I and II ) , FNR upregulates transcription from PnikABCDE and some component of the hydrogenase assembly pathway blocks NikR function .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16159764" target="_blank">16159764</a>]<br></li>
<li>NikR function when hydrogenase synthesis is high and nickel concentrations are limiting ( Fig. 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16159764" target="_blank">16159764</a>]<br></li>
<li>Genetic evidence indicates that the primary function of NikR is to repress transcription of the nikABCDE genes when nickel is present at concentrations of 250 mM or mor C Fig. 3 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10595554" target="_blank">10595554</a>]<br></li>
<li>In the presence of excess intracellular nickel , the nickel-dependent repressor NikR binds to an operator sequence within the nikABCDE promoter , turning off expression of the transporter and decreasing nickel import .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12970756" target="_blank">12970756</a>]<br></li>
<li>In the presence of nitrate at low nickel concentrations ( III ) , NarL and NikR strongly repress transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16159764" target="_blank">16159764</a>]<br></li>
<li>At high nickel concentrations ( IV ) , NikR is the dominant regulator of PnikABCDE expression regardless of the presence of nitrate ( i.e. , NarL ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16159764" target="_blank">16159764</a>]<br></li>
<li>A decrease in oxygen tension results in activation of FNR and upregulation of nikABCDE expression , while the presence of excess nickel activates NikR , which overrides the action of FNR and results in repression of nikABCDE transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16159764" target="_blank">16159764</a>]<br></li>
<li>NikR , a protein of the ribbon-helix-helix family of transcription factors , represses expression of the nikABCDE operon in the presence of excessive concentrations of intracellular nickel .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10787413" target="_blank">10787413</a>]<br></li>
<li>Transcription of the operon encoding this nickel importer , nikABCDE , is repressed in the presence of nickel by the transcription factor NikR ( 3 , 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16945905" target="_blank">16945905</a>]<br></li>
<li>Both organisms use transcription factor NikR to inhibit the transcription of the nickel uptake machine in the presence of surplus nickel ( 5 , 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18193897" target="_blank">18193897</a>]<br></li>
<li>NikR regulates NikABCDE synthesis in a nickel-dependent fashion so that transporter expression is minimized under conditions of abundant extracellular metal ( Rowe et al. , 2005 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16956381" target="_blank">16956381</a>]<br></li>
<li>The pattern of Pnik regulation determined thus far has indicated that transcription of nikABCDE is simple and is not tightly linked to the regulation of hydrogenase expression , raising the possibility that NikABCDE levels could be unnecessarily high under conditions of known low hydrogenase expression ( i.e. , growth in nitrate ) and/or low extracellular nickel concentrations ( inactive NikR ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16159764" target="_blank">16159764</a>]<br></li>
<li>NikR further represses NikABCDE expression at higher nickel concentrations ( IV ) , providing nearly complete repression under these conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16159764" target="_blank">16159764</a>]<br></li>
<li>At higher nickel concentrations , NikR repression exhibits a consistent dependence on the added nickel concentration .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16159764" target="_blank">16159764</a>]<br></li>
<li>The nikABCDE operon involved in nickel uptake is expressed under anaerobic-growth conditions ( 53 ) , and its expression is speciﬁcally repressed by nickel via the metalloregulator NikR ( 12 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21665978" target="_blank">21665978</a>]<br></li>
<li>Nitrate ( NarLX ) and nickel ( NikR ) independently regulate P expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16159764" target="_blank">16159764</a>]<br></li>
<li>The nickel dependence of PrcnA expression suggested that NikR does not regulate both Pnik and PrcnA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16956381" target="_blank">16956381</a>]<br></li>
<li>In contrast , the six-coordinate Ni ( II ) sites of NikR and RcnR appear to be designed for spontaneous metal release in response to decreasing metal ion concentration , as both of these sites function to respond to fluctuations in intracellular nickel concentrations that occur under stress conditions .9 , 63 The use of a single cysteine thiolate ligation in NikR and RcnR would seem to be an ideal choice for a metal ion electronic structure sensor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18505253" target="_blank">18505253</a>]<br></li>
<li>( 8 ) Iwig , J. S. , and Chivers , P. T. ( 2010 ) Coordinating intracellular nickel-metal-site structure-function relationships and the NikR and RcnR repressors .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23737072" target="_blank">23737072</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22971172" target="_blank">22971172</a>]<br></li>
<li>Divalent nickel and copper bind tightly to NikR and both R are able to order the 3 helix , while divalent zinc binds less R tightly and is incapable of ordering the 3 helix ( Figure 4 and Figure S1 of the Supporting Information ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18193897" target="_blank">18193897</a>]<br></li>
<li>Proteins rich in histidine and cysteine often bind divalent transition metals ( 18 ) ; in NikR , these residues may bind nickel ions ( 7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10787413" target="_blank">10787413</a>]<br></li>
<li>Finally , an abundant intracellular metal ion such as potassium binds at the interface of the MBD and RHHs , serving to stabilize this NikR configuration ( IV f V ) repression of the nik operon , it is important that NikR be highly responsive to nickel ions , but not be activated by other intracellular transition metals .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18193897" target="_blank">18193897</a>]<br></li>
<li>As NikR is the only ribbon-helix-helix transcriptional-repressor that is known to be metal-responsive , it is possible that this helix is important for the allosteric regulation of NikR by nickel .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12970756" target="_blank">12970756</a>]<br></li>
<li>Either nickel ions enter the cell by some previously unidentiﬁed pathway or NikR is activated in vivo by a different transition metal .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16159764" target="_blank">16159764</a>]<br></li>
<li>The high afﬁnity of NikR for metal ions ( 8 , 41 ) suggests different possibilities for the activation of NikR in the absence of added nickel .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16159764" target="_blank">16159764</a>]<br></li>
<li>The disordered-to-ordered transition of helix 3 within the MBD upon nickel binding gave preliminary insight into the structural changes that take place upon activation of NikR by metal ions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16945905" target="_blank">16945905</a>]<br></li>
<li>In the Ni2 + - bound form , NikR represses transcription from Pnik , providing a link between the two metal pools .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16956381" target="_blank">16956381</a>]<br></li>
<li>Although the protein 2 was purified with one nickel ion bound per monomer and was yellow in color , crystals of fulllength NikR were colorless , presumably because the ADA buffer chelated the nickel at the concentrations used for crystallization .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12970756" target="_blank">12970756</a>]<br></li>
<li>When NikR is incubated with PAR , only approximately half of the nickel bound to the protein is released into solution over the course of 7 h ( Figure S2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20583753" target="_blank">20583753</a>]<br></li>
<li>In the absence of excess Ni21 during column loading , NikR stripped nickel from the column and failed to-bind efficiently ( ,10 % bound ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10787413" target="_blank">10787413</a>]<br></li>
<li>As described previously , residues 62 -- 80 of NikR are disordered in the absence of nickel but form a loop and a stable - helix ( helix 3 ) when nickel is bound because of the direct involvement of His-76 in nickel coordination ( 13 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16945905" target="_blank">16945905</a>]<br></li>
<li>( PDB entry 1Q5V ) and Ni2 - bound MBD and NikR structures ( PDB entries 1Q5Y and 2HZA , respectively ) indicates that the R3 helices ( residues 65-78 ) of the MBD are flexible in the absence of nickel and become structured when nickel binds to the protein ( 19 , 20 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18193897" target="_blank">18193897</a>]<br></li>
<li>Addition of higher concentrations of Ni21 resulted in protein aggregation at the NikR concentrations used in this experiment .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10787413" target="_blank">10787413</a>]<br></li>
<li>The second motif , DXGXVXXFXDDXIXXR , corresponds to residues 104 to 119 of NikR and it is not as well conserved as the ﬁrst one in the 16 Fur-like proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9882686" target="_blank">9882686</a>]<br></li>
<li>The reactivities of the cysteine residues in the various states of NikR were examined first .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20583753" target="_blank">20583753</a>]<br></li>
<li>The ﬁrst motif consists of STQHHHXXL , corresponding to residues 73 to 81 of NikR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9882686" target="_blank">9882686</a>]<br></li>
<li>The Helicobacter site is in the NikR promoter , suggesting that NikR may negatively regulate its own synthesis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10787413" target="_blank">10787413</a>]<br></li>
<li>The degree of NikR inhibition depends upon the level of hydrogenase expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16159764" target="_blank">16159764</a>]<br></li>
<li>The structure of NikR reveals the architecture of a protein capable of this level of regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12970756" target="_blank">12970756</a>]<br></li>
<li>Deletion of rcnR results in constitutive rcnA expression and a corresponding decrease in NikR activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16956381" target="_blank">16956381</a>]<br></li>
<li>Moreover , expression was not constitutive , which would have been expected when NikR was involved in repression of rcnA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17120142" target="_blank">17120142</a>]<br></li>
<li>NikABCDE-independent regulation of PnikABCDE expression by NikR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16159764" target="_blank">16159764</a>]<br></li>
<li>Relatively little is currently known about NikR repressor or how it regulates gene expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10787413" target="_blank">10787413</a>]<br></li>
<li>Formate likely does not act directly to affect RcnR and NikR function .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16956381" target="_blank">16956381</a>]<br></li>
<li>Nickel-dependent repression by NikR was still observed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16956381" target="_blank">16956381</a>]<br></li>
<li>Functional similarity between NikR and Fur proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9882686" target="_blank">9882686</a>]<br></li>
<li>Carbamidomethylation of Cysteine Residues in NikR Using Iodoacetamide .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20583753" target="_blank">20583753</a>]<br></li>
<li>Black lines , NikR ; gray lines , apoNikR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10787413" target="_blank">10787413</a>]<br></li>
<li>NikR -- Operator Interactions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16945905" target="_blank">16945905</a>]<br></li>
<li>Metal Binding to NikR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18193897" target="_blank">18193897</a>]<br></li>
<li>Metal salts and NikR were added as noted .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20583753" target="_blank">20583753</a>]<br></li>
				</div></ul><br>
			<label class="collapse_DOM" for="_2"><font size="5px">NikR protein domains</font></label>
				<input id="_2" type="checkbox">
				<div class="DOM"><br><ul><li>Structures of Ni2 + - bound EcNikR and EcNikR-DNA were obtained from the Protein Data Bank ( PDB IDs 2HZA and 2HZV , respectively ) .9,32 Missing protein residues were added with only backbone and Cβ atoms in approximate positions in COOT .37 Missing heavy atoms and all hydrogen atoms were built with ICBuild and HBuild functions in CHARMM version 33b2 using the all-atom ( CHARMM27 ) parameters for proteins , nucleic acids , and potassium ions .38 Nickel parameters ( van der Waals radius of 1.4125 Å and ε of − 0.010 ) were obtained from Dr. Johan Bredenberg ( personal communication ) and thiolate parameters from published studies .39 Histidines were modeled as δ - or ε-protonated , depending on the likely hydrogen-bonding pattern ; i.e. , heavy atoms of hydrogen-bonding residues that were within 3.5 Å of one another suggest that the corresponding nitrogen on His should be protonated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19621966" target="_blank">19621966</a>]<br></li>
<li>Replacing Q118 with alanine , which would disrupt the putative hydrogen bond interaction but not the potassium site , does result in a small but signi cant reduction in the activity of NikR in vivo , ﬁ suggesting that this residue plays a minor role in the allosteric effect of nickel as previously suggested by molecular dynamics simulations .39 However , Q118A NikR is still able to repress transcription from the nik promoter in response to nickel supplementation of the growth media , indicating that it is not required for the full function of the protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23737072" target="_blank">23737072</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22971172" target="_blank">22971172</a>]<br></li>
<li>Several published sequence alignments of NikR homologues34 ,39 reveal that although many NikR proteins have a basic residue at one or both of the positions corresponding to K64 and R65 of E. coli NikR ,39,41 this chemistry is not conserved .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23737072" target="_blank">23737072</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22971172" target="_blank">22971172</a>]<br></li>
<li>Instead , the results reported here suggest that the nonspeciﬁc electrostatic interactions between residues K64 and R65 and the DNA-phosphodiester backbone are essential for the function of NikR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23737072" target="_blank">23737072</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22971172" target="_blank">22971172</a>]<br></li>
<li>Given that both K64 and R65 appear to make nonspeciﬁc contacts with the DNA , it is feasible that they perform the same function such that removing only one basic residue does not hinder the activity of NikR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23737072" target="_blank">23737072</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22971172" target="_blank">22971172</a>]<br></li>
<li>Residues 49 to 55 of the E. coli NikR sequence probably form a flexible interdomain linker since this region is relatively glycine-rich and contains gaps in the alignment ~ Fig. 2 !	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10595554" target="_blank">10595554</a>]<br></li>
<li>This motif is located in a region that shows the highest solvent accessibility score in NikR and that is composed of a histidine-rich stretch of 5 amino acids .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9882686" target="_blank">9882686</a>]<br></li>
<li>Apo and apo-SeMet NikR crystals belonged to space group P212121 and were cryoprotected using Paratone-N oil .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12970756" target="_blank">12970756</a>]<br></li>
<li>Difference-distance matrix analysis using ESCET19 and ∆ Φ , ∆ Ψ plots20 of the four apo NikR molecules support the notion that this poorly conserved linker serves as an interdomain hinge , with vertices at residues 41 and 48 -- 49 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12970756" target="_blank">12970756</a>]<br></li>
<li>In apo-NikR , three of the four 3 helices are disordered-and not traceable in the electron density maps ( residues 62 2 + 77 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18193897" target="_blank">18193897</a>]<br></li>
<li>Helix α3 was removed in three apo NikR molecules because of high B-factors and a lack of interpretable electron density in omit maps .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12970756" target="_blank">12970756</a>]<br></li>
<li>We also note that the pattern of sequence conservation among different NikR subfamily members matches closely to the patterns of structurally and functionally important residues that have been established by mutagenesis of Arc repressor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10595554" target="_blank">10595554</a>]<br></li>
				<label class="DOM_collapse_similarity" for="dom_similarity_13">
Arrows and filled diamonds indicate guanine residues protected by NikR from methylation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10787413" target="_blank">10787413</a>]<br></label>
<input id="dom_similarity_13", type="checkbox">
<div class="DOM_similar">
<li>Filled diamonds indicate guanine residues protected from methylation by NikR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10787413" target="_blank">10787413</a>]<br></li></div>
<li>The final model refined to an Rwork of 24.9 % and an Rfree of 27.3 % and includes one subunit of the MBD NikR ( residues 50-61 and 80-130 ) , 22 water molecules , and two zinc ions ( Table 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18193897" target="_blank">18193897</a>]<br></li>
<li>The antibody was raised against a NikR peptide ( residues 104 − 121 ) and puriﬁed using afinity chromatography with resin bearing the same peptide .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23737072" target="_blank">23737072</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22971172" target="_blank">22971172</a>]<br></li>
<li>The second motif , DXGXVXXFXDDXIXXR , corresponds to residues 104 to 119 of NikR and it is not as well conserved as the ﬁrst one in the 16 Fur-like proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9882686" target="_blank">9882686</a>]<br></li>
<li>The reactivities of the cysteine residues in the various states of NikR were examined first .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20583753" target="_blank">20583753</a>]<br></li>
<li>The ﬁrst motif consists of STQHHHXXL , corresponding to residues 73 to 81 of NikR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9882686" target="_blank">9882686</a>]<br></li>
<li>This result was confirmed by 5,5 ´ - dithionitro-benzoic acid ( DTNB ) , a thiol-specific labeling reagent , titration of thiol groups in NikR , which contains two cysteines ( Cys95 and the nonconserved Cys128 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12524532" target="_blank">12524532</a>]<br></li>
<li>Modification of apo-NikR with excess iodoacetamide results in two population of species bearing one or two labels , indicating that both cysteines are susceptible to modification ( Figure S4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20583753" target="_blank">20583753</a>]<br></li>
<li>The presence of two Ni2 + - responsive regulators in E. coli raises the question of how Ni2 + partitions between NikR and RcnR so that repression of nikABCDE occurs before activation of rcnA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16956381" target="_blank">16956381</a>]<br></li>
<li>However , NikR contains two motifs which are perfectly conserved in the four Fur proteins described above .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9882686" target="_blank">9882686</a>]<br></li>
<li>This suggests that it may not always be straightforward to predict the dimer recognition subsites for other NikR family members .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10787413" target="_blank">10787413</a>]<br></li>
<li>Interestingly , RcnA affects NikR function and is necessary for hydrogenase activity in the presence of NikR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16956381" target="_blank">16956381</a>]<br></li>
<li>Carbamidomethylation of Cysteine Residues in NikR Using Iodoacetamide .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20583753" target="_blank">20583753</a>]<br></li>
<li>Carbethoxylation of Histidine Residues in NikR Using Diethyl Pyrocarbonate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20583753" target="_blank">20583753</a>]<br></li>
<li>The RHH domains align well structurally with both apo-NikR ( rmsd 1.16 -- 1.35 Å for 84 C atoms ) and nickel-bound NikR ( rmsd 1.06 -- 1.16 Å for 84 C atoms ) , whereas the MBD closely resembles the nickel-bound forms of both the isolated MBD ( rmsd 0.95 -- 1.02 Å for 304 C atoms ) and full-length NikR ( rmsd 0.81 -- 0.84 Å for 304 C atoms ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16945905" target="_blank">16945905</a>]<br></li>
<li>Indeed , the RHH domains of NikR are connected to the MBD by flexible linkers ( Figs. 1 and 2 ) that vary significantly in length and amino acid sequence as a function of species , making it unlikely that an allosteric signal could be propagated via this route .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16945905" target="_blank">16945905</a>]<br></li>
<li>The previously determined crystal structure of Ni2 + - MBD ( PDB entry 1Q5Y ) ( 19 ) shows that nickel binds in a square planar geometry at the interface between two subunits of the NikR tetramer and is coordinated by H87 , H89 , and C95 of one monomer and H76 of a neighboring monomer ( Figure 2a ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18193897" target="_blank">18193897</a>]<br></li>
<li>N ickel is a vital nutritional requirement for many microorganisms , but it is toxic if the intracellular levels and distribution are not strictly controlled .1 − 4 As for other metals , nickel homeostasis is often governed by metalloregulatory proteins that directly modulate the expression of genes following metal-binding .5,6 An example is the nickelresponsive NikR ,7 − 9 which in Escherichia coli ( E. coli ) represses transcription of the nikABCDE operon encoding a nickelspeci c importer .10,11 ﬁ The tetrameric structure of NikR ( Figure 1 ) consists of a core of four metal-binding domains ( MBDs ) and two ﬂanking ribbon − helix − helix DNA-binding domains ( DBDs ) .12,13 NikR binds stoichiometric nickel ( 4 nickel ions per tetramer ) with picomolar afinity , which consequently activates DNA-binding to the nik recognition sequence .12,14 − 17 These nickel ions are located at the center of the protein and are coordinated in a square-planar geometry by H87 , H89 , and C95 from one MBD and H76 ′ from an opposing MBD .12,13,18 There is also evidence for the binding of additional nickel ions to NikR which increase the afinity of the protein − DNA complex , although the physiological relevance of these weaker nickel sites has not yet been de ned .14,16,19 − 21 ﬁ The mechanism of nickel-activated DNA-binding by NikR is not clear .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23737072" target="_blank">23737072</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22971172" target="_blank">22971172</a>]<br></li>
<li>The previously reported crystal structure of apo-EcNikR revealed a modular domain organization , with two dimeric ribbon -- helix -- helix ( RHH ) domains flanking a tetrameric metalbinding domain ( MBD ) ( Fig. 1A ; ref .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16945905" target="_blank">16945905</a>]<br></li>
<li>Introduction Monitoring the concentration of metal ions within the cell is of crucial importance , as a relatively small concentration of some metal ions is necessary for a number of cellular processes to occur , but often too much of that same ion may cause cell damage and death .1 -- 4 Nickel is an essential cofactor for a number of bacterial enzymes that are involved in cell survival , including hydrogenase and urease , two enzymes vital to the survival of the pathogenic bacteria Helicobacter pylori .1 The concentration of nickel in many bacteria is regulated by the nickel-binding transcription factor , NikR , which transcriptionally represses genes encoding nickel-specific importers .5 The best-characterized NikR homologues are from Escherichia coli ( EcNikR ) , H. pylori ( HpNikR ) , and Pyrococcus horikoshi ( PhNikR ) .6 -- 9 Each is a homotetramer composed of two domains : the central tetrameric metal-binding domain ( MBD ) and two flanking dimeric ribbon -- helix -- helix ( RHH ) domains ( Figure 1a ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19621966" target="_blank">19621966</a>]<br></li>
<li>A similar conformation was observed in one crystal form of nickel-bound PhNikR grown in the absence of DNA ( PDB ID code 2BJ9 ) , although in that structure the two RHH domains are oriented toward opposite faces of the MBD ( trans conformation as opposed to cis ) ( 14 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16945905" target="_blank">16945905</a>]<br></li>
<li>Structures of PhNikR also show different relative orientations of the MBD and RHH domains when compared with each other or to the structures of EcNikR , further supporting the idea that NikR is flexible in the absence of DNA ( 14 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16945905" target="_blank">16945905</a>]<br></li>
<li>However , the orientations of the RHH DNA-binding domains of the repressor relative to the MBD are dramatically different compared with the structures of EcNikR alone ( Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16945905" target="_blank">16945905</a>]<br></li>
<li>Moreover , the highly conserved positions in the sequence alignment of NikR N-terminal domains generally correspond to positions in Arc repressor where single alanine mutations disrupt folding or function , and nonconserved NikR positions correspond to positions in Arc where mutations are silent ~ see Fig. 2 !	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10595554" target="_blank">10595554</a>]<br></li>
<li>The C-terminal regulatory domains of two NikR monomers form what we will refer to as a core dimer ( red and green chains make up one core dimer and yellow and blue chains make up the other core dimer in Fig. 1a ) , and these dimers come together to form a tetramer with approximate D2 symmetry .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12970756" target="_blank">12970756</a>]<br></li>
<li>Each monomer in the C C-domain homology to other regulatory domains The C domain of NikR is structurally homologous to several classes of proteins containing an ` ACT-like ' domain13 and is most similar to the regulatory domains of the metabolic enzymes phosphoglycerate dehydrogenase ( PGDH ) 14 , ATP phosphoribosyltransferase ( ATPPRTase ) 15 and phenylalanine hydroxlyase ( PheOH ) 16 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12970756" target="_blank">12970756</a>]<br></li>
<li>Notably , the regulatory domains of NikR are similar in structure to regulatory domains of allosteric enzymes such as PGDH .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12970756" target="_blank">12970756</a>]<br></li>
<li>This observation confirms that the crystal structure represents a specific NikR -- operator complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16945905" target="_blank">16945905</a>]<br></li>
<li>NikR -- operator complex structure and the mechanism of repressor activation by metal ions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18586269" target="_blank">18586269</a>]<br></li>
<li>( 13 ) Schreiter , E. R. , Wang , S. C. , Zamble , D. B. , and Drennan , C. L. ( 2006 ) NikR − operator complex structure and the mechanism of repressor activation by metal ions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23737072" target="_blank">23737072</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22971172" target="_blank">22971172</a>]<br></li>
<li>It has been proposed that these nickel-activated contacts weakly anchor NikR to the DNA in a sequence-independent manner , allowing it to eficiently scan in one dimension for the nik recognition sequence .25 Domain − domain motions about the ﬂexible linker between the MBD and DBD , supported by molecular dynamics calculations ,40 would allow the protein to frequently sample the down-cis conformation and read the DNA sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23737072" target="_blank">23737072</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22971172" target="_blank">22971172</a>]<br></li>
<li>For example , nickel-dependent stabilization of the loop/helix α3 was also observed by X-ray crystallography in several subunits of NikR from Pyrococcus horikoshii , but this NikR homologue does not have basic residues at the equivalent positions .34 As an alternative , the authors predicted hydrogen bonding between a backbone amide nitrogen of the protein loop and the DNA , as observed in other types of transcription factors .42 Furthermore , in vitro analysis of NikR from H. pylori demonstrated that mutation of the corresponding Q76 and R77 to alanine weakened complex formation with one of its recognition sequences by several orders of magnitude ,33 although DNA-binding could still be detected .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23737072" target="_blank">23737072</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22971172" target="_blank">22971172</a>]<br></li>
<li>To identify potential sites of close contact between NikR and the operator site , we performed dimethylsulfate protection experiments .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10787413" target="_blank">10787413</a>]<br></li>
<li>In vivo and in vitro experiments will help to determine the precise DNA motif involved in the contact with NikR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9882686" target="_blank">9882686</a>]<br></li>
<li>First , specific hydrogen bonding interactions are made by the side chains of Arg-3 and Thr-5 from the - sheets of the RHH domains to nucleotide bases in the operator major groove , contacting the 5 bp of each operator half-site that abrogate NikR binding when mutated ( ref .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16945905" target="_blank">16945905</a>]<br></li>
<li>The NikR -- DNA complex structure shows the importance of this conformational change in creating 6 -- 8 polar interactions between the loop containing residues Lys-64 and Arg-65 and the phosphate backbone of operator DNA ( Fig. 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16945905" target="_blank">16945905</a>]<br></li>
<li>The NikR molecule makes initial contacts to DNA , stabilized via hydrogen bonds and electrostatic interactions between residues in the loop of the MBD and the phosphate backbone ( II f III ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18193897" target="_blank">18193897</a>]<br></li>
<li>The side chain of Arg-119 , in a loop connecting the last - helix and - strand of NikR , extends between the phosphates of the DNA minor groove .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16945905" target="_blank">16945905</a>]<br></li>
<li>Interactions also are made between a loop leading into helix 3 of the MBD and the center of the NikR operator , including the backbone amide nitrogen of Arg-65 and the side chains of Lys-64 and Arg-65 , which make polar interactions with DNA backbone phosphates ( asterisk in Figs. 2 A and 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16945905" target="_blank">16945905</a>]<br></li>
<li>X-ray absorption spectroscopy reveals that nickel in the holo-NikR protein is bound in a novel four-coordinate planar site consisting of two histidines , one additional O - or N-donor ligand and one S-donor ligand .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12524532" target="_blank">12524532</a>]<br></li>
<li>This residue was of particular interest because the corresponding residue in H. pylori NikR , Glu104 , serves as a ligand of a secondary nickel site ( 26 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20583753" target="_blank">20583753</a>]<br></li>
<li>Furthermore , R65 is in hydrogen-bonding proximity to Q118 , a residue that contributes a backbone amide carbonyl oxygen as a ligand in a potassium-binding site .13 This potassium is bound at the interface of the MBD and DBD and is critical for NikR − DNA complex formation ,24 so Q118 could serve as a link between nickel-binding and an activated DNA-binding conformation of E. coli NikR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23737072" target="_blank">23737072</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22971172" target="_blank">22971172</a>]<br></li>
<li>In this study , classical chemical modification combined with mass spectrometric peptide mapping has been used as a sensitive approach for the direct identification of cysteine and histidine residues in both the highand the low-affinity Ni ( II ) sites of E. coli NikR , and the locations of these residues are consistent with their roles as critical ligands in the allosteric metalloregulation of operator/promoter binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20583753" target="_blank">20583753</a>]<br></li>
<li>To identify residues in E. coli NikR of potential significance to the binding of nickel , we used a combination of chemical modification and mass spectrometry .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20583753" target="_blank">20583753</a>]<br></li>
<li>In the E. coli operator , residues important for NikR binding account for fewer than half of the base pairs that are related by dyad symmetry .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10787413" target="_blank">10787413</a>]<br></li>
<li>Ni2 + binds to the high-affinity site of NikR , stabilizing the R3 helix and its preceding loop ( I f II ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18193897" target="_blank">18193897</a>]<br></li>
<li>The relative mass difference of three carbethoxy groups between the apo - and Ni ( II ) - loaded forms of NikR is consistent with the roles of His76 , His87 , and His89 as high-affinity Ni ( II ) - binding ligands .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20583753" target="_blank">20583753</a>]<br></li>
<li>The central high-affinity Ni2 + - binding sites , which are conserved among all NikR homologues , are sensory sites that specifically bind the scarce cellular Ni2 + ion , enabling binding to the nik promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19621966" target="_blank">19621966</a>]<br></li>
<li>E. coli NikR contains two cysteines at positions 95 and 128 , and Cys95 is a ligand in the high-affinity Ni ( II ) site ( 14 , 19 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20583753" target="_blank">20583753</a>]<br></li>
<li>Purified NikR binds two nickel ions per dimer with high affinity in a manner that gives rise to distinctive changes in both the UV absorbance and circular dichroism spectra , although the function of nickel-binding to this site is unknown .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10787413" target="_blank">10787413</a>]<br></li>
<li>These data reveal a role for the high-afﬁnity nickel-binding site in NikR function at low total nickel concentrations .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16159764" target="_blank">16159764</a>]<br></li>
<li>Strong operator binding requires the continual presence of 20 -- 50 nickel , inmM dicating the presence of a low affinity nickel-binding site , and NikR dimers also contain two high affinity nickel-binding sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10787413" target="_blank">10787413</a>]<br></li>
<li>The aim of this study was to resolve some of the discussion in the literature with regard to the distinct nickel-binding sites of E. coli NikR and to identify residues that contribute to the elusive low-affinity Ni ( II ) site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20583753" target="_blank">20583753</a>]<br></li>
<li>NikR contains two Ni ( II ) binding sites , termed high - and low-affinity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18505253" target="_blank">18505253</a>]<br></li>
<li>Despite the evidence that metal-binding to two distinct sites on NikR , referred to as the high - and low-affinity Ni ( II ) sites , is required for Ni ( II ) - selective DNA-binding by the protein , the role of the latter set of Ni ( II ) ions in the activation of NikR remains controversial , and the position of the putative low-affinity Ni ( II ) - binding site ( s ) on NikR has not been determined .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20583753" target="_blank">20583753</a>]<br></li>
<li>These experiments support the conclusion that C128 is not a ligand of the low-affinity Ni ( II ) site of NikR and that this residue does not make a significant contribution to Ni ( II ) - responsive DNA-binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20583753" target="_blank">20583753</a>]<br></li>
<li>In this study we confirm that NikR has a high-affinity Ni ( II ) - binding site that is maintained upon DNA-binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20583753" target="_blank">20583753</a>]<br></li>
<li>To examine if the high-affinity Ni ( II ) coordination site changes upon DNA-binding , XAS analyses of holo-NikR in the presence and absence of an oligonclueotide containing the nik recognition sequence were performed by our laboratory .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20583753" target="_blank">20583753</a>]<br></li>
<li>This unusual DNA-binding fold contains several conserved positively charged amino acids in α1 and α2 , and two of these residues were shown to be necessary for high-affinity DNA-binding of MtCsoR .2 RcnR , the first biochemically described member of the RcnR/CsoR family , was identified as a transcriptional regulator of the rcnA gene ,5 which encodes a Ni ( II ) / Co ( II ) efflux protein in E. coli .8 RcnA not only prevents accumulation of toxic metal levels8 but also blocks premature activation of a second transcription factor , NikR , through competition for Ni ( II ) , allowing nickel ions to be trafficked to the hydrog-enase enzymes .5 Unlike the trigonal planar Cu ( I ) site of MtCsoR , Ni ( II ) and Co ( II ) bind to RcnR at pseudo-octahedral six-coordinate sites .4 RcnR uses residues in positions equivalent to those used by MtCsoR to bind metal ions , in addition to ligands in an Nterminal metal-binding motif .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19703465" target="_blank">19703465</a>]<br></li>
<li>An examination of the crystal structures of E. coli NikR revealed multiple potential metal-binding sites on the protein in addition to the high-affinity nickel-binding sites , which is not surprising given that each NikR monomer contains 12 histidines and 2 cysteines as well as 16 aspartates/glutamates .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20583753" target="_blank">20583753</a>]<br></li>
<li>UV-visible and CD spectra , the peaks attributed to nickel-binding required the presence of both the metal ion and NikR and were absent following incubation with 50 mM EDTA for 12 h at room temperature .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10787413" target="_blank">10787413</a>]<br></li>
<li>A comparison between E. coli apo-NikR 1 Abbreviations : EXAFS , extended X-ray absorption fine structure ; HEPES , 4 - ( 2-hydroxyethyl ) -1 - piperazineethanesulfonic acid ; GuHCl , guanidine hydrochloride ; MBD , metal-binding domain of NikR , residues 49-133 ; NTA , nitrilotriacetic acid ; PAR , 4 - ( 2-pyridylazo ) - resorcinol ; RHH , ribbon-helix-helix domain of NikR , residues 1-48 ; rmsd , root-mean-square deviation ; Tris , tris ( hydroxymethyl ) ami-nomethane ; XANES , X-ray absorption near-edge spectroscopy .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18193897" target="_blank">18193897</a>]<br></li>
<li>These second metal-binding sites are composed of two conserved amino acid residues from the RHH domain , E30 and D34 , and a number of backbone carbonyls from residues in the MBD ( Figure 1b ) .9 Biochemical data and crystal structures of various NikR proteins have informed as well as raised interesting questions regarding these metal-binding sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19621966" target="_blank">19621966</a>]<br></li>
<li>To investigate how different metals bind NikR and to explore the issue of specificity , we have determined the structures of the E. coli NikR metal-binding domain ( MBD ) 1 with Zn2 + and Cu2 + bound and with no metals bound .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18193897" target="_blank">18193897</a>]<br></li>
<li>This domain retains all key metal-binding features of the full-length NikR while crystallizing much more readily .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18193897" target="_blank">18193897</a>]<br></li>
<li>As described below , we first studied nickel-binding by the intact NikR repressor , then assayed NikR binding to a DNA fragment containing the entire nik promoter region by electro-phoretic mobility shift experiments , and finally used footprinting and mutational studies to define the operator site and its functional role in greater detail .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10787413" target="_blank">10787413</a>]<br></li>
<li>Despite the wealth of structural information about NikR proteins in multiple free and ligand-bound states , the mechanism of metal-activated DNA-binding remains undeﬁned .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23737072" target="_blank">23737072</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22971172" target="_blank">22971172</a>]<br></li>
<li>In contrast , neither metal affords tight DNA-binding by D34A-NikR , and no shift indicative of a 4:1 NikR -- DNA complex is observed ( Fig. 5 ) , demonstrating that Asp-34 is critical for the DNA-binding response of NikR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16945905" target="_blank">16945905</a>]<br></li>
<li>Comparison of NikR with 24 Fur and Fur-like proteins of known function show conserved amino acids in N-terminal or C-terminal regions presumed important for either DNA-binding or metal-binding selectivity , respectively ( Fig. 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10648098" target="_blank">10648098</a>]<br></li>
<li>The His87-X-His89-X5-Cys95-X-Glu97 nickel-binding motif of NikR ( Fig. 3b ) is reminiscent of Zn-finger consensus sequences24 ; however , secondary structure predictions suggest that the Ni-binding domain is composed of β-sheet structure .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12524532" target="_blank">12524532</a>]<br></li>
<li>The NikR proteins also contain homologous C-terminal sequences of ; 80 residues with a unique His-X13-His-X10-His-X-His-X5-Cys motif that probably forms a nickel-binding site ~ Fig. 2 , bottom !	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10595554" target="_blank">10595554</a>]<br></li>
<li>The data presented here along with previous studies suggest that there are at least three factors that are likely important in the formation of a stable complex between NikR and the nik operator DNA with stoichiometric nickel present : ( 1 ) hydrogen bonding and electrostatic interactions formed between MBD and DNA upon R3 ordering , ( 2 ) the proper orientation of the RHHs down such that residues of this domain can make specific contacts with DNA , and ( 3 ) the binding of K + , or a similar cellular abundant cation , to a site between the RHHs and MBD , stabilizing the RHH in a down , cis DNA-binding orientation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18193897" target="_blank">18193897</a>]<br></li>
<li>The apo-NikR structure revealed a conformation that is not suitable for interaction with operator DNA ( 13 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16945905" target="_blank">16945905</a>]<br></li>
<li>The DNA-binding domains of NikR are linked to the tetrameric C domain by residues that are partially disordered in one monomer of the apo NikR structure and have different conformations in the others .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12970756" target="_blank">12970756</a>]<br></li>
<li>The nickel-activated NikR structure demonstrates that stoichiometric nickel-binding alone does not preorganize a conformation capable of operator binding ( Fig. 1B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16945905" target="_blank">16945905</a>]<br></li>
<li>The crystal structure of Ni ( II ) - NikR bound to the nik operator sequence revealed that residues in the loop preceding helix α3 in the metal-binding domain , which becomes structurally ordered upon stoichiometric nickel-binding , interact with the DNA backbone .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23737072" target="_blank">23737072</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22971172" target="_blank">22971172</a>]<br></li>
<li>The high-affinity metalbinding sites have been well characterized as nickel-binding sites , which upon nickel-binding induce a conformational change allowing NikR to bind DNA .10,11 Interestingly , nickel-binding does not appear to affect the conformation of the RHH DNA-binding domains that make specific contacts to DNA , instead altering the structure of the MBD that makes nonspecific DNA contacts ( Figure 1 ) .9 In particular , Ni2 + binding orders the α-helix 3 through its coordination of helical residue H76 ; conversely , H76 and α-helix 3 are disordered in the apo EcNikR structure .8 Crystallographic and mutagenesis data suggest that , like NikR 's high-affinity site , the second metal-binding sites are also important in modulating the affinity of NikR for DNA .9 The location of these sites at the interface of the MBD and RHH domains suggests that a metal in this site would stabilize the `` down-cis '' or the DNA-binding mode of EcNikR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19621966" target="_blank">19621966</a>]<br></li>
<li>The RHH domains are separated from the MBD by flexible linkers , and crystal structures have captured the RHH in a number of different arrangements with respect to the MBD , suggesting that the RHH domains are free to move with respect to the MBD .8,9 However , the relevant DNA-binding conformation for the RHH domains appears to be a `` down-cis '' arrangement , as seen in the EcNikR-DNA crystal structure ( Figure 1b ) .9 In addition , two identical second metal-binding sites are formed when NikR assumes a `` down-cis '' arrangement and are located at the interface of the RHH and MBD domains ( Figure 1b ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19621966" target="_blank">19621966</a>]<br></li>
<li>This conformation of NikR results in a new protein -- protein interface between the RHH and MBD and creates a second metal-binding site at this interface , which is discussed in detail below .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16945905" target="_blank">16945905</a>]<br></li>
<li>The secondary metal-binding site , formed at the interface between the RHH and MBD domains in the DNA-bound conformation of NikR and occupied by potassium in our structure , adds an additional level of complexity to the function of this transcription factor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16945905" target="_blank">16945905</a>]<br></li>
<li>The conserved secondary metal-binding site created between the RHH and MBD domains can be occupied by the intracellularly abundant metal ion K and can lock the otherwise flexible NikR in a conformation suitable for DNA-binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16945905" target="_blank">16945905</a>]<br></li>
<li>The crystal structure of fulllength NikR with stoichiometric nickel reveals domains that are comparable with the two previously determined structures of NikR , with the MBD closely matching the structure of the isolated nickel-bound MBD tetramer ( rmsd 0.89 -- 0.90 Å for 304 C atoms ) , and the RHH DNA-binding domains similar to those of the apo-NikR structure ( Fig. 1 A ) ( rmsd 0.82 -- 1.21 Å for 84 C atoms ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16945905" target="_blank">16945905</a>]<br></li>
<li>The DNA curves smoothly around one face of the NikR tetramer , bending 22 ° over its length and contacting both RHH DNA-binding domains and two of the four MBD subunits ( Fig. 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16945905" target="_blank">16945905</a>]<br></li>
<li>The initial crystal structure of apo-NikR revealed a conformation of the tetrameric protein in which the two DNA-binding domains are facing in opposite directions ( 14 ) , and it was reasonable to suggest that nickel-binding causes the RHH motifs to swing down to the same face of the protein so that they can concurrently bind to the DNA-recognition sequence without significant distortion of the double helix .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20583753" target="_blank">20583753</a>]<br></li>
<li>In the crystal structure of metal-free E. coli NikR the two DNA-binding domains are facing in opposite directions ( Figure 1 ) , in a conformation unsuitable for simultaneous DNA-binding , and the structure of the nickel-loaded protein revealed that the metal does not produce any dramatic changes in the tertiary structure .12,13 Instead , a metal-induced stabilization of secondary structural elements of the MBD was noted .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23737072" target="_blank">23737072</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22971172" target="_blank">22971172</a>]<br></li>
<li>In the crystal structure of the nickel-activated NikR tetramer in complex with its operator DNA ( Figs. 1C and 2A ) , the individual RHH and MBDs are again very similar to the previously determined structures of NikR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16945905" target="_blank">16945905</a>]<br></li>
<li>Our structures of apo-NikR , nickel-activated NikR and the NikR -- DNA complex allow us to evaluate mechanistic hypotheses regarding the metal ion-activated repressor function of EcNikR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16945905" target="_blank">16945905</a>]<br></li>
<li>Along with the previously published structure of apo-NikR , these structures allow us to evaluate functional proposals for how metal ions activate NikR , delineate the drastic conformational changes required for operator recognition , and describe the formation of a second metal-binding site in the presence of DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16945905" target="_blank">16945905</a>]<br></li>
<li>To understand how the high-affinity nickel-binding site of NikR is able to accommodate these other metal ions , and to improve our understanding of NikR 's mechanism of binding to DNA , we have determined structures of the metalbinding domain ( MBD ) of NikR in the apo form and in complex with Cu2 + and Zn2 + ions and compared them with the previously published structures with Ni2 + .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18193897" target="_blank">18193897</a>]<br></li>
<li>The disordered-to-ordered transition of helix 3 within the MBD upon nickel-binding gave preliminary insight into the structural changes that take place upon activation of NikR by metal ions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16945905" target="_blank">16945905</a>]<br></li>
<li>A crystal structure of the nickel-bound form of the isolated MBD showed that the high-affinity metal-binding site of NikR is located at the subunit interfaces , with the four nickel ions each bridging two subunits of the tetramer .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16945905" target="_blank">16945905</a>]<br></li>
<li>In the absence of bound nickel ions , the - helix contributing His-76 ( helix 3 ) to the metal-binding site is structurally flexible and was disordered in the apo-NikR crystal structure .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16945905" target="_blank">16945905</a>]<br></li>
<li>How stoichiometric nickel-binding activates DNA-binding by NikR is not completely understood , but it is likely that the mechanism involves organization of stable structures in the metal-binding domains ( 19-21 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20583753" target="_blank">20583753</a>]<br></li>
<li>The structure of the DNA complex of NikR loaded with stoichiometric nickel revealed two additional metal ions at the interface of the DNA-binding and metal-binding domains which were modeled as potassium cations ( Figure 1 ) ( 19 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20583753" target="_blank">20583753</a>]<br></li>
<li>These ions appear to be optimally placed to enhance the DNA-binding activity of NikR because they would reinforce the conformation of the protein with the DNA-binding domains in the cis orientation , and it was possible that they act as placeholders for the low-affinity nickel sites ( 19 , 23 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20583753" target="_blank">20583753</a>]<br></li>
<li>These structures establish basic architectural features of NikR , including the relative positions of the DNA-binding domains , the nickel-binding sites and the tetramerization interface .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12970756" target="_blank">12970756</a>]<br></li>
<li>In a second model of NikR activation , nickel would exert its influence on gene transcription by formation or stabilization of the tetrameric form of NikR that is required for DNA-binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12970756" target="_blank">12970756</a>]<br></li>
<li>Site-directed mutation of His87 , His89 , Cys95 or Glu97 in NikR to alanine eliminates high-affinity nickel-binding and abolishes DNA-binding but maintains stable protein folding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12524532" target="_blank">12524532</a>]<br></li>
<li>An unanticipated feature of the NikR structure is that the nickel coordination responds to DNA-binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12524532" target="_blank">12524532</a>]<br></li>
<li>Although binding of nickel to the high-affinity site of NikR does not directly alter the conformation of the RHH domains for DNA-binding , it does have a very important short-range effect on the conformation of NikR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16945905" target="_blank">16945905</a>]<br></li>
<li>Here we report the structure of the high-affinity nickel-binding site in NikR and show that it responds dramatically to DNA-binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12524532" target="_blank">12524532</a>]<br></li>
<li>We report here structural studies of the high-affinity nickel-binding site in NikR and evidence showing that its structure is sensitive to the DNA-bound state of NikR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12524532" target="_blank">12524532</a>]<br></li>
<li>Helix α3 and the preceding loop of two of the MBDs are circled and are only resolved in one of the four MBDs in the apo-NikR structure but become ordered upon stoichiometric nickel-binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23737072" target="_blank">23737072</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22971172" target="_blank">22971172</a>]<br></li>
<li>The regulatory domain of NikR contains four nickel-binding sites at the tetramer interface , each exhibiting a novel square-planar coordination by three histidines and one cysteine side chain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12970756" target="_blank">12970756</a>]<br></li>
<li>PGDH and the C domain of NikR share a common topology , a similar quaternary structure for two protein monomers and nearly identical positions for binding of their respective allosteric ligands , serine and nickel ( Fig. 1c , d ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12970756" target="_blank">12970756</a>]<br></li>
<li>Indeed , the nickel-binding sites of two NikR monomers can nearly be superimposed on two PGDH allosteric effector sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12970756" target="_blank">12970756</a>]<br></li>
<li>Our results indicate that NikR contains two distinct types of nickel-binding sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10787413" target="_blank">10787413</a>]<br></li>
<li>For NikR , the subsite spacing suggests that adjacently bound DNA-binding domains would be roughly 34 Å apart , making cooperative contacts between these N-terminal domains highly improbable in the absence of extraordinary DNA bending .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10787413" target="_blank">10787413</a>]<br></li>
<li>This spatial separation of the two DNA-binding domains explained how NikR could recognize two operator subsites distantly separated by two turns of DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16945905" target="_blank">16945905</a>]<br></li>
<li>Although apo NikR is approximately the correct size to span the larger operator , closer inspection of Figure 4b shows that the DNA-binding domains are not positioned to allow both to bind simultaneously to a linear B-form DNA molecule .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12970756" target="_blank">12970756</a>]<br></li>
<li>It is our proposal that nickel and copper 's ability to ligate this residue and in turn `` tie down '' the R3 helix and its preceding loop is related to the ability of Ni2 + - and Cu2 + - bound NikR 's ability to bind DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18193897" target="_blank">18193897</a>]<br></li>
<li>The structures of apo - , 2 + 2 + Zn - , and Cu - MBD in combination with previously determined structures and DNA-binding data provide further evidence that the ordering of the R3 helix is an important step in NikR 's recognition of DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18193897" target="_blank">18193897</a>]<br></li>
<li>Here we illustrate a correlation between the ability of a metal to order the R3 helix and that metal 's ability to induce DNA-binding and use this information along with previous results to describe a mechanism for binding of NikR to DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18193897" target="_blank">18193897</a>]<br></li>
<li>Thus , this network could be involved in stabilizing both helix R3 and the overall structure of NikR for DNA-binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18193897" target="_blank">18193897</a>]<br></li>
<li>Helix R3 , residues 65-78 ( Figure 1b , c ) , has been implicated in the mechanism of binding of NikR to DNA , due to the fact that nickel-binding orders this helix and there are contacts made by residues of this helix and the preceding loop to DNA ( Figure 1c ) ( 20 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18193897" target="_blank">18193897</a>]<br></li>
<li>Ordering of the R3 helix and the loop preceding it ( loop 64-65 ) has previously been implicated in the mechanism of binding of NikR to DNA ( 19 , 20 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18193897" target="_blank">18193897</a>]<br></li>
<li>E. coli NikR is a modular protein : residues at the N terminus ( Met1 -- His47 ) show homology with ribbon-helix-helix transcription factors , including the Arc , Mnt , CopG , MetJ , TraY and ω repressors , and are responsible for binding DNA6 , whereas the C-terminal two-thirds of the protein is responsible for tetramerization and highaffinity nickel-binding4 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12970756" target="_blank">12970756</a>]<br></li>
<li>RESULTS Overall features of full-length apo-NikR NikR is a homotetramer with a simple modular design : dimeric ribbon-helix-helix DNA-binding domains are attached at either end of the tetrameric C-terminal regulatory domain ( Fig. 1a ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12970756" target="_blank">12970756</a>]<br></li>
<li>The ribbon-helix-helix domain and operator recognition The DNA-binding domain of NikR has a dimeric structure similar to other proteins in the ribbon-helix-helix family .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12970756" target="_blank">12970756</a>]<br></li>
<li>Mutagenesis experiments and homology with other ribbon-helix-helix proteins suggest that residues on the surface of the antiparallel β-sheet in the NikR DNA-binding domain should contact the major groove in each nik operator subsite6 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12970756" target="_blank">12970756</a>]<br></li>
<li>Finally , we show that the N-terminal domain of NikR binds to a DNA site near the promoter of the nikABCDE operon and also responds to mutations in the putative b-sheet as expected for a ribbon-helix-helix protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10595554" target="_blank">10595554</a>]<br></li>
<li>In the case of the N-terminal domain of NikR , there are a number of indicators in addition to sequence similarity that support its assignment to the ribbon-helix-helix family : the N-terminal domain of NikR matches best to Arc , a b-a-a protein , in structure prediction experiments ; this domain is stable to limited proteolysis and has a CD spectrum expected for a protein with the same a-helical content as Arc ; the native N-terminal domain , like Arc , is dimeric and shows concentration dependent stability ; and mutation of a predicted DNA-binding residue reduces DNA-binding but does not affect protein secondary structure or stability .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10595554" target="_blank">10595554</a>]<br></li>
<li>Complementary interaction of the nickel-stabilized MBD with DNA would localize NikR to the DNA helix and allow cooperative interaction of both RHH domains .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16945905" target="_blank">16945905</a>]<br></li>
<li>NikR is a homotetrameric protein arranged with a central tetrameric MBD and two flanking dimeric DNA-binding ribbon-helix-helix ( RHH ) domains ( Figure 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18193897" target="_blank">18193897</a>]<br></li>
<li>The MBD of NikR ( residues 48-133 ) contains the residues involved in high-affinity metal-binding but lacks the N-terminal ribbon-helix-helix ( RHH ) DNA-binding domains .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18193897" target="_blank">18193897</a>]<br></li>
<li>First , nickel-binding creates or stabilizes a form of NikR in which the ribbon-helix-helix domains are positioned correctly for operator subsite binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12970756" target="_blank">12970756</a>]<br></li>
<li>NikR is the only metal-responsive member of the ribbon-helix-helix ( RHH ) 1 family of DNA-binding proteins ( 8-10 ) , and it controls transcription of the nik operon , which encodes an ABC-type nickel uptake transporter ( 11 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20583753" target="_blank">20583753</a>]<br></li>
<li>NikR is the only known metal-responsive member of the ribbon-helix-helix family of transcription factors , and its structure has a quaternary arrangement consisting of two dimeric DNA-binding domains separated by a tetrameric regulatory domain that binds nickel .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12970756" target="_blank">12970756</a>]<br></li>
<li>Copyright © 1999 The Protein Society NikR is a ribbon-helix-helix DNA-binding protein PETER T. CHIVERS and ROBERT T. SAUER Department of Biology , Massachusetts Institute of Technology , Cambridge , Massachusetts 02139 ~ Received April 15 , 1999 ; Accepted August 13 , 1999 !	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10595554" target="_blank">10595554</a>]<br></li>
				<label class="DOM_collapse_similarity" for="dom_similarity_143">
Chivers , P.T. & Sauer , R.T. NikR is a ribbon-helix-helix DNA-binding protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12524532" target="_blank">12524532</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12970756" target="_blank">12970756</a>]<br></label>
<input id="dom_similarity_143", type="checkbox">
<div class="DOM_similar">
<li>Chivers , P. T. , and Sauer , R. T. ( 1999 ) NikR is a ribbon-helix-helix DNA-binding protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20583753" target="_blank">20583753</a>]<br></li>
<li>( 41 ) Chivers , P. T. , and Sauer , R. T. ( 1999 ) NikR is a ribbon-helix-helix DNA-binding protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23737072" target="_blank">23737072</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22971172" target="_blank">22971172</a>]<br></li></div>
<li>DNA-binding of wild-type and mutant domains NikR 's function as a genetic repressor suggests that it is likely to be a sequence specific DNA-binding protein ~ de Pina et al. , 1999 !	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10595554" target="_blank">10595554</a>]<br></li>
<li>Finally , NikR is a transcriptional repressor and DNA-binding protein and thus shares common functions with other members of the b-a-a superfamily .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10595554" target="_blank">10595554</a>]<br></li>
<li>Experiments with 10 pM protein revealed close to half-maximal DNA-binding with H125N or H92A NikR in the presence o excess Ni ( II ) ( Figure 6 and Table 1 ) , similar to wild-type protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20583753" target="_blank">20583753</a>]<br></li>
<li>Cys128 , His123 0 , and His1250 compose a cluster of metal-binding residues that appear to be suitably located to bind a Ni ( II ) ion at the interface with the DNA-binding domain ; however , these experiments also demonstrated that substitution of His125 did not affect the low-affinity Ni ( II ) - induced DNA-binding response of NikR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20583753" target="_blank">20583753</a>]<br></li>
<li>These data support the model that the nonspeciﬁc interactions between the metal-binding domain of the protein and the DNA phosphodiester backbone are critical for the Ni ( II ) - responsive activity of E. coli NikR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23737072" target="_blank">23737072</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22971172" target="_blank">22971172</a>]<br></li>
<li>To determine whether NarL - and NikR-dependent regulation of Pnik expression were independent , LacZ levels were measured in nikR-deﬁcient and nikR narLX-deﬁcient strains containing wild-type NikR or a mutant protein , the Cys95Ala protein , which lacks high-afﬁnity nickel-binding activity and shows no DNA-binding in vitro with up to 1 M NiCl2 ( 7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16159764" target="_blank">16159764</a>]<br></li>
<li>For example , the crystal structure of Helicobacter pylori NikR revealed external nickelbinding sites at the surface of the tetramerization interface of the protein ( 26 ) , and a recent study demonstrated that these external sites are important for DNA-binding to some promoters ( 27 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20583753" target="_blank">20583753</a>]<br></li>
<li>However , biochemical characterization of the isolated mutant protein demonstrated that both the highafinity nickel-binding activity and the overall secondary structure are preserved , suggesting that the substitutions speciﬁcally disrupt the nickel-responsive DNA-binding activity of NikR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23737072" target="_blank">23737072</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22971172" target="_blank">22971172</a>]<br></li>
<li>Glu-30 , Arg-33 , and Asp-34 originate from the second helix of the RHH domain , explaining the high degree of conservation along this helix among NikR sequences that is not seen for other RHH repressors .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16945905" target="_blank">16945905</a>]<br></li>
<li>The high degree of conservation of the residues of the second helix of the ribbon-helix-helix among NikR homologs suggests that this helix could be a region to which a signal is propagated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12970756" target="_blank">12970756</a>]<br></li>
<li>The topology and tertiary organization of the NikR C domain also matches the regulatory domain of LrpA17 , thought to be representative of the leucine-responsive regulatory protein ( Lrp ) family of transcription factors .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12970756" target="_blank">12970756</a>]<br></li>
<li>Nevertheless , both profile searches and structure prediction suggest that NikR is a member of the ribbon-helix-helix protein family .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10595554" target="_blank">10595554</a>]<br></li>
				<label class="DOM_collapse_similarity" for="dom_similarity_158">
NikR is the only known metal-responsive member of the ribbon-helix-helix family of transcription factors18 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12524532" target="_blank">12524532</a>]<br></label>
<input id="dom_similarity_158", type="checkbox">
<div class="DOM_similar">
<li>NikR is the only known metal-responsive member of the ribbon-helix-helix family of transcription factors .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12970756" target="_blank">12970756</a>]<br></li></div>
<li>Here , we report profile searches that identify the N-terminal domain of NikR repressor as a potential member of the ribbon-helix-helix family and demonstrate that the biochemical and biophysical properties of this NikR domain are consistent with a b-a-a fold .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10595554" target="_blank">10595554</a>]<br></li>
<li>The N-terminal sequences of two NikR monomers intertwine to form each C2-sym-metric ribbon-helix-helix domain , as has been observed for other family members7 -- 11 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12970756" target="_blank">12970756</a>]<br></li>
<li>RESULTS In a previous study ( 7 ) , we showed that the purified ribbon-helix-helix domain of NikR protected two regions of DNA within the nik promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10787413" target="_blank">10787413</a>]<br></li>
<li>Moreover , these sequences were roughly centered within the two regions protected by the ribbon-helix-helix domain of NikR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10787413" target="_blank">10787413</a>]<br></li>
<li>However , unlike the N-termi-nal domain alone , full-length NikR also protected the central region of the operator site from DNase I digestion .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10787413" target="_blank">10787413</a>]<br></li>
<li>This result makes it highly unlikely that NikR makes base-specific interactions with this central region of the operator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10787413" target="_blank">10787413</a>]<br></li>
<li>The Ni-NikR structure was solved by molecular replacement by using the program EPMR ( 31 ) with a model of a dimer of the MBD ( PDB ID code 1Q5Y ) and a dimer of the RHH domain from the apo-NikR structure ( ID code 1Q5V ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16945905" target="_blank">16945905</a>]<br></li>
<li>Coordinates and structure factors for the Apo-NikR and Nibound regulatory domain structures have been deposited in the Protein Data Bank ( accession codes 1Q5V and 1Q5Y , respectively ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12970756" target="_blank">12970756</a>]<br></li>
<li>NikR contains two domains : an N-domain ( ∼ 45 residues ) , which forms dimers and contacts operator DNA18 , and a C-domain ( ∼ 85 residues ) , which forms tetramers and binds Ni2 + ions avidly ( Kd = 7 pM ) 19 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12524532" target="_blank">12524532</a>]<br></li>
<li>If the N-terminal region of NikR does adopt the b-a-a fold , then it would be expected to fold independently as a dimer and to have substantial a-helical secondary structure .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10595554" target="_blank">10595554</a>]<br></li>
<li>Therefore , a slightly shorter fragment consisting of E. coli NikR residues 1 to 48 with a C-terminal 6His-tag was cloned , expressed , and purified .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10595554" target="_blank">10595554</a>]<br></li>
<li>Structure predictiona results for NikR N-terminal domains Z-scoreb NikR E. coli ~ 1 -- 48 !	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10595554" target="_blank">10595554</a>]<br></li>
<li>Analytical ultracentrifugation A protein sample containing 6.25 mM NikR N-terminal domain in S buffer were centrifuged at 31,000 rpm in a Beckman Optim XLA analytical ultracentrifuge .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10595554" target="_blank">10595554</a>]<br></li>
<li>Consistent with the role of NikR as a transcriptional repressor , we have previously demonstrated that the N-terminal domain dimer binds weakly to two regions within the nikABCDE promoter ( Pnik ) ( 7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10787413" target="_blank">10787413</a>]<br></li>
<li>The secondary structure of the NikR N-terminal domain was lost in a cooperative fashion in both thermal and GuHCl denaturation ~ Fig. 3B , C !	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10595554" target="_blank">10595554</a>]<br></li>
<li>The Arc structure scored highest against four of the five NikR N-terminal domain sequences and was the second highest hit for the remaining sequence , although the statistical significance of these scores was borderline ~ Table 2 !	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10595554" target="_blank">10595554</a>]<br></li>
<li>The N-terminal domain sequences of five NikR proteins were submitted for structure prediction by the method of Fischer and 9 24 Eisenberg ~ 1996 !	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10595554" target="_blank">10595554</a>]<br></li>
<li>The N-terminal domain sequences of five NikR proteins were submitted for structure prediction with the following options : H3P25no ; TOPITS5no ; HTML5yes ; and PROFILESEARCH5no .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10595554" target="_blank">10595554</a>]<br></li>
<li>Each of these NikR proteins consists of an N-terminal rib-bon-helix-helix domain of approximately 50 residues and a NikRspecific C-terminal domain of about 80 residues .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10787413" target="_blank">10787413</a>]<br></li>
<li>These results support the model that the N-terminal domain of E. coli NikR adopts a homodimeric b-a-a fold .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10595554" target="_blank">10595554</a>]<br></li>
<li>Stabilities of wild-type and RA3 NikR N-terminal domain dimers a Tm Cm m ~ 8C !	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10595554" target="_blank">10595554</a>]<br></li>
<li>Dnase I footprinting of the nikABCDE promoter region by 6.25 mM of the wild-type or RA3 N-terminal domain of NikR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10595554" target="_blank">10595554</a>]<br></li>
<li>In contrast , more than 95 % of NikR N-terminal domain was soluble after dialysis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10595554" target="_blank">10595554</a>]<br></li>
<li>bSWISSPROT ID number tion for the N-terminal domain of NikR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10595554" target="_blank">10595554</a>]<br></li>
<li>Biophysical properties of NikR N-terminal domain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10595554" target="_blank">10595554</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10595554" target="_blank">10595554</a>]<br></li>
<li>The gray bars indicate bases previously shown to be protected by the NikR N-terminal domain ( 7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10787413" target="_blank">10787413</a>]<br></li>
<li>The corresponding C-terminal domain fragment of NikR ~ residues 54 -- 138 !	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10595554" target="_blank">10595554</a>]<br></li>
<li>The outer flanks of the NikR footprint were similar to those of the isolated N-terminal domain footprint ( 7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10787413" target="_blank">10787413</a>]<br></li>
<li>Biophysical and biochemical characterization of the N-terminal domain of NikR show that it has many features expected of a b-a-a protein including a-helical content , dimeric solution form , concentration dependent thermal stability , and ability to bind DNA in sequence-specific manner .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10595554" target="_blank">10595554</a>]<br></li>
<li>Indeed , a fragment corresponding to the C-terminal domain of E. coli NikR stably folds in solution , is a monomer , and binds nickel ~ P.T. Chivers & R.T. Sauer , unpubl .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10595554" target="_blank">10595554</a>]<br></li>
<li>Benanti , E. L. , and Chivers , P. T. ( 2007 ) The N-terminal arm of the Helicobacter pylori Ni - dependent transcription factor NikR is 2þ required for specific DNA-binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20583753" target="_blank">20583753</a>]<br></li>
				<label class="DOM_collapse_similarity" for="dom_similarity_191">
Chivers , P.T. & Sauer , R.T. NikR repressor : high-affinity nickel-binding to the Cterminal domain regulates binding to operator DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12524532" target="_blank">12524532</a>]<br></label>
<input id="dom_similarity_191", type="checkbox">
<div class="DOM_similar">
<li>Chivers , P.T. & Sauer , R.T. NikR repressor : high-affinity nickel-binding to the C-termi-nal domain regulates binding to operator DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12970756" target="_blank">12970756</a>]<br></li></div>
<li>NikR repressor : high-afﬁnity nickel-binding to the C-terminal domain regulates binding to operator DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16159764" target="_blank">16159764</a>]<br></li>
				<label class="DOM_collapse_similarity" for="dom_similarity_194">
Chivers , P. T. , and Sauer , R. T. ( 2002 ) NikR repressor : High-affinity nickel-binding to the C-terminal domain regulates binding to operator DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20583753" target="_blank">20583753</a>]<br></label>
<input id="dom_similarity_194", type="checkbox">
<div class="DOM_similar">
<li>( 16 ) Chivers , P. T. , and Sauer , R. T. ( 2002 ) NikR repressor : highaffinity nickel-binding to the C-terminal domain regulates binding to operator DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23737072" target="_blank">23737072</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22971172" target="_blank">22971172</a>]<br></li></div>
<li>Ni site structure responds to DNA-binding Despite the observations that all the amino acids required for the high-affinity nickel-binding site in NikR are contained in the C-terminal domain and that high-affinity nickel-binding is retained in the C-terminal fragment , the structure of the nickel site is sensitive to DNA-binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12524532" target="_blank">12524532</a>]<br></li>
<li>In addition to both GTATGA sites and nickel , high affinity operator binding also requires the C-terminal domain of NikR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10787413" target="_blank">10787413</a>]<br></li>
<li>Because nickel-binding and DNA-binding are mediated by different domains within NikR , a communication link between the two domains is implicated , consistent with the finding that the nickel-binding site in a fragment corresponding to the C-terminal domain of NikR is structurally distinct from that found in holo-NikR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12524532" target="_blank">12524532</a>]<br></li>
<li>The C-terminal nickel-binding domain of NikR contains several conserved residues that are potential nickel ligands .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12524532" target="_blank">12524532</a>]<br></li>
<li>XANES and EXAFS spectra of the nickel-binding site in the isolated C-terminal domain were clearly different from those of intact NikR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12524532" target="_blank">12524532</a>]<br></li>
<li>Binding was dramatically reduced if nickel was not present throughout the binding assay or if the NikR C-terminal domain was removed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10787413" target="_blank">10787413</a>]<br></li>
<li>In preliminary experiments , we have also found that a subset of mutations within the DNA region protected by the N-terminal domain reduce the affinity of NikR for the operator ~ data not shown !	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10595554" target="_blank">10595554</a>]<br></li>
<li>In principle , the C-terminal domain might help mediate direct Ni21-dependent contacts with operator DNA and/or mediate cooperative interactions between NikR molecules bound to each operator subsite .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10787413" target="_blank">10787413</a>]<br></li>
<li>The position of the C-terminal regulatory domain enforces a large spacing between the contacts that each NikR DNA-binding domain can make with the nik operator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12970756" target="_blank">12970756</a>]<br></li>
<li>These experiments support a model for NikR function in which the N-terminal domain binds DNA subsites and the C-terminal domain binds Ni21 and regulates operator binding via specific interactions between adjacently bound NikR dimers .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10787413" target="_blank">10787413</a>]<br></li>
<li>To test this model , DNase footprinting was used to assay binding of the N-terminal domain of E. coli NikR to a DNA fragment containing the nikABCDE promoter region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10595554" target="_blank">10595554</a>]<br></li>
<li>These results confirm that the N-terminal domain of NikR is responsible for DNA-recognition .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10595554" target="_blank">10595554</a>]<br></li>
<li>DNA encoding the NikR N-terminal domain ~ residues 1 -- 48 !	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10595554" target="_blank">10595554</a>]<br></li>
<li>, another prediction of the b-a-a model for the N-terminal domain of NikR is that a mutation at the first surface position of the presumed b-sheet should interfere with DNA-binding but not affect protein stability .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10595554" target="_blank">10595554</a>]<br></li>
<li>mutant of the NikR N-terminal domain and assayed protein stability and DNA protection .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10595554" target="_blank">10595554</a>]<br></li>
<li>Because both Ni21 ions and the C-terminal domain are also required for the observation of stable complexes , it seems likely that Ni21-dependent interactions between the C-terminal domains of NikR mediate dimer-dimer contacts .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10787413" target="_blank">10787413</a>]<br></li>
<li>It seems likely that NikR senses high concentrations of nickel through interactions of this metal with its C-terminal domain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10595554" target="_blank">10595554</a>]<br></li>
<li>It seems likely , therefore , that contacts involving the C-terminal domain of NikR play some role in cooperative binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10787413" target="_blank">10787413</a>]<br></li>
<li>In experiments using the purified N-terminal domain of NikR , no mobility shift complex was observed at protein concentrations as high as 10 mM , either in the presence or absence of nickel ( data not shown ) , indicating that the C-terminal domain is required for the formation of a stable mobility shift complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10787413" target="_blank">10787413</a>]<br></li>
<li>For example , although the isolated N-terminal domain of NikR protects the operator in footprinting experiments ( 7 ) , this binding is observed at concentrations 10,000-fold higher than those required for binding of intact NikR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10787413" target="_blank">10787413</a>]<br></li>
<li>To gain insight into how nickel regulates NikR function , thereby regulating its own uptake , we have determined the crystal structures of two forms of E. coli NikR : full-length apo NikR at a resolution of 2.3 Å and the nickel-bound C-terminal regulatory domain at a resolution of 1.4 Å .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12970756" target="_blank">12970756</a>]<br></li>
<li>Protein concentrations were determined using extinction coefficients of E280 5 3,960 M21 cm21 for the full-length protein at pH 7.5 and E 5 2,600 M21 21 295 cm for the N-terminal domain of NikR at pH 12 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10595554" target="_blank">10595554</a>]<br></li>
<li>These structural findings along with biochemical data on NikR support a hypothesis that ordering of the R3 helix is important for repressor activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18193897" target="_blank">18193897</a>]<br></li>
<li>NikR crystal structures sometimes show little or no electron density in the region of helix R3 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18193897" target="_blank">18193897</a>]<br></li>
<li>This network stretches across the MBD interface and includes two residues from the R3 helix , but the network is not ordered in the apo-NikR ( 19 ) or apo-MBD structures .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18193897" target="_blank">18193897</a>]<br></li>
<li>As described previously , residues 62 -- 80 of NikR are disordered in the absence of nickel but form a loop and a stable - helix ( helix 3 ) when nickel is bound because of the direct involvement of His-76 in nickel coordination ( 13 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16945905" target="_blank">16945905</a>]<br></li>
<li>One impact of nickel is organization of helix α3 and the preceding loop , which is predominantly disordered in the structure of E. coli NikR in the apo form .12,13 Ordering of loop / helix α3 was also observed in the structure of the MBD loaded with Cu ( II ) ,23 which binds in a very similar square-planar site as Ni ( II ) ,23,38 but not when the protein was loaded with noncognate zinc ions that bind with very different coordination .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23737072" target="_blank">23737072</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22971172" target="_blank">22971172</a>]<br></li>
<li>Divalent nickel and copper bind tightly to NikR and both R are able to order the 3 helix , while divalent zinc binds less R tightly and is incapable of ordering the 3 helix ( Figure 4 and Figure S1 of the Supporting Information ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18193897" target="_blank">18193897</a>]<br></li>
<li>Proteins rich in histidine and cysteine often bind divalent transition metals ( 18 ) ; in NikR , these residues may bind nickel ions ( 7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10787413" target="_blank">10787413</a>]<br></li>
<li>Indeed , our results suggest that relatively distant residues on the EcNikR protein itself may influence the identity of the ion in the second metalbinding site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19621966" target="_blank">19621966</a>]<br></li>
<li>Taken together , results from crystallography and mobility-shift assays suggest that this conserved second metal site is physiologically relevant , critical for the function of NikR , and well suited to bind a metal such as K .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16945905" target="_blank">16945905</a>]<br></li>
<li>In addition , crystallographic data on EcNikR do not support the idea that Ni2 + binds in the second metal sites , instead showing multiple low-affinity sites on the surface of the EcNikR protein .9 The suggestion that K + might be the relevant metal in the second metal-binding sites also comes from both biochemical and structural data .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19621966" target="_blank">19621966</a>]<br></li>
<li>Crystal structures of EcNikR and the metal-binding sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19621966" target="_blank">19621966</a>]<br></li>
<li>Supporting Information Available : Average energies for both the EcNikR high-affinity site studies ( Figure S1 ) and the EcNikR-DNA second metal-binding site studies ( Figure S3 ) ; metal-ligand distances at one of the four high-affinity metal-binding sites for the four known ligands during the EcNikR high-affinity site studies ( Figure S2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19621966" target="_blank">19621966</a>]<br></li>
<li>We begin with an analysis of the EcNikR structure when either K + or Ni2 + occupies the high-affinity or the second metal-binding sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19621966" target="_blank">19621966</a>]<br></li>
<li>In total , these data clarify the physical basis for metal ion specificity in both the high-affinity and second metal-binding sites in EcNikR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19621966" target="_blank">19621966</a>]<br></li>
<li>To determine which ion is preferred at the second metal-binding site and the physical basis for any preference of one ion over another in both the second metal-binding site and the high-affinity sites , we conducted a series of detailed molecular simulations on the EcNikR structure .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19621966" target="_blank">19621966</a>]<br></li>
<li>To decipher the physical basis underlying any difference in metal-binding specificity at both the high-affinity and second metal-binding sites , we conducted a series of detailed molecular simulations on the EcNikR structure .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19621966" target="_blank">19621966</a>]<br></li>
<li>An analysis of Ni2 + coordination geometries obtained from eight structures in the Protein Data Bank and over 1000 structures in the Cambridge Structural Database reveals that similar geometries have been observed in other crystal structures that contain bound Ni2 + ions .20 -- 22 Molecular dynamics simulations of EcNikR complexed to DNA with either K + or Ni2 + in the second metal-binding site confirm that the octahedral geometry and the distorted square-pyramidal geometry for K + and Ni2 + , respectively , represent stable geometries for both ions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19621966" target="_blank">19621966</a>]<br></li>
<li>In the presence of DNA , a second metal-binding site is observed at the interface generated between the RHH domain and the MBD of EcNikR ( Figs. 1C , 2 A , and 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16945905" target="_blank">16945905</a>]<br></li>
<li>The second metal-binding sites in EcNikR are located at the interface of the MBD and RHH domains and are composed solely of oxygen ligands from each domain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19621966" target="_blank">19621966</a>]<br></li>
<li>This same secondary metal-binding site was observed in the structure of nickel-activated PhNikR determined from crystals soaked in 5 mM Ni2 and 1 mM phosphate in the absence of DNA ( Fig. 7 B and C ) ( 14 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16945905" target="_blank">16945905</a>]<br></li>
<li>An analysis of the energetic contributions to the electrostatic binding free energy suggests that , while the interaction between EcNikR and DNA is more favorable when the second site contains Ni2 + , the large desolvation penalty associated with moving Ni2 + from solution to the relatively buried second site offsets this favorable interaction term .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19621966" target="_blank">19621966</a>]<br></li>
<li>This raises the possibility that EcNikR DNA-binding may be afforded by another monovalent cation in the second metal-binding site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19621966" target="_blank">19621966</a>]<br></li>
<li>Electrostatic binding free energy calculations , however , suggest that EcNikR binding to DNA is more favorable when the second metal-binding site contains K + .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19621966" target="_blank">19621966</a>]<br></li>
<li>In the first reaction , EcNikR binds DNA with K + in the second metal-binding site , and the associated free energy difference is ΔGK .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19621966" target="_blank">19621966</a>]<br></li>
<li>Although under these conditions PhNikR has nickel in this metal-binding site , two pieces of evidence indicate that this second site is not occupied by nickel in our EcNikR-DNA structure : metal analysis and anomalous dispersion .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16945905" target="_blank">16945905</a>]<br></li>
<li>To explore whether the second metal-binding site can rearrange to adopt conformations that are favorable for Ni2 + , we replaced K + with Ni2 + in the crystal structure of EcNikR bound to DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19621966" target="_blank">19621966</a>]<br></li>
				</div></ul><br>
			<label class="collapse_EVO" for="_3"><font size="5px">NikR evolution</font></label>
				<input id="_3" type="checkbox">
				<div class="EVO"><br><ul><li>Apo and apo-SeMet NikR crystals belonged to space group P212121 and were cryoprotected using Paratone-N oil .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12970756" target="_blank">12970756</a>]<br></li>
<li>The isolated MBD is tetrameric and has a topology identical to that of the MBD portion of the fulllength NikR protein ( Figure 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18193897" target="_blank">18193897</a>]<br></li>
<li>However , the addition of the CopG and NikR families to the b-a-a class has more than doubled the number of known superfamily members in the past year alone .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10595554" target="_blank">10595554</a>]<br></li>
<li>Finally , NikR is a transcriptional repressor and DNA-binding protein and thus shares common functions with other members of the b-a-a superfamily .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10595554" target="_blank">10595554</a>]<br></li>
<li>Percentage pairwise sequence identity between ribbon-helix-helix subfamily members a Arc Mnt MetJ TraY CopG NikR P03050b P03049 P08338 P06627 P13920 P28910 Arc Mnt 13 11 17 19 22 -- 17 13 11 15 17 15 15 11 17 -- 37 MetJ TraY CopG NikR Fig. 5 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10595554" target="_blank">10595554</a>]<br></li>
<li>Metal-responsive DNA-binding proteins that control transporter and enzyme gene expression in bacteria are well known .1 These metalloregulators have been divided into several structural families , named after their founding members : MerR , ArsR/SmtB , DtxR , Fur , NikR , and CopY .1 Each family , with the exception of NikR , contains sub-families that respond to distinct metals , or other small molecules .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18505253" target="_blank">18505253</a>]<br></li>
<li>This suggests that it may not always be straightforward to predict the dimer recognition subsites for other NikR family members .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10787413" target="_blank">10787413</a>]<br></li>
<li>The topology and tertiary organization of the NikR C domain also matches the regulatory domain of LrpA17 , thought to be representative of the leucine-responsive regulatory protein ( Lrp ) family of transcription factors .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12970756" target="_blank">12970756</a>]<br></li>
<li>The ribbon-helix-helix domain and operator recognition The DNA-binding domain of NikR has a dimeric structure similar to other proteins in the ribbon-helix-helix family .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12970756" target="_blank">12970756</a>]<br></li>
<li>The NikR half-sites are similar in size to subsites recognized by other ribbon-helix-helix proteins , but the spacing between these sites is much greater than for other family members .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10787413" target="_blank">10787413</a>]<br></li>
<li>NikR is the only metal-responsive member of the ribbon-helix-helix ( RHH ) 1 family of DNA-binding proteins ( 8-10 ) , and it controls transcription of the nik operon , which encodes an ABC-type nickel uptake transporter ( 11 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20583753" target="_blank">20583753</a>]<br></li>
				<label class="EVO_collapse_similarity" for="evo_similarity_12">
NikR is the only known metal-responsive member of the ribbon-helix-helix family of transcription factors18 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12524532" target="_blank">12524532</a>]<br></label>
<input id="evo_similarity_12", type="checkbox">
<div class="EVO_similar">
<li>NikR is the only known metal-responsive member of the ribbon-helix-helix family of transcription factors .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12970756" target="_blank">12970756</a>]<br></li></div>
<li>Here , we report profile searches that identify the N-terminal domain of NikR repressor as a potential member of the ribbon-helix-helix family and demonstrate that the biochemical and biophysical properties of this NikR domain are consistent with a b-a-a fold .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10595554" target="_blank">10595554</a>]<br></li>
<li>Nevertheless , both profile searches and structure prediction suggest that NikR is a member of the ribbon-helix-helix protein family .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10595554" target="_blank">10595554</a>]<br></li>
				</div></ul><br>
			<label class="collapse_RP" for="_4"><font size="5px">Processes regulated by NikR</font></label>
				<input id="_4" type="checkbox">
				<div class="RP"><br><ul><li>The final plasmid construct ( pNIK103 ) is a derivative of pET22-b ( Novagen , Madison , WI ) and encodes the wild-type NikR repressor under transcriptional-control of a T7 promoter and an lac operator site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10787413" target="_blank">10787413</a>]<br></li>
<li>Since a protein homologous to NikR has recently been identified in the gastric pathogen H. pylori ( De Pina et al. 1999 ) whose urease activity acts as a virulence factor , the possibility that NikR participates in bacterial pathogenesis should also be considered .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10648098" target="_blank">10648098</a>]<br></li>
<li>The Fur proteins act as negative regulators of the expression of bacterial operons for highaffinity iron uptake pathways and thus display a similar regulatory role to that of NikR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10648098" target="_blank">10648098</a>]<br></li>
<li>In Escherichia coli , one of the principle mechan-isms of nickel homeostasis is the tight control of expression of a nickel importer by the nickel-responsive transcriptional regulator NikR ( 5-7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20583753" target="_blank">20583753</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_5">
Leitch , S. , Bradley , M. J. , Rowe , J. L. , Chivers , P. T. , and Maroney , M. J. ( 2007 ) Nickel-specific response in the transcriptional regulator , Escherichia coli NikR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20583753" target="_blank">20583753</a>]<br></label>
<input id="rp_similarity_5", type="checkbox">
<div class="RP_similar">
<li>( 38 ) Leitch , S. , Bradley , M. J. , Rowe , J. L. , Chivers , P. T. , and Maroney , M. J. ( 2007 ) Nickel-specific response in the transcriptional regulator , Escherichia coli NikR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23737072" target="_blank">23737072</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22971172" target="_blank">22971172</a>]<br></li></div>
<li>Such cross-regulation has been observed in species like Helicobacter pylori , which utilizes the NikR regulator to coordinate expression of both iron and nickel metal homeostasis systems ( 13 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21097627" target="_blank">21097627</a>]<br></li>
<li>In contrast to Helicobacter pylori , NikR from E. coli is not involved in positive gene-regulation ( Ernst et al. 2005 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17120142" target="_blank">17120142</a>]<br></li>
<li>Ernst FD , Kuipers EJ , Heijens A , et al. 2005 The nickelresponsive regulator NikR controls activation and repression of gene transcription in Helicobacter pylori .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17120142" target="_blank">17120142</a>]<br></li>
<li>Introduction Monitoring the concentration of metal ions within the cell is of crucial importance , as a relatively small concentration of some metal ions is necessary for a number of cellular-processes to occur , but often too much of that same ion may cause cell damage and death .1 -- 4 Nickel is an essential cofactor for a number of bacterial enzymes that are involved in cell survival , including hydrogenase and urease , two enzymes vital to the survival of the pathogenic bacteria Helicobacter pylori .1 The concentration of nickel in many bacteria is regulated by the nickel-binding transcription factor , NikR , which transcriptionally represses genes encoding nickel-specific importers .5 The best-characterized NikR homologues are from Escherichia coli ( EcNikR ) , H. pylori ( HpNikR ) , and Pyrococcus horikoshi ( PhNikR ) .6 -- 9 Each is a homotetramer composed of two domains : the central tetrameric metal-binding domain ( MBD ) and two flanking dimeric ribbon -- helix -- helix ( RHH ) domains ( Figure 1a ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19621966" target="_blank">19621966</a>]<br></li>
<li>Transcriptome analysis of a deletion mutant and in vitro promoter-binding assays have revealed that H. pylori NikR ( HpNikR ) regulates transcription of not only the nickel importer and urease enzyme , but also other regulatory networks , some through repression of the gene encoding the ferric uptake regulator ( Fur ) protein ( 5 -- 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16945905" target="_blank">16945905</a>]<br></li>
<li>NikR , a protein of the ribbon-helix-helix family of transcription factors , represses expression of the nikABCDE operon in the presence of excessive concentrations of intracellular nickel .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10787413" target="_blank">10787413</a>]<br></li>
<li>Crystal structure of the nickel-responsive transcription NikR is a metal-responsive transcription factor that controls nickel uptake in Escherichia coli by regulating expression of a nickelspecific ATP-binding cassette ( ABC ) transporter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12970756" target="_blank">12970756</a>]<br></li>
<li>Structural Basis of the Metal Specificity for Nickel Regulatory Protein NikR † , ‡ ABSTRACT : In the presence of excess nickel , Escherichia coli NikR regulates cellular nickel uptake by suppressing the transcription of the nik operon , which encodes the nickel uptake transporter , NikABCDE .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18193897" target="_blank">18193897</a>]<br></li>
<li>Nickel binding at high-affinity sites in E. coli NikR ( EcNikR ) facilitates EcNikR binding to the nik operon , where it then suppresses transcription of genes encoding the nickel uptake transporter , NikABCDE .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19621966" target="_blank">19621966</a>]<br></li>
<li>In addition to controlling the transcription of NixA ( 6 ) , NikR regulates a number of genes in H. pylori , including urease accessory protein UreA ( 12 ) and outer membrane proteins FepA3 ( 13 ) , FrpB4 ( 2 , 13 ) , and ExbB ( 14 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18193897" target="_blank">18193897</a>]<br></li>
<li>Perspectives The transcription of nik genes is stimulated by the global regulator FNR under anoxic conditions , and negatively controlled by a new repressor protein , NikR , responding specifically to Ni .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10648098" target="_blank">10648098</a>]<br></li>
				</div></ul><br>
			<label class="collapse_SIT" for="_5"><font size="5px">NikR binding site</font></label>
				<input id="_5" type="checkbox">
				<div class="SIT"><br><ul><li>( a ) Ni2 + - bound EcNikR ( PDB 2HZA ) and the square-planar highaffinity nickel-binding site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19621966" target="_blank">19621966</a>]<br></li>
<li>Percentage pairwise sequence identity between ribbon-helix-helix subfamily members a Arc Mnt MetJ TraY CopG NikR P03050b P03049 P08338 P06627 P13920 P28910 Arc Mnt 13 11 17 19 22 -- 17 13 11 15 17 15 15 11 17 -- 37 MetJ TraY CopG NikR Fig. 5 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10595554" target="_blank">10595554</a>]<br></li>
<li>Is the extended inverted-repeat sequence important for the function of NikR or an additional transcription factor ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10787413" target="_blank">10787413</a>]<br></li>
<li>Fig. 2 Multiple sequence alignment of NikR and Fur-like proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10648098" target="_blank">10648098</a>]<br></li>
<li>Residues 49 to 55 of the E. coli NikR sequence probably form a flexible interdomain linker since this region is relatively glycine-rich and contains gaps in the alignment ~ Fig. 2 !	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10595554" target="_blank">10595554</a>]<br></li>
<li>The NikR proteins also contain homologous C-terminal sequences of ; 80 residues with a unique His-X13-His-X10-His-X-His-X5-Cys motif that probably forms a nickel binding site ~ Fig. 2 , bottom !	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10595554" target="_blank">10595554</a>]<br></li>
<li>Homology searches identified putative operator sequences for NikR in Salmonella typhimurium , Klebsiella pneumoniae , Helicobacter pylori , and Bradyrhizobium japonicum ( Fig. 7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10787413" target="_blank">10787413</a>]<br></li>
<li>These results and the fact that other ribbon-helix-helix dimers recognize DNA subsites of 4 -- 6 base pairs suggested that operator bases within or near these 59-GTATG-39 sequences were likely to comprise the principal base-specific determinants of NikR recognition .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10787413" target="_blank">10787413</a>]<br></li>
<li>Samples of 30-40 μM apo - , Ni ( II ) - , and excess Ni ( II ) - NikR ( NikR in the presence of 5 equiv of excess Ni ( II ) ) were incubated with 1 equiv of an HPLC-purified 54-bp DNA fragment from the nik promoter that contains the NikR recognition sequence ( Table S1 ) for 1 h at room temperature in 20 mM Tris-HCl buffer at pH 7.6 , 100 mM KCl , and 3 mM MgCl2 followed by modification with a 50-fold molar excess of iodoa-cetamide for 20 min at room temperature in the dark .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20583753" target="_blank">20583753</a>]<br></li>
<li>One set of sites ( four per tetramer ) has picomolar affinity for nickel , and occupancy of these sites results in NikR binding to a region spanning ~ 40 operator base pairs ( bp ) with an affinity in the mid-nanomolar range .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12970756" target="_blank">12970756</a>]<br></li>
<li>Stoichiometric nickel ions activate NikR to bind a 28-bp palindromic operator sequence ( GTATGA-N16-TCATAC ) within the promoter of nikABCDE with low-nanomolar affinity ( 9 , 11 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16945905" target="_blank">16945905</a>]<br></li>
<li>In the presence of 1:1 nickel , intact NikR binds with 35 nM affinity to an operator site that spans the -- 10 position of the nikABCDE promoter and contains dyad-symmetric half-sites , 5 ′ - GTATGA-3 ′ , separated by 16 base pairs ( bp ) 20 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12524532" target="_blank">12524532</a>]<br></li>
<li>Nickel K-edge XAS data for NikR -- DNA complexes ( Figs. 1 , 2 ; Table 1 ) indicate that large structural changes were observed when NikR was bound to a 12 bp operator DNA half site or to 50 bp operator DNA containing the full dyad symmetric binding site ( full site , Figs. 1 , 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12524532" target="_blank">12524532</a>]<br></li>
<li>A sixcoordinate nickel site composed of O - or N-donor ligands , but lacking cysteine , forms when NikR binds to operator DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12524532" target="_blank">12524532</a>]<br></li>
<li>Obvious operator site sequences were not found in several bacteria and archaea containing NikR genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10787413" target="_blank">10787413</a>]<br></li>
<li>The central high-affinity Ni2 + - binding sites , which are conserved among all NikR homologues , are sensory sites that specifically bind the scarce cellular Ni2 + ion , enabling binding to the nik promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19621966" target="_blank">19621966</a>]<br></li>
<li>The second , the promoter of the nikABCDE operon , contains a partially conserved FNR box located immediately upstream of an inverted-repeated sequence composed of a 14-base dyad that could serve as a binding site for the NikR regulatory protein ( Navarro et al. 1993 ; V. Desjardin and M.-A .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10648098" target="_blank">10648098</a>]<br></li>
<li>This hypothesis is supported by the ﬁnding , in the promoter-operator region of the nik operon ( 11 ) , of an inverted-repeated sequence composed of a 14-base dyad , AA TCAGTATGACGA-N10-TCGTCATACTTATT , which may serve as a NikR binding site and of a partially conserved FNR box located just upstream .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9882686" target="_blank">9882686</a>]<br></li>
<li>The latter nonspeciﬁc interactions could anchor NikR to the DNA , allowing the protein to initiate a one-dimensional search for its palindromic operator sequence in the nik 25 promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23737072" target="_blank">23737072</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22971172" target="_blank">22971172</a>]<br></li>
<li>To examine if the high-affinity Ni ( II ) coordination site changes upon DNA binding , XAS analyses of holo-NikR in the presence and absence of an oligonclueotide containing the nik recognition sequence were performed by our laboratory .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20583753" target="_blank">20583753</a>]<br></li>
<li>The next set of experiments was performed in the presence of an oligonucleotide containing the sequence from the nik promo-ter that is recognized by NikR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20583753" target="_blank">20583753</a>]<br></li>
				</ul></div><br>
			<label class="collapse_TU" for="_6"><font size="5px">Transcription units regulated by NikR</font></label>
				<input id="_6" type="checkbox">
				<div class="TU"><br><ul>				<label class="TU_collapse_similarity" for="tu_similarity_1">
( A ) Filled squares , Pnik-lacZ expression in glucose for the wild type ( WT ) ( RZ4500 plus pNIK103 ) ; open squares , nikR ( PC269 plus pNIK103 C95A NikR ) ; and open circles , nikABCDE ( PC379 plus pNIK103 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16159764" target="_blank">16159764</a>]<br></label>
<input id="tu_similarity_1", type="checkbox">
<div class="TU_similar">
<li>( B ) Filled squares , Pnik-lacZ expression in glucose plus 15 mM KNO3 for the wild type ( RZ4500 plus pNIK103 ) ; open squares , nikR ( PC269 plus pNIK103 C95A NikR ) ; and open circles , nikABCDE ( PC379 plus pNIK103 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16159764" target="_blank">16159764</a>]<br></li></div>
<li>They also suggest that NikR is active when nitrate is present in the growth media but that its function is somehow inhibited under conditions that favor the expression of hydrogenase isozymes and their corresponding assembly proteins ( Fig. 2B ) , such as the presence of formate in the growth medium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16159764" target="_blank">16159764</a>]<br></li>
<li>Moreover , expression was not constitutive , which would have been expected when NikR was involved in repression of rcnA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17120142" target="_blank">17120142</a>]<br></li>
<li>Relatively little is currently known about NikR repressor or how it regulates gene expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10787413" target="_blank">10787413</a>]<br></li>
<li>In the presence of nitrate ( II ) , NikABCDE synthesis is repressed by both the NarLX two-component system and NikR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16159764" target="_blank">16159764</a>]<br></li>
<li>It is activated by fumarate nitrate regulatory protein ( Fnr ) , an oxygen-sensitive transcriptional factor , and repressed by NikR when intracellular nickel concentrations are high ( 5 , 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10787413" target="_blank">10787413</a>]<br></li>
<li>Transcription of the operon encoding this nickel importer , nikABCDE , is repressed in the presence of nickel by the transcription factor NikR ( 3 , 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16945905" target="_blank">16945905</a>]<br></li>
<li>NikR , a protein of the ribbon-helix-helix family of transcription factors , represses expression of the nikABCDE operon in the presence of excessive concentrations of intracellular nickel .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10787413" target="_blank">10787413</a>]<br></li>
<li>The nikABCDE operon involved in nickel uptake is expressed under anaerobic growth conditions ( 53 ) , and its expression is speciﬁcally repressed by nickel via the metalloregulator NikR ( 12 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21665978" target="_blank">21665978</a>]<br></li>
<li>The transcription of the NikABCDE importer is repressed by the NikR metalloregulator ( 12 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21665978" target="_blank">21665978</a>]<br></li>
<li>The nickel dependence of PrcnA expression suggested that NikR does not regulate both Pnik and PrcnA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16956381" target="_blank">16956381</a>]<br></li>
<li>Therefore , expression of the nikR gene is independent of nickel , which may function as a corepressor of NikR in the regulation of expression of the nik operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9882686" target="_blank">9882686</a>]<br></li>
				</div></ul></div><br>
	</div>
</body>
</html>