<!doctype html>
<style>
@charset "utf-8";
body {
	height: 100%;
	margin: 0;
	padding: 0;
}

.banner {
	position: fixed;
	top: 0;
	width: 100%;
	height: 65px;
	background: gray;
	line-height: 65px;
}
.spacer {
	height: 10px;
}

.main-content {
	padding: 5%;
}

.collapse_ACT {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-style: none solid none none;
	border-color: #B4045F;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_ACT + input{
	display: none;
}
.collapse_ACT + input + div{
	display: none;
}
.collapse_ACT + input:checked + div{
	display: block;
}
.ACT {
	text-align: justify;
	text-justify: inter-word;
}
.ACT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B4045F;
}
.ACT_collapse_similarity:hover{
	background: #e5d3dc;
}
.ACT_collapse_similarity + input {
	display: none;
}
.ACT_collapse_similarity + input + div{
	display: none;
}

.ACT_collapse_similarity + input:checked + div{
	display: block;
}
.ACT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.ACT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B4045F;
	padding-left: 5px;
}
.ACT_collapse_similarity:hover{
	background: #e5d3dc;
}
.ACT_collapse_similarity + input {
	display: none;
}
.ACT_collapse_similarity + input + div{
	display: none;
}
.ACT_collapse_similarity + input:checked + div{
	display: block;
}
.ACT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.ACT_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #B4045F;
}
.ACT_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_DOM {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-style: none solid none none;
	border-color: #B404AE;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_DOM + input{
	display: none;
}
.collapse_DOM + input + div{
	display: none;
}
.collapse_DOM + input:checked + div{
	display: block;
}
.DOM {
	text-align: justify;
	text-justify: inter-word;
}
.DOM_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B404AE;
	padding-left: 5px;
}
.DOM_collapse_similarity:hover{
	background: #e8d7e7;
}
.DOM_collapse_similarity + input {
	display: none;
}
.DOM_collapse_similarity + input + div{
	display: none;
}
.DOM_collapse_similarity + input:checked + div{
	display: block;
}
.DOM_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}.DOM_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #B404AE;
}
.DOM_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_EVO {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #7401DF;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_EVO + input{
	display: none;
}
.collapse_EVO + input + div{
	display: none;
}
.collapse_EVO + input:checked + div{
	display: block;
}
.EVO {
	text-align: justify;
	text-justify: inter-word;
}
.EVO_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #7401DF;
	padding-left: 5px;
}
.EVO_collapse_similarity:hover{
	background: #e2d7ed;
}
.EVO_collapse_similarity + input {
	display: none;
}
.EVO_collapse_similarity + input + div{
	display: none;
}
.EVO_collapse_similarity + input:checked + div{
	display: block;
}
.EVO_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.EVO_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #7401DF;
}
.EVO_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_RP {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #0174DF;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_RP + input{
	display: none;
}
.collapse_RP + input + div{
	display: none;
}
.collapse_RP + input:checked + div{
	display: block;
}
.RP {
	text-align: justify;
	text-justify: inter-word;
}
.RP_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #0174DF;
	padding-left: 5px;
}
.RP_collapse_similarity:hover{
	background: #c9d7e2;
}
.RP_collapse_similarity + input {
	display: none;
}
.RP_collapse_similarity + input + div{
	display: none;
}
.RP_collapse_similarity + input:checked + div{
	display: block;
}
.RP_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.RP_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #0174DF;
}
.RP_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_SIT {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #0489B1;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_SIT + input{
	display: none;
}
.collapse_SIT + input + div{
	display: none;
}
.collapse_SIT + input:checked + div{
	display: block;
}
.SIT {
	text-align: justify;
	text-justify: inter-word;
}
.SIT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #0489B1;
	padding-left: 5px;
}
.SIT_collapse_similarity:hover{
	background: #d0dfe2;
}
.SIT_collapse_similarity + input {
	display: none;
}
.SIT_collapse_similarity + input + div{
	display: none;
}
.SIT_collapse_similarity + input:checked + div{
	display: block;
}
.SIT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.SIT_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #0489B1;
}
.SIT_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_TU {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #04B486;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_TU + input{
	display: none;
}
.collapse_TU + input + div{
	display: none;
}
.collapse_TU + input:checked + div{
	display: block;
}
.TU {
	text-align: justify;
	text-justify: inter-word;
}
.TU_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #04B486;
	padding-left: 5px;
}
.TU_collapse_similarity:hover{
	background: #d0e2de;
}
.TU_collapse_similarity + input {
	display: none;
}
.TU_collapse_similarity + input + div{
	display: none;
}
.TU_collapse_similarity + input:checked + div{
	display: block;
}
.TU_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.TU_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #04B486;
}
.TU_similar > a {
	text-decoration: none;
	color: gray;
}

ul { 
	list-style-type: none;
	padding: 0;
	margin: 0;
}
ul li > a {
	text-decoration: none;
	color: gray;
}
ul li:hover, ul li:focus {
	background: #F0F0F0;
}

ul li:first-child span {
  border-left:none;  
}

</style>
<html>
<head>
<meta charset="utf-8">
<title>TF properties - NorR</title>
<meta name="theme-color" content="#ffffff">
</head>
<body>
	<div class="banner">
		<center><font color="white" size="16px" face="arial">NorR</font></center>
	</div>
	<div align="center" class="main-content">
		<div class="spacer">
	</div><br>			<label class="collapse_ACT" for="_1"><font size="5px">NorR transcriptional activity</font></label>
				<input id="_1" type="checkbox">
				<div class="ACT"><br><ul><li>Indeed , the hcp gene is regulated by HcpR ( highlighted in yellow ) in many anaerobic bacteria , by NsrR in facultative anaerobic enterobacteria , and some b - and a-proteobacteria ( in magenta ) , by NorR in most Vibrionales ( in green ) , and by DNR in A. ferrooxidans and Thermochromatium tepidum ( in blue ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16261196" target="_blank">16261196</a>]<br></li>
<li>BBRC Biochemical and Biophysical Research Communications 328 ( 2005 ) 540 -- 544 www.elsevier.com/locate/ybbrc Binding of NorR to three D A sites is essential for pro activation of the ﬂavorubredoxin gene , the nitric oxide reductase f Escherichia coli N moter o Marta C. Justino , Vera M.M. Gonçalves , Lıgia M. Saraiva * Instituto de Tecnologia Quımica e Biológica , Universidade Nova de Lisboa , Apartado 127 Avenida da República ( EAN ) , 2781-901 Oeiras , Portugal Received 22 December 2004 Available online 13 January 2005 Abstract NorR is a nitric oxide sensor that in Escherichia coli regulates the gene encoding for ﬂavorubredoxin , an enzyme involved in nitrosative detoxiﬁcation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15694381" target="_blank">15694381</a>]<br></li>
<li>It is interesting to note that NorR , Fur , SoxR , and OxyR all appear to contain redox-active cysteines and , in some cases , iron .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14718666" target="_blank">14718666</a>]<br></li>
<li>Binding reactions shown in lanes 3 to 14 contained increasing concentrations of NorR identical to those shown in Fig. 1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15375149" target="_blank">15375149</a>]<br></li>
<li>To this end puriﬁed NorR was incubated with increasing concentrations of the DNA fragment P321 ( 320 bp ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15694381" target="_blank">15694381</a>]<br></li>
<li>Under these conditions , the behavior of the E. coli wild-type strain was rescued ( Fig. 3B ) , showing that when it is expressed independently of its own promoter , i.e. , not regulated by NorR , FlRd confers protection to E. coli against macrophages .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22563051" target="_blank">22563051</a>]<br></li>
<li>This mechanism of norR repression would function to maintain the very low levels of NorR required to activate norV expression , whilst allowing the norR promoter to be de-repressed should NorR levels fall below a critical threshold level .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16417519" target="_blank">16417519</a>]<br></li>
<li>NorR D99A is unable to activate expression of norV in vivo , whilst NorR C113G activates norV expression constitutively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16417519" target="_blank">16417519</a>]<br></li>
<li>Mechanism of transcriptional regulation by the Escherichia coli nitric oxide sensor NorR N.P. Tucker * 1 , B. D'Autre ́aux * , S. Spiro † and R. Dixon * * Department of Molecular Microbiology , John Innes Centre , Norwich NR4 7UH , U.K. , and † School of Biology , Georgia Institute of Technology , 310 Ferst Drive , Atlanta , GA 30332-0230 , U.S.A. Abstract Nitric oxide ( NO ) is a highly reactive water-soluble gas encountered by bacteria endogenously as an intermediate of denitriﬁcation and exogenously as one of the radical species deployed by macrophages against invading pathogens .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16417519" target="_blank">16417519</a>]<br></li>
<li>NorR Protein of Escherichia coli Activates Expression o Flavorubredoxin Gene norV in Response to Reactive Nitrogen Species The f the Matthew I. Hutchings , † Neeraj Mandhana , and Stephen Spiro * School of Biological Sciences , University of East Anglia , Norwich NR4 7TJ , United Kingdom Received 28 March 2002/Accepted 9 May 2002 The Escherichia coli norVW genes encode a ﬂavorubredoxin and NADH :( ﬂavo ) rubredoxin reductase , respectively , which are involved in nitric oxide detoxiﬁcation under anaerobic-growth conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12142437" target="_blank">12142437</a>]<br></li>
<li>We suggest that NorR , and perhaps the unidentified regulators , were evolved to sense reactive nitrogen species , whereas Fur , SoxR , and OxyR were evolved to sense iron and reactive oxygen species and are collaterally activated by reactive nitrogen species .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14718666" target="_blank">14718666</a>]<br></li>
<li>Keywords : NorR ; Nitric oxide ; Escherichia coli ; Flavorubredoxin gene regulation NorR is a r54 ( RpoN ) - dependent regulator , present in a wide range of organisms , proposed to be involved in the transcriptional response to nitrosative stress [ 1 -- 7 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15694381" target="_blank">15694381</a>]<br></li>
<li>In Escherichia coli , the transcriptional-regulators NorR and NsrR mediate adaptive responses to NO by controlling the expression of genes encoding enzymes that reduce or oxidize NO to less toxic species ( Mukhopadhyay et al. , 2004 ; Bodenmiller and Spiro , 2006 ; Spiro , 2007 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19656291" target="_blank">19656291</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_14">
Mukhopadhyay , P. , Zheng , M. , Bedzyk , L.A. , LaRossa , R.A. , and Storz , G. ( 2004 ) Prominent roles of the NorR and Fur regulators in the Escherichia coli transcriptional response to reactive nitrogen species .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15101990" target="_blank">15101990</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19656291" target="_blank">19656291</a>]<br></label>
<input id="act_similarity_14", type="checkbox">
<div class="ACT_similar">
<li>Mukhopadhyay , P. , Zheng , M. , Bedzyk , L. A. , LaRossa , R. A. & Storz , G. Prominent roles of the NorR and Fur regulators in the Escherichia coli transcriptional response to reactive nitrogen species .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16193057" target="_blank">16193057</a>]<br></li>
<li>[ 4 ] P. Mukhopadhyay , M. Zheng , L.A. Bedzyk , R.A. LaRossa , G. Storz , Prominent roles of the NorR and Fur regulators in the Escherichia coli transcriptional response to reactive nitrogen species , Proc .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15694381" target="_blank">15694381</a>]<br></li></div>
<li>Thus , we propose that the E. coli response to reactive nitrogen species is brought about by the modification of some transcription factors , such as NorR , that control the expression of NO detoxifying activities and are dedicated to sensing reactive nitrogen species , and other redoxactive transcription factors such as Fur , SoxR , and OxyR , where the reaction with reactive nitrogen species is incidental .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14718666" target="_blank">14718666</a>]<br></li>
<li>Proc Natl Acad Sci USA 98:10108 -- 10112 Hutchings MI , Mandhana N , Spiro S ( 2002 ) The NorR Protein of Escherichia coli activates expression of the Xavorubredoxin gene norV in response to reactive nitrogen species .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17024490" target="_blank">17024490</a>]<br></li>
<li>The NorR protein of Escherichia coli activates expression of the ﬂavorubredoxin gene norV in response to reactive nitrogen species .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17449618" target="_blank">17449618</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16428390" target="_blank">16428390</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16193057" target="_blank">16193057</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15375149" target="_blank">15375149</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_20">
[ 10 ] Hutchings , M.I. , Mandhana , N. and Spiro , S. ( 2002 ) The NorR protein of Escherichia coli activates expression of the # avorubredoxin gene norV in response to reactive nitrogen species .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12586421" target="_blank">12586421</a>]<br></label>
<input id="act_similarity_20", type="checkbox">
<div class="ACT_similar">
<li>[ 3 ] M.I. Hutchings , N. Mandhana , S. Spiro , The NorR protein of Escherichia coli activates expression of the ﬂavorubredoxin gene norV in response to reactive nitrogen species , J. Bacteriol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15694381" target="_blank">15694381</a>]<br></li>
<li>Hutchings MI , Mandhana N & Spiro S ( 2002 ) The NorR protein of Escherichia coli activates expression of the ﬂavorubredoxin gene norV in response to reactive nitrogen species .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22029434" target="_blank">22029434</a>]<br></li>
<li>Hutchings , M.I. , Mandhana , N. , and Spiro , S. ( 2002 ) The NorR protein of Escherichia coli activates expression of the ﬂavorubredoxin gene norV in response to reactive nitrogen species .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19656291" target="_blank">19656291</a>]<br></li></div>
<li>Transcriptional activation by sigma 54 ( j ) - dependent activators 54 is usually controlled by sensory modules11 , and in the case of NorR the amino-terminal GAF domain ( named for cyclic GMP-speciﬁc and stimulated phosphodiesterases , Anabaena adenylate cyclases and E. coli FhlA ) is the probable site of signal sensing ( Fig. 1a ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16193057" target="_blank">16193057</a>]<br></li>
<li>Deletion of the GAF domain ( NorRDGAF ) results in constitutive signal-independent transcriptional activation by NorR in vivo ( Fig. 1e ) , implying that the function of the GAF domain is to sense the signal and inhibit the ATPase activity of the central AAA domain ( Fig. 1a ) þ 3,10 when NorR is in its inactive state .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16193057" target="_blank">16193057</a>]<br></li>
<li>Although the high affinity NorR 1 site is suitably positioned to act as an upstream activ-ator sequence for norVW , it is possible that co-operative binding of NorR to all three sites is necessary for maximal promoter activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15667300" target="_blank">15667300</a>]<br></li>
<li>Taking advantage of the fact that the 54-dependent transcriptional-activators bind to DNA even when they are in their inactive conﬁguration , we have deﬁned three NorR binding sites by using puriﬁed protein in DNase I footprinting and methylation protection assays .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15375149" target="_blank">15375149</a>]<br></li>
<li>The present work shows that although puriﬁed NorR can bind independently to each of three binding sites in the ﬂavorubredoxin gene promoter , the presence of all sites is required for in vivo nitric oxide-dependent induction of the ﬂavorubredoxin gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15694381" target="_blank">15694381</a>]<br></li>
<li>Nitroprusside was used as the source of RNIs in these experiments , since the homologous NorR protein of R. eutropha can be activated by nitroprusside in vivo ( 11 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12142437" target="_blank">12142437</a>]<br></li>
<li>The NorR protein of E. coli appears to be a true orthologue of NorR of R. eutropha ( 11 ) in that it is activated in vivo by sources of RNIs , speciﬁcally nitroprusside .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12142437" target="_blank">12142437</a>]<br></li>
<li>Transcriptional regulation of the dissimilatory nitric oxides metabolism in bacteria is diverse and involves FNR-like transcription factors HcpR , DNR , and NnrR ; two-component systems NarXL and NarQP ; NO-responsive activator NorR ; and nitrite-sensitive repressor NsrR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16261196" target="_blank">16261196</a>]<br></li>
<li>To gain insight into the nature of the active form of NorR , we sought evidence for the presence of an NO adduct in E. coli whole cells using electron paramagnetic resonance ( EPR ) spectroscopy .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16193057" target="_blank">16193057</a>]<br></li>
<li>NorR as isolated was essentially EPR silent , suggesting that it was puriﬁed in the ferrous form .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22563051" target="_blank">22563051</a>]<br></li>
<li>Thus , NorR is a transcriptional-activator protein that responds to NO and it is tempting to suggest that the NorR-orthologues located upstream and divergently transcribed from their corresponding hmp genes in Vibrio cholerae , V. parahemolyticus , V. vulniWcus , Pseudomonas putida , P. aeruginosa , Azotobacter vinelandii , Burkholderia fungorum and Burkholderia , sp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17024490" target="_blank">17024490</a>]<br></li>
<li>Puriﬁed NorR binds to the norR-norVW intergenic region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15375149" target="_blank">15375149</a>]<br></li>
<li>NorR activates transcription under both aerobic and anaerobic conditions , suggesting there may be additional NorR targets , since the flavorubredoxin is only active in the absence of oxygen [ 6,9 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15667300" target="_blank">15667300</a>]<br></li>
<li>Under anaerobic conditions NorR up-regulates the E. coli flavorubredoxin , NorV , and its associated oxidoreductase , NorW , which function to reduce NO to N O [ 10 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21265775" target="_blank">21265775</a>]<br></li>
<li>Under anaerobic conditions , the transcription of nrfA is induced , and so both NrfA and NorV , which is activated by NorR , are important for survival .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21265775" target="_blank">21265775</a>]<br></li>
<li>The key NO detoxifying enzymes are the ﬂavohaemoglobin ( encoded by the hmp gene ) and the ﬂavorubredoxin ( encoded by norVW ) , which are regulated by NsrR and NorR respectively ( Hutchings et al. , 2002 ; Gardner et al. , 2003 ; Bodenmiller and Spiro , 2006 ) , and the respiratory nitrite reductase , Nrf , which reduces both nitrite and NO to ammonia ( Poock et al. , 2002 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19656291" target="_blank">19656291</a>]<br></li>
<li>The core regulon members , that is , genes regulated by the nitrogen-oxide responsive factors NsrR , HcpR , DNR , NnrR , and NorR in many genomes , are the hybrid cluster protein gene hcp , the NO-detoxifying ﬂavohemoglobin hmp , two hypothetical genes dnrN and nnrS , NO reductase operon norVW , and multiple denitriﬁcation genes , nir , nor , and nos , encoding nitrite , NO , and nitrous oxide reductases , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16261196" target="_blank">16261196</a>]<br></li>
<li>All three regulons are present only in three Ralstonia species , where NsrR controls the NO-detoxifying ﬂavohemoglobin , whereas NorR and DNR regulate the denitriﬁcation genes ( nir , nor , and nos ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16261196" target="_blank">16261196</a>]<br></li>
<li>The lack of differences in the survival of the wild type and the norV mutant upon incubation in macrophages observed within the ﬁrst 6 h of macrophage infection may be due , in light of our earlier ﬁndings that norV induction in response to NO is compromised when H2O2 is present , to the NorR inhibition caused by the macrophage oxidative burst .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22563051" target="_blank">22563051</a>]<br></li>
<li>Over time , macrophages decrease the oxidative burst while NO production increases , which will allow binding of NO to NorR and subsequent triggering of norV expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22563051" target="_blank">22563051</a>]<br></li>
<li>Disruption of each NorR binding site prevented activation of norV-lacZ expression in cultures grown anaerobically with nitrite , a known inducer of norVW expression [ 10 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16417519" target="_blank">16417519</a>]<br></li>
<li>Given the relatively low concentration of NO required for anaerobic norV induction ( Fig. 1B ) , NO is the most probable physiological signal modulating NorR and norVW transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12529359" target="_blank">12529359</a>]<br></li>
<li>Data presented in this paper dem-onstrate that the NorR protein is indeed required for transcription of norV and that expression of the structural genes is activated in both aerobic and anaerobic cultures by RNIs .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12142437" target="_blank">12142437</a>]<br></li>
<li>In the denitrifier Ralstonia eutropha , the tripartite transcription factor NorR regulates denitrification , norA1B1 transcription , and NOR activity expression in a 54-dependent mechanism in response to exposures to sodium nitroprusside , the NO donor compound NOC18 , or during growth with nitrite or nitrate ( 31 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12529359" target="_blank">12529359</a>]<br></li>
<li>In E. coli , NorR activates norVW transcription in anaerobic cultures in response to NO , nitrate , and nitrite ( which may act as sources of endogenous NO ) and nitroprusside and in aerobic cultures in response to S-nitrosoglutathione , acidiﬁed nitrite , and nitroprusside ( 4 , 10 , 15 , 19 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15375149" target="_blank">15375149</a>]<br></li>
<li>NorR is a σ 54-dependent transcriptional-activator required for the expression of norVW in response to NO , nitrate , nitrite and nitroprusside under both aerobic and anaerobic conditions [ 4,6 -- 9 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15667300" target="_blank">15667300</a>]<br></li>
<li>NorR is a DNA-binding enhancer protein that in response to low concentrations of cytoplasmic NO activates norVW transcription by the r54 version of RNA polymerase ( Hutchings et al. , 2002 ; Gardner et al. , 2003 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22029434" target="_blank">22029434</a>]<br></li>
<li>The only regulator known to serve exclusively as an NO sensor in E. coli is NorR , which activates transcription of the norVW genes encoding the ﬂavorubredoxin and associated ﬂavoprotein that together reduce NO to nitrous oxide ( 7 , 13 , 14 , 16 , 18 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16428390" target="_blank">16428390</a>]<br></li>
<li>NO activates NorR , the transcription factor that regulates the NO reductase genes , norVW [ 12 -- 15 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16417494" target="_blank">16417494</a>]<br></li>
<li>The norVW genes are activated in response to NO by NorR ( 7 , 14 , 42 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16428390" target="_blank">16428390</a>]<br></li>
<li>Transcription of the norVW genes is activated in response to NO by the σ 54-dependent regulator NorR , a member of the prokaryotic enhancer binding protein family .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16417519" target="_blank">16417519</a>]<br></li>
<li>The fact that NorR can regulate gene expression under oxic conditions ( which inactivate the NO reductase activity of the ﬂavorubredoxin ) raises the possibility that there are additional genes regulated by NorR , the products of which have roles in aerobically growing cultures .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15375149" target="_blank">15375149</a>]<br></li>
<li>As has been previously noted ( 11 ) , the product of ygaA is 42 % identical in sequence to the NorR protein of Ralstonia eutropha , which activates expression of a nitric oxide reductase in response to NO and to reactive nitrogen intermediates ( RNIs ) generated from sodium nitroprusside ( 11 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12142437" target="_blank">12142437</a>]<br></li>
<li>YgaA strongly resembles the Ralstonia eutropha NorR protein that regulates expression of heme b3-iron nitric oxide reductase in response to NO [ 9 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12586421" target="_blank">12586421</a>]<br></li>
<li>The homologous NorR protein of Ralstonia eutropha regulates expression of an NO reductase and can also be activated by nitroprusside and , presumably , NO ( 20 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15375149" target="_blank">15375149</a>]<br></li>
<li>For example , NorR regulates expression of the norAB genes encoding the respiratory NO reductase in Ralstonia eutropha [ 6 ] , and the flavohaemoglobin gene fhp in Pseudomonas aeru-54 ginosa [ 7 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16417519" target="_blank">16417519</a>]<br></li>
<li>NorR is a r54-dependent transcriptional-activator that regulates expression of the NO reductase operons , norVW in E. coli and norAB in R. eutropha [ 21,22 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16261196" target="_blank">16261196</a>]<br></li>
<li>Conclusions Our current model for the mechanism of transcriptional activation by NorR in response to NO is shown in Figure 1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16417519" target="_blank">16417519</a>]<br></li>
<li>A major protection mechanism against NO in the cytoplasm is provided by ﬂavorubredoxin and its reductase , NorVW , which are synthesized in response to NO activation of the transcription activator NorR ( 15 , 20 , 22 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17449618" target="_blank">17449618</a>]<br></li>
<li>This mechanism enables direct transduction of 54 the NO signal by NorR to the activation of NO detoxiﬁcation processes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16193057" target="_blank">16193057</a>]<br></li>
<li>This is in agreement with earlier proposals stating that activation of norV by NO occurs via NorR [ 2 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15694381" target="_blank">15694381</a>]<br></li>
<li>Interestingly , nitroprusside-elicited norV transcription was increased 5-fold by normoxic O suggesting 2 mechanisms for NorR activation involving O2-derived reactive nitrogen intermediates rather than NO per se .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12529359" target="_blank">12529359</a>]<br></li>
<li>The transcription of the norV gene is strongly upregulated in cells cultured anaerobically and exposed to NO , through the activation of the nitric oxide sensor NorR ( 12 , 21 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22563051" target="_blank">22563051</a>]<br></li>
<li>In the absence of NO , there was no measurable NOR activity expressed ( Fig. 3A , open bars ) indicating that overexpression of wild-type NorR does not by itself increase NorVW expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12529359" target="_blank">12529359</a>]<br></li>
<li>Interestingly , the NO-mediated induction of NOD activity was significantly ( p 0.05 ) reduced in strains expressing NorR and elevated NOR activity ( Fig. 3B ) thus suggesting an indirect role for NorR and NOR in regulating NOD expression by reducing NO levels .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12529359" target="_blank">12529359</a>]<br></li>
<li>The iron centre in NorR seems to be in the reduced state because , in the absence of NO , EPR signals indicative of a ferric species were not detected either in whole cells ( spectra 3 in Fig. 1b , c ) or with puriﬁed reconstituted proteins ( Supplementary Fig. 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16193057" target="_blank">16193057</a>]<br></li>
<li>NorR apoprotein prepared under aerobic conditions ( and thus devoid of iron ) was unable to drive the formation of open promoter complexes either in the presence or absence of NO ( Fig. 3a , lanes 13 and 14 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16193057" target="_blank">16193057</a>]<br></li>
<li>Given that the compromised expression of FlRd in cells treated with NO plus H2O2 could not be attributed to lowered NorR levels , we hypothesized that the ligation of NO to the mononuclear iron site located in the GAF domain of NorR , which triggers NorR activation of the norV promoter ( 6 ) , was impaired in the presence of hydrogen peroxide , an issue that was studied by EPR spectro-scopy .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22563051" target="_blank">22563051</a>]<br></li>
<li>Moreover , the presence of a low-afinity binding site is enough to ensure speciﬁc binding of NorR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15694381" target="_blank">15694381</a>]<br></li>
<li>The active site of NorR is a di-iron center that can be directly nitrosylated in the presence of very low concentrations of NO ( D'Autreaux et al. , 2005 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22029434" target="_blank">22029434</a>]<br></li>
<li>Altogether , it was concluded that the presence of H2O2 inhibits the binding of NO to the iron center of NorR due to oxidation of the mononuclear iron center impairing the formation of the nitrosylated ferrous site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22563051" target="_blank">22563051</a>]<br></li>
<li>Therefore , NO-activated NorR-Fe ( II ) is apparently not sensitive to oxygen , which is consistent with the presence of NorR activity in aerobic cultures9 ,14 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16193057" target="_blank">16193057</a>]<br></li>
<li>However , the ATPase activities of NorRDGAF and NO-treated NorR-Fe ( II ) markedly increased in the presence of DNA containing NorR binding sites ( Fig. 3c ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16193057" target="_blank">16193057</a>]<br></li>
<li>A expected , upon incubation with NO transcription activity was readily detected for promoter PL321 and no increase in the activity was observed for the promoter PL0 that lacks any NorR binding site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15694381" target="_blank">15694381</a>]<br></li>
<li>The enhancer dependency of the ATPase activity potentially increases the speciﬁcity of NO signalling by NorR towards the regulation of speciﬁc target promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16193057" target="_blank">16193057</a>]<br></li>
<li>The basal level of the ATPase activity of NorR remained essentially unchanged when the protein was exposed to H2O2 ; in contrast , when it was treated with NO , the activity increased approximately 7-fold .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22563051" target="_blank">22563051</a>]<br></li>
<li>Since our results suggest that hydrogen peroxide prevents binding of NO , the ATPase activity of NorR under these stress conditions was examined ( Table 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22563051" target="_blank">22563051</a>]<br></li>
<li>However , in the presence of both NO and H2O2 , no enhancement of the activity was seen , thus conﬁrming that hydrogen peroxide hinders the NO-dependent ATPase activity of NorR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22563051" target="_blank">22563051</a>]<br></li>
<li>It was previously shown that the Fe ( II ) - GAF domain of NorR inhibits its ATPase activity and that binding of NO to the iron center is required to stimulate the ATPase activity ( 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22563051" target="_blank">22563051</a>]<br></li>
<li>The binding of NO to the ferrous iron center stimulates the ATPase activity of NorR and enables NorR to activate the transcription of norV ( 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22563051" target="_blank">22563051</a>]<br></li>
<li>Binding of NO stimulates the ATPase activity of NorR , enabling the activation of transcription by RNA polymerase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16193057" target="_blank">16193057</a>]<br></li>
<li>Binding of NO to the ferrous iron centre stimulates the ATPase activity of NorR , enabling NorR to activate transcription by σ 54-RNA polymerase [ 3 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16417519" target="_blank">16417519</a>]<br></li>
<li>EPR spectra were recorded at a 9.39-MHz microwave frequency , 2.4-mW microwave power , and 6 to 7 K. TABLE 2 ATPase activity of NorR upon treatment with NO and H O a 2 2 ATPase activity ( mol ATP / Presence of NO min-mg protein ) 0.4 0.1 0.4 0.1 2.9 0.4 Presence of H2O2 1.1 0.1 a NorR was treated for 5 min with 100 M H2O2 , 20 M NO , or a mixture of the two .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22563051" target="_blank">22563051</a>]<br></li>
<li>We therefore assayed the ATPase activity of NorR to obtain further evidence that the protein is activated by the formation of a nitrosyl -- iron complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16193057" target="_blank">16193057</a>]<br></li>
<li>Accordingly , triggering of the NorR ATPase activity , a requisite for ﬂavorubredoxin activation , was impaired by treatment of the protein with the double stress .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22563051" target="_blank">22563051</a>]<br></li>
<li>The formation of the mono-nitrosyl iron complex upon binding of NO to the GAF domain is presumably responsible for a conformational change in NorR that releases repression of the AAA + domain ( Figure 1C ) , allowing ATP hydrolysis and the isomerization of σ 54-RNA polymerase , resulting in open complex formation ( Figure 1D ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16417519" target="_blank">16417519</a>]<br></li>
<li>Alternatively , the NorR N-termi-nal domain could activate transcription by interacting with a signal transducing protein as described for NifL/NifA ( 45 ) or by binding NO directly as the formate-sensing transcription regulator FhlA binds formate ( 46 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12529359" target="_blank">12529359</a>]<br></li>
<li>Because the transcriptional activation mediated by this domain of NorR is apparently inhibited by the GAF domain in the absence of the signal , we anticipated that the coordination of NO to the Fe ( II ) centre might activate NorR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16193057" target="_blank">16193057</a>]<br></li>
<li>When purified under anaerobic conditions , NorR contains a mononuclear iron centre in the GAF domain that is required for sensing the NO signal [ 3 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16417519" target="_blank">16417519</a>]<br></li>
				</div></ul><br>
			<label class="collapse_DOM" for="_2"><font size="5px">NorR protein domains</font></label>
				<input id="_2" type="checkbox">
				<div class="DOM"><br><ul><li>A second inframe deletion of amino acids 30 -- 214 was created by digesting pUC19NorR with Bcl1 and religating .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12529359" target="_blank">12529359</a>]<br></li>
<li>This procedure generated an in-frame fusion of NorR that deleted amino acids 30 -- 164 and added a glutamate residue at the junction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12529359" target="_blank">12529359</a>]<br></li>
<li>A search of GenBankTM ( NCBI ) with the N-terminal 182 amino acids of NorR identifies several NorR orthologues ( Fig. 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12529359" target="_blank">12529359</a>]<br></li>
<li>Comparison of the norVW promoters of enteric bacteria along with known and putative NorR-regulated promoters from Vibrio , Ralstonia and Pseudomonas species suggests that NorR binding sites contain an invariant GT ( N7 ) AC motif ﬂanking an AT-rich central region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15667300" target="_blank">15667300</a>]<br></li>
<li>Binding of NorR to fragments P32 and P21 , which contain two putative NorR binding sites , gave two discrete products on the retardation gel ( Fig. 1B-P32 and P21 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15694381" target="_blank">15694381</a>]<br></li>
<li>To localize NorR binding sites more precisely and to establish which guanosine residues make close contacts with bound protein , methylation protection patterns were determined on both strands of the DNA fragment , using dimethyl sulfate as a footprinting reagent ( Fig. 3A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15375149" target="_blank">15375149</a>]<br></li>
<li>Clear protection of guanosine residues by NorR was observed at all three of the sites identiﬁed by DNase I footprinting ( compare Fig. 2 and 3A ) , thus conﬁrming the locations of the NorR binding sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15375149" target="_blank">15375149</a>]<br></li>
<li>except V. cholerae , P. profundus d-proteobacteria NorR HcpR Ferredoxin-like protein , component of hydroxylamine reductase ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16261196" target="_blank">16261196</a>]<br></li>
<li>Positions G5 and T6 and complementary positions A14 and C15 are the most conserved ones throughout the NorR family , being only partially replaced in Polaromonas sp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16261196" target="_blank">16261196</a>]<br></li>
<li>The mechanism of NorR activation remains to be elucidated , but the E. coli NorR protein has seven cysteines , three of which are conserved in R. eutropha NorR1 and NorR2 and two of which are not conserved but are found in a CXXC motif characteristic of redox-active cysteines .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14718666" target="_blank">14718666</a>]<br></li>
<li>The characterization of NorR binding sites will facilitate the identiﬁcation of any novel members of the NorR regulon and leads to the conclusion that NorR DNA-recognition sites are partially conserved in several proteobacteria .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15375149" target="_blank">15375149</a>]<br></li>
<li>The consensus sequences of NorR and NorR2 recognition motifs identiﬁed in various taxonomic groups have only a limited number of universally conserved positions ( Figure 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16261196" target="_blank">16261196</a>]<br></li>
<li>In contrast , cells expressing the GAF-truncated and constitutively active form of NorR , NorRDGAF , NorR contained 0.7 and 0.9 Fe atoms per monomer , respectively , suggesting that NorR coordinates a single mononuclear iron atom per monomer .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16193057" target="_blank">16193057</a>]<br></li>
<li>Lane 14 contained no NorR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15375149" target="_blank">15375149</a>]<br></li>
<li>The puriﬁed NorR protein contained 0.9 Fe atom per monomer .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22563051" target="_blank">22563051</a>]<br></li>
<li>In contrast to wild-type NorR , the anaerobically puriﬁed NorR D99A mutant protein did not contain iron .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16193057" target="_blank">16193057</a>]<br></li>
<li>It is interesting to note that NorR , Fur , SoxR , and OxyR all appear to contain redox-active cysteines and , in some cases , iron .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14718666" target="_blank">14718666</a>]<br></li>
<li>Detailed procedures used for site-directed mutagenesis and assays of NorR activity in vivo , mass spectrometry , determination of the NO dissociation constant and kinetic parameters , open complex assays , protein puriﬁcations and details of chemical and biochemicals used are provided in the Supplementary Methods .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16193057" target="_blank">16193057</a>]<br></li>
<li>Moreover , the presence of a low-afinity binding site is enough to ensure speciﬁc binding of NorR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15694381" target="_blank">15694381</a>]<br></li>
<li>The long noncoding 54 region upstream of ybiJ contains neither a predicted promoter nor a strongly predicted site for NorR binding , suggesting that ybiJ is not directly activated by NorR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15375149" target="_blank">15375149</a>]<br></li>
<li>Induction of norVW occurs upon ligation of NO to NorR and binding of the regulator to three motifs present in the promoter region of norVW ( 20 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22563051" target="_blank">22563051</a>]<br></li>
<li>( B ) EMSA analysis of the interaction of NorR with DNA fragments containing segments of the norVW promoter region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15694381" target="_blank">15694381</a>]<br></li>
<li>These results demonstrate the signaling function of the Nterminal domain of the E. coli norVW transcription regulator NorR similar to that described for other tripartite regulators ( 31 , 42 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12529359" target="_blank">12529359</a>]<br></li>
<li>Hence , it might be concluded that NorR needs to be in the reduced state in order to bind NO and to promote the transcription of norV , i.e. , the iron oxidation state inﬂuences the activation of NorR ( Fig. 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22563051" target="_blank">22563051</a>]<br></li>
<li>This is in agreement with earlier proposals stating that activation of norV by NO occurs via NorR [ 2 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15694381" target="_blank">15694381</a>]<br></li>
<li>We therefore assayed the ATPase activity of NorR to obtain further evidence that the protein is activated by the formation of a nitrosyl -- iron complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16193057" target="_blank">16193057</a>]<br></li>
<li>The s54-dependent transcriptional activator NorR is stimulated by the formation of a nitrosyl species at a mono-nuclear iron site in its signalling domain ( D ′ Autréaux et al. , 2005 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19656291" target="_blank">19656291</a>]<br></li>
<li>The NorR signalling mechanism reveals an unprecedented biological role for a mono-nitrosyl non-haem iron complex in NO sensing , which may serve as a paradigm for NO co-ordination in NO sensors that contain non-haem iron .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16417519" target="_blank">16417519</a>]<br></li>
<li>Furthermore , conserved aspartate residues in the NorR N-terminal signaling domain suggest the potential for phosphorylation by a sensor histidine-kinase similar to that described for the NtrB/NtrC pair ( 43 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12529359" target="_blank">12529359</a>]<br></li>
<li>We also show that a deletion within the conserved NorR N-terminal 81 domain activates NorVW expression independent of NO exposure , thus demonstrating the role of the N terminus in NO sensing and signaling .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12529359" target="_blank">12529359</a>]<br></li>
<li>Furthermore , an internal deletion of the N-terminal domain of NorR activated NOR expression independent of NO exposure .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12529359" target="_blank">12529359</a>]<br></li>
<li>Similar to other tripartite regulators in this family , deletion of the N-terminal signaling or inhibitory domain of NorR activated NorVW expression independent of NO ( Fig. 3A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12529359" target="_blank">12529359</a>]<br></li>
<li>A role for the NorR N-termi-nal domain as a transducer or sensor for NO is suggested .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12529359" target="_blank">12529359</a>]<br></li>
<li>The NorR N-terminal domain contains potential metal-liganding histidine and cysteine resi-dues that could form the NO sensor module .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12529359" target="_blank">12529359</a>]<br></li>
<li>Deletion of the GAF-domain ( NorRDGAF ) results in constitutive signal-independent transcriptional activation by NorR in vivo ( Fig. 1e ) , implying that the function of the GAF-domain is to sense the signal and inhibit the ATPase activity of the central AAA domain ( Fig. 1a ) þ 3,10 when NorR is in its inactive state .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16193057" target="_blank">16193057</a>]<br></li>
<li>As a positive control for activity assays , we used the truncated form of NorR ( NorRDGAF ) , which lacks the GAF-domain and gives rise to signal-independent transcriptional activation in vivo ( Fig. 1a , e ) 3,10 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16193057" target="_blank">16193057</a>]<br></li>
<li>The second expresses the NorR GAF-domain only ( designated GAFNorR ) , from residues 1 to 177 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16193057" target="_blank">16193057</a>]<br></li>
<li>To analyse the influence of the mutations on NorR binding , we performed in vitro methylation protection experiments with a constitutively active form of NorR that lacks the N-terminal GAF-domain ( NorR GAF ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16417519" target="_blank">16417519</a>]<br></li>
<li>NorR was believed to function in a similar fashion since deletion of the N-terminal GAF-domain results in a constitutively active protein in vivo [ 10 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16417519" target="_blank">16417519</a>]<br></li>
<li>Given that the compromised expression of FlRd in cells treated with NO plus H2O2 could not be attributed to lowered NorR levels , we hypothesized that the ligation of NO to the mononuclear iron site located in the GAF-domain of NorR , which triggers NorR activation of the norV promoter ( 6 ) , was impaired in the presence of hydrogen peroxide , an issue that was studied by EPR spectro-scopy .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22563051" target="_blank">22563051</a>]<br></li>
<li>Because the transcriptional activation mediated by this domain of NorR is apparently inhibited by the GAF-domain in the absence of the signal , we anticipated that the coordination of NO to the Fe ( II ) centre might activate NorR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16193057" target="_blank">16193057</a>]<br></li>
<li>It was previously shown that the Fe ( II ) - GAF-domain of NorR inhibits its ATPase activity and that binding of NO to the iron center is required to stimulate the ATPase activity ( 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22563051" target="_blank">22563051</a>]<br></li>
<li>The NorR GAF-domain senses NO by a novel mechanism As noted above , the N-terminal signalling domain of NorR is a GAF-domain that apparently represses the ATPase activity of the AAA + domain in the absence of signal , a mode of action shared by many EBPs [ 14 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16417519" target="_blank">16417519</a>]<br></li>
<li>We have recently demonstrated that the N-terminal GAF ( cGMP-speciﬁc and - stimulated phosphodiesterases , Anabaena adenylate cyclases and Escherichia coli FhlA ) domain of NorR contains a non-haem mononuclear iron centre and senses NO by formation of a mono-nitrosyl iron complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16417519" target="_blank">16417519</a>]<br></li>
<li>Here we show that the regulatory domain of NorR contains a mononuclear non-haem iron centre , which reversibly binds NO .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16193057" target="_blank">16193057</a>]<br></li>
<li>When purified under anaerobic conditions , NorR contains a mononuclear iron centre in the GAF-domain that is required for sensing the NO signal [ 3 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16417519" target="_blank">16417519</a>]<br></li>
<li>To determine whether the iron centre in NorR is required for its activity , we substituted a candidate ligand to the iron , the strictly conserved aspartate residue at position 99 in the GAF-domain of NorR , for alanine ( NorR D99A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16193057" target="_blank">16193057</a>]<br></li>
<li>NorR D99A and NorR C113G both appear to lack iron in the GAF-domain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16417519" target="_blank">16417519</a>]<br></li>
<li>Furthermore , the identity of the EPR spectra obtained with full-length NorR and GAFNorR ( Fig. 2 ) indicates that all of the protein ligands to the nitrosyl -- iron species are located in the GAF-domain , and that the other domains of NorR do not inﬂuence the electronic properties of the mononitrosyl -- iron centre .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16193057" target="_blank">16193057</a>]<br></li>
<li>Alignment of the GAF-domains of NorR orthologues reveals a number of conserved residues that may either serve as ligands to the ferrous iron centre or have a role in signal communication .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16417519" target="_blank">16417519</a>]<br></li>
<li>Sigma-54-dependent NorR regulon found in some gamma - and beta-proteobacteria contains various enzymes involved in the NO detoxification .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16261196" target="_blank">16261196</a>]<br></li>
<li>We observe that the catalytic-activity of NorR is reduced to background levels when any one of the three NorR binding sites is disrupted , suggesting that the organization of the enhancer binding sites is optimized to assist in the formation of a higher oligomeric state required for ATPase activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16417519" target="_blank">16417519</a>]<br></li>
<li>However , in the presence of both NO and H2O2 , no enhancement of the activity was seen , thus conﬁrming that hydrogen peroxide hinders the NO-dependent ATPase activity of NorR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22563051" target="_blank">22563051</a>]<br></li>
<li>Using the constitutively active form of NorR ( NorR GAF ) , we have demonstrated that the ATPase activity of NorR is dependent on enhancer binding [ 3 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16417519" target="_blank">16417519</a>]<br></li>
<li>The ATPase activity is DNA concentration-dependent and requires a DNA template containing the NorR binding sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16417519" target="_blank">16417519</a>]<br></li>
<li>The primary sequence of E. coli NorR predicts a tripartite structure [ 5 ] , that includes an N-terminal GAF-domain required to sense NO [ 2 ] , a central domain , and a C-terminal region containing a helix -- turn -- helix motif typical of DNA-binding proteins [ 11 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15694381" target="_blank">15694381</a>]<br></li>
<li>Figure 1 Model of NorR-dependent norVW activation ( A ) NorR ( the AAA + and C-terminal domains are depicted as a green oval and the GAF-domain by a blue box ) binds to three sites upstream of the norVW promoter and core RNA polymerase ( RNAP , purple ) associated with σ 54 ( yellow ) binds to the norVW promoter at sequences centred at − 12 and − 24 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16417519" target="_blank">16417519</a>]<br></li>
<li>The NorR protein is organized into three domains : an N-terminal GAF-domain that is potentially a site for an interaction with a small molecule ligand ( 8 ) , a central domain that is predicted to interact with 54-containing RNA polymerase and to hydrolyze ATP , and a C-terminal DNA-binding domain ( 11 , 14 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12142437" target="_blank">12142437</a>]<br></li>
<li>NorR is a 54-dependent transcription factor formed by three domains : an N-terminal regulatory GAF-domain harboring a mononuclear iron site that binds NO , a central AAA domain responsible for ATPase activity and interaction with the 54 subunit of RNA polymerase , and a C-terminal DNA-binding domain that interacts with enhancer sequences ( 6 , 12 , 20 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22563051" target="_blank">22563051</a>]<br></li>
<li>NorR contains three domains : ( i ) a C-terminal DNA-binding domain predicted to interact with enhancer sequences , ( ii ) a central domain belonging to the AAA + ( ATPase associated with various cellular activities ) family of proteins , which is responsible for ATPase activity and interaction with σ 54 , and ( iii ) an N-terminal GAF ( cGMPspecific and - stimulated phosphodiesterases , Anabaena ad-enylate cyclases and E. coli FhlA ) domain , a ubiquitous signalling domain that is the site of signal sensing [ 3 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16417519" target="_blank">16417519</a>]<br></li>
<li>However , internal deletion of NorR , eliminating amino acids 30 -- 164 containing the putative signaling domain ( 31 , 39 ) , but retaining the entire central 54-interacting ATPase domain and the C-terminal DNA-binding domain , induced NOR activity in the absence of NO ( Fig. 3A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12529359" target="_blank">12529359</a>]<br></li>
<li>Further deletion of NorR to amino acid 214 , eliminating part of the 54-interacting ATPase domain , did not induce - galactosidase activity ( data not shown ) thus further delineating the requirement for 54 interaction with NorR for transcriptional activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12529359" target="_blank">12529359</a>]<br></li>
<li>Induction of norVW Transcription Is Dependent on 54 -- The central domain of the tripartite NorR protein is highly homologous with 54-dependent response regulators ( 31 , 39 ) thus suggesting an important role for 54 in the NO response .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12529359" target="_blank">12529359</a>]<br></li>
<li>Similar to other σ 54-dependent transcriptional activators , NorR has a modular domain architecture [ 10 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15667300" target="_blank">15667300</a>]<br></li>
<li>To investigate whether NorR GAF promotes isomerization of σ 54-RNA polymerase holoenzyme , we employed a gel retardation assay to detect the formation of heparin-resistant open complexes , using a DNA template containing the norR -- norVW intergenic region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16417519" target="_blank">16417519</a>]<br></li>
<li>DNA looping , facilitated by IHF , may enable this form of the NorR oligomer to make contact with σ 54-RNA polymerase bound as a closed complex ( Figure 1B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16417519" target="_blank">16417519</a>]<br></li>
<li>IHF site 1 is located between the σ 54 promoter and the norV-proximal NorR binding site ( NorR1 ) consistent with a role in facilitating an interaction between DNA-bound NorR and σ 54-RNA polymerase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15667300" target="_blank">15667300</a>]<br></li>
<li>( D ) NorR catalyses the hydrolysis of ATP , resulting in the isomerization of σ 54-RNA polymerase and the formation of an open complex ( black dotted lines ) and transcription initiation ( red arrow ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16417519" target="_blank">16417519</a>]<br></li>
<li>The requirement for ATP in the open complex assays probably reﬂects the need for nucleotide hydrolysis by the AAA domain of þ NorR in remodelling the j - RNA polymerase-promoter complex to 54 activate transcription22 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16193057" target="_blank">16193057</a>]<br></li>
<li>The formation of the mono-nitrosyl iron complex upon binding of NO to the GAF-domain is presumably responsible for a conformational change in NorR that releases repression of the AAA + domain ( Figure 1C ) , allowing ATP hydrolysis and the isomerization of σ 54-RNA polymerase , resulting in open complex formation ( Figure 1D ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16417519" target="_blank">16417519</a>]<br></li>
<li>Expression of norVW requires the NO sensing regulatory protein NorR and is dependent on RNA polymerase containing the alternative sigma factor , σ 54 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15667300" target="_blank">15667300</a>]<br></li>
<li>Clearly , the DNA-dependent ATPase activity of full-length NorR requires activation of the ironcontaining species by NO , which may account for the signal-dependent isomerization of j - RNA polymerase that we observe 54 in the open complex assays .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16193057" target="_blank">16193057</a>]<br></li>
<li>Binding of NO stimulates the ATPase activity of NorR , enabling the activation of transcription by RNA polymerase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16193057" target="_blank">16193057</a>]<br></li>
<li>Binding of NO to the ferrous iron centre stimulates the ATPase activity of NorR , enabling NorR to activate transcription by σ 54-RNA polymerase [ 3 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16417519" target="_blank">16417519</a>]<br></li>
<li>For example , NorR 111 contains an His - X-Cys113 site reminiscent of the Cys75-X-77 His heme iron ligand-switch motif in the carbon monoxide-sensing CooA of Rhodospirillum rubrum ( 47 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12529359" target="_blank">12529359</a>]<br></li>
				<label class="DOM_collapse_similarity" for="dom_similarity_76">
haem iron centre in the transcription factor NorR senses nitric oxide .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22029434" target="_blank">22029434</a>]<br></label>
<input id="dom_similarity_76", type="checkbox">
<div class="DOM_similar">
<li>A non-haem iron centre in the transcription factor NorR senses nitric oxide .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22563051" target="_blank">22563051</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16428390" target="_blank">16428390</a>]<br></li></div>
				</div></ul><br>
			<label class="collapse_EVO" for="_3"><font size="5px">NorR evolution</font></label>
				<input id="_3" type="checkbox">
				<div class="EVO"><br><ul><li>NorR contains three domains : ( i ) a C-terminal DNA-binding domain predicted to interact with enhancer sequences , ( ii ) a central-domain belonging to the AAA + ( ATPase associated with various cellular activities ) family of proteins , which is responsible for ATPase activity and interaction with σ 54 , and ( iii ) an N-terminal GAF ( cGMPspecific and - stimulated phosphodiesterases , Anabaena ad-enylate cyclases and E. coli FhlA ) domain , a ubiquitous signalling domain that is the site of signal sensing [ 3 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16417519" target="_blank">16417519</a>]<br></li>
<li>As has been previously noted ( 11 ) , the product of ygaA is 42 % identical in sequence to the NorR protein of Ralstonia eutropha , which activates expression of a nitric oxide reductase in response to NO and to reactive nitrogen intermediates ( RNIs ) generated from sodium nitroprusside ( 11 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12142437" target="_blank">12142437</a>]<br></li>
<li>It should be noted that TetR , QacR , and the putative protein NorR are homologous , BltR and BmrR are members of the MerR family of transcriptional repressors ( 3 ) , and EmrR is a member of the MarR family of regulatory proteins ( 160 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8987357" target="_blank">8987357</a>]<br></li>
<li>Induction of norVW Transcription Is Dependent on 54 -- The central-domain of the tripartite NorR protein is highly homologous with 54-dependent response regulators ( 31 , 39 ) thus suggesting an important role for 54 in the NO response .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12529359" target="_blank">12529359</a>]<br></li>
<li>Transcription of the norVW genes is activated in response to NO by the σ 54-dependent regulator NorR , a member of the prokaryotic enhancer binding protein family .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16417519" target="_blank">16417519</a>]<br></li>
<li>Expression of norVW requires the σ 54-dependent NO-sensing transcriptional regulator NorR , a member of the prokaryotic EBP ( enhancer binding protein ) family [ 2 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16417519" target="_blank">16417519</a>]<br></li>
<li>NorR belongs to the family of two-component response regulators ( 42 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12529359" target="_blank">12529359</a>]<br></li>
<li>EPR spectroscopy showed that the nitrosyl complex in the GAF domain of NorR belongs to the -LCB- Fe ( NO ) -RCB- 7 ( S 1/4 3/2 ) family .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16193057" target="_blank">16193057</a>]<br></li>
				</div></ul><br>
			<label class="collapse_RP" for="_4"><font size="5px">Processes regulated by NorR</font></label>
				<input id="_4" type="checkbox">
				<div class="RP"><br><ul><li>NsrR , unlike 2 NorR , appears to be a global regulator of NO-induced stress and has been identified and studied in a wide range of Gram-negative and Gram-positive bacteria including Bacillus subtilis [ 12 ] , Salmonella enterica [ 13 ] , and the obligate human pathogens Neisseria meningitidis [ 14 ] and N. gonorrhoeae [ 15 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18989365" target="_blank">18989365</a>]<br></li>
<li>g008 Genes Functions Regulators Genomes Core members of the regulatory and metabolic networks dnrN Hypothetical cytoplasmic protein involved in anaerobic NO resistance HcpR Desulfuromonas HcpR2 NsrR NorR NnrR HcpR B. fragilis Enterobacteria , Shewanella spp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16261196" target="_blank">16261196</a>]<br></li>
<li>, T. denitrificans , C. violaceum Six denitrifying a-proteobacteria P. profundum Hypothetical heme-copper-contain-ing membrane protein nnrS NorR DNR NnrR NsrR DNR NnrR Periplasmic nitrite reductase nir genes ( nirS or nirK ) Cytoplasmic NO reductase norVW NsrR NorR HcpR E. coli , S. typhimurium , Erwinia spp. , two Vibrio spp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16261196" target="_blank">16261196</a>]<br></li>
<li>NsrR NorR HcpR NO reductase , membrane-bound nor genes , in particular norB NsrR NorR DNR NnrR NsrR DNR NnrR NsrR Periplasmic nitrous oxide reductase nos genes Flavohemoglobin , NO dioxygenase hmp DNR NorR Additional network m aprAB-qmoABC , sat bers em APS reductase , Qmo complex , ATP sulfurylase ( sulfate-reduction ) Putative sulfite/nitrite reductase , polyferredoxin Thiosulfate reductase phsA Ferredoxins-like Polyferredoxins-like NADH dehydrogenase Contain cytochrome c heme-bind-ing site Unknown Hemerythrin-like HcpR Desulfovibrio spp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16261196" target="_blank">16261196</a>]<br></li>
<li>These results demonstrate that YgaA ( NorR ) and YgaK ( NorV , flavoRb ) are essential for anaerobic NO consumption .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11751865" target="_blank">11751865</a>]<br></li>
<li>These results demonstrate important , albeit conditional , roles for YgaA ( NorR ) and YgaK ( NorV ) in NO detoxification .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11751865" target="_blank">11751865</a>]<br></li>
<li>oxide , mono-nitrosyl non-haem iron complex , NorR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16417519" target="_blank">16417519</a>]<br></li>
<li>strain TH2 ( BAC16772 ) , and Burkholderia fungorum ( ZP00028693 ) ge-nomes are found divergently transcribed from flavohemoglobin ( hmp ) genes suggesting a potential role for NorR in regulating NODs in response to NO .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12529359" target="_blank">12529359</a>]<br></li>
<li>g006 NorR Regulon In Vibrionales , the hcp-hcr operon is preceded by a gene that encodes a homolog of the NO-responsive regulator NorR , named NorR2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16261196" target="_blank">16261196</a>]<br></li>
<li>BBRC Biochemical and Biophysical Research Communications 328 ( 2005 ) 540 -- 544 www.elsevier.com/locate/ybbrc Binding of NorR to three D A sites is essential for pro activation of the ﬂavorubredoxin gene , the nitric oxide reductase f Escherichia coli N moter o Marta C. Justino , Vera M.M. Gonçalves , Lıgia M. Saraiva * Instituto de Tecnologia Quımica e Biológica , Universidade Nova de Lisboa , Apartado 127 Avenida da República ( EAN ) , 2781-901 Oeiras , Portugal Received 22 December 2004 Available online 13 January 2005 Abstract NorR is a nitric oxide sensor that in Escherichia coli regulates the gene encoding for ﬂavorubredoxin , an enzyme involved in nitrosative detoxiﬁcation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15694381" target="_blank">15694381</a>]<br></li>
<li>We have now demonstrated that the E. coli NorR homolog ( b2709 ) encoded by ygaA ( 36 ) and the adjacent gene ygaK ( encoding a flavoRb ( b2710 ) ) ( Fig. 1 ) are each required for expression of the efficient and inducible anaerobic NO meta-bolic activity ( Figs. 2 and 3 ) , the protection of NO-sensitive [ 4Fe-4S ] - containing ( de ) hydratases ( Figs. 4 and 6 ) , and the resistance of E. coli to NO-mediated growth inhibition under various metabolic conditions ( Fig. 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11751865" target="_blank">11751865</a>]<br></li>
<li>Production of recombinant E. coli NorR protein , ATPase activity assays , and electron paramagnetic resonance ( EPR ) studies .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22563051" target="_blank">22563051</a>]<br></li>
<li>( A ) Whole-cell EPR analysis of E. coli overexpressing NorR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22563051" target="_blank">22563051</a>]<br></li>
<li>E. coli and related microbes contain norR orthologues suggesting a global regulatory role for NorR in controlling defenses ( i.e. norVW , norBC , and hmp ) against the incipient toxicity of NO and secondarily derived reactive nitrogen species ( 8 , 31 , 32 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12529359" target="_blank">12529359</a>]<br></li>
<li>Keywords : NorR ; Nitric oxide ; Escherichia coli ; Flavorubredoxin gene regulation NorR is a r54 ( RpoN ) - dependent regulator , present in a wide range of organisms , proposed to be involved in the transcriptional response to nitrosative stress [ 1 -- 7 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15694381" target="_blank">15694381</a>]<br></li>
<li>These assays showed that NorR , a homolog of NO-responsive transcription factors in Ralstonia eutrophus , and Fur , the global repressor of ferric ion uptake , are major regulators of the response to reactive nitrogen species .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14718666" target="_blank">14718666</a>]<br></li>
<li>Mukhopadhyay P , Zheng M , Bedzyk LA , LaRossa RA & Storz G ( 2004 ) Prominent roles of the NorR and Fur regulators in the Escherichia coli transcriptional response to reactive nitrogen Outten FW , Djaman O & Storz G ( 2004 ) A suf operon requirement for Fe -- S cluster assembly during iron starvation in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16553864" target="_blank">16553864</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_18">
Mukhopadhyay P , Zheng M , Bedzyk LA , LaRossa RA , Storz G. Prominent roles of the NorR and Fur regulators in the Escherichia coli transcriptional response to reactive nitrogen species .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23075318" target="_blank">23075318</a>]<br></label>
<input id="rp_similarity_18", type="checkbox">
<div class="RP_similar">
<li>Mukhopadhyay , P. , Zheng , M. , Bedzyk , L. A. , LaRossa , R. A. & Storz , G. Prominent roles of the NorR and Fur regulators in the Escherichia coli transcriptional response to reactive nitrogen species .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16193057" target="_blank">16193057</a>]<br></li>
<li>[ 4 ] P. Mukhopadhyay , M. Zheng , L.A. Bedzyk , R.A. LaRossa , G. Storz , Prominent roles of the NorR and Fur regulators in the Escherichia coli transcriptional response to reactive nitrogen species , Proc .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15694381" target="_blank">15694381</a>]<br></li></div>
<li>Hutchings MI , Mandhana N , Spiro S ( 2002 ) The NorR Protein of Escherichia coli Activates Expression of the Flavorubredoxin Gene norV in Response to Reactive Nitrogen Species .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18989365" target="_blank">18989365</a>]<br></li>
<li>NorR Protein of Escherichia coli Activates Expression o Flavorubredoxin Gene norV in Response to Reactive Nitrogen Species The f the Matthew I. Hutchings , † Neeraj Mandhana , and Stephen Spiro * School of Biological Sciences , University of East Anglia , Norwich NR4 7TJ , United Kingdom Received 28 March 2002/Accepted 9 May 2002 The Escherichia coli norVW genes encode a ﬂavorubredoxin and NADH :( ﬂavo ) rubredoxin reductase , respectively , which are involved in nitric oxide detoxiﬁcation under anaerobic growth conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12142437" target="_blank">12142437</a>]<br></li>
<li>In this context , the E. coli NorR protein should be favored as a model for examining effects of reactive nitrogen species on transcription-factor-activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14718666" target="_blank">14718666</a>]<br></li>
<li>Proc Natl Acad Sci USA 98:10108 -- 10112 Hutchings MI , Mandhana N , Spiro S ( 2002 ) The NorR Protein of Escherichia coli activates expression of the Xavorubredoxin gene norV in response to reactive nitrogen species .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17024490" target="_blank">17024490</a>]<br></li>
<li>The NorR protein of Escherichia coli activates expression of the ﬂavorubredoxin gene norV in response to reactive nitrogen species .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17449618" target="_blank">17449618</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16428390" target="_blank">16428390</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16193057" target="_blank">16193057</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15375149" target="_blank">15375149</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_26">
[ 10 ] Hutchings , M.I. , Mandhana , N. and Spiro , S. ( 2002 ) The NorR protein of Escherichia coli activates expression of the # avorubredoxin gene norV in response to reactive nitrogen species .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12586421" target="_blank">12586421</a>]<br></label>
<input id="rp_similarity_26", type="checkbox">
<div class="RP_similar">
<li>[ 3 ] M.I. Hutchings , N. Mandhana , S. Spiro , The NorR protein of Escherichia coli activates expression of the ﬂavorubredoxin gene norV in response to reactive nitrogen species , J. Bacteriol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15694381" target="_blank">15694381</a>]<br></li>
<li>Hutchings MI , Mandhana N & Spiro S ( 2002 ) The NorR protein of Escherichia coli activates expression of the ﬂavorubredoxin gene norV in response to reactive nitrogen species .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22029434" target="_blank">22029434</a>]<br></li>
<li>Hutchings , M.I. , Mandhana , N. , and Spiro , S. ( 2002 ) The NorR protein of Escherichia coli activates expression of the ﬂavorubredoxin gene norV in response to reactive nitrogen species .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19656291" target="_blank">19656291</a>]<br></li></div>
<li>We demonstrate here the role of the E. coli NorR homolog YgaA in controlling NOR expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11751865" target="_blank">11751865</a>]<br></li>
<li>doi :10.1016 / j.bbrc .2005.01.008 Materials and methods Cloning and expression of E. coli NorR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15694381" target="_blank">15694381</a>]<br></li>
<li>YgaA strongly resembles the Ralstonia eutropha NorR protein that regulates expression of heme b3-iron nitric oxide reductase in response to NO [ 9 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12586421" target="_blank">12586421</a>]<br></li>
<li>For example , NorR regulates expression of the norAB genes encoding the respiratory NO reductase in Ralstonia eutropha [ 6 ] , and the flavohaemoglobin gene fhp in Pseudomonas aeru-54 ginosa [ 7 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16417519" target="_blank">16417519</a>]<br></li>
<li>In E. coli , NorR activates norVW transcription in anaerobic cultures in response to NO , nitrate , and nitrite ( which may act as sources of endogenous NO ) and nitroprusside and in aerobic cultures in response to S-nitrosoglutathione , acidiﬁed nitrite , and nitroprusside ( 4 , 10 , 15 , 19 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15375149" target="_blank">15375149</a>]<br></li>
<li>In Escherichia coli , the transcriptional regulators NorR and NsrR mediate adaptive responses to NO by controlling the expression of genes encoding enzymes that reduce or oxidize NO to less toxic species ( Mukhopadhyay et al. , 2004 ; Bodenmiller and Spiro , 2006 ; Spiro , 2007 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19656291" target="_blank">19656291</a>]<br></li>
<li>Another NO-responsive transcriptional factor , r54-depend-ent NorR , activates expression of the NO reductases norVW in E. coli and norAB in R. eutropha [ 21,22 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16261196" target="_blank">16261196</a>]<br></li>
<li>NorR is a r54-dependent transcriptional activator that regulates expression of the NO reductase operons , norVW in E. coli and norAB in R. eutropha [ 21,22 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16261196" target="_blank">16261196</a>]<br></li>
<li>In Escherichia coli , NorR activates the transcription of the norVW genes encoding a ﬂavorubredoxin ( FlRd ) and an associated ﬂavoprotein , respect-ively , which together have NADH-dependent NO reductase activity10 ,12 -- 14 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16193057" target="_blank">16193057</a>]<br></li>
<li>NorR of Escherichia coli is a 54-dependent transcriptional activator that regulates expression of the norVW genes encoding ﬂavorubredoxin and its associated redox partner , respectively ( 4 , 10 , 15 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15375149" target="_blank">15375149</a>]<br></li>
				</div></ul><br>
			<label class="collapse_SIT" for="_5"><font size="5px">NorR binding site</font></label>
				<input id="_5" type="checkbox">
				<div class="SIT"><br><ul><li>To construct pUC19NorR 30 -- 164 , a 1.1-kb fragment encoding the C terminus was PCR-amplified using pUC19NorR as the template and using the oligonucleotide primers 5 - GTTGCGGATCCAACAACTGGAAAGCCAGAATATGC-3 and 5-CATGCCTGCAGGATTTCTATCAGGCCG-3 containing BamHI and PstI sites ( underlined ) , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12529359" target="_blank">12529359</a>]<br></li>
<li>Sensor DNA Binding Activity of the Escherichia coli Nitric Oxide NorR Suggests a Conserved Target Sequence in Diverse Proteobacteria Nicholas P. Tucker ,1 Benoît D'Autre ́aux ,1 David J. Studholme ,2 Stephen Spiro ,3 * and Ray Dixon1 John Innes Centre , Colney , Norwich ,1 and Wellcome Trust Sanger Institute , Hinxton , Cambridge ,2 United Kingdom , and School of Biology , Georgia Institute of Technology , Atlanta , Georgia3 Received 6 May 2004/Accepted 7 July 2004 The Escherichia coli nitric oxide sensor NorR was shown to bind to the promoter region of the norVW transcription unit , forming at least two distinct complexes detectable by gel retardation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15375149" target="_blank">15375149</a>]<br></li>
				<label class="SIT_collapse_similarity" for="sit_similarity_3">
Tucker , N. P. , D'Autre ́aux , B. , Studholme , D. J. , Spiro , S. & Dixon , R. DNA binding activity of the Escherichia coli nitric oxide sensor NorR suggests a conserved target sequence in diverse Proteobacteria .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16193057" target="_blank">16193057</a>]<br></label>
<input id="sit_similarity_3", type="checkbox">
<div class="SIT_similar">
<li>Tucker NP , D'Autreaux B , Studholme DJ , Spiro S , Dixon R ( 2004 ) DNA binding activity of the Escherichia coli nitric oxide sensor NorR suggests a conserved target sequence in diverse proteobacteria .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18989365" target="_blank">18989365</a>]<br></li>
<li>[ 6 ] N.P. Tucker , B. D Autreaux , D.J. Studholme , S. Spiro , R. Dixon , DNA binding activity of the Escherichia coli nitric oxide sensor NorR suggests a conserved target sequence in diverse proteobacteria , J. Bacteriol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15694381" target="_blank">15694381</a>]<br></li></div>
<li>The consensus sequences of NorR and NorR2 recognition motifs identiﬁed in various taxonomic groups have only a limited number of universally conserved positions ( Figure 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16261196" target="_blank">16261196</a>]<br></li>
<li>The NorR1 and NorR2 consensus sequences differ signiﬁcantly in four positions ( C7 , G13 for NorR1 and G2 , C18 for NorR2 ) , allowing for discrimination of the target sites by the NorR paralogs in these species .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16261196" target="_blank">16261196</a>]<br></li>
<li>There are very similar patterns of methylation protection and enhancement at each site , indicating that NorR interacts with each sequence in a similar manner .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15375149" target="_blank">15375149</a>]<br></li>
<li>In the unﬁnished genome of P. stutzeri , the candidate tandem NorR sites followed by candidate r54 promoters were found upstream of the hmp and dnrN genes , but the norR gene was not found in the sequenced portion of the genome .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16261196" target="_blank">16261196</a>]<br></li>
<li>Figure 1 Model of NorR-dependent norVW activation ( A ) NorR ( the AAA + and C-terminal domains are depicted as a green oval and the GAF domain by a blue box ) binds to three sites upstream of the norVW promoter and core RNA polymerase ( RNAP , purple ) associated with σ 54 ( yellow ) binds to the norVW promoter at sequences centred at − 12 and − 24 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16417519" target="_blank">16417519</a>]<br></li>
<li>Gel retardation experiments in 10 mM Tris-HCl ( pH 8.0 ) -- 5 mM MgCl2 -- 30 mM KCl -- 0.4 mg of BSA/ml demonstrated that NorR binds to a 362-bp DNA fragment spanning the norR-norVW intergenic region ( Fig. 1A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15375149" target="_blank">15375149</a>]<br></li>
<li>The most likely explanation for this failure of norR to complement the norR mutation in trans is that the mutation disrupts a cis-acting sequence ( perhaps a NorR binding site ) within the norR coding region that is required for norVW expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12142437" target="_blank">12142437</a>]<br></li>
<li>As part of an effort to understand the mechanism of regulation of the norVW promoter and to help identify additional NorR targets , we have set out to determine the DNA sequence ( s ) recognized by NorR in the norR-norVW intergenic region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15375149" target="_blank">15375149</a>]<br></li>
<li>Role of the enhancer sites in NorR - dependent regulation To investigate the role of individual NorR binding sites in promoter regulation , we disrupted each binding site in the norR -- norVW intergenic region individually , by altering the conserved motif from GT - ( N ) - AC to GG - ( N ) - CC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16417519" target="_blank">16417519</a>]<br></li>
<li>Puriﬁed NorR binds to the norR-norVW intergenic region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15375149" target="_blank">15375149</a>]<br></li>
<li>We have previously identified three binding sites in the E. coli norR -- norVW intergenic region to which NorR binds co-operatively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16417519" target="_blank">16417519</a>]<br></li>
<li>NorR bound co-operatively to multiple sites in the norR/norVW intergenic region in gel retardation assays .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15667300" target="_blank">15667300</a>]<br></li>
<li>DNA fragments containing combinations of the NorR binding sites ( norboxes ) , with sizes ranging from 109 to 320 bp , were PCR ampliﬁed using chromosomal DNA of E. coli K-12 ATCC 23716 as template .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15694381" target="_blank">15694381</a>]<br></li>
<li>Alignments of the norVW promoters from E. coli , S. enterica serovar Typhimurium , and Shigella ﬂexneri and the chromosomal and megaplasmid copies of the norAB promoter from R. eutropha ( 20 ) indicate that each contains three conserved NorR binding sites with the minimal consensus GT - ( N7 ) - AC ( Fig. 5B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15375149" target="_blank">15375149</a>]<br></li>
<li>NorR binds to three tandem operator regions , inverted-repeats with degenerate consensus GT - ( N7 ) - AC , which are localized upstream of the r54 promoter site [ 23 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16261196" target="_blank">16261196</a>]<br></li>
<li>We anticipate that the functional oligomeric state of NorR is at least a hexamer , since the symmetrical nature of the NorR consensus binding site suggests that NorR binds to each site as a dimer .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16417519" target="_blank">16417519</a>]<br></li>
<li>Moreover , the presence of a low-afinity binding site is enough to ensure speciﬁc binding of NorR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15694381" target="_blank">15694381</a>]<br></li>
<li>Identification of the consensus NorR binding site will facilitate the search for other potential members of the NorR regulon in diverse bacteria .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15667300" target="_blank">15667300</a>]<br></li>
<li>The derived consensus sequence for NorR binding sites was used to search for novel members of the E. coli NorR regulon and to show that NorR binding sites are partially conserved in other members of the proteobacteria .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15375149" target="_blank">15375149</a>]<br></li>
<li>The sequences of the three NorR binding sites that we have identiﬁed will facilitate future efforts to identify additional members of the NorR regulon by other techniques .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15375149" target="_blank">15375149</a>]<br></li>
<li>The identiﬁcation of a consensus for NorR binding sites will help to elucidate additional members of the NorR regulon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15667300" target="_blank">15667300</a>]<br></li>
<li>The long noncoding 54 region upstream of ybiJ contains neither a predicted promoter nor a strongly predicted site for NorR binding , suggesting that ybiJ is not directly activated by NorR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15375149" target="_blank">15375149</a>]<br></li>
<li>Although the high affinity NorR 1 site is suitably positioned to act as an upstream activ-ator sequence for norVW , it is possible that co-operative binding of NorR to all three sites is necessary for maximal promoter activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15667300" target="_blank">15667300</a>]<br></li>
<li>We have puriﬁed NorR and shown that it binds to three sites in the norVW promoter region , located 75 -- 140 bp upstream of the experimentally veriﬁed transcription start site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15667300" target="_blank">15667300</a>]<br></li>
<li>DNase I footprinting and methylation protection experiments identified three NorR binding sites located 75 -- 140 bp upstream of the norV transcription start site [ 13 ] ( Figure 1B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15667300" target="_blank">15667300</a>]<br></li>
<li>Three binding sites for NorR and two integration host factor binding sites were identiﬁed in the norR-norV intergenic region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15375149" target="_blank">15375149</a>]<br></li>
<li>We have also identiﬁed two binding sites for the integration host factor , one between the NorR sites and the σ 54-RNA polymerase binding site , and a second downstream of the norVW transcription start site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15667300" target="_blank">15667300</a>]<br></li>
<li>Using S1 nuclease mapping experiments , we have determined that the norR transcription initiation site is in the centre of NorR binding site 3 , which is the closest of the three sites to the norR coding sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16417519" target="_blank">16417519</a>]<br></li>
<li>From these footprinting experiments , we have been able to deduce a consensus for NorR binding sites , which is GT ( N7 ) AC ( A ) Consensus NorR binding site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15667300" target="_blank">15667300</a>]<br></li>
<li>Alignment of the three E. coli NorR binding sites revealed that sites 1 and 3 both contain the inverted-repeat GTCA - ( N3 ) - TGAC , while site 2 is GTCA - ( N3 ) - CGAC ( Fig. 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15375149" target="_blank">15375149</a>]<br></li>
<li>( B ) Alignment of conserved NorR binding sites in σ 54-dependent promoters upstream of genes implicated in NO detoxiﬁcation from various bacterial species .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15667300" target="_blank">15667300</a>]<br></li>
<li>Conserved sequence elements in predicted NorR binding sites and 54 promoters are highlighted .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15375149" target="_blank">15375149</a>]<br></li>
<li>Footprinting NorR binding sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15375149" target="_blank">15375149</a>]<br></li>
<li>NorR sites are labelled 1 -- 3 as in Figure 1 and the σ 54-RNAP binding site is labelled σ 54 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15667300" target="_blank">15667300</a>]<br></li>
				</ul></div><br>
			<label class="collapse_TU" for="_6"><font size="5px">Transcription unit regulated by NorR</font></label>
				<input id="_6" type="checkbox">
				<div class="TU"><br><ul><li>A non-haem iron centre in the transcription factor did not exhibit the g 1/4 4 EPR signal ( Fig. 1d , spectrum 1 ) , demonstrating that this signal does not arise from other proteins as a consequence of NorR-dependent activation of gene expression , and that the paramagnetic centre is located within the GAF domain of NorR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16193057" target="_blank">16193057</a>]<br></li>
<li>Under these conditions , the behavior of the E. coli wild-type strain was rescued ( Fig. 3B ) , showing that when it is expressed independently of its own promoter , i.e. , not regulated by NorR , FlRd confers protection to E. coli against macrophages .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22563051" target="_blank">22563051</a>]<br></li>
<li>Transcriptome analysis of the Escherichia coli norR mutant strain grown under aerobic conditions and submitted to S-nitrosoglutathione ( GSNO ) showed that several genes are regulated by NorR [ 4 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15694381" target="_blank">15694381</a>]<br></li>
<li>NorR binds co-operatively to three enhancer sites to regulate transcription of both norVW and the divergently transcribed norR gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16417519" target="_blank">16417519</a>]<br></li>
<li>In the unﬁnished genome of P. stutzeri , the candidate tandem NorR sites followed by candidate r54 promoters were found upstream of the hmp and dnrN genes , but the norR gene was not found in the sequenced portion of the genome .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16261196" target="_blank">16261196</a>]<br></li>
<li>Intriguingly , NorR homologs are also located adjacent to the flavoHb gene ( hmp ) in both Vibrio cholera ( 37 ) and Pseudomonas aeruginosa ( 38 ) , suggesting a common control for NO detoxification systems in various organisms .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11751865" target="_blank">11751865</a>]<br></li>
<li>Neither NorR nor 54 was essential for NO-mediated induction of the NO di-oxygenase ( flavohemoglobin ) encoded by hmp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12529359" target="_blank">12529359</a>]<br></li>
<li>strain TH2 ( BAC16772 ) , and Burkholderia fungorum ( ZP00028693 ) ge-nomes are found divergently transcribed from flavohemoglobin ( hmp ) genes suggesting a potential role for NorR in regulating NODs in response to NO .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12529359" target="_blank">12529359</a>]<br></li>
<li>Thus , NorR is a transcriptional activator protein that responds to NO and it is tempting to suggest that the NorR-orthologues located upstream and divergently transcribed from their corresponding hmp genes in Vibrio cholerae , V. parahemolyticus , V. vulniWcus , Pseudomonas putida , P. aeruginosa , Azotobacter vinelandii , Burkholderia fungorum and Burkholderia , sp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17024490" target="_blank">17024490</a>]<br></li>
<li>g006 NorR Regulon In Vibrionales , the hcp-hcr operon is preceded by a gene that encodes a homolog of the NO-responsive regulator NorR , named NorR2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16261196" target="_blank">16261196</a>]<br></li>
<li>NorR Protein of Escherichia coli Activates Expression o Flavorubredoxin Gene norV in Response to Reactive Nitrogen Species The f the Matthew I. Hutchings , † Neeraj Mandhana , and Stephen Spiro * School of Biological Sciences , University of East Anglia , Norwich NR4 7TJ , United Kingdom Received 28 March 2002/Accepted 9 May 2002 The Escherichia coli norVW genes encode a ﬂavorubredoxin and NADH :( ﬂavo ) rubredoxin reductase , respectively , which are involved in nitric oxide detoxiﬁcation under anaerobic growth conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12142437" target="_blank">12142437</a>]<br></li>
<li>This arrangement of regulatory elements indicates dual positive control of the hmp expression by different NO-responsive activators , r54-dependent NorR and r70-depend-ent DNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16261196" target="_blank">16261196</a>]<br></li>
<li>It was recently reported that a member of this family ( NorR , formerly known as YgaA ) in E. coli is responsible for the induction of the norVW operon ( encoding a Xavorubredoxin , divergently transcribed from norR ) in response to NO ( Hutchings et al. 2002 ; Gardner et al. 2003 ; daCosta et al. 2003 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17024490" target="_blank">17024490</a>]<br></li>
<li>The key NO detoxifying enzymes are the ﬂavohaemoglobin ( encoded by the hmp gene ) and the ﬂavorubredoxin ( encoded by norVW ) , which are regulated by NsrR and NorR respectively ( Hutchings et al. , 2002 ; Gardner et al. , 2003 ; Bodenmiller and Spiro , 2006 ) , and the respiratory nitrite reductase , Nrf , which reduces both nitrite and NO to ammonia ( Poock et al. , 2002 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19656291" target="_blank">19656291</a>]<br></li>
<li>The core regulon members , that is , genes regulated by the nitrogen-oxide responsive factors NsrR , HcpR , DNR , NnrR , and NorR in many genomes , are the hybrid cluster protein gene hcp , the NO-detoxifying ﬂavohemoglobin hmp , two hypothetical genes dnrN and nnrS , NO reductase operon norVW , and multiple denitriﬁcation genes , nir , nor , and nos , encoding nitrite , NO , and nitrous oxide reductases , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16261196" target="_blank">16261196</a>]<br></li>
<li>The transcription of the norV gene is strongly upregulated in cells cultured anaerobically and exposed to NO , through the activation of the nitric oxide sensor NorR ( 12 , 21 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22563051" target="_blank">22563051</a>]<br></li>
<li>The homologous NorR protein of Ralstonia eutropha regulates expression of an NO reductase and can also be activated by nitroprusside and , presumably , NO ( 20 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15375149" target="_blank">15375149</a>]<br></li>
<li>Another NO-responsive transcriptional factor , r54-depend-ent NorR , activates expression of the NO reductases norVW in E. coli and norAB in R. eutropha [ 21,22 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16261196" target="_blank">16261196</a>]<br></li>
<li>NorR is a r54-dependent transcriptional activator that regulates expression of the NO reductase operons , norVW in E. coli and norAB in R. eutropha [ 21,22 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16261196" target="_blank">16261196</a>]<br></li>
<li>NorR of Escherichia coli is a 54-dependent transcriptional activator that regulates expression of the norVW genes encoding ﬂavorubredoxin and its associated redox partner , respectively ( 4 , 10 , 15 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15375149" target="_blank">15375149</a>]<br></li>
<li>NorR is a σ 54-dependent transcriptional activator required for the expression of norVW in response to NO , nitrate , nitrite and nitroprusside under both aerobic and anaerobic conditions [ 4,6 -- 9 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15667300" target="_blank">15667300</a>]<br></li>
<li>The fact that NorR can regulate gene expression under oxic conditions ( which inactivate the NO reductase activity of the ﬂavorubredoxin ) raises the possibility that there are additional genes regulated by NorR , the products of which have roles in aerobically growing cultures .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15375149" target="_blank">15375149</a>]<br></li>
<li>The only regulator known to serve exclusively as an NO sensor in E. coli is NorR , which activates transcription of the norVW genes encoding the ﬂavorubredoxin and associated ﬂavoprotein that together reduce NO to nitrous oxide ( 7 , 13 , 14 , 16 , 18 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16428390" target="_blank">16428390</a>]<br></li>
<li>The norVW genes are activated in response to NO by NorR ( 7 , 14 , 42 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16428390" target="_blank">16428390</a>]<br></li>
<li>DISCUSSION Our data demonstrate that the exposure of E. coli to NO induces transcription of the norV and norW genes via a NorR and 54-dependent mechanism .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12529359" target="_blank">12529359</a>]<br></li>
<li>The expression of the two most strongly regulated genes , norV and norW , has been shown to be regulated by NorR by assays of lacZ operon fusions ( 15 , 16 , 22 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14718666" target="_blank">14718666</a>]<br></li>
<li>NorR D99A is unable to activate expression of norV in vivo , whilst NorR C113G activates norV expression constitutively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16417519" target="_blank">16417519</a>]<br></li>
<li>Under anaerobic conditions , the transcription of nrfA is induced , and so both NrfA and NorV , which is activated by NorR , are important for survival .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21265775" target="_blank">21265775</a>]<br></li>
<li>Data presented in this paper dem-onstrate that the NorR protein is indeed required for transcription of norV and that expression of the structural genes is activated in both aerobic and anaerobic cultures by RNIs .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12142437" target="_blank">12142437</a>]<br></li>
<li>21 Expression of NorR and GAFNorR was induced at an absorbance ( A600nm ) of 0.6 with 50 mM isopropyl-b-D-thiogalactoside .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16193057" target="_blank">16193057</a>]<br></li>
<li>NorR was produced by cloning the entire norR DNA coding region ( 1631 bp ) into NdeI -- EcoRI digested pT7-7 , and expression in E. coli BL21 ( DE3 ) cells grown at 28 C , until OD600 0.4 and induced with 500 lM IPTG for 4 h. NorR was then puriﬁed from cell extracts using Q-Sepharose XK26/14 and Superdex 75 ( XK26/50 ) columns .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15694381" target="_blank">15694381</a>]<br></li>
				</div></ul></div><br>
	</div>
</body>
</html>