<!doctype html>
<style>
@charset "utf-8";
body {
	height: 100%;
	margin: 0;
	padding: 0;
}

.banner {
	position: fixed;
	top: 0;
	width: 100%;
	height: 65px;
	background: gray;
	line-height: 65px;
}
.spacer {
	height: 10px;
}

.main-content {
	padding: 5%;
}

.collapse_ACT {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-style: none solid none none;
	border-color: #B4045F;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_ACT + input{
	display: none;
}
.collapse_ACT + input + div{
	display: none;
}
.collapse_ACT + input:checked + div{
	display: block;
}
.ACT {
	text-align: justify;
	text-justify: inter-word;
}
.ACT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B4045F;
}
.ACT_collapse_similarity:hover{
	background: #e5d3dc;
}
.ACT_collapse_similarity + input {
	display: none;
}
.ACT_collapse_similarity + input + div{
	display: none;
}

.ACT_collapse_similarity + input:checked + div{
	display: block;
}
.ACT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.ACT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B4045F;
	padding-left: 5px;
}
.ACT_collapse_similarity:hover{
	background: #e5d3dc;
}
.ACT_collapse_similarity + input {
	display: none;
}
.ACT_collapse_similarity + input + div{
	display: none;
}
.ACT_collapse_similarity + input:checked + div{
	display: block;
}
.ACT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.ACT_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #B4045F;
}
.ACT_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_DOM {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-style: none solid none none;
	border-color: #B404AE;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_DOM + input{
	display: none;
}
.collapse_DOM + input + div{
	display: none;
}
.collapse_DOM + input:checked + div{
	display: block;
}
.DOM {
	text-align: justify;
	text-justify: inter-word;
}
.DOM_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B404AE;
	padding-left: 5px;
}
.DOM_collapse_similarity:hover{
	background: #e8d7e7;
}
.DOM_collapse_similarity + input {
	display: none;
}
.DOM_collapse_similarity + input + div{
	display: none;
}
.DOM_collapse_similarity + input:checked + div{
	display: block;
}
.DOM_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}.DOM_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #B404AE;
}
.DOM_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_EVO {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #7401DF;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_EVO + input{
	display: none;
}
.collapse_EVO + input + div{
	display: none;
}
.collapse_EVO + input:checked + div{
	display: block;
}
.EVO {
	text-align: justify;
	text-justify: inter-word;
}
.EVO_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #7401DF;
	padding-left: 5px;
}
.EVO_collapse_similarity:hover{
	background: #e2d7ed;
}
.EVO_collapse_similarity + input {
	display: none;
}
.EVO_collapse_similarity + input + div{
	display: none;
}
.EVO_collapse_similarity + input:checked + div{
	display: block;
}
.EVO_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.EVO_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #7401DF;
}
.EVO_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_RP {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #0174DF;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_RP + input{
	display: none;
}
.collapse_RP + input + div{
	display: none;
}
.collapse_RP + input:checked + div{
	display: block;
}
.RP {
	text-align: justify;
	text-justify: inter-word;
}
.RP_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #0174DF;
	padding-left: 5px;
}
.RP_collapse_similarity:hover{
	background: #c9d7e2;
}
.RP_collapse_similarity + input {
	display: none;
}
.RP_collapse_similarity + input + div{
	display: none;
}
.RP_collapse_similarity + input:checked + div{
	display: block;
}
.RP_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.RP_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #0174DF;
}
.RP_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_SIT {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #0489B1;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_SIT + input{
	display: none;
}
.collapse_SIT + input + div{
	display: none;
}
.collapse_SIT + input:checked + div{
	display: block;
}
.SIT {
	text-align: justify;
	text-justify: inter-word;
}
.SIT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #0489B1;
	padding-left: 5px;
}
.SIT_collapse_similarity:hover{
	background: #d0dfe2;
}
.SIT_collapse_similarity + input {
	display: none;
}
.SIT_collapse_similarity + input + div{
	display: none;
}
.SIT_collapse_similarity + input:checked + div{
	display: block;
}
.SIT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.SIT_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #0489B1;
}
.SIT_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_TU {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #04B486;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_TU + input{
	display: none;
}
.collapse_TU + input + div{
	display: none;
}
.collapse_TU + input:checked + div{
	display: block;
}
.TU {
	text-align: justify;
	text-justify: inter-word;
}
.TU_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #04B486;
	padding-left: 5px;
}
.TU_collapse_similarity:hover{
	background: #d0e2de;
}
.TU_collapse_similarity + input {
	display: none;
}
.TU_collapse_similarity + input + div{
	display: none;
}
.TU_collapse_similarity + input:checked + div{
	display: block;
}
.TU_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.TU_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #04B486;
}
.TU_similar > a {
	text-decoration: none;
	color: gray;
}

ul { 
	list-style-type: none;
	padding: 0;
	margin: 0;
}
ul li > a {
	text-decoration: none;
	color: gray;
}
ul li:hover, ul li:focus {
	background: #F0F0F0;
}

ul li:first-child span {
  border-left:none;  
}

</style>
<html>
<head>
<meta charset="utf-8">
<title>TF properties - NtrC</title>
<meta name="theme-color" content="#ffffff">
</head>
<body>
	<div class="banner">
		<center><font color="white" size="16px" face="arial">NtrC</font></center>
	</div>
	<div align="center" class="main-content">
		<div class="spacer">
	</div><br>			<label class="collapse_ACT" for="_1"><font size="5px">NtrC transcriptional activity</font></label>
				<input id="_1" type="checkbox">
				<div class="ACT"><br><ul><li>The RR-bound 32P was stably retained for the A1 group RRs such as pArcA , pCpxR , pCusR , pKdpE , and pNarL but was rapidly released for the A2 group RRs , including pBaeR , pCheB , pCheY , pCreB , pDcuR , pNarP , pNtrC , pTorR , pUhpA , pYehT , and pYfhA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15522865" target="_blank">15522865</a>]<br></li>
<li>a Results Chimeric XNtrC proteins are active in vivo and in vitro To test the dimerization properties of various response-regulator domains , we used the X system introduced by Hu et al. ( 1990 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7641688" target="_blank">7641688</a>]<br></li>
<li>DSETIGFF 209 QSSRVMAAVVRE.ARRMAPLDAPLLIEGETGTGKELLARACHLASPR.GQSPFMALNCAGLPESMAETELFGYGPGAFEGARPEGKLGLL 213 GHTPTMRRVFDQ.IRRVAKWNSTVLVLGESGTGKELIASAIHYKSPRAHR.PFVRLNCAALPETLLESELFGHEKGAFTGA.VKQRKGRF 237 GHSPAYKRICET.IDKAARGRVSVLLLGETGVGKEVIARSVHLRSERAEQ.PFVAVNCAAIPPDLIESELFGVDKGAYTGA.VNARAGRF 137 GESPPMR.ALRNQIGKLARSQAPVYISGESGSGKELVARLIHEQGPRIER.PFVPVNCGAIPSELMESEFFGHKKGSFTGA.IEDKQGLF 142 GEAPAMQDVFRI.IGRLSRSSISVLINGESGTGKELVAHALHRHSPRAKA.PFIALNMAAIPKDLIESELFGHEKGAFTGA.NTIRQGRF TyrR NtrC Walker B • • • • • • • • • 288 EQANGGSVLLDEIGEMSPR.MQAKLLRFLNDGTFRR.VGEDHEVHVDVRVICATQKNLVELVQKGMFREDLYYRLNVLTLNLPPLRDCPQ 99 EYANKGTVLLDGIAELS.LSLQAKLLRFLTDGSFRR.VGEEKEHYANVRVICTSQVPLHLLVEQGKVRADLFHRLNVLTINVPALRDRMA 297 ELTAGGTLFLDGVGEMSPR.LQAKLLRFLQDGCFRR.VGSDEEVYLDVRVICATQVDLSELCAKGEFRQDLYHRLNVLSLHIPPLRECLD 300 EQADGGTLFLDEIGEISPM.FQAKLLRVLQEGEFER.VGGNQTVRVNVRIVAATNRDLESEVEKGKFREDLYYRLNVMAIRIPPLRERTA 324 ERANGGTIFLDEVIELTPR.AQATLLRVLQEGELER.VGGDRTRKVDVRLITATNENLEEAVKMGRFRADLFFRLNVFPVHIPPLRERVE 224 QAASGGTLFLDEVADL.PMAMQVKLLRAIQE.KAVRAVGGQQEVAVDVRILCATHKDLAAEVGAGRFRQDLYYRLNVIELRVPPLRERRE 229 EQADGGTLFLDEIGDM.PLDVQTRLLRVLADGQFYR.VGGYAPVKVDVRIIAATHQNLEQRVQEGKFREDLFHRLNVIRVHLPPLRERRE 3 PASE-Linker • • • • • • • • 376 DIMP.LTELFVARFADEQ.GVPRPKLA.AD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11923293" target="_blank">11923293</a>]<br></li>
<li>YRVGGYAPVKV DVRIIAATHQNLEQRVQEG KFREDLFHRLNVIRVHLPPLRERREDI YRVGGYAPVKV DVRIIAATHQNLELRVQEG KFREDLFHRLNVIRVHLPPLRERREDI TTVGGRTPIKT DVRIVAASNKDLRILIQQG LFREDLFFRLNVVPLRVPPLRERIEDL ALPD PPRIMATTQVDLGALMEAGRRFRQDLYYRLGGVTLAVPALRERVEDI TPLGTNEVRPV NLRVVAAAKIDLGDPAVRG DFREDLYYRLNVVTISIPPLRERRDDI TPLGTNLTRPV DIRWAAAKVDLGDPAARG DFREDLYYRLNVVTLSIPPLRERRDDI RP FRLVGVGSASLVEQAAAN QIAAELYYCFAMTQIACQSLSQRPDDI ERVGDPVTRRA DVRILAATNRDLGAMVAQG QFREDLLYRLNVIVLNLPPLRERAEDI QRVGSNQIISV DVRLIAATHRDLAAEVNAG RFRQDLYYRLNVVAIEVPSLRQRREDI ERVGGDRTRKV DVRLITATNENLEEAVKMG RFRADLFFRLNVFPVHIPPLRERVEDI ERVGGSKPVKV DVRIIAATNRNLVEMVEQG TFREDLYYRLNVFPITVPPLRERGSDV ERVGGNTTIHV DLRVIAATNRNLAEMVADG TFAEDLYYRLNVFPITIPPLRERGSDI ERLGSNKIIQT DVRLIAATNRDLKKMVADR EFRSDLYYRLNVFPIHLPPLRERPEDI ERVGGDETLRV NVRIIAATNRHLEEEVRLG HFREDLYYRLNVMPIALPPLRERQEDI DRVGGSKPVKV NIRILATSNRDLAQAVKDG TFREDLLYRLNVVNLRLPPLRERPADV ERLGGTKTLKV DVRVICATNKNLEVAVLRG EFRADLYYRINVVPIILPPLRQRDGDI ERVGGTKTLKV DVRLIFATNKDLEMAVQNG EFREDLYYRISGVPLILPPLRHRDGDI ERVGGTRTLKV NVRLVCATNKDLETAVAAG EFRADLYYRINVVPITLPPLRQRDGDI ERVGGAKTIKV DTRT ~ AATNRDLEDAVARG QSRADLHPRICVVPSVLPPLRCRRSDI ERVGGNRTLKV DqRLVCATNKNLEEAVSKG EFRADLYYRIHVVPLILPPLRERPGDI ERVGSNHTIKV DVRVIAATNRNLEEAVARS EFRADLYYRISVVPLLLPPLRERESDI RRVGEDHEVHV DVRVICATQKNLVELVQKG MFREDLYYRLNVLTLNLPPLRDCPQDI NRLGESEHKRTSNVLFICATTENPSSALLKT FLRRIPMTIHIPSLEERSLNE NtrC Eco NtrC Kpn NtrC Bpa NifRl Rca DctD Rle DctD Rme PgtA Sty AIgB Pae HydG Eco XylR Ppu AnfA Avi VnfA Avi FhlA Eco NifA Kpn FIbD Ccr NifA Rle NifA ~ me NifA rlBT NifA Rca NifA Aca Ni fA Bj a TyrR Eco LevR Bsu ERVGG ... .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7699720" target="_blank">7699720</a>]<br></li>
<li>Proteins in classes 4 , 5 and 6 are exceptions 7O ENAF I0 20 30 40 50 60 NtrC NtrC NifRl NtrC PgtA DctD FixJ NodW CheY SpoOF RRHM RRHM OmpR PhoB CreB ArcA PetR VirG CutR KdpE PhoB PhoP TctD AIgB YLB3 RcsC BvgS HydG CheY FrzE BarA TrpO HvgA FimZ DegU NarL AIgRI YecB UhpA RcsB Agm & CheB CheB FrzG Spo0A MrkE AgrA Coma F!bD Bpa Rme Rca Eco Sty Rle Rme Bja Bsu Bsu Cca Pae Eco Bsu ECO Eco Rca Atu Sli Eco Eco Sty Sty Pae Lin Eco Bpe Eco Eco M ~ a Eco Pae Bpe Eco Bsu Eco Pae Eco Eco Eco Pae Bsu Eco Mxa Bsu Kpn Sau Bsu Ccr 6 ILVADDDTAIKTVLNQA 6 ILVADDD ~ IRTVLNQA 5 VLVADDDRTIRTVLTQA 6 V % ~ VDDDSSIR ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7699720" target="_blank">7699720</a>]<br></li>
<li>) Major clusters 1 , 2 , and 3 are associated primarily or exclusively with the three major families of DNA binding NtrC Bpa NtrC Rme NifRl Rca Nt rC Eco PgtA Sty DetD Rle FixJ Kme NodW Bja CheY Bsu SpoOF BSu RRHM Cca RRHM Pae OmpR Eco PhoB Bsu CreB Eco ArcA Eco PetR Rca VirG Atu CutR Sli KdpE ECO PhoB Eco PhoP Sty TctD Sty A!gB Pae YLB3 Lin RcsC Eco BvgS Bpe HydG Eco CheY Eco FrzE Nc .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7699720" target="_blank">7699720</a>]<br></li>
<li>aerobic to anaerobic anaerobic to aerobic ArgR CysB BirA DgsA MalT GadW MetJ NikR OxyR SoxR SoxS DcuR CueR FadR HcaR NtrC PhoB PurR SgrR TdcA TrpR ZraR ArcA CpxR CRP CusR FhlA Fis FNR Fur H-NS IHF IscR Lrp ModE NarL NrdR PdhR FlhDC GlcC EvgA GadX LeuO PspF RbsR steady states Figure 3 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22870390" target="_blank">22870390</a>]<br></li>
<li>sitesa % CGb Count % Non-codinge Ada 3 38.7 AraC 6 40.3 2 3 100 100 47 76 100 28 33 32 73 38 100 52 100 86 79 89 100 67 26 100 28 100 70 37 67 28 100 82 100 47 50 51 100 100 38 33 64 3 6 100 100 38 54 100 23 40 18 43 29 100 38 100 39 27 53 58 71 19 100 23 33 47 100 40 65 21 23 60 25 100 41 100 26 43 36 100 35 35 30 64 36 23 22 26 14 100 83 28 ArcA 12 20.0 ArgR 17 22.9 CarP 2 46.0 Crp 49 37.0 CspA 4 41.3 CynR 2 33.3 CysB 3 34.2 CytR 5 37.8 DeoR 3 22.9 DnaA 8 36.7 FadR 7 47.0 FarR 4 20.0 FhlA 3 42.0 Fis 19 38.2 FlhCD 3 53.8 Fnr 9 25.8 FruR 12 42.7 Fur 9 23.4 GalR 7 43.8 GcvA 4 22.5 GlpR 13 39.6 HipB 4 40.9 Hns 15 31.5 IclR 2 26.7 Ihf 26 32.6 IlvY 2 32.4 LacI 3 44.4 LexA 18 33.9 Lrp 18 28.9 MalT 10 61.0 MarR 2 37.5 MelR 2 33.3 MetJ 15 40.1 MetR 8 33.3 NagC 6 22.5 NarL 9 34.6 NtrC 6 51.9 OmpR 9 26.7 OxyR 4 42.3 PdhR 3 33.3 PhoB 15 29.7 PurR 23 45.3 RhaS 2 50.0 RpoD15 27 37.4 RpoD16 48 34.9 RpoD17 116 37.3 RpoD18 34 38.0 RpoD19 25 38.2 RpoE 3 38.9 RpoH13 8 42.1 RpoH14 7 44.3 RpoN 7 40.6 RpoS17 15 43.0 RpoS18 7 45.2 SoxS 14 40.8 TorR 4 37.5 TrpR 4 38.5 Tus 6 26.8 TyrR 17 33.2 a The number of input sites used to construct the matrix .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9813115" target="_blank">9813115</a>]<br></li>
<li>of additional probable sites in predictionsd 3 3 7e 7 -- 4 22e 0 1 0 2 -- -- 5 0 1 -- 13 7 18 1 0 6e 1 1 1 3 Ada AraC ArcA ArgR CarP CpxR Crp CspA CynR CysB CytR DeoR DnaA FadR FhlA Fis FlhCD Fnr FruR Fur GalR GcvA GlpR IciA ( ArgP ) IclR Ihf IlvY LacI LexA Lrp MalT MarR MelR MetJ MetR Mlc ( DgsA ) ModE NagC NarL NarP NtrC OmpR OxyR PdhR PhoB PurR RhaR RhaS SoxR SoxS TorR TrpR TyrR Stem -- loopsf 6 2 0 5 2 9 1 3 4 5e 5 -- 5 5 4 6 3 4 2 -- 1 3 5 1 3 -- aThe number of genes in our study set with documented binding sites for each TF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11160901" target="_blank">11160901</a>]<br></li>
<li>Kutsche , M. , Leimkühler , S. , Angermüller , S. , and Klipp , W. ( 1996 ) Promoters controlling expression of the alternate nitrogenase and the molybdenum uptake system in are activated by NtrC , independent of s54 , and are repressed by molybdenum .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466267" target="_blank">9466267</a>]<br></li>
<li>Kutsche , M. , Leikkuhler , S. , Angermuller , S. , and Klipp , W. ( 1996 ) Promoters controlling expression of the alternate nitrogenase and the molybdenum uptake system in Rhodo-bacter capsulatus are activated by NtrC , independent of s 54 , and are repressed by molybdenum .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9044285" target="_blank">9044285</a>]<br></li>
<li>FEMS Microbiol Rev 22:503 -- 521 Kutsche M , Leimkuhler S , Angermuller S , Klipp W ( 1996 ) Promoters controlling expression of the alternative nitrogenase and the molybdenum uptake system in Rhodobacter capsulatus are activated by NtrC , independent of sigma54 , and repressed by molybdenum .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16205910" target="_blank">16205910</a>]<br></li>
<li>Kutshe , M. , Leimkiihler , S. , Angemuller , S. & Klipp , W. ( 1996 ) Prornoters controlling expression of the alternative nitrogenase and the mo-lybdenum uptake system in Rhodobacter capsulatus are activated by NtrC , independent of os4a , nd repressed by molybdenum , 1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9210473" target="_blank">9210473</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_14">
Promoters controlling expression of the alternative nitrogenase and the molybdenum uptake system in Rhodobacter capsulatus are activated by NtrC , independent of 54 , and repressed by molybdenum .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12029041" target="_blank">12029041</a>]<br></label>
<input id="act_similarity_14", type="checkbox">
<div class="ACT_similar">
<li>Promoters controlling expression of the alternative nitrogenase and the molybdenum uptake system in Rhodobacter capsulatus are activated by NtrC , independent of s54 , and repressed by molybdenum .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9721306" target="_blank">9721306</a>]<br></li>
<li>Promoters controlling expression of the alternative nitrogenase and the molybdenum uptake system in Rhodobacter capsulatus are activated by NtrC , independent of 054 , and repressed by molybdenum .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10206709" target="_blank">10206709</a>]<br></li></div>
<li>Under N-excess conditions , 16 genes of the NtrC regulon showed decreased expression ( glnK , amtB , nac , nagA , yhhQ , cbl , glnG , b0296 , b1008 , b1011 -- 12 , b1036 , b1486 -- 87 , b1932 , b1973 ) and three genes increased expression ( osmY , csgD , csgG , see also the Supplementary Table containing the normalized gene expression ratios of all hybridization experiments ) as was observed when comparing the transcriptomes at 3.33 and 1.67 mM glucose feed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17273855" target="_blank">17273855</a>]<br></li>
<li>The structural basis for regulated assembly and function of the transcriptional-activator NtrC Sacha De Carlo ,1,7 Baoyu Chen ,3,7 Timothy R. Hoover ,5 Elena Kondrashkina ,6 Eva Nogales ,1,2,9 and B. Tracy Nixon4 ,8 1Howard Hughes Medical Institute , Department of Molecular and Cell Biology , University of California at Berkeley , Berkeley , California 94720 , USA ; 2Lawrence Berkeley National Laboratory , Berkeley , California 94720 , USA ; 3Integrative Biosciences Graduate Degree Program -- Chemical Biology and 4Department of Biochemistry and Molecular Biology , The Pennsylvania State University , University Park , Pennsylvania 16802 , USA ; 5Department of Microbiology , University of Georgia , Athens , Georgia 30602 , USA ; 6BioCAT at APS/Argonne National Lab , Illinois Institute of Technology , Argonne , Illinois 60439 , USA In two-component signal transduction , an input triggers phosphorylation of receiver domains that regulate the status of output modules .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16751184" target="_blank">16751184</a>]<br></li>
<li>Our structures of ADP - and ADP-AlFx bound NtrC ( S160F , 3-Ala ) provide data supporting that model , as well as clear evidence that the GAFTGA loop region forms a raised rim above the pore of the ring in the ADP-AlF state , but then relaxes in the ADP-bound state ( Figs .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16751184" target="_blank">16751184</a>]<br></li>
<li>Electron microscopy and image processing The oligomeric form of Mg2 + / BeF − 3 - activated NtrC ( S160F , 3-Ala ) was isolated by size-exclusion chromatography using a 2.4-mL Superdex 200 gel filtration column ( Amersham Pharmacia Biotech ) in a SMART system in the presence of either ADP or ADP-AlFx .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16751184" target="_blank">16751184</a>]<br></li>
<li>To demonstrate that the same cleavages occurred in populations containing heterodimers as in popu-Fe lations of NtrCD86C homodimers , we demonstrated that antibodies directed against the C terminus of NtrC detected bands 29 , 4 , and 5 ( Fig. 4B , lane 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10960104" target="_blank">10960104</a>]<br></li>
<li>FEMS Microbiology Letters 167 ( 1998 ) 51 ^ 56 Transcriptional repression of gdhA in Escherichia coli is mediated e Nac protein by th Laura Camarena * , Sebastian Poggio , Norma Garcè  a , Aurora Osorio Departamento de Biologè  a Molecular , Instituto de Investigaciones Biomeèdicas , Universidad Nacional Autoènoma de Meèxico , A.P. 70-228 , Meèxico 04510 D.F. , Mexico Received 3 March 1998 ; revised 28 July 1998 ; accepted 31 July 1998 Abstract In this work we show that the nac gene from Escherichia coli is transcriptionally active , and that its expression is dependent on NRI ( NtrC ) and sigma-54 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9785451" target="_blank">9785451</a>]<br></li>
<li>JOURNAL OF BACTERIOLOGY , Jan. 1997 , p. 41 -- / 97 / $ 04.0010 Copyright q 1997 , American Society for Microbiology Characterization of fhlA Mutations Resulting in Ligand - Independent Transcriptional Activation and ATP Hydrolysis INGRID KORSA AND AUGUST BÖCK * Lehrstuhl für Mikrobiologie der Universität München , D-80638 Munich , Germany Received 19 August 1996/Accepted 21 October 1996 The FhlA protein belongs to the NtrC family of transcriptional-regulators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8981978" target="_blank">8981978</a>]<br></li>
<li>ArgR-Independent Induction and ArgR-Dependent Superinduction of the astCADBE Operon in Escherichia coli Alexandros K. Kiupakis and Larry Reitzer * olecular and Cell Biology , The University of Texas at Dallas , Richardson , Texas 75083-0688 Received 7 November 2001/Accepted 8 March 2002 ent of M Departm For Escherichia coli , growth in the absence of ammonia is termed nitrogen limited and results in the induction of genes that assimilate other nitrogen sources , a response mediated by 54 and nitrogen regulator I ( NRI , also called NtrC ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12003934" target="_blank">12003934</a>]<br></li>
<li>JOURNAL OF BACTERIOLOGY , May 1995 , p. 2490 -- / 95 / $ 04.0010 Copyright q 1995 , American Society for Microbiology Active Contribution of Two Domains to Cooperative DNA Binding of the Enhancer-Binding Protein Nitrogen Regulator I ( NtrC ) of Escherichia coli : Stimulation by Phosphorylation and the Binding of ATP PO CHEN † AND LAWRENCE J. REITZER * Department of Molecular and Cell Biology , The University of Texas at Dallas , Richardson , Texas Received 27 October 1994/Accepted 31 January 1995 Activation by the prokaryotic activator nitrogen regulator I ( NRI , or NtrC ) of Escherichia coli requires an interaction between two NRI dimers .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7730282" target="_blank">7730282</a>]<br></li>
<li>Examples of transcriptional control in which the action of ArgR occurs in combination with ( an ) other regulatory protein ( s ) and co-factors have been documented previously for CRP , NtrC , Lrp , and IHF .22,23,29,50 Furthermore , artP mRNA levels were found to be induced in the stationary phase by EσS , the general stress-specific RNA polymerase ,51 upregulated ( 1.9-fold ) by the superoxide generating agent paraquat , an inducer of the soxRS system , and by sodium salicylate ( NaSal ) , an inducer of the marRA system ( 2.5-fold ) ,52 and down-regulated by Lrp ( about sixfold ) .53 Genome-wide expression profiling studies in E. coli indicate that argT and hisJ transcription are activated by NtrC ( GlnG ) ,45 and that in anaerobiosis hisJ expression is strongly increased in fnr ( 25-fold ) and arcA ( 60-fold ) mutants .54,55 None of the underlying molecular mechanisms has been unraveled yet , but the induction of E. coli artP in the stationary phase appears to depend on several secondary promoters ,51 upstream of PartP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17850814" target="_blank">17850814</a>]<br></li>
<li>This is the case for TyrR ,23 MalT ,24 BirA ,25 and NtrC .26 For eukaryotic nuclear receptors , including the vitamin D and retinoic acid receptors , regulatory ligands determine not only the oligomeric state of the receptor but als in ¯ uence the assembly of heterodimers of mixed receptor type , processes that have consequences for target site selection .27,28 In general , these effectorlinked assembly processes provide an indirect means of regulating DNA target-site occupancy , and ultimately transcriptional activity , via small changes in ligand concentration .25 One of the most interesting aspects of the Lrp regulon is the different regulatory patterns that relate to Lrp-leucine interactions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11575919" target="_blank">11575919</a>]<br></li>
<li>Residues that differ in the nuclear magnetic resonance ( NMR ) structures of unphosphorylated and phosphorylated NtrC ( 206 ) are mapped in green onto the corresponding surface of CheY ; residues that show signiﬁcant NMR chemical shift perturbations in phosphorylated CheY are mapped in blue ( 199 ) ; and overlapping residues appear as cyan .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10966457" target="_blank">10966457</a>]<br></li>
<li>Sedim en tation velo city d trC an com plexes compared with hydrod D20 , w 107 D 107 ( cm2 sÿ1 ) ( cm2 sÿ1 ) model ynamic m M a sed ( kDa ) odels ata of N d DNA s20 , w ( S ) Mcalc ( kDa ) smodel ( S ) DNA duplex ES-1 2.9 0.1 3.0 7.8 1.1 8.0 20 3 20 ( single NtrC binding site ) DNA duplex ES-2 3.7 5.3 35 6 37 3.9 0.1 5.9 0.9 ( two NtrC binding sites ) NtrC protein 5.2 0.2 5.3 4.7 0.7 4.6 103 16 108 6.3 0.2 NtrC protein , ES-1 DNA 6.2 4.7 0.7 9.5 1.3 c 4.1 112 17 408c 128 473b 14.4 b 2.7 b 12.6 0.4 NtrC protein , ES-1 DNA , CH2NO5PNa2 , MgCl2 , 6.5 0.3 d 6.4 ± 3.4 ± 145 NtrC protein , ES-2 DNA DNA duplex in excess NtrC protein , ES-2 DNA NtrC protein in excess 9.0 0.3 9.0 3.0 253 2.6 0.4 290 46 483c 5.8 0.9 c 14.4 0.4 15.0 2.5 507 , NtrC protein , ES-2 DNA CH2NO5PNa2 , MgCl2 , DNA duplex in excess 3.5 0.6 c 394c 14.4 470 2.7 12.1 0.4 , NtrC protein , ES-2 DNA CH2NO5PNa2 , MgCl2 , NtrC protein in excess The sedimentation and diffusion coef ® cients given in Table 2 were corrected to standard conditions ( 20 C , H2O ) according to equations ( 9 ) and ( 10 ) and are compared to the corresponding values calculated with the program HYDRO for the model complexes shown in Figure 8 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9600853" target="_blank">9600853</a>]<br></li>
<li>Molecular weight ( M ) of NtrC and DNA complexes determined by analytical ultracentrifugation M measured n20 C M calculated ( kDa ) ( ml gÿ1 ) ( kDa ) Association state DNA duplex ES-1 20 2 0.538 20 DNA duplex ( single NtrC binding site ) DNA duplex ES-2 37 2 37 DNA duplex 0.538 ( two NtrC binding sites ) NtrC protein 102 3 NtrC dimer 0.740 108 NtrC protein , ES-1 DNA 134 8 1 NtrC dimer 1 ES-1 duplex 0.709 128 Multiple speciesa 0.723 a 128-513a 417 56 NtrC protein , ES-1 DNA CH2NO5PNa2 , MgCl2 161 17b NtrC protein , ES-2 DNA DNA duplex in excess 145 0.689 1 NtrC dimer 1 ES-2 duplex NtrC protein , ES-2 DNA NtrC protein in excess 262 12 253 0.711 2 NtrC dimers 1 ES-2 duplex NtrC protein , ES-2 DNA , MgCl2 8 240 253 0.711 2 NtrC dimers 1 ES-2 duplex 2 NtrC dimers 1 ES-2 duplex NtrC protein , ES-2 DNA , CH2NO5PNa2 7 247 253 0.711 4 NtrC dimers 2 ES-2 duplexes NtrC protein , ES-2 DNA , CH2NO5PNa2 , MgCl2 , 500 25 507 0.711 DNA duplex in excess NtrC protein , ES-2 DNA CH2NO5PNa2 , MgCl2 , NtrC protein in excess , 4 NtrC dimers 1 ES-2 duplex 451 16 470 0.724 The data are averages of 5 to 20 measurements and were determined as described in Materials and Methods .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9600853" target="_blank">9600853</a>]<br></li>
<li>Size and shape parameters of NtrC and DNA complexes determined from sedimentation velocity analysis at 4 C d D 107 4 , buf ÿ ( cm2 s 1 ) H Oa 2 ( g/g ) b c f/fo s4 , buf ( S ) Vhyd ( cm3 ) Vmod ( cm3 ) 4.8 10ÿ20 5.1 10ÿ20 1.24 DNA duplex ES-1 1.5 0.1 4.0 0.92 ( single NtrC binding site ) 8.7 10ÿ20 1.0 10ÿ19 DNA duplex ES-2 4.1 0.89 1.45 2.0 0.1 ( two NtrC binding sites ) 2.0 10ÿ19 2.1 10ÿ19 2.7 0.1 NtrC protein 2.4 0.37 1.32 2.5 10ÿ19 2.6 10ÿ19 3.3 0.1 6.5 0.2 e NtrC protein , ES-1 DNA 2.4 0.45 1.34 8.9 10ÿ19 9.2 10ÿ19 4.9 0.42 1.53 2.8 10ÿ19 3.0 10ÿ19 3.4 0.2 f ± 0.50 1.57 NtrC protein , ES-2 DNA DNA duplex in excess 4.9 10ÿ19 5.1 10ÿ19 NtrC protein , ES-2 DNA NtrC protein in excess 4.7 0.2 1.3 0.45 1.47 9.8 10ÿ19 1.0 10ÿ18 , NtrC protein , ES-2 DNA CH2NO5PNa2 , MgCl2 , DNA duplex in excess 7.5 0.2 3.0 0.45 1.41 8.9 10ÿ19 9.2 10ÿ19 , 1.8 6.3 0.2 0.41 1.59 NtrC protein , ES-2 DNA CH2NO5PNa2 , MgCl2 , NtrC protein in excess The data are averages of two or three measurements .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9600853" target="_blank">9600853</a>]<br></li>
<li>On the other hand , nine species of RR ( AtoC , CheY , CusR , HydG , KdpE , NarL , NarP , NtrC , and YfhA ) were phosphoryl-ated by non-cognate HKs besides their cognate HKs .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15522865" target="_blank">15522865</a>]<br></li>
<li>Indeed , there was more than a 20-fold increase in crl transcript abundance under nitrogen starvation as observed by quantitative real-time PCR ( qRT-PCR ) ( Fig. 1A ) , a result confirmed by Northern blot hybridization ( Fig. 1C ) , and this increase was abolished by removal of NtrC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24474781" target="_blank">24474781</a>]<br></li>
<li>It has been shown that many EBPs are present at low concentrations in unstimulated cells and that environmental stimuli result , via positive autoregulation , in an increase in the amount of these activators , e.g. , NtrC ( 20 , 22 , 23 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9260970" target="_blank">9260970</a>]<br></li>
<li>Further , NtrC of E. coli , Klebsiella pneumoniae , and Bradyrhizobium parasponiae as well as NifR1 of Rhodobacter capsulatus cluster tightly together in both trees while the DctD Rle , DctD Rme , and PgtA Sty proteins are more distant .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7699720" target="_blank">7699720</a>]<br></li>
<li>Lanes 9 -- 12 represent the pre-incubation of 1 mM NtrC with increasing concentrations of PspA ( 0 , 0.5 , 1 and 2.5 mM light bars ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12079332" target="_blank">12079332</a>]<br></li>
<li>Lanes 13 -- 15 represent the pre-incubation of 1 mM NtrC with equivalent volumes of mock as a control for increasing concentrations of PspA ( 0.5 , 1 and 2.5 mM dark bars ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12079332" target="_blank">12079332</a>]<br></li>
<li>Typically , bacterial transcriptional-regulators use a helix-turn-helix ( HTH ) motif to-bind DNA and commonly form homodimers ( e.g. , cI , cro , trp repressor , CRP , NtrC , MarR , and FadR ) ( 2 , 22 , 36 , 40 , 42 , 43 , 49 , 60 , 64 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21685294" target="_blank">21685294</a>]<br></li>
<li>28 -- 33 How , for transcription factors such as MalT , NtrC or TyrR , differential spatial relative organizations of protein protomers and cognate sites are reconciled remains to be determined .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15491603" target="_blank">15491603</a>]<br></li>
<li>Phosphorylation site of NtrC , a protein phosphatase whose covalent intermediate activates transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8098993" target="_blank">8098993</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8226691" target="_blank">8226691</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10074086" target="_blank">10074086</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7730282" target="_blank">7730282</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9367763" target="_blank">9367763</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15755952" target="_blank">15755952</a>]<br></li>
<li>Arg.Lys.Leu.Ala.Lys.Arg.Leu.Gly.Val.Ser.HiB.Thr.Ala.Ile.Ala.Asn.Lys.Leu.Arg Cro NifA NtrC FhIA TyrR ng .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1943694" target="_blank">1943694</a>]<br></li>
<li>By 54 activating glnAp1 and downregulating glnAp2 , the overall effect of CRP -- cAMP on glnA expression is an approximately fourfold reduction , which correlates with the reduction of g-glutamyl transferase activities regulon , NtrB ( also called NRII ) and NtrC ( also called NRI ) ( Reitzer and Magasanik , 1986 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11532153" target="_blank">11532153</a>]<br></li>
<li>In this case , CRP -- cAMP might 54 compete with activators , such as NtrC , NifA , DctD , XylR , etc. for Es to downregulate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11532153" target="_blank">11532153</a>]<br></li>
<li>As many factors are involved ( CRP , cAMP , s , Es , NtrC , 54 54 NtrC-phosphate , NifA , etc. ) , it is difﬁcult to quantify their The precise mechanisms involved in inhibition by CRP -- cAMP at glnAp2 are not known .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11532153" target="_blank">11532153</a>]<br></li>
<li>Reactions which contained NtrA , NtrC , or both factors were tested in the presence or absence of cAMP and CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2468650" target="_blank">2468650</a>]<br></li>
<li>Similarly , when constitutively expressed NifA is used as the activator , CRP -- cAMP downregulates glnAp2 less than when NtrC is used as the activator ( 11-fold versus 21-fold , Fig. 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11532153" target="_blank">11532153</a>]<br></li>
<li>This might be as a result of the fact that the concentrations or the activities of the repressor CRP -- cAMP versus the activator NtrC or NifA are different under the conditions tested .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11532153" target="_blank">11532153</a>]<br></li>
<li>Seven major families , namely NtrC , MCMs , McrB , chelatases , YifB , MoxR , and dynein-midasin were identiﬁed within this lineage .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15037234" target="_blank">15037234</a>]<br></li>
<li>To identify region of NtrC that undergo a phosphorylation-dependent conformational change , protein cleavage experiments were performed with Fe-EDTA derivatives of mutant NtrC proteins ( MBP fusion versions ; 94 kDa ) carrying single cysteine res-Fe idues in positions described in the Introduction ( NtrCD86C , A89CFe V115CFe S160FFe NtrC , NtrC , and NtrC ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10960104" target="_blank">10960104</a>]<br></li>
<li>Moreover , changes in the chemical shifts of the active site residues D11 , D12 and D54 and of residues G36 , G62 and M81 were cumulative in N termini carrying one , two or three substitutions in helices 4 and 5 , paralleling the increasing activity of the intact NtrC proteins carrying these substitutions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9367763" target="_blank">9367763</a>]<br></li>
<li>Magnesium binding and structural changes in NtrCN-term carrying constitutive amino acid substitutions Mg2 is required for the in vitro transcription assay used to measure the activity of intact NtrC and is absolutely required for phosphorylation of D54 ( Stock et al. , 1993 ; A. North , personal communication ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9367763" target="_blank">9367763</a>]<br></li>
<li>First , there is a direct correlation between increasing activity of fulllength NtrC and larger structural changes in the 3445 face of NtrCN-term in a series of mutant forms carrying combinations of constitutive substitutions in NtrCN-term .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9367763" target="_blank">9367763</a>]<br></li>
<li>This site , located upstream of the rutA promoter , was previously proposed to be involved in RutR mediated repression of the rutABCDEFG operon that is transcribed from a r54 promoter and regulated by NtrC , the general regulator of nitrogen metabolism [ 1 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25685666" target="_blank">25685666</a>]<br></li>
<li>For example , the regulators in the subfamily consisting of NtrC and DctD are involved in cellular response to low external levels of nitrogen and act with RpoN ( a54 ) to activate their target genes ( 34 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2404948" target="_blank">2404948</a>]<br></li>
<li>The narX and narL genes , as mapped in this region with the correct orientation ( 9 ) , encode proteins with apparent molecular weights of 62,000-66 ,000 and OmpR is involved in osmotic regulation of the synthesis of outer membrane proteins , and NtrC is involved in nitrogen control of several nitrogen assimilatory genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2657652" target="_blank">2657652</a>]<br></li>
<li>Activator proteins in the NtrC family interact directly with E 54 to stimulate transcription from 54-dependent promoters ( 27 ) , which are generally involved in nitrogen regulation in E. coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15805526" target="_blank">15805526</a>]<br></li>
<li>NtrC family activators are generally involved in nitrogen regulation in E. coli , since nitrogen assimilation consumes energy and intermediates of central me-tabolism ( 27 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19633077" target="_blank">19633077</a>]<br></li>
<li>The results reveal that NtrC couples these two major bacterial stress responses to manage conditions of nitrogen limitation , and provide novel mechanistic insights into how a speciﬁc nutritional stress leads to elevating ppGpp levels in bacteria .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24947454" target="_blank">24947454</a>]<br></li>
<li>, Schneider , B.L. , Gu , W. , and Reitzer , L.J. ( 1992 ) Role of nitrogen regulator I ( NtrC ) , the transcriptional-activator of glnA in enteric bacteria , in reducing expression of glnA during nitrogen-limiting growth .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9720863" target="_blank">9720863</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12453212" target="_blank">12453212</a>]<br></li>
<li>NtrC , PhoB and CheB , the subjects of this study , belong to different subfamilies of response-regulators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7641688" target="_blank">7641688</a>]<br></li>
<li>Phosphorylation-mediated dimerization of the receiver domains of the NtrC and PhoB proteins has recently been reported and is proposed to result in their activation as transcriptional-regulators in vivo ( 16 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8892825" target="_blank">8892825</a>]<br></li>
<li>We suggest that NtrC and PhoB represent class of one response-regulators , where activation is mediated by dimerization of the phosphorylated receiver modules .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7641688" target="_blank">7641688</a>]<br></li>
<li>A common switch in activation of the response-regulators NtrC and PhoB : phosphorylation induces dimerization of the receiver modules During signal transduction , response-regulators of two-component systems are phosphorylated in a conserved receiver module .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7641688" target="_blank">7641688</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_64">
A common switch in activation of the response-regulators NtrC and PhoB : phosphorylation induces dimerization of the recei-ver modules .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9878422" target="_blank">9878422</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9600853" target="_blank">9600853</a>]<br></label>
<input id="act_similarity_64", type="checkbox">
<div class="ACT_similar">
<li>A common switch in activation of the response-regulators NtrC and PhoB : phosphorylation induced dimerization of receiver modules .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9973351" target="_blank">9973351</a>]<br></li>
<li>A common switch in activation of the response-regulators NtrC and PhoB : phosphorylation induces dimerization of the receiver modules .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659677" target="_blank">15659677</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8892825" target="_blank">8892825</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8981978" target="_blank">8981978</a>]<br></li></div>
<li>Activities of NtrC proteins carrying combinations of constitutive amino acid substitutions Since the activities of three of the MBP-NtrC proteins carrying single constitutive substitutions in the N-terminal domain could be increased by phosphorylation , the substitutions were combined to determine whether proteins with multiple substitutions would be more active .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9367763" target="_blank">9367763</a>]<br></li>
<li>F13202 lacks glnB , resulting in the constitutive phosphorylation of the introduced NtrC hybrid proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7641688" target="_blank">7641688</a>]<br></li>
<li>PII is not inactivated in a strain lacking UTase , leading to the constitutive dephosphorylation of the introduced NtrC hybrid proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7641688" target="_blank">7641688</a>]<br></li>
<li>In that case , a decrease in the mobility of the label observed upon phosphorylation of deri-D86C vatized NtrC also required phosphorylation of derivat-ized NtrCD86C , S160F .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10960104" target="_blank">10960104</a>]<br></li>
<li>However , no homologues of YcgR were found , although a loose similarity with the region around the phosphorylation site of Spo0A , Spo0F , and NtrC was detected .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11031114" target="_blank">11031114</a>]<br></li>
<li>Examples include NtrC , which is activated by phosphorylation of N-terminal Asp54 and NifA , which is negatively regulated by NifL through speciﬁc interactions with the NifA N-terminal domain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12079332" target="_blank">12079332</a>]<br></li>
<li>We infer that PhoB dimerizes upon phosphorylation , as does XN-NtrCRO lacking the C-terminal constitutive dimerization domain of NtrC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7641688" target="_blank">7641688</a>]<br></li>
<li>Note that this dimerization reaction of the monomeric XN-NtrCRO corresponds to the tetramerization reaction induced by phosphorylation of dimeric wild-type NtrC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7641688" target="_blank">7641688</a>]<br></li>
<li>Mechanism of activation of NtrC The dimerization induced by phosphorylation of kNNtrCRO corresponds to the tetramerization triggered by phosphorylation of wild-type NtrC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7641688" target="_blank">7641688</a>]<br></li>
<li>Some response-regulators oligomerize upon phosphorylation and then bind to DNA in their oligomerized forms ( e.g. , NtrC and UhpA ) ( 38 , 40 , 41 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14996819" target="_blank">14996819</a>]<br></li>
<li>It has been demonstrated that this is caused by the phosphorylation of NtrC by acetyl phosphate ( 23 , 24 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9892680" target="_blank">9892680</a>]<br></li>
<li>Only the latter conditions will lead to phosphorylation of NtrC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9600853" target="_blank">9600853</a>]<br></li>
<li>Phosphorylation-induced signal propagation in the response-regulator NtrC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16751184" target="_blank">16751184</a>]<br></li>
<li>Under nitro-gen-limited conditions , NtrB promotes the phosphorylation of the response-regulator NtrC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9892680" target="_blank">9892680</a>]<br></li>
<li>N limitation results in the phosphorylation of the response-regulator NtrC ( product of glnG ) by its cognate sensor kinase NtrB ( product of glnL ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24947454" target="_blank">24947454</a>]<br></li>
<li>In other two-component systems , it has been shown that acetyl phosphate phosphorylates the response-regulators CheY , NtrC , AlgR , NarQ , and NarX in the absence of the histidine kinase and ATP ( 21-24 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7568033" target="_blank">7568033</a>]<br></li>
<li>between EnyZ and OmpR were first shown to occur between the histidine kinase NRI , ( NtrB ) and NRI ( NtrC ) of the nitrogen regulation system ( 23 , 32 , 45 ) and between the histidine kinrase CheA and two effector-molecules , CheY and CheB , of the chemotaxis response ( 15 , 16 , 42 , 43 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2160945" target="_blank">2160945</a>]<br></li>
<li>By analogy to NtrC ( NRI , ginG ) and NtrB ( NRII , glnL ) in nitrogen control ( 31 ) , CheY and CheA in chemotaxis ( 24 ) , and OmpR and EnvZ in osmoregulation ( 2 , 19 , 26 ) , PhoR may be an autophosphorylase that also phosphorylates and dephosphorylates PhoB to convert it between active and inactive forms .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2160940" target="_blank">2160940</a>]<br></li>
<li>This hypothesis is based on the striking similarities that are shared at the protein sequence level for the regulators PhoB , NtrC , CheY , and OmpR and for their respective sensory proteins PhoR , NtrB , CheA , and EnvZ , which are now thought to act in this way ( 7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2160940" target="_blank">2160940</a>]<br></li>
<li>However , as Mlc shows no homology to any of the known effector components of two-component systems , e.g. OmpR , PhoB , NtrC , this would seem to exclude a classical two-component-type regulatory mechanism .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10411743" target="_blank">10411743</a>]<br></li>
<li>As a consequence of the homology with NtrC , there was also homology with other regulatory proteins which respond to other environmental stimuli such as phosphate limitation ( PhoB ) , altered osmolarity ( OmpR ) , presence of female cells ( SfrA ) , presence of plant exudate , .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2666400" target="_blank">2666400</a>]<br></li>
<li>Amino acid sequence comparisons showed that UhpA was homologous to a family of bacterial regulatory proteins , some of which act as transcriptional-activators ( OmpR , PhoB , NtrC , and DctD ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3042748" target="_blank">3042748</a>]<br></li>
<li>The N-terminal two-thirds of the NarL protein has extensive homology with those of a diverse set of prokaryotic regulatory proteins , including OmpR , PhoB , SfrA , UhpA , CheY , CheB , NtrC , DctD , FixJ , VirG , SpoOF , and SpoOA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2657652" target="_blank">2657652</a>]<br></li>
<li>Therefore , the most proteins of this family are subdivided into four different groups according to their C-terminal domain : ( i ) NarL , DegU , UvrC ORF2 , UhpA , and FixJ ; ( ii ) OmpR , PhoB , PhoM ORF2 , SfrA , and VirG ; ( iii ) NtrC , DctD , and PgtA ; and ( iv ) CheY and SpoOF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2657652" target="_blank">2657652</a>]<br></li>
<li>One family includes OmpR , PhoB , SfrA , VirG , CheB , CheY , Spo0A , SpoOF , NtrC , and DctD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3042748" target="_blank">3042748</a>]<br></li>
<li>The match to the other proteins , although obvious , was weaker , ranging from about 25 % identity with NtrC and CheY to 19 % for OmpR , SfrA , and PhoB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3042748" target="_blank">3042748</a>]<br></li>
<li>The matches of UhpA with DctD , NtrC , PhoB , and CheY were at least 12 standard deviations ( SDs ) above random .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3042748" target="_blank">3042748</a>]<br></li>
<li>Furthermore , comparison of the electrophoretic mobility of P-UhpA in native gels of various polyacrylamide concentrations , in comparison to the migration of protein standards , indicated that P-UhpA was monomeric in solution under those conditions .3 The rates of phosphorylation and dephosphorylation of UhpA were comparable to those seen for OmpR and PhoB ( 13 , 34 ) and were much slower than the corresponding processes in CheY and NtrC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8999880" target="_blank">8999880</a>]<br></li>
<li>The phosphorylation level was from 1 % to 10 % for TorR , PhoB , CusR , BasR , RstA , UvrY , ArcA , BaeR , NtrC , KdpE , UhpA , and PhoP ( in decreasing order ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15522865" target="_blank">15522865</a>]<br></li>
<li>In class I ( e.g. NtrC and PhoB ) , this interaction leads to the inhibition of the receiver module .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7641688" target="_blank">7641688</a>]<br></li>
<li>Members of this class are NtrC and PhoB , other candidates are ArcA , OmpR and KdpE .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7641688" target="_blank">7641688</a>]<br></li>
<li>This homology was equally spread over all functional domains on NtrC and GInG , as delineated by Drummond et al. ( 11 ) , although in the case of HydG , the interdomain region C was not particularly hydro-philic , as had been found to be the case for NtrC and related proteins such as OmpR and SfrA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2666400" target="_blank">2666400</a>]<br></li>
<li>GENES & DEVELOPMENT 20:1485 -- 1495 © 2006 by Cold Spring Harbor Laboratory Press ISSN 0890-9369/06 ; www.genesdev.org Results Our studies of the activated , oligomeric form of NtrC were facilitated by conditions that stably activate an NtrC variant that combined an S160F mutation , which favors an equilibrium toward the active , oligomeric state , and three alanine substitutions ( R456A , N457A , and R461A ) in the DNA-binding domain that prevent aggregation of the protein in the activated state ( Rombel et al. 1998 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16751184" target="_blank">16751184</a>]<br></li>
<li>Taken together , this indicates that DhaR like TyrR of E. coli and NtrC of Rhodobacter capsulatus ( Studholme and Dixon , 2003 ) and LevR of Lactobacillus casei ( Maze et al , 2004 ) activates an Es70 rather than a Es54 complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15616579" target="_blank">15616579</a>]<br></li>
<li>In this class , KdpD/KdpE , NarP and NtrB/NtrC are known to be involved in response to turgor pressure , nitrate and nitrogen availability respectively ( Laimins et al. , 1981 ; Darwin and Stewart , 1995 ; Zimmer et al. , 2000 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12366850" target="_blank">12366850</a>]<br></li>
<li>In nitrogen limitation , expression of the major transcript is dependent on the RNA polymerase sigma factor o - ^ together with phosphorylated NtrC ( Hirschman et ai , 1985 ; Hunt and Magasanik , 1985 ; Reitzer and Magasanik , 1985 ; Ninfa and Magasanik , 1986 ; Magasanik , 1989 ; Weiss etal .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8412694" target="_blank">8412694</a>]<br></li>
<li>For example , the arginine catabolic operon , astCADBE , not only responds to Ntr control ( via NtrC ) , but also to specific hyperinduction by arginine ( via ArgR ) , and activation by σS ( Schneider et al. , 1998 ; Kiupakis and Reitzer , 2002 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23531166" target="_blank">23531166</a>]<br></li>
<li>The role of these sites seems to be to increase the local concentration of the activator NtrC , as is characteristic of eukaryotic enhancers ( reviewed in Kustu et al. , 1991 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7545108" target="_blank">7545108</a>]<br></li>
<li>We propose therefore that a physiological role of CRP -- cAMP activation of glnAp1 is to partially compensate for CRP -- cAMP downregulation of Q 2001 Blackwell Science Lt For s - dependent promoters , upstream activator 54 sequences ( UAS ) are the speciﬁc target sites for transcriptional-activators belonging to the NtrC family ( NtrC , NifA , etc. , reviewed in Kustu et al. , 1989 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11532153" target="_blank">11532153</a>]<br></li>
<li>These results are consistent with previous studies showing that nitrogen-deﬁciency-induced glnA expression is controlled by s - dependent glnAp2 , while poor carbon-induced glnA 54 expression is controlled by CRP -- cAMP-activated , s - dependent glnAp1 ( MacNeil et al. , 1982b ; Reitzer 70 To explore the role of CRP -- cAMP and/or NtrC DNA-binding sites in the control of glnA , two derivatives of pKU100 were constructed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11532153" target="_blank">11532153</a>]<br></li>
<li>However , substitutions in the GAFTGA loop of NtrC caused binding to the enhancer or even nonspecific DNA to act as a negative allosteric regulator of the interaction with RNA polymerase ( Yan and Kustu 1999 ) , and prior studies of DctD , activated by deleting its N-terminal receiver domain , showed that ADP-AlFx , but not ADP or ADP-BeFx , caused hypersensitivity to DNase I to appear between the tandem binding sites in the dctA UAS ( Wang et al. 2003 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16751184" target="_blank">16751184</a>]<br></li>
<li>Additional , low-affinity binding sites on the promoter-proximal side of the glnA enhancer were seen in both in vitro ( Hirschman et al. 1985 ) and then in in vivo footprints of activated NtrC ( Sasse-Dwight and Gralla 1988 ) and reported to be necessary for high-level transcription activation ( Lilja et al. 2004 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16751184" target="_blank">16751184</a>]<br></li>
<li>CRP -- cAMP could interact with a-CTD in solution and inhibit binding of Es RNA polymerase to some target 54 promoters , or alter the conformation of promoter-bound Es RNA polymerase and so limit activation by NtrC 54 ( Wang et al. , 1998 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11532153" target="_blank">11532153</a>]<br></li>
<li>The amino acid sequence of PhoB protein is similar to those of several regulatory proteins , including OmpR , Dye , ORF2 ( of the phoM operon ) , NtrC , VirG , SpoOA and SPOOF , most of which are probably transcriptional-regulators for global genetic regulatory systems in bacteria ( Comeau et al. , 1985 ; Drury & Buxton , 1985 ; MacFarlane & Merrick , 1985 ; Amemura al. , 1986 ; IIrummond al. , 1986 ; et et Nixon et al. , 1986 ; Winans et al. , 1986 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3054125" target="_blank">3054125</a>]<br></li>
<li>By contrast to the case for NtrC , some other receiver domains ( e.g. that of DctD , which is also an activator of s54-holoenzyme , and those of FixJ , SpoOA and CheB ) apparently work as intramole-cular repressors , i.e. the removal of such domains activates the remainder of the protein ( Gu et al. , 1994 ; Huala et al. , 1992 ; Ireton et al. , 1993 ; Kahn & Ditta , 1991 ; Simms et al. , 1985 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9367763" target="_blank">9367763</a>]<br></li>
<li>One example is NtrC , where deletion of the receiver module results in a protein inactive as a transcriptional-activator ( Drummond al. , 1990 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7641688" target="_blank">7641688</a>]<br></li>
<li>Constitutive amino acid substitutions in NtrC act positively NtrCN-term is required for transcriptional activation by the remainder of the protein ( Drummond et al. , 1990 ; Flashner et al. , 1995 ; D. S. Weiss et al. , 1992 ) and hence constitutively activating amino acid substitutions in NtrCN-term must represent a gain of function .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9367763" target="_blank">9367763</a>]<br></li>
<li>This is due to the accumulation of acetyl phosphate in the cell , which serves as a phospho donor for NtrC ( Feng et al. , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7641688" target="_blank">7641688</a>]<br></li>
<li>The same phosphorylation of NtrC can be reproduced in vitro by the addition of a phosphorylating chemical like carbamyl phosphate ( CH2NO5PNa2 ) in the presence of MgCl2 ( Feng et al. , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9600853" target="_blank">9600853</a>]<br></li>
<li>Acetyl phosphate , in particular , makes significant contributions to in vivo NtrC phosphorylation levels , but other phosphodonors may also ( Feng et al. , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8098993" target="_blank">8098993</a>]<br></li>
<li>AcP acts as a phospho-donor for the autophosphorylation of the proteins CheY and OmpR ( Lukat et ai , 1992 ) , NtrC ( Feng et ai .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7934904" target="_blank">7934904</a>]<br></li>
<li>1990 ; Wanner and Wilmes-Riesenberg , 1992 ) , NtrC ( Feng etai .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7934904" target="_blank">7934904</a>]<br></li>
<li>Like NtrC and NifA , FhIA activity regulates promoters that are recognized by o ^ '' instead of a ™ ( Birkmann et at .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8412675" target="_blank">8412675</a>]<br></li>
<li>Modulation of its activity , unlike that of NtrC , depends on protein regulators : PspA ( negative ) or PspB and PspC ( positive ) ( Shingler , 1996 ; Elderkin et al. , 2002 ; Hankamer et al. , 2004 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17185540" target="_blank">17185540</a>]<br></li>
<li>As deduced from gel ® ltration experiments , unphosphorylated NtrC forms a dimer free in solution , and it has been concluded from gel electrophoretic analysis that a dimer is bound to a single NtrC binding site on the DNA ( Klose et al. , 1994 ; Porter et al. , 1993 ; Reitzer & Magasanik , 1983 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9600853" target="_blank">9600853</a>]<br></li>
<li>Association states of unphosphorylated NtrC In previous studies it was concluded from gel ® ltration and gel electrophoresis data that unpho-sphorylated NtrC forms a dimer and binds as a dimer to a single site ( Klose et al. , 1994 ; Porter et al. , 1993 ; Reitzer & Magasanik , 1983 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9600853" target="_blank">9600853</a>]<br></li>
<li>Discussion Constitutive substitutions in the receiver domain of NtrC create an active state Like forms of NtrC carrying the D54E substitution at the site of phosphoryation ( Klose et al. , 1993 ) , mutant forms carrying the D86N ( Flashner et al. , 1995 ) , A89T or V115I substitutions activate transcription in vitro without being phosphorylated , indicating that they are in an active state .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9367763" target="_blank">9367763</a>]<br></li>
<li>When puri ® ed and analyzed in vitro , the NtrC constitutive proteins activated transcription without being phosphorylated ( Flashner et al. , 1995 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9367763" target="_blank">9367763</a>]<br></li>
<li>When phosphorylated and bound at these sites , NtrC promotes formation of open transcription complexes between RNA polymerase and glnAp2 ( Popham et al. , 1989 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8098993" target="_blank">8098993</a>]<br></li>
<li>It has recently been demonstrated that NtrC is probably involved in the ammonium-promoted inhibition of the nitrate/nitrite transporter ( Kobayashi et al. 1997 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9873074" target="_blank">9873074</a>]<br></li>
<li>This result differs from prior suggestions that activated NtrC is an octamer ( Rippe et al. 1998 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16751184" target="_blank">16751184</a>]<br></li>
<li>MBP-NtrC fusions are known to be as active as the corresponding free NtrC proteins ( Klose et al. , 1994 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9367763" target="_blank">9367763</a>]<br></li>
<li>In vivo the NtrC protein is activated under conditions of nitrogen limitation ( Keener & Kustu , 1988 ; Ninfa & Magasanik , 1986 ; Weiss & Magasanik , 1988 ) as a consequence of phosphorylation of the aspartate residue at position 54 ( Sanders et al. , 1992 ) in its amino ( N ) - terminal receiver domain ( Drummond et al. , 1986 ; Keener & Kustu , 1988 ; Ninfa & Magasanik , 1986 ; Weiss & Magasanik , 1988 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9367763" target="_blank">9367763</a>]<br></li>
<li>In response to nitrogen limitation , the ATPase activity of NtrC is positively regulated by phosphorylation of its N-terminal domain by the NtrB kinase and stimulated by DNA binding ( Weiss et al. , 1991 , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17185540" target="_blank">17185540</a>]<br></li>
<li>These results have led to the idea that NtrC binds as an octameric complex at the enhancer upon phosphorylation of the protein ( Wyman et al. , 1997 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9600853" target="_blank">9600853</a>]<br></li>
<li>PhoB has in common with NtrC that phosphorylation enhances its DNA binding ability ( Makino et al. , 1989 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7641688" target="_blank">7641688</a>]<br></li>
<li>Unphosphorylated NtrC is dimeric and can bind to single target sites , but phosphorylation greatly enhances cooperative binding to tandem sites by augmenting interactions between NtrC dimers ( Weiss et al. , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8098993" target="_blank">8098993</a>]<br></li>
<li>Phosphorylation of the receiver module of NtrC induces the ATPase activity in NtrC , which is essential for activation of transcription ( Popham et al. , 1989 ; Weiss et al. , 1991 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7641688" target="_blank">7641688</a>]<br></li>
<li>In stark contrast , positively regulated activators like NtrC require the presence of the phosphorylated receiver domain to function in ATPase and transcription activation assays ( Drummond et al. 1990 ; Weiss et al. 1992a , b ; Klose et al. 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16751184" target="_blank">16751184</a>]<br></li>
<li>Furthermore , it has been shown in a number of studies that the unpho-sphorylated protein can not activate transcription , and deletion of the second NtrC binding site leads to a large reduction in the activation rate ( Austin & Dixon , 1992 ; Mettke et al. , 1995 ; Porter et al. , 1993 ; Weiss et al. , 1991 , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9600853" target="_blank">9600853</a>]<br></li>
<li>The phosphorylation state of NtrC and its ability to form larger complexes at the enhancer appear to be interconnected ( Porter et al. , 1993 ; Weiss et al. , 1992 ; Wyman et al. , 1997 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9600853" target="_blank">9600853</a>]<br></li>
<li>The transcrip tionally active form of NtrC is probably tetrameric ( Feng et al. , 1992 ; Weiss et al. , 1991 , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8098993" target="_blank">8098993</a>]<br></li>
<li>There is strong evidence ( Weiss et al. , 1992 ; Porter et al. , 1993 ) that activation of NtrC occurs in several steps .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7641688" target="_blank">7641688</a>]<br></li>
<li>Phosphorylation of D54 results in oligomeri-zation of NtrC ( Porter et al. , 1993 ; V. Weiss et al. , 1992 ; Wyman et al. , 1997 ) and activates its essential ATPase activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9367763" target="_blank">9367763</a>]<br></li>
<li>Implications of the model for the phosphorylated NtrC complex It is known that a single phosphorylated NtrC dimer is not able to activate transcription and that phosphorylation induces the ATPase activity of NtrC ( Austin & Dixon , 1992 ; Mettke et al. , 1995 Porter et al. , 1993 ; Weiss et al. , 1991 , 1992 ; Wyman et al. , 1997 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9600853" target="_blank">9600853</a>]<br></li>
<li>N-NtrC was as efficient in repression of XPR-lacZ expression as wild-type X repressor and repression was independent of the degree of phosphorylation ( Figure 3A and B ) , since NtrC is a dimer already in its unphosphorylated form ( Reitzer and Magasanik , 1983 ) , and a tetramer or higher oligomer in its phosphorylated form when bound to DNA ( Weiss et al. , 1992 ; North et al. , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7641688" target="_blank">7641688</a>]<br></li>
<li>In our case Kd is 4.6 10ÿ10 M for unphosphorylated NtrC ( Sevenich et al. , 1998 ) and 4 10ÿ12 M for the cooperative binding of phosphorylated NtrC with k12 230 ( Porter et al. , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9600853" target="_blank">9600853</a>]<br></li>
<li>There are several lines of evidence that , for wild-type NtrC , addition of DNA promotes the formation of the active NtrC-P tetramer ( Austin and Dixon , 1992 ; Weiss et al. , 1992 ; Porter et al. , 1993 ) due to the co-operative binding of phosphorylated NtrC to two adjacent binding sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7641688" target="_blank">7641688</a>]<br></li>
<li>Wild-type NtrC is phosphorylated in vivo by the NtrB protein ( nitrogen regulatory protein B , also designated nitrogen regulator II or NRII ) at Asp54 which greatly increases the cooperativity of DNA binding to two sites ( Chen & Reitzer , 1995 ; Porter et al. , 1993 ; Weiss et al. , 1992 ) and activates the ATPase function of NtrC ( Weiss et al. , 1991 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9600853" target="_blank">9600853</a>]<br></li>
<li>However , in the literature it has also been reported that binding of two unphosphorylated NtrC dimers to the enhancer occurs with a cooperativity constant of 20 to 100 , as determined by gel-shift experiments , nitrocellulose ® lter binding assays and quantitative DNA footprinting ( Chen & Reitzer , 1995 ; Porter et al. , 1993 ; Weiss et al. , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9600853" target="_blank">9600853</a>]<br></li>
<li>Native or a DNA-binding mutant form of NtrC and the isolated central domains of NifA and DctD , when overproduced , also have the ability to activate transcription ( Dixon , 1984 ; Huala & Ausubel , 1989 ; Huala et al. , 1992 ; Morett & Segovia , 1993 ; Reitzer & Magasanik , 1986 ; North & Kustu , 1997 ) , indicating that these forms of the activators can interact directly from solution with s54-RNAP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9878422" target="_blank">9878422</a>]<br></li>
<li>Previous studies have shown that NtrC , a widely studied member of this family of activators , binds to its ELE and activates transcription through the assembly of large oligomers in which some of the dimers are not in direct contact with the DNA ( Wyman et al. , 1997 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15528672" target="_blank">15528672</a>]<br></li>
<li>Nonetheless , the portion of UhpB which appears to be involved in the binding of UhpA overlaps the homologous region of a VanSderived peptide which binds VanR ( 46 ) , and portions of NtrB and EnvZ which are necessary for phosphatase activity on NtrC and OmpR , respectively ( 19 , 23 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11325944" target="_blank">11325944</a>]<br></li>
<li>However , the V shaped model for the NtrC dimer conformation presented here has the additional advantage that it can be easily extended to a model for the octameric association states adopted by the phosphorylated protein as shown in Figure 8c and d. Association states of phosphorylated NtrC In vivo phosphorylation is brought about by the histidine kinase NtrB ( see Fiedler & Weiss , 1995 , and references therein ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9600853" target="_blank">9600853</a>]<br></li>
<li>Biochemical characterization of the system that regulates glutamine synthetase expression has shown that gene expression is stimulated in vitro by a cascade involving phosphorylation of the NtrC protein by the NtrB kinase ( Ninfa and Magasanik , 1986 ; Ninfa et al. , 1987 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3280143" target="_blank">3280143</a>]<br></li>
<li>PII regulates the ratio of phosphatase ( P ) to autokinase ( K ) activities in NtrB to adjust the phosphorylation state of NtrC , which in turn controls glutamine synthetase expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8098993" target="_blank">8098993</a>]<br></li>
<li>In studies of the degree of phosphorylation of NtrC by NtrB and ATP it was found that the maximal degree of phosphorylation of NtrC is only 35 % at 37 C due to the autopho-sphatase activity of NtrC ( Weiss & Magasanik , 1988 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9600853" target="_blank">9600853</a>]<br></li>
<li>NtrB controls the phosphorylation state of NtrC through reciprocal kinase and phosphatase activities ( Keener and Kustu , 1988 ; Ninfa and Magasanik , 1986 ; Weiss and Magasanik , 1988 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8098993" target="_blank">8098993</a>]<br></li>
<li>Phosphorylation of NtrC is catalysed by NtrB , which has also an NtrC-phosphatase activity ( Ninfa and Magasanik , 1986 ; Keener and Kustu , 1988 ; Weiss and Magasanik , 1988 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8412694" target="_blank">8412694</a>]<br></li>
<li>NtrB in turn senses the cell 's PII modification state Unmodified PII interacts with NtrB , augmenting its phosphatase activity toward NtrC ( Ninfa and Magasanik , 1986 ; Keener and Kustu , 1988 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8098993" target="_blank">8098993</a>]<br></li>
<li>, 1988 ) , and from EnvZ to NtrC ( NR , ) ( Ninfa , Igo , and Silhavy , cited in Magasanik , 1988 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2201868" target="_blank">2201868</a>]<br></li>
<li>Transcriptional activation by NtrC requires phosphorylation of its receiver module ( Ninfa and Magasanik , 1986 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8098993" target="_blank">8098993</a>]<br></li>
<li>Due to the differences in methodology between transcription assays described11 and in this study ( Figure 2 ) the inhibi-tory effects of PspA on sigma isomerisation cata-lysed by NifA central domain and phosphorylated NtrC were analysed to further conﬁrm that negative effects of PspA are speciﬁc to PspF , and do not play a role in global regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12079332" target="_blank">12079332</a>]<br></li>
<li>PspA inhibits PspF-dependent transcription but has no effect on NtrC or NifA Previous analysis of the activity of PspA in vitro demonstrated that PspA inhibits PspF-depen-dent transcription from the s54-dependent pspA promoter , with almost complete inhibition at a ratio of 4 nM PspFDHTH to 300 nM PspA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12079332" target="_blank">12079332</a>]<br></li>
<li>Thus , under conditions employed here , PspA speciﬁcally inhibited transcription activation mediated by PspFDHTH and had no detect-able effect on the NtrC and NifA activators tested .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12079332" target="_blank">12079332</a>]<br></li>
<li>The possibility was raised by the fact that PspA functions in trans upon the AAAþ domain of PspF common to all s54 activators and the reported inﬂuence of PspA upon activation by NtrC .11 We , on the other hand , were unable to show any negative regulation of the AAAþ central domain of NifA nor NtrC by PspA ( Figures 2 ( c ) and 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12079332" target="_blank">12079332</a>]<br></li>
<li>The best-characterized examples of this are at the glnHp2 promoter where activation by NtrC requires IHF ( 4 ) and IHF-dependent NifA activation of the nifH promoter ( 16 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8376324" target="_blank">8376324</a>]<br></li>
<li>These results are in contrast to the case of NtrC , in which the central catalytic domain by itself is not active ( 39 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8606168" target="_blank">8606168</a>]<br></li>
<li>This reaction is dependent on ATP hydrolysis , most likely catalyzed by the central domain of NtrC ( Weisset al. , 1991 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8098993" target="_blank">8098993</a>]<br></li>
<li>Upon phosphorylation , NtrC forms oligomers ( likely to be octamers ) that can hydrolyze ATP and couple the energy available from ATP hydrolysis to the formation of open complexes by s54 holoenzyme ( 1 , 31 , 36 , 42 , 43 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10960104" target="_blank">10960104</a>]<br></li>
<li>This process involves phosphorylation of NtrC , assembly of a multimeric NtrC complex at the enhancer DNA sequence , interaction of this complex with promoter bound RNAP s54 via DNA looping , and hydrolysis of ATP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9600853" target="_blank">9600853</a>]<br></li>
<li>In the case of NtrC , ATP hydrolysis is required during the process of activation to convert the or '' holoenzyme DNA complex from a closed to an open configuration ( 6 , 31 , 57 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8449883" target="_blank">8449883</a>]<br></li>
<li>The NMR structure of NtrC shows a displacement of β4 , β5 , αC , and αD away from the active site , thereby exposing a hydrophobic surface that is proposed to allow transmission of the activation signal to the effector domain ( 206 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10966457" target="_blank">10966457</a>]<br></li>
<li>PrpR of S. enterica belongs to the NtrC family of transcriptional-activators and has been known to regulate the expression of the prpBCDE operon in response to propionate catabolism ( 32 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15805526" target="_blank">15805526</a>]<br></li>
<li>Experiments are in pro-gress to determine the af ® nity of the different NtrC association states to the polymerase complex , in order to provide further details on the activation mechanism by which NtrC operates .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9600853" target="_blank">9600853</a>]<br></li>
<li>Perhaps an excess of the unmodified form can still bind to DNA , as is the case with NtrC ( 17 ) , and interfere with activation by the correctly modified form .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3042748" target="_blank">3042748</a>]<br></li>
<li>It should be recalled that physical and biochemical evidence for DNA looping already exists in the literature , especially in the case of the araBAD operon , and for activation by NtrC of the ginA promoter of E. coli ( reviewed in reference 16 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7928991" target="_blank">7928991</a>]<br></li>
<li>Phosphorylated NtrC is active as a transcriptional-activator for the Ntr regulon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9892680" target="_blank">9892680</a>]<br></li>
<li>Spohn , G. and Scarlato , V. ( 1999 ) Motility of Helicobacter pylori is coordinately regulated by the transcriptional-activator FlgR , an NtrC homolog .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19969540" target="_blank">19969540</a>]<br></li>
<li>Effect of overproduced HycA on the transcriptional activation Considering the fact that residues 131 and 132 of NtrC were assigned to the linker region between domain A and C [ 46 ] we chose to separate FhlA between residue 378 and 379 for further investigations .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10848985" target="_blank">10848985</a>]<br></li>
<li>However , a speci ® c biological function of this association state seems unlikely since all known biological activities of NtrC like transcriptional activation , ATPase activity and DNA binding can be reproduced in vitro at nanomolar concentrations .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9600853" target="_blank">9600853</a>]<br></li>
<li>Therefore , it seems improbable that CRP -- cAMP speciﬁcally interferes with NtrC-mediated transcriptional activation , for example by somehow interfering with NtrC0s activity , e.g. by disrupting the histidyl -- aspartyl phosphorelay between NtrB and NtrC proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11532153" target="_blank">11532153</a>]<br></li>
<li>Higher mRNA levels in the first strain in experiments a -- f are indicative of Nac activation , whereas higher levels in the first strain in experiments c -- h are indicative of NtrC activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11121068" target="_blank">11121068</a>]<br></li>
<li>Interestingly , ﬁve of the cysteine substitutions appear to result in loss of transcriptional activation by NtrC in vivo ( J. Lee , unpublished results ) , commensurate with the postulated role of this helix in signal propagation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10960104" target="_blank">10960104</a>]<br></li>
<li>Through a complex cascade of events , glutamine starvation leads to phosphorylation ( and activation ) of the transcriptional-regulator NtrC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9495755" target="_blank">9495755</a>]<br></li>
<li>In NtrC it is induced by phosphorylation of the N-terminal domain and required for ATPase activity that is coupled to transcriptional activation ( 46 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11923293" target="_blank">11923293</a>]<br></li>
<li>Genetic evidence for activation of the positive transcriptional-regulator XylR , a member of the NtrC family of regulators , by effector binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8981978" target="_blank">8981978</a>]<br></li>
<li>[ 22 ] L.S. Jenkins , W.D. Nunn , Regulation of the ato operon by the atoC gene [ 23 ] S.C. Porter , A.K. North , S. Kustu , Mechanism of transcriptional activation by NtrC , in : J.A. Hoch , T.J. Silhavy ( Eds .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16153782" target="_blank">16153782</a>]<br></li>
<li>Porter , S.C. , North , A.K. , and Kustu , S. ( 1995 ) Mechanism of transcriptional activation by NtrC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12366850" target="_blank">12366850</a>]<br></li>
<li>Mechan-ism of transcriptional activation by NtrC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9878422" target="_blank">9878422</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_187">
Mechanism of transcriptional activation by NtrC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16751184" target="_blank">16751184</a>]<br></label>
<input id="act_similarity_187", type="checkbox">
<div class="ACT_similar">
<li>Mechanism of transcriptional activation by NtrC , p. 147 -- 158 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10960104" target="_blank">10960104</a>]<br></li></div>
<li>The special case of NR ( I ) ( NtrC ) is also included in the table as it represses its own synthesis from a sigma 70 promoter but activates it from an overlapping sigma 54 promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1943993" target="_blank">1943993</a>]<br></li>
<li>This N is released as ammonium by cytidine deaminase ( cdd , b2143 ) , which is not known to be under NtrC control , to yield uridine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16540542" target="_blank">16540542</a>]<br></li>
<li>As expected for an operon under NtrC control , levels of b1012 mRNA were very low in both strains on NH4Cl as N source .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16540542" target="_blank">16540542</a>]<br></li>
<li>The hypothesis of back-to-front dimers as intermediates in the activation process is consistent with the multiple species of intermediate sedimentation velocities that we observed for partially activated NtrC ( S160F , 3-Ala ) ( Supplementary Fig .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16751184" target="_blank">16751184</a>]<br></li>
<li>Guinier plots for these data were linear , demonstrating the absence of aggregates ( e.g. , the plot for the peak of NtrC [ S160F , 3-Ala ] is shown in Supplementary Fig .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16751184" target="_blank">16751184</a>]<br></li>
<li>Instead , like OmpR , NtrC can acquire phosphoryl groups from several sources .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8098993" target="_blank">8098993</a>]<br></li>
<li>Four of these are activated by the NR ( I ) ( NtrC ) activator and three are activated by the FhlB activator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1943993" target="_blank">1943993</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1943993" target="_blank">1943993</a>]<br></li>
<li>We collected SAXS/WAXS data for the activated , oligo-meric NtrC bound to ADP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16751184" target="_blank">16751184</a>]<br></li>
<li>Therefore a new full data set was obtained for activated NtrC ( S160F , 3-Ala ) in the presence of ADP-AlFx .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16751184" target="_blank">16751184</a>]<br></li>
<li>EM model for activated NtrC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16751184" target="_blank">16751184</a>]<br></li>
<li>Full-length model of NtrC for the ADP and ADP-AlFx states .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16751184" target="_blank">16751184</a>]<br></li>
<li>B , Uuftiple aJtgnment of OrfXB and NtrC rxjnxjiogies .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8412694" target="_blank">8412694</a>]<br></li>
<li>NtrC dimers are drawn in red and in yellow .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9600853" target="_blank">9600853</a>]<br></li>
<li>To identify the genomic loci bound by NtrC in E. coli during N starvation , we sampled cells for ChIP-seq analysis at t 1/4 N and NtrC-bound DNA was precipitated with anti-FLAG antibodies .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24947454" target="_blank">24947454</a>]<br></li>
<li>NtrC binds to DNA , and at least for NtrC and NifA , it has been demonstrated that they perform their function in a coordinate action with a specific sigma factor ( NtrA ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2666400" target="_blank">2666400</a>]<br></li>
<li>Structural a Functional Analyses of Activating Amino Acid Substitutions in the Receiver om NtrC : Eviden o n Activating Surface nd D ain of ce f r a Michael Nohaile1 , Dorothee Kern1 , David Wemmer1 * , Kenneth Stedman2 and Sydney Kustu2 1 Department of Chemistry University of California Berkeley & Structural Biology Division , Lawrence Berkeley National Laboratory , Berkeley CA 94720 , USA 2 Department of Molecular and Cell Biology , University of California , Berkeley , CA 94720 USA # 1997 Academic Press Limited Keywords : two-component systems ; constitutive mutant forms of NtrC ; phosphorylation ; signal transduction ; NMR chemical shift analysis * Corresponding author Introduction The bacterial enhancer-binding protein NtrC activates transcription when phosphorylated on aspartate 54 in its amino ( N ) - terminal regulatory domain or when altered by constitutively activating amino acid substitutions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9367763" target="_blank">9367763</a>]<br></li>
<li>osphorylation-Induced Signal Propagation in the Regulator NtrC Ph Response JONGHUI LEE ,1 JEFFREY T. OWENS ,2 † INGYU HWANG ,3 ‡ CLAUDE MEARES ,2 AND SYDNEY KUSTU3 * Department of Molecular and Cell Biology , University of California , Berkeley , California 94720-32041 ; Department of Chemistry , University of California , Davis , California 956162 ; and Department of Plant and Microbial Biology , University of California , Berkeley , California 94720-31023 Received 28 December 1999/Accepted 19 June 2000 The bacterial enhancer-binding protein NtrC is a well-studied response-regulator in a two-component regulatory system .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10960104" target="_blank">10960104</a>]<br></li>
<li>Abbreviations used : HMQC , heteronuclear multiple quantum coherence ; HSQC , heteronuclear single quantum coherence ; MBP , maltose binding protein ; NMR , nuclear magnetic resonance ; NOESY , nuclear Overhauser and exchange spectroscopy ; NtrC , nitrogen regulatory protein C ; NtrCN-term , N-terminal domain of NtrC ; PCR , polymerase chain reaction ; TOCSY , total correlated spectroscopy .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9367763" target="_blank">9367763</a>]<br></li>
<li>The NtrC protein concentration was determined by measuring the UV absorbance at 280 nm in the presence of 6 M guanidine hydrochloride , using an extinction coefﬁcient of 1.11 3 105 M21 cm21 for the MBP-NtrC fusion protein ( 7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10960104" target="_blank">10960104</a>]<br></li>
<li>Therefore , if complexes C2 and C3 represent 4 and 6 NanR monomers bound , respectively ( Fig. 6 ) , some oligomerization must occur through solely protein-protein interactions with bound NanR , as observed for NtrC ( 32 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23935044" target="_blank">23935044</a>]<br></li>
<li>Increasing the concentration of PspA still further may have resulted in some inhibition of either NtrC or NifA-dependent sigma isomerisation but was not experimentally possible due to accessible protein concentrations .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12079332" target="_blank">12079332</a>]<br></li>
<li>Expression of the putA gene encoding proline dehydrogenase from Rhodobacter capsulatus is independent of NtrC regulation but requires an Lrp-like activator protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18586269" target="_blank">18586269</a>]<br></li>
<li>Some of these proteins , i.e. NtrC , DctD and HydG , are activated by phosphorytation carried out by specific histidine protein kinases ( see Stock et ai , 1989 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1943694" target="_blank">1943694</a>]<br></li>
<li>On the basis of sequence homology with other transcriptional-activators such as NtrC , HydG , and KlebsieUa pneumoniae NifA proteins , the FhlA protein was deduced to be a transcriptional-activator controlling the production of FHL .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2118503" target="_blank">2118503</a>]<br></li>
<li>In this regard , the FhlA protein is similar to the NifA protein of K. pneumoniae and is distinct from the NtrC protein ' .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2118503" target="_blank">2118503</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_214">
Atkinson , M.R. , Kamberov , E.S. , Weiss , R.L. and Ninfa , A.J. ( 1994 ) Reversible uridylylation of the Escherichia coli PII signal transduction protein regulates its ability to stimulate the dephosphorylation of the transcription factor nitrogen regulator I ( NRI or NtrC ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7641688" target="_blank">7641688</a>]<br></label>
<input id="act_similarity_214", type="checkbox">
<div class="ACT_similar">
<li>Atkinson , M.R. , Kamberov , E.S. , Weiss , R.L. , and Ninfa , A.J. ( 1994 ) Reversible uridylylation of the Escherichia coli PII signal transduction protein regulates its ability to stimulate the dephosphorylation of the transcription factor Nitrogen Regulator I ( NRI or NtrC ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9720863" target="_blank">9720863</a>]<br></li></div>
<li>Transcription of most is apparently activated by NtrC in conjunction with s54-holoenzyme , but transcription of some ( ' 9 operons ; 25 genes ) appears to depend on the adapter Nac in conjunction with s70-holoenzyme .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11121068" target="_blank">11121068</a>]<br></li>
<li>NtrC is activated by phoshorylation of the aspartate 54 residue located in the N-terminal receiver domain causing oligomerisation , and DctD by its phosphorylated receiver domain relieving inhibition imposed by the non-phosphorylated recei-ver domain upon the central AAAþ domain ( Figure 1 ) .24 Some activators of s54-dependent transcription are regulated by the binding of a small ligand to the N-terminal domain , such as binding xylene to XylR .24 In contrast , other s54 activators are regulated through their N-terminal domains being targeted by a partner second protein which imposes negative control .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12079332" target="_blank">12079332</a>]<br></li>
<li>In contrast , the glnAp2 promoter is s - dependent 54 and its transcription can be activated either by its cognate activator NtrC under nitrogen-deﬁciency conditions ( Fig. 1 ) , or by the alternative activator , NifA ( Figs 4 and 7B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11532153" target="_blank">11532153</a>]<br></li>
<li>Schulz A , Langowski J & Rippe K ( 2000 ) The effect of the DNA conformation on the rate of NtrC activated transcription of Escherichia coli RNA polymerase.sigma ( 54 ) holoenzyme .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16553838" target="_blank">16553838</a>]<br></li>
<li>ATPγS causes a mutant NtrC that activates transcription in the absence of phosphorylation to oligomerize to a hexamer or octamer in the absence of DNA ( 48 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11923293" target="_blank">11923293</a>]<br></li>
<li>Geometric and dynamic requirements for DNA looping , wrapping and unwrapping in the activation of E. coli glnAp2 transcription by NtrC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16751184" target="_blank">16751184</a>]<br></li>
<li>NtrC activates relA transcription in N-starved E. coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24947454" target="_blank">24947454</a>]<br></li>
<li>A mutation in ntrC which changes glycine 173 to asparagine results in a mutant NtrC protein that can repress but not activate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8449883" target="_blank">8449883</a>]<br></li>
<li>Such substitutions would be unlikely to activate NtrC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9367763" target="_blank">9367763</a>]<br></li>
<li>Conjugation was around 50 % , and derivatized NtrC proteins retained some ability to activate transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10960104" target="_blank">10960104</a>]<br></li>
<li>Deutsches Krebsforschungszentrum Abteilung Biophysik der Makromoleku È le , Im Neuenheimer Feld 280 D-69120 , Heidelberg , Germ any # 1998 Academic Press Limited Keywords : enhancer ; protein-DNA interaction ; transcription ; RNA polymerase * Corresponding author Abbreviations used : NtrC , nitrogen regulatory protein C ; NtrC-P , phosphorylated NtrC ; RNAP s54 , E. coli RNA polymerase s54 holoenzyme ; NtrB , nitrogen regulatory protein B ; AUC , analytical ultracentrifugation ; M , molecular weight ; s , sedimentation coef ® cient ; D , diffusion constant ; BSA , bovine serum albumin .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9600853" target="_blank">9600853</a>]<br></li>
<li>Thus , bands 29 and 4 appear to be C-terminal cleavage fragments from unphosphorylated NtrC ( also seen when the protein is phosphorylated ) , whereas band 5 is a C-terminal fragment obtained only when the protein is phosphorylated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10960104" target="_blank">10960104</a>]<br></li>
<li>Nitrogen-regulated Ntr promoters of Escherichia coli are activated by the phosphorylated form of the glnG ( ntrC ) protein NRI ~ P ( NtrC ~ P ) ( Ninfa and Magasanik , 1986 ) , which binds to enhancer sequences found upstream from the regulated promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12453212" target="_blank">12453212</a>]<br></li>
<li>NtrC protein , a positive regulator for genes involved in nitrogen metabolism , was shown to be phosphorylated by NtrB protein , and the phosphorylated form was the active form for transcriptional activation of the genes involved in nitrogen fixation ( Ninfa & Magasanik , 1986 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3054125" target="_blank">3054125</a>]<br></li>
<li>Prominent examples are the bacterial nitrogen uptake regulating transcription factor , NtrC , which is phosphoryiated in its active form by NtrB ( Austin and Dixon , 1992 ; Sanders etai , 1992 ) , or the catabolite regulatory protein , CRP , which is activated by oAMP ( Botsford and Harman , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7512187" target="_blank">7512187</a>]<br></li>
<li>This suggests that kN-NtrC is phosphorylated in vivo by NtrB , is able to tetramerize or oligomerize to the active ATPase and can interact with the o54 RNA polymerase holo-enzyme to activate transcription at the glnAp2 promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7641688" target="_blank">7641688</a>]<br></li>
<li>For example , transcription from the glnAp2 is activated when the NtrC protein is promoter phosphorylated by the action of the NtrB protein ( 17 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3042748" target="_blank">3042748</a>]<br></li>
<li>It appears questionable whether the 19 S NtrC complex is of biological relevance , because the required protein concentrations are well above those found in the bacterial cell which are estimated to be 10 to up to 100 nM under nitrogen limiting conditions ( Reitzer & Magasanik , 1983 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9600853" target="_blank">9600853</a>]<br></li>
<li>Also , induction of NtrC activity by nitrogen starvation was tested .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17185540" target="_blank">17185540</a>]<br></li>
<li>This nitrogen regulation is not mediated by the known regulators that respond to nitrogen limitation , Nac or NtrC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12460564" target="_blank">12460564</a>]<br></li>
<li>glnG encodes NtrC , the transcriptional-activator of nitrogen assimilation and scavenging genes under conditions of nitro-gen-limited growth ( 41 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15629914" target="_blank">15629914</a>]<br></li>
<li>3 Simulation of Conductance by Amt B Regulation of Ammonia Transport In most bacteria , Amt genes are induced by nitrogen limitation , which is controlled by an activator , NtrC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17119155" target="_blank">17119155</a>]<br></li>
<li>In contrast , the arginine-ornithine-lysine binding protein which interacts with the same ABC transporter is induced under nitrogenlimiting conditions and controlled by NtrC ( 61 ) , suggesting that the LAO system serves as a scavenging system for nitro-gen-rich amino acids under conditions of nitrogen starvation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21441513" target="_blank">21441513</a>]<br></li>
<li>For example , the sensor NtrB responds to nitrogen limitation and transmits that information to the regulator NtrC , which then activates transcription of nitrogen-regulated operons such as ginA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2649492" target="_blank">2649492</a>]<br></li>
<li>This second protein is neither RcsB itself nor NtrC , the protein that interacts with RpoN to stimulate synthesis of some genes during nitrogen deprivation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1943696" target="_blank">1943696</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_241">
Interaction of purified NtrC protein with nitrogen regulated promoters from Klebsiella pneumoniae .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3025172" target="_blank">3025172</a>]<br></label>
<input id="act_similarity_241", type="checkbox">
<div class="ACT_similar">
<li>Interaction of purified NtrC protein with nitrogen regulated promoter from Klebsiella pneumoniae .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2448295" target="_blank">2448295</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2871943" target="_blank">2871943</a>]<br></li></div>
<li>In another procaryotic regulatory system which senses the availability of nitrogen in the extracellular environment , phosphorylation of an aspartic acid residue ( 34 ) of the transcription activator protein NtrC by the sensory protein NtrB , which are structurally similar to OmpR and EnvZ , respectively ( 26 ) , activates transcription of the ginA gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2656631" target="_blank">2656631</a>]<br></li>
<li>Glutamate at the site of phosphorylation of nitrogen-regulatory protein NtrC mimics aspartyl-phos-phate and activates the protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9422609" target="_blank">9422609</a>]<br></li>
<li>The NtrC protein also activates expression of the nitrogen assimilatory control gene ( nac ) in Klebsiella aerogenes , which is known to regulate expression of glutamate synthase ( 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955321" target="_blank">8955321</a>]<br></li>
<li>In all , ' 2 % of the E. coli genome appears to be under NtrC control , although transcription of some operons depends on the nitrogen assimilation control protein , which serves as an adapter between NtrC and s70-dependent promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11121068" target="_blank">11121068</a>]<br></li>
<li>The nitrogen assimilation control ( Nac ) protein activates transcription of s70-dependent genes , whereas NtrC activates transcription of s54-dependent genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11121068" target="_blank">11121068</a>]<br></li>
<li>against nitrogen limitation Nitrogen regulatory protein C ( NtrC ) of enteric bacteria activates transcription of genesyoperons whose products minimize the slowing of growth under nitrogen-limiting conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11121068" target="_blank">11121068</a>]<br></li>
<li>One consequence is activation of transcription of genes under control of nitrogen regulatory protein C ( NtrC ) , the products of which ameliorate slowing of growth .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11121068" target="_blank">11121068</a>]<br></li>
<li>The autokinase is more active in glnL ( Up ) [ NtrB ( Con ) ] strains than in wild-type strains , and hence genes and operons under NtrC ( GlnG ) control are overexpressed even under conditions of nitrogen excess .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11121068" target="_blank">11121068</a>]<br></li>
<li>The nitrogen regulation stress response is an adaptive mechanism used by nitrogen-starved Escherichia coli to scavenge for alternative nitrogen sources and requires the global transcriptional-regulator NtrC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24947454" target="_blank">24947454</a>]<br></li>
<li>A Nitroregulation Circuit in E. coli The NtrB-NtrC pathway regulates glutamine synthetase expression in response to changes in nitrogen availability .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8098993" target="_blank">8098993</a>]<br></li>
<li>The GlnL ( NtrB ) protein is a histidine autokinase that donates phosphoryl groups to response-regulator GlnG ( NtrC ) and thereby activates it ( 18 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11121068" target="_blank">11121068</a>]<br></li>
<li>The phosphorylation state of the ` receiver ' protein of the two-component system ( NRI or NtrC ) determines its ability to activate transcription and is controlled by interaction with the transmitter protein ( NRII or NtrB ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12123449" target="_blank">12123449</a>]<br></li>
<li>The NRII ( NtrB ) - NRI ( NtrC ) two-component system regulates the transcription of nitrogen-regulated ( ntr ) genes in response to signals of nitrogen and carbon sufﬁciency ( reviewed in references 16 , 23 , 24 , 27 , 33 , 34 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10074086" target="_blank">10074086</a>]<br></li>
<li>DNA-binding , nucleotide cycle , and interaction 54 with : the transition state of NtrC exposes the GAFTGA loops and fixes the orientation of the DNA-binding domains The complex operon glnAntrBC , encoding glutamine synthetase as well as the histidine kinase NtrB and NtrC itself , is one of the well-characterized genetic loci that are regulated by NtrC ( for reviews , see Merrick and Edwards 1995 ; Reitzer 2003 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16751184" target="_blank">16751184</a>]<br></li>
<li>One such example is the s54-specific activators the oligomerization of which is modulated by the binding of effector ( s ) or covalent modification , as shown with XylR and NtrC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10559195" target="_blank">10559195</a>]<br></li>
<li>Despite the striking differences in the functioning of NtrC and TyrR , both have harnessed oligomerization , ATP binding , and ligand binding to regulate transcription in different ways .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11923293" target="_blank">11923293</a>]<br></li>
<li>For NtrC , activation leads to a competence in ATP hydrolysis achieved by increases in self-association of NtrC protomers , but ATP binding is not subject to activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12079332" target="_blank">12079332</a>]<br></li>
<li>For the discussion of these data one important parameter is the effect of ATP on NtrC binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9600853" target="_blank">9600853</a>]<br></li>
<li>The % C indicates the percentage of DNA bound by NtrC in comparison with unbound DNA in lane 1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24947454" target="_blank">24947454</a>]<br></li>
<li>NtrC is believed to be in a dimer-octamer or - hexamer equilibrium in the presence of DNA ( 42,47 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11923293" target="_blank">11923293</a>]<br></li>
<li>Phosphorylation induces strong co-operative binding of NtrC-P to DNA , leading to tetramerization or oligomerization of NtrC upon binding to DNA , which in turn induces the conformational change of the output domain to the active-conformation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7641688" target="_blank">7641688</a>]<br></li>
<li>NtrC dimers , capable of binding to DNA ( 184 ) , oligomerize into octamers upon phosphorylation ( 185 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10966457" target="_blank">10966457</a>]<br></li>
<li>Proteins were incubated with DNA for ten minutes at 37 C to allow enhancer-binding by NtrC , closed complex formation by polymerase , and phosphorylation of NtrC where applicable .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9367763" target="_blank">9367763</a>]<br></li>
<li>These include failure to activate , impaired DNA binding , and constitutive activation of NtrC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16751184" target="_blank">16751184</a>]<br></li>
<li>Typical data for the binding of NtrC to the DNA duplexes ES-1 and ES-2 are presented in Figure 3b .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9600853" target="_blank">9600853</a>]<br></li>
<li>One batch of the puri ® ed synthetic ES-2 duplexes showed a residual amount of 10 to 15 % free DNA duplex being present even in the presence of excess of NtrC protein at concentrations well above the dissociation constant for NtrC binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9600853" target="_blank">9600853</a>]<br></li>
<li>Upon lowering the protein concentration so that the DNA was in excess ( Figure 7a , green and black curves ) the s4 , buf value shifted to an extrapolated 3.4 ( 0.2 ) S at zero protein concentration which was predicted for that of a single NtrC dimer bound to ES-2 ( Tables 2 and 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9600853" target="_blank">9600853</a>]<br></li>
<li>For unphosphorylated NtrC protein in the absence of DNA a molecular weight of 102 ( 3 ) kDa was measured in the concentration range of 2 to 8 mM NtrC monomer .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9600853" target="_blank">9600853</a>]<br></li>
<li>It is possible that phosphorylation of the ArcA protein also facilitates cooperative interactions of mol-ecules bound to the DNA , as is apparent in the case of the NtrC protein ( see chapters 5 and 9 in reference 20 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8892825" target="_blank">8892825</a>]<br></li>
<li>In addition to the two DNA bound NtrC dimers , the two additional NtrC dimers had to be bound via protein-protein interactions , because the size of the DNA duplex excludes binding of more than two NtrC dimers to the DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9600853" target="_blank">9600853</a>]<br></li>
<li>In addition to the two DNA bound NtrC dimers that are depicted in red , two additional NtrC dimers ( yellow ) are bound via protein-protein interactions if the protein is phosphorylated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9600853" target="_blank">9600853</a>]<br></li>
<li>The search for a possible enhancer was focused on sequences resembling those of PspF - and 54 NtrC-binding sites , since PspF is the activator of the sdependent stress-induced pspA -- E operon , and NtrC 54 activates a few s - dependent promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17185540" target="_blank">17185540</a>]<br></li>
<li>FlbD of Caulobacter crescentus is a homologue of the NtrC ( NR1 ) protein and activates cr54-dependent flagellar gene promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8449883" target="_blank">8449883</a>]<br></li>
<li>NtrC appears to act as an en-hancer-binding protein , since it activates transcription at o'54-dependent promoters in a manner independent of the orientation or distance of its DNA-binding sites with respect to the transcription start site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1569008" target="_blank">1569008</a>]<br></li>
<li>A common concept for the action of these regulators is that inducer binding has the same consequence on ATPase activity and transcription activation as phosphorylation has in the case of the NtrC subclass ( 9 , 14 , 15 , 40 , 44 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8981978" target="_blank">8981978</a>]<br></li>
<li>Unlike NtrC , however , FhlA is activated not by phosphorylation but rather by direct binding of the effector formate ( 15 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8981978" target="_blank">8981978</a>]<br></li>
<li>It was found that the binding of formate phenome-nologically affects these activities of FhlA in the same way that phosphorylation affects NtrC activity ( 40 , 44 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8981978" target="_blank">8981978</a>]<br></li>
<li>Substitutions at a single amino acid residue in the nitrogenregulated activator protein NtrC differentially influence its activity in response to phosphorylation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16751184" target="_blank">16751184</a>]<br></li>
<li>This can take place either by phosphorylation as in the case for NtrC [ 17 ] and many other regulators , by protein ± protein interaction as shown for NifA [ 18 ] or by binding of an effector-molecule .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10848985" target="_blank">10848985</a>]<br></li>
<li>The phosphorylation of NtrC by NtrB was also recently shown to increase the affinity of the activator for the upstream NtrC-binding sites of nifLA ( 17 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2656631" target="_blank">2656631</a>]<br></li>
<li>Active contribution of two domains to cooperative DNA binding of the enhancer-binding protein nitrogen regulator I ( NtrC ) of Escherichia coli : stimulation by phosphorylation and the binding of ATP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9600853" target="_blank">9600853</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12003934" target="_blank">12003934</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9878422" target="_blank">9878422</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8606168" target="_blank">8606168</a>]<br></li>
<li>The intrinsic ATPase activity associated with the c54-dependent enhancer-binding proteins like NtrC and NifA is also present in FhlA and this activity of FhlA also responded positively to the presence of formate [ 8 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10689164" target="_blank">10689164</a>]<br></li>
<li>The bacterial enhancer binding protein NtrC is a molecular machine : ATP hydrolysis is coupled to transcriptional activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8981978" target="_blank">8981978</a>]<br></li>
<li>Rombel , I. , North , A. , Hwang , I. , Wyman , C. and Kustu , S. ( 1998 ) The bacterial enhancer-binding protein NtrC as a molecular machine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19969540" target="_blank">19969540</a>]<br></li>
<li>The best-characterized protein of this family , NtrC , has been shown to possess ATPase activity that is phosphorylationdependent and stimulated by site-specific DNA-binding [ 41,48 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17537579" target="_blank">17537579</a>]<br></li>
<li>The NtrC family response-regulators bind to enhancer-like sequences , but DNA binding is not affected by protein phosphorylation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11976297" target="_blank">11976297</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_289">
The procaryotic enhancer binding protein NtrC has an ATPase activity which is phosphorylation and DNA dependent .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8981978" target="_blank">8981978</a>]<br></label>
<input id="act_similarity_289", type="checkbox">
<div class="ACT_similar">
<li>The prokaryotic enhancer binding protein NtrC has an ATPase activity which is phosphorylation and DNA dependent .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8449883" target="_blank">8449883</a>]<br></li></div>
<li>Flashner , Y. , Weiss , D.S. , Keener , J. , and Kustu , S. ( 1995 ) Constitutive forms of the enhancer-binding protein NtrC : evidence that essential oligomerization determinants lie in the central activation domain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9720863" target="_blank">9720863</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_292">
Mutant form of the enhancer-binding protein NtrC can activate transcription from solution .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9878422" target="_blank">9878422</a>]<br></label>
<input id="act_similarity_292", type="checkbox">
<div class="ACT_similar">
<li>Mutant forms of the enhancer-binding protein NtrC can activate transcription from solution .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16751184" target="_blank">16751184</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10960104" target="_blank">10960104</a>]<br></li></div>
<li>[ Keywords : NtrC ; gene regulation ; AAA + ATPase assembly ; transcription activation ; enhancer-binding protein ; two-component signal transduction ] Supplemental material is available at http://www.genesdev.org .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16751184" target="_blank">16751184</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_295">
Weiss , D.S. , Batut , J. , Klose , K.E. , Keener , J. , and Kustu , S. ( 1991 ) The phosphorylated form of the enhancer-binding protein NtrC has an ATPase activity that is essential for activation of transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12453212" target="_blank">12453212</a>]<br></label>
<input id="act_similarity_295", type="checkbox">
<div class="ACT_similar">
<li>The phosphorylated form of the enhancer binding protein NtrC has an ATPase activity that is essential for activation of transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16751184" target="_blank">16751184</a>]<br></li>
<li>The phosphorylated form of the enhancer-binding protein NtrC has an ATPase activity that is essential for activation of transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8449883" target="_blank">8449883</a>]<br></li></div>
<li>At this locus , NtrC binds to an enhancer/operator region located between 100 and 150 bp upstream of glnA , and to an operator located between glnA and ntrB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16751184" target="_blank">16751184</a>]<br></li>
<li>At such promoters , hexamers of phosphorylated NtrC bind to enhancer-like sequences located B100 -- 150 nucleotides upstream of the transcription start site and in a reaction hydrolysing adenosine triphosphate ( ATP ) , convert a transcriptionally inactive initial s54-RNAp-promoter complex to a transcriptionally active one17 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24947454" target="_blank">24947454</a>]<br></li>
<li>Previous data showed that PspA inhibited NtrC-dependent transcription , suggesting that PspA might play a role in global regulation .11 In our hands E. coli NtrC s54-dependent transcription was largely unaffected by the addition of PspA ( data not shown ) in contrast to previous ﬁndings .11 Differences in the phosphorylation conditions ( carbamyl phosphate versus activation by NtrB ) used to generate phosphorylated active NtrC , rather than the sensitivity of phosphorylated NtrC to PspA might explain apparently different results .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12079332" target="_blank">12079332</a>]<br></li>
<li>Discussion Mechanism of positive activation in 54-dependent AAA + ATPases : direct contribution to hexamer stability by phosphorylated NtrC receiver domains Regulation of the assembly of 54-dependent AAA + ATPases can be achieved by either a negative or a positive mechanism .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16751184" target="_blank">16751184</a>]<br></li>
<li>In Figure 2 activation of RNAP s54 transcription by phosphorylated His-tagged NtrC ( lane 3 ) was demonstrated by the appearance of the speci ® c 155 nucleotide transcript .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9600853" target="_blank">9600853</a>]<br></li>
<li>When a speciﬁc aspartate residue in the receiver domain of NtrC is phosphorylated , the dimeric protein forms an oligomer that is capable of ATP hydrolysis and transcriptional activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10960104" target="_blank">10960104</a>]<br></li>
<li>In this conformation higher order NtrC complexes can be stabilized by interaction between the phosphorylated receiver domain and the central activation domain of different NtrC dimers .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9600853" target="_blank">9600853</a>]<br></li>
<li>c , Comparison of triple substitution to wildtype : wild-type MBP-NtrC ( open square ) was phosphorylated with 10 mM carbamoyl phosphate for ten minutes at 37 C ( in the presence of DNA and s54-holoenzyme ) before open complex formation was initiated by addition of ATP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9367763" target="_blank">9367763</a>]<br></li>
<li>Thus , oligomerization of phosphorylated NtrC dimers at the enhancer is required for the formation of an active NtrC complex and appears to be a crucial feature of its biological function .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9600853" target="_blank">9600853</a>]<br></li>
<li>However , it appears likely that some conformation changes of the NtrC dimer occur in the phosphorylated octameric complex as compared to the free unpho-sphorylated dimer .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9600853" target="_blank">9600853</a>]<br></li>
<li>D86CFe Bio Subunit exchange between NtrC and NtrCD86C ( 2:1 ) yielded the hetero-dimers represented in panel C. All samples were phosphorylated and cleaved .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10960104" target="_blank">10960104</a>]<br></li>
<li>EMSA , EM and footprinting experiments were used to check the possible binding of PspF and NtrC-P ( NtrC phosphorylated in vitro by carbamyl phosphate ) to the 707 bp DNA fragment W1-W2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17185540" target="_blank">17185540</a>]<br></li>
<li>Under conditions where NtrC is phosphorylated in the cell , repression was higher at 54 % ( Figure 3A ) and 52 % ( Figure 3B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7641688" target="_blank">7641688</a>]<br></li>
<li>Under these conditions , NtrC was 50 to 100 % phosphorylated as judged by mass spectrometry ( I. Hwang and D. Yan , unpublished data ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10960104" target="_blank">10960104</a>]<br></li>
<li>The phosphorylated forms of these proteins then transfer phosphate to NtrC ( 22 , 49 ) and CheY ( 16 , 52 ) , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2649492" target="_blank">2649492</a>]<br></li>
<li>The in situ phosphorylated NtrC ( NtrC-P ) bound to the relA and positive control glnA probes ( Fig. 2a ) but not the other probes ( Supplementary Fig. 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24947454" target="_blank">24947454</a>]<br></li>
<li>The CpUpGpG RNA product is only synthesized under conditions when NtrC is phosphorylated in situ with carbamyl phosphate and s54-RNAp is present ( Supplementary Fig. 4C , lanes 4 and 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24947454" target="_blank">24947454</a>]<br></li>
<li>Transcription was studied in the absence ( lane 1 ) or presence of NtrC ( lane 2 and 3 ) , either unphosphorylated ( lane 2 ) or phosphorylated by the addition of 25 mM carbamyl phosphate ( lane 3 ) as described in Materials and Methods .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9600853" target="_blank">9600853</a>]<br></li>
<li>In experiments with phosphorylated NtrC , carbamoyl phosphate was added to 10 mM .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9367763" target="_blank">9367763</a>]<br></li>
<li>E. coli NtrC was phosphorylated as described above and all reactions using NtrC contained 10 mM carbamyl phosphate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12079332" target="_blank">12079332</a>]<br></li>
<li>NtrC was phosphorylated to NtrC-P before use by the addition of carbamyl phosphate ( 10 mM ) to the reaction mixture .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17185540" target="_blank">17185540</a>]<br></li>
<li>If NtrC was phosphorylated by the addition of carbamyl phosphate ( CH2NO5PNa2 ) and MgCl2 to the buffer its association properties changed dramatically .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9600853" target="_blank">9600853</a>]<br></li>
<li>Thus , we estimate that only a quarter to a third of the NtrC monomers are phosphorylated in equilibirium by carbamyl phosphate in our samples .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9600853" target="_blank">9600853</a>]<br></li>
<li>When phosphorylated , it yielded slightly more of bands 1 and 5 ( lane 4 versus lane 5 ) , which depend on phosD86CFe phorylation of NtrC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10960104" target="_blank">10960104</a>]<br></li>
<li>In contrast , acetyl phosphate will not accumulate , and thus NtrC will not be phosphorylated , when cells are grown with glucose as the sole carbon source .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7641688" target="_blank">7641688</a>]<br></li>
<li>In Fig. 6A , NtrC was phosphorylated with wild-type NtrB until steady-state levels were reached .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9892680" target="_blank">9892680</a>]<br></li>
<li>NtrC is phosphorylated in the absence of NtrB , when cells are grown on pyruvate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7641688" target="_blank">7641688</a>]<br></li>
<li>b , Complexes formed by phosphorylated NtrC with ES-2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9600853" target="_blank">9600853</a>]<br></li>
<li>Phosphorylated XN-NtrCRO is active as an ATPase To test the activity of our NtrC derivatives XN-NtrCRO and XN-NtrCo , we perfonned an ATPase assay .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7641688" target="_blank">7641688</a>]<br></li>
<li>First , purified hybrid proteins containing the receiver domain of NtrC ( kN-NtrCRO and XN-NtrCR ) were phosphorylated in vitro in the presence of NtrB and ATP to the same extent as wild-type NtrC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7641688" target="_blank">7641688</a>]<br></li>
<li>Third , purified kN-NtrCRO is active in vitro as an ATPase when phosphorylated in the presence of phosphoamidate , a low molecular weight phospho donor for wild-type NtrC ( see Figure 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7641688" target="_blank">7641688</a>]<br></li>
<li>The triply substituted protein MBP-NtrC ( D86N/A89T/V115I ) , though more active than MBP-NtrC ( D86N/A89T ) , was still not as active as phosphorylated MBP-NtrC ( wild-type ) ( Figure 2c ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9367763" target="_blank">9367763</a>]<br></li>
<li>The activity in the presence of 30 mM phosphoamidate is -10 - fold lower than that of wild-type NtrC ( not shown ) under the same conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7641688" target="_blank">7641688</a>]<br></li>
<li>The in vitro activity of all three fusion proteins carrying amino acid substitutions away from the active site was greatly stimulated by phosphoryl-ation with the low molecular mass phosphodonor carbamoyl phosphate and activities of the phosphorylated proteins were equivalent to that of phosphorylated NtrC ( wild-type ) ( not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9367763" target="_blank">9367763</a>]<br></li>
<li>This signal transductiodn itions of nitrogen limitation thceells contain an elevated consystem consistosf fourproteins : the transcription factorcentration of glutamine synthetase and the enzyme is highly NRI ( NtrC ) , which activategslnA transcription when it active , while under conditions of nitrogen excess the cells con-is phosphorylated , the kinase/phosphatase proteNiRnII tain a lower concentration of the enzyme and it is lesasctive .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7961766" target="_blank">7961766</a>]<br></li>
<li>It has been demonstrated that NtrB is a kinase/phosphatase capable of converting NtrC to a phosphorylated active form ( 55 , 56 ) and that CheA is autophosphorylated at a His residue and able to transfer the phosphoryl group to CheB and CheY ( 57 , 58 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2657652" target="_blank">2657652</a>]<br></li>
<li>Expression of glnA , the structural gene for glutamine synthetase , strongly depends on the amount of NtrC phosphorylated in the cell and thus is a direct measure of the activity of the NtrB derivatives ( 8 , 21 , 22 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9892680" target="_blank">9892680</a>]<br></li>
				</div></ul><br>
			<label class="collapse_DOM" for="_2"><font size="5px">NtrC protein domains</font></label>
				<input id="_2" type="checkbox">
				<div class="DOM"><br><ul><li>a Results Chimeric XNtrC proteins are active in vivo and in vitro To test the dimerization properties of various response regulator domains , we used the X system introduced by Hu et al. ( 1990 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7641688" target="_blank">7641688</a>]<br></li>
<li>In eukaryotes , the proteasomal ATPase family has undergone a massive proliferation : 6 orthologous lineages , namely S4 , S6a , S6b , S7 , S8 , S10a , and S10b , can be traced back to the common ancestor of all crown group eukaryotes Table 1 Clamp loader clade Pre-sensor 1 hairpin superclade SR [ CAT ] motif associated with Arginine ﬁnger b-hairpin inserted before Sensor 1 motif * HolB/DNAX family SFIII helicase clade [ viruses ] Zn cluster between helix-1 and strand 2 ** Hol B subfamily [ B ] ** DNAX subfamily [ B , Pl ] * N * P * O CLDV D5 ATPase family ositive strand RNA virus helicase fam ther viral helicases ily * RFC family ** MTH241/RFC2 subfamily [ AE ] HslU/ClpX/Lon / ClpAB-C clade Extended loop after strand 2 * HslU/ClpX family ( In eukaryotes includes RFC2 , RFC3 , RFC4 , RFC5 ) ** MTH240/RFC1 subfamily [ AE ] ( In eukaryotes includes RFC1 , Chl12 , RAD24 , Yor144c ) * WHIP family [ B , E ] ** HslU subfamily [ B , Kin , Api ] ** ClpX subfamily [ B , E ] * Lon family Fused to a C-terminal WC domain Fused to a Lon-protease at the C-terminus ** Bacterial lon protease subfamily [ B , E ] Fused to a LAN domain at the N-terminus ** Archaeal lon protease subfamily [ A > B ] * ClpAB-C family NRhD motif associated with Arginine ﬁnger ** ClpAB-C subfamily [ B , E ] ** Torsin subfamily [ E ] Torsin has a distinct a helical N-terminal domain RuvB family [ B , E ] Has C-terminal wHTH domain ; Helical segment between strand 5 and C-terminal bundle DnaA/ORC clade Two helices of approximately equal length after strand 2 * DnaA-family ** DnaA subfamily [ B ] ** DnaC subfamily [ B ] * CDC6/ORC family [ A , E ] ** CDC6/ORC [ A ] ** Orc1p , Orc4p , Orc5p , and Cdc6p subfamilies [ E ] Classical AAA clade and its divergent relatives Helix-2 insert clade Small helix between strand-2 and helix-2 , [ GN ] R motif associated with arginine ﬁnger * FtsH family [ B , E ] Fused to a metalloprotease at the C-terminus * CDC48 family [ A , E ] DPBB at N-terminus , 2 AAA + domains * Proteasomal ATPase family [ A , E ] ** S4 , S6a , S6b , S7 , S8 , S10a and S10b subfamilies [ E ] b -- a -- b insert in helix-2 NtrC/MCM group Have [ AG ] [ FL ] T motif in helix-2 before b -- a -- b insert * NtrC family [ B ] * MCM family [ A , E ] Fused to a Zn ribbon at N-terminus and a wHTH at the C-terminus * Chelatase / YifB group ** Chelatase family [ A , B , plants ] Associated with a vWA domain ** YifB family [ B ] Fused to a Lon Protease at its N-terminus , insert of Zn cluster after strand 4 * McrB family [ B > A , E ] * Mox R group ** MoxR family [ A , B ] ** Dynein/Midasin family * Katanin p60/Fidgetin family [ E ] * NSF1 family [ E ] * Pex1/6 subfamily [ E ] * Bcs1p subfamily [ E ] ( Possible relatives of classical AAA clade ) * AFG1 family [ B , E ] * ClpAB-N family [ B > A , E ] * TIP49 family [ A , E ] N-terminal module with characteristic H between Helix-2 and Strand 3 [ ST ] H motif , Predicted b barrel 6 Tandem AAA + domains ** Dynein subfamily [ E ] ** Midasin subfamily [ E ] Abbreviations : A , Archaea ; B , Bacteria ; E , Eukaryotes ; Api , Apicomplexans ; Kin , Kinetoplastids ; and Pl , Plants `` > '' indicates sporadic lateral transfer .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15037234" target="_blank">15037234</a>]<br></li>
<li>HydG showed considerable homology with respect to the TTA 0M ATC ACG CGC AAA ACG CTA TTG GCA AAA CTG TCG CGT TAG TTC TGC TGO 00r TMG Lau Gly Ibm Thr Ar ; Lys Thr Lau Lau Ala Lys Iou Ber Ar ; STP presumed DNA-binding consensus ( 90 % ) , as well with the other functional domains that can be discerned on NtrC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2666400" target="_blank">2666400</a>]<br></li>
<li>Residues that differ in the nuclear magnetic resonance ( NMR ) structures of unphosphorylated and phosphorylated NtrC ( 206 ) are mapped in green onto the corresponding surface of CheY ; residues that show signiﬁcant NMR chemical shift perturbations in phosphorylated CheY are mapped in blue ( 199 ) ; and overlapping residues appear as cyan .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10966457" target="_blank">10966457</a>]<br></li>
<li>When NtrCD54C was chemically cleaved at cysteine residues ( see Materials and Methods ) and the fragments so generated were used as mo-lecular weight standards , bands 2/29 migrated slightly faster than the C-terminal fragment carrying the sequence from res-idue 54 to the end of NtrC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10960104" target="_blank">10960104</a>]<br></li>
<li>To determine whether the three cleavages giving rise to bands 1 and 5 , bands 2 and 4 , and bands 3 and 29 occurred within a monomer or between monomers , we ﬁrst assessed whether monomerization of NtrC by means of a C-terminal truncation [ D ( 444 -- 469 ) ] affected the efﬁciencies with which these cleavages occurred .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10960104" target="_blank">10960104</a>]<br></li>
<li>To demonstrate that the same cleavages occurred in populations containing heterodimers as in popu-Fe lations of NtrCD86C homodimers , we demonstrated that antibodies directed against the C terminus of NtrC detected bands 29 , 4 , and 5 ( Fig. 4B , lane 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10960104" target="_blank">10960104</a>]<br></li>
<li>c , Comparison of triple substitution to wildtype : wild-type MBP-NtrC ( open square ) was phosphorylated with 10 mM carbamoyl phosphate for ten minutes at 37 C ( in the presence of DNA and s54-holoenzyme ) before open complex formation was initiated by addition of ATP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9367763" target="_blank">9367763</a>]<br></li>
<li>However , compatible with the low activity of NtrC ( V115I ) , the changes in the latter two regions affected fewer residues and were smaller than the changes seen in the singly and doubly substituted N termini discussed above .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9367763" target="_blank">9367763</a>]<br></li>
				<label class="DOM_collapse_similarity" for="dom_similarity_10">
Drummond , M.H. , Contreras , A. and Mitchenall , L.A. ( 1990 ) The function of isolated domains and chimaeric proteins constructed from the transcriptional activators NifA and NtrC of Klebsiella pneumoniae .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7641688" target="_blank">7641688</a>]<br></label>
<input id="dom_similarity_10", type="checkbox">
<div class="DOM_similar">
<li>The function of isolated domains and chimaeric proteins constructed from the transcriptional activators NifA and NtrC of Klebsiella pneumoniae .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16751184" target="_blank">16751184</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9367763" target="_blank">9367763</a>]<br></li></div>
<li>We first generated an NtrC ATPase monomer model using the threading function of the DeepView program ( Guex and Peitsch 1997 ) on the NtrC1 ATPase structure .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16751184" target="_blank">16751184</a>]<br></li>
<li>By analogy to the case for muscle and yeast glycogen phosphorylases ( 14 , 19 , 20 ) , this intermolecular interaction may rearrange the dimer-interface to promote phosphorylation-dependent oligomerization of NtrC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10960104" target="_blank">10960104</a>]<br></li>
<li>Interestingly , ﬁve of the cysteine substitutions appear to result in loss of transcriptional activation by NtrC in vivo ( J. Lee , unpublished results ) , commensurate with the postulated role of this helix in signal propagation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10960104" target="_blank">10960104</a>]<br></li>
<li>This suggests further that the active NtrC-P tetramer is actually composed of two active dimeric units , where each subunit originally belongs to a different constitutive NtrC dimer .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7641688" target="_blank">7641688</a>]<br></li>
<li>We would like to note that in the model of the octameric complex ( Figure 8c and d ) it would be enough if one or two NtrC monomers ( i.e. a degree of 13 to 25 % phosphorylation ) are phosphorylated to stabilize the complex by interaction between two different NtrC dimers .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9600853" target="_blank">9600853</a>]<br></li>
<li>NtrC must be phosphorylated in order to carry out itsactivation function ( Ninfa and Magasanik , 1986 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8444880" target="_blank">8444880</a>]<br></li>
<li>Thus , oligomerization of phosphorylated NtrC dimers at the enhancer is required for the formation of an active NtrC complex and appears to be a crucial feature of its biological function .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9600853" target="_blank">9600853</a>]<br></li>
<li>However , it appears likely that some conformation changes of the NtrC dimer occur in the phosphorylated octameric complex as compared to the free unpho-sphorylated dimer .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9600853" target="_blank">9600853</a>]<br></li>
<li>This suggests that kN-NtrC is phosphorylated in vivo by NtrB , is able to tetramerize or oligomerize to the active ATPase and can interact with the o54 RNA polymerase holo-enzyme to activate transcription at the glnAp2 promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7641688" target="_blank">7641688</a>]<br></li>
<li>It has been demonstrated that NtrB is a kinase/phosphatase capable of converting NtrC to a phosphorylated active form ( 55 , 56 ) and that CheA is autophosphorylated at a His residue and able to transfer the phosphoryl group to CheB and CheY ( 57 , 58 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2657652" target="_blank">2657652</a>]<br></li>
<li>In the NtrB/NtrC system , a His residue in NtrB and an Asp residue in NtrC were identified as the phosphorylated amino acids ( 59 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2657652" target="_blank">2657652</a>]<br></li>
<li>Apparently the G domain is able to fold independently into an active domain and phosphorylate the histidine residue of the H domain in trans , and the phosphorylated H -- N fragment is able to pass its phosphate group on to NtrC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9892680" target="_blank">9892680</a>]<br></li>
<li>-------- N A V E E V I E N 0 G I S L D - P T S N N VN UhPA DctD NtrC PgtA VirG CheT Oin * Dye PhoB FIG. 2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3042748" target="_blank">3042748</a>]<br></li>
<li>This mechanism implies that part of the binding energy gained by the interaction of the phosphorylated receiver modules in wild-type NtrC is used for the conformational change in the output domains .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7641688" target="_blank">7641688</a>]<br></li>
<li>This explains why the interaction between two isolated receiver modules appears to be stronger than the interaction between two phosphorylated NtrC molecules .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7641688" target="_blank">7641688</a>]<br></li>
<li>We suggest that NtrC and PhoB represent class of one response regulators , where activation is mediated by dimerization of the phosphorylated receiver modules .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7641688" target="_blank">7641688</a>]<br></li>
<li>A common switch in activation of the response regulators NtrC and PhoB : phosphorylation induces dimerization of the receiver modules During signal transduction , response regulators of two-component systems are phosphorylated in a conserved receiver module .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7641688" target="_blank">7641688</a>]<br></li>
				<label class="DOM_collapse_similarity" for="dom_similarity_29">
A common switch in activation of the response regulators NtrC and PhoB : phosphorylation induced dimerization of receiver modules .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9973351" target="_blank">9973351</a>]<br></label>
<input id="dom_similarity_29", type="checkbox">
<div class="DOM_similar">
<li>A common switch in activation of the response regulators NtrC and PhoB : phosphorylation induces dimerization of the receiver modules .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8981978" target="_blank">8981978</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659677" target="_blank">15659677</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8892825" target="_blank">8892825</a>]<br></li></div>
<li>In class I ( e.g. NtrC and PhoB ) , this interaction leads to the inhibition of the receiver module .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7641688" target="_blank">7641688</a>]<br></li>
<li>The receiver-domains of NtrC and PhoB are dimerization domains We have shown that the unphosphorylated receiver-domains of NtrC and PhoB are unable to dimerize when attached to their cognate output domains : unphosphoryl-ated XN-PhoB and XN-NtrCRO , which both carry the receiver and output domains , are monomeric proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7641688" target="_blank">7641688</a>]<br></li>
<li>Analysis of these hybrid proteins shows that the receiver modules of NtrC and PhoB are potential dimerization domains .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7641688" target="_blank">7641688</a>]<br></li>
<li>We show that the receiver modules of NtrC and PhoB potential dimerization are domains .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7641688" target="_blank">7641688</a>]<br></li>
<li>Discussion Our results establish that the receiver modules of NtrC and PhoB are dimerization domains .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7641688" target="_blank">7641688</a>]<br></li>
<li>These relatively conservative substitutions affect residues that are not identical either among activators of s54-holoen-zyme that contain receiver-domains ( e.g. DctD , AlgB ; see Pao & Saier , 1995 ) or even among NtrC proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9367763" target="_blank">9367763</a>]<br></li>
<li>Our structure of the activated NtrC suggests that phenotypes of substitutions in residues 149 -- 156 of NtrC helix N arise from specific changes in the interface between ATPase and receiver-domains , and it suggests intermediates for the assembly process .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16751184" target="_blank">16751184</a>]<br></li>
<li>Based on the conserved structure of receiver-domains ( Volz , 1993 ) , data on the structure of activated forms of CheY and NtrC may be relevant to the structural basis for activation of receiver-domains generally .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9367763" target="_blank">9367763</a>]<br></li>
<li>Higher mRNA levels in the first strain in experiments a -- f are indicative of Nac activation , whereas higher levels in the first strain in experiments c -- h are indicative of NtrC activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11121068" target="_blank">11121068</a>]<br></li>
<li>For NtrC , activation leads to a competence in ATP hydrolysis achieved by increases in self-association of NtrC protomers , but ATP-binding is not subject to activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12079332" target="_blank">12079332</a>]<br></li>
<li>In the case of NtrC , the hydrolysis of ATP at this site is necessary for the activation of the relevant 54 promoter by converting it from a closed to an open complex ( 6 , 31 , 42 , 57 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8449883" target="_blank">8449883</a>]<br></li>
<li>In the case of NtrC , ATP hydrolysis is required during the process of activation to convert the or '' holoenzyme DNA complex from a closed to an open configuration ( 6 , 31 , 57 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8449883" target="_blank">8449883</a>]<br></li>
<li>GENES & DEVELOPMENT 20:1485 -- 1495 © 2006 by Cold Spring Harbor Laboratory Press ISSN 0890-9369/06 ; www.genesdev.org Results Our studies of the activated , oligomeric form of NtrC were facilitated by conditions that stably activate an NtrC variant that combined an S160F mutation , which favors an equilibrium toward the active , oligomeric state , and three alanine substitutions ( R456A , N457A , and R461A ) in the DNA-binding domain that prevent aggregation of the protein in the activated state ( Rombel et al. 1998 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16751184" target="_blank">16751184</a>]<br></li>
<li>In titrations of our various protein preparations we observed the maximal amount of transcription-activation at 15 to 20 nM NtrC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9600853" target="_blank">9600853</a>]<br></li>
<li>However , a speci ® c biological function of this association state seems unlikely since all known biological activities of NtrC like transcriptional activation , ATPase activity and DNA-binding can be reproduced in vitro at nanomolar concentrations .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9600853" target="_blank">9600853</a>]<br></li>
<li>A common concept for the action of these regulators is that inducer-binding has the same consequence on ATPase activity and transcription-activation as phosphorylation has in the case of the NtrC subclass ( 9 , 14 , 15 , 40 , 44 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8981978" target="_blank">8981978</a>]<br></li>
				<label class="DOM_collapse_similarity" for="dom_similarity_47">
Constitutive forms of the enhancer-binding protein NtrC : evidence that essential oligomerization determinants lie in the central activation domain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8981978" target="_blank">8981978</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9600853" target="_blank">9600853</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9367763" target="_blank">9367763</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10960104" target="_blank">10960104</a>]<br></label>
<input id="dom_similarity_47", type="checkbox">
<div class="DOM_similar">
<li>Flashner , Y. , Weiss , D.S. , Keener , J. & Kustu , S. ( 1995 ) Constitutive forms of the enhancer-binding protein NtrC : evidence that essential oligomerization determinants lie in the central activation domain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10848985" target="_blank">10848985</a>]<br></li>
<li>Flashner , Y. , Weiss , D.S. , Keener , J. , and Kustu , S. ( 1995 ) Constitutive forms of the enhancer-binding protein NtrC : evidence that essential oligomerization determinants lie in the central activation domain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9720863" target="_blank">9720863</a>]<br></li>
<li>[ 47 ] Y. Flashner , D.S. Weiss , J. Keener , S. Kustu , Constitutive forms of the enhancer-binding protein NtrC : evidence that essential oligomerization determinants lie in the central activation domain , J. Mol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17537579" target="_blank">17537579</a>]<br></li>
<li>[ 5 ] Y. Flashner , D.S. Weiss , J. Keener , S. Kustu , Constitutive forms of the enhancer-binding protein NtrC : evidence that essential oligomerization determinants lie in the central activation domain , J. Mol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21115262" target="_blank">21115262</a>]<br></li></div>
<li>Repressor forms of the enhancer-binding protein NtrC : some fail in coupling ATP hydrolysis to open complex formation by s54-holoenzyme .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9600853" target="_blank">9600853</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9260970" target="_blank">9260970</a>]<br></li>
<li>At high concentrations many of the 54-dependent ATPase domains ( including NtrC 's ) can act in the absence of DNA-binding activity and in an enhancer-independent fashion ( North and Kustu 1997 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16751184" target="_blank">16751184</a>]<br></li>
<li>Wyman C , Rombel I , North AK , Bustamante C & Kustu S ( 1997 ) Unusual oligomerization required for activity of NtrC , a bacterial enhancer-binding protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16553838" target="_blank">16553838</a>]<br></li>
<li>Unusual oligomerization required for activity of NtrC , a bacterial enhancer-binding protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9367763" target="_blank">9367763</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14996819" target="_blank">14996819</a>]<br></li>
<li>The bacterial enhancer binding protein NtrC is a molecular machine : ATP hydrolysis is coupled to transcriptional activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8981978" target="_blank">8981978</a>]<br></li>
				<label class="DOM_collapse_similarity" for="dom_similarity_57">
The phosphorylated form of the enhancer binding protein NtrC has an ATPase activity that is essential for activation of transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16751184" target="_blank">16751184</a>]<br></label>
<input id="dom_similarity_57", type="checkbox">
<div class="DOM_similar">
<li>The phosphorylated form of the enhancer-binding protein NtrC has an ATPase activity that is essential for activation of transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8449883" target="_blank">8449883</a>]<br></li></div>
				<label class="DOM_collapse_similarity" for="dom_similarity_59">
The procaryotic enhancer binding protein NtrC has an ATPase activity which is phosphorylation and DNA dependent .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8981978" target="_blank">8981978</a>]<br></label>
<input id="dom_similarity_59", type="checkbox">
<div class="DOM_similar">
<li>The prokaryotic enhancer-binding protein NtrC has an ATPase activity which is phosphorylation and DNA dependent .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9878422" target="_blank">9878422</a>]<br></li>
<li>The prokaryotic enhancer binding protein NtrC has an ATPase activity which is phosphorylation and DNA dependent .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8449883" target="_blank">8449883</a>]<br></li></div>
<li>These domains either connect the NtrC ATPase to the two-component signaling systems or function as small mol-ecule-binding domains that enable the ATPase to sense various metabolites in the environment ( Rombel et al. , 1998 ; Zhang et al. , 2002 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15037234" target="_blank">15037234</a>]<br></li>
<li>The DNA-binding domains are localized to the bottom side of the NtrC ring , and therefore binding to the enhancer is on the opposite face of the ATPase from its interaction site with 54 polymerase and promoter DNA , which occurs via the GAFTGA loop region ( Lee et al. 2003 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16751184" target="_blank">16751184</a>]<br></li>
<li>Conclusions The structural model that we describe for activated NtrC , in which the activated receiver-domain of one subunit is in contact with the ATPase domain of a second one to stabilize ring assembly , differs dramatically from the model proposed to explain how two-component signal transduction regulates assembly of AAA + ATPase domains in the enhancer-binding proteins NtrC1 and DctD ( Lee et al. 2003 ; Doucleff et al. 2005 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16751184" target="_blank">16751184</a>]<br></li>
<li>Active contribution of two domains to cooperative DNA-binding of the enhancer-binding protein nitrogen regulator I ( NtrC ) of Escherichia coli : stimulation by phosphorylation and the binding of ATP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9600853" target="_blank">9600853</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12003934" target="_blank">12003934</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9878422" target="_blank">9878422</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8606168" target="_blank">8606168</a>]<br></li>
<li>These DNA-binding domains can be subdivided into three major families , represented by OmpR , NarL , and NtrC ( 14 , 7 , and 4 members , respectively , in E. coli ) ( 4 , 63 , 133 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10966457" target="_blank">10966457</a>]<br></li>
<li>This segment of the protein is 45 % homologous to the putative DNA-binding region ( helix-turn-helix motif ; 22 ) of the HydG and NtrC proteins ( Fig. 9 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2118503" target="_blank">2118503</a>]<br></li>
<li>Alignment of possible DNA-binding motifs in the NtrC , FhlA , and HydG proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2118503" target="_blank">2118503</a>]<br></li>
<li>These include failure to activate , impaired DNA-binding , and constitutive activation of NtrC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16751184" target="_blank">16751184</a>]<br></li>
<li>DNA-binding and oligomerization of NtrC studied by ¯ uorescence anisotropy and ¯ uorescence correlation spectroscopy .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9600853" target="_blank">9600853</a>]<br></li>
<li>Given that the glnA enhancer region contains two tandem motifs for binding NtrC dimers , formation of a hexamer upon activation would imply that a third dimer would have to be recruited and incorporated to close the ring , possibly aided by binding to nonspecific DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16751184" target="_blank">16751184</a>]<br></li>
<li>( ii ) In the unphosphorylated state one NtrC dimer was bound to one binding site as determined with DNA oli-gonucleotide duplexes containing one or two binding sites ( concentration range studied 50 to 1000 nM NtrC dimer ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9600853" target="_blank">9600853</a>]<br></li>
<li>This demonstrates the binding of two NtrC dimers to the two DNA-binding sites in agreement with a calculated molecular weight of 253 kDa for this complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9600853" target="_blank">9600853</a>]<br></li>
<li>The in vitro activity of all three fusion proteins carrying amino acid substitutions away from the active site was greatly stimulated by phosphoryl-ation with the low molecular mass phosphodonor carbamoyl phosphate and activities of the phosphorylated proteins were equivalent to that of phosphorylated NtrC ( wild-type ) ( not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9367763" target="_blank">9367763</a>]<br></li>
<li>Moreover , changes in the chemical shifts of the active site residues D11 , D12 and D54 and of residues G36 , G62 and M81 were cumulative in N termini carrying one , two or three substitutions in helices 4 and 5 , paralleling the increasing activity of the intact NtrC proteins carrying these substitutions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9367763" target="_blank">9367763</a>]<br></li>
<li>In fact , amino acid residues 89 and 115 of NtrC proteins from several alpha-proteobacteria are naturally threonine or isoleucine , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9367763" target="_blank">9367763</a>]<br></li>
<li>Substitutions at a single amino acid residue in the nitrogenregulated activator protein NtrC differentially influence its activity in response to phosphorylation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16751184" target="_blank">16751184</a>]<br></li>
<li>It has been shown that this amino acid sequence contains residues important for ATP-binding , ATPase activity , and interaction of NtrC with s54-RNAP ( 17 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9260970" target="_blank">9260970</a>]<br></li>
<li>Western analysis of cleavage products from the truncated monomeric D86CFe protein NtrC , D444 -- 469 using antibodies directed against the N and C termini of NtrC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10960104" target="_blank">10960104</a>]<br></li>
<li>Obviously , other arrangements of the different NtrC domains are possible that would also lead to a good ® t with the AUC data obtained for the unphosphorylated protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9600853" target="_blank">9600853</a>]<br></li>
<li>NtrC and Lrp are identical in the two strains , and catabolite gene activator protein and the a subunit of RNA polymerase differ by only one and two amino acids in these bacteria .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9495755" target="_blank">9495755</a>]<br></li>
<li>NtrC , this surface , which is the one we have derivatized , presumably contacts the remainder of the protein ( Fig. 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10960104" target="_blank">10960104</a>]<br></li>
<li>Evidence for a second interaction between the regulatory amino-terminal and central output domains of the response regulator NtrC ( nitrogen regulator I ) in Esch-erichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659676" target="_blank">15659676</a>]<br></li>
<li>Alterations of highly conserved residues in the regulatory domains of nitrogen regulator I ( NtrC ) of Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7730282" target="_blank">7730282</a>]<br></li>
<li>[ 50 ] J.B. Moore , S.P. Shiau , L.J. Reitzer , Alterations of highly conserved residues in the regulatory domain of nitrogen regulator I ( NtrC ) of [ 51 ] J.-M .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16153782" target="_blank">16153782</a>]<br></li>
				<label class="DOM_collapse_similarity" for="dom_similarity_86">
Altera-tions of highly conserved residues in the regulatory domain of nitrogen regulator I ( NtrC ) of Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9367763" target="_blank">9367763</a>]<br></label>
<input id="dom_similarity_86", type="checkbox">
<div class="DOM_similar">
<li>Alterations of highly conserved residues in the regulatory domain of nitrogen regulator I ( NtrC ) of Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12003934" target="_blank">12003934</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11812125" target="_blank">11812125</a>]<br></li></div>
<li>Due to the differences in methodology between transcription assays described11 and in this study ( Figure 2 ) the inhibi-tory effects of PspA on sigma isomerisation cata-lysed by NifA central domain and phosphorylated NtrC were analysed to further conﬁrm that negative effects of PspA are speciﬁc to PspF , and do not play a role in global regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12079332" target="_blank">12079332</a>]<br></li>
<li>Recently determined structures of the phosphorylated regulatory domains of NtrC ( 206 ) and FixJ ( 159a ) and of a stably modiﬁed phosphoprotein analog , phosphono-CheY ( 207 ; CJ Halkides , MM McEvoy , P Matsumura , K Volz , FW Dahlquist , manuscript in preparation ) conﬁrm that phosphorylation causes long-range structural perturbations , altering the molecular surface of the regulatory domain ( Figure 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10966457" target="_blank">10966457</a>]<br></li>
<li>Sequence analysis of rmpA suggests it is embedded in a larger open reading frame , rmpA2 , which is similar to RcsA in the N terminus , has a LuxR-like C terminus ( as do RcsA and RcsB ) , and a region with homology to a domain of the sigma-54-dependent response regulator , NtrC ( 112 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16153174" target="_blank">16153174</a>]<br></li>
<li>The `` pre-sensor-1 b-hairpin '' ( PS1BH ) superclade All the remaining lineages of AAA + domains , namely , the SF3 helicases , HslU and ClpX , Lon , torsin , the C-terminal AAA + domain of ClpAB , RuvB , NtrC , MoxR , and its relatives , dynein , midasin , and the chelatases , can be uniﬁed into a vast monophyletic assemblage .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15037234" target="_blank">15037234</a>]<br></li>
<li>The NtrC family has diversiﬁed within the bacteria into several distinct subfamilies that have characteristic fusions with various N-terminal domains .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15037234" target="_blank">15037234</a>]<br></li>
<li>The response regulator NtrC contains an N-terminal receiver and two additional domains needed to activate transcription from the glnAp2 promoter , which uses the o54 form of RNA polymerase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8098993" target="_blank">8098993</a>]<br></li>
<li>The open symbols refer to unphosphorylated NtrC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9600853" target="_blank">9600853</a>]<br></li>
<li>The value of n refers to a complex of four NtrC dimers and two ES-1 duplexes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9600853" target="_blank">9600853</a>]<br></li>
<li>The boxes designate the Q-linker of TyrR , the Walker A and B ATP-binding motifs , the PASE-linker of NtrC ( 32 ) , and the helix-turn-helix motif based on the structure of the C-terminus of H. influenzae TyrR ( 34 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11923293" target="_blank">11923293</a>]<br></li>
<li>The next region of about 24 amino acids ( region II ; amino acids 370 to 393 ) is about 60 % homologous with the HydG protein and is less homologous ( 46 % ) with the NtrC and NifA proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2118503" target="_blank">2118503</a>]<br></li>
<li>The sequence of TyrR protein from amino acids 223 to 434 is homologous to the central domains of NtrC and NifA and a variety of other activator proteins from diverse organisms ( 20 , 53 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8449883" target="_blank">8449883</a>]<br></li>
<li>The NifA/NtrC regulators also contain an ATP-binding site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8246840" target="_blank">8246840</a>]<br></li>
<li>The central domain is similar to the 230 amino acid conserved ATP-binding domains of the sigma 54-interaction family , which includes NtrC and NifA that are involved in nitrogen and nitrogenase regulation respectively ( Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11923293" target="_blank">11923293</a>]<br></li>
<li>Not-withstanding the fact that the TyrR protein contains an ATP-binding domain and a helix-tum-helix DNA-bind-ing domain which are structurally homologous to domains of similar functions in proteins such as NifA , NtrC , DctD and XylR , it differs from them in a number of respects .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1943694" target="_blank">1943694</a>]<br></li>
<li>The central and C-terminal domains of TyrR do show certain homologies with members of the NtrC family of activator proteins , in which both a DNA-binding protein and a histidine protein kinase are required for activation ( 21 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7798138" target="_blank">7798138</a>]<br></li>
<li>The central domain of TyrR contains an ATP-binding site and is homologous with the NtrC family of activator proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8449883" target="_blank">8449883</a>]<br></li>
<li>The blots were incubated with primary antibodies raised against MBP ( New England Biolabs , Beverly , Mass. ) , NtrC , the N-terminal domain of NtrC ( 124 residues ; Zymed , South San Francisco , Calif. ) , the C-terminal domain of NtrC ( 90 resi-dues ; Zymed ) , or the metal chelate ( CHA255 [ 35 ] ; a kind gift from David Goodwin , Stanford University ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10960104" target="_blank">10960104</a>]<br></li>
<li>Because the truncated form of NtrC lacks the last 25 residues of the protein , C-terminal cleavage fragments should migrate faster on SDS-polyacryl-amide gels , and this was the case for bands 4 and 5 ( Fig. 1B Fe FIG. 1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10960104" target="_blank">10960104</a>]<br></li>
<li>For each band , the results of Western analysis using antibodies directed against MBP , the N-terminal domain of NtrC ( N ) , the C-terminal domain ( C ) , or the iron chelate ( Fe ) are indicated , as is the reaction of streptavidin with biotin ( Bio ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10960104" target="_blank">10960104</a>]<br></li>
<li>Band 1 , which was detected only with antibodies directed against the N terminus of NtrC , is an N-terminal fragment .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10960104" target="_blank">10960104</a>]<br></li>
<li>Band 29 from truncated NtrCD86C , D444 -- 469 was detected by both anti-N-terminal and anti-C-terminal antibodies to NtrC and was also detected by antibodies to the iron chelate , indicating that it contains residue 86 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10960104" target="_blank">10960104</a>]<br></li>
<li>However , EPR analysis could not give us any information on where the interdomain interaction between a4 of the N-terminal domain and the remainder of the protein was occurring , and we were unable to determine whether the interaction occurred within a phosphorylated NtrC dimer or required the formation of larger oligomers .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10960104" target="_blank">10960104</a>]<br></li>
<li>Thus , bands 29 and 4 appear to be C-terminal cleavage fragments from unphosphorylated NtrC ( also seen when the protein is phosphorylated ) , whereas band 5 is a C-terminal fragment obtained only when the protein is phosphorylated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10960104" target="_blank">10960104</a>]<br></li>
<li>Thus , the cleavage that occurred uniquely in phosphorylated NtrC apparently occurred outside its N-termi-nal domain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10960104" target="_blank">10960104</a>]<br></li>
<li>Finally , bands 3 and 29 appear to be N - and C-terminal partners , respectively , of a cleavage that occurs within the N-terminal domain of unphosphorylated NtrC ( Fig. 2 and 3 ; Table 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10960104" target="_blank">10960104</a>]<br></li>
<li>Together with the size of band 1 ( see below ) , immunological evidence was commensurate with the view that the cleavage that gave rise to bands 1 and 5 occurred outside the N-terminal domain of NtrC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10960104" target="_blank">10960104</a>]<br></li>
<li>Given the predicted position of cleavage that gave rise to band 29 , i.e. , that it occurred prior to position 86 in the N-terminal domain of NtrC , detection by anti-N-terminal antibodies is expected based on the results obtained for band 4 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10960104" target="_blank">10960104</a>]<br></li>
<li>Constitutively activating amino acid substitutions allow definition of an activating surface of NtrCN-term The approach of using constitutively activating amino acid substitutions to investigate structural changes involved in activation of receiver-domains , such as that of NtrC , is attractive for several reasons .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9367763" target="_blank">9367763</a>]<br></li>
<li>First , larger chemical shift changes in mutant forms of NtrCN-term carrying multiple amino acid substitutions correlate with greater activity of the corresponding intact NtrC proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9367763" target="_blank">9367763</a>]<br></li>
<li>Mechanism of activation of NtrC and other response regulator proteins Mutant forms of NtrCN-term carrying constitutive amino acid substitutions have been used to de ® ne the 3445 face not only as a region of the molecule that undergoes structural changes in the activated state but as a region that is likely to play an important role in signal transduction to the remainder of the protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9367763" target="_blank">9367763</a>]<br></li>
<li>Structural a Functional Analyses of Activating Amino Acid Substitutions in the Receiver om NtrC : Eviden o n Activating Surface nd D ain of ce f r a Michael Nohaile1 , Dorothee Kern1 , David Wemmer1 * , Kenneth Stedman2 and Sydney Kustu2 1 Department of Chemistry University of California Berkeley & Structural Biology Division , Lawrence Berkeley National Laboratory , Berkeley CA 94720 , USA 2 Department of Molecular and Cell Biology , University of California , Berkeley , CA 94720 USA # 1997 Academic Press Limited Keywords : two-component systems ; constitutive mutant forms of NtrC ; phosphorylation ; signal transduction ; NMR chemical shift analysis * Corresponding author Introduction The bacterial enhancer-binding protein NtrC activates transcription when phosphorylated on aspartate 54 in its amino ( N ) - terminal regulatory domain or when altered by constitutively activating amino acid substitutions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9367763" target="_blank">9367763</a>]<br></li>
<li>Moreover , structural differences between mutant and wild-type forms provide evidence that changes in a speci ® c surface of the N-terminal domain of NtrC are associated with transcriptional activation of the fulllength protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9367763" target="_blank">9367763</a>]<br></li>
<li>Comparison of amide chemical shifts for constitutively active mutant forms and the wild-type form of the N-terminal domain of NtrC ( NtrCN-term ) Figure 3 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9367763" target="_blank">9367763</a>]<br></li>
<li>Structural and functional analyses of activating amino acid substitutions in the receiver-domain of NtrC : evidence for an activating surface .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10960104" target="_blank">10960104</a>]<br></li>
<li>In the present work we examine structural and functional differences between the wild-type receiver-domain of NtrC and mutant forms carrying constitutively activating substitutions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9367763" target="_blank">9367763</a>]<br></li>
<li>NMR spectra of the phosphorylated receiver-domain of wild-type NtrC and of constitutive mutant forms with amino acid substitutions in a4 indicated that conformational changes occurred in the so-called 3445 face of the molecule ( 22 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10960104" target="_blank">10960104</a>]<br></li>
<li>The use of this assay along with the known NMR structure of the N-terminal domain of NtrC ( Volkman et al. , 1995 ) allowed us to perform parallel functional and structural investigations of the changes caused by individual constitutive amino acid substitutions in the Nterminal domain of NtrC and by combinations of such substitutions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9367763" target="_blank">9367763</a>]<br></li>
<li>Three of them were found to carry amino acid substitutions in the N-terminal domain of NtrC ; aspartate 86 to asparagine ( D86N ) , alanine 89 to threonine ( A89T ) and valine 115 to isoleucine ( V115I ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9367763" target="_blank">9367763</a>]<br></li>
<li>Activities of NtrC proteins carrying combinations of constitutive amino acid substitutions Since the activities of three of the MBP-NtrC proteins carrying single constitutive substitutions in the N-terminal domain could be increased by phosphorylation , the substitutions were combined to determine whether proteins with multiple substitutions would be more active .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9367763" target="_blank">9367763</a>]<br></li>
<li>Combining constitutive amino acid substitutions in the N-terminal domain of NtrC increases activity Combinations of constitutive amino acid substitutions in helices 4 and 5 of the receiver-domain of NtrC gave increased activity ( Figure 2b ) , as did phosphorylation of proteins carrying the individual substitutions ( not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9367763" target="_blank">9367763</a>]<br></li>
<li>Plasmid pJES779 ( K. Klose , personal communication ) , which was used for the construction of all mutant forms of NtrC carrying multiple amino acid substitutions in its N-terminal domain , was made by site-directed mutagenesis of pJES608 using the oligonucleotide : 50-GCGCATTCCAATCTAGATGCC-30 , which introduces the D86N substitution and an XbaI site into pJES608 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9367763" target="_blank">9367763</a>]<br></li>
<li>In order to understand the structural basis for activation of NtrC , 15N-1H HSQC spectra of mutant forms of the N-terminal domain ( NtrCN-term ) carrying constitutive amino acid substitutions were acquired and compared to the spectrum of the wild-type domain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9367763" target="_blank">9367763</a>]<br></li>
<li>Transcriptional activation by NtrC containing amino acid substitutions in its N-terminal domain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9367763" target="_blank">9367763</a>]<br></li>
<li>Plasmid constructions : strategy for construction of plasmids encoding mutant forms of NtrC containing constitutive amino acid substitutions in its N-terminal domain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9367763" target="_blank">9367763</a>]<br></li>
<li>Within the PS1BH clade , chelatases , MoxR , YifB , McrB , Dynein-midasin , NtrC , and MCMs form a monophyletic assembly deﬁned by a distinct insert in helix-2 of the conserved ATPase core , and additional helical segment between the core ATPase domain and the C-terminal a-helical bundle .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15037234" target="_blank">15037234</a>]<br></li>
<li>Recently , limited proteolysis combined with mass spectrometry identified a linker called the PASE-linker located beyond the highly conserved core region of NtrC and immediately before the region that forms the four-helix bundle of its C-terminal domain ( 32,33 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11923293" target="_blank">11923293</a>]<br></li>
<li>Keener , J. , and Kustu , S. ( 1988 ) Protein klnase and phospho proteín phosphatase activities of nitrogen regulatory proteins NTRB and NtrC of enteric bacteria : roles 01 conserved amino terminal domain of NTRC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1447980" target="_blank">1447980</a>]<br></li>
				<label class="DOM_collapse_similarity" for="dom_similarity_135">
Protein kinase and phosphoprotein phosphatase activities of nitrogen regulatory proteins NtrB and NtrC of enteric bacteria : roles of conserved amino terminal domain of NtrC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11325944" target="_blank">11325944</a>]<br></label>
<input id="dom_similarity_135", type="checkbox">
<div class="DOM_similar">
<li>Protein kinase and phosphoprotein phosphatase activities of nitrogen regulatory proteins NtrB and NtrC of enteric bacteria : roles of the conserved N-terminal domain of NtrC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14996819" target="_blank">14996819</a>]<br></li></div>
<li>Hence , UhpA only has an N-terminal motif ( about 120 amino acids ) in common with the OmpR group and the NtrC group of regulatory proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3042748" target="_blank">3042748</a>]<br></li>
<li>Members of this subfamily have an amino-terminal phosphorylation domain characteristic of so-called two-component regulators , such as OmpR , CheY , PhoB , and NtrC , and a carboxyl-terminal domain conserved among many transcriptional activators , including LuxR and MalT .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1569008" target="_blank">1569008</a>]<br></li>
<li>Sequence similarities within the amino-terminal domain place UhpA in the family of regulatory proteins which includes OmpR , CheY , PhoB , and NtrC ( 25 , 34 ) , whereas sequence relatedness in the carboxyl-terminal domain indicates that UhpA is a member of the UhpA-LuxR family of transcriptional activators ( 12 , 35 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1569008" target="_blank">1569008</a>]<br></li>
<li>Indeed , when comparing HydG with NtrC ( Fig. 6 ) , it was found that the absolute homology between residues 1 and 60 only 21 % , whereas was the homology between residues and 50 % , 61 122 was another indication that the essential region in the B domain is located somewhat more distal from the N terminus .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2666400" target="_blank">2666400</a>]<br></li>
<li>( MBP is fused to the N terminus of NtrC ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10960104" target="_blank">10960104</a>]<br></li>
<li>Chemical cleavage of NtrC at cysteine residues .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10960104" target="_blank">10960104</a>]<br></li>
<li>Effect of overproduced HycA on the transcriptional activation Considering the fact that residues 131 and 132 of NtrC were assigned to the linker region between domain A and C [ 46 ] we chose to separate FhlA between residue 378 and 379 for further investigations .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10848985" target="_blank">10848985</a>]<br></li>
<li>In the conserved domain of the reguiator NtrC there are three candidate Asp residues which may serve as the phosphoryl group acceptor ( Weiss and Magasanik , 1988 ; Keener and Kustu , 1988 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2201868" target="_blank">2201868</a>]<br></li>
<li>Such conformational dynamics may help NtrC assemble into a full hexamer or increase the ability of its ATPase domain to interact with the factor at the other side of the ring ( Fig. 4B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16751184" target="_blank">16751184</a>]<br></li>
<li>We previously studied phosphorylation-dependent conformational changes in 4 of the receiver-domain of NtrC by a derivatizing proteins carrying single cysteine substitutions in this helix with a cysteine-speciﬁc nitroxide spin label and monitoring the mobility of the nitroxide side chain so introduced by electron paramagnetic resonance ( EPR ) spectroscopy ( 12 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10960104" target="_blank">10960104</a>]<br></li>
<li>The NMR structure of NtrC shows a displacement of β4 , β5 , αC , and αD away from the active site , thereby exposing a hydrophobic surface that is proposed to allow transmission of the activation signal to the effector domain ( 206 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10966457" target="_blank">10966457</a>]<br></li>
<li>The possibility was raised by the fact that PspA functions in trans upon the AAAþ domain of PspF common to all s54 activators and the reported inﬂuence of PspA upon activation by NtrC .11 We , on the other hand , were unable to show any negative regulation of the AAAþ central domain of NifA nor NtrC by PspA ( Figures 2 ( c ) and 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12079332" target="_blank">12079332</a>]<br></li>
<li>The activation domain of AtoC , like that of NtrC , belongs to the AAA + ( for `` ATPases associated with diverse cellular activities '' ) superfamily [ 38 ] , members of which participate in diverse cellular functions ( [ 39 ] and references therein ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17537579" target="_blank">17537579</a>]<br></li>
<li>These include the CheY-like receiver-domain in NtrC , the GAF-domain in FhlA , the ACT and PAS domains in TyrR , and the 4-vinyl reductase ( 4VR ) domain in XylR ( Fig. 5 ; Anantharaman et al. , 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15037234" target="_blank">15037234</a>]<br></li>
<li>Discussion Constitutive substitutions in the receiver-domain of NtrC create an active state Like forms of NtrC carrying the D54E substitution at the site of phosphoryation ( Klose et al. , 1993 ) , mutant forms carrying the D86N ( Flashner et al. , 1995 ) , A89T or V115I substitutions activate transcription in vitro without being phosphorylated , indicating that they are in an active state .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9367763" target="_blank">9367763</a>]<br></li>
<li>This confirms that the deleted Cterminal domain is the major dimerization domain of NtrC ( Porter et al. , 1993 ; Shiau et al. , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7641688" target="_blank">7641688</a>]<br></li>
<li>There is a quantitative in vitro assay for activation of the output domain of NtrC with highly puri ® ed components ( Wedel et al. , 1990 ; Weiss et al. , 1991 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9367763" target="_blank">9367763</a>]<br></li>
<li>In a study of constitutive NtrC forms transcriptional activation and ATP hydrolysis were analyzed in vitro for the N-NtrCS160F mutant which is lacking the N-terminal receiver-domain and has a point mutation in the central activation domain ( Flashner et al. , 1995 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9600853" target="_blank">9600853</a>]<br></li>
<li>First , purified hybrid proteins containing the receiver-domain of NtrC ( kN-NtrCRO and XN-NtrCR ) were phosphorylated in vitro in the presence of NtrB and ATP to the same extent as wild-type NtrC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7641688" target="_blank">7641688</a>]<br></li>
<li>The isolated receiver-domain of NtrC is a constitutive dimer independent of phosphorylation To determine whether the receiver-domain or the output domain of NtrC is responsible for the dimerization of XN-NtrCRO , in our next experiment we fused each domain separately to the DNA-binding domain of X repressor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7641688" target="_blank">7641688</a>]<br></li>
<li>Surprisingly , the hybrid protein carrying the DNA-binding domain of X repressor and the receiver-domain of NtrC ( XN-NtrCR ) , behaved as a constitutive dimer independent of the degree of phosphorylation , leading to -72 % repression of XPR-lacZ expression in the experiment in both Figure 3A and B. DNA-binding properties of X,-NtrCRO and r-NtrCR To corroborate these results , we purified XN-NtrCRO and XN-NtrCR and studied the DNA-binding properties of the phosphorylated and unphosphorylated proteins with a nitrocellulose filter binding assay in vitro , using a synthetic oligonucleotide carrying the operator site XOR1 of XPR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7641688" target="_blank">7641688</a>]<br></li>
<li>NtrC lacking the dimerization domain is a monomer and dimerizes upon phosphorylation AN-NtrC lacking the C-terminal domain ( AN-NtrCRO ) repressed XPR to 35 % in its unphosphorylated form ( Figure 3A and B ) , indicating that unphosphorylated kN-NtrCRO is a monomeric protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7641688" target="_blank">7641688</a>]<br></li>
<li>We therefore deleted next the C-terminal dimerization domain of XN-NtrC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7641688" target="_blank">7641688</a>]<br></li>
<li>We infer that PhoB dimerizes upon phosphorylation , as does XN-NtrCRO lacking the C-terminal constitutive dimerization domain of NtrC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7641688" target="_blank">7641688</a>]<br></li>
<li>The molecular mass of these truncated proteins determined with size-exclusion chromatography confirms that NtrC lacking the C-terminal domain ( NtrCRO ) is a monomer , whereas the isolated receiver-domain of NtrC ( NtrCR ) is a dimer ( not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7641688" target="_blank">7641688</a>]<br></li>
<li>We fused the DNA-binding domain of X repressor together with its flexible linker region to NtrC ( kN-NtrC ) , to NtrC lacking the C-terminal domain ( kN-NtrCRO ) , to the receiver module of NtrC ( kN-NtrCR ) or to the output domain of NtrC ( Figure 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7641688" target="_blank">7641688</a>]<br></li>
<li>To test this , we overexpressed and purified , in independent experiments , NtrC derivatives consisting of receiver and output domain ( NtrCRO ) and the receiver-domain of NtrC alone ( NtrCR ) , not fused to the N-terminal domain of X repressor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7641688" target="_blank">7641688</a>]<br></li>
<li>Our structures , together with previous biochemical data showing that the activated receiver-domain of one subunit is in contact with the ATPase domain of a second , nonidentical subunit , offer an explanation for the positive mode of regulation in NtrC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16751184" target="_blank">16751184</a>]<br></li>
<li>One domain ( residues 144 to 345 ) shows extensive similarity to NifA/NtrC regulators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8246840" target="_blank">8246840</a>]<br></li>
<li>The model for activated NtrC presented here shows residues D149 , F151 , R152 , and R156 of helix N facing helix 4 of the receiver-domain , directly contacting residues L87 , D88 , A90 , and V91 ( see Supplementary Fig .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16751184" target="_blank">16751184</a>]<br></li>
<li>To explore the site of intermonomer interaction in more detail , we have placed single cysteine residues at 10 locations within the ﬁrst long a-helix in the central domain of NtrC , which has been postulated to be responsible for communication between the receiver-domain and the remainder of the central domain ( 25 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10960104" target="_blank">10960104</a>]<br></li>
<li>The receiver-domain of NtrC is connected to its central output domain , the only portion of the molecule for which no structural information is available , via a ﬂexible and protease-sensitive linker ( ; 16 residues ) ( 6 , 15 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10960104" target="_blank">10960104</a>]<br></li>
<li>DISCUSSION In the absence of structural information on either the phosphorylated receiver-domain of NtrC or its central output domain , we derivatized single cysteine residues at strategic locations in the receiver-domain with an Fe-EDTA chelate to explore the pathway of signal transduction between domains .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10960104" target="_blank">10960104</a>]<br></li>
<li>When a speciﬁc aspartate residue in the receiver-domain of NtrC is phosphorylated , the dimeric protein forms an oligomer that is capable of ATP hydrolysis and transcriptional activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10960104" target="_blank">10960104</a>]<br></li>
<li>Thus , as is true for NtrC , the phosphorylated receiver-domain of CheB apparently forms a new contact with the remainder of the protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10960104" target="_blank">10960104</a>]<br></li>
<li>In contrast , for the spatial arrangement in the phosphorylated NtrC complex suggested here the interaction between a receiver-domain and one or even two activation domains of different NtrC dimers is easily possible .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9600853" target="_blank">9600853</a>]<br></li>
<li>In this conformation higher order NtrC complexes can be stabilized by interaction between the phosphorylated receiver-domain and the central activation domain of different NtrC dimers .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9600853" target="_blank">9600853</a>]<br></li>
<li>Furthermore , the model presented in Figure 8 suggests that interactions occur between the phosphorylated receiver-domain and the central activation domain of different NtrC dimers .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9600853" target="_blank">9600853</a>]<br></li>
<li>It is known that the phosphorylated receiver-domain of NtrC must act positively on its central domain , because removing the receiver-domain does not substitute for phosphorylation ( 10 , 44 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10960104" target="_blank">10960104</a>]<br></li>
<li>A chemical protein cleavage method was used to investigate signal propagation from the phosphorylated receiver-domain of NtrC , which acts positively , to its central output domain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10960104" target="_blank">10960104</a>]<br></li>
<li>The results are commensurate with the hypothesis that a-helix 4 of the phosphorylated receiver-domain of NtrC interacts with the beginning of the central domain for signal propagation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10960104" target="_blank">10960104</a>]<br></li>
<li>In contrast to fusion proteins carrying receiver and output domain of NtrC or PhoB , the isolated receiver modules in both XN-NtrCR and XN-PhoBR , but not the fusion protein XN-NtrCo carrying only the output domain of NtrC , behaved as constitutive dimers independent of phosphorylation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7641688" target="_blank">7641688</a>]<br></li>
<li>In the absence of phosphorylation , the iron chelate tethered at positions 86 and 89 of NtrC cleaved at two positions within the receiver-domain of the protein ( Fig. 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10960104" target="_blank">10960104</a>]<br></li>
<li>They imply that the phosphorylation-dependent interdomain and intermolecular interactions between the receiver-domain of one subunit and the output domain of its partner subunit in an NtrC dimer precede -- and may give rise to -- the oligomerization needed for transcriptional activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10960104" target="_blank">10960104</a>]<br></li>
<li>Since both domains are connected via a ¯ exible linker it appears unlikely that phosphorylation of the receiver-domain would change the conformation of the activation domain in the same NtrC monomer .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9600853" target="_blank">9600853</a>]<br></li>
<li>Cleavage occurred outside the receiver-domain itself and on the partner subunit of the derivatized monomer in an NtrC dimer .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10960104" target="_blank">10960104</a>]<br></li>
<li>Ribbon diagram of the receiver-domain of NtrC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9367763" target="_blank">9367763</a>]<br></li>
<li>Phosphorylation of the NtrC receiver-domain is shown by yellow or red spots in the receiver-domain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9600853" target="_blank">9600853</a>]<br></li>
<li>High resolution solution structure of the beryllofluoride-activated NtrC receiver-domain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16751184" target="_blank">16751184</a>]<br></li>
<li>Three dimensional solution structure of the N-terminal receiver-domain of NtrC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16751184" target="_blank">16751184</a>]<br></li>
<li>Consequently , additional interactions between the receiver or output domain of NtrC , triggered by phosphorylation , can not be detected in XN-NtrC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7641688" target="_blank">7641688</a>]<br></li>
<li>The isolated receiver-domain of NtrC appears to remain monomeric even after phosphorylation ( 16 , 40 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10960104" target="_blank">10960104</a>]<br></li>
<li>This reaction is dependent on ATP hydrolysis , most likely catalyzed by the central domain of NtrC ( Weisset al. , 1991 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8098993" target="_blank">8098993</a>]<br></li>
<li>Effects of PspA on NtrC and NifA ( central domain ) - dependent sigma isomerisation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12079332" target="_blank">12079332</a>]<br></li>
<li>These results are in contrast to the case of NtrC , in which the central catalytic domain by itself is not active ( 39 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8606168" target="_blank">8606168</a>]<br></li>
<li>It has been proposed that the interaction of the phosphorylated N-terminal domain of NtrC with the central domain may be with a long a-helix that precedes the mononucleotide-binding fold ( 25 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10960104" target="_blank">10960104</a>]<br></li>
<li>In contrast to NtrC , the ligand binding sites that promote the higher order oligomerization of TyrR are located solely within the central domain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11923293" target="_blank">11923293</a>]<br></li>
<li>The other domain is homologous to a part of the NifA and NtrC proteins , which are activators of a54-dependent transcription , possibly connecting it to regulation of nitrogen metabolism .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8246840" target="_blank">8246840</a>]<br></li>
<li>The central domain of TyrR shares extensive homology with the central domains of the transcription activators belonging to the NtrC family ( Morett and Segovia , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12207706" target="_blank">12207706</a>]<br></li>
<li>DISCUSSION The central domain of TyrR links it both structurally and functionally with the large family of bacterial transcriptional regulators typified by NtrC and NifA ( 1,3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11923293" target="_blank">11923293</a>]<br></li>
<li>This domain differs from those of NtrC , DctD , and other members of the family of proteins with which TyrR shares regions of homology ( 53 ) , and little is known about its function .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8449883" target="_blank">8449883</a>]<br></li>
<li>Activator proteins in the NtrC family have a strongly conserved central domain that interacts directly with Es 54 and thus stimulates transcription from s - dependent 54 promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11532153" target="_blank">11532153</a>]<br></li>
<li>In addition , we describe a number of mutations in the central domain of TyrR and compare the roles of this region in TyrR and in the NtrC family of related proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8449883" target="_blank">8449883</a>]<br></li>
<li>By analogy with the NtrC case , it is reasonable to suggest that the role of the N-terminal domain of the TyrR protein is to originate transcriptional regulatory signals thatare transmitted via the second domain to anoperator recognition region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8444880" target="_blank">8444880</a>]<br></li>
<li>Another rel-ated regulatory protein , NtrC completely lacks the B domain , but shows a 20 % ideintity to NarL with respect to its A domain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2648330" target="_blank">2648330</a>]<br></li>
<li>This contention is supported by the phenotype of mutants altered in other regulators of the NtrC family : DctD and some constitutive mutants of NtrC function in the absence of the N-terminal domain which is the site for phosphorylation , and XylR ( 10 ) and DmpR ( 40 ) become constitutively active in ATP hydrolysis and transcriptional activation when the binding domain has been removed ( 13 , 17 , 27 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8981978" target="_blank">8981978</a>]<br></li>
<li>This indicated that none of the substitutions greatly perturbed the active site in the N-terminal domain or disrupted phosphorylation-induced activation of the remainder of NtrC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9367763" target="_blank">9367763</a>]<br></li>
<li>In NtrC it is induced by phosphorylation of the N-terminal domain and required for ATPase activity that is coupled to transcriptional activation ( 46 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11923293" target="_blank">11923293</a>]<br></li>
<li>Neither NtrC , which contains a single cysteine in a5 of the N-terminal domain , nor Fe NtrCS160C , which contains a single cysteine at the beginning of the central domain of NtrC , yielded more than small amounts of cleavage products and these were unchanged by phosphor-ylation ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10960104" target="_blank">10960104</a>]<br></li>
<li>The iron chelate reagent Fe-BABE was conjugated onto unique cysteines introduced into the N-terminal domain of NtrC , and the conjugated proteins were subjected to Fe-dependent cleavage with or without prior phosphor-ylation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10960104" target="_blank">10960104</a>]<br></li>
<li>RESULTS Phosphorylation-dependent chemical cleavage of mutant NtrC proteins by Fe-EDTA chelates tethered to single cysteines in a4 of the N-terminal domain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10960104" target="_blank">10960104</a>]<br></li>
<li>The major dimerization determinants of the nitrogen regulatory protein NtrC from enteric bacteria lie in its carboxy-terminal domain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8981978" target="_blank">8981978</a>]<br></li>
<li>The proposed oligomerization determinants of NtrC are located in the central domain ( Porter et al. , 1995 ) , while the major dimerization determinants lie in its C-terminal domain ( Klose et al. , 1994 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9878422" target="_blank">9878422</a>]<br></li>
<li>The major dimerisation determinants of NtrC are located in its carboxy-terminal domain , and indirect evidence points to oligomerization determinants being located in the central domain ( 46,48 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11923293" target="_blank">11923293</a>]<br></li>
<li>The N-terminal domain contains a highly conserved aspartyl resi-due ( Asp-54 in UhpA ) , which is the site of phosphorylation in homologous proteins , CheY and NtrC ( 9 , 10 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8999880" target="_blank">8999880</a>]<br></li>
<li>It is not a part of a two-protein component system and it lacks the amino-terminal domain that is present on NtrC and DctO .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1943694" target="_blank">1943694</a>]<br></li>
<li>The phosphorylated residue in NtrC is an aspartate ( 49 ) located in the amino-terminal domain ( 22 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2649492" target="_blank">2649492</a>]<br></li>
<li>Forexample , the N-terminal domain of NtrC contains a residue ( Asp64 ) that is phosphorylated in a reaction catalyzed by NtrB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8444880" target="_blank">8444880</a>]<br></li>
<li>The D86C substitution in the N-terminal domain of NtrC , which was derivatized with Fe-BABE , is indicated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10960104" target="_blank">10960104</a>]<br></li>
<li>The mobility of the MBP -- N-terminal protein ( a protein containing only MBP and the N-terminal domain of NtrC ) is indicated by an arrow .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10960104" target="_blank">10960104</a>]<br></li>
<li>The amino ( N ) - terminal receiver-domain of NtrC modulates the function of its adjacent output domain , which activates transcription by the s54 holoenzyme .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10960104" target="_blank">10960104</a>]<br></li>
<li>The N-terminal domain of NtrC , which acts positively on the remainder of the protein , is homologous to a large family of signal transduction domains called receiver-domains .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9367763" target="_blank">9367763</a>]<br></li>
<li>The NtrC family has 2 effector domains C-terminal to the receiver-domain , and ca. 460 amino acids .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8780507" target="_blank">8780507</a>]<br></li>
<li>Furthermore , conserved aspartate residues in the NorR N-terminal signaling domain suggest the potential for phosphorylation by a sensor histidine-kinase similar to that described for the NtrB/NtrC pair ( 43 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12529359" target="_blank">12529359</a>]<br></li>
<li>Bands 1 and 5 , whose combined size is close to that of full-length NtrC , provided evidence for a phosphorylation-de-pendent conformational change in NtrC that affected a4 of its N-terminal domain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10960104" target="_blank">10960104</a>]<br></li>
<li>Examples include NtrC , which is activated by phosphorylation of N-terminal Asp54 and NifA , which is negatively regulated by NifL through speciﬁc interactions with the NifA N-terminal domain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12079332" target="_blank">12079332</a>]<br></li>
<li>Evidence that the phosphorylation-dependent cleavage occurs outside the N-terminal domain of NtrC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10960104" target="_blank">10960104</a>]<br></li>
<li>To provide further evidence that the phosphorylation-dependent cleavage of NtrCD86CFe occurred outside the N-terminal domain of the protein , we compared the size of band 1 , the N-terminal cleavage product , to that of the intact N-terminal domain ( residues 1 to 124 ) of NtrC ( MBP fusion proteins in both cases ) ( 12 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10960104" target="_blank">10960104</a>]<br></li>
<li>Bantscheff , M. , Weiss , V. & Glocker , M.O. ( 1999 ) Identification of linker regions and domain borders of the transcription activator protein NtrC from Escherichia coli by limited proteolysis , in-gel digestion , and mass spectrometry .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10848985" target="_blank">10848985</a>]<br></li>
				<label class="DOM_collapse_similarity" for="dom_similarity_226">
Yang XF , Ji Y , Schneider BL , Reitzer L. Phosphorylation-independent dimer-dimer interactions by the enhancer-binding activator NtrC of Escherichia coli : a third function for the C-terminal domain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23531166" target="_blank">23531166</a>]<br></label>
<input id="dom_similarity_226", type="checkbox">
<div class="DOM_similar">
<li>Phosphorylation-independent dimer dimer interactions by the enhancer-binding activator NtrC of Escherichia coli : a third function for the C-terminal domain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659676" target="_blank">15659676</a>]<br></li>
<li>Phosphorylation-independent dimer-dimer interactions by the enhancer-binding activator NtrC of Escherichia coli : a third function for the C-terminal domain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17616594" target="_blank">17616594</a>]<br></li></div>
<li>( vi ) From the sedimentation data a model was derived in which the NtrC dimer adopts a V shaped structure with the DNA-binding domains being located at the bottom and the two receiver-domains at the top of the V .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9600853" target="_blank">9600853</a>]<br></li>
<li>To derive a model of the NtrC dimer we started with the simplest conformation and assumed that the four NtrC domains form a linear array of beads with the above dimensions in the order given by the amino acid sequence , i.e. receiver , Q-linker , activator and DNA-binding domain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9600853" target="_blank">9600853</a>]<br></li>
<li>Plasmid pgln31 ( Weglenski et al. , 1989 ) was used as a template for the NtrC domains , plasmid pHA3 ( Hartmann and Boos , 1993 ) as a template for the PhoB domains and E.coli chromosomal DNA for the amplification of the receiver-domain of CheB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7641688" target="_blank">7641688</a>]<br></li>
<li>We fused various domains of the response regulators NtrC , PhoB or CheB to the DNA-binding domain of X repressor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7641688" target="_blank">7641688</a>]<br></li>
<li>Using the N-terminal-DNA-binding-domain of X repressor as a reporter domain for dimerization in vivo , examined we the interaction between receiver and output domains of NtrC , PhoB and CheB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7641688" target="_blank">7641688</a>]<br></li>
<li>Results Activities of NtrC proteins carrying single constitutive amino acid substitutions Transcriptional activation by maltose-binding protein ( MBP ) fusions to NtrC carrying single constitutive amino acid substitutions in its N-terminal domain was assessed in a single-cycle transcription assay .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9367763" target="_blank">9367763</a>]<br></li>
<li>After mutagenesis , the portion of the DNA containing the substitutions was subcloned into vectors for expression of maltose binding protein fusions to NtrC , MBP-fusion , or for expression of the isolated N-terminal domain , N-terminus .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9367763" target="_blank">9367763</a>]<br></li>
<li>Since it has been shown that dimerization of Ntr is mediated by the C-terminal DNA-binding domain ( Klose et al. , 1994 ) , we derived the NtrC dimer structure by rotating two monomers which contacted each other at the DNA-binding domain until the calculated s and D values matched the measured ones ( Table 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9600853" target="_blank">9600853</a>]<br></li>
<li>Finally , several independent fusions of the AAA + domain with different versions of the DNA-binding HTH domain are seen in the NtrC , MCM , Cdc6 , DnaA , and RuvB families ( Fig. 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15037234" target="_blank">15037234</a>]<br></li>
<li>Next , the NMR structures of the NtrC BeF − 3 - bound receiver-domain ( PDB entry 1KRW ) and 3-Ala DNA-binding domain ( PDB entry 1NTC ) were positioned in the remaining density by hand .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16751184" target="_blank">16751184</a>]<br></li>
<li>Solution structure of the DNA-binding domain of NtrC with three alanine substitutions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16751184" target="_blank">16751184</a>]<br></li>
<li>Recently , the structure of the DNA-binding domain of a mutated NtrC was solved by NMR spectroscopy [ 25 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10848985" target="_blank">10848985</a>]<br></li>
<li>The sequenced region of OrfXB corresponds to ( i ) part of the transcription activator domain of NtrC , which interacts with o - ̂ ' * containing RNA polymerase , and ( ii ) a C-terminal DNA-binding domain ( Drummond etai .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8412694" target="_blank">8412694</a>]<br></li>
<li>Members of the NtrC in subfamily contain , addition to this phosphorylation a domain , large central activation domain consensus containing nucleotide-binding motifs and a carboxyl-terminal domain necessary for specific DNA-binding activity ( 34 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1569008" target="_blank">1569008</a>]<br></li>
<li>The C-terminal DNA-binding domain of NtrC is not required for the ATPase activity of NtrC , since phosphoryl-ated XN-NtrCRO lacking the C-terminal domain showed ATPase activity in vitro ( Figure 5 ) ; however , phosphorylation was absolutely required for activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7641688" target="_blank">7641688</a>]<br></li>
<li>In response to nitrogen limitation , the ATPase activity of NtrC is positively regulated by phosphorylation of its N-terminal domain by the NtrB kinase and stimulated by DNA-binding ( Weiss et al. , 1991 , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17185540" target="_blank">17185540</a>]<br></li>
<li>In vivo the NtrC protein is activated under conditions of nitrogen limitation ( Keener & Kustu , 1988 ; Ninfa & Magasanik , 1986 ; Weiss & Magasanik , 1988 ) as a consequence of phosphorylation of the aspartate residue at position 54 ( Sanders et al. , 1992 ) in its amino ( N ) - terminal receiver-domain ( Drummond et al. , 1986 ; Keener & Kustu , 1988 ; Ninfa & Magasanik , 1986 ; Weiss & Magasanik , 1988 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9367763" target="_blank">9367763</a>]<br></li>
<li>From previous stu-dies it has been concluded that the E. coli NtrC monomer and the closely related protein from Sal-monella consists of four domains ( Fiedler & Weiss , 1995 ; North et al. , 1996 ; Osuna et al. , 1997 ) : The amino-terminal receiver-domain ( residues 1 to 120 ; Volkman et al. , 1995 ) , a ¯ exible glutamine rich lin-ker ( residues 121 to 133 ) , the central activation domain ( residues 134 to 371 ; Osuna et al. , 1997 ) , and the C-terminal DNA-binding domain that is homologous to FIS protein ( residues 372 to 469 ; Kostrewa et al. , 1991 , 1992 ; North et al. , 1993 ; Pan et al. , 1996 ; Yuan et al. , 1991 ) ( note that the E. coli NtrC sequence given by North et al. ( 1993 ) has been revised by Blattner et al. ( 1997 ) ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9600853" target="_blank">9600853</a>]<br></li>
<li>NtrC is activated by phoshorylation of the aspartate 54 residue located in the N-terminal receiver-domain causing oligomerisation , and DctD by its phosphorylated receiver-domain relieving inhibition imposed by the non-phosphorylated recei-ver domain upon the central AAAþ domain ( Figure 1 ) .24 Some activators of s54-dependent transcription are regulated by the binding of a small ligand to the N-terminal domain , such as binding xylene to XylR .24 In contrast , other s54 activators are regulated through their N-terminal domains being targeted by a partner second protein which imposes negative control .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12079332" target="_blank">12079332</a>]<br></li>
<li>NtrC is composed of three domains ( 23 , 27 ) : the N-terminal receiver-domain ( ; 124 res-idues ) , which contains the site of phosphorylation , aspartate 54 ; a central output domain ( ; 240 residues ) , which is directly responsible for activation of transcription by the s54 holoenzyme form of RNA polymerase ; and a C-terminal DNA-bind-ing domain ( ; 90 residues ) , which carries the major dimerization determinants for the protein and mediates binding to transcriptional enhancers .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10960104" target="_blank">10960104</a>]<br></li>
				</div></ul><br>
			<label class="collapse_EVO" for="_3"><font size="5px">NtrC evolution</font></label>
				<input id="_3" type="checkbox">
				<div class="EVO"><br><ul><li>Other common regulator families are ArsR ( 175 ) , AsnC ( 140 ) , Crp ( 233 ) , DeoR ( 17 ) , GntR ( 29 , 104 , 213 ) , IclR ( 214 ) , LacI ( 253 ) , LuxR/UphA ( 112 ) , LysR ( 111 ) , MarR ( 56 ) , MerR ( 107 ) , NtrC ( 189 ) , TetR ( 198 ) , YedF/YeeD/YhhP ( 14 ) , and YhdG/YjbN/YohI ( 15 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9409145" target="_blank">9409145</a>]<br></li>
<li>HydG showed considerable homology with respect to the TTA 0M ATC ACG CGC AAA ACG CTA TTG GCA AAA CTG TCG CGT TAG TTC TGC TGO 00r TMG Lau Gly Ibm Thr Ar ; Lys Thr Lau Lau Ala Lys Iou Ber Ar ; STP presumed DNA-binding consensus ( 90 % ) , as well with the other functional domains that can be discerned on NtrC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2666400" target="_blank">2666400</a>]<br></li>
<li>Abstract : Third International Conference on Molecular Biology of Hydrogenases , Troia 1991 , pp 63-64 Richaud P , Colbeau A , Toussaint B , Vignais PM ( 1991 ) Identification and sequence analysis of the hupR1 gene , which encodes a response regulator of the NtrC family required for hydrogenase expression in Rhodobacter capsulatus .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1482271" target="_blank">1482271</a>]<br></li>
<li>These DNA-binding domains can be subdivided into three major families , represented by OmpR , NarL , and NtrC ( 14 , 7 , and 4 members , respectively , in E. coli ) ( 4 , 63 , 133 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10966457" target="_blank">10966457</a>]<br></li>
<li>In E. coli , members of these homologous families have been identified in more than 10 different adaptive response systems ( for a review , see reference 38 ) , including chemotaxis ( CheA-CheY ) , nitrogen regulation ( NtrB-NtrC ) , phosphate regulation ( PhoR-PhoB ) , nitrate reductase regulation ( NarX-NarL ) , oxygen regulation ( ArcB-ArcA ) , and osmotic porin regulation ( EnvZ-OmpR ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8282700" target="_blank">8282700</a>]<br></li>
<li>Existing structural information on DctD and NtrC1 has been used to provide a model of how two-component signal transduction can regulate assembly of their AAA + ATPase domains ( Lee et al. 2003 ) , but this model fails to explain regulation in the closely related protein NtrC ( 40 % sequence identity , 60 % sequence similarity ) ( Lee 2000 ; Lee et al. 2000 ; Hastings et al. 2003 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16751184" target="_blank">16751184</a>]<br></li>
<li>The entire 20-amino acid region of TdcR displays 25 % and 35 % identity to the NtrC and Fis polypeptides which activate , respectively , a54-dependent promoters associated with nitrogen regulation ( Buikema et al. 1985 ; Drummond et al. 1986 ) and Hin-mediated D N A inversion in vivo and in vitro ( Johnson et al. 1988 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2573820" target="_blank">2573820</a>]<br></li>
<li>NtrC ( also called NRI ) is the transcriptional activator for the ntr regulon ( Magasanik , 1988 ) and belongs to the subfamily of activators for a ; 54-dependent promoters ( Kustu et al. , 1991 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7641688" target="_blank">7641688</a>]<br></li>
<li>JOURNAL OF BACTERIOLOGY , Jan. 1997 , p. 41 -- / 97 / $ 04.0010 Copyright q 1997 , American Society for Microbiology Characterization of fhlA Mutations Resulting in Ligand - Independent Transcriptional Activation and ATP Hydrolysis INGRID KORSA AND AUGUST BÖCK * Lehrstuhl für Mikrobiologie der Universität München , D-80638 Munich , Germany Received 19 August 1996/Accepted 21 October 1996 The FhlA protein belongs to the NtrC family of transcriptional regulators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8981978" target="_blank">8981978</a>]<br></li>
<li>The activation domain of AtoC , like that of NtrC , belongs to the AAA + ( for `` ATPases associated with diverse cellular activities '' ) superfamily [ 38 ] , members of which participate in diverse cellular functions ( [ 39 ] and references therein ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17537579" target="_blank">17537579</a>]<br></li>
<li>One domain ( residues 144 to 345 ) shows extensive similarity to NifA/NtrC regulators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8246840" target="_blank">8246840</a>]<br></li>
<li>NtrC , PhoB and CheB , the subjects of this study , belong to different subfamilies of response regulators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7641688" target="_blank">7641688</a>]<br></li>
<li>The similarities between the chemotaxis and the NtrB-NtrC systems suggest that other members of the family of the two-component regulatory system may transmit signals in an analogous manner .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3280143" target="_blank">3280143</a>]<br></li>
<li>FleQ is homologous to NtrC of the two-component regulatory systems .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14638413" target="_blank">14638413</a>]<br></li>
<li>This is in contrast to the situation in the NtrB/NtrC family of two-component systems , in which signal transduction has been shown to involve phosphorylation of the regulator by the sensor ( Ninfa and Magasanik , 1986 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2691840" target="_blank">2691840</a>]<br></li>
<li>Pseudomonas aeruginosa AlgB , a two-component response regulator of the NtrC family , is required for algD transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8449883" target="_blank">8449883</a>]<br></li>
<li>One of these was the 35-40 % identity of the central and carboxyl-terminal domains of the TyrR protein ( a repressor for genes of aromatic amino acid biosynthesis and transport ) with a number of regulator proteins such as NtrC , NifA , XylR , FhIA , HydG and DctD , which activate expression from promoters ustng the E < r54 RNA polymerase holoenzyme ( Stock et ai , 1989 ; Schlensog and Bock , 1990 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1943694" target="_blank">1943694</a>]<br></li>
<li>This segment of the protein is 45 % homologous to the putative DNA-binding region ( helix-turn-helix motif ; 22 ) of the HydG and NtrC proteins ( Fig. 9 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2118503" target="_blank">2118503</a>]<br></li>
<li>The extent of homology of this segment of the FhaA protein with the NtrC and NifA proteins is considerably lower ( about 30 % ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2118503" target="_blank">2118503</a>]<br></li>
<li>The next region of about 24 amino acids ( region II ; amino acids 370 to 393 ) is about 60 % homologous with the HydG protein and is less homologous ( 46 % ) with the NtrC and NifA proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2118503" target="_blank">2118503</a>]<br></li>
<li>The next region ( IV ) of about 55 amino acids ( positions 606 to 660 ) , with homology of about 22 to 27 % o with the HydG , NtrC , and NifA proteins , is unique to the FhrA protein , although the four proteins probably diverged from the same or a similar ancestral protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2118503" target="_blank">2118503</a>]<br></li>
<li>The best-characterized protein of this family , NtrC , has been shown to possess ATPase activity that is phosphorylationdependent and stimulated by site-specific DNA-binding [ 41,48 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17537579" target="_blank">17537579</a>]<br></li>
<li>The N-terminal domain contains a highly conserved aspartyl resi-due ( Asp-54 in UhpA ) , which is the site of phosphorylation in homologous proteins , CheY and NtrC ( 9 , 10 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8999880" target="_blank">8999880</a>]<br></li>
<li>The same situation may wellapply to NtrC , DctD , NifA , and other members of this family of proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8444880" target="_blank">8444880</a>]<br></li>
<li>This domain differs from those of NtrC , DctD , and other members of the family of proteins with which TyrR shares regions of homology ( 53 ) , and little is known about its function .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8449883" target="_blank">8449883</a>]<br></li>
<li>The central-domain is similar to the 230 amino acid conserved ATP-binding domains of the sigma 54-interaction family , which includes NtrC and NifA that are involved in nitrogen and nitrogenase regulation respectively ( Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11923293" target="_blank">11923293</a>]<br></li>
<li>The sequence of TyrR protein from amino acids 223 to 434 is homologous to the central-domains of NtrC and NifA and a variety of other activator proteins from diverse organisms ( 20 , 53 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8449883" target="_blank">8449883</a>]<br></li>
<li>The central and C-terminal domains of TyrR do show certain homologies with members of the NtrC family of activator proteins , in which both a DNA-binding protein and a histidine protein kinase are required for activation ( 21 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7798138" target="_blank">7798138</a>]<br></li>
<li>The central-domain of TyrR contains an ATP-binding site and is homologous with the NtrC family of activator proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8449883" target="_blank">8449883</a>]<br></li>
<li>The central-domain of TyrR shares extensive homology with the central-domains of the transcription activators belonging to the NtrC family ( Morett and Segovia , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12207706" target="_blank">12207706</a>]<br></li>
<li>The N-terminal domain of NtrC , which acts positively on the remainder of the protein , is homologous to a large family of signal transduction domains called receiver domains .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9367763" target="_blank">9367763</a>]<br></li>
<li>The NtrC family has 2 effector domains C-terminal to the receiver domain , and ca. 460 amino acids .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8780507" target="_blank">8780507</a>]<br></li>
<li>Previous studies have shown that NtrC , a widely studied member of this family of activators , binds to its ELE and activates transcription through the assembly of large oligomers in which some of the dimers are not in direct contact with the DNA ( Wyman et al. , 1997 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15528672" target="_blank">15528672</a>]<br></li>
<li>One family includes OmpR , PhoB , SfrA , VirG , CheB , CheY , Spo0A , SpoOF , NtrC , and DctD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3042748" target="_blank">3042748</a>]<br></li>
<li>The match to the other proteins , although obvious , was weaker , ranging from about 25 % identity with NtrC and CheY to 19 % for OmpR , SfrA , and PhoB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3042748" target="_blank">3042748</a>]<br></li>
<li>Members of this subfamily have an amino-terminal phosphorylation domain characteristic of so-called two-component regulators , such as OmpR , CheY , PhoB , and NtrC , and a carboxyl-terminal domain conserved among many transcriptional activators , including LuxR and MalT .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1569008" target="_blank">1569008</a>]<br></li>
<li>The NarL structure is a model for other members of the FixJ or LuxR family of bacterial transcriptional activators , and possibly to the more distant OmpR and NtrC families as well .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8780507" target="_blank">8780507</a>]<br></li>
<li>Amino acid sequence comparisons showed that UhpA was homologous to a family of bacterial regulatory proteins , some of which act as transcriptional activators ( OmpR , PhoB , NtrC , and DctD ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3042748" target="_blank">3042748</a>]<br></li>
<li>Sequence similarities within the amino-terminal domain place UhpA in the family of regulatory proteins which includes OmpR , CheY , PhoB , and NtrC ( 25 , 34 ) , whereas sequence relatedness in the carboxyl-terminal domain indicates that UhpA is a member of the UhpA-LuxR family of transcriptional activators ( 12 , 35 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1569008" target="_blank">1569008</a>]<br></li>
<li>Interestingly , comparison of OrfXB with data base sequences revealed homology with the NtrC family of transcriptional activators ( Fig. 6B ) ( for reviews , see Kustu et ai , 1989 ; Thony and Hennecke , 1989 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8412694" target="_blank">8412694</a>]<br></li>
<li>Together , all these families are related to the McrB family , to the exclusion of the NtrC and MCM families ( Fig. 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15037234" target="_blank">15037234</a>]<br></li>
<li>PrpR is an NtrC family activator which interacts directly with RpoN RNA polymerase ( 17 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19633077" target="_blank">19633077</a>]<br></li>
<li>HbpR , a new member of the XylR/DmpR subclass within the NtrC family of bacterial transcriptional activators , regulates expression of 2-hydroxybiphenyl metabo-lism in Pseudomonas azelaica HBP1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14523117" target="_blank">14523117</a>]<br></li>
<li>Genetic evidence for activation of the positive transcriptional regulator XylR , a member of the NtrC family of regulators , by effector binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8981978" target="_blank">8981978</a>]<br></li>
<li>Cloning and nucleotide sequence of the gene encoding the positive regulator ( DmpR ) of the phenol catabolic pathway encoded by pVI150 and identiﬁcation of DmpR as a member of the NtrC family of transcriptional activators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8981978" target="_blank">8981978</a>]<br></li>
<li>PrpR of S. enterica belongs to the NtrC family of transcriptional activators and has been known to regulate the expression of the prpBCDE operon in response to propionate catabolism ( 32 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15805526" target="_blank">15805526</a>]<br></li>
<li>The regulators at the top of the polar ﬂagellar hierarchy belong to the NtrC family of s54-associated transcription activators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19969540" target="_blank">19969540</a>]<br></li>
<li>AtoC is a member of the NtrC -- NifA family of j54-RNA polymerase transcriptional activators [ 23 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16153782" target="_blank">16153782</a>]<br></li>
<li>FhlA belongs to the large NtrC family of transcriptional regulators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8981978" target="_blank">8981978</a>]<br></li>
<li>DISCUSSION The central-domain of TyrR links it both structurally and functionally with the large family of bacterial transcriptional regulators typified by NtrC and NifA ( 1,3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11923293" target="_blank">11923293</a>]<br></li>
				</div></ul><br>
			<label class="collapse_RP" for="_4"><font size="5px">Processes regulated by NtrC</font></label>
				<input id="_4" type="checkbox">
				<div class="RP"><br><ul><li>Doubling regulated regulated of altered time Identiﬁed function Class system ( HK/RR ) Relevant functions genes genes genes ( min ) in this experiment Strain name Mutation ntrol o o C N 1 4 5 27 BW25113 WT or little signiﬁcant alteration CreC/CreB 1 1 2 Catabolite repression ( Wanner , 1996 ) Drug resistance ( Kato et al. , 2000 ) BW26983 DE ( creABCD ) 154 29 2 3 5 EvgS/EvgA BW27869 DE ( evgAS ) 1291 29 2 6 8 YgiY/YgiX NarX/NarL BW27551 BW27864 DE ( ygiXY ) 1302 DE ( narXL ) 1316 27 29 5 4 9 Nitrate regulation ( Stewart and Rabin , 1995 ) Hexose phosphate uptake ( Kadner , 1995 ) Phosphate regulation ( Wanner , 1995 ) Copper response ( Munson et al. , 2000 ) Nitrate regulation ( Stewart and Rabin ,1995 ) UhpB/UhpA 6 3 9 BW27871 DE ( uhpAB ) 1322 28 PhoR/PhoB 1 8 9 BW24476 DE ( phoBR ) 580 28 CusS/CusR 2 7 BW28077 DE ( cusRS ) 1204 27 9 / NarP 9 BW27873 DnarP1312 33 12 3 YedV/YedW YfhK / YehU/YehT KdpD/KdpE DE ( yedWV ) 1298 DyfhK1328 DE ( yehUT ) 1324 DE ( kdpABCDE ) 1224 26 28 27 27 13 15 20 20 6 7 7 8 BW BW BW BW 27550 27872 27877 27564 13 10 7 10 Potassium transport ( Walderhaug et al. , 1992 ) Nitrogen regulation ( Porter et al. , 1995 ) NtrB/NtrC BW27880 DE ( ntrBC ) 1318 31 20 8 12 Signiﬁcant alteration BaeS/BaeR HydH/HydG ( Nagasawa et al. , 1993 ) Response to zinc and lead ( Leonhartsberger et al. , 2001 ) Hydrogen peroxide sensitivity ( Pernestig et al. , 2001 ) 14 11 BW27553 BW27867 DE ( baeSR ) 579 DE ( hydHG ) 1336 8 22 24 29 31 13 2 BarA / BW29434 DbarA1358 24 26 30 7 / FimZ BW28078 BW27558 Enterochelin DﬁmZ1214 22 27 29 31 28 27 4 PhoQ/PhoP Response to magnesium ( Kato et al. , 1999 ) ( Nagasawa et al. , 1993 ) Response to cell envelope stress ( De Wulf et al. , 1999 ) DE ( phoPQ ) 1244 20 16 12 16 BasS/BasR CpxA/CpxR BW27848 BW27559 DE ( basRS ) 1287 DE ( cpxRA ) 623 32 32 29 29 Enterochelin 12 20 22 17 RstB/RstA YojN ( Hpt ) BW27552 BW27866 DE ( rstAB ) 1278 DyojN1332 34 37 26 29 Enterochelin Capsular synthesis ( Takeda et al. , 2001 ) 14 Control of TMAO respiratory system ( Simon et al. , 1994 ) 38 24 TorS/TorR BW26423 26 DE ( torSTRCAD ) 518 a .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12366850" target="_blank">12366850</a>]<br></li>
<li>Amino acid biosynthesis Stress adaptation ( 0-3 ,827 ) RNAp Input ( 0-1 ,631 ) RNAp Input ( 0-1 ,547 ) RNAp Input ( 0-6 ,019 ) RNAp Input ( 0-3 ,827 ) RNAp N + ( 0-1 ,631 ) RNAp N + ( 0-1 ,547 ) RNAp N + ( 0-6 ,019 ) RNAp N + ( 0-3 ,827 ) RNAp N -- ( 0-1 ,631 ) RNAp N -- ( 0-1 ,547 ) RNAp N -- ( 0-6 ,019 ) RNAp N -- grxB mdtH hisG hisD yoeB hisL yefM yceB yajG bolA tig rem hokD relE ydfV flxA relB ( p ) ppGpp ( p ) ppGpp Transcription N stress charged tRNA uncharged tRNA ( 0-5 ,207 ) RNAp Input Ribosome Translation NtrB P ( 0-5 ,207 ) RNAp N + P NtrC ( 0-1 ,938 ) RNAp Input ( 0-5 ,207 ) RNAp N -- NtrC ( 0-1 ,938 ) RNAp N + ( p ) ppGpp transcription activation P σ54 RNAp nlpD rpoS relA ( 0-1 ,938 ) RNAp N ¬ ( 0-9 ,150 ) RNAp Input ( p ) ppGpp ATP + rpsA cmk GDP or GTP RelA 3 ' ( 0-9 ,150 ) RNAp N + RelA ( 0-2 ,163 ) RNAp Input 5 ' AMP + ( 0-9 ,150 ) RNAp N -- ( 0-2 ,163 ) RNAp N + ( p ) ppGpp rpoH ftsX ( 0-2 ,163 ) RNAp N ¬ ( 0-14 ,396 ) RNAp Input ( p ) ppGpp ( p ) ppGpp rpmH rnpA yidD dnaA Nucleotide metabolism ( 0-14 ,396 ) RNAp N + ppGpp degradation/synthesis ( 0-3 ,406 ) RNAp Input ( 0-834 ) ( 0-981 ) ( 0-14 ,396 ) RNAp N -- RNAp Input RNAp Input ( 0-3 ,406 ) RNAp N + ( 0-834 ) ( 0-981 ) RNAp N + RNAp N + nadB rseA rpoE ( 0-3 ,406 ) RNAp N ¬ ( 0-834 ) ( 0-981 ) RNAp N -- RNAp N -- apt yieP gmk rrsC rpoZ ybaN spoT Figure 3  NtrC couples the Ntr stress response and stringent-response in N-starved E. coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24947454" target="_blank">24947454</a>]<br></li>
<li>is TyrR E. coli ( P07604 ) , is TyrR Haemophilus influenzae ( P44694 ) , is PhhR Psedomonas putida ( Q52177 ) , is NifA Azotobacter Vinelandii ( P09570 ) , is XylR Psedomonas putida ( P21940 ) , is PilR Psedomonas aeruginosa ( Q00934 ) , is NtrC E. coli ( P06713 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11923293" target="_blank">11923293</a>]<br></li>
<li>Abstract : Third International Conference on Molecular Biology of Hydrogenases , Troia 1991 , pp 63-64 Richaud P , Colbeau A , Toussaint B , Vignais PM ( 1991 ) Identification and sequence analysis of the hupR1 gene , which encodes a response regulator of the NtrC family required for hydrogenase expression in Rhodobacter capsulatus .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1482271" target="_blank">1482271</a>]<br></li>
<li>NtrC apparently activated transcription of the putrescine-transport operon ( b0854 -- 57 ) , the b1006 -- 12 operon ( ORFs ) , the transport and catabolic operon for D-alanine-D-alanine ( b1483 -- 88 ) ( Fig. 2 ) , and the probable orni-thine aminotransferase operon ( b3073 ) , in addition to operons known or thought to be under its control ( Table 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11121068" target="_blank">11121068</a>]<br></li>
<li>The value of 2.5 was the minimum that yielded at least one gene in each operon known from previous biochemical and genetic studies to be under NtrC control in enteric bacteria [ nine operons containing 27 genes were : cbl ( b1987 ) , which encodes a regulator for sulfur metab-olism ( see below ) ; ompF ( b0929 ) , which encodes a large outermembrane pore ( see below ) ; and three operons of unknown function ( b1006 -- 12 , b1783 -- 84 , and b1932 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11121068" target="_blank">11121068</a>]<br></li>
<li>The cycA gene ( b4208 ) , which encodes a transporter for D-ala , is under Nac control and the ddp operon ( b1483 -- 88 ) , which encodes a transporter and a specific dipeptidase for - ala - - ala ( 10 ) , is under NtrC control .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11121068" target="_blank">11121068</a>]<br></li>
<li>The product of the b1006 gene was predicted to be a nucleobase-transporter ( 7 ) , and most operons under NtrC control are involved in transport and or catabolism of alternative N sources .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16540542" target="_blank">16540542</a>]<br></li>
<li>Finally , NtrC appeared to activate a group of genes ( b3504 -- 17 ) around the hdeAB ( H-NS-determined expression ) operon ( b3509 -- 10 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11121068" target="_blank">11121068</a>]<br></li>
<li>One of these was the 35-40 % identity of the central and carboxyl-terminal domains of the TyrR protein ( a repressor for genes of aromatic-amino-acid biosynthesis and transport ) with a number of regulator proteins such as NtrC , NifA , XylR , FhIA , HydG and DctD , which activate expression from promoters ustng the E < r54 RNA polymerase holoenzyme ( Stock et ai , 1989 ; Schlensog and Bock , 1990 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1943694" target="_blank">1943694</a>]<br></li>
<li>The narX and narL genes , as mapped in this region with the correct orientation ( 9 ) , encode proteins with apparent molecular weights of 62,000-66 ,000 and OmpR is involved in osmotic regulation of the synthesis of outer membrane proteins , and NtrC is involved in nitrogen control of several nitrogen assimilatory genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2657652" target="_blank">2657652</a>]<br></li>
<li>Effect of NtrA and NtrC proteins on the synthesis of the glycogen-biosynthesis enzymes in S-30 extracts .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2468650" target="_blank">2468650</a>]<br></li>
<li>For example , NtrC activates the 54-dependent glnAp2 promoter but represses the 54-dependent glnAp promoter in E. coli 1 ( 30 ) , while BvgA functions as both a transcriptional activator of multiple virulence factors and a repressor of genes required for motility and/or survival in several Bordetella species ( 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17616594" target="_blank">17616594</a>]<br></li>
<li>Control at the transcriptional level has been demonstrated for both inﬂux and efflux genes , by several speciﬁc ( ArgR , ArgP ) and more general ( NtrC , Lrp ) repressors and activators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19906180" target="_blank">19906180</a>]<br></li>
<li>Expression of the operon is elevated in a strain that carries an ntrB ( Con ) ( ntrB [ constitutive ] ) mutation , which increases transcription of all genes under NtrC control ( 59 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20400551" target="_blank">20400551</a>]<br></li>
<li>Our results reveal a far wider ranging control of genes occurs through a direct interface of the master Ntr stress response transcription activator NtrC with the bacterial stringent-response system .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24947454" target="_blank">24947454</a>]<br></li>
<li>NtrC activates relA transcription in N-starved E. coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24947454" target="_blank">24947454</a>]<br></li>
<li>Phosphorylation site of NtrC , a protein phosphatase whose covalent intermediate activates transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8098993" target="_blank">8098993</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10074086" target="_blank">10074086</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8226691" target="_blank">8226691</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7730282" target="_blank">7730282</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9367763" target="_blank">9367763</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15755952" target="_blank">15755952</a>]<br></li>
<li>NtrC also activates transcription of genes encoding regulatory proteins : ntrB and ntrC ( glnL and glnG , respectively ) , glnK , and nac ( 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11121068" target="_blank">11121068</a>]<br></li>
<li>The nitrogen assimilation control ( Nac ) protein activates transcription of s70-dependent genes , whereas NtrC activates transcription of s54-dependent genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11121068" target="_blank">11121068</a>]<br></li>
<li>The NtrC protein also activates expression of the nitrogen assimilatory control gene ( nac ) in Klebsiella aerogenes , which is known to regulate expression of glutamate synthase ( 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955321" target="_blank">8955321</a>]<br></li>
<li>against nitrogen limitation Nitrogen regulatory protein C ( NtrC ) of enteric bacteria activates transcription of genesyoperons whose products minimize the slowing of growth under nitrogen-limiting conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11121068" target="_blank">11121068</a>]<br></li>
<li>Glutamate at the site of phosphorylation of nitrogen-regulatory protein NtrC mimics aspartyl-phos-phate and activates the protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9422609" target="_blank">9422609</a>]<br></li>
<li>Strain MX902 carrying an insertion in the glnG ( ntrC ) gene ( glnG74 : : Tn5 ) , which encodes for the NRI protein ( NtrC ) , was used to determine GDH activity in N-excess or N-limiting conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9785451" target="_blank">9785451</a>]<br></li>
<li>Among genes that have been known to be under the control of NtrB/NtrC , only the glnP gene encoding glutamine-transport protein was downregulated in the DntrBC mutant .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12366850" target="_blank">12366850</a>]<br></li>
<li>Expression of the putA gene encoding proline dehydrogenase from Rhodobacter capsulatus is independent of NtrC regulation but requires an Lrp-like activator protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18586269" target="_blank">18586269</a>]<br></li>
<li>Substitutions at a single amino acid residue in the nitrogenregulated activator protein NtrC differentially influence its activity in response to phosphorylation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16751184" target="_blank">16751184</a>]<br></li>
<li>b , NtrC protein ( left curve ) and NtrC ES-1 complex ( right curve ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9600853" target="_blank">9600853</a>]<br></li>
<li>In E. coli , members of these homologous families have been identified in more than 10 different adaptive response systems ( for a review , see reference 38 ) , including chemotaxis ( CheA-CheY ) , nitrogen regulation ( NtrB-NtrC ) , phosphate regulation ( PhoR-PhoB ) , nitrate reductase regulation ( NarX-NarL ) , oxygen regulation ( ArcB-ArcA ) , and osmotic porin regulation ( EnvZ-OmpR ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8282700" target="_blank">8282700</a>]<br></li>
<li>To gain insight into the response of E. coli to environmental challenges at the expression level , global gene-expression analyses using DNA microarrays ( Rhodius et al. 2002 ; Khodursky et al. 2003 ) were widely used , e.g. to characterize the responses to exposure to hydrogen peroxide ( Zheng et al. 2001 ) or to elucidate the involved gene regulatory circuits , e.g. by the central regulators of nitrogen metabolism NtrC and Nac ( Zimmer et al. 2000 ) or by the sigma-factor of the general stress response σS ( Weber et al. 2005 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17273855" target="_blank">17273855</a>]<br></li>
<li>NtrC family activators are generally involved in nitrogen regulation in E. coli , since nitrogen assimilation consumes energy and intermediates of central me-tabolism ( 27 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19633077" target="_blank">19633077</a>]<br></li>
<li>As NtrC protein levels are controlled by the glnAp2 promoter , the dual regulatory role of the CRP -- cAMP complex ( i.e. the activation of s - dependent promoters involved in carbon 70 metabolism and the downregulation of s - dependent 54 promoters , in particular the glnAp2 promoter , involved in nitrogen assimilation ) might provide a novel and general regulatory linkage between carbon metabolism and nitrogen assimilation in E. coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11532153" target="_blank">11532153</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_33">
Atkinson , M.R. , Kamberov , E.S. , Weiss , R.L. , and Ninfa , A.J. ( 1994 ) Reversible uridylylation of the Escherichia coli PII signal-transduction protein regulates its ability to stimulate the dephosphorylation of the transcription factor Nitrogen Regulator I ( NRI or NtrC ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9720863" target="_blank">9720863</a>]<br></label>
<input id="rp_similarity_33", type="checkbox">
<div class="RP_similar">
<li>Atkinson , M.R. , Kamberov , E.S. , Weiss , R.L. and Ninfa , A.J. ( 1994 ) Reversible uridylylation of the Escherichia coli PII signal-transduction protein regulates its ability to stimulate the dephosphorylation of the transcription factor nitrogen regulator I ( NRI or NtrC ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7641688" target="_blank">7641688</a>]<br></li>
<li>Reversible uridylylation of the Escherichia coli PII signal-transduction protein regulates its ability to stimulate the dephosphorylation of the transcription factor nitrogen regulator I ( NRI or NtrC ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16740928" target="_blank">16740928</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10074086" target="_blank">10074086</a>]<br></li></div>
<li>Through a complex cascade of events , glutamine starvation leads to phosphorylation ( and activation ) of the transcriptional regulator NtrC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9495755" target="_blank">9495755</a>]<br></li>
<li>Biochemical characterization of the system that regulates glutamine synthetase expression has shown that gene-expression is stimulated in vitro by a cascade involving phosphorylation of the NtrC protein by the NtrB kinase ( Ninfa and Magasanik , 1986 ; Ninfa et al. , 1987 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3280143" target="_blank">3280143</a>]<br></li>
<li>A Nitroregulation Circuit in E. coli The NtrB-NtrC pathway regulates glutamine synthetase expression in response to changes in nitrogen availability .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8098993" target="_blank">8098993</a>]<br></li>
<li>PII regulates the ratio of phosphatase ( P ) to autokinase ( K ) activities in NtrB to adjust the phosphorylation state of NtrC , which in turn controls glutamine synthetase expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8098993" target="_blank">8098993</a>]<br></li>
<li>Among the scavenging regulons , Crp , which allows the use of alternative carbon sources , and the two component NtrB/NtrC system , which controls scavenging for nitrogen are by far important in practice .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21272324" target="_blank">21272324</a>]<br></li>
<li>For example , the regulators in the subfamily consisting of NtrC and DctD are involved in cellular response to low external levels of nitrogen and act with RpoN ( a54 ) to activate their target genes ( 34 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2404948" target="_blank">2404948</a>]<br></li>
<li>, Schneider , B.L. , Gu , W. , and Reitzer , L.J. ( 1992 ) Role of nitrogen regulator I ( NtrC ) , the transcriptional activator of glnA in enteric bacteria , in reducing expression of glnA during nitrogen-limiting growth .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9720863" target="_blank">9720863</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12453212" target="_blank">12453212</a>]<br></li>
<li>NtrC is transcriptional a a activator ( and in some cases a repressor ) of several operons involved in nitrogen assimilation ( nif , hut , and put ) and can be present in an active or inactive state ( 14 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2666400" target="_blank">2666400</a>]<br></li>
<li>Introduction Nitrogen regulatory protein C ( NtrC , also designated nitrogen regulator I or NRI ) from Escherichia coli is one of the activator proteins of E. coli RNA polymerase s54 holoenzyme ( RNAP s54 ) for a var-iety of genes that are involved in nitrogen utiliz-ation ( Magasanik , 1996 ; North et al. , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9600853" target="_blank">9600853</a>]<br></li>
<li>NtrC protein , a positive regulator for genes involved in nitrogen metabolism , was shown to be phosphorylated by NtrB protein , and the phosphorylated form was the active form for transcriptional activation of the genes involved in nitrogen-fixation ( Ninfa & Magasanik , 1986 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3054125" target="_blank">3054125</a>]<br></li>
<li>glnG encodes NtrC , the transcriptional activator of nitrogen assimilation and scavenging genes under conditions of nitro-gen-limited growth ( 41 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15629914" target="_blank">15629914</a>]<br></li>
<li>RpoN-containing polymerase always requires a transcriptional activator , a role played by nitrogen regulatory protein C ( NtrC ) under nitrogen-stress conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24474781" target="_blank">24474781</a>]<br></li>
<li>The operon also encodes a 2CST pathway , composed of NRII ( also known as NtrB and encoded by glnL ) and its cognate RR NRI ( also known as NtrC and encoded by glnG ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15755952" target="_blank">15755952</a>]<br></li>
<li>Regulation of Autophosphorylation of Escherichia coli Nitrogen Regulator II by the PII Signal Transduction Protein PENG JIANG AND ALEXANDER J. NINFA * Department of Biological Chemistry , University of Michigan Medical School , Ann Arbor , Michigan Received 30 October 1998/Accepted 14 January 1999 The nitrogen regulator II ( NRII or NtrB ) - NRI ( NtrC ) two-component signal-transduction-system regulates the transcription of nitrogen-regulated genes in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10074086" target="_blank">10074086</a>]<br></li>
<li>The NRII ( NtrB ) - NRI ( NtrC ) two-component system regulates the transcription of nitrogen-regulated ( ntr ) genes in response to signals of nitrogen and carbon sufﬁciency ( reviewed in references 16 , 23 , 24 , 27 , 33 , 34 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10074086" target="_blank">10074086</a>]<br></li>
<li>Expression of the glnALG ( or glnA-ntrBC ) operon from the glnAp2 promoter is required for optimal GS synthesis and for high levels of two important regulators of the Ntr response , nitrogen regulator II ( NRII , also called NtrB ) and nitrogen regulator I ( NRI , also called NtrC ) , which are required for maximal GS synthesis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12730324" target="_blank">12730324</a>]<br></li>
<li>Unmodiﬁed PII interacts with nitrogen regulator II ( NRII , also called NtrB ) and stimulates the dephosphorylation of nitrogen regulator I ( NR1 , also called NtrC ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11528004" target="_blank">11528004</a>]<br></li>
<li>The genes of glnALG code for glutamine synthetase , nitrogen regulator II ( NRII , or NtrB ) , and nitrogen regulator I ( NRI , or NtrC ) , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7730282" target="_blank">7730282</a>]<br></li>
<li>The degradation of arginine as the sole nitrogen source limits growth and requires the transcriptional activator nitrogen regulator I ( NRI , also called NtrC ) , the product of glnG ( or ntrC ) ( 23 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9696779" target="_blank">9696779</a>]<br></li>
<li>Alternative sigma factor 54 and 54-dependent activator nitrogen regulator I ( NRI , also called NtrC ) mediate this nitro-gen-regulated ( Ntr ) response ( 25 , 33 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12003934" target="_blank">12003934</a>]<br></li>
<li>Expression of Ntr genes often requires the alternate sigma factor 54 and the response regulator nitrogen regulator I ( NRI ; also called NtrC ) ( 30 , 31 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12446648" target="_blank">12446648</a>]<br></li>
<li>Most Ntr genes have either a 54-dependent promoter , which requires nitrogen regulator I ( NRI ; also called NtrC ) , or a 70-dependent promoter , which requires Nac .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12446648" target="_blank">12446648</a>]<br></li>
<li>The nitrogen regulation stress response is an adaptive mechanism used by nitrogen-starved Escherichia coli to scavenge for alternative nitrogen sources and requires the global transcriptional regulator NtrC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24947454" target="_blank">24947454</a>]<br></li>
<li>ArgR-Independent Induction and ArgR-Dependent Superinduction of the astCADBE Operon in Escherichia coli Alexandros K. Kiupakis and Larry Reitzer * olecular and Cell Biology , The University of Texas at Dallas , Richardson , Texas 75083-0688 Received 7 November 2001/Accepted 8 March 2002 ent of M Departm For Escherichia coli , growth in the absence of ammonia is termed nitrogen limited and results in the induction of genes that assimilate other nitrogen sources , a response mediated by 54 and nitrogen regulator I ( NRI , also called NtrC ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12003934" target="_blank">12003934</a>]<br></li>
<li>In this class , KdpD/KdpE , NarP and NtrB/NtrC are known to be involved in response to turgor pressure , nitrate and nitrogen availability respectively ( Laimins et al. , 1981 ; Darwin and Stewart , 1995 ; Zimmer et al. , 2000 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12366850" target="_blank">12366850</a>]<br></li>
<li>In E. coli , nitrogen-limited growth induces the nitrogen-regulated ( Ntr ) response , which requires σ54 ( RpoN ) , and the sensor kinase-response regulator pair NtrB-NtrC ( Zimmer et al. , 2000 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23531166" target="_blank">23531166</a>]<br></li>
<li>Under nitro-gen-limited conditions , NtrB promotes the phosphorylation of the response regulator NtrC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9892680" target="_blank">9892680</a>]<br></li>
<li>The GlnL ( NtrB ) protein is a histidine autokinase that donates phosphoryl groups to response regulator GlnG ( NtrC ) and thereby activates it ( 18 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11121068" target="_blank">11121068</a>]<br></li>
				</div></ul><br>
			<label class="collapse_SIT" for="_5"><font size="5px">NtrC binding site</font></label>
				<input id="_5" type="checkbox">
				<div class="SIT"><br><ul><li>Tracks with the input DNA control tag density ( denoted as input ) and with the genomic loci bound by NtrC identiﬁed by the peak-calling algorithm Site Identiﬁcation for Short Sequence Reads ( denoted as SISSRS ) at t 1/4 N are also shown for comparison .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24947454" target="_blank">24947454</a>]<br></li>
<li>Genomic loci bound by NtrC and RNAp were identiﬁed using the peak-calling algorithm SISSRs ( Site Identiﬁcation 33 for Short Sequence Reads ) , with peaks deﬁned as signiﬁcant with a P-value of o 0.005 and if they showed greater than 9-fold enrichment in the immunoprecipitated sample compared with the input control .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24947454" target="_blank">24947454</a>]<br></li>
<li>Transcription assays with NtrC contained 10 mM carbamyl phosphate .31 Open complex assays Open complex assays were conducted with 16 nM end-labelled 88mer homoduplex double-stranded DNA þ consisting of the 260 to 28 S. meliloti nifH promoter sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12079332" target="_blank">12079332</a>]<br></li>
<li>NtrC binds to DNA , and at least for NtrC and NifA , it has been demonstrated that they perform their function in a coordinate action with a specific sigma factor ( NtrA ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2666400" target="_blank">2666400</a>]<br></li>
<li>We have demonstrated that the nitrogen regulatory protein NtrC , product of gene ntrC ( glnG in Escherichia eoli ) binds to a specific sequence in the dhuA promoter region ( Ames and Nikaido 1985 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3118148" target="_blank">3118148</a>]<br></li>
<li>Nitrogen-regulated Ntr promoters of Escherichia coli are activated by the phosphorylated form of the glnG ( ntrC ) protein NRI ~ P ( NtrC ~ P ) ( Ninfa and Magasanik , 1986 ) , which binds to enhancer sequences found upstream from the regulated promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12453212" target="_blank">12453212</a>]<br></li>
<li>Figure 3 compares the sequences of 11 proteins in the region identified by Drummond et al. as a central domain in NtrC and NifA ( 5 , 12 , 20 , 21 , 25 , 33 , 43 , 44 , 50 , 52 , 54 , 59 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8449883" target="_blank">8449883</a>]<br></li>
<li>By analogy with NifA ( Buck et al. , 1986 ; Fischer , 1994 ) , and NtrC ( Porter et al. , 1993 ) , the sequences marked UAS I and UAS II are likely to represent the core binding site for PspF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9878422" target="_blank">9878422</a>]<br></li>
<li>These sequences are compared to the consensus sequences of PspF ( Dworkin et al. , 1997 ) and NtrC ( Weiss et al. , 1992 ) at the bottom of the ﬁgure .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17185540" target="_blank">17185540</a>]<br></li>
<li>Therefore , we tested NtrC binding to these regions by electrophoretic mobility shift assays ( EMSA ) with puriﬁed E. coli NtrC and DNA probe sequences ( designed following ChIP-seq analysis ) upstream of ﬂgMN , dicC , ﬂiC , relA , ssrS , soxR and yjcZ-proP ( referred to as ﬂgN , dicC , ﬂiC , relA , ssrS , soxR and yjcZ probes ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24947454" target="_blank">24947454</a>]<br></li>
<li>Representative autoradiographs of non-denaturing gels showing the binding of NtrC to 32P-labelled DNA probes with sequences corresponding to the upstream regions of glnA ( positions 273 to þ 71 relative to the translation-start site of GlnA ) and relA ( positions 928 to 592 relative to the translationstart site of RelA ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24947454" target="_blank">24947454</a>]<br></li>
<li>In both NtrC and TyrR , oligomerization provides a means of recruiting multiple DNA binding sites to form a stable protein-DNA complex with possibly altered tyrosine concentration would promote hexamerization , and binding to the adjacent weak box .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11923293" target="_blank">11923293</a>]<br></li>
<li>s54 -- RNAP binding to the putative P6 promoter IHF , NtrC and PspF binding upstream of the P6 promoter Analysis of the regulatory region of the rpoH gene revealed the presence of a putative IHF-binding site , with remarkable homology to the consensus site ( Craig & Nash , 1984 ; Goodrich et al. , 1990 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17185540" target="_blank">17185540</a>]<br></li>
<li>Sequence analysis of the rpoH regulatory region indicated the presence of some imperfect NtrC - binding sites and PspF-binding sites spread over the whole region of interest ( Fig. 1b ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17185540" target="_blank">17185540</a>]<br></li>
<li>binding site binding site 1 2 3 4 5 6 7 8 9 10 11 12 13 ( b ) s54 -- RNAP binding to the putative P6 promoter The search for NtrC - or PspF-binding sites by sequence analysis of the 707 bp W1-W2 DNA fragment resulted in a complicated picture of several imperfect binding sites for each of these proteins ( Fig. 1b ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17185540" target="_blank">17185540</a>]<br></li>
<li>Although we have detected some putative NtrC binding sites by sequence analysis , we have not yet identified the pairs of sites that usually constitute functional enhancers ( 53 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11121068" target="_blank">11121068</a>]<br></li>
<li>( iv ) The phosphorylated protein formed a speci ® c complex at the DNA duplex with the enhancer sequence ( two NtrC binding sites ) that consisted of four dimers ( concentration range studied 100 to 1000 nM NtrC dimer ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9600853" target="_blank">9600853</a>]<br></li>
<li>A speci ® c complex formed at the DNA duplex ES-2 with the enhancer sequence ( two NtrC binding sites ) that consisted of four NtrC dimers ( open circles Figure 3b , and ® lled squares Figure 5 , Table 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9600853" target="_blank">9600853</a>]<br></li>
<li>The NtrC binding sites are shaded gray and the recognition sequence is printed in bold .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9600853" target="_blank">9600853</a>]<br></li>
<li>This template has an enhancer sequence with two NtrC binding sites , identical to the sequence of the ES-2 duplex , at a distance of about 460 bp from the promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9600853" target="_blank">9600853</a>]<br></li>
<li>The ES-1 duplex ( 32 bp ) has a single binding site and the ES-2 duplex ( 59 bp ) has two NtrC binding sites as indicated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9600853" target="_blank">9600853</a>]<br></li>
<li>The ES-2 DNA duplex sequence is given in Figure 1 and has two binding sites for NtrC indicated by the gray shading .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9600853" target="_blank">9600853</a>]<br></li>
<li>To demonstrate in vitro that P4 represents a bona ﬁde NtrC and s54-RNAp-dependent promoter , we conducted a small primed RNA ( spRNA ) synthesis assay20 using puriﬁed E. coli NtrC , s54 and RNAp and using puriﬁed E. coli NtrC , s54 and RNAp and a promoter DNA fragment ( encompassing sequences 261 to þ 75 with respect to P4 TSS ) and thus containing the NtrC binding site and P4 promoter region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24947454" target="_blank">24947454</a>]<br></li>
<li>At such promoters , hexamers of phosphorylated NtrC bind to enhancer-like sequences located B100 -- 150 nucleotides upstream of the transcription start site and in a reaction hydrolysing adenosine triphosphate ( ATP ) , convert a transcriptionally inactive initial s54-RNAp-promoter complex to a transcriptionally active one17 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24947454" target="_blank">24947454</a>]<br></li>
<li>The three major differences that separate s54 from the s70 family of the other s factors are : ( i ) unlike members of s70 family , 54 s is able to bind promoter DNA in the absence of core RNA polymerase ( 7 ) ; ( ii ) regulatory proteins like NtrB and NtrC activate s54 holoenzyme ( 12,13 ) ; ( iii ) s54 recognizes promoter sequences with conserved GG and GC elements located 24 to 12 nucleotides upstream of the transcription start site ( 3,7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19969540" target="_blank">19969540</a>]<br></li>
<li>At this locus , NtrC binds to an enhancer/operator region located between 100 and 150 bp upstream of glnA , and to an operator located between glnA and ntrB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16751184" target="_blank">16751184</a>]<br></li>
<li>If the perfect inverted-repeat just upstream of the rut promoter serves as a binding site for RutR , this auxiliary regulator may hinder contact between NtrC and 54-holoenzyme by stiffening the DNA between the enhancer and the promoter ( 15 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16540542" target="_blank">16540542</a>]<br></li>
<li>In contrast the OmpR binding sites on ompC and ompF promoters , and the NtrC binding sites on gZnA promoter are located more than 50 bp upstream from the - 10 region ( Reitzer & Magasanik , 1985 ; Norioka et al. , 1986 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3054125" target="_blank">3054125</a>]<br></li>
<li>For the nitrogenregulated genes expressed by a54-carrying RNA polymerase , binding sites for the activator NtrC or NifA are located in the range of 100-200 bp upstream of the transcription start site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7545108" target="_blank">7545108</a>]<br></li>
<li>In support of NtrC control , there is a putative s54 promoter for the ddp operon located 53 bp upstream of the translational start for ddpX ( 6 , 49 ) and a putative NtrC-binding site 140 bp upstream .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11121068" target="_blank">11121068</a>]<br></li>
<li>With one exception , the b1012-like operon appeared to have a consensus 54 promoter and at least one upstream NtrC binding site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16540542" target="_blank">16540542</a>]<br></li>
				</ul></div><br>
			<label class="collapse_TU" for="_6"><font size="5px">Transcription units regulated by NtrC</font></label>
				<input id="_6" type="checkbox">
				<div class="TU"><br><ul><li>Kutsche , M. , Leimkühler , S. , Angermüller , S. , and Klipp , W. ( 1996 ) Promoters controlling expression of the alternate nitrogenase and the molybdenum uptake system in are activated by NtrC , independent of s54 , and are repressed by molybdenum .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9466267" target="_blank">9466267</a>]<br></li>
<li>Kutsche , M. , Leikkuhler , S. , Angermuller , S. , and Klipp , W. ( 1996 ) Promoters controlling expression of the alternate nitrogenase and the molybdenum uptake system in Rhodo-bacter capsulatus are activated by NtrC , independent of s 54 , and are repressed by molybdenum .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9044285" target="_blank">9044285</a>]<br></li>
<li>Kutshe , M. , Leimkiihler , S. , Angemuller , S. & Klipp , W. ( 1996 ) Prornoters controlling expression of the alternative nitrogenase and the mo-lybdenum uptake system in Rhodobacter capsulatus are activated by NtrC , independent of os4a , nd repressed by molybdenum , 1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9210473" target="_blank">9210473</a>]<br></li>
<li>FEMS Microbiol Rev 22:503 -- 521 Kutsche M , Leimkuhler S , Angermuller S , Klipp W ( 1996 ) Promoters controlling expression of the alternative nitrogenase and the molybdenum uptake system in Rhodobacter capsulatus are activated by NtrC , independent of sigma54 , and repressed by molybdenum .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16205910" target="_blank">16205910</a>]<br></li>
				<label class="TU_collapse_similarity" for="tu_similarity_5">
Promoters controlling expression of the alternative nitrogenase and the molybdenum uptake system in Rhodobacter capsulatus are activated by NtrC , independent of 54 , and repressed by molybdenum .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12029041" target="_blank">12029041</a>]<br></label>
<input id="tu_similarity_5", type="checkbox">
<div class="TU_similar">
<li>Promoters controlling expression of the alternative nitrogenase and the molybdenum uptake system in Rhodobacter capsulatus are activated by NtrC , independent of s54 , and repressed by molybdenum .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9721306" target="_blank">9721306</a>]<br></li>
<li>Promoters controlling expression of the alternative nitrogenase and the molybdenum uptake system in Rhodobacter capsulatus are activated by NtrC , independent of 054 , and repressed by molybdenum .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10206709" target="_blank">10206709</a>]<br></li></div>
<li>Indeed , there was more than a 20-fold increase in crl transcript abundance under nitrogen starvation as observed by quantitative real-time PCR ( qRT-PCR ) ( Fig. 1A ) , a result confirmed by Northern blot hybridization ( Fig. 1C ) , and this increase was abolished by removal of NtrC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24474781" target="_blank">24474781</a>]<br></li>
<li>FEMS Microbiology Letters 167 ( 1998 ) 51 ^ 56 Transcriptional repression of gdhA in Escherichia coli is mediated e Nac protein by th Laura Camarena * , Sebastian Poggio , Norma Garcè  a , Aurora Osorio Departamento de Biologè  a Molecular , Instituto de Investigaciones Biomeèdicas , Universidad Nacional Autoènoma de Meèxico , A.P. 70-228 , Meèxico 04510 D.F. , Mexico Received 3 March 1998 ; revised 28 July 1998 ; accepted 31 July 1998 Abstract In this work we show that the nac gene from Escherichia coli is transcriptionally active , and that its expression is dependent on NRI ( NtrC ) and sigma-54 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9785451" target="_blank">9785451</a>]<br></li>
<li>For example , the arginine catabolic operon , astCADBE , not only responds to Ntr control ( via NtrC ) , but also to specific hyperinduction by arginine ( via ArgR ) , and activation by σS ( Schneider et al. , 1998 ; Kiupakis and Reitzer , 2002 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23531166" target="_blank">23531166</a>]<br></li>
<li>It should be recalled that physical and biochemical evidence for DNA looping already exists in the literature , especially in the case of the araBAD operon , and for activation by NtrC of the ginA promoter of E. coli ( reviewed in reference 16 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7928991" target="_blank">7928991</a>]<br></li>
<li>Under N-excess conditions , 16 genes of the NtrC regulon showed decreased expression ( glnK , amtB , nac , nagA , yhhQ , cbl , glnG , b0296 , b1008 , b1011 -- 12 , b1036 , b1486 -- 87 , b1932 , b1973 ) and three genes increased expression ( osmY , csgD , csgG , see also the Supplementary Table containing the normalized gene expression ratios of all hybridization experiments ) as was observed when comparing the transcriptomes at 3.33 and 1.67 mM glucose feed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17273855" target="_blank">17273855</a>]<br></li>
<li>The special case of NR ( I ) ( NtrC ) is also included in the table as it represses its own synthesis from a sigma 70 promoter but activates it from an overlapping sigma 54 promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1943993" target="_blank">1943993</a>]<br></li>
<li>The NtrA and NtrC proteins did not enhance the expression of glgC with pOP12 in the dipeptide synthesis assay , and in this respect the dipeptide assay confirmed the results of the S-30 analyses .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2468650" target="_blank">2468650</a>]<br></li>
<li>ginD is trans-cribed from its own promoter , ginE is cotranscribed wrth another gene , orfXE , whereas gInB is partly cotranscribed with a gene encoding a homologue of the transcription activator NtrC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8412694" target="_blank">8412694</a>]<br></li>
<li>Expression of the putA gene encoding proline dehydrogenase from Rhodobacter capsulatus is independent of NtrC regulation but requires an Lrp-like activator protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18586269" target="_blank">18586269</a>]<br></li>
<li>For the nitrogenregulated genes expressed by a54-carrying RNA polymerase , binding sites for the activator NtrC or NifA are located in the range of 100-200 bp upstream of the transcription start site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7545108" target="_blank">7545108</a>]<br></li>
<li>In support of NtrC control , there is a putative s54 promoter for the ddp operon located 53 bp upstream of the translational start for ddpX ( 6 , 49 ) and a putative NtrC-binding site 140 bp upstream .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11121068" target="_blank">11121068</a>]<br></li>
<li>This site , located upstream of the rutA promoter , was previously proposed to be involved in RutR mediated repression of the rutABCDEFG operon that is transcribed from a r54 promoter and regulated by NtrC , the general regulator of nitrogen metabolism [ 1 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25685666" target="_blank">25685666</a>]<br></li>
<li>The nitrogen-regulated glnALG genes consist of an operon whose expression is regulated by GlnG/NtrC at three promoters ; two located upstream of glnA ( glnAP1 and glnAP2 ) and a third located upstream of glnL ( glnLP ) [ reviewed in ( 28 ) ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15718303" target="_blank">15718303</a>]<br></li>
<li>The NtrC regulatory site is located upstream of argT .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17074904" target="_blank">17074904</a>]<br></li>
<li>This N is released as ammonium by cytidine deaminase ( cdd , b2143 ) , which is not known to be under NtrC control , to yield uridine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16540542" target="_blank">16540542</a>]<br></li>
<li>The operon also encodes a 2CST pathway , composed of NRII ( also known as NtrB and encoded by glnL ) and its cognate RR NRI ( also known as NtrC and encoded by glnG ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15755952" target="_blank">15755952</a>]<br></li>
<li>Expression of the operon is elevated in a strain that carries an ntrB ( Con ) ( ntrB [ constitutive ] ) mutation , which increases transcription of all genes under NtrC control ( 59 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20400551" target="_blank">20400551</a>]<br></li>
<li>NtrC also activates transcription of genes encoding regulatory proteins : ntrB and ntrC ( glnL and glnG , respectively ) , glnK , and nac ( 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11121068" target="_blank">11121068</a>]<br></li>
<li>Nac does not affect Ntr expression levels One possible explanation for the effect of the nac mutation is that , like the mutation of glnL ( ntrB ) , it resulted in reduced expression of Ntr genes by lowering levels of NRI ~ P ( NtrC ~ P ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12123449" target="_blank">12123449</a>]<br></li>
<li>The NtrC protein also activates expression of the nitrogen assimilatory control gene ( nac ) in Klebsiella aerogenes , which is known to regulate expression of glutamate synthase ( 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8955321" target="_blank">8955321</a>]<br></li>
<li>Among genes that have been known to be under the control of NtrB/NtrC , only the glnP gene encoding glutamine transport protein was downregulated in the DntrBC mutant .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12366850" target="_blank">12366850</a>]<br></li>
<li>Although the reasons are not obvious , growth on glutamine is known to elevate expression of genes under NtrC control ( 50 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11121068" target="_blank">11121068</a>]<br></li>
<li>The autokinase is more active in glnL ( Up ) [ NtrB ( Con ) ] strains than in wild-type strains , and hence genes and operons under NtrC ( GlnG ) control are overexpressed even under conditions of nitrogen excess .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11121068" target="_blank">11121068</a>]<br></li>
<li>As is the case for operons known to be activated by NtrC , most differences in mRNA levels between the glnL ( Up ) strain and the glnG ( ntrC ) strain seen with ammonia as sole nitrogen source were also apparent with glutamine ( Fig. 1 ; Tables 1 and 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11121068" target="_blank">11121068</a>]<br></li>
<li>To determine which of the operons under NtrC control depended on the NtrC-Nac regulatory cascade , we compared mRNA levels in the glnL ( Up ) strain to those in a glnL ( Up ) nac strain on ammonia and on glutamine as nitrogen source ( genes under Nac control detected ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11121068" target="_blank">11121068</a>]<br></li>
<li>The sequence of the NtrC gene in pNTRC-3 was con - ® rmed by DNA sequencing .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9600853" target="_blank">9600853</a>]<br></li>
<li>NtrC apparently activated transcription of the putrescine transport operon ( b0854 -- 57 ) , the b1006 -- 12 operon ( ORFs ) , the transport and catabolic operon for D-alanine-D-alanine ( b1483 -- 88 ) ( Fig. 2 ) , and the probable orni-thine aminotransferase operon ( b3073 ) , in addition to operons known or thought to be under its control ( Table 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11121068" target="_blank">11121068</a>]<br></li>
<li>The rut operon was discovered in E. coli K-12 as one of the most highly expressed operons under NtrC control .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20400551" target="_blank">20400551</a>]<br></li>
<li>The cycA gene ( b4208 ) , which encodes a transporter for D-ala , is under Nac control and the ddp operon ( b1483 -- 88 ) , which encodes a transporter and a specific dipeptidase for - ala - - ala ( 10 ) , is under NtrC control .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11121068" target="_blank">11121068</a>]<br></li>
<li>Further , the expression levels of glnK mRNA , an Ntr stress response gene directly activated by NtrC , at t 1/4 N ( that is , the time-point 20 min after N runs out ) are similar in the wild-type NCM3722 and NCM3722 : glnG-FLAG strains , and as expected is not readily detected in the NCM3722 : DglnG strain ( Fig. 1c ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24947454" target="_blank">24947454</a>]<br></li>
<li>To unambiguously establish that relA transcription is activated by NtrC during N starvation , we measured relA mRNA levels in wild-type NCM3722 and NCM3722 : DglnG strains at t 1/4 N relative to t 1/4 N þ .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24947454" target="_blank">24947454</a>]<br></li>
<li>Prominent examples are the bacterial nitrogen uptake regulating transcription factor , NtrC , which is phosphoryiated in its active form by NtrB ( Austin and Dixon , 1992 ; Sanders etai , 1992 ) , or the catabolite regulatory protein , CRP , which is activated by oAMP ( Botsford and Harman , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7512187" target="_blank">7512187</a>]<br></li>
<li>In contrast , the glnAp2 promoter is s - dependent 54 and its transcription can be activated either by its cognate activator NtrC under nitrogen-deﬁciency conditions ( Fig. 1 ) , or by the alternative activator , NifA ( Figs 4 and 7B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11532153" target="_blank">11532153</a>]<br></li>
<li>Here , we show that transcription of relA , a key gene responsible for the synthesis of ppGpp , is activated by NtrC during nitrogen starvation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24947454" target="_blank">24947454</a>]<br></li>
<li>NtrC activates relA transcription in N-starved E. coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24947454" target="_blank">24947454</a>]<br></li>
<li>glnG encodes NtrC , the transcriptional activator of nitrogen assimilation and scavenging genes under conditions of nitro-gen-limited growth ( 41 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15629914" target="_blank">15629914</a>]<br></li>
<li>In contrast , the arginine-ornithine-lysine binding protein which interacts with the same ABC transporter is induced under nitrogenlimiting conditions and controlled by NtrC ( 61 ) , suggesting that the LAO system serves as a scavenging system for nitro-gen-rich amino acids under conditions of nitrogen starvation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21441513" target="_blank">21441513</a>]<br></li>
<li>It should be noted that three regulatory systems have been found not to be involved in gigCA expression , the nitrogen starvation system , mediated by NtrC and NtrA or or54 ( 32 , 38 ) ; heat shock , mediated by o32 ( 32 ) ; and the katF-dependent system ( 13 , 38a ) , implying that the effects of csrA are not mediated indirectly via these global systems .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8393005" target="_blank">8393005</a>]<br></li>
<li>PrpR of S. enterica belongs to the NtrC family of transcriptional activators and has been known to regulate the expression of the prpBCDE operon in response to propionate catabolism ( 32 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15805526" target="_blank">15805526</a>]<br></li>
<li>In E. coli , the genes are organ-ized in two adjacent transcription units of identical polarity , argT and hisJQMP ( Fig. 2 ) , whose expression is positively controlled by the nitrogen regulatory protein NtrC in response to nitrogen starvation ( Zimmer et al. , Fig. 1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17074904" target="_blank">17074904</a>]<br></li>
<li>The expression of allS is , however , independent of the nitrogen-assimilation regulators NtrC and Nac ( Zimmer et al. , 2000 ; Muse et al. , 2003 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18957590" target="_blank">18957590</a>]<br></li>
<li>Finally , a divergently transcribed E. coli gene b1201 ( ycgU ; here designated dhaR ; see Fig. 1 ) was found to encode a putative transcriptional activator , homologous to DhaR , the putative regulator of dhaK expression in Citrobacter freundii ( 2 ) , as well as the NtrC response regulator of the nitrogen two-component system ( 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11021910" target="_blank">11021910</a>]<br></li>
<li>Activation of the expression of gInA and of other nitrogen-regulated genes requires both the a ^ * factor ( encoded by the rpoN gene , aiso calied ntrA ) , and the transcriptional activator NtrC ( Hirschman etai , 1985 ; Hunt and Magasanik , 1985 ; Reitzer and Magasanik , 1985 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8412694" target="_blank">8412694</a>]<br></li>
<li>This regulation of gene expression requires the NRI -- NRII ( NtrC -- NtrB ) two-component signal transduction system , encoded by two genes located downstream from the structural gene for glutamine synthetase , glnA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12123449" target="_blank">12123449</a>]<br></li>
				</div></ul></div><br>
	</div>
</body>
</html>