<!doctype html>
<style>
@charset "utf-8";
body {
	height: 100%;
	margin: 0;
	padding: 0;
}

.banner {
	position: fixed;
	top: 0;
	width: 100%;
	height: 65px;
	background: gray;
	line-height: 65px;
}
.spacer {
	height: 10px;
}

.main-content {
	padding: 5%;
}

.collapse_ACT {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-style: none solid none none;
	border-color: #B4045F;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_ACT + input{
	display: none;
}
.collapse_ACT + input + div{
	display: none;
}
.collapse_ACT + input:checked + div{
	display: block;
}
.ACT {
	text-align: justify;
	text-justify: inter-word;
}
.ACT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B4045F;
}
.ACT_collapse_similarity:hover{
	background: #e5d3dc;
}
.ACT_collapse_similarity + input {
	display: none;
}
.ACT_collapse_similarity + input + div{
	display: none;
}

.ACT_collapse_similarity + input:checked + div{
	display: block;
}
.ACT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.ACT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B4045F;
	padding-left: 5px;
}
.ACT_collapse_similarity:hover{
	background: #e5d3dc;
}
.ACT_collapse_similarity + input {
	display: none;
}
.ACT_collapse_similarity + input + div{
	display: none;
}
.ACT_collapse_similarity + input:checked + div{
	display: block;
}
.ACT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.ACT_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #B4045F;
}
.ACT_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_DOM {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-style: none solid none none;
	border-color: #B404AE;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_DOM + input{
	display: none;
}
.collapse_DOM + input + div{
	display: none;
}
.collapse_DOM + input:checked + div{
	display: block;
}
.DOM {
	text-align: justify;
	text-justify: inter-word;
}
.DOM_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B404AE;
	padding-left: 5px;
}
.DOM_collapse_similarity:hover{
	background: #e8d7e7;
}
.DOM_collapse_similarity + input {
	display: none;
}
.DOM_collapse_similarity + input + div{
	display: none;
}
.DOM_collapse_similarity + input:checked + div{
	display: block;
}
.DOM_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}.DOM_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #B404AE;
}
.DOM_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_EVO {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #7401DF;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_EVO + input{
	display: none;
}
.collapse_EVO + input + div{
	display: none;
}
.collapse_EVO + input:checked + div{
	display: block;
}
.EVO {
	text-align: justify;
	text-justify: inter-word;
}
.EVO_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #7401DF;
	padding-left: 5px;
}
.EVO_collapse_similarity:hover{
	background: #e2d7ed;
}
.EVO_collapse_similarity + input {
	display: none;
}
.EVO_collapse_similarity + input + div{
	display: none;
}
.EVO_collapse_similarity + input:checked + div{
	display: block;
}
.EVO_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.EVO_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #7401DF;
}
.EVO_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_RP {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #0174DF;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_RP + input{
	display: none;
}
.collapse_RP + input + div{
	display: none;
}
.collapse_RP + input:checked + div{
	display: block;
}
.RP {
	text-align: justify;
	text-justify: inter-word;
}
.RP_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #0174DF;
	padding-left: 5px;
}
.RP_collapse_similarity:hover{
	background: #c9d7e2;
}
.RP_collapse_similarity + input {
	display: none;
}
.RP_collapse_similarity + input + div{
	display: none;
}
.RP_collapse_similarity + input:checked + div{
	display: block;
}
.RP_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.RP_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #0174DF;
}
.RP_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_SIT {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #0489B1;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_SIT + input{
	display: none;
}
.collapse_SIT + input + div{
	display: none;
}
.collapse_SIT + input:checked + div{
	display: block;
}
.SIT {
	text-align: justify;
	text-justify: inter-word;
}
.SIT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #0489B1;
	padding-left: 5px;
}
.SIT_collapse_similarity:hover{
	background: #d0dfe2;
}
.SIT_collapse_similarity + input {
	display: none;
}
.SIT_collapse_similarity + input + div{
	display: none;
}
.SIT_collapse_similarity + input:checked + div{
	display: block;
}
.SIT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.SIT_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #0489B1;
}
.SIT_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_TU {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #04B486;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_TU + input{
	display: none;
}
.collapse_TU + input + div{
	display: none;
}
.collapse_TU + input:checked + div{
	display: block;
}
.TU {
	text-align: justify;
	text-justify: inter-word;
}
.TU_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #04B486;
	padding-left: 5px;
}
.TU_collapse_similarity:hover{
	background: #d0e2de;
}
.TU_collapse_similarity + input {
	display: none;
}
.TU_collapse_similarity + input + div{
	display: none;
}
.TU_collapse_similarity + input:checked + div{
	display: block;
}
.TU_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.TU_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #04B486;
}
.TU_similar > a {
	text-decoration: none;
	color: gray;
}

ul { 
	list-style-type: none;
	padding: 0;
	margin: 0;
}
ul li > a {
	text-decoration: none;
	color: gray;
}
ul li:hover, ul li:focus {
	background: #F0F0F0;
}

ul li:first-child span {
  border-left:none;  
}

</style>
<html>
<head>
<meta charset="utf-8">
<title>TF properties - OmpR</title>
<meta name="theme-color" content="#ffffff">
</head>
<body>
	<div class="banner">
		<center><font color="white" size="16px" face="arial">OmpR</font></center>
	</div>
	<div align="center" class="main-content">
		<div class="spacer">
	</div><br>			<label class="collapse_ACT" for="_1"><font size="5px">OmpR transcriptional activity</font></label>
				<input id="_1" type="checkbox">
				<div class="ACT"><br><ul><li>Following transfer of phosphate toOmpR , aliquots ( represent - b-complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8175665" target="_blank">8175665</a>]<br></li>
<li>In the wild-type strain , pYqjA-YqjB , pYgdD-GcvA and pOmpR activate both fusions and pFadR activates omrB-lacZ ( Fig. 3 ) , with the strongest signal from pYgdD-GcvA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16359331" target="_blank">16359331</a>]<br></li>
<li>In this context , pYqjA-YqjB , pOmpR and pFadR activate both fusions ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16359331" target="_blank">16359331</a>]<br></li>
<li>After prolonged incubation , the level of phosphorylated RRs increased up to completion for pBasR , pHydG , pOmpR , pPhoB , and pUvrY ( group B1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15522865" target="_blank">15522865</a>]<br></li>
<li>The dicative of long range interactions betweenOmpR mol - OmpR472 mutant protein is unable to-bind to the -42 to -52 ecules .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8175665" target="_blank">8175665</a>]<br></li>
<li>PG-V-SR -- LL 60 70 80 90 i00 SRV RDEEVYDRS IDVLILRLRR KLEADPSSPQLIKTARGAGYFF SRV REEEVYDRS IDVLIFRLRR KLEGDPTSPQLIKTARGAGYFF SRV REEEVYDRS IDVLI LRLRR KLEGDPTTPQLIKTARGAGYFF VTG RELKPHDRTVDVTIRRIRK HFESTPDTPEIIATIHGEGYRF '' ~ 2 , ~ GY TPERHL DTRVVDVHISRLRA KLEDDPSNPELILTSRGTGYLF \ q , ~ GY TAERQV DTRVVDVHISRLRA KLEDDPSNPDLILTARGTGYLF LQLY PDAELRESHTIDVLMGRLRK KIQAQYPH DVITTVRGQGYLF '' , ~ G PNAVEHSHYLRIYMGHLRQ KLEQDPARPRHFITETGIGYRF 2 , GR EYSAME RSIDVQISRLRR MVEEDPAHPRYIQTVWGLGYVF , ~ A2RSASAEEAAGDRAVDVQ ITRLRR KIEPDPREPRYLQTVRGLGYML '' , ' > ~ G TNVYVEDRTVDVHIRRLRK ALEPG GHDRMVQTVRGTGYRF \ ~ ' , ; G GNVYVEERTVDMD IRRLRK ALGE VYENLVQTVRGTGYRF V > ~ E DAQDTYDRTVDTHIKTLRA KLRAINPDLSPINTHRGMGYSL VS 'N YDFAGDTRIVDVHISHLRPTKIENNTKKPIYIKTIRGLGYKL VLG KRLTPFDRAIDMHISNLRR KLPDRKDGHPWFKTLRGRGYLM AWD ENTDPFTNVVRVTVMTLRR KLGE PPVIVTVPGSGYRI EQGFEVDDSSLTQAISTLRK MLKDSTKSPQYVKTVPKRGYQL VirG Atu6 VirG Arh VirG Atu5 ArcA Eco RRHM CYca RRHM POae PhoP Sty KdpE Eco OmpR Eco PetR Rca PhoB Eco PhoB Pae CreB Eco PhoB Bsu YjjE Eco CutR Sli ToxR Vch 119 Fig. 5 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7699720" target="_blank">7699720</a>]<br></li>
<li>AGRT6 VIRG AGRRA YIIA ECOLI TOXR VIBCH + RRHM Pae Plastid response-regulator OmpR Eco Porin bioynthesis , osmolarity PhoB Bsu Alkaline phosphatase regulator CreB Eco Catabolite repression ArcA Eco Aerobic respiratory control PetR Rca Cytochrome bc1 complex CutR Sli Copper metabolism KdpE Eco K + transport PhoB Eco Phosphate regulon PhoB Pae Phosphate regulon PhoP Sty Virulence TctD Sty Tricarboxylate transport VirG Atu6 Virulence VirG Atu5 Virulence VirG Arh Virulence YjjE Eco Unknown ToxR Vch Virulence cholera toxin Class 3 : DNA binding domains of family 3 FixJ Rme Nitrogen fixation FixJ Bja Nitrogen fixation NodW Bja Nodulation BvgA Bpe Virulence FimZ Eco Fimbrial synthesis DegU Bsu Extracellular degradative enzymes NarL Eco Nitrate reductase & formate dehydrogenase TrpO Pae Unknown YecB Eco Unknown RcsB Eco Colanic acid capsule biosynthesis B AgmR Fae Glycerol metabolism UhpA Eco Uptake of hexose phosphates Coma Bsu Competence GerE Bsu SigK txn .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7699720" target="_blank">7699720</a>]<br></li>
<li>Proteins in classes 4 , 5 and 6 are exceptions 7O ENAF I0 20 30 40 50 60 NtrC NtrC NifRl NtrC PgtA DctD FixJ NodW CheY SpoOF RRHM RRHM OmpR PhoB CreB ArcA PetR VirG CutR KdpE PhoB PhoP TctD AIgB YLB3 RcsC BvgS HydG CheY FrzE BarA TrpO HvgA FimZ DegU NarL AIgRI YecB UhpA RcsB Agm & CheB CheB FrzG Spo0A MrkE AgrA Coma F!bD Bpa Rme Rca Eco Sty Rle Rme Bja Bsu Bsu Cca Pae Eco Bsu ECO Eco Rca Atu Sli Eco Eco Sty Sty Pae Lin Eco Bpe Eco Eco M ~ a Eco Pae Bpe Eco Bsu Eco Pae Eco Eco Eco Pae Bsu Eco Mxa Bsu Kpn Sau Bsu Ccr 6 ILVADDDTAIKTVLNQA 6 ILVADDD ~ IRTVLNQA 5 VLVADDDRTIRTVLTQA 6 V % ~ VDDDSSIR ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7699720" target="_blank">7699720</a>]<br></li>
<li>) Major clusters 1 , 2 , and 3 are associated primarily or exclusively with the three major families of DNA binding NtrC Bpa NtrC Rme NifRl Rca Nt rC Eco PgtA Sty DetD Rle FixJ Kme NodW Bja CheY Bsu SpoOF BSu RRHM Cca RRHM Pae OmpR Eco PhoB Bsu CreB Eco ArcA Eco PetR Rca VirG Atu CutR Sli KdpE ECO PhoB Eco PhoP Sty TctD Sty A!gB Pae YLB3 Lin RcsC Eco BvgS Bpe HydG Eco CheY Eco FrzE Nc .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7699720" target="_blank">7699720</a>]<br></li>
<li>sitesa % CGb Count % Non-codinge Ada 3 38.7 AraC 6 40.3 2 3 100 100 47 76 100 28 33 32 73 38 100 52 100 86 79 89 100 67 26 100 28 100 70 37 67 28 100 82 100 47 50 51 100 100 38 33 64 3 6 100 100 38 54 100 23 40 18 43 29 100 38 100 39 27 53 58 71 19 100 23 33 47 100 40 65 21 23 60 25 100 41 100 26 43 36 100 35 35 30 64 36 23 22 26 14 100 83 28 ArcA 12 20.0 ArgR 17 22.9 CarP 2 46.0 Crp 49 37.0 CspA 4 41.3 CynR 2 33.3 CysB 3 34.2 CytR 5 37.8 DeoR 3 22.9 DnaA 8 36.7 FadR 7 47.0 FarR 4 20.0 FhlA 3 42.0 Fis 19 38.2 FlhCD 3 53.8 Fnr 9 25.8 FruR 12 42.7 Fur 9 23.4 GalR 7 43.8 GcvA 4 22.5 GlpR 13 39.6 HipB 4 40.9 Hns 15 31.5 IclR 2 26.7 Ihf 26 32.6 IlvY 2 32.4 LacI 3 44.4 LexA 18 33.9 Lrp 18 28.9 MalT 10 61.0 MarR 2 37.5 MelR 2 33.3 MetJ 15 40.1 MetR 8 33.3 NagC 6 22.5 NarL 9 34.6 NtrC 6 51.9 OmpR 9 26.7 OxyR 4 42.3 PdhR 3 33.3 PhoB 15 29.7 PurR 23 45.3 RhaS 2 50.0 RpoD15 27 37.4 RpoD16 48 34.9 RpoD17 116 37.3 RpoD18 34 38.0 RpoD19 25 38.2 RpoE 3 38.9 RpoH13 8 42.1 RpoH14 7 44.3 RpoN 7 40.6 RpoS17 15 43.0 RpoS18 7 45.2 SoxS 14 40.8 TorR 4 37.5 TrpR 4 38.5 Tus 6 26.8 TyrR 17 33.2 a The number of input sites used to construct the matrix .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9813115" target="_blank">9813115</a>]<br></li>
<li>of additional probable sites in predictionsd 3 3 7e 7 -- 4 22e 0 1 0 2 -- -- 5 0 1 -- 13 7 18 1 0 6e 1 1 1 3 Ada AraC ArcA ArgR CarP CpxR Crp CspA CynR CysB CytR DeoR DnaA FadR FhlA Fis FlhCD Fnr FruR Fur GalR GcvA GlpR IciA ( ArgP ) IclR Ihf IlvY LacI LexA Lrp MalT MarR MelR MetJ MetR Mlc ( DgsA ) ModE NagC NarL NarP NtrC OmpR OxyR PdhR PhoB PurR RhaR RhaS SoxR SoxS TorR TrpR TyrR Stem -- loopsf 6 2 0 5 2 9 1 3 4 5e 5 -- 5 5 4 6 3 4 2 -- 1 3 5 1 3 -- aThe number of genes in our study set with documented binding sites for each TF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11160901" target="_blank">11160901</a>]<br></li>
<li>JouRNAL OF BACTERIOLOGY , July 1981 , p. / 81/07025504 $ 02.00 / 0 NOTES Identification of OmpR : a Positive Regulatory Protein Controlling Expression of the Major Outer Membrane Matrix Porin Proteins of Escherichia coli K-12 RONALD K. TAYLOR , ' MICHAEL N. HALL ,2 LYNN ENQUIST ,3 AND THOMAS J. SILHAVY4 * Department ofBiological Sciences , University ofMaryland Baltimore County , Catonsvilk , Maryland 212281 ; Department ofMicrobiology and Molecular Genetics , Harvard Medical School , Boston , Massachusetts 021152 ; Laboratory ofMolecular Virology , National Cancer Institute , National Institutes of Health , Bethesda , Maryland 202053 ; and Cancer Biology Program , National Cancer Institute Frederick Cancer Research Center , Frederick , Maryland 21701 ' Received 30 January 1981/Accepted 13 March 1981 We report here on the cloning of a gene located within the ompB locus of E. coli K-12 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6263866" target="_blank">6263866</a>]<br></li>
<li>Intramolecular second-site revertants to the phosphorylation site mutation in OmpR , a kinase-depen - dent transcriptional-activator in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11812125" target="_blank">11812125</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_14">
[ 10 ] Pru `` , B.M. ( 1998 ) Acetyl phosphate and the phosphorylation of M OmpR are involved in the regulation of the cell division rate in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11287152" target="_blank">11287152</a>]<br></label>
<input id="act_similarity_14", type="checkbox">
<div class="ACT_similar">
<li>Acetyl phosphate and the phosphorylation of OmpR are involved in the regulation of the cell division rate in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25667010" target="_blank">25667010</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10894724" target="_blank">10894724</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15755952" target="_blank">15755952</a>]<br></li>
<li>Acetyl phosphate and the phosphorylation of OmpR are involved in the regulation of cell division rate in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15516564" target="_blank">15516564</a>]<br></li></div>
<li>JOURNAL OF BACTERIOLOGY , Aug. 1995 , p. 4696 -- / 95 / $ 04.0010 Copyright q 1995 , American Society for Microbiology Modulation of Flagellar Expression in Escherichia coli by Acetyl Phosphate and the Osmoregulator OmpR SOOAN SHIN AND CHANKYU PARK * Department of Biological Sciences , Korea Advanced Institute of Science and Technology , Yusong-Ku , Taejon , Republic of Korea Received 19 December 1994/Accepted 3 June 1995 During the search for unknown factors involved in motility , we have found that expression of the ﬂagellar master operon ﬂhDC is affected by mutations of the pta and ackA genes , encoding phosphotransacetylase and acetate kinase , respectively ( S. Shin , J. Sheen , and C. Park , Korean J. Microbiol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7642497" target="_blank">7642497</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_18">
Shin , S. , and Park , C. ( 1995 ) Modulation of ﬂagellar expression in Escherichia coli by acetyl phosphate and the osmo-regulator OmpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16194231" target="_blank">16194231</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16135237" target="_blank">16135237</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19843224" target="_blank">19843224</a>]<br></label>
<input id="act_similarity_18", type="checkbox">
<div class="ACT_similar">
<li>Shin , S. , and Park , C. ( 1995 ) Modulation of ﬂagellar expression in Escherichia coli by acetyl phosphate and the osmoregulator OmpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9484901" target="_blank">9484901</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12123461" target="_blank">12123461</a>]<br></li>
<li>Shin , S. and Park , C. ( 1995 ) Modulation of ﬂagellar expression in Escherichia coli by acetyl phosphate and the osmoregulator OmpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19969540" target="_blank">19969540</a>]<br></li>
<li>[ 12 ] Shin , S. and Park , C. ( 1995 ) Modulation of # agellar expression in Escherichia coli by acetyl phosphate and the osmoregulator OmpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11287152" target="_blank">11287152</a>]<br></li>
<li>Shin , S. , and Park , C. ( 1995 ) Modulation of flagellar expression in Escherichia coli by acetyl phosphate and the osmoregulator OmpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11298291" target="_blank">11298291</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8809743" target="_blank">8809743</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8866483" target="_blank">8866483</a>]<br></li></div>
				<label class="ACT_collapse_similarity" for="act_similarity_23">
Modulation of ﬂagellar expression in Esche-richia coli by acetyl phosphate and the osmoregulator OmpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11104815" target="_blank">11104815</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21685294" target="_blank">21685294</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15755952" target="_blank">15755952</a>]<br></label>
<input id="act_similarity_23", type="checkbox">
<div class="ACT_similar">
<li>Modulation of ﬂagellar expression in Escherichia coli by acetyl phosphate and the osmoregulator OmpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8550423" target="_blank">8550423</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10601207" target="_blank">10601207</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15175316" target="_blank">15175316</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16159758" target="_blank">16159758</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10913108" target="_blank">10913108</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26078445" target="_blank">26078445</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15516564" target="_blank">15516564</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24272779" target="_blank">24272779</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12620868" target="_blank">12620868</a>]<br></li>
<li>Modulation of flagellar expression in Escherichia coli by acetyl phosphate and the osmoregulator OmpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25667010" target="_blank">25667010</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22461489" target="_blank">22461489</a>]<br></li>
<li>Modulation of ¯ agellar expression in Escherichia coli by acetyl phosphate and the osmoregulator OmpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11601842" target="_blank">11601842</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11031114" target="_blank">11031114</a>]<br></li></div>
<li>Ghou , J.H. , Greenberg , J , T , , and Demple , B. ( 1993 ) Posttranscriptional repression of Escherichia coli OmpR protein in response to redox stress ; positive controi of the micF antisense RNA by the soxHS locus , J Bacterioi M5 : 1026-1031 , Gompan , I. , and Touati , D. ( 1993 ) Interaction of six global transcription regulators in expression of manganese superoxide dismutase in Escherichia coli K-12 , J Bacteriot 175 : 1687-1696 , Goyer , J , , Andersen , J , , Forst , S.S , , Inouye , M. , and Deiius , N. ( 1990 ) m/cF RNA in ompB mutants of Escherichia coti : different pathways regulate m/cF RNA levels in response to osmoiarity and temperature change , J Bacteriot 172 ; 4143-4150 , Gunningham , R , P , , Saporito , S.M. , Spitzer , S.G. , and Weiss , B. ( 1986 ) Endonuclease IV ( nfo ) mutant of Escherichia coli J Bacteriot 16S : 1120-1127 , Gallegos , M,-T , , Michan , G , , and Ramos .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7891555" target="_blank">7891555</a>]<br></li>
<li>FEMS Microbiology Letters 186 ( 2000 ) 257 ^ 262 www.fems-microbiology.org Negative regulation of the bolA1p of Escherichia coli K-12 by the R for transcription factor Omp osmolarity response genes Kaneyoshi Yamamoto a , Rie Nagura a , Hiroyuki Tanabe a , Nobuyuki Fujita b , Akira Ishihama b , Ryutaro Utsumi a ; * a Department of Agricultural Chemistry , Kinki University , 3327-204 Nakamachi , Nara 631-8505 , Japan b Department of Molecular Genetics , National Institute of Genetics , Mishima , Shizuoka 411-8540 , Japan Received 10 January 2000 ; received in revised form 13 March 2000 ; accepted 21 March 2000 Abstract To test whether OmpR is involved in regulation of the bolA1p , we investigated possible effects of ompR mutation on transcription from bolA1p .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10802181" target="_blank">10802181</a>]<br></li>
<li>Yamamoto , K. , Nagura , R. , Tanabe , H. , Fujita , N. , Ishihama , A. , and Utsumi , R. ( 2000 ) Negative regulation of the bolA1p of Escherichia coli K-12 by the transcription factor OmpR for osmolarity response genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16194231" target="_blank">16194231</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12366850" target="_blank">12366850</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12492863" target="_blank">12492863</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_30">
J Bacteriol 184:290 -- 301 Jubelin G , Vianney A , Beloin C , Ghigo J-M , Lazzaroni J-C , Lejeune P , Dorel C ( 2005 ) CpxR/OmpR interplay regulates curli gene expression in response to osmolarity in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21559929" target="_blank">21559929</a>]<br></label>
<input id="act_similarity_30", type="checkbox">
<div class="ACT_similar">
<li>[ 31 ] G. Jubelin , A. Vianney , C. Beloin , J.M. Ghigo , J.C. Lazzaroni , P. Lejeune , C. Dorel , CpxR/OmpR interplay regulates curli gene expression in response to osmolarity in Escherichia coli , J. Bacteriol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16487683" target="_blank">16487683</a>]<br></li>
<li>[ PubMed : 9351822 ] Jubelin G , Vianney A , Beloin C , Ghigo J-M , Lazzaroni J-C , Lejeune P , Dorel C. CpxR/OmpR interplay regulates curli gene expression in response to osmolarity in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20487295" target="_blank">20487295</a>]<br></li>
<li>Jubelin , G. , Vianney , A. , Beloin , C. , Ghigo , J.M. , Lazzaroni , J.C. , Lejeune , P. , and Dorel , C. ( 2005 ) CpxR/OmpR interplay regulates curli gene expression in response to osmolarity in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22250746" target="_blank">22250746</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17010156" target="_blank">17010156</a>]<br></li>
<li>, Lejeune , P. , and Dorel , C. ( 2005 ) CpxR/OmpR interplay regulates curli gene expression in response to osmolarity in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19432797" target="_blank">19432797</a>]<br></li>
<li>CpxR/OmpR Interplay regulates curli gene expression in response to osmolarity in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17468243" target="_blank">17468243</a>]<br></li>
<li>CpxR/OmpR interplay regulates curli gene expression in response to osmolarity in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16513732" target="_blank">16513732</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21415116" target="_blank">21415116</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16997959" target="_blank">16997959</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16077119" target="_blank">16077119</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23564175" target="_blank">23564175</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20466769" target="_blank">20466769</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24272779" target="_blank">24272779</a>]<br></li></div>
<li>The CpxR , OmpR and RcsB -- RcsA proteins are good candidates for the control of curli gene expression in response to cell envelope alterations .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16000739" target="_blank">16000739</a>]<br></li>
<li>This study was undertaken to clarify how the interplay between the two regulatory proteins , OmpR and CpxR , can affect the transcription of the curli gene in response to variation of the medium osmolarity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15743952" target="_blank">15743952</a>]<br></li>
<li>Expression of Outer Membrane Proteins in Escherichia coli Growing at Acid pH MIKIKO SATO , KAZUHIRO MACHIDA , ERI ARIKADO , HIROMI SAITO , TOMOHITO KAKEGAWA , AND HIROSHI KOBAYASHI * Faculty of Pharmaceutical Sciences , Chiba University , Inage-ku , Chiba 263-8522 , Japan Received 27 September 1999/Accepted 20 December 1999 It is generally accepted for Escherichia coli that ( i ) the level of OmpC increases with increased osmolarity when cells are growing in neutral and alkaline media , whereas the level of OmpF decreases at high osmolarity , and that ( ii ) the two-component system composed of OmpR ( regulator ) and EnvZ ( sensor ) regulates porin expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10698756" target="_blank">10698756</a>]<br></li>
<li>OmpR and RcsB abolish temporal and spatial changes in expression of flhD in Escherichia coli biofilm .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25667010" target="_blank">25667010</a>]<br></li>
<li>Slauch , J.M. , Garrett , S. , Jackson , D.E. , and Silhavy , T.J. ( 1988 ) EnvZ functions through OmpR to control porin gene expression in Escherichia coli K-12 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19432797" target="_blank">19432797</a>]<br></li>
<li>Differential expression of the OmpF and OmpC porin proteins in Escherichia coli K-12 depends upon the level of active OmpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16077119" target="_blank">16077119</a>]<br></li>
<li>Residues that are conserved in all four sequences are marked with asterisks 57 58 59 60 KdpE PhoB CreB OmpR * * * * ** * * ***** *** * ** GDGIEFIRDLRQ WSA VPVIVLSARSEESDKIAALDAGADDYLSKPFGIGELQARLRV GSGIQFIKHLKRESMTRDIPVVMLTARGEEEDRVRGLETGADDYITKPFSPKELVARIKA ISGFELCRQLLA LHPALPVLFLTARSEEVDRLLGLEIGADDYVAKPFSPREVCARVRT EDGLSICRRLRS QSNPMPIIMVTAKGEEVDRIVGLEIGADDYIPKPFNPRELLARIRA KdpE PhoB CreB OmpR 114 118 117 118 ** ** * ALRRHSATTAPDP LVKFSDVTVDLAARVIHRGEEEVHLTPIEFRLAGRCSTMPEK VMRRISP MAVEE VIEMQGLSLDPTSHRVMAGEEPLEMGPTEFKLLHFFMTHPER LLRRVKKFSTPSP VIRIGHFELNEPAAQISWFDTPLALTRYEFLLLKTLLKSPGR VLRRQANELPGAPSQEEAVIAFGKFKLNLGTREMFREDEPMPLTSGEFAVLKALVSHPRE KdpE PhoB CreB OmpR KdpE YSPSGPVLNQVWGPNAVEHSHYLRIYMGHLRQ 201 PhoB VYSREQLLNHVWGTNVYVEDRTVDVHIRRLRK 205 CreB VWSRQQLMDSVWEDAQDTYDRTVDTHIKTLRA 204 OmpR PLSRDKLMNLARGREYSAMERSIDVQISRLRR 210 FIG. 6 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1532388" target="_blank">1532388</a>]<br></li>
<li>EnvZcontrols theconcentration of phosphorylated OmpR to mediate osmoregulation of the porin genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8098993" target="_blank">8098993</a>]<br></li>
<li>Russo , F.D. , and Silhavy , T.J. ( 1991 ) EnvZ controls the concentration of phosphorylated OmpR to mediate osmo-regulation of the porin genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8821933" target="_blank">8821933</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_46">
EnvZ controls the concentration of phosphorylated OmpR to med-iate osmoregulation of the porin genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11812125" target="_blank">11812125</a>]<br></label>
<input id="act_similarity_46", type="checkbox">
<div class="ACT_similar">
<li>EnvZ controls the concentration of phosphorylated OmpR to mediate osmoregulation of the porin genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15466019" target="_blank">15466019</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8501030" target="_blank">8501030</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8113170" target="_blank">8113170</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10074086" target="_blank">10074086</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15175316" target="_blank">15175316</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9529892" target="_blank">9529892</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16077119" target="_blank">16077119</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9422609" target="_blank">9422609</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11717281" target="_blank">11717281</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7642497" target="_blank">7642497</a>]<br></li></div>
<li>J Biol Chem 264:18693 -18700 Russo FD , Silhavy TJ ( 1991 ) EnvZ controls the concentration of phosphorylated OmpR to mediate osmoregulation of the porin genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8413182" target="_blank">8413182</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_49">
Russo , F.D. , and Silhavy , T.J. ( 1991 ) EnvZ controls the concentration of phosphorylated OmpR to mediate osmoregulation of the porin genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10594827" target="_blank">10594827</a>]<br></label>
<input id="act_similarity_49", type="checkbox">
<div class="ACT_similar">
<li>Russo FD , Silhavy TJ ( 1991 ) EnvZ controls the concentration of phosphorylated OmpR to mediate osmoregulation of the porin genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17941710" target="_blank">17941710</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24062463" target="_blank">24062463</a>]<br></li></div>
<li>Flores-Valdez , M.A. , Puente , J.L. , and Calva , E. ( 2003 ) Negative osmoregulation of the Salmonella ompS1 porin gene independently of OmpR in an hns background .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22295907" target="_blank">22295907</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_52">
OmpR and LeuO positively regulate the Salmonella enterica serovar Typhi ompS2 porin gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18055596" target="_blank">18055596</a>]<br></label>
<input id="act_similarity_52", type="checkbox">
<div class="ACT_similar">
<li>Fernandez-Mora , M. , Puente , J.L. , and Calva , E. ( 2004 ) OmpR and LeuO positively regulate the Salmonella enterica serovar Typhi ompS2 porin gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22295907" target="_blank">22295907</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17493135" target="_blank">17493135</a>]<br></li></div>
<li>Genes most highly repressed upon IbsC , ShoB , LdrD and TisB overexpression.a Average fold repression dusB/yhdG ﬁs yeeF apt efeU ( ycdN ) CRP , IHF , Fis tRNA dihydrouridine synthase B Transcriptional activator Putative transporter Adenine phosphoribosyl transferase Inactive ferrous iron permease 6.5 7.7 0.9 0.9 0.8 0.9 1.2 1.1 1.1 1.2 1.2 2.1 4.3 3.8 4.7 2.3 1.2 17.8 6.1 21.2 CpxR Inactive ferrous iron transporter Inactive ferrous iron transporter Porin Unknown efeO ( ycdO ) efeB ( ycdB ) ompF yagU 14.3 2.6 6.1 26.7 1.0 1.8 1.4 1.7 1.2 1.8 0.9 2.5 1.2 1.2 0.5 1.2 OmpR , EnvYd a. Genes and operons which are repressed 5-fold in three independent experiments are listed with the average fold repression for the three experiments .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18710431" target="_blank">18710431</a>]<br></li>
<li>MetR MetR metH NR ( I ) , CRP ginApl ... ¬ -4 - II * NR ( I ) NR ( I ) a II I gInL NR ( I ) NFR ( l ) , ' OmpR OmpR O OmO pC pR momp R II OmpR OmpR OmpR ompF I OxyR OxyR ahpC  ' OxyROxyR   OxyR OxyR katG a a ~ ~ OxyR oxyR ' PapB I a PapB , CRP aPapB PapB papB , ' , +92 +125 PhoB phoA PhoB PhoB I Pho B PhoB PhoB phOB 4 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1943993" target="_blank">1943993</a>]<br></li>
<li>) 1,2 139 CRP cytR 0 1 p 1,2 142 CRP deop2 1 1 p 1,2 142 , 162 CRP galpi 1 1 0 1,2 60 CRP glpD 0 1 0 2,4 174 CRP glnApl 1 1 0 4 119 CRP ilvB 0 0 0 4 39 CRP lac 0 1 0 1,2 87 CRP malT 0 0 0 1,2 88 CRP melR 0 0 0 1,2 169 CRP nagB 1 0 0 4 108 CRP nagE 1 0 0 4 108 CRP pBRp4 0 0 0 2,4 113 CRP putA 1 1 0 4 47 CRP rpoHpS 1 0 0 4 99 CRP tnaA 0 0 0 2,4 27 CRP tsx 0 1 p 4 142 , 165 FNR dmsA 0 0 0 4 35 FNR fdrA 0 0 0 1,4 35 , 61 FNR , Y gipA 0 0 0 4 35 FNR , Y narG 0 0 0 1,4 35 , 75 FNR nirB 0 1 0 1,4 35 , 61 , 62 FNR PFLp6 1 0 0 4 133 FNR PFLp7 1 0 0 4 75 , 133 IlvY ilvC 1 0 p 3 170 MalT malE 1 1 R 1,2 114 , 115 MalT malK 1 1 R 1,2 114 , 115 MalT malPQ 0 0 0 1,4 138 MetR metE 1 1 0 1,3 161 MetR metH 0 0 0 4 160 , 161 OmpR ompC 0 0 p 1,2 80 , 81 , 154 OmpR ompF 0 0 p 1,2 38 , 116 , 154 OxyR ahpC 0 0 0 3 146 OxyR katG 0 0 0 3 146 , 149 PapB papB 1 1 R 1,2 37 , 41 PhoB phoA 0 0 0 4 84 PhoB phoB 0 0 0 1,4 84 , 154 PhoB phoE 0 0 P ( ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1943993" target="_blank">1943993</a>]<br></li>
<li>A wide range of environmental stimuli affect the expression of the sRNAs including oxidative-stress ( OxyR activates OxyS ) ; cell surface stress ( RcsB activates RprA ) ; glycine status of the cell ( GcvA activates GcvB ) ; osmotic imbalance ( OmpR activates OmrA/B ) ; quality of carbon source/growth rate [ McaS ; transcription factor ( s ) unknown ] ; low temperature [ DsrA ; transcription factor ( s ) unknown ] and O tension ( ArcA-ArcB 2 represses ArzA under anaerobic conditions ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22250746" target="_blank">22250746</a>]<br></li>
<li>Bacteria and plasmids used Relevant properties Reference 13 13 10 9 5 Strain F - 1acU169 araD rpsL relA thiflbB MC4100 envZII MC4100 A ( malT-ompB ) MC4100 envZ22 Apr ; vector , pBR322 ; cloned genes , ompR and envZ pYK12 ; ompR and enVZV-1243 MC4100 MH1461 SG480A76 SG477 pYK12 pSF112 OmpR in the cell containing the envZVaI243 allele was elevated 2.4-fold .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2160945" target="_blank">2160945</a>]<br></li>
<li>CpxA , the inner-membrane protein , phosphorylates its conserved histidine residue in response to signals and then transphosphorylates the conserved asparate residue in CpxR .2 -- 7 ) Activated CpxR , the phosphorylated form ( pCpxR ) , activates transcription of a set of genes , all of which are involved in envelope protein folding and proteolysis .8 -- 10 ) CpxP , a small periplasmic protein , is thought negatively to regulate CpxA activity by interacting with its periplasmic domain ,5,11,12 ) but CpxP is not required for induction of genes regulated by CpxA/CpxR in the response to envelope stress .13 ) NlpE is an outermembrane lipoprotein that activates the CpxA/CpxR regulon when overexpressed .14 ) Adhesion to hydrophobic surfaces induces the CpxA/CpxR regulon in a NlpE-dependent manner .15 ) CpxR is an OmpR family transcription factor that contains a helix-turn-helix motif at its N-terminus .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16861804" target="_blank">16861804</a>]<br></li>
<li>of T-even like phages and also acts as a porin with low permeability that allows slow penetration of small solutes ; induced by low pH during aerobic or anaerobic-growth but decreases by phosphate limitation and SOS induction Forms passive diffusion pores which allow small-molecular-wt hydrophilic materials across the outer membrane ; negatively regulated by Lrp Forms passive diffusion pores which allow low-mol wt hydrophilic materials across the outer membrane ; receptor for the bacteriophage T2 ; positively regulated by Lrp ; decreases during phosphate limitation and after benzoic acid treatment and treatment with superoxidegenerating agents such as menadione P33602 NuoG NusA 4.53 / 54,870.92 4.55 / 61,127 P0AFF6 133 , 179 , 212 , 228 , 294 P0A780 N utilization substance protein B 6.6 / 15,689.06 NusB 179 NusG 6.33 / 20,400.32 5.93 / 22,161 5.44 / 14,684 ( 5 -- 6 ) 6.40 / 21,860 ( 6 -- 11 ) P0AFG0 Transcription antitermination protein 179 , 286 , 287 OmpA 5.55 / 34,419 5.36 / 32,707 5.60 / 35,172.28 P0A910 Outer membrane protein A 75 , 179 , 228 , 274 , 287 , 293 , 294 , 321 , 323 5.34 / 28,591 5.50 / 33,875 ( DIGE 4.5 -- 6.5 ) 5.38 / 33,875 ( DIGE 4.5 -- 6.5 ) 5.23 / 14,346 ( 5 -- 6 ) 5.25 / 13,108 ( 5 -- 6 ) 4.48 / 38,307.50 -- / 36,000 Outer membrane protein C P06996 OmpC 61 OmpF 4.61 / 36,170 4.64 / 37,084.49 Outer membrane protein F P02931 61 , 88 , 179 , 228 , 293 , 294 , 321 4.56 / 37,316 ( DIGE 4.5 -- 6.5 ) 4.64 / 37,000 ( DIGE 4.5 -- 6.5 ) 4.71 / 37,000 ( DIGE 4.5 -- 6.5 ) Continued on facing pag The N terminus of this protein is required for the transcriptional expression of both major outer membrane protein genes ompF and ompC ; its carboxyl-terminal moiety mediates the multimerization of the ompR protein Protease that can cleave T7 RNA polymerase , ferric enterobactin receptor protein ( FEP ) , antimicrobial peptide protamine , and other proteins Induced by low or high pH 79 OmpR P0AA16 Transcriptional regulatory protein 6.04 / 27,353.62 1 OmpT Protease VII P09169 179 , 321 5.38 / 33,477.72 5.36 / 37,000 ( DIGE 4.5 -- 6.5 ) OmpX Outer membrane protein X P0A917 5.30 / 16,382.89 5.25 / 16,126 ( DIGE 4.5 -- 6.5 ) 5.15 / 16,000 ( DIGE 4.5 -- 6.5 ) 5.93 / 56,137 5.70 / 57,487 5.64 / 56,808 5.44 / 54,493 5.93 / 55,635 ( DIGE 4.5 -- 6.5 ) 5.82 / 55,131 ( DIGE 4.5 -- 6.5 ) 5.71 / 52,681 ( DIGE 4.5 -- 6.5 ) 5.13 / 50,973 ( 5 -- 6 ) 5.47 / 50,921 ( 5 -- 6 ) 5.63 / 50,921 ( 5 -- 6 ) 5.65 / 50,915 ( 5.5 -- 6.7 ) 179 , 274 , 321 Component of the oligopeptide permease and binding-protein-dependent transport system ; binds peptides up to 5 amino acids long with high afﬁnity ; increases by high pH during anaerobic-growth and after benzoic acid treatment OppA 5.85 / 58,359.84 Periplasmic oligopeptide-binding protein P23843 179 , 228 , 286 , 287 , 294 , 321 , 323 Part of the binding-protein-dependent transport system for oligopeptides ; probably responsible for energy coupling to the transport system Provides resistance to osmotic stress ; may be important for stationary-phase survival ; induced by elevated osmotic pressure in the growth medium Preferentially metabolizes organic hydroperoxides over inorganic hydrogen peroxide ; induced by elevated osmotic pressure and during the decelerating phase , before entry into stationary phase ; regulated by a complex network of several global regulators , including at least S , Lrp , and H-NS Induced by elevated osmotic pressure Osmotically and stationary phase inducible and controlled by several global regulators , including S , cAMP-CRP , IHF , and Lrp Induced by growth in glucose-mineral medium of elevated osmotic strength ; controlled by S Induced by growth in glucose-mineral medium of elevated osmotic strength ; controlled by S May play a role in bacterial envelope integrity ; strongly associated with the peptidoglycan Involved in pantothenate biosynthesis : ﬁrst step See PanB : ﬁnal step .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16760308" target="_blank">16760308</a>]<br></li>
<li>Surprisingly , even though plasmids pYqjA-YqjB and pYgdD-GcvA act upstream of OmpR to activate omrA-lacZ and omrB-lacZ fusions , they had a stronger effect on these fusions than pOmpR itself ( Fig. 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16359331" target="_blank">16359331</a>]<br></li>
<li>Expression of an ompR -- lacZ fusion was slightly increased in a Dtol mutant , but two-dimensional electrophoretic analysis of the OmpR proﬁle did not reveal any signiﬁcant modiﬁcation , since our OmpR antibody detected only one spot for OmpR ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16000739" target="_blank">16000739</a>]<br></li>
<li>Preliminary experiments using a chromosomal csgD-3xFLAG gene carrying the M4 mutation ( Figure 3 ) indicates that most of the inhibition of CsgD synthesis is due to direct targeting of csgD rather than through OmpR ; csgD mRNA levels were decreased to E50 % in the presence of OmrA-wt ( which matches the ompR but not csgD-M4 target ) , but to E90 % by OmrA-M4 ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20407422" target="_blank">20407422</a>]<br></li>
<li>When the csgD gene is in its normal chromosomal context , this repression ( Figure 3 ) must reﬂect the combination of at least two effects : one is the direct targeting of the csgD mRNA , the other inhibited synthesis of OmpR ( see above ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20407422" target="_blank">20407422</a>]<br></li>
<li>The present study reveals that , in addition to transcriptional repression by phosphorylated OmpR ( Shin and Park , 1995 ) and HdfR ( Ko and Park , 2000 ) and activation by cAMP-CRP ( Soutourina et al. , 1999 ) , flhDC is regulated post-transcriptionally by CsrA-mediated transcript stabilization .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11298291" target="_blank">11298291</a>]<br></li>
<li>Expression of FlhDC is tightly regulated by multiple transcription factors , including CRP , H-NS and OmpR , and hence the genes of the ﬂagellar cascade are regulated in response to many different environmental inputs ( Shin and Park , 1995 ; Sout-ourina et al. , 1999 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19843224" target="_blank">19843224</a>]<br></li>
<li>These environmental signals are mediated at the gene expression level by a number of regulators , such as OmpR , H-NS , Besides its role as activator of the csgBA operon , CsgD regulates a number of genes involved in biofilm formation and production of cell-surface-associated structures ( Latasa et al. , 2005 ; Gibson et al. , 2006 ) , as well as genes involved in transport , metabolism and gene regulation ( Chirwa & Abbreviation : CV , crystal violet .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18599830" target="_blank">18599830</a>]<br></li>
<li>RESULTS ArcA downregulates the transcription of fatty acid transport genes under anaerobic conditions The expression of fadL , which encodes one of the outer-membrane-bound proteins involved in the uptake of fatty acids , is regulated by at least three transcription factors : Crp , OmpR and FadR ( Berg et al. , 2004 ; Busch & Saier , 2002 ; Clark & Cronan , 1996 ; DiRusso et al. , 1992 ; Cronan & Subrahmanyam , 1998 ; Higashitani et al. , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16849788" target="_blank">16849788</a>]<br></li>
<li>( d ) Homology among the PhoB , OmpR and Dye proteins Our previous work ( Shinagawa et al. , 1983 ) and that by Guan et al. ( 1983 ) suggested that the phoB gene is physiologically regulated by inorganic phosphate in the medium and genetically regulated by phoR , phoM and a cluster of genes in the pst region , in the same manner as phoA expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3537313" target="_blank">3537313</a>]<br></li>
<li>On the basis of additional homology within the carboxy-terminal domain , TorR has been recognized as a member of the OmpR subfamily of response-regulators , which includes numerous members in E. coil ( such as ArcA , OmpR , or PhoB ) but also in many other bacterial species ( Aiba et aL , 1993 ; Mills et aL , 1993 ; Nagasawa et aL , 1993 ; Parkinson and Kofoid , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596446" target="_blank">8596446</a>]<br></li>
<li>For some TorR-related proteins such as OmpR or ArcA , it has also been proposed that phosphoryl-ation increases the tendency of the N-terminal ( receiver ) domain to form multimers ( Nakashima et aL , 1991 ; Tardat and Touati , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596446" target="_blank">8596446</a>]<br></li>
<li>It is very well documented in the case of OmpR , the regulatory protein most related to CpxR , that excess of P ~ OmpR in the cell switches its function from activator to repressor ( Rampersaud et al. , 1994 ; Harlocker et al. , 1995 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9130712" target="_blank">9130712</a>]<br></li>
<li>Since onipF transcription is dependent on OmpR ( Norioko et al. 1986 ; Mizuno et al. 1988 ) , it was of interest to examine whether increased amounts of this activator could reverse IHF inhibition .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1736095" target="_blank">1736095</a>]<br></li>
<li>To determine whether this was the case , the OmpR protein was purified according to the method of Jo et al. ( 20 ) and added to the in vitro transcription reactions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1906064" target="_blank">1906064</a>]<br></li>
<li>Increasing the half-life of OmpR ~ P ( through envZ mutations or MzrA overexpression ) would facilitate higher-affinity interaction with DNA ( Head et al. , 1998 ) and would enable OmpR to-bind to promoters it does not normally regulate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19432797" target="_blank">19432797</a>]<br></li>
<li>Both the unphosphorylated and phosphorylated forms of OmpR bind to the target sites , but phospho-OmpR has demonstrably higher affinity ( Aiba et al. , 1989c ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8098993" target="_blank">8098993</a>]<br></li>
<li>Phosphorylation may promote oligomerization of OmpR , which is normally monomeric , thereby enhancing its ability to-bind at tandemly repeated target sites ( Nakashima et al. , 1991 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8098993" target="_blank">8098993</a>]<br></li>
<li>Further supporting these results , it was found that part of the sequence bound by OmpR shares similarity with the OmpR consensus sequence previously proposed ( Harlocker et al. , 1995 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25667010" target="_blank">25667010</a>]<br></li>
<li>For the EnvZ -RCB- OmpR system it has recently been shown that P C OmpR is stabilized when bound to its own target promoter DNA ( Qin et al. , 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11739766" target="_blank">11739766</a>]<br></li>
<li>This followed a study that demonstrated enhancement of OmpR phosphorylation when it was bound to promoter DNA ( Ames et al. , 1999 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11739766" target="_blank">11739766</a>]<br></li>
<li>It is also possible that IHF regulates-the-activity of other kinases or phosphatases which , under certain conditions , may alter the binding properties of OmpR ( Igo et al. 1989b ; Slauch and Silhavy 1989 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1736095" target="_blank">1736095</a>]<br></li>
<li>We also Stimulation of OmpR is responsible for the activation of csgDEFG in the Dtol mutant In the Dtol mutants , the Rcs phosphorelay is activated ( Clavel et al. , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16000739" target="_blank">16000739</a>]<br></li>
<li>Expression of these operons is positively regulated by OmpR ( via the activation of CsgD ) ( Romling et al. , 1998 ) , and negatively regulated by CpxR ( Dorel et al. , 1999 ; Otto & Silhavy , 2002 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16000739" target="_blank">16000739</a>]<br></li>
<li>Moreover , CpxR and the other nine regulators belong to the same OmpR subfamily , and tor function ( Stock et al. , 1989 : Parkinson and Kofoid , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8294007" target="_blank">8294007</a>]<br></li>
<li>These include activating factors as , e.g. cAMP-CRP , H-NS and CsrA as well as repressing factors such as RcsB/A , OmpR and LrhA ( recently summarized in Prüß et al. , 2006 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17537210" target="_blank">17537210</a>]<br></li>
<li>cAMP-CRP ( Jackson et al. 2002a ) , RpoS ( Adams and McLean 1999 ) and OmpR ( Vidal et al. 1998 ) stimulate or repress bioWlm formation in E. coli by regulating , directly Communicated by Jorge Membrillo-Hernández .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21559929" target="_blank">21559929</a>]<br></li>
<li>Conversely , an insertion in OmpR decreases GE activity -4 - fold , probably because such strains underexpress OmpC and OmpF ( Mecsas et al. , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7889934" target="_blank">7889934</a>]<br></li>
<li>Furthermore , the transport of LCFAs is abolished during high external osmotic pressure , due to repression by OmpR ( Higashitani et al. , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16849788" target="_blank">16849788</a>]<br></li>
<li>It is also known that LCFA transport ( fadL ) is repressed by OmpR in response to the high external osmotic pressure ( Clark & Cronan , 1996 ; Higashitani et al. , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16849788" target="_blank">16849788</a>]<br></li>
<li>AcP acts as a phospho-donor for the autophosphorylation of the proteins CheY and OmpR ( Lukat et ai , 1992 ) , NtrC ( Feng et ai .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7934904" target="_blank">7934904</a>]<br></li>
<li>This homology was equally spread over all functional domains on NtrC and GInG , as delineated by Drummond et al. ( 11 ) , although in the case of HydG , the interdomain region C was not particularly hydro-philic , as had been found to be the case for NtrC and related proteins such as OmpR and SfrA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2666400" target="_blank">2666400</a>]<br></li>
<li>However , because the Cpx pathway controls the transcription of degP ( Danese et al. 1995 ; Raina et al. 1995 ) , and because CpxR is homologous to the OmpR subfamily of two-component transcription factors ( Dong et al. 1993 ) , the simplest interpretation of the results presented in Figure lb is that activation of the Cpx pathway stimulates dsbA transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9159399" target="_blank">9159399</a>]<br></li>
<li>CpxR shares homology with other two-component DNA-binding proteins , including OmpR and ArcA ( Dong et al. 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7883164" target="_blank">7883164</a>]<br></li>
<li>Based on sequence homologies with other regulatory proteins of two component systems , such as OmpR or CheA , it seems that CpxR ( Dong et al. , 1993 ) could be activated through phosphorylation by CpxA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9130712" target="_blank">9130712</a>]<br></li>
<li>The CpxR and CpxA proteins ( Weber and Silverman , 1988 ; Dong et al. , 1993 ) are highly homologous to the OmpR EnvZ two component system .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9130712" target="_blank">9130712</a>]<br></li>
<li>None of the transcription factors involved in fatty acid regulation ( Crp , OmpR and FadR ) is oxygen sensing , though , recently , the ArcBA two-component signal transduction system , which senses oxygen indirectly through a change in redox potential , has been implicated in the negative regulation of aerobic fatty acid metabolism ( Georgellis et al. , 2001 ; Clark & Cronan , 1996 ; Malpica et al. , 2004 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16849788" target="_blank">16849788</a>]<br></li>
<li>However , as Mlc shows no homology to any of the known effector components of two-component systems , e.g. OmpR , PhoB , NtrC , this would seem to exclude a classical two-component-type regulatory mechanism .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10411743" target="_blank">10411743</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_98">
Lee , A.K. , Detweiler , C.S. , and Falkow , S. ( 2000 ) OmpR regulates the two-component system SsrA -- SsrB in Salmonella pathogenicity island 2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21276095" target="_blank">21276095</a>]<br></label>
<input id="act_similarity_98", type="checkbox">
<div class="ACT_similar">
<li>OmpR regulates the two-component system SsrA-SsrB in Salmonella pathogenicity island 2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15995204" target="_blank">15995204</a>]<br></li>
<li>OmpR regulates the two-component system SsrA-ssrB in Salmonella pathogenicity island 2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14665678" target="_blank">14665678</a>]<br></li></div>
<li>In sequence alignment comparisons , this Asp residue is conserved in all response-regulators ( Fig. 2 ) , and it has been shown to be the site of phosphorylation , and thereby activation , of CheY , OmpR , and VirG ( 15 , 18 , 27 ; reviewed in reference 40 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2066339" target="_blank">2066339</a>]<br></li>
<li>The residue corresponding to Asp-57 of CheY is invariant in response-regulators and has been shown to be the site of protein phosphorylation in CheY , OmpR , and VirG ( 15 , 18 , 27 ; reviewed in reference 40 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2066339" target="_blank">2066339</a>]<br></li>
<li>Acetyl-P is also known to phosphorylate various response-regulators , including CheY , NRI , PhoB , and OmpR ( 11 , 28 , 52 ) , and is thus suggested to inﬂuence either the sensitivity or the magnitude of an adaptive response ( 29 , 51 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7642497" target="_blank">7642497</a>]<br></li>
<li>between EnyZ and OmpR were first shown to occur between the histidine kinase NRI , ( NtrB ) and NRI ( NtrC ) of the nitrogen regulation system ( 23 , 32 , 45 ) and between the histidine kinrase CheA and two effector-molecules , CheY and CheB , of the chemotaxis response ( 15 , 16 , 42 , 43 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2160945" target="_blank">2160945</a>]<br></li>
<li>Likewise , OmpR , CheY , CheB , and NRI belong to a structurally conserved class of effector-molecules in which the aspartic acid residues represented by Asp-12 and Asp-55 of OmpR are invariant ( 41 , 43 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2160945" target="_blank">2160945</a>]<br></li>
<li>These include CreB and PhoB of enteric bacteria which are responsive to cytoplasmic catabolite concentrations and extracellular inorganic phosphate , respectively ; OmpR of enteric bacteria , involved in responses to osmotic conditions ; VirG proteins of agrobacterial species , activators of the Ti or Ri plasmid-encoded virulence op-erons ; and ArcA , an E. coli activator of genes encoding proteins which function in aerobic metabolism .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7699720" target="_blank">7699720</a>]<br></li>
<li>In reexamining the homology in light of the corrected OmpR sequence , the two proteins appear to be even more closely related than was originally thought ; when the two proteins are realigned , homology increases from thb 28 % reported by Drury and Buxton ( 3 ) to approximately 40 % , indicating a very high degree of functional similarity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2997120" target="_blank">2997120</a>]<br></li>
<li>Lane 1 , total proteins produced by JA221 containing pBR322 ; lane 2 , total proteins produced by JA221 containingpSN142 ; lane3 , purified OmpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3023382" target="_blank">3023382</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3023382" target="_blank">3023382</a>]<br></li>
<li>The N-terminal two-thirds of the NarL protein has extensive homology with those of a diverse set of prokaryotic regulatory proteins , including OmpR , PhoB , SfrA , UhpA , CheY , CheB , NtrC , DctD , FixJ , VirG , SpoOF , and SpoOA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2657652" target="_blank">2657652</a>]<br></li>
<li>The amino acid sequence of PhoB protein is similar to those of several regulatory proteins , including OmpR , Dye , ORF2 ( of the phoM operon ) , NtrC , VirG , SpoOA and SPOOF , most of which are probably transcriptional-regulators for global genetic regulatory systems in bacteria ( Comeau et al. , 1985 ; Drury & Buxton , 1985 ; MacFarlane & Merrick , 1985 ; Amemura al. , 1986 ; IIrummond al. , 1986 ; et et Nixon et al. , 1986 ; Winans et al. , 1986 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3054125" target="_blank">3054125</a>]<br></li>
<li>The carboxyl ` terminal domain of NarL shares only short patches of similarity with PhoB , OmpR , and SfrA , but it is strikingly similar to the analogous regions of DegU , UhpA , and FixJ ( Fig. 3 and Table 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2649492" target="_blank">2649492</a>]<br></li>
<li>The induction was independent of the RelA/SpoT , RpoH , KatF , OmpR , AppY , Lrp , PhoB and H-NS proteins during stress conditions that are known to induce or activate these global regulators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1453957" target="_blank">1453957</a>]<br></li>
<li>The corresponding regulatory proteins ( in gray rounded rectangles ) are FadR , the CRP-cAMP complex , ArcA-P , and OmpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20622065" target="_blank">20622065</a>]<br></li>
<li>The sizes of these proteins are similar ; the PhoB , OmpR and Dye proteins consist of 228 , 239 and 238 amino acids , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3537313" target="_blank">3537313</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3537313" target="_blank">3537313</a>]<br></li>
<li>The match to the other proteins , although obvious , was weaker , ranging from about 25 % identity with NtrC and CheY to 19 % for OmpR , SfrA , and PhoB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3042748" target="_blank">3042748</a>]<br></li>
<li>The PhoB , OmpR and VirG proteins appear to be more related to one another than to NarL as they have 34 % -40 % identity in the B domain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2648330" target="_blank">2648330</a>]<br></li>
<li>Therefore , the most proteins of this family are subdivided into four different groups according to their C-terminal domain : ( i ) NarL , DegU , UvrC ORF2 , UhpA , and FixJ ; ( ii ) OmpR , PhoB , PhoM ORF2 , SfrA , and VirG ; ( iii ) NtrC , DctD , and PgtA ; and ( iv ) CheY and SpoOF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2657652" target="_blank">2657652</a>]<br></li>
<li>One family includes OmpR , PhoB , SfrA , VirG , CheB , CheY , Spo0A , SpoOF , NtrC , and DctD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3042748" target="_blank">3042748</a>]<br></li>
<li>As a consequence of the homology with NtrC , there was also homology with other regulatory proteins which respond to other environmental stimuli such as phosphate limitation ( PhoB ) , altered osmolarity ( OmpR ) , presence of female cells ( SfrA ) , presence of plant exudate , .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2666400" target="_blank">2666400</a>]<br></li>
<li>Extensive similarity of N-terminal amino acid sequence of NarL to other prokaryotic regulatory proteins such as OmpR and PhoB ( Fig. 3 ) strongly suggests that NarL acts directly on the regulatory region of the narK and narC operons after some modification to an active form in the presence of nitrate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2657652" target="_blank">2657652</a>]<br></li>
<li>Amino acid sequence comparisons showed that UhpA was homologous to a family of bacterial regulatory proteins , some of which act as transcriptional-activators ( OmpR , PhoB , NtrC , and DctD ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3042748" target="_blank">3042748</a>]<br></li>
<li>Primary amino acid sequence homology alignment of NarL to OmpR , VirG , and PhoB proteins from procaryotic microorganisms .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2648330" target="_blank">2648330</a>]<br></li>
<li>In the presence of 8 mM F1 -- F2 -- F3 ( note that one DNA fragment contains six OmpR-P binding sites , and at the start of the reaction the molar ratio between OmpR and DNA is 1:2 ) , approximately 6.4 % of OmpR was converted into the phosphorylated form after 35 min incubation at room temperature , as quantified by comparing with serial dilution of OmpR-P produced by GST -- EnvZc11 ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11158569" target="_blank">11158569</a>]<br></li>
<li>OmpR produced in the parent strain grown in the presence of 20 % sucrose showed a 270-fold-higher DNA-binding activity , and the envZJJ allele stimulated DNA-binding activity approximately 230-fold relative to the level found in the parent strain in the absence of sucrose .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2656631" target="_blank">2656631</a>]<br></li>
<li>Although the partially purified OmpR from MC4100 grown in nutrient broth did not produce detectable bound forms of fragment I-Ilb-IIl ( lanes 1 to 3 ) , the DNA-binding activity of the partially pure OmpR from MC4100 grown in the presence of 20 % sucrose became detectable at the lower levels of added OmpR ( lane 4 ) and progressively increased as more OmpR was added ( lanes 5 and 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2656631" target="_blank">2656631</a>]<br></li>
<li>In contrast to the OmpR from the parent , partially purified OmpR derived from the envZll strain still exhibited very high binding activity ; as little as 25 ng was required to detect bound forms of fragment I-Ilb-Ill ( lane 7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2656631" target="_blank">2656631</a>]<br></li>
<li>However , in the presence of DNA , a significantly higher amount of OmpR is able to remain phosphorylated , indicating one or more of the above reactions is affected by DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11158569" target="_blank">11158569</a>]<br></li>
<li>Therefore , it seems likely that OmpR , in its phosphorylated form , binds to DNA sequences upstream of position − 60 of the omrA and omrB promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16359331" target="_blank">16359331</a>]<br></li>
<li>and the ompC promoterfragments , suggesting that OmpR binds to specific DNA sequences within these fragments .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3023382" target="_blank">3023382</a>]<br></li>
<li>Afﬁnity cleavage reactions suggest that OmpR binds to DNA targets in direct repeats ( 17 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11976297" target="_blank">11976297</a>]<br></li>
<li>Recall that for the sigma 70 activators OmpR and CRP , activation diminishes precipitously as the DNA elements are moved out of the nearby proximal position ( 40 , 81 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1943993" target="_blank">1943993</a>]<br></li>
<li>This suggests that in the absence of Nonidet P-40 , OmpR may form amultimer or aggregate that is unable to-bind to DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3023382" target="_blank">3023382</a>]<br></li>
<li>Unlike OmpR , the DBDs of both PhoB and KdpE bind their cognate DNA with moderate affinities ( range of Kds 100 -- 400 nM ) ; however , only the DBD of PhoB can initiate transcription [ 40 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22291906" target="_blank">22291906</a>]<br></li>
<li>Considering the high degree of similarity in the structure of the OmpR family of proteins , the corresponding loop in TorR is a candidate for TorI binding , and ongoing studies focus on the characterization of TorI TorR interactions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15197250" target="_blank">15197250</a>]<br></li>
<li>The slow mobility complexes , observed when the two proteins are simultaneously added into the binding reactions ( Fig. 4A , lanes 3 to 6 ) , also upshift in the presence of anti-OmpR antibodies ( Fig. 4B , lanes 3 to 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15743952" target="_blank">15743952</a>]<br></li>
<li>The culture was monitored until thecells were in RESULTS mid-log growth andthenharvested by centrifugation.Forprocaine DNA Shift Experiments with Phosphorylated OmpR-DNA treatment , procaine was added to tcoeallsfinal concentrationof 10mM , binding of phosphorylated and nonphosphorylated OmpR was and cells were harvested1h later.Cell pellets were suspended in m0 5M Tris , 5 % glycerol , and 5 mM glycine betaine , pH 7.8 ( FPLC buffer ) and examined using a 314-bp DNA fragment encompassing the redisrupted by passage through a French Press .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8175665" target="_blank">8175665</a>]<br></li>
<li>The mechanism by which OmpR phosphorylation stimulates ompC expression may involve enhanced binding of the OmpR-phosphate to its cognate DNA-binding sequence of ompC ( 2 ) , facilitation of open complex formation ( 36 , 45 ) , or stabilization of the binding of RNA polymerase to the promoter sequence of ompC ( 29 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2160945" target="_blank">2160945</a>]<br></li>
<li>The inability of KdpEDBD to promote gene expression is similar to that of the DBD of OmpR [ 39 ] and is distinct from the DBD of PhoB , which expresses the reporter gene by binding with 7-fold greater affinity than unphosphorylated PhoB to the pho box sequence [ 40 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22291906" target="_blank">22291906</a>]<br></li>
<li>The downstream binding region from 14 to 149 is further extended to 217 , including the transcription initiation site , when OmpR is phosphorylated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7642497" target="_blank">7642497</a>]<br></li>
<li>The osmoregulator protein , OmpR , when phosphorylated , negatively regulatesflagellar synth-esis by binding to the promoter region of the flhD operon ( Shin and Park , 1995 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8866483" target="_blank">8866483</a>]<br></li>
<li>The cyclic AMP ( cAMP ) receptor protein-cAMP complex and the OmpR protein are known to affect the expression of the ﬂhDC operon by binding to its promoter region ( 21 , 23 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10913108" target="_blank">10913108</a>]<br></li>
<li>The functional similarity between tppB and ydgR , as well as the dependence of ydgR transcription on OmpR observed in our DNA microarray experiments , led us to hypothesize that ydgR is equivalent to the serovar Typhimurium gene tppB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15175316" target="_blank">15175316</a>]<br></li>
<li>The inability of mutant KdpE R193A to-bind to DNA is interesting because in most other family members the equivalent residue is of opposite charge as observed in PhoB ( D196 ) and OmpR ( D202 ) ( Fig .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22291906" target="_blank">22291906</a>]<br></li>
<li>The apparent dissociation constant for EnvZc binding to OmpR in the curve shown is 519 nM , for T83I it is 199 nM .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11812125" target="_blank">11812125</a>]<br></li>
<li>The autophosphorylation of EnvZc and the phosphotransfer from phosphorylated EnvZc to OmpR were not affected by DNA , whereas the autodephosphorylation of OmpR-P was inhibited slightly .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11158569" target="_blank">11158569</a>]<br></li>
<li>The total fraction of bound DNA was then plotted as a function of OmpR concentration .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9096305" target="_blank">9096305</a>]<br></li>
<li>Results The Effect of DNA on the Overall Phosphorylation Reaction of OmpR by EnvZc .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11158569" target="_blank">11158569</a>]<br></li>
<li>The different OmpR molecules eluted from the DEAE-cellulose column at the same salt concentration , and only a single peak of OmpR was detected , as judged by Western blot analysis and DNA-binding activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2656631" target="_blank">2656631</a>]<br></li>
<li>The DNA binding activityof OmpR is itself regulatedby For example , pleiotropic envZ mutants such as em2473 and phosphorylationthroughthekinaseproteinEnvZ .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8175665" target="_blank">8175665</a>]<br></li>
<li>The primary result of this action seems to be changes in the DNA-binding properties of OmpR ( 1 , 6 , 25 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1648075" target="_blank">1648075</a>]<br></li>
<li>The N-terminalportion of the OmpR protein is phosphorylated , and consequently the DNA-binding properties of the C-terminal portionwould bechanged .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2668281" target="_blank">2668281</a>]<br></li>
<li>The DNA binding activityof this pg ) , filteredthrougha 0.22-pm filter , anddiluted 2-fold withFPLC newly phosphorylated OmpR was examined by mixing a porbuffer .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8175665" target="_blank">8175665</a>]<br></li>
<li>Since the F2 site is not capable of independently binding OmpR even in the presence of the kinase , these experiments also suggest that cooperative interactions play an important role in the binding of OmpR to these sites and that phosphor-ylation stimulates this cooperativity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9096305" target="_blank">9096305</a>]<br></li>
<li>However , these experiments establish two basic aspects of the interaction of OmpR with its target sites : the binding of the unphosphorylated form of OmpR can be cooperative and these cooperative interactions can be further stimulated by phosphor-ylation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9096305" target="_blank">9096305</a>]<br></li>
<li>It should be noted that one of the OmpR binding sites infadL is also located in the region between-40 and - 80 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8413182" target="_blank">8413182</a>]<br></li>
<li>The binding of OmpR to these regions may be facilitated by the presence of the bent DNA in the region between - 101 and -71 and by IHF binding at one or both of its sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8113170" target="_blank">8113170</a>]<br></li>
<li>When CD6 was used as a probe , both OmpR and IHF bound to the same probe ( Fig. 9b ) , as expected from the map of binding sites ( Fig. 9a ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20466769" target="_blank">20466769</a>]<br></li>
<li>Since there are two IHF-binding sites within IHF-2 , the CD3 probe formed multiple species of OmpR -- IHF -- CD3 complex , e.g. IHF -- OmpR -- CD3 and IHF -- IHF -- OmpR -- CD3 , should be formed in the simultaneous presence of two factors , thereby giving a broad distribution of complexes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20466769" target="_blank">20466769</a>]<br></li>
<li>It is likely that members of the NarL family bind to dyad targets , in contrast to the binding of OmpR family response-regulators to direct-repeat targets .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11976297" target="_blank">11976297</a>]<br></li>
<li>To determine if this sequence functions as a binding site for the OmpR protein , we assayed binding of the FPLC-puriﬁed OmpR and OmpR234 proteins to DNA fragments containing the transcriptional-regulatory region of csgD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11717281" target="_blank">11717281</a>]<br></li>
<li>In order to understand how OmpR interacts with CpxR to affect regulation at the csgD locus , we ﬁrst assayed the binding of each puriﬁed regulator to a DNA fragment containing the transcriptional-regulatory region of csgD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15743952" target="_blank">15743952</a>]<br></li>
<li>Okamura , H. , Hanaoka , S. , Nahadoi , A. , Makino , K. , and Nishimura , Y. ( 2000 ) Structural comparision of the PhoB and OmpR DNA-binding/transactivation domain and the arrangement of PhoB molecules on the phosphate box .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12123454" target="_blank">12123454</a>]<br></li>
<li>Structural comparison of the PhoB and OmpR DNA-binding/transactivation domains and the arrangement of PhoB molecules on the phosphate box .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11976297" target="_blank">11976297</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11489853" target="_blank">11489853</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10986267" target="_blank">10986267</a>]<br></li>
<li>Model 1 also includes the possibility that NanR binds to DNA as a monomer , as in the case of WH proteins of the OmpR family , which bind to tandem repeats with a monomer binding to each successive major groove ( 4 , 15 , 16 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12897000" target="_blank">12897000</a>]<br></li>
<li>We suggest that CpxR , via its speciﬁc binding sites arranged over an extended region of DNA , functions as a modulator of OmpR activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15743952" target="_blank">15743952</a>]<br></li>
<li>We further proposed that the main difference between activation and repression of ompF entails differential DNA binding on the part of OmpR ( 39 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1648075" target="_blank">1648075</a>]<br></li>
<li>Moreover , the mutations are in the region that by deletion analysis is vital for ompF activation and OmpR DNA binding in general ( 15 , 16 , 24 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1648075" target="_blank">1648075</a>]<br></li>
<li>For T215 of KdpE , variants with mutations at equivalent residues in OmpR and PhoB are also defective in DNA binding due to loss of H-bonding with the DNA backbone [ 28,49 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22291906" target="_blank">22291906</a>]<br></li>
<li>A different route to gene regulation was proposed for OmpR wherein DBD of OmpR binding to DNA at low affinity was incapable of transcriptional initiation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22291906" target="_blank">22291906</a>]<br></li>
<li>This finding suggests either that other proteins or factors in the extract helped to stabilize the binding of OmpR to the smaller oligonucleotide fragment or that the affinity of OmpR for the DNA fragment was altered during purification .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2656631" target="_blank">2656631</a>]<br></li>
<li>DISCUSSION In this paper , we examined the effects of phosphorylation of the transcriptional-regulator OmpR on its binding to single and multiple DNA-binding sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9096305" target="_blank">9096305</a>]<br></li>
<li>Either they did not systematically compare OmpR binding in the presence and absence of kinase or they did not clearly separate the effects of phosphorylation on intrinsic binding vs. cooperative binding of OmpR to the DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9096305" target="_blank">9096305</a>]<br></li>
<li>Comprehensive analysis of OmpR phosphorylation , dimerization , and DNA binding supports a canonical model for activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25161193" target="_blank">25161193</a>]<br></li>
<li>Although previous studies have examined the binding of OmpR to DNA , the impact of OmpR phosphorylation on this binding has never been fully evaluated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9096305" target="_blank">9096305</a>]<br></li>
<li>Thus , the dramatic differences in DNA-binding activity we observed were not due to dialyzable inhibitory or stimulatory components in the extracts and presumably reflect changes in the binding affinity of OmpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2656631" target="_blank">2656631</a>]<br></li>
<li>In order to eliminate the g possibility of nonspeciﬁc binding of OmpR , the remaining portion of pSS11 DNA was digested with BglI and ScaI , yielding 178 - , 364 - , 904 - , and 1,421-bp fragments of DNA , which were then used as nonspeciﬁc DNA probes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7642497" target="_blank">7642497</a>]<br></li>
<li>In order to monitor nonspecific DNA binding by OmpR , a 32P-labeledBamHI - at -73 and -71 were changed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2553720" target="_blank">2553720</a>]<br></li>
<li>DNA binding of OmpR and T83I to the high af ® nity F1 and C1 binding sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11812125" target="_blank">11812125</a>]<br></li>
<li>DNase I footprinting analysis was also performed to assess the DNA binding ability of OmpR and T83I .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11812125" target="_blank">11812125</a>]<br></li>
<li>However , whether CpxR competes with OmpR for binding to the DNA is an open question .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15743952" target="_blank">15743952</a>]<br></li>
<li>Keywords : OmpR ; OmpF ; Osmoregulation ; DNA-binding ; Escherichia cob 1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7557323" target="_blank">7557323</a>]<br></li>
<li>DNA binding activities of pure OmpR following phosfootprinting of the region between -360 to -380 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8175665" target="_blank">8175665</a>]<br></li>
<li>Phosphorylation of OmpR enhances its DNA-binding activity [ 7 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9016718" target="_blank">9016718</a>]<br></li>
<li>B , DNA-binding assay of OmpR to the digest of pGR141 ( omPC ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3023382" target="_blank">3023382</a>]<br></li>
<li>First , the ompF4 and ompFJ8 mutations directly affect the binding of OmpR to the DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1648075" target="_blank">1648075</a>]<br></li>
<li>ProColumn fractions of 0.75 ml were collected in 1.5-ml microcentrifuge tein-DNA complexes were allowed to form , and complexes were tubes containing0 .25 ml of 80 % glycerol.Asmall aliquot was takefrnom resolved by nondenaturing electrophoresis.A representative each tube and assayed for OmpR DNA binding activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8175665" target="_blank">8175665</a>]<br></li>
<li>Diluted OmpR was added to each reaction mixture at the concentrations indicated in the legend to Fig. 3 , and the DNA binding reaction was allowed to proceed for 25 min at room temperature .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8113170" target="_blank">8113170</a>]<br></li>
<li>When increasing amounts of CpxR were simultaneously added to-binding reactions containing a constant quantity of OmpR , a major DNA-protein complex of slow mobility was observed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15743952" target="_blank">15743952</a>]<br></li>
<li>To determine whether OmpR is part of this slow mobility complex , antibodies raised to OmpR were added to the binding reactions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15743952" target="_blank">15743952</a>]<br></li>
<li>OmpR is cytoplasmic DNA-binding protein which binds to a sites upstream of the promoters of both porin ( 17 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1370954" target="_blank">1370954</a>]<br></li>
<li>On the basis of these findings , we propose that the app Y gene product is a DNA-binding protein which binds to some promoters including that of appA and stimulates transcription in analogy with CRP or OmpR proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2537825" target="_blank">2537825</a>]<br></li>
<li>This extinction coef ® cient was estimated from the tryptophan and tyrosine content of the protein .54 Fluorescence anisotropy was used to determine the equilibrium binding of EnvZc to OmpR and the mutant derivative in solution .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11812125" target="_blank">11812125</a>]<br></li>
<li>We investigated the binding ability of another OmpR family protein , PhoP , and that of the bovine serum albumin to the ung promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15576781" target="_blank">15576781</a>]<br></li>
<li>Recently , the OmpR protein was purified , and its binding to the upstream regions was dem-onstrated biochemically ( Y.-L .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3536870" target="_blank">3536870</a>]<br></li>
<li>In vitro footprintin9 analysis of binding of OmpR protein to fadL .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8413182" target="_blank">8413182</a>]<br></li>
<li>Gel retardation analysis of binding of OmpR protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8413182" target="_blank">8413182</a>]<br></li>
<li>Presumably in vivo , this increased stability promotes OmpR binding at promoters that it does not normally regulate , resulting in a pleiotropic phenotype .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19432797" target="_blank">19432797</a>]<br></li>
<li>These results supported the idea that formation of complex a results from initial binding of OmpR to the higher affinity site and subsequent binding of OmpR to the - 71 to - 64-bp lower affinity site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7557323" target="_blank">7557323</a>]<br></li>
<li>First , these equations made it possible to determine the degree of cooperative binding between OmpR molecules using calculations that are independent of the concentration of the OmpR protein and its affinity for its binding site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9096305" target="_blank">9096305</a>]<br></li>
<li>This method of calculating the degree of cooperativity uses equations that are independent of the concentration of OmpR and its affinity for a binding site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9096305" target="_blank">9096305</a>]<br></li>
<li>Indeed , no OmpR complex could be detected in E. coli when the ﬁrst OmpR binding site S1 was deleted ( Fig. 1A , lane 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15743952" target="_blank">15743952</a>]<br></li>
<li>To further characterize the binding of OmpR to the downstream site , we created a 27-bp fragment encompassing the -84 to -58 - bp sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7557323" target="_blank">7557323</a>]<br></li>
<li>Under these conditions , the occupancy of one OmpR binding site stimulates the occupancy of the adjacent site by ' 1,000 fold .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9096305" target="_blank">9096305</a>]<br></li>
<li>It is posible that SecB could bind to putative OmpR intermediates , and binding of SecB to OmpR lead to subsequent degradation of OmpR by proteases .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8522138" target="_blank">8522138</a>]<br></li>
<li>OmpR and T83I binding to EnvZc .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11812125" target="_blank">11812125</a>]<br></li>
<li>It is unclear how many OmpR monomers are binding to each site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1648075" target="_blank">1648075</a>]<br></li>
<li>About 10-fold higher concentrations of OmpR were necessary for detection of binding to thepromoter fragments when the detergentNonidet P-40 was eliminated from the binding buffer ( data notshown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3023382" target="_blank">3023382</a>]<br></li>
<li>These data were used to calculate the degree of cooperativity of binding of OmpR using equations derived by Hudson and Fried ( 9 ) and by Tsai et al. ( 10 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9096305" target="_blank">9096305</a>]<br></li>
<li>Biochemical and mutagenesis data are consistent with a model in which the concentration of phosphorylated OmpR increases with increasing osmolarity ; this results in increased occupancy of the different OmpR-binding sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9016718" target="_blank">9016718</a>]<br></li>
<li>As the concentration of phosphorylated OmpR increases , however , a low-afﬁnity ompC binding site is occupied by OmpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16194231" target="_blank">16194231</a>]<br></li>
<li>As shown in Fig. 3 , the fragments A , B and D containing the putative OmpR binding region formed complexes only when incubated with 3.6 pmol of OmpR phosphorylated by acetylphosphate ( Fig. 3b , c ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10802181" target="_blank">10802181</a>]<br></li>
<li>As expected , the stimulatory effect of the kinase on OmpR binding is not observed with this mutant form of OmpR ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9096305" target="_blank">9096305</a>]<br></li>
<li>Our laboratory has previously shown that DNA-binding stimulates the rate of OmpR phosphoryl-ation by acetyl phosphate .28 Thus , we tested whether incubation of T83I with acetyl phosphate and the high af ® nity OmpR-binding sites from the ompF or ompC promoters ( F1 or C1 ) would stimulate phosphorylation of T83I .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11812125" target="_blank">11812125</a>]<br></li>
<li>T83I binds to the high affinity sites F1 and C1 Since our previous studies have demonstrated interactions between the amino and carboxylterminal domains of OmpR , it was of interest to determine whether the substitution that altered its phosphorylation properties would also affect its DNA binding ability .25,28 For measurements of DNA binding , we used ¯ uorescein-labeled oligo-nucleotides that correspond to the high af ® nity sites F1 and C1 upstream from the ompF and ompC promoters , respectively ( see Figure 11 , below , for a diagram of the binding sites ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11812125" target="_blank">11812125</a>]<br></li>
<li>Therefore , it was not possible to draw firm conclusions concerning the effects of phosphorylation on th intrinsic binding of OmpR to a single site or on the cooperative binding of OmpR to multiple sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9096305" target="_blank">9096305</a>]<br></li>
<li>Based on these results , we conclude that the binding of OmpR to the ompF regulatory region is highly cooperative and that the role of OmpR phosphorylation is to enhance the cooperative interactions between OmpR molecules bound at adjacent sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9096305" target="_blank">9096305</a>]<br></li>
<li>Our results indicate that phosphorylation has little effect on the intrinsic binding of OmpR to the single site F1 and that the primary effect of phosphorylation at ompF is to stimulate cooperative interactions between OmpR mol-ecules bound at adjacent sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9096305" target="_blank">9096305</a>]<br></li>
<li>Furthermore , although phosphorylation results in a slight increase in OmpR binding to a single site , the primary role of phosphorylation in ompF regulation is to facilitate cooperative interactions between OmpR molecules bound at adjacent sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9096305" target="_blank">9096305</a>]<br></li>
<li>Although the F1-F1 fragment does not carry the configuration of sites normally found at the ompF regulatory region , these experiments established that the unphosphorylated form of OmpR can manifest cooperative interactions and that these interactions can be further stimulated by phosphorylation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9096305" target="_blank">9096305</a>]<br></li>
<li>In this paper , we examine how phosphorylation influences the ability of OmpR to-bind to the OmpR-binding sites in the ompF regulatory region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9096305" target="_blank">9096305</a>]<br></li>
<li>The major conclusions from our studies are that binding of OmpR to the ompF regulatory region is cooperative and that phosphorylation stimulates these cooperative interactions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9096305" target="_blank">9096305</a>]<br></li>
<li>The second line of evidence , which highlights the importance of OmpR phosphorylation and cooperativity in the binding of OmpR to the ompF regulatory region , comes from our site-inactivation experiments .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9096305" target="_blank">9096305</a>]<br></li>
<li>Occupancy of the UhpA-binding sites in the uhpT promoter shows behavior similar to that of OmpR and BvgA , namely , different afﬁnities for strong and weak sites , apparently cooperative occupancy of the weak sites , and a marked increase in afﬁnity for all sites upon phosphorylation of UhpA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10913075" target="_blank">10913075</a>]<br></li>
<li>In our study , a nearly 10-fold difference in the binding of OmpR , due to phosphorylation , was observed in addition to the extension of protected sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7642497" target="_blank">7642497</a>]<br></li>
<li>We interpret these results to mean that binding of OmpR to the mutant F2 site is dependent on its ability to interact cooperatively with OmpR molecules bound at the flanking F1 and F3 sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9096305" target="_blank">9096305</a>]<br></li>
<li>Hence , the apparent increase in binding to this `` single site '' upon phosphorylation was most likely due to the stimulation of cooperative interactions between OmpR molecules bound at this incomplete site rather than due to an increased affinity of OmpR for the DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9096305" target="_blank">9096305</a>]<br></li>
<li>Lanes 2 through 8 of Figure 8 ( c ) demonstrate that wildtype OmpR binds to the three previously characterized binding regions at ompC , from ÿ105 to ÿ36 upstream of the transcriptional start site .47 In Figure 8 ( d ) , it is apparent that T83I binds the ompC promoter differently than does wild-type OmpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11812125" target="_blank">11812125</a>]<br></li>
<li>In the OmpR system , ompC low-afﬁnity binding sites are occupied at low concentrations of OmpR , leading to very low levels of transcriptional activation of ompC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16194231" target="_blank">16194231</a>]<br></li>
<li>In addition to the three OmpR binding sites located in the ompC-micF intergenic region and required for transcriptional activation of micF by OmpR , they identiﬁed a region between 2148 and 2131 bp relative to the start site for micF transcription that was responsible for OmpR-independent repression of micF expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8002608" target="_blank">8002608</a>]<br></li>
<li>Purified OmpR binds to the DNA fragment encompassing the - 102 to -58 - bp region forming two OmpR/ompF complexes that could be resolved by gel retardation [ lo ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7557323" target="_blank">7557323</a>]<br></li>
<li>It should be noted that binding of purified OmpR to the - 193 to + 121-bp ompF fragment also produced two distinct ompF/OmpR complexes [ lo ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7557323" target="_blank">7557323</a>]<br></li>
<li>Characterization of OmpR binding sequences in the upstream region of the ompF promoter essential for transcriptional activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1648075" target="_blank">1648075</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1943993" target="_blank">1943993</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10913075" target="_blank">10913075</a>]<br></li>
<li>Of the 77 mutant phages originally isolated , 25 were still dependent on OmpR for ompF promoter activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1648075" target="_blank">1648075</a>]<br></li>
<li>Furthermore , the results imply that the difference between activation and repression of ompF involves differential interaction between OmpR and the ompF promoter region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1648075" target="_blank">1648075</a>]<br></li>
<li>OmpR is a DNA-binding protein that stimulates transcription of both ompF and ompC ( Jo et al. 1986 ; Norioka et al. 1986 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8413182" target="_blank">8413182</a>]<br></li>
<li>OmpR is a DNA-binding protein that regulates ompF and ompC transcription ( 31 , 32 , 45 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2203749" target="_blank">2203749</a>]<br></li>
<li>Tsung , K. , Brissette , R.E. , and Inouye , M. ( 1989 ) Identification of the DNA-binding domain of the OmpR protein required for transcriptional activation of the ompF and ompC genes of Escherichia coil by in vivo DNA footprinting .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596446" target="_blank">8596446</a>]<br></li>
<li>Location of DNA-binding segment of a positive regulator , OmpR , involved in activation of the ompF and ompC genes of Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11812125" target="_blank">11812125</a>]<br></li>
<li>' DNA Binding of the Purified OmpR-The ability of the purified OmpR to-bind DNA was investigated using the promoter fragments of ompF and ompC in a DNA-binding assay .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3023382" target="_blank">3023382</a>]<br></li>
<li>DNA-binding assay of OmpR by using 32P-labeled ng ( lane 3 ) of OmpR ( molar ratio of OmpR and DNA , 66 : l ) ; promoter fragments of ompF and ompC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3023382" target="_blank">3023382</a>]<br></li>
<li>A , DNA-binding assay of OmpR to thelabeledompF Theresults described above clearly demonstratethatthe promoter fragment ; B , DNA-binding assay of OmpR to the labeled purified OmpR preparation specifically binds both the ompF ompC promoter fragment .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3023382" target="_blank">3023382</a>]<br></li>
<li>Paradoxically , enhanced binding of OmpR to the upstream sequences of ompF was found to be correlated with the repression of ompF ( 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2160945" target="_blank">2160945</a>]<br></li>
<li>OmpR activates transcription of ompF and ompC by binding to specific sequences upstream of the promoters in these operons ( 31 , 41 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1885551" target="_blank">1885551</a>]<br></li>
<li>Phosphorylation of OmpR was shown to enhance binding to the upstream OmpR-binding regions of both ompF and ompC ( 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2160945" target="_blank">2160945</a>]<br></li>
<li>The specific binding of OmpR tothe ompF and ompC promoterfragments was also determinedusingfragments labeled with 32Pat their3 ` ends .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3023382" target="_blank">3023382</a>]<br></li>
<li>Characterization types of OmpR-dependent interactions occur with oligonucle - of OmpR binding to theF boxes in both ompF and ompC are otides 631/632 ( i.e. the OmpR binding site ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2553720" target="_blank">2553720</a>]<br></li>
<li>OmpR binding sites C1 , C2 , and C3 at ompC and F1 , F2 , F3 , and F4 at ompF are also indicated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16077119" target="_blank">16077119</a>]<br></li>
<li>Interaction between phosphorylated OmpR and a series of binding sites that differ in their affinities for OmpR binding regulates the ompC and ompF promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11029695" target="_blank">11029695</a>]<br></li>
<li>This well orchestrated gene regulation of ompF and ompC by OmpR-P is achieved by the combination of the two-domain structure of OmpR , the cellular concentration of OmpR-P , and the highly sophisticated hierarchical arrangement of OmpR-P-binding sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16618701" target="_blank">16618701</a>]<br></li>
<li>How might the effect of phosphorylation on the cooperative DNA-binding properties of OmpR explain the observed regulation of ompF transcription in vivo ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9096305" target="_blank">9096305</a>]<br></li>
<li>How does the binding of OmpR to the - 71 to - 62-bp region function in the regulation of ompF ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7557323" target="_blank">7557323</a>]<br></li>
<li>Unlike the regulation of the mode of OmpR binding to ompF , OmpR does not seem to act as an activator in the ﬂagellar promoter , because the ﬂagellar regulon is expressed in ompRnull or pta mutants , in which phospho-OmpR does not exist .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7642497" target="_blank">7642497</a>]<br></li>
<li>On the basis of ompF promoter bending and the presence of a negative OmpR-binding site in position - 3 8 1 to -353 , Slauch and Silhavy ( 1991 ) speculated that the negative reguiation of the ompF promoter during growth in high osmoiarity involved a DNA loop structure generated by OmpR binding in position - 3 8 1 to - 3 5 3 and DNA bending in position - 1 0 1 to - 7 1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8057841" target="_blank">8057841</a>]<br></li>
<li>These results provide evidence that envZ function influences the DNA-binding activity of OmpR and suggest that high-affinity binding of OmpR to the upstream sequences of ompF is correlated with the repression of OmpF production .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2656631" target="_blank">2656631</a>]<br></li>
<li>As shown in Fig. 8C , the nine protected regions show minimal overlap with the known binding sites for integration host factor ( 21 ) , and OmpR ( 56 ) , in agreement with the absence of an effect of Lrp on the increase in ompC transcription caused by sucrose .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8002608" target="_blank">8002608</a>]<br></li>
<li>Transcriptional level CpxR-P not only modulates the action of different regulators , such as BaeR , Lrp and OmpR , by binding to the same regulatory sequences ( see above ) , but it also represses the gene transcription of other signaling systems , such as σE and Aer .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16487683" target="_blank">16487683</a>]<br></li>
<li>Our results are consistent with the assumption that Lrp and OmpR bind independently and that binding of Lrp to a promoter containing bound OmpR represses transcription of ompC even in the presence of OmpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8002608" target="_blank">8002608</a>]<br></li>
<li>These sites could probably not recruit three dimers of CpxR at the same time , but they could certainly disturb OmpR binding and , consequently , csgD transcription activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15743952" target="_blank">15743952</a>]<br></li>
<li>Furthermore in vitro , the phospho-OmpR was found to-bind to the OmpR binding region of bolA1p and repress the transcription by EcS or EcD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10802181" target="_blank">10802181</a>]<br></li>
<li>At present it is not clear whether binding of OmpR to all four sites is necessary for repression , or whether binding to each site affects transcription differently .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8413182" target="_blank">8413182</a>]<br></li>
<li>I n uiuo , OmpR binding to P3 promotem r ay stimulate transcription from P2 andP1 promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3023382" target="_blank">3023382</a>]<br></li>
<li>This suggests that CpxR binding upstream of ompC acts in conjunction with OmpR binding to activate transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16077119" target="_blank">16077119</a>]<br></li>
<li>In vitro binding experiments show that both the OmpR wild-type and OmpR234 mutant proteins can recognize this sequence , although OmpR234 displays a slightly higher binding afﬁnity than the wild-type protein , which might be enough to improve activation of csgD transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11717281" target="_blank">11717281</a>]<br></li>
<li>For transcription activation , the OmpR protein binds to the OmpR site generally located upstream from the promoter 335 , and interacts directly with a subunit of RNA polymerase [ 19,20 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10802181" target="_blank">10802181</a>]<br></li>
<li>From these experiments we conclude that the OmpR protein interacts directly with and activates transcription from the csgD promoter by binding to a 20-bp region centered at 49.5 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11717281" target="_blank">11717281</a>]<br></li>
<li>IHF can also inhibit transcription indirectly through modulation of a regulator protein , in the ompF promoter region , IHF binds to two sites that flank the binding region of the OmpR regulator that is situated between - 6 0 and -100 ( Fig. 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7651128" target="_blank">7651128</a>]<br></li>
<li>Asincubation of OmpR with EnvZ ( C ) and ATP was ing 25 ngof OmpR ) were added to a 15-pl DNA binding reaction con - extended to increase the level of phospho-OmpR , more of the taining labeledompF DNA ( about 5,000 cpm ) , 50 m ~ `` i s , 15 % glycerol , radiolabeled DNA appeared as the slower moving a-complex DISCUSSION PAR0106 -460 to 1150 ( H-81 pink red pARo101 -100 to 1150 ( A-8 ) red red PAR0107 -150 to t150 ( S-8 ) red red PAR0105 -460 t o tl5O IS-8 ) pink red polnt nutation In Cd box pAROlOB -460 CO 1150 ( H-8 ) pink red point nutatlon in Ff box pRS414 cloning vector whit .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8175665" target="_blank">8175665</a>]<br></li>
<li>OmpR - dependent and OmpR-independent responses of Escherichia coli to sublethal attack by the neutrophil bactericidal/permeability increasing protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15755952" target="_blank">15755952</a>]<br></li>
<li>Phosphorylation-dependent conformational changes in OmpR , osmoregulatory DNA-binding protein of Escherichia coli ABSTRACT Osmoregulated porin gene expression in Escherichia coli is controlled by the two-component regulatory system EnvZ and OmpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7568033" target="_blank">7568033</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_270">
C-terminal DNA binding stimulates N-terminal phosphorylation of the outer membrane protein regulator OmpR from Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11739766" target="_blank">11739766</a>]<br></label>
<input id="act_similarity_270", type="checkbox">
<div class="ACT_similar">
<li>C-Terminal DNA binding stimulates N-terminal phosphorylation of the outer membrane protein regulator OmpR from Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11812125" target="_blank">11812125</a>]<br></li></div>
				<label class="ACT_collapse_similarity" for="act_similarity_272">
Phosphorylation-dependent conformational changes in OmpR , an osmoregulatory DNA-binding protein of Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11812125" target="_blank">11812125</a>]<br></label>
<input id="act_similarity_272", type="checkbox">
<div class="ACT_similar">
<li>Phosphorylation-depen-dent conformational changes in OmpR , an osmoregulatory DNA-binding protein of Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15743952" target="_blank">15743952</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8892825" target="_blank">8892825</a>]<br></li></div>
<li>Kenney , L.J. , Bauer , M.D. , and Silhavy , T.J. ( 1995 ) Phosphorylation-dependent conformational changes in OmpR , an osmoregulatory DNA-binding protein of Escher-ichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16194231" target="_blank">16194231</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_275">
[ 13 ] Kenney , L. , Bauer , M.D. and Silhavy , T.J. ( 1995 ) Phosphorylationdependent conformational changes in OmpR , an osmoregulatory DNA-binding protein of Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10802181" target="_blank">10802181</a>]<br></label>
<input id="act_similarity_275", type="checkbox">
<div class="ACT_similar">
<li>Kenney , L.J. , Bauer , M.D. , and Silhavy , T.J. ( 1995 ) Phospho-rylation-dependent conformational changes in OmpR , an osmoregulatory DNA-binding protein of Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16135237" target="_blank">16135237</a>]<br></li></div>
<li>OmpR , the regulatory protein of the EnvZ-OmpR system , was shown to-bind to DNA in vitro in its non-phosphorylated form , albeit with a 10-fold lower efficiency than the phosphorylated form ; the same DNA fragment was recognized by both forms ( Aiba ef a / .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8412671" target="_blank">8412671</a>]<br></li>
<li>Howments , 5 pg of EnvZ ( C ) was phosphorylated for 15 min at room tem - ever , brief incubation of OmpR with EnvZ ( C ) and ATP produced perature in a reaction consisting of 5 m ATP , 25 mM HEPES , 1m ~ two protein-DNA complexes shown in Fig. 1 as `` a '' - and `` b '' - MgCl , , 1m CaCl , , and 50mM KCl .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8175665" target="_blank">8175665</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_279">
D31 Rampersaud , A. and Inouye , M. ( 1991 ) Procaine , a local anesthetic , signals through the EnvZ receptor to change the DNA binding affinity of the transcriptional-activator protein OmpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7557323" target="_blank">7557323</a>]<br></label>
<input id="act_similarity_279", type="checkbox">
<div class="ACT_similar">
<li>Rampersaud , A. , and Inouye , M. ( 1991 ) Procaine , a local anesthetic , signals through the EnvZ receptor to change the DNA binding afﬁnity of the transcriptional-activator protein OmpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16359331" target="_blank">16359331</a>]<br></li></div>
				<label class="ACT_collapse_similarity" for="act_similarity_281">
Procaine , a local anesthetic , signals through EnvZ to change the DNA binding af ® nity of the transcriptional-activator protein , OmpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11601842" target="_blank">11601842</a>]<br></label>
<input id="act_similarity_281", type="checkbox">
<div class="ACT_similar">
<li>Procaine , a local anesthetic , signals through the EnvZ receptor to change the DNA binding affinity of the transcriptional-activator protein OmpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8098993" target="_blank">8098993</a>]<br></li></div>
<li>Phosphorylation of a bacterial activator protein , OmpR , by a protein kinase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8098993" target="_blank">8098993</a>]<br></li>
<li>This work was supported , in part , by grants from the Universite d'Aix-Marseille II and MRT fellowships to C.J and G.S. References Aiba , H. , Nakasai , F. , Mizushima , S. , and Mizuno , T. ( 1989 ) Phosphorylation of a bacterial activator protein , OmpR , by a protein kinase , EnvZ , results in stimulation of its DNA-binding ability .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596446" target="_blank">8596446</a>]<br></li>
<li>This work was supported , in part , by a grant from the Association pour la Recherche sur le Cancer ( No. 6791 ) , References Aiba , H. , Nakasai , F. , Mizushima , S. , and Mizuno , T. ( 1989 ) Phosphorylation of a bacterial activator protein , OmpR , by a protein kinase , EnvZ , results in stimulation of its DNA-binding ability .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8412671" target="_blank">8412671</a>]<br></li>
<li>J Biol Chem 264:14090 -14094 Aiba H , Nakasai F , Mizushima S , Mizuno T ( 1989b ) Phosphorylation of a bacterial activator protein , OmpR , by a protein kinase , EnvZ , results in stimulation of its DNA-binding ability .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8413182" target="_blank">8413182</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_287">
Phosphorylaton of a bacterial activator protein , OmpR , by a protein kinase , EnvZ , results in stimulation of its DNA-binding ability .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20466769" target="_blank">20466769</a>]<br></label>
<input id="act_similarity_287", type="checkbox">
<div class="ACT_similar">
<li>Phosphorylation of a bacterial activator protein , OmpR , by a protein kinase , EnvZ , results in stimulation of its DNA-binding ability .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8113170" target="_blank">8113170</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2160945" target="_blank">2160945</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7642497" target="_blank">7642497</a>]<br></li>
<li>Phosphorylation of a bacterial activator protein , OmpR , by a protein kinase , EnvZ , results in a stimulation of its DNA-binding ability .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1648075" target="_blank">1648075</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1532388" target="_blank">1532388</a>]<br></li></div>
<li>Lanes 2-4 represent DNA binding activities resulting from 10-min were mutated in thFef box ( -360 to -3691 , changing the sequencefrom incubations of EnvZ ( C ) with ATP in the absence of OmpR ( lane 21 , ATTACATTGC to ATTGTCGTGC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8175665" target="_blank">8175665</a>]<br></li>
<li>Thus , different levels of EnvZ in IHF mutants could lead to changes in the binding properties of OmpR in MH760 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2203749" target="_blank">2203749</a>]<br></li>
<li>Again , OmpR had to be pre-incubated with EnvZ and ATP for the binding to occur , implying that only the phosphorylated form of OmpR bound the DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8413182" target="_blank">8413182</a>]<br></li>
<li>Thus , EnvZ appears to promote optimal DNA-binding activity of OmpR to fragment Ila ( Fig. 4 and 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2656631" target="_blank">2656631</a>]<br></li>
<li>EnvZ phosphorylates OmpR , a cytoplasmic DNA-binding protein , on an aspartyl residue .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7568033" target="_blank">7568033</a>]<br></li>
<li>EnvZ enhances the DNA-binding properties of OmpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1648075" target="_blank">1648075</a>]<br></li>
<li>Exactly how the binding of MzrA to EnvZ modulates EnvZ enzymatic activity remains to be determined , but we demonstrated that it does so by increasing OmpR ~ P stability .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19432797" target="_blank">19432797</a>]<br></li>
<li>This is consistent with the hypothesis that EnvZ enhances the binding capabilities of OmpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1648075" target="_blank">1648075</a>]<br></li>
<li>For example , the two-component systems that respond to anaerobiosis ( ArcB/ArcA ) or osmolarity ( EnvZ/OmpR ) constitute signal transduction pathways in which both the membrane sensor protein and the DNA-binding effector are post-translationally activated via sequential phosphorylations ( for a review see Wurgler-Murphy and Saito , 1997 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9564045" target="_blank">9564045</a>]<br></li>
<li>Previous studies suggest that EnvZ functions as a membrane sensor relaying information to the DNA-binding protein , OmpR , which in turn activates expression of the appropriate promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2557454" target="_blank">2557454</a>]<br></li>
<li>EnvZ is the sensor kinase that phosphorylates OmpR , a DNA-binding regulatory protein , under conditions of high osmolarity [ 67 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17941710" target="_blank">17941710</a>]<br></li>
<li>Increasing osmolarity of the growth environment activates the sensor protein EnvZ and leads to increased phosphorylation of OmpR , resulting in enhancement of its DNA-binding ability ( 1 , 13 , 17 , 24 , 46 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11717281" target="_blank">11717281</a>]<br></li>
<li>Expression of 13-galactosidase in a coupled transcription-translation assay 1 23 4 pmol of ONPG hydrolyzed/min 814 1,963 2,431 80,417 104,170 76,661 98 o ( mppRRT50p6l.a2s ) m1id ¬ ¬ + ¬ ¬ + ¬ Promotera protein OmpR mcbAp mcbAp ¬ + + - + + ¬ mcbAp lacUVSp lacUV5p lacUVSp No DNA * :0 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1906064" target="_blank">1906064</a>]<br></li>
<li>As opposed to results observed with OmpR , several protein-DNA complexes of decreasing mobility could be resolved with an increasing concentration of CpxR ( Fig. 2A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15743952" target="_blank">15743952</a>]<br></li>
<li>concentrations of purified ( nonphosphorylated ) OmpR hardly DNA Binding and Footprinting-For DNA mobility shiftexperi - formed protein-DNA complexes ( Fig. 1 , lanes 2 and 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8175665" target="_blank">8175665</a>]<br></li>
<li>If OmpR is bound to a particular fragment , the DNA.protein complex migrates more slowly than unbound fragment , resulting in the disappearance of that specific band .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3023382" target="_blank">3023382</a>]<br></li>
<li>The level of DNA binding of the extracts derived from MC4100 grown in nutrient broth was very low ; 30 , ug of total protein ( -30 ng of OmpR ) was required to detect bound forms a and b of fragment Ila .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2656631" target="_blank">2656631</a>]<br></li>
<li>marked contrast , the DNA binding produced by extracts derived from MC4100 grown in the presence of sucrose was much higher ; only 3 , ug of total protein ( -7 ng of OmpR ) was required to produce readily detectable bound form a of fragment Ila .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2656631" target="_blank">2656631</a>]<br></li>
<li>The amounts of transcription factors added were : CpxR and IHF , 0 , 0.5 , 1.0 , 1.5 , 2.0 , 2.5 , 3.0 , 3.5 , 4.0 , 4.5 , 5.0 , 5.5 and 6.0 pmol for lanes 1 -- 13 ; H-NS , 0 , 2.5 , 5.0 , 7.5 , 10 , 12.5 , 15 , 17.5 , 20 , 22.5 , 25 , 27.5 and 30 pmol for lanes 1 -- 13 ; OmpR and RstA , 0 , 5 , 10 , 15 , 20 , 25 , 30 , 35 , 40 , 45 , 50 , 55 and 60 pmol for lanes 1 -- 13 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20466769" target="_blank">20466769</a>]<br></li>
<li>The amounts of proteins used were ( from lane 2 to 5 ) : 1.25 , 2.5 , 5 and 10 pmol for IHF ; 2.5 , 5 , 10 and 20 pmol for CpxR ; 5 , 10 , 20 and 40 pmol for OmpR and H-NS in the absence ( H-NS and IHF ) or presence ( CpxR , OmpR and RstA ) of acetyl phosphate ( 10 mM ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20466769" target="_blank">20466769</a>]<br></li>
<li>Briefly , E. coli whole-cell lysates were prepared by sonication after lysozyme treatment , and treated with antibodies , which were prepared in rabbits against purified transcription factors , CsgD , RstA , OmpR , CpxR , IHF and H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20466769" target="_blank">20466769</a>]<br></li>
<li>At the times indicated , whole-cell lysates were prepared and subjected to quantitative Western blot analysis using antibodies against IHF , OmpR , H-NS , CpxR and RstA ( upper panels ) , and anti-RpoA ( bottom panel ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20466769" target="_blank">20466769</a>]<br></li>
<li>In growing cells , the intracellular concentrations of two nucleoid proteins , IHF and H-NS , are similar and higher than those of the specific regulators , RstA , OmpR and CpxR ( Fig. 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20466769" target="_blank">20466769</a>]<br></li>
<li>Based on the results described herein , we propose a novel regulation model of cooperation between two positive factors ( OmpR and IHF ; and RstA and IHF ) and also between two negative factors ( CpxR and H-NS ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20466769" target="_blank">20466769</a>]<br></li>
<li>In addition to the competition between positive and negative factors , we found unexpected cooperation between two negative factors , CpxR and H-NS , and between two pairs of positive factors , OmpR and IHF , and RstA and IHF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20466769" target="_blank">20466769</a>]<br></li>
<li>Previously , we classiﬁed three factors ( OmpR , RstA and IHF ) as activators and two factors ( CpxR and H-NS ) as repressors , and found novel modes of their interplay .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20874755" target="_blank">20874755</a>]<br></li>
<li>In the simultaneous presence of MlrA , OmpR and IHF , all three regulators bind to the same CD6 probe forming MlrA -- OmpR -- IHF -- DNA quaternary complexes ( Fig. 5c ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20874755" target="_blank">20874755</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20874755" target="_blank">20874755</a>]<br></li>
<li>Fluorescently labelled DNA probe of the csgD promoter CD1 fragment ( 1 pmol ) was incubated with increasing concentrations of purified IHF ( lanes 1 and 2 , 0 pmol ; lane 3 , 2.5 pmol , lane 4 , 5.0 pmol ; lanes 5 and 6 , 10 pmol ) in the absence ( lane 1 , 6 ) or presence ( lanes 2 -- 5 ) of OmpR ( 80 pmol ) and subjected to DNase I footprinting assays as described in Methods .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20466769" target="_blank">20466769</a>]<br></li>
<li>Identical concentrations of OmpR failed to retard a DNA fragment encompassing the csgB promoter ( 173 to 88 ; data not shown ) , strongly suggesting that the csgD promoter is the main target for OmpR regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11717281" target="_blank">11717281</a>]<br></li>
<li>Addition of a fourfold excess of unlabeled csgD promoter was required to increase the total amount of DNA-OmpR complexes and to allow for their immunodetection ( Fig. 4C , compare lanes 2 and 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15743952" target="_blank">15743952</a>]<br></li>
<li>Curli production at low osmolarity results from transcriptional activation of the csgD promoter by OmpR and OmpR234 , OmpR234 appearing to be a much more efﬁcient activator than OmpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11717281" target="_blank">11717281</a>]<br></li>
<li>Increased production of the CsgD transcription activator results in activation of the csgBA operon ; when CsgD is expressed independently of OmpR , the latter becomes dispensable for activation of the csgB promoter ( Table 2 ) Both genetic and biochemical evidence shows that OmpR binds to an imperfect direct repeat ( 5 - GTTACATTTA/GTT ACATGTT-3 ) centered at position 49.5 relative to the transcriptional start site of the csgD promoter ( Table 2 and Fig. 1 and 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11717281" target="_blank">11717281</a>]<br></li>
<li>OmpR was shown to-bind to a consensus sequence located 49.5 bp upstream of the transcriptional start site of csgD , and this subsequently activates the csgDEFG expression that , in turn , increases the expression of csgBA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15743952" target="_blank">15743952</a>]<br></li>
<li>Overexpression of CsgD resulted in stimulation of csgBA expression , suggesting that OmpR is required for activation of the csgD but not the csgBA promoter ( Table 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11717281" target="_blank">11717281</a>]<br></li>
<li>Curiously , constitutive activation of the OmpR response-regulator has a repressive effect on csgD expression , which has been ascribed to the binding of phosphorylated OmpR to multiple low-affinity binding sites in the csgD promoter ( Gerstel et al. , 2003 ; 2006 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22250746" target="_blank">22250746</a>]<br></li>
<li>CpxR , CsgD , CRP , RstA , OmpR and Cra-binding sites are indicated csgB coding sequence TTGTTTTTCATGTTGTCACCCTGGACCTGGTCGTACATTTAAGAAATTAAATCATTTCAACTTGGTTGT TAACGCAACCTGTATTTTGTTAACGCTGCGTTACGATGGAAAGTATGTCTGCGGAAATATTTTTAATAA csgBp CTCACCCGCGCGTTGTATTTTTCTTTTTTAGTTCATACCAAAGTATTAAAACCCACGTAAATACGCTGT TATCTACGCAAAAAAATATTTTGTTTTCTTTTAAATCTCCGTTTTCCGCTATCAAAAAGCACCAGACAG TCATTCTTCTTGCCCGTCGCTGATTGCTGCGATATGTCTTGCGCACAAGCCGTGACAAACCTCGTCTTT TTGCAGTAAAAAATTGTCCACGGAGGTGTGGAGAAAAAACAAGAACGTTTTACATGACGAAAGGACTAC ACCGAAATATTTTTTATATGCATTATTAGTAAGTTATCACCATTTGTATGATTTTTTAAAATTGTGCAA TAAAAACCAAATGTACAACTTTTCTATCATTTCTAAACTTAATAAAACCTTAAGGTTAACATTTTAATA csgDp1 TAACGAGTTACATTTAGTTACATGTTTAACACTTGATTTAAGATTTGTAATGGCTAGATTGAAATCAGA csgDp2 TGTAATCCATTAGTTTTATATTTTACCCATTTAGGGCTGATTTATTACTACACACAGCAGTGCAACATC TGTCAGTACTTCTGGTGCTTCTATTTTAGAGGCAGCTGTCAGGTGTGCGATCAATAAAAAAAGCGGGGT csgD coding sequence TTCATCATGTTTAATGAAGTCCATAGTAT RstA-P CpxR-P CpxR-P CpxR-P csgDp1 OmpR csgDp2 Cra Cra CRP-AMP CRP-AMP CpxR-P CpxR-P Cra CpxR-P Cra CsgD csgBp J. Altenbuchner and S. E. Lindow , respectively , while bacteriophage P1vir was a kind gift from S. K. Apte .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21559929" target="_blank">21559929</a>]<br></li>
<li>All five transcription factors , CpxR , OmpR , RstA , IHF and H-NS , formed binary DNA complexes in vitro in the absence of other factors ( Fig. 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20466769" target="_blank">20466769</a>]<br></li>
<li>Interplay between MlrA , OmpR and IHF Three positive factors , IHF , OmpR and RstA , can associate simultaneously within the promoter-proximal hot-spot II region of transcription factor binding ( Fig. 6 ) , and cooperate with each other for activation of the csgD promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20874755" target="_blank">20874755</a>]<br></li>
<li>In this report , we describe the regulatory modes of five transcription factors , CpxR , OmpR , RstA , IHF and H-NS , and their interplays in binding in vitro to the csgD promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20466769" target="_blank">20466769</a>]<br></li>
<li>DNA curvature induced by MlrA should inﬂuence not only the activity of neighbouring DNA such as binding afﬁnity to positive factors , IHF , OmpR and RstA , and/or to negative fractors , H-NS and CpxR , but also the mode of the molecular interplay between csgD promoter-bound transcription factors .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20874755" target="_blank">20874755</a>]<br></li>
<li>CRP , CpxR , CsgD , H-NS , IHF , MlrA , OmpR , and RstA on the csgD promoter ( Ogasawara et al. 2010a , b ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23233451" target="_blank">23233451</a>]<br></li>
<li>Five factors ( OmpR , RstA , IHF , CpxR and H-NS ) bind to the promoter-proximal hot-spot II , while four factors ( IHF , MlrA , CpxR and H-NS ) bind to the promoter-distal hot-spot I. P1 and P2 represent the transcription initiation site P1 ( Hammar et al. , 1995 ) and P2 promoter ( Ogasawara et al. , 2007a ) , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20874755" target="_blank">20874755</a>]<br></li>
<li>The apparent affinity for the csgD promoter was found to be high for IHF and CpxR , followed by OmpR , H-NS and RstA ( Fig. 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20466769" target="_blank">20466769</a>]<br></li>
<li>The binding sites of MlrA , OmpR , RstA , CpxR , IHF and H-NS were determined by DNase-1 footprinting assays ( Fig. 2 ; and Ogasawara et al. , 2010 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20874755" target="_blank">20874755</a>]<br></li>
<li>Interplay between three activators , MlrA , OmpR and IHF , was also analysed with respect to control of this complex promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20874755" target="_blank">20874755</a>]<br></li>
<li>One explanation for the lack of csgD promoter activation in the mutants lacking OmpR and IHF is that MlrA requires these two activators for function .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20874755" target="_blank">20874755</a>]<br></li>
<li>When bacteria encounter environmental conditions that fully activate the Cpx pathway , CpxR-P should bind at sites 1 and 2 and allow an extremely efﬁcient csgD repression even if OmpR is still bound to the csgD promoter ( as is the case in our in vitro experiments in Fig. 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15743952" target="_blank">15743952</a>]<br></li>
<li>In contrast , the straight regulatory path CpxR-CsgD-FliA may not correspond to a predominant route in exponential-growth-phase , probably because the inhibitory effect of CpxR on csgD is impaired by the presence of OmpR that shares the same binding region as CpxR on the csgD promoter ( 26 , 49 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24272779" target="_blank">24272779</a>]<br></li>
<li>In the case of csgD , the OmpR activator binds to a unique site at the csgD promoter , whereas CpxR-P binds cooperatively to multiple sites , forming a DNA -- protein complex of very slow mobility .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16487683" target="_blank">16487683</a>]<br></li>
<li>RESULTS OmpR binds to a unique site at the csgD promoter , whereas CpxR binds cooperatively to multiple sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15743952" target="_blank">15743952</a>]<br></li>
<li>Since OmpR and CpxR share a common binding site ( Fig. 3 ) and CpxR seems to-bind preferentially in a cooperative fashion , we directly examined the possibility of a competitive binding of the two proteins at the csgD promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15743952" target="_blank">15743952</a>]<br></li>
<li>( d ) Model of cooperation between OmpR and IHF in binding to the csgD promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20466769" target="_blank">20466769</a>]<br></li>
<li>Since OmpR and RstA must compete with each other in binding to the csgD promoter , the repression of csgD by RstA must be due to competitive inhibition of OmpR binding to the csgD promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17468243" target="_blank">17468243</a>]<br></li>
<li>In principle , YdaM-generated c-di-GMP may modulate the action of any of these regulators ( i.e. OmpR , IHF , MlrA , CpxR , H-NS ) at the csgD promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17010156" target="_blank">17010156</a>]<br></li>
<li>These results indicate that , in addition to CpxR , the csgD promoter also binds OmpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15743952" target="_blank">15743952</a>]<br></li>
<li>These in vitro experiments dem-onstrate that OmpR and CpxR bind simultaneously at the csgD promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15743952" target="_blank">15743952</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_346">
OmpR and CpxR bind simultaneously at the csgD promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15743952" target="_blank">15743952</a>]<br></label>
<input id="act_similarity_346", type="checkbox">
<div class="ACT_similar">
<li>CpxR and OmpR bind simultaneously at the promoter region of csgD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15743952" target="_blank">15743952</a>]<br></li></div>
<li>It encodes an HPt-speci cents c phospho-histidine phosphatase that inhibits the transfer of phosphate groups between the histidine kinase ArcB and its non-cognate response-regulator OmpR , and it modulates the response of the ArcBA signalling pathway under certain anaerobic respiratory growth conditions [ 23 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12900013" target="_blank">12900013</a>]<br></li>
<li>These include , for example , the osmotic response-regulator OmpR , [ 384 ] the phosphate response-regulator PhoB [ 394 ± 396 ] and the nitrogen response-regulator NRI , the ¯ agellar regulator [ 388 , 389 ] and the chemotaxis response-regulator , CheY [ 390 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10396600" target="_blank">10396600</a>]<br></li>
<li>An analysis of the role played by the RcsC hybrid sensor kinase domains is presented here , which unveils the existence of a signalling pathway independent of and alternative to the RcsD and RcsB phosphorelay proteins , involving the OmpR response-regulator , which acts by repressing the pgaABCD operon and can be phosphorylated by acetyl phosphate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25667010" target="_blank">25667010</a>]<br></li>
<li>Analysis of pgaA -- cat expression in ackA , ompR and ackA -- ompR backgrounds To further study the role played by the phospho-donor acetyl phosphate and the response-regulator OmpR in the RcsC-pga signalling pathway , plasmids pKK232pgaA1 and pKK232pgaA2N were transformed into the ackA and ompR mutants , carrying pMPMABD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25667010" target="_blank">25667010</a>]<br></li>
<li>This signalling pathway involves the secondary Received 1 December 2014 Accepted 29 January 2015 metabolite acetyl phosphate and the response-regulator OmpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25667010" target="_blank">25667010</a>]<br></li>
<li>Thus , a current working model would involve the metabolite acetyl phosphate acting as a phospho-donor and phosphorylating the response-regulator OmpR , aside from another unknown response-regulator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25667010" target="_blank">25667010</a>]<br></li>
<li>AJW678 pMPMABD AJW678 pMPM-T6W OmpR is one of the response-regulators that could be modified by acetyl phosphate to affect biofilm formation AJW678 To determine whether a response-regulator affected by acetyl phosphate levels could be involved in the regulatory pathway studied here , deletion mutants in some of the corresponding genes ( i.e. cpxR , ompR , phoR , uvrY ) from the Keio collection were analysed for biofilm formation ( Fredericks et al. , 2006 ; Matsubara & Mizuno , 1999 ; McCleary & Stock , 1994 ; Tomenius et al. , 2005 ; Wolfe et al. , 2008 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25667010" target="_blank">25667010</a>]<br></li>
<li>Even though some response-regulator proteins , such as CheY , PhoB and OmpR , are known to be phosphorylated in vitro with acetyl phosphate ( Lukat et al. , 1992 ; McCleary & Stock , 1994 ) , phosphorylation in vitro of YedW by acetyl phosphate has not yet been proven .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25568260" target="_blank">25568260</a>]<br></li>
<li>However , OmpR still needs to be activated , probably by receiving a phosphoryl group from acetyl phosphate ( Bang et al. , 2000 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17185552" target="_blank">17185552</a>]<br></li>
<li>It seems that OmpR ordinarily can acquire some of its phosphates from other sources ( Forst et al. , 1990 ) , perhaps small molecules like acetyl phosphate or even other kinases .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8098993" target="_blank">8098993</a>]<br></li>
<li>At a high medium osmolality , acetylphosphate , an intermediate in the interconversion of acetate and acetyl-CoA by acetate kinase and phosphotransacetylase , has a negative inﬂuence on the expression of the chemotaxis and ﬂagellum genes , as under these conditions , phosphorylated OmpR inhibits the expression of ﬂhDC ( 60 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12620868" target="_blank">12620868</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_359">
motility and flhDC transcription are inhibited under conditions of high acetyl phosphate levels and high osmolarity by the phosphorylated form of OmpR ( PruÈû and Wolfe , 1994 ; Shin and Park , 1995 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11298291" target="_blank">11298291</a>]<br></label>
<input id="act_similarity_359", type="checkbox">
<div class="ACT_similar">
<li>In contrast , motility and ﬂhDC expression are inhibited under conditions of high acetyl phosphate levels and high osmolarity by the phosphorylated form of OmpR ( Prüß and Wolfe , 1994 ; Shin and Park , 1995 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12123461" target="_blank">12123461</a>]<br></li></div>
<li>OmpR , phosphorylated by acetyl phosphate , inhibits # hDC expression [ 12 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11287152" target="_blank">11287152</a>]<br></li>
<li>Therefore , we propose that the phosphorylated OmpR , generated by either osmolarity change or the internal level of acetyl phosphate , negatively regulates the expression of ﬂagella .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7642497" target="_blank">7642497</a>]<br></li>
<li>In vivo , about 100 genes respond to the status of the acetyl P pool , and many of these acetyl P-responsive genes are known to be regulated by a subset of RRs , including OmpR , NRI , and RcsB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15755952" target="_blank">15755952</a>]<br></li>
<li>Thus , future in vitro kinetic studies should be performed in the presence of a crowding agent on RRs , e.g. , OmpR or NRI , that readily respond to changes in the acetyl P pool .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15755952" target="_blank">15755952</a>]<br></li>
<li>This indicates that acetyl-P is able to phosphorylate OmpR , despite the presence of phosphatase and kinase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7642497" target="_blank">7642497</a>]<br></li>
<li>We believe that the difference in responsiveness to acetyl-P between the two-component system with EnvZ and OmpR and other two-component systems lies in the presence of excess OmpR relative to EnvZ .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7642497" target="_blank">7642497</a>]<br></li>
<li>However , a recent experiment ( 28 ) demonstrated that acetyl-P is able to phosphorylate OmpR in the presence of EnvZ .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7642497" target="_blank">7642497</a>]<br></li>
<li>Subsequent studies demonstrated that OmpR could directly repress ﬂhDC transcription in an acetyl P-dependent manner , despite the presence of its cognate EnvZ ( 355 , 408 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15755952" target="_blank">15755952</a>]<br></li>
<li>The ability of mutant response-regulators to discriminate between phosphodonors has previously been observed in another OmpR mutant , V203 M. 25 In addition , a CheYT87I mutant is not phosphorylated by acetyl phosphate ,45 yet it is phosphorylated by its kinase , CheA .24 A FixJ mutant , T82I , is phosphorylated by the FixL kinase , but with much slower kinetics .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11812125" target="_blank">11812125</a>]<br></li>
<li>When OmpR is incubated with acetyl phosphate , a faster peak appears , shown in Figure 3 ( b ) , that corresponds to OmpR-P .10 When the T83I mutant is incubated with acetyl phosphate and the products analyzed on C4 reversed-phase HPLC , no new peak appears , and only the unphosphorylated protein peak is present ( Figure 3 ( d ) ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11812125" target="_blank">11812125</a>]<br></li>
<li>Indeed , acetyl P and ArcB provide OmpR with the phosphate necessary to induce the protective response to the neutrophil bactericidal/permeability-increasing ( BPI ) protein ( 354 ) ( P. Prohinar and J. Weiss , personal communication ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15755952" target="_blank">15755952</a>]<br></li>
<li>Fortunately , such an acetyl P-insensitive OmpR mutant exists ( 448 ) , and it fails to mediate the OmpR-dependent response to the bactericidal/permeability-increasing protein ( 354 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15755952" target="_blank">15755952</a>]<br></li>
<li>OmpR protein phosphorylated with acetyl-P ( C and D ) was added to the reaction mixture as follows : lane 1 , none ; lane 2 , 25 ng ; lane 3 , 50 ng ; lane 4 , 100 ng ; lane 5 , 200 ng ; lane 6 , 300 ng ; lane 7 , 350 ng ; lane 8 , 400 ng .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7642497" target="_blank">7642497</a>]<br></li>
<li>Since phosphorylation by acetyl phosphate is relatively slow , we were interested in determining whether another small molecule phosphodonor , phosphoramidate , could phosphorylate T83I .27 After a two hour incubation with phosphoramidate , 100 % of the OmpR protein is phosphorylated ( Figure 3 ( f ) ) , indicating that phosphoramidate is more ef ® cient than acetyl phosphate as a phosphodonor ( compare Figure 3 ( b ) and ( f ) ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11812125" target="_blank">11812125</a>]<br></li>
<li>After incubation of OmpR or T83I with acetyl phosphate for three hours at room temperature , the samples were separated on C4 and two peaks corresponding to OmpR and OmpR-P are evident in ( b ) .10 ( d ) In contrast , there is no additional peak in the presence of acetyl phosphate with T83I .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11812125" target="_blank">11812125</a>]<br></li>
<li>Phosphorylation of OmpR and T83I by small molecule phosphate donors ; 10 mM OmpR or T83I in a total volume of 200 ml was injected onto a C4 HPLC column .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11812125" target="_blank">11812125</a>]<br></li>
<li>When T83I is added to the reaction in place of OmpR , it is also phosphorylated ( lanes 4-5 ) , even though it can not be phosphorylated by small molecule phosphate donors ( Figure 3 ( d ) and ( h ) ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11812125" target="_blank">11812125</a>]<br></li>
<li>Lanes 2-8 show the result obtained when 2.9 , 2.4 , 2.0 , 1.6 , 1.2 , 0.8 , and 0.4 mM total T83I ( 26 % of which is phosphorylated ) is included in the reaction Phosphorylation properties are altered in T83I In contrast to wild-type OmpR , T83I is not phosphorylated by the small phosphate donors acetyl phosphate and phosphoramidate ( Figure 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11812125" target="_blank">11812125</a>]<br></li>
<li>However , the phosphate group of OmpR appears to be considerably more stable than that of NR , , CheY , or CheB ( 5 , 18 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2160945" target="_blank">2160945</a>]<br></li>
<li>This phosphate group is transferred subsequently to the conserved Asp-55 residue of OmpR forming phosphorylated OmpR ( OmpR-P ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11158569" target="_blank">11158569</a>]<br></li>
<li>This results in the net breakdown of ATP and the subsequent release of inorganic phosphate from OmpR ~ P .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19432797" target="_blank">19432797</a>]<br></li>
<li>Because of the instability of the acyl-phosphate bonds of the OmpR class of molecules , successful attempts to isolate phosphorylated effector-molecules from cells labeled with 32Pi have not been reported .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2160945" target="_blank">2160945</a>]<br></li>
<li>In the presence of acetyl phosphate , OmpR is phosphorylated at only one site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7568033" target="_blank">7568033</a>]<br></li>
<li>Various amounts ( 0-20 , ug ) of purified OmpR were phosphorylated by acetyl [ 32P ] phosphate for 30 min , the peak of the time course in Fig. 1A .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7568033" target="_blank">7568033</a>]<br></li>
<li>LrhA was involved in regulating acetyl phosphate levels , we would expect a decrease in ompF9-lacZ levels due to indirect effects upon OmpR activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9882671" target="_blank">9882671</a>]<br></li>
<li>On DISCUSSION Acetyl phosphate can phosphorylate OmpR directly , yielding high levels of OmpR-P .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7568033" target="_blank">7568033</a>]<br></li>
<li>Importantly , the source for OmpR phosphorylation under this growth condition seems to be acetyl phosphate : accumulation of acetyl phosphate in the ackA mutant has a negative effect on biofilm production , while the converse is true in the double pta -- ackA mutant where no acetyl phosphate is produced ( Fig. 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25667010" target="_blank">25667010</a>]<br></li>
<li>It is possible , however , that OmpR * is aberrant and that it was detected simply because the level of OmpR-P produced by acetyl phosphate is very high .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7568033" target="_blank">7568033</a>]<br></li>
<li>To circumvent this problem , we tried acetyl phosphate as a phosphorylating agent for OmpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7568033" target="_blank">7568033</a>]<br></li>
<li>Ten micrograms of OmpR was incubated with acetyl [ 32P ] phosphate for the times indicated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7568033" target="_blank">7568033</a>]<br></li>
<li>We have used acetyl phosphate to phosphorylate OmpR and have examined the effects of limited proteolysis by trypsin to detect conformational changes in OmpR in response to phosphorylation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7568033" target="_blank">7568033</a>]<br></li>
<li>Next , to examine the effect of OmpR phosphorylation on the binding pattern , we included acetyl phosphate in the DNase I reaction mixtures .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15175316" target="_blank">15175316</a>]<br></li>
<li>OmpR was phosphorylated as before and applied to a NAP-5 column ( Pharmacia ) equilibrated with phosphorylation buffer lacking acetyl phosphate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7568033" target="_blank">7568033</a>]<br></li>
<li>A partial explanation of this effect may be the direct phosphorylation of OmpR by acetyl phosphate ( 269 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14665678" target="_blank">14665678</a>]<br></li>
<li>The low OmpF expression seen in an envZ null background is thought to be due to the phosphorylation of OmpR by acetyl phosphate ( Hsing and Silhavy , 1997 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19432797" target="_blank">19432797</a>]<br></li>
<li>To test this possibility , we took advantage of the fact that in the envZ strain EPB91 , phosphorylation of OmpR by acetyl phosphate leads to an OmpF / OmpC phenotype ( 27 ; data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16077119" target="_blank">16077119</a>]<br></li>
<li>In envZ : : cam null strains , the orphaned response-regulator OmpR is dependent upon phosphorylation by acetyl phosphate for DNA-bind-ing activity at ompF ( 17 ) , thereby allowing us to use OmpR-dependent activation at ompF to monitor acetyl phosphate levels .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9882671" target="_blank">9882671</a>]<br></li>
<li>Third , OmpR protein binds to the DNA fragment containing the ﬂhDC promoter , and its afﬁnity is increased with phosphorylation by acetyl phosphate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7642497" target="_blank">7642497</a>]<br></li>
<li>It should be noted that although it takes 3 h to reach 40 % phosphorylation of OmpR in the presence of 3,000-fold excess acetyl phosphate without DNA ( 24 ) , it takes less than 20 sec to transfer almost all of the phosphoryl group from 1 mM EnvZc-P to 4 mM OmpR ( Fig. 3A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11158569" target="_blank">11158569</a>]<br></li>
<li>Thus , the DNA-binding activity of OmpR , RstA and CpxR was examined in the presence of acetyl phosphate for phosphorylation in vitro ( note that phosphorylated forms are shown as OmpRp , RstAp and CpxRp in the respective figures ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20466769" target="_blank">20466769</a>]<br></li>
<li>On the basis of acetyl phosphate being only a phosphor donor for OmpR but not a phosphatase for OmpR-P and that the half-life of OmpR-P in the presence of DNA is slightly longer than that in the absence of DNA , it was concluded that DNA enhances the rate of phosphorylation of OmpR but not the rate of dephosphorylation of OmpR-P .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11158569" target="_blank">11158569</a>]<br></li>
<li>The 2 - to 7-fold stimulation of OmpR phosphorylation by acetyl phosphate in the presence of DNA may be a unique feature of OmpR phosphorylation by a small molecule .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11158569" target="_blank">11158569</a>]<br></li>
<li>Previously , Ames et al. reported that OmpR phosphorylation by acetyl phosphate and EnvZ were approximately 2-fold and 2-to 7-fold , respectively , stimulated by C1 DNA fragments ( 24 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11158569" target="_blank">11158569</a>]<br></li>
<li>Recently it was reported that phosphorylation of OmpR by either acetyl phosphate or EnvZ was enhanced in the presence of a DNA fragment containing OmpR-P-binding regions ( 24 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11158569" target="_blank">11158569</a>]<br></li>
<li>Based on the fact that the stability of OmpR-P was affected only slightly by the presence of DNA , these authors concluded that the increased level of OmpR-P in the presence of DNA is caused by the stimulation of the phosphorylation reaction of OmpR by acetyl phosphate and EnvZ .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11158569" target="_blank">11158569</a>]<br></li>
<li>This result supports a key prediction of the current model of porin regulation , which states that the differential regulation of OmpF and OmpC is a direct consequence of the cellular level of the active form of OmpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9422609" target="_blank">9422609</a>]<br></li>
<li>Therefore , this study provides strong evidence supporting the current model , which states that the level of the active form of OmpR 17 TABLE 2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9422609" target="_blank">9422609</a>]<br></li>
<li>However , in an envZ473 , $ ( ompR-ZacZ ) ZZ ( Hyb ) fusion strain in which OmpR + is present , EnvZ473 , working through OmpR + , confers a dominant OmpF - , OmpC + phenotype ( lane H ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2557454" target="_blank">2557454</a>]<br></li>
<li>As expected , the procaine treatment led to a drastic change in the amount of OmpC and OmpF porins , dependent upon OmpR and independent of OmrA or OmrB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16359331" target="_blank">16359331</a>]<br></li>
<li>Overexpression of OmpR presumably leads to the overexpression of OmpC or OmpF , both known to induce E activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17416652" target="_blank">17416652</a>]<br></li>
<li>Conversely , OmpR mutants with auto-phosphorylation defects have OmpF - OmpC - phenotypes , reflecting the inability to phosphorylate OmpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8098993" target="_blank">8098993</a>]<br></li>
<li>If MzrA ( YqjB ) directly inﬂuences OmpR , regardless of how OmpR gets phosphorylated , we may still observe the negative effects of MzrA ( YqjB ) on OmpF , LamB and MalE in an envZ null background .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19432797" target="_blank">19432797</a>]<br></li>
<li>To further evaluate a requirement of OmpR for the negative effect of MzrA ( YqjB ) overexpression on OMPs and MalE , we employed another wellstudied ompR allele , ompR472 ( Hall and Silhavy , 1981 ) , which produces an OmpF + OmpC - phenotype without inﬂuencing LamB and MalE expression ( Fig. 1B and C , lane 7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19432797" target="_blank">19432797</a>]<br></li>
<li>Again , we failed to see any negative effect of MzrA ( YqjB ) overexpression on OmpF , LamB or MalE ( Fig. 1B and C , lane 8 ) , thus emphasizing the need for wild type OmpR to observe the effects of MzrA ( YqjB ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19432797" target="_blank">19432797</a>]<br></li>
<li>When OmpR and EnvZc are mixed together with ATP , the following four reactions occur simultaneously in solution : autophosphorylation , EnvZc 1 ATP 3 EnvZc-P 1 ADP ; phosphotransfer , EnvZc-P 1 OmpR 3 EnvZc 1 OmpR-P ; autophosphatase , OmpR-P 3OmpR 1 Pi ; and EnvZc phosphatase , OmpR-P 3 OmpR 1 Pi .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11158569" target="_blank">11158569</a>]<br></li>
<li>A B C 2 3 4 5 6 7 8 9 10 I -- 4 500 ¬ ' ¬ - '' * Phospho-OmpR .5400 ca cXi 300-V qV4MPs a S Aft c ArcB + cytoplasmic membrane ¬ Phospho-AroB ATP ¬ c - Phospho-ArcB Phospho-ArcB + OmpR o - Phospho-OmpR ° 200-la Fig. 5 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7957084" target="_blank">7957084</a>]<br></li>
<li>A , OmpR was incubated at different concentrations ( 0 , 5 , 10 , 20 , 40 , and 80 nM ) with EnvZc11 in the reaction buffer in the presence ( lanes 1 -- 12 ) or absence ( lanes 13 -- 18 ) of 1 mM ATP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16618701" target="_blank">16618701</a>]<br></li>
<li>Complex II also forms at much lower OmpR concentrations ( compare Fig. 2B , complex II lane 2 in the absence and presence of kinase ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9096305" target="_blank">9096305</a>]<br></li>
<li>When EnvZc and OmpR were mixed together in the presence of 50 mM ATP and 5 mM Mg21 , the production of OmpR-P was barely detectable ( Fig. 1A , lane 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11158569" target="_blank">11158569</a>]<br></li>
<li>In the absence of DNA , the net result of these reactions is the hydrolysis of ATP , an ATPase reaction , and OmpR mainly exists as unphosphor-ylated because of the apparent strong phosphatase activity of EnvZc over its kinase activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11158569" target="_blank">11158569</a>]<br></li>
<li>It was also possible to eliminate the stimulation by performing the experiment using wild-type OmpR and the MBP-EnvZ kinase , but in the absence of ATP ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9096305" target="_blank">9096305</a>]<br></li>
<li>Phosphorylation Experiments-For studies on the transfer of the phosphoryl group from the EnvZ protein to the OmpR protein , the radioactivelyphosphorylated EnvZ * or EnvZ11 * ( 1p ~ ) ea , ch of which was free from ATP , was incubated with purified OmpR , OmpR3 , or OmpR77 ( 1.5 p ~ in ) TEDG buffer containing 5 mM MgC12and 50 mM KCl , either in the presence or absence of 0.1 mM ATP , at room temperature .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2668281" target="_blank">2668281</a>]<br></li>
<li>In this analysis , the DNA fragment containing the F1 site was labeled and incubated with different concentrations of OmpR in the presence or absence of a fusion protein kinase , MBP-EnvZ .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9096305" target="_blank">9096305</a>]<br></li>
<li>For this analysis , OmpR was titrated using the F1-F1 fragment in the absence or presence of the kinase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9096305" target="_blank">9096305</a>]<br></li>
<li>JOURNAL OF BACTERIOLOGY , Jan. 1998 , p. 171 -- / 98 / $ 04.0010 Copyright © 1998 , American Society for Microbiology Differential Expression of the OmpF and OmpC P Proteins in Escherichia coli K-12 Depends upon the Level of Active OmpR orin CHUNG-YU LAN AND MICHELE M. IGO * Section of Microbiology , Division of Biological Sciences , University of California , Davis , Davis , California 95616 Received 17 July 1997/Accepted 28 October 1997 We have generated a mutant form of the OmpR regulatory protein , OmpRD55E , that is active independent of the EnvZ kinase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9422609" target="_blank">9422609</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_426">
Primary characterization of the protein products of the Esche-richia coli ompB locus : structure and regulation of synthesis of the OmpR and EnvZ proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3531171" target="_blank">3531171</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2160945" target="_blank">2160945</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2656631" target="_blank">2656631</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3536870" target="_blank">3536870</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1885551" target="_blank">1885551</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1370290" target="_blank">1370290</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2404948" target="_blank">2404948</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2649492" target="_blank">2649492</a>]<br></label>
<input id="act_similarity_426", type="checkbox">
<div class="ACT_similar">
<li>Primary characterization of the protein products of Escherichia coli OmpB locus : structure and of regulation synthesis of the OmpR and EnvZ proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1532388" target="_blank">1532388</a>]<br></li></div>
<li>Comeau , D.E. , Ikenaka , K. , Tsung , K. and Inouye , M. : Primary characterization of the protein products of the Escherichia co/i ompB locus : structure and regulation of synthesis of the OmpR and EnvZ proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8294007" target="_blank">8294007</a>]<br></li>
<li>164 , No. 2 Primary Characterization of the Protein Products of the Escherichia coli ompB Locus : Structure and Regulation of Synthesis of the OmpR and EnvZ Proteins DOROTHY E. COMEAU , KAZUHIRO IKENAKA , KANGLA TSUNG , AND MASAYORI INOUYE * Department of Biochemistry , State University of New York at Stony Brook , Stony Brook , New York 11794 Received 10 June 1985/Accepted 15 August 1985 The ompB operon of Escherichia col contains the structural genes for two proteins , OmpR and EnvZ , which control the osmoregulated biosynthesis of the porin proteins OmpF and OmpC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2997120" target="_blank">2997120</a>]<br></li>
<li>174 , No. 3 Positive and Negative Control of ompB Transcription in Escherichia coli by Cyclic AMP and the Cyclic AMP Receptor Protein LIN HUANG , PING TSUI , t AND MARTIN FREUNDLICH * Department of Biochemistry and Cell Biology , State University of New York , Stony Brook , New York 11794-5215 Received 12 August 1991/Accepted 20 November 1991 The ompB operon encodes OmpR and EnvZ , two proteins that are necessary for the expression and osmoregulation of the OmpF and OmpC porins in Escherchia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1310090" target="_blank">1310090</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_431">
Expression and osmoregulation of these genes involves primarily transcriptional control mediated by OmpR and EnvZ , the protein products of the ompB operon ( 22 , 23 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1885551" target="_blank">1885551</a>]<br></label>
<input id="act_similarity_431", type="checkbox">
<div class="ACT_similar">
<li>The expression and osmoregulation of these genes are primarily under transcriptional control mediated by OmpR and EnvZ , the protein products of the ompB operon ( 22 , 23 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1310090" target="_blank">1310090</a>]<br></li></div>
<li>The ompB locus codes for two proteins , EnvZ and OmpR , which are thought to be involved in the regulation of porin protein expression in response to osmolarity change of the growth medium ( 13-15 , 17 , 36 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1695892" target="_blank">1695892</a>]<br></li>
<li>The protein products of the ompB operon , OmpR and EnvZ , are required for the expression and osmoregulation of the major outer membrane proteins OmpF and OmpC ( 16 , 17 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2203749" target="_blank">2203749</a>]<br></li>
<li>The narX and narL genes , as mapped in this region with the correct orientation ( 9 ) , encode proteins with apparent molecular weights of 62,000-66 ,000 and OmpR is involved in osmotic regulation of the synthesis of outer membrane proteins , and NtrC is involved in nitrogen control of several nitrogen assimilatory genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2657652" target="_blank">2657652</a>]<br></li>
<li>Among the plasmids that reduced function were some that may titrate E or reduce stress that signals to E ; some stimulatory plasmids identiﬁed in screen III act by activating OmpR , presumably increasing the synthesis of outer membrane proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17416652" target="_blank">17416652</a>]<br></li>
<li>It seems likely , therefore , that the Dye and OmpR proteins are related , and Dye may thus be involved in the osmoregulation of envelope protein genes as well as being required for sex factor gene expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2984198" target="_blank">2984198</a>]<br></li>
<li>A more likelypossibility is that Dye and OmpR recognize the same effector-molecule , probably involved in osmoregulation , while controlling different genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2984198" target="_blank">2984198</a>]<br></li>
<li>Besides the outer membrane porins , other genes ( 49 ) -- those encoding the tripeptide permease TppB of Salmonella typhimurium , the outer membrane protease Opr , the PhoA and PhoE proteins of the Pho regulon , and the positive regulator MalT of the maltose operon -- are known to be regulated by EnvZ and OmpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7642497" target="_blank">7642497</a>]<br></li>
<li>T h e bacteria1 response-regulator OmpR is a transcription factor that differentially regulates the expression of genes encoding the major outer membrane porin proteins , OmpF and OmpC [ l ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9016718" target="_blank">9016718</a>]<br></li>
<li>Correlation with mutagenesis data Transcription of the genes encoding the OmpF and OmpC proteins is regulated by OmpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9016718" target="_blank">9016718</a>]<br></li>
<li>Theoretically , this could be either due to direct regulation ( OmpR-binding to the promoter regions of omrA and omrB genes and subsequent regulation of transcription ) or to an indirect effect of the change in OmpC and OmpF porins ( or other OmpR regulated proteins ) in response to EnvZ-OmpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16359331" target="_blank">16359331</a>]<br></li>
<li>The essence of the model is that the EnvZ protein senses the medium osmolarity and then forces the OmpR protein to take one of two alternative structures , which positively regulate OmpC and OmpF synthesis respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3294816" target="_blank">3294816</a>]<br></li>
<li>Based on the analysis of these mutants , a model has been proposed for the osmoregulation of OmpF and OmpC synthesis , in which the OmpR protein can take two alternative structures , depending on the medium osmolarity , each of which positively regulates either OmpF or OmpC synthesis ( Hall and Silhavy 1981b ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3010044" target="_blank">3010044</a>]<br></li>
<li>These results support the view that the OmpR protein is directly involved in osmoregulation by taking two alternative structures depending on the medium osmolarity , each of which positively regulates either OmpF or OmpC synthesis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3010044" target="_blank">3010044</a>]<br></li>
<li>Hall and Silhavy have suggested that the carboxy terminus of OmpR contains A dimerization site which allows for the modulation of porin gene transcriptional activation , the monomer preferentially activating OmpF production and the dimer activating OmpC ; they further suggest that the functional state of the OmpR protein is controlled in some manner by the EnvZ protein ( 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2997120" target="_blank">2997120</a>]<br></li>
<li>In the model of Hall and Silhavy ( 11 ) , OmpR functions as a positive regulator of OmpF and OmpC transcription , and the envZ gene product is an envelope protein which senses the environment and influences porin synthesis by controlling the multimerization of the OmpR protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6317653" target="_blank">6317653</a>]<br></li>
<li>This may be because the effects of OmpR -- EnvZ on these genes were detected under speciﬁc growth conditions ( Case et al. , 1986 ; Yamamoto et al. , 2000 ) that differ from those we used .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12366850" target="_blank">12366850</a>]<br></li>
<li>Other affected genes include lamB and malE , which are normally not inﬂuenced by EnvZ/OmpR unless MzrA is overexpressed or in the presence of pleiotropic envZ mutations ( Wanner et al. , 1979 ; Wandersman et al. , 1980 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19432797" target="_blank">19432797</a>]<br></li>
<li>EnvZ is thought to be responsible for mediating the conversion of OmpR into the different forms ( Hall & Silhavy , 1981b ; Slauch et al. , 1988 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2557454" target="_blank">2557454</a>]<br></li>
<li>EnvZ is autophosphorylated in the presence of ATP , and the phosphoryl group on EnvZ is transferred to OmpR ( Igo and Silhavy 1988 ; Igo et al. 1989a ; Aiba et al. 1989a , b , c ; Aiba and Mizuno 1990 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8413182" target="_blank">8413182</a>]<br></li>
<li>It is worth noting that previously characterized pleiotropic EnvZ mutants were also shown to have diminished phosphatase activity , thus resulting in an elevated half-life of OmpR ~ P ( Aiba et al. , 1989 ; Tokishita et al. , 1991 ; Hsing et al. , 1998 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19432797" target="_blank">19432797</a>]<br></li>
<li>Furthermore , these fusions are also induced by procaine , a local anaesthetic that activates OmpR via EnvZ ( Taylor et al. , 1983 ; Ramp-ersaud and Inouye , 1991 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16359331" target="_blank">16359331</a>]<br></li>
<li>The products of the CAT reaction are indicated by arrowheads When O m p R is incubated with E n v Z l l , a mutant form of EnvZ that is defective in depbosphorylation but not in phosphorylation of OmpR , the phosphorylated form of O m p R accumulates stably ( Aiba et al. 1989b ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8413182" target="_blank">8413182</a>]<br></li>
<li>EnvZ 11 protein , which is a mutant form of EnvZ with normal kinase but decreased phosphatase activity , and OmpR protein were purified using published procedures ( Aiba et al. 1989a ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8413182" target="_blank">8413182</a>]<br></li>
<li>The ability of OmpR to activate transcription is controlled by its phosphorylation by a histidine protein kinase , EnvZ , which responds to changes in environmental osmolarity ( Igo and Silhavy 1988 ; Igo et al. 1989a , b ; Aiba et al. 1989a , b , c ; Aiba and Mizuno 1990 ; Slauch and Silhavy 1991 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8413182" target="_blank">8413182</a>]<br></li>
<li>EnvZ is an inner membrane protein ( Forst et al. 1987 ) that senses the osmolarity of the growth medium and acts as a kinase and as a phosphatase to alter the phosphorylation state of OmpR ( Aiba et al. 1989 ; Forst et al. 1989a ; Igo et al. 1989a , b ; Aiba and Minino 1990 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1736095" target="_blank">1736095</a>]<br></li>
<li>Among the five factors examined , the three response-regulators , OmpR , RstA and CpxR , are organized in two-component systems , and are converted to active forms only when phosphorylated by the respective sensor kinases , EnvZ ( Aiba et al. , 1989 ; Yamamoto et al. , 2005 ) , RstB ( Minagawa et al. , 2003 ; Yamamoto et al. , 2005 ) and CpxA ( Raivio & Silhavy , 1997 ; Yamamoto et al. , 2005 ) , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20466769" target="_blank">20466769</a>]<br></li>
<li>Stimulation of sensor kinase autophosphorylation by the presence of response-regulators has been observed for other two-component systems , such as FixL/FixJ , EnvZ/OmpR and RegS/RegR ( Tuckerman et al. , 2001 ; Mattison & Kenney , 2002 ; Emmerich et al. , Fig. 6 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15073297" target="_blank">15073297</a>]<br></li>
<li>A second set of the sRNAs , consisting of MicC , MicF , OmrA and OmrB , is controlled by the stress-induced EnvZ -- OmpR two-component system , and contributes to adjusting the levels of the major porins OmpC , OmpF and a number of non-porin OMPs , respectively , to environmental conditions ( Delihas and Forst , 2001 ; Chen et al. , 2004 ; Guillier and Gottesman , 2006 ; 2008 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19400782" target="_blank">19400782</a>]<br></li>
<li>Expression of tppB is induced by anaerobiosis and requires the OmpR and EnvZ proteins , which are also required for normal expression and regulation of the two major outermembrane porins ( Jamieson & Higgins , 1984 , 1986 : Gibson et al. , 1987 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2821267" target="_blank">2821267</a>]<br></li>
<li>mzrA expression is negatively controlled by EnvZ/OmpR Kato et al. ( 2003 ) discovered a negative regulatory feedback loop in which the expression of pmrD was shown to be negatively regulated by PmrA , the response-regulator of the PmrB/PmrA two-component system ; PmrD has been shown to promote phosphorylation of PmrA ( Kato and Groisman , 2004 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19432797" target="_blank">19432797</a>]<br></li>
<li>Although at present there is no biochemical evidence showing MzrA increases OmpR ~ P stability , the observation that overexpression of MzrA produces a phenotype that is almost identical to that seen in a strain expressing a pleiotropic envZ allele , indicates that the biochemical state of EnvZ/OmpR in these two genetic backgrounds is also very similar ; i.e. both backgrounds likely increase OmpR ~ P stability .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19432797" target="_blank">19432797</a>]<br></li>
<li>This and additional genetic and biochemical evidence suggest that the interaction of MzrA with EnvZ would either enhance EnvZ 's kinase activity or reduce its phosphatase activity , thus elevating the steady state levels of ~ OmpR P. Furthermore , our data show that MzrA links the two-component envelope stress response-regulators , CpxA/CpxR and EnvZ/OmpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19432797" target="_blank">19432797</a>]<br></li>
<li>Parallel tests with two other two-component regulatory systems showed that EnvZ/OmpR , known to be osmotically induced , was not induced under these conditions , although CpxA/CpxR , which monitors protein folding in the periplasm , was modestly induced , consistent with other ﬁndings ( 34 , 75 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16153174" target="_blank">16153174</a>]<br></li>
<li>We have provided evidence that MzrA links two major signal transduction regulatory systems : CpxA/CpxR and EnvZ/OmpR , which respond to different environmental and cellular signals to regulate outer membrane composition .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19432797" target="_blank">19432797</a>]<br></li>
<li>Thus , despite responding to different stimuli , MzrA-mediated linking of the CpxA/CpxR and EnvZ / OmpR regulatory systems would allow cells to amplify their response to appropriately adjust the envelope composition ( see Fig. 10 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19432797" target="_blank">19432797</a>]<br></li>
<li>Genes encoding OM proteins and surface appendages are potential negative targets of OmrA and OmrB Identifying EnvZ and OmpR as the regulators of omrA and omrB did not immediately lead us to the targets of these sRNAs , because no phenotypes due to post-tran-scriptional regulation by the EnvZ-OmpR system have been reported .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16359331" target="_blank">16359331</a>]<br></li>
<li>We have previously shown that both OmrA and OmrB are made in response to the activation of the EnvZ-OmpR two-component system and that their production results in a decrease in the level of several OMPs , such as the OmpT protease and the CirA , FecA and FepA receptors for iron -- siderophore complexes ( 20 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18953042" target="_blank">18953042</a>]<br></li>
<li>In the EnvZ-OmpR two-component system , OmpR , like RcsAB , negatively regulates ﬂhDC transcription ( 32 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23893115" target="_blank">23893115</a>]<br></li>
<li>Osmotic repression by the EnvZ/OmpR two-component signal transduction pathway may ensure that EcnB transcription is activated only in moribund cells .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9677290" target="_blank">9677290</a>]<br></li>
<li>CpxR indirectly regulates the transcription of the sRNAs OmrA and OmrB by activating transcription of the inner membrane protein MzrA ; MzrA physically interacts with EnvZ , increasing the activity of the EnvZ/OmpR two-component system , which activates omrA and omrB transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25246476" target="_blank">25246476</a>]<br></li>
<li>This transcriptional control is exerted via the well-characterized EnvZ/OmpR two-component signal transduction system in which EnvZ , an inner membrane histidine protein kinase , senses osmotic signals and transmits them to the transcription factor OmpR by phosphorylating it ( Pratt and Silhavy , 1995 ; Hoch , 2000 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11029695" target="_blank">11029695</a>]<br></li>
<li>RyhB is regulated by the Fur repressor ( 38 ) , RprA is regulated by the RcsC/RcsD/RcsB phosphorelay ( 34 ) , OmrA and OmrB are regulated by the EnvZ/OmpR two-component system ( 21 ) , and MicA and RybB are regulated by E ( 16 , 27 , 50 , 59 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18978044" target="_blank">18978044</a>]<br></li>
<li>We have shown that transcription of the sRNAs is activated by the two-component system EnvZ-OmpR and that , in turn , OmrA and OmrB negatively regulate the expression of multiple genes encoding OM proteins , including ompT , cirA , fecA and fepA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16359331" target="_blank">16359331</a>]<br></li>
<li>DISCUSSION OmrA and OmrB , two adjacent and partially homologous Hfq-binding sRNAs , are positively regulated by the EnvZ / OmpR two-component system and negatively regulate a set of outer membrane proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18953042" target="_blank">18953042</a>]<br></li>
<li>Our ﬁnding that the EnvZ/OmpR signaling pathway mediates motility suppression in high [ NaCl ] is consistent with previous work that found phosphorylated OmpR to negatively regulate the expression of ﬂagella by interacting with the promoter region of ﬂhDC [ 66 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17941710" target="_blank">17941710</a>]<br></li>
<li>It is important to note that MzrA expression from the pTrc99A clone was unaffected in various mutant OmpR and EnvZ backgrounds ( Fig. 1D ) , demonstrating that a lack of negative effect on OMPs or MalE was not the result of reduced MzrA expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19432797" target="_blank">19432797</a>]<br></li>
<li>Although the porin expression proﬁle in cells either lacking or overexpressing MzrA was drastically different compared with that seen in wild type MzrA + cells , porin expression still responded to changes in medium osmolarity , pH or the presence of procaine in the medium , suggesting that MzrA is not the primary receptor of these three environmental cues that inﬂuence porin expression via EnvZ/OmpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19432797" target="_blank">19432797</a>]<br></li>
<li>The fact that porin expression still responds to medium osmolarity in cells lacking or overexpressing MzrA suggets that MzrA is not the primarily receptor of medium osmolarity , rather , it appears to bias EnvZ/OmpR activity to favour OmpC expression over OmpF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19432797" target="_blank">19432797</a>]<br></li>
<li>However , porin expression in strains devoid of MzrA or overexpressing it is still sensitive to medium osmolarity , pH and procaine , all of which modulate EnvZ/OmpR activities .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19432797" target="_blank">19432797</a>]<br></li>
<li>Regulation of porin expression in response to osmolarity involves the EnvZ and OmpR proteins ( 15 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8002608" target="_blank">8002608</a>]<br></li>
<li>OmpR is a transcriptional-regulator of omrA and omrB If EnvZ and OmpR are needed for the expression of OmrA and OmrB , we might expect these sRNAs to be induced at high osmolarity , when the level of phosphorylated OmpR is expected to be high .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16359331" target="_blank">16359331</a>]<br></li>
<li>These results suggest that EnvZ regulates porin expression by controlling the level of phosphorylated OmpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2557454" target="_blank">2557454</a>]<br></li>
<li>These results indicate that OmpR phosphorylated by EnvZ negatively regulates the expression of bolA1p dependent on RpoS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10802181" target="_blank">10802181</a>]<br></li>
<li>Model for the transcriptional regulation of porin expression by OmpR and EnvZ .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2557454" target="_blank">2557454</a>]<br></li>
<li>Osmotic-stress regulates the ratio of phosphatase ( P ) to autokinase ( K ) activities in EnvZ to adjust the phosphorylation state of OmpR , which in turn controls porin expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8098993" target="_blank">8098993</a>]<br></li>
<li>Perhaps high expression of ompB in IHF mutants leads to changed levels of EnvZ in relation to OmpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2203749" target="_blank">2203749</a>]<br></li>
<li>If MzrA activates the EnvZ/OmpR regulatory system then omrAB expression is expected to increase when MzrA is overproduced and decrease in the absence of MzrA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19432797" target="_blank">19432797</a>]<br></li>
<li>Interestingly , we discovered here that like the case with pmrD-PmrB/PmrA , mzrA expression is negatively regulated by EnvZ/OmpR that are post-translationally modulated by MzrA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19432797" target="_blank">19432797</a>]<br></li>
<li>Although OmpR phosphorylation depends primarily on EnvZ , the following evidence supports the existence of an alternative pathway that involves acetyl P. Mutants that lack EnvZ retain some ability to phosphorylate OmpR ( 134 ) and hence can still regulate OmpC and OmpF expression ( 133 , 394 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15755952" target="_blank">15755952</a>]<br></li>
<li>In contrast , using a sitedirected mutagenesis approach and in vitro phosphorylation of EnvZ , we obtained results that led us to conclude that ompC expression was correlated with increased levels of nonphosphorylated OmpR ( 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2160945" target="_blank">2160945</a>]<br></li>
<li>In contrast , our analysis using the ompR1 allele in the envZ247 mutant under the different sets of expression conditions illustrates the dependence of wild-type OmpR on phosphorylation by either EnvZ or other phosphate donors .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9422609" target="_blank">9422609</a>]<br></li>
<li>resultsindicate that OmpR bindssequentially to these se-It is generally believed that at low osmolarity , EnvZ main - quences in a manner determinedby the phosphorylated state of tains relatively low levels of phosphorylated OmpR , thereby the protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8175665" target="_blank">8175665</a>]<br></li>
<li>At low osmolarity , OmpF predominates in the outer membrane , while at high osmolarity the OmpF porin is replaced by OmpC .2 OmpF has a larger pore and a faster ¯ ow rate than OmpC .3 EnvZ is an inner membrane protein that is phosphorylated by intracellular ATP at histidine 243 in response to a signal related to the environmental osmolarity .4 ± 6 It transfers this phosphoryl group to aspartic acid 55 of OmpR .7 It has been suggested that EnvZ also catalyzes the dephosphorylation of phospho-OmpR ( OmpR-P ) ,8 further controlling the cellular levels of OmpR-P .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11812125" target="_blank">11812125</a>]<br></li>
<li>Three two-component systems ( Cpx , EnvZ/OmpR , and Rcs ) are implicated in the regulation of curli biogenesis in E. coli and contribute to the modiﬁcation of the bacterial surface in response to any change in osmolarity , among others factors ( 43 , 44 , 55 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15743952" target="_blank">15743952</a>]<br></li>
<li>EnvZ/OmpR , H-NS , and the Cpx and Rcs systems all appear to affect bioﬁlm formation by regulating the production of curli ﬁmbriae in response to osmotic conditions ( 16 , 39 , 54 , 67 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16997959" target="_blank">16997959</a>]<br></li>
<li>Expression of the ecnB gene is induced by high osmolarity in stationary phase , positively regulated by RpoS ( 66 ) and negatively regulated by the EnvZ/OmpR two-compo-nent regulator ( 12 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15576800" target="_blank">15576800</a>]<br></li>
<li>Thus , environmental osmotic activity is sensed by the sensor component EnvZ of the archetypal two-compo-nent system , EnvZ-OmpR , and high osmolarity results in the phosphorylation of OmpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14665678" target="_blank">14665678</a>]<br></li>
<li>The OmpR-EnvZ TCS responds to changes in osmolarity and positively regulates csgD expression ( 13 , 35 ) , and thus the expression of csgD is activated by the EnvZ-OmpR TCS under high-osmolarity conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17468243" target="_blank">17468243</a>]<br></li>
<li>The E pathway is activated by misfolded OMPs that accumulate in the periplasm under conditions of cell envelope stress ( 51 ) , and the EnvZ-OmpR system is responsive to high osmolarity ( reviewed in reference 36 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19734312" target="_blank">19734312</a>]<br></li>
<li>Two osmotic responses observed in E. coli , osmotaxis ( 24 ) and control of porin synthesis by the EnvZ-OmpR proteins ( 3 , 9 ) , appear to sense medium osmolarity ; both responses are at control levels in medium of low osmolarity and are activated in medium of high osmolarity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1532388" target="_blank">1532388</a>]<br></li>
<li>We now report a novel chromosomal bacteriolytic module of Escherichia coli called the entericidin locus , which is activated in stationary phase under high osmolarity conditions by sS and simultaneously repressed by the osmoregulatory EnvZ/OmpR signal transduction pathway .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9677290" target="_blank">9677290</a>]<br></li>
<li>EnvZ and OmpR regulate the level of the two major outer membrane ( OM ) porins in E. coli , OmpC and OmpF , and although the molecular basis is not known , EnvZ is induced by high osmolarity ( Pratt and Silhavy , 1995a ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16359331" target="_blank">16359331</a>]<br></li>
<li>The EnvZ/OmpR system is known to control the expression of outer membrane porins in response to medium osmolarity ( Aiba and Mizuno , 1990 ; Pratt and Silhavy , 1995 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12366850" target="_blank">12366850</a>]<br></li>
<li>Introduction The expression of the major outer membrane proteins , OmpF and OmpC , in Escherichia coli is regulated at the transcriptional level by the products of the ompB operon , OmpR and EnvZ ( Garrett et al. , 1983 ; Hall & Silhavy , 1981b ; Sarma & Reeves , 1977 ) in response to changes in medium osmolarity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2557454" target="_blank">2557454</a>]<br></li>
<li>The expression of the major outer membrane proteins OmpF and OmpC in E. coli is regulated at the transcriptional level by OmpR and EnvZ in response to changes in osmolarity of the medium ( Kawaji et al. 1979 ; Hall and Silhavy 1979 , 1981 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8413182" target="_blank">8413182</a>]<br></li>
<li>However , the environmental cues to which they respond are distinct ; whereas alkaline pH or misfolded pilus subunits activate CpxA/CpxR , medium osmolarity is the best-characterized effector of EnvZ/OmpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19432797" target="_blank">19432797</a>]<br></li>
<li>Because EnvZ not only has kinase activity to phosphorylate OmpR but also phosphatase activity toward phosphorylated OmpR ( OmpR-P ) , it is able to regulate the level of OmpR-P in the cell depending upon the medium osmolarity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16618701" target="_blank">16618701</a>]<br></li>
<li>The cellular level of OmpR-P is thus finely tuned by dual EnvZ functions , OmpR kinase and OmpR-P phosphatase , whose ratios are regulated by the medium osmolarity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16618701" target="_blank">16618701</a>]<br></li>
<li>EnvZ protein is known to phosphorylate OmpR by modulating its kinase and phosphatase activities in response to the osmolarity of the medium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7642497" target="_blank">7642497</a>]<br></li>
<li>A final word about the model : we believe that the various forms of OmpR exist in equilibrium with each other and that EnvZ functions to alter this equilibrium in response to changes in medium osmolarity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2557454" target="_blank">2557454</a>]<br></li>
<li>Synthesis of these porins is regulated in the osmolarity of the external response to medium by two-component regulatory system consisting of a the products of the ompB locus , OmpR and EnvZ ( 7 , 8 , 22 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1370954" target="_blank">1370954</a>]<br></li>
<li>How this could occur is unclear , since differential synthesis of OmpF and OmpC is usually mediated by the products of the ompB operon , OmpR and EnvZ , in response to the osmolarity of the growth medium ( 22 , 23 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1310090" target="_blank">1310090</a>]<br></li>
<li>For example , the OmpF and OmpC porins are reciprocally regulated in response to the osmolarity of the medium through the action of the OmpR and EnvZ proteins ( 16 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2644187" target="_blank">2644187</a>]<br></li>
<li>By analogy to NtrC ( NRI , ginG ) and NtrB ( NRII , glnL ) in nitrogen control ( 31 ) , CheY and CheA in chemotaxis ( 24 ) , and OmpR and EnvZ in osmoregulation ( 2 , 19 , 26 ) , PhoR may be an autophosphorylase that also phosphorylates and dephosphorylates PhoB to convert it between active and inactive forms .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2160940" target="_blank">2160940</a>]<br></li>
<li>This hypothesis is based on the striking similarities that are shared at the protein sequence level for the regulators PhoB , NtrC , CheY , and OmpR and for their respective sensory proteins PhoR , NtrB , CheA , and EnvZ , which are now thought to act in this way ( 7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2160940" target="_blank">2160940</a>]<br></li>
<li>Most NtrC-like regulators function in pairs ( PhoRPhoB , EnvZ-OmpR , CpxA-SfrA , VirA-VirG , DctB-DctD , CheA-CheB , CheY ) , including NifA , which is modified by the NifL product .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2666400" target="_blank">2666400</a>]<br></li>
<li>Nonetheless , the portion of UhpB which appears to be involved in the binding of UhpA overlaps the homologous region of a VanSderived peptide which binds VanR ( 46 ) , and portions of NtrB and EnvZ which are necessary for phosphatase activity on NtrC and OmpR , respectively ( 19 , 23 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11325944" target="_blank">11325944</a>]<br></li>
<li>Furthermore , we demonstrate that this transcriptional modulation is mediated by a direct interaction of the osmoregulator OmpR with the promoter region of ﬂhDC , suggesting that the repression of ﬂhDC may occur through the phosphorylation , by either EnvZ or acetyl-P , of OmpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7642497" target="_blank">7642497</a>]<br></li>
<li>Osmoregulationof porin proteins occupancy of binding sites in the ompF promoter and OmpR is controlled at the transcriptional level by the OmpR and EnvZ phosphorylation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8175665" target="_blank">8175665</a>]<br></li>
<li>In E. coli , the RcsBCD , CpxAR , BaeSR , and EnvZ-OmpR signal transduction systems are generally believed to sense and respond to stresses that damage or alter the cell envelope ( 7 -- 11 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23893115" target="_blank">23893115</a>]<br></li>
<li>Although the cloned C. freundii entericidin locus is expressed in E. coli independently of EnvZ/OmpR , a unique ompR-like gene called ecnR is linked to the C. freundii enterici-din locus and may potentially provide repression in this organism .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9677290" target="_blank">9677290</a>]<br></li>
<li>Abbreviations used : OmpR-P , phospho-OmpR ; EnvZ-P , phospho-EnvZ ; EnvZc , carboxyl terminal fragment of EnvZc ; Pi , inorganic phosphate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11812125" target="_blank">11812125</a>]<br></li>
<li>EnvZ is autophosphorylated when incubated with ATP , and readily transfers the phosphate group to OmpR [ 12-141 ; EnvZ also exhibits phosphatase activity for phospho-OmpR [ 12,13 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8522138" target="_blank">8522138</a>]<br></li>
<li>If acetyl phosphate phosphorylates OmpR appropriately , then OmpR-P should exhibit the properties of an acyl phosphate and should be sensitive to dephosphorylation by the phosphatase activity of EnvZ ( 29 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7568033" target="_blank">7568033</a>]<br></li>
<li>Previous studies indicate that in the absence of EnvZ , the wild-type OmpR protein can be phosphorylated by other cellular phosphate donors , such as nonpartner kinases or acetyl phosphate , and is inﬂuenced by the copy number ( 8 , 11 , 22 -- 24 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9422609" target="_blank">9422609</a>]<br></li>
<li>Although acetyl phosphate can phosphorylate OmpR in vivo , EnvZ controls the cellular OmpR-P concentration and mediates osmoregulation ( 3 -- 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11158569" target="_blank">11158569</a>]<br></li>
<li>Considering these data together , OmpR appears to be phosphorylated by acetyl phosphate , but not by EnvZ , to act as a repressor of the pga operon by direct binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25667010" target="_blank">25667010</a>]<br></li>
<li>T83I is phosphorylated slowly compared to wild-type OmpR T83I binds EnvZ as well as wild-type OmpR It was possible that the slow rate of phosphoryl-ation of T83I by EnvZ was a result of the substitution disrupting the interaction between EnvZ and OmpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11812125" target="_blank">11812125</a>]<br></li>
<li>Radioactively phosphorylated EnvZ * was purified and then incubated with OmpR in the absence of ATP in a buffer containing 5 mM M & l2 and 50 mMKC1 at room temperature for 30 s. Subsequently ,0.1 mM nonlabeled ATP was addedt Th ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2668281" target="_blank">2668281</a>]<br></li>
<li>Radioactively phosphorylated EnvZ * and EnvZ11 ' ( 1PM ) were purified and then incubatedwith purifiedOmpR and OmpR3 ( 1.5 pM ) in a reaction mixture containing5 mM MgC12 and 50mM KC1at room temperature in the following combinations : panels A and B , EnvZ * - OmpR panel C , EnvZ11 * - OmpR panel D , EnvZ * - OmpR3 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2668281" target="_blank">2668281</a>]<br></li>
<li>Because all of these plasmids act through EnvZ-OmpR , it is likely that the concentration of phosphorylated OmpR in the presence of some of these plasmids is sufﬁcient to activate omrB transcription , but not omrA transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16359331" target="_blank">16359331</a>]<br></li>
<li>Phosphorylation and Dephosphorylation of the OmpR Pm-tein Mediated by the EnvZ Protein-As demonstrated previously ( 21 ) , EnvZ * is capable of phosphorylating OmpR in vitro in the presence of ATP , the phosphorylated OmpR then undergoing dephosphorylation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2668281" target="_blank">2668281</a>]<br></li>
<li>EnvZ , phosphorylated directly from ATP , transfers its histidyl phosphate to OmpR and subsequently stimulates OmpR ~ P dephosphorylation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19432797" target="_blank">19432797</a>]<br></li>
<li>EnvZ is a bifunctional enzyme that possesses both OmpR kinase and phosphorylated OmpR ( OmpR-P ) phosphatase activities and thus controls the cellular level of OmpR-P .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11158569" target="_blank">11158569</a>]<br></li>
<li>As a result , OmpR will not be phosphorylated or dephosphorylated by EnvZ .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15175316" target="_blank">15175316</a>]<br></li>
<li>This portion of EnvZ is also capable of dephosphorylating the phosphorylated form of OmpR , OmpR-P ( 3 , 4 , 21 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8113170" target="_blank">8113170</a>]<br></li>
<li>EnvZ also acts as a phosphatase , converting phosphorylated OmpR to its dephosphorylated form .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8002608" target="_blank">8002608</a>]<br></li>
<li>In the absence of OmpR , there is a low basal level of ATPase activity associated with EnvZ phosphorylation and spontaneous dephosphorylation .25,33 This rate ( 7.1 nmol/ml per minute ) was subtracted from the OmpR or T83I-stimulated rate and only the stimulated rates are shown .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11812125" target="_blank">11812125</a>]<br></li>
<li>In another procaryotic regulatory system which senses the availability of nitrogen in the extracellular environment , phosphorylation of an aspartic acid residue ( 34 ) of the transcription activator protein NtrC by the sensory protein NtrB , which are structurally similar to OmpR and EnvZ , respectively ( 26 ) , activates transcription of the ginA gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2656631" target="_blank">2656631</a>]<br></li>
<li>Article and publication date are at www.pnas.orgycgiydoiy10 .1073 ypnas .031383098 ined the effect of OmpR-P-bound DNA on EnvZ function and found that the addition of a DNA fragment has a negligible effect on OmpR phosphorylation by EnvZc but dramatically reduces OmpR-P dephosphorylation by EnvZc .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11158569" target="_blank">11158569</a>]<br></li>
<li>Among the four reactions occurring in the mixture , only the EnvZ phosphatase activity was inhibited severely by the speciﬁc DNA , in contrast to the previous report by Kenney and her associates that DNA stimulates OmpR phosphorylation by EnvZ [ Ames , S. K. , Frankema , N. & Kenney , L. J. ( 1999 ) Proc .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11158569" target="_blank">11158569</a>]<br></li>
<li>In the former study , P C OmpR -- DNA sequestration appeared to inhibit phosphatase activity by EnvZ [ via squelching of ( P C ) OmpR ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11739766" target="_blank">11739766</a>]<br></li>
<li>Thus , unlike that conferred by wild-type OmpR , the phenotype conferred by OmpRD55E is not affected by the presence or absence of the EnvZ kinase , suggesting that this mutant protein bypasses the requirement for protein phosphorylation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9422609" target="_blank">9422609</a>]<br></li>
<li>Thus , the mutant T83I binds to EnvZc with slightly higher af ® nity than wild-type OmpR , and the decrease in phosphorylation observed with T83I is not due to an inability to interact with EnvZ .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11812125" target="_blank">11812125</a>]<br></li>
<li>The increased phosphorylation of OmpR may be due to enhanced EnvZ-kinase activity , reduced EnvZ-phosphatase activity , changes in the stability of the phosphate group of OmpR-phosphate , or a combination of these events .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2160945" target="_blank">2160945</a>]<br></li>
<li>Finally ; EnvZ , in the absence of OmpR , might indirectly increase the acetyl P pool to a level that increases the phosphorylation of PhoB via acetyl P , a scenario that resembles the connection between GS and acetyl P ( 322 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15755952" target="_blank">15755952</a>]<br></li>
<li>Biochemical characterization of this mutant indicates that this alteration of the phosphorylation site prevents the phosphorylation of OmpR by the EnvZ kinase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9096305" target="_blank">9096305</a>]<br></li>
<li>Thus , it is likely that some fraction of OmpR beyond the reach of the phosphatase and kinase activities of EnvZ would be available for phosphorylation by acetyl-P .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7642497" target="_blank">7642497</a>]<br></li>
<li>The EnvZ-independent phosphorylation of OmpR depends , in large part , on the ability of cells to synthesize acetyl P ( 291 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15755952" target="_blank">15755952</a>]<br></li>
<li>Even in cells that retain EnvZ , acetyl P can contribute to OmpR phosphorylation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15755952" target="_blank">15755952</a>]<br></li>
<li>EnvZ-P ADP ( auto-phosphorylation ) ; EnvZ-P OmpR !	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11812125" target="_blank">11812125</a>]<br></li>
<li>Given the fact that they also act upstream of EnvZ ( Fig. 3 ) , they will induce OmpR phosphorylation , whereas with pOmpR , OmpR will be overproduced , but not necessarily activated by phosphorylation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16359331" target="_blank">16359331</a>]<br></li>
<li>n this study , we characterized mutant EnvZ and OmpR proteins in terms of their in vitro phosphorylation and dephosphorylation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2668281" target="_blank">2668281</a>]<br></li>
<li>The simplest interpretation of these observations as to the phosphorylation and dephosphorylation of the mutant proteins is that EnvZ * is involved not only in OmpR phosphorylation but also in OmpR dephosphorylation and that this particular mutant , EnvZ11 * , can not effectively mediate the OmpR dephosphorylation reaction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2668281" target="_blank">2668281</a>]<br></li>
<li>EnvZ is an inner membrane protein ( 16 ) which is thought to sense the osmolarity of the medium and to subsequently alter OmpR by phosphorylation and dephosphorylation ( 3 , 4 , 17 , 28 , 29 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1310090" target="_blank">1310090</a>]<br></li>
<li>Autoradiograms showing the phosphorylation and dephosphorylation reactions of the OmpR protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2668281" target="_blank">2668281</a>]<br></li>
<li>phosphorylation and dephosphorylation abilities as to the activator protein , OmpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24966114" target="_blank">24966114</a>]<br></li>
<li>First , EnvZ is a histidine kinase ( 1 , 7 , 12 , 14 ) that responds to environmental stress by converting OmpR from an inactive form to an active protein via phosphorylation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9422609" target="_blank">9422609</a>]<br></li>
<li>DISCUSSION Based onthe results presented in this andprevious papers , the most probable scheme for the phosphorylation and dephosphorylation of the positive regulator OmpR , whichis mediated by the protein kinase EnvZ , is summarized in Fig. 6 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2668281" target="_blank">2668281</a>]<br></li>
<li>Autoradiograms showingthe time course of the phosphorylation of the OmpR protein by the EnvZ protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2668281" target="_blank">2668281</a>]<br></li>
<li>We provide evidence that the EnvZ protein is involved not only in OmpR phosphorylation but also in OmpR dephosphorylation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2668281" target="_blank">2668281</a>]<br></li>
<li>Fromthese results , we concludethat theEnvZ protein is involved not only in OmpR phosphorylation but also in OmpR dephosphorylation and that ATP is required for the dephosphorylationreaction mediated by the EnvZ protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2668281" target="_blank">2668281</a>]<br></li>
<li>The cysteine 354 and 277 residues of Salmonella enterica serovar Typhi EnvZ are determinants of autophosphorylation and OmpR phosphorylation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25667010" target="_blank">25667010</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_566">
The osmotic sensor , EnvZ , located in the isolated cytoplasmic membrane displays its phosphorylation and dephosphorylation abilities as to the activator , OmpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14996819" target="_blank">14996819</a>]<br></label>
<input id="act_similarity_566", type="checkbox">
<div class="ACT_similar">
<li>The osmotic sensor , EnvZ , located in the isolated cytoplasmic membrane displays its phosphorylation and dephoshorylation abilities as to the activator protein , OmpR , J. Biochem .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17537579" target="_blank">17537579</a>]<br></li>
<li>The osmotic sensor , EnvZ , located in the isolated cytoplasmic membrane displays its phosphorylation and dephosphorylation abilities as to the activator protein , OmpR , J. Biochem .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16153782" target="_blank">16153782</a>]<br></li></div>
<li>The activity of OmpR is regulated through a phosphorylation and dephosphorylation cascade by the sensor kinase EnvZ [ 18 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10802181" target="_blank">10802181</a>]<br></li>
<li>The EnvZ sensor protein monitors external signals and modulates the activity of the OmpR response-regulator by phosphorylation and dephosphorylation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15466019" target="_blank">15466019</a>]<br></li>
<li>The critical role of DNA in the equilibrium between Escherichia coli modulates its porin expression through a histidine kinase , EnvZ , and its cognate response-regulator , OmpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11158569" target="_blank">11158569</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_572">
The critical role of DNA in the equilibrium between OmpR and phosphorylated OmpR mediated by EnvZ in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11739766" target="_blank">11739766</a>]<br></label>
<input id="act_similarity_572", type="checkbox">
<div class="ACT_similar">
<li>The critical role of DNA in the equilibrium between OmpR and phosphorylated OmpR mediated by EnvZ in Escheri-chia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11812125" target="_blank">11812125</a>]<br></li></div>
<li>The partially purified form of OmpR was used in this study since , unlike purified forms of OmpR , it retained high affinity binding and did not require the addition of EnvZ and ATP to stimulate binding h101 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7557323" target="_blank">7557323</a>]<br></li>
<li>The amounts of partially purified OmpR was estimatedon the basisof comparisons with known amounts Under these experimental conditions , EnvZ ( C ) alone or low of pure OmpR in Western blots using anti-OmpaRntibody .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8175665" target="_blank">8175665</a>]<br></li>
<li>The demonstration that OmrA and OmrB are activated by EnvZ and OmpR and that they regulate multiple OM proteins , coupled with previous ﬁndings for other sRNAs , suggests that a major cellular role of regulatory RNA molecules is to modulate and ﬁne-tune the cell surface .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16359331" target="_blank">16359331</a>]<br></li>
<li>The EnvZ-OmpR system and at least ﬁve sRNAs could thus act in co-ordination to regulate the OM composition , which should ensure the entry of essential nutrients in the cell as well as the exclusion of toxic compounds and also minimize the possibility of recognition by the host .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16359331" target="_blank">16359331</a>]<br></li>
<li>Tpp is induced by anaerobic-growth and is positively regulated by the EnvZ/OmpR legulatory system ( Gibson et aL , 1987 ) , The dipeptide permease ( Dpp ) , like the Opp , is a binding-protein-dependent system .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7536291" target="_blank">7536291</a>]<br></li>
<li>The fact that the phenotype conferred by the ompR1 allele is different with these three envZ alleles supports the hypothesis that OmpR can be phosphorylated by other phosphate donors in the absence of EnvZ .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9422609" target="_blank">9422609</a>]<br></li>
<li>The EnvZ-OmpR system is a central element of this network , since phosphorylated OmpR ( OmpR-P ) is absolutely required for OmpF and OmpC expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16077119" target="_blank">16077119</a>]<br></li>
<li>The sequences of KdpD and KdpE show they are members of the sensor-effector class of regulatory proteins : the C-terminal half of KdpD is homologous to sensors such as EnvZ and PhoR , and KdpE is homologous to effectors such as OmpR and PhoB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1532388" target="_blank">1532388</a>]<br></li>
<li>The results presented in this study newly provide several implications not only as to the gene organization of the ompB operon , but also as to the functions of the OmpR and EnvZ proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3294816" target="_blank">3294816</a>]<br></li>
<li>The EnvZ Protein Is Involved in the Dephosphorylation of the OmpR Protein-The fact that EnvZ11 * was defective in OmpR dephosphorylation suggests that the EnvZ protein is involved in OmpR dephosphorylation as well .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2668281" target="_blank">2668281</a>]<br></li>
<li>The phosphorylated OmpR protein undergoes dephosphorylation , which is also mediated by the EnvZ protein and ATP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2668281" target="_blank">2668281</a>]<br></li>
<li>The pleio-tropic effect of regulation by EnvZ and OmpR suggests the role of OmpR as a global regulator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7642497" target="_blank">7642497</a>]<br></li>
<li>The phosphoryl group on the EnvZ protein is rapidly transferred to the OmpR pro ( t2e1i ) n .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2668281" target="_blank">2668281</a>]<br></li>
<li>The transfer of the phosphoryl group from EnvZ1t1o * OmpR was as rapid as in the case of EnvZ * .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2668281" target="_blank">2668281</a>]<br></li>
<li>The results indicatedthat therate of dephosphorylationof OmpR3 was very slow as compared with that in the case of OmpR , although the transferof the phosphoryl group from EnvZ * to OmpR3 occurred rapidly .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2668281" target="_blank">2668281</a>]<br></li>
<li>The hatched and shaded rectangles denote the OmpR and EnvZ proteins , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2668281" target="_blank">2668281</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2668281" target="_blank">2668281</a>]<br></li>
<li>The Pi produced in the presence of OmpR or T83I was obtained by subtracting the Pi produced in the presence of EnvZ alone from the total Pi produced .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11812125" target="_blank">11812125</a>]<br></li>
<li>The discovery of this property of yqjB led to its new designation , mzrA -- modulator of EnvZ and OmpR A .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19432797" target="_blank">19432797</a>]<br></li>
<li>The experiment shown in Figure 6 represents the ® rst direct measurements of OmpR and EnvZ interaction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11812125" target="_blank">11812125</a>]<br></li>
<li>This result is consistent with those of previous studies ( 19 , 23 ) , which suggested that increasing the total amount of OmpR in the cell does not change the total amount of active OmpR ( OmpR-P ) in the cell when EnvZ is present .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9422609" target="_blank">9422609</a>]<br></li>
<li>Consistent with this , omr : : lacZ activity increased 4.5-fold in response to MzrA overexpression and this increase was absolutely dependent on the EnvZ/OmpR system ( Fig. 3B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19432797" target="_blank">19432797</a>]<br></li>
<li>Such negative feedback-regulatory loop appears to be well suited to re-establish homeostasis when the ` signals ' that activate the mzrA-EnvZ/OmpR cascade have dissipated ( Fig. 10 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19432797" target="_blank">19432797</a>]<br></li>
<li>Therefore , multiple systems ( EnvZ/OmpR , Cpx , Rcs , and H-NS ) appear to be involved in sensing environmental osmolarity , leading to sophisticated regulation of the curli genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15743952" target="_blank">15743952</a>]<br></li>
<li>However , in this analysis , we did not detect regulation of some genes , such as bolA and malT , that were known to be under the control of OmpR -- EnvZ .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12366850" target="_blank">12366850</a>]<br></li>
<li>Interestingly , not all genes regulated by OmpR and EnvZ seem to be sensitive to osmotic change .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15175316" target="_blank">15175316</a>]<br></li>
<li>Unfortunately , without a better understanding of the stimulus for EnvZ and the mechanism of transcriptional control by OmpR and CpxR , it is difﬁcult to determine the regulatory logic for the porin promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16077119" target="_blank">16077119</a>]<br></li>
<li>Perhaps the altered activity of the multiple ompB promoters due to cyclic AMP-CRP control leads to a change in the ratio of OmpR and EnvZ .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1310090" target="_blank">1310090</a>]<br></li>
<li>To accomplish this regulation , the sensor EnvZ is thought to control-the-activity of the transcriptional factor OmpR in two ways .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9422609" target="_blank">9422609</a>]<br></li>
<li>Thus , it would appear subordinate that of that regulation of EnvZ synthesis is to OmpR , relying on reinitiation at the ATGA sequence and providing a mechanism for controling the ratio of the two proteins in the cell ; this ratio may be important for appropriate modulation of OmpR activities by EnvZ .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2997120" target="_blank">2997120</a>]<br></li>
<li>Cells expressing these hybrid proteins were assayed for their effect on the levels of the porins OmpF and OmpC , whose transcription depends on the phosphorylation of OmpR by EnvZ .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11325944" target="_blank">11325944</a>]<br></li>
<li>Using similar techniques , we investigated how OmpR and EnvZ regulate the transcription of tppB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15175316" target="_blank">15175316</a>]<br></li>
<li>Two cellular proteins , OmpR and EnvZ , are known to participate in this regulation at the level of transcription ( 1,2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2436145" target="_blank">2436145</a>]<br></li>
<li>Furthermore , in vitro transcription of the fadL gene was strongly repressed by the addition of OmpR and EnvZ proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8413182" target="_blank">8413182</a>]<br></li>
<li>Effect of OmpR and EnvZ proteins onfadL transcription in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8413182" target="_blank">8413182</a>]<br></li>
<li>J Bacteriot 160 : 122-130 , Gibson , M.M , , Ellis , E.M. , Graeme-Cook , K.A. , and Higgins , CF. ( 1987 ) OmpR and EnvZ are pleiotropic regulatory proteins : positive regulation of the tripeptide permease ( fppS ) of Saimonetta typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7536291" target="_blank">7536291</a>]<br></li>
<li>It is also unclear as to whether or not the two regulatory proteins , OmpR and EnvZ , cooperate , and if so , how .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3536870" target="_blank">3536870</a>]<br></li>
<li>EnvZ ATPase activity , which is inde-c pendent of OmpR , was subtracted from the total ATPase activity when both proteins were present and the corrected values are indicated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19432797" target="_blank">19432797</a>]<br></li>
<li>Under these conditions , the anaeroresponsive ArcB / ArcA system , the osmoresponsive EnvZ/OmpR system and the response-regulator UvrY showed the most drastic changes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12366850" target="_blank">12366850</a>]<br></li>
<li>In this case , the process of signal transduction is accomplished through weH-documented phosphotransfer involving the sensory protein kinase and response-regulator ( EnvZ and OmpR ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1630316" target="_blank">1630316</a>]<br></li>
<li>This adaptive response is mediated by EnvZ ( sensory kinase ) and OmpR ( response-regulator ) ( for a review see Parkinson , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7957084" target="_blank">7957084</a>]<br></li>
<li>To determine whether the EnvZ-OmpR response was activated in CS448-3 , we removed either EnvZ , the sensor kinase , or OmpR , the response-regulator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23893115" target="_blank">23893115</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_615">
Phosphorylation alters the interaction of the response-regulator OmpR with its sensor kinase EnvZ .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15073297" target="_blank">15073297</a>]<br></label>
<input id="act_similarity_615", type="checkbox">
<div class="ACT_similar">
<li>Mattison , K. , and Kenney , L.J. ( 2002 ) Phosphorylation alters the interaction of the response-regulator OmpR with its sensor kinase EnvZ .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19432797" target="_blank">19432797</a>]<br></li></div>
<li>We further demonstrate that induction of the dnaK and ibp promoters does not depend upon EnvZ -- OmpR , s S or sE , but results from increased translation of s32 upon osmotic upshift .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10594827" target="_blank">10594827</a>]<br></li>
<li>We propose that the effects of mdoA and galU mutations are mediated by OmpR , although the mechanism by which MDO molecules interact with EnvZ is still unclear .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7642497" target="_blank">7642497</a>]<br></li>
<li>Although plasmid pAT429 should produce the EnvZ protein in addition to the OmpR protein , no band corresponding to the EnvZ protein was detected ( lane b ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3294816" target="_blank">3294816</a>]<br></li>
<li>OmpR is the regulator and is phosphorylated by the sensor protein EnvZ .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10698756" target="_blank">10698756</a>]<br></li>
<li>This protein could be replacing OmpR when it is missing and not be dephosphorylated by EnvZ .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25667010" target="_blank">25667010</a>]<br></li>
<li>In conclusion , we have isolated a mutant form of OmpR that is active independent of EnvZ and have used this mutant protein to examine the effects of different levels of active OmpR in the cell .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9422609" target="_blank">9422609</a>]<br></li>
<li>In this study , we directly tested this model by generating a mutant OmpR protein that is active in the absence of the EnvZ kinase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9422609" target="_blank">9422609</a>]<br></li>
<li>In contrast to the case of the OmpR protein , however , little is known about the functional role of the EnvZ protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3536870" target="_blank">3536870</a>]<br></li>
<li>In a modification of this model , the EnvZ protein regulates OmpR synthesis ( 17 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6317653" target="_blank">6317653</a>]<br></li>
<li>Thus , the OmpF - , OmpC + phenotype conferred by enwZ473 working through OmpR + is dominant to the otherwise EnvZ-independent OmpR-LacZ fusion protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2557454" target="_blank">2557454</a>]<br></li>
<li>same as Taking these results together , it is likely that the ompR3-ompR77 region is responsible for interaction of the OmpR protein with EnvZ , and lack of proper interaction would result in the OmpF - OmpCc phenotype regardless of whether the mutation site is on the OmpR or EnvZ protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3536870" target="_blank">3536870</a>]<br></li>
<li>Depending on the respective activation of the EnvZ/OmpR and Cpx pathway , a certain competition could occur and would moderately disturb OmpR-dependent csgD activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15743952" target="_blank">15743952</a>]<br></li>
<li>Therefore , the activation of the omr-lacZ fusions by pYqjA-YqjB , pYgdD-GcvA or pFadR requires EnvZ-OmpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16359331" target="_blank">16359331</a>]<br></li>
<li>phoR creC mutants that also lacked the RR OmpR , while retaining its cognate HK EnvZ , displayed 50 to 200 times more BAP activity on glucose and 2 to 3 times more activity on pyruvate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15755952" target="_blank">15755952</a>]<br></li>
<li>After addition of OmpR , the phosphoryl group from EnvZ ~ P is transferred to OmpR and the mass of the labelled band shifts ( lanes 2 -- 4 and 6 -- 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19432797" target="_blank">19432797</a>]<br></li>
<li>In the presence of wildtype OmpR , EnvZ-P transfers the phosphoryl group to OmpR , as evidenced by a labeled OmpR-P band ( lanes 2-3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11812125" target="_blank">11812125</a>]<br></li>
<li>Following autophosphorylation , EnvZ transfers its high-energy phosphoryl group to its cognate response-regulator , OmpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15175316" target="_blank">15175316</a>]<br></li>
<li>In response to environmental signals , EnvZ autophosphorylates a histidyl residue in the cytoplasmic domain and transfers this phosphoryl group to an aspartyl residue on OmpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8002608" target="_blank">8002608</a>]<br></li>
<li>Recently , we demonstrated that the ratio of the kinase to the phosphatase activity of EnvZ regulates OmpR function [ 15-171 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8522138" target="_blank">8522138</a>]<br></li>
<li>These experiments support the hypothesis that the stimulatory effect of the EnvZ kinase is due to its ability to phosphorylate OmpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9096305" target="_blank">9096305</a>]<br></li>
<li>For X4 and X3 , it is possible that premature termination of OmpR translation creates severe polarity within the operon , resulting in little or no EnvZ synthesis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2997120" target="_blank">2997120</a>]<br></li>
<li>Provided that thewild-type EnvZ * was added to the reaction mixture , ATP-dependent OmpR dephosphorylation wasclearlyobserved .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2668281" target="_blank">2668281</a>]<br></li>
<li>This required that the entericidin locus could be activated by sS and could not be repressed by EnvZ/OmpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9677290" target="_blank">9677290</a>]<br></li>
<li>In all cases , EnvZ-dependent activation of PhoB required the absence of OmpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15755952" target="_blank">15755952</a>]<br></li>
<li>In part , the EnvZ / OmpR pathway is activated by its modulator , MzrA , whose synthesis is controlled by both E and CpxRA ( 9 , 11 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22544271" target="_blank">22544271</a>]<br></li>
<li>In marked contrast to the case of EnvZ * , however , OmpR dephosphorylation was not significantly stimulated by the addition of ATP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2668281" target="_blank">2668281</a>]<br></li>
<li>In such pairs , exemplified by EnvZ and OmpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1943696" target="_blank">1943696</a>]<br></li>
<li>In turn , the phosphate moiety of EnvZ can be transferred - to a carboxyl side chain of OmpR ( 1 , 5 , 18 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2160945" target="_blank">2160945</a>]<br></li>
<li>That is , under anaerobic-growth conditions , both ArcB and EnvZ appear to be involved in phosphotransfer to OmpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12366850" target="_blank">12366850</a>]<br></li>
<li>Thus , MzrA appears to alter the output of the EnvZ/OmpR system but not its ability to receive and respond to various environmental signals .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19432797" target="_blank">19432797</a>]<br></li>
<li>Thus , YqjB was renamed MzrA , for modulator of EnvZ and OmpR A and will be henceforth referred to by this designation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19432797" target="_blank">19432797</a>]<br></li>
<li>Thus , the actions of MzrA appear to be speciﬁc to the EnvZ/OmpR regulon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19432797" target="_blank">19432797</a>]<br></li>
<li>Thus , MzrA interacts with EnvZ , but not with OmpR or CpxA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19432797" target="_blank">19432797</a>]<br></li>
<li>EnvZ controls-the-activity of OmpR , the transcriptional-regulator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8522138" target="_blank">8522138</a>]<br></li>
<li>* EnvZ is thought to modulate the functional activity of OmpR ( 7 , 14 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2160945" target="_blank">2160945</a>]<br></li>
<li>When EnvZ , ATP and OmpR are mixed together , the following reactions occur : EnvZ ATP !	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11812125" target="_blank">11812125</a>]<br></li>
<li>I A EnvZ * + OmpR + T P R 0 ¬ - 0 EnvZ EnvZ 1 OmpR ¬ - OmpR -0 - A '' + ATP ¬ - Pi Pi ¬ I 0 ' 0.5 ' 1.5 ' 3 ' 0 .5 ' 1 .5 ' 0 3 ¬ ¬ ¬ C EnvG + OmpR D ~ ~ ; + 0 n y d 3 OmpR ¬ -0 mpR3 7 + ATP 0 ' 0.5 ' 1.5 ' 3 I 0 0.5 ' 1.5 ' 3 ' B EnvZl ; + OmpR ?? A EnvZ * + ompRn * OvmpR ?? - EnvZl1 EnvZ * .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2668281" target="_blank">2668281</a>]<br></li>
<li>I OmpR EnvZ OmpR-P OmpR Wm3 Pi FIG. 6 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2668281" target="_blank">2668281</a>]<br></li>
<li>In the tol mutants , activation of csgDEFG occurred via OmpR and was dominant upon repression by RcsB and RcsA , while these two regulatory proteins repressed csgBA through a dominant effect on the activator protein CsgD , thus affecting curli synthesis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16000739" target="_blank">16000739</a>]<br></li>
<li>Possibly , signals that act on the promoter of rcsC and , via OmpR , on rcsD provide an additional boost to the activation of RcsB-dependent targets under speciﬁc conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16153174" target="_blank">16153174</a>]<br></li>
<li>Under such conditions , the activation of csgDEFG probably occurs via OmpR , which is dominant upon repression by RcsAB , while RcsB and RcsA could repress csgAB through a dominant negative effect on CsgD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16000739" target="_blank">16000739</a>]<br></li>
<li>Insertion of the C/A-rich element resulted in an approximately twofold increase in OmpR GFP synthesis in the in vitro translation assay ( Fig. 7E ; cf. lanes 3 and 9 ) , confirming the stimulatory effect of this element on mRNA translation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17974919" target="_blank">17974919</a>]<br></li>
<li>Quantification of the data by scanning densitometry showed that IHF inhibition was lowered from 72 % to 27 % by increased amounts of OmpR ( Fig. 5b ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1736095" target="_blank">1736095</a>]<br></li>
<li>The data in Fig. 5a show that increasing the amount of partially purified OmpR in the in vitro transcription assay from 100 ng to 150 ng strongly reduced inhibition by IHF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1736095" target="_blank">1736095</a>]<br></li>
<li>free Consistent with this idea are the results showing that IHF inhibition of transcription can be strongly reduced by increasing the levels of partially purified OmpR normally added to the in vitro assay ( Fig. 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1736095" target="_blank">1736095</a>]<br></li>
<li>Fig. 3A shows that as the amount of OmpR in the reaction mixture was increased , a band corresponding to the 310-bp XbaI-BamHI fragment containing the ompFpromoter ( indicated by an arrow in Fig. 3A ) disappeared .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3023382" target="_blank">3023382</a>]<br></li>
<li>Whether the increase in the total amount of wild-type OmpR also results in an increase in the amount of active OmpR ( i.e. , OmpR-P ) in the cell can not be detera A 1,050-bp EcoRI-HindIII PCR-generated DNA fragment containing the ompR1 sequence from position 2127 to 1923 from the start point of ompR transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9422609" target="_blank">9422609</a>]<br></li>
<li>Recently , it was demonstrated that microcin C51 production depends on OmpR ( 24 ) and that the promoter of the fatty-acid receptor gene , fadL , is negatively regulated at high osmolarity by interacting with phospho-OmpR ( 15 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7642497" target="_blank">7642497</a>]<br></li>
<li>In this report we present data indicating that ( 1 ) uptake of long-chain fatty acids in E. coli and gene expression of fadL are both regulated by environmental osmolarity , ( 2 ) transcription of thefadL gene in vitro is repressed by phosphorylated OmpR , and ( 3 ) phosphorylated OmpR specifically binds to the promoter region of fadL Materials and methods Bacterial strains , media , and growth conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8413182" target="_blank">8413182</a>]<br></li>
<li>From these results , we propose that oleic acid uptake in E. coli is osmotically regulated , at least when thefadR control is derepressed , through transcription of fadL , which is repressed by the phosphorylated form of OmpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8413182" target="_blank">8413182</a>]<br></li>
<li>I has been shown that the amount of the phosphorylated form of OmpR increases when the cells are grown at high osmolarity ( Forst et al. 1990 ; Russo and Silhavy 1991 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8413182" target="_blank">8413182</a>]<br></li>
<li>The OmpR in these strains does not respond to osmolarity and is believed to be locked into the high osmolarity form that normally negatively regulates ornpF ( Slauch and Silhávy 1989 ; Tsung et al. 1989 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1736095" target="_blank">1736095</a>]<br></li>
<li>Under our conditions , band-shift assays indicate that OmpR binds to a unique site ( Fig. 1A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15743952" target="_blank">15743952</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_670">
Under high-osmolarity conditions , OmpR binds to the region between -56 and -41 and to the region between -384 and -351 ( Fig. 6B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8113170" target="_blank">8113170</a>]<br></label>
<input id="act_similarity_670", type="checkbox">
<div class="ACT_similar">
<li>Under high-osmolarity conditions ( B ) , OmpR binds to region between -384 and -351 and the region between -56 and - 41 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8113170" target="_blank">8113170</a>]<br></li></div>
<li>Furthermore , since the experimental conditions for growth and the swarm assay are kept at low osmolarity , the level of phospho-OmpR in pta mutants would be almost negligible .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7642497" target="_blank">7642497</a>]<br></li>
<li>The OmpR pathway further activates this Sdependent transcription at low osmolarity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14978043" target="_blank">14978043</a>]<br></li>
<li>The model suggests that the low-osmolarity form of OmpR binds to the positively acting site ( -64 to -100 ) in the ompF promoter and activates transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1648075" target="_blank">1648075</a>]<br></li>
<li>The ompF gene , with its high af-ﬁnity OmpR-binding sites , is transcribed even when the phosphorylated OmpR is scarce ( i.e. , under low-osmolarity conditions ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14665678" target="_blank">14665678</a>]<br></li>
<li>The overexpressed wild-type OmpR confers osmoresponsiveness under these high and low osmolarity conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11812125" target="_blank">11812125</a>]<br></li>
<li>In medium of low to moderate osmolarity , csgD is activated by OmpR , which results in high curli production by swimming cells ( G. Jubelin and P. Stewart , unpublished observations ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15743952" target="_blank">15743952</a>]<br></li>
<li>In addition , in experiments where the osmolarity of the medium was changed , we detected no change in transcription levels of mcbAp , indicating that , unlike other promoters regulated by OmpR , mcbAp does not display osmoregulation ( results not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1906064" target="_blank">1906064</a>]<br></li>
<li>However , because of the pleiotropic phenotypes , it becomes questionable whether the form of OmpR created in an envZ473 background is similar to or different from the form of OmpR created by EnvZ + in high osmolarity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2557454" target="_blank">2557454</a>]<br></li>
<li>Because our results argue for the existence of at least three forms of OmpR , and because both the low osmolarity form and the high osmolarity form require input from EnvZ , it seems likely that the null state of OmpR ( Fig. 7 , triangle ) is the unphosphorylated form .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2557454" target="_blank">2557454</a>]<br></li>
<li>If this is correct , and if phosphorylation is the sole mechanism by which EnvZ creates both the low osmolarity form ( Fig. 7 , square ) and the high osmolarity form ( Fig. 7 , circle ) , then the most likely explanation for the molecular difference between these forms is a difference in the level of OmpR phosphorylation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2557454" target="_blank">2557454</a>]<br></li>
<li>+ strain , the mutant OmpR protein exists primarily in the high osmolarity form .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2557454" target="_blank">2557454</a>]<br></li>
<li>According to this model , the key to osmoregulation is the dominance of the high osmolarity form of OmpR to the low osmolarity form .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2557454" target="_blank">2557454</a>]<br></li>
<li>This is particularly true for the pleiotropic envZ alleles , which create a form of OmpR that resembles , but is not identical to , the form of OmpR seen under conditions of high osmolarity ( 39 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1648075" target="_blank">1648075</a>]<br></li>
<li>In order to assess the role of OmpR phosphorylation in motility , we measured the expression of ﬂhDC9-lac under different osmolarity conditions ( Fig. 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7642497" target="_blank">7642497</a>]<br></li>
<li>High osmolarity results in a high degree of phosphorylation of OmpR , which subsequently acts as a repressor of ompFexpression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7651128" target="_blank">7651128</a>]<br></li>
<li>High osmolarity promotes auto-phosphorylation , leading to an increase in OmpR phosphorylation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8098993" target="_blank">8098993</a>]<br></li>
<li>This implies that the repression of ﬂagellar synthesis at high osmolarity is due to the phosphorylation of OmpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7642497" target="_blank">7642497</a>]<br></li>
<li>Changes in osmolarity as well as envZ11 mutations affect ﬂagellar synthesis ( Fig. 3 ) , suggesting that the effects of high concentrations of salts on the synthesis of ﬂagella ( 42 ) involve the phosphorylation of OmpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7642497" target="_blank">7642497</a>]<br></li>
<li>When the osmolarity of the medium is high , levels of phosphorylated OmpR are increased , ompC expres ¬ * Corresponding author .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8002608" target="_blank">8002608</a>]<br></li>
<li>Since the amount of OmpR protein immunoprecipitated from the different extracts was similar ( Fig. 1B , compare lanes 2 and 4 ) , these results indicate that the phosphorylation of OmpR was indeed elevated in cells grown in high-osmolarity medium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2160945" target="_blank">2160945</a>]<br></li>
<li>Analysis of the data shown in Fig. 1 by laser scanning of the radiolabeled and immunostained bands indicated that the level of phosphorylation of OmpR was elevated approximately fourfold in cells grown in the high-osmolarity medium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2160945" target="_blank">2160945</a>]<br></li>
<li>OmpR is phosphorylated in the envZ22 ( Am ) strain minimal medium ( Fig. 4A , lane grown in 3 ) and the level of phosphorylation of OmpR is enhanced by growth in the high-osmolarity medium ( Fig. 4A , lane 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2160945" target="_blank">2160945</a>]<br></li>
<li>We tend to favor this explanation , because wildtype OmpR also functions better when cells are grown in medium of high osmolarity ( Fig. 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2557454" target="_blank">2557454</a>]<br></li>
<li>For example , the intracellular levels of OmpR increased in MC4100 grown in the presence of sucrose ( Fig. 1 , lanes 5 and 6 ) , whereas there was no apparent change in the OmpR levels in MH1461 ( lanes 7 and 8 ) and SG477 ( lanes 9 and 10 ) when the cells were grown in the high-osmolarity medium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2656631" target="_blank">2656631</a>]<br></li>
<li>Searches for bacterial regulatory factors controlling such systems in S. enterica sero-var Typhimurium have revealed the RpoS , Fur , PhoP , and OmpR proteins , which are responsible for the regulation of different sets of acid shock proteins ( ASPs ) ( 5 , 6 , 7 , 18 , 19 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12670971" target="_blank">12670971</a>]<br></li>
<li>Under anaerobic-growth conditions , however , the anaero-responsive ArcB also participates in porin gene regulation in a manner requiring OmpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12366850" target="_blank">12366850</a>]<br></li>
<li>Examples of disruption of transcriptional repression in E. coli include ompC , where an IS1 element insertion perturbed regulation by OmpR ( 39 ) ; sodA , where an IS5 element was suggested to abolish repression by Fur ( 8 ) , and fnr , where the autoregulation was postulated to be disrupted by IS5 element insertion ( 7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15516564" target="_blank">15516564</a>]<br></li>
<li>Transcription factors including NusG , OmpR , and RcsB , which are generally not considered to be under redox control , were also associated with thioredoxin , providing compelling evidence for an extensively coupled network of redox regulation of E. coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16760308" target="_blank">16760308</a>]<br></li>
<li>We have investigated the possibility that the transcriptional regulation of the csg genes by tol -- pal occurs through OmpR , CpxR and/or RcsB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16000739" target="_blank">16000739</a>]<br></li>
<li>OmpR is also a transcriptional-activator of two sRNAs , OmrA and OmrB , involved in the posttranscriptional regulation of several OMPs ( 22 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17416652" target="_blank">17416652</a>]<br></li>
<li>This modiﬁcation of the regulatory response , which is expected to favor the survival of E. coli in the intestinal tract , occurs through the cross talk activation of OmpR by the ArcB sensor , which senses the anaerobic condition .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14665678" target="_blank">14665678</a>]<br></li>
<li>Further analysis of the yﬁD promoter revealed the absence of consensus sites for regulators such as PhoP and OmpR that are known to be involved in the response of E. coli to acid .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11932447" target="_blank">11932447</a>]<br></li>
<li>Sterospeciﬁc positioning of the cis-acting sequence with respect to the canonical promoter is required for activation of the ompC gene by positive regulator , OmpR , in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7836315" target="_blank">7836315</a>]<br></li>
<li>OmpR regulates The stationary-phase acid-tolerance response of Salmonella enterica serovar Typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12670971" target="_blank">12670971</a>]<br></li>
<li>OmpR regulates the sta-tionary-phase acid tolerance response of Salmonella enterica serovar Typhi-murium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15755952" target="_blank">15755952</a>]<br></li>
<li>OmpR regulates the stationary-phase acid resistance response of Salmonella enterica serovar Typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17185552" target="_blank">17185552</a>]<br></li>
<li>Recently , the OmpR response-regulator protein was reported to be required ( as a ﬁrst ASP ) for efﬁcient induction of the stationary-phase ATR of S. enterica serovar Typhimurium ( 5 , 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12670971" target="_blank">12670971</a>]<br></li>
<li>In Salmonella enterica serovar Typhimurium , the response-regulator OmpR , independently of its cognate kinase EnvZ , is required for the stationary-phase acid tolerance response .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17185552" target="_blank">17185552</a>]<br></li>
<li>OmpR is a cytosolic response-regulator , which binds to the promoter region of the porin genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19432797" target="_blank">19432797</a>]<br></li>
<li>25 The response-regulator OmpR-P represses biofilm production by directly interacting with the pga operon promoter region 20 15 To evaluate whether the effect of OmpR was direct on the pga operon , an EMSA was set up ( Fig. 6b ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25667010" target="_blank">25667010</a>]<br></li>
<li>OmpR acts as a transcriptional-regulator of OMPs , most notably OmpC and OmpF , in response to changes in osmolarity of the envi ronment ( 41 , 64 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17416652" target="_blank">17416652</a>]<br></li>
<li>The existence of envZ mutations that prevent conversion of OmpR among the various forms in response to osmolarity is predicted by our model .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2557454" target="_blank">2557454</a>]<br></li>
<li>The important question that remains is how the appropriate OmpR conformation is generated in response to high osmolarity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11812125" target="_blank">11812125</a>]<br></li>
<li>-- OmpR TCS in response to differences in osmolarity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17064377" target="_blank">17064377</a>]<br></li>
<li>Well-studied examples are found among members of the AraC , LysR , and LuxR families , as well as in the OmpR and NarL families of phosphorylation-dependent response-regulator proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10913075" target="_blank">10913075</a>]<br></li>
<li>The hybrid sensor BarA ( also called AirS ) was ﬁrst identi-ﬁed by its ability to activate the OmpR response-regulator ( 28 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12533459" target="_blank">12533459</a>]<br></li>
<li>The cross-talk between BarA and OmpR , however , could not be demonstrated in vitro , and the cognate response-regulator for BarA was not identified ( 11 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11022030" target="_blank">11022030</a>]<br></li>
<li>Rhee , J.E. , Sheng , W. , Morgan , L.K. , Nolet , R. , Liao , X. , and Kenney , L.J. ( 2008 ) Amino-acids important for DNA recognition by the response-regulator OmpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19432797" target="_blank">19432797</a>]<br></li>
<li>Whether cross-talk or another interaction between RcsC , OmpR ( and perhaps with another unknown response-regulator ) or NhaR occurs should be investigated further .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25667010" target="_blank">25667010</a>]<br></li>
<li>This protein , UspA ( universai stress protein A ) , is thus a generai , non-specific responder to growth arrest ( this paper ; Nystrom and Neidhardt , 1992 ) , and its induction is independent of RpoS as weli as of aii other giobal reguiators so far tested , inciuding PhoB , ReiA , SpoT , AppY , OmpR , H-ns , Lrp , and RpoH ( Nystrom and Neidhardt , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8152377" target="_blank">8152377</a>]<br></li>
<li>Figueroa-Bossi , N. , Schwartz , A. , Guillemardet , B. , D'Heyge ̀re , F. , ACKNOWLEDGEMENTS We thank undergraduate summer student Estefanıa Morales Ruiz from the Programa del Verano de la Investigación Cientıfica , Academia Mexicana de Ciencias A.C. for her assistance with some experiments ; Marcos Fernández for his advice in the footprinting experiments and for purifying the OmpR protein ; and all members of our laboratory for their critical input .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25667010" target="_blank">25667010</a>]<br></li>
<li>89 Residues 89-95 in the OmpR protein , - rg Val-Asp - `` A - Ile-Val-95 G - '' ly-Leu , were foundto be similar in either charge or residue when compared with residues 24-30 in the P22 repressor , 24 - Lys-Met-Leu-Gly - '' '' - Val-Gly-Val ( single underline denotes resi - '' due having similar chemical property and double underline denotes matching residue ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6292199" target="_blank">6292199</a>]<br></li>
<li>We found that the partially pure OmpR from both the parent and envZl1 strains could no longer bind to fragment Ila even at the levels of approximately 1 , ug of OmpR , in contrast to the much lower levels of OmpR required when the protein was present in the soluble extracts .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2656631" target="_blank">2656631</a>]<br></li>
<li>Since EnvZ473 must still communicate with OmpR2 , we suggest that the form of OmpR that turns off ompF is closely related to the form of the protein that causes the various pleiotropic effects .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2557454" target="_blank">2557454</a>]<br></li>
<li>A possiblesimple interpretation is that the phosphorylated and nonphosphorylated OmpR protein might represent two alternative structures that respectively activate the ompC and ompF genes , e.g. the phosphorylated form of the OmpR protein preferentially binds to theompC promoter , whereas the nonphosphorylated form of the OmpR protein preferentially binds to the ompF promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2668281" target="_blank">2668281</a>]<br></li>
<li>Provided that these in vitro observations reflect the in vivo situation , the results presented in this study suggest that the accumulation of the phosphorylated form of the OmpR protein in cells results in preferential activation of the ompC gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2668281" target="_blank">2668281</a>]<br></li>
<li>Therefore , more detail studies are required to elucidate the molecular mechanism underlying the activation of the ompF and ompC genes by the OmpR protein , although it is most likely that the accumulation of the phosphorylated OmpR protein in cells results in preferential activation of the ompC gene , as discussed above .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2668281" target="_blank">2668281</a>]<br></li>
<li>Furthermore , comparison of the electrophoretic mobility of P-UhpA in native gels of various polyacrylamide concentrations , in comparison to the migration of protein standards , indicated that P-UhpA was monomeric in solution under those conditions .3 The rates of phosphorylation and dephosphorylation of UhpA were comparable to those seen for OmpR and PhoB ( 13 , 34 ) and were much slower than the corresponding processes in CheY and NtrC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8999880" target="_blank">8999880</a>]<br></li>
<li>The positions of standard proteins are indicated by a-e ; a , phosphorylase B ( Mr , 92.5 kDa ) ; b , ovalbumin ( Mr , 45 kDa ) ; c , carbonic anhydrase ( Mr ,31 kDa ) ; d , soybean trypsin inhibitor ( Mr , 21.5 kDa ) ; e , lysozyme ( M , , 14.4 kDa ) RESULTS Purification of OmpR-Plasmid pSN142 can overproduce OmpR protein upon induction with isopropyl P-D-thiogalac-topyranoside ( Fig. 2 lane 2 ) and was used for purification of OmpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3023382" target="_blank">3023382</a>]<br></li>
<li>The proteins isolated from these mutants were found to be defective in seemingly the same aspect , i.e. OmpR dephosphorylation.SinceEnvZ11 * can not effectively mediate OmpR dephosphorylation and OmpR3 hardly undergoes dephosphorylation , in both cases the phosphorylated form of the OmpR protein is consequently accumulated under the in vitro experimental conditions used ( see Fig. 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2668281" target="_blank">2668281</a>]<br></li>
<li>Whether or not these sites are involved in the dominant negative effect of certain mutations in envZ ( 105 , 286 , 289 ) , resulting in an overphosphorylated OmpR protein ( 2 , 224 , 291 ) , is still unclear .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9529892" target="_blank">9529892</a>]<br></li>
<li>The first approximately 200 residues of KdpE and three other effector-proteins , PhoB ( 28 ) , CreB ( formerly PhoM-orf2 , the hypothetical PhoB protein of PhoM/Pho2 ) ( 2 , 48 ) , and OmpR ( 8 ) of E. coli are aligned with the TREE program ( 13 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1532388" target="_blank">1532388</a>]<br></li>
<li>When the MalT protein was compared with the HtpR , RpoD , OmpR , CAP or Fnr proteins of E. cold , together with the N , CI and CII proteins of bacteriophage i , no significant homologies were detected .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3015733" target="_blank">3015733</a>]<br></li>
<li>( d ) Homology among the PhoB , OmpR and Dye proteins We searched for proteins with homology to the PhoB protein , using the NBRF protein sequence data base .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3537313" target="_blank">3537313</a>]<br></li>
<li>The ratio of the phosphorylated form of OmpR and CpxR in the binding reactions was determined by using bidimensional electrophoresis combined with immunodetection : 30 % of each protein was phosphorylated under the conditions described here ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15743952" target="_blank">15743952</a>]<br></li>
<li>Abbreviations : OmpR-P , phosphorylated form of OmpR ; MBP , maltose binding protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9096305" target="_blank">9096305</a>]<br></li>
<li>Therefore , the accumulation of the phosphorylated form of the OmpR protein , which otherwise would be caused by EnvZ11 * , isseemingly prevented in the phosphotransfer reactions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2668281" target="_blank">2668281</a>]<br></li>
<li>In addition , the purified OmpR protein is not phosphorylated , since the theoretical and measured masses are virtually identical ( Table 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7568033" target="_blank">7568033</a>]<br></li>
<li>When the purified phos-pho-ArcBc was incubated with purified OmpR , this protein was phosphorylated ( Figure SC , lane 9 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7957084" target="_blank">7957084</a>]<br></li>
<li>Its mass differs from the phosphorylated protein by about 99 Da and from OmpR by about 16 Da .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7568033" target="_blank">7568033</a>]<br></li>
<li>For example , ArcB can efﬁciently phosphorylate the chemotaxis protein CheY and the osmoregulator OmpR in addition to its cognate effector ArcA ( 32 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12107134" target="_blank">12107134</a>]<br></li>
<li>For this purpose , the OmpR protein was labeled with [ 3H ] arginine , purified from minicells as described under `` Experimental Procedures , '' and subjected to sequential Edman deg-radations .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6292199" target="_blank">6292199</a>]<br></li>
<li>OmpR protein was detected by using anti-OmpR serum and 1251-conjugated protein A. Increasing amounts ( three doses ) of purified OmpR and each of the individual extracts were used for the calculation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2656631" target="_blank">2656631</a>]<br></li>
<li>EnvZ473 working through OmpR + confers an envZ473 phenotype even in the presence of the OmpR2 protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2557454" target="_blank">2557454</a>]<br></li>
<li>Digestion was much slower compared to unphosphorylated OmpR , as evidenced by the presence of intact protein remaining even after 2 hr ( lane 10 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7568033" target="_blank">7568033</a>]<br></li>
<li>This observation implies a negative role of OmpR protein in regulating the ﬂhDC promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7642497" target="_blank">7642497</a>]<br></li>
<li>This band was identified as the OmpR protein as follows ; ( a ) it migrates at thesame position as the OmpR protein , labeled with [ 35S ] methionine , inSDS-polyacrylamide gel electrophoresis ( see Ref .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6292199" target="_blank">6292199</a>]<br></li>
<li>This supports the view that the OmpR protein is directly involved in osmoregulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3010044" target="_blank">3010044</a>]<br></li>
<li>This mutant was unable to phosphorylate or dephosphorylate its cognate regulator protein OmpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7730273" target="_blank">7730273</a>]<br></li>
<li>OmpR protein was present in the reactions at the followingconcentrations : lune 1 , O ng ; lane 2,670 ng ; lane 3,1.34 pg ; lane 4 , 2.68 pg .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3023382" target="_blank">3023382</a>]<br></li>
<li>Lane 2 shows the CpxP-AP fusion , MalE , and OmpR proteins from wholecell protein extracts .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9473036" target="_blank">9473036</a>]<br></li>
<li>Thus , by the above argument , the OmpR portion of this hybrid protein is locked into a given conformation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2557454" target="_blank">2557454</a>]<br></li>
<li>Purified OmpR protein was included in the reaction as specified .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3023382" target="_blank">3023382</a>]<br></li>
<li>Overproduction and puriﬁcation of OmpR-His6 protein , preparation of OmpR antibodies .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15743952" target="_blank">15743952</a>]<br></li>
<li>The levels of OmpR protein under low-copy-number conditions ( lanes 3 and 6 ) were close to those of the wild-type MC4100 strain , in which OmpR is present in a single copy ( lane 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9422609" target="_blank">9422609</a>]<br></li>
<li>The addition of the kinase to the reaction results in the phosphorylation of the OmpR protein ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9096305" target="_blank">9096305</a>]<br></li>
<li>The promoters regulated by OmpR and BvgA are activated in a graded manner in response to the level of the phosphorylated protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10913075" target="_blank">10913075</a>]<br></li>
<li>The region between 14 and 149 ﬁrst binds the OmpR protein , and then the upstream region from 2135 to 2179 is occupied with increasing OmpR concentrations .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7642497" target="_blank">7642497</a>]<br></li>
<li>The OmpR protein was partially purified from the indicated cells as described in Materials and Methods .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2656631" target="_blank">2656631</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2656631" target="_blank">2656631</a>]<br></li>
<li>The blot containing radiolabeled OmpR was subjected autoradiography , to and OmpR protein was subsequently detected by the conjugated alkaline phosphatase staining procedure .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2160945" target="_blank">2160945</a>]<br></li>
<li>The band most probably corresponds to the OmpR protein , as judged by its molecular mass .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3294816" target="_blank">3294816</a>]<br></li>
<li>The OmpR protein Rampersaud , A. , S. L. Harlocker , and M. Inouye .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8002608" target="_blank">8002608</a>]<br></li>
<li>In fact , it was demonstrated that this half of the OmpR protein itself retains thebinding ability as to theompF and ompC promoter DNAs .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2668281" target="_blank">2668281</a>]<br></li>
<li>In order to clarify whether the OmpR protein does indeed bind to the DNA via either one of these two regions , further biochemical studies will be necessitated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6292199" target="_blank">6292199</a>]<br></li>
<li>In order to establish whether CsgD alone is sufﬁcient for csgB activation , we overexpressed the CsgD protein in an OmpR strain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11717281" target="_blank">11717281</a>]<br></li>
<li>To conﬁrm that the increased copy number of the ompR gene resulted in an increase in the cellular level of the OmpR protein , we performed Western immunoblot analysis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9422609" target="_blank">9422609</a>]<br></li>
<li>In addition , increasing the copy number of the ompR gene led to increased levels of OmpR protein ( lanes 3 to 5 and 6 to 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9422609" target="_blank">9422609</a>]<br></li>
<li>In addition , it has been shown that MH760 remains OmpC - even in the presence of increased amounts of the altered OmpR protein ( 25 , 33 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2203749" target="_blank">2203749</a>]<br></li>
<li>The availability of nmpC-phoA fusions allowed us to demonstrate that nmpC expression is sensitive to catabolite repression and dependent on the CRP-cAfvlP complex , the OmpR protein during growth in alkaline medium ( Heyde et ai , 1991 ) , and several environmental conditions including pHo and osmolarity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8057841" target="_blank">8057841</a>]<br></li>
<li>Coyer and coworkers ( 10 ) have shown that thermoregulation of micF RNA levels is still seen in mutant strains lacking a functional OmpR protein , and they have also demonstrated that a shift from M-9 minimal medium to Luria broth ( LB ) results in increased expression of micF in both ompR and 1 ompR backgrounds .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8002608" target="_blank">8002608</a>]<br></li>
<li>Previous studies have shown that micF expression is increased when cells are shifted from a glucose minimal medium to LB and that increased micF transcription does not require a functional OmpR protein ( 10 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8002608" target="_blank">8002608</a>]<br></li>
<li>Second , the envZ473 allele , which produces a porin protein expression pattern essentially identical to that of envZII , requires the presence of a functional OmpR to express its phenotype ( 29 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2656631" target="_blank">2656631</a>]<br></li>
<li>nanC expression is also activated by the regulators cyclic AMP-catabolite activator protein , OmpR , and CpxR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15743943" target="_blank">15743943</a>]<br></li>
<li>For mcbAp , the OmpR protein has been shown to be necessary for maximal expression , although not for stationary-phase induction per se .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1906064" target="_blank">1906064</a>]<br></li>
<li>The signaling mechanisms that prevail in OmpR , MarA , and SoxS action are known .9,10,14 Activation of micF by Rob in response to cationic peptide antibiotics is not as well formulated .32 Transcriptional activators of micF Transcriptional regulators are known to activate micF expression in response to environmental factors ( Figure 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11601842" target="_blank">11601842</a>]<br></li>
<li>A CheYT87I mutant is non-chemotactic and exhibits only counterclockwise ¯ agellar rotation .24,38 The homologous FixJT82I mutant is seriously inhibited in its ability to activate transcription of a p-nifA-lacZ or a p - ® xK-lacZ fusion .39 An isoleucine substitution at this conserved position in other response-regulators is thus disruptive to signaling , just as we have shown for OmpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11812125" target="_blank">11812125</a>]<br></li>
<li>The concentration of OmpR in the assay was approximately 100 nM when 100 ng of partially punfied OmpR was used overlaps the -35 ( Tsui and Freundlich 1988 ; Griffo et al. 1989 ; Kur et al. 1989 ) or -10 ( Thompson and Mosig 1988 ; Tsui and Freundlich 1988 ) region or to sites that are upstream ( Huang et al. 1990 ) or downstream ( Krause and Higgin % 1986 ) from promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1736095" target="_blank">1736095</a>]<br></li>
<li>The PhoP sequence was aligned along the secondary structures of PhoB ( Sola et al. , 1999 ; Okamura et al. , 2000 ) and OmpR ( Martinez-Hackert and Stock , 1997a ; 1997b ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12123454" target="_blank">12123454</a>]<br></li>
<li>b-Galactosidase assay Ammonium hydrogen phosphoramidate ( PA ) was synthesized according to Sheridan et al. 51 The resulting crystalline product was dissolved to 0.5 M in water and immediately used to phosphorylate OmpR and T83I .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11812125" target="_blank">11812125</a>]<br></li>
<li>The two previously deﬁned Hfq-dependent sRNA regulators of CsgD translation , OmrA and OrmB , also repress the translation of their transcriptional-regulator OmpR , which in turn is essential for csgD transcription ( Guillier and Gottesman , 2006 ; 2008 ; Holmqvist et al. , 2010 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22250746" target="_blank">22250746</a>]<br></li>
<li>16 , 1 ¬ ompB asmoiaiity ) OmpR IHF IHF '60 -35 -10 • 170 QmpR CmpROmpR IHF IHF -360 -60 -170 The architectural roie of IHF in gene expression is not only limited to o - ̂ ** promoters but is probably also involved in the activation of the c ' ^ ^ - dependent promoter of the narGHJI operon ( Schroder et at .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7651128" target="_blank">7651128</a>]<br></li>
<li>The OmpR regulator is directly involved in activation of curli expression , in concert with several other regulatory proteins such as RpoS , H-NS , CpxR , and other yet unknown factors ( Romling et al. , 1998b ; Prigent-Combaret et al. , 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14523117" target="_blank">14523117</a>]<br></li>
<li>The activator effect of OmpR is absolutely necessary for csgD expression ( Prigent-Combaret et al. , 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16000739" target="_blank">16000739</a>]<br></li>
<li>The ompC gene , which was underexpressed in this condition ( Table 3 ) , has a complex regulation including protein regulators ( OmpR , CpxR , Lrp , IHF and EnvY ) and small regulatory RNAs ( micC , rseX and rybB ) that respond to different signals [ 42-46 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22513097" target="_blank">22513097</a>]<br></li>
<li>The phosphorylation of OmpR and some of its effects on gene regulation , either as an activator or as a repressor , have been evidenced by different groups ( Prüss , 1998 ; Shin & Park , 1995 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25667010" target="_blank">25667010</a>]<br></li>
<li>The helix-loop-he-lix-containing RRs of the OmpR family , OmpR and PhoB , bind in a direct or head-to-tail arrangement to recognition sites upstream or adjacent to the RNAP-binding region , where they interact with - CTD or - CTD , respectively ( 16 , 17 , 32 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11976297" target="_blank">11976297</a>]<br></li>
<li>The OmpR family member PhoB binds to direct repeat target sites in the promoter ( 32 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11976297" target="_blank">11976297</a>]<br></li>
<li>The highly phosphorylated form of OmpR binds not only to the region beiween -60 and - 1 0 0 , but alsc between - 4 0 and - 6 0 and further upstream , around position -- 360 ( Fig. 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7651128" target="_blank">7651128</a>]<br></li>
<li>The high-osmolarity form of OmpR also binds this region and , in addition , binds to the sites at -40 to -60 and at -350 to -380 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1648075" target="_blank">1648075</a>]<br></li>
<li>The region between - 56 and -41 is also protected by OmpR , but the affinity of OmpR for this region is much weaker than that for the -96 to -63 region ( 34 , 50 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8113170" target="_blank">8113170</a>]<br></li>
<li>( b ) OmpR-P binds with higher affinity than OmpR to the pgaA regulatory region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25667010" target="_blank">25667010</a>]<br></li>
<li>The fadL gene encoding the outer membrane long-chain fatty acid transporter is also repressed by OmpR [ 91 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10396600" target="_blank">10396600</a>]<br></li>
<li>The large amount of OmpR under the noninduced condition also implies the role of OmpR as a global regulator that might control some physiological events other than the expression of outer membrane porin .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7642497" target="_blank">7642497</a>]<br></li>
<li>The transcription of rprA is activated by the Rcs phosphorelay system OxyS ArcZ S σ RprA DsrA CsgD McaS GcvB OmpR OmrA OmrB Fig. 9 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22250746" target="_blank">22250746</a>]<br></li>
<li>The level of phosphorylated RRs was less than 1 % for CreB , HydG , YedW , NarL , CheB , OmpR , DcuR , CpxR , YehT , and NarP ( in decreasing order ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15522865" target="_blank">15522865</a>]<br></li>
<li>The addition of ATP to threeaction mixture resulted in dramatic acceleration of the dephosphorylationof OmpR ( panel B ) , whereas when ATP was not added , the phosphorylated OmpR wasrelatively stable ( panel A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2668281" target="_blank">2668281</a>]<br></li>
<li>The other two plasmids isolated in screen III ( Table 2 ) acted upstream of OmpR and therefore , we believe , act by signaling to OmpR , indirectly increasing Omp synthesis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17416652" target="_blank">17416652</a>]<br></li>
<li>The EnvY polypeptide is , like OmpR , dispensable under laboratory growth conditions , and the advantages of regulating OmpF and OmpC remain speculative .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6317653" target="_blank">6317653</a>]<br></li>
<li>The differences in purification of OmpR from MC4100 ( 15-fold ) and the envZJJ strain ( 135-fold ) reflect the presence of an ompR-bearing multicopy plasmid contained in MC4100 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2656631" target="_blank">2656631</a>]<br></li>
<li>The bound form of fragment I-Ilb-Ill produced by the partially pure OmpR from envZJJ was retarded much less than was the bound form produced by the OmpR from MC4100 ( e.g. , lane 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2656631" target="_blank">2656631</a>]<br></li>
<li>The phsphorylated OmpR is a transcriptional effector of both porin genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23274360" target="_blank">23274360</a>]<br></li>
<li>The levels of phosphorylation of OmpR may determine the extent of activation or repression of micF and ompC genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1695892" target="_blank">1695892</a>]<br></li>
<li>The mechan-ism of transcriptional activation of micF and ompC by OmpR as well as a role of IHF in this activation need to be better understood .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11601842" target="_blank">11601842</a>]<br></li>
<li>The role of RcsB and OmpR in csgDEFG activation was therefore investigated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16000739" target="_blank">16000739</a>]<br></li>
<li>The longest regulation path in the network is IHF → OmpR → FlhDC → FliA and further to seven operons ( marked yellow in Fig. 1B ) related to flagella motility .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15603590" target="_blank">15603590</a>]<br></li>
<li>The data show that OmpR phosphor-ylation is linear with increasing concentrations of OmpR ( Fig. 1B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7568033" target="_blank">7568033</a>]<br></li>
<li>The footprinting presence of OmpR did not give clear results as to the tran - was carried out asdescribed in Fig. 5 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3023382" target="_blank">3023382</a>]<br></li>
<li>The mobilities of nonspeciﬁc DNAs within the pT7/T3a19 vector were unaltered in the presence of different amounts of puriﬁed OmpR ( Fig. 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7642497" target="_blank">7642497</a>]<br></li>
<li>The cells were harvested and disrupted with aFrench press , and OmpR was purified by ultracentrifugation , ammonium sulfate precipitation , gel filtration , and DEAE ion-exchange chromatography .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3023382" target="_blank">3023382</a>]<br></li>
<li>The concentration of OmpR was determined by densitometric scanning of the autoradiographs from Western blot analysis using enhanced chemiluminescence ( ECL , Amersham ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7557323" target="_blank">7557323</a>]<br></li>
<li>The concentration of OmpR was determined by the BCA assay from Pierce and was compared to bovine serum albumin standards .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7568033" target="_blank">7568033</a>]<br></li>
<li>The effects GcvA were not due to the reduction of OmpR levels ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8522138" target="_blank">8522138</a>]<br></li>
<li>The same repression , due to the envZ11 mutation , which is known to activate OmpR constitutively , was observed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7642497" target="_blank">7642497</a>]<br></li>
<li>The labelled fragments were then incubated with nonphosphorylated OmpR ( A and B ) at the following amounts : lane 1 , none ; lane 2 , 44 ng ; lane 3 , 87.5 ng ; lane 4 , 175 ng ; lane 5 , 350 ng ; lane 6 , 700 ng ; lane 7 , 1400 ng ; lane 8 , 2800 ng .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7642497" target="_blank">7642497</a>]<br></li>
<li>The a-complexmigrated slower than the to ice , and approximately 125 ng of OmpR was added to the chilled mixture .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8175665" target="_blank">8175665</a>]<br></li>
<li>The windows are clearly visible in the lanes with 100 or 200 ng of OmpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7642497" target="_blank">7642497</a>]<br></li>
<li>The amounts of OmpR were as follows : lane 1 , none ; lane 2 , 50 ng ; lane 3 , 100 ng ; lane 4 , 200 ng ; lane 5 , 400 ng .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7642497" target="_blank">7642497</a>]<br></li>
<li>The samples analyzed were : lane 8 , phospho-ArcBC alone ; lane 9 , phospho-ArcBC and OmpR ; lane 10 , phospho-ArcBC and OmpR-D55Q .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7957084" target="_blank">7957084</a>]<br></li>
<li>The possibitity is discussed that the ncgative effects of IHF in this system are mediated through OmpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1736095" target="_blank">1736095</a>]<br></li>
<li>The micromolar concentrations of OmpR were as follows : lanes 2 , 0 ; lanes 3 , 38 ; lanes 4 , 19 ; lanes 5 , 9.5 ; lanes 6 , 4.75 ; lanes 7 , 2.38 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8113170" target="_blank">8113170</a>]<br></li>
<li>The exact molecular nature of the various forms of OmpR is unknown .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2557454" target="_blank">2557454</a>]<br></li>
<li>The activity of mcbp is known to be a ¡ ected positively by OmpR [ 9 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10802181" target="_blank">10802181</a>]<br></li>
<li>The role of the ﬂagella was assessed using a non-motile mutant ; the absence of roles for type 1 ﬁmbriae , curli , the OmpR system and Ag43 is inferred in part from lack of insertions in these loci in our transposon screen .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11555281" target="_blank">11555281</a>]<br></li>
<li>The positions of primers D4 and D5 , used for ampliﬁcation of the wild-type csgB promoter , are indicated RESULTS OmpR protein promotes curli production by activation of csgD promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11717281" target="_blank">11717281</a>]<br></li>
<li>The type of mutant that we are interested in will overcome the negative regulation of envZ473 but still require OmpR for promoter activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1648075" target="_blank">1648075</a>]<br></li>
<li>In thecase of ompC , the vicinity of promoters P2 and P3 was shown to be necessary for OmpR activation of ompC ( 14 ) ; our present results are also consistent with this .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3023382" target="_blank">3023382</a>]<br></li>
<li>In order to obtain these results , micromolar concentrations of OmpR were required , and it was necessary to perform the footprinting reactions at low KCl concentrations ( 20 mM ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8113170" target="_blank">8113170</a>]<br></li>
<li>For example , a substitution of Y102 for cysteine results in constitutive activation of OmpR ,44 suggesting that restriction of the movement of the conserved tyrosine residue contributes to the mutant phenotype of both CheY-T87I and OmpRT83I .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11812125" target="_blank">11812125</a>]<br></li>
<li>These results were consistent with the hypothesis of an increase in the active form of OmpR or OmpR234 in the Dtol mutant , leading to an overexpression of csgD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16000739" target="_blank">16000739</a>]<br></li>
<li>These results support the view that both the mutant proteins , EnvZ11 * and OmpR3 , are defective in seemingly the same aspect , i.e. OmpR dephosphorylation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2668281" target="_blank">2668281</a>]<br></li>
<li>Binding to the lower affinity site appeared to be cooperative , since interaction with this site was greatly enhanced by the presence of OmpR at the higher affinity site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7557323" target="_blank">7557323</a>]<br></li>
<li>This would suggest that the presence of OmpR bound at the flanking sites F1 and F3 is sufficient to stabilize the interaction of OmpR molecules with the mutated F2 site , and furthermore , that this stabilization only occurs under conditions where OmpR can be phosphorylated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9096305" target="_blank">9096305</a>]<br></li>
<li>However , under conditions where OmpR is phosphorylated , the F2 site is also occupied due to cooperative interactions between OmpR-P molecules bound at the F1 site and the weaker F2 site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9096305" target="_blank">9096305</a>]<br></li>
<li>Stimulation of OmpR Cooperative Binding by Phosphorylation -- Although both OmpR and OmpR-P exist as monomers in solution , two OmpR-P molecules cooperatively bind to the F1 site ( 11 , 12 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16618701" target="_blank">16618701</a>]<br></li>
<li>OmpR may cooperatively bind to this site and provide contact surfaces that are required for OmpR-OmpR interactions involved in the formation of the repression loop .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7557323" target="_blank">7557323</a>]<br></li>
<li>At present , it is not clear how many OmpR monomers bind to the different sites and whether or not OmpR bound at the positive site participates in the formation of the DNA loop .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8113170" target="_blank">8113170</a>]<br></li>
<li>Such loop formation has Alternatively , these bands may reflect discrete monomeric / been proposed for the regulation of other bacterial operons multimeric species of OmpR bound to the probe .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2553720" target="_blank">2553720</a>]<br></li>
<li>The OmpR molecules bound to these regions probably interact , thereby forming a repressive loop ( Slauch and Silhavy , 1991 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7651128" target="_blank">7651128</a>]<br></li>
<li>The interaction between OmpR molecules bound at these lower affinity sites is believed to induce the formation of a DNA loop which acts to repress ompF [ 9-111 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7557323" target="_blank">7557323</a>]<br></li>
<li>Loop formation is thought to result from the interaction between OmpR bound at the - 384 to -351 - bp and -56 to -41 - bp sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7557323" target="_blank">7557323</a>]<br></li>
<li>101,387-396 OmpR molecules bound at thisregion are stabilized by OmpR 7 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8175665" target="_blank">8175665</a>]<br></li>
<li>high affinity site then OmpR molecules bound to this region Chem .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8175665" target="_blank">8175665</a>]<br></li>
<li>Comparison of KdpEDBD to DBDs of OmpR ( 1OPC ) [ 9 ] , PhoB ( 1GXQ ) , and PhoB bound to DNA ( 1GXP ) [ 42 ] revealed similar overall structures with Ca root mean square deviations ( rmsd ) from 1.38 to 1.71 Å .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22291906" target="_blank">22291906</a>]<br></li>
<li>Importantly , the high affinity of OmpR for the F1bF1b fragment can be further enhanced by its phosphorylation because more than 90 % of the DNA is bound at 5 nM concentration of OmpR-P ( lane 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16618701" target="_blank">16618701</a>]<br></li>
<li>As expected , neither OmpR-P nor OmpR bound to the ler fragment , an unrelated DNA region used as a negative control .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25667010" target="_blank">25667010</a>]<br></li>
<li>Finally , three genes are involved in regulation of motility and bioﬁlm formation ( ydeH [ upregulation ] and ﬂhC OM IM IM P P Regulation ﬂhC mzrA ydeH Transcriptional regulator of ﬂagella Modulator of EnvZ/OmpR C E IM Diguanylate cyclase regulator of motility and bioﬁlm E C Translation rmf Ribosome modulation factor C raiA Ribosome-associated inhibitor a , upregulated transcription in the presence of NlpE overexpression ; , downregulation in the presence of NlpE overexpression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23564175" target="_blank">23564175</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_848">
Growth phase and OmpR regulation of transcription of the microcin B17 genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1906064" target="_blank">1906064</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2165476" target="_blank">2165476</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7836279" target="_blank">7836279</a>]<br></label>
<input id="act_similarity_848", type="checkbox">
<div class="ACT_similar">
<li>Growth phase and OmpR regulation of transcription of microcin B17 genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2537825" target="_blank">2537825</a>]<br></li></div>
				<label class="ACT_collapse_similarity" for="act_similarity_850">
[ 9 ] Hernandez-Chico , C. , San Millan , J.L. , Kolter , R. and Moreno , F. ( 1986 ) Growth phase and OmpR regulation of transcription of microcin B17 genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10802181" target="_blank">10802181</a>]<br></label>
<input id="act_similarity_850", type="checkbox">
<div class="ACT_similar">
<li>ndez-Chico , C. , San Millan , J.L. , Kolter , R. , and Moreno , F. ( 1986 ) Growth phase and OmpR regulation of transcription of MccB17 genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8709849" target="_blank">8709849</a>]<br></li></div>
<li>We found that high levels of CpxR-P failed to activate transcription of ompC in the absence of OmpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16077119" target="_blank">16077119</a>]<br></li>
<li>Even the levels of in vivo in the though transcription absence of functional OmpR are high ( Table 2 ) , it was possible that activity detectable in vitro might require this positive regulatory factor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1906064" target="_blank">1906064</a>]<br></li>
<li>In the past , it had not been possible to determine unambiguously whether OmpR played a role in stationary-phase induction or simply stimulated transcription during both logarithmic growth and stationary phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1906064" target="_blank">1906064</a>]<br></li>
<li>This result , together with our genetic analysis of tppB transcription , supports the idea that the phosphorylation of OmpR plays an important role in the transcriptional activation of tppB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15175316" target="_blank">15175316</a>]<br></li>
<li>This adds to the complexity of mechanism of transcription activation of these genes by OmpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11601842" target="_blank">11601842</a>]<br></li>
<li>This expression could be due to the activation of tppB transcription by the unphosphorylated form of OmpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15175316" target="_blank">15175316</a>]<br></li>
<li>For example , IHF has been reported to inhibit transcription when bound to a site that Crude OmpR ( ng ) zu CS-910 scanning densitometer .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1736095" target="_blank">1736095</a>]<br></li>
<li>Inhibition of ompF transcription by IHF in the presence of OmpR ( ng ) : 100 , 72 % , 125 , 51 % ; 150 , 27 % .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1736095" target="_blank">1736095</a>]<br></li>
<li>An alternative explanation for the observed repression of transcription by OmpR is that the enhancing e ¡ ect of UP element is interfered with OmpR which is bound between the promoter and the UP element .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10802181" target="_blank">10802181</a>]<br></li>
<li>We interpret this to mean that OmpR serves to stimulate transcription from this promoter but does not mediate stationary-phase induction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1906064" target="_blank">1906064</a>]<br></li>
<li>Since OmpR activates the transcription of omrA and omrB , these sRNAs should limit their own synthesis by directly downregulating their activator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18953042" target="_blank">18953042</a>]<br></li>
<li>( B ) Model depicting how acetyl P can inﬂuence diverse cellular processes through the RR OmpR and its ability to repress transcription of ﬂhDC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15755952" target="_blank">15755952</a>]<br></li>
<li>Potential interactions between OmpR and other factors known to regulate fadL transcription are another interesting subject for future studies .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8413182" target="_blank">8413182</a>]<br></li>
<li>Thus , QseB may be acting in a similar manner to OmpR to regulate the transcription of ﬂhDC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16135237" target="_blank">16135237</a>]<br></li>
<li>to many transcriptional-activators , including OmpR and NarL , appears to activate transcription of uhpT .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1479338" target="_blank">1479338</a>]<br></li>
<li>Wetherefore of a Maxam-Gilbert `` G + A '' cleavagereaction of the labeledpromoter attempted to test tahbeility of OmpR to activate transcription fragment .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3023382" target="_blank">3023382</a>]<br></li>
<li>Under the conditions of our transcription experiments , OmpR acted only as a repressor offadL .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8413182" target="_blank">8413182</a>]<br></li>
<li>17118 Interaction of OmpR efficient transcription to occur .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3023382" target="_blank">3023382</a>]<br></li>
<li>Concentrations of OmpR above 150 ng were inhibitory to transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1736095" target="_blank">1736095</a>]<br></li>
<li>In addition to the repression by YjjQ shown here , transcription initiation at the ﬂhDC promoter is known to be controlled by OmpR , CRP , RcsA-RcsB and RcsB , HfdR , DksA / ppGpp , LrhA , and MatA ( EcpR ) ( 37 , 39 , 41 -- 46 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26078445" target="_blank">26078445</a>]<br></li>
<li>The bolA1p promoter was transcribed by EcD as well as EcS ( Fig. 4A , B ) , but was repressed in the presence of phospho-OmpR while transcription of the tac promoter was una ¡ ected by the OmpR protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10802181" target="_blank">10802181</a>]<br></li>
<li>In vitro transcription from thefadL promoter was repressed by addition of OmpR to the reaction , if the OmpR was pre-incubated with E n v Z l l in the presence of ATP ( Fig. 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8413182" target="_blank">8413182</a>]<br></li>
<li>For this purpose , we employed the gel shift assay using phosphorylated and unphosphorylated forms of OmpR and 32P-labeled bolA1p promoter Fig. 1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10802181" target="_blank">10802181</a>]<br></li>
<li>Cells growing exponentially on an acetogenic carbon source possess elevated levels of acetyl P. Consequently , acetyl P acts a phosphoryl donor for OmpR , which then binds to and represses transcription from the ﬂhDC promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15755952" target="_blank">15755952</a>]<br></li>
<li>DNase I footprinting revealed the regions of the ﬂhDC promoter protected by OmpR in the presence or absence of phosphorylation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7642497" target="_blank">7642497</a>]<br></li>
<li>We are currently attempting to purify the different forms of OmpR to examine directly whether I H F influences how OmpR interacts with the ornpF promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1736095" target="_blank">1736095</a>]<br></li>
<li>Dual autoregulatory effects a OmpR is excluded because there is no known natural promoter repressed by it .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1943993" target="_blank">1943993</a>]<br></li>
<li>Interaction of OmpR with the ﬂhDC promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7642497" target="_blank">7642497</a>]<br></li>
<li>As in the case of the OmpF promoter ( 38 ) , it is likely that OmpR binds hierarchically to the ﬂhDC promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7642497" target="_blank">7642497</a>]<br></li>
<li>When a mutant OmpR lacking the presumed phospho-accepting site ( named OmpR-D55Q ; Kanamaru and Mizuno , 1992 ) was subjected to the same analysis , this particular mutant protein did not undergo phosphorylation in the presence of phospho-ArcBc ( Figure 5C , lane 10 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7957084" target="_blank">7957084</a>]<br></li>
<li>From an overexposed film , the level of OmpR phosphorylation in the absence of DNA was estimated to be approximately 0.12 % , 50-fold less than that in the presence of DNA ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11158569" target="_blank">11158569</a>]<br></li>
<li>In those studies , it was shown that the function of OmpR phosphorylation is to enhance ( approximately 10-fold [ 2 ] ) the afﬁnity of this protein to target DNA , without altering its speciﬁcity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7642497" target="_blank">7642497</a>]<br></li>
<li>ArcBc undergoes phosphorylation and functions as a phosphodonor for OmpR in vitro To verify the hypothetical view described above , we focused our attention on ArcB since this sensory kinase has been characterized intensively in terms of its physiological and catalytic functions ( luchi , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7957084" target="_blank">7957084</a>]<br></li>
<li>Thus , such factors as the stoichiometry of OmpR phosphorylation ( number of phosphate groups per OmpR molecule ) , the modulation of the actual sites of phosphorylation , and the existence of other covalent modifications may ultimately affect the functional activity of OmpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2160945" target="_blank">2160945</a>]<br></li>
<li>For the phosphorylation of OmpR , 10 mM acetyl-P ( Sigma Co. ) was added to the reaction mixture .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7642497" target="_blank">7642497</a>]<br></li>
<li>After the phosphorylation of OmpR with radioactively phosphorylated EnvZ11 * in the absence of ATP , nonlabeledEnvZ * was added to the reaction mixture together with ATP ( panel B ) or without ATP ( panel A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2668281" target="_blank">2668281</a>]<br></li>
<li>Whether this is a qualitative difference due to multiple sites of phosphorylation or simply a quantitative difference in the concentration of phosphorylated OmpR remains to be determined .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2557454" target="_blank">2557454</a>]<br></li>
<li>The fact that phosphorylation stimulates cooperativity leads us to propose a model to explain why phosphorylation is required for OmpR function in vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9096305" target="_blank">9096305</a>]<br></li>
<li>The in vivo phosphorylation approach was also used to show that OmpR can be phosphorylated in an envZ null strain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2160945" target="_blank">2160945</a>]<br></li>
<li>The mutant is phosphorylated in vitro , but non-chemotactic in vivo , indicating a block in activation subsequent to phosphorylation .24 We made the equivalent mutation in OmpR in order to characterize its properties in vivo and in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11812125" target="_blank">11812125</a>]<br></li>
<li>Recent experiments on in vivo phosphorylation of OmpR indicate that levels of OmpR-phosphate in the envZII strain are about fourfold higher than in MC4100 ( S. Forst , unpublished data ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1695892" target="_blank">1695892</a>]<br></li>
<li>Whether the higher levels of OmpR-phosphate in these cells were due to an actual increase in the concentration of OmpR-phosphate or whether they represent multiple phosphorylation events of a subpopulation of OmpR is not presently known .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2160945" target="_blank">2160945</a>]<br></li>
<li>This suggests that phosphorylation is not directly required for activating RNA polymerase , but perhaps simply to change the conformation of OmpR into the various forms .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2557454" target="_blank">2557454</a>]<br></li>
<li>This suggests that a conformational change results from phosphorylation of OmpR and that this change alters the of this sensitivity specific tryptic site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7568033" target="_blank">7568033</a>]<br></li>
<li>Puri cents cation and phosphorylation of OmpR As described previously [ 12 ] , OmpR was expressed and puri cents ed by using QIAexpress Ni-NTA Protein Puri cents cation System ( Qiagen ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10802181" target="_blank">10802181</a>]<br></li>
<li>To confirm that this stimulation is due to phosphorylation of OmpR , we repeated the above experiments using a mutant form of OmpR , OmpR D55Q .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9096305" target="_blank">9096305</a>]<br></li>
<li>Thus , both sets of experiments underscore the importance of the stimulatory effect of phosphorylation on cooperative interactions between OmpR molecules .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9096305" target="_blank">9096305</a>]<br></li>
<li>However , this approach does not permit one to analyze how pertinent environmental signals modulate the phosphorylation of OmpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2160945" target="_blank">2160945</a>]<br></li>
<li>It is known that such an increase normally enhances OmpR phosphorylation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7642497" target="_blank">7642497</a>]<br></li>
<li>OmpR phosphorylation ( Russo and Silhavy , 1991 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8098993" target="_blank">8098993</a>]<br></li>
<li>B. OmpR ~ P persists after phosphorylation by EnvZc R397L .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19432797" target="_blank">19432797</a>]<br></li>
<li>Our the functionof OmpR through phosphorylation ( 11-16 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8175665" target="_blank">8175665</a>]<br></li>
<li>Evidence , both in vivo and in vitro , indicates that this effect is mediated by the osmoregulator OmpR , involving phosphorylation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7642497" target="_blank">7642497</a>]<br></li>
<li>In vitro analysis of phosphorylation with ArcBC and OmpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7957084" target="_blank">7957084</a>]<br></li>
<li>In vivo phosphorylation of OmpR in MC4100 and the MH1461 ( envZIJ ) strain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2160945" target="_blank">2160945</a>]<br></li>
<li>In vivo phosphorylation of OmpR in E. coli parent strain MC4100 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2160945" target="_blank">2160945</a>]<br></li>
<li>In OmpF regulation , phosphorylated OmpR ( phospho-OmpR ) plays a role as an activator when its level is low and as a repressor when its level is high .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7642497" target="_blank">7642497</a>]<br></li>
<li>envZll produce very high levels of phosphorylated OmpR and Phosphorylation is believed to change the function of have an OmpF - , OmpC ' outer membraneporin pattern ( 12,18 , OmpRfrom an activator to a repressor molecule .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8175665" target="_blank">8175665</a>]<br></li>
<li>EnvZc11 mixture was M transferred to the tube containing different amounts of OmpR or PrS2-OmpR , and the reaction mixture was further incubated for 20 min to generate phosphorylated OmpR or PrS2-OmpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16618701" target="_blank">16618701</a>]<br></li>
<li>Under this condition , 100 % of OmpR or PrS2-OmpR was phosphorylated as judged from C4 reverse phase high pressure liquid chromatography .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16618701" target="_blank">16618701</a>]<br></li>
<li>This could be due to a low content of the phosphorylated form of OmpR ( Cai & Inouye , 2002 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16000739" target="_blank">16000739</a>]<br></li>
<li>For instance , the level of phosphorylated form is different between independently puriﬁed OmpR preparations ( Ishihama A. , unpublished data ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26010043" target="_blank">26010043</a>]<br></li>
<li>Abbreviations : OmpR-P , phosphorylated OmpR ; EnvZc-P , phosphorylated EnvZc ; GST , glutathione S-transferase ; OmpR , N-terminal half fragment of OmpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11158569" target="_blank">11158569</a>]<br></li>
<li>OmpR can rapidly be interconverted between active ( phosphorylated ) and inactive forms .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8002608" target="_blank">8002608</a>]<br></li>
<li>Mass spectrometric analysis of the phosphorylated species showed an increase in molecular mass of 81 Da , indicating the covalent attachment of a single phosphate group , as found for P-OmpR ( 44 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8999880" target="_blank">8999880</a>]<br></li>
<li>OmpR-P has a measured mass of 27,438 Da ( calculated , 27,434 Da ) , an increase of about 83 Da compared to OmpR ; thus , OmpR-P is phosphorylated at only one site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7568033" target="_blank">7568033</a>]<br></li>
<li>The mass spectrometry data indicate that OmpR is phosphorylated on only one residue , with an accompanying increase in mass of about 83 Da .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7568033" target="_blank">7568033</a>]<br></li>
<li>OmpR protein , whether added to the extract in partially purified form or expressed in the assay mixture on a second plasmid , pRT506 .2 ( 43 ) , can stimulate mcbAp activity up to threefold .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1906064" target="_blank">1906064</a>]<br></li>
<li>For partially purified samples , 4-36 ng of OmpR MC4100 , MH1471 , and MH760 , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8175665" target="_blank">8175665</a>]<br></li>
<li>One hundred nanograms of partially purified OmpR was added to the reactions shown in lanes 3-7 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1736095" target="_blank">1736095</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1736095" target="_blank">1736095</a>]<br></li>
<li>( C ) Immunodetection of OmpR in DNA complexes after transfer of EMSA gel proteins onto a membrane as described in Materials and Methods .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15743952" target="_blank">15743952</a>]<br></li>
<li>To analyze the phosphotransfer from EnvZc-P to OmpR , EnvZc was phosphorylated first with [ g-32P ] ATP and purified free of ATP as described in Materials and Methods .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11158569" target="_blank">11158569</a>]<br></li>
<li>Purification of Proteins -- EnvZc [ T247R ] previously termed EnvZc11 , 2 OmpR , and PrS - OmpR were purified as described previously ( 11 , 15 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16618701" target="_blank">16618701</a>]<br></li>
<li>As described in Materials and Methods , GST -- EnvZc11 phosphorylates more than 95 % OmpR under the condition used in the experiment .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11158569" target="_blank">11158569</a>]<br></li>
<li>To gain insight ' into the nature of envZ473 , we have asked whether OmpR2 is defective with respect , to the form of OmpR created by EnvZ473 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2557454" target="_blank">2557454</a>]<br></li>
<li>Therefore , although OmpR exists in solution as a monomer under most conditions that have been examined ( 5 , 14 , 15 ) , OmpR is unable to form stable proteinyDNA complexes as a monomer .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9096305" target="_blank">9096305</a>]<br></li>
<li>Phosphorylation of OmpR results in a marked increase in afﬁnity for all sites ( 10 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10913075" target="_blank">10913075</a>]<br></li>
<li>Phosphorylation changes the conformation of OmpR and stimulates the ability of OmpR to interact with speciﬁc sites upstream of its target genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15175316" target="_blank">15175316</a>]<br></li>
<li>Thus , the modest induction of bioﬁlm formation by NaCl in an nhaR mutant ( Fig. 1C ) may be due to the effect of OmpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16997959" target="_blank">16997959</a>]<br></li>
<li>As shown in Fig. 6 , OmpR appeared to-bind to theompC XbaI and the BamHI ends in separate experiments .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3023382" target="_blank">3023382</a>]<br></li>
<li>However , OmpR modestly activates bioﬁlm formation in these strains ( reference 38 and data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16997959" target="_blank">16997959</a>]<br></li>
<li>After completely transferring its phosphoryl group to OmpR within 40 sec , EnvZc functions as phosphatase toward OmpR-P .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11158569" target="_blank">11158569</a>]<br></li>
<li>They could reflect different conformations of OmpR , different molecular species produced , for example , by a covalent modification , or some combination of both .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2557454" target="_blank">2557454</a>]<br></li>
<li>During a growth shift , for example , from outside-the-host to inside-the-host conditions , the roles of OmpR and Lrp may be temporally different as well as functionally different .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8002608" target="_blank">8002608</a>]<br></li>
<li>In an extreme example of this - type , the OmpR regulator has been shown to repress and activate from the very same position ( 155 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1943993" target="_blank">1943993</a>]<br></li>
<li>In another construction , a HindIII site OmpR in the absence of ATP ( lane 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8175665" target="_blank">8175665</a>]<br></li>
<li>In vivo , bolA1p was found to be repressed by OmpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10802181" target="_blank">10802181</a>]<br></li>
<li>In contrast , no signiﬁcant MzrA -- OmpR interactions ( lanes 6 and 7 ) , or MzrA -- CpxA interactions ( lanes 8 and 9 ) were detected .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19432797" target="_blank">19432797</a>]<br></li>
<li>In the presence of T83I , the rate of Pi release is 0.5 nmol/ml per minute , compared to 4.6 nmol/ml per minute for OmpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11812125" target="_blank">11812125</a>]<br></li>
<li>In further determinations , this nearly twofold difference is maintained , as the average apparent Kd value for OmpR at C1 is 101 ( 59 ) nM and for T83I it is 59 ( 19 ) nM .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11812125" target="_blank">11812125</a>]<br></li>
<li>Upon addition of OmpR , there is a large increase in Pi production , that increases with increasing incubation time ( Figure 5 , triangles ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11812125" target="_blank">11812125</a>]<br></li>
<li>Furthermore , when phospho-OmpR was used , the protected regions were extended from 2135 to 2116 and from 14 to 217 , with an increase in afﬁnity of about 10-fold .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7642497" target="_blank">7642497</a>]<br></li>
<li>A threefold increase in the amount of OmpR did not cause any significant change in the pattern of protected regions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8413182" target="_blank">8413182</a>]<br></li>
<li>This result is due to the bifunctional feature of EnvZc , which not only phosphorylates OmpR but also dephosphorylates the resulting OmpR-P .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11158569" target="_blank">11158569</a>]<br></li>
<li>It would not be surprising if , like NanR , NagC , and OmpR , CpxR plays a role in the control of type 1 ﬁmbriation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15743943" target="_blank">15743943</a>]<br></li>
<li>Contributions of AcrD and MdtABC to multidrug resistance mediated by overexpression of BaeR , CpxR , and OmpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12618449" target="_blank">12618449</a>]<br></li>
<li>CpxR and OmpR control the formation of adhesion structures such as curli and Pap pili ( 9 , 17 , 21 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15743943" target="_blank">15743943</a>]<br></li>
<li>1992 ) , OmpR ( Forst etai , 1990 ) , and CheY ( Dailey and Berg , 1993 ) in the absence of their cognate kinases , PhoR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7934904" target="_blank">7934904</a>]<br></li>
<li>Asterisks ( * ) indicate the conserved aa residues crucial for the transcriptional regula ¬ 232 CpxR nvsas ... OmpR ArcA VirG PhoP V & `` R PhoB CreB TctD KdpE hrpdgska .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8294007" target="_blank">8294007</a>]<br></li>
<li>OmpR , SfrA , PhoB , and VirG are one such group .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2404948" target="_blank">2404948</a>]<br></li>
<li>PhoB CreB OmpR FIG. 7 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1532388" target="_blank">1532388</a>]<br></li>
<li>Members of this class are NtrC and PhoB , other candidates are ArcA , OmpR and KdpE .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7641688" target="_blank">7641688</a>]<br></li>
<li>227 OmpR ArcA VirG PhoP Va `` R PhoB CreB TctD KdpE kINnlaRgre .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8294007" target="_blank">8294007</a>]<br></li>
<li>This showed that osmolal regulation but not temperature regulation of micF depends on these upstream sequences and suggested that while osmolal regulation of the micF gene depends on OmpR , thermal regulation does not .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1695892" target="_blank">1695892</a>]<br></li>
<li>Second , sequences required for OmpR stimulation are located upstream of -54 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1906064" target="_blank">1906064</a>]<br></li>
<li>First , our results indicate that the regulation of tppB under aerobic and anaerobic conditions is dependent on OmpR ( Table 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15175316" target="_blank">15175316</a>]<br></li>
<li>Experiments are in progress to determine whether cells containing pRT506-2 produce OmpR in greater than normal amounts , and , if so , how this affects porn regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6263866" target="_blank">6263866</a>]<br></li>
<li>ompR3 c ( dom ) ( dom ) Current models for porin regulation invoke the presence of multiple forms or conformations of OmpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2557454" target="_blank">2557454</a>]<br></li>
<li>Our ﬁnding on the involvement of OmpR in ﬂagellar regulation appears to shed light on earlier queries raised by independent observations .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7642497" target="_blank">7642497</a>]<br></li>
<li>Transcriptional regulation of bolA1p by OmpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10802181" target="_blank">10802181</a>]<br></li>
<li>OmpR and MipA-YeaE regulation of rybB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17416652" target="_blank">17416652</a>]<br></li>
<li>Like repression of ﬂagellar biosynthesis , this cell division behavior requires PTA , OmpR , and FlhD ( but not FlhC ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15755952" target="_blank">15755952</a>]<br></li>
<li>It seems likely that they correspond to fragments IV and V of OmpR , and data presented below support this view .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7568033" target="_blank">7568033</a>]<br></li>
<li>However , the presence of several nearly perfect CpxR consensus inside the OmpR box strongly suggests the existence of a competition between OmpR and CpxR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15743952" target="_blank">15743952</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_966">
( b ) Gel shift assay in the presence of both IHF and OmpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20466769" target="_blank">20466769</a>]<br></label>
<input id="act_similarity_966", type="checkbox">
<div class="ACT_similar">
<li>( c ) Gel shift assay in the presence of both IHP and OmpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20466769" target="_blank">20466769</a>]<br></li></div>
<li>These five peptides correspond to residues 1-192 , 1-190 , 1-142 , 193-239 , and 200-239 of OmpR , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7568033" target="_blank">7568033</a>]<br></li>
<li>However , increasing the amount of OmpRD55E should correlate with an increasing amount of active OmpR in the cell .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9422609" target="_blank">9422609</a>]<br></li>
<li>Potential roles of DsbA , DsbB , and OmpR in OmpF synthesis and maturation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26209678" target="_blank">26209678</a>]<br></li>
<li>Like OmpR P , RcsB P represses ﬂagellar synthesis directly ( 138 , 330 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15755952" target="_blank">15755952</a>]<br></li>
<li>Lanes 2-7 show the cleavage pattern obtained when 1.0 , 2.0 , 3.2 , 4.5 , 5.8 , and 7.0 mM OmpR was included in the reaction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11812125" target="_blank">11812125</a>]<br></li>
<li>Recently , we found that native PAGE is another approach to detecting the interaction between OmpR and EnvZc .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11158569" target="_blank">11158569</a>]<br></li>
<li>After 15 minutes at room temperature , 40 mg of OmpR was added .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11812125" target="_blank">11812125</a>]<br></li>
<li>Western immunoblot analysis of OmpR levels in the cells .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9422609" target="_blank">9422609</a>]<br></li>
<li>Thus , we attempted to phosphorylate OmpR in vivo by labeling cells with 32Pi .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2160945" target="_blank">2160945</a>]<br></li>
<li>They hypothesize that cells degrade serine and synthesize acetylphosphate , which is used to phosphorylate OmpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11104815" target="_blank">11104815</a>]<br></li>
<li>Nara , F. , Matsuyama , S. , Mizuno , T. , and Mizushima , S. ( 1986 ) from methylation by OmpR , which most likely indicates that Mol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2553720" target="_blank">2553720</a>]<br></li>
<li>Instead , like OmpR , NtrC can acquire phosphoryl groups from several sources .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8098993" target="_blank">8098993</a>]<br></li>
<li>Deleted in proof OmpR was involved , perhaps occupancy of Fb facilitates 11 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2553720" target="_blank">2553720</a>]<br></li>
<li>This extensive characterization parallels and corroborates the OmpR mutagenesis data analysis presented here .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9016718" target="_blank">9016718</a>]<br></li>
<li>This OmpR was enriched approximately 135-fold .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2656631" target="_blank">2656631</a>]<br></li>
<li>This serine is not present in OmpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11812125" target="_blank">11812125</a>]<br></li>
<li>OmpR was not added to lanes 1,4 , 7 and 10 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7557323" target="_blank">7557323</a>]<br></li>
<li>Lane l. no OmpR ; lanes 2 and 5 , 100 ng OmpR ; lanes 3 and 6 , 125 ng OmpR ; lanes 4 and 7 , 150 ng OmpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1736095" target="_blank">1736095</a>]<br></li>
<li>Lane 1 represents DNase I cleavage in the absence of OmpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11812125" target="_blank">11812125</a>]<br></li>
<li>Phosphorylation Stimulates Cooperative Interactions Between OmpR Molecules Bound at Neighboring Sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9096305" target="_blank">9096305</a>]<br></li>
<li>Phosphorylation of OmpR in a strain containing envZVal .243 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2160945" target="_blank">2160945</a>]<br></li>
<li>OmpR Phosphorylation by Acetyl Phosphate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7568033" target="_blank">7568033</a>]<br></li>
<li>Phosphorylation of OmpR in MH1461 ( envZII ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2160945" target="_blank">2160945</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_991">
OmpR Phosphorylation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7568033" target="_blank">7568033</a>]<br></label>
<input id="act_similarity_991", type="checkbox">
<div class="ACT_similar">
<li>Phosphorylation of OmpR in MC4100 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2160945" target="_blank">2160945</a>]<br></li></div>
<li>Dye OmpR proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3531171" target="_blank">3531171</a>]<br></li>
<li>nearly identical , 27,346 for Dye and 27,353 for OmpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2997120" target="_blank">2997120</a>]<br></li>
<li>Initial interaction of OmpR with the - 100 * Corresponding author .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7557323" target="_blank">7557323</a>]<br></li>
<li>As with most of the RRs , OmpR lacks this property [ 63 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16487683" target="_blank">16487683</a>]<br></li>
<li>Therefore , we assayed some of these constructs in OmpR + and OmpR - isogenic strains .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1906064" target="_blank">1906064</a>]<br></li>
<li>OmpR , which it can both phosphorylate or dephosphorylate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16359331" target="_blank">16359331</a>]<br></li>
<li>OmpR Dephosphorylation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7568033" target="_blank">7568033</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_1000">
+ OmpR ' ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2668281" target="_blank">2668281</a>]<br></label>
<input id="act_similarity_1000", type="checkbox">
<div class="ACT_similar">
<li>OmpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1736095" target="_blank">1736095</a>]<br></li></div>
<li>Our data suggest that OmpR orchestrates activation of curli at csgD , but the multiple repressions at csgBA offer numerous possibilities of regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16000739" target="_blank">16000739</a>]<br></li>
<li>Depending on the specific sites bound by OmpR , OmpR functions as a transcriptional-activator or repressor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9016718" target="_blank">9016718</a>]<br></li>
<li>Phosphorylated PhoB is the activator for the pho regulon ( Wanner , 1993 ) and belongs to the OmpR subfamily of transcriptional-activators for a70-dependent promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7641688" target="_blank">7641688</a>]<br></li>
<li>As in the CRP studies , the activation is maximal when the OmpR sites are restricted to the same side of the helix as the natural site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1943993" target="_blank">1943993</a>]<br></li>
<li>Activation by MerR requires residue 323 ( 7 ) , and both residues 322 and 323 have been identiﬁed as important for OmpR activation ( 30 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10852886" target="_blank">10852886</a>]<br></li>
<li>A transcriptional fusion to csgD was inactive in both Dtol ompR : : Tn10 and ompR : : Tn10 backgrounds , while the effects of Dtol and ompR334 , an allele that increases the activation of csgD by OmpR , were cumulative in the stationary phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16000739" target="_blank">16000739</a>]<br></li>
<li>OmpR is required for transcriptional activation of the csgDEFG operon when CpxR acts as a repressor ( 19 , 57 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15743952" target="_blank">15743952</a>]<br></li>
<li>OmpR mutants speci ® cally defective for transcriptional activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9677290" target="_blank">9677290</a>]<br></li>
<li>A major conclusion of our studies with the tppB - lacZ fusion construct is that the transcriptional activation of tppB is dependent on OmpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15175316" target="_blank">15175316</a>]<br></li>
<li>The hypersignaling mutation envZ11 , which causes increased phosphorylation of OmpR due to a defect in dephosphorylation ( 14 ) , exhibits a nonmotile phenotype ( data not shown ) , which indeed results in severe reduction in ﬂhDC expression below the level of that of the ackA mutant .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7642497" target="_blank">7642497</a>]<br></li>
<li>Another consideration is that a multicopy ompR effect , which in some overexpression of ompC ( 11 , studies was shown to result in conjunction with instead of the increased 24 , 38 ) , in or phosphorylation of OmpR caused the elevated expression of ompC in the ompB deletion strain containing the multicopy plasmid pSF112 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2160945" target="_blank">2160945</a>]<br></li>
<li>Thus , the enhanced level of OmpR-phosphate in this strain grown in highosmolarity medium is correlated with increased expression These results clearly demonstrate that OmpR can of ompC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2160945" target="_blank">2160945</a>]<br></li>
<li>Since ompC can be expressed in vitro in the absence of added OmpR ( 35 ) , an analysis of the role of phosphorylation of OmpR on in vitro expression of ompC has not yet been reported .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2160945" target="_blank">2160945</a>]<br></li>
<li>We present results that support the view that increased phosphorylation of OmpR is correlated with enhanced expression of ompC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2160945" target="_blank">2160945</a>]<br></li>
<li>The results presented in this study , together with those of Aiba et al. ( 1 ) , support the idea that increased phosphorylation of OmpR results in enhanced expression of ompC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2160945" target="_blank">2160945</a>]<br></li>
<li>Since the OmpR transcriptional-regulator activates expression of both omrA and omrB genes , this newly identified control should result in an autoregulatory loop limiting the amount of OmrA/B sRNAs .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18953042" target="_blank">18953042</a>]<br></li>
<li>Finally , we ﬁnd that one direct target of OmrA and OmrB is the mRNA encoding the transcriptional-regulators of these sRNAs , EnvZ and OmpR , thus providing a feedback loop in which activation of transcription will downregulate expression of the activators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18953042" target="_blank">18953042</a>]<br></li>
<li>As shown in Fig. 1B , tppB - lacZ fusion expression is extremely low in the strain containing the ompR101 mutation , conﬁrming that the activation of tppB transcription is dependent on OmpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15175316" target="_blank">15175316</a>]<br></li>
<li>Despite weak csgD expression in the absence of the OmpR activator , repression was still observed in high-sucrose medium ( Fig. 6B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15743952" target="_blank">15743952</a>]<br></li>
<li>However , the above experiments can not exclude that downregulation of csgD expression by OmrA/B occurred through inhibition of the expression of the transcriptional-activator OmpR ; OmpR is a known OmrA/B target ( Guillier and Gottesman , 2008 ) and promotes csgD transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20407422" target="_blank">20407422</a>]<br></li>
<li>The sRNA-induced decrease in csgD mRNA from its chromosomal copy could be dominated by inhibition of OmpR , the activator required for csgD expression ; inhibition of OmpR would cause lower transcription levels .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20407422" target="_blank">20407422</a>]<br></li>
<li>At low levels of wild-type OmpR , low levels of OmpF expression are observed ( lane 11 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9422609" target="_blank">9422609</a>]<br></li>
<li>It is apparent from the results presented that under the conditions used here ( growth in rich medium ) , both exponential - and stationary-phase levels of expression from the wild-type mcbAp promoter are reduced to a similar extent in the OmpR - background , resulting in an induction ratio similar to that observed for OmpR + .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1906064" target="_blank">1906064</a>]<br></li>
<li>Overexpression of OmpR , by increasing the expression of E the major OMPs , would be expected to increase activity ( 3 ) and therefore rybB expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17416652" target="_blank">17416652</a>]<br></li>
<li>Consistent with multicopy OmpR increasing expression of RybB ( Fig. 1B and Table 2 ) , the speciﬁc activity of the long rybB-lacZ was reduced approximately ﬁvefold in the ompR mutant in stationary phase ( Fig. 1A ) , suggesting that OmpR is necessary for the full activity of the rybB-lacZ fusion .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17416652" target="_blank">17416652</a>]<br></li>
<li>Two other plasmids , including the one carrying emrR , failed to increase expression in the ompR mutant ( Fig. 1B ) , suggesting that the stimulatory activity of these plasmids was upstream of OmpR and depends upon OmpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17416652" target="_blank">17416652</a>]<br></li>
<li>They proposed that the high motility of MG1655 is due to insertion element insB-5 , which prevents binding of the negative regulator OmpR , and thus increase the expression of ﬂhDC operon for ﬂagellar synthesis ( Supplementary Figure S1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19969540" target="_blank">19969540</a>]<br></li>
<li>Most recently , Ozawa and Mizushima ( 38 ) have suggested that OmpF synthesis is regulated by OmpR and that OmpF acts as a negative regulator of OmpC expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6317653" target="_blank">6317653</a>]<br></li>
<li>We therefore propose that the regulatory role of phospho-OmpR in ﬂagellar synthesis is negative , whereas it is both positive and negative in porin expression ( 39 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7642497" target="_blank">7642497</a>]<br></li>
<li>It remains to be shown whether the dominant-negative effect of GST-Zc or GST-ZBc on porin expression results from sequestration of OmpR ( 34 ) , as proposed for the effect by GST-Bc on UhpA , or from its action as P-OmpR phosphatase ( 15 , 51 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11325944" target="_blank">11325944</a>]<br></li>
<li>To test this hypothesis , we used these sets of conditions to examine the effect of the cellular level of active OmpR on porin expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9422609" target="_blank">9422609</a>]<br></li>
<li>Effects of the level of OmpRD55E and wild-type OmpR on porin expression in the presence of three different alleles of envZ .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9422609" target="_blank">9422609</a>]<br></li>
<li>Besides requiring sS and MlrA , csgD expression is also modulated by IHF and the two-component response-regulators OmpR and CpxR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17010156" target="_blank">17010156</a>]<br></li>
<li>OmpR was shown to activate csgD expression , resulting in curli production .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15743952" target="_blank">15743952</a>]<br></li>
<li>Two other regulators , OmpR and CpxR , activate nanC expression , although a direct control has not been proven .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15743943" target="_blank">15743943</a>]<br></li>
<li>Furthermore , the following evidence indicates that the phosphorylation of OmpR , a trans factor for osmoregulation , regulates ﬂagellar expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7642497" target="_blank">7642497</a>]<br></li>
<li>Although ompR null mutants do not exhibit an increase in malT expression , mutations in envZ leading to uncontrolled phosphorylation of OmpR reduce the transcription of malT ( 33 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9529892" target="_blank">9529892</a>]<br></li>
<li>Although it is not clari cents ed how much OmpR is phosphorylated by stress , bolA1p expression may be induced with a decreased phospho-OmpR concentration in the cells under the stress .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10802181" target="_blank">10802181</a>]<br></li>
<li>A possible explanation for these observations is that omrB expression could be activated at lower levels of phosphorylated OmpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16359331" target="_blank">16359331</a>]<br></li>
<li>Depletion of the acetogenic carbon source leads to reduction of the acetyl P pool , the accumulation of nonphosphorylated OmpR , and , thus , the derepression of ﬂhDC transcription and ﬂagellar expression ( 355 , 358 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15755952" target="_blank">15755952</a>]<br></li>
<li>Moreover , this expression is dependent on the presence of OmpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15175316" target="_blank">15175316</a>]<br></li>
<li>Involvement of OmpR in acetyl-P effect on ﬂhDC expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7642497" target="_blank">7642497</a>]<br></li>
<li>Shin and Park showed that acetylphosphate inhibited ﬂhDC expression through OmpR ( 117 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11104815" target="_blank">11104815</a>]<br></li>
<li>When cells accumulate acetyl-P , and thus have increased levels of phospho-OmpR , the ﬂagellar expression is substantially reduced .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7642497" target="_blank">7642497</a>]<br></li>
<li>On the other hand , MicC RNA levels were increased , indicating that OmpR , directly or indirectly , represses MicC expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15466019" target="_blank">15466019</a>]<br></li>
<li>In vivo and in vitro evidence suggested that at least some of the effects by IHF are due to direct or indirect interactions of IHF with OmpR , the major transcriptional-regulator of these operons ( Huang etal .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1447969" target="_blank">1447969</a>]<br></li>
<li>In the exponential phase , only a few differences were observed between the wild-type strains and the mutants defective in FIS , H-NS , IHF , OmpR , cAMP , and ppGpp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11292794" target="_blank">11292794</a>]<br></li>
<li>We examined the effects of some global regulators ( factor inversion stimulation [ FIS ] , H-NS , OmpR , cAMP , ppGpp , and IHF ) on the expression of sra by using global regulator-defec-tive mutants lysogenized with lpF13 ( Psra-lacZ ) or carrying pFF6 ( Psra-lacZ ) and by measuring b-galactosidase activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11292794" target="_blank">11292794</a>]<br></li>
<li>However , details of the mechanisms of ﬁne control of sra expression in the exponential phase as well as in the stationary phase by sigma S , OmpR , FIS , IHF , cAMP , and/or ppGpp remain to be investigated further .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11292794" target="_blank">11292794</a>]<br></li>
<li>The OmpR Protein of Escherichia coli Binds to Sites in theompF Promoter Regionin aHierarchical Manner Determinedby its Degree of Phosphorylation * Arfaan RampersaudS , Susan L. Hadockerg , and MasayoriInouyeOn From the $ Departmentof Pathology , the Ohio State University , Columbus , Ohio 43210 and the § Departmentof Biochemistry , Universityof Medicine and Dentistry , the Robert Wood Johnson Medical School , Piscataway , New Jersey 08854 In Eecherichia coli the ompF gene encodes a major and increase the relative amounts of OmpR-phosphate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8175665" target="_blank">8175665</a>]<br></li>
<li>JOURNAL OF BACTERIOLOGY , June 1989 , p. / 89/062949 -07 $ 02.00 / 0 Copyright © 1989 , American Society for Microbiology DNA-Binding Properties of the Transcription Activator ( OmpR ) for the Upstream Sequences of ompF in Escherichia coli Are Altered by envZ Mutations and Medium Osmolarity STEVEN A. FORST , * JORGE DELGADO , AND MASAYORI INOUYE Department of Biochemistry , University of Medicine and Dentistry of New Jersey , Robert Wood Johnson Medical School at Rutgers , 675 Hoes Lane , Piscataway , New Jersey 08854 by the high-osmolarity growth conditions ( 4 , 33 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2656631" target="_blank">2656631</a>]<br></li>
<li>JOURNAL OF BACTERIOLOGY , June 1990 , p. / 90/063473 -05 $ 02.00 / 0 Copyright © 1990 , American Society for Microbiology of Phosphorylation of OmpR , the Transcription Activator the ompF and ompC Genes in Escherichia coli STEVEN FORST , t * JORGE DELGADO , ARFAAN RAMPERSAUD , AND MASAYORI INOUYE Department of Biochemistry , Robert Wood Johnson Medical School , University of Medicine and Dentistry of New Jersey , Piscataway , New Jersey 08854 Received 18 December 1989/Accepted 7 March 1990 An in vivo approach was taken to assess whether the phosphorylated state of the transcription activator OmpR was affected by changes in the osmolarity of the growth medium or by mutations in envZ , the gene encoding the inner membrane histidine kinase that phosphorylates OmpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2160945" target="_blank">2160945</a>]<br></li>
<li>Printed in the U.S.A. Transcription Regulation of o Transcription Factor , OmpR * mpF and ompC by a Single Received for publication , March 6 , 2006 , and in revised form , April 14 , 2006 Published , JBC Papers in Press , April 17 , 2006 , DOI 10.1074 / jbc.M602112200 Takeshi Yoshida1 , Ling Qin1 , Linda A. Egger , and Masayori Inouye2 From the Department of Biochemistry , Robert Wood Johnson Medical School , Piscataway , New Jersey 08854 The ompF and ompC genes of Escherichia coli are reciprocally regulated by a single transcription factor , phosphorylated OmpR ( OmpR-P ) , depending upon medium osmolarity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16618701" target="_blank">16618701</a>]<br></li>
<li>Evidence for the Physiological Importancoef the Phosphotransfer between theTwo Regulatory Components , EnvZ andOmpR , in Osmoregulation inEscherichia coli * ( Received for publication , March 28 , 1989 ) Hirofumi AibaS , FujieNakasaiS , Shoji Mizushimag , and Takeshi MizunoST From the $ Laboratory of Microbiology , School of Agriculture , Nagoya University , Chikusa-ku , Nagoya 464 and the $ Institute of Applied Microbiology , the University of Tokyo , Yuyoi , Bunkyo-ku , Tokyo 113 , Japan Previously , the transfer of the phosphoryl group between the EnvZand OmpR proteins , whichare involved in activation of the ompF and ompC genes in response to the medium osmolarity , has been demonstrated in vitroI .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2668281" target="_blank">2668281</a>]<br></li>
<li>JOURNAL OF BACTERIOLOGY , JUIY 1991 , p. / 91/134039 -10 $ 02.00 / 0 Copyright © ) 1991 , American Society for Microbiology F Mutations That Result in O Constitutive Expression cis-Acting omp mpR-Dependent JAMES M. SLAUCHt * AND THOMAS J. SILHAVY Department of Molecular Biology , Lewis Thomas Laboratory , Princeton University , Princeton , New Jersey 08544-1014 Received 3 December 1990/Accepted 30 April 1991 OmpR and EnvZ differentially control the transcription of the major outer membrane porin genes , ompF and ompC , in Escherichia coli in response to the osmolarity of the medium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1648075" target="_blank">1648075</a>]<br></li>
<li>12.38 -8.24 -59.96 1.13 ± 0.55 ± 0.50 ± 0.02 0.04 0.02 1.27 ± 0.50 ± 0.20 ± 0.03 0.03 0.03 / h ) / h ) Fig. 2 Comparison of the transcript levels of the E. coli wild type under different pH values at 7.0 and 6.0 : a global regulatory genes , b PhoB regulatory genes , c acid-inducible genes , d metabolic pathway genes , e respiratory chain genes during micro-aerobic continuous culture at the dilution rate of 0.1 h -1 Fig. 3 a , b Overall metabolic regulation of the E. coli during acidic condition ( a ) Acid Condition Hns ATP ADP PhoQ PhoQ P ATP ADP ¬ EvgS EvgS P cAMP - evgA Pho P PhoP Crp EvgA P B1500 EvgA cAMP-Crp ¬ RpoS b1500 ydeo ¬ YdeO gadE GadE O O GadW GadX hdeA gadB gadc gadW gadA gadX HdeA GadA GadB GadC Acid resistance ( b ) Acidic Condition ( H + ) ( indirect ) EnvZ PhoR fnr OxyR ldhA arcA P Fnr ArcA SoxR / S L PhoR - P OmpR OmpR - P DH pfl yfiD ompC TCA cycle genes : gltA sdhC mdh sodA PhoB - P PhoB Lactate OmpC ompF asr YfiD Pfl OmpF Asr Formate Acetate ackA pta Acid resistance Moreover , in accord with the proteomic analysis , the expression of yﬁD : : lac reporter fusion and the intracellular content of YﬁD were found to be high during growth at low pH for both aerobic and anaerobic cultures ( Neil et al. 2002 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23274360" target="_blank">23274360</a>]<br></li>
<li>The OmpR protein of Escherichia coli binds to sites in the ompF promoter region in a hierarchical manner determined by its degree of phosphoryl-ation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11812125" target="_blank">11812125</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_1059">
Rampersaud , A. , Harlocker , S.L. , and Inouye , M. ( 1994 ) The OmpR protein of Escherichia coil binds to sites in the ompF promoter region in a hierarchical manner determined by its degree of phosphorylation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596446" target="_blank">8596446</a>]<br></label>
<input id="act_similarity_1059", type="checkbox">
<div class="ACT_similar">
<li>[ lOI Rampersaud , A. , Harlocker , S.L. and Inouye , M. ( 1994 ) The OmpR protein of Escherichia coli binds to sites in the ompF promoter region in a hierarchical manner determined by its degree of phosphorylation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7557323" target="_blank">7557323</a>]<br></li>
<li>Rampersaud , A. , Harlocker , S.L. and Inouye , M. ( 1994 ) The OmpR protein of Escherichia coli binds to sites of the ompF promoter region in a hierarchical manner determined by its degree of phosphorylation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9130712" target="_blank">9130712</a>]<br></li>
<li>Rampersaud , A. , Harlocker , S.L. , and Inouye , M. ( 1994 ) The OmpR protein of Escherichia coli binds to sites in the ompF promoter region in a hierarchical manner determined by its degree of phosphorylation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16194231" target="_blank">16194231</a>]<br></li>
<li>The OmpR protein of Escherichia coli binds to sites in the ompF promoter region in a hierarch-ical manner determined by its degree of phosphorylation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7642497" target="_blank">7642497</a>]<br></li>
<li>The OmpR protein of Escherichia coli binds to sites in the ompF promoter region in a hierarchical manner determined by its degree of phosphorylation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10648521" target="_blank">10648521</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25161193" target="_blank">25161193</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10913075" target="_blank">10913075</a>]<br></li></div>
<li>These results suggest that OmpR binds to the -71 to -64 - bp region and that this sequence plays a role in the regulation of ompF in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7557323" target="_blank">7557323</a>]<br></li>
<li>MICROBIOLOGY LETTERS ELSEVIER F'EMSMicrobiology Letters 131 ( 1995 ) 147-151 Molecular analysis of OmpR binding sequences involved in the regulation of ompF in Escherichia coli S. Forst * , I. Kalve , W. Durski Department of Biological Sciences , Universiryof Wisconsin-Milwaukee , PO Box 413 , Milwaukee , WI53201 , USA Received 9 May 199.5 ; revised 21 June 1995 ; accepted 23 June 1995 Abstract OmpR , the transcriptional-regulatory protein of ompF , had not been previously shown to specifically bind to the - 70 to - 60-bp region of ompF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7557323" target="_blank">7557323</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_1067">
Harlocker , S. , Bergstrom , L. and Inouye , M. ( 1995 ) Tandem binding of six OmpR proteins to the ompF upstream regulatory sequence of Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9130712" target="_blank">9130712</a>]<br></label>
<input id="act_similarity_1067", type="checkbox">
<div class="ACT_similar">
<li>Harlocker , S.L. , Bergstrom , L. , and Inouye , M. ( 1995 ) Tandem binding of six OmpR proteins to the ompF upstream regulatory sequence of Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16359331" target="_blank">16359331</a>]<br></li></div>
<li>OmpR activity might also be stimulated , leading to an increase in ompC ( Bernadac et al. , 1998 ; Lazzaroni et al. , 1986 ) and csgDEFG expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16000739" target="_blank">16000739</a>]<br></li>
<li>A similar model of transcriptional regulation occurs in the OmpR regulation of ompC ( Bergstrom et al. , 1998 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16194231" target="_blank">16194231</a>]<br></li>
<li>A low degree of phosphorylation of OmpR is thought to activáte ~ m ~ ~ e x p r e s sw l i hoine a highly phosphorylated form of OmpR is believed to activate ompC expression and inhibit ompF ( Forst et al. 1989a ; Slauch and Silhavy 1989 ; Aiba and Mizuno 1990 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1736095" target="_blank">1736095</a>]<br></li>
<li>It has been proposed that this differential regulation is mediated by various forms or states of OmpR ; one form activates ompF and the other form activates ompC ( Hall & , Silhavy , 1981b ; Slauch et aE .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2557454" target="_blank">2557454</a>]<br></li>
<li>Results Differential regulation of the porin genes is thought to involve multiple forms of OmpR ; one form activates ompF and the other form activates ompC ( Hall & Silhavy , 19815 ; Slauch et al. , 1988 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2557454" target="_blank">2557454</a>]<br></li>
<li>The ompR234 mutation corresponded to an L43R transition near the phosphorylation site of OmpR ( D55 ) , leading to an increase in csgDEFG transcription ( Vidal et al. , 1998 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16000739" target="_blank">16000739</a>]<br></li>
<li>The phosphorylation of OmpR leads to approximately tenfold stimulation of its binding affinity for the ornpFand ompC promoters ( Aiba et al. 1989b ) , and the binding affects transcription of these genes ( Igo et al. 1989a , b ; Aiba and Mizuno 1990 ; Slauch and Silhavy 1991 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8413182" target="_blank">8413182</a>]<br></li>
<li>In the absence of OmpR phosphorylation , OmpR may be able to-bind F1 , but it is unable to-bind F2 in sufficient quantities to influence ompF transcription ( Fig. 5A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9096305" target="_blank">9096305</a>]<br></li>
<li>, 1986 ) , gave the expected result ; the mutant OmpR did not bind to the ompC promoter and bound to only a subset of the sites in the ompF promoter ( Mizuno et al. , 1988 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2557454" target="_blank">2557454</a>]<br></li>
<li>Moreover , phosphor-ylated OmpR is more active in stimulating transcription in vitro from the ompF promoter ( Igo et al. , 1989a ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2557454" target="_blank">2557454</a>]<br></li>
<li>In contrast , Igo et al. ( 17 , 18 ) showed that ompF transcription is stimulated in vitro by the phosphorylation of OmpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2160945" target="_blank">2160945</a>]<br></li>
<li>Since Igo et al. ( 17 , 18 ) demonstrated that phosphorylation of OmpR enhanced the in vitro transcription of ompF , there appears to be a requirement for OmpR phosphorylation in the transcription of both ompF and ompC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2160945" target="_blank">2160945</a>]<br></li>
<li>D035 .8 D038 .3 RpoS B015 .5 C013 .5 OxyR Cd , QN , HP , uspA PS , NS , Ile , Cd , QN , HP , DNP , 42 NS DNP fumC glpK dnaN ompC DNP , 42 RpoS OmpR , Lrp Lrp , RpoS FIG. 4 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7559363" target="_blank">7559363</a>]<br></li>
<li>In the absence of the main activator ( BaeR for acrD and mtdA , OmpR for ompC ) , CpxR-P can not act as a transcriptional-activator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16487683" target="_blank">16487683</a>]<br></li>
<li>Indeed , ﬂuorescence analysis showed that a cpxA ompR double mutant failed to express either porin when it was grown in glucose minimal medium ( data not shown ) , indicating that in the absence of OmpR , CpxR-P can not act as a transcriptional-activator of ompC or ompF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16077119" target="_blank">16077119</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_1084">
[ 7 ] Forst , S. , Delgado , J. , Rampersaud , A. and Inouye , M. ( 1990 ) In uiuo phosphorylation of OmpR , the transcription activator of the ompF and ompC genes in Escherichia cofi .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7557323" target="_blank">7557323</a>]<br></label>
<input id="act_similarity_1084", type="checkbox">
<div class="ACT_similar">
<li>Forst , S. , Delgado , J. , Rampersaud , A. , and Inouye , M. ( 1990 ) In vivo phosphorylation of OmpR , the transcription activator of the ompF and ompC genes in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8412671" target="_blank">8412671</a>]<br></li></div>
<li>References Aiba H , Mizuno T ( 1990 ) Phosphorylation of a bacterial activator protein , OmpR , by a protein kinase , EnvZ , stimulates the transcription of the ompF and ompC genes in Escheriehia eoli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8413182" target="_blank">8413182</a>]<br></li>
<li>Phosphorylation of a bacterial activator protein , OmpR , by a protein kinase , EnvZ , stimulates the transcription of the ompF and ompC genes in Eschenchia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1532173" target="_blank">1532173</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_1088">
htm References Aiba , H. , and Mizuno , T. ( 1990 ) Phosphorylation of a bacterial activator protein , OmpR , by a protein kinase , EnvZ , stimulates the transcription of the ompF and ompC genes in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12366850" target="_blank">12366850</a>]<br></label>
<input id="act_similarity_1088", type="checkbox">
<div class="ACT_similar">
<li>Phosphorylation of a bacterial activator protein , OmpR , by a protein kinase , EnvZ , stimulates the transcription of the ompF and ompC genes in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8113170" target="_blank">8113170</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1885551" target="_blank">1885551</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1310090" target="_blank">1310090</a>]<br></li></div>
<li>OmpR is the well-known transcriptional-regulator of ompF and ompC porin genes in response to osmolarity change .9,10,72 OmpR is phosphorylated by EnvZ and by low molecular weight phosphodo-nors such as acetyl phosphate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11601842" target="_blank">11601842</a>]<br></li>
<li>The activated CpxA/CpxR and sE regulons turn on the synthesis of MzrA , which , in turn through its interaction with EnvZ , modulates EnvZ/OmpR 's activity to favour the accumulation of OmpR ~ P. High levels of OmpR ~ P upregulates the expression of ompC , csgD and omrAB , but downregulates the expression of ompF , lamB and malE .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19432797" target="_blank">19432797</a>]<br></li>
<li>ompB also has a dual role in E. coli , since its products , OmpR and EnvZ , control the levels of the OmpF and OmpC porins through transcriptional regulation of the ompF and ompC operons ( 22 , 23 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1310090" target="_blank">1310090</a>]<br></li>
<li>IHF is also a transcriptional-regulator of the ompB operon whose protein products , EnvZ and OmpR , are required for expression and osmoregulation of ompF and ompC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1447969" target="_blank">1447969</a>]<br></li>
<li>The products of the ompB operon , OmpR and EnvZ , are necessary for the osmoregulation and expression of ompF and ompC ( 16 , 17 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2203749" target="_blank">2203749</a>]<br></li>
<li>For example , the EnvZ protein may be involved in modification of the OmpR protein , which in turn positively regulates either the ompF or ompC gene , depending on the type of modification .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3536870" target="_blank">3536870</a>]<br></li>
<li>In many of these cases , it is not clear whether these conditions affect ompF and ompC directly or indirectly , through changes in the levels of OmpR and EnvZ ( 22 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1310090" target="_blank">1310090</a>]<br></li>
<li>I 4 , -116 239 1 450 247 Thr - Arg ( C17074G ) 247 MH1461 450 2391 Leu - Gln DISCUSSION The OmpR and EnvZ proteins are involved in osmoregula-of the ompF and ompC ( 11 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3536870" target="_blank">3536870</a>]<br></li>
<li>Thus , ompF and ompC are regulated reciprocally by the cellular level of OmpR-P ( for a review of the EnvZ -- OmpR signal transduction pathway , see refs .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11158569" target="_blank">11158569</a>]<br></li>
<li>The biochemical properties of EnvZ suggest that its role in this signal transduction pathway is control the level of OmpR to phosphorylation in the cell and that it is the level of OmpR-P which determines whether ompF or ompC is expressed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8113170" target="_blank">8113170</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_1100">
Delgado , J. and Inouye , M. ( 19891 Phosphorylation of OmpR by the osmosensor EnvZ modulates the expression of the ompF and ompC genes in Escherichia co/i .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8522138" target="_blank">8522138</a>]<br></label>
<input id="act_similarity_1100", type="checkbox">
<div class="ACT_similar">
<li>[ 51Forst , S. , Delgado , J. and Inouye , M. ( 1989 ) Phosphorylation of OmpR by the osmosensor EnvZ modulates expression of the ompF and ompC genes in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7557323" target="_blank">7557323</a>]<br></li>
<li>Forst , S. , Delgado , J , , and Inouye , M , ( 1989 ) Phosphorylation of OmpR by the osmosensor EnvZ modulates expression of the ompF and ompC genes in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2201868" target="_blank">2201868</a>]<br></li></div>
<li>Introduction The expression of the ompF and ompC porin genes in Escherichia coli is controlled by the response-regulatory protein , OmpR , and the histidine kinase sensor protein , EnvZ [ l-4 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7557323" target="_blank">7557323</a>]<br></li>
<li>A. ( Received 14 April 1989 , and in revised form 28 June 1989 ) The two-component regulatory system , OmpR and EnvZ , in Escherichia coli controls the differential expression of ompF and ompC in response to medium osmolarity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2557454" target="_blank">2557454</a>]<br></li>
<li>The two-component regulatory system , OmpR and EnvZ , in Escherichia coli , controls the expression of ompF and ompC , the genes for the major outer membrane porin proteins , in response to media osmolarity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1648075" target="_blank">1648075</a>]<br></li>
<li>Hall and Silhavy have proposed a model which suggests that OmpR is a positive regulatory factor which activates transcription of both ompF and ompC and that EnvZ is an osmosensor , presumably in the membrane , which somehow modulates the functional state of OmpR to allow the preferential expression of one porin or the other ( 7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2997120" target="_blank">2997120</a>]<br></li>
<li>Characterization of these deletion strains showed that the OmpR protein is necessary for transcription of the ompC and ompF genes , and the EnvZ protein is essential for normal regulation of the ompC and ompF expression , which is affected by the medium osmolarity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3294816" target="_blank">3294816</a>]<br></li>
<li>Alternatively , if there are two types of OmpR whose interconversion is mediatedby EnvZ depending upon the osmolaritoyf the culturemedium , the present OmpR preparation may be enriched in OmpR that m isore favorable for ompF expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3023382" target="_blank">3023382</a>]<br></li>
<li>According to the current model , the EnvZ/OmpR two-compo-nent system regulates the expression of ompF and ompC in response to varying osmolarity ( 20 , 35 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15175316" target="_blank">15175316</a>]<br></li>
<li>Recent studies have established that Lrp exerts its effect on ompF and ompC expression independently of OmpR and EnvZ and that OmpF and OmpC expression is normally regulated by osmolarity in an Irp strain ( 46 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li>
<li>However , EnvZ is required for the expression of both ompF and ompC ( Garrett et al. , 1983 , 1985 ; Mizuno & Mizushima , 1987 ) and therefore the switch that controls porin gene expression can not be simply the presence or absence of a signal from EnvZ to OmpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2557454" target="_blank">2557454</a>]<br></li>
<li>We have shown elsewhere ( Slauch et al. , 1988 ) , and above , that OmpR under certain conditions is capable of activating ompF and ompC expression in the absence of EnvZ .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2557454" target="_blank">2557454</a>]<br></li>
<li>Because the OmpRD55E mutant was active in the absence of EnvZ , we were able to use this mutant to examine wheth-er the cellular level of an active form of OmpR controls the differential expression of ompF and ompC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9422609" target="_blank">9422609</a>]<br></li>
<li>According to this model , the differential expression of ompF and ompC is a direct consequence of the intracellular concentrations of the phosphorylated form of OmpR , OmpR-P .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9422609" target="_blank">9422609</a>]<br></li>
<li>The identiﬁcation and characterization of new OmpR-regulated genes will be valuable for uncovering the general mechanism by which OmpR controls the transcription of its regulon members and for discovering the global role of OmpR in the E. coli stress response .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15175316" target="_blank">15175316</a>]<br></li>
<li>The OmpR protein , a positive regulator of these genes , is involved in the osmoregulation ( Dairi et al. 1985 ; Nara et al. 1984 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3010044" target="_blank">3010044</a>]<br></li>
<li>The ompF gene encoding an outer membrane porin protein is transcribed by Es70 and regulated by OmpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10368152" target="_blank">10368152</a>]<br></li>
<li>The OmpR et al. , protein is a regulator for the and genes ompF ompC coding for outer membrane porin proteins ( Hall & Silhavy , 1981 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3537313" target="_blank">3537313</a>]<br></li>
<li>As illustrated in Fig. 3b , it has been reported that the expressions of ompC and ompF genes for outer membrane porin proteins are regulated by a sensory histidine kinase EnvZ and cognate response-regulator OmpR , where EnvZ regulates the phosphorylation of OmpR ( Castanié - Cornet et al. 2010 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23274360" target="_blank">23274360</a>]<br></li>
<li>The network of genes regulated by the CpxA/CpxR and EnvZ/OmpR systems is not entirely unique ; for example , both systems independently regulate transcription of csgD , ompC and ompF ( Batchelor et al. , 2005 ; Jubelin et al. , 2005 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19432797" target="_blank">19432797</a>]<br></li>
<li>The E. coli osmosensor EnvZ can be altered by mutation to a phosphatase - kinase + species , and this mutant EnvZ hyperphosphorylates its cognate response-regulator , OmpR , ultimately altering transcription from its normal downstream targets , ompF and ompC , as well as other loci ( Slauch et al. 1988 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7883164" target="_blank">7883164</a>]<br></li>
<li>Neither ss , nor EnvZ -- OmpR participate in the osmotic induction of the ibp and dnaK promoters The general stress factor , sS , is responsible for the transcription of a number of genes implicated in the management of hyperosmotic stress ( Hengge-Aronis , 1996a , b ; Loewen et al. , 1998 ) and both translation of the rpoS mRNA and half-life of the protein increase in response to osmotic upshift ( Mufﬂer et al. , 1996a ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10594827" target="_blank">10594827</a>]<br></li>
<li>The regulation of csgDEFG transcription is controlled by at least eight different regulators ( reviewed by Barnhart et al. , 2006 ) , three of which ( EnvZ/OmpR , CpxRA and RcsABCD ) are known to respond to a wide range of envelope stresses ( Barnhart & Chapman , 2006 ; Barnhart et al. , 2006 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21415116" target="_blank">21415116</a>]<br></li>
<li>However , results presented here from wild-type mcbAp and deleted promoters indicate that the sequences needed for OmpR activation lie upstream of 6i 0.1 0 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1906064" target="_blank">1906064</a>]<br></li>
<li>T83I does not need to be phosphorylated in order to compete with wild-type OmpR , when OmpR and T83I/D55A are coexpressed , the b-galactosidase activity expressed from the ompF and ompC promoters remains less than that observed in the parent strains ( Figure 9 , lanes 4 and 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11812125" target="_blank">11812125</a>]<br></li>
<li>With wild-type OmpR , presumably three dimers normally bind to C1-C2-C3 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11812125" target="_blank">11812125</a>]<br></li>
<li>If T83I can bind to ompF promo-ter sequences in vivo , the T83I phenotype should be dominant to wild-type OmpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11812125" target="_blank">11812125</a>]<br></li>
<li>Therefore , it is not surprising that OmpR2 is incapable of activating the ompF4 and ompF18 promoters and the question becomes , how does the wild-type OmpR bind and activate the mutant ompF promoters ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1648075" target="_blank">1648075</a>]<br></li>
<li>DNA footprinting studies with mutant OmpR proteins suggest that a class of mutant proteins termed OmpR2 proteins , which we have shown can no longer negatively regulate ompF ( 39 ) , can only bind to a t Present address : Department of Microbiology and Molecular Genetics , Harvard Medical School , 200 Longwood Ave. , Boston , MA 02115 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1648075" target="_blank">1648075</a>]<br></li>
<li>OmpR has been shown to-bind in a hierarchical fashion to the - 100 to - 40-bp and - 384 to - 351-bp regions of ompF [ 8-111 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7557323" target="_blank">7557323</a>]<br></li>
<li>We introduced the - 70 TA substitution into ompF and found that it did not affect the regulation of ompF ( data not shown ) , suggesting that OmpR could still bind to the mutated -71 to -62 - bp sequence when flanking DNA is present .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7557323" target="_blank">7557323</a>]<br></li>
<li>The hypothesis that the -55 to -42 - bp region functions as a repression site is based on the finding that OmpR derived from an ompR mutant strain ( MH760 , ompR2 ) , that is defective in the os-molarity-induced repression of ompF , does not bind to this region [ 12 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7557323" target="_blank">7557323</a>]<br></li>
<li>That this mutated F2 site can still bind OmpR in the presence of kinase could explain why it has been difficult to obtain single mutations in the ompF regulatory region that affect activation or repression by OmpR in vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9096305" target="_blank">9096305</a>]<br></li>
<li>We propose that OmpR bound at the - 71 to - 62-bp site may also participate in the repression of ompF by potentiating loop formation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7557323" target="_blank">7557323</a>]<br></li>
<li>We propose that under conditions in which the levels of OmpR-phosphate are elevated binding of OmpR to the - 71 to - 62-bp region triggers the transition between ompF activation and ompF repression by facilitating the formation of a repression loop .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7557323" target="_blank">7557323</a>]<br></li>
<li>We would predict that this loop forms between OmpR bound at the site between -384 to -351 and OmpR bound at one of the sites located immediately upstream of the ompF promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8113170" target="_blank">8113170</a>]<br></li>
<li>One interpretation of this result is that the OmpR bound at the site located between - 56 and -41 interacts with OmpR bound at the site located between - 384 to - 351 to form a DNA loop , thereby repressing ompF transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8113170" target="_blank">8113170</a>]<br></li>
<li>In an attempt to understand the effect of envZ on OmpR function , in particular the repression of ompF expression , we examined the levels and electrophoretic properties of OmpR contained in soluble extracts derived from the parent strain MC4100 , the envZ missense strain MH1461 ( envZI1 ) , and the envZ nonsense strain SG477 ( envZ22 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2656631" target="_blank">2656631</a>]<br></li>
<li>The data obtained indicate that OmpR works in both a positive and negative fashion to control the transcription of ompF and this result forms the basis of a model for porin regulation that explains the switch from OmpF to OmpC production in response to increasing medium osmolarity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2557454" target="_blank">2557454</a>]<br></li>
<li>Here , we present evidence t > hat OmpR works both positively and negatively to control the transcription of ompF , and we propose a model that explains the nature of the switch from OmpF and OmpC production in response to medium osmolarity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2557454" target="_blank">2557454</a>]<br></li>
<li>We have previously provided evidence that OmpR works both positively and negatively at the ompF promoter to give the characteristic switch from OmpF to OmpC production with increasing osmolarity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1648075" target="_blank">1648075</a>]<br></li>
<li>Previously , we have presented evidence that OmpR works both positively and negatively at the ompF promoter to control transcription , and we have proposed that osmoregulation of ompF was primarily the result of OmpR repressing transcription in high osmolarity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1648075" target="_blank">1648075</a>]<br></li>
<li>If , on the other hand , the regulation of the porin genes normally involves a form of OmpR that positively activates ompC and , at the same time , negatively regulates ompF , then we would predict that both an OmpF - and an OmpC-constitutive phenotype would often occur simultaneously and would always be dominant , regardless of whether the phenotypes are conferred by a mutant OmpR , a mutant EnvZ , or wild-type proteins in high osmolarity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2557454" target="_blank">2557454</a>]<br></li>
<li>We demonstrate that OmpR produced in cells grown in high-osmolarity medium and in the envZl l mutant strain bound much more tightly to the OmpR-binding region of ompF than did OmpR are formed by the porin proteins OmpF and OmpC ( 22 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2656631" target="_blank">2656631</a>]<br></li>
<li>Genetic studies indicate that in vivo the or @ promoter is more efficiently activated at relatively low OmpR phosphorylation states , whereas at high phospho-OmpR levels , the ompC promoter is activated and the ompF promoter is repressed ( Russo and Silhavy , 1991 ; Slauch and Silhavy , 1989 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8098993" target="_blank">8098993</a>]<br></li>
<li>This implies that there must be both positive and negative sites in the ompF promoter region at which OmpR works to activate or inhibit transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2557454" target="_blank">2557454</a>]<br></li>
<li>This inhibition was reversed by increasing the concentration o $ OmpR , a positive activator required for ompF expression , suggesting that IHF may inhibit ompF transcription by altering how OmpR interacts with the onzpF promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1736095" target="_blank">1736095</a>]<br></li>
<li>OmpR can activate or repress at a single promoter , ompF , depending on its phosphorylation state , the site at which it binds , and the property of the promoter itself ( 1 , 29 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1532173" target="_blank">1532173</a>]<br></li>
<li>Under low-osmolarity conditions , OmpR binds to the region between - 96 and - 63 , thereby facilitating the binding of RNA polymerase to the ompF promoter ( Fig. 6A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8113170" target="_blank">8113170</a>]<br></li>
<li>OmpR can activate or repress the ompF gene depending on osmolarity conditions and the strength of the promoter ( 30 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7836283" target="_blank">7836283</a>]<br></li>
<li>Based on the position of the F2 site relative to the start point of ompF transcription , we postulate that occupancy of the F2 site allows OmpR to activate transcription from the ompF promoter under low osmolarity conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9096305" target="_blank">9096305</a>]<br></li>
<li>Without the ability to loop the DNA , OmpR can not repress ompF transcription in high osmolarity , resulting in constitutive expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1648075" target="_blank">1648075</a>]<br></li>
<li>The concomitant binding of OmpR to these regions results in the formation of a DNA loop that is responsible for repressing ompF expression under high-osmolarity conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8113170" target="_blank">8113170</a>]<br></li>
<li>We propose that this tight hierarchical binding of a transcription factor , OmpR , allows distinct stepwise regulation of ompF and ompC transcription , which minimizes their overlapping expression upon changes in the medium osmolarity to achieve the reciprocal expression of ompF and ompC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16618701" target="_blank">16618701</a>]<br></li>
<li>OmpR is believed to activate transcription of ompF and ompC in a reciprocal manner depending upon the osmolarity of the culture medium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3023382" target="_blank">3023382</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_1156">
OmpR promotes the transcription of ompF in conditions of low osmolarity ; at high osmolarity , OmpR represses transcription of ompF and activates transcription of ompC [ 2,3 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9016718" target="_blank">9016718</a>]<br></label>
<input id="act_similarity_1156", type="checkbox">
<div class="ACT_similar">
<li>Under conditions of low osmolarity , OmpR activates transcription of ompF , while under high osmolarity it represses transcription of ompF and activaies transcription of ompC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9016718" target="_blank">9016718</a>]<br></li></div>
<li>To regulate porin expression , the cell need only regulate the amount of the high osmolarity form , because this form of OmpR both activates ompC and inhibits ompF expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2557454" target="_blank">2557454</a>]<br></li>
<li>Phosphorylated OmpR activates expression of the OmpF porin at low osmolarity , and at high osmolarity represses ompF transcription and activates expression of OmpC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11812125" target="_blank">11812125</a>]<br></li>
<li>Then , as the level of wildtype OmpR increases , the expression of both ompF and ompC is activated ( lane 12 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9422609" target="_blank">9422609</a>]<br></li>
<li>Moreover , if a functional high osmolarity state is supplied , e.g. by the presence of wild-type OmpR , ompC expression is activated and ompF expression is osmoregulated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2557454" target="_blank">2557454</a>]<br></li>
<li>Finally , at high levels of wild-type OmpR , ompF expression is repressed and ompC expression is activated ( lane 13 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9422609" target="_blank">9422609</a>]<br></li>
<li>According to this model , under low-osmolarity conditions ( A ) , OmpR binds to the positive site between -96 and -63 upstream of ompF , thereby activating ompF expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8113170" target="_blank">8113170</a>]<br></li>
<li>Our current hypothesis , based on genetic analysis , is that at low osmolarity , low concentrations of phosphoOmpR ( OmpR-P ) interact with high-affinity sites at ompF to stimulate its expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7568033" target="_blank">7568033</a>]<br></li>
<li>A low-osmolarity environment results in a low-phosphoryiation state of OmpR , which is required to activate ompF expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7651128" target="_blank">7651128</a>]<br></li>
<li>In media of low osmolarity , levels of phosphorylated OmpR are low and ompF expression is activated ( 52 , 53 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8002608" target="_blank">8002608</a>]<br></li>
<li>For example , the OmpR protein , which regulates porin gene expression in E. coli , binds with different afﬁnities to three or four sites in the ompC and ompF promoters , respectively ( 13 , 14 , 25 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10913075" target="_blank">10913075</a>]<br></li>
<li>Because of the genera-lity of this phenotype , we suggest that it reflects the normal regulation of the porin genes and that OmpR functions in both a positive and a negative fashion to regulate the expression of ompF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2557454" target="_blank">2557454</a>]<br></li>
<li>Phospho-OmpR binds to the promoters of the porin genes to regulate the expression of ompF and ompC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7568033" target="_blank">7568033</a>]<br></li>
<li>Mutations to the corresponding OmpR residue ( R209 ) responsible for DNA backbone interactions impair its ability to stimulate expression of reporter genes fused to ssrA , ompF , and ompC promoters [ 28 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22291906" target="_blank">22291906</a>]<br></li>
<li>Our results indicate that all three of the OmpR binding sites at the ompC promoter must be ® lled in order to activate gene expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11812125" target="_blank">11812125</a>]<br></li>
<li>As a result , OmpR is now able to occupy the activating sites upstream of ompC , leading to the expression of ompC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8113170" target="_blank">8113170</a>]<br></li>
<li>In addition , this form of OmpR negatively regulates ompF expression by binding to a site ( s ) ( negative site ) in the ompF promoter region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2557454" target="_blank">2557454</a>]<br></li>
<li>These results further clarify the mechanism by which OmpR negatively regulates ompF expression , suggesting a model in which OmpR forms a repressive loop in the ompF promoter region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1648075" target="_blank">1648075</a>]<br></li>
<li>In order to substantiate various aspects of our model for differential regulation of the ompF promoter , we have isolated and characterized cis-acting mutations that affect negative regulation by OmpR and , accordingly , confer constitutive expression on ompF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1648075" target="_blank">1648075</a>]<br></li>
<li>In any event , these results suggest that an increase in the phosphor-ylation of OmpR was correlated with enhanced expression of ompC and possibly with the repression of ompF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2160945" target="_blank">2160945</a>]<br></li>
<li>In order to circumvent this problem , we developed a way to phosphorylate OmpR in vivo to further examine the differential expression of ampC and ompF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2160945" target="_blank">2160945</a>]<br></li>
<li>In other words , in an envZ473 background , ompF expression is always repressed by OmpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1648075" target="_blank">1648075</a>]<br></li>
<li>In this paper , we show that the purified OmpR binds specifically to the ompF and ompC promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3023382" target="_blank">3023382</a>]<br></li>
<li>OmpR binds to multiple sites in the ompF and ompC promoters , whose afﬁnities are quite different and are affected by the presence of ﬂanking sites ( 18 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11976297" target="_blank">11976297</a>]<br></li>
<li>The phosphorylation of OmpR at Asp-55 enhances its ability to-bind to the regulatory sequences upstream of the ompC and ompF promoters ( 16 , 17 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11158569" target="_blank">11158569</a>]<br></li>
<li>The current model for ompF repression proposes that OmpR binds in a hierarchical fashion to form a repression loop [ 9,10 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7557323" target="_blank">7557323</a>]<br></li>
<li>The phosphorylated form of OmpR binds to regulatory sites upstream of ompF and ompC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11601842" target="_blank">11601842</a>]<br></li>
<li>The results indicated that the phosphorylated form of the OmpR protein binds to both the ompF and ompC promoter DNAs more efficiently ( about 10-fold ) than thenonphosphorylated form does .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2668281" target="_blank">2668281</a>]<br></li>
<li>The model also predicts that binds to additional sites OmpR upstream of ompF , causing ompF expression to be repressed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8113170" target="_blank">8113170</a>]<br></li>
<li>The OmpR regulator binds to both the ompC and ompF promoters , but at low concentrations of OmpR-P , ompF is activated , while at high concentrations of OmpR-P , ompF is repressed and ompC is activated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15466019" target="_blank">15466019</a>]<br></li>
<li>The DNase I footprinting studies indicate that when OmpR is bound solely at the C1 site , it is not suf ® cient to activate ompC transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11812125" target="_blank">11812125</a>]<br></li>
<li>The purified OmpR wasalsoshown tobe functionally active in transcription in vitro ; OmpR activates the expression of ompF and inhibits the transcription of ompC from the most distal of its three tandem promoters , P3 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3023382" target="_blank">3023382</a>]<br></li>
<li>The major effect of ipeX on ompC expression was exerted posttranscriptionally , whereas the truncated OmpR protein reduced ompC transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16385048" target="_blank">16385048</a>]<br></li>
<li>There is evidence that OmpR interacts directly with the alpha subunit of RNA polymerase in the process of negatively regulating ompF transcription ( 9 , 21 , 38 ) , suggesting that both OmpR and RNA polymerase are present in the looped complex that results in repression of ompF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1648075" target="_blank">1648075</a>]<br></li>
<li>RNA levels for genes regulated by OmpR did not change ( ompC ) or were decreased ( fadL and ompF ) rather than showing an increase , like those for the chemotaxis and ﬂagellum genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12620868" target="_blank">12620868</a>]<br></li>
<li>Although the level of OmpR ~ P in vivo has not been determined , it is presumed that the [ OmpR ~ P ] increases with increasing osmolarity , promoting activation and then repression of ompF and activation of ompC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19432797" target="_blank">19432797</a>]<br></li>
<li>I H F inhibition of ompF transcription was reversed b increased amounts of OmpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1736095" target="_blank">1736095</a>]<br></li>
<li>OmpR mutants have been widely studied to OmpR mutants that fail to activate transcription of ompC , but that can repress transcription of ompF , belong to the class of activation mutants .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9016718" target="_blank">9016718</a>]<br></li>
<li>Using an in uitro transcription system , we also show that thepurified OmpR activates thexpression of ompF and inhibits the transcriptionof ompC from promoterP3 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3023382" target="_blank">3023382</a>]<br></li>
<li>Phosphorylated OmpR positively regulates ompC transcription , and it negatively regulates ompF transcription ( 47 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8002608" target="_blank">8002608</a>]<br></li>
<li>However , increased phosphorylation of OmpR results in activation of transcription only at certain promoters , such as ompC , while other genes , such as ompF , are downregulated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11717281" target="_blank">11717281</a>]<br></li>
<li>However , when the concentration of activated OmpR increases , additional binding of these molecules results in increased transcription of ompC and repression of ompF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14665678" target="_blank">14665678</a>]<br></li>
<li>These results suggest that OmpR may interact with the flanking sequences contributed by fragment I , fragment III , or both and that the OmpR of the envZJJ strain binds differently to the ompF upstream sequence than does the OmpR of the parent strain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2656631" target="_blank">2656631</a>]<br></li>
<li>Studies of the activation of malT by CRP and of ompF by OmpR give clues to how more distant activators can regulate RNA polymer-ase ( 81 , 88 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1943993" target="_blank">1943993</a>]<br></li>
<li>OmpR acts as a positive tivated , while the ompF geneis repressed ( 7 , 12 , 18 , 19 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8175665" target="_blank">8175665</a>]<br></li>
<li>OmpR is a transcriptional-regulator for ompF , and consequently loss of susceptibility by the ompR strain most likely is related to this function .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26209678" target="_blank">26209678</a>]<br></li>
<li>OmpR is a known transcriptional-regulator of ompF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26209678" target="_blank">26209678</a>]<br></li>
<li>Although we do not have evidence for the interaction between OmpR proteins bound to the upstream and downstream regions , it is possible that they associate to form a loop , as in ompF regulation ( 38 , 47 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7642497" target="_blank">7642497</a>]<br></li>
<li>Since we found that ArcB is capable of phosphorylating the non-cognate OmpR under certain conditions , we then wondered whether or not ArcB is somehow implicated in the regulation of ompC and ompF under physiological conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7957084" target="_blank">7957084</a>]<br></li>
<li>We took advantage of this strong phenotype to isolate cis mutations in the ompF promoter that overcome negative regulation by OmpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1648075" target="_blank">1648075</a>]<br></li>
<li>We propose that this affects looping of the DNA required for negative regulation of the ompF promoter by OmpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1648075" target="_blank">1648075</a>]<br></li>
<li>How looping of the ompF promoter DNA by OmpR results in negative regulation of transcription is unclear .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1648075" target="_blank">1648075</a>]<br></li>
<li>These results are again in agreement with osmotic regulation of ompF being mediated directly by OmpR ( 22 , 23 , 42 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8002608" target="_blank">8002608</a>]<br></li>
<li>These results are consistent with the proposal that osmoregulation of ompF expression is the result of negative regulation by OmpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1648075" target="_blank">1648075</a>]<br></li>
<li>We have presented evidence that osmoregulation of ompF is the result of negative regulation by OmpR ( 39 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1648075" target="_blank">1648075</a>]<br></li>
<li>Model for the regulation of ompF by OmpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8113170" target="_blank">8113170</a>]<br></li>
				</div></ul><br>
			<label class="collapse_DOM" for="_2"><font size="5px">OmpR protein domains</font></label>
				<input id="_2" type="checkbox">
				<div class="DOM"><br><ul><li>P22 repressor , residues 24-28 , are predicted to be in an a-helical region as are theOmpR protein residues 89-90 and 95 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6292199" target="_blank">6292199</a>]<br></li>
<li>As proposed previously ( 6 , 7 ) , theOmpR protein is considered to contain at least two functional domains : one is responsible for interaction with the ompF and ompC promoter DNAs and the otherfor interaction with the EnvZ protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2668281" target="_blank">2668281</a>]<br></li>
<li>T87A and T87C substitutions in CheY retain some ability to mediate clockwise ¯ agellar rotation , and in the presence of a D13K substitution , these two substitutions nearly restore clockwise rotation .38 The crystal structures of several phosphorylated response regulators have been solved , and they suggest a role for the conserved hydroxyl and aromatic residues in signal propagation ,20 ± 23 for a review , see Robinson et al. 42 Upon phosphoryl-ation , the conserved threonine ( T83 in OmpR ) moves towards the phosphorylated aspartate ( D55 ) , allowing the tyrosine ( Y102 ) to rotate inwards and ® ll the space where T83 used to reside ( see Figure 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11812125" target="_blank">11812125</a>]<br></li>
<li>Radioactively phosphorylated EnvZ * was purified and then incubated with OmpR in the absence of ATP in a buffer containing 5 mM M & l2 and 50 mMKC1 at room temperature for 30 s. Subsequently ,0.1 mM nonlabeled ATP was addedt Th ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2668281" target="_blank">2668281</a>]<br></li>
<li>Our results suggest that OmpR undergoes a specific conformational change when phosphorylated and demonstrate that residues Arg-192 to Arg-199 are surface exposed , consistent with the hypothesis that this region contacts the a subunit of RNA polymerase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7568033" target="_blank">7568033</a>]<br></li>
<li>Escherichia coli positive regulator OmpR has a large loop structure at the putative RNA polymerase interaction site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11812125" target="_blank">11812125</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10648521" target="_blank">10648521</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11601842" target="_blank">11601842</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26209678" target="_blank">26209678</a>]<br></li>
<li>OmpR controiied transcription is thought to function through a direct interaction with the asubunit of RNA polymerase [ 19-211 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9016718" target="_blank">9016718</a>]<br></li>
<li>Suppressor mutations in rpoA suggest that OmpR controls transcription by direct interaction with the alpha subunit of RNA polymerase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8098993" target="_blank">8098993</a>]<br></li>
<li>For instance , transcriptional activation by OmpR involves interaction with the α subunit of RNA polymerase ( 171 , 172 ) , whereas PhoB interacts with σ 70 ( 173 , 174 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10966457" target="_blank">10966457</a>]<br></li>
<li>For transcription-activation , the OmpR protein binds to the OmpR site generally located upstream from the promoter 335 , and interacts directly with a subunit of RNA polymerase [ 19,20 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10802181" target="_blank">10802181</a>]<br></li>
<li>E-mail address of the corresponding author : KenneyL@ohsu.edu 0022-2836/02 / 040497 ± 15 $ 35.00 / 0 # 2002 Academic Pres Results T83I can not activate transcription in vivo We ® rst determined the phenotype that resulted from the threonine to isoleucine substitution at amino acid residue 83 of OmpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11812125" target="_blank">11812125</a>]<br></li>
<li>Interestingly , in the wild-type background , a T83A mutation in ompR results in an OmpF ÿ OmpC phenotype .41 This implies that T83 is a critical residue for OmpR function , even if some substitutions are less deleterious than the isoleucine described here .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11812125" target="_blank">11812125</a>]<br></li>
<li>For example , a substitution of Y102 for cysteine results in constitutive activation of OmpR ,44 suggesting that restriction of the movement of the conserved tyrosine residue contributes to the mutant phenotype of both CheY-T87I and OmpRT83I .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11812125" target="_blank">11812125</a>]<br></li>
<li>Likewise , OmpR , CheY , CheB , and NRI belong to a structurally conserved class of effector molecules in which the aspartic acid residues represented by Asp-12 and Asp-55 of OmpR are invariant ( 41 , 43 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2160945" target="_blank">2160945</a>]<br></li>
<li>Such an arginine residue is conserved at structurally equivalent positions in many members of the OmpR/PhoB family , with the exception of KdpE , OmpR , and DrrB ( Fig .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22291906" target="_blank">22291906</a>]<br></li>
<li>Rev. 4 9 , 1-32 either Fa or Fc indicates that the central C residue in Fb 623-630 plays an essential role for OmpR recognition .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2553720" target="_blank">2553720</a>]<br></li>
<li>In particular , residues 83-89 in the OmpR protein , -- 89 Ala-Lys-Gly-Glu-Gl-u-Vawle , re fountdo be similar to residues = i 9 25 19-25 , T '' y - i t hr-Ala-Lys-Asp-Leu-Gl-Vanl , he phage A cro protein ( double underline denotes matching residue ) ( 37 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6292199" target="_blank">6292199</a>]<br></li>
<li>89 Residues 89-95 in the OmpR protein , - rg Val-Asp - `` A - Ile-Val-95 G - '' ly-Leu , were foundto be similar in either charge or residue when compared with residues 24-30 in the P22 repressor , 24 - Lys-Met-Leu-Gly - '' '' - Val-Gly-Val ( single underline denotes resi - '' due having similar chemical property and double underline denotes matching residue ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6292199" target="_blank">6292199</a>]<br></li>
<li>Wurtzel et al. ( 32 ) noted that in OmpR there was a region from residues 207-263 where there were almost exclusively basic residues , having a net charge of +12 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2984198" target="_blank">2984198</a>]<br></li>
<li>Activation by MerR requires residue 323 ( 7 ) , and both residues 322 and 323 have been identiﬁed as important for OmpR activation ( 30 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10852886" target="_blank">10852886</a>]<br></li>
<li>Residues within this region of a-CTD have been implicated in interactions with MerR ( residues 311 and 323 ) ( 7 ) and OmpR ( resi-dues 322 and 323 ) ( 32 ) , suggesting that this may be a common region for a-CTD contacts with activators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11073923" target="_blank">11073923</a>]<br></li>
<li>We were surprised to find that the mass spectrum of phosphorylated OmpR actually contained three peaks .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7568033" target="_blank">7568033</a>]<br></li>
<li>The mass spectrometry data indicate that OmpR is phosphorylated on only one residue , with an accompanying increase in mass of about 83 Da .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7568033" target="_blank">7568033</a>]<br></li>
<li>This phosphate group is transferred subsequently to the conserved Asp-55 residue of OmpR forming phosphorylated OmpR ( OmpR-P ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11158569" target="_blank">11158569</a>]<br></li>
<li>It has been known that EnvZ is the osmo sensor able to sense external osmolarity , where it undergoes autophosphorylation at His residue site and transfers the phosphate group to Asp residue site of OmpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23274360" target="_blank">23274360</a>]<br></li>
<li>From these results we conclude that ArcB has an additional phosphodonor site containing a crucial histidine ( His7 17 ) in its C-terminal region , from which a phosphoryl group can be transferred to the aspartate residue ( Asp55 ) in OmpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7957084" target="_blank">7957084</a>]<br></li>
<li>These are ( see Figure 1 ) : an aspartate residue that is the site of phosphorylation ( D55 in OmpR ) , two aspartate residues that coordinate a magnesium ion ( D11 and D12 ) , a lysine residue that interacts with the phosphoryl group ( K105 ) , an aromatic residue that functions as a rotamer ( Y102 ) , and a side-chain hydroxyl residue ( T83 ) that also interacts with the phosphoryl group .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11812125" target="_blank">11812125</a>]<br></li>
<li>Finally , primary characterization of the wild-type OmpR and EnvZ revealed several points significant to their function and possible interaction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2997120" target="_blank">2997120</a>]<br></li>
<li>Interestingly , EnvZ does not require the integrity of this coiled-coil region for autophosphorylation , phosphotransfer , or phosphatase activity in vitro , suggesting that the points of contact with OmpR do not lie in this region ( 33 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11325944" target="_blank">11325944</a>]<br></li>
<li>EnvZ is autophosphorylated when incubated with ATP , and readily transfers the phosphate group to OmpR [ 12-141 ; EnvZ also exhibits phosphatase activity for phospho-OmpR [ 12,13 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8522138" target="_blank">8522138</a>]<br></li>
<li>Interestingly , EnvZ also possesses a phosphatase activity that removes the phosphoryl group from the phosphorylated form of OmpR ( OmpR-P ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15175316" target="_blank">15175316</a>]<br></li>
<li>An EnvZ mutant with constitutive phosphatase activity is not able to elicit biofilm formation To further explore any role of EnvZ in this signalling pathway , a mutant in the cytoplasmic domain of EnvZ that abolishes the kinase activity and preserves the phosphatase activity was generated , as EnvZ is the cognate sensor for OmpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25667010" target="_blank">25667010</a>]<br></li>
<li>The interference by the phosphoryl transfer domains present in GST-Bc or GST-Zc with UhpA or OmpR action was highly speciﬁc : expression of the UhpB domain had no effect on OmpR function , or conversely , expression of the EnvZ domain had no effect on UhpA function .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11325944" target="_blank">11325944</a>]<br></li>
<li>In response to environmental signals , EnvZ autophosphorylates a histidyl residue in the cytoplasmic domain and transfers this phosphoryl group to an aspartyl residue on OmpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8002608" target="_blank">8002608</a>]<br></li>
<li>The partially purified form of OmpR was used in this study since , unlike purified forms of OmpR , it retained high affinity binding and did not require the addition of EnvZ and ATP to stimulate binding h101 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7557323" target="_blank">7557323</a>]<br></li>
<li>The cysteine 354 and 277 residues of Salmonella enterica serovar Typhi EnvZ are determinants of autophosphorylation and OmpR phosphorylation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25667010" target="_blank">25667010</a>]<br></li>
<li>If the OmpR and AraC families were related to the LysR-family , then helix-turn-helix predictions would be expected near the amino termini .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3413113" target="_blank">3413113</a>]<br></li>
<li>The transcription factors NarL and OmpR each contain a recognition helix ( magenta ) , whereas CheB contains a catalytic triad ( magenta spheres ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10966457" target="_blank">10966457</a>]<br></li>
<li>Structure/function relationships in OmpR and other winged-helix transcription factors .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15175316" target="_blank">15175316</a>]<br></li>
<li>T h e P sheet is unique to the OmpR subfamily T h e relationship of OmpRc with the winged H T H family of proteins has helped to define , by analogy , the function of some of the structural elements present in the OmpRc structure .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9016718" target="_blank">9016718</a>]<br></li>
<li>We thus divided the repertoire of helix-turn-helix TFs into three categories : ( a ) classical HTH ( cHTH ) proteins with a tight loop between H-1 and H-2 , and no additional elements in the C-terminus , e.g. the GalR/LacI , family ( see Table 1 ) ; ( b ) the winged HTH proteins ( wHTH ) containing the C-terminal - hairpin and , in some proteins additional - sheets in between H-1 and H-2 , such as the OmpR , family ; and ( c ) the miscellaneous HTH ( mHTH ) , an heterogeneous collection of proteins that differ from the typical HTH in certain features , such as shortened helices or the incorporation into other super-structures , e.g. Rob in E. coli ( Tobes and Ramos , 2002 ) , or members of the enhancer-binding proteins ( EBP ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15556475" target="_blank">15556475</a>]<br></li>
<li>One potential candidate is Thr-87 , which is located near the active site in the CheY structure and is conserved in OmpR ( Thr-83 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7568033" target="_blank">7568033</a>]<br></li>
<li>OmpR may cooperatively bind to this site and provide contact surfaces that are required for OmpR-OmpR interactions involved in the formation of the repression loop .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7557323" target="_blank">7557323</a>]<br></li>
<li>1 4 6 , 23-43 base pairs ( presumably involved with OmpR recognition ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2553720" target="_blank">2553720</a>]<br></li>
<li>This result , together with our genetic analysis of tppB transcription , supports the idea that the phosphorylation of OmpR plays an important role in the transcriptional activation of tppB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15175316" target="_blank">15175316</a>]<br></li>
<li>The mutant is phosphorylated in vitro , but non-chemotactic in vivo , indicating a block in activation subsequent to phosphorylation .24 We made the equivalent mutation in OmpR in order to characterize its properties in vivo and in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11812125" target="_blank">11812125</a>]<br></li>
<li>Hall and Silhavy have suggested that the carboxy terminus of OmpR contains A dimerization site which allows for the modulation of porin gene transcriptional activation , the monomer preferentially activating OmpF production and the dimer activating OmpC ; they further suggest that the functional state of the OmpR protein is controlled in some manner by the EnvZ protein ( 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2997120" target="_blank">2997120</a>]<br></li>
<li>Depending on the respective activation of the EnvZ/OmpR and Cpx pathway , a certain competition could occur and would moderately disturb OmpR-dependent csgD activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15743952" target="_blank">15743952</a>]<br></li>
<li>In all cases , EnvZ-dependent activation of PhoB required the absence of OmpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15755952" target="_blank">15755952</a>]<br></li>
<li>Results and discussion Strticture determination T h e structure of OmpRc , a fragment of OmpR corresponding to residues Ala130-Ala239 , was determined by the multiwavelength anomalous diffraction ( MAD ) method using crystals of the selenomethionine substituted protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9016718" target="_blank">9016718</a>]<br></li>
<li>Materials and methods Purification of wild-type and selenomethionylprotein The 12 kDa DNA-bindingdomain of OmpR from E. col ; , OmpRc , composed of the C-terminal residues Ala130-Ala239 and an initiator methionine residue , was overexpressed using a T7 expression vector and purified from E. col ; BL21 cells .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9016718" target="_blank">9016718</a>]<br></li>
<li>It is , however , reasonable to assume that sequencIeis involved in OmpR binding to these regions , since a much longer region was protected from DNase Idigestion in ompF containing three copies of the sequenceI incomparisonwith ompC containing only one copy of the sequence I. Various DYA-bindingproteinshavebeen described ( 31-35 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3023382" target="_blank">3023382</a>]<br></li>
<li>In the 3D structure of OmpR and PhoB , a protruding loop located between the two helices proved to be part of the RNA-polymerase-binding region ( 28 , 31 , 32 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15197250" target="_blank">15197250</a>]<br></li>
<li>Indeed , mutant variants of the OmpR binding site C1 in the ompC regulatory region indicate that a G residue at this position is essential for independent OmpR binding ( unpublished data ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9096305" target="_blank">9096305</a>]<br></li>
<li>It should be noted that one of the OmpR binding sites infadL is also located in the region between-40 and - 80 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8413182" target="_blank">8413182</a>]<br></li>
<li>The black box shows the putative phospho-OmpR binding region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10802181" target="_blank">10802181</a>]<br></li>
<li>These data provide the evidence that the phospho-OmpR binding region is present within the 351 to 35 region of bolA1p promoter , which contains the putative OmpR binding region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10802181" target="_blank">10802181</a>]<br></li>
<li>A dominant mutation in Escherichia coli OmpR lies within a domain which is highly conserved in a large family of bacterial regulatory proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2656631" target="_blank">2656631</a>]<br></li>
<li>In order to clarify whether the OmpR protein does indeed bind to the DNA via either one of these two regions , further biochemical studies will be necessitated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6292199" target="_blank">6292199</a>]<br></li>
<li>Previously , we used dimethyl sulfate and hydroxyl radical footprinting analyses to define the regions of contact between OmpR and the DNA ( 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9096305" target="_blank">9096305</a>]<br></li>
<li>Identification of the DNA-binding domain of the OmpR protein required for transcription-activation of the ompF and ompC genes of Esch-erichia coli by in vivo footprinting .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1695892" target="_blank">1695892</a>]<br></li>
<li>Tsung , K. , Brissette , R.E. , and Inouye , M. ( 1989 ) Identification of the DNA-binding domain of the OmpR protein required for transcriptional activation of the ompF and ompC genes of Escherichia coil by in vivo DNA footprinting .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596446" target="_blank">8596446</a>]<br></li>
<li>Identification of the DNA-binding domain of the OmpR protein required for transcriptional activation of the ompF and ompC genes of Esche-richia coli by in vivo DNA footprinting .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8113170" target="_blank">8113170</a>]<br></li>
				<label class="DOM_collapse_similarity" for="dom_similarity_64">
Identiﬁcation of the DNA-binding domain of the OmpR protein required for transcriptional activation of the ompF and ompC genes of Escherichia coli by in vivo DNA foot printing .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15466019" target="_blank">15466019</a>]<br></label>
<input id="dom_similarity_64", type="checkbox">
<div class="DOM_similar">
<li>Identification of the DNA-binding domain of the OmpR protein required for transcriptional activation of the ompF and ompC genes of Escherichia coli by in vivo DNA footprinting .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2203749" target="_blank">2203749</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1370290" target="_blank">1370290</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1310090" target="_blank">1310090</a>]<br></li>
<li>Identi ® cation of the DNA-binding domain of the OmpR protein required for transcriptional activation of the ompF and ompC genes of Escherichia coli by in vivo DNA footprinting .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11601842" target="_blank">11601842</a>]<br></li>
<li>Identificatio of the DNA-binding domain of the OmpR protein required for transcriptional activation of the ompF and ompC genes of Escherichia coli by in vivo DNA footprinting .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1943993" target="_blank">1943993</a>]<br></li>
<li>Identiﬁcation of the DNA-binding domain of the OmpR protein required for transcriptional activation of the ompF and ompC genes of Escherichia coli by in vivo DNA footprinting .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8002608" target="_blank">8002608</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9371449" target="_blank">9371449</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16077119" target="_blank">16077119</a>]<br></li></div>
<li>OmpR is a DNA-binding protein that interacts may interactselectively with one region over another with the with the upstream promoter region of the ompF and ompC ultimate outcome affecting ompF expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8175665" target="_blank">8175665</a>]<br></li>
<li>Comprehensive analysis of OmpR phosphorylation , dimerization , and DNA-binding supports a canonical model for activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25161193" target="_blank">25161193</a>]<br></li>
<li>Moreover , the mutations are in the region that by deletion analysis is vital for ompF activation and OmpR DNA-binding in general ( 15 , 16 , 24 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1648075" target="_blank">1648075</a>]<br></li>
<li>We envision that the OmpR molecules interact and loop the DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1648075" target="_blank">1648075</a>]<br></li>
<li>Structural comparison of the PhoB and OmpR DNA-binding/transactivation domains and the arrangement of PhoB molecules on the phosphate box .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11976297" target="_blank">11976297</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11489853" target="_blank">11489853</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10986267" target="_blank">10986267</a>]<br></li>
<li>The x-ray structures of DrrD and DrrB , OmpR supported by recent structural studies of the dimerized N-terminal domain of ArcA ( 20 ) and PhoB ( 21 ) , suggesting an intriguing insight into the role of phosphorylation and subsequent interaction between OmpR-P molecules through the N-terminal 4 - 5 - 5 region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16618701" target="_blank">16618701</a>]<br></li>
<li>Mutational analyses suggest that interaction with the C-terminal domain of the asubunit of RNA polymerase is mediated by the C-ter-mina1 domain of OmpR [ 19-211 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9016718" target="_blank">9016718</a>]<br></li>
<li>Thus the relatively small C-terminal domain of OmpR must accommodate numerous inter - and intramolecular interactions : a regulatory interaction with the N-terminal domain , dimerization or perhaps oligomerization with itself , DNA-binding , and interaction with the asubunit of RNA polymerase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9016718" target="_blank">9016718</a>]<br></li>
<li>The binding of OmpR to these sites facilitates the interaction between OmpR and the C-terminal domain of the - subunit of RNA polymerase and results in transcriptional activation ( 39 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15175316" target="_blank">15175316</a>]<br></li>
<li>homologues from Thermotoga maritima , revealed that the N-terminal acts with the C-terminal domain is highly conserved among the OmpR / domain of the molecule interacts with its C-terminal domain ( 18 , 19 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16618701" target="_blank">16618701</a>]<br></li>
<li>These results may imply that the OmpR protein has a domain structure in which the N-terminal portion and the C-terminal portion play different roles in osmoregulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3010044" target="_blank">3010044</a>]<br></li>
<li>These results indicate that the N-terminal domain is required for OmpR to bind to the F1a site in the FIGURE 1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16618701" target="_blank">16618701</a>]<br></li>
<li>The variable architectures of effector domains are illustrated by ( b ) the C-terminal domain of E. coli OmpR , ( c ) E. coli NarL , and ( d ) S. typhimurium methylesterase CheB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10966457" target="_blank">10966457</a>]<br></li>
<li>The OmpR has a single C-terminal domain and ca. 230 amino acids .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8780507" target="_blank">8780507</a>]<br></li>
<li>the activi6 of the C-terminal domain of OmpR is regulated by the N-terminal domain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9016718" target="_blank">9016718</a>]<br></li>
<li>The C-terminal domain of OmpR ( OmpRc ) was purified as described previously ( 16 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16618701" target="_blank">16618701</a>]<br></li>
<li>Results : We have determined the crystal structure of the Escherichia coli OmpR C-terminal domain at 1.95 A resolution .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9016718" target="_blank">9016718</a>]<br></li>
<li>First , oligomerization can be impaired by alterations in the domain of OmpR of E. coli that is homologous to the N-terminal domain of NRI ( 15 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7730282" target="_blank">7730282</a>]<br></li>
<li>For instance , residues lying in the HTH turn region of PhoB and OmpR were suggested to contact RNAP s and a subunits respectively ( Makino et al. , 1996 ; Kenney , 2002 ) ; the AR region of Fis protein , localized in the turn between helices B and C , adjacent to the HTH motif in the DNA-binding domain , was shown to be critical for contact with the RNAP a-CTD ` 273 determinant ' ( Gosink et al. , 1996 ; Bokal et al. , 1997 ; McLeod et al. , 1999 ; Cheng et al. , 2000 ; Aiyar et al. , 2002 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15255893" target="_blank">15255893</a>]<br></li>
<li>Nevertheless , full-length ArcA-P has been reported to form oligomers ( 19 ) , as have both the isolated N-terminal and C-terminal domains ( 16 ) , suggesting that although the minimal DNA-binding unit is likely a dimer , as demonstrated for PhoB and OmpR ( 20 -- 22 ) , oligomerization beyond a dimer may explain binding to multiple direct repeats .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25161193" target="_blank">25161193</a>]<br></li>
<li>Many of the residues in OmpR that have been identified from mutagenesis studies as being important for DNA-binding are conserved or are conservatively substituted in other family members .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9016718" target="_blank">9016718</a>]<br></li>
<li>For T215 of KdpE , variants with mutations at equivalent residues in OmpR and PhoB are also defective in DNA-binding due to loss of H-bonding with the DNA backbone [ 28,49 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22291906" target="_blank">22291906</a>]<br></li>
<li>This is an especially important issue , since no standard DNA-binding motif can be recognized in OmpR member of this subfamily of or any response regulators ( 1 , 39 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7568033" target="_blank">7568033</a>]<br></li>
<li>These DNA-binding domains can be subdivided into three major families , represented by OmpR , NarL , and NtrC ( 14 , 7 , and 4 members , respectively , in E. coli ) ( 4 , 63 , 133 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10966457" target="_blank">10966457</a>]<br></li>
<li>Both the C-terminal domain and phosphorylated OmpR have been shown to bind , as dimers to a twenty base pair region of DNA containing a tandem ten base pair repeat [ 4,6,12-141 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9016718" target="_blank">9016718</a>]<br></li>
<li>Therefore , we attempted to use a longer fragment to detect the DNA-binding activity of OmpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2656631" target="_blank">2656631</a>]<br></li>
<li>Thus , EnvZ appears to promote optimal DNA-binding activity of OmpR to fragment Ila ( Fig. 4 and 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2656631" target="_blank">2656631</a>]<br></li>
<li>The C-terminal cytoplasmic domain of EnvZ is capable of autophosphorylation and of transferring the phosphate to the DNA-binding protein , OmpR ( 3 , 12 , 20 , 22 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8113170" target="_blank">8113170</a>]<br></li>
<li>EnvZ phosphorylates OmpR , a cytoplasmic DNA-binding protein , on an aspartyl residue .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7568033" target="_blank">7568033</a>]<br></li>
<li>The phosphoryl group is then transferred from EnvZ to an aspartate residue in OmpR , a cytoplasmic DNA-binding protein ( 4-8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7568033" target="_blank">7568033</a>]<br></li>
<li>ToxR , however , differs from the OmpR class of response proteins in at least two fundamental ways : it is membrane bound , and like CadC , the conserved DNA-binding domain is at the amino terminus ( 23 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1370290" target="_blank">1370290</a>]<br></li>
<li>OmpR belongs to a subfamily of at least 50 response regulators with homologous C-terminal DNA-binding domains of approximately 98 arnino acids .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9016718" target="_blank">9016718</a>]<br></li>
<li>OmpR is one of the most extensively studied members of a subfamily of more than 50 response regulator protein characterized by sequence similarity in their C-terminal DNA-binding domains .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9016718" target="_blank">9016718</a>]<br></li>
<li>Relationshipto other members of the OmpR/PhoB family of transcriptionalregulators T h e DNA-binding domain of OmpR exhibits sequence similarity over its entire length with a family of over 50 different transcriptional regulators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9016718" target="_blank">9016718</a>]<br></li>
<li>There is significant variation in the degree of sequence similarity among different DNA-binding domains , with members of the family exhibiting from 20 % to 65 % sequence identity with the 98 amino acid DNA-binding domain of OmpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9016718" target="_blank">9016718</a>]<br></li>
<li>Although a major advance has been made with solving the crystal structure of the carboxy terminal end of OmpR ,77,78 co-crystals with DNA-binding sites have yet to be generated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11601842" target="_blank">11601842</a>]<br></li>
<li>The crystal structure of the Escherichia coli C-terminal DNA-binding domain of OmpR ( OmpRc ) , determined at 1.95 A resolution , has revealed that OmpRc belongs to the ` winged-helix-turn-helix ` family of DNA-binding proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9016718" target="_blank">9016718</a>]<br></li>
<li>TorR belongs to the OmpR response-regulator family , in which the C-terminal domain includes a winged helix -- turn -- helix DNA-binding structure ( 28 -- 30 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15197250" target="_blank">15197250</a>]<br></li>
<li>Crystal structures of the DNA-binding domain of OmpR ( 167 , 168 ) deﬁne a novel subclass of winged-helix transcription factors ( Figure 4b ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10966457" target="_blank">10966457</a>]<br></li>
<li>The DNA-binding domain of OmpR : crystal structures of a winged helix transcription factor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11601842" target="_blank">11601842</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11812125" target="_blank">11812125</a>]<br></li>
				<label class="DOM_collapse_similarity" for="dom_similarity_109">
Martinez-Hackert , E. , and Stock , A.M. ( 1997b ) The DNA-binding domain of OmpR : crystal structure of a winged helix transcription factor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12123454" target="_blank">12123454</a>]<br></label>
<input id="dom_similarity_109", type="checkbox">
<div class="DOM_similar">
<li>The DNA-binding domain of OmpR : crystal structure of a winged helix transcription factor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15175316" target="_blank">15175316</a>]<br></li></div>
<li>Again , while the OmpR DNA-binding domain clusters with the homologous PetR domain , its receiver module is sandwiched in between those of the algal chloroplast proteins and PhoB of B. subtilis in Fig. 2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7699720" target="_blank">7699720</a>]<br></li>
<li>Strudure and tertiary fold T h e 12kDa DNA-binding domain of E. coli OmpR used in this study consists of the C-terminal residues Ala130-Ala239 and an initiator methionine residue ( the numbering used in the text refers to intact OmpR ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9016718" target="_blank">9016718</a>]<br></li>
<li>A novel type of mutation in the phosphorylation domain of the activator protein , OmpR , results in a defect in its phosphorylation-dependent DNA-binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7641688" target="_blank">7641688</a>]<br></li>
<li>C-terminal DNA-binding stimulates N-terminal phosphorylation of the outer-membrane-protein regulator OmpR from Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11739766" target="_blank">11739766</a>]<br></li>
<li>This domain is the only region conserved between OmpR and NRI , suggesting that oligomerization , which enhances DNA-binding for both proteins , may be a conserved function .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7730282" target="_blank">7730282</a>]<br></li>
<li>It is tempting to speculate this that region may represent at least part of the DNA-binding domain of OmpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7568033" target="_blank">7568033</a>]<br></li>
<li>EnvZ can phosphorylate OmpR , which contains an N-terminal receiver module followed by a C-terminal DNA-binding domain ( Figure 1 ; Aiba et al. , 1989a ; Forst et al. , 1989 ; Igo et al. , 1989 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7957084" target="_blank">7957084</a>]<br></li>
<li>This indicates that the N-terminal domain of OmpR is responsible for its cooperative binding to DNA , and its phosphorylation is likely to play an important role in enhancing the interaction between two N-terminal domains of the OmpR-P dimer on DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16618701" target="_blank">16618701</a>]<br></li>
<li>Our results are consistent with a two-domain structure for OmpR , an N-terminal phosphorylation domain joined to a C-terminal DNA-binding domain by a flexible linker region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7568033" target="_blank">7568033</a>]<br></li>
<li>Previous studies have shown that the DNA-binding activity of intact OmpR is greatly enhanced when the N-terminal domain is phosphorylated [ 7,45 ] , thus the N-terminal domain somehow modulates the ability of the C-terminal domain to bind DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9016718" target="_blank">9016718</a>]<br></li>
<li>The N-terminal Domain Is Required for Cooperative Binding of OmpR to the F1 Site -- We next compared the DNA-binding activity of the C-terminal DNA-binding domain of OmpR ( OmpRc ) with that of OmpR and OmpR-P by DNase I footprinting .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16618701" target="_blank">16618701</a>]<br></li>
<li>OmpR binding domain are critical forOmpRrecognition .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2553720" target="_blank">2553720</a>]<br></li>
<li>domain that resembles the DNA-binding domain of OmpR and related transcriptional regulators ( 3 , 19 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8808945" target="_blank">8808945</a>]<br></li>
<li>T h e structure of the C-terminal domain of E. col ; OmpR ( OmpRc ) , reported here , provides the first structural information for the OmpRPhoB family of DNA-binding domains .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9016718" target="_blank">9016718</a>]<br></li>
<li>To date , no structural information exists regarding the C-terminal DNA-binding domain of OmpR , although crystallization has been reported ( 17 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7568033" target="_blank">7568033</a>]<br></li>
<li>T h e isolated C-terminal domain is capable of binding DNA , although with significantly lower affinity than is observed for intact phosphorylated OmpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9016718" target="_blank">9016718</a>]<br></li>
<li>Genetic studies and in vitr-o DNA-binding assays have localized the DNA-binding activity to the C-terminal domain of OmpR [ 10,11 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9016718" target="_blank">9016718</a>]<br></li>
<li>N-terminal domain is required for cooperative binding of OmpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16618701" target="_blank">16618701</a>]<br></li>
<li>C , DNase I footprinting experiment was carried out using the C-terminal DNA-binding domain of OmpR ( OmpRc ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16618701" target="_blank">16618701</a>]<br></li>
<li>Each N - and C-terminal domain of OmpR has a distinct role in the DNA-binding of OmpR-P .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16618701" target="_blank">16618701</a>]<br></li>
<li>OmpR has an N-terminal receiver module and a C-ter-minal DNA-binding domain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8098993" target="_blank">8098993</a>]<br></li>
<li>Based on the crystal structures of the PhoB N-terminal domain ( Sola et al. , 1999 ) , the PhoB Cterminal domain ( Okamura et al. , 2000 ) and the OmpR C-terminal domain ( Martinez-Hackert and Stock , 1997b ) , together with the sequence similarity among PhoB , OmpR and PhoP , we could predict the domain organization for PhoP ( Fig. 9A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12123454" target="_blank">12123454</a>]<br></li>
<li>For some TorR-related proteins such as OmpR or ArcA , it has also been proposed that phosphoryl-ation increases the tendency of the N-terminal ( receiver ) domain to form multimers ( Nakashima et aL , 1991 ; Tardat and Touati , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596446" target="_blank">8596446</a>]<br></li>
<li>The two smallest fragments ( IV and V ) have N termini that correspond to amino acids 193 and 200 of OmpR , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7568033" target="_blank">7568033</a>]<br></li>
<li>Mass spectrometry of trypsin-digested OmpR and OmpR-P identified C-terminal fragments consisting of residues 193-239 and 200-239 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7568033" target="_blank">7568033</a>]<br></li>
<li>The C-terminal domain of the a subunit ( aCTD ) interacts with A1T-rich UP ( upstream activating region ) elements in the 260 to 240 region ( 31 ) and with many transcription regulators that bind upstream of the RNAP-bind-ing site , including CAP , OmpR , OxyR , Ada , and BvgA ( 4 , 5 , 10 , 23 , 38 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10572130" target="_blank">10572130</a>]<br></li>
<li>Unphosphorylated OmpR , the C-terminal domain and phosphorylated OmpR have increasing binding affinities , respectively [ 7,13,48,53 ] The structure of OmpRc , together with structural knowledge of the N-terminal domain by analogy with its homolog CheY ( the response regulator that controls flagellar rotation in bacteria1 chemotaxis ) [ 54,55 ] , permits definition of the boundaries of the linker region that joins the two domains .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9016718" target="_blank">9016718</a>]<br></li>
<li>The identified site at Lys-142 and the unidentified site N terminal to it closely bracket a region of OmpR that likely serves as the linker between the N - and C-terminal domains .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7568033" target="_blank">7568033</a>]<br></li>
<li>Comparison of the A domains of PhoB , OmpR and VirG to a similar amino terminal region in NarL ( positions 41-161 ) showed 27 % , 29 % and 24 % identity respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2648330" target="_blank">2648330</a>]<br></li>
<li>( see Response regulators share an N-terminal receiver module of - 120 amino acids , in which an invariant aspartate residue is located around the center ( see OmpR in Figure 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7957084" target="_blank">7957084</a>]<br></li>
<li>The predicted amino-terminal domain of CadC displays high homoiogy to the carboxy-terminal region of prokaryotic transcriptional activators involved in environmental sensing , including PhoP from Bacillus subtilis , OmpR from E. coti , VirG from Agrobacter-ium tumefaciens and ToxR from Vibrio choierae ( Watson e ( ai , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8057841" target="_blank">8057841</a>]<br></li>
<li>Genetic analysis has identified five amino acid residues in OmpR that are required for transcriptional activation : Arg-42 in the N-terminal domain and Pro-179 , Glu-193 , Ala-196 , and Glu-198 in the C-terminal domain ( 20 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7568033" target="_blank">7568033</a>]<br></li>
<li>Members of this subfamily have an amino-terminal phosphorylation domain characteristic of so-called two-component regulators , such as OmpR , CheY , PhoB , and NtrC , and a carboxyl-terminal domain conserved among many transcriptional activators , including LuxR and MalT .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1569008" target="_blank">1569008</a>]<br></li>
<li>Sequence similarities within the amino-terminal domain place UhpA in the family of regulatory proteins which includes OmpR , CheY , PhoB , and NtrC ( 25 , 34 ) , whereas sequence relatedness in the carboxyl-terminal domain indicates that UhpA is a member of the UhpA-LuxR family of transcriptional activators ( 12 , 35 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1569008" target="_blank">1569008</a>]<br></li>
<li>Hence , UhpA only has an N-terminal motif ( about 120 amino acids ) in common with the OmpR group and the NtrC group of regulatory proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3042748" target="_blank">3042748</a>]<br></li>
<li>OmpR is a 27kDa protein containing an N-terminal domain of approximately 125 amino acids joined by a small linker region to a C-terminal domain of approximately 98 residues .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9016718" target="_blank">9016718</a>]<br></li>
<li>The N-terminal part displayed similarity with the effector domain of these proteins , which contains the most-conserved amino acids that correspond to Asp-12 , Asp-55 , and Lys-105 of OmpR ( 23 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8083154" target="_blank">8083154</a>]<br></li>
<li>This mutation changes the amino acid at position 55 , the site of OmpR phosphorylation , from an aspartic acid residue to glutamine ( 16 , 17 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9096305" target="_blank">9096305</a>]<br></li>
<li>Analysis by site-directed mutagenesis of the amino acid residues involved in phosphotransfer between the two regulatory components , EnvZ and OmpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2066339" target="_blank">2066339</a>]<br></li>
<li>In particular , the 54 residues at the amino-terminal end of Dye were especially acidic , with 12 aspartic plus glutamic acid residues and only 2 lysine plus arginine residues ; comparable figures for the same region of OmpR were 10 acidic and 7 basic residues .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2984198" target="_blank">2984198</a>]<br></li>
<li>PrS2-OmpR is a hybrid protein , in which two tandemly repeated N-terminal domains ( a total of 184 amino acid residues ) from Myxococcus xanthus protein S is linked to the N-terminal end of the OmpR molecule .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16618701" target="_blank">16618701</a>]<br></li>
<li>The positions of amino acid residues in the OmpR and EnvZ portions are numbered from the respective NH2 termini gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3536870" target="_blank">3536870</a>]<br></li>
<li>The content of hydrophobic amino acid residues in the OmpR protein is calculated as 43 % .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6292199" target="_blank">6292199</a>]<br></li>
<li>The ompR77 mutation caused a Leu-to-Gln conversion at amino acid 16 from the NH2 terminus of the OmpR protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3536870" target="_blank">3536870</a>]<br></li>
<li>The hybrid protein specified by this fusion contains all but the last amino acid residue of OmpR fused in-frame to a large carboxy-terminal fragment of / % galactosidase ( Berman & Jackson , 1984 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2557454" target="_blank">2557454</a>]<br></li>
<li>The OmpR protein produced from this construction is a protein fusion with lipoprotein and has an extra sequence , Lys-Gly-Gly-Ile , inserted between theinitiator methionine andthe second amino acid residue , Gln .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3023382" target="_blank">3023382</a>]<br></li>
<li>Besides having a region of 19 adjacent amino acid residues homologous with OmpR ( with one exception ) , there is also quite extensive homology with OmpR in the rest of the Dye protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2984198" target="_blank">2984198</a>]<br></li>
<li>The region of residues 19-23 in the cro protein is a-helical and has been proposed to interact with its target DNA.When the amino acid sequence of the OmpR protein was analyzed for predicted secondary structure accordingto Chou and Fasman ( 38 ) , residues 84-90 were predicted to be in an a-helical region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6292199" target="_blank">6292199</a>]<br></li>
<li>Our laboratory and others have also shown that DNA-binding in the carboxyl terminus can affect both phosphorylation and dephosphorylation kin-etics at the amino terminus of OmpR .28,46 Figure 10 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11812125" target="_blank">11812125</a>]<br></li>
<li>It has altered depho-sphorylation properties , although the substitution is located in the DNA-binding domain at the carboxyl terminus .25 Clearly , substitutions that affect the phosphorylation site are not limited to the amino-terminal phosphorylation domain of OmpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11812125" target="_blank">11812125</a>]<br></li>
<li>Our combined results from ¯ uorescence aniso-tropy and DNase I footprinting indicate that a substitution in the phosphorylation site of OmpR affects its DNA-binding properties in the carboxyl terminus .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11812125" target="_blank">11812125</a>]<br></li>
<li>According to the af ® nity model of porin regulation , differential regulation of the porin genes is controlled by occupancy of OmpR-P binding sites of varying af ® nities .9 However , there is only a twofold difference in the binding af ® nity of OmpR-P for the ompF and ompC sites ( F1-F2-F3 and C1-C2-C3 ) , and the question of how OmpR binding site occupancy determines which porin is expressed remains .10 The response regulator OmpR ( 27 kDa ) consists of an N-terminal phosphorylation domain and a Cterminal DNA-binding domain ,11,12 joined by a protease-sensitive , ¯ exible linker of 15 amino acid residues .13 The C-terminal domain of OmpR contains a winged-helix-turn-helix DNA-binding motif .14,15 The phosphorylation site , aspartic acid 55 , is located in the amino terminus , referred to as the receiver-domain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11812125" target="_blank">11812125</a>]<br></li>
<li>We are interested in the molecular mechanism responsible for transmission of the conformational change associated with phosphorylation of OmpR at aspartic acid 55 in the N terminus that results in enhanced af ® nity of the C-terminal DNA-binding domain for DNA .10 In the single domain chemotaxis response regulator CheY , a substitution has previously been identi ® ed that converts the conserved active site threonine at position 87 to isoleu-cine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11812125" target="_blank">11812125</a>]<br></li>
				<label class="DOM_collapse_similarity" for="dom_similarity_164">
Tran , V.K. , Oropeza , R. , and Kenney , L.J. ( 2000 ) A single amino acid substitution in the C terminus of OmpR alters DNA-recognition and phosphorylation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19432797" target="_blank">19432797</a>]<br></label>
<input id="dom_similarity_164", type="checkbox">
<div class="DOM_similar">
<li>A single amino acid substitution in the C terminus of OmpR alters DNA-recognition and phosphorylation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11812125" target="_blank">11812125</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15755952" target="_blank">15755952</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15175316" target="_blank">15175316</a>]<br></li></div>
<li>We have characterized a substitution in the amino-terminal phosphorylation domain of OmpR , T83I , its phenotype is OmpFÿ OmpCÿ .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11812125" target="_blank">11812125</a>]<br></li>
<li>We have established that a single substitution in the amino terminus ( the phosphorylation domain ) of OmpR can interfere with transcriptional activation at ompF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11812125" target="_blank">11812125</a>]<br></li>
<li>T83I binds to the high affinity sites F1 and C1 Since our previous studies have demonstrated interactions between the amino and carboxylterminal domains of OmpR , it was of interest to determine whether the substitution that altered its phosphorylation properties would also affect its DNA-binding ability .25,28 For measurements of DNA-binding , we used ¯ uorescein-labeled oligo-nucleotides that correspond to the high af ® nity sites F1 and C1 upstream from the ompF and ompC promoters , respectively ( see Figure 11 , below , for a diagram of the binding sites ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11812125" target="_blank">11812125</a>]<br></li>
<li>In our previous studies with wild-type OmpR , we observed saturable binding to any oligonucleotide that contained either the F1 or C1 site .10 Thus , the T83I mutant , containing a single amino acid substitution , behaves differently from OmpR at the ompF regulatory region , where it exhibits high af ® nity binding , compared with the ompC regulatory region , where it does not exhibit saturable binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11812125" target="_blank">11812125</a>]<br></li>
<li>For one , the carboxy terminus of OmpR is not a highly basic domain , as was originally believed , and thus it is , unlikely that the OmpR and EnvZ proteins interact via the ionic attraction between acidic EnvZ and basic OmpR amino acid residues , as had been previously proposed ( 20 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2997120" target="_blank">2997120</a>]<br></li>
<li>Thus , it is probable that the carboxy terminus of OmpR contains a critical functional domain which may be highly sensitive to mutation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2997120" target="_blank">2997120</a>]<br></li>
<li>OmpR is represented as a two-domain molecule where shaded regions denote the amino terminus and open regions indicate the carboxyl terminus , joined by a linker .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11812125" target="_blank">11812125</a>]<br></li>
<li>The largest three fragments ( I-III ) have the N terminus of OmpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7568033" target="_blank">7568033</a>]<br></li>
<li>Fragments I and II also have the N terminus of OmpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7568033" target="_blank">7568033</a>]<br></li>
<li>Dye lacks this basis carboxyl terminus , but has a central part of the molecule identical to the OmpR protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2984198" target="_blank">2984198</a>]<br></li>
<li>In order to investigate this binding , we measured ¯ uorescence anisotropy of an OmpR protein labeled with ¯ uorescein at the amino terminus .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11812125" target="_blank">11812125</a>]<br></li>
<li>OmpR ( triangles ) or T83I ( circles ) were labeled at the amino terminus with ¯ uorescein succinimidyl ester and 20-30 nM protein was included in a binding reaction as described in Materials and Methods .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11812125" target="_blank">11812125</a>]<br></li>
<li>The resulting plasmid , pET20-OmpR , contains ompR under the control of the T7 promoter and with its own translational regulatory signals , and it expresses an OmpR protein with a His6 motif at its C terminus .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15743952" target="_blank">15743952</a>]<br></li>
<li>The 23 kDa protein was predicted to be a regulatory protein by us [ 7 ] and we have recently noted that it contains elements placing it in the OmpR category of proteins [ 121 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2556297" target="_blank">2556297</a>]<br></li>
<li>One wouldpredict arginine residues at these positions only if the synthesis of the OmpR protein initiated at thefirst ATG , located at nucleotide 252 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6292199" target="_blank">6292199</a>]<br></li>
<li>It is reasonable to assume that these two classes of mutations define two functional domains of the OmpR protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2997120" target="_blank">2997120</a>]<br></li>
<li>OmpR and its subfamily of response regulators are thought to consist of two domains .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7568033" target="_blank">7568033</a>]<br></li>
<li>Finally , the structure and function of the OmpR protein should be discussed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2668281" target="_blank">2668281</a>]<br></li>
<li>Consistent with this , no detectable protein complex is formed from isolated N-ancl C-terminal domains of OmpR ( A West , unpublished daya ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9016718" target="_blank">9016718</a>]<br></li>
<li>Recently , we demonstrated that the N-terminal portion of the OmpR protein contains a site involved in phosphorylation by the EnvZ protein ( 38 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2668281" target="_blank">2668281</a>]<br></li>
<li>In other words , is a particular OmpR mutant protein locked in a certain conformation and therefore defective in formation of the other form ( s ) , or are all forms present , but one simply non-functional ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2557454" target="_blank">2557454</a>]<br></li>
<li>In vitro studies revealed that OmpR binds to three sites ( designated C1 , C2 , and C3 ) upstream of the ompC promoter ( 41 , 66 ) and four sites ( one distal site , F4 , and three * Corresponding author .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16077119" target="_blank">16077119</a>]<br></li>
				</div></ul><br>
			<label class="collapse_EVO" for="_3"><font size="5px">OmpR evolution</font></label>
				<input id="_3" type="checkbox">
				<div class="EVO"><br><ul><li>The most abundant families described in prokaryotes , the LysR ( 16 % of the whole dataset ) , and the OmpR families ( 7 % of the TF dataset ) , have the wHTH motif , while the GalR/LacI and MerR families with the cHTH motif are scarce in all genomes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15556475" target="_blank">15556475</a>]<br></li>
<li>OmpR , a well-characterized member of the largest subfamily of RRs , functions as both an activator and repressor to regulate differentially the expression of the ompC and ompF genes that encode outer membrane porin proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10966457" target="_blank">10966457</a>]<br></li>
<li>The N-terminal part displayed similarity with the effector domain of these proteins , which contains the most-conserved amino acids that correspond to Asp-12 , Asp-55 , and Lys-105 of OmpR ( 23 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8083154" target="_blank">8083154</a>]<br></li>
<li>Likewise , OmpR , CheY , CheB , and NRI belong to a structurally conserved class of effector molecules in which the aspartic acid residues represented by Asp-12 and Asp-55 of OmpR are invariant ( 41 , 43 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2160945" target="_blank">2160945</a>]<br></li>
<li>nearly identical , 27,346 for Dye and 27,353 for OmpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2997120" target="_blank">2997120</a>]<br></li>
<li>These groups are the LysR and OmpR protein families .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3413113" target="_blank">3413113</a>]<br></li>
<li>28 % of the Dye protein was homologous with OmpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2984198" target="_blank">2984198</a>]<br></li>
<li>Recently , the Dye protein of E. colt ( Drury and Buxton 1985 ) has been found to be highly homologous to the OmpR protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3010044" target="_blank">3010044</a>]<br></li>
<li>Phosphorylated PhoB is the activator for the pho regulon ( Wanner , 1993 ) and belongs to the OmpR subfamily of transcriptional activators for a70-dependent promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7641688" target="_blank">7641688</a>]<br></li>
<li>Members of this subfamily have an amino-terminal phosphorylation domain characteristic of so-called two-component regulators , such as OmpR , CheY , PhoB , and NtrC , and a carboxyl-terminal domain conserved among many transcriptional activators , including LuxR and MalT .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1569008" target="_blank">1569008</a>]<br></li>
<li>On the basis of additional homology within the carboxy-terminal domain , TorR has been recognized as a member of the OmpR subfamily of response regulators , which includes numerous members in E. coil ( such as ArcA , OmpR , or PhoB ) but also in many other bacterial species ( Aiba et aL , 1993 ; Mills et aL , 1993 ; Nagasawa et aL , 1993 ; Parkinson and Kofoid , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596446" target="_blank">8596446</a>]<br></li>
<li>Moreover , CpxR and the other nine regulators belong to the same OmpR subfamily , and tor function ( Stock et al. , 1989 : Parkinson and Kofoid , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8294007" target="_blank">8294007</a>]<br></li>
<li>In E. coli , members of these homologous families have been identified in more than 10 different adaptive response systems ( for a review , see reference 38 ) , including chemotaxis ( CheA-CheY ) , nitrogen regulation ( NtrB-NtrC ) , phosphate regulation ( PhoR-PhoB ) , nitrate reductase regulation ( NarX-NarL ) , oxygen regulation ( ArcB-ArcA ) , and osmotic porin regulation ( EnvZ-OmpR ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8282700" target="_blank">8282700</a>]<br></li>
<li>These DNA-binding domains can be subdivided into three major families , represented by OmpR , NarL , and NtrC ( 14 , 7 , and 4 members , respectively , in E. coli ) ( 4 , 63 , 133 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10966457" target="_blank">10966457</a>]<br></li>
<li>The NarL structure is a model for other members of the FixJ or LuxR family of bacterial transcriptional activators , and possibly to the more distant OmpR and NtrC families as well .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8780507" target="_blank">8780507</a>]<br></li>
<li>The existence of multiple binding sites with differing afﬁnities is found for other response regulators , including OmpR ( 14 , 19 , 25 , 26 ) , NarL ( 18 ) , and BvgA ( 37 ) , as well as members of the LysR and AraC families of activators ( reviewed in references 8 and 30 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10913075" target="_blank">10913075</a>]<br></li>
<li>Well-studied examples are found among members of the AraC , LysR , and LuxR families , as well as in the OmpR and NarL families of phosphorylation-dependent response regulator proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10913075" target="_blank">10913075</a>]<br></li>
<li>Greater similarity ( Fig. 3 ) was found with ArcA ( 43 % identity ) , OmpR ( 32 % identity ) , and PhoB ( 29 % identity ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8083154" target="_blank">8083154</a>]<br></li>
<li>Primary amino acid sequence homology alignment of NarL to OmpR , VirG , and PhoB proteins from procaryotic microorganisms .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2648330" target="_blank">2648330</a>]<br></li>
<li>Comparison of the A domains of PhoB , OmpR and VirG to a similar amino-terminal region in NarL ( positions 41-161 ) showed 27 % , 29 % and 24 % identity respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2648330" target="_blank">2648330</a>]<br></li>
<li>The PhoB , OmpR and VirG A domains showed a similar to somewhat higher identity ( 28-45 % ) when compared to one another ( 21 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2648330" target="_blank">2648330</a>]<br></li>
<li>The PhoB , OmpR and VirG proteins appear to be more related to one another than to NarL as they have 34 % -40 % identity in the B domain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2648330" target="_blank">2648330</a>]<br></li>
<li>It wouldbe worth mentioning that the amino acid conversions found in the OmpR3 and OmpR77 proteins , respectively , are located adjacent to the putative phosphoacceptor site ( aspartic acid ) proposed for the CheY protein , which is one of the OmpR family ( 39 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2668281" target="_blank">2668281</a>]<br></li>
<li>ompFPro represents the recognition site of OmpR , a RR belonging to the same protein family as KdpE ( Fig. 1B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22291906" target="_blank">22291906</a>]<br></li>
<li>Such an arginine residue is conserved at structurally equivalent positions in many members of the OmpR/PhoB family , with the exception of KdpE , OmpR , and DrrB ( Fig .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22291906" target="_blank">22291906</a>]<br></li>
<li>The inability of mutant KdpE R193A to bind to DNA is interesting because in most other family members the equivalent residue is of opposite charge as observed in PhoB ( D196 ) and OmpR ( D202 ) ( Fig .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22291906" target="_blank">22291906</a>]<br></li>
<li>The relationships among the six OmpR family members have been reported ( 6 , 8 , 32-34 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3413113" target="_blank">3413113</a>]<br></li>
<li>The OmpR family member PhoB binds to direct repeat target sites in the promoter ( 32 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11976297" target="_blank">11976297</a>]<br></li>
<li>The sequences of KdpD and KdpE show they are members of the sensor-effector class of regulatory proteins : the C-terminal half of KdpD is homologous to sensors such as EnvZ and PhoR , and KdpE is homologous to effectors such as OmpR and PhoB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1532388" target="_blank">1532388</a>]<br></li>
<li>PgtA shows a closer match to the nitrogen-regulatory proteins than to the OmpR family .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3042748" target="_blank">3042748</a>]<br></li>
<li>A dominant mutation in Escherichia coli OmpR lies within a domain which is highly conserved in a large family of bacterial regulatory proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2656631" target="_blank">2656631</a>]<br></li>
<li>The match to the other proteins , although obvious , was weaker , ranging from about 25 % identity with NtrC and CheY to 19 % for OmpR , SfrA , and PhoB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3042748" target="_blank">3042748</a>]<br></li>
<li>One family includes OmpR , PhoB , SfrA , VirG , CheB , CheY , Spo0A , SpoOF , NtrC , and DctD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3042748" target="_blank">3042748</a>]<br></li>
<li>Amino acid sequence comparisons showed that UhpA was homologous to a family of bacterial regulatory proteins , some of which act as transcriptional activators ( OmpR , PhoB , NtrC , and DctD ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3042748" target="_blank">3042748</a>]<br></li>
<li>Sequence similarities within the amino-terminal domain place UhpA in the family of regulatory proteins which includes OmpR , CheY , PhoB , and NtrC ( 25 , 34 ) , whereas sequence relatedness in the carboxyl-terminal domain indicates that UhpA is a member of the UhpA-LuxR family of transcriptional activators ( 12 , 35 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1569008" target="_blank">1569008</a>]<br></li>
<li>If the OmpR and AraC families were related to the LysR family , then helix-turn-helix predictions would be expected near the amino termini .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3413113" target="_blank">3413113</a>]<br></li>
<li>However , in each case , helix-turn-helix predictions are made for stretches found near the middle of each OmpR and AraC family protein ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3413113" target="_blank">3413113</a>]<br></li>
<li>The oppositely oriented open reading frame ( ecnR ) , was identi ® ed in a MOTIFS search of Prosite as a member of the response regulator family of transcription factors , which include OmpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9677290" target="_blank">9677290</a>]<br></li>
<li>OmpR family transcription factors are thought to function as dimers ( or multimers ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16861804" target="_blank">16861804</a>]<br></li>
<li>CpxR is a member of the OmpR family of winged helix-turn-helix transcription factors .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15576781" target="_blank">15576781</a>]<br></li>
<li>Structural relationships in the OmpR family of winged-helix transcription factors .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10648521" target="_blank">10648521</a>]<br></li>
<li>This hypothesis is probably correct as TorR belongs to the OmpR family of response regulators ( 17 , 21 , 27 , 35 ) , and members of this family have been shown to bind cooperatively to multiple direct repeats ( 12 , 29 , 30 , 38 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10648521" target="_blank">10648521</a>]<br></li>
<li>It is likely that members of the NarL family bind to dyad targets , in contrast to the binding of OmpR family response regulators to direct-repeat targets .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11976297" target="_blank">11976297</a>]<br></li>
<li>Considering the high degree of similarity in the structure of the OmpR family of proteins , the corresponding loop in TorR is a candidate for TorI binding , and ongoing studies focus on the characterization of TorI TorR interactions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15197250" target="_blank">15197250</a>]<br></li>
<li>Model 1 also includes the possibility that NanR binds to DNA as a monomer , as in the case of WH proteins of the OmpR family , which bind to tandem repeats with a monomer binding to each successive major groove ( 4 , 15 , 16 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12897000" target="_blank">12897000</a>]<br></li>
<li>OmpR belongs to a subfamily of at least 50 response regulators with homologous C-terminal DNA-binding domains of approximately 98 arnino acids .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9016718" target="_blank">9016718</a>]<br></li>
<li>OmpR is one of the most extensively studied members of a subfamily of more than 50 response regulator protein characterized by sequence similarity in their C-terminal DNA-binding domains .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9016718" target="_blank">9016718</a>]<br></li>
<li>PhoP is a member of the OmpR family transcription factors characterized by the presence of a conserved N-terminal receiver domain that controls the function of the Cterminal DNA-binding domain ( Martinez-Hackert and Stock , 1997a ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12123454" target="_blank">12123454</a>]<br></li>
<li>Because OmpR belongs to the largest response regulator family ( OmpR/PhoB ) in E. coli , the DNA binding of transcription factors in galloping mode may be widely found in E. coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16618701" target="_blank">16618701</a>]<br></li>
<li>T h e structure of the C-terminal domain of E. col ; OmpR ( OmpRc ) , reported here , provides the first structural information for the OmpRPhoB family of DNA-binding domains .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9016718" target="_blank">9016718</a>]<br></li>
<li>The crystal structure of the Escherichia coli C-terminal DNA-binding domain of OmpR ( OmpRc ) , determined at 1.95 A resolution , has revealed that OmpRc belongs to the ` winged helix-turn-helix ` family of DNA-binding proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9016718" target="_blank">9016718</a>]<br></li>
<li>TorR belongs to the OmpR response-regulator family , in which the C-terminal domain includes a winged helix -- turn -- helix DNA-binding structure ( 28 -- 30 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15197250" target="_blank">15197250</a>]<br></li>
<li>Many of the residues in OmpR that have been identified from mutagenesis studies as being important for DNA binding are conserved or are conservatively substituted in other family members .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9016718" target="_blank">9016718</a>]<br></li>
<li>T h e relatively low leve1 of sequence identity in the OmpR family of DNA-binding domains is not altogethe Three views of the electrostatic surface potential of OmpRc .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9016718" target="_blank">9016718</a>]<br></li>
<li>Relationshipto other members of the OmpR/PhoB family of transcriptionalregulators T h e DNA-binding domain of OmpR exhibits sequence similarity over its entire length with a family of over 50 different transcriptional regulators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9016718" target="_blank">9016718</a>]<br></li>
<li>There is significant variation in the degree of sequence similarity among different DNA-binding domains , with members of the family exhibiting from 20 % to 65 % sequence identity with the 98 amino acid DNA-binding domain of OmpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9016718" target="_blank">9016718</a>]<br></li>
<li>OmpRc , however , also contains an additional four-stranded antiparallel flsheet that distinguishes the OmpR family from al1 previously defined classes of winged HTH proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9016718" target="_blank">9016718</a>]<br></li>
<li>We thus divided the repertoire of helix-turn-helix TFs into three categories : ( a ) classical HTH ( cHTH ) proteins with a tight loop between H-1 and H-2 , and no additional elements in the C-terminus , e.g. the GalR/LacI , family ( see Table 1 ) ; ( b ) the winged HTH proteins ( wHTH ) containing the C-terminal - hairpin and , in some proteins additional - sheets in between H-1 and H-2 , such as the OmpR , family ; and ( c ) the miscellaneous HTH ( mHTH ) , an heterogeneous collection of proteins that differ from the typical HTH in certain features , such as shortened helices or the incorporation into other super-structures , e.g. Rob in E. coli ( Tobes and Ramos , 2002 ) , or members of the enhancer-binding proteins ( EBP ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15556475" target="_blank">15556475</a>]<br></li>
<li>Members Group function Ec1 So2 Bs3 Group function LuxR/UhpA or OmpR family transcriptional response regulator of two-component regulatory system LysR family transcriptional regulator GntR or DeoR family transcriptional regulator sensory histidine kinase in two-component regulatory system sigma54 activator protein , enhancer binding protein AraC/XylS family transcriptional regulator ROK family transcriptional regulator/sugar kinase IclR family transcriptional regulator methyl-accepting chemotaxis protein MerR family transcriptional regulator DNA-binding regulatory protein AraC/XylS family transcriptional regulator MarR family transcriptional reguator AsnC family transcriptional regulator 46 20 18 18 13 13 11 10 9 18 14 13 13 11 16 13 8 1 oxidoreductase , Fe-S-binding 31 10 11 13 6 oxidoreductase , NAD ( P ) - binding oxidoreductase4 , NAD ( P ) - binding aldehyde dehydrogenase , NAD ( P ) - binding oxidoreductase , FAD/NAD ( P ) - binding sugar kinase respiratory reductase , alpha subunit aldo-keto reductase , NAD ( P ) - binding phosphatase nucleoside diphosphate ( Nudix ) hydrolase 43 30 22 5 14 14 7 8 8 7 5 8 8 2 7 5 1No .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15757509" target="_blank">15757509</a>]<br></li>
<li>From among these suppressor genes , we focused particularly on arcB and barA in this study , which also belong to the sensory kinase family ( luchi et al. , 1990 ; Nagasawa et al. , 1992 ) , because they are unique in the sense that they contain both transmitter and receiver modules in their primary amino acid sequences ( Figure 1 ; Kazuya Ishige , Shinobu Nagasawa , Shin-ichi Tokishita and Takeshi Mizunol Laboratory of Molecular Microbiology , School of Agriculture , Nagoya University , Chikusa-ku , Nagoya 464 , Japan ` Corresponding author Communicated by C.Lazdanski The osmoregulatory expression of ompC and ompF in Escherichia coli is mediated by a pair of bacterial signal transduction proteins , EnvZ ( sensory kinase ) and OmpR ( response regulator ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7957084" target="_blank">7957084</a>]<br></li>
<li>However , because the Cpx pathway controls the transcription of degP ( Danese et al. 1995 ; Raina et al. 1995 ) , and because CpxR is homologous to the OmpR subfamily of two-component transcription factors ( Dong et al. 1993 ) , the simplest interpretation of the results presented in Figure lb is that activation of the Cpx pathway stimulates dsbA transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9159399" target="_blank">9159399</a>]<br></li>
<li>The CpxR and CpxA proteins ( Weber and Silverman , 1988 ; Dong et al. , 1993 ) are highly homologous to the OmpR EnvZ two component system .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9130712" target="_blank">9130712</a>]<br></li>
<li>CpxR shares homology with other two-component DNA-binding proteins , including OmpR and ArcA ( Dong et al. 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7883164" target="_blank">7883164</a>]<br></li>
<li>OmpR and EnvZ are well-characterized members of the two-component osmoregulatory system ( Figure S12C ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17941710" target="_blank">17941710</a>]<br></li>
<li>OmpR belongs to the EnvZ-OmpR two-component system .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16359331" target="_blank">16359331</a>]<br></li>
<li>OmpR and EnvZ are members of a large class of two-component regulatory systems found in a variety of prokaryotic organisms ( Ronson et al. , 1987 ; Stock et aE .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2557454" target="_blank">2557454</a>]<br></li>
<li>EnvZ and OmpR belong to a family of `` two-component '' systems which are involved in adaptive responses to a variety of environmental stimuli in prokaryotes [ I 11 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8522138" target="_blank">8522138</a>]<br></li>
<li>OmpR and EnvZ belong to a large family of two-component regulatory systems found in a variety of prokaryotes ( 28 ; for review , see reference 2 or 40 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1648075" target="_blank">1648075</a>]<br></li>
<li>The torS and torR genes encode a two-component regulatory system in which TorS is an unor-thodox sensor containing three phosphorylation sites and TorR a response regulator of the OmpR family ( 8 , 9 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17040909" target="_blank">17040909</a>]<br></li>
<li>Since OmpR and EnvZ belong to this family of two-compo-nent regulatory systems , a similar phosphorylation pathway may exist for the differential expression of ompF and ompC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2656631" target="_blank">2656631</a>]<br></li>
<li>\ b Background : The differential expression of the ompfand ompC genes is regulated by two proteins that belong to the two component family of signal transduction proteins : the histidine kinase , EnvZ , and the response regulator , OmpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9016718" target="_blank">9016718</a>]<br></li>
<li>OmpR and EnvZ are members of a family of bacterial signal transduction proteins ( 36 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2203749" target="_blank">2203749</a>]<br></li>
				</div></ul><br>
			<label class="collapse_RP" for="_4"><font size="5px">Processes regulated by OmpR</font></label>
				<input id="_4" type="checkbox">
				<div class="RP"><br><ul><li>AGRT6 VIRG AGRRA YIIA ECOLI TOXR VIBCH + RRHM Pae Plastid response regulator OmpR Eco Porin bioynthesis , osmolarity PhoB Bsu Alkaline phosphatase regulator CreB Eco Catabolite repression ArcA Eco Aerobic respiratory control PetR Rca Cytochrome bc1 complex CutR Sli Copper metabolism KdpE Eco K + transport PhoB Eco Phosphate regulon PhoB Pae Phosphate regulon PhoP Sty Virulence TctD Sty Tricarboxylate transport VirG Atu6 Virulence VirG Atu5 Virulence VirG Arh Virulence YjjE Eco Unknown ToxR Vch Virulence cholera toxin Class 3 : DNA binding domains of family 3 FixJ Rme Nitrogen fixation FixJ Bja Nitrogen fixation NodW Bja Nodulation BvgA Bpe Virulence FimZ Eco Fimbrial synthesis DegU Bsu Extracellular degradative enzymes NarL Eco Nitrate reductase & formate dehydrogenase TrpO Pae Unknown YecB Eco Unknown RcsB Eco Colanic acid capsule biosynthesis B AgmR Fae Glycerol metabolism UhpA Eco Uptake of hexose phosphates Coma Bsu Competence GerE Bsu SigK txn .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7699720" target="_blank">7699720</a>]<br></li>
<li>Dual regulation by phosphoOmpR of ssrA/B gene-expression in Salmonella pathogenicity island 2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15995204" target="_blank">15995204</a>]<br></li>
<li>This hypothesis is consistent with the notion that a genome evolves from a set of precursor genes to a mature size by gene duplications and increasing modiﬁcations ( Yanai et al. , 2000 ) Table 1 TFs identiﬁed in complete genomes Family DBD Role PFAM Annotations Total 63 328 Function MarR w R 502 Multiple antibiotic resistance response Mercury resistance regulator MerR c R 244 MetJ -- Helmann et al. ( 1989 ) ; Zeng et al. ( 1998 ) Old et al. ( 1993 ) ; Wu et al. ( 1993 ) 48 235 10 R 0 Methionine biosynthesis repressor Molybdenum metabolism regulation R ModE 30 NadRa c 798 w 0 20 12 R 0 Foster and Penfound ( 1993 ) Regulation of NAD-synthesis and nucleotide-transport Regulation of the nag ( N-acetylglucosamine ) operon Biosynthesis of membrane components Probable growth inhibitor regulator NagR/XylRa w 244 A 2 31 1 A 135 OmpR w 722 638 n/d n/d 0 PemK -- PhoU -- Pérez - Rueda and Collado-Vides ( 2000 ) 36 62 0 0 -- -- Phosphate assimilation Ribose phosphate isomerase 137 R R 94 RpiR c Sorensen and Hove-Jensen ( 1996 ) Aramaki et al. ( 1995 ) 157 6 865 12 897 TetR/AcrR m Tetracycline resistance repressors Tryptophan operon repressor 12 29 TrpR WrbA R m Yang et al. ( 1993 ) Yang et al. ( 1993 ) -- -- R -- Enhancer in the formation and/or stability of noncovalent complexes between TrpR and DNA The notation is as follows : family name ( Hyp : hypothetical families ) ; DBD structure ( w , winged ; m , miscellaneous , and c , classic ) ; HLH , helix-loop-helix ; , beta-strand antiparallel ; CSD , cold shock DNA-binding domain-like ; and Zn , zinc-ﬁnger ; ND , not determined .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15556475" target="_blank">15556475</a>]<br></li>
<li>These are ( see Figure 1 ) : an aspartate residue that is the site of phosphorylation ( D55 in OmpR ) , two aspartate residues that coordinate a magnesium ion ( D11 and D12 ) , a lysine residue that interacts with the phosphoryl group ( K105 ) , an aromatic residue that functions as a rotamer ( Y102 ) , and a side-chain hydroxyl residue ( T83 ) that also interacts with the phosphoryl group .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11812125" target="_blank">11812125</a>]<br></li>
<li>JouRNAL OF BACTERIOLOGY , July 1981 , p. / 81/07025504 $ 02.00 / 0 NOTES Identification of OmpR : a Positive Regulatory Protein Controlling Expression of the Major Outer Membrane Matrix Porin Proteins of Escherichia coli K-12 RONALD K. TAYLOR , ' MICHAEL N. HALL ,2 LYNN ENQUIST ,3 AND THOMAS J. SILHAVY4 * Department ofBiological Sciences , University ofMaryland Baltimore County , Catonsvilk , Maryland 212281 ; Department ofMicrobiology and Molecular Genetics , Harvard Medical School , Boston , Massachusetts 021152 ; Laboratory ofMolecular Virology , National Cancer Institute , National Institutes of Health , Bethesda , Maryland 202053 ; and Cancer Biology Program , National Cancer Institute Frederick Cancer Research Center , Frederick , Maryland 21701 ' Received 30 January 1981/Accepted 13 March 1981 We report here on the cloning of a gene located within the ompB locus of E. coli K-12 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6263866" target="_blank">6263866</a>]<br></li>
<li>Well studied regulatory cases include OxyR regulation of OxyS RNA ( oxidative stress ) ( 4 ) , Fur regulation of RyhB RNA ( iron limitation ) ( 5 ) , cAMP-CRP regulation of CyaR and Spot 42 RNA ( glucose limitation ) ( 6 -- 9 ) , OmpR regulation of MicF , OmrA , and OmrB RNAs ( osmotic shock ) ( 10 , 11 ) , E-mediated transcription of MicA and RybB RNAs ( envelope stress ) ( 2 , 12 ) , PhoPQ regulation of MgrR ( Mg2 / Ca2 transport and virulence ) ( 13 ) , and LuxO control of Qrr1 -- 4 ( quorum-sensing in Vibrio ) ( 14 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20075074" target="_blank">20075074</a>]<br></li>
<li>A wide range of environmental stimuli affect the expression of the sRNAs including oxidative stress ( OxyR activates OxyS ) ; cell surface stress ( RcsB activates RprA ) ; glycine status of the cell ( GcvA activates GcvB ) ; osmotic imbalance ( OmpR activates OmrA/B ) ; quality of carbon source/growth rate [ McaS ; transcription factor ( s ) unknown ] ; low temperature [ DsrA ; transcription factor ( s ) unknown ] and O tension ( ArcA-ArcB 2 represses ArzA under anaerobic conditions ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22250746" target="_blank">22250746</a>]<br></li>
<li>None of the transcription factors involved in fatty acid regulation ( Crp , OmpR and FadR ) is oxygen sensing , though , recently , the ArcBA two-component signal-transduction-system , which senses oxygen indirectly through a change in redox potential , has been implicated in the negative regulation of aerobic fatty-acid-metabolism ( Georgellis et al. , 2001 ; Clark & Cronan , 1996 ; Malpica et al. , 2004 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16849788" target="_blank">16849788</a>]<br></li>
<li>To further evaluate a requirement of OmpR for the negative effect of MzrA ( YqjB ) overexpression on OMPs and MalE , we employed another wellstudied ompR allele , ompR472 ( Hall and Silhavy , 1981 ) , which produces an OmpF + OmpC - phenotype without inﬂuencing LamB and MalE expression ( Fig. 1B and C , lane 7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19432797" target="_blank">19432797</a>]<br></li>
<li>Since acetyl P also contributes to OmpR autoinduction , it may help regulate the larger subset of OmpR-dependent yet FlhD-inde-pendent genes ( 330 ) , including many associated with virulence ( e.g. , ssrAB , which encodes a 2CST pathway in S. enterica [ 261 ] ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15755952" target="_blank">15755952</a>]<br></li>
<li>aCTD participates in the function of many transcriptional activators , including CAP ( 34 ) , OmpR ( 38 ) , SoxS ( 17 ) , Ada ( 23 ) , and BvgA of Bordetella pertussis ( 4 ) , as well as in antitermination by l N protein ( 35 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10572130" target="_blank">10572130</a>]<br></li>
<li>Directly repeated DNA sequences appear to be involved in DNA-protein interactions for many prokaryotic regulatory proteins including lambda cII ( 45 ) , OmpR ( 46 ) ) ToxR ( 47 ) ) PhoB ( 44,48 ) , anO d xyR ( 49 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1447222" target="_blank">1447222</a>]<br></li>
<li>In plasmid pAT2OO5 , the promoter was positioned 17-bp upstream from the newly determined termination codon ( TGA ) for OmpR translation , which overlaps a possible initiation codon ( ATG ) for EnvZ translation as shown in Fig. 4 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3294816" target="_blank">3294816</a>]<br></li>
<li>Most NtrC-like regulators function in pairs ( PhoRPhoB , EnvZ-OmpR , CpxA-SfrA , VirA-VirG , DctB-DctD , CheA-CheB , CheY ) , including NifA , which is modified by the NifL product .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2666400" target="_blank">2666400</a>]<br></li>
<li>Like repression of ﬂagellar biosynthesis , this cell-division behavior requires PTA , OmpR , and FlhD ( but not FlhC ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15755952" target="_blank">15755952</a>]<br></li>
<li>The two previously deﬁned Hfq-dependent sRNA regulators of CsgD translation , OmrA and OrmB , also repress the translation of their transcriptional regulator OmpR , which in turn is essential for csgD transcription ( Guillier and Gottesman , 2006 ; 2008 ; Holmqvist et al. , 2010 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22250746" target="_blank">22250746</a>]<br></li>
<li>The complex transcriptional regulation of the csgDEFG operon will also have to be considered , which occurs in stationary phase and involves several global regulators such as RpoS , HNS , IHF and OmpR ( Barnhart and Chapman , 2006 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20407422" target="_blank">20407422</a>]<br></li>
<li>In E. coli , several global regulators such as H-NS ( Bertin et al. , 1994 ) , OmpR ( Shin & Park , 1995 ) , CRP-cAMP ( Soutourina et al. , 1999 ) , LrhA ( Lehnen et al. , 2002 ) and RcsAB ( Francez-Charlot et al. , 2003 ) are directly involved in the complex genetic regulatory hierarchy that assures ordered assembly of ﬂagellar components .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20528937" target="_blank">20528937</a>]<br></li>
<li>2010 SGM Printed in Great Britain In concert with the complex mode of regulation for biofilm-formation , the expression of csgD has been proposed to be under the control of various stress-sensing regulatory proteins such as OmpR ( Romling et al. , 1998 ; Vidal et al. , 1998 ; Prigent-Combaret et al. , 1999 , 2000 , 2001 ; Gerstel & Romling , 2001 ; Gerstel et al. , 2003 ) , CpxR ( PrigentCombaret et al. , 2001 ; Jubelin et al. , 2005 ) , Crl ( Bougdour et al. , 2004 ; Robbe-Saule et al. , 2006 ) , CRP ( Zheng et al. , 2004 ) , H-NS ( Arnqvist et al. , 1994 ; Gerstel et al. , 2003 ; Olsen et al. , 1993 ) , IHF ( Gerstel et al. , 2003 ; Gerstel & Romling , 2001 ) , MlrA ( Brown et al. , 2001 ) and RcsB ( Ferrieres & Clarke , 2003 ; Vianney et al. , 2005 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20466769" target="_blank">20466769</a>]<br></li>
<li>Furthermore , a number of transcription factors are involved in the regulation of the csgD promoter , including CpxR ( Jubelin et al. , 2005 ) , Crl ( Bougdour et al. , 2004 ) , H-NS ( Gerstel et al. , 2003 ) , IHF ( Gerstel et al. , 2003 , 2006 ) , OmpR ( Vidal et al. , 1998 ; Gerstel et al. , 2003 , 2006 ) , RstA ( Ogasawara et al. , 2007a ) , MlrA ( Brown et al. , 2001 ) , RcsB ( Ferrieres & Clarke , 2003 ; Vianney et al. , 2005 ) and CRP ( Zheng et al. , 2004 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20874755" target="_blank">20874755</a>]<br></li>
<li>The OmpR regulator is directly involved in activation of curli expression , in concert with several other regulatory proteins such as RpoS , H-NS , CpxR , and other yet unknown factors ( Romling et al. , 1998b ; Prigent-Combaret et al. , 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14523117" target="_blank">14523117</a>]<br></li>
<li>The stationary phase global regulator RpoS ( s ) s ( Olsén et al. , 1993 ; Römling et al. , 1998b ) and other global regulators including H-NS ( Olsén et al. , 1993 ; Arnqvist et al. , 1994 ) , Crl ( Arnqvist et al. , 1992 ) and OmpR ( Römling et al. , 1998b ; Vidal et al. , 1998 ) can effect curli synthesis by promoting or suppressing expression of the curli-encoding genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11489123" target="_blank">11489123</a>]<br></li>
<li>The stationary phase global regulator RpoS ( S ) ( Olsén et al. 1993 ; Römling et al. 1998 ) and other global regulators including H-NS ( Arnqvist et al. 1994 ; Olsén et al. 1993 ) , Crl ( Arnqvist et al. 1992 ) , OmpR ( Römling et al. 1998 ; Vidal et al. 1998 ) , MlrA ( Brown et al. 2001 ) and CRP ( Zheng et al. 2004 ) can aVect curli synthesis by promoting or suppressing expression of the curli-encoding genes by regulating the expression of csgD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21559929" target="_blank">21559929</a>]<br></li>
<li>We examined the effects of some global regulators ( factor inversion stimulation [ FIS ] , H-NS , OmpR , cAMP , ppGpp , and IHF ) on the expression of sra by using global regulator-defec-tive mutants lysogenized with lpF13 ( Psra-lacZ ) or carrying pFF6 ( Psra-lacZ ) and by measuring b-galactosidase activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11292794" target="_blank">11292794</a>]<br></li>
<li>The induction was independent of the RelA/SpoT , RpoH , KatF , OmpR , AppY , Lrp , PhoB and H-NS proteins during stress conditions that are known to induce or activate these global regulators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1453957" target="_blank">1453957</a>]<br></li>
<li>Lrp , leucine-responsive protein ; OxyR , oxidation stress regulator ; RpoS , RNA polymerase sigma factor for stationary phase ; OmpR , outer membrane protein regulator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7559363" target="_blank">7559363</a>]<br></li>
<li>Studies are in progress to determine whether ompA deletion affects known regulators of ﬁmB and ﬁmE such as leucine response regulatory protein ( Lrp ) , integration host factor , and OmpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16988236" target="_blank">16988236</a>]<br></li>
<li>AJW678 pMPMABD AJW678 pMPM-T6W OmpR is one of the response regulators that could be modified by acetyl phosphate to affect biofilm-formation AJW678 To determine whether a response regulator affected by acetyl phosphate levels could be involved in the regulatory pathway studied here , deletion mutants in some of the corresponding genes ( i.e. cpxR , ompR , phoR , uvrY ) from the Keio collection were analysed for biofilm-formation ( Fredericks et al. , 2006 ; Matsubara & Mizuno , 1999 ; McCleary & Stock , 1994 ; Tomenius et al. , 2005 ; Wolfe et al. , 2008 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25667010" target="_blank">25667010</a>]<br></li>
<li>It encodes an HPt-speci cents c phospho-histidine phosphatase that inhibits the transfer of phosphate groups between the histidine kinase ArcB and its non-cognate response regulator OmpR , and it modulates the response of the ArcBA signalling-pathway under certain anaerobic respiratory growth conditions [ 23 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12900013" target="_blank">12900013</a>]<br></li>
<li>This signalling-pathway involves the secondary Received 1 December 2014 Accepted 29 January 2015 metabolite acetyl phosphate and the response regulator OmpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25667010" target="_blank">25667010</a>]<br></li>
<li>Analysis of pgaA -- cat expression in ackA , ompR and ackA -- ompR backgrounds To further study the role played by the phospho-donor acetyl phosphate and the response regulator OmpR in the RcsC-pga signalling-pathway , plasmids pKK232pgaA1 and pKK232pgaA2N were transformed into the ackA and ompR mutants , carrying pMPMABD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25667010" target="_blank">25667010</a>]<br></li>
<li>We reasoned that comparative biochemical studies on the EnvZ-OmpR and KdpD-KdpE systems should provide a general insight into signal-transduction in response to environmental osmotic stimuli , in which a mechanism that involves coupling of an external physical force to internal genetic regulation might be implied .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8437514" target="_blank">8437514</a>]<br></li>
<li>These include , for example , the osmotic-response regulator OmpR , [ 384 ] the phosphate response regulator PhoB [ 394 ± 396 ] and the nitrogen response regulator NRI , the ¯ agellar regulator [ 388 , 389 ] and the chemotaxis response regulator , CheY [ 390 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10396600" target="_blank">10396600</a>]<br></li>
<li>In E. coli , members of these homologous families have been identified in more than 10 different adaptive response systems ( for a review , see reference 38 ) , including chemotaxis ( CheA-CheY ) , nitrogen regulation ( NtrB-NtrC ) , phosphate regulation ( PhoR-PhoB ) , nitrate reductase regulation ( NarX-NarL ) , oxygen regulation ( ArcB-ArcA ) , and osmotic porin regulation ( EnvZ-OmpR ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8282700" target="_blank">8282700</a>]<br></li>
<li>to many transcriptional activators , including OmpR and NarL , appears to activate transcription of uhpT .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1479338" target="_blank">1479338</a>]<br></li>
<li>This result , together with our genetic analysis of tppB transcription , supports the idea that the phosphorylation of OmpR plays an important role in the transcriptional activation of tppB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15175316" target="_blank">15175316</a>]<br></li>
<li>Transcription factors including NusG , OmpR , and RcsB , which are generally not considered to be under redox control , were also associated with thioredoxin , providing compelling evidence for an extensively coupled network of redox regulation of E. coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16760308" target="_blank">16760308</a>]<br></li>
<li>We are model of regulation of micF in which OmpR plays propose a a role in osmoregulation of micF but other ( unknown ) factors are involved in thermoregulation ( Fig. 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1695892" target="_blank">1695892</a>]<br></li>
<li>OmpR is also a transcriptional activator of two sRNAs , OmrA and OmrB , involved in the posttranscriptional regulation of several OMPs ( 22 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17416652" target="_blank">17416652</a>]<br></li>
<li>J Biol Chem 264:14090 -14094 Aiba H , Nakasai F , Mizushima S , Mizuno T ( 1989b ) Phosphorylation of a bacterial activator protein , OmpR , by a protein kinase , EnvZ , results in stimulation of its DNA-binding ability .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8413182" target="_blank">8413182</a>]<br></li>
<li>Phosphorylation of a bacterial activator protein , OmpR , by a protein kinase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8098993" target="_blank">8098993</a>]<br></li>
<li>Although OmpR phosphorylation depends primarily on EnvZ , the following evidence supports the existence of an alternative pathway that involves acetyl P. Mutants that lack EnvZ retain some ability to phosphorylate OmpR ( 134 ) and hence can still regulate OmpC and OmpF expression ( 133 , 394 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15755952" target="_blank">15755952</a>]<br></li>
<li>We have used acetyl phosphate to phosphorylate OmpR and have examined the effects of limited proteolysis by trypsin to detect conformational changes in OmpR in response to phosphorylation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7568033" target="_blank">7568033</a>]<br></li>
<li>Furthermore , the following evidence indicates that the phosphorylation of OmpR , a trans factor for osmoregulation , regulates ﬂagellar expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7642497" target="_blank">7642497</a>]<br></li>
<li>In order to assess the role of OmpR phosphorylation in motility , we measured the expression of ﬂhDC9-lac under different osmolarity conditions ( Fig. 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7642497" target="_blank">7642497</a>]<br></li>
<li>High osmolarity results in a high degree of phosphorylation of OmpR , which subsequently acts as a repressor of ompFexpression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7651128" target="_blank">7651128</a>]<br></li>
<li>Changes in osmolarity as well as envZ11 mutations affect ﬂagellar synthesis ( Fig. 3 ) , suggesting that the effects of high concentrations of salts on the synthesis of ﬂagella ( 42 ) involve the phosphorylation of OmpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7642497" target="_blank">7642497</a>]<br></li>
<li>Potential roles of DsbA , DsbB , and OmpR in OmpF synthesis and maturation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26209678" target="_blank">26209678</a>]<br></li>
<li>OmpR , a central integrator of several cellular responses in Yersinia enterocolitica .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25667010" target="_blank">25667010</a>]<br></li>
<li>12.38 -8.24 -59.96 1.13 ± 0.55 ± 0.50 ± 0.02 0.04 0.02 1.27 ± 0.50 ± 0.20 ± 0.03 0.03 0.03 / h ) / h ) Fig. 2 Comparison of the transcript levels of the E. coli wild type under different pH values at 7.0 and 6.0 : a global regulatory genes , b PhoB regulatory genes , c acid-inducible genes , d metabolic pathway genes , e respiratory chain genes during micro-aerobic continuous culture at the dilution rate of 0.1 h -1 Fig. 3 a , b Overall metabolic regulation of the E. coli during acidic condition ( a ) Acid Condition Hns ATP ADP PhoQ PhoQ P ATP ADP ¬ EvgS EvgS P cAMP - evgA Pho P PhoP Crp EvgA P B1500 EvgA cAMP-Crp ¬ RpoS b1500 ydeo ¬ YdeO gadE GadE O O GadW GadX hdeA gadB gadc gadW gadA gadX HdeA GadA GadB GadC Acid resistance ( b ) Acidic Condition ( H + ) ( indirect ) EnvZ PhoR fnr OxyR ldhA arcA P Fnr ArcA SoxR / S L PhoR - P OmpR OmpR - P DH pfl yfiD ompC TCA-cycle genes : gltA sdhC mdh sodA PhoB - P PhoB Lactate OmpC ompF asr YfiD Pfl OmpF Asr Formate Acetate ackA pta Acid resistance Moreover , in accord with the proteomic analysis , the expression of yﬁD : : lac reporter fusion and the intracellular content of YﬁD were found to be high during growth at low pH for both aerobic and anaerobic cultures ( Neil et al. 2002 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23274360" target="_blank">23274360</a>]<br></li>
<li>These include CreB and PhoB of enteric bacteria which are responsive to cytoplasmic catabolite concentrations and extracellular inorganic phosphate , respectively ; OmpR of enteric bacteria , involved in responses to osmotic conditions ; VirG proteins of agrobacterial species , activators of the Ti or Ri plasmid-encoded virulence op-erons ; and ArcA , an E. coli activator of genes encoding proteins which function in aerobic metabolism .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7699720" target="_blank">7699720</a>]<br></li>
<li>Correlation with mutagenesis data Transcription of the genes encoding the OmpF and OmpC proteins is regulated by OmpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9016718" target="_blank">9016718</a>]<br></li>
<li>The narX and narL genes , as mapped in this region with the correct orientation ( 9 ) , encode proteins with apparent molecular weights of 62,000-66 ,000 and OmpR is involved in osmotic regulation of the synthesis of outer membrane proteins , and NtrC is involved in nitrogen control of several nitrogen assimilatory genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2657652" target="_blank">2657652</a>]<br></li>
<li>RESULTS ArcA downregulates the transcription of fatty-acid-transport genes under anaerobic conditions The expression of fadL , which encodes one of the outer-membrane-bound proteins involved in the uptake of fatty acids , is regulated by at least three transcription factors : Crp , OmpR and FadR ( Berg et al. , 2004 ; Busch & Saier , 2002 ; Clark & Cronan , 1996 ; DiRusso et al. , 1992 ; Cronan & Subrahmanyam , 1998 ; Higashitani et al. , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16849788" target="_blank">16849788</a>]<br></li>
<li>CpxA , the inner-membrane protein , phosphorylates its conserved histidine residue in response to signals and then transphosphorylates the conserved asparate residue in CpxR .2 -- 7 ) Activated CpxR , the phosphorylated form ( pCpxR ) , activates transcription of a set of genes , all of which are involved in envelope protein-folding and proteolysis .8 -- 10 ) CpxP , a small periplasmic protein , is thought negatively to regulate CpxA activity by interacting with its periplasmic domain ,5,11,12 ) but CpxP is not required for induction of genes regulated by CpxA/CpxR in the response to envelope stress .13 ) NlpE is an outermembrane lipoprotein that activates the CpxA/CpxR regulon when overexpressed .14 ) Adhesion to hydrophobic surfaces induces the CpxA/CpxR regulon in a NlpE-dependent manner .15 ) CpxR is an OmpR family transcription factor that contains a helix-turn-helix motif at its N-terminus .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16861804" target="_blank">16861804</a>]<br></li>
<li>Neither ss , nor EnvZ -- OmpR participate in the osmotic induction of the ibp and dnaK promoters The general stress factor , sS , is responsible for the transcription of a number of genes implicated in the management of hyperosmotic stress ( Hengge-Aronis , 1996a , b ; Loewen et al. , 1998 ) and both translation of the rpoS mRNA and half-life of the protein increase in response to osmotic upshift ( Mufﬂer et al. , 1996a ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10594827" target="_blank">10594827</a>]<br></li>
<li>Expression of FlhDC is tightly regulated by multiple transcription factors , including CRP , H-NS and OmpR , and hence the genes of the ﬂagellar cascade are regulated in response to many different environmental inputs ( Shin and Park , 1995 ; Sout-ourina et al. , 1999 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19843224" target="_blank">19843224</a>]<br></li>
<li>TABLE 3 Gene knockout strains from the Keio collection that are no longer susceptible to killing by E. coli 25 Log10 CFU/ml SEM recovered after coculture withc : Gene Strain IDa BW25113 51H12 53B1 65F11 67G8 Strain or gene functionb Wild-type E. coli K-12 from Keio collection ; positive control knockout None atpA atpF dsbA dsbB E. coli 25 E. coli 25 mcpM 8.20 0.04 6.57 0.26 7.33 0.17 8.22 0.11 8.33 0.17 3.48 0.30 5.61 0.20 6.81 0.39 7.37 0.06 7.60 0.38 ATP synthase , F1 complex , subunit ; also called papA ATP synthase , Fo complex , subunit ; also called papF Periplasmic protein disulﬁde isomerase ; involved with disulﬁde bond formation Disulﬁde oxidoreductase ; involved with disulﬁde bond formation membrane protein ; oxidizes periplasmic DsbA Outer membrane protein , porin Transcriptional regulatory protein OmpR ; response regulator for osmoregulation 7G4 5F1 ompF ompR 6.98 0.14 7.97 0.08 6.33 0.33 8.27 0.07 a Strain identiﬁer ( ID ) or Keio collection clone ID .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26209678" target="_blank">26209678</a>]<br></li>
<li>Primary amino acid sequence homology alignment of NarL to OmpR , VirG , and PhoB proteins from procaryotic microorganisms .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2648330" target="_blank">2648330</a>]<br></li>
<li>Drury , L.S. and Buxton , R.S. : DNA sequence analysis of the dye gene of Escherichia coli reveals amino acid homology between the Dye and OmpR proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2670682" target="_blank">2670682</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8294007" target="_blank">8294007</a>]<br></li>
<li>The results of the nucleotide sequence analysis of the dye gene , presented in this paper , are compatible with the idea that theDye protein is involved in the control of genes coding for envelope proteins , since part of the derived amino acid sequence of the Dye protein is homologous with the OmpR protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2984198" target="_blank">2984198</a>]<br></li>
<li>The amino acid sequence of the PhoB protein deduced from the nucleotide sequence of the gene revealed extensive homology with those of the OmpR ' and Dye proteins , which are also positive regulators for several genes coding for envelope proteins of E. coli ( Hall & Silhavy , 1981 ; Drurp & Buxton , 1985 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3537313" target="_blank">3537313</a>]<br></li>
<li>The OmpR protein stimulates curli production through a regulatory cascade pathway , i.e. via activation of the csgD regulatory gene , which in turn up-regulates the csgBA operon , encoding the curli subunits .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14523117" target="_blank">14523117</a>]<br></li>
<li>The ﬂhD operon regulatory region is a direct target for several regulatory proteins : These are the cyclic AMP-catabo-lite gene activator protein complex , which is an activator in response to an alleviation of catabolite repression ( 60 ) ; OmpR P , which represses at high osmolarity ( 57 ) ; histonelike nucleoid-structuring ( H-NS ) protein , an activator ( 5 , 60 ) ; LrhA , a repressor ( 27 ) ; RcsAB , a repressor ( 17 ) ; and HdfR , a repressor ( 23 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15516564" target="_blank">15516564</a>]<br></li>
<li>Negative and positive regulation of the non-osmoregulated ompS1 porin gene in Salmonella typhi : a novel regulatory mechanism that involves OmpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25667010" target="_blank">25667010</a>]<br></li>
<li>Key words : OmpR - EnvZ - Phosphorylation scriptional control - DNA binding Tran ¬ Introduction ThefadL gene of Escherichia coli has been shown to play a central role in the uptake of exogenous long-chain-fatty-acids ( Nunn and Sirnons 1978 ; Nunn et al. 1979 ; Maloy et al. 1981 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8413182" target="_blank">8413182</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_67">
: Identification of OmpR : a positive regulatory protein controlling expression of the major outer membrane matrix porin proteins of Escherichia cog K-12 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6363215" target="_blank">6363215</a>]<br></label>
<input id="rp_similarity_67", type="checkbox">
<div class="RP_similar">
<li>Identification of OmpR : a positive regulatory protein controlling expression of the major outer membrane matrix porin proteins of Escherichia coli K-12 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6754693" target="_blank">6754693</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1906064" target="_blank">1906064</a>]<br></li></div>
<li>Batchelor E , Goulian M. Imaging OmpR localization in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21147125" target="_blank">21147125</a>]<br></li>
<li>Shin S , Park C. Modulation of flagellar expression in Escherichia coli by acetyl phosphate and the osmoregulator OmpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12144778" target="_blank">12144778</a>]<br></li>
<li>Mutations that speciﬁcally affect transcriptional-control by OmpR and EnvZ , a two-component regulatory system that controls porin gene-expression in E. coli , have also been isolated and lie in the same region of rpoA ( 28 , 30 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7836282" target="_blank">7836282</a>]<br></li>
<li>Slauch , J.M. , Garrett , S. , Jackson , D.E. , and Silhavy , T.J. ( 1988 ) EnvZ functions through OmpR to control porin gene-expression in Escherichia coli K-12 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19432797" target="_blank">19432797</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_73">
EnvZ functions through OmpR to control porin gene-expression in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2656631" target="_blank">2656631</a>]<br></label>
<input id="rp_similarity_73", type="checkbox">
<div class="RP_similar">
<li>EnvZ functions through OmpR to control porin gene-expression in Escherichia coli K-12 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1648075" target="_blank">1648075</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2203749" target="_blank">2203749</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7883164" target="_blank">7883164</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10698756" target="_blank">10698756</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1370954" target="_blank">1370954</a>]<br></li></div>
<li>164 , No. 2 Primary Characterization of the Protein Products of the Escherichia coli ompB Locus : Structure and Regulation of Synthesis of the OmpR and EnvZ Proteins DOROTHY E. COMEAU , KAZUHIRO IKENAKA , KANGLA TSUNG , AND MASAYORI INOUYE * Department of Biochemistry , State University of New York at Stony Brook , Stony Brook , New York 11794 Received 10 June 1985/Accepted 15 August 1985 The ompB operon of Escherichia col contains the structural genes for two proteins , OmpR and EnvZ , which control the osmoregulated biosynthesis of the porin proteins OmpF and OmpC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2997120" target="_blank">2997120</a>]<br></li>
<li>174 , No. 3 Positive and Negative Control of ompB Transcription in Escherichia coli by Cyclic AMP and the Cyclic AMP Receptor Protein LIN HUANG , PING TSUI , t AND MARTIN FREUNDLICH * Department of Biochemistry and Cell Biology , State University of New York , Stony Brook , New York 11794-5215 Received 12 August 1991/Accepted 20 November 1991 The ompB operon encodes OmpR and EnvZ , two proteins that are necessary for the expression and osmoregulation of the OmpF and OmpC porins in Escherchia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1310090" target="_blank">1310090</a>]<br></li>
<li>Received 9 December 1988/Accepted 3 March 1989 Expression in Escherichia coli of the genes that encode the major outer membrane porin proteins ( OmpF and OmpC ) is regulated by the transcription activator protein OmpR and the receptorlike protein EnvZ , which is located in the inner membrane .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2656631" target="_blank">2656631</a>]<br></li>
<li>JOURNAL OF BACTERIOLOGY , JUIY 1991 , p. / 91/134039 -10 $ 02.00 / 0 Copyright © ) 1991 , American Society for Microbiology F Mutations That Result in O Constitutive Expression cis-Acting omp mpR-Dependent JAMES M. SLAUCHt * AND THOMAS J. SILHAVY Department of Molecular Biology , Lewis Thomas Laboratory , Princeton University , Princeton , New Jersey 08544-1014 Received 3 December 1990/Accepted 30 April 1991 OmpR and EnvZ differentially control the transcription of the major outer membrane porin genes , ompF and ompC , in Escherichia coli in response to the osmolarity of the medium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1648075" target="_blank">1648075</a>]<br></li>
<li>It seems likely , therefore , that the Dye and OmpR proteins are related , and Dye may thus be involved in the osmoregulation of envelope protein genes as well as being required for sex factor gene-expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2984198" target="_blank">2984198</a>]<br></li>
<li>The transcriptional activator protein OmpR , best known for its role in osmoregulation of the major outer membrane proteins ( 21 , 32 ) is required for the expression of the plasmid-encoded E. coli microcin B17 ( mbc ) genes ( 22 ) , the Salmonella typhimu-rium tppB gene coding for a tripeptide permease ( 18 ) , and the Serratia liquefaciens exoenzyme phospolipase , phlA ( Givskov and Molin , unpublished ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2537825" target="_blank">2537825</a>]<br></li>
<li>The EnvZ/OmpR system is known to control the expression of outer membrane porins in response to medium osmolarity ( Aiba and Mizuno , 1990 ; Pratt and Silhavy , 1995 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12366850" target="_blank">12366850</a>]<br></li>
<li>The ompB locus codes for two proteins , EnvZ and OmpR , which are thought to be involved in the regulation of porin protein expression in response to osmolarity change of the growth medium ( 13-15 , 17 , 36 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1695892" target="_blank">1695892</a>]<br></li>
<li>Regulation of porin expression in response to osmolarity involves the EnvZ and OmpR proteins ( 15 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8002608" target="_blank">8002608</a>]<br></li>
<li>T h e bacteria1 response regulator OmpR is a transcription factor that differentially regulates the expression of genes encoding the major outer membrane porin proteins , OmpF and OmpC [ l ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9016718" target="_blank">9016718</a>]<br></li>
<li>The archetype of this basic His-Asp system is the E. coli EnvZ-OmpR osmosensing pathway responsible for modulating the expression of the outer membrane porin proteins OmpF and OmpC ( 224 ; Figure 1a ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10966457" target="_blank">10966457</a>]<br></li>
<li>By serving as a phosphodonor for the autophosphorylation of the two-component response regulator , OmpR , acetyl-P functions in the control of ﬂagellar synthesis ( 31 , 37 ) , cell-division ( 29 ) , and the expression of outer membrane porins ( 11 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10894724" target="_blank">10894724</a>]<br></li>
<li>The large amount of OmpR under the noninduced condition also implies the role of OmpR as a global regulator that might control some physiological events other than the expression of outer membrane porin .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7642497" target="_blank">7642497</a>]<br></li>
<li>Previous genetic and biochemical data suggest that EnvZ functions as an environmental sensor relaying information to the transcriptional activator OmpR , which in turn controls the expression of both porin genes ( for reviews .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2557454" target="_blank">2557454</a>]<br></li>
<li>Osmotic stress regulates the ratio of phosphatase ( P ) to autokinase ( K ) activities in EnvZ to adjust the phosphorylation state of OmpR , which in turn controls porin expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8098993" target="_blank">8098993</a>]<br></li>
<li>Since the OmpR transcriptional regulator activates expression of both omrA and omrB genes , this newly identified control should result in an autoregulatory loop limiting the amount of OmrA/B sRNAs .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18953042" target="_blank">18953042</a>]<br></li>
<li>A prime example of a 2CS controlling the expression of sRNAs is EnvZ/OmpR , which activates the expression of three sRNA genes ( micF , omrA , and omrB ) and represses the expression of one sRNA gene ( micC ) ( 23 -- 25 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25246476" target="_blank">25246476</a>]<br></li>
<li>We have shown that transcription of the sRNAs is activated by the two-component system EnvZ-OmpR and that , in turn , OmrA and OmrB negatively regulate the expression of multiple genes encoding OM proteins , including ompT , cirA , fecA and fepA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16359331" target="_blank">16359331</a>]<br></li>
<li>It seems likely that overexpression of many genes perturbs the cell envelope and/or osmolarity control sufﬁciently to cause induction of the EnvZ/OmpR system and therefore stimulate E expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17416652" target="_blank">17416652</a>]<br></li>
<li>A number of RRs , other than OmpR , regulate the expression of these acetyl P-responsive genes ( Fig. 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15755952" target="_blank">15755952</a>]<br></li>
<li>The activated EnvZ/OmpR system then inﬂuences the expression of a number of downstream target genes , Fig. 3 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19432797" target="_blank">19432797</a>]<br></li>
<li>Such environmental signals are mediated at the gene-expression level by a number of regulators , including , besides CsgD , global regulatory proteins such as OmpR , H-NS , CpxR , and * Corresponding author .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17873038" target="_blank">17873038</a>]<br></li>
<li>These environmental signals are mediated at the gene-expression level by a number of regulators , such as OmpR , H-NS , Besides its role as activator of the csgBA operon , CsgD regulates a number of genes involved in biofilm-formation and production of cell-surface-associated structures ( Latasa et al. , 2005 ; Gibson et al. , 2006 ) , as well as genes involved in transport , metabolism and gene regulation ( Chirwa & Abbreviation : CV , crystal violet .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18599830" target="_blank">18599830</a>]<br></li>
<li>nanC expression is also activated by the regulators cyclic AMP-catabolite activator protein , OmpR , and CpxR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15743943" target="_blank">15743943</a>]<br></li>
<li>Global regulators such as H-NS ( 27 , 28 ) , OmpR ( 30 ) , and IHF ( 36 ) control the expression of the curli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14978043" target="_blank">14978043</a>]<br></li>
<li>OmpR was shown to activate csgD expression , resulting in curli production .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15743952" target="_blank">15743952</a>]<br></li>
<li>Such reprogramming of gene-expression is likely to be due to changes in environmental physicochemical conditions and to involve two-component systems such as EnvZ/OmpR ( 41 ) and CpxA/CpxR ( 15 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11717281" target="_blank">11717281</a>]<br></li>
<li>Reﬂecting such situations , expression of the csgD gene is known to be positively or negatively regulated by a number of transcription factors , including MlrA ( 5 ) , Crl ( 3 ) , RpoS ( 3 ) , integration host factor ( 6 ) , Rcs ( 34 ) , the EnvZ-OmpR TCS ( 13 , 35 ) and the CpxAR TCS ( 13 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17468243" target="_blank">17468243</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_103">
and Inouye , M. ( 19931 Phenotypic revertant mutations of a new OmpR2 mutant ( V203Ql of Escherichia coli lie in the emZ gene , which encodes the OmpR kinase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8522138" target="_blank">8522138</a>]<br></label>
<input id="rp_similarity_103", type="checkbox">
<div class="RP_similar">
<li>Phenotypic revertant mutations of a new OmpR2 mutant ( V203Q ) of Escherichia coli lie in the envZ gene , which encodes in the OmpR kinase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9260955" target="_blank">9260955</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10049393" target="_blank">10049393</a>]<br></li></div>
<li>The CpxR , OmpR and RcsB -- RcsA proteins are good candidates for the control of curli gene-expression in response to cell envelope alterations .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16000739" target="_blank">16000739</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_106">
J Bacteriol 184:290 -- 301 Jubelin G , Vianney A , Beloin C , Ghigo J-M , Lazzaroni J-C , Lejeune P , Dorel C ( 2005 ) CpxR/OmpR interplay regulates curli gene-expression in response to osmolarity in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21559929" target="_blank">21559929</a>]<br></label>
<input id="rp_similarity_106", type="checkbox">
<div class="RP_similar">
<li>Jubelin , G. , Vianney , A. , Beloin , C. , Ghigo , J.M. , Lazzaroni , J.C. , Lejeune , P. , and Dorel , C. ( 2005 ) CpxR/OmpR interplay regulates curli gene-expression in response to osmolarity in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22250746" target="_blank">22250746</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17010156" target="_blank">17010156</a>]<br></li>
<li>[ 31 ] G. Jubelin , A. Vianney , C. Beloin , J.M. Ghigo , J.C. Lazzaroni , P. Lejeune , C. Dorel , CpxR/OmpR interplay regulates curli gene-expression in response to osmolarity in Escherichia coli , J. Bacteriol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16487683" target="_blank">16487683</a>]<br></li>
<li>[ PubMed : 9351822 ] Jubelin G , Vianney A , Beloin C , Ghigo J-M , Lazzaroni J-C , Lejeune P , Dorel C. CpxR/OmpR interplay regulates curli gene-expression in response to osmolarity in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20487295" target="_blank">20487295</a>]<br></li></div>
				<label class="RP_collapse_similarity" for="rp_similarity_110">
, Lejeune , P. , and Dorel , C. ( 2005 ) CpxR/OmpR interplay regulates curli gene-expression in response to osmolarity in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19432797" target="_blank">19432797</a>]<br></label>
<input id="rp_similarity_110", type="checkbox">
<div class="RP_similar">
<li>Lejeune P & Dorel C ( 2005 ) CpxR/OmpR interplay regulates curli gene-expression in response to osmolarity in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20874755" target="_blank">20874755</a>]<br></li>
<li>CpxR/OmpR Interplay regulates curli gene-expression in response to osmolarity in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17468243" target="_blank">17468243</a>]<br></li>
<li>CpxR/OmpR interplay regulates curli gene-expression in response to osmolarity in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16513732" target="_blank">16513732</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21415116" target="_blank">21415116</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16997959" target="_blank">16997959</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23564175" target="_blank">23564175</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16077119" target="_blank">16077119</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20466769" target="_blank">20466769</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24272779" target="_blank">24272779</a>]<br></li></div>
<li>One contained the gene for the response regulator OmpR ; the second contained mipA , encoding a murein hydrolase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17416652" target="_blank">17416652</a>]<br></li>
<li>OmpR regulates the stationary-phase acid resistance response of Salmonella enterica serovar Typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17185552" target="_blank">17185552</a>]<br></li>
<li>-- OmpR TCS in response to differences in osmolarity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17064377" target="_blank">17064377</a>]<br></li>
<li>Several other small-sized proteins that connect TCSs , such as PmrD , connecting the PhoQ/PhoP and PmrB/PmrA systems in Salmonella bacteria ( 16 , 19 ) , MzrA in E. coli , connecting the CpxA/CpxR and EnvZ/OmpR systems ( 11 ) , and IraM in E. coli , connecting PhoQ/PhoP and response regulator RssB ( 3 ) , have been previously reported .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21193607" target="_blank">21193607</a>]<br></li>
<li>Further analysis of the yﬁD promoter revealed the absence of consensus sites for regulators such as PhoP and OmpR that are known to be involved in the response of E. coli to acid .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11932447" target="_blank">11932447</a>]<br></li>
<li>Comparisons presented here are with the following sensorregulator transcription regulators : DegS-DegU , which controls extracellular enzyme synthesis in Bacillus subtilis ( 14 , 24 ) ; UhpB-UhpA , which controls hexose phosphate uptake in E. coli ( 37 , 50 ) ; FixL-FixJ , which controls nitrogen-fixation in Rhizobium melifoti ( 5 ) ; PhoR-PhoB , which controls the phosphate regulqn in E. coli ( 46 ) ; EnvZ-OmpR , which controls outer memitEane protein-synthesis in E. coli ( 12 ) ; and CpxA-SfrA , which controls surface properties in E. coli ( 2 , 30 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2649492" target="_blank">2649492</a>]<br></li>
<li>Interestingly , EnvZ also possesses a phosphatase activity that removes the phosphoryl group from the phosphorylated form of OmpR ( OmpR-P ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15175316" target="_blank">15175316</a>]<br></li>
<li>At low osmolarity , OmpF predominates in the outer membrane , while at high osmolarity the OmpF porin is replaced by OmpC .2 OmpF has a larger pore and a faster ¯ ow rate than OmpC .3 EnvZ is an inner membrane protein that is phosphorylated by intracellular ATP at histidine 243 in response to a signal related to the environmental osmolarity .4 ± 6 It transfers this phosphoryl group to aspartic acid 55 of OmpR .7 It has been suggested that EnvZ also catalyzes the dephosphorylation of phospho-OmpR ( OmpR-P ) ,8 further controlling the cellular levels of OmpR-P .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11812125" target="_blank">11812125</a>]<br></li>
<li>The entericidins are small , amphipathic a-helical lipoproteins encoded by tandem genes , which function as an antidote/toxin pair under positive control from sS and negative control from the osmosensing EnvZ/OmpR two-component signal-transduction-pathway .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9677290" target="_blank">9677290</a>]<br></li>
<li>Transcription factor OmpR is a receiver regulator of the two component system for transcription regulation of a set of the osmolarity response genes in bacteria .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10802181" target="_blank">10802181</a>]<br></li>
<li>In agreement with our ﬁndings , recent studies have shown that the Cpx response leads to upregulated production of the omrAB gene cluster through its direct regulation of mzrA , a gene encoding a small membrane protein capable of activating the EnvZ-OmpR two-component system , which in turn regulates omrAB transcription ( 68 , 77 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23564175" target="_blank">23564175</a>]<br></li>
<li>mzrA expression is negatively controlled by EnvZ/OmpR Kato et al. ( 2003 ) discovered a negative regulatory feedback loop in which the expression of pmrD was shown to be negatively regulated by PmrA , the response regulator of the PmrB/PmrA two-component system ; PmrD has been shown to promote phosphorylation of PmrA ( Kato and Groisman , 2004 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19432797" target="_blank">19432797</a>]<br></li>
<li>Following autophosphorylation , EnvZ transfers its high-energy phosphoryl group to its cognate response regulator , OmpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15175316" target="_blank">15175316</a>]<br></li>
<li>\ b Background : The differential expression of the ompfand ompC genes is regulated by two proteins that belong to the two component family of signal-transduction proteins : the histidine kinase , EnvZ , and the response regulator , OmpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9016718" target="_blank">9016718</a>]<br></li>
<li>T h e response regulator OmpR and the osmosensor histidine protein kinase EnvZ function as a two component phosphotransfer signal ' transduction system [ 8,9 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9016718" target="_blank">9016718</a>]<br></li>
<li>In this case , the process of signal-transduction is accomplished through weH-documented phosphotransfer involving the sensory protein kinase and response regulator ( EnvZ and OmpR ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1630316" target="_blank">1630316</a>]<br></li>
<li>Microbiology ( 2003 ) , 149 , 2847 -- 2857 DOI 10.1099 / mic.0.26306-0 The curli biosynthesis regulator CsgD co-ordinates the expression of both positive and negative determinants for bioﬁlm formation in Escherichia coli Eva Brombacher ,1 Corinne Dorel ,2 Alexander J. B. Zehnder1 and Paolo Landini1 1Swiss Federal Institute of Environmental Technology ( EAWAG ) , Überlandstrasse 133 , CH-8600 Dübendorf , Switzerland Correspondence Paolo Landini 2Unite ́ de Microbiologie et Génétique ( CNRS UMR 5122 ) , Institut National des Sciences Appliquées de Lyon , 10 rue Dubois , 69622 Villeurbanne Cedex , France landini@eawag.ch Production of curli , extracellular structures important for bioﬁlm formation , is positively regulated by OmpR , which constitutes with the EnvZ protein an osmolarity-sensing two-component regulatory system .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14523117" target="_blank">14523117</a>]<br></li>
<li>The Escherichia coli tppB ( ydgR ) Gene Represents a New Class of OmpR-Regulated Genes Ee-Been Goh , Dominic F. Siino , and Michele M. Igo * icrobiology , Division of Biological Sciences , University of California , Davis , Davis , California 95616 Received 1 December 2003/Accepted 2 March 2004 Section of M The EnvZ/OmpR two-component regulatory system plays a critical role in the Escherichia coli stress response .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15175316" target="_blank">15175316</a>]<br></li>
<li>Moreover , it has been reported that acetyl phosphate plays a role in protein-acetylation by modifying hundreds of proteins in E. coli , among them OmpR ( Kuhn et al. , 2014 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25667010" target="_blank">25667010</a>]<br></li>
<li>The OmpR protein , a positive regulator of these genes , is involved in the osmoregulation ( Dairi et al. 1985 ; Nara et al. 1984 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3010044" target="_blank">3010044</a>]<br></li>
<li>Remarkably , the OmpR protein has been implicated as an important regulator of biofilm-formation in E. coli and in other bacteria such as Yersinia ( Brzostek et al. , 2012 ; Samanta et al. , 2013 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25667010" target="_blank">25667010</a>]<br></li>
<li>Analysis by site-directed mutagenesis of the amino acid residues involved in phosphotransfer between the two regulatory components , EnvZ and OmpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2066339" target="_blank">2066339</a>]<br></li>
<li>The pleio-tropic effect of regulation by EnvZ and OmpR suggests the role of OmpR as a global regulator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7642497" target="_blank">7642497</a>]<br></li>
<li>OmpR Pi ( intrinsic dephosphorylation ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11812125" target="_blank">11812125</a>]<br></li>
<li>The simplest interpretation of these observations as to the phosphorylation and dephosphorylation of the mutant proteins is that EnvZ * is involved not only in OmpR phosphorylation but also in OmpR dephosphorylation and that this particular mutant , EnvZ11 * , can not effectively mediate the OmpR dephosphorylation reaction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2668281" target="_blank">2668281</a>]<br></li>
<li>The EnvZ Protein Is Involved in the Dephosphorylation of the OmpR Protein-The fact that EnvZ11 * was defective in OmpR dephosphorylation suggests that the EnvZ protein is involved in OmpR dephosphorylation as well .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2668281" target="_blank">2668281</a>]<br></li>
<li>The sensor protein , EnvZ , controls the phosphorylation of OmpR , a regulatory protein that has a DNA binding property ( 39 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7642497" target="_blank">7642497</a>]<br></li>
<li>The EnvZ sensor protein monitors external signals and modulates the activity of the OmpR response regulator by phosphorylation and dephosphorylation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15466019" target="_blank">15466019</a>]<br></li>
<li>phosphorylation and dephosphorylation abilities as to the activator protein , OmpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24966114" target="_blank">24966114</a>]<br></li>
<li>We provide evidence that the EnvZ protein is involved not only in OmpR phosphorylation but also in OmpR dephosphorylation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2668281" target="_blank">2668281</a>]<br></li>
<li>Recently , we demonstrated that the N-terminal portion of the OmpR protein contains a site involved in phosphorylation by the EnvZ protein ( 38 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2668281" target="_blank">2668281</a>]<br></li>
<li>JOURNAL OF BACTERIOLOGY , Aug. 1995 , p. 4696 -- / 95 / $ 04.0010 Copyright q 1995 , American Society for Microbiology Modulation of Flagellar Expression in Escherichia coli by Acetyl Phosphate and the Osmoregulator OmpR SOOAN SHIN AND CHANKYU PARK * Department of Biological Sciences , Korea Advanced Institute of Science and Technology , Yusong-Ku , Taejon , Republic of Korea Received 19 December 1994/Accepted 3 June 1995 During the search for unknown factors involved in motility , we have found that expression of the ﬂagellar master operon ﬂhDC is affected by mutations of the pta and ackA genes , encoding phosphotransacetylase and acetate kinase , respectively ( S. Shin , J. Sheen , and C. Park , Korean J. Microbiol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7642497" target="_blank">7642497</a>]<br></li>
<li>The phosphorylation of OmpR and some of its effects on gene regulation , either as an activator or as a repressor , have been evidenced by different groups ( Prüss , 1998 ; Shin & Park , 1995 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25667010" target="_blank">25667010</a>]<br></li>
<li>In vitro analysis of phosphorylation with ArcBC and OmpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7957084" target="_blank">7957084</a>]<br></li>
<li>In vivo phosphorylation of OmpR in E. coli parent strain MC4100 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2160945" target="_blank">2160945</a>]<br></li>
<li>Signal transduction and osmoregulation in Escherichia coli : a novel mutant of the positive regulator , OmpR , that functions in a phosphorylation-independent manner .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7730282" target="_blank">7730282</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11812125" target="_blank">11812125</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_150">
[ 10 ] Pru `` , B.M. ( 1998 ) Acetyl phosphate and the phosphorylation of M OmpR are involved in the regulation of the cell-division rate in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11287152" target="_blank">11287152</a>]<br></label>
<input id="rp_similarity_150", type="checkbox">
<div class="RP_similar">
<li>Acetyl phosphate and the phosphorylation of OmpR are involved in the regulation of the cell-division rate in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25667010" target="_blank">25667010</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10894724" target="_blank">10894724</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15755952" target="_blank">15755952</a>]<br></li>
<li>Acetyl phosphate and the phosphorylation of OmpR are involved in the regulation-of-cell-division rate in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15516564" target="_blank">15516564</a>]<br></li></div>
<li>The data obtained indicate that OmpR works in both a positive and negative fashion to control the transcription of ompF and this result forms the basis of a model for porin regulation that explains the switch from OmpF to OmpC production in response to increasing medium osmolarity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2557454" target="_blank">2557454</a>]<br></li>
<li>OmpR also regulates ompF post-transcriptionally through micF but primarily in low-to medium osmolarity conditions .8,2 The micF gene has a complex promoter region that serves as the site of interaction for several transcription factors ( Figure 4 ( a ) ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11601842" target="_blank">11601842</a>]<br></li>
<li>Our current hypothesis , based on genetic analysis , is that at low osmolarity , low concentrations of phosphoOmpR ( OmpR-P ) interact with high-affinity sites at ompF to stimulate its expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7568033" target="_blank">7568033</a>]<br></li>
<li>We propose that this tight hierarchical binding of a transcription factor , OmpR , allows distinct stepwise regulation of ompF and ompC transcription , which minimizes their overlapping expression upon changes in the medium osmolarity to achieve the reciprocal expression of ompF and ompC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16618701" target="_blank">16618701</a>]<br></li>
<li>Phosphorylated OmpR positively regulates ompC transcription , and it negatively regulates ompF transcription ( 47 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8002608" target="_blank">8002608</a>]<br></li>
<li>Under conditions of low osmolarity , OmpR activates transcription of ompF , while under high osmolarity it represses transcription of ompF and activaies transcription of ompC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9016718" target="_blank">9016718</a>]<br></li>
<li>Phosphorylated OmpR activates expression of the OmpF porin at low osmolarity , and at high osmolarity represses ompF transcription and activates expression of OmpC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11812125" target="_blank">11812125</a>]<br></li>
<li>Evidence for the Physiological Importancoef the Phosphotransfer between theTwo Regulatory Components , EnvZ andOmpR , in Osmoregulation inEscherichia coli * ( Received for publication , March 28 , 1989 ) Hirofumi AibaS , FujieNakasaiS , Shoji Mizushimag , and Takeshi MizunoST From the $ Laboratory of Microbiology , School of Agriculture , Nagoya University , Chikusa-ku , Nagoya 464 and the $ Institute of Applied Microbiology , the University of Tokyo , Yuyoi , Bunkyo-ku , Tokyo 113 , Japan Previously , the transfer of the phosphoryl group between the EnvZand OmpR proteins , whichare involved in activation of the ompF and ompC genes in response to the medium osmolarity , has been demonstrated in vitroI .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2668281" target="_blank">2668281</a>]<br></li>
<li>OmpR is a DNA-binding protein which acts as both an activator and a repressor to control ompF transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8113170" target="_blank">8113170</a>]<br></li>
<li>Phosphorylation of a bacterial activator protein , OmpR , by a protein kinase , EnvZ , stimulates the transcription of the ompF and ompC genes in Eschenchia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1532173" target="_blank">1532173</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_163">
htm References Aiba , H. , and Mizuno , T. ( 1990 ) Phosphorylation of a bacterial activator protein , OmpR , by a protein kinase , EnvZ , stimulates the transcription of the ompF and ompC genes in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12366850" target="_blank">12366850</a>]<br></label>
<input id="rp_similarity_163", type="checkbox">
<div class="RP_similar">
<li>Phosphorylation of a bacterial activator protein , OmpR , by a protein kinase , EnvZ , stimulates the transcription of the ompF and ompC genes in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8113170" target="_blank">8113170</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1885551" target="_blank">1885551</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1310090" target="_blank">1310090</a>]<br></li></div>
<li>Cold Spring Harbor Laboratory Press , Cold Spring Harbor , New York Norioka S , Ramakrishnan G , Ikenaka K , lnouye M ( 1986 ) Interaction of a transcriptional activator , OmpR , with reciprocally osmoregulated genes , ompF and ompC , of Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8413182" target="_blank">8413182</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_166">
In vivo phosphorylation of OmpR , the transcription activator of the ompF and ompC genes in Escherichia co/i .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8098993" target="_blank">8098993</a>]<br></label>
<input id="rp_similarity_166", type="checkbox">
<div class="RP_similar">
<li>In vivo phosphorylation of OmpR , the transcription activator of the ompF and ompC genes in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10894724" target="_blank">10894724</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11717281" target="_blank">11717281</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8002608" target="_blank">8002608</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8113170" target="_blank">8113170</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16077119" target="_blank">16077119</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15755952" target="_blank">15755952</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9422609" target="_blank">9422609</a>]<br></li></div>
<li>[ 7 ] Forst , S. , Delgado , J. , Rampersaud , A. and Inouye , M. ( 1990 ) In uiuo phosphorylation of OmpR , the transcription activator of the ompF and ompC genes in Escherichia cofi .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7557323" target="_blank">7557323</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_169">
J Biol Chem 267:8685 -8691 Forst S , Delgado J , Rampersaud A , Inouye M ( 1990 ) In vivo phosphorylation of OmpR , the transcription activator of the ompF and ompC genes in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8413182" target="_blank">8413182</a>]<br></label>
<input id="rp_similarity_169", type="checkbox">
<div class="RP_similar">
<li>Forst , S. , Delgado , J. , Rampersaud , A. , and Inouye , M. ( 1990 ) In vivo phosphorylation of OmpR , the transcription activator of the ompF and ompC genes in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8412671" target="_blank">8412671</a>]<br></li></div>
<li>Intramolecular second-site revertants to the phosphorylation site mutation in OmpR , a kinase-depen - dent transcriptional activator in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11812125" target="_blank">11812125</a>]<br></li>
<li>Identiﬁcation of a phosphorylation site and functional analysis of conserved aspartic acid res-idues of OmpR , a transcriptional activator for ompF and ompC in Esche-richia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11717281" target="_blank">11717281</a>]<br></li>
<li>Identi ® cation of a phosphorylation site and functional analysis of conserved aspartic acid resi-dues of OmpR , a transcriptional activator for ompF and ompC in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11812125" target="_blank">11812125</a>]<br></li>
<li>OmpR , consisting of 239-amino-acid residues , is a transcriptional regulator of the ompF and ompC genes ( for review , see [ IO ] ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8522138" target="_blank">8522138</a>]<br></li>
<li>Microbio1 Rev 51 :301 -319 Forst S. Conieau D , Norioka S. Inouye M ( 1987 ) Localization and membrane topology of EnvZ , a protein involved in osmo-regulation of OmpF and OmpC in Esclrerichiu coli.J Biol Chem , 262 : 16433-16438 Forst S , Delgado J. Inouye M ( 1989a ) Phosphorylation of OmpR by the osmosensor EnvZ modulates expression of the ompF and oinpC genes in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1736095" target="_blank">1736095</a>]<br></li>
<li>These mutants exhibit different phenotypes as to expression of the OmpF and OmpC proteins , indicating that the OmpR protein is directly involved in osmoregulatory expression of the ompF and ompC genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3536870" target="_blank">3536870</a>]<br></li>
<li>According to their model , the OmpR protein acts as a positive regulator for expression of the ompF and ompC genes , and the envZ product , a possible envelope protein , acts as an osmosensor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3536870" target="_blank">3536870</a>]<br></li>
<li>Location of the phosphorylation site and DNA binding site of a positive regulator , OmpR , involved in the activation of the osmoregulatory genes of Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11812125" target="_blank">11812125</a>]<br></li>
<li>Location of DNA-binding segment of a positive regulator , OmpR , involved in activation of the ompF and ompC genes of Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11812125" target="_blank">11812125</a>]<br></li>
<li>OmpR is a response regulator involved in expression of ompF and ompC in E. coli ( 20 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26209678" target="_blank">26209678</a>]<br></li>
<li>Genes Dev 3:1725 -1734 Jo Y-L , Nara F , Ichihara S , Mizuno T , Mizushima S ( 1986 ) Purification and characterization of the OmpR protein , a positive regulator involved in osmoregulatory expression of the ompF and ompC genes in Eseheriehia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8413182" target="_blank">8413182</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_182">
Puriﬁcation and characterization of the OmpR protein , a positive regulator involved in osmoregulatory expression of the ompF and ompC genes in Esch-erichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11717281" target="_blank">11717281</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8002608" target="_blank">8002608</a>]<br></label>
<input id="rp_similarity_182", type="checkbox">
<div class="RP_similar">
<li>Puriﬁcation and characterization of the OmpR protein , a positive regulator involved in osmoregulatory expression of the ompF and ompC genes in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7642497" target="_blank">7642497</a>]<br></li>
<li>Purification and characterization of the OmpR protein , a positive regulator involved in osmoregulatory expression of the ompF and ompC genes in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8113170" target="_blank">8113170</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1906064" target="_blank">1906064</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2160945" target="_blank">2160945</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2656631" target="_blank">2656631</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1885551" target="_blank">1885551</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8206845" target="_blank">8206845</a>]<br></li></div>
<li>Novel rpoA mutation that interferes with the function of OmpR and EnvZ , positive regulators of the ompF and ompC genes that code for outermembrane proteins in Escherichia coli K-12 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8113170" target="_blank">8113170</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8383109" target="_blank">8383109</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1648075" target="_blank">1648075</a>]<br></li>
<li>OmpR , a well-characterized member of the largest subfamily of RRs , functions as both an activator and repressor to regulate differentially the expression of the ompC and ompF genes that encode outer membrane porin proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10966457" target="_blank">10966457</a>]<br></li>
<li>and ompF expression in Escherichia coli Tian Jin ` , Masayori Inouye A Received 26 July 1995 ; revised29 August 1995 : accepted 30 August 1995 Osmoregulation of the porin genes , otnpF and ompC of E.scherichiu co/i , occurs at the level of transcription through the action of EnvZ and OmpR proteins as well as at the level of translation through micF antisense RNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8522138" target="_blank">8522138</a>]<br></li>
<li>Some systems output a graded response , such as the EnvZ-OmpR system that mediates the differential expression of porin genes ompF and ompC ( 224 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10966457" target="_blank">10966457</a>]<br></li>
<li>Introduction The expression of the ompF and ompC porin genes in Escherichia coli is controlled by the response regulatory protein , OmpR , and the histidine kinase sensor protein , EnvZ [ l-4 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7557323" target="_blank">7557323</a>]<br></li>
<li>The two-component regulatory system , OmpR and EnvZ , in Escherichia coli , controls the expression of ompF and ompC , the genes for the major outer membrane porin proteins , in response to media osmolarity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1648075" target="_blank">1648075</a>]<br></li>
<li>ompB also has a dual role in E. coli , since its products , OmpR and EnvZ , control the levels of the OmpF and OmpC porins through transcriptional regulation of the ompF and ompC operons ( 22 , 23 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1310090" target="_blank">1310090</a>]<br></li>
<li>Expression and osmoregulation of these genes involves primarily transcriptional-control mediated by OmpR and EnvZ , the protein products of the ompB operon ( 22 , 23 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1885551" target="_blank">1885551</a>]<br></li>
<li>IHF is also a transcriptional regulator of the ompB operon whose protein products , EnvZ and OmpR , are required for expression and osmoregulation of ompF and ompC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1447969" target="_blank">1447969</a>]<br></li>
<li>The EnvZ-OmpR two-component system represses the transcription of the large diameter pore porin gene ompF and activates the gene for the small pore porin ompC , consistent with the idea that limiting entry of possible toxic compounds may be important ( Nikaido , 2003 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16359331" target="_blank">16359331</a>]<br></li>
<li>I 4 , -116 239 1 450 247 Thr - Arg ( C17074G ) 247 MH1461 450 2391 Leu - Gln DISCUSSION The OmpR and EnvZ proteins are involved in osmoregula-of the ompF and ompC ( 11 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3536870" target="_blank">3536870</a>]<br></li>
<li>For example , the EnvZ protein may be involved in modification of the OmpR protein , which in turn positively regulates either the ompF or ompC gene , depending on the type of modification .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3536870" target="_blank">3536870</a>]<br></li>
<li>The OmpR et al. , protein is a regulator for the and genes ompF ompC coding for outer membrane porin proteins ( Hall & Silhavy , 1981 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3537313" target="_blank">3537313</a>]<br></li>
<li>As illustrated in Fig. 3b , it has been reported that the expressions of ompC and ompF genes for outer membrane porin proteins are regulated by a sensory histidine kinase EnvZ and cognate response regulator OmpR , where EnvZ regulates the phosphorylation of OmpR ( Castanié - Cornet et al. 2010 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23274360" target="_blank">23274360</a>]<br></li>
<li>Besides affecting transcription of ompC and ompF , the activated EnvZ/OmpR two-component system influences transcription of a number of genes whose product help reduce stress ( Gerken et al. 2009 and references therein ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20487295" target="_blank">20487295</a>]<br></li>
<li>These mutations affect the expression of genes that are archetypal members of the EnvZ/OmpR regulon , such as ompF and ompC , and also inﬂuence the expression of genes , including lamB and malE , normally not controlled by the EnvZ/OmpR regulon ( Wandersman et al. , 1980 ; Case et al. , 1986 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19432797" target="_blank">19432797</a>]<br></li>
<li>Ghou , J.H. , Greenberg , J , T , , and Demple , B. ( 1993 ) Posttranscriptional repression of Escherichia coli OmpR protein in response to redox stress ; positive controi of the micF antisense RNA by the soxHS locus , J Bacterioi M5 : 1026-1031 , Gompan , I. , and Touati , D. ( 1993 ) Interaction of six global transcription regulators in expression of manganese superoxide dismutase in Escherichia coli K-12 , J Bacteriot 175 : 1687-1696 , Goyer , J , , Andersen , J , , Forst , S.S , , Inouye , M. , and Deiius , N. ( 1990 ) m/cF RNA in ompB mutants of Escherichia coti : different pathways regulate m/cF RNA levels in response to osmoiarity and temperature change , J Bacteriot 172 ; 4143-4150 , Gunningham , R , P , , Saporito , S.M. , Spitzer , S.G. , and Weiss , B. ( 1986 ) Endonuclease IV ( nfo ) mutant of Escherichia coli J Bacteriot 16S : 1120-1127 , Gallegos , M,-T , , Michan , G , , and Ramos .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7891555" target="_blank">7891555</a>]<br></li>
<li>Therefore , multiple systems ( EnvZ/OmpR , Cpx , Rcs , and H-NS ) appear to be involved in sensing environmental osmolarity , leading to sophisticated regulation of the curli genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15743952" target="_blank">15743952</a>]<br></li>
<li>The identiﬁcation and characterization of new OmpR-regulated genes will be valuable for uncovering the general mechanism by which OmpR controls the transcription of its regulon members and for discovering the global role of OmpR in the E. coli stress response .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15175316" target="_blank">15175316</a>]<br></li>
<li>FEMS Microbiology Letters 186 ( 2000 ) 257 ^ 262 www.fems-microbiology.org Negative regulation of the bolA1p of Escherichia coli K-12 by the R for transcription factor Omp osmolarity response genes Kaneyoshi Yamamoto a , Rie Nagura a , Hiroyuki Tanabe a , Nobuyuki Fujita b , Akira Ishihama b , Ryutaro Utsumi a ; * a Department of Agricultural Chemistry , Kinki University , 3327-204 Nakamachi , Nara 631-8505 , Japan b Department of Molecular Genetics , National Institute of Genetics , Mishima , Shizuoka 411-8540 , Japan Received 10 January 2000 ; received in revised form 13 March 2000 ; accepted 21 March 2000 Abstract To test whether OmpR is involved in regulation of the bolA1p , we investigated possible effects of ompR mutation on transcription from bolA1p .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10802181" target="_blank">10802181</a>]<br></li>
<li>Negative regulation of the bolA1p of Escherichia coli K-12 by the transcription factor OmpR for osmolarity response genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16159766" target="_blank">16159766</a>]<br></li>
<li>Yamamoto , K. , Nagura , R. , Tanabe , H. , Fujita , N. , Ishihama , A. , and Utsumi , R. ( 2000 ) Negative regulation of the bolA1p of Escherichia coli K-12 by the transcription factor OmpR for osmolarity response genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16194231" target="_blank">16194231</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12366850" target="_blank">12366850</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12492863" target="_blank">12492863</a>]<br></li>
<li>Transport/solute diffusion dctA efeU fadL mglB nhaB ompF putP C4 dicarboxylate transporter Ferrous iron-transporter IM IM E Long-chain fatty acid uptake Galactose periplasmic binding protein Sodium/proton antiporter Porin Sodium/proline symporter Putative peptide importer Proton-dependent peptide importer IM PP IM OM IM IM IM E sbmA tppB P Protein folding cpxP dsbA spy yccA Putative periplasmic chaperone Disfulﬁde oxidase Chaperone Regulator of FtsH proteolysis E PP PP PP IM E E E Cell wall dacC slt ycbB ygaU IM D-Alanyl-D-alanine carboxypeptidase , PBP6 Lytic murein transglycosylase L,D-Transpeptidase LysM cell-wall-degradation motif OM IM IM P P Regulation ﬂhC mzrA ydeH Transcriptional regulator of ﬂagella Modulator of EnvZ/OmpR C E IM Diguanylate cyclase regulator of motility and bioﬁlm E C Translation rmf Ribosome modulation factor C raiA Ribosome-associated inhibitor a , upregulated transcription in the presence of NlpE overexpression ; , downregulation in the presence of NlpE overexpression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23564175" target="_blank">23564175</a>]<br></li>
<li>Finally , three genes are involved in regulation of motility and bioﬁlm formation ( ydeH [ upregulation ] and ﬂhC OM IM IM P P Regulation ﬂhC mzrA ydeH Transcriptional regulator of ﬂagella Modulator of EnvZ/OmpR C E IM Diguanylate cyclase regulator of motility and bioﬁlm E C Translation rmf Ribosome modulation factor C raiA Ribosome-associated inhibitor a , upregulated transcription in the presence of NlpE overexpression ; , downregulation in the presence of NlpE overexpression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23564175" target="_blank">23564175</a>]<br></li>
<li>Fold change in mRNA level with Effect of ∆ envZ-ompRa Function and comments pBAD-OmrA pBAD-OmrB Gene A ompT − 9.1 ± 4.3 − 4.1 ± 0.2 − 3.8 ± 1.1 − 3.0 ± 1.0 − 3.1 ± 0.6 − 10.1 ± 8.3 − 6.1 ± 2.6 − 6.3 ± 2.4 − 3.5 ± 0.7 − 1.9 ± 0.4 OM protein , protease cirA yjcG yjcH yahO dld 2.7 OM protein , receptor for colicin I , participates in iron-transport Acetate/glyoxalate permease ORF , hypothetical protein ORF , hypothetical protein − 2.6 ± 0.0 − 2.6 ± 0.1 − 1.3 ± 0.1 − 1.3 ± 0.1 − 1.3 ± 0.0 0.04 ± 1.7 − 0.3 ± 1.9 − 0.08 ± 1.6 − 2.3 ± 0.8 − 2.2 ± 0.1 − 2.1 ± 0.1 − 1.6 ± 0.2 − 1.3 ± 0.1 − 1.4 ± 0.2 − 2.3 ± 0.8 − 0.04 ± 1.6 − 1.8 ± 0.3 − 1.2 ± 0.2 − 2.2 ± 0.4 − 1.7 ± 0.0 D-lactate dehydrogenase , FAD protein , NADH independent Major type 1 subunit ﬁmbrin ( pilin ) Fimbrial protein Periplasmic chaperone , required for type 1 ﬁmbriae OM protein ; export and assembly of type 1 ﬁmbrae , interrupted Fimbrial morphology Fimbrial morphology Minor ﬁmbrial subunit , D-mannose speciﬁc adhesin ORF , hypothetical protein OM protein , receptor for transport of vitamin B12 , E colicins , and bacteriophage BF23 Putative periplasmic binding transport protein Kinase/phosphatase sensor for OmpR , modulates expression of ompF and ompC Response regulator affecting transcription of ompF and ompC Probable serine transporter L-serine dehydratase ( deaminase ) ﬁmA ﬁmI 2.4 ﬁmC ﬁmD ﬁmF ﬁmG ﬁmH yohN btuB ﬂiY 2.8 − 2.2 ± 0.3 − 2.3 ± 0.5 − 2.2 ± 0.6 − 2.1 ± 0.2 envZ ompR sdaC sdaB csgG csgF csgE csgD csgB csgA folP glmM gntP ykfBA osmBA yiaL − 2.2 ± 0.6 − 1.7 ± 0.4 − 1.7 ± 0.2 − 1.3 ± 0.3 − 1.7 ± 0.5 − 5.1 ± 2.0 − 1.6 ± 0.4 − 0.3 ± 1.8 − 2.6 ± 0.1 − 2.0 ± 0.0 − 1.9 ± 0.4 − 2.3 ± 0.8 − 1.2 ± 0.1 − 0.0 ± 1.7 − 2.0 ± 0.2 − 2.2 ± 0.4 − 2.5 ± 1.1 − 2.1 ± 0.3 − 1.5 ± 0.4 − 2.0 ± 0.2 − 4.6 ± 1.5 − 1.6 ± 0.2 − 0.3 ± 1.8 − 1.9 ± 0.3 − 1.9 ± 0.4 − 1.8 ± 0.3 − 1.5 ± 0.3 − 2.2 ± 0.6 − 2.1 ± 0.5 − 1.7 ± 0.8 Curli production component Curli production assembly/transport component , 2nd curli operon Curli production assembly/transport component Transcriptional activator Curlin , minor subunit precursor Curlin , major subunit 7,8-dihydropteroate synthase ; in operon with the adjacent glmM ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16359331" target="_blank">16359331</a>]<br></li>
<li>G Plasmid isolate ( fragment boundaries ) Name pK4-14 ( 1862417 -- 1865705 ) yeaD yeaE mipA rrb2 ( rep : : kan ) and rrb4 ( zaa-1000 : : kan ) Yes ; stim ulation Unknown function , conserved Methylglyoxal reductase Scaffolding protein for murein polymerase and murein hydrolase Unknown function , conserved yeaG rrb4 ( zaa-1000 : : kan ) pK4-29 ( 1403515 -- 1405770 ) No Putative LysR-type transcriptional regulators sRNA Unknown function Unknown function , conserved abgR isrA ydaL ydaM rrb4 ( zaa-1000 : : kan ) pK4-55 ( 3531018 -- 3535617 ) Yes ; stimulation ( complementation ) Phosphoenolpyruvate carboxykinase Inner membrane osomosenser protein , regulates activity of OmpR Transcriptional regulator involved in osmoregulation of OmpC and OmpF Transcriptional elongation factor Unknown function pckA envZ ompR greB yhgF rrb2 ( rep : : kan ) pK2-8 ( 2805651 -- 2809436 ) No High-afﬁnity transport for glycine , betaine , and proline Uncharacterized member of the major facilitator superfamily of transporters Predicted transporter Predicted inner membrane protein Negatively regulates transcription of the emrRAB operon encoding subunits of a multidrug efﬂux pump ; acts here via OmpR proX ygaY ygaZ ygaH emrR a Description of genes from EcoCyc ( 27 , 28 ) and results from this paper .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17416652" target="_blank">17416652</a>]<br></li>
<li>Doubling component regulated regulated of altered time Identiﬁed function system ( HK/RR ) Relevant functions genes genes genes ( min ) in this experiment Strain name Mutation Class DcuS/DcuR Response to C4-dicarboxylate ( Golby et al. , 1999 ) Chemotaxis ( Stock and Surette , 1996 ) 10 29 39 27 BW 27878 DE ( dcuSR ) 1330 28079a Flagellar synthesis CheA/CheB , CheYa 13 28 41 25 BW DE ( ﬂhEAB cheZYBR tap tar cheWA motBA ﬂhCD IS1 ) 1218a DE ( atoSC ) 569 11 32 43 27 Flagellar synthesis BW28878 AtoS/AtoC Acetoacetate-metabolism ( Chen et al. , 1991 ; Rhie and Dennis , 1995 ) YpdA/YpdB Energy metabolism , BW27875 ﬂagellar synthesis BW27865 DE ( ypdAB ) 1334 24 18 42 31 NarQ / DnarQ1314 25 45 29 Nitrate regulation ( Stewart and Rabin , 1995 ) 20 / YfhA / RcsB DyfhA1326 DrcsB1320 17 11 46 46 28 38 BW BW 29430 27870 29 35 Capsular synthesis ( Stout and Gottesman , 1990 ) Regulation of the citrate fermentation ( Kaspar and Bott , 2002 ) Sigma-S degradation ( Klauck et al. , 2001 ) Flagellar synthesis , maltose-transport Flagellar synthesis , maltose-transport CitA/CitB DE ( citAB ) 1289 23 61 BW 27876 31 38 / RssB BW27555 23 64 25 41 DrssB1273 rpoS regulon Drastic alteration Respiratory control ( Iuchi and Lin , 1995 ) / ArcA 72 38 49 110 DarcA43 TCA-cycle , ﬂagellar synthesis Respiration , rpoS regulon , maltose-transport Amino acid metabolism ( cys , leu ) , transporter , ﬂagellar synthesis , enterochelin TCA-cycle , chemotaxis , osmotic adaptation , protein peptide secretion , purine-ribonucleotide-biosynthesis , rpoS regulon , enterochelin , maltose-transport BW 27422 Hydrogen peroxide sensitivity ( Pernestig et al. , 2001 ) / UvrY 86 37 45 123 DuvrY1296 BW 27874 EnvZ/OmpR 71 54 125 35 DE ( ompR-envZ ) 520 Osmotic regulation BW 26424 ( Pratt and Silhavy , 1995 ) Respiratory control ( Iuchi and 147 Lin , 1995 ) ArcB / 112 259 44 BW26422 DarcB41 a .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12366850" target="_blank">12366850</a>]<br></li>
<li>Effect of genes whose inactivation prevented repression of hdeAB by M arA on MarA-mediated MD R Level of MarA-mediated MD a level ) R ( % of parental Locusb Gene Protein/function FOX MIN NOR CHL Transcriptional regulators Cold shock protein CspG cspGpc crp hns ompR b0990 b3357 b1237 b3405 54 21 19 67 91 33 23 77 93 29 20 93 75 38 Cyclic AMP receptor protein DNA-binding protein H-NS OmpR transcriptional dual regulator 35 100 Enzymes Acid phosphatase appA cyaA degP maoC metL pcnB recD treC ynfE b0980 b3806 b0161 b1387 b3940 b0143 b2819 b4239 b1587 68 43 84 41 91 61 91 23 77 46 68 93 46 100 32 100 40 100 Adenylate cyclase Serine protease Do Putative ring-cleavage enzyme of phenylacetate-degradation Aspartate kinase/homoserine dehydrogenase Poly ( A ) polymerase I ; RNA modiﬁcation RecBCD DNA helicase/exonuclease Trehalose-6-phosphate hydrolase Oxidoreductase subunit paralog of DmsA 10 80 94 94 50 75 100 33 50 85 93 Transport acrA acrB mhpT nikD tolC b0463 b0462 b0353 b3479 b3035 16 6 68 68 0 8 8 40 40 0 9 AcrAB-TolC multidrug efﬂux system AcrAB-TolC multidrug efﬂux system Putative propionic acid transporter Subunit of nickel ABC transporter TolC outer membrane channel 100 50 0 100 125 40 91 68 0 Others Putative membrane protein alx b3088 b3388 b2816 b2350 120 40 74 68 68 46 91 91 100 50 75 93 65 30 65 65 Predicted membrane-anchored protein tRNA-methionine CPS-53 ( KpLE1 ) prophage ; bactoprenol-linked glucose translocase ( ﬂippase ) Conserved protein Predicted unconserved protein damX metV yfdG yibLpc yniDpc b3602 b4535 68 94 68 77 50 100 50 125 a Abbreviations : FOX , cefoxitin ; NOR , norﬂoxacin ; CHL , chloramphenicol ; MIN , minocycline .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20211899" target="_blank">20211899</a>]<br></li>
<li>of T-even like phages and also acts as a porin with low permeability that allows slow penetration of small solutes ; induced by low pH during aerobic or anaerobic growth but decreases by phosphate limitation and SOS induction Forms passive diffusion pores which allow small-molecular-wt hydrophilic materials across the outer membrane ; negatively regulated by Lrp Forms passive diffusion pores which allow low-mol wt hydrophilic materials across the outer membrane ; receptor for the bacteriophage T2 ; positively regulated by Lrp ; decreases during phosphate limitation and after benzoic acid treatment and treatment with superoxidegenerating agents such as menadione P33602 NuoG NusA 4.53 / 54,870.92 4.55 / 61,127 P0AFF6 133 , 179 , 212 , 228 , 294 P0A780 N utilization substance protein B 6.6 / 15,689.06 NusB 179 NusG 6.33 / 20,400.32 5.93 / 22,161 5.44 / 14,684 ( 5 -- 6 ) 6.40 / 21,860 ( 6 -- 11 ) P0AFG0 Transcription antitermination protein 179 , 286 , 287 OmpA 5.55 / 34,419 5.36 / 32,707 5.60 / 35,172.28 P0A910 Outer membrane protein A 75 , 179 , 228 , 274 , 287 , 293 , 294 , 321 , 323 5.34 / 28,591 5.50 / 33,875 ( DIGE 4.5 -- 6.5 ) 5.38 / 33,875 ( DIGE 4.5 -- 6.5 ) 5.23 / 14,346 ( 5 -- 6 ) 5.25 / 13,108 ( 5 -- 6 ) 4.48 / 38,307.50 -- / 36,000 Outer membrane protein C P06996 OmpC 61 OmpF 4.61 / 36,170 4.64 / 37,084.49 Outer membrane protein F P02931 61 , 88 , 179 , 228 , 293 , 294 , 321 4.56 / 37,316 ( DIGE 4.5 -- 6.5 ) 4.64 / 37,000 ( DIGE 4.5 -- 6.5 ) 4.71 / 37,000 ( DIGE 4.5 -- 6.5 ) Continued on facing pag The N terminus of this protein is required for the transcriptional expression of both major outer membrane protein genes ompF and ompC ; its carboxyl-terminal moiety mediates the multimerization of the ompR protein Protease that can cleave T7 RNA polymerase , ferric enterobactin receptor protein ( FEP ) , antimicrobial peptide protamine , and other proteins Induced by low or high pH 79 OmpR P0AA16 Transcriptional regulatory protein 6.04 / 27,353.62 1 OmpT Protease VII P09169 179 , 321 5.38 / 33,477.72 5.36 / 37,000 ( DIGE 4.5 -- 6.5 ) OmpX Outer membrane protein X P0A917 5.30 / 16,382.89 5.25 / 16,126 ( DIGE 4.5 -- 6.5 ) 5.15 / 16,000 ( DIGE 4.5 -- 6.5 ) 5.93 / 56,137 5.70 / 57,487 5.64 / 56,808 5.44 / 54,493 5.93 / 55,635 ( DIGE 4.5 -- 6.5 ) 5.82 / 55,131 ( DIGE 4.5 -- 6.5 ) 5.71 / 52,681 ( DIGE 4.5 -- 6.5 ) 5.13 / 50,973 ( 5 -- 6 ) 5.47 / 50,921 ( 5 -- 6 ) 5.63 / 50,921 ( 5 -- 6 ) 5.65 / 50,915 ( 5.5 -- 6.7 ) 179 , 274 , 321 Component of the oligopeptide permease and binding-protein-dependent transport system ; binds peptides up to 5 amino acids long with high afﬁnity ; increases by high pH during anaerobic growth and after benzoic acid treatment OppA 5.85 / 58,359.84 Periplasmic oligopeptide-binding protein P23843 179 , 228 , 286 , 287 , 294 , 321 , 323 Part of the binding-protein-dependent transport system for oligopeptides ; probably responsible for energy coupling to the transport system Provides resistance to osmotic stress ; may be important for stationary-phase survival ; induced by elevated osmotic pressure in the growth medium Preferentially metabolizes organic hydroperoxides over inorganic hydrogen peroxide ; induced by elevated osmotic pressure and during the decelerating phase , before entry into stationary phase ; regulated by a complex network of several global regulators , including at least S , Lrp , and H-NS Induced by elevated osmotic pressure Osmotically and stationary phase inducible and controlled by several global regulators , including S , cAMP-CRP , IHF , and Lrp Induced by growth in glucose-mineral medium of elevated osmotic strength ; controlled by S Induced by growth in glucose-mineral medium of elevated osmotic strength ; controlled by S May play a role in bacterial envelope integrity ; strongly associated with the peptidoglycan Involved in pantothenate-biosynthesis : ﬁrst step See PanB : ﬁnal step .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16760308" target="_blank">16760308</a>]<br></li>
<li>Functionb Other regulator ( s ) Reference ( s ) Rich Minimal Medium Medium No 15 12 Yes 24 18 No 6 6 ybaY gabP ugpE Unknown GABA-transporter protein b0453 b2663 b3451 GabC ( ) , Nac ( ) PhoBR ( ) 55 , 66 , 93 15 G-3-P transporter membrane bound component ATP binding protein of G-3-P transporter Putative binding protein of ABC transporter Hypothetical ATP-binding protein of ABC transporter Beta subunit of nitrate reductase-II Subunit of cytochrome bd-II oxidase Lysine decarboxylase Induced by alkylating agents ; protects DNA from alkylation damage Hydroperoxidase II Universal stress protein B ; required for ethanol resistance Periplasmic protein induced by high osmolarity ; function unknown Multidrug efﬂux pump No 20 9 Yes 6 2 No 28 21 ugpC ydcS yehX b3450 b1440 b2129 PhoBR ( ) Nac ( ) 15 93 narY appB IdcC aidB b1467 b0979 b0186 b4187 13 23 Yes Yes Yes Yes 10 25 7 36 2 7 AppR ( ) 47 , 79 45 , 46 11 33 Lrp ( ) Yes 40 66 Yes 4 8 katE uspB b1732 b3494 74 26 Yes 35 64 osmY Lrp ( ) , Crp ( ) , IHF ( ) EvgA ( ) , Lrp ( ) b4376 20 , 90 , 91 No 31 yhiV b3514 56 , 57 , 74 , 78 56 , 57 , 78 25 72 , 74 12 77 24 4 yhiU mltB otsA ecnB argH aroM Multidrug efﬂux pump EvgA ( ) b3513 b2701 b1896 8 2 Yes No Yes Yes No No 39 2 31 23 2 15 Membrane-bound lytic transglycosylase B Trehalose phosphate synthase Bacteriolytic lipoprotein entericidin B Argino-succinate lyase Unknown ; may play a role in aromatic-amino-acid biosynthesis Putative lipase 2,5-diketo-D-gluconic acid reductase A ; catalyzes the conversion of 2,5-KGDR to 2-KLG Aldehyde dehydrogenase B ; converts lactaldehyde and NAD to lactate and NADH Succinate-semialdehyde dehydrogenase Membrane-bound subunit of carbonphosphorus lyase complex Transaldolase A ; converts Sed-7-P and Gly-3-P to Fru-6-P and Ery-4-P Putative glutathione S-transferase Putative dehydrogenase Unknown Unknown Unknown Unknown Unknown Unknown Unknown Unknown Unknown Unknown Unknown Unknown Unknown Unknown Unknown Unknown Unknown Lrp ( ) 57 74 5 78 ( ) EnvZ/OmpR ArgR ( ) TyrR ( ) b3960 b0390 No 15 No 8 yhjY yqhE b3548 b3012 9 92 11 aldB 22 Yes 21 b3588 74 , 87 gabD phnP B2661 B4092 16 8 No No GabC ( ) , Nac ( ) PhnF and PhnO ( ) 55 , 93 51 13 2 talA B2464 26 20 No CreBC ( ) 7 , 70 yfcG yliI yjbJ yjbE ygaU ygdI ygaF yjgR ydaM ydcK yhiN yebF yjiN yeaG yodC ybiO yhjG yphA yhjD B2302 B0837 B4045 B4026 B2665 B2809 B2660 B4263 B1341 B1428 B3492 B1847 B4336 B1783 B1957 B0808 B3524 B2543 B3522 52 37 37 17 22 22 4 7 3 6 16 15 5 41 4 13 41 13 17 24 5 No Yes No No No No No No No No No No No Yes No No No No Yes SdiA ( ) 84 66 179 58 12 5 7 3 23 9 7 25 6 10 17 26 15 18 34 Lrp ( ) 74 63 , 74 Lrp ( ) Lrp ( ) 63 a Genes were identiﬁed using the BLAST algorithm ( 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15576800" target="_blank">15576800</a>]<br></li>
				</div></ul><br>
			<label class="collapse_SIT" for="_5"><font size="5px">OmpR binding site</font></label>
				<input id="_5" type="checkbox">
				<div class="SIT"><br><ul><li>Similarities ' at NarX residues : Sequence Sequence 330 to 445 to 510 to 524 to DegS UhpB FixL PhoR EnvZ CpxA 20 ( 2 ) 16 ( 3 ) 6 ( 5 ) 10 ( 5 ) 8 ( 5 ) 13 ( 4 ) 11 ( 1 ) 12 ( 4 ) 3 ( 2 ) 4 ( 3 ) 4 ( 2 ) 3 ( 1 ) 1 ( 1 ) 2 ( 1 ) 3 ( 1 ) 6 ( 3 ) 7 ( 0 ) 9 ( 0 ) 4 ( 1 ) 1 ( 2 ) 0 ( 2 ) 0 ( 1 ) 0 ( 1 ) 0 ( 1 ) DegU UhpA FixJ PhoB OmpR SfrA 42 ( 15 ) 38 ( 7 ) 26 ( 9 ) 31 ( 13 ) 35 ( 8 ) 30 ( 11 ) 30 ( 7 ) 26 ( 8 ) 27 ( 6 ) 10 ( 12 ) 10 ( 13 ) 9 ( 10 ) We thank J. A. DeMoss and E. L. Sodergren for sharing data in advance of publication and T. Thannhauser for synthesizing oligo-nucleotides .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2649492" target="_blank">2649492</a>]<br></li>
<li>Residues that are conserved in all four sequences are marked with asterisks 57 58 59 60 KdpE PhoB CreB OmpR * * * * ** * * ***** *** * ** GDGIEFIRDLRQ WSA VPVIVLSARSEESDKIAALDAGADDYLSKPFGIGELQARLRV GSGIQFIKHLKRESMTRDIPVVMLTARGEEEDRVRGLETGADDYITKPFSPKELVARIKA ISGFELCRQLLA LHPALPVLFLTARSEEVDRLLGLEIGADDYVAKPFSPREVCARVRT EDGLSICRRLRS QSNPMPIIMVTAKGEEVDRIVGLEIGADDYIPKPFNPRELLARIRA KdpE PhoB CreB OmpR 114 118 117 118 ** ** * ALRRHSATTAPDP LVKFSDVTVDLAARVIHRGEEEVHLTPIEFRLAGRCSTMPEK VMRRISP MAVEE VIEMQGLSLDPTSHRVMAGEEPLEMGPTEFKLLHFFMTHPER LLRRVKKFSTPSP VIRIGHFELNEPAAQISWFDTPLALTRYEFLLLKTLLKSPGR VLRRQANELPGAPSQEEAVIAFGKFKLNLGTREMFREDEPMPLTSGEFAVLKALVSHPRE KdpE PhoB CreB OmpR KdpE YSPSGPVLNQVWGPNAVEHSHYLRIYMGHLRQ 201 PhoB VYSREQLLNHVWGTNVYVEDRTVDVHIRRLRK 205 CreB VWSRQQLMDSVWEDAQDTYDRTVDTHIKTLRA 204 OmpR PLSRDKLMNLARGREYSAMERSIDVQISRLRR 210 FIG. 6 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1532388" target="_blank">1532388</a>]<br></li>
<li>For construction of plasmids for His-tagged transcription factors , pCpxR , pH-NS , pOmpR and pRstA , DNA fragments corresponding to the coding sequences of the respective transcription factors were amplified by PCR using E. coli W3110 genomic DNA as a template and pairs of primers ( Table 1 ) designed to hybridize upstream or downstream of each coding sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20466769" target="_blank">20466769</a>]<br></li>
<li>Expression and purification of transcription factors For construction of plasmids ( pMlrA and pOmpR ) for expression of His-tagged MlrA and OmpR , DNA fragments corresponding to the coding sequences of their respective transcription factors were ampliﬁed by PCR using E. coli W3110 genome DNA as a template and pairs of primers , which were designed so as to hybridize upstream or downstream of each coding sequence ( Table S2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20874755" target="_blank">20874755</a>]<br></li>
<li>The precise positions of nucleotides were determined by comparison with a parallel run of the Maxam-Gilbert G reaction ( lane M ) The result indicates that OmpR pre-incubated with EnvZ 11 plus ATP protected two regions between nucleo-tides +45 and +90 and between +100 and +127 , downstream from the transcriptional start site offadL .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8413182" target="_blank">8413182</a>]<br></li>
<li>Phosphorylation Stimulates Cooperative Interactions Between OmpR Molecules Bound at Neighboring Sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9096305" target="_blank">9096305</a>]<br></li>
<li>As shown in Fig. 6 , OmpR appeared to bind to theompC XbaI and the BamHI ends in separate experiments .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3023382" target="_blank">3023382</a>]<br></li>
<li>Directly repeated sequences are also thought to be the recognition sites for other transcriptional activators such as OmpR and ToxR proteins ( Mizuno 8r Mizushima , 1986 ; Miller et al. , 1987 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3054125" target="_blank">3054125</a>]<br></li>
<li>Phosphorylation may promote oligomerization of OmpR , which is normally monomeric , thereby enhancing its ability to bind at tandemly repeated target sites ( Nakashima et al. , 1991 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8098993" target="_blank">8098993</a>]<br></li>
<li>MzrA links CpxA/CpxR and EnvZ/OmpR regulons The yqjA and mzrA open reading frames are separated by three nucleotides and are thought to be cotranscribed from s70 ( p1 ) - and sE ( p2 ) - dependent promoters located upstream of yqjA ( Dartigalongue et al. , 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19432797" target="_blank">19432797</a>]<br></li>
<li>The OmpR site necessary for negative osmoregulation is located at approximately positions -55 to -41 ( Mizuno et al. 1988 ; Tsung et al. 1989 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1736095" target="_blank">1736095</a>]<br></li>
<li>OmpR , the response regulator of the EnvZ-OmpR two-component system , which senses the environmental osmolarity , was found to bind a 24 bp sequence ( 262 to 239 ) ( Fig. 5 ) including a direct repeat sequence of ( 259 ) TTACATTTAgTTACATGTT ( 240 ) ( Fig. 6 ) , which agrees with the OmpR box ( Harlocker et al. , 1995 ; Pratt & Silhavy , 1995 ; Ishihama , 2010 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20466769" target="_blank">20466769</a>]<br></li>
<li>None of the portions of the OmpR protein was , however , found to coincide with the consensus sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3010044" target="_blank">3010044</a>]<br></li>
<li>Further supporting these results , it was found that part of the sequence bound by OmpR shares similarity with the OmpR consensus sequence previously proposed ( Harlocker et al. , 1995 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25667010" target="_blank">25667010</a>]<br></li>
<li>The ompR Promoter-The open reading frame between nucleotides 252 and 1103 in Fig. 3 is the coding sequence for the OmpR protein based upon the protein sequencing data described above .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6292199" target="_blank">6292199</a>]<br></li>
<li>In addition , the sequence ATGA was found beginning at bp 968 ; this sequence forms the TGA termination codon for the OmpR protein as well as a possible ATG initiation codon in the reading frame of EnvZ .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2997120" target="_blank">2997120</a>]<br></li>
<li>However , it now seems clear from the data presented in this paper that the sequence ATGA at bp 968 forms an bverlapping terminator for OmpR translation and initiator for EnvZ translation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2997120" target="_blank">2997120</a>]<br></li>
<li>The coding regions of OmpR and OmpR234 were ampliﬁed by PCR from chromosomal DNA of , respectively , MG1655 and MG1655 ompR234 strains and by using the primers R1 ( 5 - AGTACAAACCATGGAAGAGAACTAC-3 ) and R2 ( 5 - CTTCGTACGCGAAAGCTTTATTAAACTG-3 ) carrying , respectively , an NcoI and an HindIII site ( italic sequences ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11717281" target="_blank">11717281</a>]<br></li>
<li>In case of OmpR , the isolated DBD ( OmpRc ) binds weakly to recognition sites F1 and C1 at the ompR and ompC promoters respectively [ 57,58 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22291906" target="_blank">22291906</a>]<br></li>
<li>To test this hypothesis , Igo and Silhavy ( 23 ) examined the sequences between - 1158 and - 238 for a region that was protected by OmpR from DNase I digestion .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8113170" target="_blank">8113170</a>]<br></li>
<li>Both sequences are derived by deduction from the DNA sequences , Dye in the present paper and OmpR in Ref .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2984198" target="_blank">2984198</a>]<br></li>
<li>and the ompC promoterfragments , suggesting that OmpR binds to specific DNA sequences within these fragments .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3023382" target="_blank">3023382</a>]<br></li>
<li>Therefore , it seems likely that OmpR , in its phosphorylated form , binds to DNA sequences upstream of position − 60 of the omrA and omrB promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16359331" target="_blank">16359331</a>]<br></li>
<li>Similarly , the OmpR boxes found in the ompC and ompF promoters correspond to asymmetric , directly repeated decameric sequences , and they are also A/T rich , as are the pho and tor boxes ( Rampersaud et aL , 1994 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596446" target="_blank">8596446</a>]<br></li>
<li>If T83I can bind to ompF promo-ter sequences in vivo , the T83I phenotype should be dominant to wild-type OmpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11812125" target="_blank">11812125</a>]<br></li>
<li>MICROBIOLOGY LETTERS ELSEVIER F'EMSMicrobiology Letters 131 ( 1995 ) 147-151 Molecular analysis of OmpR binding sequences involved in the regulation of ompF in Escherichia coli S. Forst * , I. Kalve , W. Durski Department of Biological Sciences , Universiryof Wisconsin-Milwaukee , PO Box 413 , Milwaukee , WI53201 , USA Received 9 May 199.5 ; revised 21 June 1995 ; accepted 23 June 1995 Abstract OmpR , the transcriptional regulatory protein of ompF , had not been previously shown to specifically bind to the - 70 to - 60-bp region of ompF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7557323" target="_blank">7557323</a>]<br></li>
<li>Bergstrom , L.C. , Qin , L. , Harlocker , S.L. , Egger , L.A. , and Inouye , M. ( 1998 ) Hierarchical and co-operative binding of OmpR to a fusion construct containing the ompC and ompF upstream regulatory sequences of Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16135237" target="_blank">16135237</a>]<br></li>
				<label class="SIT_collapse_similarity" for="sit_similarity_28">
Harlocker , S. , Bergstrom , L. and Inouye , M. ( 1995 ) Tandem binding of six OmpR proteins to the ompF upstream regulatory sequence of Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9130712" target="_blank">9130712</a>]<br></label>
<input id="sit_similarity_28", type="checkbox">
<div class="SIT_similar">
<li>Harlocker , S.L. , Bergstrom , L. , and Inouye , M. ( 1995 ) Tandem binding of six OmpR proteins to the ompF upstream regulatory sequence of Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16359331" target="_blank">16359331</a>]<br></li>
<li>Tandem binding of six OmpR proteins to the ompF upstream regulatory sequence of Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25667010" target="_blank">25667010</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11976297" target="_blank">11976297</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15175316" target="_blank">15175316</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10648521" target="_blank">10648521</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9677290" target="_blank">9677290</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11717281" target="_blank">11717281</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9813115" target="_blank">9813115</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20466769" target="_blank">20466769</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12897000" target="_blank">12897000</a>]<br></li></div>
<li>Deletion of the tothis sequence andstrainsharboringthis allele produce uruOl and u r d sites decreases the ability ofAraCto bind to the OmpR constitutively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8175665" target="_blank">8175665</a>]<br></li>
<li>CpxR , CsgD , CRP , RstA , OmpR and Cra-binding sites are indicated csgB coding sequence TTGTTTTTCATGTTGTCACCCTGGACCTGGTCGTACATTTAAGAAATTAAATCATTTCAACTTGGTTGT TAACGCAACCTGTATTTTGTTAACGCTGCGTTACGATGGAAAGTATGTCTGCGGAAATATTTTTAATAA csgBp CTCACCCGCGCGTTGTATTTTTCTTTTTTAGTTCATACCAAAGTATTAAAACCCACGTAAATACGCTGT TATCTACGCAAAAAAATATTTTGTTTTCTTTTAAATCTCCGTTTTCCGCTATCAAAAAGCACCAGACAG TCATTCTTCTTGCCCGTCGCTGATTGCTGCGATATGTCTTGCGCACAAGCCGTGACAAACCTCGTCTTT TTGCAGTAAAAAATTGTCCACGGAGGTGTGGAGAAAAAACAAGAACGTTTTACATGACGAAAGGACTAC ACCGAAATATTTTTTATATGCATTATTAGTAAGTTATCACCATTTGTATGATTTTTTAAAATTGTGCAA TAAAAACCAAATGTACAACTTTTCTATCATTTCTAAACTTAATAAAACCTTAAGGTTAACATTTTAATA csgDp1 TAACGAGTTACATTTAGTTACATGTTTAACACTTGATTTAAGATTTGTAATGGCTAGATTGAAATCAGA csgDp2 TGTAATCCATTAGTTTTATATTTTACCCATTTAGGGCTGATTTATTACTACACACAGCAGTGCAACATC TGTCAGTACTTCTGGTGCTTCTATTTTAGAGGCAGCTGTCAGGTGTGCGATCAATAAAAAAAGCGGGGT csgD coding sequence TTCATCATGTTTAATGAAGTCCATAGTAT RstA-P CpxR-P CpxR-P CpxR-P csgDp1 OmpR csgDp2 Cra Cra CRP-AMP CRP-AMP CpxR-P CpxR-P Cra CpxR-P Cra CsgD csgBp J. Altenbuchner and S. E. Lindow , respectively , while bacteriophage P1vir was a kind gift from S. K. Apte .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21559929" target="_blank">21559929</a>]<br></li>
<li>A monomer of OmpR is thought to bind to a DNA region that spans ten base pairs .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9016718" target="_blank">9016718</a>]<br></li>
<li>Three base pairs were Lanes 5-8 represent DNA binding activities of OmpR following a 30-s altered in the Cd box ( -42 to -51 ) , which changed the sequence from ( lane5 ) ,1 - min ( lane 6 ) ,5 - min ( lane 7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8175665" target="_blank">8175665</a>]<br></li>
<li>Increased production of the CsgD transcription activator results in activation of the csgBA operon ; when CsgD is expressed independently of OmpR , the latter becomes dispensable for activation of the csgB promoter ( Table 2 ) Both genetic and biochemical evidence shows that OmpR binds to an imperfect direct repeat ( 5 - GTTACATTTA/GTT ACATGTT-3 ) centered at position 49.5 relative to the transcriptional start site of the csgD promoter ( Table 2 and Fig. 1 and 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11717281" target="_blank">11717281</a>]<br></li>
<li>Thus , OmpR when pre-incubated with EnvZ11 , binds at least four regions of DNA near the fadL promoter , two upstream and two downstream from the transcriptional start site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8413182" target="_blank">8413182</a>]<br></li>
<li>The results of gel retardation and DNase I protection experiments indicated that OmpR , after incubation with the protein kinase EnvZ , specifically binds to at least four sites around the fadL promoter , two upstream and two downstream from the transcriptional start site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8413182" target="_blank">8413182</a>]<br></li>
<li>Gel retardation ( Fig. 6 ) and footprinting ( Figs. 7 and 8 ) experiments further indicated that OmpR specifically binds four regions around the promoter of thefadL gene , two upstream and two downstream of the transcriptional initiation site ( Fig. 9 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8413182" target="_blank">8413182</a>]<br></li>
<li>Our DNase I footprinting analysis provides strong evidence that OmpR binds to a speciﬁc site in the tppB regulatory region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15175316" target="_blank">15175316</a>]<br></li>
<li>Their preliminary DNA footprinting studies suggested that OmpR binds to a region from approximately -380 to -350 upstream of the ompF transcription start site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8113170" target="_blank">8113170</a>]<br></li>
<li>DNase I protection studies reveal that OmpR binds to the region between -40 and -100 from the start site of transcription ( 24 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1648075" target="_blank">1648075</a>]<br></li>
<li>Recent studies from this laboratory reveal that OmpR also binds to the region from approximately -350 to -380 from the start site of transcription ( 12 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1648075" target="_blank">1648075</a>]<br></li>
<li>Loop formation is thought to result from the interaction between OmpR bound at the - 384 to -351 - bp and -56 to -41 - bp sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7557323" target="_blank">7557323</a>]<br></li>
<li>OmpR has previously been shown to bind to a series of sites upstream of ompF ( 25 , 26 ) that are located between - 96 and - 41 ( Fig. 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8113170" target="_blank">8113170</a>]<br></li>
<li>We would predict that this loop forms between OmpR bound at the site between -384 to -351 and OmpR bound at one of the sites located immediately upstream of the ompF promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8113170" target="_blank">8113170</a>]<br></li>
<li>One interpretation of this result is that the OmpR bound at the site located between - 56 and -41 interacts with OmpR bound at the site located between - 384 to - 351 to form a DNA loop , thereby repressing ompF transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8113170" target="_blank">8113170</a>]<br></li>
<li>Purified OmpR binds to the DNA fragment encompassing the - 102 to -58 - bp region forming two OmpR/ompF complexes that could be resolved by gel retardation [ lo ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7557323" target="_blank">7557323</a>]<br></li>
<li>The hypothesis that the -55 to -42 - bp region functions as a repression site is based on the finding that OmpR derived from an ompR mutant strain ( MH760 , ompR2 ) , that is defective in the os-molarity-induced repression of ompF , does not bind to this region [ 12 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7557323" target="_blank">7557323</a>]<br></li>
<li>Here we identified that five regulatory proteins ( CpxR , OmpR , RstA , IHF and H-NS ) bind within a narrow region of about 229 bp along the csgD promoter ( see Fig. 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20466769" target="_blank">20466769</a>]<br></li>
<li>As shown in Fig. 2A , OmpR protects a region within the tppB regulatory region that is approximately 30 bp in length .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15175316" target="_blank">15175316</a>]<br></li>
<li>Examination of the region protected by OmpR revealed that it contains a T-N-A-C-A sequence at the beginning of each half-site ( Fig. 2C ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15175316" target="_blank">15175316</a>]<br></li>
<li>The location of the region protected by OmpR relative to the DNA sequence is indicated in Fig. 3C .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8113170" target="_blank">8113170</a>]<br></li>
<li>These results suggest that OmpR binds to the -71 to -64 - bp region and that this sequence plays a role in the regulation of ompF in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7557323" target="_blank">7557323</a>]<br></li>
<li>Discussion In this study we show that the -71 to -62 - bp region contains a nucleotide sequence to which OmpR binds .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7557323" target="_blank">7557323</a>]<br></li>
<li>Both OmpR and IHF are able to bind to the IHF-2 site simultaneously .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20466769" target="_blank">20466769</a>]<br></li>
<li>A new EcoRI site was constructed by oligonucleotide-directedsite-specificmutagenesis ( 19 ) of the sequence encoding the amino-terminal portion of OmpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3023382" target="_blank">3023382</a>]<br></li>
<li>a , Termination site of OmpR ; b , initiation site of EnvZ according to the previously published sequence ( 35 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2997120" target="_blank">2997120</a>]<br></li>
<li>Therefore , if sequence Iis a recognition site for OmpR , three OmpRmolecules maybind cooperatively with the sametopology on the DNAhelix .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3023382" target="_blank">3023382</a>]<br></li>
<li>We did not ﬁnd consensus binding sequences for 38 other forms of RNA polymerase ( except Es , whose promoter recognition sequences are similar to those of Es70 [ 32 ] ) , nor did we detect consensus binding sequences for OmpR , a transcriptional activator that functions at promoters for other OMPs ( 16 ) , or either direct or inverted-repeats which are often recognized by transcriptional activators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7836315" target="_blank">7836315</a>]<br></li>
<li>Two consensus sequences were found in these protected sites and are likelyto play an important role in OmpR recognition of these promoter regions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3023382" target="_blank">3023382</a>]<br></li>
<li>These binding sites must be close to the -35 consensus regions of the omp promoters for proper regulation ( Maeda and Mizuno , 1990 ) suggesting that OmpR controls transcription by influencing binding of RNA polymerase to the promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8098993" target="_blank">8098993</a>]<br></li>
<li>The helix-loop-he-lix-containing RRs of the OmpR family , OmpR and PhoB , bind in a direct or head-to-tail arrangement to recognition sites upstream or adjacent to the RNAP-binding region , where they interact with - CTD or - CTD , respectively ( 16 , 17 , 32 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11976297" target="_blank">11976297</a>]<br></li>
<li>The high-osmolarity form of OmpR also binds this region and , in addition , binds to the sites at -40 to -60 and at -350 to -380 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1648075" target="_blank">1648075</a>]<br></li>
<li>RESULTS OmpR binds to a unique site at the csgD promoter , whereas CpxR binds cooperatively to multiple sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15743952" target="_blank">15743952</a>]<br></li>
<li>The OmpR family member PhoB binds to direct repeat target sites in the promoter ( 32 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11976297" target="_blank">11976297</a>]<br></li>
<li>H pall CTTRTCTCCRCCCRGGCCTGGTCGGCRGGCTTTCRGTTRRACGRRTTTTCTTCCTOTGGCCTGGGC +1 +45 +90 +100 +127 H I -80 -165 -4 -40 -130 t f I I I Transcription Met Translation [ I : OmpR binding site -- 1 and + 16 with respect to the f a d L transcriptional start site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8413182" target="_blank">8413182</a>]<br></li>
<li>The 41 codons of ecnA are preceded by a ribosome binding site ( rbs ) and then followed by the RpoS promoter element of ecnB ( ÿ35 , ÿ10 ) , an AseI restriction site , the ecnB transcriptional start site ( 1 ) , and a tandem repeat ( >> > ) characteristic of a putative OmpR binding site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9677290" target="_blank">9677290</a>]<br></li>
<li>The transcriptional start site for EcEcnB is marked ( 1 ) as is the tandem repeat characteristic of an OmpR binding site ( Pratt & Silhavy , 1995 ; Harlocker et al. , 1995 ) , which is present only in EcEcnB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9677290" target="_blank">9677290</a>]<br></li>
<li>C. Sequence comparison between the putative OmpR binding sites in omrA and omrB promoters and the critical OmpR binding sites present in ompC ( ompC1 ) and ompF ( ompF1 ) promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16359331" target="_blank">16359331</a>]<br></li>
<li>In particular , the GxxxC motif is the only conserved sequence among the seven OmpR-binding sites in ompC or ompF promoters and the two sets of AC base pairs were shown to be important for OmpR binding in vivo ( Pratt and Silhavy , 1995b ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16359331" target="_blank">16359331</a>]<br></li>
<li>OmpR is a DNA-binding protein consisting of 239 amino acid residues that activates transcription of ompF and ompC binding to sequences approximately 90 base pairs upstream of the transcription initiation site of these genes ( 5 , 9 , 11 , 13 , 18 , 23 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2656631" target="_blank">2656631</a>]<br></li>
<li>OmpR is a DNA-binding protein consisting of 239 amino acid residues that activates transcription of ompF and ompC by binding to a consensus sequence 90 base pairs upstream from the transcription initiation site of these genes ( 7 , 14 , 20 , 21 , 26 , 27 , 35 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2160945" target="_blank">2160945</a>]<br></li>
<li>The phosphorylated form of OmpR binds to regulatory sites upstream of ompF and ompC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11601842" target="_blank">11601842</a>]<br></li>
<li>The phosphorylation of OmpR at Asp-55 enhances its ability to bind to the regulatory sequences upstream of the ompC and ompF promoters ( 16 , 17 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11158569" target="_blank">11158569</a>]<br></li>
<li>EnvZ , an inner membrane protein that has both a kinase and a phosphatase activity , senses the environment and transmits the signal from the medium to OmpR , a positive regulator that binds specific DNA sequences upstream from the ompF and ompC promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9192630" target="_blank">9192630</a>]<br></li>
<li>OmpR activates transcription of ompF and ompC by binding to specific sequences upstream of the promoters in these operons ( 31 , 41 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1885551" target="_blank">1885551</a>]<br></li>
<li>OmpR is a DNA-binding protein that controls transcription of ompF and ompC by binding to specific sequences upstream from the promoters in these operons ( 38 , 40 , 50 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1310090" target="_blank">1310090</a>]<br></li>
<li>Second , sequences required for OmpR stimulation are located upstream of -54 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1906064" target="_blank">1906064</a>]<br></li>
<li>This latter observation is consistent with the location of OmpR binding sites on other OmpR-regulated promoters , where the OmpR binding sequences have been located upstream and relatively distant from the -35 region of the promoter ( 30 , 32 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1906064" target="_blank">1906064</a>]<br></li>
<li>Other unknowns are how OmpR activates micF transcription while binding at far upstream sequences , i.e. in the ÿ156 to ÿ216 region ( Figure 4 ( a ) ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11601842" target="_blank">11601842</a>]<br></li>
<li>Characterization of OmpR binding sequences in the upstream region of the ompF promoter essential for transcriptional activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8098993" target="_blank">8098993</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1648075" target="_blank">1648075</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1943993" target="_blank">1943993</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10913075" target="_blank">10913075</a>]<br></li>
<li>On the other hand , extensive analyses of the promoter regions of ompC and ompF revealed that , in addition to the canonical -- 35 and -- 10 promoter regions , a 60 to 70 base pair ( bp ) sequence residing upstream of the -- 35 region is required for functioning of these promoters , and that these regions are most probably the binding site for OmpR protein ( 10-12 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3294816" target="_blank">3294816</a>]<br></li>
<li>OmpR is a DNA-binding protein that recognizes sequences in the promoter region of both ompF and ompC ( Jo et al. , 1986 ; Mizuno et al. , 1988 ; Norioka et aZ .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2557454" target="_blank">2557454</a>]<br></li>
<li>In contrast the OmpR binding sites on ompC and ompF promoters , and the NtrC binding sites on gZnA promoter are located more than 50 bp upstream from the - 10 region ( Reitzer & Magasanik , 1985 ; Norioka et al. , 1986 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3054125" target="_blank">3054125</a>]<br></li>
<li>Interestingly , the DNA upstream of the malT promoter whose deletion results in an increase in malT expression contains sequences that are similar to sequences identiﬁed as binding sites for phosphorylated OmpR ( 133 , 155 ) ( Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9529892" target="_blank">9529892</a>]<br></li>
<li>These sequences encompassed putative integration host factor ( IHF ) binding sites and are located just downstream of the repressor region we identified in nmpC and downstream of the negative binding site of OmpR in ompF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8057841" target="_blank">8057841</a>]<br></li>
<li>The sequencing of the ﬂhDC upstream region revealed the presence of two integration host factor binding sequences ( 50 ) , each with one mismatch , between the two OmpR binding sites ( Fig. 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7642497" target="_blank">7642497</a>]<br></li>
<li>of both bands ( lanes 2-7 ) , whereas a change at -95 from T Although we can not eliminate the possibility that theloss of to A in the Fa box substantially increased the appearance of nucleotides further upstream to the OmpR binding site could both bands a and b ( lanes 8-13 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2553720" target="_blank">2553720</a>]<br></li>
<li>While most SELEX experiments are based on an oligonucleo-tide pool containing a large randomized region ¯ anked by constant but non-binding sequences , the OmpR experiment used random sequence anchored by a partial binding site ( Harlocker et al. , 1995 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9813115" target="_blank">9813115</a>]<br></li>
<li>The positions of the binding sites for the catabolite gene activator protein ( CAP ) , the osmoregulator OmpR and the H-NS protein ( black lines below sequence ) are taken from Shin and Park ( 1995 ) and Soutourina et al. ( 1999 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12123461" target="_blank">12123461</a>]<br></li>
<li>These include a CAP ( ` catabolite gene activator protein ' ) binding site centred around -- 71.5 ( class I site ) , two OmpR ( osmoregulation ) binding sites at positions -- 100 to -- 163 and +1 to +66 , respectively , and multiple binding sites for the H-NS protein from position -- 178 up to the coding region ( Shin and Park , 1995 ; Soutourina et al. , 1999 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12123461" target="_blank">12123461</a>]<br></li>
<li>OmpR was shown to bind to a consensus sequence located 49.5 bp upstream of the transcriptional start site of csgD , and this subsequently activates the csgDEFG expression that , in turn , increases the expression of csgBA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15743952" target="_blank">15743952</a>]<br></li>
<li>Another OmpR binding site located 360-In Escherichia coli , two major outer membrane porin pro- 380 bp upstream of the transcriptional start site has recently teins , OmpFand OmpC , are differentially regulatedinre - been identified ( 23 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8175665" target="_blank">8175665</a>]<br></li>
<li>high affinity site then OmpR molecules bound to this region Chem .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8175665" target="_blank">8175665</a>]<br></li>
<li>It is interesting to note that the high af ® nity OmpR binding site at the ompC locus of Salmonella typhi is located in the C2 pos ition , i.e. closer to the ÿ35 region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11812125" target="_blank">11812125</a>]<br></li>
<li>At the ompC promoter , the motif maps upstream of the core promoter ( centered at position 64 relative to the transcription start site of the main P1 promoter ) and overlaps the binding sites of the activator OmpR ( see Fig .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26078445" target="_blank">26078445</a>]<br></li>
<li>In the case of the ompC and ornpFpromoters , the OmpR binding sites are located within a region between nucleotides - 4 0 and - 100 with respect to the transcriptional start site ( Maeda et al. 1988 ; Rampersaud et al. 1989 ) , and OmpR acts as a transcriptional activator ( for review see Collado-Vides et al. 1991 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8413182" target="_blank">8413182</a>]<br></li>
<li>In addition to the three OmpR binding sites located in the ompC-micF intergenic region and required for transcriptional activation of micF by OmpR , they identiﬁed a region between 2148 and 2131 bp relative to the start site for micF transcription that was responsible for OmpR-independent repression of micF expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8002608" target="_blank">8002608</a>]<br></li>
<li>For transcription activation , the OmpR protein binds to the OmpR site generally located upstream from the promoter 335 , and interacts directly with a subunit of RNA polymerase [ 19,20 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10802181" target="_blank">10802181</a>]<br></li>
<li>OmpR binds within the IHF-C site ( 5 ) , and the cyclic AMP-binding protein ( cyclic AMP-CRP ) binds at a site located within the IHF-B binding region and activates at least one additional promoter ( 27 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1885551" target="_blank">1885551</a>]<br></li>
<li>From these experiments we conclude that the OmpR protein interacts directly with and activates transcription from the csgD promoter by binding to a 20-bp region centered at 49.5 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11717281" target="_blank">11717281</a>]<br></li>
<li>Del-etion of the OmpR-binding sequences results in loss of osmoregulation of micF .8 Three binding sites for OmpR have been de ® ned73 ,74 ( Figure 4 ( a ) ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11601842" target="_blank">11601842</a>]<br></li>
<li>In addition , an integration host factor ( IHF ) binding site has been identi ® ed at the ÿ66 position ( Figure 4 ( a ) ) .76 There is a high degree of phylogenetic conservation of nucleotide sequences in transcription factor binding sites .6 An understanding of the interaction of OmpR with the micF regulatory DNA sequences offers a particular challenge .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11601842" target="_blank">11601842</a>]<br></li>
<li>Gel shift and footprinting analyses have demonstrated the binding of OmpR to a variety of DNA sequences .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9016718" target="_blank">9016718</a>]<br></li>
<li>Two adjacent recognition sites are necessary for DNA binding and are recognized by two OmpR molecules [ 13,48 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9016718" target="_blank">9016718</a>]<br></li>
<li>We suggest that CpxR , via its speciﬁc binding sites arranged over an extended region of DNA , functions as a modulator of OmpR activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15743952" target="_blank">15743952</a>]<br></li>
<li>A comparable overall conclusion of sequential loading at half-sites of F1a and F1b of F1 promoter by phosphorylated form of full-length OmpR was reached by Inouye and colleagues using EMSA with F1b site binding more tightly than F1a [ 27 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22291906" target="_blank">22291906</a>]<br></li>
<li>Moreover , it has been reported that an OmpR binding site is comprised of two contiguous decameric repeats , more or less conserved , and that one OmpR molecule binds one DNA repeat ( 8 , 9 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10648521" target="_blank">10648521</a>]<br></li>
<li>For the promoter with a deletion of the putative OmpR binding site , a 119-bp fragment was ampliﬁed using two primers , D1 and D3 ( 5 - GGATTACATCTGATTTCAATCTAGCCATTACAAATCTTAA ATCAAGTGTTCTCGTTATATTAAAATG-3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11717281" target="_blank">11717281</a>]<br></li>
<li>These data have led to a model for ArcA and other OmpR / FIG 1 ( A ) Sequence logo for the minimal ArcA binding site consisting of two 10-bp direct repeat elements ( 5 = - ATGTTAAAAA-1-ATGTTAAAAA-3 =) ( 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25161193" target="_blank">25161193</a>]<br></li>
<li>These studies indicated that each individual OmpR binding site spans '18 bp and contains two half-sites oriented as asymmetric direct repeats .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9096305" target="_blank">9096305</a>]<br></li>
<li>Another regulatory site , termed that OmpR binding at the -360 to -380 region requires the Cd box , is between -42 and -52 and is a 10-bp sequence previous binding at downstream sequences , wihsic inh - unrelated to the -70 to -100 region ( see Figs. 8 and 9 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8175665" target="_blank">8175665</a>]<br></li>
<li>This result suggested that two distinct OmpR binding sequences existed within the - 102 to -58 - bp sequence of ompF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7557323" target="_blank">7557323</a>]<br></li>
<li>This suggested that a second OmpR binding site exists within the -71 to -64 - bp region of ompF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7557323" target="_blank">7557323</a>]<br></li>
<li>We show that the - 102 to - 76-bp sequence of ompF has a high affinity binding site for OmpR and produced a single OmpR/ompF complex ( complex b ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7557323" target="_blank">7557323</a>]<br></li>
<li>To further characterize the binding of OmpR to the downstream site , we created a 27-bp fragment encompassing the -84 to -58 - bp sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7557323" target="_blank">7557323</a>]<br></li>
<li>Considering the pga regulatory region as a whole , OmpR binding to this sequence could be hindering RNA polymerase or NhaR interactions with the pgaA promoter , as the OmpR-protected region extends from 248 to 210 , where a downstream NhaR-binding box and the 235 hexamer are also located ( Fig .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25667010" target="_blank">25667010</a>]<br></li>
<li>Fig. 4A shows the binding of OmpR to the F1-F2-F3 fragment containing the wild-type sequence in the region from 299 to 235 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9096305" target="_blank">9096305</a>]<br></li>
<li>Consensus indicates the OmpR binding sequence [ 18 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10802181" target="_blank">10802181</a>]<br></li>
<li>The consensus sequence of OmpR binding is TTTACTTTTTG-AACAT-TT [ 18 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10802181" target="_blank">10802181</a>]<br></li>
<li>To determine if this sequence functions as a binding site for the OmpR protein , we assayed binding of the FPLC-puriﬁed OmpR and OmpR234 proteins to DNA fragments containing the transcriptional regulatory region of csgD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11717281" target="_blank">11717281</a>]<br></li>
<li>S1 ) demonstrate the binding of OmpR to the pgaA regulatory region and allow us to identify the sequence protected by it .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25667010" target="_blank">25667010</a>]<br></li>
<li>The putative OmpR binding site is an imperfect direct repeat sequence centered at 49.5 relative to the transcriptional start site of csgD ( Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11717281" target="_blank">11717281</a>]<br></li>
<li>The D3 primer sequence is identical to the sequence of D2 for its ﬁrst 25 bases and then contains a 20-bp deletion corresponding to the putative OmpR binding site ( Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11717281" target="_blank">11717281</a>]<br></li>
<li>The sequence of the ﬁrst OmpR binding site ( S1 ) was highly conserved in both species , as described in Fig. 1B .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15743952" target="_blank">15743952</a>]<br></li>
<li>The presence of a possible OmpR binding site in the agfD promoter region of S. enterica serovar Typhimurium that is almost identical to the csgD promoter sequence of E. coli has already been suggested by Römling et al. ( 44 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11717281" target="_blank">11717281</a>]<br></li>
<li>The sequence of the putative OmpR-binding site identiﬁed in the omrB promoter seems closer to an optimum site , deﬁned by alignment of known OmpR-binding sites and in vitro binding by OmpR ( Harlocker et al. , 1995 ; Huang and Igo , 1996 ) , than is the site in the omrA promoter ( Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16359331" target="_blank">16359331</a>]<br></li>
<li>T h e nature of the OmpR-binding site was until recently poorly understood , and the bases specifically recognized b q OmpR were not accurately defined .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9016718" target="_blank">9016718</a>]<br></li>
<li>Evidence for multiple OmpR-binding sites in the upstream activation sequence of the ompC promoter in Escherichia coli : a single OmpR - binding site is capable of activating the promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11812125" target="_blank">11812125</a>]<br></li>
<li>( B ) Comparison of E. coli and S. enterica serotype Typhimurium OmpR binding sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15743952" target="_blank">15743952</a>]<br></li>
<li>Nucleotides in grey indicate the putative OmpR binding sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16359331" target="_blank">16359331</a>]<br></li>
<li>At present it is not clear whether binding of OmpR to all four sites is necessary for repression , or whether binding to each site affects transcription differently .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8413182" target="_blank">8413182</a>]<br></li>
<li>Since the OmpR binding site overlaps with the RNA polymerase binding site , the OmpR binding might interfere with the binding of RNA polymerase to the bolA1p .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10802181" target="_blank">10802181</a>]<br></li>
<li>The putative OmpR binding site is boxed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11717281" target="_blank">11717281</a>]<br></li>
<li>The OmpR binding site is boxed , and six putative CpxR binding sites are underlined .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15743952" target="_blank">15743952</a>]<br></li>
<li>OmpR and CpxR recognition sites overlap immediately upstream of the RNA polymerase binding site ( 44 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15743952" target="_blank">15743952</a>]<br></li>
<li>The LrhA binding site partially overlaps with the upstream OmpR site , whereas the hypersensitive sites are located within the downstream OmpR site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12123461" target="_blank">12123461</a>]<br></li>
<li>This sequence , 5 - GTTACATTTA/GTTACATGTT -3 , closely resembles the consensus for OmpR binding sites proposed by Harlocker et al. ( 21 ) , and its location would be consistent with possible OmpR-RNA polymerase interactions at the csgD promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11717281" target="_blank">11717281</a>]<br></li>
<li>This sequence is very similar to the consensus for OmpR binding sites proposed by several groups ( 21 , 22 , 37 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11717281" target="_blank">11717281</a>]<br></li>
<li>A single unique protection sequence was identified for OmpR and RstA , while two sites were identified with IHF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20466769" target="_blank">20466769</a>]<br></li>
<li>Based on both in vivo and in vitro analyses , we identiﬁed MlrA as a positive factor of the csgD promoter by directly binding to its upstream region ( 113 to 146 ) with a palindromic sequence of AAAATTGTA CA ( 12N ) TGTACAATTTT between the binding sites of two activators , IHF and OmpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20874755" target="_blank">20874755</a>]<br></li>
<li>The binding sites of OmpR , RstA , CpxR , IHF and H-NS were identified by DNase I footprinting assays ( for gel patterns see Fig. 5 ) and are shown along the csgD promoter sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20466769" target="_blank">20466769</a>]<br></li>
<li>The binding sites of IHF , OmpR , RstA and CpxR are indicated along the csgD promoter sequence while the H-NS-binding sequence is shown by a dotted line between the duplex sequences .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20466769" target="_blank">20466769</a>]<br></li>
<li>The intergenic space between the ompN and micC genes is 229 bp , which is slightly shorter than the 256-bp spacer region between ompC and micF , and we could not ﬁnd any sequence elements in the micC and ompN promoter region that were similar to the OmpR binding sites in the micF and ompC promoter region ( 27 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15466019" target="_blank">15466019</a>]<br></li>
<li>The nucleotide sequence of the whole region including the four OmpR binding sites is shown in Fig. 9 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8413182" target="_blank">8413182</a>]<br></li>
<li>Nucleotide sequence of OmpR binding sites in the fadL promoter region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8413182" target="_blank">8413182</a>]<br></li>
<li>The upper portion of the figure shows the nucleotide sequence of the 426 bp HpaII fragment that contains the fadL promoter region and the four OmpR binding sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8413182" target="_blank">8413182</a>]<br></li>
				</ul></div><br>
			<label class="collapse_TU" for="_6"><font size="5px">Transcription unit regulated by OmpR</font></label>
				<input id="_6" type="checkbox">
				<div class="TU"><br><ul><li>Genes most highly repressed upon IbsC , ShoB , LdrD and TisB overexpression.a Average fold repression dusB/yhdG ﬁs yeeF apt efeU ( ycdN ) CRP , IHF , Fis tRNA dihydrouridine synthase B Transcriptional activator Putative transporter Adenine phosphoribosyl transferase Inactive ferrous iron permease 6.5 7.7 0.9 0.9 0.8 0.9 1.2 1.1 1.1 1.2 1.2 2.1 4.3 3.8 4.7 2.3 1.2 17.8 6.1 21.2 CpxR Inactive ferrous iron transporter Inactive ferrous iron transporter Porin Unknown efeO ( ycdO ) efeB ( ycdB ) ompF yagU 14.3 2.6 6.1 26.7 1.0 1.8 1.4 1.7 1.2 1.8 0.9 2.5 1.2 1.2 0.5 1.2 OmpR , EnvYd a. Genes and operons which are repressed 5-fold in three independent experiments are listed with the average fold repression for the three experiments .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18710431" target="_blank">18710431</a>]<br></li>
<li>Under such conditions , the activation of csgDEFG probably occurs via OmpR , which is dominant upon repression by RcsAB , while RcsB and RcsA could repress csgAB through a dominant negative effect on CsgD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16000739" target="_blank">16000739</a>]<br></li>
<li>Flores-Valdez , M.A. , Puente , J.L. , and Calva , E. ( 2003 ) Negative osmoregulation of the Salmonella ompS1 porin gene independently of OmpR in an hns background .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22295907" target="_blank">22295907</a>]<br></li>
<li>Dual regulation by phosphoOmpR of ssrA/B gene expression in Salmonella pathogenicity island 2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15995204" target="_blank">15995204</a>]<br></li>
<li>The amino acid sequences used in the comparison were taken from the European Molecular Biology Laboratory ( EMBL ) data base and included those of factors acting positively on initiation of transcription such as CRP , Fnr , and OmpR of E. coli ; the spoOA , spoOF , spoOB , and spoOH gene products from B. subtilis ; and the sigma factors encoded by rpoD and rpoH of E. coli , rpoN of Klebsiella pneumoniae , and sigE of B. subtilis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2537825" target="_blank">2537825</a>]<br></li>
<li>The complex transcriptional regulation of the csgDEFG operon will also have to be considered , which occurs in stationary phase and involves several global regulators such as RpoS , HNS , IHF and OmpR ( Barnhart and Chapman , 2006 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20407422" target="_blank">20407422</a>]<br></li>
				<label class="TU_collapse_similarity" for="tu_similarity_7">
motility and flhDC transcription are inhibited under conditions of high acetyl phosphate levels and high osmolarity by the phosphorylated form of OmpR ( PruÈû and Wolfe , 1994 ; Shin and Park , 1995 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11298291" target="_blank">11298291</a>]<br></label>
<input id="tu_similarity_7", type="checkbox">
<div class="TU_similar">
<li>In contrast , motility and ﬂhDC expression are inhibited under conditions of high acetyl phosphate levels and high osmolarity by the phosphorylated form of OmpR ( Prüß and Wolfe , 1994 ; Shin and Park , 1995 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12123461" target="_blank">12123461</a>]<br></li></div>
<li>Transcriptional level CpxR-P not only modulates the action of different regulators , such as BaeR , Lrp and OmpR , by binding to the same regulatory sequences ( see above ) , but it also represses the gene transcription of other signaling systems , such as σE and Aer .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16487683" target="_blank">16487683</a>]<br></li>
<li>DNA curvature induced by MlrA should inﬂuence not only the activity of neighbouring DNA such as binding afﬁnity to positive factors , IHF , OmpR and RstA , and/or to negative fractors , H-NS and CpxR , but also the mode of the molecular interplay between csgD promoter-bound transcription factors .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20874755" target="_blank">20874755</a>]<br></li>
<li>Therefore , it seems likely that OmpR , in its phosphorylated form , binds to DNA sequences upstream of position − 60 of the omrA and omrB promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16359331" target="_blank">16359331</a>]<br></li>
<li>Since OmpR activates the transcription of omrA and omrB , these sRNAs should limit their own synthesis by directly downregulating their activator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18953042" target="_blank">18953042</a>]<br></li>
<li>Expression of these operons is positively regulated by OmpR ( via the activation of CsgD ) ( Romling et al. , 1998 ) , and negatively regulated by CpxR ( Dorel et al. , 1999 ; Otto & Silhavy , 2002 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16000739" target="_blank">16000739</a>]<br></li>
<li>RESULTS ArcA downregulates the transcription of fatty acid transport genes under anaerobic conditions The expression of fadL , which encodes one of the outer-membrane-bound proteins involved in the uptake of fatty-acids , is regulated by at least three transcription factors : Crp , OmpR and FadR ( Berg et al. , 2004 ; Busch & Saier , 2002 ; Clark & Cronan , 1996 ; DiRusso et al. , 1992 ; Cronan & Subrahmanyam , 1998 ; Higashitani et al. , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16849788" target="_blank">16849788</a>]<br></li>
<li>It is also known that LCFA transport ( fadL ) is repressed by OmpR in response to the high external osmotic pressure ( Clark & Cronan , 1996 ; Higashitani et al. , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16849788" target="_blank">16849788</a>]<br></li>
<li>The profiles were essentially the same as those of the original ompR mutants , suggesting that the overproduced mutant OmpR proteins dominate over the OmpR1 protein ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3010044" target="_blank">3010044</a>]<br></li>
<li>The resulting plasmid , pET20-OmpR , contains ompR under the control of the T7 promoter and with its own translational regulatory signals , and it expresses an OmpR protein with a His6 motif at its C terminus .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15743952" target="_blank">15743952</a>]<br></li>
<li>Genes encoding OM proteins and surface appendages are potential negative targets of OmrA and OmrB Identifying EnvZ and OmpR as the regulators of omrA and omrB did not immediately lead us to the targets of these sRNAs , because no phenotypes due to post-tran-scriptional regulation by the EnvZ-OmpR system have been reported .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16359331" target="_blank">16359331</a>]<br></li>
<li>The signaling mechanisms that prevail in OmpR , MarA , and SoxS action are known .9,10,14 Activation of micF by Rob in response to cationic peptide antibiotics is not as well formulated .32 Transcriptional activators of micF Transcriptional regulators are known to activate micF expression in response to environmental factors ( Figure 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11601842" target="_blank">11601842</a>]<br></li>
<li>Expression and purification of transcription factors For construction of plasmids ( pMlrA and pOmpR ) for expression of His-tagged MlrA and OmpR , DNA fragments corresponding to the coding sequences of their respective transcription factors were ampliﬁed by PCR using E. coli W3110 genome DNA as a template and pairs of primers , which were designed so as to hybridize upstream or downstream of each coding sequence ( Table S2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20874755" target="_blank">20874755</a>]<br></li>
<li>In the model of Hall and Silhavy ( 11 ) , OmpR functions as a positive regulator of OmpF and OmpC transcription , and the envZ gene product is an envelope protein which senses the environment and influences porin synthesis by controlling the multimerization of the OmpR protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6317653" target="_blank">6317653</a>]<br></li>
<li>The bolA1p promoter was transcribed by EcD as well as EcS ( Fig. 4A , B ) , but was repressed in the presence of phospho-OmpR while transcription of the tac promoter was una ¡ ected by the OmpR protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10802181" target="_blank">10802181</a>]<br></li>
<li>In this paper , we have shown that bolA1p transcription can be controlled negatively by phospho-OmpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10802181" target="_blank">10802181</a>]<br></li>
<li>In contrast , the additionof OmpR blocked the cellfree transcription of the ompC gene from its most upstream promoter , P3 ( Fig. 8B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3023382" target="_blank">3023382</a>]<br></li>
<li>The ompF gene encoding an outer membrane porin protein is transcribed by Es70 and regulated by OmpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10368152" target="_blank">10368152</a>]<br></li>
<li>The fadL gene encoding the outer membrane long-chain fatty acid transporter is also repressed by OmpR [ 91 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10396600" target="_blank">10396600</a>]<br></li>
<li>From these results , we propose that oleic acid uptake in E. coli is osmotically regulated , at least when thefadR control is derepressed , through transcription of fadL , which is repressed by the phosphorylated form of OmpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8413182" target="_blank">8413182</a>]<br></li>
<li>Furthermore , in vitro transcription of the fadL gene was strongly repressed by the addition of OmpR and EnvZ proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8413182" target="_blank">8413182</a>]<br></li>
<li>In this report we present data indicating that ( 1 ) uptake of long-chain-fatty-acids in E. coli and gene expression of fadL are both regulated by environmental osmolarity , ( 2 ) transcription of thefadL gene in vitro is repressed by phosphorylated OmpR , and ( 3 ) phosphorylated OmpR specifically binds to the promoter region of fadL Materials and methods Bacterial strains , media , and growth conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8413182" target="_blank">8413182</a>]<br></li>
<li>Tpp is induced by anaerobic growth and is positively regulated by the EnvZ/OmpR legulatory system ( Gibson et aL , 1987 ) , The dipeptide permease ( Dpp ) , like the Opp , is a binding-protein-dependent system .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7536291" target="_blank">7536291</a>]<br></li>
<li>Expression of the tppB gene , which encodes the tripeptide permease , is only induced during anaerobic growth ( 11 , 12 ) and depends upon the function of the positive regulatory proteins OmpR and EnvZ ( 1 ) ; the physiological role of this transport system is not yet understood .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3301822" target="_blank">3301822</a>]<br></li>
<li>However , deletion of the acrD gene , which is the gene most highly controlled by OmpR among these exporter genes , did not affect the ompR-mediated drug resistance .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12618449" target="_blank">12618449</a>]<br></li>
<li>They proposed that the high motility of MG1655 is due to insertion element insB-5 , which prevents binding of the negative regulator OmpR , and thus increase the expression of ﬂhDC operon for ﬂagellar synthesis ( Supplementary Figure S1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19969540" target="_blank">19969540</a>]<br></li>
<li>16 , 1 ¬ ompB asmoiaiity ) OmpR IHF IHF '60 -35 -10 • 170 QmpR CmpROmpR IHF IHF -360 -60 -170 The architectural roie of IHF in gene expression is not only limited to o - ̂ ** promoters but is probably also involved in the activation of the c ' ^ ^ - dependent promoter of the narGHJI operon ( Schroder et at .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7651128" target="_blank">7651128</a>]<br></li>
<li>Another metabolic intermediate , acetylphosphate , affected expression of the ﬂhDC operon in E. coli , and this effect was dependent on OmpR , another pleio-tropic regulatory protein ( 117 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11104815" target="_blank">11104815</a>]<br></li>
<li>In this report we show that curli biosynthesis is subject to a complex regulatory network : the OmpR protein positively regulates curli expression by binding the csgDEFG promoter region at position 49.5 relative to the transcriptional start site and by activating its transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11717281" target="_blank">11717281</a>]<br></li>
<li>The osmoregulator protein , OmpR , when phosphorylated , negatively regulatesflagellar synth-esis by binding to the promoter region of the flhD operon ( Shin and Park , 1995 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8866483" target="_blank">8866483</a>]<br></li>
<li>The cyclic AMP ( cAMP ) receptor protein-cAMP complex and the OmpR protein are known to affect the expression of the ﬂhDC operon by binding to its promoter region ( 21 , 23 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10913108" target="_blank">10913108</a>]<br></li>
<li>Mutations eliminating the stationary-phase sigma factor RpoS and the response regulator OmpR did not affect pgaA - lacZ expression ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16997959" target="_blank">16997959</a>]<br></li>
<li>However , details of the mechanisms of ﬁne control of sra expression in the exponential phase as well as in the stationary phase by sigma S , OmpR , FIS , IHF , cAMP , and/or ppGpp remain to be investigated further .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11292794" target="_blank">11292794</a>]<br></li>
<li>Although it is not clari cents ed how much OmpR is phosphorylated by stress , bolA1p expression may be induced with a decreased phospho-OmpR concentration in the cells under the stress .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10802181" target="_blank">10802181</a>]<br></li>
<li>Under the same conditions , the phenotypic similarities observed between the pta and ompR null mutants indicate that OmpR and phospho-OmpR are not required for the expression of ﬂagella .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7642497" target="_blank">7642497</a>]<br></li>
<li>We therefore propose that the regulatory role of phospho-OmpR in ﬂagellar synthesis is negative , whereas it is both positive and negative in porin expression ( 39 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7642497" target="_blank">7642497</a>]<br></li>
<li>Regulation of porin gene expression by osmolarity via OmpR is a well-studied phenomenon ( 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15743943" target="_blank">15743943</a>]<br></li>
<li>Since the CpxA and CpxR proteins control the expression of degP at the transcriptional level , and since CpxR is homologous to the OmpR transcription factor , we reasoned that the Cpx signal transduction system would likely control its other regulatory targets at the transcriptional level as well .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9473036" target="_blank">9473036</a>]<br></li>
<li>Previous studies have shown that micF expression is increased when cells are shifted from a glucose minimal medium to LB and that increased micF transcription does not require a functional OmpR protein ( 10 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8002608" target="_blank">8002608</a>]<br></li>
<li>OmpR was shown to activate csgD expression , resulting in curli production .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15743952" target="_blank">15743952</a>]<br></li>
<li>Besides requiring sS and MlrA , csgD expression is also modulated by IHF and the two-component response regulators OmpR and CpxR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17010156" target="_blank">17010156</a>]<br></li>
<li>nanC expression is also activated by the regulators cyclic AMP-catabolite activator protein , OmpR , and CpxR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15743943" target="_blank">15743943</a>]<br></li>
<li>Two other regulators , OmpR and CpxR , activate nanC expression , although a direct control has not been proven .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15743943" target="_blank">15743943</a>]<br></li>
<li>Despite weak csgD expression in the absence of the OmpR activator , repression was still observed in high-sucrose medium ( Fig. 6B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15743952" target="_blank">15743952</a>]<br></li>
<li>However , the above experiments can not exclude that downregulation of csgD expression by OmrA/B occurred through inhibition of the expression of the transcriptional activator OmpR ; OmpR is a known OmrA/B target ( Guillier and Gottesman , 2008 ) and promotes csgD transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20407422" target="_blank">20407422</a>]<br></li>
<li>174 , No. 3 Positive and Negative Control of ompB Transcription in Escherichia coli by Cyclic AMP and the Cyclic AMP Receptor Protein LIN HUANG , PING TSUI , t AND MARTIN FREUNDLICH * Department of Biochemistry and Cell Biology , State University of New York , Stony Brook , New York 11794-5215 Received 12 August 1991/Accepted 20 November 1991 The ompB operon encodes OmpR and EnvZ , two proteins that are necessary for the expression and osmoregulation of the OmpF and OmpC porins in Escherchia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1310090" target="_blank">1310090</a>]<br></li>
<li>Besides the outer membrane porins , other genes ( 49 ) -- those encoding the tripeptide permease TppB of Salmonella typhimurium , the outer membrane protease Opr , the PhoA and PhoE proteins of the Pho regulon , and the positive regulator MalT of the maltose operon -- are known to be regulated by EnvZ and OmpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7642497" target="_blank">7642497</a>]<br></li>
<li>Correlation with mutagenesis data Transcription of the genes encoding the OmpF and OmpC proteins is regulated by OmpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9016718" target="_blank">9016718</a>]<br></li>
<li>Received 9 December 1988/Accepted 3 March 1989 Expression in Escherichia coli of the genes that encode the major outer membrane porin proteins ( OmpF and OmpC ) is regulated by the transcription activator protein OmpR and the receptorlike protein EnvZ , which is located in the inner membrane .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2656631" target="_blank">2656631</a>]<br></li>
<li>OmpR activates transcription of ompF and ompC by binding to specific sequences upstream of the promoters in these operons ( 31 , 41 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1885551" target="_blank">1885551</a>]<br></li>
<li>OmpR is a DNA-binding protein that controls transcription of ompF and ompC by binding to specific sequences upstream from the promoters in these operons ( 38 , 40 , 50 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1310090" target="_blank">1310090</a>]<br></li>
<li>The purified OmpR wasalsoshown tobe functionally active in transcription in vitro ; OmpR activates the expression of ompF and inhibits the transcription of ompC from the most distal of its three tandem promoters , P3 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3023382" target="_blank">3023382</a>]<br></li>
<li>Phosphorylated OmpR positively regulates ompC transcription , and it negatively regulates ompF transcription ( 47 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8002608" target="_blank">8002608</a>]<br></li>
<li>The ompF gene , with its high af-ﬁnity OmpR-binding sites , is transcribed even when the phosphorylated OmpR is scarce ( i.e. , under low-osmolarity conditions ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14665678" target="_blank">14665678</a>]<br></li>
<li>Phosphorylated OmpR activates expression of the OmpF porin at low osmolarity , and at high osmolarity represses ompF transcription and activates expression of OmpC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11812125" target="_blank">11812125</a>]<br></li>
				<label class="TU_collapse_similarity" for="tu_similarity_63">
OmpR promotes the transcription of ompF in conditions of low osmolarity ; at high osmolarity , OmpR represses transcription of ompF and activates transcription of ompC [ 2,3 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9016718" target="_blank">9016718</a>]<br></label>
<input id="tu_similarity_63", type="checkbox">
<div class="TU_similar">
<li>Under conditions of low osmolarity , OmpR activates transcription of ompF , while under high osmolarity it represses transcription of ompF and activaies transcription of ompC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9016718" target="_blank">9016718</a>]<br></li></div>
<li>Without the ability to loop the DNA , OmpR can not repress ompF transcription in high osmolarity , resulting in constitutive expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1648075" target="_blank">1648075</a>]<br></li>
<li>The concomitant binding of OmpR to these regions results in the formation of a DNA loop that is responsible for repressing ompF expression under high-osmolarity conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8113170" target="_blank">8113170</a>]<br></li>
<li>These results are consistent with the proposal that osmoregulation of ompF expression is the result of negative regulation by OmpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1648075" target="_blank">1648075</a>]<br></li>
<li>How looping of the ompF promoter DNA by OmpR results in negative regulation of transcription is unclear .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1648075" target="_blank">1648075</a>]<br></li>
<li>This suggests that wild-type OmpR is capable of negatively regulating the expression of ompF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2557454" target="_blank">2557454</a>]<br></li>
<li>Then , as the level of wildtype OmpR increases , the expression of both ompF and ompC is activated ( lane 12 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9422609" target="_blank">9422609</a>]<br></li>
<li>Finally , at high levels of wild-type OmpR , ompF expression is repressed and ompC expression is activated ( lane 13 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9422609" target="_blank">9422609</a>]<br></li>
<li>In media of low osmolarity , levels of phosphorylated OmpR are low and ompF expression is activated ( 52 , 53 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8002608" target="_blank">8002608</a>]<br></li>
<li>In ompF , the two IHF binding sites flank the OmpR sites that are necessary for operon expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1736095" target="_blank">1736095</a>]<br></li>
<li>In other words , in an envZ473 background , ompF expression is always repressed by OmpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1648075" target="_blank">1648075</a>]<br></li>
<li>We propose that this form of OmpR is responsible for activating expression of ompF and does so by binding to a site ( s ) ( positive site ) in the promoter region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2557454" target="_blank">2557454</a>]<br></li>
<li>These data suggest that IHF exerts negative control of ompC expression in two ways-directly , by binding upstream from the ompC promoter region , and indirectly , by influencing the expression of OmpR and EnvZ .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2203749" target="_blank">2203749</a>]<br></li>
<li>Since ompC can be expressed in vitro in the absence of added OmpR ( 35 ) , an analysis of the role of phosphorylation of OmpR on in vitro expression of ompC has not yet been reported .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2160945" target="_blank">2160945</a>]<br></li>
<li>As illustrated in Fig. 3b , it has been reported that the expressions of ompC and ompF genes for outer membrane porin proteins are regulated by a sensory histidine kinase EnvZ and cognate response regulator OmpR , where EnvZ regulates the phosphorylation of OmpR ( Castanié - Cornet et al. 2010 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23274360" target="_blank">23274360</a>]<br></li>
<li>In many of these cases , it is not clear whether these conditions affect ompF and ompC directly or indirectly , through changes in the levels of OmpR and EnvZ ( 22 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1310090" target="_blank">1310090</a>]<br></li>
<li>Expression of ompC and ompF ( encoding the OmpC and OmpF porins ) is mainly regulated by the osmoresponsive EnvZ/OmpR system .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12366850" target="_blank">12366850</a>]<br></li>
<li>ompB also has a dual role in E. coli , since its products , OmpR and EnvZ , control the levels of the OmpF and OmpC porins through transcriptional regulation of the ompF and ompC operons ( 22 , 23 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1310090" target="_blank">1310090</a>]<br></li>
<li>OmpR and RcsB abolish temporal and spatial changes in expression of flhD in Escherichia coli biofilm .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25667010" target="_blank">25667010</a>]<br></li>
<li>Because of the genera-lity of this phenotype , we suggest that it reflects the normal regulation of the porin genes and that OmpR functions in both a positive and a negative fashion to regulate the expression of ompF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2557454" target="_blank">2557454</a>]<br></li>
<li>OmpR , a well-characterized member of the largest subfamily of RRs , functions as both an activator and repressor to regulate differentially the expression of the ompC and ompF genes that encode outer membrane porin proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10966457" target="_blank">10966457</a>]<br></li>
				<label class="TU_collapse_similarity" for="tu_similarity_85">
Delgado , J. and Inouye , M. ( 19891 Phosphorylation of OmpR by the osmosensor EnvZ modulates the expression of the ompF and ompC genes in Escherichia co/i .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8522138" target="_blank">8522138</a>]<br></label>
<input id="tu_similarity_85", type="checkbox">
<div class="TU_similar">
<li>Phosphorylation of OmpR by the osmosensor EnvZ modulates expression of the ompF and ompC genes in Escherichia co / / .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8098993" target="_blank">8098993</a>]<br></li>
<li>Phosphorylation of OmpR by the osmosensor EnvZ modulates expression of the ompF and ompC genes in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8113170" target="_blank">8113170</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2160940" target="_blank">2160940</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2203749" target="_blank">2203749</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9422609" target="_blank">9422609</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1512197" target="_blank">1512197</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26209678" target="_blank">26209678</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1885551" target="_blank">1885551</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11601842" target="_blank">11601842</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1695892" target="_blank">1695892</a>]<br></li>
<li>Phosphorylation of OmpR by the osmosensor EnvZ modulates the expression of the ompF and ompC genes in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2160945" target="_blank">2160945</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1532388" target="_blank">1532388</a>]<br></li>
<li>of Phosphorylation OmpR by the osmosensor EnvZ modulates expression of the ompF and ompC genes in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1310090" target="_blank">1310090</a>]<br></li>
<li>[ 51Forst , S. , Delgado , J. and Inouye , M. ( 1989 ) Phosphorylation of OmpR by the osmosensor EnvZ modulates expression of the ompF and ompC genes in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7557323" target="_blank">7557323</a>]<br></li>
<li>Forst , S. , Delgado , J , , and Inouye , M , ( 1989 ) Phosphorylation of OmpR by the osmosensor EnvZ modulates expression of the ompF and ompC genes in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2201868" target="_blank">2201868</a>]<br></li></div>
<li>Figure 6 also shows that a change in the relative amounts of OmpR and EnvZ in cells does not affect the regulatory profile of the ompC and ompF expression , and the polycistronic organization of the ompR and envZ genes ATK2 ( A b PAT2OO2 PAT2005 a Sucrose 0 15 0 15 0 15 Fig. 6 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3294816" target="_blank">3294816</a>]<br></li>
<li>Characterization of these deletion strains showed that the OmpR protein is necessary for transcription of the ompC and ompF genes , and the EnvZ protein is essential for normal regulation of the ompC and ompF expression , which is affected by the medium osmolarity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3294816" target="_blank">3294816</a>]<br></li>
<li>It is not yet fully understood how expression of the ompC and ompF genes is reciprocally regulated by their common regulatory factors , OmpR and EnvZ .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3294816" target="_blank">3294816</a>]<br></li>
<li>Some systems output a graded response , such as the EnvZ-OmpR system that mediates the differential expression of porin genes ompF and ompC ( 224 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10966457" target="_blank">10966457</a>]<br></li>
<li>Introduction The expression of the ompF and ompC porin genes in Escherichia coli is controlled by the response regulatory protein , OmpR , and the histidine kinase sensor protein , EnvZ [ l-4 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7557323" target="_blank">7557323</a>]<br></li>
<li>Puriﬁcation and characterization of the OmpR protein , a positive regulator involved in osmoregulatory expression of the ompF and ompC genes in Esch-erichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11717281" target="_blank">11717281</a>]<br></li>
<li>Phospho-OmpR binds to the promoters of the porin genes to regulate the expression of ompF and ompC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7568033" target="_blank">7568033</a>]<br></li>
<li>The OmpR protein acts as a positive regulator for expression of both the ompF and ompC genes by binding to the upstream region of their promoters ( Inokuchi et al. 1984 ; Dairi etal .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3010044" target="_blank">3010044</a>]<br></li>
<li>It is thus clear that the OmpR protein is the positiveregulator specific for the ompF and ompC genes ( 13-16 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2668281" target="_blank">2668281</a>]<br></li>
<li>The network of genes regulated by the CpxA/CpxR and EnvZ/OmpR systems is not entirely unique ; for example , both systems independently regulate transcription of csgD , ompC and ompF ( Batchelor et al. , 2005 ; Jubelin et al. , 2005 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19432797" target="_blank">19432797</a>]<br></li>
<li>We have shown elsewhere ( Slauch et al. , 1988 ) , and above , that OmpR under certain conditions is capable of activating ompF and ompC expression in the absence of EnvZ .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2557454" target="_blank">2557454</a>]<br></li>
<li>These mutations affect the expression of genes that are archetypal members of the EnvZ/OmpR regulon , such as ompF and ompC , and also inﬂuence the expression of genes , including lamB and malE , normally not controlled by the EnvZ/OmpR regulon ( Wandersman et al. , 1980 ; Case et al. , 1986 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19432797" target="_blank">19432797</a>]<br></li>
<li>However , in this analysis , we did not detect regulation of some genes , such as bolA and malT , that were known to be under the control of OmpR -- EnvZ .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12366850" target="_blank">12366850</a>]<br></li>
<li>Not all genes that may be affected by the activated CpxA/CpxR and EnvZ/OmpR regulons are shown .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19432797" target="_blank">19432797</a>]<br></li>
<li>The regulatory genes ompR and envZ constitute an operon like the phoB-phoR operon , and the EnvZ protein is presumed to be a membrane protein that functions as an osmo-sensor and modifies the function of the OmpR protein ( Hall & Silhavy , 1981 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3537313" target="_blank">3537313</a>]<br></li>
<li>IHF is also a transcriptional regulator of the ompB operon whose protein products , EnvZ and OmpR , are required for expression and osmoregulation of ompF and ompC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1447969" target="_blank">1447969</a>]<br></li>
<li>The products of the ompB operon , OmpR and EnvZ , are necessary for the osmoregulation and expression of ompF and ompC ( 16 , 17 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2203749" target="_blank">2203749</a>]<br></li>
<li>The expression and osmoregulation of these genes is thought to bebediated primarily by OmpR and EnvZ , the protein qioducts of the ompB operon ( Hall and Silhavy 1981a , b ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1736095" target="_blank">1736095</a>]<br></li>
				<label class="TU_collapse_similarity" for="tu_similarity_110">
Expression and osmoregulation of these genes involves primarily transcriptional control mediated by OmpR and EnvZ , the protein products of the ompB operon ( 22 , 23 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1885551" target="_blank">1885551</a>]<br></label>
<input id="tu_similarity_110", type="checkbox">
<div class="TU_similar">
<li>The expression and osmoregulation of these genes are primarily under transcriptional control mediated by OmpR and EnvZ , the protein products of the ompB operon ( 22 , 23 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1310090" target="_blank">1310090</a>]<br></li></div>
<li>Expression of the ecnB gene is induced by high osmolarity in stationary phase , positively regulated by RpoS ( 66 ) and negatively regulated by the EnvZ/OmpR two-compo-nent regulator ( 12 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15576800" target="_blank">15576800</a>]<br></li>
<li>Reﬂecting such situations , expression of the csgD gene is known to be positively or negatively regulated by a number of transcription factors , including MlrA ( 5 ) , Crl ( 3 ) , RpoS ( 3 ) , integration host factor ( 6 ) , Rcs ( 34 ) , the EnvZ-OmpR TCS ( 13 , 35 ) and the CpxAR TCS ( 13 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17468243" target="_blank">17468243</a>]<br></li>
<li>The OmpR-EnvZ TCS responds to changes in osmolarity and positively regulates csgD expression ( 13 , 35 ) , and thus the expression of csgD is activated by the EnvZ-OmpR TCS under high-osmolarity conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17468243" target="_blank">17468243</a>]<br></li>
<li>A prime example of a 2CS controlling the expression of sRNAs is EnvZ/OmpR , which activates the expression of three sRNA genes ( micF , omrA , and omrB ) and represses the expression of one sRNA gene ( micC ) ( 23 -- 25 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25246476" target="_blank">25246476</a>]<br></li>
<li>We also conﬁrmed that expression of both omrA-lux and omrB-lux was activated by the Cpx response indirectly via the Cpx-regulated connector protein MzrA and the EnvZ/OmpR 2CS as previously reported ( reference 42 and data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25246476" target="_blank">25246476</a>]<br></li>
<li>The EnvZ/OmpR system is known to control the expression of outer membrane porins in response to medium osmolarity ( Aiba and Mizuno , 1990 ; Pratt and Silhavy , 1995 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12366850" target="_blank">12366850</a>]<br></li>
<li>EnvZ and OmpR regulate the level of the two major outer membrane ( OM ) porins in E. coli , OmpC and OmpF , and although the molecular basis is not known , EnvZ is induced by high osmolarity ( Pratt and Silhavy , 1995a ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16359331" target="_blank">16359331</a>]<br></li>
<li>These results suggest that the EnvZ-OmpR two-component system not only changes the cell surface by modulating the expression of the major OM porins OmpC and OmpF , but further remodels it via expression of omrA and omrB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16359331" target="_blank">16359331</a>]<br></li>
<li>Expression of the general porins OmpC and OmpF is regulated by osmolarity via the two-component regulatory system EnvZ-OmpR ( 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15743943" target="_blank">15743943</a>]<br></li>
<li>We have shown that transcription of the sRNAs is activated by the two-component system EnvZ-OmpR and that , in turn , OmrA and OmrB negatively regulate the expression of multiple genes encoding OM proteins , including ompT , cirA , fecA and fepA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16359331" target="_blank">16359331</a>]<br></li>
<li>Expression of OmrA and OmrB in exponentially growing cells is transcriptionally controlled by the EnvZ-OmpR two-component system .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20407422" target="_blank">20407422</a>]<br></li>
<li>The EnvZ-OmpR two-component system represses the transcription of the large diameter pore porin gene ompF and activates the gene for the small pore porin ompC , consistent with the idea that limiting entry of possible toxic compounds may be important ( Nikaido , 2003 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16359331" target="_blank">16359331</a>]<br></li>
<li>Such reprogramming of gene expression is likely to be due to changes in environmental physicochemical conditions and to involve two-component systems such as EnvZ/OmpR ( 41 ) and CpxA/CpxR ( 15 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11717281" target="_blank">11717281</a>]<br></li>
<li>Osmolarity sensing occurs through the two-component regulatory systems EnvZ / OmpR and CpxA/CpxR ( 37 , 38 ) , but temperature-sensing of curli expression is not understood .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14978043" target="_blank">14978043</a>]<br></li>
<li>Mutations that speciﬁcally affect transcriptional control by OmpR and EnvZ , a two-component regulatory system that controls porin gene expression in E. coli , have also been isolated and lie in the same region of rpoA ( 28 , 30 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7836282" target="_blank">7836282</a>]<br></li>
<li>EnvZ controls porin gene expression by modulating OmpR ( 37 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1648075" target="_blank">1648075</a>]<br></li>
<li>Therefore , expression of these fusions is completely dependent upon the EnvZ-OmpR two-component system , at least in exponential phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16359331" target="_blank">16359331</a>]<br></li>
<li>Among them was ompC , whose expression is under control of the sensor kinase/response regulator system EnvZ/OmpR ( 25 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17015655" target="_blank">17015655</a>]<br></li>
<li>Transcription of ompC and ompF is regulated by the two-component regulatory system EnvZ/OmpR ( 25 , 45 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17015655" target="_blank">17015655</a>]<br></li>
<li>The ompC and micF genes shares regulatory sites for the OmpR osmoregulator , and expression of both genes is turned on at high osmolarity via the EnvZ-OmpR two-component system .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17007876" target="_blank">17007876</a>]<br></li>
<li>OmpR is known to regulate the expression of ompF and ompC via a two-component regulatory system in E. coli ( 17 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26209678" target="_blank">26209678</a>]<br></li>
<li>It is important to note that MzrA expression from the pTrc99A clone was unaffected in various mutant OmpR and EnvZ backgrounds ( Fig. 1D ) , demonstrating that a lack of negative effect on OMPs or MalE was not the result of reduced MzrA expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19432797" target="_blank">19432797</a>]<br></li>
<li>This required that the entericidin locus could be activated by sS and could not be repressed by EnvZ/OmpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9677290" target="_blank">9677290</a>]<br></li>
<li>In part , the EnvZ / OmpR pathway is activated by its modulator , MzrA , whose synthesis is controlled by both E and CpxRA ( 9 , 11 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22544271" target="_blank">22544271</a>]<br></li>
<li>It was also possible to eliminate the stimulation by performing the experiment using wild-type OmpR and the MBP-EnvZ kinase , but in the absence of ATP ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9096305" target="_blank">9096305</a>]<br></li>
<li>Furthermore , these fusions are also induced by procaine , a local anaesthetic that activates OmpR via EnvZ ( Taylor et al. , 1983 ; Ramp-ersaud and Inouye , 1991 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16359331" target="_blank">16359331</a>]<br></li>
				</div></ul></div><br>
	</div>
</body>
</html>