<!doctype html>
<style>
@charset "utf-8";
body {
	height: 100%;
	margin: 0;
	padding: 0;
}

.banner {
	position: fixed;
	top: 0;
	width: 100%;
	height: 65px;
	background: gray;
	line-height: 65px;
}
.spacer {
	height: 10px;
}

.main-content {
	padding: 5%;
}

.collapse_ACT {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-style: none solid none none;
	border-color: #B4045F;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_ACT + input{
	display: none;
}
.collapse_ACT + input + div{
	display: none;
}
.collapse_ACT + input:checked + div{
	display: block;
}
.ACT {
	text-align: justify;
	text-justify: inter-word;
}
.ACT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B4045F;
}
.ACT_collapse_similarity:hover{
	background: #e5d3dc;
}
.ACT_collapse_similarity + input {
	display: none;
}
.ACT_collapse_similarity + input + div{
	display: none;
}

.ACT_collapse_similarity + input:checked + div{
	display: block;
}
.ACT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.ACT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B4045F;
	padding-left: 5px;
}
.ACT_collapse_similarity:hover{
	background: #e5d3dc;
}
.ACT_collapse_similarity + input {
	display: none;
}
.ACT_collapse_similarity + input + div{
	display: none;
}
.ACT_collapse_similarity + input:checked + div{
	display: block;
}
.ACT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.ACT_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #B4045F;
}
.ACT_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_DOM {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-style: none solid none none;
	border-color: #B404AE;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_DOM + input{
	display: none;
}
.collapse_DOM + input + div{
	display: none;
}
.collapse_DOM + input:checked + div{
	display: block;
}
.DOM {
	text-align: justify;
	text-justify: inter-word;
}
.DOM_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B404AE;
	padding-left: 5px;
}
.DOM_collapse_similarity:hover{
	background: #e8d7e7;
}
.DOM_collapse_similarity + input {
	display: none;
}
.DOM_collapse_similarity + input + div{
	display: none;
}
.DOM_collapse_similarity + input:checked + div{
	display: block;
}
.DOM_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}.DOM_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #B404AE;
}
.DOM_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_EVO {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #7401DF;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_EVO + input{
	display: none;
}
.collapse_EVO + input + div{
	display: none;
}
.collapse_EVO + input:checked + div{
	display: block;
}
.EVO {
	text-align: justify;
	text-justify: inter-word;
}
.EVO_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #7401DF;
	padding-left: 5px;
}
.EVO_collapse_similarity:hover{
	background: #e2d7ed;
}
.EVO_collapse_similarity + input {
	display: none;
}
.EVO_collapse_similarity + input + div{
	display: none;
}
.EVO_collapse_similarity + input:checked + div{
	display: block;
}
.EVO_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.EVO_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #7401DF;
}
.EVO_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_RP {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #0174DF;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_RP + input{
	display: none;
}
.collapse_RP + input + div{
	display: none;
}
.collapse_RP + input:checked + div{
	display: block;
}
.RP {
	text-align: justify;
	text-justify: inter-word;
}
.RP_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #0174DF;
	padding-left: 5px;
}
.RP_collapse_similarity:hover{
	background: #c9d7e2;
}
.RP_collapse_similarity + input {
	display: none;
}
.RP_collapse_similarity + input + div{
	display: none;
}
.RP_collapse_similarity + input:checked + div{
	display: block;
}
.RP_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.RP_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #0174DF;
}
.RP_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_SIT {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #0489B1;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_SIT + input{
	display: none;
}
.collapse_SIT + input + div{
	display: none;
}
.collapse_SIT + input:checked + div{
	display: block;
}
.SIT {
	text-align: justify;
	text-justify: inter-word;
}
.SIT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #0489B1;
	padding-left: 5px;
}
.SIT_collapse_similarity:hover{
	background: #d0dfe2;
}
.SIT_collapse_similarity + input {
	display: none;
}
.SIT_collapse_similarity + input + div{
	display: none;
}
.SIT_collapse_similarity + input:checked + div{
	display: block;
}
.SIT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.SIT_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #0489B1;
}
.SIT_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_TU {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #04B486;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_TU + input{
	display: none;
}
.collapse_TU + input + div{
	display: none;
}
.collapse_TU + input:checked + div{
	display: block;
}
.TU {
	text-align: justify;
	text-justify: inter-word;
}
.TU_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #04B486;
	padding-left: 5px;
}
.TU_collapse_similarity:hover{
	background: #d0e2de;
}
.TU_collapse_similarity + input {
	display: none;
}
.TU_collapse_similarity + input + div{
	display: none;
}
.TU_collapse_similarity + input:checked + div{
	display: block;
}
.TU_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.TU_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #04B486;
}
.TU_similar > a {
	text-decoration: none;
	color: gray;
}

ul { 
	list-style-type: none;
	padding: 0;
	margin: 0;
}
ul li > a {
	text-decoration: none;
	color: gray;
}
ul li:hover, ul li:focus {
	background: #F0F0F0;
}

ul li:first-child span {
  border-left:none;  
}

</style>
<html>
<head>
<meta charset="utf-8">
<title>TF properties - OxyR</title>
<meta name="theme-color" content="#ffffff">
</head>
<body>
	<div class="banner">
		<center><font color="white" size="16px" face="arial">OxyR</font></center>
	</div>
	<div align="center" class="main-content">
		<div class="spacer">
	</div><br>			<label class="collapse_ACT" for="_1"><font size="5px">OxyR transcriptional activity</font></label>
				<input id="_1" type="checkbox">
				<div class="ACT"><br><ul><li>TA BL Act 0 0 0 Pr Reg Op AraC AraC AraC ArgR ArgR ArgR ArgR ArgR ArgR BioB BioB CRP CRP CRP CytR CytR CytR DeoR DeoR DnaA FNR Fur Fur Fur Fur GaIR GaIR GlpR IlvY LacI LexA LexA LexA LexA LexA LexA LexA LexA LexA MetJ MetJ MetJ MetJ MetR MetJ NR ( I ) NR ( I ) OxyR PapB PifC PurR PurR PurR PurR PurR PurR PurR PurR PurR PutA PutA RafR TetR TetR TetR TrpR araBAD araC araE argCBH argE argF argl argRpl carABp2 bioA bioB cya ompA spf ccd cytR tsx deopi deop2 rpoHp4 fnr cir pl cir p2 ffhuA * iucApl galpl galp2 glpD ilv Y lac cloDF14 colElpl lexA recA ssb sulA uvrA uvrBp2 uvrD metApi metB * metJpl * metE metR metF glnApi glnL oxyR papB pifC purC purE purF purH purL purM purR pyrC pyrD * putA * putP * rafA * tet tetRpl tetRp2 aroH 1,2 1,2 1 69 , 102 , 135 71 1 1 R R 1 0 1 1 0 0 1 1 1 1 0 0 0 0 0 0 1 1 1 0 1 1 0 0 1 1 0 1 0 0 1 0 0 0 0 0 1 0 1 1 1 1 1 0 1 1 0 1 0 0 0 0 0 0 0 0 0 0 1 1 0 1 1 1 0 2 1,4 1 4 4 2 1,2 1 1 2 2 4 4 1 4 1,2 1,2 2 4 2 2 4 2 1,2 1,2 2 3 1,2 4 2,4 2 2 4 4 2,4 2 2 4 1,2 1,2 1,4 1,3 1,4 1,2 1,2 3 2 1 3 3,4 1,3 3 3,4 4 1,2 1,3,4 4 2 2 3 2 2 2 2 145 25 25 25 25 76 18 , 125 24 , 104 24,104 2 97 109 86 142 86 , 142 4 , 86 , 163 , 164 4 , 86 , 163 , 164 167 35 45 45 23 , 30 16 , 29 , 30 1 , 85 1 , 36,101 166 33 , 34 11 , 77 11 , 77 32 92,160 140 , 160 140 , 160 106 , 158 20 , 146 , 149 37 64 49 49 , 168 122 , 123 49 49 , 89 89,141 89 , 124 19 , 171 171 47 47 6 52 , 53 52 , 53 52 , 53 65 1 0 0 0 0 0 0 0 0 0 0 0 1 1 1 0 1 0 0 0 0 0 0 1 1 1 0 1 0 1 0 0 0 0 0 0 0 0 0 0 1 0 0 1 0 0 1 0 0 0 0 0 0 0 0 0 0 1 0 0 0 0 0 0 0 P P P P P P 0 0 0 0 0 0 0 0 R R P 0 0 0 0 P R R P P R 0 P P 0 0 0 0 0 0 0 0 0 0 0 0 P 0 0 R 0 0 0 0 0 0 0 R only 0 0 0 0 P P P P 0 0 P P P P 0 0 0 P ( ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1943993" target="_blank">1943993</a>]<br></li>
<li>) 1,2 139 CRP cytR 0 1 p 1,2 142 CRP deop2 1 1 p 1,2 142 , 162 CRP galpi 1 1 0 1,2 60 CRP glpD 0 1 0 2,4 174 CRP glnApl 1 1 0 4 119 CRP ilvB 0 0 0 4 39 CRP lac 0 1 0 1,2 87 CRP malT 0 0 0 1,2 88 CRP melR 0 0 0 1,2 169 CRP nagB 1 0 0 4 108 CRP nagE 1 0 0 4 108 CRP pBRp4 0 0 0 2,4 113 CRP putA 1 1 0 4 47 CRP rpoHpS 1 0 0 4 99 CRP tnaA 0 0 0 2,4 27 CRP tsx 0 1 p 4 142 , 165 FNR dmsA 0 0 0 4 35 FNR fdrA 0 0 0 1,4 35 , 61 FNR , Y gipA 0 0 0 4 35 FNR , Y narG 0 0 0 1,4 35 , 75 FNR nirB 0 1 0 1,4 35 , 61 , 62 FNR PFLp6 1 0 0 4 133 FNR PFLp7 1 0 0 4 75 , 133 IlvY ilvC 1 0 p 3 170 MalT malE 1 1 R 1,2 114 , 115 MalT malK 1 1 R 1,2 114 , 115 MalT malPQ 0 0 0 1,4 138 MetR metE 1 1 0 1,3 161 MetR metH 0 0 0 4 160 , 161 OmpR ompC 0 0 p 1,2 80 , 81 , 154 OmpR ompF 0 0 p 1,2 38 , 116 , 154 OxyR ahpC 0 0 0 3 146 OxyR katG 0 0 0 3 146 , 149 PapB papB 1 1 R 1,2 37 , 41 PhoB phoA 0 0 0 4 84 PhoB phoB 0 0 0 1,4 84 , 154 PhoB phoE 0 0 P ( ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1943993" target="_blank">1943993</a>]<br></li>
<li>JOURNAL OF BACTERIOLOGY , Mar. 1995 , p. 1275 -- / 95 / $ 04.0010 Copyright q 1995 , American Society for Microbiology Mutational Analysis of the Redox-Sensitive Transcriptional Regulator OxyR : Regions Important for Oxidation and Transcriptional Activation INES KULLIK ,1 † MICHEL B. TOLEDANO ,1 ‡ LOUIS A. TARTAGLIA ,2 § AND GISELA STORZ1 * Cell Biology and Metabolism Branch , National Institute of Child Health and Human Development , Bethesda , Maryland 20892,1 and Division of Biochemistry and Molecular Biology , University of California , Berkeley , California 947202 Received 26 July 1994/Accepted 15 December 1994 OxyR is a redox-sensitive transcriptional-regulator of the LysR family which activates the expression of genes important for the defense against hydrogen peroxide in Escherichia coli and Salmonella typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7868602" target="_blank">7868602</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_4">
Kullik , I. , Stevens , J. , Toledano , M.B. and Storz , G. ( 1995 ) Mutational analysis of the redox-sensitive transcriptional-regulator OxyR : regions important for DNA binding and multimerization .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9312050" target="_blank">9312050</a>]<br></label>
<input id="act_similarity_4", type="checkbox">
<div class="ACT_similar">
<li>Kullik , I. , Stevens , J. , Toledano , M.B. , and Storz , G. ( 1995a ) Mutational analysis of the redox-sensitive transcriptional-regulator OxyR : regions important for DNA binding and multimerization .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15255893" target="_blank">15255893</a>]<br></li></div>
<li>Mutational analysis of the redox-sensitive transcriptional-regulator OxyR : regions important for oxidation and transcriptional activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20118257" target="_blank">20118257</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10094691" target="_blank">10094691</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12029051" target="_blank">12029051</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17464064" target="_blank">17464064</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11443092" target="_blank">11443092</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22904281" target="_blank">22904281</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21378183" target="_blank">21378183</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12644490" target="_blank">12644490</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626295" target="_blank">8626295</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8990283" target="_blank">8990283</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25266384" target="_blank">25266384</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10400599" target="_blank">10400599</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_7">
J Bacteriol 174:7104 -- 7111 Kullik I , Toledano MB , Tartaglia LA , Storz G ( 1995 ) Mutational analysis of the redox-sensitive transcriptional-regulator OxyR : regions important for oxidation and transcriptional activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8588737" target="_blank">8588737</a>]<br></label>
<input id="act_similarity_7", type="checkbox">
<div class="ACT_similar">
<li>[ PubMed : 17289666 ] Kullik I , Toledano MB , Tartaglia LA , Storz G. Mutational analysis of the redox-sensitive transcriptional-regulator OxyR : regions important for oxidation and transcriptional activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21143317" target="_blank">21143317</a>]<br></li>
<li>Kullik , I. , Toledano , M.B. , Tartaglis , L.A. , and Storz , G. ( 1995 ) Mutational analysis of the redox-sensitive transcriptional-regulator OxyR : regions important for oxidation and transcriptional activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10692164" target="_blank">10692164</a>]<br></li>
<li>Kullik , I. , Toledano , M.B. , Tartagelia , L.A. and Storz , G. ( 1995 ) Mutational analysis of the redox-sensitive transcriptional-regulator OxyR : regions important for oxidation and transcriptional activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11407111" target="_blank">11407111</a>]<br></li>
<li>Kullik , I. , Toledano , M.B. , Tartaglia , L.A. , and Storz , G. ( 1995b ) Mutational analysis of the redox-sensitive transcriptional-regulator OxyR : regions important for oxidation and transcriptional activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15255893" target="_blank">15255893</a>]<br></li>
<li>Kullik , I. , Toledano , M.B. , Tartaglia , L.A. , and Storz , G. ( 1995 ) Mutational analysis of the redox-sensitive transcriptional-regulator OxyR : regions important for oxidation and transcriptional activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10564497" target="_blank">10564497</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11972787" target="_blank">11972787</a>]<br></li>
<li>Kullik , I. , Toledano , M.B. , Tartaglia , L.A. and Storz , G. ( 1995 ) Mutational analysis of the redox-sensitive transcriptional-regulator OxyR : regions important for oxidation and transcriptional activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7556074" target="_blank">7556074</a>]<br></li></div>
<li>Crp , GalS MglAB Crp , RbsR RbsAB Crp , GalS MglA Crp , GntR GntK Crp , EbgR EbgA Crp , LacI LacZ Crp , RhaS RhaBD Crp , AraC AraBD Crp , FucR FucK Crp , YiaJ LyxK Crp , FucR FucA Crp , YiaJ SgbE RcsB WcaB CpxR LpxA RcsB WcaB CpxR LpxD RcsB B2060 PhoB PhnCKLN RcsB B2060 PhoB PstB RcsB B2060 PhoB PhoH MhpR MhpF Fur EntA FecI , Fur FecI UidR , UxuR UidR Crp , FucR FucR Crp , Fur Fur Homologous pairs of transcription factors with autoregulation IdnR PurR AsnC Lrp Fnr Crp Homologous triplets of transcription factors with autoregulation TorR PhoB CpxR EmrR PdhR ExuR Eight homologous transcription factors with autoregulation DsdC GcvA CynR Hcar IlvY LysR OxyR CysB Figure 6 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12582243" target="_blank">12582243</a>]<br></li>
<li>aerobic to anaerobic anaerobic to aerobic ArgR CysB BirA DgsA MalT GadW MetJ NikR OxyR SoxR SoxS DcuR CueR FadR HcaR NtrC PhoB PurR SgrR TdcA TrpR ZraR ArcA CpxR CRP CusR FhlA Fis FNR Fur H-NS IHF IscR Lrp ModE NarL NrdR PdhR FlhDC GlcC EvgA GadX LeuO PspF RbsR steady states Figure 3 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22870390" target="_blank">22870390</a>]<br></li>
<li>sitesa % CGb Count % Non-codinge Ada 3 38.7 AraC 6 40.3 2 3 100 100 47 76 100 28 33 32 73 38 100 52 100 86 79 89 100 67 26 100 28 100 70 37 67 28 100 82 100 47 50 51 100 100 38 33 64 3 6 100 100 38 54 100 23 40 18 43 29 100 38 100 39 27 53 58 71 19 100 23 33 47 100 40 65 21 23 60 25 100 41 100 26 43 36 100 35 35 30 64 36 23 22 26 14 100 83 28 ArcA 12 20.0 ArgR 17 22.9 CarP 2 46.0 Crp 49 37.0 CspA 4 41.3 CynR 2 33.3 CysB 3 34.2 CytR 5 37.8 DeoR 3 22.9 DnaA 8 36.7 FadR 7 47.0 FarR 4 20.0 FhlA 3 42.0 Fis 19 38.2 FlhCD 3 53.8 Fnr 9 25.8 FruR 12 42.7 Fur 9 23.4 GalR 7 43.8 GcvA 4 22.5 GlpR 13 39.6 HipB 4 40.9 Hns 15 31.5 IclR 2 26.7 Ihf 26 32.6 IlvY 2 32.4 LacI 3 44.4 LexA 18 33.9 Lrp 18 28.9 MalT 10 61.0 MarR 2 37.5 MelR 2 33.3 MetJ 15 40.1 MetR 8 33.3 NagC 6 22.5 NarL 9 34.6 NtrC 6 51.9 OmpR 9 26.7 OxyR 4 42.3 PdhR 3 33.3 PhoB 15 29.7 PurR 23 45.3 RhaS 2 50.0 RpoD15 27 37.4 RpoD16 48 34.9 RpoD17 116 37.3 RpoD18 34 38.0 RpoD19 25 38.2 RpoE 3 38.9 RpoH13 8 42.1 RpoH14 7 44.3 RpoN 7 40.6 RpoS17 15 43.0 RpoS18 7 45.2 SoxS 14 40.8 TorR 4 37.5 TrpR 4 38.5 Tus 6 26.8 TyrR 17 33.2 a The number of input sites used to construct the matrix .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9813115" target="_blank">9813115</a>]<br></li>
<li>of additional probable sites in predictionsd 3 3 7e 7 -- 4 22e 0 1 0 2 -- -- 5 0 1 -- 13 7 18 1 0 6e 1 1 1 3 Ada AraC ArcA ArgR CarP CpxR Crp CspA CynR CysB CytR DeoR DnaA FadR FhlA Fis FlhCD Fnr FruR Fur GalR GcvA GlpR IciA ( ArgP ) IclR Ihf IlvY LacI LexA Lrp MalT MarR MelR MetJ MetR Mlc ( DgsA ) ModE NagC NarL NarP NtrC OmpR OxyR PdhR PhoB PurR RhaR RhaS SoxR SoxS TorR TrpR TyrR Stem -- loopsf 6 2 0 5 2 9 1 3 4 5e 5 -- 5 5 4 6 3 4 2 -- 1 3 5 1 3 -- aThe number of genes in our study set with documented binding sites for each TF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11160901" target="_blank">11160901</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_18">
The dps promoter is activated by OxyR during growth and by IHF and sigma S in stationary phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17906139" target="_blank">17906139</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21239586" target="_blank">21239586</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11292794" target="_blank">11292794</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9829938" target="_blank">9829938</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19376854" target="_blank">19376854</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8779563" target="_blank">8779563</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12218039" target="_blank">12218039</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11133959" target="_blank">11133959</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17873038" target="_blank">17873038</a>]<br></label>
<input id="act_similarity_18", type="checkbox">
<div class="ACT_similar">
<li>The dps promoter is activated by OxyR during growth and by IHF and sS in stationary phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9294462" target="_blank">9294462</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11073934" target="_blank">11073934</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10892648" target="_blank">10892648</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8631665" target="_blank">8631665</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7768855" target="_blank">7768855</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8763946" target="_blank">8763946</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9512707" target="_blank">9512707</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10542180" target="_blank">10542180</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9324269" target="_blank">9324269</a>]<br></li>
<li>The dps promoter is activated by OxyR during growth and by IHF and S in stationary phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15489429" target="_blank">15489429</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15716429" target="_blank">15716429</a>]<br></li>
<li>The dps promoter is activated by OxyR during growth and by IHF and as in stationary phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8087856" target="_blank">8087856</a>]<br></li>
<li>The dps promoter is activated by OxyR during growth and by IHF and ss in stationary phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9335308" target="_blank">9335308</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7751275" target="_blank">7751275</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9230301" target="_blank">9230301</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8990283" target="_blank">8990283</a>]<br></li>
<li>The dps promoter is activated by OxyR during growth and by IHF and ss during stationary phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7868602" target="_blank">7868602</a>]<br></li></div>
<li>We thank the Fondation pour la Recherche MeÂdicale for a fellowship to F.R. , and the NATO Science Program for a Collaborative Research Grant ( CRG 972150 ) References Altuvia , S. , Almiron , M. , Huisman , G. , Kolter , R. , and Storz , G. ( 1994 ) The dps promoter is activated by OxyR during growth and by IHF and s in stationary phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10692161" target="_blank">10692161</a>]<br></li>
<li>pKOXS B. Demple Media and cell growth conditions Altuvia , S. , Almiron , M. , Huisman , G. , Kolter , R. , and Storz , G. ( 1994 ) The dps promoter is activated by OxyR during P growth and by IHF and os in stationary phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8899708" target="_blank">8899708</a>]<br></li>
<li>Referencec Altuvia , S. , Almirón , M. , Huisman , G. , Kolter , R. , and Ston , G. ( 1994 ) The dps promoter is activated by OxyR during growth and by IHF and os in stationary phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596457" target="_blank">8596457</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_27">
[ 5 ] S. Altuvia , M. Almiron , G. Huisman , R. Kolter , G. Storz , The dps promoter is activated by OxyR during growth and by IHF and sigma S in stationary phase , Mol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15878120" target="_blank">15878120</a>]<br></label>
<input id="act_similarity_27", type="checkbox">
<div class="ACT_similar">
<li>[ 24 ] Altuvia S. , Almiron M. , Huisman G. , Kolter R. , Storz G , The dps promoter is activated by OxyR during growth and by IHF and sigma S in stationary phase , Mol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9924977" target="_blank">9924977</a>]<br></li></div>
<li>J Mol Biol 215 :403 410 Altuvia S , Almir6n M , Huisman G , Kolter R , Storz G ( 1994 ) The dps promoter is activated by OxyR during growth and by IHF and as in stationary phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8709966" target="_blank">8709966</a>]<br></li>
<li>Genes Dev 6:2646 -2654 Altuvia S , Almiron M , Huisman G , Kolter R , Storz G ( 1994 ) The dps promoter is activated by OxyR during growth and by IHF and ~ rs in stationary phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7476863" target="_blank">7476863</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_31">
Altuvia , S. , Almiron , M. , Gjalt , H. , Kolter , R. , and Storz , G. ( 1994 ) The dps promoter is activated by OxyR during growth and by IHF and sS in stationary phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18452510" target="_blank">18452510</a>]<br></label>
<input id="act_similarity_31", type="checkbox">
<div class="ACT_similar">
<li>References Altuvia , S. , Almiron , M. , Huisman , G. , Kolter , R. , and Storz , G. ( 1994 ) The dps promoter is activated by OxyR during growth and by IHF and sS in stationary phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15101990" target="_blank">15101990</a>]<br></li>
<li>References Altuvia , S. , AlmiroÂn , M. , Huisman , G. , Kolter , R. , and Storz , G. ( 1994 ) The dps promoter is activated by OxyR during growth and by IHF and sS in stationary phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10760142" target="_blank">10760142</a>]<br></li>
<li>References Altuvia , S. , Almirón , M. , Huisman , G. , Kolter , R. , and Storz , G. ( 1994 ) The dps promoter is activated by OxyR during growth and by IHF and ss in stationary phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767577" target="_blank">9767577</a>]<br></li>
<li>References Altuvia , S. , Almiron , M. , Huisman , G. , Kolter , R. , and Storz , G. ( 1994 ) The dps promoter is activated by OxyR during growth and by IHF and ss in stationary phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9663683" target="_blank">9663683</a>]<br></li>
<li>Altuvia , S. , AlmiroÂn , M. , Huisman , G. , Kolter , R. , and Storz , G. ( 1994 ) The dps promoter is activated by OxyR during S growth and by IHF and s in stationary phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11251833" target="_blank">11251833</a>]<br></li>
<li>Altuvia , S. , Almirón , M. , Huisman , G. , Kolter , R. , and Storz , G. ( 1994 ) The dps promoter is activated by OxyR during growth and by IHF and S in stationary phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23139412" target="_blank">23139412</a>]<br></li>
<li>References Altuvia , S. , Almiron , M. , Huisman , G. , Kolter , R. , and Storz , G. ( 1994 ) The dps promoter is activated by OxyR during growth and by IHF and sigma S in stationary phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15009899" target="_blank">15009899</a>]<br></li>
<li>References Altuvia S , Almiron M , Huisman G , Kolter R , Storz G ( 1994 ) The dps promoter is activated by OxyR during growth and by IHF and sigma S in stationary phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16404567" target="_blank">16404567</a>]<br></li>
<li>References Altuvia S , Almiron M , Hulsman G , Kolter R , Storz G ( 1994 ) The dps promoter is activated by OxyR during growth and by IHF and rS in stationary phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9197421" target="_blank">9197421</a>]<br></li>
<li>References Altuvia S , Almiron M , Huisman G , Kolter R & Storz G ( 1994 ) The dps promoter is activated by OxyR during growth and by IHF S and s in stationary phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19076234" target="_blank">19076234</a>]<br></li>
<li>Altuvia , S. , Almiron , M. , Huisman , G. , Kolter , R. , and Storz G. ( 1994 ) The dps promoter is activated by OxyR during growth and by IHF and sigmaS in stationary phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17224607" target="_blank">17224607</a>]<br></li>
<li>Altuvia , S. , Almiron , M. , Huisman , G. , Kolter , R. and Storz , G. ( 1994 ) The dps promoter is activated by OxyR during growth and by IHF and s S in stationary phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11230141" target="_blank">11230141</a>]<br></li></div>
<li>In stationary phase , OxyR , for unknown reasons , is not activated by hydrogen peroxide ( Altuvia et al. , 1994 ; Gonzá - lez-Flecha and Demple , 1995 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9230301" target="_blank">9230301</a>]<br></li>
<li>Transcription of the single dps promoter requires os and integration host factor during entry into stationary phase , but is dependent on 070 and OxyR for induction in the presence of hydrogen peroxide ( Altuvia et al. , 1994 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596457" target="_blank">8596457</a>]<br></li>
<li>Since OxyR is strongly activated by hydrogen peroxide during log-phase growth , ss is not needed , and ss repression by oxyS prevents redundancy .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9230301" target="_blank">9230301</a>]<br></li>
<li>To further characterize the OxyR regulation of dps , we also examined the induction of dps transcript in response to different concentrations of hydrogen peroxide in both log and stationary phase cells .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7984106" target="_blank">7984106</a>]<br></li>
<li>Transcription of this gene depends on Eo `` 7 ° in the presence of a transcriptional-activator , OxyR , and depends on E ( ~ ~ in the absence of OxyR in stationary phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8825088" target="_blank">8825088</a>]<br></li>
<li>In these experiments , due to the toxicity of L-ascorbate to the OxyR mutant , cells were first grown to stationary phase in CAA and then incubated for 3 h in the presence of 10 mM L-ascorbate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17906139" target="_blank">17906139</a>]<br></li>
<li>This OxyR-dependent induction occurs oniy during iog phase , aithough the OxyR protein is present in stationary phase , in the stationary phase ceits , dps is expressed in a a ^ - and iHF-dependent manner .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7984106" target="_blank">7984106</a>]<br></li>
<li>In this context it is interesting to consider that : ( i ) transcription of LucZ , another positively regulated gene , is also down-regu-lated during starvation [ 6 ] ; and ( ii ) the dps promoter [ 20 ] is differentially regulated , being OxyR-and a '' - dependent during exponential-growth , and IHF-and as-dependent during stationary phase , even though the OxyR protein is present in both phases [ 20 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7557316" target="_blank">7557316</a>]<br></li>
<li>The altered metabolism or altered levels of intracellular compounds such as iron , NADPH/NADP '' ^ and GSH/GSSG in the starved and stationary phase cells may also impinge on OxyR activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7984106" target="_blank">7984106</a>]<br></li>
<li>An example of such regulation is provided by the dps gene , which is strongly activated by E S in stationary phase or under other stress conditions ( dps is a core gene ) ( Table 1 ) but which can also be activated by E 70 cooperating with the H2O2-activated regulator OxyR ( 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15716429" target="_blank">15716429</a>]<br></li>
<li>Some class II mutants , which are activated by OxyR , showed increased levels of expression at the onset of stationary phase ( Fig. 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8990283" target="_blank">8990283</a>]<br></li>
<li>There are several possibl explanations for this lack of induction , ( i ) RNA polymerase-a ™ holoenzyme may be altered so that the polymerase can not be activated by OxyR in stationary phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7984106" target="_blank">7984106</a>]<br></li>
<li>Reference Regulon Gene Encode protein Known function SoxRS regulatory protein keeps superoxide levels low SoxS protein SoxRS Li and Demple , 1994 Touati , 1992 Fraenkel and Parola , 1972 Gruer and Guest , 1994 soxS sodA zwf acnA Manganese superoxide dismutase glucose-6-phosphate dehydrogenase superoxide resistant aconitase supplies NADPH as reducing power replaces inactivated aconitase in Kreb 's cycle heat stable fumarase C multidrug efflux pump replaces inactivated fumarase in Kreb 's cycle efflux pumps in inner membrane to maintain levels of antibiotics and other compounds Catalyses hydrogen peroxide to water and oxygen Liochev and Fridovich , 1992 Ma et al. , 1995 fumC acrAB Christman et al. , 1989 Tartaglia et al. , 1989 OxyR katG Hydroperoxidase 1 ahpC Alkyl hydroperoxide reductase Detoxifies damaging hydroperoxides Gene Forward primer 5V -- 3V Reverse primer 5V -- 3V promoter size gcc act ctt tcg cac tta acg gta atc gcg tta ctc atc tt soxS gca taa atc tgc ctc ttt tca 361 sodA gca tat tca tct cca gta cc 304 gaa aga aat cga aat gca gtt ggt cct gga tgg gcg cgt tt zwf gca tgt cat tct cct taa gtt 301 acnA gca tag ctc ctc ctt aat gac 237 gaa tgc acc cgc tgt gtg aaa gat ctc act gaa caa atc cg fumC gca tga cct gct cct cac ct 165 acrAB gca tat gta aac ctc gag tg 330 gtg tgg ctt tta tga aaa tca gct tag atc agg tga ttg cc katG gca tca atg tgc tcc cct ct 328 ahpC gca tct ata ctt cct ccg tgt 306 The resulting fusions were transformed into E. coli strain Top10 and screened for green fluorescence emission on Luria-Bertani ( LB ) ( 10 g/L NaCl , 5 g/L yeast extract , 10 g/L tryptone , pH 7.2 ) plates containing 100 Ag/mL ampicil-lin and a corresponding inducer ( paraquat or hydrogen peroxide ) when excited by a hand-held UV light model UVGL-25 , UVP ( San Gabriel , CA ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15672380" target="_blank">15672380</a>]<br></li>
<li>12.38 -8.24 -59.96 1.13 ± 0.55 ± 0.50 ± 0.02 0.04 0.02 1.27 ± 0.50 ± 0.20 ± 0.03 0.03 0.03 / h ) / h ) Fig. 2 Comparison of the transcript levels of the E. coli wild type under different pH values at 7.0 and 6.0 : a global regulatory genes , b PhoB regulatory genes , c acid-inducible genes , d metabolic pathway genes , e respiratory chain genes during micro-aerobic continuous culture at the dilution rate of 0.1 h -1 Fig. 3 a , b Overall metabolic regulation of the E. coli during acidic condition ( a ) Acid Condition Hns ATP ADP PhoQ PhoQ P ATP ADP ¬ EvgS EvgS P cAMP - evgA Pho P PhoP Crp EvgA P B1500 EvgA cAMP-Crp ¬ RpoS b1500 ydeo ¬ YdeO gadE GadE O O GadW GadX hdeA gadB gadc gadW gadA gadX HdeA GadA GadB GadC Acid resistance ( b ) Acidic Condition ( H + ) ( indirect ) EnvZ PhoR fnr OxyR ldhA arcA P Fnr ArcA SoxR / S L PhoR - P OmpR OmpR - P DH pfl yfiD ompC TCA cycle genes : gltA sdhC mdh sodA PhoB - P PhoB Lactate OmpC ompF asr YfiD Pfl OmpF Asr Formate Acetate ackA pta Acid resistance Moreover , in accord with the proteomic analysis , the expression of yﬁD : : lac reporter fusion and the intracellular content of YﬁD were found to be high during growth at low pH for both aerobic and anaerobic cultures ( Neil et al. 2002 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23274360" target="_blank">23274360</a>]<br></li>
<li>Endogenous protein S-nitrosylation in E. coli : regulation by OxyR Divya Seth1 , Alfred Hausladen1 , Ya-Juan Wang2 , and Jonathan S. Stamler1 ,3 1Institute for Transformative Molecular Medicine and Department of Medicine , Case Western Reserve University School of Medicine and University Hospitals Case Medical Center , Cleveland , OH 44106 , USA 2Center for Proteomics and Bioinformatics , Case Western Reserve University School of Medicine , Cleveland , OH 44106 , USA Abstract Endogenous S-nitrosylation of proteins , a principal mechanism of cellular signaling in eukaryotes , has not been observed in microbes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22539721" target="_blank">22539721</a>]<br></li>
<li>OxyR and SoxRS regulation of fur .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10714981" target="_blank">10714981</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17464064" target="_blank">17464064</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11443092" target="_blank">11443092</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11395452" target="_blank">11395452</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21378183" target="_blank">21378183</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11443091" target="_blank">11443091</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18227264" target="_blank">18227264</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11931550" target="_blank">11931550</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_60">
Zheng , M. , Doan , B. , Schneider , T.D. and Storz , G. ( 1999 ) OxyR and SoxRS regulation of fur .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17921503" target="_blank">17921503</a>]<br></label>
<input id="act_similarity_60", type="checkbox">
<div class="ACT_similar">
<li>Zheng , M. , Doan , B. , Schneider , T. D. , and Storz , G. ( 1999 ) OxyR and SoxRS regulation of fur .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22223481" target="_blank">22223481</a>]<br></li>
<li>Zheng , M. , Doan , B. , Schneider , T.D. , and Storz , G. ( 1999 ) OxyR and SoxRS regulation of fur .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15009899" target="_blank">15009899</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14617150" target="_blank">14617150</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20015147" target="_blank">20015147</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15101990" target="_blank">15101990</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10564485" target="_blank">10564485</a>]<br></li>
<li>[ 26 ] M. Zheng , B. Doan , T.D. Schneider , G. Storz , OxyR and SoxRS regulation of fur , J. Bacteriol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16183297" target="_blank">16183297</a>]<br></li>
<li>[ 6 ] M. Zheng , B. Doan , T.D. Schneider , G. Storz , OxyR and SoxRS regulation of fur , J. Bacteriol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15878120" target="_blank">15878120</a>]<br></li>
<li>Zheng M , Doan B , Schneider TD , Storz G : OxyR and SoxRS regulation of fur .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21272324" target="_blank">21272324</a>]<br></li>
<li>[ PubMed : 4352182 ] Zheng M , Doan B , Schneider TD , Storz G. OxyR and SoxRS regulation of fur .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19400769" target="_blank">19400769</a>]<br></li></div>
<li>The YaaA Protein of the Escherichia coli OxyR Regulon Lessens Hydrogen Peroxide Toxicity by Diminishing the Amount of Intracellular Unincorporated Iron † Yuanyuan Liu , Sarah C. Bauer , and James A. Imlay * Department of Microbiology , University of Illinois , Urbana , Illinois 61801 Received 2 January 2011/Accepted 26 February 2011 Hydrogen peroxide ( H2O2 ) is commonly formed in microbial habitats by either chemical oxidation processes or host defense responses .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21378183" target="_blank">21378183</a>]<br></li>
<li>Hydrogen Escherichia coli Isc ironsulfur assembly system , and OxyR induces the Suf system to compensate peroxide inactivates the Soojin Jang # and James A. Imlay * Department of Microbiology , University of Illinois , Urbana , IL 61801 bstract A Environmental H2O2 creates several injuries in Escherichia coli , including the oxidative conversion of dehydratase [ 4Fe-4S ] clusters to an inactive [ 3Fe-4S ] form .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21143317" target="_blank">21143317</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_69">
Hydrogen peroxide inactivates the Esche-richia coli Isc iron-sulphur assembly system , and OxyR induces the Suf system to compensate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21378183" target="_blank">21378183</a>]<br></label>
<input id="act_similarity_69", type="checkbox">
<div class="ACT_similar">
<li>Hydrogen peroxide inactivates the Escherichia coli Isc iron-sulphur assembly system , and OxyR induces the Suf system to compensate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23075318" target="_blank">23075318</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25266384" target="_blank">25266384</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21338418" target="_blank">21338418</a>]<br></li></div>
<li>Three reporter strains , one lacking IscR , another lacking OxyR and their isogenic wild-type strain , were assayed after 60 min of wrwycr treatment .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22096151" target="_blank">22096151</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22096151" target="_blank">22096151</a>]<br></li>
<li>Details are shown in Supplementary Table S2 , but briefly this consisted of 184 genes that could be divided into the following categories : repressed by Fur , 63 ; activated by Fur , 15 ; repressed by CpxR , 21 ; activated by CpxR , 37 ; repressed by SoxS , 1 ; activated by SoxS , 23 ; repressed by OxyR , 2 ; activated by OxyR , 16 ; activated by CusR , 6 ; activated by BaeR , 9 ; repressed by Zur , 4 ; repressed by ArsR , 3 ; activated by ZraR , 3 ; activated by ZntR , 1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22016571" target="_blank">22016571</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_73">
Statistical modelling of the transcriptomic data generated from the Zn shift focused on the role of ten key regulators ; ArsR , BaeR , CpxR , CusR , Fur , OxyR , SoxS , ZntR , ZraR and Zur .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22016571" target="_blank">22016571</a>]<br></label>
<input id="act_similarity_73", type="checkbox">
<div class="ACT_similar">
<li>The analysis of the transcriptomic data generated from the Zn shift focused on the role of ten key regulators : ArsR , BaeR , CpxR , CusR , Fur , OxyR , SoxS , ZntR , ZraR and Zur .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22016571" target="_blank">22016571</a>]<br></li></div>
<li>They are involved in processes as diverse as respiration , gene regulation , cofactor biosynthesis , and DNA-RNA metabolism .7 There is tremendous interest in understanding the mechanisms of Fe-S cluster biosynthesis , one reason being that several human diseases , including Friedreich 's ataxia , are the result of genetic defects in the protein machinery that makes Fe-S clusters .7 10.1021 / pr100920r 2010 American Chemical Society 6740 Journal of Proteome Research 2010 , 9 , 6740 -- 6751 Published on Web 10/11/201 The SUF system of E. coli is comprised of complex protein machinery , which , under the control of OxyR , IHF , IscR , andFur , synthesizes Fe S clusters during the potentially cytotoxic conditions of iron limitation and oxidative-stress .6,11 -14 ,17,18 The SUF system is encoded by the sufABCDSE operon .14 SufS is a cysteine desulfurase that mobilizes inorganic sulfur via a pyridoxal-5 ′ - phosphate cofactor and a conserved cysteine ( C ) residue and transfers the sulfur atom through a proposed persulﬁde transfer mechanism to the conserved C51 residue on the sulfur acceptor protein SufE.19-21 SufB , SufC , and SufD form a complex ( SufBC2D , hereafter referred to as SufBCD ) 22 that stimulates this cysteine desulfurase activity through an unknown mechanism ,21 though SufC is known to be an atypical ABC transporter ATPase .12 In vitro evidence suggests that SufE transfers the sulfur atom to SufB within the SufBCD complex and that SufBCD acts as a scaffold upon which an Fe-S cluster is assembled.22-24 It is not known whether the persulﬁde sulfur on SufE is liberated via nucleophilic attack by the recipient Cys thiol on the terminal sulfur of the persulﬁde , which would result in persulﬁde transfer , or via nucleophilic attack on the bridging sulfur , which would produce a disulﬁde-linked complex between SufE and the recipient protein and liberate sulﬁde that may then bind a free Cys residue for use in assembling an Fe-S cluster .5 It also remains to be determined whether the assembled cluster on SufBCD is transferred to a target apoprotein directly or via SufA.22,25-27 It is possible that SufE transfers sulfur to proteins other than SufB , as both the iron source and the means by which the sulfur on SufE is incorporated into an Fe-S cluster remain unknown .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20936830" target="_blank">20936830</a>]<br></li>
<li>However , unlike Lrp and OxyR binding to their operator sequences at pap and ag43 respectively ( Nou et al. , 1993 ; Haagmans and van der Woude , 2000 ) , methylation does not seem to affect the binding of NagC to O .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17238917" target="_blank">17238917</a>]<br></li>
<li>Wallecha , A. , Correnti , J. , Munster , V. , and van der Woude , M. ( 2003 ) Phase variation of Ag43 is independent of the oxidation state of OxyR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23139412" target="_blank">23139412</a>]<br></li>
<li>Molecular Microbiology ( 2000 ) 35 ( 4 ) , 877 ± 887 Phase variation of g43 in Escherichia coli : e d abrogates OxyR scription A ent methylation mediated repression Dam-dep n binding and OxyR ¬ of tran W. Haagmans and M. van der Woude * Introduction Department of Microbiology , University of Pennsylvania , 202A Johnson Pavilion , 3610 Hamilton Walk , Philadelphia , PA 19104 , USA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10692164" target="_blank">10692164</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_79">
Haagmans , W. , and Van der Woude , M. ( 2000 ) Phase variation of Ag43 in Escherichia coli : Dam-dependent methylation abrogates OxyR binding and OxyR-mediated repression of transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11972787" target="_blank">11972787</a>]<br></label>
<input id="act_similarity_79", type="checkbox">
<div class="ACT_similar">
<li>Haagmans , W. , and van der Woude , M. ( 2000 ) Phase variation of Ag43 in E. coli : Dam-dependent methylation abrogates OxyR binding and OxyR-mediated repression of transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17238917" target="_blank">17238917</a>]<br></li></div>
<li>Ten mutants were grouped as class II fusions because they were OxyR activated ( required functional OxyR for maximal expression of the lacZ fusion ; hps-2 , hps-4 , hps-5 , hps-10 , hps-14 , hps-16 , hps-20 , hps-21 , hps-24 , and hps-27 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8990283" target="_blank">8990283</a>]<br></li>
<li>Classiﬁcation of hps : : lacZ mutants Known mutants with similar phenotypesa Class Fusions Characteristic ( s ) Independent of OxyR I hps-1 , hps-3 , hps-7 , hps-11 , hps-12 , hps-15 , hps-18 , hps-23 hsp-2 , hps-4 , hps-5 , hps-10 , hps-14 , hps-16 , hps-20 , hps-21 , hps-24 , hps-27 hps-6 , hps-13 , hps-19 , hps-22 , hps-25 , hps-26 sodA , sodB , xthA , dnaK OxyR activated ; requirement of functional OxyR for maximum expression OxyR repressed ; expression of the fusion reduced in the presence of OxyR a See text for discussion and references .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8990283" target="_blank">8990283</a>]<br></li>
<li>However , the OxyR switch is poised at -185 mV , about 90 mV higher than the E. coli cytoplasm ( rather than 120 mV lower in the case of FLP ) , so OxyR exists mainly in the reduced ( inactive ) form under conditions of normoxia ( Zheng et al. , 1998 ) whilst FLP is oxidized and active under these conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11407111" target="_blank">11407111</a>]<br></li>
<li>This can be due to the ability of s70-containing RNA polymer-ase to activate transcription at otherwise sS-depen-dent promoters if a stress-regulated activator , such as e.g. OxyR , can bind at an appropriate upstream position ( Altuvia et al. , 1994 ; Becker-Hapak & Eisenstark , 1995 ; Ivanova et al. , 1994 ; Lomovskaya et al. , 1994 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9512707" target="_blank">9512707</a>]<br></li>
<li>Although OxyR can activate transcription by Es70 , and IHF can enhance transcription by Es38 ( Altuvia et al. , 1994 ; Jeong et al. , 2006 ) , the molecular basis of how dps expression is regulated by growth-phase is not known .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18452510" target="_blank">18452510</a>]<br></li>
<li>We and others have observed that several genes such as katG , dps , and gorA , are regulated by both OxyR and ss ( Altuvia et al. , 1994 ; Ivanova et al. , 1994 ; Becker-Hapak and Eisenstark , 1995 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9230301" target="_blank">9230301</a>]<br></li>
<li>If OxyR mediates the high expression of ahpC observed in cell cultures grown in the presence of L-ascorbate , other genes regulated by OxyR such as dps ( Altuvia et al. , 1994 ) must be activated under these conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17906139" target="_blank">17906139</a>]<br></li>
<li>Phase-variable expression of Ag43 is due to the concerted actions of Dam-methylase ( positive regulation ) and OxyR ( negative regulation ) ( Henderson et al. , 1999 ; Schembri and Klemm , 2001 ; Waldron et al. , 2002 ; Schembri et al. , 2003 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20345943" target="_blank">20345943</a>]<br></li>
<li>Depletion of GSH pools by MG , leading to a transient change in cytoplasmic redox potential , may be sufficient to explain the increased transcription of some of the genes under OxyR control , whereas direct covalent modiﬁcation of OxyR by MG seems less likely to be the mechanism ( Zheng et al. , 1998 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23646895" target="_blank">23646895</a>]<br></li>
<li>Genes that respond to OxyR should , therefore , comprise key defenses against H2O2 ; indeed , although Hpx − mutants can grow in aerobic medium , Hpx − oxyR mutants can not ( Park et al. , 2005 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21143317" target="_blank">21143317</a>]<br></li>
<li>Hpx − ΔoxyR mutants are not viable in aerobic medium ( Park et al. , 2005 ) , which precluded the most direct test of this notion ; however , the OxyR system is also induced in ahpCF mutants , due to modestly elevated levels of intracellular H2O2 ( Seaver & Imlay , 2001b ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21143317" target="_blank">21143317</a>]<br></li>
<li>In E coli , intracellular H2O2 stress activates the OxyR transcription factor , which then directly stimulates synthesis of approximately two dozen proteins ( Zheng et al. , 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19400769" target="_blank">19400769</a>]<br></li>
<li>Many of the proteins involved in protection against H2O2 stress are positively regulated by the OxyR transcription factor , which is fully activated by the H2O2 that accumulates in Hpx − mutants ( Zheng et al. , 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21338418" target="_blank">21338418</a>]<br></li>
<li>Another transcription factor , OxyR , induces expression of mntH during periods of H2O2 stress ( Kehres et al. , 2000a , Kehres et al. , 2002b ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19400769" target="_blank">19400769</a>]<br></li>
<li>The induction of mntH is especially pronounced during during H2O2 stress , since OxyR directly activates its transcription ( Kehres et al. , 2002 ; Anjem et al. , 2009 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21338418" target="_blank">21338418</a>]<br></li>
<li>Upon oxidation , OxyR becomes a transcriptional-activator for multiple genes involved in the protection against oxidative-stress ( Christman et al. , 1989 ; Zheng et al. , 1998 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10692164" target="_blank">10692164</a>]<br></li>
<li>Under conditions of oxidative-stress , OxyR undergoes reversible formation of a disulphide bond , allowing it to activate the transcription of genes required for oxidative-stress defence ( Zheng et al. , 1998 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11972787" target="_blank">11972787</a>]<br></li>
<li>Escherichia coli cells utilize two transcription factors OxyR and SoxR to sense the oxidants and then induce various genes against oxidative-stress that are involved in removing oxidants , repairing damaged cell components and maintaining reducing conditions in the cell ( Bauer et al. , 1999 ; Storz and Imlay , 1999 ; Pomposiello and Demple , 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12773378" target="_blank">12773378</a>]<br></li>
<li>Transcription of fur is under positive control of both OxyR and SoxRS ( Zheng et al. , 1999 ) , which in turn are activated by oxidative-stress .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14617150" target="_blank">14617150</a>]<br></li>
<li>It is possible that OxyR might be activated by nitrosative stress , as suggested by Kim et al. ( 2002 ) , but analysis of the yhjA mutant phenotype did not support this idea .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17464064" target="_blank">17464064</a>]<br></li>
<li>The Suf pathway has been suggested to function under conditions of oxidative and iron stress ( Lee et al. , 2003 ; 2004 ; Outten et al. , 2004 ) via OxyR and Fur , respectively , but the ﬁnding that IscR also controls the Suf pathway suggests that this alternate pathway is involved in Fe-S biogenesis under normal aerobic-growth conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16677314" target="_blank">16677314</a>]<br></li>
<li>By stimulating OxyR expression , Puu can cause an increase in the expression of katG ; as a consequence of this stimulatory effect , putrescine can increase cell survival under oxidative-stress ( Tkachenko et al. 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24952731" target="_blank">24952731</a>]<br></li>
<li>During normal growth as well as upon oxidative-stress , OxyR also acts to repress its own expression ( Christman et al. , 1989 ; Tao et al. , 1991 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8087856" target="_blank">8087856</a>]<br></li>
<li>Removal of OxyR , IHF , ORE-I , or ORE-II , did not abolish the ORE-III mediated oxidative induction of the sufA gene ( Lee et al. , 2004 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16824106" target="_blank">16824106</a>]<br></li>
<li>The sufABCDSE operon is directly regulated by OxyR , IHF , Fur and an unidentiﬁed oxidant-responsive ( ORE ) activator ( Lee and Roe , 1997 ; Lee et al. , 2003 ; 2004 ; Outten et al. , 2004 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16824106" target="_blank">16824106</a>]<br></li>
<li>For example , gene expression of sufABCDSE is induced by H2O2 ( Zheng et al. 2001 ) , and oxidant induction of the suf operon is regulated by OxyR , IHF , Fur and IscR ( Lee et al. 2008 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19919618" target="_blank">19919618</a>]<br></li>
<li>The SoxRS regulon ( Hidalgo et al. , 1995 ; Gaudu and Weiss , 1996 ; Scandalios , 1997 ) and OxyR regulon ( Halliwell and Gutteridge , 1989 ; Zheng et al. , 1998 ) spe-cifically respond to the superoxide anion and hydrogen peroxide , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15672380" target="_blank">15672380</a>]<br></li>
<li>The OxyR response to hydrogen peroxide ( H2O2 ) is known to be transient ( Zheng et al. , 1998 ; Aslund et al. , 1999 ; Carmel-Harel and Storz , 2000 ) and would ﬁt the kinetics observed here .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23646895" target="_blank">23646895</a>]<br></li>
<li>In a similar way , whereas most of the hydrogen peroxide-induced genes are regulated by OxyR , some genes are still induced by hydrogen peroxide in the absence of OxyR or SoxR , suggesting the presence of other regulator ( s ) responding to peroxide stress ( Zheng et al. , 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15009899" target="_blank">15009899</a>]<br></li>
<li>The OxyR protein acts as a transcriptional-regulator initiating , directly or indirectly , the synthesis of more than 40 gene products to counter the effects of peroxide stress , although transcription of some of these genes is also altered during exposure to heat-shock or redox-cycling agents ( Greenberg & Demple , 1989 ; Storz & Zheng , 2000 ; Walkup & Kogoma , 1989 ; Zheng et al. , 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17464064" target="_blank">17464064</a>]<br></li>
<li>Because the OxyR system is activated by sub-micromolar concentrations of intracellular H2O2 ( Aslund et al. , 1999 ) , we sought to test the role of Suf when cells were chronically exposed to low , physiological doses .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21143317" target="_blank">21143317</a>]<br></li>
<li>The OxyR stress response is activated by as little as 0.1 μM H2O2 ( Aslund et al. , 1999 ) , and we confirmed that an mntH ' - lacZ + fusion was strongly induced in the Hpx - mutant when it was transferred from anaerobic to aerobic media ( Fig. 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19400769" target="_blank">19400769</a>]<br></li>
<li>gulon is dedicated to the suppression of Fenton chemistry re R xy e O h T The OxyR regulon has been calibrated by evolution to be activated by about 0.1 μM intracellular H2O2 , suggesting that this dose is physiologically relevant ( Aslund et al. , 1999 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19400769" target="_blank">19400769</a>]<br></li>
<li>Members of the OxyR regulon are activated by the oxidized form of OxyR , which occurs by direct oxidation of the protein by H2O2 or by the cellular thiol-Ê disulphide status ( Aslund et al. , 1999 ; Storz and Imlay , 1999 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10692164" target="_blank">10692164</a>]<br></li>
<li>The OxyR regulon , activated by H2O2 , increases the level of protective enzymes such as hydroperoxidases , glutathione reductase Dps ( a small protein that protects DNA from oxi-dative damage ) and a small regulatory mRNA oxyS ( Altuvia et al. , 1994 ; Storz and Altuvia , 1994 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767577" target="_blank">9767577</a>]<br></li>
<li>Some of these proteins are involved in the response of bacteria to environmental adversity , such as excessive oxidation in the case of OxyR ( Christman et al. , 1989 ; Storz et al. , 1990 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8168494" target="_blank">8168494</a>]<br></li>
<li>OxyR behaves as a transcriptional autorepressor under both inducing and non-inducing conditions , but it activates different regulon promoters only after H2O2 treatment ( Storz et al. 1990 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11479716" target="_blank">11479716</a>]<br></li>
<li>OxyR usually exists in the reduced state under normal growth conditions ( Åslund et al. , 1999 ; Storz and Imlay , 1999 ; Carmel-Harel and Storz , 2000 ; Schembri and Klemm , 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11972787" target="_blank">11972787</a>]<br></li>
<li>Whereas OxyR responds primarily to H2O2 and nitrosylating agents , SoxR is known to respond primarily to superoxide and nitric oxide ( Nunoshiba et al. , 1993 ; Hausladen et al. , 1996 ; Storz and Imlay , 1999 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12773378" target="_blank">12773378</a>]<br></li>
<li>Diverse types of redox sensors have been characterized , such as SoxR , OxyR , Fnr and ArcB ( Spiro and Guest , 1990 ; Amabile-Cuevas and Demple , 1991 ; Zheng et al. , 1998 ; Georgellis et al. , 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23506073" target="_blank">23506073</a>]<br></li>
<li>It was previously shown that OxyR regulates suf in response to H2O2 and that OxyR binds to the suf promoter ( Zheng et al. , 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15101990" target="_blank">15101990</a>]<br></li>
<li>In agreement with this notion , the induction of the isc operon by H2O2 is dependent on IscR ( Outten et al. , 2004 ) , and is independent of OxyR , unlike the induction of the suf operon ( Zheng et al. , 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16677314" target="_blank">16677314</a>]<br></li>
<li>However , induction is independent of OxyR and SoxRS ( Zheng et al. 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12382038" target="_blank">12382038</a>]<br></li>
<li>In E. coli this regulon is controlled by the OxyR protein , which responds to as little as 0.2 micromolar intracellular H2O2 ( CarmelHarel & Storz , 2000 , Zheng et al. , 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21143317" target="_blank">21143317</a>]<br></li>
<li>The observation that suf is controlled by the OxyR ( Zheng et al. , 2001 , Lee et al. , 2004 ) protein prompted us to examine whether the Suf system becomes important when cells are exposed to H2O2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21143317" target="_blank">21143317</a>]<br></li>
<li>The global iron regulator Fur is induced by OxyR in response to H2O2 ( Zheng et al. , 1999 ) , which may lead to the repression of iron uptake genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15101990" target="_blank">15101990</a>]<br></li>
<li>The b-galactosidase speciﬁc activities reported are cysteine of OxyR ( Zheng et al. , 1998 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16804186" target="_blank">16804186</a>]<br></li>
<li>The Fur protein is not only auto-regulated and re-gulated by the cAMP-CAP catabolite repression system ( De Lorenzo et al. , 1988 ) , but also by OxyR and SoxRS ( Zheng et al. , 1999 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11931550" target="_blank">11931550</a>]<br></li>
<li>Many of the genes implicated in the defence against ROS are inducible and belong to regulons that enable bacteria to cope with various stresses : the SoxRS regulon , including sodA and nfo , which responds to redox-cycling drugs ; the OxyR regulon , including katG and dps , which is induced in actively growing cells in response to H,O,-mediated oxidation ; the RpoS ( aS ) regulon , including also katG METHODS and dps , which is induced when bacteria enter stationary Bacterial strains .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10439394" target="_blank">10439394</a>]<br></li>
<li>origin yhjA FNR/OxyR yhjA OxyR Exposure of the yhjA mutant to nitrosative stress reagents ( sodium nitroprusside , S-nitrosoglutathione , 3 - [ 2-hydroxy-1 - ( 1-methylethyl ) -2 - nitrosohydrazino ] -1 - propanamine , 3 - [ 2-hydroxy-1-methyl-2-nitrosohydrazino ] - N-methyl-1-propanamine ) and the antimicrobial compound sodium hypochlorite in both disc diffusion and killing curve assays indicated no increased sensitivity compared to the parent strain , suggesting that yhjA has a speciﬁc role in the peroxide stress response ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17464064" target="_blank">17464064</a>]<br></li>
<li>x Induction of the sufA operon encoding Fe-S assembly proteins by superoxide generators and hydrogen peroxide : involvement of OxyR , IHF and an unidentiﬁed o xidant-responsive factor Joon-Hee Lee , Won-Sik Yeo and Jung-Hye Roe * Laboratory of Molecular Microbiology , School of Biological Sciences , and Institute of Microbiology , Seoul National University , Seoul 151-742 , Korea .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15009899" target="_blank">15009899</a>]<br></li>
<li>JOURNAL OF BACTERIOLOGY , Sept. 1997 , p. 5967 -- / 97 / $ 04.0010 Copyright © 1997 , American Society for Microbiology oxyR-Dependent Induction of Escherichia coli grx Gene Expression by Peroxide Stress KAZUYUKI TAO * Radioisotope Center , University of Tokyo , Bunkyo-ku , Tokyo 113 , Japan Received 28 April 1997/Accepted 18 July 1997 The Escherichia coli OxyR protein is a transcriptional-activator for a number of genes induced in response to low concentrations of hydrogen peroxide .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9294462" target="_blank">9294462</a>]<br></li>
<li>109 , 262-266 ( 1991 ) Purification and Characterization of the Escherichia coli OxyR Protein , the Positive Regulator for a Hydrogen Peroxide-Inducible Regulon1 Kazuyuki Tao , * 2 Kozo Makino , '' Shuji Yonei , * Atsuo Nakata , '' and Hideo Shinagawa ** J ` Laboratory of Radiation Biology , Faculty of Science , Kyoto University , Sakyo-ku , Kyoto 606 ; and `` Department of Experimental Chemotherapy , Research Institute for Microbial Diseases , Osaka University , Siuta , Osaka 565 Received for publication , September 18 , 1990 The Escherichia coli oxyR gene is required for the induction of a regulon that is inducible by hydrogen peroxide and confers resistance to oxidative-stresses .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1864839" target="_blank">1864839</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_134">
Tao , K , , Makino , K , , Yonei , S. , Nakata , A. , and Shinagawa , H. -LCB- 1991 ) Purification and characterization of the Eseherichia coli OxyR protein , the positive regulator for a hydrogen peroxide-inducible regulon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7984106" target="_blank">7984106</a>]<br></label>
<input id="act_similarity_134", type="checkbox">
<div class="ACT_similar">
<li>Tao , K. , Makino , K. , Yonei , S. , Nakata , A. , and Shinagawa , H. ( 1991 ) Purification and characterization of the Escherichia coii OxyR protein , the positive reguiator for a hydrogen peroxide-inducible regulon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8483417" target="_blank">8483417</a>]<br></li></div>
<li>Considering that sufA is induced by almost all oxidants we tested , and that OxyR is rather limited to peroxide response , it could be hypothesized that the ORE-III binding factor would respond to both superoxide and peroxide .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15009899" target="_blank">15009899</a>]<br></li>
<li>Thus the mntH response to peroxide is almost completely mediated by OxyR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12029030" target="_blank">12029030</a>]<br></li>
<li>OxyR and IscR contributed almost equally to activate the sufA operon in response to oxidants .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16824106" target="_blank">16824106</a>]<br></li>
<li>The other two IscR binding regions have not yet been tested for their role in suf regulation in vivo ; only weak binding of region 3 was observed by high IscR concentrations in vitro , and integration host factor ( IHF ) , which activates suf in conjunction with OxyR in response to peroxide stress , exhibits stronger binding afﬁnity than does IscR for region 2 ( 11 , 12 , 14 , 15 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25266384" target="_blank">25266384</a>]<br></li>
<li>The regulation of yhjA expression is interesting , because transcription is dependent on two transcription factors , FNR and OxyR , that respond to two seemingly incompatible signals , i.e. oxygen starvation and peroxide stress .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17464064" target="_blank">17464064</a>]<br></li>
<li>These include SoxR , OxyR , FNR , MetR , and Fur ( 5 , 6 , 10 , 12 , 17 ) , though in each case the principal function of the regulator is to sense another signal ( superoxide , hydrogen peroxide , oxygen , homocysteine , and iron , respectively ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16428390" target="_blank">16428390</a>]<br></li>
<li>Consistent with a biological role for increased levels of OxyR during exponential-growth , the crp cya strain had lower activities of catalase hydroperoxidase I and glutathione reductase and an increased sensitivity to exogenously added hydrogen peroxide .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9324269" target="_blank">9324269</a>]<br></li>
<li>First , INH might act like a peroxide and activate OxyR , thereby leading to increased levels of expression of the protective hydroperoxidase I and alkyl hydroperoxide reductase activities .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7986015" target="_blank">7986015</a>]<br></li>
<li>To directly compare the induction of OxyR and SoxR target genes by both hydrogen peroxide and superoxide-gen-erating compounds , we treated wild-type cells with either 0 , 0.01 , 0.03 , 0.1 , 0.3 , and 1 mM hydrogen peroxide or the same concentrations of paraquat ( methyl viologen ) , a standard inducer of the soxRS regulon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11443091" target="_blank">11443091</a>]<br></li>
<li>As expected , oxyS was strongly induced by all concentrations of hydrogen peroxide and soxS was induced by all concentrations of paraquat , consistent with the notion that OxyR primarily senses hydrogen peroxide while SoxR primarily responds to superoxide-generating compounds .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11443091" target="_blank">11443091</a>]<br></li>
<li>Specifically , the sodA , fumC , zwf , acnA , acrAB , and soxS genes from the SoxRS regulon and the katG and ahpC genes from OxyR regulon , which respond to the superoxide anion and hydrogen peroxide , were studied .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15672380" target="_blank">15672380</a>]<br></li>
<li>For example , the isc operon , which speciﬁes Fe-S cluster formation and repair activities , was induced by hydrogen peroxide in strains lacking either OxyR or the superoxide response-regulators SoxRS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11443091" target="_blank">11443091</a>]<br></li>
<li>For example , by examining the hydrogen peroxide response in strains lacking the OxyS small RNA regulator , we may be able to differentiate those genes regulated directly by OxyR and those regulated indirectly through OxyS RNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11443091" target="_blank">11443091</a>]<br></li>
<li>By measuring gene expression in isogenic wild-type and oxyR deletion strains , we conﬁrmed that the peroxide response-regulator OxyR activates most of the highly hydrogen peroxide-inducible genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11443091" target="_blank">11443091</a>]<br></li>
<li>Control of ahpC expression by the OxyR global regulator in response to L-ascorbate concentration is consistent with the formation of hydrogen peroxide under our experimental conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17906139" target="_blank">17906139</a>]<br></li>
<li>Since expression of glutathione reductase is also regulated by OxyR , two components in the glutathione-depen-dent hydrogen donor system are induced by peroxide stress in an oxyR-dependent fashion .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9294462" target="_blank">9294462</a>]<br></li>
<li>To determine whether the increased expression of oxyR-regulated genes was associated with an increase in OxyR , we treated exponentially growing cells with hydrogen peroxide and examined the amount of OxyR and rate of OxyR synthesis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2183352" target="_blank">2183352</a>]<br></li>
<li>During exponential-growth , the expression of defence genes is regulated by the redox-sensitive OxyR protein , and the genes are induced only after cells are treated with hydrogen peroxide .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7984106" target="_blank">7984106</a>]<br></li>
<li>Our results show that OxyR , a transcriptional-activator of genes induced by hydrogen peroxide , is activated directly by oxidation and is therefore both the sensor and transducer of an oxidative-stress signal .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2183352" target="_blank">2183352</a>]<br></li>
<li>Upon oxidation , OxyR activates the transcription of these genes and is therefore both the sensor and transducer of the hydrogen peroxide stress signal during exponential-growth .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7984106" target="_blank">7984106</a>]<br></li>
<li>For example , the Escherichia coli OxyR protein is the direct link between hydrogen peroxide stress and the activation of defense genes such as catalase , i.e. OxyR is both a sensor and a transcriptional-activator ( for a review , see Hidalgo and Demple , 1997a ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9564045" target="_blank">9564045</a>]<br></li>
<li>Deletion strains lacking OxyR , the transcriptional-activator of the regulon of hydrogen peroxide-inducible genes , were the most susceptible .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7986015" target="_blank">7986015</a>]<br></li>
<li>Overlap of the OxyR and SoxS binding sites suggests the possibility of interference between these two proteins under oxidative-stress conditions , particularly in the presence of superoxide which would be dismutated to hydrogen peroxide in the cell .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11741843" target="_blank">11741843</a>]<br></li>
<li>At present there are no data avialable pertaining to the question whether hydrogen peroxide might not , for example , convert OxyR to an inducer for mom in the absence of C ; mom expression has always been assayed under conditions where oxidative-stress was not applied .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2551682" target="_blank">2551682</a>]<br></li>
<li>The increased induction of yodA in the oxyR mutant cells may be explained by the fact that many oxidative-stress defence genes , such as katG , require OxyR for their expression , and hydrogen peroxide and other reactive oxygen species are thus likely to accumulate in this mutant .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12480884" target="_blank">12480884</a>]<br></li>
<li>If the expression level of the trxC9-9lacZ fusion is taken as a measure of the amount of activated OxyR in the cell , then the OxyR A221E mutation should make the cells completely resist-ant to effects of oxidative-stress by hydrogen peroxide .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10644706" target="_blank">10644706</a>]<br></li>
<li>In contrast , suf is regulated by OxyR and Fur , which are optimized to sense hydrogen peroxide stress and iron starvation , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15101990" target="_blank">15101990</a>]<br></li>
<li>Since the toxicities of both hydrogen peroxide and superoxide are exacerbated by iron , Fur induction by both OxyR and SoxRS reﬂects the chemistry of oxidative-stress .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10419964" target="_blank">10419964</a>]<br></li>
<li>Responses of OxyR Regulon Genes Similarly , time course studies were also performed for stress genes in OxyR Regulon , katG and ahpC , in response to various concentrations of paraquat ( 10 -- 500 AM ) and hydrogen peroxide ( 0.5 -- 10 mM ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15672380" target="_blank">15672380</a>]<br></li>
<li>High levels of hydrogen peroxide cause oxidative-stress for the bacteria and induce a response mediated by the regulatory protein OxyR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10644706" target="_blank">10644706</a>]<br></li>
<li>The MntH importer is also induced as an essential part of the OxyR response during periods of hydrogen peroxide ( H2O2 ) stress [ 5,11,12 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25774656" target="_blank">25774656</a>]<br></li>
<li>The stress probes within the SoxRS regulon responded to paraquat treatment ; the stress probes belonging to the OxyR response responded to both hydrogen peroxide and paraquat treatment ( Figs. 2 , 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15672380" target="_blank">15672380</a>]<br></li>
<li>The induction of OxyR regulon genes to superoxide stress is explained by the production of hydrogen peroxide from superoxide anion by MnSOD , which is induced by paraquat .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15672380" target="_blank">15672380</a>]<br></li>
<li>The katG gene has been shown to be part of the OxyR regulon , which responds to oxidative-stress , including the presence of hydrogen peroxide ( 2 , 25 ) as part of a broader response illicited by redox-cycling reagents ( 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2254248" target="_blank">2254248</a>]<br></li>
<li>The observed OxyR-mediated oxidative-stress response is consist-ent with previous reports that show that fumarate reductase produces superoxide in the presence of O , and that sufficient 2 superoxide dismutase is present in anaerobic cells to produce hydrogen peroxide and thus activate OxyR ( 43 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16857675" target="_blank">16857675</a>]<br></li>
<li>Under aerobic conditions , OxyR can be activated in vitro by oxidation or nitrosylation of a critical regulatory Cys thiol ( 15 ) , but S-oxidation ( which is mediated by hydrogen peroxide ) would be inconsistent with anaerobiosis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22539721" target="_blank">22539721</a>]<br></li>
<li>We have prevented OxyR activation on removal of the 100 mM DTT by the addition of catalase ( which breaks down hydrogen peroxide ) , suggesting that concentrations of hydrogen peroxide capable of activating OxyR are present under the aerobic conditions ( 15 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2183352" target="_blank">2183352</a>]<br></li>
<li>It is often under conditions of high aeration that bacteria are exposed to hydrogen peroxide ( Seaver & Imlay , 2004 ) , leading to activation of OxyR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17464064" target="_blank">17464064</a>]<br></li>
<li>This suggestion was based upon observations that HCP is oxidized by hydrogen peroxide , that an hcp mutant is more sensitive to hydrogen peroxide than its parent , and that hcp transcription is regulated by OxyR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17449618" target="_blank">17449618</a>]<br></li>
<li>The OxyR transcription factor is central to the adaptive response to hydrogen peroxide and oxyR deficient mutants are highly sensitive to this oxidant .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10760142" target="_blank">10760142</a>]<br></li>
<li>Similarly , the OxyR transcription factor , discovered as the primary regulator of the response to hydrogen peroxide ( H O ) , later was reported to activate target genes on 2 2 exposure to S-nitrosylated cysteine ( 27 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14718666" target="_blank">14718666</a>]<br></li>
<li>( ii ) The wild-type OxyR protein may be modified in such a way that it can no longer be oxidized or no longer activate transcription , ( iii ) Anti-oxidant activities expressed in stationary phase , such as HPII catalase , may detoxify the hydrogen peroxide so effectively that concentrations of hydrogen peroxide sufficient to activate OxyR are not reached .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7984106" target="_blank">7984106</a>]<br></li>
<li>Although we purified OxyR protein from cells which had not been treated with hydrogen peroxide , our OxyR protein activated the transcription initiation of katG in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1864839" target="_blank">1864839</a>]<br></li>
<li>If cells are treated with hydrogen peroxide during exponential-growth , dps transcription is induced by OxyR , which activates cĵ ° - RNA polymerase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7984106" target="_blank">7984106</a>]<br></li>
<li>When 10 bp were inserted between contact points 2 and 3 in the ahpC-bindingsite ( A1A2XA ~ ) , reduced instead of oxidized OxyR bound preferentially in vitro , and ahpC was no longer induced after exposure to hydrogen peroxide in vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8087856" target="_blank">8087856</a>]<br></li>
<li>Paraquat , a superoxide-generating compound , indirectly activates the conversion to oxidized OxyR by generating hydrogen peroxide at low levels ( 40 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12644490" target="_blank">12644490</a>]<br></li>
<li>In vivo , hydrogen peroxide is required to activate and oxidize OxyR , but OxyR purified aerobically ( i.e. , in the absence of reductants ) is predominantly in the oxidized form .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8087856" target="_blank">8087856</a>]<br></li>
<li>For reasons that are not understood , wild-type OxyR displayed an uninducible or reduced phenotype on the MacConkey-galactose medium , even in the presence of hydrogen peroxide .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7868602" target="_blank">7868602</a>]<br></li>
<li>The results obtained therefore suggest that the OxyR protein can bind DNA with reasonable affinity and specificity even in the absence of hydrogen peroxide treatment .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2693740" target="_blank">2693740</a>]<br></li>
<li>Speciﬁcally , in the presence of hydrogen peroxide , OxyR becomes oxidized , which results in a change in its structure , which in turn contributes to a change in protein-DNA contacts .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12029051" target="_blank">12029051</a>]<br></li>
<li>Since the mutations in hps genes conferred a hydrogen perox-ide-sensitive phenotype and since some of the hps : : lacZ fusions appeared to be activated by OxyR , we tested whether these hps : : lacZ fusions were hydrogen peroxide inducible ( i.e. , exhibited increased expression in presence of 60 mM hydrogen peroxide ) in accordance with the known function of OxyR protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8990283" target="_blank">8990283</a>]<br></li>
<li>In this study we placed a lacZ operon fusion into the katE gene and transduced the fusion into various genetic backgrounds to determine whether its expression is subject to regulation by katF , OxyR , a hydrogen peroxide-inducible regulatory protein ( 3 ) , or both katF and OxyR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3045091" target="_blank">3045091</a>]<br></li>
<li>These results confirm that only HPI is subject to regulation by hydrogen peroxide and is dependent on the OxyR protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3045091" target="_blank">3045091</a>]<br></li>
<li>Hydrogen peroxide induces the synthesis of 30 proteins in S. typhimurium , of which 9 are regulated by OxyR ( 10 , 28 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8990283" target="_blank">8990283</a>]<br></li>
<li>It has been demonstrated that it is the OxyR protein itself which is sensitive to hydrogen peroxide , activating the protein through the reversible formation of a disulﬁde bond ( 3 , 28 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9922271" target="_blank">9922271</a>]<br></li>
<li>Hydrogen peroxide directly activates OxyR through the formation of an intramolecular disul-ﬁde bond ( 22 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11443092" target="_blank">11443092</a>]<br></li>
<li>We found that dps mRNA leveis are controiied by rpoS-encoded a ^ , the transcriptionai activator OxyR and the histone-iike iHF protein , in exponentiaily growing ceiis , dps is induced by treatment with hydrogen peroxide in an OxyR-dependent manner .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7984106" target="_blank">7984106</a>]<br></li>
<li>From the array data , we assumed that the annotated ybjM gene was induced by OxyR ; however , primer extension and Northern experiments showed that hydrogen peroxide treatment actually leads to the induction of a transcript on the opposite strand .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11443091" target="_blank">11443091</a>]<br></li>
<li>A transcript encoding Fur was induced by hydrogen peroxide in a wild-type strain but not in a DoxyR strain , and DNase I footprinting assays showed that OxyR binds to the fur promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10419964" target="_blank">10419964</a>]<br></li>
<li>Oxidized OxyR binds directly to the trxC promoter and induces its expression in response to elevated hydrogen peroxide levels or the disruption of one or several of the cytoplasmic redox pathways .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10644706" target="_blank">10644706</a>]<br></li>
<li>OxyR , which senses elevated levels of hydrogen peroxide , binds to the fur promoter and induces the expression of a transcript encoding Fur .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10419964" target="_blank">10419964</a>]<br></li>
<li>These results suggest that increased expression of the oxyR regulon after treatment with hydrogen peroxide is not a result of increased levels or synthesis of OxyR , but is rather a consequence of a modification of preexisting OxyR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2183352" target="_blank">2183352</a>]<br></li>
<li>This indication of a role for OxyR in trxC expression is further supported by the finding that hydrogen peroxide increases the amount of thioredoxin 2 in cells 20-fold .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10644706" target="_blank">10644706</a>]<br></li>
<li>Subsequent experiments conﬁrmed OxyR binding to this region and showed that fur expression is induced by OxyR after hydrogen peroxide treatment .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10419964" target="_blank">10419964</a>]<br></li>
<li>As shown in Fig. 1B , hemH induction by hydrogen peroxide in vivo was clearly dependent on OxyR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11443091" target="_blank">11443091</a>]<br></li>
<li>Expression of the catalase KatG , known to be regulated by OxyR , was enhanced three times when 3 mM hydrogen peroxide was treated ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23506073" target="_blank">23506073</a>]<br></li>
<li>Levels of Grx4 did not change after treatment with hydrogen peroxide , or in null mutants for catalases or OxyR , suggesting that grxD is not part of the OxyR regulon ( as grxA is ( 51 ) ) , and is not involved in responses against hydrogen peroxide .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15833738" target="_blank">15833738</a>]<br></li>
<li>These OxyR molecules are poised to be activated by low amounts of hydrogen peroxide which presumably enter the cell by diffusion .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2183352" target="_blank">2183352</a>]<br></li>
<li>OxyR is activated at specific concentrations of hydrogen peroxide in exponentially growing cells .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7984106" target="_blank">7984106</a>]<br></li>
<li>However , OxyR is unable to activate the A1A2XA3A4mutant promoter after treatment with hydrogen peroxide .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8087856" target="_blank">8087856</a>]<br></li>
<li>Whereas the synthesis of OxyR was unchanged in untreated cells , cells treated with hydrogen peroxide showed a decrease rather than an increase in OxyR synthesis during the first 10 minutes after treatment with hydrogen peroxide .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2183352" target="_blank">2183352</a>]<br></li>
<li>These immunoblots showed that there is no significant increase in the amount of OxyR after the treatment with hydrogen peroxide .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2183352" target="_blank">2183352</a>]<br></li>
<li>No increase in amount or synthesis of OxyR after treatment with hydrogen peroxide .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2183352" target="_blank">2183352</a>]<br></li>
<li>Effect of hydrogen peroxide on the amount and synthesis of OxyR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2183352" target="_blank">2183352</a>]<br></li>
<li>DISCUSSION We have shown here ( i ) that E. coli maintains the intracellular concentration of H2O2 within a surprisingly narrow range ( ; 0.2 mM ) although the metabolic generation of H2O2 varies .10 - fold during growth ; ( ii ) that a signiﬁcant amount of this homeostatic control during exponential-growth seems to be accomplished by the transcriptional regulation of the katG-encoded HP-I ; ( iii ) that the variation of HP-I catalase levels reﬂects a continuous variation in the degree of OxyR activation ; ( iv ) that the protective role of OxyR activation is a pop-ulation-wide phenomenon , rather than one restricted to occasional cells undergoing more intense oxidative-stress ; ( v ) that the physiological trigger point for OxyR seems to lie at ; 0.2 mM , the level above which katG up-regulation is observed ; and ( vi ) that ahpFC plays an antimutator role but does not regulate H2O2 levels .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8990289" target="_blank">8990289</a>]<br></li>
<li>Interestingly , neither SoxRS nor OxyR , which have been persistently proposed to be critical NO stress response sensorregulators ( 49 , 50 ) appear to be involved in the regulation of either hmp or norVW in E. coli ( Fig. 5 ) ( 48 ) .2 Future investigations will aim to further clarify the diverse roles and mechanisms of NO defense genes , enzymes , and regulators in microbial adaptations to NO in vitro and in various models of infection .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12529359" target="_blank">12529359</a>]<br></li>
<li>In our MS analysis , we found only five OxyR-regulated proteins ( supplemental Table S1 ) : all three catalases of E. coli ( AhpF , KatG , and KatE ) ; OxyR itself , which is down-regulated by oxi-dative stress ; and Fur , which is up-regulated by OxyR but is also a usual contaminant during His tag purification ( 32 , 33 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23139412" target="_blank">23139412</a>]<br></li>
<li>Phase Variation of Ag43 Is Independent of the Oxidation State of OxyR nu Wallecha , Jason Correnti , Vincent Munster , and Marjan van der Woude * A Department of Microbiology , University of Pennsylvania , Philadelphia , Pennsylvania 19104 Received 24 October 2002/Accepted 14 January 2003 OxyR is a DNA binding protein that differentially regulates a cell 's response to hydrogen peroxide-mediated oxidative-stress .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12644490" target="_blank">12644490</a>]<br></li>
<li>Therefore , the lower level of transcription in the oxyR background seems to be caused by the absence of functional OxyR and not by the increase in oxidative-stress .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10692164" target="_blank">10692164</a>]<br></li>
<li>Firstly , the absence of a change in the level of agn - lacZ transcription and in the percentage of ON cells after a transient oxidative-stress indicates that a change in the oxidation state of OxyR does not result in an alleviation of repression and methylation of the GATC sequences ( Table 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12644490" target="_blank">12644490</a>]<br></li>
<li>However , DNA that is resistant to cleavage by DpnI must be non-methylated at all three GATC sequences and , in DNA that is resistant to cleavage by MboI , all three sequences must be OxyR ( C199S ) can repress agn transcription in vivo In the absence of oxidative-stress , the majority of the cellular OxyR protein is in the reduced state .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10692164" target="_blank">10692164</a>]<br></li>
<li>A similarly high DNA binding specificity was observed for the reduced ( transcriptionallyinactive ) form of OxyR , consistent with a model in which the OxyR protein is bound toits recognition sequenceseven in the absence of an oxidative-stress .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1730735" target="_blank">1730735</a>]<br></li>
<li>Our in vitro analyses with puriﬁed , oxidized OxyR directly support the conclusion that the repression of agn43 transcription , which occurs under oxidative-stress , is a direct result of binding of oxidized OxyR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12644490" target="_blank">12644490</a>]<br></li>
<li>Since these genes are immediately activated by oxidized OxyR upon oxi-dative stress , this indicates that binding of oxidized OxyR to agn43 DNA will occur in a cell under oxidative-stress as long as this site is unmethylated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12644490" target="_blank">12644490</a>]<br></li>
<li>OxyR-dependent regulation of Ag43 expression is therefore unusual in ﬁrstly that OxyR binding at agn43 is dependent on the methylation state of Dam target sequences in its binding site and secondly that OxyR-dependent repression appears to be independent of hydrogen-peroxide mediated oxidative-stress and the oxidation state of OxyR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12644490" target="_blank">12644490</a>]<br></li>
<li>However , differences between binding of OxyR and the mechanism of repression of mom and agn43 make it difﬁcult to extrapolate our results to the role of oxidative-stress in regulation of mom .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12644490" target="_blank">12644490</a>]<br></li>
<li>This is consistent with methylation protection at agn43 by bound OxyR in this strain , which suggests that OxyR can be bound to the agn43 regulatory region in vivo under persistent oxidative-stress .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12644490" target="_blank">12644490</a>]<br></li>
<li>This suggests that , under peroxide-mediated stress , when oxidized OxyR is known to-bind to the katG and oxyRS regions and activate transcription , it can also be bound to the unmethylated agn43 regulatory region and thus repress agn43 transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12644490" target="_blank">12644490</a>]<br></li>
<li>In the presence of hydrogen peroxide , reduced cytosolic OxyR is oxidized , and this latter form activates the transcription of genes required for oxidative-stress protection ( reviewed by Storz and Imlay , 1999 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10692164" target="_blank">10692164</a>]<br></li>
<li>Purification of the OxyR protein showed that oxidized but not reduced OxyR activates transcription of oxidative-stress-inducible genes in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2183352" target="_blank">2183352</a>]<br></li>
<li>Taken together , the lack of an effect of peroxide-mediated oxidative-stress on Ag43 expression indicates that , in cells with OxyR mainly or completely in the oxidized form , agn43 transcription is repressed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12644490" target="_blank">12644490</a>]<br></li>
<li>This repression in vitro is consistent with the interpretation that in vivo under hydrogen peroxide-induced oxidative-stress agn43 repression is mediated by oxidized OxyR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12644490" target="_blank">12644490</a>]<br></li>
<li>HPI is an adaptive enzyme regulated by OxyR , a major regulatory transcription factor responding to oxidative-stress w5 ,6 x. HPII is expressed maximally in stationary phase , and its expression is completely dependent on the product of katF w2 ,7 x. Although katF was first identified as a positive regulator of HPII w2 ,7 x , it has been found to control the expression of many stationary phase specific and osmotically regulated genes w8x .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9199247" target="_blank">9199247</a>]<br></li>
<li>A wide range of environmental stimuli affect the expression of the sRNAs including oxidative-stress ( OxyR activates OxyS ) ; cell surface stress ( RcsB activates RprA ) ; glycine status of the cell ( GcvA activates GcvB ) ; osmotic imbalance ( OmpR activates OmrA/B ) ; quality of carbon source/growth rate [ McaS ; transcription factor ( s ) unknown ] ; low temperature [ DsrA ; transcription factor ( s ) unknown ] and O tension ( ArcA-ArcB 2 represses ArzA under anaerobic conditions ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22250746" target="_blank">22250746</a>]<br></li>
<li>The gorA gene does not show strong OxyR regulation ( 7 ) , but dps , trxC , and hemH ar clearly induced by OxyR in response to oxidative-stress ( 9 , 17 , 24 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11443092" target="_blank">11443092</a>]<br></li>
<li>Since OxyR mediates a response to oxidative-stress in E. coli , we ﬁrst evaluated the possibility of oxygen dependence of its growth-phase-depen-dent regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9324269" target="_blank">9324269</a>]<br></li>
<li>Expression of Dps is known to be regulated by OxyR , in response to oxidative-stress ( 2 ) , and three nucleoid proteins , IHF , Fis , and H-NS , in response to growth-phase ( 2 , 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21239586" target="_blank">21239586</a>]<br></li>
<li>The H2O2 that is released by PEA catabolism is known to cause oxidative-stress , activating the OxyR regulon and so stimulating the synthesis of enzymes that remove H2O2 ( 44 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24013633" target="_blank">24013633</a>]<br></li>
<li>Thus the Suf system is critical for growth during H2O2 stress , and OxyR must induce it to achieve sufficient levels .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21143317" target="_blank">21143317</a>]<br></li>
<li>These results suggested that the ahpCF and the katG ahpCF mutant strains are in a constant state of oxidative-stress and that OxyR is constitutively activated because the peroxides produced by normal aerobic metabolism are not dissipated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7986015" target="_blank">7986015</a>]<br></li>
<li>Therefore , OxyR can oxyR momR act to repress and expression during normal growth and activate katG , ahpC , dps , and gorAupon oxidative-stress ( Figure 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8087856" target="_blank">8087856</a>]<br></li>
<li>The two binding modes probably allow OxyR to repress the oxyR and mom promoters during normal growth , while activating katG , ahpCF , dps , gorA , and oxyS only upon oxidative-stress .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7868602" target="_blank">7868602</a>]<br></li>
<li>The shift in binding also allows OxyR to repress its own expression under both oxidizing and reducing conditions , while only activating katG , ahpC , gorA , and dps upon oxidative-stress .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8087856" target="_blank">8087856</a>]<br></li>
<li>Thus , the profiles of `` metabolic '' transcripts and the lack of induction of OxyR - and SoxRS-regulated transcripts ( 37 ) as aerobiosis increased ( Table 2 ) suggest that remodeling metab-olism is sufficient to manage endogenous oxidative-stress under the conditions investigated here .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21252224" target="_blank">21252224</a>]<br></li>
<li>The antioxidant adaptive response to sublethal UVA irradiation may involve the induction of oxidative defens pathways that include OxyR , Fur and SoxRS , leading to coordinated gene expression of antioxidant enzymes including HPI , HPI , FeSOD , MnSOD and GR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16183297" target="_blank">16183297</a>]<br></li>
<li>Regulation of the isc operon is mediated by IscR , whereas the suf operon requires OxyR and IHF for the response to oxidative-stress and Fur for induction by iron starvation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15101990" target="_blank">15101990</a>]<br></li>
<li>Therefore , we conclude that the oxidant induction of sufA is independent of regulation by Fur , and there must be another oxidant-responsive factor in addition to OxyR that induces the suf operon under oxidative-stress conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15009899" target="_blank">15009899</a>]<br></li>
<li>Transcription of fhuF is derepressed by a low iron level in the cell via the iron regulator Fur and repressed by OxyR , an oxidative response-regulator ( 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14756576" target="_blank">14756576</a>]<br></li>
<li>One target gene , identiﬁed using this approach and experimentally conﬁrmed to be regulated by OxyR , encodes the iron metab-olism regulator Fur , demonstrating coordinate regulation between oxidative-stress response and iron metabolism ( 23 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11443092" target="_blank">11443092</a>]<br></li>
<li>Keywords : oxidative-stress response ; SoxRS regulon ; OxyR regulon ; green fluorescent protein ; quantitative and kinetic monitoring INTRODUCTION Oxidative stresses are generated through a variety of mechanisms including normal aerobic metabolism , exposure to ultraviolet light , metal ions , and oxidative compounds , or released from macrophages during infection ( Halliwell and Gutteridge , 1989 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15672380" target="_blank">15672380</a>]<br></li>
<li>Since some of the genes participating in stress response , including katG , dps , gadB and gorA , are regulated by both ss and OxyR , it was suggested that repression of rpoS by OxyS may prevent redundant utilization of transcriptional-regulators [ 14 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21106118" target="_blank">21106118</a>]<br></li>
<li>To elucidate the mechanisms by which bacterial cells sense oxidative-stress and then induce a defense response , we purified the OxyR protein to homogeneity and studied its ability to-bind oxyR-regulated promoters and to activate transcription in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2183352" target="_blank">2183352</a>]<br></li>
<li>Thus , protein S-nitrosylation during ARN generates a nitrosative stress that is ameliorated by OxyR , and OxyR ( and possibly other genes in its regulon ) appears to protect against nitrosative stress at least in part by augmenting protein denitrosylation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22539721" target="_blank">22539721</a>]<br></li>
<li>Expression of katG following H202 induction ( 21 , 34 ) is positively regulated by OxyR ( 8 , 30 ) , a protein responsible for the positive regulation of a regulon involved in protection against oxidative-stress .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3045098" target="_blank">3045098</a>]<br></li>
<li>OxyR , the regulatory protein of the oxidative-stress regulon , is bound specifically to the DNA of its target genes even in the absence of the covalent modification required for the activation of transcription ( 26 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7901196" target="_blank">7901196</a>]<br></li>
<li>We are using the cloned oxyR gene to overproduce OxyR and to allow purification of the protein to determine the exact manner of OxyR activation by oxidative-stress .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2471187" target="_blank">2471187</a>]<br></li>
<li>To further characterize OxyR activation after oxidative-stress , we purified the OxyR protein to homogeneity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2183352" target="_blank">2183352</a>]<br></li>
<li>Fortuitously , we learned that MomR is identical to OxyR , a regulatory protein responding to oxidative-stress .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2551682" target="_blank">2551682</a>]<br></li>
<li>OxyR is normally involved in protection against oxidative-stress as it activates a regulon of peroxide-inducible genes ( 21 , 22 ) , but it also represses the ﬂu gene ( 12 , 15 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10648536" target="_blank">10648536</a>]<br></li>
<li>For SoxR and OxyR , the corresponding gene products decrease the extent of intracellular oxidative-stress ( e.g. catalase , superoxide dismutase ) and repair damaged macromolecules ( 15 , 38 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22223481" target="_blank">22223481</a>]<br></li>
<li>In the presence of H2O2 , OxyR is oxidized ( ox ) and induces the expression of oxidative-stress response genes like catalases .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23139412" target="_blank">23139412</a>]<br></li>
<li>One simple explanation for this finding is that , due to the absence of thioredoxins , the equilibrium of reduced and oxidized OxyR is shifted more toward the oxidized ( i.e. active ) , form and therefore the oxidative-stress response including the expression of catalase and alkyl hydroperoxidase is partially activated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10644706" target="_blank">10644706</a>]<br></li>
<li>In summary , these data demonstrate that ychF is negatively regulated by OxyR in response to oxidative-stress .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23139412" target="_blank">23139412</a>]<br></li>
<li>This demonstrates that IscR positively contributes to induce the sufA gene as much as OxyR does in response to oxidative-stress .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16824106" target="_blank">16824106</a>]<br></li>
<li>This gene is regulated by the oxidative-stress sensor OxyR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17906139" target="_blank">17906139</a>]<br></li>
<li>Additionally , these genes are regulated by transcription factors speciﬁc for certain stress responses ( e.g. , OxyR , which is involved in the oxidative-stress response ) or more-global regulators , such as H-NS , IHF , cyclic AMP ( cAMP ) - cAMP receptor protein ( CRP ) , and Lrp ( 104 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16760308" target="_blank">16760308</a>]<br></li>
<li>Thus , it is possible that OxyR represses these genes in response to oxidative-stress .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11443091" target="_blank">11443091</a>]<br></li>
<li>By contrast , genes in the known oxidative-stress regulon of OxyR ( katG , sufA and grxA ) ( 26 ) showed increased transcription in response to oxidants ( fig .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22539721" target="_blank">22539721</a>]<br></li>
<li>In addition , upon oxidative-stress , transcriptional-dual-regulator OxyR induces katG42 and the regulatory RNA OxyS32 which inhibits translation of the rpoS message43 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21516113" target="_blank">21516113</a>]<br></li>
<li>The expression of trxC is regulated by the transcriptional-activator OxyR in response to oxidative-stress .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10644706" target="_blank">10644706</a>]<br></li>
<li>Introduction The OxyR transcriptional-regulator of Escherichia coli and Salmonella typhimurium activates the expression of defense genes in response to oxidative-stress .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8087856" target="_blank">8087856</a>]<br></li>
<li>As an antioxidant defense pleiotropic regulator , OxyS is positively regulated by OxyR , which is a transcriptional-activator under oxidative-stress [ 14 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21106118" target="_blank">21106118</a>]<br></li>
<li>OxyR is also a re-dox-sensitive transcriptional-activator involved in oxidative-stress response by E. coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8588737" target="_blank">8588737</a>]<br></li>
<li>Our data suggest that OxyR is the sole transcriptional-regulator of yaaA in response to oxidative-stress .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21378183" target="_blank">21378183</a>]<br></li>
<li>OxyR is a global regulator that is oxidized in response to oxidative-stress ( 38 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20118257" target="_blank">20118257</a>]<br></li>
<li>This unusual use of FNR and OxyR to activate expression suggests that yhjA plays a speciﬁc role in countering oxidative-stress in E. coli , possibly in particular micro-aerobic environments .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17464064" target="_blank">17464064</a>]<br></li>
<li>To date , it is only known that its expression is triggered by oxidative-stress in an OxyR - , SoxRS - , Fis - , RpoS - , cAMP - , or RecA-independent way ( 36 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15546870" target="_blank">15546870</a>]<br></li>
<li>These results show that OxyR represses fhuF expression under conditions of oxidative-stress .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11443092" target="_blank">11443092</a>]<br></li>
<li>No increased RNAP binding was observed at the genes for GSH biosynthesis that might be expected under conditions of oxidative-stress , but this may simply reﬂect the hierarchy of gene expression with the OxyR regulon ( Carmel-Harel and Storz , 2000 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23646895" target="_blank">23646895</a>]<br></li>
<li>Thus , the proximity of the OxyR binding site to the dsbG start codon led us to speculate that OxyR binding to this site i required for OxyR activation of dsbG expression in response to oxidative-stress .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11443092" target="_blank">11443092</a>]<br></li>
<li>Regardless , given the two overlapping OxyR and Fur binding sites , it is clear that fhuF expression is tightly regulated in response to both oxidative-stress and intracellular iron levels .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11443092" target="_blank">11443092</a>]<br></li>
<li>Cell , Vol .78 , 897-909 , September9 , 1994 , Copyright © 1994by CellPress Redox-Dependent Shift of OxyR-DNA Contacts along an Extended DNA-Binding Site : A Mechanism for Differential Promoter Selection Michel B. Toledano , * Ines Kullik , * Frank Trinh , * t Paul T. Baird , * ~ Thomas D. Schneider , § and Gisela Storz * * Cell Biology and Metabolism Branch National Institute of Child Health and Human Development National Institutes of Health Bethesda , Maryland 20892 § Laboratory of Mathematical Biology National Cancer Institute Frederick , Maryland 21702 Summary The redox-sensitive OxyR protein activates the transcription of antioxidant defense genes in response to oxidative-stress and represses its own expression under both oxidizing and reducing conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8087856" target="_blank">8087856</a>]<br></li>
<li>Recently , workers in this laboratory have provided evidence ( Henderson et al. , 1997b ; Henderson and Owen , 1999 ; Roche et al. , 2001 ) that the Ag43 phase switch is regulated by a novel mechanism involving Dam and OxyR , a LysR-type transcriptional-regulator better known for its role in activating the expression of antioxidant defence genes in response to oxidative-stress ( Toledano et al. , 1994 ; Kullik et al. , 1995 ; Zheng et al. , 1998 ; Åslund et al. , 1999 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11972787" target="_blank">11972787</a>]<br></li>
<li>In this communication , we report on the sequence of agn43 and ﬂanking regions , show that Ag43 belongs to the class of proteins known as bacterial autotransporters ( for a review , see reference 32 ) , and demonstrate that phase switching is regulated by a novel mechanism involving DNA methylation and OxyR , a LysR-type transcriptional-activator better known for its ability to control expression of proteins important in oxidative-stress ( 26 , 43 , 44 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10094691" target="_blank">10094691</a>]<br></li>
<li>However , others have reported that Cys-199 is modified to S-NO , S-OH , and S-SG and have proposed that these modifications of OxyR elicit different responses at increasing concentrations of GSNO ( 30 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14718666" target="_blank">14718666</a>]<br></li>
<li>Thioether and thioester modifications of particular substrates may have different effects on protein function , by analogy to the different posttranslational configurations induced in OxyR by reactive oxygen and nitrogen species ( 23 ) , thereby representing a combinatorial code for redox regulation by plant metabolites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16030151" target="_blank">16030151</a>]<br></li>
<li>From these results , we propose that the oxidant induction of the sufA operon is mediated through OxyR , IHF , plus an unidentiﬁed oxidant-responsive factor , and that the suf gene products are needed to Accepted 17 November , 2003 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15009899" target="_blank">15009899</a>]<br></li>
<li>From these results , we propose that a speciﬁc factor , most likely an oxidant-responsive activator other than OxyR and SoxRS , binds to ORE-III and mediates the induction of sufA by oxidants .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15009899" target="_blank">15009899</a>]<br></li>
<li>In order to estimate the effect of OxyR in more detail , we analysed the induction of sufA transcripts by PMS and H2O2 treatment in wild type and DoxyR mutant by S1 mapping .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15009899" target="_blank">15009899</a>]<br></li>
<li>These results clearly dem-onstrated that the sufA promoter is still inducible by PMS and H2O2 in the absence of OxyR , and suggest the presence of ( a ) responsible regulator ( s ) responding to both PMS and H2O2 , with similar kinetics of induction and feedback regulation to OxyR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15009899" target="_blank">15009899</a>]<br></li>
<li>In this report , we present evidence that the sufA promoter is induced by various superoxide generators as well as by H2O2 , and induction occurs even in the absence of SoxRS and OxyR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15009899" target="_blank">15009899</a>]<br></li>
<li>This indicates that sufA induction by superoxide generators proceeds not only independently of SoxRS and OxyR , but becomes even more sensitized in the absence of those .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15009899" target="_blank">15009899</a>]<br></li>
<li>However , although as yet we do not know the pattern of regulation of nhaR , our results suggest that like the other LysR-OxyR members , the level of NhaR in the wild-type cells is very low .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8168494" target="_blank">8168494</a>]<br></li>
<li>When anaerobic cultures of Hpx − mutants were diluted into aerobic medium , the suf operon was expressed at a level 40-fold higher than in wild-type cells , consistent with its induction by the OxyR protein ( Fig .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21143317" target="_blank">21143317</a>]<br></li>
<li>The use of similar sensing mechanisms by structurally unrelated transcription factors , such as OxyR and SoxR , suggests that there may be a limited number of biologically effective solutions to the problem of setting transcriptional switches within specific redox windows The conformational changes induced in CRP , FNR and FNR-like proteins upon perception of the appropriate signal ` switch ' the regulators from their inactive to active forms .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11407111" target="_blank">11407111</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_291">
Hattman , S. , and Sun , W. ( 1997 ) Escherichia coli OxyR modulation of bacteriophage Mu mom expression in dam 1 cells can be attributed to its ability to-bind hemimethylated Pmom promoter DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10692164" target="_blank">10692164</a>]<br></label>
<input id="act_similarity_291", type="checkbox">
<div class="ACT_similar">
<li>Escherichia coli OxyR modulation of bacteriophage Mu mom expression in dam cells can be attributed to its ability to-bind hemimethylated Pmom promoter DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12644490" target="_blank">12644490</a>]<br></li></div>
<li>Escherichia coli OxyR protein represses the unmethylated bacteriophage Mu mom operon without blocking binding of the transcriptional-activator C. Nucleic Acids Res .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12029051" target="_blank">12029051</a>]<br></li>
<li>To understand the molecular mechanism through which avicins regulate protein function , we examined their effects on the par-adigmatic redox-responsive transcriptional-activator , OxyR of Escherichia coli , which protects bacterial cells against oxidative and nitrosative stresses .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16030151" target="_blank">16030151</a>]<br></li>
<li>To elucidate the nature of the function-regulating interaction of avicins with cysteine residues , we explored their effects on a well characterized redox-regulated response that is controlled by the Escherichia coli transcriptional-activator OxyR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16030151" target="_blank">16030151</a>]<br></li>
<li>D035 .8 D038 .3 RpoS B015 .5 C013 .5 OxyR Cd , QN , HP , uspA PS , NS , Ile , Cd , QN , HP , DNP , 42 NS DNP fumC glpK dnaN ompC DNP , 42 RpoS OmpR , Lrp Lrp , RpoS FIG. 4 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7559363" target="_blank">7559363</a>]<br></li>
<li>Since we previously found that OxyR negatively autoregulates its own expression in vivo ( 10 ) , we also examined the ability of OxyR to regulate oxyR transcription under oxidizing ( 1 mM DTT ) and reducing ( 100 mM DTT ) conditions with purified OxyR and RNA polymerase in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2183352" target="_blank">2183352</a>]<br></li>
<li>To understand how OxyR acts to differentially regulate gene expression under oxidizing and reducing conditions and to gain insight into the consequences of an extended binding site , we analyzed the binding of OxyR to its target DNAs under oxidizing and reducing conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8087856" target="_blank">8087856</a>]<br></li>
<li>Our in vivo experiments suggest that the two modes of binding allow OxyR to regulate different promoters under oxidizing and reducing conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8087856" target="_blank">8087856</a>]<br></li>
<li>However , the nature of the conformational change as well as how OxyR acts to regulate different promoters under oxidizing and reducing conditions are not understood .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8087856" target="_blank">8087856</a>]<br></li>
<li>OxyR was previously shown to-bind differently under oxidizing and reducing conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8087856" target="_blank">8087856</a>]<br></li>
<li>These results suggest that , in vivo , OxyR is able to-bind to the A1A2XA ~ mutant promoter under reducing conditions and causes a slight increase in expression during normal growth .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8087856" target="_blank">8087856</a>]<br></li>
<li>The oxyR and oxyS promoters overlap , and OxyR bound to its site at these promoters acts to both repress oxyR expression ( under oxidizing and reducing conditions ) and activate oxyS expression ( only under oxidizing conditions ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7868602" target="_blank">7868602</a>]<br></li>
<li>At the oxyRSpromoter , OxyR binds under both oxidizing and reducing conditions and is a repressor of its own expression and an activator of oxyS.When two nucleotides at contact point 5 in the oxyRS site were mutated to the corresponding bases in the ahpC promoter ( O1020304X ) , reduced OxyR was no longer able to-bind in vitro and did not repress the oxyR promoter in vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8087856" target="_blank">8087856</a>]<br></li>
<li>MetR MetR metH NR ( I ) , CRP ginApl ... ¬ -4 - II * NR ( I ) NR ( I ) a II I gInL NR ( I ) NFR ( l ) , ' OmpR OmpR O OmO pC pR momp R II OmpR OmpR OmpR ompF I OxyR OxyR ahpC  ' OxyROxyR   OxyR OxyR katG a a ~ ~ OxyR oxyR ' PapB I a PapB , CRP aPapB PapB papB , ' , +92 +125 PhoB phoA PhoB PhoB I Pho B PhoB PhoB phOB 4 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1943993" target="_blank">1943993</a>]<br></li>
<li>Although it is still possible that the OxyR protein mediates the protection upstream from the ahpC and katG promoters by activating another binding factor , we do not favor this hypothesis , because substantially purified OxyR preparations show the same protection patterns ( G. Storz , L. A. Tartaglia & B. N. Ames , unpublished results ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2693740" target="_blank">2693740</a>]<br></li>
<li>OxyR Protein Binds to the Regulatory Regions of the oxyR and katG Genes -- Experiments with the oxyR ' - ` lacZ fusion gene support the view that OxyR protein negatively regulates its own synthesis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1864839" target="_blank">1864839</a>]<br></li>
<li>Sequences bound by OxyR at the following promoters ( 13 ) katG 5 ' - ATGTAAGATCrCAACTATCGCATCCGTGGATTAATTCAATTATAA-3 ' ahpC 5 ' - GGGTTGTTAGTTAACGCITATTGATTTGATAATGGAAACGCATTAG-3 ' oxyR 5 ' - GCCACGATAGTTCATGGCGATAGGTAGAATAGCAATGAACGATTA-3 ' 20 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2183352" target="_blank">2183352</a>]<br></li>
<li>To investigate this possibility and further characterize the reduced form of OxyR , we assayed the ability of OxyR to-bind to the katG , ahpC , and oxyR promoters under both conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2183352" target="_blank">2183352</a>]<br></li>
<li>Binding of oxidized and reduced OxyR to the katG , ahpC , and oxyR promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2183352" target="_blank">2183352</a>]<br></li>
<li>At the katG and ahpC promoters , OxyR binds preferentially under oxidizing conditions and acts as an activator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8087856" target="_blank">8087856</a>]<br></li>
<li>Should OxyR be bound at the ahpC , katG , dps , or gorA promoters under reducing conditions , the protein might induce these genes under inappropriate conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8087856" target="_blank">8087856</a>]<br></li>
<li>We postulate that low amounts of reduced OxyR molecules are present in the cells at all times , bound to the promoters of oxyR-regulated genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2183352" target="_blank">2183352</a>]<br></li>
<li>The autoregulated oxyR gene in E. coli is also not linked to any of the genes for which it serves as an activator ( 9 ) , but although DNase I footprints are qualitatively different with reduced and oxidized forms of highly purified OxyR protein , appreciable quantitative and differences in binding to positively negatively regulated promoters have not been observed ( 37 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1706701" target="_blank">1706701</a>]<br></li>
<li>Some of the other bacterial regulators that show homology to OxyR have also been found to be bound to the promoters that they regulate in both the presence and absence of inducer ( 22 , 24 ) suggesting that members of this family of regulators may all be ideally designed for responses requiring immediate induction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2183352" target="_blank">2183352</a>]<br></li>
<li>DNase I assays in the presence of 1 and 100 mM DTT clearly show that OxyR binds to all three promoters , with high affinity under both oxidizing and reducing conditions ( Fig. 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2183352" target="_blank">2183352</a>]<br></li>
<li>In air-saturated buffers , OxyR is active in the presence of 10 mM DTT ( Fig. 3C , lanes 1 and 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2183352" target="_blank">2183352</a>]<br></li>
<li>However , OxyR incubated in the presence of 100 mM DTT no longer activated expression of katG ( Fig. 3B , lane 3 compared to lane 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2183352" target="_blank">2183352</a>]<br></li>
<li>If the transcription assays were performed in the presence of 10 mM DTT under a nitrogen atmosphere , OxyR no longer activated katG expression ( Fig. 3C , lanes 3 and 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2183352" target="_blank">2183352</a>]<br></li>
<li>Suspecting that OxyR could be activated solely on release from the reducing environment of the cell into air-saturated buffers [ even buffers containing 1 mM dithiothrei-tol ( DTT ) ] , we prepared extracts in the presence of various antioxidants and reductants .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2183352" target="_blank">2183352</a>]<br></li>
<li>Since OxyR is activated simply by the presence of air in 10 mM DTT in our in vitro assays , we suggest that reactive oxygen species are generated by the dissolved oxygen in our purification and assay buffers .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2183352" target="_blank">2183352</a>]<br></li>
<li>This correlation is consistent with the locked OFF phenotype that is seen for an isolate with an increased total cellular level of OxyR due to the presence of an OxyRproducing plasmid ( 18 , 23 , 44 , 46 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20118257" target="_blank">20118257</a>]<br></li>
<li>Finally , the presence of unmethylated regulatory region in the ahpCF katG mutant is consistent with methylation protection by bound OxyR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12644490" target="_blank">12644490</a>]<br></li>
<li>hps-19 : : lacZ expression was repressed in the presence of OxyR ; thus , this fusion is a class III fusion mutant .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8990283" target="_blank">8990283</a>]<br></li>
<li>Transcription from a katG - lacZ fusion increases in the presence of oxidized OxyR ( 33 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12644490" target="_blank">12644490</a>]<br></li>
<li>Nucleoprotein complexes were allowed to form with 7 pmol of either SeqA or OxyR , followed by the addition of Dam in the presence or absence of the methyl group donor S-adenosylmethionine ( SAM ) ( Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20118257" target="_blank">20118257</a>]<br></li>
<li>To determine whether fhuF is regulated by OxyR , the wild-type and the DoxyR and Dfur mutant strains were grown in the absence and presence of the iron chelator 2,29-dipyridyl .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11443092" target="_blank">11443092</a>]<br></li>
<li>Even though agn0 ± lacZ transcription occurs in the absence of OxyR , the level of transcription in oxyR isolates was consistently about twofold lower than that of phase ON cells in the presence of the wild-type allele ( Table 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10692164" target="_blank">10692164</a>]<br></li>
<li>In addition , the DNAase I protection pattern for the C199S protein is identical to the pattern observed for wild-type OxyR in the presence of 200 mM DTT ( Figure 5A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8087856" target="_blank">8087856</a>]<br></li>
<li>In the presence of the OxyR protein , transcription from these promoters was greatly stimulated by wild-type RNA polymerase ( Fig. 1 , B and 0 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8483417" target="_blank">8483417</a>]<br></li>
<li>We suggest that NorR , and perhaps the unidentified regulators , were evolved to sense reactive nitrogen species , whereas Fur , SoxR , and OxyR were evolved to sense iron and reactive oxygen species and are collaterally activated by reactive nitrogen species .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14718666" target="_blank">14718666</a>]<br></li>
<li>Furthermore , Fur is activated by OxyR and SoxRS , regulators of the antioxidant defense response in E. coli , thereby reducing the production of highly reactive hydroxyl radicals and other toxic oxygen species through the Fenton reaction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15878120" target="_blank">15878120</a>]<br></li>
<li>We do not know whether 2 2 the mechanistic basis of this difference stems from a difference in OxyR afﬁnity , but the end effect resembles the graded responses of members of the SOS regulon , some of which are turned on rapidly and others of which are activated later ( 41 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21378183" target="_blank">21378183</a>]<br></li>
<li>Members of the OxyR regulon are , in general , either repressed or activated by speciﬁcally one form of OxyR or the other ( 9 , 40 , 42 , 47 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12029051" target="_blank">12029051</a>]<br></li>
<li>In E. coli , the SoxR and OxyR regulatory proteins are activated by NO , and mutants with defects in these systems are more sensitive to RNIs and nitrosative stress ( 7 , 12 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12142437" target="_blank">12142437</a>]<br></li>
<li>The ﬁrst of these to be found , the OxyR system , is triggered when peroxide oxidizes an activated thiolate residue to a sulfenic acid on the OxyR transcription factor ( 31 , 55 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21378183" target="_blank">21378183</a>]<br></li>
<li>The activated OxyR protein then directly stimulates transcription of genes involved in defense .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21378183" target="_blank">21378183</a>]<br></li>
<li>The molecular mechanism by which the OxyR protein is activated by oxidative-stress is of interest .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2471187" target="_blank">2471187</a>]<br></li>
<li>The diversity of genes activated by OxyR and SoxR illustrates the variety of cellular defense mechanisms against oxi-dative stress .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11395452" target="_blank">11395452</a>]<br></li>
<li>Thus growth defects arise and mutagenesis becomes prominent when intracellular H2O2 concentrations reach 1 μM , and it is fitting that the OxyR protein becomes activated when H2O2 levels approach this value .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19400769" target="_blank">19400769</a>]<br></li>
<li>In this connection , OxyR protein was activated by exposure to oxygen during extraction from cells ( 29 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1400156" target="_blank">1400156</a>]<br></li>
<li>It is also unlikely that OxyR is activated by the formation of an intermolecular disulfide bond with a small molecule , such as glutathione , since OxyR is activated by gel filtration that separates the protein from all small molecules in our assays .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2183352" target="_blank">2183352</a>]<br></li>
<li>It was recently reported that OxyR is activated through the formation of a disulﬁde bond and is deactivated by enzymatic reduction with glutaredoxin 1 [ 20 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11440837" target="_blank">11440837</a>]<br></li>
<li>For instance , OxyR is activated by the transient formation of a disulfide bond .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10644706" target="_blank">10644706</a>]<br></li>
<li>Alternatively , OxyR may be activated by direct binding of the reactive oxygen species .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2183352" target="_blank">2183352</a>]<br></li>
<li>However , it is poss-ible also that the OxyR binding activity was activated by oxidation upon release from the reducing environment of an E. coli cell .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2693740" target="_blank">2693740</a>]<br></li>
<li>RclR now joins the relatively small group of regulators whose functions are specifically activated by oxidation ( e.g. OxyR and SoxR ) ( 13 , 14 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24078635" target="_blank">24078635</a>]<br></li>
<li>These results indicate that hcp responded to OxyR activated by S-nitrosylation rather than S-oxidation ( S-OH , S-S , S-SG ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22539721" target="_blank">22539721</a>]<br></li>
<li>OxyR can be activated by several redox-related modifications of a single Cys-199 ( OxyR-SX ) , including S-nitrosylation , S-OH , and S-glutathione .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16030151" target="_blank">16030151</a>]<br></li>
<li>Although we did not speciﬁcally address whether the difference in activity for the short and long PsufA promoter-lacZ fusions was due to the presence or absence of the OxyR binding site , OxyR is not presumed to be active under the growth conditions used for the strain constructions , and we found no mutations within oxyR that may have resulted in a constitutively active regulator ( 45 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25266384" target="_blank">25266384</a>]<br></li>
<li>Both OxyR and IHF are required for suf induction in response to H2O2 To further characterize the suf transcriptional response to H2O2 and 2,2 cents - dipyridyl , we deﬁned the regulatory features of the suf promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15101990" target="_blank">15101990</a>]<br></li>
<li>The induction was dependent on OxyR , IHF and an unidenti-ﬁed factor that binds to the ORE-III site in close proximity to the -35 region of the sufA promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15009899" target="_blank">15009899</a>]<br></li>
<li>The suf promoter is activated independently by apo-IscR and OxyR ( Yeo et al. , 2006 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22096151" target="_blank">22096151</a>]<br></li>
<li>The induction of suf operon promoter activity by wrwycr treatment agrees with this idea , and implicates IscR ( and not OxyR ) as being necessary for this induction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22096151" target="_blank">22096151</a>]<br></li>
<li>Our results suggest that the dps promoter is recognized by both o ^ ° - hoioenzyme and < r ® - holoenzyme , since OxyR acts through a ^ ° and the starts of the OxyR - and ( T ^ - dependent transcripts are identicai .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7984106" target="_blank">7984106</a>]<br></li>
<li>Binding of OxyR to the promoter region provides an explanation of how OxyR can autoregulate its own synthesis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2551682" target="_blank">2551682</a>]<br></li>
<li>864 F. W. Outten , O. Djaman and G. Storz Independent binding of IHF , OxyR and Fur to the suf promoter Fig. 4 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15101990" target="_blank">15101990</a>]<br></li>
<li>Possibly the dps promoter is not ideally bound by either holoenzyme leading to the need for IHF to enhance the binding of o ^ - RNA polymerase and OxyR to enhance the binding of a '' ° - RNA polymerase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7984106" target="_blank">7984106</a>]<br></li>
<li>We show that both OxyR and IHF bind upstream of the dps promoter , and both mechanisms of activation act independently of each other .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7984106" target="_blank">7984106</a>]<br></li>
<li>OxyR and IHF can simultaneously bind to the dps promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7984106" target="_blank">7984106</a>]<br></li>
<li>Moreover , in gel retardation assays in which both FNR * and OxyR were present , the migration of the labelled yhjA promoter was slower than in the presence of either FNR * or OxyR alone , suggesting that both proteins occupy the yhjA promoter simultaneously ( Fig. 1 , lane 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17464064" target="_blank">17464064</a>]<br></li>
<li>No additional products were observed in the presence of OxyR ( ox ) , indicating that it was also not acting as an activator for an alternative agn43 promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12644490" target="_blank">12644490</a>]<br></li>
<li>To learn more about the nature of the repression we investigated the ability of the OxyR protein to-bind its own promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2471187" target="_blank">2471187</a>]<br></li>
<li>The wild-type RNA polymerase could bind to the katG promoter just downstream of the OxyR-binding sites only in the presence of OxyR protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8483417" target="_blank">8483417</a>]<br></li>
<li>This indicates that RNA polymerase alone can not bind to the katG promoter in the absence of OxyR protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8483417" target="_blank">8483417</a>]<br></li>
<li>This indicates that the mutant polymerase can not bind detectably to the katG promoter even in the presence of OxyR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8483417" target="_blank">8483417</a>]<br></li>
<li>The OxyR system is calibrated appropriately , responding when H2O2 levels exceed 0.2 micromolar ( Carmel-Harel & Storz , 2000 , Imlay , 2008a ) -- that is , at a dose above which the half-time of Fenton reactions approaches 1 min .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21143317" target="_blank">21143317</a>]<br></li>
<li>The inactivation of OxyR by 100 mM DTT was readily reversible since OxyR regained the ability to activate katG expression upon removal of the high concentration of DTT by gel filtration ( Fig. 3B , lane 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2183352" target="_blank">2183352</a>]<br></li>
<li>The ability of FNR , located in a typical class II position , to only weakly activate expression of yhjA in the absence of OxyR suggests that the role of OxyR might be to counteract the effects of an as-yet-unidentiﬁed regulator that inhibits the action of FNR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17464064" target="_blank">17464064</a>]<br></li>
<li>The results also show that OxyR binds on one face of the DNA helix as has been reported for the LysRtype regulators Trpl , CatR , and NodD ( Chang and Crawford , 1990 ; Rothmel , et al. , 1991 ; Fisher and Long , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8087856" target="_blank">8087856</a>]<br></li>
<li>The agn43 gene is repressed when its regulatory region binds reduced OxyR , a feature it shares with the regulatory regions of mom and oxyRS , the other members of the OxyR regulon that are repressed by reduced OxyR ( Toledano et al. , 1994 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11972787" target="_blank">11972787</a>]<br></li>
<li>The oxyRpromoter is repressed by OxyR and strains carrying deletions of the oxyR structural gene show increased 13-galactosidaseactivityfrom an oxyR-lacZfusion ( Christman et al. , 1989 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8087856" target="_blank">8087856</a>]<br></li>
<li>The in vitro sensitivity of RclR to H2O2 was much lower leucine codons , and one CCC proline codon ) ( 46 ) , which than that reported for the bona fide H2O2-sensing activator resulted in poor overexpression from the native gene sequence OxyR , which is present in the oxidized , active form without the ( data not shown ) , RclR was overexpressed for purification from addition of H2O2 in vitro ( 47 ) and is fully activated by as little as an rclR gene that was codon-optimized for expression in E. coli 10 M H2O2 in vivo ( 48 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24078635" target="_blank">24078635</a>]<br></li>
<li>The katG gene was shown to be activated by nitrosylated OxyR in vitro ( 30 ) , and a plasmid-borne katG -- lacZ fusion was reported to be induced by 1 mM S-nitrosocysteine in vivo ( 27 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14718666" target="_blank">14718666</a>]<br></li>
<li>The oxidized form of OxyR appears to be more efficient than the reduced form in repressing its own expression , in agreement with the observation that the synthesis of OxyR is decreased in cells treated with hydrogen peroxide compared to untreated cells ( Fig. 1B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2183352" target="_blank">2183352</a>]<br></li>
<li>The cellular response to peroxide that is mediated by oxidized OxyR is transient , since the inducing agent , peroxide , is enzymatically degraded rapidly and , in addition , since oxidized OxyR in the cell is converted back to the reduced state within 30 min ( 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12644490" target="_blank">12644490</a>]<br></li>
<li>The presence of bands when using a dps probe and cells extracts of cells grown in the presence of L-ascorbate ( Fig. 8b ) reinforce the idea that OxyR mediates the high-level expression of ahpC in the presence of L-ascorbate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17906139" target="_blank">17906139</a>]<br></li>
<li>The oxyR expression was reduced substantially in the presence of both reduced and oxidized OxyR ( Fig. 4 , lanes 2 and 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2183352" target="_blank">2183352</a>]<br></li>
<li>The expression of the lacZ fusion in 8 mutants was independent of OxyR , 10 mutants required OxyR for expression , and 6 mutants showed reduced levels of expression in the presence of OxyR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8990283" target="_blank">8990283</a>]<br></li>
<li>The oxidized OxyR binds to promoter elements of the genes it regulates , resulting in the onset of their expression , and the protein also represses its own synthesis ( 3 , 19 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9922271" target="_blank">9922271</a>]<br></li>
<li>OxyR-dependent Peroxide Induction of trxC The genetic analysis strongly suggested that OxyR regulates trxC expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10644706" target="_blank">10644706</a>]<br></li>
<li>The difference between our results and theirs can be reconciled by taking into consideration that different expression systems for OxyR were used , which can lead to different cellular levels of OxyR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12644490" target="_blank">12644490</a>]<br></li>
<li>The combined effect of OxyR and Dam on ﬂu results in phase-variable expression of Ag43 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10648536" target="_blank">10648536</a>]<br></li>
<li>The interconversion of OxyR between the active and inactive forms is also readily reversible within the 5-minute period required to remove the DTT .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2183352" target="_blank">2183352</a>]<br></li>
<li>The C199S mutant was unable to activate the antioxidant genes and gave rise to an extended footprint characteristic of reduced OxyR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7868602" target="_blank">7868602</a>]<br></li>
<li>Storz , G. , Tartaglia , L.A. , and Ames , B.N. ( 1990a ) The OxyR regulon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23646895" target="_blank">23646895</a>]<br></li>
<li>The oxyR transcriptton is repressed by addition ot OxyR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8483417" target="_blank">8483417</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8483417" target="_blank">8483417</a>]<br></li>
<li>RESULTS The gene yaaA is induced by OxyR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21378183" target="_blank">21378183</a>]<br></li>
<li>The mntH gene is additionally repressed by iron-loaded Fur , and both it and dps are induced by H2O2-oxidized OxyR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25774656" target="_blank">25774656</a>]<br></li>
<li>The progressive decrease in grxA transcript abundance suggests that OxyR is most active during the early stages of adaptation to aerobic metabolism .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16857675" target="_blank">16857675</a>]<br></li>
<li>The ahpC transcript was induced by PMS in a similar pattern to sufA , demonstrating that PMS can activate OxyR as suggested .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15009899" target="_blank">15009899</a>]<br></li>
<li>The affinities of oxidized OxyR for the IzatC , oxyR , and orf0promoter fragments and of reduced OxyR for the ahp , IzatG , oxyR , and orf0promoter fragments were calculated in an identical manner , and the results are shown inTable I. Identicalconditions were used for the reduced OxyR binding experiments except 100 mM DTT was included in the binding mix 10 min prior to the addition of the DNA probes which results in the complete or near complete conversion of OxyR to its reduced form ( Storz et aZ .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1730735" target="_blank">1730735</a>]<br></li>
<li>The percent shifted DNA in an EMSA with increasing amounts of puriﬁed wild-type OxyR is shown .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12644490" target="_blank">12644490</a>]<br></li>
<li>The footprinting experiments described here suggest that the OxyR protein binds to DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2471187" target="_blank">2471187</a>]<br></li>
<li>The absence of and decrease in percentage of ON colonies in isolates with OxyR ( A233V ) and OxyR ( H198R ) , respectively , appear to be a result of the decreased binding afﬁnity of these mutants for agn43 DNA and not to be related to the redox state of OxyR ( Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12644490" target="_blank">12644490</a>]<br></li>
<li>The presence of the bend is consistent with the presence of strong DNAase I hypersensitive sites in the centers of the reduced OxyR-wt and OxyR-C 199S footprints and may be imposed by the altered DNA contacts required for reduced OxyR binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8087856" target="_blank">8087856</a>]<br></li>
<li>The OxyR protein-DNA complex bound to protein A-Sepharose was then collected by centrifugation and washed three times with 500 ml of binding buffer .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9294462" target="_blank">9294462</a>]<br></li>
<li>The half-maximal binding concentration was determined by plotting the decrease in free DNA versus the OxyR concentration .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1730735" target="_blank">1730735</a>]<br></li>
<li>The binding of OxyR to methylated and non-methylated Ag43 DNA was determined using EMSAs performed with Ag43 DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10692164" target="_blank">10692164</a>]<br></li>
<li>The binding of OxyR to methylated DNA was also examined ( Fig. 4B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11972787" target="_blank">11972787</a>]<br></li>
<li>The observation that reduced OxyR could act as a repressor suggested that OxyR was capable of binding to DNA under both oxidizing and reducing conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2183352" target="_blank">2183352</a>]<br></li>
<li>Effects of Binding Site Mutations on Promoter Function In Vivo The binding experiments showed that OxyR binds different sequences under oxidizing and reducing conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8087856" target="_blank">8087856</a>]<br></li>
<li>The above experiments indicate that OxyR recognizes the ahpC - , IzatC - , oxyR - , and orf0-binding sites with haigh and similar affinity , and it does this in both its oxidized and reduced forms .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1730735" target="_blank">1730735</a>]<br></li>
<li>The DNA sequence of the oxyRS promoter region is such that both these patterns of binding are possible , and thus OxyR regulates its own expression , when reduced and oxidized , simply by repositioning itself on the DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17464064" target="_blank">17464064</a>]<br></li>
<li>For many LTTRs , including CysB ( 15 ) , OxyR ( 33 ) , OccR ( 1 , 31 , 69 ) , NocR ( 31 ) , CatR ( 52 ) , ClcR ( 40 ) , and CbbR ( 18 ) , DNA bending by the protein is more pronounced in the absence of the activating coeffector than in its presence , leading to the suggestion that bending is inversely correlated with gene activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22904281" target="_blank">22904281</a>]<br></li>
<li>Under this condition manganese uptake ensues when the OxyR regulator senses 2 2 ) were pre-cultured in anoxic M9 glucose/casamino acids medium and then diluted at time zero into aerated medium of the same composition .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25774656" target="_blank">25774656</a>]<br></li>
<li>Phase variation of pap and agn are both Dam dependent , and the regulation of each operon involves the DNA methylation-sensitive binding of a transcriptional-regulator , Lrp and OxyR respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10692164" target="_blank">10692164</a>]<br></li>
<li>OxyR is a peroxide-sensing , global transcriptional-regulator ( reviewed in references 11 , 16 , 23 , and 31 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12644490" target="_blank">12644490</a>]<br></li>
<li>Multidegenerate DNA recogmtion by the OxyR transcriptional-regulator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8087856" target="_blank">8087856</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_410">
, Storz , G , , and Ames , B.N , ( 1992 ) Muitidegenerate DNA recognition by the OxyR transcriptional-regulator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7891555" target="_blank">7891555</a>]<br></label>
<input id="act_similarity_410", type="checkbox">
<div class="ACT_similar">
<li>Tartaglia , L.A. , Gimeno , C.J. , Storz , G. , and Ames , B.N. ( 1992 ) Multidegenerate DNA recognition by the OxyR transcriptional-regulator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11972787" target="_blank">11972787</a>]<br></li></div>
<li>OxyR has provided a model for understanding the ubiquitous influence of NO redox on cellular function , and it is therefore likely that avicinylation will regulate additional classes of proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16030151" target="_blank">16030151</a>]<br></li>
<li>However , an intramolecular disulﬁde bridge is unlikely to be the redox-active center of OxyR , since only C-199 proved to be critical , and the mutant in which all other cysteines were mutated to serine was still active in vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7868602" target="_blank">7868602</a>]<br></li>
<li>It is interesting to note that NorR , Fur , SoxR , and OxyR all appear to contain redox-active cysteines and , in some cases , iron .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14718666" target="_blank">14718666</a>]<br></li>
<li>Both the redox loop and the β-sheet undergo major conformational changes upon OxyR oxidation ( 33 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23690622" target="_blank">23690622</a>]<br></li>
<li>M206 in HypT would be located in the redox loop in OxyR where the active-site Cys208 is located , and M230 in HypT would be located next to a β-sheet in OxyR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23690622" target="_blank">23690622</a>]<br></li>
<li>M205 in the redox loop of OxyR corresponds to M206 in HypT ( Fig .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23690622" target="_blank">23690622</a>]<br></li>
<li>To elucidate the mechanism of OxyR activation during ARN , we focused on hcp ( encoding a hybrid cluster protein ) , which is highly induced in an OxyR-dependent manner ( 40-fold versus 5-fold in WT versus ΔoxyR ) ( table S1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22539721" target="_blank">22539721</a>]<br></li>
<li>This model resembles the mechanism of redox activation of OxyR , in which initial oxidation of Cys-199 precedes formation of a Cys-199 -- Cys-208 disulfide bond in the fully activated protein ( 57 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24078635" target="_blank">24078635</a>]<br></li>
<li>Earlier studies concluded that OxyR activation involves formation of an intramolecular disulfide bond between Cys-199 and -208 , representing an `` on -- off switch '' ( 19 , 22 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16030151" target="_blank">16030151</a>]<br></li>
<li>Although the redox-active Cys ¬ 199 residue in OxyR probably can undergo modification to S-NO , this form of the protein is at best capable of limited activation of a subset of promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14718666" target="_blank">14718666</a>]<br></li>
<li>Modification of OxyR Cys-199 is necessary for transcriptional activation by redox-active molecules , but Cys-208 or -180 may also influence activity ( 23 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16030151" target="_blank">16030151</a>]<br></li>
<li>Thus , it seemed possible that the induction of TrxC that we observed in the ahpCF mutants was due to activation of OxyR and that the one uncharacterized mutation in DR352 was in the oxyR gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10644706" target="_blank">10644706</a>]<br></li>
<li>However , the high concentrations of DTT might inhibit activation by denaturing OxyR or by chelating a metal required for function .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2183352" target="_blank">2183352</a>]<br></li>
<li>Presumably , the rate of OxyR reduction by 100 mM DTT , but not 10 mM DTT , is sufficient to inhibit OxyR activation by the low concentrations of oxidants present in our buffer solutions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2183352" target="_blank">2183352</a>]<br></li>
<li>Together , these results suggest that OxyR is required for repression and that Dam is required to alleviate this repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10692164" target="_blank">10692164</a>]<br></li>
<li>1 These results indicate that Dam is required for full activation even in the absence of the repressor OxyR ( Table 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10692164" target="_blank">10692164</a>]<br></li>
<li>These results suggest an OxyR dependence in the L-ascorbate activation of ahpC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17906139" target="_blank">17906139</a>]<br></li>
<li>The increasing intensity of the retardation band corresponding to the complex OxyR-DNA when L-ascorbate was increased from 1 to 5 or 10 mM in the culture suggests increased H2O2 formation as well as increased oxidation of OxyR with its concomitant activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17906139" target="_blank">17906139</a>]<br></li>
<li>The reason for this is not clear , but it suggests that OxyR contributes to transcriptional activation as well as to repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10692164" target="_blank">10692164</a>]<br></li>
<li>The formation of H2O2 in our cultures , as indicated above , guarantees the required activation of OxyR through oxidation of this transcriptional factor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17906139" target="_blank">17906139</a>]<br></li>
<li>We assumed that OxyR binding to its target promoters is required for transcriptional activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7868602" target="_blank">7868602</a>]<br></li>
<li>Avicinylation of OxyR results in transcriptional activation of the target genes , katG and oxyS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16030151" target="_blank">16030151</a>]<br></li>
<li>Here and in the accompanying paper ( 20 ) , we identify regions of the OxyR protein critical for activation and repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7868602" target="_blank">7868602</a>]<br></li>
<li>This suggests that OxyR-dependent activation of agn is mediated indirectly by OxyR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10692164" target="_blank">10692164</a>]<br></li>
<li>Interestingly , this transcriptional activation was independent of both OxyR and SoxRS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14594836" target="_blank">14594836</a>]<br></li>
<li>Its induction was used to measure OxyR activation and served as a control .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8990283" target="_blank">8990283</a>]<br></li>
<li>Transcriptional activation by mutant OxyR proteins in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7868602" target="_blank">7868602</a>]<br></li>
<li>OxyR mediates H2O2 activation of mntH .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12029030" target="_blank">12029030</a>]<br></li>
<li>OxyR activation in peroxidase-deficient strains .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7986015" target="_blank">7986015</a>]<br></li>
<li>RESULTS OxyR activation of fur .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10419964" target="_blank">10419964</a>]<br></li>
<li>RESULTS Lack of OxyR activation by INH .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7986015" target="_blank">7986015</a>]<br></li>
<li>Alternatively , if the constitutive phenotype was caused only by the abnormal exposure of an activation domain , the mutants should bind in the extended conformation characteristic of reduced wild-type OxyR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7868602" target="_blank">7868602</a>]<br></li>
<li>Although reduced OxyR does not repress oxySexpression , it may oxyS-35 be repositioned over the sequence under reducing conditions to prevent inappropriate activation of oxyS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8087856" target="_blank">8087856</a>]<br></li>
<li>Results Exposure of reduced OxyR to room air [ which generates Cys-199-SOH ( 23 ) ] or to F094 ( an unspecified mixture of avicin D and G ) , avicin D or G ( avicins : OxyR , 1:10 -- 100:1 ) under strictly anaerobic conditions resulted in OxyR activation as assessed by in vitro transcriptional activation of katG and oxyS ( Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16030151" target="_blank">16030151</a>]<br></li>
<li>Most protein cysteine residues react far too sluggishly to be affected by micromolar H2O2 , but the OxyR , AhpC , and OhrR proteins feature active-site cysteines that are activated by local context so they are easily oxidized to sulfenic acid ( Imlay , 2008a ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21143317" target="_blank">21143317</a>]<br></li>
<li>Exposure to H2O2 leads to induction of at least 30 proteins , 9 of which are under direct positive control of the OxyR protein ( 7 , 17 , 18 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8779563" target="_blank">8779563</a>]<br></li>
<li>Repression of fhuF transcription by OxyR may lead to decreased levels of FhuF protein which in turn would slow iron uptake , thus providing another mechanism for the cell to minimize the escalation of oxidative damage by the Fenton reaction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11443092" target="_blank">11443092</a>]<br></li>
<li>These observations suggest that OxyR protein functions as a transcriptional-repressor for oxyR , in addition to its known function as the transcriptional-activator for the oxyR regulon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1864839" target="_blank">1864839</a>]<br></li>
<li>These results suggest that a change in the quality of OxyR protein rather than in the quantity is involved in the activation of the genes in the oxyR regulon , and that OxyR protein always functions as the repressor for the oxyR gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1864839" target="_blank">1864839</a>]<br></li>
<li>OxyR protein specifically bound to the upstream regulatory regions of the oxyR and katG genes as demonstrated by the gel-retardation assay and the DNase I footprinting experiment , and activated the transcription initiation of the katG gene in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1864839" target="_blank">1864839</a>]<br></li>
<li>These results indicate that the OxyR protein purified in this study is in its active form for enhancing the transcription initiation of the genes under its control in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1864839" target="_blank">1864839</a>]<br></li>
<li>Deletion of OxyR resulted in large increases in protein Snitrosylation , and S-nitrosylation of OxyR induced transcription from a regulon that is distinct from the regulon induced by OxyR oxidation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22539721" target="_blank">22539721</a>]<br></li>
<li>OxyR protein activates transcription of genes in the oxyR regulon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1864839" target="_blank">1864839</a>]<br></li>
<li>The 5 ' - end-labeled Hin-dm-Pvull fragment of pKT908 ( Fig. 1 ) was partially digested with DNase I in the presence or absence of OxyR protein , and the digest was analyzed by gel electrophoresis ( Fig. 3A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1864839" target="_blank">1864839</a>]<br></li>
<li>When OxyR protein was included in the transcription reaction , the amount of the shorter transcript increased and the longer transcript became undetectable ( Fig. 5 , lane 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1864839" target="_blank">1864839</a>]<br></li>
<li>The amount of this shorter transcript increased upon addition of the OxyR protein , but the longer one became undetectable under the same conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8483417" target="_blank">8483417</a>]<br></li>
<li>By the addition of OxyR protein , however , the amount of the shorter transcript did not increase but instead decreased .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8483417" target="_blank">8483417</a>]<br></li>
<li>As with FLR it is the disulfide form of OxyR that is the transcriptionally active form .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11407111" target="_blank">11407111</a>]<br></li>
<li>Therefore , OxyR protein might be converted to the active form during purification by some oxidative processes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1864839" target="_blank">1864839</a>]<br></li>
<li>5 ' - End-labeled probe DNAs were incubated with purified OxyR protein at various concentrations , and subjected to acrylamide gel electrophoresis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1864839" target="_blank">1864839</a>]<br></li>
<li>a-256 and a-235 , also gave similar results in the absence of OxyR protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8483417" target="_blank">8483417</a>]<br></li>
<li>Lane 1 , no protein ; lanes 2 and 3 , FNR * ; lanes 4 and 5 , OxyR ; lane 6 , FNR * and OxyR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17464064" target="_blank">17464064</a>]<br></li>
<li>An essential role for S-nitrosylation of OxyR Cys199 in hcp induction during ARN was demonstrated by the finding that transcription of hcp was not increased during ARN in ΔoxyR E. coli overexpressing a mutant form of OxyR in which Ser replaces Cys at position 199 ( Cys199Ser ) ( Fig. 3C ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22539721" target="_blank">22539721</a>]<br></li>
<li>hmp gene expression in the presence of NOz is independent of the SoxS , OxyR , and FNR transcription factors and instead relies on inactivation of the iron-de-pendent Fur repressor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9852058" target="_blank">9852058</a>]<br></li>
<li>In addition , the presence of two OxyR sites may be required to allow for coordinate regulation by both the OxyR and Fur transcription factors .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11443092" target="_blank">11443092</a>]<br></li>
<li>Thus , nitrosylation of iron atoms , especially ironsulfur clusters , in enzymes such as aconitase and fumarase or in the transcription factors FNR , Fur , SoxR , OxyR , and NsrR inactivates their functions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22029434" target="_blank">22029434</a>]<br></li>
<li>Thus , we propose that the E. coli response to reactive nitrogen species is brought about by the modification of some transcription factors , such as NorR , that control the expression of NO detoxifying activities and are dedicated to sensing reactive nitrogen species , and other redoxactive transcription factors such as Fur , SoxR , and OxyR , where the reaction with reactive nitrogen species is incidental .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14718666" target="_blank">14718666</a>]<br></li>
<li>Reduced OxyR makes contact at positions I , II , IV and V , resulting in transcription repression , whereas oxidized OxyR makes contact at positions I , II , III and IV , resulting in transcription activation ( Toledano et al. , 1994 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11972787" target="_blank">11972787</a>]<br></li>
<li>In the case of transcription activation , the OxyR protein binds to the promoter region between - 3 5 to - 80 relative to the OxyR-dependent transcription start points ( Tartaglia ef al. , 1989 ; Tao ef ai .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8483417" target="_blank">8483417</a>]<br></li>
<li>[ 20 ] Zheng , M. ; Åslund , F. ; Storz , G. Activation of the OxyR transcription factor by reversible disulﬁde bond formation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11440837" target="_blank">11440837</a>]<br></li>
<li>Repression of agn0 ± lacZ transcription required OxyR , whereas activation required Dam .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10692164" target="_blank">10692164</a>]<br></li>
<li>Nitrosative stress : activation of the transcription factor OxyR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12142437" target="_blank">12142437</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9019397" target="_blank">9019397</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16428390" target="_blank">16428390</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_474">
Hausladen , A. , Privalle , C. T. , Keng , T. , DeAngelo , J. & Stamler , J. S. Nitrosative stress : activation of the transcription factor OxyR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16193057" target="_blank">16193057</a>]<br></label>
<input id="act_similarity_474", type="checkbox">
<div class="ACT_similar">
<li>Hausladen , A. , Privalle , C.T. , Keng , T. , DeAngello , J. and Stamler , J.S. ( 1996 ) Nitrosative stress : activation of the transcription factor OxyR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12093725" target="_blank">12093725</a>]<br></li>
<li>Hausladen , A. , Privalle , C.T. , Keng , T. , DeAngelo , J. and Stamler , J.S. ( 1996 ) Nitrosative stress : activation of the transcription factor OxyR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12773378" target="_blank">12773378</a>]<br></li></div>
<li>These results suggest that the ability of OxyR to activate transcription is sensitive to the oxidation state of OxyR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2183352" target="_blank">2183352</a>]<br></li>
<li>These parallel assays ( in which the only variable is the presence or absence of oxygen ) strengthen our conclusion that the ability of OxyR to activate transcription is directly dependent on the state of OxyR oxidation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2183352" target="_blank">2183352</a>]<br></li>
<li>A cofactor or metal associated with OxyR might also be sensitive to oxidation ; however , the addition of chelators did not effect the ability of OxyR to activate transcription ( 15 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2183352" target="_blank">2183352</a>]<br></li>
<li>OxyR is sensitive to oxidation and reduction , and only oxidized OxyR is able to activate transcription of its target genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7868602" target="_blank">7868602</a>]<br></li>
<li>OxyR activates transcription on oxidation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2183352" target="_blank">2183352</a>]<br></li>
<li>Upon oxidation , in a process that alters the structure of the complex , the already-bound OxyR becomes competent to activate transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7901196" target="_blank">7901196</a>]<br></li>
<li>Since oxidation is necessary for OxyR to activate transcription in vitro , we suggest that C-199 is the redox center of the protein and that the C199S protein is locked in the reduced conformation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7868602" target="_blank">7868602</a>]<br></li>
<li>In vitro transcription experiments with purified components showed that the OxyR protein is sensitive to oxidation and that only the oxidized form of OxyR is able to activate transcription ( Storz et al. , 1990 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8087856" target="_blank">8087856</a>]<br></li>
<li>In vitro transcription assays demonstrated that avicins activate OxyR and its target genes katG and oxyS in a DTT-reversible manner .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16030151" target="_blank">16030151</a>]<br></li>
<li>We also assayed the ability of purified OxyR to activate transcription in the presence of 1 and 100 mM DTT to determine whether the purified protein showed the same sensitivity to oxidation and reduction as seen for the cell extracts .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2183352" target="_blank">2183352</a>]<br></li>
<li>To distinguish between these possibilities , we assayed the ability of OxyR to Downloade activate transcription under anaerobic conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2183352" target="_blank">2183352</a>]<br></li>
<li>( B ) OxyR was assayed for the ability to activate katG transcription in the presence of 1 or 100 mM DTT .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2183352" target="_blank">2183352</a>]<br></li>
<li>( C ) OxyR ( 1 RI of the centrifiuged sample described for ( A ) was assayed for the ability to activate katG transcription in the presence of air - or nitrogensaturated buffers .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2183352" target="_blank">2183352</a>]<br></li>
<li>We chose to randomly mutagenize the entire oxyR gene and screen for OxyR mutants altered in their ability to activate transcription of the oxyS promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7868602" target="_blank">7868602</a>]<br></li>
<li>We then examined the effect of the mutations on the ability of OxyR to activate and repress transcription of the oxyR - , oxyS - , and ahpC-lacZconstructs .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8087856" target="_blank">8087856</a>]<br></li>
<li>mechanism by which OxyR regulates the expression of the oxyR regulon , we first assayed the ability of extracts from OxyR-overpro-ducing cells to activate expression of the ahp genes in an in vitro transcription-translation assay .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2183352" target="_blank">2183352</a>]<br></li>
<li>To determine whether OxyR was the sole sensor of oxidation and the only component required to activate oxyR-regulated gene expression , we examined the ability of purified OxyR to regulate expression of oxyR-regulated genes in more defined transcription assays .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2183352" target="_blank">2183352</a>]<br></li>
<li>Repression by oxidized OxyR without a decrease in bla expression was observed in coupled transcription-translation assays in which all components are in greater excess ( 18 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2183352" target="_blank">2183352</a>]<br></li>
<li>When binding to the promoter region , OxyR represses its own expression , and upon oxidation , it activates oxyS transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7868602" target="_blank">7868602</a>]<br></li>
<li>OxyR represses its own expression , as well as transcription of the Mu phage mom and the E. coli ﬂu genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11443092" target="_blank">11443092</a>]<br></li>
<li>For example , a model based solely on sites from which OxyR activates transcription might be stronger in predicting othe sites where OxyR binds as an activator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11443092" target="_blank">11443092</a>]<br></li>
<li>At the oxyR promoter , OxyR activates transcription in one direction and represses transcription in the other , whereas OxyR probably only acts to induce transcription at the katG and ahpC promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2183352" target="_blank">2183352</a>]<br></li>
<li>Binding of oxidized OxyR at these latter regions activates transcription of katG and oxyS , respectively ( 33 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12644490" target="_blank">12644490</a>]<br></li>
<li>Our analyses suggest that OxyR bound to hemimethylated DNA is stable enough to be maintained and repress transcription throughout a cell cycle .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20118257" target="_blank">20118257</a>]<br></li>
<li>In the absence of the inducers for OxyR production , in both MV268 and MV307 , transcription of agn0 ± lacZ was observed at levels similar to that of MV263 ( 1076 ^ 50 MU and 1180 ^ 57 MU respectively ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10692164" target="_blank">10692164</a>]<br></li>
<li>Expression of agn43 -- lacZ in plasmids pDEW15 and pDEW16 also decreased signiﬁcantly when OxyR was overexpressed , but transcription was still very much locked ON .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11972787" target="_blank">11972787</a>]<br></li>
<li>Methylation of the GATC sequences in the 5 - UTR is also required for full transcription even in the absence of OxyR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12029051" target="_blank">12029051</a>]<br></li>
<li>This was tested by performing an in vitro transcription assay with methylated and unmethylated DNA , in the presence and absence of OxyR ( C199S ) ( Fig. 5 , lanes 1 and 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12029051" target="_blank">12029051</a>]<br></li>
<li>In vitro transcription of both methylated and unmethylated templates was also performed in the presence of OxyR ( C199S ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12029051" target="_blank">12029051</a>]<br></li>
<li>Collectively , these data establish that hcp transcription is selectively activated by S-nitrosylation of OxyR in vitro and during ARN .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22539721" target="_blank">22539721</a>]<br></li>
<li>In fact , preliminary i-n vitro transcription studies indicate that the transcription stimulatory activity of OxyR becomes activated simply upon extraction into air-saturated buffers .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2693740" target="_blank">2693740</a>]<br></li>
<li>IHF and OxyR both function as activators , while Fis and H-NS repress transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21239586" target="_blank">21239586</a>]<br></li>
<li>In the latter case , due to interaction with OxyR , C might have lost its capability to activate transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2551682" target="_blank">2551682</a>]<br></li>
<li>In this state , OxyR binds to its own regulatory region and represses transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10692164" target="_blank">10692164</a>]<br></li>
<li>Oxidized OxyR represses agn43 transcription in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12644490" target="_blank">12644490</a>]<br></li>
<li>Endogenous S-nitrosylation during anaerobic respiration is controlled by the transcription factor OxyR , previously thought to operate only under aerobic conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22539721" target="_blank">22539721</a>]<br></li>
<li>The transcription factor OxyR , activated by oxidation under aerobic conditions , is also activated by S-nitrosylation under anaerobic conditions and thereby induces expression in a distinct regulon , which reveals that a transcriptional-activator can independently control two distinct regulons through alternative posttranslational modifications .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22539721" target="_blank">22539721</a>]<br></li>
<li>It remains to be determined whether the oxidized form of OxyR can repress transcription , and what the contribution of either form is for phase variation in vivo at physiological concentrations of OxyR protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10692164" target="_blank">10692164</a>]<br></li>
<li>The effect of oxidized OxyR on in vitro transcription was studied by the addition of 9.5 and 19 nM puriﬁed OxyR to the reaction mix .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12644490" target="_blank">12644490</a>]<br></li>
<li>The tetrameric OxyR protein exists in two forms , reduced and oxidized , and only the oxidized form activates transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11479716" target="_blank">11479716</a>]<br></li>
<li>The transcription of nhaA , which is positively regulated by NhaR ( 29 ) , a member of the LysR-OxyR family ( 7 , 14 , 30 , 34 ) , is speciﬁcally induced by intracellular Na1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11133959" target="_blank">11133959</a>]<br></li>
<li>The synthesis of these antioxidant activities is regulated directly by the redox-sensitive transcription factor OxyR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9230301" target="_blank">9230301</a>]<br></li>
<li>The finding that DTT , a reductant , could reversibly inactivate the OxyR protein suggested that the redox state of the OxyR protein was coupled to its ability to stimulate transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2183352" target="_blank">2183352</a>]<br></li>
<li>OxyR in vivo may be sensitive to changes in the H2O2 levels rather than to the absolute values , analogous to signal processing in chemotaxis ( 22 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8990289" target="_blank">8990289</a>]<br></li>
<li>This is in agreement with previous data showing that H2O2 activates OxyR by specific cysteine oxidation ( 27 ) but does not significantly change the cellular OxyR concentration ( 28 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23139412" target="_blank">23139412</a>]<br></li>
<li>OxyR responds to the presence of H2O2 and appears to be sensitive to oxidation and reduction at a single cysteine residue .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8588737" target="_blank">8588737</a>]<br></li>
<li>Furthermore , products unique to the anaerobic regulon protected against S-nitrosothiols , and anaerobic-growth of E. coli lacking OxyR was impaired on nitrate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22539721" target="_blank">22539721</a>]<br></li>
<li>To protect itself , H2O2-stressed E. coli activates the OxyR regulon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21143317" target="_blank">21143317</a>]<br></li>
<li>A previous microarray study suggested that yaaA , an uncharacterized gene found in many bacteria , was induced by H2O2 in Escherichia coli as part of its OxyR regulon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21378183" target="_blank">21378183</a>]<br></li>
<li>YchF Expression Is Controlled by OxyR in Response to H2O2 -- The YchF-dependent inhibition of KatG in the presence of H2O2 would be detrimental for the cell , and therefore , we addressed how cells prevented this inhibition under physiological conditions , i.e. when ychF was not overexpressed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23139412" target="_blank">23139412</a>]<br></li>
<li>We now show that OxyR also inhibits ychF synthesis , but this inhibition is only observed after exposing cells to H2O2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23139412" target="_blank">23139412</a>]<br></li>
<li>At low H2O2 concentrations , repression of YchF by OxyR is relieved , and phosphorylation of YchF increases its affinity for KatG , resulting in its inhibition .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23139412" target="_blank">23139412</a>]<br></li>
<li>Since OxyR can be activated either by challenge with an oxidant such as H2O2 or directly by a change in the cellular thiol-disulfide status , we determined the effect of a change in the latter state of the cell on the expression of trxC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10644706" target="_blank">10644706</a>]<br></li>
<li>However , under the same conditions its expression was induced 10-fold in a catalase/peroxidase-de ﬁcient strain ( Hpx ) , which accumulates up to 1 M endogenous H2O2 and in which the OxyR response is fully activated ( 43 ) ( Fig. 2A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21378183" target="_blank">21378183</a>]<br></li>
<li>KatG is the major catalase during exponential-growth of E. coli , and its expression is induced by OxyR in response to H2O2 ( 26 , 27 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23139412" target="_blank">23139412</a>]<br></li>
<li>Although AhpCF and KatG expression is regulated by OxyR in an H2O2-dependent manner , the third catalase in E. coli , KatE , is also under the control of S and is induced during stationary phase ( 29 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23139412" target="_blank">23139412</a>]<br></li>
<li>HP-I is active as a tetramer of 81-kDa subunits ( 6 ) and is transcriptionally induced by OxyR as a part of a genetic response to H2O2 ( 35 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8990289" target="_blank">8990289</a>]<br></li>
<li>YchF Is Part of the OxyR Regulon -- The H2O2 response in most bacteria is coordinated by the LysR-type transcriptional-regulator OxyR ( 32 , 35 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23139412" target="_blank">23139412</a>]<br></li>
<li>These results clearly demonstrate that H2O2 stress conditions that are regularly used to induce the OxyR response ( e.g. , references 1 , 12 and 17 ) strongly activate the in vivo transcription of the SoxRS regulon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11073934" target="_blank">11073934</a>]<br></li>
<li>Thus , upon H2O2 exposure , OxyR activates the transcription of H2O2-scavenging enzymes like KatG and at the same time represses the transcription of the KatG inhibitor YchF ( Fig. 7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23139412" target="_blank">23139412</a>]<br></li>
<li>In the absence of H2O2 , the transcription factor OxyR is kept in its reduced ( red ) state by the glutaredoxin GlrX , which allows the basal expression of katG and ychF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23139412" target="_blank">23139412</a>]<br></li>
<li>YchF is negatively regulated by OxyR in response to H2O2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23139412" target="_blank">23139412</a>]<br></li>
<li>We furthermore showed that YchF expression is repressed by the transcription factor OxyR and further post-translationally modified in response to H2O2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23139412" target="_blank">23139412</a>]<br></li>
<li>YchF expression is negatively regulated by the transcription factor OxyR in response to H2O2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23139412" target="_blank">23139412</a>]<br></li>
<li>All three genes were strongly induced by H2O2 , but showed differences in induction by GSNO and NaNO ; grxA was 2 induced moderately by both compounds via OxyR and ahpC showed slight induction , whereas there was no detectable induction of katG .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14718666" target="_blank">14718666</a>]<br></li>
<li>Redox-cycling agents also activate the syntheses of proteins that are induced by H2O2 and controlled by OxyR , due to the spontaneous and superoxide dismutase-mediated conversion of O z2 2 to H2O2 ( 4 , 15 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11073934" target="_blank">11073934</a>]<br></li>
<li>Interestingly , while OxyR , LysR , NodD , IlvY , and MetR all negatively regulate their own genes , they positively regulate a transcript from an overlapping promoter on the complementary strand ( 29 , 36 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2471187" target="_blank">2471187</a>]<br></li>
<li>For example , a family of regulatory proteins such as OxyR , LysR , CysB , NodD , and MetR , that act as positive regulator for other genes , were shown to negatively regulate their own expression ( 30 , 32 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1715982" target="_blank">1715982</a>]<br></li>
<li>We also identiﬁed several genes , including uxuA , encoding mannonate hydrolase , whose expression might be repressed by OxyR , since their expression was elevated in the DoxyR mutant strain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11443091" target="_blank">11443091</a>]<br></li>
<li>To further explore the OxyR role in the responses to GSNO or NaNO2 and to compare these to the OxyR-mediated response to H2O2 , we also assayed the expression of three target genes by primer extension ( Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14718666" target="_blank">14718666</a>]<br></li>
<li>To determine whether INH acts to oxidize OxyR , leading to the induction of OxyR-regulated genes , the expression of the oxyS gene in response to INH was assayed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7986015" target="_blank">7986015</a>]<br></li>
<li>In this respect it should also be most interesting to learn where OxyR binding sites are located in promoters that are activated and how the constitutive mutant oxyR2 ( Christman et al. , 1985 ) , behaves with respect to mom gene expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2551682" target="_blank">2551682</a>]<br></li>
<li>During normal growth , OxyR also acts as a repressor , negatively autoregulating its own expression and the expression of the Mu phage mom gene ( 6 , 14 , 38 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7868602" target="_blank">7868602</a>]<br></li>
<li>OxyR , AraC , and MalT present three unique examples of how shifts in DNA contacts can lead to changes in gene expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8087856" target="_blank">8087856</a>]<br></li>
<li>This function would be similar to a function ascribed to the ferritinlike , nonspeciﬁc DNA binding protein Dps ( 13 ) , whose expression is also regulated by OxyR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10419964" target="_blank">10419964</a>]<br></li>
<li>To identify additional OxyR-regulated genes , I attempted to clone promoter DNA fragments whose expression is regulated by OxyR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9294462" target="_blank">9294462</a>]<br></li>
<li>In summary , the evidence presented here strongly suggests that Ag43 is an autotransporter whose phase-variable expression is transcriptionally regulated by DNA methylation and by OxyR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10094691" target="_blank">10094691</a>]<br></li>
<li>This suggested that the lack of peroxidase expression in these strains allows endogenous oxidants to accumulate , and this leads not only to constitutive OxyR activation but also to INH susceptibility .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7986015" target="_blank">7986015</a>]<br></li>
<li>Ascorbate activation of ahpC is dependent on OxyR Two lines of evidence confirm that the increased expression of ahpC observed in the presence of L-ascorbate is controlled by the activator OxyR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17906139" target="_blank">17906139</a>]<br></li>
<li>OxyR is an activator of katG and ahpCF expression and negatively regulates its own expression ( 10 , 13 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2183352" target="_blank">2183352</a>]<br></li>
<li>Eliminating OxyR function resulted in the abolition of the observed increase in trxC9-9lacZ expression ( Table II ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10644706" target="_blank">10644706</a>]<br></li>
<li>A mutation activating OxyR would cause increased expression of AHPR , which Table 2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7644465" target="_blank">7644465</a>]<br></li>
<li>Thus , a thiol-based modification by avicin activates OxyR , and DTT sensitivity indicates that the modification is reversible rather than thioether-based .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16030151" target="_blank">16030151</a>]<br></li>
<li>OxyR and SoxR activate fur expression upon H O or superoxide ( i.e. redox-cycling drugs ( 14 ) ) 2 2 treatment by 10-fold , leading to a 2-fold increase in Fur protein ( 39 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22223481" target="_blank">22223481</a>]<br></li>
<li>It regulates the expression of the superoxide dismutases ( repression of the manganese superoxide dismutase and activation of the iron one ) ( 22 , 23 ) , and fur expression is under control of OxyR and SoxRS ( 24 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12475930" target="_blank">12475930</a>]<br></li>
<li>Antioxidant defense systems are mediated through regulators ( OxyR , SoxRS , Fur and RpoS ) whose expression is most often induced by ROS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16183297" target="_blank">16183297</a>]<br></li>
<li>It is possible that the expression of other transcripts that are induced independently of OxyR is under the control of the SoxRS regulators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11443091" target="_blank">11443091</a>]<br></li>
<li>It is recognized that NO at low concentrations has a protective role for bacteria , as it activates transcription factors , such as SoxRS , OxyR , and Fur , whose regulons encode antioxidant enzymes ( 17 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22563051" target="_blank">22563051</a>]<br></li>
<li>To assess whether the OxyR and SoxRS induction of fur transcription leads to changes in Fur protein levels , we carried out immunoblot analysis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10419964" target="_blank">10419964</a>]<br></li>
<li>The induction of fur transcription by OxyR and SoxRS leads to an increase in Fur protein concentration .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10419964" target="_blank">10419964</a>]<br></li>
<li>UVA Adaptive Response - · O H O 2 2 2 SoxRS Fur OxyR RpoS ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16183297" target="_blank">16183297</a>]<br></li>
<li>OxyR - and SoxRS-independent induction of the isc operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11443091" target="_blank">11443091</a>]<br></li>
<li>Wei , Q. , Minh , P. N. , Dötsch , A. , Hildebrand , F. , Panmanee , W. , Elfarash , A. , Schulz , S. , Plaisance , S. , Charlier , D. , Hassett , D. , Häussler , S. , and Cornelis , P. ( 2012 ) Global regulation of gene expression by OxyR in an important human opportunistic pathogen .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23139412" target="_blank">23139412</a>]<br></li>
<li>However , given our previous ﬁndings that OxyR regulates the expression of the iron metabolism regulator Fur , possible OxyR regulation of a putative iron reductase was of interest .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11443092" target="_blank">11443092</a>]<br></li>
<li>Understanding OxyR , Fur , and MntR regulation of mntH should enable further regulatory questions to be addressed , particularly questions regarding the sensitivity of mntH expression to growth medium and growth state .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12029030" target="_blank">12029030</a>]<br></li>
<li>rpoS - , OxyR - , and IHF-dependent regulation of dps expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7559363" target="_blank">7559363</a>]<br></li>
<li>OxyR - , and IHP-dependent regulation of dps expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7715442" target="_blank">7715442</a>]<br></li>
<li>We observed that Fe2 and Mn2 still repressed mntH : : lacZ when OxyR regulation was inactivated by a cis - or trans-acting mutation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12029030" target="_blank">12029030</a>]<br></li>
<li>Redox regulation of OxyR requires speciﬁc disulﬁde bond formation involving a rapid kinetic reaction path .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21378183" target="_blank">21378183</a>]<br></li>
<li>Both OxyR and glutathione are also involved in the regulation of katG ( HPI ) [ 13,33 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16183297" target="_blank">16183297</a>]<br></li>
<li>To test whether INH , like peroxides , activates OxyR , the effect of INH on OxyR regulation was examined .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7986015" target="_blank">7986015</a>]<br></li>
<li>Therefore the regulation of d'ps by a ^ and OxyR is independent in vivo , and OxyR probably does not activate a ^ - holoenzyme .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7984106" target="_blank">7984106</a>]<br></li>
<li>RESULTS Mutant OxyR and agn43 regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12644490" target="_blank">12644490</a>]<br></li>
<li>the role of oxidized OxyR in agn43 regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12644490" target="_blank">12644490</a>]<br></li>
<li>Three mutants exhibited dual regulation by OxyR and RpoS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8990283" target="_blank">8990283</a>]<br></li>
<li>Noncanonical modes of regulation by OxyR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11443092" target="_blank">11443092</a>]<br></li>
<li>Results are expressed as percent induction by SNO-OxyR , normalized with respect to bla control ( n = 3 separate SNO-OxyR preparations , ± SEM ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22539721" target="_blank">22539721</a>]<br></li>
<li>Activation of OxyR by avicin in situ was demonstrated by examining the induction of HPI .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16030151" target="_blank">16030151</a>]<br></li>
<li>Thus , OxyR regulates S-nitrosylation in E. coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22539721" target="_blank">22539721</a>]<br></li>
<li>which is repressed by OxyR ( 11 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2183352" target="_blank">2183352</a>]<br></li>
<li>No response of OxyR was detected .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22016571" target="_blank">22016571</a>]<br></li>
<li>No OxyR was added to reactions shown in lanes 3 and 4 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12644490" target="_blank">12644490</a>]<br></li>
<li>Thus , neither OxyR nor SoxRS activates the oxyR gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9324269" target="_blank">9324269</a>]<br></li>
<li>oxyR Is Negatively Autoregulated by the Binding of OxyR to Its Own Promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2471187" target="_blank">2471187</a>]<br></li>
<li>This induction requires the positive activator OxyR .4 HPII , on the other hand , is induced in stationary phase and is rpoS-regulated .4 We demonstrate here that a non-growing Fur-deficient mutant displays about 30-fold lower HPII activity than the parental strain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14980071" target="_blank">14980071</a>]<br></li>
<li>These data suggested that OxyR activity varies continuously during growth , and this hypothesis was tested by monitoring the activity of another oxyR-regulated enzyme , glutathione reductase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8990289" target="_blank">8990289</a>]<br></li>
<li>As in the case of pHA233 , the b-galactosidase activity encoded by pHA2331 increased about threefold with H2O2 treatment in an oxyR-dependent fashion ( Fig. 1C and D ) , con-ﬁrming that transcription from the grx promoter is regulated by OxyR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9294462" target="_blank">9294462</a>]<br></li>
<li>Manganese is an essential cofactor of manganese-containing superoxide dismutase , but this enzyme is not co-regulated by OxyR , as one might expect it to be if the intended effect of mntH induction were to charge this enzyme .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19400769" target="_blank">19400769</a>]<br></li>
<li>The observation that the ahpC and ryhB genes , which are regulated primarily by OxyR and Fur , respectively , are induced by PMS ( Fig. 8 ) and menadione ( W. Yeo , unpublished result ) suggests that the superoxide generators do not necessarily exert their effects via superoxides only .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15009899" target="_blank">15009899</a>]<br></li>
<li>For comparison , we analysed the induction of ahpC transcripts , which are regulated primarily by OxyR , and the ryhB transcripts regulated by Fur ( Masse and Gottesman , 2002 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15009899" target="_blank">15009899</a>]<br></li>
<li>However , the only OxyR target genes , sufA and fhuF , showing 5-fold induction by GSNO or NaNO2 , are known to be regulated by both Fur and OxyR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14718666" target="_blank">14718666</a>]<br></li>
<li>Furthermore , these transcripts ( except dadX ) are regulated either by ArcA , FNR , or OxyR , possibly accounting for the increased abundance of these transcripts in the presence of O2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16857675" target="_blank">16857675</a>]<br></li>
<li>Interestingly , the mntH , yebN , and dps promoters are all thought be regulated by Fur and/or OxyR in addition to MntR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21239586" target="_blank">21239586</a>]<br></li>
<li>Since the hemF gene product is required for functional HPI catalase synthesis , it is not surprising that hemF and katG are regulated by OxyR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8990283" target="_blank">8990283</a>]<br></li>
<li>encoding HPI catalase , is regulated by both OxyR and a ^ ( Ivanova et ai , 1994 ; Mukhopadhyay and Schellhorn , 1994 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7984106" target="_blank">7984106</a>]<br></li>
<li>Glutathione , encoded by gorA , is regulated by OxyR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16183297" target="_blank">16183297</a>]<br></li>
<li>This suggests that ychF expression is regulated by OxyR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23139412" target="_blank">23139412</a>]<br></li>
<li>These changes may involve an increase in the number of bound protomers ( Schell , 1993 ) or ` translocation ' of the tetrameric protein from one binding site to another , observed as a change in position and length of the protein footprint on DNA , e.g. OxyR ( Toledano et al. , 1994 ) , CysB ( Hryniewicz and Kredich , 1994 ; 1995 ) and OccR ( Akakura and Winans , 2002a , b ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15255893" target="_blank">15255893</a>]<br></li>
<li>We therefore postulated that in this experiment , the bias to the OFF phase may have been obtained by an increase in the local subcellular concentration of free OxyR as a result of the displacement of OxyR by the passage of the replication fork .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20118257" target="_blank">20118257</a>]<br></li>
<li>In this study , we dissect the roles of the three 5 cents - GATC-3 cents sites in Ag43 phase switching by performing sitedirected mutagenesis of the target sequences in all possible combinations and in a manner that allows discrimination between their abilities to act as substrates for Dam-mediated methylation and in OxyR interactions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11972787" target="_blank">11972787</a>]<br></li>
<li>It will be interesting to determine whether Dam and OxyR are not only necessary , but also sufficient , to regulate phase variation of Ag43 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10692164" target="_blank">10692164</a>]<br></li>
<li>Once OxyR is bound , Dam can not access these target sequences , leading to preferred inheritance of the OFF phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24837285" target="_blank">24837285</a>]<br></li>
<li>Since puriﬁed OxyR protein speciﬁcally binds to upstream regulatory regions of the genes under its control ( 19 , 20 ) , I tried to clone DNA fragments which would speciﬁcally bind to OxyR for identiﬁcation of OxyR-regulated genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9294462" target="_blank">9294462</a>]<br></li>
<li>Based on the extensive nucleotide homology present in the regulatory regions agn43 and other genes ( oxyR and mom ) known to be negatively regulated by OxyR , one can predict an OxyR binding region upstream of the agn43 gene and extending between positions 295 and 249 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10094691" target="_blank">10094691</a>]<br></li>
<li>Regulation of the suf operon by oxidants 1751 Our study also suggests that OxyR that binds at the ORE-I site ( -236 to -197 ) , most likely exerts its activating effect on RNA polymerase via looping , and the IHF binding at ORE-II around -145 nt assists this long-range activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15009899" target="_blank">15009899</a>]<br></li>
<li>Whether OxyR and the ORE-III binding factor activate the sufA promoter complex in an additive way , or whether OxyR activation requires ORE-III binding factor needs further investigation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15009899" target="_blank">15009899</a>]<br></li>
<li>DAM and OxyR repressor compete for binding to sequences in the flu promoter , with OxyR binding resulting in repression and blocking of methylation , while DAM methylation is required for full gene activation [ 17 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18282299" target="_blank">18282299</a>]<br></li>
<li>In addition , this weak binding must allow for autorepression by the mutant proteins , since none of the constitutively active mutants showed elevated levels of OxyR protein by immuno-blot assays ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7868602" target="_blank">7868602</a>]<br></li>
<li>Even if it turns out that it is just a coincidence that OxyR binds to this operator and that this binding is affected by Dam methylation , it is fascinating to see how a phage has recruited a cellular protein and provided it with a novel function .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2551682" target="_blank">2551682</a>]<br></li>
<li>Because the DNA concentrations used in these experiments were < 5 X 10 '' ' M , and therefore protein was in excess and [ O X Y R ] ~ [ O ~ x ~ yRIf , , , , the concentration of OxyR required for half-maximal binding isvery close to the appare dissociation constant .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1730735" target="_blank">1730735</a>]<br></li>
<li>Previously , it was shown that OxyR is required for repression of the Ag43-encoding gene , agn43 , and that Dam-dependent methylation of three GATC target sequences in the regulatory region abrogates OxyR binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12029051" target="_blank">12029051</a>]<br></li>
<li>Conversely , OxyR binding to this region can occur when the GATC sequences are unmethylated and results in the OFF phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20118257" target="_blank">20118257</a>]<br></li>
<li>Thus , OxyR binding sequesters the agn43 GATC sequences from Dam when it is bound to the hemimethylated region as well as the unmethylated region ( 15 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20118257" target="_blank">20118257</a>]<br></li>
<li>As the nonmethylated state correlates with the OFF transcription phase , the data suggest that OxyR mediates agn repression by binding at this region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10692164" target="_blank">10692164</a>]<br></li>
<li>In the current model for Ag43 phase variation , OxyR represses agn43 transcription by binding to a region upstream of the coding sequence , resulting in the OFF phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12029051" target="_blank">12029051</a>]<br></li>
<li>Taken together , the data support a model in which the ON state for Ag43 expression is obtained by DNA methylationdependent abrogation of OxyR binding and in which the OFF phase is a result of repression mediated by binding of either the reduced or the oxidized form of OxyR to unmethylated agn43 DNA ( Fig. 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12644490" target="_blank">12644490</a>]<br></li>
<li>In summary , the data presented here indicate that the OFF phase is a result of OxyR ( C199S ) - dependent repression of a 70-dependent promoter upstream of the GATC-I sequence , which is mediated by OxyR binding at unmethylated agn43 DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12029051" target="_blank">12029051</a>]<br></li>
<li>This is in agreement with the previously proposed model , in which OxyR is a repressor of Ag43 transcription , and OxyR-mediated repression is alleviated by methylation of the GATC sequences ( Henderson and Owen , 1999 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10692164" target="_blank">10692164</a>]<br></li>
<li>Here , we examine whether binding of OxyR ( C199S ) mediates repression of Ag43 transcription in vivo by analysing transcription and the methylation state of the GATC sequ-ences in a background containing only OxyR ( C199S ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10692164" target="_blank">10692164</a>]<br></li>
<li>Thus , regulation of mom and agn have in common that methylation of the GATC sequences in the OxyR binding site abrogates OxyR binding and thereby alleviates repression of transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10692164" target="_blank">10692164</a>]<br></li>
<li>OxyR and Fur repression of fhuF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11443092" target="_blank">11443092</a>]<br></li>
<li>It is likely that OxyR binding to fhuF-2 also contributes to repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11443092" target="_blank">11443092</a>]<br></li>
<li>This is consistent with the fact that OxyR binding is Dam dependent and with the requirement of OxyR for repression ( 15 , 18 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12029051" target="_blank">12029051</a>]<br></li>
<li>Thus , mom and agn43 regulation have Dam-dependent binding of OxyR in common , but their mechanisms of activation and repression differ .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12029051" target="_blank">12029051</a>]<br></li>
<li>No OxyR binding sites were predicted upstream of ytfK , ydcH , ydeN , and yaeH , so these genes may or may not be repressed directly by OxyR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11443091" target="_blank">11443091</a>]<br></li>
<li>pDGK261 , pDGK262 , and pDGK263 are identical to pMLZ104 except for block substitution mutations in the OxyR - , Fur - , and MntR-binding motifs , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12029030" target="_blank">12029030</a>]<br></li>
<li>We have performed this by mutational inactivation in a manner unlikely to affect OxyR binding , namely by A-to-T , G-to-C and C-to-G mutations cents cents in 5 - GATC-3 sites 1 -- 3 respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11972787" target="_blank">11972787</a>]<br></li>
<li>Thus , Dam could affect fimB expression by inhibiting NanR and NagC binding to their cognate sites as it can do for Lrp and OxyR ( 49 , 51 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15534208" target="_blank">15534208</a>]<br></li>
<li>Thus , we propose that the auxiliary binding sites function as a synthetic OxyR sink that increases the OxyR concentration locally and thereby biases the competition between OxyR and Dam to favor OxyR binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20118257" target="_blank">20118257</a>]<br></li>
<li>One model to explain this effect is that the auxiliary OxyR binding sites allow novel interactions between distant OxyR tetramers and stabilize the interaction at agn43 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20118257" target="_blank">20118257</a>]<br></li>
<li>The effect of this artiﬁcial system of auxiliary OxyR binding sites can be compared to that occurring naturally for lac regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20118257" target="_blank">20118257</a>]<br></li>
<li>Binding of OxyR prevents Dam from accessing the target sequences in the binding site , allowing them to remain unmethylated throughout the cell cycle , even in the presence of Dam ( 7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12644490" target="_blank">12644490</a>]<br></li>
<li>RESULTS Affinity of OxyR for Its Functional Binding Sites-To increase the pool of information on the OxyR recognition sequences , we localized and sequenced an additional OxyR-binding site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1730735" target="_blank">1730735</a>]<br></li>
<li>Although the new binding site was near dsbG , Northern blotting and primer extension assays showed that OxyR binding to the dsbG-proximal site led to the induction of a second ahpCF transcript , while OxyR binding to the ahpCF-proximal site leads to the induction of both dsbG and ahpC transcripts .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11443092" target="_blank">11443092</a>]<br></li>
<li>Our discovery of a binding site proximal to the dsbG gene led us to suggest that OxyR binding to this site would lead to OxyR regulation of dsbG .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11443092" target="_blank">11443092</a>]<br></li>
<li>OxyR binding upstream of dsbG .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11443092" target="_blank">11443092</a>]<br></li>
<li>OxyR binding upstream of dsbG activates ahpCF expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11443092" target="_blank">11443092</a>]<br></li>
<li>This result suggested that OxyR binding to the site adjacent to dsbG was functioning to activate expression of a transcript encoding ahpC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11443092" target="_blank">11443092</a>]<br></li>
<li>Unexpectedly , we found that OxyR binding to the dsbG-proximal site regulates the expression of the divergent ahpC transcript , while OxyR binding to the ahpC-proximal site regulates expression of the dsbG gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11443092" target="_blank">11443092</a>]<br></li>
<li>In the presence of hydrogen peroxide , mntH is activated by OxyR , acting through an OxyR-binding motif centered just upstream of the likely 35 RNA polymerase-binding site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12029030" target="_blank">12029030</a>]<br></li>
<li>One interesting prediction of the Fur binding model was the presence of two clusters of Fur binding sites that overlapped the OxyR binding sites in the fhuF promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11443092" target="_blank">11443092</a>]<br></li>
<li>We conﬁrmed the presence of OxyR binding to the three sites by in vitro DNase I footprinting assays .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11443092" target="_blank">11443092</a>]<br></li>
<li>To further elucidate the role of OxyR binding to the site upstream of dsbG , we examined RNA polymerase binding in the presence of OxyR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11443092" target="_blank">11443092</a>]<br></li>
<li>RNA polymerase binding was observed in the presence of OxyR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11443092" target="_blank">11443092</a>]<br></li>
<li>This suggests that the ORE-I binding factor , most probably OxyR , exerts its activating role only in the presence of IHF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15009899" target="_blank">15009899</a>]<br></li>
<li>This result indicated that the presence of a specific sequence at the fifth contact point was critical for reduced OxyR binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8087856" target="_blank">8087856</a>]<br></li>
<li>Our investigation of OxyR binding has led us to propose that , by remodeling its DNA contacts in response to changes in the redox state of the cell , OxyR can impose opposite regulatory effects on the divergent oxyR and oxySpromoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8087856" target="_blank">8087856</a>]<br></li>
<li>In that case the HM-DNA is formed before the OxyR is dislodged and becomes available ( Fig. 4B and C ) , increasing the chance that Dam accesses the DNA before OxyR and resulting in a maintenance of the on phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20118257" target="_blank">20118257</a>]<br></li>
<li>When DNA replication of the fully methylated sequence in an on-phase cell generates HM-DNA , an opportunity arises to change the expression phase : Dam access is required to maintain the ON phase , whereas OxyR binding can be the ﬁrst step to switch to the OFF phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20118257" target="_blank">20118257</a>]<br></li>
<li>Furthermore , OxyR titration as a result of genome rep-lication is unlikely to be signiﬁcant for phase variation even though the OxyR/HM-DNA complex was not stable in the presence of competitor OxyR binding sites ( Fig. 2b ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20118257" target="_blank">20118257</a>]<br></li>
<li>This suggests that this OxyR binding to HM-DNA is stable even in the presence of Dam and therefore could be maintained for a generation and thereby could initiate a switch to the off phase in an on-phase cell .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20118257" target="_blank">20118257</a>]<br></li>
<li>Importantly , our ﬁndings regarding stability in the presence of Dam-SAM support a model in which OxyR binding to HM-DNA in replicating ON-phase cells initiates a switch to the OFF phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20118257" target="_blank">20118257</a>]<br></li>
<li>Expression of agn43 -- lacZ was restored to its full level in an oxyR mutant , indicating that the low level of expression in a wild-type background resulted from binding of singly methylated DNA by OxyR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11972787" target="_blank">11972787</a>]<br></li>
<li>Binding of OxyR protein to mutated agn43 DNA Unmethylated DNA was ampliﬁed from the wild type and each mutant and examined for binding by OxyR protein ( Fig. 4A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11972787" target="_blank">11972787</a>]<br></li>
<li>Indeed , OxyR binds wild-type hemimethylated agn43 DNA , but with a decreased afﬁnity compared to unmethylated DNA ( 10 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12029051" target="_blank">12029051</a>]<br></li>
<li>OxyR in complex with agn43 unmethylated DNA ( UM-DNA ) was more stable in the presence of an excess of HM-DNA ( Fig. 2c ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20118257" target="_blank">20118257</a>]<br></li>
<li>A decrease in concentration of OxyR , like a decrease in afﬁnity , will affect the competition with Dam for the agn43 binding site , which leads to decreased OxyR binding , and thus results in an increased incidence of the ON phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12644490" target="_blank">12644490</a>]<br></li>
<li>These in vitro analyses indicate that SeqA bound to agn43 DNA is displaced rapidly by both Dam and OxyR and thus suggest that SeqA binding at agn43 HM-DNA will not bias the competition between Dam and OxyR for this substrate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20118257" target="_blank">20118257</a>]<br></li>
<li>However , these differences in repression at the population level can also be the result of differences in the binding afﬁnity of the OxyR proteins for agn43 DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12644490" target="_blank">12644490</a>]<br></li>
<li>Our data suggest that the OxyR-agn43 contacts are different but that the afﬁnity is nevertheless relatively high , which supports these authors ' conclusion that DNA binding of OxyR and OxyR-dependent regulation are versatile ( 6 , 39 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12644490" target="_blank">12644490</a>]<br></li>
<li>Certainly , there can be very little doubt from the combined results presented here ( in particular , the locked ON phenotypes displayed by pDEW11 -- 16 in both wildtype and dam backgrounds , the absence of OxyR binding to methylated/unmethylated DNA from these constructs and the presence of a consensus OxyR binding site within the agn43 regulatory region ) that OxyR is capable of binding to the agn43 promoter and that the adenine residues of 5 cents - GATC-3 cents sites 2 and 3 are critically involved in OxyR recognition .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11972787" target="_blank">11972787</a>]<br></li>
<li>It is therefore likely that bound OxyR exerts its regulatory effect by blocking binding of RNA polymerase to the agn43 promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11972787" target="_blank">11972787</a>]<br></li>
<li>Since IHF has also been shown to cause a strong bend upon binding DNA and both regulators bind to unusually long sequences , the combined binding of OxyR and IHF may lead to an unusual promoter structure .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7984106" target="_blank">7984106</a>]<br></li>
<li>Since the auto-regulatory binding site does not contain any Dam sites , it is obvious that the ability of MomR/OxyR to discriminate between methylated and unmethylated DNA , as in the mom promoter , appears not to be needed for its cellular function .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2551682" target="_blank">2551682</a>]<br></li>
<li>Binding of OxyR to the hcp promoter was verified by el ectrophoretic mobility shift assay , and the interaction was inhibited by DNA containing the katG OxyR binding site ( fig .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22539721" target="_blank">22539721</a>]<br></li>
<li>Fig. 3 shows that very high concentrations of nonspecific DNA are required to compete for the binding of OxyR to itsfunctional binding site on the oxyR promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1730735" target="_blank">1730735</a>]<br></li>
<li>These proteins carry helix-turn-helixDNA-binding domains similar to those found in regulators that bind to shorter DNA sequences , raising the question as to why OxyR and other LysR-type regulators bind to such extended sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8087856" target="_blank">8087856</a>]<br></li>
<li>Electrophoresis Mobility Shift Assays DNA binding of OxyR was detected by the GelshiftTM EMSA kit ( Active Motif , La Hulpe , Belgium ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23139412" target="_blank">23139412</a>]<br></li>
<li>This was expected , since methylation can not take place , and therefore OxyR binding can not be abrogated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12029051" target="_blank">12029051</a>]<br></li>
<li>This buffer is close to optimal for SeqA binding and still allows OxyR binding and DNA methylation to occur .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20118257" target="_blank">20118257</a>]<br></li>
<li>This could be attributed to a low level of binding of OxyR to methylated template in the transcription buffer , which lacks nonspeciﬁc competitor DNA , as was observed by EMSA with 8 nM OxyR ( C199S ) ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12029051" target="_blank">12029051</a>]<br></li>
<li>This suggests that a similar meth-ylation pattern is required to allow transcription of the agn43 gene and that all sites have to be methylated to prevent binding of OxyR to the agn43 promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10094691" target="_blank">10094691</a>]<br></li>
<li>This protection is not seen with extracts made from strains that do not overproduce OxyR and suggests that the mechanism by which OxyR negatively autoregulates its is and blocking , expression by binding to , therefore the DNA that are for its own sequences required transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2471187" target="_blank">2471187</a>]<br></li>
<li>This ﬁnding raises questions as to the role of OxyR binding to coding sequences and whether other transcription factors bind to coding sequences .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11443092" target="_blank">11443092</a>]<br></li>
<li>In contrast , no shift was observed when methylated DNA was used as a probe , even when using 10-fold more OxyR than was required to shift equal amounts of non-methylated DNA fully ( Fig. 3B and C ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10692164" target="_blank">10692164</a>]<br></li>
<li>Similar to the results obtained with OxyR ( WT ) , OxyR ( C199S ) bound to nonmethylated agn DNA , but no shift was observed with methylated DNA ( Fig. 3C ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10692164" target="_blank">10692164</a>]<br></li>
<li>However , neither the presence of a low level of reduced OxyR nor the possibility that DNA methylation was slightly affected can be fully excluded .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12644490" target="_blank">12644490</a>]<br></li>
<li>Only DNA ampliﬁed from those constructs with substitutions of the adenines in the second or third 5 cents - GATC-3 cents sites ( pDEW11 -- pDEW16 ) remained unshifted in the presence of OxyR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11972787" target="_blank">11972787</a>]<br></li>
<li>Methylated DNA from plasmids pAg43 , pDEW10 -- pDEW16 , pDEW20 and pDEW21 remained unshifted in the presence of OxyR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11972787" target="_blank">11972787</a>]<br></li>
<li>However , DNA was present in a single retarded band with nonmethylated DNA as the probe and extract from induced cultures of MV239 containing OxyR ( WT ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10692164" target="_blank">10692164</a>]<br></li>
<li>Binding of OxyR to biotinylated DNA was detected by chemiluminescence using horseradish peroxidase-coupled streptavidin .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23139412" target="_blank">23139412</a>]<br></li>
<li>Binding of OxyR to biotinylated DNA was detected via streptavidin-conjugated horseradish peroxidase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23139412" target="_blank">23139412</a>]<br></li>
<li>Unmethylated DNA from the locked OFF triple mutant pDEW25 was also bound by OxyR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11972787" target="_blank">11972787</a>]<br></li>
<li>There is , however , no pre-cedence for this type of interaction for OxyR , even though the interaction between OxyR multimers bound to DNA remains to be fully resolved ( 28 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20118257" target="_blank">20118257</a>]<br></li>
<li>In contrast , pDEW13 and pDEW14 became locked ON in a state of relatively low expression , suggesting that , when only one 5 cents - GATC-3 cents site was intact , OxyR was able to-bind , albeit with poor afﬁnity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11972787" target="_blank">11972787</a>]<br></li>
<li>When three GATC sequences in the agn43 regulatory region are unmethylated , OxyR can bind and repress transcription , resulting in the OFF phase , but , when these GATC sequences are methylated , OxyR does not bind and expression is ON ( 12 , 17 , 18 , 36 , 37 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12644490" target="_blank">12644490</a>]<br></li>
<li>However , OxyR will not bind mom DNA or repress transcription when the GATC sequences are fully methylated ( BoÈlker and Kahmann , 1989 ; Sun and Hattman , 1996 ; 1998 ; Hattman and Sun , 1997 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10692164" target="_blank">10692164</a>]<br></li>
<li>This showed that OxyR can bind to agn43 DNA that is methylated at any of the three 5 cents - GATC-3 cents sites singly .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11972787" target="_blank">11972787</a>]<br></li>
<li>This mutant is locked in the reduced conformation , eliminating the need for the high concentrations of DTT that are required to maintain reduced OxyR in vitro but which interfere with the hydroxyl radical reaction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8087856" target="_blank">8087856</a>]<br></li>
<li>Since uxuB and gntP also showed higher induction ratios in the absence of OxyR , it is intriguing to speculate that the two divergent promoters are repressed by oxidized OxyR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11443091" target="_blank">11443091</a>]<br></li>
<li>Given this observation , it seems contradictory that the suf operon is induced by oxidized OxyR while fhuF is repressed by oxidized OxyR ( 41 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11443091" target="_blank">11443091</a>]<br></li>
<li>Both forms of the OxyR protein can bind to specific sites of the target genes , but only the oxidized form can act as an activator ( Storz et ai .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8483417" target="_blank">8483417</a>]<br></li>
<li>Puriﬁed , wild-type OxyR is in the oxidized form , since it was puriﬁed under aerobic conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12644490" target="_blank">12644490</a>]<br></li>
<li>These authors proposed that ﬁmbrial production leads to a change in the environmental redox potential that would result in the conversion of cytoplasmic OxyR to the oxidized form .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12644490" target="_blank">12644490</a>]<br></li>
<li>Our experiments have shown that OxyR ( C199S ) , which can not be oxidized , is sufficient to obtain OxyR-mediated repression when overexpressed , and our data suggest that OxyR may also be involved in activation indirectly .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10692164" target="_blank">10692164</a>]<br></li>
<li>Our in vivo data show that repression of agn43 occurs when the cellular pool of OxyR is mainly in the oxidized form , indicating that the oxidized form of OxyR is functioning as a repressor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12644490" target="_blank">12644490</a>]<br></li>
<li>Alternatively , oxidation of OxyR resulted in abrogation of repression , but a lack of rapid methylation allowed OxyR to rebind after being converted back to the reduced form .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12644490" target="_blank">12644490</a>]<br></li>
<li>OxyR binds to the Ag43 regulatory region in vitro , and Dam-dependent methylation reduces the affinity The data described above show that OxyR is required for transcriptional repression and for protection from Dam-dependent methylation of the GATC sequences in agn .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10692164" target="_blank">10692164</a>]<br></li>
<li>The data obtained with MV307 therefore confirm that OxyR is required for transcriptional repression and for protecting the GATC sequences from Dam-dependent methylation , but do not address the role of the reduced versus the oxidized form in agn repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10692164" target="_blank">10692164</a>]<br></li>
<li>In contrast , a more obvious difference in mobility was seen for ( uninduced ) OxyRC199S binding to the oxyRSpromoter probes , suggesting that reduced OxyR also induces a bend in the oxyRSsite that is relaxed upon oxidation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8087856" target="_blank">8087856</a>]<br></li>
<li>Also , the difference in binding activity 2040 Degenerate DNA between reduced ( inactive ) and oxidized ( active ) OxyR wasof interest since increased binding affinity for its recognition sequences upon activation is a potential level of regulation of the OxyR-stimulated promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1730735" target="_blank">1730735</a>]<br></li>
<li>Therefore , the conversion of reduced OxyR to a transcriptional-activator upon oxidation is not likely due to significantly increased DNA-binding by the oxidized form of OxyR , but is probably due to a conformational change in OxyR already bound to the promoter regions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2183352" target="_blank">2183352</a>]<br></li>
<li>Hence , structural and genetic studies of the differences between the oxidized and reduced forms of OxyR should provide important insights into transcriptional activation in general .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2183352" target="_blank">2183352</a>]<br></li>
<li>In contrast to the activation of DNA binding observed in RclR , however , OxyR and SoxR bind to DNA in both their reduced and oxidized forms , with oxidation resulting in structural changes that lead to transcriptional activation ( 47 , 61 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24078635" target="_blank">24078635</a>]<br></li>
<li>Results Phase variation of agn is regulated at the level of transcription and requires OxyR and Dam In order to examine the regulation of Ag43 expression , we constructed a transcriptional fusion of the upstream region of agn43 , referred to here as agn , to lacZ .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10692164" target="_blank">10692164</a>]<br></li>
<li>Our conclusion that oxidized OxyR as well as reduced OxyR can repress agn43 transcription has implications for a model proposed by Schembri et al. , which addresses the coordinated regulation between ﬁmbrial production and Ag43 expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12644490" target="_blank">12644490</a>]<br></li>
<li>These results suggest that OxyR , which is mainly in the oxidized form , was bound and repressed agn43 transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12644490" target="_blank">12644490</a>]<br></li>
<li>In contrast , our results indicate that oxidized OxyR can efﬁciently repress agn43 transcription , which suggests that the coordinate regulation is not mediated by the differential oxidation state of OxyR at the level of agn43 transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12644490" target="_blank">12644490</a>]<br></li>
<li>To determine whether the observed repression in the OFF cells can be mediated by oxidized OxyR , we examined in vitro the interactions of oxidized OxyR with agn43 regulatory region DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12644490" target="_blank">12644490</a>]<br></li>
<li>We previously reported that the reduced form of OxyR is sufﬁcient for repression of transcription of agn43 from unmethylated template DNA , which is essential for deoxyadenosine methylase ( Dam ) - and OxyR-dependent phase variation of agn43 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12644490" target="_blank">12644490</a>]<br></li>
<li>In vitro transcription from an unmethylated template was repressed by OxyR ( C199S ) , which resembles the reduced form of OxyR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12029051" target="_blank">12029051</a>]<br></li>
<li>OxyR ( C199S ) resembles the reduced form of OxyR in DNA binding and regulatory properties , binds unmethylated agn43 DNA in vitro , and is sufﬁcient to repress agn43 transcription in vivo ( 15 , 42 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12029051" target="_blank">12029051</a>]<br></li>
<li>It was previously shown that oxidized OxyR binds to unmethylated agn43 DNA in vitro ( 7 , 12 ) , but its role in agn43 regulation was not addressed further .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12644490" target="_blank">12644490</a>]<br></li>
<li>Neither the oxidized nor reduced form of OxyR could bind to the methylated agn43 template .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12644490" target="_blank">12644490</a>]<br></li>
<li>The results described above show that the oxidized form of OxyR binds to unmethylated agn43 DNA and represses agn43 transcription in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12644490" target="_blank">12644490</a>]<br></li>
<li>The afﬁnity of oxidized OxyR for unmethylated and hemimethylated agn43 DNA is the same as the afﬁnity of OxyR ( C199S ) , which behaves like the reduced form ( 12 , 20 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12644490" target="_blank">12644490</a>]<br></li>
<li>In vitro , both oxidized OxyR and OxyR ( C199S ) , which is locked in the reduced conformation , bound to the agn regulatory region , but methylation of the three GATC sequences abrogated this binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10692164" target="_blank">10692164</a>]<br></li>
<li>As aerobically prepared OxyR ( WT ) is in the oxidized state and OxyR ( C199S ) is locked in the reduced conformation , these data indicate that both the oxidized and the reduced forms of OxyR bind to non-methylated agn regulatory region DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10692164" target="_blank">10692164</a>]<br></li>
<li>Thus , as shown in Figure 7B , a site that was normally bound by both reduced and oxidized OxyR was converted to a site that was only bound tightly by oxidized OxyR , solely by the substitution of 2 bp at position 5 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8087856" target="_blank">8087856</a>]<br></li>
<li>Most members of the OxyR regulon are regulated as a result of binding of either the reduced or the oxidized form of OxyR ( 40 , 49 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12029051" target="_blank">12029051</a>]<br></li>
<li>In recent studies on the DNA binding properties of OxyR , we found that the reduced and oxidized ( activated ) forms of OxyR require different DNA contacts for binding ( 41 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7868602" target="_blank">7868602</a>]<br></li>
<li>In support of this conclusion , the footprints of oxidized and reduced OxyR are different .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2183352" target="_blank">2183352</a>]<br></li>
<li>In contrast , both the oxidized and reduced forms of OxyR can function as repressors in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8087856" target="_blank">8087856</a>]<br></li>
				</div></ul><br>
			<label class="collapse_DOM" for="_2"><font size="5px">OxyR protein domains</font></label>
				<input id="_2" type="checkbox">
				<div class="DOM"><br><ul><li>To assess the potential localization of the Met residues in HypT , we performed a secondary structure and disorder prediction and aligned the HypT sequence with that of the LysR-type transcription factor OxyR from E. coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23690622" target="_blank">23690622</a>]<br></li>
<li>Hydrogen peroxide activates the transcription factor OxyR through the oxidation of two cysteines and formation of an intramolecular disulﬁde bond ( 34 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10419964" target="_blank">10419964</a>]<br></li>
<li>Activation of the OxyR transcrip ¬ tion factor by reversible disulfide bond formation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15672380" target="_blank">15672380</a>]<br></li>
				<label class="DOM_collapse_similarity" for="dom_similarity_4">
G. ( 1998 ) Activation of the OxyR transcription factor by reversible disulﬁde bond formation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15009899" target="_blank">15009899</a>]<br></label>
<input id="dom_similarity_4", type="checkbox">
<div class="DOM_similar">
<li>Activation of the OxyR transcription factor by reversible disulﬁde bond formation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10094691" target="_blank">10094691</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11741843" target="_blank">11741843</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11073934" target="_blank">11073934</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11443092" target="_blank">11443092</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10419964" target="_blank">10419964</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21378183" target="_blank">21378183</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12644490" target="_blank">12644490</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11443091" target="_blank">11443091</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16804186" target="_blank">16804186</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9922271" target="_blank">9922271</a>]<br></li></div>
<li>They are involved in processes as diverse as respiration , gene regulation , cofactor biosynthesis , and DNA-RNA metabolism .7 There is tremendous interest in understanding the mechanisms of Fe-S cluster biosynthesis , one reason being that several human diseases , including Friedreich 's ataxia , are the result of genetic defects in the protein machinery that makes Fe-S clusters .7 10.1021 / pr100920r 2010 American Chemical Society 6740 Journal of Proteome Research 2010 , 9 , 6740 -- 6751 Published on Web 10/11/201 The SUF system of E. coli is comprised of complex protein machinery , which , under the control of OxyR , IHF , IscR , andFur , synthesizes Fe S clusters during the potentially cytotoxic conditions of iron limitation and oxidative stress.6,11-14 ,17,18 The SUF system is encoded by the sufABCDSE operon .14 SufS is a cysteine desulfurase that mobilizes inorganic sulfur via a pyridoxal-5 ′ - phosphate cofactor and a conserved cysteine ( C ) residue and transfers the sulfur atom through a proposed persulﬁde transfer mechanism to the conserved C51 residue on the sulfur acceptor protein SufE.19-21 SufB , SufC , and SufD form a complex ( SufBC2D , hereafter referred to as SufBCD ) 22 that stimulates this cysteine desulfurase activity through an unknown mechanism ,21 though SufC is known to be an atypical ABC transporter ATPase .12 In vitro evidence suggests that SufE transfers the sulfur atom to SufB within the SufBCD complex and that SufBCD acts as a scaffold upon which an Fe-S cluster is assembled.22-24 It is not known whether the persulﬁde sulfur on SufE is liberated via nucleophilic attack by the recipient Cys thiol on the terminal sulfur of the persulﬁde , which would result in persulﬁde transfer , or via nucleophilic attack on the bridging sulfur , which would produce a disulﬁde-linked complex between SufE and the recipient protein and liberate sulﬁde that may then bind a free Cys residue for use in assembling an Fe-S cluster .5 It also remains to be determined whether the assembled cluster on SufBCD is transferred to a target apoprotein directly or via SufA.22,25-27 It is possible that SufE transfers sulfur to proteins other than SufB , as both the iron source and the means by which the sulfur on SufE is incorporated into an Fe-S cluster remain unknown .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20936830" target="_blank">20936830</a>]<br></li>
<li>YchF Does Not Influence the Cellular Concentrations of KatG and OxyR -- Based on the presence of a putative nucleic-acid-binding motif within YchF ( 2 , 4 ) and its ability to bind to E. coli ribosomes ( 3 ) , it appeared likely that ychF overexpression influenced the expression of oxidative stress response proteins like OxyR and the catalase KatG ( 26 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23139412" target="_blank">23139412</a>]<br></li>
				<label class="DOM_collapse_similarity" for="dom_similarity_8">
Mutational analysis of the redox-sensitive transcriptional regulator OxyR : regions important for DNA-binding and multimerization .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10094691" target="_blank">10094691</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11073934" target="_blank">11073934</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7868602" target="_blank">7868602</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22904281" target="_blank">22904281</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8990283" target="_blank">8990283</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10648536" target="_blank">10648536</a>]<br></label>
<input id="dom_similarity_8", type="checkbox">
<div class="DOM_similar">
<li>Kullik , I. , Stevens , J. , Toledano , M.B. and Storz , G. ( 1995 ) Mutational analysis of the redox-sensitive transcriptional regulator OxyR : regions important for DNA-binding and multimerization .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9312050" target="_blank">9312050</a>]<br></li>
<li>Kullik , I. , Stevens , J. , Toledano , M.B. , and Storz , G. ( 1995a ) Mutational analysis of the redox-sensitive transcriptional regulator OxyR : regions important for DNA-binding and multimerization .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15255893" target="_blank">15255893</a>]<br></li></div>
<li>Mutational analysis of the redox-sensitive transcriptional regulator OxyR : regions important for oxidation and transcriptional activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20118257" target="_blank">20118257</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10094691" target="_blank">10094691</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12029051" target="_blank">12029051</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17464064" target="_blank">17464064</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11443092" target="_blank">11443092</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22904281" target="_blank">22904281</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21378183" target="_blank">21378183</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10648536" target="_blank">10648536</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12644490" target="_blank">12644490</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8626295" target="_blank">8626295</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8990283" target="_blank">8990283</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25266384" target="_blank">25266384</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10400599" target="_blank">10400599</a>]<br></li>
				<label class="DOM_collapse_similarity" for="dom_similarity_12">
J Bacteriol 174:7104 -- 7111 Kullik I , Toledano MB , Tartaglia LA , Storz G ( 1995 ) Mutational analysis of the redox-sensitive transcriptional regulator OxyR : regions important for oxidation and transcriptional activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8588737" target="_blank">8588737</a>]<br></label>
<input id="dom_similarity_12", type="checkbox">
<div class="DOM_similar">
<li>[ PubMed : 17289666 ] Kullik I , Toledano MB , Tartaglia LA , Storz G. Mutational analysis of the redox-sensitive transcriptional regulator OxyR : regions important for oxidation and transcriptional activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21143317" target="_blank">21143317</a>]<br></li>
<li>Kullik , I. , Toledano , M.B. , Tartaglia , L.A. , and Storz , G. ( 1995b ) Mutational analysis of the redox-sensitive transcriptional regulator OxyR : regions important for oxidation and transcriptional activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15255893" target="_blank">15255893</a>]<br></li>
<li>Kullik , I. , Toledano , M.B. , Tartaglia , L.A. , and Storz , G. ( 1995 ) Mutational analysis of the redox-sensitive transcriptional regulator OxyR : regions important for oxidation and transcriptional activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10564497" target="_blank">10564497</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11972787" target="_blank">11972787</a>]<br></li>
<li>Kullik , I. , Toledano , M.B. , Tartaglia , L.A. and Storz , G. ( 1995 ) Mutational analysis of the redox-sensitive transcriptional regulator OxyR : regions important for oxidation and transcriptional activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7556074" target="_blank">7556074</a>]<br></li>
<li>Kullik , I. , Toledano , M.B. , Tartaglis , L.A. , and Storz , G. ( 1995 ) Mutational analysis of the redox-sensitive transcriptional regulator OxyR : regions important for oxidation and transcriptional activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10692164" target="_blank">10692164</a>]<br></li>
<li>Kullik , I. , Toledano , M.B. , Tartagelia , L.A. and Storz , G. ( 1995 ) Mutational analysis of the redox-sensitive transcriptional regulator OxyR : regions important for oxidation and transcriptional activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11407111" target="_blank">11407111</a>]<br></li></div>
<li>It is interesting to note that NorR , Fur , SoxR , and OxyR all appear to contain redox-active cysteines and , in some cases , iron .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14718666" target="_blank">14718666</a>]<br></li>
<li>NO also has been shown to react directly with the 2Fe -- 2S centers of the E. coli SoxR dimer to generate a dinitrosyl -- iron -- dithiol form of SoxR ( 51 ) , and as described above , GSNO can react with the redox-active C199 residue in OxyR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14718666" target="_blank">14718666</a>]<br></li>
<li>Methylation interference experiments show that themajority of important close contacts made by the OxyR protein are not atpositions of absolute homology but rather areat positions of degenerate homology .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1730735" target="_blank">1730735</a>]<br></li>
<li>To elucidate the nature of the function-regulating interaction of avicins with cysteine residues , we explored their effects on a well characterized redox-regulated response that is controlled by the Escherichia coli transcriptional activator OxyR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16030151" target="_blank">16030151</a>]<br></li>
<li>It is known that these responses depend on select transcription factors ( e.g. OxyR , SoxR , PerR ) , which are able to specifically sense particular oxidants , often through the oxidation state of conserved cysteine residues .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24078635" target="_blank">24078635</a>]<br></li>
<li>Like OxyR , YchF contains several cysteine residues , and their oxidation by H2O2 could influence the ATPase activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23139412" target="_blank">23139412</a>]<br></li>
<li>OxyR responds to the presence of H2O2 and appears to be sensitive to oxidation and reduction at a single cysteine residue .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8588737" target="_blank">8588737</a>]<br></li>
<li>Conversion between the two forms of OxyR is rapid and reversible .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2183352" target="_blank">2183352</a>]<br></li>
<li>This feature may be important for OxyR function .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8087856" target="_blank">8087856</a>]<br></li>
<li>OxyR MnSOD sodA HPI katG ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15878120" target="_blank">15878120</a>]<br></li>
<li>Although the redox-active Cys ¬ 199 residue in OxyR probably can undergo modification to S-NO , this form of the protein is at best capable of limited activation of a subset of promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14718666" target="_blank">14718666</a>]<br></li>
<li>In the case of OxyR transcription factor , the Cys -- Cys pair is involved in redox sensing ( 45 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20156994" target="_blank">20156994</a>]<br></li>
<li>This model resembles the mechanism of redox activation of OxyR , in which initial oxidation of Cys-199 precedes formation of a Cys-199 -- Cys-208 disulfide bond in the fully activated protein ( 57 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24078635" target="_blank">24078635</a>]<br></li>
<li>We have shown that OxyR activation by H O is caused by disulfide bond formation 2 2 between residues Cys-199 and Cys-208 ( 28 , 29 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14718666" target="_blank">14718666</a>]<br></li>
<li>Whether OxyR and the ORE-III binding factor activate the sufA promoter complex in an additive way , or whether OxyR activation requires ORE-III binding factor needs further investigation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15009899" target="_blank">15009899</a>]<br></li>
<li>Hence , structural and genetic studies of the differences between the oxidized and reduced forms of OxyR should provide important insights into transcriptional activation in general .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2183352" target="_blank">2183352</a>]<br></li>
<li>In contrast to the activation of DNA-binding observed in RclR , however , OxyR and SoxR bind to DNA in both their reduced and oxidized forms , with oxidation resulting in structural changes that lead to transcriptional activation ( 47 , 61 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24078635" target="_blank">24078635</a>]<br></li>
<li>The formation of H2O2 in our cultures , as indicated above , guarantees the required activation of OxyR through oxidation of this transcriptional factor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17906139" target="_blank">17906139</a>]<br></li>
<li>OxyR mediates H2O2 activation of mntH .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12029030" target="_blank">12029030</a>]<br></li>
<li>OxyR activation in peroxidase-deficient strains .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7986015" target="_blank">7986015</a>]<br></li>
<li>To further characterize OxyR activation after oxidative stress , we purified the OxyR protein to homogeneity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2183352" target="_blank">2183352</a>]<br></li>
<li>Here and in the accompanying paper ( 20 ) , we identify regions of the OxyR protein critical for activation and repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7868602" target="_blank">7868602</a>]<br></li>
<li>One could speculate that , if HypT and OxyR are indeed structurally similar , oxidation-mediated conformational rearrangements in HypT alter contacts within the dimer/tetramer .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23690622" target="_blank">23690622</a>]<br></li>
<li>M206 in HypT would be located in the redox loop in OxyR where the active-site Cys208 is located , and M230 in HypT would be located next to a β-sheet in OxyR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23690622" target="_blank">23690622</a>]<br></li>
<li>M205 in the redox loop of OxyR corresponds to M206 in HypT ( Fig .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23690622" target="_blank">23690622</a>]<br></li>
<li>Both the redox loop and the β-sheet undergo major conformational changes upon OxyR oxidation ( 33 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23690622" target="_blank">23690622</a>]<br></li>
<li>The β-sheet forms a pseudohelical loop after OxyR oxidation and is important for dimer interaction ( 33 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23690622" target="_blank">23690622</a>]<br></li>
<li>Symbols are described above ; each oval represents an OxyR dimer , M designates a methyl group , and the open arrow indicates DNA replication .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20118257" target="_blank">20118257</a>]<br></li>
<li>Oxidized OxyR-DNA Contacts To examine the OxyR-DNA contacts , we analyzed the binding of oxidized OxyR to the natural oxyRS , katG , and ahpC sites with hydroxyl radical footprinting and missing nucleoside interference assays .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8087856" target="_blank">8087856</a>]<br></li>
<li>Redox-Dependent Repositioning of DNA Contacts The hydroxyl radical interference assays presented here demonstrate that OxyR shifts its DNA contacts between oxidizingand reducing conditions and that the two forms of OxyR require different contacts for binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8087856" target="_blank">8087856</a>]<br></li>
<li>This suggestion is based on the observations that the OxyR motif shows 2-fold dyad symmetry and that OxyR has a single helix-turn-helixDNA-binding domain that is unlikely to span two helical turns .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8087856" target="_blank">8087856</a>]<br></li>
				<label class="DOM_collapse_similarity" for="dom_similarity_50">
It shows a 2-fold symmetry , and the spacing of the elements suggests that OxyR contacts four helical turns .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9312050" target="_blank">9312050</a>]<br></label>
<input id="dom_similarity_50", type="checkbox">
<div class="DOM_similar">
<li>The motif shows 2-fold symmetry , and the spacing of the ATAG elements suggests that OxyR contacts four helical turns .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8087856" target="_blank">8087856</a>]<br></li></div>
<li>Fourth , the clusters of missing nucleosides on the two strands lie across from each other in the major groove ( see Figure 4 ) , suggesting that OxyR contacts bases in the major groove .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8087856" target="_blank">8087856</a>]<br></li>
<li>In addition , ~ repressor contacts five to seven nucleotides in each region , while OxyR only contacts one to three nucleotides in each region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8087856" target="_blank">8087856</a>]<br></li>
<li>We suggest that the oxidized OxyR protein is a dimer of dimers and that each of the four subunits contacts one of the four major grooves ( illustrated in Figure 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8087856" target="_blank">8087856</a>]<br></li>
<li>We suggest that when OxyR alters its DNA contacts upon oxidation , two of the subunits remain bound while the other two subunits are simply shifted one helical turn ( so that subunits bound at grooves 4 and 5 are shifted to grooves 3 and 4 , respectively ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8087856" target="_blank">8087856</a>]<br></li>
<li>In contrast , reduced OxyR contacts ATAGnT elements in two pairs of major grooves separated by one turn of the DNA helix ( Toledano et al. , 1994 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17464064" target="_blank">17464064</a>]<br></li>
				<label class="DOM_collapse_similarity" for="dom_similarity_57">
Oxidized OxyR binds in four adjacent major grooves , while reduced OxyR contacts two pairs of adjacent major grooves separated by one helical turn .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7868602" target="_blank">7868602</a>]<br></label>
<input id="dom_similarity_57", type="checkbox">
<div class="DOM_similar">
<li>In contrast , reduced OxyR contacts two pairs of adjacent major grooves separated by one helical turn .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8087856" target="_blank">8087856</a>]<br></li></div>
<li>This pattern of protection further supports the conclusion that oxidized OxyR binds to four major grooves protecting the five adjacent minor grooves .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8087856" target="_blank">8087856</a>]<br></li>
<li>Third , the missing nucleosides are located approximately 10 bp apart , parallel to the helical axis of the DNA , implying that OxyR binds on one face of the helix in four successive grooves .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8087856" target="_blank">8087856</a>]<br></li>
<li>Here , we report that oxidized OxyR recognizes a motif comprised of four ATAGnt elements spaced at 10 bp intervals and contacts these elements in four adjacent major grooves on one face of the DNA helix .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8087856" target="_blank">8087856</a>]<br></li>
<li>These experiments revealed that oxidized OxyR binds to an ATAGnt motif on one face of the DNA helix at four adjacent major grooves .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8087856" target="_blank">8087856</a>]<br></li>
<li>Speciﬁcally , in the presence of hydrogen peroxide , OxyR becomes oxidized , which results in a change in its structure , which in turn contributes to a change in protein-DNA contacts .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12029051" target="_blank">12029051</a>]<br></li>
<li>Second , the critical contacts defined by the interference assays map to the four ATAGnt boxes that define the OxyR motif , supporting the conclusion that OxyR contacts the DNA at these sequences .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8087856" target="_blank">8087856</a>]<br></li>
<li>We propose that specific DNA-recognition by an OxyR tetramer is achieved with four contacts of intermediate affinity allowing OxyR to reposition its DNA contacts and target alternate sets of promoters as the cellular redox state is altered .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8087856" target="_blank">8087856</a>]<br></li>
<li>Therefore , we propose that oxidized OxyR achieves tight specific binding by th combination of four contacts of intermediate affinity , rather than by two contacts of high affinity as seen for many of the helix-turn-helixproteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8087856" target="_blank">8087856</a>]<br></li>
<li>While repressor contacts DNA in two regions , OxyR makes contacts in four regions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8087856" target="_blank">8087856</a>]<br></li>
<li>These patterns are consistent with our suggestion that OxyR makes four intermediate strength contacts , while the repressor binds to DNA with two strong contacts .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8087856" target="_blank">8087856</a>]<br></li>
<li>Leucine residues that are spaced at intervals of seven amino acids reminiscent of a leucine-zipper dimerization domain are highly conserved among the LysR-OxyR family of regulators ( 15 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2183352" target="_blank">2183352</a>]<br></li>
<li>Perhaps the characteristics of the above CysB variants closely resemble those of dimeric OxyR mutants , for which the possibility of dimerization via either the `` dimerization domain '' or the `` tetramerization domain '' was suggested ( 9 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11038360" target="_blank">11038360</a>]<br></li>
<li>There are several possible ways to align the mom-binding site with the other three sequences that would result in 6 residues being conserved across the approximately 45-bp regions defined by OxyR can except either one of two different base pairs at a given position , yet can stillacquire specificity by distinguishing them from the remaining two base pairs .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1730735" target="_blank">1730735</a>]<br></li>
<li>Using footprinting data from both strands to more accurately locate the binding site centers ( possible dyad axis ) , it became clear that one of the OxyR recognition sequences , the binding site upstream of the oxyR gene , did contain a good dyad symmetry ( 13 out of 21 residues ) with an axis at the exact center of the experimentally determined binding site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1730735" target="_blank">1730735</a>]<br></li>
<li>In the ﬁnal mutant , pDEW25 , all three 5 cents - GATC-3 cents sites were disrupted in a manner thought unlikely to affect OxyR binding directly .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11972787" target="_blank">11972787</a>]<br></li>
<li>The alignment predicts that adenine residues of the second and third 5 cents - GATC-3 cents sites are likely to make contact with OxyR , because of overlap with OxyR contact elements I and II .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11972787" target="_blank">11972787</a>]<br></li>
<li>In an attempt to disrupt methylation of the second and third 5 cents - GATC-3 cents sites without affecting OxyR binding directly , nucleotides less likely than adenine to make contact with OxyR were altered .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11972787" target="_blank">11972787</a>]<br></li>
<li>Studies of OxyR interaction with agn43 DNA showed that methylation of any two 5 cents - GATC-3 cents sites was necessary and sufﬁcient to block binding of the repressor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11972787" target="_blank">11972787</a>]<br></li>
<li>The restoration of phase variation resulting from the mutations in pDEW20 and pDEW21 indicated that the adenine residues of the second and third 5 cents - GATC-3 cents sites were probably important for OxyR recognition and that methylation of any two sites was sufﬁcient to allow phase variation of agn43 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11972787" target="_blank">11972787</a>]<br></li>
<li>Results and discussion Model of DNA-binding domain ( DBD ) of CysB The solved structure of CbnR tetramer ( Muraoka et al. , 2003 ) as well as the model of full-length OxyR dimer ( Zaim and Kierzek , 2003 ) predict that ª88 N-terminal res-idues of LysR-type proteins form DNA-binding domains ( DBDs ) containing the winged HTH structure .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15255893" target="_blank">15255893</a>]<br></li>
<li>The presence of a high number of degenerate homologies ( positions at which only two of the four possible base pairs are represented ) led to the suggestion that OxyR may have a `` degenerate recognition code '' ( Tartagliaet al. , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8087856" target="_blank">8087856</a>]<br></li>
<li>The arrangement of subunits shown in Figure 8 would constitute an unusual configuration for a tetramer and implies that the OxyR protein may have two oligomerization domains .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8087856" target="_blank">8087856</a>]<br></li>
<li>The configuration also requires that OxyR undergoes a conformational change upon the interaction of two dimers to prevent further oligomerization .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8087856" target="_blank">8087856</a>]<br></li>
<li>The binding reacttons wtth 1O0ng of purified OxyR were carried out in 0.5 x TM wtth 12.5 mM Tris-CI and without glycerol and DTT .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8087856" target="_blank">8087856</a>]<br></li>
<li>The start of the fhuF message was mapped to two adjacent A residues located 30 bases upstream of the ATG start codon , situated in the middle of the high-afﬁnity OxyR binding site ( Fig. 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11443092" target="_blank">11443092</a>]<br></li>
<li>The structures of the cofactor-binding domains of two family members , CysB and OxyR , have been solved by X-ray crystallography ( Tyrrell et al. , 1997 ; Choi et al. , 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15255893" target="_blank">15255893</a>]<br></li>
<li>The katG OxyR binding motif and the putative ychF OxyR binding motif are shown below the sequence logo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23139412" target="_blank">23139412</a>]<br></li>
<li>The two modes of binding are essential for OxyR to function as both an activator and a repressor in vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8087856" target="_blank">8087856</a>]<br></li>
<li>Assuming the OxyR protein is a dimer ( the multimeric form suggested by preliminary cross-linking studies ) , these titration studies indicate that 65 % of the protein in our purified OxyR preparations isactive for DNA-binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1730735" target="_blank">1730735</a>]<br></li>
<li>The OxyR protein may therefore be an example of a sequence specific DNA-binding protein that did not evolve the complex features required for the positioning of amino acid side chains such that multiple contacts canbe made with a single base pair .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1730735" target="_blank">1730735</a>]<br></li>
<li>By the method for detection of potential 2cro-like DNA-binding region in proteins developed by Dodd and Egan ( 1987 ) , a relatively high score ( 1523 ) was calculated for the OxyR protein in this region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2511419" target="_blank">2511419</a>]<br></li>
<li>In the N-terminal highly homologous region , the sequence of 20 amino acids of OxyR protein ( residues 18-37 ) is similar to a helix-turn-helix motif of DNA-binding proteins ( Pabo and Sauer 1984 ) ( Fig. 7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2511419" target="_blank">2511419</a>]<br></li>
<li>Because OxyR protein has an amino acid sequence similar to the DNA-binding domain of well characterized DNA-binding proteins ( 4 , 5 ) , OxyR protein may bind to the regulatory region of the oxyR gene and repress the transcription from its promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1864839" target="_blank">1864839</a>]<br></li>
<li>The asterisks indicate conserved residues in the aligned DNA sequences ; a dash ( -- ) represents a space introduced between OxyR contact points III and IV to improve alignment with the agn43 regulatory region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11972787" target="_blank">11972787</a>]<br></li>
<li>The presence within the mom and agn43 regulatory regions of 7-bp ( AATAACC ) and 6-bp ( CGATTA ) homologous sequences ( corresponding to the extremities of the MuC binding domain of mom [ 13 ] ) suggests that another protein apart from OxyR and perhaps analogous to MuC may be required for agn43 transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10094691" target="_blank">10094691</a>]<br></li>
<li>The residue is conserved among all of the OxyR homologues ( not shown ) , and it is located in the C-terminal region of the protein 22 residues from the active site cysteine 199 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10644706" target="_blank">10644706</a>]<br></li>
<li>The GrxA protein participates in the regulatory feedback loop that controls OxyR activity ( 45 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16857675" target="_blank">16857675</a>]<br></li>
<li>The region of OxyR protein-binding was determined more precisely by a DNase I protection experiment .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1864839" target="_blank">1864839</a>]<br></li>
<li>For comparison , Fig. 2B shows sequence alignments within DBDs of some LTTRs ( CbnR , CysB , OxyR and GcvA ) and residues involved in DNA-binding ( blue ) or pc function ( red ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15255893" target="_blank">15255893</a>]<br></li>
<li>How this occurs mechanistically , particularly with the MntR-binding motif relatively distant from the OxyR binding site , is an intriguing question .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12029030" target="_blank">12029030</a>]<br></li>
<li>It is also noteworthy that the methylation of both guanine and adenine residues interfered with OxyR binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1730735" target="_blank">1730735</a>]<br></li>
<li>Since methylation also abrogates OxyR binding , this indicates that methylation plays a dual role in facilitating agn43 transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12029051" target="_blank">12029051</a>]<br></li>
<li>Strains ( D1210/pAQ25 ) ( Fig. 2A ) that overproduce the OxyR protein , do washed with 500ml of buffer Z containing 0.1 M KCI and 500 ml of buffer Z containing 0.2 M KCI and then eluted with a 700-ml linear gradient of buffer Z containing 0.2 to 0.5 M KCI .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2183352" target="_blank">2183352</a>]<br></li>
<li>Repeated attempts in this laboratory to ascertain OxyR contact points in agn43 regulatory regions using in vivo dimethyl sulphate footprinting have proved inconclusive .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11972787" target="_blank">11972787</a>]<br></li>
<li>An OxyR tetramer is thought to contact each of these conserved elements on one face of the double helix ( Toledano et al. , 1994 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11972787" target="_blank">11972787</a>]<br></li>
<li>Reduced OxyR makes contact at positions I , II , IV and V , resulting in transcription repression , whereas oxidized OxyR makes contact at positions I , II , III and IV , resulting in transcription-activation ( Toledano et al. , 1994 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11972787" target="_blank">11972787</a>]<br></li>
<li>Underlined and labelled with roman numerals are the nucleotides in oxyRS that make contact with OxyR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11972787" target="_blank">11972787</a>]<br></li>
<li>Alternatively , one OxyR dimer may be flipped relative to the other dimer , requiring the subunit bound at contact point 5 to shift to contact point 3 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8087856" target="_blank">8087856</a>]<br></li>
<li>Some experimental results obtained for NahR ( 7 ) , AmpR ( 8 ) , and OxyR ( 9 ) indicate involvement of the C-terminal domain in oligomer formation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11038360" target="_blank">11038360</a>]<br></li>
<li>Together with the nonbinding mutants described in the accompanying paper ( 20 ) , these mutants have allowed us to deﬁne functional domains in the OxyR protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7868602" target="_blank">7868602</a>]<br></li>
<li>Experiments described in the accompanying paper also indicate that the C-terminal domain is involved in oli-gomerization of OxyR ( 20 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7868602" target="_blank">7868602</a>]<br></li>
<li>Alternatively , if the constitutive phenotype was caused only by the abnormal exposure of an activation domain , the mutants should bind in the extended conformation characteristic of reduced wild-type OxyR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7868602" target="_blank">7868602</a>]<br></li>
<li>We propose that the oxidation of the wild-type OxyR protein exposes a domain which allows OxyR to recruit RNA polymerase and that the constitutive phenotype of our mutants may be due to exposure of this domain under both oxidizing and reducing conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7868602" target="_blank">7868602</a>]<br></li>
<li>In the present study , we used site-directed as well as random mutagenesis to characterize the domains of the OxyR protein required to sense oxidative stress and affect transcription initiation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7868602" target="_blank">7868602</a>]<br></li>
<li>The C-terminal domain of the a subunit ( aCTD ) interacts with A1T-rich UP ( upstream activating region ) elements in the 260 to 240 region ( 31 ) and with many transcription regulators that bind upstream of the RNAP-bind-ing site , including CAP , OmpR , OxyR , Ada , and BvgA ( 4 , 5 , 10 , 23 , 38 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10572130" target="_blank">10572130</a>]<br></li>
<li>This suggests that OxyR is in direct contact with the C-terminal contact site I region of the a subunit , and that the contact site for OxyR is located near amino acid residue 265 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8483417" target="_blank">8483417</a>]<br></li>
<li>Thus the contact site for OxyR in the a-subunit is probably located near amino acid 265 , overlapping the subsite for ORP -LCB- Zou ef a / .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8483417" target="_blank">8483417</a>]<br></li>
<li>An important direction for further studies will be to elucidate this reciprocal interaction between OxyR and RNA polymerase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7868602" target="_blank">7868602</a>]<br></li>
<li>The domains in OxyR and other LysR-family members that touch RNA polymerase have not been identiﬁed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7868602" target="_blank">7868602</a>]<br></li>
<li>RNA polymerase binding was observed in the presence of OxyR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11443092" target="_blank">11443092</a>]<br></li>
<li>When the reconstituted RNA polymerase containing wild-type a subunit was used , two transcripts ( plus template-sized transcript ) were observed from the katG promoter in the absence of OxyR protein ( Fig. 1A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8483417" target="_blank">8483417</a>]<br></li>
<li>In vitro transcriptio experiments clearly demonstrated the involvement of the a subunit C-terminal region in transcription-activation by OxyR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8483417" target="_blank">8483417</a>]<br></li>
<li>The OxyR protein was found to belong to the class that requires the C-terminal region of the a subunit for transcription-activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8483417" target="_blank">8483417</a>]<br></li>
<li>Results Transcription activation by RNA polymerase containing the C-terminal truncated a subunit The oxidized form of the purified OxyR protein can activate transcription in vitro from the # ooH katG and ahp promoters by wild-type RNA polynnerase holoenzyme containing o - ̂ '' ( Storz et ai .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8483417" target="_blank">8483417</a>]<br></li>
<li>To examine whether the C-terminal domain of the RNA polymerase a subunit is necessary for transcription-activation of the katG promoter by the OxyR protein , we carried out / n-v/fro-mixed transcription experiments of the lacUV5 and katG promoters in the presence or absence of the OxyR protein using wild-type and mutant RNA polymerase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8483417" target="_blank">8483417</a>]<br></li>
<li>Summary The role of the a subunit of Escherichia coii RNA polymerase Jn transcription-activation by the OxyR protein was investigated using / n-Wfro-reconstituted RNA polymerase containing a subunits carrying C-ter-minal truncations or an amino acid substitution .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8483417" target="_blank">8483417</a>]<br></li>
<li>To examine whether these mutations also affect transcription-activation by the OxyR protein , we carried out in vitro transcription using a reconstituted RNA polymerase containing one of these point-mutated a subunits , a-129 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8483417" target="_blank">8483417</a>]<br></li>
<li>The mutated alanine residue ( A233 ) is located in a C-terminal region which shows homology with other LysR-type transcriptional regulators and which may be involved in contact with RNA polymerase and/or in the multimerization of OxyR ( 44 , 73 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10094691" target="_blank">10094691</a>]<br></li>
<li>Furthermore , DNase I footprinting experiments using mutant RNA polymerases containing C-terminally truncated n subunits indicated that the C-terminal region of the RNA polymerase u subunits plays an essential role in co-operative binding of the OxyR protein on the katG promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8483417" target="_blank">8483417</a>]<br></li>
<li>859-864 Involvement of the RNA polymerase a subunit terminal region in co-operative interaction an transcriptional activation with OxyR protein C ¬ d Kazuyuki Tao , ^ * Nobuyuki Fujita ^ and Akira Ishihama ^ ^ Radioisotope Centre , University of Tokyo , Buni < yo-ku , Tokyo 113 , Japan , ^ Department of Moiecuiar Genetics , National institute of Genetics , Misfiima , Shizuoka 411 , Japan .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8483417" target="_blank">8483417</a>]<br></li>
				<label class="DOM_collapse_similarity" for="dom_similarity_129">
Involvement of the RNA polymerase alpha-subunit C-terminal region in co-operative interaction and transcriptional activation with OxyR protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7798138" target="_blank">7798138</a>]<br></label>
<input id="dom_similarity_129", type="checkbox">
<div class="DOM_similar">
<li>Involvement of the RNA polymerase a subunit C-terminal region in co-operative interaction and transcriptional activation with OxyR protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7868602" target="_blank">7868602</a>]<br></li></div>
<li>Taken together , these results indicate that the OxyR protein activates transcription from the katG promoter , at least in part , by enhancing the initial and stable binding of RNA polymerase to the katG promoter , and that the C-terminal region of the RNA polymerase a subunit is essential for this promoter complex formation , presumably through direct protein-protein contact .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8483417" target="_blank">8483417</a>]<br></li>
<li>Together , these results suggest that direct proteir > - protein contact between the OxyR protein and the C-termina ) contact site i region of the RNA polymerase a subunit piays an essential role in transcription-activation at the OxyR-dependent promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8483417" target="_blank">8483417</a>]<br></li>
<li>Tao , K. , Zou , C. , Fujita , N. , and Ishihama , A. ( 1995 ) Mapping of the OxyR protein contact site in the C-terminal region of RNA polymerase alpha subunit .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15009899" target="_blank">15009899</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15255893" target="_blank">15255893</a>]<br></li>
				<label class="DOM_collapse_similarity" for="dom_similarity_134">
Mapping of the OxyR protein contact site in the Cterminal region of RNA polymerase alpha subunit .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9512708" target="_blank">9512708</a>]<br></label>
<input id="dom_similarity_134", type="checkbox">
<div class="DOM_similar">
<li>Mapping of the OxyR protein contact site in the C-terminal region of RNA polymerase a-subunit .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9324270" target="_blank">9324270</a>]<br></li>
<li>Mapping of the OxyR protein contact site in the C-terminal region of RNA polymerase alpha subunit .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10094691" target="_blank">10094691</a>]<br></li></div>
<li>Michael acceptor sites are apparently not involved in the modification , but rather the reactive ( electrophilic ) carbonyl group of the outer monoterpene side chain ( structure I ) subserves transesterification ( oxyester 3 thioester ) of OxyR cysteine thiol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16030151" target="_blank">16030151</a>]<br></li>
<li>For example , the reaction of hydrogen peroxide with nucleophilic cysteine residue sulfhydryls in the transcription regulators OxyR and OhrR of bacteria and Orp1 of Saccharomyces cerevisiae generates cysteine sulfenic acid ( R-S-OH ) , an intermediate that initiates a series of events leading to expression of antioxidant genes .2,3 Many of these cysteine residue sulfhydryls have been shown to have a pKa below that of free cysteine ( 8.3 ) , suggesting that low pKa cysteine residues are redox active .2,4 Sulfur trafﬁcking pathways are among the systems that are induced during oxidative stress , and these pathways also use nucleo-philic cysteine residue sulfhydryls.5-7 In the ﬁrst step of sulfur trafﬁcking , a cysteine desulfurase abstracts inorganic sulfur from the cysteine substrate using its own reactive cysteine residue sulfhydryl , thereby forming a persulﬁde ( R-S-SH ) intermediate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20936830" target="_blank">20936830</a>]<br></li>
<li>Bacterial OxyR shares some similarities with mammalian Nrf2 : both regulate the expression of detoxifying enzymes and antioxidant proteins , both contain critical Cys residues that serve sensory and regulatory roles , and both are activated by peroxides .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16030151" target="_blank">16030151</a>]<br></li>
<li>Mass spectrometric analysis of activated OxyR revealed thioesteriﬁcation of the critical regulatory cysteine , Cys-199 , to an avicin fragment comprising the outer monoterpene side chain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16030151" target="_blank">16030151</a>]<br></li>
<li>To determine whether the cysteine residues in OxyR were important for sensing the oxidative stress signal , we changed each of the six cysteines to serine by site-directed mutagenesis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7868602" target="_blank">7868602</a>]<br></li>
<li>Using site-directed mutagenesis , we found that one cysteine residue ( C-199 ) is critical for the redox sensitivity of OxyR , and a C-1993S mutation appears to lock the OxyR protein in the reduced form .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7868602" target="_blank">7868602</a>]<br></li>
<li>Using site-directed mutagenesis , we found that only one ( C-199 ) of six cysteine resi-dues was critical for the induction of defense genes by OxyR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7868602" target="_blank">7868602</a>]<br></li>
<li>The six possible redox-reactive cysteines in OxyR were mutagenized to serine , and a single cysteine ( Cys-199 ) was found to be critical for activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7868602" target="_blank">7868602</a>]<br></li>
<li>Each OxyR monomer has six cysteine residues , of which only Cys-199 is essential for regulation by redox agents ( 21 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16030151" target="_blank">16030151</a>]<br></li>
<li>It may be significant that none ofthe six cysteine residues found in the OxyR protein is present in LysR or NodD since cysteines might easily be oxidized .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2471187" target="_blank">2471187</a>]<br></li>
<li>Most protein cysteine residues react far too sluggishly to be affected by micromolar H2O2 , but the OxyR , AhpC , and OhrR proteins feature active-site cysteines that are activated by local context so they are easily oxidized to sulfenic acid ( Imlay , 2008a ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21143317" target="_blank">21143317</a>]<br></li>
<li>In addition , a mutant form of OxyR in which five of the six cysteine residues have been induction of converted to serine shows wild-type oxyR-regulated genes in vivo ( 15 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2183352" target="_blank">2183352</a>]<br></li>
<li>Residues that are identical in four of six sequences are in boldface , the bars mark the helix-turn-helix motif , and the six cysteine residues in OxyR are underlined .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7868602" target="_blank">7868602</a>]<br></li>
<li>The mutants describe here will be useful for future biochemical studies of the modiﬁcations of the cysteine residues in OxyR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7868602" target="_blank">7868602</a>]<br></li>
<li>Having identiﬁed C-199 as a critical amino acid for OxyR activity , we wanted to determine what other amino acids were required for OxyR to act as an activator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7868602" target="_blank">7868602</a>]<br></li>
<li>If a sulfenic acid proves to be the redox center of OxyR , the H-114 and H-198 residues may act to form hydrogen bonds with the sulfenic acid .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7868602" target="_blank">7868602</a>]<br></li>
<li>Structural studies of OxyR should help to elucidate whether the regulator carries a sulfenic acid stabilized by histidine residues and give additional insights into the domains of OxyR and other LysR-family proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7868602" target="_blank">7868602</a>]<br></li>
<li>TtdR , which shares 44 % identity with OxyR ( data not shown ) , contains two cysteines , cysteine 288 and cysteine , which might participate in the constitution 296 of a redox sensor as has been shown for cysteine and 199 Bacteria were grown aerobically or anaerobically in M63 minimal medium with gluconate ( Gln ) as carbon source in the presence of L-tartrate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16804186" target="_blank">16804186</a>]<br></li>
<li>For many years , OxyR had been thought to contain only one essential cysteine residue ( Cys199 ) whose oxidation to a sulfenic acid was solely responsible for sensing and activating the response to hydrogen peroxide ( Storz et al. , 1990 ; Kullik et al. , 1995 ) However , it is now clear that OxyR has two essential cysteines , Cysl99 and Cys208 , and that the hydrogen peroxide response involves Cys 199 oxidation to a sulfenic acid followed by the formation of an intramolecular disulfide bridge with Cys208 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11407111" target="_blank">11407111</a>]<br></li>
<li>Most data support a model in which the peroxide-mediated oxidative stress signal is transduced by oxidation of two cysteine residues in OxyR , which leads to the formation of an intramolecular disulﬁde bridge .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12644490" target="_blank">12644490</a>]<br></li>
				</div></ul><br>
			<label class="collapse_EVO" for="_3"><font size="5px">OxyR evolution</font></label>
				<input id="_3" type="checkbox">
				<div class="EVO"><br><ul><li>TtdR , which shares 44 % identity with OxyR ( data not shown ) , contains two cysteines , cysteine 288 and cysteine , which might participate in the constitution 296 of a redox sensor as has been shown for cysteine and 199 Bacteria were grown aerobically or anaerobically in M63 minimal medium with gluconate ( Gln ) as carbon source in the presence of L-tartrate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16804186" target="_blank">16804186</a>]<br></li>
<li>Moreover , direct comparison of the OxyR and MuC binding sites of the mom regulatory region with that of agn43 shows a region displaying 65.6 % identity at the nucleotide level ( Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10094691" target="_blank">10094691</a>]<br></li>
<li>NAC from K. aerogenes is 50 % identical to OxyR ( a regulator that senses oxidative stress ) from E. coli ( 41 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9495755" target="_blank">9495755</a>]<br></li>
<li>Structural studies of OxyR should help to elucidate whether the regulator carries a sulfenic acid stabilized by histidine residues and give additional insights into the domains of OxyR and other LysR family proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7868602" target="_blank">7868602</a>]<br></li>
<li>Christman , M.F. , Sforz , G. , and Ames , B.N. ( 1989 ) OxyR , a putative regulafor of hydrogen peroxide-inducible genes in Eschericlva ooH and SaimoneHa typhimurium , is homo-logous to a family of bacteriai regulatory proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7565112" target="_blank">7565112</a>]<br></li>
<li>OxyR , a positive regulator of typhimurium , is homologous to a family of bacterial regulatory proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15672380" target="_blank">15672380</a>]<br></li>
				<label class="EVO_collapse_similarity" for="evo_similarity_7">
OxyR , a positive regulator of hydrogen peroxide-inducible genes in Escherichia coiland Salmonella typhimurium , ts homologous to a family of bacterial regulatory proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8087856" target="_blank">8087856</a>]<br></label>
<input id="evo_similarity_7", type="checkbox">
<div class="EVO_similar">
<li>OxyR , a positive regulator of hydrogen peroxide-inducible genes in Escherichia coli and Salmonella typhimurium , is homologous to a family of bacterial regulator proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1943993" target="_blank">1943993</a>]<br></li>
<li>OxyR , a positive regulator of hydrogen peroxide-inducible genes in Escherichia coli and Salmonella typhimurium , is homologous to a family of bacterial regulatory proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7868602" target="_blank">7868602</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1706701" target="_blank">1706701</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7751275" target="_blank">7751275</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8990283" target="_blank">8990283</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9922271" target="_blank">9922271</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9324269" target="_blank">9324269</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8932307" target="_blank">8932307</a>]<br></li>
<li>OxyR , a positive regulator of hydrogen peroxide -- inducible genes in Escherichia coli and Salmonella typhimurium , is homologous to a family of bacterial regulatory proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9230301" target="_blank">9230301</a>]<br></li>
<li>OxyR , a positive regulator of hydrogen peroxide-inducible genes in Escherichia coli and Salmonella typhimurium is homologous to a family of bacterial regulatory proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11133959" target="_blank">11133959</a>]<br></li>
<li>Christman , M.F. , Storz , G. , and Ames , B.N. ( 1989 ) OxyR , a positive regulator of hydrogen peroxide-inducible genes in Escherichia coli and Salmonella typhimurium , is homologous to a family of bacterial regulatory proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8483417" target="_blank">8483417</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10692164" target="_blank">10692164</a>]<br></li>
<li>Christman , M. F. , Storz , G. , and Ames , B. N. ( 1989 ) OxyR , a positive regulator of hydrogen peroxide-inducible genes in Escherichia coli and Salmonella typhimurium , is homologous to a family of bacterial regulatory proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10794600" target="_blank">10794600</a>]<br></li>
<li>Christman , M.F. , Storz , G. and Ames , B.N. ( 1989 ) OxyR , a positive regulator of hydrogen peroxide-inducible genes in Escherichia coli and Salmonella typhimurium , is homologous to a family of bacterial regulatory proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9312050" target="_blank">9312050</a>]<br></li>
<li>Cell 41 :753 -762 Christman MF , Storz G , Ames BN ( 1989 ) OxyR , a positive regulator of hydrogen peroxide-inducible genes in Escherichia coli and Salmonella typhimurium , is homologous to a family of bacterial regulatory proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8709966" target="_blank">8709966</a>]<br></li>
<li>Cell 441:753 -- 762 Christman MF , Storz G , Ames B ( 1989 ) OxyR , a positive regulator of hydrogen peroxide-inducible genes in Escherichia coli and Salmonella typhimurium , is homologous to a family of bacterial regulatory proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11479716" target="_blank">11479716</a>]<br></li></div>
<li>Like many other members of the LysR family , OxyR acts as both a positive and a negative regulator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2183352" target="_blank">2183352</a>]<br></li>
<li>For example , a family of regulatory proteins such as OxyR , LysR , CysB , NodD , and MetR , that act as positive regulator for other genes , were shown to negatively regulate their own expression ( 30 , 32 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1715982" target="_blank">1715982</a>]<br></li>
<li>The OxyR protein shows homology to a family of positive regulatory proteins including LysR in E. coli and NodD in Rhizobium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2471187" target="_blank">2471187</a>]<br></li>
<li>DISCUSSION The cloning and DNA sequence of the E. coli oxyR gene , a positive regulator of a regulon for defenses against oxidative stress , revealed that OxyR is a 34-kDa protein that shows significant homology to a family of bacterial regulatory proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2471187" target="_blank">2471187</a>]<br></li>
<li>OxyR protein is a positive regulator for a number of genes induced by oxidative stresses and , consistent with this , an homology search revealed that the protein is homologous to a large family of bacterial activator proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2511419" target="_blank">2511419</a>]<br></li>
<li>Here we report that OxyR is a member of a family of bacterial regulator proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2471187" target="_blank">2471187</a>]<br></li>
<li>In the N-terminal highly homologous region , the sequence of 20 amino acids of OxyR protein ( residues 18-37 ) is similar to a helix-turn-helix motif of DNA binding proteins ( Pabo and Sauer 1984 ) ( Fig. 7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2511419" target="_blank">2511419</a>]<br></li>
<li>An interesting feature of this region was the nucleotide sequence homology ( 39.5 % identity ) shared between it and the promoter regions of the mom and oxyR genes , both of which are known to bind OxyR , a LysR-type transcriptional activator ( 12 , 22 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10094691" target="_blank">10094691</a>]<br></li>
<li>This region shows the highest degree of amino acid sequence similarity among family members and is believed to participate in DNA-binding according to mutational studies performed on several LTTRs , e.g. NahR ( Schell et al. , 1990 ) , OxyR ( et al. , 1995a ) , GcvA ( Jourdan and Stauffer , 1998 ) , CysB ( Lochowska et al. , 2001 ) and CrgA ( Deghmane and Taha , 2003 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15255893" target="_blank">15255893</a>]<br></li>
<li>These results suggest that the Nterminal region of OxyR is involved in DNA binding , similarly to those of the other nine members of the family ( Hen-ikoff et al. 1988 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2511419" target="_blank">2511419</a>]<br></li>
<li>The structures of the cofactor-binding domains of two family members , CysB and OxyR , have been solved by X-ray crystallography ( Tyrrell et al. , 1997 ; Choi et al. , 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15255893" target="_blank">15255893</a>]<br></li>
<li>A DNase I hypersensitive site was also previously seen for IlvY ( a member of the family of regulators showing homology to OxyR ) in the absence but not in the presence of the inducer of the IlvY-mediated response ( 22 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2183352" target="_blank">2183352</a>]<br></li>
<li>Some of the other bacterial regulators that show homology to OxyR have also been found to be bound to the promoters that they regulate in both the presence and absence of inducer ( 22 , 24 ) suggesting that members of this family of regulators may all be ideally designed for responses requiring immediate induction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2183352" target="_blank">2183352</a>]<br></li>
<li>The OxyR protein is a member of the LysR Received 10 February , 1994 ; revised 12 April , 1994 ; accepted 19 April , 1994 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7984106" target="_blank">7984106</a>]<br></li>
<li>Leucine residues that are spaced at intervals of seven amino acids reminiscent of a leucine zipper dimerization domain are highly conserved among the LysR-OxyR family of regulators ( 15 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2183352" target="_blank">2183352</a>]<br></li>
<li>The primary structure of OxyR protein , deduced from the nucleotide sequence , revealed a high degree of homology to the sequences of a large family of bacterial activator proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2511419" target="_blank">2511419</a>]<br></li>
<li>The amino acid sequence of OxyR protein , deduced from the nucleotide sequence , shows a high degree of homology to the sequences of a number of bacterial activator proteins including LysR , CysB , IlvY , MetR and NodD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2511419" target="_blank">2511419</a>]<br></li>
<li>The amino acid sequence of OxyR protein deduced from the nucleotide sequence reveals a high degree of homology to those of several bacterial activator proteins including LysR , CysB , IlvY , MetR , and NodD , which were collectively called LysR family by Henikoff et aL ( 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1864839" target="_blank">1864839</a>]<br></li>
<li>The regulatory region of this cah gene , as well as of all other members of this family for which sequence is available , has retained the essential features for OxyR - and Dam-dependent phase variation ( 37 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12644490" target="_blank">12644490</a>]<br></li>
<li>The domains in OxyR and other LysR family members that touch RNA polymerase have not been identiﬁed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7868602" target="_blank">7868602</a>]<br></li>
<li>The positions of the mutations causing the constitutive phenotype are shown on an alignment of OxyR and ﬁve other members of the LysR family in Fig. 2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7868602" target="_blank">7868602</a>]<br></li>
<li>Protein sequence alignment of six LysR family members and locations of the OxyR mutations causing a constitutively active phenotype .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7868602" target="_blank">7868602</a>]<br></li>
<li>The transcription of nhaA , which is positively regulated by NhaR ( 29 ) , a member of the LysR-OxyR family ( 7 , 14 , 30 , 34 ) , is speciﬁcally induced by intracellular Na1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11133959" target="_blank">11133959</a>]<br></li>
<li>The expression of nhaA is positively regulated by nhaR ( 39 ) , a member of the LysR-OxyR family ( 6 , 16 , 42 , 46 ) , and is induced by Na1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8932307" target="_blank">8932307</a>]<br></li>
<li>OxyR is a member of the LysR family of bacterial regulators , many of which also show extended footprints and different binding properties under inducing and uninducing conditions ( reviewed by Schell , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8087856" target="_blank">8087856</a>]<br></li>
<li>Comparison of the amino and sequence of OxyR protein with the LysR family consensus sequence ( Henikoff et al. 1988 ) slightly modified by in ~ uding ~ e OxyR sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2511419" target="_blank">2511419</a>]<br></li>
<li>1 ) The OxyR protein is a member of a large family of bacterial regulators that have helix-turn-helix motifs ( Christman et al. , 1989 ; Henikoff et al. , 1988 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1730735" target="_blank">1730735</a>]<br></li>
<li>nhaR is a regulatory protein which binds DNA ( RahavManor et al. , 1992 ) and exhibits homology with the LysR-OxyR family of positive regulators ( Christman et al. , 1989 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8168494" target="_blank">8168494</a>]<br></li>
<li>NhaR is homologous to a large family of positive regulators in prokaryotes , the LysR-OxyR family ( Henikoff et al. , 1988 ; Christman et al. , 1989 ; RahavManor et al. , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9312050" target="_blank">9312050</a>]<br></li>
<li>Sequence analysis of the oxyR gene revealed that OxyR is a member of a large family of bacterial regulators that includes the E. coli regulatory protein LysR and the Rhizobium regulatory protein NodD ( 10-12 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2183352" target="_blank">2183352</a>]<br></li>
<li>The induction of 9 of the 30 proteins is under positive control of the oxyR gene product ( 10 , 28 ) , OxyR , a member of the LysR family of transcriptional regulators ( 11 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8990283" target="_blank">8990283</a>]<br></li>
<li>The dps gene , which is transcribed from a single promoter , is controlled by OxyR , which similar to NhaR , is a member of the LysR-OxyR family of positive activators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11133959" target="_blank">11133959</a>]<br></li>
<li>NhaR is the ﬁrst member of the LysR-OxyR family of activator proteins to be shown to overcome H-NS-mediated repression at a promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8932307" target="_blank">8932307</a>]<br></li>
<li>OxyR belongs to the LysRtype family of transcriptional regulators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8588737" target="_blank">8588737</a>]<br></li>
<li>OxyR is 34 kDa in size and belongs to the LysR-type family of transcriptional regulators ( 6 , 14 , 37 , 43 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7868602" target="_blank">7868602</a>]<br></li>
<li>The sequence of the OxyR protein is highly homologous to the consensus sequence and , therefore , it belongs to the family .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2511419" target="_blank">2511419</a>]<br></li>
<li>The gene nhaR of this operon encodes an autoregulatory protein that activates nhaAR transcription and is homologous to the LysR-OxyR family of prokaryotic transcriptional regulators ( 57 , 64 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16997959" target="_blank">16997959</a>]<br></li>
				</div></ul><br>
			<label class="collapse_RP" for="_4"><font size="5px">Processes regulated by OxyR</font></label>
				<input id="_4" type="checkbox">
				<div class="RP"><br><ul><li>Genes transcribed more efﬁciently in the presence of EsS Genes and Gene product b promoter regions a Transcription levels ( EsS/Es70 ratio ) d Stress response yieF sE ( 59 ) ; sS ( 19 ) sS , RcsBA ( 60 ) sS , RcsBA ( 60 ) Chromate reductase b3713 N.A. 1.50 1.51 osmB Intergenic region including osmB promoter Osmolarity-inducible lipoprotein b1283 b2688 1.51 2.21 gshA g-glutamate -- cysteine ligase ( glutathione-biosynthesis ) ahpF Alkyl-peroxidase reductase pphA Protein phosphatase macA Macrolide resistance efﬂux pump DNA and RNA-metabolism and modiﬁcation ppk Polyphosphate kinase , part of RNA degradosome , Intergenic region including xapA promoter xapA Xanthosine phosphorylase , nucleotide-synthesis/degradation gyrB B subunit of DNA gyrase recT Rac prophage , recombinase , in recET-lar-ydaCQ operon lar Rac prophage , restriction alleviation and modiﬁcation enhancement , in recET-lar-ydaCQ operon ydaC Rac prophage , in recET-lar-ydaCQ operon ydaQ Rac prophage , possible recombinase , in recET-lar-ydaCQ operon Polyamine-metabolism puuB g-glutamyl -- putrescine oxidase gabD Succinate semialdehyde dehydrogenase speB Agmatinase ( in arginine/putrescine-degradation pathway ) Transcription regulation Intergenic region including appY promoter appY Regulator of hyaABCDEF operon yciT Putative deoR-type transcription regulator ilvY Regulator of ilvC operon rhaS Regulator of rhamnose-transport Sugar metabolism treF Trehalase OxyR ( 61 ) sH ( 62,63 ) b0606 b1838 b0878 1.52 1.54 1.54 Regulator of intracellular sS concentrations ( 3 ) XapR ( 64 ) XapR ( 64 ) b2501 1.50 N.A. b2407 1.62 1.56 b3699 b1349 1.65 1.93 Fis ( 65 ) sS-Dependent in bioﬁlm cells ( 18 ) - growing b1348 1.93 b1347 b1346 1.74 1.66 as recT as recT sS ( 19 ) b1301 b2661 b2937 1.61 1.65 1.66 sS , Nac ( 9,66 ) 1.69 1.51 1.56 1.75 1.81 N.A. b0564 b1284 b3773 b3905 H-NS ( 67 ) H-NS ( 67 ) CRP ( 29 ) ; RhaS ( 68 ) sS ( 9,19 ) sS ( 19 ) sS ( 7 ) 1.53 1.55 1.56 1.57 b3519 b4230 b1897 b1901 ytfT Putative galactose ABC transporter otsB Trehalose-6-phosphate phosphatise araF Arabinose transporter , component of an ABC transport system glgP Glycogen phosphorylase , part of glgCAP operon gpmM Putative 2,3-bisphosphoglycerate mutase glvC Sugar transport phosphotransferase Other metabolic functions Intergenic region including hyaABCDEF promoter hyaA Hydrogenase small subunit ; in hyaABCDEF operon ( 69 ) ; sS ( 19 ) CRP , AraC CRP ( 70 ) ; sS controls glgCAP operon ( 19 ) 1.60 b3428 1.61 1.79 b3612 b3683 2.02 N.A. sS , AppY , anaerobic regulation by ArcA and NarP/NarL ( 20,71,72 ) as hyaA 1.68 b0972 Hydrogenase large subunit ; in hyaABCDEF operon Hydrogenase subunit ( nickel-binding protein ) ; in hyaABCDEF operon SecY-interacting protein Acetyl-muramic acid permease Transcriptional repressor of murQP Trans-aconitate methylatransferase Predicted oxidoreductase , Fe-S protein ; in ydhYVWXUT operon Predicted oxidoreductase , Fe-S protein ; in ydhYVWXUT operon hyaB 1.66 b0973 as hyaA hyaF 1.50 b0977 syd 1.59 1.59 1.62 1.60 b2793 b2429 b2427 b1519 murP murR tam CRP , MurR ( 73 ) sS ( 74 ) ydhY FNR , NarL ( 75 ) 1.80 b1674 FNR , NarL ( 75 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21398637" target="_blank">21398637</a>]<br></li>
<li>Inner membrane protein ; in yhbE-obgE operon GTP-binding protein , involved in ppGpp turnover ; in yhbE-obgE operon ubiX b2311 1.90 as cvpA c2854 2.00 N.A. yjeF 1.88 b4167 yjeE 1.95 b4168 amiB 2.02 b4169 mutL b4170 2.06 2.36 1.91 b3184 b3183 yhbE obgE Transcription regulation cspA Intergenic region including cspA promoter Cold shock protein A 1.85 2.34 N.A. b3556 Upregulated in an rpoS mutant derivative of OH157 : H7 EDL 933 ( 19 ) alpA 2.31 b2624 CP4-57 prophage gene , regulator of tmRNAs Metabolic functions fhuF ydhR artJ Fur , OxyR ( 84 ) 1.83 1.91 2.13 b4367 b1667 b0860 Iron reductase Putative monooxygenase Arginine transporter Upregulated in a bioﬁlm-growing rpoS mutant derivative of MG1655 ( 18 ) Unknown and miscellaneous functions ycgY Unknown 1.83 b1196 ( continued 1.85 1.92 b2017 b2119 yefM yehL Antitoxin in yefM-yoeB toxin-antitoxin system Unknown , possible component of ABC transport system Unknown , putative membrane protein Putative lipoprotein 2.06 2.10 b1821 b1705 yebN ydiE Non-coding RNAs isrB spf ssrS Small RNA 2.25 2.72 3.74 b4434 b3864 b2911 Small RNA , regulates DNA polymerase I activity 6S RNA , modulates s70 activity Upregulated in a rpoS mutant derivative of MG1655 ( 20 ) Intergenic regions , genes c1714 not annotated in MG1655 Unknown , annotated in CFT073 , upstream of cls , transcribed in opposite direction Intergenic region including yahA promoter region Unknown , annotated in O157 : H7 SAKAI , possible prophage gene ( renD ) Intergenic region between argT and hisJ Intergenic region upstream bssS ( regulator of bioﬁlm formation ) Unknown , annotated in CFT073 , upstream of cobU Intergenic region upstream of lpd Intergenic region upstream of lrhA Annotated in O157 : H7 OH157 : H7 EDL 933 , caiC Annotated in CFT073 , wcaM Annotated in O157 : H7 OH157 : H7 EDL 933 , rfaG Unknown , annotated in CFT073 Intergenic region upstream yqaBA Intergenic region downstream of ompT Unknown , annotated in CFT073 ( antisense of cspC ) Intergenic region , between yobF and yebO Annotated in O157 : H7 OH157 : H7 EDL 933 ( antisense of yegI ) Annotated in CFT073 homologous to aslA Annotated in CFT073 as dppC Intergenic region , between yobD and yebN Unknown , annotated in CFT073 , ydfR Annotated in O157 : H7 OH157 : H7 EDL 933 ( toxic peptide ) Annotated in O157 : H7 OH157 : H7 EDL 933 ; yjgM , putative acetyltransferase N.A. 1.83 N.A. 1.85 N.A. 1.86 ECs1613 N.A. N.A. 1.92 1.92 N.A. 1.94 c2481 ... .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21398637" target="_blank">21398637</a>]<br></li>
<li>WT , MG1655 ; vector , empty vector control ( pACYC184 ) ; pGSO58 , plasmid-borne mutant OxyR A233V that constitutively overexpresses OxyR-regulated genes ; ahp ( AL154 ) , NADH peroxidase mutant ; Hpx ( AL145 ) , catalase / peroxidase-deﬁcient mutant .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21378183" target="_blank">21378183</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21378183" target="_blank">21378183</a>]<br></li>
<li>12.38 -8.24 -59.96 1.13 ± 0.55 ± 0.50 ± 0.02 0.04 0.02 1.27 ± 0.50 ± 0.20 ± 0.03 0.03 0.03 / h ) / h ) Fig. 2 Comparison of the transcript levels of the E. coli wild type under different pH values at 7.0 and 6.0 : a global regulatory genes , b PhoB regulatory genes , c acid-inducible genes , d metabolic pathway genes , e respiratory chain genes during micro-aerobic continuous culture at the dilution rate of 0.1 h -1 Fig. 3 a , b Overall metabolic regulation of the E. coli during acidic condition ( a ) Acid Condition Hns ATP ADP PhoQ PhoQ P ATP ADP ¬ EvgS EvgS P cAMP - evgA Pho P PhoP Crp EvgA P B1500 EvgA cAMP-Crp ¬ RpoS b1500 ydeo ¬ YdeO gadE GadE O O GadW GadX hdeA gadB gadc gadW gadA gadX HdeA GadA GadB GadC Acid resistance ( b ) Acidic Condition ( H + ) ( indirect ) EnvZ PhoR fnr OxyR ldhA arcA P Fnr ArcA SoxR / S L PhoR - P OmpR OmpR - P DH pfl yfiD ompC TCA-cycle genes : gltA sdhC mdh sodA PhoB - P PhoB Lactate OmpC ompF asr YfiD Pfl OmpF Asr Formate Acetate ackA pta Acid resistance Moreover , in accord with the proteomic analysis , the expression of yﬁD : : lac reporter fusion and the intracellular content of YﬁD were found to be high during growth at low pH for both aerobic and anaerobic cultures ( Neil et al. 2002 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23274360" target="_blank">23274360</a>]<br></li>
<li>Endogenous protein-S-nitrosylation in E. coli : regulation by OxyR Divya Seth1 , Alfred Hausladen1 , Ya-Juan Wang2 , and Jonathan S. Stamler1 ,3 1Institute for Transformative Molecular Medicine and Department of Medicine , Case Western Reserve University School of Medicine and University Hospitals Case Medical Center , Cleveland , OH 44106 , USA 2Center for Proteomics and Bioinformatics , Case Western Reserve University School of Medicine , Cleveland , OH 44106 , USA Abstract Endogenous S-nitrosylation of proteins , a principal mechanism of cellular signaling in eukaryotes , has not been observed in microbes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22539721" target="_blank">22539721</a>]<br></li>
<li>Dam - and OxyR-Dependent Phase Variation of agn43 : Essential Elements and Evidence for a New Role of DNA Methylation A nu Wallecha , Vincent Munster , Jason Correnti , Teresa Chan , and Marjan van der Woude * Department of Microbiology , University of Pennsylvania , Philadelphia , Pennsylvania 19104 Received 1 March 2002/Accepted 10 March 2002 Phase variation of the outer membrane protein Ag43 in E. coli requires deoxyadenosine methylase ( Dam ) and OxyR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12029051" target="_blank">12029051</a>]<br></li>
<li>Exposure of cells to H2O2 in a range from 5 to 200 M activates OxyR and enhances the synthesis of 40 proteins ( 183 ) , including HPI catalase ( KatG ) ( 279 ) , an NADPH-de-pendent alkyl hydroperoxide reductase ( AhpCF ) ( 279 ) , gluta-thione reductase ( GorA ) ( 183 ) , and Dps , which nonspeciﬁcally binds DNA to protect cells from H2O2 toxicity ( 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16760308" target="_blank">16760308</a>]<br></li>
<li>OxyR can be activated by several redox-related modifications of a single Cys-199 ( OxyR-SX ) , including S-nitrosylation , S-OH , and S-glutathione .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16030151" target="_blank">16030151</a>]<br></li>
<li>Hydrogen Escherichia coli Isc ironsulfur assembly system , and OxyR induces the Suf system to compensate peroxide inactivates the Soojin Jang # and James A. Imlay * Department of Microbiology , University of Illinois , Urbana , IL 61801 bstract A Environmental H2O2 creates several injuries in Escherichia coli , including the oxidative conversion of dehydratase [ 4Fe-4S ] clusters to an inactive [ 3Fe-4S ] form .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21143317" target="_blank">21143317</a>]<br></li>
<li>JOURNAL OF BACTERIOLOGY , Mar. 1995 , p. 1275 -- / 95 / $ 04.0010 Copyright q 1995 , American Society for Microbiology Mutational Analysis of the Redox-Sensitive Transcriptional Regulator OxyR : Regions Important for Oxidation and Transcriptional Activation INES KULLIK ,1 † MICHEL B. TOLEDANO ,1 ‡ LOUIS A. TARTAGLIA ,2 § AND GISELA STORZ1 * Cell Biology and Metabolism Branch , National Institute of Child Health and Human Development , Bethesda , Maryland 20892,1 and Division of Biochemistry and Molecular Biology , University of California , Berkeley , California 947202 Received 26 July 1994/Accepted 15 December 1994 OxyR is a redox-sensitive transcriptional regulator of the LysR family which activates the expression of genes important for the defense against hydrogen peroxide in Escherichia coli and Salmonella typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7868602" target="_blank">7868602</a>]<br></li>
<li>JOURNAL OF BACTERIOLOGY , Jan. 1997 , p. 330 -- / 97 / $ 04.0010 Copyright q 1997 , American Society for Microbiology Identiﬁcation and Characterization of Hydrogen Peroxide-Sensitive Mutants of Escherichia coli : Genes That Require OxyR for Expression SUMAN MUKHOPADHYAY AND HERB E. SCHELLHORN * Department of Biology , McMaster University , Hamilton , Ontario L8S 4K1 , Canada Received 4 June 1996/Accepted 1 November 1996 Escherichia coli produces an inducible set of proteins that protect the cell from exogenous peroxide stress .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8990283" target="_blank">8990283</a>]<br></li>
<li>JOURNAL OF BACTERIOLOGY , Sept. 1997 , p. 5967 -- / 97 / $ 04.0010 Copyright © 1997 , American Society for Microbiology oxyR-Dependent Induction of Escherichia coli grx Gene Expression by Peroxide Stress KAZUYUKI TAO * Radioisotope Center , University of Tokyo , Bunkyo-ku , Tokyo 113 , Japan Received 28 April 1997/Accepted 18 July 1997 The Escherichia coli OxyR protein is a transcriptional activator for a number of genes induced in response to low concentrations of hydrogen peroxide .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9294462" target="_blank">9294462</a>]<br></li>
<li>They are involved in processes as diverse as respiration , gene regulation , cofactor-biosynthesis , and DNA-RNA-metabolism .7 There is tremendous interest in understanding the mechanisms of Fe-S cluster biosynthesis , one reason being that several human diseases , including Friedreich 's ataxia , are the result of genetic defects in the protein machinery that makes Fe-S clusters .7 10.1021 / pr100920r 2010 American Chemical Society 6740 Journal of Proteome Research 2010 , 9 , 6740 -- 6751 Published on Web 10/11/201 The SUF system of E. coli is comprised of complex protein machinery , which , under the control of OxyR , IHF , IscR , andFur , synthesizes Fe S clusters during the potentially cytotoxic conditions of iron limitation and oxidative stress.6,11-14 ,17,18 The SUF system is encoded by the sufABCDSE operon .14 SufS is a cysteine desulfurase that mobilizes inorganic sulfur via a pyridoxal-5 ′ - phosphate cofactor and a conserved cysteine ( C ) residue and transfers the sulfur atom through a proposed persulﬁde transfer mechanism to the conserved C51 residue on the sulfur acceptor protein SufE.19-21 SufB , SufC , and SufD form a complex ( SufBC2D , hereafter referred to as SufBCD ) 22 that stimulates this cysteine desulfurase activity through an unknown mechanism ,21 though SufC is known to be an atypical ABC transporter ATPase .12 In vitro evidence suggests that SufE transfers the sulfur atom to SufB within the SufBCD complex and that SufBCD acts as a scaffold upon which an Fe-S cluster is assembled.22-24 It is not known whether the persulﬁde sulfur on SufE is liberated via nucleophilic attack by the recipient Cys thiol on the terminal sulfur of the persulﬁde , which would result in persulﬁde transfer , or via nucleophilic attack on the bridging sulfur , which would produce a disulﬁde-linked complex between SufE and the recipient protein and liberate sulﬁde that may then bind a free Cys residue for use in assembling an Fe-S cluster .5 It also remains to be determined whether the assembled cluster on SufBCD is transferred to a target apoprotein directly or via SufA.22,25-27 It is possible that SufE transfers sulfur to proteins other than SufB , as both the iron source and the means by which the sulfur on SufE is incorporated into an Fe-S cluster remain unknown .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20936830" target="_blank">20936830</a>]<br></li>
<li>These proteins are associated with the regulation of a large ( ArcA and Fis ) or small number ( CadC , UidR , and OxyR ) of downstream targets , indicating that both global and specific transcriptional regulators were identified in our screen .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18824089" target="_blank">18824089</a>]<br></li>
<li>We identified both xenobiotic-metabolism ( AhpC , CpxA , and CynR ) and stress based ( ArcA and OxyR ) regulatory proteins as being over-represented amongst the MMS-toxicity modulating proteins , which was expected and served as another validation for our algorithm .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18824089" target="_blank">18824089</a>]<br></li>
<li>To determine whether the SoxRS and OxyR regulators play more important roles at different time points or under other conditions , we also assayed the dose dependence ( Fig. 2A ) and time course of induction ( Fig. 2B ) for representative target genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14718666" target="_blank">14718666</a>]<br></li>
<li>Since methylation also abrogates OxyR binding , this indicates that methylation plays a dual role in facilitating agn43 transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12029051" target="_blank">12029051</a>]<br></li>
<li>Thus , upon H2O2 exposure , OxyR activates the transcription of H2O2-scavenging enzymes like KatG and at the same time represses the transcription of the KatG inhibitor YchF ( Fig. 7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23139412" target="_blank">23139412</a>]<br></li>
<li>It is recognized that NO at low concentrations has a protective role for bacteria , as it activates transcription factors , such as SoxRS , OxyR , and Fur , whose regulons encode antioxidant enzymes ( 17 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22563051" target="_blank">22563051</a>]<br></li>
<li>Oxidized OxyR then activates transcription of antioxidant genes , including katG ( encoding hydroperoxidase I ) , ahpCF ( encoding an alkyl hydroperoxide reductase ) , dps ( encoding a nonspeciﬁc DNA binding protein ) , gorA ( encoding glutathione reductase ) , grxA ( encoding glutaredoxin I ) , and oxyS ( encoding a small regulatory RNA ) ( reviewed in reference 17 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11443092" target="_blank">11443092</a>]<br></li>
<li>Activated OxyR then induces transcription of a set of antioxidant genes , including katG ( hydroperoxidase I ) , ahpCF ( alkylhydroperoxidase ) , dps ( a nonspeciﬁc DNA binding protein ) , gorA ( glutathione reductase ) , grxA ( glutaredoxin I ) , and oxyS ( a regulatory RNA ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10419964" target="_blank">10419964</a>]<br></li>
<li>Oxidized OxyR then induces the transcription of a set of genes , including katG ( hydroperoxidase I ) , dps ( a nonspeciﬁc DNA binding protein ) , ahpCF ( alkyl hydroperoxide reductase ) and gorA ( glutathione reductase ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11073934" target="_blank">11073934</a>]<br></li>
<li>Upon exposure to hydrogen peroxide , OxyR induces the expression of several genes , including katG ( encoding HPI catalase ) , ahpCF ( encoding an alkyl hydroperoxide reductase ) , dps ( encoding a nonspeciﬁc DNA-binding protein ) , gorA ( encoding glutathione reductase ) , and oxyS ( encoding a small untranslated , regulatory RNA ) , and the cells become more resistant to oxidative stress ( 3 , 4 , 39 , 41 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7868602" target="_blank">7868602</a>]<br></li>
<li>Upon exposure to hydrogen peroxide , the OxyR protein induces the expression of a set of antioxidants encoding genes including katG ( hydro-peroxide I ) , ahpCF ( alkylhydroperoxide reductase ) , gorA ( glutathione reductase ) , dps ( a non-specific DNA binding protein ) , grxA ( glutaredoxin I ) , fur ( Fur repressor ) and oxyS ( a regulatory RNA ) ( Storz and Imlay , 1999 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10760142" target="_blank">10760142</a>]<br></li>
<li>Many of the genes implicated in the defence against ROS are inducible and belong to regulons that enable bacteria to cope with various stresses : the SoxRS regulon , including sodA and nfo , which responds to redox-cycling drugs ; the OxyR regulon , including katG and dps , which is induced in actively growing cells in response to H,O,-mediated oxidation ; the RpoS ( aS ) regulon , including also katG METHODS and dps , which is induced when bacteria enter stationary Bacterial strains .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10439394" target="_blank">10439394</a>]<br></li>
<li>In cases where multiple promoters with different regulation have been recognized , they are distinguished as P1 , P2 etc. - Fold up-regulation relative to the anaerobic steady state at the indicated times following the introduction of air 5 min 10 min 15 min 60 min 4.8 5.2 4.6 3.1 3.1 3.5 2.1 2.0 4.2 4.2 5.1 4.6 Oxidative stress and iron-sulfur-cluster-assembly genes Relevant regulatory proteins ahpC alkyl hydroperoxide reductase small subunit ahpF alkyl hydroperoxide reductase large subunit dadA D-amino acid dehydrogenase dadX alanine racemase OxyR * ( ) MetJ ( ) OxyR * ( ) MetJ ( ) P1 , CRP * ( / ) Lrp * ( ) P2 , CRP * ( ) Lrp * ( / ) P3 , CRP * ( ) Lrp * ( ) P1 , CRP * ( / ) Lrp * ( ) P2 , CRP * ( ) Lrp * ( / ) P3 , CRP * ( ) Lrp * ( ) IscR * ( ) SoxRS * ( ) OxyR * ( ) OxyR * ( ) IscR * ( ) IscR * ( ) IscR * ( ) IscR * ( ) IscR * ( ) IscR * ( ) IscR * ( ) OxyR * ( ) P1 , H-NS ( ) Lrp * ( ) NhaR ( ) RcsB ( P2 , H-NS ( ) Lrp * ( ) IscR * ( ) OxyR ( ) IHF ( ) Fur SoxR ( ) SoxS ( ) OxyR * ( ) IHF * ( ) Fur ( ) IscR * ( ) OxyR * ( ) IHF * ( ) Fur ( ) IscR * ( ) OxyR * ( ) IHF * ( ) Fur ( ) IscR * ( ) 3.3 2.1 3.4 1.9 3.2 1.3 7.2 1.6 2.2 2.8 6.3 6.1 3.4 3.0 2.1 5.3 2.1 2.8 3.0 9.1 5.8 4.1 fdx ferredoxin fur ferric uptake regulator 2.1 3.4 2.4 2.8 3.3 10.1 7.0 5.1 2.0 1.9 1.8 2.7 1.8 1.8 2.8 2.5 11.2 5.7 4.4 1.8 1.1 3.6 grxA glutaredoxin 1 hscA co-chaperone involved in IscU iron-sulfur-cluster-assembly hscB co-chaperone involved in IscU iron-sulfur-cluster-assembly iscA iron-sulfur-cluster-assembly protein iscR transcriptional regulator iscS cysteine desulfurase iscU scaffold protein involved in iron-sulfur-cluster-assembly iscX possible role in iron-sulfur-cluster-biosynthesis katG bifunctional hydroperoxidase I osmC organic hydroperoxidase Within 5 min of air exposure the abundance of the soxS transcript is increased 8-fold ( Table 2 ) , indicating activation of the sensor SoxR , which enhances soxS expression , which in turn controls the SoxRS regulon ( 39 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16857675" target="_blank">16857675</a>]<br></li>
<li>Genes selected from SoxRS and OxyR regulons for stress probe constructs include : SoxRS regulator ( soxS ) , superoxide dismutase ( sodA ) , glucose-6-phosphate dehydrogenase ( zwf ) , superoxide resistant aco-nitase ( acnA ) , heat stable fumarase C ( fumC ) , multidrug efflux pump protein ( acrAB ) , hydroperoxidase I ( katG ) , and alkyl hydroperoxide reductase ( ahpC ) ( Table I ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15672380" target="_blank">15672380</a>]<br></li>
<li>The sets of genes in SoxRS regulon ( soxS , sodA , zwf , acnA , fumC , and acrAB ) and OxyR regulon ( katG and ahpC ) were selected to make the stress probes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15672380" target="_blank">15672380</a>]<br></li>
<li>Several global regulatory systems , such as ArcAB , Fnr , OxyR , and SoxRS are known to control the adaptation of E. coli to aerobic environments ( 5 -- 9 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10794600" target="_blank">10794600</a>]<br></li>
<li>These results show that the control of iron metabolism in E. coli is an integral part of the antioxidant defense-response and that the regulation of Fur by OxyR and SoxRS directly reﬂects the chemistry between iron and reactive-oxygen-species .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10419964" target="_blank">10419964</a>]<br></li>
<li>Furthermore , Fur is activated by OxyR and SoxRS , regulators of the antioxidant defense-response in E. coli , thereby reducing the production of highly reactive hydroxyl radicals and other toxic oxygen species through the Fenton reaction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15878120" target="_blank">15878120</a>]<br></li>
<li>Well studied regulatory cases include OxyR regulation of OxyS RNA ( oxidative stress ) ( 4 ) , Fur regulation of RyhB RNA ( iron limitation ) ( 5 ) , cAMP-CRP regulation of CyaR and Spot 42 RNA ( glucose limitation ) ( 6 -- 9 ) , OmpR regulation of MicF , OmrA , and OmrB RNAs ( osmotic shock ) ( 10 , 11 ) , E-mediated transcription of MicA and RybB RNAs ( envelope stress ) ( 2 , 12 ) , PhoPQ regulation of MgrR ( Mg2 / Ca2 transport and virulence ) ( 13 ) , and LuxO control of Qrr1 -- 4 ( quorum-sensing in Vibrio ) ( 14 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20075074" target="_blank">20075074</a>]<br></li>
<li>A wide range of environmental stimuli affect the expression of the sRNAs including oxidative stress ( OxyR activates OxyS ) ; cell surface stress ( RcsB activates RprA ) ; glycine status of the cell ( GcvA activates GcvB ) ; osmotic imbalance ( OmpR activates OmrA/B ) ; quality of carbon source/growth rate [ McaS ; transcription factor ( s ) unknown ] ; low temperature [ DsrA ; transcription factor ( s ) unknown ] and O tension ( ArcA-ArcB 2 represses ArzA under anaerobic conditions ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22250746" target="_blank">22250746</a>]<br></li>
<li>Some of these proteins are involved in the response of bacteria to environmental adversity , such as excessive oxidation in the case of OxyR ( Christman et al. , 1989 ; Storz et al. , 1990 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8168494" target="_blank">8168494</a>]<br></li>
<li>During H2O2 stress the OxyR system also induces Dps , a ferritin-like protein that sequesters loose iron and thereby diminishes the amount of Fenton-based DNA damage ( Grant et al. , 1998 , Ilari et al. , 2002 , Park et al. , 2005 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19400769" target="_blank">19400769</a>]<br></li>
<li>The OxyR regulon , activated by H2O2 , increases the level of protective enzymes such as hydroperoxidases , glutathione reductase Dps ( a small protein that protects DNA from oxi-dative damage ) and a small regulatory mRNA oxyS ( Altuvia et al. , 1994 ; Storz and Altuvia , 1994 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767577" target="_blank">9767577</a>]<br></li>
<li>After treatment of cells with H2O2 , OxyR induces a set of genes with clearly antioxidant activities .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11931550" target="_blank">11931550</a>]<br></li>
<li>YjiE differs from OxyR and SoxR as it does not regulate genes that are apparently involved in the degradation of ROS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22223481" target="_blank">22223481</a>]<br></li>
<li>OxyR , a transcription factor involved in stimulation of peroxide detoxification genes , is directly modified by NO via S-nitrosylation and assists in protecting the bacterium from the NO donor S-nitrosocysteine ( 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9852058" target="_blank">9852058</a>]<br></li>
<li>Thus , OxyR serves as a master regulator of S-nitrosylation , and alternative posttranslational-modifications of OxyR control distinct transcriptional responses .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22539721" target="_blank">22539721</a>]<br></li>
<li>The transcription factor OxyR , activated by oxidation under aerobic conditions , is also activated by S-nitrosylation under anaerobic conditions and thereby induces expression in a distinct regulon , which reveals that a transcriptional activator can independently control two distinct regulons through alternative posttranslational-modifications .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22539721" target="_blank">22539721</a>]<br></li>
<li>Cys199 was identified as the sole site of S-nitrosylation in OxyR by mass spectrometry ( see figs .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22539721" target="_blank">22539721</a>]<br></li>
<li>Thus , OxyR regulates S-nitrosylation in E. coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22539721" target="_blank">22539721</a>]<br></li>
<li>( F ) S-nitrosylation of OxyR during anaerobic growth on nitrate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22539721" target="_blank">22539721</a>]<br></li>
<li>( G ) Dynamic S-nitrosylation and denitrosylation of OxyR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22539721" target="_blank">22539721</a>]<br></li>
<li>Thus , protein-S-nitrosylation during ARN generates a nitrosative stress that is ameliorated by OxyR , and OxyR ( and possibly other genes in its regulon ) appears to protect against nitrosative stress at least in part by augmenting protein denitrosylation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22539721" target="_blank">22539721</a>]<br></li>
<li>Components of the regulon induced by S-nitrosylated OxyR ( including hcp ) protect against endogenous nitrosative stress , and protection is conferred at least in part by limiting S-nitrosylation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22539721" target="_blank">22539721</a>]<br></li>
<li>These results suggest that a change in the quality of OxyR protein rather than in the quantity is involved in the activation of the genes in the oxyR regulon , and that OxyR protein always functions as the repressor for the oxyR gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1864839" target="_blank">1864839</a>]<br></li>
<li>Furthermore , the OxyR protein functions as a repressor of its own gene ( Christman efa / .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8483417" target="_blank">8483417</a>]<br></li>
<li>Deletion of OxyR resulted in large increases in protein Snitrosylation , and S-nitrosylation of OxyR induced transcription from a regulon that is distinct from the regulon induced by OxyR oxidation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22539721" target="_blank">22539721</a>]<br></li>
<li>OxyR activates transcription on oxidation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2183352" target="_blank">2183352</a>]<br></li>
<li>OxyR protein activates transcription of genes in the oxyR regulon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1864839" target="_blank">1864839</a>]<br></li>
<li>The activated OxyR protein then directly stimulates transcription of genes involved in defense .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21378183" target="_blank">21378183</a>]<br></li>
<li>OxyR is an activator of a regulon of genes and additionally acts as an autorepressor ( 34 , 35 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10648536" target="_blank">10648536</a>]<br></li>
<li>Storz G , Altuvia S : OxyR regulon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11448770" target="_blank">11448770</a>]<br></li>
<li>To protect itself , H2O2-stressed E. coli activates the OxyR regulon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21143317" target="_blank">21143317</a>]<br></li>
<li>109 , 262-266 ( 1991 ) Purification and Characterization of the Escherichia coli OxyR Protein , the Positive Regulator for a Hydrogen Peroxide-Inducible Regulon1 Kazuyuki Tao , * 2 Kozo Makino , '' Shuji Yonei , * Atsuo Nakata , '' and Hideo Shinagawa ** J ` Laboratory of Radiation Biology , Faculty of Science , Kyoto University , Sakyo-ku , Kyoto 606 ; and `` Department of Experimental Chemotherapy , Research Institute for Microbial Diseases , Osaka University , Siuta , Osaka 565 Received for publication , September 18 , 1990 The Escherichia coli oxyR gene is required for the induction of a regulon that is inducible by hydrogen peroxide and confers resistance to oxidative stresses .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1864839" target="_blank">1864839</a>]<br></li>
<li>DISCUSSION The cloning and DNA sequence of the E. coli oxyR gene , a positive regulator of a regulon for defenses against oxidative stress , revealed that OxyR is a 34-kDa protein that shows significant homology to a family of bacterial regulatory proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2471187" target="_blank">2471187</a>]<br></li>
<li>Tao , K. , Makino , K. , Yonei , S. , Nakata , A. , and Shinagawa , H. ( 1989 ) Moiecuiar cloning and nucieotide sequencing of oxyR , the positive reguiatory gene of a regulon for an adaptive response-to-oxidative-stress in Escherichia coii \ homologies between OxyR protein and a famiiy of bacteriai activator proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8483417" target="_blank">8483417</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_60">
Molecular cloning and nucleotide sequencing of oxyR , the positive regulatory gene of a regulon for an adaptive response-to-oxidative-stress in Escherichia coli : homologies between OxyR protein and a family of bacterial activator proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11073934" target="_blank">11073934</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7868602" target="_blank">7868602</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8990289" target="_blank">8990289</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1999392" target="_blank">1999392</a>]<br></label>
<input id="rp_similarity_60", type="checkbox">
<div class="RP_similar">
<li>[ 9 ] Tao , K. ; Makino , K. ; Yonei , S. ; Nacata , A. ; Shinagawa , H. Molecular cloning and nucleotide seguencing of oxyR , the positive regulatory gene of a regulon for an adaptive response-to-oxidative-stress in Escherichia coli : homologies between OxyR protein and a family of bacterial activator proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11440837" target="_blank">11440837</a>]<br></li>
<li>[ 8 ] Tao , K. , Makino , K. , Yonei , S. , Nacata , A. and Shinagawa , H. ( 1989 ) Molecular cloning and nucleotide sequencing of oxyR , the positive regulatory gene of a regulon for an adaptive response-to-oxidative-stress in Escherichia coli : Homologies between OxyR protein and a family of bacterial activator proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10802173" target="_blank">10802173</a>]<br></li></div>
<li>Heni-koff et al. ( 29 ) have recently recognized that these proteins and others [ E. coli IlvY , regulator of ilvC and other genes in the isoleucine-valine biosynthetic pathway ( 30 ) ; E. coli and S. typhimurium CysB , positive regulator of the cysteine regulon ( 31 ) ; S. typhimurium MetR , regulator of the metE and metH genes ( 32 ) ; and the Enterobacter cloacae ampR regulator of the ampC gene , which confers resistance to cepha-losporin ( 33 ) ] comprise a family of bacterial regulatory OxyR - A HT .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2471187" target="_blank">2471187</a>]<br></li>
<li>For example , a family of regulatory proteins such as OxyR , LysR , CysB , NodD , and MetR , that act as positive regulator for other genes , were shown to negatively regulate their own expression ( 30 , 32 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1715982" target="_blank">1715982</a>]<br></li>
<li>For example , the reaction of hydrogen peroxide with nucleophilic cysteine residue sulfhydryls in the transcription regulators OxyR and OhrR of bacteria and Orp1 of Saccharomyces cerevisiae generates cysteine sulfenic acid ( R-S-OH ) , an intermediate that initiates a series of events leading to expression of antioxidant genes .2,3 Many of these cysteine residue sulfhydryls have been shown to have a pKa below that of free cysteine ( 8.3 ) , suggesting that low pKa cysteine residues are redox active .2,4 Sulfur trafﬁcking pathways are among the systems that are induced during oxidative stress , and these pathways also use nucleo-philic cysteine residue sulfhydryls.5-7 In the ﬁrst step of sulfur trafﬁcking , a cysteine desulfurase abstracts inorganic sulfur from the cysteine substrate using its own reactive cysteine residue sulfhydryl , thereby forming a persulﬁde ( R-S-SH ) intermediate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20936830" target="_blank">20936830</a>]<br></li>
<li>The Escherichia coli yhjA gene , encoding a predicted cytochrome c peroxidase , is regulated by FNR and OxyR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20522491" target="_blank">20522491</a>]<br></li>
<li>Microbiology ( 2007 ) , 153 , 1499 -- 1507 DOI 10.1099 / mic .0.2006 / 004838-0 The Escherichia coli yhjA gene , encoding a predicted cytochrome c peroxidase , is regulated FNR and OxyR by Jonathan D. Partridge , Robert K. Poole and Jeffrey Green Department of Molecular Biology and Biotechnology , The University of Shefﬁeld , Shefﬁeld S10 2TN , UK Correspondence Jeffrey Green jeff.green@shefﬁeld.ac.uk The Escherichia coli FNR protein is an oxygen-responsive global transcription factor , and OxyR is a key regulator of the peroxide stress response .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17464064" target="_blank">17464064</a>]<br></li>
<li>Responses of OxyR Regulon Genes Similarly , time course studies were also performed for stress genes in OxyR Regulon , katG and ahpC , in response to various concentrations of paraquat ( 10 -- 500 AM ) and hydrogen peroxide ( 0.5 -- 10 mM ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15672380" target="_blank">15672380</a>]<br></li>
<li>The hps-7DoxyR double mutant exhibited a synergistic hypersensitive phenotype and had an approximately 50-fold-reduced viability compared to that of the wild-type strain , indicating that other genes ( such as hps-7 ) may be involved in hydrogen peroxide stress response independent of OxyR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8990283" target="_blank">8990283</a>]<br></li>
<li>Control of ahpC expression by the OxyR global regulator in response to L-ascorbate concentration is consistent with the formation of hydrogen peroxide under our experimental conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17906139" target="_blank">17906139</a>]<br></li>
<li>Identiﬁcation and characterization of hydrogen peroxide-sensitive mutants of Escherichia coli : genes that require OxyR for expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9294462" target="_blank">9294462</a>]<br></li>
<li>By measuring gene-expression in isogenic wild-type and oxyR deletion strains , we conﬁrmed that the peroxide response regulator OxyR activates most of the highly hydrogen peroxide-inducible genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11443091" target="_blank">11443091</a>]<br></li>
<li>In response-to-hydrogen-peroxide ( H O ) or S-nitrosothiols , OxyR triggers 2 2 the expression of multiple protective genes ( 18 -- 20 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16030151" target="_blank">16030151</a>]<br></li>
<li>For example , the Escherichia coli OxyR protein is the direct link between hydrogen peroxide stress and the activation of defense genes such as catalase , i.e. OxyR is both a sensor and a transcriptional activator ( for a review , see Hidalgo and Demple , 1997a ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9564045" target="_blank">9564045</a>]<br></li>
<li>In Escherichia coli , the key regulator of the adaptive response to protect against hydrogen peroxide is the OxyR transcription factor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10760142" target="_blank">10760142</a>]<br></li>
<li>OxyR , a positive regulator of hydrogen peroxide-inducible genes in Escherichia coli and Salmonella typhimurium , is homologous to a family of bacterial regulator proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1943993" target="_blank">1943993</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_77">
Puriﬁcation and characterization of the Escherichia coli OxyR protein , a positive regulator for a hydrogen peroxide-inducible regulon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8990283" target="_blank">8990283</a>]<br></label>
<input id="rp_similarity_77", type="checkbox">
<div class="RP_similar">
<li>Puriﬁcation and characterization of the Escherichia coli OxyR protein , the positive regulator for a hydrogen peroxide-inducible regulon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7868602" target="_blank">7868602</a>]<br></li></div>
<li>Amino acids involved in a deﬁned activity regulation scheme would be expected to be conserved [ e.g. , the active-site Cys in Hsp33 , oxidative stress regulator OxyR , NemR , and HypR ( 16 -- 18 , 33 ) , and the Met in calmodulin kinase II ( 32 ) ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23690622" target="_blank">23690622</a>]<br></li>
<li>Cell , Vol .78 , 897-909 , September9 , 1994 , Copyright © 1994by CellPress Redox-Dependent Shift of OxyR-DNA Contacts along an Extended DNA-Binding Site : A Mechanism for Differential Promoter Selection Michel B. Toledano , * Ines Kullik , * Frank Trinh , * t Paul T. Baird , * ~ Thomas D. Schneider , § and Gisela Storz * * Cell Biology and Metabolism Branch National Institute of Child Health and Human Development National Institutes of Health Bethesda , Maryland 20892 § Laboratory of Mathematical Biology National Cancer Institute Frederick , Maryland 21702 Summary The redox-sensitive OxyR protein activates the transcription of antioxidant defense genes in response-to-oxidative-stress and represses its own expression under both oxidizing and reducing conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8087856" target="_blank">8087856</a>]<br></li>
<li>Interestingly , neither SoxRS nor OxyR , which have been persistently proposed to be critical NO stress response sensorregulators ( 49 , 50 ) appear to be involved in the regulation of either hmp or norVW in E. coli ( Fig. 5 ) ( 48 ) .2 Future investigations will aim to further clarify the diverse roles and mechanisms of NO defense genes , enzymes , and regulators in microbial adaptations to NO in vitro and in various models of infection .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12529359" target="_blank">12529359</a>]<br></li>
<li>SoxRS ( Escherichia coli ) OxyR ( Escherichia coli ) Oxidative stress ( O2̇ ¯ ) Oxidative stress ( H2O2 ) Redox reaction Fe-S -- Redox reaction at cysteine residu Table 2 FNR regulated genes , regulatory signals , and additional regulators acting at the relevant promoters in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8588737" target="_blank">8588737</a>]<br></li>
<li>Lrp , leucine-responsive protein ; OxyR , oxidation stress regulator ; RpoS , RNA polymerase sigma factor for stationary phase ; OmpR , outer membrane protein regulator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7559363" target="_blank">7559363</a>]<br></li>
<li>Discussion The Ag43 protein of E. coli is expressed in a phasevariable manner , and this phase variation has been shown to be dependent on the OxyR transcriptional regulator and on deoxyadenosine methyltransferase ( Dam ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11972787" target="_blank">11972787</a>]<br></li>
<li>The second regulator of Ag43 phase variation is OxyR , which is a DNA binding protein that acts as a global regulator to mediate protection against oxidative stress .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12029051" target="_blank">12029051</a>]<br></li>
<li>Our previous data suggested that YaaA helped to control the iron level during peroxide stress ; hence , we wanted to check whether YaaA worked with other proteins that also control iron-homeostasis during the OxyR response , including the iron uptake regulator ( Fur ) , iron-scavenging protein ( Dps ) , and manganese transporter ( MntH ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21378183" target="_blank">21378183</a>]<br></li>
<li>The Suf pathway has been suggested to function under conditions of oxidative and iron stress ( Lee et al. , 2003 ; 2004 ; Outten et al. , 2004 ) via OxyR and Fur , respectively , but the ﬁnding that IscR also controls the Suf pathway suggests that this alternate pathway is involved in Fe-S biogenesis under normal aerobic growth conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16677314" target="_blank">16677314</a>]<br></li>
<li>Another transcription factor , OxyR , induces expression of mntH during periods of H2O2 stress ( Kehres et al. , 2000a , Kehres et al. , 2002b ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19400769" target="_blank">19400769</a>]<br></li>
<li>In E coli , intracellular H2O2 stress activates the OxyR transcription factor , which then directly stimulates synthesis of approximately two dozen proteins ( Zheng et al. , 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19400769" target="_blank">19400769</a>]<br></li>
<li>Many of the proteins involved in protection against H2O2 stress are positively regulated by the OxyR transcription factor , which is fully activated by the H2O2 that accumulates in Hpx − mutants ( Zheng et al. , 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21338418" target="_blank">21338418</a>]<br></li>
<li>The OxyR protein acts as a transcriptional regulator initiating , directly or indirectly , the synthesis of more than 40 gene products to counter the effects of peroxide stress , although transcription of some of these genes is also altered during exposure to heat shock or redox-cycling agents ( Greenberg & Demple , 1989 ; Storz & Zheng , 2000 ; Walkup & Kogoma , 1989 ; Zheng et al. , 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17464064" target="_blank">17464064</a>]<br></li>
<li>Escherichia coli cells utilize two transcription factors OxyR and SoxR to sense the oxidants and then induce various genes against oxidative stress that are involved in removing oxidants , repairing damaged cell components and maintaining reducing conditions in the cell ( Bauer et al. , 1999 ; Storz and Imlay , 1999 ; Pomposiello and Demple , 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12773378" target="_blank">12773378</a>]<br></li>
<li>The OxyR protein is a bifunctional protein , acting as both a sensor cf oxidative stress and a transcription activator of genes that respond to the stress ( Storz et ai .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8483417" target="_blank">8483417</a>]<br></li>
<li>Upon oxidation , OxyR becomes a transcriptional activator for multiple genes involved in the protection against oxidative stress ( Christman et al. , 1989 ; Zheng et al. , 1998 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10692164" target="_blank">10692164</a>]<br></li>
<li>Under conditions of oxidative stress , OxyR undergoes reversible formation of a disulphide bond , allowing it to activate the transcription of genes required for oxidative stress defence ( Zheng et al. , 1998 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11972787" target="_blank">11972787</a>]<br></li>
<li>The antioxidant adaptive response to sublethal UVA irradiation may involve the induction of oxidative defens pathways that include OxyR , Fur and SoxRS , leading to coordinated gene-expression of antioxidant enzymes including HPI , HPI , FeSOD , MnSOD and GR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16183297" target="_blank">16183297</a>]<br></li>
<li>Keywords : oxidative stress response ; SoxRS regulon ; OxyR regulon ; green fluorescent protein ; quantitative and kinetic monitoring INTRODUCTION Oxidative stresses are generated through a variety of mechanisms including normal aerobic metabolism , exposure to ultraviolet light , metal ions , and oxidative compounds , or released from macrophages during infection ( Halliwell and Gutteridge , 1989 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15672380" target="_blank">15672380</a>]<br></li>
<li>This unusual use of FNR and OxyR to activate expression suggests that yhjA plays a speciﬁc role in countering oxidative stress in E. coli , possibly in particular micro-aerobic environments .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17464064" target="_blank">17464064</a>]<br></li>
<li>During normal growth as well as upon oxidative stress , OxyR also acts to repress its own expression ( Christman et al. , 1989 ; Tao et al. , 1991 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8087856" target="_blank">8087856</a>]<br></li>
<li>These results show that OxyR represses fhuF expression under conditions of oxidative stress .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11443092" target="_blank">11443092</a>]<br></li>
<li>Therefore , OxyR can oxyR momR act to repress and expression during normal growth and activate katG , ahpC , dps , and gorAupon oxidative stress ( Figure 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8087856" target="_blank">8087856</a>]<br></li>
<li>The shift in binding also allows OxyR to repress its own expression under both oxidizing and reducing conditions , while only activating katG , ahpC , gorA , and dps upon oxidative stress .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8087856" target="_blank">8087856</a>]<br></li>
<li>Recently , workers in this laboratory have provided evidence ( Henderson et al. , 1997b ; Henderson and Owen , 1999 ; Roche et al. , 2001 ) that the Ag43 phase switch is regulated by a novel mechanism involving Dam and OxyR , a LysR-type transcriptional regulator better known for its role in activating the expression of antioxidant defence genes in response-to-oxidative-stress ( Toledano et al. , 1994 ; Kullik et al. , 1995 ; Zheng et al. , 1998 ; Åslund et al. , 1999 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11972787" target="_blank">11972787</a>]<br></li>
<li>HPI is an adaptive enzyme regulated by OxyR , a major regulatory transcription factor responding to oxidative stress w5 ,6 x. HPII is expressed maximally in stationary phase , and its expression is completely dependent on the product of katF w2 ,7 x. Although katF was first identified as a positive regulator of HPII w2 ,7 x , it has been found to control the expression of many stationary phase specific and osmotically regulated genes w8x .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9199247" target="_blank">9199247</a>]<br></li>
<li>Additional regulators acting on some of these genes include transcription factors speciﬁc for certain stress response , such as OxyR , and also some more global regulators , such as H-NS or Lrp ( Hengge-Aronis , 1996b and references herein ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9663683" target="_blank">9663683</a>]<br></li>
<li>Additionally , these genes are regulated by transcription factors speciﬁc for certain stress responses ( e.g. , OxyR , which is involved in the oxidative stress response ) or more-global regulators , such as H-NS , IHF , cyclic AMP ( cAMP ) - cAMP receptor protein ( CRP ) , and Lrp ( 104 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16760308" target="_blank">16760308</a>]<br></li>
<li>OxyR is normally involved in protection against oxidative stress as it activates a regulon of peroxide-inducible genes ( 21 , 22 ) , but it also represses the ﬂu gene ( 12 , 15 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10648536" target="_blank">10648536</a>]<br></li>
<li>Expression of katG following H202 induction ( 21 , 34 ) is positively regulated by OxyR ( 8 , 30 ) , a protein responsible for the positive regulation of a regulon involved in protection against oxidative stress .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3045098" target="_blank">3045098</a>]<br></li>
<li>In contrast , expression analysis has demonstrated that the suf operon is controlled by OxyR and Fur , an H2O2-sensing regulator and an iron-dependent repressor , respectively , suggesting a stress response function for the SUF machinery under conditions of oxidative stress and iron limitation ( 36 , 37 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15496591" target="_blank">15496591</a>]<br></li>
<li>One target gene , identiﬁed using this approach and experimentally conﬁrmed to be regulated by OxyR , encodes the iron metab-olism regulator Fur , demonstrating coordinate regulation between oxidative stress response and iron metabolism ( 23 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11443092" target="_blank">11443092</a>]<br></li>
<li>Three regulons and one sigma factor are known to be involved in the cellular response to oxi-dative stress : OxyR ( oxidative stress response regulon ) , SoxRS ( superoxide stress regulon ) , Fur ( ferric uptake regulator ) , and RpoS ( stationary phase sigma factor ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15878120" target="_blank">15878120</a>]<br></li>
<li>OxyR regulon controls lipid peroxidation-mediated oxidative stress in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23831895" target="_blank">23831895</a>]<br></li>
<li>The primary regulators of the oxidative stress response , OxyR and SoxRS , stimulate Escherichia coli fur transcription from different promoters , which complies with the function of Fur in protecting cells against oxidative damage ( Zheng et al , 1999 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17268550" target="_blank">17268550</a>]<br></li>
<li>We found that regulators of the Escherichia coli responses to oxidative stress , OxyR and SoxRS , activate the expression of Fur , the global repressor of ferric ion uptake .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10419964" target="_blank">10419964</a>]<br></li>
<li>DISCUSSION We have shown that the primary regulators of the E. coli response-to-oxidative-stress , OxyR and SoxRS , modulate the levels of the ferric uptake regulator Fur .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10419964" target="_blank">10419964</a>]<br></li>
<li>In addition , upon oxidative stress , transcriptional dual regulator OxyR induces katG42 and the regulatory RNA OxyS32 which inhibits translation of the rpoS message43 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21516113" target="_blank">21516113</a>]<br></li>
<li>The katG gene has been shown to be part of the OxyR regulon , which responds to oxidative stress , including the presence of hydrogen peroxide ( 2 , 25 ) as part of a broader response illicited by redox-cycling reagents ( 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2254248" target="_blank">2254248</a>]<br></li>
<li>Thus , OxyR controls the expression of a large set of genes that form a nitrosative stress regulon that is distinct from the OxyR regulon activated by oxidative stress .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22539721" target="_blank">22539721</a>]<br></li>
<li>In the presence of H2O2 , OxyR is oxidized ( ox ) and induces the expression of oxidative stress response genes like catalases .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23139412" target="_blank">23139412</a>]<br></li>
<li>Thus , it is possible that OxyR represses these genes in response-to-oxidative-stress .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11443091" target="_blank">11443091</a>]<br></li>
<li>By contrast , genes in the known oxidative stress regulon of OxyR ( katG , sufA and grxA ) ( 26 ) showed increased transcription in response to oxidants ( fig .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22539721" target="_blank">22539721</a>]<br></li>
<li>OxyR directly detects oxidative stress and is a transcriptional activator of stress response genes including katG , encoding hydroperoxidase I ( HPI ) [ 5 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15878120" target="_blank">15878120</a>]<br></li>
<li>As an antioxidant defense pleiotropic regulator , OxyS is positively regulated by OxyR , which is a transcriptional activator under oxidative stress [ 14 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21106118" target="_blank">21106118</a>]<br></li>
<li>The OxyR protein is a transcriptional activator of genes important for the defense against oxidative stress in Esche-richia coli and Salmonella typhimurium ( 14 , 37 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7868602" target="_blank">7868602</a>]<br></li>
<li>Introduction The OxyR transcriptional regulator of Escherichia coli and Salmonella typhimurium activates the expression of defense genes in response-to-oxidative-stress .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8087856" target="_blank">8087856</a>]<br></li>
<li>OxyR is also a re-dox-sensitive transcriptional activator involved in oxidative stress response by E. coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8588737" target="_blank">8588737</a>]<br></li>
<li>The expression of trxC is regulated by the transcriptional activator OxyR in response-to-oxidative-stress .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10644706" target="_blank">10644706</a>]<br></li>
<li>Expression of Dps is known to be regulated by OxyR , in response-to-oxidative-stress ( 2 ) , and three nucleoid proteins , IHF , Fis , and H-NS , in response to growth phase ( 2 , 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21239586" target="_blank">21239586</a>]<br></li>
<li>This demonstrates that IscR positively contributes to induce the sufA gene as much as OxyR does in response-to-oxidative-stress .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16824106" target="_blank">16824106</a>]<br></li>
<li>In summary , these data demonstrate that ychF is negatively regulated by OxyR in response-to-oxidative-stress .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23139412" target="_blank">23139412</a>]<br></li>
<li>Our data suggest that OxyR is the sole transcriptional regulator of yaaA in response-to-oxidative-stress .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21378183" target="_blank">21378183</a>]<br></li>
<li>OxyR is a global regulator that is oxidized in response-to-oxidative-stress ( 38 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20118257" target="_blank">20118257</a>]<br></li>
<li>Zheng M , Doan B , Schneider TD , Storz G : OxyR and SoxRS regulation of fur .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21272324" target="_blank">21272324</a>]<br></li>
<li>It regulates the expression of the superoxide dismutases ( repression of the manganese superoxide dismutase and activation of the iron one ) ( 22 , 23 ) , and fur expression is under control of OxyR and SoxRS ( 24 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12475930" target="_blank">12475930</a>]<br></li>
<li>However , given our previous ﬁndings that OxyR regulates the expression of the iron metabolism regulator Fur , possible OxyR regulation of a putative iron reductase was of interest .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11443092" target="_blank">11443092</a>]<br></li>
<li>In this study , we report the identiﬁcation of several genes that require OxyR for expression , including hemF ( encoding coproporphyrinogen III oxidase ) , rcsC ( encoding a sensor-regulator protein of capsular polysaccharide-synthesis genes ) , and an open reading frame , f497 , that is similar to arylsulfatase-encoding genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8990283" target="_blank">8990283</a>]<br></li>
<li>Recently , Muk-hopadhyay and Schellhorn ( 14 ) demonstrated with gene fusions that OxyR activates transcription from the promoters of hemF ( encoding coprophyrinogen III oxidase ) , rcsC ( encoding a sensor regulator protein of capsular polysaccharide-synthesis ) , and a gene encoding a putative f497 protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9294462" target="_blank">9294462</a>]<br></li>
<li>Van Dyk et al. ( 2001a ) showed that reporter gene fusions could be an important alternative to other gene-expression analyses ( e.g. , DNA microarray ) by analyzing the heat shock , SOS , OxyR , SoxRS , and cya/crp stress-responsive regulons via luxCDABE fusions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15672380" target="_blank">15672380</a>]<br></li>
<li>Wei , Q. , Minh , P. N. , Dötsch , A. , Hildebrand , F. , Panmanee , W. , Elfarash , A. , Schulz , S. , Plaisance , S. , Charlier , D. , Hassett , D. , Häussler , S. , and Cornelis , P. ( 2012 ) Global regulation-of-gene-expression by OxyR in an important human opportunistic pathogen .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23139412" target="_blank">23139412</a>]<br></li>
<li>hmp gene-expression in the presence of NOz is independent of the SoxS , OxyR , and FNR transcription factors and instead relies on inactivation of the iron-de-pendent Fur repressor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9852058" target="_blank">9852058</a>]<br></li>
<li>The expression level of hcaR was also measured in soxR and oxyR mutants because SoxR [ 5 ] and OxyR [ 31 ] regulators modulate expression of many genes including sodA , gor , katG , ahpC and ahpF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15808934" target="_blank">15808934</a>]<br></li>
<li>We also identiﬁed several genes , including uxuA , encoding mannonate hydrolase , whose expression might be repressed by OxyR , since their expression was elevated in the DoxyR mutant strain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11443091" target="_blank">11443091</a>]<br></li>
<li>OxyR is an activator of katG and ahpCF expression and negatively regulates its own expression ( 10 , 13 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2183352" target="_blank">2183352</a>]<br></li>
<li>Upon oxidation , the OxyR regulator activates the expression of defense genes which include katG , ahpCF ( 13 ) , and a gene encoding a small nontranslated RNA designated oxyS ( 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7986015" target="_blank">7986015</a>]<br></li>
<li>Additional genes are indirectly controlled by OxyR as it induces the synthesis of OxyS , a small non-coding RNA that controls the expression of additional genes ( 36 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23139412" target="_blank">23139412</a>]<br></li>
<li>In addition , OxyR regulates the expression of the grxA and gor genes , coding for glutaredoxin 1 and glutathione oxidoreductase , respectively ( 29 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10644706" target="_blank">10644706</a>]<br></li>
<li>The class III fusions indicate that OxyR acts as a repressor and represses a group of genes in E. coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8990283" target="_blank">8990283</a>]<br></li>
<li>OxyR represses its own expression , as well as transcription of the Mu phage mom and the E. coli ﬂu genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11443092" target="_blank">11443092</a>]<br></li>
<li>To determine whether OxyR was the sole sensor of oxidation and the only component required to activate oxyR-regulated gene-expression , we examined the ability of purified OxyR to regulate expression of oxyR-regulated genes in more defined transcription assays .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2183352" target="_blank">2183352</a>]<br></li>
<li>To determine whether INH acts to oxidize OxyR , leading to the induction of OxyR-regulated genes , the expression of the oxyS gene in response to INH was assayed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7986015" target="_blank">7986015</a>]<br></li>
<li>The oxyR gene encodes the transcription factor OxyR , which regulates the expression of several antioxidant genes ( e.g. , those for catalase hydroperoxidase I and glutathione reductase ) in re sponse to hydrogen peroxide ( 2 , 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9922271" target="_blank">9922271</a>]<br></li>
<li>mechanism by which OxyR regulates the expression of the oxyR regulon , we first assayed the ability of extracts from OxyR-overpro-ducing cells to activate expression of the ahp genes in an in vitro transcription-translation assay .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2183352" target="_blank">2183352</a>]<br></li>
<li>OxyR also controls the expression of seven or eight other H2O2-inducible proteins in Salmonella typhimurium and E. coli ( 5 , 15 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8990289" target="_blank">8990289</a>]<br></li>
<li>A key question in OxyR - and Dam-dependent phase variation of Ag43 is whether the switch in expression phase requires a speciﬁc cell-cycle or metabolic event .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12644490" target="_blank">12644490</a>]<br></li>
<li>Effect of OxyR on b-galactosidase expression of hps : : lacZ fusions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8990283" target="_blank">8990283</a>]<br></li>
<li>Effect of OxyR on growth phase-dependent expression of hps : : lacZ fusions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8990283" target="_blank">8990283</a>]<br></li>
<li>( C ) Dependence of hcp expression during ARN on OxyR Cys199 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22539721" target="_blank">22539721</a>]<br></li>
<li>rpoS - , OxyR - , and IHF-dependent regulation of dps expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7559363" target="_blank">7559363</a>]<br></li>
<li>OxyR - , and IHP-dependent regulation of dps expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7715442" target="_blank">7715442</a>]<br></li>
<li>Apart from OxyR , the Dam protein , responsible for meth-ylating GATC sites of DNA , is also involved in the expression of Ag43 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10648536" target="_blank">10648536</a>]<br></li>
<li>OxyR also has been shown to be a repressor of its own expression as well as that of fhuF ( encoding a ferric ion reductase ) and ﬂu ( encoding the antigen 43 outer membrane protein ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11443091" target="_blank">11443091</a>]<br></li>
<li>Our current dat show that OxyR also represses the expression of fhuF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11443092" target="_blank">11443092</a>]<br></li>
<li>The expression of this gene , which is predicted to encode a CCP , is shown to be driven from an FNR-dependent class II promoter that is also dependent on OxyR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17464064" target="_blank">17464064</a>]<br></li>
<li>In conclusion , it has been shown that the YhjA protein is likely to be a CCP that contributes to the survival of E. coli upon exposure to peroxide stress , and that expression of the yhjA gene is driven from an FNR-dependent class II promoter that also requires OxyR for activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17464064" target="_blank">17464064</a>]<br></li>
<li>OxyR is important for maintaining and establishing the sequestration that is essential for phase variation and imposes heritability on the gene-expression phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20118257" target="_blank">20118257</a>]<br></li>
<li>The increased expression from a heterologous oxyR : : lacZ gene fusion in an oxyR deletion strain compared to an oxyR ' strain suggests that OxyR is also a negative regulator of its own expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2471187" target="_blank">2471187</a>]<br></li>
<li>Overall , the pattern of OxyR protein expression paralleled the transcriptional behavior of the oxyR gene reasonably well .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9324269" target="_blank">9324269</a>]<br></li>
<li>The result suggests that OxyR protein represses the expression of the oxyR gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1864839" target="_blank">1864839</a>]<br></li>
<li>At the oxyR-binding site , OxyR not only represses expression of its own gene , but also activates the expression of oxyS , a gene transcribed divergently from oxyR ( S. Altuvia and G. S. , unpublished data ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8087856" target="_blank">8087856</a>]<br></li>
<li>By stimulating oxyR expression , putrescine also caused an increase in the expression of katG , the target gene for OxyR ( Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11479716" target="_blank">11479716</a>]<br></li>
<li>DISCUSSION The results presented here show an oscillation in oxyR expression during aerobic growth in E. coli , including a pronounced increase in OxyR during early exponential phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9324269" target="_blank">9324269</a>]<br></li>
<li>The expression of Ag43 is phase variable , and the shift from Ag43 ( ON ) to Ag43 ( OFF ) cells results from combined regulation of the ﬂu gene by the Dam methylase and the transcriptional regulator OxyR ( 21 , 22 , 42 , 50 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659678" target="_blank">15659678</a>]<br></li>
<li>Escherichia coli OxyR protein represses the unmethylated bacteriophage Mu mom operon without blocking binding of the transcriptional activator C. Nucleic Acids Res .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12029051" target="_blank">12029051</a>]<br></li>
<li>Bolker and Kahmann have also found that OxyR is a repressor of the mom gene ( encoding a DNA-modification function ) of phage Mu ( 11 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2183352" target="_blank">2183352</a>]<br></li>
<li>During normal growth , OxyR also acts as a repressor , negatively autoregulating its own expression and the expression of the Mu phage mom gene ( 6 , 14 , 38 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7868602" target="_blank">7868602</a>]<br></li>
<li>OxyR protein also represses its own synthesis ( 10 , 39 , 42 ) as well as that of the Mu phage mom gene ( 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8990283" target="_blank">8990283</a>]<br></li>
<li>OxyR can act as a transcriptional activator ( 39 , 42 , 43 ) and a repressor , repressing its own expression ( 10 ) and that of the mom gene of phage Mu ( 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8990283" target="_blank">8990283</a>]<br></li>
<li>LTTRs typically form dimers and tetramers ( e.g. OxyR ( 17 ) , CysB ( 18 ) , TsaR ( 19 ) , nitrogen assimilation control protein ( 20 ) ) , but also octamers ( CrgA ( 21 ) ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22223481" target="_blank">22223481</a>]<br></li>
<li>Also , Lrp , RpoS , and OxyR have been reported previously ( 8 , 21 , 28 ) to be involved in the regulation of some Gsps reported here to be IHF dependent ( Table 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7559363" target="_blank">7559363</a>]<br></li>
<li>Both OxyR and glutathione are also involved in the regulation of katG ( HPI ) [ 13,33 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16183297" target="_blank">16183297</a>]<br></li>
<li>Our computational search predicted an OxyR binding site at position 4549044 between uxuAB ( encoding mannonate hydrolase and mannon-ate oxidoreductase ) and the divergent gntP gene ( encoding a possible gluconate permease ) ( Fig. 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11443091" target="_blank">11443091</a>]<br></li>
<li>For comparison , Fig. 2B shows sequence alignments within DBDs of some LTTRs ( CbnR , CysB , OxyR and GcvA ) and residues involved in DNA binding ( blue ) or pc function ( red ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15255893" target="_blank">15255893</a>]<br></li>
				</div></ul><br>
			<label class="collapse_SIT" for="_5"><font size="5px">OxyR binding site</font></label>
				<input id="_5" type="checkbox">
				<div class="SIT"><br><ul><li>- e. y e oAlaLeuSerGlnGlnlleArgGlnLeuGluGluSerLjeuGlyValProLeuPheA3pAr 4-247 7-250 10.8 26.6 LysR 25-55 26-56 5.9 51.6 90-151 92-153 5.6 32.3 4.7 29.9 CysB ( E. coli ) TAGCGGGCGAACGATTCGTCTCACTGATGCAGGAGAAGTCTGGCGACAGi * ACGCCAuCCG gSerGlyArgThrrleArgLeuThrAspAlaGlyGluValTrpArgGlnTyrAlaSerAr CysB ( Salmonella typhimurium ) 25-55 26-56 5.6 48.4 GGCGTTACAGGAACTGGGGGCGGGTAAACGGGCGATTCATGATGTTGCCGATCTGACGCG gAlaLeuGlnGluLeuGlyAlaGlyLysArgAlaIleHisAspValAlaAspLeuThrAr 90-151 92-153 5.6 30.6 4.7 30.3 AGGATCGCTGCGTATCGCCGTCACCCCCACCTTTACGAGCTACTTTATCGGCCCCTTAAT gGlySerLeuArgrleAlaValThrProThrPheThrSerTyrPhel leGlyProLeuMe 4-185 4-185 10.5 32.0 IlvY ( E. coli ) GGCGGATTTCTATGCGCGCTATCCCAGCATCACGCTCCAGCTACAGGAAATGTCGCAGGA tAlaAspPheTyrAlaArgTyrProSerIleThr ' euGlnLeuGlnGluMeetSerGlnGl 9-58 14-63 7.5 46.0 AmpR ( Enterobacter ) 93-124 96-127 3.6 40.6 AmpR ( Rhodobacter ) 15-50 20-55 5.7 52.8 3.7 28.6 uLysIleGluA3pMetLeuCysArgA3pGluLeuA3pValGlyIleAlaPheAlaProVa GCATTCGCCGGAGCTGGAGGCAATTCCTTTACTGACAGAAAGTTTAGCGTTAGTCGTGGC lHi3SerProGluLeuGluAlaIleProLeuLeuThrGluSerLeuAlaLeuValValAl GCAACATCATCCGCTGGCCGTCCATGAACAGGTGGCGTTGAGTCGCTTGCATGATGAAAA aGlnHi3HisProLeuAlaValHisGluGlnValAlaLeuSerArgLeuHisA3pGluLy ACTGGTCCTGCTCAGCGCGGAATTTGCCACCAGAGAGCAAATTGACCACTACTGCGAGAA 3LeuValLeuLeuSerAlaGluPheAlaThrArgGluGlnIleAspHi3TyrCy3GluLy OxyR ( E. coli ) 1-242 1-242 14.4 31.4 a Sequences were analyzed by using the Molecular Biology Information Resource site ( Baylor College of Medicine , Houston , Tex. ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1592818" target="_blank">1592818</a>]<br></li>
				<label class="SIT_collapse_similarity" for="sit_similarity_2">
In contrast , reduced OxyR contacts two pairs of adjacent major grooves separated by one helical turn .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8087856" target="_blank">8087856</a>]<br></label>
<input id="sit_similarity_2", type="checkbox">
<div class="SIT_similar">
<li>In contrast , reduced OxyR binds to two pairs of adjacent major grooves separated by one helical turn .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8087856" target="_blank">8087856</a>]<br></li>
<li>While oxidized OxyR binds four adjacent major grooves ( grooves 1,2 , 3 , 4 ) , reduced OxyR binds two pairs of adjacent major grooves separated by one helical turn ( grooves 1 , 2 , 4 , 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8087856" target="_blank">8087856</a>]<br></li>
<li>Oxidized OxyR binds in four adjacent major grooves , while reduced OxyR contacts two pairs of adjacent major grooves separated by one helical turn .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7868602" target="_blank">7868602</a>]<br></li>
<li>Oxidized OxyR binds to four adjacent major grooves , while reduced OxyR binds to two pairs of adjacent major grooves separated by one helical turn .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8087856" target="_blank">8087856</a>]<br></li></div>
<li>This pattern of protection further supports the conclusion that oxidized OxyR binds to four major grooves protecting the five adjacent minor grooves .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8087856" target="_blank">8087856</a>]<br></li>
<li>Third , the missing nucleosides are located approximately 10 bp apart , parallel to the helical axis of the DNA , implying that OxyR binds on one face of the helix in four successive grooves .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8087856" target="_blank">8087856</a>]<br></li>
<li>Oxidized OxyR binds as a tetramer to a sequence of degenerate twofold symmetry , making contact at four adjacent major grooves on one face of the DNA ( 5 , 48 , 52 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11741843" target="_blank">11741843</a>]<br></li>
<li>Here , we report that oxidized OxyR recognizes a motif comprised of four ATAGnt elements spaced at 10 bp intervals and contacts these elements in four adjacent major grooves on one face of the DNA helix .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8087856" target="_blank">8087856</a>]<br></li>
<li>These experiments revealed that oxidized OxyR binds to an ATAGnt motif on one face of the DNA helix at four adjacent major grooves .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8087856" target="_blank">8087856</a>]<br></li>
<li>To test whether the FNR sites upstream of yecI , yjhB , yfgF , yhjA and smf contributed to regulation of these genes , the corresponding promoter regions ( e.g. MC4100 oxyR-Kan Zheng et al. ( 1999 ) Lab collection JRG1728 JRG3462 MC1000 D ( tyrR-fnr-rac-trg ) 17 zdd-230-Tn9 CAG627 pGS771 Meng et al. ( 1998 ) This work MC1000/lRZ5 ( PyhjA -- lacZ ) from pGS1634 JRG5066 JRG5067 JRG1728/lRZ5 ( PyhjA -- lacZ ) from pGS1634 W3110 DyhjA This work JRG5354 JRG5393 MC1000 oxyR-Kan This work This work JRG5393/lRZ5 ( PyhjA -- lacZ ) from pGS1634 JRG5609 JRG5641 MC1000/lRZ5 ( PyhjA0 -- lacZ ) from pGS2018 MC1000/lRZ5 ( PyhjA9 -- lacZ ) from pGS1829 This work This work JRG5694 MC1000 D ( lacIPOZYA ) X74 galU galK D ( araABC-leu ) XL-1 Blue pMV107 F , l 2 2 Lab collection MV247 W3110 Haagmans & van der Woude ( 2000 ) Lab collection Plasmids pACYC184 Source of cat cassette for linear transformation PyfgF -- lacZ fusion in pRS415 ( 192 bp promoter fragment , 2 +84 relative to the transcript start ) , Ap R Martinez et al. ( 1988 ) This work pGS1633 108 to pGS1634 This work PyhjA -- lacZ fusion in pRS415 ( 187 bp promoter fragment , 2127 to +60 relative to the transcript start ) , Ap R PyecI -- lacZ fusion in pRS415 ( 399 bp promoter fragment ) , Ap R Psmf -- lacZ fusion in pRS415 ( 383 bp promoter fragment ) , Ap R PyjhB -- lacZ fusion in pRS415 ( 450 bp promoter fragment ) , Ap R PyhjA -- lacZ fusion in pRS415 with the FNR-binding site altered from TTGAT-N4-ATCAA to ATCAT-N4-ATGAT , Ap R PyhjA : : lacZ fusion in pRS415 with the OxyR-binding site altered from ATAGgcacaggCTATcttattgATAGtTtatAttcAT to TATCgcacaggGATActtattgTATCtAtatTttcTA , Ap R pGS1740 pGS1740a This work This work pGS1828 pGS1829 This work This work pGS2018 This work Haagmans & van der Woude ( 2000 ) Simons et al. ( 1987 ) pMV107 pRS415 pZE24 with a 929 bp fragment encoding OxyR lacYZA-based promoter-fusion vector , Ap R pTP223 lam , bet and exo genes of phage l , under the control of the lac promoter , Tet R Poteete & Fenton ( 1984 ) Oligonucleotides JDP1 This work This work TTTTGAATCAACTGTAGGTGGGTATC TTTTGGATCCGTGTTATTCTAATATAGCCC JDP2 JDP3 This work This work TTTTGAATTCTGTAATAATGATGAATTAACATG TTTTGGATCCGCAGAATCCCCCACATCC JDP4 JDP5 This work This work TTTGAATTCCCGCAATCATGATATGCC TTTGGATCCCGGTGGTGAGTTATTGTG JDP6 JDP7 This work This work CGTAATCCATCAACTGTTTCG GCCTAAAACCTTATTATTTTCAC JDP8 FyecI TTTTGAATCTTAATTATCACTGGGCCCATTAATTAACAG TTTTGGATCCAAGCATTCCAGCGGATCCCATACCTAATAT RyecI Fsmf TTTTGAATCTCTAAAAGCTCTGAATTCARTTAACACTAGC TTTTGGATCCTTTCTGTATCGAGGATCCTTATCTCCCTGC Rsmf Cultures of MC1000 ( lac ) and JRG1728 ( lac fnr ) transformed with plasmids pGS1828 ( PyjhB -- lacZ ) , pGS1633 ( PyfgF -- lacZ ) , pGS1634 ( PyhjA -- lacZ ) , pGS1740 ( PyecI -- lacZ ) , pGS1740a ( Psmf -- lacZ ) , were grown under aerobic and anaerobic conditions at 37 uC in L broth to OD600 0.4 -- 0.6 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17464064" target="_blank">17464064</a>]<br></li>
<li>Sequences bound by OxyR at the following promoters ( 13 ) katG 5 ' - ATGTAAGATCrCAACTATCGCATCCGTGGATTAATTCAATTATAA-3 ' ahpC 5 ' - GGGTTGTTAGTTAACGCITATTGATTTGATAATGGAAACGCATTAG-3 ' oxyR 5 ' - GCCACGATAGTTCATGGCGATAGGTAGAATAGCAATGAACGATTA-3 ' 20 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2183352" target="_blank">2183352</a>]<br></li>
<li>OxyR-Binding Sites Selected from Pools of Oligonucleotides Carrying 30 or 45 Random Bases ( A ) Sequences of 38 selected probes with 30 random bases shown to be bound specifically by OxyR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8087856" target="_blank">8087856</a>]<br></li>
<li>( C ) Sequences of 16 selected probes with 45 random bases shown to be bound specifically by OxyR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8087856" target="_blank">8087856</a>]<br></li>
<li>When 10 bp were inserted between contact points 2 and 3 in the ahpC-bindingsite ( A1A2XA ~ ) , reduced instead of oxidized OxyR bound preferentially in vitro , and ahpC was no longer induced after exposure to hydrogen peroxide in vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8087856" target="_blank">8087856</a>]<br></li>
<li>t e '' ~ N Figure 3 Hydroxyl Radical Footprints and Missing Nucleoside Contacts for OxyR Binding to the oxyRS , katG , and ahpCPromoters ( A ) Fragments ( 100 bp ) of pGSO40-pGSO42 , labeled at the EcoRI site , were incubated with OxyR as indicated and then cleaved with DNAase I or with iron/hydrogen peroxide/ascorbate as described in the Experimental Procedures .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8087856" target="_blank">8087856</a>]<br></li>
<li>( B ) Fragments ( 100 bp ) of pGSO40-pGSO42 , labeled at the Hindlll site , were treated with iron/hydrogen peroxide/ascorbate and then incubated with the indicated dilutions of a 0.5 mg/ml solution of punfied OxyR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8087856" target="_blank">8087856</a>]<br></li>
<li>457 1 2 29.3 27.0 26.0 23.0 21.8 19.1 18.3 18.0 17.6 17.4 17.3 17.3 16.4 16.0 15.8 15.8 15.5 15.5 15.3 15.1 13.1 11.4 10.3 7.4 637851 890054 710103 2467337 638089 4603357 4603273 3666966 4131337 1762663 1475504 2069355 2167805 3266461 3272480 1211333 2698016 1596260 4156029 1118367 848228 2716640 497211 3643862 Upstream of dsbG Upstream of grxA Upstream of fur In yfdI Upstream of ahpC Upstream of fhuF-2 Upstream of fhuF-1 Upstream of yhjA Upstream of katG Upstream of sufA Upstream of ynbA Upstream of ﬂu In gatR In yhaC In yhaU Upstream of elbA Downstream of yfhL Upstream of ydeK Upstream of oxyR Upstream of yceA Upstream of dps Upstream of trxC Upstream of hemH Upstream of gorA Improved 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 RESULTS Initial search for new OxyR binding sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11443092" target="_blank">11443092</a>]<br></li>
<li>To our knowledge , this is the fourth time that methylation protection has been demonstrated for a bacterial regulatory system in vitro , with the interactions of Lrp with the papA-papB intergenic region , OxyR with ag43 and GutR with its operator sequence being the ﬁrst three ( van der Woude et al. , 1998 ; Correnti et al. , 2002 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17238917" target="_blank">17238917</a>]<br></li>
<li>Comparison of the amino and sequence of OxyR protein with the LysR family consensus sequence ( Henikoff et al. 1988 ) slightly modified by in ~ uding ~ e OxyR sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2511419" target="_blank">2511419</a>]<br></li>
<li>Both of the OxyR footprints extend into the -35 sigma 70 consensus sequence , which typically also makes contacts with RNA polymerase ( Siebenlist et al. , 1980 ; von Hippel et al. , 1984 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2693740" target="_blank">2693740</a>]<br></li>
<li>Also , the quality of the proposed dyads improves when allowances are made for the degenerate consensus which OxyR is likely to recognize .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1730735" target="_blank">1730735</a>]<br></li>
<li>To assess the potential localization of the Met residues in HypT , we performed a secondary structure and disorder prediction and aligned the HypT sequence with that of the LysR-type transcription factor OxyR from E. coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23690622" target="_blank">23690622</a>]<br></li>
<li>Therefore , despite the dissimilarity in sequence among the four functional OxyR recognition sequences , enough sequence-specific information must be contained in them toaccount for the fact that all four sequences are recognized with such similar and high specificity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1730735" target="_blank">1730735</a>]<br></li>
<li>Residues that are identical in four of six sequences are in boldface , the bars mark the helix-turn-helix motif , and the six cysteine residues in OxyR are underlined .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7868602" target="_blank">7868602</a>]<br></li>
<li>Second , the critical contacts defined by the interference assays map to the four ATAGnt boxes that define the OxyR motif , supporting the conclusion that OxyR contacts the DNA at these sequences .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8087856" target="_blank">8087856</a>]<br></li>
<li>First , OxyR DNAase I footprints are unusually long and span 45 bp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8087856" target="_blank">8087856</a>]<br></li>
<li>Three ORE cis-acting DNA sites have been mapped and found to bind their cognate regulators ; ORE-I ( from -236 to -197 nt from the transcriptional start site ) for OxyR , ORE-II ( from -156 to -127 ) for IHFs , and ORE-III ( from -56 to -35 ) for an unidentiﬁed activator responding to both superoxide generators and hydrogen peroxide ( Lee et al. , 2004 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16824106" target="_blank">16824106</a>]<br></li>
<li>A hallmark of the members of the OxyR regulon is that either the reduced or the oxidized form of OxyR binds to the regulatory region or that , if both forms bind , the DNA-protein contact sites differ ( 34 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12644490" target="_blank">12644490</a>]<br></li>
<li>Both forms of the OxyR protein can bind to specific sites of the target genes , but only the oxidized form can act as an activator ( Storz et ai .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8483417" target="_blank">8483417</a>]<br></li>
<li>The sites bound by OxyR are quite large ( > 45 nucleotides ) suggesting that OxyR might bind as a multimer , but no obvious inverted or tandem repeats are seen within the protected regions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2183352" target="_blank">2183352</a>]<br></li>
<li>Synthetic sites bound by OxyR were sequestered away from the pool of random oligonucleo-tides by a shift in gel mobility .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8087856" target="_blank">8087856</a>]<br></li>
<li>To test whether OxyR bound to these sites , we carried out DNase I footprinting experiments .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11443092" target="_blank">11443092</a>]<br></li>
<li>The seven OxyR target sequences analyzed previously ( 25 ) and two new sites ( grxA at position 207 in GenBank entry M13449 and a second Mu phage mom site at position 59 in GenBank entry V01463 ) were used to generate an individual information weight matrix ( 26 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10419964" target="_blank">10419964</a>]<br></li>
<li>In this study , we dissect the roles of the three 5 cents - GATC-3 cents sites in Ag43 phase switching by performing sitedirected mutagenesis of the target sequences in all possible combinations and in a manner that allows discrimination between their abilities to act as substrates for Dam-mediated methylation and in OxyR interactions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11972787" target="_blank">11972787</a>]<br></li>
<li>Thus , there is evidence that the 5 cents - GATC-3 cents sites in the agn43 regulatory region are protected from methylation in phase OFF but not in phase ON cells and that OxyR can bind to the agn43 regulatory region in a manner that is abrogated by methylation of the target DNA ( Haagmans and Van der Woude , 2000 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11972787" target="_blank">11972787</a>]<br></li>
<li>This showed that OxyR can bind to agn43 DNA that is methylated at any of the three 5 cents - GATC-3 cents sites singly .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11972787" target="_blank">11972787</a>]<br></li>
<li>The observation that OxyR binds unusually long and dispersed sequences raises the question of what is the physiological advantage of an extended DNA-bindingsite .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8087856" target="_blank">8087856</a>]<br></li>
<li>However , OxyR will not bind mom DNA or repress transcription when the GATC sequences are fully methylated ( BoÈlker and Kahmann , 1989 ; Sun and Hattman , 1996 ; 1998 ; Hattman and Sun , 1997 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10692164" target="_blank">10692164</a>]<br></li>
<li>A quantitative model based on information theory was constructed for describing the DNA recognition sequences for the OxyR transcription factor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11443092" target="_blank">11443092</a>]<br></li>
<li>At the right border of the region conferring methylation dependence the regions protected by MomR / OxyR and by C overlap by only 4 bp ( see Figure 6 ) , leaving at least two possibilities to achieve repression : OxyR binding could block access of C protein to its binding site , alternatively , both proteins could bind simultaneously .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2551682" target="_blank">2551682</a>]<br></li>
<li>Analysis of the OxyR footprint of the mom and oxyR loci indicates that the essential protein-DNA contacts are made with the ﬁrst 5 bp of the protected region , covering the ﬁrst GATC site in mom ( 13 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10094691" target="_blank">10094691</a>]<br></li>
<li>Once OxyR is bound , Dam can not access these target sequences , leading to preferred inheritance of the OFF phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24837285" target="_blank">24837285</a>]<br></li>
<li>Binding of three non-homologous sequences by purified OxyR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2183352" target="_blank">2183352</a>]<br></li>
<li>Perhaps the different sequences at the three promoters dictate different conforDownloade mational changes and roles for OxyR at the three promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2183352" target="_blank">2183352</a>]<br></li>
<li>These 51 nt include the 35 and 10 regions of the promoter , the three GATC sequences , and nucleotides that are predicted to be essential for binding of OxyR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12029051" target="_blank">12029051</a>]<br></li>
<li>Both the oxidized and the reduced forms of the OxyR protein are capable of binding to three diverse sequences upstream of OxyR-regulated promoters , but the interactions of the two forms of OxyR with the promoter regions are different .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2183352" target="_blank">2183352</a>]<br></li>
<li>These results showed that purified OxyR is capable of binding to the three different sequences at the katG , ahpC , and oxyR promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2183352" target="_blank">2183352</a>]<br></li>
<li>Why the OxyR protein hasevolvedto recognize such diverse sequences is not clear .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1730735" target="_blank">1730735</a>]<br></li>
<li>Possibly two different isoforms of the OxyR protein recognize the two different sequences , or expression ure8 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2693740" target="_blank">2693740</a>]<br></li>
<li>Potential dyad symmetries were found in each of the OxyR recognition sequences .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1730735" target="_blank">1730735</a>]<br></li>
<li>Matches to the FNR and OxyR consensus sequences are highlighted in bold .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17464064" target="_blank">17464064</a>]<br></li>
<li>The heavy brackets denote the oxidized footprint of OxyR , the light brackets denote the reduced extended footprint of OxyR , and the dotted lines indicate the sequences protected by RNA polymerase binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7868602" target="_blank">7868602</a>]<br></li>
<li>After DNase I digestion , samples were run in Darallel with a Maxam-Gilbert G/A seauencine ladder ( iane 6 ) on an 8 % polyacrylamide iequencing gef : The regions protected from DNase I digestion are indicated by brackets at the right of OxyR results in the binding of other factors to the two promoter sequences .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2693740" target="_blank">2693740</a>]<br></li>
<li>The sequences protected from DNase I digestion by OxyR and IHF binding are indicated by the brackets .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7984106" target="_blank">7984106</a>]<br></li>
<li>It is likely that OxyR binding to these sequences allows for RNA polymerase recruitment to the promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11443092" target="_blank">11443092</a>]<br></li>
<li>Since the higherafﬁnity fhuF-1 binding site covers the 11 , 210 , and 235 sequences of the fhuF promoter , we suggest that OxyR represses fhuF expression by blocking RNA polymerase binding to the promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11443092" target="_blank">11443092</a>]<br></li>
<li>To estimate the apparent dissociation constants for OxyR binding its specific recognition sequences , the gel mobility shift assay was used .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1730735" target="_blank">1730735</a>]<br></li>
<li>We postulated that the information for the binding of reduced OxyR could be contained in sequences at position 3 ( affecting benda-bility ) , at position 5 ( affecting specific contacts ) , or throughout the entire site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8087856" target="_blank">8087856</a>]<br></li>
<li>RESULTS Affinity of OxyR for Its Functional Binding Sites-To increase the pool of information on the OxyR recognition sequences , we localized and sequenced an additional OxyR-binding site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1730735" target="_blank">1730735</a>]<br></li>
<li>Since IHF has also been shown to cause a strong bend upon binding DNA and both regulators bind to unusually long sequences , the combined binding of OxyR and IHF may lead to an unusual promoter structure .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7984106" target="_blank">7984106</a>]<br></li>
<li>These proteins carry helix-turn-helixDNA-binding domains similar to those found in regulators that bind to shorter DNA sequences , raising the question as to why OxyR and other LysR-type regulators bind to such extended sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8087856" target="_blank">8087856</a>]<br></li>
<li>OxyR protein purified in its oxidized form was found to bind four of its non-homologous , functionaD l NA-binding sites with over 10g-fold higher affinity than random DNA sequences .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1730735" target="_blank">1730735</a>]<br></li>
<li>This ﬁnding raises questions as to the role of OxyR binding to coding sequences and whether other transcription factors bind to coding sequences .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11443092" target="_blank">11443092</a>]<br></li>
<li>Only three positions out of 45 bp are conserved among the seven OxyR-binding sites if the sequences are aligned with respect to the experimentally defined oxidized OxyR DNAase I footprints ( see below ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8087856" target="_blank">8087856</a>]<br></li>
<li>A key aspect of this phase variation system is that OxyR binding is abrogated when three Dam target sequences in its binding sites are methylated ( resulting in the ON phase ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24837285" target="_blank">24837285</a>]<br></li>
<li>Comparison of the nucleotide sequences of the OxyR protein binding sites in oxyR , mom , and katG .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1864839" target="_blank">1864839</a>]<br></li>
<li>Gray rectangles indicate OxyR tetramer binding regions , asterisks indicate mutated GATC sequences , and n indicates the number of OxyR binding sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20118257" target="_blank">20118257</a>]<br></li>
<li>The asterisks indicate conserved residues in the aligned DNA sequences ; a dash ( -- ) represents a space introduced between OxyR contact points III and IV to improve alignment with the agn43 regulatory region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11972787" target="_blank">11972787</a>]<br></li>
<li>The 10 sequence of the promoter and three Dam target sequences were protected in an in vitro DNase I footprint assay by OxyR ( ox ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12644490" target="_blank">12644490</a>]<br></li>
<li>The DNase I-protected region of unmethylated agn43 DNA in the presence of oxidized OxyR included the 10 sequence of the agn43 promoter and the three GATC sequences ( Fig. 3 , lane 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12644490" target="_blank">12644490</a>]<br></li>
<li>The region of DNA that was protected from DNase I cleavage by OxyR ( C199S ) runs from 16 to 42 relative to the transcription initiation site , which includes the 10 region of the phasevarying promoter and the three GATC sequences .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12029051" target="_blank">12029051</a>]<br></li>
<li>These GATC sequences were not protected from Dam-dependent methylation in an oxyR background , suggesting that OxyR binding results in Dam-depen-dent methylation protection in OFF cells .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10692164" target="_blank">10692164</a>]<br></li>
<li>First , methylation of these sequences abrogates OxyR binding and thus alleviates OxyR-dependent repression ( Fig. 6 ) ( 15 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12029051" target="_blank">12029051</a>]<br></li>
<li>Thus , these observations and our data are consistent with a model in which the binding of OxyR blocks methylation of the GATC sequences in agn and , conversely , methylation of these sequences blocks OxyR binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10692164" target="_blank">10692164</a>]<br></li>
<li>Thus , regulation of mom and agn have in common that methylation of the GATC sequences in the OxyR binding site abrogates OxyR binding and thereby alleviates repression of transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10692164" target="_blank">10692164</a>]<br></li>
<li>Importantly , this experimental strategy for the ﬁrst time allowed the phase variation of agn43 to be locked OFF without altering either the agn43 OxyR binding site af-ﬁnity , the methylation target sequences , or the cellular concentration of the regulatory factors .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20118257" target="_blank">20118257</a>]<br></li>
<li>A sequence with an OxyR binding site was generated by amplifying the agn43 binding site with point mutations in all three GATC sequences ( 46 ) from pMV140 using oMV419 and oMV420 and cloned as an SalI fragment into pMV244 or pMV245 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20118257" target="_blank">20118257</a>]<br></li>
<li>Three Dam target ( GATC ) sequences are contained within the binding site for OxyR at agn43 , and methylation on both strands at this sequence prevents OxyR binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20118257" target="_blank">20118257</a>]<br></li>
<li>Each site has the same binding afﬁnity for reduced OxyR as that for the unmethylated agn43 sequence , but they do contain GATC sequences ( 46 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20118257" target="_blank">20118257</a>]<br></li>
<li>The ON phase is a result of abrogation of OxyR binding and direct enhancement of transcription , both mediated by methylation of the agn43 GATC sequences .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12029051" target="_blank">12029051</a>]<br></li>
<li>The repressor is the OxyR DNA-binding protein , and its binding site was found to be located downstream of the agn43 transcription start site in a region of DNA that encompasses three 5 cents - GATC-3 cents sequences that are subject to Dam-mediated DNA methylation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11972787" target="_blank">11972787</a>]<br></li>
<li>The transition to the ON phase of Ag43 not only requires dissociation of OxyR from the DNA but also methylation of the three target GATC sequences in the agn43 OxyR binding site to prevent reassociation of OxyR ( 7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12644490" target="_blank">12644490</a>]<br></li>
<li>The DNA probe includes the agn43 OxyR binding site and the three Dam target sequences .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20118257" target="_blank">20118257</a>]<br></li>
<li>Binding of OxyR prevents Dam from accessing the target sequences in the binding site , allowing them to remain unmethylated throughout the cell cycle , even in the presence of Dam ( 7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12644490" target="_blank">12644490</a>]<br></li>
<li>The 101-nt region that is present in the agn - lacZ fusion of MV556 contains only the promoter region of the nt 9251 transcription initiation site and the OxyR binding region that contains the Dam target sequences ( see Fig. 7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12029051" target="_blank">12029051</a>]<br></li>
<li>Furthermore , in vitro transcription from only the unmethylated template , which is a binding substrate for OxyR , was repressed in the presence of OxyR ( C199S ) , consistent with the OxyR ( C199S ) binding region that overlaps the 10 region of the promoter and contains the Dam target sequences ( Fig. 3 and 7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12029051" target="_blank">12029051</a>]<br></li>
<li>In summary , the 101-nt region that contains the promoter and the OxyR binding site which contains the three GATC sequences is sufﬁcient for Dam - and OxyR-dependent phase variation Methylation of the agn43 GATC sequences is required for full agn43 transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12029051" target="_blank">12029051</a>]<br></li>
<li>The results presented here show that Dam - and OxyR-dependent phase variation requires only a promoter and an appropriately positioned OxyR binding site that contains Dam target sequences .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12029051" target="_blank">12029051</a>]<br></li>
<li>Furthermore , the promoter and the OxyR binding site containing Dam target sequences appear to be necessary and sufﬁcient to obtain phase variation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12029051" target="_blank">12029051</a>]<br></li>
<li>Three GATC sequences in the agn regulatory region are protected from methylation in an OxyR-dependent manner The data presented above show that Ag43 transcription is Dam dependent , and that OxyR is a repressor ( Table 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10692164" target="_blank">10692164</a>]<br></li>
<li>OxyR binds to the Ag43 regulatory region in vitro , and Dam-dependent methylation reduces the affinity The data described above show that OxyR is required for transcriptional repression and for protection from Dam-dependent methylation of the GATC sequences in agn .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10692164" target="_blank">10692164</a>]<br></li>
<li>This is consistent with a competition model in which Dam and OxyR share a preference for speciﬁc DNA sequences in the regulatory region of the agn43 gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11972787" target="_blank">11972787</a>]<br></li>
<li>When three GATC sequences in the agn43 regulatory region are unmethylated , OxyR can bind and repress transcription , resulting in the OFF phase , but , when these GATC sequences are methylated , OxyR does not bind and expression is ON ( 12 , 17 , 18 , 36 , 37 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12644490" target="_blank">12644490</a>]<br></li>
<li>In addition , OxyR can bind to the agn43 regulatory region , but only when three GATC sequences in the regulatory region are unmethylated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12029051" target="_blank">12029051</a>]<br></li>
<li>Consistent with this , the region protected by OxyR ( ox ) from DNase I includes the GATC sequences .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12644490" target="_blank">12644490</a>]<br></li>
<li>Consistent with this and the role of Dam in OxyR binding , OxyR ( C199S ) protected from DNase I digestion the agn43 regulatory region from 16 to 42 , which includes the three GATC sequences .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12029051" target="_blank">12029051</a>]<br></li>
<li>OxyR does not bind directly to the regulatory region of agn when the agn GATC sequences are methylated ( Fig. 3 ) , indicating that there is only the one binding site in this region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10692164" target="_blank">10692164</a>]<br></li>
<li>Thus , OxyR binding sequesters the agn43 GATC sequences from Dam when it is bound to the hemimethylated region as well as the unmethylated region ( 15 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20118257" target="_blank">20118257</a>]<br></li>
<li>Previously , it was shown that OxyR is required for repression of the Ag43-encoding gene , agn43 , and that Dam-dependent methylation of three GATC target sequences in the regulatory region abrogates OxyR binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12029051" target="_blank">12029051</a>]<br></li>
<li>In vitro , both oxidized OxyR and OxyR ( C199S ) , which is locked in the reduced conformation , bound to the agn regulatory region , but methylation of the three GATC sequences abrogated this binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10692164" target="_blank">10692164</a>]<br></li>
<li>Therefore , we examined whether OxyR binding to the Ag43 regulatory region is affected by the methylation state of the GATC sequences .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10692164" target="_blank">10692164</a>]<br></li>
<li>There are three GATC sequences in this putative OxyR binding region of agn .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10692164" target="_blank">10692164</a>]<br></li>
<li>The ON phase is obtained by methylation of the three GATC sequences in the regulatory region , which abrogates the binding of OxyR at that GATC-containing region ( 19 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12029051" target="_blank">12029051</a>]<br></li>
<li>A simple model for the regulation of Ag43 phase variation is one in which the OxyR protein binds to unmethylated 5 cents - GATC-3 cents sequences in the region of the agn43 promoter and represses transcription by excluding or impeding the movement of RNA polymerase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11972787" target="_blank">11972787</a>]<br></li>
<li>This suggests that OxyR binds at the GATC-containing region of agn .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10692164" target="_blank">10692164</a>]<br></li>
<li>Unexpectedly , our studies showed that OxyR binds to sequences within the coding region of yfdI .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11443092" target="_blank">11443092</a>]<br></li>
<li>Figure 4 ( lanes 4 and 5 ) shows that extracts from the strain that overproduces the OxyR protein specifically protect sequences upstream from ahpC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2693740" target="_blank">2693740</a>]<br></li>
<li>Extracts of strains that overproduce the OxyR protein protect regions upstream of the oxyR-regulated oxyR , katG , and ahpC promoters from deoxyribonuclease ( DNase ) I digestion although the protected sequences show very little sequence similarity ( 13 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2183352" target="_blank">2183352</a>]<br></li>
<li>To determine whether OxyR purified to homogeneity on the basis of binding to the katG sequence could still bind the ahpC and oxyR sequences , we incubated samples of the fraction shown in Fig. 2B with katG , ahpC , and oxyR promoter fragments and assayed for the ability to protect against DNase I digestion .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2183352" target="_blank">2183352</a>]<br></li>
<li>The region of the predicted helix-turn-helix motif is boxed Homology between OxyR and other proteins Homologies were searched for between the amino acid sequence of the OxyR protein deduced from the nucleotide sequence and sequences of other proteins in current data-bases .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2511419" target="_blank">2511419</a>]<br></li>
<li>These results suggest that the OxyR protein specifically recognizes seemingly dissimilar sequences through the use oafmultidegenerate recognition code The chemical basis for a plausible degenerate recognition systemis discussed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1730735" target="_blank">1730735</a>]<br></li>
<li>Binding of OxyR to its recognition sequence is proposed to occur as a dimer of dimers ( 9 , 42 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12029051" target="_blank">12029051</a>]<br></li>
<li>For comparison , Fig. 2B shows sequence alignments within DBDs of some LTTRs ( CbnR , CysB , OxyR and GcvA ) and residues involved in DNA binding ( blue ) or pc function ( red ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15255893" target="_blank">15255893</a>]<br></li>
<li>The OxyR sequence logo is below the sequence in the 3 -5 orientation and corresponds to the lower DNA strand .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11741843" target="_blank">11741843</a>]<br></li>
<li>For example , a model based solely on sites from which OxyR activates transcription might be stronger in predicting othe sites where OxyR binds as an activator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11443092" target="_blank">11443092</a>]<br></li>
<li>:1 o Sequence Determinants for Reduced OxyR Binding We also examined OxyR-C199S binding to katGand ahpC fragments and observed a striking lack of binding ( Figure 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8087856" target="_blank">8087856</a>]<br></li>
<li>Together , the alignments of the natural and synthetic OxyR-binding sites and the results of the hydroxyl radical footprinting and interference assays provide strong evidence that oxidized OxyR binds in four adjacent major grooves .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8087856" target="_blank">8087856</a>]<br></li>
<li>Molecular information theory analysis of aligned OxyR ( 44 ) and SoxS ( 55 ) binding sites ( Fig. 6 ) predict that the major grooves facing each of these two proteins are off-center by ca. 5 nt , suggesting that SoxS and OxyR bind to opposite faces of the grxA-ybjC intergenic region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11741843" target="_blank">11741843</a>]<br></li>
<li>Our finding that only a single CysB tetramer binds to a 45-bp site attribute of LysR-type proteins may be indicative of a common in recognizing large binding sites , which have been reported to be between 36 and 55 bp for OxyR ( 56 ) , AmpR ( 33 ) , NodD ( 11 ) , NahR ( 25 , 53 ) , and OccR ( 61 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8206845" target="_blank">8206845</a>]<br></li>
<li>In contrast to the other repressed genes , however , OxyR binds to two separate sites in the fhuF promoter region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11443092" target="_blank">11443092</a>]<br></li>
<li>Interestingly , the two OxyR sites in the fhuF promoter overlapped with two regions bound by the Fur repressor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11443092" target="_blank">11443092</a>]<br></li>
<li>One interesting prediction of the Fur binding model was the presence of two clusters of Fur binding sites that overlapped the OxyR binding sites in the fhuF promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11443092" target="_blank">11443092</a>]<br></li>
<li>To determine which of these alignments is most significant and to understand how the OxyR protein recognizes its binding sites will require further protection , interference and mutant binding studies , obpc AAT GAATCAGCAAA CAACAAT k & G TTCTCTCTAA 0xyR `` Conserved '' ----- G -- AG-T ------- AT-G ------ T-e-v ------- A Figure 10 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2693740" target="_blank">2693740</a>]<br></li>
<li>In addition to predicting OxyR binding sites upstream of the fur ( 23 ) , dsbG , and fhuF genes , our computer model predicted OxyR sites centered 9 bp upstream of the nmpC and 81 bp upstream of the ybaL start codons as well as sites within ORFs .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11443092" target="_blank">11443092</a>]<br></li>
<li>Furthermore , OxyR titration as a result of genome rep-lication is unlikely to be signiﬁcant for phase variation even though the OxyR/HM-DNA complex was not stable in the presence of competitor OxyR binding sites ( Fig. 2b ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20118257" target="_blank">20118257</a>]<br></li>
<li>To furtherdefine the mechanisms by which a protein can specifically recognize dissimilar functional binding sites , we have continued ourinvestigation of the OxyR protein 's DNA binding properties .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1730735" target="_blank">1730735</a>]<br></li>
<li>Computation-directed identiﬁcation of OxyR DNA binding sites in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20118257" target="_blank">20118257</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12029051" target="_blank">12029051</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21378183" target="_blank">21378183</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11443091" target="_blank">11443091</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12644490" target="_blank">12644490</a>]<br></li>
<li>Interestingly , using the second model , four of the known OxyR binding sites , at the dps , trxC , hemH , and gor promoters , were no longer among the 20 sites with the highest information content .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11443092" target="_blank">11443092</a>]<br></li>
<li>The information content of the 20 sites ranges from 11.8 to 26.0 bits , and ﬁve of the six E. coli OxyR binding sites used in the model fell within the top 20 sequences .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11443092" target="_blank">11443092</a>]<br></li>
<li>With this approach , previously identiﬁed OxyR binding sites ( 31 ) were used as a model to search the E. coli genome sequence ( 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10419964" target="_blank">10419964</a>]<br></li>
<li>Our initial model was used to search the entire E. coli genome sequence for additional OxyR binding sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11443092" target="_blank">11443092</a>]<br></li>
<li>We used an algorithm based on information theory ( 26 ) that uses previously identiﬁed OxyR binding sites as a model to search through the entire E. coli genome for new OxyR binding sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10419964" target="_blank">10419964</a>]<br></li>
<li>We used an algorithm based on information theory ( 11 ) that uses previously identi-ﬁed OxyR binding site sequences as a model to search through the entire Escherichia coli genome for putative OxyR binding sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11443092" target="_blank">11443092</a>]<br></li>
<li>By analyzing54 synthetic binding sites selected from pools of random oligonucleotides , we are now able to define a consensus motif ( ATAGntnnnanCTATnnnnnnnATAGntnnnanCTAT ) for the binding of oxidized OxyR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8087856" target="_blank">8087856</a>]<br></li>
<li>We have not seen any obvious homologies to consensus sequence binding sites for LexA , SoxS , OxyR , or KatF in the promoter regions of the fpg or mutY operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10400579" target="_blank">10400579</a>]<br></li>
<li>Since binding to these different sites is specific and of high affinity , what sequence information allows OxyR to discriminate between these sites and nonspecific DNA ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8087856" target="_blank">8087856</a>]<br></li>
<li>We have found that OxyR recognizes its different functional binding sites with high and nearly equal affinity and specificitydespite the lack of sequence similarity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1730735" target="_blank">1730735</a>]<br></li>
<li>The OxyR logo was generated from an alignment of the grxA sequence and its complement and seven other OxyR binding sites and their complement taken from Schneider ( 44 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11741843" target="_blank">11741843</a>]<br></li>
<li>B , upper panel , sequence logo based on previously identified OxyR binding sites ( 27 , 33 , 59 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23139412" target="_blank">23139412</a>]<br></li>
<li>SoxS and OxyR binding sites are indicated by square brackets above and below the sequence , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11741843" target="_blank">11741843</a>]<br></li>
<li>Our data indicate that the overlap of the SeqA and OxyR binding sites at agn43 has no signiﬁcance and that SeqA does not play a role in the sequestration of the agn43 region from Dam .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20118257" target="_blank">20118257</a>]<br></li>
<li>Dam methylation of these sites would prevent binding of OxyR , thereby allowing transcription of agn43 to proceed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11972787" target="_blank">11972787</a>]<br></li>
<li>Methylation of these sites would block binding of OxyR , thereby allowing RNA polymerase to bind and transcribe agn43 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11972787" target="_blank">11972787</a>]<br></li>
<li>No OxyR binding sites were predicted upstream of ytfK , ydcH , ydeN , and yaeH , so these genes may or may not be repressed directly by OxyR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11443091" target="_blank">11443091</a>]<br></li>
<li>Depending on the distance of binding sites from transcription start points , the OxyR protein acts as a repressor or an activator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8483417" target="_blank">8483417</a>]<br></li>
<li>The two MomR/OxyR binding sites we have analysed share several features ; they both comprise 43 bp , of which 20 bp are identical .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2551682" target="_blank">2551682</a>]<br></li>
<li>Predicted OxyR binding sites ( 41 ) of 7.0 bits centered at position 4549044 , 8.8 bits centered at position 2784053 , and 5.4 bits centered at position 2784276 are underlined .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11443091" target="_blank">11443091</a>]<br></li>
<li>Our computational search for OxyR binding sites allowed us to identify several new OxyR target genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11443092" target="_blank">11443092</a>]<br></li>
<li>Introduction of auxiliary OxyR binding sites up to 466 bp from agn43 affects phase variation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20118257" target="_blank">20118257</a>]<br></li>
<li>OxyR binding sites were introduced as auxiliary binding sites for agn43 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20118257" target="_blank">20118257</a>]<br></li>
<li>Like the OxyR binding sites , no obvious homologies were found between these two binding sites in either orientation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2693740" target="_blank">2693740</a>]<br></li>
<li>Derivatives of pMV244 and pMV245 with additional OxyR and SeqA binding sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20118257" target="_blank">20118257</a>]<br></li>
<li>Locations and alignment of OxyR binding sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10094691" target="_blank">10094691</a>]<br></li>
<li>MuC and OxyR binding sites are overlined by thick lines .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10094691" target="_blank">10094691</a>]<br></li>
<li>This work was initiated by a search for additional OxyR binding sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10419964" target="_blank">10419964</a>]<br></li>
<li>Improved search for new OxyR binding sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11443092" target="_blank">11443092</a>]<br></li>
<li>In an attempt to disrupt methylation of the second and third 5 cents - GATC-3 cents sites without affecting OxyR binding directly , nucleotides less likely than adenine to make contact with OxyR were altered .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11972787" target="_blank">11972787</a>]<br></li>
<li>( A ) Schematic representation of the agnK12 regulatory region from positions 35 to 223 showing the OxyR binding region , methylation sites , ORF , and terminator , as indicated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24837285" target="_blank">24837285</a>]<br></li>
<li>Evidence is presented which suggests that OxyR may act as a repressor of Ag43 transcription by binding to unmethylated GATC sites in the regulatory region of the gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10094691" target="_blank">10094691</a>]<br></li>
<li>Based upon analysis of mutants and upon sequence comparisons , it is proposed that OxyR may act as a repressor of Ag43 transcription by binding to unmethylated GATC sites in the regulatory region of the gene and that methylation of these sites prevents OxyR binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10094691" target="_blank">10094691</a>]<br></li>
<li>OxyR binding sites contain a directly repeated sequence , and the region protected from DNaseI digestion increases in thepresence of inducer ( 49 ) in a manner similar to thatobserved for araIBAD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1447222" target="_blank">1447222</a>]<br></li>
<li>To facilitate the dissection of this process further , a novel approach was introduced that does not alter the sequence of the regulatory region or the cellular concentration of Dam or OxyR , which consists of inserting auxiliary OxyR binding sites upstream of the regulatory region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20118257" target="_blank">20118257</a>]<br></li>
<li>Certainly , there can be very little doubt from the combined results presented here ( in particular , the locked ON phenotypes displayed by pDEW11 -- 16 in both wildtype and dam backgrounds , the absence of OxyR binding to methylated/unmethylated DNA from these constructs and the presence of a consensus OxyR binding site within the agn43 regulatory region ) that OxyR is capable of binding to the agn43 promoter and that the adenine residues of 5 cents - GATC-3 cents sites 2 and 3 are critically involved in OxyR recognition .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11972787" target="_blank">11972787</a>]<br></li>
<li>These and other observations strongly favoured a proposed model for the Ag43 phase switch in which binding of OxyR to unmethylated 5 cents - GATC-3 cents sites within the promoter region of agn43 represses transcription , possibly by causing RNA polymerase to be excluded .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11972787" target="_blank">11972787</a>]<br></li>
<li>These results place the agn43 promoter upstream of the three 5 cents - GATC-3 cents sites contained within the proposed binding site for OxyR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11972787" target="_blank">11972787</a>]<br></li>
<li>In order to test the efﬁcacy of this model , we constructed a plasmid-borne transcriptional fusion of the agn43 regulatory region to lacZ and located the agn43 transcriptional start site and promoter upstream of , and adjacent to , three 5 cents - GATC-3 cents sites , which lie within a proposed OxyR binding site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11972787" target="_blank">11972787</a>]<br></li>
<li>In the current model for Ag43 phase variation , OxyR represses agn43 transcription by binding to a region upstream of the coding sequence , resulting in the OFF phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12029051" target="_blank">12029051</a>]<br></li>
<li>Introducing auxiliary OxyR binding sites as far upstream of the agn43 transcription start site as 466 bp was sufﬁcient to introduce a bias to the OFF phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20118257" target="_blank">20118257</a>]<br></li>
<li>Speciﬁcally , by introducing OxyR binding sites upstream as far as 466 bp of the regulatory region , the phase variation of agn43 was altered with a bias to the OFF phase ( Table 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20118257" target="_blank">20118257</a>]<br></li>
<li>However , unlike Lrp and OxyR binding to their operator sequences at pap and ag43 respectively ( Nou et al. , 1993 ; Haagmans and van der Woude , 2000 ) , methylation does not seem to affect the binding of NagC to O .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17238917" target="_blank">17238917</a>]<br></li>
<li>Possible Dyad Symmetry in the 0xyR-binding Sites-Pre-vious reports have described a lack of obvious dyad symmetry in the OxyR recognition sequences ( Bolker and Kahmann , 1989 ; Tartaglia et al. , 1989 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1730735" target="_blank">1730735</a>]<br></li>
<li>These changes may involve an increase in the number of bound protomers ( Schell , 1993 ) or ` translocation ' of the tetrameric protein from one binding site to another , observed as a change in position and length of the protein footprint on DNA , e.g. OxyR ( Toledano et al. , 1994 ) , CysB ( Hryniewicz and Kredich , 1994 ; 1995 ) and OccR ( Akakura and Winans , 2002a , b ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15255893" target="_blank">15255893</a>]<br></li>
<li>OxyR binds adjacent to the 235 sequence of the positively regulated promoters , and Tao et al. have shown that OxyR acts cooperatively to increase RNA polymerase binding to the katG promoter ( 36 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7868602" target="_blank">7868602</a>]<br></li>
<li>We aiso note that , although the high levol of sequence degeneracy in the proposed soxbox sequence is unusuai for fhe binding sife of a prokaryotic transcriptional activator , a similar degeneracy has been reported for the DNA-binding sites of OxyR , the transcriptional activator of the hydrogen peroxide stimuton ( Tarfaglia et al. .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7891555" target="_blank">7891555</a>]<br></li>
<li>Interestingly , a high binding afﬁnity of OxyR for agn43 DNA had been predicted by Zheng et al. , based on analysis of a binding sequence with contacts at four adjacent major grooves ( 39 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12644490" target="_blank">12644490</a>]<br></li>
<li>Row 1 ( K12 ) , shared sequence between MV301 and MV500 ; row 2 ( mot ) , binding motif for the reduced form of OxyR , from Kullik et al. ( 24 ) ; row 3 ( oxybs ) , DNase I footprint with OxyR ( C199S ) ( Fig. 2 ) indicated by a hatched bar ( , DNase I hypersensitive site ) ; row 4 ( cons ) , nucleotides conserved between rows 1 , 5 , 6 , and 7 ( underlined nucleotides are conserved only between rows 1 and 5 ) ; rows 5 to 8 , sequences from E. coli RS218 ( RS ) , S. ﬂexneri ( AF200692 ) ( 1 ) ( sh ) , E. coli O157 : H7 ( O157 ) ( AE005174 ) ( 31 ) , and phage Mu ( Mu ) ( V01463 ) ( 20 ) , respectively ( see text ) FIG. 8 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12029051" target="_blank">12029051</a>]<br></li>
<li>In contrast , the GATC sequences in the Ag43 regulatory region sequence seem to be located within one binding region of an OxyR multimer ( G. Storz , unpublished results ; Toledano et al. , 1994 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10692164" target="_blank">10692164</a>]<br></li>
<li>The sequence of the regulatory region of agn has homology to the consensus OxyR binding sequence and to the OxyR binding site of the phage Mu mom gene ( Kahmann , 1983 ; Toledano et al. , 1994 ; Hasman et al. , 1999 ; Henderson and Owen , 1999 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10692164" target="_blank">10692164</a>]<br></li>
<li>This regulatory region has homology to the OxyR binding site consensus sequence , and OxyR protects the agn regulatory region from DNase I digestion from basepairs 9235 to 9293 , which contains the three GATC sequences ( G. Storz , personal communication ; Toledano et al. , 1994 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10692164" target="_blank">10692164</a>]<br></li>
<li>There is substantial sequence homology between the regulatory region of agn and that of phage Mu mom at the OxyR binding site at mom , including the presence of three similarly spaced GATC sequences ( Henderson and Owen , 1999 ; Henderson et al. , 1999 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10692164" target="_blank">10692164</a>]<br></li>
<li>280 ± 18 250 ± 27 180 ± 9 175 ± 18 Activation of yhjA is co-dependent on the OxyR system Further inspection of the yhjA promoter region identiﬁed a potential binding site for the oxidized ( disulphide ) form of the transcriptional regulator OxyR that matched the consensus ( ATAGnTnnnAnCTATnnnnnnnATAGnTnnn-AnCTAT ) in 16 ( shown in bold ) of 20 bases ( Toledano et al. , 1994 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17464064" target="_blank">17464064</a>]<br></li>
<li>In addition , the nucleotide sequence of the protected region in the grx promoter showed a high degree of homology ( 17 of the 20 bases matched ) to the consensus motif for oxidized OxyR ( Fig. 5 ) proposed by Toledano et al. based on results obtained by selection of binding sites from pools of random-sequence oligonucleotides ( 21 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9294462" target="_blank">9294462</a>]<br></li>
<li>The OxyR binding region determined by footprinting ( Zheng et al. , 2001 ) , an IHF consensus sequence and the predicted Fur-box are also indicated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15009899" target="_blank">15009899</a>]<br></li>
<li>However , based on the binding of OxyR to oligonucleotides of random sequence , a consensus OxyR-binding motif has been proposed previously comprising the sequence ATAGntnnnanCTATnnnnnnnATAGntnnnanCTAT ( Toledano et al. , 1994 ; see also Zheng et al. , 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11972787" target="_blank">11972787</a>]<br></li>
<li>Recently , a detailed consensus motif was deﬁned for the binding of OxyR ( Toledano et al. , 1994 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9312050" target="_blank">9312050</a>]<br></li>
<li>In the deletion series from the BamHI site , the OxyR + phenotype was lost when the deletion extended to 377 bp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2511419" target="_blank">2511419</a>]<br></li>
<li>In a computational scan of the E. coli genome , we predicted a putative OxyR site centered at 1762663 , 253 bp upstream of the sufA start codon ( 41 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11443091" target="_blank">11443091</a>]<br></li>
<li>It is shown that the yhjA gene of E. coli possesses a class II FNR site and an OxyR site upstream of the yhjA transcript start .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17464064" target="_blank">17464064</a>]<br></li>
<li>The purified OxyR protein protected from DNase I digestion the regulatory regions of oxyR from +15 to - 3 1 , and katG from - 3 3 to - 8 0 relative to the start site of transcription on the coding strand .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1864839" target="_blank">1864839</a>]<br></li>
<li>The oxidized and the reduced forms of OxyR are as indicated , and the transcription start site is indicated by an arrow .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12644490" target="_blank">12644490</a>]<br></li>
<li>The transcript start ( arrow ) , putative '' 10 box ( italicized ) , FNR site ( boxed ) , and OxyR site ( underlined ) are indicated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17464064" target="_blank">17464064</a>]<br></li>
<li>Oxidized OxyR bound at a single site located at -197 to -244 , as has been previously reported ( Zheng et al. , 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15101990" target="_blank">15101990</a>]<br></li>
<li>To identify the region of the sufA promoter controlled by IscR , either nucleotides − 393 to +90 ( containing the previously described OxyR binding site located from − 254 to − 198 ; Zheng et al. , 2001 ) or − 200 to +40 relative to the transcription start site ( lacking the OxyR binding site ) was fused to lacZ on the chromosome , and β-galactosidase activity was assayed from cells grown under aerobic conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16677314" target="_blank">16677314</a>]<br></li>
<li>OxyR-dependent activation of this gene in response to H2O2 has also been demonstrated , and the OxyR binding site has been mapped to a region of about 50 bp centred around 220 nucleotides upstream from the transcription start site ( Zheng et al. , 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15009899" target="_blank">15009899</a>]<br></li>
<li>There are several possible ways to align the mom-binding site with the other three sequences that would result in 6 residues being conserved across the approximately 45-bp regions defined by OxyR can except either one of two different base pairs at a given position , yet can stillacquire specificity by distinguishing them from the remaining two base pairs .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1730735" target="_blank">1730735</a>]<br></li>
<li>Using footprinting data from both strands to more accurately locate the binding site centers ( possible dyad axis ) , it became clear that one of the OxyR recognition sequences , the binding site upstream of the oxyR gene , did contain a good dyad symmetry ( 13 out of 21 residues ) with an axis at the exact center of the experimentally determined binding site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1730735" target="_blank">1730735</a>]<br></li>
<li>Importantly , its location downstream of the transcription start site is typical for genes that are repressed by OxyR ( 33 ) , whereas for genes that are activated by OxyR , the consensus motif is located upstream of the start site as in KatG ( Fig. 5B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23139412" target="_blank">23139412</a>]<br></li>
<li>However , this single OxyR binding site was far up stream from the end of the primer extension product , and at all other known OxyR-activated promoters , the transcription factor binds at a position overlapping or directly upstream of the 235 sequence of the promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11443091" target="_blank">11443091</a>]<br></li>
<li>For all previously known OxyR-activated genes , OxyR binds at a site directly upstream of the 235 region of a target gene ( 19 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11443092" target="_blank">11443092</a>]<br></li>
<li>For OxyR binding to the bottom strand , the 378-bp NotI-BamHI fragment of pGSO133 was labeled with 32P at the BamHI site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11443091" target="_blank">11443091</a>]<br></li>
<li>For OxyR binding to the bottom strand relative to the dsbG promoter , the 32P-labeled primer 710 and unlabeled primer 709 were used to PCR amplify a 430-bp fragment containing both the ahpC and dsbG promoter sequences .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11443092" target="_blank">11443092</a>]<br></li>
<li>The computational search for OxyR binding sites ( 41 ) predicted a putative OxyR binding site centered at position 922026 in the yljA promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11443091" target="_blank">11443091</a>]<br></li>
<li>Our computational search ( 41 ) predicted an OxyR binding site of 9.6 bits centered at position 922026 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11443091" target="_blank">11443091</a>]<br></li>
<li>The putative OxyR binding site upstream of ychF overlaps with the RNA polymerase binding site , which is typical for genes that are negatively regulated by LysR-type transcriptional regulators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23139412" target="_blank">23139412</a>]<br></li>
<li>In the presence of hydrogen peroxide , mntH is activated by OxyR , acting through an OxyR-binding motif centered just upstream of the likely 35 RNA polymerase-binding site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12029030" target="_blank">12029030</a>]<br></li>
<li>In contrast , activating OxyR binding sites as in katG are located 40 -- 60 nucle-otides upstream of the RNA polymerase binding site ( 32 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23139412" target="_blank">23139412</a>]<br></li>
<li>How this occurs mechanistically , particularly with the MntR-binding motif relatively distant from the OxyR binding site , is an intriguing question .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12029030" target="_blank">12029030</a>]<br></li>
<li>ORE-I lies between-236 and -197 nucleotides from the transcription start site , overlapping extensively with the OxyR binding site reported previously .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15009899" target="_blank">15009899</a>]<br></li>
<li>C ells re q u ire manganese import during H2O2 stress We deleted the OxyR binding site upstream of mntH .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19400769" target="_blank">19400769</a>]<br></li>
<li>An initial OxyR binding site model was constructed using nine OxyR target sequences .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11443092" target="_blank">11443092</a>]<br></li>
<li>Alternatively , further improvements in our OxyR binding site model are needed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11443092" target="_blank">11443092</a>]<br></li>
<li>Surprisingly , the binding was to sequences upstream rather than downstream of the OxyR binding site relative to the dsbG start codon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11443092" target="_blank">11443092</a>]<br></li>
<li>Expression of katG is activated by OxyR binding at a site centred at 252 relative to the transcript start .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17464064" target="_blank">17464064</a>]<br></li>
<li>As seen in Fig. 6 , the SoxS binding site identiﬁed in this study overlaps the OxyR binding site ( 47 ) of the divergently transcribed grxA ( glutaredoxin-1 ) gene that lies directly upstream of the ybjC sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11741843" target="_blank">11741843</a>]<br></li>
<li>This was also the case when - galactosidase activity was measured from a shorter sufA promoter ( bp 200 to 40 ) - lacZ fusion that lacks the upstream OxyR binding site ( Fig. 2B and D ) ( 11 , 12 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25266384" target="_blank">25266384</a>]<br></li>
<li>The 39 end of the potential OxyR site is located only 8 bp upstream of the 59 end of the FNR site ( Fig. 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17464064" target="_blank">17464064</a>]<br></li>
<li>DNase I footprinting assays conﬁrmed that oxidized OxyR bound to the predicted site centered 54 bp upstream of the dsbG gene and 238 bp upstream of a known OxyR binding site in the promoter region of the divergently transcribed ahpC gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11443092" target="_blank">11443092</a>]<br></li>
<li>Oxidized OxyR binding to the predicted site centered 40 bp upstream of the fhuF gene was conﬁrmed by DNase I footprinting , but these assays further revealed a second higher-afﬁnity site in the fhuF promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11443092" target="_blank">11443092</a>]<br></li>
<li>Our analysis of the DNA sequence upstream of yaaA revealed a potential OxyR binding site approximately 74 bp upstream of the yaaA start codon ( Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21378183" target="_blank">21378183</a>]<br></li>
<li>The computer calculation predicted an OxyR binding site centered at position 637851 , 54 bp upstream of the dsbG start codon and 238 bp away from the center of a known OxyR binding site proximal to the divergently transcribed ahpC gene ( Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11443092" target="_blank">11443092</a>]<br></li>
<li>The computer search indicated an OxyR binding site at coordinate 710103 , centered at 234 bp upstream of the start codon of the fur gene ( Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10419964" target="_blank">10419964</a>]<br></li>
<li>The computer search indicated an OxyR binding site centered at coordinate 4603357 , 126 bases upstream of the start codon of the fhuF gene ( Fig. 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11443092" target="_blank">11443092</a>]<br></li>
<li>Interestingly , we observed another oxidized OxyR binding site ( fhuF-1 ) of slightly higher afﬁnity centered at coordinate 4603273 , 42 bases upstream of the fhuF start codon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11443092" target="_blank">11443092</a>]<br></li>
<li>The start of the fhuF message was mapped to two adjacent A residues located 30 bases upstream of the ATG start codon , situated in the middle of the high-afﬁnity OxyR binding site ( Fig. 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11443092" target="_blank">11443092</a>]<br></li>
<li>In addition , there is a high degree of conservation ( 15 of 20 nt ) between the binding motif for reduced OxyR and the sequence at agn43 ( Fig. 7B , rows 1 to 3 ) ( 42 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12029051" target="_blank">12029051</a>]<br></li>
<li>To eliminate the site of negative autoregulation and to allow for optimal expression of the oxyR message , the oxyR promoter region ( including the OxyR binding site and oxyR Shine-Dalgamo sequence ) was replaced with an efficiently translated Shine-Dalgarno sequence ( 16 ) ( Fig. 2A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2183352" target="_blank">2183352</a>]<br></li>
<li>Interestingly , the OxyR binding site is located at -197 to -244 , much further from the -35 promoter element than is typically seen with OxyR regulated genes ( Fig. 3A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15101990" target="_blank">15101990</a>]<br></li>
<li>It is peculiar that OxyR which normally acts as an activator represses transcription from the mom promoter , although the binding site is located in a region where binding could potentially stimulate transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2551682" target="_blank">2551682</a>]<br></li>
<li>Finally , we have shown that OxyR binds to the trxC promoter region via a DNA sequence that corresponds to a consensus sequence for OxyR binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10644706" target="_blank">10644706</a>]<br></li>
<li>The regulatory region of agn43 contains a SeqA binding site in addition to and overlapping with the OxyR binding site , and SeqA binds to hemimethylated agn43 DNA in vitro ( 15 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20118257" target="_blank">20118257</a>]<br></li>
<li>The GATC-I and GATC-II mutations were introduced outside the predicted OxyR-DNA contact sites to minimize direct effects of the sequence change on OxyR binding ( Fig. 7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12029051" target="_blank">12029051</a>]<br></li>
<li>This result indicated that the presence of a specific sequence at the fifth contact point was critical for reduced OxyR binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8087856" target="_blank">8087856</a>]<br></li>
<li>The determinants for the binding of reduced OxyR are specified by the sequence at contact point 5 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8087856" target="_blank">8087856</a>]<br></li>
<li>The mom promoter is located downstream of the OxyR binding site and GATC sequences and has poor homology to the canonical promoter sequence , and its transcription requires MuC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12029051" target="_blank">12029051</a>]<br></li>
<li>The nucleotide sequence of the OxyR binding site m the katG promoter was aligned to match maximally to those of mom and oxyR Arrows indicate the repeats of the consensus nonamer sequence -LCB- 18 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1864839" target="_blank">1864839</a>]<br></li>
<li>The OxyR binding site overlaps the -- 35 region of the katG promoter which does not show good homology to the consensus sequence ( 8 , our unpublished result ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1864839" target="_blank">1864839</a>]<br></li>
<li>The reason for OxyR binding to the lower-afﬁnity fhuF-2 site is less obvious .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11443092" target="_blank">11443092</a>]<br></li>
<li>The location of the predicted OxyR site is shown by a sequence walker ( 27 ) , in which the rectangle surrounding the adenine at position 710103 indicates the center ( zero base ) of the binding site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10419964" target="_blank">10419964</a>]<br></li>
<li>The katG OxyR binding motif and the putative ychF OxyR binding motif are shown below the sequence logo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23139412" target="_blank">23139412</a>]<br></li>
<li>The chromosomal mntH ( NI ) allele , in which the OxyR binding site is removed and replaced with a flp scar sequence , was also generated by the Red recombinase method .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19400769" target="_blank">19400769</a>]<br></li>
<li>The OxyR logo was generated from both strands of DNA sequence due to the symmetrical nature of the binding site ( 45 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11741843" target="_blank">11741843</a>]<br></li>
<li>Purified OxyR or IHF was incubated with the binding site DNA for 10 min , then the IHF or OxyR was added and assayed for protection from DNase I digestion .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7984106" target="_blank">7984106</a>]<br></li>
<li>Finally , DNase I footprinting experiments showed OxyR binding to the site predicted to be within the coding sequence of yfdI .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11443092" target="_blank">11443092</a>]<br></li>
<li>The results of the two selection experiments led us to propose the consensus motif ATAGntnnnanCTATnnnnn-nnATAGntnnnanCTATfor OxyR binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8087856" target="_blank">8087856</a>]<br></li>
<li>There may be a distinct consensus sequence , albeit one with complicated rules , as is the case for the consensus sequence for OxyR binding ( 152 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7968922" target="_blank">7968922</a>]<br></li>
<li>A close look at the trxC-binding site revealed that 12 out of 20 bases matched the proposed OxyR consensus motif ( 22 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10644706" target="_blank">10644706</a>]<br></li>
<li>They proposed a consensus nonamer sequence , which repeats four times in each binding site and might be the recognition sequence for OxyR protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1864839" target="_blank">1864839</a>]<br></li>
<li>An interesting feature of this region was the nucleotide sequence homology ( 39.5 % identity ) shared between it and the promoter regions of the mom and oxyR genes , both of which are known to bind OxyR , a LysR-type transcriptional activator ( 12 , 22 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10094691" target="_blank">10094691</a>]<br></li>
<li>An in vitro DNase I footprinting experiment showed that oxidized OxyR binds to the hemH promoter centered at 497211 , overlapping and just upstream of the 235 region ( Fig. 1C ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11443091" target="_blank">11443091</a>]<br></li>
<li>The oxidized wild-type OxyR protein clearly protected a region located immediately upstream of the 235 sequence of the trxC promoter from digestion ( Fig. 4B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10644706" target="_blank">10644706</a>]<br></li>
<li>The oxidized form of OxyR clearly bound a region overlapping and directly upstream of the promoter sequence ( from - 2 6 to - 7 5 on the top strand .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7984106" target="_blank">7984106</a>]<br></li>
<li>The induction was dependent on OxyR , IHF and an unidenti-ﬁed factor that binds to the ORE-III site in close proximity to the -35 region of the sufA promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15009899" target="_blank">15009899</a>]<br></li>
<li>The DNase I footprinting carried out using a 378-bp fragment from the sufA-ydiH intragenic region showed that oxidized OxyR exclusively bound to the predicted site ( Fig. 2C ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11443091" target="_blank">11443091</a>]<br></li>
<li>These showed that , like FNR , OxyR was able to bind at the unaltered yhjA promoter region , but not at the same promoter fragment carrying mutations in the predicted OxyR site ( Fig. 1 , lanes 4 and 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17464064" target="_blank">17464064</a>]<br></li>
<li>The selected oligonucleotides were cloned and subsequently sequenced if oxidized OxyR bound with an affinity at least equal to that of a control oxyRS promoter site oligonucleotide .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8087856" target="_blank">8087856</a>]<br></li>
<li>Thus , as shown in Figure 7B , a site that was normally bound by both reduced and oxidized OxyR was converted to a site that was only bound tightly by oxidized OxyR , solely by the substitution of 2 bp at position 5 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8087856" target="_blank">8087856</a>]<br></li>
				</ul></div><br>
			<label class="collapse_TU" for="_6"><font size="5px">Transcription units regulated by OxyR</font></label>
				<input id="_6" type="checkbox">
				<div class="TU"><br><ul>				<label class="TU_collapse_similarity" for="tu_similarity_1">
Lee , J.H. , Yeo , W.S. , and Roe , J.H. ( 2004 ) Induction of the sufA operon encoding Fe-S assembly proteins by superoxide generators and hydrogen peroxide : involvement of OxyR , IHF and an unidentiﬁed oxidant-responsive factor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25430730" target="_blank">25430730</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16677314" target="_blank">16677314</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16824106" target="_blank">16824106</a>]<br></label>
<input id="tu_similarity_1", type="checkbox">
<div class="TU_similar">
<li>Induction of the sufA operon encoding Fe-S assembly proteins by superoxide generators and hdyrogen peroxide : involvement of OxyR , IHF and an unidentified oxidant-responsive factor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19400769" target="_blank">19400769</a>]<br></li>
<li>Induction of the sufA operon encoding Fe-S assembly proteins by superoxide generators and hydrogen peroxide : involvement of OxyR , IHF and an unidentified oxidant-responsive factor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21143317" target="_blank">21143317</a>]<br></li>
<li>Induction of the sufA operon encoding Fe-S assembly proteins by superoxide generators and hydrogen peroxide : involvement of OxyR , IHF and an unidentiﬁed oxidant-responsive factor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659690" target="_blank">15659690</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25266384" target="_blank">25266384</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21378183" target="_blank">21378183</a>]<br></li></div>
<li>The sufABCDSE operon is directly regulated by OxyR , IHF , Fur and an unidentiﬁed oxidant-responsive ( ORE ) activator ( Lee and Roe , 1997 ; Lee et al. , 2003 ; 2004 ; Outten et al. , 2004 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16824106" target="_blank">16824106</a>]<br></li>
<li>From these results , we propose that the oxidant induction of the sufA operon is mediated through OxyR , IHF , plus an unidentiﬁed oxidant-responsive factor , and that the suf gene products are needed to Accepted 17 November , 2003 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15009899" target="_blank">15009899</a>]<br></li>
<li>Since OxyR is strongly activated by hydrogen peroxide during log-phase growth , ss is not needed , and ss repression by oxyS prevents redundancy .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9230301" target="_blank">9230301</a>]<br></li>
				<label class="TU_collapse_similarity" for="tu_similarity_8">
The dps promoter is activated by OxyR during growth and by IHF and sigma S in stationary phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17906139" target="_blank">17906139</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21239586" target="_blank">21239586</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11292794" target="_blank">11292794</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9829938" target="_blank">9829938</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19376854" target="_blank">19376854</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8779563" target="_blank">8779563</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12218039" target="_blank">12218039</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11133959" target="_blank">11133959</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17873038" target="_blank">17873038</a>]<br></label>
<input id="tu_similarity_8", type="checkbox">
<div class="TU_similar">
<li>The dps promoter is activated by OxyR during growth and by IHF and sS in stationary phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9294462" target="_blank">9294462</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11073934" target="_blank">11073934</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10892648" target="_blank">10892648</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8631665" target="_blank">8631665</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7768855" target="_blank">7768855</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8763946" target="_blank">8763946</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9512707" target="_blank">9512707</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10542180" target="_blank">10542180</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9324269" target="_blank">9324269</a>]<br></li>
<li>The dps promoter is activated by OxyR during growth and by IHF and ss in stationary phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9335308" target="_blank">9335308</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7751275" target="_blank">7751275</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9230301" target="_blank">9230301</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8990283" target="_blank">8990283</a>]<br></li>
<li>The dps promoter is activated by OxyR during growth and by IHF and ss during stationary phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7868602" target="_blank">7868602</a>]<br></li>
<li>The dps promoter is activated by OxyR during growth and by IHF and S in stationary phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15489429" target="_blank">15489429</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15716429" target="_blank">15716429</a>]<br></li>
<li>The dps promoter is activated by OxyR during growth and by IHF and as in stationary phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8087856" target="_blank">8087856</a>]<br></li></div>
<li>pKOXS B. Demple Media and cell growth conditions Altuvia , S. , Almiron , M. , Huisman , G. , Kolter , R. , and Storz , G. ( 1994 ) The dps promoter is activated by OxyR during P growth and by IHF and os in stationary phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8899708" target="_blank">8899708</a>]<br></li>
<li>Referencec Altuvia , S. , Almirón , M. , Huisman , G. , Kolter , R. , and Ston , G. ( 1994 ) The dps promoter is activated by OxyR during growth and by IHF and os in stationary phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596457" target="_blank">8596457</a>]<br></li>
				<label class="TU_collapse_similarity" for="tu_similarity_16">
[ 5 ] S. Altuvia , M. Almiron , G. Huisman , R. Kolter , G. Storz , The dps promoter is activated by OxyR during growth and by IHF and sigma S in stationary phase , Mol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15878120" target="_blank">15878120</a>]<br></label>
<input id="tu_similarity_16", type="checkbox">
<div class="TU_similar">
<li>[ 24 ] Altuvia S. , Almiron M. , Huisman G. , Kolter R. , Storz G , The dps promoter is activated by OxyR during growth and by IHF and sigma S in stationary phase , Mol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9924977" target="_blank">9924977</a>]<br></li></div>
<li>J Mol Biol 215 :403 410 Altuvia S , Almir6n M , Huisman G , Kolter R , Storz G ( 1994 ) The dps promoter is activated by OxyR during growth and by IHF and as in stationary phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8709966" target="_blank">8709966</a>]<br></li>
				<label class="TU_collapse_similarity" for="tu_similarity_19">
Altuvia , S. , Almiron , M. , Gjalt , H. , Kolter , R. , and Storz , G. ( 1994 ) The dps promoter is activated by OxyR during growth and by IHF and sS in stationary phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18452510" target="_blank">18452510</a>]<br></label>
<input id="tu_similarity_19", type="checkbox">
<div class="TU_similar">
<li>References Altuvia , S. , Almiron , M. , Huisman , G. , Kolter , R. , and Storz , G. ( 1994 ) The dps promoter is activated by OxyR during growth and by IHF and sS in stationary phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15101990" target="_blank">15101990</a>]<br></li>
<li>References Altuvia , S. , AlmiroÂn , M. , Huisman , G. , Kolter , R. , and Storz , G. ( 1994 ) The dps promoter is activated by OxyR during growth and by IHF and sS in stationary phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10760142" target="_blank">10760142</a>]<br></li></div>
				<label class="TU_collapse_similarity" for="tu_similarity_22">
Genes Dev 6:2646 -2654 Altuvia S , Almiron M , Huisman G , Kolter R , Storz G ( 1994 ) The dps promoter is activated by OxyR during growth and by IHF and ~ rs in stationary phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7476863" target="_blank">7476863</a>]<br></label>
<input id="tu_similarity_22", type="checkbox">
<div class="TU_similar">
<li>References Altuvia , S. , Almiron , M. , Huisman , G. , Kolter , R. , and Storz , G. ( 1994 ) The dps promoter is activated by OxyR during growth and by IHF and sigma S in stationary phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15009899" target="_blank">15009899</a>]<br></li>
<li>References Altuvia S , Almiron M , Huisman G , Kolter R , Storz G ( 1994 ) The dps promoter is activated by OxyR during growth and by IHF and sigma S in stationary phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16404567" target="_blank">16404567</a>]<br></li>
<li>References Altuvia , S. , Almirón , M. , Huisman , G. , Kolter , R. , and Storz , G. ( 1994 ) The dps promoter is activated by OxyR during growth and by IHF and ss in stationary phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767577" target="_blank">9767577</a>]<br></li>
<li>References Altuvia , S. , Almiron , M. , Huisman , G. , Kolter , R. , and Storz , G. ( 1994 ) The dps promoter is activated by OxyR during growth and by IHF and ss in stationary phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9663683" target="_blank">9663683</a>]<br></li>
<li>References Altuvia S , Almiron M , Hulsman G , Kolter R , Storz G ( 1994 ) The dps promoter is activated by OxyR during growth and by IHF and rS in stationary phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9197421" target="_blank">9197421</a>]<br></li>
<li>Altuvia , S. , AlmiroÂn , M. , Huisman , G. , Kolter , R. , and Storz , G. ( 1994 ) The dps promoter is activated by OxyR during S growth and by IHF and s in stationary phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11251833" target="_blank">11251833</a>]<br></li>
<li>Altuvia , S. , Almirón , M. , Huisman , G. , Kolter , R. , and Storz , G. ( 1994 ) The dps promoter is activated by OxyR during growth and by IHF and S in stationary phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23139412" target="_blank">23139412</a>]<br></li>
<li>References Altuvia S , Almiron M , Huisman G , Kolter R & Storz G ( 1994 ) The dps promoter is activated by OxyR during growth and by IHF S and s in stationary phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19076234" target="_blank">19076234</a>]<br></li>
<li>Altuvia , S. , Almiron , M. , Huisman , G. , Kolter , R. , and Storz G. ( 1994 ) The dps promoter is activated by OxyR during growth and by IHF and sigmaS in stationary phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17224607" target="_blank">17224607</a>]<br></li>
<li>Altuvia , S. , Almiron , M. , Huisman , G. , Kolter , R. and Storz , G. ( 1994 ) The dps promoter is activated by OxyR during growth and by IHF and s S in stationary phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11230141" target="_blank">11230141</a>]<br></li></div>
<li>Upon entry into stationary phase , transcription of dps is ensured by Es , stimulated by the nucleoids associated protein IHF and OxyR independent .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10692161" target="_blank">10692161</a>]<br></li>
<li>In this context it is interesting to consider that : ( i ) transcription of LucZ , another positively regulated gene , is also down-regu-lated during starvation [ 6 ] ; and ( ii ) the dps promoter [ 20 ] is differentially regulated , being OxyR-and a '' - dependent during exponential growth , and IHF-and as-dependent during stationary phase , even though the OxyR protein is present in both phases [ 20 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7557316" target="_blank">7557316</a>]<br></li>
<li>Interestingly , dps is controlled by OxyR and is s70 dependent only during the exponential phase , while it becomes sS dependent and OxyR independent in the stationary phase ( 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11133959" target="_blank">11133959</a>]<br></li>
<li>This OxyR-dependent induction occurs oniy during iog phase , aithough the OxyR protein is present in stationary phase , in the stationary phase ceits , dps is expressed in a a ^ - and iHF-dependent manner .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7984106" target="_blank">7984106</a>]<br></li>
<li>To determine whether fhuF is regulated by OxyR , the wild-type and the DoxyR and Dfur mutant strains were grown in the absence and presence of the iron chelator 2,29-dipyridyl .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11443092" target="_blank">11443092</a>]<br></li>
<li>Although it is still possible that the OxyR protein mediates the protection upstream from the ahpC and katG promoters by activating another binding factor , we do not favor this hypothesis , because substantially purified OxyR preparations show the same protection patterns ( G. Storz , L. A. Tartaglia & B. N. Ames , unpublished results ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2693740" target="_blank">2693740</a>]<br></li>
<li>Components of the regulon induced by S-nitrosylated OxyR ( including hcp ) protect against endogenous nitrosative stress , and protection is conferred at least in part by limiting S-nitrosylation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22539721" target="_blank">22539721</a>]<br></li>
<li>A previous microarray study suggested that yaaA , an uncharacterized gene found in many bacteria , was induced by H2O2 in Escherichia coli as part of its OxyR regulon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21378183" target="_blank">21378183</a>]<br></li>
<li>Hpx − ΔoxyR mutants are not viable in aerobic medium ( Park et al. , 2005 ) , which precluded the most direct test of this notion ; however , the OxyR system is also induced in ahpCF mutants , due to modestly elevated levels of intracellular H2O2 ( Seaver & Imlay , 2001b ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21143317" target="_blank">21143317</a>]<br></li>
<li>Redox-cycling agents also activate the syntheses of proteins that are induced by H2O2 and controlled by OxyR , due to the spontaneous and superoxide dismutase-mediated conversion of O z2 2 to H2O2 ( 4 , 15 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11073934" target="_blank">11073934</a>]<br></li>
<li>Although reduced OxyR does not repress oxySexpression , it may oxyS-35 be repositioned over the sequence under reducing conditions to prevent inappropriate activation of oxyS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8087856" target="_blank">8087856</a>]<br></li>
<li>Expression of the hcaA , A2 , C , B , D operon is not affected by soxR or oxyR disruption ( data not shown ) and thus can not be considered as a target of OxyR or SorR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15808934" target="_blank">15808934</a>]<br></li>
<li>Since uxuB and gntP also showed higher induction ratios in the absence of OxyR , it is intriguing to speculate that the two divergent promoters are repressed by oxidized OxyR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11443091" target="_blank">11443091</a>]<br></li>
<li>Given this observation , it seems contradictory that the suf operon is induced by oxidized OxyR while fhuF is repressed by oxidized OxyR ( 41 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11443091" target="_blank">11443091</a>]<br></li>
<li>Therefore , we conclude that the oxidant induction of sufA is independent of regulation by Fur , and there must be another oxidant-responsive factor in addition to OxyR that induces the suf operon under oxidative stress conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15009899" target="_blank">15009899</a>]<br></li>
<li>Primer extension assays showed that isc operon induction by hydrogen peroxide ( Fig. 5B ) and paraquat ( data not shown ) was independent of both OxyR and SoxRS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11443091" target="_blank">11443091</a>]<br></li>
<li>OxyR - and SoxRS-independent induction of the isc operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11443091" target="_blank">11443091</a>]<br></li>
<li>In this report , we present evidence that the sufA promoter is induced by various superoxide generators as well as by H2O2 , and induction occurs even in the absence of SoxRS and OxyR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15009899" target="_blank">15009899</a>]<br></li>
<li>This indicates that sufA induction by superoxide generators proceeds not only independently of SoxRS and OxyR , but becomes even more sensitized in the absence of those .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15009899" target="_blank">15009899</a>]<br></li>
<li>YchF Expression Is Controlled by OxyR in Response to H2O2 -- The YchF-dependent inhibition of KatG in the presence of H2O2 would be detrimental for the cell , and therefore , we addressed how cells prevented this inhibition under physiological conditions , i.e. when ychF was not overexpressed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23139412" target="_blank">23139412</a>]<br></li>
<li>For example , the reaction of hydrogen peroxide with nucleophilic cysteine residue sulfhydryls in the transcription regulators OxyR and OhrR of bacteria and Orp1 of Saccharomyces cerevisiae generates cysteine sulfenic acid ( R-S-OH ) , an intermediate that initiates a series of events leading to expression of antioxidant genes .2,3 Many of these cysteine residue sulfhydryls have been shown to have a pKa below that of free cysteine ( 8.3 ) , suggesting that low pKa cysteine residues are redox active .2,4 Sulfur trafﬁcking pathways are among the systems that are induced during oxidative stress , and these pathways also use nucleo-philic cysteine residue sulfhydryls.5-7 In the ﬁrst step of sulfur trafﬁcking , a cysteine desulfurase abstracts inorganic sulfur from the cysteine substrate using its own reactive cysteine residue sulfhydryl , thereby forming a persulﬁde ( R-S-SH ) intermediate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20936830" target="_blank">20936830</a>]<br></li>
<li>Cell , Vol .78 , 897-909 , September9 , 1994 , Copyright © 1994by CellPress Redox-Dependent Shift of OxyR-DNA Contacts along an Extended DNA-Binding Site : A Mechanism for Differential Promoter Selection Michel B. Toledano , * Ines Kullik , * Frank Trinh , * t Paul T. Baird , * ~ Thomas D. Schneider , § and Gisela Storz * * Cell Biology and Metabolism Branch National Institute of Child Health and Human Development National Institutes of Health Bethesda , Maryland 20892 § Laboratory of Mathematical Biology National Cancer Institute Frederick , Maryland 21702 Summary The redox-sensitive OxyR protein activates the transcription of antioxidant defense genes in response to oxidative stress and represses its own expression under both oxidizing and reducing conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8087856" target="_blank">8087856</a>]<br></li>
<li>This unusual use of FNR and OxyR to activate expression suggests that yhjA plays a speciﬁc role in countering oxidative stress in E. coli , possibly in particular micro-aerobic environments .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17464064" target="_blank">17464064</a>]<br></li>
<li>Transcription of fhuF is derepressed by a low iron level in the cell via the iron regulator Fur and repressed by OxyR , an oxidative response regulator ( 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14756576" target="_blank">14756576</a>]<br></li>
<li>Therefore , the lower level of transcription in the oxyR background seems to be caused by the absence of functional OxyR and not by the increase in oxidative stress .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10692164" target="_blank">10692164</a>]<br></li>
<li>In E. coli the oxidative stress response is coordinated by the transcription factors SoxRS ( responding to superoxide ) and OxyR ( responding to hydrogen peroxide ) ( reviewed by Storz and Zheng ( 37 ) ; Green and Paget ( 38 ) ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16857675" target="_blank">16857675</a>]<br></li>
<li>This demonstrates that IscR positively contributes to induce the sufA gene as much as OxyR does in response to oxidative stress .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16824106" target="_blank">16824106</a>]<br></li>
<li>In summary , these data demonstrate that ychF is negatively regulated by OxyR in response to oxidative stress .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23139412" target="_blank">23139412</a>]<br></li>
<li>In the presence of H2O2 , OxyR is oxidized ( ox ) and induces the expression of oxidative stress response genes like catalases .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23139412" target="_blank">23139412</a>]<br></li>
<li>Thus , it is possible that OxyR represses these genes in response to oxidative stress .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11443091" target="_blank">11443091</a>]<br></li>
<li>In contrast , expression analysis has demonstrated that the suf operon is controlled by OxyR and Fur , an H2O2-sensing regulator and an iron-dependent repressor , respectively , suggesting a stress response function for the SUF machinery under conditions of oxidative stress and iron limitation ( 36 , 37 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15496591" target="_blank">15496591</a>]<br></li>
<li>To date , it is only known that its expression is triggered by oxidative stress in an OxyR - , SoxRS - , Fis - , RpoS - , cAMP - , or RecA-independent way ( 36 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15546870" target="_blank">15546870</a>]<br></li>
<li>In fact , of the 22 genes that belong to the OxyR regulon ( 70 ) , 10 genes were observed here to be under S control , including the core genes dps and yaiA , the noncore genes dsbG and hemH , and the sufABCDSE operon ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15716429" target="_blank">15716429</a>]<br></li>
<li>Many of the genes implicated in the defence against ROS are inducible and belong to regulons that enable bacteria to cope with various stresses : the SoxRS regulon , including sodA and nfo , which responds to redox-cycling drugs ; the OxyR regulon , including katG and dps , which is induced in actively growing cells in response to H,O,-mediated oxidation ; the RpoS ( aS ) regulon , including also katG METHODS and dps , which is induced when bacteria enter stationary Bacterial strains .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10439394" target="_blank">10439394</a>]<br></li>
<li>The observation that the ahpC and ryhB genes , which are regulated primarily by OxyR and Fur , respectively , are induced by PMS ( Fig. 8 ) and menadione ( W. Yeo , unpublished result ) suggests that the superoxide generators do not necessarily exert their effects via superoxides only .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15009899" target="_blank">15009899</a>]<br></li>
<li>All three genes were strongly induced by H2O2 , but showed differences in induction by GSNO and NaNO ; grxA was 2 induced moderately by both compounds via OxyR and ahpC showed slight induction , whereas there was no detectable induction of katG .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14718666" target="_blank">14718666</a>]<br></li>
<li>We also discovered that the sufA , sufB , sufC , sufD , sufS , and sufE genes are part of the OxyR regulon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11443091" target="_blank">11443091</a>]<br></li>
<li>However , the only OxyR target genes , sufA and fhuF , showing 5-fold induction by GSNO or NaNO2 , are known to be regulated by both Fur and OxyR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14718666" target="_blank">14718666</a>]<br></li>
<li>OxyR is known to repress several genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12644490" target="_blank">12644490</a>]<br></li>
<li>YjiE differs from OxyR and SoxR as it does not regulate genes that are apparently involved in the degradation of ROS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22223481" target="_blank">22223481</a>]<br></li>
<li>No OxyR binding sites were predicted upstream of ytfK , ydcH , ydeN , and yaeH , so these genes may or may not be repressed directly by OxyR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11443091" target="_blank">11443091</a>]<br></li>
<li>OxyR-dependent genes identified in this analysis may include genes that respond directly to OxyR , as well as genes or operons that may be activated by other OxyR-dependent genes ( and does not preclude codependence on other transcriptional regulators ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22539721" target="_blank">22539721</a>]<br></li>
<li>Expression of agn43 -- lacZ in plasmids pDEW15 and pDEW16 also decreased signiﬁcantly when OxyR was overexpressed , but transcription was still very much locked ON .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11972787" target="_blank">11972787</a>]<br></li>
<li>of Lac colonies of MV462 ( oxyR ) conﬁrms that wild-type OxyR , which is mainly if not exclusively in the reduced form , represses agn43 transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12644490" target="_blank">12644490</a>]<br></li>
<li>In addition , OxyR induces transcription of fur , whose product represses rpoS transcription [ 17,18 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21106118" target="_blank">21106118</a>]<br></li>
<li>In this state , OxyR binds to its own regulatory region and represses transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10692164" target="_blank">10692164</a>]<br></li>
<li>Oxidized OxyR represses agn43 transcription in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12644490" target="_blank">12644490</a>]<br></li>
<li>Our conclusion that oxidized OxyR as well as reduced OxyR can repress agn43 transcription has implications for a model proposed by Schembri et al. , which addresses the coordinated regulation between ﬁmbrial production and Ag43 expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12644490" target="_blank">12644490</a>]<br></li>
<li>This suggests that , under peroxide-mediated stress , when oxidized OxyR is known to bind to the katG and oxyRS regions and activate transcription , it can also be bound to the unmethylated agn43 regulatory region and thus repress agn43 transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12644490" target="_blank">12644490</a>]<br></li>
<li>These results suggest that OxyR , which is mainly in the oxidized form , was bound and repressed agn43 transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12644490" target="_blank">12644490</a>]<br></li>
<li>However , the results presented here indicate that both the reduced and the oxidized forms of OxyR can repress agn43 transcription when the agn43 GATC sites are unmethylated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12644490" target="_blank">12644490</a>]<br></li>
<li>Both the oxidized ( C ) and reduced forms ( B ) of OxyR repress transcription from the unmethylated agn43 template .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12644490" target="_blank">12644490</a>]<br></li>
<li>When three GATC sequences in the agn43 regulatory region are unmethylated , OxyR can bind and repress transcription , resulting in the OFF phase , but , when these GATC sequences are methylated , OxyR does not bind and expression is ON ( 12 , 17 , 18 , 36 , 37 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12644490" target="_blank">12644490</a>]<br></li>
<li>It seems possible that phase switching of Ag43 expression is not simply the result of straightforward competition between Dam and OxyR for unmethylated DNA in the agn43 regulatory region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11972787" target="_blank">11972787</a>]<br></li>
<li>The agn43 gene is repressed when its regulatory region binds reduced OxyR , a feature it shares with the regulatory regions of mom and oxyRS , the other members of the OxyR regulon that are repressed by reduced OxyR ( Toledano et al. , 1994 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11972787" target="_blank">11972787</a>]<br></li>
				<label class="TU_collapse_similarity" for="tu_similarity_88">
The afﬁnity of oxidized OxyR for unmethylated agn43 DNA is similar to that for katG and oxyRS DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12644490" target="_blank">12644490</a>]<br></label>
<input id="tu_similarity_88", type="checkbox">
<div class="TU_similar">
<li>The afﬁnity of oxidized OxyR for unmethylated agn43 DNA was similar not only to OxyR ( C199S ) for agn43 DNA but also to the oxyRS and katG regulatory regions ( Fig. 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12644490" target="_blank">12644490</a>]<br></li></div>
<li>In its transcription direction and localization of promoters with respect to its target gene nhaA , nhaR differs from the LysR-OxyR family .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8168494" target="_blank">8168494</a>]<br></li>
<li>The gene nhaR of this operon encodes an autoregulatory protein that activates nhaAR transcription and is homologous to the LysR-OxyR family of prokaryotic transcriptional regulators ( 57 , 64 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16997959" target="_blank">16997959</a>]<br></li>
<li>In ST-209 the transposon was inserted into the promoter region of the ahpCF operon so that the PBAD promoter could regulate transcription of genes normally under the control of the OxyR transcription factor [ 12 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15380559" target="_blank">15380559</a>]<br></li>
<li>In this paper , we analysed agn0 ± lacZ transcription , and our data strongly indicate that Dam and OxyR control phase variation by regulating the transcription of the Ag43-encoding gene agn .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10692164" target="_blank">10692164</a>]<br></li>
<li>Transcription of the agnK12 gene is under the control of both phase variation , exercised by the proteins Dam and OxyR ( 3 , 9 -- 12 ) , and antitermination of transcription at the agnK12 leader terminator , as shown here .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24837285" target="_blank">24837285</a>]<br></li>
<li>Expression of katG is activated by OxyR binding at a site centred at 252 relative to the transcript start .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17464064" target="_blank">17464064</a>]<br></li>
<li>Importantly , its location downstream of the transcription start site is typical for genes that are repressed by OxyR ( 33 ) , whereas for genes that are activated by OxyR , the consensus motif is located upstream of the start site as in KatG ( Fig. 5B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23139412" target="_blank">23139412</a>]<br></li>
<li>Collectively , these data establish that hcp transcription is selectively activated by S-nitrosylation of OxyR in vitro and during ARN .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22539721" target="_blank">22539721</a>]<br></li>
<li>UT , untreated ( B ) In vitro transcription at hcp promoter by S-nitrosylated but not S-oxidized OxyR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22539721" target="_blank">22539721</a>]<br></li>
<li>SNO-OxyR strongly induced transcription at the hcp promoter ( Fig. 3B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22539721" target="_blank">22539721</a>]<br></li>
<li>S-oxidized OxyR did not stimulate transcription at the hcp promoter ( Fig. 3B ) , whereas both S-oxidized and S-nitrosylated OxyR activated the katG promoter ( Fig. 3B and fig .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22539721" target="_blank">22539721</a>]<br></li>
<li>At the oxyR promoter , OxyR activates transcription in one direction and represses transcription in the other , whereas OxyR probably only acts to induce transcription at the katG and ahpC promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2183352" target="_blank">2183352</a>]<br></li>
<li>The longer RNA seems to be transcribed from a weak promoter located upstream of the katG promoter , and this transcription is repressed by OxyR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8483417" target="_blank">8483417</a>]<br></li>
<li>The binding sites of the OxyR protein on the ahpC gene and the promoter transcribed opposite the oxyR gene are also located at similar positions relative to the transcription start point ( Tartaglia ef ai , 1989 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8483417" target="_blank">8483417</a>]<br></li>
<li>Should OxyR be bound at the ahpC , katG , dps , or gorA promoters under reducing conditions , the protein might induce these genes under inappropriate conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8087856" target="_blank">8087856</a>]<br></li>
<li>Oxidized OxyR then induces the transcription of a set of genes , including katG ( hydroperoxidase I ) , dps ( a nonspeciﬁc DNA binding protein ) , ahpCF ( alkyl hydroperoxide reductase ) and gorA ( glutathione reductase ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11073934" target="_blank">11073934</a>]<br></li>
<li>OxyR protein , which is in its active form for enhancing the transcription initiation at the E. coli katG promoter , was purified as described previously fTaoef a / .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8483417" target="_blank">8483417</a>]<br></li>
<li>These results indicate that the OxyR protein purified in this study is in its active form for enhancing the transcription initiation of the genes under its control in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1864839" target="_blank">1864839</a>]<br></li>
<li>These results suggest that OxyR protein binds to the upstream regulatory regions of the oxyR and katG genes and regulates the transcription initiation of these genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1864839" target="_blank">1864839</a>]<br></li>
<li>OxyR protein activates transcription of genes in the oxyR regulon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1864839" target="_blank">1864839</a>]<br></li>
<li>OxyR is sensitive to oxidation and reduction , and only oxidized OxyR is able to activate transcription of its target genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7868602" target="_blank">7868602</a>]<br></li>
<li>In vitro transcription assays demonstrated that avicins activate OxyR and its target genes katG and oxyS in a DTT-reversible manner .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16030151" target="_blank">16030151</a>]<br></li>
<li>Purification of the OxyR protein showed that oxidized but not reduced OxyR activates transcription of oxidative stress-inducible genes in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2183352" target="_blank">2183352</a>]<br></li>
<li>By contrast , genes in the known oxidative stress regulon of OxyR ( katG , sufA and grxA ) ( 26 ) showed increased transcription in response to oxidants ( fig .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22539721" target="_blank">22539721</a>]<br></li>
<li>The autoregulated oxyR gene in E. coli is also not linked to any of the genes for which it serves as an activator ( 9 ) , but although DNase I footprints are qualitatively different with reduced and oxidized forms of highly purified OxyR protein , appreciable quantitative and differences in binding to positively negatively regulated promoters have not been observed ( 37 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1706701" target="_blank">1706701</a>]<br></li>
<li>OxyR Protein Binds to the Regulatory Regions of the oxyR and katG Genes -- Experiments with the oxyR ' - ` lacZ fusion gene support the view that OxyR protein negatively regulates its own synthesis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1864839" target="_blank">1864839</a>]<br></li>
<li>Antioxidant defense genes induced by OxyR upon exposure to hydrogen peroxide include katGand ahpCF , which encode hydroperoxidase I and an alkyl hydroperoxide reductase , respectively ( Tartaglia et al. , 1989 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8087856" target="_blank">8087856</a>]<br></li>
<li>HPI synthesis is controlled by OxyR ( 6 ) , a hydrogen peroxide-activated sigma factor which is not required for katE expression ( 25 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1385595" target="_blank">1385595</a>]<br></li>
<li>Oxidized OxyR binds directly to the trxC promoter and induces its expression in response to elevated hydrogen peroxide levels or the disruption of one or several of the cytoplasmic redox pathways .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10644706" target="_blank">10644706</a>]<br></li>
<li>OxyR , which senses elevated levels of hydrogen peroxide , binds to the fur promoter and induces the expression of a transcript encoding Fur .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10419964" target="_blank">10419964</a>]<br></li>
<li>The oxyR gene encodes the transcription factor OxyR , which regulates the expression of several antioxidant genes ( e.g. , those for catalase hydroperoxidase I and glutathione reductase ) in re sponse to hydrogen peroxide ( 2 , 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9922271" target="_blank">9922271</a>]<br></li>
<li>By measuring gene expression in isogenic wild-type and oxyR deletion strains , we conﬁrmed that the peroxide response regulator OxyR activates most of the highly hydrogen peroxide-inducible genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11443091" target="_blank">11443091</a>]<br></li>
<li>In this study we placed a lacZ operon fusion into the katE gene and transduced the fusion into various genetic backgrounds to determine whether its expression is subject to regulation by katF , OxyR , a hydrogen peroxide-inducible regulatory protein ( 3 ) , or both katF and OxyR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3045091" target="_blank">3045091</a>]<br></li>
<li>Studies with the oxyR ' - ` lacZ fusion gene indicate that synthesis of OxyR protein is not affected by treatment with hydrogen peroxide ( Table I ) , but the expression of katG is induced in an oxyR-dependent fashion ( 1 , 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1864839" target="_blank">1864839</a>]<br></li>
<li>A recent microarray expression analysis has shown that suf operon is regulated by OxyR and induced upon treatment with hydrogen peroxide ( 28 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12089140" target="_blank">12089140</a>]<br></li>
<li>Subsequent experiments conﬁrmed OxyR binding to this region and showed that fur expression is induced by OxyR after hydrogen peroxide treatment .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10419964" target="_blank">10419964</a>]<br></li>
<li>The induction of OxyR regulon genes to superoxide stress is explained by the production of hydrogen peroxide from superoxide anion by MnSOD , which is induced by paraquat .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15672380" target="_blank">15672380</a>]<br></li>
<li>A subset of these genes is induced by hydrogen peroxide and is controlled at the transcriptional level by the OxyR protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8990283" target="_blank">8990283</a>]<br></li>
<li>During exponential growth , the expression of defence genes is regulated by the redox-sensitive OxyR protein , and the genes are induced only after cells are treated with hydrogen peroxide .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7984106" target="_blank">7984106</a>]<br></li>
<li>Expression of the catalase KatG , known to be regulated by OxyR , was enhanced three times when 3 mM hydrogen peroxide was treated ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23506073" target="_blank">23506073</a>]<br></li>
<li>This suggestion was based upon observations that HCP is oxidized by hydrogen peroxide , that an hcp mutant is more sensitive to hydrogen peroxide than its parent , and that hcp transcription is regulated by OxyR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17449618" target="_blank">17449618</a>]<br></li>
<li>We previously observed that OxyR negatively regulates its own expression ( 10 ) and found that cloning the oxyR promoter and open reading frame behind the well-transcribed tac promoter did not result in large overproduction of OxyR ( Fig. 2A , pAQ28 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2183352" target="_blank">2183352</a>]<br></li>
<li>In contrast , the XO403020 ~ ( oxyR ) fusion ( which is poorly bound by reduced OxyR ) gave rise to equal levels of J3-galactosidase expression in wild-type and oxyR deletion strains , suggesting OxyR is no longer able to repress the XO403020 ~ ( oxyR ) mutant promoter in vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8087856" target="_blank">8087856</a>]<br></li>
<li>Since we previously found that OxyR negatively autoregulates its own expression in vivo ( 10 ) , we also examined the ability of OxyR to regulate oxyR transcription under oxidizing ( 1 mM DTT ) and reducing ( 100 mM DTT ) conditions with purified OxyR and RNA polymerase in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2183352" target="_blank">2183352</a>]<br></li>
<li>At the oxyRSpromoter , OxyR binds under both oxidizing and reducing conditions and is a repressor of its own expression and an activator of oxyS.When two nucleotides at contact point 5 in the oxyRS site were mutated to the corresponding bases in the ahpC promoter ( O1020304X ) , reduced OxyR was no longer able to bind in vitro and did not repress the oxyR promoter in vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8087856" target="_blank">8087856</a>]<br></li>
<li>The lengths of the OxyR footprints at the katG and oxyR promoters are also different under oxidizing and reducing conditions ( Fig. 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2183352" target="_blank">2183352</a>]<br></li>
<li>The oxyR and oxyS promoters overlap , and OxyR bound to its site at these promoters acts to both repress oxyR expression ( under oxidizing and reducing conditions ) and activate oxyS expression ( only under oxidizing conditions ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7868602" target="_blank">7868602</a>]<br></li>
<li>Formerly , it was considered that OxyR behaves as a transcriptional autorepressor under both inducing and non-inducing conditions ( Christman et al. 1989 ) , but several recent lines of evidence suggest that variation of oxyR expression has an important biological role .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11479716" target="_blank">11479716</a>]<br></li>
<li>Ascorbate activation of ahpC is dependent on OxyR Two lines of evidence confirm that the increased expression of ahpC observed in the presence of L-ascorbate is controlled by the activator OxyR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17906139" target="_blank">17906139</a>]<br></li>
<li>We examined the transcription factors known to respond to nitric oxide ( SoxS and OxyR ) or to influence hmp transcription in E. coli ( Fnr ) for their contribution to aerobic hmp expression in S. typhimurium ( 2 , 15 , 27 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9852058" target="_blank">9852058</a>]<br></li>
<li>Indeed , in Salmonella typhimurium , expression of hmp also is induced after NO treatment ( 18 ) , and its control is independent of the SoxS and OxyR transcription factors but relies on the irondependent Fur repressor ( 18 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12475930" target="_blank">12475930</a>]<br></li>
<li>hmp gene expression in the presence of NOz is independent of the SoxS , OxyR , and FNR transcription factors and instead relies on inactivation of the iron-de-pendent Fur repressor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9852058" target="_blank">9852058</a>]<br></li>
<li>This protection is not seen with extracts made from strains that do not overproduce OxyR and suggests that the mechanism by which OxyR negatively autoregulates its is and blocking , expression by binding to , therefore the DNA that are for its own sequences required transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2471187" target="_blank">2471187</a>]<br></li>
<li>In the absence of H2O2 , the transcription factor OxyR is kept in its reduced ( red ) state by the glutaredoxin GlrX , which allows the basal expression of katG and ychF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23139412" target="_blank">23139412</a>]<br></li>
<li>Another transcription factor , OxyR , induces expression of mntH during periods of H2O2 stress ( Kehres et al. , 2000a , Kehres et al. , 2002b ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19400769" target="_blank">19400769</a>]<br></li>
<li>YchF expression is negatively regulated by the transcription factor OxyR in response to H2O2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23139412" target="_blank">23139412</a>]<br></li>
<li>We furthermore showed that YchF expression is repressed by the transcription factor OxyR and further post-translationally modified in response to H2O2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23139412" target="_blank">23139412</a>]<br></li>
<li>Finally , we show that YchF expression is repressed by the transcription factor OxyR in an H2O2-dependent manner .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23139412" target="_blank">23139412</a>]<br></li>
<li>For example , a family of regulatory proteins such as OxyR , LysR , CysB , NodD , and MetR , that act as positive regulator for other genes , were shown to negatively regulate their own expression ( 30 , 32 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1715982" target="_blank">1715982</a>]<br></li>
<li>Additional genes are indirectly controlled by OxyR as it induces the synthesis of OxyS , a small non-coding RNA that controls the expression of additional genes ( 36 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23139412" target="_blank">23139412</a>]<br></li>
<li>To further explore the OxyR role in the responses to GSNO or NaNO2 and to compare these to the OxyR-mediated response to H2O2 , we also assayed the expression of three target genes by primer extension ( Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14718666" target="_blank">14718666</a>]<br></li>
<li>All of the mutants showed the same relative expression seen with the oxyS gene , suggesting that the individual OxyR mutants had similar effects on all target genes ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7868602" target="_blank">7868602</a>]<br></li>
<li>To determine whether INH acts to oxidize OxyR , leading to the induction of OxyR-regulated genes , the expression of the oxyS gene in response to INH was assayed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7986015" target="_blank">7986015</a>]<br></li>
<li>OxyR also has been shown to be a repressor of its own expression as well as that of fhuF ( encoding a ferric ion reductase ) and ﬂu ( encoding the antigen 43 outer membrane protein ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11443091" target="_blank">11443091</a>]<br></li>
<li>OxyR represses its own expression , as well as transcription of the Mu phage mom and the E. coli ﬂu genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11443092" target="_blank">11443092</a>]<br></li>
<li>OxyR protein also represses its own synthesis ( 10 , 39 , 42 ) as well as that of the Mu phage mom gene ( 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8990283" target="_blank">8990283</a>]<br></li>
<li>OxyR can act as a transcriptional activator ( 39 , 42 , 43 ) and a repressor , repressing its own expression ( 10 ) and that of the mom gene of phage Mu ( 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8990283" target="_blank">8990283</a>]<br></li>
<li>During normal growth , OxyR also acts as a repressor , negatively autoregulating its own expression and the expression of the Mu phage mom gene ( 6 , 14 , 38 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7868602" target="_blank">7868602</a>]<br></li>
<li>We also identiﬁed several genes , including uxuA , encoding mannonate hydrolase , whose expression might be repressed by OxyR , since their expression was elevated in the DoxyR mutant strain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11443091" target="_blank">11443091</a>]<br></li>
<li>Unlike in class II mutants , in class III mutants the expression of the lacZ fusion was repressed by a functional OxyR protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8990283" target="_blank">8990283</a>]<br></li>
<li>hps-19 : : lacZ expression was repressed in the presence of OxyR ; thus , this fusion is a class III fusion mutant .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8990283" target="_blank">8990283</a>]<br></li>
<li>However , OxyR incubated in the presence of 100 mM DTT no longer activated expression of katG ( Fig. 3B , lane 3 compared to lane 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2183352" target="_blank">2183352</a>]<br></li>
<li>If the transcription assays were performed in the presence of 10 mM DTT under a nitrogen atmosphere , OxyR no longer activated katG expression ( Fig. 3C , lanes 3 and 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2183352" target="_blank">2183352</a>]<br></li>
<li>The oxidized form of OxyR appears to be more efficient than the reduced form in repressing its own expression , in agreement with the observation that the synthesis of OxyR is decreased in cells treated with hydrogen peroxide compared to untreated cells ( Fig. 1B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2183352" target="_blank">2183352</a>]<br></li>
<li>The oxyR expression was reduced substantially in the presence of both reduced and oxidized OxyR ( Fig. 4 , lanes 2 and 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2183352" target="_blank">2183352</a>]<br></li>
<li>Activation of katG expression in A vitro by oxidized but not reduced OxyR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2183352" target="_blank">2183352</a>]<br></li>
<li>Overall , the pattern of OxyR protein expression paralleled the transcriptional behavior of the oxyR gene reasonably well .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9324269" target="_blank">9324269</a>]<br></li>
<li>The expression of the control bla gene was unaffected by the different OxyR mutants .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7868602" target="_blank">7868602</a>]<br></li>
<li>The increased expression from a heterologous oxyR : : lacZ gene fusion in an oxyR deletion strain compared to an oxyR ' strain suggests that OxyR is also a negative regulator of its own expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2471187" target="_blank">2471187</a>]<br></li>
<li>The expression of the oxyR ' - ` lacZ gene was higher in an oxyiZ-deletion strain than in the oxyR + strain , and was repressed by overexpressing the OxyR protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1864839" target="_blank">1864839</a>]<br></li>
<li>The result suggests that OxyR protein represses the expression of the oxyR gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1864839" target="_blank">1864839</a>]<br></li>
<li>OxyR is an activator of katG and ahpCF expression and negatively regulates its own expression ( 10 , 13 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2183352" target="_blank">2183352</a>]<br></li>
<li>The expression level of hcaR was also measured in soxR and oxyR mutants because SoxR [ 5 ] and OxyR [ 31 ] regulators modulate expression of many genes including sodA , gor , katG , ahpC and ahpF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15808934" target="_blank">15808934</a>]<br></li>
<li>Increasing amounts of OxyR increased the transcription of both the ahpC ( 15 ) and katG genes ( Fig. 3A ) while having no effect on the expression of the bla gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2183352" target="_blank">2183352</a>]<br></li>
<li>By stimulating oxyR expression , putrescine also caused an increase in the expression of katG , the target gene for OxyR ( Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11479716" target="_blank">11479716</a>]<br></li>
<li>At the oxyR-binding site , OxyR not only represses expression of its own gene , but also activates the expression of oxyS , a gene transcribed divergently from oxyR ( S. Altuvia and G. S. , unpublished data ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8087856" target="_blank">8087856</a>]<br></li>
<li>When binding to the promoter region , OxyR represses its own expression , and upon oxidation , it activates oxyS transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7868602" target="_blank">7868602</a>]<br></li>
<li>OxyR binding upstream of dsbG activates ahpCF expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11443092" target="_blank">11443092</a>]<br></li>
<li>These assays also showed that OxyR does not actively repress oxyS expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8087856" target="_blank">8087856</a>]<br></li>
<li>These results show that OxyR represses fhuF expression under conditions of oxidative stress .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11443092" target="_blank">11443092</a>]<br></li>
<li>Our current dat show that OxyR also represses the expression of fhuF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11443092" target="_blank">11443092</a>]<br></li>
<li>Thus , it is conceivable that OxyR is regulating the expression of a transcript encoded by the strand opposite yfdI .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11443092" target="_blank">11443092</a>]<br></li>
<li>This suggests that ychF expression is regulated by OxyR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23139412" target="_blank">23139412</a>]<br></li>
<li>This finding suggests that OxyR is a negative regulator of its own expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2471187" target="_blank">2471187</a>]<br></li>
<li>Here both FNR and OxyR are shown to regulate expression of the E. coli yhjA gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17464064" target="_blank">17464064</a>]<br></li>
<li>These data suggest that OxyR is required for expression of yhjA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17464064" target="_blank">17464064</a>]<br></li>
<li>For this reason , yaaA mutants resemble dps mutants in that they do not exhibit unusual sensitivity in experiments that involve a sudden bolus of H2O2 ; a period of OxyR regulon expression is required for it to become functional .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21378183" target="_blank">21378183</a>]<br></li>
<li>Interestingly , in ahp-deﬁcient ( catalase-proﬁcient ) mutants that experience minor H2O2 stress ( 44 ) , the expression of yaaA was at the basal level during aerobic culture ( Fig. 2A ) , even though katG , another member of the OxyR regulon , was strongly induced ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21378183" target="_blank">21378183</a>]<br></li>
<li>It regulates the expression of the superoxide dismutases ( repression of the manganese superoxide dismutase and activation of the iron one ) ( 22 , 23 ) , and fur expression is under control of OxyR and SoxRS ( 24 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12475930" target="_blank">12475930</a>]<br></li>
<li>There are 10 genes of unknown function among the most highly induced genes whose expression is independent of OxyR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11443091" target="_blank">11443091</a>]<br></li>
<li>Antioxidant defense systems are mediated through regulators ( OxyR , SoxRS , Fur and RpoS ) whose expression is most often induced by ROS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16183297" target="_blank">16183297</a>]<br></li>
<li>It is possible that the expression of other transcripts that are induced independently of OxyR is under the control of the SoxRS regulators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11443091" target="_blank">11443091</a>]<br></li>
<li>However , under the same conditions its expression was induced 10-fold in a catalase/peroxidase-de ﬁcient strain ( Hpx ) , which accumulates up to 1 M endogenous H2O2 and in which the OxyR response is fully activated ( 43 ) ( Fig. 2A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21378183" target="_blank">21378183</a>]<br></li>
<li>KatG is the major catalase during exponential growth of E. coli , and its expression is induced by OxyR in response to H2O2 ( 26 , 27 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23139412" target="_blank">23139412</a>]<br></li>
<li>Although AhpCF and KatG expression is regulated by OxyR in an H2O2-dependent manner , the third catalase in E. coli , KatE , is also under the control of S and is induced during stationary phase ( 29 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23139412" target="_blank">23139412</a>]<br></li>
<li>We previously showed that a decrease in intracellular gluta-thione by nonpenetrating oxidant ferricyanide accelerated the H2O2-induced expression of the katG gene controlled by OxyR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11440837" target="_blank">11440837</a>]<br></li>
<li>The H2O2-resistant function of the SUF system is in agreement with the expression profile of the suf operon , i.e. the chromosomal suf operon is markedly induced by OxyR upon treatment with H2O2 ( 36 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15496591" target="_blank">15496591</a>]<br></li>
<li>For example , gene expression of sufABCDSE is induced by H2O2 ( Zheng et al. 2001 ) , and oxidant induction of the suf operon is regulated by OxyR , IHF , Fur and IscR ( Lee et al. 2008 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19919618" target="_blank">19919618</a>]<br></li>
<li>These results suggested that the ahpCF and the katG ahpCF mutant strains are in a constant state of oxidative stress and that OxyR is constitutively activated because the peroxides produced by normal aerobic metabolism are not dissipated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7986015" target="_blank">7986015</a>]<br></li>
<li>The OxyR stress response is activated by as little as 0.1 μM H2O2 ( Aslund et al. , 1999 ) , and we confirmed that an mntH ' - lacZ + fusion was strongly induced in the Hpx - mutant when it was transferred from anaerobic to aerobic media ( Fig. 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19400769" target="_blank">19400769</a>]<br></li>
<li>Transcription of fur is under positive control of both OxyR and SoxRS ( Zheng et al. , 1999 ) , which in turn are activated by oxidative stress .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14617150" target="_blank">14617150</a>]<br></li>
<li>During normal growth as well as upon oxidative stress , OxyR also acts to repress its own expression ( Christman et al. , 1989 ; Tao et al. , 1991 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8087856" target="_blank">8087856</a>]<br></li>
<li>Therefore , OxyR can oxyR momR act to repress and expression during normal growth and activate katG , ahpC , dps , and gorAupon oxidative stress ( Figure 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8087856" target="_blank">8087856</a>]<br></li>
<li>The shift in binding also allows OxyR to repress its own expression under both oxidizing and reducing conditions , while only activating katG , ahpC , gorA , and dps upon oxidative stress .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8087856" target="_blank">8087856</a>]<br></li>
<li>The ahpCF operon is controlled by OxyR , a positive regulator of the oxidation stress response ( 16 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7644465" target="_blank">7644465</a>]<br></li>
<li>The expression of trxC is regulated by the transcriptional activator OxyR in response to oxidative stress .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10644706" target="_blank">10644706</a>]<br></li>
<li>The gorA gene does not show strong OxyR regulation ( 7 ) , but dps , trxC , and hemH ar clearly induced by OxyR in response to oxidative stress ( 9 , 17 , 24 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11443092" target="_blank">11443092</a>]<br></li>
<li>DISCUSSION The data presented here strongly suggest that phase variation of Ag43 is not affected in cells experiencing peroxidemediated oxidative stress and furthermore that under these conditions the OFF phase can be a result of repression by oxidized OxyR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12644490" target="_blank">12644490</a>]<br></li>
<li>Expression of Dps is known to be regulated by OxyR , in response to oxidative stress ( 2 ) , and three nucleoid proteins , IHF , Fis , and H-NS , in response to growth phase ( 2 , 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21239586" target="_blank">21239586</a>]<br></li>
<li>Expression of yhjA was found to be dependent on this unusual combination of FNR and OxyR under conditions of oxygen starvation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17464064" target="_blank">17464064</a>]<br></li>
<li>The regulation of yhjA expression is interesting , because transcription is dependent on two transcription factors , FNR and OxyR , that respond to two seemingly incompatible signals , i.e. oxygen starvation and peroxide stress .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17464064" target="_blank">17464064</a>]<br></li>
<li>HPI is an adaptive enzyme regulated by OxyR , a major regulatory transcription factor responding to oxidative stress w5 ,6 x. HPII is expressed maximally in stationary phase , and its expression is completely dependent on the product of katF w2 ,7 x. Although katF was first identified as a positive regulator of HPII w2 ,7 x , it has been found to control the expression of many stationary phase specific and osmotically regulated genes w8x .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9199247" target="_blank">9199247</a>]<br></li>
<li>The transcription of ~ 60 genes or operons was dependent on OxyR under anaerobic conditions ( tables S1 and S2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22539721" target="_blank">22539721</a>]<br></li>
<li>The transcription factor OxyR , activated by oxidation under aerobic conditions , is also activated by S-nitrosylation under anaerobic conditions and thereby induces expression in a distinct regulon , which reveals that a transcriptional activator can independently control two distinct regulons through alternative posttranslational modifications .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22539721" target="_blank">22539721</a>]<br></li>
<li>Our results show that dps is regulated independently by OxyR and a ^ and is a part of both types of defence regulon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7984106" target="_blank">7984106</a>]<br></li>
<li>Although OxyR can activate transcription by Es70 , and IHF can enhance transcription by Es38 ( Altuvia et al. , 1994 ; Jeong et al. , 2006 ) , the molecular basis of how dps expression is regulated by growth phase is not known .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18452510" target="_blank">18452510</a>]<br></li>
<li>Altuvia et al. ( 1994 ) reported that the dps promoter can be transcribed by Es during exponential phase , and that such transcription 70 depends on stimulation by the activator OxyR in oxidative conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10692161" target="_blank">10692161</a>]<br></li>
<li>We and others have observed that several genes such as katG , dps , and gorA , are regulated by both OxyR and ss ( Altuvia et al. , 1994 ; Ivanova et al. , 1994 ; Becker-Hapak and Eisenstark , 1995 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9230301" target="_blank">9230301</a>]<br></li>
<li>If OxyR mediates the high expression of ahpC observed in cell cultures grown in the presence of L-ascorbate , other genes regulated by OxyR such as dps ( Altuvia et al. , 1994 ) must be activated under these conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17906139" target="_blank">17906139</a>]<br></li>
<li>If cells are treated with hydrogen peroxide during exponential growth , dps transcription is induced by OxyR , which activates cĵ ° - RNA polymerase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7984106" target="_blank">7984106</a>]<br></li>
<li>It has been recently shown not only that katG ( 23 , 29 ) , dps ( 3 ) , and gorA ( 4 ) are members of the OxyR regulon but that their growth phase expression is regulated by RpoS , the second principal sigma factor responsible for starvation stress-induced gene expression ( 24 , 26 , 41 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8990283" target="_blank">8990283</a>]<br></li>
<li>Since it is known that the expression of katG ( 23 , 29 ) , dps ( 3 ) , and gorA ( 4 ) genes is regulated by both OxyR ( in a hydrogen peroxide-inducible manner ) and RpoS in a growth phase-dependent manner , we wanted to determine whether any of the hps genes were also regulated by RpoS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8990283" target="_blank">8990283</a>]<br></li>
<li>The expression of katG ( encoding HPI hydroperoxidase ) increases to a maximum as the cultures enter into stationary phase and is regulated at the level of transcription by RpoS ; however , the hydrogen peroxide inducibility is regulated in exponentialphase cultures at the level of transcription by OxyR ( 10 , 28 , 29 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8990283" target="_blank">8990283</a>]<br></li>
<li>The presence of bands when using a dps probe and cells extracts of cells grown in the presence of L-ascorbate ( Fig. 8b ) reinforce the idea that OxyR mediates the high-level expression of ahpC in the presence of L-ascorbate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17906139" target="_blank">17906139</a>]<br></li>
<li>rpoS - , OxyR - , and IHF-dependent regulation of dps expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7559363" target="_blank">7559363</a>]<br></li>
<li>OxyR - , and IHP-dependent regulation of dps expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7715442" target="_blank">7715442</a>]<br></li>
<li>oxyR Is Negatively Autoregulated by the Binding of OxyR to Its Own Promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2471187" target="_blank">2471187</a>]<br></li>
<li>which is repressed by OxyR ( 11 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2183352" target="_blank">2183352</a>]<br></li>
<li>, 1994 ) are alsoregulated by both OxyR and 0 ' .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8899708" target="_blank">8899708</a>]<br></li>
<li>The plasmid pASK-IBA3C-OxyR was constructed by amplifying oxyR from chromosomal E. coli DNA using the primers OxyRFw and OxyRrev .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23139412" target="_blank">23139412</a>]<br></li>
<li>The oxyR transcriptton is repressed by addition ot OxyR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8483417" target="_blank">8483417</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8483417" target="_blank">8483417</a>]<br></li>
<li>Binding of OxyR to the promoter region provides an explanation of how OxyR can autoregulate its own synthesis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2551682" target="_blank">2551682</a>]<br></li>
<li>The extensive mutagenesis of the promoter to inactivate the OxyR site is likely to have disrupted other regulatory elements in this region ( 294 to 257 ) , but this was not sufﬁcient to enhance FNR-mediated yhjA expression ( Table 3 ) , and suggests that any such regulatory element is located upstream ( 2127 to 294 ) of the OxyR site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17464064" target="_blank">17464064</a>]<br></li>
<li>The oxidized OxyR binds to promoter elements of the genes it regulates , resulting in the onset of their expression , and the protein also represses its own synthesis ( 3 , 19 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9922271" target="_blank">9922271</a>]<br></li>
<li>The DNA sequence of the oxyRS promoter region is such that both these patterns of binding are possible , and thus OxyR regulates its own expression , when reduced and oxidized , simply by repositioning itself on the DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17464064" target="_blank">17464064</a>]<br></li>
<li>The expression pattern of the hps-19 : : lacZ mutant suggests that the promoter element ( s ) is recognized by both RpoS and OxyR and that OxyR acts as a repressor , as is evident from its own expression ( 10 , 44 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8990283" target="_blank">8990283</a>]<br></li>
<li>The stationary phase-induced , rpoSdependent expression of Dps ( compare lanes 7 and 10 ) was also not affected by the lack of OxyR ( compare lanes 7 and 9 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7984106" target="_blank">7984106</a>]<br></li>
<li>The transcription of nhaA , which is positively regulated by NhaR ( 29 ) , a member of the LysR-OxyR family ( 7 , 14 , 30 , 34 ) , is speciﬁcally induced by intracellular Na1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11133959" target="_blank">11133959</a>]<br></li>
<li>The expression of nhaA is positively regulated by nhaR ( 39 ) , a member of the LysR-OxyR family ( 6 , 16 , 42 , 46 ) , and is induced by Na1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8932307" target="_blank">8932307</a>]<br></li>
<li>The dps gene , which is transcribed from a single promoter , is controlled by OxyR , which similar to NhaR , is a member of the LysR-OxyR family of positive activators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11133959" target="_blank">11133959</a>]<br></li>
<li>The katG gene was shown to be activated by nitrosylated OxyR in vitro ( 30 ) , and a plasmid-borne katG -- lacZ fusion was reported to be induced by 1 mM S-nitrosocysteine in vivo ( 27 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14718666" target="_blank">14718666</a>]<br></li>
<li>RESULTS The gene yaaA is induced by OxyR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21378183" target="_blank">21378183</a>]<br></li>
<li>The mntH gene is additionally repressed by iron-loaded Fur , and both it and dps are induced by H2O2-oxidized OxyR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25774656" target="_blank">25774656</a>]<br></li>
<li>Both OxyR and glutathione are also involved in the regulation of katG ( HPI ) [ 13,33 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16183297" target="_blank">16183297</a>]<br></li>
<li>Primer extension assays showed that transcription of the OxyR-regulated oxyS gene was not significantly induced by INH in wild-type cells , indicating that INH does not activate OxyR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7986015" target="_blank">7986015</a>]<br></li>
<li>encoding HPI catalase , is regulated by both OxyR and a ^ ( Ivanova et ai , 1994 ; Mukhopadhyay and Schellhorn , 1994 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7984106" target="_blank">7984106</a>]<br></li>
<li>Since the hemF gene product is required for functional HPI catalase synthesis , it is not surprising that hemF and katG are regulated by OxyR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8990283" target="_blank">8990283</a>]<br></li>
<li>The Escherichia coli yhjA gene , encoding a predicted cytochrome-c peroxidase , is regulated by FNR and OxyR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20522491" target="_blank">20522491</a>]<br></li>
<li>Glutathione , encoded by gorA , is regulated by OxyR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16183297" target="_blank">16183297</a>]<br></li>
<li>Therefore , it is not clear whether the grx gene is regulated by OxyR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9294462" target="_blank">9294462</a>]<br></li>
				</div></ul></div><br>
	</div>
</body>
</html>