<!doctype html>
<style>
@charset "utf-8";
body {
	height: 100%;
	margin: 0;
	padding: 0;
}

.banner {
	position: fixed;
	top: 0;
	width: 100%;
	height: 65px;
	background: gray;
	line-height: 65px;
}
.spacer {
	height: 10px;
}

.main-content {
	padding: 5%;
}

.collapse_ACT {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-style: none solid none none;
	border-color: #B4045F;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_ACT + input{
	display: none;
}
.collapse_ACT + input + div{
	display: none;
}
.collapse_ACT + input:checked + div{
	display: block;
}
.ACT {
	text-align: justify;
	text-justify: inter-word;
}
.ACT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B4045F;
}
.ACT_collapse_similarity:hover{
	background: #e5d3dc;
}
.ACT_collapse_similarity + input {
	display: none;
}
.ACT_collapse_similarity + input + div{
	display: none;
}

.ACT_collapse_similarity + input:checked + div{
	display: block;
}
.ACT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.ACT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B4045F;
	padding-left: 5px;
}
.ACT_collapse_similarity:hover{
	background: #e5d3dc;
}
.ACT_collapse_similarity + input {
	display: none;
}
.ACT_collapse_similarity + input + div{
	display: none;
}
.ACT_collapse_similarity + input:checked + div{
	display: block;
}
.ACT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.ACT_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #B4045F;
}
.ACT_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_DOM {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-style: none solid none none;
	border-color: #B404AE;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_DOM + input{
	display: none;
}
.collapse_DOM + input + div{
	display: none;
}
.collapse_DOM + input:checked + div{
	display: block;
}
.DOM {
	text-align: justify;
	text-justify: inter-word;
}
.DOM_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B404AE;
	padding-left: 5px;
}
.DOM_collapse_similarity:hover{
	background: #e8d7e7;
}
.DOM_collapse_similarity + input {
	display: none;
}
.DOM_collapse_similarity + input + div{
	display: none;
}
.DOM_collapse_similarity + input:checked + div{
	display: block;
}
.DOM_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}.DOM_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #B404AE;
}
.DOM_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_EVO {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #7401DF;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_EVO + input{
	display: none;
}
.collapse_EVO + input + div{
	display: none;
}
.collapse_EVO + input:checked + div{
	display: block;
}
.EVO {
	text-align: justify;
	text-justify: inter-word;
}
.EVO_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #7401DF;
	padding-left: 5px;
}
.EVO_collapse_similarity:hover{
	background: #e2d7ed;
}
.EVO_collapse_similarity + input {
	display: none;
}
.EVO_collapse_similarity + input + div{
	display: none;
}
.EVO_collapse_similarity + input:checked + div{
	display: block;
}
.EVO_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.EVO_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #7401DF;
}
.EVO_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_RP {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #0174DF;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_RP + input{
	display: none;
}
.collapse_RP + input + div{
	display: none;
}
.collapse_RP + input:checked + div{
	display: block;
}
.RP {
	text-align: justify;
	text-justify: inter-word;
}
.RP_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #0174DF;
	padding-left: 5px;
}
.RP_collapse_similarity:hover{
	background: #c9d7e2;
}
.RP_collapse_similarity + input {
	display: none;
}
.RP_collapse_similarity + input + div{
	display: none;
}
.RP_collapse_similarity + input:checked + div{
	display: block;
}
.RP_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.RP_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #0174DF;
}
.RP_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_SIT {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #0489B1;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_SIT + input{
	display: none;
}
.collapse_SIT + input + div{
	display: none;
}
.collapse_SIT + input:checked + div{
	display: block;
}
.SIT {
	text-align: justify;
	text-justify: inter-word;
}
.SIT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #0489B1;
	padding-left: 5px;
}
.SIT_collapse_similarity:hover{
	background: #d0dfe2;
}
.SIT_collapse_similarity + input {
	display: none;
}
.SIT_collapse_similarity + input + div{
	display: none;
}
.SIT_collapse_similarity + input:checked + div{
	display: block;
}
.SIT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.SIT_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #0489B1;
}
.SIT_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_TU {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #04B486;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_TU + input{
	display: none;
}
.collapse_TU + input + div{
	display: none;
}
.collapse_TU + input:checked + div{
	display: block;
}
.TU {
	text-align: justify;
	text-justify: inter-word;
}
.TU_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #04B486;
	padding-left: 5px;
}
.TU_collapse_similarity:hover{
	background: #d0e2de;
}
.TU_collapse_similarity + input {
	display: none;
}
.TU_collapse_similarity + input + div{
	display: none;
}
.TU_collapse_similarity + input:checked + div{
	display: block;
}
.TU_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.TU_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #04B486;
}
.TU_similar > a {
	text-decoration: none;
	color: gray;
}

ul { 
	list-style-type: none;
	padding: 0;
	margin: 0;
}
ul li > a {
	text-decoration: none;
	color: gray;
}
ul li:hover, ul li:focus {
	background: #F0F0F0;
}

ul li:first-child span {
  border-left:none;  
}

</style>
<html>
<head>
<meta charset="utf-8">
<title>TF properties - PaaX</title>
<meta name="theme-color" content="#ffffff">
</head>
<body>
	<div class="banner">
		<center><font color="white" size="16px" face="arial">PaaX</font></center>
	</div>
	<div align="center" class="main-content">
		<div class="spacer">
	</div><br>			<label class="collapse_ACT" for="_1"><font size="5px">PaaX transcriptional activity</font></label>
				<input id="_1" type="checkbox">
				<div class="ACT"><br><ul><li>del Peso-Santos , T. , Bartolome-Martin , D. , Fernández , C. , Alonso , S. , García , J.L. , Díaz , E. , et al. ( 2006 ) Coregulation by phenylacetyl-coenzyme A-responsive PaaX integrates control of the upper and lower pathways for catabolism of styrene by Pseudomonas sp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24983528" target="_blank">24983528</a>]<br></li>
<li>Regulation of Penicillin G Acylase Gene Expression in Escherichia coli by Repressor PaaX and the cAMP-cAM Receptor Protein Complex * □ S P Received for publication , April 20 , 2004 , and in revised form , May 14 , 2004 Published , JBC Papers in Press , May 24 , 2004 , DOI 10.1074 / jbc.M404348200 Hyoung Seok Kim ‡ , Tae Sun Kang § , Joon Sik Hyun § , and Hyen Sam Kang ¶ From the Department of Microbiology , School of Biological Sciences , Seoul National University , San 56-1 , Shillim Kwanak-gu , 151-742 , Korea - dong , paa operon , from binding to DNA ( 12 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15159386" target="_blank">15159386</a>]<br></li>
<li>First , Ferrandez et al. ( 12 ) observed that mutations within our 6-base element reduced PaaX binding and resulted in less repression in the - galactosidase assay , and the effect of 2-base substitutions within the 6-base element was greater than that of 1-base substitutions ( pAFPA2-M1 , - M5 , - M9 , and - M16 in Fig. 5 and Table III of Ref .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15159386" target="_blank">15159386</a>]<br></li>
<li>Increasing concentrations of PaaX cause a decrease in the yield of the RNAP-PZ complex and the appearance of a gel retardation band corresponding to the PaaX-PZ complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24983528" target="_blank">24983528</a>]<br></li>
<li>In addition to the operon-specific regulation mediated by the PaaX repressor , the paa catabolic cluster of E. coli is subjected to control by the two global regulators , CRP ( 23 ) and IHF ( 22 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10766858" target="_blank">10766858</a>]<br></li>
<li>This diversity in the mechanism of transcriptional repression by PaaX is noteworthy because it acts as an upstream repressor and functions by displacing the activator , cAMP-CRP , in the pga promoter , unlike most bacterial repressors that inhibit either RNA polym-erase , open-complex formation , or transcriptional activation by binding close to the transcription initiation site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15159386" target="_blank">15159386</a>]<br></li>
<li>In IG this model the transcriptional-repressor , PaaX , and the transcriptional-activator , the cAMP-CRP complex , compete for binding to the upstream region of the pga promoter that overlaps with the CRP-binding site I. Details are given in the text .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15159386" target="_blank">15159386</a>]<br></li>
<li>When PAA is supplied it enters the nucleoplasm and is converted to PAA-CoA by PaaK , and this then inhibits PaaX from binding to the pga operator by blocking the physical interaction between PaaX monomers .2 The cAMP-CRP complex is thus now able to-bind to the unoccupied CRP-binding site I after binding to the higher affinity CRP-binding site II , and then fully activates pga transcription in the manner typical of class III CRP-dependent promoter ( Fig. 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15159386" target="_blank">15159386</a>]<br></li>
<li>In view of the fact that the operator for PaaX covers only the CRP-binding site I , cAMP-CRP could conceivably bind to site II when PAA is absent .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15159386" target="_blank">15159386</a>]<br></li>
<li>In the absence of PAA , pga is poorly expressed because the cAMP-CRP complex can not bind to the CRP-binding site I as the latter is occupied by PaaX .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15159386" target="_blank">15159386</a>]<br></li>
<li>Fig. 7 , panel A , shows that PaaX binds very tightly to the pga promoter in the absence of PAA but fails to-bind in its presence and that , conversely , the cAMP-CRP complex binds strongly to the pga promoter in the presence of PAA but hardly at all in its absence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15159386" target="_blank">15159386</a>]<br></li>
<li>Mutually exclusive binding of PaaX and cAMP-CRP to the pga promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15159386" target="_blank">15159386</a>]<br></li>
<li>Panel A , competitive binding of PaaX and CRP to the pga promoter according to the presence or absence of PAA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15159386" target="_blank">15159386</a>]<br></li>
<li>However , RNAP is able to-bind to the Pa promoter even in the presence of increasing concentrations of PaaX ( Fig .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24983528" target="_blank">24983528</a>]<br></li>
<li>RNAP is able to-bind to the Pa promoter even in the presence of increasing concentrations of PaaX , and a low mobility complex due to the interaction of the PA probe with RNAP , PaaX and cAMP-CRP is observed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24983528" target="_blank">24983528</a>]<br></li>
<li>Thus , these data strongly suggest that the mechanism of repression of PaaX on its own promoter is based on the steric hindrance of RNAP binding to the Px promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24983528" target="_blank">24983528</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_17">
In vitro binding of puriﬁed PaaX , RNAP and CRP to the Pz promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24983528" target="_blank">24983528</a>]<br></label>
<input id="act_similarity_17", type="checkbox">
<div class="ACT_similar">
<li>In vitro binding of puriﬁed PaaX , RNAP and CRP to the Pa promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24983528" target="_blank">24983528</a>]<br></li></div>
<li>Under normal conditions , PaaX , the repressor of the paa operon encoded by paaX , binds to the divergent Pa and Pz promoters to prevent transcription of the meta-cleavage pathway gene cluster .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15159386" target="_blank">15159386</a>]<br></li>
<li>Although it is likely that the effect of the PaaX protein on the Pa and Pz promoters will involve different mechanisms of repression , validation of this assumption requires further research .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10766858" target="_blank">10766858</a>]<br></li>
<li>Mechanism of PaaX-mediated repression at the Px , Pa and Pz promoters The fact that the PaaX operator region overlaps the − 10 box of Px suggests that PaaX binding prevents the RNAP interaction and , therefore , leads to repression of Px .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24983528" target="_blank">24983528</a>]<br></li>
<li>However , when PAA is present , PaaK , the PAA-CoA ligase encoded by paaK , converts PAA to PAA-CoA and this inhibits PaaX from binding to the promoter and initiates expression from the Pa and Pz promoters ( 12 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15159386" target="_blank">15159386</a>]<br></li>
<li>PAA Relieves PaaX Repression of pga Expression -- As stated , PAA-CoA , generated from PAA by PaaK , inhibits PaaX from binding to the Pa and Pz promoters of the paa operon ( 12 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15159386" target="_blank">15159386</a>]<br></li>
<li>Therefore , these data taken together revealed the specific and independent binding of PaaX to the Pz and Pa promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10766858" target="_blank">10766858</a>]<br></li>
<li>B , binding affinity of PaaX to wild-type and mutant Pa promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10766858" target="_blank">10766858</a>]<br></li>
<li>Regulation of pga Expression by PaaX and the cAMP-CRP Complex -- To demonstrate antagonistic regulation of pga expression by PaaX as repressor , and CRP as activator , we constructed 3 - cMyc-tagged derivatives of recombinant paaX and crp and performed ChIP assays .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15159386" target="_blank">15159386</a>]<br></li>
<li>Panel B , PaaX repression of pga during normal growth .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15159386" target="_blank">15159386</a>]<br></li>
<li>During normal growth of E. coli W , PaaX is expressed and binds to its cognate region on the pga operator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15159386" target="_blank">15159386</a>]<br></li>
<li>Because the major difference between the paaX mutant and the wild type is the presence of free PaaX in the latter that can bind to the pga promoter and be released from it by PAA-CoA , we infer that relief from PaaX repression is necessary for efficient expression of pga .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15159386" target="_blank">15159386</a>]<br></li>
<li>'' Our findings therefore provide answers to some of the questions posed by Diaz et al. ( 10 ) concerning the regulatory proteins involved in PAA induction ; PaaX is responsible for repression of pga expression and PAA for derepression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15159386" target="_blank">15159386</a>]<br></li>
<li>First , we have demonstrated that PaaX represses pga transcription , and this accounts both for negative regulation and PAA induction of pga expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15159386" target="_blank">15159386</a>]<br></li>
<li>Third , these findings have led us to characterize `` PAA induction , '' the increase of PGA production in the presence of PAA , as primarily resulting from relief of pga from repression by PaaX .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15159386" target="_blank">15159386</a>]<br></li>
<li>Thus , since we have shown here that PAA induction consists primarily of relief of pga from PaaX repression by PAA , we suggest that the increase in expression of pga in response to PAA be renamed `` PAA derepression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15159386" target="_blank">15159386</a>]<br></li>
<li>We therefore hypothesized that the increase in pga expression that occurs in response to exogenous PAA was due to PAA-CoA-mediated relief of pga from repression by PaaX .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15159386" target="_blank">15159386</a>]<br></li>
<li>We conclude that the process of `` PAA induction '' actually involves relief of pga from repression by PaaX .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15159386" target="_blank">15159386</a>]<br></li>
<li>PAA relieves PaaX repression of pga expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15159386" target="_blank">15159386</a>]<br></li>
<li>It is worth noting that the FadR transcriptional-repressor , which is regulated by acyl-CoA compounds and shows local similarity to PaaX , is also controlling the expression of genes involved in b-oxidation mechanisms ( 73 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9748275" target="_blank">9748275</a>]<br></li>
<li>PaaX and PaaY may be involved in regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12846838" target="_blank">12846838</a>]<br></li>
<li>However , in the presence of a functional PaaX repressor , the absence of a functional paaY gene leads to a long lag phase in PA-containing minimal medium ( Fig. 3B ) , suggesting an inefficient induction of the paa genes and , therefore , the involvement of PaaY in the regulatory circuit .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24983528" target="_blank">24983528</a>]<br></li>
<li>Moreover , we have observed that PaaY plays a second and unexpected role by facilitating the induction of the paa genes whose expression is inhibited by the speciﬁc transcriptional-repressor PaaX .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24983528" target="_blank">24983528</a>]<br></li>
<li>Lanes 3 -- 5 , increasing concentrations of PaaX ( 10 , 50 and 100 nM , respectively ) , the PX probe and RNAP ( 150 nM ) were added simultaneously .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24983528" target="_blank">24983528</a>]<br></li>
<li>Lane 6 , PaaX ( 100 nM ) , PX probe and RNAP ( 150 nM ) were incubated in the presence of 500 μM PA-CoA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24983528" target="_blank">24983528</a>]<br></li>
<li>B. Lane 1 shows the free PX probe ; lanes 2 -- 9 show retardation assays containing 60 nM of puriﬁed PaaX protein in the presence of 0 , 25 , 50 , 100 , 200 , 400 , 700 and 1000 μM of PA-CoA respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24983528" target="_blank">24983528</a>]<br></li>
<li>Lanes 8 -- 11 , show footprinting assays containing 27 nM of puriﬁed PaaX in the presence of 0 , 1 , 10 and 20 μM of PA-CoA respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24983528" target="_blank">24983528</a>]<br></li>
<li>Taking into account that PaaX recognizes PA-CoA as inducer , it is tempting to speculate that a CoA derivative resulting from the PA degradation pathway , e.g. 2-hydroxycyclohepta-1 ,4,6 - triene-1-carboxyl-CoA or 1,2-epoxyphenylacetyl-CoA , might act as an anti-effector ( antagonist ) of PaaX by interacting with this repressor and blocking its PA-CoA dependent release from the cognate promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24983528" target="_blank">24983528</a>]<br></li>
<li>Thus , the regulatory function of PaaY , which is only observed in the presence of PaaX and a functional PA degradation pathway , could be ascribed to the hydrolysis of some CoA derivative ( s ) formed during the catabo-lism of PA and that acts as an antagonist of PaaX , preventing the PA-CoA dependent activation of the paa genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24983528" target="_blank">24983528</a>]<br></li>
<li>The repressor effect of PaaX on Pa could not be significantly alleviated by growing the cells in the presence of 5 mM PA ( Table IV ) , which indicates that this aromatic compound is not the true inducer of the pathway .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9748275" target="_blank">9748275</a>]<br></li>
<li>The repressor effect of PaaX on the Pa promoter in E. coli W14 ( lacZ2 ) cells , containing the Pa-lacZ fusion into the chromosome and the paaX gene in a plasmid , could not be alleviated by growing the cells in the presence of PA , suggesting that this aromatic compound is not the inducer of the pathway ( 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10766858" target="_blank">10766858</a>]<br></li>
<li>In this work , using different in vivo and in vitro approaches , we have demonstrated that in E. coli , the paaX and paaY genes form a true operon that is autoregulated by PaaX in a PA-CoA dependent manner .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24983528" target="_blank">24983528</a>]<br></li>
<li>In this sense , gel retardation assays have confirmed PA-CoA as the effector-molecule .4 Therefore , PaaX constitutes the first reported transcription factor regulated by CoA derivatives that controls the catabolism of aromatic compounds .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9748275" target="_blank">9748275</a>]<br></li>
<li>Interestingly , PA , CoA , benzoyl-CoA , acetoacetyl-CoA or butyryl-CoA added at 0.8 mM to the assay were not able to inhibit the binding of PaaX to the PX probe ( data not shown ) , reinforcing the previous observation that PA-CoA is the true inducer .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24983528" target="_blank">24983528</a>]<br></li>
<li>PA-CoA , but not PA , speciﬁcally inhibits binding of PaaX to the target sequences of Pa and Pz , thus conﬁrming the ﬁrst intermediate of the pathway as the true inducer ( Ferrández et al. , 2000 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24983528" target="_blank">24983528</a>]<br></li>
<li>PA-coenzyme A ( PA-CoA ) , but not PA , specifically inhibited binding of PaaX to the target sequences , thus confirming the first intermediate of the pathway as the true inducer and PaaX as the only bacterial regulatory protein described so far that responds to an aryl-CoA compound .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10766858" target="_blank">10766858</a>]<br></li>
<li>As PA-CoA is the true inducer , PaaX becomes the first regulator of an aromatic catabolic pathway that responds to a CoA derivative .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9748275" target="_blank">9748275</a>]<br></li>
<li>As the repression caused by PaaX was only alleviated by PA in the presence of the PaaK protein ( Table IV ) , PA-CoA appears to be the true inducer of the paa-encoded pathway .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9748275" target="_blank">9748275</a>]<br></li>
<li>PA-CoA ( red ovals ) produced by PaaK binds to PaaX for derepressing the system .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24983528" target="_blank">24983528</a>]<br></li>
<li>These data , therefore , indicate that PaaX behaves as a transcriptional-repressor of Pz and suggest that the product of the reaction catalyzed by PaaK , i.e. PA-CoA , is the true inducer of this promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10766858" target="_blank">10766858</a>]<br></li>
<li>Figure 1 shows that PaaX represses the expression of the Px : : lacZ fusion and that this effect can be reverted in the presence of both PA and the PA-CoA ligase ( PaaK ) , suggesting that the Px promoter is controlled by the PaaX repressor and that PA-CoA , but not PA , is the inducer molecule .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24983528" target="_blank">24983528</a>]<br></li>
<li>Figure 2B shows that PA-CoA inhibits the binding of PaaX to the Px promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24983528" target="_blank">24983528</a>]<br></li>
<li>Other CoA derivatives such as acetyl-CoA and free CoA were also unable to abolish binding of PaaX to the Pa promoter ( data not shown ) , thus confirming PA-CoA as the specific inducer molecule of the paa catabolic genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10766858" target="_blank">10766858</a>]<br></li>
<li>This was confirmed in vitro through gel retardation assays which revealed that binding of PaaX to the Pa and Pz promoters is specifically prevented by PA-CoA and not by PA ( Fig. 2 ) , benzoyl-CoA , acetyl-CoA , or free CoA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10766858" target="_blank">10766858</a>]<br></li>
<li>Moreover , we have tested by gel retardation assays the ability of PA-CoA to inhibit the binding of PaaX to the Px promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24983528" target="_blank">24983528</a>]<br></li>
<li>We therefore tested by gel retardation assays the ability of PA and PA-CoA to inhibit binding of PaaX to the Pa promoter region in vitro ( Fig. 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10766858" target="_blank">10766858</a>]<br></li>
				</div></ul><br>
			<label class="collapse_DOM" for="_2"><font size="5px">PaaX protein domains</font></label>
				<input id="_2" type="checkbox">
				<div class="DOM"><br><ul><li>( iii ) The hypersensitive sites detected by DNase I footprinting were located centrally , indicating that the right half ought to play some role in PaaX binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15159386" target="_blank">15159386</a>]<br></li>
<li>The DNase I-hyper-sensitive sites observed in ORA and ORZ were spaced at approximately 10 nucleotide intervals , corresponding to about one helix turn ( Figs. 3 and 4 ) , which suggests binding of PaaX to one side of the double helix .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10766858" target="_blank">10766858</a>]<br></li>
<li>The dyad symmetry of the PaaX-binding site and the probability that the repressor-DNA interaction is mediated by a putative helix-turn-helix motif , amino acid residues 39 -- 64 of PaaX ( 2 ) , that shares similarity to that of transcriptional repressors of the GntR-family ( 5 , 6 ) suggest that the regulator is in an oligomeric form when bound to DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10766858" target="_blank">10766858</a>]<br></li>
<li>The PaaX repressor is a 316-amino acid protein with a stretch of 25 amino acids ( residues 39 -- 64 ) that shares signiﬁcant similarity with the helix-turn-helix motif for DNA-recognition and binding of transcriptional regulators from the GntR-family such as FadR ( DiRusso et al. , 1992 ) and GntR ( Haydon and Guest , 1991 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24983528" target="_blank">24983528</a>]<br></li>
<li>The PaaX protein ( 316-amino acid length ) contains a stretch of 25 residues at amino acids 39 -- 64 that shares similarity with the helix-turn-helix motif predicted to be important for DNA rec-ognition and binding in other transcriptional repressors such as GntR ( 71 ) and FadR ( 72 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9748275" target="_blank">9748275</a>]<br></li>
<li>The PaaX repressor is a 316-amino acid protein that shows 41.4 % amino acid sequence identity with its ortholog PhaN ( recently renamed as PaaN ( 4 ) ) from the PA degradation pathway of P. putida U ( 1 ) and contains a stretch of 25 residues at amino acids 39 -- 64 that shares similarity with the helix-turn-helix motif for DNA-recognition and binding of transcriptional repressors from the GntR-family such as FadR ( 5 ) and GntR ( 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10766858" target="_blank">10766858</a>]<br></li>
<li>Characterization of a Conserved PaaX Binding Motif -- Alignment of the PaaX binding sequences identified by DNase I footprinting analyses revealed that each of the operator regions , ORA in promoter Pa and ORZ in promoter Pz , contained a conserved 15-bp imperfect palindromic motif ( Fig. 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10766858" target="_blank">10766858</a>]<br></li>
<li>The two sequence repeats of the PaaX binding motif are related by a ( pseudo ) dyad axis through the central T base at position 8 ( Fig. 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10766858" target="_blank">10766858</a>]<br></li>
<li>Mutagenesis of the PaaX Binding Motif -- Base substitutions in the PaaX binding motif were introduced through recombinant PCR using the plasmid pAAD as template .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10766858" target="_blank">10766858</a>]<br></li>
<li>Base substitution analyses within the PaaX binding motif .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10766858" target="_blank">10766858</a>]<br></li>
<li>Taken together , these data suggest that the conserved motif present in the Pa and Pz promoters is critical for PaaX repressor binding , with the LH region of the motif being much more important for repressor binding than the RH region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10766858" target="_blank">10766858</a>]<br></li>
<li>The PaaX binding motif is boxed , with its LH and RH ( in italics ) regions indicated by convergent arrows .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10766858" target="_blank">10766858</a>]<br></li>
<li>Convergent arrows indicate the LH and RH regions of the PaaX binding motif .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10766858" target="_blank">10766858</a>]<br></li>
				</div></ul><br>
			<label class="collapse_EVO" for="_3"><font size="5px">PaaX evolution</font></label>
				<input id="_3" type="checkbox">
				<div class="EVO"><br><ul><li>It is likely that other regulatory proteins showing a significant amino acid sequence similarity with PaaX , e.g. the equivalent PaaN repressor ( 41.4 % identity ) of the PA degradation pathway from P. putida U ( 1 ) , the putative regulator , ORF13 ( 31.5 % identity ) , of a predicted PA catabolic pathway from Bacillus halodurans ( GenBankTM accession number AB011837 ) , and the putative repressor ( GenBankTM accession number AF042490 ) ( 24.1 % identity ) of the 4-chloro-benzoate dehalogenation pathway involving formation of CoA derivatives in Arthrobacter sp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10766858" target="_blank">10766858</a>]<br></li>
<li>The dyad symmetry of the PaaX-binding site and the probability that the repressor-DNA interaction is mediated by a putative helix-turn-helix motif , amino acid residues 39 -- 64 of PaaX ( 2 ) , that shares similarity to that of transcriptional repressors of the GntR family ( 5 , 6 ) suggest that the regulator is in an oligomeric form when bound to DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10766858" target="_blank">10766858</a>]<br></li>
<li>The PaaX repressor is a 316-amino acid protein with a stretch of 25 amino acids ( residues 39 -- 64 ) that shares signiﬁcant similarity with the helix-turn-helix motif for DNA recognition and binding of transcriptional regulators from the GntR family such as FadR ( DiRusso et al. , 1992 ) and GntR ( Haydon and Guest , 1991 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24983528" target="_blank">24983528</a>]<br></li>
<li>The PaaX protein ( 316-amino acid length ) contains a stretch of 25 residues at amino acids 39 -- 64 that shares similarity with the helix-turn-helix motif predicted to be important for DNA rec-ognition and binding in other transcriptional repressors such as GntR ( 71 ) and FadR ( 72 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9748275" target="_blank">9748275</a>]<br></li>
<li>The PaaX repressor is a 316-amino acid protein that shows 41.4 % amino acid sequence identity with its ortholog PhaN ( recently renamed as PaaN ( 4 ) ) from the PA degradation pathway of P. putida U ( 1 ) and contains a stretch of 25 residues at amino acids 39 -- 64 that shares similarity with the helix-turn-helix motif for DNA recognition and binding of transcriptional repressors from the GntR family such as FadR ( 5 ) and GntR ( 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10766858" target="_blank">10766858</a>]<br></li>
<li>Whereas PaaX is a member of the GntR family of regulators , PaaR is usually a TetR-like repressor ( Hamlin et al. , 2009 ) , although a PaaR regulator of the AraC family has been also proposed in Rhodococcus jostii RHA1 ( Navarro-Llorens et al. , 2005 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24983528" target="_blank">24983528</a>]<br></li>
<li>Whereas the LH region of the PaaX binding motif is rather conserved in the operator sequences of other bacterial regulators such as GntR ( 6 ) and PdhR ( 27 ) ( which belong to the GntR family ) and , surprisingly , CRP ( 23 , 25 ) and LacI ( 26 ) ( which belong to the GalR-LacI family ) , the RH region differs from that of other operators ( Fig. 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10766858" target="_blank">10766858</a>]<br></li>
				</div></ul><br>
			<label class="collapse_RP" for="_4"><font size="5px">Processes regulated by PaaX</font></label>
				<input id="_4" type="checkbox">
				<div class="RP"><br><ul><li>Other identiﬁed genes include those encoding DNA helicases ( UvrD , RecG , RecQ ) , truncated forms of the DNA mismatch-repair protein ( MutS ) and a primosomal component ( DnaT ) , a negative modulator of initiation of replication/GATC-binding protein ( SeqA ) , a stationary phase regulator AppY , a transcriptional regulator PaaX and three putative open reading frames , ycgW , yfjY and yjiD , encoding hypothetical proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15225322" target="_blank">15225322</a>]<br></li>
<li>SeqA ( region no. 3 ) encodes a negative modulator of initiation of replication and a GATC-binding protein ( Hiraga et al. , 1998 ; Shakibai et al. , 1998 ; Guarne et al. , 2002 ) ; b1399 ( region no. 6 ) encodes a transcriptional regulator PaaX for phenylacetic acid degradation ( Ferrandez et al. , 2000 ) ; and yfjY ( region no. 8 ) encodes a hypothetical protein ( Blattner et al. , 1997 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15225322" target="_blank">15225322</a>]<br></li>
<li>Antagonist-CoA ( Ant-CoA ; green ovals ) represents a CoA-intermediate generated by the paa pathway , e.g. 2-hydroxycyclohept-1 ,4,6 - triene-1-carboxy-CoA , which blocks the catabolic pathway ( orange hammer ) and acts as an anti-repressor on PaaX ( blue arrow ) avoiding the induction of the paa genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24983528" target="_blank">24983528</a>]<br></li>
<li>Kim , H.S. , Kang , T.S. , Hyun , J.S. , and Kang , H.S. ( 2004 ) Regulation of penicillin G acylase gene-expression in Escherichia coli by repressor PaaX and the cAMP-cAMP receptor protein complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24983528" target="_blank">24983528</a>]<br></li>
<li>PaaX and PaaY may be involved in regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12846838" target="_blank">12846838</a>]<br></li>
<li>It is worth noting that the FadR transcriptional repressor , which is regulated by acyl-CoA compounds and shows local similarity to PaaX , is also controlling the expression of genes involved in b-oxidation mechanisms ( 73 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9748275" target="_blank">9748275</a>]<br></li>
<li>First , we have shown that PaaX is a transcriptional repressor of pga expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15159386" target="_blank">15159386</a>]<br></li>
<li>First , we have demonstrated that PaaX represses pga transcription , and this accounts both for negative regulation and PAA induction of pga expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15159386" target="_blank">15159386</a>]<br></li>
				</div></ul><br>
			<label class="collapse_SIT" for="_5"><font size="5px">PaaX binding site</font></label>
				<input id="_5" type="checkbox">
				<div class="SIT"><br><ul><li>To generate mutations in the left-half site of the PaaX binding motif , a first PCR reaction was performed using two pairs of primers as follows : PA5 -- 1 ( 59-CAATCTCGGAATGCG-CATG-39 , which encodes amino acids 38 -- 44 of PaaA and hybridizes with the non-coding strand of the paaA gene between nucleotides 2885 and 2867 of the paa gene cluster ( 2 ) , underlined are the bases that reconstruct a BbuI restriction site ) and the degenerated MUT-PA5 ( 59-CTTTCATAAAACAATGWKMTTCGTGTTTTTAATT-39 , which hybridizes between nucleotides 2688 and 2721 enclosing the transcription start sites of Pa ( 2 ) , see also Fig. 4 , where K is G or T , M is A or C , and W is A or T ) ; and PAP ( 59-GGTCTAGAGTTATCAAAATAGAGTGCG-39 , located between the Pz and Pa promoters , nucleotides 2611 -- 2631 of the paa gene cluster ( 2 ) ( see Fig. 4 ) , where the engineered XbaI site is underlined ) and the degenerated MUT-PA3 ( 59-AATTAAAAACAC-GAAKMWCATTGTTTTATGAAAG-39 , which is complementary to the MUT-PA5 oligonucleotide ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10766858" target="_blank">10766858</a>]<br></li>
<li>Similarly , to generate mutations in the right-half site of the PaaX binding motif , a first PCR reaction was performed using two pairs of primers : PA5 -- 1 and the degenerated MUT-PA52 ( 59-CATAAAA-CAATGTGATTCGWSWWTTTAATTAATTCACG-39 , which hybridizes between nucleotides 2692 and 2729 enclosing the transcription start sites of Pa ( 2 ) , see also Fig. 4 ; where S means C or G ) ; and PAP and the degenerated MUT-PA32 ( 59-CGTGAATTAATTAAAWWSWCGAATCA-CATTGTTTTATG-39 , which is complementary to the MUT-PA52 oligo-nucleotide ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10766858" target="_blank">10766858</a>]<br></li>
<li>His-PaaX indeed protected only one segment of the pga upstream region , of about 50 bases in length , from 126 to 77 , a very unusual upstream binding site for a bacterial repressor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15159386" target="_blank">15159386</a>]<br></li>
<li>In addition , in view of the fact that the PaaX operator ( from 126 to 77 ) completely overlaps the weaker CRP-binding site ( site I from 119 to 100 ) , we would anticipate that competition between PaaX and cAMP-CRP for binding to the pga upstream region would influence pga expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15159386" target="_blank">15159386</a>]<br></li>
<li>This suggests that PaaX binds to DNA in pairwise fashion , in the form of dimers or tetramers , etc. , leaving an unprotected central region exposed to attack by DNase I. Identification of a New PaaX-binding Consensus Sequence -- To determine the precise sequence to which PaaX binds we constructed six plasmids each with a single base substitution in the presumed operator region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15159386" target="_blank">15159386</a>]<br></li>
<li>Because a single base substitution in Fig. 2 , boxed regions A and B , is enough to abolish operator activity , it seems reasonable to suppose that PaaX binds in a pairwise manner , with one molecule recognizing region A and the other region B .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15159386" target="_blank">15159386</a>]<br></li>
<li>These findings therefore permit us to identify a new consensus sequence for PaaX ( Fig. 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15159386" target="_blank">15159386</a>]<br></li>
<li>We believe that we have also identified the consensus sequence recognized by PaaX .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15159386" target="_blank">15159386</a>]<br></li>
<li>A consensus palindromic sequence ( TGATTC - ( N26 -- 28 ) - GAATCA/G ) has been shown to be the PaaX operator in E. coli ( Ferrández et al. , 2000 ; Kim et al. , 2004 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24983528" target="_blank">24983528</a>]<br></li>
<li>The in silico analysis of the Px promoter region revealed a potential PaaX binding site ( TGATTC - ( N26 ) - GAATCG ) between positions − 9 and + 29 with respect to the transcriptional start site described for the paaX gene ( Ferrández et al. , 1998 ) ( Fig .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24983528" target="_blank">24983528</a>]<br></li>
<li>Since sequences with dyad symmetry within the operator regions are usually the binding sites of bacterial transcriptional regulators ( 21 ) , we predicted that the described conserved motifs might be the binding sites of PaaX within ORA and ORZ .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10766858" target="_blank">10766858</a>]<br></li>
<li>Whereas the LH region of the PaaX binding motif is rather conserved in the operator sequences of other bacterial regulators such as GntR ( 6 ) and PdhR ( 27 ) ( which belong to the GntR family ) and , surprisingly , CRP ( 23 , 25 ) and LacI ( 26 ) ( which belong to the GalR-LacI family ) , the RH region differs from that of other operators ( Fig. 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10766858" target="_blank">10766858</a>]<br></li>
<li>Alignment was done with the nucleotide sequences of the binding sites for the PaaX repressor in Pa ( PaaX ( Pa ) ) and Pz ( PaaX ( Pz ) ) promoters , the CRP activator in lac promoter ( CRP - ( Plac ) ) ( 25 ) , the LacI repressor ( 26 ) , the GntR repressor ( 6 ) , and the PdhR repressor ( 27 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10766858" target="_blank">10766858</a>]<br></li>
<li>Nucleotides of the RH region are shown in italics , and numbers refer to the position of the nucleotides within the 15-bp PaaX binding motif .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10766858" target="_blank">10766858</a>]<br></li>
<li>Sequence comparison analysis between ORA and ORZ revealed a 15-bp consensus motif , WWTRTGATTCGYGWT , located at the 59-end of the non-coding strands ( Fig. 4 ) , which was shown by point mutation analyses to be indispensable for binding of PaaX to each promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10766858" target="_blank">10766858</a>]<br></li>
<li>The region within the Pa and Pz promoters that is protected by the PaaX repressor in DNase I footprinting assays contains a conserved 15-base pair imperfect palindromic sequence motif that was shown , through mutational analysis , to be indispensable for PaaX binding and repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10766858" target="_blank">10766858</a>]<br></li>
<li>Characterization of a Conserved PaaX Binding Motif -- Alignment of the PaaX binding sequences identified by DNase I footprinting analyses revealed that each of the operator regions , ORA in promoter Pa and ORZ in promoter Pz , contained a conserved 15-bp imperfect palindromic motif ( Fig. 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10766858" target="_blank">10766858</a>]<br></li>
<li>The putative PaaX operator shows a palindromic sequence that is in good agreement with those of other PaaX operator sequences previously reported ( Fig .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24983528" target="_blank">24983528</a>]<br></li>
<li>The two sequence repeats of the PaaX binding motif are related by a ( pseudo ) dyad axis through the central T base at position 8 ( Fig. 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10766858" target="_blank">10766858</a>]<br></li>
				</ul></div><br>
			<label class="collapse_TU" for="_6"><font size="5px">Transcription units regulated by PaaX</font></label>
				<input id="_6" type="checkbox">
				<div class="TU"><br><ul><li>Under normal conditions , PaaX , the repressor of the paa operon encoded by paaX , binds to the divergent Pa and Pz promoters to prevent transcription of the meta-cleavage pathway gene cluster .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15159386" target="_blank">15159386</a>]<br></li>
<li>These observations suggest that PaaX might control the transcription of the paaXY operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24983528" target="_blank">24983528</a>]<br></li>
<li>In this work , using different in vivo and in vitro approaches , we have demonstrated that in E. coli , the paaX and paaY genes form a true operon that is autoregulated by PaaX in a PA-CoA dependent manner .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24983528" target="_blank">24983528</a>]<br></li>
<li>The P a a Results and discussion The Px promoter drives the expression of the paaXY operon and is controlled by PaaX in a PA-CoA dependent manner Using different experimental approaches , e.g. real-time polymerase chain reaction ( RT-PCR ) analysis of the co-expression of paaX and paaY genes and Western blot analysis of PaaY content in the E. coli wild-type and paaXY mutant strains , we have demonstrated that paaX and paaY genes constitute a single operon ( Fig .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24983528" target="_blank">24983528</a>]<br></li>
<li>First , we have demonstrated that PaaX represses pga transcription , and this accounts both for negative regulation and PAA induction of pga expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15159386" target="_blank">15159386</a>]<br></li>
<li>To check whether PaaX regulates in vivo its own expression , we used the E. coli AF141X strain that harbours a Px : : lacZ translational fusion inserted into its chromosome ( Table S1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24983528" target="_blank">24983528</a>]<br></li>
<li>To check whether the expression of the divergently transcribed Pz promoter , which drives the expression of the paaZ catabolic gene , was also regulated by PaaX , we have engineered a reporter Pz-lacZ fusion within a mini-Tn5 vector .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10766858" target="_blank">10766858</a>]<br></li>
<li>RESULTS The PaaX Repressor Regulates the Pz Promoter -- We have previously shown that the Pa promoter , which drives the expression of the paaABCDEFGHIJK catabolic operon , is repressed by the PaaX regulator ( 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10766858" target="_blank">10766858</a>]<br></li>
<li>However , when PAA is present , PaaK , the PAA-CoA ligase encoded by paaK , converts PAA to PAA-CoA and this inhibits PaaX from binding to the promoter and initiates expression from the Pa and Pz promoters ( 12 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15159386" target="_blank">15159386</a>]<br></li>
<li>Figure 1 shows that PaaX represses the expression of the Px : : lacZ fusion and that this effect can be reverted in the presence of both PA and the PA-CoA ligase ( PaaK ) , suggesting that the Px promoter is controlled by the PaaX repressor and that PA-CoA , but not PA , is the inducer molecule .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24983528" target="_blank">24983528</a>]<br></li>
				</div></ul></div><br>
	</div>
</body>
</html>