<!doctype html>
<style>
@charset "utf-8";
body {
	height: 100%;
	margin: 0;
	padding: 0;
}

.banner {
	position: fixed;
	top: 0;
	width: 100%;
	height: 65px;
	background: gray;
	line-height: 65px;
}
.spacer {
	height: 10px;
}

.main-content {
	padding: 5%;
}

.collapse_ACT {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-style: none solid none none;
	border-color: #B4045F;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_ACT + input{
	display: none;
}
.collapse_ACT + input + div{
	display: none;
}
.collapse_ACT + input:checked + div{
	display: block;
}
.ACT {
	text-align: justify;
	text-justify: inter-word;
}
.ACT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B4045F;
}
.ACT_collapse_similarity:hover{
	background: #e5d3dc;
}
.ACT_collapse_similarity + input {
	display: none;
}
.ACT_collapse_similarity + input + div{
	display: none;
}

.ACT_collapse_similarity + input:checked + div{
	display: block;
}
.ACT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.ACT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B4045F;
	padding-left: 5px;
}
.ACT_collapse_similarity:hover{
	background: #e5d3dc;
}
.ACT_collapse_similarity + input {
	display: none;
}
.ACT_collapse_similarity + input + div{
	display: none;
}
.ACT_collapse_similarity + input:checked + div{
	display: block;
}
.ACT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.ACT_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #B4045F;
}
.ACT_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_DOM {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-style: none solid none none;
	border-color: #B404AE;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_DOM + input{
	display: none;
}
.collapse_DOM + input + div{
	display: none;
}
.collapse_DOM + input:checked + div{
	display: block;
}
.DOM {
	text-align: justify;
	text-justify: inter-word;
}
.DOM_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B404AE;
	padding-left: 5px;
}
.DOM_collapse_similarity:hover{
	background: #e8d7e7;
}
.DOM_collapse_similarity + input {
	display: none;
}
.DOM_collapse_similarity + input + div{
	display: none;
}
.DOM_collapse_similarity + input:checked + div{
	display: block;
}
.DOM_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}.DOM_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #B404AE;
}
.DOM_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_EVO {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #7401DF;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_EVO + input{
	display: none;
}
.collapse_EVO + input + div{
	display: none;
}
.collapse_EVO + input:checked + div{
	display: block;
}
.EVO {
	text-align: justify;
	text-justify: inter-word;
}
.EVO_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #7401DF;
	padding-left: 5px;
}
.EVO_collapse_similarity:hover{
	background: #e2d7ed;
}
.EVO_collapse_similarity + input {
	display: none;
}
.EVO_collapse_similarity + input + div{
	display: none;
}
.EVO_collapse_similarity + input:checked + div{
	display: block;
}
.EVO_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.EVO_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #7401DF;
}
.EVO_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_RP {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #0174DF;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_RP + input{
	display: none;
}
.collapse_RP + input + div{
	display: none;
}
.collapse_RP + input:checked + div{
	display: block;
}
.RP {
	text-align: justify;
	text-justify: inter-word;
}
.RP_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #0174DF;
	padding-left: 5px;
}
.RP_collapse_similarity:hover{
	background: #c9d7e2;
}
.RP_collapse_similarity + input {
	display: none;
}
.RP_collapse_similarity + input + div{
	display: none;
}
.RP_collapse_similarity + input:checked + div{
	display: block;
}
.RP_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.RP_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #0174DF;
}
.RP_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_SIT {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #0489B1;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_SIT + input{
	display: none;
}
.collapse_SIT + input + div{
	display: none;
}
.collapse_SIT + input:checked + div{
	display: block;
}
.SIT {
	text-align: justify;
	text-justify: inter-word;
}
.SIT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #0489B1;
	padding-left: 5px;
}
.SIT_collapse_similarity:hover{
	background: #d0dfe2;
}
.SIT_collapse_similarity + input {
	display: none;
}
.SIT_collapse_similarity + input + div{
	display: none;
}
.SIT_collapse_similarity + input:checked + div{
	display: block;
}
.SIT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.SIT_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #0489B1;
}
.SIT_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_TU {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #04B486;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_TU + input{
	display: none;
}
.collapse_TU + input + div{
	display: none;
}
.collapse_TU + input:checked + div{
	display: block;
}
.TU {
	text-align: justify;
	text-justify: inter-word;
}
.TU_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #04B486;
	padding-left: 5px;
}
.TU_collapse_similarity:hover{
	background: #d0e2de;
}
.TU_collapse_similarity + input {
	display: none;
}
.TU_collapse_similarity + input + div{
	display: none;
}
.TU_collapse_similarity + input:checked + div{
	display: block;
}
.TU_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.TU_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #04B486;
}
.TU_similar > a {
	text-decoration: none;
	color: gray;
}

ul { 
	list-style-type: none;
	padding: 0;
	margin: 0;
}
ul li > a {
	text-decoration: none;
	color: gray;
}
ul li:hover, ul li:focus {
	background: #F0F0F0;
}

ul li:first-child span {
  border-left:none;  
}

</style>
<html>
<head>
<meta charset="utf-8">
<title>TF properties - PhoB</title>
<meta name="theme-color" content="#ffffff">
</head>
<body>
	<div class="banner">
		<center><font color="white" size="16px" face="arial">PhoB</font></center>
	</div>
	<div align="center" class="main-content">
		<div class="spacer">
	</div><br>			<label class="collapse_ACT" for="_1"><font size="5px">PhoB transcriptional activity</font></label>
				<input id="_1" type="checkbox">
				<div class="ACT"><br><ul><li>After prolonged incubation , the level of phosphorylated RRs increased up to completion for pBasR , pHydG , pOmpR , pPhoB , and pUvrY ( group B1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15522865" target="_blank">15522865</a>]<br></li>
<li>Most NtrC-like regulators function in pairs ( PhoRPhoB , EnvZ-OmpR , CpxA-SfrA , VirA-VirG , DctB-DctD , CheA-CheB , CheY ) , including NifA , which is modified by the NifL product .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2666400" target="_blank">2666400</a>]<br></li>
<li>Crp , GalS MglAB Crp , RbsR RbsAB Crp , GalS MglA Crp , GntR GntK Crp , EbgR EbgA Crp , LacI LacZ Crp , RhaS RhaBD Crp , AraC AraBD Crp , FucR FucK Crp , YiaJ LyxK Crp , FucR FucA Crp , YiaJ SgbE RcsB WcaB CpxR LpxA RcsB WcaB CpxR LpxD RcsB B2060 PhoB PhnCKLN RcsB B2060 PhoB PstB RcsB B2060 PhoB PhoH MhpR MhpF Fur EntA FecI , Fur FecI UidR , UxuR UidR Crp , FucR FucR Crp , Fur Fur Homologous pairs of transcription factors with autoregulation IdnR PurR AsnC Lrp Fnr Crp Homologous triplets of transcription factors with autoregulation TorR PhoB CpxR EmrR PdhR ExuR Eight homologous transcription factors with autoregulation DsdC GcvA CynR Hcar IlvY LysR OxyR CysB Figure 6 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12582243" target="_blank">12582243</a>]<br></li>
<li>aerobic to anaerobic anaerobic to aerobic ArgR CysB BirA DgsA MalT GadW MetJ NikR OxyR SoxR SoxS DcuR CueR FadR HcaR NtrC PhoB PurR SgrR TdcA TrpR ZraR ArcA CpxR CRP CusR FhlA Fis FNR Fur H-NS IHF IscR Lrp ModE NarL NrdR PdhR FlhDC GlcC EvgA GadX LeuO PspF RbsR steady states Figure 3 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22870390" target="_blank">22870390</a>]<br></li>
<li>sitesa % CGb Count % Non-codinge Ada 3 38.7 AraC 6 40.3 2 3 100 100 47 76 100 28 33 32 73 38 100 52 100 86 79 89 100 67 26 100 28 100 70 37 67 28 100 82 100 47 50 51 100 100 38 33 64 3 6 100 100 38 54 100 23 40 18 43 29 100 38 100 39 27 53 58 71 19 100 23 33 47 100 40 65 21 23 60 25 100 41 100 26 43 36 100 35 35 30 64 36 23 22 26 14 100 83 28 ArcA 12 20.0 ArgR 17 22.9 CarP 2 46.0 Crp 49 37.0 CspA 4 41.3 CynR 2 33.3 CysB 3 34.2 CytR 5 37.8 DeoR 3 22.9 DnaA 8 36.7 FadR 7 47.0 FarR 4 20.0 FhlA 3 42.0 Fis 19 38.2 FlhCD 3 53.8 Fnr 9 25.8 FruR 12 42.7 Fur 9 23.4 GalR 7 43.8 GcvA 4 22.5 GlpR 13 39.6 HipB 4 40.9 Hns 15 31.5 IclR 2 26.7 Ihf 26 32.6 IlvY 2 32.4 LacI 3 44.4 LexA 18 33.9 Lrp 18 28.9 MalT 10 61.0 MarR 2 37.5 MelR 2 33.3 MetJ 15 40.1 MetR 8 33.3 NagC 6 22.5 NarL 9 34.6 NtrC 6 51.9 OmpR 9 26.7 OxyR 4 42.3 PdhR 3 33.3 PhoB 15 29.7 PurR 23 45.3 RhaS 2 50.0 RpoD15 27 37.4 RpoD16 48 34.9 RpoD17 116 37.3 RpoD18 34 38.0 RpoD19 25 38.2 RpoE 3 38.9 RpoH13 8 42.1 RpoH14 7 44.3 RpoN 7 40.6 RpoS17 15 43.0 RpoS18 7 45.2 SoxS 14 40.8 TorR 4 37.5 TrpR 4 38.5 Tus 6 26.8 TyrR 17 33.2 a The number of input sites used to construct the matrix .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9813115" target="_blank">9813115</a>]<br></li>
<li>of additional probable sites in predictionsd 3 3 7e 7 -- 4 22e 0 1 0 2 -- -- 5 0 1 -- 13 7 18 1 0 6e 1 1 1 3 Ada AraC ArcA ArgR CarP CpxR Crp CspA CynR CysB CytR DeoR DnaA FadR FhlA Fis FlhCD Fnr FruR Fur GalR GcvA GlpR IciA ( ArgP ) IclR Ihf IlvY LacI LexA Lrp MalT MarR MelR MetJ MetR Mlc ( DgsA ) ModE NagC NarL NarP NtrC OmpR OxyR PdhR PhoB PurR RhaR RhaS SoxR SoxS TorR TrpR TyrR Stem -- loopsf 6 2 0 5 2 9 1 3 4 5e 5 -- 5 5 4 6 3 4 2 -- 1 3 5 1 3 -- aThe number of genes in our study set with documented binding sites for each TF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11160901" target="_blank">11160901</a>]<br></li>
<li>1.3 1.9 5.0 2.4 4.7 2.7 2.6 1.4 PspF , IHF PhoB RbsR , Crp 37.9 22.0 2.6 1.2 1.8 2.3 0.8 1.1 Inner membrane Unknown Ribose transport Ribose transport Ribose transport Ribose transport Ribose transport Ribose transport pspG ( yjbO ) ytfK rbsD rbsA rbsC rbsB rbsK rbsR 0.7 3.8 1.8 1.0 1.6 1.4 1.2 0.8 0.9 3.0 8.2 5.4 21.3 6.6 9.0 2.8 manX Mannose permease 4.2 2.1 1.1 5.0 manY Mannose permease 3.4 2.6 1.1 5.2 manZ Mannose permease 6.5 5.7 1.0 16.7 NagC , DgsA , Crp a. Genes and operons which are induced 10-fold in three independent experiments are listed with the average fold induction for the three experiments .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18710431" target="_blank">18710431</a>]<br></li>
<li>23.5 20.0 3.8 16.3 9.1 12.7 Maltose transport MalT , Crp malK lamB malM ibpB ibpA htpG Maltose transport MalT , Crp 18.6 5.3 3.3 39.0 24.4 11.7 32.7 13.3 9.1 24.2 20.2 8.2 2.6 2.1 1.7 4.6 4.1 2.0 48.6 19.2 20.0 52.6 35.9 16.6 s32 Heat shock Heat shock s32 Heat shock chaperone s32 Protease ; chaperone Regulator Regulator ; chaperone Protease Periplasmic clpB 11.5 12.9 17.3 21.0 20.4 14.3 3.3 3.2 1.3 1.8 2.5 2.9 8.0 2.7 2.9 24.5 6.8 5.7 1.7 1.3 yceP/bssS cpxP ( b3913 ) degP ( htrA ) spy CpxR CpxR , H-NS CpxR , BaeSR CpxR CpxR Unknown Unknown ydeH yebE 49.2 44.8 63.1 21.0 30.7 32.2 56.9 33.1 9.5 1.1 1.6 6.1 7.3 7.3 2.6 1.5 1.6 2.0 1.4 1.2 1.3 2.9 Unknown ; signal peptide Putative transporter Regulator ; phage shock protein Regulator ; phage shock protein Regulator ; phage shock protein Inner membrane Sulphurtransferase yncJ ( b1436 ) yhjX pspA pspB pspC pspD pspE 1.6 44.1 1.1 1.6 1.7 1.2 1.2 3.1 165.6 1.8 1.5 2.1 1.2 1.5 PspF , IHF 1.3 1.9 5.0 2.4 4.7 2.7 2.6 1.4 PspF , IHF PhoB RbsR , Crp 37.9 22.0 2.6 1.2 1.8 2.3 0.8 1.1 Inner membrane Unknown Ribose transport Ribose transport Ribose transport Ribose transport Ribose transport Ribose transport pspG ( yjbO ) ytfK rbsD rbsA rbsC rbsB rbsK rbsR 0.7 3.8 1.8 1.0 1.6 1.4 1.2 0.8 0.9 3.0 8.2 5.4 21.3 6.6 9.0 2.8 manX Mannose permease 4.2 2.1 1.1 5.0 manY Mannose permease 3.4 2.6 1.1 5.2 manZ Mannose permease 6.5 5.7 1.0 16.7 NagC , DgsA , Crp a. Genes and operons which are induced 10-fold in three independent experiments are listed with the average fold induction for the three experiments .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18710431" target="_blank">18710431</a>]<br></li>
<li>Residues that are conserved in all four sequences are marked with asterisks 57 58 59 60 KdpE PhoB CreB OmpR * * * * ** * * ***** *** * ** GDGIEFIRDLRQ WSA VPVIVLSARSEESDKIAALDAGADDYLSKPFGIGELQARLRV GSGIQFIKHLKRESMTRDIPVVMLTARGEEEDRVRGLETGADDYITKPFSPKELVARIKA ISGFELCRQLLA LHPALPVLFLTARSEEVDRLLGLEIGADDYVAKPFSPREVCARVRT EDGLSICRRLRS QSNPMPIIMVTAKGEEVDRIVGLEIGADDYIPKPFNPRELLARIRA KdpE PhoB CreB OmpR 114 118 117 118 ** ** * ALRRHSATTAPDP LVKFSDVTVDLAARVIHRGEEEVHLTPIEFRLAGRCSTMPEK VMRRISP MAVEE VIEMQGLSLDPTSHRVMAGEEPLEMGPTEFKLLHFFMTHPER LLRRVKKFSTPSP VIRIGHFELNEPAAQISWFDTPLALTRYEFLLLKTLLKSPGR VLRRQANELPGAPSQEEAVIAFGKFKLNLGTREMFREDEPMPLTSGEFAVLKALVSHPRE KdpE PhoB CreB OmpR KdpE YSPSGPVLNQVWGPNAVEHSHYLRIYMGHLRQ 201 PhoB VYSREQLLNHVWGTNVYVEDRTVDVHIRRLRK 205 CreB VWSRQQLMDSVWEDAQDTYDRTVDTHIKTLRA 204 OmpR PLSRDKLMNLARGREYSAMERSIDVQISRLRR 210 FIG. 6 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1532388" target="_blank">1532388</a>]<br></li>
<li>pvirighFeLnepaaqisw fDtp.LaLTryEFlLLktL1 kspgrVwSRq hevqqlgeLi Fhdegy.fll qgqp.LaLTp rEqaLLtvb yrrtrpvSRq dplvkfsdvtvdlaar.vih rgeeavhLTp iEFrLagrcstmpekyspsg ~ PR ArcA VirG PhoP Va `` R PhoB CreB TctD KdpE and Silverman , with the second viously published by Weber and Silverman ( Weber 1988 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8294007" target="_blank">8294007</a>]<br></li>
<li>PG-V-SR -- LL 60 70 80 90 i00 SRV RDEEVYDRS IDVLILRLRR KLEADPSSPQLIKTARGAGYFF SRV REEEVYDRS IDVLIFRLRR KLEGDPTSPQLIKTARGAGYFF SRV REEEVYDRS IDVLI LRLRR KLEGDPTTPQLIKTARGAGYFF VTG RELKPHDRTVDVTIRRIRK HFESTPDTPEIIATIHGEGYRF '' ~ 2 , ~ GY TPERHL DTRVVDVHISRLRA KLEDDPSNPELILTSRGTGYLF \ q , ~ GY TAERQV DTRVVDVHISRLRA KLEDDPSNPDLILTARGTGYLF LQLY PDAELRESHTIDVLMGRLRK KIQAQYPH DVITTVRGQGYLF '' , ~ G PNAVEHSHYLRIYMGHLRQ KLEQDPARPRHFITETGIGYRF 2 , GR EYSAME RSIDVQISRLRR MVEEDPAHPRYIQTVWGLGYVF , ~ A2RSASAEEAAGDRAVDVQ ITRLRR KIEPDPREPRYLQTVRGLGYML '' , ' > ~ G TNVYVEDRTVDVHIRRLRK ALEPG GHDRMVQTVRGTGYRF \ ~ ' , ; G GNVYVEERTVDMD IRRLRK ALGE VYENLVQTVRGTGYRF V > ~ E DAQDTYDRTVDTHIKTLRA KLRAINPDLSPINTHRGMGYSL VS 'N YDFAGDTRIVDVHISHLRPTKIENNTKKPIYIKTIRGLGYKL VLG KRLTPFDRAIDMHISNLRR KLPDRKDGHPWFKTLRGRGYLM AWD ENTDPFTNVVRVTVMTLRR KLGE PPVIVTVPGSGYRI EQGFEVDDSSLTQAISTLRK MLKDSTKSPQYVKTVPKRGYQL VirG Atu6 VirG Arh VirG Atu5 ArcA Eco RRHM CYca RRHM POae PhoP Sty KdpE Eco OmpR Eco PetR Rca PhoB Eco PhoB Pae CreB Eco PhoB Bsu YjjE Eco CutR Sli ToxR Vch 119 Fig. 5 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7699720" target="_blank">7699720</a>]<br></li>
<li>AGRT6 VIRG AGRRA YIIA ECOLI TOXR VIBCH + RRHM Pae Plastid response-regulator OmpR Eco Porin bioynthesis , osmolarity PhoB Bsu Alkaline phosphatase regulator CreB Eco Catabolite repression ArcA Eco Aerobic respiratory control PetR Rca Cytochrome bc1 complex CutR Sli Copper metabolism KdpE Eco K + transport PhoB Eco Phosphate regulon PhoB Pae Phosphate regulon PhoP Sty Virulence TctD Sty Tricarboxylate transport VirG Atu6 Virulence VirG Atu5 Virulence VirG Arh Virulence YjjE Eco Unknown ToxR Vch Virulence cholera toxin Class 3 : DNA binding domains of family 3 FixJ Rme Nitrogen fixation FixJ Bja Nitrogen fixation NodW Bja Nodulation BvgA Bpe Virulence FimZ Eco Fimbrial synthesis DegU Bsu Extracellular degradative enzymes NarL Eco Nitrate reductase & formate dehydrogenase TrpO Pae Unknown YecB Eco Unknown RcsB Eco Colanic acid capsule biosynthesis B AgmR Fae Glycerol metabolism UhpA Eco Uptake of hexose phosphates Coma Bsu Competence GerE Bsu SigK txn .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7699720" target="_blank">7699720</a>]<br></li>
<li>Proteins in classes 4 , 5 and 6 are exceptions 7O ENAF I0 20 30 40 50 60 NtrC NtrC NifRl NtrC PgtA DctD FixJ NodW CheY SpoOF RRHM RRHM OmpR PhoB CreB ArcA PetR VirG CutR KdpE PhoB PhoP TctD AIgB YLB3 RcsC BvgS HydG CheY FrzE BarA TrpO HvgA FimZ DegU NarL AIgRI YecB UhpA RcsB Agm & CheB CheB FrzG Spo0A MrkE AgrA Coma F!bD Bpa Rme Rca Eco Sty Rle Rme Bja Bsu Bsu Cca Pae Eco Bsu ECO Eco Rca Atu Sli Eco Eco Sty Sty Pae Lin Eco Bpe Eco Eco M ~ a Eco Pae Bpe Eco Bsu Eco Pae Eco Eco Eco Pae Bsu Eco Mxa Bsu Kpn Sau Bsu Ccr 6 ILVADDDTAIKTVLNQA 6 ILVADDD ~ IRTVLNQA 5 VLVADDDRTIRTVLTQA 6 V % ~ VDDDSSIR ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7699720" target="_blank">7699720</a>]<br></li>
<li>) Major clusters 1 , 2 , and 3 are associated primarily or exclusively with the three major families of DNA binding NtrC Bpa NtrC Rme NifRl Rca Nt rC Eco PgtA Sty DetD Rle FixJ Kme NodW Bja CheY Bsu SpoOF BSu RRHM Cca RRHM Pae OmpR Eco PhoB Bsu CreB Eco ArcA Eco PetR Rca VirG Atu CutR Sli KdpE ECO PhoB Eco PhoP Sty TctD Sty A!gB Pae YLB3 Lin RcsC Eco BvgS Bpe HydG Eco CheY Eco FrzE Nc .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7699720" target="_blank">7699720</a>]<br></li>
<li>Its preference at the P1 position of the peptide substrate is Glu Leu Ala His Val Phe Ile Gly Lys Thr ; induced by heat-shock Involved in essential step in gluconeogenesis when pyruvate and lactate are used as a carbon source PpiD P77241 179 Phosphoenolpyruvate synthase 4.93 / 87,303.88 4.94 / 86,407 P23538 PpsA 212 , 228 , 294 Probable zinc protease Tail-speciﬁc protease P31828 P23865 PqqL Prc 5.95 / 104,656.47 6.04 / 74,323.25 179 179 Involved in the cleavage of a C-terminal peptide of 11 residues from the precursor form of penicillin-binding protein 3 ; may be involved in protection of the bacterium from thermal and osmotic stresses Directs the termination of translation in response to the peptide chain termination codons UGA and UAA Increases the formation of ribosomal termination complexes and stimulates activities of RF-1 and RF-2 ; binds guanine nucleotides and has strong preference for UGA stop codons ; may interact directly with the ribosome ; the stimulation of RF-1 and RF-2 is signiﬁcantly reduced by GTP and GDP but not by GMP Catalyzes the NADPH dependent reduction of L-gamma-glutamyl 5-phosphate into L-glutamate 5-semialdehyde and phosphate in proline biosynthesis ; spontaneously undergoes cyclization to form 1-pyrroline-5-carboxylate ; Involved in proline biosynthesis 4.64 / 41,250.73 Peptide chain release factor 2 ( RF-2 ) P07012 PrfB 1 5.65 / 59,442.89 P0A7I4 Peptide chain RF-3 PrfC 179 ProA 5.42 / 44,630.05 5.39 / 51,249 ( 5 -- 6 ) Gamma-glutamyl phosphate reductase P07004 179 , 287 179 P0A9L8 5.64 / 28,144.88 Pyrroline-5-carboxylate reductase Prolyl-tRNA synthetase Glycine betaine-binding periplasmic protein ProC ProS ProX P16659 P0AFM2 179 , 287 5.06 / 55,572 ( 4.5 -- 5.5 ) 5.12 / 63,692.60 5.65 / 33,726.86 Member of a multicomponent binding-protein-dependent transport system ( the ProU transporter ) which serves as the glycine betaine/L-proline transporter ; induced by high pH during anaerobic-growth , osmotic stress , and the stationary phase Utilized by both the de novo and the salvage pathways by which endogenously formed or exogenously added pyrimidine , purine , or pyridine bases are converted to the corresponding ribonucleoside monophosphates 5.12 / 63,692.60 5.65 / 33,726.86 103 , 179 , 191 , 321 , 323 5.65 / 33,203 ( DIGE 4.5 -- 6.5 ) 5.44 / 33,098 ( DIGE 4.5 -- 6.5 ) 5.23 / 34,087.08 P0A717 Ribose-phosphate pyrophosphokinase Prs 179 179 Psd 5.51 / 35,934.42 P0A8K1 Phosphatidylserine decarboxylase proenzyme Continued on facing pag Required for binding-protein-mediated phosphate transport ; induced by phosphate deprivation ; subject to positive control by PhoB and to negative control by PhoR Involved in conversion of acetate to acetyl-CoA ; induced by high pH but decreases during phosphate limitation Endopeptidase that degrades small peptides of less than 7 kDa , such as glucagon and insulin Component of the phosphoenolpyruvate-dependent sugar PTS , a major carbohydrate active-transport system .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16760308" target="_blank">16760308</a>]<br></li>
<li>MetR MetR metH NR ( I ) , CRP ginApl ... ¬ -4 - II * NR ( I ) NR ( I ) a II I gInL NR ( I ) NFR ( l ) , ' OmpR OmpR O OmO pC pR momp R II OmpR OmpR OmpR ompF I OxyR OxyR ahpC  ' OxyROxyR   OxyR OxyR katG a a ~ ~ OxyR oxyR ' PapB I a PapB , CRP aPapB PapB papB , ' , +92 +125 PhoB phoA PhoB PhoB I Pho B PhoB PhoB phOB 4 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1943993" target="_blank">1943993</a>]<br></li>
<li>) 1,2 139 CRP cytR 0 1 p 1,2 142 CRP deop2 1 1 p 1,2 142 , 162 CRP galpi 1 1 0 1,2 60 CRP glpD 0 1 0 2,4 174 CRP glnApl 1 1 0 4 119 CRP ilvB 0 0 0 4 39 CRP lac 0 1 0 1,2 87 CRP malT 0 0 0 1,2 88 CRP melR 0 0 0 1,2 169 CRP nagB 1 0 0 4 108 CRP nagE 1 0 0 4 108 CRP pBRp4 0 0 0 2,4 113 CRP putA 1 1 0 4 47 CRP rpoHpS 1 0 0 4 99 CRP tnaA 0 0 0 2,4 27 CRP tsx 0 1 p 4 142 , 165 FNR dmsA 0 0 0 4 35 FNR fdrA 0 0 0 1,4 35 , 61 FNR , Y gipA 0 0 0 4 35 FNR , Y narG 0 0 0 1,4 35 , 75 FNR nirB 0 1 0 1,4 35 , 61 , 62 FNR PFLp6 1 0 0 4 133 FNR PFLp7 1 0 0 4 75 , 133 IlvY ilvC 1 0 p 3 170 MalT malE 1 1 R 1,2 114 , 115 MalT malK 1 1 R 1,2 114 , 115 MalT malPQ 0 0 0 1,4 138 MetR metE 1 1 0 1,3 161 MetR metH 0 0 0 4 160 , 161 OmpR ompC 0 0 p 1,2 80 , 81 , 154 OmpR ompF 0 0 p 1,2 38 , 116 , 154 OxyR ahpC 0 0 0 3 146 OxyR katG 0 0 0 3 146 , 149 PapB papB 1 1 R 1,2 37 , 41 PhoB phoA 0 0 0 4 84 PhoB phoB 0 0 0 1,4 84 , 154 PhoB phoE 0 0 P ( ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1943993" target="_blank">1943993</a>]<br></li>
<li>12.38 -8.24 -59.96 1.13 ± 0.55 ± 0.50 ± 0.02 0.04 0.02 1.27 ± 0.50 ± 0.20 ± 0.03 0.03 0.03 / h ) / h ) Fig. 2 Comparison of the transcript levels of the E. coli wild type under different pH values at 7.0 and 6.0 : a global regulatory genes , b PhoB regulatory genes , c acid-inducible genes , d metabolic pathway genes , e respiratory chain genes during micro-aerobic continuous culture at the dilution rate of 0.1 h -1 Fig. 3 a , b Overall metabolic regulation of the E. coli during acidic condition ( a ) Acid Condition Hns ATP ADP PhoQ PhoQ P ATP ADP ¬ EvgS EvgS P cAMP - evgA Pho P PhoP Crp EvgA P B1500 EvgA cAMP-Crp ¬ RpoS b1500 ydeo ¬ YdeO gadE GadE O O GadW GadX hdeA gadB gadc gadW gadA gadX HdeA GadA GadB GadC Acid resistance ( b ) Acidic Condition ( H + ) ( indirect ) EnvZ PhoR fnr OxyR ldhA arcA P Fnr ArcA SoxR / S L PhoR - P OmpR OmpR - P DH pfl yfiD ompC TCA cycle genes : gltA sdhC mdh sodA PhoB - P PhoB Lactate OmpC ompF asr YfiD Pfl OmpF Asr Formate Acetate ackA pta Acid resistance Moreover , in accord with the proteomic analysis , the expression of yﬁD : : lac reporter fusion and the intracellular content of YﬁD were found to be high during growth at low pH for both aerobic and anaerobic cultures ( Neil et al. 2002 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23274360" target="_blank">23274360</a>]<br></li>
<li>Dual Transcriptional Regulation of the Escherichia coli Phosphate-Starvation-Inducible psiE Gene of the Phosphate Regulon by PhoB and the Cyclic AMP ( cAMP ) - cAMP Receptor Protein Complex SOO-KI KIM ,1 SIGENOBU KIMURA ,2 HIDEO SHINAGAWA ,2 ATSUO NAKATA ,3 KI-SUNG LEE ,1 BARRY L. WANNER ,4 AND KOZO MAKINO2 * Research Center for Biomedicinal Resources ( Bio-Med RRC ) , Paichai University , Taejon 302-735 , Korea1 ; Department of Molecular Microbiology , Research Institute for Microbial Diseases , Osaka University , 3-1 Yamadaoka , Suita , Osaka 565-0871 ,2 and Department of Marine Biotechnology , Faculty of Technology , Fukuyama University , Fukuyama , Hiroshima 729-0292 ,3 Japan ; and Department of Biological Sciences , Purdue University , West Lafayette , Indiana 479074 Received 10 March 2000/Accepted 7 July 2000 We have shown that the Escherichia coli phosphate-starvation-inducible psiE gene is regulated by both phosphate and the carbon source by using both lacZ and chloramphenicol acetyltransferase gene ( cat ) fusions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10986267" target="_blank">10986267</a>]<br></li>
<li>Genome-Wide PhoB Binding and Gene Expression Profiles Reveal the Hierarchical Gene Regulatory Network of Phosphate Starvation in Escherichia coli Chi Yang , Tzu-Wen Huang , Shiau-Yi Wen , Chun-Yang Chang , Shih-Feng Tsai , Whei-Fen Wu , Chuan-1 3 1 4 3 4 Hsiung Chang * 1,2 1 Institute of Biomedical Informatics , Center for Systems and Synthetic Biology , National Yang Ming University , Taipei , Taiwan , 2 Center for Systems and Synthetic Biology , National Yang Ming University , Taipei , Taiwan , 3 Division of Molecular and Genomic Medicine , National Health Research Institutes , Miaoli , Taiwan , 4 Department of Agricultural Chemistry , National Taiwan University , Taipei , Taiwan Abstract The phosphate-starvation response in bacteria has been studied extensively for the past few decades and the phosphatelimiting signal is known to be mediated via the PhoBR two-component system .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23071782" target="_blank">23071782</a>]<br></li>
<li>Citation : Yang C , Huang T-W , Wen S-Y , Chang C-Y , Tsai S-F , et al. ( 2012 ) Genome-Wide PhoB Binding and Gene Expression Profiles Reveal the Hierarchical Gene Regulatory Network of Phosphate Starvation in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23071782" target="_blank">23071782</a>]<br></li>
<li>To study the in vivo function of IciA , we examined iciA gene expression under abnormal growth conditions Communicated by H. Ikeda J. S. Han á J. Y. Park á Y. S. Lee á D. S. Hwang ( & Institute of Molecular Biology and Genetics , Seoul National University , Seoul 151-742 , Korea E-mail : dshwang@plaza.snu.ac.kr Tel. : +82 -2 -880 -7524 ; Fax : +82 -2 -874 -1206 ) J. S. Han Department of Microbiology , Seoul National University , Seoul 151-742 , Korea J. Y. Park á D. S. Hwang Interdisciplinary Program in G Seoul National University , Seoul 151-742 , Korea enetic Engineering , B. Tho È ny Division of Clinical Chemistry and Biochemistry , Departments of Pediatrics , University of Zu È rich , Steinwiesstrasse 75 , CH-8032 Zu È rich , Switzerland This report shows that the activation of the iciA gene upon phosphate depletion is dependent on PhoB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10589831" target="_blank">10589831</a>]<br></li>
<li>Among the RRs tested , five species of RR ( CitB , CpxR , PhoB , RssB , and YhjB ) enhanced dephosphoryl-ation of more than two species of HK , and four species ( CusR , HydG , PhoP , and YfhA ) enhanced dephosphorylation of one non-cognate HK .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15522865" target="_blank">15522865</a>]<br></li>
<li>For instance , dephosphorylation of pDcuS was enhanced by four non-cognated RRs ( CitB , CpxR , PhoB , and RssB ) and dephosphorylation of pEnvZ , pNtrB , and pRstB was stimulated each by three non-cognate RRs ( Fig. 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15522865" target="_blank">15522865</a>]<br></li>
<li>On the basis of additional homology within the carboxy-terminal domain , TorR has been recognized as a member of the OmpR subfamily of response-regulators , which includes numerous members in E. coil ( such as ArcA , OmpR , or PhoB ) but also in many other bacterial species ( Aiba et aL , 1993 ; Mills et aL , 1993 ; Nagasawa et aL , 1993 ; Parkinson and Kofoid , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596446" target="_blank">8596446</a>]<br></li>
<li>This model is supported by the requirement for phosphorylation of ArcA to-bind to DR3 , which is also known to promote dimer formation among OmpR/PhoB response-regulators ( 20 , 21 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25161193" target="_blank">25161193</a>]<br></li>
<li>Acetyl-P is also known to phosphorylate various response-regulators , including CheY , NRI , PhoB , and OmpR ( 11 , 28 , 52 ) , and is thus suggested to inﬂuence either the sensitivity or the magnitude of an adaptive response ( 29 , 51 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7642497" target="_blank">7642497</a>]<br></li>
<li>In class I ( e.g. NtrC and PhoB ) , this interaction leads to the inhibition of the receiver module .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7641688" target="_blank">7641688</a>]<br></li>
<li>NtrC , PhoB and CheB , the subjects of this study , belong to different subfamilies of response-regulators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7641688" target="_blank">7641688</a>]<br></li>
<li>Several recent studies demonstrate in vivo activation of the response-regulators , PhoB ( Lee etai .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7934904" target="_blank">7934904</a>]<br></li>
<li>Phosphorylation-mediated dimerization of the receiver domains of the NtrC and PhoB proteins has recently been reported and is proposed to result in their activation as transcriptional-regulators in vivo ( 16 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8892825" target="_blank">8892825</a>]<br></li>
<li>We suggest that NtrC and PhoB represent class of one response-regulators , where activation is mediated by dimerization of the phosphorylated receiver modules .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7641688" target="_blank">7641688</a>]<br></li>
<li>A common switch in activation of the response-regulators NtrC and PhoB : phosphorylation induces dimerization of the receiver modules During signal transduction , response-regulators of two-component systems are phosphorylated in a conserved receiver module .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7641688" target="_blank">7641688</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_34">
A common switch in activation of the response-regulators NtrC and PhoB : phosphorylation induced dimerization of receiver modules .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9973351" target="_blank">9973351</a>]<br></label>
<input id="act_similarity_34", type="checkbox">
<div class="ACT_similar">
<li>A common switch in activation of the response-regulators NtrC and PhoB : phosphorylation induces dimerization of the receiver modules .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8981978" target="_blank">8981978</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659677" target="_blank">15659677</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8892825" target="_blank">8892825</a>]<br></li>
<li>A common switch in activation of the response-regulators NtrC and PhoB : phosphorylation induces dimerization of the recei-ver modules .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9878422" target="_blank">9878422</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9600853" target="_blank">9600853</a>]<br></li></div>
<li>Although , KdpE does not contain identical residues ( it is Y191 , I194 , H198 and T217 respectively at positions equivalent to those of PhoB shown in the previous line ) , the properties of three side chains are similar except for KdpE T217 and PhoB R219 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22291906" target="_blank">22291906</a>]<br></li>
<li>At least four sets of gene pairs , cusR/cusC , prpR/prpB , feaR/feaB , and yedW/yedX , have the potential to form feed forward loops with PhoB regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23071782" target="_blank">23071782</a>]<br></li>
<li>Figure S3 Purity data for purification of PhoB-His fusion Figure S4 Novel PhoB bindings located in the coding regions may participate in gene regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23071782" target="_blank">23071782</a>]<br></li>
<li>55 different TFs binding sites were detected with high confidence , of which the most frequently found were those for CRP , Fis , PhoB , RhaS , PurR and FNR ( Table S4 and Figure S6 ) , of which , CRP , Fis and FNR are general TFs that regulate many genes in E. coli ( Table S5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19838305" target="_blank">19838305</a>]<br></li>
<li>In this study , chromatin immunoprecipitation for PhoB was combined with high-density tiling array ( ChIP-chip ) as well as gene expression microarray to reveal the first global down-stream responses of the responding regulator , PhoB in E. coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23071782" target="_blank">23071782</a>]<br></li>
<li>g003 In summary , our genome-wide approach for characterizing the roles of PhoB by ChIP-chip and gene expression array provides a comprehensive global binding profile of PhoB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23071782" target="_blank">23071782</a>]<br></li>
<li>In the sense strand , PhoB protein protected the DNA segment spanning nucleo-tides -20 to -101 from DNase I digestion and G residues at positions -28 , -44 , -53 , -57 , -61 , -66 , -68 , -72 , -76 , -79 , -85 , -88 , and -90 from methylation , and it enhanced the methylation of G residues at -65 , -84 , and -87 ( Fig. 6A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1987150" target="_blank">1987150</a>]<br></li>
<li>[ 21 ] K. Makino , H. Shinagawa , M. Amemura , S. Kimura , A. Nakata , Activation of pstS transcription by PhoB , J. Mol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15313253" target="_blank">15313253</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_45">
[ 20 ] K. Makino , M. Amemura , T. Kawamoto , S. Kimura , H. Shinagawa , A. Nakata , M. Suzuki , DNA binding of PhoB and its interaction with RNA polymerase , J. Mol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15313253" target="_blank">15313253</a>]<br></label>
<input id="act_similarity_45", type="checkbox">
<div class="ACT_similar">
<li>Makino , K. , Amemura , M. , Kawamoto , T. , Kimura , S. , Shinagawa , H. , Nakata , A. , and Suzuki , M. ( 1996 ) DNA binding of PhoB and its interaction with RNA polymerase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15255893" target="_blank">15255893</a>]<br></li></div>
<li>However , the superposed structures reveal KdpE H198 is less-likely to participate in base specific interactions due to its shorter side-chain length when compared to R201 of PhoB ( Fig. 2C ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22291906" target="_blank">22291906</a>]<br></li>
<li>If the structure of PhoP is similar to that of PhoB , intra - and intermolecular interactions of PhoP might protect Arg-87 and Lys-101 from digestion by trypsin .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12123454" target="_blank">12123454</a>]<br></li>
<li>D196 of PhoB does not contribute to DNA interactions [ 42 ] , however studies on PhoP from M. tuberculosis an ortholog of PhoB suggest a potential role for the equivalent residue E215 in base-specific interactions [ 50 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22291906" target="_blank">22291906</a>]<br></li>
<li>To understand the contribution of each DR in ArcA repression of icdA , they were mutated toward or away from 5 = - TGTTA-3 = ( Fig. 1C ) , the most conserved sequence within each ArcA DR element ( hereinafter referred to as the consensus ) and , based on the PhoB DNA cocrystal structure ( 15 ) , the region likely contacted by ArcA in the major groove .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25161193" target="_blank">25161193</a>]<br></li>
<li>It is possible that ArcA-P binds to the icdA promoter as a dimer of dimers ; one dimer binds DR1 and DR2 , as depicted in the PhoB and KdpE DNA cocrystal structures ( 15 , 18 ) , and the second dimer binds DR3 but only weakly to adjacent DNA sequence , such that no footprint is observed ( Fig. 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25161193" target="_blank">25161193</a>]<br></li>
<li>Tandem DNA recognition by PhoB , a two-component signal transduction transcriptional-activator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18375564" target="_blank">18375564</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25161193" target="_blank">25161193</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12897000" target="_blank">12897000</a>]<br></li>
<li>No large structural changes ( Ca rmsd 1.15 Å ) were reported for PhoB in free and bound conformations [ 42 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22291906" target="_blank">22291906</a>]<br></li>
<li>Comparison of KdpEDBD to DBDs of OmpR ( 1OPC ) [ 9 ] , PhoB ( 1GXQ ) , and PhoB bound to DNA ( 1GXP ) [ 42 ] revealed similar overall structures with Ca root mean square deviations ( rmsd ) from 1.38 to 1.71 Å .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22291906" target="_blank">22291906</a>]<br></li>
<li>Likewise , the a-helical and b-stand elements of KdpEDBD superpose well onto PhoB in DNA bound conformation with deviations restricted to loops connecting a7-a8 and a8 to the bhairpin structure ( Fig. 2C ; the DNA corresponds to the PhoB recognition sequence ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22291906" target="_blank">22291906</a>]<br></li>
<li>C. Superposition of KdpEDBD onto the structure of PhoB bound to DNA ( PDB code : 1GXP ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22291906" target="_blank">22291906</a>]<br></li>
<li>Residues in KdpE targeted for mutagenesis namely R193 , R200 and T215 are indicated by stars , whereas the + sign points to R219 of PhoB that interacts with the minor groove of DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22291906" target="_blank">22291906</a>]<br></li>
<li>PhoB has in common with NtrC that phosphorylation enhances its DNA binding ability ( Makino et al. , 1989 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7641688" target="_blank">7641688</a>]<br></li>
<li>In this design , the genome-wide map of interactions between PhoB and E. coli genomic DNA was constructed ( Figure S2A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23071782" target="_blank">23071782</a>]<br></li>
<li>However , the global DNA binding profile of the response-regulator PhoB and the PhoB downstream responses are currently unclear .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23071782" target="_blank">23071782</a>]<br></li>
<li>Cooperative binding of PhoB ( DBD ) to its cognate DNA sequence -- a combined application of single-molecule and ensemble methods .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25161193" target="_blank">25161193</a>]<br></li>
<li>In fulllength PhoB and MtrA [ 41 ] , the interactions of the RD with DBD sterically prevent the latter from binding to DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22291906" target="_blank">22291906</a>]<br></li>
<li>Okamura , H. , Hanaoka , S. , Nahadoi , A. , Makino , K. , and Nishimura , Y. ( 2000 ) Structural comparision of the PhoB and OmpR DNA-binding/transactivation domain and the arrangement of PhoB molecules on the phosphate box .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12123454" target="_blank">12123454</a>]<br></li>
<li>Structural comparison of the PhoB and OmpR DNA-binding/transactivation domains and the arrangement of PhoB molecules on the phosphate box .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11976297" target="_blank">11976297</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11489853" target="_blank">11489853</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10986267" target="_blank">10986267</a>]<br></li>
<li>If phosphate becomes limiting , phosphorylated PhoB dimers may have a higher afﬁnity for other PhoB binding sites and thereby derepress the eda P4 promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659677" target="_blank">15659677</a>]<br></li>
<li>Suggested binding sites for CRP , NAC , PhoB , and NsrR are underlined .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24013633" target="_blank">24013633</a>]<br></li>
<li>While unphosphorylated PhoB possesses DNA binding capability , phosphorylated PhoB dimerizes , which increases its afﬁnity for PhoB binding sites ( 9 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659677" target="_blank">15659677</a>]<br></li>
<li>This protein , UspA ( universai stress protein A ) , is thus a generai , non-specific responder to growth arrest ( this paper ; Nystrom and Neidhardt , 1992 ) , and its induction is independent of RpoS as weli as of aii other giobal reguiators so far tested , inciuding PhoB , ReiA , SpoT , AppY , OmpR , H-ns , Lrp , and RpoH ( Nystrom and Neidhardt , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8152377" target="_blank">8152377</a>]<br></li>
<li>Phosphorylated BarA , in turn , acts as an efficient phosphoryl group donor to UvrY but not to the non-cognate response-regulators ArcA , PhoB , or CpxR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11022030" target="_blank">11022030</a>]<br></li>
<li>Third , BarA catalyzes the phosphorylation of UvrY but not the phosphorylation of the non-cognate response-regulators ArcA , PhoB , or CpxR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11022030" target="_blank">11022030</a>]<br></li>
<li>The protein concentrations ( in pmol ) were 50 9BarA , 100 UvrY , 100 ArcA , 100 PhoB , 100 CpxR , and 50 ArcB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11022030" target="_blank">11022030</a>]<br></li>
<li>A , comparison of the ability of 9BarA to phosphorylate the UvrY , ArcA , PhoB , and CpxR proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11022030" target="_blank">11022030</a>]<br></li>
<li>Comparison of the abilities of H1-D1 to phosphorylate the ArcA , CpxR , and PhoB proteins and the inﬂuence of H2 on the reaction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9286997" target="_blank">9286997</a>]<br></li>
<li>Equimolar amounts of purified ArcA , PhoB , and CpxR were each incubated with 9BarA in the presence of [ g-32P ] ATP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11022030" target="_blank">11022030</a>]<br></li>
<li>H1-D1 ( 50 pmol ) was incubated in 5-ml reaction mixtures with [ g-32P ] ATP for 2 min with ArcA ( 100 pmol ) , CpxR ( 200 pmol ) , or PhoB ( 100 pmol ) in the presence ( 1 ) or absence of H2 ( 100 pmol ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9286997" target="_blank">9286997</a>]<br></li>
<li>Although CreC has been shown previously to phosphorylate PhoB only in the absence of PhoR , overproduction of CreC can result in Transcriptome analysis of a two-component system 285 phosphorylation of PhoB in both the presence and the absence of PhoR ( J. A. Masella and B. L. Wanner , unpublished data ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12366850" target="_blank">12366850</a>]<br></li>
<li>Purification of PhoB and PhoR1084 proteins and phosphor-ylation of PhoB by PhoR1084 in the presence of ATP were done as described previously ( 14 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1840580" target="_blank">1840580</a>]<br></li>
<li>Purification of PhoB and PhoR1084 proteins and phosphorylation of PhoB by PhoR1084 were done as described previously ( 22 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8444794" target="_blank">8444794</a>]<br></li>
<li>Unfortunately , this argument is based on studies of PhoB and CheY , two RRs that appear unresponsive to acetyl P in the presence of their cognate HKs .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15755952" target="_blank">15755952</a>]<br></li>
<li>In the presence of PhoR , however , PhoM is not able to affect the activity of PhoB ( 21 ) ; instead , it has long been thought to act through its cognate RR , which is encoded by phoM-orf2 ( 25 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18375564" target="_blank">18375564</a>]<br></li>
<li>By analogy to NtrC ( NRI , ginG ) and NtrB ( NRII , glnL ) in nitrogen control ( 31 ) , CheY and CheA in chemotaxis ( 24 ) , and OmpR and EnvZ in osmoregulation ( 2 , 19 , 26 ) , PhoR may be an autophosphorylase that also phosphorylates and dephosphorylates PhoB to convert it between active and inactive forms .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2160940" target="_blank">2160940</a>]<br></li>
<li>These include CreB and PhoB of enteric bacteria which are responsive to cytoplasmic catabolite concentrations and extracellular inorganic phosphate , respectively ; OmpR of enteric bacteria , involved in responses to osmotic conditions ; VirG proteins of agrobacterial species , activators of the Ti or Ri plasmid-encoded virulence op-erons ; and ArcA , an E. coli activator of genes encoding proteins which function in aerobic metabolism .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7699720" target="_blank">7699720</a>]<br></li>
<li>However , as Mlc shows no homology to any of the known effector components of two-component systems , e.g. OmpR , PhoB , NtrC , this would seem to exclude a classical two-component-type regulatory mechanism .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10411743" target="_blank">10411743</a>]<br></li>
<li>The phosphorylation level was from 1 % to 10 % for TorR , PhoB , CusR , BasR , RstA , UvrY , ArcA , BaeR , NtrC , KdpE , UhpA , and PhoP ( in decreasing order ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15522865" target="_blank">15522865</a>]<br></li>
<li>Furthermore , comparison of the electrophoretic mobility of P-UhpA in native gels of various polyacrylamide concentrations , in comparison to the migration of protein standards , indicated that P-UhpA was monomeric in solution under those conditions .3 The rates of phosphorylation and dephosphorylation of UhpA were comparable to those seen for OmpR and PhoB ( 13 , 34 ) and were much slower than the corresponding processes in CheY and NtrC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8999880" target="_blank">8999880</a>]<br></li>
<li>227 OmpR ArcA VirG PhoP Va `` R PhoB CreB TctD KdpE kINnlaRgre .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8294007" target="_blank">8294007</a>]<br></li>
<li>Asterisks ( * ) indicate the conserved aa residues crucial for the transcriptional regula ¬ 232 CpxR nvsas ... OmpR ArcA VirG PhoP V & `` R PhoB CreB TctD KdpE hrpdgska .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8294007" target="_blank">8294007</a>]<br></li>
<li>Members of this class are NtrC and PhoB , other candidates are ArcA , OmpR and KdpE .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7641688" target="_blank">7641688</a>]<br></li>
<li>OmpR , SfrA , PhoB , and VirG are one such group .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2404948" target="_blank">2404948</a>]<br></li>
<li>PhoB CreB OmpR FIG. 7 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1532388" target="_blank">1532388</a>]<br></li>
<li>Unlike OmpR , the DBDs of both PhoB and KdpE bind their cognate DNA with moderate affinities ( range of Kds 100 -- 400 nM ) ; however , only the DBD of PhoB can initiate transcription [ 40 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22291906" target="_blank">22291906</a>]<br></li>
<li>For T215 of KdpE , variants with mutations at equivalent residues in OmpR and PhoB are also defective in DNA binding due to loss of H-bonding with the DNA backbone [ 28,49 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22291906" target="_blank">22291906</a>]<br></li>
<li>The inability of mutant KdpE R193A to-bind to DNA is interesting because in most other family members the equivalent residue is of opposite charge as observed in PhoB ( D196 ) and OmpR ( D202 ) ( Fig .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22291906" target="_blank">22291906</a>]<br></li>
<li>The carboxyl ` terminal domain of NarL shares only short patches of similarity with PhoB , OmpR , and SfrA , but it is strikingly similar to the analogous regions of DegU , UhpA , and FixJ ( Fig. 3 and Table 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2649492" target="_blank">2649492</a>]<br></li>
<li>The N-terminal two-thirds of the NarL protein has extensive homology with those of a diverse set of prokaryotic regulatory proteins , including OmpR , PhoB , SfrA , UhpA , CheY , CheB , NtrC , DctD , FixJ , VirG , SpoOF , and SpoOA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2657652" target="_blank">2657652</a>]<br></li>
<li>Therefore , the most proteins of this family are subdivided into four different groups according to their C-terminal domain : ( i ) NarL , DegU , UvrC ORF2 , UhpA , and FixJ ; ( ii ) OmpR , PhoB , PhoM ORF2 , SfrA , and VirG ; ( iii ) NtrC , DctD , and PgtA ; and ( iv ) CheY and SpoOF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2657652" target="_blank">2657652</a>]<br></li>
<li>One family includes OmpR , PhoB , SfrA , VirG , CheB , CheY , Spo0A , SpoOF , NtrC , and DctD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3042748" target="_blank">3042748</a>]<br></li>
<li>The match to the other proteins , although obvious , was weaker , ranging from about 25 % identity with NtrC and CheY to 19 % for OmpR , SfrA , and PhoB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3042748" target="_blank">3042748</a>]<br></li>
<li>The matches of UhpA with DctD , NtrC , PhoB , and CheY were at least 12 standard deviations ( SDs ) above random .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3042748" target="_blank">3042748</a>]<br></li>
<li>The amino acid sequence of PhoB protein is similar to those of several regulatory proteins , including OmpR , Dye , ORF2 ( of the phoM operon ) , NtrC , VirG , SpoOA and SPOOF , most of which are probably transcriptional-regulators for global genetic regulatory systems in bacteria ( Comeau et al. , 1985 ; Drury & Buxton , 1985 ; MacFarlane & Merrick , 1985 ; Amemura al. , 1986 ; IIrummond al. , 1986 ; et et Nixon et al. , 1986 ; Winans et al. , 1986 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3054125" target="_blank">3054125</a>]<br></li>
<li>This hypothesis is based on the striking similarities that are shared at the protein sequence level for the regulators PhoB , NtrC , CheY , and OmpR and for their respective sensory proteins PhoR , NtrB , CheA , and EnvZ , which are now thought to act in this way ( 7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2160940" target="_blank">2160940</a>]<br></li>
<li>( d ) Homology among the PhoB , OmpR and Dye proteins We searched for proteins with homology to the PhoB protein , using the NBRF protein sequence data base .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3537313" target="_blank">3537313</a>]<br></li>
<li>As a consequence of the homology with NtrC , there was also homology with other regulatory proteins which respond to other environmental stimuli such as phosphate limitation ( PhoB ) , altered osmolarity ( OmpR ) , presence of female cells ( SfrA ) , presence of plant exudate , .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2666400" target="_blank">2666400</a>]<br></li>
<li>Extensive similarity of N-terminal amino acid sequence of NarL to other prokaryotic regulatory proteins such as OmpR and PhoB ( Fig. 3 ) strongly suggests that NarL acts directly on the regulatory region of the narK and narC operons after some modification to an active form in the presence of nitrate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2657652" target="_blank">2657652</a>]<br></li>
<li>Amino acid sequence comparisons showed that UhpA was homologous to a family of bacterial regulatory proteins , some of which act as transcriptional-activators ( OmpR , PhoB , NtrC , and DctD ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3042748" target="_blank">3042748</a>]<br></li>
<li>Primary amino acid sequence homology alignment of NarL to OmpR , VirG , and PhoB proteins from procaryotic microorganisms .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2648330" target="_blank">2648330</a>]<br></li>
<li>The PhoP sequence was aligned along the secondary structures of PhoB ( Sola et al. , 1999 ; Okamura et al. , 2000 ) and OmpR ( Martinez-Hackert and Stock , 1997a ; 1997b ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12123454" target="_blank">12123454</a>]<br></li>
<li>The sequences of KdpD and KdpE show they are members of the sensor-effector class of regulatory proteins : the C-terminal half of KdpD is homologous to sensors such as EnvZ and PhoR , and KdpE is homologous to effectors such as OmpR and PhoB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1532388" target="_blank">1532388</a>]<br></li>
<li>The first approximately 200 residues of KdpE and three other effector-proteins , PhoB ( 28 ) , CreB ( formerly PhoM-orf2 , the hypothetical PhoB protein of PhoM/Pho2 ) ( 2 , 48 ) , and OmpR ( 8 ) of E. coli are aligned with the TREE program ( 13 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1532388" target="_blank">1532388</a>]<br></li>
<li>The induction was independent of the RelA/SpoT , RpoH , KatF , OmpR , AppY , Lrp , PhoB and H-NS proteins during stress conditions that are known to induce or activate these global regulators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1453957" target="_blank">1453957</a>]<br></li>
<li>The sizes of these proteins are similar ; the PhoB , OmpR and Dye proteins consist of 228 , 239 and 238 amino acids , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3537313" target="_blank">3537313</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3537313" target="_blank">3537313</a>]<br></li>
<li>It should be noted that the RNA polymerase holoenzyme containing r70 is also involved in the transcriptional activation by PhoB , where Fig. 1a shows the down-regulation of rpoD as well as rpoS , and the transcript levels of gadX and gadC decreased , but not gadA in the phoB mutant .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23274360" target="_blank">23274360</a>]<br></li>
<li>As the activity of the eptC-lacZ transcriptional fusion was not totally abolished in phoB mutants , it suggests that a constitutive basal transcriptional activity is present , which is independent of both PhoB and BasR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23372159" target="_blank">23372159</a>]<br></li>
<li>( d ) Homology among the PhoB , OmpR and Dye proteins Our previous work ( Shinagawa et al. , 1983 ) and that by Guan et al. ( 1983 ) suggested that the phoB gene is physiologically regulated by inorganic phosphate in the medium and genetically regulated by phoR , phoM and a cluster of genes in the pst region , in the same manner as phoA expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3537313" target="_blank">3537313</a>]<br></li>
<li>However , from the results obtained by using a phoB ' - cat fusion gene , Guan et al. ( 1983 ) suggested that expression of phoB is positively regulated by the PhoB protein while our results obtained b using a phoB ' - ` la & fusion gene indicated that the PhoB protein is not related to the regulation of the gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3537313" target="_blank">3537313</a>]<br></li>
<li>as Guan et al. ( 1983 ) suggested , or due to some residual PhoB activity of the product ' of phoR ' - ` la & fusion gene that contains about SOS / , of the ph , oB coding region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3537313" target="_blank">3537313</a>]<br></li>
<li>PhoR and PhoB have been found to be structurally as well as functionally similar to many two-component regulators for genetic systems that respond to environmental changes ( Makino et al. , 1986a , b ; Ronson et al. , 1987 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3054125" target="_blank">3054125</a>]<br></li>
<li>It was also shown that PhoR was autophosphorylated at His 213 , and transferred phosphoryl group to PhoB at Asp 53 to activate it ( Makino , K. et al. , op .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2657652" target="_blank">2657652</a>]<br></li>
<li>directly or via its acceptors might activate PhoR in the periplasm and that asr is under control of PhoB -- PhoR operon ( Suziedeliené et al. 1999 ) , where it was also reported that asr expression was abolished in the phoB -- phoR deletion mutant .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23274360" target="_blank">23274360</a>]<br></li>
<li>We obtained evidence that P1 is under negative control of GntR , P2 is under negative control of KdgR , and P4 is under negative control of PhoB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659677" target="_blank">15659677</a>]<br></li>
<li>These results suggest that the promoter P1 is activated by PhoB , and the CAT activities in ANCH1 reflect the activity of the promoter P2 , which is PhoB independent .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8444794" target="_blank">8444794</a>]<br></li>
<li>These results , together with measurements of the in vivo psiE promoter activity under various conditions , show that expression of the psiE gene is under direct positive and negative control by PhoB and cAMP-CRP , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10986267" target="_blank">10986267</a>]<br></li>
<li>First , the level of repression at XPR in vivo was higher under conditions where the proteins were phosphorylated in the cell ( 55 % for XN-NtrCRO and 65 % for XN-PhoB ; Figures 3 and 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7641688" target="_blank">7641688</a>]<br></li>
<li>Under high-glucose and LP conditions , where the amount of PhoB is large and that of cAMP-CRP is small , PhoB activates transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10986267" target="_blank">10986267</a>]<br></li>
<li>Transcription from the promoter would be blocked by cAMP-CRP under low-glucose and HP conditions , because of a high concentration of cAMP-CRP and low concentration of PhoB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10986267" target="_blank">10986267</a>]<br></li>
<li>Under low-glu-cose and LP conditions , where both proteins are abundant , PhoB and cAMP-CRP would compete for binding to the psiE promoter region , which contains overlapping sequences recognized by these proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10986267" target="_blank">10986267</a>]<br></li>
<li>Lanes 1 to 13 , 0 , 1.99 , 8.3 , 24.9 , 49.8 , 58.1 , 66.4 , 49.8 , 0 , 1.66 , 24.9 , 58.1 , and 59.1 pmol of PhoB , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659677" target="_blank">15659677</a>]<br></li>
<li>Lanes 1 and 2 are the products of S , mapping without or with PhoB prot , ein , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3054125" target="_blank">3054125</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3054125" target="_blank">3054125</a>]<br></li>
<li>From the gene expression microarray data analysis , 287 differentially expressed genes were identified in the presence of PhoB activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23071782" target="_blank">23071782</a>]<br></li>
<li>Gene expression microarray and analysis The Affymetrix E. coli Genome 2.0 array was used to investigate gene expression status in the presence and the absence of PhoB activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23071782" target="_blank">23071782</a>]<br></li>
<li>This may be a clue that PhoB cooperates with other factors to modulate transcription since other factors may reduce the effect of PhoB regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23071782" target="_blank">23071782</a>]<br></li>
<li>In our study , all of these genes showed down-regulation and their transcription repressor , PuuR , in turn was up-regulated in the presence of PhoB activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23071782" target="_blank">23071782</a>]<br></li>
<li>However , they both showed down regulation in the presence of PhoB from our microarray datasets .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23071782" target="_blank">23071782</a>]<br></li>
<li>It is interesting that genes in this 14 kb region all showed up-regulation in the presence of PhoB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23071782" target="_blank">23071782</a>]<br></li>
<li>Likewise , loss-of-function mutations in the pst genes lead to the constitutive phosphorylation of PhoB , regardless of the external Pi concentration .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14526009" target="_blank">14526009</a>]<br></li>
<li>Differentially expressed genes containing putative PhoB binding sites in their coding regions were selected to confirm that PhoB binds to their coding regions and may participate in regulating them .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23071782" target="_blank">23071782</a>]<br></li>
<li>The results demonstrate that PhoB binds to the coding regions of the three genes despite the weak binding to yahA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23071782" target="_blank">23071782</a>]<br></li>
<li>Some of the new genes found to be regulated by PhoB were involved in acid and metal ion stress responses , which indicate the additional roles of PhoB as a global regulator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17020555" target="_blank">17020555</a>]<br></li>
<li>The gadA and gadB genes for glutamate decarobxylase isozymes in a glutamate-dependent acid response system were also PhoBdependently up-regulated in this study ( Table S2 ) , and this means that these genes were also indirectly regulated by PhoB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17020555" target="_blank">17020555</a>]<br></li>
<li>Most genes involved in the phosphate uptake and metabolism are regulated by PhoB except for the genes for the low-afﬁnity phosphate uptake system ( Wanner , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17020555" target="_blank">17020555</a>]<br></li>
<li>The hierarchical structure of phosphate sensing regulatory network It is worth noting that our gene expression data showed 287 genes affected by PhoB while only 19 out of these 287 genes were considered to be directly regulated PhoB targets ( Table 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23071782" target="_blank">23071782</a>]<br></li>
<li>All genes , tdk , insZ , adhE , ychE , oppABCDF and yciU were up-regulated with PhoB activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23071782" target="_blank">23071782</a>]<br></li>
<li>These ten genes are likely to be directly regulated by PhoB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23071782" target="_blank">23071782</a>]<br></li>
<li>There were 287 differentially expressed genes that were directly or indirectly regulated by PhoB ( Table S1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23071782" target="_blank">23071782</a>]<br></li>
<li>The other phoB transformants ( pBS12 , pBS14 and pBS15 ) grew on the Cm plates indicating that the putative promoters upstream of pstC , pstB and phoU are not dependent on PhoB , and therefore , should not be induced by Pi-limitation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20963440" target="_blank">20963440</a>]<br></li>
<li>The inability of KdpEDBD to promote gene expression is similar to that of the DBD of OmpR [ 39 ] and is distinct from the DBD of PhoB , which expresses the reporter gene by binding with 7-fold greater affinity than unphosphorylated PhoB to the pho box sequence [ 40 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22291906" target="_blank">22291906</a>]<br></li>
<li>The crystal structure of PhoB complexed with Pho-box DNA ( PhoB is a member of the OmpR-family transcription factor , like CpxR ) indicated that two molecules of PhoB bind in tandem to the direct repeat sequence within the Pho-box , and that the proteinprotein interaction between two PhoB proteins induces DNA curvature .16 ) The hypersensitive sites make it appear to the binding of CpxR onto the promoter induces DNA curvature .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16861804" target="_blank">16861804</a>]<br></li>
<li>The helix-loop-he-lix-containing RRs of the OmpR family , OmpR and PhoB , bind in a direct or head-to-tail arrangement to recognition sites upstream or adjacent to the RNAP-binding region , where they interact with - CTD or - CTD , respectively ( 16 , 17 , 32 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11976297" target="_blank">11976297</a>]<br></li>
<li>The OmpR family member PhoB binds to direct repeat target sites in the promoter ( 32 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11976297" target="_blank">11976297</a>]<br></li>
<li>The creC gene is part of a four-gene cluster , creABCD ( 11 ) ; the functions of CreA and CreD are unknown , but CreB is a putative response-regulator and is homologous to PhoB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11350954" target="_blank">11350954</a>]<br></li>
<li>The activation of PhoB by acetylphosphate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15755952" target="_blank">15755952</a>]<br></li>
<li>McCleary , W.R. ( 1996 ) The activation of PhoB by acetylpho-sphate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9720863" target="_blank">9720863</a>]<br></li>
<li>McCleary , W.R. ( 1996 ) The activation of PhoB by acetylphosphate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12135487" target="_blank">12135487</a>]<br></li>
<li>The transformants were PhoB - , and one of them was designated ANCB1-5 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2651888" target="_blank">2651888</a>]<br></li>
<li>The optimal condition for observing the PhoB regulation was ﬁrstly determined .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17020555" target="_blank">17020555</a>]<br></li>
<li>The carboxy-terminal half of PhoB was suggested to be involved in transcriptional activation and DNA-binding activity ( 26 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2676981" target="_blank">2676981</a>]<br></li>
<li>The coloring scheme : green , KdpEDBD ; purple , PhoB and yellow/orange , DNA strands .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22291906" target="_blank">22291906</a>]<br></li>
<li>The mipA gene also has a PhoB binding signal in its upstream region and is considered as a directly regulated target .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23071782" target="_blank">23071782</a>]<br></li>
<li>The interaction of phosphorylated PhoB with this region was also examined by methylation protection experiments ( Fig. 6B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1840580" target="_blank">1840580</a>]<br></li>
<li>The afﬁnity of PhoB for its binding site at P4 was comparable to the afﬁnity of PhoB for its binding site at phoA ( Fig. 6B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659677" target="_blank">15659677</a>]<br></li>
<li>Therefore , although this sequence appeared to function as the recognition site for PhoB , it appeared not to function as a cis-acting regulatory element .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1987150" target="_blank">1987150</a>]<br></li>
<li>In this is the Pi regard , Ugp only previously reported transporter for a P1 ester the synthesis of which is P1 regulated and under PhoB and PhoR control .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1846145" target="_blank">1846145</a>]<br></li>
<li>( iii ) Lysogens were made , and one that was phenotypically PhoB + and PhoR + was UV induced .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2824439" target="_blank">2824439</a>]<br></li>
<li>Cells were induced for PhoB synthesis by a temperature upshift , and incubation was continued for an additional 6 to 8 h .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10094685" target="_blank">10094685</a>]<br></li>
<li>In addition to CRP and NAC , feaR may also be repressed by PhoB ( 20 ) and ArcA ( P. J. Kiley , personal communication ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24013633" target="_blank">24013633</a>]<br></li>
<li>uncovered a second transcript from Pugp2 , whose expression may depend on the cAMP-CRP complex and be repressed by PhoB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1987150" target="_blank">1987150</a>]<br></li>
<li>Finally , we also observed an indirectly regulated 14 kb region to which PhoB did not tend to-bind to ( Figure S5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23071782" target="_blank">23071782</a>]<br></li>
<li>This experiment also showed that unphosphorylated PhoB can bind .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659677" target="_blank">15659677</a>]<br></li>
<li>Where do PhoB and cAMP-CRP bind ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10986267" target="_blank">10986267</a>]<br></li>
<li>We show that the former is regulated by cyclic AMP ( cAMP ) and its receptor protein , CRP , and that the latter is regulated by phosphorylated PhoB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1987150" target="_blank">1987150</a>]<br></li>
<li>The data also suggest that transcription from this start site is positively regulated by PhoB and negatively regulated by cAMP-CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10986267" target="_blank">10986267</a>]<br></li>
<li>mkndem Va `` R DkItGLtiGA DDYitKPFrp 1ELiARVkaqLRRyk ... kf sgvkaqnanv PhoB DrVrGLEtGA DDYitKPFspkELvARIkavmispma .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8294007" target="_blank">8294007</a>]<br></li>
<li>EEVYDRSIDVLILRLRRKLEADPSSPQLIKTARGAGYFFDA 301 351 RF LYRTALKHEAIALLATKFICPYVIAHRHLAVRQPGDDLSGGAELRDFAEPPA DVQVSHGGTMAA PhoB OmnpR VirG Figure 5 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2648330" target="_blank">2648330</a>]<br></li>
<li>Figure 8 shows repression of XPR-lacZ expression by the PhoB hybrid proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7641688" target="_blank">7641688</a>]<br></li>
<li>That is , PhoB could either directly or indirectly control the expression of this sRNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14526009" target="_blank">14526009</a>]<br></li>
<li>Therefore , PhoB is not required for iraP expression or activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17640895" target="_blank">17640895</a>]<br></li>
<li>For example , PhoB regulates cusR and both PhoB and CusR regulators regulate cusC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23071782" target="_blank">23071782</a>]<br></li>
<li>Binding of PhoB to this site may down-regulate the pKD 513 pKD 5131 FIG. 8 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1987150" target="_blank">1987150</a>]<br></li>
<li>.4 PhoB PhoB P phoE a hBoE i h P PhoB phoS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1943993" target="_blank">1943993</a>]<br></li>
<li>Binding of PhoB to its targets in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23071782" target="_blank">23071782</a>]<br></li>
<li>Therefore , we are conﬁdent in proposing the following model : CreC responds to changes in the growth medium and/or me-tabolite pool levels and autophosphorylates ; this phosphate is passed on to CreB , which binds to cre tag sequences ; at some promoters ( e.g. , creD or talA ) , CreB works as a transcriptional-activator by recruiting RNA polymerase in a manner typical of other winged-helix RR proteins , such as PhoB ( 7 ) ; at other promoters ( e.g. , malE ) , the binding of CreB to the cre tag represses transcription through promoter occlusion .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18375564" target="_blank">18375564</a>]<br></li>
<li>Comparison of transcriptome profiles between wild-type and pho B mutant strains Based on the results of the phoB and phoR expression proﬁles obtained in the wild-type strain under a P-limiting condition , the time point when PhoB was most activated was determined : the time between the exponential and stationary phases , just after phosphate was completely exhausted .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17020555" target="_blank">17020555</a>]<br></li>
<li>Alternatively , EnvZ-P might donate phosphate to an unidentiﬁed RR , whose action combines with that of PhoB P ( derived from acetyl P ) to activate PhoA transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15755952" target="_blank">15755952</a>]<br></li>
<li>PhoB was phosphorylated with acetyl phosphate solution ( 20 g of PhoB , 50 mM Tris-HCl [ pH 8 ] , 53 mM MgCl2 , 50 mM acetyl phosphate ) for 1 h at 37 ° C .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659677" target="_blank">15659677</a>]<br></li>
<li>They showed that during growth on pyruvate minimal medium , acetyl phosphate levels build up , causing PhoB to become phosphorylated and inducing Bap production ( 24 , 25 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18375564" target="_blank">18375564</a>]<br></li>
<li>On the basis of this finding , it was suggested that the normal function of CreC is to monitor changes in carbon supply and that cross-talk with PhoB reflects similarities between CreC and PhoR such that both serve as phosphate donors for PhoB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11350954" target="_blank">11350954</a>]<br></li>
<li>In the presence of a functional PhoR the crosstalk between PhoM and PhoB would be neither effective nor noticeable , since in the absence of phosphate the phosphorylation of PhoB by PhoR would greatly exceed that catalyzed by PhoM , and , in the presence of phosphate , the phosphorylation of PhoB by PhoM would be most readily counteracted by its dephosphorylation by PhoR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2574599" target="_blank">2574599</a>]<br></li>
<li>In E. coli , the PhoR -- PhoB two-component regulatory system senses the environmental phosphate concentration and responds to phosphate-starvation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17020555" target="_blank">17020555</a>]<br></li>
<li>When the external phosphate is limiting for growth , PhoR activates PhoB by phosphorylation ( 13 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1987150" target="_blank">1987150</a>]<br></li>
<li>Both the catabolite regulatory sensor CreC and acetyl phosphate activate the PhoB protein by phosphorylation in the absence of the PhoR protein ( 32 , 39 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8226621" target="_blank">8226621</a>]<br></li>
<li>In this case the modulator PhoR appears to activate the effector PhoB in the absence of phosphate and to inactivate it when phosphate is provided .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2574599" target="_blank">2574599</a>]<br></li>
<li>These include , for example , the osmotic response-regulator OmpR , [ 384 ] the phosphate response-regulator PhoB [ 394 ± 396 ] and the nitrogen response-regulator NRI , the ¯ agellar regulator [ 388 , 389 ] and the chemotaxis response-regulator , CheY [ 390 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10396600" target="_blank">10396600</a>]<br></li>
<li>Even though some response-regulator proteins , such as CheY , PhoB and OmpR , are known to be phosphorylated in vitro with acetyl phosphate ( Lukat et al. , 1992 ; McCleary & Stock , 1994 ) , phosphorylation in vitro of YedW by acetyl phosphate has not yet been proven .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25568260" target="_blank">25568260</a>]<br></li>
<li>Thus , studying the genome-wide regulation exercised by PhoB in response to phosphate-starvation is required to understand the underlying mechanisms of bacterial adaptation to phosphate-starvation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23071782" target="_blank">23071782</a>]<br></li>
<li>This suggests that proper adaptation and response to phosphate-starvation can be achieved by the high levels of PhoB and PhoR at the onset of phosphate exhaustion .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17020555" target="_blank">17020555</a>]<br></li>
<li>They proposed , and later proved , that this regulator is acetyl phosphate , which is a phosphoryl donor for PhoB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18375564" target="_blank">18375564</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_197">
, Wilmes-Riesenberg , M.R. , and Wanner , B.L. ( 1996 ) Involvement of the sensor kinase EnvZ in the in vivo activation of the response-regulator PhoB by acetyl phosphate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12366850" target="_blank">12366850</a>]<br></label>
<input id="act_similarity_197", type="checkbox">
<div class="ACT_similar">
<li>Involvement of the sensor kinase EnvZ in the in vivo activation of the response-regulator PhoB by acetyl phosphate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9286997" target="_blank">9286997</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15755952" target="_blank">15755952</a>]<br></li></div>
<li>cause it can act as a phosphate donor for PhoB , a response-regulator that controls expression of the pho regulon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11350954" target="_blank">11350954</a>]<br></li>
<li>DISCUSSION Expression of E. coli Genes at Eight Genetic Loci Is Subject to CreBC Control -- The E. coli CreBC TCR was discovered fortuitously because its signal sensor , CreC , can act as phosphate donor for the pho regulon response-regulator , PhoB , to activate transcription of phoA in mutants that lack the normal pho regulon signal sensor , PhoR ( 5 , 8 -- 10 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11350954" target="_blank">11350954</a>]<br></li>
<li>The phoR gene product modulates PhoB function in response to the phosphate concentrations in the medium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2676981" target="_blank">2676981</a>]<br></li>
<li>The sensor kinase of this machinery , PhoR , plays an important role to pass the limited environmental phosphate signal to its response-regulator , PhoB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23071782" target="_blank">23071782</a>]<br></li>
<li>The phospho-PhoR protein then functions as a protein kinase and transmits the phosphate limitation signal to the response-regulator PhoB , through phosphorylation of an aspartate residue .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10589831" target="_blank">10589831</a>]<br></li>
<li>The activity of PhoB as a transcriptional activater is modified by PhoR protein in response to the phosphate concentration in the medium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1987150" target="_blank">1987150</a>]<br></li>
<li>The sensor kinase ( SK ) PhoR responds to a decrease in the concentration of inorganic phosphate ( Pi ) , and under conditions of Pi starvation the protein autophosphorylates and then trans-phosphorylates the response-regulator ( RR ) , PhoB ( 15 , 25 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18375564" target="_blank">18375564</a>]<br></li>
<li>Phosphate starvation causes the sensor kinase PhoR to phosphorylate the response-regulator PhoB , which in turn activates phosphate-starvation genes ( 32 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659677" target="_blank">15659677</a>]<br></li>
<li>When the concentration of environmental phosphate is low ( generally , below 4 mM ) , PhoR ( sensor kinase ) senses it and activates PhoB ( response-regulator , transcriptional-activator ) by phosphor-ylation ( Wanner , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17020555" target="_blank">17020555</a>]<br></li>
<li>Results and Discussion Genome-wide mapping of PhoB binding profiles We applied the ChIP-chip techniques to measure the binding of PhoB across the whole genome under the phosphate-limiting condition ( Figure 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23071782" target="_blank">23071782</a>]<br></li>
<li>Our results unveiled the genome-wide binding profile of PhoB and the downstream responses under phosphate-starvation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23071782" target="_blank">23071782</a>]<br></li>
<li>This integrative genome-wide approach allowed us to identify 54 PhoB binding targets and 287 differentially expressed genes in the presence of PhoB activity during phosphate-starvation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23071782" target="_blank">23071782</a>]<br></li>
<li>Candidates for PhoP regulation in E. coli include the phosphate-starvation-inducible psiG , psiH , psil , psiJ , psiL , and psiO genes , which are also induced by starvation for carbon and nitrogen and do not seem to be under the control of PhoB , a PhoP homolog which regulates the expression of several phosphate-starvation loci ( 32 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1530848" target="_blank">1530848</a>]<br></li>
<li>There is a physiological rationale for the repression of feaR by PhoB and ArcA : O2 is required for the catab-olism of monoamines , and H2O2 is a threat to membrane integrity during phosphate-starvation ( 20 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24013633" target="_blank">24013633</a>]<br></li>
<li>E. coli W1485 induces Eda in response to phosphate limitation ( Fig. 3 ) , and the P4 promoter region is bound by PhoB in vitro ( Fig. 6B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659677" target="_blank">15659677</a>]<br></li>
<li>While the mechanism of Eda regulation by PhoB remains to be elucidated , it is clear that Eda is induced by phosphate limitation in a PhoB-dependent manner .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659677" target="_blank">15659677</a>]<br></li>
<li>The P1 promoter was induced by phosphate limitation and dependent on PhoB , while the P2 promoter was constitutive with respect to phosphate concentration in the medium ( Fig. 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8444794" target="_blank">8444794</a>]<br></li>
<li>Preliminary results from our laboratory suggest that ' PhoB does indeed bind specifically to the conserved phosphate box .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3537313" target="_blank">3537313</a>]<br></li>
<li>The data suggest that PhoB plays protective roles in membrane integrity and oxidative-stress reduction during phosphate-starvation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23071782" target="_blank">23071782</a>]<br></li>
<li>Overall , the data suggest that PhoB specifically regulates a relatively small group of genes , which influence a large group of downstream genes during phosphate-starvation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23071782" target="_blank">23071782</a>]<br></li>
<li>Under phosphate-replete conditions , unphosphorylated , monomeric PhoB may bind to its P4 operator to repress eda expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659677" target="_blank">15659677</a>]<br></li>
<li>Under phosphate excess conditions , XN-PhoB repressed XPR to 35 % , indicating that dephosphorylated XN-PhoB is predominantly a monomer .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7641688" target="_blank">7641688</a>]<br></li>
<li>Cells were grown either under phosphate-starvation conditions , where PhoB is phosphorylated , or under phosphate excess conditions , where PhoB is dephosphorylated ( Wanner , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7641688" target="_blank">7641688</a>]<br></li>
<li>P4 is active under conditions of phosphate-starvation and is directly controlled by PhoB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659677" target="_blank">15659677</a>]<br></li>
<li>As seen in Fig. 1E , the mRNA for phoA , a known target of PhoB regulation , is induced under our phosphate-starvation conditions and this induction is abolished in a phoB mutant , as expected .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17640895" target="_blank">17640895</a>]<br></li>
<li>The residual induction in the absence of ppGpp may reflect a modest effect of PhoB ( Fig. 1E ) and/or direct or indirect effects of phosphate-starvation on mRNA stability .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17640895" target="_blank">17640895</a>]<br></li>
<li>Phosphate starvation results in induction of P4 , which is directly mediated by PhoB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659677" target="_blank">15659677</a>]<br></li>
<li>Since Eda is apparently induced by phosphate-starvation ( Fig. 3 ) , we sought to determine if Eda synthesis is under PhoB control .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659677" target="_blank">15659677</a>]<br></li>
<li>From these results we conclude that the modest induction of Eda associated with phosphate-starvation is mediated by negative transcriptional control by PhoB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659677" target="_blank">15659677</a>]<br></li>
<li>Since all the genes identified as being under the regulation of the PhoB protein code for envelope proteins , the outer membrane ( phoE ) , periplasmic proteins ( phoA , phoS and ugpB ) , and inner membrane proteins ( pho W , p & A and p & B ) , and since the Dye protein regulates the synthesis of envelope proteins , all three regulons may be involved in a system by which the cell adapts to environmental change by modifying the envelope components .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3537313" target="_blank">3537313</a>]<br></li>
<li>B , BamHI ; H , HindIII ; M , MaeIII ANCS3 ( phoB63 ) ANCB1-5 ( AphoB ) Discussion We could selectively clone the promoter region of pstS by specific binding to the PhoB protein while those of phoA , phoB , and phoEwere not found among the 50 clones examined .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2651888" target="_blank">2651888</a>]<br></li>
<li>Portions of one helix recognize the 235 element and are preceded and followed by resi-dues that affect activation by proteins that bind near 235 , such as activation of lPRM by cI protein ( 24 ) , of the ada and aidB promoters by Ada ( 23 ) , of the pho regulon promoters by PhoB ( 21 ) , and of the phage Mu middle promoter by Mor protein ( 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10572130" target="_blank">10572130</a>]<br></li>
<li>Fluorescence anisotropic monitoring of protein-DNA interactions showed that the DBD of PhoB binds pho box DNA containing two half-sites with a 7-fold higher affinity ( 63 nM ) than the nonphosphorylated , full-length PhoB ( 440 nM ) [ 40 ] , whereas upon phosphorylation the affinity increases to 9.7 nM [ 56 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22291906" target="_blank">22291906</a>]<br></li>
<li>This results in the phosphorylation of PhoB via phospho-relay , causing an increase in its affinity for the `` pho box , '' a specific DNA binding motif , that is promoterproximal in all pho regulon genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11350954" target="_blank">11350954</a>]<br></li>
<li>It has been established that pho box DNA binding by E. coli PhoB is not dependent upon PhoB phosphorylation , but that phosphorylation increases the afﬁnity of PhoB for the pho box .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18375564" target="_blank">18375564</a>]<br></li>
<li>The D N A fragments isolated from plasmids pSD310 and pSD317 which contained two and one pho box , respectively , bound to PhoB protein ( lanes 2 , 4 ) , while that isolated from plasmid pSD320 , which had deleted both pho boxes from the 3 ' end up to - 6 3 , did not bind to PhoB ( lanes 5 , 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2651888" target="_blank">2651888</a>]<br></li>
<li>As to the upstream deletions , those isolated from pSD508 and pSD514 , which deleted up to - 6 9 and - 4 4 , and contained both pho boxes and only the downstream pho box , respectively , bound to PhoB ( lanes 8 , 10 ) , while that isolated from pSD520 , which deleted up to + 6 , and contained none of the pho box , did not bind to the protein ( lanes 11 , 12 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2651888" target="_blank">2651888</a>]<br></li>
<li>There is also an 18-bp consensus-like pho box around − 30 in the asr promoter and the response-regulator PhoB can bind the asr promoter in vitro [ 32 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15313253" target="_blank">15313253</a>]<br></li>
<li>Binding of PhoB to the pho box stimulates ( or in some cases represses ) transcription of the downstream gene ( 5 -- 7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11350954" target="_blank">11350954</a>]<br></li>
<li>PhoB has also been demonstrated to-bind to poor consensus pho box sequences leading to gene activation [ 6,14 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15313253" target="_blank">15313253</a>]<br></li>
<li>Similarly , binding of PhoB-P to the upstream PhoB box at the pstS promoter occurs at a lower concentration of PhoB-P than when it binds to the adjacent , downstream box ( 34 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25161193" target="_blank">25161193</a>]<br></li>
<li>Subsequently , the phosphorylated PhoB activates the transcription of at least 31 genes by binding to the Pho box in the promoter region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17020555" target="_blank">17020555</a>]<br></li>
<li>Deletions extending into the upstream pho box but retaining the downstream pho box greatly reduced promoter activity , but the remaining activity was still regulated by phosphate levels in the medium and by the PhoB protein , indicating that each pho box is functional , No activity was observed in deletion mutants which lacked the remaining pho box or the - 1 0 region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2651888" target="_blank">2651888</a>]<br></li>
<li>In the present work , we purified PhoB protein produced in a p & S mutant cell in which the pho regulon was expressed constitutively and , therefore , we expected the PhoB protein to be mostly in its active form .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3054125" target="_blank">3054125</a>]<br></li>
<li>According to a model for this circuit ( Tommassen & regulatory Lugtenberg , 1982 ; Tommassen et al. , 1982a ) , the PhoB protein functions as an activator of transcription of the structural genes of the pho regulon , whereas phoB expression in its turn is modulated by the products of phoR and phoM .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2828642" target="_blank">2828642</a>]<br></li>
<li>The phosphorylated PhoB protein then binds to the Pho box found within the promoters of genes of the Pho regulon , such as phnC , phoA , phoB , phoH and others ( Kasahara et al. 1991 ; Kim et al. 1993 ; Makino et al. 1986 , 1988 , 1991 ; Tommassen et al. 1987 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10589831" target="_blank">10589831</a>]<br></li>
<li>Methylation of the purine bases in the pho box region was either inhibited or enhanced by the phosphorylated PhoB protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1840580" target="_blank">1840580</a>]<br></li>
<li>PhoB protein may guide RNA polymerase to the pstS promoter by interacting with the pho box and activate initiation of pstS transcription specifically .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3054125" target="_blank">3054125</a>]<br></li>
<li>These results suggest that the PhoB protein binds to each pho box of pstS independently .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2651888" target="_blank">2651888</a>]<br></li>
<li>Each pho box of pstS was demonstrated to be independently bound by the PhoB protein ( Fig. 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2651888" target="_blank">2651888</a>]<br></li>
<li>Makino et al. ( 11 , 12 ) found a consensus sequence shared by the regulatory regions of phoA , phoB , phoE , and pstS which was named the pho box , and they showed that PhoB protein binds to the pho boxes of pstS and phoB and activates the transcription of pstS in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2676981" target="_blank">2676981</a>]<br></li>
<li>The activator protein PhoB binds the pho box in a way which activates transcription of the pho genes ( 14 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1987150" target="_blank">1987150</a>]<br></li>
<li>The results described in this paper show that PhoB protein binds to the pho box and activates transcription in Gtro from the site that reflects initiation in , uivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3054125" target="_blank">3054125</a>]<br></li>
<li>Recently , we purified the PhoB protein and showed that it binds to the consensus sequence for the regulatory regions of the genes of the phosphate regulon and activates transcription of these genes in vitro ( 14 ; unpublished results ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3531171" target="_blank">3531171</a>]<br></li>
<li>Our findings indicate that PhoB protein recognizes and binds to the pho box and activates transcription of the genes in the phosphate regulon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3054125" target="_blank">3054125</a>]<br></li>
<li>One of these sites overlaps the pho box , suggesting that IHF may inhibit phoA expression by binding to this sequence and interfering with the binding of the phosphorylated PhoB protein , the activator of the PHO genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9871104" target="_blank">9871104</a>]<br></li>
<li>To examine whether PhoB protein also binds to the pho boxes of other genes in the pho regulon , the PhoH protein binding to the regulatory region of phoB was studied by DNase T and methylation protection experiments .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3054125" target="_blank">3054125</a>]<br></li>
<li>PhoR phosphorylates the transcriptional-regulator PhoB , which then binds to a pho box consensus sequence within the promoter of genes in the pho regulon ( 46 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11489853" target="_blank">11489853</a>]<br></li>
<li>Upon Pi-starvation the PhoB protein becomes phosphorylated , binds to the pho box , interacts with the s70 subunit of RNA polymerase , and facilitates the transcription of the PHO regulon genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9871104" target="_blank">9871104</a>]<br></li>
<li>In its phosphorylated form , PhoB binds more tightly to the pho box DNA sequence and interacts with RNA polymerase to stimulate the transcription of a regulon of genes that encode products that have a collective role in raising the concentration of cellular Pi ( 7 -- 9 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18375564" target="_blank">18375564</a>]<br></li>
<li>Hence , the reduced afﬁnity of unphosphorylated PhoB for the pho box can be mitigated by the overproduction of PhoB , which activates pho regulon gene expression even in the absence of an active PhoR signal sensor ( 12 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18375564" target="_blank">18375564</a>]<br></li>
<li>In response to phosphate limitation , PhoB binds to the Pho box and transmits the phosphate-limiting signal to To date , thirty-one responding genes composed of nine transcription units are known to be regulated by PhoB , while several other genes lack direct evidence of PhoB binding in E. coli [ 9 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23071782" target="_blank">23071782</a>]<br></li>
<li>When phosphorylated , PhoB activates transcription of the PHO regulon , including phoA , which encodes bacterial alkaline phosphatase ( BAP ) ( 287 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15755952" target="_blank">15755952</a>]<br></li>
<li>This would mirror the known mode of action of PhoBR , in which phosphate-starvation leads to autophosphorylation of PhoR and phospho-relay to PhoB , which then binds to the pho box and activates or represses transcription of pho-regulon genes ( 5 , 7 , 8 , 26 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11350954" target="_blank">11350954</a>]<br></li>
<li>concentrations of phosphate are low , PhoB is phosphorylated by PhoR and activates the transcription of the Pho regulon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17640895" target="_blank">17640895</a>]<br></li>
<li>When the concentration of Pi in the medium decreases below a certain level , the sensor protein PhoR auto-phosphorylates and transfers the Pi group to the regulatory protein PhoB , which in turn binds to the PHO-boxes and allows transcription of the PHO genes by interacting with Es ( Wanner , 1996 ; Makino et al. , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15347756" target="_blank">15347756</a>]<br></li>
<li>When the concentration of Pi in the medium falls below 4 lM , the sensor protein PhoR phosphorylates PhoB , and the phosphorylated PhoB binds to the PHO-boxes , interacts with the r70 factor of RNA polymerase and initiates transcription ( Makino et al. 1993 ; Wanner 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12471449" target="_blank">12471449</a>]<br></li>
<li>Because the p & S gene product , the peri-plasmic phosphate-binding protein , is responsible for monitoring the levels of phosphate in t , hc medium and for the transduction of the phosphate signal to PhoB protein via several components of the signal transduction system ( Nakat ' a et al. , 1987 ; Shinagawa et al. , 1987 ) , a certain level of pstS expression should be maintained in the cell grown with excess phosphate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3054125" target="_blank">3054125</a>]<br></li>
<li>Since activation of phoB requires the PhoR protein or PhoM protein in addition to the PhoB protein ( Wanner & Latterell , 1980 ; Shinagawa et al. , 1983 ; Guan et al. ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3537313" target="_blank">3537313</a>]<br></li>
<li>Genetic and physiological studies suggest that PhoB protein has two forms , an active form or an inactive form , depending on the external levels of phosphate ( Shinagawa et al. , 1987 ; Wanner , 1987 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3054125" target="_blank">3054125</a>]<br></li>
<li>( 1988 ) 203 , 85-95 Regulation of the Phosphate Regulon f Escherichia coli o Activation by PhoB of pstS Transcription Protein in Vitro Kozo Makinot , Hideo Shinagawa , Mitsuko Amemura Sigenobu Kimura , Atsuo Nakata Department of Experimental Chemotherapy The Research Institute for Microbial Diseases Osaka University , 3-1 Yamadaoka , Suita , Osaka , Japan 565 and Akira Ishihama Department of Molecular Genetics National Institute of Genetics 1111 Yata , Mishima , Shizuoka , Japan 41 I ( Received 24 July 1987 , and in revised form 1 April 1988 ) Expression of the genes in the phosphate regulon , including the pstS ( phoS ) and phoH genes , is positively regulated by PhoB protein when phosphate is limited .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3054125" target="_blank">3054125</a>]<br></li>
<li>Makino , K. , Shinagawa , H. , Amemura , M. , Kawamoto , T. , Yamada , M. and Nakata , A. ( 1989 ) Signal transduction in the phosphate regulon of Escherichia coli involves phosphotransfer between PhoR and PhoB proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7641688" target="_blank">7641688</a>]<br></li>
<li>Lee , B.L. Wanner , K. Makino , Dual transcriptional regulation of the Escherichia coli phosphate-starvation-inducible psiE gene of the phosphate regulon by PhoB and the cyclic AMP ( cAMP ) -- cAMP receptor protein complex , J. Bacteriol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15313253" target="_blank">15313253</a>]<br></li>
<li>Autophosphorylation and activation of transcriptional-activator PhoB of Escherichia coli by acetyl phosphate in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15755952" target="_blank">15755952</a>]<br></li>
<li>Regulation of the phosphate regulon of Escherichia coli : activation of pstS transcription by PhoB protein in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10986267" target="_blank">10986267</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1987150" target="_blank">1987150</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8444794" target="_blank">8444794</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15755952" target="_blank">15755952</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1840580" target="_blank">1840580</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2676981" target="_blank">2676981</a>]<br></li>
<li>Makino , K. , Shinagawa , H. , Amemura , M. , Kimura , S. , and Nakata , A. ( 1988 ) Regulation of the phosphate regulon of Escherichia coil Activation of pstS transcription by PhoB protein in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596446" target="_blank">8596446</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_275">
J Mol Biol 190:37 ± 44 Makino K , Shinagawa H , Amemura M , Kimura S , Nakata A , Ishihama A ( 1988 ) Regulation of the phosphate regulon of Escherichia coli : activation of pstS transcription by PhoB protein in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10589831" target="_blank">10589831</a>]<br></label>
<input id="act_similarity_275", type="checkbox">
<div class="ACT_similar">
<li>J Mol Biol 190 : 37-44 Makino K , Shinagawa H , Amemura M , Kimura S , Nakata A , Ishihama A ( 1988 ) Regulation of the phosphate regulon of Eseheriehia coli : activation ofpstS transcription by PhoB protein in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2651888" target="_blank">2651888</a>]<br></li></div>
<li>The addition of PhoB protein to the invitro system containing RNA polymerase holo-enzyme increased `` faithful transcription '' but decreased initiation from the upstream site , which was not detected in viva .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3054125" target="_blank">3054125</a>]<br></li>
<li>By sequence homology , phoM was predicted to encode an SK protein , and it was later shown that PhoM can autophosphorylate and then trans-phosphorylate PhoB in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18375564" target="_blank">18375564</a>]<br></li>
<li>PhoB protein and PhoR1084 protein , which lacks the amino-terminal hydrophobic region and is constitutively active as a kinase ( 32 ) , were purified , and in vitro transcription was done essentially as described by Makino et al. ( 13 , 14 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1987150" target="_blank">1987150</a>]<br></li>
<li>We examined whether the phosphorylated PhoB protein activates transcription initiation of the ugp operon by RNA polymerase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1987150" target="_blank">1987150</a>]<br></li>
<li>Also , the PhoB protein activates transcription only when phosphorylated , and the PhoR protein is a PhoB protein kinase ( 15 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8226621" target="_blank">8226621</a>]<br></li>
<li>To elucidate the function of PhoR protein and to test the validity of our model , we purified PhoB protein and studied its interaction with the pstS gene in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3054125" target="_blank">3054125</a>]<br></li>
<li>We are preparing in vitro transcription experiments , using the phoS and the phoA genes cloned on the vectors as templates , to see the stimulatory effect of the PhoB protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6365894" target="_blank">6365894</a>]<br></li>
<li>Activation of transcription of the ugp operon by phosphor-ylated PhoB protein in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1987150" target="_blank">1987150</a>]<br></li>
<li>Activation of pstS transcription by PhoB protein in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12471449" target="_blank">12471449</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10094685" target="_blank">10094685</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14529615" target="_blank">14529615</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1943993" target="_blank">1943993</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7798138" target="_blank">7798138</a>]<br></li>
<li>Synthesis of PhoB protein was induced by adding 1 mM-isopropyl-p-r > - thiogalact , opyranoside to the culture grown in T broth to 0.5 unit of A , , , , and the cells were harvested after overnight incubation at 37 ° C with shaking .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3054125" target="_blank">3054125</a>]<br></li>
<li>In contrast to RutR , PhoB is a well-conserved protein and the phosphate-sensing mechanism is vital for survival ; thus PhoB is likely well-evolved and its bindings in the coding regions may have biological functions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23071782" target="_blank">23071782</a>]<br></li>
<li>The region containing thepho box was bound by PhoB protein , the transcriptional-activator of thepho regulon , as revealed by footprinting .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8444794" target="_blank">8444794</a>]<br></li>
<li>PhoB is a transcriptional-activator , and PhoR is a partner protein that activates PhoB when P , is limiting and inactivates PhoB when Pi is in excess ( 45 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2160940" target="_blank">2160940</a>]<br></li>
<li>The PhoB protein is the transcriptional-activator , and the PhoR protein is the As such , the PhoR protein has dual Pi sensor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8226621" target="_blank">8226621</a>]<br></li>
<li>Under conditions of Pi limitation , the histidine kinase PhoR autophosphorylates and transfers the P moiety to the regulatory i protein PhoB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25645557" target="_blank">25645557</a>]<br></li>
<li>promoters by the PhoB and PhoRA proteins during Pi limitation is controlled by the amount of phospho-PhoB protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8226621" target="_blank">8226621</a>]<br></li>
<li>Although the molecular mechanisms of this process are largely unknown , the final outcome seems to be activation or deactivation of PhoB protein as the activator , depending on the scarcity or abundance of external phosphate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2676981" target="_blank">2676981</a>]<br></li>
<li>PhoB is known to be a transcriptional-activator of the Pho regulon , expression of which is activated during phosphate-starvation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10589831" target="_blank">10589831</a>]<br></li>
<li>The phoR product is a pro-tein-histidine kinase , which when autophosphorylated ( at phosphate-starvation ) phosphorylates PhoB , a transcriptional-activator of genes belonging to the pho regulon ( 34 , 35 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10094685" target="_blank">10094685</a>]<br></li>
<li>Responding to phosphate limitation , the phoR gene product undergoes autophosphorylation with ATP and acts as a protein kinase for the activator protein PhoB ( 22 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8444794" target="_blank">8444794</a>]<br></li>
<li>Derepression of the proteins of the pho regulon during phosphate limitation is associated with increased synthesis of the PhoB activator protein ( 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3042748" target="_blank">3042748</a>]<br></li>
<li>External phosphate levels are perceived by phosphatebinding protein , and the signal is transmitted by the functions of the products of the Pst-phoU operon ( 17 ) and the phoR gene to PhoB protein , which is the direct transcriptional-activator for the genes in the pho regulon , including the phoB-phoR operon ( 17 , 26 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2676981" target="_blank">2676981</a>]<br></li>
<li>The highly conserved region in this vicinity may be important for receiving environmental signals because the mutant is insensitive to the concentration of phosphate and therefore the PhoR69 protein is considered to activate PhoB constitutively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2676981" target="_blank">2676981</a>]<br></li>
<li>The inability to detect the corresponding transcript in uivo does not exclude the possibility that it is synthesized at a low level in the cell grown with excessphosphate and that , as in the in-vitro case , it is repressed by PhoB protein in the cell grown with limited phosphate , because expression of p & S is induced several hundredfold by phosphate-starvation ( unpublished results ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3054125" target="_blank">3054125</a>]<br></li>
<li>The 229-amino-acid ORF2 protein is highly homologous to the PhoB protein , a positive regulatory protein for the phosphate regulon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3531171" target="_blank">3531171</a>]<br></li>
<li>The downstream promoter was induced by phosphate-starvation and requires the PhoB protein for its activation as do the other pho regulon promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1987150" target="_blank">1987150</a>]<br></li>
<li>The pstS gene belongs to the phosphate regulon whose expression is induced by phosphate-starvation and regulated positively by the PhoB protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2651888" target="_blank">2651888</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_304">
The ugp operon is induced by phosphate-starvation and positively regulated by PhoB protein , as are other genes in the pho regulon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1987150" target="_blank">1987150</a>]<br></label>
<input id="act_similarity_304", type="checkbox">
<div class="ACT_similar">
<li>DISCUSSION The ugp operon is known to be induced by phosphate-starvation and regulated positively by PhoB protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1987150" target="_blank">1987150</a>]<br></li></div>
<li>Strong bands of the ugpl transcript were observed only in the reaction mixture with phosphorylated PhoB protein added ( Fig. 5 , lane 3 ) , and weak bands of the ugp2 transcript were found only in the mixture without PhoB protein ( Fig. 5 , lane 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1987150" target="_blank">1987150</a>]<br></li>
<li>Therefore , phosphorylated PhoB protein greatly stimulated the ugp transcription initiated from the ugpl site , which is the same as the start site of the transcript found in the pho constitutive strain ( Fig. 4 ) , and appeared to inhibit the ugp2 transcription ( Fig. 5 , lane 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1987150" target="_blank">1987150</a>]<br></li>
<li>These results indicate that activation of the iciA gene is dependent upon the PhoB protein Fig. 1A ± C PhoB-dependent activation of iciA expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10589831" target="_blank">10589831</a>]<br></li>
<li>The labeled DNA was treated with dimethyl sulfate in the absence ( lanes 1 ) or presence ( lanes 2 ) of phosphorylated PhoB and then subjected to the Maxam-Gilbert G > A reaction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8444794" target="_blank">8444794</a>]<br></li>
<li>The DNA fragment was methylated in the absence ( lane 2 ) or in the presence ( lane 3 ) of PhoB protein ( 16 pmol ) , and subjected to the Maxam-Gilbert G reaction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3054125" target="_blank">3054125</a>]<br></li>
<li>DNA fragments were methylated or digested in the presence ( + ) or in the absence ( - ) of PhoB protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1987150" target="_blank">1987150</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1987150" target="_blank">1987150</a>]<br></li>
<li>Binding of PhoB protein to DNA fragments was measured by the method of Singh et al. ( 1986 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2651888" target="_blank">2651888</a>]<br></li>
<li>Ex Pi , excess phosphate ; Li Pi , Limiting phosphate b The plasmid containing the 959 bp pstS promoter region Binding of the PhoB protein to DNA fragments containing the pho box Binding of PhoB to the promoter region covering the pho boxes in pstS was demonstrated by footprinting experiments ( Makino et al. 1988 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2651888" target="_blank">2651888</a>]<br></li>
<li>To further examine if these eight sites are directly bound by PhoB , we used the gel mobility shift assay to detect the protein-DNA interactions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23071782" target="_blank">23071782</a>]<br></li>
<li>The positions of PhoB protein are marked by arrowheads .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3054125" target="_blank">3054125</a>]<br></li>
<li>PhoB protein appeared to slightly repress Pugp2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1987150" target="_blank">1987150</a>]<br></li>
<li>So PhoB protein was added in lanes 1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3054125" target="_blank">3054125</a>]<br></li>
<li>Assay of PhoB protein binding to DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2651888" target="_blank">2651888</a>]<br></li>
<li>Therefore , the pstS promoter region was highly selected in the DNA population bound to the PhoB protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2651888" target="_blank">2651888</a>]<br></li>
<li>The asr promoter DNA region was demonstrated to-bind PhoB protein in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10094685" target="_blank">10094685</a>]<br></li>
<li>Since ackA overexpression also increased the intracellular ATP pool , the authors proposed that elevated ATP levels might facilitate autophosphorylation of a modulator protein that could cross talk to PhoB ( 264 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15755952" target="_blank">15755952</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_322">
Amemura M , Makino K , Shinagawa H , Nakata A ( 1990 ) Cross talk to the phosphate regulon of Escherichia coli by PhoM protein : PhoM is a histidine protein kinase and catalyzes phosphorylation of PhoB and PhoM-open reading frame 2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24062463" target="_blank">24062463</a>]<br></label>
<input id="act_similarity_322", type="checkbox">
<div class="ACT_similar">
<li>Cross talk to the phosphate regulation of Escherichia coli by PhoM protein : PhoM is a histidine protein kinase and catalyzes phosphorylation of PhoB and PhoM-open reading frame 2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11325944" target="_blank">11325944</a>]<br></li>
<li>Cross talk to the phosphate regulon of Escherichia coli by PhoM protein : PhoM is a histidine protein kinase and catalyzes phosphorylation of PhoB and PhoM opon reading frame 2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15755952" target="_blank">15755952</a>]<br></li>
<li>Cross talk to the phosphate regulon of Escherichia coli PhoM protein : PhoM is a histidine protein kinase and catalyzes phosphorylation of PhoB and PhoM-open reading frame 2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8550505" target="_blank">8550505</a>]<br></li>
<li>Cross talk to the phosphate regulon of Escherichia coli by PhoM protein : PhoM is a histidine protein kinase and catalyzes phosphorylation of PhoB and PhoM-open reading frame 2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18375564" target="_blank">18375564</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11004187" target="_blank">11004187</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9286997" target="_blank">9286997</a>]<br></li></div>
<li>In these experiments , PhoR protein and ATP were added to the reaction mixture for phosphorylation of PhoB protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1987150" target="_blank">1987150</a>]<br></li>
<li>Therefore , the PhoRA protein is probably a kinase that activates the PhoB protein by phosphorylation , and the PhoRR protein might act as a protein phosphatase that inactivates the phospho-PhoB protein by dephosphorylation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8226621" target="_blank">8226621</a>]<br></li>
<li>PhoR is the histidine protein kinase that activates PhoB by phosphorylation and ( presumably ) inactivates phosphorylated PhoB ( phospho-PhoB ) by dephosphorylation in response to the extracellular phosphate concentration ( 14 , 17 , 22 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10986267" target="_blank">10986267</a>]<br></li>
<li>Cross-talk between the histidine protein kinase VanS and the response-regulator PhoB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11325944" target="_blank">11325944</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11053370" target="_blank">11053370</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9286997" target="_blank">9286997</a>]<br></li>
<li>These genes are transcriptionally activated by the PhoB protein , a response-regulator of the Pho regulon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10589831" target="_blank">10589831</a>]<br></li>
<li>Signal is further transduced to an activator protein ( PhoB ) or to another regulatory factor , produced by the cell in response to acidiﬁcation , or both .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10094685" target="_blank">10094685</a>]<br></li>
<li>We thus examined whether or not the positive regulator ( PhoB ) , which is responsible for induction of the so-called E. coli pfio regulon including the phoA gene ( Wanner , 1987 ) , is involved in this particular response of CbpA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7997164" target="_blank">7997164</a>]<br></li>
<li>Given that PhoB and CreB are highly similar ( 25 ) , we next tested whether CreB overproduction in E. coli DH5 would stimulate cre regulon gene expression even during aerobic-growth-on-glucose , when the CreBC TCS is not activated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18375564" target="_blank">18375564</a>]<br></li>
<li>On the contrary , low extracellular Pi concentrations reduce transport by Pst , which increases the kinase activity of PhoR , resulting in the accumulation of phosphorylated PhoB ( PhoB-P ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14526009" target="_blank">14526009</a>]<br></li>
<li>Under low-Pi conditions PhoR autop-hosphorylates and transfers the Pi moiety to PhoB ( Wanner 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20963440" target="_blank">20963440</a>]<br></li>
<li>Under conditions of Pi excess , PhoR dephosphorylates PhoB , thereby ending the induced expression of the PHO regulon ( 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25645557" target="_blank">25645557</a>]<br></li>
<li>Under excess-Pi conditions , PhoR phosphatase activity predominates , it dephosphorylates PhoB thereby preventing transcription of the PHO genes ( Carmany et al. 2003 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20963440" target="_blank">20963440</a>]<br></li>
<li>For example , the HK PhoR senses environmental Pi ( directly or indirectly ) , is autophosphorylated and transfers the phosphoryl group to the RR PhoB for activation of the Pho regulon ( Wanner , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12366850" target="_blank">12366850</a>]<br></li>
<li>When levels of extracellular phosphate are depleted , PhoR , the sensor , becomes autophosphorylated at a histidine resi-due , and in turn phosphorylates an aspartate residue in the cytoplasmic regulator PhoB , which ® nally binds to Pho boxes located within the promoters of genes of the Pho regulon , causing transcriptional activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10589831" target="_blank">10589831</a>]<br></li>
<li>PhoB and PhoR regulate genes such as phoA and phn ( psiD ) in the PHO regulon for P assimilation in response to environmental Pi levels .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2160940" target="_blank">2160940</a>]<br></li>
<li>The expression of the E. coli Pho regulon is activated under conditions of Pi limitation , a process requiring PhoB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7592415" target="_blank">7592415</a>]<br></li>
<li>The phn gene cluster has a role in P assimilation because its expression is induced during Pi limitation in a manner that requires the PHO regulon regulatory proteins , PhoB and PhoR ( 54 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1846145" target="_blank">1846145</a>]<br></li>
<li>The transcription of the psi genes involved in phosphorus assimilation is regulated by the PhoB and PhoR proteins ( 27 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2824439" target="_blank">2824439</a>]<br></li>
<li>Consequently , unphosphorylated PhoB can not activate the transcription of the Pho regulon , a group of genes that encode proteins involved in scavenging and metabolizing phosphorus sources other than Pi .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14526009" target="_blank">14526009</a>]<br></li>
<li>Thus , our experiments show that genomic pitB encodes a functional Pi transporter repressed/inactivated by the pho regulon , either directly through PhoB or via another pathway controlled by the pho regulon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11489853" target="_blank">11489853</a>]<br></li>
<li>PhoB is a monomer and dimerizes upon phosphorylation We next examined the dimerization properties of PhoB , the transcriptional-activator for the pho regulon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7641688" target="_blank">7641688</a>]<br></li>
<li>Phosphorylated PhoB is the activator for the pho regulon ( Wanner , 1993 ) and belongs to the OmpR subfamily of transcriptional-activators for a70-dependent promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7641688" target="_blank">7641688</a>]<br></li>
<li>A negative growth phenotype is expected because PhoB is the transcriptional-activator of Pho regulon promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7592415" target="_blank">7592415</a>]<br></li>
<li>( PhoB is a required transcriptional-activator for genes in the PHO regulon [ 30 , 52 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1846145" target="_blank">1846145</a>]<br></li>
<li>phoR68 mutant background ) it can regulate the production of Bap through trans-phosphorylating PhoB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18375564" target="_blank">18375564</a>]<br></li>
<li>To measure CreC activation , they exploited the ability of active CreC to trans-phosphorylate PhoB ( and so stimulate Bap production ) in a phoR68 mutant background .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18375564" target="_blank">18375564</a>]<br></li>
<li>Truncations of VanS that inhibit PhoB activation by functional VanS appeared to form inactive heterodimers with VanS rather than to inhibit PhoB directly ( 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11053370" target="_blank">11053370</a>]<br></li>
<li>Mechanism of activation for transcription factor PhoB suggested by different modes of dimerization in the inactive and active states .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17005198" target="_blank">17005198</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18375564" target="_blank">18375564</a>]<br></li>
<li>Under other conditions , the HK CreC detects a signal ( as yet unknown ) , is autophosphorylated and transfers the phosphoryl group to both its partner RR CreB and PhoB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12366850" target="_blank">12366850</a>]<br></li>
<li>As proposed by Wanner and colleagues , it is conceivable that EnvZ or some other HK senses acetyl P , autophosphorylates in response using ATP as its phosphoryl donor , and then donates that phosphoryl group to PhoB ( 238 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15755952" target="_blank">15755952</a>]<br></li>
<li>They proposed two alternative mechanisms for acetyl P-dependent activation of PhoB : ( i ) indirect activation through a HK that senses acetyl P or ( ii ) direct activation , most probably by phosphor-ylation ( 464 , 466 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15755952" target="_blank">15755952</a>]<br></li>
<li>In all cases , EnvZ-dependent activation of PhoB required the absence of OmpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15755952" target="_blank">15755952</a>]<br></li>
<li>Clearly , some inconsistencies remain , the most pressing being the observation that the acetyl P-dependent activation o PhoB depends on EnvZ , at least under certain environmental conditions ( 238 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15755952" target="_blank">15755952</a>]<br></li>
<li>Whereas acetyl phosphate may , under certain conditions , act by phosphorylating PhoB directly ( that is in the absence of an apparent or known HK ; Wanner and Wilmes-Riesenberg , 1992 ) , the apparent phosphorylation of PhoB by EnvZ requires acetyl phosphate ( Kim et al. , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12366850" target="_blank">12366850</a>]<br></li>
<li>Finally ; EnvZ , in the absence of OmpR , might indirectly increase the acetyl P pool to a level that increases the phosphorylation of PhoB via acetyl P , a scenario that resembles the connection between GS and acetyl P ( 322 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15755952" target="_blank">15755952</a>]<br></li>
				</div></ul><br>
			<label class="collapse_DOM" for="_2"><font size="5px">PhoB protein domains</font></label>
				<input id="_2" type="checkbox">
				<div class="DOM"><br><ul><li>Residues involved in interactions of CheY with FliM ( 209 , 213 , 216 ) , CheA ( 214 , 215 ) , and CheZ ( 212 , 213 , 216 ) are shown in magenta ; interactions of Spo0F with KinA and Spo0B ( 217 ) in cyan ; interactions of PhoB with PhoR ( 218 ) and a proposed dimerization surface of PhoB ( 157 ) in gold ; the domain interface of NarL ( 158 ) in red ; and the domain interface of CheB ( 159 ) in blue .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10966457" target="_blank">10966457</a>]<br></li>
<li>inactivated the The phoB23 mutation , which completely PhoB function ( Table 3 ) , had a change in the ninth amino acid , where the negatively charged amino acids Glu or Asp are conserved , to an oppositely charged amino acid , Lys , suggesting that a negatively charged amino acid in this position may be essential for the function .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2676981" target="_blank">2676981</a>]<br></li>
<li>A search using program PLsearch and alignment using program LOCAL revealed a second domain of about 120 amino acid residues in the C-terminal region that shows homology with the regulator proteins of two-com-ponent systems ( Fig. 6 ) , such as PhoB -LCB- Makino ef ai , 1986a ) , ArcA ( Dye ) ( Drury and Buxton , 1985 ) , Orf2 of the phoM operon -LCB- Amemura ef ai .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2201868" target="_blank">2201868</a>]<br></li>
<li>After alignment is optimized by adding a maximum of two gaps in any one sequence , 36 % of the residues of KdpE are identical to those of PhoB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1532388" target="_blank">1532388</a>]<br></li>
<li>Although , KdpE does not contain identical residues ( it is Y191 , I194 , H198 and T217 respectively at positions equivalent to those of PhoB shown in the previous line ) , the properties of three side chains are similar except for KdpE T217 and PhoB R219 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22291906" target="_blank">22291906</a>]<br></li>
<li>This phospho-PhoR dimer has the kinase activity that can transfer the two phosphoryl groups to the aspartate residue in each of the PhoB monomers [ 1 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23071782" target="_blank">23071782</a>]<br></li>
<li>When levels of extracellular phosphate are depleted , PhoR , the sensor , becomes autophosphorylated at a histidine resi-due , and in turn phosphorylates an aspartate residue in the cytoplasmic regulator PhoB , which ® nally binds to Pho boxes located within the promoters of genes of the Pho regulon , causing transcriptional activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10589831" target="_blank">10589831</a>]<br></li>
<li>Unfortunately , this argument is based on studies of PhoB and CheY , two RRs that appear unresponsive to acetyl P in the presence of their cognate HKs .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15755952" target="_blank">15755952</a>]<br></li>
<li>Mechanism of activation for transcription factor PhoB suggested by different modes of dimerization in the inactive and active states .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17005198" target="_blank">17005198</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18375564" target="_blank">18375564</a>]<br></li>
<li>Several recent studies demonstrate in vivo activation of the response regulators , PhoB ( Lee etai .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7934904" target="_blank">7934904</a>]<br></li>
<li>In all cases , EnvZ-dependent activation of PhoB required the absence of OmpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15755952" target="_blank">15755952</a>]<br></li>
<li>The activation of PhoB by acetylphosphate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15755952" target="_blank">15755952</a>]<br></li>
<li>In class I ( e.g. NtrC and PhoB ) , this interaction leads to the inhibition of the receiver module .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7641688" target="_blank">7641688</a>]<br></li>
<li>We suggest that NtrC and PhoB represent class of one response regulators , where activation is mediated by dimerization of the phosphorylated receiver modules .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7641688" target="_blank">7641688</a>]<br></li>
<li>A common switch in activation of the response regulators NtrC and PhoB : phosphorylation induces dimerization of the receiver modules During signal transduction , response regulators of two-component systems are phosphorylated in a conserved receiver module .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7641688" target="_blank">7641688</a>]<br></li>
				<label class="DOM_collapse_similarity" for="dom_similarity_16">
A common switch in activation of the response regulators NtrC and PhoB : phosphorylation induced dimerization of receiver modules .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9973351" target="_blank">9973351</a>]<br></label>
<input id="dom_similarity_16", type="checkbox">
<div class="DOM_similar">
<li>A common switch in activation of the response regulators NtrC and PhoB : phosphorylation induces dimerization of the receiver modules .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659677" target="_blank">15659677</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8981978" target="_blank">8981978</a>]<br></li></div>
<li>The receiver-domains of NtrC and PhoB are dimerization domains We have shown that the unphosphorylated receiver-domains of NtrC and PhoB are unable to dimerize when attached to their cognate output domains : unphosphoryl-ated XN-PhoB and XN-NtrCRO , which both carry the receiver and output domains , are monomeric proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7641688" target="_blank">7641688</a>]<br></li>
<li>Discussion Our results establish that the receiver modules of NtrC and PhoB are dimerization domains .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7641688" target="_blank">7641688</a>]<br></li>
<li>Analysis of these hybrid proteins shows that the receiver modules of NtrC and PhoB are potential dimerization domains .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7641688" target="_blank">7641688</a>]<br></li>
<li>We show that the receiver modules of NtrC and PhoB potential dimerization are domains .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7641688" target="_blank">7641688</a>]<br></li>
<li>In EnvZ and PhoB , the phosphatase activity has been found to reside in the dimerization and histidine phosphorylation domain ( H1 in RcsC ) ( 14 , 119 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16153174" target="_blank">16153174</a>]<br></li>
<li>We infer that PhoB dimerizes upon phosphorylation , as does XN-NtrCRO lacking the C-terminal constitutive dimerization domain of NtrC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7641688" target="_blank">7641688</a>]<br></li>
<li>In contrast to fusion proteins carrying receiver and output domain of NtrC or PhoB , the isolated receiver modules in both XN-NtrCR and XN-PhoBR , but not the fusion protein XN-NtrCo carrying only the output domain of NtrC , behaved as constitutive dimers independent of phosphorylation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7641688" target="_blank">7641688</a>]<br></li>
<li>The unphosphorylated receiver-domain of PhoB silences the activity of its output domain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18375564" target="_blank">18375564</a>]<br></li>
<li>Plasmid pgln31 ( Weglenski et al. , 1989 ) was used as a template for the NtrC domains , plasmid pHA3 ( Hartmann and Boos , 1993 ) as a template for the PhoB domains and E.coli chromosomal DNA for the amplification of the receiver-domain of CheB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7641688" target="_blank">7641688</a>]<br></li>
<li>The isolated receiver-domain of PhoB is a constitutive dimer independent of phosphorylation Next we determined the oligomerization state of XN-PhoBR , the fusion protein consisting of the DNA-binding domain of X repressor and the receiver-domain of PhoB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7641688" target="_blank">7641688</a>]<br></li>
<li>We fused the DNA-binding domain of X repressor to the entire PhoB ( kN-PhoB ) and to the receiver-domain of PhoB ( XN-PhoBR ) ( Figure 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7641688" target="_blank">7641688</a>]<br></li>
<li>We fused various domains of the response regulators NtrC , PhoB or CheB to the DNA-binding domain of X repressor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7641688" target="_blank">7641688</a>]<br></li>
<li>Using the N-terminal-DNA-binding-domain of X repressor as a reporter domain for dimerization in vivo , examined we the interaction between receiver and output domains of NtrC , PhoB and CheB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7641688" target="_blank">7641688</a>]<br></li>
<li>Based on the crystal structures of the PhoB N-terminal domain ( Sola et al. , 1999 ) , the PhoB Cterminal domain ( Okamura et al. , 2000 ) and the OmpR C-terminal domain ( Martinez-Hackert and Stock , 1997b ) , together with the sequence similarity among PhoB , OmpR and PhoP , we could predict the domain organization for PhoP ( Fig. 9A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12123454" target="_blank">12123454</a>]<br></li>
<li>Sola , M. , Gomis-Ruth , F.X. , Serrano , L. , Gonzalez , A. , and Coll , M. ( 1999 ) Three-dimentional crystal structure of the transcription factor PhoB receiver-domain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12123454" target="_blank">12123454</a>]<br></li>
<li>Three-dimensional crystal structure of the transcription factor PhoB receiver-domain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10960104" target="_blank">10960104</a>]<br></li>
<li>In the case of PhoB , the loop b5a5 is involved in PhoB dimerization ( Sola et al. , 1999 ) , and the a5 helix contacts with the C-terminal domain of PhoB ( Allen et al. , 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12123454" target="_blank">12123454</a>]<br></li>
<li>Allen , M.P. , Zumbrennen , K.B. , and McCleary , W.R. ( 2001 ) Genetic evidence that a5 helix of the receiver-domain of PhoB is involved in interdomain interactions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12123454" target="_blank">12123454</a>]<br></li>
<li>In the sense strand , PhoB protein protected the DNA segment spanning nucleo-tides -20 to -101 from DNase I digestion and G residues at positions -28 , -44 , -53 , -57 , -61 , -66 , -68 , -72 , -76 , -79 , -85 , -88 , and -90 from methylation , and it enhanced the methylation of G residues at -65 , -84 , and -87 ( Fig. 6A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1987150" target="_blank">1987150</a>]<br></li>
<li>In the antisense strand , PhoB protein protected the DNA segment spanning -14 to -100 from the nuclease digestion , and G and A residues at positions -19 , -26 , -37 , -48 , -70 , -81 , -92 , and -95 from methylation , and it enhanced the methylation of the G residue at position -86 ( Fig. 6B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1987150" target="_blank">1987150</a>]<br></li>
<li>Two of these residues ( E575 and E591 ) are also involved in the interaction with the activator proteins PhoB and cI , respectively ( 19 , 35 ) , suggesting that these amino acids might be surface exposed and accessible to different activators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11073893" target="_blank">11073893</a>]<br></li>
<li>Several amino acids in both region 4.2 and in the basic cluster have been found to be important for interaction with transcription activators , such as PhoB , bacteriophage l cI protein , and FNR ( 29 , 30 ) .2 A set of substitutions between amino acids 570 and 580 of s70 severely impairs transcription-activation by PhoB at the pstS promoter ( 31 ) and also affects activation by cI at l PRM promoter ( 30 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9582376" target="_blank">9582376</a>]<br></li>
<li>One of the residues important for meAda-s70 interaction , glutamic acid at position 575 ( Glu-575 ) , also plays a role in transcription-activation by PhoB ( 29 ) , suggesting that the location of this residue allows interaction with different transcription activators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9582376" target="_blank">9582376</a>]<br></li>
<li>Among other proteins which do not require contact site I ( 5 ) , PhoB at the pstS promoter , the cyclic AMP receptor protein at the gal promoter , and l cI at the PRM promoter interact at contact site II , which lies within the s subunit of the RNA polymerase holoenzyme ( 10 , 12 , 14 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7798138" target="_blank">7798138</a>]<br></li>
<li>Mutational analysis of the role of the ﬁrst helix of region 4.2 of the s70 subunit of Escherichia coli RNA polymerase in transcriptional activation by activator protein PhoB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10572130" target="_blank">10572130</a>]<br></li>
				<label class="DOM_collapse_similarity" for="dom_similarity_43">
J Mol Biol 203:85 -- 95 Makino K , Amemura M , Kim SK , Nakata A , Shinagawa H ( 1993 ) Role of the sigma 70 subunit of RNA polymerase in transcriptional activation by activator protein PhoB in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12471449" target="_blank">12471449</a>]<br></label>
<input id="dom_similarity_43", type="checkbox">
<div class="DOM_similar">
<li>J Bacteriol 188:8496 -- 8503 Makino K , Amemura M , Kim SK , Nakata A , Shinagawa H ( 1993 ) Role of the sigma 70 subunit of RNA polymerase in transcriptional activation by activator protein PhoB in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20963440" target="_blank">20963440</a>]<br></li></div>
<li>Role of the ar70 subunit of RNA polymerase in transcriptional activation by activator protein PhoB in Esch-erichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8468304" target="_blank">8468304</a>]<br></li>
<li>Role of the sigma 70 subunit of RNA polymerase in transcriptional activation by activator protein PhoB in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15347756" target="_blank">15347756</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9512708" target="_blank">9512708</a>]<br></li>
<li>For instance , transcriptional activation by OmpR involves interaction with the α subunit of RNA polymerase ( 171 , 172 ) , whereas PhoB interacts with σ 70 ( 173 , 174 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10966457" target="_blank">10966457</a>]<br></li>
<li>Transcriptional activation by phospho-PhoB involves pro-tein-protein interaction with the s70 subunit of RNA polymer-ase holoenzyme ( 13 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10986267" target="_blank">10986267</a>]<br></li>
<li>The interaction of phosphorylated PhoB with this region was also examined by methylation protection experiments ( Fig. 6B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1840580" target="_blank">1840580</a>]<br></li>
<li>Therefore , these Gs may be very important for interaction with phosphorylated PhoB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8444794" target="_blank">8444794</a>]<br></li>
<li>Where do PhoB and cAMP-CRP bind ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10986267" target="_blank">10986267</a>]<br></li>
<li>How does RNA polymerase interact with the promoter ( s ) in the presence or absence of PhoB and/or cAMP-CRP ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10986267" target="_blank">10986267</a>]<br></li>
<li>Anti-PhoB antibody speciﬁcally interacts with the protein released from the asr DNA-protein complex C1 ( Fig. 8A , lanes 1 to 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10094685" target="_blank">10094685</a>]<br></li>
<li>Gel mobility shift assays with a DNA probe containing the eda P4 promoter region conﬁrmed binding by puriﬁed , phosphorylated PhoB ( Fig. 7B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659677" target="_blank">15659677</a>]<br></li>
<li>PhoB protein-binding site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1840580" target="_blank">1840580</a>]<br></li>
<li>It remains to be seen whether this region of the PhoB protein also has such a comformation and is a DNA-binding domain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3537313" target="_blank">3537313</a>]<br></li>
<li>In the 3D structure of OmpR and PhoB , a protruding loop located between the two helices proved to be part of the RNA-polymerase-binding region ( 28 , 31 , 32 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15197250" target="_blank">15197250</a>]<br></li>
<li>Proc Natl Acad Sci USA 48:1230 -- 1237 Makino K , Amemura M , Kawamoto T , Kimura S , Shinagawa H , Nakata A , Suzuki M ( 1996 ) DNA-binding of PhoB and its interaction with RNA polymerase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16404567" target="_blank">16404567</a>]<br></li>
<li>PhoB protein may consist of three functional domains , a regulatory domain with which PhoR protein interacts , a DNA-binding domain and a domain fo interaction with RNA polymerase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3054125" target="_blank">3054125</a>]<br></li>
<li>DNA-binding of PhoB and its interaction with RNA polymerase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25161193" target="_blank">25161193</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11976297" target="_blank">11976297</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10648521" target="_blank">10648521</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15347756" target="_blank">15347756</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25645557" target="_blank">25645557</a>]<br></li>
<li>The interaction or cumulative action of PhoB protein and the putative factor lead to asr transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10094685" target="_blank">10094685</a>]<br></li>
<li>The reaction mixtures containing a linear DNA template ( Fig. 1C , line d ) , PhoR1084 protein , ATP , and RNA polymerase , with or without PhoB protein , were incubated at 37 ° C for 10 min to form a transcription-initiation complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1987150" target="_blank">1987150</a>]<br></li>
<li>The phosphorylated PhoB binds to the PHO boxes , where it interacts with E 70 ( 70 associated with the core RNA polymerase ) ( 7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25645557" target="_blank">25645557</a>]<br></li>
<li>Activation of in-vitro transcription by PhoB protein required both the normal sigma factor ( a7 ' ) and core RNA polymerase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3054125" target="_blank">3054125</a>]<br></li>
<li>The C-terminal domain of s70 ( sCTD ) contains a putative helix-turn-helix motif ( region 4.2 , residues 570 to 613 ) which recognizes the 235 promoter element ( 15 ) and some of the transcription activators that bind near the 235 region , including l cI , PhoB , Ada , cyclic AMP ( cAMP ) receptor protein ( CAP ) -- activating region-3 ( AR3 ) , FNR , and phage Mu Mor proteins ( 1 , 21 , 23 -- 25 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10572130" target="_blank">10572130</a>]<br></li>
<li>Speciﬁc surfaces of the RNAP components RpoD ( 70 ) or the N-terminal domain of RpoA are sites of functional contact by some transcription factors that bind near the 35 promoter region , such as FNR , PhoB , or cI protein ( 25 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11976297" target="_blank">11976297</a>]<br></li>
<li>Footprinting experiments suggest t , hat the pstS promoter contains two PhoB binding sites that cover the proposed pho boxes , and that the phoB promoter contains one PhoB binding site that ccvers the pho box .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3054125" target="_blank">3054125</a>]<br></li>
<li>The region from nucleo-tide 12 to nucleotide 55 , which contains the putative pho box , was protected by phosphorylated PhoB protein from DNase I digestion .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1840580" target="_blank">1840580</a>]<br></li>
<li>Footprinting experiments showed the region with which PhoB protein actually interacts ( Fig. 6 and 7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1987150" target="_blank">1987150</a>]<br></li>
<li>The results obtained from these two experiments suggest that PhoB protein interacts with the ugp promoter in the region spanning -14 to -101 , which contains the three tandemly arranged pho boxes and a `` half pho box '' ( Fig. 7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1987150" target="_blank">1987150</a>]<br></li>
<li>The results ohtained from these two experiments suggest that PhoB protein interacts with the pstS gene in thP region spanning -20 to -65 , which includes two pho boxes ( Fig. 9 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3054125" target="_blank">3054125</a>]<br></li>
<li>Since the amino-involved in the terminal half of PhoB was suggested to be activator func-regulatory function of the carboxy-terminal tion ( 26 ) , the highly conserved region in the vicinity of the mutation site may be related to the interaction with PhoR for activation of PhoB protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2676981" target="_blank">2676981</a>]<br></li>
<li>To elucidate the function of PhoR protein and to test the validity of our model , we purified PhoB protein and studied its interaction with the pstS gene in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3054125" target="_blank">3054125</a>]<br></li>
<li>Altered recognition mutants of the response regulator PhoB : a new genetic strategy for studying protein-protein interactions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11325944" target="_blank">11325944</a>]<br></li>
<li>The orientation and protein-protein contacts between the N - and C-terminal domains within an ArcA-P dimer are based on crystallographic data from ArcA and PhoB , respectively ( 15 , 16 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25161193" target="_blank">25161193</a>]<br></li>
<li>The sequence data combined with the known amino-terminal amino acid sequence of a PhoB-LacZ hybrid protein suggest that the PhoB protein consists of 228 amino acid residues with a M , of 26,302 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3537313" target="_blank">3537313</a>]<br></li>
<li>Such a sequence is also found in the PhoB protein , residues 205 to 218 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3537313" target="_blank">3537313</a>]<br></li>
<li>However , we have no biochemical data t ' o distinguish between the two hypothetical forms of PhoB protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3054125" target="_blank">3054125</a>]<br></li>
<li>D196 of PhoB does not contribute to DNA interactions [ 42 ] , however studies on PhoP from M. tuberculosis an ortholog of PhoB suggest a potential role for the equivalent residue E215 in base-specific interactions [ 50 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22291906" target="_blank">22291906</a>]<br></li>
<li>Residues T194 , V197 , R201 and R219 ( that penetrates the minor groove is labeled in red ) of PhoB have been shown to be form base specific interactions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22291906" target="_blank">22291906</a>]<br></li>
<li>Residues in KdpE targeted for mutagenesis namely R193 , R200 and T215 are indicated by stars , whereas the + sign points to R219 of PhoB that interacts with the minor groove of DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22291906" target="_blank">22291906</a>]<br></li>
<li>Interaction between the R219 residue and an AT-rich minor groove was reported previously in the crystal structure of the DBD of PhoB with pho box DNA ( Fig. 2C , R219 is shown in stick representation ) [ 42 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22291906" target="_blank">22291906</a>]<br></li>
<li>The interactions between R203 and T217 of PhoB with each other and to DNA position the wing region of wHTH motif into the minor groove [ 42 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22291906" target="_blank">22291906</a>]<br></li>
<li>KdpEDBD lacks a conserved arginine residue in the b-hairpin of the wHTH motif that interacts with the DNA minor groove as observed in PhoB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22291906" target="_blank">22291906</a>]<br></li>
<li>T h e pattern of sequence conservation within the OmpRPhoB family indicates a common fold for al1 of the DNA-binding domains , and the OmpRc fold can be used as a model to correlate mutational analyses in different proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9016718" target="_blank">9016718</a>]<br></li>
<li>From our ChIP-chip analysis , significant PhoB binding peaks were detected and the putative PhoB binding motifs were also identified within the coding regions ( panel A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23071782" target="_blank">23071782</a>]<br></li>
<li>Therefore , the amino-terminal domains of the ORF2 and PhoB proteins may be involved in the reception of physiological signals and may affect the function in the carboxyl-terminal domains that interact with the regulatory regions of the genes to be regulated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3531171" target="_blank">3531171</a>]<br></li>
<li>Members of this subfamily have an amino-terminal phosphorylation domain characteristic of so-called two-component regulators , such as OmpR , CheY , PhoB , and NtrC , and a carboxyl-terminal domain conserved among many transcriptional activators , including LuxR and MalT .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1569008" target="_blank">1569008</a>]<br></li>
<li>Sequence similarities within the amino-terminal domain place UhpA in the family of regulatory proteins which includes OmpR , CheY , PhoB , and NtrC ( 25 , 34 ) , whereas sequence relatedness in the carboxyl-terminal domain indicates that UhpA is a member of the UhpA-LuxR family of transcriptional activators ( 12 , 35 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1569008" target="_blank">1569008</a>]<br></li>
<li>The x-ray structures of DrrD and DrrB , OmpR supported by recent structural studies of the dimerized N-terminal domain of ArcA ( 20 ) and PhoB ( 21 ) , suggesting an intriguing insight into the role of phosphorylation and subsequent interaction between OmpR-P molecules through the N-terminal 4 - 5 - 5 region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16618701" target="_blank">16618701</a>]<br></li>
<li>Comparison of the A domains of PhoB , OmpR and VirG to a similar amino terminal region in NarL ( positions 41-161 ) showed 27 % , 29 % and 24 % identity respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2648330" target="_blank">2648330</a>]<br></li>
<li>For instance , residues lying in the HTH turn region of PhoB and OmpR were suggested to contact RNAP s and a subunits respectively ( Makino et al. , 1996 ; Kenney , 2002 ) ; the AR region of Fis protein , localized in the turn between helices B and C , adjacent to the HTH motif in the DNA-binding domain , was shown to be critical for contact with the RNAP a-CTD ` 273 determinant ' ( Gosink et al. , 1996 ; Bokal et al. , 1997 ; McLeod et al. , 1999 ; Cheng et al. , 2000 ; Aiyar et al. , 2002 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15255893" target="_blank">15255893</a>]<br></li>
<li>For T215 of KdpE , variants with mutations at equivalent residues in OmpR and PhoB are also defective in DNA-binding due to loss of H-bonding with the DNA backbone [ 28,49 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22291906" target="_blank">22291906</a>]<br></li>
<li>Structural comparison of the PhoB and OmpR DNA-binding/transactivation domains and the arrangement of PhoB molecules on the phosphate box .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11976297" target="_blank">11976297</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11489853" target="_blank">11489853</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10986267" target="_blank">10986267</a>]<br></li>
				<label class="DOM_collapse_similarity" for="dom_similarity_95">
Suzuki , M. , and Brenner , S.E. ( 1995 ) Classiﬁcation of multihelical DNA-binding domains and application to predict the DBD structures of s factor , LysR , OmpR/PhoB , CENP-B , Rap1 and XylS/Ada/AraC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15255893" target="_blank">15255893</a>]<br></label>
<input id="dom_similarity_95", type="checkbox">
<div class="DOM_similar">
<li>Classiﬁcation of multi-helical DNA-binding domains and application to predict the DBD structures of s factor , LysR , OmpR/PhoB , CENP-B , Rap1 , and XylS/Ada/AraC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8980677" target="_blank">8980677</a>]<br></li></div>
<li>For members of the OmpR/PhoB family the few analyses of DNA-protein interactions under equilibrium conditions available describe binding to a pair of half-sites that form a single recognition site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22291906" target="_blank">22291906</a>]<br></li>
<li>To our knowledge , ours is the first report of equilibrium binding analysis at half-sites of a recognition site for a member of the OmpR/PhoB family , the largest group among all RRs .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22291906" target="_blank">22291906</a>]<br></li>
<li>Nevertheless , full-length ArcA-P has been reported to form oligomers ( 19 ) , as have both the isolated N-terminal and C-terminal domains ( 16 ) , suggesting that although the minimal DNA-binding unit is likely a dimer , as demonstrated for PhoB and OmpR ( 20 -- 22 ) , oligomerization beyond a dimer may explain binding to multiple direct repeats .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25161193" target="_blank">25161193</a>]<br></li>
<li>Tandem direct repeat element recognition by an ArcA dimer is supported by the cocrystal structure of the C-terminal DNA-binding domain of the closely related response regulator PhoB bound to its tandem direct repeat site as a head-to-tail dimer ( 15 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25161193" target="_blank">25161193</a>]<br></li>
<li>PhoB family , it was proposed that this region may also serve as an interBecause the 4 - 5 - 5 structure of the N-terminal domain that inter - face to stabilize dimer formation on DNA ( 18 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16618701" target="_blank">16618701</a>]<br></li>
<li>The carboxy-terminal half of PhoB was suggested to be involved in transcriptional activation and DNA-binding activity ( 26 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2676981" target="_blank">2676981</a>]<br></li>
<li>By comparing the similarity in the sequencesand the functions of these regulatory proteins , the amino-terminal half of PhoB was postulated to contain the regulatory domain and the carboxy-terminal half to contain the DNA-binding domain ( Shinagawa et al. , 1987 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3054125" target="_blank">3054125</a>]<br></li>
<li>T h e structure of the C-terminal domain of E. col ; OmpR ( OmpRc ) , reported here , provides the first structural information for the OmpRPhoB family of DNA-binding domains .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9016718" target="_blank">9016718</a>]<br></li>
<li>Two architectural motifs prevail in DNA-binding domains ( DBDs ) : the winged-helix-turn-helix ( wHTH ) , which is exemplified by the OmpR/PhoB family of proteins [ 9 ] that constitute .60 % of all DNA-binding RRs , and the classic helix-turn-helix motif observed in the NarL/FixJ [ 10 ] and NtrC/DctD families [ 11 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22291906" target="_blank">22291906</a>]<br></li>
<li>The structural and biochemical data suggest two subgroups within the family based on the oligomerization state of the RRs after phosphorylation : ( i ) those that form dimers before binding DNA as exemplified by PhoB [ 26 ] and ( ii ) and other OmpR-like proteins that form stable dimers only after binding DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22291906" target="_blank">22291906</a>]<br></li>
<li>The recent structural characterization of full-length KdpE , another OmpR/PhoB family member , bound to its direct repeat site conﬁrmed these different domain symmetries while identifying an additional level of asymmetry resulting from intramolecular contacts between the receiver and DNA-binding domains within one KdpE subunit ( 18 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25161193" target="_blank">25161193</a>]<br></li>
<li>Again , while the OmpR DNA-binding domain clusters with the homologous PetR domain , its receiver module is sandwiched in between those of the algal chloroplast proteins and PhoB of B. subtilis in Fig. 2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7699720" target="_blank">7699720</a>]<br></li>
<li>Relationshipto other members of the OmpR/PhoB family of transcriptionalregulators T h e DNA-binding domain of OmpR exhibits sequence similarity over its entire length with a family of over 50 different transcriptional regulators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9016718" target="_blank">9016718</a>]<br></li>
<li>Recent structural studies of proteins in the OmpR/PhoB family suggest that the N-terminal domain is important to cooperative OmpR-P DNA-binding activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16618701" target="_blank">16618701</a>]<br></li>
<li>g003 OmpR/PhoB family appear to be refractory to crystallization with the sole exception of DNA-binding domain of PhoB [ 42 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22291906" target="_blank">22291906</a>]<br></li>
<li>Analysis KdpEDBD structure superposed on PhoB-DNA complex revealed : ( i ) R193 , conserved only among KdpE orthologs ( Fig. 2B lower panel ) had the potential to form base specific contacts ( Fig. 2C ) and ( ii ) residues conserved in OmpR/PhoB family namely R200 and T214 of KdpE superpose well onto R203 and T217 of PhoB that form a hydrogen bond ( R203 NH1 T217OG1 in PhoB ) in addition to salt bridges to the backbone in PhoB-DNA structure .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22291906" target="_blank">22291906</a>]<br></li>
<li>Overall , the structure of KdpDDBD resembles that of other members of the OmpR/PhoB family of proteins : It has a central three-helix core ( a6-a7-a8 ) bookended by two b-sheets containing four and two strands at the N - and Ctermini , respectively ( Fig. 2A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22291906" target="_blank">22291906</a>]<br></li>
<li>Such an arginine residue is conserved at structurally equivalent positions in many members of the OmpR/PhoB family , with the exception of KdpE , OmpR , and DrrB ( Fig .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22291906" target="_blank">22291906</a>]<br></li>
<li>The monomeric state of KdpEDBD is consistent with previous studies on DBDs of other OmpR/PhoB family members .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22291906" target="_blank">22291906</a>]<br></li>
<li>Although the structure of KdpEDBD is similar to that of other members of the OmpR/PhoB family of proteins , several important differences exist .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22291906" target="_blank">22291906</a>]<br></li>
<li>T h e residue corresponding to Ser181 is highly conserved throughout the OmpRPhoB family as either a serine or a threonine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9016718" target="_blank">9016718</a>]<br></li>
<li>T h e N-terminal four-stranded antiparallel j3 sheet that distinguishes OmpRc from other winged H T H proteins is conserved in the OmpRlPhoB subfamily .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9016718" target="_blank">9016718</a>]<br></li>
				</div></ul><br>
			<label class="collapse_EVO" for="_3"><font size="5px">PhoB evolution</font></label>
				<input id="_3" type="checkbox">
				<div class="EVO"><br><ul><li>Members of this family include such diverse transcriptional regulators as Escherichia coli PhoB ( of the phosphate assimilation pathway ) , Enterococcus faecium VanR ( which controls resistance to the antibiotic vancomycin ) , and Agrobacterium tumefaciens VirG ( involved in the establishment of crown gall tumors in plants ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9016718" target="_blank">9016718</a>]<br></li>
<li>The PhoB protein is homologous to a family of bacterial regulatory proteins that act as a part of the sensory and regulatory mechanisms responsible for phenotypic ad-aptation in response to environmental changes ( 18 , 23 , 38 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2676981" target="_blank">2676981</a>]<br></li>
<li>These genes are under the transcriptional control of the PhoB and PhoR proteins , which belong to a large family of two-compo-nent regulatory systems that respond to a variety of environmental stimuli ( 15 , 26 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10986267" target="_blank">10986267</a>]<br></li>
<li>T h e structure of the C-terminal domain of E. col ; OmpR ( OmpRc ) , reported here , provides the first structural information for the OmpRPhoB family of DNA-binding domains .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9016718" target="_blank">9016718</a>]<br></li>
<li>Greater similarity ( Fig. 3 ) was found with ArcA ( 43 % identity ) , OmpR ( 32 % identity ) , and PhoB ( 29 % identity ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8083154" target="_blank">8083154</a>]<br></li>
<li>Comparison of the A domains of PhoB , OmpR and VirG to a similar amino-terminal region in NarL ( positions 41-161 ) showed 27 % , 29 % and 24 % identity respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2648330" target="_blank">2648330</a>]<br></li>
<li>The PhoB , OmpR and VirG A domains showed a similar to somewhat higher identity ( 28-45 % ) when compared to one another ( 21 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2648330" target="_blank">2648330</a>]<br></li>
<li>The PhoB , OmpR and VirG proteins appear to be more related to one another than to NarL as they have 34 % -40 % identity in the B domain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2648330" target="_blank">2648330</a>]<br></li>
<li>DrrD is a response regulator protein of the OmpR/PhoB subfamil and the sequence identities between DrrD and the PmrA proteins are about 35 % .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17686460" target="_blank">17686460</a>]<br></li>
<li>Primary amino acid sequence homology alignment of NarL to OmpR , VirG , and PhoB proteins from procaryotic microorganisms .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2648330" target="_blank">2648330</a>]<br></li>
<li>Amino acid sequence comparisons showed that UhpA was homologous to a family of bacterial regulatory proteins , some of which act as transcriptional activators ( OmpR , PhoB , NtrC , and DctD ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3042748" target="_blank">3042748</a>]<br></li>
<li>Sequence similarities within the amino-terminal domain place UhpA in the family of regulatory proteins which includes OmpR , CheY , PhoB , and NtrC ( 25 , 34 ) , whereas sequence relatedness in the carboxyl-terminal domain indicates that UhpA is a member of the UhpA-LuxR family of transcriptional activators ( 12 , 35 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1569008" target="_blank">1569008</a>]<br></li>
<li>One family includes OmpR , PhoB , SfrA , VirG , CheB , CheY , Spo0A , SpoOF , NtrC , and DctD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3042748" target="_blank">3042748</a>]<br></li>
<li>The match to the other proteins , although obvious , was weaker , ranging from about 25 % identity with NtrC and CheY to 19 % for OmpR , SfrA , and PhoB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3042748" target="_blank">3042748</a>]<br></li>
<li>Overall , the structure of KdpDDBD resembles that of other members of the OmpR/PhoB family of proteins : It has a central three-helix core ( a6-a7-a8 ) bookended by two b-sheets containing four and two strands at the N - and Ctermini , respectively ( Fig. 2A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22291906" target="_blank">22291906</a>]<br></li>
<li>The sequences of KdpD and KdpE show they are members of the sensor-effector class of regulatory proteins : the C-terminal half of KdpD is homologous to sensors such as EnvZ and PhoR , and KdpE is homologous to effectors such as OmpR and PhoB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1532388" target="_blank">1532388</a>]<br></li>
<li>B. Conservation of the sequence in the wHTH motif across members of the OmpR/PhoB family ( upper panel ) and between KdpE orthologs ( lower panel ) presented in logo format derived from multiple sequence alignments [ 61 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22291906" target="_blank">22291906</a>]<br></li>
<li>Such an arginine residue is conserved at structurally equivalent positions in many members of the OmpR/PhoB family , with the exception of KdpE , OmpR , and DrrB ( Fig .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22291906" target="_blank">22291906</a>]<br></li>
<li>The inability of mutant KdpE R193A to bind to DNA is interesting because in most other family members the equivalent residue is of opposite charge as observed in PhoB ( D196 ) and OmpR ( D202 ) ( Fig .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22291906" target="_blank">22291906</a>]<br></li>
<li>The logos were created from a multiple sequence alignment made using sequences from known 3D structures of DBDs from OmpR/PhoB family members ( Fig .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22291906" target="_blank">22291906</a>]<br></li>
<li>t003 Figure S2 Multiple sequence alignment of amino acid sequences of members of the OmpR/PhoB family .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22291906" target="_blank">22291906</a>]<br></li>
<li>The monomeric state of KdpEDBD is consistent with previous studies on DBDs of other OmpR/PhoB family members .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22291906" target="_blank">22291906</a>]<br></li>
<li>Although the structure of KdpEDBD is similar to that of other members of the OmpR/PhoB family of proteins , several important differences exist .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22291906" target="_blank">22291906</a>]<br></li>
<li>Two architectural motifs prevail in DNA binding domains ( DBDs ) : the winged helix-turn-helix ( wHTH ) , which is exemplified by the OmpR/PhoB family of proteins [ 9 ] that constitute .60 % of all DNA binding RRs , and the classic helix-turn-helix motif observed in the NarL/FixJ [ 10 ] and NtrC/DctD families [ 11 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22291906" target="_blank">22291906</a>]<br></li>
<li>Relationshipto other members of the OmpR/PhoB family of transcriptionalregulators T h e DNA-binding domain of OmpR exhibits sequence similarity over its entire length with a family of over 50 different transcriptional regulators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9016718" target="_blank">9016718</a>]<br></li>
<li>g003 OmpR/PhoB family appear to be refractory to crystallization with the sole exception of DNA binding domain of PhoB [ 42 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22291906" target="_blank">22291906</a>]<br></li>
<li>Because OmpR belongs to the largest response regulator family ( OmpR/PhoB ) in E. coli , the DNA binding of transcription factors in galloping mode may be widely found in E. coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16618701" target="_blank">16618701</a>]<br></li>
<li>Optimization of such architectural features may be a common strategy to either dampen or enhance the sensitivity of DNA binding among the members of the large OmpR/PhoB family of regulators as well as other transcription factors .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25161193" target="_blank">25161193</a>]<br></li>
<li>Given the conserved dimerization mode and binding of direct repeat DNA sites among response regulators within the OmpR/PhoB family ( 16 ) , this architectural plasticity may be a common regulatory strategy , particularly for regulators that act as repressors at many targets .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25161193" target="_blank">25161193</a>]<br></li>
<li>The OmpR family member PhoB binds to direct repeat target sites in the promoter ( 32 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11976297" target="_blank">11976297</a>]<br></li>
<li>For members of the OmpR/PhoB family the few analyses of DNA-protein interactions under equilibrium conditions available describe binding to a pair of half-sites that form a single recognition site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22291906" target="_blank">22291906</a>]<br></li>
<li>To our knowledge , these data are the first to describe in quantitative terms the binding at half-sites under equilibrium conditions for a member of the ubiquitous OmpR/PhoB family of proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22291906" target="_blank">22291906</a>]<br></li>
<li>To our knowledge , ours is the first report of equilibrium binding analysis at half-sites of a recognition site for a member of the OmpR/PhoB family , the largest group among all RRs .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22291906" target="_blank">22291906</a>]<br></li>
<li>On the basis of additional homology within the carboxy-terminal domain , TorR has been recognized as a member of the OmpR subfamily of response regulators , which includes numerous members in E. coil ( such as ArcA , OmpR , or PhoB ) but also in many other bacterial species ( Aiba et aL , 1993 ; Mills et aL , 1993 ; Nagasawa et aL , 1993 ; Parkinson and Kofoid , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8596446" target="_blank">8596446</a>]<br></li>
<li>Phosphorylated PhoB is the activator for the pho regulon ( Wanner , 1993 ) and belongs to the OmpR subfamily of transcriptional activators for a70-dependent promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7641688" target="_blank">7641688</a>]<br></li>
<li>Members of this subfamily have an amino-terminal phosphorylation domain characteristic of so-called two-component regulators , such as OmpR , CheY , PhoB , and NtrC , and a carboxyl-terminal domain conserved among many transcriptional activators , including LuxR and MalT .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1569008" target="_blank">1569008</a>]<br></li>
<li>It encodes a 25,000-Da protein which shares homology with response regulators of two-component systems and belongs to the OmpR-PhoB subclass .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8083154" target="_blank">8083154</a>]<br></li>
<li>NblR belongs to the two-component regulators of the OmpR/PhoB family .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14638413" target="_blank">14638413</a>]<br></li>
<li>In E. coli , members of these homologous families have been identified in more than 10 different adaptive response systems ( for a review , see reference 38 ) , including chemotaxis ( CheA-CheY ) , nitrogen regulation ( NtrB-NtrC ) , phosphate regulation ( PhoR-PhoB ) , nitrate reductase regulation ( NarX-NarL ) , oxygen regulation ( ArcB-ArcA ) , and osmotic porin regulation ( EnvZ-OmpR ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8282700" target="_blank">8282700</a>]<br></li>
<li>NtrC , PhoB and CheB , the subjects of this study , belong to different subfamilies of response regulators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7641688" target="_blank">7641688</a>]<br></li>
<li>An in vitro study raised the possibility of interaction between hetero-pairs of response regulators from the OmpR/PhoB subfamily ( 27 ) , and Knusten et al. ( 28 ) suggested that a ComE/BlpR heterodimer ( these proteins belong to the LytR subfamily of response regulator ) recognizes a hybrid DNA motif for convergent gene regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20189963" target="_blank">20189963</a>]<br></li>
				</div></ul><br>
			<label class="collapse_RP" for="_4"><font size="5px">Processes regulated by PhoB</font></label>
				<input id="_4" type="checkbox">
				<div class="RP"><br><ul><li>Most NtrC-like regulators function in pairs ( PhoRPhoB , EnvZ-OmpR , CpxA-SfrA , VirA-VirG , DctB-DctD , CheA-CheB , CheY ) , including NifA , which is modified by the NifL product .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2666400" target="_blank">2666400</a>]<br></li>
<li>12.38 -8.24 -59.96 1.13 ± 0.55 ± 0.50 ± 0.02 0.04 0.02 1.27 ± 0.50 ± 0.20 ± 0.03 0.03 0.03 / h ) / h ) Fig. 2 Comparison of the transcript levels of the E. coli wild type under different pH values at 7.0 and 6.0 : a global regulatory genes , b PhoB regulatory genes , c acid-inducible genes , d metabolic pathway genes , e respiratory chain genes during micro-aerobic continuous culture at the dilution rate of 0.1 h -1 Fig. 3 a , b Overall metabolic regulation of the E. coli during acidic condition ( a ) Acid Condition Hns ATP ADP PhoQ PhoQ P ATP ADP ¬ EvgS EvgS P cAMP - evgA Pho P PhoP Crp EvgA P B1500 EvgA cAMP-Crp ¬ RpoS b1500 ydeo ¬ YdeO gadE GadE O O GadW GadX hdeA gadB gadc gadW gadA gadX HdeA GadA GadB GadC Acid resistance ( b ) Acidic Condition ( H + ) ( indirect ) EnvZ PhoR fnr OxyR ldhA arcA P Fnr ArcA SoxR / S L PhoR - P OmpR OmpR - P DH pfl yfiD ompC TCA-cycle genes : gltA sdhC mdh sodA PhoB - P PhoB Lactate OmpC ompF asr YfiD Pfl OmpF Asr Formate Acetate ackA pta Acid resistance Moreover , in accord with the proteomic analysis , the expression of yﬁD : : lac reporter fusion and the intracellular content of YﬁD were found to be high during growth at low pH for both aerobic and anaerobic cultures ( Neil et al. 2002 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23274360" target="_blank">23274360</a>]<br></li>
<li>Dual Transcriptional Regulation of the Escherichia coli Phosphate-Starvation-Inducible psiE Gene of the Phosphate Regulon by PhoB and the Cyclic AMP ( cAMP ) - cAMP Receptor Protein Complex SOO-KI KIM ,1 SIGENOBU KIMURA ,2 HIDEO SHINAGAWA ,2 ATSUO NAKATA ,3 KI-SUNG LEE ,1 BARRY L. WANNER ,4 AND KOZO MAKINO2 * Research Center for Biomedicinal Resources ( Bio-Med RRC ) , Paichai University , Taejon 302-735 , Korea1 ; Department of Molecular Microbiology , Research Institute for Microbial Diseases , Osaka University , 3-1 Yamadaoka , Suita , Osaka 565-0871 ,2 and Department of Marine Biotechnology , Faculty of Technology , Fukuyama University , Fukuyama , Hiroshima 729-0292 ,3 Japan ; and Department of Biological Sciences , Purdue University , West Lafayette , Indiana 479074 Received 10 March 2000/Accepted 7 July 2000 We have shown that the Escherichia coli phosphate-starvation-inducible psiE gene is regulated by both phosphate and the carbon source by using both lacZ and chloramphenicol acetyltransferase gene ( cat ) fusions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10986267" target="_blank">10986267</a>]<br></li>
<li>The genes , wzxC ( colanic acid exporter ) , wcaJ ( predicted UDP-glucose lipid carrier transferase ) , wcaK ( predicted pyruvyl transferase ) , wcaL ( predicted glycosyl transferase ) and wcaM ( predicted colanic-acid-biosynthesis protein ) are involved in the colanic-acid-biosynthesis and transportation pathway and were observed to be up-regulated in the wild-type strain relative to the PhoB knock-out strain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23071782" target="_blank">23071782</a>]<br></li>
<li>We obtained evidence that P1 is under negative control of GntR , P2 is under negative control of KdgR , and P4 is under negative control of PhoB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659677" target="_blank">15659677</a>]<br></li>
<li>In addition , post-transcriptional modifications , such as mRNA-degradation or small RNA regulation , may also be other reasons for reductions in PhoB up-regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23071782" target="_blank">23071782</a>]<br></li>
<li>In addition , PhoB also indirectly affects genes participating in protecting cells from oxidative stress and genes that balance the level of methylglyoxal .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23071782" target="_blank">23071782</a>]<br></li>
<li>In this study , chromatin immunoprecipitation for PhoB was combined with high-density tiling array ( ChIP-chip ) as well as gene-expression microarray to reveal the first global down-stream responses of the responding regulator , PhoB in E. coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23071782" target="_blank">23071782</a>]<br></li>
<li>g003 In summary , our genome-wide approach for characterizing the roles of PhoB by ChIP-chip and gene-expression array provides a comprehensive global binding profile of PhoB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23071782" target="_blank">23071782</a>]<br></li>
<li>This results in the phosphorylation of PhoB via phospho-relay , causing an increase in its affinity for the `` pho box , '' a specific DNA binding motif , that is promoterproximal in all pho regulon genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11350954" target="_blank">11350954</a>]<br></li>
<li>Therefore , the PhoRA protein is probably a kinase that activates the PhoB protein by phosphorylation , and the PhoRR protein might act as a protein phosphatase that inactivates the phospho-PhoB protein by dephosphorylation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8226621" target="_blank">8226621</a>]<br></li>
<li>Negative control of target genes by PhoB , although rare , has been observed by others ( Barry Wanner , personal communication ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659677" target="_blank">15659677</a>]<br></li>
<li>Other uncharacterized PhoB targets were also identified in our study , and the overall target genes were classified into six groups with functions involved in transcriptional regulation , transportation , metabolism , membrane structure , unknown function and pseudogene ( Table 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23071782" target="_blank">23071782</a>]<br></li>
<li>PhoB is involved in the regulation-of-transporter systems and membrane component rearrangement Out of the 287 genes showing differential expression in our gene-expression experiments , more than 60 genes encode the proteins of transporter systems ( Table S1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23071782" target="_blank">23071782</a>]<br></li>
<li>Since all the genes identified as being under the regulation of the PhoB protein code for envelope proteins , the outer membrane ( phoE ) , periplasmic proteins ( phoA , phoS and ugpB ) , and inner membrane proteins ( pho W , p & A and p & B ) , and since the Dye protein regulates the synthesis of envelope proteins , all three regulons may be involved in a system by which the cell adapts to environmental change by modifying the envelope components .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3537313" target="_blank">3537313</a>]<br></li>
<li>PhoB is involved in oxidative stress protection Previous studies described that although cells stop growing , bacteria will still undergo aerobic-respiration during phosphate starvation [ 24 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23071782" target="_blank">23071782</a>]<br></li>
<li>PhoB participates in protecting cells during phosphate starvation We have presented that during phosphate starvation , PhoB is involved in triggering the membrane component rearrangement for membrane integrity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23071782" target="_blank">23071782</a>]<br></li>
<li>Overall , the data suggest that PhoB specifically regulates a relatively small group of genes , which influence a large group of downstream genes during phosphate starvation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23071782" target="_blank">23071782</a>]<br></li>
<li>Candidates for PhoP regulation in E. coli include the phosphate starvation-inducible psiG , psiH , psil , psiJ , psiL , and psiO genes , which are also induced by starvation for carbon and nitrogen and do not seem to be under the control of PhoB , a PhoP homolog which regulates the expression of several phosphate starvation loci ( 32 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1530848" target="_blank">1530848</a>]<br></li>
<li>Additionally , PhoB regulates the expression of a number of genes that are involved in other metabolisms such as de novo biosynthesis of NAD ( Hove-Jensen , 1996 ) , initiation of chromosome replication ( Han et al. , 1999 ) , acid shock response ( Suziedeliene et al. , 1999 ) and general stress response by RpoS ( Gerard et al. , 1999 ; Ruiz & Silhavy , 2003 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17020555" target="_blank">17020555</a>]<br></li>
<li>Most genes involved in the phosphate uptake and metabolism are regulated by PhoB except for the genes for the low-afﬁnity phosphate uptake system ( Wanner , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17020555" target="_blank">17020555</a>]<br></li>
<li>Some of the new genes found to be regulated by PhoB were involved in acid and metal ion stress responses , which indicate the additional roles of PhoB as a global regulator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17020555" target="_blank">17020555</a>]<br></li>
<li>When phosphorylated , PhoB activates transcription of the PHO regulon , including phoA , which encodes bacterial alkaline phosphatase ( BAP ) ( 287 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15755952" target="_blank">15755952</a>]<br></li>
<li>Like , for example , phoA , which encodes the periplasmic enzyme alkaline phosphatase ( 26 ) , the pst-phoU operon is part of the pho regulon , which is under the control of a two-component regulatory system consisting of the proteins PhoB and PhoR ( 29 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11443103" target="_blank">11443103</a>]<br></li>
<li>The PHO regulon includes ( i ) the phoBR operon , which encodes the environmental Pi sensor , PhoR , and the PHO regulon activator , PhoB ( 29 , 30 ) ; ( ii ) the aforementioned pstSCAB-phoU operon ; ( iii ) the phoA operon ( 34 ) , which encodes bacterial alkaline phosphatase ( Bap ) , a nonspecific phosphomonoesterase ; ( iv ) the phoE gene , which encodes an outer membrane porin selective for polyanions ( 5 ) ; ( v ) the ugpBAEC transporter for glycerol 3-phosphate ( 39 ) which , like Pst , encodes a binding protein-dependent system ; and ( vi ) the phnC to phnP gene cluster , the subject of this paper .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1846145" target="_blank">1846145</a>]<br></li>
<li>The transcription of the psi genes involved in phosphorus assimilation is regulated by the PhoB and PhoR proteins ( 27 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2824439" target="_blank">2824439</a>]<br></li>
<li>Consequently , unphosphorylated PhoB can not activate the transcription of the Pho regulon , a group of genes that encode proteins involved in scavenging and metabolizing phosphorus sources other than Pi .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14526009" target="_blank">14526009</a>]<br></li>
<li>PhoB and PhoR regulate genes such as phoA and phn ( psiD ) in the PHO regulon for P assimilation in response to environmental Pi levels .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2160940" target="_blank">2160940</a>]<br></li>
<li>The phn gene cluster has a role in P assimilation because its expression is induced during Pi limitation in a manner that requires the PHO regulon regulatory proteins , PhoB and PhoR ( 54 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1846145" target="_blank">1846145</a>]<br></li>
<li>PhoB is known to be a transcriptional activator of the Pho regulon , expression of which is activated during phosphate starvation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10589831" target="_blank">10589831</a>]<br></li>
<li>The expression of the E. coli Pho regulon is activated under conditions of Pi limitation , a process requiring PhoB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7592415" target="_blank">7592415</a>]<br></li>
<li>Members of this family include such diverse transcriptional regulators as Escherichia coli PhoB ( of the phosphate assimilation pathway ) , Enterococcus faecium VanR ( which controls resistance to the antibiotic vancomycin ) , and Agrobacterium tumefaciens VirG ( involved in the establishment of crown gall tumors in plants ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9016718" target="_blank">9016718</a>]<br></li>
<li>Lee , B.L. Wanner , K. Makino , Dual transcriptional regulation of the Escherichia coli phosphate starvation-inducible psiE gene of the phosphate regulon by PhoB and the cyclic AMP ( cAMP ) -- cAMP receptor protein complex , J. Bacteriol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15313253" target="_blank">15313253</a>]<br></li>
<li>Therefore , PhoR and PhoB proteins make up a two-component signal-transduction-pathway in E. coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10589831" target="_blank">10589831</a>]<br></li>
<li>Signal transduction in the phosphate regulon of Escherichia coli involves phosphotransfer between PhoR and PhoB proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1846145" target="_blank">1846145</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10986267" target="_blank">10986267</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8226621" target="_blank">8226621</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10094685" target="_blank">10094685</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18375564" target="_blank">18375564</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1987150" target="_blank">1987150</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8444794" target="_blank">8444794</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1512197" target="_blank">1512197</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1840580" target="_blank">1840580</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1532388" target="_blank">1532388</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_36">
Makino , K. , Shinagawa , H. , Amemura , M. , Kawamoto , T. , Yamada , M. and Nakata , A. ( 1989 ) Signal transduction in the phosphate regulon of Escherichia coli involves phosphotransfer between PhoR and PhoB proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7641688" target="_blank">7641688</a>]<br></label>
<input id="rp_similarity_36", type="checkbox">
<div class="RP_similar">
<li>J Mol Biol 203:85 ± 95 Makino K , Shinagawa H , Amemura M , Kawamoto T , Yamada M , Nakata A ( 1989 ) Signal transduction in the phosphate regulon of Escherichia coli involves phosphotransfer between PhoR and PhoB proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10589831" target="_blank">10589831</a>]<br></li></div>
<li>We thus examined whether or not the positive regulator ( PhoB ) , which is responsible for induction of the so-called E. coli pfio regulon including the phoA gene ( Wanner , 1987 ) , is involved in this particular response of CbpA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7997164" target="_blank">7997164</a>]<br></li>
<li>cause it can act as a phosphate donor for PhoB , a response regulator that controls expression of the pho regulon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11350954" target="_blank">11350954</a>]<br></li>
<li>DISCUSSION Expression of E. coli Genes at Eight Genetic Loci Is Subject to CreBC Control -- The E. coli CreBC TCR was discovered fortuitously because its signal sensor , CreC , can act as phosphate donor for the pho regulon response regulator , PhoB , to activate transcription of phoA in mutants that lack the normal pho regulon signal sensor , PhoR ( 5 , 8 -- 10 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11350954" target="_blank">11350954</a>]<br></li>
<li>The Pst transport system is associated with the PhoR histidine kinase/phosphatase , which controls the phosphorylation state of the response regulator PhoB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14526009" target="_blank">14526009</a>]<br></li>
<li>Phosphate starvation causes the sensor kinase PhoR to phosphorylate the response regulator PhoB , which in turn activates phosphate starvation genes ( 32 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659677" target="_blank">15659677</a>]<br></li>
<li>The sensor kinase of this machinery , PhoR , plays an important role to pass the limited environmental phosphate signal to its response regulator , PhoB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23071782" target="_blank">23071782</a>]<br></li>
<li>The phospho-PhoR protein then functions as a protein kinase and transmits the phosphate limitation signal to the response regulator PhoB , through phosphorylation of an aspartate residue .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10589831" target="_blank">10589831</a>]<br></li>
<li>Thus , it can be said that PhoB plays an additional role in regulating metal ion metabolism or stress response in E. coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17020555" target="_blank">17020555</a>]<br></li>
<li>This suggests that PhoB plays a protective role for the oxidative stress which occurs during phosphate starvation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23071782" target="_blank">23071782</a>]<br></li>
<li>The two-component regulatory system PhoR/PhoB regulates the expression of phosphate starvation stress response genes ( 12 , 13 ) , but our findings add to evidence that ppGpp plays an important role in this stress condition as well .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17640895" target="_blank">17640895</a>]<br></li>
<li>Comparisons presented here are with the following sensorregulator transcription regulators : DegS-DegU , which controls extracellular enzyme synthesis in Bacillus subtilis ( 14 , 24 ) ; UhpB-UhpA , which controls hexose phosphate uptake in E. coli ( 37 , 50 ) ; FixL-FixJ , which controls nitrogen-fixation in Rhizobium melifoti ( 5 ) ; PhoR-PhoB , which controls the phosphate regulqn in E. coli ( 46 ) ; EnvZ-OmpR , which controls outer memitEane protein-synthesis in E. coli ( 12 ) ; and CpxA-SfrA , which controls surface properties in E. coli ( 2 , 30 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2649492" target="_blank">2649492</a>]<br></li>
<li>These include CreB and PhoB of enteric bacteria which are responsive to cytoplasmic catabolite concentrations and extracellular inorganic phosphate , respectively ; OmpR of enteric bacteria , involved in responses to osmotic conditions ; VirG proteins of agrobacterial species , activators of the Ti or Ri plasmid-encoded virulence op-erons ; and ArcA , an E. coli activator of genes encoding proteins which function in aerobic metabolism .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7699720" target="_blank">7699720</a>]<br></li>
<li>These include , for example , the osmotic-response regulator OmpR , [ 384 ] the phosphate response regulator PhoB [ 394 ± 396 ] and the nitrogen response regulator NRI , the ¯ agellar regulator [ 388 , 389 ] and the chemotaxis response regulator , CheY [ 390 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10396600" target="_blank">10396600</a>]<br></li>
<li>In E. coli , members of these homologous families have been identified in more than 10 different adaptive response systems ( for a review , see reference 38 ) , including chemotaxis ( CheA-CheY ) , nitrogen regulation ( NtrB-NtrC ) , phosphate regulation ( PhoR-PhoB ) , nitrate reductase regulation ( NarX-NarL ) , oxygen regulation ( ArcB-ArcA ) , and osmotic porin regulation ( EnvZ-OmpR ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8282700" target="_blank">8282700</a>]<br></li>
<li>Because the p & S gene product , the peri-plasmic phosphate-binding protein , is responsible for monitoring the levels of phosphate in t , hc medium and for the transduction of the phosphate signal to PhoB protein via several components of the signal-transduction-system ( Nakat ' a et al. , 1987 ; Shinagawa et al. , 1987 ) , a certain level of pstS expression should be maintained in the cell grown with excess phosphate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3054125" target="_blank">3054125</a>]<br></li>
<li>Although PhoB typically acts as a transcriptional activator , it is a repressor of eda expression ( Fig. 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659677" target="_blank">15659677</a>]<br></li>
<li>External phosphate levels are perceived by phosphatebinding protein , and the signal is transmitted by the functions of the products of the Pst-phoU operon ( 17 ) and the phoR gene to PhoB protein , which is the direct transcriptional activator for the genes in the pho regulon , including the phoB-phoR operon ( 17 , 26 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2676981" target="_blank">2676981</a>]<br></li>
<li>Since PhoB transcriptional activator of the PHO regulon ( 39 , 42 ) , phoB mutations abolish the expression of several phoB-dependent promoters , including phn ( psiD ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2155195" target="_blank">2155195</a>]<br></li>
<li>Directly repeated DNA sequences appear to be involved in DNA-protein interactions for many prokaryotic regulatory proteins including lambda cII ( 45 ) , OmpR ( 46 ) ) ToxR ( 47 ) ) PhoB ( 44,48 ) , anO d xyR ( 49 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1447222" target="_blank">1447222</a>]<br></li>
<li>The induction was independent of the RelA/SpoT , RpoH , KatF , OmpR , AppY , Lrp , PhoB and H-NS proteins during stress conditions that are known to induce or activate these global regulators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1453957" target="_blank">1453957</a>]<br></li>
<li>Primary amino acid sequence homology alignment of NarL to OmpR , VirG , and PhoB proteins from procaryotic microorganisms .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2648330" target="_blank">2648330</a>]<br></li>
<li>The amino acid sequence of the PhoB protein deduced from the nucleotide sequence of the gene revealed extensive homology with those of the OmpR ' and Dye proteins , which are also positive regulators for several genes coding for envelope proteins of E. coli ( Hall & Silhavy , 1981 ; Drurp & Buxton , 1985 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3537313" target="_blank">3537313</a>]<br></li>
<li>Doubling regulated regulated of altered time Identiﬁed function Class system ( HK/RR ) Relevant functions genes genes genes ( min ) in this experiment Strain name Mutation ntrol o o C N 1 4 5 27 BW25113 WT or little signiﬁcant alteration CreC/CreB 1 1 2 Catabolite repression ( Wanner , 1996 ) Drug resistance ( Kato et al. , 2000 ) BW26983 DE ( creABCD ) 154 29 2 3 5 EvgS/EvgA BW27869 DE ( evgAS ) 1291 29 2 6 8 YgiY/YgiX NarX/NarL BW27551 BW27864 DE ( ygiXY ) 1302 DE ( narXL ) 1316 27 29 5 4 9 Nitrate regulation ( Stewart and Rabin , 1995 ) Hexose phosphate uptake ( Kadner , 1995 ) Phosphate regulation ( Wanner , 1995 ) Copper response ( Munson et al. , 2000 ) Nitrate regulation ( Stewart and Rabin ,1995 ) UhpB/UhpA 6 3 9 BW27871 DE ( uhpAB ) 1322 28 PhoR/PhoB 1 8 9 BW24476 DE ( phoBR ) 580 28 CusS/CusR 2 7 BW28077 DE ( cusRS ) 1204 27 9 / NarP 9 BW27873 DnarP1312 33 12 3 YedV/YedW YfhK / YehU/YehT KdpD/KdpE DE ( yedWV ) 1298 DyfhK1328 DE ( yehUT ) 1324 DE ( kdpABCDE ) 1224 26 28 27 27 13 15 20 20 6 7 7 8 BW BW BW BW 27550 27872 27877 27564 13 10 7 10 Potassium transport ( Walderhaug et al. , 1992 ) Nitrogen regulation ( Porter et al. , 1995 ) NtrB/NtrC BW27880 DE ( ntrBC ) 1318 31 20 8 12 Signiﬁcant alteration BaeS/BaeR HydH/HydG ( Nagasawa et al. , 1993 ) Response to zinc and lead ( Leonhartsberger et al. , 2001 ) Hydrogen peroxide sensitivity ( Pernestig et al. , 2001 ) 14 11 BW27553 BW27867 DE ( baeSR ) 579 DE ( hydHG ) 1336 8 22 24 29 31 13 2 BarA / BW29434 DbarA1358 24 26 30 7 / FimZ BW28078 BW27558 Enterochelin DﬁmZ1214 22 27 29 31 28 27 4 PhoQ/PhoP Response to magnesium ( Kato et al. , 1999 ) ( Nagasawa et al. , 1993 ) Response to cell envelope stress ( De Wulf et al. , 1999 ) DE ( phoPQ ) 1244 20 16 12 16 BasS/BasR CpxA/CpxR BW27848 BW27559 DE ( basRS ) 1287 DE ( cpxRA ) 623 32 32 29 29 Enterochelin 12 20 22 17 RstB/RstA YojN ( Hpt ) BW27552 BW27866 DE ( rstAB ) 1278 DyojN1332 34 37 26 29 Enterochelin Capsular synthesis ( Takeda et al. , 2001 ) 14 Control of TMAO respiratory system ( Simon et al. , 1994 ) 38 24 TorS/TorR BW26423 26 DE ( torSTRCAD ) 518 a .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12366850" target="_blank">12366850</a>]<br></li>
<li>AGRT6 VIRG AGRRA YIIA ECOLI TOXR VIBCH + RRHM Pae Plastid response regulator OmpR Eco Porin bioynthesis , osmolarity PhoB Bsu Alkaline phosphatase regulator CreB Eco Catabolite repression ArcA Eco Aerobic respiratory control PetR Rca Cytochrome bc1 complex CutR Sli Copper metabolism KdpE Eco K + transport PhoB Eco Phosphate regulon PhoB Pae Phosphate regulon PhoP Sty Virulence TctD Sty Tricarboxylate transport VirG Atu6 Virulence VirG Atu5 Virulence VirG Arh Virulence YjjE Eco Unknown ToxR Vch Virulence cholera toxin Class 3 : DNA binding domains of family 3 FixJ Rme Nitrogen fixation FixJ Bja Nitrogen fixation NodW Bja Nodulation BvgA Bpe Virulence FimZ Eco Fimbrial synthesis DegU Bsu Extracellular degradative enzymes NarL Eco Nitrate reductase & formate dehydrogenase TrpO Pae Unknown YecB Eco Unknown RcsB Eco Colanic acid capsule biosynthesis B AgmR Fae Glycerol metabolism UhpA Eco Uptake of hexose phosphates Coma Bsu Competence GerE Bsu SigK txn .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7699720" target="_blank">7699720</a>]<br></li>
<li>Its preference at the P1 position of the peptide substrate is Glu Leu Ala His Val Phe Ile Gly Lys Thr ; induced by heat shock Involved in essential step in gluconeogenesis when pyruvate and lactate are used as a carbon source PpiD P77241 179 Phosphoenolpyruvate synthase 4.93 / 87,303.88 4.94 / 86,407 P23538 PpsA 212 , 228 , 294 Probable zinc protease Tail-speciﬁc protease P31828 P23865 PqqL Prc 5.95 / 104,656.47 6.04 / 74,323.25 179 179 Involved in the cleavage of a C-terminal peptide of 11 residues from the precursor form of penicillin-binding protein 3 ; may be involved in protection of the bacterium from thermal and osmotic stresses Directs the termination of translation in response to the peptide chain termination codons UGA and UAA Increases the formation of ribosomal termination complexes and stimulates activities of RF-1 and RF-2 ; binds guanine nucleotides and has strong preference for UGA stop codons ; may interact directly with the ribosome ; the stimulation of RF-1 and RF-2 is signiﬁcantly reduced by GTP and GDP but not by GMP Catalyzes the NADPH dependent reduction of L-gamma-glutamyl 5-phosphate into L-glutamate 5-semialdehyde and phosphate in proline-biosynthesis ; spontaneously undergoes cyclization to form 1-pyrroline-5-carboxylate ; Involved in proline-biosynthesis 4.64 / 41,250.73 Peptide chain release factor 2 ( RF-2 ) P07012 PrfB 1 5.65 / 59,442.89 P0A7I4 Peptide chain RF-3 PrfC 179 ProA 5.42 / 44,630.05 5.39 / 51,249 ( 5 -- 6 ) Gamma-glutamyl phosphate reductase P07004 179 , 287 179 P0A9L8 5.64 / 28,144.88 Pyrroline-5-carboxylate reductase Prolyl-tRNA synthetase Glycine betaine-binding periplasmic protein ProC ProS ProX P16659 P0AFM2 179 , 287 5.06 / 55,572 ( 4.5 -- 5.5 ) 5.12 / 63,692.60 5.65 / 33,726.86 Member of a multicomponent binding-protein-dependent transport system ( the ProU transporter ) which serves as the glycine betaine/L-proline-transporter ; induced by high pH during anaerobic growth , osmotic stress , and the stationary phase Utilized by both the de novo and the salvage pathways by which endogenously formed or exogenously added pyrimidine , purine , or pyridine bases are converted to the corresponding ribonucleoside monophosphates 5.12 / 63,692.60 5.65 / 33,726.86 103 , 179 , 191 , 321 , 323 5.65 / 33,203 ( DIGE 4.5 -- 6.5 ) 5.44 / 33,098 ( DIGE 4.5 -- 6.5 ) 5.23 / 34,087.08 P0A717 Ribose-phosphate pyrophosphokinase Prs 179 179 Psd 5.51 / 35,934.42 P0A8K1 Phosphatidylserine decarboxylase proenzyme Continued on facing pag Required for binding-protein-mediated phosphate-transport ; induced by phosphate deprivation ; subject to positive control by PhoB and to negative control by PhoR Involved in conversion of acetate to acetyl-CoA ; induced by high pH but decreases during phosphate limitation Endopeptidase that degrades small peptides of less than 7 kDa , such as glucagon and insulin Component of the phosphoenolpyruvate-dependent sugar PTS , a major carbohydrate active-transport system .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16760308" target="_blank">16760308</a>]<br></li>
<li>23.5 20.0 3.8 16.3 9.1 12.7 Maltose transport MalT , Crp malK lamB malM ibpB ibpA htpG Maltose transport MalT , Crp 18.6 5.3 3.3 39.0 24.4 11.7 32.7 13.3 9.1 24.2 20.2 8.2 2.6 2.1 1.7 4.6 4.1 2.0 48.6 19.2 20.0 52.6 35.9 16.6 s32 Heat shock Heat shock s32 Heat shock chaperone s32 Protease ; chaperone Regulator Regulator ; chaperone Protease Periplasmic clpB 11.5 12.9 17.3 21.0 20.4 14.3 3.3 3.2 1.3 1.8 2.5 2.9 8.0 2.7 2.9 24.5 6.8 5.7 1.7 1.3 yceP/bssS cpxP ( b3913 ) degP ( htrA ) spy CpxR CpxR , H-NS CpxR , BaeSR CpxR CpxR Unknown Unknown ydeH yebE 49.2 44.8 63.1 21.0 30.7 32.2 56.9 33.1 9.5 1.1 1.6 6.1 7.3 7.3 2.6 1.5 1.6 2.0 1.4 1.2 1.3 2.9 Unknown ; signal peptide Putative transporter Regulator ; phage-shock protein Regulator ; phage-shock protein Regulator ; phage-shock protein Inner membrane Sulphurtransferase yncJ ( b1436 ) yhjX pspA pspB pspC pspD pspE 1.6 44.1 1.1 1.6 1.7 1.2 1.2 3.1 165.6 1.8 1.5 2.1 1.2 1.5 PspF , IHF 1.3 1.9 5.0 2.4 4.7 2.7 2.6 1.4 PspF , IHF PhoB RbsR , Crp 37.9 22.0 2.6 1.2 1.8 2.3 0.8 1.1 Inner membrane Unknown Ribose transport Ribose transport Ribose transport Ribose transport Ribose transport Ribose transport pspG ( yjbO ) ytfK rbsD rbsA rbsC rbsB rbsK rbsR 0.7 3.8 1.8 1.0 1.6 1.4 1.2 0.8 0.9 3.0 8.2 5.4 21.3 6.6 9.0 2.8 manX Mannose permease 4.2 2.1 1.1 5.0 manY Mannose permease 3.4 2.6 1.1 5.2 manZ Mannose permease 6.5 5.7 1.0 16.7 NagC , DgsA , Crp a. Genes and operons which are induced 10-fold in three independent experiments are listed with the average fold induction for the three experiments .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18710431" target="_blank">18710431</a>]<br></li>
<li>E. coli genes whose relative expression levels were increased or decreased by baeR ampliﬁcation Effect of BaeR on gene-expression ( fold change ) Genea b no.b Known or predicted functionc Microarray Real-time PCR ncreased expression creD I 24 24 22 22 20 19 15 14 35,000 15,000 37 21 79 76 17 73 b4400 b1743 b2074 b2075 b2079 b4034 b1982 b3552 b2957 b1983 b4035 Tolerance to colicin E2 Periplasmic protein related to spheroblast formation spy mdtA ( yegM ) mdtB ( yegN ) baeR malE amn yiaD ansB yeeN malK Membrane protein inolved in drug-resistance Multidrug-transport protein ( RND family ) Response regulator in two-component regulatory system with BaeS Maltose transport protein , chemotaxis ( ABC superfamily ) AMP nucleosidase Putative outer membrane protein Periplasmic L-asparaginase II Conserved protein Maltose transport protein ( ABC superfamily ) ( N terminal ) : phenotypic repressor of mal operon ( C terminal ) Putative ( D ) - glucarate/galactarate transport protein ( MFS family ) Glutathionylspermidine amidase ( N terminal ) ; glutathionylspermidine synthetase ( C terminal ) Periplasmic protein of mal regulon Maltoporin , high-afﬁnity receptor for maltose and maltose oligosaccharides NAD kinase Putative membrane protein Multidrug-transport protein ( RND family ) Methyl-accepting chemotaxis protein 1 , serine sensor receptor Sensory histidine kinase in two-component regulatory system wtih BaeR Outer membrane channel ; tolerance to colicin E1 Conserved hypothetical protein Response regulator in two-component regulatory system with PhoR ( or CreC ) Putative transferase Putative S-transferase Methyl-accepting chemotaxis protein II , aspartate sensor receptor Flagellar biosynthesis ( D ) - Galactarate dehydrogenase Methyl-accepting chemotaxis protein IV , peptide sensor receptor Alpha-dehydro-beta-deoxy-D-glucarate aldolase Glutamate methyltransferase , chemotactic response regulator Putative oxidoreductase with FAD/NAD ( P ) - binding domaind Conserved protein Phosphonate transport protein ( ABC superfamily , ) Maltose transport protein ( ABC superfamily , membrane ) Suppressor of lon ; inhibitor of cell-division and FtsZ ring formation upon DNA damage or inhibition 3-Deoxy-D-arabinoheptulosonate-7-phosphate synthase ( DAHP synthetase ) , tyrosine repressible Flagellar biosynthesis ; hook-ﬁlament junction protein Maltose transport protein ( ABC superfamily , membrane ) Flagellar biosynthesis ; ﬁlament capping protein ; enables ﬁlament assembly MIP channel , glycerol diffusion Flagellar biosynthesis ; putative export chaperone for FliD Conserved protein in phn operon Putative cysteine hydrolase Chemotactic sensory histidine kinase ( soluble ) in two-component regulatory system with CheB and CheY e14 prophage ; isocitrate dehydrogenase , speciﬁc for NADP Sensory histidine kinase in two-component regulatory system with PhoB Qin prophage Flagellar biosynthesis ; repressor of class 3a and 3b operons ( RﬂA activity ) Enables ﬂagellar motor rotation , linking torque machinery to cell wall e14 prophage ; putative exisionase Proton conductor component of motor , torque generator Tagatoso-6-phosphate aldolase I , subunit together with AgaY Purine-binding chemotaxis protein ; regulation Chemotactic response , CheY protein phophatase Putative dehydrogenase Nucleoside channel ; receptor of phage T6 and colicin K Aminoglycoside/multidrug efﬂux pump ( RND family ) Putative multidrug-transport protein ( MFS family ) b3127 b2988 garP ( yhaU ) gsp 13 13 86 29 12 12 10 9.3 9.1 8.9 8.0 7.9 7.8 7.8 7.3 7.2 7.1 6.6 6.5 6.0 6.0 5.8 5.7 5.6 5.6 5.5 5.4 35 26 malM lamB b4037 b4036 b2615 b3716 b2076 b4355 b2078 b3035 b3717 b0399 b2267 b2989 b1886 b1923 b3128 b1885 b3126 b1884 b3690 b2473 b4105 b4033 b0958 18 50 190 14 120 12 48 18 15 14 6.3 6.4 18 7.4 32 9.1 130 9.1 140 28 5.2 yfjB yieI mdtC ( yegO ) tsr baeS tolC yieJ phoB elaA yghU cheM ( tar ) ﬂiC garD ( yhaG ) tap garL ( yhaF ) cheR yidS ypfH phnD malF sulA 5.4 5.4 5.3 5.3 5.2 5.0 5.0 5.0 5.0 9.5 5.9 19 8.1 10 13 60 7.3 8.1 aroF b2601 b1083 b4032 b1924 b3927 b1926 b4098 b0897 b1888 ﬂgL malG ﬂiD glpF ﬂiT phnJ ycaC cheA 31 15 4.9 4.9 4.7 4.7 4.5 4.5 4.4 4.4 4.3 4.1 4.1 4.1 4.1 4.0 icdA phoR b1136 b0400 b1566 b1925 b1889 b1141 b1890 b3132 b1887 b1881 b0801 b0411 b2470 b2077 7.7 6.2 6.4 19 6.6 23 6.9 39 8.2 5.8 35 110 ﬂxA ﬂiS motB b1141 motA agaZ cheW cheZ ybiC tsx acrD mdtD ( yegB ) ecreased expressio ibpB ibpA n D Small heat shock protein Small heat shock protein b3686 b3687 0.17 0.16 0.25 0.21 a Based on information from the EcoGene database ( http://bmb.med.maiami.edu/ecogene/ecoweb/ ) ( 55 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15716448" target="_blank">15716448</a>]<br></li>
				</div></ul><br>
			<label class="collapse_SIT" for="_5"><font size="5px">PhoB binding site</font></label>
				<input id="_5" type="checkbox">
				<div class="SIT"><br><ul><li>Distance to the Peak center Gene a translation start site c Log2-ratio b Sequence motif identified by MEME ( A ) Transcription regulation 331900.5 yahA 0.7814 CATTAATATATCTGTGAC 291.5 347564.5 prpR 21.9058 TTGCAACAATTATGAAAC 252.5 416096.5 phoB 8.5176 TTTTCATAAATCTGTCAT 272.5 594808.5 cusR 1.0307 AAATGACAAAATTGTCAT 277.5 1445357.5 feaR 21.4766 TTGCAACACAAATGCAAC 243.5 1626323.5 ydfH 1.1505 TTGTCACAAAAAAGTGGT 246.5 ¬ 1975957.5 flhD ATGACATCAACTTGTCAT 39.5 ¬ 2036855.5 yedW TCATTACAAAATTGTAAT 253.5 20.7827 3670466.5 yhjC 291.5 GTTTCACAATGTTGTCAT ( B ) Transportation 259612.5 phoE 8.2168 ATTTATTAAATCTGTAAT 2134.5 2.7525 cusC 594808.5 AAATGACAAAATTGTCAT 279.5 ¬ gsiD 871560.5 ACGCAACAAAACGGTCAT 626.5 ACGTATTATATTTGTCAT ¬ mdoC 1107869.5 384.5 ¬ TTGACAGATAGTTGTCAT yddB 1575128.5 1428.5 20.9238 TTGTATGACAAATGTCAC yegH 2136092.5 230.5 ¬ GTATCTTAACAATGTGAT mglB 2238620.5 2242.5 ¬ TCGTCACATTATTGCAAT ygfU 3029561.5 234.5 3590504.5 ugpB 2.7377 CTATCTTACAAATGTAAC 2105.5 ¬ ATGAAACACTGTTGTAAA yhhS 3609847.5 224.5 3710399.5 ¬ TTTTAACGTAACAGTCAC yhjX 2225.5 3910006.5 pstS 6.2008 CTGTCATAAAACTGTCAT 297.5 ¬ ATGTGACAAATCTGCCAA frwC 4140311.5 2239.5 ¬ ATGTGACAAATCTGCCAA ptsA 4140311.5 269.5 ¬ GATTCACAAATCTGTCAC yjbB 4225453.5 2212.5 4323404.5 phnC 8.2648 AATTAACCAAATCGTCAC 253.5 ( C ) Metabolism ¬ ATTTATTAAATCTGTAAT 259612.5 proB 2153.5 ¬ 2186.5 TTGCAACAATTATGAAAC 347564.5 prpB 400879.5 phoA 6.0955 271.5 CTGTCATAAAGTTGTCAC 20.6432 2117.5 TTTTCATAAATCTGTCAT 416096.5 sbcD 0.9983 408.5 ATGTCAAAAAAATGCCAC 466908.5 cof ¬ 2423.5 TTGTCAAAATGTTGTAGC 659845.5 lipA ¬ 933.5 GAGTGACGATACCGTGAC 713691.5 pgm 295.5 TTACGACAACAATGTCAA ¬ 791061.5 galE ¬ 1147.5 GTATGAAATAAATGTGAC 1369357.5 ycjM ¬ 2192.5 TTGCAACACAAATGCAAC 1445357.5 feaB 0.6879 803.5 CTGTAATAAAAATTTCAC 1564180.5 yddV ¬ 560.5 ATGCCAGAAAACGGTCAT 1628303.5 ydfI ¬ 682.5 GTAAAACAGAGCTGTGAA 1654897.5 speG 2.0870 2180.5 TTGCATAATTAATGTAAA 1864699.5 mipA ¬ 2255.5 TTGCATAATTAATGTAAA 1864699.5 yeaG ¬ 195.5 GATTAATAAAACTGTAAT 2487014.5 yfdE 2670.5 TCGTCACATTATTGCAAT ¬ 3029561.5 ygfT 555.5 TAGAAATACCGTTGTCAT ¬ 3079040.5 tktA 2170.5 CTGTCATACACCCGTCAC 3241621.5 uxaA ¬ 290.5 GTGACATAAATACGTCAT ¬ 3935233.5 rbsK ( D ) Membrane protein PhoB operates cooperative regulatory mechanisms In addition to the formation of FFL network motifs , three more observations revealed that PhoB cooperates with other transcription factors to modulate downstream responses .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23071782" target="_blank">23071782</a>]<br></li>
<li>Residues that are conserved in all four sequences are marked with asterisks 57 58 59 60 KdpE PhoB CreB OmpR * * * * ** * * ***** *** * ** GDGIEFIRDLRQ WSA VPVIVLSARSEESDKIAALDAGADDYLSKPFGIGELQARLRV GSGIQFIKHLKRESMTRDIPVVMLTARGEEEDRVRGLETGADDYITKPFSPKELVARIKA ISGFELCRQLLA LHPALPVLFLTARSEEVDRLLGLEIGADDYVAKPFSPREVCARVRT EDGLSICRRLRS QSNPMPIIMVTAKGEEVDRIVGLEIGADDYIPKPFNPRELLARIRA KdpE PhoB CreB OmpR 114 118 117 118 ** ** * ALRRHSATTAPDP LVKFSDVTVDLAARVIHRGEEEVHLTPIEFRLAGRCSTMPEK VMRRISP MAVEE VIEMQGLSLDPTSHRVMAGEEPLEMGPTEFKLLHFFMTHPER LLRRVKKFSTPSP VIRIGHFELNEPAAQISWFDTPLALTRYEFLLLKTLLKSPGR VLRRQANELPGAPSQEEAVIAFGKFKLNLGTREMFREDEPMPLTSGEFAVLKALVSHPRE KdpE PhoB CreB OmpR KdpE YSPSGPVLNQVWGPNAVEHSHYLRIYMGHLRQ 201 PhoB VYSREQLLNHVWGTNVYVEDRTVDVHIRRLRK 205 CreB VWSRQQLMDSVWEDAQDTYDRTVDTHIKTLRA 204 OmpR PLSRDKLMNLARGREYSAMERSIDVQISRLRR 210 FIG. 6 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1532388" target="_blank">1532388</a>]<br></li>
<li>Similarities ' at NarX residues : Sequence Sequence 330 to 445 to 510 to 524 to DegS UhpB FixL PhoR EnvZ CpxA 20 ( 2 ) 16 ( 3 ) 6 ( 5 ) 10 ( 5 ) 8 ( 5 ) 13 ( 4 ) 11 ( 1 ) 12 ( 4 ) 3 ( 2 ) 4 ( 3 ) 4 ( 2 ) 3 ( 1 ) 1 ( 1 ) 2 ( 1 ) 3 ( 1 ) 6 ( 3 ) 7 ( 0 ) 9 ( 0 ) 4 ( 1 ) 1 ( 2 ) 0 ( 2 ) 0 ( 1 ) 0 ( 1 ) 0 ( 1 ) DegU UhpA FixJ PhoB OmpR SfrA 42 ( 15 ) 38 ( 7 ) 26 ( 9 ) 31 ( 13 ) 35 ( 8 ) 30 ( 11 ) 30 ( 7 ) 26 ( 8 ) 27 ( 6 ) 10 ( 12 ) 10 ( 13 ) 9 ( 10 ) We thank J. A. DeMoss and E. L. Sodergren for sharing data in advance of publication and T. Thannhauser for synthesizing oligo-nucleotides .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2649492" target="_blank">2649492</a>]<br></li>
<li>Shown below the logo representation are the sequences of the wHTH motif of KdpE and PhoB ( upper panel ) and that of KdpE in the lower panel .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22291906" target="_blank">22291906</a>]<br></li>
<li>The most significant 18 bp pattern was identified ( evalue = 2.2 e218 ) , meaning that all currently identified targets of PhoB share a significant conserved pattern ( Table 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23071782" target="_blank">23071782</a>]<br></li>
<li>Likewise , the a-helical and b-stand elements of KdpEDBD superpose well onto PhoB in DNA bound conformation with deviations restricted to loops connecting a7-a8 and a8 to the bhairpin structure ( Fig. 2C ; the DNA corresponds to the PhoB recognition sequence ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22291906" target="_blank">22291906</a>]<br></li>
<li>Tandem direct repeat element recognition by an ArcA dimer is supported by the cocrystal structure of the C-terminal DNA binding domain of the closely related response regulator PhoB bound to its tandem direct repeat site as a head-to-tail dimer ( 15 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25161193" target="_blank">25161193</a>]<br></li>
<li>To understand the contribution of each DR in ArcA repression of icdA , they were mutated toward or away from 5 = - TGTTA-3 = ( Fig. 1C ) , the most conserved sequence within each ArcA DR element ( hereinafter referred to as the consensus ) and , based on the PhoB DNA cocrystal structure ( 15 ) , the region likely contacted by ArcA in the major groove .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25161193" target="_blank">25161193</a>]<br></li>
<li>It is possible that ArcA-P binds to the icdA promoter as a dimer of dimers ; one dimer binds DR1 and DR2 , as depicted in the PhoB and KdpE DNA cocrystal structures ( 15 , 18 ) , and the second dimer binds DR3 but only weakly to adjacent DNA sequence , such that no footprint is observed ( Fig. 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25161193" target="_blank">25161193</a>]<br></li>
<li>These pho boxes comprise two well-conserved 7-bp direct repeats , which are separated by poorly conserved 4-bp spacer sequences , suggesting that PhoB molecules always bind on the same face of the DNA helix ( 20 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10986267" target="_blank">10986267</a>]<br></li>
<li>The variations in sequences between a8 helices of PhoB and KdpE possibly reflect the differences in recognition sites of the two proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22291906" target="_blank">22291906</a>]<br></li>
<li>The fact that all known pho boxes are composed of a pair of repeated sequences ( Makino et al. , 1986a ) suggests that the active form of PhoB proteins is a dimer .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3054125" target="_blank">3054125</a>]<br></li>
<li>Motif identification To find the position weight matrix ( PWM ) of PhoB binding sites , E. coli K12 MG1655 sequences of all the enriched regions were extracted from NCBI RefSeq ( accession no .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23071782" target="_blank">23071782</a>]<br></li>
<li>Identification of new genes regulated by PhoB 3 In order to determine the presence of the PhoB-binding motif , the sequences of the 500 bp long upstream regions of the 18 putative PhoB-regulated genes were examined .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17020555" target="_blank">17020555</a>]<br></li>
<li>Preliminary results from our laboratory suggest that ' PhoB does indeed bind specifically to the conserved phosphate box .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3537313" target="_blank">3537313</a>]<br></li>
<li>A classic example is that of the pho operon , where the PhoB binding site is centered at 240 , occupying what would have been the 235 box .13,14 Other studies , such as string search analysis ,4 revealed a third element upstream from the 235 box , the UP element , originally associated with `` bent DNA '' ,15 and more recently associated with the binding region of the carboxy-terminal domain of the alpha subunit of RNAP .16 In about 3 % of the E. coli promoters , the UP element is located approximately 4 bp upstream of the 2 35 region conferring additional strength to the promoter .17 Different particular combinations of the elements mentioned above , some of them mutually substitu-table , allow a region to be recognized by the RNAP and transcription to initiate , deﬁning a functional promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14529615" target="_blank">14529615</a>]<br></li>
<li>Therefore , although this sequence appeared to function as the recognition site for PhoB , it appeared not to function as a cis-acting regulatory element .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1987150" target="_blank">1987150</a>]<br></li>
<li>PhoB has also been demonstrated to bind to poor consensus pho box sequences leading to gene activation [ 6,14 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15313253" target="_blank">15313253</a>]<br></li>
<li>PhoB protein probably makes contact with the DNA on the same side of t , he chain , as one turn of the helix of B-form DNA is 10.5 bp long , similar to the distance between the centers of the repeated sequences comprising the pho box .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3054125" target="_blank">3054125</a>]<br></li>
<li>The bracketed area shows the protected region by PhoB protein , and the numbers on the right indicate those of the DNA sequence of the promoter relative to the mRNA start site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3054125" target="_blank">3054125</a>]<br></li>
<li>The control region of pst consists of a -- 10 promoter sequence and two PHO-boxes ( Fig. 1 ) that are recognized by PhoB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12471449" target="_blank">12471449</a>]<br></li>
<li>The PhoB box is located downstream of the RstA box and overlaps with the promoter 35 sequence ( Fig. 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17468243" target="_blank">17468243</a>]<br></li>
<li>PhoB is the transcriptional activator , which acts by binding pho box sites immediately upstream of all known pho regulon promoters ( 16 , 17 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10986267" target="_blank">10986267</a>]<br></li>
<li>A putative binding site for the PhoB protein ( pho box ) is found upstream of the phoH coding region ( Fig .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8444794" target="_blank">8444794</a>]<br></li>
<li>PhoB protein binding site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1840580" target="_blank">1840580</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8444794" target="_blank">8444794</a>]<br></li>
<li>One of these sites overlaps the pho box , suggesting that IHF may inhibit phoA expression by binding to this sequence and interfering with the binding of the phosphorylated PhoB protein , the activator of the PHO genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9871104" target="_blank">9871104</a>]<br></li>
<li>Since the - 10 sequence of the putative upstream promoter ( TATTC at -46 in Fig. 9 ) overlaps one of the pho boxes , repression of transcription in vitro may be due to the binding of PhoB protein to these sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3054125" target="_blank">3054125</a>]<br></li>
<li>Boxes , pho boxes and the 210 region of psiE ; brackets , regions protected from DNase I digestion in the presence of PhoB and cAMP-CRP ; asterisks , positions identical with the consensus sequence for the cAMP-CRP binding site ( 7 ) ; thick line above pho box 1 , putative 235 sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10986267" target="_blank">10986267</a>]<br></li>
<li>To identify the PhoB protein recognition sequence , the pho box , we examined the binding of PhoB to the regulatory region of the phn operon by CRYPTIC AND FUNCTIONAL phn OPERONS IN E. COLI K-12 ( Mu - trp ' - acZ ' ) ( - ) pMCD1 ( 9.1 kb ) ( 5.7 kb ) ( % .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1840580" target="_blank">1840580</a>]<br></li>
<li>In the ugp regulatory region we could identify several nucleotide sequences that were similar to the consensus sequence for the recognition sites of PhoB protein ( pho box ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1987150" target="_blank">1987150</a>]<br></li>
<li>The overall arrangement of the cre tag is very similar to that of the PhoB binding site , the pho box ( 6 , 7 , 26 ) , although the exact sequence is different .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11350954" target="_blank">11350954</a>]<br></li>
<li>To study further the requirement of the exact sequence for the binding and define the pho box more precisely , we examined the PhoB proteinbinding activity of the various D N A fragments covering the pho box region ofpstS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2651888" target="_blank">2651888</a>]<br></li>
<li>The phospho-PhoB dimer is the active form of the transcriptional factor that recognizes the previously characterized PhoB recognition consensus sequence CTGTCAT-A ( AT ) A ( TA ) - CTGT ( CA ) A ( CT ) ( Pho box ) and regulates its target genes [ 1,10,11 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23071782" target="_blank">23071782</a>]<br></li>
<li>Phosphorylated PhoB binds to a consensus sequence , known as the PHO-box , located in the promoter region of all PHO genes , where it interacts with RNA polymerase associated with r70 and facilitates transcription initiation ( Makino et al. 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20963440" target="_blank">20963440</a>]<br></li>
<li>PhoR phosphorylates the transcriptional regulator PhoB , which then binds to a pho box consensus sequence within the promoter of genes in the pho regulon ( 46 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11489853" target="_blank">11489853</a>]<br></li>
<li>Phosphorylated PhoB binds to the pho box , a consensus sequence of the pho regulon promoters , and activates transcription with RNA polymerase containing the major sigma factor , c70 ( 23 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8444794" target="_blank">8444794</a>]<br></li>
<li>We discovered a consensus sequence shared by the regulatory regions of phoA , phoB , phoE and pstS , which we named the pho box , and proposed that , it might be the specific sequence that PhoB protein recognizes to activate transcription of the genes 85 0 1988 Academic Press Limik t Author to whom correspondence should be addressed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3054125" target="_blank">3054125</a>]<br></li>
<li>Makino et al. ( 11 , 12 ) found a consensus sequence shared by the regulatory regions of phoA , phoB , phoE , and pstS which was named the pho box , and they showed that PhoB protein binds to the pho boxes of pstS and phoB and activates the transcription of pstS in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2676981" target="_blank">2676981</a>]<br></li>
<li>We have found a consensus sequence in the regulatory regions of genes in the pho regulon including pstS and phoB , which we designated the ph , o box , and proposed that it is the site recognized by PhoB protein to activate transcription of these genes ( Makino et al. , 1986a ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3054125" target="_blank">3054125</a>]<br></li>
<li>Recently , we purified the PhoB protein and showed that it binds to the consensus sequence for the regulatory regions of the genes of the phosphate regulon and activates transcription of these genes in vitro ( 14 ; unpublished results ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3531171" target="_blank">3531171</a>]<br></li>
<li>Our findings indicate that PhoB protein recognizes and binds to the pho box and activates transcription of the genes in the phosphate regulon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3054125" target="_blank">3054125</a>]<br></li>
<li>Subsequently , the phosphorylated PhoB activates the transcription of at least 31 genes by binding to the Pho box in the promoter region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17020555" target="_blank">17020555</a>]<br></li>
<li>The regulatory protein PhoB is known to bind speciﬁcally to a conserved DNA region consisting of 18 nt ( the Pho box ) , which is a part of promoters for the genes belonging to the pho regulon ( 35 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10094685" target="_blank">10094685</a>]<br></li>
<li>Among the 18 new putative genes that were found to be under the control of the PhoB transcriptional regulator , ﬁve genes that contain the consensus Pho box were identiﬁed by sequence analysis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17020555" target="_blank">17020555</a>]<br></li>
<li>The protected region from DNase I digestion and the affected region from methylation by PhoB protein covered the consensus sequence for the ph. 0 box ( Figs 8 and 9 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3054125" target="_blank">3054125</a>]<br></li>
<li>Primer extension analysis of the asr transcript revealed a region similar to the Pho box ( the consensus sequence found in promoters transcriptionally activated by the PhoB protein ) upstream from the determined transcription start .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10094685" target="_blank">10094685</a>]<br></li>
<li>A region similar to the consensus sequence for the PhoB regulated promoters ( Pho box ) is boxed and labeled .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10094685" target="_blank">10094685</a>]<br></li>
<li>Indicated in the ﬁgure are : i ) the location of the four direct repeats ( identiﬁed by numbered arrows ) [ 6 ] ; ii ) the center of the intrinsic DNA bend ( ) identiﬁed by permutation experiments [ 9 ] ; iii ) the three H-NS binding sites ( striped boxes ) [ 9 ] ; iv ) the seven FIS binding sites [ 10 ] identiﬁed by footprinting ; and v ) the location of the presumed PhoB box [ 7 ] , whose sequence is given in the text ( thick frame ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11278076" target="_blank">11278076</a>]<br></li>
<li>Under low-glu-cose and LP conditions , where both proteins are abundant , PhoB and cAMP-CRP would compete for binding to the psiE promoter region , which contains overlapping sequences recognized by these proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10986267" target="_blank">10986267</a>]<br></li>
<li>Suggested binding sites for CRP , NAC , PhoB , and NsrR are underlined .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24013633" target="_blank">24013633</a>]<br></li>
<li>Determination of PhoB and cAMP-CRP binding sites within the psiE regulatory region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10986267" target="_blank">10986267</a>]<br></li>
<li>The helix-loop-he-lix-containing RRs of the OmpR family , OmpR and PhoB , bind in a direct or head-to-tail arrangement to recognition sites upstream or adjacent to the RNAP-binding region , where they interact with - CTD or - CTD , respectively ( 16 , 17 , 32 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11976297" target="_blank">11976297</a>]<br></li>
<li>The crystal structure of PhoB complexed with Pho-box DNA ( PhoB is a member of the OmpR-family transcription factor , like CpxR ) indicated that two molecules of PhoB bind in tandem to the direct repeat sequence within the Pho-box , and that the proteinprotein interaction between two PhoB proteins induces DNA curvature .16 ) The hypersensitive sites make it appear to the binding of CpxR onto the promoter induces DNA curvature .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16861804" target="_blank">16861804</a>]<br></li>
<li>The OmpR family member PhoB binds to direct repeat target sites in the promoter ( 32 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11976297" target="_blank">11976297</a>]<br></li>
<li>Gel mobility shift experiments The synthetic single-stranded 60 bp DNA fragments centered at PhoB putative binding sites were used in these experiments ( Table S3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23071782" target="_blank">23071782</a>]<br></li>
<li>For members of the OmpR/PhoB family the few analyses of DNA-protein interactions under equilibrium conditions available describe binding to a pair of half-sites that form a single recognition site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22291906" target="_blank">22291906</a>]<br></li>
				</ul></div><br>
			<label class="collapse_TU" for="_6"><font size="5px">Transcription units regulated by PhoB</font></label>
				<input id="_6" type="checkbox">
				<div class="TU"><br><ul><li>1.3 1.9 5.0 2.4 4.7 2.7 2.6 1.4 PspF , IHF PhoB RbsR , Crp 37.9 22.0 2.6 1.2 1.8 2.3 0.8 1.1 Inner membrane Unknown Ribose transport Ribose transport Ribose transport Ribose transport Ribose transport Ribose transport pspG ( yjbO ) ytfK rbsD rbsA rbsC rbsB rbsK rbsR 0.7 3.8 1.8 1.0 1.6 1.4 1.2 0.8 0.9 3.0 8.2 5.4 21.3 6.6 9.0 2.8 manX Mannose permease 4.2 2.1 1.1 5.0 manY Mannose permease 3.4 2.6 1.1 5.2 manZ Mannose permease 6.5 5.7 1.0 16.7 NagC , DgsA , Crp a. Genes and operons which are induced 10-fold in three independent experiments are listed with the average fold induction for the three experiments .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18710431" target="_blank">18710431</a>]<br></li>
<li>When either was complemented by lysogenization with the 4 ) 80 d13 ( phoB + ) phage , the lysogens were not only BAP positive but also BAP constitutive , i.e. , the mutants were phenotypically both PhoB - and PhoR - .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2824439" target="_blank">2824439</a>]<br></li>
<li>As the activity of the eptC-lacZ transcriptional fusion was not totally abolished in phoB mutants , it suggests that a constitutive basal transcriptional activity is present , which is independent of both PhoB and BasR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23372159" target="_blank">23372159</a>]<br></li>
<li>( 1988 ) 203 , 85-95 Regulation of the Phosphate Regulon f Escherichia coli o Activation by PhoB of pstS Transcription Protein in Vitro Kozo Makinot , Hideo Shinagawa , Mitsuko Amemura Sigenobu Kimura , Atsuo Nakata Department of Experimental Chemotherapy The Research Institute for Microbial Diseases Osaka University , 3-1 Yamadaoka , Suita , Osaka , Japan 565 and Akira Ishihama Department of Molecular Genetics National Institute of Genetics 1111 Yata , Mishima , Shizuoka , Japan 41 I ( Received 24 July 1987 , and in revised form 1 April 1988 ) Expression of the genes in the phosphate regulon , including the pstS ( phoS ) and phoH genes , is positively regulated by PhoB protein when phosphate is limited .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3054125" target="_blank">3054125</a>]<br></li>
<li>It was found that the expressions of the amn ( AMP nucleosidase ) , yibD ( metal ion stress response gene ) and ytfK ( hypothetical protein ) genes were activated by PhoB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17020555" target="_blank">17020555</a>]<br></li>
<li>We discovered a consensus sequence shared by the regulatory regions of phoA , phoB , phoE and pstS , which we named the pho box , and proposed that , it might be the specific sequence that PhoB protein recognizes to activate transcription of the genes 85 0 1988 Academic Press Limik t Author to whom correspondence should be addressed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3054125" target="_blank">3054125</a>]<br></li>
<li>A third factor found to affect eda transcription is PhoB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659677" target="_blank">15659677</a>]<br></li>
<li>Each pho box of pstS was demonstrated to be independently bound by the PhoB protein ( Fig. 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2651888" target="_blank">2651888</a>]<br></li>
<li>In-vitro transcription of pstS was activated by the PhoB protein , and the initiation site of transcription agreed with the ire-vivo initiation site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3054125" target="_blank">3054125</a>]<br></li>
<li>Activation of transcription of the ugp operon by phosphor-ylated PhoB protein in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1987150" target="_blank">1987150</a>]<br></li>
<li>Activation of pstS transcription by PhoB protein in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12471449" target="_blank">12471449</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10094685" target="_blank">10094685</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14529615" target="_blank">14529615</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1943993" target="_blank">1943993</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7798138" target="_blank">7798138</a>]<br></li>
<li>To examine the possibility that PhoB protein is the specific sigma factor for the pho genes , pstS transcription was studied in various in-vitro systems .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3054125" target="_blank">3054125</a>]<br></li>
<li>PhoB protein could not replace a '' for pstS transcription , so it is probably not a sigma factor of E. co & .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3054125" target="_blank">3054125</a>]<br></li>
<li>Functional categories altered by PhoB To assess the gene expression status affected by PhoB , RNA samples were extracted from the and the MG1655 strains under the same condition used for the ChIP-chip experiments .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23071782" target="_blank">23071782</a>]<br></li>
<li>Given that PhoB and CreB are highly similar ( 25 ) , we next tested whether CreB overproduction in E. coli DH5 would stimulate cre regulon gene expression even during aerobic growth on glucose , when the CreBC TCS is not activated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18375564" target="_blank">18375564</a>]<br></li>
<li>The other phoB transformants ( pBS12 , pBS14 and pBS15 ) grew on the Cm plates indicating that the putative promoters upstream of pstC , pstB and phoU are not dependent on PhoB , and therefore , should not be induced by Pi-limitation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20963440" target="_blank">20963440</a>]<br></li>
<li>The PhoB box is located downstream of the RstA box and overlaps with the promoter 35 sequence ( Fig. 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17468243" target="_blank">17468243</a>]<br></li>
<li>The most likely explanation is that PhoB represses the pitB gene because this has already been observed for a pitB-like pit gene in Rhizobium meliloti using a pit : : lacZ fusion .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11489853" target="_blank">11489853</a>]<br></li>
<li>The gadA and gadB genes for glutamate decarobxylase isozymes in a glutamate-dependent acid response system were also PhoBdependently up-regulated in this study ( Table S2 ) , and this means that these genes were also indirectly regulated by PhoB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17020555" target="_blank">17020555</a>]<br></li>
<li>The data also suggest that transcription from this start site is positively regulated by PhoB and negatively regulated by cAMP-CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10986267" target="_blank">10986267</a>]<br></li>
<li>The two upstream regions of yhjC and yegH may be positively regulated by PhoB based on our reporter gene assay .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23071782" target="_blank">23071782</a>]<br></li>
<li>The mipA gene also has a PhoB binding signal in its upstream region and is considered as a directly regulated target .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23071782" target="_blank">23071782</a>]<br></li>
<li>This is illustrated by comparing the effects of phosphate regulatory mutations on their expression , as shown in Table 4 : ( 1 ) PhoB mutations nearly abolish phoA transcription but reduce p & E induction only partially ( about lo-fold ; compare lines ( 11 ) and ( 12 ) ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3540312" target="_blank">3540312</a>]<br></li>
<li>Like , for example , phoA , which encodes the periplasmic enzyme alkaline phosphatase ( 26 ) , the pst-phoU operon is part of the pho regulon , which is under the control of a two-component regulatory system consisting of the proteins PhoB and PhoR ( 29 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11443103" target="_blank">11443103</a>]<br></li>
<li>The transcription of the psi genes involved in phosphorus assimilation is regulated by the PhoB and PhoR proteins ( 27 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2824439" target="_blank">2824439</a>]<br></li>
<li>Transcription is regulated by a two-component system that is composed of the proteins PhoB and PhoR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15347756" target="_blank">15347756</a>]<br></li>
<li>The genes and operons of the PHO regulon are co-regulated by a two-component system composed of the regulatory proteins PhoB and PhoR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12471449" target="_blank">12471449</a>]<br></li>
<li>However , while the expression of phoA and other members of the Pho regulon are regulated by the PhoR / PhoB two-component regulatory system in response to phosphate deprivation ( Wanner 1996 ) , the IraP-depen-dent stabilization of S was not affected by a phoB mutation ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16600914" target="_blank">16600914</a>]<br></li>
<li>The two-component regulatory system PhoR/PhoB regulates the expression of phosphate starvation stress response genes ( 12 , 13 ) , but our findings add to evidence that ppGpp plays an important role in this stress condition as well .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17640895" target="_blank">17640895</a>]<br></li>
<li>Hence , the reduced afﬁnity of unphosphorylated PhoB for the pho box can be mitigated by the overproduction of PhoB , which activates pho regulon gene expression even in the absence of an active PhoR signal sensor ( 12 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18375564" target="_blank">18375564</a>]<br></li>
<li>Under conditions of Pi excess , PhoR dephosphorylates PhoB , thereby ending the induced expression of the PHO regulon ( 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25645557" target="_blank">25645557</a>]<br></li>
<li>The phn gene cluster has a role in P assimilation because its expression is induced during Pi limitation in a manner that requires the PHO regulon regulatory proteins , PhoB and PhoR ( 54 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1846145" target="_blank">1846145</a>]<br></li>
<li>Candidates for PhoP regulation in E. coli include the phosphate starvation-inducible psiG , psiH , psil , psiJ , psiL , and psiO genes , which are also induced by starvation for carbon and nitrogen and do not seem to be under the control of PhoB , a PhoP homolog which regulates the expression of several phosphate starvation loci ( 32 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1530848" target="_blank">1530848</a>]<br></li>
<li>The inability to detect the corresponding transcript in uivo does not exclude the possibility that it is synthesized at a low level in the cell grown with excessphosphate and that , as in the in-vitro case , it is repressed by PhoB protein in the cell grown with limited phosphate , because expression of p & S is induced several hundredfold by phosphate starvation ( unpublished results ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3054125" target="_blank">3054125</a>]<br></li>
<li>The expression ofphoA and pstS was more strictly regulated by PhoB protein than that of phoB and phoE ; low levels of residual expression of the latter genes were observed in the phoB-phoR deletion mutant .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2676981" target="_blank">2676981</a>]<br></li>
<li>The highly conserved sequence in the regulatory region of phoB for the gene belonging t , o the phosphate regulon ( Fig. 4 ) suggests that the phoB gene as well as the phoA , phoS and phoE genes are positively regulated by the PhoB protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3537313" target="_blank">3537313</a>]<br></li>
<li>The downstream promoter was induced by phosphate starvation and requires the PhoB protein for its activation as do the other pho regulon promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1987150" target="_blank">1987150</a>]<br></li>
<li>The pstS gene belongs to the phosphate regulon whose expression is induced by phosphate starvation and regulated positively by the PhoB protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2651888" target="_blank">2651888</a>]<br></li>
				<label class="TU_collapse_similarity" for="tu_similarity_39">
The ugp operon is induced by phosphate starvation and positively regulated by PhoB protein , as are other genes in the pho regulon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1987150" target="_blank">1987150</a>]<br></label>
<input id="tu_similarity_39", type="checkbox">
<div class="TU_similar">
<li>DISCUSSION The ugp operon is known to be induced by phosphate starvation and regulated positively by PhoB protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1987150" target="_blank">1987150</a>]<br></li></div>
<li>PhoB is known to be a transcriptional activator of the Pho regulon , expression of which is activated during phosphate starvation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10589831" target="_blank">10589831</a>]<br></li>
<li>P4 is active under conditions of phosphate starvation and is directly controlled by PhoB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659677" target="_blank">15659677</a>]<br></li>
<li>Since Eda is apparently induced by phosphate starvation ( Fig. 3 ) , we sought to determine if Eda synthesis is under PhoB control .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659677" target="_blank">15659677</a>]<br></li>
<li>As seen in Fig. 1E , the mRNA for phoA , a known target of PhoB regulation , is induced under our phosphate starvation conditions and this induction is abolished in a phoB mutant , as expected .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17640895" target="_blank">17640895</a>]<br></li>
<li>This suggests that proper adaptation and response to phosphate starvation can be achieved by the high levels of PhoB and PhoR at the onset of phosphate exhaustion .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17020555" target="_blank">17020555</a>]<br></li>
<li>The pathway for iraP induction is not known but is independent of the PhoR/PhoB phosphate regulatory pathway .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16600914" target="_blank">16600914</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16600914" target="_blank">16600914</a>]<br></li>
<li>Under phosphate-replete conditions , unphosphorylated , monomeric PhoB may bind to its P4 operator to repress eda expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659677" target="_blank">15659677</a>]<br></li>
<li>These data show that expression of psiE by phosphate starvation requires PhoB ; however a substantial basal level is expressed indepen-a Cells were grown overnight in Tris-glucose medium and were assayed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10986267" target="_blank">10986267</a>]<br></li>
<li>Comparison of transcriptome profiles between wild-type and pho B mutant strains Based on the results of the phoB and phoR expression proﬁles obtained in the wild-type strain under a P-limiting condition , the time point when PhoB was most activated was determined : the time between the exponential and stationary phases , just after phosphate was completely exhausted .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17020555" target="_blank">17020555</a>]<br></li>
<li>directly or via its acceptors might activate PhoR in the periplasm and that asr is under control of PhoB -- PhoR operon ( Suziedeliené et al. 1999 ) , where it was also reported that asr expression was abolished in the phoB -- phoR deletion mutant .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23274360" target="_blank">23274360</a>]<br></li>
<li>According to a model for this circuit ( Tommassen & regulatory Lugtenberg , 1982 ; Tommassen et al. , 1982a ) , the PhoB protein functions as an activator of transcription of the structural genes of the pho regulon , whereas phoB expression in its turn is modulated by the products of phoR and phoM .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2828642" target="_blank">2828642</a>]<br></li>
<li>External phosphate levels are perceived by phosphatebinding protein , and the signal is transmitted by the functions of the products of the Pst-phoU operon ( 17 ) and the phoR gene to PhoB protein , which is the direct transcriptional activator for the genes in the pho regulon , including the phoB-phoR operon ( 17 , 26 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2676981" target="_blank">2676981</a>]<br></li>
<li>Guan et al. ( 8 ) reported that expression of phoB requires functional PhoB protein , but Shinagawa et al. ( 27 ) reported that expression of phoB was not much affected by phoBJ9 or phoB62 mutation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2676981" target="_blank">2676981</a>]<br></li>
<li>These results confirm the finding by Guan et al. ( 8 ) that phoB transcription is positively regulated by PhoB protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2676981" target="_blank">2676981</a>]<br></li>
<li>However , from the results obtained by using a phoB ' - cat fusion gene , Guan et al. ( 1983 ) suggested that expression of phoB is positively regulated by the PhoB protein while our results obtained b using a phoB ' - ` la & fusion gene indicated that the PhoB protein is not related to the regulation of the gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3537313" target="_blank">3537313</a>]<br></li>
<li>From our gene expression analysis , the yeaE gene encoding the methylglyoxal reductase was up-regulated in the wild-type E. coli strain compared to the PhoB knock-out strain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23071782" target="_blank">23071782</a>]<br></li>
<li>These results , together with measurements of the in vivo psiE promoter activity under various conditions , show that expression of the psiE gene is under direct positive and negative control by PhoB and cAMP-CRP , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10986267" target="_blank">10986267</a>]<br></li>
<li>Although PhoB typically acts as a transcriptional activator , it is a repressor of eda expression ( Fig. 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659677" target="_blank">15659677</a>]<br></li>
<li>Therefore , PhoB is not required for iraP expression or activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17640895" target="_blank">17640895</a>]<br></li>
<li>Enhanced expression of iciA-lacZ was dependent upon the PhoB protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10589831" target="_blank">10589831</a>]<br></li>
<li>These results indicate that activation of the iciA gene is dependent upon the PhoB protein Fig. 1A ± C PhoB-dependent activation of iciA expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10589831" target="_blank">10589831</a>]<br></li>
<li>As the mRNA expression levels of the tktB and xasA genes do not correlate with the results of the reporter gene assay , the expression of these two genes seems to be affected indirectly by PhoB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17020555" target="_blank">17020555</a>]<br></li>
<li>The results suggested thaf the expression of cbpA is not dependenf on the function of PhoB ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7997164" target="_blank">7997164</a>]<br></li>
<li>The alternative possibility that the phn-128 and phn-129 insertions are in a gene ( s ) for a new transcriptional activator is unlikely because the expression of all reporter fusions to these phn genes appears to be activated by PhoB ( see below and data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7592415" target="_blank">7592415</a>]<br></li>
<li>In addition to CRP and NAC , feaR may also be repressed by PhoB ( 20 ) and ArcA ( P. J. Kiley , personal communication ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24013633" target="_blank">24013633</a>]<br></li>
<li>In this design , the genome-wide map of interactions between PhoB and E. coli genomic DNA was constructed ( Figure S2A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23071782" target="_blank">23071782</a>]<br></li>
<li>In order to investigate the global biological roles played by PhoB under the phosphate-limiting condition , the COG functional distribution of these differentially expressed genes was plotted ( Figure 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23071782" target="_blank">23071782</a>]<br></li>
				</div></ul></div><br>
	</div>
</body>
</html>