<!doctype html>
<style>
@charset "utf-8";
body {
	height: 100%;
	margin: 0;
	padding: 0;
}

.banner {
	position: fixed;
	top: 0;
	width: 100%;
	height: 65px;
	background: gray;
	line-height: 65px;
}
.spacer {
	height: 10px;
}

.main-content {
	padding: 5%;
}

.collapse_ACT {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-style: none solid none none;
	border-color: #B4045F;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_ACT + input{
	display: none;
}
.collapse_ACT + input + div{
	display: none;
}
.collapse_ACT + input:checked + div{
	display: block;
}
.ACT {
	text-align: justify;
	text-justify: inter-word;
}
.ACT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B4045F;
}
.ACT_collapse_similarity:hover{
	background: #e5d3dc;
}
.ACT_collapse_similarity + input {
	display: none;
}
.ACT_collapse_similarity + input + div{
	display: none;
}

.ACT_collapse_similarity + input:checked + div{
	display: block;
}
.ACT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.ACT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B4045F;
	padding-left: 5px;
}
.ACT_collapse_similarity:hover{
	background: #e5d3dc;
}
.ACT_collapse_similarity + input {
	display: none;
}
.ACT_collapse_similarity + input + div{
	display: none;
}
.ACT_collapse_similarity + input:checked + div{
	display: block;
}
.ACT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.ACT_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #B4045F;
}
.ACT_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_DOM {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-style: none solid none none;
	border-color: #B404AE;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_DOM + input{
	display: none;
}
.collapse_DOM + input + div{
	display: none;
}
.collapse_DOM + input:checked + div{
	display: block;
}
.DOM {
	text-align: justify;
	text-justify: inter-word;
}
.DOM_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B404AE;
	padding-left: 5px;
}
.DOM_collapse_similarity:hover{
	background: #e8d7e7;
}
.DOM_collapse_similarity + input {
	display: none;
}
.DOM_collapse_similarity + input + div{
	display: none;
}
.DOM_collapse_similarity + input:checked + div{
	display: block;
}
.DOM_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}.DOM_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #B404AE;
}
.DOM_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_EVO {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #7401DF;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_EVO + input{
	display: none;
}
.collapse_EVO + input + div{
	display: none;
}
.collapse_EVO + input:checked + div{
	display: block;
}
.EVO {
	text-align: justify;
	text-justify: inter-word;
}
.EVO_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #7401DF;
	padding-left: 5px;
}
.EVO_collapse_similarity:hover{
	background: #e2d7ed;
}
.EVO_collapse_similarity + input {
	display: none;
}
.EVO_collapse_similarity + input + div{
	display: none;
}
.EVO_collapse_similarity + input:checked + div{
	display: block;
}
.EVO_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.EVO_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #7401DF;
}
.EVO_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_RP {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #0174DF;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_RP + input{
	display: none;
}
.collapse_RP + input + div{
	display: none;
}
.collapse_RP + input:checked + div{
	display: block;
}
.RP {
	text-align: justify;
	text-justify: inter-word;
}
.RP_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #0174DF;
	padding-left: 5px;
}
.RP_collapse_similarity:hover{
	background: #c9d7e2;
}
.RP_collapse_similarity + input {
	display: none;
}
.RP_collapse_similarity + input + div{
	display: none;
}
.RP_collapse_similarity + input:checked + div{
	display: block;
}
.RP_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.RP_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #0174DF;
}
.RP_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_SIT {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #0489B1;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_SIT + input{
	display: none;
}
.collapse_SIT + input + div{
	display: none;
}
.collapse_SIT + input:checked + div{
	display: block;
}
.SIT {
	text-align: justify;
	text-justify: inter-word;
}
.SIT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #0489B1;
	padding-left: 5px;
}
.SIT_collapse_similarity:hover{
	background: #d0dfe2;
}
.SIT_collapse_similarity + input {
	display: none;
}
.SIT_collapse_similarity + input + div{
	display: none;
}
.SIT_collapse_similarity + input:checked + div{
	display: block;
}
.SIT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.SIT_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #0489B1;
}
.SIT_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_TU {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #04B486;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_TU + input{
	display: none;
}
.collapse_TU + input + div{
	display: none;
}
.collapse_TU + input:checked + div{
	display: block;
}
.TU {
	text-align: justify;
	text-justify: inter-word;
}
.TU_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #04B486;
	padding-left: 5px;
}
.TU_collapse_similarity:hover{
	background: #d0e2de;
}
.TU_collapse_similarity + input {
	display: none;
}
.TU_collapse_similarity + input + div{
	display: none;
}
.TU_collapse_similarity + input:checked + div{
	display: block;
}
.TU_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.TU_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #04B486;
}
.TU_similar > a {
	text-decoration: none;
	color: gray;
}

ul { 
	list-style-type: none;
	padding: 0;
	margin: 0;
}
ul li > a {
	text-decoration: none;
	color: gray;
}
ul li:hover, ul li:focus {
	background: #F0F0F0;
}

ul li:first-child span {
  border-left:none;  
}

</style>
<html>
<head>
<meta charset="utf-8">
<title>TF properties - PrpR</title>
<meta name="theme-color" content="#ffffff">
</head>
<body>
	<div class="banner">
		<center><font color="white" size="16px" face="arial">PrpR</font></center>
	</div>
	<div align="center" class="main-content">
		<div class="spacer">
	</div><br>			<label class="collapse_ACT" for="_1"><font size="5px">PrpR transcriptional activity</font></label>
				<input id="_1" type="checkbox">
				<div class="ACT"><br><ul><li>As was mentioned earlier , the typical distance between GT and AC in most FADR-subfamily TF-binding sites is 3 nt ( DgoR , ExuR , FadR , GlcC , LldR , PdhR etc ) , although in several orthologous groups it is 2 nt ( e.g. , GntR , HpxS , HypR , MdcY , PrpR , UxuR ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26151451" target="_blank">26151451</a>]<br></li>
<li>Both probes identiÔ¨Åed the same proteins : AtoC , the product of b1201 , FhlA , HyfR , NRI , PrpR , PspF , RtcR , YfhA YgaA , YgeV , and ZraR ( HydH ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11528004" target="_blank">11528004</a>]<br></li>
<li>Genetic evidence obtained from previous studies suggested that PrpR likely sensed a small-molecule metabolite of propio-nate , not propionate itself , as the signal for activation of 3Present address : Department of Microbiology , University of Iowa , IA 52246 , USA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15528672" target="_blank">15528672</a>]<br></li>
<li>atabolite Repression of the Propionate Catabolic Genes in C Escherichia coli and Salmonella enterica : Evidence for Involvement of the Cyclic AMP Receptor Protein Sung Kuk Lee ,1 Jack D. Newman ,1 and Jay D. Keasling1 ,2 * Department of Chemical Engineering , University of California ,1 and Synthetic Biology Department , Physical Biosciences Division , Lawrence Berkeley National Laboratory ,2 Berkeley , California Received 24 September 2004/Accepted 3 January 2005 Previous studies with Salmonella enterica serovar Typhimurium LT2 demonstrated that transcriptional activation of the prpBCDE operon requires the function of transcription factor PrpR , sigma-54 , and IHF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15805526" target="_blank">15805526</a>]<br></li>
<li>Microbiology ( 2004 ) , 150 , 3877 -- 3887 DOI 10.1099 / mic.0.27299-0 2-Methylcitrate-dependent activation of the propionate catabolic operon ( prpBCDE ) of Salmonella enterica by the PrpR protein Sergio Palacios3 and Jorge C. Escalante-Semerena Department of Bacteriology , University of Wisconsin , 1710 University Avenue , Madison , WI 53726-4087 , USA Correspondence Jorge C. Escalante-Semerena escalante@bact.wisc.edu The function of the PrpR protein of Salmonella enterica serovar Typhimurium LT2 was studied in vitro and in vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15528672" target="_blank">15528672</a>]<br></li>
<li>While the expression of the mhpR , melR and malT regulators depends only on the presence of CRP ( Webster et al. , 1988 ; Chapon & Kolb , 1983 ) , the expression of rhaSR and prpR depends on the presence of CRP and the transcription activators RhaR and PrpR , respectively ( Tobin & Schleif , 1990a , b ; Wickstrum et al. , 2005 ; Lee et al. , 2005 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20966094" target="_blank">20966094</a>]<br></li>
<li>This promoter is not regulated by glucose or glycerol ( data not shown ) ( 41 ) , so that PrpR will be expressed at a consistent level regardless of catabolite repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15805526" target="_blank">15805526</a>]<br></li>
<li>Nevertheless , PprpBCDE activity consistently decreased to near background levels in the presence of glucose or glycerol in cells expressing PrpR from Pzt1 ( Fig. 3A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15805526" target="_blank">15805526</a>]<br></li>
<li>At present , it is not clear why such a high level of PrpR is present under normal growth conditions in the absence of propionate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19633077" target="_blank">19633077</a>]<br></li>
<li>A more detailed analysis of this and other structural differences is needed , and would provide important information to understand the activation mechanism of PrpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15528672" target="_blank">15528672</a>]<br></li>
<li>2-Methyl-citrate or a product of its metabolism has been proposed to-bind PrpR and induce the operon ( 155 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11528004" target="_blank">11528004</a>]<br></li>
<li>PrpR was unresponsive to citrate ( a close structural analogue of 2-MC ) and to propionate , suggesting that 2-MC , not propionate , is the metabolite that signals the presence of propionate in the environment to S. enterica .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15528672" target="_blank">15528672</a>]<br></li>
<li>PrpR speciÔ¨Åcally binds 2-MC , as shown by its unresponsiveness to citrate , a closely related structural analogue .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15528672" target="_blank">15528672</a>]<br></li>
<li>These results , taken together with previous data ( 32 , 47 ) , indicate that no fewer than four proteins ( IHF , 54 , PrpR , and CRP ) are involved in the regulation of prpB-CDE transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15805526" target="_blank">15805526</a>]<br></li>
<li>Antibodies against AscG , cyclic AMP receptor protein ( CRP ) , and PrpR were raised in rabbits by injection of the respective puriÔ¨Åed proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19633077" target="_blank">19633077</a>]<br></li>
<li>Implications of PrpR and CRP binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15805526" target="_blank">15805526</a>]<br></li>
<li>Determination of intracellular concentrations of AscG , CRP , and PrpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19633077" target="_blank">19633077</a>]<br></li>
<li>Regulation of prpBCDE expression might be affected by different concentrations of PrpR , which could explain how high concentrations of PrpR would lead to a decrease in promoter activity , but we do not know how this inhibition might occur .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15805526" target="_blank">15805526</a>]<br></li>
<li>This result adds another layer of complexity to a promoter already known to require two other proteins , IHF and PrpR , for transcriptional activation ( Fig. 1B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15805526" target="_blank">15805526</a>]<br></li>
<li>To further conÔ¨Årm the involvement of the cAMP-CRP complex in the activation of the prpBCDE promoter , PprpBCDE-rfp expression was measured in crp - and cya-deÔ¨Åcient strains expressing PrpR in trans .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15805526" target="_blank">15805526</a>]<br></li>
<li>Taken together , these results demonstrate that the dramatic decrease in prpBCDE promoter activity is most likely the result of decreased PrpR production in the crp and cya mutant strains in combination with the loss of cAMP-CRP activation of PprpBCDE itself .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15805526" target="_blank">15805526</a>]<br></li>
<li>In this study , we have shown that the 54-dependent prpBCDE promoter is positively regulated by CRP together with PrpR , which implies direct contact between CRP and E 54 or activation via DNA conformational changes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15805526" target="_blank">15805526</a>]<br></li>
<li>In vitro 2-MC-dependent activation of the prpB promoter by PrpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15528672" target="_blank">15528672</a>]<br></li>
<li>These data and those obtained with control experiments ( Fig. 4b ) indicated that binding of PrpR to the DNA used in the assay was speciÔ¨Åc .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15528672" target="_blank">15528672</a>]<br></li>
<li>The lack of an effect by 2-MC or ATP/ADP on the ability of PrpR to-bind DNA was surprising because of the known stimulatory effect of the co-activator and ATP on DNA binding by other s - dependent transcriptional-activators 54 ( Austin & Dixon , 1992 ; Lee et al. , 1994 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15528672" target="_blank">15528672</a>]<br></li>
<li>The ATPase speciÔ¨Åc activity of wild-type PrpR and of PrpR and PrpR mutant proteins was A162T R24S determined as a function of the concentration of 2-MC in the reaction mixture .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15528672" target="_blank">15528672</a>]<br></li>
<li>The ATPase speciÔ¨Åc activity of the constitutive isoform PrpR did not change when DNA c containing the PrpR binding site was added to the reaction mixture ( speciÔ¨Åc activity was 237 , 254 , 257 and 253 % liberated PO4 min at 0 , 0 ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15528672" target="_blank">15528672</a>]<br></li>
<li>The fact that a single amino acid change ( PrpR ) can A162T completely bypass the need for 2-MC without affecting the ATPase activity of the protein indicates that the effect of 2-MC binding on PrpR-dependent activation of prpBCDE expression is not due to an increase in the ATPase activity of the protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15528672" target="_blank">15528672</a>]<br></li>
<li>In fact , under the conditions tested , the ATPase activity in the wild-type PrpR protein was not stimulated by addition of 2-MC over a broad range of concentrations ( 0 ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15528672" target="_blank">15528672</a>]<br></li>
<li>The ATPase activity in the c PrpR protein , however , is only 2 ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15528672" target="_blank">15528672</a>]<br></li>
<li>It is clear that the ATPase activity of the constitutive protein PrpR is not stimulated by addi-c tion of DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15528672" target="_blank">15528672</a>]<br></li>
<li>The PrpR protein is a sensor of 2-methylcitrate ( 2-MC ) , an intermediate of the 2-methylcitric acid cycle used by this bacterium to convert propionate to pyruvate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15528672" target="_blank">15528672</a>]<br></li>
<li>Residue A162 is critical to 2-MC sensing , since the mutant PrpR protein PrpRA162T was as active as the PrpRc protein in the absence of 2-MC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15528672" target="_blank">15528672</a>]<br></li>
<li>It is possible that PrpR represents a case in which the binding of 2-MC affects other aspects of the activation process .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15528672" target="_blank">15528672</a>]<br></li>
<li>In agreement with the slight increase in prpR expression in the ascG mutant , the mRNA level of prpB , the activation target of PrpR , signiÔ¨Åcantly increased ( Fig. 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19633077" target="_blank">19633077</a>]<br></li>
<li>Increasing amounts of 2-MC in the reaction mixture correlated with increased transcription of PrpR ( Fig. 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15528672" target="_blank">15528672</a>]<br></li>
<li>This may be attributable to basal or `` leaky '' transcription from Pzt1 , resulting in an increase in the levels of PrpR , indicating that prpBCDE expression might be tightly controlled by the levels of the secondary activator , PrpR ( 13 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15805526" target="_blank">15805526</a>]<br></li>
<li>The expression of the prpBCDE operon of enteric bacteria , which codes for 2-meth-ylcitrate synthase , 2-methylcitrate dehydratase , and 2-methyl-citrate lyase , which are key enzymes of the methylcitrate cycle , and propionyl-CoA synthetase , depends on PrpR , IHF , and NtrA ( 10 , 27 , 51 , 68 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12620868" target="_blank">12620868</a>]<br></li>
<li>PrpR of S. enterica belongs to the NtrC family of transcriptional-activators and has been known to regulate the expression of the prpBCDE operon in response to propionate catabolism ( 32 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15805526" target="_blank">15805526</a>]<br></li>
<li>PrpR protein bound to the DNA region between the prpBCDE operon and prpR ( Figs 1a and 4a ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15528672" target="_blank">15528672</a>]<br></li>
				</div></ul><br>
			<label class="collapse_DOM" for="_2"><font size="5px">PrpR protein domains</font></label>
				<input id="_2" type="checkbox">
				<div class="DOM"><br><ul><li>Mutational analysis was performed to gain insights into the structural changes of PrpR required to transduce the effect exerted by the binding of 2-MC to other domains of the PrpR protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15528672" target="_blank">15528672</a>]<br></li>
<li>A more detailed analysis of this and other structural differences is needed , and would provide important information to understand the activation mechanism of PrpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15528672" target="_blank">15528672</a>]<br></li>
<li>Fractions containing PrpR protein were dialysed against 10 mM Tris/HCl buffer , pH 7 ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15528672" target="_blank">15528672</a>]<br></li>
<li>The PrpR protein was excised from the GST N-terminal fusion using thrombin protease .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15528672" target="_blank">15528672</a>]<br></li>
<li>Excision from the tag leaves two amino acid residues from the GST protein tag in the N-terminus of PrpR ( glycine -- serine ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15528672" target="_blank">15528672</a>]<br></li>
<li>PrpR has ATPase activity that is not affected by DNA or 2-MC s - dependent activators typically have an ATPase activity 54 that is essential for transcriptional activation ( Austin & Dixon , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15528672" target="_blank">15528672</a>]<br></li>
<li>It is possible that PrpR represents a case in which the binding of 2-MC affects other aspects of the activation process .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15528672" target="_blank">15528672</a>]<br></li>
<li>It is possible that ATP-binding , hydrolysis , oligomerization and DNA-binding are unregulated events in PrpR , and 2-MC binding may expose the GAFTGA motif , a highly conserved region in the catalytic domain found to be essential for interaction with RNA polymerase / 54 s and for transduction of the ATP hydrolysis signal ( Lee et al. , 2003 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15528672" target="_blank">15528672</a>]<br></li>
<li>The ATPase speciÔ¨Åc activity of the constitutive isoform PrpR did not change when DNA c containing the PrpR binding site was added to the reaction mixture ( speciÔ¨Åc activity was 237 , 254 , 257 and 253 % liberated PO4 min at 0 , 0 ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15528672" target="_blank">15528672</a>]<br></li>
<li>This suggested that the A domain had a negative effect on the transcriptional-activity of PrpR , and that binding of the co-activator modulated this effect ( Palacios & Escalante-Semerena , 2000 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15528672" target="_blank">15528672</a>]<br></li>
<li>PrpR , however , contains an alanine -- serine substitution in this motif ( GAFTGS ) that does not affect the transcriptional-activity of the protein ( Palacios & Escalante-Semerena , 2000 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15528672" target="_blank">15528672</a>]<br></li>
<li>We note that none of the mutations that led to 2-MC-independent PrpR activity were mapped in the linker region between the A and C domains ( Table 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15528672" target="_blank">15528672</a>]<br></li>
<li>All lesions causing constitutive PrpR activity were mapped to the N-terminal domain of the protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15528672" target="_blank">15528672</a>]<br></li>
<li>All the causal mutations were mapped to the A domain of PrpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15528672" target="_blank">15528672</a>]<br></li>
<li>One protein ( PrpR ) showed activity similar to that A162T of the PrpR protein , in which the entire A domain was c missing ( Table 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15528672" target="_blank">15528672</a>]<br></li>
<li>It appears that the highest level of ATPase activity in PrpR can only be attained by removal of the entire sensing-domain ( PrpR ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15528672" target="_blank">15528672</a>]<br></li>
<li>The signal-sensing ( A ) domain of PrpR regulates its activity Previous studies showed that the removal of the 209 N-terminal residues which compose the signal-sensing Fig. 4 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15528672" target="_blank">15528672</a>]<br></li>
<li>Role of 2-MC The signal-sensing-domain of PrpR has a negative effect on the activity of the protein , since removal of the domain yields a constitutively active protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15528672" target="_blank">15528672</a>]<br></li>
<li>5 100 2 prpR JE4766 JE4593 PrpR ( WT ) PrpR ( DA domain ) c 940 ¬± 45 62 prpR234 prpR235 prpR236 JE6634 JE6635 180 ¬± 26 260 ¬± 26 19 28 12 17 F129L E50K prpR237 prpR238 JE6636 JE6637 190 ¬± 4 980 ¬± 15 20 13 65 D37N A162T 104 S33R F49L prpR239 prpR240 JE6638 JE6639 580 ¬± 22 415 ¬± 21 62 44 38 28 E172G F49S prpR241 prpR242 JE6672 JE6673 65 ¬± 4 170 ¬± 9 7 4 18 11 S33N E142K prpR243 prpR244 JE6674 JE6675 500 ¬± 4 530 ¬± 77 53 56 33 35 R24S prpR245 prpR246 JE6865 JE6762 490 ¬± 10 85 ¬± 12 52 9 32 6 Y180H L47P S21P prpR247 prpR248 JE6763 JE6764 360 ¬± 22 90 ¬± 5 38 24 9 6 R24H G104E prpR249 prpR250 JE6765 JE6766 350 ¬± 47 70 ¬± 14 37 23 7 5 K81H F178C prpR251 prpR252 JE6767 JE6768 350 ¬± 12 570 ¬± 13 37 23 61 36 38 23 340 ¬± 34 JE6769 D31G prpR253 PrpR is different from other well-studied members of this family of activators , such as XylR and DmpR , in which the region linking the sensing and catalytic domains contains a coiled-coil structure that is important for controlling binding of the effector molecule and activation of the protein ( Garmendia & de Lorenzo , 2000 ; O'Neill et al. , 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15528672" target="_blank">15528672</a>]<br></li>
<li>When the primary amino acid sequence of PrpR was analysed using the COILS coiled-coil structure algorithm ( Lupas et al. , 1991 ) , no coiled-coil structures were predicted in PrpR ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15528672" target="_blank">15528672</a>]<br></li>
<li>A coiled-coil structure was not detected in several other members of this family of activators that sense smallmolecule effectors , including PrpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15528672" target="_blank">15528672</a>]<br></li>
<li>We propose that PrpR and other members of this family of regulators that sense small-molecule co-activators and lack this coiled-coil motif constitute a subclass which uses a different mechanism for signal propagation within the protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15528672" target="_blank">15528672</a>]<br></li>
				</div></ul><br>
			<label class="collapse_EVO" for="_3"><font size="5px">PrpR evolution</font></label>
				<input id="_3" type="checkbox">
				<div class="EVO"><br><ul><li>In addition to 54 , expression requires IHF and PrpR , which is homologous to NRI ( 122 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11528004" target="_blank">11528004</a>]<br></li>
<li>One protein ( PrpR ) showed activity similar to that A162T of the PrpR protein , in which the entire A domain was c missing ( Table 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15528672" target="_blank">15528672</a>]<br></li>
<li>A coiled-coil structure was not detected in several other members of this family of activators that sense smallmolecule effectors , including PrpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15528672" target="_blank">15528672</a>]<br></li>
<li>We propose that PrpR and other members of this family of regulators that sense small-molecule co-activators and lack this coiled-coil motif constitute a subclass which uses a different mechanism for signal propagation within the protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15528672" target="_blank">15528672</a>]<br></li>
<li>PrpR of S. enterica belongs to the NtrC family of transcriptional activators and has been known to regulate the expression of the prpBCDE operon in response to propionate catabolism ( 32 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15805526" target="_blank">15805526</a>]<br></li>
<li>One unit is the single gene prpR , which encodes a regulatory protein , PrpR , a member of the RpoN-dependent activator family ( 22 , 40 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19633077" target="_blank">19633077</a>]<br></li>
<li>PrpR is an NtrC family activator which interacts directly with RpoN RNA polymerase ( 17 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19633077" target="_blank">19633077</a>]<br></li>
				</div></ul><br>
			<label class="collapse_RP" for="_4"><font size="5px">Processes regulated by PrpR</font></label>
				<input id="_4" type="checkbox">
				<div class="RP"><br><ul><li>As observed for the MhpR regulator , CRP also regulates expression of other transcription factors in E. coli such as MelR ( Webster et al. , 1988 ) , MalT ( Chapon & Kolb , 1983 ) , RhaS ( Wickstrum et al. , 2005 ) , MaoB ( Yamashita et al. , 1996 ) , HcaR ( Turlin et al. , 2001 ) , GalS ( Weickert & Adhya , 1993 ) , RpoH ( Kallipolitis & Valentin-Hansen , 1998 ) , BlgG ( Gulati & Mahadevan , 2000 ) , PrpR ( Lee et al. , 2005 ) and Fis ( Nasser et al. , 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20966094" target="_blank">20966094</a>]<br></li>
<li>E. coli operons with conÔ¨Årmed 54-dependent promoters Promoter sequencea Operon Regulator Function argT-hisJQMP NRI-NRII Arginine , histidine-transport astCADBE NRI-NRII-ArgR Arginine catabolism atoDAEB AtoC-AtoS Acetoacetate-catabolism fdhF FhlA FDH glnALG NRI-NRII Nitrogen assimilation glnHPQ NRI-NRII Glutamine-transport glnK-amtB NRI-NRII PII paralog , NH3 transport hycABCDEFGHI FhlA Hydrogenase hydN-hypF FhlA Hydrogenase hypABCDE-fhlA FhlA Hydrogenase nac NRI-NRII Nitrogen assimilation prpBCDE PrpR Propionate catabolism pspABCDE PspF Phage shock and other stresses rtcBA RtcR RNA terminal cyclase ygjG NRI-NRII ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11528004" target="_blank">11528004</a>]<br></li>
<li>The expression of the prpBCDE operon of enteric bacteria , which codes for 2-meth-ylcitrate synthase , 2-methylcitrate dehydratase , and 2-methyl-citrate lyase , which are key enzymes of the methylcitrate cycle , and propionyl-CoA synthetase , depends on PrpR , IHF , and NtrA ( 10 , 27 , 51 , 68 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12620868" target="_blank">12620868</a>]<br></li>
<li>One possibility is that PrpR has an as yet unidentiÔ¨Åed physiological role , including the regulation of other genes besides the only known target , the prpBCDE operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19633077" target="_blank">19633077</a>]<br></li>
				</div></ul><br>
			<label class="collapse_SIT" for="_5"><font size="5px">PrpR binding site</font></label>
				<input id="_5" type="checkbox">
				<div class="SIT"><br><ul><li>We then cloned sequences of about 500 bp in length upstream from the initiation codons of the ascF , prpR , and prpB genes ( see Fig. 3 and 8 for the gene organization ) , each including transcription initiation sites and recognition sequences for a transcription factor , i.e. , AscG ( ascF ) , CRP ( ascF and prpB ) , or PrpR ( prpB ) , and inserted them into pGRP to construct the promoter assay vectors pGRK259 ( pGRPascF ) , pGRK559 ( pGRPascG ) , and pGRN024 ( pGRPprpR ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19633077" target="_blank">19633077</a>]<br></li>
<li>Recently , it has been reported that PrpR bound to two regions in S. enterica that have the consensus sequence 5 - CGTTTCATGAAACG-3 and span bases 55 to 68 and 77 to 90 from the start codon of the prpR gene , as shown in Fig. 1 ( 33 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15805526" target="_blank">15805526</a>]<br></li>
<li>PrpR bound to two regions which have the consensus sequence 59-CGTTTCATGAAACG-39 and which span from 2110 to 2123 and from 2132 to 2145 bases from the 224 element of the prpB promoter ( Fig. 5b ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15528672" target="_blank">15528672</a>]<br></li>
<li>The proposed ATG start sites for PrpR and PrpB are boxed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15805526" target="_blank">15805526</a>]<br></li>
<li>PrpR binding sites were deÔ¨Åned by a region of strong DNase I hypersensitivity located at the 39 end of the symmetric regions ( Fig. 5a ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15528672" target="_blank">15528672</a>]<br></li>
<li>DNA footprinting studies identiÔ¨Åed the site in the region between prpR and the prpBCDE operon to which the PrpR protein binds .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15528672" target="_blank">15528672</a>]<br></li>
<li>DNA footprinting The sequence of the site to which PrpR binds was determined using a DNase I protection assay .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15528672" target="_blank">15528672</a>]<br></li>
<li>The function of the divergent prpBCDE promoter was completely destroyed even by deleting prpR promoter sequences ( with PrpR supplied in trans ) ( pAPF7 ) , suggesting that the PrpR-binding site for activation of the prpBCDE promoter overlaps the prpR promoter itself ( Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15805526" target="_blank">15805526</a>]<br></li>
<li>Results from DNase I protection experiments indicated that the PrpR binding site spanned nucleotides 110 to 145 relative to the PprpBCDE promoter ( Fig. 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15528672" target="_blank">15528672</a>]<br></li>
<li>Interactions of PrpR with its DNA binding site The DNA sequence recognized by PrpR is composed of two sites , which were termed proximal ( P ) and distal ( D ) to reÔ¨Çect their location relative to the prpB promoter ( Fig. 5b ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15528672" target="_blank">15528672</a>]<br></li>
				</ul></div><br>
			<label class="collapse_TU" for="_6"><font size="5px">Transcription unit regulated by PrpR</font></label>
				<input id="_6" type="checkbox">
				<div class="TU"><br><ul><li>2-Methyl-citrate or a product of its metabolism has been proposed to bind PrpR and induce the operon ( 155 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11528004" target="_blank">11528004</a>]<br></li>
<li>To determine whether 2-MC was required for the trans-criptional activation of the prpBCDE genes or not , co-activator-dependent transcription of the prpB promoter by PrpR was tested in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15528672" target="_blank">15528672</a>]<br></li>
<li>One unit is the single gene prpR , which encodes a regulatory protein , PrpR , a member of the RpoN-dependent activator family ( 22 , 40 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19633077" target="_blank">19633077</a>]<br></li>
<li>PrpR of S. enterica belongs to the NtrC family of transcriptional activators and has been known to regulate the expression of the prpBCDE operon in response to propionate catabolism ( 32 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15805526" target="_blank">15805526</a>]<br></li>
<li>One possibility is that PrpR has an as yet unidentiÔ¨Åed physiological role , including the regulation of other genes besides the only known target , the prpBCDE operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19633077" target="_blank">19633077</a>]<br></li>
<li>One unit , prpBCDE , encodes most of the enzymes Transcription of the prpBCDE operon of S. enterica is known to require the functions of the PrpR , RpoN and IHF proteins ( Palacios & Escalante-Semerena , 2000 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15528672" target="_blank">15528672</a>]<br></li>
				</div></ul></div><br>
	</div>
</body>
</html>