<!doctype html>
<style>
@charset "utf-8";
body {
	height: 100%;
	margin: 0;
	padding: 0;
}

.banner {
	position: fixed;
	top: 0;
	width: 100%;
	height: 65px;
	background: gray;
	line-height: 65px;
}
.spacer {
	height: 10px;
}

.main-content {
	padding: 5%;
}

.collapse_ACT {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-style: none solid none none;
	border-color: #B4045F;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_ACT + input{
	display: none;
}
.collapse_ACT + input + div{
	display: none;
}
.collapse_ACT + input:checked + div{
	display: block;
}
.ACT {
	text-align: justify;
	text-justify: inter-word;
}
.ACT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B4045F;
}
.ACT_collapse_similarity:hover{
	background: #e5d3dc;
}
.ACT_collapse_similarity + input {
	display: none;
}
.ACT_collapse_similarity + input + div{
	display: none;
}

.ACT_collapse_similarity + input:checked + div{
	display: block;
}
.ACT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.ACT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B4045F;
	padding-left: 5px;
}
.ACT_collapse_similarity:hover{
	background: #e5d3dc;
}
.ACT_collapse_similarity + input {
	display: none;
}
.ACT_collapse_similarity + input + div{
	display: none;
}
.ACT_collapse_similarity + input:checked + div{
	display: block;
}
.ACT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.ACT_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #B4045F;
}
.ACT_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_DOM {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-style: none solid none none;
	border-color: #B404AE;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_DOM + input{
	display: none;
}
.collapse_DOM + input + div{
	display: none;
}
.collapse_DOM + input:checked + div{
	display: block;
}
.DOM {
	text-align: justify;
	text-justify: inter-word;
}
.DOM_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B404AE;
	padding-left: 5px;
}
.DOM_collapse_similarity:hover{
	background: #e8d7e7;
}
.DOM_collapse_similarity + input {
	display: none;
}
.DOM_collapse_similarity + input + div{
	display: none;
}
.DOM_collapse_similarity + input:checked + div{
	display: block;
}
.DOM_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}.DOM_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #B404AE;
}
.DOM_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_EVO {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #7401DF;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_EVO + input{
	display: none;
}
.collapse_EVO + input + div{
	display: none;
}
.collapse_EVO + input:checked + div{
	display: block;
}
.EVO {
	text-align: justify;
	text-justify: inter-word;
}
.EVO_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #7401DF;
	padding-left: 5px;
}
.EVO_collapse_similarity:hover{
	background: #e2d7ed;
}
.EVO_collapse_similarity + input {
	display: none;
}
.EVO_collapse_similarity + input + div{
	display: none;
}
.EVO_collapse_similarity + input:checked + div{
	display: block;
}
.EVO_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.EVO_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #7401DF;
}
.EVO_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_RP {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #0174DF;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_RP + input{
	display: none;
}
.collapse_RP + input + div{
	display: none;
}
.collapse_RP + input:checked + div{
	display: block;
}
.RP {
	text-align: justify;
	text-justify: inter-word;
}
.RP_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #0174DF;
	padding-left: 5px;
}
.RP_collapse_similarity:hover{
	background: #c9d7e2;
}
.RP_collapse_similarity + input {
	display: none;
}
.RP_collapse_similarity + input + div{
	display: none;
}
.RP_collapse_similarity + input:checked + div{
	display: block;
}
.RP_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.RP_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #0174DF;
}
.RP_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_SIT {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #0489B1;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_SIT + input{
	display: none;
}
.collapse_SIT + input + div{
	display: none;
}
.collapse_SIT + input:checked + div{
	display: block;
}
.SIT {
	text-align: justify;
	text-justify: inter-word;
}
.SIT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #0489B1;
	padding-left: 5px;
}
.SIT_collapse_similarity:hover{
	background: #d0dfe2;
}
.SIT_collapse_similarity + input {
	display: none;
}
.SIT_collapse_similarity + input + div{
	display: none;
}
.SIT_collapse_similarity + input:checked + div{
	display: block;
}
.SIT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.SIT_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #0489B1;
}
.SIT_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_TU {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #04B486;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_TU + input{
	display: none;
}
.collapse_TU + input + div{
	display: none;
}
.collapse_TU + input:checked + div{
	display: block;
}
.TU {
	text-align: justify;
	text-justify: inter-word;
}
.TU_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #04B486;
	padding-left: 5px;
}
.TU_collapse_similarity:hover{
	background: #d0e2de;
}
.TU_collapse_similarity + input {
	display: none;
}
.TU_collapse_similarity + input + div{
	display: none;
}
.TU_collapse_similarity + input:checked + div{
	display: block;
}
.TU_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.TU_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #04B486;
}
.TU_similar > a {
	text-decoration: none;
	color: gray;
}

ul { 
	list-style-type: none;
	padding: 0;
	margin: 0;
}
ul li > a {
	text-decoration: none;
	color: gray;
}
ul li:hover, ul li:focus {
	background: #F0F0F0;
}

ul li:first-child span {
  border-left:none;  
}

</style>
<html>
<head>
<meta charset="utf-8">
<title>TF properties - PuuR</title>
<meta name="theme-color" content="#ffffff">
</head>
<body>
	<div class="banner">
		<center><font color="white" size="16px" face="arial">PuuR</font></center>
	</div>
	<div align="center" class="main-content">
		<div class="spacer">
	</div><br>			<label class="collapse_ACT" for="_1"><font size="5px">PuuR transcriptional activity</font></label>
				<input id="_1" type="checkbox">
				<div class="ACT"><br><ul><li>PuuR and pACYC184 vector were 128 and 30 , 108 and 22 , 78 and 16 , and 76 and 15 nmol/mg protein , respectively , when cells were harvested at A540 = 0.5 ( Fig. 2b ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23719730" target="_blank">23719730</a>]<br></li>
<li>ns , p C 0.05 , * p \ 0.05 , ** p \ 0.01 1.2 + PUT , pACYC-PotFGHI + PUT , pACYC-PuuP + PUT , pACYC-PuuP + PuuR + PUT , pACYC184 + PUT , pACYC-PotFGHI + PUT , pACYC-PuuP + PUT , pACYC-PuuP + PuuR + PUT , pACYC184 1.2 1.0 1.0 .8 0 0.8 0.6 0.6 0.4 0.4 Polyamine A540 ( nmol/mg protein ) 0.2 0.2 - PUT , pACYC184 - PUT , pACYC184 0 0 0 2 4 6 8 10 12 14 Time ( h ) 0 2 4 6 8 10 12 14 Time ( h ) B Polyamine content PUT SPD PUT SPD 200 00 2 ** * * 150 * 150 * * ns 100 100 ns ns 50 50 ns ns ns ns 0 0 Polyamine pACYC - pACYC - pACYC - pACYC184 PotFGHI PuuP PuuP + PuuR pACYC - pACYC - pACYC - pACYC184 PotFGHI PuuP PuuP + PuuR A 54 ( nmol/mg protein ) Comparison of expression level of PotFGHI and PuuP The levels of PotFGHI and PuuP proteins and those of potFGHI and puuP mRNAs were analyzed by Western blotting and dot blot analysis , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23719730" target="_blank">23719730</a>]<br></li>
<li>These results indicated that excess putrescine is appropriately catabolized by products of the puu gene cluster that are regulated by ArcA , FNR , and PuuR to decrease the toxicity of putrescine and to utilize putrescine as both nitrogen and carbon sources .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18495664" target="_blank">18495664</a>]<br></li>
<li>These strains were designed by increasing puuA and puuD copy numbers using a pUC19 vector in the puuR ( 3Ala ) host cell , in which repression of puu gene expression by PuuR was disrupted .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22522900" target="_blank">22522900</a>]<br></li>
<li>These results suggest that PuuR binds to the intergenic region between puuA and puuD for repression of the puu genes , but PuuR does not bind to the intergenic region between puuP and puuA or puuR and puuC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22522900" target="_blank">22522900</a>]<br></li>
<li>At low intracellular putrescine concentrations , PuuR binds to the intergenic region between puuA and puuD and represses the expression of the puu genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22522900" target="_blank">22522900</a>]<br></li>
<li>For example , it appears that putrescine binds only weakly to the PuuR repressor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23531166" target="_blank">23531166</a>]<br></li>
<li>The binding of wild-type PuuR to a DNA probe containing PuuR recognition sites was diminished with increasing putrescine concentrations in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22522900" target="_blank">22522900</a>]<br></li>
<li>After the Puu pathway utilizes putrescine and its concentration in the cell decreases , PuuR may bind to DNA again and represses the transcription of puu genes ( Fig. 7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22522900" target="_blank">22522900</a>]<br></li>
<li>The results presented above suggest that the Puu pathway will work when the putrescine concentration in the cell is relatively high , because PuuR remains bound to DNA at low concentrations of putrescine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22522900" target="_blank">22522900</a>]<br></li>
<li>The 32P-labeled 6 probes ( 1.35 fmol ) were incubated in the absence ( lane 1 ) or presence of increasing concentrations of puriﬁed His6-PuuR ( lane 2 , 1 pmol ; lane 3 , 5 pmol ; lane 4 , 10 pmol ; lane 5 , 20 pmol ; lane 6 , 40 pmol ) , and then subjected to DNase I footprinting assays ( panel B represents segment 1 from panel A ; panel C represents segment 2 from panel A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22522900" target="_blank">22522900</a>]<br></li>
<li>Lane 2 , presence of cell extract with His6-PuuR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22522900" target="_blank">22522900</a>]<br></li>
<li>Puriﬁed His6-PuuR bound to probe 2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22522900" target="_blank">22522900</a>]<br></li>
<li>In our study , all of these genes showed down-regulation and their transcription repressor , PuuR , in turn was up-regulated in the presence of PhoB activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23071782" target="_blank">23071782</a>]<br></li>
<li>When puuP was expressed together with puuR in the presence of 0.4 % glucose , the level of PuuP protein was repressed by PuuR , which was parallel with the level of of puuR mRNA and Western blotting of PuuP protein in E. coli MA261 Dcya , crp and its parent strain E. coli MA261 were performed as described in `` Materials and methods '' .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23719730" target="_blank">23719730</a>]<br></li>
<li>Furthermore , deletion of PuuR increases expression from the puuA and puuD promoters 15-fold , but putrescine induces only four-fold ( for cells grown in glucose-ammonia minimal medium ) ( Schneider and Reitzer , 2012 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23531166" target="_blank">23531166</a>]<br></li>
<li>Loss of PuuR elevated expression from the puuD promoter 22-fold in glucose-ammonia medium , 6-fold in glucose-ammonia-putrescine medium , and 1.6-fold in glucose-putrescine medium ( Fig. 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22636776" target="_blank">22636776</a>]<br></li>
<li>The complex of cAMP and CRP ( cAMP receptor protein ) inhibited the expression of PuuR in the absence of glucose .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23719730" target="_blank">23719730</a>]<br></li>
<li>Expression of PuuP was reduced in the presence of PuuR , a negative regulator for PuuP , and expression of PuuR was positively regulated by glucose , which reduces the level of cAMP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23719730" target="_blank">23719730</a>]<br></li>
<li>It also became clear that PuuR expression is positively regulated by glucose , which decreases the level of cAMP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23719730" target="_blank">23719730</a>]<br></li>
<li>When PuuP was expressed in the presence of PuuR , cell growth was delayed , because PuuR is a negative regulator of PuuP expression ( Nemoto et al. 2012 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23719730" target="_blank">23719730</a>]<br></li>
<li>The results indicate that the putrescine uptake by PotFGHI is more effective than that by PuuP in the presence of glucose , and conﬁrmed that PuuR is an effective negative regulator for PuuP expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23719730" target="_blank">23719730</a>]<br></li>
<li>Results obtained in this study indicate that both PotFGHI and PuuP contribute to the growth of polyamine-deﬁcient cells cultured in the presence of 0.4 % ( 22.2 mM ) enhanced through repression of PuuR expression , and PotFGHI activity was inhibited by accumulated polyamines glucose and 10 mM putrescine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23719730" target="_blank">23719730</a>]<br></li>
				</div></ul><br>
			<label class="collapse_DOM" for="_2"><font size="5px">PuuR protein domains</font></label>
				<input id="_2" type="checkbox">
				<div class="DOM"><br><ul><li>The N-terminal region of PuuR ( residues 1 to 70 ) shows signiﬁcant similarity to members of the HTH-XRE-family of transcriptional regulators ( 25 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22522900" target="_blank">22522900</a>]<br></li>
<li>These results indicate that the adenine residues in the mutation site on probe 10 mut1 and mut2 are important for binding PuuR , whereas the residues in probe 10 mut3 are not .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22522900" target="_blank">22522900</a>]<br></li>
<li>In this study , we report the identiﬁcation of binding sites and recognition motifs for PuuR DNA-binding in the puu gene cluster and discuss the mechanisms by which PuuR regulates the Puu pathway and intracellular putrescine concentration .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22522900" target="_blank">22522900</a>]<br></li>
<li>To further investigate the importance of individual bases in the predicted 15-bp PuuR recognition motif for PuuR binding , point mutations were introduced into the FC site ( Fig. 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22522900" target="_blank">22522900</a>]<br></li>
<li>According to the program , this alanine substitution ought to disable the formation of the PuuR HTH motif .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22522900" target="_blank">22522900</a>]<br></li>
<li>Three amino acids in the puuR ( 3Ala ) mutants that were predicted to comprise a key portion of the HTH DNA-binding motif of PuuR by a secondary structure prediction program ( Genetyx software , Genetyx Corp. ) were mutated to alanine residues ( Thr33Ala , His34Ala , and Ser35Ala ) ( 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22522900" target="_blank">22522900</a>]<br></li>
<li>One gene in the puu gene cluster encodes a transcription factor , PuuR , which has a helix-turn-helix DNA-binding motif .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22522900" target="_blank">22522900</a>]<br></li>
<li>DNA microarray analysis of an E. coli puuR mutant , in which three amino acid residues in the helix-turn-helix DNA-binding motif of PuuR were mutated to alanine to eliminate DNA-binding of PuuR , suggested that PuuR is a negative regulator of puu genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22522900" target="_blank">22522900</a>]<br></li>
<li>We predicted that puuR encodes a transcriptional regulator because a secondary structure prediction made from the amino acid sequence of PuuR revealed a helix-turn-helix ( HTH ) DNA-binding motif .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22522900" target="_blank">22522900</a>]<br></li>
				</div></ul><br>
			<label class="collapse_EVO" for="_3"><font size="5px">PuuR evolution</font></label>
				<input id="_3" type="checkbox">
				<div class="EVO"><br><ul><li>The N-terminal region of PuuR ( residues 1 to 70 ) shows signiﬁcant similarity to members of the HTH-XRE family of transcriptional regulators ( 25 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22522900" target="_blank">22522900</a>]<br></li>
				</div></ul><br>
			<label class="collapse_RP" for="_4"><font size="5px">Processes regulated by PuuR</font></label>
				<input id="_4" type="checkbox">
				<div class="RP"><br><ul><li>Expression of PuuP was reduced in the presence of PuuR , a negative regulator for PuuP , and expression of PuuR was positively regulated by glucose , which reduces the level of cAMP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23719730" target="_blank">23719730</a>]<br></li>
<li>Results Putrescine uptake by PotFGHI and PuuP The genes encoding proteins for putrescine uptake -- PotFGHI and PuuP -- and for PuuR , a negative regulator of puuP gene-expression ( Kurihara et al. 2008 ; Nemoto et al. 2012 ; Pistocchi et al. 1993 ) are illustrated in Fig. 1a .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23719730" target="_blank">23719730</a>]<br></li>
<li>Mechanism for Regulation of the Putrescine Utilization Pathway by the Transcription Factor PuuR in Escherichia coli K-12 Naoki Nemoto , a Shin Kurihara , a , b Yuzuru Kitahara , a , b Kei Asada , a , b Kenji Kato , b and Hideyuki Suzukia Division of Applied Biology , Graduate School of Science and Technology , Kyoto Institute of Technology , Goshokaido-cho , Matsugasaki , Sakyo-ku , Kyoto , Japan , a and Division of Integrated Life Science , Graduate School of Biostudies , Kyoto University , Oiwake-cho , Kitashirakawa , Sakyo-ku , Kyoto , Japanb In Escherichia coli , putrescine is metabolized to succinate for use as a carbon and nitrogen source by the putrescine utilization pathway ( Puu pathway ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22522900" target="_blank">22522900</a>]<br></li>
<li>[ PubMed : 9495755 ] Nemoto N , Kurihara S , Kitahara Y , Asada K , Kato K , Suzuki H. Mechanism for regulation of the putrescine utilization pathway by the transcription factor PuuR in Escherichia coli K-12 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23531166" target="_blank">23531166</a>]<br></li>
<li>J Bacteriol 178:5522 -- 5528 Nemoto N , Kurihara S , Kitahara Y , Asada K , Kato K , Suzuki H ( 2012 ) Mechanism for regulation of the putrescine utilization pathway by the transcription factor PuuR in Escherichia coli Oliver DB , Beckwith J ( 1982 ) Regulation of a membrane component required for protein-secretion in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23719730" target="_blank">23719730</a>]<br></li>
<li>Regulation of the Puu pathway by PuuR seems to be a simple , primary regulatory system that maintains the putrescine concentration in the cell , although there is a possibility of other regulatory mechanisms such as an aeration shift by FNR or ArcA for expression of the Puu pathway ( 14 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22522900" target="_blank">22522900</a>]<br></li>
<li>Regulation of GP pathway genes : PuuR and putrescine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22636776" target="_blank">22636776</a>]<br></li>
<li>These results suggest that PuuR regulates the intracellular putrescine concentration by the transcriptional regulation of genes in the Puu pathway , including puuR itself .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22522900" target="_blank">22522900</a>]<br></li>
				</div></ul><br>
			<label class="collapse_SIT" for="_5"><font size="5px">PuuR binding site</font></label>
				<input id="_5" type="checkbox">
				<div class="SIT"><br><ul><li>A 12-amino-acid sequence with His tag of pQE-80L , MRGSHHHHHHGS , was added to the N-terminal end of the PuuR protein to form His6-PuuR protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22522900" target="_blank">22522900</a>]<br></li>
<li>Puriﬁed His6-PuuR bound to probe 2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22522900" target="_blank">22522900</a>]<br></li>
<li>( D ) An alignment of the nucleotide sequences of all four sites protected by PuuR , which were observed in the DNase I footprint assay .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22522900" target="_blank">22522900</a>]<br></li>
<li>Nucleotide sequence alignments of all four sites protected by PuuR revealed a predicted 15-bp PuuR recognition motif ( Fig. 2D ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22522900" target="_blank">22522900</a>]<br></li>
<li>Bases in the predicted 15-bp PuuR recognition sequence ( AAAATATA ATGAACA ) are shown , and bases not in the predicted consensus sequence are shown with underlined letters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22522900" target="_blank">22522900</a>]<br></li>
<li>Mutations were introduced into the FC site because while the FC site is partially matched to the consensus sequence , PuuR recognizes the sequence motif in the FC site for binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22522900" target="_blank">22522900</a>]<br></li>
<li>Mapping of the four PuuR binding sites revealed that FA is located 2 to 16 bp downstream of the puuA transcription start site , FB overlaps with the transcription start site of puuD , FC is located between the transcription start site and the initiation codon of puuD , and FD is located downstream from the initiation codon of puuD ( Fig. 4B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22522900" target="_blank">22522900</a>]<br></li>
<li>The positions of the four PuuR binding sites , FA , FB , FC , and FD , are noted .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22522900" target="_blank">22522900</a>]<br></li>
				</ul></div><br>
			<label class="collapse_TU" for="_6"><font size="5px">Transcription units regulated by PuuR</font></label>
				<input id="_6" type="checkbox">
				<div class="TU"><br><ul><li>These strains were designed by increasing puuA and puuD copy numbers using a pUC19 vector in the puuR ( 3Ala ) host cell , in which repression of puu gene expression by PuuR was disrupted .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22522900" target="_blank">22522900</a>]<br></li>
<li>Furthermore , deletion of PuuR increases expression from the puuA and puuD promoters 15-fold , but putrescine induces only four-fold ( for cells grown in glucose-ammonia minimal medium ) ( Schneider and Reitzer , 2012 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23531166" target="_blank">23531166</a>]<br></li>
<li>This might suggest that the activities encoded by the puuA and puuD transcripts promote expression of the puuCBE operon , perhaps via the predicted transcription factor PuuR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16857675" target="_blank">16857675</a>]<br></li>
<li>The results shown above suggest that the promoters for puuA and puuD are located in the intergenic regions of puuA and puuD and are negatively regulated by PuuR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22522900" target="_blank">22522900</a>]<br></li>
<li>Results Putrescine uptake by PotFGHI and PuuP The genes encoding proteins for putrescine uptake -- PotFGHI and PuuP -- and for PuuR , a negative regulator of puuP gene expression ( Kurihara et al. 2008 ; Nemoto et al. 2012 ; Pistocchi et al. 1993 ) are illustrated in Fig. 1a .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23719730" target="_blank">23719730</a>]<br></li>
<li>When puuP was expressed together with puuR in the presence of 0.4 % glucose , the level of PuuP protein was repressed by PuuR , which was parallel with the level of of puuR mRNA and Western blotting of PuuP protein in E. coli MA261 Dcya , crp and its parent strain E. coli MA261 were performed as described in `` Materials and methods '' .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23719730" target="_blank">23719730</a>]<br></li>
<li>Expression of PuuP was reduced in the presence of PuuR , a negative regulator for PuuP , and expression of PuuR was positively regulated by glucose , which reduces the level of cAMP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23719730" target="_blank">23719730</a>]<br></li>
<li>It also became clear that PuuR expression is positively regulated by glucose , which decreases the level of cAMP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23719730" target="_blank">23719730</a>]<br></li>
				</div></ul></div><br>
	</div>
</body>
</html>