<!doctype html>
<style>
@charset "utf-8";
body {
	height: 100%;
	margin: 0;
	padding: 0;
}

.banner {
	position: fixed;
	top: 0;
	width: 100%;
	height: 65px;
	background: gray;
	line-height: 65px;
}
.spacer {
	height: 10px;
}

.main-content {
	padding: 5%;
}

.collapse_ACT {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-style: none solid none none;
	border-color: #B4045F;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_ACT + input{
	display: none;
}
.collapse_ACT + input + div{
	display: none;
}
.collapse_ACT + input:checked + div{
	display: block;
}
.ACT {
	text-align: justify;
	text-justify: inter-word;
}
.ACT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B4045F;
}
.ACT_collapse_similarity:hover{
	background: #e5d3dc;
}
.ACT_collapse_similarity + input {
	display: none;
}
.ACT_collapse_similarity + input + div{
	display: none;
}

.ACT_collapse_similarity + input:checked + div{
	display: block;
}
.ACT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.ACT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B4045F;
	padding-left: 5px;
}
.ACT_collapse_similarity:hover{
	background: #e5d3dc;
}
.ACT_collapse_similarity + input {
	display: none;
}
.ACT_collapse_similarity + input + div{
	display: none;
}
.ACT_collapse_similarity + input:checked + div{
	display: block;
}
.ACT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.ACT_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #B4045F;
}
.ACT_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_DOM {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-style: none solid none none;
	border-color: #B404AE;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_DOM + input{
	display: none;
}
.collapse_DOM + input + div{
	display: none;
}
.collapse_DOM + input:checked + div{
	display: block;
}
.DOM {
	text-align: justify;
	text-justify: inter-word;
}
.DOM_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B404AE;
	padding-left: 5px;
}
.DOM_collapse_similarity:hover{
	background: #e8d7e7;
}
.DOM_collapse_similarity + input {
	display: none;
}
.DOM_collapse_similarity + input + div{
	display: none;
}
.DOM_collapse_similarity + input:checked + div{
	display: block;
}
.DOM_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}.DOM_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #B404AE;
}
.DOM_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_EVO {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #7401DF;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_EVO + input{
	display: none;
}
.collapse_EVO + input + div{
	display: none;
}
.collapse_EVO + input:checked + div{
	display: block;
}
.EVO {
	text-align: justify;
	text-justify: inter-word;
}
.EVO_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #7401DF;
	padding-left: 5px;
}
.EVO_collapse_similarity:hover{
	background: #e2d7ed;
}
.EVO_collapse_similarity + input {
	display: none;
}
.EVO_collapse_similarity + input + div{
	display: none;
}
.EVO_collapse_similarity + input:checked + div{
	display: block;
}
.EVO_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.EVO_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #7401DF;
}
.EVO_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_RP {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #0174DF;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_RP + input{
	display: none;
}
.collapse_RP + input + div{
	display: none;
}
.collapse_RP + input:checked + div{
	display: block;
}
.RP {
	text-align: justify;
	text-justify: inter-word;
}
.RP_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #0174DF;
	padding-left: 5px;
}
.RP_collapse_similarity:hover{
	background: #c9d7e2;
}
.RP_collapse_similarity + input {
	display: none;
}
.RP_collapse_similarity + input + div{
	display: none;
}
.RP_collapse_similarity + input:checked + div{
	display: block;
}
.RP_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.RP_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #0174DF;
}
.RP_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_SIT {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #0489B1;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_SIT + input{
	display: none;
}
.collapse_SIT + input + div{
	display: none;
}
.collapse_SIT + input:checked + div{
	display: block;
}
.SIT {
	text-align: justify;
	text-justify: inter-word;
}
.SIT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #0489B1;
	padding-left: 5px;
}
.SIT_collapse_similarity:hover{
	background: #d0dfe2;
}
.SIT_collapse_similarity + input {
	display: none;
}
.SIT_collapse_similarity + input + div{
	display: none;
}
.SIT_collapse_similarity + input:checked + div{
	display: block;
}
.SIT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.SIT_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #0489B1;
}
.SIT_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_TU {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #04B486;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_TU + input{
	display: none;
}
.collapse_TU + input + div{
	display: none;
}
.collapse_TU + input:checked + div{
	display: block;
}
.TU {
	text-align: justify;
	text-justify: inter-word;
}
.TU_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #04B486;
	padding-left: 5px;
}
.TU_collapse_similarity:hover{
	background: #d0e2de;
}
.TU_collapse_similarity + input {
	display: none;
}
.TU_collapse_similarity + input + div{
	display: none;
}
.TU_collapse_similarity + input:checked + div{
	display: block;
}
.TU_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.TU_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #04B486;
}
.TU_similar > a {
	text-decoration: none;
	color: gray;
}

ul { 
	list-style-type: none;
	padding: 0;
	margin: 0;
}
ul li > a {
	text-decoration: none;
	color: gray;
}
ul li:hover, ul li:focus {
	background: #F0F0F0;
}

ul li:first-child span {
  border-left:none;  
}

</style>
<html>
<head>
<meta charset="utf-8">
<title>TF properties - QseB</title>
<meta name="theme-color" content="#ffffff">
</head>
<body>
	<div class="banner">
		<center><font color="white" size="16px" face="arial">QseB</font></center>
	</div>
	<div align="center" class="main-content">
		<div class="spacer">
	</div><br>			<label class="collapse_ACT" for="_1"><font size="5px">QseB transcriptional activity</font></label>
				<input id="_1" type="checkbox">
				<div class="ACT"><br><ul><li>Another intriguing aspect of our ﬁndings is that cross-talk is not pro-miscuous , as no other sensor was identiﬁed in our screen and deletion of pmrB in the qseC mutant entirely suppressed QseB activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24062463" target="_blank">24062463</a>]<br></li>
<li>In addition to these amino acid-related transcription factors , Lrp was found to regulate a total of 15 transcription factors ( Fig. 6 , grey symbols ) , which are involved in the stress-response and life-style selection of E. coli , such as GadW for acid response , YedW for copper and peroxide response , EnvR for response to drugs , QseB for quorum sensing , BluR for bioﬁlm formation , and MatA and SlyA for planktonic growth .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2162465" target="_blank">2162465</a>]<br></li>
<li>We have previously reported that the ΔqseC-related defects in pathogenic E. coli are a result of constitutive activation of the QseB RR , resulting in increased qseB expression , aberrant regulation of conserved metabolic pathways , and down-regulation of viru-lence-associated genes ( 8 , 9 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24062463" target="_blank">24062463</a>]<br></li>
<li>It should also be noted that purified QseB did not bind to the 5 - UTR of the mqsR-ygiT operon ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19690171" target="_blank">19690171</a>]<br></li>
<li>However , although PmrB readily phosphorylates QseB in vitro , it is signiﬁcantly less efﬁcient at dephosphorylating QseB , compared with QseC , thereby explaining the increased levels of active QseB in the qseC mutant .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24062463" target="_blank">24062463</a>]<br></li>
<li>Thus , although PmrB can phosphorylate QseB at rates relatively comparable to QseC , it is signiﬁcantly less efﬁcient in its ability to dephosphorylate QseB ∼ P , resulting in the disproportionate activation of QseB in the qseC mutant .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24062463" target="_blank">24062463</a>]<br></li>
<li>If PmrB phosphorylates QseB in the absence of QseC , then ablating the PmrB kinase activity by site-directed mutagenesis should yield the same outcome as the pmrB deletion .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24062463" target="_blank">24062463</a>]<br></li>
<li>This study documents another strong interaction between a RR and a noncognate sensor : PmrB readily phosphorylates QseB ; however , PmrB phosphatase activity toward QseB is slow , thus explaining the observed QseB activation when QseC is not there .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24062463" target="_blank">24062463</a>]<br></li>
<li>We performed a time course , assessing dephosphorylation of in vitro phosphorylated QseB ( QseB ∼ P ) incubated with PmrB-enriched membrane vesicles ( MV ) , MV lacking PmrB , or QseC-enriched MV .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24062463" target="_blank">24062463</a>]<br></li>
<li>QseB puriﬁcation , generation of QseC - and PmrB-enriched membranes , and execution of phosphotransfer assays were performed as described in ref .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24062463" target="_blank">24062463</a>]<br></li>
<li>In striking contrast , 44 % of QseB was dephosphorylated after 5 min of incubation in the presence of QseC , and by 60 min of incubation , only 2.5 % of QseB ∼ P remained phosphorylated ( Fig. 2E ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24062463" target="_blank">24062463</a>]<br></li>
<li>Incubation of QseB ∼ P with MV lacking PmrB or QseC or incubation of QseB in the absence of MV altogether did not impact the QseB phosphorylation state ( Fig. 2D , Center and Right ) , indicating that QseB ∼ P does not get dephosphorylated by other kinases/phosphatases in the MV and it does not undergo signiﬁcant spontaneous dephosphorylation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24062463" target="_blank">24062463</a>]<br></li>
<li>We have shown , however , that qseB and qseC are transcriptionally linked ( Clarke and Sperandio , 2005 ) and that QseBC compromise a functional signalling cascade where , upon autophosphorylation , QseC transfers its phosphate to QseB ( M.B. Clarke and V. Sperandio , unpubl .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16135237" target="_blank">16135237</a>]<br></li>
<li>This conclusion is strengthened by the observation that , upon autophosphorylation , QseC transfers its phosphate to QseB ( M.B. Clarke and V. Sperandio , unpubl .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16194231" target="_blank">16194231</a>]<br></li>
<li>When cell-to-cell signalling systems are activated , QseB is expressed and then phosphorylated by QseC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16194231" target="_blank">16194231</a>]<br></li>
<li>When cell-to-cell signalling is activated , we hypothesize that QseC , a predicted sensor kinase , actively autophosphorylates itself on a conserved histidine residue and transfers its phosphate to a conserved aspartate residue of QseB , its putative response-regulator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16135237" target="_blank">16135237</a>]<br></li>
<li>When quorum sensing occurs , QseC autophosphorylates and then transfers its phosphate to a conserved aspartate residue on QseB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16194231" target="_blank">16194231</a>]<br></li>
<li>After 120 min , QseC transferred a large proportion of its phosphate to QseB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16803956" target="_blank">16803956</a>]<br></li>
<li>Consequently , when the autophosphorylation of QseC is blocked by PE , QseC can not transfer a phosphate to QseB , and unphosphorylated QseB can not interact with the flhDC promoter to activate transcription ( 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16803956" target="_blank">16803956</a>]<br></li>
<li>QseB QseC activate transcription of the flagella regulon responsible for swimming motility in EHEC ( 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16803956" target="_blank">16803956</a>]<br></li>
<li>Upon sensing these signals , QseC autophosphorylates and then transfers its phosphate to QseB , which then regulates both is own transcription and the transcription of the flagella and motility genes ( 8 , 20 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16803956" target="_blank">16803956</a>]<br></li>
<li>The cognate QseC was , as expected , superior in its capacity to phosphorylate greater amounts of QseB within the same time frame , as indicated by kinetic analysis ( Fig. 2 B and C ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24062463" target="_blank">24062463</a>]<br></li>
<li>This puriﬁed QseB protein should be mostly unphosphorylated , as it was highly overexpressed in the strain and its cognate response-regulator , QseC , was expressed at normal , wildtype levels .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16135237" target="_blank">16135237</a>]<br></li>
<li>Of relevance to EHEC , QseB QseC comprise a two-component system , in which QseC is the predicted HK and QseB the predicted response-regulator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16803956" target="_blank">16803956</a>]<br></li>
<li>QseB is predicted to be the cognate response-regulator for QseC ( Fig. 3A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16803956" target="_blank">16803956</a>]<br></li>
<li>The QseC HK initiated autophosphorylation after 10 min in the presence of epi and transferred its phosphate to its cognate response-regulator , QseB , after 30 min ( Fig. 3B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16803956" target="_blank">16803956</a>]<br></li>
<li>Here , we report that in the case of the quorum-sensing Escherichia coli ( Qse ) BC TCS , absence of the cognate sensor QseC leads to robust , constitutive activation of the QseB response-regulator by the noncognate polymyxin resistance ( Pmr ) sensor kinase PmrB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24062463" target="_blank">24062463</a>]<br></li>
<li>Bearson BL , Bearson SM , Lee IS , Brunelle BW ( 2010 ) The Salmonella enterica serovar Typhimurium QseB response-regulator negatively regulates bacterial motility and swine colonization in the absence of the QseC sensor kinase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24062463" target="_blank">24062463</a>]<br></li>
<li>In order to test our hypothesis , we puriﬁed QseB in native conditions to perform DNA binding assays .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16194231" target="_blank">16194231</a>]<br></li>
<li>In order to study the binding of QseB to the ﬂhDC promoter , EMSAs were performed using puriﬁed QseB-His and PCR-ampliﬁed DNA probes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16135237" target="_blank">16135237</a>]<br></li>
<li>Electrophoretic mobility shift assays ( EMSAs ) and DNase I footprints In order to study the direct binding of QseB to its own promoter , EMSAs were performed using the puriﬁed QseB-His and PCR ampliﬁed DNA probes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16194231" target="_blank">16194231</a>]<br></li>
<li>Puriﬁcation of QseB , EMSAs and DNaseI footprinting The E. coli strain containing pVS154 was grown at 37 ∞ C in LB to an OD600 of 0.7 , at which point arabinose was added to a ﬁnal volume of 0.2 % and allowed to induce for 3 h. His-tagged QseB protein was then puriﬁed under native conditions using nickel columns according to the manufacturer 's instructions ( Qiagen ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16135237" target="_blank">16135237</a>]<br></li>
<li>QseB protein was puriﬁed under native conditions and stained with Coomassie .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16194231" target="_blank">16194231</a>]<br></li>
<li>This appears to be the ﬁrst report about the regulation of bioﬁlm by QseB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16352847" target="_blank">16352847</a>]<br></li>
<li>AI-2 increases motility in E. coli through QseB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16352847" target="_blank">16352847</a>]<br></li>
<li>Based on this increase in motility through QseB upon AI-2 addition , we hypothesized that AI-2 induces bioﬁlm formation by inducing motility and that this increase in motility leads to increased attachment .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16352847" target="_blank">16352847</a>]<br></li>
<li>However , loss of PmrA leads to constitutive , low-level induction of qseBC that is abolished in the absence of QseB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24062463" target="_blank">24062463</a>]<br></li>
<li>In addition to PmrB activating QseB on the protein level , we found that the PmrA response-regulator contributes to qseB transcription in the absence of QseC and PmrA speciﬁcally binds the qseBC promoter , indicative of a direct regulation of qseBC gene transcription by PmrAB under physiological conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24062463" target="_blank">24062463</a>]<br></li>
<li>Taken together , these data indicate that in the absence of QseC , qseBC up-regulation comes from a possible synergistic effect of aberrantly activated QseB ∼ P by PmrB and the activity of PmrA ( the phosphorylation state for this function remains unknown ) on the qseBC promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24062463" target="_blank">24062463</a>]<br></li>
<li>Signal reception by PmrB leads to phosphorylation of PmrA and QseB and changes the afﬁnity of the RR for the qseBC promoter and/or allows synergistic binding ( and transcription ) that is controlled by the phosphatase activity of QseC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24062463" target="_blank">24062463</a>]<br></li>
<li>When both sites are occupied by QseB , auto-regulation occurs , and qseBC transcription is greatly increased .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16194231" target="_blank">16194231</a>]<br></li>
<li>Loss of QseC function tips this delicate balance toward more phosphorylated QseB , leading to elevated qseBC transcription , more QseB ∼ P , and global gene deregulation , as we previously reported .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24062463" target="_blank">24062463</a>]<br></li>
<li>QseC is a sensor histidine kinase , and QseB is a transcription regulator that binds to the 5 - UTR of the qseBC operon and activates transcription of this operon ( 50 , 51 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19690171" target="_blank">19690171</a>]<br></li>
<li>Addition of the PmrAB activating signal , ferric iron , induced qseBC gene expression in the wild-type UPEC strain in a PmrB-dependent manner that appears to involve QseB and PmrA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24062463" target="_blank">24062463</a>]<br></li>
<li>Based on these observations , QseBC is subject to PmrB control under physiological conditions in a manner that appears to involve PmrA and QseB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24062463" target="_blank">24062463</a>]<br></li>
<li>If PmrB activates both QseB and PmrA in response to Fe3 + , this observation may be the result of functional redundancy between the two RRs that allows signal-dependent promoter activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24062463" target="_blank">24062463</a>]<br></li>
<li>Absence of PmrA may also alter the kinase/phosphatase kinetics of PmrB to QseB , thereby contributing to the observed reduction in qseB transcript .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24062463" target="_blank">24062463</a>]<br></li>
<li>We show that qseBC are transcribed in an operon and have identiﬁed the minimal region necessary for QseB to activate qseBC Genomic c D D N N A A Negative control ( no RTase Fig. 1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16194231" target="_blank">16194231</a>]<br></li>
<li>These results indicate that QseB may act to autoregulate its own transcription through binding to low - and high-afﬁnity sites found in its promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16194231" target="_blank">16194231</a>]<br></li>
<li>Phosphorylated QseB is then free to-bind both the high - and low-afﬁnity binding sites , and to autoactivate its own transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16194231" target="_blank">16194231</a>]<br></li>
<li>This suggests that , even though the binding sites may be reversed , QseB is acting in a similar manner to activate transcription of both its own promoter and the ﬂhDC promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16194231" target="_blank">16194231</a>]<br></li>
<li>Thus , QseB may be acting in a similar manner to OmpR to regulate the transcription of ﬂhDC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16135237" target="_blank">16135237</a>]<br></li>
<li>According to our data , QseB may be acting in a similar manner on its own promoter to regulate transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16194231" target="_blank">16194231</a>]<br></li>
<li>These results suggest to us that phosphorylation of QseB is necessary for it to be able to actively bind its regulatory promoter region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16194231" target="_blank">16194231</a>]<br></li>
<li>These analyses have allowed us to determine the potential consensus sequence to which QseB binds in order to regulate transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16135237" target="_blank">16135237</a>]<br></li>
<li>Phosphorylated QseB binds to and activates the transcription of the flhDC promoter , which encodes the FlhDC master regulators of the flagella regulon , and also binds to its own promoter ( 8 , 20 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16803956" target="_blank">16803956</a>]<br></li>
<li>QseB binds to two regions in the ﬂhDC promoter To determine if QseB directly interacts with the ﬂhDC promoter region to activate transcription , we puriﬁed QseB protein under native conditions as described previously ( M.B. Clarke and V. Sperandio , submitted ) to use in electrophoretic mobility shift assays ( EMSAs ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16135237" target="_blank">16135237</a>]<br></li>
<li>Finally , electro-phoretic mobility shift assays , competition experiments and DNase I footprints were performed and demonstrated that QseB directly binds to two sites in its own promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16194231" target="_blank">16194231</a>]<br></li>
<li>We also demonstrate that QseB binds directly to its own promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16194231" target="_blank">16194231</a>]<br></li>
<li>These studies suggest a mechanism by which QseB binds directly to two sites in its own promoter in order to activate its own transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16194231" target="_blank">16194231</a>]<br></li>
<li>The addition of unlabelled bla probe , as a negative control reaction , shows no competition , indicating that QseB speciﬁcally binds to two different areas of its own promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16194231" target="_blank">16194231</a>]<br></li>
<li>Consequently , addition of an exogenous small phosphor-donor such as acetyl phosphate allowed phospho-QseB to speciﬁcally shift its promoter region ( as compared with a bla negative control ) , indicating that QseB phosphorylation is essential for DNA binding and , consequently , transcriptional regulation ( Fig. 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16194231" target="_blank">16194231</a>]<br></li>
<li>We can not rule out the possibility that QseB is phosphorylated by other sensor kinases .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16135237" target="_blank">16135237</a>]<br></li>
<li>We hypothesized that the addition of 0.1 M acetyl phosphate , a small phosphodonor , to the EMSA reaction might result in a phosphorylated , active QseB protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16194231" target="_blank">16194231</a>]<br></li>
<li>Our studies demonstrate that QseB is speciﬁcally binding to two areas of the ﬂhDC promoter ( Fig. 8B ) , in a similar pattern to the binding of QseB to its own promoter ( M.B. Clarke and V. Sperandio , submitted ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16135237" target="_blank">16135237</a>]<br></li>
<li>450 M. B. Clarke and V. Sperandio when phosphorylated QseB was added concentrations .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16194231" target="_blank">16194231</a>]<br></li>
<li>We aligned these ﬂhDC sequences ( Fig. 8A ) with sequences bound by QseB in the qseBC promoter ( Clarke and Sperandio , 2005 ) to determine a consensus sequence to which phosphorylated QseB binds .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16135237" target="_blank">16135237</a>]<br></li>
<li>Furthermore , the observation that , upon phosphorylation , QseC will transfer its phosphate to QseB ( M.B. Clarke and V. Sperandio , unpublished data ) and that phosphorylated QseB speciﬁcally binds to the qseBC promoter further enhances this suggestion .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16194231" target="_blank">16194231</a>]<br></li>
<li>To investigate whether or not the phosphorylation of QseB was necessary for binding to the ﬂhDC promoter fragments , we performed EMSAs to which 0.1 M of the phosphodonor acetyl phosphate was added .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16135237" target="_blank">16135237</a>]<br></li>
<li>Given that phosphorylated QseB appeared to be binding to two separate regions of the ﬂhDC promoter , we were interested in whether or not QseB bound cooperatively or in a speciﬁc order to the promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16135237" target="_blank">16135237</a>]<br></li>
<li>Double EMSA with QseB In order to determine whether QseB bound cooperatively or in a speciﬁc order to the qseBC promoter , we performed double-gel shifts ( Fig. 7A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16194231" target="_blank">16194231</a>]<br></li>
<li>Higher concentrations of phosphorylated QseB ( 8 -- 16 mg ) appear to saturate both binding sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16135237" target="_blank">16135237</a>]<br></li>
<li>End-labelled promoter fragments were incubated at identical concentrations ( 5 ng ) together with puriﬁed , phosphorylated QseB protein in increasing concentrations ( 0 -- 16 mg ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16194231" target="_blank">16194231</a>]<br></li>
<li>The competition assay was performed using 3 mg of puriﬁed , phosphorylated QseB and add-ing increasing amounts of unlabelled qseBC promoter ( -120 bp to +130 bp ) probe .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16194231" target="_blank">16194231</a>]<br></li>
<li>An EMSA was performed using phosphorylated QseB and a negative control , the bla promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16194231" target="_blank">16194231</a>]<br></li>
<li>Double EMSA and competitions of qseBC fragments using phosphorylated QseB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16194231" target="_blank">16194231</a>]<br></li>
<li>As binding afﬁnity of phosphorylated QseB to each promoter region may play a role in the regulation of ﬂhDC , we designed a competition assay to determine the relative afﬁnity of QseB for the binding sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16135237" target="_blank">16135237</a>]<br></li>
<li>These data further substantiate the theory that phosphorylated QseB has increased binding afﬁnity for the proximal region ( 120 bp to +130 bp ) of the qseBC promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16194231" target="_blank">16194231</a>]<br></li>
<li>When QseB is highly phosphorylated by QseC , QseB is available to-bind to its low-afﬁnity binding site in the distal promoter region ( -350 bp to -500 bp ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16194231" target="_blank">16194231</a>]<br></li>
<li>Figure 7B shows that phosphorylated QseB , added in increasing concentrations from 0 mg ml-1 to 5 mg ml-1 , protects a region of the distal promoter from -710 bp to -723 bp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16135237" target="_blank">16135237</a>]<br></li>
<li>Figure 5A reveals that phosphorylated QseB binds to and shifts the -300 bp to +50 bp ﬂhDC promoter fragment .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16135237" target="_blank">16135237</a>]<br></li>
<li>As we observed that phosphorylated QseB also bound directly to the distal ﬂhDC promoter region ( -650 bp to -900 bp ) , we again performed DNaseI footprint analysis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16135237" target="_blank">16135237</a>]<br></li>
				</div></ul><br>
			<label class="collapse_DOM" for="_2"><font size="5px">QseB protein domains</font></label>
				<input id="_2" type="checkbox">
				<div class="DOM"><br><ul><li>DNaseI footprint analyses have allowed us to determine the speciﬁc binding sites of phosphorylated QseB protein to the ﬂhDC promoter ( Fig. 7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16135237" target="_blank">16135237</a>]<br></li>
<li>Additionally , we determined the minimal regions necessary for QseBC activation of ﬂhDC and QseB binding sites within the ﬂhDC promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16135237" target="_blank">16135237</a>]<br></li>
<li>This phosphorylated QseB was incubated with the two ﬂhDC promoter regions that showed QseBC-dependent activation in the deletion analyses ( Fig. 4B and C ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16135237" target="_blank">16135237</a>]<br></li>
<li>In agreement with these studies , we have previously shown that the QseC phosphatase domain is critical in maintaining optimal phosphorylated QseB levels and gene expression ( 9 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24062463" target="_blank">24062463</a>]<br></li>
<li>When cell-to-cell signalling is activated , we hypothesize that QseC , a predicted sensor kinase , actively autophosphorylates itself on a conserved histidine residue and transfers its phosphate to a conserved aspartate residue of QseB , its putative response regulator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16135237" target="_blank">16135237</a>]<br></li>
<li>When quorum sensing occurs , QseC autophosphorylates and then transfers its phosphate to a conserved aspartate residue on QseB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16194231" target="_blank">16194231</a>]<br></li>
<li>QseC also contains an ATPase domain , which may allow it to exhibit phosphatase activity towards QseB , and a conserved EAL-domain , which has been implicated in the formation of bioﬁlms in Vibrio cholerae ( Tischler and Camilli , 2004 ) ( Fig. 1A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16194231" target="_blank">16194231</a>]<br></li>
<li>Results The QseBC two-component system is transcribed in an operon and autoregulates its own transcription The predicted response regulator , QseB , shares domain homology with typical response regulators , and also appears to contain a conserved helix -- turn -- helix DNA-binding domain ( Fig. 1A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16194231" target="_blank">16194231</a>]<br></li>
<li>QseB is predicted to be the response regulator , while QseC shares domain homology with sensor kinases .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16194231" target="_blank">16194231</a>]<br></li>
<li>QseB , the putative response regulator , contains both a typical response regulator domain that may receive the phosphate from QseC and a DNA-binding domain that may allow QseB to regulate gene expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16194231" target="_blank">16194231</a>]<br></li>
<li>These possibilities open unique avenues of study , investigating not only contacts between PmrA and QseB , but also potential -- yet unexplored -- interactions of QseC with PmrA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24062463" target="_blank">24062463</a>]<br></li>
<li>This study documents another strong interaction between a RR and a noncognate sensor : PmrB readily phosphorylates QseB ; however , PmrB phosphatase activity toward QseB is slow , thus explaining the observed QseB activation when QseC is not there .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24062463" target="_blank">24062463</a>]<br></li>
<li>It is thus possible that the interacting interfaces of PmrB and QseC share common features that enable similar protein − protein interactions with the QseB RR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24062463" target="_blank">24062463</a>]<br></li>
				</div></ul><br>
			<label class="collapse_EVO" for="_3"><font size="5px">QseB evolution</font></label>
				<input id="_3" type="checkbox">
				<div class="EVO"><br><ul><li>QseBC are homologous to the PmrAB two-component system of S. typhimurium , with QseB being the predicted response regulator and QseC being the putative sensor kinase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16194231" target="_blank">16194231</a>]<br></li>
				</div></ul><br>
			<label class="collapse_RP" for="_4"><font size="5px">Processes regulated by QseB</font></label>
				<input id="_4" type="checkbox">
				<div class="RP"><br><ul><li>Consequently , addition of an exogenous small phosphor-donor such as acetyl phosphate allowed phospho-QseB to speciﬁcally shift its promoter region ( as compared with a bla negative control ) , indicating that QseB phosphorylation is essential for DNA binding and , consequently , transcriptional regulation ( Fig. 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16194231" target="_blank">16194231</a>]<br></li>
<li>As expected if MqsR was required for signal-transduction to QseB , expression of qseB was induced fourfold in a dose-dependent manner in M9C glu ( Fig. 6B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16352847" target="_blank">16352847</a>]<br></li>
<li>Addition of the PmrAB activating signal , ferric iron , induced qseBC gene-expression in the wild-type UPEC strain in a PmrB-dependent manner that appears to involve QseB and PmrA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24062463" target="_blank">24062463</a>]<br></li>
<li>Based on these observations , QseBC is subject to PmrB control under physiological conditions in a manner that appears to involve PmrA and QseB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24062463" target="_blank">24062463</a>]<br></li>
<li>Interestingly , the genes coding for local regulators of the genes for individual amino acids are under the control of Lrp , including AdiY ( a regulator of Arg regulon ) , CysB ( a regulator of the Cys regulon ) , GadW ( a regulator of Glu reg ¬ Lrp Network Y J YhjI AdiY OLm r pR BluR YbaO YedW CadX CadW SdiA PhoP H-NS Cbl CysB TyrR TdcR DeoT TdcA CadX CadW DctR SdiA EnvR TdcA TdcA FabR StpA GadW LeuO GadE GadW GadX DtcR SlyA LeuO BglG StpA QseB Rob LrhA MatA NanR ~ 20 TFs FlhDC FliE LrhA FlhDC ~ 10 TFs FliZ MatA FlhDC QseCB Fig. 6 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2162465" target="_blank">2162465</a>]<br></li>
<li>In addition to these amino acid-related transcription factors , Lrp was found to regulate a total of 15 transcription factors ( Fig. 6 , grey symbols ) , which are involved in the stress-response and life-style selection of E. coli , such as GadW for acid response , YedW for copper and peroxide response , EnvR for response-to-drugs , QseB for quorum-sensing , BluR for bioﬁlm formation , and MatA and SlyA for planktonic growth .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2162465" target="_blank">2162465</a>]<br></li>
<li>Phosphorylated QseB binds to and activates the transcription of the flhDC promoter , which encodes the FlhDC master regulators of the flagella regulon , and also binds to its own promoter ( 8 , 20 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16803956" target="_blank">16803956</a>]<br></li>
<li>The array results also showed that MqsR regulates motility by controlling not only QseB ( Fig. 5A ) but also csrA , which is downregulated in the mqsR mutant ( twofold ) and which regulates motility master regulon expression in E. coli ( 50 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16352847" target="_blank">16352847</a>]<br></li>
<li>We also found that AI-2 stimulates bioﬁlm formation via the uncharacterized protein MqsR by showing that MqsR induces motility ( Fig. 4 ) and bioﬁlm formation in both batch and continuous systems ( Fig. 1B and 2 ) , that AI-2 stimulates motility through MotA ( Fig. 5B ) and bioﬁlm formation through MqsR ( Fig. 1A ) , and that MqsR stimulates QseB ( Fig. 5A ) , which controls motility in E. coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16352847" target="_blank">16352847</a>]<br></li>
<li>Using this infection model , we showed that a qseC mutant is attenuated for virulence in rabbits ( Fig. 4 ) , thereby establishing that QseB QseC are involved in pathogenesis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16803956" target="_blank">16803956</a>]<br></li>
<li>In this study , we determined the source of QseB phosphorylation , using uropathogenic E. coli ( UPEC ) as a model organism .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24062463" target="_blank">24062463</a>]<br></li>
<li>Kostakioti M , Hadjifrangiskou M , Pinkner JS , Hultgren SJ ( 2009 ) QseC-mediated dephosphorylation of QseB is required for expression of genes associated with virulence in uropathogenic Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24062463" target="_blank">24062463</a>]<br></li>
<li>Loss of QseC function tips this delicate balance toward more phosphorylated QseB , leading to elevated qseBC transcription , more QseB ∼ P , and global gene deregulation , as we previously reported .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24062463" target="_blank">24062463</a>]<br></li>
<li>QseB and QseC are a two-component , quorum-sensing controlled regulator system for motility ( 45 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16352847" target="_blank">16352847</a>]<br></li>
<li>The genes encoding these putative E. coli regulators , renamed quorum-sensing E. coli regulators B and C ( QseBC ) , encode a predicted 24 kDa response regulator ( QseB ) and a predicted 50 kDa sensor kinase ( QseC ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11929534" target="_blank">11929534</a>]<br></li>
<li>As part of the 300-gene , quorum-sensing regulon in E. coli ( 15 , 36 , 44 ) , qseBC ( b3025 and b3026 ) are organized in an operon in the E. coli chromosome with QseB playing a role as a response regulator and QseC playing a role as the sensor kinase ( 45 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16352847" target="_blank">16352847</a>]<br></li>
				</div></ul><br>
			<label class="collapse_SIT" for="_5"><font size="5px">QseB binding site</font></label>
				<input id="_5" type="checkbox">
				<div class="SIT"><br><ul><li>These analyses have allowed us to determine the potential consensus sequence to which QseB binds in order to regulate transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16135237" target="_blank">16135237</a>]<br></li>
<li>We aligned these ﬂhDC sequences ( Fig. 8A ) with sequences bound by QseB in the qseBC promoter ( Clarke and Sperandio , 2005 ) to determine a consensus sequence to which phosphorylated QseB binds .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16135237" target="_blank">16135237</a>]<br></li>
<li>These studies suggest a mechanism by which QseB binds directly to two sites in its own promoter in order to activate its own transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16194231" target="_blank">16194231</a>]<br></li>
<li>These results suggested that two binding sites for QseB might exist .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16194231" target="_blank">16194231</a>]<br></li>
<li>Data obtained from the double-gel shifts ( Fig. 7 ) may indicate that phosphorylated QseB is binding ﬁrst to a high-afﬁnity site in the proximal promoter fragment ( -120 bp to +130 bp ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16194231" target="_blank">16194231</a>]<br></li>
<li>A high concentration of phosphorylated QseB may then saturate the qseBC high-afﬁnity binding site ( -120 bp to +130 bp ) , followed by binding of the low-afﬁnity site ( -360 bp to -500 bp ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16194231" target="_blank">16194231</a>]<br></li>
<li>Figure 5A reveals that phosphorylated QseB binds to and shifts the -300 bp to +50 bp ﬂhDC promoter fragment .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16135237" target="_blank">16135237</a>]<br></li>
<li>As we observed that phosphorylated QseB also bound directly to the distal ﬂhDC promoter region ( -650 bp to -900 bp ) , we again performed DNaseI footprint analysis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16135237" target="_blank">16135237</a>]<br></li>
<li>When QseB is highly phosphorylated by QseC , QseB is available to bind to its low-afﬁnity binding site in the distal promoter region ( -350 bp to -500 bp ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16194231" target="_blank">16194231</a>]<br></li>
<li>These results indicate that phosphorylated QseB is spe-ciﬁcally binding to the ﬂhDC promoter in two regions , the -900 bp to -650 bp region , and the -300 bp to +50 bp region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16135237" target="_blank">16135237</a>]<br></li>
<li>Additionally , our studies suggest that the distal , high-afﬁnity binding site for QseB is located between -650 bp and -900 bp on the ﬂhDC promoter ( Figs 4 -- 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16135237" target="_blank">16135237</a>]<br></li>
<li>These data suggest that phosphorylated QseB may have a higher-afﬁnity binding site located in the distal ﬂhDC promoter ( -650 bp to -900 bp ) and a lower-afﬁnity binding site located in the proximal ﬂhDC promoter ( -300 bp to +50 bp ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16135237" target="_blank">16135237</a>]<br></li>
<li>Figure 6 indicates that QseB may have a high-afﬁnity binding site , which is located in the more distal promoter region ( -650 bp to -900 bp ) , and a low-afﬁnity binding site , which is located in the proximal promoter region ( -300 bp to +50 bp ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16135237" target="_blank">16135237</a>]<br></li>
<li>Given that phosphorylated QseB appeared to be binding to two separate regions of the ﬂhDC promoter , we were interested in whether or not QseB bound cooperatively or in a speciﬁc order to the promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16135237" target="_blank">16135237</a>]<br></li>
<li>We also performed electrophoretic mobility shift assays , competition experiments and DNaseI footprints , which suggest that QseB directly binds the ﬂhDC promoter at high - and low-afﬁnity binding sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16135237" target="_blank">16135237</a>]<br></li>
<li>DNaseI footprints In order to determine the speciﬁc binding sites of QseB to the ﬂhDC promoter regions , we performed DNaseI footprint experiments .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16135237" target="_blank">16135237</a>]<br></li>
				</ul></div><br>
			<label class="collapse_TU" for="_6"><font size="5px">Transcription unit regulated by QseB</font></label>
				<input id="_6" type="checkbox">
				<div class="TU"><br><ul><li>Addition of the PmrAB activating signal , ferric-iron , induced qseBC gene expression in the wild-type UPEC strain in a PmrB-dependent manner that appears to involve QseB and PmrA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24062463" target="_blank">24062463</a>]<br></li>
<li>The array results also showed that MqsR regulates motility by controlling not only QseB ( Fig. 5A ) but also csrA , which is downregulated in the mqsR mutant ( twofold ) and which regulates motility master regulon expression in E. coli ( 50 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16352847" target="_blank">16352847</a>]<br></li>
<li>We also found that AI-2 stimulates bioﬁlm formation via the uncharacterized protein MqsR by showing that MqsR induces motility ( Fig. 4 ) and bioﬁlm formation in both batch and continuous systems ( Fig. 1B and 2 ) , that AI-2 stimulates motility through MotA ( Fig. 5B ) and bioﬁlm formation through MqsR ( Fig. 1A ) , and that MqsR stimulates QseB ( Fig. 5A ) , which controls motility in E. coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16352847" target="_blank">16352847</a>]<br></li>
<li>It should also be noted that purified QseB did not bind to the 5 - UTR of the mqsR-ygiT operon ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19690171" target="_blank">19690171</a>]<br></li>
<li>Results The QseBC two-component system is transcribed in an operon and autoregulates its own transcription The predicted response regulator , QseB , shares domain homology with typical response regulators , and also appears to contain a conserved helix -- turn -- helix DNA-binding domain ( Fig. 1A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16194231" target="_blank">16194231</a>]<br></li>
<li>We show that qseBC are transcribed in an operon and have identiﬁed the minimal region necessary for QseB to activate qseBC Genomic c D D N N A A Negative control ( no RTase Fig. 1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16194231" target="_blank">16194231</a>]<br></li>
<li>We know , however , that qseBC are transcribed in an operon and that QseC directly phosphorylates QseB in vitro ( M.B. Clarke and V. Sperandio , unpubl .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16194231" target="_blank">16194231</a>]<br></li>
<li>QseC is a sensor histidine kinase , and QseB is a transcription regulator that binds to the 5 - UTR of the qseBC operon and activates transcription of this operon ( 50 , 51 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19690171" target="_blank">19690171</a>]<br></li>
<li>After conﬁrming that qseBC are transcribed in an operon ( Fig. 1C ) and mapping the TSS responsive to QseBC ( Fig. 3 , lanes 3 and 4 ) , we showed that QseB and QseC regulate their own transcription through direct binding of QseB to proximal ( -120 bp to +130 bp ) and distal ( -360 bp to -500 bp ) regions of their promoter ( Fig. 7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16194231" target="_blank">16194231</a>]<br></li>
<li>This ﬂhDC promoter region is also upstream of the divergently transcribed yecG gene , suggesting that QseB may also play a role in the regulation of yecG expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16135237" target="_blank">16135237</a>]<br></li>
<li>Phosphorylated QseB binds to and activates the transcription of the flhDC promoter , which encodes the FlhDC master regulators of the flagella regulon , and also binds to its own promoter ( 8 , 20 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16803956" target="_blank">16803956</a>]<br></li>
<li>Upon sensing these signals , QseC autophosphorylates and then transfers its phosphate to QseB , which then regulates both is own transcription and the transcription of the flagella and motility genes ( 8 , 20 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16803956" target="_blank">16803956</a>]<br></li>
<li>When both sites are occupied by QseB , auto-regulation occurs , and qseBC transcription is greatly increased .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16194231" target="_blank">16194231</a>]<br></li>
<li>These results indicate that QseB may act to autoregulate its own transcription through binding to low - and high-afﬁnity sites found in its promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16194231" target="_blank">16194231</a>]<br></li>
<li>As many two-component systems also autoregulate their own transcription ( Bijlsma and Groisman , 2003 ) , our study aimed to investigate whether QseB may also act in this fashion .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16194231" target="_blank">16194231</a>]<br></li>
				</div></ul></div><br>
	</div>
</body>
</html>