<!doctype html>
<style>
@charset "utf-8";
body {
	height: 100%;
	margin: 0;
	padding: 0;
}

.banner {
	position: fixed;
	top: 0;
	width: 100%;
	height: 65px;
	background: gray;
	line-height: 65px;
}
.spacer {
	height: 10px;
}

.main-content {
	padding: 5%;
}

.collapse_ACT {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-style: none solid none none;
	border-color: #B4045F;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_ACT + input{
	display: none;
}
.collapse_ACT + input + div{
	display: none;
}
.collapse_ACT + input:checked + div{
	display: block;
}
.ACT {
	text-align: justify;
	text-justify: inter-word;
}
.ACT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B4045F;
}
.ACT_collapse_similarity:hover{
	background: #e5d3dc;
}
.ACT_collapse_similarity + input {
	display: none;
}
.ACT_collapse_similarity + input + div{
	display: none;
}

.ACT_collapse_similarity + input:checked + div{
	display: block;
}
.ACT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.ACT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B4045F;
	padding-left: 5px;
}
.ACT_collapse_similarity:hover{
	background: #e5d3dc;
}
.ACT_collapse_similarity + input {
	display: none;
}
.ACT_collapse_similarity + input + div{
	display: none;
}
.ACT_collapse_similarity + input:checked + div{
	display: block;
}
.ACT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.ACT_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #B4045F;
}
.ACT_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_DOM {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-style: none solid none none;
	border-color: #B404AE;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_DOM + input{
	display: none;
}
.collapse_DOM + input + div{
	display: none;
}
.collapse_DOM + input:checked + div{
	display: block;
}
.DOM {
	text-align: justify;
	text-justify: inter-word;
}
.DOM_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B404AE;
	padding-left: 5px;
}
.DOM_collapse_similarity:hover{
	background: #e8d7e7;
}
.DOM_collapse_similarity + input {
	display: none;
}
.DOM_collapse_similarity + input + div{
	display: none;
}
.DOM_collapse_similarity + input:checked + div{
	display: block;
}
.DOM_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}.DOM_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #B404AE;
}
.DOM_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_EVO {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #7401DF;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_EVO + input{
	display: none;
}
.collapse_EVO + input + div{
	display: none;
}
.collapse_EVO + input:checked + div{
	display: block;
}
.EVO {
	text-align: justify;
	text-justify: inter-word;
}
.EVO_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #7401DF;
	padding-left: 5px;
}
.EVO_collapse_similarity:hover{
	background: #e2d7ed;
}
.EVO_collapse_similarity + input {
	display: none;
}
.EVO_collapse_similarity + input + div{
	display: none;
}
.EVO_collapse_similarity + input:checked + div{
	display: block;
}
.EVO_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.EVO_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #7401DF;
}
.EVO_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_RP {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #0174DF;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_RP + input{
	display: none;
}
.collapse_RP + input + div{
	display: none;
}
.collapse_RP + input:checked + div{
	display: block;
}
.RP {
	text-align: justify;
	text-justify: inter-word;
}
.RP_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #0174DF;
	padding-left: 5px;
}
.RP_collapse_similarity:hover{
	background: #c9d7e2;
}
.RP_collapse_similarity + input {
	display: none;
}
.RP_collapse_similarity + input + div{
	display: none;
}
.RP_collapse_similarity + input:checked + div{
	display: block;
}
.RP_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.RP_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #0174DF;
}
.RP_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_SIT {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #0489B1;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_SIT + input{
	display: none;
}
.collapse_SIT + input + div{
	display: none;
}
.collapse_SIT + input:checked + div{
	display: block;
}
.SIT {
	text-align: justify;
	text-justify: inter-word;
}
.SIT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #0489B1;
	padding-left: 5px;
}
.SIT_collapse_similarity:hover{
	background: #d0dfe2;
}
.SIT_collapse_similarity + input {
	display: none;
}
.SIT_collapse_similarity + input + div{
	display: none;
}
.SIT_collapse_similarity + input:checked + div{
	display: block;
}
.SIT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.SIT_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #0489B1;
}
.SIT_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_TU {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #04B486;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_TU + input{
	display: none;
}
.collapse_TU + input + div{
	display: none;
}
.collapse_TU + input:checked + div{
	display: block;
}
.TU {
	text-align: justify;
	text-justify: inter-word;
}
.TU_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #04B486;
	padding-left: 5px;
}
.TU_collapse_similarity:hover{
	background: #d0e2de;
}
.TU_collapse_similarity + input {
	display: none;
}
.TU_collapse_similarity + input + div{
	display: none;
}
.TU_collapse_similarity + input:checked + div{
	display: block;
}
.TU_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.TU_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #04B486;
}
.TU_similar > a {
	text-decoration: none;
	color: gray;
}

ul { 
	list-style-type: none;
	padding: 0;
	margin: 0;
}
ul li > a {
	text-decoration: none;
	color: gray;
}
ul li:hover, ul li:focus {
	background: #F0F0F0;
}

ul li:first-child span {
  border-left:none;  
}

</style>
<html>
<head>
<meta charset="utf-8">
<title>TF properties - RbsR</title>
<meta name="theme-color" content="#ffffff">
</head>
<body>
	<div class="banner">
		<center><font color="white" size="16px" face="arial">RbsR</font></center>
	</div>
	<div align="center" class="main-content">
		<div class="spacer">
	</div><br>			<label class="collapse_ACT" for="_1"><font size="5px">RbsR transcriptional activity</font></label>
				<input id="_1" type="checkbox">
				<div class="ACT"><br><ul><li>The absence of a branch at the @ - carbon of thLe u may confer the required additional flexibility to allow a shorter turn in RbsR , although the very short two-residue turn is bound to affect the conformations of the amino acids leading into the helices at that site t Materials and methods Materials 5 ' - [ a-35S ] Thio-dATP , ' 4C-methylatpe rodtein molecular weight standards , and ~ - [ ~ ~ S ] methionw ineere obtained from Amersham .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1304369" target="_blank">1304369</a>]<br></li>
<li>Therefore , the observed ompF-1acZ repression and ompC-1acZ activation by pWF2 is likely due to the overexpression of RbsR , although one can not exclude a possibility of the effect of rbsK , which may be transcribed through the vector 's promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8522138" target="_blank">8522138</a>]<br></li>
<li>IS + versus hs normallzed lo the eno control 2.30 2.23 0.31 4.74 1.99 0.25 1.90 1.18 0.41 Transcriptionalregulation in GlpR and RbsR repressor-deficientstrains As glpK and glpQ expression is regulated by GlpR ( Cozzarelli et a / .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8899705" target="_blank">8899705</a>]<br></li>
<li>Expression plasmid pRsbR of His-tagged RbsR protein was constructed essentially according to the standard procedure in this laboratory ( Shimada et al. , 2005 ; Yamamoto et al. , 2005 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23651393" target="_blank">23651393</a>]<br></li>
<li>Level of RbsR binding Level of RbsR binding Results and discussion 2500 Search for the regulation targets of RbsR by ( a ) RbsR - / rbsD 2000 Genomic SELEX screening 1500 To identify the whole set of promoters , genes and ope-rons under the direct control of RbsR , we performed Genomic SELEX screening ( Shimada et al. , 2005 ) , in which puriﬁed His-tagged RbsR was mixed with a collection of E. coli genome fragments of 200 -- 300 bp in length and RbsR-bound DNA fragments were afﬁnity-isolated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23651393" target="_blank">23651393</a>]<br></li>
<li>Expression and puriﬁcation of RbsR protein Escherichia coli BL21 ( pRbsR ) transformant was grown in LB broth in the presence of 50 lg mL 1 ampicillin , and expression of RbsR was induced in mid-log phase by add-ing 1 mM IPTG .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23651393" target="_blank">23651393</a>]<br></li>
<li>Theresults , together with the controlexperiments run simultaneously , indicate that RbsR does bind rbs0 , and of the sugars tested , only ribosweas effective indisruptingthat complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1304369" target="_blank">1304369</a>]<br></li>
<li>No ancillaryfactors were needed to form theRbsR-rbsO complex when pure RbsR was tested .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1304369" target="_blank">1304369</a>]<br></li>
<li>, 1984 ) was hybridized to a probe for rbsS no significant difference in RNA levels of fis ' and fis strains was observed ( Fig. 5B ) , suggesting that FIS positively regulates the repressor , RbsR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8899705" target="_blank">8899705</a>]<br></li>
<li>Repressor of cytR Repressor of deoR UdP Conversion of uridine to uracil plus CytR ribose-1-phosphate Conversion of deoxyribose-5-phosphate CytR , DeoR to glyceraldehyde-3-phosphate deoCABD mflADR rbsACB CVtR deoR Transport and assimilation of o-mannitol MtlR Transport and assimilation of o-ribose RbsR Transcriptional repressor Transcriptional repressor Q 1996 Blackwell Science Ltd , Molecular Microbiobgy ,22 , 21-2 Fig. 6 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8899705" target="_blank">8899705</a>]<br></li>
<li>Thus , it is most likely that the RbsR and PurRproteins have opposite binding characteristics to their respective operators with highest affinity being the open form for RbsR and the closed form for PurR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1304370" target="_blank">1304370</a>]<br></li>
<li>Since suppression leads to upregulation of rbsDAC , it seems likely that the extension of RbsR decreases its ability to repress .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15466022" target="_blank">15466022</a>]<br></li>
<li>Of special interest , however , is theobservationthatthe C-terminal270 residues of RbsR are homologousto the ribosebindingprotein.Homology between short sequence fragments of arabinose ( ABP ) , galactose ( GBP ) , and ribose ( RBP ) binding proteins and thelac and gal repressors had been noted in 1983 ( Muller-Hill , 1983 ) , and hypothesesrelating to the sugarbindingsiteinLacI ( Sams et al. , 1984 ) and various functional sites on LacI ( Gordon et al. , 1988 ) were proposed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1304370" target="_blank">1304370</a>]<br></li>
<li>This possibility was examined by Northern blot analysis of udk mRNA in the presence and absence of RbsR ( see below ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23651393" target="_blank">23651393</a>]<br></li>
<li>To increase the expression of RbsR , the rbsR genewasplacedintoboth pBR32 v ) U z a DP + C 0 V a N N z n 0 Z m 0 Qa ( u 2 ( u LL z , rr ) LL E Q - 200 Kdal - loo 92.5 - 69 -46 Fig. 2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1304369" target="_blank">1304369</a>]<br></li>
<li>To examine the expression and binding of RbsR in single copy , extract was prepared from MRil21 ( characterized as rbsR + ) grown in M9 medium containing either glucose , glycerol , D-ribose + glucose , or D-ribose + glycerol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1304369" target="_blank">1304369</a>]<br></li>
<li>To determine if the lack of binding was due to anuncharacterized rbsR strain or to low RbsR expression , extracts were prepared from the strain MRil2 ( 1dem-onstrated to be rbsR + [ Lopilato et al. , 19841 ) grown in sugars to control the inductionby ribose or to control the catabolite repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1304369" target="_blank">1304369</a>]<br></li>
<li>The labeled fragment was bound under the conditions determined by the mobility shift assay using an extract containing partially purified RbsR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1304369" target="_blank">1304369</a>]<br></li>
<li>In practice , considerable similarity between the regulatory protein and the regulated protein is only seen in the regulatory protein RbsR and a periplasmic component RbsB of HAT , which is encoded by the operon rbsDABCK under the repression by RbsR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8729576" target="_blank">8729576</a>]<br></li>
<li>MICROBIOLOGY LETTERS R E S E A R C H L E T T E R Involvement of the ribose operon repressor RbsR in regulation of purine nucleotide synthesis in Escherichia coli Tomohiro Shimada1 ,2,3 , Ayako Kori1 & Akira Ishihama1 ,2 1 2 Department of Frontier Bioscience , Hosei University , Koganei , Tokyo ; Research Center for Micro-Nano Technology , Hosei University , Koganei , Tokyo ; and Chemical Resources Laboratory , Tokyo Institute of Technology , Nagatsuda , Yokohama , Japan 3 Correspondence : Akira Ishihama , Hosei University , Department of Frontier Bioscience , Abstract Escherichia coli is able to utilize D-ribose as its sole carbon source .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23651393" target="_blank">23651393</a>]<br></li>
<li>[ 6 ] Mauzy , C.A. and Hermodson , M.A. ( 1992 ) Structural and functional analyses of the repressor , RbsR , of the ribose operon of Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11750821" target="_blank">11750821</a>]<br></li>
<li>Copyright 0 1992 The Protein Society - ~ '' '' etween rbs repressor and Structural h omology b ribose binding protein implies functional similarity ~ C.A. MAUZY AND M.A. HERMODSON Department of Biochemistry , Purdue University , West Lafayette , Indiana 47907 ( RECEIVED November 18 , 1991 ; REVISED MANUSCRIPRTECEIVEDFebruary 12 , 1992 ) Abstract The deduced amino acidsequenceof the rbs repressor , RbsR , of Escherichia coli is homologous over its C-terminal 272 residues to the entire sequenceof the periplasmic ribose binding protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1304370" target="_blank">1304370</a>]<br></li>
<li>The mechanism by which RbsR protein activates the expression of ompC-1acZ in high osmolarity can not be explained at present .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8522138" target="_blank">8522138</a>]<br></li>
<li>The hydrophobic coreresidues in RbsR would be `` 4 , V-7 , A-8 , V-17 , V-20 , 1-21 , A-38 , 1-39 , L-42 , and Y-44 by homology to LacI ( Fig. 1 ; Kaptein et al. , 1985 ; Gordon et al. , 1988 ; Mauzy & Hermodson , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1304369" target="_blank">1304369</a>]<br></li>
<li>The level of purH mRNA also increased in the rbsR mutant ( Fig. 4 , purH lane ) , implying a repressive role of RbsR in transcription of the purHG operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23651393" target="_blank">23651393</a>]<br></li>
<li>The rbsDACBK operon is the only target of regulation so far identiﬁed to be under the control of RbsR ( Lopilato et al. , 1984 ; Mauzy & Hermodson , 1992a ; Laikova et al. , 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23651393" target="_blank">23651393</a>]<br></li>
<li>Search for the regulation targets of RbsR in the The rbsDACBK operon is induced by D-ribose that is sensed by RsbR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23651393" target="_blank">23651393</a>]<br></li>
<li>Other sugarssuch as ribose-5-phosphate , ribose-1-phosphate ( Fig. 7 ) , galactose , arabinose , L-ribose , and allose ( not shown ) do notaffect RbsR bindingto the proposed operator sequence found in pCMO1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1304369" target="_blank">1304369</a>]<br></li>
<li>RbsR does not bind to either the pCM03 fragment or the pCM04 fragment in the absence or presence of D-ribose ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1304369" target="_blank">1304369</a>]<br></li>
<li>However , it was shown that RbsR does bind to thepCMO1 fragment ( Fig. 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1304369" target="_blank">1304369</a>]<br></li>
<li>The binding affinity of RbsR for the pCMOl fragment decreases upon the addition of D-ribose ( Fig. 7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1304369" target="_blank">1304369</a>]<br></li>
<li>Competition between unlabeled and labeled pCMOl fragment was demonstrated , indicative of the specific nature of the binding of RbsR to the pCMOl fragment .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1304369" target="_blank">1304369</a>]<br></li>
<li>Thus , we concluded that RbsR is involved in repression of the entire pathway of de novo synthesis of purine nucleotides starting from the uptake of D-ribose .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23651393" target="_blank">23651393</a>]<br></li>
<li>The ribose operon repressor RbsR represses not only the uptake and initial step metabolism of D-ribose to generate D-ribose 5 ′ - phosphate but also the de novo synthesis of purine nucleotides from D-ribose 5 ′ - phosphate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23651393" target="_blank">23651393</a>]<br></li>
<li>Because ribose induces the rbs operon , RbsR with ribose ( or a ribose metabolite ) bound , i.e. , the closed conformer in thebindingproteinportion , mustbe the conformation that has the lower affinity for the operator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1304370" target="_blank">1304370</a>]<br></li>
<li>In the absence of inducer ribose , RbsR alone bound at least six regions ( Fig. 1a ) , three within intergenic spacers and three within open reading frames ( Table 1A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23651393" target="_blank">23651393</a>]<br></li>
<li>Members of these families have the opposite modes of action : RbsR is a classical repressor whose true inducer is D-ribose , whereas AraC and XylR are primarily activators of transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11750821" target="_blank">11750821</a>]<br></li>
<li>Closed circles represent the level of RbsR binding in the absence of D-ribose , whereas closed circles represent the level of RbsR binding in the presence of inducer D-ribose .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23651393" target="_blank">23651393</a>]<br></li>
<li>In the presence of inducer D-ribose , however , RbsR activates the salvage pathway of purine nucleotide synthesis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23651393" target="_blank">23651393</a>]<br></li>
<li>In the absence of inducer D-ribose , the ribose operon is repressed by a LacI-type transcription factor RbsR , which is encoded by a gene located downstream of this ribose operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23651393" target="_blank">23651393</a>]<br></li>
<li>nucleotide ; RbsR , ribose ; transcription factor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23651393" target="_blank">23651393</a>]<br></li>
<li>Finally , candidate cAMP receptor-protein sites were identified in the regulatory regions of the majority of RbsR - , AraC - , and XylR-regulated operons .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11750821" target="_blank">11750821</a>]<br></li>
<li>Keywords : Computer analysis ; Transcriptional regulation ; RbsR ; AraC ; XylR ; cAMP receptor protein 1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11750821" target="_blank">11750821</a>]<br></li>
<li>Candidate binding sites for RbsR and AraC were detected .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11750821" target="_blank">11750821</a>]<br></li>
<li>ons of the RbsR , AraC and XylR regulons .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11750821" target="_blank">11750821</a>]<br></li>
<li>Only a few of the transporters and regulators in this group bind the same substrates ; galactose ( MglB and GalR ) , ribose ( RbsR and RbsB ) and xylose ( XylF and XylR ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15757509" target="_blank">15757509</a>]<br></li>
<li>R , RbsR ; G , GalR ; L , Lad ; M , MalI ; C , CytR ; P , PurR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1304370" target="_blank">1304370</a>]<br></li>
<li>1.3 1.9 5.0 2.4 4.7 2.7 2.6 1.4 PspF , IHF PhoB RbsR , Crp 37.9 22.0 2.6 1.2 1.8 2.3 0.8 1.1 Inner membrane Unknown Ribose transport Ribose transport Ribose transport Ribose transport Ribose transport Ribose transport pspG ( yjbO ) ytfK rbsD rbsA rbsC rbsB rbsK rbsR 0.7 3.8 1.8 1.0 1.6 1.4 1.2 0.8 0.9 3.0 8.2 5.4 21.3 6.6 9.0 2.8 manX Mannose permease 4.2 2.1 1.1 5.0 manY Mannose permease 3.4 2.6 1.1 5.2 manZ Mannose permease 6.5 5.7 1.0 16.7 NagC , DgsA , Crp a. Genes and operons which are induced 10-fold in three independent experiments are listed with the average fold induction for the three experiments .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18710431" target="_blank">18710431</a>]<br></li>
<li>23.5 20.0 3.8 16.3 9.1 12.7 Maltose transport MalT , Crp malK lamB malM ibpB ibpA htpG Maltose transport MalT , Crp 18.6 5.3 3.3 39.0 24.4 11.7 32.7 13.3 9.1 24.2 20.2 8.2 2.6 2.1 1.7 4.6 4.1 2.0 48.6 19.2 20.0 52.6 35.9 16.6 s32 Heat shock Heat shock s32 Heat shock chaperone s32 Protease ; chaperone Regulator Regulator ; chaperone Protease Periplasmic clpB 11.5 12.9 17.3 21.0 20.4 14.3 3.3 3.2 1.3 1.8 2.5 2.9 8.0 2.7 2.9 24.5 6.8 5.7 1.7 1.3 yceP/bssS cpxP ( b3913 ) degP ( htrA ) spy CpxR CpxR , H-NS CpxR , BaeSR CpxR CpxR Unknown Unknown ydeH yebE 49.2 44.8 63.1 21.0 30.7 32.2 56.9 33.1 9.5 1.1 1.6 6.1 7.3 7.3 2.6 1.5 1.6 2.0 1.4 1.2 1.3 2.9 Unknown ; signal peptide Putative transporter Regulator ; phage shock protein Regulator ; phage shock protein Regulator ; phage shock protein Inner membrane Sulphurtransferase yncJ ( b1436 ) yhjX pspA pspB pspC pspD pspE 1.6 44.1 1.1 1.6 1.7 1.2 1.2 3.1 165.6 1.8 1.5 2.1 1.2 1.5 PspF , IHF 1.3 1.9 5.0 2.4 4.7 2.7 2.6 1.4 PspF , IHF PhoB RbsR , Crp 37.9 22.0 2.6 1.2 1.8 2.3 0.8 1.1 Inner membrane Unknown Ribose transport Ribose transport Ribose transport Ribose transport Ribose transport Ribose transport pspG ( yjbO ) ytfK rbsD rbsA rbsC rbsB rbsK rbsR 0.7 3.8 1.8 1.0 1.6 1.4 1.2 0.8 0.9 3.0 8.2 5.4 21.3 6.6 9.0 2.8 manX Mannose permease 4.2 2.1 1.1 5.0 manY Mannose permease 3.4 2.6 1.1 5.2 manZ Mannose permease 6.5 5.7 1.0 16.7 NagC , DgsA , Crp a. Genes and operons which are induced 10-fold in three independent experiments are listed with the average fold induction for the three experiments .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18710431" target="_blank">18710431</a>]<br></li>
<li>Crp , GalS MglAB Crp , RbsR RbsAB Crp , GalS MglA Crp , GntR GntK Crp , EbgR EbgA Crp , LacI LacZ Crp , RhaS RhaBD Crp , AraC AraBD Crp , FucR FucK Crp , YiaJ LyxK Crp , FucR FucA Crp , YiaJ SgbE RcsB WcaB CpxR LpxA RcsB WcaB CpxR LpxD RcsB B2060 PhoB PhnCKLN RcsB B2060 PhoB PstB RcsB B2060 PhoB PhoH MhpR MhpF Fur EntA FecI , Fur FecI UidR , UxuR UidR Crp , FucR FucR Crp , Fur Fur Homologous pairs of transcription factors with autoregulation IdnR PurR AsnC Lrp Fnr Crp Homologous triplets of transcription factors with autoregulation TorR PhoB CpxR EmrR PdhR ExuR Eight homologous transcription factors with autoregulation DsdC GcvA CynR Hcar IlvY LysR OxyR CysB Figure 6 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12582243" target="_blank">12582243</a>]<br></li>
<li>aerobic to anaerobic anaerobic to aerobic ArgR CysB BirA DgsA MalT GadW MetJ NikR OxyR SoxR SoxS DcuR CueR FadR HcaR NtrC PhoB PurR SgrR TdcA TrpR ZraR ArcA CpxR CRP CusR FhlA Fis FNR Fur H-NS IHF IscR Lrp ModE NarL NrdR PdhR FlhDC GlcC EvgA GadX LeuO PspF RbsR steady states Figure 3 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22870390" target="_blank">22870390</a>]<br></li>
				</div></ul><br>
			<label class="collapse_DOM" for="_2"><font size="5px">RbsR protein domains</font></label>
				<input id="_2" type="checkbox">
				<div class="DOM"><br><ul><li>In the absence of inducer ribose , RbsR alone bound at least six regions ( Fig. 1a ) , three within intergenic spacers and three within open reading frames ( Table 1A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23651393" target="_blank">23651393</a>]<br></li>
<li>83 Determination of the translational start codon of rbsR To determine the start codon of the rbsR gene , a rbsRlacZ fusion was constructed to produce a fusion protein containing the N-terminal 266 amino acids from RbsR fused to 1,016 residues ofP-galactosidase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1304369" target="_blank">1304369</a>]<br></li>
<li>There is one region of uncertainty regarding placement of gaps between residues 135 and 155 of RbsR where Figures 1 and 2 differ slightly .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1304370" target="_blank">1304370</a>]<br></li>
<li>Indeed , in the caseof RbsR , theabsence of one residue in this position indicates that this region does not require athree-residue turn .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1304369" target="_blank">1304369</a>]<br></li>
<li>For the RBP/RbsR comparison , 63 of the 268 residues compared ( 23 % ) align with identical residues .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1304370" target="_blank">1304370</a>]<br></li>
<li>Copyright 0 1992 The Protein Society - ~ '' '' etween rbs repressor and Structural h omology b ribose binding protein implies functional similarity ~ C.A. MAUZY AND M.A. HERMODSON Department of Biochemistry , Purdue University , West Lafayette , Indiana 47907 ( RECEIVED November 18 , 1991 ; REVISED MANUSCRIPRTECEIVEDFebruary 12 , 1992 ) Abstract The deduced amino acidsequenceof the rbs repressor , RbsR , of Escherichia coli is homologous over its C-terminal 272 residues to the entire sequenceof the periplasmic ribose binding protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1304370" target="_blank">1304370</a>]<br></li>
<li>By homology to LacI ( Mauzy & Hermodson , 1992 ) the first 60 residues of RbsR form theDNA-binding domain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1304369" target="_blank">1304369</a>]<br></li>
<li>The proposed turnregion between helix-1 and helix3 in RbsR is truncated by one residue as compared to the other members of this repressor family ( Fig. 1 ; Mauzy & Hermodson , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1304369" target="_blank">1304369</a>]<br></li>
<li>The N-terminal 57 resi-dues of RbsR and 58 residues of GalR , homologous to the Lac1 DNA-binding domain , are not shown.The numbers above the alignments refer to the ABP sequence and those below to the RbsR sequence with the last numeral above or below the indicated residue .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1304370" target="_blank">1304370</a>]<br></li>
<li>The peak regions with the binding activity of puriﬁed RbsR protein were expanded .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23651393" target="_blank">23651393</a>]<br></li>
				</div></ul><br>
			<label class="collapse_EVO" for="_3"><font size="5px">RbsR evolution</font></label>
				<input id="_3" type="checkbox">
				<div class="EVO"><br><ul><li>For the RBP/RbsR comparison , 63 of the 268 residues compared ( 23 % ) align with identical residues .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1304370" target="_blank">1304370</a>]<br></li>
<li>It is interesting to note that PurR andRbsR are themost similar proteins in this family at the sequence level ( 47 % identical ) , something one may not expect for proteins with significantly different mechanisms .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1304370" target="_blank">1304370</a>]<br></li>
<li>s 84 The high degree of sequence identity within the family of repressors to which RbsR belongs ( Fig. 1 ) leavesno doubt that these six proteins share a common ancestorB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1304370" target="_blank">1304370</a>]<br></li>
<li>We have now completed the sequence analysis of this open reading frame and have shown that it encodes RbsR , a member of a family of repressorproteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1304369" target="_blank">1304369</a>]<br></li>
<li>RbsR is a member of a repressor family including GalR , Lacl , Mali , CytR , and PurR [ 36 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8522138" target="_blank">8522138</a>]<br></li>
<li>PurR is a member of the lac repressor family , which includes LacI ( 4 , 9 ) , GalR ( 46 ) , RbsR ( 25 ) , MalI ( 36 ) , RafR ( 3 ) , RbtR ( 50 ) , CytR ( 44 ) , EbgR ( 43 ) , CcpA ( 13 ) , and FruR ( 16 , 45 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1400170" target="_blank">1400170</a>]<br></li>
<li>The proposed turnregion between helix-1 and helix3 in RbsR is truncated by one residue as compared to the other members of this repressor family ( Fig. 1 ; Mauzy & Hermodson , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1304369" target="_blank">1304369</a>]<br></li>
<li>We show here that RbsR ( Mauzy & Hermodson , 1992 ) is a member of a family of homologous repressor proteins , including Lad .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1304370" target="_blank">1304370</a>]<br></li>
<li>RbsR is a member of a family of homologous repressor proteins having very similar DNA-binding sites and binding to similar operator sequences .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1304369" target="_blank">1304369</a>]<br></li>
<li>The C-terminal 270 resi-dues of RbsR are homologous to the ribose-binding protein RbsB ( Mauzy & Hermodson , 1992b ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23651393" target="_blank">23651393</a>]<br></li>
<li>Members of these families have the opposite modes of action : RbsR is a classical repressor whose true inducer is D-ribose , whereas AraC and XylR are primarily activators of transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11750821" target="_blank">11750821</a>]<br></li>
<li>The systems for utilization of these sugars are regulated on the transcriptional level by protein factors which belong to di ¡ erent structural families : the ribose regulator ( RbsR ) belongs to the LacI family , whereas the arabinose ( AraC ) and xylose ( XylR ) regulators are members of the AraC family .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11750821" target="_blank">11750821</a>]<br></li>
<li>The system for transport and utilization of these sugars are regulated on the transcriptional level by transcription factors which belong to different structural families : the ribose regulator ( RbsR ) belongs to the LacI family ( Mauzy & Hermodson , 1992a ) , whereas the arabinose ( AraC ) ( Stoner & Schleif , 1983 ; Miyada et al. , 1984 ) and xylose ( XylR ) ( Song & Park , bers of these families have the opposite modes of action : RbsR is a classical repressor whose inducer is D-ribose , whereas AraC and XylR are activators of transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23651393" target="_blank">23651393</a>]<br></li>
				</div></ul><br>
			<label class="collapse_RP" for="_4"><font size="5px">Processes regulated by RbsR</font></label>
				<input id="_4" type="checkbox">
				<div class="RP"><br><ul><li>presence of ribose Transcription regulation in vivo of the set of RbsR target genes involved in nucleotide metabolism 150 -- Ribose + Ribose ( a ) ( b ) 100 400 200 50 0 0 262000 263000 1699000 1700000 260000 261000 1701000 1702000 add ydgJ Level of RbsA-binding proB proA thrW malY 100 500 80 ( c ) 400 ( d ) 60 40 300 200 20 0 2142000 2143000 udk 100 0 2144000 yegE 3168000 3169000 3170000 qseB qseC 2145000 3167000 2500 2000 ( 120 100 e ) ( f ) 80 60 0 4204000 rbsA 0 3930000 3931000 3932000 4205000 4206000 4207000 purH rrsE rbsD kup Fig. 2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23651393" target="_blank">23651393</a>]<br></li>
<li>[ RbsR PIR accession number A41828 .1 Keywords : Escherichia coli ; repressors ; ribose transport and utilization ; transcription regulation The rbs operon of Escherichia coliK12 encodes the highaffinity membrane transport system for ribose ( Iida et al. , 1984 ; Lopilato et al. , 1984 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1304369" target="_blank">1304369</a>]<br></li>
<li>Northern blot analysis of mRNA for RbsR target genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23651393" target="_blank">23651393</a>]<br></li>
<li>MICROBIOLOGY LETTERS R E S E A R C H L E T T E R Involvement of the ribose operon repressor RbsR in regulation of purine-nucleotide-synthesis in Escherichia coli Tomohiro Shimada1 ,2,3 , Ayako Kori1 & Akira Ishihama1 ,2 1 2 Department of Frontier Bioscience , Hosei University , Koganei , Tokyo ; Research Center for Micro-Nano Technology , Hosei University , Koganei , Tokyo ; and Chemical Resources Laboratory , Tokyo Institute of Technology , Nagatsuda , Yokohama , Japan 3 Correspondence : Akira Ishihama , Hosei University , Department of Frontier Bioscience , Abstract Escherichia coli is able to utilize D-ribose as its sole carbon source .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23651393" target="_blank">23651393</a>]<br></li>
<li>On the other hand , the level of both add and udk mRNAs decreased in the rbsR mutant ( Fig. 4 , add and udk lanes ) , implying a positive role of RbsR in transcription of the add and udk-dcd operons , both participating in the salvage pathway of nucleotide-synthesis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23651393" target="_blank">23651393</a>]<br></li>
<li>Since RbsR inﬂuences transcription of the udk and dcd operons , RsbR may also be involved in the regulation of salvage pathway of pyrimidine nucleotides .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23651393" target="_blank">23651393</a>]<br></li>
<li>After Genomic SELEX screening , however , we have identiﬁed that RbsR binds not only to the rbs promoter but also to the promoters of a set of genes involved in purine-nucleotide-metabolism .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23651393" target="_blank">23651393</a>]<br></li>
<li>After Genomic SELEX screening , we identiﬁed novel regulation targets of RbsR , including the genes involved in the salvage pathways of purine-nucleotide-synthesis ( shown against a gray background ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23651393" target="_blank">23651393</a>]<br></li>
<li>Taken together , we propose that RbsR is a global regulator for switch control between the de novo synthesis of purine nucleotides and its salvage pathway .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23651393" target="_blank">23651393</a>]<br></li>
<li>Thus , we concluded that RbsR is involved in repression of the entire pathway of de novo synthesis of purine nucleotides starting from the uptake of D-ribose .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23651393" target="_blank">23651393</a>]<br></li>
<li>The ribose operon repressor RbsR represses not only the uptake and initial step metabolism of D-ribose to generate D-ribose 5 ′ - phosphate but also the de novo synthesis of purine nucleotides from D-ribose 5 ′ - phosphate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23651393" target="_blank">23651393</a>]<br></li>
<li>DOI : 10.1111 / 1574-6968 .12172 Editor : Olga Ozoline Northern blotting analysis indicated that RbsR represses the purHD operon for de novo synthesis of purine nucleotide but activates the add and udk genes Keywords Escherichia coli ; genomic SELEX ; purine involved in the salvage pathway of purine-nucleotide-synthesis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23651393" target="_blank">23651393</a>]<br></li>
<li>Moreover , RbsR is an activator for the salvage pathway genes of nucleotide-synthesis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23651393" target="_blank">23651393</a>]<br></li>
<li>In the presence of inducer D-ribose , however , RbsR activates the salvage pathway of purine-nucleotide-synthesis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23651393" target="_blank">23651393</a>]<br></li>
<li>The results herein described indicate that RbsR is involved in regulation of not only the rbs operon for transport and utilization of D-ribose but also a set of genes for both de novo and salvage pathways of pur-ine nucleotide-synthesis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23651393" target="_blank">23651393</a>]<br></li>
<li>On the other hand , PurH ( AICAR transformylase/IMP cyclohydrolase ) and PurD ( phosphoribosylamine-glycine ligase ) are involved in the de novo synthesis of purine , indicating the involvement of RbsR in regulation of the genes for purine-nucleotide-synthesis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23651393" target="_blank">23651393</a>]<br></li>
<li>Operons which have been experimentally verified as being regulated by CRP Gene function Reference e ROMA Operon nam Other CRP sites regulators b Score KE/wt Position Sites a WT HL HL/wt KE 1.96 0.85 # bglG 1.73 -- 61.5 AACTGCGAGCATGGTCATATTT 11.2 Fis Positive regulation of bgl ( 49 ) operon Uptake of C4-dicarboxylic ( 50 ) acids D-ribose high-affinity ( 51 ) transport system Positive regulator for rhabad ( 52 ) operon Succinate dehydrogenase , ( 53 ) cytochrome b556 Tryptophanase leader peptide ( 54 ) PTS-system enzyme II , ( 55 ) trehalose specific Probable formate-transporter ( 43 ) Mgl repressor , galactose ( 56 ) operon inducer Sn-glycerol-3-phosphate ( 57 ) permease PTS-system , maltose and ( 58 ) glucose-specific II ABC Regulator of melibiose operon ( 59 ) Galactose-binding transport ( 56 ) protein PTS-system , glucose-specific ( 35 ) IIBC component PTS-system protein hpr ( 60 ) Orf , hypothetical protein ( 61 ) Aspartate ammonia-lyase ( 62 ) ( aspartase ) 2.44 1.09 # dctA 1.85 -- 81.5 TTGTGCGAGCCAGCTCAAACTT 14.1 DcuR , ArcA 3.80 0.89 # rbsDACBK R 4.10 -- 61.5 CGTTTCGAGGTTGATCACATTT 9.4 RbsR 1.46 0.86 # rhaSR 1.66 -- 113.5 TGATGTGATGCTCACCGCATTT 12.4 RhaR 2.71 1.11 # sdhCDAB 2.38 -- 83.5 TATCGTGACCTGGATCACTGTT 16.4 ArcA , FNR 1.98 0.95 # 3.09 1.35 # tnaLAB treBC 2.08 -- 4.73 -- 61.5 60.5 GATTGTGATTCGATTCACATTT 19.6 AATTGTGATCTTCGCTGCGTTT 8.8 1.39 # 1.09 # 3.90 1.59 # 2.49 1.22 # galS 41.5 41.5 AGATATGATCTATATCAATTTC 10.1 FNR TGCTGTGACTCGATTCACGAAG 10.4 GalR , GalS 2.21 # 4.51 0.89 # glpTQ ATGTGTGCGGCAATTCACATTT 17.2 GlpR 41.5 2.94 # 6.24 1.71 # malXY TTATGTGACAGATAAAACGTTT 11.2 49.5 1.17 # 1.78 # 2.29 1.20 # 6.78 1.49 # melR AACCGTGCTCCCACTCGCAGTC 5.5 MelR ATCTGTGAGTGATTTCACAGTA 16.6 41.5 41.5 mglBAC 1.13 # 2.11 1.14 # ptsG AAACGTGATAGCCGTCAAACAA 14.3 Mlc 40.5 1.56 # 1.10 # 2.65 # 2.95 1.01 # 1.88 0.97 # 6.29 2.50 # yhfA aspA TTTTATGATTTGGTTCAATTCT 9.5 TAATGTGACGTCCTTTGCATAC 11.7 42.5 44.5 90.5 AGCGGTGATCTATTTCACAAAT 15.5 FNR TAAAGTGATCCAGATTACGGTA 14.2 40.5 94.5 deoCABD TTATTTGAACCAGATCGCATTA 17.1 CytR , DeoR 2.37 -- 3.22 2.37 -- 2-Deoxyribose-5-phosphate ( 21 ) aldolase AATTGTGATGTGTATCGAAGTG 11.9 41.5 90.5 4.38 1.72 # AAATGTGAATTGCCGCACACAT 17.2 FNR , ArcA , ( 57 ) glpACB 3.31 -- Sn-glycerol-3-phosphate dehydrogenase FlhD , GlpR AATGACGCATGAAATCACGTTT 3.3 40.5 92.5 2.17 # 4.26 2.59 # manXYZ PTS enzyme IIAB , mannase-specific GAATGTGACAAGGATATTTTAC 2.4 Mlc , NagC ( 63,36 ) ATTACGGATCTTCATCACATAA 7.6 AAATGTATGACAGATCACTATT 12.6 CytR 40.5 89.5 1.39 # 3.25 1.17 # nupC Permease of transport system ( 64 ) for 3 nucleosides TAGTGTGTGTCAGATCTCGTTT 12.6 40.5 92.5 4.45 1.01 # nupG Transport of nucleosides , permease protein AAATGTTATCCACATCACAATT 20.4 CytR , DeoR 3.25 -- ( 65 ) TTATTTGCCACAGGTAACAAAA 10.6 ATTTCATCTCTATGTCACATTT 7.4 CytR 42.5 93.5 41.5 40.5 1.15 # rpoH Sigma ( 32 ) factor ( 22 ) 2.18 1.75 -- ACTTGTGGATAAAATCACGGTC 2.2 AACTGTGAAACGAAACATATTT 12.9 CytR 2.40 # 3.96 1.19 # tsx Receptor of phage T6 and ( 66 ) colicin K 74.5 AAACGTGAACGCAATCGATTAC 5.8 a The centre of an experimentally verified CRP-binding site relative to corresponding transcription start site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15520470" target="_blank">15520470</a>]<br></li>
<li>23.5 20.0 3.8 16.3 9.1 12.7 Maltose transport MalT , Crp malK lamB malM ibpB ibpA htpG Maltose transport MalT , Crp 18.6 5.3 3.3 39.0 24.4 11.7 32.7 13.3 9.1 24.2 20.2 8.2 2.6 2.1 1.7 4.6 4.1 2.0 48.6 19.2 20.0 52.6 35.9 16.6 s32 Heat shock Heat shock s32 Heat shock chaperone s32 Protease ; chaperone Regulator Regulator ; chaperone Protease Periplasmic clpB 11.5 12.9 17.3 21.0 20.4 14.3 3.3 3.2 1.3 1.8 2.5 2.9 8.0 2.7 2.9 24.5 6.8 5.7 1.7 1.3 yceP/bssS cpxP ( b3913 ) degP ( htrA ) spy CpxR CpxR , H-NS CpxR , BaeSR CpxR CpxR Unknown Unknown ydeH yebE 49.2 44.8 63.1 21.0 30.7 32.2 56.9 33.1 9.5 1.1 1.6 6.1 7.3 7.3 2.6 1.5 1.6 2.0 1.4 1.2 1.3 2.9 Unknown ; signal peptide Putative transporter Regulator ; phage-shock protein Regulator ; phage-shock protein Regulator ; phage-shock protein Inner membrane Sulphurtransferase yncJ ( b1436 ) yhjX pspA pspB pspC pspD pspE 1.6 44.1 1.1 1.6 1.7 1.2 1.2 3.1 165.6 1.8 1.5 2.1 1.2 1.5 PspF , IHF 1.3 1.9 5.0 2.4 4.7 2.7 2.6 1.4 PspF , IHF PhoB RbsR , Crp 37.9 22.0 2.6 1.2 1.8 2.3 0.8 1.1 Inner membrane Unknown Ribose transport Ribose transport Ribose transport Ribose transport Ribose transport Ribose transport pspG ( yjbO ) ytfK rbsD rbsA rbsC rbsB rbsK rbsR 0.7 3.8 1.8 1.0 1.6 1.4 1.2 0.8 0.9 3.0 8.2 5.4 21.3 6.6 9.0 2.8 manX Mannose permease 4.2 2.1 1.1 5.0 manY Mannose permease 3.4 2.6 1.1 5.2 manZ Mannose permease 6.5 5.7 1.0 16.7 NagC , DgsA , Crp a. Genes and operons which are induced 10-fold in three independent experiments are listed with the average fold induction for the three experiments .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18710431" target="_blank">18710431</a>]<br></li>
				</div></ul><br>
			<label class="collapse_SIT" for="_5"><font size="5px">RbsR binding site</font></label>
				<input id="_5" type="checkbox">
				<div class="SIT"><br><ul><li>Isolation and separation of the DNA products from both theRbsR bound and unbound fragments reveal that thesecond guanine from the5 ' end of the operator sequence is missing from the sequence ( Fig. 4 ) in the RbsR-bound form on both strands ( Fig. 9A , B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1304369" target="_blank">1304369</a>]<br></li>
<li>DNA-bind-ing assays indicate that RbsR binds to a region of perfect dyad symmetry spanning the rbs operon transcriptional start site and that theaffinity for the rbs operator is reduced by addition of ribose , consistent with ribose being the inducer of the operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1304369" target="_blank">1304369</a>]<br></li>
<li>In E. coli , RbsR was experimentally shown to bind to a palindromic sequence upstream of the rbs operon [ 6 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11750821" target="_blank">11750821</a>]<br></li>
<li>RbsR binds to the operator of a palindromic sequence in the rbs operon promoter ( Mauzy & Hermodson , 1992a ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23651393" target="_blank">23651393</a>]<br></li>
<li>Candidate binding sites for RbsR and AraC were detected .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11750821" target="_blank">11750821</a>]<br></li>
<li>RbsR is a member of a family of homologous repressor proteins having very similar DNA-binding sites and binding to similar operator sequences .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1304369" target="_blank">1304369</a>]<br></li>
<li>The presence of a highscore RbsR candidate site upstream of rbsD ( Table 1 ) can be explained by the conjecture that the rbsR regulator gene has been deleted quite recently , so that the sequence drift has not yet destroyed the site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11750821" target="_blank">11750821</a>]<br></li>
<li>The RbsR box was predicted to be composed of the palindromic octanucleotide sequence including the rbsR mutant .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23651393" target="_blank">23651393</a>]<br></li>
<li>Although this sequence contains only one RbsR binding site without a dyad symmetry , the RbsR protein may repress the transcription of ompF by binding the ompF sequence described in Fig. 4 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8522138" target="_blank">8522138</a>]<br></li>
<li>The RbsR binding site was determined by footprinting of the rhs operator sequence [ 38 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8522138" target="_blank">8522138</a>]<br></li>
<li>The identical sequences between ompF and RbsR binding site are underlined .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8522138" target="_blank">8522138</a>]<br></li>
<li>The ompF sequence alignment with the sequence of RbsR binding site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8522138" target="_blank">8522138</a>]<br></li>
<li>The DNA sequence between - 17 to -36 of ompF regulatory region alignment with RbsR binding sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8522138" target="_blank">8522138</a>]<br></li>
				</ul></div><br>
			<label class="collapse_TU" for="_6"><font size="5px">Transcription units regulated by RbsR</font></label>
				<input id="_6" type="checkbox">
				<div class="TU"><br><ul><li>1.3 1.9 5.0 2.4 4.7 2.7 2.6 1.4 PspF , IHF PhoB RbsR , Crp 37.9 22.0 2.6 1.2 1.8 2.3 0.8 1.1 Inner membrane Unknown Ribose transport Ribose transport Ribose transport Ribose transport Ribose transport Ribose transport pspG ( yjbO ) ytfK rbsD rbsA rbsC rbsB rbsK rbsR 0.7 3.8 1.8 1.0 1.6 1.4 1.2 0.8 0.9 3.0 8.2 5.4 21.3 6.6 9.0 2.8 manX Mannose permease 4.2 2.1 1.1 5.0 manY Mannose permease 3.4 2.6 1.1 5.2 manZ Mannose permease 6.5 5.7 1.0 16.7 NagC , DgsA , Crp a. Genes and operons which are induced 10-fold in three independent experiments are listed with the average fold induction for the three experiments .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18710431" target="_blank">18710431</a>]<br></li>
<li>IS + versus hs normallzed lo the eno control 2.30 2.23 0.31 4.74 1.99 0.25 1.90 1.18 0.41 Transcriptionalregulation in GlpR and RbsR repressor-deficientstrains As glpK and glpQ expression is regulated by GlpR ( Cozzarelli et a / .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8899705" target="_blank">8899705</a>]<br></li>
<li>Expression and puriﬁcation of RbsR protein Escherichia coli BL21 ( pRbsR ) transformant was grown in LB broth in the presence of 50 lg mL 1 ampicillin , and expression of RbsR was induced in mid-log phase by add-ing 1 mM IPTG .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23651393" target="_blank">23651393</a>]<br></li>
<li>The mechanism by which RbsR protein activates the expression of ompC-1acZ in high osmolarity can not be explained at present .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8522138" target="_blank">8522138</a>]<br></li>
<li>Control of expression of RbsR is not well understood .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1304369" target="_blank">1304369</a>]<br></li>
<li>This possibility was examined by Northern blot analysis of udk mRNA in the presence and absence of RbsR ( see below ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23651393" target="_blank">23651393</a>]<br></li>
<li>In the absence of inducer D-ribose , the ribose operon is repressed by a LacI-type transcription factor RbsR , which is encoded by a gene located downstream of this ribose operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23651393" target="_blank">23651393</a>]<br></li>
<li>Since promoters of the E. coli genome are sometimes located within upstream gene ORF , the udk-dcd operon was supposed to be under the control of RbsR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23651393" target="_blank">23651393</a>]<br></li>
<li>Search for the regulation targets of RbsR in the The rbsDACBK operon is induced by D-ribose that is sensed by RsbR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23651393" target="_blank">23651393</a>]<br></li>
<li>The level of purH mRNA also increased in the rbsR mutant ( Fig. 4 , purH lane ) , implying a repressive role of RbsR in transcription of the purHG operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23651393" target="_blank">23651393</a>]<br></li>
				</div></ul></div><br>
	</div>
</body>
</html>