<!doctype html>
<style>
@charset "utf-8";
body {
	height: 100%;
	margin: 0;
	padding: 0;
}

.banner {
	position: fixed;
	top: 0;
	width: 100%;
	height: 65px;
	background: gray;
	line-height: 65px;
}
.spacer {
	height: 10px;
}

.main-content {
	padding: 5%;
}

.collapse_ACT {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-style: none solid none none;
	border-color: #B4045F;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_ACT + input{
	display: none;
}
.collapse_ACT + input + div{
	display: none;
}
.collapse_ACT + input:checked + div{
	display: block;
}
.ACT {
	text-align: justify;
	text-justify: inter-word;
}
.ACT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B4045F;
}
.ACT_collapse_similarity:hover{
	background: #e5d3dc;
}
.ACT_collapse_similarity + input {
	display: none;
}
.ACT_collapse_similarity + input + div{
	display: none;
}

.ACT_collapse_similarity + input:checked + div{
	display: block;
}
.ACT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.ACT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B4045F;
	padding-left: 5px;
}
.ACT_collapse_similarity:hover{
	background: #e5d3dc;
}
.ACT_collapse_similarity + input {
	display: none;
}
.ACT_collapse_similarity + input + div{
	display: none;
}
.ACT_collapse_similarity + input:checked + div{
	display: block;
}
.ACT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.ACT_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #B4045F;
}
.ACT_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_DOM {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-style: none solid none none;
	border-color: #B404AE;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_DOM + input{
	display: none;
}
.collapse_DOM + input + div{
	display: none;
}
.collapse_DOM + input:checked + div{
	display: block;
}
.DOM {
	text-align: justify;
	text-justify: inter-word;
}
.DOM_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B404AE;
	padding-left: 5px;
}
.DOM_collapse_similarity:hover{
	background: #e8d7e7;
}
.DOM_collapse_similarity + input {
	display: none;
}
.DOM_collapse_similarity + input + div{
	display: none;
}
.DOM_collapse_similarity + input:checked + div{
	display: block;
}
.DOM_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}.DOM_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #B404AE;
}
.DOM_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_EVO {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #7401DF;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_EVO + input{
	display: none;
}
.collapse_EVO + input + div{
	display: none;
}
.collapse_EVO + input:checked + div{
	display: block;
}
.EVO {
	text-align: justify;
	text-justify: inter-word;
}
.EVO_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #7401DF;
	padding-left: 5px;
}
.EVO_collapse_similarity:hover{
	background: #e2d7ed;
}
.EVO_collapse_similarity + input {
	display: none;
}
.EVO_collapse_similarity + input + div{
	display: none;
}
.EVO_collapse_similarity + input:checked + div{
	display: block;
}
.EVO_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.EVO_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #7401DF;
}
.EVO_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_RP {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #0174DF;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_RP + input{
	display: none;
}
.collapse_RP + input + div{
	display: none;
}
.collapse_RP + input:checked + div{
	display: block;
}
.RP {
	text-align: justify;
	text-justify: inter-word;
}
.RP_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #0174DF;
	padding-left: 5px;
}
.RP_collapse_similarity:hover{
	background: #c9d7e2;
}
.RP_collapse_similarity + input {
	display: none;
}
.RP_collapse_similarity + input + div{
	display: none;
}
.RP_collapse_similarity + input:checked + div{
	display: block;
}
.RP_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.RP_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #0174DF;
}
.RP_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_SIT {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #0489B1;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_SIT + input{
	display: none;
}
.collapse_SIT + input + div{
	display: none;
}
.collapse_SIT + input:checked + div{
	display: block;
}
.SIT {
	text-align: justify;
	text-justify: inter-word;
}
.SIT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #0489B1;
	padding-left: 5px;
}
.SIT_collapse_similarity:hover{
	background: #d0dfe2;
}
.SIT_collapse_similarity + input {
	display: none;
}
.SIT_collapse_similarity + input + div{
	display: none;
}
.SIT_collapse_similarity + input:checked + div{
	display: block;
}
.SIT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.SIT_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #0489B1;
}
.SIT_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_TU {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #04B486;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_TU + input{
	display: none;
}
.collapse_TU + input + div{
	display: none;
}
.collapse_TU + input:checked + div{
	display: block;
}
.TU {
	text-align: justify;
	text-justify: inter-word;
}
.TU_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #04B486;
	padding-left: 5px;
}
.TU_collapse_similarity:hover{
	background: #d0e2de;
}
.TU_collapse_similarity + input {
	display: none;
}
.TU_collapse_similarity + input + div{
	display: none;
}
.TU_collapse_similarity + input:checked + div{
	display: block;
}
.TU_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.TU_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #04B486;
}
.TU_similar > a {
	text-decoration: none;
	color: gray;
}

ul { 
	list-style-type: none;
	padding: 0;
	margin: 0;
}
ul li > a {
	text-decoration: none;
	color: gray;
}
ul li:hover, ul li:focus {
	background: #F0F0F0;
}

ul li:first-child span {
  border-left:none;  
}

</style>
<html>
<head>
<meta charset="utf-8">
<title>TF properties - RcnR</title>
<meta name="theme-color" content="#ffffff">
</head>
<body>
	<div class="banner">
		<center><font color="white" size="16px" face="arial">RcnR</font></center>
	</div>
	<div align="center" class="main-content">
		<div class="spacer">
	</div><br>			<label class="collapse_ACT" for="_1"><font size="5px">RcnR transcriptional activity</font></label>
				<input id="_1" type="checkbox">
				<div class="ACT"><br><ul><li>Peter T. Chivers Jeffrey S. Iwig and Department of Biochemistry and Molecular Biophysics , Washington University School of Medicine , 660 South Euclid Avenue , Box 8231 , St. Louis , MO 63110 , USA Received 15 June 2009 ; received in revised form 14 August 2009 ; accepted 14 August 2009 Available online 21 August 2009 Escherichia coli RcnR is a founding member of a recently discovered large and widespread structural family of bacterial transcription factors that are predicted to respond to a variety of environmental stresses .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19703465" target="_blank">19703465</a>]<br></li>
<li>This assay , utilizing topoisomerase I , has been used to show topological changes induced by DNA-binding proteins , including Orc 1,9 and enzymes , such as lambda family recombinases .10 A 297-bp fragment of PrcnA was cloned into pBS , and the resulting plasmid was preincubated with RcnR ; topoisomerase I was then used to relax negative supercoils of the plasmid , and the topoisomers were resolved on chloroquine agarose gel .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19703465" target="_blank">19703465</a>]<br></li>
<li>Interestingly , both RcnR and FrmR appear to be sensitive to C-termina extensions because proteins modiﬁed in this way were rendered inactive in vivo ( Herring & Blattner 2004 ) , ( data not shown for RcnR ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17120142" target="_blank">17120142</a>]<br></li>
<li>Experiments were also carried out in the presence of 200 μM NiCl2 or 200 μM CoCl2 instead of EDTA and under these conditions RcnR remained tetrameric ( see Supporting Information Figure S2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18505253" target="_blank">18505253</a>]<br></li>
<li>A range of concentrations of NiCl2 or CoCl2 , were incubated with 0.5 μM mag-fura 2 ( Molecular Probes ; Carlsbad , CA ) or 0.3 μM fura-2 , respectively , for 3 h in the presence or absence of the indicated concentrations of RcnR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18505253" target="_blank">18505253</a>]<br></li>
<li>Mag-fura 2 ( 0.5 μM ) fluorescence ( λex = 380 nm , λem = 505 nm ) in the absence or presence of RcnR monomer ( 1.2 μM ) and the indicated concentrations of NiCl2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18505253" target="_blank">18505253</a>]<br></li>
<li>Fura 2 ( 0.3 μM ) fluorescence ( λex = 360 nm , λem = 500 nm ) in the absence or presence of RcnR monomer ( 360 nM ) and the indicated concentrations of CoCl2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18505253" target="_blank">18505253</a>]<br></li>
<li>63Ni content ( atoms cfu-1 ) Hydrogenase-3 activity Strain 24 58 68 7923 80 2338 Wild-type DnikR DrcnA DnikA DnikR rcnA DnikR nikA DrcnR White White Red Red White Red White 79 6 154 3 198 258 J. S. Iwig , J. L. Rowe and P. T. Chivers the presence of one of these metals , RcnR has a weaker affinity for PrcnA , and transcription can occur .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16956381" target="_blank">16956381</a>]<br></li>
<li>From left to right : G/A ladder , no RcnR , 250 to 7 nM RcnR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19703465" target="_blank">19703465</a>]<br></li>
<li>From left to right : G/A ladder , no RcnR , 100 nM RcnR , 50 nM RcnR , 20 nM RcnR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19703465" target="_blank">19703465</a>]<br></li>
<li>With the use of ITC , titration of site 2 into RcnR in buffer P showed a signal change for an exothermic reaction , demonstrating that RcnR can indeed bind to this short DNA duplex ( Fig. 4a ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19703465" target="_blank">19703465</a>]<br></li>
<li>RcnR was dialyzed against the stock of buffer solution to prevent mismatches in buffer composition .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19703465" target="_blank">19703465</a>]<br></li>
<li>RcnR and FrmR share 40 % sequence identity and both may be repressors that uncover their speciﬁc promoters in the presence of different inducers , metal cations or formaldehyde , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17120142" target="_blank">17120142</a>]<br></li>
<li>RcnR and CsoR share three conserved ligand positions ( 35 , 60 , and 64 ) , suggesting that one or more of these residues may be essential for the allosteric response to metal binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18505253" target="_blank">18505253</a>]<br></li>
<li>Repression of the Ni2 / Co2 efﬂux transporter RcnA by apo-RcnR is relieved in M63/glucose minimal medium as nickel levels increase above 500 nM ( 25 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21097627" target="_blank">21097627</a>]<br></li>
<li>Nickel ions in pool B enter the cell through an unidentiﬁed pathway and are available to other Ni-requiring enzymes , such as Glx I , as well as the Ni2 + - efflux protein RcnA , and the regulatory proteins RcnR and NikR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16956381" target="_blank">16956381</a>]<br></li>
<li>In contrast , the six-coordinate Ni ( II ) sites of NikR and RcnR appear to be designed for spontaneous metal release in response to decreasing metal ion concentration , as both of these sites function to respond to fluctuations in intracellular nickel concentrations that occur under stress conditions .9 , 63 The use of a single cysteine thiolate ligation in NikR and RcnR would seem to be an ideal choice for a metal ion electronic structure sensor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18505253" target="_blank">18505253</a>]<br></li>
<li>Formate likely does not act directly to affect RcnR and NikR function .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16956381" target="_blank">16956381</a>]<br></li>
<li>( 8 ) Iwig , J. S. , and Chivers , P. T. ( 2010 ) Coordinating intracellular nickel-metal-site structure-function relationships and the NikR and RcnR repressors .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23737072" target="_blank">23737072</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22971172" target="_blank">22971172</a>]<br></li>
<li>These features have been demonstrated for the Ni ( II ) - responsive transcriptional-repressor , NikR , from E. coli , which has been shown to-bind to a number of metals with high affinity7 but responds in vivo only to the presence of Ni ( II ) ions .8 Recently , we identified a nickel - and cobalt-responsive metalloregulator , E. coli RcnR ,9 that represses transcription of rcnA , which encodes a nickel and cobalt efflux protein .10 RcnR exhibits only minimal sequence identity with the previously described families of transcriptional-regulators leading to the proposal of a new metalloregulator family .9 RcnR homologs are found throughout the eubacterial kingdom , emphasizing their widespread role in regulating the cell physiology of microbes from diverse growth niches .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18505253" target="_blank">18505253</a>]<br></li>
<li>The RcnR - and NikR-mediated transcriptional responses at added NiCl2 concentrations 10 nM are independent of NikABCDE .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16956381" target="_blank">16956381</a>]<br></li>
<li>The nickel-dependent regulation of yqjH and yqjI by YqjI is physiologically consistent with the nickel homeostasis systems in E. coli that are regulated by the NikR and RcnR transcription factors .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21097627" target="_blank">21097627</a>]<br></li>
<li>The presence of two Ni2 + - responsive regulators in E. coli raises the question of how Ni2 + partitions between NikR and RcnR so that repression of nikABCDE occurs before activation of rcnA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16956381" target="_blank">16956381</a>]<br></li>
<li>The two nickel-responsive transcriptional-regulators from E. coli , RcnR and NikR , operate in a sequential fashion in response to increasing intracellular nickel9 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18505253" target="_blank">18505253</a>]<br></li>
<li>The same induction level was observed in the rcnR mutant , indicating that saturating concentrations of Ni or Co fully alleviate repression via unbinding of Ni - or Co-loaded RcnR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21040754" target="_blank">21040754</a>]<br></li>
<li>However , when measured in an rcnR background , the induction level of rcnR increased tenfold , indicating that this regulation is mediated by RcnR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21040754" target="_blank">21040754</a>]<br></li>
<li>A single shifted band was observed with increasing RcnR concentration when 1 mM EDTA was present in the binding buffer ( Fig. 4A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16956381" target="_blank">16956381</a>]<br></li>
<li>The RcnR protein unfolded cooperatively as a function of increasing urea concentration ( Figure 2B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18505253" target="_blank">18505253</a>]<br></li>
<li>The distribution shifted by − 0.8 supercoils in the presence of RcnR , indicating that the protein introduced a single negative supercoil into the plasmid .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19703465" target="_blank">19703465</a>]<br></li>
<li>The presence of RcnR resulted in a shift in the Ni-dependent fluorescence quenching curve ( Figure 4A ) , indicating competition for Ni-binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18505253" target="_blank">18505253</a>]<br></li>
<li>Protein samples were in Buffer M. Electronic Spectroscopy of Ni or Co-substituted RcnR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18505253" target="_blank">18505253</a>]<br></li>
<li>Text in parentheses refers to the identity of RcnR Ni/Co coordinating residues .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18505253" target="_blank">18505253</a>]<br></li>
<li>The analysis of first coordination sphere EXAFS data for Co ( II ) - RcnR also reveals the presence of a S-donor ligand .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18505253" target="_blank">18505253</a>]<br></li>
<li>To test this , we measured the activity of a PrcnR-lacZ fusion with increasing Ni ( II ) concentrations in the presence and in the absence of RcnR ( Supplementary Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19703465" target="_blank">19703465</a>]<br></li>
<li>In the presence of RcnR , LacZ activity increases by ∼ 2-fold upon addition of 1 mM Ni ( II ) , while constitutive , high activity is observed for a strain lacking RcnR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19703465" target="_blank">19703465</a>]<br></li>
<li>Co ( II ) - responsiveness -- Mutant RcnR constructs were tested for LacZ activity in the presence of 200 □ M CoCl2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18505253" target="_blank">18505253</a>]<br></li>
<li>LacZ activity is increased ~ 20-fold under these experimental conditions when wild-type RcnR is present ( Figure 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18505253" target="_blank">18505253</a>]<br></li>
<li>All RcnR mutants , except H3L , bound ~ 1 equivalent of Ni/RcnR monomer ( Supporting Information Figure S9 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18505253" target="_blank">18505253</a>]<br></li>
<li>The regulator Fur and the putative regulator RcnR are involved in this process and each is necessary for metal-dependent regulation Acknowledgments This work was supported by the Land Sachsen-Anhalt ( HWP 56 IF ) and Deutsche Forschungsgemeinschaft ( Ni262/3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17120142" target="_blank">17120142</a>]<br></li>
<li>The diversity in operator site organization may thus provide diversity in gene regulation among RcnR homologs which are involved in metal sensing and in oxidative-stress response .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21040754" target="_blank">21040754</a>]<br></li>
<li>Because O2 inactivates FNR and prevents Pnik expression , the nickel ions required for RcnR inactivation must enter the cells in the absence of NikABCDE .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16956381" target="_blank">16956381</a>]<br></li>
<li>Although the induction is weak , RcnR clearly regulates transcription of its own gene in response to the same Ni ( II ) concentrations that control regulation of rcnA .5 Quantitation of the core binding site as a function of RcnR concentration and subsequent fitting using a Hill model demonstrated apparent positive cooperativity for DNA binding ( n = 3.3 ) , suggesting that multiple protomers bind to this DNA fragment .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19703465" target="_blank">19703465</a>]<br></li>
<li>RcnR directly regulates transcription of the gene encoding the RcnA nickel and cobalt efflux protein by coordination of DNA-binding and metal-binding activities .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19703465" target="_blank">19703465</a>]<br></li>
<li>Puriﬁed RcnR binds directly to the rcnA promoter DNA fragment and this interaction is inhibited by nickel and cobalt .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16956381" target="_blank">16956381</a>]<br></li>
<li>Despite the lack of a known DNA binding motif , RcnR represses transcription by directly binding to the rcnA promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18505253" target="_blank">18505253</a>]<br></li>
<li>Involvement of Fur in expression control of rcnA suggested that RcnR -- Fur interaction is required for promoter binding or for functional alteration of Fur except for overproduction of RcnR in a Dfur strain still inhibited rcnA expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17120142" target="_blank">17120142</a>]<br></li>
<li>Intrinsic bending upstream of rcnA caused by the G-tracts may aid in RcnR binding , but it may also decrease the strength the promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19703465" target="_blank">19703465</a>]<br></li>
<li>a To investigate if RcnR exerts repression of rcnA-expression through direct binding to the promoter-region upstream of rcnA , the RcnR protein was puriﬁed as N-terminal as well as Cterminal strep-tag fusion protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17120142" target="_blank">17120142</a>]<br></li>
<li>Because this is negative evidence , it can not be ruled out that RcnR might bind to the rcn-promoter region as a repressor of rcnA transcription under different experimental conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17120142" target="_blank">17120142</a>]<br></li>
<li>Overproduction of rcnR inhibited induction of rcnA by metal cations but RcnR did not bind to the rcnA promoter in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17120142" target="_blank">17120142</a>]<br></li>
<li>However , RcnR alone was not suficient for this regulatory process because rcnA was still induced by metal cations in the absence of RcnR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17120142" target="_blank">17120142</a>]<br></li>
<li>Thus , the mechanism by which RcnR represses RcnA expression remains elusive Figure 5 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17120142" target="_blank">17120142</a>]<br></li>
<li>The extension of the footprint into rcnR itself suggested that RcnR regulates its own expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19703465" target="_blank">19703465</a>]<br></li>
<li>The analogy is not perfect because ZntR acts as a transcriptional-activator of the zinc exporter ZntA in the presence of excess zinc , whereas rcnA expression is derepressed by inactivation of RcnR in the presence of excess nickel .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16956381" target="_blank">16956381</a>]<br></li>
<li>In addition to rcnA , RcnR controls expression of its own gene in response to Ni and Co , but the two genes are differentially expressed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21040754" target="_blank">21040754</a>]<br></li>
<li>yohN lies directly downstream of rcnA , and we show here that it is coexpressed with rcnA and that its expression is induced by Ni and Co via RcnR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21665978" target="_blank">21665978</a>]<br></li>
<li>RcnR specifically binds to the rcnR -- rcnA intergenic region , resulting in the repression of both rcnR and rcnA at low Ni ( II ) levels .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19703465" target="_blank">19703465</a>]<br></li>
<li>In contrast , RcnR-dependent repression of rcnA is maintained until nikABCDE expression is fully repressed , then increasing Ni ( II ) releases RcnR from DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18505253" target="_blank">18505253</a>]<br></li>
<li>Under steady state growth conditions , the NikR repressor responds to lower ambient Ni ( II ) concentrations than RcnR , as nikABCDE expression is completely repressed at nickel levels that exhibit minimal rcnA expression .9 As Ni ( II ) levels increase , RcnR-repression of rcnA transcription is diminished allowing full induction of RcnA expression to prevent excess metal accumulation in the cell .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18505253" target="_blank">18505253</a>]<br></li>
<li>These affinities make RcnR sensitive to low levels of free nickel and cobalt ions ( lower limit ~ 1 nM ) in E. coli that , in the absence of other effects , would lead to relatively rapid induction of rcnA expression in response to excess metal .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18505253" target="_blank">18505253</a>]<br></li>
<li>Unexpectedly , C-terminally extended RcnR was only inactive for nickel or cobalt dependent induction of rcnA but overproduction of RcnR ( either with N - or C-terminal strep-tag extensions ) prevented iron-dependent regulation of rcnA in a Dfur background ( Figure 7C and data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17120142" target="_blank">17120142</a>]<br></li>
<li>This positive iron-regulation was somewhat surprising , because Fur exerts its mode of regulation by repression Fur interacted with the rcnAp promoter in vivo Gene expression of rcnR responded to iron and RcnR may function as a negative regulator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17120142" target="_blank">17120142</a>]<br></li>
<li>This indicated that RcnR was involved in negative regulation of rcnA by iron .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17120142" target="_blank">17120142</a>]<br></li>
<li>This suggested that not only RcnR but also Fur was involved in iron-dependent negative regulation of rcnA and that the presence of both RcnR and Fur led to repression of DrcnA : : lacZ in the presence of iron .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17120142" target="_blank">17120142</a>]<br></li>
<li>Nickel and cobalt-dependent regulation of rcnA expression required the newly identiﬁed transcriptional-repressor RcnR ( formerly YohL ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16956381" target="_blank">16956381</a>]<br></li>
<li>Thus , Fur and RcnR regulation of rcnA expression is not interdependent .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17120142" target="_blank">17120142</a>]<br></li>
<li>Previous studies in E. coli also have shown that rcnA may be regulated by Fur in response to iron and that the RcnR protein ( that represses rcnA expression ) is itself induced by iron via a Fur-independent pathway ( 29 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21097627" target="_blank">21097627</a>]<br></li>
<li>Likely , high iron conditions repressed rcnA expression through an increase of RcnR protein levels .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17120142" target="_blank">17120142</a>]<br></li>
<li>Nevertheless , in a wild-type strain both RcnR and Fur had to be present to inhibit induction of rcnA by iron ( Figures 5 , 7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17120142" target="_blank">17120142</a>]<br></li>
<li>However , there is no evidence that Ni-Fur is the regulator of rcnA and this would not explain the observed positive regulation by metals except for interplay of Ni-Fur with RcnR that seems not to occur ( Figures 5 , 7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17120142" target="_blank">17120142</a>]<br></li>
<li>It is unclear if DNA wrapping by RcnR provides a physiological advantage for transcriptional repression or metal sensing .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19703465" target="_blank">19703465</a>]<br></li>
<li>If MraZ indeed binds to DNA as a dodeca-mer , it might wrap the DNA around its toroidal structure similar to the E. coli transcriptional-repressor RcnR ( 65 ) and DNA gyrase ( 66 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24659771" target="_blank">24659771</a>]<br></li>
<li>All four sets of mutations had striking effects on RcnR binding , leading to decreases in affinity and increases in ΔHobs over the full temperature range tested ( 8 -- 30 ° C ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19703465" target="_blank">19703465</a>]<br></li>
<li>RcnR DNA affinity increases with length of flanking DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19703465" target="_blank">19703465</a>]<br></li>
<li>Both metals bind to RcnR with nanomolar affinity and stabilize the protein to denaturation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18505253" target="_blank">18505253</a>]<br></li>
<li>One RcnR tetramer binds to single-site duplexes by ITC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19703465" target="_blank">19703465</a>]<br></li>
<li>Although the duplexes used in ITC were not as large as those used for footprinting , these experiments provide no evidence that more than two RcnR tetramers can bind to PrcnA and instead demonstrate that one tetramer binds to each TACT-G6-N-AGTA motif .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19703465" target="_blank">19703465</a>]<br></li>
<li>Together , these in vitro experiments demonstrate that RcnR binds speciﬁcally to PrcnA , and this interaction is negatively affected by the presence of nickel ions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16956381" target="_blank">16956381</a>]<br></li>
<li>RcnR is necessary for nickel-dependent PrcnA activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16956381" target="_blank">16956381</a>]<br></li>
<li>Therefore , all of the experimental data are consistent with two RcnR tetramers binding to PrcnA , although the possibility of additional tetramers binding to large wrapped DNA duplexes can not be formally excluded .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19703465" target="_blank">19703465</a>]<br></li>
<li>LacZ expression data and in vitro DNA binding experiments reveal that RcnR acts by binding to PrcnA in the absence of Ni2 + ( or Co2 + ) and repressing transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16956381" target="_blank">16956381</a>]<br></li>
				</div></ul><br>
			<label class="collapse_DOM" for="_2"><font size="5px">RcnR protein domains</font></label>
				<input id="_2" type="checkbox">
				<div class="DOM"><br><ul><li>Since there is only one Cys ligand present in EcRcnR , the two metals both bind Cys35 in six-coordinate sites that also contain at least two imidazole ligands .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18505253" target="_blank">18505253</a>]<br></li>
<li>In the co-crystal structure of this complex , sequence-specific contacts from two α-helix -- minor groove interactions induce significant kinking of the DNA away from the protein and distortion of the minor groove .21 A-DNA has been reported to play a role in DNA-binding by DNA polymerases ,19 small acid-soluble proteins in dormant Bacillus spores ,22,23 and catabolite repressor protein .24 Nofootprinting assays .26 DNA is thought to wrap around the C-terminal domain of GyrA utilizing a large basic patch on the outer edge of the protein , but this domain folds into a β-pinwheel structure27 that is unlike that of MtCsoR .2 Similar to DNA gyrase , our studies indicate that the DNA wraps around RcnR instead of looping away from the protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19703465" target="_blank">19703465</a>]<br></li>
<li>RcnR is an all-helical protein with poorly conserved loop sequences , which suggests that an α-helix of the protein binds in the DNA minor groove .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19703465" target="_blank">19703465</a>]<br></li>
<li>tably , the eukaryotic Tc3 transposase recognizes an A-DNA -- B-DNA junction that depends upon a G7 tract .25 Unlike RcnR , a helix -- turn -- helix motif binds in the DNA major groove near this junction , and the DNA bends sharply away from the protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19703465" target="_blank">19703465</a>]<br></li>
<li>In this study , we extended our previous studies on RcnR using biophysical and biochemical techniques to probe the mechanism of DNA interaction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19703465" target="_blank">19703465</a>]<br></li>
<li>We found that an RcnR tetramer recognizes a TACT-G6-N-AGTA motif , of which there are two in the rcnA -- rcnR intergenic region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19703465" target="_blank">19703465</a>]<br></li>
<li>To test the prediction that specific binding was dependent on the TACT-G6-N1-AGTA motif , we used isothermal titration calo-rimetry ( ITC ) to understand thermodynamic aspects of the RcnR -- DNA interaction in short oligonucleotide duplexes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19703465" target="_blank">19703465</a>]<br></li>
<li>A model for the interaction between RcnR and PrcnA must account for both the specific binding to a pair of TACT-G6-N-AGTA motifs and the surprising nonspecific interactions with the flanking DNA that lead to DNA wrapping .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19703465" target="_blank">19703465</a>]<br></li>
<li>The spacing of these half-sites ( one turn of A - / B-form DNA ) is similar to the spacing of helical side-chain residues in helix α1 ( ∼ 34 Å ) in two monomers on the same face of the putative RcnR tetramer , which is consistent with the stoichiometry of the RcnR interaction with site 1 and site 2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19703465" target="_blank">19703465</a>]<br></li>
<li>RcnR makes nonspecific contacts with DNA flanking the core binding site The footprinting results demonstrated that RcnR -- DNA contacts are made well outside of the core protection region , which suggests that RcnR binding affinity should change with DNA length .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19703465" target="_blank">19703465</a>]<br></li>
<li>Our data indicate that wrapping is another feature of the RcnR -- DNA complex as the protein makes nonspecific contacts with the DNA over an ∼ 50-bp stretch on either side of the core binding site in addition to the specific interactions made at each operator site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19703465" target="_blank">19703465</a>]<br></li>
<li>Despite the lack of a known DNA-binding motif , RcnR represses transcription by directly binding to the rcnA promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18505253" target="_blank">18505253</a>]<br></li>
<li>ITC experiments revealed that two RcnR tetra-mers bind to the 44-bp core DNA-binding site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19703465" target="_blank">19703465</a>]<br></li>
<li>We show here that the two promoters are convergent and that the two RcnR DNA-binding boxes are symmetrically located from each transcription start site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21040754" target="_blank">21040754</a>]<br></li>
<li>RcnR is structurally similar to CsoR , a Cu-sensing repressor whose crystallographic structure reveals a homodimeric state in which Cu ( I ) is coordinated at the interface of the two monomers [ 8 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21040754" target="_blank">21040754</a>]<br></li>
<li>tate of RcnR S ly b m e s s A Sedimentation equilibrium experiments were performed to rigorously characterize the oligomeric state of the protein and determine whether metal-binding caused any changes in RcnR assembly state .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18505253" target="_blank">18505253</a>]<br></li>
<li>Using the PAR assay for transition metal ion detection ,30 purified RcnR contained < 0.04 metal equivalents per monomer .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18505253" target="_blank">18505253</a>]<br></li>
<li>The number of metal-binding sites per RcnR monomer was also determined using the colorimetric divalent metal indicator , PAR ( described in the Experimental Section ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18505253" target="_blank">18505253</a>]<br></li>
<li>These data are all consistent with one metal-binding site per RcnR monomer , or four binding sites per tetramer .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18505253" target="_blank">18505253</a>]<br></li>
<li>Thus , the RcnR class may be unique because it has recruited residues from the Nterminal region to aid in metal recognition .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18505253" target="_blank">18505253</a>]<br></li>
<li>The identification of four RcnR References residues that when mutated drastically affect metal-binding and metal-responsiveness suggests that RcnR/CsoR-like sequences can be analyzed to make predictions about ligand-binding specificities .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18505253" target="_blank">18505253</a>]<br></li>
<li>Mutation of RcnR identified specific residues involved in metal-binding and support the ligand set determined from EXAFS analysis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18505253" target="_blank">18505253</a>]<br></li>
<li>RcnR and CsoR share three conserved ligand positions ( 35 , 60 , and 64 ) , suggesting that one or more of these residues may be essential for the allosteric response to metal-binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18505253" target="_blank">18505253</a>]<br></li>
<li>CsoR coordinates Cu ( I ) with three ligands ( Cys35 , His60 , Cys64 , using RcnR numbering ) ,12 , 79 while RcnR responds biologically to metals that require higher coordination spheres by recruiting ligands from the N-terminal region , which is not used by CsoR for metal coordination , as well as a residue changes at position 64 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18505253" target="_blank">18505253</a>]<br></li>
<li>The H64L Ni ( II ) - RcnR spectrum contained two weak features at 380 nm and 600 nm not seen for WT .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18505253" target="_blank">18505253</a>]<br></li>
<li>In contrast , the H60L Ni ( II ) - RcnR spectrum showed neither an increase in the intensity of d-d features nor a significant perturbation of the 280 nm peak , suggesting that this residue is not involved in Ni coordination .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18505253" target="_blank">18505253</a>]<br></li>
<li>Moreover , RcnR DNA-binding is speciﬁcally modulated by one Ni or Co equivalent and not by other metals .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21040754" target="_blank">21040754</a>]<br></li>
<li>Here , using in vitro footprint experiments , we present evidence that addition of one Ni or Co equivalent leads to the release of RcnR interaction from the two recognition sites lying in the rcnRercnA promotereoperator region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21040754" target="_blank">21040754</a>]<br></li>
<li>The identification of the Ni ( II ) and Co ( II ) binding sites in RcnR , in combination with bioinformatics analysis of all RcnR/CsoR family members , identify a four amino acid fingerprint that likely defines ligand-binding specificity , leading to an emerging picture of the similarities and differences between different classes of RcnR/CsoR proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18505253" target="_blank">18505253</a>]<br></li>
<li>n io s s u c is D RcnR plays a pivotal role in intracellular sensing of Ni ( II ) and Co ( II ) ions in E. coli , where it must specifically interact with these metal ions when they are deemed to be in excess .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18505253" target="_blank">18505253</a>]<br></li>
<li>Text in parentheses refers to the identity of RcnR Ni/Co coordinating residues .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18505253" target="_blank">18505253</a>]<br></li>
<li>The A2 * mutation likely impairs Ni ( II ) - responsiveness by forming a square-planar ATCUN motif ( Xaa-Xaa-His ) .70 , 71 The tripartite coordination of Ni ( II ) and Co ( II ) by the two N-terminal residues of RcnR is a key aspect of its selectivity towards metals with preferences for higher coordination numbers .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18505253" target="_blank">18505253</a>]<br></li>
<li>The higher coordination number observed in NmtR is achieved by the recruitment of two His ligands from a short C-terminal extension in the protein that is not present in CzrA ,76 similar to the recruitment of N-terminal residues for Ni/Co coordination by RcnR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18505253" target="_blank">18505253</a>]<br></li>
<li>The involvement of the N-terminal amino group and the side-chain of the second residues is similar to that observed for Ni-SOD .52 , 53 In this case , the deprotonated main chain amide of residue 2 also coordinates the Ni atom , suggesting that RcnR also uses main chain coordination by the amide of His3 for Ni ( II ) and Co ( II ) binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18505253" target="_blank">18505253</a>]<br></li>
<li>The differential contributions of Cys35 and His60 to Co ( II ) - and Ni ( II ) - responsiveness suggest that these residues play different roles in the structural response of the RcnR to metal-binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18505253" target="_blank">18505253</a>]<br></li>
<li>The Cys35 mutant data suggest that Ni ( II ) dependent allostery originates from intramonomer metal coordination , in contrast to Co ( II ) - RcnR and Cu ( I ) - CsoR where this residue is essential for activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18505253" target="_blank">18505253</a>]<br></li>
<li>This unusual DNA-binding fold contains several conserved positively charged amino acids in α1 and α2 , and two of these residues were shown to be necessary for high-affinity DNA-binding of MtCsoR .2 RcnR , the first biochemically described member of the RcnR/CsoR family , was identified as a transcriptional regulator of the rcnA gene ,5 which encodes a Ni ( II ) / Co ( II ) efflux protein in E. coli .8 RcnA not only prevents accumulation of toxic metal levels8 but also blocks premature activation of a second transcription factor , NikR , through competition for Ni ( II ) , allowing nickel ions to be trafficked to the hydrog-enase enzymes .5 Unlike the trigonal planar Cu ( I ) site of MtCsoR , Ni ( II ) and Co ( II ) bind to RcnR at pseudo-octahedral six-coordinate sites .4 RcnR uses residues in positions equivalent to those used by MtCsoR to bind metal ions , in addition to ligands in an Nterminal metal-binding motif .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19703465" target="_blank">19703465</a>]<br></li>
<li>Ni ( II ) and Co ( II ) do not commonly adopt the trigonal planar geometry ( two cysteines and one histidine ) observed in the Cu ( I ) - CsoR structure ,12 suggesting that the RcnR protein must supply additional metal-binding ligands .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18505253" target="_blank">18505253</a>]<br></li>
<li>RcnR is functionally analogous to M. tuberculosis NmtR and KmtR , which also utilize six-coordinate geometry to bind Ni ( II ) and Co ( II ) between two monomer chains .6 , 76 , 77 However , KmtR and NmtR use only N - and O-ligands for metal coordination .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18505253" target="_blank">18505253</a>]<br></li>
<li>RcnR ( Pfam accession # 02583 ) contains a charged N-terminal region ( amino acids 1 -- 34 ) , followed by a highly conserved stretch of primarily non-polar residues ( amino acids 35 -- 64 ) , and a variable segment consisting of the ﬁnal 36 residues ( Fig. 2B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16956381" target="_blank">16956381</a>]<br></li>
<li>RcnR residues Cys35 and His60 are invariant among homologous proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16956381" target="_blank">16956381</a>]<br></li>
<li>Effect of side-chain substitution of conserved residues on RcnR function in vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18505253" target="_blank">18505253</a>]<br></li>
<li>Mutated RcnR residues included all five histidine residues ( 3 are highly conserved ) , a subset of conserved Glu and Asp residues , and the highly conserved Cys35 ( see Supporting Information ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18505253" target="_blank">18505253</a>]<br></li>
<li>The difference between EXAFS and mutagenesis , combined with structural homology , in assigning the number of His ligands is not unreasonable owing to the effect of His conformation on fitting these residues to EXAFS data .72 A sequence alignment of the predicted RcnR proteins ( see Supporting Information ) does not reveal a highly conserved sixth residue with a side chain likely to function in metal-binding ( Asp , Glu , Met , His , Cys ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18505253" target="_blank">18505253</a>]<br></li>
<li>Together , these in vitro experiments demonstrate that RcnR binds speciﬁcally to PrcnA , and this interaction is negatively affected by the presence of nickel ions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16956381" target="_blank">16956381</a>]<br></li>
<li>160 140 120 ΔrcnR LacZ activity ( Miller Units Competition between nickel homeostasis pathways The identiﬁcation of the RcnA-RcnR nickel homeostasis module raised two important questions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16956381" target="_blank">16956381</a>]<br></li>
<li>The presence of two Ni2 + - responsive regulators in E. coli raises the question of how Ni2 + partitions between NikR and RcnR so that repression of nikABCDE occurs before activation of rcnA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16956381" target="_blank">16956381</a>]<br></li>
				</div></ul><br>
			<label class="collapse_EVO" for="_3"><font size="5px">RcnR evolution</font></label>
				<input id="_3" type="checkbox">
				<div class="EVO"><br><ul><li>RcnR has 40 % sequence identity with E. coli FrmR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16956381" target="_blank">16956381</a>]<br></li>
<li>RcnR and FrmR share 40 % sequence identity and both may be repressors that uncover their speciﬁc promoters in the presence of different inducers , metal cations or formaldehyde , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17120142" target="_blank">17120142</a>]<br></li>
<li>The identification of the Ni ( II ) and Co ( II ) binding sites in RcnR , in combination with bioinformatics analysis of all RcnR/CsoR family members , identify a four amino acid fingerprint that likely defines ligand-binding specificity , leading to an emerging picture of the similarities and differences between different classes of RcnR/CsoR proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18505253" target="_blank">18505253</a>]<br></li>
<li>RcnR is a member of an uncharacterized family of regulatory proteins found throughout the eubacterial kingdom .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16956381" target="_blank">16956381</a>]<br></li>
<li>To our knowledge , these results provide the ﬁrst evidence of DNA-binding by a member of the RcnR protein family .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16956381" target="_blank">16956381</a>]<br></li>
<li>This difference also manifests itself in the metal-binding affinities of the two proteins , wherein KmtR has a higher relative affinity for Ni ( II ) and Co ( II ) than NmtR .77 NmtR and KmtR , like RcnR , are members of a large family of metalloregulators ( ArsR/SmtB ) that show distinct metal-binding specificities that correlate with differences in coordination number .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18505253" target="_blank">18505253</a>]<br></li>
<li>This mode of regulation is also seen for the ArsR/SmtB family of metalloregulators ( Busenlehner et al. , 2003 ) ; however , RcnR shows little sequence or structural similarity with this family .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16956381" target="_blank">16956381</a>]<br></li>
<li>Additionally , the expected RcnR protein is about 30-residues shorter than most ArsR/SmtB family members .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16956381" target="_blank">16956381</a>]<br></li>
				</div></ul><br>
			<label class="collapse_RP" for="_4"><font size="5px">Processes regulated by RcnR</font></label>
				<input id="_4" type="checkbox">
				<div class="RP"><br><ul><li>The diversity in operator site organization may thus provide diversity in gene regulation among RcnR homologs which are involved in metal sensing and in oxidative stress response .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21040754" target="_blank">21040754</a>]<br></li>
<li>This expression of rcnR from a high-copy number vector may have led to a constitutively repressed rcnA gene , making it probable that apo-RcnR ( rather than a hypothetical metallo-RcnR ) is the active regulator conﬁg-uration .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17120142" target="_blank">17120142</a>]<br></li>
<li>This shows that RcnR controls its own expression and suggests that supplementary regulatory events may control the expression of rcnR and therefore metal efﬂux .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21040754" target="_blank">21040754</a>]<br></li>
<li>Nickel and cobalt-dependent regulation of rcnA expression required the newly identiﬁed transcriptional repressor RcnR ( formerly YohL ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16956381" target="_blank">16956381</a>]<br></li>
<li>rcnA expression is controlled by a newly identiﬁed nickel-dependent regulator RcnR ( formerly YohL ) in E. coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16956381" target="_blank">16956381</a>]<br></li>
<li>Based on the results described below , we have renamed this gene RcnR , a repressor of rcnA expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16956381" target="_blank">16956381</a>]<br></li>
<li>In addition to rcnA , RcnR controls expression of its own gene in response to Ni and Co , but the two genes are differentially expressed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21040754" target="_blank">21040754</a>]<br></li>
<li>Previous studies in E. coli also have shown that rcnA may be regulated by Fur in response to iron and that the RcnR protein ( that represses rcnA expression ) is itself induced by iron via a Fur-independent pathway ( 29 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21097627" target="_blank">21097627</a>]<br></li>
<li>This suggested that RcnR was involved in expression control of rcnA and that RcnR probably serves as a negative regulator for rcnA expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17120142" target="_blank">17120142</a>]<br></li>
<li>This indicated that RcnR was involved in negative regulation of rcnA by iron .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17120142" target="_blank">17120142</a>]<br></li>
<li>This suggested that not only RcnR but also Fur was involved in iron-dependent negative regulation of rcnA and that the presence of both RcnR and Fur led to repression of DrcnA : : lacZ in the presence of iron .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17120142" target="_blank">17120142</a>]<br></li>
<li>Because RcnR homologues are preferentially located near genes encoding metal requiring enzymes , or genes important in metal trafficking , they likely play a role in metal homeostasis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16956381" target="_blank">16956381</a>]<br></li>
<li>RcnR directly regulates transcription of the gene encoding the RcnA nickel and cobalt efflux protein by coordination of DNA-binding and metal-binding activities .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19703465" target="_blank">19703465</a>]<br></li>
<li>Mutation of RcnR identified specific residues involved in metal binding and support the ligand set determined from EXAFS analysis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18505253" target="_blank">18505253</a>]<br></li>
				</div></ul><br>
			<label class="collapse_SIT" for="_5"><font size="5px">RcnR binding site</font></label>
				<input id="_5" type="checkbox">
				<div class="SIT"><br><ul><li>There are two similar sequences , TGCTCAGGGGGAGTA , upstream of yfaX , encoding a putative transcription factor , and TTATTGGG-GGAAGTA , upstream of insH , encoding a transposase ; however , it is not obvious how these could be linked to nickel physiology or if RcnR could interact with these sites given the number of base substitutions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19703465" target="_blank">19703465</a>]<br></li>
<li>DNAse I footprints in the presence of puriﬁed RcnR conﬁrmed the occupancy of a 43-bp region covering the two boxes ( Fig. 2A ) using here a 325-bp probe located from 166 to þ159 bp relative to the ATG of rcnR [ 7 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21040754" target="_blank">21040754</a>]<br></li>
<li>Puriﬁed RcnR was titrated against a 160 bp 32P-end-labelled PrcnA fragment in binding buffer at 22 ° C containing either 1 mM EDTA ( A ) or 100 mM NiCl2 ( B ) prior to electrophoresis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16956381" target="_blank">16956381</a>]<br></li>
<li>B. Sequence alignment of E. coli RcnR and homologues .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16956381" target="_blank">16956381</a>]<br></li>
<li>RcnR has 40 % sequence identity with E. coli FrmR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16956381" target="_blank">16956381</a>]<br></li>
<li>Our studies demonstrated that the G-tracts of the RcnR operator sites convey A-form-like properties to PrcnA , suggesting that these sequences could provide a structural basis for DNA recognition .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19703465" target="_blank">19703465</a>]<br></li>
<li>The minor groove offers fewer possibilities for hydrogen-bond donor -- acceptor interactions18 such that additional sequence or structural information may be present in the RcnR binding sites to facilitate specific ( i.e. , highaffinity ) binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19703465" target="_blank">19703465</a>]<br></li>
<li>Core RcnR binding site constructs were cloned into pBS using the oligonucleotide pairs JI350/JI351 ( core ) and JI369/JI370 ( single site ) , which were purchased phosphorylated at the 5 ′ end and contain overhangs equivalent to those generated by EagI and XhoI .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19703465" target="_blank">19703465</a>]<br></li>
<li>Despite the lack of a known DNA binding motif , RcnR represses transcription by directly binding to the rcnA promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18505253" target="_blank">18505253</a>]<br></li>
<li>The two repeated RcnR binding boxes are framed with dashed lines 3.2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21040754" target="_blank">21040754</a>]<br></li>
<li>We show here that the two promoters are convergent and that the two RcnR DNA binding boxes are symmetrically located from each transcription start site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21040754" target="_blank">21040754</a>]<br></li>
<li>Protection patterns that extended beyond this sequence are not shown Results DNase I footprinting reveals an extended binding site for RcnR We showed previously by mobility shift assay that RcnR forms a specific complex with a 160-bp fragment of PrcnA .5 However , this experiment did not reveal a specific DNA motif that was recognized by the protein as there are several sets of inverted-repeats in this region that could potentially interact with RcnR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19703465" target="_blank">19703465</a>]<br></li>
<li>TACT inverted-repeats are critical for high-affinity DNA binding by RcnR The role of the T1A2C3T4 inverted-repeats in RcnR binding was tested specifically by mutation of pairs of equivalent positions in the inverted-repeat in site 1 and by monitoring protein binding by ITC ( see Table S2 for constructs ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19703465" target="_blank">19703465</a>]<br></li>
<li>Two sequence features were found to be important for high-affinity DNA binding by RcnR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19703465" target="_blank">19703465</a>]<br></li>
<li>With the identification of specific sequence requirements for RcnR binding , the DNA recognition sites of RcnR proteins from other organisms have also been identified , revealing potentially distinct mechanisms of binding among this family due to single or dual binding sites and the sequence of the site itself .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19703465" target="_blank">19703465</a>]<br></li>
<li>PrcnA with sequence boundaries for different duplexes used in ITC experiments is shown in ( e ) Two RcnR tetramers bind to the full core binding site The interaction of RcnR with a 44-bp DNA duplex spanning both operator sites ( site 12 , Fig. 4e ) was also examined using ITC to determine the stoichi-ometry of the complex and examine any potential cooperativity between bound tetramers ( Fig. 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19703465" target="_blank">19703465</a>]<br></li>
<li>ITC experiments revealed that two RcnR tetra-mers bind to the 44-bp core DNA binding site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19703465" target="_blank">19703465</a>]<br></li>
<li>DNA flanking the core binding site could contain additional RcnR binding sites or form specific or nonspecific contacts with repressors bound to the core site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19703465" target="_blank">19703465</a>]<br></li>
<li>Mobility shift assays were conducted with DNA duplexes from 81 to 300 base pairs , all centered on the core RcnR binding site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19703465" target="_blank">19703465</a>]<br></li>
<li>RcnR bound site 1 and site 2 with a stoichiometry of approximately one tetramer per duplex ( site 1 , 1.3 ± 0.1 ; site 2 , 1.2 ± 0.1 ) and ΔCp , obs values of − 0.55 and − 0.52 kcal / ( mol K ) , respectively , over a temperature range of 8 to 34 ° C .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19703465" target="_blank">19703465</a>]<br></li>
<li>Similarly tandem repeats of RcnR binding site is not an absolute rule , a single site being present in many cases ( not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21040754" target="_blank">21040754</a>]<br></li>
<li>One hundred micromolar concentration of site 1 altered sequence DNA duplexes was titrated into 10 μM RcnR tetramer in buffer P between 8 and 34 ° C .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19703465" target="_blank">19703465</a>]<br></li>
<li>These data are all consistent with one metal binding site per RcnR monomer , or four binding sites per tetramer .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18505253" target="_blank">18505253</a>]<br></li>
<li>A single RcnR tetramer binds to each operator site The footprinting results identified a likely RcnR binding site , but they did not reveal the stoichiometry of binding , nor did they precisely define the key bases necessary for the interaction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19703465" target="_blank">19703465</a>]<br></li>
<li>One RcnR tetramer binds to single-site duplexes by ITC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19703465" target="_blank">19703465</a>]<br></li>
<li>A single RcnR tetramer recognizes a site with a single G-tract and therefore both A -- B junctions ( one on each side ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19703465" target="_blank">19703465</a>]<br></li>
<li>RcnR binds the intergenic rcnRercnA region on the palindromic sequence TACT-G6-N-AGTA , which is repeated twice and interacts * Corresponding author at : MAP UMR 5240 , Bat Lwoff e 10 rue Dubois , F-69622 Villeurbanne Cedex , France .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21040754" target="_blank">21040754</a>]<br></li>
<li>The sequence surrounding the two start sites contains a palindromic sequence , TACT-N7-AGTA recognized by RcnR [ 7 ] , which is repeated twice , with the two directly repeated dyad symmetry sequences separated by 5 bp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21040754" target="_blank">21040754</a>]<br></li>
<li>We found that RcnR specifically recognizes a pair of TACT-G6-N-AGTA operator sites likely through minor groove contacts and nonspecifically binds to flanking DNA to promote DNA wrapping .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19703465" target="_blank">19703465</a>]<br></li>
<li>A model for the interaction between RcnR and PrcnA must account for both the specific binding to a pair of TACT-G6-N-AGTA motifs and the surprising nonspecific interactions with the flanking DNA that lead to DNA wrapping .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19703465" target="_blank">19703465</a>]<br></li>
<li>We found that an RcnR tetramer recognizes a TACT-G6-N-AGTA motif , of which there are two in the rcnA -- rcnR intergenic region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19703465" target="_blank">19703465</a>]<br></li>
<li>We show that the promoters of rcnR and rcnA are convergent and that apo-RcnR binds on symmetrically located sequences in this intergenic region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21040754" target="_blank">21040754</a>]<br></li>
<li>RcnR binds to an extended region of PrcnA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19703465" target="_blank">19703465</a>]<br></li>
<li>Interestingly , RcnR binds to this fragment with similar apparent affinity as the construct with both binding sites ( Kapp values of 5 ± 1 × 10 − 9 and 14 ± 2 × 10 − 9 M for single and dual sites , respectively ) , but there was no extended footprint beyond the 22-bp protected region expected for sit Fig. 5 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19703465" target="_blank">19703465</a>]<br></li>
<li>RcnR is an all-helical protein with poorly conserved loop sequences , which suggests that an α-helix of the protein binds in the DNA minor groove .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19703465" target="_blank">19703465</a>]<br></li>
<li>tably , the eukaryotic Tc3 transposase recognizes an A-DNA -- B-DNA junction that depends upon a G7 tract .25 Unlike RcnR , a helix -- turn -- helix motif binds in the DNA major groove near this junction , and the DNA bends sharply away from the protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19703465" target="_blank">19703465</a>]<br></li>
<li>The centers of protected regions were separated by 9 to 11 bases , demonstrating that RcnR binds to one face of the DNA duplex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19703465" target="_blank">19703465</a>]<br></li>
<li>The fact that RcnR binds downstream of both transcription start sites suggests that repression of transcription hampers RNA polymerase progression in both directions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21040754" target="_blank">21040754</a>]<br></li>
				</ul></div><br>
			<label class="collapse_TU" for="_6"><font size="5px">Transcription units regulated by RcnR</font></label>
				<input id="_6" type="checkbox">
				<div class="TU"><br><ul><li>Strains also contained either pNIK103 or pRcnR , which are the E. coli nikR and rcnR genes cloned into pET-22b respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16956381" target="_blank">16956381</a>]<br></li>
<li>However the topology of the RcnR binding box is diverse as genes under its control may also be transcribed in the same orientation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21040754" target="_blank">21040754</a>]<br></li>
<li>This positive iron-regulation was somewhat surprising , because Fur exerts its mode of regulation by repression Fur interacted with the rcnAp promoter in vivo Gene expression of rcnR responded to iron and RcnR may function as a negative regulator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17120142" target="_blank">17120142</a>]<br></li>
<li>This expression of rcnR from a high-copy number vector may have led to a constitutively repressed rcnA gene , making it probable that apo-RcnR ( rather than a hypothetical metallo-RcnR ) is the active regulator conﬁg-uration .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17120142" target="_blank">17120142</a>]<br></li>
<li>This shows that RcnR controls its own expression and suggests that supplementary regulatory events may control the expression of rcnR and therefore metal efﬂux .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21040754" target="_blank">21040754</a>]<br></li>
<li>The extension of the footprint into rcnR itself suggested that RcnR regulates its own expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19703465" target="_blank">19703465</a>]<br></li>
<li>A search of regions upstream of known E. coli genes did not identify any other identical sequences , suggesting that RcnR only regulates expression of rcnR and rcnA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19703465" target="_blank">19703465</a>]<br></li>
<li>However , RcnR alone was not suficient for this regulatory process because rcnA was still induced by metal cations in the absence of RcnR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17120142" target="_blank">17120142</a>]<br></li>
<li>In E. coli RcnR alone does not function as a metal-dependent repressor of rcnA expression because rcnA is still induced by metals in a DrcnR deletion strain ( Figure 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17120142" target="_blank">17120142</a>]<br></li>
<li>Conversely , the addition of Fe ( II ) did not alter rcnA expression in fur wild-type when RcnR was over-produced ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17120142" target="_blank">17120142</a>]<br></li>
<li>In addition to rcnA , RcnR controls expression of its own gene in response to Ni and Co , but the two genes are differentially expressed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21040754" target="_blank">21040754</a>]<br></li>
<li>It has been previously shown that the transcription of rcnA is induced by Ni or Co ( 44 ) via the metalloregulator RcnR ( 3 , 26 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21665978" target="_blank">21665978</a>]<br></li>
<li>rcnA expression is induced by Ni and Co ( 3 , 44 ) via the control of the metalloregulator RcnR ( 26 , 29 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21665978" target="_blank">21665978</a>]<br></li>
<li>The expression of rcnA is induced by Ni and Co via the metallo-regulator RcnR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21040754" target="_blank">21040754</a>]<br></li>
<li>yohN lies directly downstream of rcnA , and we show here that it is coexpressed with rcnA and that its expression is induced by Ni and Co via RcnR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21665978" target="_blank">21665978</a>]<br></li>
<li>In contrast , RcnR-dependent repression of rcnA is maintained until nikABCDE expression is fully repressed , then increasing Ni ( II ) releases RcnR from DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18505253" target="_blank">18505253</a>]<br></li>
<li>rcnA expression is controlled by a newly identiﬁed nickel-dependent regulator RcnR ( formerly YohL ) in E. coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16956381" target="_blank">16956381</a>]<br></li>
<li>In the present study we elucidated that expression of rcnA was inﬂu-enced by YohL , which we therefore denominated RcnR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17120142" target="_blank">17120142</a>]<br></li>
<li>Thus , Fur and RcnR regulation of rcnA expression is not interdependent .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17120142" target="_blank">17120142</a>]<br></li>
<li>This suggested that RcnR was involved in expression control of rcnA and that RcnR probably serves as a negative regulator for rcnA expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17120142" target="_blank">17120142</a>]<br></li>
<li>This indicated that RcnR was involved in negative regulation of rcnA by iron .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17120142" target="_blank">17120142</a>]<br></li>
<li>Previous studies in E. coli also have shown that rcnA may be regulated by Fur in response to iron and that the RcnR protein ( that represses rcnA expression ) is itself induced by iron via a Fur-independent pathway ( 29 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21097627" target="_blank">21097627</a>]<br></li>
<li>Likely , high iron conditions repressed rcnA expression through an increase of RcnR protein levels .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17120142" target="_blank">17120142</a>]<br></li>
				</div></ul></div><br>
	</div>
</body>
</html>