<!doctype html>
<style>
@charset "utf-8";
body {
	height: 100%;
	margin: 0;
	padding: 0;
}

.banner {
	position: fixed;
	top: 0;
	width: 100%;
	height: 65px;
	background: gray;
	line-height: 65px;
}
.spacer {
	height: 10px;
}

.main-content {
	padding: 5%;
}

.collapse_ACT {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-style: none solid none none;
	border-color: #B4045F;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_ACT + input{
	display: none;
}
.collapse_ACT + input + div{
	display: none;
}
.collapse_ACT + input:checked + div{
	display: block;
}
.ACT {
	text-align: justify;
	text-justify: inter-word;
}
.ACT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B4045F;
}
.ACT_collapse_similarity:hover{
	background: #e5d3dc;
}
.ACT_collapse_similarity + input {
	display: none;
}
.ACT_collapse_similarity + input + div{
	display: none;
}

.ACT_collapse_similarity + input:checked + div{
	display: block;
}
.ACT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.ACT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B4045F;
	padding-left: 5px;
}
.ACT_collapse_similarity:hover{
	background: #e5d3dc;
}
.ACT_collapse_similarity + input {
	display: none;
}
.ACT_collapse_similarity + input + div{
	display: none;
}
.ACT_collapse_similarity + input:checked + div{
	display: block;
}
.ACT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.ACT_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #B4045F;
}
.ACT_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_DOM {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-style: none solid none none;
	border-color: #B404AE;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_DOM + input{
	display: none;
}
.collapse_DOM + input + div{
	display: none;
}
.collapse_DOM + input:checked + div{
	display: block;
}
.DOM {
	text-align: justify;
	text-justify: inter-word;
}
.DOM_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B404AE;
	padding-left: 5px;
}
.DOM_collapse_similarity:hover{
	background: #e8d7e7;
}
.DOM_collapse_similarity + input {
	display: none;
}
.DOM_collapse_similarity + input + div{
	display: none;
}
.DOM_collapse_similarity + input:checked + div{
	display: block;
}
.DOM_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}.DOM_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #B404AE;
}
.DOM_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_EVO {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #7401DF;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_EVO + input{
	display: none;
}
.collapse_EVO + input + div{
	display: none;
}
.collapse_EVO + input:checked + div{
	display: block;
}
.EVO {
	text-align: justify;
	text-justify: inter-word;
}
.EVO_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #7401DF;
	padding-left: 5px;
}
.EVO_collapse_similarity:hover{
	background: #e2d7ed;
}
.EVO_collapse_similarity + input {
	display: none;
}
.EVO_collapse_similarity + input + div{
	display: none;
}
.EVO_collapse_similarity + input:checked + div{
	display: block;
}
.EVO_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.EVO_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #7401DF;
}
.EVO_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_RP {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #0174DF;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_RP + input{
	display: none;
}
.collapse_RP + input + div{
	display: none;
}
.collapse_RP + input:checked + div{
	display: block;
}
.RP {
	text-align: justify;
	text-justify: inter-word;
}
.RP_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #0174DF;
	padding-left: 5px;
}
.RP_collapse_similarity:hover{
	background: #c9d7e2;
}
.RP_collapse_similarity + input {
	display: none;
}
.RP_collapse_similarity + input + div{
	display: none;
}
.RP_collapse_similarity + input:checked + div{
	display: block;
}
.RP_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.RP_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #0174DF;
}
.RP_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_SIT {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #0489B1;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_SIT + input{
	display: none;
}
.collapse_SIT + input + div{
	display: none;
}
.collapse_SIT + input:checked + div{
	display: block;
}
.SIT {
	text-align: justify;
	text-justify: inter-word;
}
.SIT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #0489B1;
	padding-left: 5px;
}
.SIT_collapse_similarity:hover{
	background: #d0dfe2;
}
.SIT_collapse_similarity + input {
	display: none;
}
.SIT_collapse_similarity + input + div{
	display: none;
}
.SIT_collapse_similarity + input:checked + div{
	display: block;
}
.SIT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.SIT_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #0489B1;
}
.SIT_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_TU {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #04B486;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_TU + input{
	display: none;
}
.collapse_TU + input + div{
	display: none;
}
.collapse_TU + input:checked + div{
	display: block;
}
.TU {
	text-align: justify;
	text-justify: inter-word;
}
.TU_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #04B486;
	padding-left: 5px;
}
.TU_collapse_similarity:hover{
	background: #d0e2de;
}
.TU_collapse_similarity + input {
	display: none;
}
.TU_collapse_similarity + input + div{
	display: none;
}
.TU_collapse_similarity + input:checked + div{
	display: block;
}
.TU_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.TU_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #04B486;
}
.TU_similar > a {
	text-decoration: none;
	color: gray;
}

ul { 
	list-style-type: none;
	padding: 0;
	margin: 0;
}
ul li > a {
	text-decoration: none;
	color: gray;
}
ul li:hover, ul li:focus {
	background: #F0F0F0;
}

ul li:first-child span {
  border-left:none;  
}

</style>
<html>
<head>
<meta charset="utf-8">
<title>TF properties - RstA</title>
<meta name="theme-color" content="#ffffff">
</head>
<body>
	<div class="banner">
		<center><font color="white" size="16px" face="arial">RstA</font></center>
	</div>
	<div align="center" class="main-content">
		<div class="spacer">
	</div><br>			<label class="collapse_ACT" for="_1"><font size="5px">RstA transcriptional activity</font></label>
				<input id="_1" type="checkbox">
				<div class="ACT"><br><ul><li>With respect to the number of DNA-binding sites , Dan is different from the well-characterized global regulators so far examined by the genomic SELEX , including CRP , Cra , RstA , PdhR , RutR , TyrR , NemR , AllR and CitB ( 7 -- 10,12,13 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20156994" target="_blank">20156994</a>]<br></li>
<li>These observations altogether indicate that PhoQP is directly involved in positive regulation of curli formation on one hand , and on the other hand , we found that the csgD expression is negatively regulated by RstA , implying that the curli formation is under negative regulation by PhoQP-RstBA relay .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17468243" target="_blank">17468243</a>]<br></li>
<li>In the absence of RstA , however , translation of rpoS is enhanced ( Sugiura et al. , 2003 ) while overproduction of RstA induces degradation of RpoS in Salmonella ( Cabeza et al. , 2007 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20466769" target="_blank">20466769</a>]<br></li>
<li>Stable binding of RstA to these sequences might require an effector for RstA activation or its fully phosphorylated state .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17468243" target="_blank">17468243</a>]<br></li>
<li>Fluorescence-labeled DNA probes of asr ( A ) or csgD ( B ) were incubated at 37 ° C for 30 min with the indicated amount ( 0 , 10 , 20 , 40 , and 80 pmol ) of RstA in the absence or presence of acetyl phosphate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17468243" target="_blank">17468243</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_6">
Fluores-cence-labeled DNA probe of the asr promoter fragment was incubated with increasing concentrations of the puriﬁed RstA ( lane 1 , 0 pmol ; lane 2 , 10 pmol ; lane 3 , 20 pmol ; lane 4 , 40 pmol ; lane 5 , 80 pmol ) and subjected to DNase I footprinting assays as described in Materials and Methods .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17468243" target="_blank">17468243</a>]<br></label>
<input id="act_similarity_6", type="checkbox">
<div class="ACT_similar">
<li>Fluores-cence-labeled DNA probe of the csgD promoter fragment was incubated with increasing concentrations of the puriﬁed RstA ( lane 1 , 0 pmol ; lane 2 , 10 pmol ; lane 3 , 20 pmol ; lane 4 , 40 pmol ; lane 5 , 80 pmol ) and subjected to DNase I footprinting assays as described in Materials and Methods .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17468243" target="_blank">17468243</a>]<br></li></div>
<li>Thus , the DNA-binding activity of OmpR , RstA and CpxR was examined in the presence of acetyl phosphate for phosphorylation in vitro ( note that phosphorylated forms are shown as OmpRp , RstAp and CpxRp in the respective figures ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20466769" target="_blank">20466769</a>]<br></li>
<li>( b ) The gel shift assay was carried out using RstA and IHF with FITC-labelled CD3 probe DNA in the presence of acetyl phosphate ( 10 mM ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20466769" target="_blank">20466769</a>]<br></li>
<li>When the gel shift assay of RstA was carried out in the presence of acetyl phosphate , however , little enhancement of its afﬁnity to the target DNA was observed ( Fig. 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17468243" target="_blank">17468243</a>]<br></li>
<li>CpxR , CsgD , CRP , RstA , OmpR and Cra-binding sites are indicated csgB coding sequence TTGTTTTTCATGTTGTCACCCTGGACCTGGTCGTACATTTAAGAAATTAAATCATTTCAACTTGGTTGT TAACGCAACCTGTATTTTGTTAACGCTGCGTTACGATGGAAAGTATGTCTGCGGAAATATTTTTAATAA csgBp CTCACCCGCGCGTTGTATTTTTCTTTTTTAGTTCATACCAAAGTATTAAAACCCACGTAAATACGCTGT TATCTACGCAAAAAAATATTTTGTTTTCTTTTAAATCTCCGTTTTCCGCTATCAAAAAGCACCAGACAG TCATTCTTCTTGCCCGTCGCTGATTGCTGCGATATGTCTTGCGCACAAGCCGTGACAAACCTCGTCTTT TTGCAGTAAAAAATTGTCCACGGAGGTGTGGAGAAAAAACAAGAACGTTTTACATGACGAAAGGACTAC ACCGAAATATTTTTTATATGCATTATTAGTAAGTTATCACCATTTGTATGATTTTTTAAAATTGTGCAA TAAAAACCAAATGTACAACTTTTCTATCATTTCTAAACTTAATAAAACCTTAAGGTTAACATTTTAATA csgDp1 TAACGAGTTACATTTAGTTACATGTTTAACACTTGATTTAAGATTTGTAATGGCTAGATTGAAATCAGA csgDp2 TGTAATCCATTAGTTTTATATTTTACCCATTTAGGGCTGATTTATTACTACACACAGCAGTGCAACATC TGTCAGTACTTCTGGTGCTTCTATTTTAGAGGCAGCTGTCAGGTGTGCGATCAATAAAAAAAGCGGGGT csgD coding sequence TTCATCATGTTTAATGAAGTCCATAGTAT RstA-P CpxR-P CpxR-P CpxR-P csgDp1 OmpR csgDp2 Cra Cra CRP-AMP CRP-AMP CpxR-P CpxR-P Cra CpxR-P Cra CsgD csgBp J. Altenbuchner and S. E. Lindow , respectively , while bacteriophage P1vir was a kind gift from S. K. Apte .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21559929" target="_blank">21559929</a>]<br></li>
<li>Among the five factors examined , the three response-regulators , OmpR , RstA and CpxR , are organized in two-component systems , and are converted to active forms only when phosphorylated by the respective sensor kinases , EnvZ ( Aiba et al. , 1989 ; Yamamoto et al. , 2005 ) , RstB ( Minagawa et al. , 2003 ; Yamamoto et al. , 2005 ) and CpxA ( Raivio & Silhavy , 1997 ; Yamamoto et al. , 2005 ) , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20466769" target="_blank">20466769</a>]<br></li>
<li>In the rstAB mutant grown at pH 7.0 , the level of csgD mRNA increased slightly ( Fig. 2b , lane 5 ) while overproduction of RstA resulted in significant reduction of csgD mRNA ( Ogasawara et al. , 2007a ) , these results together indicating a negative role of RstA in csgD transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20466769" target="_blank">20466769</a>]<br></li>
<li>In the presence of MlrA overexpression , an increase of csgD mRNA was observed in the mutants lacking Crl ( Fig. 4 , lanes 5 and 6 ) , H-NS ( Fig. 4 , lanes 7 and 8 ) , RstA ( Fig. 4 , lanes 11 and 12 ) , CpxR ( Fig. 4 , lanes 13 and 14 ) and RcsB ( Fig. 4 , lanes 15 and 16 ) , indicating that MlrA functions independent of these regulators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20874755" target="_blank">20874755</a>]<br></li>
<li>In contrast , transcription of csgD was repressed by overexpression of RstA , indicating that RstA is a repressor for csgD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17468243" target="_blank">17468243</a>]<br></li>
<li>When CD6 was used as a probe , both RstA and IHF bound to the probe ( Fig. 10a ) as expected from the map of binding sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20466769" target="_blank">20466769</a>]<br></li>
<li>The inﬂuence of RstA on the csgD promoter was then examined by direct measurement of csgD mRNA after increasing expression of RstA in the rstA-null mutant .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17468243" target="_blank">17468243</a>]<br></li>
<li>Since OmpR and RstA must compete with each other in binding to the csgD promoter , the repression of csgD by RstA must be due to competitive inhibition of OmpR binding to the csgD promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17468243" target="_blank">17468243</a>]<br></li>
<li>Cooperative binding of the two activators RstA and IHF to the csgD promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20466769" target="_blank">20466769</a>]<br></li>
<li>( c ) Model of cooperation between RstA and IHF in binding to the csgD promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20466769" target="_blank">20466769</a>]<br></li>
<li>d. Also regulated by Crp , IHF , Lrp , CpxR , RstA , Fur as well as sE ( Rhodius et al. , 2006 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18710431" target="_blank">18710431</a>]<br></li>
<li>The amounts of transcription factors added were : CpxR and IHF , 0 , 0.5 , 1.0 , 1.5 , 2.0 , 2.5 , 3.0 , 3.5 , 4.0 , 4.5 , 5.0 , 5.5 and 6.0 pmol for lanes 1 -- 13 ; H-NS , 0 , 2.5 , 5.0 , 7.5 , 10 , 12.5 , 15 , 17.5 , 20 , 22.5 , 25 , 27.5 and 30 pmol for lanes 1 -- 13 ; OmpR and RstA , 0 , 5 , 10 , 15 , 20 , 25 , 30 , 35 , 40 , 45 , 50 , 55 and 60 pmol for lanes 1 -- 13 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20466769" target="_blank">20466769</a>]<br></li>
<li>The amounts of proteins used were ( from lane 2 to 5 ) : 1.25 , 2.5 , 5 and 10 pmol for IHF ; 2.5 , 5 , 10 and 20 pmol for CpxR ; 5 , 10 , 20 and 40 pmol for OmpR and H-NS in the absence ( H-NS and IHF ) or presence ( CpxR , OmpR and RstA ) of acetyl phosphate ( 10 mM ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20466769" target="_blank">20466769</a>]<br></li>
<li>Briefly , E. coli whole-cell lysates were prepared by sonication after lysozyme treatment , and treated with antibodies , which were prepared in rabbits against purified transcription factors , CsgD , RstA , OmpR , CpxR , IHF and H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20466769" target="_blank">20466769</a>]<br></li>
<li>At the times indicated , whole-cell lysates were prepared and subjected to quantitative Western blot analysis using antibodies against IHF , OmpR , H-NS , CpxR and RstA ( upper panels ) , and anti-RpoA ( bottom panel ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20466769" target="_blank">20466769</a>]<br></li>
<li>In growing cells , the intracellular concentrations of two nucleoid proteins , IHF and H-NS , are similar and higher than those of the specific regulators , RstA , OmpR and CpxR ( Fig. 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20466769" target="_blank">20466769</a>]<br></li>
<li>Previously , we classiﬁed three factors ( OmpR , RstA and IHF ) as activators and two factors ( CpxR and H-NS ) as repressors , and found novel modes of their interplay .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20874755" target="_blank">20874755</a>]<br></li>
<li>Based on the results described herein , we propose a novel regulation model of cooperation between two positive factors ( OmpR and IHF ; and RstA and IHF ) and also between two negative factors ( CpxR and H-NS ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20466769" target="_blank">20466769</a>]<br></li>
<li>In addition to the competition between positive and negative factors , we found unexpected cooperation between two negative factors , CpxR and H-NS , and between two pairs of positive factors , OmpR and IHF , and RstA and IHF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20466769" target="_blank">20466769</a>]<br></li>
<li>Interplay between MlrA , OmpR and IHF Three positive factors , IHF , OmpR and RstA , can associate simultaneously within the promoter-proximal hot-spot II region of transcription factor binding ( Fig. 6 ) , and cooperate with each other for activation of the csgD promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20874755" target="_blank">20874755</a>]<br></li>
<li>Five factors ( OmpR , RstA , IHF , CpxR and H-NS ) bind to the promoter-proximal hot-spot II , while four factors ( IHF , MlrA , CpxR and H-NS ) bind to the promoter-distal hot-spot I. P1 and P2 represent the transcription initiation site P1 ( Hammar et al. , 1995 ) and P2 promoter ( Ogasawara et al. , 2007a ) , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20874755" target="_blank">20874755</a>]<br></li>
<li>The apparent affinity for the csgD promoter was found to be high for IHF and CpxR , followed by OmpR , H-NS and RstA ( Fig. 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20466769" target="_blank">20466769</a>]<br></li>
<li>CRP , CpxR , CsgD , H-NS , IHF , MlrA , OmpR , and RstA on the csgD promoter ( Ogasawara et al. 2010a , b ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23233451" target="_blank">23233451</a>]<br></li>
<li>The binding sites of MlrA , OmpR , RstA , CpxR , IHF and H-NS were determined by DNase-1 footprinting assays ( Fig. 2 ; and Ogasawara et al. , 2010 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20874755" target="_blank">20874755</a>]<br></li>
<li>All five transcription factors , CpxR , OmpR , RstA , IHF and H-NS , formed binary DNA complexes in vitro in the absence of other factors ( Fig. 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20466769" target="_blank">20466769</a>]<br></li>
<li>In this report , we describe the regulatory modes of five transcription factors , CpxR , OmpR , RstA , IHF and H-NS , and their interplays in binding in vitro to the csgD promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20466769" target="_blank">20466769</a>]<br></li>
<li>DNA curvature induced by MlrA should inﬂuence not only the activity of neighbouring DNA such as binding afﬁnity to positive factors , IHF , OmpR and RstA , and/or to negative fractors , H-NS and CpxR , but also the mode of the molecular interplay between csgD promoter-bound transcription factors .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20874755" target="_blank">20874755</a>]<br></li>
<li>In the mutant lacking RstA , the acid-induced activation of the asr promoter was completely abolished ( Fig. 6A , compare lanes 2 and 8 ) , indicating that RstA is indeed involved in the activation of the asr promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17468243" target="_blank">17468243</a>]<br></li>
<li>Again , virtually no asr activity was detected in the rstA knockout mutant ( Fig. 6B , lanes 7 and 8 ) , supporting the pre diction that RstA is needed for transcription activation of the asr gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17468243" target="_blank">17468243</a>]<br></li>
<li>To conﬁrm the involvement of both PhoP and RstA in acid-induced transcription of the asr gene , the promoter assay was performed in the presence of low ( 0.24 mM ) or high ( 30 mM ) concentrations of Mg2 and at both pH 7.0 ( Fig. 6D ) and pH 4.5 ( Fig. 6E ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17468243" target="_blank">17468243</a>]<br></li>
<li>On the other hand , the activity of lacUV5 promoter used as a reference stayed constant irrespective of the presence or absence of RstA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17468243" target="_blank">17468243</a>]<br></li>
<li>In vivo transcription of the asr promoter was observed only in the presence of RstA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17468243" target="_blank">17468243</a>]<br></li>
<li>The activity of the asr promoter was low in the absence of RstA , but in the presence of added RstA , it increased in a dose-dependent manner .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17468243" target="_blank">17468243</a>]<br></li>
<li>Several lines of experimental evidence indicated that the expression of asr is activated by RstA in response to low Mg2 availability in the environment , while that of csgD is repressed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17468243" target="_blank">17468243</a>]<br></li>
<li>An acid-stress response-regulator , RstA , activates csgD only under acidic conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20466769" target="_blank">20466769</a>]<br></li>
<li>Previously we noticed that RstA is involved in asr activation for the acid stress response ( Ogasawara et al. , 2007a ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20466769" target="_blank">20466769</a>]<br></li>
<li>This ﬁnding is in agreement with our previous observation that the level of transphosphorylation in vitro of RstA by phosphorylated RstB is lower than that for other RRs of E. coli TCSs or that the dephosphorylation activity of phosphorylated RstA is higher than that for other RRs of E. coli TCSs ( 37 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17468243" target="_blank">17468243</a>]<br></li>
<li>The phosphorylation level was from 1 % to 10 % for TorR , PhoB , CusR , BasR , RstA , UvrY , ArcA , BaeR , NtrC , KdpE , UhpA , and PhoP ( in decreasing order ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15522865" target="_blank">15522865</a>]<br></li>
<li>The trans-phos-phorylation was experimentally demonstrated in vitro , for the first time , for 9 pairs , CusS-CusR , RstB-RstA , YedV-YedW , BaeS-BaeR , YehU-YehT , YfhK-YfhA , HydH-HydG , BasSBasR , and CreC-CreB ( see Fig. 3 ) , supporting the predicted HK-RR pairs .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15522865" target="_blank">15522865</a>]<br></li>
<li>A total of 30 sensor HKs , including as yet uncharacterized puta - tive HKs ( BaeS , BasS , CreC , CusS , HydH , RstB , YedV , and YfhK ) , and a total of 34 RRs , including putative RRs ( BaeR , BasR , CreB , CusR , HydG , RstA , YedW , YfhA , YgeK , and YhjB ) , have been suggested to exist in Esche - richia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15522865" target="_blank">15522865</a>]<br></li>
<li>Trans-phosphorylation of presumed cog - nate RRs by HKs was detected , for the first time , for eight pairs , BaeS-BaeR , BasS-BasR , CreC-CreB , CusS-CusR , HydH-HydG , RstB-RstA , YedV-YedW , and YfhK-YfhA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15522865" target="_blank">15522865</a>]<br></li>
				</div></ul><br>
			<label class="collapse_DOM" for="_2"><font size="5px">RstA protein domains</font></label>
				<input id="_2" type="checkbox">
				<div class="DOM"><br><ul><li>Based on the site of RstA binding , it can be predicted that it serves as a class I transcription factor , which makes contact with the C-terminal domain for supporting RNA polymerase binding to the asr promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17468243" target="_blank">17468243</a>]<br></li>
				</div></ul><br>
			<label class="collapse_RP" for="_4"><font size="5px">Processes regulated by RstA</font></label>
				<input id="_4" type="checkbox">
				<div class="RP"><br><ul><li>We also observed downregulation of torS ( encodes an HK ) in the DbasRS and DntrBC mutants , downregulation of evgS ( encodes an HK ) in the DdcuSR mutant , downregulation of the rstAB ( encodes the RstB/RstA TCS ) in the DphoPQ mutant and upregulation of atoS ( encodes an HK ) in the DyfhA ( RR ) mutant .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12366850" target="_blank">12366850</a>]<br></li>
<li>Doubling regulated regulated of altered time Identiﬁed function Class system ( HK/RR ) Relevant functions genes genes genes ( min ) in this experiment Strain name Mutation ntrol o o C N 1 4 5 27 BW25113 WT or little signiﬁcant alteration CreC/CreB 1 1 2 Catabolite repression ( Wanner , 1996 ) Drug resistance ( Kato et al. , 2000 ) BW26983 DE ( creABCD ) 154 29 2 3 5 EvgS/EvgA BW27869 DE ( evgAS ) 1291 29 2 6 8 YgiY/YgiX NarX/NarL BW27551 BW27864 DE ( ygiXY ) 1302 DE ( narXL ) 1316 27 29 5 4 9 Nitrate regulation ( Stewart and Rabin , 1995 ) Hexose phosphate uptake ( Kadner , 1995 ) Phosphate regulation ( Wanner , 1995 ) Copper response ( Munson et al. , 2000 ) Nitrate regulation ( Stewart and Rabin ,1995 ) UhpB/UhpA 6 3 9 BW27871 DE ( uhpAB ) 1322 28 PhoR/PhoB 1 8 9 BW24476 DE ( phoBR ) 580 28 CusS/CusR 2 7 BW28077 DE ( cusRS ) 1204 27 9 / NarP 9 BW27873 DnarP1312 33 12 3 YedV/YedW YfhK / YehU/YehT KdpD/KdpE DE ( yedWV ) 1298 DyfhK1328 DE ( yehUT ) 1324 DE ( kdpABCDE ) 1224 26 28 27 27 13 15 20 20 6 7 7 8 BW BW BW BW 27550 27872 27877 27564 13 10 7 10 Potassium transport ( Walderhaug et al. , 1992 ) Nitrogen regulation ( Porter et al. , 1995 ) NtrB/NtrC BW27880 DE ( ntrBC ) 1318 31 20 8 12 Signiﬁcant alteration BaeS/BaeR HydH/HydG ( Nagasawa et al. , 1993 ) Response to zinc and lead ( Leonhartsberger et al. , 2001 ) Hydrogen peroxide sensitivity ( Pernestig et al. , 2001 ) 14 11 BW27553 BW27867 DE ( baeSR ) 579 DE ( hydHG ) 1336 8 22 24 29 31 13 2 BarA / BW29434 DbarA1358 24 26 30 7 / FimZ BW28078 BW27558 Enterochelin DﬁmZ1214 22 27 29 31 28 27 4 PhoQ/PhoP Response to magnesium ( Kato et al. , 1999 ) ( Nagasawa et al. , 1993 ) Response to cell envelope stress ( De Wulf et al. , 1999 ) DE ( phoPQ ) 1244 20 16 12 16 BasS/BasR CpxA/CpxR BW27848 BW27559 DE ( basRS ) 1287 DE ( cpxRA ) 623 32 32 29 29 Enterochelin 12 20 22 17 RstB/RstA YojN ( Hpt ) BW27552 BW27866 DE ( rstAB ) 1278 DyojN1332 34 37 26 29 Enterochelin Capsular synthesis ( Takeda et al. , 2001 ) 14 Control of TMAO respiratory system ( Simon et al. , 1994 ) 38 24 TorS/TorR BW26423 26 DE ( torSTRCAD ) 518 a .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12366850" target="_blank">12366850</a>]<br></li>
<li>Furthermore , a number of transcription factors are involved in the regulation of the csgD promoter , including CpxR ( Jubelin et al. , 2005 ) , Crl ( Bougdour et al. , 2004 ) , H-NS ( Gerstel et al. , 2003 ) , IHF ( Gerstel et al. , 2003 , 2006 ) , OmpR ( Vidal et al. , 1998 ; Gerstel et al. , 2003 , 2006 ) , RstA ( Ogasawara et al. , 2007a ) , MlrA ( Brown et al. , 2001 ) , RcsB ( Ferrieres & Clarke , 2003 ; Vianney et al. , 2005 ) and CRP ( Zheng et al. , 2004 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20874755" target="_blank">20874755</a>]<br></li>
<li>An acid-stress response regulator , RstA , activates csgD only under acidic conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20466769" target="_blank">20466769</a>]<br></li>
<li>Previously we noticed that RstA is involved in asr activation for the acid stress response ( Ogasawara et al. , 2007a ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20466769" target="_blank">20466769</a>]<br></li>
<li>These observations altogether raise a possibility that RstA is a typical bifunctional regulator , acting as both an activator and a repressor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17468243" target="_blank">17468243</a>]<br></li>
<li>RstA is the response regulator of the RstAB two-component system that controls the expression of many genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19101473" target="_blank">19101473</a>]<br></li>
<li>The findings reported in this article provide further support to the notion that the PhoP PhoQ two-component system is a central regulator that controls the expression and or activity of other regulatory proteins and systems when cells experience low Mg2 , including the two-component system RstA RstB of E. coli , ( 47 ) , the SpiR SsrB two-component system ( 48 ) , and Salmonella the Salmonella transcription factor SlyA ( 49 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16938894" target="_blank">16938894</a>]<br></li>
				</div></ul><br>
			<label class="collapse_SIT" for="_5"><font size="5px">RstA binding site</font></label>
				<input id="_5" type="checkbox">
				<div class="SIT"><br><ul><li>After searching for this RstA box-like sequence along the entire E. coli genome , we found nine other possible targets of RstA , including the nohB , leuS , modA , ybiL , ompF , narG , narU , sdaA , and hdeA genes , each carrying the RstA box sequence within 500 bp upstream of the respective initiation codon ( Fig. 5B ) , among which the expression level of narG has been reported to decrease in an rstAB-deﬁcient strain ( 22 ) , while the overexpression of RstA represses ompF transcription ( H. Aiba , personal communication ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17468243" target="_blank">17468243</a>]<br></li>
<li>CpxR , CsgD , CRP , RstA , OmpR and Cra-binding sites are indicated csgB coding sequence TTGTTTTTCATGTTGTCACCCTGGACCTGGTCGTACATTTAAGAAATTAAATCATTTCAACTTGGTTGT TAACGCAACCTGTATTTTGTTAACGCTGCGTTACGATGGAAAGTATGTCTGCGGAAATATTTTTAATAA csgBp CTCACCCGCGCGTTGTATTTTTCTTTTTTAGTTCATACCAAAGTATTAAAACCCACGTAAATACGCTGT TATCTACGCAAAAAAATATTTTGTTTTCTTTTAAATCTCCGTTTTCCGCTATCAAAAAGCACCAGACAG TCATTCTTCTTGCCCGTCGCTGATTGCTGCGATATGTCTTGCGCACAAGCCGTGACAAACCTCGTCTTT TTGCAGTAAAAAATTGTCCACGGAGGTGTGGAGAAAAAACAAGAACGTTTTACATGACGAAAGGACTAC ACCGAAATATTTTTTATATGCATTATTAGTAAGTTATCACCATTTGTATGATTTTTTAAAATTGTGCAA TAAAAACCAAATGTACAACTTTTCTATCATTTCTAAACTTAATAAAACCTTAAGGTTAACATTTTAATA csgDp1 TAACGAGTTACATTTAGTTACATGTTTAACACTTGATTTAAGATTTGTAATGGCTAGATTGAAATCAGA csgDp2 TGTAATCCATTAGTTTTATATTTTACCCATTTAGGGCTGATTTATTACTACACACAGCAGTGCAACATC TGTCAGTACTTCTGGTGCTTCTATTTTAGAGGCAGCTGTCAGGTGTGCGATCAATAAAAAAAGCGGGGT csgD coding sequence TTCATCATGTTTAATGAAGTCCATAGTAT RstA-P CpxR-P CpxR-P CpxR-P csgDp1 OmpR csgDp2 Cra Cra CRP-AMP CRP-AMP CpxR-P CpxR-P Cra CpxR-P Cra CsgD csgBp J. Altenbuchner and S. E. Lindow , respectively , while bacteriophage P1vir was a kind gift from S. K. Apte .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21559929" target="_blank">21559929</a>]<br></li>
<li>Among the regulation targets of RstA identified by genomic SELEX , we found the consensus sequence of TACATNTNGTTACA as a common sequence recognized by RstA ( Ishihama , 2010 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20466769" target="_blank">20466769</a>]<br></li>
<li>Once the consensus recognition sequence is predicted after SELEX screening , one can extend the search of other RstA targets against the whole genome sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17468243" target="_blank">17468243</a>]<br></li>
<li>This RstA-box-like sequence ( 256 ) TACATTAGTTACA ( 244 ) is pres-ent within the region protected by RstA ( 261 to 243 ) ( Fig. 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20466769" target="_blank">20466769</a>]<br></li>
				<label class="SIT_collapse_similarity" for="sit_similarity_6">
The bold line under the sequence indicates the RstA-binding sequence as detected by the protection pattern from DNase treatment , while two arrows between the DNA strands indicate the conserved direct repeat of the RstA box .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17468243" target="_blank">17468243</a>]<br></label>
<input id="sit_similarity_6", type="checkbox">
<div class="SIT_similar">
<li>The bold line under the sequence indicates the RstA-binding sequence as detected by protection pattern from DNase treatment while two arrows between the DNA strands indicate the conserved direct repeat of the RstA box .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17468243" target="_blank">17468243</a>]<br></li></div>
<li>Using a DNase I footprinting assay , we determined the RstA-binding sites ( RstA boxes ) with the consensus sequence TACATNTNGTTACA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17468243" target="_blank">17468243</a>]<br></li>
<li>Using the newly identiﬁed RstA box sequence , we could identify nine other candidates of the RstA target .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17468243" target="_blank">17468243</a>]<br></li>
<li>The PhoB box is located downstream of the RstA box and overlaps with the promoter 35 sequence ( Fig. 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17468243" target="_blank">17468243</a>]<br></li>
<li>In the two RstA-binding sequences on the asr and csgD promoters , a 14-bp-long TACATNTNGTTACA sequence ( Fig. 5A ) , which includes a direct TACA repeat , exists ( this sequence is hereafter referred to as the RstA box ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17468243" target="_blank">17468243</a>]<br></li>
<li>The RstA-binding consensus sequence , the RstA box , was identiﬁed on both the asr and csgD promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17468243" target="_blank">17468243</a>]<br></li>
<li>On the asr promoter fragment , RstA was found to bind to the 23-bp-long sequence between positions 77 and 55 upstream of the asr transcriptional start position ( or between positions 126 and 104 from the translation start site ) ( Fig. 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17468243" target="_blank">17468243</a>]<br></li>
<li>On the other hand , on the csgD promoter fragment , RstA bound to a 19-bp sequence between positions 71 and 53 upstream of the transcriptional start position of csgD promoter P2 ( or between positions 61 and 43 of promoter P1 ) ( Fig. 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17468243" target="_blank">17468243</a>]<br></li>
<li>Here we identified that five regulatory proteins ( CpxR , OmpR , RstA , IHF and H-NS ) bind within a narrow region of about 229 bp along the csgD promoter ( see Fig. 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20466769" target="_blank">20466769</a>]<br></li>
<li>A single unique protection sequence was identified for OmpR and RstA , while two sites were identified with IHF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20466769" target="_blank">20466769</a>]<br></li>
<li>The binding sites of IHF , OmpR , RstA and CpxR are indicated along the csgD promoter sequence while the H-NS-binding sequence is shown by a dotted line between the duplex sequences .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20466769" target="_blank">20466769</a>]<br></li>
<li>Both RstA and IHF are able to bind to the IHF-2 site simultaneously .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20466769" target="_blank">20466769</a>]<br></li>
<li>Five factors ( OmpR , RstA , IHF , CpxR and H-NS ) bind to the promoter-proximal hot-spot II , while four factors ( IHF , MlrA , CpxR and H-NS ) bind to the promoter-distal hot-spot I. P1 and P2 represent the transcription initiation site P1 ( Hammar et al. , 1995 ) and P2 promoter ( Ogasawara et al. , 2007a ) , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20874755" target="_blank">20874755</a>]<br></li>
<li>For construction of plasmids for His-tagged transcription factors , pCpxR , pH-NS , pOmpR and pRstA , DNA fragments corresponding to the coding sequences of the respective transcription factors were amplified by PCR using E. coli W3110 genomic DNA as a template and pairs of primers ( Table 1 ) designed to hybridize upstream or downstream of each coding sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20466769" target="_blank">20466769</a>]<br></li>
<li>For the construction of IPTG ( isopropyl - - D-thiogalac-topyranoside ) - inducible RstA expression plasmid pKH13-1 , a DNA fragment corresponding to the RstA-coding region was ampliﬁed by PCR using E. coli W3110 genome DNA as a template and a pair of primers , RSTAF ( 5 - CGTTT TTATAGGATCCGTGAATGTTATGAA-3 ) and RSTAR ( 5 - CAGTTTTTTC GCGGCCGCTTCCCATGCATG-3 ) , and , after digestion with BamHI and NotI , cloned into pET21a ( ) ( Novagen ) at the corresponding sites ( 38 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17468243" target="_blank">17468243</a>]<br></li>
<li>For construction of an arabinose-inducible RstA expression plasmid , a DNA fragment ( 830 bp ) containing the rstA coding region was prepared by PCR using E. coli KP7600 genome DNA as a template , along with a pair of primers , rstA-BAD-EcoRI-F ( 5 - TAGAAAGAATTCATAGTAAGTAAAAACAGC-3 ) and rstA-BAD-XbaI-R ( 5 - CAAGGATCTAGACAAACAATAACAGGTAAA-3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17468243" target="_blank">17468243</a>]<br></li>
<li>For the genomic SELEX screening of RstA-binding sequences , 5 pmol of DNA fragments and 20 pmol of His-tagged RstA were mixed in a binding buffer ( 10 mM Tris-HCl , pH 7.8 at 4 ° C , 3 mM Mg acetate , 150 mM NaCl , 1.25 g/ml bovine serum albumin [ BSA ] ) and incubated for 30 min at 37 ° C .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17468243" target="_blank">17468243</a>]<br></li>
<li>For isolation of DNA sequences that are recognized by E. coli RstA protein , we employed the genomic SELEX system ( 27 ) , in which a complete library of genome DNA fragments ( 100 to 300 bp in length ) is used as DNA substrates in place of a mixture of synthetic oligonucleotides with all possible sequences ( 29 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17468243" target="_blank">17468243</a>]<br></li>
				</ul></div><br>
			<label class="collapse_TU" for="_6"><font size="5px">Transcription units regulated by RstA</font></label>
				<input id="_6" type="checkbox">
				<div class="TU"><br><ul><li>The location of the RstA-binding site and the direction of transcription of two neighboring genes suggested the possibility that RstA regulates transcription of the asr gene ( Table 1 and Fig. 1 ) , which is known to be highly induced upon exposure to acidic conditions ( pH 5.0 ) ( 25 , 33 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17468243" target="_blank">17468243</a>]<br></li>
<li>This RstA site is located upstream of both of the neighboring genes , csgD ( regulator of the csgBA operon ) and csgB ( minor curlin subunit ) ( Table 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17468243" target="_blank">17468243</a>]<br></li>
<li>From the location of this RstA site , as determined by the DNase I footprinting assay ( see below ) , RstA bound on this site was considered to regulate the csgD gene encoding a FixJ family transcription factor , which positively regulates the csgBA operon encoding the minor and major subunits of curli ( 11 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17468243" target="_blank">17468243</a>]<br></li>
<li>These observations altogether indicate that PhoQP is directly involved in positive regulation of curli formation on one hand , and on the other hand , we found that the csgD expression is negatively regulated by RstA , implying that the curli formation is under negative regulation by PhoQP-RstBA relay .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17468243" target="_blank">17468243</a>]<br></li>
<li>DNA curvature induced by MlrA should inﬂuence not only the activity of neighbouring DNA such as binding afﬁnity to positive factors , IHF , OmpR and RstA , and/or to negative fractors , H-NS and CpxR , but also the mode of the molecular interplay between csgD promoter-bound transcription factors .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20874755" target="_blank">20874755</a>]<br></li>
<li>In contrast , transcription of csgD was repressed by overexpression of RstA , indicating that RstA is a repressor for csgD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17468243" target="_blank">17468243</a>]<br></li>
<li>Here we found that transcription of csgD is repressed by overexpression of RstA ( Fig. 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17468243" target="_blank">17468243</a>]<br></li>
<li>RstA is the response regulator of the RstAB two-component system that controls the expression of many genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19101473" target="_blank">19101473</a>]<br></li>
<li>E. coli rstA mutant JD22755 ( pBAD18 , control plasmid ) ( lanes 1 and 3 ) or JD22755 ( pBADrstA , RstA expression plasmid ) ( lanes 2 and 4 ) were grown in LB medium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17468243" target="_blank">17468243</a>]<br></li>
				</div></ul></div><br>
	</div>
</body>
</html>