<!doctype html>
<style>
@charset "utf-8";
body {
	height: 100%;
	margin: 0;
	padding: 0;
}

.banner {
	position: fixed;
	top: 0;
	width: 100%;
	height: 65px;
	background: gray;
	line-height: 65px;
}
.spacer {
	height: 10px;
}

.main-content {
	padding: 5%;
}

.collapse_ACT {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-style: none solid none none;
	border-color: #B4045F;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_ACT + input{
	display: none;
}
.collapse_ACT + input + div{
	display: none;
}
.collapse_ACT + input:checked + div{
	display: block;
}
.ACT {
	text-align: justify;
	text-justify: inter-word;
}
.ACT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B4045F;
}
.ACT_collapse_similarity:hover{
	background: #e5d3dc;
}
.ACT_collapse_similarity + input {
	display: none;
}
.ACT_collapse_similarity + input + div{
	display: none;
}

.ACT_collapse_similarity + input:checked + div{
	display: block;
}
.ACT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.ACT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B4045F;
	padding-left: 5px;
}
.ACT_collapse_similarity:hover{
	background: #e5d3dc;
}
.ACT_collapse_similarity + input {
	display: none;
}
.ACT_collapse_similarity + input + div{
	display: none;
}
.ACT_collapse_similarity + input:checked + div{
	display: block;
}
.ACT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.ACT_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #B4045F;
}
.ACT_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_DOM {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-style: none solid none none;
	border-color: #B404AE;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_DOM + input{
	display: none;
}
.collapse_DOM + input + div{
	display: none;
}
.collapse_DOM + input:checked + div{
	display: block;
}
.DOM {
	text-align: justify;
	text-justify: inter-word;
}
.DOM_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B404AE;
	padding-left: 5px;
}
.DOM_collapse_similarity:hover{
	background: #e8d7e7;
}
.DOM_collapse_similarity + input {
	display: none;
}
.DOM_collapse_similarity + input + div{
	display: none;
}
.DOM_collapse_similarity + input:checked + div{
	display: block;
}
.DOM_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}.DOM_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #B404AE;
}
.DOM_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_EVO {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #7401DF;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_EVO + input{
	display: none;
}
.collapse_EVO + input + div{
	display: none;
}
.collapse_EVO + input:checked + div{
	display: block;
}
.EVO {
	text-align: justify;
	text-justify: inter-word;
}
.EVO_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #7401DF;
	padding-left: 5px;
}
.EVO_collapse_similarity:hover{
	background: #e2d7ed;
}
.EVO_collapse_similarity + input {
	display: none;
}
.EVO_collapse_similarity + input + div{
	display: none;
}
.EVO_collapse_similarity + input:checked + div{
	display: block;
}
.EVO_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.EVO_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #7401DF;
}
.EVO_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_RP {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #0174DF;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_RP + input{
	display: none;
}
.collapse_RP + input + div{
	display: none;
}
.collapse_RP + input:checked + div{
	display: block;
}
.RP {
	text-align: justify;
	text-justify: inter-word;
}
.RP_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #0174DF;
	padding-left: 5px;
}
.RP_collapse_similarity:hover{
	background: #c9d7e2;
}
.RP_collapse_similarity + input {
	display: none;
}
.RP_collapse_similarity + input + div{
	display: none;
}
.RP_collapse_similarity + input:checked + div{
	display: block;
}
.RP_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.RP_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #0174DF;
}
.RP_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_SIT {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #0489B1;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_SIT + input{
	display: none;
}
.collapse_SIT + input + div{
	display: none;
}
.collapse_SIT + input:checked + div{
	display: block;
}
.SIT {
	text-align: justify;
	text-justify: inter-word;
}
.SIT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #0489B1;
	padding-left: 5px;
}
.SIT_collapse_similarity:hover{
	background: #d0dfe2;
}
.SIT_collapse_similarity + input {
	display: none;
}
.SIT_collapse_similarity + input + div{
	display: none;
}
.SIT_collapse_similarity + input:checked + div{
	display: block;
}
.SIT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.SIT_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #0489B1;
}
.SIT_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_TU {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #04B486;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_TU + input{
	display: none;
}
.collapse_TU + input + div{
	display: none;
}
.collapse_TU + input:checked + div{
	display: block;
}
.TU {
	text-align: justify;
	text-justify: inter-word;
}
.TU_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #04B486;
	padding-left: 5px;
}
.TU_collapse_similarity:hover{
	background: #d0e2de;
}
.TU_collapse_similarity + input {
	display: none;
}
.TU_collapse_similarity + input + div{
	display: none;
}
.TU_collapse_similarity + input:checked + div{
	display: block;
}
.TU_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.TU_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #04B486;
}
.TU_similar > a {
	text-decoration: none;
	color: gray;
}

ul { 
	list-style-type: none;
	padding: 0;
	margin: 0;
}
ul li > a {
	text-decoration: none;
	color: gray;
}
ul li:hover, ul li:focus {
	background: #F0F0F0;
}

ul li:first-child span {
  border-left:none;  
}

</style>
<html>
<head>
<meta charset="utf-8">
<title>TF properties - SgrR</title>
<meta name="theme-color" content="#ffffff">
</head>
<body>
	<div class="banner">
		<center><font color="white" size="16px" face="arial">SgrR</font></center>
	</div>
	<div align="center" class="main-content">
		<div class="spacer">
	</div><br>			<label class="collapse_ACT" for="_1"><font size="5px">SgrR transcriptional activity</font></label>
				<input id="_1" type="checkbox">
				<div class="ACT"><br><ul><li>aerobic to anaerobic anaerobic to aerobic ArgR CysB BirA DgsA MalT GadW MetJ NikR OxyR SoxR SoxS DcuR CueR FadR HcaR NtrC PhoB PurR SgrR TdcA TrpR ZraR ArcA CpxR CRP CusR FhlA Fis FNR Fur H-NS IHF IscR Lrp ModE NarL NrdR PdhR FlhDC GlcC EvgA GadX LeuO PspF RbsR steady states Figure 3 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22870390" target="_blank">22870390</a>]<br></li>
<li>The Clusters of Orthologous Groups database , available through the National Center for Biotechnology Information , was used to identify proteins in COG4533 , which are closely related evolutionarily to SgrR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15522088" target="_blank">15522088</a>]<br></li>
<li>Endogenous SgrR could not be detected by Western blotting using a polyclonal antibody raised against SgrR-His ( data not shown ) , indicating that cellular levels of SgrR are quite low .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17209026" target="_blank">17209026</a>]<br></li>
<li>Addition of MG had no signiﬁcant effect on the extent of negative autoregulation by SgrR when sgrR expression was induced by IPTG from Plac-sgrR ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17209026" target="_blank">17209026</a>]<br></li>
<li>However , this is almost certainly not the only mechanism for control of SgrR amounts , since mutation of the sequences required for negative autoregulation ( 5-bp substitution as shown in Fig. 1 ) did not allow accumulation of SgrR to detectable levels ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17209026" target="_blank">17209026</a>]<br></li>
<li>As SgrR is the ﬁrst member of this family to be characterized , little was known , prior to this work , about the mechanism of regulation of target genes or about the regulation of expression and activity of the SgrR protein itself .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17209026" target="_blank">17209026</a>]<br></li>
<li>If this was the case , SgrR would be similar in some respects to the LysR family of proteins , which autoregulate their own expression and often the expression of divergent target genes and bind to DNA sites with and without signal ( 15 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17209026" target="_blank">17209026</a>]<br></li>
<li>To examine the DNA binding characteristics of SgrR in vitro , an epitope-tagged SgrR-His protein was overexpressed and puriﬁed by Ni2 - nitrilotriacetic acid afﬁnity chromatography .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17209026" target="_blank">17209026</a>]<br></li>
<li>These observations , coupled with our ﬁnding that SgrR binds in vitro to a speciﬁc DNA fragment without the addition of exogenous small molecules , suggests that SgrR DNA binding is independent of signal .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17209026" target="_blank">17209026</a>]<br></li>
<li>The yfdZ gene , encoding a putative aminotransferase , was identiﬁed in this screen ; the yfdZ promoter contains an SgrR binding site , and transcriptional fusions indicate that yfdZ is activated by SgrR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17209026" target="_blank">17209026</a>]<br></li>
<li>The regulated degradation of ptsG mRNA , which we now know to be caused by SgrS , is correlated with intracellular accumulation of G6P or fructose-6-phosphate ( F6P ) ( 10 ) , suggesting that SgrR might directly bind G6P or F6P when levels increase past a certain threshold .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17209026" target="_blank">17209026</a>]<br></li>
<li>Our results suggest that under conditions of glucose phosphate accumulation , SgrR activates SgrS synthesis , causing degradation of ptsG mRNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15522088" target="_blank">15522088</a>]<br></li>
<li>We had hypothesized that the activity of SgrR might be regulated at the level of DNA binding depending on the presence of the stress signal , that is , that SgrR binding to its target sequences may only occur in the presence of MG ( 21 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17209026" target="_blank">17209026</a>]<br></li>
<li>The activity of the ypdA promoter was low and was similar in wildtype and sgrR mutant hosts in the absence and presence of MG , suggesting that ypdA is not regulated by SgrR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17209026" target="_blank">17209026</a>]<br></li>
<li>As shown in Fig. 2B ( sgrR : : cm ) , the absence of SgrR did not affect the basa level of setA - lacZ activity in the absence of stress but abolished activation following the exposure of cells to MG .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20971900" target="_blank">20971900</a>]<br></li>
<li>The analyses of PsgrR - and PsgrS-lacZ fusions indicate that while SgrR binding determinants located in the sgrR coding sequence are not necessary for activation of sgrS , they are important for full autoregulation of sgrR ( Fig. 2A and 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17209026" target="_blank">17209026</a>]<br></li>
<li>As predicted by the presence of this domain , we ﬁnd that SgrR binds to a speciﬁc DNA sequence required for transcriptional regulation of sgrS ( Fig. 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17209026" target="_blank">17209026</a>]<br></li>
<li>Based on the predicted domain structure ( Fig. 7A ) , we hypothesize that SgrR acts directly as a sensor of the intracellular phosphosugar concentration and as a DNA-binding protein to allow activation of sgrS transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15522088" target="_blank">15522088</a>]<br></li>
<li>In vitro studies demonstrated that puriﬁed SgrR binds speciﬁcally to sgrS promoter DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17209026" target="_blank">17209026</a>]<br></li>
<li>This result suggested that , unlike the activation activity of SgrR at the sgrS promoter , the autorepression activity of the SgrR protein is not signiﬁcantly affected by the stress .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17209026" target="_blank">17209026</a>]<br></li>
<li>Puriﬁed SgrR binds speciﬁcally to sgrS promoter DNA in vitro ; a mutation in the site required for in vivo activation and autorepression abrogates in vitro SgrR binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17209026" target="_blank">17209026</a>]<br></li>
<li>The same SgrR binding site is required for activation of sgrS and autorepression of sgrR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17209026" target="_blank">17209026</a>]<br></li>
<li>This corresponds to the region that would contain the translation initiation sequences for sgrR , suggesting that SgrR binding at this site for activation of sgrS transcription may result in simultaneous negative autoregulation of sgrR transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17209026" target="_blank">17209026</a>]<br></li>
<li>This binding may then activate SgrR to a conformation competent to-bind DNA and activate transcription at the sgrS promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15522088" target="_blank">15522088</a>]<br></li>
<li>Thus , Cra bound on sgrS may lead to the repression of sgrR , thereby resulting in a decrease in activator SgrR levels .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21115656" target="_blank">21115656</a>]<br></li>
<li>Expression of sgrS is induced in response to aMG by an activator protein , SgrR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15522088" target="_blank">15522088</a>]<br></li>
<li>They have also found that the expression of sgrS is regulated by a novel transcriptional-activator SgrR that may activate sgrS transcription in response to high intracellular levels of phosphosugars .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15650111" target="_blank">15650111</a>]<br></li>
<li>SgrR with bound sugar-phos-phate would then activate transcription of sgrS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17209026" target="_blank">17209026</a>]<br></li>
<li>Given the domain structure of SgrR and the evidence showing that SgrR requires a stress signal in order to activate sgrS expression , we hypothesize that binding of a small-molecule inducer may alter the function of the SgrR protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17209026" target="_blank">17209026</a>]<br></li>
<li>As described in Results , we found that SgrR was required for aMG-inducible sgrS expression and thus , the recovery from phosphosugar stress .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15522088" target="_blank">15522088</a>]<br></li>
<li>Vanderpool and Gottesman ( 45 ) identiﬁed a transcriptional-activator called SgrR , which is activated under these conditions and causes enhanced transcription of a small RNA designated SgrS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16855233" target="_blank">16855233</a>]<br></li>
<li>Analyses of the PsgrR1-lacZ fusion showed that in the absence of stress , transcription of the sgrR promoter was consistently at least 1.5-fold higher in an sgrR mutant background than in sgrR cells , demonstrating that SgrR at least modestly negatively autoregulates its own transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17209026" target="_blank">17209026</a>]<br></li>
<li>Without stress , the activity of the longer PsgrR2-lacZ fusion was 3-fold-higher in the sgrR mutant host than in the wild type ( Fig. 2B ) , again suggesting that SgrR negatively autoregulates its own transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17209026" target="_blank">17209026</a>]<br></li>
<li>These results conﬁrm that SgrR negatively autoregulates its own transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17209026" target="_blank">17209026</a>]<br></li>
<li>In this study , we examined the regulation of sgrR expression and found that SgrR negatively autoregulates its own transcription in the presence and absence of stress .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17209026" target="_blank">17209026</a>]<br></li>
<li>Consistent with this prediction , we found that SgrR negatively autoregulates its own transcription ( Fig. 2A and B ) under both stress and nonstress conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17209026" target="_blank">17209026</a>]<br></li>
<li>SgrR activities : activation versus repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17209026" target="_blank">17209026</a>]<br></li>
<li>However , the negative autoregulation phenotype in the absence of glucose-phosphate stress ( Fig. 2 ) strongly suggests that SgrR can bind DNA in a repression-competent form in the absence of the stress signal .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17209026" target="_blank">17209026</a>]<br></li>
<li>Negative autoregulation by SgrR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17209026" target="_blank">17209026</a>]<br></li>
<li>Thus , it is plausible that additional factor ( s ) other than RNase E , enolase , Hfq , SgrS , and SgrR are also involved in the regulation of stability of ptsG mRNA in response to phosphosugar stress .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15650111" target="_blank">15650111</a>]<br></li>
<li>It is therefore possible that increased endogenous pyruvate concentrations caused by the kdgR deletion alter signaling through SgrR and , subsequently , stress-dependent activation of sgrS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20971900" target="_blank">20971900</a>]<br></li>
<li>Vanderpool and Gottesman have reported that SgrR stimulates yfdZ expression 2-fold and mediates activation by sugar-phosphate stress ( 35 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20729367" target="_blank">20729367</a>]<br></li>
<li>Minimally , these results demonstrate that , in addition to sgrS , at least one unlinked gene is activated by SgrR in response to sugar-phosphate stress .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17209026" target="_blank">17209026</a>]<br></li>
<li>This result demonstrated that SgrR is required for stress-dependent induction of setA transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20971900" target="_blank">20971900</a>]<br></li>
<li>Collectively , these results suggested that under glucose-phosphate stress conditions , KdgR represses the activation of sgrS ( and very slightly of setA ) indirectly by altering signaling through the stress response system and therefore activity of SgrR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20971900" target="_blank">20971900</a>]<br></li>
<li>The growth defect and differences in signaling through SgrR ( for activation of sgrS transcription ) in the setA mutant under certain conditions conﬁrm its connection to the stress response .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20971900" target="_blank">20971900</a>]<br></li>
<li>While sgrS transcription is activated by SgrR only in the presence of a stress signal ( 21 ) , repression of sgrR by SgrR occurs independently of stress ( Fig. 2 and 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17209026" target="_blank">17209026</a>]<br></li>
<li>Taken together , these in vivo results suggest that the same SgrR binding site is required for activation of sgrS transcription under glucose-phosphate stress conditions and negative autoregulation of sgrR transcription in the presence and absence of stress .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17209026" target="_blank">17209026</a>]<br></li>
<li>Genetic analyses of the regulation of sgrR ( Fig. 2 ) and sgrS ( 21 ) ( Fig. 3 ) showed that SgrR is required for activation of sgrS transcription under glucose-phosphate stress conditions and autorepression of sgrR , prompting the hypothesis that SgrR directly binds DNA to mediate these effects .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17209026" target="_blank">17209026</a>]<br></li>
<li>During glucose-phosphate stress , SgrR activates transcription of sgrS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23873911" target="_blank">23873911</a>]<br></li>
<li>During glucose-phosphate stress , SgrR is required for activation of sgrS transcription ( 21 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17209026" target="_blank">17209026</a>]<br></li>
<li>It therefore remains possible that SgrR senses another small molecule that accumulates or is depleted under glucose-phosphate stress conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17209026" target="_blank">17209026</a>]<br></li>
<li>The present study was undertaken in order to further understand the roles played by the small RNA SgrS and its transcriptional-regulator SgrR in the response to glucose-phos-phate stress .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17209026" target="_blank">17209026</a>]<br></li>
<li>A model for the roles of SgrR and SgrS in the response to glucose PTS-dependent phosphosugar stress .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15522088" target="_blank">15522088</a>]<br></li>
<li>The transcription factor SgrR activates transcription of the small RNA SgrS under stress conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20971900" target="_blank">20971900</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_56">
The novel transcription factor SgrR coordinates the response to glucose-phosphate stress .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20729367" target="_blank">20729367</a>]<br></label>
<input id="act_similarity_56", type="checkbox">
<div class="ACT_similar">
<li>Vanderpool , C.K. and Gottesman , S. ( 2007 ) The novel transcription factor SgrR coordinates the response to glucose-phosphate stress .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21504983" target="_blank">21504983</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21422074" target="_blank">21422074</a>]<br></li></div>
<li>SgrS or SgrR are strongly inhibited under these conditions and fail to recover signiﬁcantly ( 12 , 21 ) , indicating that this pathway is critical for the glucose-phosphate stress response .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17209026" target="_blank">17209026</a>]<br></li>
<li>This study demonstrates that setA expression is induced in response to glucose-phosphate stress , and this requires SgrR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20971900" target="_blank">20971900</a>]<br></li>
<li>SgrR bound to the wild-type Psgr fragment to produce a shifted band over a range of protein concentrations ( Fig. 4 ) ; the apparent KD for this interaction was 5 nM .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17209026" target="_blank">17209026</a>]<br></li>
<li>We currently do not know the identity of the putativ small-molecule signal that is bound by SgrR and modulates its activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17209026" target="_blank">17209026</a>]<br></li>
<li>Transcription of PsgrS -- lacZ was still completely dependent upon aMG in DsgrR : : cm ( pLCV3 ) cells , strongly suggesting that it is the activity of SgrR rather than its synthesis that is affected by aMG .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15522088" target="_blank">15522088</a>]<br></li>
<li>We theorized that clones like pBRlib15 negatively affected PsgrS-lacZ activity because they bound free SgrR protein and titrated it away from the chromosomal fusion .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17209026" target="_blank">17209026</a>]<br></li>
<li>Taken together , these data suggest that mlc affects activity of PsgrS-lacZ by a mechanism independent of repression of ptsG and probably not involving titration of the SgrR protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17209026" target="_blank">17209026</a>]<br></li>
				</div></ul><br>
			<label class="collapse_DOM" for="_2"><font size="5px">SgrR protein domains</font></label>
				<input id="_2" type="checkbox">
				<div class="DOM"><br><ul><li>The resulting construct , pET-sgrR , encodes a recombinant SgrR protein containing the T7 tag epitope at the N terminus and the His6 epitope at the C terminus .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17209026" target="_blank">17209026</a>]<br></li>
<li>A. Schematic diagram of the functional domains of SgrR , predicted by the Interpro database .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15522088" target="_blank">15522088</a>]<br></li>
<li>While the biochemical properties of SgrR have not yet been studied , we hypothesize that SgrR may act as a sensor of the intracellular accumulation of phosphoglucose by binding these molecules in the C-terminal sol-ute-binding domain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15522088" target="_blank">15522088</a>]<br></li>
<li>JOURNAL OF BACTERIOLOGY , Mar. 2007 , p. 2238 -- / 07 / $ 08.00 0 doi :10.1128 / JB.01689-06 The Novel Transcription Factor SgrR Coordinates the Response to Glucose-Phosphate Stress † Carin K. Vanderpool * and Susan Gottesman Laboratory of Molecular Biology , National Cancer Institute , Bethesda , Maryland 20892 Received 1 November 2006/Accepted 21 December 2006 SgrR is the ﬁrst characterized member of a family of bacterial transcription factors containing an Nterminal DNA-binding domain and a C-terminal solute binding domain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17209026" target="_blank">17209026</a>]<br></li>
<li>Expression of sgrS is regulated by a novel transcriptional activator , SgrR ( formerly YabN ) , which has a putative DNA-binding domain and a solute-binding domain similar to those found in certain transport proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15522088" target="_blank">15522088</a>]<br></li>
<li>However , the domain of SgrR similar to the periplasmic solute-binding proteins is at the C-termi-nus of SgrR ( Fig. 7A ) , beginning at amino acid position ª140 in the SgrR sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15522088" target="_blank">15522088</a>]<br></li>
<li>Many SgrR family members are erroneously annotated in bacterial genomes as `` putative periplasmic transport protein '' due to the strong homology of the C-terminal domain with metal or sugar binding transport proteins found in the periplasm .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17209026" target="_blank">17209026</a>]<br></li>
<li>SgrR is the ﬁrst characterized member of a previously unstudied family of proteins that have been annotated primarily as solute binding proteins or periplasmic transport proteins due to their conserved C-terminal solute binding domains .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17209026" target="_blank">17209026</a>]<br></li>
<li>However , as we observed previously ( 21 ) , SgrR family members also contain N-terminal-DNA-binding-domains , suggesting that these proteins are instead transcription factors .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17209026" target="_blank">17209026</a>]<br></li>
<li>Instead , conservation of the N-terminus ( predicted DNA-binding domain ) suggests that SgrR and its other family members may all function as transcriptional regulators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15522088" target="_blank">15522088</a>]<br></li>
<li>SgrR is the deﬁning member of a previously uncharacterized protein family with a conserved wingedhelix type DNA-binding domain and a solute-binding domain similar to sugar-binding domains .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15522088" target="_blank">15522088</a>]<br></li>
<li>SgrR and its family members are characterized by a two-domain structure with an N-termi-nal DNA-binding domain of the winged helix family and a solute binding domain at the C terminus .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17209026" target="_blank">17209026</a>]<br></li>
<li>The SgrR protein contains two domains , an N-terminal-DNA-binding-domain of the winged-helix family and a C-terminal solute binding domain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17209026" target="_blank">17209026</a>]<br></li>
<li>Mutation of the known SgrR binding site in this short region ( clone pBRCV3 carries the 5-bp substitution , as in Fig. 1 ) completely abrogated titration .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17209026" target="_blank">17209026</a>]<br></li>
<li>The setA mutation also affects the stress-re-sponsive activation of sgrS in a medium-speciﬁc manner , suggesting that SetA activity affects signaling through SgrR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20971900" target="_blank">20971900</a>]<br></li>
<li>The analyses of PsgrR - and PsgrS-lacZ fusions indicate that while SgrR binding determinants located in the sgrR coding sequence are not necessary for activation of sgrS , they are important for full autoregulation of sgrR ( Fig. 2A and 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17209026" target="_blank">17209026</a>]<br></li>
<li>Puriﬁed SgrR binds speciﬁcally to sgrS promoter DNA in vitro ; a mutation in the site required for in vivo activation and autorepression abrogates in vitro SgrR binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17209026" target="_blank">17209026</a>]<br></li>
<li>An SgrR binding site in the sgrR-sgrS intergenic region is required in vivo for both SgrR-dependent activation of sgrS and autorepression of sgrR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17209026" target="_blank">17209026</a>]<br></li>
<li>The same SgrR binding site is required for activation of sgrS and autorepression of sgrR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17209026" target="_blank">17209026</a>]<br></li>
<li>Given the domain structure of SgrR and the evidence showing that SgrR requires a stress signal in order to activate sgrS expression , we hypothesize that binding of a small-molecule inducer may alter the function of the SgrR protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17209026" target="_blank">17209026</a>]<br></li>
<li>As predicted by the presence of this domain , we ﬁnd that SgrR binds to a speciﬁc DNA sequence required for transcriptional regulation of sgrS ( Fig. 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17209026" target="_blank">17209026</a>]<br></li>
<li>Based on the predicted domain structure ( Fig. 7A ) , we hypothesize that SgrR acts directly as a sensor of the intracellular phosphosugar concentration and as a DNA-binding protein to allow activation of sgrS transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15522088" target="_blank">15522088</a>]<br></li>
				</div></ul><br>
			<label class="collapse_EVO" for="_3"><font size="5px">SgrR evolution</font></label>
				<input id="_3" type="checkbox">
				<div class="EVO"><br><ul><li>In some genomes , the genes encoding SgrR family members are divergent from genes predicted to be involved in transport processes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17209026" target="_blank">17209026</a>]<br></li>
<li>Intriguingly , in Bacillus subtilis , the gene yhjP , encoding an SgrR family member , is divergently transcribed from the yhjO gene , predicted to encode a major facilitator family efﬂux pump , homologous to SetA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15522088" target="_blank">15522088</a>]<br></li>
<li>SgrR is a member of a novel family of bacterial transcription factors ( NCBI Clusters of Orthologous Groups designation COG4533 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17209026" target="_blank">17209026</a>]<br></li>
<li>JOURNAL OF BACTERIOLOGY , Mar. 2007 , p. 2238 -- / 07 / $ 08.00 0 doi :10.1128 / JB.01689-06 The Novel Transcription Factor SgrR Coordinates the Response to Glucose-Phosphate Stress † Carin K. Vanderpool * and Susan Gottesman Laboratory of Molecular Biology , National Cancer Institute , Bethesda , Maryland 20892 Received 1 November 2006/Accepted 21 December 2006 SgrR is the ﬁrst characterized member of a family of bacterial transcription factors containing an Nterminal DNA binding domain and a C-terminal solute binding domain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17209026" target="_blank">17209026</a>]<br></li>
<li>SgrR belongs to a well-conserved family of proteins of unknown function , all sharing conservation throughout their length but annotated as periplasmic binding proteins ( COG4533 ) , similar to periplasmic protein components of ABC-transporters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15522088" target="_blank">15522088</a>]<br></li>
<li>Expression of sgrS is regulated by a novel transcriptional activator , SgrR ( formerly YabN ) , which has a putative DNA-binding domain and a solute-binding domain similar to those found in certain transport proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15522088" target="_blank">15522088</a>]<br></li>
<li>Instead , conservation of the N-terminus ( predicted DNA-binding domain ) suggests that SgrR and its other family members may all function as transcriptional regulators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15522088" target="_blank">15522088</a>]<br></li>
<li>SgrR and its family members are characterized by a two-domain structure with an N-termi-nal DNA binding domain of the winged helix family and a solute binding domain at the C-terminus .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17209026" target="_blank">17209026</a>]<br></li>
<li>The SgrR protein contains two domains , an N-terminal DNA binding domain of the winged-helix family and a C-terminal solute binding domain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17209026" target="_blank">17209026</a>]<br></li>
<li>However , as we observed previously ( 21 ) , SgrR family members also contain N-terminal DNA binding domains , suggesting that these proteins are instead transcription factors .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17209026" target="_blank">17209026</a>]<br></li>
<li>Many SgrR family members are erroneously annotated in bacterial genomes as `` putative periplasmic transport protein '' due to the strong homology of the C-terminal domain with metal or sugar binding transport proteins found in the periplasm .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17209026" target="_blank">17209026</a>]<br></li>
<li>SgrR is the ﬁrst characterized member of a previously unstudied family of proteins that have been annotated primarily as solute binding proteins or periplasmic transport proteins due to their conserved C-terminal solute binding domains .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17209026" target="_blank">17209026</a>]<br></li>
<li>SgrR is the deﬁning member of a previously uncharacterized protein family with a conserved wingedhelix type DNA-binding domain and a solute-binding domain similar to sugar-binding domains .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15522088" target="_blank">15522088</a>]<br></li>
				</div></ul><br>
			<label class="collapse_RP" for="_4"><font size="5px">Processes regulated by SgrR</font></label>
				<input id="_4" type="checkbox">
				<div class="RP"><br><ul><li>Continued Gene product b Genes and promoter regions a Unknown and miscellaneous functions yccT Unknown Induced in stationary phase ( 76 ) b0964 b2447 b3679 b1586 b2602 b0255 b2625 1.51 1.52 1.53 1.54 1.55 1.56 1.57 yffP Predicted protein , prophage yidK Putative membrane transporter ynfD Predicted protein yﬁL Putative lipoprotein yi91a Unknown , in CP4-6 prophage sequence yfjL Unknown , possible prophage gene yagL Unknown , prophage protein sS ( 9 ) sS-Dependent in bioﬁlm-grow cells ( 18 ) eutH , in eutHA operon , is sS-dependent ( 9 ) sS ( 9 ) b0278 1.59 ing Reactivating factor for ethanolamine ammonia lyase Predicted outer membrane protein Hypothetical protein Unknown Unknown , mutant affecting swarming motility Phantom gene b2451 1.59 eutA b3524 b0625 b1964 b2510 b2596 1.61 1.66 1.69 1.74 1.79 yhjG ybeH yedS yfgJ G7353 Upstream of a ribosome modulation factor induced in stationary phase ( yﬁA ) yqiG Unknown , interrupted by IS element Intergenic region including ycbX promoter Unknown Unknown Intergenic region downstream of yfjH Unknown b3046 N.A. b0947 b2623 N.A. b1228 1.81 2.26 2.05 2.30 2.01 3.21 ycbX yfjH ychS Non-coding RNAs ryeE ( cyaR ) CRP , sE ( 77,78 ) b4438 Small RNA , promotes degradation of ompX and nadE RNA Small RNA , inhibits ptsG translation/SgrT protein sibD/sibE non coding RNA - ibsD/ibsE toxic peptides 1.50 sgrS/sgrT 1.51 b4577 SgrR protein ECs3934 1.62 b4447 b4664 Complex locus including two non coding RNAs overlapped by two small ORFs encoding a putative toxin/antitoxin system micA small RNA ( downregulates ompA expression ) not annotated in MG1655 Unknown , annotated in CFT073 Intergenic region between syd ( predicted protein ) and csrB ( ncRNA ) , antisense Unknown , annotated in CFT073 , downstream of malM Between fabB and ycfJ , antisense Unknown , annotated in O157 : H7 SAKAI Intergenic region , upstream of predicted helicase ygcB Unknown , annotated in CFT073 ( antisense of atpC ) Intergenic region upstream of gadA , antisense Unknown , annotated in CFT073 , upstream of aqpZ Unknown , annotated in CFT073 , upstream of dacA , transcribed in opposite direction Intergenic region upstream emrKY ( TolC-multidrug efﬂux pump ) Intergenic region upstream of yohJK ( inner membrane proteins ) Intergenic region between yehI-yehK , antisense Unknown , annotated in CFT073 ( antisense of menB ) Unknown , annotated in CFT073 , upstream of perM gene , transcribed in opposite direction Intergenic region , upstream of yhjB Intergenic region upstream of yqcE Unknown , annotated in CFT073 ( antisense of aspA ) Intergenic region upstream of rrsH ribosomal operon ( transcribed in opposite direction ) b4442 2.00 micA Intergenic regions , genes c3878 1.50 1.50 N.A. N.A. 1.50 N.A. c5008 ... .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21398637" target="_blank">21398637</a>]<br></li>
<li>A plasmid library screen identiﬁed clones that alter expression of a PsgrS-lacZ fusion ; some act by titrating endogenous SgrR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17209026" target="_blank">17209026</a>]<br></li>
<li>Thus , it is plausible that additional factor ( s ) other than RNase E , enolase , Hfq , SgrS , and SgrR are also involved in the regulation of stability of ptsG mRNA in response to phosphosugar stress .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15650111" target="_blank">15650111</a>]<br></li>
<li>SgrR * P SgrS EIIC EIIB PtsG EIIC Hfq EIIB P P P sgrS P SgrS synthesis Stress : Sugar-P P SgrS : Target base pairing Hfq accumulation SgrR SgrS sRNA/Target degradation RNaseE ptsG mRNA RNA regulator of phosphosugar stress response 1085 accumulation of excess phosphosugars that inhibit cell-growth .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15522088" target="_blank">15522088</a>]<br></li>
<li>Our results suggest that under conditions of glucose phosphate accumulation , SgrR activates SgrS synthesis , causing degradation of ptsG mRNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15522088" target="_blank">15522088</a>]<br></li>
<li>Previously , we reported genetic evidence that SgrR activates the divergently transcribed gene sgrS , which encodes a small RNA required for recovery from glucose-phosphate stress .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17209026" target="_blank">17209026</a>]<br></li>
<li>Under glucose-phosphate stress conditions , E. coli cells initiate a unique cellular response mediated by the transcription factor SgrR , which activates transcription of a small RNA ( sRNA ) , SgrS ( Fig. 1A ) ( 32 -- 34 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20971900" target="_blank">20971900</a>]<br></li>
<li>Furthermore , we found that a dedicated pathway to deal with glucose-phosphate stress involved , in addition to SgrS , a novel transcription factor protein , SgrR ( 21 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17209026" target="_blank">17209026</a>]<br></li>
<li>The novel transcription factor SgrR coordinates the response to glucose-phosphate stress .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20729367" target="_blank">20729367</a>]<br></li>
<li>During glucose-phosphate stress , SgrR activates transcription of sgrS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23873911" target="_blank">23873911</a>]<br></li>
<li>In some genomes , the genes encoding SgrR family members are divergent from genes predicted to be involved in transport processes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17209026" target="_blank">17209026</a>]<br></li>
<li>RESULTS Regulation of divergent sgrR and sgrS genes : SgrR is an activator and a repressor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17209026" target="_blank">17209026</a>]<br></li>
<li>Expression of sgrS is induced in response to aMG by an activator protein , SgrR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15522088" target="_blank">15522088</a>]<br></li>
				</div></ul><br>
			<label class="collapse_SIT" for="_5"><font size="5px">SgrR binding site</font></label>
				<input id="_5" type="checkbox">
				<div class="SIT"><br><ul><li>At the Asp-Pro-stop sequence in the SgrR gene , the ribosome pauses robustly with a score of 84 , meaning that a ribosome is 84 times more likely to be found at this site than a typical position in the SgrR ORF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23431150" target="_blank">23431150</a>]<br></li>
<li>This corresponds to the region that would contain the translation initiation sequences for sgrR , suggesting that SgrR binding at this site for activation of sgrS transcription may result in simultaneous negative autoregulation of sgrR transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17209026" target="_blank">17209026</a>]<br></li>
<li>This result suggested that there were sequences between the limits of clones pBRCV16 and pBRCV17 , within the coding region for SgrR , that were important for SgrR binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17209026" target="_blank">17209026</a>]<br></li>
<li>Alignment of PsgrS sequences from closely related organisms showed that the sequences in this region were highly conserved ( 21 ) , supporting the notion that a conserved SgrR binding site may occur here .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17209026" target="_blank">17209026</a>]<br></li>
<li>This mutation in pBRCV20 abrogated the titration effect and restored PsgrS-lacZ activity to 100 % ( Fig. 6A ) , strongly suggesting that these sequences constitute an SgrR binding site and that the wild-type yfdZ-ypdA intergenic region titrates SgrR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17209026" target="_blank">17209026</a>]<br></li>
<li>A putative SgrR binding site was identiﬁed by sequence comparison with PsgrS ( Fig. 6B ) , and a substitution mutation in these sequences abrogated the titration ( Fig. 6A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17209026" target="_blank">17209026</a>]<br></li>
				</ul></div><br>
			<label class="collapse_TU" for="_6"><font size="5px">Transcription units regulated by SgrR</font></label>
				<input id="_6" type="checkbox">
				<div class="TU"><br><ul><li>Continued Gene product b Genes and promoter regions a Unknown and miscellaneous functions yccT Unknown Induced in stationary phase ( 76 ) b0964 b2447 b3679 b1586 b2602 b0255 b2625 1.51 1.52 1.53 1.54 1.55 1.56 1.57 yffP Predicted protein , prophage yidK Putative membrane transporter ynfD Predicted protein yﬁL Putative lipoprotein yi91a Unknown , in CP4-6 prophage sequence yfjL Unknown , possible prophage gene yagL Unknown , prophage protein sS ( 9 ) sS-Dependent in bioﬁlm-grow cells ( 18 ) eutH , in eutHA operon , is sS-dependent ( 9 ) sS ( 9 ) b0278 1.59 ing Reactivating factor for ethanolamine ammonia lyase Predicted outer membrane protein Hypothetical protein Unknown Unknown , mutant affecting swarming motility Phantom gene b2451 1.59 eutA b3524 b0625 b1964 b2510 b2596 1.61 1.66 1.69 1.74 1.79 yhjG ybeH yedS yfgJ G7353 Upstream of a ribosome modulation factor induced in stationary phase ( yﬁA ) yqiG Unknown , interrupted by IS element Intergenic region including ycbX promoter Unknown Unknown Intergenic region downstream of yfjH Unknown b3046 N.A. b0947 b2623 N.A. b1228 1.81 2.26 2.05 2.30 2.01 3.21 ycbX yfjH ychS Non-coding RNAs ryeE ( cyaR ) CRP , sE ( 77,78 ) b4438 Small RNA , promotes degradation of ompX and nadE RNA Small RNA , inhibits ptsG translation/SgrT protein sibD/sibE non coding RNA - ibsD/ibsE toxic peptides 1.50 sgrS/sgrT 1.51 b4577 SgrR protein ECs3934 1.62 b4447 b4664 Complex locus including two non coding RNAs overlapped by two small ORFs encoding a putative toxin/antitoxin system micA small RNA ( downregulates ompA expression ) not annotated in MG1655 Unknown , annotated in CFT073 Intergenic region between syd ( predicted protein ) and csrB ( ncRNA ) , antisense Unknown , annotated in CFT073 , downstream of malM Between fabB and ycfJ , antisense Unknown , annotated in O157 : H7 SAKAI Intergenic region , upstream of predicted helicase ygcB Unknown , annotated in CFT073 ( antisense of atpC ) Intergenic region upstream of gadA , antisense Unknown , annotated in CFT073 , upstream of aqpZ Unknown , annotated in CFT073 , upstream of dacA , transcribed in opposite direction Intergenic region upstream emrKY ( TolC-multidrug efﬂux pump ) Intergenic region upstream of yohJK ( inner membrane proteins ) Intergenic region between yehI-yehK , antisense Unknown , annotated in CFT073 ( antisense of menB ) Unknown , annotated in CFT073 , upstream of perM gene , transcribed in opposite direction Intergenic region , upstream of yhjB Intergenic region upstream of yqcE Unknown , annotated in CFT073 ( antisense of aspA ) Intergenic region upstream of rrsH ribosomal operon ( transcribed in opposite direction ) b4442 2.00 micA Intergenic regions , genes c3878 1.50 1.50 N.A. N.A. 1.50 N.A. c5008 ... .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21398637" target="_blank">21398637</a>]<br></li>
<li>If this was the case , SgrR would be similar in some respects to the LysR family of proteins , which autoregulate their own expression and often the expression of divergent target genes and bind to DNA sites with and without signal ( 15 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17209026" target="_blank">17209026</a>]<br></li>
<li>Addition of MG had no signiﬁcant effect on the extent of negative autoregulation by SgrR when sgrR expression was induced by IPTG from Plac-sgrR ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17209026" target="_blank">17209026</a>]<br></li>
<li>They have also found that the expression of sgrS is regulated by a novel transcriptional activator SgrR that may activate sgrS transcription in response to high intracellular levels of phosphosugars .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15650111" target="_blank">15650111</a>]<br></li>
<li>Expression of sgrS is induced in response to aMG by an activator protein , SgrR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15522088" target="_blank">15522088</a>]<br></li>
<li>In vivo studies of sgrR regulation revealed that the SgrR protein functions as an activator of target genes such as sgrS and as a repressor of its own transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17209026" target="_blank">17209026</a>]<br></li>
<li>RESULTS Regulation of divergent sgrR and sgrS genes : SgrR is an activator and a repressor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17209026" target="_blank">17209026</a>]<br></li>
<li>Analyses of the PsgrR1-lacZ fusion showed that in the absence of stress , transcription of the sgrR promoter was consistently at least 1.5-fold higher in an sgrR mutant background than in sgrR cells , demonstrating that SgrR at least modestly negatively autoregulates its own transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17209026" target="_blank">17209026</a>]<br></li>
<li>Without stress , the activity of the longer PsgrR2-lacZ fusion was 3-fold-higher in the sgrR mutant host than in the wild type ( Fig. 2B ) , again suggesting that SgrR negatively autoregulates its own transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17209026" target="_blank">17209026</a>]<br></li>
<li>These results conﬁrm that SgrR negatively autoregulates its own transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17209026" target="_blank">17209026</a>]<br></li>
<li>In this study , we examined the regulation of sgrR expression and found that SgrR negatively autoregulates its own transcription in the presence and absence of stress .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17209026" target="_blank">17209026</a>]<br></li>
<li>Consistent with this prediction , we found that SgrR negatively autoregulates its own transcription ( Fig. 2A and B ) under both stress and nonstress conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17209026" target="_blank">17209026</a>]<br></li>
<li>Stress-responsive activation of sgrS requires the transcription factor SgrR , which is encoded by the sgrR gene divergently transcribed from sgrS ( Fig. 1B ) ( 34 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20971900" target="_blank">20971900</a>]<br></li>
<li>Under stress conditions , SgrS RNA is highly transcribed by the action of SgrR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15650111" target="_blank">15650111</a>]<br></li>
<li>This result is not surprising because we have shown that synthesis of the sRNA SgrS ( which encodes sgrT ) under stress conditions depends on the transcription factor SgrR ( 6 , 17 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18042713" target="_blank">18042713</a>]<br></li>
<li>This study demonstrates that setA expression is induced in response to glucose-phosphate stress , and this requires SgrR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20971900" target="_blank">20971900</a>]<br></li>
<li>Our data are consistent with an operonic arrangement for sgrS and setA , with transcription initiating upstream of sgrS at a promoter that is activated under stress conditions by the transcription factor SgrR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20971900" target="_blank">20971900</a>]<br></li>
<li>The growth defect and differences in signaling through SgrR ( for activation of sgrS transcription ) in the setA mutant under certain conditions conﬁrm its connection to the stress response .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20971900" target="_blank">20971900</a>]<br></li>
<li>Taken together , these in vivo results suggest that the same SgrR binding site is required for activation of sgrS transcription under glucose-phosphate stress conditions and negative autoregulation of sgrR transcription in the presence and absence of stress .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17209026" target="_blank">17209026</a>]<br></li>
<li>DISCUSSION The small RNA SgrS and the transcription factor that controls sgrS expression , SgrR , are both required for recovery from glucose-phosphate stress ( 21 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17209026" target="_blank">17209026</a>]<br></li>
<li>Analysis of fusions indicated that the yfdZ gene is positively regulated by SgrR ( Fig. 6C ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17209026" target="_blank">17209026</a>]<br></li>
<li>This suggests that yfdZ expression may be controlled by two promoters , one SgrR dependent and one SgrR independent .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17209026" target="_blank">17209026</a>]<br></li>
				</div></ul></div><br>
	</div>
</body>
</html>