<!doctype html>
<style>
@charset "utf-8";
body {
	height: 100%;
	margin: 0;
	padding: 0;
}

.banner {
	position: fixed;
	top: 0;
	width: 100%;
	height: 65px;
	background: gray;
	line-height: 65px;
}
.spacer {
	height: 10px;
}

.main-content {
	padding: 5%;
}

.collapse_ACT {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-style: none solid none none;
	border-color: #B4045F;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_ACT + input{
	display: none;
}
.collapse_ACT + input + div{
	display: none;
}
.collapse_ACT + input:checked + div{
	display: block;
}
.ACT {
	text-align: justify;
	text-justify: inter-word;
}
.ACT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B4045F;
}
.ACT_collapse_similarity:hover{
	background: #e5d3dc;
}
.ACT_collapse_similarity + input {
	display: none;
}
.ACT_collapse_similarity + input + div{
	display: none;
}

.ACT_collapse_similarity + input:checked + div{
	display: block;
}
.ACT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.ACT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B4045F;
	padding-left: 5px;
}
.ACT_collapse_similarity:hover{
	background: #e5d3dc;
}
.ACT_collapse_similarity + input {
	display: none;
}
.ACT_collapse_similarity + input + div{
	display: none;
}
.ACT_collapse_similarity + input:checked + div{
	display: block;
}
.ACT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.ACT_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #B4045F;
}
.ACT_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_DOM {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-style: none solid none none;
	border-color: #B404AE;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_DOM + input{
	display: none;
}
.collapse_DOM + input + div{
	display: none;
}
.collapse_DOM + input:checked + div{
	display: block;
}
.DOM {
	text-align: justify;
	text-justify: inter-word;
}
.DOM_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B404AE;
	padding-left: 5px;
}
.DOM_collapse_similarity:hover{
	background: #e8d7e7;
}
.DOM_collapse_similarity + input {
	display: none;
}
.DOM_collapse_similarity + input + div{
	display: none;
}
.DOM_collapse_similarity + input:checked + div{
	display: block;
}
.DOM_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}.DOM_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #B404AE;
}
.DOM_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_EVO {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #7401DF;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_EVO + input{
	display: none;
}
.collapse_EVO + input + div{
	display: none;
}
.collapse_EVO + input:checked + div{
	display: block;
}
.EVO {
	text-align: justify;
	text-justify: inter-word;
}
.EVO_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #7401DF;
	padding-left: 5px;
}
.EVO_collapse_similarity:hover{
	background: #e2d7ed;
}
.EVO_collapse_similarity + input {
	display: none;
}
.EVO_collapse_similarity + input + div{
	display: none;
}
.EVO_collapse_similarity + input:checked + div{
	display: block;
}
.EVO_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.EVO_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #7401DF;
}
.EVO_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_RP {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #0174DF;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_RP + input{
	display: none;
}
.collapse_RP + input + div{
	display: none;
}
.collapse_RP + input:checked + div{
	display: block;
}
.RP {
	text-align: justify;
	text-justify: inter-word;
}
.RP_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #0174DF;
	padding-left: 5px;
}
.RP_collapse_similarity:hover{
	background: #c9d7e2;
}
.RP_collapse_similarity + input {
	display: none;
}
.RP_collapse_similarity + input + div{
	display: none;
}
.RP_collapse_similarity + input:checked + div{
	display: block;
}
.RP_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.RP_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #0174DF;
}
.RP_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_SIT {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #0489B1;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_SIT + input{
	display: none;
}
.collapse_SIT + input + div{
	display: none;
}
.collapse_SIT + input:checked + div{
	display: block;
}
.SIT {
	text-align: justify;
	text-justify: inter-word;
}
.SIT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #0489B1;
	padding-left: 5px;
}
.SIT_collapse_similarity:hover{
	background: #d0dfe2;
}
.SIT_collapse_similarity + input {
	display: none;
}
.SIT_collapse_similarity + input + div{
	display: none;
}
.SIT_collapse_similarity + input:checked + div{
	display: block;
}
.SIT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.SIT_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #0489B1;
}
.SIT_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_TU {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #04B486;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_TU + input{
	display: none;
}
.collapse_TU + input + div{
	display: none;
}
.collapse_TU + input:checked + div{
	display: block;
}
.TU {
	text-align: justify;
	text-justify: inter-word;
}
.TU_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #04B486;
	padding-left: 5px;
}
.TU_collapse_similarity:hover{
	background: #d0e2de;
}
.TU_collapse_similarity + input {
	display: none;
}
.TU_collapse_similarity + input + div{
	display: none;
}
.TU_collapse_similarity + input:checked + div{
	display: block;
}
.TU_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.TU_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #04B486;
}
.TU_similar > a {
	text-decoration: none;
	color: gray;
}

ul { 
	list-style-type: none;
	padding: 0;
	margin: 0;
}
ul li > a {
	text-decoration: none;
	color: gray;
}
ul li:hover, ul li:focus {
	background: #F0F0F0;
}

ul li:first-child span {
  border-left:none;  
}

</style>
<html>
<head>
<meta charset="utf-8">
<title>TF properties - TdcA</title>
<meta name="theme-color" content="#ffffff">
</head>
<body>
	<div class="banner">
		<center><font color="white" size="16px" face="arial">TdcA</font></center>
	</div>
	<div align="center" class="main-content">
		<div class="spacer">
	</div><br>			<label class="collapse_ACT" for="_1"><font size="5px">TdcA transcriptional activity</font></label>
				<input id="_1" type="checkbox">
				<div class="ACT"><br><ul><li>aerobic to anaerobic anaerobic to aerobic ArgR CysB BirA DgsA MalT GadW MetJ NikR OxyR SoxR SoxS DcuR CueR FadR HcaR NtrC PhoB PurR SgrR TdcA TrpR ZraR ArcA CpxR CRP CusR FhlA Fis FNR Fur H-NS IHF IscR Lrp ModE NarL NrdR PdhR FlhDC GlcC EvgA GadX LeuO PspF RbsR steady states Figure 3 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22870390" target="_blank">22870390</a>]<br></li>
<li>Although the precise mechanism of transcriptional regulation has yet to be elucidated , it has been established that the cAMP receptor protein ( CRP ) , integration host factor and two proteins , termed TdcR and TdcA , are necessary for positive regulation ( Wu et al. , 1992 ; Hagewood et al. , 1994 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11251844" target="_blank">11251844</a>]<br></li>
<li>Future studies will involve developing an in vitro assay with all the transcription factor components , which should help to elucidate how this promoter is regulated in response to oxygen and how the TdcA and TdcR proteins regulate the initiation process .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11251844" target="_blank">11251844</a>]<br></li>
<li>Thus , it appears reasonable that TdcA could function as a putative DNA binding protein to enhance tdc transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8413189" target="_blank">8413189</a>]<br></li>
<li>Does TdcA bind to tdcP DNA to facilitate transcription ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7928991" target="_blank">7928991</a>]<br></li>
<li>If this notion is correct , the function of TdcR is to initially increase the cellular level of TdcA , which then sustains the high steady-state level of tdc transcription in the absence of TdcR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7928991" target="_blank">7928991</a>]<br></li>
<li>No such active complex is formed in the absence of TdcR at a low concentration of TdcA ( from a single-copy gene ) , thus reducing tdc transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7928991" target="_blank">7928991</a>]<br></li>
<li>Whether TdcA and TdcR function together to activate the anaerobic expression of tdcA remains to be resolved .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11251844" target="_blank">11251844</a>]<br></li>
<li>Taken together , these findings indicate that , in the deletion construct CRP , TdcA and TdcR are functional in controlling transcription of the operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11251844" target="_blank">11251844</a>]<br></li>
<li>This indicates that the other tdc regulators , TdcA and TdcR , govern anaerobic transcription activation by CRP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11251844" target="_blank">11251844</a>]<br></li>
<li>Remarkably , transcription in vivo from the mutant promoter was unaffected aerobically , which indicates that TdcA , bound to the presumptive co-inducer , together with IHF and TdcR are all essential for transcriptional activation , even when the cAMP ± CRP complex is not limiting .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11251844" target="_blank">11251844</a>]<br></li>
				</div></ul><br>
			<label class="collapse_DOM" for="_2"><font size="5px">TdcA protein domains</font></label>
				<input id="_2" type="checkbox">
				<div class="DOM"><br><ul><li>For example , TdcA has highly conserved residues at positions 28 , 32 , 38 and 39 , hydrophobic residues at positions 31 and 33 , and a basic residue at position 40 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8413189" target="_blank">8413189</a>]<br></li>
<li>These results might tend to suggest that the TdcA-mediated regulation of the tdc operon does not appear to depend on its D N A binding activity to tdcP , however , we can not completely rule out the possibility that some missing factor ( s ) might be involved in promoting tdcP-TdcA interaction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8413189" target="_blank">8413189</a>]<br></li>
<li>We reported earlier ( 3 ) that , despite numerous attempts , no TdcA-specific shift in gel mobility in vitro of tdcP DNA was observed , and it was speculated that some missing factor ( s ) might be involved in promoting tdcP-TdcA interaction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7928991" target="_blank">7928991</a>]<br></li>
<li>We previously reported that the TdcA polypeptide consists of 312 amino acid residues with calculated molecular weight of 35 544 , and in minicells produced a polypeptide of expected length ( Goss and Datta 1985 ; Goss et al. 1988 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8413189" target="_blank">8413189</a>]<br></li>
<li>Thus , the tdcP DNA region around -175 appears to titrate the TdcA polypeptide in vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7928991" target="_blank">7928991</a>]<br></li>
<li>Thus , it appears reasonable that TdcA could function as a putative DNA-binding protein to enhance tdc transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8413189" target="_blank">8413189</a>]<br></li>
<li>Thus , TdcA could potentially function as a DNA-binding protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7928991" target="_blank">7928991</a>]<br></li>
<li>The amino acid sequene at the amino-terminal end of TdcA revealed a significant homology to the helix-turn-helix motifs of typical DNA-binding proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8413189" target="_blank">8413189</a>]<br></li>
<li>The amino acid sequence of TdcA revealed a significant sequence similarity with the bacterial LysR-family of transcriptional activators and contains a putative helix-turn-helix motif for DNA-binding ( 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7928991" target="_blank">7928991</a>]<br></li>
<li>Analysis of the TdcA sequence according to the method of Dodd and Egan ( 1987 ) revealed that the amino acid sequence between residues 24 and 43 shows a significant homology to the helix-turn-helix motifs of several DNA-binding proteins ( Fig. 3 ; Pabo and Sauer 1984 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8413189" target="_blank">8413189</a>]<br></li>
<li>Helix-turn-helix motif in TdcA polypeptide as compared with that of several known DNA-binding proteins ( taken from Pabo and Sauer 1984 ; Dodd and Egan 1987 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8413189" target="_blank">8413189</a>]<br></li>
<li>TdcA is a member o f the L y s R family o f proteins The lysR gene of E. coli encodes a protein of 311 amino acid residues , which activates transcription of lysA , the structural gene for diaminopimelate decarboxylase , involved in lysine biosynthesis ( Stragier and Patte 1983 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8413189" target="_blank">8413189</a>]<br></li>
<li>The on the top numbers represent amino acid residues of TdcA polypeptide .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8413189" target="_blank">8413189</a>]<br></li>
				</div></ul><br>
			<label class="collapse_EVO" for="_3"><font size="5px">TdcA evolution</font></label>
				<input id="_3" type="checkbox">
				<div class="EVO"><br><ul><li>TdcA is a member o f the L y s R family o f proteins The lysR gene of E. coli encodes a protein of 311 amino acid residues , which activates transcription of lysA , the structural gene for diaminopimelate decarboxylase , involved in lysine biosynthesis ( Stragier and Patte 1983 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8413189" target="_blank">8413189</a>]<br></li>
				<label class="EVO_collapse_similarity" for="evo_similarity_2">
Our results suggest that TdcA is related to the LysR family of proteins by common ancestry and , based on its functional role in tdc expression , belongs to the LysR family of transcriptional activators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8413189" target="_blank">8413189</a>]<br></label>
<input id="evo_similarity_2", type="checkbox">
<div class="EVO_similar">
<li>Thus , TdcA appears to be related to the LysR family of proteins by common ancestry , and based on its functional role in tdc expression , belongs to the LysR family of transcriptional activators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8413189" target="_blank">8413189</a>]<br></li></div>
<li>TdcR is predicted to be a DNA-binding protein ( Schweizer and Datta , 1989 ) , and TdcA is a member of the LysR family of transcription factors ( Ganduri et al. , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11251844" target="_blank">11251844</a>]<br></li>
<li>Based on structural and functional similarities , TdcA appears to belong to the LysR family of bacterial activator proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8413189" target="_blank">8413189</a>]<br></li>
<li>Mol Gen Genet ( 1993 ) 240:395 -402 © Springer-Verlag 1993 TdcA , a transcriptional activator of the tdcABC operon of Escherichia coil , is a member of the LysR family of proteins Received : 21 December 1992 / Accepted : 8 March 1993 Abstract .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8413189" target="_blank">8413189</a>]<br></li>
				<label class="EVO_collapse_similarity" for="evo_similarity_7">
TdcA , a transcriptional activator of the tdcABC operon of Escherichia coli , is a member of the LysR family of proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9829935" target="_blank">9829935</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7928991" target="_blank">7928991</a>]<br></label>
<input id="evo_similarity_7", type="checkbox">
<div class="EVO_similar">
<li>Ganduri , Y.L. , Sadda , S.R. , Datta , M.W. , Jambukeswaran , R.K. , and Datta , P. ( 1993 ) TdcA , a transcriptional activator of the tdcABC operon of Escherichia coli , is a member of the LysR family of proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11251844" target="_blank">11251844</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9484901" target="_blank">9484901</a>]<br></li></div>
<li>Figure 2 shows the sequence alignment of TdcA with LysR , as well as with a consensus sequence deduced from multiple alignment of nine LysR family members ( Henikoff et al. 1988 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8413189" target="_blank">8413189</a>]<br></li>
<li>A comparison of TdcA with the LysR family consensus sequence also shows a high degree of similarity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8413189" target="_blank">8413189</a>]<br></li>
<li>The amino acid sequences of TdcA and LysR revealed 17 % identity and an additional 23 % similarity throughout their entire lengths although the proteins are more similar near their amino-terminal ends .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8413189" target="_blank">8413189</a>]<br></li>
<li>The amino acid sequence of TdcA revealed a significant sequence similarity with the bacterial LysR family of transcriptional activators and contains a putative helix-turn-helix motif for DNA binding ( 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7928991" target="_blank">7928991</a>]<br></li>
				</div></ul><br>
			<label class="collapse_RP" for="_4"><font size="5px">Processes regulated by TdcA</font></label>
				<input id="_4" type="checkbox">
				<div class="RP"><br><ul><li>TdcA is a member o f the L y s R family o f proteins The lysR gene of E. coli encodes a protein of 311 amino acid residues , which activates transcription of lysA , the structural gene for diaminopimelate decarboxylase , involved in lysine-biosynthesis ( Stragier and Patte 1983 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8413189" target="_blank">8413189</a>]<br></li>
<li>Likewise , transcription factors of the genes for amino-acid-metabolism , including AdiY and CysB ( regulator of Cys synthesis ) , LeuO ( regulator of Leu synthesis ) , and TdcA and TdcR ( reg ¬ The change in amino acid levels was interconnected with the changes in the level of intermediate metabolites of carbohydrate-catabolism and energy metabolism .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2162465" target="_blank">2162465</a>]<br></li>
<li>Interestingly , the genes coding for local regulators of the genes for individual amino acids are under the control of Lrp , including AdiY ( a regulator of Arg regulon ) , CysB ( a regulator of the Cys regulon ) , GadW ( a regulator of Glu reg ¬ Lrp Network Y J YhjI AdiY OLm r pR BluR YbaO YedW CadX CadW SdiA PhoP H-NS Cbl CysB TyrR TdcR DeoT TdcA CadX CadW DctR SdiA EnvR TdcA TdcA FabR StpA GadW LeuO GadE GadW GadX DtcR SlyA LeuO BglG StpA QseB Rob LrhA MatA NanR ~ 20 TFs FlhDC FliE LrhA FlhDC ~ 10 TFs FliZ MatA FlhDC QseCB Fig. 6 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2162465" target="_blank">2162465</a>]<br></li>
				</div></ul><br>
			<label class="collapse_SIT" for="_5"><font size="5px">TdcA binding site</font></label>
				<input id="_5" type="checkbox">
				<div class="SIT"><br><ul><li>It has been proposed that TdcA binds to a site 175 bp upstream of the tdcA promoter ( Hagewood et al. , 1994 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11251844" target="_blank">11251844</a>]<br></li>
<li>TdcA is a LysR-type regulator ( Ganduri et al. , 1993 ) , and a mutational analysis of the tdcR ± tdcA intergenic region suggested that TdcA may bind to an inverted-repeat 175 bp upstream of the tdcA transcription start site ( Hagewood et al. , 1994 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11251844" target="_blank">11251844</a>]<br></li>
<li>The results summarized in this article suggest potential interaction between TdcR and TdcA and the involvement of a specific tdcP sequence at -175 for TdcA binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7928991" target="_blank">7928991</a>]<br></li>
<li>Sequence alignment of TdcA with LysR also showed considerable sequence similarity throughout their entire lengths .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8413189" target="_blank">8413189</a>]<br></li>
<li>Figure 2 shows the sequence alignment of TdcA with LysR , as well as with a consensus sequence deduced from multiple alignment of nine LysR family members ( Henikoff et al. 1988 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8413189" target="_blank">8413189</a>]<br></li>
<li>A comparison of TdcA with the LysR family consensus sequence also shows a high degree of similarity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8413189" target="_blank">8413189</a>]<br></li>
				</ul></div><br>
			<label class="collapse_TU" for="_6"><font size="5px">Transcription unit regulated by TdcA</font></label>
				<input id="_6" type="checkbox">
				<div class="TU"><br><ul><li>Requirementof tdcA for tdcB-lacZ expression Expt Straina ( plasmid ) Relevant genotype LacZ activityin TdcBactivityin TYE H4 TYE TH274 ) ~ YG2413 ( pYG18 ) TH274 ) ~ YG2413 ( pYG113 ) TH274LYG2413 ( pYG112 ) TH274 ) ~ SH241 ( loYG18 ) TH274 ) ~ SH241 ( pYG113 ) TH274 ) ~ SH241 ( pYG112 ) TH274 ) ~ YG2413 ( pYG184 ) TH274LYG2413 ( pYG234 ) TH274 ~ , SH241 ( pYG184 ) TH274 ; ~ SH241 ( pYG234 ) tdcR + tdcAaB-lacZ ( vector ) 155 ( 1.0 ) 744 ( 4.8 ) 754 ( 4.9 ) 756 ( 1.0 ) 775 ( 1.0 ) 839 ( 1.1 ) 210 ( 1.0 ) 980 ( 4.7 ) 817 ( 1.0 ) 503 ( 0.6 ) 267 ( 1.0 ) 1266 ( 4.7 ) 1348 ( 5.0 ) 1053 ( 1.0 ) 704 ( 0.7 ) 1097 ( 1.0 ) 237 ( 1.0 ) 947 ( 4.0 ) 1084 ( 1.0 ) 745 ( 0.7 ) 361 210 tdcR + tdcAAB-lacZ ( tdcR lacPtdcA + B ' ) tdcR + tdcAaB - lacZ ( tdcR lacPtdcA + B + C + ) tdcR + tdcAB-lacZ ( vector ) tdcR + tdcAB-lacZ ( tdcR lacPtdcA + B ' ) tdcR + tdcAB-lacZ ( tdcR lacPtdcA + B+C + ) tdcR + tdcA ~ B-lacZ ( vector ) tdcR + tdcA ~ B-IacZ ( tdcR tdcPA + ) tdcR -- tdcAB-lacZ ( vector ) tdcR + tdcAB-lacZ ( tdcR tdcPA + ) 36850 ND ND ND ND II III IV For culture conditions , enzymeassays and specificactivities , see tion ofvectorcontrolare shownin parentheses.ND , not determined Materialsand methods.Relativevaluesof LacZexpressionas a func - , Strain TH274is Mac A ( tdcR-tdcABC ) Helix-turn-helix motif in TdcA How does a trans-acting protein activate gene transcription ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8413189" target="_blank">8413189</a>]<br></li>
<li>TcbC does not functionally replace any of the tdc operon regulators Expression of the tdc operon is controlled directly by a number of positive transcriptional regulators , including CRP , IHF , TdcA and TdcR ( Wu et al. , 1992 ; Hagewood et al. , 1994 ) , as well as being controlled indirectly by FNR ( Chattopadhyay et al. , 1997 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11251844" target="_blank">11251844</a>]<br></li>
<li>Transcript levels were low and equivalent under all conditions tested ( Fig. 8 , lanes 10 ± 12 ) , indicating that TdcA and/or TdcR mediate the anaerobic induction of operon expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11251844" target="_blank">11251844</a>]<br></li>
<li>Further experiments are necessary to elucidate the biochemical mechanism of tdc operon expression by TdcA and its mutual interactions , if any , with the other trans-acting regulatory factors in transcription activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8413189" target="_blank">8413189</a>]<br></li>
<li>It has been found that both TdcR and TdcA , which have helix-turn-helix motifs , appear to be involved in efficient expression of the tdc operon ( 22 , 27 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1328166" target="_blank">1328166</a>]<br></li>
<li>ogt and abgT occurred in a context that suggested an operon structure with two additional open reading frames upstream of abgT ( given labels b1337 and b1338 ) adjacent to a divergently transcribed lysRlike regulatory gene ( b1339 ) that encodes a protein 27 % identical to TdcA , a LysR-like transcription activator of the threonine dehydratase operon ( 10 , 31 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9829935" target="_blank">9829935</a>]<br></li>
<li>TdcA is a member o f the L y s R family o f proteins The lysR gene of E. coli encodes a protein of 311 amino acid residues , which activates transcription of lysA , the structural gene for diaminopimelate decarboxylase , involved in lysine biosynthesis ( Stragier and Patte 1983 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8413189" target="_blank">8413189</a>]<br></li>
				</div></ul></div><br>
	</div>
</body>
</html>