<!doctype html>
<style>
@charset "utf-8";
body {
	height: 100%;
	margin: 0;
	padding: 0;
}

.banner {
	position: fixed;
	top: 0;
	width: 100%;
	height: 65px;
	background: gray;
	line-height: 65px;
}
.spacer {
	height: 10px;
}

.main-content {
	padding: 5%;
}

.collapse_ACT {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-style: none solid none none;
	border-color: #B4045F;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_ACT + input{
	display: none;
}
.collapse_ACT + input + div{
	display: none;
}
.collapse_ACT + input:checked + div{
	display: block;
}
.ACT {
	text-align: justify;
	text-justify: inter-word;
}
.ACT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B4045F;
}
.ACT_collapse_similarity:hover{
	background: #e5d3dc;
}
.ACT_collapse_similarity + input {
	display: none;
}
.ACT_collapse_similarity + input + div{
	display: none;
}

.ACT_collapse_similarity + input:checked + div{
	display: block;
}
.ACT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.ACT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B4045F;
	padding-left: 5px;
}
.ACT_collapse_similarity:hover{
	background: #e5d3dc;
}
.ACT_collapse_similarity + input {
	display: none;
}
.ACT_collapse_similarity + input + div{
	display: none;
}
.ACT_collapse_similarity + input:checked + div{
	display: block;
}
.ACT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.ACT_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #B4045F;
}
.ACT_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_DOM {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-style: none solid none none;
	border-color: #B404AE;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_DOM + input{
	display: none;
}
.collapse_DOM + input + div{
	display: none;
}
.collapse_DOM + input:checked + div{
	display: block;
}
.DOM {
	text-align: justify;
	text-justify: inter-word;
}
.DOM_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B404AE;
	padding-left: 5px;
}
.DOM_collapse_similarity:hover{
	background: #e8d7e7;
}
.DOM_collapse_similarity + input {
	display: none;
}
.DOM_collapse_similarity + input + div{
	display: none;
}
.DOM_collapse_similarity + input:checked + div{
	display: block;
}
.DOM_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}.DOM_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #B404AE;
}
.DOM_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_EVO {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #7401DF;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_EVO + input{
	display: none;
}
.collapse_EVO + input + div{
	display: none;
}
.collapse_EVO + input:checked + div{
	display: block;
}
.EVO {
	text-align: justify;
	text-justify: inter-word;
}
.EVO_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #7401DF;
	padding-left: 5px;
}
.EVO_collapse_similarity:hover{
	background: #e2d7ed;
}
.EVO_collapse_similarity + input {
	display: none;
}
.EVO_collapse_similarity + input + div{
	display: none;
}
.EVO_collapse_similarity + input:checked + div{
	display: block;
}
.EVO_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.EVO_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #7401DF;
}
.EVO_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_RP {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #0174DF;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_RP + input{
	display: none;
}
.collapse_RP + input + div{
	display: none;
}
.collapse_RP + input:checked + div{
	display: block;
}
.RP {
	text-align: justify;
	text-justify: inter-word;
}
.RP_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #0174DF;
	padding-left: 5px;
}
.RP_collapse_similarity:hover{
	background: #c9d7e2;
}
.RP_collapse_similarity + input {
	display: none;
}
.RP_collapse_similarity + input + div{
	display: none;
}
.RP_collapse_similarity + input:checked + div{
	display: block;
}
.RP_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.RP_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #0174DF;
}
.RP_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_SIT {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #0489B1;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_SIT + input{
	display: none;
}
.collapse_SIT + input + div{
	display: none;
}
.collapse_SIT + input:checked + div{
	display: block;
}
.SIT {
	text-align: justify;
	text-justify: inter-word;
}
.SIT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #0489B1;
	padding-left: 5px;
}
.SIT_collapse_similarity:hover{
	background: #d0dfe2;
}
.SIT_collapse_similarity + input {
	display: none;
}
.SIT_collapse_similarity + input + div{
	display: none;
}
.SIT_collapse_similarity + input:checked + div{
	display: block;
}
.SIT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.SIT_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #0489B1;
}
.SIT_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_TU {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #04B486;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_TU + input{
	display: none;
}
.collapse_TU + input + div{
	display: none;
}
.collapse_TU + input:checked + div{
	display: block;
}
.TU {
	text-align: justify;
	text-justify: inter-word;
}
.TU_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #04B486;
	padding-left: 5px;
}
.TU_collapse_similarity:hover{
	background: #d0e2de;
}
.TU_collapse_similarity + input {
	display: none;
}
.TU_collapse_similarity + input + div{
	display: none;
}
.TU_collapse_similarity + input:checked + div{
	display: block;
}
.TU_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.TU_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #04B486;
}
.TU_similar > a {
	text-decoration: none;
	color: gray;
}

ul { 
	list-style-type: none;
	padding: 0;
	margin: 0;
}
ul li > a {
	text-decoration: none;
	color: gray;
}
ul li:hover, ul li:focus {
	background: #F0F0F0;
}

ul li:first-child span {
  border-left:none;  
}

</style>
<html>
<head>
<meta charset="utf-8">
<title>TF properties - UhpA</title>
<meta name="theme-color" content="#ffffff">
</head>
<body>
	<div class="banner">
		<center><font color="white" size="16px" face="arial">UhpA</font></center>
	</div>
	<div align="center" class="main-content">
		<div class="spacer">
	</div><br>			<label class="collapse_ACT" for="_1"><font size="5px">UhpA transcriptional activity</font></label>
				<input id="_1" type="checkbox">
				<div class="ACT"><br><ul><li>The RR-bound 32P was stably retained for the A1 group RRs such as pArcA , pCpxR , pCusR , pKdpE , and pNarL but was rapidly released for the A2 group RRs , including pBaeR , pCheB , pCheY , pCreB , pDcuR , pNarP , pNtrC , pTorR , pUhpA , pYehT , and pYfhA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15522865" target="_blank">15522865</a>]<br></li>
<li>AGRT6 VIRG AGRRA YIIA ECOLI TOXR VIBCH + RRHM Pae Plastid response-regulator OmpR Eco Porin bioynthesis , osmolarity PhoB Bsu Alkaline phosphatase regulator CreB Eco Catabolite repression ArcA Eco Aerobic respiratory control PetR Rca Cytochrome bc1 complex CutR Sli Copper metabolism KdpE Eco K + transport PhoB Eco Phosphate regulon PhoB Pae Phosphate regulon PhoP Sty Virulence TctD Sty Tricarboxylate transport VirG Atu6 Virulence VirG Atu5 Virulence VirG Arh Virulence YjjE Eco Unknown ToxR Vch Virulence cholera toxin Class 3 : DNA binding domains of family 3 FixJ Rme Nitrogen fixation FixJ Bja Nitrogen fixation NodW Bja Nodulation BvgA Bpe Virulence FimZ Eco Fimbrial synthesis DegU Bsu Extracellular degradative enzymes NarL Eco Nitrate reductase & formate dehydrogenase TrpO Pae Unknown YecB Eco Unknown RcsB Eco Colanic acid capsule biosynthesis B AgmR Fae Glycerol metabolism UhpA Eco Uptake of hexose phosphates Coma Bsu Competence GerE Bsu SigK txn .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7699720" target="_blank">7699720</a>]<br></li>
<li>Proteins in classes 4 , 5 and 6 are exceptions 7O ENAF I0 20 30 40 50 60 NtrC NtrC NifRl NtrC PgtA DctD FixJ NodW CheY SpoOF RRHM RRHM OmpR PhoB CreB ArcA PetR VirG CutR KdpE PhoB PhoP TctD AIgB YLB3 RcsC BvgS HydG CheY FrzE BarA TrpO HvgA FimZ DegU NarL AIgRI YecB UhpA RcsB Agm & CheB CheB FrzG Spo0A MrkE AgrA Coma F!bD Bpa Rme Rca Eco Sty Rle Rme Bja Bsu Bsu Cca Pae Eco Bsu ECO Eco Rca Atu Sli Eco Eco Sty Sty Pae Lin Eco Bpe Eco Eco M ~ a Eco Pae Bpe Eco Bsu Eco Pae Eco Eco Eco Pae Bsu Eco Mxa Bsu Kpn Sau Bsu Ccr 6 ILVADDDTAIKTVLNQA 6 ILVADDD ~ IRTVLNQA 5 VLVADDDRTIRTVLTQA 6 V % ~ VDDDSSIR ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7699720" target="_blank">7699720</a>]<br></li>
<li>Amino acid sequence comparisons showed that UhpA was homologous to a family of bacterial regulatory proteins , some of which act as transcriptional-activators ( OmpR , PhoB , NtrC , and DctD ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3042748" target="_blank">3042748</a>]<br></li>
<li>The carboxyl ` terminal domain of NarL shares only short patches of similarity with PhoB , OmpR , and SfrA , but it is strikingly similar to the analogous regions of DegU , UhpA , and FixJ ( Fig. 3 and Table 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2649492" target="_blank">2649492</a>]<br></li>
<li>The N-terminal two-thirds of the NarL protein has extensive homology with those of a diverse set of prokaryotic regulatory proteins , including OmpR , PhoB , SfrA , UhpA , CheY , CheB , NtrC , DctD , FixJ , VirG , SpoOF , and SpoOA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2657652" target="_blank">2657652</a>]<br></li>
<li>Therefore , the most proteins of this family are subdivided into four different groups according to their C-terminal domain : ( i ) NarL , DegU , UvrC ORF2 , UhpA , and FixJ ; ( ii ) OmpR , PhoB , PhoM ORF2 , SfrA , and VirG ; ( iii ) NtrC , DctD , and PgtA ; and ( iv ) CheY and SpoOF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2657652" target="_blank">2657652</a>]<br></li>
<li>Construction of arabinose-inducible expression system of RRs The coding sequences of six species of NarL-family RR ( EvgA , NarL , NarP , RcsB , UhpA , and UvrY ) were PCR-ampliﬁed using 5 - proximal and 3 - proximal primers of 0 0 each open reading frame ( for the sequence , see Table S1 0 in Supporting Information ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26010043" target="_blank">26010043</a>]<br></li>
<li>As an initial attempt to identify the stage 3 cross talk at the step of target promoter recognition by RR , we analyzed in this study the cross-recognition of target promoters by six NarL-family RRs , EvgA , NarL , NarP , RcsB , UhpA , and UvrY .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26010043" target="_blank">26010043</a>]<br></li>
<li>( B ) Test promoter RR Test promoter Gene product Protein function EvgA NarL NarP RcsB UhpA UvrY ydeO nirB nirB wza uhpT csrB AraC family transcription factor Nitrite reductase enzyme Regulation of acid resistance genes Reduction of nitrite to ammonia Reduction of nitrite to ammonia Capsular polysaccharide translocation Hexose-6-phosphate : phosphate antiporter Regulatory sRNA for carbon storage Nitrite reductase enzyme Outer membrane lipoprotein Major facilitator superfamily ( MFS ) transporter Small regulatory RNA ( sRNA ) Cross talk in vivo in promoter recognition between six NarL-family RRs The stage 3 cross talk in vivo was examined for a total of 30 combinations between six RRs and ﬁve representative promoters [ note that NarL and NarP recognize the same nirB promoter ] under the induction condition characteristic to each RR or in the presence and absence of over-expressed RRs .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26010043" target="_blank">26010043</a>]<br></li>
<li>As described previously ( 33 ) , activation by UhpA shows apparent cooperative behavior , and the amount of transcript synthesized was ﬁt to the following equation for sigmoidal rate dependence : n n v Vm ( UhpA ) / S0 .5 ( UhpA ) where Vm is maximal velocity and S0 .5 is the concentration of UhpA for half-maximal velocity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11976297" target="_blank">11976297</a>]<br></li>
<li>Puri-ﬁcation under non-denaturing conditions resulted in a highly concentrated , soluble His - UhpA fraction , whereas over - `` ' produced His - UhpB largely ended up in the insoluble ' fraction , due to the formation of inclusion bodies .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11739766" target="_blank">11739766</a>]<br></li>
<li>Expression of UhpA from qmulticopy plasmids subverts the normal regulatory control by Glu6P and UhpBC function , and results in elevated and constitutive production of UhpT ( Shattuck-Eidens & Kadner , 1983 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1479338" target="_blank">1479338</a>]<br></li>
<li>The Histidine Kinase Domain of UhpB Inhibits UhpA Action at the Escherichia coli uhpT Promoter JESSE S. WRIGHT , IGOR N. OLEKHNOVICH , GAIL TOUCHIE , † AND ROBERT J. KADNER * Department of Microbiology , School of Medicine , University of Virginia , Charlottesville , Virginia 22908-0734 Received 12 June 2000/Accepted 22 August 2000 The histidine kinase ( HK ) component of many two-component regulatory systems exhibits regulated ability to phosphorylate itself and to participate in transfer of phosphate to and from its cognate response-regulator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11053370" target="_blank">11053370</a>]<br></li>
<li>The Phosphoryl Transfer Domain of UhpB Interacts with the Response Regulator UhpA JESSE S. WRIGHT III AND ROBERT J. KADNER * Department of Microbiology , School of Medicine , University of Virginia , Charlottesville , Virginia 22908-0734 Received 1 November 2000/Accepted 27 February 2001 Bacterial two-component regulatory systems control the expression of target genes through regulated changes in protein phosphorylation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11325944" target="_blank">11325944</a>]<br></li>
<li>Thus , our current model for regulation can account for the necessity for all three regulatory proteins by assuming that UhpB modifies UhpA in some way to convert it to an active form .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3301805" target="_blank">3301805</a>]<br></li>
<li>UhpA and UhpB together form a two-component regulatory system that activates expression of the transporter , UhpT , after extracellular G6P binds to the membrane receptor , UhpC ( 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9837961" target="_blank">9837961</a>]<br></li>
<li>Induction of uhpT gene expression in response to the presence of extracellular glucose-6-phosphate ( Glc6P ) is mediated by an unusual two-component regulatory system comprised of three proteins , the histidine kinase ( HK ) UhpB , the response-regulator ( RR ) UhpA , and the membrane receptor UhpC ( 20 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11325944" target="_blank">11325944</a>]<br></li>
<li>β-Galactosidase Activity + G6P -- G6P -- cAMP +5 mM cAMP 1400 1200 1400 B A 1200 1000 800 600 400 200 0 200 0 1 8 7 6 5 4 3 2 cya + cya + crp + Genotype : crp + Plasmid : Vector ∆ cya ∆ cya ∆ crp ∆ crp UhpA UhpA Vector Vector UhpA UhpA Vector Figure 2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10512697" target="_blank">10512697</a>]<br></li>
<li>Unlike the wild-type situation , a GST fusion variant carrying one of the UhpB-activating substitutions , R324C , displayed autokinase activity and phosphate transfer to UhpA but retained the ability to sequester UhpA when it was altered in the conserved residues important for phosphate transfer .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11053370" target="_blank">11053370</a>]<br></li>
<li>This activating change does not appear to alter the interaction of UhpB with UhpA , since GST-Bc-R324C still showed a strong dominant-negative effect when its autokinase function was abrogated by the substitutions in conserved regions important for phosphotransfer activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11053370" target="_blank">11053370</a>]<br></li>
<li>When the phosphatase assay with GST-Bc was carried out in the presence of Mg ions , all of the label released from P-UhpA was of low molecular weight and was presumably Pi .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11325944" target="_blank">11325944</a>]<br></li>
<li>When the phosphatase assay was carried out in the presence of the metal chelator EDTA , the half-life of P-UhpA in the sample incubated with GST was greatly extended , from 10 to 20 min to around 60 min ( Fig. 3A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11325944" target="_blank">11325944</a>]<br></li>
<li>The addition of the wildtype form of GST-Bc greatly increased the rate of dephosphor-ylation of P-UhpA compared to the rate in the presence of GST ( Fig. 3A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11325944" target="_blank">11325944</a>]<br></li>
<li>The rate of dephosphorylation of P-UhpA by GST-Bc was substantially slowed in the presence of EDTA but still showed a half-life of ,10 min .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11325944" target="_blank">11325944</a>]<br></li>
<li>Taken together , these results show that GST-Bc can accelerate dephosphorylation of P-UhpA and that reverse phosphotransfer to GST-Bc can be trapped in the presence of EDTA or the absence of the HK nucleotide-binding domain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11325944" target="_blank">11325944</a>]<br></li>
<li>P-UhpA dephosphorylation was measured following further incubation in the presence of 1.5 mM GST or GST-Bc in the absence of ATP ( ﬁrst panel ) , in the presence of 1 mM ATP ( second panel ) , and in the presence of 15 mM EDTA ( third panel ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11325944" target="_blank">11325944</a>]<br></li>
<li>To demonstrate directly this cophosphatase activity , GST-Bc was added to 32P-labeled His6-UhpA prepared by incubation with acetyl [ 32P ] phosphate and then puriﬁed by elution from an Ni21-chelate matrix .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11325944" target="_blank">11325944</a>]<br></li>
<li>In the presence of 1 mM unlabeled ATP added together with UhpA , the transfer of labeled phosphate from P-GST-Bc to wild-type UhpA was greatly slowed , but the retention of label on both Uhp proteins was much more prolonged than in the absence of added ATP ( Fig. 5C ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11053370" target="_blank">11053370</a>]<br></li>
<li>The GST-Bc R324C variant also appeared to possess the ability to dephosphorylate P-UhpA , since the phosphate transferred to UhpA by the action of GST-Bc was much more labile than expected from the half-life of P-UhpA formed in the absence of UhpB by phosphate transfer from acetyl phosphate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11325944" target="_blank">11325944</a>]<br></li>
<li>The cophosphatase activity of GST-Bc was expected to exist based on the ﬁnding that the turnover of P-UhpA was much more rapid when it was formed by phosphate transfer from GST-Bc R324C than when it was produced in the absence of GST-Bc by phosphate transfer from acetyl phosphate ( 4 , 49 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11325944" target="_blank">11325944</a>]<br></li>
<li>The ability of phosphorylated GST-Bc R324C to mediate phosphate transfer to UhpA was shown by allowing formation of P-GST-Bc by autophosphorylation with [ g-32P ] ATP for 20 min followed by the addition of UhpA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11053370" target="_blank">11053370</a>]<br></li>
<li>The active variant of GST-Bc could participate in phosphate transfer to UhpA , but it also seems to mediate the dephosphorylation of P-UhpA , as shown by the relatively rapid loss of labeled phosphate from UhpA in the presence of UhpB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11053370" target="_blank">11053370</a>]<br></li>
<li>When P-GST-Bc R324C was incubated with UhpA D54N , lacking the site of phosphorylation ( Fig. 5B ) , there was no transfer of labeled phosphate to UhpA , and the amount of label on GST-Bc was retained .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11053370" target="_blank">11053370</a>]<br></li>
<li>Reverse transfer of phosphate from P-UhpA to GST-Bc was observed in the presence of the metal chelator EDTA and depended on the presence of His-313 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11325944" target="_blank">11325944</a>]<br></li>
<li>Reverse transfer of phosphate from P-UhpA to GST-Bc was found to occur in the presence of EDTA and to be dependent on His-313 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11325944" target="_blank">11325944</a>]<br></li>
<li>Strikingly , label was seen to be transferred from P-UhpA to GST-Bc when EDTA was present .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11325944" target="_blank">11325944</a>]<br></li>
<li>As expected , phosphate transfer from P-UhpA to GST-Bc was seen only when EDTA was present .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11325944" target="_blank">11325944</a>]<br></li>
<li>Transfer of phosphate from P-UhpA to GST-Bc was observed to occur under certain conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11325944" target="_blank">11325944</a>]<br></li>
<li>Chelation may trap phosphate transferred to GST-Bc by preventing its transfer back to UhpA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11325944" target="_blank">11325944</a>]<br></li>
<li>This interference phenotype could be explained by the ability of the GST-Bc protein to accelerate dephosphorylation of P-UhpA that was phosphorylated by the chromosome-encoded UhpBC complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11053370" target="_blank">11053370</a>]<br></li>
<li>This dominantnegative interference could result from the ability of GST-Bc to-bind and sequester the RR UhpA and to accelerate its dephosphorylation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11325944" target="_blank">11325944</a>]<br></li>
<li>In contrast , the dominant-negative effect of GST-Bc does not require the presence of intact UhpB and appears to occur through direct action on UhpA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11053370" target="_blank">11053370</a>]<br></li>
<li>It remains to be shown whether the dominant-negative effect of GST-Zc or GST-ZBc on porin expression results from sequestration of OmpR ( 34 ) , as proposed for the effect by GST-Bc on UhpA , or from its action as P-OmpR phosphatase ( 15 , 51 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11325944" target="_blank">11325944</a>]<br></li>
<li>There was strong interference by GST-Bc on activation by the UhpB-independent UhpA H170Y variant or by overexpressed UhpA-D54N lacking the site of phosphorylation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11053370" target="_blank">11053370</a>]<br></li>
<li>These results show that GST-Bc has a direct inhibitory effect on UhpA that is independent of its state of phosphorylation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11053370" target="_blank">11053370</a>]<br></li>
<li>The inhibitory effect of GST-Bc was seen even when UhpA could drive uhpT transcription in its unphosphor-ylated state , as occurs with the phosphorylation-independent H170Y variant of UhpA carrying the replacement of His-170 with Tyr or upon overexpression of UhpA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11325944" target="_blank">11325944</a>]<br></li>
<li>Dephosphorylation of P-UhpA is shown to be catalyzed by GST-Bc and presumably also occurs with the full-length protein , but the squelching phenotype occurs even under conditions where phosphorylation of UhpA is not involved .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11325944" target="_blank">11325944</a>]<br></li>
<li>Effect of GST-Bc expression when UhpA is also overexpressed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11053370" target="_blank">11053370</a>]<br></li>
<li>In the latter case , GST-Bc even inhibited the activity of overexpressed UhpA D54N , which is unable to be phosphorylated ( 4 , 49 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11325944" target="_blank">11325944</a>]<br></li>
<li>In the footprinting assay ( data not shown ) , the protection or enhancement of DNase cleavage of sites in the 280 to 232 region by the presence of 800 nM UhpA was completely reversed by the presence of 1,600 nM GST-Bc but was not affected by 1,600 nM GST .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11053370" target="_blank">11053370</a>]<br></li>
<li>When expressed in the absence of IPTG induction , the GST-Bc derivatives with insertions at residues 387 , 473 , and 489 exhibited the same strong dominant-negative interference as did GST-Bc , suggesting that these insertions did not affect the region involved in the docking of UhpA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11325944" target="_blank">11325944</a>]<br></li>
<li>Phosphorylation of UhpA by Acetyl Phosphate -- Several response-regulators can be phosphorylated and activated by incubation with low molecular weight phospho-donors , such as acetyl phosphate ( 13 , 29 -- 31 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8999880" target="_blank">8999880</a>]<br></li>
<li>This phenomenon strongly indicates that UhpA can use acetyl phosphate as phosphoryl donor in vivo , like many other response-regulators , and thus can b activated in the absence of kinase and phosphatase activity by UhpB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12107138" target="_blank">12107138</a>]<br></li>
<li>Although phosphorylation of UhpA by soluble forms of UhpB has not been obtained yet , phosphate transfer from acetyl phosphate , a characteristic of many response-regulators ( 13 , 30 , 31 , 33 , 43 ) , was demonstrated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8999880" target="_blank">8999880</a>]<br></li>
<li>This may explain why UhpT expression can be higher in the absence of UhpBC , during growth on pyruvate , whe UhpA is supposed to be phoshorylated by acetyl phosphate ( Fig. 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12107138" target="_blank">12107138</a>]<br></li>
<li>Purified UhpA was phosphorylated by acetyl phosphate in a reaction that was dependent on 21 Mg and on the presence of aspartate 54 , the site of phosphorylation in homologous response-regulators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8999880" target="_blank">8999880</a>]<br></li>
<li>Fractional occupancy was measured in the presence of increasing concentrations of UhpA or of P-UhpA , prepared by incubation with acetyl phosphate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10913075" target="_blank">10913075</a>]<br></li>
<li>To test for changes in state of oligomerization following phosphorylation , samples of UhpA were incubated with acetyl [ 32P ] phosphate in the presence or absence of EDTA and were separated by PAGE under nondenaturing conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8999880" target="_blank">8999880</a>]<br></li>
<li>This truncated protein was able to accept phosphate from P-UhpA in the presence of EDTA and displayed a low level of phosphate transfer in the absence of EDTA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11325944" target="_blank">11325944</a>]<br></li>
<li>Transfer of phosphate was completely blocked in the presence of 10 mM EDTA ( lane 1 ) or when aspartate-54 of UhpA was replaced with asparagine ( lane 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8999880" target="_blank">8999880</a>]<br></li>
<li>Properties of P-UhpA -- The extent of incorporation of radio-activity , the kinetics of labeling , and the lack of phosphorylation of the D54N variant suggested that treatment with acetyl phosphate resulted in the phosphorylation of Asp-54 on the majority of the UhpA molecules .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8999880" target="_blank">8999880</a>]<br></li>
<li>The UhpA protein was added at the indicated concentrations in its native state ( open symbols ) or after phosphorylation by incubation with acetyl phosphate ( solid symbols ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10913075" target="_blank">10913075</a>]<br></li>
<li>The transfer of phosphate to purified UhpA from acetyl [ 32P ] phosphate and the properties of the phosphorylated protein were investigated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8999880" target="_blank">8999880</a>]<br></li>
<li>Due to the enzymic procedure used to synthesize acetyl [ $ # P ] phosphate , this reaction resulted in phosphorylated UhpA with a high speciﬁc activity , but a low chemical concentration ( C 0 ± 5 µM ) and an excess of non-phos-phorylated UhpA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11739766" target="_blank">11739766</a>]<br></li>
<li>P-UhpA was isolated by adsorption to Ni21-conjugated agarose beads , washed , and eluted to remove excess acetyl [ 32P ] phosphate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11325944" target="_blank">11325944</a>]<br></li>
<li>To assay phosphatase activity ( in UhpBC-enriched membranes ) towards P C UhpA , His - UhpA was ﬁrst phos - ' phorylated using acetyl phosphate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11739766" target="_blank">11739766</a>]<br></li>
<li>By ﬁrst phosphorylating UhpA , using acetyl [ $ # P ] phosphate , this could be further addressed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11739766" target="_blank">11739766</a>]<br></li>
<li>UhpA ( 10 µM ) was phosphorylated using acetyl phosphate , as described in Methods .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11739766" target="_blank">11739766</a>]<br></li>
<li>In both cases , P C UhpA ( free of remaining acetyl phosphate , ATP and -RCB- or ADP ) was collected by exchanging the buffer via a BioGel P-6 microcolumn equilibrated in TEDG buffer .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11739766" target="_blank">11739766</a>]<br></li>
<li>His - UhpA ( 5 -- 15 µM ) and acetyl [ $ # P ] phosphate ' ± ° were mixed , and incubated for 2 5 h at 30 C. Similarly , nonradioactive P C UhpA was obtained by incubating 5 -- 15 M µ His - UhpA with 10 mM acetyl phosphate and 10 mM MgCl ' # in TEDG buffer ( 50 mM Tris -RCB- HCl pH 7 ± 5 ; 1 mM DTT ; 0 ± 5 mM EDTA ; 10 % , v -RCB- v , glycerol ) , at 30 ° C for 2 ± 5 h .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11739766" target="_blank">11739766</a>]<br></li>
<li>Phosphorylation of UhpA -- UhpA was phosphorylated by incubation at 37 ° C in a mixture containing 13 mM UhpA and 10 mM acetyl phosphate in buffer D ( 50 mM Tris-HCl , pH 7.5 , 6 mM MgCl2 , 1 mM DTT ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8999880" target="_blank">8999880</a>]<br></li>
<li>32 Fig. 2B demonstrates that [ P ] phosphate was transferred from acetyl phosphate to UhpA in a reaction containing 5 mM MgCl2 , 1 mM DTT , 50 mM Tris-HCl ( pH 7.5 ) , 4.8 mM UhpA , and 20 mM acetyl [ 32P ] phosphate at 37 ° C ( lane 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8999880" target="_blank">8999880</a>]<br></li>
<li>Activation of EvgA could be achieved upon exposure to an acidic condition ( Eguchi & Utsumi 2014 ) , whereas UhpA was activated after the addition of inducer glucose-6-phosphate ( Weston & Kadner 1988 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26010043" target="_blank">26010043</a>]<br></li>
<li>Next , we addressed Uhp induction in early-exponential-phase ( OD600 0.3 ) wild-type cells ( RK5115 ) grown on pyruvate ( compared with growth-on-succinate ) and a low optimal phosphate concentration ( 400 M ) upon addition of glucose 6-phosphate to ﬁnd out whether Uhp induction rates were affected when UhpA was activated by UhpB ( cells plus glucose 6-phosphate , i.e. , kinase on ) and ( supposedly ) acetyl phosphate simultaneously .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12107138" target="_blank">12107138</a>]<br></li>
<li>Furthermore , Uhp induction rates reached signiﬁcantly higher levels in uhpBC cells , via pyruvate-mediated activation of 420 UhpA , than in wild-type cells ( maximally ) induced with glucose 6-phosphate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12107138" target="_blank">12107138</a>]<br></li>
<li>Furthermore , we measured a small additive pyruvate-mediated effect , presumably due to acetyl phosphate , on glucose 6-phos-phate-triggered UhpA activation in early-exponential-phase wild-type cells , which adds to the relevance of acetyl phosphate as a physiologically signiﬁcant signaling molecule ( 33 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12107138" target="_blank">12107138</a>]<br></li>
<li>UhpA activation could be made independent of transmembrane signaling when uhpBC cells were grown on pyruvate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12107138" target="_blank">12107138</a>]<br></li>
<li>As pyruvate-mediated activation of UhpA in uhpBC cells could result in considerably stronger UhpT expression than glucose 6-phosphate-dependent activation through UhpBC , the observed positive cooperativity for the overall pathway in wild-type cells may reﬂect the previously described cooperative binding of UhpA to the uhpT promoter ( J. L. Dahl et al. , J. Biol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12107138" target="_blank">12107138</a>]<br></li>
<li>Induction by extracellular glucose-6-phosphate ( Glu6P ) of the sugar phosphate transporter UhpT requires transcription activation by the response-regulator UhpA ( 3 , 18 , 29 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11053370" target="_blank">11053370</a>]<br></li>
<li>Inorganic phosphate interfered with glucose 6-phosphate-dependent , UhpBC-mediated , as well as pyruvatemediated activation of UhpA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12107138" target="_blank">12107138</a>]<br></li>
<li>There was no detectable difference in the electrophoretic mobilities of the phosphorylated and unphosphorylated forms of UhpA during SDS-PAGE ( Fig. 2A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8999880" target="_blank">8999880</a>]<br></li>
<li>As shown in Fig. 8A , addition of unmodified UhpA in the concentration range up to 3.4 mM resulted in the characteristic , progressive decrease in mobility of the A1 DNA fragment ( coordinates 2142 to 168 ) ( lanes 2 -- 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8999880" target="_blank">8999880</a>]<br></li>
<li>As shown in Fig. 8B , the variation of DNA probe mobility as a function of the concentration of UhpA [ D54N ] protein was roughly identical in the absence or presence of acetyl phosphate and was similar to that of unphosphorylated wild-type UhpA ( compare to Fig. 8A , lanes 2 -- 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8999880" target="_blank">8999880</a>]<br></li>
<li>As the UhpA concentration was increased , there was a progressive decrease in the mobility of the protein-DNA complex , associated with a wavy electrophoretic pattern , until a maximally retarded complex of defined mobility and typical sharp appearance was eventually achieved .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8999880" target="_blank">8999880</a>]<br></li>
<li>The progressive decrease in probe mobility with increasing UhpA concentration may result from the protein concentration-dependent rebinding of the probe to UhpA molecules .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8999880" target="_blank">8999880</a>]<br></li>
<li>Analogous involvement of helix R1 in intermolecular contacts is also found in UhpA and in the CheY/CheZ complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9521685" target="_blank">9521685</a>]<br></li>
<li>All these HKs are , however , known to form TCS with the respective cognate RR partner ( ArcA for pArcB ; CheB and CheY for pCheA ; and UhpA for pUhpB ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15522865" target="_blank">15522865</a>]<br></li>
<li>With haploid levels of UhpA , addition of Glu6P resulted in strong ( > 100-fold ) induction in b-galactosidase activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10512697" target="_blank">10512697</a>]<br></li>
<li>Nonetheless , the portion of UhpB which appears to be involved in the binding of UhpA overlaps the homologous region of a VanSderived peptide which binds VanR ( 46 ) , and portions of NtrB and EnvZ which are necessary for phosphatase activity on NtrC and OmpR , respectively ( 19 , 23 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11325944" target="_blank">11325944</a>]<br></li>
<li>This interference phenotype could not result solely from the phosphatase activity of UhpB , because interference affected both overexpressed UhpA and UhpA variants which are active in the absence of phosphorylation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11053370" target="_blank">11053370</a>]<br></li>
<li>This interference can not be explained solely by UhpB phosphatase activity and must involve the binding and sequestration of UhpA by inactive UhpB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11053370" target="_blank">11053370</a>]<br></li>
<li>Another possibility was that overexpressed UhpA is activated by covalent modification by another sensor kinase in some form of cross-talk .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8999880" target="_blank">8999880</a>]<br></li>
<li>UhpA is activated by its phosphorylation on aspartate-54 by the membrane-bound UhpBC sensor kinase complex in response to extracellular Glu6P .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10572130" target="_blank">10572130</a>]<br></li>
<li>We presume that in the uninduced state , the kinase activity of UhpB is inactive , but the phospha-tase activity is available to reverse any adventitious phosphorylation of UhpA by other sensor-kinases .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8144492" target="_blank">8144492</a>]<br></li>
<li>The dependence of UhpB kinase activity on the ATP concentration implies that in vivo , with cytoplasmic ATP concentrations in the order of 4 -- 5 mM ( Rohwer et al. , 1996 ) , the UhpBC sensor will be ` kinase on ' with respect to UhpA phosphorylation , as long as extracellular G6P is being sensed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11739766" target="_blank">11739766</a>]<br></li>
<li>The requirement for UhpC to activate kinase activity allowed the demonstration that the unactivated form of UhpB confers a very strong interference with UhpA action .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11053370" target="_blank">11053370</a>]<br></li>
<li>The activity of UhpA is controlled by its phosphorylation by the cognate membrane-bound sensor kinase complex composed of UhpB and UhpC ( 21 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11976297" target="_blank">11976297</a>]<br></li>
<li>Unlabeled 1 mM ATP or 10 mM EDTA was incubated with the UhpA prior to its addition to P-UhpB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11053370" target="_blank">11053370</a>]<br></li>
<li>Symbols indicate different UhpA : UhpB ratios : - , 0:1 ; squares , 10:1 ; circles , 5:1 ; triangles , 2 ± 5:1 ; diamonds , 0 ± 5:1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11739766" target="_blank">11739766</a>]<br></li>
<li>Under these conditions , .80 % of UhpA molecules are phosphorylated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10913075" target="_blank">10913075</a>]<br></li>
<li>Under these conditions , sensing of G6P by UhpC induces autophosphorylation in UhpB and P C UhpB instantaneously serves as a substrate for UhpA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11739766" target="_blank">11739766</a>]<br></li>
<li>However , comparison of the afﬁn-ities of UhpA and P-UhpA was complicated by the decreased solubility or stability of P-UhpA relative to UhpA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10913075" target="_blank">10913075</a>]<br></li>
<li>Maximal activation was achieved at 100 to 120 nM UhpA , and half-maximal activity was obtained at around 80 nM UhpA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10512697" target="_blank">10512697</a>]<br></li>
<li>LTKRERQV 150 149 UhpA 145 RcsB PAILSAVLDLDIEGIVLKQGAPTDLPKALAALQKGKKFTPESVSRLLEKISAGGYGDKR ... .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2404948" target="_blank">2404948</a>]<br></li>
<li>Concentrations of UhpA above 300 nM were increasingly inhibitory .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10512697" target="_blank">10512697</a>]<br></li>
<li>When P C UhpA is added to G6P-induced UhpBC-enriched membrane vesicles , rapid UhpA dephosphorylation occurs .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11739766" target="_blank">11739766</a>]<br></li>
<li>Therefore , UhpBC is quickly desensitized to the kinase off/phosphatase on state , causing phosphorylated UhpA ( P-UhpA ) dephosphorylation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12107138" target="_blank">12107138</a>]<br></li>
<li>Reverse phosphotransfer requires the presence of His-313 , but the data do not allow us to conclude whether His-313 is the site of phosphorylation or whether reverse transfer is an obligate step in P-UhpA dephosphorylation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11325944" target="_blank">11325944</a>]<br></li>
<li>To assay phosphoryl transfer to UhpA , His - UhpA was added ' to the autophosphorylation mixture ( now at pH 7 ± 5 ) after 2 -- 3 min incubation with [ γ - $ # P ] ATP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11739766" target="_blank">11739766</a>]<br></li>
<li>Subsequently , puriﬁed His - ' UhpA was added .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11739766" target="_blank">11739766</a>]<br></li>
<li>Inclusion of G6P via the freeze -- thaw treatment gave the most reproducible results ( as determined via Phosphor ¬ Phosphorylation and dephosphorylation of UhpA by membrane-bound UhpB To investigate phosphoryl transfer from UhpB to UhpA , membrane-bound UhpB was ﬁrst incubated with [ γ - $ # P ] ATP to achieve maximum autophosphorylation of the kinase ( Fig. 3a , lane 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11739766" target="_blank">11739766</a>]<br></li>
<li>It would only be necessary that the interaction with the extramembranous loops blocks some step of the phosphate transfer to UhpA , while allowing the phosphatase function to remain accessible and active .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8144492" target="_blank">8144492</a>]<br></li>
<li>In our phosphoryl transfer assay the default phosphatase state ( Wright et al. , 2000 ) of UhpB was only overcome for a few minutes after the addition of ATP , whereas , as demonstrated with the P C UhpA phosphatase assay , the presence of G6P in UhpBC-enriched vesicles could not switch off UhpB phosphatase activity towards P C UhpA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11739766" target="_blank">11739766</a>]<br></li>
<li>One model proposes that cytoplasmic loops formed by the transmembrane segments of UhpB or UhpC or both bind specifically to regions of the carboxylterminal portion of UhpB and thereby block its kinase activity or its ability to transfer its covalently bound phosphate to UhpA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8144492" target="_blank">8144492</a>]<br></li>
<li>The possibility of squelching -RCB- sequestration of UhpA and OmpR by the phosphoryl transfer domain of UhpB and EnvZ , respectively , has recently been described in detail ( Wright & Kadner , 2001 ; Wright et al. , 2000 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11739766" target="_blank">11739766</a>]<br></li>
<li>Phosphate transfer from P-UhpA to the liberated phosphoryl transfer domain also occurred .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11325944" target="_blank">11325944</a>]<br></li>
<li>Phosphate transfer from P-UhpB to UhpA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11053370" target="_blank">11053370</a>]<br></li>
<li>Our conclusion at this time is that the phosphoryl transfer domain of UhpB can recognize and bind UhpA and can accept phosphate from P-UhpA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11325944" target="_blank">11325944</a>]<br></li>
<li>Some insertions in uhpB , typiﬁed by uhpB87 :4 , conferred very low Uhp expression which was somewhat higher than that of the uninduced wild type , presumably owing to loss of P-UhpA phosphatase activity , but was unaffected by the presence of Glu6P , as is seen for uhpB null mutations ( 11 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11053370" target="_blank">11053370</a>]<br></li>
<li>One hypothesis is that , in the absence of induction , UhpB functions to maintain UhpA in an inactive state and thereby contributes to the extremely low basal level of expression in the Uhp system .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1569007" target="_blank">1569007</a>]<br></li>
<li>In the absence of induction , it may act as a protein phosphatase to maintain UhpA in its unphosphorylated , inactive state .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1569007" target="_blank">1569007</a>]<br></li>
<li>Activation by multicopy uhpA was not affected by the status of uhpC but was decreased in the absence of uhpB , suggesting a role for UhpB in directly activating UhpA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3042748" target="_blank">3042748</a>]<br></li>
<li>Transport was partially inducible and somewhat lower in the uhpA strain RK7621 and the uhpB strain RK7622 , consistent with a role for uhpB in stimulating activation by UhpA , even in the presence of multicopy uhpA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3042748" target="_blank">3042748</a>]<br></li>
<li>Furthermore , comparison of the electrophoretic mobility of P-UhpA in native gels of various polyacrylamide concentrations , in comparison to the migration of protein standards , indicated that P-UhpA was monomeric in solution under those conditions .3 The rates of phosphorylation and dephosphorylation of UhpA were comparable to those seen for OmpR and PhoB ( 13 , 34 ) and were much slower than the corresponding processes in CheY and NtrC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8999880" target="_blank">8999880</a>]<br></li>
<li>The matches of UhpA with DctD , NtrC , PhoB , and CheY were at least 12 standard deviations ( SDs ) above random .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3042748" target="_blank">3042748</a>]<br></li>
<li>The phosphorylation level was from 1 % to 10 % for TorR , PhoB , CusR , BasR , RstA , UvrY , ArcA , BaeR , NtrC , KdpE , UhpA , and PhoP ( in decreasing order ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15522865" target="_blank">15522865</a>]<br></li>
<li>The UhpA - plasmids pLAW014 , pLAW016 , pLAW022 , pLAW027 , pLAW025 , pLAW026 , pLAW015 , and pLAW019 all restored growth on F6P to hosts that carried transposon insertions in uhpC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3038843" target="_blank">3038843</a>]<br></li>
<li>Plasmids pLAW028 , pLAW029 , pLAW006 , pLAW007 , and pLAW030 conferred constitutive Uhp expression on UhpA - hosts .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3038843" target="_blank">3038843</a>]<br></li>
<li>This constitutive expression of UhpT is attributable to activation of UhpA by AcP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26010043" target="_blank">26010043</a>]<br></li>
<li>The functions and interactions of the two modules of UhpA were investigated here by isotation and anatysis of in-frame detetions within the receiver module , of point mutations that confer transcription activation in the absence of kinase activity , and ot truncations that remove increasing iengths of protein sequence from the Cterminus .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7596290" target="_blank">7596290</a>]<br></li>
<li>The two-turn deletion variants also exhibited expression that required UhpA but not induction by Glu6P .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10913075" target="_blank">10913075</a>]<br></li>
<li>The requirement for UhpA is consistent with previous observations that it is necessary for uhpT expression , as transport activity is virtually undetectable in the absence of uhpA even when uhpT is in multicopy ( Weston & Kadner , 1988 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1479338" target="_blank">1479338</a>]<br></li>
<li>The uhpT-lacZ expression in this mutant decreased more than 10-fold relative to the D22 variant , and it was further reduced in the absence of UhpA ( Table 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10913075" target="_blank">10913075</a>]<br></li>
<li>The absence of UhpA led to a slight increase in expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10913075" target="_blank">10913075</a>]<br></li>
<li>The constitutive expression seen with the elevated copy number of uhpA is independent of UhpB and may reflect the increased equilibrium concentration of the activated conformer of UhpA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3301805" target="_blank">3301805</a>]<br></li>
<li>The overproduction of UhpA from multicopy plasmids results in high-level , constitutive expression of UhpT even in the absence of UhpB and UhpC functions , consistent with the hypothesis that UhpA is a transcriptional-activator ( 29 , 40 , 41 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1569008" target="_blank">1569008</a>]<br></li>
<li>DOMAIN STRUCTURE OF UhpA The transcription activator protein UhpA is absolutely required for Uhp expression , and its overproduction results in constitutive Uhp expression independent of the presence of functional forms of UhpB and UhpC ( Weston and Kadner , 1988 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8144492" target="_blank">8144492</a>]<br></li>
<li>The expression of UhpB or its cytoplasmic C-terminal segment as a GST fusion protein results in speciﬁc blockage of the Uhp signaling process by inactivation of the transcription activator UhpA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11325944" target="_blank">11325944</a>]<br></li>
<li>The observed presence of a single locus of cleavage sites in the middle of each region is most consistent with the conclusion that UhpA binds to each region as a dyad or tail-to-tail protein dimer .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11976297" target="_blank">11976297</a>]<br></li>
<li>The work presented here confirms this hypothesis by demonstrating that UhpA binds preferentially and specifically to the 264 element .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8999880" target="_blank">8999880</a>]<br></li>
<li>The activation of UhpA by these other proteins would not respond to the presence of UhpB or inducer .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3042748" target="_blank">3042748</a>]<br></li>
<li>The average hydropathy values for UhpA and UhpB are 0.18 and 0.19 , respectively , which are in the range of soluble proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3301805" target="_blank">3301805</a>]<br></li>
<li>The Uhp - phenotype resulting from the titration of UhpA , as measured in column B , is also manifested as resistance to fosfomycin .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1569008" target="_blank">1569008</a>]<br></li>
<li>The sequence recognized by UhpA is predicted to be 5 ' - GGCAAAACNNNGAAA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1569008" target="_blank">1569008</a>]<br></li>
<li>The stoichiometry and orientation of bound UhpA molecules were assessed by an afﬁnity cleavage technique .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11976297" target="_blank">11976297</a>]<br></li>
<li>Residue K593 of 70 has been found to be important for several transcription activators , including AraC , UhpA , cI , FNR , Ada , RhaR ( this study ) , and RhaS ( 4 , 24 , 27 , 35 , 36 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15342598" target="_blank">15342598</a>]<br></li>
<li># 1999 Academic Press Keywords : UhpA ; CAP ; gene expression ; transcriptional activation ; response-regulator * Corresponding author Introduction Present address : J. L. Dahl , Department of Microbiology and Immunology , University of Arizona School of Medicine , Tucson , AZ 85724 , USA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10512697" target="_blank">10512697</a>]<br></li>
<li>Protein Phosphorylation Affects Binding of the Escherichia coli Transcription Activator UhpA to the uhpT Promoter * Expression of the Escherichia coli sugar phosphate transporter UhpT is induced by extracellular glucose 6-phosphate through a transmembrane signaling proc-ess dependent on the sensor kinase UhpB and the UhpT homolog , UhpC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8999880" target="_blank">8999880</a>]<br></li>
<li>Mutational Scanning and Afﬁnity Cleavage Analysis of UhpA-Binding Sites in the Escherichia coli uhpT Promoter Igor N. Olekhnovich and Robert J. Kadner * ent of M Departm icrobiology , School of Medicine , University of Virginia , Charlottesville , Virginia 22908 Received 9 October 2001/Accepted 3 February 2002 UhpA , a member of the NarL family of response-regulators , activates transcription of the Escherichia coli uhpT gene for the sugar phosphate transporter UhpT in response to extracellular glucose-6-phosphate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11976297" target="_blank">11976297</a>]<br></li>
<li>Transcription activation by UhpA is stimulated by the catabolite gene activator protein ( CAP ) - cyclic AMP complex and depends on the C-terminal domains of the RNA polymerase RpoA and RpoD subunits .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11976297" target="_blank">11976297</a>]<br></li>
<li>Transmembrane induction of uhpT expression by external glucose 6-phosphate is positively regulated by the promoter-speciﬁc activator protein UhpA and the global regulator catabolite gene activator protein ( CAP ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7896692" target="_blank">7896692</a>]<br></li>
<li>Transcription of the Escherichia coli uhpT gene , encoding the inducible sugar phosphate transporter , is dependent on the response-regulator UhpA and is stimulated by the cyclic AMP receptor protein ( CAP ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10572130" target="_blank">10572130</a>]<br></li>
<li>Transcription of the Escherichia coli uhpT gene , encoding the organophosphate transporter , requires the response-regulator UhpA and is stimulated by the global regulator protein CAP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10512697" target="_blank">10512697</a>]<br></li>
<li>Normally the phosphorylated form of UhpA activates transcription , but overexpression of UhpA results in phos-phorylation-independent action ( Webber & Kadner , 1995 ; Weston & Kadner , 1988 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10512697" target="_blank">10512697</a>]<br></li>
<li>Thus , only the sequences from 40 to 33 altered in variants M11 and M12 contributed strongly to transcription activated by UhpA alone .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11976297" target="_blank">11976297</a>]<br></li>
<li>The role of these residues was veriﬁed by in vitro transcription assays , showing that residues 593 , 598 , and 599 were required for UhpA activation as well as other transcription activators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10572130" target="_blank">10572130</a>]<br></li>
<li>Thus , several residues in sCTD appear to be important for UhpA activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10572130" target="_blank">10572130</a>]<br></li>
<li>These results show that the K593A and R599A substitutions interfere with UhpA-dependent transcription activation but are only moderately impaired for transcription stimulated by UhpA plus CAP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10572130" target="_blank">10572130</a>]<br></li>
<li>Taken together , these results suggest that UhpA contributes to transcription activation in several ways .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11976297" target="_blank">11976297</a>]<br></li>
<li>This promoter was strongly activated by UhpA plus CAP even though UhpA-dependent transcription was greatly impaired .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10572130" target="_blank">10572130</a>]<br></li>
<li>CAP binds to the uhpT promoter at a single site , centered at ÿ103 .5 bp relative to the start of transcription , and UhpA binds to multiple sites between positions ÿ80 and ÿ32 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10512697" target="_blank">10512697</a>]<br></li>
<li>Activation by UhpA requires a promoter element centered at 264 bp , relative to the start of transcription , and activation by CAP requires a DNA site centered at position 2103.5 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7896692" target="_blank">7896692</a>]<br></li>
<li>To investigate the action of UhpA and CAP , we developed an in vitro transcription system that reproduces the dependence on UhpA and the stimulation by CAP that is seen in vivo ( 30 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10572130" target="_blank">10572130</a>]<br></li>
<li>The lack of effect of CAP on the maximal in vitro transcription contrasts to the in vivo ® nding that CAP stimulation occurs even when the levels of UhpA are elevated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10512697" target="_blank">10512697</a>]<br></li>
<li>Effect of CAP on in vitro transcription activity Addition of CAP-cAMP complex along with saturating levels of UhpA ( 200 nM ) stimulated uhpT transcription , but did not activate any other transcription sites on the PT380 fragment .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10512697" target="_blank">10512697</a>]<br></li>
<li>W region variants was stimulated by CAP to a much greater degree than were other substitutions , suggesting that CAP can activate transcription as long as UhpA molecules are bound to the S region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11976297" target="_blank">11976297</a>]<br></li>
<li>Since CAP can not independently activate transcription from so far upstream ( 2 , 42 ) , it was of interest to test whether UhpA or CAP action was related to speciﬁc contacts with RNAP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10572130" target="_blank">10572130</a>]<br></li>
<li>Because CAP can not activate uhpT transcription by itself , UhpA must activate transcription in two ways , one involving interaction with sCTD and the other involving interaction with CAP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10572130" target="_blank">10572130</a>]<br></li>
<li>We conclude that CAP and UhpA must bind in proximity for either to activate transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10913075" target="_blank">10913075</a>]<br></li>
<li>sCTD contributes to the action of UhpA alone , but not to that of UhpA plus CAP , whereas aCTD is important for CAP action .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10572130" target="_blank">10572130</a>]<br></li>
<li>We are currently attempting to isolate mutants of CAP , UhpA , and RNA polymerase that are speciﬁcally affected in transcription activation at uhpTp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7896692" target="_blank">7896692</a>]<br></li>
<li>Action of UhpA and CAP proteins was not affected by template supercoiling .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10512697" target="_blank">10512697</a>]<br></li>
<li>Transcription activation by UhpA or CAP was not apparently affected by template supercoiling .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10512697" target="_blank">10512697</a>]<br></li>
<li>These results suggest that UhpA must bind to the promoter prior to RNAP for optimal activity , and that assembly of UhpA onto the promoter is the rate-limiting process .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10512697" target="_blank">10512697</a>]<br></li>
<li>The M4 variant showed a lesser degree of impairment in UhpA occupancy , and the M1 and M2 variants had little effect on UhpA occupancy , despite the substantial decrease in promoter activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11976297" target="_blank">11976297</a>]<br></li>
<li>Removal of as few as eight amino acids resulted in a form of UhpA that not only was unable to activate uhpT transcription , but even interfered with the function of a resident wild-type version of UhpA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8144492" target="_blank">8144492</a>]<br></li>
<li>By analogy with other two-component systems , we suspect that the presence of Glu6P in the medium triggers the phosphorylation pathway whereby UhpB phosphorylates itself on the invariant histidine residue and allows transfer of that phosphate to UhpA , which becomes activated to allow transcription at the uhpT promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8144492" target="_blank">8144492</a>]<br></li>
<li>The UhpA protein is a response-regulator protein whose action is absotutety required for transcription at the uhpT promoter Its activity is controiied by two membrane proteins : UhpB , whioh is retated in sequence to the sensor kinases ; and UhpC , which is related in sequence to UhpT and simitar transport proteins ( Isiand et ai .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7596290" target="_blank">7596290</a>]<br></li>
<li>It is more likely that UhpB protein serves to alter UhpA covalently to convert it to a form that stimulates uhpT transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3042748" target="_blank">3042748</a>]<br></li>
<li>Previous studies have required for shown that all three uhp regulatory genes are in uhpT expression and support a model for uhp regulation membrane-localized UhpB and UhpC proteins which the respond to the presence of external Glu6P and convert the protein form able activate uhpT transcription UhpA to a to 31 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1569007" target="_blank">1569007</a>]<br></li>
<li>Based on these results and the properties of the predicted Uhp polypeptides , we proposed that UhpA is a positive activator of uhpT transcription and that UhpB and UhpC are membrane proteins that detect the presence of external G6P and regulate the ability of UhpA to stimulate uhpT transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3042748" target="_blank">3042748</a>]<br></li>
<li>These results led to a model of regulation in which the UhpB and UhpC regulatory proteins prevent the ability of UhpA to activate transcription of the uhpT gene under noninducing conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3038843" target="_blank">3038843</a>]<br></li>
<li>It is likely , but not yet proven , that occupancy by UhpA of the ÿ35 region is necessary for transcription activation , because base changes there result in complete loss of promoter function ( Merkel et al. , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10512697" target="_blank">10512697</a>]<br></li>
<li>Phosphorylated UhpA exhibits enhanced afinity for speciﬁc target sequences in the uhpT promoter region ( Dahl et al. , 1997 ; Merkel et al. , 1992 ) , through which it can stimulate UhpT expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11739766" target="_blank">11739766</a>]<br></li>
<li>Finally , our quantiﬁcation of the in vivo concentration of UhpA ( 2 -- 3 µM ; unpublished data ) , in combination with the recently reported afinity of ( P C ) UhpA for the promoter region of uhpT ( Chen & Kadner , 2000 ; Olekhnovich et al. , 1999 ) , implies that the uhpT promoter region is already saturated with UhpA even in the absence of extracellular G6P .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11739766" target="_blank">11739766</a>]<br></li>
<li>Addition of 5 mM cAMP partially restored uhpT promoter activity , and the degree of restoration was comparable in cells expressing normal or ampli ® ed levels of UhpA ( Figure 2 ( b ) , strains 6 and 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10512697" target="_blank">10512697</a>]<br></li>
<li>As shown by its high activity in the presence of an effective 235 region , the inactivity of the uhpT promoter in the absence of UhpA is the result of its weak 235 element and not the consequence of silencing activity , as proposed for the blocking by NarL of the silencing of the nir promoter by FIS ( factor for inversion stimulation ) and another protein ( 35 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10913075" target="_blank">10913075</a>]<br></li>
<li>It is unlikely that negative or silencing proteins participate in maintaining the extremely low basal promoter activity , because uhpT expression is still absolutely dependent on UhpA in the in vitro system in the absence of any other proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10512697" target="_blank">10512697</a>]<br></li>
<li>The 6-bp linker substitution at positions 251 to 256 in the downstream half of the 64 element eliminated uhpT promoter activity , increased the 2 concentration of UhpA necessary to-bind to DNA , and prevented the formation of the maximally retarded UhpA-DNA complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8999880" target="_blank">8999880</a>]<br></li>
<li>Here we report that phosphorylation of UhpA by acetyl phosphate occurs on Asp-54 , that UhpA binds specifically to target sequences in the uhpT promoter , and that phosphorylation strongly affects the ability of UhpA to-bind to the 264 element and a downstream region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8999880" target="_blank">8999880</a>]<br></li>
<li>Phosphorylation of UhpA Enhances Its DNA Binding Activity -- The effect of phosphorylation by acetyl phosphate on the ability of UhpA to-bind to the uhpT promoter region was examined in the electrophoretic mobility shift assay .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8999880" target="_blank">8999880</a>]<br></li>
<li>DNase I footprinting of UhpA at the uhpT promoter and in vitro transcription were assayed in the presence of GST and GST-Bc .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11053370" target="_blank">11053370</a>]<br></li>
<li>The uhpT-speciﬁc in vitro transcription activated by 220 nM UhpA was completely blocked by the presence of 200 nM GST-Bc but was unaffected by GST ( Fig. 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11053370" target="_blank">11053370</a>]<br></li>
<li>GST-Bc inhibits DNA binding and transcription activation by UhpA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11053370" target="_blank">11053370</a>]<br></li>
<li>The liberated HK domain carried in GST-Bc was found to prevent both the binding of UhpA at the uhpT promoter and transcription activation in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11325944" target="_blank">11325944</a>]<br></li>
<li>This result suggests that binding of UhpA to the promoter must precede the binding of RNAP for effective transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10512697" target="_blank">10512697</a>]<br></li>
<li>We can not detect an effect of UhpA on RNAP binding to the promoter , because there is no obvious DNase footprint by RNAP even in the presence of all transcription factors ( I.N.O. & R.J.K. , unpublishe result ) , and the results of gel shift experiments with RNAP are not interpretable owing to its strong binding to DNA ends .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10512697" target="_blank">10512697</a>]<br></li>
<li>Overall binding of UhpA is relatively weaker than that of other transcription factors , as indicated by the concentrations of UhpA needed for activity in gel shift , footprinting , and in vitro transcription assays .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10512697" target="_blank">10512697</a>]<br></li>
<li>The pattern of in vitro binding by UhpA agreed with the in vivo observations that phosphorylation-in-dependent assembly of the transcription initiation complex can occur at elevated concentrations of UhpA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8999880" target="_blank">8999880</a>]<br></li>
<li>Assuming that the constitutive expression conferred by multicopy uhpA re ¯ ects the functional saturation of the UhpA-binding sites in the uhpT promoter , we conclude that CAP does not act solely to increase binding of UhpA to its target sites , but must stimulate transcription in a manner separate from the action of UhpA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10512697" target="_blank">10512697</a>]<br></li>
<li>Overexpression of UhpA did not reduce the degree of CAP stimulation of uhpT-lacZ expression , showing that CAP action is more complex than enhancement of the binding of UhpA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10512697" target="_blank">10512697</a>]<br></li>
<li>Results Overexpression of UhpA does not decrease the stimulation by CAP If the binding of CAP to the uhpT promoter serves only to facilitate the binding of UhpA , then the degree of stimulation of uhpT expression by CAP should be reduced when UhpA levels are increased .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10512697" target="_blank">10512697</a>]<br></li>
<li>The binding of CAP confers a slight increase of UhpA binding to its upstream sites , and the stimulation of the uhpT promoter by CAP appears to require the functioning of a-CTD ( 24 ) but not the activating surfaces used at other CAP-dependent promoters ( 21 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10913075" target="_blank">10913075</a>]<br></li>
<li>Binding of UhpA and CAP to the uhpT promoter DNase-footprinting techniques were used to examine directly whether CAP or UhpA affected the af ® nity or position of binding of the other protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10512697" target="_blank">10512697</a>]<br></li>
<li>DISCUSSION The UhpA protein was overexpressed and purified to show its specific binding to DNA sequences in the uhpT promoter region and to compare its binding properties with those expected from the regulation of uhpT expression in the intact cell .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8999880" target="_blank">8999880</a>]<br></li>
<li>The titration of chromosomally encoded UhpA by the uhpT promoter on multicopy plasmids results in a greatly decreased ability to utilize Fru6P as the sole carbon source .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1569008" target="_blank">1569008</a>]<br></li>
<li>The ® nding indicates that CAP can not act solely to help UhpA bind to the promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10512697" target="_blank">10512697</a>]<br></li>
<li>The speciﬁc transcription activator UhpA is absolutely required for transcription activity , and in its absence , no promoter activity is detected ( 21 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7896692" target="_blank">7896692</a>]<br></li>
<li>The absence of a ÿ35 element to specify proper RNAP binding in the uhpT promoter is probably replaced by the binding of UhpA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10512697" target="_blank">10512697</a>]<br></li>
<li>The number of UhpA molecules that bind to the promoter is not known , but the extent of the activator-binding regions in the uhpT promoter appear to be simpler than for other response-regulators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8999880" target="_blank">8999880</a>]<br></li>
<li>The action of CAP bound at -103 in the uhpT is thus expected to require the participation promoter of UhpA as a transcriptional-activator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1569008" target="_blank">1569008</a>]<br></li>
<li>The uhpT promoter was activated by not only its cognate UhpA but also a noncognate UvrY ( see Table 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26010043" target="_blank">26010043</a>]<br></li>
<li>For example , at the uhpT promoter , CRP binds at position 2103.5 and acts in conjunction with the uhp-speciﬁc activator , UhpA , bound at position 264 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10852886" target="_blank">10852886</a>]<br></li>
<li>Upon phosphoryl transfer , the response-regulator UhpA ( presumably phosphorylated on the conserved Asp-54 ) shows enhanced afﬁnity for the uhpT promoter and , together with the catabolite gene activator protein , initiates uhpT transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12107138" target="_blank">12107138</a>]<br></li>
<li>It remains to be determined where the UhpA protein binds in the uhpT promoter region and whether its activity is regulated by some covalent modification or by its release from the membrane .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3301805" target="_blank">3301805</a>]<br></li>
<li>Binding of UhpA to the 232 region of the uhpT promoter appears to be important for its action .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10572130" target="_blank">10572130</a>]<br></li>
<li>To better understand the binding and action of UhpA , we analyzed a systematic set of base substitutions throughout the UhpA-binding region in the uhpT promoter for their effect on UhpA occupancy and transcription activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11976297" target="_blank">11976297</a>]<br></li>
<li>Subsequently , this phosphoryl group is transferred to a conserved aspartate residue in the response-regulator UhpA , which then initiates uhpT transcription , via binding to the uhpT promoter region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11739766" target="_blank">11739766</a>]<br></li>
<li>Activated ( i.e. , phosphorylated ) UhpA binds to the promoter region of uhpT , resulting in initiation of transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12107138" target="_blank">12107138</a>]<br></li>
<li>Phosphorylation of UhpA strongly increases but is not required for binding to the 80 to 32 region of the uhpT promoter ( all promoter coordinates are relative to the transcription start site ) ( 9 , 10 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11976297" target="_blank">11976297</a>]<br></li>
<li>This may imply that maximal in vivo P C UhpA levels are low and/or that , to keep sufﬁcient P C UhpA accumulated to induce uhpT transcription , the uhpT promoter DNA itself is involved in stabilization/sequestration of P C UhpA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11739766" target="_blank">11739766</a>]<br></li>
<li>Perhaps owing to the absence of a 235 element , the uhpT promoter is absolutely dependent on phosphorylated UhpA ( P-UhpA ) , but unphosphorylated UhpA can activate transcription when it is overexpressed or altered by certain mutations .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10913075" target="_blank">10913075</a>]<br></li>
<li>We have previously shown that UhpA can bind to sites in the uhpT promoter and activate its transcription in an in vitro system with puriﬁed components .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11053370" target="_blank">11053370</a>]<br></li>
<li>Dependence of in vitro uhpT transcription on concentration of UhpA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11976297" target="_blank">11976297</a>]<br></li>
<li>Phosphate transfer from UhpB to UhpA stimulates the ability of UhpA to-bind at the uhpT promoter and to activate its transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11325944" target="_blank">11325944</a>]<br></li>
<li>The GST fusion protein inhibited the ability of UhpA to-bind and activate transcription at the uhpT promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11053370" target="_blank">11053370</a>]<br></li>
<li>GST-Bc inhibited the ability of UhpA to-bind and to activate the in vitro transcription at the uhpT promoter ; both processes were measured in the absence of phosphorylation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11053370" target="_blank">11053370</a>]<br></li>
<li>Involvement of the amino-terminal phosphorylation module of UhpA in activation of uhpT transcription in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14996819" target="_blank">14996819</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11053370" target="_blank">11053370</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12107138" target="_blank">12107138</a>]<br></li>
<li>Protein phosphorylation affects binding of the Escherichia coli transcription activator UhpA to the uhpT promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11976297" target="_blank">11976297</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11325944" target="_blank">11325944</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12107134" target="_blank">12107134</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10512697" target="_blank">10512697</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11739766" target="_blank">11739766</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10913075" target="_blank">10913075</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10572130" target="_blank">10572130</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11053370" target="_blank">11053370</a>]<br></li>
<li>It has been shown previously in vitro that binding of UhpA to the uhpT promoter exhibits positive cooperativity and that binding occurs with much higher afﬁnity when UhpA is phosphorylated ( 5 , 6 , 22 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12107138" target="_blank">12107138</a>]<br></li>
<li>Phosphorylated UhpA exhibits a strongly increased afﬁnity for the uhpT promoter ( 5 , 6 ) , implying increased positive cooperativity ( 6 ) , possibly approaching the upper value of this range .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12107138" target="_blank">12107138</a>]<br></li>
<li>Thus , regulated phosphorylation of the response-regulator UhpA enhances its occupancy of the weak sites where favorable contacts can allow the binding of RNA polymerase to the promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10913075" target="_blank">10913075</a>]<br></li>
<li>Occupancy of the UhpA-binding sites in the uhpT promoter shows behavior similar to that of OmpR and BvgA , namely , different afﬁnities for strong and weak sites , apparently cooperative occupancy of the weak sites , and a marked increase in afﬁnity for all sites upon phosphorylation of UhpA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10913075" target="_blank">10913075</a>]<br></li>
<li>Phosphorylation of UhpA ( Fig. 1 ) markedly increased its binding to the uhpT promoter , with half-maximal protection of all sites at 5 to 8 nM P-UhpA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10913075" target="_blank">10913075</a>]<br></li>
<li>Thus , phosphorylation of UhpA may enhance cooperativity between UhpA monomers at the uhpT promoter to enable formation of a maximally retarded gel complex and extension of the binding to the low affinity site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8999880" target="_blank">8999880</a>]<br></li>
<li>UhpA can be phosphorylated on aspartate-54 by acetyl-phosphate , and phosphorylation stimulates its binding to the uhpT promoter ( 2 , 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11053370" target="_blank">11053370</a>]<br></li>
<li>Effect of the phosphorylation of UhpA on binding to sites in the uhpT promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10913075" target="_blank">10913075</a>]<br></li>
<li>To examine the effect of protein phosphorylation on the binding of UhpA to target sequences in the uhpT promoter region , the UhpA protein was overexpressed and purified .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8999880" target="_blank">8999880</a>]<br></li>
<li>When the number of promoter copies is increased by plasmid carriage , the promoters on some plasmids are occupied and others are not , but now phosphor-ylation of UhpA allows the binding of all available UhpA molecules , leading to the further inducibility .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10913075" target="_blank">10913075</a>]<br></li>
<li>In the presence of AcP , however , the binding afﬁnity to the cognate promoters signiﬁcantly increased for EvgA , NarL , UhpA , and UvrY , but little enhancement was observed for NarP and RcsB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26010043" target="_blank">26010043</a>]<br></li>
<li>In the presence of acetyl phosphate for in vitro phosphorylation of RRs , the promoter-binding afﬁnity was signiﬁcantly enhanced for EvgA , NarL , UhpA , and UvrY .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26010043" target="_blank">26010043</a>]<br></li>
<li>In the bipartite activation model , UhpA bound to the downstream W region might allow recruitment and positioning of RNAP through interaction with - CTD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11976297" target="_blank">11976297</a>]<br></li>
<li>Gel-mobility shift assays indicated the low affinity binding of UhpA to sequences downstream of the 264 element .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8999880" target="_blank">8999880</a>]<br></li>
<li>Binding to the 264 element exhibited positive cooperativity and was stimulated severalfold by phosphorylation of UhpA , whereas extension to the downstream region was more strongly affected by phosphorylation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8999880" target="_blank">8999880</a>]<br></li>
<li>Binding of UhpA to the 264 element appears to be less strongly affected by its phosphorylation than was the ability to extend the binding complex to the downstream region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8999880" target="_blank">8999880</a>]<br></li>
<li>These results are consistent with the premise that occupancy and activation from the strong sites are less dependent on UhpA phosphorylation than is activation from the weak site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10913075" target="_blank">10913075</a>]<br></li>
<li>Occupancy of the ÿ50 to ÿ32 region is weaker than occupancy of the strong sites from ÿ80 to ÿ50 , but is stimulated by phosphorylation of UhpA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10512697" target="_blank">10512697</a>]<br></li>
<li>DNase I and hydroxyl radical footprinting suggested that lower concentrations of UhpA were required for occupancy of the strong binding sites between positions 280 and 250 than for occupancy of the weak sites between 250 and 232 , but the relative afﬁnities and the effect of UhpA phosphorylation were not quantiﬁed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10913075" target="_blank">10913075</a>]<br></li>
<li>This behavior ﬁts the initial premise that the strong sites are at least partially occupied by UhpA in the absence of its phosphorylation , as was suggested from the binding process in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10913075" target="_blank">10913075</a>]<br></li>
<li>UhpA-dependent activity under noninducing conditions appears to result from the binding of unphosphorylated UhpA to the strong sites , which are now in the position normally occupied by the weak sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10913075" target="_blank">10913075</a>]<br></li>
<li>These results showed that the presence of the weak sites did not affect the binding of UhpA to the strong sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10913075" target="_blank">10913075</a>]<br></li>
<li>As was seen for UhpA , the binding of CAP to its target sites was increased about twofold by the presence of UhpA ( Figure 3 ( b ) ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10512697" target="_blank">10512697</a>]<br></li>
<li>Phosphorylation of UhpA strongly increased its afﬁnity for both sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10913075" target="_blank">10913075</a>]<br></li>
<li>Phosphor-ylation of UhpA greatly increased site occupancy and afﬁnity , especially of the weak sites , but was not necessary for binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10913075" target="_blank">10913075</a>]<br></li>
<li>CAP conferred a moderate increase ( ca twofold ) in the ability of UhpA to-bind within the ÿ64 region , but had no apparent effect on UhpA binding from positions ÿ32 to ÿ48 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10512697" target="_blank">10512697</a>]<br></li>
<li>Thirdly , CAP may reposition UhpA bound to uhpTp rather than stabilize binding of the activator , as is seen with CAP and MalT interaction at the malK promoter ( 32 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7896692" target="_blank">7896692</a>]<br></li>
<li>If UhpA action depends on its binding to the ÿ35 region of the promoter , it is likely that bound UhpA contacts RNAP to affect its binding or activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10512697" target="_blank">10512697</a>]<br></li>
<li>UhpA bound to the upstream S region could enhance binding of UhpA to the W region and interact directly with CAP and - CTD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11976297" target="_blank">11976297</a>]<br></li>
<li>Perhaps these effects on protein binding are mediated by changes in DNA conformation rather than CAP-UhpA contact , since CAP binding induces the appearance of two hydroxyl radical-hypersensitive sites at positions ÿ62 and ÿ43 in the UhpA-binding region ( J.L.D. , unpublished results ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10512697" target="_blank">10512697</a>]<br></li>
<li>Second , activation by unphosphorylated UhpA occurred because its strong sites are adjacent to the RNAP-binding site ; this activation required the proximity of CAP and UhpA on the DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10913075" target="_blank">10913075</a>]<br></li>
<li>Thus , CAP and UhpA have a modest effect on the binding of the other protein to adjacent sites , but CAP does not appear to facilitate binding of UhpA to the sites extending to ÿ32 which may be the sites relevant for transcription activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10512697" target="_blank">10512697</a>]<br></li>
<li>DISCUSSION This study provides information regarding the recognition sequences for UhpA binding , the number and orientation of bound UhpA molecules , the contribution of UhpA-binding sites to transcription activation , and the existence of cooperativity in UhpA binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11976297" target="_blank">11976297</a>]<br></li>
<li>Based on these results , we tested whether UhpA activation was affected by the helical phasing between the strong and weak sites and whether deletion of the weak sites to bring the strong sites into proximity to the RNAP-binding region would allow UhpA to activate uhpT transcription without the need for phosphorylation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10913075" target="_blank">10913075</a>]<br></li>
<li>Transcription activation by the phosphorylation-independent binding of UhpA occurs only when the strong sites are near the RNAP-binding region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10913075" target="_blank">10913075</a>]<br></li>
<li>Thus , binding of UhpA to the W region is crucial for UhpA-dependent activation and depends on occupancy of the S region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11976297" target="_blank">11976297</a>]<br></li>
<li>This stimulation affected only UhpA binding to its strong sites and not to the ÿ35 region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10512697" target="_blank">10512697</a>]<br></li>
<li>At higher concentrations of UhpA , binding was extended to the 232 region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8999880" target="_blank">8999880</a>]<br></li>
<li>The ability to compete for UhpA was eliminated by the substitutions for -51 to -56 and -72 to -77 , which are within the -64 element , confirming the importance of the -64 element for UhpA binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1569008" target="_blank">1569008</a>]<br></li>
<li>The cooperative binding of UhpA molecules is suggested by the sigmoid dependence of DNA binding and transcription activation on the concentration of UhpA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11976297" target="_blank">11976297</a>]<br></li>
<li>The cooperative but low-afﬁnity binding of UhpA to its DNA targets complicates further mutational veriﬁcation of this binding consensus .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11976297" target="_blank">11976297</a>]<br></li>
<li>The weak binding of RNAP may reﬂect the weak binding of UhpA to its target sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10572130" target="_blank">10572130</a>]<br></li>
<li>The different levels of basal and induced activity in the D20 , D21 , and D22 variants suggest that the helical phasing between the binding sites for UhpA and RNAP may be important for activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10913075" target="_blank">10913075</a>]<br></li>
<li>The W region substitutions M9 , M11 , and M12 were strongly impaired for UhpA binding and for transcription driven by UhpA and RNAP , suggesting that UhpA occupancy of the W region was crucial for activation by UhpA in the absence of CAP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11976297" target="_blank">11976297</a>]<br></li>
<li>The S region variants M3 to M5 , which decreased UhpA binding , also decreased binding to the W region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11976297" target="_blank">11976297</a>]<br></li>
<li>The presence of CAP partly overcomes the decreased promoter activity of W region variants or RpoD mutants , but we have not found a defect in CAP activation due to impaired binding of UhpA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11976297" target="_blank">11976297</a>]<br></li>
<li>Unlabeled compet-itor DNA was added in H2O , dried by lyophilization in the reaction tube , and dissolved in binding assay components before addition of UhpA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8999880" target="_blank">8999880</a>]<br></li>
<li>Binding of UhpA to its DNA target displayed unusual behavior in the electrophoretic mobility shift assay .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8999880" target="_blank">8999880</a>]<br></li>
<li>When RcsA is limiting [ Ion * cells ) and RcsB is not activated by phosphorylation , the cps region is not transcribed The predicted protein sequences of both RcsA and RcsB contain a carboxy-terminal domain similar to that found in a class of DNA-binding proteins exemplified by UhpA , MalT , and FixJ ( Fig. 1 ; Stout etal .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1943696" target="_blank">1943696</a>]<br></li>
<li>It is likely that the weak yet speciﬁc binding of UhpA to the DNA is a general adaptation designed to allow genome-bound transcription factors to maintain communication with their cognate membrane-bound sensor kinase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10572130" target="_blank">10572130</a>]<br></li>
<li>It might be interesting to consider possible explanations for why expression of uhpT , which is so tightly and speci ® cally regulated , relies on the weak binding of UhpA and RNAP to their DNA targets .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10512697" target="_blank">10512697</a>]<br></li>
<li>One possibility is that the truncated species forms a mixed oligomer with the wild-type version of UhpA that is inactive for DNA binding or activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8144492" target="_blank">8144492</a>]<br></li>
<li>To understand the basis for the lack of further induction when uhpA is multicopy will require information about the nature of the activation of UhpA and the effect of this putative modification on the DNA-binding properties of the protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3042748" target="_blank">3042748</a>]<br></li>
<li>Acetyl phosphate had no apparent effect upon the DNA binding activity of the UhpA [ D54N ] protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8999880" target="_blank">8999880</a>]<br></li>
<li>One interpretation of these findings is that phosphorylation of UhpA might enhance its binding to its target sequences , but that DNA binding or formation of the active structure on the DNA could be achieved by unphosphorylated UhpA at elevated concentrations .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8999880" target="_blank">8999880</a>]<br></li>
<li>Perhaps the interdomain interaction in UhpA does not occlude the DNA-binding site , but the conformational change induced by phosphorylation might enhance the interaction on the DNA surface of UhpA monomers .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8999880" target="_blank">8999880</a>]<br></li>
<li>Tight binding of UhpA to DNA , typical of the regulatory proteins that are modulated by small-molecule effectors , would prevent this communication .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10512697" target="_blank">10512697</a>]<br></li>
<li>Hence , loose binding of P-UhpA to its target sequences may be designed to allow its ready dissociation from the DNA for diffusion to UhpB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10512697" target="_blank">10512697</a>]<br></li>
<li>Since this cutting did not depend on the iron atom attached near the DNA-binding surface , it may indicate that UhpA binding distorts the DNA and increases the effectiveness of hydroxyl radical attack .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11976297" target="_blank">11976297</a>]<br></li>
<li>These include changes in DNA conformation , effects on RNA polymerase binding and the formation of the closed promoter complex , stimulation or stabili-zation of the transition to open complexes , and possibly acceleration of promoter clearance ( Busby & Ebright , 1994 ; Hochschild & Dove , 1998 ; kinase UhpB , together control the level of phosphorylation of the response-regulator UhpA ( Island & Kadner , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10512697" target="_blank">10512697</a>]<br></li>
<li>It appears , therefore , that both protein concentration or relative stoichiometry and phosphorylation are important parameters for UhpA activity , and that the requirement for phosphorylation of UhpA is lost when it is overexpressed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8999880" target="_blank">8999880</a>]<br></li>
<li>The site of phosphorylation of UhpA is not necessary for uhp activation by overexpressed UhpA but is necessary for UhpA action at normal levels of UhpA or for interference by the truncated species .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7596290" target="_blank">7596290</a>]<br></li>
<li>The D54N variant of UhpA , in which aspartate 54 at the putative site of phosphorylation is replaced with asparagine , is totally inactive for Uhp expression when in single gene copy , but is as active as the wild-type protein when overexpressed ( 17 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8999880" target="_blank">8999880</a>]<br></li>
<li>In DNase footprinting experiments , occupancy of the strong-site sequences in the wild-type and D22 promoter fragments showed similar dependence on the concentration of UhpA and the same increase in occupancy in response to phosphorylation of UhpA ( Fig. 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10913075" target="_blank">10913075</a>]<br></li>
<li>Although it is dif ® cult to quantify the concentration-dependent behavior of UhpA owing to its low solubility , experiments are in progress to de ® ne the effect of phosphorylation on UhpA function .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10512697" target="_blank">10512697</a>]<br></li>
<li>( b ) Time course of ( de ) phosphorylation of UhpA ( open symbols , regular lines ) and UhpB ( solid symbols , dotted lines ) , at increasing UhpA : UhpB ratios .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11739766" target="_blank">11739766</a>]<br></li>
<li>Cellular expression studies showed that the roughly tenfold activation of uhpT expression by CAP still occurred in cells in which the level of UhpA was increased suf ® ciently to allow constitutive , phosphorylation-independent activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10512697" target="_blank">10512697</a>]<br></li>
<li>Overexpression of UhpA resulted in high level , constitutive uhpT expression and loss of the normal requirements for inducer , UhpBC function , and the site of phosphorylation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8999880" target="_blank">8999880</a>]<br></li>
<li>The structure of the related protein NarL provides a framework to model transcription activation functions , although NarL and UhpA differ in the effect of phosphorylation on function .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11976297" target="_blank">11976297</a>]<br></li>
<li>Occupancy of the weak sites may be critical for transcription activation and may occur by oligomerization of UhpA molecules along the DNA in response to its phosphorylation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10913075" target="_blank">10913075</a>]<br></li>
<li>These proteins are thought to regulate the phosphorylation of the transcription activator , UhpA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8999880" target="_blank">8999880</a>]<br></li>
<li>Overexpression of UhpA allows phosphorylation-independent transcription activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11976297" target="_blank">11976297</a>]<br></li>
<li>Therefore , in G6P-induced cells phosphorylation of a relatively small fraction of the total amount of UhpA may well be suficient to induce uhpT transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11739766" target="_blank">11739766</a>]<br></li>
<li>In wild-type cells , uhpT transcription is absolutely controled by the phosphorylation state of UhpA , which is determined by the membrane-bound UhpBC signaling complex in response to external Glu6P .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10512697" target="_blank">10512697</a>]<br></li>
<li>When present in a single copy , the deletion promoters can be fully occupied by UhpA , so that phosphorylation of UhpA does not increase binding and gene expression noticeably .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10913075" target="_blank">10913075</a>]<br></li>
<li>Future studies will address the effect of these changes on UhpA binding and test whether the access of the histidine to phosphorylation can be affected by the binding of inducer and the presence of UhpC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11325944" target="_blank">11325944</a>]<br></li>
<li>These data show that phosphorylation of UhpA strongly affects its ability to-bind to its target DNA sequences , and are consistent with the conclusion that the requirement for phosphorylation can be circumvented by overexpression of UhpA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8999880" target="_blank">8999880</a>]<br></li>
<li>Some response-regulators oligomerize upon phosphorylation and then bind to DNA in their oligomerized forms ( e.g. , NtrC and UhpA ) ( 38 , 40 , 41 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14996819" target="_blank">14996819</a>]<br></li>
<li>Frequent dissociation of UhpA from the DNA can allow adjustment of its level of phosphorylation to reﬂect the current state of occupancy of the receptor for Glu6P and hence to allow efﬁcient regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10572130" target="_blank">10572130</a>]<br></li>
<li>In the case of UhpA , the primary effect of phosphorylation seems to affect its ability to extend cooperatively along the DNA helix to the downstream sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8999880" target="_blank">8999880</a>]<br></li>
<li>Lik NarL , it binds DNA following phosphorylation of the same aspartic acid : Asp54 in UhpA vs Asp59 in NarL .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9521685" target="_blank">9521685</a>]<br></li>
<li>Studies are under way to identify the effect of phosphorylation on UhpA structure and activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8999880" target="_blank">8999880</a>]<br></li>
<li>Subsequently , upon addition of UhpA , very rapid and transient UhpA phosphorylation takes place .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11739766" target="_blank">11739766</a>]<br></li>
<li>RESULTS Effect of UhpA phosphorylation on site occupancy .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10913075" target="_blank">10913075</a>]<br></li>
<li>Activation by UhpA does not require its phosphorylation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10512697" target="_blank">10512697</a>]<br></li>
<li>UhpA binds to a dyad element centered at position 264 , and phosphorylated UhpA can then bind to low-afﬁnity sites that extend to position 232 ( 7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10572130" target="_blank">10572130</a>]<br></li>
<li>It could bind UhpA and remove it from the vicinity of the genome .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3042748" target="_blank">3042748</a>]<br></li>
<li>This result suggests that either half of the -64 element can bind UhpA , but not as well as the intact element .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1569008" target="_blank">1569008</a>]<br></li>
<li>can bind UhpA , but not as well as the intact element .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8999880" target="_blank">8999880</a>]<br></li>
<li>Since UhpA binds near the 235 region , we expected that UhpA might contact the CTD of RpoD , as found for some transcription activators that bind in a similar position .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10572130" target="_blank">10572130</a>]<br></li>
<li>UhpA differs from NarL in that phosphoryl-ation of UhpA is not required for its ability to-bind DNA and activate transcription ( Dahl et al. , 1997 ; J. Li et al. , 1994 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10512697" target="_blank">10512697</a>]<br></li>
<li>UhpA binds to multiple sites between positions ÿ80 and ÿ32 ( Dahl et al. , 1997 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10512697" target="_blank">10512697</a>]<br></li>
<li>UhpA induced the appearance of DNase-hypersensi-tive cleavage sites at 58 on the top strand and at 80 , 74 , FIG. 3 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11976297" target="_blank">11976297</a>]<br></li>
<li>Extension of the footprint to the low affinity site ( 250 to 232 ) occurred with P-UhpA at 140 nM , but required much higher concentrations of unmodified UhpA ( 1.3 mM ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8999880" target="_blank">8999880</a>]<br></li>
<li>When UhpA was added last , Vm was only 75 % of the value when RNAP was added last , suggesting that a substantial proportion of the template molecules are inactivated or form a dead-end complex when RNAP is allowed to interact with promoter DNA in the absence of UhpA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10512697" target="_blank">10512697</a>]<br></li>
<li>This model is difficult to reconcile with our finding that unphosphorylated UhpA binds to specific targets in DNA , although with somewhat lower affinity than does P-UhpA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8999880" target="_blank">8999880</a>]<br></li>
<li>Two possibilities are that aCTD interacts directly with both CAP and UhpA or that aCTD binds to nonspeciﬁc DNA upstream of the CAP site which is brought into reach of RNAP by the DNA-bending activity of CAP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10572130" target="_blank">10572130</a>]<br></li>
<li>Because formation of the hypersensitive sites required aCTD but not upstream DNA , we favor the view that aCTD binds between the CAP and UhpA sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10572130" target="_blank">10572130</a>]<br></li>
<li>Following incubation with P-UhpA , portions were removed at 0 , 5 , 10 , 20 , and 30 min and analyzed as described for panel A. Hydrolysis of P-UhpA was determined in the absence of 15 mM EDTA ( panels 1 and 3 ) and in its presence ( panels 2 and 4 ) phosphotransfer has been observed for other HK-RR pairs in the absence of EDTA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11325944" target="_blank">11325944</a>]<br></li>
<li>f DNA cleavage of the S or W regions following incubation of DNA with FeBABE-modiﬁed UhpA proteins in the presence of hydrogen peroxide and ascorbate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11976297" target="_blank">11976297</a>]<br></li>
<li>In panels A and B , lane 1 shows the product of the afﬁnity cleavage reaction following addition of hydrogen peroxide and ascorbate with wild-type UhpA-FeBABE ; lane 2 shows the afﬁnity cleavage products with UhpA E187C-FeBABE ; lane 3 shows the DNase I cleavage products in the absence of UhpA proteins ; lanes 4 and 5 show DNase protection reactions in the presence of wild-type UhpA-FeBABE and UhpA E187C-FeBABE , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11976297" target="_blank">11976297</a>]<br></li>
<li>Promo-ter fragment D1 ( ÿ250 to 50 ) , labeled with 32P on the 50 end of the top strand , was incubated with the indicated concentrations of UhpA or CAP , in the absence and presence of saturating concentrations of the other protein , namely , 140 nM CAP or 800 nM UhpA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10512697" target="_blank">10512697</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10512697" target="_blank">10512697</a>]<br></li>
<li>After one or ten minutes at 37 C , 5 ml of TXN buffer containing NTP substrates and heparin was added to yield the following ® nal concentrations : 1 nM DNA template ; 14-220 nM UhpA ; 20 nM CAP ; 35 nM RNAP ; 50 mg/ml heparin ; NTPs , which are ATP , CTP , GTP , 200 mM ; and 40 mM [ a32P ] UTP ( 2.5 Ci/nmol ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10512697" target="_blank">10512697</a>]<br></li>
<li>Abortive initiation assays and kinetic analysis For lag-time assays , DNA template ( PT380 ) , UhpA or RNAP , CAP , CpU and [ a-32P ] CTP were incubated for ten minutes in 143 ml of TXN buffer at 37 C and combined with 7 ml of RNAP or UhpA at time zero to yield the following ® nal concentrations : 2 nM DNA template ; 220 nM UhpA ; 20 nM CAP ; 10-35 nM RNAP ; 0.5 mM CpU ; 50 mM [ a-32P ] CTP ( 0.4 to 0.8 Ci/nmol ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10512697" target="_blank">10512697</a>]<br></li>
<li>To test the effect of order of addition of UhpA and RNAP , the abortive initiation assay was started by adding UhpA to the DNA template that was preincubated with saturating levels of RNAP CpU [ a-32P ] CTP with or without CAP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10512697" target="_blank">10512697</a>]<br></li>
<li>Product formation by 30 nM RNAP plus 220 nM UhpA is indicated by open bars ; product formed with the additional presence of 20 nM CAP is indicated by solid bars .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11976297" target="_blank">11976297</a>]<br></li>
<li>Promoter templates , PT380 and supercoiled plasmid pUT1 DNA ( 1 nM ) , were preincubated at 37 C with 200 nM UhpA in the absence or presence of 20 nM CAP-cAMP as indicated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10512697" target="_blank">10512697</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10512697" target="_blank">10512697</a>]<br></li>
<li>They were incubated at 37 ° C in TXN buffer with 220 nM UhpA in the absence or presence of 20 nM CAP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11976297" target="_blank">11976297</a>]<br></li>
<li>The PT380 DNA template was ® rst incubated with saturating levels of RNAP or UhpA , in the absence and presence of CAP , for ten minutes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10512697" target="_blank">10512697</a>]<br></li>
<li>The productive initiation assay ( Material and Methods ) was used , in which the PT380 template DNA was preincubated with 30 nM RNAP or 220 nM UhpA , in the absence or presence of 20 nM CAP , as indicated , for ten minutes at 37 C .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10512697" target="_blank">10512697</a>]<br></li>
<li>The DNA was pre-incubated in the presence of ( a ) 220 nM UhpA , in the absence ( open symbols ) and presence ( ® lled symbols ) of 20 nM CAP , or ( b ) 30 nM RNAP , in the absence of presence of CAP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10512697" target="_blank">10512697</a>]<br></li>
<li>To measure the rate of dissociation of the open complex , RNAP was allowed to-bind to promoter DNA for 60 minutes in the presence of UhpA with or without CAP , followed by addition of poly [ d ( IC ) ] at concentrations from 10 to 100 mM .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10512697" target="_blank">10512697</a>]<br></li>
<li>When CAP and UhpA were pre-incubate with the promoter DNA for ® ve minutes , and RNAP was then added for one minute before NTPs and heparin , the presence of CAP stimulated formation of the 233 nt run-off transcript by a factor of 3.7 ( Figure 5 , lanes 1 and 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10512697" target="_blank">10512697</a>]<br></li>
<li>When the productive initiation reactions were initiated by the addition of RNAP to template that had been pre-incubated with UhpA in the absence and presence of CAP ( Figure 6 ( b ) ) , the t values were 3.3 and 1.0 minute , respectively ( Table 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10512697" target="_blank">10512697</a>]<br></li>
<li>When the reaction was initiated by the addition of UhpA to template DNA that had been pre-incu-bated with RNAP in the absence and presence of CAP ( Figure 6 ( a ) ) , the calculated t values were 5.7 and 1.9 minutes , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10512697" target="_blank">10512697</a>]<br></li>
<li>Fractional occupancy was calculated as -LCB- 1 2 [ ( band intensity in the presence of UhpA 2 background from empty lane ) / ( band intensity of nearby band 2 background ) ] / [ ( band intensity in the absence of UhpA 2 background ) / ( band intensity of nearby band 2 background ) ] -RCB- .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10913075" target="_blank">10913075</a>]<br></li>
<li>This rapid dephosphorylation occurred independently of the presence or absence of intra-vesicular G6P ( compare open and solid symbols ) and the absolute concentration of P C UhpA ( compare Fig. 4a and 4b ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11739766" target="_blank">11739766</a>]<br></li>
<li>The open promoter complexes formed in the pre-sence of UhpA dissociated rapidly in presence of heparin and more slowly with poly [ d ( IC ) ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10512697" target="_blank">10512697</a>]<br></li>
<li>The continual release of UhpA from the DNA may also be the basis for the instability of the open promoter complexes that are formed in the presence of UhpA alone .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10512697" target="_blank">10512697</a>]<br></li>
<li>Kinetic analyses of productive and abortive initiation showed that CAP acted both to stabilize by ® vefold the open promoter complexes formed in the presence of UhpA and to enhance by twofold the rate of their formation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10512697" target="_blank">10512697</a>]<br></li>
<li>Finding that the stimulatory effect of CAP may involve more than changes in UhpA binding , we demonstrated that open promoter complexes are formed only in the presence of UhpA , but are dissociated by poly-anions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10512697" target="_blank">10512697</a>]<br></li>
<li>The presence of 220 nM UhpA , as was used in the experiment in Fig. 3 , resulted in near-maximal rates of transcription at both promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11976297" target="_blank">11976297</a>]<br></li>
<li>The presence of UhpA resulted in protection of positions from ÿ32 to ÿ80 with different af ® nities .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10512697" target="_blank">10512697</a>]<br></li>
<li>The presence of 140 nM CAP did not affect the positions that were protected by UhpA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10512697" target="_blank">10512697</a>]<br></li>
<li>The values for k were 8 10ÿ4 sÿ1 r in the presence of UhpA , and 2 10ÿ4 sÿ1 in the presence of UhpA CAP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10512697" target="_blank">10512697</a>]<br></li>
<li>DNase I hypersensitivity occurred at positions 280 and 274 only in the presence of all three factors , i.e. , RNAP , UhpA , and CAP ( Fig. 6A , lane 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10572130" target="_blank">10572130</a>]<br></li>
<li>In the presence of UhpA and CAP , the steady-state rate of oligonucleotide synthesis was increased by a factor of 4 at each RNAP concentration .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10512697" target="_blank">10512697</a>]<br></li>
<li>The relative amounts of the uhpT transcript ( Txn level ) , determined by PhosphorImager quantitation , are presented relative to that for the wild-type RNAP in the presence of UhpA FIG. 6 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10572130" target="_blank">10572130</a>]<br></li>
<li>CAP - ( cAMP ) is required for high-level constitutive uhpT promoter activity in presence of overexpressed UhpA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10512697" target="_blank">10512697</a>]<br></li>
<li>Transcription of uhpT was measured in the presence of UhpA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10572130" target="_blank">10572130</a>]<br></li>
<li>Transcription of the uhpT promoter in vitro requires the presence of UhpA , but its phosphorylation is not required when it * Corresponding author .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11053370" target="_blank">11053370</a>]<br></li>
<li>As expected , uhpT expression from the supercoiled template required the presence of UhpA ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10512697" target="_blank">10512697</a>]<br></li>
<li>The expression of uhpT-lacZ was , however , not observed in the presence of all six RRs including UhpA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26010043" target="_blank">26010043</a>]<br></li>
<li>The a-CTD , which is necessary for substantial activation of uhpT transcription ( 24 ) , may contact both CAP and UhpA at the 280 region , as suggested by the formation of a DNase-hyper-sensitive site there when in the presence of all three proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10913075" target="_blank">10913075</a>]<br></li>
<li>CAP is required for efﬁcient transcription , but maximal in vitro transcription can occur in the absence of CAP with longer incubation of the template with UhpA and RNAP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10572130" target="_blank">10572130</a>]<br></li>
<li>Single-round transcription assays were carried out in the presence of 30 nM RNAP and indicated concentrations of UhpA , as described in Materials and Methods .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11976297" target="_blank">11976297</a>]<br></li>
<li>( B ) uhpT transcription ( Txn ) in the presence of UhpA without and with CAP as indicated , calculated relative to the amount of transcription produced by the wild-type RNAP in the presence of UhpA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10572130" target="_blank">10572130</a>]<br></li>
<li>Promoter activity of the uhpT variants was measured with single-round transcription assays with puriﬁed RNAP and UhpA in the absence or presence of CAP , as described previously ( 33 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11976297" target="_blank">11976297</a>]<br></li>
<li>DNA templates were preincubated at 25 ° C with UhpA in the absence and presence of CAP at concentrations previously found to be maximally stimulatory for uhpT transcription ( 30 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10572130" target="_blank">10572130</a>]<br></li>
<li>e uhpT-speciﬁc transcription driven by 30 nM RNAP -- 220 nM UhpA variant , in the presence or absence of 20 nM CAP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11976297" target="_blank">11976297</a>]<br></li>
<li>In all cases , expression of - galactosidase was fully dependent on the presence of UhpA and induction by Glc6P ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11976297" target="_blank">11976297</a>]<br></li>
<li>In the presence of UhpA alone , the lag time t values were in the range of four to six minutes ( Figure 7 ( c ) ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10512697" target="_blank">10512697</a>]<br></li>
<li>In the presence of RNAP and UhpA alone , the tobs value was 9 ( 1 ) minutes , compared to 5.7 minutes in the productive initiation assay and 3.9 minutes when RNAP is added last .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10512697" target="_blank">10512697</a>]<br></li>
<li>Sequence homology suggests that UhpB may be a histidine-protein kinase and , in the presence of Glu6P , may phosphorylate and activate UhpA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1569007" target="_blank">1569007</a>]<br></li>
<li>Dephosphorylation of P-UhpA appeared to be increased somewhat in the presence of 1 mM ATP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11325944" target="_blank">11325944</a>]<br></li>
<li>Unlike the situation with EnvZ and NtrB ( 14 , 22 , 31 ) , dephosphorylation of P-UhpA by GSTBc did not require the presence of ATP , although ATP may stimulate it somewhat ( but see reference 51 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11325944" target="_blank">11325944</a>]<br></li>
				</div></ul><br>
			<label class="collapse_DOM" for="_2"><font size="5px">UhpA protein domains</font></label>
				<input id="_2" type="checkbox">
				<div class="DOM"><br><ul><li>Additional fusions were obtained by in vivo transposition of TnphoA into an expression vector in which uhpBC was inserted downstream of the lac promoter , with amino acid 11 of LacZ fused to residue 189 of UhpA via the pUC8 polylinker .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1569007" target="_blank">1569007</a>]<br></li>
<li>These mutant alleles are designated by the position of the amino acid residue preceding the site of insertion and the number of amino acids inserted ; thus , the genotype uhpA15 :4 indicates that the encoded UhpA protein contains a four-amino-acid insertion between amino acids 15 and 16 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11053370" target="_blank">11053370</a>]<br></li>
<li>The only differences between these constructs and the fusions constructed in vitro were four amino acids inserted at UhpA residue 189 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1569007" target="_blank">1569007</a>]<br></li>
<li>His6-UhpA with an extension containing six His residues at the N terminus of UhpA ( 49 ) was phosphorylated with 20 to 25 mM acetyl [ 32P ] phosphate by incubation at 37 ° C for 2 to 3 h in assay buffer ( 50 mM HEPES [ pH 8.0 ] , 5 mM MgCl2 , 1 mM dithiothreitol ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11325944" target="_blank">11325944</a>]<br></li>
<li>Cysteine residues were introduced into UhpA or a variant carrying an N-terminal six-His extension , at positions 154 , 155 , 181 , 182 , 183 , 185 , 189 , and 191 ( Table 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11976297" target="_blank">11976297</a>]<br></li>
<li>Disruption of the 232 element with a 6-bp linker substitution eliminated promoter activity , but the 232 element did not compete for UhpA binding in vivo and its function is yet unknown .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8999880" target="_blank">8999880</a>]<br></li>
<li>Variant 7 ( 235 to 230 ) contains a linker substitution in the 232 element and exhibits complete loss of promoter activity , but gave a normal pattern of UhpA binding in the in vivo titration assay ( 18 ) and in this gel shift assay ( Fig. 7C ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8999880" target="_blank">8999880</a>]<br></li>
<li>These results indicate that open complex formation requires UhpA and that CAP stabilizes the open complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10512697" target="_blank">10512697</a>]<br></li>
<li>We are currently attempting to isolate mutants of CAP , UhpA , and RNA polymerase that are speciﬁcally affected in transcription-activation at uhpTp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7896692" target="_blank">7896692</a>]<br></li>
<li>Transcription activation by UhpA is stimulated by the catabolite gene activator protein ( CAP ) - cyclic AMP complex and depends on the C-terminal domains of the RNA polymerase RpoA and RpoD subunits .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11976297" target="_blank">11976297</a>]<br></li>
<li>It would only be necessary that the interaction with the extramembranous loops blocks some step of the phosphate transfer to UhpA , while allowing the phosphatase function to remain accessible and active .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8144492" target="_blank">8144492</a>]<br></li>
<li>Reverse transfer of phosphate from P-UhpA to GST-Bc was observed in the presence of the metal chelator EDTA and depended on the presence of His-313 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11325944" target="_blank">11325944</a>]<br></li>
<li>( B ) Reverse phosphotransfer was detected by incubation of P-UhpA with GST-Bc variants carrying substitutions in the H box ( H313E and H313Q ) or the catalytic domain ( H428Q ) ( 49 ) or with the nucleotide-binding domain removed FIG. 4 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11325944" target="_blank">11325944</a>]<br></li>
<li>Taken together , these results show that GST-Bc can accelerate dephosphorylation of P-UhpA and that reverse phosphotransfer to GST-Bc can be trapped in the presence of EDTA or the absence of the HK nucleotide-binding domain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11325944" target="_blank">11325944</a>]<br></li>
<li>Interestingly , the truncated species GST-Bc366 , which lacks the nucleotide-bind-ing domain , exhibited a low level of reverse phosphotransfer from P-UhpA in the absence of EDTA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11325944" target="_blank">11325944</a>]<br></li>
<li>The interference phenotype was localized to the C-terminal half of UhpB , since liberation of this HK domain from the transmembrane N-terminal half did not result in activation of UhpA but retained the strong dominant-negative interference .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11053370" target="_blank">11053370</a>]<br></li>
<li>The GST-Bc fusion carrying UhpB residues 273 to 366 ( GST-Bc366 ) and lacking the HK domain appeared to be able to dephosphorylate P-UhpA , but at a lower rate than the other GST-Bc proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11325944" target="_blank">11325944</a>]<br></li>
<li>The insertion at residue 288 affected the interference phenotype by activating UhpB function , hence affecting regulation rather than UhpA docking .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11325944" target="_blank">11325944</a>]<br></li>
<li>The restoration of the interference phenotype to these constitutively active UhpB variants following mutagenesis of conserved residues needed for autokinase activity , and the interference of UhpB with phosphorylation-independent variants of UhpA , indicate that the interference by UhpB involves both cophosphatase and sequestration activity on UhpA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11053370" target="_blank">11053370</a>]<br></li>
<li>The genetic requirements for interference indicated that the overexpressed receiver module competed with intact UhpA for phosphate residues carried on UhpB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7596290" target="_blank">7596290</a>]<br></li>
<li>Recently , the interaction of the transmitter domain of UhpB with UhpA has been described in detail ( Wright & Kadner , 2001 ; Wright et al. , 2000 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11739766" target="_blank">11739766</a>]<br></li>
<li>Thus , numerous positions in the putative helical regions of the phosphoryl transfer domain appear to affect both the regulation of autokinase activity and the ability to interact with UhpA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11325944" target="_blank">11325944</a>]<br></li>
<li>Taken together , these results indicate that the phosphoryl transfer-dimerization domain of UhpB participates in the speciﬁc binding of UhpA , in the control of autokinase activity , and in the dephosphorylation of P-UhpA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11325944" target="_blank">11325944</a>]<br></li>
<li>The interference by the phosphoryl transfer domains present in GST-Bc or GST-Zc with UhpA or OmpR action was highly speciﬁc : expression of the UhpB domain had no effect on OmpR function , or conversely , expression of the EnvZ domain had no effect on UhpA function .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11325944" target="_blank">11325944</a>]<br></li>
<li>Phosphate transfer from P-UhpA to the liberated phosphoryl transfer domain also occurred .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11325944" target="_blank">11325944</a>]<br></li>
<li>Our conclusion at this time is that the phosphoryl transfer domain of UhpB can recognize and bind UhpA and can accept phosphate from P-UhpA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11325944" target="_blank">11325944</a>]<br></li>
<li>The phosphoryl transfer domain of UhpB interacts with the response regulator UhpA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11739766" target="_blank">11739766</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12107138" target="_blank">12107138</a>]<br></li>
<li>To help localize the portion of GST-Bc ( containing UhpB residues 273 to 500 ) which is necessary for the inhibition of UhpA action , we took advantage of several Pstl linker insertions in this region ( 18 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11325944" target="_blank">11325944</a>]<br></li>
<li>GST-Bc variants with truncations of this region with fusion junctions at UhpB residues 282 , 288 , and 293 did not confer activation of GST-Bc kinase activity and still showed the strong interference phenotype , as did the GST-Bc ( 273 -- 500 ) species , indicating that the putative linker region is not involved in contact with UhpA or in keeping the kinase activity in a silent state .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11325944" target="_blank">11325944</a>]<br></li>
<li>This activating change does not appear to alter the interaction of UhpB with UhpA , since GST-Bc-R324C still showed a strong dominant-negative effect when its autokinase function was abrogated by the substitutions in conserved regions important for phosphotransfer activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11053370" target="_blank">11053370</a>]<br></li>
<li>The effect of several mutations affecting conserved residues in GST-Bc on the dephosphorylation of P-UhpA and on reverse phosphate transfer to GST-Bc was studied ( Fig. 3B ) using shorter time intervals than in the previous experiment .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11325944" target="_blank">11325944</a>]<br></li>
<li>Unlike the wild-type situation , a GST fusion variant carrying one of the UhpB-activating substitutions , R324C , displayed autokinase activity and phosphate transfer to UhpA but retained the ability to sequester UhpA when it was altered in the conserved residues important for phosphate transfer .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11053370" target="_blank">11053370</a>]<br></li>
<li>The GST-Bc R324C variant had not lost the ability to sequester UhpA , which was revealed by the restoration of the dominantnegative behavior when its autokinase activity was eliminated by mutations affecting conserved residues in the catalytic N or G boxes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11325944" target="_blank">11325944</a>]<br></li>
<li>In the representative comparison of close relatives of NarL shown in Figure 4 of reference 1 , DegU is seen to have an interdomain loop that is 13 residues longer than that of NarL , whereas UhpA and TrpO have loops that are 10 residues shorter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9521685" target="_blank">9521685</a>]<br></li>
<li>apparently shorter linker in UhpA and TrpO does not necessarily imply any difference in N and C domain geometry from that seen in NarL .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9521685" target="_blank">9521685</a>]<br></li>
<li>Purified UhpA was phosphorylated by acetyl phosphate in a reaction that was dependent on 21 Mg and on the presence of aspartate 54 , the site of phosphorylation in homologous response regulators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8999880" target="_blank">8999880</a>]<br></li>
<li>The RR UhpA , which is composed of a receiver-domain that belongs to the RC1 subfamily of receiver-domains ( Grebe & Stock , 1999 ) and an output domain belonging to the FixJ -RCB- NarL subfamily ( Parkinson & Kofoid , 1992 ) , becomes activated by phosphorylation of Asp-54 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11739766" target="_blank">11739766</a>]<br></li>
<li>The strongiy conserved sequences in the receiver moduie inciude the aspartyt residue ( Asp-54 in UhpA ) that is the sife of phosphorytation in CheY and NtrC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7596290" target="_blank">7596290</a>]<br></li>
<li>Although it seems most likely that sCTD contacts UhpA , the aCTD contacts are not yet known .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10572130" target="_blank">10572130</a>]<br></li>
<li>Analogous involvement of helix R1 in intermolecular contacts is also found in UhpA and in the CheY/CheZ complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9521685" target="_blank">9521685</a>]<br></li>
<li>Webber and Kadner ( 16 ) show that phosphorylation of UhpA promotes oligomerization along the surface of a DNA helix , and that this oligomerization occurs via contacts between helixes R1 in the UhpA molecules .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9521685" target="_blank">9521685</a>]<br></li>
<li>For example , NarX and CpxA are identical at seven of eight residues in the region immediately preceeding the Gly-Leu-Gly-Leu motif ( residues 510 to 523 to NarX ) ; NarX , DegS , and UhpA share little similarity over the same region ( Fig. 3 and Table 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2649492" target="_blank">2649492</a>]<br></li>
<li>It is also possible to assign the motif in the highly homologous region with FixJ and UhpA between Leu 171 and Lys 192 , although this region is most likely to form the helix-beta-helix structure ( Fig. 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2657652" target="_blank">2657652</a>]<br></li>
<li>UhpA has a net negative charge , and the other three proteins bear net positive charges .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3301805" target="_blank">3301805</a>]<br></li>
<li>NarL , DegU , and UhpA also share uniqde : ly conserved residues within their amino-terminal domains ( Fig. 3 and Table 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2649492" target="_blank">2649492</a>]<br></li>
<li>Within the aligned regions of these proteins , 60 % of the amino acid residues in UhpA were found in an identical position in at least one of the other proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3042748" target="_blank">3042748</a>]<br></li>
<li>Sequences homologous to this conserved 22-amino-acid region are found near the C terminus of UhpA ( amino acid residues 153 to 174 ) and are compared with those of several other regulatory proteins in Table 1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3301805" target="_blank">3301805</a>]<br></li>
<li>Residues in UhpA with a double underline are those in which the same residue is present in at least three of the other regulatory proteins ; those with a single underline are replaced by homologous amino acids in at least seven of the other proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3301805" target="_blank">3301805</a>]<br></li>
<li>Hence , UhpA only has an N-terminal motif ( about 120 amino acids ) in common with the OmpR group and the NtrC group of regulatory proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3042748" target="_blank">3042748</a>]<br></li>
<li>Sequence similarities within the amino-terminal domain place UhpA in the family of regulatory proteins which includes OmpR , CheY , PhoB , and NtrC ( 25 , 34 ) , whereas sequence relatedness in the carboxyl-terminal domain indicates that UhpA is a member of the UhpA-LuxR family of transcriptional activators ( 12 , 35 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1569008" target="_blank">1569008</a>]<br></li>
<li>These were constructed by ligation into the PstI-digested plasmid pSU19 of PstI fragments extending from the coding region for residue 189 of UhpA to the coding region for residue 91 of UhpC or to the uhpT promoter region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11053370" target="_blank">11053370</a>]<br></li>
<li>Since UhpA binds near the 235 region , we expected that UhpA might contact the CTD of RpoD , as found for some transcription activators that bind in a similar position .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10572130" target="_blank">10572130</a>]<br></li>
<li>As seen at other promoters , these sCTD residues are needed for the action of the transcription factor bound near position 235 , in this case UhpA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10572130" target="_blank">10572130</a>]<br></li>
<li>Perhaps UhpA plays a similar role at the uhpT promoter to alter DNA geometry and allow contact of RNAP and CAP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10572130" target="_blank">10572130</a>]<br></li>
<li>Other studies suggested that UhpA at the 235 region interacts with the s70 subunit of the RNAP holoenzyme to enhance RNAP binding and to allow low-level transcription ( 24 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10913075" target="_blank">10913075</a>]<br></li>
<li>Subsequently , this phosphoryl group is transferred to a conserved aspartate residue in the response-regulator UhpA , which then initiates uhpT transcription , via binding to the uhpT promoter region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11739766" target="_blank">11739766</a>]<br></li>
<li>Taken together , these results support the theory that activation of the uhpT promoter occurs by two independent interactions of domains of RNAP with UhpA molecules at opposite ends of the UhpA-binding region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10913075" target="_blank">10913075</a>]<br></li>
<li>If UhpA action depends on its binding to the ÿ35 region of the promoter , it is likely that bound UhpA contacts RNAP to affect its binding or activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10512697" target="_blank">10512697</a>]<br></li>
<li>Thus , binding of UhpA to the W region is crucial for UhpA-dependent activation and depends on occupancy of the S region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11976297" target="_blank">11976297</a>]<br></li>
<li>Because CAP can not activate uhpT transcription by itself , UhpA must activate transcription in two ways , one involving interaction with sCTD and the other involving interaction with CAP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10572130" target="_blank">10572130</a>]<br></li>
<li>In a second model , CAP may act to stabilize UhpA bound to uhpTp by using a contact region other than that between amino acids 156 and 162 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7896692" target="_blank">7896692</a>]<br></li>
<li>In the bipartite activation model , UhpA bound to the downstream W region might allow recruitment and positioning of RNAP through interaction with - CTD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11976297" target="_blank">11976297</a>]<br></li>
<li>Strong stimulation of transcription requires the interaction of the a-CTD with UhpA and CAP in the 280 region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10913075" target="_blank">10913075</a>]<br></li>
<li>The a-CTD , which is necessary for substantial activation of uhpT transcription ( 24 ) , may contact both CAP and UhpA at the 280 region , as suggested by the formation of a DNase-hyper-sensitive site there when in the presence of all three proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10913075" target="_blank">10913075</a>]<br></li>
<li>Effect of cysteine substitutions in the C-terminal domain of UhpA on DNA-binding , transcription activity , and site-speciﬁc cleavage D NA c ding Transcription activity bin Relative protein yieldb UhpA varianta FeBABE cleavagef In vitroe d S W LacZ P CAP CA -- Wild type 100 100 H-WT 100 300 83 100 -- H-A154C 100 100 H-V155C 66 37 H-G181C 24 53 H-V182C 33 38 S183C 15 63 D185C 43 51 E187C 100 70 A189C 42 179 R191C 10 47 a UhpA proteins carried the indicated cysteine substitutions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11976297" target="_blank">11976297</a>]<br></li>
<li>Thus , BvgA Bpe , TrpO Pae , and FimZ Eco cluster together in both trees , but FixJ , which clusters with UhpA Eco in the phylogenetic tree of the DNA-binding domains ( Fig. 8 ) , is very distant from UhpA Eco in the phylogenetic tree of the receiver modules ( Fig. 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7699720" target="_blank">7699720</a>]<br></li>
<li>The sequence of the first 20 N-terminal residues was determined by automated Edman degradation and matched exactly that of the deduced UhpA polypeptide ( 28 ) , showing that the N-terminal methionine residue was retained in the product .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8999880" target="_blank">8999880</a>]<br></li>
<li>Thus , several residues in sCTD appear to be important for UhpA activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10572130" target="_blank">10572130</a>]<br></li>
<li>The role of these residues was veriﬁed by in vitro transcription assays , showing that residues 593 , 598 , and 599 were required for UhpA activation as well as other transcription activators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10572130" target="_blank">10572130</a>]<br></li>
<li>UhpA and its homoiogues appear to be composed of two modules : the receiver module which contains the putative site of phosphorylation , and the activation module whose predicted helix-turn-helix motif is related to that present in many transcription activators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7596290" target="_blank">7596290</a>]<br></li>
<li>The N-terminal domain contains a highly conserved aspartyl resi-due ( Asp-54 in UhpA ) , which is the site of phosphorylation in homologous proteins , CheY and NtrC ( 9 , 10 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8999880" target="_blank">8999880</a>]<br></li>
<li>UhpA contains the amino-terminal phosphorylation module of about 120 amino acids that is characteristic of and highly conserved among response regulator proteins , while the carboxylterminal region is related to only a subfamily of transcription activator proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8144492" target="_blank">8144492</a>]<br></li>
<li>UhpA contains four cysteines in the N-terminal phosphorylation module .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11976297" target="_blank">11976297</a>]<br></li>
<li>NarL , FimZ , UhpA and RcsB of subgroup ( c ) also certainly fall into the category of response regulator from the existence of phosphoryl-ation domain , although SdiA and MalT show the similarity to the above four regulatory proteins only in the C-terminal region containing the helix-turn-helix motif .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8729576" target="_blank">8729576</a>]<br></li>
<li>RcsB also has more extended homology in its C-terminal domain to the subfamily of regulators containing FixJ , DegU , NarL , and UhpA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2404948" target="_blank">2404948</a>]<br></li>
<li>A third subfamily of transcriptional activator proteins includes UhpA , NarL , DegU , and FixJ ( 34 ) ; they possess the amino-terminal phosphorylation domain , a short variable region , and a carboxyl-terminal region that contains a predicted helix-turn-helix motif .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1569008" target="_blank">1569008</a>]<br></li>
<li>When RcsA is limiting [ Ion * cells ) and RcsB is not activated by phosphorylation , the cps region is not transcribed The predicted protein sequences of both RcsA and RcsB contain a carboxy-terminal domain similar to that found in a class of DNA-binding proteins exemplified by UhpA , MalT , and FixJ ( Fig. 1 ; Stout etal .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1943696" target="_blank">1943696</a>]<br></li>
<li>UhpA has high sequence similarity to the nitrate-responsive RR NarL ( 1 ) , whose structure reveals a two-domain protein with an N-terminal CheY-like phosphorylation domain and a C-termi-nal DNA-binding helix-turn-helix domain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11053370" target="_blank">11053370</a>]<br></li>
<li>Within the C-terminal region specific to UhpA is a potential helix-turn-helix motif that could constitute a DNA-binding domain ( 7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3042748" target="_blank">3042748</a>]<br></li>
<li>Cysteine residues were introduced at nine positions of the C-terminal domain of UhpA adjacent to the region of homology to the DNA-binding helix-turn-helix motif of NarL ( 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11976297" target="_blank">11976297</a>]<br></li>
<li>The UhpA and FixJ proteins of the first group were shown to have helix-turn-helix DNA-binding motif in the C-terminal domain ( 45 , 46 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2657652" target="_blank">2657652</a>]<br></li>
<li>The structure of the sCTD may resemble that of the C-terminal helix-turn-helix DNA-binding domain of NarL and UhpA ( 25 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10572130" target="_blank">10572130</a>]<br></li>
<li>The recent presentation of the structure of NarL , a close homolog of UhpA , reveals the presence of a DNA-binding helix-turn-helix motif in the C-terminal domain that is occluded by interaction with the N-terminal phosphorylation module ( 15 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8999880" target="_blank">8999880</a>]<br></li>
<li>Consistent with the binding of UhpA near the 235 region , residues K593 and K599 in the C-terminal region of RpoD were necessary for efﬁcient uhpT expression in response to UhpA alone .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10572130" target="_blank">10572130</a>]<br></li>
<li>The ability of UhpA to activate uhpT transcription depended on certain residues ( K593 , L598 , and K599 ) in the 268 C-terminal domain of RpoD ( - CTD ) ( 34 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11976297" target="_blank">11976297</a>]<br></li>
<li>The dimeric nature of UhpA binding is indicated by the pattern of DNA cleavage by UhpA molecules carrying a hydroxyl radical-generating moiety near its DNA-binding domain ( I. N. Olekhnovich and R. J. Kadner , unpublished data ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10913075" target="_blank">10913075</a>]<br></li>
<li>The liberated HK domain carried in GST-Bc was found to prevent both the binding of UhpA at the uhpT promoter and transcription-activation in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11325944" target="_blank">11325944</a>]<br></li>
<li>The DNA-binding and activation domain of UhpA is well conserved and widely distributed in several families of transcriptional regulators , but it has not been extensively analyzed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11976297" target="_blank">11976297</a>]<br></li>
<li>The size of UhpA ( 196 amino acids ) and its sequence homology patterns suggest the existence of two domains : at the amino end the phosphorylation module analogous to CheY ( amino acids 1 to 120 ) , and at the carboxyl end an activation/DNA-binding module ( residues 135 to 196 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8144492" target="_blank">8144492</a>]<br></li>
<li>The major difference in sequence between UhpA and NarL is that the flexible hinge in UhpA is 10 residues shorter than in NarL , which might rotate the C-terminal region relative to the phosphorylation module and thereby allow continuous exposure of the DNA-binding surface .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8999880" target="_blank">8999880</a>]<br></li>
<li>We tested nine positions ﬂanking the DNA-binding region , but only one , UhpA E187C , fulﬁlled all these criteria .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11976297" target="_blank">11976297</a>]<br></li>
<li>The UhpA activator protein contains a region with substantial homology to the areas of DNA-binding regulatory proteins that are responsible for the sequence-specific interactions between the protein and its DNA target ( 19 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3301805" target="_blank">3301805</a>]<br></li>
<li>The -64 element appears to be the only region required for in vivo UhpA binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1569008" target="_blank">1569008</a>]<br></li>
<li>The iron chelate FeBABE [ iron ( S ) -1 - ( p-bromoacetamidobenzyl ) EDTA ] was covalently attached to engineered cysteine residues near the DNA-binding region in UhpA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11976297" target="_blank">11976297</a>]<br></li>
<li>One possibility is that the truncated species forms a mixed oligomer with the wild-type version of UhpA that is inactive for DNA-binding or activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8144492" target="_blank">8144492</a>]<br></li>
<li>In this study , cysteine residues introduced into UhpA were derivatized with a DNA cleavage reagent , and the resulting pattern of hydroxyl radical-mediated DNA cleavage was consistent with the binding of two dimers of UhpA , with each dimer occupying one region in tail-tail manner .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11976297" target="_blank">11976297</a>]<br></li>
<li>Since the extent of labeling at position 187 and the other four cysteine residues in UhpA was not known , the effect of deri-vatization by FeBABE on DNA-binding could not be determined .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11976297" target="_blank">11976297</a>]<br></li>
				</div></ul><br>
			<label class="collapse_EVO" for="_3"><font size="5px">UhpA evolution</font></label>
				<input id="_3" type="checkbox">
				<div class="EVO"><br><ul><li>UhpA showed the closest match to DctD , with 30 % identity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3042748" target="_blank">3042748</a>]<br></li>
<li>Arabinose ( % ) 0 0.002 0.02 0.2 EvgA NarL NarP RcsB UhpA UvrY Figure 1 Artiﬁcial expression of RRs .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26010043" target="_blank">26010043</a>]<br></li>
<li>The mHTH class includes the three large families , AraC/XylS ( 8.5 % ) , LuxR/UhpA ( 5.7 % ) , and TetR/AcrR ( 9.4 % ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15556475" target="_blank">15556475</a>]<br></li>
<li>Members Group function Ec1 So2 Bs3 Group function LuxR/UhpA or OmpR family transcriptional response regulator of two-component regulatory system LysR family transcriptional regulator GntR or DeoR family transcriptional regulator sensory histidine kinase in two-component regulatory system sigma54 activator protein , enhancer binding protein AraC/XylS family transcriptional regulator ROK family transcriptional regulator/sugar kinase IclR family transcriptional regulator methyl-accepting chemotaxis protein MerR family transcriptional regulator DNA-binding regulatory protein AraC/XylS family transcriptional regulator MarR family transcriptional reguator AsnC family transcriptional regulator 46 20 18 18 13 13 11 10 9 18 14 13 13 11 16 13 8 1 oxidoreductase , Fe-S-binding 31 10 11 13 6 oxidoreductase , NAD ( P ) - binding oxidoreductase4 , NAD ( P ) - binding aldehyde dehydrogenase , NAD ( P ) - binding oxidoreductase , FAD/NAD ( P ) - binding sugar kinase respiratory reductase , alpha subunit aldo-keto reductase , NAD ( P ) - binding phosphatase nucleoside diphosphate ( Nudix ) hydrolase 43 30 22 5 14 14 7 8 8 7 5 8 8 2 7 5 1No .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15757509" target="_blank">15757509</a>]<br></li>
<li>The RR UhpA , which is composed of a receiver domain that belongs to the RC1 subfamily of receiver domains ( Grebe & Stock , 1999 ) and an output domain belonging to the FixJ -RCB- NarL subfamily ( Parkinson & Kofoid , 1992 ) , becomes activated by phosphorylation of Asp-54 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11739766" target="_blank">11739766</a>]<br></li>
<li>The similarity of UhpA and UhpB to receiver and transmitter modules of sensory transduction pathways has been described ( 14 , 28 , 31 ) , and the correction of two short regions in the published sequence at the carboxyl-terminal end of UhpB resulted in an increased match to other members of the protein kinase family .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1569007" target="_blank">1569007</a>]<br></li>
<li>presence of members of the LuxR/UhpA family in bacteria .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15556475" target="_blank">15556475</a>]<br></li>
<li>Alignment of UhpA with homologous proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3042748" target="_blank">3042748</a>]<br></li>
<li>The N-terminal domain contains a highly conserved aspartyl resi-due ( Asp-54 in UhpA ) , which is the site of phosphorylation in homologous proteins , CheY and NtrC ( 9 , 10 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8999880" target="_blank">8999880</a>]<br></li>
<li>The sequence of the DctD protein , which controls expression of the dicarboxylic acid transport system in Rhizobium leguminosarum ( 20 ) , provided the key to the relationship of UhpA to the first family of regulatory proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3042748" target="_blank">3042748</a>]<br></li>
<li>Amino acid sequence comparisons showed that UhpA was homologous to a family of bacterial regulatory proteins , some of which act as transcriptional activators ( OmpR , PhoB , NtrC , and DctD ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3042748" target="_blank">3042748</a>]<br></li>
<li>Sequence similarities within the amino-terminal domain place UhpA in the family of regulatory proteins which includes OmpR , CheY , PhoB , and NtrC ( 25 , 34 ) , whereas sequence relatedness in the carboxyl-terminal domain indicates that UhpA is a member of the UhpA-LuxR family of transcriptional activators ( 12 , 35 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1569008" target="_blank">1569008</a>]<br></li>
<li>Sequence comparisons of the Uhp regulatory proteins showed that UhpA and UhpB contain sequence motifs characteristic of members of the widespread group of two-component bacterial regulatory systems ( Kofoid and Parkinson , 1988 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8144492" target="_blank">8144492</a>]<br></li>
<li>The UhpA and UhpB proteins are members of families of two-component regulatory proteins , which also include OmpR-EnvZ , CheY-CheA , PhoB-PhoR , t Present address : Food and Drug Administration , Rockville , MD 20850 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1569008" target="_blank">1569008</a>]<br></li>
<li>The UhpA protein is required for expression ; its deduced amino acid sequence shows that it belongs to a subfamily of bacterial transcription regulators including NarL , DegU , and FixJ .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1569008" target="_blank">1569008</a>]<br></li>
<li>UhpA is a transcription activator protein with extensive amino acid sequence similarity to the phosphate-accepting transcription activators of two-component regulatory response systems ( 41 , 46 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7896692" target="_blank">7896692</a>]<br></li>
<li>Sequence comparisons show that UhpA is a member of the famity of response-regulator proteins that inciudes NarL , NarP and RcsB from # coli , DegU and ComA from Bacittus subtilis , FixJ from Rhizobium species , and others [ Pao etai .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7596290" target="_blank">7596290</a>]<br></li>
<li>Sequence comparisons group UhpA into a family of response-regulator proteins that includes NarL from E. coli , DegU from Bacillus subtilis , and FixJ from Rhizobium species ( 7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8999880" target="_blank">8999880</a>]<br></li>
<li>The regulon-speciﬁc regulator UhpA is a member of the NarL family of response regulator ( RR ) proteins of two-component regulatory systems ( reviewed in reference 40 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11976297" target="_blank">11976297</a>]<br></li>
<li>In this connection , it is worth comparing NarL with another member of the response regulator family , the Escherichia coli UhpA protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9521685" target="_blank">9521685</a>]<br></li>
<li>UhpA belongs to the NarL family of response regulators ( 11 , 29 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10913075" target="_blank">10913075</a>]<br></li>
<li>The 196-residue UhpA protein is a member of the NarL family of response regulators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10512697" target="_blank">10512697</a>]<br></li>
<li>NarL , FimZ , UhpA and RcsB of subgroup ( c ) also certainly fall into the category of response regulator from the existence of phosphoryl-ation domain , although SdiA and MalT show the similarity to the above four regulatory proteins only in the C-terminal region containing the helix-turn-helix motif .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8729576" target="_blank">8729576</a>]<br></li>
<li>RcsB also has more extended homology in its C-terminal domain to the subfamily of regulators containing FixJ , DegU , NarL , and UhpA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2404948" target="_blank">2404948</a>]<br></li>
<li>This family also includes UhpA ( 45 ) , FixJ ( 46 ) , DegU ( 47 , 48 ) , and UvrC ORF2 ( 49 ) , which , in addition to the homologous N-terminal domain , shared extensive similarity to NarL over their entire lengths ( Fig. 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2657652" target="_blank">2657652</a>]<br></li>
<li>The DNA-binding and activation domain of UhpA is well conserved and widely distributed in several families of transcriptional regulators , but it has not been extensively analyzed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11976297" target="_blank">11976297</a>]<br></li>
<li>UhpA has high sequence similarity to the nitrate-responsive RR NarL ( 1 ) , whose structure reveals a two-domain protein with an N-terminal CheY-like phosphorylation domain and a C-termi-nal DNA-binding helix-turn-helix domain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11053370" target="_blank">11053370</a>]<br></li>
<li>The C-termi-nal domains of the UhpA family of response regulators contain a highly conserved segment predicted to form a helix-turn-helix motif that is similar to the DNA-binding regions of some transcription factors that are not regulated by protein phosphoryl-ation , such as MalT from E. coli and LuxR from Vibrio fischeri ( 14 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8999880" target="_blank">8999880</a>]<br></li>
				</div></ul><br>
			<label class="collapse_RP" for="_4"><font size="5px">Processes regulated by UhpA</font></label>
				<input id="_4" type="checkbox">
				<div class="RP"><br><ul><li>Gene Description PQ-activated 2.0 2.8 b0463 b0605 b1415 b0863 b0864 b0710 b1378 b1452 b2351 b2962 b4131 b4133 b2198 b0429 b2752 b2414 b1190 b4383 b0184 b0812 b0684 b3288 b3924 b1611 b0683 b2094 b2093 b2091 b0450 b0720 b2552 b0440 b2237 b4036 b0096 b4034 b4035 b0168 b1531 b0086 b0578 b2281 b2279 b0113 b4025 b1101 b1658 b3317 b3319 b3305 b3985 b3318 b3312 b3314 b3321 b3316 b0724 b3908 b4062 b0729 b3708 b0850 b2523 b3520 b4217 b1852 acrA ahpC aldA artI artP b0710 b1378 b1452 b2351 b2962 cadA cadC ccmD cyoD cysD cysK dadX deoB dnaE dps ﬂdA fmt fpr fumC fur gatA gatB gatD glnK gltA hmpA hupB inaA lamB lpxC malE malK map marA murF nfnB nuoI nuoK pdhR pgi ptsG purR rplB rplD rplF rplJ rplW rpmC rpsC rpsJ rpsS sdhB sodA soxS sucD tnaA ybjC yfhI yhjB ytfK zwf AcrAB efﬂux system effects Mar multiple resistance Alkyl hydroperoxide reductase small subunit 2.3 1.9 1.9 2.0 1.7 1.9 1.7 2.4 1.8 2.4 1.7 1.7 1.7 1.9 1.7 1.9 2.0 1.7 4.7 1.8 3.6 6.9 2.2 2.9 3.7 1.7 1.8 3.1 1.8 1.7 2.9 4.8 2.0 2.1 1.9 1.7 2.6 1.7 2.1 1.8 1.9 1.9 2.1 3.3 1.9 1.9 2.5 1.7 1.7 1.8 1.7 1.7 2.0 1.7 1.8 12.3 2.3 1.9 2.4 1.7 1.9 3.4 2.3 2.7 Aldehyde dehydrogenase , NAD linked Periplasmic binding protein of Arg transport system Arg periplasmic transport system Function unknown Putative pyruvate-ﬂavodoxin oxidoreductase Function unknown CPS-53 prophage ; putative bactoprenol glucosyl transferase Function unknown Lysine decarboxylase Regulatory gene Cytochrome c-related Cytochrome o oxidase subunit IV Sulfate adenylyltransferase Cysteine synthase A Alanine racemase Deoxyribouratase , phosphopentomutase DNA polymerase III , alpha subunit Stress response DNA-binding protein Flavodoxin Methionyl-tRNA formyltransferase Ferredoxin NADP1 reductase ; anaerobic Fumarase C Ferric iron uptake , negative regulatory gene Galactitol-speciﬁc enzyme IIA of PTSb Galactitol-speciﬁc enzyme IIB of PTS Galactitol-1-phosphate dehydrogenase Regulated through NRI/NRII two-component regulatory system Citrate synthase Dihydropteridine reductase 2 and nitric oxide dioxygenase NAD/FAD Histone-like protein HU-alpha , HU-1 Function unknown Maltose high-afﬁnity uptake Cell envelope and cell separation Maltose-binding protein , periplasmic ; transport and chemotaxis Maltose transport complex , ATP-binding subunit Methionine aminopeptidase Transcription activator of multiple-antibiotic-resistance system D-Alanyl :D - alanine-adding enzyme Resistance to nitrofurantoin ; a nitroreductase NADH dehydrogenase I subunit NADH dehydrogenase I subunit Pyruvate-dehydrogenase repressor Glucose phosphate isomerase PTS family enzyme IIC , glucose-speciﬁc Purine repressor 50S ribosomal subunit protein L2 50S ribosomal subunit protein L4 50S ribosomal subunit protein L6 50S ribosomal subunit protein L10 50S ribosomal subunit protein L23 50S ribosomal subunit protein L29 30S ribosomal subunit protein S3 30S ribosomal subunit protein S10 30S ribosomal subunit protein S19 Succinate dehydrogenase iron-sulfur protein Superoxide dismutase , Mn Regulatory protein of soxRS regulon Succinyl-coenzyme A synthetase alpha subunit Tryptophanase Function unknown Peptidase B ; aminopeptidase Putative transcriptional regulator ( LuxR/UhpA family ) Function unknown Glucose-6-phosphate dehydrogenase activity Continued on following page TABLE 1 -- Continued Gene type Description No .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11395452" target="_blank">11395452</a>]<br></li>
<li>Description Expression ratio etabolism Synthesis and m bioF pﬂD yfdW 8-Amino-7-oxononanoate synthase Formate acetyltransferase 2 b0776 b3951 b2374 21.1 13.9 12.1 Putative enzyme ; formyl-CoAa transferase monomer , subunit of formyl-CoA transferase ORF , hypothetical protein ; triphosphoribosyl-dephospho-CoA synthase Putative aldose-1-epimerase L-Tartrate dehydratase , subunit A Endonuclease III , speciﬁc for apurinic and/or apyrimidinic sites Glucitol operon activator Reactivating factor for ethanolamine ammonia lyase EutBC ORF , hypothetical protein ; protein with a sugar isomerase domain ; GutQ has similarity to E. coli K1 Fumarase B fumarate hydratase class I ; anaerobic isozyme Putative xylanase Putative beta-xylosidase Phosphomethylpyrimidine kinase ( thiamine-biosynthesis ) 2,3-Dihydroxyphenylpropionate 1,2-dioxygenase Putative glycosyl transferase ; colanic acid related Thiamine-biosynthesis , thiazole moiety Probable nitrate reductase 3 Putative prismane ; hybrid cluster protein/hydroxylamine reductase Hydrogenase 4 membrane subunit Hydrogenase 4 Fe-S subunit Phosphoribosylaminoimidazole-succinocarboxamide synthetase , SAICAR synthetase Citrate lyase synthetase ( citrate [ pro-3S ] - lyase ligase ) Putative regulator for colanic-acid-synthesis Diaminopimelate decarboxylase ORF , hypothetical protein ; member of the GMP family of beta barrel pores Trehalase 6-P hydrolase Regulator of acetyl-CoA synthetase Ornithine carbamoyltransferase 2 , chain F ( arginine-biosynthesis ) Regulator for metE and metH , homocysteine transcriptional activator Phosphotransferase system N-acetylgalactosameine-speciﬁc IIC component 2 Putative oxidoreductase ; reductase enzyme which can convert oxidized ﬂavorubredoxin to its reduced form ( NO removal ) Part of regulation of tor operon , periplasmic Probable large subunit , hydrogenade 2 b0613 b3879 b3061 b1633 b2706 b2451 b2708 12.1 11.3 11.3 10.6 9.8 9.8 9.2 citG yihR ttdA nth gutM eutA gutQ b4122 b3719 b0271 b2103 b0348 b2057 b3992 b2206 b0873 b2484 b2481 b2476 fumB yieL yagH thiD mhpB wcaC thiF napA ybjW hyfD hyfA purC 9.2 8.6 8 8 7.5 7 7 7 6.5 6.5 6.5 6.1 b0618 b2059 b2838 b1337 b4239 b4017 b0273 b3828 b3134 b2711 citC 6.1 5.7 5.3 5.3 4.9 4.9 4.9 4.6 4.6 4.3 133 377 wcaA lysA abgB treC arp argF metR agaW ygbD b0994 b2994 torT hybC 4.3 4.3 342 567 Metabolism and transport wza Putative polysaccharide export protein 4 379 b2062 Transport yccW focB Putative oxidoreductase ; putative methyltransferase b0967 b2492 17.1 13.9 367 282 Probable formate-transporter ( formate channel 2 ) ; FocB formate FNTb transporter Putative resistance protein ( transport ) ORF , hypothetical protein Putative permease Uracil transport Putative amino acid/amine-transport protein ; YhfM methionine APCc transporter Anaerobic dicarboxylate transport ATP-binding component of transport system for maltose Integral transmembrane protein ; acridine resistance Arginine 3rd transport system periplasmic binding protein Phenylalanine-speciﬁc transport system ORF , hypothetical protein ; putative transport protein Phosphotransferase system , N-acetylglucosamine enzyme IID component 1 b3874 b4307 b3877 b2497 b3370 yihN yjhQ yihP uraA yhfM 462 12.1 11.3 10.6 8.6 7.5 b4123 b4035 b3266 b0860 b0576 b1115 b3140 dcuB malK acrF artJ pheP ycfT agaD 446 371 6.5 6.1 5.7 5.3 4.9 4.3 4 1,034 Structure acrE b3265 brane Transmembrane protein affects septum formation and cell mem permeability Homolog of Salmonella P ring of ﬂagellar basal body Sensor histidine protein kinase phosphorylates UhpA Flagellar biosynthesis 385 24.3 b1080 b3668 b0229 ﬂgI 6.5 4.6 4.3 uhpB fhiA Continued on following pag TABLE 2 -- Continued Expression ratio Protein size ( aa ) Group and gene b no .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16385049" target="_blank">16385049</a>]<br></li>
<li>Activates MinC 0.6 Multicopy on plasmid in strain with a deletion of envZ induces ompC expression 0.4 Beta-ketoacyl-acyl carrier protein synthase I 0.5 Flagellar synthesis ; ﬂagellar regulon member 0.5 Cyclase component of IGP synthase complex 0.5 L-Xylulose kinase , cryptic 0.5 Involved in chromosome partitioning 0.3 Nitrate reductase alpha subunit 0.6 Formate hydrogen-lyase activity 0.2 Outer membrane protein , porin 0.3 Aspartate transcarbamylase , catalytic subunit 0.5 Dihydroorotate oxidase 0.6 Membrane protein 0.5 Superoxide dismutase , Fe 0.6 Superoxide dismutase , Cu , Zn 0.4 Uracil concentration dependence of pyr mutants ; Ura ABC transporter 0.4 Function unknown 0.6 Putative transcriptional repressor ( LuxR/UhpA family ) 0.6 Rac prophage 0.5 Ribosome-associated factor , stabilizes ribosomes against dissociation 0.4 Putative NADP-binding dehydrogenase 0.5 Putative transmembrane protein 0.4 Anaerobic pathway , L-serine deaminase , L-serine dehydratase 0.5 Putative transcriptional regulator ( AraC/XylS family ) 0.5 Putative hexulose-6-phosphate isomerase 0.6 Putative aldose-1-epimerase 0.5 Putative acyltransferase 0.5 Function unknown 0.6 Putative outer membrane protein 0.4 Putative transport protein 0.6 Function unknown 0.4 Function unknown modulated by either exposure to PQ or expression of SoxS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11395452" target="_blank">11395452</a>]<br></li>
<li>Catabolite response Deoxirribose assimilation D 87 135 Crp w 27 Pérez - Rueda and Collado-Vides ( 2000 ) 220 -- R 235 10 DeoR DicA 35 w Pérez - Rueda and Collado-Vides ( 2000 ) -- n/d 0 -- -- n/d HLH R 4 DicC DnaA 0 Transcription and replication regulation Positive regulator of cytochrome c552 77 0 -- Pérez - Rueda and Collado-Vides ( 2000 ) 41 -- n/d DniR 0 Kajie et al. ( 1991 ) 209 8 215 -- A Ebp FhlC 77 m Pérez - Rueda and Collado-Vides ( 2000 ) Pérez - Rueda and Collado-Vides ( 2000 ) n/d 0 -- 7 -- Pérez - Rueda and Collado-Vides ( 2000 ) Schnetz ( 2002 ) n/d R FhlD FrvR 0 -- Phosphoenolpyruvate sugar phosphotransferase system operon regulation Iron assimilation 8 0 -- -- 128a Fur R w Achenbach and Yang ( 1997 ) 107 460 35 Carbon sources assimilation Nguyen and Saier ( 1995 ) ; Weickert and Adhya ( 1992 ) 388 GalR/LacI 55 R c -- 13 GcvR GntR Rigali et al. ( 2002 ) Glycine cleavage operon repression 0 R 735 775 Ghrist and Stauffer ( 1998 ) 96 R w Yamada and Saier ( 1988 ) Activator of the glucitol operon in E. coli High-frequency persistence of the lethal effects of inhibition of either DNA or peptidoglycan-synthesis regulation ( HipAB ) -- 8 GutM 0 A -- Black et al. ( 1991 ) -- 1 HipA 1 0 n/d -- n/d HipB HtgA Black et al. ( 1991 ) Missiakas et al. ( 1993 ) 87 0 -- -- -- n/d 7 Heat shock regulation 0 -- HycA HypF 6 -- n/d ZnF A 0 -- 0 Bouhouche et al. ( 2000 ) Gui et al. ( 1996 ) 27 -- Glyoxylate bypass operon Integration host factor 31 106 161 w R -- IclR Ihf -- 15 8 Pérez - Rueda and Collado-Vides ( 2000 ) Campoy et al. ( 2002 ) -- D 43 -- DNA-repair 54 LexA R w 102 279 Miller and Bassler ( 2001 ) Schell ( 1993 541 LuxR/UhpA m A Quorum sensing 1548 LysR 1271 Amino acid biosynthesis regulation D w 3.3 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15556475" target="_blank">15556475</a>]<br></li>
<li>CsgD ( curlin subunit gene D ) is a FixJ/LuxR/UhpA family transcription factor that regulates the csgBAC and csgDEFG operons for the synthesis , secretion , and assembly of curli components ( 10 , 20 , 22 , 27 , 41 , 42 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21421764" target="_blank">21421764</a>]<br></li>
<li>AGRT6 VIRG AGRRA YIIA ECOLI TOXR VIBCH + RRHM Pae Plastid response regulator OmpR Eco Porin bioynthesis , osmolarity PhoB Bsu Alkaline phosphatase regulator CreB Eco Catabolite repression ArcA Eco Aerobic respiratory control PetR Rca Cytochrome bc1 complex CutR Sli Copper metabolism KdpE Eco K + transport PhoB Eco Phosphate regulon PhoB Pae Phosphate regulon PhoP Sty Virulence TctD Sty Tricarboxylate transport VirG Atu6 Virulence VirG Atu5 Virulence VirG Arh Virulence YjjE Eco Unknown ToxR Vch Virulence cholera toxin Class 3 : DNA binding domains of family 3 FixJ Rme Nitrogen fixation FixJ Bja Nitrogen fixation NodW Bja Nodulation BvgA Bpe Virulence FimZ Eco Fimbrial synthesis DegU Bsu Extracellular degradative enzymes NarL Eco Nitrate reductase & formate dehydrogenase TrpO Pae Unknown YecB Eco Unknown RcsB Eco Colanic acid capsule biosynthesis B AgmR Fae Glycerol metabolism UhpA Eco Uptake of hexose phosphates Coma Bsu Competence GerE Bsu SigK txn .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7699720" target="_blank">7699720</a>]<br></li>
<li>Doubling regulated regulated of altered time Identiﬁed function Class system ( HK/RR ) Relevant functions genes genes genes ( min ) in this experiment Strain name Mutation ntrol o o C N 1 4 5 27 BW25113 WT or little signiﬁcant alteration CreC/CreB 1 1 2 Catabolite repression ( Wanner , 1996 ) Drug resistance ( Kato et al. , 2000 ) BW26983 DE ( creABCD ) 154 29 2 3 5 EvgS/EvgA BW27869 DE ( evgAS ) 1291 29 2 6 8 YgiY/YgiX NarX/NarL BW27551 BW27864 DE ( ygiXY ) 1302 DE ( narXL ) 1316 27 29 5 4 9 Nitrate regulation ( Stewart and Rabin , 1995 ) Hexose phosphate uptake ( Kadner , 1995 ) Phosphate regulation ( Wanner , 1995 ) Copper response ( Munson et al. , 2000 ) Nitrate regulation ( Stewart and Rabin ,1995 ) UhpB/UhpA 6 3 9 BW27871 DE ( uhpAB ) 1322 28 PhoR/PhoB 1 8 9 BW24476 DE ( phoBR ) 580 28 CusS/CusR 2 7 BW28077 DE ( cusRS ) 1204 27 9 / NarP 9 BW27873 DnarP1312 33 12 3 YedV/YedW YfhK / YehU/YehT KdpD/KdpE DE ( yedWV ) 1298 DyfhK1328 DE ( yehUT ) 1324 DE ( kdpABCDE ) 1224 26 28 27 27 13 15 20 20 6 7 7 8 BW BW BW BW 27550 27872 27877 27564 13 10 7 10 Potassium transport ( Walderhaug et al. , 1992 ) Nitrogen regulation ( Porter et al. , 1995 ) NtrB/NtrC BW27880 DE ( ntrBC ) 1318 31 20 8 12 Signiﬁcant alteration BaeS/BaeR HydH/HydG ( Nagasawa et al. , 1993 ) Response to zinc and lead ( Leonhartsberger et al. , 2001 ) Hydrogen peroxide sensitivity ( Pernestig et al. , 2001 ) 14 11 BW27553 BW27867 DE ( baeSR ) 579 DE ( hydHG ) 1336 8 22 24 29 31 13 2 BarA / BW29434 DbarA1358 24 26 30 7 / FimZ BW28078 BW27558 Enterochelin DﬁmZ1214 22 27 29 31 28 27 4 PhoQ/PhoP Response to magnesium ( Kato et al. , 1999 ) ( Nagasawa et al. , 1993 ) Response to cell envelope stress ( De Wulf et al. , 1999 ) DE ( phoPQ ) 1244 20 16 12 16 BasS/BasR CpxA/CpxR BW27848 BW27559 DE ( basRS ) 1287 DE ( cpxRA ) 623 32 32 29 29 Enterochelin 12 20 22 17 RstB/RstA YojN ( Hpt ) BW27552 BW27866 DE ( rstAB ) 1278 DyojN1332 34 37 26 29 Enterochelin Capsular synthesis ( Takeda et al. , 2001 ) 14 Control of TMAO respiratory system ( Simon et al. , 1994 ) 38 24 TorS/TorR BW26423 26 DE ( torSTRCAD ) 518 a .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12366850" target="_blank">12366850</a>]<br></li>
<li>Table 1 NarL-family RRs and their promoters ( A ) Test response regulator SK RR Regulatory roles Regulation target operons ( total number of operons ) EvgA NarL NarP RcsB UhpA UvrY gadE-mdtEF , acrD , evgAS , emrKY , -- ( 8 ) EvgS NarX Regulator for acid and multidrug-resistance Regulator against high-level nitrate and nitrite Regulator against low-level nitrate and nitrite Regulator of colanic acid in bioﬁlm formation citCDEFXG , nirBDC , hyaABCDEF , narGHJI , -- ( 33 ) hyaABCDEF , napFDAGHEC , nirBDC , -- ( 11 ) leuO , bglGFB , yidL , sfsB , ynbABCD , -- ( 35 ) uhpT ( 1 ) NarQ RcsCD Regulator of hexose phosphate uptake Regulator of carbon storage system UhpBC BarA csrB , csrC ( 2 ) Bold indicates the tested RR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26010043" target="_blank">26010043</a>]<br></li>
<li>Stage-3 -- AcP + AcP K R Signal S R Promoter NarL ( pmol ) 0 1 5 10 50 100 ydeP NarL ( pmol ) 0 1 5 10 50 100 ydeP ydeP Low pH Na + K + EvgS EvgA NarL nirB NarQ NarX Nitrate Nitrite NarL NarP ( pmol ) NarP ( pmol ) nirB Nitrate Nitrite NarQ NarX 0 1000 ydeP 0 1000 ydeP NarP arP N wza Saturated fatty acid * RcsB RcsD uhpT Glucose UhpB UhpA RcsB ( pmol ) RcsB ( pmol ) 0 1 5 10 50 100 0 1 5 10 50 100 RcsB ydeP ydeP 6-phosphate csrB BarA UvrY Formate Figure 5 Cross talks in the three stages of signal-transduction involving six NarL-family RRs .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26010043" target="_blank">26010043</a>]<br></li>
<li>The Uhp - phenotype resulting from the titration of UhpA , as measured in column B , is also manifested as resistance to fosfomycin .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1569008" target="_blank">1569008</a>]<br></li>
<li>The Phosphoryl Transfer Domain of UhpB Interacts with the Response Regulator UhpA JESSE S. WRIGHT III AND ROBERT J. KADNER * Department of Microbiology , School of Medicine , University of Virginia , Charlottesville , Virginia 22908-0734 Received 1 November 2000/Accepted 27 February 2001 Bacterial two-component regulatory systems control the expression of target genes through regulated changes in protein-phosphorylation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11325944" target="_blank">11325944</a>]<br></li>
<li>Plasmids pLAW028 , pLAW029 , pLAW006 , pLAW007 , and pLAW030 conferred constitutive Uhp expression on UhpA - hosts .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3038843" target="_blank">3038843</a>]<br></li>
<li>provide new information about the function of the uhp regulatory genes and about the aberrant expression of uhpT that occurs in response to overproduction of UhpA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3042748" target="_blank">3042748</a>]<br></li>
<li>P C UhpA dephosphorylation by UhpBC-enriched membranes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11739766" target="_blank">11739766</a>]<br></li>
<li>RESULTS Effect of UhpA phosphorylation on site occupancy .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10913075" target="_blank">10913075</a>]<br></li>
<li>Overexpression of UhpA resulted in high level , constitutive uhpT expression and loss of the normal requirements for inducer , UhpBC function , and the site of phosphorylation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8999880" target="_blank">8999880</a>]<br></li>
<li>Comparisons presented here are with the following sensorregulator transcription regulators : DegS-DegU , which controls extracellular enzyme synthesis in Bacillus subtilis ( 14 , 24 ) ; UhpB-UhpA , which controls hexose phosphate uptake in E. coli ( 37 , 50 ) ; FixL-FixJ , which controls nitrogen-fixation in Rhizobium melifoti ( 5 ) ; PhoR-PhoB , which controls the phosphate regulqn in E. coli ( 46 ) ; EnvZ-OmpR , which controls outer memitEane protein-synthesis in E. coli ( 12 ) ; and CpxA-SfrA , which controls surface properties in E. coli ( 2 , 30 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2649492" target="_blank">2649492</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_18">
Action of receiver and activator modules of UhpA in transcriptional-control of the Escherichia coli sugar phosphate-transport system .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11053370" target="_blank">11053370</a>]<br></label>
<input id="rp_similarity_18", type="checkbox">
<div class="RP_similar">
<li>Action of receiver and activator modules of UhpA in transcriptional control of the Escherichia coli sugar phosphate-transport system .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10512697" target="_blank">10512697</a>]<br></li></div>
<li>UhpA is a positive regulator for a gene ( uhpT ) required for uptake of hexose phosphates in E. coli ( 11 ) , and RcsB is a positive regulator of the capsule genes that are necessary for synthesis of a surface polysaccharide of E. coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2404948" target="_blank">2404948</a>]<br></li>
<li>Transcription of the Escherichia coli uhpT gene , encoding the inducible sugar phosphate-transporter , is dependent on the response regulator UhpA and is stimulated by the cyclic AMP receptor protein ( CAP ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10572130" target="_blank">10572130</a>]<br></li>
<li>Transcription of the Escherichia coli uhpT gene , encoding the organophosphate-transporter , requires the response regulator UhpA and is stimulated by the global regulator protein CAP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10512697" target="_blank">10512697</a>]<br></li>
<li>The uhpT promoter , which controls expression of the sugar phosphate-transport system in Escherichia coli , possesses multiple sites for its speciﬁc activator protein , UhpA , and a single site for binding of the global regulator , the catabolite gene activator protein ( CAP ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10913075" target="_blank">10913075</a>]<br></li>
<li>Transmembrane induction of uhpT expression by external glucose 6-phosphate is positively regulated by the promoter-speciﬁc activator protein UhpA and the global regulator catabolite gene activator protein ( CAP ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7896692" target="_blank">7896692</a>]<br></li>
				</div></ul><br>
			<label class="collapse_SIT" for="_5"><font size="5px">UhpA binding site</font></label>
				<input id="_5" type="checkbox">
				<div class="SIT"><br><ul><li>Similarities ' at NarX residues : Sequence Sequence 330 to 445 to 510 to 524 to DegS UhpB FixL PhoR EnvZ CpxA 20 ( 2 ) 16 ( 3 ) 6 ( 5 ) 10 ( 5 ) 8 ( 5 ) 13 ( 4 ) 11 ( 1 ) 12 ( 4 ) 3 ( 2 ) 4 ( 3 ) 4 ( 2 ) 3 ( 1 ) 1 ( 1 ) 2 ( 1 ) 3 ( 1 ) 6 ( 3 ) 7 ( 0 ) 9 ( 0 ) 4 ( 1 ) 1 ( 2 ) 0 ( 2 ) 0 ( 1 ) 0 ( 1 ) 0 ( 1 ) DegU UhpA FixJ PhoB OmpR SfrA 42 ( 15 ) 38 ( 7 ) 26 ( 9 ) 31 ( 13 ) 35 ( 8 ) 30 ( 11 ) 30 ( 7 ) 26 ( 8 ) 27 ( 6 ) 10 ( 12 ) 10 ( 13 ) 9 ( 10 ) We thank J. A. DeMoss and E. L. Sodergren for sharing data in advance of publication and T. Thannhauser for synthesizing oligo-nucleotides .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2649492" target="_blank">2649492</a>]<br></li>
<li>To examine the action of the carboxyl-terminal region , we have prepared constructs encoding forms of UhpA truncated from that end by deletion of the nucleotide sequence at the 3 ' end and replacement with translation termination signals .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8144492" target="_blank">8144492</a>]<br></li>
<li>The amino acid sequence of UhpA is strongly conserved , with only five conservative substitutions among 196 codons ( nucleotides 101 to 692 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1569007" target="_blank">1569007</a>]<br></li>
<li>Thus , sequence comparisons show that UhpA and VOV L. 171 ~ ~ ~ R ~ EGU ~ L ~ A ~ TIOO ~ NuhINp E. COLI 0 , 1988 O79L08 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3042748" target="_blank">3042748</a>]<br></li>
<li>UhpA , which bears sequence relatedness * Authorforcorrespondence.Tel .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1479338" target="_blank">1479338</a>]<br></li>
<li>The sequence recognized by UhpA is predicted to be 5 ' - GGCAAAACNNNGAAA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1569008" target="_blank">1569008</a>]<br></li>
<li>On the basis of these results and comparisons of the sequences of the two halves of the inverted-repeat in E. coli and S. typhimunium , it is likely that the sequence recognized by UhpA includes 5 ' - GGCAAAACNNNGAAA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1569008" target="_blank">1569008</a>]<br></li>
<li>Plasmids bearing deletions that removed downstream sequences up to position -46 strongly competed for UhpA , as indicated by their Uhp - phenotype .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1569008" target="_blank">1569008</a>]<br></li>
<li>UhpA binds to multiple sites between positions ÿ80 and ÿ32 ( Dahl et al. , 1997 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10512697" target="_blank">10512697</a>]<br></li>
<li>The insertion or deletion of one helical turn of DNA resulted in the loss of promoter activity and of occupancy by UhpA of the remaining weak-site sequences but was accompanied by normal occupancy of the strong site and no change in the gel retardation behavior of the promoter fragments .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10913075" target="_blank">10913075</a>]<br></li>
<li>In the V10 variant , P-UhpA protects about 20 bp of the weak-site sequences , as well as the strong site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10913075" target="_blank">10913075</a>]<br></li>
<li>These results confirm the conclusions from the gel mobility shift analysis that the 264 element ( 250 to 280 ) is the high affinity binding site for P-UhpA , and show that the sequences immediately downstream and overlapping the putative RNA polymerase-binding region are the low affinity binding region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8999880" target="_blank">8999880</a>]<br></li>
<li>DNase I and hydroxyl radical protection footprinting extended the results of the gel shift experiment by confirming that the 264 element , from coordinates 280 to 250 , was the site of high affinity binding by UhpA and showing that the extension of UhpA binding to downstream sequences occurred at higher concentrations .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8999880" target="_blank">8999880</a>]<br></li>
<li>Here we report that phosphorylation of UhpA by acetyl phosphate occurs on Asp-54 , that UhpA binds specifically to target sequences in the uhpT promoter , and that phosphorylation strongly affects the ability of UhpA to bind to the 264 element and a downstream region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8999880" target="_blank">8999880</a>]<br></li>
<li>The ﬁve sequences of the uhpT promoter that UhpA protects from hydroxyl radicals ( 10 ) ﬁt the consensus sequence AAAYY in the S region ( where Y C or T ) and ARRYY in the W region ( where R A or G ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11976297" target="_blank">11976297</a>]<br></li>
<li>UhpA binds to a dyad element centered at position 264 , and phosphorylated UhpA can then bind to low-afﬁnity sites that extend to position 232 ( 7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10572130" target="_blank">10572130</a>]<br></li>
<li>UhpA and LuxR bind immediately upstream of the RNA polymerase-binding site in uhpT and luxICDABE ( luxI-E ) promoters , respectively ( 27 ) , while DegU binds 300 bp upstream of the start site of the degQ promoter ( 20 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1569008" target="_blank">1569008</a>]<br></li>
<li>Activation by UhpA requires a promoter element centered at 264 bp , relative to the start of transcription , and activation by CAP requires a DNA site centered at position 2103.5 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7896692" target="_blank">7896692</a>]<br></li>
<li>UhpA binds to multiple sites in the uhpT promoter between positions 280 and 232 upstream of the transcription start site , and CAP binds to a single site centered at position 2103.5 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10572130" target="_blank">10572130</a>]<br></li>
<li>CAP binds to the uhpT promoter at a single site , centered at ÿ103 .5 bp relative to the start of transcription , and UhpA binds to multiple sites between positions ÿ80 and ÿ32 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10512697" target="_blank">10512697</a>]<br></li>
<li>The entire transcription control region of the uhpT promoter is relatively small , at 120 bp , and the sites for UhpA binding are limited to the 50-bp region from 280 to 230 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8999880" target="_blank">8999880</a>]<br></li>
<li>Gel electrophoretic mobility shift and DNase I and hydroxyl radical protection assays showed that UhpA bound specifically to the region of the uhpT promoter extending from 280 to 250 bp , relative to the transcription start site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8999880" target="_blank">8999880</a>]<br></li>
<li>Phosphorylation of UhpA strongly increases but is not required for binding to the 80 to 32 region of the uhpT promoter ( all promoter coordinates are relative to the transcription start site ) ( 9 , 10 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11976297" target="_blank">11976297</a>]<br></li>
<li>The locations of the ribosome binding site ( RBS ) , the transcription start point ( right-angled arrow ) , the 210 region , the short inverted-repeat centered at position 232.5 , the proposed site for UhpA action centered at 264 , and the DNA site for CAP are indicated ( 21 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7896692" target="_blank">7896692</a>]<br></li>
<li>There appear to be four elements in the promoter : a typical -10 region , an element centered at -32 which could be involved in RNA polymerase binding , a large inverted-repeat centered at -64 which appears to be the site of UhpA binding , and a CAP-binding sequence centered at -103 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1569008" target="_blank">1569008</a>]<br></li>
<li>At least four sequence elements are required for maximal expression , a 210 region , an inverted-repeat centered at position 232.5 , an inverted-repeat centered at position 64 ( which appears to be the DNA site for UhpA ) , 2 and an inverted-repeat centered at position 2103.5 that matches the consensus sequence for CAP-binding DNA sites ( Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7896692" target="_blank">7896692</a>]<br></li>
<li>This promoter combined the D22 deletion with the insertion of a random 22-bp sequence at position 285 between the UhpA site and the CAP site , to return the CAP-binding site to the same position as in the wild-type promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10913075" target="_blank">10913075</a>]<br></li>
<li>A sequence element of about 40 bp , centered at -64 , has been implicated as the site for UhpA action .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8144492" target="_blank">8144492</a>]<br></li>
<li>Mutational analysis was used to try to identify sequences important for UhpA binding ; ﬁndings are summarized in Fig. 8 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11976297" target="_blank">11976297</a>]<br></li>
<li>These sites were protected by UhpA to a similar degree as were the ﬂanking wildtype sequences .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11976297" target="_blank">11976297</a>]<br></li>
<li>This protection of nonnative sequences inserted within the target site indicates that occupancy by UhpA of a speciﬁc region is not highly dependent on the sequence of that region , as long as the ﬂanking sequences are intact , consistent with a cooperative binding process .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11976297" target="_blank">11976297</a>]<br></li>
<li>UhpA binding to the W sites shows cooperative dependence on occupancy of the S region , but also speciﬁc recognition of sequences in the W region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11976297" target="_blank">11976297</a>]<br></li>
<li>DISCUSSION This study provides information regarding the recognition sequences for UhpA binding , the number and orientation of bound UhpA molecules , the contribution of UhpA-binding sites to transcription activation , and the existence of cooperativity in UhpA binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11976297" target="_blank">11976297</a>]<br></li>
<li>Sequences inserted between the DNA binding sites for CAP and UhpA are shown in boldface and are underlined .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7896692" target="_blank">7896692</a>]<br></li>
<li>These results showed that binding of UhpA to the weak sites depends on both the proper nucleotide sequence and the orientation relative to the strong sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10913075" target="_blank">10913075</a>]<br></li>
<li>Deﬁnition of the UhpA recognition sequence required knowledge of the stoichiometry and orientation of the binding sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11976297" target="_blank">11976297</a>]<br></li>
<li>The Hill coefﬁcients for UhpA binding were ca. 2 at the S region and ca. 3 at the W region ( 9 ) , and the coefﬁcient for transcription activation was in the range of 4 to 6 , which is consistent with the proposal that at least four monomeric UhpA molecules must bind simultaneously for efﬁcient transcription ( 10 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11976297" target="_blank">11976297</a>]<br></li>
<li>The observed presence of a single locus of cleavage sites in the middle of each region is most consistent with the conclusion that UhpA binds to each region as a dyad or tail-to-tail protein dimer .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11976297" target="_blank">11976297</a>]<br></li>
<li>The strongest cleavage speciﬁc for UhpA E187C occurred in the center of the S region , where symmetric cleavage sites were ﬂanked by regions of protection .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11976297" target="_blank">11976297</a>]<br></li>
				</ul></div><br>
			<label class="collapse_TU" for="_6"><font size="5px">Transcription unit regulated by UhpA</font></label>
				<input id="_6" type="checkbox">
				<div class="TU"><br><ul><li>The absence of the llvN and UhpA polypeptides when pLAW034 was used ( Fig. 6 , lane 4 ) showed that both of those genes are transcribed in the same direction , from left to right in the diagram of Fig. 6 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3038843" target="_blank">3038843</a>]<br></li>
<li>Expression of the uhpT promoter is induced by extracellular glucose-6-phosphate ( Glu6P ) acting through an unusual two-component regulatory system ( 17 ) in which the membrane-bound UhpBC sensor kinase complex regulates the phosphorylation and activation of the response regulator UhpA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10913075" target="_blank">10913075</a>]<br></li>
<li>Note that CAP alone does not allow any uhpT transcription in the absence of UhpA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10572130" target="_blank">10572130</a>]<br></li>
<li>This result suggested that overexpressed RpoA interferes with uhpT expression and that aCTD participates in transcription activation by CAP but perhaps not by that of UhpA alone .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10572130" target="_blank">10572130</a>]<br></li>
<li>Transmembrane induction of uhpT expression by external glucose 6-phosphate is positively regulated by the promoter-speciﬁc activator protein UhpA and the global regulator catabolite gene activator protein ( CAP ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7896692" target="_blank">7896692</a>]<br></li>
<li>Transcription of the Escherichia coli uhpT gene , encoding the inducible sugar phosphate transporter , is dependent on the response regulator UhpA and is stimulated by the cyclic AMP receptor protein ( CAP ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10572130" target="_blank">10572130</a>]<br></li>
<li>The Escherichia coli uhpT gene encodes the sugar phosphate transporter UhpT and is induced by extracellular glucose-6-phosphate ( Glc6P ) through action of the transcription activators UhpA and CAP ( 22 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11976297" target="_blank">11976297</a>]<br></li>
<li>The basis for this difference in growth response is under investigation , and it may reflect titration of limiting amounts of the UhpA activator by the multiple copies of the promoter upstream of the nonfunctional uhpT ' gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3038843" target="_blank">3038843</a>]<br></li>
<li>UhpA is essential for detect-able uhpT expression in vivo and in vitro ( 33 , 42 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11976297" target="_blank">11976297</a>]<br></li>
<li>The uhpT-lacZ expression in this mutant decreased more than 10-fold relative to the D22 variant , and it was further reduced in the absence of UhpA ( Table 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10913075" target="_blank">10913075</a>]<br></li>
<li>The uhpT promoter was activated by not only its cognate UhpA but also a noncognate UvrY ( see Table 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26010043" target="_blank">26010043</a>]<br></li>
<li>When present in a single copy , the deletion promoters can be fully occupied by UhpA , so that phosphorylation of UhpA does not increase binding and gene expression noticeably .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10913075" target="_blank">10913075</a>]<br></li>
<li>In all cases , expression of - galactosidase was fully dependent on the presence of UhpA and induction by Glc6P ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11976297" target="_blank">11976297</a>]<br></li>
				</div></ul></div><br>
	</div>
</body>
</html>