<!doctype html>
<style>
@charset "utf-8";
body {
	height: 100%;
	margin: 0;
	padding: 0;
}

.banner {
	position: fixed;
	top: 0;
	width: 100%;
	height: 65px;
	background: gray;
	line-height: 65px;
}
.spacer {
	height: 10px;
}

.main-content {
	padding: 5%;
}

.collapse_ACT {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-style: none solid none none;
	border-color: #B4045F;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_ACT + input{
	display: none;
}
.collapse_ACT + input + div{
	display: none;
}
.collapse_ACT + input:checked + div{
	display: block;
}
.ACT {
	text-align: justify;
	text-justify: inter-word;
}
.ACT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B4045F;
}
.ACT_collapse_similarity:hover{
	background: #e5d3dc;
}
.ACT_collapse_similarity + input {
	display: none;
}
.ACT_collapse_similarity + input + div{
	display: none;
}

.ACT_collapse_similarity + input:checked + div{
	display: block;
}
.ACT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.ACT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B4045F;
	padding-left: 5px;
}
.ACT_collapse_similarity:hover{
	background: #e5d3dc;
}
.ACT_collapse_similarity + input {
	display: none;
}
.ACT_collapse_similarity + input + div{
	display: none;
}
.ACT_collapse_similarity + input:checked + div{
	display: block;
}
.ACT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.ACT_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #B4045F;
}
.ACT_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_DOM {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-style: none solid none none;
	border-color: #B404AE;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_DOM + input{
	display: none;
}
.collapse_DOM + input + div{
	display: none;
}
.collapse_DOM + input:checked + div{
	display: block;
}
.DOM {
	text-align: justify;
	text-justify: inter-word;
}
.DOM_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B404AE;
	padding-left: 5px;
}
.DOM_collapse_similarity:hover{
	background: #e8d7e7;
}
.DOM_collapse_similarity + input {
	display: none;
}
.DOM_collapse_similarity + input + div{
	display: none;
}
.DOM_collapse_similarity + input:checked + div{
	display: block;
}
.DOM_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}.DOM_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #B404AE;
}
.DOM_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_EVO {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #7401DF;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_EVO + input{
	display: none;
}
.collapse_EVO + input + div{
	display: none;
}
.collapse_EVO + input:checked + div{
	display: block;
}
.EVO {
	text-align: justify;
	text-justify: inter-word;
}
.EVO_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #7401DF;
	padding-left: 5px;
}
.EVO_collapse_similarity:hover{
	background: #e2d7ed;
}
.EVO_collapse_similarity + input {
	display: none;
}
.EVO_collapse_similarity + input + div{
	display: none;
}
.EVO_collapse_similarity + input:checked + div{
	display: block;
}
.EVO_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.EVO_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #7401DF;
}
.EVO_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_RP {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #0174DF;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_RP + input{
	display: none;
}
.collapse_RP + input + div{
	display: none;
}
.collapse_RP + input:checked + div{
	display: block;
}
.RP {
	text-align: justify;
	text-justify: inter-word;
}
.RP_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #0174DF;
	padding-left: 5px;
}
.RP_collapse_similarity:hover{
	background: #c9d7e2;
}
.RP_collapse_similarity + input {
	display: none;
}
.RP_collapse_similarity + input + div{
	display: none;
}
.RP_collapse_similarity + input:checked + div{
	display: block;
}
.RP_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.RP_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #0174DF;
}
.RP_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_SIT {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #0489B1;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_SIT + input{
	display: none;
}
.collapse_SIT + input + div{
	display: none;
}
.collapse_SIT + input:checked + div{
	display: block;
}
.SIT {
	text-align: justify;
	text-justify: inter-word;
}
.SIT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #0489B1;
	padding-left: 5px;
}
.SIT_collapse_similarity:hover{
	background: #d0dfe2;
}
.SIT_collapse_similarity + input {
	display: none;
}
.SIT_collapse_similarity + input + div{
	display: none;
}
.SIT_collapse_similarity + input:checked + div{
	display: block;
}
.SIT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.SIT_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #0489B1;
}
.SIT_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_TU {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #04B486;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_TU + input{
	display: none;
}
.collapse_TU + input + div{
	display: none;
}
.collapse_TU + input:checked + div{
	display: block;
}
.TU {
	text-align: justify;
	text-justify: inter-word;
}
.TU_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #04B486;
	padding-left: 5px;
}
.TU_collapse_similarity:hover{
	background: #d0e2de;
}
.TU_collapse_similarity + input {
	display: none;
}
.TU_collapse_similarity + input + div{
	display: none;
}
.TU_collapse_similarity + input:checked + div{
	display: block;
}
.TU_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.TU_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #04B486;
}
.TU_similar > a {
	text-decoration: none;
	color: gray;
}

ul { 
	list-style-type: none;
	padding: 0;
	margin: 0;
}
ul li > a {
	text-decoration: none;
	color: gray;
}
ul li:hover, ul li:focus {
	background: #F0F0F0;
}

ul li:first-child span {
  border-left:none;  
}

</style>
<html>
<head>
<meta charset="utf-8">
<title>TF properties - XylR</title>
<meta name="theme-color" content="#ffffff">
</head>
<body>
	<div class="banner">
		<center><font color="white" size="16px" face="arial">XylR</font></center>
	</div>
	<div align="center" class="main-content">
		<div class="spacer">
	</div><br>			<label class="collapse_ACT" for="_1"><font size="5px">XylR transcriptional activity</font></label>
				<input id="_1" type="checkbox">
				<div class="ACT"><br><ul><li>YRVGGYAPVKV DVRIIAATHQNLEQRVQEG KFREDLFHRLNVIRVHLPPLRERREDI YRVGGYAPVKV DVRIIAATHQNLELRVQEG KFREDLFHRLNVIRVHLPPLRERREDI TTVGGRTPIKT DVRIVAASNKDLRILIQQG LFREDLFFRLNVVPLRVPPLRERIEDL ALPD PPRIMATTQVDLGALMEAGRRFRQDLYYRLGGVTLAVPALRERVEDI TPLGTNEVRPV NLRVVAAAKIDLGDPAVRG DFREDLYYRLNVVTISIPPLRERRDDI TPLGTNLTRPV DIRWAAAKVDLGDPAARG DFREDLYYRLNVVTLSIPPLRERRDDI RP FRLVGVGSASLVEQAAAN QIAAELYYCFAMTQIACQSLSQRPDDI ERVGDPVTRRA DVRILAATNRDLGAMVAQG QFREDLLYRLNVIVLNLPPLRERAEDI QRVGSNQIISV DVRLIAATHRDLAAEVNAG RFRQDLYYRLNVVAIEVPSLRQRREDI ERVGGDRTRKV DVRLITATNENLEEAVKMG RFRADLFFRLNVFPVHIPPLRERVEDI ERVGGSKPVKV DVRIIAATNRNLVEMVEQG TFREDLYYRLNVFPITVPPLRERGSDV ERVGGNTTIHV DLRVIAATNRNLAEMVADG TFAEDLYYRLNVFPITIPPLRERGSDI ERLGSNKIIQT DVRLIAATNRDLKKMVADR EFRSDLYYRLNVFPIHLPPLRERPEDI ERVGGDETLRV NVRIIAATNRHLEEEVRLG HFREDLYYRLNVMPIALPPLRERQEDI DRVGGSKPVKV NIRILATSNRDLAQAVKDG TFREDLLYRLNVVNLRLPPLRERPADV ERLGGTKTLKV DVRVICATNKNLEVAVLRG EFRADLYYRINVVPIILPPLRQRDGDI ERVGGTKTLKV DVRLIFATNKDLEMAVQNG EFREDLYYRISGVPLILPPLRHRDGDI ERVGGTRTLKV NVRLVCATNKDLETAVAAG EFRADLYYRINVVPITLPPLRQRDGDI ERVGGAKTIKV DTRT ~ AATNRDLEDAVARG QSRADLHPRICVVPSVLPPLRCRRSDI ERVGGNRTLKV DqRLVCATNKNLEEAVSKG EFRADLYYRIHVVPLILPPLRERPGDI ERVGSNHTIKV DVRVIAATNRNLEEAVARS EFRADLYYRISVVPLLLPPLRERESDI RRVGEDHEVHV DVRVICATQKNLVELVQKG MFREDLYYRLNVLTLNLPPLRDCPQDI NRLGESEHKRTSNVLFICATTENPSSALLKT FLRRIPMTIHIPSLEERSLNE NtrC Eco NtrC Kpn NtrC Bpa NifRl Rca DctD Rle DctD Rme PgtA Sty AIgB Pae HydG Eco XylR Ppu AnfA Avi VnfA Avi FhlA Eco NifA Kpn FIbD Ccr NifA Rle NifA ~ me NifA rlBT NifA Rca NifA Aca Ni fA Bj a TyrR Eco LevR Bsu ERVGG ... .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7699720" target="_blank">7699720</a>]<br></li>
<li>Activators that , like PspF , lack an N-terminal domain ( e.g. CheB , HrpR and HrpS ) or in which the N-terminal domain has been deleted ( e.g. DctD , NifA , XylR and DmpR ) have elevated or constitutive activity ( Ronson , 1988 ; Huala & Ausubel , 1989 ; Huala et al. , 1992 ; Gu et al. , 1994 ; Lee et al. , 1994 ; Xiao et al. , 1994 ; Berger et al. , 1995 ; Grimm et al. , 1995 ; Shingler & Pavel , 1995 ; Perez-Martin & Lorenzo , 1996b ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9878422" target="_blank">9878422</a>]<br></li>
<li>Search with the XylR pro cents le and comparison of genes having strong candidate sites in the upstream regions have lead to the identi cents cation , in the E. coli , S. typhi and K. pneumoniae genomes , of putative operons , which are likely to be regulated by XylR and whose products probably mediate transport and hydrolysis of xylooligosaccharides .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11750821" target="_blank">11750821</a>]<br></li>
<li>Although only a minor shift was observed with unliganded XylR by mobility shift assay , the footprinting assay clearly demonstrated that XylR binds to IF1 in both induced and uninduced conditions and that three half sites , IF2 , IA1 , and IA2 , were occupied under induced conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9371449" target="_blank">9371449</a>]<br></li>
<li>A new XylR version with novel mutations was reported , in which the natural response to m-xylene was inhibited while the nonnative acquired response to 2,4-DNT was enhanced ( de las Heras and de Lorenzo 2011 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25981994" target="_blank">25981994</a>]<br></li>
<li>It is tempting to propose that , in all of these cases , ATP hydrolysis is coupled to dissociation of the oligo-meric structure formed by the protein , as inferred for XylR ( 28 ) and as suggested for MalT .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10559195" target="_blank">10559195</a>]<br></li>
<li>As shown in Fig. 6 , guanine residues at positions A-59 , A-61 , F-59 , and F-38 on the top strand and at positions A-49 , A-70 , F-68 , and F-67 on the bottom strand were protected by XylR from methylation upon xylose induction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9371449" target="_blank">9371449</a>]<br></li>
<li>When there is no inducer , only two positions of IF1 , F-59 and F-65 ( hotspot ) , showed modiﬁcations on the top strand , suggesting that XylR protein interacts with the region under this condition .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9371449" target="_blank">9371449</a>]<br></li>
<li>Pérez - Martin , J. and De Lorenzo , V. ( 1995 ) The sigma 54-dependent promoters Ps of the TOL plasmid of Pseudomonas patida requires HU for transcriptional activation in vivo by XylR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9218807" target="_blank">9218807</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_10">
An upstream XylR - and IHF-induced nucleoprotein complex regulates the n54-dependent Pu promoter of TOL plasmid .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1433305" target="_blank">1433305</a>]<br></label>
<input id="act_similarity_10", type="checkbox">
<div class="ACT_similar">
<li>An upstream XylR - and IHF-induced nucleoprotein complex regulates the s54-dependent Pu promoter of TOL plasmid .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9098055" target="_blank">9098055</a>]<br></li></div>
				<label class="ACT_collapse_similarity" for="act_similarity_12">
In vitro activities of an N-terminal truncated form of XylR , a s54 ± dependent transcriptional-activator of Pseudo-monas putida .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9878422" target="_blank">9878422</a>]<br></label>
<input id="act_similarity_12", type="checkbox">
<div class="ACT_similar">
<li>In vitro activities of an N-terminal truncated form of XylR , a s54-dependent transcriptional-activator of Pseudo-monas putida .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8981978" target="_blank">8981978</a>]<br></li></div>
<li>Going around the tree in the clockwise direction , a single protein , the XylR protein of Pseudomonas putida , which is not a response-regulator , is found by itself and does not cluster with any of the other effector domains of this group .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7699720" target="_blank">7699720</a>]<br></li>
<li>Activation of the transcriptional-regulator XylR of Pseudomonas putida by release of repression between functional domains .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15528672" target="_blank">15528672</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8981978" target="_blank">8981978</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9098055" target="_blank">9098055</a>]<br></li>
<li>The latter was shown for DmpR , XylR and PhhR , which respond to aromatic inducers [ 19 ± 21 ] and other regulators , among them FhlA , which is activated by formate , the substrate of the FHL complex [ 13 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10848985" target="_blank">10848985</a>]<br></li>
<li>DISCUSSION FhlA , like XylR ( 7 , 18 , 27 ) and DmpR ( 37 , 38 , 40 ) , is activated not by phosphorylation but by binding of a ligand , in this case formate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8981978" target="_blank">8981978</a>]<br></li>
<li>The observation that overexpression of the N-terminally truncated but not full-length RtcR induces transcription of rtcA and rtcB suggests that , as in the case of several previously characterized s54-specific regulators ( e.g. XylR , DmpR , DctD , and LevR ( 66 -- 69 ) ) , the N-terminal domain represses activity of RtcR , and its deletion makes the regulator constitutively active .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9738023" target="_blank">9738023</a>]<br></li>
<li>NtrC is activated by phoshorylation of the aspartate 54 residue located in the N-terminal receiver domain causing oligomerisation , and DctD by its phosphorylated receiver domain relieving inhibition imposed by the non-phosphorylated recei-ver domain upon the central AAAþ domain ( Figure 1 ) .24 Some activators of s54-dependent transcription are regulated by the binding of a small ligand to the N-terminal domain , such as binding xylene to XylR .24 In contrast , other s54 activators are regulated through their N-terminal domains being targeted by a partner second protein which imposes negative control .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12079332" target="_blank">12079332</a>]<br></li>
<li>In this case , CRP -- cAMP might 54 compete with activators , such as NtrC , NifA , DctD , XylR , etc. for Es to downregulate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11532153" target="_blank">11532153</a>]<br></li>
<li>The inducer for NagC is GlcNAc-6-P ( Plumbridge , 1991 ) , while for XylR from B. subtilis , it is the unphosphorylated sugar , xylose ( Gärtner et al. , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767573" target="_blank">9767573</a>]<br></li>
<li>This may also explain why CRP -- cAMP did not show any signiﬁcant effect on the expression of XylR activated operons under activating conditions with high inducer concentrations ( Hugouvieux-Cotte-Pattat et al. , 1990 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11532153" target="_blank">11532153</a>]<br></li>
<li>Before the DMS treatment , amounts of XylR expressed were conﬁrmed by Western analysis with anti-XylR antiserum to be similar in induced and uninduced conditions ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9371449" target="_blank">9371449</a>]<br></li>
<li>Without XylR , there is no change in DMS modiﬁcation regardless of the presence of inducer ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9371449" target="_blank">9371449</a>]<br></li>
<li>The experimental data [ 4 ] are indicated : the guanines protected from the DMS methylation by XylR are shaded and the adenines whose methyl-ation was enhanced in the presence of XylR are shaded and marked with asterisks .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11750821" target="_blank">11750821</a>]<br></li>
<li>Using genetic approaches , they concluded that this repression was XylR dependent , though it is not known whether this involved direct repression of ribose metabolic gene expression or an indirect mechanism , for example , involving the xylose transporters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20023096" target="_blank">20023096</a>]<br></li>
<li>Only a few of the transporters and regulators in this group bind the same substrates ; galactose ( MglB and GalR ) , ribose ( RbsR and RbsB ) and xylose ( XylF and XylR ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15757509" target="_blank">15757509</a>]<br></li>
<li>Members of these families have the opposite modes of action : RbsR is a classical repressor whose true inducer is D-ribose , whereas AraC and XylR are primarily activators of transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11750821" target="_blank">11750821</a>]<br></li>
<li>Keywords : Computer analysis ; Transcriptional regulation ; RbsR ; AraC ; XylR ; cAMP receptor protein 1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11750821" target="_blank">11750821</a>]<br></li>
<li>ons of the RbsR , AraC and XylR regulons .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11750821" target="_blank">11750821</a>]<br></li>
<li>( B ) The xylose metabolic genes and transporters are regulated by XylR , an AraC-type positive regulator , in a manner analogous to that of arabinose ( 38 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20023096" target="_blank">20023096</a>]<br></li>
<li>AraC activates the arabinose-catabolism genes araAB , araE , and araFGH , and suppresses the xylose-catabolism genes xylAB and xylFGH by inhibiting XylR , the xylose transcriptional-activator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26045332" target="_blank">26045332</a>]<br></li>
<li>Based on our results with the arabinose transporter deletion strains , arabinose likely inhibits xylose gene expression by one of two mechanisms : either ( i ) arabinose-bound AraC binds to the PxylA promoter and prevents it from being activated by xylose-bound XylR or ( ii ) arabinose directly binds XylR and inhibits its activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20023096" target="_blank">20023096</a>]<br></li>
<li>Therefore , we conclude that intracellular arabinose is not inhibiting the activity of XylR but rather is repressing PxylA promoter activity through the action of AraC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20023096" target="_blank">20023096</a>]<br></li>
<li>Alternatively , if repression is due to inhibition of XylR by arabinose , then repression should still occur .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20023096" target="_blank">20023096</a>]<br></li>
<li>Because repression is AraC dependent yet did not involve an arabinose metabolic intermediate , we suspected that intracellular arabinose was somehow interacting with either AraC or XylR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20023096" target="_blank">20023096</a>]<br></li>
<li>These results show that repression is dependent only on the activity of AraC and does not involve a mechanism where arabinose inhibits-the-activity of XylR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20023096" target="_blank">20023096</a>]<br></li>
<li>This suggests some sort of functional interaction between CRP and XylR , e.g. co-opera-tive binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11750821" target="_blank">11750821</a>]<br></li>
<li>Conserved relative positioning of XylR and CRP in gamma Proteobacteria suggests close functional interaction between these regulators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11750821" target="_blank">11750821</a>]<br></li>
<li>These results are consistent with the locations of various regulatory sites proposed and imply an involvement of CRP and XylR in PA activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9371449" target="_blank">9371449</a>]<br></li>
<li>However , in most cases , the positions of CRP and XylR sites conform to the period of the DNA helix .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11750821" target="_blank">11750821</a>]<br></li>
<li>Binding of XylR to the xyl promoter was enhanced by the presence of D-xylose , suggesting that transcription was positively regulated by XylR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9371449" target="_blank">9371449</a>]<br></li>
<li>The xylA promoter is positively regulated by XylR in response to xylose [ 20 ] and can equivalently be used to determine relative intracellular xylose concentration .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22952739" target="_blank">22952739</a>]<br></li>
<li>D-Xylose enhanced the binding of XylR to the target DNA , which is proportional to the XylR concentration ( Fig. 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9371449" target="_blank">9371449</a>]<br></li>
<li>Two regions of enhanced XylR binding in the presence of xylose were revealed in the xylAB/xylFGH intergenic space by DMS footprinting [ 4 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11750821" target="_blank">11750821</a>]<br></li>
<li>Binding of XylR to the PA and PF promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9371449" target="_blank">9371449</a>]<br></li>
<li>It was experimentally demonstrated that XylR acts as a transcriptional-activator for the promoters of the xylAB and xylFGH operons , but not for the xylR promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11750821" target="_blank">11750821</a>]<br></li>
<li>Xylose-bound XylR is believed to activate the transcription of the xylFGH , xylR , xylAB , and xylE operons .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20023096" target="_blank">20023096</a>]<br></li>
<li>One such example is the s54-specific activators the oligomerization of which is modulated by the binding of effector ( s ) or covalent modification , as shown with XylR and NtrC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10559195" target="_blank">10559195</a>]<br></li>
<li>Genetic evidence for activation of the positive transcriptional-regulator XylR , a member of the NtrC family of regulators , by effector binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8981978" target="_blank">8981978</a>]<br></li>
<li>These observations demonstrate that FhlA binds to one side of the DNA , a binding mode also pointed out for XylR [ 48 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10848985" target="_blank">10848985</a>]<br></li>
<li>XylR multimerization , which occurs upon UAS binding , seems to depend on the binding of both the inducer and ATP ( 28 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10559195" target="_blank">10559195</a>]<br></li>
<li>ATP binding to the s54-dependent activator XylR triggers a protein multimerization cycle catalyzed by UAS DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9878422" target="_blank">9878422</a>]<br></li>
<li>The XylR protein expressed was active in DNA binding as measured by b-galactosidase activities from xylG1-lacZ in a xylR-negative background .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9371449" target="_blank">9371449</a>]<br></li>
<li>The binding of XylR to the promoter region was initially attempted with puriﬁed XylR protein , which turned out to be unsuccessful due to a loss of DNA-binding activity of the pu-riﬁed XylR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9371449" target="_blank">9371449</a>]<br></li>
				</div></ul><br>
			<label class="collapse_DOM" for="_2"><font size="5px">XylR protein domains</font></label>
				<input id="_2" type="checkbox">
				<div class="DOM"><br><ul><li>5 100 2 prpR JE4766 JE4593 PrpR ( WT ) PrpR ( DA domain ) c 940 ± 45 62 prpR234 prpR235 prpR236 JE6634 JE6635 180 ± 26 260 ± 26 19 28 12 17 F129L E50K prpR237 prpR238 JE6636 JE6637 190 ± 4 980 ± 15 20 13 65 D37N A162T 104 S33R F49L prpR239 prpR240 JE6638 JE6639 580 ± 22 415 ± 21 62 44 38 28 E172G F49S prpR241 prpR242 JE6672 JE6673 65 ± 4 170 ± 9 7 4 18 11 S33N E142K prpR243 prpR244 JE6674 JE6675 500 ± 4 530 ± 77 53 56 33 35 R24S prpR245 prpR246 JE6865 JE6762 490 ± 10 85 ± 12 52 9 32 6 Y180H L47P S21P prpR247 prpR248 JE6763 JE6764 360 ± 22 90 ± 5 38 24 9 6 R24H G104E prpR249 prpR250 JE6765 JE6766 350 ± 47 70 ± 14 37 23 7 5 K81H F178C prpR251 prpR252 JE6767 JE6768 350 ± 12 570 ± 13 37 23 61 36 38 23 340 ± 34 JE6769 D31G prpR253 PrpR is different from other well-studied members of this family of activators , such as XylR and DmpR , in which the region linking the sensing and catalytic domains contains a coiled-coil structure that is important for controlling binding of the effector molecule and activation of the protein ( Garmendia & de Lorenzo , 2000 ; O'Neill et al. , 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15528672" target="_blank">15528672</a>]<br></li>
<li>The role of the interdomain B linker in the activation of the XylR protein of Pseudomonas putida .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15528672" target="_blank">15528672</a>]<br></li>
<li>Going around the tree in the clockwise direction , a single protein , the XylR protein of Pseudomonas putida , which is not a response regulator , is found by itself and does not cluster with any of the other effector domains of this group .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7699720" target="_blank">7699720</a>]<br></li>
<li>Activation of the transcriptional regulator XylR of Pseudomonas putida by release of repression between functional domains .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15528672" target="_blank">15528672</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8981978" target="_blank">8981978</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9098055" target="_blank">9098055</a>]<br></li>
<li>These include the CheY-like receiver-domain in NtrC , the GAF-domain in FhlA , the ACT and PAS domains in TyrR , and the 4-vinyl reductase ( 4VR ) domain in XylR ( Fig. 5 ; Anantharaman et al. , 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15037234" target="_blank">15037234</a>]<br></li>
<li>For DmpR , XylR and DctD it was shown that truncated proteins that lack the N-terminal A domain are constitutively active [ 26 ± 29 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10848985" target="_blank">10848985</a>]<br></li>
<li>This contention is supported by the phenotype of mutants altered in other regulators of the NtrC family : DctD and some constitutive mutants of NtrC function in the absence of the N-terminal domain which is the site for phosphorylation , and XylR ( 10 ) and DmpR ( 40 ) become constitutively active in ATP hydrolysis and transcriptional activation when the binding domain has been removed ( 13 , 17 , 27 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8981978" target="_blank">8981978</a>]<br></li>
<li>Removal of the A domain , like in XylR [ 28 ] , renders ATPase activity independent of the effector .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10848985" target="_blank">10848985</a>]<br></li>
<li>shown that DmpR and XylR bind the respective phenolic li-gands with an N-terminal domain ( 39 ) ; also , stimulation of the ATPase activity by phenols has been demonstrated with the aid of a partially puriﬁed DmpR preparation ( 40 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8981978" target="_blank">8981978</a>]<br></li>
<li>NtrC is activated by phoshorylation of the aspartate 54 residue located in the N-terminal receiver-domain causing oligomerisation , and DctD by its phosphorylated receiver-domain relieving inhibition imposed by the non-phosphorylated recei-ver domain upon the central AAAþ domain ( Figure 1 ) .24 Some activators of s54-dependent transcription are regulated by the binding of a small ligand to the N-terminal domain , such as binding xylene to XylR .24 In contrast , other s54 activators are regulated through their N-terminal domains being targeted by a partner second protein which imposes negative control .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12079332" target="_blank">12079332</a>]<br></li>
<li>( A ) Alignment of the HTH motifs in the N-terminal regions of NagC and Mlc from E.coli and the XylR repressor from Bacillus subtilis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11139621" target="_blank">11139621</a>]<br></li>
<li>Not-withstanding the fact that the TyrR protein contains an ATP-binding domain and a helix-tum-helix DNA-bind-ing domain which are structurally homologous to domains of similar functions in proteins such as NifA , NtrC , DctD and XylR , it differs from them in a number of respects .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1943694" target="_blank">1943694</a>]<br></li>
<li>A similarity of XylR protein to other AraC-family proteins whose DNA-binding motifs are organized as two helix-turn-helix ( HTH ) structures lies in the COOH-terminal portion of the peptide as for other proteins in the group ( 9 , 45 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9371449" target="_blank">9371449</a>]<br></li>
<li>This family consists of two classes of proteins : the transcriptional regulatory proteins exempliﬁed by NagC and XylR , and the somewhat smaller proteins encoding sugar kinases which are missing the N-terminal helix-turn-helix DNA-binding domain present in the transcription factors .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9864311" target="_blank">9864311</a>]<br></li>
<li>The predicted DNA-binding domain of XylR contains two HTH structures in the C-terminal region , comprising 304 to 323 and 353 to 372 amino acid residues .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9371449" target="_blank">9371449</a>]<br></li>
<li>The amino-terminal domain of the prokaryotic enhancer-binding protein XylR is a speciﬁc intramolecular repressor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15528672" target="_blank">15528672</a>]<br></li>
<li>It is possible that unliganded XylR and some other DNA-binding protein ( s ) may associate with this region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9371449" target="_blank">9371449</a>]<br></li>
<li>The NagC and XylR repressors show signiﬁcant differences in their protein sequences in the region , which are predicted to form the helix -- turn -- helix DNA-binding motif .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9484892" target="_blank">9484892</a>]<br></li>
				</div></ul><br>
			<label class="collapse_EVO" for="_3"><font size="5px">XylR evolution</font></label>
				<input id="_3" type="checkbox">
				<div class="EVO"><br><ul><li>It was reported that AlsK has similarity to XylR of Bacillus subtilis ( 23.6 % identity [ 3 ] ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9401019" target="_blank">9401019</a>]<br></li>
<li>In our analysis , however , AlsK is more similar to glucokinase of Streptomyces coelicolor ( 25.9 % ) than to XylR of B. subtilis ( 19.1 % ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9401019" target="_blank">9401019</a>]<br></li>
<li>Other members of the FhlA subclass are NifA ( 8 ) , DmpR ( 37 , 38 ) , and XylR ( 18 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8981978" target="_blank">8981978</a>]<br></li>
<li>One of these was the 35-40 % identity of the central and carboxyl-terminal domains of the TyrR protein ( a repressor for genes of aromatic amino acid biosynthesis and transport ) with a number of regulator proteins such as NtrC , NifA , XylR , FhIA , HydG and DctD , which activate expression from promoters ustng the E < r54 RNA polymerase holoenzyme ( Stock et ai , 1989 ; Schlensog and Bock , 1990 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1943694" target="_blank">1943694</a>]<br></li>
<li>Regarding carbon metabolism , members of the family control the degradation of sugars such as arabinose ( AraC ) , cellobiose ( CelD ) , melibiose ( MelR and MsmR ) , rafﬁnose ( RafR and MsmR ) , rhamnose ( RhaR and RhaS ) , and xylose ( XylR ) ; certain amino acids such as valine ( MmsR ) , arginine ( AdiY ) , and ornithine ( OruR ) ; alcohols such as 1,2-propanediol ( PocR ) ; alkylbenzoates ( XylS ) ; p-hydroxyphenylacetic acid ( HpaA ) ; and herbicides such as S-ethyl dipropylthiocarbamate ( TchR ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9409145" target="_blank">9409145</a>]<br></li>
<li>In the Sinorhizobium meliloti and Agrobacterium tumefaciens ge-nomes , the xylA and xylB genes form a putative operon with a LacI family regulator , which might function as XylR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11750821" target="_blank">11750821</a>]<br></li>
<li>Members of these families have the opposite modes of action : RbsR is a classical repressor whose true inducer is D-ribose , whereas AraC and XylR are primarily activators of transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11750821" target="_blank">11750821</a>]<br></li>
<li>The systems for utilization of these sugars are regulated on the transcriptional level by protein factors which belong to di ¡ erent structural families : the ribose regulator ( RbsR ) belongs to the LacI family , whereas the arabinose ( AraC ) and xylose ( XylR ) regulators are members of the AraC family .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11750821" target="_blank">11750821</a>]<br></li>
<li>The system for transport and utilization of these sugars are regulated on the transcriptional level by transcription factors which belong to different structural families : the ribose regulator ( RbsR ) belongs to the LacI family ( Mauzy & Hermodson , 1992a ) , whereas the arabinose ( AraC ) ( Stoner & Schleif , 1983 ; Miyada et al. , 1984 ) and xylose ( XylR ) ( Song & Park , bers of these families have the opposite modes of action : RbsR is a classical repressor whose inducer is D-ribose , whereas AraC and XylR are activators of transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23651393" target="_blank">23651393</a>]<br></li>
<li>In this regard , XylR is similar to other members of the AraC family regulators that bind to direct repeat structures with a gap of 4 bp , spanning about four helical turns as a whole ( 23 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9371449" target="_blank">9371449</a>]<br></li>
<li>The norR promoter was signiﬁcantly more active in the norR mutant , which is consistent with negative autoregu lation , as is seen for other members of the 54-dependent family of regulators , such as XylR ( 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12142437" target="_blank">12142437</a>]<br></li>
<li>A similarity of XylR protein to other AraC family proteins whose DNA-binding motifs are organized as two helix-turn-helix ( HTH ) structures lies in the COOH-terminal portion of the peptide as for other proteins in the group ( 9 , 45 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9371449" target="_blank">9371449</a>]<br></li>
<li>This family consists of two classes of proteins : the transcriptional regulatory proteins exempliﬁed by NagC and XylR , and the somewhat smaller proteins encoding sugar kinases which are missing the N-terminal helix-turn-helix DNA binding domain present in the transcription factors .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9864311" target="_blank">9864311</a>]<br></li>
<li>The peculiarity of the NagC and Mlc proteins is that the sequence of the amino acids in their recognition helices is almost identical and much more similar to each other than to the equivalent sequences in the XylR family ( Fig. 1A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11139621" target="_blank">11139621</a>]<br></li>
<li>Mlc is a member of the so-called ROK family of proteins ( Titgemeyer et al. , 1994 ) , which includes NagC , the repressor for the nagE -- BACD operons ( Plumbridge , 1991 ; Plumbridge and Kolb , 1995 ) , and XylR , the repressor of the xylose operons , from a number of Gram-positive bacteria plus certain glucose and fructose kinases .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767573" target="_blank">9767573</a>]<br></li>
<li>The first class is the transcriptional regulators , notably the family of various xylose repressors ( XylR ) in Gram-positive bacteria , and NagC and Mlc in E.coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11139621" target="_blank">11139621</a>]<br></li>
<li>Genetic evidence for activation of the positive transcriptional regulator XylR , a member of the NtrC family of regulators , by effector binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8981978" target="_blank">8981978</a>]<br></li>
				</div></ul><br>
			<label class="collapse_RP" for="_4"><font size="5px">Processes regulated by XylR</font></label>
				<input id="_4" type="checkbox">
				<div class="RP"><br><ul><li>Furthermore , computer analysis permitted us to classify the sdaR gene product within a set of proteins that present a conserved helix-turn-helix motif , including other gene-expression regulators such as XylR ( 13 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10762278" target="_blank">10762278</a>]<br></li>
<li>is TyrR E. coli ( P07604 ) , is TyrR Haemophilus influenzae ( P44694 ) , is PhhR Psedomonas putida ( Q52177 ) , is NifA Azotobacter Vinelandii ( P09570 ) , is XylR Psedomonas putida ( P21940 ) , is PilR Psedomonas aeruginosa ( Q00934 ) , is NtrC E. coli ( P06713 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11923293" target="_blank">11923293</a>]<br></li>
<li>One of these was the 35-40 % identity of the central and carboxyl-terminal domains of the TyrR protein ( a repressor for genes of aromatic-amino-acid biosynthesis and transport ) with a number of regulator proteins such as NtrC , NifA , XylR , FhIA , HydG and DctD , which activate expression from promoters ustng the E < r54 RNA polymerase holoenzyme ( Stock et ai , 1989 ; Schlensog and Bock , 1990 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1943694" target="_blank">1943694</a>]<br></li>
<li>Regarding carbon metabolism , members of the family control the degradation of sugars such as arabinose ( AraC ) , cellobiose ( CelD ) , melibiose ( MelR and MsmR ) , rafﬁnose ( RafR and MsmR ) , rhamnose ( RhaR and RhaS ) , and xylose ( XylR ) ; certain amino acids such as valine ( MmsR ) , arginine ( AdiY ) , and ornithine ( OruR ) ; alcohols such as 1,2-propanediol ( PocR ) ; alkylbenzoates ( XylS ) ; p-hydroxyphenylacetic acid ( HpaA ) ; and herbicides such as S-ethyl dipropylthiocarbamate ( TchR ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9409145" target="_blank">9409145</a>]<br></li>
<li>Mlc is a member of the so-called ROK family of proteins ( Titgemeyer et al. , 1994 ) , which includes NagC , the repressor for the nagE -- BACD operons ( Plumbridge , 1991 ; Plumbridge and Kolb , 1995 ) , and XylR , the repressor of the xylose operons , from a number of Gram-positive bacteria plus certain glucose and fructose kinases .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9767573" target="_blank">9767573</a>]<br></li>
<li>VirB is itself an activator of the ipaABCD virulence regulon ; S. ﬂexneri and S. sonnei proteins are identical to the S. dysenteriae protein 339 V38K 97 Shigella dysenteriae SP : Q04248 129 , 220 , 264 262 VirF Shigella ﬂexneri Shigella sonnei 53 271 VirF Yersinia enterocolitica Transcriptional activator of the Yersinia virulence regulon comprising yop , ysc , yadA and ylpA genes ; the Y. pseudotuberculosis protein is 99 % identical to the Y. enterocolitica protein SP : P13225 Yersinia pseudotuberculosis Escherichia coli Probable transcriptional activator of the xylBAFGHR operon , which seems to be implicated in the catabolism of xylose Similar to E. coli XylR Transcriptional activator of the pWW0 plasmid meta operon ( xylXYZLTEGFJQKIH ) , required for the degradation of benzoate and substituted derivatives Transcriptional activator of the pWW53 plasmid meta 1 and 2 operons for benzoate ca-tabolism and substituted derivatives SP : P37390 231 392 XylR XylR XylS Haemophilus inﬂuenzae Pseudomonas putida 74 387 321 SP : P45043 SP : P07859 119 , 169 , 235 10 321 XylS1 Pseudomonas putida SP : Q04710 TABLE 1 -- Continued Function Protein Microorganism No .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9409145" target="_blank">9409145</a>]<br></li>
<li>Upstream regulatory sequence for transcriptional activator XylR in the first operon of xylene-metabolism on the TOL plasmid .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1433305" target="_blank">1433305</a>]<br></li>
<li>AraC activates the arabinose-catabolism genes araAB , araE , and araFGH , and suppresses the xylose-catabolism genes xylAB and xylFGH by inhibiting XylR , the xylose transcriptional activator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26045332" target="_blank">26045332</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_9">
[ 4 ] Song , S. and Park , C. ( 1997 ) Organization and regulation of the D-xylose operons in Escherichia coli K-12 : XylR acts as a transcriptional activator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11750821" target="_blank">11750821</a>]<br></label>
<input id="rp_similarity_9", type="checkbox">
<div class="RP_similar">
<li>Organization and regulation of the D-xylose operons in Escherichia coli K-12 : XylR acts as a transcriptional activator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20023096" target="_blank">20023096</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9401019" target="_blank">9401019</a>]<br></li></div>
				</div></ul><br>
			<label class="collapse_SIT" for="_5"><font size="5px">XylR binding site</font></label>
				<input id="_5" type="checkbox">
				<div class="SIT"><br><ul><li>CRP site Score Operon Site The AraC regulon E. coli CG D araBAD/araC 3.17 7.90 8.13 8.31 7.19 7.00 6.26 3.48 7.66 8.11 7.44 7.00 2.82 7.90 8.35 7.75 6.81 7.50 6.22 3.14 3.97 7.72 6.97 gAaaccAATtgTCCATA D TaaCAaAAgtgTCtATA - ( 4 ) - cgGCAGAAaAgTCCAcA TAGCAttTTtATCCATA - ( 4 ) - TAGCgGATcctaCCtgA cAaGGATTtccagGCTA - ( 4 ) - TATGGATTAATCTGCTg TATGtcTTtTcCcGCTA - ( 4 ) - TATGcAcgtTctcaCTg D ttATtTGcaCggcgTCACAcTT 3131 4.07 7 araFGH 7 cgATGTGATaTtGcTCtCcTaT AatTGgaATaTccATCACATaa tAtccTGcaagctATCACtTTa 3.67 4.25 3.21 D araE araJ cAGCAatTTAATCCATA - ( 4 ) - TgctgttTccgaCCtgA cAGCAGgATAATgaATA - ( 4 ) - ggGgcGAATtATCtcTt D CG D araBAD/araC S. typhi gAaacaAATtgTCCATA TAaCAGAAgtgTCtATA - ( 4 ) - TgGCtGgAatgTCCAcA TAGCAttTTtgTCCATA - ( 4 ) - TAGCgGATcctgCCtgA D D AtATtTGcaCagcgTCACAcTT AAgTGTaATaTccATCACATaT 3131 3131 4.05 4.78 D araE araJ TAGCcatTTAATCCATA - ( 4 ) - TgcCgttTccAgCCtgA TAGCAtAATAATgaATA - ( 4 ) - aAGacGAATAAggaATg ggcaccAATtgTCCATA TAaCAaAAgtgTCtATA - ( 4 ) - cgGCAGAAaAgTCCAcA cAGCAaAATAATCCATA - ( 4 ) - TAGCgaATccggCCtgA D D araBAD/araC K. pneumoniae UG D D AtATtTGcaCggcgTCACAcTT 3130 4.08 7 araFGH cAaGGATaAgcCTGCTg - ( 4 ) - TATaGATgATcCTGCTA TATGtcaTtTTtTGCTA - ( 4 ) - TATGtAcgcAatTaCTg cAGCAatATAgTCCATA - ( 4 ) - TgGCttATcctgCCtgA cATGaAcgAATCaGgct - ( 4 ) - TATGGAggAAaCTGCcg TAGCtGAATgtgaCATA 7 cgATGTGATaatGcTCtCgTaT AagTtTaATgTccATCACAaaa 3161 3131 3.63 4.41 D araE araJ 7 CG D araBA/araFGHC Y. pestis D TAGttaATTtATgCATA D cAGtttATTtATCCATA - ( 4 ) - cAGCAaATTtAaCCtcg aAGCAGAAaAgTgCATA - ( 4 ) - cAGCtatTggcTCCtcA D ActTtTGActgAcATCACAaaa 3146 4.17 yAGCakaWtwrTCCATA The AraC half-site consensus The XylR regulon E. coli CG D xylAB/xylFGHR xylFGHR/xylAB xylR xylE GTGAAtTATCtcAATaG - ( 4 ) - GTGAAATAACaTAATTG 3112 accAAAaATCGTAATcG - ( 4 ) - aTaAAAatctGTAATTG 3131 4.13 3.73 3.51 3.59 4.17 3.41 4.05 2.97 4.49 3.06 4.16 3.28 3.79 4.07 3.85 4.75 3.12 3.05 3.85 10.21 9.51 tttTGcGAgCgAGcgCACAcTT AAgTaaGATCTcGgTCAtAaaT gttctTGATtTtGATaAaAaTT tttcGTGcTCTgagTCACggca AttTtgGATaattATCACAaTT D 7 tttTTACGTTATTTgtt - ( 4 ) - CAcTTACGATAaTTCtC 3219 aaGAcATtACGTAAacG - ( 4 ) - GTaAAAaATgaTAATTG 3107 8.52 8.94 D tAtcacaATtaAGATCACAgaa tAAcGctAcCacGATCACATaa AtATaTcgTtggcgTCACAaaa tttTGaGAgCcAGAgCACATTT AAAaGcGATCgcGATCgaATcc D 7.04 8.21 9.55 8.36 yicJI yagGH aacAAgaATCaTAAaTt - ( 4 ) - accAgATATCGgAATat GcaAAATAACGTAATTc - ( 4 ) - aTaAgATATgaccATTG GgGgAtTcTCtTAATaG - ( 4 ) - GTGAAATAACGTAATTG acaAAAaAcCGTAATat - ( 4 ) - aTaAgAaATgacAATTG D CG D xylAB/xylR xylR/xylAB S. typhi D tgcTGTGATgaAttTCgCATaa tggcGctAcCctGATCACAgaa AAcTGTGAcCacctTagCAaaT AAAcGTGcgCcAGcTCgCAaaa AAATacGATCgccgTCAtAaTT D 8.00 7.63 9.04 7.84 9.06 9.34 7.44 7.42 yicJI xylKL aacAAgaATCGTAATTt - ( 4 ) - accAAATAACGgAATat actAAtaAACGcAATcc - ( 4 ) - GTaAgATtTaaTAATTG CAATTACGTTATTTCAC - ( 4 ) - CcATTgaGAgAaTcCAC acGAAATtTCaTAAcgG - ( 4 ) - aTaAgAaccgGTAATTG GaGcAtTtTCtcAATcG - ( 4 ) - GTGAAATAACGTAATTG D 7 K. pneumoniae UG xylFGHR xylFGHR xylA2/xynTB xynTB/xylA2 yicJI xylMN D D AttTGcGAgaTgGcTCACATTT cttTGcttcgagGATCACAgaa AttccctAcCgcGATCACAgaa AtgTGTGAcaacttTCtCAgTT D GTaAAAaAACaTAATTa - ( 4 ) - aTttAAaAACGTAATaG aacAAgaAACGTAATct - ( 4 ) - acGAgATATCGcAATTc aacAgAaAACaTAAacc - ( 4 ) - GcGAAAaATgacAATaG D D Table 2 ( Continued ) Genome Operon Site Pos .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11750821" target="_blank">11750821</a>]<br></li>
<li>3220 3160 Score 4.55 4.09 CG Y. pestis xylAB/xylFGHR xylFGHR/xylAB aatAAtTcTCtgAATTt - ( 4 ) - GTGAAATAACGTAATTG 3198 accAAAaAACaTAATTG - ( 4 ) - aTGAAAatctGTAATTG 3128 D ttATGaGATCTAcAcCACAaTT D AAAcaTGATCgttATCAtAaaa CG D H. in # uenzae HI1112-3 / HI1111-09 HI1111-09 / HI1112-3 HI1106 xylAB/xylFGH GcaAAtaATCaacATaG - ( 4 ) - aTtAAATAACaTAATTG 398 GTGAAAaAACGTAATaa - ( 4 ) - GaaAgATtTtaTAATTG 3113 9.20 9.29 AAcTGTGATCcAcgcCACAgTT 3120 4.18 D tAtTaTaActTAaATagCAaTT tttTGTGAcCcAGtcCACAaTa 342 3118 3.45 4.59 D A. actinomycetem-UG comitans aacAAtaATCtTAAaTc - ( 4 ) - GcGAAATAACGTAATTG 396 8.96 D xylFGH/xylAB xylR GaGAAAaAACGTAATaG - ( 4 ) - GgaAgATcTgaTAATTG 3139 9.09 393 4.12 tttTtTGAgtgAaATCACAgag RtgAAAWAwCrTAATTG The XylR half-site consensus The divergently arranged operons are separated by a slash .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11750821" target="_blank">11750821</a>]<br></li>
<li>A consensus sequence , -- gaAa-a -- a-AAT , presumably for the binding of monomeric XylR emerges from the comparison of the four half sites ( IA1 , IA2 , IF1 , and IF2 ) that are rich in adenines .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9371449" target="_blank">9371449</a>]<br></li>
<li>These genes , together with xylAB , are positively regulated by the same protein ( XylR [ Fig. 4 ] ) , which appears to recognize the consensus sequences located in their promoter regions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9371449" target="_blank">9371449</a>]<br></li>
<li>For the xylAB and xylFGH promoters , these sequences were probed by direct binding in vitro with XylR , whereas the same consensus sequence in the xylE promoter was found merely by prediction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9371449" target="_blank">9371449</a>]<br></li>
<li>The improved XylR site consensus was proposed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11750821" target="_blank">11750821</a>]<br></li>
<li>In both cases , the CRP site is upstream of the XylR site , either immediately adjacent or separated by 10 bp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11750821" target="_blank">11750821</a>]<br></li>
<li>The upstream region of the xylKL operon of S. typhi contains the CRP and XylR sites with an unusually long spacer of 21 bp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11750821" target="_blank">11750821</a>]<br></li>
<li>It is likely that XylR binds as a dimeric form , each molecule occupying two helical turns of each half site as in AraC ( 2 , 23 ) or as in other helix-turn-helix proteins ( 1 ) such as cI ( 16 ) and Cro ( 49 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9371449" target="_blank">9371449</a>]<br></li>
<li>Interestingly , the conserved bases in each half site , both in IA and in IF , are repeated with a periodicity of 20 or 21 bp ( about two helical turns ) , suggesting that the contacts of each XylR mono-mer occur on the same side of the DNA surface .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9371449" target="_blank">9371449</a>]<br></li>
<li>In vivo footprinting analysis revealed that XylR binds to at least two DNA regions , IA and IF , each with a direct repeat .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9371449" target="_blank">9371449</a>]<br></li>
<li>In this regard , XylR is similar to other members of the AraC family regulators that bind to direct repeat structures with a gap of 4 bp , spanning about four helical turns as a whole ( 23 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9371449" target="_blank">9371449</a>]<br></li>
				</ul></div><br>
			<label class="collapse_TU" for="_6"><font size="5px">Transcription units regulated by XylR</font></label>
				<input id="_6" type="checkbox">
				<div class="TU"><br><ul><li>Before the DMS treatment , amounts of XylR expressed were conﬁrmed by Western analysis with anti-XylR antiserum to be similar in induced and uninduced conditions ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9371449" target="_blank">9371449</a>]<br></li>
<li>3220 3160 Score 4.55 4.09 CG Y. pestis xylAB/xylFGHR xylFGHR/xylAB aatAAtTcTCtgAATTt - ( 4 ) - GTGAAATAACGTAATTG 3198 accAAAaAACaTAATTG - ( 4 ) - aTGAAAatctGTAATTG 3128 D ttATGaGATCTAcAcCACAaTT D AAAcaTGATCgttATCAtAaaa CG D H. in # uenzae HI1112-3 / HI1111-09 HI1111-09 / HI1112-3 HI1106 xylAB/xylFGH GcaAAtaATCaacATaG - ( 4 ) - aTtAAATAACaTAATTG 398 GTGAAAaAACGTAATaa - ( 4 ) - GaaAgATtTtaTAATTG 3113 9.20 9.29 AAcTGTGATCcAcgcCACAgTT 3120 4.18 D tAtTaTaActTAaATagCAaTT tttTGTGAcCcAGtcCACAaTa 342 3118 3.45 4.59 D A. actinomycetem-UG comitans aacAAtaATCtTAAaTc - ( 4 ) - GcGAAATAACGTAATTG 396 8.96 D xylFGH/xylAB xylR GaGAAAaAACGTAATaG - ( 4 ) - GgaAgATcTgaTAATTG 3139 9.09 393 4.12 tttTtTGAgtgAaATCACAgag RtgAAAWAwCrTAATTG The XylR half-site consensus The divergently arranged operons are separated by a slash .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11750821" target="_blank">11750821</a>]<br></li>
<li>Cross-regulation by XylR and DmpR activators of Pseudomonas putida suggests that transcriptional control of biodegradative operons evolves independently of catabolic genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22904281" target="_blank">22904281</a>]<br></li>
<li>These genes , together with xylAB , are positively regulated by the same protein ( XylR [ Fig. 4 ] ) , which appears to recognize the consensus sequences located in their promoter regions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9371449" target="_blank">9371449</a>]<br></li>
<li>AraC activates the arabinose-catabolism genes araAB , araE , and araFGH , and suppresses the xylose-catabolism genes xylAB and xylFGH by inhibiting XylR , the xylose transcriptional activator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26045332" target="_blank">26045332</a>]<br></li>
<li>It was experimentally demonstrated that XylR acts as a transcriptional activator for the promoters of the xylAB and xylFGH operons , but not for the xylR promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11750821" target="_blank">11750821</a>]<br></li>
<li>Xylose-bound XylR is believed to activate the transcription of the xylFGH , xylR , xylAB , and xylE operons .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20023096" target="_blank">20023096</a>]<br></li>
				</div></ul></div><br>
	</div>
</body>
</html>