<!doctype html>
<style>
@charset "utf-8";
body {
	height: 100%;
	margin: 0;
	padding: 0;
}

.banner {
	position: fixed;
	top: 0;
	width: 100%;
	height: 65px;
	background: gray;
	line-height: 65px;
}
.spacer {
	height: 10px;
}

.main-content {
	padding: 5%;
}

.collapse_ACT {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-style: none solid none none;
	border-color: #B4045F;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_ACT + input{
	display: none;
}
.collapse_ACT + input + div{
	display: none;
}
.collapse_ACT + input:checked + div{
	display: block;
}
.ACT {
	text-align: justify;
	text-justify: inter-word;
}
.ACT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B4045F;
}
.ACT_collapse_similarity:hover{
	background: #e5d3dc;
}
.ACT_collapse_similarity + input {
	display: none;
}
.ACT_collapse_similarity + input + div{
	display: none;
}

.ACT_collapse_similarity + input:checked + div{
	display: block;
}
.ACT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.ACT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B4045F;
	padding-left: 5px;
}
.ACT_collapse_similarity:hover{
	background: #e5d3dc;
}
.ACT_collapse_similarity + input {
	display: none;
}
.ACT_collapse_similarity + input + div{
	display: none;
}
.ACT_collapse_similarity + input:checked + div{
	display: block;
}
.ACT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.ACT_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #B4045F;
}
.ACT_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_DOM {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-style: none solid none none;
	border-color: #B404AE;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_DOM + input{
	display: none;
}
.collapse_DOM + input + div{
	display: none;
}
.collapse_DOM + input:checked + div{
	display: block;
}
.DOM {
	text-align: justify;
	text-justify: inter-word;
}
.DOM_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B404AE;
	padding-left: 5px;
}
.DOM_collapse_similarity:hover{
	background: #e8d7e7;
}
.DOM_collapse_similarity + input {
	display: none;
}
.DOM_collapse_similarity + input + div{
	display: none;
}
.DOM_collapse_similarity + input:checked + div{
	display: block;
}
.DOM_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}.DOM_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #B404AE;
}
.DOM_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_EVO {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #7401DF;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_EVO + input{
	display: none;
}
.collapse_EVO + input + div{
	display: none;
}
.collapse_EVO + input:checked + div{
	display: block;
}
.EVO {
	text-align: justify;
	text-justify: inter-word;
}
.EVO_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #7401DF;
	padding-left: 5px;
}
.EVO_collapse_similarity:hover{
	background: #e2d7ed;
}
.EVO_collapse_similarity + input {
	display: none;
}
.EVO_collapse_similarity + input + div{
	display: none;
}
.EVO_collapse_similarity + input:checked + div{
	display: block;
}
.EVO_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.EVO_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #7401DF;
}
.EVO_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_RP {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #0174DF;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_RP + input{
	display: none;
}
.collapse_RP + input + div{
	display: none;
}
.collapse_RP + input:checked + div{
	display: block;
}
.RP {
	text-align: justify;
	text-justify: inter-word;
}
.RP_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #0174DF;
	padding-left: 5px;
}
.RP_collapse_similarity:hover{
	background: #c9d7e2;
}
.RP_collapse_similarity + input {
	display: none;
}
.RP_collapse_similarity + input + div{
	display: none;
}
.RP_collapse_similarity + input:checked + div{
	display: block;
}
.RP_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.RP_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #0174DF;
}
.RP_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_SIT {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #0489B1;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_SIT + input{
	display: none;
}
.collapse_SIT + input + div{
	display: none;
}
.collapse_SIT + input:checked + div{
	display: block;
}
.SIT {
	text-align: justify;
	text-justify: inter-word;
}
.SIT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #0489B1;
	padding-left: 5px;
}
.SIT_collapse_similarity:hover{
	background: #d0dfe2;
}
.SIT_collapse_similarity + input {
	display: none;
}
.SIT_collapse_similarity + input + div{
	display: none;
}
.SIT_collapse_similarity + input:checked + div{
	display: block;
}
.SIT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.SIT_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #0489B1;
}
.SIT_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_TU {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #04B486;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_TU + input{
	display: none;
}
.collapse_TU + input + div{
	display: none;
}
.collapse_TU + input:checked + div{
	display: block;
}
.TU {
	text-align: justify;
	text-justify: inter-word;
}
.TU_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #04B486;
	padding-left: 5px;
}
.TU_collapse_similarity:hover{
	background: #d0e2de;
}
.TU_collapse_similarity + input {
	display: none;
}
.TU_collapse_similarity + input + div{
	display: none;
}
.TU_collapse_similarity + input:checked + div{
	display: block;
}
.TU_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.TU_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #04B486;
}
.TU_similar > a {
	text-decoration: none;
	color: gray;
}

ul { 
	list-style-type: none;
	padding: 0;
	margin: 0;
}
ul li > a {
	text-decoration: none;
	color: gray;
}
ul li:hover, ul li:focus {
	background: #F0F0F0;
}

ul li:first-child span {
  border-left:none;  
}

</style>
<html>
<head>
<meta charset="utf-8">
<title>TF properties - YdeO</title>
<meta name="theme-color" content="#ffffff">
</head>
<body>
	<div class="banner">
		<center><font color="white" size="16px" face="arial">YdeO</font></center>
	</div>
	<div align="center" class="main-content">
		<div class="spacer">
	</div><br>			<label class="collapse_ACT" for="_1"><font size="5px">YdeO transcriptional activity</font></label>
				<input id="_1" type="checkbox">
				<div class="ACT"><br><ul><li>GadE cascade induced a total of 19 operons including aidB , blc , cbpAM , elaB , gabDTP , pagP , sufABCDSE , ycaC , yfcG , ygaM , and yjjUV ( see Table 1 ) , of which the yfcG gene encodes a disulfide reductase [ 37 ] and the sufABCDSE operon encodes the complex biosynthetic machinery for iron-sulfur clusters in several enzymes which have critical cysteine residues [ 38 ] , suggesting a relationship between the function of YdeO and cysteine metab-olism .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25375160" target="_blank">25375160</a>]<br></li>
<li>All of the seven operons identified by ChIP-chip as being potentially regulated by YdeO were confirmed as being under the direct control of YdeO using RT-qPCR , EMSA , DNaseI-footprinting and reporter assays .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25375160" target="_blank">25375160</a>]<br></li>
<li>Interestingly the high level of ydeO-lacZ was detected in both pH 5.5 and 7.0 when E. coli were grown under anaerobic conditions ( Fig. 3C ) , implying that YdeO plays a role in E. coli respiration under anaerobic conditions , such as in the animal intestine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25375160" target="_blank">25375160</a>]<br></li>
<li>These findings also suggest that YdeO activated genes play an important role in primary adaptation , which enables the cell to colonize animal intestines by contributing to adaptation to acidic conditions in the stomach and to anaerobic conditions in the intestine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25375160" target="_blank">25375160</a>]<br></li>
<li>Characterization of the YdeO Regulon in Escherichia coli Yuki Yamanaka , Taku Oshima , Akira Ishihama , Kaneyoshi Yamamoto * 1 3 1,2 1,2 1 Department of Frontier Bioscience , Hosei University , Koganei , Tokyo , Japan , 2 Micro-Nano Technology Research Center , Hosei University , Koganei , Tokyo , Japan , 3 Graduate School of Information Sciences , Nara Institute of Science and Technology , Ikoma , Nara , Japan Abstract Enterobacteria are able to survive under stressful conditions within animals , such as acidic conditions in the stomach , bile salts during transfer to the intestine and anaerobic conditions within the intestine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25375160" target="_blank">25375160</a>]<br></li>
<li>Yamanaka , Y. , Oshima , T. , Ishihama , A. & Yamamoto , K. ( 2014 ) The role of the Escherichia coli YdeO regulator in response to acidic and anaerobic stresses .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26010043" target="_blank">26010043</a>]<br></li>
<li>Together , these data show that despite co-reliance on EvgAS , there is a degree of mechanistic independence between the regulatory elements governing rapid and delayed responses to low pH at the gadE promoter , which corroborates and extends recent findings .28 In summary , the above data show that through EvgAS , a strong response to a pH 5.7 shift is initially transmitted through the circuit via YdeO , which we show here negatively auto-regulates its own production to dampen this response .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20603130" target="_blank">20603130</a>]<br></li>
<li>It is also unclear whether YdeO induces slp-yhiF , hdeAB-yhiD , hdeD , gadA and gadBC directly or via YhiE .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12694615" target="_blank">12694615</a>]<br></li>
<li>These results suggest that YdeO binds to the downstream region of hdeD , upregulates yhiUV and , consequently , induces multidrug resistance in a DacrB strain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12694615" target="_blank">12694615</a>]<br></li>
<li>YdeO induces Slp-YhiF , HdeA , HdeD , YhiE and GadABC , which induce acid resistance by working co-operatively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12694615" target="_blank">12694615</a>]<br></li>
<li>The contributions of YhiD and HdeB to acid resistance are unclear.YdeO also induces YhiUV , which induce multidrug resistance .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12694615" target="_blank">12694615</a>]<br></li>
<li>EvgS EvgS P PhoQ EvgS SafA PhoQ P inner ATP EvgA membrane ATP ADP P EvgA ADP yegZ yegR ydeO safA ydeP yfdX yfdE yfdV oxc frc ( yfdU ) ( yfdW ) emrY emrK evgA evgS YdeO ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19352034" target="_blank">19352034</a>]<br></li>
<li>Although the yiiS and uspD genes , encoding conserved proteins with unidentified function , were expressed even without the over-expression of YdeO , their expressions were further increased after YdeO expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25375160" target="_blank">25375160</a>]<br></li>
<li>It is inconsistent with the RT-qPCR result that the mRNA level of gadW increased in the presence of YdeO expression as detected by RT-qPCR ( Table 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25375160" target="_blank">25375160</a>]<br></li>
<li>The data nonetheless demonstrate how both YdeO and PhoP each contribute globally to both activation and fine-tuning repression within this transcriptional network .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20603130" target="_blank">20603130</a>]<br></li>
<li>The data suggest that RpoS does not absolutely depend on YdeO or PhoP to activate transcription at those promoters that are sensitive to its presence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20603130" target="_blank">20603130</a>]<br></li>
<li>It was reported elsewhere that the YdeO section of the cascade can be bypassed .24,26 Our data show that this is true ( see later time points in Fig. 3a , vii -- viii ) but that EvgAmediated induction of gadE and gadB in the absence of YdeO is severely delayed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20603130" target="_blank">20603130</a>]<br></li>
<li>Furthermore , data to be presented below will suggest that EvgA alone may in fact have little effect on gadE promoter activity , as additional regulatory interactions linked to EvgAS are required for residual induction of the gadE promoter in the absence of YdeO .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20603130" target="_blank">20603130</a>]<br></li>
<li>Indeed , the gadB and gadA promoters have hallmarks of RpoS-regulated promoters .19,51 RpoS functioned as an activator in both the presence and absence of either PhoP or YdeO , in some cases , having a quantifiable effect only in the absence of YdeO-mediated activation ( e.g. , at slpp and hdeAp ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20603130" target="_blank">20603130</a>]<br></li>
<li>In the absence of YdeO , however , the presence of PhoP contributes , along with RpoS and GadE , to delayed and gradual induction in gadE and gadB promoter activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20603130" target="_blank">20603130</a>]<br></li>
<li>As had already been observed at the gadE and gadB promoters , promoters usually subject to strong YdeO-dependent induction ( groups 3 and 4 ) displayed a delayed and gradual increase in activity in the absence of YdeO .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20603130" target="_blank">20603130</a>]<br></li>
<li>Surprisingly , we found that PhoP was also required for activation of gadE and gadB promoters in the absence of YdeO ( Fig. 6d ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20603130" target="_blank">20603130</a>]<br></li>
<li>Thus , in the absence of YdeO , at least two additional transcription factors , namely , RpoS and PhoP , must be present for the delayed EvgAS-dependent activation of the gadE and gadB promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20603130" target="_blank">20603130</a>]<br></li>
<li>Thus , the two situations are not directly comparable , and it is far from clear that PhoP is involved in activation of the gadE and gadB promoters in the presence of YdeO , as is the case for RpoS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20603130" target="_blank">20603130</a>]<br></li>
<li>This order of induction is consistent with YdeO first inducing the gadE promoter and then GadE in turn activating the gadB promoter .18,27 A transcriptional cascade from EvgA to YdeO to GadE to GadB must be completed to enable a rapid increase in glutamate-dependent AR To start to assign the regulators responsible for the kinetics seen in Fig. 2b , we re-assayed this set of promoters , in addition to a similarly constructed evgAS -- Lux promoter fusion ( Table 2 ) , in strain derivatives in which the key regulators EvgS , EvgA , YdeO , GadE , and RpoS had been removed by mutation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20603130" target="_blank">20603130</a>]<br></li>
<li>We noted that a putative PhoP binding site has also been identified in the gadE promoter , although this was proposed as an activating site , not repression as here .30 In order to shed light on whether PhoP could regulate the gadE and gadB promoters independently of YdeO , we measured the activities of the gadE and gadB promoters in the ydeO phoP double-mutant background and compared the activities obtained t those in a ydeO single mutant , where gadE and gadB promoter activity was induced after a delay in an EvgA - and RpoS-dependent fashion ( Fig. 3a , vii -- viii ; Fig. 3c ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20603130" target="_blank">20603130</a>]<br></li>
<li>However , we found that in the absence of YdeO , PhoP-dependent mgtA promoter activity increased , suggesting an increase in the levels of active PhoP compared to levels in the wild type ( Fig. 5b , iv ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20603130" target="_blank">20603130</a>]<br></li>
<li>In this particular case , the divergently transcribed gadY promoter is dependent on YdeO for low pH activation , and gadY promoter activity is increased substantially in a phoP mutant .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20603130" target="_blank">20603130</a>]<br></li>
<li>Following pH 5.7 induction , a sharp increase in AR2 was observed in the wild-type strain and this depended on the regulation conferred by EvgA for up to 2 h after induction ( Fig. 3b ) , in agreement with earlier findings that also showed a key role for evgA in mounting AR2 in low pH shifted log-phase but not stationary-phase cells .24 The initial increase in AR2 was also entirely dependent on YdeO , but at later time points post-induction , an increase in AR2 occurred in the absence of ydeO ( Fig. 3b ) , consistent with our findings concerning the kinetics of gadB promoter regulation ( Figs. 2 and 3a ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20603130" target="_blank">20603130</a>]<br></li>
<li>In agreement with this hypothesis , it is shown in Fig. 3a , vi , that removal of YdeO by mutation leads to a greater than 2-fold increase in transcription of the b1500-ydeO promoter following induction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20603130" target="_blank">20603130</a>]<br></li>
<li>YdeO and PhoP negatively feedback to the b1500-ydeO and ydeP promoters causing a transient surge in transcriptional activation To investigate the relative contribution of the YdeO and PhoP regulators in controlling the induction surge at the b1500-ydeO promoter , we constructed a ydeO phoP double mutant and analysed b1500-ydeO promoter activity in this background .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20603130" target="_blank">20603130</a>]<br></li>
<li>Both PhoP and YdeO contribute to negative feedback at the b1500 promoter , generating surge-like kinetics .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20603130" target="_blank">20603130</a>]<br></li>
<li>This novel YdeO-mediated repression of b1500-ydeO promoter activity occurs within 10 min of induction of YdeO synthesis and contributes substantially in setting the steady-state level of activity observed at this promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20603130" target="_blank">20603130</a>]<br></li>
<li>Although this could be due to direct repression by YdeO at the mgtA promoter , it could also be explained by our novel finding that YdeO negatively regulates b1500 and ydeO expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20603130" target="_blank">20603130</a>]<br></li>
<li>Taken together , these physiological systems activated by YdeO stimulate stress response and respiration .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25375160" target="_blank">25375160</a>]<br></li>
<li>Glucose induction of P3 and P3P2 is independent of YdeO , GadXW , EvgAS and Crp We then asked whether any of the E. coli-speciﬁc regulators YdeO , GadX , GadW or EvgAS were needed for glucose-dependent induction of P3P2 -- lacZ .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19220752" target="_blank">19220752</a>]<br></li>
<li>B. Effects of YdeO , GadXW and EvgAS on glucose-dependent induction of P3P2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19220752" target="_blank">19220752</a>]<br></li>
<li>First , YdeO can activate promoters P3 and P2 during exponential-growth in minimal medium when pH falls below pH 7 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19220752" target="_blank">19220752</a>]<br></li>
<li>Promoters P3 and P2 can be activated by GadX and GadW during growth to stationary phase , or by YdeO during log-phase growth in minimal medium below pH 7 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19220752" target="_blank">19220752</a>]<br></li>
<li>Data revealed that YdeO and GadX activate P3 , P2 and P3P2 , while GadE autoactivates P1 and represses P3 and P3P2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19220752" target="_blank">19220752</a>]<br></li>
<li>In contrast to the effect of YdeO and GadX on the P2 and P3 promoters , overexpression of GadE , as predicted , did not activate P2 , P3 or the combined P3 P2 fusions in Salmonella during exponential-growth ( Fig. 6A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19220752" target="_blank">19220752</a>]<br></li>
<li>The gadA/BC genes , activated in response to acid stress and in stationary phase cells , are subjected to complex circuits of regulation involving 70 , S , cAMP receptor protein , H-NS , EvgAS , TorRS , GadE , GadX , GadW , and YdeO .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15795232" target="_blank">15795232</a>]<br></li>
<li>Transcription of hdeAB depends on complex circuits of regulation involving RpoD ( 70 ) , RpoS ( 38 ) , H-NS , cyclic AMP receptor protein , GadE , GadX , GadW , GadY , EvgAS , YdeO , Lrp , MarA , SoxRS , TorRS , and TrmE , among other regulators ( see Fig. 1 and references in the legend ; see Fig. 2 for the DNA binding sites of some of these regulators in the hdeAB promoter ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18083817" target="_blank">18083817</a>]<br></li>
<li>Amongst genes downstream of a YdeO-binding site , 19 genes , ( nhaA , nhaR , hyaA , hyaB , hyaC , hyaD , hyaE , hyaF , appC , appB , appA , yiiS , yiiT / uspD , slp , dctR , gadE , mdtE , mdtF , and gadW ) were induced more than 3-fold by the over-expression of YdeO ; while three genes , yccA , yiiR , and yhiS , were not affected in both duplicate experiments .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25375160" target="_blank">25375160</a>]<br></li>
<li>These 19 genes induced by YdeO constitute a total of 7 transcriptional units , nhaAR , hyaABCDEF , appCBA , yiiS-yiiT/uspD , slp-dctR , gadE-mdtEF , and gadW .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25375160" target="_blank">25375160</a>]<br></li>
<li>The acid resistance induced by YdeO overexpression was abolished by deletion of yhiE , gadC , slp-yhiF , hdeA or hdeD , genes induced by YdeO overexpression , suggesting that several genes orchestrate YdeO-induced acid resis ¬ Accepted 20 January , 2003 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12694615" target="_blank">12694615</a>]<br></li>
<li>Several genes induced by overexpression of YdeO and GadX overlapped ; however , yhiE was induced only by YdeO .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12694615" target="_blank">12694615</a>]<br></li>
<li>Many of the YdeO-activated genes were also activated by GadX , whereas yhiEUV were induced only by YdeO .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12694615" target="_blank">12694615</a>]<br></li>
<li>12.38 -8.24 -59.96 1.13 ± 0.55 ± 0.50 ± 0.02 0.04 0.02 1.27 ± 0.50 ± 0.20 ± 0.03 0.03 0.03 / h ) / h ) Fig. 2 Comparison of the transcript levels of the E. coli wild type under different pH values at 7.0 and 6.0 : a global regulatory genes , b PhoB regulatory genes , c acid-inducible genes , d metabolic pathway genes , e respiratory chain genes during micro-aerobic continuous culture at the dilution rate of 0.1 h -1 Fig. 3 a , b Overall metabolic regulation of the E. coli during acidic condition ( a ) Acid Condition Hns ATP ADP PhoQ PhoQ P ATP ADP ¬ EvgS EvgS P cAMP - evgA Pho P PhoP Crp EvgA P B1500 EvgA cAMP-Crp ¬ RpoS b1500 ydeo ¬ YdeO gadE GadE O O GadW GadX hdeA gadB gadc gadW gadA gadX HdeA GadA GadB GadC Acid resistance ( b ) Acidic Condition ( H + ) ( indirect ) EnvZ PhoR fnr OxyR ldhA arcA P Fnr ArcA SoxR / S L PhoR - P OmpR OmpR - P DH pfl yfiD ompC TCA cycle genes : gltA sdhC mdh sodA PhoB - P PhoB Lactate OmpC ompF asr YfiD Pfl OmpF Asr Formate Acetate ackA pta Acid resistance Moreover , in accord with the proteomic analysis , the expression of yﬁD : : lac reporter fusion and the intracellular content of YﬁD were found to be high during growth at low pH for both aerobic and anaerobic cultures ( Neil et al. 2002 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23274360" target="_blank">23274360</a>]<br></li>
<li>DELETION OF EvgA , YdeO , H-NS DOES NOT HAVE OBVIOUS EFFECT ON THE TRANSCRIPTION FROM INDIVIDUAL PROMOTERS UNDER ANAEROBIC CONDITION Our investigation above showed that ArcA and MnmE are the major activators responsible for the expression of gadE-mdtEF under anaerobic condition , whereas other regulators tested did not show obvious regulatory effects .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23874328" target="_blank">23874328</a>]<br></li>
<li>Knowledge about the regulatory functions of YdeO is limited except that it is known that YdeO activates transcription of the gad system components , gadE , gadA and gadBC [ 7,8,9 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25375160" target="_blank">25375160</a>]<br></li>
<li>Induction of NhaR , GadE , and GadW by YdeO Four transcription factors , the LysR-type NhaR , the LuxR-type GadE , and the AraC-type GadW , were found to be under the direct control of YdeO ( see Figs. 1 -- 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25375160" target="_blank">25375160</a>]<br></li>
<li>Thus , as GadE is produced , it will begin to shut down the YdeO activation pathway but stimulate its own synthesis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15489450" target="_blank">15489450</a>]<br></li>
<li>Among those indirectly YdeO activated via GadE ( group 4 ) , the hdeD and hdeA promoters appeared not to be sensitive to this feedback .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20603130" target="_blank">20603130</a>]<br></li>
<li>GadE induced by YdeO stimulated the transcription of hdeAB-yhiD , hdeD , gadAX , gadCB , mdtEF , gadW , and yhiM as well as those previously reported promoters [ 9,34,35 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25375160" target="_blank">25375160</a>]<br></li>
<li>Consistent with this idea , overexpressing YdeO also increased GAD production and relied on GadE to do so ( Fig. 1B , compare lanes 2 and 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15489450" target="_blank">15489450</a>]<br></li>
<li>The effects of EvgA and YdeO on GAD production also appear additive ( Fig. 3A , EK596 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15489450" target="_blank">15489450</a>]<br></li>
<li>However , we failed to detect NhaR and GadW even in the presence of YdeO expression ( Fig. 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25375160" target="_blank">25375160</a>]<br></li>
<li>The expression of nhaR-lux , slp-lux , and gadE-lux was found to be activated in the presence of the YdeO-expressing plasmid ( Fig. 3A ) , indicating that YdeO is also a positive regulator for these promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25375160" target="_blank">25375160</a>]<br></li>
<li>Accordingly the constructed nhaRlux reporter plasmid containing this novel nhaR promoter was also activated in the presence of YdeO expression ( Fig. 3A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25375160" target="_blank">25375160</a>]<br></li>
<li>It is now known that E. coli possesses a number of effectors that enable it to survive otherwise lethal doses of inorganic and organic acid stress ( termed acid resistance or AR ) , many of which are not present in closely related bacteria such as Salmonella spp .8 The function of each AR effector depends on the precise nature of the acid stress .9 -- 13 For example , two isomeric glutamate decarboxylases encoded by gadA and gadB and a glutamate-γ-aminobutyric acid antiporter gadC contribute to AR only when glutamate is available in the extracellular environment ,10 a phenomenon termed glutamate-dependent AR or AR2 , whereas the periplasmic chaperone encoded by hdeA only appears to contribute to AR in dense cell populations .11,14 Several effectors reside within a 15-kb genomic patch termed the `` acid fitness island '' ( AFI ) and , along with the non-proximal gadBC locus , are co-regulated under diverse situations .15 -- 18 In E. coli K-12 , these effectors are subject to tight transcriptional control and are expressed at low levels during rapid growth at neutral pH , due in part to global H-NS-mediated repression15 ,19 and Lon-mediated degradation of the central activator protein GadE .20 For induction of this latter set of genes , distinct regulatory networks operate depending on the inducing conditions , which include a shift to sublethal pH21 and growth into stationary phase .19,22 A drop from pH 7 to sub-lethal pH 5.5 induces a number of AR effectors via the two-component system EvgA/EvgS , which is encoded outside of the AFI .23,24 Two intermediate transcription factors , YdeO and the AFI-encoded GadE , are transcriptionally activated by the presumably phosphorylated response-regulator EvgA .18,24 -- 26 YdeO can also activate gadE transcription forming a coherent feed-forward loop18 ( Fig. 1a ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20603130" target="_blank">20603130</a>]<br></li>
<li>Although we reported low-level induction of gadX by EvgA overexpression ( Masuda and Church , 2002 ) , we could not detect gadX induction by either EvgA overexpression in the absence of ydeO or YdeO overexpression ( http://arep.med.harvard.edu/cgi-bin/ ExpressDBecoli/EXDStart ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12694615" target="_blank">12694615</a>]<br></li>
<li>Although gadX is predicted to be co-transcribed with gadA ( Tramonti et al. , 2002 ) , EvgA overexpression in the absence of ydeO or YdeO overexpression upregulated gadA but not gadX .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12694615" target="_blank">12694615</a>]<br></li>
<li>Thus , in the present study , we ﬁrst examined whether previously identiﬁed regulators , such as EvgA , YdeO , MnmE , GadE , GadX , and H-NS , also participate in the anaerobic activation of mdtEF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23874328" target="_blank">23874328</a>]<br></li>
<li>Interestingly , except for mnmE , deletion of the regulators previously shown to participate in the regulation of mdtEF under aerobic condition , such as EvgA , YdeO , GadE , and GadX , caused no or only a moderate decrease ( 17 -- 40 % ) of the transcription of PgadE-lacZ , and deletion of H-NS caused a moderate increase of PgadE-lacZ activity , suggesting that these regulators are not responsible for the activation of mdtEF under anaerobic condition .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23874328" target="_blank">23874328</a>]<br></li>
<li>Involvement of EvgA , YdeO , H-NS , CRP , GadX , as well as a small RNA DsrA in the regulation of the mdtEF gene in the same operon has also been reported ( Nishino and Yamaguchi , 2002 , 2004 ; Oshima et al. , 2006 ; Nishino et al. , 2008a , b , 2009 , 2011 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23874328" target="_blank">23874328</a>]<br></li>
<li>More than ten transcription regulators including EvgA , YdeO , GadE , TorR , H-NS , PhoP , RpoS , CRP , MnmE , GadX , and GadW have been found to participate in the regulation of gadE during various circumstances of acid stresses through at least ﬁve different regulatory circuits ( Ma et al. , 2004 ; Sayed and Foster , 2009 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23874328" target="_blank">23874328</a>]<br></li>
<li>INVOLVEMENT OF DIFFERENT REGULATORS IN THE ANAEROBIC EXPRESSION OF mdtEF FROM THAT OF AEROBIC CONDITION Previous studies focusing on the expression of gadE-mdtEF operon during acid resistance have revealed that several regulators , such as EvgA , YdeO , MnmE , GadE , GadX , CRP , and H-NS , are involved in the regulation of gadE-mdtEF operon ( Gong et al. , 2004 ; Ma et al. , 2004 ; Hirakawa et al. , 2006 ; Tramonti et al. , 2008 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23874328" target="_blank">23874328</a>]<br></li>
<li>Thus , we propose a complex regulatory cascade in which global regulators ( RpoS , CRP , HN-S , EvgA , etc. ) inﬂuence the expression levels and/or activities of GadX and GadW , which in turn activate the expression and/or activities of transcription activators ( e.g. , YdeO and YhiE ) that directly activate subsets of target genes involved in AR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12730179" target="_blank">12730179</a>]<br></li>
<li>A complex acid resistance regulon including the gad system is regulated by transcription factors GadX-GadW and EvgA-YdeO , as well as by RpoS , H-NS , and cyclic AMP ( 11 , 12 , 44 , 79 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15601715" target="_blank">15601715</a>]<br></li>
<li>Their regulation is known to be mediated by transcription factors GadXGadW and EvgA-YdeO , as well as by RpoS , H-NS , and cyclic AMP [ 60,61,74 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17026754" target="_blank">17026754</a>]<br></li>
<li>Reporter assays for transcriptional regulation by YdeO .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25375160" target="_blank">25375160</a>]<br></li>
<li>Expression of the ﬁrst major regulator immediately downstream of EvgA , YdeO , is itself completely dependent on RcsB , making a direct effect of RcsB on EvgS or EvgA more likely .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21571995" target="_blank">21571995</a>]<br></li>
<li>Finally , the data set confirmed that all the effects of EvgA and YdeO ( and B1500 ) were specific to pH 5.7 inducing conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20603130" target="_blank">20603130</a>]<br></li>
<li>Promoters are named according to the first gene in the operon they control and have been grouped according to regulatory factors required for activation during the first hour after shift to pH 5.7 , as deduced from ( c ) below : group 1 : EvgA ; group 2 : EvgA , B1500 , and PhoP ; group 3 : EvgA and YdeO ; group 4 : EvgA , YdeO , and GadE ; and group 5 : no activation observed during this period .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20603130" target="_blank">20603130</a>]<br></li>
<li>Group 1 ( b1500p and ydePp ) is activated by EvgA only ; group 2 ( mgtAp , phoPp and gadWp ) is activated by EvgA , B1500 , and PhoP ; group 3 ( gadEp , gadYp , and slpp ) is activated by EvgA and YdeO ; and group 4 ( hdeAp , hdeDp , gadAp , and gadBp ) is activated by EvgA , YdeO , and GadE .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20603130" target="_blank">20603130</a>]<br></li>
<li>EvgA and YdeO activate GAD expression in exponentialphase cells grown in minimal medium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15489450" target="_blank">15489450</a>]<br></li>
<li>In contrast to the situation in LB , the acid-induced ( pH 5.8 ) , exponential-phase expression of P3P2 -- lacZ during growth in minimal medium primarily required EvgA and YdeO ( Fig. 7C , bars 6 , 8 and 9 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19220752" target="_blank">19220752</a>]<br></li>
<li>For instance , EvgA , YdeO and GadE itself form a branched regulatory loop that activates gadE during exponential-growth in pH 5.5 minimal medium ( Ma et al. , 2004 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17660422" target="_blank">17660422</a>]<br></li>
<li>GadX binds to the gadE control region Regulators EvgA , YdeO and GadE bind to subfragments of the 798 bp gadE upstream control region ( Ma et al. , 2004 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17660422" target="_blank">17660422</a>]<br></li>
<li>Recently gadE ( yhiE ) has been identiﬁed as inducible by low pH , its inactivation being detrimental to acid resistance in minimal medium ( Tucker et al. , 2002 ) , or to acid resistance caused by EvgA overexpression ( Masuda & Church , 2002 , 2003 ) , and gadE has been proposed to be regulated by YdeO ( Masuda & Church , 2003 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14702398" target="_blank">14702398</a>]<br></li>
<li>First , the response-regulator EvgA and the AraC-family activator YdeO create a feed-forward gadE activation circuit that functions in minimal glucose medium ( Ma et al. , 2004 ; Masuda & Church , 2003 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17660422" target="_blank">17660422</a>]<br></li>
<li>EvgA is likely to directly upregulate operons in the ﬁrst class , and indirectly upregulate operons in the second class via YdeO .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12694615" target="_blank">12694615</a>]<br></li>
<li>These results suggest that EvgA directly regulates the former genes , and indirectly regulates the latter genes via YdeO .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12694615" target="_blank">12694615</a>]<br></li>
<li>EvgA overexpression induced the expression of ydeO , an AraC/XylS regulator , indicating that some of the genes induced by EvgA overexpression might be directly induced by YdeO instead of EvgA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12399493" target="_blank">12399493</a>]<br></li>
<li>YdeO , one of the genes directly induced by phosphorylated EvgA , then up-regulates gadE , the master regulator of acid resist-ance genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19352034" target="_blank">19352034</a>]<br></li>
<li>We eliminated the other four genes from further consideration , as these genes might be false positives or induced only by the higher level of YdeO relative to the YdeO levels induced by EvgA overexpression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12694615" target="_blank">12694615</a>]<br></li>
<li>This would not happen if YdeO or EvgA activated other genes necessary for AR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15489450" target="_blank">15489450</a>]<br></li>
<li>We hypothesized that increased levels of YdeO might induce some EvgA-activated genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12694615" target="_blank">12694615</a>]<br></li>
<li>Our results suggest that the gad system is positively regulated by at least two AraC/XylS-type regulators , YdeO and GadX , and also suggest that EvgA and YdeO regulate acid resistance genes more globally than GadX .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12694615" target="_blank">12694615</a>]<br></li>
<li>These results suggest that yhiE is essential for acid resistance induced by YdeO overexpression and that EvgA overexpression induces yhiE via the induction of YdeO .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12694615" target="_blank">12694615</a>]<br></li>
<li>The multidrug resistance caused by overexpression of EvgA and YdeO is due to the induction of mdtEF gene expression ( 6 , 7 , 14 , 20 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16885453" target="_blank">16885453</a>]<br></li>
<li>html ) , activation of the EvgS/EvgA system increased the expression of as many as 225 open reading frames ( ORFs ) including ones encoding transcriptional factors such as YdeO and GadE .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19352034" target="_blank">19352034</a>]<br></li>
<li>Figure 7 indicates that GadE bound to fragments F1 and F2 ( Fig. 7A ) , EvgA bound to F2 and F3 ( Fig. 7B ) , while YdeO bound to F1 and F3 but not to F2 ( Fig. 7C ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15489450" target="_blank">15489450</a>]<br></li>
<li>Further studies are necessary to clarify how EvgA and YdeO regulate the gadE-mdtEF operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19352034" target="_blank">19352034</a>]<br></li>
<li>Next we tested whether EvgA and YdeO affected gadE expression under physiological conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15489450" target="_blank">15489450</a>]<br></li>
<li>EvgA , GadE , and YdeO directly bind the gadE promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15489450" target="_blank">15489450</a>]<br></li>
<li>We presented evidence above that EvgA and YdeO directly activate gadE .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15489450" target="_blank">15489450</a>]<br></li>
<li>EvgA directly activates gadE without assistance from YdeO , and YdeO can activate gadE without EvgA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15489450" target="_blank">15489450</a>]<br></li>
<li>The accumulated data suggest that EvgA , YdeO , and GadE all contribute to gadE induction in response to acid .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15489450" target="_blank">15489450</a>]<br></li>
<li>Regulation of gadE expression involves such regulators as EvgA , YdeO , GadE , PhoP , TrmE , GadX , and GadW ( 12 , 21 , 24 , 30 , 33 ) , which makes GadE the center of regulation for the induction of acid resistance .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21193607" target="_blank">21193607</a>]<br></li>
<li>The phosphorylated EvgA induces the b1500-ydeO operon , and the transcriptional-regulator , YdeO , sequentially induces another transcriptional-regulator , GadE .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17998538" target="_blank">17998538</a>]<br></li>
<li>The Northern blot shown in Fig. 1C indicates that EvgA overexpression does induce GadE and that this induction is heavily dependent on YdeO .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15489450" target="_blank">15489450</a>]<br></li>
<li>B , EvgA dependent and YdeO partly dependent expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19352034" target="_blank">19352034</a>]<br></li>
<li>Thus , EvgA , YdeO , and GadE appear to affect only induction of glutamate-dependent AR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15489450" target="_blank">15489450</a>]<br></li>
<li>Induction of GadA or GadB by overexpressing EvgA and YdeO is dependent on GadE .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15489450" target="_blank">15489450</a>]<br></li>
<li>YdeO then activates AR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15489450" target="_blank">15489450</a>]<br></li>
<li>The previous result established that EvgA and YdeO activate glutamate-depen dent AR under physiological conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15489450" target="_blank">15489450</a>]<br></li>
<li>The fusions were transferred to Salmonella , which lacks this AR system , and plasmid-borne E. coli-speciﬁc regulators EvgA , YdeO , GadE and GadX were introduced .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19220752" target="_blank">19220752</a>]<br></li>
<li>The signiﬁcance of this control extends beyond AR , since GadE , as well as EvgA and YdeO , affect multiple aspects of cell physiology ( 10 , 17 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15489450" target="_blank">15489450</a>]<br></li>
<li>The overexpression results suggested a linear control circuit proceeding from EvgA to YdeO to GadE , which would then activate the gadA or gadBC genes ( 13 , 17 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15489450" target="_blank">15489450</a>]<br></li>
<li>However , it is not clear what purpose this regulation serves in vivo since several studies have shown that EvgA can activate the gadE promoter in a ydeO mutant .24,26 GadE directly activates its own expression while repressing production of YdeO .24 Many structural genes within the AFI , as well as the gadBC locus , are activated directly by GadE , although effectors such as slp appear not to be under GadE control .15,18,27 The contribution of the alternate general stress and stationary phase sigma factor RpoS may also have a role under low pH inducing conditions , since GadE positive feedback can be further supported by its presence .28 The homologous transcription factors * Corresponding author .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20603130" target="_blank">20603130</a>]<br></li>
<li>Note that the safA promoter drives expression of both safA ( also called b1500 ) and ydeO ( 20 ) ; SafA activates the PhoPQ system in response to acid , and YdeO directly activates GadE expression as well as the expression of several other acid resistance genes ( 9 -- 11 , 13 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21571995" target="_blank">21571995</a>]<br></li>
<li>The level of GadE increased in the YY5002 harboring the YdeO-expression plasmid , supporting the prediction that the gadE gene is under the direct and positive control of YdeO .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25375160" target="_blank">25375160</a>]<br></li>
<li>The yehX gene was induced by the YdeO-expression plasmid in both the parent strain and the gadE mutant but the osmF and yehY genes , and parts of osmF-yehYXW transcription unit , were not induced in the gadE mutant ( Fig. 6 and Table 1 ) , implying that GadE activates the known promoters located at the upstream of yehX which is possibly activated by YdeO .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25375160" target="_blank">25375160</a>]<br></li>
<li>The ﬁve other genes ( Fig. 2B , C ) were presumed to be regulated by YdeO or GadE because these genes were all up-regulated by overexpression of YdeO in the absence of evgA .12 ) The gene gadE also showed a decreased level of transcripts in the evgS1 ydeO strain ( Fig. 2B lane 5 ) , suggesting its regulation by YdeO .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19352034" target="_blank">19352034</a>]<br></li>
<li>Among them , we selected ORFs that are directly regulated by EvgA ( ydeP , ydeO , yfdX , frc , yegR , and emrK ) 12 ) and those that are presumed to be regulated by YdeO and GadE ( gadE , gadA , gadB , hdeA , and hdeD ) 12 ) for precise estimation of the transcriptional activation of these genes in the presence of the constitutive evgS1 mutation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19352034" target="_blank">19352034</a>]<br></li>
<li>As shown in EvgS/EvgA up-regulated genes targeted by EvgA , YdeO , and GadE Next , we constructed an evgS1 and ydeO-disrupted strain ( MGR2001 ydeO ) and an evgS1 and gadE-disrupted strain ( MGR2001 gadE ) in order to classify the genes regulated by YdeO or GadE .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19352034" target="_blank">19352034</a>]<br></li>
				</div></ul><br>
			<label class="collapse_EVO" for="_3"><font size="5px">YdeO evolution</font></label>
				<input id="_3" type="checkbox">
				<div class="EVO"><br><ul><li>The role of GadX , GadW , and YdeO , all of which are members of the AraC-like family of transcriptional regulators , is restricted to spe-ciﬁc circuits .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16980449" target="_blank">16980449</a>]<br></li>
<li>In addition , overexpression of gadX induced the expression of several genes encoding regulatory proteins such as YdeO , a member of the AraC/XylS transcriptional regulator family .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14702398" target="_blank">14702398</a>]<br></li>
<li>YdeO is a transcription factor , belonging to the AraC/XylS family .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25375160" target="_blank">25375160</a>]<br></li>
<li>Because YdeO is a member of the AraC/XylS family of transcriptional regulators ( Gallegos et al. , 1997 ) , YdeO might upregulate other acid resistance genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12694615" target="_blank">12694615</a>]<br></li>
<li>YdeO is a member of the AraC/XylS family of transcriptional regulators ( 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12399493" target="_blank">12399493</a>]<br></li>
<li>YdeP is a putative oxidoreductase that has homology to the a subunit of E. coli formate dehydrogenase H. YdeO and YhiE are members of the AraC/XylS family ( Gallegos et al. , 1997 ) and LuxR superfamily ( Fuqua et al. , 1994 ; Robison and Church , 1994 ) of transcriptional regulators respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12694615" target="_blank">12694615</a>]<br></li>
<li>YdeO is among them , and it is a member of the AraC/XylS family ( Gallegos et al. 1997 ) and LuxR super family ( Fukuda et al. 1994 ; Robison and Church 1994 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23274360" target="_blank">23274360</a>]<br></li>
				</div></ul><br>
			<label class="collapse_RP" for="_4"><font size="5px">Processes regulated by YdeO</font></label>
				<input id="_4" type="checkbox">
				<div class="RP"><br><ul><li>These findings also suggest that YdeO activated genes play an important role in primary adaptation , which enables the cell to colonize animal intestines by contributing to adaptation to acidic conditions in the stomach and to anaerobic conditions in the intestine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25375160" target="_blank">25375160</a>]<br></li>
<li>In addition to confirming several regulatory interactions that have been revealed by previous studies , this approach has identified new components in this system that lead to complex dynamics of gene-expression following a drop in pH , including an auto-regulatory loop involving the YdeO activator protein and novel roles for the PhoP protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20603130" target="_blank">20603130</a>]<br></li>
<li>12.38 -8.24 -59.96 1.13 ± 0.55 ± 0.50 ± 0.02 0.04 0.02 1.27 ± 0.50 ± 0.20 ± 0.03 0.03 0.03 / h ) / h ) Fig. 2 Comparison of the transcript levels of the E. coli wild type under different pH values at 7.0 and 6.0 : a global regulatory genes , b PhoB regulatory genes , c acid-inducible genes , d metabolic pathway genes , e respiratory chain genes during micro-aerobic continuous culture at the dilution rate of 0.1 h -1 Fig. 3 a , b Overall metabolic regulation of the E. coli during acidic condition ( a ) Acid Condition Hns ATP ADP PhoQ PhoQ P ATP ADP ¬ EvgS EvgS P cAMP - evgA Pho P PhoP Crp EvgA P B1500 EvgA cAMP-Crp ¬ RpoS b1500 ydeo ¬ YdeO gadE GadE O O GadW GadX hdeA gadB gadc gadW gadA gadX HdeA GadA GadB GadC Acid resistance ( b ) Acidic Condition ( H + ) ( indirect ) EnvZ PhoR fnr OxyR ldhA arcA P Fnr ArcA SoxR / S L PhoR - P OmpR OmpR - P DH pfl yfiD ompC TCA-cycle genes : gltA sdhC mdh sodA PhoB - P PhoB Lactate OmpC ompF asr YfiD Pfl OmpF Asr Formate Acetate ackA pta Acid resistance Moreover , in accord with the proteomic analysis , the expression of yﬁD : : lac reporter fusion and the intracellular content of YﬁD were found to be high during growth at low pH for both aerobic and anaerobic cultures ( Neil et al. 2002 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23274360" target="_blank">23274360</a>]<br></li>
<li>Within this YdeO regulon , we identified four stress-response transcription factors , DctR , NhaR , GadE , and GadW and enzymes for anaerobic-respiration .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25375160" target="_blank">25375160</a>]<br></li>
<li>INVOLVEMENT OF DIFFERENT REGULATORS IN THE ANAEROBIC EXPRESSION OF mdtEF FROM THAT OF AEROBIC CONDITION Previous studies focusing on the expression of gadE-mdtEF operon during acid resistance have revealed that several regulators , such as EvgA , YdeO , MnmE , GadE , GadX , CRP , and H-NS , are involved in the regulation of gadE-mdtEF operon ( Gong et al. , 2004 ; Ma et al. , 2004 ; Hirakawa et al. , 2006 ; Tramonti et al. , 2008 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23874328" target="_blank">23874328</a>]<br></li>
<li>Activation of this system initiates a transcriptional cascade of AR genes encoding regulators EvgA , YdeO , and GadE ( see Fig. 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21193607" target="_blank">21193607</a>]<br></li>
<li>Thus , we propose a complex regulatory cascade in which global regulators ( RpoS , CRP , HN-S , EvgA , etc. ) inﬂuence the expression levels and/or activities of GadX and GadW , which in turn activate the expression and/or activities of transcription activators ( e.g. , YdeO and YhiE ) that directly activate subsets of target genes involved in AR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12730179" target="_blank">12730179</a>]<br></li>
<li>Expression and puriﬁcation of MBP-GadE , MBP-YdeO , and His6-EvgA fusion proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15489450" target="_blank">15489450</a>]<br></li>
<li>This system likely includes several regulons , which are under hierarchical control of a number of regulatory proteins : the two-component signal-transduction-system EvgA/EvgS , GadE , YdeO , GadX , GadW as well as the global regulators RpoS and H -- NS [ 18,19,22,23,35 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15313253" target="_blank">15313253</a>]<br></li>
<li>Identification of the targets directly controlled by H-NS H-NS modulates the expression of several regulators controlling acid stress resistance including HdfR , RcsD , EvgA , YdeO , YdeP , GadE , GadW , GadX , AdiY and CadC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21034467" target="_blank">21034467</a>]<br></li>
<li>Expression of these components is under the control of a complex network of transcription factors , including GadE , GadX , GadW , EvgA , YdeO , and H-NS [ 1 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25375160" target="_blank">25375160</a>]<br></li>
<li>Since a number of regulatory proteins , including RpoS , Fur , PhoPQ , Ada , PhoBR , EvgAS , GadE , and YdeO , have been reported to be involved in expression of the acid survival system ( 2 , 19 , 33 ) , transcription regulations of the ASP genes must be different .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17468243" target="_blank">17468243</a>]<br></li>
<li>A model for regulatory network of acid resistance genes In this study , we have identiﬁed genes directly upregulated by EvgA and genes upregulated by EvgA-induced YdeO .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12694615" target="_blank">12694615</a>]<br></li>
<li>Note that the safA promoter drives expression of both safA ( also called b1500 ) and ydeO ( 20 ) ; SafA activates the PhoPQ system in response to acid , and YdeO directly activates GadE expression as well as the expression of several other acid resistance genes ( 9 -- 11 , 13 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21571995" target="_blank">21571995</a>]<br></li>
<li>The gadA/BC genes , activated in response to acid stress and in stationary phase cells , are subjected to complex circuits of regulation involving 70 , S , cAMP receptor protein , H-NS , EvgAS , TorRS , GadE , GadX , GadW , and YdeO .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15795232" target="_blank">15795232</a>]<br></li>
<li>In this regulatory process H-NS represses the expression of gadE ( encoding the central activator of the glutamate-dependent acid resistance pathway ) both in a direct and an indirect way , via EvgA , YdeO , GadX and GadW [ 1,7,8 ] , while it decreases rcsD expression , essential to the phosphorylation of RcsB ( the capsular synthesis regulator component ) required for the formation of the regulatory complex with GadE [ 6 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21034467" target="_blank">21034467</a>]<br></li>
<li>Indeed , the EvgA-YdeO regulatory pathway plays a secondary modulator role in the glutamate-dependent acid stress response , in comparison to H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21034467" target="_blank">21034467</a>]<br></li>
<li>It is now known that E. coli possesses a number of effectors that enable it to survive otherwise lethal doses of inorganic and organic acid stress ( termed acid resistance or AR ) , many of which are not present in closely related bacteria such as Salmonella spp .8 The function of each AR effector depends on the precise nature of the acid stress .9 -- 13 For example , two isomeric glutamate decarboxylases encoded by gadA and gadB and a glutamate-γ-aminobutyric acid antiporter gadC contribute to AR only when glutamate is available in the extracellular environment ,10 a phenomenon termed glutamate-dependent AR or AR2 , whereas the periplasmic chaperone encoded by hdeA only appears to contribute to AR in dense cell populations .11,14 Several effectors reside within a 15-kb genomic patch termed the `` acid fitness island '' ( AFI ) and , along with the non-proximal gadBC locus , are co-regulated under diverse situations .15 -- 18 In E. coli K-12 , these effectors are subject to tight transcriptional-control and are expressed at low levels during rapid growth at neutral pH , due in part to global H-NS-mediated repression15 ,19 and Lon-mediated degradation of the central activator protein GadE .20 For induction of this latter set of genes , distinct regulatory networks operate depending on the inducing conditions , which include a shift to sublethal pH21 and growth into stationary phase .19,22 A drop from pH 7 to sub-lethal pH 5.5 induces a number of AR effectors via the two-component system EvgA/EvgS , which is encoded outside of the AFI .23,24 Two intermediate transcription factors , YdeO and the AFI-encoded GadE , are transcriptionally activated by the presumably phosphorylated response regulator EvgA .18,24 -- 26 YdeO can also activate gadE transcription forming a coherent feed-forward loop18 ( Fig. 1a ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20603130" target="_blank">20603130</a>]<br></li>
<li>Key words : transcriptional cascade ; EvgS/EvgA ; YdeO ; GadE ; acid resistance The EvgS/EvgA two-component signal-transduction-system in Escherichia coli ( E. coli ) 1 ) is highly homologous to the virulence-related BvgS/BvgA system of Bordetella pertusis and the KvgS/KvgA system , which is present only in virulent strains of Klebsiella pneumoniae .2 ) A BLAST search also indicated the presence of homologous systems in other microorganisms such as Shigella ﬂexineri , Pseudomonas aeruginosa , and Yersi-nia pestis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19352034" target="_blank">19352034</a>]<br></li>
<li>In this study , we showed that YdeO directly regulates the expression of three transcription factor genes , nhaR , gadE , and gadW ( see Fig. 7 ) , proving the novel transcription cascade : EvgAS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25375160" target="_blank">25375160</a>]<br></li>
<li>In addition , overexpression of gadX induced the expression of several genes encoding regulatory proteins such as YdeO , a member of the AraC/XylS transcriptional regulator family .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14702398" target="_blank">14702398</a>]<br></li>
<li>Indole signaling for mdtEF expression is independent of the EvgSA pathway but mediated by GadX , which is also an AraC / XylS regulator that cooperates with YdeO ( 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16885453" target="_blank">16885453</a>]<br></li>
<li>These results altogether indicate that YdeO plays a role as a positive regulator for expression of all seven operons , nhaAR , hyaABC-DEF , appCBA , yiiS-yiiT/uspD , gadW , gadE-mdtEF , and slp-dctR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25375160" target="_blank">25375160</a>]<br></li>
<li>The expression of nhaR-lux , slp-lux , and gadE-lux was found to be activated in the presence of the YdeO-expressing plasmid ( Fig. 3A ) , indicating that YdeO is also a positive regulator for these promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25375160" target="_blank">25375160</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_25">
Yamanaka , Y. , Oshima , T. , Ishihama , A. & Yamamoto , K. ( 2014 ) The role of the Escherichia coli YdeO regulator in response to acidic and anaerobic stresses .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26010043" target="_blank">26010043</a>]<br></label>
<input id="rp_similarity_25", type="checkbox">
<div class="RP_similar">
<li>The role of YdeO regulator for Escherichia coli response to acidic and anaerobic stresses .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25406449" target="_blank">25406449</a>]<br></li></div>
<li>Taken all together , we suggest that YdeO is a key regulator of E. coli survival in both acidic and anaerobic conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25375160" target="_blank">25375160</a>]<br></li>
<li>Taking these observations together we propose that YdeO is the regulator which coordinates the response to acid and anaerobic conditions in E. coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25375160" target="_blank">25375160</a>]<br></li>
<li>Results and discussion Induction of acid resistance by overexpression of YdeP , YdeO and YhiE in exponentially growing cells We have reported that overexpression of the response regulator EvgA confers acid resistance to exponentially growing E. coli ( Masuda and Church , 2002 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12694615" target="_blank">12694615</a>]<br></li>
<li>YdeO ( an AraC/XylS regulator for E. coli acid resistance ) ( 13 , 20 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16885453" target="_blank">16885453</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_31">
Yamanaka Y , Ishihama A , Yamamoto K ( 2012 ) Induction of YdeO , a regulator for acid resistance genes , by ultraviolet irradiation in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25375160" target="_blank">25375160</a>]<br></label>
<input id="rp_similarity_31", type="checkbox">
<div class="RP_similar">
<li>Induction of YdeO , a regulator for acid resistance genes , by ultraviolet irradiation in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25406449" target="_blank">25406449</a>]<br></li></div>
<li>YdeO induces Slp-YhiF , HdeA , HdeD , YhiE and GadABC , which induce acid resistance by working co-operatively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12694615" target="_blank">12694615</a>]<br></li>
<li>The reduction in EvgA-induced acid resistance by ydeO deletion independently of ydeP deletion suggests that YdeO induces gene ( s ) involved in acid resistance except for ydeP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12694615" target="_blank">12694615</a>]<br></li>
<li>These results suggest that YdeO induces acid resistance gene ( s ) in addition to yhiE .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12694615" target="_blank">12694615</a>]<br></li>
<li>To conﬁrm the involvement of YdeP , YdeO and YhiE in acid resistance , we constructed expression plasmids , introduced them into MG1655 ( ATCC 47076 ) and tested the effect of overexpressing these proteins on acid resistance .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12694615" target="_blank">12694615</a>]<br></li>
<li>Construction of plasmids for gene-expression YdeO , YdeP , YhiE and GadX expression plasmids , pQEydeO , pQEydeP , pQEyhiE and pQEgadX , respectively , were constructed as follows .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12694615" target="_blank">12694615</a>]<br></li>
<li>Induction of gene-expression by YdeO .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25375160" target="_blank">25375160</a>]<br></li>
				</div></ul><br>
			<label class="collapse_SIT" for="_5"><font size="5px">YdeO binding site</font></label>
				<input id="_5" type="checkbox">
				<div class="SIT"><br><ul><li>Construction of YdeO expression plasmids To construct pYY0401 for YdeO-3xFLAG expression , DNA fragments containing the ydeO coding region were amplified by PCR using E. coli YY5001 genomic DNA , including the 3xflag tag at the end of ydeO as a template , and a pair of primers , YDEOF-2 and YDEOR-3 , in which the Bam HI and Eco RI sites were included ( see sequences in Table .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25375160" target="_blank">25375160</a>]<br></li>
<li>It should be noted , however , that a version of this site ( TGGCAAatTaGCCA ) is also located at -55 bp from the start of the T3 transcript making it tempting to suggest this sequence might allow YdeO and GadX to activate both promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19220752" target="_blank">19220752</a>]<br></li>
<li>Even though this YdeO-box like sequence exists within the appC promoter , which is located immediately downstream of a YdeO-binding site ( see Fig. 1 ) , the binding in vitro of YdeO to the appC promoter probe was not high ( see Fig. 2 ) , implying that an as yet unidentified additional transcription factor or DNA secondary structure is needed for efficient binding of YdeO to the target promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25375160" target="_blank">25375160</a>]<br></li>
<li>These results indicate that YdeO directly binds to at least these six sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25375160" target="_blank">25375160</a>]<br></li>
<li>However , no sites were found corresponding to a predicted EvgA binding consensus sequence , and nothing is known of potential YdeO binding sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15489450" target="_blank">15489450</a>]<br></li>
<li>Location of YdeO binding sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25375160" target="_blank">25375160</a>]<br></li>
<li>Electrophoretic mobility shift assays ( EMSA ) were used to test whether His-tagged EvgA , MBP-YdeO , and MBP-GadE proteins bound to different regions within the hdeD-gadE intergenic sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15489450" target="_blank">15489450</a>]<br></li>
<li>GadX binds to the gadE control region Regulators EvgA , YdeO and GadE bind to subfragments of the 798 bp gadE upstream control region ( Ma et al. , 2004 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17660422" target="_blank">17660422</a>]<br></li>
				</ul></div><br>
			<label class="collapse_TU" for="_6"><font size="5px">Transcription units regulated by YdeO</font></label>
				<input id="_6" type="checkbox">
				<div class="TU"><br><ul><li>An MBP-YdeO fusion protein was constructed by excising ydeO from pQEydeO ( 17 ) using BamHI and HindIII and religating the fragment downstream of MalE in plasmid pMALc2E ( New England Biolabs ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15489450" target="_blank">15489450</a>]<br></li>
<li>Since the appC promoter is transcribed in vivo by RNA polymerase containing the RpoS sigma factor and is induced by AppY [ 32,33 ] , one possibility is that AppY and/or RpoS sigma are required for the efficient binding of YdeO to the appC promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25375160" target="_blank">25375160</a>]<br></li>
<li>( b ) Transcriptional fusions of the b1500-ydeO , gadE , gadBC , and asr gene promoters were constructed in the pLUX vector , and the response of each of these promoters to a shift from pH 7 to pH values ranging between 6.3 and 4.9 ( as well as a mock induction at pH 7 ) was assayed in an E. coli MG1655 wild-type genetic back ¬ GadX and GadW , along with the small regulatory RNA GadY , collaborate with RpoS to relieve H-NS-mediated repression of the system when cultures enter into stationary phase16 but have also been reported to have a role during low pH response .20,29 In addition , GadW is under direct control of the PhoPQ two-component system ,30 and it has been shown that EvgAS can cross-talk to PhoPQ via the connector protein encoded by b1500 ( also known as safA26 ) , which resides directly upstream of and in the same operon as YdeO .18,31 As a first step to building a mathematical description of this complex network , we wished to assess whether a dynamic approach to monitoring gene expression within the network was both feasible and valid and in particular whether such an approach would reveal any novel aspects of the network that had not been detected to date .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20603130" target="_blank">20603130</a>]<br></li>
<li>It is also unclear whether YdeO induces slp-yhiF , hdeAB-yhiD , hdeD , gadA and gadBC directly or via YhiE .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12694615" target="_blank">12694615</a>]<br></li>
<li>GadE induced by YdeO stimulated the transcription of hdeAB-yhiD , hdeD , gadAX , gadCB , mdtEF , gadW , and yhiM as well as those previously reported promoters [ 9,34,35 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25375160" target="_blank">25375160</a>]<br></li>
<li>Genes induced in both strains are organized into a total of 12 transcriptional units ( Table 1 ) , including five transcription units , hyaABCDEF , appCBA , slp-dctR , and nhaAR , that are under the direct control of YdeO ( see above ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25375160" target="_blank">25375160</a>]<br></li>
<li>Thus , we conclude that YdeO is a positive regulator for transcription of operons controlled by seven promoters , the nhaR promoter , hyaA promoter , appC promoter , yiiS promoter , slp promoter , gadE promoter , and gadW promoter ( Fig. 7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25375160" target="_blank">25375160</a>]<br></li>
<li>Amongst genes downstream of a YdeO-binding site , 19 genes , ( nhaA , nhaR , hyaA , hyaB , hyaC , hyaD , hyaE , hyaF , appC , appB , appA , yiiS , yiiT / uspD , slp , dctR , gadE , mdtE , mdtF , and gadW ) were induced more than 3-fold by the over-expression of YdeO ; while three genes , yccA , yiiR , and yhiS , were not affected in both duplicate experiments .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25375160" target="_blank">25375160</a>]<br></li>
<li>These 19 genes induced by YdeO constitute a total of 7 transcriptional units , nhaAR , hyaABCDEF , appCBA , yiiS-yiiT/uspD , slp-dctR , gadE-mdtEF , and gadW .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25375160" target="_blank">25375160</a>]<br></li>
<li>These results altogether indicate that YdeO plays a role as a positive regulator for expression of all seven operons , nhaAR , hyaABC-DEF , appCBA , yiiS-yiiT/uspD , gadW , gadE-mdtEF , and slp-dctR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25375160" target="_blank">25375160</a>]<br></li>
<li>It is inconsistent with the RT-qPCR result that the mRNA level of gadW increased in the presence of YdeO expression as detected by RT-qPCR ( Table 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25375160" target="_blank">25375160</a>]<br></li>
<li>The reporter and RT-qPCR assays indicated that all of the promoters located downstream of these YdeO-binding sites are activated by YdeO ( see Table 1 and Fig. 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25375160" target="_blank">25375160</a>]<br></li>
<li>Knowledge about the regulatory functions of YdeO is limited except that it is known that YdeO activates transcription of the gad system components , gadE , gadA and gadBC [ 7,8,9 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25375160" target="_blank">25375160</a>]<br></li>
<li>At present , it is known that the expression of the essential regulator GadE , a LuxR-like protein that is encoded by the gadE gene and located in the AFI ( Fig. 1 ) , is under the control of three different activation circuits ( for review , see Foster , 2004 ) : ( i ) the EvgAS/YdeO-dependent circuit activates gadE during the exponential phase of growth in minimal medium ( pH 5.5 ) containing glucose and relies on the activity of both the EvgAS two-component system and the activator YdeO , a member of the AraC/XylS family of transcriptional regulators , ( ii ) the TrmE-dependent circuit activates gadE in the stationary phase in unbuffered LB medium containing glucose and relies on the activity of TrmE , an Era-like molecular switch GTPase , and ( iii ) the GadX/GadW-dependent circuit is required for gadE activation upon entry into the stationary phase in buffered LB medium and involves the transcriptional regulators GadX and GadW , the genes of which are located downstream of gadA and gadX respectively ( Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18808381" target="_blank">18808381</a>]<br></li>
<li>YdeO is an AraC-like transcription factor , as are GadX and the product of the gadW gene , which lies immediately downstream of gadX .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12730179" target="_blank">12730179</a>]<br></li>
<li>To construct pYdeO for expression of intact YdeO , DNA fragments containing the ydeO coding region were amplified by PCR using E. coli W3110 type A [ 14 ] genomic DNA as a template and the primers , YDEOF-2 and YDEOR-2 ( see sequences in Table .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25375160" target="_blank">25375160</a>]<br></li>
<li>The YdeO-expression plasmid , pYdeO ( ydeO ) , and the empty expression vector pTrc99A were transformed into these E. coli strains and the transformants were grown in LB medium until log phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25375160" target="_blank">25375160</a>]<br></li>
<li>However , the fact that YdeO induced yhiE but GadX did not suggests that YdeO binds to the downstream region of hdeD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12694615" target="_blank">12694615</a>]<br></li>
<li>Several genes induced by overexpression of YdeO and GadX overlapped ; however , yhiE was induced only by YdeO .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12694615" target="_blank">12694615</a>]<br></li>
<li>Many of the YdeO-activated genes were also activated by GadX , whereas yhiEUV were induced only by YdeO .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12694615" target="_blank">12694615</a>]<br></li>
				<label class="TU_collapse_similarity" for="tu_similarity_21">
The second class contains 12 operons induced by YdeO overexpression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12694615" target="_blank">12694615</a>]<br></label>
<input id="tu_similarity_21", type="checkbox">
<div class="TU_similar">
<li>The second class contains 12 operons or genes induced by YdeO overexpression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19352034" target="_blank">19352034</a>]<br></li></div>
<li>Operons or genes in the second class were induced by EvgA overexpression only in the presence of ydeO , which suggests the following model of the regulatory network of the acid resistance genes : EvgA directly upregulates operons or genes containing ydeO in the ﬁrst class and indirectly upregulates operons or genes in the second class containing gadE , a LuxR-family regulator gene , via YdeO .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19352034" target="_blank">19352034</a>]<br></li>
<li>In addition , overexpression of gadX induced the expression of several genes encoding regulatory proteins such as YdeO , a member of the AraC/XylS transcriptional regulator family .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14702398" target="_blank">14702398</a>]<br></li>
<li>EvgA overexpression induced the expression of ydeO , an AraC/XylS regulator , indicating that some of the genes induced by EvgA overexpression might be directly induced by YdeO instead of EvgA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12399493" target="_blank">12399493</a>]<br></li>
<li>These results suggest that YdeO binds to the downstream region of hdeD , upregulates yhiUV and , consequently , induces multidrug resistance in a DacrB strain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12694615" target="_blank">12694615</a>]<br></li>
<li>YdeO induces Slp-YhiF , HdeA , HdeD , YhiE and GadABC , which induce acid resistance by working co-operatively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12694615" target="_blank">12694615</a>]<br></li>
<li>The contributions of YhiD and HdeB to acid resistance are unclear.YdeO also induces YhiUV , which induce multidrug resistance .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12694615" target="_blank">12694615</a>]<br></li>
<li>The acid resistance induced by YdeO overexpression was abolished by deletion of yhiE , gadC , slp-yhiF , hdeA or hdeD , genes induced by YdeO overexpression , suggesting that several genes orchestrate YdeO-induced acid resis ¬ Accepted 20 January , 2003 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12694615" target="_blank">12694615</a>]<br></li>
<li>The multidrug resistance caused by overexpression of EvgA and YdeO is due to the induction of mdtEF gene expression ( 6 , 7 , 14 , 20 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16885453" target="_blank">16885453</a>]<br></li>
<li>The reduction in EvgA-induced acid resistance by ydeO deletion independently of ydeP deletion suggests that YdeO induces gene ( s ) involved in acid resistance except for ydeP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12694615" target="_blank">12694615</a>]<br></li>
<li>These results suggest that yhiE is essential for acid resistance induced by YdeO overexpression and that EvgA overexpression induces yhiE via the induction of YdeO .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12694615" target="_blank">12694615</a>]<br></li>
<li>These results are similar to our previous observation that EvgA overexpression does not confer acid resistance to the ydeP-b1500-ydeO deletion strain , and are consistent with the hypothesis that YdeO induces yhiE expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12694615" target="_blank">12694615</a>]<br></li>
<li>Although this could be due to direct repression by YdeO at the mgtA promoter , it could also be explained by our novel finding that YdeO negatively regulates b1500 and ydeO expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20603130" target="_blank">20603130</a>]<br></li>
<li>Accordingly the constructed nhaRlux reporter plasmid containing this novel nhaR promoter was also activated in the presence of YdeO expression ( Fig. 3A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25375160" target="_blank">25375160</a>]<br></li>
<li>Although the yiiS and uspD genes , encoding conserved proteins with unidentified function , were expressed even without the over-expression of YdeO , their expressions were further increased after YdeO expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25375160" target="_blank">25375160</a>]<br></li>
<li>However , it is not clear what purpose this regulation serves in vivo since several studies have shown that EvgA can activate the gadE promoter in a ydeO mutant .24,26 GadE directly activates its own expression while repressing production of YdeO .24 Many structural genes within the AFI , as well as the gadBC locus , are activated directly by GadE , although effectors such as slp appear not to be under GadE control .15,18,27 The contribution of the alternate general stress and stationary phase sigma factor RpoS may also have a role under low pH inducing conditions , since GadE positive feedback can be further supported by its presence .28 The homologous transcription factors * Corresponding author .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20603130" target="_blank">20603130</a>]<br></li>
<li>Note that the safA promoter drives expression of both safA ( also called b1500 ) and ydeO ( 20 ) ; SafA activates the PhoPQ system in response to acid , and YdeO directly activates GadE expression as well as the expression of several other acid resistance genes ( 9 -- 11 , 13 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21571995" target="_blank">21571995</a>]<br></li>
<li>In previous studies , it was found that expression of the mdtEF genes is up-regulated by the bacterial two-component signal transduction system EvgSA ( 20 ) and the AraC/XylS acid resistance regulator YdeO ( 14 ) , while their expression is repressed by a histone-like protein , H-NS ( 21 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16885453" target="_blank">16885453</a>]<br></li>
<li>The results revealed that a total of 106 genes were markedly affected by YdeO expression in the wild-type and included 53 up - and the same number of down-regulated genes ( Tables S2 and S3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25375160" target="_blank">25375160</a>]<br></li>
<li>Among the 53 genes up-regulated by YdeO expression , clustering analysis showed 23 genes were induced in both the parent strain and the gadE-deficient mutant and 30 genes induced in the wild-type but not the gadE-deficient mutant ( Fig. 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25375160" target="_blank">25375160</a>]<br></li>
<li>On the other hand , the rest of the 30 up-regulated genes forming 21 transcription units were induced in the wild-type but not in the gadE-deficient mutant ( Table 1 ) , indicating that these 21 transcription units are under the direct control of GadE but the indirect control of YdeO .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25375160" target="_blank">25375160</a>]<br></li>
<li>Construction of plasmids for gene expression YdeO , YdeP , YhiE and GadX expression plasmids , pQEydeO , pQEydeP , pQEyhiE and pQEgadX , respectively , were constructed as follows .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12694615" target="_blank">12694615</a>]<br></li>
<li>An evgA-lacZ fusion gene was then constructed to examine whether this operon was also under acid control and whether YdeO or GadE affected expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15489450" target="_blank">15489450</a>]<br></li>
<li>GadE , GadX and YdeO differentially affect the expression of individual gadE promoters Plasmids expressing the E. coli-speciﬁc gadE , gadX and ydeO genes were then introduced into the Salmonella strains .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19220752" target="_blank">19220752</a>]<br></li>
<li>Next we tested whether EvgA and YdeO affected gadE expression under physiological conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15489450" target="_blank">15489450</a>]<br></li>
<li>However , neither EvgA nor YdeO is required for gadE expression in stationary phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17660422" target="_blank">17660422</a>]<br></li>
<li>This will diminish YdeO expression and the YdeO-dependent transcription of gadE from P3P2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19220752" target="_blank">19220752</a>]<br></li>
<li>Genes up-regulated by YdeO expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25375160" target="_blank">25375160</a>]<br></li>
<li>Induction of gene expression by YdeO .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25375160" target="_blank">25375160</a>]<br></li>
<li>The level of GadE increased in the YY5002 harboring the YdeO-expression plasmid , supporting the prediction that the gadE gene is under the direct and positive control of YdeO .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25375160" target="_blank">25375160</a>]<br></li>
<li>The yehX gene was induced by the YdeO-expression plasmid in both the parent strain and the gadE mutant but the osmF and yehY genes , and parts of osmF-yehYXW transcription unit , were not induced in the gadE mutant ( Fig. 6 and Table 1 ) , implying that GadE activates the known promoters located at the upstream of yehX which is possibly activated by YdeO .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25375160" target="_blank">25375160</a>]<br></li>
<li>However , continuous expression of gadA , gadBC , and the hde genes is likely necessary to cope with permanent acid stress ; this requires continuously high expression of GadE , probably mediated by the EvgSA/YdeO pathway .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15716429" target="_blank">15716429</a>]<br></li>
<li>GlcNAc-induced mdtEF gene expression is mediated by neither the EvgSA-YdeO nor the RpoS-GadX signaling pathway .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16885453" target="_blank">16885453</a>]<br></li>
<li>The results indicated that evgAS is not acid induced and is not subject to autoinduction and that neither YdeO nor GadE had any major effect on evgA expression ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15489450" target="_blank">15489450</a>]<br></li>
<li>The expression of glutamate-dependent acid resistance genes is induced at low pH and is subject to regulation by GadE , GadW , GadX , TrmE , HNS , TorR , Crp , YdeO , the EvgSA two-component regulatory system and the sigma factor RpoS ( reviewed in Foster 2004 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17273855" target="_blank">17273855</a>]<br></li>
				</div></ul></div><br>
	</div>
</body>
</html>