<!doctype html>
<style>
@charset "utf-8";
body {
	height: 100%;
	margin: 0;
	padding: 0;
}

.banner {
	position: fixed;
	top: 0;
	width: 100%;
	height: 65px;
	background: gray;
	line-height: 65px;
}
.spacer {
	height: 10px;
}

.main-content {
	padding: 5%;
}

.collapse_ACT {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-style: none solid none none;
	border-color: #B4045F;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_ACT + input{
	display: none;
}
.collapse_ACT + input + div{
	display: none;
}
.collapse_ACT + input:checked + div{
	display: block;
}
.ACT {
	text-align: justify;
	text-justify: inter-word;
}
.ACT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B4045F;
}
.ACT_collapse_similarity:hover{
	background: #e5d3dc;
}
.ACT_collapse_similarity + input {
	display: none;
}
.ACT_collapse_similarity + input + div{
	display: none;
}

.ACT_collapse_similarity + input:checked + div{
	display: block;
}
.ACT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.ACT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B4045F;
	padding-left: 5px;
}
.ACT_collapse_similarity:hover{
	background: #e5d3dc;
}
.ACT_collapse_similarity + input {
	display: none;
}
.ACT_collapse_similarity + input + div{
	display: none;
}
.ACT_collapse_similarity + input:checked + div{
	display: block;
}
.ACT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.ACT_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #B4045F;
}
.ACT_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_DOM {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-style: none solid none none;
	border-color: #B404AE;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_DOM + input{
	display: none;
}
.collapse_DOM + input + div{
	display: none;
}
.collapse_DOM + input:checked + div{
	display: block;
}
.DOM {
	text-align: justify;
	text-justify: inter-word;
}
.DOM_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B404AE;
	padding-left: 5px;
}
.DOM_collapse_similarity:hover{
	background: #e8d7e7;
}
.DOM_collapse_similarity + input {
	display: none;
}
.DOM_collapse_similarity + input + div{
	display: none;
}
.DOM_collapse_similarity + input:checked + div{
	display: block;
}
.DOM_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}.DOM_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #B404AE;
}
.DOM_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_EVO {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #7401DF;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_EVO + input{
	display: none;
}
.collapse_EVO + input + div{
	display: none;
}
.collapse_EVO + input:checked + div{
	display: block;
}
.EVO {
	text-align: justify;
	text-justify: inter-word;
}
.EVO_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #7401DF;
	padding-left: 5px;
}
.EVO_collapse_similarity:hover{
	background: #e2d7ed;
}
.EVO_collapse_similarity + input {
	display: none;
}
.EVO_collapse_similarity + input + div{
	display: none;
}
.EVO_collapse_similarity + input:checked + div{
	display: block;
}
.EVO_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.EVO_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #7401DF;
}
.EVO_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_RP {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #0174DF;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_RP + input{
	display: none;
}
.collapse_RP + input + div{
	display: none;
}
.collapse_RP + input:checked + div{
	display: block;
}
.RP {
	text-align: justify;
	text-justify: inter-word;
}
.RP_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #0174DF;
	padding-left: 5px;
}
.RP_collapse_similarity:hover{
	background: #c9d7e2;
}
.RP_collapse_similarity + input {
	display: none;
}
.RP_collapse_similarity + input + div{
	display: none;
}
.RP_collapse_similarity + input:checked + div{
	display: block;
}
.RP_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.RP_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #0174DF;
}
.RP_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_SIT {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #0489B1;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_SIT + input{
	display: none;
}
.collapse_SIT + input + div{
	display: none;
}
.collapse_SIT + input:checked + div{
	display: block;
}
.SIT {
	text-align: justify;
	text-justify: inter-word;
}
.SIT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #0489B1;
	padding-left: 5px;
}
.SIT_collapse_similarity:hover{
	background: #d0dfe2;
}
.SIT_collapse_similarity + input {
	display: none;
}
.SIT_collapse_similarity + input + div{
	display: none;
}
.SIT_collapse_similarity + input:checked + div{
	display: block;
}
.SIT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.SIT_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #0489B1;
}
.SIT_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_TU {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #04B486;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_TU + input{
	display: none;
}
.collapse_TU + input + div{
	display: none;
}
.collapse_TU + input:checked + div{
	display: block;
}
.TU {
	text-align: justify;
	text-justify: inter-word;
}
.TU_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #04B486;
	padding-left: 5px;
}
.TU_collapse_similarity:hover{
	background: #d0e2de;
}
.TU_collapse_similarity + input {
	display: none;
}
.TU_collapse_similarity + input + div{
	display: none;
}
.TU_collapse_similarity + input:checked + div{
	display: block;
}
.TU_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.TU_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #04B486;
}
.TU_similar > a {
	text-decoration: none;
	color: gray;
}

ul { 
	list-style-type: none;
	padding: 0;
	margin: 0;
}
ul li > a {
	text-decoration: none;
	color: gray;
}
ul li:hover, ul li:focus {
	background: #F0F0F0;
}

ul li:first-child span {
  border-left:none;  
}

</style>
<html>
<head>
<meta charset="utf-8">
<title>TF properties - YefM</title>
<meta name="theme-color" content="#ffffff">
</head>
<body>
	<div class="banner">
		<center><font color="white" size="16px" face="arial">YefM</font></center>
	</div>
	<div align="center" class="main-content">
		<div class="spacer">
	</div><br>			<label class="collapse_ACT" for="_1"><font size="5px">YefM transcriptional activity</font></label>
				<input id="_1" type="checkbox">
				<div class="ACT"><br><ul><li>35 , No. 1 325 -- 339 doi :10.1093 / nar/gkl1028 of YefM -- YoeB with paired DNA palindromes exerts transcriptional autorepression Barbara Kędzierska , Lu-Yun Lian and Finbarr Hayes * 1 Faculty of Life Sciences and Manchester Interdisciplinary Biocentre , The University of Manchester , 131 Princess Street , Manchester M1 7DN , UK and School of Biological Sciences , University of Liverpool , 1 Crown Street , Liverpool L69 7ZB , UK Received October 6 , 2006 ; Revised and Accepted November 13 , 2006 ABSTRACT INTRODUCTION Toxin -- antitoxin ( TA ) complexes function in programmed cell death or stress response mechanisms in bacteria .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17170003" target="_blank">17170003</a>]<br></li>
<li>Subsequently , YefM protein was provided in trans from an arabinose-inducible promoter , and its effect on b-galactosidase production by the yefM -- lacZ fusion was examined : b-galactosidase levels were reduced 5.5-fold to 739 ± 164 U in the presence of YefM ( Figure 1C ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17170003" target="_blank">17170003</a>]<br></li>
<li>Upon binding to the YefM homodimer , certain mechanistic conformational changes occur in the three pBAD33Txe + pBAD24 pBAD33 + pBAD24Axe pBAD33Txe + pBAD24Axe pBAD33Txe + pBAD24YefM 1.5 1.2 0.9 0.6 0.3 0 0 120 240 180 60 Time ( min ) Fig. 2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24028219" target="_blank">24028219</a>]<br></li>
<li>YefM concentrations ( mM , left to right ) : 0 , 0.01 , 0.025 , 0.05 , 0.1 , 0.25 , 0.5 , 1.0 , 2.5 and 5.0 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17170003" target="_blank">17170003</a>]<br></li>
<li>The values obtained in the experiments without any repression ( vector pBBRlux bearing the yefM -- yoeB promoter co-introduced with pBAD33 and pKK223-3 plasmids ) were considered as ` no repression ' , and presented values ( fold repression ) reﬂect the efﬁciency of luminescence in the tested systems ( where YefM alone or together with YoeB or TxeAsp83Tyr was present in pBAD33 and pKK223-3 plasmids ) relative to that of the control ( i.e. without any repression ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24028219" target="_blank">24028219</a>]<br></li>
<li>Here we dissect the role of the YefM -- YoeB complex in modulating its own synthesis : YefM is the primary transcriptional-repressor of the yefM-yoeB cassette , with YoeB acting as a repression enhancer .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17170003" target="_blank">17170003</a>]<br></li>
<li>In summary , the repression data corroborate the ﬁnding in toxicity and antitoxicity assays that the Asp83Tyr substitution in Txe allows for effective interaction of this toxin with the YefM antitoxin .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24028219" target="_blank">24028219</a>]<br></li>
<li>The positions corresponding to Tyr82 in YoeB and Tyr53 in YefM are the negatively charged Asp83 in Txe and positively charged Arg54 in Axe , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24028219" target="_blank">24028219</a>]<br></li>
<li>These results suggested that Axe -- Txe and YefM -- YoeB interactions are speciﬁc , and no crossinteractions occur .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24028219" target="_blank">24028219</a>]<br></li>
<li>( B ) Toxicity of Txe is inhibited by Axe , but not YefM .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24028219" target="_blank">24028219</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24028219" target="_blank">24028219</a>]<br></li>
<li>Degradation of YefM , in turn , leads to activation of YoeB and mRNA cleavage .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19400780" target="_blank">19400780</a>]<br></li>
<li>As YefM is a Lon substrate ( Christensen et al. , 2004 ) , YefM is degraded and YeoB mRNA interferase is activated and leads to mRNA cleavage at stop-codons .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19400780" target="_blank">19400780</a>]<br></li>
<li>In contrast , the repressor -- corepressor complex recognizes both motifs more avidly , impyling that YefM within the complex has an enhanced DNA-binding affinity compared to free YefM .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17170003" target="_blank">17170003</a>]<br></li>
<li>Structure determination of free YefM2 , and of YefM and YefM-YoeB structures bound to DNA , will provide further crucial insights into the mechanism by which YefM -- YoeB exerts transcriptional regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17170003" target="_blank">17170003</a>]<br></li>
<li>This distinction was used to assess whether YefM or YefM-YoeB-His6 underwent detectable structural changes when bound to operator DNA , and vice versa .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17170003" target="_blank">17170003</a>]<br></li>
<li>A position on the lower strand that is hypersenstive to DNase I cleavage in the presence of YefM and YefM -- YoeB -- His6 is highlighted by the star .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17170003" target="_blank">17170003</a>]<br></li>
<li>CD analysis of the YefM and YefM -- YoeB -- His6 proteins in the absence and presence of DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17170003" target="_blank">17170003</a>]<br></li>
<li>In the absence of highresolution information about the tertiary structure of free YefM , it remains to be clariﬁed whether YefM undergoes structural transitions between its free state and when associated with YoeB , although this seems likely .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17170003" target="_blank">17170003</a>]<br></li>
<li>The results suggest that YefM undergoes structural transitions when bound to DNA , and that the alterations are induced speciﬁcally by its cognate binding site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17170003" target="_blank">17170003</a>]<br></li>
				</div></ul><br>
			<label class="collapse_DOM" for="_2"><font size="5px">YefM protein domains</font></label>
				<input id="_2" type="checkbox">
				<div class="DOM"><br><ul><li>Prediction software tools for disordered regions within proteins also suggest that YefM is wellstructured , with only a patch of amino acids close to the C-terminus expected to be unfolded ( Figure 9 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17170003" target="_blank">17170003</a>]<br></li>
<li>However , native YefM puriﬁed here exhibited distinct CD minima at 208 and 222 nm that are characteristic of a-helix content and helix -- helix interactions , respectively , and which indicate that the protein was at least partly folded ( Figure 4A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17170003" target="_blank">17170003</a>]<br></li>
<li>This contention is supported by NMR spectra of YefM , which display signatures characteristic of a well-ordered protein containing both a-helical and b-strand secondary structure .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17170003" target="_blank">17170003</a>]<br></li>
<li>Overall , the NMR data suggests that the YefM protein exists either as a mixture of folded and unfolded states , or that the protein has folded and unfolded regions in its tertiary structure .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17170003" target="_blank">17170003</a>]<br></li>
<li>YefM possesses extensive secondary structure CD analysis suggested that YefM is an unfolded protein that entirely lacks secondary structure ( 24 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17170003" target="_blank">17170003</a>]<br></li>
<li>Based on CD analysis , YefM has been described as an intrinsically unfolded protein that typiﬁes a novel family of proteins that entirely lack any secondary structure ( 24 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17170003" target="_blank">17170003</a>]<br></li>
<li>It is known that the C-termi-nal domain of YefM is engaged in contacts with the toxin and a tag at this position could potentially Fig. 7 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24028219" target="_blank">24028219</a>]<br></li>
<li>The genes are all labelled as per the gene products : SymE = SymE family ; cHTH = a Cro/cI type transcriptional regulator , RHH = a MetJ/Arc-type transcription factor , YefM = a YefM-type transcription factor , RelE = a toxin homologous to the RelE toxins , Prim-Hel = the gene encoding a two module protein with a N-terminal DNAG-type primase domain and C-terminal MCM-like AAA + helicase domain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17462020" target="_blank">17462020</a>]<br></li>
<li>These residues , via intermolecular van der Waals interactions , bind the amphipathic a-helix H4 in YefM antitoxin , notably residues Leu64 ( Ile65 in Axe ) , Met65 ( Arg66 in Axe ) , Ile68 ( Asp69 in Axe ) and Leu71 ( Phe72 in Axe ) [ 15 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24028219" target="_blank">24028219</a>]<br></li>
<li>YefM belongs to a unique group of DNA-binding proteins with a distinctive fold typiﬁed as the Phd superfamily , whereas YoeB forms a compact globular structure consisting of a ﬁve-stranded b sheet and two a helices and is related structurally to the RelE superfamily of toxins [ 15 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24028219" target="_blank">24028219</a>]<br></li>
<li>In contrast , the C-terminal halves of the two proteins are implicated in interactions with their cognate toxins and share very little similarity ( 21,33,54 ) , which has been proposed to reﬂect modular exchange of DNA-binding and toxin recognition domains among proteins related to Phd ( 54 ) , including YefM homologues .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17170003" target="_blank">17170003</a>]<br></li>
<li>This observation correlates with the suggestion that a conserved basic patch in the N-terminal domain of YefM might be involved in DNA-recognition ( 21 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17170003" target="_blank">17170003</a>]<br></li>
<li>This correlates with biophysical and structural studies which demonstrated that puriﬁed YefM -- YoeB complex consisted of either a 1:2 or 2:1 mix of the two proteins ( 21,24 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17170003" target="_blank">17170003</a>]<br></li>
<li>In contrast , the repressor -- corepressor complex recognizes both motifs more avidly , impyling that YefM within the complex has an enhanced DNA-binding affinity compared to free YefM .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17170003" target="_blank">17170003</a>]<br></li>
<li>Similar DNA-binding patterns were observed with YefM-His6 ( data not shown ) indicating that the C-terminal tag does not detectably impede interaction of YefM with DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17170003" target="_blank">17170003</a>]<br></li>
<li>Notably , resi-dues Asp83 in Txe and Arg54 in Axe are conserved at corresponding positions in S. pneumoniae YefM and YoeB , respectively , compared with the Tyr residues at the equivalent positions in E. coli YefM and YoeB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24028219" target="_blank">24028219</a>]<br></li>
<li>Moreover , the mycobacterial proteins possess Tyr residues at the positions corresponding to Tyr53 in YefM and Tyr82 ( Fig. 7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24028219" target="_blank">24028219</a>]<br></li>
<li>The data with E. coli , enterococcal , streptococcal and mycobacterial YefM -- YoeB homologs are consistent with a key role in toxin -- antitoxin interaction for residues at positions equivalent to Tyr53 in YefM and Tyr82 in YoeB of E. coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24028219" target="_blank">24028219</a>]<br></li>
<li>The only pair of residues that interact with each other and possess pronounced physicochemical differences are Tyr82 of YoeB and Tyr53 of YefM .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24028219" target="_blank">24028219</a>]<br></li>
<li>In contrast , toxicity of the Txe variant bearing the Asp83Tyr change in which a Txe-speciﬁc residue was altered to a YoeB-speciﬁc amino acid was signiﬁcantly impaired by YefM ( Fig. 5C ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24028219" target="_blank">24028219</a>]<br></li>
<li>The analyses of amino acid sequence and the complex structures suggest that the mechanisms of inhibition of YoeB homologs by YefM homologs , including Axe -- Txe , are highly similar despite changes in amino acid composition at some positions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24028219" target="_blank">24028219</a>]<br></li>
<li>The residues described that differ between YoeB and Txe toxins and between YefM and Axe antitoxins may be implicated in the contact speci-ﬁcity of these complexes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24028219" target="_blank">24028219</a>]<br></li>
<li>Based on molecular modeling , selected amino acid residues were changed to assess which positions were crucial in the speciﬁcity of toxin -- antitoxin interaction in the related Axe -- Txe and YefM -- YoeB complexes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24028219" target="_blank">24028219</a>]<br></li>
<li>All residues chosen for mutational analysis are depicted in Fig. 3B , C. Substitutions of Txe-speciﬁc residues by YoeB-speciﬁc residues neither abolishes toxicity entirely nor abolishes effective interactions with the Axe antitoxin On the basis of the molecular modeling described above , ﬁve candidate amino acids in the Txe toxin were targeted which might be implicated in the speci-ﬁcity of antitoxin interactions in Axe -- Txe and YefM -- YoeB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24028219" target="_blank">24028219</a>]<br></li>
<li>Therefore the Asp83Tyr substitution in Txe may make this toxin sufﬁciently similar to YoeB to permit effective interactions with YefM .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24028219" target="_blank">24028219</a>]<br></li>
<li>Ovals indicate residues crucial for YoeB toxicity ( dotted line ) and important for YefM -- YoeB interactions which cause changes in the toxin active site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24028219" target="_blank">24028219</a>]<br></li>
<li>In summary , upon formation of the complex , YoeB residues necessary for endoribonuclease activity are hidden by binding of the YefM antitoxin .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24028219" target="_blank">24028219</a>]<br></li>
<li>A mechanistic conformational change of the three C-terminal amino acids of YoeB ( Tyr82 , His83 , Tyr84 ) takes place upon YefM binding which destabilizes a canonical conformation in the catalytic center of the endoribonuclease [ 15 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24028219" target="_blank">24028219</a>]<br></li>
<li>The three residues at the C-terminal tip of YoeB that form part of the atypical endoribonucleolytic fold are rearranged into a less favourable conformation when in complex with the YefM dimer , partly explaining the mechanism by which the antitoxin blocks the toxic activity of YoeB ( 21 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17170003" target="_blank">17170003</a>]<br></li>
<li>The N-terminal segments of the YefM monomers form a symmetrical dimer .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24028219" target="_blank">24028219</a>]<br></li>
<li>The N-terminal regions of YefM form a symmetrical dimer within the YefM2 -- YoeB complex and do not contact YoeB directly in the crystal structure ( 21 ) , although the YefM recognition determinant that interacts most strongly with YoeB was previously mapped to the N-terminal segment of the protein ( 24 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17170003" target="_blank">17170003</a>]<br></li>
<li>One of the YefM C-terminal segments exclusively binds the YoeB monomer , whereas the other YefM C-terminus remains structurally disordered [ 15 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24028219" target="_blank">24028219</a>]<br></li>
<li>It is difﬁcult to envisage how YefM might achieve dimerization and become extensively structured principally by the interaction of a single a-helix at the C-terminal end of one YefM monomer with YoeB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17170003" target="_blank">17170003</a>]<br></li>
<li>The YefM -- YoeB complex forms a heterotrimer that contains one YoeB monomer and two YefM subunits .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24028219" target="_blank">24028219</a>]<br></li>
<li>The recently-described tertiary structure of the YefM -- YoeB heterotrimeric complex consists of one YoeB monomer associated with a YefM homodimer .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17170003" target="_blank">17170003</a>]<br></li>
<li>Paired 50-TGTACA-30 motifs are common in yefM-yoeB regulatory regions in diverse genomes suggesting that interaction of YefM -- YoeB with these motifs is a conserved mechanism of operon autoregulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17170003" target="_blank">17170003</a>]<br></li>
<li>A single amino acid substitution in the Txe toxin not only made this protein susceptible to the inhibitory action of YefM antitoxin , but also allowed formation of an active repressor -- corepressor complex of the yefM -- yoeB endoribonuclease that cleaves mRNA molecules three promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24028219" target="_blank">24028219</a>]<br></li>
<li>Although complex formation apparently was inefﬁcient , the interaction of YefM with the yefM-yoeB promoter region was speciﬁc as no retarded species were observed in EMSA with an unrelated substrate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17170003" target="_blank">17170003</a>]<br></li>
				</div></ul><br>
			<label class="collapse_EVO" for="_3"><font size="5px">YefM evolution</font></label>
				<input id="_3" type="checkbox">
				<div class="EVO"><br><ul><li>The genes are all labelled as per the gene products : SymE = SymE family ; cHTH = a Cro/cI type transcriptional regulator , RHH = a MetJ/Arc-type transcription factor , YefM = a YefM-type transcription factor , RelE = a toxin homologous to the RelE toxins , Prim-Hel = the gene encoding a two module protein with a N-terminal DNAG-type primase domain and C-terminal MCM-like AAA + helicase domain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17462020" target="_blank">17462020</a>]<br></li>
<li>YefM belongs to a unique group of DNA-binding proteins with a distinctive fold typiﬁed as the Phd superfamily , whereas YoeB forms a compact globular structure consisting of a ﬁve-stranded b sheet and two a helices and is related structurally to the RelE superfamily of toxins [ 15 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24028219" target="_blank">24028219</a>]<br></li>
<li>Cherny I & Gazit E ( 2004 ) The YefM antitoxin deﬁnes a family of natively unfolded proteins : implications as a novel antibacterial target .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17263853" target="_blank">17263853</a>]<br></li>
				<label class="EVO_collapse_similarity" for="evo_similarity_4">
Alignments of selected members of the YefM ( A ) and YoeB ( B ) families of proteins using CLUSTALW2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24028219" target="_blank">24028219</a>]<br></label>
<input id="evo_similarity_4", type="checkbox">
<div class="EVO_similar">
<li>Protein sequence analysis Alignments of selected members of the YefM and YoeB families of proteins were performed using CLUSTALW2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24028219" target="_blank">24028219</a>]<br></li></div>
<li>The mycobacterial proteins show high identity -- 42 % for antitoxin and 46 % for the toxin -- to E. coli YefM -- YoeB proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24028219" target="_blank">24028219</a>]<br></li>
<li>Results of molecular modeling and protein -- protein docking show a high similarity between Axe -- Txe and YefM -- YoeB structures ( Fig. 3A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24028219" target="_blank">24028219</a>]<br></li>
<li>( A ) Model of structural similarity between Axe -- Txe and YefM -- YoeB protein complexes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24028219" target="_blank">24028219</a>]<br></li>
<li>The plasmid-encoded Axe -- Txe complex of Enterococcus faecium is homologous to YefM -- YoeB [ 13 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24028219" target="_blank">24028219</a>]<br></li>
<li>The antitoxins exhibit more sequence divergence than the toxins : YefM shows ~ 25 % identity overall with Axe [ 13 ] ; Txe and YoeB are ~ 50 % identical .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24028219" target="_blank">24028219</a>]<br></li>
				</div></ul><br>
			<label class="collapse_RP" for="_4"><font size="5px">Processes regulated by YefM</font></label>
				<input id="_4" type="checkbox">
				<div class="RP"><br><ul><li>35 , No. 1 325 -- 339 doi :10.1093 / nar/gkl1028 of YefM -- YoeB with paired DNA palindromes exerts transcriptional autorepression Barbara Kędzierska , Lu-Yun Lian and Finbarr Hayes * 1 Faculty of Life Sciences and Manchester Interdisciplinary Biocentre , The University of Manchester , 131 Princess Street , Manchester M1 7DN , UK and School of Biological Sciences , University of Liverpool , 1 Crown Street , Liverpool L69 7ZB , UK Received October 6 , 2006 ; Revised and Accepted November 13 , 2006 ABSTRACT INTRODUCTION Toxin -- antitoxin ( TA ) complexes function in programmed-cell-death or stress response mechanisms in bacteria .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17170003" target="_blank">17170003</a>]<br></li>
<li>Inhibition of translation by VapC mediates degradation of YefM antitoxin by Lon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19400780" target="_blank">19400780</a>]<br></li>
<li>Degradation of YefM , in turn , leads to activation of YoeB and mRNA-cleavage .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19400780" target="_blank">19400780</a>]<br></li>
<li>We have shown previously that TA systems are important for biofilm-formation using MqsR ( Ren et al. , 2004a ; González Barrios et al. , 2006 ; Zhang et al. , 2008 ) and the E. coli Δ5 strain , which lacks five TA systems ( listed as toxin/antitoxin : MazF/MazE , RelE/RelB , ChpB , YoeB/YefM and YafQ/DinJ ) ( Kim et al. , 2009 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20105222" target="_blank">20105222</a>]<br></li>
<li>Using a strain deficient in six major TA systems , Δ6 ( MazF/MazE , RelE/RelB , ChpB , YoeB/YefM , YafQ/DinJ , and MqsR/MqsA ) , we provide insights into extracellular stress and both the general stress response and the switch from planktonic growth to biofilm-formation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21516113" target="_blank">21516113</a>]<br></li>
<li>We used a genetic background devoid of the major E. coli TA pairs via the Δ5 strain2 , which lacks the MazF/MazE , RelE/RelB , ChpB , YoeB / YefM , and YafQ/DinJ TA systems ( Supplementary Results , Supplementary Table 1 ) and the Δ6 strain which also lacks MqsR/MqsA ( Δ5 mqsRA ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21516113" target="_blank">21516113</a>]<br></li>
<li>Many chromosomal TA systems have been characterized in Escherichia coli including ( listed as toxin/antitoxin ) MazF/MazE ( Aizenman et al. , 1996 ) , RelE/RelB ( Gotfredsen and Gerdes , 1998 ) , ChpB/ChpS ( Masuda et al. , 1993 ) , YoeB/YefM ( Cherny and Gazit , 2004 ) , YafQ / DinJ ( Motiejūnaitė et al. , 2007 ) and YhaV/PrlF ( Schmidt et al. , 2007 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20105222" target="_blank">20105222</a>]<br></li>
<li>Unlike most TApairs including RelE/RelB , YoeB/YefM and MazF / MazE , the MqsR toxin gene precedes the antitoxin gene in the bicistronic operon ( Shah et al. , 2006 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20105222" target="_blank">20105222</a>]<br></li>
				</div></ul><br>
			<label class="collapse_SIT" for="_5"><font size="5px">YefM binding site</font></label>
				<input id="_5" type="checkbox">
				<div class="SIT"><br><ul><li>Native YefM weakly bound the 99 bp oligonucleotide , producing a single-retarded complex in EMSA ( Figure 2A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17170003" target="_blank">17170003</a>]<br></li>
<li>YefM and YefM -- YoeB recognize DNA palindromes with common core sequences Three versions of YefM were puriﬁed to > 95 % homogen-eity and tested in EMSA with a 99 bp double-strande oligonucleotide substrate encompassing the yefM-yoeB promoter region : native YefM isolated following cleavage from a fusion of the C-terminus of the protein to the N-terminus of an intein tag ; YefM with a hexahistidine tag at its C-terminus ( YefM-His6 ) ; and YefM with a 21 amino acid extension , including a decahistidine tag , at its N-terminus ( His10-YefM ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17170003" target="_blank">17170003</a>]<br></li>
				<label class="SIT_collapse_similarity" for="sit_similarity_3">
( A ) Far UV CD spectrum of YefM alone ( 10 mM ) , and in the presence of 2 mM 99 bp oligonucleotides with the yefM-yoeB promoter -- operator region , the same region but with mutations in the S palindrome ( Figure 2D ) , and without the promoter -- operator sequences .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17170003" target="_blank">17170003</a>]<br></label>
<input id="sit_similarity_3", type="checkbox">
<div class="SIT_similar">
<li>( B ) Far UV CD spectrum of YefM -- YoeB -- His6 alone ( 5 mM ) , and in the presence of 99 bp oligonucleotides with the yefM-yoeB promoter -- operator region , the same region but with mutations in the S palindrome ( Figure 2D ) , and without the promoter -- operator sequences .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17170003" target="_blank">17170003</a>]<br></li></div>
<li>Preferential binding of YefM to the L repeat might be facilitated by the additional palindromic nucleotides that ﬂank the hexameric core sequence within this repeat , but that are absent from the S repeat .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17170003" target="_blank">17170003</a>]<br></li>
<li>YefM binds the long palindrome , followed sequentially by short palindrome recognition .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17170003" target="_blank">17170003</a>]<br></li>
<li>Intriguingly , the inverted-repeats bound by Phd include core 50-GTAC-30 motifs ( 33 ) , identical to the central tetranucleotides of the L and S repeats recognized by YefM ( Figure 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17170003" target="_blank">17170003</a>]<br></li>
<li>The primary region protected by YefM against DNase I attack includes a 50-TCATTGTACAATGA-30 palindrome ( L [ long ] repeat ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17170003" target="_blank">17170003</a>]<br></li>
<li>The 50-TGTACA-30 core of this inverted-repeat is also present in the secondary region of YefM protection ( S [ short ] repeat ) ( Figure 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17170003" target="_blank">17170003</a>]<br></li>
				</ul></div><br>
			<label class="collapse_TU" for="_6"><font size="5px">Transcription units regulated by YefM</font></label>
				<input id="_6" type="checkbox">
				<div class="TU"><br><ul><li>It was not feasible to use wild-type Txe as a control in these experiments as it is not neutralized by YefM and therefore remains an active endoribo-nuclease .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24028219" target="_blank">24028219</a>]<br></li>
<li>More infrequently , SymE family genes are found in operons encoding transcription factors of the YefM or MetJ/Arc ribbon -- helix -- helix type ( RHH ) superfamilies , which are also represented in toxin-antitoxin operons .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17462020" target="_blank">17462020</a>]<br></li>
<li>YefM and Phd are homologues , although Doc and Yoe are not ( 18,19 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17170003" target="_blank">17170003</a>]<br></li>
<li>The only pair of two-hybrid plasmids showing interactions for both Axe -- Txe and YefM -- YoeB was that in which the antitoxin gene was cloned in the pKNT25 vector and the toxin gene was located on pUT18C ( Figs 1 and S1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24028219" target="_blank">24028219</a>]<br></li>
<li>Hence , a fusion of the promoter with the lux operon was constructed and luminescence was measured in cultures of bacteria expressing genes coding for the YefM antitoxin and either YoeB or Txe-Asp83Tyr .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24028219" target="_blank">24028219</a>]<br></li>
<li>Disruption of the S repeat did not detectably perturb either unregulated expression from the resulting yefM ( S0 ) - lacZ fusion or repression of this fusion by YefM in vivo ( Figure 1C ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17170003" target="_blank">17170003</a>]<br></li>
<li>The yefM-yoeB operator DNA showed no signiﬁcant alterations in these characteristics in the presence YefM or YefM-YoeB-His6 ( data not shown ) , suggesting that the operator site does not undergo major structural transitions when bound by its cognate proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17170003" target="_blank">17170003</a>]<br></li>
<li>RESULTS YefM and YoeB are the transcriptional repressor and corepressor , respectively , of the yefM-yoeB genes During in vitro transcriptional analysis of the histidine biosynthesis ( his ) genes in E.coli K-12 , a cryptic transcript was detected from a promoter orientated divergently to the his operon promoter ( 41,42 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17170003" target="_blank">17170003</a>]<br></li>
<li>( C ) Autoregulation of yefM-yoeB expression by YefM and YefM-YoeB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17170003" target="_blank">17170003</a>]<br></li>
<li>However , the YefM homologue Axe does not repress the yefM-yoeB promoter in vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17170003" target="_blank">17170003</a>]<br></li>
				</div></ul></div><br>
	</div>
</body>
</html>