<!doctype html>
<style>
@charset "utf-8";
body {
	height: 100%;
	margin: 0;
	padding: 0;
}

.banner {
	position: fixed;
	top: 0;
	width: 100%;
	height: 65px;
	background: gray;
	line-height: 65px;
}
.spacer {
	height: 10px;
}

.main-content {
	padding: 5%;
}

.collapse_ACT {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-style: none solid none none;
	border-color: #B4045F;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_ACT + input{
	display: none;
}
.collapse_ACT + input + div{
	display: none;
}
.collapse_ACT + input:checked + div{
	display: block;
}
.ACT {
	text-align: justify;
	text-justify: inter-word;
}
.ACT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B4045F;
}
.ACT_collapse_similarity:hover{
	background: #e5d3dc;
}
.ACT_collapse_similarity + input {
	display: none;
}
.ACT_collapse_similarity + input + div{
	display: none;
}

.ACT_collapse_similarity + input:checked + div{
	display: block;
}
.ACT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.ACT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B4045F;
	padding-left: 5px;
}
.ACT_collapse_similarity:hover{
	background: #e5d3dc;
}
.ACT_collapse_similarity + input {
	display: none;
}
.ACT_collapse_similarity + input + div{
	display: none;
}
.ACT_collapse_similarity + input:checked + div{
	display: block;
}
.ACT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.ACT_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #B4045F;
}
.ACT_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_DOM {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-style: none solid none none;
	border-color: #B404AE;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_DOM + input{
	display: none;
}
.collapse_DOM + input + div{
	display: none;
}
.collapse_DOM + input:checked + div{
	display: block;
}
.DOM {
	text-align: justify;
	text-justify: inter-word;
}
.DOM_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B404AE;
	padding-left: 5px;
}
.DOM_collapse_similarity:hover{
	background: #e8d7e7;
}
.DOM_collapse_similarity + input {
	display: none;
}
.DOM_collapse_similarity + input + div{
	display: none;
}
.DOM_collapse_similarity + input:checked + div{
	display: block;
}
.DOM_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}.DOM_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #B404AE;
}
.DOM_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_EVO {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #7401DF;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_EVO + input{
	display: none;
}
.collapse_EVO + input + div{
	display: none;
}
.collapse_EVO + input:checked + div{
	display: block;
}
.EVO {
	text-align: justify;
	text-justify: inter-word;
}
.EVO_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #7401DF;
	padding-left: 5px;
}
.EVO_collapse_similarity:hover{
	background: #e2d7ed;
}
.EVO_collapse_similarity + input {
	display: none;
}
.EVO_collapse_similarity + input + div{
	display: none;
}
.EVO_collapse_similarity + input:checked + div{
	display: block;
}
.EVO_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.EVO_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #7401DF;
}
.EVO_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_RP {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #0174DF;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_RP + input{
	display: none;
}
.collapse_RP + input + div{
	display: none;
}
.collapse_RP + input:checked + div{
	display: block;
}
.RP {
	text-align: justify;
	text-justify: inter-word;
}
.RP_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #0174DF;
	padding-left: 5px;
}
.RP_collapse_similarity:hover{
	background: #c9d7e2;
}
.RP_collapse_similarity + input {
	display: none;
}
.RP_collapse_similarity + input + div{
	display: none;
}
.RP_collapse_similarity + input:checked + div{
	display: block;
}
.RP_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.RP_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #0174DF;
}
.RP_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_SIT {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #0489B1;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_SIT + input{
	display: none;
}
.collapse_SIT + input + div{
	display: none;
}
.collapse_SIT + input:checked + div{
	display: block;
}
.SIT {
	text-align: justify;
	text-justify: inter-word;
}
.SIT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #0489B1;
	padding-left: 5px;
}
.SIT_collapse_similarity:hover{
	background: #d0dfe2;
}
.SIT_collapse_similarity + input {
	display: none;
}
.SIT_collapse_similarity + input + div{
	display: none;
}
.SIT_collapse_similarity + input:checked + div{
	display: block;
}
.SIT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.SIT_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #0489B1;
}
.SIT_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_TU {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #04B486;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_TU + input{
	display: none;
}
.collapse_TU + input + div{
	display: none;
}
.collapse_TU + input:checked + div{
	display: block;
}
.TU {
	text-align: justify;
	text-justify: inter-word;
}
.TU_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #04B486;
	padding-left: 5px;
}
.TU_collapse_similarity:hover{
	background: #d0e2de;
}
.TU_collapse_similarity + input {
	display: none;
}
.TU_collapse_similarity + input + div{
	display: none;
}
.TU_collapse_similarity + input:checked + div{
	display: block;
}
.TU_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.TU_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #04B486;
}
.TU_similar > a {
	text-decoration: none;
	color: gray;
}

ul { 
	list-style-type: none;
	padding: 0;
	margin: 0;
}
ul li > a {
	text-decoration: none;
	color: gray;
}
ul li:hover, ul li:focus {
	background: #F0F0F0;
}

ul li:first-child span {
  border-left:none;  
}

</style>
<html>
<head>
<meta charset="utf-8">
<title>TF properties - ZntR</title>
<meta name="theme-color" content="#ffffff">
</head>
<body>
	<div class="banner">
		<center><font color="white" size="16px" face="arial">ZntR</font></center>
	</div>
	<div align="center" class="main-content">
		<div class="spacer">
	</div><br>			<label class="collapse_ACT" for="_1"><font size="5px">ZntR transcriptional activity</font></label>
				<input id="_1" type="checkbox">
				<div class="ACT"><br><ul><li>Location and functional classification of 32-regulon members A. Location Periplasm ( 2 ) Regulon members DsbC YceI CreC CutE FtsH FxsA GntYa HflX HflK HflC HtpX LipBa LspA MenA PgpA PhoQ YbeXa YbeZa YceJa YcjFa ClpB ClpP ClpX CreB DnaJ DnaK FkpB GapA GlnS GroELS GrpE HepA Hfq HolC Hsp33 HslU HslV HtpG IbpA IbpB IleS IspH Lon MiaA Mlc MutL MutM NarP NusB PhoP PrlC PyrF RecA RecJ RdgB RpoD rrnB RrmJ SdaA TopA TyrR ValS XerD YciH YfjN YrfH ZntR CreA FolP RibE ThiL YadF YafD YafE YbeD YbeY YceP YciH YciM YciS YcjX YdhQ YeaD YfjN YgaD YgbF YggW YhdN YhiQ YnfK Inner membrane ( 18 ) Cytoplasm ( 48 ) Unknown ( 25 ) B. Functional category Regulon members Metabolism CutE FolP GapA IspH LipB LspA MenA PgpA PyrF RibE SdaA ThiL YadF YafE YceJ YggW Chaperone/folding catalysts ClpB DnaJ DnaK DsbC FkpB GroELS GrpE Hsp33 HtpG IbpA IbpB Protein degradation ClpP ClpX FtsH HflX HflK HflC HslU HslV HtpX Lon PrlC DNA modification HolC MutL MutM RecA RecJ RdgB TopA XerD RNA state GlnS Hfq IleS MiaA rrnB RrmJ ValS YciH YfjN YrfH Transcription regulators CreB CreC HepA Mlc NarP NusB PhoP PhoQ RpoD TyrR ZntR Transporter YbeX YbeZ Miscellaneous FxsA GntY YciH YfjN YhiQ Unknown function CreA YafD YbeD YbeY YceI YceP YciM YciS YcjF YcjX YdhQ YeaD YgaD YgbF YhdN YnfK Only regulon members encoded on the chromosome and with a confirmed 32-dependent promoter are considered .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16818608" target="_blank">16818608</a>]<br></li>
<li>All of the ZntR derivatives were active at PzntA-1 in the absence or presence of Table 1 Primers used for mutagenesis written in a 50 -- 30 orientation Primer name Sequence CAGCAGATGATGGAGGCTGAGGTGCGTACTGAAGG CCAGCGATTGAAATTTATCCGGGCCGCCAGACAACTAGG GCATCGATCCTGAGGCCCATACCTGTCAGGAGTCAAAAGGC GCATCGATCCTGAACACGCTACATGTCAGGAGTCAAAAGGC CGATCCTGAACACCATACCGCTCAGGAGTCAAAAGGC CGATCCTGAACACCATACCTCACAGGAGTCAAAAGGC CGCCTTAACGATGCCGCATGTGGGACTGCTCATAGC CGCCTTAACGATGCCTCATGTGGGACTGCTCATAGC CGCCTTAACGATGCCTGTGCAGGGACTGCTCATAGC H29A H53A H76A H77A C79A C79S C114A C114S C115A CGCCTTAACGATGCCTGTTCAGGGACTGCTCATAGC GGGACTGCTCATTAGCAGTGTTTATGCATCGATTCTTGAAGC GGGACTGCTCATTAGCAGTGTTTATTCATCGATTCTTGAAGC GGCGTTAAGAGTGGCGCTTGATTTTTTGCACTGGCAGG GGCGTTAAGAGTGGTTCATGATTTTTTGCACTGGCAGG GGCGTTAAGAGTGGTCATTGATTTTTTGCACTGGCAGG GGCGTTAAGAGTGGTCTCTAATTTTTTGCACTGGCAGG GGCGTTAAGAGTGGTTACTAGTTTTTTGCACTGGCAGG GGCGTTAAGAGTGGTGGTTGATTTTTTGCACTGGCAGG C115S C124A C124S C141A C141S C141H C141L C141Y C141G GGCGTTAAGAGTGGTGATTGATTTTTTGCACTGGCAGG GCCTGTTGTGGGACTGCTGCTAGCCAGTGTTTATTGTTCG CGCCAGGGTTTTCCCAGTCACGAC AGCGGATAACAATTTCACACAGGA C141D H119A M13-Forward M13-Reverse Only one of the primer pairs is shown as the second primer was an exact complement .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12445820" target="_blank">12445820</a>]<br></li>
<li>std , mo-lecular mass standards ; lane 1 , uninduced cells ; lane 2 , induced cells ; lane 3 , freeze-thaw supernatant ; lane 4 , 45 % ( NH ) SO precipitate ; 4 2 4 lane 5 , 15 mg of purified ZntR ; lane 6 , 30 mg of purified ZntR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10608803" target="_blank">10608803</a>]<br></li>
<li>[ 52 ] Singh , V.K. , Xiong , A. , Usgaard , T.R. , Chakrabarti , S. , Deora , R. , Misra , T.K. and Jayaswal , R.K. ( 1999 ) ZntR is an autoregulatory protein and negatively regulates the chromosomal zinc resistance op-eron znt of Staphylococcus aureus .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12829264" target="_blank">12829264</a>]<br></li>
<li>Details are shown in Supplementary Table S2 , but briefly this consisted of 184 genes that could be divided into the following categories : repressed by Fur , 63 ; activated by Fur , 15 ; repressed by CpxR , 21 ; activated by CpxR , 37 ; repressed by SoxS , 1 ; activated by SoxS , 23 ; repressed by OxyR , 2 ; activated by OxyR , 16 ; activated by CusR , 6 ; activated by BaeR , 9 ; repressed by Zur , 4 ; repressed by ArsR , 3 ; activated by ZraR , 3 ; activated by ZntR , 1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22016571" target="_blank">22016571</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_6">
Statistical modelling of the transcriptomic data generated from the Zn shift focused on the role of ten key regulators ; ArsR , BaeR , CpxR , CusR , Fur , OxyR , SoxS , ZntR , ZraR and Zur .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22016571" target="_blank">22016571</a>]<br></label>
<input id="act_similarity_6", type="checkbox">
<div class="ACT_similar">
<li>The analysis of the transcriptomic data generated from the Zn shift focused on the role of ten key regulators : ArsR , BaeR , CpxR , CusR , Fur , OxyR , SoxS , ZntR , ZraR and Zur .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22016571" target="_blank">22016571</a>]<br></li></div>
<li>In vitro transcription studies ( Outten and O'Halloran , 2001 ) revealed that halfmaximal Zur repression occurs at ﬁvefold lower Zn2 + ion concentrations than the half-maximal value of ZntR activation so that gene transcription occurs for either one of Department of Biochemistry and Molecular Biophysics , Washington University School of Medicine , St Louis , MO , 63110 , USA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16956381" target="_blank">16956381</a>]<br></li>
<li>Note that the value axes are of different ranges ZntR derivatives suggests that the conserved cysteine residues play a central role in the mechanism of induction and are likely to act as ligands for metal-ions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12445820" target="_blank">12445820</a>]<br></li>
<li>It is unclear why the C141A or C141H mutations alter the function of ZntR to such a degre although we would hypothesise that the presence of a thiol group at this position may alter the access of metalions to other binding sites which drive the torsional mechanism of induction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12445820" target="_blank">12445820</a>]<br></li>
<li>Mobility shift assays demonstrate that puriﬁed ZntR protein bound speciﬁcally to PzntA , conﬁrming that the pro-duct of zntR is the cognate regulatory protein for zntA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10048032" target="_blank">10048032</a>]<br></li>
<li>In contrast to the CueR metal-binding site , the coordination environment of the metal-binding loop in ZntR is optimal for binding of divalent metals .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12958362" target="_blank">12958362</a>]<br></li>
<li>MlrACTD contains a RxxD motifs , i.e. a signature involved in c-di-GMP binding in several types of c-di-GMP effector-proteins ( Schirmer and Jenal , 2009 ) , but none of the cysteines , which in metal-responding MerR-like proteins , e.g. MerR , ZntR and CueR , are involved in ligand binding to the CTD ( Helmann et al. , 1990 ; Brown et al. , 2003 ; Changela et al. , 2003 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22783906" target="_blank">22783906</a>]<br></li>
<li>As argued for E. coli ZntR and Zur , this makes it unlikely that these metal sensor proteins function under thermodynamic control , where the metal binding a ⁄ nity of the sensor protein is appropriately tuned to 1 / [ metal ] free such that a successful stress response can be mounted [ 4 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12829264" target="_blank">12829264</a>]<br></li>
<li>Two species of cytosolic transcription factor , ZntR and Zur , are involved in transcription regulation of the zinc response genes ( 3 , 24 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16159766" target="_blank">16159766</a>]<br></li>
<li>Nevertheless , these preliminary data suggest that binding of ZntR to the promoter does not actively repress transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10048032" target="_blank">10048032</a>]<br></li>
<li>In the putative promoter ( 200 bp upstream of the pitA start codon ) , no DNA-binding sites for the following transcription factors are evident : MerR , CueR , ZntR ( all MerR family ) , Zur , NikR , CusR , ZraR , CpxR and ModE .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19054109" target="_blank">19054109</a>]<br></li>
<li>The Bradford assay was found to overestimate the ZntR concentration by 2.65 when using IgG as the standard .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10608803" target="_blank">10608803</a>]<br></li>
<li>The structures of Cu - , Ag - , and Au - bound forms of CueR were determined to 2.2 , 2.1 , and 2.5 Å resolution , respectively , whereas the structure of an N-terminally truncated fragment of ZntR bound to Zn2 was solved to 1.9 Å resolution ( 13 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12958362" target="_blank">12958362</a>]<br></li>
<li>In E. coli , three regulators -- ZntR , ZraS/R , and Zur -- are known to respond to Zn2 + .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11831459" target="_blank">11831459</a>]<br></li>
<li>ZntR upregulates synthesis of ZntA , a Zn2 + - speciﬁc export protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16956381" target="_blank">16956381</a>]<br></li>
<li>However , it is possible that it acts as a sensor of extracellular Zn2 while the ZntR system responds to intracellular Zn2 ( 99 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11528004" target="_blank">11528004</a>]<br></li>
<li>ZntR responds to Zn2 , Cd2 and Pb2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11243806" target="_blank">11243806</a>]<br></li>
<li>In a wide range of other organisms P-type ATPases have one of which is the Fet3 ferroxidase necessary for iron Abbreviations : mRNA , messenger RNA ; copA , ybaR , gene for copper ef̄ ux pump , copR , ybbI , regulatory gene for copA ; MerR mercury responsive transcriptional-activator ; lacZ , b-galactosidase gene ; ZntR , zink responsive transcriptional-activator ; PCR , polymerase chain reaction * Corresponding author .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11167016" target="_blank">11167016</a>]<br></li>
<li>This mutability of ZntR is in sharp contrast to that seen during mutational analysis of MerR [ 31,36 -- 38 ] which appears to be rather more refractory to functional alteration [ 30 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12445820" target="_blank">12445820</a>]<br></li>
<li>The transcriptional-regulators under 32 control mostly sense extracellular conditions : Mlc senses glucose , Pho P/Q senses low Mg + + and is induced in acid stress , ZntR responds to excess Zn + + , and NarP induction requires nitrite/nitrate under anaerobic conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16818608" target="_blank">16818608</a>]<br></li>
<li>The analogy is not perfect because ZntR acts as a transcriptional-activator of the zinc exporter ZntA in the presence of excess zinc , whereas rcnA expression is derepressed by inactivation of RcnR in the presence of excess nickel .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16956381" target="_blank">16956381</a>]<br></li>
<li>Multiple alignment of MerR [ 8 ] , ZntR [ 10 ] , CadR [ 27 ] , CueR [ 25,26 ] , and PbrR [ 28 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12445820" target="_blank">12445820</a>]<br></li>
<li>Lanes 1 and 8 are DNA-only controls , lanes 2 -- 6 contain puriﬁed ZntR at 0 - , 5 - , 10 - , 20 - and 100-fold dilutions of puriﬁed ZntR ; lane 7 contains puriﬁed MerR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10048032" target="_blank">10048032</a>]<br></li>
<li>This work demonstrates that these residues are important for ZntR function and structure , as they are in MerR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12445820" target="_blank">12445820</a>]<br></li>
<li>These experiments indicate significant similarities between ZntR and MerR function in vivo and lead us to address the question of how ZntR activates transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10608803" target="_blank">10608803</a>]<br></li>
<li>Based on the sequence similarities among CueR , ZntR , and MerR ( Fig. 6B ) , it is likely that CueR detects cytoplasmic copper stress and activates transcription in response to increasing copper concentrations .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10915804" target="_blank">10915804</a>]<br></li>
<li>The Cd - and Pb-dependent trans-criptional up-regulation of zntA is also mediated by ZntR ( Binet & Poole , 2000 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12480884" target="_blank">12480884</a>]<br></li>
<li>The induction of zntA transcription requires the ZntR transcription activator , while znuC transcription is repressed by the Zur transcription repressor ( 3 , 25 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16159766" target="_blank">16159766</a>]<br></li>
<li>Members of the MerR/ZntR family or of the + + + ArsR/SmtB family regulate Pb2 , Cd2 , and Zn2 extrusion The regulator of zntA in E. coli , ZntR , induces zntA 2 + 2 + transcription at 19 µM Cd or 100 µM Zn ( Noll 2 + & Lutsenko 2000 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11831459" target="_blank">11831459</a>]<br></li>
<li>The expression of zntA is regulated by ZntR ( Brocklehurst et al. , 1999 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11243806" target="_blank">11243806</a>]<br></li>
<li>ZntR , a member of the ArsR family , regulates the expression of zntA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11831459" target="_blank">11831459</a>]<br></li>
<li>The footprinting data for ZntR are summarized in Fig. 5 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10608803" target="_blank">10608803</a>]<br></li>
<li>The in vitro activation and the sequence of metal-dependent changes in the ZntR footprinting data show many similarities with those obtained for MerR ( 22 , 24 , 25 , 37 ) and SoxR , another MerR homologue in E. coli that responds to oxidative-stress ( 40 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10608803" target="_blank">10608803</a>]<br></li>
<li>ZntR responds to Cd as well as to Zn ( Brocklehurst II II et al. , 1999 ) , and CueR shows 37 % identity to ZntR across the length of the protein sequence ( Fig. 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11136469" target="_blank">11136469</a>]<br></li>
<li>The response to Hg ( II ) may result from cross-reactivity of ZntR with Hg ( II ) , as there are cysteine residues in ZntR ( Fig. 4 ) at positions corresponding to those responsible for binding Hg ( II ) in MerR ( Cys-82 , Cys-117 and Cys-126 ; Shewchuk et al. , 1989 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10048032" target="_blank">10048032</a>]<br></li>
<li>Apo - and Zn-ZntR would follow this same mechanism , although the extent of bending and underwinding remains to be seen in this system .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10608803" target="_blank">10608803</a>]<br></li>
<li>This result is of particular interest given that H53 is conserved in ZntR , CadR [ 27 ] , and PbrR [ 28 ] yet only Zn ( II ) induction is affected by its replacement .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12445820" target="_blank">12445820</a>]<br></li>
<li>The alterations ZntR ( H76A ) and ZntR ( H77A ) appeared to diminish ZntR-mediated Zn ( II ) induction to some extent but each shows a similar proﬁle to that for ZntR w.t. , this is surprising given the conservation seen between ZntR [ 10 ] , MerR [ 8 ] and CueR [ 25,26 ] in this region ( Fig. 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12445820" target="_blank">12445820</a>]<br></li>
<li>ZntR ( C141A ) showed dramatic differences in its Zn ( II ) - induction proﬁle with respect to w.t. ZntR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12445820" target="_blank">12445820</a>]<br></li>
<li>Most interestingly , replacement of C141 with alanine increased the maximal induction of ZntR in response to Zn ( II ) ( 2.5-fold ) but surprisingly changed the sensitivity of the regulator/promoter complex , with maximal induction now occurring at 0.1 mM Zn ( II ) whereas ZntR w.t. ( and most of the active derivatives ) showed maximal induction in the 0.9 -- 1.2 mM range .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12445820" target="_blank">12445820</a>]<br></li>
<li>The critical feature of this promoter is that ZntR w.t. induces transcription in the absence of Zn ( II ) , i.e. , interaction of ZntR w.t. with PzntA-1 alone can be measured [ 10 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12445820" target="_blank">12445820</a>]<br></li>
<li>Data from the promoter deletion derivatives PzntA-2 and PzntA-2-IR also demonstrate that , in the absence of Zn ( II ) , binding of ZntR to the promoter appears to result in approximately fourfold induction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10048032" target="_blank">10048032</a>]<br></li>
<li>Whilst exhibiting little alteration in its response to Zn ( II ) , ZntR ( H76A ) induces PzntA to a greater extent in the presence of Cd ( II ) and Pb ( II ) ( Figs. 4B -- D ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12445820" target="_blank">12445820</a>]<br></li>
<li>Transcriptional activity of ZntR derivatives H29A , H53A , H76A , H77A , and H119A at PzntA The transcriptional proﬁles of ZntR ( H29A ) , ZntR ( H53A ) , and ZntR ( H119A ) ( Fig. 1 ) show that no signiﬁcant induction is mediated by these ZntR derivatives in response to Zn ( II ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12445820" target="_blank">12445820</a>]<br></li>
<li>Transcriptional response of PzntA to ZntR and its derivatives .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12445820" target="_blank">12445820</a>]<br></li>
<li>When ZntR is titrated with excess TPEN present ( 10 mM ) but no added Zn ( II ) , zntA transcript levels increase very slightly up to 20 nM ZntR , and by 500 nM ZntR , they drop off to a value that is ; 20 % lower than RNAP alone ( lanes 2 -- 8 in panel A and graph in panel B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10608803" target="_blank">10608803</a>]<br></li>
<li>Similarly , with constant [ ZntR ] and increasing [ Zn ( II ) ] , halfmaximal transcriptional activation is achieved at 5.9 6 0.2 mM Zn ( II ) in the presence of 10 mM TPEN ( lanes 17 -- 24 in panel A and graph in panel C ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10608803" target="_blank">10608803</a>]<br></li>
<li>Without TPEN in the transcription buffer , ZntR did not require added Zn ( II ) for activation ( data not shown ) , suggesting that the buffer still contains sufficient levels of Zn ( II ) to activate ZntR despite chelexing overnight .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10608803" target="_blank">10608803</a>]<br></li>
<li>The fact that TPEN is required to see the lowest levels of transcript in the absence of Zn ( II ) suggests that ZntR is very sensitive to free Zn ( II ) concentrations in the buffer .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10608803" target="_blank">10608803</a>]<br></li>
<li>A , apo-ZntR ; B , Zn-ZntR ; C , Zn-ZntR-RNAP ; and D , Zn-ZntR-RNAP with UpA , ATP , GTP , and UTP added to the reaction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10608803" target="_blank">10608803</a>]<br></li>
<li>A transcriptionally active , heparin-resistant open RNAP complex at PzntA is only formed when ZntR , Zn ( II ) , and RNAP are present in the reaction mixture .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10608803" target="_blank">10608803</a>]<br></li>
<li>Zn-ZntR was also prepared under anaerobic conditions in a glove box .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10608803" target="_blank">10608803</a>]<br></li>
<li>While MerR binds 1 mol of Hg per MerR dimer ( 22 , 63 ) , ZntR can bind 1 -- 2 mol of Zn ( II ) per monomer depending on the buffer conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10608803" target="_blank">10608803</a>]<br></li>
<li>It is tempting to suggest that since alterations in these residues appear to affect the substrate range of ZntR that they act as ligands for one of the two Zn ( II ) - ions predicted to-bind each ZntR monomer , perhaps acting in a structural role rather than a primary driver of DNA distortion .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12445820" target="_blank">12445820</a>]<br></li>
<li>Our result , that the expression of zraP is under control of the HydH/G system , seems to be at ® rst in contrast with the report that this gene is activated by a MerR-like transcriptional-activator ( PMTR ) , which shares signi ® cant homology with ZntR ( Noll et al. , 1998 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11243806" target="_blank">11243806</a>]<br></li>
<li>Protein/DNA footprinting and in vivo transcriptional assays reveal that , like MerR , ZntR functions primarily as a transcriptional-activator , but it only weakly , if at all , represses expression of the zntA gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10608803" target="_blank">10608803</a>]<br></li>
<li>The mechanism of target gene activation by oxidized SoxR involves promoter distortion , being similar to that by MerR in response to Hg2 + ( Ansari et al. , 1995 ; Hidalgo and Demple , 1997 ) and to that by another MerR family member ZntR in response to Zn2 + ( Outten et al. , 1999 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12773378" target="_blank">12773378</a>]<br></li>
<li>A very recent paper demonstrated that ZntR activates the transcription of zntA by a DNA distortion mechanism similar to MerR regulation of Hg ( II ) resistance genes [ 11 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10802061" target="_blank">10802061</a>]<br></li>
<li>ZntR has been demonstrated to-bind DNA speci cents cally at the zntA promoter region [ 10 ] and activate transcription by a DNA distortion mechanism [ 11 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10802061" target="_blank">10802061</a>]<br></li>
<li>The addition of Zn ( II ) to ZntR converts it to a transcriptional-activator protein that introduces changes in the DNA conformation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10608803" target="_blank">10608803</a>]<br></li>
<li>The transcription and footprinting data presented here provide strong evidence that the stress-responsive transcriptional activation of PzntA by ZntR involves a MerR-like DNA distortion mechanism .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10608803" target="_blank">10608803</a>]<br></li>
<li>DNA Distortion Mechanism for Transcriptional Activation by ZntR , a Zn ( II ) - responsive MerR Homologue in Escherichia coli * ( Received for publication , August 7 , 1999 , and in revised form , November 3 , 1999 ) Caryn E. Outten ‡ , F. Wayne Outten § , and Thomas V. O'Halloran ‡ § ¶ From the ‡ Department of Chemistry and the § Department of Biochemistry , Molecular Biology , and Cell Biology , Northwestern University , Evanston , Illinois 60208 MerR-like DNA distortion mechanisms have been proposed for a variety of stress-responsive transcription factors .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10608803" target="_blank">10608803</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_69">
DNA distortion mechanism for transcriptional activation by ZntR , a Zn ( II ) - responsive MerR homologue in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12144778" target="_blank">12144778</a>]<br></label>
<input id="act_similarity_69", type="checkbox">
<div class="ACT_similar">
<li>1999 DNA distortion mechanism for transcriptional activation by ZntR , a Zn ( II ) - responsive MerR homologue in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11831459" target="_blank">11831459</a>]<br></li>
<li>[ 16 ] C.E. Outten , F.W. Outten , T.V. O Halloran , DNA distortion mechanism for transcriptional activation by ZntR , a Zn ( II ) ¬ responsive MerR homologue in Escherichia coli , J. Biol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12445820" target="_blank">12445820</a>]<br></li>
<li>Outten , C.E. , Outten , F.W. , and O'Hallaran , T.V. ( 1999 ) DNA distortion mechanism for transcriptional activation by ZntR , a Zn ( II ) - responsive MerR homologue in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11489123" target="_blank">11489123</a>]<br></li>
<li>[ 18 ] Outten , C.E. , Outten , F.W. and O'Halloran , T.V. ( 1999 ) DNA distortion mechanism for transcriptional activation by ZntR , a Zn ( II ) - responsive MerR homologue in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12829264" target="_blank">12829264</a>]<br></li>
<li>Outten , C.E. , Outten , F.W. , and O'Halloran , T.V. ( 1999 ) DNA distortion mechanism for transcriptional activation by ZntR , a Zn ( II ) - responsive MerR homologue in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16956381" target="_blank">16956381</a>]<br></li>
<li>Outten , C.E. , Outten , F.W. and O'Halloran , T.V. ( 1999 ) DNA distortion mechanism for transcriptional activation by ZntR , a Zn ( II ) - responsive MerR homologue in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12773378" target="_blank">12773378</a>]<br></li></div>
				<label class="ACT_collapse_similarity" for="act_similarity_76">
[ 10 ] K.R. Brocklehurst , J.L. Hobman , B. Lawley , L. Blank , S.J. Marshall , N.L. Brown , A.P. Morby , ZntR is a Zn ( II ) - responsive MerR-like transcriptional-regulator of zntA in Escherichia coli , Mol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12445820" target="_blank">12445820</a>]<br></label>
<input id="act_similarity_76", type="checkbox">
<div class="ACT_similar">
<li>Brocklehurst KR , Hobman JL , Lawley B , Blank L , Marshall SJ , Brown NL & Morby AP ( 1999 ) ZntR is a Zn ( II ) - responsive MerR-like transcriptional-regulator of zntA in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19054109" target="_blank">19054109</a>]<br></li>
<li>Brocklehurst , K.R. , Hobman , J.L. , Lawley , B. , Blank , L. , Marshall , S.J. , Brown , N.L. , and Morby , A.P. ( 1999 ) ZntR is a Zn ( II ) - responsive MerR-like transcriptional-regulator of zntA in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11136469" target="_blank">11136469</a>]<br></li>
<li>Brocklehurst , K. , Hobman , J. , Lawley , B. , Blank , L. , Marshall , S. , Brown , N. , and Morby , A. ( 1999 ) ZntR is a Zn ( II ) - responsive MerR-like transcriptional-regulator of zntA in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16956381" target="_blank">16956381</a>]<br></li></div>
<li>ZntR is a MerR-like regulator that binds as a repressor to the zntA 2 + 2 + promoter ; in the presence of Cd or Zn , ZntR is converted into a transcriptional-activator that changes the conformation of the promoter region and makes it a better substrate for the RNA polymerase ( Outten et al. 1999 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11831459" target="_blank">11831459</a>]<br></li>
<li>zntA expression is regulated by ZntR , a transcriptional-regulator belonging to the MerR family , which activates zntA expression in the presence of Zn ( II ) [ 10 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10802061" target="_blank">10802061</a>]<br></li>
				</div></ul><br>
			<label class="collapse_DOM" for="_2"><font size="5px">ZntR protein domains</font></label>
				<input id="_2" type="checkbox">
				<div class="DOM"><br><ul><li>DAR carried out the analysis of putative Zur and ZntR binding sites and B12 riboswitches , analysis of putative back-up enzyme paralogs , and assisted in drafting the paper .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19822009" target="_blank">19822009</a>]<br></li>
<li>Initial experimentation demonstrated that replacement of C141 in ZntR with either an alanine or serine residue had very different effects on function .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12445820" target="_blank">12445820</a>]<br></li>
<li>For SoxR , a MerR homolog that uses an Fe-S cluster to sense oxidative stress ( 26 ) , the alignment suggests that the 2Fe-2S cluster will bind in a loop resembling the ZntR binuclear Zn2 site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12958362" target="_blank">12958362</a>]<br></li>
<li>In ZntR , both metal-binding domains are well ordered with two Zn2 ions bound at each site through Cys114 , Cys115 , His119 , Cys124 , and Cys79 ( Fig. 3A ) , which is supported by biochemical data showing a diminished response to Zn2 upon mutation of any of these resi-dues ( 22 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12958362" target="_blank">12958362</a>]<br></li>
<li>The model of ZntR ( Fig. 6A ) predicts that H29 lies close to the DNA-binding H-T-H region of ZntR and is distant from a proposed binding site at the dimer-interface although our data show that this residue has a signiﬁcant effect on the selectivity and dynamic range of ZntR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12445820" target="_blank">12445820</a>]<br></li>
<li>In contrast to the CueR metal-binding site , the coordination environment of the metal-binding loop in ZntR is optimal for binding of divalent metals .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12958362" target="_blank">12958362</a>]<br></li>
<li>The overall structure of the truncated ZntR dimer is similar to that of CueR and consists of one helix-turn-helix motif of the DNA-binding domain , the dimerization helix , and an intact metal-binding domain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12958362" target="_blank">12958362</a>]<br></li>
<li>A residue at the N-terminus of the dimerization helix extends across the dimer-interface to contact either the metal-binding loop ( Ser77 in CueR ) or directly to the metal ion ( Cys79 in ZntR ) and plays a unique role in discriminating between 1 and 2 ions ( Fig. 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12958362" target="_blank">12958362</a>]<br></li>
<li>As argued for E. coli ZntR and Zur , this makes it unlikely that these metal sensor proteins function under thermodynamic control , where the metal-binding a ⁄ nity of the sensor protein is appropriately tuned to 1 / [ metal ] free such that a successful stress response can be mounted [ 4 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12829264" target="_blank">12829264</a>]<br></li>
<li>Identi cents cation of the metal-binding sites on ZntR may lead to understanding the basis of metal recognition and signal transduction and perhaps engineering of the protein to modify metal speci cents city .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10802061" target="_blank">10802061</a>]<br></li>
<li>A model for the interaction of Zn-ZntR with the wrapped RNAPzDNA open complex is given in Fig. 8 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10608803" target="_blank">10608803</a>]<br></li>
<li>The standard error ( vertical bars ) in each case is derived from a triplicate data set Transcriptional activity of ZntR derivatives C79A/S , C114A/S , C115A/S , and C124A/S at PzntA In contrast to the histidine modiﬁed ZntR derivatives , replacement of the ﬁrst four cysteine residues resulted in a complete loss of activity with Zn ( II ) as the inducer whether replaced with alanine or serine ( Fig. 2 ) and ZntR modiﬁed at positions C79 , C114 , and C124 also shows no activity with Cd ( II ) or Pb ( II ) ( Figs. 4A -- C ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12445820" target="_blank">12445820</a>]<br></li>
<li>It is unsurprising , given the chemical similarity of Zn ( II ) , Cd ( II ) , and Pb ( II ) , that simplistic interpretations of the structure -- function relationships within ZntR are not easily deﬁned , although the three-dimensional structure of BmrR [ 29 ] has allowed us to create a model of the ﬁrst 95 amino acids of ZntR ( Fig. 6A ) which positions three of the histidine ( H53 , H76 , and H77 ) and one of the cysteine residues ( C79 ) in regions expected to interact with metal-ions , e.g. , at the proposed dimer-interface ( see Fig. 6B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12445820" target="_blank">12445820</a>]<br></li>
<li>A coiledcoil structure ( Fig. 6B , shown in magenta ) is present in BmrR [ 29 ] and a similar structure is thought to link the two domains of ZntR ( and other MerR-like regulators ) [ 30 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12445820" target="_blank">12445820</a>]<br></li>
<li>Furthermore , like Hg-MerR , Zn-ZntR greatly enhances RNAP binding and open complex formation at the target promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10608803" target="_blank">10608803</a>]<br></li>
<li>There are three cysteine residues in ZntR at positions corresponding to those responsible for binding Hg ( II ) in MerR [ 10 ] and which may be responsible for the cross-induction of the zntA promoter by Hg ( II ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10802061" target="_blank">10802061</a>]<br></li>
<li>The region of merR corresponding to the N-terminal region of MerR was ampliﬁed by PCR using primers MZ4 and M13 reverse , and the region of zntR corresponding to the C-terminal of ZntR was ampliﬁed using pri-mers MZ3 and M13 forward .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10048032" target="_blank">10048032</a>]<br></li>
<li>A hybrid regulatory protein containing the N-terminal region of MerR and the C-terminal region of ZntR showed an altered speciﬁcity to that of Pzmer/MerR , responding to Zn ( II ) and , to a lesser extent , to Hg ( II ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10048032" target="_blank">10048032</a>]<br></li>
<li>This result demonstrates that the ZntR C-terminal region is capable of recognizing Zn ( II ) and participating in a MerR-like activation mechanism when attached to a heterologous DNA-binding domain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10048032" target="_blank">10048032</a>]<br></li>
<li>CueR consists of binding region and a C-terminal metal-binding region , that are also conserved in both ZntR and MerR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10915804" target="_blank">10915804</a>]<br></li>
<li>C-terminal region of ZntR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10048032" target="_blank">10048032</a>]<br></li>
<li>Hybrid proteins with MerR ( N-terminal ) and ZntR ( C-terminal ) allowed the mer promoter to be activated by zinc ( Brocklehurst et al. , 1999 ) , indicating that these two proteins operate by similar mechanisms and confirming that the discriminatory metal recognition is in the C-terminal half of the protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11136469" target="_blank">11136469</a>]<br></li>
<li>Two of the three cysteines in MerR ( Cys-117 and Cys-126 ) that are involved in mercury binding ( Helmann et al. , 1990 ) are conserved between MerR , ZntR and CueR , and the discriminatory residues must therefore lie elsewhere in the proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11136469" target="_blank">11136469</a>]<br></li>
<li>These three cysteine residues are conserved with respect to other MerR-family members ( Fig. 5 ) , and correspond to residues C79 , C114 , and C124 , respectively , in ZntR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12445820" target="_blank">12445820</a>]<br></li>
<li>Interestingly these cysteine residues are conserved in the zinc responsive protein ZntR from E. coli , and the two C-terminal cysteines are even conserved in CopR ( Fig. 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11167016" target="_blank">11167016</a>]<br></li>
<li>Like CopR these proteins had a spacing of seven amino acids between the two conserved C-terminal cysteine residues , whereas MerR proteins and ZntR have a spacing of eight or nine amino acid residues , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11167016" target="_blank">11167016</a>]<br></li>
<li>CueR does , however , lack a Cys residue that is conserved at approximately position 80 in the ZntR and MerR sequence ( Fig. 6B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10915804" target="_blank">10915804</a>]<br></li>
<li>Furthermore , the spacing between two other highly conserved Cys residues is shorter in CueR than in either ZntR or MerR ( Fig. 6B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10915804" target="_blank">10915804</a>]<br></li>
<li>The other essential cysteine for metal recognition in MerR ( Cys-82 ) is found in ZntR , but not in CueR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11136469" target="_blank">11136469</a>]<br></li>
<li>Transcriptional activity of C141 ZntR derivatives at PzntA The C-terminal residue of ZntR ( position 141 ) is a cysteine and shows no sequence conservation with other MerR-family members except CueR ( Fig. 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12445820" target="_blank">12445820</a>]<br></li>
<li>Furthermore , the ZntR structure reveals how the prototypical metalloregulatory protein in this family , MerR ( 27 ) , 2 could bind Hg using a trigonal metal-thiolate coordination environment ( 28 ) involving two cysteines from one monomer and a third cysteine , the equivalent to Cys79 in ZntR , from the other monomer ( 24 , 29 -- 31 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12958362" target="_blank">12958362</a>]<br></li>
<li>The metal selectivity and Zn ( II ) affinity of ZntR are being evaluated and may shed light on the molecular basis of metal ion recognition by this MerR-family member .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10608803" target="_blank">10608803</a>]<br></li>
<li>To understand metal sensitivity and selectivity in the MerR-family of metalloregulatory proteins at the atomic level , we solved the x-ray crystal structures of metal-bound forms of two representative members that discriminate between metal ions with a 1 and a 2 charge : E. coli CueR and E. coli ZntR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12958362" target="_blank">12958362</a>]<br></li>
<li>This work demonstrates that these residues are important for ZntR function and structure , as they are in MerR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12445820" target="_blank">12445820</a>]<br></li>
<li>Irrespective of the detailed function of these residues , these results show that all ﬁve histidine residues are important for ZntR function .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12445820" target="_blank">12445820</a>]<br></li>
<li>Taken together , these data show that both cysteine and histidine residues are important in the function of ZntR and that alteration of the C-terminal amino acid can have a profound effect on the properties of this regulator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12445820" target="_blank">12445820</a>]<br></li>
<li>Note that the value axes are of different ranges ZntR derivatives suggests that the conserved cysteine residues play a central role in the mechanism of induction and are likely to act as ligands for metal-ions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12445820" target="_blank">12445820</a>]<br></li>
<li>It is tempting to suggest that since alterations in these residues appear to affect the substrate range of ZntR that they act as ligands for one of the two Zn ( II ) - ions predicted to bind each ZntR monomer , perhaps acting in a structural role rather than a primary driver of DNA distortion .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12445820" target="_blank">12445820</a>]<br></li>
<li>It is unsurprising that alterations of these residues abrogate the transcriptional response of ZntR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12445820" target="_blank">12445820</a>]<br></li>
				</div></ul><br>
			<label class="collapse_EVO" for="_3"><font size="5px">ZntR evolution</font></label>
				<input id="_3" type="checkbox">
				<div class="EVO"><br><ul><li>ZntR responds to Cd as well as to Zn ( Brocklehurst II II et al. , 1999 ) , and CueR shows 37 % identity to ZntR across the length of the protein sequence ( Fig. 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11136469" target="_blank">11136469</a>]<br></li>
<li>The predicted sequence of the ZntR protein exhibits 34 % identity with the MerR metalloregulatory protein , an Hg ( II ) - specific receptor that regulates expression of bacterial plasmid-encoded mercury detoxification genes ( 20 -- 29 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10608803" target="_blank">10608803</a>]<br></li>
<li>A coiledcoil structure ( Fig. 6B , shown in magenta ) is present in BmrR [ 29 ] and a similar structure is thought to link the two domains of ZntR ( and other MerR-like regulators ) [ 30 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12445820" target="_blank">12445820</a>]<br></li>
<li>The mechanism of target gene activation by oxidized SoxR involves promoter distortion , being similar to that by MerR in response to Hg2 + ( Ansari et al. , 1995 ; Hidalgo and Demple , 1997 ) and to that by another MerR family member ZntR in response to Zn2 + ( Outten et al. , 1999 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12773378" target="_blank">12773378</a>]<br></li>
<li>ZntR is one of a growing number of proteins belonging to the MerR family that respond to a metal ion other than Hg ( II ) , including PMTR , which is implicated in Zn ( II ) - dependent signalling in Proteus mirabilis ( Noll et al. , 1998 ) , a Co ( II ) - responsive regulator , CoaR , in Synechocystis PCC6803 ( J. Rutherford and N. J. Robinson , personal communication ) and a Pb ( II ) - responsive regulator , PbrR , in Alcaligenes eutrophus ( Borremans et al. , 1997 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10048032" target="_blank">10048032</a>]<br></li>
<li>Among the other members of the MerR family are several metalresponsive regulators of gene expression ; they respond to cobalt in Synechocystis PCC 6803 ( CoaR ; Rutherford et al. , 1999 ) , zinc in E. coli ( ZntR ; Brocklehurst et al. , 1999 ) and lead in Ralstonia metallidurans CH34 ( PbrR ; B. Borremans , D. van der Lelie , J. L. Hobman and N. L. Brown , unpublished data ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11136469" target="_blank">11136469</a>]<br></li>
<li>CueR is the sixth member of the MerR family that responds to metal ions , the others being MerR itself ( Foster and Brown , 1985 ; Lund et al. , 1986 ) , PbrR ( B. Borremans , D. van der Lelie , J. L. Hobman and N. L. Brown , unpublished data ) , PMTR ( Noll et al. , 1998 ) , ZntR ( Brocklehurst et al. , 1999 ) and CoaR ( Rutherford et al. , 1999 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11136469" target="_blank">11136469</a>]<br></li>
<li>In comparison , BluR shows only low overall identity to other MerR-like proteins of E. coli ( 12 % to SoxR , 17 % to CueR and 14 % to ZntR ) ( Fig .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22783906" target="_blank">22783906</a>]<br></li>
<li>The predicted primary sequence of ZntR shows 34 % identity with MerR , containing ﬁve cysteine and ﬁve histidine residues , and is a member of the MerR-family of regulators which includes SoxR from E. coli ( the superoxide stress response [ 11 ] ) ; TipAL from Streptomyces lividans ( thiostrepton-regulated gene expression [ 12 ] ; NolA from Bradyrhizobium japonicum ( nodulation gene expression ; [ 13 ] and BmrR and BltR from Bacillus subtilis ( expression of multidrug resistance [ 14,15 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12445820" target="_blank">12445820</a>]<br></li>
<li>The yehV gene product , particularly the N-terminus , exhibits identity to members of the mercury resistance regulator ( MerR ) family of proteins , including MerR proteins , GlnR from Bacillus subtilis , ZntR from E. coli , putative ORFs from E. coli K-12 ( YcgE ) , S. typhimurium ( ORF242 ) , Pseudomonas aeruginosa ( PA299 ) , Vibrio cholerae ( VCA0056 ) ( Fig. 2 ) and other MerR-related stress response regulators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11489123" target="_blank">11489123</a>]<br></li>
<li>The metal selectivity and Zn ( II ) affinity of ZntR are being evaluated and may shed light on the molecular basis of metal ion recognition by this MerR family member .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10608803" target="_blank">10608803</a>]<br></li>
<li>Members of the MerR/ZntR family or of the + + + ArsR/SmtB family regulate Pb2 , Cd2 , and Zn2 extrusion The regulator of zntA in E. coli , ZntR , induces zntA 2 + 2 + transcription at 19 µM Cd or 100 µM Zn ( Noll 2 + & Lutsenko 2000 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11831459" target="_blank">11831459</a>]<br></li>
<li>Members of this family include MerR [ 17 ] , ZntR [ 18 ] and CoaR [ 19 ] , the mercury - , zinc - and cobalt-inducible transcriptional activators in Escherichia coli and Synechocystis , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12829264" target="_blank">12829264</a>]<br></li>
<li>This family includes metal-responsive transcriptional activators such as the zinc sensor ZntR ( 1 , 10 ) and the mercury sensor MerR ( 11 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12958362" target="_blank">12958362</a>]<br></li>
<li>In order to delineate which mechanistic aspects are conserved among MerR family members ( 40 -- 43 ) , we have examined the mechanism that ZntR employs in transcriptional regulation of zntA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10608803" target="_blank">10608803</a>]<br></li>
<li>Also ZntR , the Zn21-responsive transcriptional regulator of zntA that belongs to the family of MerR-like prokaryotic transcriptional regulators , has been identified ( 7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10816566" target="_blank">10816566</a>]<br></li>
<li>The ZntR product is a member of the MerR family of transcriptional regulators and appears to act as a hypersensitive transcriptional switch .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10048032" target="_blank">10048032</a>]<br></li>
<li>ZntR is a member of the growing family of MerR-like pro-karyotic transcriptional regulators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10048032" target="_blank">10048032</a>]<br></li>
<li>ZntR , a member of the ArsR family , regulates the expression of zntA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11831459" target="_blank">11831459</a>]<br></li>
<li>zntA expression is regulated by ZntR , a transcriptional regulator belonging to the MerR family , which activates zntA expression in the presence of Zn ( II ) [ 10 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10802061" target="_blank">10802061</a>]<br></li>
				<label class="EVO_collapse_similarity" for="evo_similarity_21">
In addition , zntA expression has been yodA reported to be up-regulated primarily by zinc , mediated by the regulatory protein ZntR , belonging to the MerR transcriptional regulator family .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12480884" target="_blank">12480884</a>]<br></label>
<input id="evo_similarity_21", type="checkbox">
<div class="EVO_similar">
<li>zntA expression has been reported to be up-regulated primarily by Zn ( II ) , mediated by the regulatory protein ZntR , belonging to the MerR transcriptional regulator family .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10802061" target="_blank">10802061</a>]<br></li></div>
<li>These three cysteine residues are conserved with respect to other MerR family members ( Fig. 5 ) , and correspond to residues C79 , C114 , and C124 , respectively , in ZntR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12445820" target="_blank">12445820</a>]<br></li>
<li>Both this long spacer and palindromic sequences found within the 210/235 region ( Fig. 6A ) are similar to those found in promoters regulated by other members of the MerR family of transcriptional regulators , including SoxR and ZntR ( 12 , 13 , 19 , 22 , 23 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10915804" target="_blank">10915804</a>]<br></li>
				</div></ul><br>
			<label class="collapse_RP" for="_4"><font size="5px">Processes regulated by ZntR</font></label>
				<input id="_4" type="checkbox">
				<div class="RP"><br><ul><li>S. aureus also contains a chromosomal Zn ( II ) / Co ( II ) - speci cents c metalloregulatory transcriptional repressor , CzrA ( also designated ZntR ) 1 , that negatively regulates the expression of the czr operon ( Fig. 2 ) which encodes , in addition to CzrA , a membrane-bound Zn ( II ) transporter of the cation di ¡ usion facilitator family , CzrB ( ZntA ) 1 [ 29,30 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12829264" target="_blank">12829264</a>]<br></li>
<li>[ 52 ] Singh , V.K. , Xiong , A. , Usgaard , T.R. , Chakrabarti , S. , Deora , R. , Misra , T.K. and Jayaswal , R.K. ( 1999 ) ZntR is an autoregulatory protein and negatively regulates the chromosomal zinc resistance op-eron znt of Staphylococcus aureus .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12829264" target="_blank">12829264</a>]<br></li>
<li>HmrR ( heavy metal regulator ) Sinorhizobium meliloti ( GenBank Accession Number AF129004 ) , HMTR ( heavy metal dependent transcription regulator ) Proteus mirabilis ( GenBank Accession Number AF017747 ) , HmrR Rhizobium leguminosarum ( GenBank Accession Number AF127795 ) , CopR ( YbbI ) E. coli ( Swiss-Prot # P77565 ) , YbbI Haemophilus in ¯ uenzae ( GenBank Accession Number U32715 ) , ZntR E. coli ( Swiss-Prot # P36676 ) , MerR Bacillus megaterium ( Swiss-Prot # P22853 ) , MerR Pseudo-monas aeruginosae Tn501 ( Swiss-Prot # P06688 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11167016" target="_blank">11167016</a>]<br></li>
<li>Two species of cytosolic transcription factor , ZntR and Zur , are involved in transcription regulation of the zinc response genes ( 3 , 24 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16159766" target="_blank">16159766</a>]<br></li>
<li>For instance , the response-to-copper includes at least four different transcription factors , CueR , CusSR , CpxAR and YedVW , while at least four different transcription factors , Zur , ZraSR , ZntR and NhaR have been identiﬁed for zinc response ( Fig. 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17884222" target="_blank">17884222</a>]<br></li>
<li>Based on the sequence similarities among CueR , ZntR , and MerR ( Fig. 6B ) , it is likely that CueR detects cytoplasmic copper stress and activates transcription in response to increasing copper concentrations .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10915804" target="_blank">10915804</a>]<br></li>
<li>Examples of molecules associated with stress response or nutrient availability that have been shown to activate MerR-type proteins include mercury ( MerR ) ( Summers , 1992 ) , zinc ( ZntR ) ( Outten et al. , 1999 ) and other metal ions , excess nitrogen sources ( GlnR ) ( Fisher , 1999 ) , the antibiotic thiostrepton ( TipA ) ( Chiu et al. , 1999 ) and Fe2S2 accumulation from oxidative / nitrosative stress ( SoxR ) ( Bradley et al. , 1997 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11489123" target="_blank">11489123</a>]<br></li>
<li>The Escherichia coli ZntR protein , a homologue of MerR , has recently been shown to mediate Zn ( II ) - responsive regulation of zntA , a gene involved in Zn ( II ) detoxification .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10608803" target="_blank">10608803</a>]<br></li>
<li>The yehV gene product , particularly the N-terminus , exhibits identity to members of the mercury resistance regulator ( MerR ) family of proteins , including MerR proteins , GlnR from Bacillus subtilis , ZntR from E. coli , putative ORFs from E. coli K-12 ( YcgE ) , S. typhimurium ( ORF242 ) , Pseudomonas aeruginosa ( PA299 ) , Vibrio cholerae ( VCA0056 ) ( Fig. 2 ) and other MerR-related stress response regulators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11489123" target="_blank">11489123</a>]<br></li>
				</div></ul><br>
			<label class="collapse_SIT" for="_5"><font size="5px">ZntR binding site</font></label>
				<input id="_5" type="checkbox">
				<div class="SIT"><br><ul><li>All of the ZntR derivatives were active at PzntA-1 in the absence or presence of Table 1 Primers used for mutagenesis written in a 50 -- 30 orientation Primer name Sequence CAGCAGATGATGGAGGCTGAGGTGCGTACTGAAGG CCAGCGATTGAAATTTATCCGGGCCGCCAGACAACTAGG GCATCGATCCTGAGGCCCATACCTGTCAGGAGTCAAAAGGC GCATCGATCCTGAACACGCTACATGTCAGGAGTCAAAAGGC CGATCCTGAACACCATACCGCTCAGGAGTCAAAAGGC CGATCCTGAACACCATACCTCACAGGAGTCAAAAGGC CGCCTTAACGATGCCGCATGTGGGACTGCTCATAGC CGCCTTAACGATGCCTCATGTGGGACTGCTCATAGC CGCCTTAACGATGCCTGTGCAGGGACTGCTCATAGC H29A H53A H76A H77A C79A C79S C114A C114S C115A CGCCTTAACGATGCCTGTTCAGGGACTGCTCATAGC GGGACTGCTCATTAGCAGTGTTTATGCATCGATTCTTGAAGC GGGACTGCTCATTAGCAGTGTTTATTCATCGATTCTTGAAGC GGCGTTAAGAGTGGCGCTTGATTTTTTGCACTGGCAGG GGCGTTAAGAGTGGTTCATGATTTTTTGCACTGGCAGG GGCGTTAAGAGTGGTCATTGATTTTTTGCACTGGCAGG GGCGTTAAGAGTGGTCTCTAATTTTTTGCACTGGCAGG GGCGTTAAGAGTGGTTACTAGTTTTTTGCACTGGCAGG GGCGTTAAGAGTGGTGGTTGATTTTTTGCACTGGCAGG C115S C124A C124S C141A C141S C141H C141L C141Y C141G GGCGTTAAGAGTGGTGATTGATTTTTTGCACTGGCAGG GCCTGTTGTGGGACTGCTGCTAGCCAGTGTTTATTGTTCG CGCCAGGGTTTTCCCAGTCACGAC AGCGGATAACAATTTCACACAGGA C141D H119A M13-Forward M13-Reverse Only one of the primer pairs is shown as the second primer was an exact complement .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12445820" target="_blank">12445820</a>]<br></li>
<li>The MerR -- ZntR hybrid ( MZ44 ) was created using M13 universal primers , which anneal to vector sequences either side of the cloning site together with MZ3 ( 58-CCTATGGCAGCATCCGCCTATATACCGAAAGC-38 ) and MZ4 ( 58-GATCGCTTTCGGTATATAGGCGGATGC-TGCC-38 ) ( Gibco BRL ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10048032" target="_blank">10048032</a>]<br></li>
<li>Transcriptional activity of ZntR derivatives at PzntA-1 All of the ZntR derivatives created in this work were assayed for their ability to elicit transcription from a PzntA derivative promoter ( PzntA-1 [ 10 ] ) in which the spacer region between the ) 10 and ) 35 regions has been shortened from the 20 bp seen in the w.t sequence to 19 bp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12445820" target="_blank">12445820</a>]<br></li>
<li>Creation and functional assay of a MerR/ZntR hybrid protein A PzntA derivative was created in which the Pmer 7 -- 4 -- 7 inverted-repeat ( Fig. 3C ) replaced the 11 -- 11 bp inverted-repeat of PzntA ; this was designated Pzmer ( Table 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10048032" target="_blank">10048032</a>]<br></li>
<li>The footprints of ZntR , MerR , and SoxR show hypersensitivity to 5-phenyl-OP-Cu cleavage at the center of the palindromic binding site with the activating metal bound ( 24 , 55 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10608803" target="_blank">10608803</a>]<br></li>
<li>An alignment of ZntR and MerR primary amino acid sequence ( Christie et al. , 1994 ) is repeated here for clarity ( Fig. 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10048032" target="_blank">10048032</a>]<br></li>
<li>As shown in Fig. 7 , MerR , SoxR , and ZntR all bind to a palindromic sequence located between the 235 and 210 regions of their target promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10608803" target="_blank">10608803</a>]<br></li>
<li>Both this long spacer and palindromic sequences found within the 210/235 region ( Fig. 6A ) are similar to those found in promoters regulated by other members of the MerR family of transcriptional regulators , including SoxR and ZntR ( 12 , 13 , 19 , 22 , 23 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10915804" target="_blank">10915804</a>]<br></li>
<li>It is unclear why the C141A or C141H mutations alter the function of ZntR to such a degre although we would hypothesise that the presence of a thiol group at this position may alter the access of metalions to other binding sites which drive the torsional mechanism of induction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12445820" target="_blank">12445820</a>]<br></li>
<li>Zn-ZntR binds and distorts the DNA at sites directly in between the transcription bubble and the RNAP-induced bends .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10608803" target="_blank">10608803</a>]<br></li>
				</ul></div><br>
			<label class="collapse_TU" for="_6"><font size="5px">Transcription units regulated by ZntR</font></label>
				<input id="_6" type="checkbox">
				<div class="TU"><br><ul><li>Our result , that the expression of zraP is under control of the HydH/G system , seems to be at ® rst in contrast with the report that this gene is activated by a MerR-like transcriptional activator ( PMTR ) , which shares signi ® cant homology with ZntR ( Noll et al. , 1998 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11243806" target="_blank">11243806</a>]<br></li>
<li>The induction of zntA transcription requires the ZntR transcription activator , while znuC transcription is repressed by the Zur transcription repressor ( 3 , 25 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16159766" target="_blank">16159766</a>]<br></li>
<li>In this paper , we demonstrate that zntA is maximally induced by Cd ( II ) , not Zn ( II ) , as well as by Pb ( II ) , and that the Cd ( II ) - and Pb ( II ) - dependent transcriptional regulation of zntA is also mediated by ZntR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10802061" target="_blank">10802061</a>]<br></li>
<li>E. coli TG1 [ pUCDPzmer ] , E. coli TG1 ( zntR : : kan ) [ pUCDPzmer ] and E. coli TG1 ( zntR : : kan ) [ pSU18 ; pUCDPzmer ] cells showed identical induction responses to both metals ( data not shown ) , demonstrating that this promoter derivative is not regulated by ZntR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10048032" target="_blank">10048032</a>]<br></li>
<li>Expression and puriﬁcation of ZntR The primers LIC1 58-GACGACGACAAGATGTATCGCATT-GGTGAGCTGGC-38 and LIC2 58-GAGGAGAAGCCCGGTT-CAACAACCACTCTTAACGCC-38 were used to amplify zntR using Vent DNA polymerase ( New England Biolabs ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10048032" target="_blank">10048032</a>]<br></li>
				</div></ul></div><br>
	</div>
</body>
</html>