<!doctype html>
<style>
@charset "utf-8";
body {
	height: 100%;
	margin: 0;
	padding: 0;
}

.banner {
	position: fixed;
	top: 0;
	width: 100%;
	height: 65px;
	background: gray;
	line-height: 65px;
}
.spacer {
	height: 10px;
}

.main-content {
	padding: 5%;
}

.collapse_ACT {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-style: none solid none none;
	border-color: #B4045F;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_ACT + input{
	display: none;
}
.collapse_ACT + input + div{
	display: none;
}
.collapse_ACT + input:checked + div{
	display: block;
}
.ACT {
	text-align: justify;
	text-justify: inter-word;
}
.ACT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B4045F;
}
.ACT_collapse_similarity:hover{
	background: #e5d3dc;
}
.ACT_collapse_similarity + input {
	display: none;
}
.ACT_collapse_similarity + input + div{
	display: none;
}

.ACT_collapse_similarity + input:checked + div{
	display: block;
}
.ACT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.ACT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B4045F;
	padding-left: 5px;
}
.ACT_collapse_similarity:hover{
	background: #e5d3dc;
}
.ACT_collapse_similarity + input {
	display: none;
}
.ACT_collapse_similarity + input + div{
	display: none;
}
.ACT_collapse_similarity + input:checked + div{
	display: block;
}
.ACT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.ACT_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #B4045F;
}
.ACT_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_DOM {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-style: none solid none none;
	border-color: #B404AE;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_DOM + input{
	display: none;
}
.collapse_DOM + input + div{
	display: none;
}
.collapse_DOM + input:checked + div{
	display: block;
}
.DOM {
	text-align: justify;
	text-justify: inter-word;
}
.DOM_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B404AE;
	padding-left: 5px;
}
.DOM_collapse_similarity:hover{
	background: #e8d7e7;
}
.DOM_collapse_similarity + input {
	display: none;
}
.DOM_collapse_similarity + input + div{
	display: none;
}
.DOM_collapse_similarity + input:checked + div{
	display: block;
}
.DOM_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}.DOM_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #B404AE;
}
.DOM_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_EVO {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #7401DF;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_EVO + input{
	display: none;
}
.collapse_EVO + input + div{
	display: none;
}
.collapse_EVO + input:checked + div{
	display: block;
}
.EVO {
	text-align: justify;
	text-justify: inter-word;
}
.EVO_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #7401DF;
	padding-left: 5px;
}
.EVO_collapse_similarity:hover{
	background: #e2d7ed;
}
.EVO_collapse_similarity + input {
	display: none;
}
.EVO_collapse_similarity + input + div{
	display: none;
}
.EVO_collapse_similarity + input:checked + div{
	display: block;
}
.EVO_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.EVO_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #7401DF;
}
.EVO_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_RP {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #0174DF;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_RP + input{
	display: none;
}
.collapse_RP + input + div{
	display: none;
}
.collapse_RP + input:checked + div{
	display: block;
}
.RP {
	text-align: justify;
	text-justify: inter-word;
}
.RP_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #0174DF;
	padding-left: 5px;
}
.RP_collapse_similarity:hover{
	background: #c9d7e2;
}
.RP_collapse_similarity + input {
	display: none;
}
.RP_collapse_similarity + input + div{
	display: none;
}
.RP_collapse_similarity + input:checked + div{
	display: block;
}
.RP_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.RP_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #0174DF;
}
.RP_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_SIT {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #0489B1;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_SIT + input{
	display: none;
}
.collapse_SIT + input + div{
	display: none;
}
.collapse_SIT + input:checked + div{
	display: block;
}
.SIT {
	text-align: justify;
	text-justify: inter-word;
}
.SIT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #0489B1;
	padding-left: 5px;
}
.SIT_collapse_similarity:hover{
	background: #d0dfe2;
}
.SIT_collapse_similarity + input {
	display: none;
}
.SIT_collapse_similarity + input + div{
	display: none;
}
.SIT_collapse_similarity + input:checked + div{
	display: block;
}
.SIT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.SIT_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #0489B1;
}
.SIT_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_TU {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #04B486;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_TU + input{
	display: none;
}
.collapse_TU + input + div{
	display: none;
}
.collapse_TU + input:checked + div{
	display: block;
}
.TU {
	text-align: justify;
	text-justify: inter-word;
}
.TU_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #04B486;
	padding-left: 5px;
}
.TU_collapse_similarity:hover{
	background: #d0e2de;
}
.TU_collapse_similarity + input {
	display: none;
}
.TU_collapse_similarity + input + div{
	display: none;
}
.TU_collapse_similarity + input:checked + div{
	display: block;
}
.TU_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.TU_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #04B486;
}
.TU_similar > a {
	text-decoration: none;
	color: gray;
}

ul { 
	list-style-type: none;
	padding: 0;
	margin: 0;
}
ul li > a {
	text-decoration: none;
	color: gray;
}
ul li:hover, ul li:focus {
	background: #F0F0F0;
}

ul li:first-child span {
  border-left:none;  
}

</style>
<html>
<head>
<meta charset="utf-8">
<title>TF properties - AcrR</title>
<meta name="theme-color" content="#ffffff">
</head>
<body>
	<div class="banner">
		<center><font color="white" size="16px" face="arial">AcrR</font></center>
	</div>
	<div align="center" class="main-content">
		<div class="spacer">
	</div><br>			<label class="collapse_ACT" for="_1"><font size="5px">AcrR transcriptional activity</font></label>
				<input id="_1" type="checkbox">
				<div class="ACT"><br><ul><li>Keywords : AcrR ; AcrB ; Salmonella ; Eﬄux ; Repressor ; Fluoroquinolone 1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15336432" target="_blank">15336432</a>]<br></li>
<li>Gene Gene name Functional group Fold change Loga Stationary 5.4 4.4 4.5 1.7 0.3 7.5 2.9 4.2 5.1 2.1 2.4 2.7 2.3 3.0 4.5 2.6 1.8 2.0 5.7 2.7 2.7 3.1 3.2 4.3 2.8 4.3 2.1 2.5 3.0 Inhibitor of cell division upon DNA damage Putative inner membrane protein Putative inner membrane protein Putative inner membrane protein Putative 3-hexulose-6-phosphate isomerase DNA damage-inducible , inhibits UmuD processing SOS response-regulator DNA strand exchange and recombination protein Recombination and DNA repair Holliday junction helicase subunit A Mutagenesis by UV and mutagens related to Mutagenesis by UV and mutagens related to Error-prone repair : component of DNA polymerase V Error-prone repair : component of DNA polymerase V Error-prone repair Error-prone repair Error-prone repair : DNA polymerase IV UvrA with UvrBC is a DNA excision repalr enzyme DNA damage-inducible gene in SOS regulon Putative short-chain alcohol dehydrogenase Diacylglycerol kinase Putative acetyl esterase Fels-2 prophage : hypothetical protein Fels-1 prophage ; putative tail assembly protein Gifsy-2 prophage Putative cytoplasmic protein Putative inner membrane protein Putative transcriptional-repressor ( TetR/AcrR family ) Putative periplasmic protein Metabolism of surface poly - and lipopolysaccharides Cell envelope Cell envelope Cell envelope Central intermediary metabolism DNA replication , recombination , and repair DNA replication , recombination , and repair DNA replication , recombination , and repair DNA replication , recombination , and repair DNA replication , recombination , and repair DNA replication , recombination , and repair DNA replication , recombination , and repair DNA replication , recombination , and repair DNA replication , recombination , and repair DNA replication , recombination , and repair DNA replication , recombination , and repair DNA replication , recombination , and repair DNA replication , recombination , and repair DNA replication , recombination , and repair Fatty acid and phospholipid biosynthesis Fatty acid and phospholipid biosynthesis Fatty acid and phospholipid degradation Unknown Prophage functions Prophage functions Unknown Unknown Unknown Unknown 0.7 1.2 sulA STM1331 yeeA yigN sgbH dinI lexA recA recN ruvA PSLT055 PSLT054 umuC umuD mucA mucB dinB uvrA yebG STM1675 dgkA STM4032 STM2728 STM0927 STM1019 STM0718 STM1281 ydhM yebF 5.4 2.8 0.3 4.7 2.3 4.4 11.6 2.9 7.2 6.2 1.1 0.5 25.0 12.5 4.8 2.3 2.1 1.1 7.1 3.6 9.0 1.1 2.3 0.8 0.7 0.8 3.4 a Log-phase data are from [ 9 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20691712" target="_blank">20691712</a>]<br></li>
<li>It is regulated locally by the repressor AcrR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28926285" target="_blank">28926285</a>]<br></li>
<li>AcrAB regulation is mediated by five regulators : MarA , SoxS , Rob , AcrR and RamA ( Bailey et al. , 2008 ; Ma et al. , 1996 ; Randall & Woodward , 2002 ; Rosenberg et al. , 2003 ; Schneiders et al. , 2003 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21148208" target="_blank">21148208</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_5">
In Escherichia coli , AcrAB is regulated by three activators , MarA , SoxS and Rob , and one repressor , AcrR ( Ma et al. , 1996 ; Randall & Woodward , 2002 ; Rosenberg et al. , 2003 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21148208" target="_blank">21148208</a>]<br></label>
<input id="act_similarity_5", type="checkbox">
<div class="ACT_similar">
<li>In addition to RamA , AcrAB is regulated by three activators , MarA , SoxS and Rob , and one repressor , AcrR ( Ma et al. , 1996 ; Randall & Woodward , 2002 ; Rosenberg et al. , 2003 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21148208" target="_blank">21148208</a>]<br></li></div>
<li>For example , acrAB transcription in Salmonella is controlled by AcrR , a TetR-like repressor encoded nearby , and by the AraC/XylS-like regulators RamA , MarA , SoxS and Rob , involved in stress - and virulence-response ( Bailey et al. , 2010 ; Nikaido et al. , 2011 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28631419" target="_blank">28631419</a>]<br></li>
<li>Its expression is modulated precisely by its local repressor , AcrR , as well as by global transcriptional-regulators of the AraC family , RamA , MarA , SoxS , and Rob ( Nikaido et al. , 2008 ; Kehrenberg et al. , 2009 ; Pérez et al. , 2012 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28261566" target="_blank">28261566</a>]<br></li>
<li>Other regulators of acrAB such as MarA , SoxS , Rob , SdiA , and AcrR did not contrib-ute to acrAB induction by indole in Salmonella .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18577510" target="_blank">18577510</a>]<br></li>
<li>Other regulators , including MarA , SoxS , SdiA , and AcrR , are not involved in AcrAB induction by indole and bile [ 92 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19230852" target="_blank">19230852</a>]<br></li>
<li>Ruiz and Levy demonstrated that when the AcrAB-TolC pump in Escherichia coli is inactivated or inhibited , cellular metabolites accumulate , inactivating AcrR and/or upregulating soxS and marA expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28650690" target="_blank">28650690</a>]<br></li>
<li>AcrAB expression is adjusted by a local repressor , AcrR , and by the global regulators SoxS , MarA , RamA , and Rob [ 19 , 21 , 29 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27116992" target="_blank">27116992</a>]<br></li>
<li>In E. coli it is modulated at the lowest level by the local repressor AcrR .14 At a more global level , acrAB expression is modulated by stress conditions and by global regulatory genes , such as marA , soxS , and rob .14,17,18 The acrAB locus is indeed part of the mar , sox , and rob regulons of E. coli , the activation of which confers a low-level resistance to a wide range of antimicrobial agents and organic solvents .17 Previous studies clearly showed that overexpression of marA , which results from marR mutations , would down-regulate the expression of acrR , thus inducing high-level AcrAB expression and antimicrobial resistance .23 Proteins encoded by the mar locus include the transcriptional-activator , MarA , its local repressor , MarR , and MarB and MarC , two proteins with unknown functions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16201927" target="_blank">16201927</a>]<br></li>
				</div></ul><br>
			<label class="collapse_EVO" for="_3"><font size="5px">AcrR evolution</font></label>
				<input id="_3" type="checkbox">
				<div class="EVO"><br><ul><li>In P. ﬂuorescens , PsrA , a member of the TetR/AcrR family of transcriptional regulators , which directly activates the expression of the stationary-phase sigma factor RpoS ( 42 ) , is required for the expression of rsmZ ( 38 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24187088" target="_blank">24187088</a>]<br></li>
<li>Structures of AcrR and CmeR : insight into the mechanisms of transcriptional repression and multi-drug recognition in the TetR family of regulators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22948865" target="_blank">22948865</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22123696" target="_blank">22123696</a>]<br></li>
<li>At a local level acrAB expression is modulated by the local repressor AcrR , and at a global level it is modulated by MarA , SoxS , and Rob , which belong to the AraC/XylS family of transcriptional regulators ( 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19104017" target="_blank">19104017</a>]<br></li>
<li>Its expression is modulated precisely by its local repressor , AcrR , as well as by global transcriptional regulators of the AraC family , RamA , MarA , SoxS , and Rob ( Nikaido et al. , 2008 ; Kehrenberg et al. , 2009 ; Pérez et al. , 2012 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28261566" target="_blank">28261566</a>]<br></li>
				</div></ul><br>
			<label class="collapse_RP" for="_4"><font size="5px">Processes regulated by AcrR</font></label>
				<input id="_4" type="checkbox">
				<div class="RP"><br><ul><li>Other regulators , including MarA , SoxS , SdiA , and AcrR , are not involved in AcrAB induction by indole and bile [ 92 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19230852" target="_blank">19230852</a>]<br></li>
<li>Furthermore , a local repres-6 -- 10 sor , AcrR , has been identiﬁed to play a role in the regulation of acrAB genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19778917" target="_blank">19778917</a>]<br></li>
<li>[ 12 ] Ma , D. , Alberti , M. , Lynch , C. , Nikaido , H. and Hearst , J.E. ( 1996 ) The local repressor AcrR plays a modulating role in the regulation of acrAB genes of Escherichia coli by global stress [ 13 ] Randall , L.P. and Woodward , M.J. ( 2002 ) The multiple antibiotic resistance ( mar ) locus and its signiﬁcance .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15336432" target="_blank">15336432</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_4">
Ma D. , Alberti M. , Lynch C. , Nikaido H. , Hearst J.E. ( 1996 ) The local repressor AcrR plays a modulating role in the regulation of acrAB genes of Escherichia coli by global stress signals .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19120970" target="_blank">19120970</a>]<br></label>
<input id="rp_similarity_4", type="checkbox">
<div class="RP_similar">
<li>[ 54 ] D. Ma , M. Alberti , C. Lynch , H. Nikaido , J.E. Hearst , The local repressor AcrR plays a modulating role in the regulation of acrAB genes of Escherichia coli by global stress signals , Mol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19230852" target="_blank">19230852</a>]<br></li>
<li>The local repressor AcrR plays a modulating role in the regulation of acrAB genes by global stress signals .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10613858" target="_blank">10613858</a>]<br></li>
<li>The local repressor AcrR plays a modulating role in the regulation of acrAB genes of Escherichia coli by global stress signals .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18083849" target="_blank">18083849</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14742546" target="_blank">14742546</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16201927" target="_blank">16201927</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21148208" target="_blank">21148208</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17933888" target="_blank">17933888</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18407759" target="_blank">18407759</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11036033" target="_blank">11036033</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19778917" target="_blank">19778917</a>]<br></li></div>
<li>However , a number of insertions and single amino acid substitutions , duplications , and deletions that inactivate AcrR , a divergently transcribed local repressor of the acrAB operon , also result in enhanced expression of acrAB and increased ﬂuoroquinolone resistance in clinical E. coli strains [ 75,76 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19230852" target="_blank">19230852</a>]<br></li>
<li>AcrR represses the expression of acrAB locally .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28650690" target="_blank">28650690</a>]<br></li>
<li>In E. coli and S. enterica , AcrR represses expression of acrAB ( 28 , 36 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18083849" target="_blank">18083849</a>]<br></li>
<li>This pump has been reported to play a key role not only in conferring resistance to quinolones , but also in multidrug-resistance ( MDR ) ,12 -- 14 a term used for resistance to four or more antimicrobials in the case of non-typhoidal Samonella .15 In Salmonella , AcrAB expression has been reported to be under the control of several regulators : the local repressor , AcrR ,16 and three transcriptional activators , MarA , belonging to the marRAB operon ,17 SoxS , belonging to the soxRS region18 and RamA , repressed by the ramR gene .19,20 On the other hand , the ability to form a bioﬁlm in response to unfavourable growth conditions is among the virulence properties described for S. enterica , since this phenotype contributes to conferring MDR and can help bacteria to survive in hostile or suboptimal environments .21,22 Curli ﬁmbriae , also designated as thin aggregative ﬁmbriae ( Csg or Agf ) , and cellulose production have been well characterized as contributing to bioﬁlm formation and the rdar morphotype ( for red , dry and rough colony morphology ) .23 -- 27 Recently , evidence showing an association between the acquisition of quinolone resistance and a decreased expression of virulence factors has been published .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24706735" target="_blank">24706735</a>]<br></li>
<li>For example , acrAB transcription in Salmonella is controlled by AcrR , a TetR-like repressor encoded nearby , and by the AraC/XylS-like regulators RamA , MarA , SoxS and Rob , involved in stress - and virulence-response ( Bailey et al. , 2010 ; Nikaido et al. , 2011 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28631419" target="_blank">28631419</a>]<br></li>
<li>Expression of acrAB-tolC is regulated by the global regulator , MarA , which is a positive regulator ( Alekshun and Levy , 1999 ) and by AcrR ( decreased expression in 2a ) , which is a repressor speciﬁc for the acrAB operon ( Ma et al. , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18407759" target="_blank">18407759</a>]<br></li>
<li>Gene Gene name Functional group Fold change Loga Stationary 5.4 4.4 4.5 1.7 0.3 7.5 2.9 4.2 5.1 2.1 2.4 2.7 2.3 3.0 4.5 2.6 1.8 2.0 5.7 2.7 2.7 3.1 3.2 4.3 2.8 4.3 2.1 2.5 3.0 Inhibitor of cell-division upon DNA damage Putative inner membrane protein Putative inner membrane protein Putative inner membrane protein Putative 3-hexulose-6-phosphate isomerase DNA damage-inducible , inhibits UmuD processing SOS-response regulator DNA strand exchange and recombination protein Recombination and DNA-repair Holliday junction helicase subunit A Mutagenesis by UV and mutagens related to Mutagenesis by UV and mutagens related to Error-prone repair : component of DNA polymerase V Error-prone repair : component of DNA polymerase V Error-prone repair Error-prone repair Error-prone repair : DNA polymerase IV UvrA with UvrBC is a DNA excision repalr enzyme DNA damage-inducible gene in SOS regulon Putative short-chain alcohol dehydrogenase Diacylglycerol kinase Putative acetyl esterase Fels-2 prophage : hypothetical protein Fels-1 prophage ; putative tail assembly protein Gifsy-2 prophage Putative cytoplasmic protein Putative inner membrane protein Putative transcriptional repressor ( TetR/AcrR family ) Putative periplasmic protein Metabolism of surface poly - and lipopolysaccharides Cell envelope Cell envelope Cell envelope Central intermediary-metabolism DNA-replication , recombination , and repair DNA-replication , recombination , and repair DNA-replication , recombination , and repair DNA-replication , recombination , and repair DNA-replication , recombination , and repair DNA-replication , recombination , and repair DNA-replication , recombination , and repair DNA-replication , recombination , and repair DNA-replication , recombination , and repair DNA-replication , recombination , and repair DNA-replication , recombination , and repair DNA-replication , recombination , and repair DNA-replication , recombination , and repair DNA-replication , recombination , and repair Fatty acid and phospholipid-biosynthesis Fatty acid and phospholipid-biosynthesis Fatty acid and phospholipid-degradation Unknown Prophage functions Prophage functions Unknown Unknown Unknown Unknown 0.7 1.2 sulA STM1331 yeeA yigN sgbH dinI lexA recA recN ruvA PSLT055 PSLT054 umuC umuD mucA mucB dinB uvrA yebG STM1675 dgkA STM4032 STM2728 STM0927 STM1019 STM0718 STM1281 ydhM yebF 5.4 2.8 0.3 4.7 2.3 4.4 11.6 2.9 7.2 6.2 1.1 0.5 25.0 12.5 4.8 2.3 2.1 1.1 7.1 3.6 9.0 1.1 2.3 0.8 0.7 0.8 3.4 a Log-phase data are from [ 9 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20691712" target="_blank">20691712</a>]<br></li>
<li>Locus Gene function Name Fold Expression ryhB-1 ryhB-2 ryhB-1 ryhB-2 Higher in both single deletion backgrounds STM4526 Endonuclease R , host restriction 3.693 3.335 3.322 2.621 2.613 2.547 2.534 2.426 2.389 2.328 2.307 2.137 2.069 2.069 1.976 1.948 1.891 1.868 1.857 1.838 1.682 1.667 1.600 1.587 1.564 1.552 1.549 1.531 1.518 1.507 1.455 4.028 3.475 2.351 2.191 1.676 1.549 1.812 2.467 2.053 3.301 3.127 1.537 2.195 2.043 2.005 1.952 1.791 3.746 2.238 1.604 1.473 1.792 2.015 1.568 1.581 3.087 2.105 1.677 1.806 1.837 1.713 2.701 2.910 1.616 1.701 1.600 1.659 1.890 1.750 1.997 2.738 2.644 1.431 2.263 1.549 2.104 1.451 1.706 3.254 2.201 1.579 1.540 1.451 1.571 1.944 1.926 3.067 1.818 1.587 1.952 2.527 1.673 hsdR STM1793 Putative cytochrome oxidase , subunit II STM0067 Carbamoyl-phosphate synthase , large subunit STM2886 Surface presentation of antigens secretory proteins STM3014 Positive transcriptional regulator ( LysR family ) STM0843 Putative pyruvate formate lyase STM2667 Bifuctional : chorismate mutase P prephenate dehydratase STM0289 Putative cytoplasmic protein STM2976 l-Fucose isomerase STM0621 Bifunctional citrate lyase alpha chain/citrate-ACP transferase STM1614 Putative PTS-system enzyme IIC component STM2479 Putative oxidoreductase STM0138 Putative cytoplasmic protein STM0986 Putative cytoplasmic protein STM0890 ABC superfamily ( bind prot ) , arginine-transport system STM3026 Putative outer membrane protein STM1290 Glyceraldehyde-3-phosphate dehydrogenase A STM2927 Survival protein , protein damage control STM1201 DNA polymerase III , delta prime subunit STM3519 Putative cytoplasmic protein STM2127 Paral putative outer membrane receptor STM1154 Putative MFS family transport protein STM0573 Putative inner membrane protein STM3833 Putative mandelate racemase/muconate lactonizing enzyme STM0570 Outer membrane N-acetyl phenylalanine beta-naphthyl ester-cleaving esterase STM4500 Putative SAM-dependent methyltransferase STM2040 Propanediol utilization : dehydratase , large subunit STM3939 Adenylate cyclase STM3588 Putative periplasmic protein STM1904 Putative inner membrane protein STM4007 Glutamine synthetase carB sicA lysR pﬂF pheA fucI citF aegA yacG ycaQ artI gapA surE holB rtcB yegN yceE apeE yjhP pduC cyaA yhiN yecN glnA Reduced or unchanged in both single deletion backgrounds STM1005 Gifsy-2 prophage integrase 0.507 0.655 0.759 0.751 0.778 0.743 0.874 0.934 1.051 0.863 0.866 0.874 0.989 0.965 0.577 0.947 0.956 0.720 0.500 0.566 0.768 0.861 0.871 0.893 0.987 0.988 0.999 0.999 1.007 1.029 1.051 1.052 1.062 1.064 1.066 1.067 1.757 1.679 2.584 2.439 1.603 1.499 1.527 1.594 1.454 2.138 1.734 1.437 2.814 1.639 1.550 1.627 1.509 1.423 STM1555 Putative transcriptional regulator STM2031 Synthesis of vitamin B12 adenosyl cobalamide precursor STM1132 Putative sugar-transport protein STM1928 Trehalose-6-phosphate synthase STM3591 Universal stress protein A STM0532 Putative carbamate kinase STM0185 Putative glutamyl t-RNA synthetase STM4503 Putative inner membrane protein STM3446 Protein chain elongation factor EF-G , GTP-binding STM0787 Imidazolonepropionase STM2385 Putative methylase STM2513 Similar to the C-terminal region of AIDA STM1568 Formate dehydrogenase-N , cytochrome B556 ( Fdn ) gamma subunit , nitrate-inducible STM1437 Putative transcriptional repressor ( TetR/AcrR family ) STM0095 23S rRNA pseudouridylate 746 synthase STM4482 GntP family , L-idonate-transport protein STM1513 Putative cytoplasmic protein cbiE otsA uspA arcC yadB fusA hutI yfcB shdA fdnI ydhM rluA idnT Higher only in RyhB-1 deletion background 1.504 2.957 0.921 1.051 1.596 2.974 STM1037 Gifsy-2 prophage probable minor tail protein STM0299 Putative outer membrane protein safA Higher only in RyhB-2 deletion background 3.139 3.031 2.949 2.888 2.841 2.703 2.500 2.497 2.482 2.474 2.391 2.361 2.296 2.259 2.203 2.17 2.109 2.573 2.191 1.626 3.913 4.962 2.902 4.122 2.961 2.534 2.421 1.521 2.789 1.730 2.110 2.961 0.673 0.699 0.674 0.663 0.729 0.865 0.962 0.950 0.979 0.958 0.901 1.019 0.899 0.739 0.497 0.909 STM0925 Fels-1 prophage putative host speciﬁcity protein STM4096 50S ribosomal subunit protein L31 STM0223 Putative membrane-associated Zn-dependent protease STM3822 Trimethylamine N-oxide reductase subunit STM1191 50S ribosomal subunit protein L32 STM1182 Flagellar biosynthesis STM0033 Putative 5-nucleotidase STM1229 Putative cytoplasmic protein STM3750 Putative GPH family transport protein STM1719 putative ribosomal large subunit pseudouridine synthase STM0188 2 -- 5 RNA ligase STM4359 Enzyme in methyl-directed mismatch-repair , stimulates binding STM1842 Putative transcriptional repressor ( IclR family ) STM3749 Putative alpha-xylosidase STM0954 Putative inner membrane protein STM2934 ATP-sulfurylase , subunit 1 ( ATP : sulfate adenylyltransferase ) rpmE yaeL torA rpmF ﬂgJ ycfD yicJ yciL ligT mutL kdgR yicI cysN ybiI folE qor ydiR eutG glgB gdhA ygeD nagZ ygbL cysK ybaM cydB pspA ybbO cheY prpB cbiP ﬂiF yjiX the known targets were excluded from the ﬁrst ﬁltration based on log2 Cy5/Cy3 ratio in the double deletion mutant ; these targets were removed from the candidate target genes probably due to the high stringency used in our ﬁltering step .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23831078" target="_blank">23831078</a>]<br></li>
				</div></ul><br>
			<label class="collapse_SIT" for="_5"><font size="5px">AcrR binding site</font></label>
				<input id="_5" type="checkbox">
				<div class="SIT"><br><ul><li>The RamR binding pattern is also similar to the Escherichia coli AcrR repressor , which binds as a dimer of dimers to a 24-bp sequence of the acrAB operator , including two 10-bp IR sequences separated by a 4-bp linker ( 21 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22123696" target="_blank">22123696</a>]<br></li>
				</ul></div><br>
			<label class="collapse_TU" for="_6"><font size="5px">Transcription units regulated by AcrR</font></label>
				<input id="_6" type="checkbox">
				<div class="TU"><br><ul><li>Other regulators of acrAB such as MarA , SoxS , Rob , SdiA , and AcrR did not contrib-ute to acrAB induction by indole in Salmonella .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18577510" target="_blank">18577510</a>]<br></li>
<li>For example , acrAB transcription in Salmonella is controlled by AcrR , a TetR-like repressor encoded nearby , and by the AraC/XylS-like regulators RamA , MarA , SoxS and Rob , involved in stress - and virulence-response ( Bailey et al. , 2010 ; Nikaido et al. , 2011 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28631419" target="_blank">28631419</a>]<br></li>
<li>AcrR is the local repressor encoded upstream of the acrAB genes and mutations within its coding sequence have been associated with increased expression of the pump .14 In addition , three homologous transcriptional activators , RamA , SoxS and MarA , have been associated with increased acrB and tolC expression levels .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26679248" target="_blank">26679248</a>]<br></li>
<li>In E. coli , the transcription of acrAB and micF is controlled by the homologous proteins MarA , SoxS and Rob ( 9 , 10 ) , but may be affected by other proteins such as AcrR ( 11 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19120970" target="_blank">19120970</a>]<br></li>
<li>Detection of Mutations in the Genes Encoding Quinolone Protein Targets and Regulatory Loci Amplification of the QRDRs of gyrA , gyrB , parC , and parE , as well as the soxRS , marRAB , acrR and ramR regulatory loci ( as it is already known , the transcriptional regulators SoxS , MarA and RamR exert a positive effect on AcrAB/TolC expression , whereas AcrR is the local repressor ) was performed using the corresponding primers listed in Table 5 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19946377" target="_blank">19946377</a>]<br></li>
<li>Expression of acrAB-tolC is regulated by the global regulator , MarA , which is a positive regulator ( Alekshun and Levy , 1999 ) and by AcrR ( decreased expression in 2a ) , which is a repressor speciﬁc for the acrAB operon ( Ma et al. , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18407759" target="_blank">18407759</a>]<br></li>
<li>At a local level acrAB expression is modulated by the local repressor AcrR , and at a global level it is modulated by MarA , SoxS , and Rob , which belong to the AraC/XylS family of transcriptional regulators ( 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19104017" target="_blank">19104017</a>]<br></li>
<li>In E. coli it is modulated at the lowest level by the local repressor AcrR .14 At a more global level , acrAB expression is modulated by stress conditions and by global regulatory genes , such as marA , soxS , and rob .14,17,18 The acrAB locus is indeed part of the mar , sox , and rob regulons of E. coli , the activation of which confers a low-level resistance to a wide range of antimicrobial agents and organic solvents .17 Previous studies clearly showed that overexpression of marA , which results from marR mutations , would down-regulate the expression of acrR , thus inducing high-level AcrAB expression and antimicrobial resistance .23 Proteins encoded by the mar locus include the transcriptional activator , MarA , its local repressor , MarR , and MarB and MarC , two proteins with unknown functions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16201927" target="_blank">16201927</a>]<br></li>
<li>The AcrR protein is known in E. coli to repress transcription of the acrAB operon [ 12 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15336432" target="_blank">15336432</a>]<br></li>
<li>However , a number of insertions and single amino acid substitutions , duplications , and deletions that inactivate AcrR , a divergently transcribed local repressor of the acrAB operon , also result in enhanced expression of acrAB and increased ﬂuoroquinolone resistance in clinical E. coli strains [ 75,76 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19230852" target="_blank">19230852</a>]<br></li>
<li>In E. coli and S. enterica , AcrR represses expression of acrAB ( 28 , 36 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18083849" target="_blank">18083849</a>]<br></li>
<li>AcrR and AcrS repress the expression of acrAB ( Table 3 ) [ 54,55 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19230852" target="_blank">19230852</a>]<br></li>
<li>AcrR represses the expression of acrAB locally .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28650690" target="_blank">28650690</a>]<br></li>
				</div></ul></div><br>
	</div>
</body>
</html>