<!doctype html>
<style>
@charset "utf-8";
body {
	height: 100%;
	margin: 0;
	padding: 0;
}

.banner {
	position: fixed;
	top: 0;
	width: 100%;
	height: 65px;
	background: gray;
	line-height: 65px;
}
.spacer {
	height: 10px;
}

.main-content {
	padding: 5%;
}

.collapse_ACT {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-style: none solid none none;
	border-color: #B4045F;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_ACT + input{
	display: none;
}
.collapse_ACT + input + div{
	display: none;
}
.collapse_ACT + input:checked + div{
	display: block;
}
.ACT {
	text-align: justify;
	text-justify: inter-word;
}
.ACT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B4045F;
}
.ACT_collapse_similarity:hover{
	background: #e5d3dc;
}
.ACT_collapse_similarity + input {
	display: none;
}
.ACT_collapse_similarity + input + div{
	display: none;
}

.ACT_collapse_similarity + input:checked + div{
	display: block;
}
.ACT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.ACT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B4045F;
	padding-left: 5px;
}
.ACT_collapse_similarity:hover{
	background: #e5d3dc;
}
.ACT_collapse_similarity + input {
	display: none;
}
.ACT_collapse_similarity + input + div{
	display: none;
}
.ACT_collapse_similarity + input:checked + div{
	display: block;
}
.ACT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.ACT_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #B4045F;
}
.ACT_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_DOM {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-style: none solid none none;
	border-color: #B404AE;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_DOM + input{
	display: none;
}
.collapse_DOM + input + div{
	display: none;
}
.collapse_DOM + input:checked + div{
	display: block;
}
.DOM {
	text-align: justify;
	text-justify: inter-word;
}
.DOM_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B404AE;
	padding-left: 5px;
}
.DOM_collapse_similarity:hover{
	background: #e8d7e7;
}
.DOM_collapse_similarity + input {
	display: none;
}
.DOM_collapse_similarity + input + div{
	display: none;
}
.DOM_collapse_similarity + input:checked + div{
	display: block;
}
.DOM_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}.DOM_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #B404AE;
}
.DOM_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_EVO {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #7401DF;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_EVO + input{
	display: none;
}
.collapse_EVO + input + div{
	display: none;
}
.collapse_EVO + input:checked + div{
	display: block;
}
.EVO {
	text-align: justify;
	text-justify: inter-word;
}
.EVO_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #7401DF;
	padding-left: 5px;
}
.EVO_collapse_similarity:hover{
	background: #e2d7ed;
}
.EVO_collapse_similarity + input {
	display: none;
}
.EVO_collapse_similarity + input + div{
	display: none;
}
.EVO_collapse_similarity + input:checked + div{
	display: block;
}
.EVO_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.EVO_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #7401DF;
}
.EVO_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_RP {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #0174DF;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_RP + input{
	display: none;
}
.collapse_RP + input + div{
	display: none;
}
.collapse_RP + input:checked + div{
	display: block;
}
.RP {
	text-align: justify;
	text-justify: inter-word;
}
.RP_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #0174DF;
	padding-left: 5px;
}
.RP_collapse_similarity:hover{
	background: #c9d7e2;
}
.RP_collapse_similarity + input {
	display: none;
}
.RP_collapse_similarity + input + div{
	display: none;
}
.RP_collapse_similarity + input:checked + div{
	display: block;
}
.RP_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.RP_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #0174DF;
}
.RP_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_SIT {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #0489B1;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_SIT + input{
	display: none;
}
.collapse_SIT + input + div{
	display: none;
}
.collapse_SIT + input:checked + div{
	display: block;
}
.SIT {
	text-align: justify;
	text-justify: inter-word;
}
.SIT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #0489B1;
	padding-left: 5px;
}
.SIT_collapse_similarity:hover{
	background: #d0dfe2;
}
.SIT_collapse_similarity + input {
	display: none;
}
.SIT_collapse_similarity + input + div{
	display: none;
}
.SIT_collapse_similarity + input:checked + div{
	display: block;
}
.SIT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.SIT_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #0489B1;
}
.SIT_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_TU {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #04B486;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_TU + input{
	display: none;
}
.collapse_TU + input + div{
	display: none;
}
.collapse_TU + input:checked + div{
	display: block;
}
.TU {
	text-align: justify;
	text-justify: inter-word;
}
.TU_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #04B486;
	padding-left: 5px;
}
.TU_collapse_similarity:hover{
	background: #d0e2de;
}
.TU_collapse_similarity + input {
	display: none;
}
.TU_collapse_similarity + input + div{
	display: none;
}
.TU_collapse_similarity + input:checked + div{
	display: block;
}
.TU_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.TU_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #04B486;
}
.TU_similar > a {
	text-decoration: none;
	color: gray;
}

ul { 
	list-style-type: none;
	padding: 0;
	margin: 0;
}
ul li > a {
	text-decoration: none;
	color: gray;
}
ul li:hover, ul li:focus {
	background: #F0F0F0;
}

ul li:first-child span {
  border-left:none;  
}

</style>
<html>
<head>
<meta charset="utf-8">
<title>TF properties - Cra</title>
<meta name="theme-color" content="#ffffff">
</head>
<body>
	<div class="banner">
		<center><font color="white" size="16px" face="arial">Cra</font></center>
	</div>
	<div align="center" class="main-content">
		<div class="spacer">
	</div><br>			<label class="collapse_ACT" for="_1"><font size="5px">Cra transcriptional activity</font></label>
				<input id="_1" type="checkbox">
				<div class="ACT"><br><ul><li>Whether Cra binds to site 1 and/or 2 remains an open question ACKNOWLEDGMENTS This work was supported by PHS grant R01-GM62203 to J.C.E.-S .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19136587" target="_blank">19136587</a>]<br></li>
<li>The catabolite repressor/activator Cra stimulates CsgD expression and curli synthesis in E. coli by binding to four sites [ 90 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25437188" target="_blank">25437188</a>]<br></li>
<li>Cra acts as an activator or repressor , and its DNA-binding activity is abrogated by fructose-1-phosphate and fructose-1 ,6 - bisphosphate ( 30 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19136587" target="_blank">19136587</a>]<br></li>
<li>The combined absence of Cra and IscR derepresses PiscRlacZ expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19136587" target="_blank">19136587</a>]<br></li>
<li>and the catabolite repressor -RCB- activator ( Cra ) protein ( Saier & Ramseier , 1996 ; Saier et al. , 1996 ) , nitrogen utilization is controlled by the NtrBC sensor kinase -RCB- response-regulator pair in conjunction with the GlnB and GlnD proteins ( Magasanik , 1996 ; Reitzer , 1996a , b ) , phosphorus utilization is controlled by the PhoR -RCB- PhoB sensor kinase -RCB- response-regulator pair ( Wanner , 1996 ) , iron utilization is regulated by the Fur transcription factor ( Earhart , 1996 ; Gralla & Collado-Vides , 1996 ) and sulfur metabolism is controlled by the CysB transcriptional-activator ( Kredich , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11782505" target="_blank">11782505</a>]<br></li>
<li>Under the conditions investigated , it is assumed that carbon flux is controlled by the catabolite repressor/activator ( Cra ) protein , which activates gluconeogenesis enzymes ( ppsA , pckA , fbp ) and represses sugar catabolism enzymes [ 61 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20676398" target="_blank">20676398</a>]<br></li>
<li>Cra protein concentrations are given in terms of monomeric protein , although Cra is tetrameric .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19136587" target="_blank">19136587</a>]<br></li>
<li>If the effect of Cra were direct , Cra would bind to one or more nonconsensus sites within the iscR region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19136587" target="_blank">19136587</a>]<br></li>
<li>For example , Cra binds to a nonconsensus site in the cydAB promoter region only when high ( 278 nM ) concentrations of Cra and Fnr are both present ( 29 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19136587" target="_blank">19136587</a>]<br></li>
<li>Cra binds to the iscR promoter region in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19136587" target="_blank">19136587</a>]<br></li>
<li>Although the O1-site downstream of the transcriptional start site appears to be the most important for csgD expression , binding of Cra to operators O2 and O3 located upstream commonly appears to be important for maximal transcription [ 90 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25437188" target="_blank">25437188</a>]<br></li>
<li>Involvement of the Cra Global Regulatory Protein in the Expression o the iscRSUA Operon , Revealed during Studies of Tricarballylate Catabolism in Salmonella enterica † f Jeffrey A. Lewis , Jeffrey M. Boyd , Diana M. Downs , and Jorge C. Escalante-Semerena * Department of Bacteriology , University of Wisconsin -- Madison , 6478 Microbial Sciences Building , 1550 Linden Drive , Madison , Wisconsin 53706-1521 Received 6 November 2008/Accepted 6 January 2009 In Salmonella enterica , tricarballylate ( Tcb ) catabolism requires function of TcuB , a membrane-bound protein that contains [ 4Fe-4S ] clusters and heme .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19136587" target="_blank">19136587</a>]<br></li>
<li>When interacting with the genes it regulates , i.e. , in the presence of gluconeo-genic substrates ( 18 ) , the Cra protein positively regulates transcription of those genes encoding biosynthetic and oxidative enzymes ( e.g. , key enzymes in the tricarboxylic acid cycle , the glyoxylate bypass , the gluconeogenic pathway , and electron transport ) and negatively regulates transcription of genes encoding glycolytic enzymes , e.g. , key enzymes in the EmbdenMeyerhof and Entner-Doudoroff pathways ( 18 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10816546" target="_blank">10816546</a>]<br></li>
<li>Cra regulates numerous genes involved in carbon and energy metabolism and is believed to regulate the direction of carbon ﬂow ( 28 , 32 ) , but to our knowledge , there were no reports in the literature linking Cra to the regulation of iscRSUA operon expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19136587" target="_blank">19136587</a>]<br></li>
<li>This is , however , the ﬁrst evidence of a regulator , other than IscR , involved in the regulation of the iscRSUA operon and the ﬁrst evidence that the assembly of [ Fe-S ] clusters is activated in response to sources of carbon and energy whose catabolism is under the control of Cra .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19136587" target="_blank">19136587</a>]<br></li>
<li>Here we show that low concentrations of glucose and other saccharides ( 1 mM ) restore growth of an apbC strain on Tcb , that the saccharide effect is mediated via the global regulatory protein Cra , that Cra directly affects the expression of the iscRSUA operon , and that the saccharide effect results in increased levels of TcuB activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19136587" target="_blank">19136587</a>]<br></li>
<li>Although it is possible that Cra activates iscRSUA expression , the data in Fig. 4 show that expression of the operon in the absence of Cra is sufﬁcient to support growth on Tcb .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19136587" target="_blank">19136587</a>]<br></li>
<li>It is possible that further activation of iscRSUA by Cra is needed under some as-yet-unidentiﬁed growth conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19136587" target="_blank">19136587</a>]<br></li>
<li>One explanation for these results is that Cra can activate iscRSUA expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19136587" target="_blank">19136587</a>]<br></li>
<li>In the model , IscR represses iscRSUA transcription when bound to a [ 2Fe-2S ] cluster , while Cra can activate iscRSUA expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19136587" target="_blank">19136587</a>]<br></li>
				</div></ul><br>
			<label class="collapse_DOM" for="_2"><font size="5px">Cra protein domains</font></label>
				<input id="_2" type="checkbox">
				<div class="DOM"><br><ul><li>Although the O1-site downstream of the transcriptional start site appears to be the most important for csgD expression , binding of Cra to operators O2 and O3 located upstream commonly appears to be important for maximal transcription [ 90 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25437188" target="_blank">25437188</a>]<br></li>
<li>( v ) Cra protein binds to all three sites in the iscRSUA promoter region in a concentration-dependent fashion .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19136587" target="_blank">19136587</a>]<br></li>
<li>Shown in Fig. 5 is an alignment of the iscR region with the consensus Cra DNA-binding motif ( 32 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19136587" target="_blank">19136587</a>]<br></li>
<li>A 239-bp PCR product containing the three putative Cra binding sites in the iscR promoter region was ampliﬁed using the following primers : ( i ) upstream primer 5 - TATCCGCTGGTGGACGATCTT-3 and ( ii ) downstream primer 5 - TCAGGTATTTAGCGTTCCGTG-3 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19136587" target="_blank">19136587</a>]<br></li>
<li>Cra DNA-binding assays were performed on a 239-bp DNA fragment containing three putative Cra binding sites or a 330-bp positive control fragment containing the icd promoter region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19136587" target="_blank">19136587</a>]<br></li>
				</div></ul><br>
			<label class="collapse_RP" for="_4"><font size="5px">Processes regulated by Cra</font></label>
				<input id="_4" type="checkbox">
				<div class="RP"><br><ul><li>The abovementioned glucose-effect on PiscR-lacZ expression depended on a functional Cra protein ( Table 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19136587" target="_blank">19136587</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_2">
Perrenoud A , Sauer U : Impact of global transcriptional regulation by ArcA , ArcB , Cra , Crp , Cya , Fnr , and Mlc on glucose-catabolism in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24044554" target="_blank">24044554</a>]<br></label>
<input id="rp_similarity_2", type="checkbox">
<div class="RP_similar">
<li>04553.x Perrenoud A , Sauer U ( 2005 ) Impact of global transcriptional regulation by ArcA , ArcB , Cra , Crp , Fnr , and Mlc on glucose-metabolism in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18563621" target="_blank">18563621</a>]<br></li></div>
<li>Finally , studies of TCSs such as PhoP/PhoQ , Cra , and KdpD/KdpE suggest that other central homeostasis bacterial regulators will also prove to have various adaptive roles in virulence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23555240" target="_blank">23555240</a>]<br></li>
<li>KdpE interacts with Cra , which itself coordinates the E. coli response to sugar availability , thereby linking metabolism to pathogenesis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23555240" target="_blank">23555240</a>]<br></li>
<li>In S. typhimurium strain LJ705 the structural genes for EI and HPr have been deleted , and the fruR gene encoding the fructose repressor protein FruR , or Cra , has been disrupted by a Tn10 insertion ; this strain also expresses the normally cryptic EI-like protein EIFru ( EI * ) , which is part of the fru regulon ( Chin et al. , 1987 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12480889" target="_blank">12480889</a>]<br></li>
<li>When interacting with the genes it regulates , i.e. , in the presence of gluconeo-genic substrates ( 18 ) , the Cra protein positively regulates transcription of those genes encoding biosynthetic and oxidative enzymes ( e.g. , key enzymes in the tricarboxylic-acid-cycle , the glyoxylate-bypass , the gluconeogenic pathway , and electron transport ) and negatively regulates transcription of genes encoding glycolytic enzymes , e.g. , key enzymes in the EmbdenMeyerhof and Entner-Doudoroff pathways ( 18 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10816546" target="_blank">10816546</a>]<br></li>
<li>Cra regulates numerous genes involved in carbon and energy metabolism and is believed to regulate the direction of carbon ﬂow ( 28 , 32 ) , but to our knowledge , there were no reports in the literature linking Cra to the regulation of iscRSUA operon expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19136587" target="_blank">19136587</a>]<br></li>
<li>Involvement of the Cra global regulatory protein in the expression of the iscRSUA operon , revealed during studies of tricarballylate catabolism in Salmonella enterica .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22101844" target="_blank">22101844</a>]<br></li>
<li>This is , however , the ﬁrst evidence of a regulator , other than IscR , involved in the regulation of the iscRSUA operon and the ﬁrst evidence that the assembly of [ Fe-S ] clusters is activated in response to sources of carbon and energy whose catabolism is under the control of Cra .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19136587" target="_blank">19136587</a>]<br></li>
<li>Here , evidence is presented that the mutation in SR-11 Fad2 that renders the strain avirulent is in the cra gene , which encodes the Cra protein , a regulator of central carbon metabolism .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10816546" target="_blank">10816546</a>]<br></li>
<li>and the catabolite repressor -RCB- activator ( Cra ) protein ( Saier & Ramseier , 1996 ; Saier et al. , 1996 ) , nitrogen-utilization is controlled by the NtrBC sensor kinase -RCB- response regulator pair in conjunction with the GlnB and GlnD proteins ( Magasanik , 1996 ; Reitzer , 1996a , b ) , phosphorus-utilization is controlled by the PhoR -RCB- PhoB sensor kinase -RCB- response regulator pair ( Wanner , 1996 ) , iron utilization is regulated by the Fur transcription factor ( Earhart , 1996 ; Gralla & Collado-Vides , 1996 ) and sulfur-metabolism is controlled by the CysB transcriptional activator ( Kredich , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11782505" target="_blank">11782505</a>]<br></li>
<li>Under the conditions investigated , it is assumed that carbon flux is controlled by the catabolite repressor/activator ( Cra ) protein , which activates gluconeogenesis enzymes ( ppsA , pckA , fbp ) and represses sugar catabolism enzymes [ 61 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20676398" target="_blank">20676398</a>]<br></li>
<li>The Cra protein is a regulator of central carbon metabolism ( 18 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10816546" target="_blank">10816546</a>]<br></li>
<li>Regulation of expression of the ethanol dehydrogenase gene ( adhE ) in Escherichia coli by catabolite repressor activator protein Cra .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19389776" target="_blank">19389776</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_16">
The catabolite repressor -RCB- activator ( Cra ) protein of enteric bacteria .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11782505" target="_blank">11782505</a>]<br></label>
<input id="rp_similarity_16", type="checkbox">
<div class="RP_similar">
<li>The catabolite repressor / activator ( Cra ) protein of enteric bacteria .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19136587" target="_blank">19136587</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10816546" target="_blank">10816546</a>]<br></li></div>
				</div></ul><br>
			<label class="collapse_SIT" for="_5"><font size="5px">Cra binding site</font></label>
				<input id="_5" type="checkbox">
				<div class="SIT"><br><ul><li>[ 89 ] BasR/PmrA Escherichia coli K12 TTAANNTT ( N ) TTAANNTT 3 Cra Escherichia coli K12 RSTGAAWCSNTHHW CRP Escherichia coli K12 AAATGTGATCTAGATCACATTT CsgD Escherichia coli K12 CGGGKGAKNKA ‡ Escherichia coli K12 AAAAGNGNNAAAWW ‡ Salmonella enterica serovar Typhimurium ATCC14028 YGGGTGAGTYM § HGGGNDRNBNN § CpxR Escherichia coli K12 GTAA ( N5 -- 7 ) GTAA IHF Salmonella enterica serovar Typhimurium ATCC14028 WATCAA ( N4 ) TTR MlrA Escherichia coli K12 AAAATTGTACA ( 12N ) TGTACAATTTT OmpR Escherichia coli MG1655 TTTRWAYATWRDTTTACA ¶ Salmonella enterica serovar Typhimurium ATCC14028 ACNTTTNGNTACANNTAT RcdA Escherichia coli K12 TTGTNNACA RstA Escherichia coli K12 TACATNTNGTTACA † The consensus sequences are written in International Union of Pure and Applied Chemistry ( IUPAC ) nucleotide code .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25437188" target="_blank">25437188</a>]<br></li>
<li>Whether Cra binds to site 1 and/or 2 remains an open question ACKNOWLEDGMENTS This work was supported by PHS grant R01-GM62203 to J.C.E.-S .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19136587" target="_blank">19136587</a>]<br></li>
<li>Potential Cra binding sites are underlined .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19136587" target="_blank">19136587</a>]<br></li>
<li>Cra binding sites are numbered .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19136587" target="_blank">19136587</a>]<br></li>
<li>Cra is known to bind to at least one nonconsensus binding site within the cydAB promoter ( 29 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19136587" target="_blank">19136587</a>]<br></li>
<li>The promoter of iscR ( bottom sequence ) was aligned with the consensus Cra box ( top sequence ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19136587" target="_blank">19136587</a>]<br></li>
<li>The weak binding of Cra to the iscR promoter was not surprising given the lack of a consensus sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19136587" target="_blank">19136587</a>]<br></li>
<li>A 239-bp PCR product containing the three putative Cra binding sites in the iscR promoter region was ampliﬁed using the following primers : ( i ) upstream primer 5 - TATCCGCTGGTGGACGATCTT-3 and ( ii ) downstream primer 5 - TCAGGTATTTAGCGTTCCGTG-3 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19136587" target="_blank">19136587</a>]<br></li>
<li>Putative Cra binding sites within the promoter region of iscR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19136587" target="_blank">19136587</a>]<br></li>
<li>If the effect of Cra were direct , Cra would bind to one or more nonconsensus sites within the iscR region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19136587" target="_blank">19136587</a>]<br></li>
				</ul></div><br>
			<label class="collapse_TU" for="_6"><font size="5px">Transcription units regulated by Cra</font></label>
				<input id="_6" type="checkbox">
				<div class="TU"><br><ul><li>Although the O1-site downstream of the transcriptional start site appears to be the most important for csgD expression , binding of Cra to operators O2 and O3 located upstream commonly appears to be important for maximal transcription [ 90 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25437188" target="_blank">25437188</a>]<br></li>
<li>Under the conditions investigated , it is assumed that carbon flux is controlled by the catabolite repressor/activator ( Cra ) protein , which activates gluconeogenesis enzymes ( ppsA , pckA , fbp ) and represses sugar catabolism enzymes [ 61 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20676398" target="_blank">20676398</a>]<br></li>
<li>Regulation of expression of the ethanol dehydrogenase gene ( adhE ) in Escherichia coli by catabolite repressor activator protein Cra .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19389776" target="_blank">19389776</a>]<br></li>
<li>Although it is possible that Cra activates iscRSUA expression , the data in Fig. 4 show that expression of the operon in the absence of Cra is sufﬁcient to support growth on Tcb .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19136587" target="_blank">19136587</a>]<br></li>
				</div></ul></div><br>
	</div>
</body>
</html>