<!doctype html>
<style>
@charset "utf-8";
body {
	height: 100%;
	margin: 0;
	padding: 0;
}

.banner {
	position: fixed;
	top: 0;
	width: 100%;
	height: 65px;
	background: gray;
	line-height: 65px;
}
.spacer {
	height: 10px;
}

.main-content {
	padding: 5%;
}

.collapse_ACT {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-style: none solid none none;
	border-color: #B4045F;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_ACT + input{
	display: none;
}
.collapse_ACT + input + div{
	display: none;
}
.collapse_ACT + input:checked + div{
	display: block;
}
.ACT {
	text-align: justify;
	text-justify: inter-word;
}
.ACT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B4045F;
}
.ACT_collapse_similarity:hover{
	background: #e5d3dc;
}
.ACT_collapse_similarity + input {
	display: none;
}
.ACT_collapse_similarity + input + div{
	display: none;
}

.ACT_collapse_similarity + input:checked + div{
	display: block;
}
.ACT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.ACT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B4045F;
	padding-left: 5px;
}
.ACT_collapse_similarity:hover{
	background: #e5d3dc;
}
.ACT_collapse_similarity + input {
	display: none;
}
.ACT_collapse_similarity + input + div{
	display: none;
}
.ACT_collapse_similarity + input:checked + div{
	display: block;
}
.ACT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.ACT_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #B4045F;
}
.ACT_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_DOM {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-style: none solid none none;
	border-color: #B404AE;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_DOM + input{
	display: none;
}
.collapse_DOM + input + div{
	display: none;
}
.collapse_DOM + input:checked + div{
	display: block;
}
.DOM {
	text-align: justify;
	text-justify: inter-word;
}
.DOM_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B404AE;
	padding-left: 5px;
}
.DOM_collapse_similarity:hover{
	background: #e8d7e7;
}
.DOM_collapse_similarity + input {
	display: none;
}
.DOM_collapse_similarity + input + div{
	display: none;
}
.DOM_collapse_similarity + input:checked + div{
	display: block;
}
.DOM_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}.DOM_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #B404AE;
}
.DOM_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_EVO {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #7401DF;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_EVO + input{
	display: none;
}
.collapse_EVO + input + div{
	display: none;
}
.collapse_EVO + input:checked + div{
	display: block;
}
.EVO {
	text-align: justify;
	text-justify: inter-word;
}
.EVO_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #7401DF;
	padding-left: 5px;
}
.EVO_collapse_similarity:hover{
	background: #e2d7ed;
}
.EVO_collapse_similarity + input {
	display: none;
}
.EVO_collapse_similarity + input + div{
	display: none;
}
.EVO_collapse_similarity + input:checked + div{
	display: block;
}
.EVO_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.EVO_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #7401DF;
}
.EVO_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_RP {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #0174DF;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_RP + input{
	display: none;
}
.collapse_RP + input + div{
	display: none;
}
.collapse_RP + input:checked + div{
	display: block;
}
.RP {
	text-align: justify;
	text-justify: inter-word;
}
.RP_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #0174DF;
	padding-left: 5px;
}
.RP_collapse_similarity:hover{
	background: #c9d7e2;
}
.RP_collapse_similarity + input {
	display: none;
}
.RP_collapse_similarity + input + div{
	display: none;
}
.RP_collapse_similarity + input:checked + div{
	display: block;
}
.RP_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.RP_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #0174DF;
}
.RP_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_SIT {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #0489B1;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_SIT + input{
	display: none;
}
.collapse_SIT + input + div{
	display: none;
}
.collapse_SIT + input:checked + div{
	display: block;
}
.SIT {
	text-align: justify;
	text-justify: inter-word;
}
.SIT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #0489B1;
	padding-left: 5px;
}
.SIT_collapse_similarity:hover{
	background: #d0dfe2;
}
.SIT_collapse_similarity + input {
	display: none;
}
.SIT_collapse_similarity + input + div{
	display: none;
}
.SIT_collapse_similarity + input:checked + div{
	display: block;
}
.SIT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.SIT_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #0489B1;
}
.SIT_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_TU {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #04B486;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_TU + input{
	display: none;
}
.collapse_TU + input + div{
	display: none;
}
.collapse_TU + input:checked + div{
	display: block;
}
.TU {
	text-align: justify;
	text-justify: inter-word;
}
.TU_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #04B486;
	padding-left: 5px;
}
.TU_collapse_similarity:hover{
	background: #d0e2de;
}
.TU_collapse_similarity + input {
	display: none;
}
.TU_collapse_similarity + input + div{
	display: none;
}
.TU_collapse_similarity + input:checked + div{
	display: block;
}
.TU_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.TU_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #04B486;
}
.TU_similar > a {
	text-decoration: none;
	color: gray;
}

ul { 
	list-style-type: none;
	padding: 0;
	margin: 0;
}
ul li > a {
	text-decoration: none;
	color: gray;
}
ul li:hover, ul li:focus {
	background: #F0F0F0;
}

ul li:first-child span {
  border-left:none;  
}

</style>
<html>
<head>
<meta charset="utf-8">
<title>TF properties - DeoR</title>
<meta name="theme-color" content="#ffffff">
</head>
<body>
	<div class="banner">
		<center><font color="white" size="16px" face="arial">DeoR</font></center>
	</div>
	<div align="center" class="main-content">
		<div class="spacer">
	</div><br>			<label class="collapse_ACT" for="_1"><font size="5px">DeoR transcriptional activity</font></label>
				<input id="_1" type="checkbox">
				<div class="ACT"><br><ul><li>DeoR repression at-a-distance only weakly responds to changes in interoperator separation and DNA topology .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14526015" target="_blank">14526015</a>]<br></li>
<li>S. Typhimurium ST4/74 and its ST4/74 ΔgatR-HTH , ST4/74 ΔgatR-DeoR , and ST4/74 ΔgatC mutants were cultivated , and the OD600 was monitored as described in the legend to Fig. 1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27956522" target="_blank">27956522</a>]<br></li>
<li>The resulting four strains were grown in minimal medium with different carbon sources , and the levels of - galactosidase activity , to monitor expression of deoK/deoQ , and thymidine phosphorylase ( deoA ) activity , to monitor expression of the DeoR controlled deoCABD operon , were determined .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14526015" target="_blank">14526015</a>]<br></li>
<li>Since thymidine did not induce - galactosidase synthesis in the presence of both DeoQ and DeoR ( experiment 12 ) , it appeared that DeoR was subordinate to DeoQ in repression of deoK and deoQ expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14526015" target="_blank">14526015</a>]<br></li>
<li>This indicates that dRib5P , which is the true inducer of the DeoR regulon ( 15 ) , is not the endogenous inducer of DeoQ-mediated repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14526015" target="_blank">14526015</a>]<br></li>
<li>One promoter is regulated negatively by the DeoR repressor , and the other is regulated negatively by the CytR repressor and positively by the cAMP/CRP complex ( Bremer et al. 1988 ; Gerlach et al. 1990 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16489221" target="_blank">16489221</a>]<br></li>
<li>The deoKPX promoter was 10-fold stronger than the deoQ promoter , and expression was negatively regulated by DeoQ as well as by DeoR , the repressor of the deoxynucleoside catabolism operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14526015" target="_blank">14526015</a>]<br></li>
				</div></ul><br>
			<label class="collapse_DOM" for="_2"><font size="5px">DeoR protein domains</font></label>
				<input id="_2" type="checkbox">
				<div class="DOM"><br><ul><li>We also generated two partial deletion mutants lacking either the putative DNA-binding domain ( ΔgatR-HTH ) or the putative oligomerization domain ( ΔgatR-DeoR ) of GatR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27956522" target="_blank">27956522</a>]<br></li>
<li>Although DeoQ is similar to DeoR from both E. coli and Bacillus subtilis in appearing to exist as an octamer in solution ( 28 , 40 ) and containing a helix-turn-helix domain characteristic for DNA-binding proteins , it differs from the two DeoR proteins in that dRib5P is not the low-molecular-weight effector for the protein ( see above ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14526015" target="_blank">14526015</a>]<br></li>
<li>Therefore , this repressor , which belongs to the DeoR family of proteins and contains a helix-turn-helix ( HTH ) domain and a DeoR C-terminal sensor domain , was investigated here in more detail .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27956522" target="_blank">27956522</a>]<br></li>
				</div></ul><br>
			<label class="collapse_EVO" for="_3"><font size="5px">DeoR evolution</font></label>
				<input id="_3" type="checkbox">
				<div class="EVO"><br><ul><li>The DeoR family includes at least four non-orthologous groups of RhaR regulators that are characterized by distinct DNA motifs with a tandem repeat symmetry .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24391637" target="_blank">24391637</a>]<br></li>
<li>The signiﬁcant amino acid sequence identity between DeoQ and DeoR ( 36.3 % for serovar Typhimurium ) led us to look for possible cross-activity between the two repressors and their targets .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14526015" target="_blank">14526015</a>]<br></li>
<li>sty0448 encodes a protein of unknown function with signiﬁcant amino acid sequence similarity to the DeoR family of transcriptional regulatory proteins ( Hammer-Jespersen and Munch-Ptersen , 1975 ) ( Fig .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17555437" target="_blank">17555437</a>]<br></li>
<li>The deoQ gene was deduced to encode a putative regulatory protein with 37.4 % amino acid sequence identity to the DeoR repressor of E. coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14526015" target="_blank">14526015</a>]<br></li>
<li>Therefore , this repressor , which belongs to the DeoR family of proteins and contains a helix-turn-helix ( HTH ) domain and a DeoR C-terminal sensor domain , was investigated here in more detail .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27956522" target="_blank">27956522</a>]<br></li>
<li>This regulator , which we have named IgeR , is a member of the DeoR family of transcriptional regulatory proteins and is highly conserved in all S. enterica serovars .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17555437" target="_blank">17555437</a>]<br></li>
<li>Haghjoo and Galán have also previously shown that IgeR , which belongs to the DeoR family of transcription regulators , binds to the CdtB promoter and represses expression in extracellular bacteria [ 50 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19746165" target="_blank">19746165</a>]<br></li>
<li>We found that IgeR belongs to the DeoR family of transcription regulators and is widely distributed among other Salmo-nella enterica serovars .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17555437" target="_blank">17555437</a>]<br></li>
<li>Analysis of other taxonomic groups outside the γ-proteobacteria identiﬁed previously uncharacterized members of the LacI , DeoR , and AraC families as alternative transcriptional regulators of the L-rhamnose catabolic pathways ( Figure 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24391637" target="_blank">24391637</a>]<br></li>
<li>Amino acid sequence alignment of IgeR with members of the DeoR family of transcription regulators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17555437" target="_blank">17555437</a>]<br></li>
<li>A transposon mutagenesis screen identiﬁed IgeR , a transcriptional regulator belonging to the DeoR family of transcriptional regulators , as a repressor of cdtB transcription in S. Typhi [ 39 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27120620" target="_blank">27120620</a>]<br></li>
<li>IgeR belongs to the DeoR family of transcriptional repressor proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17555437" target="_blank">17555437</a>]<br></li>
<li>The ﬁrst belongs to the DeoR family of transcriptional regulators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23774596" target="_blank">23774596</a>]<br></li>
<li>It is intriguing that each of the segments ( i.e. HK , O and S ) has an ORF at one terminus which has similarity to a transcriptional regulator in the LysR , DeoR and MarR families ( Perez-Rueda & Collado-Vides , 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12177346" target="_blank">12177346</a>]<br></li>
<li>trifolii , a novel negative TF of the DeoR family has been implicated in control of the L-Rha utilization regulon , which contains two divergently transcribed operons , rhaRSTPQUK and rhaDI , encoding an ABC transporter for L-Rha uptake ( RhaSTPQ ) , an alternative kinase ( RhaK , 19 % identity to RhaB from E. coli ) , an isomerase ( RhaI ) , and a mutarotase ( RhaU , 41 % identity to RhaM from E. coli ) ( Richardson et al. , 2004 , 2008 ; Richardson and Oresnik , 2007 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24391637" target="_blank">24391637</a>]<br></li>
				</div></ul><br>
			<label class="collapse_RP" for="_4"><font size="5px">Processes regulated by DeoR</font></label>
				<input id="_4" type="checkbox">
				<div class="RP"><br><ul><li>Classiﬁcation of annotated iron-responsive promoters with respect to Fur regulation Maximum fold inductiona Function ( s ) Classe Gene Reference ( s ) iroBc , d A 3 , 4 8,900 Putative glycosyl transferase , related to UDP-glucuronosyltransferase/putative ATP binding cassette ( ABC ) transporter Isochorismate synthetase , enterochelin-biosynthesis Outer membrane porin , receptor for colicin I , requires TonB Outer membrane porin , receptor for ferric enterobactin ( enterochelin ) and colicins B and D Putative hemin uptake factor Ferric hydroxamate transport , involved in reduction of ferric iron in cytoplasmic ferrioxamine B Salmonella iron-transporter Putative DedA family , membrane protein Regulatory or redox component complexing with Bfr in iron storage and mobility Putative iron chelator utilization protein Putative transcription regulator ; AraC family Putative anaerobic dimethyl sulfoxide reductase , subunit A Putative periplasmic protein Energy transducer : uptake of iron , cyanocobalamin : sensitivity to phages , colicins Outer membrane protein receptor or transporter for ferrichrome , colicin M , and phages T1 , T5 , and phi80 ATP-dependent protease , Hsp 100 , part of novel multichaperone system with DnaK , DnaJ , and GrpE Cu ( I ) - translocating P-type ATPase Putative hemolysin Phosphoribosylpyrophosphate synthetase DNA polymerase [ III ] , chi subunit Invasion protein Putative cytoplasmic protein O-antigen ligase MFS superfamily , nitrite extrusion protein Putative molybdopterin oxidoreductase UDP-D-glucose : ( galactosyl ) lipopolysaccharide glucosyltransferase Putative cytochrome c peroxidase Putative aminotransferase in cobalamin-synthesis Propanediol utilization : polyhedral bodies Propanediol utilization : propanol dehydrogenase entCc , d cirAc fepAc , d 1,805 620 569b 6 , 7 , 26 17 , 18 , 20 , 53 47 ydiEd fhuFc 353 241 38 19 , 35 , 36 sitAc , d yqjA bfd 142b 86 81b 54 32 32 , 37 58b 55b 52b 46 22 yajHd 38 32 STM0363d STM4305 STM1586 tonBc 32 32 20 , 40 fhuAc , d 21 20 , 47 20b clpB 32 copA yafA 12 11b 32 32 32 32 32 32 32 32 32 32 32 32 32 32 2b 3 6 6b 7 17 26b 28 29b 36 274b 363b prsA holC invH ytfK rfaL narKd fdnG rfaJ STM3820 cobD pduA pduQ 42b Iron-dependent alcohol dehydrogenase of the multifunctional alcohol dehydrogenase AdhE Cytoplasmic trehalase Xanthosine phosphorylase ( purine nucleoside phosphorylase ) Putative thioredoxin-like proteins and domain Putative coiled-coil protein Membrane-associated protein of unknown function Fimbrial protein Z , putative transcriptional regulator ( LuxR/UhpA family ) Pyruvate kinase 1 ( formerly F ) , fructose stimulated Putative DNA/RNA nonspeciﬁc endonuclease Putative cytoplasmic protein Fumarase A ( fumarate hydratase class I ) , aerobic isozyme Putative cytoplasmic protein Soluble pyridine nucleotide transhydrogenase Secreted effector protein of Salmonella serovar Dublin Putative inner membrane protein Paral putative oxidoreductase Surface presentation of antigens : secretory proteins Putative phosphatase L-Rhamnose isomerase Molybdopterin-biosynthesis , protein A Ferritin-like protein Putative peptidase T ( aminotripeptidase ) MukF protein ( killing factor KicB ) Fumarate reductase , anaerobic , ﬂavoprotein subunit sn-glycerol-3-phosphate dehydrogenase ( anaerobic ) , large subunit Putative aspartate racemase Anaerobic dimethyl sulfoxide reductase , subunit A B adhE 32 treF xapAd 15 13 32 32 32 32 32 32 32 32 32 30 32 32 32 32 32 32 32 32 32 32 8b 4b 2b 4 4 4 4b 5b 5 5 6 9 9 10 10 16b 19 21 24 24 28 30b 59b 88 yhgl STM1554 ycfC ﬁmZ pykF STM1330 yhcGd fumAc STM1239 udhA sopA yhfK ysaA sicA STM3595 rhaA moaAd ftnB pepT STM0989 frdAd glpA STM4510 dmsAd 32 32 32 32 32 Continue TABLE 2 -- Continued Maximum fold inductiona Classe Gene Function ( s ) Reference ( s ) yqhC Putative transcriptional regulator ( AraC/XylS family ) 32 104 yfcZ Putative cytoplasmic protein 32 STM1851 Putative cytoplasmic protein 32 yhcO Putative cytoplasmic protein 32 STM4448 Putative periplasmic protein/putative phosphotransferase system mannitol / 32 fructose-speciﬁc IIA domain ( Ntr type ) yciG 6 Putative cytoplasmic protein 32 ydhI 4 Putative inner membrane protein 32 nagB 3 Glucosamine-6-phosphate deaminase 32 narW 3 Nitrate reductase 2 , delta subunit , assembly function 32 STM1537 4b Putative Ni/Fe-hydrogenase 1 b-type cytochrome subunit 32 STM4067 4 Putative ADP-ribosylglycohydrolase 32 cybC 5b Cytochrome b ( 562 ) 32 yeaA 6b Putative domain frequently associated with peptide methionine sulfoxide 32 reductase nirB 7b Nitrite reductase , large subunit 32 yfbE 8 LexA regulated , putative SOS-response 32 sifB 9 Salmonella translocated effector : translocated by SP1 -- 2 32 yjdE 10 Putative APC family , putrescine/ornithine-transport protein , cryptic 32 yliG 10 Putative Fe-S oxidoreductase family 1 32 ais 12 Aluminum-inducible protein 32 yﬁR 14 Putative periplasmic protein 32 aspAd 15b Aspartate ammonia-lyase ( aspartase ) 32 STM1255 15b Putative ABC transporter periplasmic binding protein 32 fucR 22b Positive regulator of the fuc operon ( DeoR family ) 32 STM3138d 23 Putative methyl-accepting chemotaxis protein 32 araE 28 MFS family , L-arabinose : proton symport protein ( low-afﬁnity transporter ) 32 eutS 28 Putative carboxysome structural protein , ethanol utilization 32 yhbU 29 Putative protease 32 STM4519 41b Putative NAD-dependent aldehyde dehydrogenase 32 mgsA 78b Methylglyoxal synthase 32 a Indicates approximate maximum induction in LB medium between high-iron ( positive numbers ) and low-iron ( negative numbers ) conditions 1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12897017" target="_blank">12897017</a>]<br></li>
<li>These regulators act as repressors in sugar me-tabolism , but further study has revealed that some DeoR regulators are involved in the cell response to environmental stress , such as superoxide stress and osmotic upshift ( 54 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23774596" target="_blank">23774596</a>]<br></li>
<li>trifolii , a novel negative TF of the DeoR family has been implicated in control of the L-Rha utilization regulon , which contains two divergently transcribed operons , rhaRSTPQUK and rhaDI , encoding an ABC transporter for L-Rha uptake ( RhaSTPQ ) , an alternative kinase ( RhaK , 19 % identity to RhaB from E. coli ) , an isomerase ( RhaI ) , and a mutarotase ( RhaU , 41 % identity to RhaM from E. coli ) ( Richardson et al. , 2004 , 2008 ; Richardson and Oresnik , 2007 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24391637" target="_blank">24391637</a>]<br></li>
<li>Using bioinformatics analyses of L-Rha utilization genes , we identiﬁed twelve groups of rhamnose-related transcriptional regulators from different protein families , AraC , DeoR , and LacI , and proposed binding site motifs for these regulators within tentatively reconstructed regulons ( Figure S5 in Supplementary Material ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24391637" target="_blank">24391637</a>]<br></li>
				</div></ul><br>
			<label class="collapse_SIT" for="_5"><font size="5px">DeoR binding site</font></label>
				<input id="_5" type="checkbox">
				<div class="SIT"><br><ul><li>The DeoR family includes at least four non-orthologous groups of RhaR regulators that are characterized by distinct DNA motifs with a tandem repeat symmetry .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24391637" target="_blank">24391637</a>]<br></li>
<li>The operators recognized by DeoR and CytR are present in the sequence of the S. Typhi tsx promoter , and the deoR and cytR genes are conserved in the S. Typhi genome sequence ( Parkhill et al. 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16489221" target="_blank">16489221</a>]<br></li>
<li>All four sites showed signiﬁcant similarity with the consensus DeoR binding site ( NTGTN10 ACAN [ 14 ] ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14526015" target="_blank">14526015</a>]<br></li>
				</ul></div><br>
			<label class="collapse_TU" for="_6"><font size="5px">Transcription units regulated by DeoR</font></label>
				<input id="_6" type="checkbox">
				<div class="TU"><br><ul><li>The deoKPX promoter was 10-fold stronger than the deoQ promoter , and expression was negatively regulated by DeoQ as well as by DeoR , the repressor of the deoxynucleoside catabolism operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14526015" target="_blank">14526015</a>]<br></li>
<li>Initiation of transcription from each of the promoters is regulated negatively both by the DeoQ repressor and by DeoR , the repressor of the deoxynucleoside catabolic deoCABD operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14526015" target="_blank">14526015</a>]<br></li>
<li>The E. coli DeoR repressor was able to repress both deoK and deoQ expression to about the same extent as DeoQ ( experiment 7 versus experiments 2 and 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14526015" target="_blank">14526015</a>]<br></li>
<li>Since thymidine did not induce - galactosidase synthesis in the presence of both DeoQ and DeoR ( experiment 12 ) , it appeared that DeoR was subordinate to DeoQ in repression of deoK and deoQ expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14526015" target="_blank">14526015</a>]<br></li>
<li>It is noteworthy that even though DeoR was capable of repressing expression of deoK , deoQ , and deoA ( experiment 7 ) , the reciprocal was not observed ( experiment 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14526015" target="_blank">14526015</a>]<br></li>
				</div></ul></div><br>
	</div>
</body>
</html>