<!doctype html>
<style>
@charset "utf-8";
body {
	height: 100%;
	margin: 0;
	padding: 0;
}

.banner {
	position: fixed;
	top: 0;
	width: 100%;
	height: 65px;
	background: gray;
	line-height: 65px;
}
.spacer {
	height: 10px;
}

.main-content {
	padding: 5%;
}

.collapse_ACT {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-style: none solid none none;
	border-color: #B4045F;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_ACT + input{
	display: none;
}
.collapse_ACT + input + div{
	display: none;
}
.collapse_ACT + input:checked + div{
	display: block;
}
.ACT {
	text-align: justify;
	text-justify: inter-word;
}
.ACT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B4045F;
}
.ACT_collapse_similarity:hover{
	background: #e5d3dc;
}
.ACT_collapse_similarity + input {
	display: none;
}
.ACT_collapse_similarity + input + div{
	display: none;
}

.ACT_collapse_similarity + input:checked + div{
	display: block;
}
.ACT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.ACT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B4045F;
	padding-left: 5px;
}
.ACT_collapse_similarity:hover{
	background: #e5d3dc;
}
.ACT_collapse_similarity + input {
	display: none;
}
.ACT_collapse_similarity + input + div{
	display: none;
}
.ACT_collapse_similarity + input:checked + div{
	display: block;
}
.ACT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.ACT_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #B4045F;
}
.ACT_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_DOM {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-style: none solid none none;
	border-color: #B404AE;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_DOM + input{
	display: none;
}
.collapse_DOM + input + div{
	display: none;
}
.collapse_DOM + input:checked + div{
	display: block;
}
.DOM {
	text-align: justify;
	text-justify: inter-word;
}
.DOM_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B404AE;
	padding-left: 5px;
}
.DOM_collapse_similarity:hover{
	background: #e8d7e7;
}
.DOM_collapse_similarity + input {
	display: none;
}
.DOM_collapse_similarity + input + div{
	display: none;
}
.DOM_collapse_similarity + input:checked + div{
	display: block;
}
.DOM_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}.DOM_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #B404AE;
}
.DOM_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_EVO {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #7401DF;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_EVO + input{
	display: none;
}
.collapse_EVO + input + div{
	display: none;
}
.collapse_EVO + input:checked + div{
	display: block;
}
.EVO {
	text-align: justify;
	text-justify: inter-word;
}
.EVO_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #7401DF;
	padding-left: 5px;
}
.EVO_collapse_similarity:hover{
	background: #e2d7ed;
}
.EVO_collapse_similarity + input {
	display: none;
}
.EVO_collapse_similarity + input + div{
	display: none;
}
.EVO_collapse_similarity + input:checked + div{
	display: block;
}
.EVO_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.EVO_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #7401DF;
}
.EVO_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_RP {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #0174DF;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_RP + input{
	display: none;
}
.collapse_RP + input + div{
	display: none;
}
.collapse_RP + input:checked + div{
	display: block;
}
.RP {
	text-align: justify;
	text-justify: inter-word;
}
.RP_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #0174DF;
	padding-left: 5px;
}
.RP_collapse_similarity:hover{
	background: #c9d7e2;
}
.RP_collapse_similarity + input {
	display: none;
}
.RP_collapse_similarity + input + div{
	display: none;
}
.RP_collapse_similarity + input:checked + div{
	display: block;
}
.RP_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.RP_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #0174DF;
}
.RP_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_SIT {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #0489B1;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_SIT + input{
	display: none;
}
.collapse_SIT + input + div{
	display: none;
}
.collapse_SIT + input:checked + div{
	display: block;
}
.SIT {
	text-align: justify;
	text-justify: inter-word;
}
.SIT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #0489B1;
	padding-left: 5px;
}
.SIT_collapse_similarity:hover{
	background: #d0dfe2;
}
.SIT_collapse_similarity + input {
	display: none;
}
.SIT_collapse_similarity + input + div{
	display: none;
}
.SIT_collapse_similarity + input:checked + div{
	display: block;
}
.SIT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.SIT_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #0489B1;
}
.SIT_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_TU {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #04B486;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_TU + input{
	display: none;
}
.collapse_TU + input + div{
	display: none;
}
.collapse_TU + input:checked + div{
	display: block;
}
.TU {
	text-align: justify;
	text-justify: inter-word;
}
.TU_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #04B486;
	padding-left: 5px;
}
.TU_collapse_similarity:hover{
	background: #d0e2de;
}
.TU_collapse_similarity + input {
	display: none;
}
.TU_collapse_similarity + input + div{
	display: none;
}
.TU_collapse_similarity + input:checked + div{
	display: block;
}
.TU_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.TU_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #04B486;
}
.TU_similar > a {
	text-decoration: none;
	color: gray;
}

ul { 
	list-style-type: none;
	padding: 0;
	margin: 0;
}
ul li > a {
	text-decoration: none;
	color: gray;
}
ul li:hover, ul li:focus {
	background: #F0F0F0;
}

ul li:first-child span {
  border-left:none;  
}

</style>
<html>
<head>
<meta charset="utf-8">
<title>TF properties - FIS</title>
<meta name="theme-color" content="#ffffff">
</head>
<body>
	<div class="banner">
		<center><font color="white" size="16px" face="arial">FIS</font></center>
	</div>
	<div align="center" class="main-content">
		<div class="spacer">
	</div><br>			<label class="collapse_ACT" for="_1"><font size="5px">FIS transcriptional activity</font></label>
				<input id="_1" type="checkbox">
				<div class="ACT"><br><ul><li>AA3YVYGCTR - [ 1DEDYL YKlKEI MEE P D Y 802 - KRE C K , FHTL - T G SE ; RSIjT 230 - - - - - - - - - - - - - - - AFIKLWYV -- DPEAWD 845 DLKV VV - E KEGWA N  L DIDIRIY  -164 --------------- RGKH VILTLDEG EA 511 MS L WS R - - - - KKI DKIREQGAELIW TJVIAIFIS R [ I - - - - A KIV-YVHLM 589 LT KRDREgLW-226 NF F VS R TN E KGEM - Y TRMAQYAVELW ¬ 821 GRRR D L AG T ; - DFGT 247 EA - E. .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1704886" target="_blank">1704886</a>]<br></li>
<li>Resistance profiles : AMP , ampicillin ; CHL , chloramphenicol ; STR , streptomycin ; FIS , sulfisoxazole ; TET , tetracycline ; AUG2 , amoxicillin -- clavulanic acid ; XNL , ceftiofur ; AXO , ceftriaxone ; FOX , cefoxitin ; KAN , kanamycin ; SXT , trimethoprim-sulfamethoxazole ; NAL , nalidixic acid .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23127704" target="_blank">23127704</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23127704" target="_blank">23127704</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_3">
Solutions of CVM N18534 S Susceptible Chicken breast FDA-CVM AMP , CHL , STR , FIS , TET AMP , CHL , STR , FIS , TET AMP , CHL , STR , FIS , TET AMP , CHL , STR , FIS , TET FSL R6-215 FSL R8-2540 CVM N6431 CVM 30662 MDR MDR MDR MDR Human Human Cornell Cornell Chicken breast Chicken breast FDA-CVM FDA-CVM AMP , CHL , STR , FIS , TET , MDR-AmpC AUG2 , XNL , AXO , FOX , KAN AMP , CHL , STR , FIS , TET , MDR-AmpC AUG2 , XNL , AXO , FOX AMP , CHL , STR , FIS , TET , AUG2 , XNL , AXO , FOX , SXT , KAN AMP , CHL , STR , FIS , TET , AUG2 , XNL , AXO , FOX , NAL , KAN FSL S5-786 Bovine Cornell CVM N176 Chicken breast FDA-CVM CVM 33831 MDR-AmpC Cattle FDA-CVM CVM 30034 Typhimurium var .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23127704" target="_blank">23127704</a>]<br></label>
<input id="act_similarity_3", type="checkbox">
<div class="ACT_similar">
<li>O :52 ( Copenhagen ) Typhimurium S Susceptible CVM N18534 S Susceptible Chicken breast FDA-CVM AMP , CHL , STR , FIS , TET AMP , CHL , STR , FIS , TET AMP , CHL , STR , FIS , TET AMP , CHL , STR , FIS , TET FSL R6-215 FSL R8-2540 CVM N6431 CVM 30662 MDR MDR MDR MDR Human Human Cornell Cornell Chicken breast Chicken breast FDA-CVM FDA-CVM AMP , CHL , STR , FIS , TET , MDR-AmpC AUG2 , XNL , AXO , FOX , KAN AMP , CHL , STR , FIS , TET , MDR-AmpC AUG2 , XNL , AXO , FOX AMP , CHL , STR , FIS , TET , AUG2 , XNL , AXO , FOX , SXT , KAN AMP , CHL , STR , FIS , TET , AUG2 , XNL , AXO , FOX , NAL , KAN FSL S5-786 Bovine Cornell CVM N176 Chicken breast FDA-CVM CVM 33831 MDR-AmpC Cattle FDA-CVM CVM 30034 Typhimurium var .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23127704" target="_blank">23127704</a>]<br></li></div>
<li>At all loci tested , FIS occupancy decreased when cells were treated with novobiocin .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22457642" target="_blank">22457642</a>]<br></li>
<li>For example , both proteins influence DNA supercoiling ; FIS represses gyrB [ 35 ] , whereas RpoS transcribes gyrB [ 36 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24358360" target="_blank">24358360</a>]<br></li>
<li>All cultures were inoculated 1/100 from an overnight culture grown under aerating conditions to ensure a starting culture which was negative for ﬁs transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17784910" target="_blank">17784910</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17784910" target="_blank">17784910</a>]<br></li>
<li>Resistance to antimicrobial agents other than ciprofloxacin : Cot , trimethoprim -- sulfamethoxazole ; Fis , sulfamethoxazole or sulfisoxazole ; Nal , nalidixic acid ; Tet , tetracycline ; Amp , ampicillin ; Chl , chloramphenicol ; Str , streptomycin .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21749779" target="_blank">21749779</a>]<br></li>
<li>This conﬁrmed that Fis acts as a transcriptional-repressor as well as an activator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15256548" target="_blank">15256548</a>]<br></li>
<li>Currently , csrAB , sirA-barA , ﬁs , ﬂiZ , fadD , ompR-envZ , fur , mlc , dsbA , rtsA , hilC , and hilD have all been shown to positively upregulate hilA expression ( 13 -- 15 , 22 -- 31 , 56 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25547794" target="_blank">25547794</a>]<br></li>
<li>enterica ARS # 576 Serovar Newport ; Aug Amp Fox Tio Chl Gen Kan Str Fis Tet ARS # 815 Serovar Newport ; Aug Amp Fox Tio Chl Gen Kan Str Fis Tet ARS # 852 Serovar Typhimurium DT104 ; Amp Chl Kan Str Fis Tet ARS # 886 Serovar Bardo ; Aug Amp Fox Tio Chl Kan Str Fis Tet Chen et al. ( 2010b ) , Lindsey et al. ( 2009 ) Chen et al. ( 2010b ) , Lindsey et al. ( 2009 ) Chen et al. ( 2010b ) , Lindsey et al. ( 2009 ) Chen et al. ( 2010b ) , Lindsey et al. ( 2009 ) Plasmid pSBardo-Kan pSe-Kan pUC19 This study plasmid isolated from ARS # 886 ; Kan This study plasmid isolated from ARS # 852 ; Kan Cloning vector ; Amp New England BioLabs a Antibiotic resistance abbreviation : Aug , amoxicillin-clavulanic acid ; Amp , ampicillin ; Fox , cefoxitin ; Tio , ceftiofur ; Chl , chloramphenicol ; Kan , kana mycin ; Str , streptomycin ; Fis , Sulﬁsoxazole ; Tet , tetracycline .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21324339" target="_blank">21324339</a>]<br></li>
<li>This is to be expected given that Fis is known to inﬂuence topoisomerase gene expression and to inﬂuence DNA topological transitions directly through its preference for binding to intermediately supercoiled molecules ( Schneider et al. , 1997 ; 1999 ; 2000 ; Weinstein-Fischer et al. , 2000 ; Keane and Dorman , 2003 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16999831" target="_blank">16999831</a>]<br></li>
<li>Positive regulators include BarA/SirA , FliZ , EnvZ/OmpR , FadD , CsrB , HilD and some nucleoid binding proteins ( Fis , HU ) , while Pag , HilE , Ams and the nucleoid binding proteins Hha and H-NS are negatively regulating hilA transcription ( Altier et al. 2000a , b Fahlen et al. 2000 ; Lucas et al. 2000 ; Wilson et al. 2001 ; Schechter et al. 2003 ; Baxter et al. 2003 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18563621" target="_blank">18563621</a>]<br></li>
<li>Its own gene , ﬁs , is among those that it represses ( Pratt et al. , 1997 ) , while it acts to upregulate many of the genes that encode components of the translation machinery of the cell ( Bosch et al. , 1990 ; Nilsson et al. , 1992 ; Ross et al. , 1990 ; Ball et al. , 1992 ; Ninnemann et al. , 1992 ; Walker et al. , 1999 ; Schneider et al. , 2003 ; Paul et al. , 2004 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17784910" target="_blank">17784910</a>]<br></li>
<li>These included the cold-shock-responsive hns gene previously shown to be activated by Fis ( Dersch et al. , 1994 ; Falconi et al. , 1996 ) , the hha gene whose product can form heteromeric complexes with H-NS and ( like H-NS ) regulates several virulence genes in response to temperature ( Madrid et al. , 2002 ; Nieto et al. , 2002 ) , and the stpA gene that encodes a paralogue of H-NS and can also form heteromers with it ( Deighan et al. , 2003 ; Free et al. , 2001 ; Johansson et al. , 2001 ; Williams et al. , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15256548" target="_blank">15256548</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_16">
Of these , much attention has been paid to the role of Fis , the factor for inversion stimulation , due to its role in controlling virulence gene expression in a variety of pathogens ( Falconi et al. , 2001 ; Goldberg et al. , 2001 ; Sheikh et al. , 2001 ; Wilson et al. , 2001 ; Schechter et al. , 2003 ; Prosseda et al. , 2004 ; Yoon et al. , 2003 ; Kelly et al. , 2004 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16999831" target="_blank">16999831</a>]<br></label>
<input id="act_similarity_16", type="checkbox">
<div class="ACT_similar">
<li>Given the role of Fis in regulating virulence genes in several pathogens ( Falconi et al. , 2001 ; Goldberg et al. , 2001 ; Sheikh et al. , 2001 ; Wilson et al. , 2001 ; Schechter et al. , 2003 ; Yoon et al. , 2003 ; Kelly et al. , 2004 ; Prosseda et al. , 2004 ) and the sensitivity of bacterial virulence gene promoters to changes in DNA supercoiling ( Dorman et al. , 1990 ; Galán and Curtiss , 1990 ; Dorman , 1991 ; Parsot and Mekalanos , 1992 ; Rohde et al. , 1994 ; 1999 ; GraeffWohlleben et al. , 1995 ; Tobe et al. , 1995 ; Falconi et al. , 1998 ; Leclerc et al. , 1998 ; Beltrametti et al. , 1999 ; Gmuender et al. , 2001 ; Bang et al. , 2002 ; Fournier and Klier , 2004 ) , we investigated the possibility that supercoiling and Fis might collaborate to modulate SPI-2 gene expression in S. Typhimurium during growth of the bacte-rium in murine macrophage .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16999831" target="_blank">16999831</a>]<br></li></div>
<li>Fis levels are very low or undetectable during very slow growth or during stationary phase but rapidly increase to over 25,000 to 50,000 dimers per cell when cells are shifted to a rich medium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7536730" target="_blank">7536730</a>]<br></li>
<li>For instance , Fis mRNA and protein levels are extremely low during stationary phase but rapidly increase over 500-fold during early logarithmic growth-phase ( 5 , 37 , 38 , 54 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9811652" target="_blank">9811652</a>]<br></li>
<li>In their absence , fis continues to be induced by nutritional upshift and repressed during stationary phase [ 4-6 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24358360" target="_blank">24358360</a>]<br></li>
<li>Keywords : hlyE , rpoS , phoP , cAMP-receptor protein , Fis protein , Catabolite repression Background The genus Salmonella includes two species , Salmonella bongori and Salmonella enterica .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24885225" target="_blank">24885225</a>]<br></li>
<li>The DNA fragments ( 1 nM ) were incubated with increasing concentrations of Fis protein ( 0 , 50 , 100 , 150 , 200 , 250 , 300 , 350 , 400 nM ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23717649" target="_blank">23717649</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23717649" target="_blank">23717649</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_23">
Wilson , R. , Libby , S. , Freet , A. , Boddicker , J. , Fahlen , T. , and Jones , B. ( 2001 ) Fis , a DNA nucleoid-associated protein , is involved in Salmonella typhimurium SPI-1 invasion gene expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15661008" target="_blank">15661008</a>]<br></label>
<input id="act_similarity_23", type="checkbox">
<div class="ACT_similar">
<li>[ 64 ] R.L. Wilson , S.J. Libby , A.M. Freet , J.D. Boddicker , T.F. Fahlen , B.D. Jones , Fis , a DNA nucleoid-associated protein , is involved in Salmonella typhimurium SPI-1 invasion gene expression , Mol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29146500" target="_blank">29146500</a>]<br></li>
<li>Wilson RL , Libby SJ , Freet AM , Boddicker JD , Fahlen TF , Jones BD : Fis , a DNA nucleoid-associated protein , is involved in Salmonella typhimurium SPI-1 invasion gene expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21418628" target="_blank">21418628</a>]<br></li></div>
<li>In contrast , Fis protein was readily detected in stationary-phase cultures that had grown under nonaerated conditions ( Fig. 1A , lane 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17784910" target="_blank">17784910</a>]<br></li>
<li>Next we tested how supercoiling responds to osmotic pressure in cells lacking FIS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21276095" target="_blank">21276095</a>]<br></li>
<li>RpoS protein is elevated in a ﬁs mutant .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15716427" target="_blank">15716427</a>]<br></li>
<li>When cells in stationary phase are batch cultured in a nutritionally rich medium , ﬁs mRNA levels increase more than 1,000-fold within the time required to initiate logarithmic cell division .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9811652" target="_blank">9811652</a>]<br></li>
<li>The rapid increase in Fis levels upon nutrient upshift is clearly beneﬁcial , since ﬁs mutant cells initiate growth in nutrient-rich medium more slowly than wild-type cells , particularly after a prolonged period in stationary phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7536730" target="_blank">7536730</a>]<br></li>
<li>In this work , we found that fis participates in hlyE repression especially in presence of glucose ( Figure 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24885225" target="_blank">24885225</a>]<br></li>
<li>Moreover , fis expression was observed to correlate directly with DNA supercoiling in non-aerated conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24358360" target="_blank">24358360</a>]<br></li>
<li>The intracellular Fis concentration varies extensively in response to nutritional conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14553938" target="_blank">14553938</a>]<br></li>
<li>A potential role for Fis in SPI-1 gene regulation could be one that is related to DNA topology in response to environmental changes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11123690" target="_blank">11123690</a>]<br></li>
<li>For example , whereas FIS is important for DNA supercoiling homeostasis during growth in well-aerated conditions , FIS appears to have a minimal role in low-oxygen conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21276095" target="_blank">21276095</a>]<br></li>
<li>Nutritional up-shift has been shown to activate fis expression , which is , in turn , increased in response to increased gyrase activity under this condition ( Schneider et al. 2000 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15235764" target="_blank">15235764</a>]<br></li>
<li>ﬁs expression in response to nutrient upshift .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7536730" target="_blank">7536730</a>]<br></li>
<li>Together , these data indicate that neither FIS nor DNA supercoiling levels can account for changes in PtopAEc and PtopASe expression in response to osmotic pressure , suggesting that other as yet unrecognized factors directly control PtopA expression in both species .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21276095" target="_blank">21276095</a>]<br></li>
<li>Sustained fis expression results in highly elevated FIS protein levels , which contribute directly to increased invasion of epithelial cells in in vitro assays [ 13 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24358360" target="_blank">24358360</a>]<br></li>
<li>This modest two-fold increase in expression corresponds to a dramatic rise in FIS protein level , which causes S. Typhimurium to become more invasive in a fis-dependent manner [ 13 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24358360" target="_blank">24358360</a>]<br></li>
<li>Fis repression was detected for all ﬁs promoters when the short constructs ( from 2168 or 2172 to 178 ) were assayed , although their regulation efﬁciencies varied .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9811652" target="_blank">9811652</a>]<br></li>
<li>This suggests that PtopASe is tuned to respond to a lower effective range of FIS concentration .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21276095" target="_blank">21276095</a>]<br></li>
<li>It is intriguing that PssrA induction does not require FIS in culture conditions that mimic the vacuolar environment [ 12 ] , nor is FIS required for PssrA induction when DNA is highly relaxed [ 20 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22457642" target="_blank">22457642</a>]<br></li>
<li>Although FIS contributes to transcriptional activation of these promoters , the finding that transcriptional activation occurs even when FIS is depleted suggests that FIS may act in part through its global control of DNA topology .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22457642" target="_blank">22457642</a>]<br></li>
<li>Because FIS transitions from a filamentous DNA-binding mode to an ordered dimer as its concentration decreases [ 24 ] , it is also possible that the depletion of FIS coupled with changes in DNA topology restructures FIS complexes into forms that favour transcription activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22457642" target="_blank">22457642</a>]<br></li>
<li>Abbreviations : Acn , aconitase ; cAcn , cytoplasmic aconitase ; CRP , cyclic-AMP receptor protein ; Fis , factor for inversion stimulation ; FNR , fumarate -- nitrate reduction regulator ; Fur , ferric uptake regulator ; IRE , iron-responsive element ; IRP-1 , iron-responsive protein 1 ; LB , Luria -- Bertani broth ; LIC , ligation-independent cloning ; PacnA , promoter region of the acnA gene ; PacnB , promoter region of the acnB gene ; SoxR ; superoxide response protein ; S. Typhimurium , Salmonella enterica serovar Typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23637460" target="_blank">23637460</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_47">
Keane , O.M. , and Dorman , C.J. ( 2003 ) The gyr genes of Salmonella enterica serovar Typhimurium are repressed by the factor for inversion stimulation , Fis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17784910" target="_blank">17784910</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16999831" target="_blank">16999831</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21276095" target="_blank">21276095</a>]<br></label>
<input id="act_similarity_47", type="checkbox">
<div class="ACT_similar">
<li>Keane OM , Dorman CJ ( 2003 ) The gyr genes of Salmonella enterica serovar Typhimurium are repressed by the factor for inversion stimulation , Fis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24358360" target="_blank">24358360</a>]<br></li>
<li>The gyr genes of Salmonella enterica serovar Typhimurium are repressed by the factor for inversion stimulation , Fis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15256548" target="_blank">15256548</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15716427" target="_blank">15716427</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15790293" target="_blank">15790293</a>]<br></li></div>
<li>The Escherichia coli Fis protein stimulates bacteriophage lambda integrative recombination in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15716427" target="_blank">15716427</a>]<br></li>
<li>Ball , C.A. , Osuna , R. , Ferguson , K.C. , and Johnson , R.C. ( 1992 ) Dramatic changes in Fis levels upon nutrient upshift in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11123690" target="_blank">11123690</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17784910" target="_blank">17784910</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_52">
[ 15 ] Ross , W. , Thompson , J.F. , Newlands , J.T. and Gourse , R.L. ( 1990 ) E. coli Fis protein activates ribosomal RNA transcription in vitro and in vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14553938" target="_blank">14553938</a>]<br></label>
<input id="act_similarity_52", type="checkbox">
<div class="ACT_similar">
<li>Ross , W. , Thompson , J.F. , Newlands , J.T. , and Gourse , R.L. ( 1990 ) E. coli Fis protein activates ribosomal RNA transcription in vitro and in vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17784910" target="_blank">17784910</a>]<br></li></div>
<li>Fis represses its mRNA levels by approximately 2.5-fold in S. typhimurium and by approximately 10-fold in E. coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7536730" target="_blank">7536730</a>]<br></li>
<li>When b-galacto-sidase activities measured in RZ211 and RJ1561 carrying different ﬁs promoter regions in plasmids pRJ1070 , pCK186 , and pMB258 were compared , negative regulation by Fis could be observed in each case ( Fig. 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9811652" target="_blank">9811652</a>]<br></li>
<li>The high-afﬁnity Fis sites III and IV in E coli , for example , which are either weakly protected or absent in S. typhimurium , may be directly or indirectly involved in achieving a more efﬁcient repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7536730" target="_blank">7536730</a>]<br></li>
<li>Osuna , R. , Lienau , D. , Hughes , K.T. , and Johnson , R.C. ( 1995 ) Sequence , regulation , and functions of ﬁs .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17784910" target="_blank">17784910</a>]<br></li>
<li>[ 7 ] R. Osuna , D. Lienau , K.T. Hughes , R.C. Johnson , Sequence , regulation , and function of ﬁs in Salmonella typhimurium , J. Bacteriol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20691712" target="_blank">20691712</a>]<br></li>
<li>Sequence , regulation , and functions of ﬁs in Salmonella typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16980465" target="_blank">16980465</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_60">
Osuna , R. , Lienau , D. , Hughes , K.T. , and Johnson , R.C. ( 1995 ) Sequence , regulation , and functions of fis in Salmonella typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11123690" target="_blank">11123690</a>]<br></label>
<input id="act_similarity_60", type="checkbox">
<div class="ACT_similar">
<li>Osuna , R. , Lienau , D. , Hughes , K.T. , and Johnson , R.C. ( 1995 ) Sequence , regulation , and functions of ﬁs in Salmo-nella typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21276095" target="_blank">21276095</a>]<br></li></div>
				<label class="ACT_collapse_similarity" for="act_similarity_62">
Kelly , A. , Goldberg , M. D. , Carroll , R. K. , Danino , V. , Hinton , J. C. , and Dorman , C. J. ( 2004 ) A global role for Fis in the transcriptional control of metabolism and type III secretion in Salmonella enterica serovar Typhi-murium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16684765" target="_blank">16684765</a>]<br></label>
<input id="act_similarity_62", type="checkbox">
<div class="ACT_similar">
<li>Kelly , A. , Goldberg , M.D. , Carroll , R.K. , Danino , V. , Hinton , J.C.D. , and Dorman , C.J. ( 2004 ) A global role for Fis in the transcriptional control of metabolism and type III secretion in Salmonella enterica serovar Typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17784910" target="_blank">17784910</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21276095" target="_blank">21276095</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16999831" target="_blank">16999831</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27206164" target="_blank">27206164</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24308841" target="_blank">24308841</a>]<br></li>
<li>Kelly , A. , Goldberg , M.D. , Carroll , R.K. , Danino , V. , Hinton , J.C. , and Dorman , C.J. ( 2004 ) A global role for Fis in the transcriptional control of metabolism and type III secretion in Salmonella enterica serovar Typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23190111" target="_blank">23190111</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17427289" target="_blank">17427289</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630976" target="_blank">17630976</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17163975" target="_blank">17163975</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18399940" target="_blank">18399940</a>]<br></li></div>
				<label class="ACT_collapse_similarity" for="act_similarity_65">
Hirsch M and Elliott T. Fis regulates transcriptional induction of RpoS in Salmonella enterica .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19909098" target="_blank">19909098</a>]<br></label>
<input id="act_similarity_65", type="checkbox">
<div class="ACT_similar">
<li>Hirsch , M. , and Elliott , T. ( 2005 ) Fis regulates transcriptional induction of RpoS in Salmonella enterica .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17784910" target="_blank">17784910</a>]<br></li></div>
<li>[ 15 ] Hirsch M , Elliot T. Fis regulates transcriptional induction of rpoS in Salmonella enterica .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18061487" target="_blank">18061487</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_68">
Hirsch M , Elliott T : Fis regulates transcriptional induction of RpoS in Salmonella enterica .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24885225" target="_blank">24885225</a>]<br></label>
<input id="act_similarity_68", type="checkbox">
<div class="ACT_similar">
<li>Fis regulates transcriptional induction of RpoS in Salmonella enterica .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20075614" target="_blank">20075614</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23385142" target="_blank">23385142</a>]<br></li></div>
<li>Fis is required for activation of ssrA expression in murine macrophages through DNA relaxation [ 56 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20861532" target="_blank">20861532</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_71">
Lim , S. , Kim , B. , Choi , H.S. , Lee , Y. , and Ryu , S. ( 2006 ) Fis is required for proper regulation of ssaG expression in Salmonella enterica serovar Typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16999831" target="_blank">16999831</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630976" target="_blank">17630976</a>]<br></label>
<input id="act_similarity_71", type="checkbox">
<div class="ACT_similar">
<li>Lim , S. , Kim , B. , Choi , H.S. , Lee , Y. and Ryu , S. ( 2006 ) Fis is required for proper regulation of ssaG expression in Salmonella enterica serovar Typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17329372" target="_blank">17329372</a>]<br></li></div>
				<label class="ACT_collapse_similarity" for="act_similarity_73">
Fis is required for proper regulation of ssaG expression in Salmonella enterica serovar Typhimu-rium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23006851" target="_blank">23006851</a>]<br></label>
<input id="act_similarity_73", type="checkbox">
<div class="ACT_similar">
<li>Fis is required for proper regulation of ssaG expression in Salmonella enterica serovar Typhi-murium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19858298" target="_blank">19858298</a>]<br></li></div>
<li>Nevertheless , novobiocin treatment caused a small increase in OmpR occupancy in Dfis mutants , indicating that OmpR binding does not absolutely require the topological constraints imposed on DNA by FIS binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22457642" target="_blank">22457642</a>]<br></li>
<li>To assess the relationship between oxygen availability , DNA supercoiling , and fis promoter activity , aeration was adjusted by growing cells in tubes with increasing culture volumes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24358360" target="_blank">24358360</a>]<br></li>
<li>have been identiﬁed as being important in activation of hilA expression , including csrAB , sirA/barA , phoBR , fadD , envZ , ﬂiZ , and ﬁs ( 2 , 3 , 26 , 32 , 53 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15731035" target="_blank">15731035</a>]<br></li>
<li>A variety of genes ( hilC , hilD , ﬁs , sirA/barA , csrAB , phoB , fadD , envZ/ompR , ﬂiZ , hilE , ams , lon , pag , and hha ) have been identiﬁed that exert positive or negative effects on hilA expression , although the mechanisms by which these gene products function remain relatively unclear .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12595445" target="_blank">12595445</a>]<br></li>
<li>The level of Fis protein then declines thereafter until it is Department of Microbiology , Moyne Institute of Preventive Medicine , Trinity College Dublin , Ireland .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17784910" target="_blank">17784910</a>]<br></li>
<li>Whereas Fis has been found to activate of ppGpp under low oxygen conditions ( Fig. 3 , B and D ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16905537" target="_blank">16905537</a>]<br></li>
<li>Therefore , we postulated that an rpoS mutant displaying elevated levels of Fis expression under these conditions should be more adept at invasion of epithelial cells .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17784910" target="_blank">17784910</a>]<br></li>
<li>This expression is associated with a downregulation of RpoS and an upregulation of Fis expression under these conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17784910" target="_blank">17784910</a>]<br></li>
<li>An rpoS mutant displayed elevated levels of Fis and had a higher frequency of epithelial cell invasion under these growth conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17784910" target="_blank">17784910</a>]<br></li>
<li>RpoS , Fis and invasion of epithelial cells Both the growth of Salmonella under non-aerated conditions and Fis protein expression are associated with invasion of epithelial cells ( Lee and Falkow , 1990 ; Wilson et al. , 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17784910" target="_blank">17784910</a>]<br></li>
<li>Fis protein binds to the PrpoS region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15716427" target="_blank">15716427</a>]<br></li>
<li>These results indicated that Fis regulates not only genes to which it binds directly , but also genes downstream of the Fis-binding genes ( Table S8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23717649" target="_blank">23717649</a>]<br></li>
<li>Most importantly , Fis was found to activate SPI genes , which are essential for the virulence of S. enterica .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23717649" target="_blank">23717649</a>]<br></li>
<li>Fis-mediated activation of the Spi-1 genes occurs through different regulatory modes : Fis binds to promoters of Spi-1 encoded genes , or controls conserved upstream regulators such as OmpR to inﬂuence the Spi-1 regulator HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26441883" target="_blank">26441883</a>]<br></li>
<li>In spite of all these changes , this protein is able to efﬁciently promote l DNA excision and ﬁs P repression and to moderately stimulate Hin-mediated DNA inversion .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9811652" target="_blank">9811652</a>]<br></li>
<li>This allows Fis to affect the homeostatic balance between the countervailing activities of DNA gyrase and DNA topoisomerase I. Third , Fis can inﬂuence DNA topology directly by binding to DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16999831" target="_blank">16999831</a>]<br></li>
<li>Proteins such as Fis , known to antagonize H-NS repression at the promoters of housekeeping genes and to-bind to highly degenerate DNA sequences , are good candidates for this role at one level .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15790293" target="_blank">15790293</a>]<br></li>
<li>Fis can act to preserve microdomains of supercoiling due to its preference for binding to DNA of intermediate superhelical density , and this allows it to sustain transcription at certain promoters that respond to changes in local DNA supercoiling levels ( Rochman et al. , 2002 ; 2004 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17784910" target="_blank">17784910</a>]<br></li>
<li>Furthermore , the small DNA-binding proteins H-NS , HU , and Fis have been demonstrated to repress hilA expression ( 18 , 51 ) , although it is unknown whether these binding proteins modulate hilA expression directly or indirectly .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11741864" target="_blank">11741864</a>]<br></li>
<li>Of the small nucleoid-binding proteins that affect hilA expression , H-NS acts as a repressor , but Fis and HU activate hilA by changing the expression of hilD ( 32 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16391141" target="_blank">16391141</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_95">
Also , Fis and CRP seem to exert the repression over hlyE through down-regulating rpoS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24885225" target="_blank">24885225</a>]<br></label>
<input id="act_similarity_95", type="checkbox">
<div class="ACT_similar">
<li>Moreover , Fis and CRP seem to exert the repression over hlyE via down-regulating rpoS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24885225" target="_blank">24885225</a>]<br></li></div>
<li>Zhi H , Wang X , Cabrera JE , Johnson RC , Jin DJ ( 2003 ) Fis stabilizes the interaction between RNA polymerase and the ribosomal promoter rrnB P1 , leading to transcriptional activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23717649" target="_blank">23717649</a>]<br></li>
<li>Both global positive regulators , such as the nucleoid proteins Fis and H-NS or the cyclic AMP ( cAMP ) - catabo-lite activator protein ( CAP ) complex ( 21 -- 23 ) , and negative regulators , such as RﬂM , the Spi1 regulator RtsB , or SlyA , bind within and act upon the ﬂhDC promoter ( 24 , 25 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24488311" target="_blank">24488311</a>]<br></li>
<li>To date , it is known that the positive regulators HilC and HilD and the negative regulators Hha and Fis can bind directly the hilA promoter ( 8 , 19 , 53 , 59 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14645287" target="_blank">14645287</a>]<br></li>
<li>Fis regulates genes encoding the T3SS and its cognate effectors as well as synthesis of ﬂagella for motility .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24308841" target="_blank">24308841</a>]<br></li>
<li>In particular , Fis regulates those genes encoding the type III secretion machinery and cognate effectors required by the bacterium for invasion of host epithelial cells , for survival in macrophage and for the deployment of ﬂagella for motility .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15256548" target="_blank">15256548</a>]<br></li>
<li>Two other NAPs , Fis and HU , positively regulate SPI-1 genes , although the mechanism of regulation is unknown and may be indirect ( 32 -- 35 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27601571" target="_blank">27601571</a>]<br></li>
<li>These results indicate that the autoregulation of tufB gene expression is uncoupled from tRNA production and show that it is a secondary regulatory mechanism specific for EF-TuB , and independent from transcriptional initiation of the operon and its regulation by Fis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26934594" target="_blank">26934594</a>]<br></li>
<li>We propose that this provides a mechanism for distinguishing the intra - and extracellular environment and for adaptation by S. Typhimurium to macrophage allowing enhanced survival through the effect of Fis and DNA relaxation on SPI-2 gene expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16999831" target="_blank">16999831</a>]<br></li>
<li>H-NS , HU , and Fis also appear to affect the derepression of hilA by HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12933899" target="_blank">12933899</a>]<br></li>
<li>Conformational changes of the upstream DNA mediated by H-NS and Fis regulate E. coli rrnB promoter activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15790293" target="_blank">15790293</a>]<br></li>
<li>Mol Microbiol 96 : Hancock , S.P. , Stella , S. , Cascio , D. , and Johnson , R.C. ( 2016 ) DNA Sequence determinants controlling affinity , stability and shape of DNA complexes bound by the nucleoid protein Fis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28631437" target="_blank">28631437</a>]<br></li>
<li>The effects of certain environmental conditions , such as growth-phase , oxygen , and osmolarity , may be mediated by small DNA binding proteins , such as Hha and Fis , whose functions can couple cellular physiology to the topology of the bacterial chromosome , altering DNA supercoiling and transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15765064" target="_blank">15765064</a>]<br></li>
<li>Most of the genes in SPI-1 , SPI-2 , SPI-4 , SPI-5 , and some genes in SPI-3 have been found to be positively regulated by Fis [ 7 ] , although the underlying regulatory mechanism remains to be clarified .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23717649" target="_blank">23717649</a>]<br></li>
<li>Using mutation and overexpression experiments , real-time PCR analysis , and electrophoretic mobility shift assays , we demonstrated that Fis regulates 63 of the 94 Salmonella pathogenicity island ( SPI ) -1 and SPI-2 genes , by three regulatory modes : i ) binds to SPI regulators in the gene body or in upstream regions ; ii ) binds to SPI genes directly to mediate transcriptional activation of themselves and downstream genes ; iii ) binds to gene encoding OmpR which affects SPI gene expression by controlling SPI regulators SsrA and HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23717649" target="_blank">23717649</a>]<br></li>
<li>Three regulatory modes were characterized by which Fis controls SPI gene transcription : i ) Fis binds to and activates SPI regulator genes ( hilC and ssrA ) ; ii ) Fis binds directly to SPI genes to enhance the transcription of these genes and those downstream ; iii ) Fis enhances the expression of the global regulator gene ompR [ 39 ] , which induces the expression of SPI positive regulator genes ( hilD and ssrA ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23717649" target="_blank">23717649</a>]<br></li>
<li>In this study , we firstly conclude the existence of three modes of Fis regulatory mechanism on SPI genes , and provide a regulatory network of Fis on SPI genes ( Figure 8 ) , including : i ) Fis binds to and activates the SPI positive regulator genes ; ii ) Fis binds to the ORF region of SPI genes to mediate transcriptional activation of these genes and those downstream genes ; iii ) Fis positively influences the SPI regulators by binding to and activating ompR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23717649" target="_blank">23717649</a>]<br></li>
<li>Although most genes on SPI were activated by Fis , a small proportion of SPI genes were down-regulated , such as STM2911 and STM2912 in SPI-1 and ttrB in SPI-2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23717649" target="_blank">23717649</a>]<br></li>
<li>Our data identify a coordinating role for Fis in the activation of virulence genes in SPI-1 , SPI-2 , some in SPI-3 , SPI-4 and SPI-5 , and are consistent with the previously demonstrated link between expression of SPI-5 genes and those of SPI-1 and SPI-2 ( Knodler et al. , 2002 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15256548" target="_blank">15256548</a>]<br></li>
<li>HU or Fis may bind t the hilA promoter and enhance its ability to be derepressed by HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12933899" target="_blank">12933899</a>]<br></li>
<li>These results suggest that Fis positively influences the SPI-1 and SPI-2 regulators by binding to and activating ompR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23717649" target="_blank">23717649</a>]<br></li>
<li>For example , OmpR promotes SPI-2 expression by directly binding to both the ssrA and ssrB promoters [ 16 ] ; PhoP activates SPI-2 transcription by directly binding to the ssrB promoter and controls SsrA levels post-transcriptionally [ 15 ] ; SlyA , HilD and Fis proteins can bind to the ssrA promoter to regulate SPI-2 genes [ 19,20,22 ] , and PmrA represses SPI-2 gene transcription through directly binding to the ssrB promoter [ 18 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29751061" target="_blank">29751061</a>]<br></li>
<li>Furthermore , Fis could clearly be detected when the rpoS mutant was grown under aerated conditions , whereas the protein was absent in wild-type cells grown under the same conditions ( Fig. 4C , lanes 1 and 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17784910" target="_blank">17784910</a>]<br></li>
<li>When the NaCl levels were raised , however , or when novobiocin ( 25 mg mL 1 ) was added to the growth medium , both the ihfB and the ﬁs mutations negatively affected pil transcription activities ( Fig. 1b ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16842356" target="_blank">16842356</a>]<br></li>
<li>This result supports the fact that glucose increases the fis expression , thereby contributing to the repression of hlyE .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24885225" target="_blank">24885225</a>]<br></li>
<li>In order to test this hypothesis , we performed a RT-PCR to detect the fis transcript of S. Typhi cultured to OD600 = 0.4 in LB or LB + glucose .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24885225" target="_blank">24885225</a>]<br></li>
<li>In this case , the DNA region from 2377 to 178 in pMB194 showed no Fis regulation , whereas the region from 2172 to 178 in pMB263 showed a 7.3-fold repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9811652" target="_blank">9811652</a>]<br></li>
<li>Together the results suggest that the full-length fis promoter region activates fis expression in response to DNA supercoiling in low-aerated conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24358360" target="_blank">24358360</a>]<br></li>
<li>Here , no effect of the ﬁs mutation on stpA expression was detected at 1 h , although stpA expression was dependent on Fis when wildtype and mutant were compared at 4 h .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15256548" target="_blank">15256548</a>]<br></li>
<li>Because PtopASe is repressed by chromosomally encoded FIS in wild-type E. coli , the presence of pBAD-ﬁs had very little additional repressive effect ( Fig. 4B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21276095" target="_blank">21276095</a>]<br></li>
<li>Conversely , PtopASe is quite unlike PtopAEc when placed in E. coli , probably due to an overriding repression of PtopASe by FIS in E. coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21276095" target="_blank">21276095</a>]<br></li>
<li>The reciprocal cross of placing PtopASe in E. coli revealed only weak PtopASe induction by novobiocin in the exogenous background , and this poor induction was not due to repression by FIS ( Fig. 6D ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21276095" target="_blank">21276095</a>]<br></li>
<li>Figure 4d clearly shows increased pixel intensity and therefore upregulation of ﬁs within the amoeba .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14502413" target="_blank">14502413</a>]<br></li>
<li>Unlike FIS , OmpR and its phospo-donor EnvZ were both absolutely required for induction of PssrA , suggesting that relaxed DNA supercoiling alone can not activate PssrA in the absence of OmpR 's ability to recruit RNAP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22457642" target="_blank">22457642</a>]<br></li>
<li>Like PssrA , PhilC required both FIS and OmpR-EnvZ for induction ; yet unlike PssrA , the absence of FIS was not compensated by increasing concentrations of novobiocin .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22457642" target="_blank">22457642</a>]<br></li>
<li>The effect of mutations in ﬁs , sirA , and csrA , which work most likely through hilD , rather than on hilA itself , closely 17,23,25 resembles the effect of DSPI1 deletion .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16443238" target="_blank">16443238</a>]<br></li>
<li>Full expression of ﬁs is required for upregulation of genes encoding secretion apparatus of T3SS and effectors required for invasion of host epithelial cells , for survival in macrophages and synthesis of ﬂagella for motility ( Kelly et al. , 2004 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24308841" target="_blank">24308841</a>]<br></li>
<li>Nilsson , L. , Vanet , A. , Vijgenboom , E. , and Bosch , L. ( 1990 ) The role of FIS in trans activation of stable RNA operons of E. coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26934594" target="_blank">26934594</a>]<br></li>
<li>Transmission of an Oxygen Availability Signal at the Salmonella enterica Serovar Typhimurium fis Promoter Andrew D. S. Cameron1 ,2 , Carsten Kröger1 , Heather J. Quinn1 , Isobel K. Scally1 , Anne J. Daly1 , Stefani C. Kary2 , Charles J. Dorman1 * 1 Department of Microbiology , Moyne Institute of Preventive Medicine , School of Genetics and Microbiology , Trinity College Dublin , Dublin , Ireland , 2 Department of Biology , University of Regina , Regina , SK , Canada Abstract The nucleoid-associated protein FIS is a global regulator of gene expression and chromosome structure in Escherichia coli and Salmonella enterica .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24358360" target="_blank">24358360</a>]<br></li>
<li>JOURNAL OF BACTERIOLOGY , Apr. 1995 , p. 2021 -- / 95 / $ 04.0010 Copyright q 1995 , American Society for Microbiology Sequence , Regulation , and Functions of ﬁs in Salmonella typhimurium ROBERT OSUNA ,1 † * DREW LIENAU ,2 KELLY T. HUGHES ,2 AND REID C. JOHNSON1 ,3 Department of Biological Chemistry , School of Medicine ,1 and Molecular Biology Institute ,3 University of California , Los Angeles , California 90024 , and Department of Microbiology , SC-42 , University of Washington , Seattle , Washington 981952 Received 27 September 1994/Accepted 1 February 1995 The ﬁs operon from Salmonella typhimurium has been cloned and sequenced , and the properties of Fisdeﬁcient and Fis-constitutive strains were examined .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7536730" target="_blank">7536730</a>]<br></li>
<li>A model is presented in which Fis binding to this site allows repression of rpoS speciﬁcally during exponential-growth , thus mediating transcriptional regulation of rpoS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15716427" target="_blank">15716427</a>]<br></li>
<li>This presence of Fis in cultures grown without aeration was shown to correlate with an increase in invasion of epithelial cells , conﬁrming that these were conditions under which SPI1 genes were induced ( Fig. 1B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17784910" target="_blank">17784910</a>]<br></li>
<li>Fis was found to act as a transcriptional-repressor of both gyr genes in S. enterica .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12898222" target="_blank">12898222</a>]<br></li>
<li>The DNA-binding activity of Fis was also found to be sufﬁcient for its function as an autoregulator but not for its role in transcription activation of rrnB P1 ( 4 , 19 , 28 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7536730" target="_blank">7536730</a>]<br></li>
<li>Because control of DNA supercoiling in response to osmotic pressure , novobiocin and FIS concentration differs between E. coli and S. enterica , we tested how a promoter that is ﬁne-tuned to supercoiling and FIS activity in one species is controlled when transferred to the other species .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21276095" target="_blank">21276095</a>]<br></li>
<li>Relaxation of DNA supercoiling caused an increase in OmpR binding to DNA and a concomitant decrease in binding by the nucleoid-associated protein FIS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22457642" target="_blank">22457642</a>]<br></li>
<li>The regulatory link between FIS and topA appears to have resulted in repeated parallel loss-of-function mutations in ﬁs during long-term E. coli evolution experiments due to the selective advantage of increased DNA supercoiling in standard culture conditions ( Crozat et al. , 2010 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21276095" target="_blank">21276095</a>]<br></li>
<li>However , the FIS binding sites in PBAD ( Cho et al. , 2008 ) were sufficient to titrate FIS and so remove repression of PtopASe .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21276095" target="_blank">21276095</a>]<br></li>
<li>It may be that titration of FIS by multicopy PtopAEc in wild-type cells prevents FIS saturation and repression of the promoter , allowing FIS to perform only an activating role .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21276095" target="_blank">21276095</a>]<br></li>
<li>In most cases , both PtopAEc and PtopASe increased expression in response to increased osmotic pressure regardless of the presence or absence of FIS ( Fig. 5A and B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21276095" target="_blank">21276095</a>]<br></li>
<li>The expression of ﬁs , which encodes a DNA binding protein known to positively regulate expression of hilA and invF ( 25 ) , was increased in L110 ( acrB : : aph ) only .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19411325" target="_blank">19411325</a>]<br></li>
<li>g004 DNA supercoiling control of S. Typhimurium fis expression DNA supercoiling is a key driver of fis expression [ 9 ] , raising the intriguing question of whether DNA supercoiling and the fis promoter respond to small incremental changes in oxygen availability .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24358360" target="_blank">24358360</a>]<br></li>
<li>We used the Ptac promoter as a second control in addition to PlacUV5 ; its activity was not increased and in fact was slightly depressed in the ﬁs background .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15716427" target="_blank">15716427</a>]<br></li>
<li>However , a clear increase in ﬁs transcription levels under low-aeration conditions in stationary phase was observed in both the wild-type and the rpoS mutant strains but absent in both strains when grown under aerating conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17784910" target="_blank">17784910</a>]<br></li>
<li>In S. enterica the absence of FIS caused a small decrease in PtopASe expression during exponential-growth and resulted in elevated PtopASe expression during stationary phase ( Fig. 4A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21276095" target="_blank">21276095</a>]<br></li>
<li>Thus in S. enterica , FIS enhances the expression of both PtopAEc and PtopASe during exponential-growth , but reduces the expression of both promoters in stationary phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21276095" target="_blank">21276095</a>]<br></li>
<li>E. coli topA was upregulated in the absence of FIS in both exponential-growth and stationary phase , whereas S. enterica topA expression was unaffected in Dﬁs cells during exponential-growth but was elevated in stationary phase relative to wild-type cells .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21276095" target="_blank">21276095</a>]<br></li>
<li>When expressed exogenously in wild-type S. enterica , PtopAEc expression is greatly decreased in stationary-phase compared with exponential-growth ; however , in the absence of FIS , PtopAEc expression is relatively insensitive to growth-phase ( Fig. 5C ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21276095" target="_blank">21276095</a>]<br></li>
<li>The correlation between DNA supercoiling and sustained fis expression in non-aerated cultures prompted us to test whether increased fis expression in the ∆ arcA mutant might also correlate with an increase in DNA supercoiling in this genetic background .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24358360" target="_blank">24358360</a>]<br></li>
<li>The expression of fis in intracellular environments may be explained in part by the recent finding that S. Typhimurium fis expression is elevated in stationary phase if oxygen availability is reduced , a condition referred to as `` sustained expression '' because the fis promoter remains active in stationary phase [ 13 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24358360" target="_blank">24358360</a>]<br></li>
<li>In these conditions , deletion of arcA caused a decrease in Pfis-1 and Pfis-2 activity , consistent with the decrease in exponential phase fis expression seen in Figure 2D .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24358360" target="_blank">24358360</a>]<br></li>
<li>As shown in Figure 1E , the presence of glucose increased fis transcription compared with bacteria grown in standard LB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24885225" target="_blank">24885225</a>]<br></li>
<li>The Salmonella ﬁs promoter in E. coli revealed no signiﬁcant difference in its transcription proﬁle whether grown in the presence or absence of aeration at 24 h ( Fig. 2C ) , despite having shown induction of ﬁs transcription when grown without aeration in stationary phase when in its native SL1344 background ( Fig. 2A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17784910" target="_blank">17784910</a>]<br></li>
<li>However , when ﬁs is supplied in trans on a plasmid , DNA inversion may be stimulated , resulting in expression of lacZ and growth of red col-onies on MacConkey-lactose .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9811652" target="_blank">9811652</a>]<br></li>
<li>Phenotypes of ﬁs mutants include loss of Hin-mediated DNA inversion , cell ﬁlamentation , reduced growth rates in rich medium , and increased lag times when the mutants are subcultured after prolonged growth in stationary phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7536730" target="_blank">7536730</a>]<br></li>
<li>Thus , up-regulation of rpoS in the ΔarcA mutant causes an elevation in sustained fis expression because of elevated RpoS activity at the fis promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24358360" target="_blank">24358360</a>]<br></li>
<li>However , for gene expression studies , cells containing pBAD33 or pBAD-ﬁs were cultured overnight in 2 ml of LB ( 0.5 % NaCl ) in tubes in the absence of glucose or arabinose ; the presence of glucose or arabinose did not have an effect on ectopic ﬁs expression levels , likely due to a loss of PBAD control in cells with altered FIS levels .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21276095" target="_blank">21276095</a>]<br></li>
<li>Since the absence of ﬁs results in increased expression of ﬂjB during early logarithmic phase , we can not correlate the reduced motility seen in ﬁs mutants with ﬂjB expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7536730" target="_blank">7536730</a>]<br></li>
<li>PhilD was unique among the four promoters in having higher expression in the absence of ompR and envZ , but it nevertheless required FIS for full activation ( Figure 1B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22457642" target="_blank">22457642</a>]<br></li>
<li>Control of fis expression is best understood in E. coli , where the global regulatory proteins CRP , IHF , and FIS modulate only the degree of induction , but none of them is absolutely required for fis expression in laboratory culture [ 4-6 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24358360" target="_blank">24358360</a>]<br></li>
<li>In contrast , strain CSH50 , which produced no detectable Fis protein in late exponential-growth-phase regardless of aeration ( Fig. 1A ) , showed only a ~ 2-fold decrease in RpoS levels in response to the presence or absence of aeration ( Fig. 4A , lanes 3 and 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17784910" target="_blank">17784910</a>]<br></li>
<li>The effect of the ﬁs knockout mutation was negative in all cases and resulted in a reduction in expression of approximately twofold ( Fig. 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15256548" target="_blank">15256548</a>]<br></li>
<li>Higher concentrations of arabinose resulted in a very marked increase in Fis protein levels , the conversion of almost all reporter plasmid to its relaxed form and a signiﬁcantly higher level of ssrA transcription than was seen in the arabinose-free control .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16999831" target="_blank">16999831</a>]<br></li>
<li>To corroborate whether an increase in the hlyE transcript due to the absence of fis in LB and in LB + glucose resulted in increased HlyE-associated hemolytic activity , erythrocyte damage produced by whole bacterial lysates of S. Typhi WT and Δfis mutants grown in LB or LB + glucose was compared by quantifying the release of hemoglobin .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24885225" target="_blank">24885225</a>]<br></li>
<li>This study examined spatial and temporal changes in the distribution of acanth-amoebae and salmonellae in coculture , in the regulation of ﬁs , and in the morphology of S. typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14502413" target="_blank">14502413</a>]<br></li>
<li>Glucose increases transcription of fis As stated above , the effect of Δcrp mutant on hlyE transcriptional level is relieved when glucose is added into the culture medium , suggesting that a second repressor is activated in some way under a glucose-rich medium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24885225" target="_blank">24885225</a>]<br></li>
<li>Effect on hilA expression decrease decrease decrease decrease decrease decrease decrease Mutation1 Comments ompR envZ [ 66 ] [ 66 ] EnvZ/OmpR is a two-component system EnvZ/OmpR is a two-component system pho-24 sirA , barA pstS fadD ﬂiZ [ 19 ] [ 64 , 65 , 68 ] , unpublished2 [ 66 ] [ 66 ] [ 66 ] , unpublished3 hyperactive PhoQ sensor kinase BarA/SirA is a two-component system acts via PhoB regulator does not act via FadR regulator transposon insertions in ﬂiA are polar on ﬂiZ ; FliZ suppresses the effects of transposon insertions in ﬂiA and ﬂhCD the Csr system regulates RNA stability Fis is a small nucleoid binding protein HU , a small nucleoid binding protein , is comprised of HupA and HupB Hha is a small nucleoid binding protein amsE encodes RNase E H-NS is a small nucleoid binding protein no known homologies PhoPQ-activated gene csrB ﬁs decrease decrease decrease decrease increase increase increase increase [ 64 , 68 ] unpublished4 ,5 unpublished4 ,5 unpublished2 [ 56 ] unpublished4 ,5 [ 56 ] [ 56 ] hupA/hupB hha amsE : : Tn5-1 hns hilE pag-1 Condition high osmolarity low oxygen late log phase near-neutral pH low osmolarity low oxygen early log phase bile [ 19 ] , unpublished3 [ 19 ] , unpublished6 [ 57 ] , unpublished7 unpublished3 [ 19 ] , unpublished3 [ 19 ] , unpublished6 [ 57 ] , unpublished7 unpublished3 osmolarity regulates hilA in the absence of OmpR/EnvZ oxygen regulates hilA in the absence of OxrA/Fnr or ArcA induction induction induction induction repression repression repression repression late log phase induces invasion activity ; RpoS is not required not mediated by PmrAB osmolarity regulates hilA expression in the absence of OmpR/EnvZ oxygen regulates hilA in the absence of OxrA/Fnr or ArcA early log phase represses invasion activity ; RpoS is not required reported to regulate invasion , prgH , invF and hilC ; effects on hilA expression not reported but very likely 1 Unless otherwise noted , the mutations are null mutations in the open reading frame listed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11755416" target="_blank">11755416</a>]<br></li>
<li>It is possible then that the fis promoter is inactive in stationary phase because DNA is in a more relaxed state and because effective concentrations of RpoD are reduced due to competition with RpoS [ 31 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24358360" target="_blank">24358360</a>]<br></li>
<li>Although topA promoter function has routinely been studied using plasmid-based systems ( Tse-Dinh , 1985 ; Tse-Dinh and Beran , 1988 ; Lesley et al. , 1990 ; Marshall et al. , 2000 ; Weinstein-Fischer and Altuvia , 2007 ) , our results indicate that in E. coli the expression of plasmid-borne PtopAEc is enhanced by FIS whereas the chromosomal copy of the promoter is repressed by FIS ( compare Figs 3A and 4A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21276095" target="_blank">21276095</a>]<br></li>
<li>Fis binds preferentially to supercoiled DNA , and Fis-promoted activation coincides with a high level of nucleoid supercoiling under anaerobic conditions in the gut during the initial infection of Salmonella ( Wilson et al. , 2001 ; Wang et al. , 2013 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26441883" target="_blank">26441883</a>]<br></li>
<li>As shown in Fig. 5 , Fis may be required for the induction of one of the genes involved in activation of hilA : envZ , fliZ , pstS , fadD ( Lucas et al. , 2000 ) , sirA ( Johnston et al. , 1996 ) , barA and csrA/B ( Altier et al. , 2000 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11123690" target="_blank">11123690</a>]<br></li>
<li>Fis also stimulates transcription of E. coli topA , the gene coding for topoisomerase I , when the bacteria experience oxidative-stress ( Weinstein-Fischer et al. , 2000 ) , something that may be experienced by S. Typhimurium when growing in macrophage ( Babior , 2000 ; Nathan and Shiloh , 2000 ) , although the severity of the oxidative-stress is unclear from an examination of the responses of classic oxidative-stress response genes ( Eriksson et al. , 2003 ; Faucher et al. , 2006 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16999831" target="_blank">16999831</a>]<br></li>
<li>At ﬁrst glance , this observation is difficult to reconcile with the ﬁnding that optimal expression of the genes in the SPI1 island requires the Fis DNA binding protein , because this is expressed best in the early stages of logarithmic growth and becomes almost undetectable in cultures approaching the end of exponential-growth ( Ball et al. , 1992 ; Nilsson et al. , 1992 ; Ninnemann et al. , 1992 ; Keane and Dorman , 2003 ; Mallik et al. , 2004 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17784910" target="_blank">17784910</a>]<br></li>
<li>As expected , the result showed that when the concentration of Fis protein ( 0 -- 400 nM ) was increased , more FisDNA complex and less free DNA were detected for all 16 genes ( Figure S2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23717649" target="_blank">23717649</a>]<br></li>
<li>Expression of hilA is increased by the pleiotropic activator proteins OmpR , PhoPQ , and the Factor for Inversion Stimulation , Fis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16443238" target="_blank">16443238</a>]<br></li>
<li>This expression pattern reﬂects the activity of the ﬁs promoter which is repressed by the Fis protein and requires negative DNA supercoiling for optimal activity ( Schneider et al. , 2000 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17784910" target="_blank">17784910</a>]<br></li>
<li>Fis activates the RpoS-dependent station-ary-phase expression of proP in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15716427" target="_blank">15716427</a>]<br></li>
<li>Xu , J. , and Johnson , R.C. ( 1995 ) Fis activates the RpoSdependent stationary-phase expression of proP in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11123690" target="_blank">11123690</a>]<br></li>
<li>The decreased DNA gyrase activity inhibited cspH transcription , and the accumulation of Fis upon nutrient up-shift in part regulated its expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15235764" target="_blank">15235764</a>]<br></li>
<li>As shown in Figs. 2 , and 4 , cspH expression in the Fis mutant was partially repressed , but the addition of gyrase inhibitor almost completely blocked expression upon nutrient up-shift .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15235764" target="_blank">15235764</a>]<br></li>
<li>From this it seems that Fis regulation does not require CRP and that full CRP regulation is hindered in the absence of Fis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15716427" target="_blank">15716427</a>]<br></li>
<li>It has been demonstrated that normal regulation is also dependent upon CRP , and in vitro results suggest ﬁs promoter activation in the absence of Fis , with Fis and CRP acting synergistically as transcriptional-repressors ( 47 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15716427" target="_blank">15716427</a>]<br></li>
<li>Fis modulates expression of various proteins involved in metabolic processes Proteins regulated by Fis in other cellular processes were primarily enzymes and transport proteins involved in the metabolism of sugars .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14553938" target="_blank">14553938</a>]<br></li>
<li>In order to understand the role of Fis in virulence gene expression of S. typhimurium , we analyzed proteins that are di ¡ erentially expressed in the presence and absence of cents s and detected 11 possible proteins upregulated by Fis and seven proteins downregulated by Fis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14553938" target="_blank">14553938</a>]<br></li>
<li>This difference in Fis-binding sites might be correlated with the difference in magnitude of negative regulation by Fis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7536730" target="_blank">7536730</a>]<br></li>
<li>As with CRP , Fis might participate in downregulation of hlyE via RpoS , and this regulation could be exerted directly due to the presence of several putative Fis binding boxes in S. Typhi rpoS promoter ( Figure 2C ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24885225" target="_blank">24885225</a>]<br></li>
<li>Fis regulation of rpoS requires the presence of a Fis site positioned at 50 with respect to PrpoS , and this site is bound by Fis in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15716427" target="_blank">15716427</a>]<br></li>
<li>We characterized the activity of promoters serving rpoS in Salmonella enterica serovar Typhimurium and report that regulation of transcription during growth into SP depends on Fis , a DNA-binding protein whose abundance is high during exponential-growth and very low in SP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15716427" target="_blank">15716427</a>]<br></li>
<li>Microbiology ( 2004 ) , 150 , 2037 -- 2053 DOI 10.1099 / mic.0.27209-0 A global role for Fis in the transcriptional control of metabolism and type III secretion in Salmonella enterica serovar Typhimurium Arlene Kelly ,1 Martin D. Goldberg ,2 Ronan K. Carroll ,1 Vittoria Danino ,2 Jay C. D. Hinton2 and Charles J. Dorman1 1Department of Microbiology , Moyne Institute of Preventive Medicine , University of Dublin , Trinity College , Dublin 2 , Ireland Correspondence Charles J. Dorman 2Molecular Microbiology Group , Institute of Food Research , Norwich Research Park , Colney , Norwich NR4 7UA , UK cjdorman@tcd.ie Fis is a key DNA-binding protein involved in nucleoid organization and modulation of many DNA transactions , including transcription in enteric bacteria .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15256548" target="_blank">15256548</a>]<br></li>
<li>After 60 to 90 min of logarithmic growth , ﬂjB mRNA levels are low or undetectable in the presence of Fis , and maximum levels are reached by about 120 min of outgrowth .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7536730" target="_blank">7536730</a>]<br></li>
<li>[ 33,34 ] Although the absence of Fis , a key regulator in S. Typhimurium virulence , up-regulates the expression of cadA and cadB , [ 35 ] it is still not known how cadA is involved in Salmonella pathogenicity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20390946" target="_blank">20390946</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_197">
Ó Cróinín , T. , and Dorman , C.J. ( 2007 ) Expression of the Fis protein is sustained in late-exponential - and stationary-phase cultures of Salmonella enterica serovar Typhimu-rium grown in the absence of aeration .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21276095" target="_blank">21276095</a>]<br></label>
<input id="act_similarity_197", type="checkbox">
<div class="ACT_similar">
<li>Expression of the Fis protein is sustained in late-exponential - and stationary-phase cultures of Salmonella enterica serovar Typhimurium grown in the absence of aeration .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23676436" target="_blank">23676436</a>]<br></li>
<li>Cróinín TÓ , Dorman CJ ( 2007 ) Expression of the Fis protein is sustained in late-exponential - and stationary-phase cultures of Salmonella enterica serovar Typhimurium grown in the absence of aeration .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24358360" target="_blank">24358360</a>]<br></li></div>
<li>Based on these results and the requirement for speciﬁc sequences upstream of the major rpoS promoter , we present an intuitive model for transcriptional regulation of rpoS in which Fis binds to and represses transcription from PrpoS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15716427" target="_blank">15716427</a>]<br></li>
<li>These results are consistent with a model in which most Fis regulation of rpoS is independent of CRP function , and is not mediated through , for example , competitive binding of the two regulators at the overlapping Fis III and CRP sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15716427" target="_blank">15716427</a>]<br></li>
<li>Figure 2 CRP and Fis participate in the repression of rpoS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24885225" target="_blank">24885225</a>]<br></li>
<li>Testing the interaction of CRP and Fis in the transcriptional regulation of rpoS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15716427" target="_blank">15716427</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_204">
Multiple effects of Fis on integration and the control of lysogeny in phage lambda .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15716427" target="_blank">15716427</a>]<br></label>
<input id="act_similarity_204", type="checkbox">
<div class="ACT_similar">
<li>Multiple effects of Fis on integration and the control of lysogeny in phage l. J. Bacteriol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9811652" target="_blank">9811652</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7536730" target="_blank">7536730</a>]<br></li></div>
<li>In addition , we implicate the alternate stress response sigma factor RpoS as playing a role in repressing Fis expression , and show that induction of Fis in the absence of aeration is facilitated by a reduction of RpoS levels under these conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17784910" target="_blank">17784910</a>]<br></li>
<li>Curiously , results of both b-galactosidase and primer extension assays indicate that the presence of sequences upstream of 2173 in the S. marcescens ﬁs P region may somehow prevent repression by Fis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9811652" target="_blank">9811652</a>]<br></li>
<li>The involvement of Fis in negative regulation of ndk was of interest given its similar role in the expression of the nupG and rbsC nucleoside transport genes , suggesting that Fis coordinates the expression of genes involved in pyrimidine metabolism .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15256548" target="_blank">15256548</a>]<br></li>
<li>Expression of ﬂagellar gene fusions in the presence and absence of Fis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15256548" target="_blank">15256548</a>]<br></li>
<li>Increased stability of l and Mu lysogens is also observed in the presence of Fis ( 3 , 5 , 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7536730" target="_blank">7536730</a>]<br></li>
<li>However , based on these results we could not conclude that Fis directly interacts with the cspH promoter , since this might also be due to the pleio-tropic effect of Fis , coordinating the expression of a large set of genes both by direct control at the level of transcription initiation and by possible indirect effects ( Drlica and Rouviere-Yaniv 1987 ; Finkel and Johnson 1992 ; McLeod et al. 2002 ; Schmid 1990 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15235764" target="_blank">15235764</a>]<br></li>
<li>Molecular Microbiology ( 2007 ) 66 ( 1 ) , 237 -- 251 doi :10.1111 / j.1365-2958.2007.05916.x First published online 3 September 2007 Expression e Fis is sustaine late-ex o ential - and stationary-phase Salmonella enterica serovar Typhim se c of aeration of th d in protein p n cultures of m w gro n in uriu the ab n e Tadhg Ó Cróinín and Charles J. Dorman * positively or negatively , depending on the location of its binding site with respect to the target promoter ( Xu and Johnson , 1995 ; Schneider et al. , 2003 ; Opel et al. , 2004 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17784910" target="_blank">17784910</a>]<br></li>
<li>Because hilA expression is also induced during mid-log to late-log growth ( Lee and Falkow , 1990 ; Bajaj et al. , 1996 ) , we tested the effect of Fis on hilA regulation and found that a mutation in the S. typhimurium fis gene results in a large negative impact on the expression of both hilA and invF and a severe virulence defect in the mouse model of S. typhimurium pathogenesis by the oral , but not the peritoneal , route of infection .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11123690" target="_blank">11123690</a>]<br></li>
<li>However , growth in the absence of aeration increases the level of negative supercoiling in the DNA of E. coli and S. Typhimurium ( Yamamoto and Droffner , 1985 ; Dorman et al. , 1988 ; Hsieh et al. , 1991a ) , and this would stimulate the ﬁs promoter if RpoD were free from competition by RpoS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17784910" target="_blank">17784910</a>]<br></li>
<li>Results are the averages from three independent experiments Fis affects the expression of cspH Fis , which is induced in the early exponential phase or upon nutrient up-shift , is a pleiotropic regulator in E. coli ( Drlica and Rouviere-Yaniv 1987 ; Finkel and Johnson 1992 ; McLeod et al. 2002 ; Schmid 1990 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15235764" target="_blank">15235764</a>]<br></li>
<li>The level of Fis in the cell is subject to complex and multifactorial control .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15256548" target="_blank">15256548</a>]<br></li>
<li>Up-regulation of the S. Typhimurium fis promoter in the Δfis mutant confirmed that FIS represses its own promoter , and this repression occurs in both aerated and non-aerated conditions ( Figure 3C & D ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24358360" target="_blank">24358360</a>]<br></li>
<li>SPI-2 expression during growth in rich medium , which roughly mimics conditions in the intestinal lumen , requires OmpR and FIS but is independent of SsrB , SlyA , and PhoP [ 12,18 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22457642" target="_blank">22457642</a>]<br></li>
<li>The expression of fis and rpoS is negatively correlated , the former being elevated in exponential-growth and the latter in stationary phase , therefore it is unsurprising that FIS and RpoS are antagonistic at some gene promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24358360" target="_blank">24358360</a>]<br></li>
<li>Null mutations in ﬁs , hupB and/or hupA decrease hilA expression and so are not likely to be 're pressors ' of hilA in any classical sense even though they do affect the regulation of hilA expression ( L. Schechter and C. Lee , unpublished observations ; Wilson et al. , in press ; B. Jones unpublished observations ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11755416" target="_blank">11755416</a>]<br></li>
<li>Falconi , M. , Proseda , G. , Giangrossi , M. , Beghetto , E. , and Colonna , B. ( 2001 ) Involvement of Fis in the H-NS-mediated regulation of virF gene in Shigella and enteroinvasive Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16999831" target="_blank">16999831</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17784910" target="_blank">17784910</a>]<br></li>
<li>Residues within the b-hairpin , in particular Val16 , Asp20 , and Val22 , are required for stimulation of Hin-mediated DNA inversion and presumably participate in transient interactions between Fis and the Hin recombinase ( 29 , 41 , 47 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9811652" target="_blank">9811652</a>]<br></li>
<li>Fis also binds to and represses the promoters of the gyrA and gyrB genes in both E. coli ( Schneider et al. , 1999 ) and S. Typhimurium ( Keane and Dorman , 2003 ) , resulting in reduced levels of DNA supercoiling .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16999831" target="_blank">16999831</a>]<br></li>
<li>Upon subculturing of stationary-phase cells in LB medium , Fis protein and mRNA levels increase over 800 - and 1,000-fold , respectively , within the period required to initiate logarithmic cell division .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7536730" target="_blank">7536730</a>]<br></li>
<li>RpoS , PhoP , CRP and Fis may be important for the proper hlyE expression inside macrophages , since the RpoS and PhoP respective regulons are activated inside these cells ; and both crp and fis expression is repressed in the same environment [ 41,43 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24885225" target="_blank">24885225</a>]<br></li>
<li>We discuss a model whereby Fis and RpoS levels vary in response to environmental signals allowing the expression and repression of SPI1 invasion genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17784910" target="_blank">17784910</a>]<br></li>
<li>Given that a previous study ( Hirsch and Elliott , 2005 ) had reported that the Fis protein can act as a repressor of RpoS transcription during exponential-growth , we decided to investigate whether RpoS in turn represses Fis expression under stationary-phase conditions and whether this contributes to the observed anti-correlation between Fis and RpoS expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17784910" target="_blank">17784910</a>]<br></li>
<li>Thus , the presence of a functional ﬁs gene allows cells to recover from stationary phase more efﬁciently and to grow faster when they are subcultured in a rich medium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7536730" target="_blank">7536730</a>]<br></li>
<li>Fis serves to autoregulate the magnitude of ﬁs mRNA levels but has little effect on the overall regulation pattern ( Fig. 7C and D ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7536730" target="_blank">7536730</a>]<br></li>
<li>IHF activation is deduced to be mediated through HilD as for Fis and HU according to the information provided in the text .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23554419" target="_blank">23554419</a>]<br></li>
<li>By co-operating with or antagonizing the action of the other regulators , Fis appears to modulate and ﬁne-tune gene expression in ways that beneﬁt the cell during growth and adaptation to environmental change .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15256548" target="_blank">15256548</a>]<br></li>
<li>When PssrA is transferred to E. coli , the ﬁne-tuning of PssrA to the supercoiling and FIS states of S. enterica results in inappropriate regulation in response to the environmental conditions that are relevant during host colonization .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21276095" target="_blank">21276095</a>]<br></li>
<li>Thus , it was of interest to determine the extent to which Fis protein in S. typhimurium resembles that in E. coli with respect to the Hin-mediated recombination reaction and its other physiological roles .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7536730" target="_blank">7536730</a>]<br></li>
<li>We then overexpressed ompR in Dfis , and found that the increased transcription of the SPI-1 and SPI-2 genes was recovered ( 1.04 to 6.36-fold increase ) , thus partly compensating for the effect caused by the fis deletion ( Figure 6C ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23717649" target="_blank">23717649</a>]<br></li>
<li>CRP and Fis participate in the repression of hlyE transcription by repressing rpoS transcription To find out whether CRP is repressing hlyE expression through a direct binding to the respective promoter region , we performed a shift mobility assay using purified CRP activated with cAMP and the DNA corresponding to the taiA -- hlyE promoter region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24885225" target="_blank">24885225</a>]<br></li>
<li>Fis somewhat increased the expression of cspH upon nutritional up-shift .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15235764" target="_blank">15235764</a>]<br></li>
<li>Since the negative effect of Fis is restricted to exponential phase , this ﬁnding supports a role for TABLE 2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15716427" target="_blank">15716427</a>]<br></li>
<li>Changing the supercoiling of the DNA also exerts an effect : Fis prefers DNA that is negatively supercoiled , a requirement that is easily met during the early stages of logarithmic growth when Fis is abundant .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28631437" target="_blank">28631437</a>]<br></li>
<li>In the absence of Fis , full upregulation is not detected .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16999831" target="_blank">16999831</a>]<br></li>
<li>In contrast , endogenous PtopASe expression was induced equally in the presence or absence of FIS ( Fig. 6B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21276095" target="_blank">21276095</a>]<br></li>
<li>PtopASe maintained wild-type expression during exponential-growth in the absence of rpoS ( Fig. 4C ) , indicating that the higher concentration of FIS was insufficient to repress PtopASe .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21276095" target="_blank">21276095</a>]<br></li>
<li>The exponential phase expression of PtopAEc was slightly elevated in the absence of rpoS , perhaps due to stimulation by elevated FIS levels .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21276095" target="_blank">21276095</a>]<br></li>
<li>Here we showed that ssaG was also dependent on the presence of a functional ﬁs gene when the bacteria grew in macrophage ( Fig. 3B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16999831" target="_blank">16999831</a>]<br></li>
<li>In E. coli , PtopAEc induction was dependent on the presence of FIS , although weak induction did occur in E. coli Dﬁs at the highest novobiocin concentration tested ( 25 mg ml-1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21276095" target="_blank">21276095</a>]<br></li>
<li>During growth , the ﬁs mutant displayed a ninefold elevation of PrpoS activity , although the increase of RpoS protein was estimated at three - to fourfold ( Fig. 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15716427" target="_blank">15716427</a>]<br></li>
<li>These results are consistent with the presence of multiple Fis binding sites in each gyr promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12898222" target="_blank">12898222</a>]<br></li>
<li>Since a Tn5 insertion in this region disrupts the positive regulation of hupB by crp-cAMP as well as the negative regulation by FIS we currently do not know if the transposon insertion increases or decreases normal expression of hupB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10767604" target="_blank">10767604</a>]<br></li>
<li>The presence of pFis349 in the wild-type strain slightly decreased PrpoS-lac activity , a ﬁnding consistent with the idea that Fis represses rpoS transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15716427" target="_blank">15716427</a>]<br></li>
<li>Fis is expressed in the intracellular niche , so why does it not repress transcription of the rpoS gene , preventing RpoS from activating spv transcription ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17784910" target="_blank">17784910</a>]<br></li>
<li>However , if Fis regulates hilD , it might be expected that activation of the hilD promoter in pJB1 could not occur in the fis background .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11123690" target="_blank">11123690</a>]<br></li>
<li>Fis protein could be detected from pBAD24ﬁs following arabinose induction in bacteria grown in either the presence or absence of aeration , showing that the arabinose-dependent regulatory circuit operated under both sets of growth conditions ( Fig. 5D ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17784910" target="_blank">17784910</a>]<br></li>
<li>Increased transcription from PrpoS in the absence of Fis eliminated the transcriptional induction as cells enter SP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15716427" target="_blank">15716427</a>]<br></li>
<li>Fis activates a chromosomal invF : : Tn5lacZY reporter in the absence of HilA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11123690" target="_blank">11123690</a>]<br></li>
<li>Our data indicate that the presence of the RpoS sigma factor correlates with repression of the ﬁs gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17784910" target="_blank">17784910</a>]<br></li>
<li>Fis modulates the kinetics of the supercoiling response to intramacrophage growth Our data showed that DNA relaxation was a key signal for upregulation of the macrophage-inducible ssrA gene , but that the presence of the Fis protein was also required for this activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16999831" target="_blank">16999831</a>]<br></li>
<li>We hypothesized that the induction of SPI1 genes under non-aerated conditions could be due to the presence of the Fis protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17784910" target="_blank">17784910</a>]<br></li>
<li>Moreover , the lack of fis clearly contributed to the RpoS accumulation even in presence of glucose , as inferred from Figure 2B .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24885225" target="_blank">24885225</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_258">
J Biol Chem 262:10362 -- 10368 Martin RG , Rosner JL ( 1997 ) Fis , an accessorial factor for transcriptional activation of the mar ( multiple antibiotic resistance ) promoter of Escherichia coli in the presence of the activator MarA , SoxS , or Rob .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15235764" target="_blank">15235764</a>]<br></label>
<input id="act_similarity_258", type="checkbox">
<div class="ACT_similar">
<li>Martin R.G. , Rosner J.L. ( 1997 ) Fis , an accessorial factor for transcriptional activation of the mar ( multiple antibiotic resistance ) promoter of Escherichia coli in the presence of the activator MarA , SoxS , or Rob .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19120970" target="_blank">19120970</a>]<br></li>
<li>Fis , an accessorial factor for transcriptional activation of the mar ( multiple antibiotic resistance ) promoter of Escherichia coli in the presence of the activator MarA , SoxS , or Rob .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19759044" target="_blank">19759044</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15073288" target="_blank">15073288</a>]<br></li></div>
<li>Many of the positive activators of hilA expression are now believed to function by posttranscriptional modiﬁcation of hilD , although the activity of the Fis protein may be more direct ( 62 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12595445" target="_blank">12595445</a>]<br></li>
<li>The retention of Fis-mediated repression of gyrA in S. enterica despite the radically different nucleotide sequence found immediately upstream of the Pribnow box is indicative of the physiological importance for the cell of regulation of gyrase expression by Fis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12898222" target="_blank">12898222</a>]<br></li>
<li>Even though the levels of RpoS increase as those of Fis decrease , both are essential for the expression of proP2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11123690" target="_blank">11123690</a>]<br></li>
<li>It can be speculated that the weak effects of Fis on effectors observed in this study result from the inability of the LT2 growth conditions ( Luria-Bertani broth ) to induce transcription of effectors [ 79,80 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23717649" target="_blank">23717649</a>]<br></li>
<li>To investigate the role of Fis in rpoS regulation , we tested the effect of a ﬁs insertion/deletion ( 49 ) on the activity of P - lac [ op ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15716427" target="_blank">15716427</a>]<br></li>
<li>Expression of the ﬂagellar protein FliC in the presence or absence of Fis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15256548" target="_blank">15256548</a>]<br></li>
<li>These results showed that the absence of RpoS in the absence or presence of aeration correlated with greater levels of Fis protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17784910" target="_blank">17784910</a>]<br></li>
<li>2-DE map comparison led to the detection of several SPI-related proteins induced in the presence of Fis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14553938" target="_blank">14553938</a>]<br></li>
<li>Additionally , we detected one SPI-2 TTSS-related protein , SseC , that showed increased levels in the presence of Fis , suggesting the possibility of an association of Fis with SPI-2 expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14553938" target="_blank">14553938</a>]<br></li>
<li>To test whether overexpression of ﬁs could compensate for titration and consequently repress PtopAEc , ﬁs was cloned in the low-copy-number vector pBAD33 under the control of the PBAD promoter ( pBAD-ﬁs ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21276095" target="_blank">21276095</a>]<br></li>
<li>The results showed that in the absence of Fis , the ssrA promoter could still respond to DNA relaxation , but the response was more pronounced ( Fig. 5C ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16999831" target="_blank">16999831</a>]<br></li>
<li>Thus , we wondered whether , in addition to stimulating the Hin-mediated DNA inversion reaction , Fis could also regulate the transcription of the ﬂjB promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7536730" target="_blank">7536730</a>]<br></li>
<li>Since the intracellular levels of this protein were comparable to or greater than those of E. coli Fis generated from pRJ807 , the reduction in the ability to repress ﬁs P could not be attributed to lower intracellular concentrations of E. carotovora Fis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9811652" target="_blank">9811652</a>]<br></li>
<li>In particular Fis seems to be well placed to coordinate the expression of genes involved in the transition from a free-living life in the gut lumen to the intracellular niche .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15256548" target="_blank">15256548</a>]<br></li>
<li>The ﬁs promoter is most active when bacteria are in the early logarithmic phase of growth , when the cell may contain up to 60,000 copies of the Fis protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12898222" target="_blank">12898222</a>]<br></li>
<li>Mean channel fluorescenc Typhimurium to macrophage is delayed in the absence of Fis ( Fig. 5 ) showing that the dynamics of DNA supercoiling regulation are altered when the ﬁs gene is inactivated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16999831" target="_blank">16999831</a>]<br></li>
<li>Sequence analysis of genes regulating intracellular levels of RpoS To identify factors that inﬂuence an increase in the RpoS levels in the KLX2 strain , sequence analysis was carried out on KLX2 and the paternal strain LT2 , for the genes known to be involved in controlling intracellular levels of RpoS ( relA , spoT , hfq , hns , arcA , arcB , ﬁs , cspC , cspE , rprA , rpoE , rssA , rssB , iraP , iraD , iraM , clpX , clpP , and dnaK ) and the rpoS gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23385142" target="_blank">23385142</a>]<br></li>
<li>Using the topA promoter from both species , we show that a combination of different DNA superhelical densities and the presence of the FIS NAP modulate the expression proﬁle of the topA gene at the level of transcription such that it responds differently in E. coli and S. enterica to salt stress , aeration and the presence of an ami-nocoumarin antibiotic , when these parameters are varied individually or in combination .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21276095" target="_blank">21276095</a>]<br></li>
<li>Thus , although the maximum ﬂjB mRNA levels are similar for wild-type and ﬁs cells at 120 min after subculturing in LB medium , the absence of Fis seems to affect the temporal regulation pattern of the ﬂjB promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7536730" target="_blank">7536730</a>]<br></li>
<li>The results showed that the SPI-2 promoters were signiﬁcantly less active in the absence of Fis , in agreement with the DNA microarray data .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15256548" target="_blank">15256548</a>]<br></li>
<li>This result suggested that although the absence of RpoS leads to higher levels of ﬁs transcription than those observed in the wild type , other factors play a role in the speciﬁc increase in transcription of ﬁs that is seen in the absence of aeration .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17784910" target="_blank">17784910</a>]<br></li>
<li>Thus , the reduction in the repression efﬁciency of P. vulgaris ﬁs P can be attributed to the loss of Fis site I in this promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9811652" target="_blank">9811652</a>]<br></li>
<li>To conﬁrm that the increase in rpoS transcription in the ﬁs mutant was due to the absence of Fis , the exponential-phase activity of PrpoS-lac [ op ] was measured in wild-type and ﬁs mutant backgrounds harboring either the single-copy ﬁs expression plasmid , pFis349 , or the empty vector control pGS349 ( 32 , 71 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15716427" target="_blank">15716427</a>]<br></li>
<li>This was in contrast to the upregulation of the pdu genes in the ﬁs mutant .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15256548" target="_blank">15256548</a>]<br></li>
<li>The elevation of PrpoS activity in the ﬁs mutant during exponential phase eliminated SP transcriptional induction , an effect that can be totally complemented by plasmid-encoded Fis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15716427" target="_blank">15716427</a>]<br></li>
<li>As it is unlikely that RpoS represses the ﬁs promoter directly , repression may involve an indirect negative effect in which RpoS upregulates an as-yet uni-dentiﬁed negative regulator of Fis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17784910" target="_blank">17784910</a>]<br></li>
<li>It appears that genes in cluster 2A are more under fis controlled than ssrAB control perhaps because their expression is more sensitive to DNA relaxation than the expression of genes in cluster 2B which appear under stringent ssrAB control .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20861532" target="_blank">20861532</a>]<br></li>
<li>Changes in SPI expression suggest the possibility that regulation of genes in SPI-2 as well as SPI-1 is affected by Fis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14553938" target="_blank">14553938</a>]<br></li>
<li>Fis is involved in SPI expression Positive regulation of SPI-1 genes by Fis was expected considering the fact that hilA , a transcriptional-regulator of SPI-1 , is upregulated by Fis [ 16 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14553938" target="_blank">14553938</a>]<br></li>
<li>None of the Fis-responsive genes found in this study is regulated by Fis alone .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15256548" target="_blank">15256548</a>]<br></li>
<li>This indicates that in LT2 Fis preferentially mediates positively regulation of genes , which is opposite from that in K12 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23717649" target="_blank">23717649</a>]<br></li>
<li>In this work , we found that glucose induces the fis expression in S. Typhi , suggesting that Fis is also regulated by its level in S. enterica ( Figure 1E ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24885225" target="_blank">24885225</a>]<br></li>
<li>In B. subtilis , the small DNA binding protein AbrB resembles Fis in that it is transcriptionally activated by environmental conditions in a growth-phase-dependent manner , particularly in the period spanning the transition between the lag and exponential-growth-phases ( 40 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9811652" target="_blank">9811652</a>]<br></li>
<li>In the present study we show that the Fis protein ( factor for inversion stimulation ) is involved in RpoS regulation during growth in rich medium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15716427" target="_blank">15716427</a>]<br></li>
<li>One of these promoters , proP1 , is activated independently of Fis in response to increases in medium osmolarity , whereas the second promoter , proP2 , is activated by high medium osmolarity in late exponential phase growth .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11123690" target="_blank">11123690</a>]<br></li>
<li>These results indicate that Fis is also involved in the regulation of invF expression in a HilA-independent manner .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11123690" target="_blank">11123690</a>]<br></li>
<li>However , in the presence of Fis , an in vivo regulatory pattern for ﬂjB that is different from that in cells unable to express ﬁs is apparent .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7536730" target="_blank">7536730</a>]<br></li>
<li>Only 145 common genes are bound by Fis in both K12 and LT2 , which is possibly due to the difference in their genome sequence , as 320 of the 943 Fis-binding genes in LT2 are not present in K12 [ 24 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23717649" target="_blank">23717649</a>]<br></li>
<li>These observations suggest that the high-afﬁnity Fis sites I and II in this region may not be sufﬁcient to cause the 10-fold regulation by Fis observed in E. coli , since these are also present in S. typhimurium , in which only a 2.5-fold regulation is observed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7536730" target="_blank">7536730</a>]<br></li>
<li>However , in these constructs , Fis repression could be detected in all the promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9811652" target="_blank">9811652</a>]<br></li>
<li>Furthermore , some differences were observed in functional loci , such as the Fis binding site in the operator of marRAB in E. coli , which is absent in Salmonella .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19120970" target="_blank">19120970</a>]<br></li>
<li>Since Fis has been implicated in catabolite repression in E. coli [ 20,21 ] , we assessed the role of Fis in hlyE transcription in S. Typhi by RT-PCR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24885225" target="_blank">24885225</a>]<br></li>
<li>Here we report that , contrary to predictions based on the classic pattern of Fis protein expression , Fis is expressed at elevated levels in S. Typhimurium bacteria grown to stationary phase under non-aerated , SPI1-inducing conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17784910" target="_blank">17784910</a>]<br></li>
<li>As Fis has been shown to modulate growth-phase-specific changes in the supercoiling level of DNA ( Schneider et al. , 1997 ) , perhaps the effect of Fis on SPI-1 gene regulation is mediated directly through changes in DNA supercoiling at SPI-1 regulatory gene promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11123690" target="_blank">11123690</a>]<br></li>
<li>In vitro analysis has shown that Hin-mediated DNA inversion is stimulated over 150-fold in the presence of Fis ( 26 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7536730" target="_blank">7536730</a>]<br></li>
<li>An increasing number of functions are being assigned to the Fis protein ( factor for inversion stimulation ) in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7536730" target="_blank">7536730</a>]<br></li>
<li>We tried to identify genes under Fis control involved in pathogenicity of Sal-monella in order to understand the role of Fis in Salmo-nella virulence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14553938" target="_blank">14553938</a>]<br></li>
<li>33 Homologue of virK in Shigella The effect of the ﬁs mutation on speciﬁc SPI-2 genes Transcriptional fusions to virulence genes in the SPI-2 pathogenicity island were tested individually for Fis activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15256548" target="_blank">15256548</a>]<br></li>
<li>The absence of an explicit RpoS effect in aerated , non-aerated , and anaerobic conditions ( Figures 2E , 4C & D ) , suggests a model in which RpoS represses fis expression only when RpoS protein levels are unusually high , as in a ΔarcA mutant .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24358360" target="_blank">24358360</a>]<br></li>
<li>These results revealed a negative correlation between expression of RpoS and Fis under non-aerated growth conditions in stationary-phase cultures .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17784910" target="_blank">17784910</a>]<br></li>
<li>Mallik P , Paul BJ , Rutherford ST , Gourse RL , Osuna R ( 2006 ) DksA is required for growth-phase-dependent regulation , growth rate-dependent control , and stringent control of fis expression in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24358360" target="_blank">24358360</a>]<br></li>
<li>The redox sensors ArcAB and FNR regulate transitions from aerobic to anaerobic-growth , making these proteins prime candidates for regulating the sustained expression of fis in low oxygen environments .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24358360" target="_blank">24358360</a>]<br></li>
<li>ArcA is a repressor of S. Typhimurium fis The redox sensors ArcAB and FNR are transcriptional-activators in low oxygen conditions , making them prime candidates for activating sustained fis expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24358360" target="_blank">24358360</a>]<br></li>
<li>The Small Nucleoid-Binding Proteins H-NS , HU , and Fis Affect hilA Expression in Salmonella enterica Serovar Typhimurium Lisa M. Schechter , * Sumita Jain , † Samina Akbar , ‡ and Catherine A. Lee Department of Microbiology and Molecular Genetics , Harvard Medical School , Boston , Massachusetts 02115 Received 1 November 2002/Returned for modiﬁcation 3 February 2003/Accepted 27 May 2003 hilA encodes an activator of Salmonella enterica serovar Typhimurium virulence genes and is transcriptionally modulated by environmental conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12933899" target="_blank">12933899</a>]<br></li>
<li>For example , Fis stimulates transcription of rRNA and tRNA genes and genes for proteins involved in translation ( 19 , 35 , 40 , 51 , 56 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7536730" target="_blank">7536730</a>]<br></li>
<li>FIS is required for full activation of both SPI-1 and SPI-2 genes in laboratory conditions [ 7 ] , and Dfis mutants are attenuated for virulence in mice [ 8 ] and show reduced survival in macrophage [ 9 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22457642" target="_blank">22457642</a>]<br></li>
<li>ArcA and FNR regulate dozens and hundreds of genes , respectively , many of which are required for metabolism in anaerobic conditions , but neither transcription factor has been implicated in controlling fis expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24358360" target="_blank">24358360</a>]<br></li>
<li>The E. coli , P. vulgaris , H. inﬂuenzae , and E. carotovora ﬁs resulted in 255 , 196 , 326 , and 493 U of b-galactosidase activity , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9811652" target="_blank">9811652</a>]<br></li>
<li>Factor for inversion stimulation ( ﬁs ) has been shown to be involved in DNA transcription , DNA replication at oriC , transposition , invasion of HEp-2 cells , and transcriptional activation and repression of several genes ( Kelly et al. , 2004 ; ) and DluxS mutant ( M ) and between wild-type cell-free supernatants and DluxS Table 6 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19909098" target="_blank">19909098</a>]<br></li>
<li>Fis has been implicated in many other processes such as stimulation of excision and integration of lambda ( 4 , 5 , 20 , 67 ) , DNA replication at oriC ( 24 , 26 , 58 ) , transposition ( 70 ) , invasion of HEp-2 cells ( 71 ) , and transcriptional activation and repression of several genes , including hns , leuV , gyr , tyrT , proP , nuo , osmE , and rRNA operons ( 3 , 12 , 21 , 33 , 56 , 68 , 72 , 74 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15716427" target="_blank">15716427</a>]<br></li>
<li>Fis activates transcription of rRNA and many other genes , including some involved in replication , so it is conceivable that Fis also works indirectly .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15716427" target="_blank">15716427</a>]<br></li>
<li>Conversely , constructs in which Fis site III was altered or deleted ( Fig. 7 , constructs B , C , and G were independent of regulation by Fis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15716427" target="_blank">15716427</a>]<br></li>
<li>Xu , J. , and Johnson , R.C. ( 1995 ) Identiﬁcation of genes negatively regulated by Fis : Fis and RpoS comodulate growth-phase-dependent gene expression in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16999831" target="_blank">16999831</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17784910" target="_blank">17784910</a>]<br></li>
<li>Published transcriptomic datasets reveal that SPI1 and ﬂa-gellar genes are coregulated in response to a number of regulatory signals , including CsrA , YfgL , Fnr , Fis , and RfaH , as well as several environmental conditions ( Table S3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22021388" target="_blank">22021388</a>]<br></li>
<li>Fis Regulates Transcriptional Induction of RpoS in Salmonella enterica Matthew Hirsch and Thomas Elliott * ent of M Departm icrobiology , Immunology , and Cell Biology , West Virginia University , M Received 23 August 2004/Accepted 19 November 2004 organtown , West Virginia The sigma factor RpoS is known to regulate at least 60 genes in response to environmental sources of stress or during growth to stationary phase ( SP ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15716427" target="_blank">15716427</a>]<br></li>
<li>Levels of Fis in E. coli vary enormously in response to changing nutritional conditions and this variation may be important for its physiological roles ( 4 , 42 , 43 , 57 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7536730" target="_blank">7536730</a>]<br></li>
<li>Because HilD is required for hilA expression ( Schechter et al. , 1999 ) , and current models suggest that HilD is also required for invF expression independently of hilA ( Lucas and Lee , 2000 ) , our data are consistent with a model in which hilD is induced by Fis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11123690" target="_blank">11123690</a>]<br></li>
<li>Fis is also required for oriC-directed DNA replication and inﬂuences the topolo-gical state of DNA in the cell by repressing DNA gyrase and activating topoisomerase I gene expression ( Gonzalez-Gil et al. , 1996 ; Ross et al. , 1990 ; Schneider et al. , 1999 ; Weinstein-Fischer et al. , 2000 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15256548" target="_blank">15256548</a>]<br></li>
<li>Fis and intramacrophage survival The SPI-2 genes that are regulated by the ssrA and ssrB regulatory genes are required for adaptation to and survival within the harsh intracellular environment encountered by Salmonella in macrophage ( Ochman et al. , 1996 ; Cirillo et al. , 1998 ; Hensel et al. , 1998 ; Hensel , 2000 ; Waterman and Holden , 2003 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16999831" target="_blank">16999831</a>]<br></li>
<li>The nucleoid-associated proteins Fis and YdgT are involved in positive and negative regulation of SPI-2 ( Kelly et al. , 2004 ; Coombes et al. , 2005 ) respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630976" target="_blank">17630976</a>]<br></li>
<li>Results Invasion of epithelial cells and Fis expression during growth with or without aeration The genes in the SPI1 pathogenicity island of S. Typhimurium are known to be Fis-dependent for expression ( Wilson et al. , 2001 ; Schechter et al. , 2003 ; Kelly et al. , 2004 ) , and previous data indicated that these genes are induced when bacteria are grown to late exponential-growth-phase without aeration ( Lee and Falkow , 1990 ; Song et al. , 2004 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17784910" target="_blank">17784910</a>]<br></li>
<li>Co-factors such as Hha and YdgT , and other nucleoid associated proteins like Fis contribute to the expression of horizontally acquired genes in Salmonella , including those that regulate virulence ( Schechter et al. , 2003 ; Silphaduang et al. , 2007 ; Vivero et al. , 2008 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29034217" target="_blank">29034217</a>]<br></li>
<li>Alternatively , because Fis is involved in the transcriptional regulation of other nucleoid-binding proteins such as HU ( Claret and RouvieÁre-Yaniv , 1996 ) and H-NS ( Falconi et al. , 1996 ) , as well as gyrase ( Schneider et al. , 1999 ) and topoisomerase I ( Weinstein-Fischer et al. , 2000 ) , its contribution to SPI-1 gene regulation could be through altering the expression of these genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11123690" target="_blank">11123690</a>]<br></li>
<li>Thus , global supercoiling homeostasis is maintained in part by FIS 's central control over GyrAB and TopA levels , and also in part by the repressive effect of each enzyme 's activity on its antagonist 's promoter : gyrA and gyrB are induced by DNA relaxation whereas topA in induced by increased negative supercoiling ( Tse-Dinh , 1985 ; Menzel and Gellert , 1987 ; Peter et al. , 2004 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21276095" target="_blank">21276095</a>]<br></li>
<li>Recently , it was reported that CspA in E. coli was induced during early exponential phase ( Brandi et al. 1999 ; Yamanaka and Inouye 2001 ) and that high concentrations of its transcriptional-activator Fis positively regulated its expression both in vivo and in vitro ( Brandi et al. 1999 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15235764" target="_blank">15235764</a>]<br></li>
<li>For example , an interesting observation made by Kelly et al. ( 52 ) was that Fis , a global transcriptional-regulator that controls both metabolism and type III secretion systems in S. typhimurium , appeared to be abundant in log phase and nearly absent in stationary phase , consistent with our results .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16684765" target="_blank">16684765</a>]<br></li>
<li>This represents a point in the growth curve where Fis is poorly detected in E. coli ( Ball et al. , 1992 ; Appleman et al. , 1998 ; Keane and Dorman , 2003 ) , although approximately 10 000 Fis dimers were found to be present in an LT2 culture grown under aerated conditions to late logarithmic stage ( Osuna et al. , 1995 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17784910" target="_blank">17784910</a>]<br></li>
<li>Fis promoter is activated by high negative superhelicity of the DNA in vivo and in vitro ( Schneider et al. , 2000 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25033662" target="_blank">25033662</a>]<br></li>
<li>Genes encoding ﬂagella [ ﬂi and ﬂg genes , ( Barak et al. , 2005 ; Jeter and Matthysse , 2005 ; Torres et al. , 2005 ) ] were not found to be more transcribed in presence of sprouts suggesting a more important role for Fis in regulation of the T3SS in our experiment .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24308841" target="_blank">24308841</a>]<br></li>
<li>Our discovery that Fis regulates the expression of all three type III secretion systems in Salmonella is in keeping with other studies that have pointed to regulatory overlaps between virulence genes and ﬂagella in Salmonella ( Eichelberg & Galan , 2000 ; Ellermeier & Slauch , 2003 ; Lawhon et al. , 2003 ) and other bacteria ( Goodier & Ahmer , 2001 ; Grant et al. , 2003 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15256548" target="_blank">15256548</a>]<br></li>
<li>In the presence of lacIq , ﬁs is constitutively expressed from pRJ807 ; therefore , IPTG was not added .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7536730" target="_blank">7536730</a>]<br></li>
<li>enhancer Fis originated by horizontal gene transfer from the transcriptional-regulator NtrC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23717649" target="_blank">23717649</a>]<br></li>
<li>Transcription from the ﬁs promoter ( ﬁs P ) is negatively regulated by Fis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9811652" target="_blank">9811652</a>]<br></li>
<li>The ﬁs promoter is stimulated by negative supercoiling of the DNA ( Schneider et al. , 2000 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15256548" target="_blank">15256548</a>]<br></li>
<li>The ﬁs gene is repressed by the stringent response and is negatively auto-regulated by the Fis protein ( Ninnemann et al. 1992 ; Walker et al. 1999 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12898222" target="_blank">12898222</a>]<br></li>
<li>Transcription of the ﬁs gene is inﬂuenced by the stringent response , is autoregulated by Fis protein and is controlled by the intracellular concentration of cytosine triphosphate ( Ball et al. , 1992 ; Walker et al. , 1999 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15256548" target="_blank">15256548</a>]<br></li>
<li>However , their promoter strengths vary , as do their responses to Fis autoregulation and integration host factor stimulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9811652" target="_blank">9811652</a>]<br></li>
<li>We believe that Fis probably acts directly as a repressor given that a speciﬁc site positioned at 50 is necessary for complete repression and that this site is speciﬁcally bound in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15716427" target="_blank">15716427</a>]<br></li>
<li>Fis ( factor for inversion stimulation ) is a nucleoid-associated protein in E. coli that regulates recombination , replication , and transcription reactions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14502413" target="_blank">14502413</a>]<br></li>
<li>H-NS regulates its own gene negatively , and this repression is antagonized by Fis , a relationship that is seen at many other H-NS-repressed promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15790293" target="_blank">15790293</a>]<br></li>
<li>Fis is a histone-like DNA binding protein that was cents rst identi cents ed as a factor from Escherichia coli that stimulates site-speci cents c DNA inversion reactions mediated by S. typhi-murium His [ 9 ] and by phage-derived Gin and Cin [ 10,11 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14553938" target="_blank">14553938</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_352">
aldB , an RpoS-dependent gene in Escherichia coli encoding an aldehyde dehydrogenase that is repressed by Fis and activated by Crp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15256548" target="_blank">15256548</a>]<br></label>
<input id="act_similarity_352", type="checkbox">
<div class="ACT_similar">
<li>aldB , an RpoS-dependent gene in Esche-richia coli encoding an aldehyde dehydrogenase that is repressed by Fis and activated by Crp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8752327" target="_blank">8752327</a>]<br></li></div>
<li>Wild-type and ﬁs mutant strains are , respectively , as follows : A , TE8744 and TE8764 ; B , TE8899 and TE8900 ; C , TE8887 and TE8888 ; D , TE8971 and TE8972 ; E , TE8868 and TE8869 ; F , TE9083 and TE9096 ; G , TE8947 and TE8949 ; H , TE6676 and TE8766 ; and I , TE6675 and TE8948 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15716427" target="_blank">15716427</a>]<br></li>
<li>Thus , the hup and ﬁs mutations may decrease hilA expression ( i ) by decreasing its derepression by HilD and/or ( ii ) by increasing its repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12933899" target="_blank">12933899</a>]<br></li>
<li>Fis Regulates B12 Biosynthesis Genes by Binding to and Activating PocR B12 is a known cofactor for numerous enzymes mediating methylation , reduction , and intramolecular rearrangements [ 59 -- Figure 6 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23717649" target="_blank">23717649</a>]<br></li>
<li>We suspected that small nucleoid-binding proteins , such as H-NS , HU , and Fis , might control hilA expression by affecting its repression and/or derepression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12933899" target="_blank">12933899</a>]<br></li>
<li>Recent genetic evidence suggests that H-NS can repress hilA under certain conditions while HU and Fis help activate hilA expression ( L. M. Schechter and C. A. Lee , unpublished results , and reference 72 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11292791" target="_blank">11292791</a>]<br></li>
<li>Furthermore , other DNA-bending modulators such as IHF and Fis counteract H-NS repression and support virulence gene activation by the virulence cluster/PAI-speci ﬁc activator proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26441883" target="_blank">26441883</a>]<br></li>
<li>For example , # agellin proteins ( FljB and FliC ) and SPI-related proteins ( SipA , SseC , and SopE2 ) are upregulated by Fis regardless of bacterial growth-phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14553938" target="_blank">14553938</a>]<br></li>
<li>Transcription of hilA is subject to direct or indirect negative control by multiple nucleoid binding proteins , including Hha , H-NS , HU , and Fis ( 10 , 23 ) , and other proteins , including HilE , Pag , and Ams ( 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15126488" target="_blank">15126488</a>]<br></li>
<li>Like Fis , HU , and IHF , H-NS is not thought to-bind any ligand that would alter its activity , and unlike these other proteins , H-NS levels in the cell are relatively constant .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15790293" target="_blank">15790293</a>]<br></li>
<li>In particular , the abundant nucleoid-associated proteins such as IHF , FIS , HU , and H-NS are known to modulate DNA supercoiling globally and locally ( reviewed in 12 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12882514" target="_blank">12882514</a>]<br></li>
<li>Of the small nucleoid-bind-ing proteins , H-NS and Hha can repress hilA expression , but FIS and HU may activate ( 12 ; citation in references 30 and 47 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12107132" target="_blank">12107132</a>]<br></li>
<li>We found that PompR is also activated by low concentrations of novobiocin during exponential-growth , and FIS and OmpR-EnvZ contribute to this induction ( Figure 1B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22457642" target="_blank">22457642</a>]<br></li>
<li>Here we describe a fundamental mechanism that activates both SPI-2 and SPI-1 promoters through changes in DNA topology , and this mechanism depends on OmpR and FIS but is independent of SsrB and HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22457642" target="_blank">22457642</a>]<br></li>
<li>When FIS levels decrease as growth slows , RpoS gains access to the dps promoter and transcription is up-regulated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24358360" target="_blank">24358360</a>]<br></li>
<li>In S. enterica there is a negative correlation between RpoS and FIS levels , so deleting rpoS results in greatly increased S. enterica FIS levels ( Ó Cróinín and Dorman , 2007 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21276095" target="_blank">21276095</a>]<br></li>
<li>However , hilD were not found to be bound by Fis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23717649" target="_blank">23717649</a>]<br></li>
<li>We proposed that hilD was probably indirectly regulated by Fis through other proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23717649" target="_blank">23717649</a>]<br></li>
<li>b BA class includes genes which are activated by Fis both in LT2 and K12 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23717649" target="_blank">23717649</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23717649" target="_blank">23717649</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_372">
d AR class includes genes which are activated by Fis in LT2 , but repressed in K12 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23717649" target="_blank">23717649</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23717649" target="_blank">23717649</a>]<br></label>
<input id="act_similarity_372", type="checkbox">
<div class="ACT_similar">
<li>e RA class includes genes which are repressed by Fis in LT2 , but activated in K12 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23717649" target="_blank">23717649</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23717649" target="_blank">23717649</a>]<br></li></div>
<li>Model for Salmonella SPI-1 gene regulation by Fis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11123690" target="_blank">11123690</a>]<br></li>
<li>Categories of genes regulated by Fis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15256548" target="_blank">15256548</a>]<br></li>
<li>Activation Repression Total Silent c Total Direct a 943 Indirect 2 b Total 2 2 Analysis of Fis-regulated Genes from RNA-seq Data a Genes which are both regulated and bound by Fis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23717649" target="_blank">23717649</a>]<br></li>
<li>a Genes which are both regulated and bound by Fis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23717649" target="_blank">23717649</a>]<br></li>
<li>Almost all genes encoding needle complex structural proteins and SPI regulators were activated by Fis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23717649" target="_blank">23717649</a>]<br></li>
<li>In the other four COGs , there were more genes repressed by Fis than those activated by Fis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23717649" target="_blank">23717649</a>]<br></li>
<li>Among 23 COGs , 17 contained more genes activated by Fis than those repressed by Fis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23717649" target="_blank">23717649</a>]<br></li>
<li>0 indicate genes normally activated by Fis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15256548" target="_blank">15256548</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15256548" target="_blank">15256548</a>]<br></li>
				</div></ul><br>
			<label class="collapse_DOM" for="_2"><font size="5px">FIS protein domains</font></label>
				<input id="_2" type="checkbox">
				<div class="DOM"><br><ul><li>BLAST searches revealed additional organisms that contain ORF1 and ﬁs .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9811652" target="_blank">9811652</a>]<br></li>
<li>Fis binding was demonstrated to at least three sites ( Fig. 8 ) , suggesting a nucleoprotein complex forms near PrpoS in vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15716427" target="_blank">15716427</a>]<br></li>
<li>The only transcription factor in cluster 2B is Fis protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20861532" target="_blank">20861532</a>]<br></li>
				<label class="DOM_collapse_similarity" for="dom_similarity_4">
J Bacteriol 179:7410 -- 7419 McLeod SM , Aiyar SE , Gourse RL , Johnson RC ( 2002 ) The C-ter-minal domains of the RNA polymerase alpha subunits : contact site with Fis and localization during co-activation with CRP at the Escherichia coli proP P2 promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15235764" target="_blank">15235764</a>]<br></label>
<input id="dom_similarity_4", type="checkbox">
<div class="DOM_similar">
<li>The C-terminal domains of the RNA polymerase alpha subunits : contact site with Fis and localization during co-activation with CRP at the Escherichia coli proP P2 promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15716427" target="_blank">15716427</a>]<br></li>
<li>McLeod , S.M. , Aiyar , S.E. , Gourse , R.L. , and Johnson , R.C. ( 2002 ) The C-terminal domains of the RNA polymerase alpha subunits : contact site with Fis and localization during co-activation with Crp at the Escherichia coli proP P2 promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16999831" target="_blank">16999831</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17784910" target="_blank">17784910</a>]<br></li></div>
				<label class="DOM_collapse_similarity" for="dom_similarity_7">
Zhi H , Wang X , Cabrera JE , Johnson RC , Jin DJ ( 2003 ) Fis stabilizes the interaction between RNA polymerase and the ribosomal promoter rrnB P1 , leading to transcriptional activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23717649" target="_blank">23717649</a>]<br></label>
<input id="dom_similarity_7", type="checkbox">
<div class="DOM_similar">
<li>Fis stabilizes the interaction between RNA polymerase and the ribosomal promoter rrnB P1 , leading to transcriptional activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15716427" target="_blank">15716427</a>]<br></li></div>
<li>Fis is a basic homodimer , with each subunit consisting of 98 amino acids .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9811652" target="_blank">9811652</a>]<br></li>
<li>It has been recently discovered that together OmpR and FIS drive low-level transcription of SPI-2 in the intestinal lumen , an environment classically thought to be the exclusive domain of SPI-1 [ 12 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22457642" target="_blank">22457642</a>]<br></li>
<li>We note that Fis was involved in three of the four cases in which RITM was altered by the loss of a regulator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16980465" target="_blank">16980465</a>]<br></li>
<li>The crystal structure of Fis revealed that each monomer consisted of four connected a-helices , with the ﬁrst 19 amino acids being disordered in the crystal ( 31 , 63 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7536730" target="_blank">7536730</a>]<br></li>
<li>An ORF ( ORF1 ) upstream of ﬁs that consists of 321 amino acids is also well conserved between the two species .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7536730" target="_blank">7536730</a>]<br></li>
<li>E. carotovora Fis contained only two amino acid changes with respect to Fis in all other enteric bacteria tested : residue 20 is alanine instead of the more common aspartic acid , and residue 48 is serine instead of asparagine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9811652" target="_blank">9811652</a>]<br></li>
<li>P. vulgaris also contained two amino acid differences compared to Fis in the other enteric bacteria : residue 14 is alanine instead of the more common serine , and residue 79 is glutamine instead of leucine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9811652" target="_blank">9811652</a>]<br></li>
<li>The gyrB region I contained two matches to the Fis binding site consensus while region II had one ( Fig. 5D ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12898222" target="_blank">12898222</a>]<br></li>
<li>The ﬁs mutation Arg-853His is located within the helix-turn-helix motif and has been shown to severely reduce DNA-binding activity , although protein expression is normal ( 48 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7536730" target="_blank">7536730</a>]<br></li>
<li>The nucleoid-associated protein Fis ( Factor for Inversion Stimulation ) relies on an HTH motif for DNA-binding and it binds as a dimer .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28631437" target="_blank">28631437</a>]<br></li>
				<label class="DOM_collapse_similarity" for="dom_similarity_19">
The molecular structure of wild-type and a mutant Fis protein : relationship between mutational changes and recombinational enhancer function or DNA-binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9811652" target="_blank">9811652</a>]<br></label>
<input id="dom_similarity_19", type="checkbox">
<div class="DOM_similar">
<li>The molecular structure of wild-type and a mutant Fis protein -- relationship between mutational changes and recombinational enhancer function or DNA-binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7536730" target="_blank">7536730</a>]<br></li></div>
<li>The transactivation region of the Fis protein that controls site-speciﬁc DNA inversion contains extended mobile b-hairpin arms .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9811652" target="_blank">9811652</a>]<br></li>
<li>This residue forms part of the ﬁrst helix of the helix-turn-helix DNA-binding motif ( helix C ) , and Gln at this position might somehow interfere with Fis binding at the l Fis site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9811652" target="_blank">9811652</a>]<br></li>
<li>A carboxy-terminal region , which included a helix-turn-helix DNA-binding motif , was required for efﬁcient DNA-binding and bending and hence was essential for the stimulation of Hin-mediated DNA inversion , l DNA excision from the chromosome , and repression of the ﬁs promoter ( 5 , 29 , 41 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9811652" target="_blank">9811652</a>]<br></li>
<li>Mutation analysis indicated the presence of at least two functional regions in Fis : a carboxy-terminal region required for efﬁcient DNA-binding and bending and a region near the amino terminus that is required for stimulation of DNA inversion but not for l DNA excision ( 28 , 48 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7536730" target="_blank">7536730</a>]<br></li>
<li>The DNA-binding activity of Fis was also found to be sufﬁcient for its function as an autoregulator but not for its role in transcription-activation of rrnB P1 ( 4 , 19 , 28 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7536730" target="_blank">7536730</a>]<br></li>
<li>DNA-binding function of Fis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7536730" target="_blank">7536730</a>]<br></li>
				</div></ul><br>
			<label class="collapse_EVO" for="_3"><font size="5px">FIS evolution</font></label>
				<input id="_3" type="checkbox">
				<div class="EVO"><br><ul><li>The response regulator OmpR , the MarR homologue SlyA , and the nucleoid protein Fis contrib-ute directly to activate the ssrA-ssrB SPI-2 regulatory locus , which encodes a two-component regulatory system ( 15 -- 18 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21059643" target="_blank">21059643</a>]<br></li>
<li>Thus , for 60 % of the ORFs showing an up or down response to the absence of Fis at 1 h ( the time point at which the protein was most abundant in the wild-type ) the response was transient .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15256548" target="_blank">15256548</a>]<br></li>
				<label class="EVO_collapse_similarity" for="evo_similarity_3">
The deduced amino acid sequence for H. inﬂuenzae Fis is about 80 % identical and 90 % similar to Fis in enteric bacteria .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9811652" target="_blank">9811652</a>]<br></label>
<input id="evo_similarity_3", type="checkbox">
<div class="EVO_similar">
<li>The deduced Fis amino acid sequence for H. inﬂuenzae is about 80 % identical and 90 % similar to Fis in enteric bacteria .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9811652" target="_blank">9811652</a>]<br></li></div>
<li>H. inﬂuenzae ﬁs is about 67 to 73 % identical to the various ﬁs sequences in enteric bacteria .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9811652" target="_blank">9811652</a>]<br></li>
<li>In contrast , the ﬁs P regions from 255 to 2135 in these bacteria were only 32 to 70 % identical , averaging 47 % identity overall .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9811652" target="_blank">9811652</a>]<br></li>
<li>A comparison of these ﬁs sequences together with those of E. coli , S. typhimurium , and H. inﬂuenzae showed 85 to 98 % , identity with sequences from K. pneumoniae , S. typhimurium , and E. coli showing 97 to 98 % identity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9811652" target="_blank">9811652</a>]<br></li>
<li>The carboxy-terminal regions corresponding to the a-helices C and D in Fis show 45 , 55 , and 70 % identities with NTRC protein from Klebsiella pneumoniae , R. capsulatus , and Bradyrhizobium parasponiae , respectively ( 14 , 25 , 44 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7536730" target="_blank">7536730</a>]<br></li>
				</div></ul><br>
			<label class="collapse_RP" for="_4"><font size="5px">Processes regulated by FIS</font></label>
				<input id="_4" type="checkbox">
				<div class="RP"><br><ul><li>Fis has a moderate effect on gyr gene-expression , inﬂuencing gyr transcription over an approximately two-fold range .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12898222" target="_blank">12898222</a>]<br></li>
<li>The global regulator Fis had no significant effect on acnA expression ( Fig. 3a ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23637460" target="_blank">23637460</a>]<br></li>
<li>Selected groups of Salmonella Typhimurium SL1344 genes with signiﬁcant ( P ,0.05 ) differential expression in response to triclosan exposure Fold change a Gene name Product Drug detoxiﬁcation acrE acrR ompC ompF ompH ompR sodA sodC tolC fumC acnA fpr acriﬂavin resistance protein E acrAB operon repressor 1.63 1.87 2.31 0.83 1.27 1.43 1.58 1.46 1.50 2.24 1.99 1.45 outer membrane porin C outer membrane porin F outer membrane protein OmpH precursor two-component response regulator manganese superoxide dismutase copper-zinc superoxide dismutase outer membrane protein fumarate hydratase class II aconitate hydratase 1 ( citrate hydrolyase 1 ) ferredoxin-NADP reductase Curli biosynthesis csgA csgDEFG major curlin subunit precursor 1.63 curli biosynthesis and regulation thereof 3.11 -- 6.06 SPI-1 sicP chaperone ( associated with virulence ) probable acyl carrier protein iron-transport proteins possible AraC-family transcriptional regulator hypothetical protein hypothetical protein tyrosine phosphatase ( associated with virulence ) 1.27 1.40 sipF sitA -- D sprA STM2904 STM2905 stpA 2.33 -- 3.42 1.52 1.77 1.79 1.40 SPI-2 spiC putative pathogenicity island 2 secreted effector protein putative pathogenicity island protein putative secretion system protein putative pathogenicity island protein putative pathogenicity island lipoprotein putative secretion system protein putative type-III-secretion proteins putative type-III-secretion protein putative type-III-secretion-system chaperone protein putative pathogenicity island protein putative pathogenicity island effector proteins putative two-component sensor kinase putative pathogenicity island protein putative pathogenicity island protein 1.74 2.62 ssaD ssaE ssaI ssaJ ssaL ssaN -- Q ssaV sscA sscB sseA -- E ssrA STM1389 2.08 1.30 1.32 1.63 1.34 -- 1.72 1.90 1.85 1.77 1.81 -- 2.32 1.61 3.79 1.45 STM1410 Fatty acid biosynthesis fabI fabA fabB fabG ) enoyl - ( acyl carrier protein ) reductase ( NADH D-3-hydroxydecanoyl - ( acyl carrier protein ) 3-oxoacyl - ( acyl carrier protein ) synthase I 3-oxoacyl - ( acyl carrier protein ) reductase 0.82 0.47 0.18 0.73 Global regulators ﬁs fnr phoP phoQ rpoS soxS DNA-binding protein fumarate and nitrate reduction regulatory protein 0.40 0.84 0.83 0.72 2.30 1.70 inants transcriptional regulatory protein PhoP , regulator of virulence determ sensor protein PhoQ , regulator of virulence determinants stress response sigma factor superoxide stress response regulator Continued Table 2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19759044" target="_blank">19759044</a>]<br></li>
<li>List of genes Description List of genes Description sseE Secreted effector protein sscB Secretion system chaparone ssaR/yscR Type III-secretion-system protein ssaV Secretion system apparatus protein SsaV fis Global DNA-binding transcriptional dual regulator sscA/cesD Putative Type III-secretion-system chaperone protein ssaN Flagellum-specific ATP synthase sseB Secreted protein EspA ssaJ Needle complex inner membrane lipoprotein SspH2 Leucine-rich repeat protein spiA/ssaC Putative outer membrane secretory protein ssaD Putative pathogenicity island protein sseD Translocation machinery component ssaS Flagellar biosynthesis protein Q sifA Secreted effector protein ssaQ Flagellar motor switch/type III secretory pathway protein Figure 13 : Cluster 7A .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20861532" target="_blank">20861532</a>]<br></li>
<li>Fis expression is subject to complex control .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12898222" target="_blank">12898222</a>]<br></li>
<li>Fis was also shown to stimulate phage l DNA excision and integration ( 2 , 3 , 57 ) , which is a mechanistically different DNA-recombination reaction ( 8 , 34 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7536730" target="_blank">7536730</a>]<br></li>
<li>The ﬁs gene encodes a factor for inversion stimulation , a DNA binding protein involved in growth phase-dependent regulation ( Sheikh et al. , 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16842356" target="_blank">16842356</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_8">
Fis , a DNA nucleoid-associated protein , is involved in Salmonella Typhimurium SPI-1 invasion gene-expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25547794" target="_blank">25547794</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19411325" target="_blank">19411325</a>]<br></label>
<input id="rp_similarity_8", type="checkbox">
<div class="RP_similar">
<li>Fis , a DNA nucleiod-associated protein , is involved in Salmonella typhimurium SPI-1 invasion gene-expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11292791" target="_blank">11292791</a>]<br></li>
<li>Fis , a DNA nucleoid-associated protein , is involved in Salmonella typhimurium SPI-1 invasion gene-expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23554419" target="_blank">23554419</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14645287" target="_blank">14645287</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15256548" target="_blank">15256548</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26441883" target="_blank">26441883</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16443238" target="_blank">16443238</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15716427" target="_blank">15716427</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15968047" target="_blank">15968047</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12595445" target="_blank">12595445</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19074398" target="_blank">19074398</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12107132" target="_blank">12107132</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923082" target="_blank">12923082</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11741864" target="_blank">11741864</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11673432" target="_blank">11673432</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27601571" target="_blank">27601571</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15731035" target="_blank">15731035</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15039320" target="_blank">15039320</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12511499" target="_blank">12511499</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18441068" target="_blank">18441068</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27886215" target="_blank">27886215</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12933899" target="_blank">12933899</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18424519" target="_blank">18424519</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14523114" target="_blank">14523114</a>]<br></li></div>
				<label class="RP_collapse_similarity" for="rp_similarity_11">
Wilson , R.L. , Libby , S.J. , Freet , A.M. , Boddicker , J.D. , Fahlen , T.F. , and Jones , B.D. ( 2001 ) Fis , a DNA nucle-oid-associated protein , is involved in Salmonella typhimu-rium SPI-1 invasion gene-expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16045614" target="_blank">16045614</a>]<br></label>
<input id="rp_similarity_11", type="checkbox">
<div class="RP_similar">
<li>Wilson , R.L. , Libby , S.J. , Freet , A.M. , Boddicker , J.D. , Fahlen , T.F. , and Jones , B.D. ( 2001 ) Fis , a DNA nucleoid-associated protein , is involved in Salmonella typhimurium SPI1 invasion gene-expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17784910" target="_blank">17784910</a>]<br></li>
<li>[ 16 ] Wilson , R.L. , Libby , S.J. , Freet , A.M. , Boddicker , J.D. , Fahlen , T.F. and Jones , B.D. ( 2001 ) Fis , a DNA nucleoid-associated protein , is involved in Salmonella typhimurium SPI-1 invasion gene-expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14553938" target="_blank">14553938</a>]<br></li>
<li>Wilson , R.L. , Libby , S.J. , Freet , A.M. , Boddicker , J.D. , Fahlen , T.F. , and Jones , B.D. ( 2001 ) Fis , a DNA nucleoid-associated protein , is involved in Salmonella typhimurium SPI-1 invasion gene-expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16999831" target="_blank">16999831</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17163975" target="_blank">17163975</a>]<br></li>
<li>Wilson , R. , Libby , S. , Freet , A. , Boddicker , J. , Fahlen , T. , and Jones , B. ( 2001 ) Fis , a DNA nucleoid-associated protein , is involved in Salmonella typhimurium SPI-1 invasion gene-expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15661008" target="_blank">15661008</a>]<br></li></div>
<li>J Bacteriol 18:6092 -- 6097 Wilson RL , Libby SJ , Freet AM , Boddicker JD , Fahlen TF , Jones BD ( 2001 ) Fis , a DNA nucleoid-associated protein , is involved in Salmonella Typhimurium SPI-1 invasion gene-expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18563621" target="_blank">18563621</a>]<br></li>
<li>[ 64 ] R.L. Wilson , S.J. Libby , A.M. Freet , J.D. Boddicker , T.F. Fahlen , B.D. Jones , Fis , a DNA nucleoid-associated protein , is involved in Salmonella typhimurium SPI-1 invasion gene-expression , Mol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29146500" target="_blank">29146500</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_18">
J Bacteriol 181:1269 -- 1280 Wilson RL , Libby SJ , Freet AM , Boddicker JD , Fahlen TF , Jones BD ( 2001 ) Fis , a DNA nucleoid-associated protein , is involved in Salmonella typhimurium SPI-1 invasion gene-expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12898222" target="_blank">12898222</a>]<br></label>
<input id="rp_similarity_18", type="checkbox">
<div class="RP_similar">
<li>Wilson RL , Libby SJ , Freet AM , Boddicker JD , Fahlen TF , Jones BD : Fis , a DNA nucleoid-associated protein , is involved in Salmonella typhimurium SPI-1 invasion gene-expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21418628" target="_blank">21418628</a>]<br></li></div>
<li>Deletion of fnr had no detectable effect on fis expression or on cell-growth ( Figure 2B , C & G ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24358360" target="_blank">24358360</a>]<br></li>
<li>Effects of a ﬁs null mutation on cell-motility .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7536730" target="_blank">7536730</a>]<br></li>
<li>enterica ARS # 576 Serovar Newport ; Aug Amp Fox Tio Chl Gen Kan Str Fis Tet ARS # 815 Serovar Newport ; Aug Amp Fox Tio Chl Gen Kan Str Fis Tet ARS # 852 Serovar Typhimurium DT104 ; Amp Chl Kan Str Fis Tet ARS # 886 Serovar Bardo ; Aug Amp Fox Tio Chl Kan Str Fis Tet Chen et al. ( 2010b ) , Lindsey et al. ( 2009 ) Chen et al. ( 2010b ) , Lindsey et al. ( 2009 ) Chen et al. ( 2010b ) , Lindsey et al. ( 2009 ) Chen et al. ( 2010b ) , Lindsey et al. ( 2009 ) Plasmid pSBardo-Kan pSe-Kan pUC19 This study plasmid isolated from ARS # 886 ; Kan This study plasmid isolated from ARS # 852 ; Kan Cloning vector ; Amp New England BioLabs a Antibiotic resistance abbreviation : Aug , amoxicillin-clavulanic acid ; Amp , ampicillin ; Fox , cefoxitin ; Tio , ceftiofur ; Chl , chloramphenicol ; Kan , kana mycin ; Str , streptomycin ; Fis , Sulﬁsoxazole ; Tet , tetracycline .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21324339" target="_blank">21324339</a>]<br></li>
<li>Recently , Fis has been implicated in the control of virulence gene-expression in pathogenic strains of E. coli ( Goldberg et al. , 2001 ; Sheikh et al. , 2001 ) , in Shigella ﬂexneri ( Falconi et al. , 2001 ) and in S. typhimurium , where it has been found to inﬂuence expression of genes within the SPI-1 patho-genicity island ( Schechter et al. , 2003 ; Wilson et al. , 2001 ; Yoon et al. , 2003 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15256548" target="_blank">15256548</a>]<br></li>
<li>In addition , nucleoid-associated proteins such as Fis , H-NS , HU and IHF inﬂuence the expression of these virulence genes ( Schechter et al. , 2003 ; Kelly et al. , 2004 ; Mangan et al. , 2006 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16999831" target="_blank">16999831</a>]<br></li>
<li>Co-factors such as Hha and YdgT , and other nucleoid associated proteins like Fis contribute to the expression of horizontally acquired genes in Salmonella , including those that regulate virulence ( Schechter et al. , 2003 ; Silphaduang et al. , 2007 ; Vivero et al. , 2008 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29034217" target="_blank">29034217</a>]<br></li>
<li>Alternatively , because Fis is involved in the transcriptional regulation of other nucleoid-binding proteins such as HU ( Claret and RouvieÁre-Yaniv , 1996 ) and H-NS ( Falconi et al. , 1996 ) , as well as gyrase ( Schneider et al. , 1999 ) and topoisomerase I ( Weinstein-Fischer et al. , 2000 ) , its contribution to SPI-1 gene regulation could be through altering the expression of these genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11123690" target="_blank">11123690</a>]<br></li>
<li>Fis also inﬂuences DNA supercoiling at a global level through its role as a transcription regulator of the genes coding for DNA gyrase and topoisomerase I ( Keane and Dorman , 2003 ; Schneider et al. , 1999 ; Weinstein-Fischer et al. , 2000 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15790293" target="_blank">15790293</a>]<br></li>
<li>Alternatively , Fis could repress the gene ( s ) shown negatively to affect hilA expression : phoP/Q ( Behlau and Miller , 1993 ; Pegues et al. , 1995 ) and hilE , ams and pag ( Fahlen et al. , 2000 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11123690" target="_blank">11123690</a>]<br></li>
<li>Its own gene , ﬁs , is among those that it represses ( Pratt et al. , 1997 ) , while it acts to upregulate many of the genes that encode components of the translation machinery of the cell ( Bosch et al. , 1990 ; Nilsson et al. , 1992 ; Ross et al. , 1990 ; Ball et al. , 1992 ; Ninnemann et al. , 1992 ; Walker et al. , 1999 ; Schneider et al. , 2003 ; Paul et al. , 2004 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17784910" target="_blank">17784910</a>]<br></li>
<li>These included the cold-shock-responsive hns gene previously shown to be activated by Fis ( Dersch et al. , 1994 ; Falconi et al. , 1996 ) , the hha gene whose product can form heteromeric complexes with H-NS and ( like H-NS ) regulates several virulence genes in response to temperature ( Madrid et al. , 2002 ; Nieto et al. , 2002 ) , and the stpA gene that encodes a paralogue of H-NS and can also form heteromers with it ( Deighan et al. , 2003 ; Free et al. , 2001 ; Johansson et al. , 2001 ; Williams et al. , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15256548" target="_blank">15256548</a>]<br></li>
<li>Many of the genes that are regulated positively by Fis in Gram-negative bacterial pathogens contribute to their virulence , and Fis has been identiﬁed as a virulence gene regulator in enteropathogenic E. coli , enteroaggre-gative E. coli , S. Typhimurium and Shigella ﬂexneri ( Falconi et al. , 2001 ; Goldberg et al. , 2001 ; Sheikh et al. , 2001 ; Wilson et al. , 2001 ; Schechter et al. , 2003 ; Yoon et al. , 2003 ; Kelly et al. , 2004 ; Prosseda et al. , 2004 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17784910" target="_blank">17784910</a>]<br></li>
<li>Positive regulators include BarA/SirA , FliZ , EnvZ/OmpR , FadD , CsrB , HilD and some nucleoid binding proteins ( Fis , HU ) , while Pag , HilE , Ams and the nucleoid binding proteins Hha and H-NS are negatively regulating hilA transcription ( Altier et al. 2000a , b Fahlen et al. 2000 ; Lucas et al. 2000 ; Wilson et al. 2001 ; Schechter et al. 2003 ; Baxter et al. 2003 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18563621" target="_blank">18563621</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_33">
Several genes have been identiﬁed including hilC ( Schechter et al. , 1999 ) , hilD ( Schechter et al. , 1999 ) , sirA/barA ( Johnston et al. , 1996 ; Altier et al. , 2000a ) , envZ/ompR ( Lucas et al. , 2000 ) , ﬁs ( Wilson et al. , 2001 ; Schechter et al. , 2003 ) , pstS ( Lucas et al. , 2000 ) , fadD ( Lucas et al. , 2000 ) , ﬂiZ ( Iyoda et al. , 2001 ) and csrAB ( Altier et al. , 2000b ) as positive effectors , and ams ( Fahlen et al. , 2000 ) , hha ( Fahlen et al. , 2001 ) , hilE ( Fahlen et al. , 2000 ) , pag ( Fahlen et al. , 2000 ) , phoP/phoQ ( Lucas et al. , 2000 ) , hupA/hupB ( Schechter et al. , 2003 ) and hns ( Schechter et al. , 2003 ) as negative regulators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15661008" target="_blank">15661008</a>]<br></label>
<input id="rp_similarity_33", type="checkbox">
<div class="RP_similar">
<li>The expression of many bacterial virulence genes is modulated by the Fis protein ( Falconi et al. , 2001 ; Goldberg et al. , 2001 ; Sheikh et al. , 2001 ; Wilson et al. , 2001 ; Schechter et al. , 2003 ; Yoon et al. , 2003 ; Kelly et al. , 2004 ; Prosseda et al. , 2004 ) and by DNA supercoiling ( Dorman et al. , 1990 ; Galán and Curtiss , 1990 ; Dorman , 1991 ; Parsot and Mekalanos , 1992 ; Rohde et al. , 1994 ; 1999 ; Graeff-Wohlleben et al. , 1995 ; Tobe et al. , 1995 ; Falconi et al. , 1998 ; Leclerc et al. , 1998 ; Beltrametti et al. , 1999 ; Gmuender et al. , 2001 ; Bang et al. , 2002 ; Fournier and Klier , 2004 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16999831" target="_blank">16999831</a>]<br></li></div>
<li>Selected groups of E. coli MG1655 genes with signiﬁcant ( P ,0.05 ) differential expression in response to triclosan exposure Fold change a Product Gene name Drug detoxiﬁcation acrB imp ﬂdA emrD marR sodC ompC ompF acridine efﬂux pump organic solvent tolerance 2.21 1.96 3.11 0.60 0.54 0.29 0.05 4.98 ﬂavodoxin 1 2-module integral membrane pump multiple antibiotic resistance regulator copper-zinc superoxide dismutase outer membrane porin C outer membrane porin F Curli biosynthesis csgA csgDEFG 0.56 major curlin subunit precursor 0.28 -- 0.41 curli biosynthesis and regulation thereof Global regulator ﬁs rpoS soxS 5.91 0.53 0.18 site-speciﬁc DNA inversion stimulation factor stress response sigma factor superoxide stress response regulator Motility ﬂgM ﬂiC ﬂiD ﬂiG -- I ﬂiL -- N ﬂiS ﬂiT ﬂiZ anti-FliA ( anti-sigma ) factor 5.19 215.55 ent structural protein ﬂagellar biosynthesis ; ﬂagellin , ﬁlam ﬂagellar biosynthesis ﬂagellar biosynthesis ﬂagellar biosynthesis ﬂagellar biosynthesis ﬂagellar biosynthesis orf , hypothetical protein 13.48 3.02 -- 8.32 2.26 -- 6.41 6.40 9.98 16.55 Continued Compared with E. coli MG1655 unexposed to triclosan .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19759044" target="_blank">19759044</a>]<br></li>
<li>In addition , we found that Fis up-regulated cobalamin ( B12 ) biosynthesis genes by controlling the B12 regulator gene , pocR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23717649" target="_blank">23717649</a>]<br></li>
<li>These include HU , H-NS , integration host factor ( IHF ) , and factor for inversion stimulation ( Fis ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9811652" target="_blank">9811652</a>]<br></li>
<li>Weinstein-Fischer , D. , Elgrably-Weiss , M. , and Altuvia , S. ( 2000 ) Escherichia coli response-to-hydrogen-peroxide : a role for DNA supercoiling , topoisomerase I and Fis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17784910" target="_blank">17784910</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21276095" target="_blank">21276095</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11123690" target="_blank">11123690</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16999831" target="_blank">16999831</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_39">
Escherichia coli response-to-hydrogen-peroxide : a role for DNA supercoiling , topoisom-erase I and Fis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12081946" target="_blank">12081946</a>]<br></label>
<input id="rp_similarity_39", type="checkbox">
<div class="RP_similar">
<li>Escherichia coli response-to-hydrogen-peroxide : a role for DNA supercoiling , topoisomerase I and Fis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15256548" target="_blank">15256548</a>]<br></li></div>
<li>sion is dependent upon several transcription factors that are important for virulence , including PhoP , RtsA , SirA , HilC , HilD , and Fis ( 9 , 10 , 35 , 61 , 63 , 78 , 90 -- 92 , 97 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15968047" target="_blank">15968047</a>]<br></li>
<li>Almost all genes encoding needle complex structural proteins and SPI regulators were activated by Fis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23717649" target="_blank">23717649</a>]<br></li>
<li>H-NS regulates its own gene negatively , and this repression is antagonized by Fis , a relationship that is seen at many other H-NS-repressed promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15790293" target="_blank">15790293</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_44">
aldB , an RpoS-dependent gene in Escherichia coli encoding an aldehyde dehydrogenase that is repressed by Fis and activated by Crp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15256548" target="_blank">15256548</a>]<br></label>
<input id="rp_similarity_44", type="checkbox">
<div class="RP_similar">
<li>aldB , an RpoS-dependent gene in Esche-richia coli encoding an aldehyde dehydrogenase that is repressed by Fis and activated by Crp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8752327" target="_blank">8752327</a>]<br></li></div>
<li>Transcriptional fusions to several early , middle , and late genes have corroborated these results , as have motility tests , which show impaired motility in a ﬁs mutant .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23554419" target="_blank">23554419</a>]<br></li>
<li>2 20 ORF1 ﬁs operon are involved in processes that are responsive to nitrogen control .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7536730" target="_blank">7536730</a>]<br></li>
<li>Like other NAPs , FIS has an architectural role in shaping DNA structure .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21276095" target="_blank">21276095</a>]<br></li>
<li>For example , whereas FIS is important for DNA supercoiling homeostasis during growth in well-aerated conditions , FIS appears to have a minimal role in low-oxygen conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21276095" target="_blank">21276095</a>]<br></li>
<li>e RA class includes genes which are repressed by Fis in LT2 , but activated in K12 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23717649" target="_blank">23717649</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23717649" target="_blank">23717649</a>]<br></li>
<li>This was consistent with a role for RpoS in repressing Fis expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17784910" target="_blank">17784910</a>]<br></li>
<li>A potential role for Fis in SPI-1 gene regulation could be one that is related to DNA topology in response to environmental changes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11123690" target="_blank">11123690</a>]<br></li>
<li>Fis was shown to play an autoregulatory role in E. coli ( 4 , 43 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7536730" target="_blank">7536730</a>]<br></li>
<li>Nutritional up-shift has been shown to activate fis expression , which is , in turn , increased in response to increased gyrase activity under this condition ( Schneider et al. 2000 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15235764" target="_blank">15235764</a>]<br></li>
<li>ﬁs expression in response-to-nutrient upshift .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7536730" target="_blank">7536730</a>]<br></li>
<li>Several S. typhimurium genes involved in polyamine-metabolism showed elevated expression in the ﬁs mutant ( Table 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15256548" target="_blank">15256548</a>]<br></li>
<li>These results suggest that changes in DNA supercoiling during non-aerated growth may be involved in sustained fis expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24358360" target="_blank">24358360</a>]<br></li>
<li>Morett E , Bork P ( 1998 ) Evolution of new protein function : recombinational Nasser W , Rochman M , Muskhelishvili G ( 2002 ) Transcriptional regulation of fis operon involves a module of multiple coupled promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23717649" target="_blank">23717649</a>]<br></li>
<li>ﬁs is a positive regulator of hilA , an activator of invasion genes , while hha is a negative regulator of hilA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19909098" target="_blank">19909098</a>]<br></li>
<li>This mutant also had increased expression of ﬁs .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19411325" target="_blank">19411325</a>]<br></li>
<li>DNA supercoiling control of fis expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24358360" target="_blank">24358360</a>]<br></li>
<li>FIS control of gyrA , gyrB and topA gene-expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21276095" target="_blank">21276095</a>]<br></li>
<li>FIS control of topA expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21276095" target="_blank">21276095</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_64">
A ) RT-PCR assay for mRNAs examining transcription of the hlyE gene from S. Typhi wild-type ( STH2370 ) , Δcrp , Δfis , Δcrp Δfis double mutant , and the complemented strains Δcrp / pTCRP and Δfis / pTFIS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24885225" target="_blank">24885225</a>]<br></label>
<input id="rp_similarity_64", type="checkbox">
<div class="RP_similar">
<li>A ) RT-PCR assay for mRNAs examining transcription of the rpoS gene from S. Typhi wild-type ( STH2370 ) , Δcrp , Δfis , Δcrp Δfis double mutant , and the complemented strains Δcrp / pTCRP and Δfis / pTFIS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24885225" target="_blank">24885225</a>]<br></li></div>
<li>In particular , the genes required for cytochrome-o-oxidase , succinyl-CoA synthetase , glutamate/aspartate-transport , trehalose-6-phosphate biosynthesis , long-chain-fatty-acids transport , spermidine/putrescine-transport , dipeptide-transport , the genes encoding the two-component tricarboxylic transport system and the site-specific DNA factor for inversion stimulation ( fis ) were among the highest repressed by ArcA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21418628" target="_blank">21418628</a>]<br></li>
<li>Other genes from lipid-biosynthesis were also found to be up-regulated in the ﬁs mutant .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15256548" target="_blank">15256548</a>]<br></li>
<li>At the 1 h time point , the genes most highly up-regulated in the ﬁs mutant were all involved in biotin-synthesis ( bioB , bioC and bioF ) ( Fig. 9 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15256548" target="_blank">15256548</a>]<br></li>
<li>This response regulates many important genes in the cell , including the ﬁs gene itself .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15256548" target="_blank">15256548</a>]<br></li>
<li>Fis is also an important regulator of transcription that can act Accepted 10 August , 2007 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17784910" target="_blank">17784910</a>]<br></li>
<li>In Escherichia coli , Fis has been shown to modulate transcription of many genes , including those encoding stable RNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15256548" target="_blank">15256548</a>]<br></li>
<li>It is apparent that many genes of unknown function showed a positive or a negative response to Fis ( supplementary data Tables S1 and S2 at http://mic.sgmjournals.org ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15256548" target="_blank">15256548</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_73">
A global role for Fis in the transcriptional-control of metabolism and type-III-secretion in Salmonella enterica serovar Typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23554419" target="_blank">23554419</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26441883" target="_blank">26441883</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21212121" target="_blank">21212121</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15790293" target="_blank">15790293</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20390946" target="_blank">20390946</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22356573" target="_blank">22356573</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19909098" target="_blank">19909098</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23873910" target="_blank">23873910</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29751061" target="_blank">29751061</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24488311" target="_blank">24488311</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27601571" target="_blank">27601571</a>]<br></label>
<input id="rp_similarity_73", type="checkbox">
<div class="RP_similar">
<li>A global role for Fis in the transcriptional-control of metab ¬ olism and type-III-secretion in Salmonella enterica serovar Typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19411325" target="_blank">19411325</a>]<br></li>
<li>A global role for Fis in the transcriptional-control of metab-olism and type-III-secretion in Salmonella enterica serovar Typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21037291" target="_blank">21037291</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16988271" target="_blank">16988271</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19114490" target="_blank">19114490</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18441068" target="_blank">18441068</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16997949" target="_blank">16997949</a>]<br></li></div>
				<label class="RP_collapse_similarity" for="rp_similarity_76">
Kelly A , Goldberg MD , Carroll RK , Danino V , Hinton JC , et al. ( 2004 ) A global role for Fis in the transcriptional-control of metabolism and type-III-secretion in Salmonella enterica serovar Typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23717649" target="_blank">23717649</a>]<br></label>
<input id="rp_similarity_76", type="checkbox">
<div class="RP_similar">
<li>[ 8 ] A. Kelly , M.D. Goldberg , R.K. Carroll , V. Danino , J.C.D. Hinton , C.J. Dorman , A global role for Fis in the transcriptional-control of metabolism and type-III-secretion in Salmonella enterica serovar Typhimurium , Microbiology 150 ( 2004 ) 2037 -- 2053 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20691712" target="_blank">20691712</a>]<br></li>
<li>Kelly A. , Goldberg M.D. , Carroll R.K. , Danino V. , Hinton J.C. , Dorman C.J. ( 2004 ) A global role for Fis in the transcriptional-control of metabolism and type-III-secretion in Salmonella enterica serovar Typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19341422" target="_blank">19341422</a>]<br></li>
<li>Kelly , A. , Goldberg , M. D. , Carroll , R. K. , Danino , V. , Hinton , J. C. , and Dorman , C. J. ( 2004 ) A global role for Fis in the transcriptional-control of metabolism and type-III-secretion in Salmonella enterica serovar Typhi-murium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16684765" target="_blank">16684765</a>]<br></li>
<li>Kelly , A. , Goldberg , M.D. , Carroll , R.K. , Danino , V. , Hinton , J.C.D. , and Dorman , C.J. ( 2004 ) A global role for Fis in the transcriptional-control of metabolism and type-III-secretion in Salmonella enterica serovar Typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17784910" target="_blank">17784910</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18031307" target="_blank">18031307</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21276095" target="_blank">21276095</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16999831" target="_blank">16999831</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27206164" target="_blank">27206164</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24308841" target="_blank">24308841</a>]<br></li>
<li>Kelly , A. , Goldberg , M.D. , Carroll , R.K. , Danino , V. , Hinton , J.C. , and Dorman , C.J. ( 2004 ) A global role for Fis in the transcriptional-control of metabolism and type-III-secretion in Salmonella enterica serovar Typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23190111" target="_blank">23190111</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17427289" target="_blank">17427289</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630976" target="_blank">17630976</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17163975" target="_blank">17163975</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18399940" target="_blank">18399940</a>]<br></li></div>
<li>Sheikh J , Hicks S , Dall ' Agnol M , Phillips AD , Nataro JP : Roles for Fis and YafK in biofilm-formation by enteroaggregative Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21418628" target="_blank">21418628</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_83">
Cho , B.K. , Knight , E.M. , Barrett , C.L. , and Palsson , B.O. ( 2008 ) Genome-wide analysis of Fis binding in Escherichia coli indicates a causative role for A - / AT-tracts .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22804842" target="_blank">22804842</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21276095" target="_blank">21276095</a>]<br></label>
<input id="rp_similarity_83", type="checkbox">
<div class="RP_similar">
<li>Cho BK , Knight EM , Barrett CL , Palsson BO ( 2008 ) Genome-wide analysis of Fis binding in Escherichia coli indicates a causative role for A2/AT-tracts .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23717649" target="_blank">23717649</a>]<br></li></div>
<li>Genome-wide analysis of Fis binding in Escherichia coli indicates a causative role for A - / AT-tracts .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25713562" target="_blank">25713562</a>]<br></li>
<li>Several groups have identiﬁed mutations in genes that result in decreased expression of hilA , indicating that csrAB , sirA/barA , pstS , fadD , envZ , ﬂiZ , hilC/sirC/sprA , ﬁs , and hilD have roles in hilA expression ( 1 , 2 , 27 , 36 , 42 , 48 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12511499" target="_blank">12511499</a>]<br></li>
<li>have been identiﬁed as being important in activation of hilA expression , including csrAB , sirA/barA , phoBR , fadD , envZ , ﬂiZ , and ﬁs ( 2 , 3 , 26 , 32 , 53 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15731035" target="_blank">15731035</a>]<br></li>
<li>A variety of genes ( hilC , hilD , ﬁs , sirA/barA , csrAB , phoB , fadD , envZ/ompR , ﬂiZ , hilE , ams , lon , pag , and hha ) have been identiﬁed that exert positive or negative effects on hilA expression , although the mechanisms by which these gene products function remain relatively unclear .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12595445" target="_blank">12595445</a>]<br></li>
<li>A number of positive regulators ( hilC , hilD , ﬁs , sirA/barA , csrAB , phoBR , fadD , envZ/ompR , and ﬂiZ ) and negative regulators ( hha , hilE , lon , ams , phoPc and pag ) have been identiﬁed that are able to alter expression of hilA transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15731035" target="_blank">15731035</a>]<br></li>
<li>A number of positive regulators of hilA expression have been identified including csrAB , sirA/barA , pstS , hilC/sirC/sprA , fis , and hilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15765064" target="_blank">15765064</a>]<br></li>
<li>In Escherichia coli , class Ia has been shown to be cell-cycle regulated [ 3,4 ] and modulated by several global transcriptional factors such as DnaA , Fis and IciA [ 5-7 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20593029" target="_blank">20593029</a>]<br></li>
<li>These genes include those that encode several two-component regulatory systems ( PhoP/Q , PhoBR , OmpR/EnvZ and SirA/BarA ) , post-transcriptional systems ( CsrAB , RNase E , PNPase and Lon ) , nucleoid proteins ( HU , FIS , H-NS and Hha ) , signaling molecules ( ppGpp and NpnN ) , and other regulatory proteins ( FliZ , FadD and CpxA ) ( 6,7,19,21,22 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17329372" target="_blank">17329372</a>]<br></li>
<li>In Figure S11 and Figure S12 , we showed that nucleoid proteins Fis and HU also independently control both hilA and hilD transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22021388" target="_blank">22021388</a>]<br></li>
<li>By regulating expression of SPI-encoded transcription factors , H-NS , FIS , OmpR , and other global regulators sit atop a hierarchical network that integrates diverse environmental and physiological cues .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22457642" target="_blank">22457642</a>]<br></li>
<li>RpoS integrates multiple environmental signals and global regulators , including CRP , Fis , and PhoP signaling-pathways , to control S. Typhi hlyE expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24885225" target="_blank">24885225</a>]<br></li>
<li>We suspected that small nucleoid-binding proteins , such as H-NS , HU , and Fis , might control hilA expression by affecting its repression and/or derepression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12933899" target="_blank">12933899</a>]<br></li>
<li>Transcription of hilA is subject to direct or indirect negative control by multiple nucleoid binding proteins , including Hha , H-NS , HU , and Fis ( 10 , 23 ) , and other proteins , including HilE , Pag , and Ams ( 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15126488" target="_blank">15126488</a>]<br></li>
<li>The expression of hilA is regulated in response to variety of environmental signals , including pH ( 6 ) , growth phase ( 56 ) , osmolarity ( 36 ) , and oxygen ( 52 ) and by a number of proteins , including BarA/SirA ( 51 ) , CpxA ( 71 ) , CsrB ( RNA ) ( 2 ) , EnvZ/OmpR ( 63 ) , FadD ( 63 ) , Fis ( 94 ) , FliZ ( 63 ) , HilC ( 83 ) , HilD ( 83 ) , Hu ( 84 ) , Mlc ( 58 ) , and RtsA ( 28 ) , which positively regulate expression , and CsrA ( 2 ) , Hha ( 29 ) , HilE ( 7 ) , H-NS ( 84 ) , Lon ( 87 ) , PhoB ( 63 ) , and PhoP/PhoQ ( 9 ) , which negatively regulate expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19074398" target="_blank">19074398</a>]<br></li>
<li>Thus , the full-length S. enterica fis promoter region may act as a topological switch that is sensitive to stress-induced duplex destabilisation and upregulates expression in non-aerated conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24358360" target="_blank">24358360</a>]<br></li>
<li>Apart from H-NS and Hha the f actor for inversion stimulation ( Fis ) , also controls Spi-1 gene-expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26441883" target="_blank">26441883</a>]<br></li>
<li>Similarly , motility and ﬂagellar genes required Fis for full expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15256548" target="_blank">15256548</a>]<br></li>
<li>Fis , a DNA binding protein , is required for full expression of both SPI1 genes and ﬂagellar genes ( 28 , 52 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18441068" target="_blank">18441068</a>]<br></li>
<li>In contrast , Fis is a site-speciﬁc DNA binding protein that induces sharp bends ( 58 ) and can behave both as an activator and a repressor of gene-expression ( 20 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11292791" target="_blank">11292791</a>]<br></li>
<li>R E S E A R C H A R T I C L E Open Access RpoS integrates CRP , Fis , and PhoP signaling pathways to control Salmonella Typhi hlyE expression Matías R Jofré1 , Leonardo M Rodríguez1 , Nicolás A Villagra1 , Alejandro A Hidalgo2 , Guido C Mora2 and Juan A Fuentes1 ,3 * Abstract Background : SPI-18 is a pathogenicity island found in some Salmonella enterica serovars , including S. Typhi .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24885225" target="_blank">24885225</a>]<br></li>
<li>Fis plays a role in Tn5 and IS50 transposition .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15716427" target="_blank">15716427</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7536730" target="_blank">7536730</a>]<br></li>
<li>Genes up-regulated by Fis are involved in translation , flagellar biosynthesis and energy metab-olism , while down-regulated genes are involved in stress responses , amino acid and nucleotide-biosynthesis , and nutrient transport [ 4,20,21 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23717649" target="_blank">23717649</a>]<br></li>
<li>Moreover , they are transcribed under similar conditions and affected by several global regulators , including CsrA , Fis , and H-NS ( recently reviewed in reference 50 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16988271" target="_blank">16988271</a>]<br></li>
<li>For example , an interesting observation made by Kelly et al. ( 52 ) was that Fis , a global transcriptional regulator that controls both metabolism and type-III-secretion-systems in S. typhimurium , appeared to be abundant in log phase and nearly absent in stationary phase , consistent with our results .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16684765" target="_blank">16684765</a>]<br></li>
<li>Our discovery that Fis regulates the expression of all three type-III-secretion-systems in Salmonella is in keeping with other studies that have pointed to regulatory overlaps between virulence genes and ﬂagella in Salmonella ( Eichelberg & Galan , 2000 ; Ellermeier & Slauch , 2003 ; Lawhon et al. , 2003 ) and other bacteria ( Goodier & Ahmer , 2001 ; Grant et al. , 2003 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15256548" target="_blank">15256548</a>]<br></li>
<li>Published transcriptomic datasets reveal that SPI1 and ﬂa-gellar genes are coregulated in response to a number of regulatory signals , including CsrA , YfgL , Fnr , Fis , and RfaH , as well as several environmental conditions ( Table S3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22021388" target="_blank">22021388</a>]<br></li>
<li>FIS is a master regulator of DNA supercoiling in E. coli , and as such is expected to play a central role in determining supercoiling set points and responses to osmotic pressure and aeration .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21276095" target="_blank">21276095</a>]<br></li>
<li>Kahramanoglou C , Seshasayee AS , Prieto AI , Ibberson D , Schmidt S , et al. ( 2011 ) Direct and indirect effects of H-NS and Fis on global gene-expression control in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23717649" target="_blank">23717649</a>]<br></li>
<li>The Small Nucleoid-Binding Proteins H-NS , HU , and Fis Affect hilA Expression in Salmonella enterica Serovar Typhimurium Lisa M. Schechter , * Sumita Jain , † Samina Akbar , ‡ and Catherine A. Lee Department of Microbiology and Molecular Genetics , Harvard Medical School , Boston , Massachusetts 02115 Received 1 November 2002/Returned for modiﬁcation 3 February 2003/Accepted 27 May 2003 hilA encodes an activator of Salmonella enterica serovar Typhimurium virulence genes and is transcriptionally modulated by environmental conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12933899" target="_blank">12933899</a>]<br></li>
<li>Factor for inversion stimulation ( ﬁs ) has been shown to be involved in DNA transcription , DNA-replication at oriC , transposition , invasion of HEp-2 cells , and transcriptional activation and repression of several genes ( Kelly et al. , 2004 ; ) and DluxS mutant ( M ) and between wild-type cell-free supernatants and DluxS Table 6 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19909098" target="_blank">19909098</a>]<br></li>
<li>For example , Fis stimulates transcription of rRNA and tRNA genes and genes for proteins involved in translation ( 19 , 35 , 40 , 51 , 56 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7536730" target="_blank">7536730</a>]<br></li>
<li>Fis ( factor for inversion stimulation ) is a nucleoid-associated protein in E. coli that regulates recombination , replication , and transcription reactions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14502413" target="_blank">14502413</a>]<br></li>
<li>Besides overall transcriptional decreases for genes related to translation , lower transcript abundances were observed for genes involved in transcription and in replication , recombination , and repair , such as rpoA ( encoding DNA-directed RNA polymerase subunit alpha ) , gyrA ( DNA gyrase subunit A ) , ﬁs ( DNA-binding protein Fis ) , and priB ( primosomal replication protein N ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25107963" target="_blank">25107963</a>]<br></li>
<li>Fis-binding sites on several genes such as rpoS and gyrB in S. enterica , through which Fis regulates these genes , have also been identified [ 16 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23717649" target="_blank">23717649</a>]<br></li>
<li>For instance , Fis stimulates transcription from rRNA and tRNA genes in E. coli ( 36 , 46 ) , a function that is most critical during exponential cell-growth ( 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9811652" target="_blank">9811652</a>]<br></li>
<li>Fis activates transcription of rRNA and many other genes , including some involved in replication , so it is conceivable that Fis also works indirectly .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15716427" target="_blank">15716427</a>]<br></li>
<li>Mol Microbiol 39:1109 -- 1115 Osuna R , Lienau D , Hughes KT , Johnson RC ( 1995 ) Sequence , regulation , and functions of fis in Salmonella typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15235764" target="_blank">15235764</a>]<br></li>
<li>[ 7 ] R. Osuna , D. Lienau , K.T. Hughes , R.C. Johnson , Sequence , regulation , and function of ﬁs in Salmonella typhimurium , J. Bacteriol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20691712" target="_blank">20691712</a>]<br></li>
<li>Sequence , regulation , and functions of ﬁs in Salmonella typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8752327" target="_blank">8752327</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15256548" target="_blank">15256548</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15790293" target="_blank">15790293</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15716427" target="_blank">15716427</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9811652" target="_blank">9811652</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16980465" target="_blank">16980465</a>]<br></li>
<li>Mallik P , Paul BJ , Rutherford ST , Gourse RL , Osuna R ( 2006 ) DksA is required for growth phase-dependent regulation , growth rate-dependent control , and stringent control of fis expression in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24358360" target="_blank">24358360</a>]<br></li>
<li>Mallik P , Pratt TS , Beach MB , Bradley MD , Undamatla J et al. ( 2004 ) Growth phase-dependent regulation and stringent control of fis are conserved processes in enteric bacteria and involve a single promoter ( fis P ) in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24358360" target="_blank">24358360</a>]<br></li>
<li>R. ( 2004 ) Growth phase-dependent regulation and stringent control of ﬁs are conserved processes in enteric bacteria and involve a single promoter ( ﬁs P ) in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17784910" target="_blank">17784910</a>]<br></li>
<li>We constructed an identical FIS : Myc fusion protein in S. enterica and used this to measure FIS occupancy of gene promoters in our experimental conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22457642" target="_blank">22457642</a>]<br></li>
<li>The FIS protein is a global regulator of supercoiling in E. coli , but it was found to have less inﬂuence over supercoiling control in S. enterica .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21276095" target="_blank">21276095</a>]<br></li>
<li>In contrast , PtopASe demonstrated baseline expression in E. coli wild type but greatly elevated expression in E. coli Dﬁs , indicating that FIS is a strong repressor of PtopASe in E. coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21276095" target="_blank">21276095</a>]<br></li>
<li>Gonzalez-Gil G , Bringmann P , Kahmann R ( 1996 ) FIS is a regulator of metabolism in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23717649" target="_blank">23717649</a>]<br></li>
<li>FIS is a regulator of metabolism in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15256548" target="_blank">15256548</a>]<br></li>
<li>Surprisingly , deletion of arcA caused an increase in fis expression in non-aerated stationary phase ( Figure 2A & D ) , implicating arcA as a repressor of sustained fis expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24358360" target="_blank">24358360</a>]<br></li>
<li>Discussion FIS is a global regulator of gene-expression , but its abundance in the cell fluctuates dramatically depending on growth phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24358360" target="_blank">24358360</a>]<br></li>
<li>* E-mail : cjdorman@tcd.ie Introduction The factor for inversion stimulation ( FIS ) is a global regulator of gene-expression and chromosome compaction in Gamma-proteobacteria .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24358360" target="_blank">24358360</a>]<br></li>
<li>The factor for inversion stimulation ( FIS ) is of particular interest because it plays diverse roles in the control of DNA supercoiling and gene-expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21276095" target="_blank">21276095</a>]<br></li>
<li>Mol Microbiol 38:526 -- 534 Wold S , Crooke E , Skarstad K ( 1996 ) The Escherichia coli Fis protein prevents initiation of DNA-replication from oriC in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15235764" target="_blank">15235764</a>]<br></li>
<li>The Escherichia coli Fis protein prevents initiation of DNA-replication from oriC in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9811652" target="_blank">9811652</a>]<br></li>
<li>The FIS protein binds and bends the origin of chromosomal DNA-replication , oriC , of Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15716427" target="_blank">15716427</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_139">
Mol Microbiol 6:3257 -- 3265 Gille H , Egan JB , Roth A , Messer W ( 1991 ) The Fis protein binds and bends the origin of chromosomal DNA-replication , oriC , of Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12898222" target="_blank">12898222</a>]<br></label>
<input id="rp_similarity_139", type="checkbox">
<div class="RP_similar">
<li>The Fis protein binds and bends the origin of chromosomal DNA-replication , oriC , of Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7536730" target="_blank">7536730</a>]<br></li></div>
<li>We decided to examine Fis protein expression as a function of growth by Western analysis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12898222" target="_blank">12898222</a>]<br></li>
<li>Effects of Fis on Escherichia coli gene-expression during different growth stages .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19411325" target="_blank">19411325</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_143">
Xu J , Johnson RC ( 1995 ) Identiﬁcation of genes negatively regulated by Fis : Fis and RpoS comodulate growth-phase-dependent gene-expression in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14502413" target="_blank">14502413</a>]<br></label>
<input id="rp_similarity_143", type="checkbox">
<div class="RP_similar">
<li>Identiﬁcation of genes negatively regulated by Fis : Fis and RpoS comodulate growth-phase-dependent gene-expression in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8752327" target="_blank">8752327</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7536730" target="_blank">7536730</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17293430" target="_blank">17293430</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15256548" target="_blank">15256548</a>]<br></li></div>
<li>Expression of AbrB , a transition state regulator from Bacillus subtilis , is growth phase dependent in a manner resembling that of Fis , the nucleoid binding protein from Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9811652" target="_blank">9811652</a>]<br></li>
<li>Fis activates the RpoS-dependent station-ary-phase expression of proP in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16332813" target="_blank">16332813</a>]<br></li>
<li>The expression of sodC , a superoxide dismutase , and ﬁs , a global regulator , was also divergent between the species ( Tables 2 and 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19759044" target="_blank">19759044</a>]<br></li>
<li>To verify the microarray results , expression levels of several representative genes including ﬁs , ﬂiC , rplB , purB , acs and poxB were also investigated by RT-PCR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19659726" target="_blank">19659726</a>]<br></li>
<li>We noticed that a ﬁs null mutation reduces cell-motility , as assayed either by direct microscopic observation or on swarm plates ( Fig. 4 ) , suggesting that Fis may somehow inﬂuence the expression of genes involved in motility .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7536730" target="_blank">7536730</a>]<br></li>
<li>Thus , ﬁlamentation of S. typhimurium ﬁs cells may be observed exclusively under conditions of rapid growth , which is consistent with a role proposed for Fis for efﬁcient initiation of DNA-replication in E. coli ( 12 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7536730" target="_blank">7536730</a>]<br></li>
<li>Keywords gyr genes Æ DNA gyrase Æ Fis protein Æ Gene regulation Æ Salmonella enterica serovar Typhimurium Introduction In Escherichia coli a complex regulatory relationship exists between the DNA-binding protein Fis and the control of DNA supercoiling .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12898222" target="_blank">12898222</a>]<br></li>
<li>doi :10.1111 / j.1365-2958.2006.05416.x First published online 25 September 2006 dulating expression of virulence g s during intracellular almonella e serovar Typhimurium n the Fis protein i Roles for D A upercoiling a d N s n mo n nterica e e growth of S Tadhg Ó Cróinín , ‡ Ronan K. Carroll , † ‡ Arlene Kelly and Charles J. Dorman * Department of Microbiology , Moyne Institute of Preventive Medicine , Trinity College Dublin , Ireland .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16999831" target="_blank">16999831</a>]<br></li>
<li>Microbiology ( 2004 ) , 150 , 2037 -- 2053 DOI 10.1099 / mic.0.27209-0 A global role for Fis in the transcriptional-control of metabolism and type-III-secretion in Salmonella enterica serovar Typhimurium Arlene Kelly ,1 Martin D. Goldberg ,2 Ronan K. Carroll ,1 Vittoria Danino ,2 Jay C. D. Hinton2 and Charles J. Dorman1 1Department of Microbiology , Moyne Institute of Preventive Medicine , University of Dublin , Trinity College , Dublin 2 , Ireland Correspondence Charles J. Dorman 2Molecular Microbiology Group , Institute of Food Research , Norwich Research Park , Colney , Norwich NR4 7UA , UK cjdorman@tcd.ie Fis is a key DNA-binding protein involved in nucleoid organization and modulation of many DNA transactions , including transcription in enteric bacteria .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15256548" target="_blank">15256548</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_154">
Ó Cróinín , T. , Carroll , R.K. , Kelly , A. , and Dorman , C.J. ( 2006 ) Roles for DNA supercoiling and the Fis protein in modulating expression of virulence genes during intracellular-growth of Salmonella enterica serovar Typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21276095" target="_blank">21276095</a>]<br></label>
<input id="rp_similarity_154", type="checkbox">
<div class="RP_similar">
<li>O'Croinin , T. , Carroll , R. , Kelly , A. , and Dorman , C.J. ( 2006 ) Roles for DNA supercoiling and the FIS protein in modulating expression of virulence genes during intracellular-growth of Salmonella enterica serovar Typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630976" target="_blank">17630976</a>]<br></li>
<li>Ó Cróinín , T. , Carroll , R.K. , Kelly , A. , and Dorman , C.J. ( 2006 ) Roles for DNA supercoiling and the FIS protein in modulating expression of virulence genes during intracellular-growth of Salmonella enterica serovar Typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17784910" target="_blank">17784910</a>]<br></li>
<li>O Croinin T , Carroll RK , Kelly A , Dorman CJ : Roles for DNA supercoiling and the Fis protein in modulating expression of virulence genes during intracellular-growth of Salmonella enterica serovar typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23295139" target="_blank">23295139</a>]<br></li>
<li>Roles for DNA supercoiling and the Fis protein in modulating expression of virulence genes during intracellular-growth of Salmonella enterica serovar Typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20861532" target="_blank">20861532</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26441883" target="_blank">26441883</a>]<br></li></div>
<li>The reciprocal modulation of RpoS and Fis expression represents an interesting trade-off between reduced expres ¬ Relevant characteristics Strain Reference/source E. coli XL-1 Blue recA1 endA1 gyrA96thi-1 hsdR17 supE44 relA1 lac [ F ′ proAB lacIqZDM15Tn10 ( TcR ) ] F -- ara D ( lac pro ) strA thi F -- ara D ( lac pro ) strA thi ﬁs : : KmR F -- y -- araD139 D ( argF-lac ) U169 rpsL150 relA1 ﬂbB5301 deoC ptsF25 rbsR Stratagene CSH50 CSH50ﬁs MC4100 S. Typhimurium LT2 SL1344 SL1344 ﬁs SL1344 rpoS SL1344 ﬁsrpoS Cold Spring Harbor Laboratories Koch et al. ( 1988 ) Casadaban ( 1976 ) Wild type rpsL hisG B.N. Ames Hoiseth and Stocker ( 1981 ) Keane and Dorman ( 2003 ) Keane ( 2002 ) This study SL1344 ﬁs : : CmR SL1344 rpoS : : KmR SL1344 rpoS : : KmRﬁs : : Cm R sion of genes involved in the stress response in favour of genes involved in the invasion of epithelial cells leading to systemic dispersion .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17784910" target="_blank">17784910</a>]<br></li>
<li>FIS is a global regulator of supercoiling levels in both organisms , but its control of supercoiling homeostasis differs between growth conditions and between the two organisms .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21276095" target="_blank">21276095</a>]<br></li>
<li>Antagonistic involvement of FIS and H-NS proteins in the transcriptional-control of hns expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9811652" target="_blank">9811652</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15256548" target="_blank">15256548</a>]<br></li>
<li>Transmission of an Oxygen Availability Signal at the Salmonella enterica Serovar Typhimurium fis Promoter Andrew D. S. Cameron1 ,2 , Carsten Kröger1 , Heather J. Quinn1 , Isobel K. Scally1 , Anne J. Daly1 , Stefani C. Kary2 , Charles J. Dorman1 * 1 Department of Microbiology , Moyne Institute of Preventive Medicine , School of Genetics and Microbiology , Trinity College Dublin , Dublin , Ireland , 2 Department of Biology , University of Regina , Regina , SK , Canada Abstract The nucleoid-associated protein FIS is a global regulator of gene-expression and chromosome structure in Escherichia coli and Salmonella enterica .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24358360" target="_blank">24358360</a>]<br></li>
<li>1,2 1,2,3 1 TEDA School of Biological Sciences and Biotechnology , Nankai University , TEDA , Tianjin , P. R. China , 2 Tianjin Key Laboratory of Microbial Functional Genomics , Tianjin , P. R. China , 3 The Key Laboratory of Molecular Microbiology and Technology , Ministry of Education , Tianjin , P. R. China Abstract Fis , one of the most important nucleoid-associated proteins , functions as a global regulator of transcription in bacteria that has been comprehensively studied in Escherichia coli K12 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23717649" target="_blank">23717649</a>]<br></li>
<li>Involvement of Escherichia coli FIS protein in maintenance of bacteriophage Mu lysogeny by the repressor : control of early transcription and inhibition of transposition .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7536730" target="_blank">7536730</a>]<br></li>
<li>Full expression of ﬁs is required for upregulation of genes encoding secretion apparatus of T3SS and effectors required for invasion-of-host epithelial cells , for survival in macrophages and synthesis of ﬂagella for motility ( Kelly et al. , 2004 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24308841" target="_blank">24308841</a>]<br></li>
<li>The microarray analysis showed acid-induced down-regulation of ﬁs , a regulatory gene whose product enhances transcription of SPI1 and SPI2 virulence genes and of motility genes in Typhimurium ( Kelly et al. , 2004 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22356573" target="_blank">22356573</a>]<br></li>
<li>Transcription of a ﬁs -- gfp fusion as a function of growth in S. Typhimurium strain SL1344 and E. coli strain CSH50 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17784910" target="_blank">17784910</a>]<br></li>
<li>This also identiﬁed a previously unrecognized osmoregulation of topA expression that is independent of FIS and supercoiling in both E. coli and S. enterica .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21276095" target="_blank">21276095</a>]<br></li>
<li>Fis has been implicated in many other processes such as stimulation of excision and integration of lambda ( 4 , 5 , 20 , 67 ) , DNA-replication at oriC ( 24 , 26 , 58 ) , transposition ( 70 ) , invasion of HEp-2 cells ( 71 ) , and transcriptional activation and repression of several genes , including hns , leuV , gyr , tyrT , proP , nuo , osmE , and rRNA operons ( 3 , 12 , 21 , 33 , 56 , 68 , 72 , 74 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15716427" target="_blank">15716427</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_170">
Synergistic role of curli and cellulose in cell adherence and biofilm-formation of attaching and effacing Escherichia coli and identification of Fis as a negative regulator of curli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28470089" target="_blank">28470089</a>]<br></label>
<input id="rp_similarity_170", type="checkbox">
<div class="RP_similar">
<li>Synergistic role of curli and cellulose in cell adherence and bioﬁlm formation of attaching and effacing Escherichia coli and identiﬁcation of Fis as a negative regulator of curli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23377943" target="_blank">23377943</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27356028" target="_blank">27356028</a>]<br></li></div>
<li>The redox sensors ArcAB and FNR regulate transitions from aerobic to anaerobic growth , making these proteins prime candidates for regulating the sustained expression of fis in low oxygen environments .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24358360" target="_blank">24358360</a>]<br></li>
<li>ArcA and FNR regulate dozens and hundreds of genes , respectively , many of which are required for metabolism in anaerobic conditions , but neither transcription factor has been implicated in controlling fis expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24358360" target="_blank">24358360</a>]<br></li>
<li>Effects of transcription factor mutations on fis expression in aerated and non-aerated conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24358360" target="_blank">24358360</a>]<br></li>
<li>Most importantly , Fis was found to activate SPI genes , which are essential for the virulence of S. enterica .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23717649" target="_blank">23717649</a>]<br></li>
<li>It is intuitively appealing that S. typhimurium can beneﬁt from coordinating the expression of its major virulence factors with its metabolism and motility , and that it should use a global regulator such as Fis to accomplish this .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15256548" target="_blank">15256548</a>]<br></li>
<li>Of the small nucleoid-binding proteins that affect hilA expression , H-NS acts as a repressor , but Fis and HU activate hilA by changing the expression of hilD ( 32 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16391141" target="_blank">16391141</a>]<br></li>
<li>Also Fis , which also promotes Spi-1 gene-expression ( see Attachment and Invasion of the Intestinal Epithelium ) , is needed for full expression of Spi-2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26441883" target="_blank">26441883</a>]<br></li>
<li>We have now established that Fis regulates the expression of genes involved with metabolism , transport , ﬂagellar biosynthesis and invasion .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15256548" target="_blank">15256548</a>]<br></li>
<li>Fis also influences the virulence of Salmonella enterica and pathogenic E. coli by regulating their virulence genes , however , the relevant mechanism is unclear .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23717649" target="_blank">23717649</a>]<br></li>
<li>Fis and virulence gene-expression Among the most strongly Fis-activated genes were the virulence genes located within the SPI pathogenicity islands ( see Fig. 2 and Fig. 3 ) and the chemotaxis / ﬂagellar regulons ( Fig. 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15256548" target="_blank">15256548</a>]<br></li>
<li>This group included most of the global regulators and many were differentially affected under the different growth conditions including Crp , Fis , ArcA , SlyA , Fur , and H-NS ( Figures 1D -- F ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25713562" target="_blank">25713562</a>]<br></li>
<li>In particular , Fis regulates those genes encoding the type-III-secretion machinery and cognate effectors required by the bacterium for invasion-of-host epithelial cells , for survival in macrophage and for the deployment of ﬂagella for motility .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15256548" target="_blank">15256548</a>]<br></li>
<li>In this study , using the pYH1 derivative ( pYDF ) and a Fis -- strain , we have shown that Fis and the putative Fisbinding region are important for the expression of cspH upon nutrient up-shift .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15235764" target="_blank">15235764</a>]<br></li>
<li>In conclusion , this study identiﬁes a new reciprocal expression proﬁle for the global regulators Fis and RpoS during growth in the absence of aeration .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17784910" target="_blank">17784910</a>]<br></li>
<li>Transcriptomic analysis of S. typhimurium has deﬁned the regulons for polynucleotide phosphorylase ( PNPase ) , CsrA , Fis , histone-like nucleoid-structuring protein ( H-NS ) and phage-shock protein Psp , as well as establishing the effect of various stresses , ranging from microgravity to temperature and bile ( Table 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16413221" target="_blank">16413221</a>]<br></li>
<li>Using a genetic approach , Fis was shown to be necessary for the induction of hilA and invF expression , genes that encode two positive regulators of several SPI-1 genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11123690" target="_blank">11123690</a>]<br></li>
<li>The regulators CsrA , Fis and H-NS were all shown to induce the expression of many SPI1 , motility and che-motaxis-related genes [ 19 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16413221" target="_blank">16413221</a>]<br></li>
<li>Both regulatory and structural genes involved in most aspects of ﬂagellar expression and function were affected by the ﬁs mutation and included genes from the early , middle and late stages of ﬂagellar biosynthesis ( Macnab , 1996 , 2003 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15256548" target="_blank">15256548</a>]<br></li>
<li>A number of regulatory genes are known to be able to inﬂuence the expression of genes involved in pathogenicity or defense against host-derived factors of S. Typhimurium ; these include the two-component phoPQ ( 53 ) and pmrAB ( 42 ) systems ( involved in sensing the transition to an intracellular location and defense against host peptides ) and the regulators of SPI-1 , invF ( 9 ) , hilC ( sprA ) ( 14 ) , and hilA ( 43 ) , and of SPI-2 , ssrA ( 39 ) , as well as global regulators , including ﬁs ( 7 , 50 ) and hns ( 28 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19411325" target="_blank">19411325</a>]<br></li>
<li>Fis regulates genes encoding the T3SS and its cognate effectors as well as synthesis of ﬂagella for motility .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24308841" target="_blank">24308841</a>]<br></li>
<li>These involve positive regulators of ssrAB or the SPI2 genes , such as Fis ( 28 ) , and negative regulators , such as H-NS ( 43 ) and YdgT ( 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19858298" target="_blank">19858298</a>]<br></li>
<li>Expression of the SPI-1 genes also requires core-genome-encoded regulators such as OmpR , Fis and H-NS [ 62 -- 65 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24603618" target="_blank">24603618</a>]<br></li>
<li>A dedicated translation factor controls the synthesis of the global regulator Fis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18621905" target="_blank">18621905</a>]<br></li>
<li>Owens , R.M. , Pritchard , G. , Skipp , P. , Hodey , M. , Connell , S.R. , Nierhaus , K.H. , and O'Connor , C.D. ( 2004 ) A dedicated translation factor controls the synthesis of the global regulator Fis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17784910" target="_blank">17784910</a>]<br></li>
<li>Fis repressed transcription of ndk , the gene that encodes nucleoside diphosphate kinase ( Table 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15256548" target="_blank">15256548</a>]<br></li>
<li>Expression of hilA : : Tn5lacZY and invF : : Tn5lacZY in the fis mutant could be restored by introducing a plasmid containing the S. typhimurium fis gene or a plasmid containing hilD , a gene encoding an AraC-like regulator of Salmonella invasion genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11123690" target="_blank">11123690</a>]<br></li>
<li>Fis is thought to be involved in the adaptation of cellular-metabolism to rapid growth by serving as an indicator molecule for the physiological state of the cell ( Nilsson et al. 1992 ; Ninnemann et al. 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15235764" target="_blank">15235764</a>]<br></li>
<li>Generally the effect of the ﬁs mutation was to decrease gene-expression , indicating a role for Fis as a transcription activator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15256548" target="_blank">15256548</a>]<br></li>
<li>We found Fis to have a positive role in the expression of ethanolamine utilization genes ( eut ) at the 4 h time point with little or no effect at 1 h ( Table 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15256548" target="_blank">15256548</a>]<br></li>
<li>In particular Fis seems to be well placed to coordinate the expression of genes involved in the transition from a free-living life in the gut lumen to the intracellular niche .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15256548" target="_blank">15256548</a>]<br></li>
<li>For example , Fis was reported to influence the transcription of the virulence genes at the locus of enterocyte effacement ( LEE ) in enteropathogenic E. coli and therefore , to affect the invasion ability of the pathogen [ 29 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23717649" target="_blank">23717649</a>]<br></li>
<li>Moreover , several of these regulators play a master role in the timing of virulence , since they drive the transition from the exponential growth to the stationary phase ( e.g. , Fis and IHF ) as well as from the extracellular to the intracellular environment ( e.g. , PhoQ-PhoP ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23554419" target="_blank">23554419</a>]<br></li>
<li>We also observed increased exponentialphase expression of the RpoS-dependent reporters katE-lac [ op ] ( threefold ) and proV-lac [ op ] ( fourfold ) in the ﬁs mutant , a ﬁnding consistent with the Western blot analysis of RpoS protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15716427" target="_blank">15716427</a>]<br></li>
<li>A number of genes encoding nucleoid-associated proteins with global regulatory roles were affected by the absence of Fis in the mutant ( supplementary data Tables S1 and S2 at http://mic.sgmjournals.org ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15256548" target="_blank">15256548</a>]<br></li>
<li>Sequence analysis of genes regulating intracellular levels of RpoS To identify factors that inﬂuence an increase in the RpoS levels in the KLX2 strain , sequence analysis was carried out on KLX2 and the paternal strain LT2 , for the genes known to be involved in controlling intracellular levels of RpoS ( relA , spoT , hfq , hns , arcA , arcB , ﬁs , cspC , cspE , rprA , rpoE , rssA , rssB , iraP , iraD , iraM , clpX , clpP , and dnaK ) and the rpoS gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23385142" target="_blank">23385142</a>]<br></li>
<li>Histogram of S. typhimurium cell length at days 1 , 3 , and 6 post inoculation in the presence of A. polyphaga Plasmid pPDT105 which carries the ﬁs promoter and gene encoding eGFP ( Clontech ) was used as an indicator of physical status of the bacteria .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14502413" target="_blank">14502413</a>]<br></li>
<li>Combining the results of ChIP-seq , RNA-seq , real-time PCR ( RTPCR ) , mutation , and cell infection experiments , we showed that Fis influences the expression of 63 of the 94 SPI-1 and SPI-2 genes , which are responsible for the invasion and intracellular replication of LT2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23717649" target="_blank">23717649</a>]<br></li>
<li>Evidence has also been provided that pdu mutations are responsible for attenuation of the virulence of Salmonella ( 14 ) and that the global Salmonella virulence regulators CsrA and Fis inﬂuence the expression of the eut gene ( 27 , 34 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21037291" target="_blank">21037291</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_210">
Goldberg , M.D. , Johnson , M. , Hinton , J.C.D. , and Williams , P.H. ( 2001 ) Role of the nucleoid-associated protein Fis in the regulation of virulence properties of enteropathogenic Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16999831" target="_blank">16999831</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17784910" target="_blank">17784910</a>]<br></label>
<input id="rp_similarity_210", type="checkbox">
<div class="RP_similar">
<li>Goldberg MD , Johnson M , Hinton JCD , Williams PH : Role of the nucleoid-associated protein Fis in the regulation of virulence properties of enteropathogenic Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21418628" target="_blank">21418628</a>]<br></li>
<li>Role of the nucleoid-associated protein Fis in the regulation of virulence properties of enteropathogenic Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14761985" target="_blank">14761985</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15256548" target="_blank">15256548</a>]<br></li></div>
<li>Fis , an accessorial factor for transcriptional activation of the mar ( multiple antibiotic resistance ) promoter of Escherichia coli in the presence of the activator MarA , SoxS , or Rob .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15073288" target="_blank">15073288</a>]<br></li>
<li>Galan B , Manso I , Kolb A , Garcia JL , Prieto MA : The role of FIS protein in the physiological control of the expression of the Escherichia coli meta-hpa operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24885225" target="_blank">24885225</a>]<br></li>
<li>We are trying to elucidate more detailed function of Fis in regulation of expression of various genes identi cents ed in this study .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14553938" target="_blank">14553938</a>]<br></li>
<li>The involvement of Fis in the control of virulence genes points to an interesting paradox .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17784910" target="_blank">17784910</a>]<br></li>
<li>The effect of aeration on the expression of Fis in S. Typhimurium SL1344 and E. coli CSH50 ( A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17784910" target="_blank">17784910</a>]<br></li>
<li>Fis affects expression of invF : : Tn5lacZY in the absence of HilA Current models for the regulation of invasion genes suggest that there is a HilA-independent pathway to invF expression that involves HilD ( Lucas and Lee , 2000 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11123690" target="_blank">11123690</a>]<br></li>
<li>The involvement of Fis in negative regulation of ndk was of interest given its similar role in the expression of the nupG and rbsC nucleoside-transport genes , suggesting that Fis coordinates the expression of genes involved in pyrimidine-metabolism .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15256548" target="_blank">15256548</a>]<br></li>
<li>As a result of our continued studies of SPI-1 gene regulation , we have discovered that the nucleoidbinding protein Fis plays a pivotal role in the expression of HilA and InvF , two activators of SPI-1 genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11123690" target="_blank">11123690</a>]<br></li>
<li>The reason that the expression of cob genes in pocR deleted strains is still dependent of Fis effect is Fis may also control the expression of B12 biosynthesis genes through other unknown negative regulators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23717649" target="_blank">23717649</a>]<br></li>
<li>Fis is required for optimal expression of SPI1 genes , and the non-aerated growth conditions that facilitate expression of an invasive phenotype also result in sustained Fis expression in late stages of growth .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17784910" target="_blank">17784910</a>]<br></li>
<li>This also suggests that at least some of the functions played by Fis ( and perhaps also the product of ORF1 ) in these bacteria are related to its growth phase-dependent expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9811652" target="_blank">9811652</a>]<br></li>
<li>In this study , we determined the genome-wide distribution of Fis-binding regions in S. enterica LT2 , analyzed the regulation of global gene transcription by Fis , and identified the molecular mechanisms by which Fis acts to influence virulence of LT2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23717649" target="_blank">23717649</a>]<br></li>
<li>In addition , we implicate the alternate stress response sigma factor RpoS as playing a role in repressing Fis expression , and show that induction of Fis in the absence of aeration is facilitated by a reduction of RpoS levels under these conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17784910" target="_blank">17784910</a>]<br></li>
<li>This is a signiﬁcant omission , becaus Communicated by W. Goebel O. M. Keane Æ C. J. Dorman ( & ) Department of Microbiology , Moyne Institute of Preventive Medicine , University of Dublin , Trinity College , Dublin 2 , Ireland E-mail : cjdorman@tcd.ie Tel. : +353 -1 -6082013 Fax : +353 -1 -6799294 DNA supercoiling has an important inﬂuence on the expression of virulence genes in S. enterica ( Marshall et al. 2000 ) and the Fis protein has been found to inﬂuence the expression of genes in the SPI-1 pathoge-nicity island ( Wilson et al. 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12898222" target="_blank">12898222</a>]<br></li>
<li>Fis plays a key role in controlling the expression of the translation machinery of the cell following nutritional upshift ( Appleman et al. , 1998 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15790293" target="_blank">15790293</a>]<br></li>
<li>In addition , we analyzed the regulation of ﬁs expression in S. typhimurium , including the role of Fis as an autoregulator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7536730" target="_blank">7536730</a>]<br></li>
<li>Therefore , we suggest that Fis plays a role in coordinating the expression of house-keeping genes with that of virulence genes as part of a regulatory mechanism controlling the transition from a free-living mode in the gut lumen to an intracellular niche .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15256548" target="_blank">15256548</a>]<br></li>
<li>In this study , we re-examined the pattern of expression of the Fis protein in the context of those growth conditions that have been described as optimal for the expression of the S. Typhimurium virulence genes in the SPI1 patho-genicity island .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17784910" target="_blank">17784910</a>]<br></li>
<li>The expression of the global regulator of transcription , ﬁs , was also increased ( Table 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19411325" target="_blank">19411325</a>]<br></li>
<li>In addition , it has been proved that CRP-cAMP modulates fis expression and the inhibition of DNA gyrase represses the expression of several CRP-cAMP sensitive genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25033662" target="_blank">25033662</a>]<br></li>
<li>The result showed that Fis activates the transcription of B12 biosynthesis genes mainly through controlling the expression of pocR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23717649" target="_blank">23717649</a>]<br></li>
<li>Concerning the ﬂagellar genes , inactivation of the ﬁs gene has been associated with diminished expression of the ﬂagellar and motility genes ( highlighting a strong effect on expression of rtsB , a regulator of ﬂagellar genes [ see below ] ) at 4 h of growth by means of transcriptomic analysis ( 305 , 306 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23554419" target="_blank">23554419</a>]<br></li>
<li>The regulon of genes whose expression is inﬂuenced by Fis in Salmonella enterica serovar Typhimurium ( S. typhimurium ) has been deﬁned by DNA microarray analysis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15256548" target="_blank">15256548</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_236">
Fis regulates transcriptional induction of RpoS in Salmonella enterica .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20075614" target="_blank">20075614</a>]<br></label>
<input id="rp_similarity_236", type="checkbox">
<div class="RP_similar">
<li>Hirsch M , Elliott T : Fis regulates transcriptional induction of RpoS in Salmonella enterica .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24885225" target="_blank">24885225</a>]<br></li></div>
<li>Fis is involved in SPI expression Positive regulation of SPI-1 genes by Fis was expected considering the fact that hilA , a transcriptional regulator of SPI-1 , is upregulated by Fis [ 16 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14553938" target="_blank">14553938</a>]<br></li>
<li>These results suggest that Fis may a ¡ ect expression of genes in SPI-2 as well as SPI-1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14553938" target="_blank">14553938</a>]<br></li>
<li>We could cents nd several virulence genes from both SPI-1 and -2 whose expressions are a ¡ ected by Fis in addition to genes involved in various aspects of cellular physiology .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14553938" target="_blank">14553938</a>]<br></li>
<li>The genes that were most strongly up-regulated by Fis were those involved in virulence and located in the pathogenicity islands SPI-1 , SPI-2 , SPI-3 and SPI-5 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15256548" target="_blank">15256548</a>]<br></li>
<li>Fis has been shown to be involved in a variety of cellular-processes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15235764" target="_blank">15235764</a>]<br></li>
<li>Fis has also been shown to bind oriC in E. coli and to play a role in initiation of DNA-replication ( 12 , 16 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7536730" target="_blank">7536730</a>]<br></li>
<li>Fis was found to act as a transcriptional repressor of both gyr genes in S. enterica .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12898222" target="_blank">12898222</a>]<br></li>
<li>Fis was also shown to participate in other processes such as stimulation of phage V excision [ 12 ] , growth modulation of phage Mu [ 13 ] , replication at oriC [ 14 ] and RNA synthesis [ 15 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14553938" target="_blank">14553938</a>]<br></li>
<li>Fis Regulates Transcriptional Induction of RpoS in Salmonella enterica Matthew Hirsch and Thomas Elliott * ent of M Departm icrobiology , Immunology , and Cell Biology , West Virginia University , M Received 23 August 2004/Accepted 19 November 2004 organtown , West Virginia The sigma factor RpoS is known to regulate at least 60 genes in response to environmental sources of stress or during growth to stationary phase ( SP ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15716427" target="_blank">15716427</a>]<br></li>
<li>Genes encoding the regulators ( Dps , Lrp , and RtsA/B ) , only RtsA/B have been global regulators fis and fur also showed elevated expression shown to be involved in hilA gene-expression ( 51 , 52 ) , and both levels in the relA spoT strain under low oxygen growth congenes showed 3 -- 5-fold reductions in expression in the absence ditions ( Fig. 3 , B and D ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16905537" target="_blank">16905537</a>]<br></li>
<li>The main regulatory proteins involved in the expression of Spi-2 include the two-component systems SpiR ( SsrA ) / SsrB , PhoP/PhoQ , OmpR/EnvZ , as well as the DNA-binding proteins HilD , SlyA ( RovA in Yersinia ) , RcsB , the iron regulator Fur , and the nucleoid-associated proteins H-NS , Hha ( YmoA in Yersinia ) , YdgT , IHF , and Fis ( Fass and Groisman , 2009 ; Choi et al. , 2014 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26441883" target="_blank">26441883</a>]<br></li>
<li>These data suggest that Fis plays a central role in coordinating the expression of both metabolic and type-III-secretion factors .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15256548" target="_blank">15256548</a>]<br></li>
<li>Fis was reported to influence the expression of many genes on SPI-1 and SPI-2 in S. enterica serovar Typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23717649" target="_blank">23717649</a>]<br></li>
<li>These results indicated that ompR plays an important role in Fis regulation of SPI-1 and SPI-2 genes , and influences the infection and replication of LT2 in host cells .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23717649" target="_blank">23717649</a>]<br></li>
<li>Our data identify a coordinating role for Fis in the activation of virulence genes in SPI-1 , SPI-2 , some in SPI-3 , SPI-4 and SPI-5 , and are consistent with the previously demonstrated link between expression of SPI-5 genes and those of SPI-1 and SPI-2 ( Knodler et al. , 2002 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15256548" target="_blank">15256548</a>]<br></li>
<li>Equal numbers of bacteria were used to inoculate the centres of semi-solid swarming agar plates and incubated at 37 6C for 8 h. Genes involved in propanediol utilization by S. typhimurium were repressed by Fis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15256548" target="_blank">15256548</a>]<br></li>
<li>Concluding remarks In an attempt to elucidate the role of Fis in virulence gene regulation in Salmonella we have studied proteome changes caused by cents s mutation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14553938" target="_blank">14553938</a>]<br></li>
<li>Besides its role in global regulation , the roles of Fis in the regulation of virulence properties in E. coli and S. enterica have been previously reported [ 7,29 -- 31 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23717649" target="_blank">23717649</a>]<br></li>
<li>Identiﬁcation and Characterization of the ﬁs Operon in Enteric Bacteria MICHAEL B. BEACH AND ROBERT OSUNA * Department of Biological Sciences , University at Albany , Albany , New York 12222 Received 28 April 1998/Accepted 9 September 1998 The small DNA binding protein Fis is involved in several different biological-processes in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9811652" target="_blank">9811652</a>]<br></li>
<li>The factor-for-inversion stimulation protein ( Fis ) is known to play a pivotal role in the expression of Salmonella pathogenicity island ( SPI ) -1 genes in addition to various cellular-processes such as recombination , replication , and transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14553938" target="_blank">14553938</a>]<br></li>
<li>Fis ( factor for inversion simulation ) is a nucleoid-associated protein that participates in the structuring of the bacterial chromosome , also known to interact at specific promoters to regulate gene transcription in E. coli and S. enterica [ 17-19 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24885225" target="_blank">24885225</a>]<br></li>
<li>[ 33,34 ] Although the absence of Fis , a key regulator in S. Typhimurium virulence , up-regulates the expression of cadA and cadB , [ 35 ] it is still not known how cadA is involved in Salmonella pathogenicity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20390946" target="_blank">20390946</a>]<br></li>
<li>Fis is a global regulatory NAP involved in the processes of replication , recombination , and transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23554419" target="_blank">23554419</a>]<br></li>
<li>We tried to identify genes under Fis control involved in pathogenicity of Sal-monella in order to understand the role of Fis in Salmo-nella virulence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14553938" target="_blank">14553938</a>]<br></li>
<li>We found that Fis acts to modulate expression of genes involved in aspects of metabolism and transport that are relevant to S. typhimurium during life in the gut .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15256548" target="_blank">15256548</a>]<br></li>
<li>These results indicate that Fis is also involved in the regulation of invF expression in a HilA-independent manner .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11123690" target="_blank">11123690</a>]<br></li>
<li>Fis stimulates the expression of proteins involved in translation The spot corresponding to elongation factor Tu ( EFTu ) showed higher expression levels in wild-type than in cents s mutant .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14553938" target="_blank">14553938</a>]<br></li>
<li>Fis modulates expression of various proteins involved in metabolic-processes Proteins regulated by Fis in other cellular-processes were primarily enzymes and transport proteins involved in the metabolism of sugars .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14553938" target="_blank">14553938</a>]<br></li>
<li>This indicates that in LT2 Fis preferentially mediates positively regulation of genes , which is opposite from that in K12 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23717649" target="_blank">23717649</a>]<br></li>
<li>Results Fis participates in the repression of hlyE at transcriptional level CRP is a regulator that acts as a cAMP receptor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24885225" target="_blank">24885225</a>]<br></li>
<li>Several global regulators activate transcription of the flhDC operon , including the cyclic AMP-catabolite repressor protein ( cAMP-CRP ) , the histone-like nucleoid-structuring protein ( H-NS ) and the nucleoid-organization protein Fis ( Kutsukake , 1997 ; Kelly et al. , 2004 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27206164" target="_blank">27206164</a>]<br></li>
<li>Oral LD50 S. typhimurium strain Intraperitoneal LD50 Discussion We have performed experiments that identify the nucleoidbinding protein Fis as an important regulator of S. typhimurium SPI-1 invasion genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11123690" target="_blank">11123690</a>]<br></li>
<li>Of the regulators that have been studied , Fis is a nucleoid-associated protein which is expressed upon nutrient upshift and the H-NS nucleoid-associated protein is involved in the integration of environmental signals .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16413221" target="_blank">16413221</a>]<br></li>
<li>In the present study we show that the Fis protein ( factor for inversion stimulation ) is involved in RpoS regulation during growth in rich medium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15716427" target="_blank">15716427</a>]<br></li>
<li>Salmonella Typhimurium Genes Involved in Virulence Regulated by luxS = Autoinducer-2 at Mid-Log Phase in the Absence of Glucose Fold change W = M Fold change wCFS = mCFS Putative identiﬁcation of JCVI database Genes 1.9 1.0 hha Hemolysin expression modulating protein ( involved in environmental regulation ) PhoPQ-activated gene Putative virulence factor Putative nicotinate-nucleotide dimethylbenzimidazolephosphoribosltransferase , homolog of virulence factor PhoP regulated : reduced macrophage survival Response regulator in protein turnover : mouse virulence Regulation of invasion genes Virulence gene homologous sequence to virK in Putative inner membrane protein Putative cytoplasmic protein Putative ﬂagellar biosynthesis = type III secretory pathway Cell invasion protein cytoplasmic Regulatory helix-turn-helix proteins , araC family Invasion genes transcription activator Protein tyrosine phosphate Chaparone , related to virulence Putative cytoplasmic protein Cell invasion protein Surface presentation of antigen secretory proteins Surface presentation of antigen secretory proteins Surface presentation of antigen secretory proteins Invasion protein Invasion protein Putative invasion protein NTP pyrophosphohydrolase Site-speciﬁc DNA inversion stimulation factor Host factor I for bacteriophage Q beta 1.4 1.3 pagP mviM cobB 2.2 1.6 1.3 1.2 pagC hnr 1.5 1.8 1.3 1.4 2.3 2.5 1.4 1.8 1.7 1.4 1.2 2.0 1.3 1.4 1.4 1.3 1.5 1.5 1.4 1.6 1.4 1.4 4.6 1.4 sirC virK avrA STM2868 orgA prgI hilD hilA sptP sicP STM2880 sipB spaR invC invB invA invE ygdP ﬁs hfq 1.2 1.4 3.1 2.1 2.7 9.6 3.3 6.1 5.9 5.0 2.9 2.8 9.2 11.2 7.5 6.8 10.5 1.0 1.0 1.2 protein Differentially regulated genes between wild type ( W ) and DluxS mutant ( M ) and between wild-type cell-free supernatants and DluxS mutant cell-free supernatants are listed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19909098" target="_blank">19909098</a>]<br></li>
<li>Molecular Microbiology ( 2001 ) 39 ( 1 ) , 79 ± 88 Fis , a N nucleoid-associated protein , is involved in murium SPI-1 invasion gen D A onella t Salm e expression yphi Rebecca L. Wilson , Stephen J. Libby , 1 2 Anne M. Freet , Jennifer D. Boddicker , 1 1 1 1 Thomas F. Fahlen and Bradley D. Jones * 1 Department of Microbiology , University of Iowa School of Medicine , Iowa City , IA 52242 , USA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11123690" target="_blank">11123690</a>]<br></li>
<li>Many genetic elements that exert regulatory effects on hilA have been identiﬁed including positive elements hilC/sirC/sprA ( 10 , 48 , 50 ) , hilD ( 50 ) , sirA ( 26 ) , ﬁs ( 55 ) , barA , csrAB ( 1 ) , and phoB , fadD , and ﬂiZ ( 39 ) as well as negative elements such as phoPQ ( 5 , 46 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11673432" target="_blank">11673432</a>]<br></li>
<li>Supernatant proteins from SL1344 ( wild type ) , BJ1625 ( fis : : kan ) , BJ70 ( hilA : : Tn5lacZY ) and BJ72 ( invF : : Tn5lacZY ) cultures .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11123690" target="_blank">11123690</a>]<br></li>
<li>Resistance to antimicrobial agents other than ciprofloxacin : Cot , trimethoprim -- sulfamethoxazole ; Fis , sulfamethoxazole or sulfisoxazole ; Nal , nalidixic acid ; Tet , tetracycline ; Amp , ampicillin ; Chl , chloramphenicol ; Str , streptomycin .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21749779" target="_blank">21749779</a>]<br></li>
<li>Resistance profiles : AMP , ampicillin ; CHL , chloramphenicol ; STR , streptomycin ; FIS , sulfisoxazole ; TET , tetracycline ; AUG2 , amoxicillin -- clavulanic acid ; XNL , ceftiofur ; AXO , ceftriaxone ; FOX , cefoxitin ; KAN , kanamycin ; SXT , trimethoprim-sulfamethoxazole ; NAL , nalidixic acid .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23127704" target="_blank">23127704</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23127704" target="_blank">23127704</a>]<br></li>
				</div></ul><br>
			<label class="collapse_SIT" for="_5"><font size="5px">FIS binding site</font></label>
				<input id="_5" type="checkbox">
				<div class="SIT"><br><ul><li>Underlined sequences indicate putative Fis boxes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24885225" target="_blank">24885225</a>]<br></li>
<li>Plasmid pMB315 contains coding sequences for E. carotovora Fis , and pMB314 contains coding sequences for P. vulgaris Fis ( Table 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9811652" target="_blank">9811652</a>]<br></li>
<li>Here , we found putative Fis binding sequences at 254 and 230 ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11544220" target="_blank">11544220</a>]<br></li>
<li>Fis binds several DNA sites near PrpoS in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15716427" target="_blank">15716427</a>]<br></li>
<li>Cooperative DNA binding is a feature common among NAPs -- like FIS and H-NS -- that bind with low-specificity to multiple proximal DNA sites [ 24,25 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22457642" target="_blank">22457642</a>]<br></li>
<li>Discussion Fis is an 11.2-kDa protein that binds to speci cents c DNA sites displaying a highly degenerate consensus sequence [ 28 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14553938" target="_blank">14553938</a>]<br></li>
<li>The DNA sequences that are bound by Fis show weak homology to a degenerate consensus ( Hengen et al. , 1997 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17784910" target="_blank">17784910</a>]<br></li>
<li>2 ChIP-seq RNA-seq Sample Mock-IP IP LT2 Dfis Number of reads 6,576,604 3,350,972 13,894,928 14,708,050 Number of mappable 5,717,748 2,017,349 13,202,514 14,089,439 reads 42 Number of reads per base118 290 272 2 Number of binding 2 a regions 885 bp 2 2 2 Average length of binding sites 349 bp a The number of Fis binding regions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23717649" target="_blank">23717649</a>]<br></li>
<li>C ) In silico sequence analysis of CRP and Fis binding boxes in S. Typhi CT18 rpoS promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24885225" target="_blank">24885225</a>]<br></li>
<li>Predicted Fis binding sites are underlined , and arrows designate putative CRP-binding half-sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15716427" target="_blank">15716427</a>]<br></li>
<li>Information analysis of Fis binding sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15716427" target="_blank">15716427</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15256548" target="_blank">15256548</a>]<br></li>
<li>CRP and FIS ( grey boxes ) and IHF ( underline ) binding sites identified in E. coli are shown [ 5,6,40 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24358360" target="_blank">24358360</a>]<br></li>
<li>Fis sites III and V are weakly bound in S. typhi-murium , and site IV is absent .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7536730" target="_blank">7536730</a>]<br></li>
<li>It is noteworthy that a 400 bp divergent promoter region is found upstream of ssaB and contains multiple binding sites for transcription factors including OmpR , SsrB , SlyA and Fis ( Lee et al. , 2000 ; Feng et al. , 2003 ; 2004 ; Kelly et al. , 2004 ; Navarre et al. , 2006 ; D. Walthers and L.J. Kenney , unpublished ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630976" target="_blank">17630976</a>]<br></li>
<li>A degenerate consensus sequence has been identiﬁed for Fis where it introduces a bend of between 40u and 90u upon binding ( Hengen et al. , 1997 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15256548" target="_blank">15256548</a>]<br></li>
<li>We believe that Fis probably acts directly as a repressor given that a speciﬁc site positioned at 50 is necessary for complete repression and that this site is speciﬁcally bound in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15716427" target="_blank">15716427</a>]<br></li>
<li>This PCR product was used as a megaprimer for yet a third PCR with an additional primer that annealed to sequences about 386 bp downstream of ﬁs and created a HindIII site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9811652" target="_blank">9811652</a>]<br></li>
<li>Open boxes , the 210 and 235 promoter sequences ; arrow , the start of transcription ; solid and open bars , represent high - and low-afﬁnity Fis sites , respectively , which are identiﬁed by roman numerals .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7536730" target="_blank">7536730</a>]<br></li>
<li>DNA fragments carrying these two genes were synthesized by PCR with pMB180 and pMB182 as DNA templates and oligonucleotides that annealed to the 59 and 39 ends of these ﬁs sequences while generating XbaI and PvuII restriction sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9811652" target="_blank">9811652</a>]<br></li>
<li>This was done by PCR with plasmids pCK186 , pMB193 , pMB194 , and pMB258 as templates , an oligonucleotide that annealed to the respective upstream ﬁs P regions and generated EcoRI sites , and the same oligonucleotides used above that annealed to ORF1 sequences and generated XbaI sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9811652" target="_blank">9811652</a>]<br></li>
<li>This NcoI site is located 78 bp downstream from the ﬁs transcription start site ( Fig. 1B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7536730" target="_blank">7536730</a>]<br></li>
<li>Solid arrow , the preferred transcription start site ; dashed arrow , a less-efﬁcient start site , inverted arrows downstream of ﬁs , a region of dyad symmetry which is predicted to serve as a signal for transcription termination .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7536730" target="_blank">7536730</a>]<br></li>
<li>Deleting regions of the fis promoter led to the discovery of a novel anaerobically-induced transcription start site ( Pfis-2 ) upstream of the primary transcription start site ( Pfis-1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24358360" target="_blank">24358360</a>]<br></li>
<li>A major determinant of growth phase-dependent fis expression is the GC-rich sequence between the − 10 RNA polymerase binding site and the transcription start site , which creates a protein-independent barrier to transcription [ 7,8 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24358360" target="_blank">24358360</a>]<br></li>
<li>The occupancy of these additional four Fis sites in E. coli might serve to hinder upstream DNA sequences from a potential role in enhancing transcription activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7536730" target="_blank">7536730</a>]<br></li>
				<label class="SIT_collapse_similarity" for="sit_similarity_26">
The promoter of the tgt/sec operon in Escherichia coli is preceded by an upstream activation sequence that contains a high afﬁnity FIS binding site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10572129" target="_blank">10572129</a>]<br></label>
<input id="sit_similarity_26", type="checkbox">
<div class="SIT_similar">
<li>The promoter of the tgt/sec operon in Escherichia coli is preceded by an upstream activation sequence that contains a high afﬁnity Fis binding site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7536730" target="_blank">7536730</a>]<br></li></div>
<li>In H. inﬂuenzae , sequences preceding ORF1 show little or no resemblance to the ﬁs P region in E. coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9811652" target="_blank">9811652</a>]<br></li>
<li>However , the sequence in this region which contains an inverted-repeat is very well conserved , which is consistent with its suggested role as a transcription terminator for the ﬁs operon ( 4 , 29 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7536730" target="_blank">7536730</a>]<br></li>
<li>The limited DNA sequence immediately downstream of the ﬁs coding region shows a 28 % divergence from E. coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7536730" target="_blank">7536730</a>]<br></li>
<li>Indeed , no BamHI sites were identiﬁed in the available sequence upstream of ﬁs ( see Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7536730" target="_blank">7536730</a>]<br></li>
<li>Nucleotide sequence of the ﬁs operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7536730" target="_blank">7536730</a>]<br></li>
<li>Both the P. vulgaris and S. marcescens ihf sites centered at 2116 contain only two mismatches from the E. coli ihf consensus sequence ( Table 3 ) , whereas the ihf sites for all other ﬁs promoters contain only one mismatch .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9811652" target="_blank">9811652</a>]<br></li>
<li>In E. coli , the region protected from DNase I digestion by Fis begins 2 bp upstream of the -- 10 box ( Schneider et al. 1999 ) , while in S. enterica it begins 16 bp upstream of this promoter hexamer ( Fig. 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12898222" target="_blank">12898222</a>]<br></li>
<li>Ó Cróinín and Dorman used a plasmid-based reporter gene fusion containing the fis promoter region up to -298 bp , thus omitting the 330 bp of upstream non-coding sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24358360" target="_blank">24358360</a>]<br></li>
<li>This fragment provided the DNA sequence for the entire ORF1 and ﬁs promoter region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7536730" target="_blank">7536730</a>]<br></li>
<li>To verify the DNA sequences of the ﬁrst six codons of ﬁs , PCR products were obtained from chromosomal DNA by using an oligonucleotide that annealed to the various ﬁs sequences with an oligonucleotide that annealed to the beginning of ORF1 or sequences with upstream of ORF1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9811652" target="_blank">9811652</a>]<br></li>
<li>Therefore , we attempted to obtain ﬁs promoter sequences from K. pneumoniae , S. marcescens , E. carotovora , and P. vulgaris by PCR with degenerate primers that annealed to a region near the 39 end of prmA and primers that annealed near the 59 end of ORF1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9811652" target="_blank">9811652</a>]<br></li>
<li>Characterized Fis sites have total scores that range from 2.5 to 15.7 , and the wellstudied ones in hin proximal to the hixL site have scores of 8 to 9 ( 27 ) , whereas the total scores for random sequences average 0 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15716427" target="_blank">15716427</a>]<br></li>
<li>Since we had previously shown that the important regulatory sequences for the E. coli ﬁs P reside in the region from 2168 to 178 relative to the predominant transcription start site ( 45 ) , we synthesized shorter DNA fragments containing the various ﬁs P regions from either 2172 or 2168 to 178 and fused them to the ( trp-lac ) W200 sequence in pRJ800 to make pMB260 , pMB261 , pMB262 , and pMB263 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9811652" target="_blank">9811652</a>]<br></li>
<li>These sequences are located at these same positions relative to the transcriptional start sites , except for that in S. typhi-murium , which is 1 bp closer to the ﬁs promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9811652" target="_blank">9811652</a>]<br></li>
<li>The promoter region for the ﬁs operon in E. coli ( 4 , 43 ) , which is located in the DNA sequence upstream of ORF1 , is completely conserved in S. typhimurium from 249 to 194 relative to the start of transcription in E. coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7536730" target="_blank">7536730</a>]<br></li>
<li>Black arrows represent promoters transcribing in the direction of ORF1 and ﬁs ; the shaded arrow represents a DNA region preceding the H. inﬂuenzae ORF1 which contains a good match to the consensus sequence for s70 promoters , but its function in vivo remains untested .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9811652" target="_blank">9811652</a>]<br></li>
<li>Five of the most conserved base pairs in the consensus are marked with an underline ( where present in each sequence ) , and the asterisk marks the axis of rotational symmetry for the Fis consensus sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15716427" target="_blank">15716427</a>]<br></li>
<li>Based on these results and the requirement for speciﬁc sequences upstream of the major rpoS promoter , we present an intuitive model for transcriptional regulation of rpoS in which Fis binds to and represses transcription from PrpoS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15716427" target="_blank">15716427</a>]<br></li>
<li>The ﬁs gene is transcribed from a single promoter , and the sequences of the ﬁs promoters from E. coli and S. Typhimurium are identical over a region extending from the translation start codon to a position lying 14 base pairs upstream of the -35 box ( Mallik et al. , 2004 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17784910" target="_blank">17784910</a>]<br></li>
<li>Seemingly , sequences upstream of 2172 in the P. vulgaris and S. marces-cens ﬁs P regions are capable of interfering with the ability of Fis to regulate transcription at ﬁs P .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9811652" target="_blank">9811652</a>]<br></li>
<li>Mutation analysis of the ﬁs promoter region is currently in progress to help elucidate the precise role of each of these Fis sites as well as of other DNA sequences in the transcription activity of the ﬁs promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7536730" target="_blank">7536730</a>]<br></li>
<li>However , no sequence similar to a FIS consensus binding site ( 33 ) is present in the miaB promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10572129" target="_blank">10572129</a>]<br></li>
<li>As was expected from the identical sequence between 249 to 194 , Fis sites I and II are present in both promoter regions and bind Fis with similar afﬁnities .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7536730" target="_blank">7536730</a>]<br></li>
<li>MarR binding sequences in the marRAB promoters are italicized grey marked and the binding site of Fis is marked in dark grey and underlined .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19120970" target="_blank">19120970</a>]<br></li>
<li>Sequences with matches to the consensus for Fis binding sites are indicated in bold , Pribnow boxes are shown in italics Fis protein binds to the S. enterica gyr promoters The interaction of Fis protein with the S. enterica gyr promoters was examined by electrophoretic mobility shift assay .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12898222" target="_blank">12898222</a>]<br></li>
<li>Only 145 common genes are bound by Fis in both K12 and LT2 , which is possibly due to the difference in their genome sequence , as 320 of the 943 Fis-binding genes in LT2 are not present in K12 [ 24 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23717649" target="_blank">23717649</a>]<br></li>
<li>The results reported here show , however , that Fis regulation is conserved in S. enterica gyrA despite the sequence diversity , and involves Fis binding sites with some homology to the consensus sequence that are otherwise unrelated to those used at gyrA in E. coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12898222" target="_blank">12898222</a>]<br></li>
<li>While Fis site II is very well conserved in all the ﬁs promoters analyzed here , some variations exist in the region corresponding to Fis site I in S. marcescens , E. caroto-vora , and P. vulgaris ( Fig. 3B ) , with the most severe disruptions to the Fis consensus sequence occurring in P. vulgaris .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9811652" target="_blank">9811652</a>]<br></li>
<li>Given the almost identical sequences at gyrB in S. enterica and E. coli , and the conservation of the Fis binding sites , both in sequence and location , it is reasonable to propose that Fis repression acts through the same mechanism at this promoter in each organism .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12898222" target="_blank">12898222</a>]<br></li>
<li>The proposed DNA consensus sequence for Fis binding is highly degenerate ( 13 , 22 ) and , therefore , can not be used to predict which of the Fis-binding sites within the E. coli ﬁs promoter region are also present in S. typhimurium , particularly in the less-conserved region upstream of 249 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7536730" target="_blank">7536730</a>]<br></li>
<li>Construct C is identical to construct A except for a set of three point mutations ( represented by an asterisk ) altering conserved nucleotides of Fis site III .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15716427" target="_blank">15716427</a>]<br></li>
<li>Fis was found to bind to a total of 309 kb ( approximately 8 % ) sequences on the LT2 genome .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23717649" target="_blank">23717649</a>]<br></li>
<li>Perhaps this represents a selective pressure acting on Fis to maintain its ability to bind to sites with a high degree of DNA sequence diversity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15256548" target="_blank">15256548</a>]<br></li>
<li>In each case Fis binds to several positions , although nearly all regulation is conferred by a single Fis binding site centered at 42 ( Pﬁs ) and 41 ( P2proP ) from the transcriptional start sites ( 41 , 50 , 72 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15716427" target="_blank">15716427</a>]<br></li>
<li>Our analysis predicted a single high-scor-ing Fis binding site centered at position 50 with respect to the PrpoS start site ( bit score of 10.9 ; TCTGCACAAAATTCCAC CGTT , Fig. 4 ; Fis site III in Fig. 7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15716427" target="_blank">15716427</a>]<br></li>
<li>The B target that has a 21-bp deletion of Fis site III also lost the ability to bind Fis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15716427" target="_blank">15716427</a>]<br></li>
<li>One of the most convincing ( and intriguing ) potential binding sites was a strong Fis site centered at bp 50 with respect to the transcriptional start .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15716427" target="_blank">15716427</a>]<br></li>
<li>In fact , we found 83 sites with scores equal to or greater than that of rpoS Fis site III within the ﬁrst 106 bp of the E. coli genomic sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15716427" target="_blank">15716427</a>]<br></li>
<li>Within each region a match to the consensus sequence for Fis binding sites was detected ( Fig. 5B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12898222" target="_blank">12898222</a>]<br></li>
<li>( B ) The consensus sequence for Fis protein binding is given , as well as the sequences of predicted Fis-binding sites near PrpoS , individually designated by Roman numerals .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15716427" target="_blank">15716427</a>]<br></li>
<li>The E. coli Fis protein has a preference for binding sites located within regions of DNA curvature and is known to bind as a dimer ( Wagner , 2000 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15256548" target="_blank">15256548</a>]<br></li>
<li>Sequence , regulation , and functions of ﬁs in Salmonella typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8752327" target="_blank">8752327</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16980465" target="_blank">16980465</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15790293" target="_blank">15790293</a>]<br></li>
<li>( A ) ﬁs nucleotide sequences from several bacteria are aligned with that of E. coli for comparison .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9811652" target="_blank">9811652</a>]<br></li>
<li>ﬁs promoter sequences .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9811652" target="_blank">9811652</a>]<br></li>
<li>( B ) Nucleotide sequences of ﬁs promoter regions from various bacteria .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9811652" target="_blank">9811652</a>]<br></li>
				</ul></div><br>
			<label class="collapse_TU" for="_6"><font size="5px">Transcription unit regulated by FIS</font></label>
				<input id="_6" type="checkbox">
				<div class="TU"><br><ul><li>For example , both proteins influence DNA supercoiling ; FIS represses gyrB [ 35 ] , whereas RpoS transcribes gyrB [ 36 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24358360" target="_blank">24358360</a>]<br></li>
<li>Under standard laboratory growth conditions , fis is highly expressed during rapid growth but is then silenced as growth slows .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24358360" target="_blank">24358360</a>]<br></li>
<li>Transcriptome analysis has revealed that S. enterica serovar Typhi fis is induced inside macrophage vacuoles despite nutrient-poor conditions and slow growth [ 14 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24358360" target="_blank">24358360</a>]<br></li>
<li>The bicistronic dusB-fis operon encodes FIS in both species .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24358360" target="_blank">24358360</a>]<br></li>
<li>This NcoI site is located 78 bp downstream from the ﬁs transcription start site ( Fig. 1B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7536730" target="_blank">7536730</a>]<br></li>
<li>We wondered whether Fis might also be required for invF expression through this pathway .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11123690" target="_blank">11123690</a>]<br></li>
<li>The nucleoid proteins HU and Fis are also required for hilA expression ( Wilson et al. , 2001 ; Schechter et al. , 2003 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16045614" target="_blank">16045614</a>]<br></li>
<li>Its own gene , ﬁs , is among those that it represses ( Pratt et al. , 1997 ) , while it acts to upregulate many of the genes that encode components of the translation machinery of the cell ( Bosch et al. , 1990 ; Nilsson et al. , 1992 ; Ross et al. , 1990 ; Ball et al. , 1992 ; Ninnemann et al. , 1992 ; Walker et al. , 1999 ; Schneider et al. , 2003 ; Paul et al. , 2004 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17784910" target="_blank">17784910</a>]<br></li>
<li>The ﬁs operon and promoter region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9811652" target="_blank">9811652</a>]<br></li>
<li>( C ) hilD and ompA primer extension products were generated by using total RNA from wild-type ( WT ) , hupA : : kan hupB : : kan ( hup ) , ﬁs3 : : cam ( ﬁs ) , and hns-1 : : kan ( hns ) derivatives of SL1344 or LT2 , grown under inducing or low-osmolarity conditions , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12933899" target="_blank">12933899</a>]<br></li>
<li>SL1344 and ﬁs , rpoS and ﬁs rpoS mutant derivates were grown under non-aerated ( SPI1 inducing ) conditions and used to invade CACO2 ( A ) or CHO ( B ) epithelial cells .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17784910" target="_blank">17784910</a>]<br></li>
<li>However , hilD were not found to be bound by Fis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23717649" target="_blank">23717649</a>]<br></li>
<li>We proposed that hilD was probably indirectly regulated by Fis through other proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23717649" target="_blank">23717649</a>]<br></li>
				<label class="TU_collapse_similarity" for="tu_similarity_14">
d AR class includes genes which are activated by Fis in LT2 , but repressed in K12 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23717649" target="_blank">23717649</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23717649" target="_blank">23717649</a>]<br></label>
<input id="tu_similarity_14", type="checkbox">
<div class="TU_similar">
<li>e RA class includes genes which are repressed by Fis in LT2 , but activated in K12 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23717649" target="_blank">23717649</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23717649" target="_blank">23717649</a>]<br></li></div>
				<label class="TU_collapse_similarity" for="tu_similarity_16">
c BR class includes genes which are repressed by Fis both in LT2 and K12 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23717649" target="_blank">23717649</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23717649" target="_blank">23717649</a>]<br></label>
<input id="tu_similarity_16", type="checkbox">
<div class="TU_similar">
<li>b BA class includes genes which are activated by Fis both in LT2 and K12 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23717649" target="_blank">23717649</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23717649" target="_blank">23717649</a>]<br></li></div>
<li>The spv genes are not regulated by Fis ( Kelly et al. , 2004 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17784910" target="_blank">17784910</a>]<br></li>
<li>a Genes which are both regulated and bound by Fis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23717649" target="_blank">23717649</a>]<br></li>
<li>c Genes which are Fis-binding genes but are not regulated by Fis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23717649" target="_blank">23717649</a>]<br></li>
<li>These results indicated that Fis regulates not only genes to which it binds directly , but also genes downstream of the Fis-binding genes ( Table S8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23717649" target="_blank">23717649</a>]<br></li>
<li>Although Fis was not found to affect genes involved in B12 production , it did repress genes ( btuB , btuC ) contributing to its uptake at the 4 h time point ( Table 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15256548" target="_blank">15256548</a>]<br></li>
<li>Although most genes on SPI were activated by Fis , a small proportion of SPI genes were down-regulated , such as STM2911 and STM2912 in SPI-1 and ttrB in SPI-2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23717649" target="_blank">23717649</a>]<br></li>
<li>The genes that were most strongly up-regulated by Fis were those involved in virulence and located in the pathogenicity islands SPI-1 , SPI-2 , SPI-3 and SPI-5 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15256548" target="_blank">15256548</a>]<br></li>
<li>The pdu operon has also been shown to be regulated by CsrA and Fis , which are both regulators of Salmonella pathogenicity ( 52 , 53 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16684765" target="_blank">16684765</a>]<br></li>
<li>Keane , O.M. , and Dorman , C.J. ( 2003 ) The gyr genes of Salmonella enterica serovar Typhimurium are repressed by the factor for inversion stimulation , Fis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17784910" target="_blank">17784910</a>]<br></li>
<li>Validation of the ChIP-seq results was performed by RT-PCR for 22 genes , including 20 Fis-binding genes and two genes ( carA and hupA ) not bound by Fis as controls .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23717649" target="_blank">23717649</a>]<br></li>
<li>Among the SPI regulators , only hilC and ssrA were both Fis-regulated and Fis-binding genes , indicating that they are regulated by Fis directly .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23717649" target="_blank">23717649</a>]<br></li>
<li>The pocR gene was found to be activated 5.27-fold by Fis and a Fis-binding site was identified within the gene ( Table S3 and S5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23717649" target="_blank">23717649</a>]<br></li>
<li>Expression of the hupA gene showed a strong requirement for Fis at 1 h and has been described previously as being activated by Fis in E. coli ( Claret & Rouvière - Yaniv , 1996 ) ( supplementary data Tables S1 and S2 at http://mic.sgmjournals.org ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15256548" target="_blank">15256548</a>]<br></li>
<li>Other virulence genes regulated by Fis ﬁs mutant/wild-type expression ratio Function Gene 1 h 4 h Chromosomal genes outside pathogenicity islands iagB Cell invasion protein Macrophage-induced gene 0 ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15256548" target="_blank">15256548</a>]<br></li>
<li>Categories of genes regulated by Fis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15256548" target="_blank">15256548</a>]<br></li>
<li>We found that genes within SPI-5 were regulated positively by Fis , with pipC showing the strong-est Fis dependence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15256548" target="_blank">15256548</a>]<br></li>
<li>The aldB gene was repressed by Fis ( Table 4 ) in agreement with previous data from E. coli ( Xu & Johnson , 1995a , b ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15256548" target="_blank">15256548</a>]<br></li>
<li>Fis represses its mRNA levels by approximately 2.5-fold in S. typhimurium and by approximately 10-fold in E. coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7536730" target="_blank">7536730</a>]<br></li>
				<label class="TU_collapse_similarity" for="tu_similarity_36">
aldB , an RpoS-dependent gene in Escherichia coli encoding an aldehyde dehydrogenase that is repressed by Fis and activated by Crp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15256548" target="_blank">15256548</a>]<br></label>
<input id="tu_similarity_36", type="checkbox">
<div class="TU_similar">
<li>aldB , an RpoS-dependent gene in Esche-richia coli encoding an aldehyde dehydrogenase that is repressed by Fis and activated by Crp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8752327" target="_blank">8752327</a>]<br></li></div>
<li>The hin gene , which encodes Hin invertase , flanked by two regions -- hixL and hixR -- is inverted by Hin invertase together with Fis protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22922144" target="_blank">22922144</a>]<br></li>
<li>H-NS regulates its own gene negatively , and this repression is antagonized by Fis , a relationship that is seen at many other H-NS-repressed promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15790293" target="_blank">15790293</a>]<br></li>
<li>In their absence , fis continues to be induced by nutritional upshift and repressed during stationary phase [ 4-6 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24358360" target="_blank">24358360</a>]<br></li>
<li>Activation of Escherichia coli leuV transcription by FIS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15716427" target="_blank">15716427</a>]<br></li>
<li>Control of topA promoters by FIS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21276095" target="_blank">21276095</a>]<br></li>
<li>DNA supercoiling control of fis expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24358360" target="_blank">24358360</a>]<br></li>
<li>FIS control of gyrA , gyrB and topA gene expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21276095" target="_blank">21276095</a>]<br></li>
<li>FIS control of topA expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21276095" target="_blank">21276095</a>]<br></li>
<li>CRP modulates ﬁs transcription by alternate formation of activating and repressing nucleoprotein complexes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15716427" target="_blank">15716427</a>]<br></li>
<li>Nasser W , Schneider R , Travers A , Muskhelishvili G ( 2001 ) CRP modulates fis transcription by alternate formation of activating and repressing nucleoprotein complexes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24358360" target="_blank">24358360</a>]<br></li>
<li>Dps is a nucleoid-associated protein with an expression profile that is the opposite of FIS ; it is absent during exponential phase growth but becomes highly abundant in stationary phase [ 3 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24358360" target="_blank">24358360</a>]<br></li>
<li>However , if cells are cultured in nonaerated conditions , fis expression is sustained during stationary phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24358360" target="_blank">24358360</a>]<br></li>
<li>The interplay between these two regulators is particularly intriguing in conditions that promote fis expression during stationary phase , when RpoS is most active .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24358360" target="_blank">24358360</a>]<br></li>
<li>It is intriguing that PssrA induction does not require FIS in culture conditions that mimic the vacuolar environment [ 12 ] , nor is FIS required for PssrA induction when DNA is highly relaxed [ 20 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22457642" target="_blank">22457642</a>]<br></li>
<li>CRP is a global-acting transcription factor that directly regulates FIS expression in E. coli [ 4 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24358360" target="_blank">24358360</a>]<br></li>
<li>Figure 4 Model showing that CRP , Fis and PhoPQ regulate taiA-hlyE operon expression via RpoS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24885225" target="_blank">24885225</a>]<br></li>
<li>Alternatively , Fis could repress the gene ( s ) shown negatively to affect hilA expression : phoP/Q ( Behlau and Miller , 1993 ; Pegues et al. , 1995 ) and hilE , ams and pag ( Fahlen et al. , 2000 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11123690" target="_blank">11123690</a>]<br></li>
<li>In E. coli , FIS binds the gyrA and gyrB promoters to repress their expression ( Schneider et al. , 1999 ) ; similarly , FIS has been found to repress S. enterica gyrA and gyrB ( Keane and Dorman , 2003 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21276095" target="_blank">21276095</a>]<br></li>
<li>Thus , the hup and ﬁs mutations may decrease hilA expression ( i ) by decreasing its derepression by HilD and/or ( ii ) by increasing its repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12933899" target="_blank">12933899</a>]<br></li>
<li>We suspected that small nucleoid-binding proteins , such as H-NS , HU , and Fis , might control hilA expression by affecting its repression and/or derepression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12933899" target="_blank">12933899</a>]<br></li>
<li>In Figure S11 and Figure S12 , we showed that nucleoid proteins Fis and HU also independently control both hilA and hilD transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22021388" target="_blank">22021388</a>]<br></li>
<li>Hirsch and Elliott showed that fis represses rpoS expression by direct binding in S. Typhimurium [ 19 ] , supporting ou observations .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24885225" target="_blank">24885225</a>]<br></li>
<li>This chromosome-based reporter differs from the study by Ó Cróinín and Dorman that first identified sustained fis expression [ 13 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24358360" target="_blank">24358360</a>]<br></li>
<li>These results suggest that changes in DNA supercoiling during non-aerated growth may be involved in sustained fis expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24358360" target="_blank">24358360</a>]<br></li>
<li>Thus a simple model predicts that fis will be expressed in oxygen-limited conditions , and our results suggest that DNA supercoiling may be an important contributor to sustained fis expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24358360" target="_blank">24358360</a>]<br></li>
<li>Promoter truncation also revealed that the shortest functional fis promoter was incapable of sustained expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24358360" target="_blank">24358360</a>]<br></li>
<li>fis expression was slightly elevated in aerated conditions ( Figure 3C ) , which was not detected using the chromosomal system -- again indicating that the plasmid-borne system enhances or exaggerates fis promoter activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24358360" target="_blank">24358360</a>]<br></li>
<li>In non-aerated conditions , sustained fis expression was lower in Pfis ( − 198 ) and was even further reduced in Pfis ( − 49 ) ( Figure 3B ) , indicating that one or more regulatory elements upstream contribute to sustained expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24358360" target="_blank">24358360</a>]<br></li>
<li>Transcription from Pfis-2 was up-regulated two-fold in non-aerated stationary phase cultures , but this accounted for only 7 ± 1 % of total fis transcripts , indicating that Pfis-2 is not responsible for sustained fis expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24358360" target="_blank">24358360</a>]<br></li>
<li>The expression of fis in intracellular environments may be explained in part by the recent finding that S. Typhimurium fis expression is elevated in stationary phase if oxygen availability is reduced , a condition referred to as `` sustained expression '' because the fis promoter remains active in stationary phase [ 13 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24358360" target="_blank">24358360</a>]<br></li>
<li>Quantitative PCR confirmed that the dusB-fis-gfpTCD fusion is a reliable reporter of fis expression in non-aerated conditions ( Figure S1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24358360" target="_blank">24358360</a>]<br></li>
<li>It has been stated that Fis regulates gene expression by increasing/decreasing its level in E. coli [ 20 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24885225" target="_blank">24885225</a>]<br></li>
<li>It has been reported previously that Fis has no effect on stpA gene expression in E. coli at 30 min following subinoculation ( Free & Dorman , 1997 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15256548" target="_blank">15256548</a>]<br></li>
<li>For E. coli cspA , it was proposed that Fis regulated th expression of cspA at 37 ° C ( 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11544220" target="_blank">11544220</a>]<br></li>
<li>Mallik P , Paul BJ , Rutherford ST , Gourse RL , Osuna R ( 2006 ) DksA is required for growth phase-dependent regulation , growth rate-dependent control , and stringent control of fis expression in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24358360" target="_blank">24358360</a>]<br></li>
<li>These results suggest that Fis may a ¡ ect gene expression in stationary phase even though Fis concentration is low in stationary phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14553938" target="_blank">14553938</a>]<br></li>
				<label class="TU_collapse_similarity" for="tu_similarity_74">
Xu , J. , and Johnson , R.C. ( 1995 ) Identiﬁcation of genes negatively regulated by Fis : Fis and RpoS comodulate growth-phase-dependent gene expression in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16999831" target="_blank">16999831</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17784910" target="_blank">17784910</a>]<br></label>
<input id="tu_similarity_74", type="checkbox">
<div class="TU_similar">
<li>Xu J , Johnson RC ( 1995 ) Identiﬁcation of genes negatively regulated by Fis : Fis and RpoS comodulate growth-phase-dependent gene expression in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14502413" target="_blank">14502413</a>]<br></li>
<li>Identiﬁcation of genes negatively regulated by Fis : Fis and RpoS comodulate growth-phase-dependent gene expression in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8752327" target="_blank">8752327</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7536730" target="_blank">7536730</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17293430" target="_blank">17293430</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15256548" target="_blank">15256548</a>]<br></li></div>
<li>Xu , J. , and Johnson , R.C. ( 1995 ) Fis activates the RpoSdependent stationary-phase expression of proP in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11123690" target="_blank">11123690</a>]<br></li>
<li>Fis activates the RpoS-dependent station-ary-phase expression of proP in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8752327" target="_blank">8752327</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15716427" target="_blank">15716427</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16332813" target="_blank">16332813</a>]<br></li>
<li>In particular , the genes required for cytochrome-o-oxidase , succinyl-CoA synthetase , glutamate/aspartate transport , trehalose-6-phosphate biosynthesis , long-chain-fatty-acids transport , spermidine/putrescine transport , dipeptide transport , the genes encoding the two-component tricarboxylic transport system and the site-specific DNA factor for inversion stimulation ( fis ) were among the highest repressed by ArcA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21418628" target="_blank">21418628</a>]<br></li>
<li>These primers generate a 1.68 kb DNA fragment containing the orf1 ± fis operon ( Osuna et al. , 1995 ) under the control of its own promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11123690" target="_blank">11123690</a>]<br></li>
<li>It appears that cloning of the complete ﬁs operon ( including its own promoter ) on a high-copy-number plasmid results in cell toxicity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7536730" target="_blank">7536730</a>]<br></li>
<li>Nucleotide sequence of the ﬁs operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7536730" target="_blank">7536730</a>]<br></li>
<li>These results demonstrated that this pattern of ﬁs expression without aeration was strain-speciﬁc and not species-speciﬁc .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17784910" target="_blank">17784910</a>]<br></li>
<li>A number of positive regulators of hilA expression have been identified including csrAB , sirA/barA , pstS , hilC/sirC/sprA , fis , and hilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15765064" target="_blank">15765064</a>]<br></li>
<li>A number of positive regulators ( hilC , hilD , ﬁs , sirA/barA , csrAB , phoBR , fadD , envZ/ompR , and ﬂiZ ) and negative regulators ( hha , hilE , lon , ams , phoPc and pag ) have been identiﬁed that are able to alter expression of hilA transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15731035" target="_blank">15731035</a>]<br></li>
<li>A variety of genes ( hilC , hilD , ﬁs , sirA/barA , csrAB , phoB , fadD , envZ/ompR , ﬂiZ , hilE , ams , lon , pag , and hha ) have been identiﬁed that exert positive or negative effects on hilA expression , although the mechanisms by which these gene products function remain relatively unclear .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12595445" target="_blank">12595445</a>]<br></li>
<li>Considering that crp and fis deletions seem to produce additive effects in hlyE transcription ( Figure 1A ) , these two repressors probably act through different pathways to repress the hlyE expression ( Figure 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24885225" target="_blank">24885225</a>]<br></li>
<li>Transcription of fis declines dramatically as cells enter stationary phase , and the FIS protein concentration drops to undetectable levels [ 1-3 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24358360" target="_blank">24358360</a>]<br></li>
<li>It may be that during stationary phase RpoS is able to gain access to and initiate transcription at the otherwise RpoDdriven fis promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24358360" target="_blank">24358360</a>]<br></li>
<li>This effect on ndk expression may reﬂect yet another route through which the Fis protein autoregulates ﬁs gene expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15256548" target="_blank">15256548</a>]<br></li>
<li>Like PssrA , PhilC required both FIS and OmpR-EnvZ for induction ; yet unlike PssrA , the absence of FIS was not compensated by increasing concentrations of novobiocin .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22457642" target="_blank">22457642</a>]<br></li>
<li>Fis is required for activation of ssrA expression in murine macrophages through DNA relaxation [ 56 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20861532" target="_blank">20861532</a>]<br></li>
<li>Effect of HU and Fis on hilA expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12933899" target="_blank">12933899</a>]<br></li>
<li>This expression is associated with a downregulation of RpoS and an upregulation of Fis expression under these conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17784910" target="_blank">17784910</a>]<br></li>
<li>These results indicate that Fis is essential for full activation of hilA : : Tn5lacZY expression from both plasmid and chromosomally located reporters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11123690" target="_blank">11123690</a>]<br></li>
<li>Because HilD is required for hilA expression ( Schechter et al. , 1999 ) , and current models suggest that HilD is also required for invF expression independently of hilA ( Lucas and Lee , 2000 ) , our data are consistent with a model in which hilD is induced by Fis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11123690" target="_blank">11123690</a>]<br></li>
<li>None of the above mentioned genes ( ﬁs , hha , hilA ) exhibited differential WT = DluxS expression at early stationary phase ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19909098" target="_blank">19909098</a>]<br></li>
<li>However , fis does directly regulate genes in this cluster but does it through expression of ssrAB operon according to our SPI-2 pathway .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20861532" target="_blank">20861532</a>]<br></li>
<li>It appears that genes in cluster 2A are more under fis controlled than ssrAB control perhaps because their expression is more sensitive to DNA relaxation than the expression of genes in cluster 2B which appear under stringent ssrAB control .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20861532" target="_blank">20861532</a>]<br></li>
<li>Effects of transcription factor mutations on fis expression in aerated and non-aerated conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24358360" target="_blank">24358360</a>]<br></li>
<li>The absence of an explicit RpoS effect in aerated , non-aerated , and anaerobic conditions ( Figures 2E , 4C & D ) , suggests a model in which RpoS represses fis expression only when RpoS protein levels are unusually high , as in a ΔarcA mutant .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24358360" target="_blank">24358360</a>]<br></li>
<li>This presence of Fis in cultures grown without aeration was shown to correlate with an increase in invasion of epithelial cells , conﬁrming that these were conditions under which SPI1 genes were induced ( Fig. 1B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17784910" target="_blank">17784910</a>]<br></li>
<li>However , a clear increase in ﬁs transcription levels under low-aeration conditions in stationary phase was observed in both the wild-type and the rpoS mutant strains but absent in both strains when grown under aerating conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17784910" target="_blank">17784910</a>]<br></li>
<li>These are exempli cents ed by the comparison of FljB , FliC , SipA , SseC , and SopE2 expression in wild-type and cents s mutant strains at exponential and stationary phase of growth , in which we could not cents nd much di ¡ erence in their expression patterns ( Fig. 2 ) : proteins positively regulated by Fis in stationary phase showed a similar positive regulation in exponential phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14553938" target="_blank">14553938</a>]<br></li>
<li>E. coli topA was upregulated in the absence of FIS in both exponential growth and stationary phase , whereas S. enterica topA expression was unaffected in Dﬁs cells during exponential growth but was elevated in stationary phase relative to wild-type cells .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21276095" target="_blank">21276095</a>]<br></li>
<li>Thus , expression of ﬁs in these clones is attributed to transcription originating in the plasmid .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7536730" target="_blank">7536730</a>]<br></li>
<li>The promoter region for the ﬁs operon in E. coli ( 4 , 43 ) , which is located in the DNA sequence upstream of ORF1 , is completely conserved in S. typhimurium from 249 to 194 relative to the start of transcription in E. coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7536730" target="_blank">7536730</a>]<br></li>
<li>While it may be tempting to speculate that repression of cadA transcription by Fis may represent a step in the expression of virulence in S. typhimurium , it is not known if lysine decarboxylase activity plays any role in S. typhimurium virulence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15256548" target="_blank">15256548</a>]<br></li>
<li>These results indicate that Fis is also involved in the regulation of invF expression in a HilA-independent manner .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11123690" target="_blank">11123690</a>]<br></li>
<li>Figure 1 Fis participates in down-regulation of hlyE expression in S. Typhi .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24885225" target="_blank">24885225</a>]<br></li>
<li>The repressive effect of Fis on hupB expression described in E. coli ( Claret & Rouvière - Yaniv , 1996 ) was not detected under the conditions used in our study .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15256548" target="_blank">15256548</a>]<br></li>
<li>It is known that the expression of fis sharply increases on nutritional up-shift and that promoter activity is autorepressed by Fis in Escherichia coli ( Ball et al. 1992 ; Nilsson et al. 1992 ; Ninnemann et al. 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15235764" target="_blank">15235764</a>]<br></li>
<li>The ﬁs gene is repressed by the stringent response and is negatively auto-regulated by the Fis protein ( Ninnemann et al. 1992 ; Walker et al. 1999 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12898222" target="_blank">12898222</a>]<br></li>
<li>Transcription of the ﬁs gene is inﬂuenced by the stringent response , is autoregulated by Fis protein and is controlled by the intracellular concentration of cytosine triphosphate ( Ball et al. , 1992 ; Walker et al. , 1999 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15256548" target="_blank">15256548</a>]<br></li>
<li>Although topA promoter function has routinely been studied using plasmid-based systems ( Tse-Dinh , 1985 ; Tse-Dinh and Beran , 1988 ; Lesley et al. , 1990 ; Marshall et al. , 2000 ; Weinstein-Fischer and Altuvia , 2007 ) , our results indicate that in E. coli the expression of plasmid-borne PtopAEc is enhanced by FIS whereas the chromosomal copy of the promoter is repressed by FIS ( compare Figs 3A and 4A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21276095" target="_blank">21276095</a>]<br></li>
<li>This potential may be abrogated by other factors , such as the relaxation of DNA supercoiling that accompanies the transition into stationary phase ( Dorman et al. , 1988 ; Ó Cróinín et al. , 2006 ) and which is known to inﬂuence the transcription of the ﬁs gene ( Schneider et al. , 2000 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17784910" target="_blank">17784910</a>]<br></li>
<li>Fis is also required for oriC-directed DNA replication and inﬂuences the topolo-gical state of DNA in the cell by repressing DNA gyrase and activating topoisomerase I gene expression ( Gonzalez-Gil et al. , 1996 ; Ross et al. , 1990 ; Schneider et al. , 1999 ; Weinstein-Fischer et al. , 2000 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15256548" target="_blank">15256548</a>]<br></li>
<li>Transcription of the ﬁs gene in strains SL1344 and CSH50 Fis protein expression is controlled at the level of ﬁs transcription ( Osuna et al. , 1995 ; Walker et al. , 1999 ) and by a post-transcriptional mechanism ( Owens et al. , 2004 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17784910" target="_blank">17784910</a>]<br></li>
<li>Recently , it was reported that CspA in E. coli was induced during early exponential phase ( Brandi et al. 1999 ; Yamanaka and Inouye 2001 ) and that high concentrations of its transcriptional activator Fis positively regulated its expression both in vivo and in vitro ( Brandi et al. 1999 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15235764" target="_blank">15235764</a>]<br></li>
<li>Results Invasion of epithelial cells and Fis expression during growth with or without aeration The genes in the SPI1 pathogenicity island of S. Typhimurium are known to be Fis-dependent for expression ( Wilson et al. , 2001 ; Schechter et al. , 2003 ; Kelly et al. , 2004 ) , and previous data indicated that these genes are induced when bacteria are grown to late exponential growth phase without aeration ( Lee and Falkow , 1990 ; Song et al. , 2004 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17784910" target="_blank">17784910</a>]<br></li>
<li>Fis represses transcription of the gyrA and gyrB genes in E. coli ( Schneider et al. 1999 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12898222" target="_blank">12898222</a>]<br></li>
<li>Genes encoding ﬂagella [ ﬂi and ﬂg genes , ( Barak et al. , 2005 ; Jeter and Matthysse , 2005 ; Torres et al. , 2005 ) ] were not found to be more transcribed in presence of sprouts suggesting a more important role for Fis in regulation of the T3SS in our experiment .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24308841" target="_blank">24308841</a>]<br></li>
<li>Only 145 common genes are bound by Fis in both K12 and LT2 , which is possibly due to the difference in their genome sequence , as 320 of the 943 Fis-binding genes in LT2 are not present in K12 [ 24 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23717649" target="_blank">23717649</a>]<br></li>
<li>Changes in SPI expression suggest the possibility that regulation of genes in SPI-2 as well as SPI-1 is affected by Fis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14553938" target="_blank">14553938</a>]<br></li>
<li>Fis is involved in SPI expression Positive regulation of SPI-1 genes by Fis was expected considering the fact that hilA , a transcriptional regulator of SPI-1 , is upregulated by Fis [ 16 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14553938" target="_blank">14553938</a>]<br></li>
<li>It is known that the 28 Fis-dependent TFs regulate 303 of those 419 genes ( Regulon DB ) ; thus , it is highly likely that Fis controls the transcription of these 303 genes by regulating corresponding TFs .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23717649" target="_blank">23717649</a>]<br></li>
<li>It is apparent that the majority of genes regulated by Fis are associated with virulence and motility/chemotaxis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15256548" target="_blank">15256548</a>]<br></li>
<li>In this work , we found that glucose induces the fis expression in S. Typhi , suggesting that Fis is also regulated by its level in S. enterica ( Figure 1E ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24885225" target="_blank">24885225</a>]<br></li>
<li>These results suggest that Fis from S. typhimurium influences the expression of cspH by binding to its promoter , although a direct interaction between Fis and the cspH promoter was not shown in this study .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15235764" target="_blank">15235764</a>]<br></li>
<li>Altogether , these results suggest that CRP and Fis are able to repress the rpoS expression by direct binding , thereby regulating hlyE expression in S. Typhi .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24885225" target="_blank">24885225</a>]<br></li>
				<label class="TU_collapse_similarity" for="tu_similarity_131">
J Bacteriol 174:8043 -- 8056 Beach MB , Osuna R ( 1998 ) Identification and characterization of the fis operon in enteric bacteria .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15235764" target="_blank">15235764</a>]<br></label>
<input id="tu_similarity_131", type="checkbox">
<div class="TU_similar">
<li>Beach MB , Osuna R ( 1998 ) Identification and characterization of the fis operon in enteric bacteria .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24358360" target="_blank">24358360</a>]<br></li></div>
<li>DNA supercoiling and fis expression were measured in parallel cultures containing either pUC18 or the Pfis ( -298 ) reporter plasmid , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24358360" target="_blank">24358360</a>]<br></li>
<li>Up-regulation of the S. Typhimurium fis promoter in the Δfis mutant confirmed that FIS represses its own promoter , and this repression occurs in both aerated and non-aerated conditions ( Figure 3C & D ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24358360" target="_blank">24358360</a>]<br></li>
<li>Transcription was moderately reduced at both Pfis-1 and Pfis-2 in the ΔarcA mutant ( Figure 4B ) , suggesting that ArcA causes mild activation of fis expression during exponential growth .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24358360" target="_blank">24358360</a>]<br></li>
<li>To verify the microarray results , expression levels of several representative genes including ﬁs , ﬂiC , rplB , purB , acs and poxB were also investigated by RT-PCR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19659726" target="_blank">19659726</a>]<br></li>
<li>The expression of sodC , a superoxide dismutase , and ﬁs , a global regulator , was also divergent between the species ( Tables 2 and 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19759044" target="_blank">19759044</a>]<br></li>
<li>Expression of the regulatory genes ﬁs and soxS was also divergent .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19759044" target="_blank">19759044</a>]<br></li>
<li>Strain H2114 was grown at 37uC to OD600 = 0.4 , and Fis expression was induced by the addition of 0.1 mM isopropyl-b-D-thiogalactopyranoside ( IPTG ) for 3 h at 30uC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23717649" target="_blank">23717649</a>]<br></li>
<li>For example , OmpR promotes SPI-2 expression by directly binding to both the ssrA and ssrB promoters [ 16 ] ; PhoP activates SPI-2 transcription by directly binding to the ssrB promoter and controls SsrA levels post-transcriptionally [ 15 ] ; SlyA , HilD and Fis proteins can bind to the ssrA promoter to regulate SPI-2 genes [ 19,20,22 ] , and PmrA represses SPI-2 gene transcription through directly binding to the ssrB promoter [ 18 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29751061" target="_blank">29751061</a>]<br></li>
<li>Fis and virulence gene expression Among the most strongly Fis-activated genes were the virulence genes located within the SPI pathogenicity islands ( see Fig. 2 and Fig. 3 ) and the chemotaxis / ﬂagellar regulons ( Fig. 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15256548" target="_blank">15256548</a>]<br></li>
<li>Using a genetic approach , Fis was shown to be necessary for the induction of hilA and invF expression , genes that encode two positive regulators of several SPI-1 genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11123690" target="_blank">11123690</a>]<br></li>
<li>Three regulatory modes were characterized by which Fis controls SPI gene transcription : i ) Fis binds to and activates SPI regulator genes ( hilC and ssrA ) ; ii ) Fis binds directly to SPI genes to enhance the transcription of these genes and those downstream ; iii ) Fis enhances the expression of the global regulator gene ompR [ 39 ] , which induces the expression of SPI positive regulator genes ( hilD and ssrA ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23717649" target="_blank">23717649</a>]<br></li>
<li>In comparisons of the expression profiles of LT2 wild-type and Dfis , 63 of the 94 SPI-1 and SPI-2 genes were found to be regulated by Fis , including 55 Fis up-regulated genes and eight Fis down-regulated genes ( Figure S1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23717649" target="_blank">23717649</a>]<br></li>
<li>The regulators CsrA , Fis and H-NS were all shown to induce the expression of many SPI1 , motility and che-motaxis-related genes [ 19 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16413221" target="_blank">16413221</a>]<br></li>
<li>Fis seem to accelerate relaxation and stabilizes the promoter topology of ssrAB to allow timely expression of Spi-2 genes ( O Croinin et al. , 2006 ; Dillon and Dorman , 2010 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26441883" target="_blank">26441883</a>]<br></li>
<li>In E. coli , FIS represses its own promoter by binding the FIS-I and FIS-II binding sites that flank the transcription start site [ 4,6 ] , and these sites are perfectly conserved between the E. coli and S. Typhimurium fis promoters ( Figure 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24358360" target="_blank">24358360</a>]<br></li>
<li>Regarding growth-phase-dependent expression , it has been reported that the E. coli rapA and fis genes , whose expression is dependent upon the growth phase , usually have a cytosine ( C ) residue as a transcriptional start site , and that these genes also have a discriminator sequence between positions -- 10 and +1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15235764" target="_blank">15235764</a>]<br></li>
<li>Both the discriminator and transcriptional start sites are important for the growth-phase-dependent expression of rapA and fis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15235764" target="_blank">15235764</a>]<br></li>
<li>RpoS , PhoP , CRP and Fis may be important for the proper hlyE expression inside macrophages , since the RpoS and PhoP respective regulons are activated inside these cells ; and both crp and fis expression is repressed in the same environment [ 41,43 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24885225" target="_blank">24885225</a>]<br></li>
<li>The expression of fis and rpoS is negatively correlated , the former being elevated in exponential growth and the latter in stationary phase , therefore it is unsurprising that FIS and RpoS are antagonistic at some gene promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24358360" target="_blank">24358360</a>]<br></li>
<li>The intimate relationship between ﬁs and ssrA gene expression is also illustrated by the striking similarity that exists between their expression patterns when S. Typhimurium is growing in macrophage ( Fig. 4A and B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16999831" target="_blank">16999831</a>]<br></li>
<li>We cloned , mapped , and sequenced the S. typhimurium ﬁs gene and upstream DNA sequences and show here that the deduced Fis amino acid sequence is identical to that in E. coli and that the overall operon organization is conserved .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7536730" target="_blank">7536730</a>]<br></li>
<li>Transductional crosses established that the ﬁs gene is located between the envB and aroE genes ( between min 70.5 and 71.1 ) , as depicted in Fig. 3 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7536730" target="_blank">7536730</a>]<br></li>
<li>ArcA and FNR regulate dozens and hundreds of genes , respectively , many of which are required for metabolism in anaerobic conditions , but neither transcription factor has been implicated in controlling fis expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24358360" target="_blank">24358360</a>]<br></li>
<li>Quantitative PCR ( qPCR ) analysis was used to measure gyrA , gyrB , topA and ﬁs gene expression in cells cultured in 60 ml of LB ( 0.5 % NaCl ) in ﬂasks ; these conditions are identical to those used in Fig. 1A and F , in which E. coli demonstrated a larger dynamic range of supercoiling changes and a more dramatic response to deletion of the ﬁs gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21276095" target="_blank">21276095</a>]<br></li>
				<label class="TU_collapse_similarity" for="tu_similarity_157">
Expression of the ssrA and ﬁs genes in macrophage and epithelial cells The expression of gfp reporter fusions to the promoters of the ssrA and ﬁs genes was monitored in bacteria growing in J774A .1 macrophage and in CACO 2 epithelial cells .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16999831" target="_blank">16999831</a>]<br></label>
<input id="tu_similarity_157", type="checkbox">
<div class="TU_similar">
<li>Expression of the ﬁs and ssrA genes during growth of S. Typhimurium in macrophage and epithelial cells .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16999831" target="_blank">16999831</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16999831" target="_blank">16999831</a>]<br></li></div>
<li>Expression of both viaB and pil was , however , optimal at 100 mM NaCl , and mutations in envZ/ompR , rcsB/rcsC , ( but not rcsA ) , tviA , ihfB or ﬁs affected transcription of both viaB and pil DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16842356" target="_blank">16842356</a>]<br></li>
<li>S. Typhimurium strain SL1344 and E. coli strain MC4100 showed comparable levels of RpoS expression , which were signiﬁcantly less than those observed in CSH50 ( Fig. 3C ) , and both strains showed comparable levels of ﬁs transcription without aeration ( compare Figs. 2A and 3B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17784910" target="_blank">17784910</a>]<br></li>
<li>Whether the differential regulation 60 of both ﬁs and soxS is a direct response to triclosan exposure or an indirect result of altered expression of higher order regulators is not known .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19759044" target="_blank">19759044</a>]<br></li>
<li>This also identiﬁed a previously unrecognized osmoregulation of topA expression that is independent of FIS and supercoiling in both E. coli and S. enterica .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21276095" target="_blank">21276095</a>]<br></li>
<li>Transcription of E. coli topA is driven by the interplay of at least four transcription start sites , allowing FIS to be both an activator and a repressor of E. coli topA expression ( WeinsteinFischer and Altuvia , 2007 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21276095" target="_blank">21276095</a>]<br></li>
<li>The factor for inversion stimulation ( FIS ) is of particular interest because it plays diverse roles in the control of DNA supercoiling and gene expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21276095" target="_blank">21276095</a>]<br></li>
<li>Falconi , M. , Brandi , A. , La Teana , A. , Gualerzi , C.O. , and Pon , C.L. ( 1996 ) Antagonistic involvement of Fis and H-NS proteins in the transcriptional control of hns expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11123690" target="_blank">11123690</a>]<br></li>
<li>Antagonistic involvement of FIS and H-NS proteins in the transcriptional control of hns expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9811652" target="_blank">9811652</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15256548" target="_blank">15256548</a>]<br></li>
<li>HilA was found to be downregulated ( twofold ) at mid-log phase which could be due to the repression of ﬁs and expression of hha ( Table 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19909098" target="_blank">19909098</a>]<br></li>
<li>The expression of ﬁs , which encodes a DNA binding protein known to positively regulate expression of hilA and invF ( 25 ) , was increased in L110 ( acrB : : aph ) only .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19411325" target="_blank">19411325</a>]<br></li>
<li>Fis somewhat increased the expression of cspH upon nutritional up-shift .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15235764" target="_blank">15235764</a>]<br></li>
<li>Half of the same mRNA preparations were used to measure bla transcription as an internal control ( lower panel ) Fig. 4 Fis affects the expression of cspH upon nutritional up-shift .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15235764" target="_blank">15235764</a>]<br></li>
<li>PhilD was unique among the four promoters in having higher expression in the absence of ompR and envZ , but it nevertheless required FIS for full activation ( Figure 1B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22457642" target="_blank">22457642</a>]<br></li>
<li>Thus in S. enterica , FIS enhances the expression of both PtopAEc and PtopASe during exponential growth , but reduces the expression of both promoters in stationary phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21276095" target="_blank">21276095</a>]<br></li>
<li>The exponential phase expression of PtopAEc was slightly elevated in the absence of rpoS , perhaps due to stimulation by elevated FIS levels .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21276095" target="_blank">21276095</a>]<br></li>
<li>PtopASe maintained wild-type expression during exponential growth in the absence of rpoS ( Fig. 4C ) , indicating that the higher concentration of FIS was insufficient to repress PtopASe .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21276095" target="_blank">21276095</a>]<br></li>
<li>In contrast , endogenous PtopASe expression was induced equally in the presence or absence of FIS ( Fig. 6B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21276095" target="_blank">21276095</a>]<br></li>
<li>As shown in Fig. 1 , the expression of hilA : : Tn5lacZY in the fis : : kan mutant was not induced throughout the growth cycle compared with the fis strain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11123690" target="_blank">11123690</a>]<br></li>
<li>Plasmid pFis349 , but not the parent plasmid , was able to restore hilA : : Tn5lacZY expression in the fis : : kan mutant ( Table 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11123690" target="_blank">11123690</a>]<br></li>
<li>During periods of rapid growth in laboratory conditions the fis promoter is highly expressed , which accounts for the high levels of FIS protein during exponential growth .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24358360" target="_blank">24358360</a>]<br></li>
<li>Genetic analysis revealed that neither the stringent-response-associated alarmone ppGpp nor the DksA protein were essential for the sustained expression of fis in stationary phase [ 21 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24358360" target="_blank">24358360</a>]<br></li>
<li>Guadarrama-Beltran S ( 2013 ) Sustained expression of fis , the gene coding for the Fis nucleoid-associated protein , during the stationary phase of growth in Salmonella enterica .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24358360" target="_blank">24358360</a>]<br></li>
<li>The reduced intensity of sustained expression in the chromosome gfp fusion compared to the plasmid-borne reporter indicates that the plasmid system is a reliable but exaggerated reporter of sustained fis expression in nonaerated conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24358360" target="_blank">24358360</a>]<br></li>
<li>Thus , the effect of the ΔarcA mutation on fis expression is not due to altered growth in non-aerated conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24358360" target="_blank">24358360</a>]<br></li>
<li>Since a Tn5 insertion in this region disrupts the positive regulation of hupB by crp-cAMP as well as the negative regulation by FIS we currently do not know if the transposon insertion increases or decreases normal expression of hupB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10767604" target="_blank">10767604</a>]<br></li>
<li>At higher concentrations of arabinose inducer , Fis protein expression reached a maximum ( A ) , SsrB was undetectable ( B ) , the pUC18 reporter plasmid became highly relaxed ( C ) , and ssrA transcription was higher than the repressed control but not as high as the level seen with 0.2 % arabinose ( D ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16999831" target="_blank">16999831</a>]<br></li>
<li>Expression of the ﬁs gene was more easily induced under low-aeration conditions , conﬁrming that other factors play a role in this low-aeration induction of the Fis protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17784910" target="_blank">17784910</a>]<br></li>
<li>b The expression of cspH -- lacZ is dependent on the growth phase in strain YH1 ( squares ) , strain YHDF ( Fis + strain containing the pYDF ) [ ( diamonds ) ] and strain YHF ( Fis -- strain containing pYH1 ) [ ( triangles ) ] at 37 ° C .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15235764" target="_blank">15235764</a>]<br></li>
<li>E. coli strain CSH50 had the highest levels of RpoS protein under these conditions ( Fig. 3C ) and showed no expression of Fis at the onset of stationary phase ( Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17784910" target="_blank">17784910</a>]<br></li>
<li>The burst of Fis expression that is induced during the recovery of cells from stationary phase as they enter the logarithmic growth phase might play a role in negatively regulating stationary phase-speciﬁc functions ( 61 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7536730" target="_blank">7536730</a>]<br></li>
<li>The same ﬁs mutant had an extended lag phase in a rich growth medium and the viability of the bacterium was compromised by constitutive expression of Fis during stationary phase ( Osuna et al. , 1995 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15256548" target="_blank">15256548</a>]<br></li>
<li>Growth phase determines the expression levels of several regulators ( e.g. , Fis is maximally expressed at 1 h of growth , and IHF peaks during the transition into the stationary phase [ 301 , 305 ] ) and inﬂuences its regulatory action ( e.g. , HilD induces SPI-2 genes during the late stationary phase [ 14 ] ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23554419" target="_blank">23554419</a>]<br></li>
<li>The factor-for-inversion stimulation protein ( Fis ) is known to play a pivotal role in the expression of Salmonella pathogenicity island ( SPI ) -1 genes in addition to various cellular processes such as recombination , replication , and transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14553938" target="_blank">14553938</a>]<br></li>
<li>The decreased DNA gyrase activity inhibited cspH transcription , and the accumulation of Fis upon nutrient up-shift in part regulated its expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15235764" target="_blank">15235764</a>]<br></li>
<li>As shown in Figs. 2 , and 4 , cspH expression in the Fis mutant was partially repressed , but the addition of gyrase inhibitor almost completely blocked expression upon nutrient up-shift .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15235764" target="_blank">15235764</a>]<br></li>
<li>Results are the averages from three independent experiments Fis affects the expression of cspH Fis , which is induced in the early exponential phase or upon nutrient up-shift , is a pleiotropic regulator in E. coli ( Drlica and Rouviere-Yaniv 1987 ; Finkel and Johnson 1992 ; McLeod et al. 2002 ; Schmid 1990 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15235764" target="_blank">15235764</a>]<br></li>
<li>Therefore , to investigate whether Fis affects the expression of cspH mRNA after the adaptation to nutritional up-shift , mRNA levels of cspH were measured in both Fis -- and Fis + strains at each growth phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15235764" target="_blank">15235764</a>]<br></li>
<li>c The expression of cspH mRNA is dependent on the growth phase in Fis + and Fis -- strains .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15235764" target="_blank">15235764</a>]<br></li>
<li>Fis affects expression of invF : : Tn5lacZY in the absence of HilA Current models for the regulation of invasion genes suggest that there is a HilA-independent pathway to invF expression that involves HilD ( Lucas and Lee , 2000 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11123690" target="_blank">11123690</a>]<br></li>
<li>The reason that the expression of cob genes in pocR deleted strains is still dependent of Fis effect is Fis may also control the expression of B12 biosynthesis genes through other unknown negative regulators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23717649" target="_blank">23717649</a>]<br></li>
<li>The result showed that Fis activates the transcription of B12 biosynthesis genes mainly through controlling the expression of pocR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23717649" target="_blank">23717649</a>]<br></li>
<li>This also suggests that at least some of the functions played by Fis ( and perhaps also the product of ORF1 ) in these bacteria are related to its growth phase-dependent expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9811652" target="_blank">9811652</a>]<br></li>
<li>In order to understand the role of Fis in virulence gene expression of S. typhimurium , we analyzed proteins that are di ¡ erentially expressed in the presence and absence of cents s and detected 11 possible proteins upregulated by Fis and seven proteins downregulated by Fis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14553938" target="_blank">14553938</a>]<br></li>
<li>In addition , we implicate the alternate stress response sigma factor RpoS as playing a role in repressing Fis expression , and show that induction of Fis in the absence of aeration is facilitated by a reduction of RpoS levels under these conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17784910" target="_blank">17784910</a>]<br></li>
<li>This anti-correlation in the levels of the Fis and RpoS proteins recalled an earlier observation that Fis is a repressor of rpoS gene transcription , at least in exponential-phase cultures ( Hirsch and Elliott , 2005 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17784910" target="_blank">17784910</a>]<br></li>
<li>[ 33,34 ] Although the absence of Fis , a key regulator in S. Typhimurium virulence , up-regulates the expression of cadA and cadB , [ 35 ] it is still not known how cadA is involved in Salmonella pathogenicity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20390946" target="_blank">20390946</a>]<br></li>
<li>Cróinín TÓ , Dorman CJ ( 2007 ) Expression of the Fis protein is sustained in late-exponential - and stationary-phase cultures of Salmonella enterica serovar Typhimurium grown in the absence of aeration .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24358360" target="_blank">24358360</a>]<br></li>
<li>Expression of ﬂagellar gene fusions in the presence and absence of Fis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15256548" target="_blank">15256548</a>]<br></li>
<li>b-Galactosidase assays were used to measure expression of lacZ in strains harbouring fusions to a selection of ﬂagellar genes in the presence and absence of the Fis protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15256548" target="_blank">15256548</a>]<br></li>
<li>The involvement of Fis in negative regulation of ndk was of interest given its similar role in the expression of the nupG and rbsC nucleoside transport genes , suggesting that Fis coordinates the expression of genes involved in pyrimidine metabolism .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15256548" target="_blank">15256548</a>]<br></li>
<li>In addition , it has been proved that CRP-cAMP modulates fis expression and the inhibition of DNA gyrase represses the expression of several CRP-cAMP sensitive genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25033662" target="_blank">25033662</a>]<br></li>
<li>Consistent with the idea that HU and Fis affect the derepression of hilA , our results show that hilA-lacZ expression is not signiﬁcantly reduced by the hupAB or ﬁs mutations in a hilCD or hilD mutant strain background ( Fig. 1DF ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12933899" target="_blank">12933899</a>]<br></li>
<li>Of the small nucleoid-binding proteins that affect hilA expression , H-NS acts as a repressor , but Fis and HU activate hilA by changing the expression of hilD ( 32 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16391141" target="_blank">16391141</a>]<br></li>
<li>These results indicate that the autoregulation of tufB gene expression is uncoupled from tRNA production and show that it is a secondary regulatory mechanism specific for EF-TuB , and independent from transcriptional initiation of the operon and its regulation by Fis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26934594" target="_blank">26934594</a>]<br></li>
<li>Expression of hilA is increased by the pleiotropic activator proteins OmpR , PhoPQ , and the Factor for Inversion Stimulation , Fis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16443238" target="_blank">16443238</a>]<br></li>
<li>In E. coli ( 55 ) and H. inﬂuenzae , the gene encoding the ribosomal protein L11 methyltransferase ( prmA ) is located upstream of the ﬁs promoter region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9811652" target="_blank">9811652</a>]<br></li>
<li>On deletion of the ﬁs gene , diminished hilA transcription , invasion , and hilA-independent invF expression ( 304 ) are observed , as well as a marked reduction in the hilD mRNA levels ( 291 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23554419" target="_blank">23554419</a>]<br></li>
<li>However , other factors are clearly involved because the absence of RpoS in the aerated culture did not lead to the same levels of Fis expression as that observed without aeration : the non-aerated Sl1344 rpoS culture expressed at least twice as much Fis protein as its aerated counterpart ( Fig. 4C , lanes 5 and 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17784910" target="_blank">17784910</a>]<br></li>
<li>These contained transposon insertions in the ihfB and ﬁs genes ( one strain of each ) , and we proceeded to show that mutations in these genes negatively affected the transcription of both the viaB gene cluster and pil .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16842356" target="_blank">16842356</a>]<br></li>
<li>A negative correlation between ﬁs expression and the status of the rpoS gene was apparent in each strain we examined ( Figs 3 and 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17784910" target="_blank">17784910</a>]<br></li>
<li>Our results indicate that the level of hilD mRNA is reduced in the hupA hupB and ﬁs mutants and increased in the hns mutant ( Fig. 2C ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12933899" target="_blank">12933899</a>]<br></li>
<li>Using the topA promoter from both species , we show that a combination of different DNA superhelical densities and the presence of the FIS NAP modulate the expression proﬁle of the topA gene at the level of transcription such that it responds differently in E. coli and S. enterica to salt stress , aeration and the presence of an ami-nocoumarin antibiotic , when these parameters are varied individually or in combination .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21276095" target="_blank">21276095</a>]<br></li>
<li>To conﬁrm that these results were speciﬁc for the ﬁs promoter and not a more general observation of transcription in the Salmonella background , the transcription of the rpsM gene was monitored in the SL1344 background and was found to be unaffected by the degree of aeration in stationary phase ( Fig. 2E ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17784910" target="_blank">17784910</a>]<br></li>
<li>23S rRNAs were included as loading control Discussion In the present report , we have shown that the expression of cspH , encoding one of the cold-shock proteins in S. typhimurium , is induced upon nutrient up-shift at 37 ° C , and that both DNA gyrase and Fis affect expression of the gene under this condition .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15235764" target="_blank">15235764</a>]<br></li>
<li>Because hilA expression is also induced during mid-log to late-log growth ( Lee and Falkow , 1990 ; Bajaj et al. , 1996 ) , we tested the effect of Fis on hilA regulation and found that a mutation in the S. typhimurium fis gene results in a large negative impact on the expression of both hilA and invF and a severe virulence defect in the mouse model of S. typhimurium pathogenesis by the oral , but not the peritoneal , route of infection .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11123690" target="_blank">11123690</a>]<br></li>
<li>We then overexpressed ompR in Dfis , and found that the increased transcription of the SPI-1 and SPI-2 genes was recovered ( 1.04 to 6.36-fold increase ) , thus partly compensating for the effect caused by the fis deletion ( Figure 6C ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23717649" target="_blank">23717649</a>]<br></li>
<li>We examined the relative contributions made by other protein transcription factors and DNA supercoiling to the expression of S. Typhimurium fis in the non-aerated growth conditions that up-regulate the fis promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24358360" target="_blank">24358360</a>]<br></li>
<li>As a negative control , a DNA probe derived from the ORF of the spvR virulence regulatory gene of S. enterica , whose expression is not inﬂuenced by Fis ( our unpublished data ) , was also incubated with Fis protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12898222" target="_blank">12898222</a>]<br></li>
<li>The absence of the Fis protein in SL1344ﬁs : : cat was conﬁrmed by Western blotting ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15256548" target="_blank">15256548</a>]<br></li>
<li>Fis repressed transcription of ndk , the gene that encodes nucleoside diphosphate kinase ( Table 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15256548" target="_blank">15256548</a>]<br></li>
<li>Fis is expressed in the intracellular niche , so why does it not repress transcription of the rpoS gene , preventing RpoS from activating spv transcription ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17784910" target="_blank">17784910</a>]<br></li>
<li>Regulators Altogether , eight regulatory genes were more transcribed in presence of sprouts , with ﬁs ( ) being the regulatory gene with highest fold expression ratio ( 38-fold ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24308841" target="_blank">24308841</a>]<br></li>
<li>Also downregulated in the ﬁs mutant was the lipoprotein gene lpp ( Table 4 ) which affects ﬂagellar assembly ( Dailey & Macnab , 2002 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15256548" target="_blank">15256548</a>]<br></li>
<li>Expression of b-galactosidase encoded by pLS31 was 18-fold higher ( 1506 Miller units ) in the wildtype strain compared with the fis : : kan mutant ( 82 Miller units ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11123690" target="_blank">11123690</a>]<br></li>
<li>Expression of the FliC protein was measured by Western blotting in wild-type strain SL1344 and its ﬁs knockout derivative , SL1344ﬁs : : cat following 4 h growth in LB at 37 6C .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15256548" target="_blank">15256548</a>]<br></li>
<li>The ninefold elevation of rpoS transcription in the ﬁs mutant was completely eliminated by pFis349 ( Fig. 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15716427" target="_blank">15716427</a>]<br></li>
<li>A similar increase in transcription was seen when ﬁs expression from the chromsome was monitored by RT-PCR from RNA isolated at 9 h growth ( C ) ; no difference was detected in the transcription of the control gene rpsM ( which encodes a ribosomal protein ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17784910" target="_blank">17784910</a>]<br></li>
<li>Reduced viability in stationary phase by constitutively expressed ﬁs has also been observed in E. coli ( 11 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7536730" target="_blank">7536730</a>]<br></li>
<li>Relative positions in DNA regions corresponding to ﬁs , ORF1 , prmA , and cah are represented by open rectangles ( not to scale ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9811652" target="_blank">9811652</a>]<br></li>
<li>( b ) Mutations in ﬁs and ihfB affect pil transcription when bacteria are grown under stress .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16842356" target="_blank">16842356</a>]<br></li>
<li>Because ﬁs is the second gene in its transcriptional unit in both E. coli and S. enterica , ﬁs lacks its own ribosome binding site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21276095" target="_blank">21276095</a>]<br></li>
				</div></ul></div><br>
	</div>
</body>
</html>