<!doctype html>
<style>
@charset "utf-8";
body {
	height: 100%;
	margin: 0;
	padding: 0;
}

.banner {
	position: fixed;
	top: 0;
	width: 100%;
	height: 65px;
	background: gray;
	line-height: 65px;
}
.spacer {
	height: 10px;
}

.main-content {
	padding: 5%;
}

.collapse_ACT {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-style: none solid none none;
	border-color: #B4045F;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_ACT + input{
	display: none;
}
.collapse_ACT + input + div{
	display: none;
}
.collapse_ACT + input:checked + div{
	display: block;
}
.ACT {
	text-align: justify;
	text-justify: inter-word;
}
.ACT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B4045F;
}
.ACT_collapse_similarity:hover{
	background: #e5d3dc;
}
.ACT_collapse_similarity + input {
	display: none;
}
.ACT_collapse_similarity + input + div{
	display: none;
}

.ACT_collapse_similarity + input:checked + div{
	display: block;
}
.ACT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.ACT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B4045F;
	padding-left: 5px;
}
.ACT_collapse_similarity:hover{
	background: #e5d3dc;
}
.ACT_collapse_similarity + input {
	display: none;
}
.ACT_collapse_similarity + input + div{
	display: none;
}
.ACT_collapse_similarity + input:checked + div{
	display: block;
}
.ACT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.ACT_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #B4045F;
}
.ACT_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_DOM {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-style: none solid none none;
	border-color: #B404AE;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_DOM + input{
	display: none;
}
.collapse_DOM + input + div{
	display: none;
}
.collapse_DOM + input:checked + div{
	display: block;
}
.DOM {
	text-align: justify;
	text-justify: inter-word;
}
.DOM_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B404AE;
	padding-left: 5px;
}
.DOM_collapse_similarity:hover{
	background: #e8d7e7;
}
.DOM_collapse_similarity + input {
	display: none;
}
.DOM_collapse_similarity + input + div{
	display: none;
}
.DOM_collapse_similarity + input:checked + div{
	display: block;
}
.DOM_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}.DOM_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #B404AE;
}
.DOM_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_EVO {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #7401DF;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_EVO + input{
	display: none;
}
.collapse_EVO + input + div{
	display: none;
}
.collapse_EVO + input:checked + div{
	display: block;
}
.EVO {
	text-align: justify;
	text-justify: inter-word;
}
.EVO_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #7401DF;
	padding-left: 5px;
}
.EVO_collapse_similarity:hover{
	background: #e2d7ed;
}
.EVO_collapse_similarity + input {
	display: none;
}
.EVO_collapse_similarity + input + div{
	display: none;
}
.EVO_collapse_similarity + input:checked + div{
	display: block;
}
.EVO_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.EVO_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #7401DF;
}
.EVO_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_RP {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #0174DF;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_RP + input{
	display: none;
}
.collapse_RP + input + div{
	display: none;
}
.collapse_RP + input:checked + div{
	display: block;
}
.RP {
	text-align: justify;
	text-justify: inter-word;
}
.RP_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #0174DF;
	padding-left: 5px;
}
.RP_collapse_similarity:hover{
	background: #c9d7e2;
}
.RP_collapse_similarity + input {
	display: none;
}
.RP_collapse_similarity + input + div{
	display: none;
}
.RP_collapse_similarity + input:checked + div{
	display: block;
}
.RP_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.RP_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #0174DF;
}
.RP_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_SIT {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #0489B1;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_SIT + input{
	display: none;
}
.collapse_SIT + input + div{
	display: none;
}
.collapse_SIT + input:checked + div{
	display: block;
}
.SIT {
	text-align: justify;
	text-justify: inter-word;
}
.SIT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #0489B1;
	padding-left: 5px;
}
.SIT_collapse_similarity:hover{
	background: #d0dfe2;
}
.SIT_collapse_similarity + input {
	display: none;
}
.SIT_collapse_similarity + input + div{
	display: none;
}
.SIT_collapse_similarity + input:checked + div{
	display: block;
}
.SIT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.SIT_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #0489B1;
}
.SIT_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_TU {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #04B486;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_TU + input{
	display: none;
}
.collapse_TU + input + div{
	display: none;
}
.collapse_TU + input:checked + div{
	display: block;
}
.TU {
	text-align: justify;
	text-justify: inter-word;
}
.TU_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #04B486;
	padding-left: 5px;
}
.TU_collapse_similarity:hover{
	background: #d0e2de;
}
.TU_collapse_similarity + input {
	display: none;
}
.TU_collapse_similarity + input + div{
	display: none;
}
.TU_collapse_similarity + input:checked + div{
	display: block;
}
.TU_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.TU_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #04B486;
}
.TU_similar > a {
	text-decoration: none;
	color: gray;
}

ul { 
	list-style-type: none;
	padding: 0;
	margin: 0;
}
ul li > a {
	text-decoration: none;
	color: gray;
}
ul li:hover, ul li:focus {
	background: #F0F0F0;
}

ul li:first-child span {
  border-left:none;  
}

</style>
<html>
<head>
<meta charset="utf-8">
<title>TF properties - FlhC</title>
<meta name="theme-color" content="#ffffff">
</head>
<body>
	<div class="banner">
		<center><font color="white" size="16px" face="arial">FlhC</font></center>
	</div>
	<div align="center" class="main-content">
		<div class="spacer">
	</div><br>			<label class="collapse_ACT" for="_1"><font size="5px">FlhC transcriptional activity</font></label>
				<input id="_1" type="checkbox">
				<div class="ACT"><br><ul><li>- DnaK - ATP - DnaK - ATP + + + + ¬ ¬ M + - + + - + M 25 26 27 28 29 30 31 32 33 34 DnaK FlhC FlhC FlhD FlhD Fig. 8 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16430704" target="_blank">16430704</a>]<br></li>
<li>However , Salmonella deficient in FlhC expression and lacking flagella failed to activate NF - B-de-pendent luciferase expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17477819" target="_blank">17477819</a>]<br></li>
<li>The FlhC and FlhD proteins form a heterotetrameric complex , FlhC2FlhD2 , that direct σ70-RNA polymerase complex to activate transcription from class 2 promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16629666" target="_blank">16629666</a>]<br></li>
<li>This indicates that FliT binds speciﬁcally to FlhC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16952964" target="_blank">16952964</a>]<br></li>
<li>This protein binds to FlhD4C2 through interaction with the FlhC subunit ( 71 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21278297" target="_blank">21278297</a>]<br></li>
<li>FliT acts as an anti-FlhD4C2 factor , which binds to the FlhD4C2 complex through interaction with the FlhC subunit and inhibits its binding to the class * Corresponding author .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21804009" target="_blank">21804009</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_7">
We conclude that FliT acts as an anti-FlhD2C2 factor , which binds to FlhD2C2 through interaction with the FlhC subunit and inhibits its binding to the class 2 promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16952964" target="_blank">16952964</a>]<br></label>
<input id="act_similarity_7", type="checkbox">
<div class="ACT_similar">
<li>FliT acts as an anti-FlhD2C2 factor , which binds to FlhD2C2 through interaction with the FlhC subunit and inhibits its binding to class 2 promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16952964" target="_blank">16952964</a>]<br></li></div>
<li>Therefore , FliT binding to FlhC may directly inhibit the DNA-binding activity of FlhD4C2 , while YdiV binding to FlhD may indirectly affect FlhD4C2 binding to DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21278297" target="_blank">21278297</a>]<br></li>
<li>However , because FliT can bind to FlhC that does not form FlhD2C2 complex ( Fig. 3B , lane 9 ) , we can not exclude a possibility that FliT binds to FlhC monomer or dimer to inhibit formation of the FlhC2 or FlhD2C2 complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16952964" target="_blank">16952964</a>]<br></li>
<li>Importantly , all alanine substitutions still retained the ability to-bind to FlhC to some extent and therefore inhibited wild-type motility ( Fig. 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21143315" target="_blank">21143315</a>]<br></li>
<li>As a result this circuit is sensitive to the availability of FliT to-bind FliD , FliT to-bind FlhC and the availability of FlhD4C2 to-bind DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21143315" target="_blank">21143315</a>]<br></li>
<li>Thus , both the kinetic pattern of ydiV repression by FliZ ( figure 3A ) and FlhC regulation by FliZ ( figure 3B ) support our fliC transcription timecourse , in which FliZ increases fliC expression at later timepoints during growth ( figure 2A-E ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25315056" target="_blank">25315056</a>]<br></li>
<li>FliZ is responsible for increased levels of FlhC protein independent of ﬂhDC transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18469103" target="_blank">18469103</a>]<br></li>
<li>This allowed us to perform half-life determinations of the FlhC protein under almost physiological conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12675803" target="_blank">12675803</a>]<br></li>
<li>Another anti-FlhD4C2 factor is the flagellar-related protein FliT , which interacts with free FlhC subunits preventing activation of flagellar class 2 promoter transcription and leading to ClpXP-dependent proteolysis ( Yamamoto and Kutsukake , 2006 ; Aldridge et al. , 2010 ; Sato et al. , 2014 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27206164" target="_blank">27206164</a>]<br></li>
<li>FliT is known to inhibit FlhD4C2-binding to DNA through its direct interaction with FlhC ( Yamamoto and Kutsukake , 2006 ) , while it also interacts with the ﬁlament-capping protein FliD and two soluble proteins , FliI and FliJ , of the ﬂagellar-speciﬁc type 3 secretion ( T3S ) apparatus ( Fraser et al. , 1999 ; Evans et al. , 2006 ; Imada et al. , 2010 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21143315" target="_blank">21143315</a>]<br></li>
<li>As a regulator of flagellar expression , FliT binds to FlhC as part of the FlhD4C2 complex and inhibits its activity ( Hung et al. , 2012 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27404739" target="_blank">27404739</a>]<br></li>
<li>FliT negatively regulates FlhDC activity by inhibiting promoter binding of the FlhDC complex and increases ClpXP-dependent proteolysis of the FlhC subunit ( Yamamoto and Kutsukake , 2006 ; Sato et al. , 2014 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26441883" target="_blank">26441883</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_20">
Claret , L. , and Hughes , C. ( 2000b ) Rapid turnover of FlhD and FlhC , the ﬂagellar regulon transcriptional-activator proteins , during Proteus swarming .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12675803" target="_blank">12675803</a>]<br></label>
<input id="act_similarity_20", type="checkbox">
<div class="ACT_similar">
<li>[ 36 ] Claret , L. and Hughes , C. ( 2000 ) Rapid turnover of FlhD and FlhC , the # agellar regulon transcriptional-activator proteins , during Proteus swarming .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11731137" target="_blank">11731137</a>]<br></li></div>
<li>In accordance with our hypothesis , the stability of epitope-tagged FlhC protein after arrest of de novo protein synthesis was increased in a ΔrfaG ΔrﬂP double mutant compared to the ΔrfaG single mutant ( Fig. 2C ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29717015" target="_blank">29717015</a>]<br></li>
<li>To show a direct interaction of ClpXP ATP-dependent protease and the FlhD/FlhC regulator proteins , biochemical analysis in vitro with puriﬁed proteins is now in progress .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11790733" target="_blank">11790733</a>]<br></li>
<li>Additionally , three proteins , STM1344 ( also named YdiV or RflP ) , STM1697 and FliT , were recently identified as negative regulators of flagellum in Salmonella by direct protein -- protein interactions : FliT binds to the FlhC subunit , and STM1344 and STM1697 interact with the FlhD subunit ( 17 -- 22 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28973452" target="_blank">28973452</a>]<br></li>
<li>Interestingly FlhD and FlhC , which are negatively regulated by RcsB , have been shown to regulate a number of E. coli genes involved in anaerobic respiration ( 71 ) , but dcuB and narH were not on the list .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17616593" target="_blank">17616593</a>]<br></li>
<li>It is known that FlhD and FlhC act together in an FlhD2C2 heterotetramer to activate transcription of the ﬂagellar regulon ( Liu and Matsumura , 1994 ; Clart and Hughes , 2000a ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12675803" target="_blank">12675803</a>]<br></li>
<li>Its products , FlhD and FlhC , act together in a FlhD/FlhC hetero-oligomer to activate the promoter of class 2 genes ( 28 , 38 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18245288" target="_blank">18245288</a>]<br></li>
<li>B A + ∆ ∆ + + ∆ DnaK ClpP ∆ ∆ DnaK + ClpP + M ∆ + ∆ + ∆ + FlhC FlhD FliC The DnaK chaperone machinery activates the FlhD2C2 complex into a functional transcriptional-regulator The immunoblotting analyses in Fig. 4 show no accumulation of FliC in the ∆ dnaK ∆ clpP cells but signiﬁcant accumulation of what appears to be the FlhD2C2 complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16430704" target="_blank">16430704</a>]<br></li>
<li>When FlhD and FlhC proteins were synthesized together , accumulation of both proteins by the depletion of ClpXP protease was conﬁrmed ( lane 2 in Fig 4B or 4C ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12675803" target="_blank">12675803</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_30">
Bacterial culture of strain CS2637 grown overnight was diluted 1:100 into fresh L-broth and followed by the induction of FlhD and FlhC expression by the addition of 20 mM IPTG for 2 h. Cells were collected , lysed and run in lane 5 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12675803" target="_blank">12675803</a>]<br></label>
<input id="act_similarity_30", type="checkbox">
<div class="ACT_similar">
<li>Bacterial culture of strain CS2637 grown overnight was diluted 1:100 into fresh L-broth and followed by the induction of FlhD and FlhC expression by the addition of 40 mM IPTG for 2 h. Cells were collected , lysed and run in lane 8 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12675803" target="_blank">12675803</a>]<br></li></div>
<li>Strains lacking FlhC production did not bind to cholesterol under any of the conditions tested , further suggesting that the serovar Typhimurium ﬂagellar ﬁlament mediates binding to cholesterol in the early stages of bioﬁlm formation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20118264" target="_blank">20118264</a>]<br></li>
<li>In this system , FlhC protein in amounts sufﬁcient enough to precipitate as an immunocomplex was produced by read-through from the PA1lac0-1 promoter even in the absence of IPTG .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12675803" target="_blank">12675803</a>]<br></li>
<li>If this were the sole mechanism by which FliZ increased FlhC , the amount of FlhC should be identical between ΔydiV and ΔydiV ΔfliZ .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25315056" target="_blank">25315056</a>]<br></li>
<li>As shown in Figure 3B , the levels of FlhC and FlhD were increased in ΔclpXP cells compared to wild type .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21689395" target="_blank">21689395</a>]<br></li>
<li>In the dnaK + cells , the β-galactosi-dase activity increased with increasing FlhD and FlhC levels .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16430704" target="_blank">16430704</a>]<br></li>
<li>In contrast , in the presence of FlhD4/His10-FlhC 2 , both RNA I and the ﬂiA transcript were produced ( Fig. 6A , lane 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21278297" target="_blank">21278297</a>]<br></li>
<li>However , activation of the hilA gene by FliZ occurs even in the absence of FlhD/FlhC [ 13 , our unpublished results ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11162188" target="_blank">11162188</a>]<br></li>
<li>Prüss and Matsumura also demonstrated that FlhD in the absence of FlhC is a regulator of cell division ( 103 ) , and this regulation occurs through induction of the acid response gene cadA , encodong lysine decarboxylase ( 101 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11104815" target="_blank">11104815</a>]<br></li>
<li>¯ hDC ÿ denotes the ¯ hD92 : : Tn10 allele ; in the absence of the FlhD and FlhC proteins , s28 expression is greatly reduced but not eliminated ( in a ¯ gM ÿ ¯ hDC background , the ¯ iC promoter is saturated , and basal s28 activities can not be measured ; see Materials and Methods for details ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11237608" target="_blank">11237608</a>]<br></li>
<li>Our previous ﬁnding that it negatively regulates the expression of the ﬂagellar regulon by controlling the turnover of the master regulator , FlhD/FlhC ( 62 ) , allowed us to hypothesize that it may regulate HilC and HilD by controlling ﬂagellar gene expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18245288" target="_blank">18245288</a>]<br></li>
<li>Our present results suggest that the FlhC and FlhD when are in FlhD2FlhC2 complex are targeted by the ClpXP protease to negatively regulate the transcription of the ﬂagellar regulon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12675803" target="_blank">12675803</a>]<br></li>
<li>We hypothesized that ClpXP may regulate HilC and HilD through the control of the ﬂagellar regulon , since ClpXP has been shown to regulate its expression negatively by degrading the FlhD/FlhC master regulator , which functions at the apex of the transcription hierarchy of the regulon ( 61 , 62 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18245288" target="_blank">18245288</a>]<br></li>
<li>FliZ activates the class 2 ﬂagellar genes only in the presence of the ﬂa-gellar activator proteins , FlhD/FlhC [ 16 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11162188" target="_blank">11162188</a>]<br></li>
<li>Levels of master regulators , FlhD and FlhC , in the ClpXP protease-depleted mutant Our previous study showed that the level of bgalactosidase activity in cells carrying the ﬂhD-lac fusion Turnover of FlhDC by ClpXP protease 445 was not changed , in contrast to the marked increase of bgalactosidase levels expressed from the ﬂiC-lac fusion , by mutations in either clpX or clpP ( Tomoyasu et al. , 2002 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12675803" target="_blank">12675803</a>]<br></li>
<li>Alternatively , folding , assembling and/or activation of FlhD and FlhC might occur independently on the DnaK chaperone machinery , though at a very low rate because the β-galactosidase activity in the ﬂiA -- lac fusion increased only slightly in the ∆ dnaK cells .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16430704" target="_blank">16430704</a>]<br></li>
<li>In contrast , in the ∆ dnaK cells , the β-galactosidase activity was signiﬁcantly lower even after 3 h of incubation with IPTG , when the FlhD and FlhC proteins reached levels similar to those in the dnaK + cells .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16430704" target="_blank">16430704</a>]<br></li>
<li>Therefore , the slower increasing of FlhD and FlhC proteins in the ∆ dnaK cells could be due to the degradation of both monomers by the increased level of protease ( s ) such as ClpAP , HslVU and Lon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16430704" target="_blank">16430704</a>]<br></li>
<li>The most convincing evidence indicating the importance of DnaK for the activation of FlhD2C2 complex is as follows ; in the wild-type cells , the levels of FlhD and FlhC decreased with time while in the ∆ dnaK ∆ clpP cells the levels of both proteins stayed constantly high , but this is contrasted by the transcriptional regulating activity of FhlD and FlhC , which was high in the wild-type cells but mark-edly low in the ∆ dnaK ∆ clpP cells .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16430704" target="_blank">16430704</a>]<br></li>
				</div></ul><br>
			<label class="collapse_DOM" for="_2"><font size="5px">FlhC protein domains</font></label>
				<input id="_2" type="checkbox">
				<div class="DOM"><br><ul><li>This protein binds to FlhD4C2 through interaction with the FlhC subunit ( 71 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21278297" target="_blank">21278297</a>]<br></li>
<li>In the FlhD2C2 complex , the FlhC subunit bears a DNA-binding activity , while the FlhD * Corresponding author .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16952964" target="_blank">16952964</a>]<br></li>
				<label class="DOM_collapse_similarity" for="dom_similarity_3">
In the FlhD4C2 complex , the FlhC subunit exhibits a DNA-binding activity , while the FlhD subunit strengthens its DNA-binding speciﬁcity and stabilizes the protein-DNA complex ( 10 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21278297" target="_blank">21278297</a>]<br></label>
<input id="dom_similarity_3", type="checkbox">
<div class="DOM_similar">
<li>The FlhC subunit has DNA-binding activity , while the FlhD subunit strengthens its DNA-binding speciﬁcity and stabilizes the protein-DNA complex ( 10 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21278297" target="_blank">21278297</a>]<br></li></div>
				</div></ul><br>
			<label class="collapse_EVO" for="_3"><font size="5px">FlhC evolution</font></label>
				<input id="_3" type="checkbox">
				<div class="EVO"><br><ul><li>Our results suggest that FlhC and FlhD belong to the small group of central regulators of cell physiology known as timing proteins ( Gottesman and Maurizi , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12675803" target="_blank">12675803</a>]<br></li>
<li>The ﬂhDC operon is the sole one belonging to class 1 and encodes FlhD and FlhC , which assemble into an FlhD4C2 heterohexamer acting as an activator of class 2 operons ( 31 , 42 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21804009" target="_blank">21804009</a>]<br></li>
				</div></ul><br>
			<label class="collapse_RP" for="_4"><font size="5px">Processes regulated by FlhC</font></label>
				<input id="_4" type="checkbox">
				<div class="RP"><br><ul><li>- ) Flagellar motor rotation protein MotB Flagellar motor rotation protein MotA Flagellar transcriptional activator FlhC Flagellar transcriptional activator FlhD Cystine ABC transporter , periplasmic cystine-binding protein FliY Flagellar biosynthesis protein FliZ RNA polymerase sigma factor for flagellar operon Lysine-N-methylase ( EC 2.1.1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26056383" target="_blank">26056383</a>]<br></li>
<li>Immunoblotting for detection of FlhC protein in various pro-tease-depleted mutants .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12675803" target="_blank">12675803</a>]<br></li>
<li>FlhC is a master regulator protein involved in flagellar biogenesis in Salmonella [ 65 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22632036" target="_blank">22632036</a>]<br></li>
<li>Interestingly , genes encoding the central regulators HilA , SsrB , and FlhC were shown to be repressed in strains unable to express YfgL .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17060472" target="_blank">17060472</a>]<br></li>
<li>ﬂhB 0.26 0.25 0.23 Flagellar biosynthesis protein FlhB ﬂhC 0.61 0.51 0.52 0.40 0.41 0.34 Transcriptional activator FlhC ﬂhD Transcriptional activator FlhD Flagellar biosynthesis sigma factor Lysine-N-methylase A subset of genes encoding phosphotransferase components , involved in the uptake of carbohydrate , was also induced ( ptsH , ptsA ( frwA ) , frwBC , treB ) in EWMM , as were several genes specifying glycolysis and pentose-phosphate-pathway enzymes ( gapA , gpmA , pykA , fbaB , pfkB , talC , rpiA , tktA ) , by 1.99 - to 5.62-fold at 45 min ( Table 11 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28553268" target="_blank">28553268</a>]<br></li>
<li>The ability of FlhC or FlhD to regulate other pathways to include those involved in respiration and cell-division may offer an explanation about the inconsistent phenotype imparted by these mutations ( 71 -- 73 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29247060" target="_blank">29247060</a>]<br></li>
<li>Interestingly FlhD and FlhC , which are negatively regulated by RcsB , have been shown to regulate a number of E. coli genes involved in anaerobic-respiration ( 71 ) , but dcuB and narH were not on the list .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17616593" target="_blank">17616593</a>]<br></li>
<li>Although it has been described that SdiA affects motility and the transcription of multiple genes involved in flagella biosynthesis ( Kanamaru et al. 2000 ; Wei et al. 2001b ) , flhC was one of the few genes unaffected by over-expressed SdiA ( Wei et al. 2001b ) , which suggest that FlhC operates independently of SdiA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12115054" target="_blank">12115054</a>]<br></li>
<li>Gene array analysis of Yersinia enterocolitica FlhD and FlhC : regulation of enzymes affecting synthesis and degradation of carbamoylphosphate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26441883" target="_blank">26441883</a>]<br></li>
<li>Prüss and Matsumura also demonstrated that FlhD in the absence of FlhC is a regulator of cell-division ( 103 ) , and this regulation occurs through induction of the acid response gene cadA , encodong lysine decarboxylase ( 101 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11104815" target="_blank">11104815</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_11">
FlhD/FlhC is a regulator of anaerobic-respiration and the entnerdoudoroff pathway through induction of the methyl-accepting chemotaxis protein Aer .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28553268" target="_blank">28553268</a>]<br></label>
<input id="rp_similarity_11", type="checkbox">
<div class="RP_similar">
<li>FlhD/FlhC is a regulator of anaerobic-respiration and the Entner-Doudoroff pathway through induction of the methyl-accepting che-motaxis protein Aer .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17616593" target="_blank">17616593</a>]<br></li></div>
				<label class="RP_collapse_similarity" for="rp_similarity_13">
Prüß BM , Campbell JW , Van Dyk TK , Zhu C , Kogan Y , Matsumura P. FlhD/FlhC is a regulator of anaerobic-respiration and the Entner-Doudoroff pathway through induction of the methyl-accepting chemotaxis protein Aer .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18486950" target="_blank">18486950</a>]<br></label>
<input id="rp_similarity_13", type="checkbox">
<div class="RP_similar">
<li>Pruss , B.M. , Campbell , J.W. , Van Dyk , T.K. , Zhu , C. , Kogan , Y. , and Matsumura , P. ( 2003 ) FlhD/FlhC is a regulator of anaerobic-respiration and the Entner -- Duodoroff pathway through induction of the methyl-accepting chemotaxis protein Aer .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12753201" target="_blank">12753201</a>]<br></li></div>
<li>This phenotype could be related to the transcriptional down-regu-lation of genes encoding proteins involved in the biosynthesis of all Salmonella type-III-secretion-systems , including the central regulators HilA , SsrB , and FlhC , in a yfgL mutant .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17060472" target="_blank">17060472</a>]<br></li>
<li>The possible role of Salmonella flagellin in the induction of type I and type II regulated ISGs in J774A .1 macrophages was assessed using a non-flagellated Salmonella strain containing a mutation in the gene encoding the global regulator of flagellin production , flhC .19 The Salmonella mutant deficient in FlhC expression was as efficient as WT Salmonella in activating iNOS ( Fig. 4A ) and Mx ( Fig. 4B ) in macrophage cells , as measured by Northern hybridization .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17477819" target="_blank">17477819</a>]<br></li>
<li>The ﬂiAZY operon requires the master ﬂagellar gene regulators , FlhD and FlhC , for expression ( 44 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11292791" target="_blank">11292791</a>]<br></li>
<li>In DdnaK cells , FlhD and FlhC proteins seemed to be assembled into hetero-tetrameric FlhD2C2 but the complex was not fully active in class 2 gene transcription , suggesting that the DnaK chaperone is involved in activating native FlhD2C2 complex into a regulator of ﬂagellar regulon expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16430704" target="_blank">16430704</a>]<br></li>
<li>Turnover of FlhD and FlhC , master regulator proteins for Salmonella flagellum-biogenesis , by the ATP-dependent ClpXP protease .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23676436" target="_blank">23676436</a>]<br></li>
<li>To test the hypothesis that secreted effector ( s ) responsible for invasion of the Salmonella SPI-1 / 2 double mutant into 3-D cells is a result of the flagellum pore ; we infected cells with a Salmonella mutant deficient in SPI-1 , SPI-2 , and the master regulators of flagellum pore-formation and synthesis , FlhC and FlhD ( Figure 2E ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21206750" target="_blank">21206750</a>]<br></li>
<li>FlhD is a component of the FlhDFlhC master regulator , essential for the transcription of genes in the ﬂagellar regulon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20802043" target="_blank">20802043</a>]<br></li>
<li>We demonstrate that DnaK regulates the expression of the Sal-monella ﬂagellar regulon by modulating the FlhD and FlhC proteins , which function as master regulators at the apex of a transcription hierarchy comprising three classes of genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16430704" target="_blank">16430704</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_23">
Proteome analysis of proteins secreted from the mutant cells deﬁcient for ﬂhDC and clpXP genes demonstrated that the ﬂhD mutation abolished the enhanced effect by clpXP mutation on the production of ﬂagellar proteins , suggesting that the ClpXP possibly deﬁnes a regulatory pathway affecting the expression of ﬂagellar regulon that is dependent on FlhD/FlhC master regulators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11790733" target="_blank">11790733</a>]<br></label>
<input id="rp_similarity_23", type="checkbox">
<div class="RP_similar">
<li>As shown in Fig. 4 , proteome analysis of the secreted proteins from the mutant cells deﬁcient for clpP and ﬂhD / ﬂhC genes demonstrated that the ﬂhD / ﬂhC mutation completely abolished the enhanced effect of the clpP mutation on the production of the ﬂagellar proteins , suggesting that the ClpXP possibly works via a pathway dependent on the FlhD/FlhC master regulators Rapid turnover of FlhD and FlhC by protease during P. mirabilis swarming was recently demonstrated ( 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11790733" target="_blank">11790733</a>]<br></li></div>
<li>Class 1 promoters drive expression of the ﬂagellar master operon that encodes the transcription factors FlhD and FlhC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26441919" target="_blank">26441919</a>]<br></li>
<li>Our previous ﬁnding that it negatively regulates the expression of the ﬂagellar regulon by controlling the turnover of the master regulator , FlhD/FlhC ( 62 ) , allowed us to hypothesize that it may regulate HilC and HilD by controlling ﬂagellar gene-expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18245288" target="_blank">18245288</a>]<br></li>
				</div></ul><br>
			<label class="collapse_SIT" for="_5"><font size="5px">FlhC binding site</font></label>
				<input id="_5" type="checkbox">
				<div class="SIT"><br><ul><li>This indicates that FliT binds speciﬁcally to FlhC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16952964" target="_blank">16952964</a>]<br></li>
<li>By analogy with this , we can assume that FlhD and FlhC may act as activator proteins which assist FliA-RNA polymerase in recognizing this putative -10 sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8206838" target="_blank">8206838</a>]<br></li>
				</ul></div><br>
			<label class="collapse_TU" for="_6"><font size="5px">Transcription units regulated by FlhC</font></label>
				<input id="_6" type="checkbox">
				<div class="TU"><br><ul><li>The class 3 operons were found to be transcribed in this condition , indicating that these operons should be directly regulated by FliA but neither by FlhD nor by FlhC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8206838" target="_blank">8206838</a>]<br></li>
<li>Thus , both the kinetic pattern of ydiV repression by FliZ ( figure 3A ) and FlhC regulation by FliZ ( figure 3B ) support our fliC transcription timecourse , in which FliZ increases fliC expression at later timepoints during growth ( figure 2A-E ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25315056" target="_blank">25315056</a>]<br></li>
<li>FlhD/FlhC regulated promoters analyzed by gene array and lacZ gene fusions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21278297" target="_blank">21278297</a>]<br></li>
<li>The operon encoding FlhD and FlhC has been reported to be regulated by multiple regulators , many of which have not been identiﬁed yet ( Kutsukake , 1997 ; Teplitski et al. , 2003 ; Clarke and Sperandio , 2005 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29473025" target="_blank">29473025</a>]<br></li>
<li>Interestingly FlhD and FlhC , which are negatively regulated by RcsB , have been shown to regulate a number of E. coli genes involved in anaerobic respiration ( 71 ) , but dcuB and narH were not on the list .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17616593" target="_blank">17616593</a>]<br></li>
<li>Although it has been described that SdiA affects motility and the transcription of multiple genes involved in flagella biosynthesis ( Kanamaru et al. 2000 ; Wei et al. 2001b ) , flhC was one of the few genes unaffected by over-expressed SdiA ( Wei et al. 2001b ) , which suggest that FlhC operates independently of SdiA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12115054" target="_blank">12115054</a>]<br></li>
<li>The class 2 operons are positively controlled by activator proteins , FlhD and FlhC , which are encoded by the class 1 genes ( 18 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8550529" target="_blank">8550529</a>]<br></li>
<li>Prüss and Matsumura also demonstrated that FlhD in the absence of FlhC is a regulator of cell division ( 103 ) , and this regulation occurs through induction of the acid response gene cadA , encodong lysine decarboxylase ( 101 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11104815" target="_blank">11104815</a>]<br></li>
<li>Initiation of the transcription of ﬂiA-fused lacZ gene by FlhD and FlhC in the absence of new synthesis of these proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16430704" target="_blank">16430704</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16430704" target="_blank">16430704</a>]<br></li>
<li>This implies that FlhD and FlhC are essential factors for expression of the class 2 operons , whereas FliA is required only for the relief of its repression by FlgM .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8206838" target="_blank">8206838</a>]<br></li>
				<label class="TU_collapse_similarity" for="tu_similarity_11">
The ﬂiA gene is positively controlled by activator proteins , FlhD and FlhC , which are encoded by the ﬂhD class 1 operon ( also called the master operon ) lying at the top of the transcription hierarchy ( 26 , 30 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11790733" target="_blank">11790733</a>]<br></label>
<input id="tu_similarity_11", type="checkbox">
<div class="TU_similar">
<li>Furthermore , the transcription of ﬂiA gene is positively controlled by the class 1 gene products , FlhD and FlhC , which are encoded by the sole operon lying at the top of the transcriptional hierarchy .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11790733" target="_blank">11790733</a>]<br></li>
<li>In turn , ﬂiA is positively controlled by the gene products FlhD and FlhC , which are encoded by the sole operon lying at the apex of the hierarchy .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16430704" target="_blank">16430704</a>]<br></li></div>
				<label class="TU_collapse_similarity" for="tu_similarity_14">
Class 1 promoters drive expression of the ﬂagellar master operon that encodes the transcription factors FlhD and FlhC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26441919" target="_blank">26441919</a>]<br></label>
<input id="tu_similarity_14", type="checkbox">
<div class="TU_similar">
<li>The class 1 promoters drive the expression of the flagellar master operon , which encodes the transcription factors FlhD and FlhC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16912280" target="_blank">16912280</a>]<br></li></div>
<li>The FlhD and FlhC proteins are required for expression of the rest of the genes in the regulon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9765570" target="_blank">9765570</a>]<br></li>
<li>At the top of the hierarchy is the ﬂhDC operon that encodes the FlhD and FlhC proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28603807" target="_blank">28603807</a>]<br></li>
<li>FlhC and FlhD form heterotetramers to activate expression of operons encoding the ﬂagellar basal body and FliA ( 56 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10714991" target="_blank">10714991</a>]<br></li>
<li>The ﬂiAZY operon requires the master ﬂagellar gene regulators , FlhD and FlhC , for expression ( 44 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11292791" target="_blank">11292791</a>]<br></li>
				</div></ul></div><br>
	</div>
</body>
</html>