<!doctype html>
<style>
@charset "utf-8";
body {
	height: 100%;
	margin: 0;
	padding: 0;
}

.banner {
	position: fixed;
	top: 0;
	width: 100%;
	height: 65px;
	background: gray;
	line-height: 65px;
}
.spacer {
	height: 10px;
}

.main-content {
	padding: 5%;
}

.collapse_ACT {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-style: none solid none none;
	border-color: #B4045F;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_ACT + input{
	display: none;
}
.collapse_ACT + input + div{
	display: none;
}
.collapse_ACT + input:checked + div{
	display: block;
}
.ACT {
	text-align: justify;
	text-justify: inter-word;
}
.ACT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B4045F;
}
.ACT_collapse_similarity:hover{
	background: #e5d3dc;
}
.ACT_collapse_similarity + input {
	display: none;
}
.ACT_collapse_similarity + input + div{
	display: none;
}

.ACT_collapse_similarity + input:checked + div{
	display: block;
}
.ACT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.ACT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B4045F;
	padding-left: 5px;
}
.ACT_collapse_similarity:hover{
	background: #e5d3dc;
}
.ACT_collapse_similarity + input {
	display: none;
}
.ACT_collapse_similarity + input + div{
	display: none;
}
.ACT_collapse_similarity + input:checked + div{
	display: block;
}
.ACT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.ACT_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #B4045F;
}
.ACT_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_DOM {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-style: none solid none none;
	border-color: #B404AE;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_DOM + input{
	display: none;
}
.collapse_DOM + input + div{
	display: none;
}
.collapse_DOM + input:checked + div{
	display: block;
}
.DOM {
	text-align: justify;
	text-justify: inter-word;
}
.DOM_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B404AE;
	padding-left: 5px;
}
.DOM_collapse_similarity:hover{
	background: #e8d7e7;
}
.DOM_collapse_similarity + input {
	display: none;
}
.DOM_collapse_similarity + input + div{
	display: none;
}
.DOM_collapse_similarity + input:checked + div{
	display: block;
}
.DOM_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}.DOM_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #B404AE;
}
.DOM_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_EVO {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #7401DF;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_EVO + input{
	display: none;
}
.collapse_EVO + input + div{
	display: none;
}
.collapse_EVO + input:checked + div{
	display: block;
}
.EVO {
	text-align: justify;
	text-justify: inter-word;
}
.EVO_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #7401DF;
	padding-left: 5px;
}
.EVO_collapse_similarity:hover{
	background: #e2d7ed;
}
.EVO_collapse_similarity + input {
	display: none;
}
.EVO_collapse_similarity + input + div{
	display: none;
}
.EVO_collapse_similarity + input:checked + div{
	display: block;
}
.EVO_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.EVO_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #7401DF;
}
.EVO_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_RP {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #0174DF;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_RP + input{
	display: none;
}
.collapse_RP + input + div{
	display: none;
}
.collapse_RP + input:checked + div{
	display: block;
}
.RP {
	text-align: justify;
	text-justify: inter-word;
}
.RP_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #0174DF;
	padding-left: 5px;
}
.RP_collapse_similarity:hover{
	background: #c9d7e2;
}
.RP_collapse_similarity + input {
	display: none;
}
.RP_collapse_similarity + input + div{
	display: none;
}
.RP_collapse_similarity + input:checked + div{
	display: block;
}
.RP_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.RP_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #0174DF;
}
.RP_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_SIT {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #0489B1;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_SIT + input{
	display: none;
}
.collapse_SIT + input + div{
	display: none;
}
.collapse_SIT + input:checked + div{
	display: block;
}
.SIT {
	text-align: justify;
	text-justify: inter-word;
}
.SIT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #0489B1;
	padding-left: 5px;
}
.SIT_collapse_similarity:hover{
	background: #d0dfe2;
}
.SIT_collapse_similarity + input {
	display: none;
}
.SIT_collapse_similarity + input + div{
	display: none;
}
.SIT_collapse_similarity + input:checked + div{
	display: block;
}
.SIT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.SIT_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #0489B1;
}
.SIT_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_TU {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #04B486;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_TU + input{
	display: none;
}
.collapse_TU + input + div{
	display: none;
}
.collapse_TU + input:checked + div{
	display: block;
}
.TU {
	text-align: justify;
	text-justify: inter-word;
}
.TU_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #04B486;
	padding-left: 5px;
}
.TU_collapse_similarity:hover{
	background: #d0e2de;
}
.TU_collapse_similarity + input {
	display: none;
}
.TU_collapse_similarity + input + div{
	display: none;
}
.TU_collapse_similarity + input:checked + div{
	display: block;
}
.TU_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.TU_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #04B486;
}
.TU_similar > a {
	text-decoration: none;
	color: gray;
}

ul { 
	list-style-type: none;
	padding: 0;
	margin: 0;
}
ul li > a {
	text-decoration: none;
	color: gray;
}
ul li:hover, ul li:focus {
	background: #F0F0F0;
}

ul li:first-child span {
  border-left:none;  
}

</style>
<html>
<head>
<meta charset="utf-8">
<title>TF properties - Fur</title>
<meta name="theme-color" content="#ffffff">
</head>
<body>
	<div class="banner">
		<center><font color="white" size="16px" face="arial">Fur</font></center>
	</div>
	<div align="center" class="main-content">
		<div class="spacer">
	</div><br>			<label class="collapse_ACT" for="_1"><font size="5px">Fur transcriptional activity</font></label>
				<input id="_1" type="checkbox">
				<div class="ACT"><br><ul><li>FNR functions alone or in synchronization with other regulators such as ArcA , RpoS , NarL , Fur or SoxR ( Spiro & Guest , 1991 ; Hassan & Sun , 1992 ; Gunsalus & Park , 1994 ; Schröder et al. , 1994 ; Park & Gunsalus , 1995 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19389776" target="_blank">19389776</a>]<br></li>
<li>Fur regulates acid Oglesby-Sherrouse , A.G. , Murphy , E.R. , 2013 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27242152" target="_blank">27242152</a>]<br></li>
<li>Increased entB expression reﬂects decreased Fur activity ( i.e. , repression ) ( 40 ; reviewed in reference 34 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18835989" target="_blank">18835989</a>]<br></li>
<li>Fur activates hilA transcription in a HilD-dependent manner .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
<li>Fur regulates HilD in a different manner than it regulates sodB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
<li>More recently , it was proposed that Fur activates hilA by repressing H-NS ( Troxell et al. 2010 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22021388" target="_blank">22021388</a>]<br></li>
<li>( B ) Fur overexpression induces hilA expression when HilD is present .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
<li>Fur does not regulate hilA expression by controlling HilE .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
<li>Fur represses iron-responsive genes inside macrophages .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24643535" target="_blank">24643535</a>]<br></li>
<li>For example , Fur represses aerobactin biosynthesis using ferrous iron , cobalt , or manganese [ 2 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22017966" target="_blank">22017966</a>]<br></li>
<li>Our data showed that Fur represses tcf expression ( Fig. 4b ) , whereas low iron conditions increased it .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26944792" target="_blank">26944792</a>]<br></li>
<li>We found that Fur represses SPI-2 expression that is activated inside macrophages and under acidic conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24643535" target="_blank">24643535</a>]<br></li>
<li>However , our results did not support this initial hypothesis since ΔfurΔfnr exhibited ~ 3.5-fold increased expression compared to Δfnr ; indicating that Fur regulation was Fnr-independent .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22017966" target="_blank">22017966</a>]<br></li>
<li>In a fur strain ( MntR ) with a mutated MntR binding site , there was no repression by Fe , indicating that MntR does not bind to the Fur binding site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659669" target="_blank">15659669</a>]<br></li>
<li>However , when MntR was mutated , repression by Mn was still evident ( although there was less repression than with wild-type MntR ) , and this repression was greatly reduced when Fur or the Fur binding site was also mutated , suggesting that Fur can also use Mn as a cofactor to-bind at the Fur binding site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659669" target="_blank">15659669</a>]<br></li>
<li>When Fur was mutated , repression by Fe was still evident , but this repression was lost when MntR or the MntR binding site was also mutated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659669" target="_blank">15659669</a>]<br></li>
<li>Fur , but not MntR , can use Co as a cofactor to repress sit transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659669" target="_blank">15659669</a>]<br></li>
<li>The Fur regulator requires iron binding for its activity and binds to promoters of iron-responsive genes to repress their transcription under ironreplete conditions ( 12 , 13 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24643535" target="_blank">24643535</a>]<br></li>
<li>f These proteins appear to require during deferrated for induction Fur iron limitation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1624426" target="_blank">1624426</a>]<br></li>
<li>Unlike results for sitABCD , neither Fur nor MntR appears to be involved in mntR transcriptional regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659669" target="_blank">15659669</a>]<br></li>
<li>Fur is known to be only a transcriptional-repressor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
<li>c Promoters known to be Fur regulated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12897017" target="_blank">12897017</a>]<br></li>
<li>Fur binding to nrdHIEF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20593029" target="_blank">20593029</a>]<br></li>
<li>Under Fe-limited conditions , relief of Fur repression allows RyhB to recruit ftnA and bfr mRNA to degradosomes ( Masse et al. , 2003 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17302823" target="_blank">17302823</a>]<br></li>
<li>Then , in the transition to stationary phase growth , other regulators , such as Fur and RpoS , contribute to increase dps transcription to up to 200 000 molecules per cell ( Almiron et al. , 1992 ; Yoo et al. , 2007 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25028458" target="_blank">25028458</a>]<br></li>
<li>However , Fur can respond to Mn , and MntR can respond to Fe to cause repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659669" target="_blank">15659669</a>]<br></li>
<li>Salmonella harbours several regulons which enable it to adapt to acidiﬁcation , especially those controlled by RpoS , Fur , PhoPQ , and OmpR/EnvZ .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16023758" target="_blank">16023758</a>]<br></li>
<li>The Fur protein was shown to repress feoB transcription under iron-replete conditions ( 3 , 11 ) , while the activator Fnr protein promotes feoB transcription under anaerobic conditions ( 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23708131" target="_blank">23708131</a>]<br></li>
<li>The iron-sensing Fur regulator represses feoB transcription under iron-replete conditions ( 3 , 11 ) while the oxygen-sensing Fnr regulator promotes feoB transcription under anaerobic conditions ( 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23708131" target="_blank">23708131</a>]<br></li>
<li>The question arose whether the iron sensing regulator Fur was involved in this iron mediated activation ( Ellermeier & Slauch , 2008 ; Teixidó et al. , 2011 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27704705" target="_blank">27704705</a>]<br></li>
<li>The iron-sensing Fur regulator represses feo transcription , and the oxygen-sensing Fnr regulator activates feo transcription ( 2 , 9 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25313398" target="_blank">25313398</a>]<br></li>
<li>Many genes that are induced under iron-starvation conditions are regulated by Fur ( Stojiljkovic et al. , 1994 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10712695" target="_blank">10712695</a>]<br></li>
<li>ipf genes are negatively regulated by Fur and activated by OmpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28817673" target="_blank">28817673</a>]<br></li>
<li>e Class A represents promoters that are negatively regulated by Fur .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12897017" target="_blank">12897017</a>]<br></li>
<li>This implies that genes controlled by H-NS are regulated by iron through Fur .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22017966" target="_blank">22017966</a>]<br></li>
<li>Fur represses siderophore synthesis genes such as entABCE and iroDEN under iron-replete conditions ( 27 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24643535" target="_blank">24643535</a>]<br></li>
<li>In this study , ftnB and fumA were positively regulated by Fur under high-iron conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12897017" target="_blank">12897017</a>]<br></li>
<li>Both bfr and ftnA are under positive regulation by Fur , and hence are expressed under iron-replete conditions consistent with iron storage/detoxification roles , while both ftnB and dps are negatively regulated by Fe ( II ) - Fur and thus expressed under conditions of iron-depletion .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21722794" target="_blank">21722794</a>]<br></li>
<li>Expression of prgH and invF decreased ninefold and sixfold under iron limited conditions , this regulatory mechanism being largely mediated by Fur .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27704705" target="_blank">27704705</a>]<br></li>
<li>Neiderhoffer et al. ( 29 ) provide evidence that the superoxide dismutase genes sodA4 and sodB are under negative and positive control , respectively , by Fur .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1624426" target="_blank">1624426</a>]<br></li>
<li>Moreover , ftnB and dps are repressed by the Fe-responsive regulator Fur and induced under conditions of Fe limitation , whereas bfr and ftnA are maximally expressed when Fe is abundant .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17302823" target="_blank">17302823</a>]<br></li>
<li>The E. coli FtnA and Bfr are indirectly activated , under Fe-rich conditions , by the Fe-dependent repressor , Fur .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17302823" target="_blank">17302823</a>]<br></li>
<li>Surprisingly , not all iron-regulated proteins were controlled by Fur and not all Fur-dependent proteins were obviously regulated by iron status .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1624426" target="_blank">1624426</a>]<br></li>
<li>The genes for these proteins are repressed by Fur when iron is abundant and are upregulated when iron is limiting ( 12 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24990908" target="_blank">24990908</a>]<br></li>
<li>As such , at high iron concentrations , iron acquisition systems are inhibited by Fur , and excess iron is stored by the iron storage proteins ( FtnA , FtnB , Bfr ) ( Andrews , 1998 ; Velayudhan et al. , 2007 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23306672" target="_blank">23306672</a>]<br></li>
<li>Therefore , Salmonella feo expression is highly induced under low-oxygen conditions with low iron ( 8 ) , where Fur activity is reduced ( 10 ) and Fnr activity is enhanced ( 11 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25313398" target="_blank">25313398</a>]<br></li>
<li>In contrast , the Helicobacter pylori ferritin , Pfr , is repressed by Fur under low-Fe conditions ( Bereswill et al. , 2000 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17302823" target="_blank">17302823</a>]<br></li>
<li>Conversely , the sRNAs are expressed under low iron conditions and their expression is repressed by Fur .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23306672" target="_blank">23306672</a>]<br></li>
<li>However , excess iron is toxic , and its uptake is exquisitely regulated by the ferric uptake regulator , Fur .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21075923" target="_blank">21075923</a>]<br></li>
<li>This sRNA is expressed in low iron conditions where the repression by Fur is inhibited ( Massé & Gottesman , 2002 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23306672" target="_blank">23306672</a>]<br></li>
<li>The sitABCD operon is induced under iron-deficient conditions in vitro and in the testinal tant is gesting is repressed by Fur .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10712695" target="_blank">10712695</a>]<br></li>
<li>Similarly , Salmonella sodB is Fur activated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
<li>This demonstrated that regulation by Fur is indeed functional in ATCC 14028 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20008066" target="_blank">20008066</a>]<br></li>
<li>It is known that the RyhB-mediated control is negatively regulated itself by Fur ( Ferric Uptake Regulator ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23831078" target="_blank">23831078</a>]<br></li>
<li>Our results confirm the prediction by Baumler et al that iroN is regulated by Fur [ 58 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22017966" target="_blank">22017966</a>]<br></li>
<li>The sit operon , for example , is dramatically regulated by oxygen , a phenomenon that is completely dependent on Fur ( 25 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
<li>NrdEF is usually repressed by Fur .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24044554" target="_blank">24044554</a>]<br></li>
<li>Accordingly , Fur represses H-NS , and thereby HilD is no longer repressed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23554419" target="_blank">23554419</a>]<br></li>
<li>The sitABCD promoter is regulated by MntR and Fur .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659669" target="_blank">15659669</a>]<br></li>
<li>These data show that sit transcription is negatively controlled by both MntR and Fur .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659669" target="_blank">15659669</a>]<br></li>
<li>It should be also reminded that MnSOD is transcriptionally repressed by Fur ( which senses intracellular iron content ) while FeSOD synthesis is positively regulated by Fur via small RNAs and Hfq ( Niederhoffer et al. 1990 ; Hassan and Schrum 1994 ; Troxell et al. 2011 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24970347" target="_blank">24970347</a>]<br></li>
<li>On the other hand , HilD activity is positively regulated by FliZ and Fur , through still unknown mechanisms ( Ellermeier and Slauch , 2008 ; Chubiz et al. , 2010 ) , as well as negatively regulated by HilE , through protein-protein interactions ( Baxter et al. , 2003 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26300871" target="_blank">26300871</a>]<br></li>
<li>Lane 1 , 14028s + MnCl2 ; lane 2 , 14028s + pfur-ha ; lane 3 , 14028s + MnCl2 + pfur-ha ; lane 4 , Δfur + MnCl2 ; lane 5 , Δfur + pfur-ha ; lane 6 , Δfur + MnCl2 + pfur-ha Figure 4 Effects of Fur , Fnr and iron chelation on transcription of ftnB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22017966" target="_blank">22017966</a>]<br></li>
<li>For such FeoB production , Fur repression should be relieved and Fnr activation should occur on feoB transcription ( Fig. 1A and 2A ) , as previously reported ( 3 , 11 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23708131" target="_blank">23708131</a>]<br></li>
<li>Fur and MntR account for all cation repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12029030" target="_blank">12029030</a>]<br></li>
<li>H-NS confirmed binding sites and transcriptional repression [ 31 ] were compared with our microarray data and Fur repression of hns [ 29 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22017966" target="_blank">22017966</a>]<br></li>
<li>The ydiE gene is Fe-Fur repressed in S. Typhimurium ( Lutz and Bujard , 1997 ; Bjarnason et al. , 2003 ) , and its induction in EWMM is consistent with FeFur repression in S. Enteritidis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28553268" target="_blank">28553268</a>]<br></li>
<li>In S. enterica serovar Typhimurium ( S. Typhimurium ) , Fur regulates systems implicated principally in iron acquisition , but also other genes involved in acid stress and adaptation , oxidative-stress resistance and virulence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23306672" target="_blank">23306672</a>]<br></li>
<li>Oglesby , A.G. , Murphy , E.R. , Iyer , V.R. and Payne , S.M. ( 2005 ) Fur regulates acid resistance in Shigella ﬂexneri via RyhB and ydeP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18267966" target="_blank">18267966</a>]<br></li>
<li>The S. Typhimurium ferritins are differentially regulated Previous studies in E. coli have shown that Fur positively regulates ftnA and bfr expression under Fe-replete conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17302823" target="_blank">17302823</a>]<br></li>
<li>Both are known to regulate virulence factors in Salmonella ( Fur positively , and HNS negatively ) [ 17,22,95 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23040276" target="_blank">23040276</a>]<br></li>
<li>These data suggest that ftnB and dps are negatively regulated by Fur in Fe-rich medium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17302823" target="_blank">17302823</a>]<br></li>
<li>Site-speciﬁc mutations in putative Fur and MntR binding sites disrupted regulation by Fur and MntR , conﬁrming that those sites are involved in transcriptional regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659669" target="_blank">15659669</a>]<br></li>
<li>OmpR , Lrp and Fur are involved in pESI fimbriae regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28817673" target="_blank">28817673</a>]<br></li>
<li>NO · interactions with NsrR and SoxR are coordinated through iron-sulfur clusters ( FeS ) , whereas NorR and Fur interact directly with Fe2 + .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22831173" target="_blank">22831173</a>]<br></li>
<li>Fur has previously been demonstrated to indirectly activate expression of ferritins including Bfr and FtnA ( Abdul-Tehrani et al. , 1999 ; Masse and Gottesman , 2002 ) and of several other Fe-containing proteins including aconitase A , fumarase A , succinate dehydrogenase and several respiratory complexes ( Masse and Gottesman , 2002 ; McHugh et al. , 2003 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17302823" target="_blank">17302823</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_77">
NsrR senses very low concentrations of NO · The iron-containing transcriptional-regulators NsrR , NorR , SoxR and Fur have been shown to respond to NO · in E. coli ( Crack et al. , 2012 ; Fleischhacker and Kiley , 2011 ; Spiro , 2006 ; Zheng and Storz , 2000 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22831173" target="_blank">22831173</a>]<br></label>
<input id="act_similarity_77", type="checkbox">
<div class="ACT_similar">
<li>Several iron-containing transcriptional-regulators , including NsrR , NorR , Fur , and SoxR , have been previously shown to respond to NO · ( Crack et al. , 2012 ; Fleischhacker and Kiley , 2011 ; Spiro , 2006 ; Zheng and Storz , 2000 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22831173" target="_blank">22831173</a>]<br></li></div>
<li>Ferric Uptake Regulator ( Fur ) positive and Gram-negative bacteria to regulate genes that encode iron-uptake and - storage proteins as a repressor or activator , respectively ( Bsat et al. , 1998 ; Escolar et al. , 1999 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27242152" target="_blank">27242152</a>]<br></li>
<li>Flagella-mediated motility is transcriptionally controlled in response to multiple stresses including heat-shock ( RpoH dependent ) , pH change ( CpxAR dependent ) , envelope stress ( OmpR/EnvZ , CpxAR and PspF dependent ) and low iron content ( Fur dependent ) ( Shin and Park , 1995 ; Prüß et al. , 2003 ; Lloyd et al. , 2004 ; Jovanovic et al. , 2006 ; Raivio et al. , 2013 ) , which are all of relevance to the conditions experienced upon EWMM exposure , as indicated by the transcriptomics data .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28553268" target="_blank">28553268</a>]<br></li>
<li>Moreover , Fur , PspF and OmpR/EnvZ regulate FlhDC ( Stojiljkovic and Hantke , 1992 ; Campoy et al. , 2002 ; Jovanovic et al. , 2006 ; Samanta et al. , 2013 ) and PspF is negatively regulated by RpoH involved in the heat-shock response ( Brissette et al. , 1991 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28553268" target="_blank">28553268</a>]<br></li>
<li>The iron-rationing response is controlled indirectly by Fur through RfrA and RfrB in Salmonella ( RyhB in E. coli ) and largely functions to repress genes encoding ironcontaining proteins in reaction to low iron conditions ( McHugh et al. , 2003 ; Massé et al. , 2005 ; Yang et al. , 2013 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28553268" target="_blank">28553268</a>]<br></li>
<li>In S. Tm , cib expression is negatively regulated in a Fur ( ferric uptake regulator ) - dependent and LexA ( locus for x-ray sensitivity A ) - dependent way ( Mankovich et al. , 1986 ; Nedialkova et al. , 2014 ) and maximally induced under iron limitation and upon exposure to DNA damaging agents .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26439675" target="_blank">26439675</a>]<br></li>
<li>In this case , HilD was subjected to positive regulation by four nodes ( SirA-BirA , HilC , RtsA and Fur ) along with self activation [ 11,14,43,44 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24079299" target="_blank">24079299</a>]<br></li>
<li>The conclusion was that all three regulators mediate Co induction , however clearly with different time-responses as IscR - , followed by Fur - , and OxyR-mediated responses occurred in this order .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21952637" target="_blank">21952637</a>]<br></li>
<li>However , in contrast to E. coli , S. Typhimurium acnA was not induced in stationary phase and was repressed , rather than activated , by Fur ( Fig. 3b ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23637460" target="_blank">23637460</a>]<br></li>
<li>For instance , each gene was regulated in the same manner in Δfur or Δfnr ; a gene activated by Fur was also activated by Fnr .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22017966" target="_blank">22017966</a>]<br></li>
<li>] rlgA map rpsB yajC nrdR 2.8 2.6 4.0 3.2 2.5 3.1 7.1 8.2 No No No No No No No No No No No No No Fnr [ 21 ] Fnr [ 21 ] Fnr [ 21 ] Fnr [ 21 ] Yes [ 12 ] Yes [ 12 ] cyoE cyoD cyoB cyoA fepA Yes [ 12 ] Yes [ 12 ] Yes [ 12,15,16,126-129 ] Yes [ 12,16,127-129 ] Yes [ 12,15,130 ] No 3.2 No 10.7 Yes [ 128,129 ] fes entC 39.8 6.8 4.1 5.6 2.7 2.8 -4.2 7.4 Yes [ 128,129 ] Yes [ 130 ] No No No No No No Fnr [ 21 ] sucC gpmA cmk Yes [ 12 ] No No No Yes [ 12,15 ] No Yes [ 15 ] No STM1013 STM1133 ydiE nth STM1586 Fnr [ 21 ] Rcs [ 131 ] 2.9 No No 76.1 -4.0 -37.3 11.4 3.1 5.8 -8.8 Fnr [ 21 ] Fnr [ 21 ] ldhA ynaF No No Yes [ 132 ] Yes [ 29 ] No No No Yes [ 12,15 ] Yes [ 29 ] No No Yes [ 12,15 ] No No tonB hns STM1795 Fnr [ 21 ] Fnr [ 21 ] STM2186 cirA eutC eutB yffB Yes [ 133 ] No No No No No No 4.0 -4.1 -3.2 2.6 4.6 9.1 53.8 Fnr [ 21 ] Fnr [ 21 ] No Yes [ 15,59 ] iroB iroN sitA No Yes [ 15,46,61,134-138 ] No Yes [ 12 ] Yes [ 101 ] Yes [ 14,79,88 ] MntR [ 61 ] No No 3.5 3.8 yggU yqjH secY bfr bfd Yes [ 100 ] No No No 4.0 3.2 5.9 11.8 Yes [ 12,14,15 ] Yes ArcA and Fnr [ 141 ] feoB [ 12,14,63,134,139,140 ] No No No No Yes [ 14,55,82,88,143-148 ] No STM3600 STM3690 rpoZ udp sodA No No Fnr [ 21 ] Fnr [ 21 ] -6.8 -4.2 3.9 -5.4 9.1 No No Yes [ 85,146,148 ] IscS [ 142 ] Fnr , ArcA , IHF , SoxRS [ 53,81 ] yjcD N 2.8 Table 4 Comparison of Differentially Expressed Genes in Δ?Δ?fur That Contain a Putative Fur Binding Site with Confirmed Data of Fur Regulation from other Studies and the Possible Involvement of other Transcription Regulators ( Continued ) No -5.8 -3.6 No dcuA aspA No Yes [ 13,15 ] NarL [ 149,150 ] ArcA [ 151 ] NsrR [ 99 ] ytfE 10.0 Yes [ 13 ] fhuF 8.5 Yes [ 12,13,15 ] No Yes [ 11,152,153 ] a Genes from the present study that are regulated by Fur and possess a putative Fur-binding motif bFold change of expression in Δfur relative to the wt 14028s c Evidence of direct Fur binding the regulatory region of the gene d Regulation by other transcription factors besides Fur Figure 6 Role of Fur in the transcriptional , translational and post-translational regulation of sodA and sodB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22017966" target="_blank">22017966</a>]<br></li>
<li>How Fur , which is thought to negatively regulate gene expression in presence of Fe2þ positively contributes to SPI1 gene expression is not well understood .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23040276" target="_blank">23040276</a>]<br></li>
<li>Of these , Fur is a reasonable candidate for contributing to PQ-induced sodA expression under aerobic-growth .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10613858" target="_blank">10613858</a>]<br></li>
<li>However , the Helicobacter pylori Fur protein can activate frpB gene transcription by directly binding its promoter ( Delany et al. , 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11932449" target="_blank">11932449</a>]<br></li>
<li>In S. enterica serovar Typhimurium , at least 30 proteins are negatively regulated at the transcription level by Fur ( 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12652904" target="_blank">12652904</a>]<br></li>
<li>Since most mRNAs targeted by RyhB are repressed at the translation level , these target genes are positively regulated by the activated Fur repressor through RyhB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22824499" target="_blank">22824499</a>]<br></li>
<li>Where class A genes are negatively regulated by Fur in the classical manner , class B genes appear to be positively regulated by Fur .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12897017" target="_blank">12897017</a>]<br></li>
<li>Either way , Fur regulates hilA by posttranslationally affecting the HilD protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
<li>Thus , under this 2.5-fold constraint , the microarray data did not identify hilA as differentially regulated by Fur .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21075923" target="_blank">21075923</a>]<br></li>
<li>Under iron-rich conditions , it has been shown that expression of ryhB gene is repressed by Fe2 + - Fur , whereas transcription of genes encoding iron-storage proteins is elevated ( Masse et al. , 2005 ; Oglesby et al. , 2005 ; Vecerek et al. , 2007 ; Ellermeier and Slauch , 2008 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27242152" target="_blank">27242152</a>]<br></li>
<li>In addition , hmpA expression is upregulated under iron-limiting conditions through NsrR , but independently of Fur ( Bang et al. , 2006 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21833325" target="_blank">21833325</a>]<br></li>
<li>For example , Fur negatively regulates transcription of HN-S in Salmonella ( Troxell et al. , 2010 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21833325" target="_blank">21833325</a>]<br></li>
<li>The action of Fur in SPI1 regulation is more complicated ( Troxell et al. 2010 ; Teixido et al. 2011 ) , but also requires HilD ( Ellermeier and Slauch 2008 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22021388" target="_blank">22021388</a>]<br></li>
<li>Attributing residual Mn2 repression to the normal Fur regulatory circuit is more problematic .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12029030" target="_blank">12029030</a>]<br></li>
<li>When Fe -- Fur is bound to the 2 + fur box , gene transcription is prevented .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16023758" target="_blank">16023758</a>]<br></li>
<li>Mechanistically , Fur is a transcriptional-repressor , suggesting that the effects on hilA must be indirect .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17208038" target="_blank">17208038</a>]<br></li>
<li>Small regulatory RNA molecules such as RyhB provide a mechanism for positive Fur regulation ; it is likely that RyhB , dependent on the Hfq RNA-binding protein for function , regulates the target gene by terminating transcription postinitiation ( 30 , 31 , 34 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12897017" target="_blank">12897017</a>]<br></li>
<li>The data shown in Fig. 4A to D indicate that Fe2 repression is not completely abolished by any mutation in the Fur regulator circuit .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12029030" target="_blank">12029030</a>]<br></li>
<li>Thus , ryhB transcription is dependent on regulation by the Fur protein in response to iron availability .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27242152" target="_blank">27242152</a>]<br></li>
<li>Regulation by the Fur transcriptional-repressor in response to iron availability is also investigated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17302823" target="_blank">17302823</a>]<br></li>
<li>This system is known to be induced by low iron in response to Fur , but also by low Mn2 + in response to MntR ( Ikeda et al. , 2005 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28553268" target="_blank">28553268</a>]<br></li>
<li>Thus , sitABCD is primarily regulated by Fur in response to iron levels .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10712695" target="_blank">10712695</a>]<br></li>
<li>In this context , both sRNAs are controlled by Fur in response to iron starvation ; however , RyhB-2 is expressed at lower levels than RyhB-1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24239962" target="_blank">24239962</a>]<br></li>
<li>Therefore , it is not surprising that Fur regulated , low-iron-induced genes such as sitABCD and sodA would answer an IVET selection in cultured hepatocytes and that these genes are also induced in the host .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10712695" target="_blank">10712695</a>]<br></li>
<li>What is interesting in this context is that three components defined as regulatory factors in the ATRs , OmpR , Fur and PhoP , were initially identified as regulators responding to osmolarity or to selected metal ions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11370771" target="_blank">11370771</a>]<br></li>
<li>While both RNAs can control sodB , it is possible that they differentially control other genes , and it will be interesting to understand the complete Fur regulon in Sal-monella .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
<li>Fur is active ( i.e. , represses ) when bound to Fe2 ( reviewed in references 8 and 19 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18835989" target="_blank">18835989</a>]<br></li>
<li>Fur is a transcriptional-repressor that is active when bound to ferrous iron ( reviewed in references 6 and 18 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17720790" target="_blank">17720790</a>]<br></li>
<li>Fur is a well-characterized transcriptional-repressor that regulates gene expression in response to iron ( 18 -- 20 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12897017" target="_blank">12897017</a>]<br></li>
<li>Fur is a well-established negative transcriptional-regulator that responds to iron availability and class A expression pro-ﬁles are consistent with traditional Fur regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12897017" target="_blank">12897017</a>]<br></li>
<li>These results suggest that repression of sitABCD and mntH is mediated by MntR binding to the MntR binding site or Fur binding to the Fur binding site , and each repressor can use either Mn or Fe as a cofactor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659669" target="_blank">15659669</a>]<br></li>
<li>Fur-mediated repression requires the Fur binding site and is most efﬁcient with Fe , but Fur also responds to Mn ( or Co ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659669" target="_blank">15659669</a>]<br></li>
<li>In a fur strain ( MntR ) with a mutated MntR binding site and in an mntR strain ( Fur ) with a mutated Fur binding site , there was little or no repression , indicating that in response to Mn , neither Fur nor MntR binds to the heterologous binding site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659669" target="_blank">15659669</a>]<br></li>
<li>In this study , we found that Fur , acting at the Fur binding site , can repress mntH and sitABCD in response to Fe or Mn .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659669" target="_blank">15659669</a>]<br></li>
<li>Our results indicate that in response to low levels of Mn , MntR is more efﬁcient than Fur at repressing sitABCD and mntH .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659669" target="_blank">15659669</a>]<br></li>
<li>Likewise , in response to low levels of Fe , Fur can repress sitABCD and mntH better than MntR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659669" target="_blank">15659669</a>]<br></li>
<li>Our studies indicate that both Fur and MntR regulate sit transcription in response to Fe and Mn , respectively , although both regulators can use the alternative metal to partially repress transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659669" target="_blank">15659669</a>]<br></li>
<li>As was reported previously ( 19 ) , the mntH promoter is regulated in response to both Mn and Fe , primarily via MntR and Fur , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659669" target="_blank">15659669</a>]<br></li>
<li>Thus , although there are subtle differences , the mntH and sit promoters are regulated similarly by Fur and MntR in response to Fe and Mn .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659669" target="_blank">15659669</a>]<br></li>
<li>As seen in Figure 2 , Fur also repressed nrdHIEF expression , increasing its transcription up to 5-fold compared to the wild-type strain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20593029" target="_blank">20593029</a>]<br></li>
<li>Previously , we suggested that Fur can repress mntH in response to Mn , but the exact mechanism was unclear ( 19 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659669" target="_blank">15659669</a>]<br></li>
<li>Key players in Salmo-nella stress response in intestinal colonisation are RpoS , PhoPQ , Fur or OmpR/EnvZ , which are not spe-ciﬁc to this genus , showing that the stress adaptation was already evolved in an ancient ancestor , but whether this was also a gut coloniser is impossible to say .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16023758" target="_blank">16023758</a>]<br></li>
<li>In many bacteria , Fur is generally involved in regulation of iron homeostasis genes , but it is also known to participate in bacterial colonization , oxidative-stress response , toxin secretion and pathogenicity [ 55 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28817673" target="_blank">28817673</a>]<br></li>
<li>Likewise , feoB expression is under dual transcriptional control by the iron-sensing Fur and oxygen-sensing Fnr regulators whereby FeoB-mediated Fe ( II ) uptake is maintained at appropriate levels in response to different levels of iron and oxygen .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23708131" target="_blank">23708131</a>]<br></li>
<li>The expression of RyhB-1 and RyhB-2 is induced to downregulate target mRNAs implicated in iron homeostasis , including its cognate repressor Fur and dispensable iron-con-taining proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24239962" target="_blank">24239962</a>]<br></li>
<li>Clearly , these results suggest Fnr is functional in Δfur and that Fur is regulating genes of anaerobic metabolism ( eut , frd , and dms operons ) through an unknown mechanism .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22017966" target="_blank">22017966</a>]<br></li>
<li>Expression of hmpA is repressed by Fur in Salmonella , making it responsive The SOS response Free radicals are potent inducers of bacterial SOS responses in vitro ( Buchmeier et al. , 1993 ; Khil and Cam-erini-Otero , 2002 ; Motohashi and Saito , 2002 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12492857" target="_blank">12492857</a>]<br></li>
<li>Fur regulation of HilD is independent of HilE and is also unaffected by RfrA and RfrB , the E. coli RyhB ortholog and paralog found in Salmonella .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
<li>In this work we have proven that Fur is indeed a transcriptional-regulator of nrdHIEF in S. Typhimurium , normally repressing its expression and causing an upregulation under iron depleted growth ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20593029" target="_blank">20593029</a>]<br></li>
<li>In summary , Fur represses expression of sRNAs RfrA and RfrB , a repression that is important for optimal growth of S. Typhi and resistance to H2O2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23306672" target="_blank">23306672</a>]<br></li>
<li>This is in contrast to what has been observed for sodB regulation ; overproduction of Fur had little if any effect on sodB expression , presumably because Fur repression of rfrA and rfrB is essentially complete in these conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
<li>Moreover , we observed that in this mutant there is a total shut down of FeSOD activity and accumulation in presence of cadmium , which is indicative of a drastic reduction of iron binding to Fur ( Taylor et al. 2009 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24970347" target="_blank">24970347</a>]<br></li>
<li>We speculate that a primary role of RstA activation of Fur in acidic conditions is to reduce ferrous iron transport , repress hns , and promote Furmediated hydroperoxidase activity ( 64 , 66 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21075923" target="_blank">21075923</a>]<br></li>
<li>Finally , this work identified twenty-six new targets of Fur regulation , and demonstrates that H-NS repressed genes are down-regulated in Δfur .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22017966" target="_blank">22017966</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_142">
Delany , I. , Spohn , G. , Rappuoli , R. , and Scarlato , V. ( 2001 ) The Fur repressor controls transcription of iron-activated and -- repressed genes in Helicobacter pylori .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17302823" target="_blank">17302823</a>]<br></label>
<input id="act_similarity_142", type="checkbox">
<div class="ACT_similar">
<li>The Fur repressor controls transcription of iron-activated and - repressed genes in Helicobacter pylori .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11932449" target="_blank">11932449</a>]<br></li></div>
<li>Moreover , under Fe-limiting conditions in which Fur would be anticipated to be inactive , levels of bfr and ftnA expression were comparable in wild type and fur mutant backgrounds .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17302823" target="_blank">17302823</a>]<br></li>
<li>The entABCDE and sufABCDS genes involved in iron-uptake and assimilation were also signiﬁcantly upregulated and 90 % of the Fur regulon was signiﬁcantly increased in immobilized cells at LEP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22356617" target="_blank">22356617</a>]<br></li>
<li>We examined isrE and ryhB transcription at stationary phase and upon exposure to hydrogen peroxide in the absence of Fur and found that while RyhB levels were further elevated in a fur mutant ( Figure 3C compare lanes 3 and 8 top and middle panels ) , no increase in IsrE RNA could be detected in fur-deﬁcient cells at both hydrogen peroxide treated and stationary phase cells ( Figure 3C compare lanes 2 and 7 top and middle panels ) , further conﬁrming that the expression control of these genes differs .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18267966" target="_blank">18267966</a>]<br></li>
<li>These results clearly indicate the requirement for Hmp and Fur in maximal survival and proliferation in activated macrophages .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17526833" target="_blank">17526833</a>]<br></li>
<li>Consistent with positive regulation by Fur , bfr and ftnA mRNA levels were approximately fourfold higher in wild-type S. Typh-imurium than in a fur mutant in Fe-rich medium , and reduced in the presence of the Fe chelator , 2,2 ′ - dipyridyl ( dip ) ( Fig. 3A and B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17302823" target="_blank">17302823</a>]<br></li>
<li>By controlling expression/activity of HilD , HilC , RtsA , and HilA , many regulators contribute to SPI-1 gene regulation in response to environmental stimuli that include temperature ( H-NS and Hha ) , osmolarity ( H-NS and Hha ) , pH ( OmpR ) , iron availability ( Fur ) , and oxygen levels ( FNR ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27601571" target="_blank">27601571</a>]<br></li>
<li>For example , NsrR and NorR are dedicated sensors of NO , and the transcriptional-regulators OxyR , SoxR , fumarate-nitrate reduction ( FNR ) , and Fur , which primarily function as sensors of O2 , reactive oxygen species ( ROS ) , and iron , also respond to RNS ( 10 , 12 , 15 , 18 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22311927" target="_blank">22311927</a>]<br></li>
<li>The proteins NsrR and NorR are NO-dedicated sensors that regulate genes encoding antinitrosative activity in aerobic and anaerobic bacteria ( 15 , 16 ) , whereas OxyR , SoxR , FNR , DksA , and Fur , which primarily serve as sensors of reactive oxygen species , O2 , nutritional starvation , and iron , contribute important aspects of the antinitrosative arsenal of enteric bacteria ( 17 -- 21 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24166960" target="_blank">24166960</a>]<br></li>
<li>Interestingly , many crucial components of the starvation response and the ATR are also well-defined virulence regulators ( 64 , 65 ) and involve , as regulators , components such as OmpR , RpoS , PhoPPhoQ and Fur , which are also found in E. colz .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11370771" target="_blank">11370771</a>]<br></li>
<li>Both mntH and sitABCD are under the control of MntR and Fur , and hence are regulated in response to both iron and manganese , and also possibly involving some overlap with respect to the metal specificities of the sensors ( Kehres et al. , 2002b ; Ikeda et al. , 2005 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21722794" target="_blank">21722794</a>]<br></li>
<li>We show the role of both core ( Lrp , Fur and OmpR ) and horizontally acquired ( KlfB and KlfL ) regulators in orchestrating these fimbriae expression and demonstrate that they are induced under micro-aerobic conditions and at the avian physiological temperature ( 41 ˚C ) compared to 37 ˚C or the ambient temperature .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28817673" target="_blank">28817673</a>]<br></li>
<li>S. Typhimurium was treated with varying concentrations of the NO · - releasing compound diethylamine NONOate ( DEA/NO ) , and hmp , norV , entC and soxS expression was determined by quantitative RT-PCR as a measure of NsrR , NorR , Fur and SoxR expression , respectively ( Experimental Procedures ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22831173" target="_blank">22831173</a>]<br></li>
<li>Therefore , Fur might indirectly repress tcf expression by repressing these sRNAs , as loss of fur or growth in low iron concomitantly increases expression of both sRNAs ( RyhB1 and RyhB2 ) and tcf .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26944792" target="_blank">26944792</a>]<br></li>
<li>Increased expression of H-NS , a known repressor of ftnB , tdc operon , and other genes , in Δfur may account for their activation by Fur .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22017966" target="_blank">22017966</a>]<br></li>
<li>Three groups were identiﬁed , consisting of promoters with increased expression when Fur was titrated ( class A ) and decreased expression when Fur was titrated ( class B ) and promoters that appeared to be regulated independently of Fur ( class C ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12897017" target="_blank">12897017</a>]<br></li>
<li>Fur directly binds to the hilD promoter and enhances its own expression ( 15 ) , and HilD functions as an activator of SPI-1 expression ( 17 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24643535" target="_blank">24643535</a>]<br></li>
<li>Understanding OxyR , Fur , and MntR regulation of mntH should enable further regulatory questions to be addressed , particularly questions regarding the sensitivity of mntH expression to growth medium and growth state .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12029030" target="_blank">12029030</a>]<br></li>
<li>https://doi.org/10.1371/journal.ppat.1006559.g006 Lrp controls klf expression , while Fur and OmpR regulate ipf Since the expression of both fimbriae was highly affected by microaerobic conditions , we sought to test a possible role of regulators known to be involved in oxygen homeostasis including OxyR , SoxR , ArcA , ArcB and FNR , [ 29 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28817673" target="_blank">28817673</a>]<br></li>
<li>As shown above , overexpression of Fur caused an eightfold increase in hilA expression , and this effect was dependent on HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
<li>Loss of both MntR and Fur allowed high-level expression under both conditions , although there was still a 27 % decrease in expression when the cells were grown anaerobically for reasons that are not clear .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659669" target="_blank">15659669</a>]<br></li>
<li>In addition , we determined that under anaerobic conditions the expression of hilA was dependent on Fur , and we demonstrated that PhoP was not involved in this regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21075923" target="_blank">21075923</a>]<br></li>
<li>Harvie D.R. , Vilchez S. , Steggles J.R. and Ellar D.J. , Bacillus cereus Fur regulates iron metabolism and is required for full virulence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16362827" target="_blank">16362827</a>]<br></li>
<li>e-mail : jordi.barbe!uab.es Department of Genetics and Microbiology , Universitat Auto ' noma de Barcelona1 and Centre de Recerca en Sanitat Animal ( CReSA ) , Universitat Auto ' noma de Barcelona Institut de Recerca i Tecnologia Agroalimenta ' ria ( UAB-IRTA ) 2 , Bellaterra , 08193 Barcelona , Spain It is known that the Fur protein negatively regulates iron-uptake systems in different bacterial species , including Salmonella typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11932449" target="_blank">11932449</a>]<br></li>
<li>We report here that Fur is required for virulence in Salmonella enterica serovar Typhimurium and that Fur is required for the activation of hilA , as well as of other HilA-dependent genes , invF and sipC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21075923" target="_blank">21075923</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_168">
Troxell , B. , M. L. Sikes , R. C. Fink , A. Vazquez-Torres , J. Jones-Carson et al. , 2010 Fur negatively regulates hns and is required for the expression of hilA and virulence in Salmonella enterica serovar Typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22021388" target="_blank">22021388</a>]<br></label>
<input id="act_similarity_168", type="checkbox">
<div class="ACT_similar">
<li>Troxell B , et al. ( 2011 ) Fur negatively regulates hns and is required for the expression of HilA and virulence in Salmonella enterica serovar Typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23690578" target="_blank">23690578</a>]<br></li>
<li>Fur negatively regulates hns and is required for the expression of HilA and virulence in Salmonella enterica serovar Typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26441883" target="_blank">26441883</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26662614" target="_blank">26662614</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23306672" target="_blank">23306672</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21722794" target="_blank">21722794</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24735176" target="_blank">24735176</a>]<br></li>
<li>Fur negatively regulates hns and is required for the expression of HilA and virulence in Salmonella enterica serovar Typhi-murium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23554419" target="_blank">23554419</a>]<br></li>
<li>Fur negatively regulates hns and is required for the expression of HilA and virulence in Salmonella enterica serovar Typhimu-rium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24166960" target="_blank">24166960</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24643535" target="_blank">24643535</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26386070" target="_blank">26386070</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25547794" target="_blank">25547794</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24990908" target="_blank">24990908</a>]<br></li></div>
				<label class="ACT_collapse_similarity" for="act_similarity_173">
doi :10.1007 / Troxell B , Sikes ML , Fink RC , Vazquez-Torres A , Jones-Car-son J , Hassan HM ( 2011 ) Fur negatively regulates hns and is required for the expression of HilA and virulence in Salmonella enterica serovar Typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24970347" target="_blank">24970347</a>]<br></label>
<input id="act_similarity_173", type="checkbox">
<div class="ACT_similar">
<li>Troxell B , Sikes ML , Fink RC , Vazquez-Torres A , Jones-Carson J , Hassan HM : Fur negatively regulates hns and is required for the expression of HilA and virulence in Salmonella enterica serovar typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23295139" target="_blank">23295139</a>]<br></li>
<li>[ 19 ] B. Troxell , M.L. Sikes , R.C. Fink , A. Vazquez-Torres , J. Jones-Carson , H.M. Hassan , Fur negatively regulates hns and is required for the expression of HilA and virulence in Salmonella enterica serovar Typhimurium , J. Bacteriol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23040276" target="_blank">23040276</a>]<br></li>
<li>Troxell B , Sikes ML , Fink RC , Vazquez-Torres A , Jones-Carson J , Hassan HM : Fur negatively regulates hns and is required for the expression of HilA and virulence in Salmonella enterica serovar Typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22017966" target="_blank">22017966</a>]<br></li></div>
<li>As arabinose was diluted away , synthesis of Crp , Fur and LacI decreased , but PspA synthesis increased , as expected .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20585446" target="_blank">20585446</a>]<br></li>
<li>Shorter moderate acid adaptation also increases total acid resistance of Salmonella but this process is rpoS-independent and requires functional Fur protein ( see below ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16023758" target="_blank">16023758</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_179">
Ikeda , J. S. , Janakiraman , A. , Kehres , D. G. , Maguire , M. E. , and Slauch , J. M. ( 2005 ) Transcriptional regulation of sitABCD of Salmonella enterica serovar typhimurium by MntR and Fur .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24596096" target="_blank">24596096</a>]<br></label>
<input id="act_similarity_179", type="checkbox">
<div class="ACT_similar">
<li>Ikeda , J.S. , Janakiraman , A. , Kehres , D.G. , Maguire , M.E. , and Slauch , J.M. ( 2005 ) Transcriptional regulation of sitABCD of Salmonella enterica serovar Typhimurium by MntR and Fur .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17555437" target="_blank">17555437</a>]<br></li></div>
<li>Ikeda JS , Janakiraman A , Kehres DG , Maguire ME , Slauch JM : Transcriptiona regulation of sitABCD of Salmonella enterica serovar typhimurium by MntR and Fur .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22017966" target="_blank">22017966</a>]<br></li>
<li>Transcriptional regulation of sitABCD of Salmonella enterica serovar Typhimurium by MntR and Fur .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20008066" target="_blank">20008066</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19049822" target="_blank">19049822</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28553268" target="_blank">28553268</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21722794" target="_blank">21722794</a>]<br></li>
<li>Although Fe ( II ) - bound Fur generally acts as a corepressor , Fur can also activate transcription either directly or indirectly , the latter involving the small regulatory RNA ryhB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21722794" target="_blank">21722794</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_184">
The Fur regulator directly binds to the ssrB promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24643535" target="_blank">24643535</a>]<br></label>
<input id="act_similarity_184", type="checkbox">
<div class="ACT_similar">
<li>FIG 4 The Fur regulator directly binds to the ssrB promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24643535" target="_blank">24643535</a>]<br></li></div>
<li>Based on this ﬁnding , we explored whether the Fur regulator could directly bind to the ssrB promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24643535" target="_blank">24643535</a>]<br></li>
<li>This result demonstrates that the Fur protein must bind to the S. typhimurium ﬂhD promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11932449" target="_blank">11932449</a>]<br></li>
<li>Of particular interest was the ﬁnding that pmg , encoding the phosphoglycerate mutase , was regulated by Fur .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7642488" target="_blank">7642488</a>]<br></li>
<li>It was hypothesized that Fur repressed the SPI-2 expression that was activated during acidiﬁcation of the phagosome .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24643535" target="_blank">24643535</a>]<br></li>
<li>In conclusion , we provided , for the ﬁrst time , evidence that Fur represses hns expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21075923" target="_blank">21075923</a>]<br></li>
<li>This observation provides further support to the possibility that Fur directly regulates ipf genes expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28817673" target="_blank">28817673</a>]<br></li>
<li>While Fur has been characterized as primarily as a transcriptional-repressor in the presence of Fe2þ [ 96 ] ; it has also been demonstrated as a transcriptional-activator [ 17 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23040276" target="_blank">23040276</a>]<br></li>
<li>In contrast , overproduction of Fur in the sodB - lacZ strain led to a modest 1.4-fold increase in activity ( sodB - lacZ pWKS30 activity , 763.15 25.39 U ; sodB - lacZ pFur activity , 1,044.06 101.23 U ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
<li>Upon nitrosylation , Fe-Fur loses its DNA-binding activity ( 54 ) , resulting in the derepression of target genes involved in iron acquisition , as observed in our RNA-seq data , as well as in other studies ( 11 , 20 , 21 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25107963" target="_blank">25107963</a>]<br></li>
<li>This result was in accordance with the Fur-dependent repression of tcf expression , because Fur binding activity occurs under iron-rich conditions ( Escolar et al. , 1999 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26944792" target="_blank">26944792</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_196">
RstA-promoted expression of the ferrous iron transporter FeoB under ironreplete conditions enhances Fur activity in Salmonella enterica .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25313398" target="_blank">25313398</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24643535" target="_blank">24643535</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19801407" target="_blank">19801407</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23708131" target="_blank">23708131</a>]<br></label>
<input id="act_similarity_196", type="checkbox">
<div class="ACT_similar">
<li>RstA-promoted expression of the ferrous iron transporter FeoB under iron-replete conditions enhances Fur activity in Salmonella enterica .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20656781" target="_blank">20656781</a>]<br></li>
<li>[ 27 ] J. Jeon , H. Kim , J. Yun , S. Ryu , E.A. Groisman , D. Shin , RstA-promoted expression of the ferrous iron transporter FeoB under iron-replete conditions enhances Fur activity in Salmonella enterica , J. Bacteriol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23040276" target="_blank">23040276</a>]<br></li>
<li>Jeon J , Kim H , Yun J , Ryu S , Groisman EA , Shin D : RstA-promoted expression of the ferrous iron transporter FeoB under iron-replete conditions enhances Fur activity in Salmonella enterica .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22017966" target="_blank">22017966</a>]<br></li></div>
<li>FIG 3 The Fur regulator represses expression of the ssrB gene inside macro-phages and under acidic conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24643535" target="_blank">24643535</a>]<br></li>
<li>Fur represses expression of the ssrB gene encoding the SsrB response-regulator under acidic conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24643535" target="_blank">24643535</a>]<br></li>
<li>Taken together , these results illustrate that the Fur regulator represses production of the SsrB regulator under acidic conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24643535" target="_blank">24643535</a>]<br></li>
<li>Consistent with these ﬁndings , the lack of Fur under anaerobic conditions enhanced FeoB production under high-iron conditions , whereas the lack of Fnr impaired FeoB production under low-iron conditions ( Fig. 1A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23708131" target="_blank">23708131</a>]<br></li>
<li>Fur activates sodB by repressing the expression of ryhB under high-iron conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
<li>Thirty-six proteins were overtly affected by iron availability , most of which were clearly under the control of Fur .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1624426" target="_blank">1624426</a>]<br></li>
<li>These results suggest that Fur and MntR are largely responsible for the increased repression of sit under anaerobic conditions compared to aerobic conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659669" target="_blank">15659669</a>]<br></li>
<li>Our data indicate that Fur controls the class Ib and III ribonucleotide reductases , either directly or indirectly , under anaerobic conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22017966" target="_blank">22017966</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_208">
Teixido , L. , B. Carrasco , J. C. Alonso , J. Barbe , and S. Campoy , 2011 Fur activates the expression of Salmonella enterica path-ogenicity island 1 by directly interacting with the hilD operator in vivo and in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22021388" target="_blank">22021388</a>]<br></label>
<input id="act_similarity_208", type="checkbox">
<div class="ACT_similar">
<li>Fur activates the expression of Salmonella enterica pathogenicity Island 1 by directly interacting with the hilD operator in vivo and in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28553268" target="_blank">28553268</a>]<br></li>
<li>Fur activates the expression of Salmonella enterica pathogenicity island 1 by directly interacting with the hilD operator in vivo and in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23376784" target="_blank">23376784</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26441883" target="_blank">26441883</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23306672" target="_blank">23306672</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24643535" target="_blank">24643535</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26386070" target="_blank">26386070</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27704705" target="_blank">27704705</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24990908" target="_blank">24990908</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27601571" target="_blank">27601571</a>]<br></li>
<li>Teixidó L , Carrasco B , Alonso JC , Barbé J , Campoy S : Fur activates the expression of Salmonella enterica pathogenicity island 1 by directly interacting with the hilD operator in vivo and in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24079299" target="_blank">24079299</a>]<br></li>
<li>Teixido , L. , Carrasco , B. , Alonso , J. C. , Barbe , J. & Campoy , S. Fur activates the expression of Salmonella enterica pathogenicity island 1 by directly interacting with the hilD operator in vivo and in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27886215" target="_blank">27886215</a>]<br></li>
<li>Teixido L , Carrasco B , Alonso JC , Barbe J , Campoy S : Fur activates the expression of Salmonella enterica pathogenicity island 1 by directly interacting with the hilD operator in vivo and in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22017966" target="_blank">22017966</a>]<br></li></div>
<li>Moreover , Fur signiﬁcantly regulates translational hilD -- lacZ fusions in a manner that mirrors the effects on hilA expression , but it does not affect transcription of hilD as monitored by transcriptional lac fusions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17208038" target="_blank">17208038</a>]<br></li>
<li>Therefore , PhoP-depen-dent regulation was not involved in the observed activation of hilA-lacZ by Fur .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21075923" target="_blank">21075923</a>]<br></li>
<li>The reduction in PrtsA and PhilD activation was consistent with earlier data , which showed that Fur regulated SPI-1 by controlling hilD expression when the hilD gene was present ( 35 , 37 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26386070" target="_blank">26386070</a>]<br></li>
<li>Figure 3 shows that Fur regulation of hilA was abrogated in any background when hilD was deleted ; in all hilD strains there was no signiﬁcant difference in hilA expression between the fur and fur strains .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
<li>However , when hilD was deleted in these strains , there was no longer a signiﬁcant effect of Fur overproduction , while HilC overproduction still induced hilD transcription at a level that was approximately the same as the level seen in the hilD background .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
<li>In the Fe2 + - bound form , E. coli Fur represses genes involved in respiration , ﬂagellar chemotaxis , the TCA cycle , glycolysis , methionine biosynthesis , phage DNA packaging , DNA synthesis , purine metabolism , and redox stress resistance .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23376784" target="_blank">23376784</a>]<br></li>
<li>Upon complexing with Fe , 2 + Fur recognises a speciﬁc DNA sequence , the fur box and binds to it .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16023758" target="_blank">16023758</a>]<br></li>
<li>While MntR primarily responds to the Mn concentration and Fur primarily responds to the Fe concentration , both proteins can use the other metal and repress transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659669" target="_blank">15659669</a>]<br></li>
<li>However , it was later revealed that the reporter fusion was to the Fur repressed iroC and not to the hmpA [ 116 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22017966" target="_blank">22017966</a>]<br></li>
<li>* , induced by iron limitation , repressed by ferrated Fur ; * * , induced by iron limitation , required deferrated Fur for induction ( 10 ) was mapped to the 13-min region of the chromosome , based on a 55 % linkage to ent .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8169207" target="_blank">8169207</a>]<br></li>
<li>Under iron-replete conditions , RstA expression retarded Salmonella growth but enabled the Fur protein to repress the target genes beyond the levels which were simply accomplished by iron .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18790861" target="_blank">18790861</a>]<br></li>
<li>This can be illustrated by the reciprocal regulation of sodB , encoding the iron-superoxide dismutase , and sodA , encoding the manganese-superoxide dismutase by Fe ( II ) - Fur which allows the cell to replace iron-superoxide dismutase with manganese-superoxide dismutase when iron levels are reduced .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21722794" target="_blank">21722794</a>]<br></li>
<li>Under these circumstances , FeoB induction by RstA activation could provide the Fur protein with additional Fe ( II ) to repress the iron response beyond the levels normally mediated by the Fur protein ( Fig. 4 and Fig. 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18790861" target="_blank">18790861</a>]<br></li>
<li>The log-phase ATR includes OmpR , the alternative ( T factor RpoS , and the iron response-regulator Fur for induction ( 64 , 67 , 68 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11370771" target="_blank">11370771</a>]<br></li>
<li>Indeed , this operon is controlled by 15 IscR , Fur and OxyR , which are the Fe -- S homeostasis - , the iron - and the H2O2-sensing regulators , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21952637" target="_blank">21952637</a>]<br></li>
<li>and the catabolite repressor -RCB- activator ( Cra ) protein ( Saier & Ramseier , 1996 ; Saier et al. , 1996 ) , nitrogen utilization is controlled by the NtrBC sensor kinase -RCB- response-regulator pair in conjunction with the GlnB and GlnD proteins ( Magasanik , 1996 ; Reitzer , 1996a , b ) , phosphorus utilization is controlled by the PhoR -RCB- PhoB sensor kinase -RCB- response-regulator pair ( Wanner , 1996 ) , iron utilization is regulated by the Fur transcription factor ( Earhart , 1996 ; Gralla & Collado-Vides , 1996 ) and sulfur metabolism is controlled by the CysB transcriptional-activator ( Kredich , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11782505" target="_blank">11782505</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_230">
Patzer , S. I. , and Hantke , K. ( 2001 ) Dual repression by Fe2 - Fur and Mn2 - MntR of the mntH gene , encoding an NRAMP-like Mn2 transporter in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24596096" target="_blank">24596096</a>]<br></label>
<input id="act_similarity_230", type="checkbox">
<div class="ACT_similar">
<li>Dual repression by Fe - Fur and Mn2-MntR of the mntH gene , encoding an NRAMP-like Mn2 transporter in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12379679" target="_blank">12379679</a>]<br></li>
<li>Dual repression by Fe2 - Fur and Mn2-MntR of the mntH gene , encoding an NRAMP-like Mn2 transporter in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12029030" target="_blank">12029030</a>]<br></li>
<li>Dual repression by Fe2 - Fur and Mn2-2 MntR of the mntH gene , encoding an NRAMP-like Mn transporter in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659669" target="_blank">15659669</a>]<br></li></div>
<li>However , Fur lacking iron was also required for the induction of six gene products .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1624426" target="_blank">1624426</a>]<br></li>
<li>Abbreviations : Acn , aconitase ; cAcn , cytoplasmic aconitase ; CRP , cyclic-AMP receptor protein ; Fis , factor for inversion stimulation ; FNR , fumarate -- nitrate reduction regulator ; Fur , ferric uptake regulator ; IRE , iron-responsive element ; IRP-1 , iron-responsive protein 1 ; LB , Luria -- Bertani broth ; LIC , ligation-independent cloning ; PacnA , promoter region of the acnA gene ; PacnB , promoter region of the acnB gene ; SoxR ; superoxide response protein ; S. Typhimurium , Salmonella enterica serovar Typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23637460" target="_blank">23637460</a>]<br></li>
<li>The fact that Mn remains in the reduced state independent of oxygen conditions may explain why MntR had much less effect on anaerobic repression than Fur did .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659669" target="_blank">15659669</a>]<br></li>
<li>However , in the absence of MntR , the promoter was still repressed threefold with 1 M Mn , and this repression was dependent on Fur .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659669" target="_blank">15659669</a>]<br></li>
<li>Most of this repression was dependent on Fur , but MntR could repress the promoter more than twofold in response to Fe .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659669" target="_blank">15659669</a>]<br></li>
<li>To directly test this hypothesis , we monitored expression of the sitABCD and mntH promoters in the presence and absence of MntR and/or Fur .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659669" target="_blank">15659669</a>]<br></li>
<li>Stability of Crp , Fur , RpoS , and PhoP proteins and their decline during growth in the absence of arabinose .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19103774" target="_blank">19103774</a>]<br></li>
<li>Decrease in amounts of Crp , Fur , RpoS , and PhoP proteins as a consequence of growth of 9021 ( Pcrp527 ) , 8848 ( Pfur33 ) , 8956 ( PrpoS183 ) , and 8918 ( PphoPQ107 ) in the absence of arabinose .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19103774" target="_blank">19103774</a>]<br></li>
<li>'' If Fur represses rfrA and rfrB and allows translation of sodB , loss of rfrA and perhaps rfrB should increase expression of sodB in the fur background .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
<li>Table 2 Number of Differentially Expressed Genes in Δfur Differentially Expressed Genes in Δfur Number of Genes `` Fur Repressed '' a Cluster of Orthologous Groups Total Number of Genes `` Fur Activated '' b No COG 39 34 3 23 11 13 6 30 16 3 9 Energy Production and Conversion Cell Cycle Control Amino Acid Metabolism and Transport Nucleotide Metabolism and Transport Carbohydrate Metabolism and Transport Coenzyme Metabolism and Transport Lipid Metabolism and Transport 18 0 16 7 7 4 9 4 6 0 5 0 5 Translation Transcription 46 0 46 11 6 17 9 2 Replication , Recombination , and Repair Cell Wall/Membrane/Envelope Biogenesis Cell Motility Post-Translational Modification , Protein Turnover , Chaperone Functions Inorganic Ion Transport and Metabolism Secondary Metabolite Biosynthesis , Transport , and Catabolism 5 1 3 14 1 10 20 5 15 0 1 1 11 22 9 19 11 7 2 4 General Functional Prediction Only Function Unknown Signal Transduction Mechanisms Intracellular Trafficking and Secretion Defense Mechanisms 4 9 2 5 2 3 1 0 3 1 0 Total 226 72 298 Categorized According to Cluster of Orthologous Groups ( COGs ) a Genes with increased expression in the absence of fur b Genes with decreased expression in the absence of fur conditions is unknown , since the repression of cydAB by H-NS does not appear to occur under anaerobic conditions [ 68 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22017966" target="_blank">22017966</a>]<br></li>
<li>f Fur Negatively Regulates hns and Is Required for the Expression o HilA and Virulence in Salmonella enterica Serovar Typhimurium † Bryan Troxell ,1 ‡ Michael L. Sikes ,1 Ryan C. Fink ,1 ¶ Andres Vazquez-Torres ,2 Jessica Jones-Carson ,2,3 and Hosni M. Hassan1 * Department of Microbiology , North Carolina State University , Raleigh , North Carolina 27695-7615 ,1 and Departments of Microbiology2 and Medicine ,3 University of Colorado Denver , Aurora , Colorado 80045 Received 11 August 2010/Accepted 29 October 2010 Iron is an essential element for the survival of living cells .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21075923" target="_blank">21075923</a>]<br></li>
<li>RstA-Promoted Expression of the Ferrous Iron Transporter Feo under Iron-Replete Conditions Enhances Fur Activity in B Salmonella enterica † Jihye Jeon ,2 ‡ Hyunkeun Kim ,1 ‡ Jiae Yun ,2 Sangryeol Ryu ,2 Eduardo A. Groisman ,3 and Dongwoo Shin1 * Department of Molecular Cell Biology , Samsung Biomedical Research Institute , Sungkyunkwan University School of Medicine , Suwon 440-746 , South Korea1 ; Department of Food and Animal Biotechnology , Department of Agricultural Biotechnology , and Center for Agricultural Biomaterials , Seoul National University , Seoul 151-921 , South Korea2 ; and Department of Molecular Microbiology , Howard Hughes Medical Institute , Washington University School of Medicine , St. Louis , Missouri 631103 Received 30 June 2008/Accepted 31 August 2008 The Fur protein is a primary regulator that monitors and controls cytoplasmic iron levels .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18790861" target="_blank">18790861</a>]<br></li>
<li>The presence of Mn caused transcriptional repression of the sitABCD and mntH promoters primarily via MntR , but Fur was also capable of some repression in response to Mn .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659669" target="_blank">15659669</a>]<br></li>
<li>The mechanism of this regulation is unclear , but the absence of Fur binding sites in the promoters of any of these genes suggests an indirect mode of regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22017966" target="_blank">22017966</a>]<br></li>
<li>JOURNAL OF BACTERIOLOGY , Oct. 1996 , p. 5683 -- / 96 / $ 04.0010 Copyright q 1996 , American Society for Microbiology T he Role of Fur in the Acid Tolerance Response of Salmonella typhimurium Is Physiologically and Genetically Separable from Its Role in Iron Acquisition HOLLY K. HALL1 ,2 AND JOHN W. FOSTER1 * Department of Microbiology and Immunology , College of Medicine ,1 and Department of Medical Technology , College of Allied Health ,2 University of South Alabama , Mobile , Alabama 36688 Received 5 April 1996/Accepted 23 July 1996 The response of Salmonella typhimurium to low pH includes a low-pH protection system called the acid tolerance response ( ATR ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8824613" target="_blank">8824613</a>]<br></li>
<li>Ferric Uptake Regulator-Dependent Antinitrosative Defenses in Salmonella enterica Serovar Typhimurium Pathogenesis Maroof Husain , a Jessica Jones-Carson , a , b Lin Liu , a Miryoung Song , a J. Royden Saah , c Bryan Troxell , d Mary Mendoza , d Hosni Hassan , d Andrés Vázquez-Torresa , e ` Department of Microbiologya and Division of Infectious Diseases , b University of Colorado School of Medicine , Aurora , Colorado , USA ; North Carolina State Laboratory of Public Health , Raleigh , North Carolina , USAc ; Department of Microbiology , North Carolina State University , Raleigh , North Carolina , USAd ; VA Eastern Colorado Health Care System , Denver , Colorado , USAe Herein we report an important role for the ferric uptake regulator ( Fur ) in the resistance of Salmonella enterica serovar Typhi-murium to the reactive nitrogen species produced by inducible nitric oxide ( NO ) synthase in an NRAMP1r murine model of acute systemic infection .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24166960" target="_blank">24166960</a>]<br></li>
<li>JOURNAL OF BACTERIOLOGY , Aug. 1995 , p. 4628 -- / 95 / $ 04.0010 Copyright q 1995 , American Society for Microbiology Fur Regulon of Salmonella typhimurium : Identiﬁcation of New Iron-Regulated Genes RENÉE M. TSOLIS , ANDREAS J. BÄUMLER , IGOR STOJILJKOVIC , AND FRED HEFFRON * Department of Molecular Microbiology and Immunology , Oregon Health Sciences University , Portland , Oregon 97201 Received 20 March 1995/Accepted 1 June 1995 In order to identify genes belonging to the Fur regulon of Salmonella typhimurium , a bank of 10,000 independent S. typhimurium MudJ insertion mutants was screened for lacZ fusions regulated by the iron response-regulator Fur .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7642488" target="_blank">7642488</a>]<br></li>
<li>H-NS binds the region of ftnB and represses transcription ( 46 ) , which indicates a possible mechanism for Fur activation of ftnB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21075923" target="_blank">21075923</a>]<br></li>
<li>Fur activation of SPI1 is not mediated through the Fur-regulated small RNAs RfrA and RfrB , which are the Salmonella ortholog and paralog of RyhB that control expression of sodB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
<li>Taken together , these results suggest that two events that occur upon feoB transcription , the relief of Fur repression and the occurrence of Fnr activation , lead to high-level production of the FeoB transporter when levels of both iron and oxygen are low .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23708131" target="_blank">23708131</a>]<br></li>
<li>This demonstrates that during intracellular growth of S. Typhimurium , the state of oxygen tension and iron valence are adequate for the activation of both Fnr and Fur , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22017966" target="_blank">22017966</a>]<br></li>
<li>In general , the activity of each strain reﬂected the additive effects of Fur - and MntR-mediated repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659669" target="_blank">15659669</a>]<br></li>
<li>Presumably , under the iron-replete conditions of our experiment , repression of rfrA and rfrB by Fur was essentially complete and increasing the concentration of Fur had little effect .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
<li>This indicates inhibition by dip Figure 3 Effects of Fur , Hfq , and manganese on the activity of superoxide dismutases .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22017966" target="_blank">22017966</a>]<br></li>
<li>Cobalt toxicity results in decreased siroheme production , increased expression of the Fur regulon , and decreased activity of Fe-S cluster proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17720790" target="_blank">17720790</a>]<br></li>
<li>Thus , as in E. coli , Fur regulation of sodB is mediated through small RNAs , but serovar Typhimurium encodes two apparently redundant small RNA molecules , RfrA and RfrB , each of which can individually repress sodB in the absence of the other and both of which contribute to Fur regulation of sodB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
<li>The requirement of Fur for the activation of hilA was ap parent as the rate of synthesis of hilA-lacZ was signiﬁcantly reduced in fur compared to 14028s ( P 0.001 , Fig. 6 ) , and lack of Fur resulted in decreased expression of HilA-activated genes ( i.e. , invF and sipC ) ( see Fig .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21075923" target="_blank">21075923</a>]<br></li>
<li>In the absence of Fe and/or Fur , repression of ryhB is relieved , leading to negative regulation of bfr and ftnA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17302823" target="_blank">17302823</a>]<br></li>
<li>The increased expression of hmpA-lacZ by the addition of dip could have been due to inactivation of Fnr , Fur , and/or NsrR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22017966" target="_blank">22017966</a>]<br></li>
<li>When Salmonella is grown under conditions of low levels of oxygen and iron ( Fig. 6A ) , Fur does not repress the feo promoter ( 2 , 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25313398" target="_blank">25313398</a>]<br></li>
<li>On the other hand , under conditions where Fur is inactive ( i.e. , via deletion of fur or iron deprivation ) [ H-NS ] increases and results in repression of hilA ( Fig. 7B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21075923" target="_blank">21075923</a>]<br></li>
<li>Although Fur repression of the iroBCDE loci is known [ 59 ] , iroN is encoded downstream of this operon and is transcribed in the opposite orientation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22017966" target="_blank">22017966</a>]<br></li>
<li>Collectively , full expression of ftnB is dependent on Fur in aerobic and anaerobic conditions , whereas Fnr is a strong activator in the absence of O2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22017966" target="_blank">22017966</a>]<br></li>
<li>NO inhibits transcription of SPI2 effectors encoded in Induction of hmp by NO The expression of the NO-detoxifying Salmonella flavohemoglobin Hmp is positively induced by the transcriptional-regulator Fur in response to nitrosative stress ( 28 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16129704" target="_blank">16129704</a>]<br></li>
<li>Recently , we demonstrated the role of Fur in HilA expression and virulence in S. Typhimurium , which is mediated by the negative regulation of H-NS by Fur under anaerobic conditions [ 29 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22017966" target="_blank">22017966</a>]<br></li>
<li>However , our data rule-out this possibility in ftnB regulation by showing the involvement of Fur in the regulation of ftnB under aerobic conditions , where Fnr is inactive .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22017966" target="_blank">22017966</a>]<br></li>
<li>Despite the fact that Fur is primarily a negative regulator , it can also act as a positive regulator since at least nine proteins require the presence of both Fur and iron for their expression and six proteins require a functional Fur free of iron for their expression [ 43 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16023758" target="_blank">16023758</a>]<br></li>
<li>Furthermore , Fur is required for the proper expression and activation of the antioxidant enzymes , FeSOD and MnSOD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22017966" target="_blank">22017966</a>]<br></li>
<li>However , the data suggest that Hfq plays little or no role in the Fur regulation of hilA , again differentiating SPI1 and sodB regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
<li>In addition , we detected increased expression of hns in Δfur ( Additional file 2 : Table S2 ) , and earlier work detected in vivo binding of Fur to the upstream region of hns [ 29 ] ; this strongly indicates that Fur directly represses hns under anaerobic conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22017966" target="_blank">22017966</a>]<br></li>
<li>Taken together , these data suggest that the presence of metals in the environment induces expression of hilA and that Fur plays a major role in this metal regulation of hilA and therefore SPI1 expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
<li>In the present study , we report that Fur regulates the expression of H-NS and this in-turn regulates the expression of hilA and virulence in mice .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21075923" target="_blank">21075923</a>]<br></li>
<li>This activation is , in part , indirect where Fur represses the expression of hns , which represses the expression of hilA and hilD [ 29 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22017966" target="_blank">22017966</a>]<br></li>
<li>( A ) Conditions that activate Fur ( i.e. , the presence of iron ) repress the expression of hns ( solid arrow ) , and allow the expression of hilA ( blocked dotted arrows ) and increased HilA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21075923" target="_blank">21075923</a>]<br></li>
<li>However , the significance of the increased expression of HscA , IscU and Fur at low temperatures in not apparent .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20806250" target="_blank">20806250</a>]<br></li>
<li>Several observations led to the Hfq regulon members being highlighted in our M9 proteomic analysis : 1 ) Hfq promotes the expression of a large class of ribosomal structural proteins , and we found differential expression of several of these genes in spaceflight ( L7/L12 , L32 , S20 , S13 , S11 , S19 , SA , L14 , L33 , S4 , L4 ) ; 2 ) Hfq regulates the expression of the Fur protein and other genes involved in iron metabolism , and we found that Fur and other iron-related genes are differentially regulated by spaceflight in M9 medium ( Fur , Dps , NifU , FepA ) ; 3 ) Several other proteins encoded by genes belonging to the Hfq regulon were also found in this analysis : NmpC , Tpx , PtsI , PtsH , SucC , LeuB , CysP , DppA , OppA , RpoZ , CsrA , RpoB , NlpB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19079590" target="_blank">19079590</a>]<br></li>
<li>Role of iron , Fur and the sRNAs RyhB1/RyhB2 in regulation of tcf expression As the Fur regulator is activated by iron and the regulation of its target genes is usually iron-dependent , the regulation of tcf by iron levels ( high and low ) was investigated in the WT and fur mutant strains .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26944792" target="_blank">26944792</a>]<br></li>
<li>Because Fur controls iron uptake in many bacteria and there is ﬁerce competition between the pathogen and the host for free iron , an understanding of how iron homeostasis regulates hilA is crucial to our knowledge of the role of Fur in the activation of SPI-1 and virulence in Salmonella .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21075923" target="_blank">21075923</a>]<br></li>
<li>The absence of IRO-28 in the fur-1 mutant ( Fig. 7C ) and its presence in the furH90R mutant ( Fig. 7E ) conﬁrm that Fur acts as a positive regulator of this gene and that FurH90R is iron blind yet active .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8824613" target="_blank">8824613</a>]<br></li>
<li>The data presented illustrate that Fur can positively and negatively regulate gene expression and that regulation can occur in the absence of the iron-sensing function of Fur .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8824613" target="_blank">8824613</a>]<br></li>
<li>The histidines of the iron-uptake regulation protein , Fur .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8824613" target="_blank">8824613</a>]<br></li>
<li>Fur regulation of hilD requires the HilD protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
<li>Fur regulation of HilD does not appear to be simply at the transcriptional or translational level but rather requires the presence of the HilD protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
<li>Rather , Fur activity requires the presence of the HilD protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
<li>This positive control must be mediated by binding of the S. typhimurium Fur protein to the flhD promoter as indicated by the fact that this promoter tests positive in a Fur titration assay .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11932449" target="_blank">11932449</a>]<br></li>
<li>Thus , our data showed that the RstA protein activates feoAB transcription independently of the repression mediated by the Fur protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18790861" target="_blank">18790861</a>]<br></li>
<li>The RstA protein maintained wild-type levels of the Fur protein but exceptionally activated transcription of the feoAB operon encoding the ferrous iron transporter FeoB by binding directly to the feoA promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18790861" target="_blank">18790861</a>]<br></li>
<li>This emphasizes that , in the presence of iron , the feoB gene should be induced by the RstA protein to repress iron-responsive genes and differs from the ﬁnding that iron chelator inactivated the Fur protein to abolish the regulatory effect of RstA on its regulated genes ( Fig. 1B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18790861" target="_blank">18790861</a>]<br></li>
<li>RCF conducted the microarrays , constructed the Fur Logo , and contributed to the editing of the manuscript .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22017966" target="_blank">22017966</a>]<br></li>
<li>The second model suggests that Fur ( or something regulated by Fur ) acts at the level of the DNA to affect the threshold of HilD required to activate the hilD promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
<li>The innate antinitrosative defenses regulated by Fur added to the adaptive response associated with the NO-detoxifying activity of the ﬂavohemoprotein Hmp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24166960" target="_blank">24166960</a>]<br></li>
<li>It should be noted that the feoA promoter mutant strains grown in LB medium still displayed the wild-type levels of Fur repression ( Fig. 3B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18790861" target="_blank">18790861</a>]<br></li>
<li>In contrast , entB expression in the yggX gshA strain was decreased by deferoxamine , indicating the presence of more iron-bound Fur .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18835989" target="_blank">18835989</a>]<br></li>
<li>While the wild-type mntH : : lacZ reporter was strongly repressed by addition of Fe2 at concentrations of 1 M or greater , this repression was diminished but not eliminated when either the Fur protein or the Fur-binding motif in the promoter or both were mutated ( Fig. 4A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12029030" target="_blank">12029030</a>]<br></li>
<li>However , the ssaG mRNA levels displayed by the ssrB mutant were unaffected by fur deletion under the same growth condition , which was in contrast to the Fur repression of ssaG expression occurring in the presence of ssrB ( Fig. 5A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24643535" target="_blank">24643535</a>]<br></li>
<li>Real-time PCR analysis revealed that upon RstA expression , transcriptional repression of the fhuF gene by the FLAG-tagged Fur protein was as efﬁcient as in the strain with the wild-type Fur protein ( see Fig .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18790861" target="_blank">18790861</a>]<br></li>
<li>Fur represses the synthesis of the small RNA RyhB , which under Fe-restricted conditions negatively regulates the synthesis of several Fe-containing proteins , including FtnA and Bfr ( Masse and Gottesman , 2002 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17302823" target="_blank">17302823</a>]<br></li>
<li>While under Fur inactive conditions ( right panel ) , the overexpression of H-NS results in the repression of ftnB and the tdc operon under anaerobic conditions ΔfurΔfnr backgrounds .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22017966" target="_blank">22017966</a>]<br></li>
<li>To reveal potentially novel aspects of the Fur regulon in Salmonella enterica serovar Typhimurium under oxygen conditions similar to that encountered in the host , we compared the transcriptional profiles of the virulent wild-type strain ( ATCC 14028s ) and its isogenic Δfur strain under anaerobic conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22017966" target="_blank">22017966</a>]<br></li>
<li>In addition , the rate of synthesis in the parent strain was greatest under anaerobic conditions due to the active roles of both Fnr and Fur ( Figure 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22017966" target="_blank">22017966</a>]<br></li>
<li>In addition , Fur is known to activate SPI1 via the activation of the positive regulators of SPI1 ( i.e. , HilA and HilD ) [ 38,39 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22017966" target="_blank">22017966</a>]<br></li>
<li>However , E. coli Fur is also known to activate the expression of many genes by either direct or indirect mechanisms , the latter involving repression of a small regulatory RNA , ryhB , which binds to and decreases the stability of its target mRNAs ( Masse et al. , 2003 ; Lee and Helmann , 2007 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21722794" target="_blank">21722794</a>]<br></li>
<li>While OxyR is not present to induce the expression of both sRNAs , Fur is being disabled by sub-millimolar levels of peroxide , releasing the repression of ryhB-1 and ryhB-2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24239962" target="_blank">24239962</a>]<br></li>
<li>In summary , we propose a model where , in response to peroxide stress , the expression of RyhB-1 and RyhB-2 is not only determined by OxyR activation , but also by an exquisite redox balance that depends on the levels of H2O2 ( generation / degradation ) and the concentration of the active form of Fur .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24239962" target="_blank">24239962</a>]<br></li>
<li>However , in response to H2O2 , this Fur repression is abolished and the apoprotein form of IscR upregulates expression of the operon [ 44 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24044554" target="_blank">24044554</a>]<br></li>
<li>They act by increasing the level of HilD protein to overcome HilE , and controlling HilD activity , or in the case of Fur , lowering the threshold required at the promoter , such that HilD activates its own promoter as well as induces expression of HilC and RtsA , which then act to amplify and accelerate SPI1 expression ( Saini et al. 2010a ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22021388" target="_blank">22021388</a>]<br></li>
<li>Slauch and colleagues proposed two models to explain Fur activation of Spi1 : either by indirect control of the HilD protein or by affecting the threshold value of HilD concentration required to activate the hilD promoter ( Ellermeier & Slauch , 2008 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27704705" target="_blank">27704705</a>]<br></li>
<li>Together , these data suggest that Fur might activate SPI1 by reducing the H-NS -- mediated silencing of the hilD promoter region and thereby lowering the threshold of HilD required to activate the hilD promoter , as we originally proposed ( Ellermeier and Slauch 2008 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22021388" target="_blank">22021388</a>]<br></li>
<li>The similarity of the sitABCD promoter with the mntH promoter and the presence of a putative MntR binding site suggested that sitABCD is regulated by MntR in addition to Fur .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659669" target="_blank">15659669</a>]<br></li>
<li>Support for this hypothesis stems from the fact that Fur-regulated genes were induced in response to acidified RNS and by the fact that in the absence of Fur , NO-treated Salmonella were completely killed after 1 h of acid challenge .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18350168" target="_blank">18350168</a>]<br></li>
<li>The addition of FeCl3 to the medium repressed expression in all strains and conditions , indicating that the Fur protein was not damaged ; rather , the increased expression reﬂected a decreased amount of Fe2 bound to the existing Fur .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18835989" target="_blank">18835989</a>]<br></li>
<li>If we disrupted the gene that Fur represses in order to activate hilA , we would expect two things : ( i ) the colonies would appear blue on LB medium with X-Gal , and ( ii ) expression of hilA would be unaffected by the presence or absence of Fur in the mutant background .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
<li>Fur is involved in metal-dependent transcriptional regulation in response to different metal concentrations and the redox state of the bacterial the metabolism of ST24CHX .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25136333" target="_blank">25136333</a>]<br></li>
<li>The Fur metalloprotein could coordinate virulence through the regulation of metabolic processes , and more speciﬁcally , in the case of enteropathogenic bacteria , it could enhance pathogenicity by regulating the acid tolerance response that increases bacterial ﬁtness in the low pH of the stomach ( 36 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24166960" target="_blank">24166960</a>]<br></li>
<li>Our investigations are consistent with a model in which Fur controls an innate mechanism of detoxiﬁcation of NO ( data herein ) , whereas the induction of the ﬂavohemoprotein Hmp in response to the nitrosylation of the NsrR repressor provides adaptive , long-term , and ampliﬁed NO resistance ( 41 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24166960" target="_blank">24166960</a>]<br></li>
<li>The relative activity , obtained by dividing the activity of the relevant strain by the activity of the ryhB acnA parent , is shown in parentheses istry , the Fur iron complex is a target of hydroxyl radical stress , and the action of YggX in preventing Fenton chemistry is most signiﬁcant for superoxide stress and upstream of labile iron effects .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18835989" target="_blank">18835989</a>]<br></li>
<li>It has been shown that the sitABCD operon , encoding a manganese transport system repressed by Fur in the presence of high iron concentrations , is not expressed in the lumen of the small intestine of the mouse but is induced upon invasion ( 25 , 26 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
<li>Production of RyhB is repressed by Fur in the presence of iron , leading indirectly to positive Fur regulation in the cell .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12897017" target="_blank">12897017</a>]<br></li>
<li>This conclusion is based on the ability of the H90R Fur mutant to mediate acid induction of speciﬁc Fur-dependent ASPs and enable induction of acid tolerance in spite of the fact that the mutant Fur can not mediate iron regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8824613" target="_blank">8824613</a>]<br></li>
<li>The fact that ipf expression is upregulated in the presence of an iron chelator and the identification of putative Fur box in the promoter of ipfA strengthen the possibility that Ipf regulation by Fur is direct .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28817673" target="_blank">28817673</a>]<br></li>
<li>Global gene expression proﬁling of intracellular Salmonella 111 centrations , the Fur protein binds to Fe2 + and represses during the intestinal phase of the infection when free iron the expression of genes involved in iron uptake in Salmo - concentrations are thought to be higher ( Gunn et al. , nella ( Tsolis et al. , 1995b ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12492857" target="_blank">12492857</a>]<br></li>
<li>In contrast , the fur mutant , which shows a constitutive expression of iron-uptake mechanisms and lacks the predicted protective effect of the Fur protein in iron-overloading conditions ( Touati et al. , 1995 ) , presents a lower level of cAMP ( Fig. 4a ) when growing in an iron-rich medium ( in the absence of DPD ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11932449" target="_blank">11932449</a>]<br></li>
<li>The combined induction of the iron-acquisition genes and repression of the iron-storage gene strongly suggest that exposure of Salmonella to EWMM induces a FeFur controlled homeostatic reaction in response to the prevailing iron-restriction conditions of the medium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28553268" target="_blank">28553268</a>]<br></li>
<li>This effect appears to be coordinated by the global Fe-regulator , Fur , and incorporates the following effects : induction of iron-uptake pathways ; down-regulation of iron-storage capacity ( along with the probable mobilization of iron stores ) ; induction of the alternative Fe-S cluster manufacturing pathway ; iron rationing with replacement of iron-dependent proteins by manganese-dependent alternatives ; and induction of manganese uptake to supply Mn2 + to the newly-induced Mn-dependent isoenzymes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28553268" target="_blank">28553268</a>]<br></li>
<li>Perhaps the increase in levels of the RyhBs could be indirect , as oxidative-stress and stationary phase decrease the levels of free iron , which leads to inactivation of the Fur repressor ( Varghese et al. , 2007 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27242152" target="_blank">27242152</a>]<br></li>
<li>Supporting this model , the concentrations of H2O2 used herein are sufﬁcient to activate OxyR [ 2 ] and to disrupt the ability of the repressor Fur to control the levels of free iron [ 50 ] , thus relieving the repression of both RyhBs .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24239962" target="_blank">24239962</a>]<br></li>
<li>Iron acquisition genes encoded by the sitABCD operon and the iroA locus were differentially induced in response to spermine NONOate ( table S2 ) , likely reflecting their derepression upon nitrosylation of the iron prosthetic group of Fur [ 38 -- 40 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18350168" target="_blank">18350168</a>]<br></li>
<li>Although Fur appears to contribute to the ATR independently from its role in iron acquisition [ 23 ] , it remains possible that nitrosylation of iron prosthetic groups of Fur in the nitrosative environment of the stomach may induce the ATR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18350168" target="_blank">18350168</a>]<br></li>
<li>Interestingly , both homologues of Tcf , i.e. Cbl and CFA/I , were also shown to be repressed by iron ( Karjalainen et al. , 1991 ; Tomich & Mohr , 2004 ) , suggesting the involvement of the iron regulator Fur .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26944792" target="_blank">26944792</a>]<br></li>
<li>Nevertheless , they all share the common functional feature of mediating positive regulation of their target genes by the Fur repressor and contribute to iron homeostasis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22824499" target="_blank">22824499</a>]<br></li>
<li>E. coli Fur typically acts as corepressor that binds to a specific DNA sequence , the Fur box , when metal-loaded and represses the transcription of downstream genes by hindering the access of RNA polymerase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21722794" target="_blank">21722794</a>]<br></li>
<li>This could be explained by the iron content of the CAS agar plates ( 10 mM ) , which was high enough to activate Fur and to trigger Fur-mediated repression of the sRNAs .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23306672" target="_blank">23306672</a>]<br></li>
<li>Under low oxygen conditions , the soluble reduced form of iron ( Fe2 ) would become available for transport , leading to repression of the Fur regulon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659669" target="_blank">15659669</a>]<br></li>
<li>Taken together , these lines of evidence suggest that the Fur repressor could interfere with binding of the SsrB activator to the ssrB promoter , leading to repression of ssrB transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24643535" target="_blank">24643535</a>]<br></li>
<li>Because of the wide ranging effects of Fur [ 27 ] , it may also contribute to other effects on virulence upon exposure to tetracycline , including increased survival during passage through the acidic conditions of the gastrointestinal tract .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18656407" target="_blank">18656407</a>]<br></li>
<li>Our data indicated that Fur was required for full virulence in mice regardless of the presence or absence of the natural resistance-associated macrophage protein 1 , Nramp1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21075923" target="_blank">21075923</a>]<br></li>
<li>The lack of FeSOD in Δfur was of no surprise , as previous studies in E. coli [ 83,84 ] have established that Fur is indirectly required for the translation of sodB via its repression of the small RNA , ryhB , which works in conjunction with the RNA chaperon protein , Hfq [ 88,89 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22017966" target="_blank">22017966</a>]<br></li>
<li>If small RNAs are involved in the Fur regulation of hilD , deletion of hfq could increase expression of hilA in the fur background .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
<li>Finally , we demonstrated that in the absence of H-NS the expression of hilA was independent of Fur .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21075923" target="_blank">21075923</a>]<br></li>
<li>Effect of Fur on the anaerobic transcriptome of S. Typhimurium Under anaerobic conditions , the absence of fur resulted in the differential expression of 298 genes ( Additional File 2 : Table S2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22017966" target="_blank">22017966</a>]<br></li>
<li>Thus , to further confirm Ipf regulation by Fur , we studied the expression of ipfA in the presence of a Fe chelator , Dip ( 2,2 ' - dipyridyl ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28817673" target="_blank">28817673</a>]<br></li>
<li>In its most common mode of regulation , in the presence of Fe2þ , Fur binds to the Fe2þ ion , and represses expression of its target genes [ 20,25 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23040276" target="_blank">23040276</a>]<br></li>
<li>( B ) In contrast , in the absence of Fur , levels of SsrB gradually increase during environmental acidiﬁcation , which then leads to the gradual induction of SPI-2 expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24643535" target="_blank">24643535</a>]<br></li>
<li>Given the global importance of Fur in intracellular iron management , a fur mutant of S. typhimurium might be compromised in its ability to resist killing within macrophages ; conversely the modest upregulation of Hmp in the absence of Fur might confer a selective advantage .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17526833" target="_blank">17526833</a>]<br></li>
<li>To conﬁrm the role of Fur in regulating hns , we used qRT-PCR to quantify the expression of hns in the presence or absence of Fur .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21075923" target="_blank">21075923</a>]<br></li>
<li>In the presence of iron , Fur negatively controls transcription of a small RNA , RyhB , which facilitates degradation of the sdhCDBA mRNA encoding succinate dehydrogenase ( 30 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20008066" target="_blank">20008066</a>]<br></li>
<li>Since enterochelin/salmochelin production is necessary for NE-enhanced growth of S. Typhimurium ( Bearson et al. 2008 ) , the accumulation of these siderophores in the presence of NE increases iron availability which concomitantly reduces the expression of genes encoding iron acquisition proteins via the iron regulator Fur .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26589218" target="_blank">26589218</a>]<br></li>
<li>In the presence of high levels of iron , the Fur protein serves as a negative regulator of genes involved in iron utilization [ 18 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18554972" target="_blank">18554972</a>]<br></li>
<li>In the presence of high iron concentrations , the activated Fur represses transcription of genes whose promoter contains the consensus FUR box sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22824499" target="_blank">22824499</a>]<br></li>
<li>Therefore , these results suggested that the presence of the iron regulator Fur inhibited the expression of the tcf ﬁmbrial cluster .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26944792" target="_blank">26944792</a>]<br></li>
<li>The oxygen tension within tomato fruits under normal storage conditions , however , has been shown to be near that of ambient air ( 36 ) , suggesting the presence of some additional environmental signal for the induction of the Fur regulon in our experiments .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25795672" target="_blank">25795672</a>]<br></li>
<li>One explanation could be that the combined chelation power of the millimolar levels of glutathione and deferoxamine limit the availability of iron to Fur more than that limited by the presence of deferoxamine alone .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18835989" target="_blank">18835989</a>]<br></li>
<li>This fact indicates that , regardless of cAMP , the Fur protein must be involved in the control of the ﬂagellar regulon and that its presence is absolutely required to achieve an optimal expression of the ﬂhD promoter , despite there being enough intracellular cAMP concentration .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11932449" target="_blank">11932449</a>]<br></li>
<li>This FeoB induction resulted in increased ferrous iron uptake , which associates with the Fur protein because lack of RstA-dependent transcriptional activation of the feoA promoter and feoB-deletion abolished repression of the Fur target genes by the RstA protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18790861" target="_blank">18790861</a>]<br></li>
<li>Supplying iron , however , reproduced the regulatory effect of RstA on the Fur protein : the fhuA transcripts of wild-type cells were ap proximately sixfold lower in the presence of 20 M ferrous sulfate ( FeSO4 ) than in its absence , and expression of the RstA protein reduced these mRNA levels even further ( Fig. 1C ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18790861" target="_blank">18790861</a>]<br></li>
<li>Despite the enhanced expression of the hilA-lacZ fusion following induction of the Fur protein with IPTG at 37 ° C , induction at 42 ° C did not increase expression of hilA-lacZ even though the levels of the Fur protein under the two sets of conditions were strikingly similar ( Fig. 4B , right panel ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26386070" target="_blank">26386070</a>]<br></li>
<li>In contrast , in the absence of Fur repression on the ssrB promoter , levels of SsrB protein would gradually increase over the course of environmental acidiﬁcation ( Fig. 6B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24643535" target="_blank">24643535</a>]<br></li>
<li>Titration of FeCl3 into the medium with 80 M CoCl2 resulted in increasing repression of entB transcription , showing that the Fur protein had not been damaged by the presence of cobalt .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17720790" target="_blank">17720790</a>]<br></li>
<li>Indeed , when S. Typhimurium is within macro-phages , repression of the Fur regulated iroBCDE promoter occurs regardless of the presence of the host metal transporter Nramp1 [ 27,28 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22017966" target="_blank">22017966</a>]<br></li>
<li>The presence of the Fur box in the promoter region of ryhB-1 , explains the Fur-mediated transcriptional regulation of RyhB-1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22824499" target="_blank">22824499</a>]<br></li>
<li>Determining the mechanism of Fur activation of HilD is an important step in understanding the regulation of the SPI1 T3SS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
<li>Taken together , these results illustrate that the iron-sensing Fur regulator negatively controls expression of SPI-2 genes during acidiﬁcation and a lack of this regulation enables the SPI-2 genes to be induced , even under intermediate acidic conditions that would not normally allow for SPI-2 induction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24643535" target="_blank">24643535</a>]<br></li>
<li>Therefore , Fur repression of SPI-2 genes inside macrophages raises the possibility that iron is present inside the Salmonella-containing phagosome .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24643535" target="_blank">24643535</a>]<br></li>
<li>We hypothesized that the RstA-promoted Fur-depen-dent repression of iron-responsive genes might be due to an increase in the Fur protein levels .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18790861" target="_blank">18790861</a>]<br></li>
<li>We hypothesized that growth of wild-type Salmonella under high-iron conditions might increase Fur activity to the levels displayed by the RstA-express-ing strain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18790861" target="_blank">18790861</a>]<br></li>
<li>This study showed that both sRNAs are regulated by iron and by Fur , but their expression differed considerably depending on the growth-phase , the oxidative-stress and iron levels in the medium , suggesting differences in their regulation ( PadalonBrauch et al. , 2008 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23306672" target="_blank">23306672</a>]<br></li>
<li>In contrast to the ASPs , iron-regulated proteins marked with squares and numbered in Fig. 7C were not ASPs but were overexpressed in both the fur-1 and iron-blind fur-9 mutants , conﬁrming that Fur negatively regulates these genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8824613" target="_blank">8824613</a>]<br></li>
<li>In contrast to fur , the sRNAs rfrA and rfrB were induced in low iron conditions , where the regulator Fur is inactive .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23306672" target="_blank">23306672</a>]<br></li>
<li>It remains to be established to what extent this is coordinated with the NikR and Fur mediated regulation in this organism lacking the NikABCDE nickel uptake system .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21722794" target="_blank">21722794</a>]<br></li>
<li>This was consistent with our observation that pyruvate levels in the medium were altered in response to iron concentration and that this response required the Fur protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10869064" target="_blank">10869064</a>]<br></li>
<li>In this respect , it is worth noting that the S. typhimurium ﬂhD promoter tested positive , in both qualitative and quantitative assays , in the FURTA test used to determine if the Fur protein binds to a given promoter in vivo ( Stojilkovic et al. , 1994 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11932449" target="_blank">11932449</a>]<br></li>
<li>Under the above conditions in which Fur repression may be relieved , both the induced RyhB-1 and RyhB-2 can form base pairing with individual target mRNAs with the aid of Hfq in an antisense manner .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27242152" target="_blank">27242152</a>]<br></li>
<li>As can be seen by the results presented in Fig. 3 , the Crp , Fur , and PhoP proteins are very stable and not subject to breakdown , whereas the RpoS protein displays no stability in the log phase ( 59 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19103774" target="_blank">19103774</a>]<br></li>
<li>In agreement with previous studies ( Nielsen et al. , 1973 ; Imamura et al. , 1996 ) , the higher level of Fe ( II ) in S. typhimurium Fur cells could stimulate the activity of the cyclic phosphodiesterase encoded by cpdA which would then produce a decrease in the intracellular cAMP level .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11932449" target="_blank">11932449</a>]<br></li>
<li>Fur , the primary iron regulatory protein in Salmonella , can activate SPI-1 through HilD in response to the high concentration of Fe in the small intestine [ 26 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28575106" target="_blank">28575106</a>]<br></li>
<li>Genes under fur control require the presence in their promoters of at least three contiguous NAT ( A -RCB- T ) AT-like hexamers , in either direct or inverse orientation , to which the Fur protein binds ( Escolar et al. , 1999 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11932449" target="_blank">11932449</a>]<br></li>
<li>This may be supported by observations in E. coli where levels of fes or ydiE mRNA , genes controlled by Fe -- Fur , were found to be the same in fur mutants as in the wild-type strain when treated with H2O2 [ 47 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16023758" target="_blank">16023758</a>]<br></li>
<li>These data were consistent with a speciﬁc role for glutathione in facilitating the binding of iron by Fur or in maintaining an environment in which iron remains bound to Fur .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18835989" target="_blank">18835989</a>]<br></li>
<li>In our model , the lack of glutathione results in increased iron available to Fenton chemistry , resulting in increased hydroxyl radical stress , which can damage Fur .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18835989" target="_blank">18835989</a>]<br></li>
<li>A comparison of b-galactosidase activity in the presence and absence of pMH152 showed that these seven transcriptional fusions were indeed regulated by Fur .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7642488" target="_blank">7642488</a>]<br></li>
<li>The OMP profile showed that the iron-uptake OMPs , repressed by Fur , were down regulated in presence of arabinose and up regulated in absence of arabinose ( data not shown ; see reference [ 29 ] ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20585446" target="_blank">20585446</a>]<br></li>
<li>Seven proteins , indicated by their identification numbers in Fig. 1C and D , remained regulated by metal chelation with DTPA even in the absence of Fur .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1624426" target="_blank">1624426</a>]<br></li>
<li>Finally , we identified new target genes regulated by Fur in S. Typhimurium , and our data support the increasing evidence of enhanced H-NS expression in Δfur .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22017966" target="_blank">22017966</a>]<br></li>
<li>qRT-PCR analysis that was applied for ipfD exhibited a negative regulatory role for Fur and showed moderately ( two-fold or less ) lower transcription of ipfD in the absence of Lrp , RpoS , SoxR , PhoP , OmpR , and OxyR compared to the wild-type background ( Fig 7E ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28817673" target="_blank">28817673</a>]<br></li>
<li>To account for such multi-factorial effects , the rule for H-NS was defined such that H-NS was turned ` OFF ' only in the presence of all five repressors ( HilD , PhoP , SlyA , IHF and Fur ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24079299" target="_blank">24079299</a>]<br></li>
<li>Likewise , Fe in the medium repressed transcription of both sit and mntH primarily via Fur , although MntR was also involved in this response .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659669" target="_blank">15659669</a>]<br></li>
<li>For example , at higher Mn concentrations , Fur causes a two - to threefold reduction in sit transcription in both the presence and absence of MntR ( Fig. 2A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659669" target="_blank">15659669</a>]<br></li>
<li>β-Galactosidase synthesis was measured from a fusion of each promoter with lacZ in both wild-type ( wt ) and fur ( Fur − ) strains growing in LB medium in the absence or presence of either glucose or DPD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11932449" target="_blank">11932449</a>]<br></li>
<li>Fur has recently been confirmed as a weak repressor of hmp transcription , and a fur mutant was also compromised for survival within macrophages even in the absence of elevated NO levels in non-stimulated macrophages .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17526833" target="_blank">17526833</a>]<br></li>
<li>We also re-examine the role of Fur and demonstrate that a fur mutant is compromised for survival within macrophages even in the absence of elevated NO levels in non-stimulated macrophages .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17526833" target="_blank">17526833</a>]<br></li>
<li>Decreased expression of cellular targets known to the negatively regulated by Fur , including those involved in glycolysis was also noted ; for example , phosphofructokinase ( pfkA ) , which was detected as down-regulated at the RNA and protein levels .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25136333" target="_blank">25136333</a>]<br></li>
<li>Interestingly , the Fur regulon was deﬁned in S. Typhimurium using bacteria grown under anaerobic conditions , and this analysis identiﬁed regulated genes not known to be controlled by Fur when bacteria were grown in the presence of oxygen ( 35 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25795672" target="_blank">25795672</a>]<br></li>
<li>induced by hydrogen peroxide , weakly induced by EDTA and unresponsive to paraquat , consistent with the presence of Fur and OxyR binding sites in the promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10844693" target="_blank">10844693</a>]<br></li>
<li>Fe in the medium greatly repressed transcription of sit and mntH only when both wild-type Fur and the Fur binding site were present .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659669" target="_blank">15659669</a>]<br></li>
<li>Thus both the Fur protein and the Fur-binding motif are involved in Fe2 - dependent , but not in acid-dependent , regulation of mntH .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12029030" target="_blank">12029030</a>]<br></li>
<li>Except for slight differences in dose response among the three Fur circuit mutants in Fig. 4A , our data are consistent 2 with the hypothesis that residual Fe repression of mntH in Fur protein and/or Fur-binding motif mutants is straightforward regulation by MntR , acting through the MntR-binding motif using Fe2 as a cofactor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12029030" target="_blank">12029030</a>]<br></li>
<li>Most importantly , repression by Fur could still be observed for promoters mutated in the MntR binding site , and repression by MntR could still be observed for promoters mutated in the Fur binding site , suggesting that the effects of mutations on or differences in inherent promoter strength did not interfere with our ability to determine the relative roles of Fur and MntR in regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659669" target="_blank">15659669</a>]<br></li>
<li>Nitrosylation of cysteines or iron cofactors in OxyR , SoxR , and NorR activates transcription , whereas the iron-nitrosyl complexes in NsrR and Fur derepress gene expression ( 12 , 13 , 15 , 18 , 39 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22311927" target="_blank">22311927</a>]<br></li>
<li>Holmes K , Mulholland F , Pearson BM , Pin C , McNicholl-Kennedy J , Ketley JM and Wells JM : Campylobacter jejuni gene expression in response to iron limitation and the role of Fur .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20003355" target="_blank">20003355</a>]<br></li>
<li>This result contrasts with the example of Bradyrhizobium japonicum , in which iron-dependent regulation of heme biosynthesis involves both Fur and a Fur-like protein Irr : Fur regulates hemA and Irr regulates hemB in response to iron ( 11 , 12 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12081946" target="_blank">12081946</a>]<br></li>
<li>By contrast , the ferric uptake regulator ( Fur ) does not appear to be involved in this process because an increase in iron would promote Fur dimerization and repression of its target promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23690578" target="_blank">23690578</a>]<br></li>
<li>Furthermore , isc mutants accumulated increased levels of pyruvate in the growth medium in response to exogenously added iron ( FeCl3 ) , and this response required a functional ferric uptake regulator , Fur .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10869064" target="_blank">10869064</a>]<br></li>
<li>This acid control also appears to be iron independent in that a Fur superrepressor and the iron-blind FurH90R still regulated hyd in response to pH. Thus , Fur appears to sense acid stress and affect the expression of some genes in an apparent iron-independent fashion .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8824613" target="_blank">8824613</a>]<br></li>
<li>Fur associates with Fe2 to repress transcription of genes involved in iron uptake and to activate expression of genes implicated in the defense against oxygen toxicity ( reviewed in references 33 and 35 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12081946" target="_blank">12081946</a>]<br></li>
<li>Fur regulates ﬁmbrial , ﬂagella , and SPI encoded genes in Salmonella Previous reports have demonstrated that Fur , in addition to regulating the cellular gene expression in response to the iron availability and its uptake , also regulates expression of genes involved in pathogenesis [ 17,22 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23040276" target="_blank">23040276</a>]<br></li>
<li>There was transcriptional induction of hilD in response to Fur , and this required the hilD promoter region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
<li>( B ) Effects of Fur and MnCl2 - Cell-free extracts were prepared from anaerobically grown cultures as in ( A ) except that 1 mM MnCl2 was added to the media .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22017966" target="_blank">22017966</a>]<br></li>
<li>The ferric uptake regulator , Fur , monitors the status of the labile iron pool and acts as a transcriptional-repressor when bound to Fe2 ( reviewed in reference 19 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18835989" target="_blank">18835989</a>]<br></li>
<li>Expression levels of the feoB gene change in response to iron and oxygen , which results from the action of the iron-sensing transcriptional-regulator Fur ( 9 ) and the oxygen-sensing transcriptional-regulator Fnr ( 10 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23708131" target="_blank">23708131</a>]<br></li>
<li>However , given that the Fur and Fnr regulators can efﬁciently control feoB mRNA levels in response to different levels of oxygen and iron , the reason why Salmonella also controls FeoB levels using the FeoC protein remains unknown .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25313398" target="_blank">25313398</a>]<br></li>
<li>This sequence , known as the Fur box , seems to be widespread in bacteria since its presence and functionality have been described in the promoter of iron-regulated genes of several bacterial species belonging to groups as different as Entero-bacteriaceae , Pseudomonadaceae , Neisseriaceae and Gram-positive bacteria ( Escolar et al. , 1999 ; Ratledge & Dover , 2000 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11932449" target="_blank">11932449</a>]<br></li>
<li>Studies on the other known Mn transport system of Salmonella , MntH , showed that Fur and MntR are important in regulation in response to Fe and Mn , and this regulation involved Fur and MntR binding sites in the promoter region of mntH ( 19 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659669" target="_blank">15659669</a>]<br></li>
<li>[ 47 ] Mukhopadhyay , P. , Zheng , M. , Bedzyk , L.A. , LaRossa , R.A. and Storz , G. ( 2004 ) Prominent roles of the NorR and Fur regulators in the Escherichia coli transcriptional response to reactive nitrogen species .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16023758" target="_blank">16023758</a>]<br></li>
<li>It is thought that in the small intestine , conditions are anaerobic and therefore replete with Fe2 , leading to an active Fur protein and the repression of genes involved in uptake of iron and other metals .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
<li>Proteins involved in nutrient transport ( Crr , DppA , Fur and SufC ) , together with ones involved in the complementa-ry processes of degradation and energy metabolism ( GcvT , GpmA and RibB ) form the other important group of proteins found here to be induced under bioﬁlm conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23376784" target="_blank">23376784</a>]<br></li>
<li>To support this hypothesis , we expected reporter activity to be similar in Δfnr and Figure 7 Representation depicting the role of Fur and H-NS in the regulation of ftnB and the tdc operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22017966" target="_blank">22017966</a>]<br></li>
<li>These results are summarized in the diagram depicted in Fig. 7 , where the presence of active Fur ( i.e. , Fur-Fe2 ) results in decrease in [ H-NS ] and increased the expression of hilA ( Fig. 7A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21075923" target="_blank">21075923</a>]<br></li>
<li>Thus , the increase in isrE and ryhB levels at stationary phase and under oxidative-stress conditions could be indirect , possibly due to a decrease in the levels of free iron that in turn leads to a decrease in the activity of Fur as a transcriptional-repressor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18267966" target="_blank">18267966</a>]<br></li>
<li>The Zur protein belongs to the Fur family of metalloregulatory proteins , and in the presence of Zn2 , it binds to a nearly perfect palindrome found in this region , blocking both znuA and znuCB transcription ( 17 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12117991" target="_blank">12117991</a>]<br></li>
<li>AcnB was assayed in a ryhB acnA background to negate regulatory effects of Fur in the mutant strains , and eliminate activity of the oxygen-stable AcnA protein ( Masse & Gottesman , 2002 ; Thorgersen & Downs , 2007 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19118370" target="_blank">19118370</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_424">
However , mutation of the MntR-binding motif alone abolishes Mn2 repression ( Fig. 4D ) ; hence the possible interaction of a Mn2 - liganded Fur protein with the Fur-binding motif does not appear to be 2 sufﬁcient to mediate Mn repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12029030" target="_blank">12029030</a>]<br></label>
<input id="act_similarity_424", type="checkbox">
<div class="ACT_similar">
<li>Further , mutation of the MntR protein and Fur-binding motif also abolishes Mn2 repression ( Fig. 4F ) ; hence the Fur protein does not appear to mediate repression by binding to the MntR-binding motif .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12029030" target="_blank">12029030</a>]<br></li></div>
<li>The most direct conclusion based on the present data is that residual Mn2 repression is mediated through the Fur protein , but the cis-acting sequence required for Mn2 - cofactored Fur to be effectiv contains elements from both the Fur - and MntR-binding motifs .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12029030" target="_blank">12029030</a>]<br></li>
<li>There is the formal possibility that repression involves some other protein which is expressed only in the presence of Fur ; however the same uncertainties as to the cis-acting binding motif apply to it as well .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12029030" target="_blank">12029030</a>]<br></li>
<li>Although the Fur-box is located proximally downstream of 1 , it is unlikely to regulate the Mn2 - dependent transcription of mntH because Fur responds to Fe2 but not Mn2 ( 18 , 19 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24596096" target="_blank">24596096</a>]<br></li>
<li>However , the predominant effect of Fur is to activate SPI1 , so the transcriptional effect on hilC seems to have little phenotypic effect .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
<li>A second retarded band seems to appear as protein concentration increases , likely indicating the presence of an additional Fur recognition sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20593029" target="_blank">20593029</a>]<br></li>
<li>Because Fur is thought to be capable of acting as only a repressor ( 27 ) , the ﬁrst model posits that Fur represses a negative regulator to `` activate '' SPI1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
<li>It is perhaps reasonable to expect that a regulator as central to the cell as Fur will exhibit different types of control over the many genes that it regulates .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1624426" target="_blank">1624426</a>]<br></li>
<li>These data suggested that a primary contributor to the inactivation of Fur was oxidative damage .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18835989" target="_blank">18835989</a>]<br></li>
<li>For example , if Fur negatively regulates the expression of a second negative regulator , the targets of that second regulator will appear to be under positive control by Fur .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1624426" target="_blank">1624426</a>]<br></li>
<li>It appears that the binding of Fe ( II ) to Fur inactivates Fur induction of IRO-28 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8824613" target="_blank">8824613</a>]<br></li>
<li>Consequently , if Fur acts as a simple repressor of the factor that controls HilD , as it does with ryhB controlling sodB , then the Fur binding site controlling the negative regulator must be far from consensus , leading to weak binding of Fur under normal laboratory conditions ; overproduction of Fur leads to further repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
<li>In E. coli hydroxiperoxidase mutants treated with micromolecular levels of hydrogen peroxide , Fur is unable to control the free iron levels since oxidation of Fe to Fe by hydrogen peroxide inactivates its 2 + 3 + repressor activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20593029" target="_blank">20593029</a>]<br></li>
<li>Further investigations are needed in order to address how Fur regulates the biosynthesis of heme proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24166960" target="_blank">24166960</a>]<br></li>
<li>GHS index [ adapted from Gill et al. ( 2001 ) ] Score Criteria of the health index 5 Mouse bright eyed and alert , has a smooth coat with a sheen , responds to a stimulus , shows interest in its environment Fur slightly ruffed , a loss of sheen to the coat , mouse remains alert and active 4 Fur noticeably ruffed , parts of coat form clumps , less interested in environment outside of cage 3 Mouse hunched over and lethargic , little interest shown in environment , fur clumped with signs of hyperventilating when handled 2 Mouse non-reactive to stimulus , fur has a bottle-brush appearance , mouse hunched over , preferring to sleep ; cold to touch .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27902414" target="_blank">27902414</a>]<br></li>
<li>The elevated expression of Fur in response to hydrogen peroxide stress will limit the amount of intracellular iron , thereby acting to reduce oxidation of biomolecules due to the generation of hydroxyl radicals ( Varghese et al. , 2007 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21722794" target="_blank">21722794</a>]<br></li>
<li>Thus , following colonization of lymphoid tissues , the Fur , Crp , PhoPQ , and/or RpoS proteins cease to be synthesized due to the absence of arabinose such that attenuation is gradually manifest in vivo to preclude induction of diseases symptoms .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19103774" target="_blank">19103774</a>]<br></li>
<li>However , we have determined that Fnr alone contributes to the activation of ftnB and that Fur is required for full induction of the gene , with Fnr exhi-biting a more pronounced role .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22017966" target="_blank">22017966</a>]<br></li>
<li>Since rtsA , for example , responds to oxygen in the absence of HilD , there must be additional oxygen regulation that is independent of Fur .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
<li>While it is plausible that inactivation of Bfr in S. Typhimuruium results in elevated free Fe leading to Fur repression of Fe acquisition and lower total Fe , we have found that the addition of a fur mutation ( which derepresses Fe uptake ) to an S. Typhimurium bfr mutant does not lead to an increase in total Fe , suggesting that the low total Fe content of the bfr mutant is not an indirect consequence of reduced Fe uptake .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17302823" target="_blank">17302823</a>]<br></li>
<li>A simple interpretation of this result is that more Fe ( II ) was available to-bind Fur in the presence of accumulated YggX .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12670952" target="_blank">12670952</a>]<br></li>
<li>Fur , when bound to Fe2 , binds to DNA sites in the promoters of genes which it directly represses ( 27 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
<li>The type of Fe-S cluster in S. Typhimurium NsrR has not yet been determined , but the ability of NsrR to respond to very low NO · concentrations in comparison to NorR , SoxR or Fur ( Fig. 1 ) , suggests that the NsrR Fe-S cluster in NsrR is configured to optimize NO · responsiveness .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22831173" target="_blank">22831173</a>]<br></li>
<li>Because exposure to low pH also leads to oxidative-stress , it is thus possible that the unifying action of Fur lies in its protection against oxidative damage .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16023758" target="_blank">16023758</a>]<br></li>
<li>This result is consistent with a low affinity association that might be expected for a ligand to allow the release of Fe ( II ) to relevant cellular processes such as Fur regulation and [ Fe-S ] cluster synthesis/repair .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12670952" target="_blank">12670952</a>]<br></li>
<li>We then examined the effects of deleting fur , overproducing Fur , and overproducing HilC , which is known to independently activate hilD transcription ( 16 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
<li>In Salmonella , for example , NO-Fe-Fur derepresses transcription of hmp ( 94 ) , whereas in E. coli hmp is derepressed by NO-Fe-Fnr and induced in response to intracellular nitrosylated homocysteine levels and MetR binding to the glyA-hmp intergenic region ( 60 , 95 , 96 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26443513" target="_blank">26443513</a>]<br></li>
<li>Like RpoS , Fur is also involved in the acid tolerance response of log phase cells [ 14,43 ] , and predominantly responds to organic acid stress [ 6 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16023758" target="_blank">16023758</a>]<br></li>
<li>We reason that the chromosomal sequence upstream of UTR1 , which is absent in pYS1300 , may still be required for MntR and Fur to exert their effects .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24596096" target="_blank">24596096</a>]<br></li>
<li>The FtnB protein , characterized for the ﬁrst time in the present study , appears to possess a number of characteristics that distinguish it from Bfr and FtnA : ( i ) despite a modest contribution to total Fe storage capacity , FtnB is necessary for efficient repair of the Fe-sulphur clustercontaining enzyme aconitase following oxidative damage ; ( ii ) FtnB ( like Dps , but in contrast to FtnA and Bfr ) is expressed during Fe-restricted conditions and repressed by Fur ; ( iii ) FtnB is required for full Salmonella virulence ; and ( iv ) in the absence of Dps , Bfr and FtnA , FtnB appears to exacerbate oxidative-stress .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17302823" target="_blank">17302823</a>]<br></li>
<li>Because Fur protein is required for the ATR and metallated Fur is considered the active form that represses iron-regulated genes , we examined whether intracellular iron concentrations might be critical for adaptation to acid stress .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8824613" target="_blank">8824613</a>]<br></li>
<li>Fur regulation of bfd suggests that when iron is low in the cell , bfd would be derepressed and its product would function to mobilize iron from bacterial storage proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12897017" target="_blank">12897017</a>]<br></li>
<li>While we know that Fur does not respond to oxygen concentration directly , under high-oxygen conditions iron is oxidized to insoluble Fe3 , leading to decreased intracellular iron and a lack of Fur binding the DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
<li>First , if a protein was normally induced by excess iron , it became uninducible in thefur mutant , suggesting that Fur may complex with iron to activate the transcription of some genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1624426" target="_blank">1624426</a>]<br></li>
<li>Iron represses feoB transcription at acidic pH. When bacteria are grown in the presence of iron , the Fur protein associates with Fe2 to repress transcription of genes involved in iron acquisition ( 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19801407" target="_blank">19801407</a>]<br></li>
<li>The Fe ( II ) - associated Fur protein resulting from sufﬁcient levels of intracellular iron binds to its speciﬁc DNA sequences on the target promoters to repress transcription of genes encoding proteins that are involved in iron acquisition ( 1 , 9 , 11 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18790861" target="_blank">18790861</a>]<br></li>
<li>This result indicated that in the presence of glutathione , more iron-bound Fur was present .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18835989" target="_blank">18835989</a>]<br></li>
<li>However , evidence exists that Fur regulates specific genes differently in the presence of ferrous iron or manganese [ 7 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22017966" target="_blank">22017966</a>]<br></li>
<li>Here we show that Fur regulates HilD posttranslationally , leading to induction of the SPI1 T3SS in high-iron conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
<li>One interpretation of this result is that in the former case , there was little iron present to-bind Fur , and in the latter , iron was present but was not available to Fur .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18835989" target="_blank">18835989</a>]<br></li>
<li>Taken together , these results show that , in a strain accumulating YggX , a lower level of exogenous iron was needed to repress the Fur regulon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12670952" target="_blank">12670952</a>]<br></li>
<li>Con-2 + sidering the low O2 conditions in the small intestine may contribute to the stability of Fe , 2 + whether low O2 indirectly influences Fur regulation on SPI-1 needs to be further investigated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28575106" target="_blank">28575106</a>]<br></li>
<li>In addition to providing a means to control Fur activity , the increase in Fe ( II ) uptake via the FeoB protein appeared to be a source of a Salmonella growth defect under iron-replete conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18790861" target="_blank">18790861</a>]<br></li>
<li>Fur repressed ssrB expression both inside macrophages and under acidic conditions , which we ascribe to the direct binding of Fur to the ssrB promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24643535" target="_blank">24643535</a>]<br></li>
<li>Under ironreplete conditions , dimeric Fe2 - bound Fur binds to consensus DNA sequences and represses the transcription of iron uptake systems ( 53 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25107963" target="_blank">25107963</a>]<br></li>
<li>Although Fur responds primarily to iron and MntR responds primarily to manganese , each metalloregulatory protein also represses transcription in the presence of the other metal , albeit at lower efﬁciency .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659669" target="_blank">15659669</a>]<br></li>
<li>For it example , will be useful to examine type C and D loci for the presence or absence of a Fur box , the identified consensus DNA sequence to which iron-complexed Fur binds ( 15 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1624426" target="_blank">1624426</a>]<br></li>
<li>When iron is available , Fur binds iron , and the iron-Fur complex binds to a DNA sequence referred to as the Fur box in the promoter region of iron regulated genes , thereby repressing transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18554972" target="_blank">18554972</a>]<br></li>
<li>Because H-NS binds to AT-rich DNA and represses transcription through topological constraint ( Lucchini et al. , 2006 ; Navarre et al. , 2006 ) , the nitrosylation of Fur could influence the regulation of horizontally acquired Salmonella SPI1 and SPI2 genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21833325" target="_blank">21833325</a>]<br></li>
<li>Under iron-limiting condition , RyhB is released from the repression by the Fur regulator , and begins to downregulate expression of non-essential iron-containing proteins ( Masse and Gottesman 2002 ; Masse et al. 2005 ; Tjaden et al. 2006 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23831078" target="_blank">23831078</a>]<br></li>
<li>When iron is scarce , Fur repression of ryhB transcription is abated , and the increase of RyhB transcripts leads to a decrease in expression of genes that encode nonessential iron-using proteins ( Masse and Gottesman , 2002 ; Masse et al. , 2005 ; Tjaden et al. , 2006 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27242152" target="_blank">27242152</a>]<br></li>
<li>Received 25 October 2012 Revised 22 December 2012 Accepted 4 January 2013 INTRODUCTION Fur ( ferric uptake regulator ) is a regulatory protein that is active in the presence of iron ( and other metals ) and its principal function is to repress the transcription of genes implicated in iron acquisition and transport ( Ernst et al. , 1978 ; Hantke , 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23306672" target="_blank">23306672</a>]<br></li>
<li>Fur is also known to assist in mounting ATR responses ( Hall and Foster , 1996 ) , in the induction of SodB expression ( Tsolis et al. , 1995 ) , and is evidently needed for the regulation of cAMP levels ( Campoy et al. , 2002 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15790293" target="_blank">15790293</a>]<br></li>
<li>Expression from PiroBCDE is under the control of the iron-responsive transcription factor Fur and induced by iron-deﬁciency [ 12 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19049822" target="_blank">19049822</a>]<br></li>
<li>Moreover , the E. coli ferritin is indirectly induced by Fur under iron-rich conditions , as Fur represses the small RNA RhyB , which directly represses ferritin transcription in the absence of iron ( Abdul-Tehrani et al. 1999 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26928316" target="_blank">26928316</a>]<br></li>
<li>fepA codes for the enterobactin receptor , which is induced under low-iron conditions and repressed by Fur in conditions of high iron ( 53 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16237024" target="_blank">16237024</a>]<br></li>
<li>The sitABCD operon is Fur regulated Both the homology with known iron transport proteins and the identification of sitABCD in an IVET selection for genes induced under low-iron conditions led us to examine the regulation of this fusion in vitro under ironstarvation conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10712695" target="_blank">10712695</a>]<br></li>
<li>The Fur regulon , those genes whose expression are under the control of Fur , contain genes that are involved in iron acquisition , which is essential for bacterial multiplication ( de Lorenzo et al. , 1987 ; Mekalanos , 1992 ; Wooldridge and Williams , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12652904" target="_blank">12652904</a>]<br></li>
<li>For example , the iron-responsive transcriptional-regulator Fur is required for expression of iron uptake systems under iron-limiting conditions and for acid-induced activation of atr genes ( Slauch et al. , 1997 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10844688" target="_blank">10844688</a>]<br></li>
<li>In contrast , in S. enterica serovar 2 Typhimurium , we see signiﬁcant Fe repression when either component of the Fur regulatory circuit has been disrupted ( Fig. 4A ) , but only when the MntR circuit is intact .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12029030" target="_blank">12029030</a>]<br></li>
<li>It is also noteworthy that Fur is required for induction of the least studied ferritin-like protein in S. Typhimurium , FtnB ( 64 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21075923" target="_blank">21075923</a>]<br></li>
<li>Transcription of the hilD gene is activated in response to Fur , but this is also dependent on the HilD protein and is in accordance with our model for SPI1 regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
<li>Because the Fur protein which binds the fur box is also involved in acid resistance [ 43 ] , we extended our experiments and found that sdiA is induced in S. typhimurium when the bacterial culture is inoculated in LB at pH 4 under fully aerobic conditions [ 172 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16023758" target="_blank">16023758</a>]<br></li>
<li>Figure 5A shows that this construct was not affected by either HilC or Fur overproduction and that the presence or absence of hilD had no effect on transcription of this fusion .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
<li>In case , a count and probability of nucleotide at particular position in Fur box encountered as zero , which is quite common , might not be converted to logarithms , so the background frequency at that position was taken under consideration and penalized by multiplying of intergenic fraction or overall frequency [ 53 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23040276" target="_blank">23040276</a>]<br></li>
<li>While the potential role of some of these regulators was not confirmed by western blotting , this analysis clearly demonstrated , that IpfD-2HA is negatively regulated by Fur and that OmpR acts as a positive regulator of ipfD at 37 ˚C ( Fig 7F ) and 41 ˚C ( Fig 7G ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28817673" target="_blank">28817673</a>]<br></li>
<li>The recent evidence that Fur displaces H-NS from regulatory sites of ftnA and not vice versa ( 45 ) indicates that once Fur is activated the cell is committed to countersilencing and repression of H-NS until iron concentrations are reduced enough to inhibit Fur binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21075923" target="_blank">21075923</a>]<br></li>
<li>Crawford & Goldberg ( 1998a ) proposed that the ironresponsive regulator , Fur , represses hmp transcription and that this repression is lifted by NO on inactivation of Fur .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17526833" target="_blank">17526833</a>]<br></li>
<li>We reveal that Fur controls SPI-2 genes via repression of SsrB activity and provide evidence that Fur represses SsrB production by binding to the ssrB promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24643535" target="_blank">24643535</a>]<br></li>
<li>Masse and Gottesman ( 2002 ) also reported that expression of certain mRNAs ( acnA , fumA , ftnA , bfr , sodB , and sdhD ) in Escherichia coli that were shown to be positively regulated by the Fur regulator is repressed by RyhB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23831078" target="_blank">23831078</a>]<br></li>
<li>This suggests that repression by Fur is occurring in vivo , and that the 2 + SCV is rich in Fe but does not contain signiﬁcant levels of Fe3 + 2 + .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12492857" target="_blank">12492857</a>]<br></li>
<li>However , in such cases it is not clear whether Fur directly regulates such genes or whether this may be caused by the indirect action of Fur as a negative regulator of a second negative regulator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16023758" target="_blank">16023758</a>]<br></li>
<li>The ferric uptake regulator Fur is also inactivated by NO · , and the derepression of Fur-regulated is observed in cells subjected to nitrosative stress ( D'Autreaux et al. , 2002 ; Spiro , 2006 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22831173" target="_blank">22831173</a>]<br></li>
<li>Fur is able to sense ﬂuctuation of free iron ; eventually activated Fur regulates its target gene by direct interaction with a so-called `` Fur box '' in the promoter regions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27242152" target="_blank">27242152</a>]<br></li>
<li>Interestingly , the regulation of tcf by Fur was observed in S. Typhimurium and S. Typhi , but not in E. coli , although Fur is highly conserved between these genera , suggesting an indirect mechanism of regulation by Fur that is speciﬁc to Salmonella .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26944792" target="_blank">26944792</a>]<br></li>
<li>Although SPI2 is not known to be regulated by Fur , there is presence of a large number of Fur-boxes in the SPI2 locus .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23040276" target="_blank">23040276</a>]<br></li>
<li>In E. coli , class Ib expression has been suggested to be regulated by the transcriptional factor Fur [ 8,9 ] , and nrdDG expression is anaerobically activated by the global regulator FNR [ 10 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20593029" target="_blank">20593029</a>]<br></li>
<li>While the SPI1 network has been demonstrated to be regulated by Fur ( although the precise mechanism is still being investigated ) [ 17,22 ] , the other systems are not known to be regulated by Fur .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23040276" target="_blank">23040276</a>]<br></li>
<li>In Salmonella Typhimurium , the expression of both RyhB-1 and RyhB-2 is negatively regulated by the Fur repressor , while stationary phase is the primary signal inducing RyhB-2 expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22824499" target="_blank">22824499</a>]<br></li>
<li>On the contrary , RyhB-2 expression is also slightly induced by activated Fur , but its primary inducing condition is the stationary growth-phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22824499" target="_blank">22824499</a>]<br></li>
<li>In E. coli , YncE is secreted into the periplasm via the Sec-dependent pathway [ 37 ] , and its expression is induced under iron-restricted conditions when repression by the Fur protein is relieved [ 38 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23936575" target="_blank">23936575</a>]<br></li>
<li>Although Yoo et al. ( 2007 ) reported that dps expression is induced during the stationary phase by the Fur regulator , our results demonstrated that this gene is also regulated by RcsB but during the exponential phase of growth .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25028458" target="_blank">25028458</a>]<br></li>
<li>Therefore , we propose that the positive regulation ftnB by Fur is mediated by the negative regulation of hns by Fur .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22017966" target="_blank">22017966</a>]<br></li>
<li>In the presence of Fe2 , mntH is repressed primarily by Fur , acting through a Fur-binding motif overlapping the 35 region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12029030" target="_blank">12029030</a>]<br></li>
<li>Transcription of the feoAB operon is negatively regulated by the Fur protein ( 12 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18790861" target="_blank">18790861</a>]<br></li>
<li>Expression of the Pasteurella multocida ompH gene is negatively regulated by the Fur protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11932449" target="_blank">11932449</a>]<br></li>
<li>Collectively , we concluded from the above set of experiments that Lrp is an activator of the klf genes , and that the ipf operon is repressed by Fur and positively regulated by OmpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28817673" target="_blank">28817673</a>]<br></li>
<li>Thus , under Fur active conditions ( left panel ) , hns is repressed by Fur thereby blocking H-NS repression of ftnB and the tdc operon ( signified by the circle with an `` X '' ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22017966" target="_blank">22017966</a>]<br></li>
<li>Ernst F.D. , Homuth G. , Stoof J. , Mader U. , Waidner B. , Kuipers E.J. , Kist M. , Kusters J.G. , Bereswill S. and Van Vliet A.H. , Iron-responsive regulation of the Helicobacter pylori iron-cofactored superoxide dismutase SodB is med-iated by Fur .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16362827" target="_blank">16362827</a>]<br></li>
<li>What is clear is that a transporter exclusively dedicated to the uptake of Mn2 is repressed strongly , via normal Fur regulation , by the minor and physiologically unimportant substrate Fe2 ( 23a ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12029030" target="_blank">12029030</a>]<br></li>
<li>The following scenario is proposed to explain these findings and to suggest that the observed regulation of ftnB by Fur is mediated by the histone-like protein H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22017966" target="_blank">22017966</a>]<br></li>
<li>The simplest explanation is that available iron and manganese become limited as the cell population increases , leading to induction of the Fur and MntR regulons .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659669" target="_blank">15659669</a>]<br></li>
<li>SPI1 is induced by iron in a Fur-dependent fashion ( Teixidó et al. , 2011 ) suggesting that the down regulation of the SPI1 genes observed here is due to the low iron concentration of egg white and consequent regulation change mediated by Fur .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28553268" target="_blank">28553268</a>]<br></li>
<li>The effect of iron availability on hmpA expression is negated by macrophage-derived defence factors ; NO is believed to mimic iron-depletion by destroying the Fe -- Fur complexes leading to derepression , even in the presence of iron ( Crawford and Goldberg , 1998 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12492857" target="_blank">12492857</a>]<br></li>
<li>Fur and the acid stress response The Fur protein is usually linked to the regulation of bacterial iron metabolism .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16023758" target="_blank">16023758</a>]<br></li>
<li>In response to oxi-dative stress , expression of the Fur protein is upregulated by the OxyR and SoxRS regulators ( 33 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18790861" target="_blank">18790861</a>]<br></li>
<li>Fur positively regulates HilA , the master regulator of SPI-1 , via HilD ( Ellermeier & Slauch , 2008 ; Teixidó et al. , 2011 ) or via the repression of the histone-like protein H-NS ( Troxell et al. , 2011b ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23306672" target="_blank">23306672</a>]<br></li>
<li>The nitric oxide ( NO ) donor spermine NONOate derepressed the Fur regulon that controls secondary lines of resistance against organic acids .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18350168" target="_blank">18350168</a>]<br></li>
<li>The negative transcriptional-regulator Fur is responsible for the regulation of a number o FIG. 3 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12897017" target="_blank">12897017</a>]<br></li>
<li>In conclusion , Fur activates hilA by repressing the expression of hns .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21075923" target="_blank">21075923</a>]<br></li>
<li>Fur regulates hilA-lacZ independent of the response-regulator PhoP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21075923" target="_blank">21075923</a>]<br></li>
<li>This again suggests a mechanism where Fur binding might relieve HNS-dependent transcription repression , or HNS binding might initiate nucleation of the protein , and repress Fur-dependent activation of gene expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23040276" target="_blank">23040276</a>]<br></li>
<li>Fur controlled SPI-2 genes via repression of the SsrB response-regulator , a primary activator of SPI-2 expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24643535" target="_blank">24643535</a>]<br></li>
<li>We further reasoned that if Fur controls SPI-2 genes via the repression of ssrB transcription , Fur repression of SPI-2 expression would be impaired when the ssrB gene is ectopically expressed from a heterologous promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24643535" target="_blank">24643535</a>]<br></li>
<li>While we can not say that Fur accounts for all of the oxygen regulation of SPI1 , it certainly contributes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
<li>[ 97 ] A. Nandal , C.C.O. Huggins , M.R. Woodhall , J. McHugh , F. Rodrıguez-Quin ̃ones , M.A. Quail , et al. , Induction of the ferritin gene ( ftnA ) of Escherichia coli by Fe ( 2þ ) - Fur is mediated by reversal of H-NS silencing and is RyhB independent , Mol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23040276" target="_blank">23040276</a>]<br></li>
<li>In E. coli , expression of sodB is activated by Fur via repression of the small RNA RyhB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
<li>Our study of the anaerobic expression of hmpA suggests that it is regulated by Fur , independent of Fnr .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22017966" target="_blank">22017966</a>]<br></li>
<li>Taken together , these data rule out the possibility that there is Fur control of SPI1 by simple activation of hilD transcription or translation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
<li>Collectively , these data demonstrate that Fur-depen-dent activation of hilA-lacZ during anaerobic-growth is due to the repression of hns by Fur .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21075923" target="_blank">21075923</a>]<br></li>
<li>In contrast to PiroBCDE , no viable concentration of cobalt or manganese altered expression from PsodB ( data not shown ) , despite PsodB being regulated by Fur , albeit indirectly .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19049822" target="_blank">19049822</a>]<br></li>
<li>A thorough analysis by two-dimensional gel electrophoresis identiﬁed 19 proteins of S. typhimurium whose expression is negatively regulated by Fur ( 17 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7642488" target="_blank">7642488</a>]<br></li>
<li>Twelve proteins were positively regulated by Fur seemingly irrespective of iron status .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8824613" target="_blank">8824613</a>]<br></li>
<li>Thus , Fur indirectly activates SPI1 via its repression of hns , demonstrating that iron metabolism can influence genes regulated by H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22017966" target="_blank">22017966</a>]<br></li>
<li>The Fur-mediated up-regulation of FnrS is unlikely to be a direct effect , however , as Fur does not directly activate transcription [ 53 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27564394" target="_blank">27564394</a>]<br></li>
<li>Fur regulation is mediated through HilD , but regulation does not appear to be simply at the level of hilD transcription or translation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
<li>It is highly likely that Salmonella possesses a variety of sensors , such as Fur , devoted to coordinating responses to NO .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21833325" target="_blank">21833325</a>]<br></li>
<li>Class C represents promoters that appear to be regulated independently of Fur .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12897017" target="_blank">12897017</a>]<br></li>
<li>Several putative genes appear to be under direct control of Fur .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22017966" target="_blank">22017966</a>]<br></li>
<li>Thus , Fur is an activator of genes that are typically induced under anaerobic conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22017966" target="_blank">22017966</a>]<br></li>
<li>The expression of ryhB , in turn , is repressed by Fur under high-iron conditions ( 38 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
<li>Regulation of SodA production is complex but includes repression by Fnr , ArcA , and Fur , such that expression is repressed under anaerobic conditions ( 16 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659669" target="_blank">15659669</a>]<br></li>
<li>The regulation of the manganese transporters by Fur is unclear , although it is possible that manganese import is not necessary under some circumstances of iron-sufficiency ( Papp-Wallace and Maguire , 2006 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21722794" target="_blank">21722794</a>]<br></li>
<li>Global regulation of gene expression by HNS and Fur Since SPI1 is a foreign fragment of DNA acquired by Salmonella during its evolution , it is understandable that it is subject to a heavy negative regulation [ 89 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23040276" target="_blank">23040276</a>]<br></li>
<li>This Fe-mediated repression by MntR is independent of Fur .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659669" target="_blank">15659669</a>]<br></li>
<li>Fur regulation of HilD is not mediated through HilE .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
<li>The mechanism of hilA regulation by Fur is unknown .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21075923" target="_blank">21075923</a>]<br></li>
<li>The fact that presence of Fur inhibits macrophage uptake is novel .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23306672" target="_blank">23306672</a>]<br></li>
<li>The mechanism of Fur activation of these putative genes is unknown .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22017966" target="_blank">22017966</a>]<br></li>
<li>These results indicated that Fur controlled regulation of ftnB is independent of Fnr .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22017966" target="_blank">22017966</a>]<br></li>
<li>Since Fur activation of hilA is clearly dependent upon HilD , we constructed hilD-lacZ fusions in a hilD background .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
<li>It is wort noting that while overproduction of Fur caused an eightfold increase in hilA expression in low-oxygen conditions , Fur overproduction induced hilA expression only threefold in highoxygen conditions ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
<li>Platero R , Peixoto L , O'Brian MR , Fabiano E : Fur is involved in manganese-dependent regulation of mntA ( sitA ) expression in Sinorhizobium meliloti .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22017966" target="_blank">22017966</a>]<br></li>
<li>Therefore , this result indicates that Fur repression of SPI-2 expression is dependent on SsrB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24643535" target="_blank">24643535</a>]<br></li>
<li>Ellermeier and Slauch initially reported that Fur activates hilA transcription in a HilD-dependent manner , and since Fur is thought to be capable of acting only as a repressor , it should repress a repressor that controls HilD ( 322 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23554419" target="_blank">23554419</a>]<br></li>
<li>Thus , one model to explain Fur regulation of SPI1 is that Fur represses a factor that negatively regulates HilD function .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
<li>Given that HilD is required for Fur regulation of hilA , it seemed possible that Fur regulates hilD expression in some manner .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
<li>Taken together , these data show that Fur activates hilA expression ( HilD activity ) in a manner that is mechanistically different from Fur activation of sodB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
<li>Because growth at 42 ° C repressed expression of the PrtsA-lacZ fusion , identifying that Fur also regulates SPI-1 in this manner would suggest that temperature control of SPI-1 may act through Fur .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26386070" target="_blank">26386070</a>]<br></li>
<li>These data suggest that Fur activates SPI1 in a manner different from that of Fur activation of sodB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17208038" target="_blank">17208038</a>]<br></li>
<li>Earlier work determined that Fur activation of hilA operate through HilD and that overexpression of HilD resulted in Fur-independent induction of hilA ( 22 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21075923" target="_blank">21075923</a>]<br></li>
<li>Again , when hilD was deleted in these strains , Fur regulation of hilD was abrogated , while overproduction of HilC still induced hilD expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
<li>Fur mediates Fe2 repression of mntH .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12029030" target="_blank">12029030</a>]<br></li>
<li>Importantly , inactivation of Fur disrupts Fe2 - dependent repression of mntH transcription but retains Mn2 - dependent repression ( 19 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24596096" target="_blank">24596096</a>]<br></li>
<li>Taken together , these results illustrate that Fur repression of ssrB transcription leads to repression of SPI-2 genes under acidic conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24643535" target="_blank">24643535</a>]<br></li>
<li>Fur negatively controls SPI-2 expression via repression of ssrB transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24643535" target="_blank">24643535</a>]<br></li>
<li>FIG 5 Fur repression of ssrB expression leads to repression of SPI-2 expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24643535" target="_blank">24643535</a>]<br></li>
<li>[ 21 ] R. Tsolis , A. Baumler , I. Stojiljkovic , F. Heffron , Fur regulon of Salmonella typhimurium : identiﬁcation of new iron - regulated genes , J. Bacteriol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23040276" target="_blank">23040276</a>]<br></li>
<li>[ 29 ] A.H. Yoo , S.W. Kim , J.E. Yu , Y.H. Kim , J. Cha , J. Oh , S.K. Eo , J.H. Lee , H.Y. Kang , Requirement of Fur for full induction of dps expression in Salmonella enterica serovar typhimurium , J. Microbiol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23159630" target="_blank">23159630</a>]<br></li>
<li>The absence of fur resulted in increased expression ( i.e. , Fur acted as a repressor ) of 226 genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22017966" target="_blank">22017966</a>]<br></li>
<li>Fur regulation of sodB and Fur regulation of hilA are mechanistically different .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
<li>First , overproduction of Fur signiﬁcantly increases expression of hilA and hilD when HilD is present .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
<li>Because of this , we wanted to observe expression of a sodB-lacZ translational fusion as a control for Fur `` activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
<li>However , Fur activation of ftnB was independent of Fnr .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22017966" target="_blank">22017966</a>]<br></li>
<li>Many other regulators encoded outside of SPI-1 affect SPI-1 genes and/or HilA expression through HilD , including HilE ( repression of hilA by binding to and preventing HilD function ) [ 31 ] , FliZ ( activation of hilA via posttranslational regulation of HilD ) [ 32 ] , DNA adenine methylase ( activation of hilA via posttranslational regulation of HilD ) [ 33 ] , EnvZ-OmpR ( activation of hilA via HilD ) [ 34 ] , CpxA/R ( repression of hilA by decreasing the stability of HilD ) [ 35 ] , BarA/SirA ( activation of hilA via activation of csrB/csrC to block CsrA repression of hilD ) [ 24 ] and Fur ( activation of hilA via an unknown regulation of HilD ) [ 26 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28575106" target="_blank">28575106</a>]<br></li>
<li>Stability of Crp , Fur , RpoS , and PhoP proteins during incubation of cultures induced for expression of these proteins prior to addition of 50 g/ml chloramphenicol of culture .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19103774" target="_blank">19103774</a>]<br></li>
<li>Inducible expression of either HilC or RtsA protein , but not of Fur , FliZ , or HilD , was sufﬁcient to activate hilA following growth at 42 ° C .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26386070" target="_blank">26386070</a>]<br></li>
<li>In this work , we demonstrated that Fur regulates expression of hilA via control of HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
<li>Fur activates expression of hilA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_584">
Fur regulates expression of hilA via HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></label>
<input id="act_similarity_584", type="checkbox">
<div class="ACT_similar">
<li>Fur regulates expression of rtsA via HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li></div>
<li>Fur regulates expression of hilA by working through HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
<li>Fur regulates expression of rtsA by controlling HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
				</div></ul><br>
			<label class="collapse_DOM" for="_2"><font size="5px">Fur protein domains</font></label>
				<input id="_2" type="checkbox">
				<div class="DOM"><br><ul><li>( A ) Motif consensus logo for Fur recognition motif was generated with software available from http://weblogo.berkeley .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27564394" target="_blank">27564394</a>]<br></li>
<li>We have also identified a Fur recognition sequence [ 33 ] ( Figure 4A ) located at position -309 within the nrdHIEF promoter region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20593029" target="_blank">20593029</a>]<br></li>
<li>indicating that although residue H-90 is needed by Fur for iron sensing , it is not required for acid pH control .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8824613" target="_blank">8824613</a>]<br></li>
<li>The Fur regulon contains genes which are involved in iron acquisition , a function which is essential for bacterial multiplication .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7642488" target="_blank">7642488</a>]<br></li>
<li>This interaction also demonstrates interaction between two regulators ( Fur and H-NS ) functioning in highly conserved physiological events , regulating a potentially toxic , but needed metal and regulating foreign DNA in a concerted manner .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22017966" target="_blank">22017966</a>]<br></li>
<li>NO · interactions with NsrR and SoxR are coordinated through iron-sulfur clusters ( FeS ) , whereas NorR and Fur interact directly with Fe2 + .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22831173" target="_blank">22831173</a>]<br></li>
<li>Transformants carrying plasmids that contain a Fur binding site or encode an iron-binding protein form red colonies on M1Fe , while other colonies will be white .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7642488" target="_blank">7642488</a>]<br></li>
<li>Transformant carrying plasmids that contain a Fur binding site or encoding an iron-binding protein form a red colony in MacConkey lactose with FeSO plates .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12652904" target="_blank">12652904</a>]<br></li>
<li>Crystal structure of the Vibrio cholerae ferric uptake regulator ( Fur ) reveals insights into metal co-ordination .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21722794" target="_blank">21722794</a>]<br></li>
				<label class="DOM_collapse_similarity" for="dom_similarity_10">
Functional domains of the Escherichia coli ferric uptake regulator protein ( Fur ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8824613" target="_blank">8824613</a>]<br></label>
<input id="dom_similarity_10", type="checkbox">
<div class="DOM_similar">
<li>Functional domains of the Escherichia coli ferric uptake regulator ( Fur ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7642488" target="_blank">7642488</a>]<br></li></div>
<li>A putative Fur binding motif was identified in 49 genes ( Table 4 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22017966" target="_blank">22017966</a>]<br></li>
<li>In this scenario , processes dependent on labile iron , such as Fur iron-binding and Fe-S cluster metabolism , could be perturbed by disrupting the intracellular thiol pools .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17720790" target="_blank">17720790</a>]<br></li>
<li>These studies conﬁrm previous ﬁndings demonstrating an important role for Fur in the ATR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8824613" target="_blank">8824613</a>]<br></li>
<li>The sitABCD promoter also contains putative Fur and MntR binding sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659669" target="_blank">15659669</a>]<br></li>
<li>We had previously noted that the S. enterica serovar Typhimurium mntH promoter contains not only OxyR and Fur binding motifs but also a novel inverted repeat element .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12029031" target="_blank">12029031</a>]<br></li>
<li>It is interesting that nuclear magnetic resonance studies examining ionization of the histidine residues suggest that Fur is a monitor of iron concentration and pH ( 40 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8824613" target="_blank">8824613</a>]<br></li>
<li>These residues occur in the amino-terminal half of Fur associated with DNA-binding ( 50 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8824613" target="_blank">8824613</a>]<br></li>
<li>Site-directed mutagenesis of the histidine and cysteine residues of E. coli Fur argue that cysteines 92 and 95 are the most important for iron-binding ( 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8824613" target="_blank">8824613</a>]<br></li>
<li>Structural dynamics and functional domains of the Fur protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7642488" target="_blank">7642488</a>]<br></li>
<li>Iron binding is necessary for the function of Fur as a transcription factor ( 12 , 13 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24643535" target="_blank">24643535</a>]<br></li>
<li>We investigated whether promoters of sRNAs which were up-regulated in the absence of Fur contained `` Fur-boxes '' or recognition motifs for Fur binding , consistent with direct repression of expression by Fur .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27564394" target="_blank">27564394</a>]<br></li>
<li>We also identiﬁed 28 nucleotides upstream of hns that contained the Fur binding motif responsible for the Fur-de-pendent activation of hilA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21075923" target="_blank">21075923</a>]<br></li>
<li>The results of fusion protein studies combining different domains of Fur and the l repressor cI857 indicate that the N terminus of Fur is involved in DNA-binding while the C terminus is required for dimerization ( 50 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8824613" target="_blank">8824613</a>]<br></li>
<li>Since the C-terminal part of Fur contains the domain necessary for repressor dimerization but not for DNA-binding ( 13 , 51 ) , the LacZ-Fur protein may form defective dimers with wild-type E. coli Fur in H1717 , thereby causing a 2 Fur phenotype .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7642488" target="_blank">7642488</a>]<br></li>
<li>Since a plausible Fur-binding motif is apparent in the mntH promoter , cis - and trans-acting Fur mutants were evaluated in add-back experiments with each metal .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12029030" target="_blank">12029030</a>]<br></li>
<li>Experiments equivalent to those with the Fur protein and Fur-binding motif mutants were performed with an MntR protein mutant and an inverted-repeat motif ( MntR-binding motif ) mutant , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12029030" target="_blank">12029030</a>]<br></li>
				<label class="DOM_collapse_similarity" for="dom_similarity_28">
However , mutation of the MntR-binding motif alone abolishes Mn2 repression ( Fig. 4D ) ; hence the possible interaction of a Mn2 - liganded Fur protein with the Fur-binding motif does not appear to be 2 sufﬁcient to mediate Mn repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12029030" target="_blank">12029030</a>]<br></label>
<input id="dom_similarity_28", type="checkbox">
<div class="DOM_similar">
<li>Further , mutation of the MntR protein and Fur-binding motif also abolishes Mn2 repression ( Fig. 4F ) ; hence the Fur protein does not appear to mediate repression by binding to the MntR-binding motif .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12029030" target="_blank">12029030</a>]<br></li></div>
<li>All four NRAMP promoters contain putative Fur and OxyR binding motifs ; however , the 22 base inverted repeat motif is absent in the Y. pestis promoters for NRAMP and for yfeABCD , a virulence-associated iron transport system in Y. pestis thought to be equivalent to S. typhimurium sitABCD ( Bearden and Perry , 1999 ; Zhou et al. , 1999 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10844693" target="_blank">10844693</a>]<br></li>
<li>The crystal structure of M. tuberculosis Zur in complex with zinc has been determined and shows similarity to iron-sensing Fur proteins ( Pohl et al. , 2003 ; Sheikh and Taylor , 2009 ) , each monomer of the dimer being a two-domain structure with an amino-terminal DNA-binding domain composed of a three-helix bundle followed by a short antiparallel b-sheet and a carboxyl-terminal metal-binding and dimerization domain ( Lucarelli et al. , 2007 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21722794" target="_blank">21722794</a>]<br></li>
<li>Fur contains 12 histidines and 4 cysteines , 2 of which are found in the CysXYCysGly motif commonly used as a ligand by iron-sulfur proteins ( 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8824613" target="_blank">8824613</a>]<br></li>
<li>Cobalt has been shown to oxidize glutathione in vitro ( reviewed in reference 7 ) and disrupts both Fur activity and the activity of Fe-S cluster proteins in a manner that is at least partially suppressed by the addition of cysteine or glutathione to the growth medium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17720790" target="_blank">17720790</a>]<br></li>
<li>It will be interesting and informative to identify the residues within Fur that serve as pH sensors and ultimately identify the acid tolerance proteins regulated by Fur .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8824613" target="_blank">8824613</a>]<br></li>
<li>The conserved residues in the Fur and Zur proteins from E. coli and S. enterica sv .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21722794" target="_blank">21722794</a>]<br></li>
<li>The lack of an obvious Fur binding motif upstream of hmpA indicates that reporter activity seen in Δfur was likely indirect .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22017966" target="_blank">22017966</a>]<br></li>
<li>The nucleotide sequence of three FURTA-positive clones contained Fur binding motifs but showed no homology to previously described Fur-regulated promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7642488" target="_blank">7642488</a>]<br></li>
				<label class="DOM_collapse_similarity" for="dom_similarity_38">
The promoters of ftnB and hmpA do not contain recognized Fur binding motifs , which indicated their probable indirect regulation by Fur .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22017966" target="_blank">22017966</a>]<br></label>
<input id="dom_similarity_38", type="checkbox">
<div class="DOM_similar">
<li>However , the promoters of ftnB and hmpA do not contain recognizable Fur binding motifs indicating their indirect regulation by Fur .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22017966" target="_blank">22017966</a>]<br></li></div>
				</div></ul><br>
			<label class="collapse_EVO" for="_3"><font size="5px">Fur evolution</font></label>
				<input id="_3" type="checkbox">
				<div class="EVO"><br><ul><li>Although there is no sequence similarity between DtxR and Fur ( less than 20 % identity ) , there is evidence for structural homology ( 12 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659669" target="_blank">15659669</a>]<br></li>
<li>Typhimurium possesses two zinc sensors , Zur ( Fur family ) and ZntR ( MerR family ) , which act together to control zinc availability .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21722794" target="_blank">21722794</a>]<br></li>
				<label class="EVO_collapse_similarity" for="evo_similarity_3">
Lee JW , Helmann JD : Functional specialization within the Fur family of metalloregulators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22017966" target="_blank">22017966</a>]<br></label>
<input id="evo_similarity_3", type="checkbox">
<div class="EVO_similar">
<li>Functional specialization within the Fur family of metalloregulators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24643535" target="_blank">24643535</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21722794" target="_blank">21722794</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24735176" target="_blank">24735176</a>]<br></li></div>
<li>OxyR , a member of the LysR family of bacterial transcriptional factors , regulates expression of many genes required for combating hydrogen peroxide ( H2O2 ) - mediated bacterial killing , such as those encoding catalase , glutathione reductase , glutaredoxin , alkyl hydroperoxide reductase , OxyS and Fur ( Christman et al. , 1985 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18757817" target="_blank">18757817</a>]<br></li>
<li>Given that RstA ( STM1475 ) is a member of the OmpR family of response regulators , it may not be surprising that H-NS regulation is iron dependent and indicates that H-NS may reduce ferrous iron acquisition and transport , thereby reducing Fur activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21075923" target="_blank">21075923</a>]<br></li>
<li>S. enterica Fur is an orthologue of the well-characterized E. coli Fur ( reviewed by Lee and Helmann , 2007 ) , these proteins sharing 99 % sequence identity over 148 amino acids .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21722794" target="_blank">21722794</a>]<br></li>
<li>The protein encoded by rosE had homology to transcriptional regulators and showed the highest sequence identity to the ArgR ( 29 % ) ( Lim et al. , 1987 ) and Fur ( 16 % ) ( Schäffer et al. , 1985 ) proteins of E. coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18331470" target="_blank">18331470</a>]<br></li>
<li>In other organisms members of the Fur family are also known to sense sufficiency of other metals , including manganese ( Mur ) and nickel ( Nur ) , the availability of haem ( Irr ) , or use metal catalyzed oxidation reactions to sense peroxide stress ( PerR ) ( Lee and Helmann , 2007 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21722794" target="_blank">21722794</a>]<br></li>
<li>The Fur-repressed divergently-oriented promoters of STnc3250 and STnc4000 may represent new examples of the bidirectional promoters that belong to the Fur regulon , such as , fepA-fes , fepD-ybdA and fepB-entCEBA [ 55 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27564394" target="_blank">27564394</a>]<br></li>
<li>These include the so-called LysNMetR regulatory protein family , the AraC protein family , the iron response-regulator Fur family , and two-component regulatory systems .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11370771" target="_blank">11370771</a>]<br></li>
				<label class="EVO_collapse_similarity" for="evo_similarity_12">
Members of the Fur protein family regulate iron and zinc transport in E. coli and characteristics of the Fur-regulated protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12897017" target="_blank">12897017</a>]<br></label>
<input id="evo_similarity_12", type="checkbox">
<div class="EVO_similar">
<li>Members of the Fur protein family regulate iron and zinc transport in E. coli and characteristics of the Fur-regulated fhuF protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18441067" target="_blank">18441067</a>]<br></li>
<li>Members of the Fur protein family regulate iron and zinc transport in Escherichia coli and characteristics of the Fur-regulated fhuF protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15790293" target="_blank">15790293</a>]<br></li></div>
<li>The Zur protein belongs to the Fur family of metalloregulatory proteins , and in the presence of Zn2 , it binds to a nearly perfect palindrome found in this region , blocking both znuA and znuCB transcription ( 17 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12117991" target="_blank">12117991</a>]<br></li>
<li>Transcriptional Regulation by Zur and Zinc Uptake Zinc-sensing Zur represents the second , in addition to iron-sensing Fur , member of the Fur family of regulators in S. enterica sv .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21722794" target="_blank">21722794</a>]<br></li>
				</div></ul><br>
			<label class="collapse_RP" for="_4"><font size="5px">Processes regulated by Fur</font></label>
				<input id="_4" type="checkbox">
				<div class="RP"><br><ul><li>Bacterial strains and plasmids used in this study Genotype or relevant characteristics Deletion endpoints or cloned end pointsb c Strains 14028 JS248 JS251 JS252 JS253 JS256 JS562 JS564 JS566 JS567 JS568 JS569 JS570 JS571 JS572 JS573 JS574 JS404 JS412 JS575 JS576 JS577 JS578 JS579 JS580 JS581 JS582 JS583 JS584 JS585 JS586 JS587 JS588 JS589 JS590 JS591 JS592 JS593 JS594 JS595 JS596 JS597 JS598 JS613 JS614 JS615 JS616 JS617 JS618 JS619 JS620 JS621 JS622 JS623 JS624 JS625 JS626 JS629 JS630 JS631 JS632 JS633 JS634 JS635 JS636 JS637 ATCCd 18 18 18 18 18 W ild-type serovar Typhimurium rtsA5 4561755 -- 4560884 3019885 -- 3021480 3012135 -- 3012976 3017865 -- 3018730 3012135 -- 3018730 1509924 -- 1509449 3017958 -- 3018729 756811 -- 756421 3715394 -- 3715551 hilA112 : : Cm hilC113 : : Cm hilD114 : : Cm hilC-D2915 : : Cm sodB111 : : Km hilD138 : : Km fur-42 : : tetAR rfrA1 : : Cm rfrA1 rfrB2 : : Cm hfq11 : : Cm hfq11 : : Km hilE115 : : aadA hilC131 : : Cm rtsA9 : : Cm tt : : pAH125 : : sitA - lac fur-41 : : Cm att : : pAH125 : : sitA - lac tt : : pDX1 : : hilA - lac hilD114 : : Cm att : : pDX1 : : hilA - lac hilC113 : : Cm att : : pDX1 : : hilA - lac hilC-D2915 : : Cm att : : pDX1 : : hilA - lac rtsA5 att : : pDX1 : : hilA - lac hilD114 : : Cm rtsA5 att : : pDX1 : : hilA - lac hilC113 : : Cm rtsA5 att : : pDX1 : : hilA - lac hilC-D2915 : : Cm rtsA5 att : : pDX1 : : hilA - lac fur-42 : : tetAR att : : pDX1 : : hilA - lac fur-42 : : tetAR hilD114 : : Cm att : : pDX1 : : hilA - lac fur-42 : : tetAR hilC113 : : Cm att : : pDX1 : : hilA - lac fur-42 : : tetAR hilC-D2915 : : Cm att : : pDX1 : : hilA - lac fur-42 : : tetAR rtsA5 att : : pDX1 : : hilA - lac fur-42 : : tetAR hilD114 : : Cm rtsA5 att : : pDX1 : : hilA - la fur-42 : : tetAR hilC113 : : Cm rtsA5 att : : pDX1 : : hilA - la fur-42 : : tetAR hilC-D2915 : : Cm rtsA5 att : : pDX1 : : hilA WKS30 att : : pDX1 : : hilA Fur att : : pDX1 : : hilA WKS30 hilD114 : : Cm att : : pDX1 : : hilA Fur hilD114 : : Cm att : : pDX1 : : hilA WKS30 hilC113 : : Cm att : : pDX1 : : hilA Fur hilC113 : : Cm att : : pDX1 : : hilA WKS30 rtsA5 att : : pDX1 : : hilA Fur rtsA5 att : : pDX1 : : hilA rfrA1 att : : pDX1 : : hilA - lac fur-42 : : tetAR rfrA1 att : : pDX1 : : hilA - lac rfrB2 : : Cm rfrA1 att : : pDX1 : : hilA - lac fur-42 : : tetAR rfrB2 : : Cm rfrA1 att : : pDX1 : : hilA - lac rfrB2 : : Cm att : : pDX1 : : hilA - lac fur-42 : : tetAR rfrB2 : : Cm att : : pDX1 : : hilA - lac ( sodB111 - lac ) hyb fur-42 : : tetAR ( sodB111 - lac ) hyb rfrA1 ( sodB111 - lac ) hyb fur-42 : : tetAR rfrA1 ( sodB111 - lac ) hyb rfrB2 : : Cm rfrA1 ( sodB111 - lac ) hyb fur-42 : : tetAR rfrB2 : : Cm rfrA1 ( sodB111 - lac ) hyb rfrB2 : : Cm ( sodB111 - lac ) hyb fur-42 : : tetAR rfrB2 : : Cm ( sodB111 - lac ) hyb hfq11 : : Cm ( sodB111 - lac ) hyb fur-42 : : tetAR hfq11 : : Cm ( sodB111 - lac ) hyb hfq11 : : Km att : : pDX1 : : hilA - lac hfq11 : : Km fur-42 : : tetAR att : : pDX1 : : hilA - lac hilE115 : : aadA att : : pDX1 : : hilA - lac hilE115 : : aadA hilD114 : : Cm att : : pDX1 : : hilA - lac hilE115 : : aadA hilC113 : : Cm att : : pDX1 : : hilA - lac hilE115 : : aadA rtsA5 att : : pDX1 : : hilA - lac fur-42 : : tetAR hilE115 : : aadA att : : pDX1 : : hilA - lac 1352993 -- 1352870 4604580 -- 4604883 4604580 -- 4604883 4763527 -- 4764108 3012137 -- 3012904 4560911 -- 4561439 25 a 25 a c c - lac p p p p p p p p Continued on following pag TABLE 1 -- Continued Deletion endpoints or cloned end pointsb Strain or plasmida c Source or reference Genotype or relevant characteristics JS638 JS639 JS640 JS641 JS642 JS643 JS644 JS645 JS646 JS647 JS648 JS649 JS650 JS651 JS652 JS653 JS654 JS655 JS656 JS659 JS660 JS661 JS662 JS667 JS668 JS697 JS698 JS699 JS700 JS701 JS702 JS703 JS704 JS705 JS706 JS707 JS708 JS709 JS710 JS711 JS712 JS713 JS714 JS715 JS716 JS717 JS718 JS719 JS720 JS721 JS722 JS723 JS724 JS725 JS726 JS727 JS728 JS729 JS730 JS731 JS732 JS733 JS734 fur-42 : : tetAR hilE115 : : aadA hilD114 : : Cm att : : pDX1 : : hilA - lac fur-42 : : tetAR hilE115 : : aadA hilC113 : : Cm att : : pDX1 : : hilA - lac fur-42 : : tetAR hilE115 : : aadA rtsA5 att : : pDX1 : : hilA - lac ( hilC - lac ) 131 rtsA5 ( hilC - lac ) 131 hilD114 : : Cm ( hilC - lac ) 131 hilD114 : : Cm rtsA5 ( hilC - lac ) 131 ( rtsA - lac ) 9 hilD114 : : Cm ( rtsA - lac ) 9 hilC113 : : Cm ( rtsA - lac ) 9 hilC-D2915 : : Cm ( rtsA - lac ) 9 fur-42 : : tetAR ( hilC - lac ) 131 fur-42 : : tetAR rtsA5 ( hilC - lac ) 131 fur-42 : : tetAR hilD114 : : Cm ( hilC - lac ) 131 fur-42 : : tetAR hilD114 : : Cm rtsA5 ( hilC - lac ) 131 fur-42 : : tetAR ( rtsA - lac ) 9 fur-42 : : tetAR hilD114 : : Cm ( rtsA - lac ) 9 fur-42 : : tetAR hilC113 : : Cm ( rtsA - lac ) 9 fur-42 : : tetAR hilC-D2915 : : Cm ( rtsA - lac ) 9 ( hilD - lac ) 138 pLS119 ( hilD - lac ) 138 pJE19 ( hilD - lac ) 138 fur-42 : : tetAR ( hilD - lac ) 138 pWKS30 ( sodB111 - lac ) hyb pFUR ( sodB111 - lac ) hyb yjeT : : hilD123 : : Km yjeT : : ( hilD - lac ) 123 yjeT : : PlacUV5 ( hilD - lac ) 123 yjeT : : ( hilD - lac ) hyb123 yjeT : : PlacUV5 ( hilD - lac ) hyb123 pWKS30 yjeT : : ( hilD - lac ) 123 pFUR yjeT : : ( hilD - lac ) 123 pBAD30 yjeT : : ( hilD - lac ) 123 pLS119 yjeT : : ( hilD - lac ) 123 pWKS30 yjeT : : PlacUV5 ( hilD - lac ) 123 pFUR yjeT : :P ( hilD - lac lacUV5 ) 123 pBAD30 yjeT : :P ( hilD - lac lacUV5 ) 123 pLS119 yjeT : : PlacUV5 ( hilD - lac ) 123 pWKS30 hilD114 : : Cm yjeT : : ( hilD - lac ) 123 pFUR hilD114 : : Cm yjeT : : ( hilD - lac ) 123 pBAD30 hilD114 : : Cm yjeT : : ( hilD - lac ) 123 pLS119 hilD114 : : Cm yjeT : : ( hilD - lac ) 123 pWKS30 hilD114 : : Cm yjeT : : PlacUV5 ( hilD - lac ) 123 pFUR hilD114 : : Cm yjeT : : PlacUV5 ( hilD - lac ) 123 pBAD30 hilD114 : : Cm yjeT : : PlacUV5 ( hilD - lac ) 123 pLS119 hilD114 : : Cm yjeT : : PlacUV5 ( hilD - lac ) 123 pWKS30 yjeT : : ( hilD - lac ) hyb123 pFUR yjeT : : ( hilD - lac ) hyb123 pBAD30 yjeT : : ( hilD - lac ) hyb123 pLS119 yjeT : : ( hilD - lac ) hyb123 pWKS30 yjeT : : PlacUV5 ( hilD - lac ) hyb123 pFUR yjeT : : PlacUV5 ( hilD - lac ) hyb123 pBAD30 yjeT : : PlacUV5 ( hilD - lac ) hyb123 pLS119 yjeT : : PlacUV5 ( hilD - lac ) hyb123 pWKS30 hilD114 : : Cm yjeT : : ( hilD - lac ) hyb123 pFUR hilD114 : : Cm yjeT : : ( hilD - lac ) hyb123 pBAD30 hilD114 : : Cm yjeT : : ( hilD - lac ) hyb123 pLS119 hilD114 : : Cm yjeT : : ( hilD - lac ) hyb123 pWKS30 hilD114 : : Cm yjeT : : PlacUV5 ( hilD - lac ) hyb123 pFUR hilD114 : : Cm yjeT : : PlacUV5 ( hilD - lac ) hyb123 pBAD30 hilD114 : : Cm yjeT : : PlacUV5 ( hilD - lac ) hyb123 pLS119 hilD114 : : Cm yjeT : : PlacUV5 ( hilD - lac ) hyb123 hilD123 : : Km 3018495 -- 3018729 Plasmids pKD46 pCP20 14 11 bla PBAD gam bet exo pSC101 oriTS bla cat cI857 PR ﬂp pSC101 oriTS Continued on following pag To see if Fur was involved in this metal regulation , we deleted fur in the hilA-lac fusion background and monitored expression with and without dipyridyl .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
<li>Strain or plasmid S. enterica serovar Typhimurium strains 14028s 41 JH352 42 HK427 This study DN263 This study EN504 This study DN235 This study Wild-type 14028s fur 14028s ssrB-HA 14028s fur ssrB-HA 14028s ssrB : : Kmr 14028s ssrB : : Kmr ssrB-HA-R fur SM-fur ( H90R ) - F SM-fur ( H90R ) - R Q-ssaB-F Q-ssaB-R Q-ssaG-F Q-ssaG-R Q-ssaM-F Q-ssaM-R Q-sseA-F Q-sseA-R Q-ssrB-F Q-ssrB-R Q-entC-F Q-entC-R Q-rrsH-F Q-rrsH-R EMSA-ssrB-F EMSA-ssrB-R Plasmids pUHE21-2lacIq P rep r q FIG 1 The Fur regulator negatively controls SPI-2 and iron-responsive genes inside macrophages .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24643535" target="_blank">24643535</a>]<br></li>
<li>A A R A A A A A HU MarT SpvR FimWYZ FlhDC FliZ A A A A A R A R A A R CsgD RtsB A A * R * R Lrp Lon DnaK Fur Mlc RNase E R R R A A A A R FadD Pag A R SlyA A a A , activation ; R , repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23554419" target="_blank">23554419</a>]<br></li>
<li>1 The abbreviations used are : Fur , ferric uptake regulator protein ; CFU , colony-forming unit ( s ) ; AP , abasic ; 1,10-phen , 1,10-phenanthro-line ; NCE , no carbon E medium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12670952" target="_blank">12670952</a>]<br></li>
<li>Reference Bioﬁlm-related function Group Quorum ( Merighi et al. 2009 ) a Membrane protein , Quorum Sensing ( AI-3 ) regulated pmrC sensing Metabolism Metabolism Putrescine transporter potF pps ( Hermans et al. 2011 ) ( White et al. 2010 ) Metabolism ( phosphoenolpyruvate synthase , conversion of pyruvate to PEP ) Needle complex inner membrane protein ( T3SS ) , PhoPQ repressed Purine synthesis , adenylosuccinate synthetase Purine synthesis , amidophosphoribosyltransferase Purine synthesis , bifunctional phosphoribosylaminoimidazolecarboxamide formyltransferase/IMP cyclohydrolase Activation of colanic-acid-biosynthesis , transcriptional co-regulator Activation of colanic-acid-biosynthesis , inhibition ﬂagellar biosynthesis Stationary phase sigma factor , stress response Synthetic RpoS-dependent promoter Regulates SPI-1 expression together with HilD and HilC Hypothetical protein ( similar to E. coli vancomycin sensitivity gene ) Responses to Acyl Homoserine Lactones ( AI-1 ) , inﬂuences plasmid encoded ﬁmbriae Transcriptional regulator in LysR family ( regulates motility in B. subtilis ) Response regulator of the BarA/SirA two-component system Fur regulated Salmonella iron-transporter Secreted effector protein ( T3SS ) prgH ( Steenackers et al. 2012 ) ( McClelland et al. 2001 ) a ( McClelland et al. 2001 ) a ( McClelland et al. 2001 ) a Virulence purA purF purH Metabolism Metabolism Metabolism rcsA ( Steenackers et al. 2012 ) Matrix rcsC ( Steenackers et al. 2012 ) Matrix rpoS ( Steenackers et al. 2012 ) ( White et al. 2006 ) ( Golubeva et al. 2012 ) ( Hermans et al. 2011 ) Regulator Synthetic Virulence Stress resistance Quorum sensing Putative sig38H4 rtsA sanA sdiA ( Steenackers et al. 2012 ) sinR ( Folkesson et al. 2002 , Marchal et al. 2004 ) ( Steenackers et al. 2012 ) Regulator sirA Metabolism Virulence sitA ( Hermans et al. 2011 ) sopB ( Hernandez et al. 2004 , Jennings et al. 2012 ) ( Steenackers et al. 2012 ) a Quorum sensing Coiled-coil protein , regulated by SdiA srgE ( Hansen et al. 2005 ) b Metabolism Stringent starvation protein , inhibits stationary-phase accumulation of H-NS Chaperon for sth ﬁmbriae synthesis , bioﬁlm formation on epithelial cells Putative cytoplasmic protein Inner membrane protein Putative ABC transporter periplasmic binding protein Putative cytoplasmic protein Diguanylate cyclase ( c-di-GMP-synthesis ) , activates cellulose production ( matrix ) Putative DNA-binding protein Putative DNA-binding protein Transcriptional regulator ( ﬂagella synthesis and pathogenicity ) Glycosyl transferase for the biosynthesis of colanic acid ( matrix ) Regulator , reduces bioﬁlm formation and induces motility Hypothetical protein Cell wall-associated hydrolase , contains a CsgD-binding site Regulatory protein Hydroxybutyrate dehydrogenase , O-Ag-capsule biosynthesis Regulator of O-Ag-capsule biosynthesis DNA-binding transcriptional repressor FabR , fatty-acid-biosynthesis Synthetic promoter , sigma-70 responsive ( growth control ) Synthetic promoter , sigma-70 responsive ( growth control ) sspA Fimbriae sthA ( Steenackers et al. 2012 ) Putative Putative Putative Putative Matrix sciJ ( Hermans et al. 2011 ) Hermans et al. unpublished ( Hermans et al. 2011 ) ( Hermans et al. 2011 ) ( Steenackers et al. 2012 ) STM0731 STM1255 STM1851 STM1987 Putative Putative Regulator STM3071 STM4423 tdcA ( Hermans et al. 2011 ) ( Hermans et al. 2011 ) ( Golubeva et al. 2012 ) Matrix wcaA ( Steenackers et al. 2012 ) Regulator Putative Putative ybaJ ybeL ydhO ( Barrios et al. 2006 ) ( Hermans et al. 2011 ) ( Ogasawara et al. 2011 ) Putative Matrix yhiW yihU ( Morgan et al. 2004 ) ( Steenackers et al. 2012 ) Matrix yihVW yijC ( Steenackers et al. 2012 ) ( Feng and Cronan 2009 ) a Metabolism ( Kim and Surette 2006 ) ( Stocki et al. 2007 ) σ70-16 σ70-7 Synthetic Synthetic Note : References for the bioﬁlm-related functions in Salmonella are shown .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24735176" target="_blank">24735176</a>]<br></li>
<li>) e , f , u i EnvZ/OmpR , PhoPQ PhoRB , CsrAB , SirA/BarA , HilA , HilC HilD , InvF b , c , f , g , h , i HilA , SirA/BarA j , , SPI4 ( svr genes ) Unknown SPI5 ( sopB , pips ) PMN migration b , c HilA , SirA/BarA j HilA , SirA/BarA j sopE Invasion , cytokine expression b , c , k , l Iron uptake a Iron limitation a Fur a Macrophage line ( IVET ) m entF Mg uptake 21 n Cation levels n PhoPQ n Macrophages mgtCB Mouse spleen and macrophage line ( IVET ) ; epithelial cell line ( other fusion studies ) m SPI2 Cation levels , Pi levels , osmolarity ( ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10844688" target="_blank">10844688</a>]<br></li>
<li>Proposed Virulence Proposed Environmental site in host genes functions conditions In vivo Regulatory factors induction Stomach atr genes Log-phase acid tolerance acid pH a response a Fur , RpoS a Adhesion b FimZ , AgfD b Fimbrial genes Intestine SPI1 Invasion , chemokine Oxygen , osmolarity , production , PMN migration , pH , growth phase , Apoptosis cations ( ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10844688" target="_blank">10844688</a>]<br></li>
<li>genes ( ArgP ) ( Fur ) ( DnaA ) Vitamin biosynthesis Infection genes b Essential genes c TF pred .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23427276" target="_blank">23427276</a>]<br></li>
<li>These predicted Fur or HNS regulated genes include genes associated with virulence as well those not related to virulence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23040276" target="_blank">23040276</a>]<br></li>
<li>Classiﬁcation of annotated iron-responsive promoters with respect to Fur regulation Maximum fold inductiona Function ( s ) Classe Gene Reference ( s ) iroBc , d A 3 , 4 8,900 Putative glycosyl transferase , related to UDP-glucuronosyltransferase/putative ATP binding cassette ( ABC ) transporter Isochorismate synthetase , enterochelin-biosynthesis Outer membrane porin , receptor for colicin I , requires TonB Outer membrane porin , receptor for ferric enterobactin ( enterochelin ) and colicins B and D Putative hemin uptake factor Ferric hydroxamate transport , involved in reduction of ferric iron in cytoplasmic ferrioxamine B Salmonella iron-transporter Putative DedA family , membrane protein Regulatory or redox component complexing with Bfr in iron storage and mobility Putative iron chelator utilization protein Putative transcription regulator ; AraC family Putative anaerobic dimethyl sulfoxide reductase , subunit A Putative periplasmic protein Energy transducer : uptake of iron , cyanocobalamin : sensitivity to phages , colicins Outer membrane protein receptor or transporter for ferrichrome , colicin M , and phages T1 , T5 , and phi80 ATP-dependent protease , Hsp 100 , part of novel multichaperone system with DnaK , DnaJ , and GrpE Cu ( I ) - translocating P-type ATPase Putative hemolysin Phosphoribosylpyrophosphate synthetase DNA polymerase [ III ] , chi subunit Invasion protein Putative cytoplasmic protein O-antigen ligase MFS superfamily , nitrite extrusion protein Putative molybdopterin oxidoreductase UDP-D-glucose : ( galactosyl ) lipopolysaccharide glucosyltransferase Putative cytochrome c peroxidase Putative aminotransferase in cobalamin-synthesis Propanediol utilization : polyhedral bodies Propanediol utilization : propanol dehydrogenase entCc , d cirAc fepAc , d 1,805 620 569b 6 , 7 , 26 17 , 18 , 20 , 53 47 ydiEd fhuFc 353 241 38 19 , 35 , 36 sitAc , d yqjA bfd 142b 86 81b 54 32 32 , 37 58b 55b 52b 46 22 yajHd 38 32 STM0363d STM4305 STM1586 tonBc 32 32 20 , 40 fhuAc , d 21 20 , 47 20b clpB 32 copA yafA 12 11b 32 32 32 32 32 32 32 32 32 32 32 32 32 32 2b 3 6 6b 7 17 26b 28 29b 36 274b 363b prsA holC invH ytfK rfaL narKd fdnG rfaJ STM3820 cobD pduA pduQ 42b Iron-dependent alcohol dehydrogenase of the multifunctional alcohol dehydrogenase AdhE Cytoplasmic trehalase Xanthosine phosphorylase ( purine nucleoside phosphorylase ) Putative thioredoxin-like proteins and domain Putative coiled-coil protein Membrane-associated protein of unknown function Fimbrial protein Z , putative transcriptional regulator ( LuxR/UhpA family ) Pyruvate kinase 1 ( formerly F ) , fructose stimulated Putative DNA/RNA nonspeciﬁc endonuclease Putative cytoplasmic protein Fumarase A ( fumarate hydratase class I ) , aerobic isozyme Putative cytoplasmic protein Soluble pyridine nucleotide transhydrogenase Secreted effector protein of Salmonella serovar Dublin Putative inner membrane protein Paral putative oxidoreductase Surface presentation of antigens : secretory proteins Putative phosphatase L-Rhamnose isomerase Molybdopterin-biosynthesis , protein A Ferritin-like protein Putative peptidase T ( aminotripeptidase ) MukF protein ( killing factor KicB ) Fumarate reductase , anaerobic , ﬂavoprotein subunit sn-glycerol-3-phosphate dehydrogenase ( anaerobic ) , large subunit Putative aspartate racemase Anaerobic dimethyl sulfoxide reductase , subunit A B adhE 32 treF xapAd 15 13 32 32 32 32 32 32 32 32 32 30 32 32 32 32 32 32 32 32 32 32 8b 4b 2b 4 4 4 4b 5b 5 5 6 9 9 10 10 16b 19 21 24 24 28 30b 59b 88 yhgl STM1554 ycfC ﬁmZ pykF STM1330 yhcGd fumAc STM1239 udhA sopA yhfK ysaA sicA STM3595 rhaA moaAd ftnB pepT STM0989 frdAd glpA STM4510 dmsAd 32 32 32 32 32 Continue TABLE 2 -- Continued Maximum fold inductiona Classe Gene Function ( s ) Reference ( s ) yqhC Putative transcriptional regulator ( AraC/XylS family ) 32 104 yfcZ Putative cytoplasmic protein 32 STM1851 Putative cytoplasmic protein 32 yhcO Putative cytoplasmic protein 32 STM4448 Putative periplasmic protein/putative phosphotransferase system mannitol / 32 fructose-speciﬁc IIA domain ( Ntr type ) yciG 6 Putative cytoplasmic protein 32 ydhI 4 Putative inner membrane protein 32 nagB 3 Glucosamine-6-phosphate deaminase 32 narW 3 Nitrate reductase 2 , delta subunit , assembly function 32 STM1537 4b Putative Ni/Fe-hydrogenase 1 b-type cytochrome subunit 32 STM4067 4 Putative ADP-ribosylglycohydrolase 32 cybC 5b Cytochrome b ( 562 ) 32 yeaA 6b Putative domain frequently associated with peptide methionine sulfoxide 32 reductase nirB 7b Nitrite reductase , large subunit 32 yfbE 8 LexA regulated , putative SOS-response 32 sifB 9 Salmonella translocated effector : translocated by SP1 -- 2 32 yjdE 10 Putative APC family , putrescine/ornithine-transport protein , cryptic 32 yliG 10 Putative Fe-S oxidoreductase family 1 32 ais 12 Aluminum-inducible protein 32 yﬁR 14 Putative periplasmic protein 32 aspAd 15b Aspartate ammonia-lyase ( aspartase ) 32 STM1255 15b Putative ABC transporter periplasmic binding protein 32 fucR 22b Positive regulator of the fuc operon ( DeoR family ) 32 STM3138d 23 Putative methyl-accepting chemotaxis protein 32 araE 28 MFS family , L-arabinose : proton symport protein ( low-afﬁnity transporter ) 32 eutS 28 Putative carboxysome structural protein , ethanol utilization 32 yhbU 29 Putative protease 32 STM4519 41b Putative NAD-dependent aldehyde dehydrogenase 32 mgsA 78b Methylglyoxal synthase 32 a Indicates approximate maximum induction in LB medium between high-iron ( positive numbers ) and low-iron ( negative numbers ) conditions 1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12897017" target="_blank">12897017</a>]<br></li>
<li>( continued ) Accession RE Gene GS Synthesis of vitamin B12 adenosyl cobalamide precursor Outer membrane porin , receptor for ferric enterobactin ( enterochelin ) and colicins B and D Nicotinamidase = pyrazinamidase Transcriptional regulator of cai and ﬁx operon Glutamate = aspartate-transporter Sigma F ( sigma 28 ) factor of RNA polymerase ABC-type transport systems Putative acyl-coA dehydrogenase Copper homeostasis protein Surface presentation of antigens Fur regulated salmonella iron-transporter Putative chemotaxis signal-transduction protein Regulator of ﬂagellar biosynthesis , Hook-ﬁlament junction protein 1 Fur regulated salmonella iron-transporter Related to carnitine-metabolism protein Putative ornithine carbamoyltransferase Dcu family anaerobic dicarboxylate transport protein Secretion system effector Homolog of sapA Response regulator in two-component regulatory system with PhoR ( or CreC ) PhoB-dependent ATP-binding pho regulon component Secretion system effector Putative thiol-alkyl hydroperoxide reductase Putative outer membrane protein Secretion system chaparone Secretion system effector Response regulator in two-component regulatory system with RstB ( OmpR family ) aga operon transcriptional repressor Putative hemolysin Outer membrane protease , receptor for phage OX2 Putative HlyD family secretion protein Putative cytoplasmic protein MFS superfamily nitrate extrusion protein Secretion system apparatus protein Secretion system apparatus protein PhoPQ-activated gene Putative PTS permease Homology with cold shock proteins Cytochrome o ubiquinol oxidase subunit III Gifsy-2 prophage putative type III secreted protein Secretion system regulator : transcriptonal activator Putative component in type IV pilin biogenesis Secretion system apparatus protein Secretion system apparatus protein Secretion system regulator : Sensor component PhoPQ-activated protein Salmonella translocated effector Secretion system effector Peptide methionine sulfoxide reductase PhoPQ-activated protein Putative transcriptional regulator in curly assembly = transport , 2nd curli operon Gifsy-2 prophage superoxide dismutase precursor ( Cu-Zn Putative outer membrane receptor Outer membrane protein E Putative outer membrane protein Reduced macrophage survival protein Similar to virK in Shigella PhoP regulated AAL20927 AAL19536 5.2 4.4 cbiM fepA AAL20218 AAL19032 AAL19615 AAL20868 AAL20413 AAL19266 AAL20823 AAL21772 AAL21742 AAL21215 AAL20841 AAL20113 AAL21743 AAL19041 AAL23284 AAL23148 AAL20325 AAL19263 AAL19351 4.4 4.1 3.9 3.9 3.9 3.9 3.8 3.8 3.5 3.5 3.5 3.4 3.4 3.2 3.2 3.0 3.0 3.0 3.1 pncA caiF gltJ ﬂiA opuBB yafH cutC invJ sitB cheW ﬂhD ﬂgK sitC ﬁxC argF dcuA sseD sapA phoB 3.3 3.6 3.6 3.6 3.7 3.7 4.0 AAL20057 AAL20326 AAL19356 AAL19492 AAL20327 AAL20320 AAL20394 phoH sseE STM0402 STM0538 sscB ssaE rstA 4.7 4.8 AAL22124 AAL20200 AAL19769 AAL21030 AAL20482 AAL20494 AAL20318 AAL20319 AAL19579 AAL23353 AAL20172 AAL19396 AAL19985 AAL20315 AAL19106 AAL20317 AAL20339 AAL20316 AAL20777 AAL20549 AAL20324 AAL23228 AAL20781 AAL20069 glpR yaoF 4.9 5.0 5.1 5.4 5.5 5.6 6.6 7.8 8.3 8.4 9.6 9.7 10.1 10.6 10.6 11.1 11.9 13.0 15.1 16.0 16.9 17.0 ompX acrA yddX narU ssaC ssaD pagP STM4535 cspH cyoC sseI ssrB hofC ssaB ssaN ssrA pagO sseJ sseC msrA pagK csgG sodC yncD 17.9 18.4 22.8 23.1 25.0 33.2 38.6 AAL19978 AAL20505 AAL21295 AAL21147 AAL20174 AAL21666 AAL20173 ) pgtE STM2245 pagC virK pagD a These are genes that are expressed greater than threefold .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18407759" target="_blank">18407759</a>]<br></li>
<li>GENES OF S. TYPHIMURIUM LUXS MUTANT PJ002 INFLUENCED BY IN VITRO SYNTHESIZED AUTOINDUCER 2 ( AI-2 ) Fold change Gene function or product Gene P sitC 10.8 2.3 2.1 2.0 1.9 1.9 1.8 1.7 1.6 1.5 1.5 Fur regulated Salmonella iron-transporter Cell invasion protein Chaperone Hsp90 , heat shock protein C 62.5 Putative outer membrane lipoprotein Oligopeptidase A O-antigen translocase in LPS biosynthesis Putative fimbriae Suppresses inhibitory activity of csra Putative fimbriae ; chaparone Sigma S ( sigma 38 ) factor of RNA polymerase Nicotinate-nucleotide dimethylbenzimmidazole-P phophoribosyl transferase Putative outer membrane protein Putative outer membrane protein Putative universal stress protein uspa and related nucleotide-binding protein Tol protein , role in outer membrane integrity Putative GPH family transport protein Heat shock protein hslj Putative cytoplasmic protein Putative Zn-dependent hydrolases of the betalactamase fold Putative metal-binding protein Maltodextrin glucosidase Putative cytoplasmic protein Putative cytoplasmic protein 0.050 0.027 0.002 0.003 0.030 0.020 0.025 0.045 0.031 0.026 0.039 prgH htpG yhdV prlC wzxE stiH tldD stbE rpoS cobT 0.045 0.033 0.022 1.7 1.7 1.8 STM1131 STM0305 ybdQ 0.039 0.023 0.028 0.009 0.039 1.8 1.8 1.8 2.1 2.3 tolR yicJ hslJ STM2868 yifR 0.039 0.020 0.007 0.019 2.3 2.5 3.1 3.3 yecA malZ STM4255 yjiS There was a significant difference ( P 0.05 ) in the gene-expression patterns in 23 genes for the AI-2 treatment compared to the control PB treatment , with roughly half the genes being upregulated ( Table 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17378703" target="_blank">17378703</a>]<br></li>
<li>Genes Differentially Expressed ( FDR < 5 % ) in 2a Based upon Microarray Comparisons a Accession RE Gene GS AAL20730 AAL21761 AAL20023 AAL20490 AAL20022 AAL21753 AAL21754 AAL21775 AAL20021 AAL21763 AAL22362 AAL19500 AAL21766 AAL19551 AAL19499 AAL19498 AAL20835 AAL21762 AAL20872 AAL20838 AAL23285 AAL21759 AAL21776 AAL21777 AAL21774 AAL20871 AAL21760 AAL19822 AAL20024 AAL20906 AAL19933 AAL23063 AAL19497 AAL21747 AAL23226 AAL21779 AAL21301 AAL21770 AAL21778 AAL19683 AAL21752 AAL20731 AAL23064 AAL20832 AAL21764 AAL20834 AAL19879 AAL21063 AAL21765 AAL21773 AAL20401 AAL21750 AAL23055 AAL22926 AAL21744 AAL21767 AAL19501 AAL23352 AAL21564 AAL21899 AAL20867 AAL21905 Cell division inhibitor Putative acyl carrier protein minD iacP 45.0 45.0 41.0 40.4 40.1 38.2 36.9 35.7 35.7 34.0 32.9 32.7 32.6 32.1 31.5 30.5 28.0 25.0 25.0 25.0 25.0 25.0 25.0 25.0 25.0 25.0 25.0 25.0 25.0 25.0 25.0 22.7 22.6 22.6 21.7 21.6 21.4 21.1 20.0 19.3 19.1 18.6 17.4 13.6 12.1 9.8 9.6 8.7 8.4 8.2 7.9 7.8 7.1 7.1 6.7 6.5 6.3 6.1 5.9 5.9 5.5 5.3 Pathogenicity island encoded protein : SPI5 New outer membrane protein Pathogenicity island encoded protein : SPI5 Cytoplasmic cell invasion protein Cell invasion protein Surface presentation of antigens Pathogenicity island encoded protein : SPI5 Cell invasion protein Phosphoenolpyruvate carboxykinase Outer membrane usher protein Surface presentation of antigens Carbon starvation protein Periplasmic chaperone , required for type 1 ﬁmbriae Fimbrial protein internal segment Methyl accepting chemotaxis protein II Cell invasion protein Filament capping protein Enables ﬂagellar motor rotation Putative carbamate kinase Chaperone related to virulence Invasion protein A Invasion protein E Surface presentation of antigens Flagellar biosynthesis ; ﬂagellin Putative cytoplasmic protein Arginine 3rd transport system Pathogenicity island encoded protein : SPI5 Putative cold-shock protein Outer membrane protein 1a ( ia ; b ; f ) , porin Putative cytoplasmic protein Major type 1 subunit ﬁmbrin ( pilin ) AraC family bacterial regulatory helix-turn-helix protein Putative cytoplasmic protein Invasion protein Cold shock-induced palmitoleoyl transferase Surface presentation of antigens Outer membrane invasion protein Succinyl-coA synthetase , alpha subunit Cytoplasmic cell invasion protein Cell division topological speciﬁcity factor Putative cytoplasmic protein Chemotaxis regulator protein Cell invasion protein Glutamate methyltransferase Similar to cspa but not cold shock induced Putative sensor = kinase in regulatory system Cell invasion protein Surface presentation of antigens Putative membrane transporter of cations and cationic drugs Putative inner membrane protein Phage lambda receptor protein Glycerol kinase Fur regulated Salmonella iron-transporter Surface presentation of antigens Minor ﬁmbrial subunit NtrC family transcriptional regulator 16S rRNA-processing protein Putative cytoplasmic protein Putative regulator of ﬂia Putative ﬁmbrial-like protein sopB nmpC pipC prgI prgH invB STM1089 sipD pckA ﬁmD sicA cstA ﬁmC ﬁmI cheM sipA ﬂiD motB arcC sicP invA invE invC ﬂiC STM2880 artJ orfX cspB ompF STM4239 ﬁmA hilC ytfK invF ddg spaP invG sucD prgJ minE yjbJ cheY sipC cheR cspD yehU sipB invI ydgF orgA lamB glpK sitD spaS ﬁmH pspF rimM yohL ﬂiZ stdA ( continued Table 3 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18407759" target="_blank">18407759</a>]<br></li>
<li>THE GENES PREDICTED BY MICROARRAY TO BE REGULATED IN SALMONELLA ENTERICA TYPHIMURIUM 14028 EXPOSURED TO 1.6 MG/ML OF NALIDIXIC ACID , SORTED ACCORDING TO FOLD REGULATION ( 50 AND 50 INDICATE FOLD REGULATION ESTIMATED BEYOND 50-FOLD ) ( CONTINUED ) Cytochrome-related transporter Putative cytoplasmic protein 2.2 2.1 cydD ydhH membrane 0.008 0.006 16419468 16419966 cytoplasmic protein fimbria hydrolase virulence transport periplasmic protein virulence oxidoreductase Putative fimbriae ; usher Exonuclease I 2.1 2.1 2.1 2.1 2.1 stbD sbcB spaS fdx ynfB 0.007 0.001 0.009 0.005 0.001 16418837 16420598 16421435 16421083 16420023 Surface presentation of antigens Ferredoxin Putative periplasmic protein Regulator in two-component system crec Gifsy-2 prophage superoxide dismutase precursor ( Cu-Zn ) Putative fimbrial protein Methionine trna synthetase Tryptophan synthase , beta protein 2.1 2.1 creB sodC 0.003 0.009 2.1 2.0 2.0 fimF metG trpB 0.006 0.002 0.008 16419059 16420690 16420256 fimbria atp binding tryptophan-biosynthesis cytoplasmic protein membrane cell-division oxidoreductase resistance transcription membrane Putative cytoplasmic protein 2.0 0.000 16419684 yceH Outer membrane protein W 2.0 2.0 2.0 2.0 2.0 2.0 0.002 0.001 0.001 0.001 0.002 0.011 16420263 16420347 16422620 16420410 16423073 16419079 ompW minE sodA yobA uxuR apeE Cell division topological specificity factor Superoxide dismutase Similar Cu resistance protein copc Transcriptional repressor for uxu operon Outer membrane N-acetyl phenylalanine esterase Cysq protein Putative outer membrane lipoprotein Transcriptional regulator UDP-N-acetyl-D-mannosaminuronic acid dehydrogenase Flagellar biosynthesis protein Putative outer membrane or exported protein Proteolytic subunit of clpa-clpp Putative outer membrane protein Putative oxidoreductase Carbon storage regulator Putative outer membrane protein , ompa family Fimbrial chaperone CPA2 family K efflux antiporter , glutathione-regulated Modulator of enterobacterial common antigen length Filament structural protein Component in transcription antitermination Enables flagellar motor rotation CDP-abequose synthase Homology to outer membrane efflux protein Fur regulated Salmonella iron-transporter Acridine efflux pump Putative periplasmic protein 2.0 2.0 2.0 2.0 cysQ yjbF sinR wecC 0.011 0.003 0.013 0.010 16422966 16422780 16418806 16422489 metal-binding lipoprotein transcription oxidoreductase 2.0 2.0 flagellum membrane flgG 0.013 0.002 16419696 16422800 STM4242 protease membrane 2.1 2.1 2.1 2.1 2.1 clpP 0.000 0.005 0.000 0.001 0.003 16418955 16422112 16420707 16421371 16418782 yhhA yohF csrA OmpA oxidoreductase rna-binding membrane fimbria 16418521 16418583 0.001 0.003 2.1 2.1 bcfG kefC membrane membrane 16422487 0.006 2.1 wzzE flagellum fijB 16421319 16422705 16420459 16420621 16418851 0.000 0.008 0.008 0.005 0.001 2.2 2.3 2.3 2.3 2.3 transcription flagellum LPS membrane NusG motB rfbJ TolC membrane membrane periplasmic protein virulence sitD acrA ytfN 16421410 16418983 16422972 0.005 0.005 0.004 2.3 2.4 2.4 16420491 0.002 Sigma F ( sigma 28 ) factor of RNA polymerase Regulator of length of O-antigen LPS Putative fimbrial chaparone protein 2.4 fliA membrane fimbria wzzB stdC 2.4 2.4 16420611 16421578 0.011 0.005 TABLE 1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17600486" target="_blank">17600486</a>]<br></li>
<li>THE GENES PREDICTED BY MICROARRAY TO BE REGULATED IN SALMONELLA ENTERICA TYPHIMURIUM 14028 EXPOSURED TO 1.6 MG/ML OF NALIDIXIC ACID , SORTED ACCORDING TO FOLD REGULATION ( 50 AND 50 INDICATE FOLD REGULATION ESTIMATED BEYOND 50-FOLD ) ( CONTINUED ) SOS-response regulator Activator protein 2.5 2.5 2.6 lexA pgtA potF transcription transcription-transport 0.001 0.001 0.011 16422795 16420940 16419385 ABC superfamily ( ) , putrescine-transporter Flagellar biosynthesis protein Putative salt-induced outer membrane protein Sigma E ( sigma 24 ) factor of RNA polymerase Rhamnosyl transferase DNA-replication initiator protein Glycosyltransferase Putative permease DNA-damage-inducible protein F Putative transcriptional regulatory protein Phob-dependent ATP-binding pho regulon component Putative invasin Outer membrane lipoprotein Putative fimbrial subunit Multiple antibiotic resistance protein Putative marr-family transcriptional regulator 1,4-alpha-glucan branching enzyme Putative oxidoreductase Putative cytoplasmic protein 2.6 2.6 fliM ydiY flagellum membrane 0.003 0.014 16420511 16419845 2.6 0.002 16421189 rpoE transcription 2.6 2.6 2.7 2.7 2.7 2.7 2.7 0.002 0.000 0.005 0.001 0.002 0.001 0.000 16420617 16422412 16422182 16420524 16422796 16422328 16419642 rfbN dnaA yhjO yedA dinF marT phoH transferase atp binding membrane membrane dna damage transcription atp binding 2.7 2.7 2.8 2.8 2.9 ychP blc stiA marB MarR 0.000 0.000 0.001 0.008 0.002 16420297 16422901 16418679 16420039 16420069 virulence lipoprotein fimbria transcription transcription glgB ydgJ 3.0 3.0 3.0 0.000 0.009 0.004 16422097 16419983 16422797 transferase oxidoreductase cytoplasmic protein metal binding cell-division , dna damage STM4239 FkbR2 ftsW uvrB Putative monoamine oxidase Essential cell-division gene 3.0 3.0 3.1 0.000 0.005 0.006 16421672 16418626 16419304 Uvrb with uvrac is a dna excision repair enzyme DNA primase Transcriptional activator of tdc operon Glycosyl hydrolase Putative outer membrane lipoprotein UDP-N-acetylmuramoylalanine-D-glutamate ligase Positive lysr family transcriptional regulator Fur regulated Salmonella iron-transporter Fur regulated Salmonella iron-transporter Regulator for maltose-metabolism UDP-N-acetylglucosamine : N-acetylmuramyl-transferase Putative outer membrane lipoprotein Component of Sec-independent protein-secretion pathway Iron-sulfur component of fhuf stability protein Rod shape-determining protein MFS superfamily nitrate extrusion protein Cytochrome BD2 subunit I Transcriptional regulator of Aas Suppressor of lon Putative outer membrane or secreted lipoprotein Transcriptional repressor of cel operon Cysteine sulfinate desulfinase Hemolysin expression modulating protein Uptake of enterochelin dnaG tdcA 3.1 3.2 3.2 3.2 3.3 transferase transcription hydrolase lipoprotein atp binding 0.005 0.000 0.000 0.002 0.000 16421767 16421803 16422096 16422811 16418625 glgX STM4253 murD transcription membrane membrane transcription membrane 0.000 0.002 0.000 0.002 0.004 3.3 3.3 3.3 3.4 3.4 LysR sitC sitB SfsA murG 16421564 16421409 16421408 16418690 16418627 lipoprotein membrane 16418577 16422539 0.003 0.001 3.4 3.5 DUF903 tatB metal-binding , 16419892 0.001 3.7 sufD 16421932 16420099 16418862 16420210 16419586 16420083 0.000 0.009 0.000 0.006 0.000 0.000 3.7 3.7 3.8 3.8 3.8 3.9 mreD narU membrane oxidoreductase transcription membrane lipoprotein CydA tyrR sulA DUF1264 transcription transcription virulence membrane celD deaD hha exbB 4.0 4.1 4.3 4.3 16419833 16421839 16418980 1642171 0.004 0.002 0.001 0.000 TABLE 1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17600486" target="_blank">17600486</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_16">
Ellermeier , J.R. , and Slauch , J.M. ( 2008 ) Fur regulates expression of the Salmonella pathogenicity island 1 type-III-secretion-system through HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19007416" target="_blank">19007416</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24354910" target="_blank">24354910</a>]<br></label>
<input id="rp_similarity_16", type="checkbox">
<div class="RP_similar">
<li>Ellermeier , J. R. , and J. M. Slauch , 2008 Fur regulates expression of the Salmonella pathogenicity island 1 type-III-secretion-system through HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20008574" target="_blank">20008574</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22021388" target="_blank">22021388</a>]<br></li></div>
				<label class="RP_collapse_similarity" for="rp_similarity_18">
Fur regulates expression of the Sal-monella pathogenicity island 1 type-III-secretion-system through HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27704705" target="_blank">27704705</a>]<br></label>
<input id="rp_similarity_18", type="checkbox">
<div class="RP_similar">
<li>Fur regulates expression of the Salmo-nella pathogenicity island 1 type-III-secretion-system through HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23554419" target="_blank">23554419</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24375101" target="_blank">24375101</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26386070" target="_blank">26386070</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27601571" target="_blank">27601571</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25547794" target="_blank">25547794</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24990908" target="_blank">24990908</a>]<br></li></div>
				<label class="RP_collapse_similarity" for="rp_similarity_20">
[ 15 ] J.R. Ellermeier , J.M. Slauch , Fur regulates expression of the Salmonella pathogenicity island 1 type-III-secretion-system through HilD , J. Bacteriol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24937451" target="_blank">24937451</a>]<br></label>
<input id="rp_similarity_20", type="checkbox">
<div class="RP_similar">
<li>[ 17 ] J.R. Ellermeier , J.M. Slauch , Fur regulates expression of the Salmonella pathogenicity island 1 type-III-secretion-system through HilD , J. Bacteriol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23040276" target="_blank">23040276</a>]<br></li>
<li>Ellermeier JR , Slauch JM : Fur regulates expression of the Salmonella pathogenicity island 1 type-III-secretion-system through HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22017966" target="_blank">22017966</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24079299" target="_blank">24079299</a>]<br></li>
<li>Ellermeier , J. R. & Slauch , J. M. Fur regulates expression of the Salmonella pathogenicity island 1 type-III-secretion-system through HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27886269" target="_blank">27886269</a>]<br></li>
<li>[ 26 ] J.R. Ellermeier , J.M. Slauch , Fur regulates expression of the Salmonella pathogenicity island 1 type-III-secretion-system through HilD , J. Bacter-iol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18656407" target="_blank">18656407</a>]<br></li>
<li>Fur regulates expression of the Salmonella pathogenicity island 1 type-III-secretion-system through HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24239962" target="_blank">24239962</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26441883" target="_blank">26441883</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26300871" target="_blank">26300871</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21075923" target="_blank">21075923</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19884332" target="_blank">19884332</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23306672" target="_blank">23306672</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20889744" target="_blank">20889744</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28575106" target="_blank">28575106</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27242152" target="_blank">27242152</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21680637" target="_blank">21680637</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21722794" target="_blank">21722794</a>]<br></li>
<li>Fur regulates expression of the Salmonella patho-genicity island 1 type-III-secretion-system through HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19049822" target="_blank">19049822</a>]<br></li>
<li>Fur regulates expression of the Salmonella pathogeni-city island 1 type-III-secretion-system through HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23831078" target="_blank">23831078</a>]<br></li></div>
<li>Taken together , these data indicate that Fur acts as a positive regulator of hilA expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
<li>A variation of this model is that Fur indirectly regulates a factor that positively affects HilD function ; e.g. , Fur represses a repressor of a factor that assists HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
<li>Fur regulates expression of the Salmonella pathogenicity Gaballa A , Antelmann H , Aguilar C , Khakh SK , Song KB , Smaldone GT , et al. .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22824499" target="_blank">22824499</a>]<br></li>
<li>Fur negatively controls SPI-2 expression via repression of ssrB transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24643535" target="_blank">24643535</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_32">
Fur regulates expression of hilA via HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></label>
<input id="rp_similarity_32", type="checkbox">
<div class="RP_similar">
<li>Fur regulates expression of rtsA via HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li></div>
<li>Fur regulates expression of hilA by working through HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
<li>Fur regulates expression of rtsA by controlling HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
<li>In this work , we demonstrated that Fur regulates expression of hilA via control of HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
<li>RESULTS Fur induces expression of hilA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
<li>Fur activates expression of hilA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
<li>OxyR and not Fur is essential for survival during adrenaline exposure Oxidative stress resistance in bacterial cells is mediated by enzymatic as well as non-enzymatic methods involving manganese [ 53,56 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18837991" target="_blank">18837991</a>]<br></li>
<li>We have now identiﬁed a novel regulatory pathway that controls Fur activity in S. enterica .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18790861" target="_blank">18790861</a>]<br></li>
<li>Figure 1 A demonstrates that Fur is required for virulence in Nramp / ( C3H/HeN ) mice .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21075923" target="_blank">21075923</a>]<br></li>
<li>One metabolic pathway that appears impacted by Fur is that required for glycerol-metabolism .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22017966" target="_blank">22017966</a>]<br></li>
<li>Our data showed that Fur represses tcf expression ( Fig. 4b ) , whereas low iron conditions increased it .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26944792" target="_blank">26944792</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_44">
The Fur regulator represses expression of SPI-2 genes during environmental acidiﬁcation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24643535" target="_blank">24643535</a>]<br></label>
<input id="rp_similarity_44", type="checkbox">
<div class="RP_similar">
<li>FIG 2 The Fur regulator represses SPI-2 expression during environmental acidiﬁcation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24643535" target="_blank">24643535</a>]<br></li></div>
<li>By this action on SsrB production , Fur was able to negatively control SPI-2 expression ( Fig. 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24643535" target="_blank">24643535</a>]<br></li>
<li>Of these , Fur is a reasonable candidate for contributing to PQ-induced sodA expression under aerobic growth .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10613858" target="_blank">10613858</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_48">
Tsolis , R.M. , Baumler , A.J. , Stojiljkovic , I. and Heffron , F. ( 1995 ) `` Fur regulon of Salmonella typhimurium : identiﬁcation of new iron-regulated genes '' , J. Bacteriol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12652904" target="_blank">12652904</a>]<br></label>
<input id="rp_similarity_48", type="checkbox">
<div class="RP_similar">
<li>Tsolis , R.M. , Bäumler , A.J. , Stojiljkovic , I. , and Heffron , F. ( 1995 ) Fur regulon of Salmonella typhimurium : identiﬁcation of new iron-regulated genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11929531" target="_blank">11929531</a>]<br></li>
<li>Tsolis , R.M. , Baumler , A.J. , Stojiljkovic , I. , and Heffron , F. ( 1995b ) Fur regulon of Salmonella typhimurium : identiﬁcation of new iron-regulated genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12492857" target="_blank">12492857</a>]<br></li>
<li>Tsolis , R.M. , Baumler , A.J. , Stojiljkovic , I. , and Heffron , F. ( 1995 ) Fur regulon of Salmonella typhimurium : identiﬁcation of new iron-regulated genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18031307" target="_blank">18031307</a>]<br></li></div>
				<label class="RP_collapse_similarity" for="rp_similarity_52">
Fur regulon of Salmonella typhimurium : identification of new ironregulated genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23306672" target="_blank">23306672</a>]<br></label>
<input id="rp_similarity_52", type="checkbox">
<div class="RP_similar">
<li>Fur regulon of Salmonella typhimurium : identiﬁcation of new ironregulated genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11932449" target="_blank">11932449</a>]<br></li></div>
				<label class="RP_collapse_similarity" for="rp_similarity_54">
[ 21 ] R. Tsolis , A. Baumler , I. Stojiljkovic , F. Heffron , Fur regulon of Salmonella typhimurium : identiﬁcation of new iron - regulated genes , J. Bacteriol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23040276" target="_blank">23040276</a>]<br></label>
<input id="rp_similarity_54", type="checkbox">
<div class="RP_similar">
<li>[ 28 ] R.M. Tsolis , A.J. Baumler , I. Stojiljkovic , F. Heffron , Fur regulon of Salmonella typhimurium : identiﬁcation of new iron-regulated genes , J. Bacteriol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16027021" target="_blank">16027021</a>]<br></li></div>
				<label class="RP_collapse_similarity" for="rp_similarity_56">
Fur regulon of Salmonella Typhimurium : identiﬁcation of new iron-regulated genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20237076" target="_blank">20237076</a>]<br></label>
<input id="rp_similarity_56", type="checkbox">
<div class="RP_similar">
<li>Fur regulon of Salmonella typhimurium : Identiﬁcation of new iron-regulated genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18835989" target="_blank">18835989</a>]<br></li>
<li>Fur regulon of Salmonella typhimurium : identiﬁcation of new iron-regulated genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15790293" target="_blank">15790293</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17720790" target="_blank">17720790</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12897017" target="_blank">12897017</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11948147" target="_blank">11948147</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10869064" target="_blank">10869064</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16237024" target="_blank">16237024</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8824613" target="_blank">8824613</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9495757" target="_blank">9495757</a>]<br></li></div>
				<label class="RP_collapse_similarity" for="rp_similarity_59">
Identiﬁcation and characterization of new iron-regulated Escherichia coli genes by a Fur titration assay .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8824613" target="_blank">8824613</a>]<br></label>
<input id="rp_similarity_59", type="checkbox">
<div class="RP_similar">
<li>Identiﬁcation and characterization of new iron-regulated Esche-richia coli genes by a Fur titration assay .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23873910" target="_blank">23873910</a>]<br></li></div>
				<label class="RP_collapse_similarity" for="rp_similarity_61">
Fur regulon in gram ¬ negative bacteria : identiﬁcation and characterization of new iron-regulated Escherichia coli genes by a fur titration assay .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659669" target="_blank">15659669</a>]<br></label>
<input id="rp_similarity_61", type="checkbox">
<div class="RP_similar">
<li>Fur regulon in Gramnegative bacteria : identiﬁcation and characterization of new iron-regulated Escherichia coli genes by a Fur titration assay .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28553268" target="_blank">28553268</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19114490" target="_blank">19114490</a>]<br></li></div>
<li>Fur has been well studied and is known to control many other genes , and this may be a way to coordinate Mn transport with other cellular functions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659669" target="_blank">15659669</a>]<br></li>
<li>Like Lrp , other core global regulators including , Fur and OmpR were found to control ipf expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28817673" target="_blank">28817673</a>]<br></li>
<li>We found that Fur represses SPI-2 expression that is activated inside macrophages and under acidic conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24643535" target="_blank">24643535</a>]<br></li>
<li>As Salmonella possesses several virulence genes that are absent from E. coli , it is possible that Fur and sRNAs RfrA and RfrB regulate distinct genes that may contribute to virulence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23306672" target="_blank">23306672</a>]<br></li>
<li>Unlike results for sitABCD , neither Fur nor MntR appears to be involved in mntR transcriptional regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659669" target="_blank">15659669</a>]<br></li>
<li>DISCUSSION Fur has been shown to be important in bacterial pathogenesis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24166960" target="_blank">24166960</a>]<br></li>
<li>Ferric uptake regulator ( Fur ) is a Fe2þ dependent transcriptional regulator in bacteria , which can act as an activator or repressor of gene-expression [ 17,19 -- 24 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23040276" target="_blank">23040276</a>]<br></li>
<li>Fur is a well-characterized transcriptional repressor that regulates gene-expression in response to iron ( 18 -- 20 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12897017" target="_blank">12897017</a>]<br></li>
<li>How Fur , which is thought to negatively regulate gene-expression in presence of Fe2þ positively contributes to SPI1 gene-expression is not well understood .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23040276" target="_blank">23040276</a>]<br></li>
<li>Since sit transcriptional activity increased during stationary phase in an RpoS-independent Fur-dependent manner , we examined whether Fur was also involved in mntR expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659669" target="_blank">15659669</a>]<br></li>
<li>Because Fur has recently been shown to play a critical role in Salmonella virulence ( 27 ) , we tested whether Fur regulates antinitrosative defenses in Salmonella pathogenesis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24166960" target="_blank">24166960</a>]<br></li>
<li>Fur has also been found to control virulence gene-expression in Salmonella .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23040276" target="_blank">23040276</a>]<br></li>
<li>Hassett DJ , Sokol PA , Howell ML , Ma JF , Schweizer HT , Ochsner U , Vasil ML : Ferric uptake regulator ( Fur ) mutants of Pseudomonas aeruginosa demonstrate defective siderophore-mediated iron uptake , altered aerobic growth , and decreased superoxide dismutase and catalase activities .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22017966" target="_blank">22017966</a>]<br></li>
<li>Interestingly , two promoters , ydiE ( putative hemin uptake factor ) and yqjH ( putative iron chelator utilization protein ) , predicted to be Fur regulated using a comparative genomic approach ( 38 ) , were also identi-ﬁed as such in this screen .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12897017" target="_blank">12897017</a>]<br></li>
<li>Notably , several proteins involved in iron utilization and uptake ( Fur , cytoplasmic ferritin , Fe-S cluster formation , bactoferrin , siderophore receptor TonB , iron-transport protein , iron-dependent alcohol dehydrogenase , and ferric enterobactin receptor ) and ribosome structure ( L7 , L32 , S20 , S13 , S11 S19 , L14 , L33 , S4 , L4 ) were identified as differentially expressed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19079590" target="_blank">19079590</a>]<br></li>
<li>The FeFur repressed sufABCDS operon , encoding an alternative Fe-S cluster biosynthetic pathway employed during redox stress and iron starvation in E. coli ( Outten et al. , 2004 ) , was also induced ( 3.77 - to 9.35-fold at 45 min ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28553268" target="_blank">28553268</a>]<br></li>
<li>Ferric Uptake Regulator ( Fur ) positive and Gram-negative bacteria to regulate genes that encode iron-uptake and - storage proteins as a repressor or activator , respectively ( Bsat et al. , 1998 ; Escolar et al. , 1999 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27242152" target="_blank">27242152</a>]<br></li>
<li>Fur represents a global regulator activated by iron and is involved in iron-homeostasis ( Escolar et al. , 1999 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26944792" target="_blank">26944792</a>]<br></li>
<li>The Fe-Fur repressed yqjH and bfd genes are considered to be involved in iron mobilization ( Miethke et al. , 2011 ; Yao et al. , 2012 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28553268" target="_blank">28553268</a>]<br></li>
<li>The iron-rationing response is controlled indirectly by Fur through RfrA and RfrB in Salmonella ( RyhB in E. coli ) and largely functions to repress genes encoding ironcontaining proteins in reaction to low iron conditions ( McHugh et al. , 2003 ; Massé et al. , 2005 ; Yang et al. , 2013 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28553268" target="_blank">28553268</a>]<br></li>
<li>Under iron-rich conditions , it has been shown that expression of ryhB gene is repressed by Fe2 + - Fur , whereas transcription of genes encoding iron-storage proteins is elevated ( Masse et al. , 2005 ; Oglesby et al. , 2005 ; Vecerek et al. , 2007 ; Ellermeier and Slauch , 2008 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27242152" target="_blank">27242152</a>]<br></li>
<li>Husain et al. , 2008 PMID : 18198179 McCollister et al. , 2011 doi : 10.1128 / AAC.01203-10 Husain et al. , 2008 PMID : 18198179 Husain et al. , 2014 PMID : 24166960 NO ( murine host ) → decrease in NADH dehydrogenase activity → NADH high in cytoplasm hydrogen peroxide protection → Direct detoxiﬁcation of NO by the NADH dehydrogenase ( RNS defense by acid-induced regulator Fur regulates NADH dehydrogenase ) Condition Result Author , weblink CARBOHYDRATE METABOLISM → Aconitase isoenzymes : acoA for oxidative stress → Repair of oxidized aconitase by bacterial frataxin ortholog proteins CyaY and YggX Carbohydrate metabolism adaptations of Salmonella during infection Baothman et al. , 2013 PMID : 23637460 Velayudhan et al. , 2014 PMID : 24421039 Bowden et al. , 2010 doi : 10.1371 / journal.pone .0013871 → Incomplete TCA helps survival and replication in resting and activated murine macrophages compared to wt → Epithelial cell infection : sucCD and gltA replicate less than wt → S. Typhimurium sucAB and sucCD attenuated in murine infection S. Typhimurium TCA-cycle mutations ( gltA , mdh , sdhCDAB , sucAB , and sucCD Inﬂuence Result Author , weblink THE BROAD INFLUENCE OF AMINO ACIDS ON METABOLIC ADAPTATION DURING INFECTION Amino acid decarboxylase systems consume protons , → Salmonella decarboxylases for lysine ( CadA ) , arginine raise cytosolic pH ( AdiA ) , and ornithine ( SpeF ) , not glutamate → acid tolerance but not essential for virulence in mice Alvarez-Ordóñez et al. , 2010 PMID : 19864032 Viala et al. , 2011 PMID : 21799843 Das et al. , 2010 doi : 10.1371 / journal .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25688337" target="_blank">25688337</a>]<br></li>
<li>Fur also plays a role in virulence of S. Typhimurium , as it is important for the establishment of a systemic infection in mice and some studies have shown that a fur mutant had a lower level of invasion in epithelial cells ( Curtiss et al. , 2009 ; Garcia-del Portillo et al. , 1993 ; Riesenberg-Wilmes et al. , 1996 ; Troxell et al. , 2011b ; Velayudhan et al. , 2007 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23306672" target="_blank">23306672</a>]<br></li>
<li>Regarding the other 10 `` bioﬁlm proteins '' found here ( BtuE , OsmY , SspA , YbbN , YhbO , Fur , SufC , GcvT , YibF and CsgF ; Table 2 ) , the ﬁrst 5 ( BtuE , OsmY , SspA , YbbN , YhbO ) are known to play an essential role in cellular stress response mechanisms .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23376784" target="_blank">23376784</a>]<br></li>
<li>The Fur protein , also required for log-phase ATR , is not involved in the stationary-phase ATR , indicating that the two acid-inducible acid tolerance systems are functionally distinct .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10735868" target="_blank">10735868</a>]<br></li>
<li>In particular , the Ferric Uptake Regulator ( Fur ) protein was shown to play a role in low fluid shear induced acid stress resistance [ 11 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19079590" target="_blank">19079590</a>]<br></li>
<li>Shorter moderate acid adaptation also increases total acid resistance of Salmonella but this process is rpoS-independent and requires functional Fur protein ( see below ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16023758" target="_blank">16023758</a>]<br></li>
<li>In addition , a Fur superrepressor that constitutively repressed iron-regulated genes mediated normal Fur-dependent acid tolerance and pH-controlled gene-expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8824613" target="_blank">8824613</a>]<br></li>
<li>Fur is a well-established negative transcriptional regulator that responds to iron availability and class A expression pro-ﬁles are consistent with traditional Fur regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12897017" target="_blank">12897017</a>]<br></li>
<li>The Fur protein is a primary transcription factor that controls expression of genes for iron acquisition ( 11 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24643535" target="_blank">24643535</a>]<br></li>
<li>Normally , Fur acts as a repressor of genes required for iron-homeostasis ( 30 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21075923" target="_blank">21075923</a>]<br></li>
<li>Lastly , our investigations indicate that Fur indirectly regulates genes that are under control of Fnr or additional regulators with an iron sulfur cluster ( i.e. , ftnB and hmpA ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22017966" target="_blank">22017966</a>]<br></li>
<li>Thus , Fur is an activator of genes that are typically induced under anaerobic conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22017966" target="_blank">22017966</a>]<br></li>
<li>This RpoS-independent transient ATR was shown to be dependent on the Fur regulator , which acts , either directly or indirectly , in the positive control of several RpoS-independent ASPs ( 17 , 18 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8824613" target="_blank">8824613</a>]<br></li>
<li>In addition , the screen identiﬁed several genes with expression known to be controlled by Fur , the ferric uptake regulator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25795672" target="_blank">25795672</a>]<br></li>
<li>Masse and Gottesman ( 2002 ) also reported that expression of certain mRNAs ( acnA , fumA , ftnA , bfr , sodB , and sdhD ) in Escherichia coli that were shown to be positively regulated by the Fur regulator is repressed by RyhB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23831078" target="_blank">23831078</a>]<br></li>
<li>In E. coli , class Ib expression has been suggested to be regulated by the transcriptional factor Fur [ 8,9 ] , and nrdDG expression is anaerobically activated by the global regulator FNR [ 10 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20593029" target="_blank">20593029</a>]<br></li>
<li>Several bacteria , including pathogens , express Mn-depen-dent transcriptional repressors that function like the regulators Fur and DtxR , which control the bacterial response to iron limitation ( 29 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12379679" target="_blank">12379679</a>]<br></li>
<li>and the catabolite repressor -RCB- activator ( Cra ) protein ( Saier & Ramseier , 1996 ; Saier et al. , 1996 ) , nitrogen-utilization is controlled by the NtrBC sensor kinase -RCB- response regulator pair in conjunction with the GlnB and GlnD proteins ( Magasanik , 1996 ; Reitzer , 1996a , b ) , phosphorus-utilization is controlled by the PhoR -RCB- PhoB sensor kinase -RCB- response regulator pair ( Wanner , 1996 ) , iron utilization is regulated by the Fur transcription factor ( Earhart , 1996 ; Gralla & Collado-Vides , 1996 ) and sulfur-metabolism is controlled by the CysB transcriptional activator ( Kredich , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11782505" target="_blank">11782505</a>]<br></li>
<li>In the Fe2 + - bound form , E. coli Fur represses genes involved in respiration , ﬂagellar chemotaxis , the TCA-cycle , glycolysis , methionine-biosynthesis , phage DNA packaging , DNA-synthesis , purine-metabolism , and redox stress resistance .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23376784" target="_blank">23376784</a>]<br></li>
<li>In E. coli , rcnA , which encodes a transporter involved in cobalt and nickel efﬂux ( 37 ) , was found to be regulated by the ferric uptake regulator ( Fur ) ( 23 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17720790" target="_blank">17720790</a>]<br></li>
<li>Several cellular targets known to be positively regulated by Fur displayed an increased expression in ST24CHX , including those involved in anaerobic metabolism , such as dimethyl sulfoxide reductase ( dmsA ) , which was up-regulated at both the RNA and protein levels , and those involved in glycerol/glycerolipid-metabolism ( glpAB ) , which were up-regulated at the RNA level .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25136333" target="_blank">25136333</a>]<br></li>
<li>Decreased expression of cellular targets known to the negatively regulated by Fur , including those involved in glycolysis was also noted ; for example , phosphofructokinase ( pfkA ) , which was detected as down-regulated at the RNA and protein levels .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25136333" target="_blank">25136333</a>]<br></li>
<li>Like RpoS , Fur is also involved in the acid tolerance response of log phase cells [ 14,43 ] , and predominantly responds to organic acid stress [ 6 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16023758" target="_blank">16023758</a>]<br></li>
<li>The Fur protein is also involved in the acid tolerance response of S. typhi-murium ( Wilmes-Riesenberg et al. , 1996 ) , although its role in iron uptake and acid resistance is physiologically and genetically separable ( Hall & Foster , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11932449" target="_blank">11932449</a>]<br></li>
<li>Fur is also involved in acid tolerance , which is important for Salmo-nella to survive the conditions encountered in the gastro-intes-tinal tract and in host cells [ 25 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18656407" target="_blank">18656407</a>]<br></li>
<li>This finding also matched previous results in which iron pathway genes in S. typhimurium changed expression in a ground-based model of microgravity , and the Fur protein was shown to play a role in stress resistance alterations induced in the same model ( 7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17901201" target="_blank">17901201</a>]<br></li>
<li>This acid control also appears to be iron independent in that a Fur superrepressor and the iron-blind FurH90R still regulated hyd in response-to-pH .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8824613" target="_blank">8824613</a>]<br></li>
<li>RyhB-1 and RyhB-2 The small RNA RyhB , ﬁrst discovered in Escherichia coli , was found to be part of the Ferric Uptake Regulator ( Fur ) regulon that participates in iron-homeostasis [ 66e68 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28302471" target="_blank">28302471</a>]<br></li>
<li>Iron responsive regulator Fur has been found to be directly or indirectly involved in regulation of oxidative stress defense [ 12 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16362827" target="_blank">16362827</a>]<br></li>
<li>S. typhimurium RB338 , which carries the fur mutation , involved in iron expresses gene products transport in a manner analogous to a Fur ' strain which is under iron limitation ( 3 , 9 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6232258" target="_blank">6232258</a>]<br></li>
<li>Fur therefore exists as a global regulator within S. enterica that allows a coordinated shift in gene-expression to allow appropriate adaptive responses to changes in iron availability and oxidative stress .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21722794" target="_blank">21722794</a>]<br></li>
<li>Lastly , Fur may act as a positive regulator of some genes in response to signals other than or in addition to iron ( type D ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1624426" target="_blank">1624426</a>]<br></li>
<li>Our investigations indicate that , in addition to playing a critical role in iron-homeostasis , Fur is an important antinitrosative determinant of Salmonella pathogenesis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24166960" target="_blank">24166960</a>]<br></li>
<li>In many bacteria , Fur is generally involved in regulation of iron-homeostasis genes , but it is also known to participate in bacterial colonization , oxidative stress response , toxin secretion and pathogenicity [ 55 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28817673" target="_blank">28817673</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_118">
Patzer , S. I. , and Hantke , K. ( 2001 ) Dual repression by Fe2 - Fur and Mn2 - MntR of the mntH gene , encoding an NRAMP-like Mn2 transporter in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24596096" target="_blank">24596096</a>]<br></label>
<input id="rp_similarity_118", type="checkbox">
<div class="RP_similar">
<li>Dual repression by Fe - Fur and Mn2-MntR of the mntH gene , encoding an NRAMP-like Mn2 transporter in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12379679" target="_blank">12379679</a>]<br></li>
<li>Dual repression by Fe2 - Fur and Mn2-MntR of the mntH gene , encoding an NRAMP-like Mn2 transporter in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12029030" target="_blank">12029030</a>]<br></li>
<li>Dual repression by Fe2 - Fur and Mn2-2 MntR of the mntH gene , encoding an NRAMP-like Mn transporter in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659669" target="_blank">15659669</a>]<br></li></div>
<li>Tardat B , Touati D : Iron and oxygen regulation of Escherichia coli MnSOD expression : competition between the global regulators Fur and ArcA for binding to DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22017966" target="_blank">22017966</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_123">
Two global regulators repress the anaerobic expression of Mn SOD in Escherichia coli : Fur ( ferric uptake regulation ) and Arc ( aerobic reparation control ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8824613" target="_blank">8824613</a>]<br></label>
<input id="rp_similarity_123", type="checkbox">
<div class="RP_similar">
<li>Two global regulators repress the anaerobic expression of MnSOD in Escherichia coli : : Fur ( ferric uptake regulation ) and Arc ( aerobic-respiration control ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18790861" target="_blank">18790861</a>]<br></li>
<li>Tardat B , Touati D : Two global regulators repress the anaerobic expression of MnSOD in Escherichia coli : : Fur ( ferric uptake regulation ) and Arc ( aerobic-respiration control ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24044554" target="_blank">24044554</a>]<br></li>
<li>Tardat B , Touati D : Two global regulators repress the anaerobic expression of MnSOD in Escherichia coli : Fur ( ferric uptake regulation ) and Arc ( aerobic-respiration control ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22017966" target="_blank">22017966</a>]<br></li></div>
<li>Expression , isolation and properties of Fur ( ferric uptake regulation ) protein of Escherichia coli K-12 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8824613" target="_blank">8824613</a>]<br></li>
<li>Fur ( ferric uptake regulation ) and CAP ( catabolite-activator protein ) modulate transcription of fur gene in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7642488" target="_blank">7642488</a>]<br></li>
<li>Contents lists available at SciVerse ScienceDirect Computers in Biology and Medicine journal homepage : www.elsevier.com/locate/cbm Interplay between Fur and HNS in controlling virulence gene expression in Salmonella typhimurium Mahendra Kumar Prajapat a , Supreet Saini b , n a Chemical Engineering , Indian Institute of Technology Gandhinagar , VGEC Campus , Chandkheda , Ahmedabad , Gujarat 382424 , India b Chemical Engineering , Indian Institute of Technology Bombay , Powai , Mumbai , Maharashtra 400067 , India abstract article info Salmonella enterica is responsible for a large number of diseases in a wide-range of hosts .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23040276" target="_blank">23040276</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_130">
Regulatory roles of Fnr , Fur , and Arc in expression of manganese-containing superoxide dismutase in Esche-richia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659669" target="_blank">15659669</a>]<br></label>
<input id="rp_similarity_130", type="checkbox">
<div class="RP_similar">
<li>Regulatory roles of FNR , Fur , and Arc in expression of manganese-containing superoxide dismutase in Esche-richia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17220229" target="_blank">17220229</a>]<br></li>
<li>Hassan HM , Sun HC : Regulatory roles of Fnr , Fur , and Arc in expression of manganese-containing superoxide dismutase in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22017966" target="_blank">22017966</a>]<br></li>
<li>Regulatory roles of Fnr , Fur , and Arc in expression of manganese-containing superoxide dismutase in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21075923" target="_blank">21075923</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9393844" target="_blank">9393844</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19389776" target="_blank">19389776</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12029030" target="_blank">12029030</a>]<br></li></div>
<li>c Expression of Fur proteins in a fr mutant ( JF2023 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1624426" target="_blank">1624426</a>]<br></li>
<li>This gene , secY , is a direct target of Fur regulation in Neisseria meningitides [ 100,101 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22017966" target="_blank">22017966</a>]<br></li>
<li>Platero R , Peixoto L , O'Brian MR , Fabiano E : Fur is involved in manganese-dependent regulation of mntA ( sitA ) expression in Sinorhizobium meliloti .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22017966" target="_blank">22017966</a>]<br></li>
<li>To investigate this issue , we determined the mRNA levels of the entC gene , a representative iron-responsive Fur target gene encoding a siderophore-synthesis protein ( 27 ) after Salmonella entry into macrophages .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24643535" target="_blank">24643535</a>]<br></li>
<li>The mntH transcription is also partially repressed by Fe2 via a global iron regulator , Fur , which specifically binds Fe2 and targets a Fur-binding site in the mntH promoter ( 11 , 20 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24596096" target="_blank">24596096</a>]<br></li>
<li>Transcriptional Regulation by Fur The ferric-uptake regulator protein Fur is the key transcriptional regulator of iron-homeostasis in S. enterica and required for virulence ( Troxell et al. , 2011 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21722794" target="_blank">21722794</a>]<br></li>
<li>Typhimurium : identification of new Fur targets Bryan Troxell1 ,3 , Ryan C Fink1 ,4 , Steffen Porwollik2 , Michael McClelland2 and Hosni M Hassan1 * Abstract Background : The Ferric uptake regulator ( Fur ) is a transcriptional regulator that controls iron-homeostasis in bacteria .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22017966" target="_blank">22017966</a>]<br></li>
<li>The Fur protein is a primary transcription factor that senses iron in the cytoplasm and controls genes for iron acquisition ( 11 , 12 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24643535" target="_blank">24643535</a>]<br></li>
<li>The ferric uptake regulator ( Fur ) protein acts as a repressor of many genes whose products are involved in iron , zinc , and manganese acquisition and uptake ( 12 , 13 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24990908" target="_blank">24990908</a>]<br></li>
<li>The expression of RyhB-1 and RyhB-2 is induced to downregulate target mRNAs implicated in iron-homeostasis , including its cognate repressor Fur and dispensable iron-con-taining proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24239962" target="_blank">24239962</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_144">
Delany , I. , Spohn , G. , Rappuoli , R. , and Scarlato , V. ( 2001 ) The Fur repressor controls transcription of iron-activated and -- repressed genes in Helicobacter pylori .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17302823" target="_blank">17302823</a>]<br></label>
<input id="rp_similarity_144", type="checkbox">
<div class="RP_similar">
<li>The Fur repressor controls transcription of iron-activated and - repressed genes in Helicobacter pylori .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11932449" target="_blank">11932449</a>]<br></li></div>
<li>Direct NO targets also include iron -- sulfur [ Fe -- S clusters , which are crucial components of many regulators , including the ferric uptake regulator Fur .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23651595" target="_blank">23651595</a>]<br></li>
<li>The iron prosthetic group of the transcriptional factor Fur ( ferric uptake regulation ) , which is involved in maintaining iron-homeostasis , has also been coopted as an NO sensor ( 86 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26443513" target="_blank">26443513</a>]<br></li>
<li>The ferric uptake regulator ( Fur ) acts as the master regulator of iron-homeostasis in the Enterobacteriaceae , controlling the iron-uptake machinery according to iron regime .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28553268" target="_blank">28553268</a>]<br></li>
<li>Traditionally , the ferric uptake regulator ( Fur ) has been described to be a major determinant in the control of iron homeo-stasis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24166960" target="_blank">24166960</a>]<br></li>
<li>In prokaryotes , Fur ( ferric uptake repressor ) is a central regulator for the maintenance of cytoplasmic iron levels .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12081946" target="_blank">12081946</a>]<br></li>
<li>The Fur ( ferric uptake regulator ) protein is the main regulator of iron-homeostasis in many bacteria .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23376784" target="_blank">23376784</a>]<br></li>
<li>The regulatory protein Fur has been known to sense cytoplasmic iron levels and to regulate vast numbers of genes involved in iron metabolism ( 1 , 9 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18790861" target="_blank">18790861</a>]<br></li>
<li>In iron-replete cells , Fur represses a number of genes involved in iron uptake , including the gene entB ( 47 ; reviewed in reference 36 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17720790" target="_blank">17720790</a>]<br></li>
<li>Iron represses feoB transcription at acidic pH. When bacteria are grown in the presence of iron , the Fur protein associates with Fe2 to repress transcription of genes involved in iron acquisition ( 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19801407" target="_blank">19801407</a>]<br></li>
<li>The Fe ( II ) - associated Fur protein resulting from sufﬁcient levels of intracellular iron binds to its speciﬁc DNA sequences on the target promoters to repress transcription of genes encoding proteins that are involved in iron acquisition ( 1 , 9 , 11 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18790861" target="_blank">18790861</a>]<br></li>
<li>Fur fine-tunes the regulation of iron-homeostasis by controlling iron-transport [ 56,58 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18837991" target="_blank">18837991</a>]<br></li>
<li>Consequently , they serve as a link that facilitates the positive regulation of certain genes by Fur , thus playing a major role in iron-homeostasis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28302471" target="_blank">28302471</a>]<br></li>
<li>Bacillus subtilis contains multiple Fur homologues : identiﬁcation of the iron uptake ( Fur ) and peroxide regulon ( PerR ) repressors .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27242152" target="_blank">27242152</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659669" target="_blank">15659669</a>]<br></li>
<li>In E. coli , rcnA expression has also been suggested to be under the control of iron-sensing Fur ( Koch et al. , 2007 ) , implying some interplay of nickel , cobalt , and iron-homeostasis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21722794" target="_blank">21722794</a>]<br></li>
<li>Under ironreplete conditions , dimeric Fe2 - bound Fur binds to consensus DNA sequences and represses the transcription of iron uptake systems ( 53 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25107963" target="_blank">25107963</a>]<br></li>
<li>The Salmonella serovar Typhimurium genome contains a number of iron-responsive genes that allow for the uptake and storage of iron , with regulation mediated primarily through the ferric uptake regulator ( Fur ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12897017" target="_blank">12897017</a>]<br></li>
<li>The regulatory protein that mediates this regulation , designated Fur ( ferric iron uptake regulator ) , binds Fe2 '' and subsequently represses the expression of the many iron acquisition genes ( 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1624426" target="_blank">1624426</a>]<br></li>
<li>Iron-sensing Fur controls the iron-dependent expression of genes involved in iron uptake and storage as well as with primary roles other than iron-homeostasis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21722794" target="_blank">21722794</a>]<br></li>
<li>For example , the iron-responsive transcriptional regulator Fur is required for expression of iron uptake systems under iron-limiting conditions and for acid-induced activation of atr genes ( Slauch et al. , 1997 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10844688" target="_blank">10844688</a>]<br></li>
<li>In high iron conditions , Fur is known to repress the transcription of genes encoding iron uptake systems and siderophores .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23306672" target="_blank">23306672</a>]<br></li>
<li>Because the Fur regulon includes genes for iron uptake and transport , such regulation results in increased expression of genes for the eficient iron uptake only when iron is a growth-limiting factor [ 42 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16023758" target="_blank">16023758</a>]<br></li>
<li>Expression of several genes controlled by Fnr and NsrR appeared to be also dependent on Fur .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22017966" target="_blank">22017966</a>]<br></li>
<li>As shown above , overexpression of Fur caused an eightfold increase in hilA expression , and this effect was dependent on HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
<li>In addition , we determined that under anaerobic conditions the expression of hilA was dependent on Fur , and we demonstrated that PhoP was not involved in this regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21075923" target="_blank">21075923</a>]<br></li>
<li>In this work we have proven that Fur is indeed a transcriptional regulator of nrdHIEF in S. Typhimurium , normally repressing its expression and causing an upregulation under iron depleted growth ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20593029" target="_blank">20593029</a>]<br></li>
<li>Interplay between Fur and HNS for global regulation of Salmonella gene-expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23040276" target="_blank">23040276</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23040276" target="_blank">23040276</a>]<br></li>
<li>Boulette ML , Payne SM : Anaerobic regulation of Shigella flexneri virulence : ArcA regulates Fur and iron acquisition genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22017966" target="_blank">22017966</a>]<br></li>
<li>Clearly , these results suggest Fnr is functional in Δfur and that Fur is regulating genes of anaerobic metabolism ( eut , frd , and dms operons ) through an unknown mechanism .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22017966" target="_blank">22017966</a>]<br></li>
<li>Iron regulation in this bacterium has primarily focused on the identiﬁcation and characterization of Fur-responsive elements , including 14 genes identiﬁed by a Fur titration assay ( 47 ) and a Salmonella-speciﬁc iron-transport system that is required for virulence ( 23 , 54 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12897017" target="_blank">12897017</a>]<br></li>
<li>We report here that Fur is required for virulence in Salmonella enterica serovar Typhimurium and that Fur is required for the activation of hilA , as well as of other HilA-dependent genes , invF and sipC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21075923" target="_blank">21075923</a>]<br></li>
<li>( A ) Fur controls expression of sodB , but not SPI1 , via RfrA and RfrB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
<li>In S. Typhi , Fur is a repressor of both sRNAs , and loss of either RfrA or RfrB resulted in distinct phenotypes , suggesting a non-redundant role for these regulatory RNAs .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23306672" target="_blank">23306672</a>]<br></li>
<li>In summary , Fur represses expression of sRNAs RfrA and RfrB , a repression that is important for optimal growth of S. Typhi and resistance to H2O2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23306672" target="_blank">23306672</a>]<br></li>
<li>Increased expression of H-NS , a known repressor of ftnB , tdc operon , and other genes , in Δfur may account for their activation by Fur .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22017966" target="_blank">22017966</a>]<br></li>
<li>Role of iron and Fur in tcf expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26944792" target="_blank">26944792</a>]<br></li>
<li>Therefore , Fur might indirectly repress tcf expression by repressing these sRNAs , as loss of fur or growth in low iron concomitantly increases expression of both sRNAs ( RyhB1 and RyhB2 ) and tcf .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26944792" target="_blank">26944792</a>]<br></li>
<li>Deletion of both NrdR and Fur , two main transcriptional regulators , indicates that Fur specifically represses the class Ib enzyme and that NrdR acts as a global repressor of all three classes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20593029" target="_blank">20593029</a>]<br></li>
<li>The proteins NsrR and NorR are NO-dedicated sensors that regulate genes encoding antinitrosative activity in aerobic and anaerobic bacteria ( 15 , 16 ) , whereas OxyR , SoxR , FNR , DksA , and Fur , which primarily serve as sensors of reactive-oxygen-species , O2 , nutritional starvation , and iron , contribute important aspects of the antinitrosative arsenal of enteric bacteria ( 17 -- 21 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24166960" target="_blank">24166960</a>]<br></li>
<li>'' If Fur represses rfrA and rfrB and allows translation of sodB , loss of rfrA and perhaps rfrB should increase expression of sodB in the fur background .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
<li>RstA-Promoted Expression of the Ferrous Iron Transporter Feo under Iron-Replete Conditions Enhances Fur Activity in B Salmonella enterica † Jihye Jeon ,2 ‡ Hyunkeun Kim ,1 ‡ Jiae Yun ,2 Sangryeol Ryu ,2 Eduardo A. Groisman ,3 and Dongwoo Shin1 * Department of Molecular Cell Biology , Samsung Biomedical Research Institute , Sungkyunkwan University School of Medicine , Suwon 440-746 , South Korea1 ; Department of Food and Animal Biotechnology , Department of Agricultural Biotechnology , and Center for Agricultural Biomaterials , Seoul National University , Seoul 151-921 , South Korea2 ; and Department of Molecular Microbiology , Howard Hughes Medical Institute , Washington University School of Medicine , St. Louis , Missouri 631103 Received 30 June 2008/Accepted 31 August 2008 The Fur protein is a primary regulator that monitors and controls cytoplasmic iron levels .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18790861" target="_blank">18790861</a>]<br></li>
<li>Accession Log2 geometric p-value mean fold change Gene Deﬁnition / function / gene product Pathogenicity island hilD SPI-1 regulator a spaP SPI-1 surface presentation of antigens sprB SPI-1 transcriptional regulator ssaL SPI-2 secretion system apparatus protein AE008831 6.95 AE008832 1.90 2.13 E 07 0.0033 AE008831 1.99 0.0033 AE008761 1.63 0.0012 Virulence-associated fur Regulator of the Fur a regulon involved in iron uptake , resistance to acid and possibly oxygen radicals mig5 Macrophage-inducible , a putative carbonic anhydrase , found on virulence plasmid 5.55 E 05 AE008728 2.23 Fig. 1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18656407" target="_blank">18656407</a>]<br></li>
<li>Fur is well characterized in Escherichia coli where it also represses the transcription of a small regulatory non-coding RNA ( sRNA ) , RyhB ( Massé & Gottesman , 2002 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23306672" target="_blank">23306672</a>]<br></li>
<li>However , reports from several laboratories , including our own , suggest that the role of Fur in cellular physiology extends beyond that of regulating iron utilization ( 18 , 19 , 23 , 32 , 37 , 47 , 49 , 51 , 53 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8824613" target="_blank">8824613</a>]<br></li>
<li>In E. coli , Fur has been shown to positively affect the expression of few genes ( e.g. , sodB ) via its negative regulation of a small RNA , RyhB ( 39 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21075923" target="_blank">21075923</a>]<br></li>
<li>Recently , Fur was shown to be involved in regulation of genes encoding regulators of SPI-1 , a group of genes encoding all the machinery of a type three secretion system essential to the invasion of epithelial cells .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23306672" target="_blank">23306672</a>]<br></li>
<li>While Fur is thought to be a simple repressor , the expression of a number of genes is decreased upon deletion of fur .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
<li>In the presence of iron , Fur negatively controls transcription of a small RNA , RyhB , which facilitates degradation of the sdhCDBA mRNA encoding succinate dehydrogenase ( 30 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20008066" target="_blank">20008066</a>]<br></li>
<li>Since enterochelin/salmochelin production is necessary for NE-enhanced growth of S. Typhimurium ( Bearson et al. 2008 ) , the accumulation of these siderophores in the presence of NE increases iron availability which concomitantly reduces the expression of genes encoding iron acquisition proteins via the iron regulator Fur .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26589218" target="_blank">26589218</a>]<br></li>
<li>In the presence of high levels of iron , the Fur protein serves as a negative regulator of genes involved in iron utilization [ 18 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18554972" target="_blank">18554972</a>]<br></li>
<li>Dubrac S , Touati D : Fur positive regulation of iron superoxide dismutase in Escherichia coli : functional analysis of the sodB promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22017966" target="_blank">22017966</a>]<br></li>
<li>Escolar L , Perez-Martin J , de Lorenzo V : Coordinated repression in vitro of the divergent fepA-fes promoters of Escherichia coli by the iron uptake regulation ( Fur ) protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22017966" target="_blank">22017966</a>]<br></li>
<li>These classes are the result of examining global effects of iron changes in the environment that may act directly ( Fur ) or indirectly ( oxidative stress ) on gene-expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12897017" target="_blank">12897017</a>]<br></li>
<li>Furthermore , the gene encoding aconitase of E. coli , an enzyme of the Krebs-cycle , contains a Fur binding site in its promoter region ( 41 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7642488" target="_blank">7642488</a>]<br></li>
<li>If small RNAs are involved in the Fur regulation of hilD , deletion of hfq could increase expression of hilA in the fur background .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
<li>OxyR , a member of the LysR family of bacterial transcriptional factors , regulates expression of many genes required for combating hydrogen peroxide ( H2O2 ) - mediated bacterial killing , such as those encoding catalase , glutathione reductase , glutaredoxin , alkyl hydroperoxide reductase , OxyS and Fur ( Christman et al. , 1985 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18757817" target="_blank">18757817</a>]<br></li>
<li>The Iron-Sensing Fur Regulator Controls Expression Timing and Levels of Salmonella Pathogenicity Island 2 Genes in the Course o Environmental Acidiﬁcation f Eunna Choi , a Hyunkeun Kim , a Hwiseop Lee , a Daesil Nam , a Jeongjoon Choi , b * Dongwoo Shina Division of Microbiology , Department of Molecular Cell Biology , Samsung Biomedical Research Institute , Sungkyunkwan University School of Medicine , Suwon , South Koreaa ; Department of Agricultural Biotechnology , Seoul National University , Seoul , South Koreab In order to survive inside macrophages , Salmonella produces a series of proteins encoded by genes within Salmonella pathoge-nicity island 2 ( SPI-2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24643535" target="_blank">24643535</a>]<br></li>
<li>JOURNAL OF BACTERIOLOGY , Oct. 1996 , p. 5683 -- / 96 / $ 04.0010 Copyright q 1996 , American Society for Microbiology T he Role of Fur in the Acid Tolerance Response of Salmonella typhimurium Is Physiologically and Genetically Separable from Its Role in Iron Acquisition HOLLY K. HALL1 ,2 AND JOHN W. FOSTER1 * Department of Microbiology and Immunology , College of Medicine ,1 and Department of Medical Technology , College of Allied Health ,2 University of South Alabama , Mobile , Alabama 36688 Received 5 April 1996/Accepted 23 July 1996 The response of Salmonella typhimurium to low pH includes a low-pH protection system called the acid tolerance response ( ATR ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8824613" target="_blank">8824613</a>]<br></li>
<li>Ferric Uptake Regulator-Dependent Antinitrosative Defenses in Salmonella enterica Serovar Typhimurium Pathogenesis Maroof Husain , a Jessica Jones-Carson , a , b Lin Liu , a Miryoung Song , a J. Royden Saah , c Bryan Troxell , d Mary Mendoza , d Hosni Hassan , d Andrés Vázquez-Torresa , e ` Department of Microbiologya and Division of Infectious Diseases , b University of Colorado School of Medicine , Aurora , Colorado , USA ; North Carolina State Laboratory of Public Health , Raleigh , North Carolina , USAc ; Department of Microbiology , North Carolina State University , Raleigh , North Carolina , USAd ; VA Eastern Colorado Health Care System , Denver , Colorado , USAe Herein we report an important role for the ferric uptake regulator ( Fur ) in the resistance of Salmonella enterica serovar Typhi-murium to the reactive nitrogen species produced by inducible nitric oxide ( NO ) synthase in an NRAMP1r murine model of acute systemic infection .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24166960" target="_blank">24166960</a>]<br></li>
<li>The absence of fur resulted in increased expression ( i.e. , Fur acted as a repressor ) of 226 genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22017966" target="_blank">22017966</a>]<br></li>
<li>The iron-sensing Fur regulator controls expression timing and levels of Salmonella pathogenicity island 2 genes in the course of environmental acidiﬁcation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26441883" target="_blank">26441883</a>]<br></li>
<li>Formation of a nitrosyl-iron-Fur complex derepresses transcription of Fur-regulated genes in Salmonella and E. coli ( 87 , 86 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26443513" target="_blank">26443513</a>]<br></li>
<li>Effect of Salmonella typhimurium Ferric Uptake Regulator ( fur ) Mutations on Iron - and pH-Regulated Protein Synthesis Fur is an important regulatory protein known to function in the presence of iron as a repressor of iron-controlled genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1624426" target="_blank">1624426</a>]<br></li>
<li>It is also not well understood whether Fur controls the expression of pathogenicity-related genes in Salmonella directly , or by its action on another global / local regulator [ 17,19,22,26 -- 28 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23040276" target="_blank">23040276</a>]<br></li>
<li>Fur Fur ( ferric uptake regulator ) , which is the major iron regulator in Salmonella , has been recently shown to activate expression of SPI1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17208038" target="_blank">17208038</a>]<br></li>
<li>In addition to affecting iron metabolism , the nitrosylation of Fur could have broad implications in Salmonella gene-expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21833325" target="_blank">21833325</a>]<br></li>
<li>The microarray data ( 64 ) identiﬁed hns , a gene encoding the histone-like protein H-NS , as part of the Fur regulon in anaerobically grown S. Typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21075923" target="_blank">21075923</a>]<br></li>
<li>The fur gene , an important regulator of the Fur operon revealed a signiﬁcant increase in transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18656407" target="_blank">18656407</a>]<br></li>
<li>Our description of gene regulator proteins also identiﬁes several sensory gene regulato systems -- Fur , RpoS , EnvZ/OmpR , PhoP/PhoQ -- as evolutionary conserved key elements in deﬁning a deeper level in the activation of virulence gene-expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15790293" target="_blank">15790293</a>]<br></li>
<li>To do this we veriﬁed the selected phenotypic characteristics of S. Typhimurium strain v9718 ( pYA4346 , pUWM1161 ) : i.e. LPS proﬁle , sugar fermentation , Fur synthesis and motility .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23913025" target="_blank">23913025</a>]<br></li>
<li>Alteration in SPI-1 , flagellin and Vi-antigen expression Multiple , ( egpag , pig , SPV , SP12 genes ) Multiple , ( eg invF ; sipB ) Osmolarity RcsB/RcsC Alternative ( T factors RpoS Starvation Multiple , ( eg spv ) Many , induction of stress tolerance , inreased intracellular survival and growth Others H-NS Temperature , ( DNAtopology ) Temperature Iron , pH Many Global gene ( co ) regulator TlpA Fur Unknown flpA Multiple , ( eg sitABCD ) Many , involved in the induction of stress tolerance LysWMetR gene regulators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11370771" target="_blank">11370771</a>]<br></li>
<li>Salmonella encodes the global regulator Fur ( ferric uptake regulator ) and the small regulatory non-coding RNAs ( sRNAs ) RfrA and RfrB , homologues of RyhB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23306672" target="_blank">23306672</a>]<br></li>
<li>Interestingly , many crucial components of the starvation response and the ATR are also well-defined virulence regulators ( 64 , 65 ) and involve , as regulators , components such as OmpR , RpoS , PhoPPhoQ and Fur , which are also found in E. colz .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11370771" target="_blank">11370771</a>]<br></li>
<li>The main regulatory proteins involved in the expression of Spi-2 include the two-component systems SpiR ( SsrA ) / SsrB , PhoP/PhoQ , OmpR/EnvZ , as well as the DNA-binding proteins HilD , SlyA ( RovA in Yersinia ) , RcsB , the iron regulator Fur , and the nucleoid-associated proteins H-NS , Hha ( YmoA in Yersinia ) , YdgT , IHF , and Fis ( Fass and Groisman , 2009 ; Choi et al. , 2014 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26441883" target="_blank">26441883</a>]<br></li>
<li>The defense network involved multiple cell targets including those associated with the synthesis and modiﬁcation of the cell wall , the SOS-response , virulence , and a shift in cellular-metabolism toward anoxic pathways , some of which were regulated by CreB and Fur .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25136333" target="_blank">25136333</a>]<br></li>
<li>Proteins involved in nutrient transport ( Crr , DppA , Fur and SufC ) , together with ones involved in the complementa-ry processes of degradation and energy metabolism ( GcvT , GpmA and RibB ) form the other important group of proteins found here to be induced under bioﬁlm conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23376784" target="_blank">23376784</a>]<br></li>
<li>This group included most of the global regulators and many were differentially affected under the different growth conditions including Crp , Fis , ArcA , SlyA , Fur , and H-NS ( Figures 1D -- F ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25713562" target="_blank">25713562</a>]<br></li>
<li>Fur represses the synthesis of the small RNA RyhB , which under Fe-restricted conditions negatively regulates the synthesis of several Fe-containing proteins , including FtnA and Bfr ( Masse and Gottesman , 2002 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17302823" target="_blank">17302823</a>]<br></li>
<li>Yet , ﬁnally , mutation of both the MntR and Fur proteins also abolishes Mn2 repression ( Fig. 4F ) , which implies that Fur is the only other trans-acting factor involved .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12029030" target="_blank">12029030</a>]<br></li>
<li>We also found that the anaerobic expression of ftnB ( encoding a ferri-tin-like protein ) and hmpA ( encoding the NO · detoxifying flavohemoglobin ) was dependent on both Fur and Fnr .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22017966" target="_blank">22017966</a>]<br></li>
<li>Two of the global regulators involved in controlling gene-expression during the infection cycle of the bacterium are Fur and HNS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23040276" target="_blank">23040276</a>]<br></li>
<li>Two of the global regulators involved in control of gene-expression are Fur and HNS [ 17,18 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23040276" target="_blank">23040276</a>]<br></li>
<li>They act by increasing the level of HilD protein to overcome HilE , and controlling HilD activity , or in the case of Fur , lowering the threshold required at the promoter , such that HilD activates its own promoter as well as induces expression of HilC and RtsA , which then act to amplify and accelerate SPI1 expression ( Saini et al. 2010a ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22021388" target="_blank">22021388</a>]<br></li>
<li>( A ) Conditions that activate Fur ( i.e. , the presence of iron ) repress the expression of hns ( solid arrow ) , and allow the expression of hilA ( blocked dotted arrows ) and increased HilA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21075923" target="_blank">21075923</a>]<br></li>
<li>In the present study , we report that Fur regulates the expression of H-NS and this in-turn regulates the expression of hilA and virulence in mice .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21075923" target="_blank">21075923</a>]<br></li>
<li>( B ) Mutation in Fur or lack of the cofactor ( i.e. , Fe2 ) results in the expression of hns and the H-NS protein ( blocked dotted arrows ) and the reduced expression of hilA ( solid arrows ) and HilA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21075923" target="_blank">21075923</a>]<br></li>
<li>In Salmonella , Fur also modulates the expression of genes involved in acid shock , adaptation ( 15 , 16 ) , and oxidative stress resistance ( 17 , 18 ) , and it plays a role in the regulation of the Salmonella pathogenicity island 1 ( SPI-1 ) genes ( e.g. , hilA and hilD ) necessary for invasion ( 19 -- 21 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24990908" target="_blank">24990908</a>]<br></li>
<li>The differential expression of stn and invA was observed among TSB-5 , TSB-6 , and TSB-7 , suggesting that the alternative sigma factor ( s ) , two-component regulatory system ( PhoPQ ) , and iron regulatory protein ( Fur ) can regulate not only the ATR but also the expression of virulence genes in S. Typhimurium cells ( 20 , 24 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20971873" target="_blank">20971873</a>]<br></li>
<li>Global gene-expression proﬁling of intracellular Salmonella 111 centrations , the Fur protein binds to Fe2 + and represses during the intestinal phase of the infection when free iron the expression of genes involved in iron uptake in Salmo - concentrations are thought to be higher ( Gunn et al. , nella ( Tsolis et al. , 1995b ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12492857" target="_blank">12492857</a>]<br></li>
<li>Additionally , OxyR and SoxRS both activate the expression of the iron regulator , Fur ( Zheng et al. , 1999 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23651595" target="_blank">23651595</a>]<br></li>
<li>The lumen of the stomach is acidic , and low pH induces the expression of RpoS and Fur , transcription factors important for acid resistance ( Foster , 1991 ; Fang et al. , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12675799" target="_blank">12675799</a>]<br></li>
<li>However , E. coli Fur is also known to activate the expression of many genes by either direct or indirect mechanisms , the latter involving repression of a small regulatory RNA , ryhB , which binds to and decreases the stability of its target mRNAs ( Masse et al. , 2003 ; Lee and Helmann , 2007 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21722794" target="_blank">21722794</a>]<br></li>
<li>Since many of the genes are regulated by the global transcriptional iron regulator Fur , real-time RT-PCR was also performed for the fur gene ; a signiﬁcant , 2.5-fold reduction in gene-expression was observed in the wild-type strain grown with 2 mM norepinephrine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18554972" target="_blank">18554972</a>]<br></li>
<li>Expression levels of the feoB gene change in response to iron and oxygen , which results from the action of the iron-sensing transcriptional regulator Fur ( 9 ) and the oxygen-sensing transcriptional regulator Fnr ( 10 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23708131" target="_blank">23708131</a>]<br></li>
<li>Two global regulators , the alternative sigma factor ss , encoded by rpoS ( 33 ) , and the iron-regulatory protein Fur ( ferric uptake regulation ) ( 21 , 22 ) , control the expression of nonoverlapping ASP subsets and signiﬁcantly contrib-ute to the development of acid tolerance ( 13 , 28 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8824613" target="_blank">8824613</a>]<br></li>
<li>Because the ferric uptake repressor protein , Fur , controls many of the processes in the cell that are responsive to iron ( 9 ) , null alleles of fur were introduced into strains DM5419 , DM5420 , and LT2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10869064" target="_blank">10869064</a>]<br></li>
<li>The regulatory effect of the RstA protein occurred through the Fur protein because the RstA protein failed to repress transcription of the iron-regu-lated genes ( i.e. , fhuA and fhuF ) in a fur deletion strain ( Fig. 1B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18790861" target="_blank">18790861</a>]<br></li>
<li>Due to the acid induction of AvrA in class 2 strains , it might be speculated that the expression of avrA is dependent on a regulatory system which resembles the earlier identiﬁed ATR regulon , a part of a global stress response network including the functions of RpoS , Fur , and PhoP/Q ( Foster et al. , 1994 ; Foster , 1995 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16377240" target="_blank">16377240</a>]<br></li>
<li>The decrease in receptor level in the Fur - strain RB338 indicates that the effect of growth temperature on the number of colicin I receptors , like that on enterochelin synthesis and transport , is independent of the fur gene product .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=6232258" target="_blank">6232258</a>]<br></li>
<li>This fact indicates that , regardless of cAMP , the Fur protein must be involved in the control of the ﬂagellar regulon and that its presence is absolutely required to achieve an optimal expression of the ﬂhD promoter , despite there being enough intracellular cAMP concentration .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11932449" target="_blank">11932449</a>]<br></li>
<li>One member of the Fur regulon is entB , encoding a protein involved in the synthesis of the siderophore enterobac-tin .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18835989" target="_blank">18835989</a>]<br></li>
<li>We also determined that the Fur protein was acting as a transcriptional repressor of the feoAB operon because deletion of the fur gene resulted in a twofold increase of the wild-type feoAB transcripts ( Fig. 2A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18790861" target="_blank">18790861</a>]<br></li>
<li>To determine the relative responsiveness of the NsrR , NorR , Fur and SoxR transcriptional regulators , the expression of individual genes that are specifically dependent on each regulator was measured following exposure to NO · .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22831173" target="_blank">22831173</a>]<br></li>
<li>A fur mutant is unable to grow on succinate as a sole carbon source due to a lack of positive regulation of the genes encoding the Fe-S cluster containing the enzymes SDH , fuma-rase A , and aconitase B ( AcnB ) of the tricarboxylic acid ( TCA ) cycle by Fur , which is mediated by the small RNA RyhB ( 26 , 27 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18835989" target="_blank">18835989</a>]<br></li>
<li>When associated with Fe2 , the Fur protein represses the expression of genes for iron acquisition , which minimizes the accumulation of free Fe2 in the cytoplasm ( 1 , 12 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19801407" target="_blank">19801407</a>]<br></li>
<li>Over time , the PmrA-regulated modifications that lower the negative charge of the bacterial surface are incorporated into the lipid A moiety while the expression of iron-transport genes is repressed due to rising cytoplasmic Fe2 + , which combines with the aporepressor Fur to dampen expression of iron acquisition determinants ( FaraldO-Gomez and Sansom , 2003 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22921935" target="_blank">22921935</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_251">
Prominent roles of NorR and Fur regulators in the Escherichia coli transcriptional response to reactive nitrogen species .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15790293" target="_blank">15790293</a>]<br></label>
<input id="rp_similarity_251", type="checkbox">
<div class="RP_similar">
<li>Prominent roles of the NorR and Fur regulators in the Escherichia coli transcriptional response to reactive nitrogen species .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25107963" target="_blank">25107963</a>]<br></li></div>
<li>For example , Fur has previously been shown to repress sitABCD , encoding a manganese transport system , in Salmonella when the bacterium is in the lumen of the small intestine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
<li>Role of the small RNA RyhB in the Fur regulon in mediating the capsular-polysaccharide-biosynthesis and iron acquisition systems in Klebsiella pneumoniae .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27242152" target="_blank">27242152</a>]<br></li>
<li>Fur is involved in metal-dependent transcriptional regulation in response to different metal concentrations and the redox state of the bacterial the metabolism of ST24CHX .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25136333" target="_blank">25136333</a>]<br></li>
<li>The iron-regulatory protein Fur has been implicated in the ATR since fur mutants are acid sensitive and cause altered expression of several acid shock proteins ( J. W. Foster , J. Bacteriol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8824613" target="_blank">8824613</a>]<br></li>
<li>The H90R iron-blind Fur protein also mediated acid shock induction of several Fur-dependent acid shock proteins and acid control of the hyd locus .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8824613" target="_blank">8824613</a>]<br></li>
<li>https://doi.org/10.1371/journal.ppat.1006559.g006 Lrp controls klf expression , while Fur and OmpR regulate ipf Since the expression of both fimbriae was highly affected by microaerobic conditions , we sought to test a possible role of regulators known to be involved in oxygen homeostasis including OxyR , SoxR , ArcA , ArcB and FNR , [ 29 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28817673" target="_blank">28817673</a>]<br></li>
<li>In addition , we screened for potential regulatory roles of global regulators , previously reported to control Salmonella pathogenicity including RpoS , PhoP , OmpR and Fur [ 30 ] as well as the leucine-responsive regulatory protein ( Lrp ) that was shown to regulate different types of fimbriae in E. coli [ 31,32 ] and Salmonella [ 33 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28817673" target="_blank">28817673</a>]<br></li>
<li>As at least in E. coli cAMP represses RpoS expression ( Ma et al. , 2003 ) , Fur activity could conceptually be linke with RpoS and concomitantly with the expression of additional virulence genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15790293" target="_blank">15790293</a>]<br></li>
<li>Fur also positively regulates other HilA-dependent genes ( invF and sipC ) involved in invasion , by repressing the expression of H-NS ( Troxell et al. , 2011 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21722794" target="_blank">21722794</a>]<br></li>
<li>Recently , we demonstrated the role of Fur in HilA expression and virulence in S. Typhimurium , which is mediated by the negative regulation of H-NS by Fur under anaerobic conditions [ 29 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22017966" target="_blank">22017966</a>]<br></li>
<li>The Fur regulon , those genes whose expression are under the control of Fur , contain genes that are involved in iron acquisition , which is essential for bacterial multiplication ( de Lorenzo et al. , 1987 ; Mekalanos , 1992 ; Wooldridge and Williams , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12652904" target="_blank">12652904</a>]<br></li>
<li>The RstA-promoted FeoB expression imports additional Fe ( II ) , elevating the active Fur [ i.e. , Fur-Fe ( II ) ] levels , which in turn represses further transcription of Furrepressed genes encoding other Fe ( II ) / Fe ( III ) transporters and proteins involved in iron assimilation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18790861" target="_blank">18790861</a>]<br></li>
<li>Alternative mechanisms for regulation in this class may involve other small RNA-dependent mechanisms ( 52 ) , Fur regulation of other regulators , and indirect induction of oxidative stress in the cell ( 46 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12897017" target="_blank">12897017</a>]<br></li>
<li>Except for Fur , inactivation of genes encoding the other stress regulators results in attenuated virulence and such mutants can therefore be considered as vaccine candidates .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16023758" target="_blank">16023758</a>]<br></li>
<li>In addition , Fur is also linked to other stresses where iron plays an important role such as the acid tolerance response ( ATR ) by assisting in SodB expression ( Dubrac & Touati , 2000 ) , repression of the iron-transport system SitABCD encoded on SPI-1 ( Janakiraman & Slauch , 2000 ) and in nitrosative stress by repressing hmpA ( Mukhopadhyay , Zheng , Bedzyk , LaRossa , & Storz , 2004 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23651595" target="_blank">23651595</a>]<br></li>
<li>Furthermore , isc mutants accumulated increased levels of pyruvate in the growth medium in response to exogenously added iron ( FeCl3 ) , and this response required a functional ferric uptake regulator , Fur .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10869064" target="_blank">10869064</a>]<br></li>
<li>Fur protein is the major regulator of iron metabolism in both E. coli and S. typhimurium , but it clearly has a very broad impact on cellular physiology ( 10 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8169207" target="_blank">8169207</a>]<br></li>
<li>In Salmonella enterica and other microorganisms , the ferric uptake regulator ( Fur ) is a transcriptional repressor of genes involved in iron uptake and metab-olism .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18835989" target="_blank">18835989</a>]<br></li>
<li>Received 25 October 2012 Revised 22 December 2012 Accepted 4 January 2013 INTRODUCTION Fur ( ferric uptake regulator ) is a regulatory protein that is active in the presence of iron ( and other metals ) and its principal function is to repress the transcription of genes implicated in iron acquisition and transport ( Ernst et al. , 1978 ; Hantke , 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23306672" target="_blank">23306672</a>]<br></li>
<li>It is thought that in the small intestine , conditions are anaerobic and therefore replete with Fe2 , leading to an active Fur protein and the repression of genes involved in uptake of iron and other metals .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
<li>The absence of IRO-28 in the fur-1 mutant ( Fig. 7C ) and its presence in the furH90R mutant ( Fig. 7E ) conﬁrm that Fur acts as a positive regulator of this gene and that FurH90R is iron blind yet active .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8824613" target="_blank">8824613</a>]<br></li>
<li>Lelong et al. ( 28 ) reported that in E. coli W3110 , Crl can interact with the ferric uptake regulator Fur , a key protein for the control of intracellular iron concentration ( see reference 8 and references therein ) and that Fur represses crl transcription ( 28 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20008066" target="_blank">20008066</a>]<br></li>
<li>The 17-kDa Fur protein exerts control over a series of genes in S. typhimurium involved with the synthesis , excretion , and recovery of the iron-chelating siderophore enterochelin ( 54 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8824613" target="_blank">8824613</a>]<br></li>
<li>The proximity of STnc4000 and STnc3250 to genes involved in iron-homeostasis raises the possibility that Fur regulation of STnc4000 and STnc3250 may be involved in the control of iron metabolism .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27564394" target="_blank">27564394</a>]<br></li>
<li>Because Fur controls iron uptake in many bacteria and there is ﬁerce competition between the pathogen and the host for free iron , an understanding of how iron-homeostasis regulates hilA is crucial to our knowledge of the role of Fur in the activation of SPI-1 and virulence in Salmonella .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21075923" target="_blank">21075923</a>]<br></li>
<li>Thus , Fur appears to sense acid stress and affect the expression of some genes in an apparent iron-independent fashion .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8824613" target="_blank">8824613</a>]<br></li>
<li>Several observations led to the Hfq regulon members being highlighted in our M9 proteomic analysis : 1 ) Hfq promotes the expression of a large class of ribosomal structural proteins , and we found differential expression of several of these genes in spaceflight ( L7/L12 , L32 , S20 , S13 , S11 , S19 , SA , L14 , L33 , S4 , L4 ) ; 2 ) Hfq regulates the expression of the Fur protein and other genes involved in iron metabolism , and we found that Fur and other iron-related genes are differentially regulated by spaceflight in M9 medium ( Fur , Dps , NifU , FepA ) ; 3 ) Several other proteins encoded by genes belonging to the Hfq regulon were also found in this analysis : NmpC , Tpx , PtsI , PtsH , SucC , LeuB , CysP , DppA , OppA , RpoZ , CsrA , RpoB , NlpB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19079590" target="_blank">19079590</a>]<br></li>
<li>( viii ) Hfq regulates the expression of the Fur protein and other genes involved in the iron response pathway , and we observed several iron utilization/storage genes with altered expression in flight ( 9 , 11 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17901201" target="_blank">17901201</a>]<br></li>
<li>Fur associates with Fe2 to repress transcription of genes involved in iron uptake and to activate expression of genes implicated in the defense against oxygen toxicity ( reviewed in references 33 and 35 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12081946" target="_blank">12081946</a>]<br></li>
<li>Fur regulates ﬁmbrial , ﬂagella , and SPI encoded genes in Salmonella Previous reports have demonstrated that Fur , in addition to regulating the cellular gene-expression in response to the iron availability and its uptake , also regulates expression of genes involved in pathogenesis [ 17,22 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23040276" target="_blank">23040276</a>]<br></li>
<li>Holmes K , Mulholland F , Pearson BM , Pin C , McNicholl-Kennedy J , Ketley JM and Wells JM : Campylobacter jejuni gene-expression in response to iron limitation and the role of Fur .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20003355" target="_blank">20003355</a>]<br></li>
<li>Fur contributed to invasion of epithelial cells , and was shown for the first time to play a role in phagocytosis and intracellular survival of S. Typhi in human macrophages .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23306672" target="_blank">23306672</a>]<br></li>
<li>nrdAB expression was also measured to rule out a global effect of the Fur knock-out on overall gene-expression ( Figure S1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20593029" target="_blank">20593029</a>]<br></li>
<li>Fur activates sodB by repressing the expression of ryhB under high-iron conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
<li>In conclusion , Fur activates hilA by repressing the expression of hns .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21075923" target="_blank">21075923</a>]<br></li>
<li>Given that SsrB functions as an activator for expression of SPI-2 genes ( 4 ) and that Fur represses ssrB expression ( Fig. 3 ) , it is possible that the Fur regulator controls SPI-2 genes in a fashion dependent on the SsrB regulator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24643535" target="_blank">24643535</a>]<br></li>
<li>Fur controlled SPI-2 genes via repression of the SsrB response regulator , a primary activator of SPI-2 expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24643535" target="_blank">24643535</a>]<br></li>
<li>Taken together , these results illustrate that the iron-sensing Fur regulator negatively controls expression of SPI-2 genes during acidiﬁcation and a lack of this regulation enables the SPI-2 genes to be induced , even under intermediate acidic conditions that would not normally allow for SPI-2 induction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24643535" target="_blank">24643535</a>]<br></li>
<li>In the present study , we report that Fur , a central regulator of iron utilization , negatively controls the expression of SPI-2 genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24643535" target="_blank">24643535</a>]<br></li>
<li>Here , we report that the iron-sensing Fur regulator also controls expression of the SPI-2 genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24643535" target="_blank">24643535</a>]<br></li>
<li>Because growth at 42 ° C repressed expression of the PrtsA-lacZ fusion , identifying that Fur also regulates SPI-1 in this manner would suggest that temperature control of SPI-1 may act through Fur .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26386070" target="_blank">26386070</a>]<br></li>
<li>This conﬁrmed that HilD controls expression of rtsA and that Fur works exclusively via HilD to control the expression of SPI1 genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
<li>In addition , Fur might contribute to Salmonella virulence by controlling expression timing and the expression level of SPI-2 genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24643535" target="_blank">24643535</a>]<br></li>
<li>Fur indirectly increases the expression of several genes that either use or store iron , including aconitase B ( AcnB ) and succinate dehydrogenase ( SDH ) , via the RyhB small RNA , whose expression is controlled by Fur ( 28 , 29 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17720790" target="_blank">17720790</a>]<br></li>
<li>First , the microarray data showed that Fur negatively regulates the expression of hns and has a predicted Fur binding site ( Table 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22017966" target="_blank">22017966</a>]<br></li>
<li>Fur represses expression of the ssrB gene encoding the SsrB response regulator under acidic conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24643535" target="_blank">24643535</a>]<br></li>
<li>FIG 3 The Fur regulator represses expression of the ssrB gene inside macro-phages and under acidic conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24643535" target="_blank">24643535</a>]<br></li>
<li>Moreover , these results also demonstrate that the S. typhimurium Fur protein positively regulates the expression of the flhD master operon governing the ﬂagellar regulon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11932449" target="_blank">11932449</a>]<br></li>
<li>It has also been reported that the Escherichia coli Fur protein is a positive regulator of sodB gene-expression , although the precise mechanism ... .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11932449" target="_blank">11932449</a>]<br></li>
<li>For example , if Fur negatively regulates the expression of a second negative regulator , the targets of that second regulator will appear to be under positive control by Fur .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1624426" target="_blank">1624426</a>]<br></li>
<li>For example , Fur is a positive regulator of the sufABCDSE operon that encodes an enzymatic pathway for the repair of oxidized [ 4Fe-4S ] clusters ( 42 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24166960" target="_blank">24166960</a>]<br></li>
<li>The nitric oxide ( NO ) donor spermine NONOate derepressed the Fur regulon that controls secondary lines of resistance against organic acids .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18350168" target="_blank">18350168</a>]<br></li>
<li>Fur , a central regulator of iron utilization ( 11 ) , has been reported to enhance SPI-1 expression ( 15 , 16 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24643535" target="_blank">24643535</a>]<br></li>
<li>The regulatory role of ferric uptake regulator ( Fur ) during anaerobic-respiration of Shewanella piezotolerans WP3 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28553268" target="_blank">28553268</a>]<br></li>
<li>Bereswill , S. , Greiner , S. , van Vliet , A.H. , Waidner , B. , Fassbinder , F. , Schiltz , E. , et al. ( 2000 ) Regulation of ferritinmediated cytoplasmic iron storage by the ferric uptake regulator homolog ( Fur ) of Helicobacter pylori .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17302823" target="_blank">17302823</a>]<br></li>
<li>Fur plays an important role in the virulence of Salmonella , regulating several virulence-asso-ciated promoters ( Tsolis et al. , 1995 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18096019" target="_blank">18096019</a>]<br></li>
<li>For example , Fur negatively regulates transcription of HN-S in Salmonella ( Troxell et al. , 2010 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21833325" target="_blank">21833325</a>]<br></li>
<li>[ 24 ] I. Delany , R. Rappuoli , V. Scarlato , Fur functions as an activator and as a repressor of putative virulence genes in Neisseria meningitidis , Mol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23040276" target="_blank">23040276</a>]<br></li>
<li>The Fur regulon includes genes involved in an array of cellular functions ( 27 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
<li>Finally , we have also identified twenty-six genes as new targets of Fur regulation in S. Typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22017966" target="_blank">22017966</a>]<br></li>
<li>In contrast , several transcription factors positively or negatively regulate SPI-1 genes by controlling hilD transcription independently of H-NS ( e.g. , Fur ) ( 37 , 38 ) , or by controlling HilD translation ( e.g. , SirA and CsrA ) ( 39 ) or activity ( e.g. , HilE and FliZ ) ( 40 , 41 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27601571" target="_blank">27601571</a>]<br></li>
<li>The results provide further evidence that Fur has an extensive impact on gene-expression and cellular physiology and suggest an explanation for the acid-sensitive nature offir mutants .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1624426" target="_blank">1624426</a>]<br></li>
<li>The results suggest for a complex interplay between Fur and HNS to globally control the cellular behavior .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23040276" target="_blank">23040276</a>]<br></li>
<li>e-mail : jordi.barbe!uab.es Department of Genetics and Microbiology , Universitat Auto ' noma de Barcelona1 and Centre de Recerca en Sanitat Animal ( CReSA ) , Universitat Auto ' noma de Barcelona Institut de Recerca i Tecnologia Agroalimenta ' ria ( UAB-IRTA ) 2 , Bellaterra , 08193 Barcelona , Spain It is known that the Fur protein negatively regulates iron-uptake systems in different bacterial species , including Salmonella typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11932449" target="_blank">11932449</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_317">
Troxell , B. , M. L. Sikes , R. C. Fink , A. Vazquez-Torres , J. Jones-Carson et al. , 2010 Fur negatively regulates hns and is required for the expression of hilA and virulence in Salmonella enterica serovar Typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22021388" target="_blank">22021388</a>]<br></label>
<input id="rp_similarity_317", type="checkbox">
<div class="RP_similar">
<li>Troxell B , et al. ( 2011 ) Fur negatively regulates hns and is required for the expression of HilA and virulence in Salmonella enterica serovar Typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23690578" target="_blank">23690578</a>]<br></li>
<li>Fur negatively regulates hns and is required for the expression of HilA and virulence in Salmonella enterica serovar Typhimu-rium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24166960" target="_blank">24166960</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24643535" target="_blank">24643535</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26386070" target="_blank">26386070</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25547794" target="_blank">25547794</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24990908" target="_blank">24990908</a>]<br></li>
<li>Fur negatively regulates hns and is required for the expression of HilA and virulence in Salmonella enterica serovar Typhi-murium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23554419" target="_blank">23554419</a>]<br></li>
<li>Fur negatively regulates hns and is required for the expression of HilA and virulence in Salmonella enterica serovar Typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26441883" target="_blank">26441883</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26662614" target="_blank">26662614</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23306672" target="_blank">23306672</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21722794" target="_blank">21722794</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24735176" target="_blank">24735176</a>]<br></li></div>
				<label class="RP_collapse_similarity" for="rp_similarity_322">
Troxell B , Sikes ML , Fink RC , Vazquez-Torres A , Jones-Carson J , Hassan HM : Fur negatively regulates hns and is required for the expression of HilA and virulence in Salmonella enterica serovar typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23295139" target="_blank">23295139</a>]<br></label>
<input id="rp_similarity_322", type="checkbox">
<div class="RP_similar">
<li>doi :10.1007 / Troxell B , Sikes ML , Fink RC , Vazquez-Torres A , Jones-Car-son J , Hassan HM ( 2011 ) Fur negatively regulates hns and is required for the expression of HilA and virulence in Salmonella enterica serovar Typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24970347" target="_blank">24970347</a>]<br></li>
<li>[ 19 ] B. Troxell , M.L. Sikes , R.C. Fink , A. Vazquez-Torres , J. Jones-Carson , H.M. Hassan , Fur negatively regulates hns and is required for the expression of HilA and virulence in Salmonella enterica serovar Typhimurium , J. Bacteriol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23040276" target="_blank">23040276</a>]<br></li>
<li>Troxell B , Sikes ML , Fink RC , Vazquez-Torres A , Jones-Carson J , Hassan HM : Fur negatively regulates hns and is required for the expression of HilA and virulence in Salmonella enterica serovar Typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22017966" target="_blank">22017966</a>]<br></li></div>
<li>Several iron-containing transcriptional regulators , including NsrR , NorR , Fur , and SoxR , have been previously shown to respond to NO · ( Crack et al. , 2012 ; Fleischhacker and Kiley , 2011 ; Spiro , 2006 ; Zheng and Storz , 2000 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22831173" target="_blank">22831173</a>]<br></li>
<li>In S. Tm , cib expression is negatively regulated in a Fur ( ferric uptake regulator ) - dependent and LexA ( locus for x-ray sensitivity A ) - dependent way ( Mankovich et al. , 1986 ; Nedialkova et al. , 2014 ) and maximally induced under iron limitation and upon exposure to DNA damaging agents .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26439675" target="_blank">26439675</a>]<br></li>
<li>Results Rationale and approach We have previously shown that HilE , FliZ , and Fur control hilA expression via HilD ( Ellermeier and Slauch 2008 ; Lin et al. 2008 ; Chubiz et al. 2010 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22021388" target="_blank">22021388</a>]<br></li>
<li>Fur has previously been demonstrated to indirectly activate expression of ferritins including Bfr and FtnA ( Abdul-Tehrani et al. , 1999 ; Masse and Gottesman , 2002 ) and of several other Fe-containing proteins including aconitase A , fumarase A , succinate dehydrogenase and several respiratory complexes ( Masse and Gottesman , 2002 ; McHugh et al. , 2003 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17302823" target="_blank">17302823</a>]<br></li>
<li>It has been reported that expression of sodB is regulated at the transcriptional and post-transcrip-tional level by several factors including Fur , NsrR and the sRNAs ryhB and fnrS ( Massé & Gottesman , 2002 ; Durand & Storz , 2010 ; Niederhoffer et al. , 1990 ; Partridge et al. , 2009 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23918818" target="_blank">23918818</a>]<br></li>
<li>Harvie D.R. , Vilchez S. , Steggles J.R. and Ellar D.J. , Bacillus cereus Fur regulates iron metabolism and is required for full virulence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16362827" target="_blank">16362827</a>]<br></li>
<li>Fur and virulence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21075923" target="_blank">21075923</a>]<br></li>
<li>A third class was Fur independent and included both negatively and positively iron-responsive genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12897017" target="_blank">12897017</a>]<br></li>
<li>Fur is a transcriptional and posttranscriptional regulator that senses iron concentration and redox state .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25795672" target="_blank">25795672</a>]<br></li>
<li>Typhimurium ( St ) Fur and Zur proteins with related Fur proteins from E. coli ( Ec ) , Pseudomonas aeruginosa ( Pa ) and Vibrio cholera ( Vc ) and Zur proteins from E. coli ( Ec ) and M. tuberculosis ( Mt ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21722794" target="_blank">21722794</a>]<br></li>
<li>The group of proteins whose expression was visible only during bioﬁlm growth included proteins involved in global regulation and stress response ( ArcA , BtuE , Dps , OsmY , SspA , TrxA , YbbN and YhbO ) , nutrient transport ( Crr , DppA , Fur and SufC ) , degradation and energy metabolism ( GcvT , GpmA , RibB ) , detoxiﬁcation ( SseA and YibF ) , DNA-metabolism ( SSB ) , curli production ( CsgF ) , and murein synthesis ( MipA ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23376784" target="_blank">23376784</a>]<br></li>
<li>The 20 proteins whose expression was visible only in the bioﬁlm-derived samples could be classiﬁed into the following ﬁve categories , based on their function : ( a ) proteins involved in global regulation and stress response ( ArcA , BtuE , Dps , OsmY , SspA , TrxA , YbbN and YhbO ) ; ( b ) proteins involved in nutrient transport ( Crr , DppA , Fur and SufC ) ; ( c ) proteins involved in degradation and energy metab-olism ( GcvT , GpmA and RibB ) ; ( d ) proteins involved in detoxiﬁcation ( SseA and YibF ) and ( e ) proteins involved in various processes , such as DNA-metabolism ( SSB ) , curli production ( CsgF ) and murein synthesis ( MipA ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23376784" target="_blank">23376784</a>]<br></li>
<li>By controlling expression/activity of HilD , HilC , RtsA , and HilA , many regulators contribute to SPI-1 gene regulation in response to environmental stimuli that include temperature ( H-NS and Hha ) , osmolarity ( H-NS and Hha ) , pH ( OmpR ) , iron availability ( Fur ) , and oxygen levels ( FNR ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27601571" target="_blank">27601571</a>]<br></li>
<li>The panel of regulatory proteins ( Table 2 ) included SPI1-encoded transcription factors ( HilA , HilC and HilD ) and transcription factors previously reported to control SPI1 gene-expression ( HilE , BarA/SirA , Fur and FliZ ) [ 17 -- 24 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27564394" target="_blank">27564394</a>]<br></li>
<li>Genes associated with DNA-repair and purine meta-bolism ( nrdAB , nth , recA , and nei ) were repressed by Fur under anaerobic conditions ( Additional file 2 : Table S2 ) , thus implicating Fur as a regulator of DNA-repair and de novo synthesis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22017966" target="_blank">22017966</a>]<br></li>
<li>Typhi Fur regulator and small RNAs RfrA and RfrB in iron-homeostasis and interaction-with-host cells .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27242152" target="_blank">27242152</a>]<br></li>
<li>Results NrdR and Fur negatively regulate nrd expression In order to assess the transcriptional role of both NrdR and Fur , unmarked nrdR and fur null mutants of Salmonella enterica serovar Typhimurium LT2 were obtained as described in Materials and Methods and then transduced into Salmonella strains containing either nrdAB-lacZ , nrdHIEF-lacZ or nrdDG-lacZ reporter fusions ( for DnrdR mutation ) or just nrdHIEF-lacZ ( for Dfur mutation ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20593029" target="_blank">20593029</a>]<br></li>
<li>In this study , we investigated the import-ance of Fur and the sRNAs RfrA and RfrB of S. Typhi for growth , resistance to oxidative stress , and during interaction-with-host intestinal epithelial cells and macrophages .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23306672" target="_blank">23306672</a>]<br></li>
<li>Microbiology ( 2013 ) , 159 , 591 -- 602 DOI 10.1099 / mic.0.064329-0 Role of the Salmonella enterica serovar Typhi Fur regulator and small RNAs RfrA and RfrB in iron-homeostasis and interaction-with-host cells Jean-Mathieu Leclerc ,1 Charles M. Dozois2 and France Daigle1 Correspondence France Daigle 1 Department of Microbiology and Immunology , University of Montreal , C.P. 6128 Succursale Centre-Ville , Montreal , QC H3C 3J7 , Canada france.daigle@umontreal.ca 2 INRS-Institut Armand-Frappier , 531 boul .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23306672" target="_blank">23306672</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_345">
[ 27 ] J.M. Leclerc , C.M. Dozois , F. Daigle , Role of the Salmonella enterica serovar Typhi Fur regulator and small RNAs RfrA and RfrB in iron-homeostasis and interaction-with-host cells , Microbiology 159 ( 2013 ) 591 -- 602 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24937451" target="_blank">24937451</a>]<br></label>
<input id="rp_similarity_345", type="checkbox">
<div class="RP_similar">
<li>[ 28 ] Leclerc JM , Dozois CM , Daigle F. Role of the Salmonella enterica serovar Typhi Fur regulator and small RNAs RfrA and RfrB in iron-homeostasis and interaction-with-host cells .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24239962" target="_blank">24239962</a>]<br></li>
<li>Role of the Salmonella enterica serovar Typhi Fur regulator and small RNAs RfrA and RfrB in iron ho-meostasis and interaction-with-host cells .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24990908" target="_blank">24990908</a>]<br></li>
<li>Role of the Salmonella enterica serovar Typhi Fur regulator and small RNAs RfrA and RfrB in iron-homeostasis and interaction-with-host cells .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26944792" target="_blank">26944792</a>]<br></li></div>
<li>Most of the recognized regulators in P. aeruginosa ( determined from the regulatory network proposed by Galán - Vásquez ) can be classified into certain groups including quorum-sensing ( RhlR , LasR , BqsR , BqsS , VqsR , VqsS , QscR and PsrA ) , biofilm ( AlgQ , AlgR , AlgU , AlgW , AlgZ , MucA , MucB , MucC , MucD and FleQ ) , anaerobic-respiration and nitrate reduction ( Anr , NarL , Dnr , RoxR and RoxS ) , iron uptake ( Fur , FpvA , PtxR , PvdS and PchR ) , antibiotic resistance ( NfxB and MexT ) and type-III-secretion-systems ( GacA , ExsA , ExsD and Vfr ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26387845" target="_blank">26387845</a>]<br></li>
<li>Functional interactions between the carbon and iron utilization regulators , Crp and Fur , in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28553268" target="_blank">28553268</a>]<br></li>
<li>Comparing conserved co-expressed regulators between E. coli and S. enterica reported by Zarrineh et al. with identified P. aeruginosa regulators showed that Fur ( responsible for transcription of genes involved in iron-homeostasis ) , sigma factor FliA and anti sigma factor FlgM ( both involved in the regulation of motility and flagellar synthesis ) and GlpR ( Glycerol-3-phosphate regulon repressor ) were conserved equivalently in E. coli , P. aeruginosa and S. enterica .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26387845" target="_blank">26387845</a>]<br></li>
<li>Oglesby AG , Murphy ER , Iyer VR , Payne SM ( 2005 ) Fur regulates acid resistance in Shigella ﬂexneri via RyhB and ydeP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24691548" target="_blank">24691548</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_353">
Fur regulates acid resistance in Shigella ﬂexneri via RyhB and ydeP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22824499" target="_blank">22824499</a>]<br></label>
<input id="rp_similarity_353", type="checkbox">
<div class="RP_similar">
<li>Oglesby , A.G. , Murphy , E.R. , Iyer , V.R. and Payne , S.M. ( 2005 ) Fur regulates acid resistance in Shigella ﬂexneri via RyhB and ydeP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18267966" target="_blank">18267966</a>]<br></li></div>
<li>These results indicate a complex interplay between Fur and HNS in regulating cellular global behavior .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23040276" target="_blank">23040276</a>]<br></li>
<li>Interplay between Fur and HNS in controlling virulence gene-expression in Salmonella typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24643535" target="_blank">24643535</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27185788" target="_blank">27185788</a>]<br></li>
<li>Genes involved in anaerobic-respiration ( dmsABC ) and ethanolamine utilization ( eutSPQTDMEJGHABCLK ) were activated by Fur ( Additional file 2 : Table S2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22017966" target="_blank">22017966</a>]<br></li>
<li>Site-speciﬁc mutations in putative Fur and MntR binding sites disrupted regulation by Fur and MntR , conﬁrming that those sites are involved in transcriptional regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659669" target="_blank">15659669</a>]<br></li>
<li>OmpR , Lrp and Fur are involved in pESI fimbriae regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28817673" target="_blank">28817673</a>]<br></li>
<li>The Fur - and MntR-controlled mntH gene , also involved in Mn2 + and Fe2 + uptake ( with a preference for Mn2 + ) , was also induced but in lesser extent ( up to 2.1-fold at 45 min ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28553268" target="_blank">28553268</a>]<br></li>
<li>In S. enterica serovar Typhimurium ( S. Typhimurium ) , Fur regulates systems implicated principally in iron acquisition , but also other genes involved in acid stress and adaptation , oxidative stress resistance and virulence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23306672" target="_blank">23306672</a>]<br></li>
<li>The iron-sensing Fur regulator represses feoB transcription under iron-replete conditions ( 3 , 11 ) while the oxygen-sensing Fnr regulator promotes feoB transcription under anaerobic conditions ( 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23708131" target="_blank">23708131</a>]<br></li>
<li>For example , three pH-regulated genes that were not subject to obvious iron regulation were nevertheless rendered uninducible by acid pH in a Fur mutant ( 19a ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1624426" target="_blank">1624426</a>]<br></li>
<li>The Fur regulon contains genes which are involved in iron acquisition , a function which is essential for bacterial multiplication .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7642488" target="_blank">7642488</a>]<br></li>
<li>The 17-kDa transcriptional regulator , Fur , plays an important role in bacterial iron-homeostasis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22017966" target="_blank">22017966</a>]<br></li>
<li>For example , Fur represses aerobactin-biosynthesis using ferrous iron , cobalt , or manganese [ 2 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22017966" target="_blank">22017966</a>]<br></li>
<li>In response to iron , Fur regulates genes encoding iron-transport systems as well as transport systems for other metals , metabolic enzymes , * Corresponding author .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659669" target="_blank">15659669</a>]<br></li>
<li>Fur represses iron-responsive genes inside macrophages .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24643535" target="_blank">24643535</a>]<br></li>
<li>Fur represses siderophore-synthesis genes such as entABCE and iroDEN under iron-replete conditions ( 27 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24643535" target="_blank">24643535</a>]<br></li>
<li>Transcriptional regulation by ferric uptake regulator ( Fur ) in pathogenic bacteria .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24990908" target="_blank">24990908</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_371">
Fur regulon in Gram-negative bacteria .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11932449" target="_blank">11932449</a>]<br></label>
<input id="rp_similarity_371", type="checkbox">
<div class="RP_similar">
<li>Fur regulon in gram-negative bacteria .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24488311" target="_blank">24488311</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23873910" target="_blank">23873910</a>]<br></li></div>
<li>Fur regulon in Gramnegative bacteria .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8824613" target="_blank">8824613</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7642488" target="_blank">7642488</a>]<br></li>
<li>Fur regulon in gramnegative bacteria .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10869064" target="_blank">10869064</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12897017" target="_blank">12897017</a>]<br></li>
<li>Fur regulates acid Oglesby-Sherrouse , A.G. , Murphy , E.R. , 2013 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27242152" target="_blank">27242152</a>]<br></li>
<li>Fur Regulated Genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22017966" target="_blank">22017966</a>]<br></li>
				</div></ul><br>
			<label class="collapse_SIT" for="_5"><font size="5px">Fur binding site</font></label>
				<input id="_5" type="checkbox">
				<div class="SIT"><br><ul><li>HNS box sequence HNS box location Strand þ þ 3006305y3006313 3010825y3010833 3011464y3011472 3012245y3012253 3013028y3013036 3018129y3018137 3018934y3018942 3019190y3019198 3019673y3019681 3020913y3020921 3023611 ... 3023619 3023757y3023765 3023758y3023766 3023975y3023983 3027358y3027366 3027433y3027441 3031507y3031515 3032259y3032267 3034613y3034621 3036799y3036807 3038496y3038504 3038724y3038732 3038966y3038974 3042887y3042895 3044398y3044406 3045587y3045595 1463152y 1465356y 1473069y 1473917y 1475863y 1479318y 1479477y 1479846y 1480623y 1484296y 1484705y 1485569y 1486355y 1488147y 1489155y 1489392y 1489509y 1491640y 1493907y 1494864y 1495827y 1497681y 1500166y 1500389y 1500827y GCTATATAA AATATATTG GCTATATTA AAAATATCG GCAATATAA GCTATATGA AATATATTG GCTATATTA GCTAAATAA AATACATCG GCAATATAA GCGATATAA GCTATATTA AAAATATCG GCTATATAA GCTAAATAA 1 2 1 2 GCTATAAAA GCGATATAA GCTATGTAA AATATATTG GCTGTATAA AAAATATCG GATATATAA AATTTATCG GCTATAAAA AATATACCG GCTATATAC AATATACCG GCTTTATAA GCTATGTAA AATATAACG ATTATATCG 1463160 1465364 1473077 1473925 1475871 1479326 1479485 1479854 1480631 1484304 1484713 1485577 1486363 1488155 1489163 1489400 þ 3 4 5 6 7 8 9 10 11 12 13 14 15 16 3 4 5 6 7 8 9 10 11 12 13 14 15 16 þ þ þ þ þ þ þ þ þ þ þ þ þ þ AATACATCG GCTATTTAA GCTATACAA GCTATAAAA AATAAATCG AATATAACG GCTTTATAA GTTATATAA AAAATATCG GCTATACAA 17 18 19 20 21 22 23 24 25 26 17 18 19 20 21 22 23 24 25 GCTACATAA GCTTTATAA ATTATATCG GCTATATCA GCTGTATAA GGTATATAA AATATATCC GCTATTTAA AATGTATCG 1489517 1491648 1493915 1494872 1495835 1497689 1500174 1500397 1500835 þ þ þ þ þ þ þ þ Frequency of Binding Sites 2.2 2.0 1.8 1.6 1.4 1.2 1.0 0.8 0.6 0.4 Fur HNS SPI2 SPI1 SPI3 SPI4 0.2 0.0 0 1x106 2x106 3x106 4x106 5x106 Base Number Fig. 2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23040276" target="_blank">23040276</a>]<br></li>
<li>( A ) Motif consensus logo for Fur recognition motif was generated with software available from http://weblogo.berkeley .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27564394" target="_blank">27564394</a>]<br></li>
<li>Fur box sequence Fur box location Strand Fur box location GCAAATAAGAATTATTTTC CGTTTATTCTTAATAAAAG TTCTTAATAAAAGTAAATA ATAAATGATAAAAAATATT GCAAATTTTTATCGCTATT AATAATGTTATAAACTTTC AGAAATTAAAATCACTTTT ATTTTAACATTATTAAATT CTATAATTTATAATAATAC TCTGACTATTATCATTATC TATAATTATAACTATAATT TGTAACCATTATTAATATC TTTTTAAGTAACGTAAACG AATAATGATCTTAATTATC TTATTACTAGAATTAATAG GGAAACCATTATTGCTATT 1 2 þ 1 2 AATAAAAATATAAATTTTC CATTTACTATTAATGATCT GTAAATGATAATTACTAGA TTTTTATTTAGCGAAATGT TTTTTGCAAATAGCAATGC CTTTTTCCATTAGTAGTCG TTTTTAAGTATAAAAATAA TTTTTAAAAATAGCGAATG TAAGATCAATATCATTTTC GTAAATAATGTTTATTAGC ATTAATCATATATATTTTT TATAAAAATAATTGTTAGT TTAAAAAATAATTCTTATT TTTTTATTAAGAATAAATT TTTAATTATATTAATTATA 3006269y 3006269y 3006275y 3010948y 3012156y 3012392y 3017894y 3018803y 3020972y 3025212y 3025847y 3026728y 3031088y 3032427y 3032427y 3040063y 3040578y 3045003y 24442y 26779y 26779y 603891y 605534y 606222y 611252y 611370y 612967y 613070y 3828073y 3828084y 3828157y 3828160y 3828174y 24460 26797 3006287 3006287 3006293 3010966 3012174 3012410 3017912 3018821 3020990 3025230 3025865 3026746 3031106 3032445 3032445 3040081 þ 3 4 5 6 7 8 9 10 11 12 13 14 15 16 3 4 5 6 7 8 9 10 11 12 13 14 15 26797 603909 605552 606240 611270 611388 612985 613088 3828091 3828102 3828175 3828178 3828192 þ þ þ þ þ þ þ þ þ þ þ þ 3040596 3045021 GGTTATAAGAAGTATTTTC GAAAAAAATAAACACTATT 17 18 Table 5 List of Fur boxes in the ﬂagellar regulon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23040276" target="_blank">23040276</a>]<br></li>
<li>After sequence analysis , only 11 clones carrying a Fur box were identiﬁed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12652904" target="_blank">12652904</a>]<br></li>
<li>Fur box sequence S. no .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23040276" target="_blank">23040276</a>]<br></li>
<li>Fur box sequence Fur box location Strand þ TTAAATAAGAATTAGTTTT TTTTAGTTAATTGTAAATT GACAGTGATAATTATTTGT CTGTCACTATTAATAAACA TTATTGCAAAAAGTAACCG ATTTTACCTTTAGTAAACA TAAAATGGAAATCATTTGT TTAAATATTAAACGTTATT CTATTGTCTAAAGCAATCG 1 2 3 4 5 6 7 8 9 424815y 433201y 436986y 436986y 1257832y 2022547y 2022547y 2022571y 2048544y 424833 433219 437004 437004 1257850 2022565 2022565 2022589 2048562 Strand Fur box location Fur box sequence S. no .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23040276" target="_blank">23040276</a>]<br></li>
<li>We have also identified a Fur recognition sequence [ 33 ] ( Figure 4A ) located at position -309 within the nrdHIEF promoter region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20593029" target="_blank">20593029</a>]<br></li>
<li>We had previously noted that the S. enterica serovar Typhimurium mntH promoter contains not only OxyR and Fur binding motifs but also a novel inverted-repeat element .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12029031" target="_blank">12029031</a>]<br></li>
<li>Fur binds upstream of hns in a metal-dependent manner .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21075923" target="_blank">21075923</a>]<br></li>
<li>Speciﬁcally , we calculated frequency of hits ( Fur boxes and HNS-nucleation sites ) along with AT % in bin size of 100,000 nucleotides .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23040276" target="_blank">23040276</a>]<br></li>
<li>Although to a different extent , both isrE and ryhB are regulated by iron metabolism and harbor consensus sites for Fur binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18267966" target="_blank">18267966</a>]<br></li>
<li>In addition , this RcsB-binding site is far from the nucleotide sequences required for Fur and RpoS binding ( 177 and 126 bp , and 259 bp , respectively , Fig. 5a ) ( Yoo et al. , 2007 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25028458" target="_blank">25028458</a>]<br></li>
<li>In addition , we also located a number of Fur and HNS binding sites in close proximity to accessory regulators hilC , sprB , and hilA [ 17,19,33,46,47 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23040276" target="_blank">23040276</a>]<br></li>
<li>Materials and methods The genome of Salmonella typhimurium LT2 14028 [ 48 ] was used as a template to perform scanning for locating Fur binding sites , HNS-nucleation sites , and computing AT base percentage in the chromosome .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23040276" target="_blank">23040276</a>]<br></li>
<li>This likely reﬂects differences in spacing between the MntR binding sites and their respective 10 sites and perhaps Fur binding sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659669" target="_blank">15659669</a>]<br></li>
<li>-10 and -35 sites are highlighted in green and the predicted Fur binding site is in bold font and underlined .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28817673" target="_blank">28817673</a>]<br></li>
<li>Again putative Fur boxes were identiﬁed upstream of these sequences ( Table II and Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12652904" target="_blank">12652904</a>]<br></li>
<li>The predicted Fur binding site is located within 385 and 357 nucleotides upstream of hns ( signiﬁed by a ﬁlled triangle in Fig. 5A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21075923" target="_blank">21075923</a>]<br></li>
<li>The predicted 35 and 10 sites of hns are located 70 and 50 nucleotides upstream of ATG , respectively , 309 and 292 nucleotides downstream from our putative Fur binding site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21075923" target="_blank">21075923</a>]<br></li>
<li>No consensus binding sites for Fur were found upstream of this ORF ( Fig. 2N and 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7642488" target="_blank">7642488</a>]<br></li>
<li>A putative Fur binding site was identiﬁed upstream of hns , but no evidence was presented for Fur binding to this region ( 64 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21075923" target="_blank">21075923</a>]<br></li>
<li>TABLE II Comparison of potential Fur box found in FURTA-positive clones with the consensus sequence found by de Lorenzo et al. ( 1987 ) Matches GATAATGAT A ATCATTATC Consensus pSC2 : pSC3 : pSC4 : pSC5 : pSC8 : pSC9 : pSC10 : pSC12 : pSC14 : pSC15 PSC16 ( 13/19 ) ( 13/19 ) ( 17/19 ) ( 14/19 ) ( 13/19 ) ( 15/19 ) ( 17/19 ) ( 12/19 ) ( 13/19 ) ( 12/19 ) ( 14/19 ) GATAACTAT T TGCATTTGC GATACTGAT T ATGGTTTAT GATATTGGT A ATTATTATC GCAAACAAT A ATAATTATC GTTAATCAT A ACCGTGATA GAAAATGAG A AGCATTAAC GATAATCAC T ATCATTATC GATACTGAT T ATGGTTTAT GCAAATGCA A ATAGTTATC CTTGAAGAT A ATGGTTACC GATAATGAA A AAGGTTATG DNA Sequencing and Analysis Automated DNA sequencing was done on the Applied Biosystems Model 373A DNA system by using the T3 and T7 universal primers and primers complementary to regions already sequenced in the plasmid .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12652904" target="_blank">12652904</a>]<br></li>
<li>The DNA sequences corresponding to the putative Fur box ( 33 ) were indicated as a gray box .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24643535" target="_blank">24643535</a>]<br></li>
<li>Upon complexing with Fe , 2 + Fur recognises a speciﬁc DNA sequence , the fur box and binds to it .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16023758" target="_blank">16023758</a>]<br></li>
<li>NrdR and Fur recognition sequences The presence of putative nrdR recognition sequences has been studied by phylogenetic profiling in numerous bacterial genomes [ 22 ] and it has been established that the NrdR recognition site consists of two tandem NrdR boxes separated by approximately 31 -- 32 bp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20593029" target="_blank">20593029</a>]<br></li>
<li>However , it is likely that the Fur site is larger than the 19-bp consensus sequence extending to position 74 in Fig. 1 , showing an F-F-x-R-R architecture as deﬁned by Lavrrar and McIntosh ( 23 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659669" target="_blank">15659669</a>]<br></li>
<li>Transcriptional start site is indicated in bold and underlined as is the 19 bp Fur box in the nrdHIEF promoter region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20593029" target="_blank">20593029</a>]<br></li>
<li>In Bacillus subtilis , the dps homologue is preceded by sequences similar to the iron box consensus element for recognition by the E. coli Fur protein ( Chen et al. , 1993 ) , which is consistent with Fur-mediated repression under Fe-replete conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17302823" target="_blank">17302823</a>]<br></li>
<li>The fur box is a 19-bp consensus sequence which is recognized by the Fur repressor protein ( Stojiljkovic et al. 1994 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12115054" target="_blank">12115054</a>]<br></li>
<li>Under ironreplete conditions , dimeric Fe2 - bound Fur binds to consensus DNA sequences and represses the transcription of iron uptake systems ( 53 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25107963" target="_blank">25107963</a>]<br></li>
<li>Every hit was located on the genome [ 56 ] and listed with its particular target , location ( in a gene , upstream or/and downstream to gene ) and sequence of Fur box in Table S1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23040276" target="_blank">23040276</a>]<br></li>
<li>Several speciﬁc base changes in the motifs were made ( 12 base changes for the Fur binding site and 16 base changes for the MntR binding site ) plus base changes between the motifs were made when restriction sites were incorporated into the region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659669" target="_blank">15659669</a>]<br></li>
<li>Mutations in the Fur binding sites decreased transcription 80 to 90 % , likely due to changes in the 35 sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659669" target="_blank">15659669</a>]<br></li>
<li>Previous studies in our laboratory indicated an upregulation of nrdHIEF under iron depletion and we identified a putative Fur binding box 309 bp upstream from the transcription start site ( Figure 4A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20593029" target="_blank">20593029</a>]<br></li>
<li>For ( A ) and ( B ) representative data are shown with the differential rate of synthesis ( U/OD600 ) ± standard deviations from three independent experiments listed Identification of new Fur targets Table 3 shows genes differentially regulated in Δfur that contain a putative Fur binding site located within -400 to +50 nucleotides relative to the translational start site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22017966" target="_blank">22017966</a>]<br></li>
<li>The promoter sequence of the adhE gene contains a Fur binding box and a NarL binding site , which , consistent with previous reports ( 31 , 32 ) , contrib-ute to the anaerobia-induced property of Salmonella .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21474796" target="_blank">21474796</a>]<br></li>
<li>STnc4000 contains a putative Fur binding site overlapping the -10 site of the promoter region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27564394" target="_blank">27564394</a>]<br></li>
<li>The second highest scoring putative Fur binding site was obtained for a sequence overlapping the -10 site of the STnc3250 promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27564394" target="_blank">27564394</a>]<br></li>
<li>Black boxes indicate the presence of a consensus Fur binding site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7642488" target="_blank">7642488</a>]<br></li>
<li>Motif analysis of the Fur recognition site A position-specific scoring matrix ( PSSM ) was generated using alignment of the homologous Salmonella sequences of 15 published Fur binding sites from E. coli ( available at http://arep .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27564394" target="_blank">27564394</a>]<br></li>
<li>In the case of the iron-storage protein ferritin , FtnA , gene expression is induced by direct interaction of Fe ( II ) - Fur with an extended Fur binding site ( containing five tandem Fur boxes ) located upstream of the ftnA promoter ( Nandal et al. , 2010 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21722794" target="_blank">21722794</a>]<br></li>
<li>We also identiﬁed 28 nucleotides upstream of hns that contained the Fur binding motif responsible for the Fur-de-pendent activation of hilA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21075923" target="_blank">21075923</a>]<br></li>
<li>Consis-2 + 2 + tent with these results in-silico ipfA promoter analysis identified putative imperfect 19-bp Fur binding site ( TAAAATGGTAATCAAATGA ) partially overlap with the -10 and the -35 sites in the ipfA promoter ( S1B Fig ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28817673" target="_blank">28817673</a>]<br></li>
<li>Bases that were changed in the Fur binding and/or MntR binding site mutants are shown below the putative binding sequences .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659669" target="_blank">15659669</a>]<br></li>
<li>In our original study , a larger region of the mntH promoter that included the MntR binding site , Fur binding site , and OxyR binding site was cloned upstream of lacZ in a single - or low-copy-number plasmid .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659669" target="_blank">15659669</a>]<br></li>
<li>A 15/19 nucleotide match to the consensus binding site for the Fe - and Mn - activated 21 21 transcriptional repressor protein Fur ( GCACTTGATAATCATTATC in S. typhimurium ) overlaps a likely 235 RNA polymerase binding site , as is the case with other well-characterized Fur boxes ( Coy and Neilands , 1991 ; Althaus et al. , 1999 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10844693" target="_blank">10844693</a>]<br></li>
<li>( a ) The dps promoter region : localization of the putative RpoS , OxyR and Fur binding sites is indicated by grey boxes , and the putative RcsB-binding site is identified by a black box .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25028458" target="_blank">25028458</a>]<br></li>
<li>The consensus OxyR binding motif ( 38 ) , Fur binding motif ( 9 , 23 ) , and MntR binding motif ( based on the mntH promoter of E. coli ) ( 30 ) are shown above the corresponding putative sites underlined in the mntH ( 19 ) and sitABCD promoter regions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659669" target="_blank">15659669</a>]<br></li>
<li>The putative Fur binding site in sitABCD ( Fig. 1 ) was identiﬁed by comparison with the Fur binding sites of mntH from E. coli ( 30 ) and Salmonella ( 19 , 22 ) and the known consensus Fur binding sequence ( 9 , 23 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659669" target="_blank">15659669</a>]<br></li>
<li>induced by hydrogen peroxide , weakly induced by EDTA and unresponsive to paraquat , consistent with the presence of Fur and OxyR binding sites in the promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10844693" target="_blank">10844693</a>]<br></li>
<li>Promoter location including the -10 and -35 boxes was predicted by BPROM , and the Fnr and Fur binding sites were predicted by the Virtual Footprint tool .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28817673" target="_blank">28817673</a>]<br></li>
				<label class="SIT_collapse_similarity" for="sit_similarity_52">
The sequence , score , and the location of Fur boxes in the ﬁmbrial genes in Salmonella are given in the Table .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23040276" target="_blank">23040276</a>]<br></label>
<input id="sit_similarity_52", type="checkbox">
<div class="SIT_similar">
<li>The sequence , score , and the location of Fur boxes in the SPI2 region are given in the table .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23040276" target="_blank">23040276</a>]<br></li>
<li>The sequence , score , and the location of Fur boxes in the ﬂagellar regulon are given in the Table .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23040276" target="_blank">23040276</a>]<br></li></div>
<li>While HNS binding of the SPI1 region has been previously identiﬁed as a silencing mechanism [ 89 ] , the role of presence of Fur binding sites in the SPI1s ; and HNS binding in the SPI2 , SPI3 , and SPI4 regions has not been previously reported .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23040276" target="_blank">23040276</a>]<br></li>
<li>In fact , sitABCD lies at the end of the SPI-1 pathogenicity island ( 43 ) such that the boundary of SPI-1 is located immediately upstream of the Fur binding site ( 26 ) and within the DNA that corresponds to the OxyR binding site o mntH ( 19 ) ( Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659669" target="_blank">15659669</a>]<br></li>
<li>In the present study , the promoter regions encompassing the MntR and Fur binding sites were cloned upstream of lacZ and integrated into the chromosome in a single copy .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659669" target="_blank">15659669</a>]<br></li>
<li>The RNA lengths ( nucleotides ) were estimated based on 50 end mapping by primer extension and the position of the last uridine at the end of the rho-independent transcription terminator As was observed for ryhB homologs in other bacteria ( 21,28,29 ) , the sequences of the S. typhimurium ryhB gene and its paralog isrE contain a highly conserved core region spanning nucleotides 37 -- 69 and consensus binding sites for Fur ( ferric uptake regulator ) that overlap their promoter regions ( Figure 3A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18267966" target="_blank">18267966</a>]<br></li>
<li>We have also analyzed the upregulation of nrdHIEF by Fur and detected a Fur recognition sequence within the nrdHIEF promoter region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20593029" target="_blank">20593029</a>]<br></li>
<li>A. Fur box mutated sequence for the nrdHIEF promoter region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20593029" target="_blank">20593029</a>]<br></li>
<li>Multiple alignment with ClustalW of the E. coli Fur box consensus sequence , S. Typhimurium LT2 Fur box within the fur promoter region and S. Typhimurium LT2 Fur box in the nrdHIEF promoter region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20593029" target="_blank">20593029</a>]<br></li>
<li>The Fe ( II ) - Fur complex binds a regulatory DNA sequence , designated Fur-box , which is located in the promoter region of Fur-regulated genes ( 11 , 16 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7642488" target="_blank">7642488</a>]<br></li>
<li>Recently , a computational analysis predicted that the consensus DNA sequence recognized by Fur ( i.e. , Fur box ) ( 11 , 12 ) exists on the ssrB promoter ( Fig. 4A ) ( 33 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24643535" target="_blank">24643535</a>]<br></li>
<li>The presence of a consensus Fur box in the promoter region of mntH suggests that Mn or Fe binding to Fur mediates this 21 21 effect on mntH transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10844693" target="_blank">10844693</a>]<br></li>
<li>E. coli Fur typically acts as corepressor that binds to a specific DNA sequence , the Fur box , when metal-loaded and represses the transcription of downstream genes by hindering the access of RNA polymerase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21722794" target="_blank">21722794</a>]<br></li>
<li>For it example , will be useful to examine type C and D loci for the presence or absence of a Fur box , the identified consensus DNA sequence to which iron-complexed Fur binds ( 15 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1624426" target="_blank">1624426</a>]<br></li>
<li>When intracellular Fe ( II ) concentrations are high , iron-metal-lated Fur complex binds to a 19-bp DNA consensus sequence , the Fur box , in the promoter region of the iron acquisition genes to repress their transcription ( 1 , 5 , 9 , 21 , 43 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8824613" target="_blank">8824613</a>]<br></li>
<li>When iron is available , Fur binds iron , and the iron-Fur complex binds to a DNA sequence referred to as the Fur box in the promoter region of iron regulated genes , thereby repressing transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18554972" target="_blank">18554972</a>]<br></li>
<li>Fur repression is known to occur due to the binding of the Fur-Fe complex to a 19-bp 2 + consensus site ( GATAATGATAATCATTATC ) found in the operator of a target promoters [ 36 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28817673" target="_blank">28817673</a>]<br></li>
				<label class="SIT_collapse_similarity" for="sit_similarity_70">
The putative 19-bp Fur box is located close to the known SsrB-binding site ( 7 ) on the ssrB promoter ( Fig. 4A ) , and Fur actually binds to DNA regions larger than the Fur box ( 31 bp ) ( 11 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24643535" target="_blank">24643535</a>]<br></label>
<input id="sit_similarity_70", type="checkbox">
<div class="SIT_similar">
<li>Because the putative 19-bp Fur box is located only 5 bp upstream of the SsrB binding site ( 7 , 33 ) on the ssrB promoter ( Fig. 4A ) , and because Fur binds to regions larger than the Fur box ( 31 bp ) ( 11 ) , it is possible that the Fur repressor competes with the SsrB activator for binding to the ssrB promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24643535" target="_blank">24643535</a>]<br></li></div>
<li>The cumulative scores for a sequence of 19 bases were used to compute the Fur box score .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23040276" target="_blank">23040276</a>]<br></li>
<li>Black boxes indicate the presence of a consensus Fur box .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12652904" target="_blank">12652904</a>]<br></li>
				<label class="SIT_collapse_similarity" for="sit_similarity_74">
Recognition of DNA by Fur : a reinterpretation of the Fur box consensus sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27242152" target="_blank">27242152</a>]<br></label>
<input id="sit_similarity_74", type="checkbox">
<div class="SIT_similar">
<li>Baichoo N , Helmann JD ( 2002 ) Recognition of DNA by Fur : a reinterpretation of the Fur box consensus sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28817673" target="_blank">28817673</a>]<br></li>
<li>Baichoo N , Helmann JD : Recognition of DNA by Fur : a reinterpretation of the Fur box consensus sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22017966" target="_blank">22017966</a>]<br></li></div>
<li>Comparison of potential Fur binding sites found in FURTA-positive clones with the consensus sequence found by de Lorenzo et al. ( 16 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7642488" target="_blank">7642488</a>]<br></li>
<li>The absence of an iron-regulated promoter in pUJ501 agrees with sequence analysis data , which indicated a lack of a consensus Fur binding site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7642488" target="_blank">7642488</a>]<br></li>
<li>Clone pSC16 , which was homologous to cytochrome O ubiquinol oxidase subunit II , had a putative Fur box matching the consensus sequence in 14 of 19 bases ( Table II and Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12652904" target="_blank">12652904</a>]<br></li>
<li>A putative Fur box was identiﬁed in the individual upstream regions of iroB and iroN that match the E. coli consensus sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12652904" target="_blank">12652904</a>]<br></li>
<li>A putative Fur box matching the consensus E. coli Fur box ( de Lorenzo et al. , 1987 ) in 17 of 19 bases was present in the upstream region of the iroB of pSC4 ( Table II and Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12652904" target="_blank">12652904</a>]<br></li>
<li>A putative Fur box matching the consensus sequence in 13 of 19 bases of E. coli was located in the upstream region of this sequence ( Table II and Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12652904" target="_blank">12652904</a>]<br></li>
<li>DNA sequence of the clone pSC12 showed homology to ord-6c with a putative Fur box matching the consensus sequence in 12 of 19 bases ( Table II and Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12652904" target="_blank">12652904</a>]<br></li>
<li>Another clone , pSC15 , showed homologous to a hypothetical glutaredoxinlike protein of S. enterica serovar Typhimurium , and the putative Fur box matched the consensus sequence in 12 of 19 bases ( Table II and Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12652904" target="_blank">12652904</a>]<br></li>
<li>Three possible Fur binding sites that match Fur consensus ( gataatgataatcattatc ) at 15 , 16 and 13 of 19 bases are indicated by dashed lines above the sequence of IsrE .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18267966" target="_blank">18267966</a>]<br></li>
<li>Fur sites at ryhB , matching 14 of 19 bases of the consensus are indicated under the sequence of ryhB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18267966" target="_blank">18267966</a>]<br></li>
<li>Search for a Fur binding site in the mntR promoter region revealed a sequence that matched 12 of 19 bases in the Fur binding consensus sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659669" target="_blank">15659669</a>]<br></li>
<li>One FURTA-positive clone , pFT36 , contained a small ORF ( 189 bp ) which was preceded by a sequence matching the consensus site for Fur binding in 16 of 19 bases .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7642488" target="_blank">7642488</a>]<br></li>
				<label class="SIT_collapse_similarity" for="sit_similarity_89">
A potential Fur binding site matching the consensus sequence in 14 of 19 bases was located upstream of this sequence ( Table II and Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12652904" target="_blank">12652904</a>]<br></label>
<input id="sit_similarity_89", type="checkbox">
<div class="SIT_similar">
<li>A potential Fur binding site matching the consensus sequence in 12 of 19 bases was located upstream of this ORF ( Fig. 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7642488" target="_blank">7642488</a>]<br></li>
<li>Approximately 50 bp upstream of this ORF was a potential Fur binding site matching the consensus sequence in 15 of 19 bases .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7642488" target="_blank">7642488</a>]<br></li></div>
<li>We took sum of each calculated likelihood score at each position in a Fur box that was score ( Fur ) and repeated this procedure for all 15 sequences and were used in constructing PSSM , the minimum value of sum of likelihood scores among all Fur boxes was set as Threshold value from Eq .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23040276" target="_blank">23040276</a>]<br></li>
<li>Because the lowest weighted score of a published Fur binding site was 7 , and a number of poorly conserved motifs had a score of approximately 7 , we chose the more conservative threshold score of 10 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27564394" target="_blank">27564394</a>]<br></li>
<li>DNA sequence analysis of these clones showed that 11 out of 15 clones had a Fur binding site ( Fur box ) , and 6 of these clones showed homology to the iron-regulated genes of S. enterica serovar Typhi and/or E. coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12652904" target="_blank">12652904</a>]<br></li>
<li>The complete nucleotide sequence of 9848 bp of the iroA locus of S. enterica serovar Choleraesuis consists of iroB , C , D , E and N genes , which are transcriptionally regulated by Fur .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12652904" target="_blank">12652904</a>]<br></li>
<li>The nucleotide sequence of three FURTA-positive clones contained Fur binding motifs but showed no homology to previously described Fur-regulated promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7642488" target="_blank">7642488</a>]<br></li>
<li>( B ) Consensus sequence for Fur recognition aligned with putative Fur recognition sites within Furregulated sRNA promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27564394" target="_blank">27564394</a>]<br></li>
<li>The Fur box was originally known to be a 19-bp palindromic consensus sequence ( GATAATGATAATCATTATC ) ( de Lorenzo et al. , 1987 ) ; later , it has been proposed that this consensus sequence contains three or more copies of hexamer ( GATAAT ) ( Baichoo and Helmann , 2002 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27242152" target="_blank">27242152</a>]<br></li>
<li>If a sequence corresponding to a Fur binding motif was identified , then this sequence was given a weighted score [ 45 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22017966" target="_blank">22017966</a>]<br></li>
<li>The published Fur binding sequences were scanned with the same PSSM to establish a minimum threshold weighted score , below which predicted motifs were considered to be false-positives .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27564394" target="_blank">27564394</a>]<br></li>
<li>This indicates that our bioinformatic analyses indeed agree with experimentally identified Fur binding sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22017966" target="_blank">22017966</a>]<br></li>
<li>A ﬁlled triangle depicted a predicted Fur binding site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21075923" target="_blank">21075923</a>]<br></li>
<li>edu [ 98 ] , using published Fur binding sites ( ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27564394" target="_blank">27564394</a>]<br></li>
<li>Fur binding sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20593029" target="_blank">20593029</a>]<br></li>
<li>Nucleotides that were highly conserved in known MntR or Fur ( 36 ) binding sites were changed in order to disrupt binding to MntR or Fur , respectively ( Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659669" target="_blank">15659669</a>]<br></li>
<li>Fur binding to nrdHIEF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20593029" target="_blank">20593029</a>]<br></li>
				</ul></div><br>
			<label class="collapse_TU" for="_6"><font size="5px">Transcription unit regulated by Fur</font></label>
				<input id="_6" type="checkbox">
				<div class="TU"><br><ul><li>Fur does not directly regulate hilD transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
<li>The Fur protein was shown to repress feoB transcription under iron-replete conditions ( 3 , 11 ) , while the activator Fnr protein promotes feoB transcription under anaerobic conditions ( 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23708131" target="_blank">23708131</a>]<br></li>
<li>The iron-sensing Fur regulator represses feoB transcription under iron-replete conditions ( 3 , 11 ) while the oxygen-sensing Fnr regulator promotes feoB transcription under anaerobic conditions ( 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23708131" target="_blank">23708131</a>]<br></li>
<li>The FeFur repressed sufABCDS operon , encoding an alternative Fe-S cluster biosynthetic pathway employed during redox stress and iron starvation in E. coli ( Outten et al. , 2004 ) , was also induced ( 3.77 - to 9.35-fold at 45 min ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28553268" target="_blank">28553268</a>]<br></li>
<li>The Fe-Fur induced ftnA gene ( encoding ferritin A , an iron-storage protein ) was suitably repressed , an effect that increased from 7 to 45 min ( 0.19-fold at 45 min ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28553268" target="_blank">28553268</a>]<br></li>
<li>In addition , the nrdHIEF genes specifying an alternative , Fe-Fur repressed , Mndependent ribonucleotide reductase ( Martin and Imlay , 2011 ) , were induced 5.18 - to 15.4-fold at 45 min .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28553268" target="_blank">28553268</a>]<br></li>
<li>More recently , it was proposed that Fur activates hilA by repressing H-NS ( Troxell et al. 2010 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22021388" target="_blank">22021388</a>]<br></li>
<li>Many genes that are induced under iron-starvation conditions are regulated by Fur ( Stojiljkovic et al. , 1994 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10712695" target="_blank">10712695</a>]<br></li>
<li>This implies that genes controlled by H-NS are regulated by iron through Fur .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22017966" target="_blank">22017966</a>]<br></li>
<li>This gene ( hmpA ) is repressed by Fnr and contained a putative Fnr binding site , but did not contain a predicted Fur binding site [ 21,95,96 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22017966" target="_blank">22017966</a>]<br></li>
<li>However , in contrast to E. coli , S. Typhimurium acnA was not induced in stationary phase and was repressed , rather than activated , by Fur ( Fig. 3b ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23637460" target="_blank">23637460</a>]<br></li>
<li>Genes involved in anaerobic respiration ( dmsABC ) and ethanolamine utilization ( eutSPQTDMEJGHABCLK ) were activated by Fur ( Additional file 2 : Table S2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22017966" target="_blank">22017966</a>]<br></li>
<li>These data suggest that ftnB and dps are negatively regulated by Fur in Fe-rich medium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17302823" target="_blank">17302823</a>]<br></li>
<li>It has been reported that in Salmonella dps expression is controlled by RpoS and Fur proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25028458" target="_blank">25028458</a>]<br></li>
<li>These results suggested that crl transcription is not repressed by Fur and Crl in ATCC 14028 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20008066" target="_blank">20008066</a>]<br></li>
<li>These data show that sit transcription is negatively controlled by both MntR and Fur .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659669" target="_blank">15659669</a>]<br></li>
<li>The sitABCD promoter is regulated by MntR and Fur .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659669" target="_blank">15659669</a>]<br></li>
<li>Siderophore expression by S. Typhi and both sRNAs were regulated by iron and by Fur .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23306672" target="_blank">23306672</a>]<br></li>
<li>ipf genes are negatively regulated by Fur and activated by OmpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28817673" target="_blank">28817673</a>]<br></li>
<li>crl transcription in E. coli W3110 is repressed by Fur ( by 100-fold ) and by Crl ( by more than 10-fold ) ( 28 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20008066" target="_blank">20008066</a>]<br></li>
<li>NrdEF is usually repressed by Fur .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24044554" target="_blank">24044554</a>]<br></li>
<li>Fur represses siderophore synthesis genes such as entABCE and iroDEN under iron-replete conditions ( 27 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24643535" target="_blank">24643535</a>]<br></li>
<li>The S. Typhimurium ferritins are differentially regulated Previous studies in E. coli have shown that Fur positively regulates ftnA and bfr expression under Fe-replete conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17302823" target="_blank">17302823</a>]<br></li>
<li>In this study , ftnB and fumA were positively regulated by Fur under high-iron conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12897017" target="_blank">12897017</a>]<br></li>
<li>Expression of prgH and invF decreased ninefold and sixfold under iron limited conditions , this regulatory mechanism being largely mediated by Fur .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27704705" target="_blank">27704705</a>]<br></li>
<li>Both bfr and ftnA are under positive regulation by Fur , and hence are expressed under iron-replete conditions consistent with iron storage/detoxification roles , while both ftnB and dps are negatively regulated by Fe ( II ) - Fur and thus expressed under conditions of iron-depletion .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21722794" target="_blank">21722794</a>]<br></li>
<li>Moreover , ftnB and dps are repressed by the Fe-responsive regulator Fur and induced under conditions of Fe limitation , whereas bfr and ftnA are maximally expressed when Fe is abundant .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17302823" target="_blank">17302823</a>]<br></li>
<li>( A ) Regulation of hilA and sitA by Fur and metal chelation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
<li>Genes associated with DNA repair and purine meta-bolism ( nrdAB , nth , recA , and nei ) were repressed by Fur under anaerobic conditions ( Additional file 2 : Table S2 ) , thus implicating Fur as a regulator of DNA repair and de novo synthesis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22017966" target="_blank">22017966</a>]<br></li>
<li>Both are known to regulate virulence factors in Salmonella ( Fur positively , and HNS negatively ) [ 17,22,95 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23040276" target="_blank">23040276</a>]<br></li>
<li>In E. coli and Salmonella , five transcription factors have been implicated : hmp transcription is activated by MetR ( Membrillo-Herna ́ndez et al. , 1998 ) and RamA ( Hernández - Urzúa et al. , 2007 ) and is repressed by NsrR ( Bodenmiller & Spiro , 2006 ) , FNR ( Cruz-Ramos et al. , 2002 ; Poole et al. , 1996 ) and Fur ( D'Autreaux et al. , 2002 ; Hernández - Urzúa et al. , 2007 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20829289" target="_blank">20829289</a>]<br></li>
<li>In S. Tm , cib expression is negatively regulated in a Fur ( ferric uptake regulator ) - dependent and LexA ( locus for x-ray sensitivity A ) - dependent way ( Mankovich et al. , 1986 ; Nedialkova et al. , 2014 ) and maximally induced under iron limitation and upon exposure to DNA damaging agents .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26439675" target="_blank">26439675</a>]<br></li>
<li>The Fe-Fur repressed yqjH and bfd genes are considered to be involved in iron mobilization ( Miethke et al. , 2011 ; Yao et al. , 2012 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28553268" target="_blank">28553268</a>]<br></li>
<li>The iron-rationing response is controlled indirectly by Fur through RfrA and RfrB in Salmonella ( RyhB in E. coli ) and largely functions to repress genes encoding ironcontaining proteins in reaction to low iron conditions ( McHugh et al. , 2003 ; Massé et al. , 2005 ; Yang et al. , 2013 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28553268" target="_blank">28553268</a>]<br></li>
<li>Both are Fe-Fur controlled , but in a reciprocal fashion -- sodA is Fe-Fur repressed whilst sodB is Fe-Fur ( via RfrA and RfrB ) induced ( Troxell et al. , 2011 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28553268" target="_blank">28553268</a>]<br></li>
<li>The ydiE gene is Fe-Fur repressed in S. Typhimurium ( Lutz and Bujard , 1997 ; Bjarnason et al. , 2003 ) , and its induction in EWMM is consistent with FeFur repression in S. Enteritidis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28553268" target="_blank">28553268</a>]<br></li>
<li>Our data showed that Fur represses tcf expression ( Fig. 4b ) , whereas low iron conditions increased it .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26944792" target="_blank">26944792</a>]<br></li>
<li>We found that Fur represses SPI-2 expression that is activated inside macrophages and under acidic conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24643535" target="_blank">24643535</a>]<br></li>
<li>Thus , Fur is an activator of genes that are typically induced under anaerobic conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22017966" target="_blank">22017966</a>]<br></li>
<li>Of these , Fur is a reasonable candidate for contributing to PQ-induced sodA expression under aerobic growth .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10613858" target="_blank">10613858</a>]<br></li>
<li>The genes for these proteins are repressed by Fur when iron is abundant and are upregulated when iron is limiting ( 12 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24990908" target="_blank">24990908</a>]<br></li>
<li>Therefore , Salmonella feo expression is highly induced under low-oxygen conditions with low iron ( 8 ) , where Fur activity is reduced ( 10 ) and Fnr activity is enhanced ( 11 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25313398" target="_blank">25313398</a>]<br></li>
<li>The first gene of this operon , STM1130 , was also down-regulated 7.9-fold in Δfur ( Additional file 2 : Table S2 ) ; however , a Fur binding site was not identified upstream of STM1130 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22017966" target="_blank">22017966</a>]<br></li>
<li>Figure 2B shows that when hilD was deleted , pFur was unable to induce expression of hilA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
<li>Again , when hilD was deleted in these strains , Fur regulation of hilD was abrogated , while overproduction of HilC still induced hilD expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
<li>Fur regulates hilA-lacZ through nucleotides upstream of hns .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21075923" target="_blank">21075923</a>]<br></li>
<li>Deletion of either hilC or rtsA in this experiment decreased expression of hilA but did not block regulation by either HilE or Fur .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
<li>Fur regulates expression of hilA by working through HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
<li>Fur regulates expression of rtsA by controlling HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
<li>Fur regulates expression of hilA via HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
<li>RESULTS Fur induces expression of hilA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
<li>Fur activates expression of hilA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
<li>The Fur regulator represses expression of SPI-2 genes during environmental acidiﬁcation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24643535" target="_blank">24643535</a>]<br></li>
<li>The absence of fur resulted in increased expression ( i.e. , Fur acted as a repressor ) of 226 genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22017966" target="_blank">22017966</a>]<br></li>
<li>Taken together , these results illustrate that Fur repression of ssrB transcription leads to repression of SPI-2 genes under acidic conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24643535" target="_blank">24643535</a>]<br></li>
<li>Fur negatively controls SPI-2 expression via repression of ssrB transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24643535" target="_blank">24643535</a>]<br></li>
<li>FIG 5 Fur repression of ssrB expression leads to repression of SPI-2 expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24643535" target="_blank">24643535</a>]<br></li>
<li>( B ) Fur overexpression induces hilA expression when HilD is present .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
<li>Fur does not regulate hilA expression by controlling HilE .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
<li>This construct expresses Fur protein at a low level unless induced by DNA damage ( 56 ) and does not contain any chromosomal DNA upstream or downstream of fur .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8824613" target="_blank">8824613</a>]<br></li>
<li>In addition , Fur could repress the sitABCD operon with Co. .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659669" target="_blank">15659669</a>]<br></li>
<li>In conclusion , we provided , for the ﬁrst time , evidence that Fur represses hns expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21075923" target="_blank">21075923</a>]<br></li>
<li>As yet , the intermediate through which Fur represses hilD translation is unknown .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17208038" target="_blank">17208038</a>]<br></li>
<li>It is known that the RyhB-mediated control is negatively regulated itself by Fur ( Ferric Uptake Regulator ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23831078" target="_blank">23831078</a>]<br></li>
<li>Moreover , the S. typhimurium ﬂhD master operon , controlling ﬂagellar gene expression , is positively regulated by Fur through an iron-independent mechanism that requires further characterization .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11932449" target="_blank">11932449</a>]<br></li>
<li>As seen in Figure 2 , Fur also repressed nrdHIEF expression , increasing its transcription up to 5-fold compared to the wild-type strain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20593029" target="_blank">20593029</a>]<br></li>
<li>As a positive control we used the Fur promoter region since Fur has already been reported to autoregulate its own expression in E. coli [ 23 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20593029" target="_blank">20593029</a>]<br></li>
<li>NsrR-dependent expression of hmp was significantly greater at DEA/NO concentrations as low as 1 μM ( corresponds to approximately 1.5 μM of released NO · ) in comparison to genes regulated by NorR , Fur or SoxR ( Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22831173" target="_blank">22831173</a>]<br></li>
<li>Indeed , as pH was lowered from pH 7.0 to pH 5.5 , the lack of Fur enabled SPI-2 gene expression to be induced at higher pH and to be expressed at higher levels .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24643535" target="_blank">24643535</a>]<br></li>
<li>Fur represses the expression of iron-responsive genes in a form associated with iron when bacteria are grown under iron-replete conditions ( 11 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24643535" target="_blank">24643535</a>]<br></li>
<li>Fur is reported to repress the iron acquisition systems including the host-inducible , virulence-associated sitABCD iron acquisition genes contained within SPI1 ( Janakiraman and Slauch , 2000 ; Zhou et al. , 1999 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15790293" target="_blank">15790293</a>]<br></li>
<li>Under iron-replete conditions , RstA expression retarded Salmonella growth but enabled the Fur protein to repress the target genes beyond the levels which were simply accomplished by iron .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18790861" target="_blank">18790861</a>]<br></li>
<li>The Fur protein is a primary transcription factor that controls expression of genes for iron acquisition ( 11 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24643535" target="_blank">24643535</a>]<br></li>
<li>Fur is required for expression of hilA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21075923" target="_blank">21075923</a>]<br></li>
<li>This indicated that Fur is required for ftnB expression , independent of Fnr .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22017966" target="_blank">22017966</a>]<br></li>
<li>How Fur , which is thought to negatively regulate gene expression in presence of Fe2þ positively contributes to SPI1 gene expression is not well understood .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23040276" target="_blank">23040276</a>]<br></li>
<li>It is wort noting that while overproduction of Fur caused an eightfold increase in hilA expression in low-oxygen conditions , Fur overproduction induced hilA expression only threefold in highoxygen conditions ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
<li>In this work we have proven that Fur is indeed a transcriptional regulator of nrdHIEF in S. Typhimurium , normally repressing its expression and causing an upregulation under iron depleted growth ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20593029" target="_blank">20593029</a>]<br></li>
				<label class="TU_collapse_similarity" for="tu_similarity_79">
Fur negatively regulates hns and is required for the expression of HilA and virulence in Salmonella enterica serovar Typhimu-rium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24166960" target="_blank">24166960</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24643535" target="_blank">24643535</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26386070" target="_blank">26386070</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25547794" target="_blank">25547794</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24990908" target="_blank">24990908</a>]<br></label>
<input id="tu_similarity_79", type="checkbox">
<div class="TU_similar">
<li>Fur negatively regulates hns and is required for the expression of HilA and virulence in Salmonella enterica serovar Typhi-murium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23554419" target="_blank">23554419</a>]<br></li>
<li>Fur negatively regulates hns and is required for the expression of HilA and virulence in Salmonella enterica serovar Typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24735176" target="_blank">24735176</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26662614" target="_blank">26662614</a>]<br></li></div>
<li>The Fur repressor controls transcription of iron-activated and - repressed genes in Helicobacter pylori .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11932449" target="_blank">11932449</a>]<br></li>
<li>Fur protein binds Fe ( II ) and represses transcription of various target genes , among them entB ( 46 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12670952" target="_blank">12670952</a>]<br></li>
<li>Expression of hmpA is repressed by Fur in Salmonella , making it responsive The SOS response Free radicals are potent inducers of bacterial SOS responses in vitro ( Buchmeier et al. , 1993 ; Khil and Cam-erini-Otero , 2002 ; Motohashi and Saito , 2002 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12492857" target="_blank">12492857</a>]<br></li>
<li>The predicted Fur binding site is located within 385 and 357 nucleotides upstream of hns ( signiﬁed by a ﬁlled triangle in Fig. 5A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21075923" target="_blank">21075923</a>]<br></li>
<li>Ellermeier and Slauch initially reported that Fur activates hilA transcription in a HilD-dependent manner , and since Fur is thought to be capable of acting only as a repressor , it should repress a repressor that controls HilD ( 322 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23554419" target="_blank">23554419</a>]<br></li>
<li>Clearly , these results suggest Fnr is functional in Δfur and that Fur is regulating genes of anaerobic metabolism ( eut , frd , and dms operons ) through an unknown mechanism .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22017966" target="_blank">22017966</a>]<br></li>
<li>Expression of several genes controlled by Fnr and NsrR appeared to be also dependent on Fur .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22017966" target="_blank">22017966</a>]<br></li>
<li>Figure Figure 5 Fur and Fnr control transcription of hmpA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22017966" target="_blank">22017966</a>]<br></li>
<li>Therefore , Fur might indirectly repress tcf expression by repressing these sRNAs , as loss of fur or growth in low iron concomitantly increases expression of both sRNAs ( RyhB1 and RyhB2 ) and tcf .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26944792" target="_blank">26944792</a>]<br></li>
<li>S. Typhimurium was treated with varying concentrations of the NO · - releasing compound diethylamine NONOate ( DEA/NO ) , and hmp , norV , entC and soxS expression was determined by quantitative RT-PCR as a measure of NsrR , NorR , Fur and SoxR expression , respectively ( Experimental Procedures ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22831173" target="_blank">22831173</a>]<br></li>
<li>'' If Fur represses rfrA and rfrB and allows translation of sodB , loss of rfrA and perhaps rfrB should increase expression of sodB in the fur background .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
<li>However , when both rfrA and rfrB were deleted , expression of the sodB - lacZ translational fusion was restored and no longer affected by loss of Fur .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
<li>In summary , Fur represses expression of sRNAs RfrA and RfrB , a repression that is important for optimal growth of S. Typhi and resistance to H2O2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23306672" target="_blank">23306672</a>]<br></li>
<li>( A ) Fur controls expression of sodB , but not SPI1 , via RfrA and RfrB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
<li>The data can not be explained by a cascade of regulators in which Fur-Fe ( II ) represses a positive regulator of IRO-28 expression because deferrated Fur is required for IRO-28 expression ( Fig. 7C and E ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8824613" target="_blank">8824613</a>]<br></li>
<li>Increased expression of H-NS , a known repressor of ftnB , tdc operon , and other genes , in Δfur may account for their activation by Fur .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22017966" target="_blank">22017966</a>]<br></li>
<li>First , overproduction of Fur signiﬁcantly increases expression of hilA and hilD when HilD is present .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
<li>Fur regulation of sodB and Fur regulation of hilA are mechanistically different .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
<li>Global regulation of gene expression by HNS and Fur 4 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23040276" target="_blank">23040276</a>]<br></li>
<li>Under iron-rich conditions , it has been shown that expression of ryhB gene is repressed by Fe2 + - Fur , whereas transcription of genes encoding iron-storage proteins is elevated ( Masse et al. , 2005 ; Oglesby et al. , 2005 ; Vecerek et al. , 2007 ; Ellermeier and Slauch , 2008 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27242152" target="_blank">27242152</a>]<br></li>
<li>In addition , hmpA expression is upregulated under iron-limiting conditions through NsrR , but independently of Fur ( Bang et al. , 2006 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21833325" target="_blank">21833325</a>]<br></li>
<li>Likewise , feoB expression is under dual transcriptional control by the iron-sensing Fur and oxygen-sensing Fnr regulators whereby FeoB-mediated Fe ( II ) uptake is maintained at appropriate levels in response to different levels of iron and oxygen .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23708131" target="_blank">23708131</a>]<br></li>
<li>The expression of RyhB-1 and RyhB-2 is induced to downregulate target mRNAs implicated in iron homeostasis , including its cognate repressor Fur and dispensable iron-con-taining proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24239962" target="_blank">24239962</a>]<br></li>
<li>The sitABCD operon is induced under iron-deficient conditions in vitro and in the testinal tant is gesting is repressed by Fur .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10712695" target="_blank">10712695</a>]<br></li>
<li>The expression of ryhB , in turn , is repressed by Fur under high-iron conditions ( 38 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
<li>fepA codes for the enterobactin receptor , which is induced under low-iron conditions and repressed by Fur in conditions of high iron ( 53 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16237024" target="_blank">16237024</a>]<br></li>
<li>Conversely , the sRNAs are expressed under low iron conditions and their expression is repressed by Fur .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23306672" target="_blank">23306672</a>]<br></li>
<li>Indeed , it has been shown previously that transcription of sitABCD is repressed by Fur in response to iron ( 18 , 43 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659669" target="_blank">15659669</a>]<br></li>
<li>Regulation of SodA production is complex but includes repression by Fnr , ArcA , and Fur , such that expression is repressed under anaerobic conditions ( 16 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659669" target="_blank">15659669</a>]<br></li>
<li>The regulation of the manganese transporters by Fur is unclear , although it is possible that manganese import is not necessary under some circumstances of iron-sufficiency ( Papp-Wallace and Maguire , 2006 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21722794" target="_blank">21722794</a>]<br></li>
<li>Global regulation of gene expression by HNS and Fur Since SPI1 is a foreign fragment of DNA acquired by Salmonella during its evolution , it is understandable that it is subject to a heavy negative regulation [ 89 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23040276" target="_blank">23040276</a>]<br></li>
<li>Therefore , it is not surprising that Fur regulated , low-iron-induced genes such as sitABCD and sodA would answer an IVET selection in cultured hepatocytes and that these genes are also induced in the host .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10712695" target="_blank">10712695</a>]<br></li>
<li>This file contains genes that were differentially expressed in 14028s , Δfur , and the fnr , which contain a putative binding site for Fur and for Fnr .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22017966" target="_blank">22017966</a>]<br></li>
<li>In addition , a Fur superrepressor that constitutively repressed iron-regulated genes mediated normal Fur-dependent acid tolerance and pH-controlled gene expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8824613" target="_blank">8824613</a>]<br></li>
<li>In Escherichia coli , it has been reported that Crl also interacts with the ferric uptake regulator Fur and that Fur represses crl transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20008066" target="_blank">20008066</a>]<br></li>
<li>Crawford & Goldberg ( 1998a ) proposed that the ironresponsive regulator , Fur , represses hmp transcription and that this repression is lifted by NO on inactivation of Fur .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17526833" target="_blank">17526833</a>]<br></li>
<li>Masse and Gottesman ( 2002 ) also reported that expression of certain mRNAs ( acnA , fumA , ftnA , bfr , sodB , and sdhD ) in Escherichia coli that were shown to be positively regulated by the Fur regulator is repressed by RyhB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23831078" target="_blank">23831078</a>]<br></li>
<li>Fur also positively regulates other HilA-dependent genes ( invF and sipC ) involved in invasion , by repressing the expression of H-NS ( Troxell et al. , 2011 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21722794" target="_blank">21722794</a>]<br></li>
<li>Although these results have been retracted ( Crawford & Goldberg , 2006 ) , and others have suggested that Fur is not involved in Salmonella hmp expression ( Bang et al. , 2006 ) , other promoters including hmp are controlled by nitrosylation of the Fur iron ( D'Autreaux et al. , 2002 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17526833" target="_blank">17526833</a>]<br></li>
<li>The Fur regulon , those genes whose expression are under the control of Fur , contain genes that are involved in iron acquisition , which is essential for bacterial multiplication ( de Lorenzo et al. , 1987 ; Mekalanos , 1992 ; Wooldridge and Williams , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12652904" target="_blank">12652904</a>]<br></li>
<li>Neiderhoffer et al. ( 29 ) provide evidence that the superoxide dismutase genes sodA4 and sodB are under negative and positive control , respectively , by Fur .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1624426" target="_blank">1624426</a>]<br></li>
<li>It has been reported that expression of sodB is regulated at the transcriptional and post-transcrip-tional level by several factors including Fur , NsrR and the sRNAs ryhB and fnrS ( Massé & Gottesman , 2002 ; Durand & Storz , 2010 ; Niederhoffer et al. , 1990 ; Partridge et al. , 2009 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23918818" target="_blank">23918818</a>]<br></li>
<li>SPI-1 expression is controlled by additional regulators located outside the island such as the Fur and BarA/SirA activators ( Fortune et al. , 2006 ; Ellermeier and Slauch , 2008 ) and the HilE repressor ( Fahlen et al. , 2000 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24354910" target="_blank">24354910</a>]<br></li>
				<label class="TU_collapse_similarity" for="tu_similarity_125">
Dual repression by Fe - Fur and Mn2-MntR of the mntH gene , encoding an NRAMP-like Mn2 transporter in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12379679" target="_blank">12379679</a>]<br></label>
<input id="tu_similarity_125", type="checkbox">
<div class="TU_similar">
<li>Dual repression by Fe2 - Fur and Mn2-MntR of the mntH gene , encoding an NRAMP-like Mn2 transporter in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12029030" target="_blank">12029030</a>]<br></li>
<li>Dual repression by Fe2 - Fur and Mn2-2 MntR of the mntH gene , encoding an NRAMP-like Mn transporter in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659669" target="_blank">15659669</a>]<br></li></div>
<li>Moreover , Fur signiﬁcantly regulates translational hilD -- lacZ fusions in a manner that mirrors the effects on hilA expression , but it does not affect transcription of hilD as monitored by transcriptional lac fusions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17208038" target="_blank">17208038</a>]<br></li>
<li>The reduction in PrtsA and PhilD activation was consistent with earlier data , which showed that Fur regulated SPI-1 by controlling hilD expression when the hilD gene was present ( 35 , 37 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26386070" target="_blank">26386070</a>]<br></li>
<li>Figure 5A shows that hilD transcription was induced by both Fur and HilC in the hilD background .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
<li>The data in Fig. 4B show that the HA-tagged Fur was not detected at the control genes , rpoD or dinP , but was enriched 103-fold at hns .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21075923" target="_blank">21075923</a>]<br></li>
<li>Importantly , Fur enrichment to the upstream region of hns was metal dependent , noting that the data in Fig. 4B were normalized to 2,2 - dipyridyl-treated controls .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21075923" target="_blank">21075923</a>]<br></li>
<li>However , it was later revealed that the reporter fusion was to the Fur repressed iroC and not to the hmpA [ 116 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22017966" target="_blank">22017966</a>]<br></li>
<li>Moreover , under Fe-limiting conditions in which Fur would be anticipated to be inactive , levels of bfr and ftnA expression were comparable in wild type and fur mutant backgrounds .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17302823" target="_blank">17302823</a>]<br></li>
<li>Cells were grown to stationary phase in LB medium at 30 ° C and 37 ° C and also on LB agar at 30 ° C , a growth condition used to study the regulation of crl expression by Fur in E. coli W3110 ( 28 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20008066" target="_blank">20008066</a>]<br></li>
<li>Every hit was located on the genome [ 56 ] and listed with its particular target , location ( in a gene , upstream or/and downstream to gene ) and sequence of Fur box in Table S1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23040276" target="_blank">23040276</a>]<br></li>
<li>We examined isrE and ryhB transcription at stationary phase and upon exposure to hydrogen peroxide in the absence of Fur and found that while RyhB levels were further elevated in a fur mutant ( Figure 3C compare lanes 3 and 8 top and middle panels ) , no increase in IsrE RNA could be detected in fur-deﬁcient cells at both hydrogen peroxide treated and stationary phase cells ( Figure 3C compare lanes 2 and 7 top and middle panels ) , further conﬁrming that the expression control of these genes differs .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18267966" target="_blank">18267966</a>]<br></li>
<li>Several cellular targets known to be positively regulated by Fur displayed an increased expression in ST24CHX , including those involved in anaerobic metabolism , such as dimethyl sulfoxide reductase ( dmsA ) , which was up-regulated at both the RNA and protein levels , and those involved in glycerol/glycerolipid metabolism ( glpAB ) , which were up-regulated at the RNA level .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25136333" target="_blank">25136333</a>]<br></li>
<li>Decreased expression of cellular targets known to the negatively regulated by Fur , including those involved in glycolysis was also noted ; for example , phosphofructokinase ( pfkA ) , which was detected as down-regulated at the RNA and protein levels .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25136333" target="_blank">25136333</a>]<br></li>
<li>Likewise , Fe in the medium repressed transcription of both sit and mntH primarily via Fur , although MntR was also involved in this response .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659669" target="_blank">15659669</a>]<br></li>
<li>This finding also matched previous results in which iron pathway genes in S. typhimurium changed expression in a ground-based model of microgravity , and the Fur protein was shown to play a role in stress resistance alterations induced in the same model ( 7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17901201" target="_blank">17901201</a>]<br></li>
<li>In addition , Fur is also linked to other stresses where iron plays an important role such as the acid tolerance response ( ATR ) by assisting in SodB expression ( Dubrac & Touati , 2000 ) , repression of the iron transport system SitABCD encoded on SPI-1 ( Janakiraman & Slauch , 2000 ) and in nitrosative stress by repressing hmpA ( Mukhopadhyay , Zheng , Bedzyk , LaRossa , & Storz , 2004 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23651595" target="_blank">23651595</a>]<br></li>
<li>Other high-scoring Fur boxes were located in the promoters of sRNA genes located adjacent to iron-associated or Fur-regulated operons .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27564394" target="_blank">27564394</a>]<br></li>
<li>Interestingly , the Fur regulon was deﬁned in S. Typhimurium using bacteria grown under anaerobic conditions , and this analysis identiﬁed regulated genes not known to be controlled by Fur when bacteria were grown in the presence of oxygen ( 35 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25795672" target="_blank">25795672</a>]<br></li>
<li>The OMP profile showed that the iron-uptake OMPs , repressed by Fur , were down regulated in presence of arabinose and up regulated in absence of arabinose ( data not shown ; see reference [ 29 ] ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20585446" target="_blank">20585446</a>]<br></li>
<li>As we have shown , sitABCD is controlled by oxygen level and growth phase via the Fur regulatory protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659669" target="_blank">15659669</a>]<br></li>
<li>For example , we have previously shown that Fur represses the sitABCD operon in the lumen and that repression is relieved once invasion occurs ( 26 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
<li>Because the Fur protein which binds the fur box is also involved in acid resistance [ 43 ] , we extended our experiments and found that sdiA is induced in S. typhimurium when the bacterial culture is inoculated in LB at pH 4 under fully aerobic conditions [ 172 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16023758" target="_blank">16023758</a>]<br></li>
<li>Transcription of the cytochrome-o ubiquinol oxidase operon ( cyoABCDE ) and the high affinity cytochrome-d terminal oxidase genes ( cydAB ) was repressed by Fur ( Additional file 2 : Table S2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22017966" target="_blank">22017966</a>]<br></li>
<li>In E. coli , YncE is secreted into the periplasm via the Sec-dependent pathway [ 37 ] , and its expression is induced under iron-restricted conditions when repression by the Fur protein is relieved [ 38 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23936575" target="_blank">23936575</a>]<br></li>
<li>Expression from PiroBCDE is under the control of the iron-responsive transcription factor Fur and induced by iron-deﬁciency [ 12 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19049822" target="_blank">19049822</a>]<br></li>
<li>Moreover , the E. coli ferritin is indirectly induced by Fur under iron-rich conditions , as Fur represses the small RNA RhyB , which directly represses ferritin transcription in the absence of iron ( Abdul-Tehrani et al. 1999 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26928316" target="_blank">26928316</a>]<br></li>
<li>In Salmonella Typhimurium , the expression of both RyhB-1 and RyhB-2 is negatively regulated by the Fur repressor , while stationary phase is the primary signal inducing RyhB-2 expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22824499" target="_blank">22824499</a>]<br></li>
<li>On the contrary , RyhB-2 expression is also slightly induced by activated Fur , but its primary inducing condition is the stationary growth phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22824499" target="_blank">22824499</a>]<br></li>
<li>Thus , under Fur active conditions ( left panel ) , hns is repressed by Fur thereby blocking H-NS repression of ftnB and the tdc operon ( signified by the circle with an `` X '' ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22017966" target="_blank">22017966</a>]<br></li>
<li>Transcription of the feoAB operon is negatively regulated by the Fur protein ( 12 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18790861" target="_blank">18790861</a>]<br></li>
<li>Expression of the Pasteurella multocida ompH gene is negatively regulated by the Fur protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11932449" target="_blank">11932449</a>]<br></li>
<li>Although Yoo et al. ( 2007 ) reported that dps expression is induced during the stationary phase by the Fur regulator , our results demonstrated that this gene is also regulated by RcsB but during the exponential phase of growth .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25028458" target="_blank">25028458</a>]<br></li>
<li>For instance , the H-NS modulator YmoA is preferentially degraded at 37 ◦ C by the ClpP and Lon proteases in Yersinia ( Jackson et al. , 2004 ) , and in Salmonella it has been shown that transcription of the hns gene -- which inﬂuences expression of the Spi-1-encoded invasion genes through the regulator HilD -- is repressed by the iron homeostasis regulator Fur ( Troxell et al. , 2011 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26441883" target="_blank">26441883</a>]<br></li>
<li>Collectively , we concluded from the above set of experiments that Lrp is an activator of the klf genes , and that the ipf operon is repressed by Fur and positively regulated by OmpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28817673" target="_blank">28817673</a>]<br></li>
<li>We then examined the effects of deleting fur , overproducing Fur , and overproducing HilC , which is known to independently activate hilD transcription ( 16 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
<li>In E. coli , Fur has been shown to positively affect the expression of few genes ( e.g. , sodB ) via its negative regulation of a small RNA , RyhB ( 39 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21075923" target="_blank">21075923</a>]<br></li>
<li>The fur gene , an important regulator of the Fur operon revealed a signiﬁcant increase in transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18656407" target="_blank">18656407</a>]<br></li>
<li>The sitABCD operon is Fur regulated Both the homology with known iron transport proteins and the identification of sitABCD in an IVET selection for genes induced under low-iron conditions led us to examine the regulation of this fusion in vitro under ironstarvation conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10712695" target="_blank">10712695</a>]<br></li>
<li>While under Fur inactive conditions ( right panel ) , the overexpression of H-NS results in the repression of ftnB and the tdc operon under anaerobic conditions ΔfurΔfnr backgrounds .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22017966" target="_blank">22017966</a>]<br></li>
<li>When Salmonella is grown under conditions of low levels of oxygen and iron ( Fig. 6A ) , Fur does not repress the feo promoter ( 2 , 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25313398" target="_blank">25313398</a>]<br></li>
<li>In high iron conditions , Fur is known to repress the transcription of genes encoding iron uptake systems and siderophores .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23306672" target="_blank">23306672</a>]<br></li>
<li>Fur activates sodB by repressing the expression of ryhB under high-iron conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
<li>Fur repressed ssrB expression both inside macrophages and under acidic conditions , which we ascribe to the direct binding of Fur to the ssrB promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24643535" target="_blank">24643535</a>]<br></li>
<li>Fur represses expression of the ssrB gene encoding the SsrB response regulator under acidic conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24643535" target="_blank">24643535</a>]<br></li>
<li>FIG 3 The Fur regulator represses expression of the ssrB gene inside macro-phages and under acidic conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24643535" target="_blank">24643535</a>]<br></li>
<li>In addition , we determined that under anaerobic conditions the expression of hilA was dependent on Fur , and we demonstrated that PhoP was not involved in this regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21075923" target="_blank">21075923</a>]<br></li>
<li>Promoter fusion experiments indicated that acnB transcription was ~ 10-fold greater than that of acnA and that acnA expression was regulated by the cyclic-AMP receptor protein ( CRP , glucose starvation ) , the fumarate nitrate reduction regulator ( FNR , oxygen starvation ) , the ferric uptake regulator ( Fur , iron starvation ) and the superoxide response protein ( SoxR , oxidative stress ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23637460" target="_blank">23637460</a>]<br></li>
<li>The decrease in the transcript levels of the hemA gene was also detected by S1 analysis ( Fig. 1C , ﬁrst and second lanes ) , and we found that this decrease was independent of both Fur and OxyR ( Fig. 8A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12081946" target="_blank">12081946</a>]<br></li>
<li>We also found that the anaerobic expression of ftnB ( encoding a ferri-tin-like protein ) and hmpA ( encoding the NO · detoxifying flavohemoglobin ) was dependent on both Fur and Fnr .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22017966" target="_blank">22017966</a>]<br></li>
<li>It has also been reported that the Escherichia coli Fur protein is a positive regulator of sodB gene expression , although the precise mechanism ... .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11932449" target="_blank">11932449</a>]<br></li>
<li>It was hypothesized that Fur repressed the SPI-2 expression that was activated during acidiﬁcation of the phagosome .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24643535" target="_blank">24643535</a>]<br></li>
<li>It was reported that the Fur regulator positively controls expression of the SPI-1 genes ( 15 , 16 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24643535" target="_blank">24643535</a>]<br></li>
<li>Fur also contributes to SPI-1 TTSS production by reducing the levels of H-NS that represses SPI-1 expression ( 16 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24643535" target="_blank">24643535</a>]<br></li>
<li>We further reasoned that if Fur controls SPI-2 genes via the repression of ssrB transcription , Fur repression of SPI-2 expression would be impaired when the ssrB gene is ectopically expressed from a heterologous promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24643535" target="_blank">24643535</a>]<br></li>
<li>Taken together , these results illustrate that the iron-sensing Fur regulator negatively controls expression of SPI-2 genes during acidiﬁcation and a lack of this regulation enables the SPI-2 genes to be induced , even under intermediate acidic conditions that would not normally allow for SPI-2 induction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24643535" target="_blank">24643535</a>]<br></li>
<li>Given that SsrB functions as an activator for expression of SPI-2 genes ( 4 ) and that Fur represses ssrB expression ( Fig. 3 ) , it is possible that the Fur regulator controls SPI-2 genes in a fashion dependent on the SsrB regulator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24643535" target="_blank">24643535</a>]<br></li>
<li>Indeed , the activation of hilA by Fur was dependent on 28 nucleotides located upstream of hns .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21075923" target="_blank">21075923</a>]<br></li>
<li>In conclusion , Fur activates hilA by repressing the expression of hns .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21075923" target="_blank">21075923</a>]<br></li>
<li>Expression of each sRNA was also induced when fur was deleted , confirming the role of Fur as a repressor for both of these sRNAs .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23306672" target="_blank">23306672</a>]<br></li>
<li>FIG 4 Growth at 42 ° C repressed expression of SPI-1 genes independently of the activator Fur .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26386070" target="_blank">26386070</a>]<br></li>
<li>Under high-iron conditions , signiﬁcant basal expression of bfd ( data not shown ) was observed , suggesting incomplete repression of the operon by Fur and , subsequently , expression of bfr .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12897017" target="_blank">12897017</a>]<br></li>
<li>The requirement of Fur for the activation of hilA was ap parent as the rate of synthesis of hilA-lacZ was signiﬁcantly reduced in fur compared to 14028s ( P 0.001 , Fig. 6 ) , and lack of Fur resulted in decreased expression of HilA-activated genes ( i.e. , invF and sipC ) ( see Fig .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21075923" target="_blank">21075923</a>]<br></li>
<li>Also , while deletion of hilC or rtsA decreased the expression of hilA and affected the fold effect of fur , it did not prevent Fur regulation of hilA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
<li>Finally , we demonstrated that in the absence of H-NS the expression of hilA was independent of Fur .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21075923" target="_blank">21075923</a>]<br></li>
				<label class="TU_collapse_similarity" for="tu_similarity_191">
Induction of the ferritin gene ( ftnA ) of Escherichia coli by Fe2þ-Fur is mediated by reversal of H-NS silencing and is RyhB independent .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21722794" target="_blank">21722794</a>]<br></label>
<input id="tu_similarity_191", type="checkbox">
<div class="TU_similar">
<li>Induction of the ferritin gene ( ftnA ) of Escherichia coli by Fe2 - Fur is mediated by reversal of H-NS silencing and is RyhB independent .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21075923" target="_blank">21075923</a>]<br></li></div>
<li>In E. coli , expression of sodB is activated by Fur via repression of the small RNA RyhB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
<li>Our study of the anaerobic expression of hmpA suggests that it is regulated by Fur , independent of Fnr .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22017966" target="_blank">22017966</a>]<br></li>
<li>NO inhibits transcription of SPI2 effectors encoded in Induction of hmp by NO The expression of the NO-detoxifying Salmonella flavohemoglobin Hmp is positively induced by the transcriptional regulator Fur in response to nitrosative stress ( 28 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16129704" target="_blank">16129704</a>]<br></li>
<li>( b ) E. coli RpoS CRP SoxR FNR Fur acnA P2 P1 S. Typhimurium Expression of S. Typhimurium acnA is regulated by CRP , FNR , Fur and SoxR Enhanced expression of the acnA gene in the acnB mutant suggested that PacnA was regulated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23637460" target="_blank">23637460</a>]<br></li>
<li>In the case of the iron-storage protein ferritin , FtnA , gene expression is induced by direct interaction of Fe ( II ) - Fur with an extended Fur binding site ( containing five tandem Fur boxes ) located upstream of the ftnA promoter ( Nandal et al. , 2010 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21722794" target="_blank">21722794</a>]<br></li>
<li>Teixido et al. ( 2010 ) showed evidence that transcription of narL and narP , encoding response regulators of the two-component system that controls the use of nitrate , was regulated by the Fur protein regardless of oxygen conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27242152" target="_blank">27242152</a>]<br></li>
<li>These data showed that the transcription of iroB is regulated by iron and by Fur , and that each of the sRNAs had a distinct influence of the expression of iroB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23306672" target="_blank">23306672</a>]<br></li>
<li>Fur activation of SPI1 is not mediated through the Fur-regulated small RNAs RfrA and RfrB , which are the Salmonella ortholog and paralog of RyhB that control expression of sodB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
<li>While OxyR is not present to induce the expression of both sRNAs , Fur is being disabled by sub-millimolar levels of peroxide , releasing the repression of ryhB-1 and ryhB-2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24239962" target="_blank">24239962</a>]<br></li>
<li>Although the Fur-box is located proximally downstream of 1 , it is unlikely to regulate the Mn2 - dependent transcription of mntH because Fur responds to Fe2 but not Mn2 ( 18 , 19 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24596096" target="_blank">24596096</a>]<br></li>
<li>Although Fur repression of the iroBCDE loci is known [ 59 ] , iroN is encoded downstream of this operon and is transcribed in the opposite orientation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22017966" target="_blank">22017966</a>]<br></li>
<li>Collectively , full expression of ftnB is dependent on Fur in aerobic and anaerobic conditions , whereas Fnr is a strong activator in the absence of O2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22017966" target="_blank">22017966</a>]<br></li>
<li>Fur directly binds to the hilD promoter and enhances its own expression ( 15 ) , and HilD functions as an activator of SPI-1 expression ( 17 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24643535" target="_blank">24643535</a>]<br></li>
<li>Fur has been recently shown to induce the expression of hilA , and SPI-1 in general , throug post-translational mechanisms acting on HilD [ 26 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18656407" target="_blank">18656407</a>]<br></li>
<li>The absence of IRO-28 in the fur-1 mutant ( Fig. 7C ) and its presence in the furH90R mutant ( Fig. 7E ) conﬁrm that Fur acts as a positive regulator of this gene and that FurH90R is iron blind yet active .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8824613" target="_blank">8824613</a>]<br></li>
<li>These results are summarized in the diagram depicted in Fig. 7 , where the presence of active Fur ( i.e. , Fur-Fe2 ) results in decrease in [ H-NS ] and increased the expression of hilA ( Fig. 7A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21075923" target="_blank">21075923</a>]<br></li>
<li>Conclusions : This study determined that Fur is a positive factor in ftnB regulation , while serving to repress the expression of hmpA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22017966" target="_blank">22017966</a>]<br></li>
<li>Expression of these genes is downregulated in Salmonella growing inside macrophages ( 35 ) , to which Fur could contribute , as evidenced by the ﬁnding that Fur represses expression of the entC gene inside macrophages ( Fig. 1C ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24643535" target="_blank">24643535</a>]<br></li>
<li>Because one of Fur 's target promoters is that corresponding to the hns gene ( 53 ) , and H-NS is known to silence expression of SPI-2 genes ( 22 ) , the increase in iron should increase expression of SPI-2 genes ; however , we observed decreased expression ( Fig. 3A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23690578" target="_blank">23690578</a>]<br></li>
<li>Furthermore , we have shown Fur binding to both this sequence and to its own promoter , indicating a direct role of the Fur protein in the regulation of class Ib nrd genes as well as in the autoregulation of fur expression similarly to what has previously been described in E. coli [ 23 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20593029" target="_blank">20593029</a>]<br></li>
<li>In its most common mode of regulation , in the presence of Fe2þ , Fur binds to the Fe2þ ion , and represses expression of its target genes [ 20,25 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23040276" target="_blank">23040276</a>]<br></li>
<li>The expression of hns was signiﬁcantly upregulated in the fur mutant , and a putative Fur binding site was identiﬁed upstream of the gene ( 64 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21075923" target="_blank">21075923</a>]<br></li>
<li>Time course analysis of SPI-2 expression after the entry of Salmonella into macrophages revealed that SPI-2 genes are induced earlier and at higher levels in the absence of the Fur regulator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24643535" target="_blank">24643535</a>]<br></li>
<li>Inspection of our Fur microarray data revealed that the expression of genes within SPI-1 ( prgK , prgH , sipADCB , sicA , and invJICBA ) was signiﬁcantly ( 1.4 - to 2-fold ) reduced in the fur mutant ( 64 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21075923" target="_blank">21075923</a>]<br></li>
<li>The lumen of the stomach is acidic , and low pH induces the expression of RpoS and Fur , transcription factors important for acid resistance ( Foster , 1991 ; Fang et al. , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12675799" target="_blank">12675799</a>]<br></li>
<li>They act by increasing the level of HilD protein to overcome HilE , and controlling HilD activity , or in the case of Fur , lowering the threshold required at the promoter , such that HilD activates its own promoter as well as induces expression of HilC and RtsA , which then act to amplify and accelerate SPI1 expression ( Saini et al. 2010a ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22021388" target="_blank">22021388</a>]<br></li>
<li>The differential expression of stn and invA was observed among TSB-5 , TSB-6 , and TSB-7 , suggesting that the alternative sigma factor ( s ) , two-component regulatory system ( PhoPQ ) , and iron regulatory protein ( Fur ) can regulate not only the ATR but also the expression of virulence genes in S. Typhimurium cells ( 20 , 24 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20971873" target="_blank">20971873</a>]<br></li>
<li>The expression data indicated that Fur and Fnr cooperate in the regulation of these 12 genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22017966" target="_blank">22017966</a>]<br></li>
<li>Thus , we showed that Fur negatively regulates the expression of hns and that the positive effect of Fur on the expression of hilA is dependent on an upstream sequence that contained the putative Fur-binding site in hns .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21075923" target="_blank">21075923</a>]<br></li>
<li>First , the microarray data showed that Fur negatively regulates the expression of hns and has a predicted Fur binding site ( Table 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22017966" target="_blank">22017966</a>]<br></li>
<li>Taken together , these data suggest that the presence of metals in the environment induces expression of hilA and that Fur plays a major role in this metal regulation of hilA and therefore SPI1 expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
<li>( A ) Conditions that activate Fur ( i.e. , the presence of iron ) repress the expression of hns ( solid arrow ) , and allow the expression of hilA ( blocked dotted arrows ) and increased HilA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21075923" target="_blank">21075923</a>]<br></li>
<li>This activation is , in part , indirect where Fur represses the expression of hns , which represses the expression of hilA and hilD [ 29 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22017966" target="_blank">22017966</a>]<br></li>
<li>It has been shown that the sitABCD operon , encoding a manganese transport system repressed by Fur in the presence of high iron concentrations , is not expressed in the lumen of the small intestine of the mouse but is induced upon invasion ( 25 , 26 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
<li>While the role of Fur has been studied in controlling the gene expression of SPI1 and SPI2 , the mechanistic details of the control is not understood .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23040276" target="_blank">23040276</a>]<br></li>
<li>Role of iron , Fur and the sRNAs RyhB1/RyhB2 in regulation of tcf expression As the Fur regulator is activated by iron and the regulation of its target genes is usually iron-dependent , the regulation of tcf by iron levels ( high and low ) was investigated in the WT and fur mutant strains .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26944792" target="_blank">26944792</a>]<br></li>
<li>Iron acquisition genes encoded by the sitABCD operon and the iroA locus were differentially induced in response to spermine NONOate ( table S2 ) , likely reflecting their derepression upon nitrosylation of the iron prosthetic group of Fur [ 38 -- 40 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18350168" target="_blank">18350168</a>]<br></li>
<li>In the presence of high iron concentrations , the activated Fur represses transcription of genes whose promoter contains the consensus FUR box sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22824499" target="_blank">22824499</a>]<br></li>
<li>The yncE gene is induced under iron restriction through the action of the global iron regulator Fur in E. coli ; however , the regulator and the iron restriction did not affect the transcription of the yncD gene ( McHugh et al. , 2003 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22150228" target="_blank">22150228</a>]<br></li>
<li>This emphasizes that , in the presence of iron , the feoB gene should be induced by the RstA protein to repress iron-responsive genes and differs from the ﬁnding that iron chelator inactivated the Fur protein to abolish the regulatory effect of RstA on its regulated genes ( Fig. 1B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18790861" target="_blank">18790861</a>]<br></li>
<li>Our study shows that Fur plays a signiﬁcant role in the expression of the SPI1 T3SS by genetically activating hilA transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
<li>In contrast , entB expression in the yggX gshA strain was decreased by deferoxamine , indicating the presence of more iron-bound Fur .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18835989" target="_blank">18835989</a>]<br></li>
<li>Each of the mutant strains showed signiﬁcant derepression of the entB gene , indicating that less iron-bound Fur was present than in the wildtype strain under the same conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18835989" target="_blank">18835989</a>]<br></li>
<li>We also determined that the Fur protein was acting as a transcriptional repressor of the feoAB operon because deletion of the fur gene resulted in a twofold increase of the wild-type feoAB transcripts ( Fig. 2A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18790861" target="_blank">18790861</a>]<br></li>
<li>Real-time PCR analysis revealed that upon RstA expression , transcriptional repression of the fhuF gene by the FLAG-tagged Fur protein was as efﬁcient as in the strain with the wild-type Fur protein ( see Fig .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18790861" target="_blank">18790861</a>]<br></li>
<li>Deregulation of SPI-2 expression displayed by the fur mutant was due to the function of Fur , as evidenced by the fact that heterologous expression of the fur gene from the plasmid-linked lac promoter complemented ssaG mRNA expression levels in the fur mutant strain similar to wildtype levels under pH 6.0 conditions ( Fig. 2B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24643535" target="_blank">24643535</a>]<br></li>
<li>However , the ssaG mRNA levels displayed by the ssrB mutant were unaffected by fur deletion under the same growth condition , which was in contrast to the Fur repression of ssaG expression occurring in the presence of ssrB ( Fig. 5A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24643535" target="_blank">24643535</a>]<br></li>
<li>Genes whose expression are under the control of this regulatory protein form the Fur regulon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7642488" target="_blank">7642488</a>]<br></li>
<li>Moreover , these results also demonstrate that the S. typhimurium Fur protein positively regulates the expression of the flhD master operon governing the ﬂagellar regulon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11932449" target="_blank">11932449</a>]<br></li>
<li>Moreover , expression of Fur from a plasmid that was introduced into the fur background , resulted in significantly reduced IpfD-2HA expression relative to its expression in the fur strain ( Fig 7H ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28817673" target="_blank">28817673</a>]<br></li>
<li>In contrast , iron repression of the ssaG gene did not occur in the fur deletion mutant ( Fig. 2C ) , suggesting that Fur represses SPI-2 expression in an iron-dependent fashion .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24643535" target="_blank">24643535</a>]<br></li>
<li>Fur ( H90R ) production from the plasmid-linked lac promoter could no longer complement ssaG expression in the fur deletion strain under pH 6.0 conditions ( Fig. 2B ) , further indicating that Fur repression of SPI-2 expression under acidic conditions is dependent on iron .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24643535" target="_blank">24643535</a>]<br></li>
<li>Whereas the E. coli aerobactin operon was repressed by Fur using either Fe or Mn as a corepressor , sodA was repressed by Fur using Fe but not Mn .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15659669" target="_blank">15659669</a>]<br></li>
<li>Thus , under this 2.5-fold constraint , the microarray data did not identify hilA as differentially regulated by Fur .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21075923" target="_blank">21075923</a>]<br></li>
<li>Since most mRNAs targeted by RyhB are repressed at the translation level , these target genes are positively regulated by the activated Fur repressor through RyhB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22824499" target="_blank">22824499</a>]<br></li>
<li>Another gene regulated by Fnr or Fur was the NO · detoxifying flavohemoglobin protein encoded by the hmpA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22017966" target="_blank">22017966</a>]<br></li>
<li>Not surprisingly , the Fur protein repressed feoB transcription even at acidic pH when iron was plentiful ( see Fig .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19801407" target="_blank">19801407</a>]<br></li>
<li>We determined that Fur represses ssrB transcription ( Fig. 3 ) by directly binding to the ssrB promoter ( Fig. 4B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24643535" target="_blank">24643535</a>]<br></li>
				</div></ul></div><br>
	</div>
</body>
</html>