<!doctype html>
<style>
@charset "utf-8";
body {
	height: 100%;
	margin: 0;
	padding: 0;
}

.banner {
	position: fixed;
	top: 0;
	width: 100%;
	height: 65px;
	background: gray;
	line-height: 65px;
}
.spacer {
	height: 10px;
}

.main-content {
	padding: 5%;
}

.collapse_ACT {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-style: none solid none none;
	border-color: #B4045F;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_ACT + input{
	display: none;
}
.collapse_ACT + input + div{
	display: none;
}
.collapse_ACT + input:checked + div{
	display: block;
}
.ACT {
	text-align: justify;
	text-justify: inter-word;
}
.ACT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B4045F;
}
.ACT_collapse_similarity:hover{
	background: #e5d3dc;
}
.ACT_collapse_similarity + input {
	display: none;
}
.ACT_collapse_similarity + input + div{
	display: none;
}

.ACT_collapse_similarity + input:checked + div{
	display: block;
}
.ACT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.ACT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B4045F;
	padding-left: 5px;
}
.ACT_collapse_similarity:hover{
	background: #e5d3dc;
}
.ACT_collapse_similarity + input {
	display: none;
}
.ACT_collapse_similarity + input + div{
	display: none;
}
.ACT_collapse_similarity + input:checked + div{
	display: block;
}
.ACT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.ACT_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #B4045F;
}
.ACT_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_DOM {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-style: none solid none none;
	border-color: #B404AE;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_DOM + input{
	display: none;
}
.collapse_DOM + input + div{
	display: none;
}
.collapse_DOM + input:checked + div{
	display: block;
}
.DOM {
	text-align: justify;
	text-justify: inter-word;
}
.DOM_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B404AE;
	padding-left: 5px;
}
.DOM_collapse_similarity:hover{
	background: #e8d7e7;
}
.DOM_collapse_similarity + input {
	display: none;
}
.DOM_collapse_similarity + input + div{
	display: none;
}
.DOM_collapse_similarity + input:checked + div{
	display: block;
}
.DOM_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}.DOM_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #B404AE;
}
.DOM_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_EVO {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #7401DF;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_EVO + input{
	display: none;
}
.collapse_EVO + input + div{
	display: none;
}
.collapse_EVO + input:checked + div{
	display: block;
}
.EVO {
	text-align: justify;
	text-justify: inter-word;
}
.EVO_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #7401DF;
	padding-left: 5px;
}
.EVO_collapse_similarity:hover{
	background: #e2d7ed;
}
.EVO_collapse_similarity + input {
	display: none;
}
.EVO_collapse_similarity + input + div{
	display: none;
}
.EVO_collapse_similarity + input:checked + div{
	display: block;
}
.EVO_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.EVO_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #7401DF;
}
.EVO_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_RP {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #0174DF;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_RP + input{
	display: none;
}
.collapse_RP + input + div{
	display: none;
}
.collapse_RP + input:checked + div{
	display: block;
}
.RP {
	text-align: justify;
	text-justify: inter-word;
}
.RP_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #0174DF;
	padding-left: 5px;
}
.RP_collapse_similarity:hover{
	background: #c9d7e2;
}
.RP_collapse_similarity + input {
	display: none;
}
.RP_collapse_similarity + input + div{
	display: none;
}
.RP_collapse_similarity + input:checked + div{
	display: block;
}
.RP_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.RP_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #0174DF;
}
.RP_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_SIT {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #0489B1;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_SIT + input{
	display: none;
}
.collapse_SIT + input + div{
	display: none;
}
.collapse_SIT + input:checked + div{
	display: block;
}
.SIT {
	text-align: justify;
	text-justify: inter-word;
}
.SIT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #0489B1;
	padding-left: 5px;
}
.SIT_collapse_similarity:hover{
	background: #d0dfe2;
}
.SIT_collapse_similarity + input {
	display: none;
}
.SIT_collapse_similarity + input + div{
	display: none;
}
.SIT_collapse_similarity + input:checked + div{
	display: block;
}
.SIT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.SIT_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #0489B1;
}
.SIT_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_TU {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #04B486;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_TU + input{
	display: none;
}
.collapse_TU + input + div{
	display: none;
}
.collapse_TU + input:checked + div{
	display: block;
}
.TU {
	text-align: justify;
	text-justify: inter-word;
}
.TU_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #04B486;
	padding-left: 5px;
}
.TU_collapse_similarity:hover{
	background: #d0e2de;
}
.TU_collapse_similarity + input {
	display: none;
}
.TU_collapse_similarity + input + div{
	display: none;
}
.TU_collapse_similarity + input:checked + div{
	display: block;
}
.TU_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.TU_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #04B486;
}
.TU_similar > a {
	text-decoration: none;
	color: gray;
}

ul { 
	list-style-type: none;
	padding: 0;
	margin: 0;
}
ul li > a {
	text-decoration: none;
	color: gray;
}
ul li:hover, ul li:focus {
	background: #F0F0F0;
}

ul li:first-child span {
  border-left:none;  
}

</style>
<html>
<head>
<meta charset="utf-8">
<title>TF properties - GlrR</title>
<meta name="theme-color" content="#ffffff">
</head>
<body>
	<div class="banner">
		<center><font color="white" size="16px" face="arial">GlrR</font></center>
	</div>
	<div align="center" class="main-content">
		<div class="spacer">
	</div><br>			<label class="collapse_ACT" for="_1"><font size="5px">GlrR transcriptional activity</font></label>
				<input id="_1" type="checkbox">
				<div class="ACT"><br><ul><li>We wanted to discriminate , whether phosphorylated or dephosphorylated GlrR is active .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20965974" target="_blank">20965974</a>]<br></li>
<li>The following strains and transformants were tested ( corresponding to the columns from left to right ) : Z197 , Z206 , Z206 + pBGG223 , Z206 + pYG6 , Z227 , Z388 , Z389 , Z389 + pBGG223 , Z389 + pYG6 , Z446 , Z362 , Z363 , Z363 + pBGG223 , Z363 + pYG6 and Z444 EMSAs using puriﬁed Y. pseudotuberculosis GlrR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20965974" target="_blank">20965974</a>]<br></li>
<li>Finally , binding of GlrR was completely prevented , when all three ABS were truncated or simultaneously mutated ( Figure 4B , Supplementary Figure S8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20965974" target="_blank">20965974</a>]<br></li>
<li>The following strains were employed ( corresponding to the columns from left to right ) : Z197 , Z206 , Z370 , Z372 , Z190 , Z196 , Z371 and Z373 Phosphorylated GlrR is active and stimulates sRNA expression GlrR contains a response-regulatory domain including the conserved putative phosphorylation site aspartate 56 at its N-terminus .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20965974" target="_blank">20965974</a>]<br></li>
<li>To obtain in vivo evidence that phosphorylated rather than dephosphorylated GlrR is active , we replaced the phosphorylation site Asp 56 in GlrR with an alanine and a glutamate residue , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20965974" target="_blank">20965974</a>]<br></li>
<li>In addition , pre-incubation of GlrR with acetyl phosphate increased its binding afﬁnity for the glmY promoter ( Figure 7A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20965974" target="_blank">20965974</a>]<br></li>
<li>To test the possible effect of phosphorylation on GlrR activity , the protein was pre-incubated at 37 C for 1 h in the absence ( left panel ) or presence ( right panel ) of 50 mM acetyl phosphate before continuing with the EMSA protocol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20965974" target="_blank">20965974</a>]<br></li>
<li>Since ongoing incubation of GlrR at 37 C resulted in increasing inactivation of the protein ( compare left panels in Figures 2A and 7A ) , a control experiment was performed in which GlrR was treated the same way but acetyl phosphate was omitted .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20965974" target="_blank">20965974</a>]<br></li>
<li>These experiments revealed that binding afﬁnity of GlrR was somewhat increased by the acetyl phosphate treatment relative to the control ( compare panels in Figure 7A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20965974" target="_blank">20965974</a>]<br></li>
<li>Most importantly , our results suggest that in most species expression of both sRNAs is controlled by s54 and the response-regulator GlrR ( Figure 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20965974" target="_blank">20965974</a>]<br></li>
<li>Response regulator GlrR binds to the glmY promoter regions of S. typhimurium and Y. pseudotuberculosis First , we wanted to verify if the putative s54-dependent glmY promoters of S. typhimurium and Y. pseudotuberculosis are controlled by the response-regulator GlrR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20965974" target="_blank">20965974</a>]<br></li>
<li>Signal Sensing through Two-Component Systems S. Typhimurium has three bEBPs ( NtrC , ZraR , and GlrR ) that are response-regulators of TCSs , in which a sensor kinase protein recognizes the cellular signal , autophosphorylates , and transfers the phosphate to a conserved aspartate residue of the response-regulator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27583250" target="_blank">27583250</a>]<br></li>
<li>GlrR binds three TGTCN GACA motifs located more than 10 100 bp upstream of glmY and thereby activates the s54-promoter , while activity of the s70-promoter is unaffected .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20965974" target="_blank">20965974</a>]<br></li>
<li>However , GlrR / GlrK strongly up-regulate glmY expression through activation of the s54-promoter , when cells enter the stationary growth-phase ( 22 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20965974" target="_blank">20965974</a>]<br></li>
<li>Phosphorylation increases activity of response-regulator GlrR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20965974" target="_blank">20965974</a>]<br></li>
<li>Transcription of Y. pseudotuberculosis glmZ depends on binding of GlrR to its three target sites upstream of the promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20965974" target="_blank">20965974</a>]<br></li>
<li>Histidine kinase GlrK phosphorylates response-regulator GlrR , which stimulates binding of GlrR to its target sites on the DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20965974" target="_blank">20965974</a>]<br></li>
<li>Finally , evidence is provided that phosphorylation of GlrR increases its activity and thereby sRNA expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20965974" target="_blank">20965974</a>]<br></li>
<li>Finally , our data indicate that phosphorylation of GlrR increases its afﬁnity for its target sites on the DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20965974" target="_blank">20965974</a>]<br></li>
<li>Taken together , the data indicate that phosphorylation of GlrR increases its DNA binding activity and thereby expression of the sRNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20965974" target="_blank">20965974</a>]<br></li>
<li>Taken together this indicates that the DNA-binding activity of GlrR is activated by its phosphorylation although it can not be excluded yet that the mutations in GlrR affected the stability rather than activity of the protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20965974" target="_blank">20965974</a>]<br></li>
<li>Cellular processes regulated by σ54-dependent bEBPs in S. Typhimurium include nitrogen metabolism in response to limiting nitrogen conditions [ NtrC ( GlnG , NRI ) , Keener and Kustu , 1988 ; Zimmer et al. , 2000 ] , transport and catabolism of D-glucosaminate ( DgaR , Miller et al. , 2013 ) and glucoselysine/fructoselysine ( GfrR , Miller et al. , 2015 ) , regulation of cytoplasmic pH homeostasis during fermentative growth by the formate-hydrogen lyase system ( FhlA , Hopper and Böck , 1995 ; Lamichhane-Khadka et al. , 2015 ) , response to assaults to the cell envelope ( PspF , Karlinsey et al. , 2010 ; Flores-Kim and Darwin , 2015 and zinc-dependent ZraR , Appia-Ayme et al. , 2012 ) , reduction of nitric oxide under anaerobic conditions ( NorR , Hutchings et al. , 2002 ; Mills et al. , 2005 ) , propionate catabolism ( PrpR , Palacios and Escalante-Semerena , 2000 ) , regulation of amino-sugar synthesis by sRNAs ( GlrR ; Gopel et al. , 2011 ) , and RNA repair/processing ( RtcR , Samuels , 2014 ; Engl et al. , 2016 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27583250" target="_blank">27583250</a>]<br></li>
<li>Accordingly , GlrR speciﬁcally binds to these regions and stimulates transcription ( Figures 2 and 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20965974" target="_blank">20965974</a>]<br></li>
<li>Accordingly , glmY and glmZ most likely originated from duplication of a single sRNA locus in an ancestor of Enterobacteriaceae and transcription of this ancient sRNA was presumably already controlled by s54 and GlrR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20965974" target="_blank">20965974</a>]<br></li>
<li>We show that both , glmY and glmZ , are controlled by GlrR and s54 in Y. pseudotuberculosis , S. typhimurium and presumably in many other species .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20965974" target="_blank">20965974</a>]<br></li>
<li>Hence , glmY and glmZ compose a regulon controlled by GlrR and s54 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20965974" target="_blank">20965974</a>]<br></li>
<li>In order to monitor activation by the cognate GlrR protein , Y. pseudotuberculosis glrR was expressed from plasmid pYG6 , while the endogenous glrR gene was deleted .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20965974" target="_blank">20965974</a>]<br></li>
<li>In these species , both sRNAs are expressed from s54-dependent promoters that require activation by GlrR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20965974" target="_blank">20965974</a>]<br></li>
<li>In conclusion , Y. pseudotuberculosis glmY as well as glmZ appear to be expressed exclusively from s54-dependent promoters that require activation by GlrR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20965974" target="_blank">20965974</a>]<br></li>
<li>These results show that Y. pseudotuberculosis glmZ is transcribed from a single s54-promoter , which requires activation by binding of GlrR to its upstream located ABS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20965974" target="_blank">20965974</a>]<br></li>
<li>In conclusion , transcription is initiated from single s54-promoters that require activation by GlrR and the same may also hold true for species of the genera Arsenophonus , Dickeya , Erwinia , Pectobacterium , Photorhabdus , Proteus and Serratia ( Figure 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20965974" target="_blank">20965974</a>]<br></li>
<li>In Y. pseudotuberculosis , transcription of glmY and glmZ is directed by single s54-promoters that require activation by GlrR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20965974" target="_blank">20965974</a>]<br></li>
				</div></ul><br>
			<label class="collapse_DOM" for="_2"><font size="5px">GlrR protein domains</font></label>
				<input id="_2" type="checkbox">
				<div class="DOM"><br><ul><li>The following strains were employed ( corresponding to the columns from left to right ) : Z197 , Z206 , Z370 , Z372 , Z190 , Z196 , Z371 and Z373 Phosphorylated GlrR is active and stimulates sRNA expression GlrR contains a response regulatory domain including the conserved putative phosphorylation site aspartate 56 at its N-terminus .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20965974" target="_blank">20965974</a>]<br></li>
<li>To obtain in vivo evidence that phosphorylated rather than dephosphorylated GlrR is active , we replaced the phosphorylation site Asp 56 in GlrR with an alanine and a glutamate residue , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20965974" target="_blank">20965974</a>]<br></li>
<li>The activator protein GlrR consists of an N-terminal response regulatory domain , a central s54-interaction module and a C-terminal helix-turn-helix DNA-binding motif .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20965974" target="_blank">20965974</a>]<br></li>
<li>GlrR binds three TGTCN GACA motifs located more than 10 100 bp upstream of glmY and thereby activates the s54-promoter , while activity of the s70-promoter is unaffected .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20965974" target="_blank">20965974</a>]<br></li>
<li>GlrR , which contains a s54 interaction domain , is absolutely required for activity of these s54-promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20965974" target="_blank">20965974</a>]<br></li>
				</div></ul><br>
			<label class="collapse_EVO" for="_3"><font size="5px">GlrR evolution</font></label>
				<input id="_3" type="checkbox">
				<div class="EVO"><br><ul><li>GlrR of E. coli shares 95 % and 87 % amino acid sequence identity with its homologs from S. typhimurium and Y. pseudotuberculosis , respect-ively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20965974" target="_blank">20965974</a>]<br></li>
				</div></ul><br>
			<label class="collapse_RP" for="_4"><font size="5px">Processes regulated by GlrR</font></label>
				<input id="_4" type="checkbox">
				<div class="RP"><br><ul><li>The microarray results for S. Typhi-murium LT2 genes within operons that meet the criteria for up-regulation , or that are known or predicted to be σ54-dependent , are shown in Table 2 and Additional file 1 glmY GlmY sRNA glmZ GlmZ sRNA GlrR NtrC [ 28 ] [ 29 ] STM4007-05 glnALG Glutamine synthetase Predicted σ54-dependent operons : STM0462-63 glnK amtB hypothetical protein PTS ( putative ) NtrC 3.6 67 11 1.8 2.5 2.4 f 17 [ 22 ] [ 18 ] STM0577-72 STM0571 STM0652 NtrC STM0649.S-53 Hydrolase ( putative ) Glutamate/aspartate-transporter [ 18 ] [ 18,30 ] STM0665-62 gltIJKL yeaGH Serine protein kinase ( putative ) STM1285-84 NtrC NtrC [ 18,30 ] [ 31,32 ] [ 5 ] [ 18 ] STM1303-07 Arginine/ornithine/glutamine-metabolism Phage shock proteins Amino acid transport ( putative ) astCABDE STM1690-86 PspF STM2361 pspABCDE ------ ubiX norV ygbD STM2360-56 100 16 13 26 5.6 71 STM2840-41 Nitric oxide reductase NorR FhlA FhlA FhlA [ 18,33 ] [ 34 ] [ 35 ] [ 35 ] STM2843-42 Hydrogenase maturation proteins Hydrogenase 3 Formate-hydrogen lyase system hydN hypF hycABCDEFGHIhypABCDE - rtcBA STM2853-44 STM2854-58 STM3521-18 RtcR RNA-repair system ( putative ) Heat shock sigma factor ( σ32 ) PTS ( putative ) Zinc resistance-associated protein [ 23 ] STM3568 1.7 [ 36,37 ] [ 18 ] [ 3,18 ] rpoH STM3772-66 39 16 3.7 1.4 29 16 STM3773 ZraR ZraR PspF STM4172 zraP STM4173-74 [ 3 ] hydHG pspG Zinc resistance two-component system Phage shock protein Formate dehydrogenase PTS ( putative ) STM4244 [ 38 ] [ 39 ] [ 18 ] STM4285 FhlA fdhF STM4535-40 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24007446" target="_blank">24007446</a>]<br></li>
<li>Cellular processes regulated by σ54-dependent bEBPs in S. Typhimurium include nitrogen metabolism in response to limiting nitrogen conditions [ NtrC ( GlnG , NRI ) , Keener and Kustu , 1988 ; Zimmer et al. , 2000 ] , transport and catabolism of D-glucosaminate ( DgaR , Miller et al. , 2013 ) and glucoselysine/fructoselysine ( GfrR , Miller et al. , 2015 ) , regulation of cytoplasmic pH homeostasis during fermentative growth by the formate-hydrogen lyase system ( FhlA , Hopper and Böck , 1995 ; Lamichhane-Khadka et al. , 2015 ) , response to assaults to the cell envelope ( PspF , Karlinsey et al. , 2010 ; Flores-Kim and Darwin , 2015 and zinc-dependent ZraR , Appia-Ayme et al. , 2012 ) , reduction of nitric oxide under anaerobic conditions ( NorR , Hutchings et al. , 2002 ; Mills et al. , 2005 ) , propionate-catabolism ( PrpR , Palacios and Escalante-Semerena , 2000 ) , regulation of amino-sugar synthesis by sRNAs ( GlrR ; Gopel et al. , 2011 ) , and RNA-repair/processing ( RtcR , Samuels , 2014 ; Engl et al. , 2016 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27583250" target="_blank">27583250</a>]<br></li>
<li>Interestingly , GlrR/GlrK have been implicated to play a role for virulence : Mutants of Y. pseudotuberculosis lacking GlrR exhibited reduced pathogenicity in mice ( 36 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20965974" target="_blank">20965974</a>]<br></li>
<li>Finally , evidence is provided that phosphorylation of GlrR increases its activity and thereby sRNA expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20965974" target="_blank">20965974</a>]<br></li>
<li>( B ) A glutamate replacement of the phosphorylation site Asp56 in GlrR strongly up-regulates glmY expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20965974" target="_blank">20965974</a>]<br></li>
<li>GlrR binds to three activator binding sites present upstream of s54-dependent promoters that control the expression of sRNA genes glmY in all species and glmZ in a subset of species .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20965974" target="_blank">20965974</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_7">
Comparison of the roles of GlrR and s54 for expression of glmY from E. coli , S. typhimurium and Y. pseudotuberculosis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20965974" target="_blank">20965974</a>]<br></label>
<input id="rp_similarity_7", type="checkbox">
<div class="RP_similar">
<li>Comparison of the roles of GlrR and s54 for expression of glmZ from E. coli , S. typhimurium and Y. pseudotuberculosis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20965974" target="_blank">20965974</a>]<br></li></div>
<li>For the sRNA glmY in S. Typhimurium and E. coli , it has previously been shown that transcription from the 54-dependent promoter , which is stimulated by the bEBP GlrR under glucosamine-6-phosphate-limiting conditions , represses expression from a 70-dependent promoter that precisely overlaps the 54-dependent promoter ( 50 , 59 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28373272" target="_blank">28373272</a>]<br></li>
<li>To test the possible effect of phosphorylation on GlrR activity , the protein was pre-incubated at 37 C for 1 h in the absence ( left panel ) or presence ( right panel ) of 50 mM acetyl phosphate before continuing with the EMSA protocol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20965974" target="_blank">20965974</a>]<br></li>
				</div></ul><br>
			<label class="collapse_SIT" for="_5"><font size="5px">GlrR binding site</font></label>
				<input id="_5" type="checkbox">
				<div class="SIT"><br><ul><li>Accordingly , GlrR speciﬁcally binds to these regions and stimulates transcription ( Figures 2 and 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20965974" target="_blank">20965974</a>]<br></li>
<li>The GlrR binding sites are likewise conserved although sequence deviations from the consensus TGTCN10GACA occur in a few cases , in particular in ABS 1 and 3 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20965974" target="_blank">20965974</a>]<br></li>
<li>Sequence motifs of a s54-promoter , three GlrR binding sites and two IHF binding sites are detect-able .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20965974" target="_blank">20965974</a>]<br></li>
<li>In these cases , sequence motifs for s54-promoters and for GlrR - and IHF-binding sites are lacking .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20965974" target="_blank">20965974</a>]<br></li>
<li>Yersinia possesses the sequences for a s54-promoter ( labeled in red ) and three GlrR binding sites upstream of both sRNA genes , while overlapping s70-promoters appear to be absent .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20965974" target="_blank">20965974</a>]<br></li>
<li>GlrR binds three TGTCN GACA motifs located more than 10 100 bp upstream of glmY and thereby activates the s54-promoter , while activity of the s70-promoter is unaffected .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20965974" target="_blank">20965974</a>]<br></li>
<li>Sequences for a p54-promoter and for binding sites of the response regulator GlrR as well as of IHF are shared features of the glmY and glmZ promoter regions of many , but not all Enterobacteriaceae We performed sequence alignments of the promoter regions of the glmY as well as glmZ genes retrieved from 39 genome sequences representing the most important genera of Enterobacteriaceae .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20965974" target="_blank">20965974</a>]<br></li>
<li>The response regulator GlrR binds the glmZ promoter region of S. typhimurium and Y. pseudotuberculosis , while the E. coli glmZ promoter is not bound The sequence alignment analysis of the glmZ promoter regions had revealed putative s54-promoters and GlrR binding sites in S. typhimurium and Y. pseudotuberculosis , while these elements are missing upstream of E. coli glmZ ( Figure 1B , Supplementary Figure S4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20965974" target="_blank">20965974</a>]<br></li>
				</ul></div><br>
			<label class="collapse_TU" for="_6"><font size="5px">Transcription units regulated by GlrR</font></label>
				<input id="_6" type="checkbox">
				<div class="TU"><br><ul><li>For the sRNA glmY in S. Typhimurium and E. coli , it has previously been shown that transcription from the 54-dependent promoter , which is stimulated by the bEBP GlrR under glucosamine-6-phosphate-limiting conditions , represses expression from a 70-dependent promoter that precisely overlaps the 54-dependent promoter ( 50 , 59 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28373272" target="_blank">28373272</a>]<br></li>
<li>Hence , expression of E. coli glmZ is not controlled by GlrR or s54 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20965974" target="_blank">20965974</a>]<br></li>
<li>E. coli glmZ is exclusively transcribed from a ppromoter , while Y. pseudotuberculosis glmZ transcription 54 depends on p and GlrR Our data suggested that glmZ of Y. pseudotuberculosis is transcribed from a single promoter that requires activation by s54 and GlrR , whereas expression of E. coli glmZ is not affected by these factors .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20965974" target="_blank">20965974</a>]<br></li>
<li>These results show that Y. pseudotuberculosis glmZ is transcribed from a single s54-promoter , which requires activation by binding of GlrR to its upstream located ABS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20965974" target="_blank">20965974</a>]<br></li>
				</div></ul></div><br>
	</div>
</body>
</html>