<!doctype html>
<style>
@charset "utf-8";
body {
	height: 100%;
	margin: 0;
	padding: 0;
}

.banner {
	position: fixed;
	top: 0;
	width: 100%;
	height: 65px;
	background: gray;
	line-height: 65px;
}
.spacer {
	height: 10px;
}

.main-content {
	padding: 5%;
}

.collapse_ACT {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-style: none solid none none;
	border-color: #B4045F;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_ACT + input{
	display: none;
}
.collapse_ACT + input + div{
	display: none;
}
.collapse_ACT + input:checked + div{
	display: block;
}
.ACT {
	text-align: justify;
	text-justify: inter-word;
}
.ACT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B4045F;
}
.ACT_collapse_similarity:hover{
	background: #e5d3dc;
}
.ACT_collapse_similarity + input {
	display: none;
}
.ACT_collapse_similarity + input + div{
	display: none;
}

.ACT_collapse_similarity + input:checked + div{
	display: block;
}
.ACT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.ACT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B4045F;
	padding-left: 5px;
}
.ACT_collapse_similarity:hover{
	background: #e5d3dc;
}
.ACT_collapse_similarity + input {
	display: none;
}
.ACT_collapse_similarity + input + div{
	display: none;
}
.ACT_collapse_similarity + input:checked + div{
	display: block;
}
.ACT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.ACT_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #B4045F;
}
.ACT_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_DOM {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-style: none solid none none;
	border-color: #B404AE;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_DOM + input{
	display: none;
}
.collapse_DOM + input + div{
	display: none;
}
.collapse_DOM + input:checked + div{
	display: block;
}
.DOM {
	text-align: justify;
	text-justify: inter-word;
}
.DOM_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B404AE;
	padding-left: 5px;
}
.DOM_collapse_similarity:hover{
	background: #e8d7e7;
}
.DOM_collapse_similarity + input {
	display: none;
}
.DOM_collapse_similarity + input + div{
	display: none;
}
.DOM_collapse_similarity + input:checked + div{
	display: block;
}
.DOM_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}.DOM_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #B404AE;
}
.DOM_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_EVO {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #7401DF;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_EVO + input{
	display: none;
}
.collapse_EVO + input + div{
	display: none;
}
.collapse_EVO + input:checked + div{
	display: block;
}
.EVO {
	text-align: justify;
	text-justify: inter-word;
}
.EVO_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #7401DF;
	padding-left: 5px;
}
.EVO_collapse_similarity:hover{
	background: #e2d7ed;
}
.EVO_collapse_similarity + input {
	display: none;
}
.EVO_collapse_similarity + input + div{
	display: none;
}
.EVO_collapse_similarity + input:checked + div{
	display: block;
}
.EVO_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.EVO_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #7401DF;
}
.EVO_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_RP {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #0174DF;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_RP + input{
	display: none;
}
.collapse_RP + input + div{
	display: none;
}
.collapse_RP + input:checked + div{
	display: block;
}
.RP {
	text-align: justify;
	text-justify: inter-word;
}
.RP_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #0174DF;
	padding-left: 5px;
}
.RP_collapse_similarity:hover{
	background: #c9d7e2;
}
.RP_collapse_similarity + input {
	display: none;
}
.RP_collapse_similarity + input + div{
	display: none;
}
.RP_collapse_similarity + input:checked + div{
	display: block;
}
.RP_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.RP_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #0174DF;
}
.RP_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_SIT {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #0489B1;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_SIT + input{
	display: none;
}
.collapse_SIT + input + div{
	display: none;
}
.collapse_SIT + input:checked + div{
	display: block;
}
.SIT {
	text-align: justify;
	text-justify: inter-word;
}
.SIT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #0489B1;
	padding-left: 5px;
}
.SIT_collapse_similarity:hover{
	background: #d0dfe2;
}
.SIT_collapse_similarity + input {
	display: none;
}
.SIT_collapse_similarity + input + div{
	display: none;
}
.SIT_collapse_similarity + input:checked + div{
	display: block;
}
.SIT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.SIT_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #0489B1;
}
.SIT_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_TU {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #04B486;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_TU + input{
	display: none;
}
.collapse_TU + input + div{
	display: none;
}
.collapse_TU + input:checked + div{
	display: block;
}
.TU {
	text-align: justify;
	text-justify: inter-word;
}
.TU_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #04B486;
	padding-left: 5px;
}
.TU_collapse_similarity:hover{
	background: #d0e2de;
}
.TU_collapse_similarity + input {
	display: none;
}
.TU_collapse_similarity + input + div{
	display: none;
}
.TU_collapse_similarity + input:checked + div{
	display: block;
}
.TU_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.TU_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #04B486;
}
.TU_similar > a {
	text-decoration: none;
	color: gray;
}

ul { 
	list-style-type: none;
	padding: 0;
	margin: 0;
}
ul li > a {
	text-decoration: none;
	color: gray;
}
ul li:hover, ul li:focus {
	background: #F0F0F0;
}

ul li:first-child span {
  border-left:none;  
}

</style>
<html>
<head>
<meta charset="utf-8">
<title>TF properties - Hha</title>
<meta name="theme-color" content="#ffffff">
</head>
<body>
	<div class="banner">
		<center><font color="white" size="16px" face="arial">Hha</font></center>
	</div>
	<div align="center" class="main-content">
		<div class="spacer">
	</div><br>			<label class="collapse_ACT" for="_1"><font size="5px">Hha transcriptional activity</font></label>
				<input id="_1" type="checkbox">
				<div class="ACT"><br><ul><li>The concentrations of Hha used were as follows : lane 1 , no protein ; lane 2 , 1 mol ; lane 3 , 0.5 mol ; lane 4 , 0.25 mol ; lane 5 , 0.13 mol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11673432" target="_blank">11673432</a>]<br></li>
<li>Further work demonstrated that puriﬁed Hha protein could bind to a hilA DNA promoter fragment .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12595445" target="_blank">12595445</a>]<br></li>
<li>It is also possible that Hha has a repressing effect on transcription of downstream invasion genes independent of hilA transcriptional repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11673432" target="_blank">11673432</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_4">
Hha is a negative modulator of hilA transcription , the Salmonella typhimurium invasion gene transcriptional-activator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15039320" target="_blank">15039320</a>]<br></label>
<input id="act_similarity_4", type="checkbox">
<div class="ACT_similar">
<li>Hha is a negative modulator of hilA transcription , the Salmo-nella typhimurium invasion gene transcriptional-activator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15765064" target="_blank">15765064</a>]<br></li></div>
				<label class="ACT_collapse_similarity" for="act_similarity_6">
Fahlen TF , Wilson RL , Boddicker JD , Jones BD : Hha is a negative modulator of transcription of hilA , the Salmonella enterica serovar Typhimurium invasion gene transcriptional-activator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21689395" target="_blank">21689395</a>]<br></label>
<input id="act_similarity_6", type="checkbox">
<div class="ACT_similar">
<li>[ 184 ] Fahlen , T.F. , Wilson , R.L. , Boddicker , J.D. and Jones , B.D. ( 2001 ) Hha is a negative modulator of transcription of hilA , the Salmonella enterica serovar typhimurium invasion gene transcriptional-activator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16023758" target="_blank">16023758</a>]<br></li>
<li>Fahlen , T.F. , Wilson , R.L. , Boddicker , J.D. , and Jones , B.D. ( 2001 ) Hha is a negative modulator of transcription of hilA , the Salmonella enterica serovar Typhimurium invasion gene transcriptional-activator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16045614" target="_blank">16045614</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28631437" target="_blank">28631437</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12492857" target="_blank">12492857</a>]<br></li>
<li>Fahlen , T. F. , Wilson , R. L. , Boddicker , J. D. , and Jones , B. D. ( 2001 ) Hha is a negative modulator of transcription of hilA , the Salmonella enterica serovar Typhimurium invasion gene transcriptional-activator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23515315" target="_blank">23515315</a>]<br></li>
<li>Fahlen , T. , Wilson , R. , Boddicker , J. , and Jones , B. ( 2001 ) Hha is a negative modulator of transcription of hilA , the Salmonella enterica serovar Typhimurium invasion gene transcriptional-activator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15661008" target="_blank">15661008</a>]<br></li></div>
<li>Moreover , H-NS and Hha have been proven to synergistically contribute to such repression under noninducing conditions ( 292 , 293 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23554419" target="_blank">23554419</a>]<br></li>
<li>This effect can be counter-balanced and modulated by other H-NS-like proteins such as Hha ( YmoA ) which modify H-NS promoted silencing by heterocomplex formation under certain growth conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26441883" target="_blank">26441883</a>]<br></li>
<li>Hha , YbaJ , and OmpA regulate Escherichia coli K12 bioﬁlm formation and conjugation plasmids abolish motility .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24735176" target="_blank">24735176</a>]<br></li>
<li>Both chromosomal - and plasmid-encoded H-NS and Hha proteins interact to modulate R27 temperature-dependent conjugative transfer [ 18 ] : either plasmid - or chromosomally-encoded H-NS and Hha proteins can repress conjugative transfer at high temperature .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19521501" target="_blank">19521501</a>]<br></li>
<li>Our results conﬁrm these observations and provide additional data for low osmolarity conditions , in which HilD and HilC activators were not essential in mutants lacking H-NS and Hha proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16443238" target="_blank">16443238</a>]<br></li>
<li>Garcıa , J. , Madrid , C. , Cendra , M. , Juárez , A. , and Pons , M. ( 2009 ) N9L and L9N mutations toggle Hha binding and hemolysin regulation by Esche-richia coli and Vibrio cholerae H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23515315" target="_blank">23515315</a>]<br></li>
<li>The data show that all three genes , hilD , hilC , and rtsA , were repressed by H-NS and/or Hha .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17675384" target="_blank">17675384</a>]<br></li>
<li>Negative modulation of hilA is also accomplished by global modulators such as Hha ( Fahlen et al. , 2001 ) and H-NS ( Olekhnovich & Kadner , 2006 ; Schechter et al. , 2003 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21680637" target="_blank">21680637</a>]<br></li>
<li>Co-factors such as Hha and YdgT , and other nucleoid associated proteins like Fis contribute to the expression of horizontally acquired genes in Salmonella , including those that regulate virulence ( Schechter et al. , 2003 ; Silphaduang et al. , 2007 ; Vivero et al. , 2008 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29034217" target="_blank">29034217</a>]<br></li>
<li>Positive regulators include BarA/SirA , FliZ , EnvZ/OmpR , FadD , CsrB , HilD and some nucleoid binding proteins ( Fis , HU ) , while Pag , HilE , Ams and the nucleoid binding proteins Hha and H-NS are negatively regulating hilA transcription ( Altier et al. 2000a , b Fahlen et al. 2000 ; Lucas et al. 2000 ; Wilson et al. 2001 ; Schechter et al. 2003 ; Baxter et al. 2003 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18563621" target="_blank">18563621</a>]<br></li>
<li>The above data suggested that Hha might be involved in the repression of genes carried in SPI-2 under certain environmental conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17307861" target="_blank">17307861</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_22">
The absence of Hha resulted in increased expression in both media , but with a greater relative increase in the low-salt medium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16443238" target="_blank">16443238</a>]<br></label>
<input id="act_similarity_22", type="checkbox">
<div class="ACT_similar">
<li>The absence of Hha resulted in increased expression in both media .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16443238" target="_blank">16443238</a>]<br></li></div>
				<label class="ACT_collapse_similarity" for="act_similarity_24">
Csn-B inhibited the secretion of SPI-1 effectors through the Hha -- H-NS regulatory pathway .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23459474" target="_blank">23459474</a>]<br></label>
<input id="act_similarity_24", type="checkbox">
<div class="ACT_similar">
<li>FIG 3 Csn-B inhibited SPI-1 effectors through the Hha -- H-NS regulatory pathway .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23459474" target="_blank">23459474</a>]<br></li></div>
<li>Collectively , these data provide evidence that Hha binds speciﬁcally to the hilA promoter , which we believe is the likely mechanism by which it represses hilA transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11673432" target="_blank">11673432</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_27">
Pons , J. I. , Rodrıguez , S. , Madrid , C. , Juárez , A. , and Nieto , J. M. ( 2004 ) In vivo increase of solubility of overexpressed Hha protein by tandem expression with interacting protein H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23515315" target="_blank">23515315</a>]<br></label>
<input id="act_similarity_27", type="checkbox">
<div class="ACT_similar">
<li>In vivo increase of solubility of overexpressed Hha protein by tandem expression with interacting protein H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16443238" target="_blank">16443238</a>]<br></li></div>
<li>It would be interesting to determine how HilE and Hha regulate hilC intracellularly ( is the mechanism similar to hilA regulation ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15039320" target="_blank">15039320</a>]<br></li>
<li>The Hha protein is able to-bind to DNA and to the H-NS protein , and has been reported to repress SPI1 ( Fahlen et al. , 2001 ) ; our data suggest that Hha plays a role in regulating Salmonella gene expression in vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12492857" target="_blank">12492857</a>]<br></li>
<li>In contrast , Boddicker et al. ( 5 ) found that the HilD protein was required to activate hilA-lacZ transcription even when several negative regulators , Hha , Ams , HilE , and/or Pag , were deleted in combination .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15126488" target="_blank">15126488</a>]<br></li>
<li>Additional repressors of hilA may include HilE , Pag , and Hha , as strains lacking these proteins exhibit increased hilA expression under inducing conditions ( 4 , 10 , 11 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12933899" target="_blank">12933899</a>]<br></li>
<li>To determine whether HilE , Pag , or Hha affects repression , rather than derepression , of hilA , the effect that hilE , pag , and hha mutations have on hilA expression should be examined under repressing environmental conditions in strains lacking HilC and HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12933899" target="_blank">12933899</a>]<br></li>
<li>Therefore , we observe that Hha enhanced H-NS DNA binding in vitro but could not form stable protein-DNA complexes in the absence of H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23515315" target="_blank">23515315</a>]<br></li>
<li>Such a model would explain why the in vitro DNA binding assays performed with purified components found that Hha can increase the ability of H-NS to-bind DNA under a narrow range of conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23515315" target="_blank">23515315</a>]<br></li>
<li>Effect on hilA expression decrease decrease decrease decrease decrease decrease decrease Mutation1 Comments ompR envZ [ 66 ] [ 66 ] EnvZ/OmpR is a two-component system EnvZ/OmpR is a two-component system pho-24 sirA , barA pstS fadD ﬂiZ [ 19 ] [ 64 , 65 , 68 ] , unpublished2 [ 66 ] [ 66 ] [ 66 ] , unpublished3 hyperactive PhoQ sensor kinase BarA/SirA is a two-component system acts via PhoB regulator does not act via FadR regulator transposon insertions in ﬂiA are polar on ﬂiZ ; FliZ suppresses the effects of transposon insertions in ﬂiA and ﬂhCD the Csr system regulates RNA stability Fis is a small nucleoid binding protein HU , a small nucleoid binding protein , is comprised of HupA and HupB Hha is a small nucleoid binding protein amsE encodes RNase E H-NS is a small nucleoid binding protein no known homologies PhoPQ-activated gene csrB ﬁs decrease decrease decrease decrease increase increase increase increase [ 64 , 68 ] unpublished4 ,5 unpublished4 ,5 unpublished2 [ 56 ] unpublished4 ,5 [ 56 ] [ 56 ] hupA/hupB hha amsE : : Tn5-1 hns hilE pag-1 Condition high osmolarity low oxygen late log phase near-neutral pH low osmolarity low oxygen early log phase bile [ 19 ] , unpublished3 [ 19 ] , unpublished6 [ 57 ] , unpublished7 unpublished3 [ 19 ] , unpublished3 [ 19 ] , unpublished6 [ 57 ] , unpublished7 unpublished3 osmolarity regulates hilA in the absence of OmpR/EnvZ oxygen regulates hilA in the absence of OxrA/Fnr or ArcA induction induction induction induction repression repression repression repression late log phase induces invasion activity ; RpoS is not required not mediated by PmrAB osmolarity regulates hilA expression in the absence of OmpR/EnvZ oxygen regulates hilA in the absence of OxrA/Fnr or ArcA early log phase represses invasion activity ; RpoS is not required reported to regulate invasion , prgH , invF and hilC ; effects on hilA expression not reported but very likely 1 Unless otherwise noted , the mutations are null mutations in the open reading frame listed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11755416" target="_blank">11755416</a>]<br></li>
<li>Hha Is a Negative Modulator of Transcription of hilA , the Salmonella enterica Serovar Typhimurium Invasion Gene Transcriptional Activator THOMAS F. FAHLEN , REBECCA L. WILSON , JENNIFER D. BODDICKER , AND BRADLEY D. JONES * Department of Microbiology , University of Iowa School of Medicine , Iowa City , Iowa 52242 Received 2 July 2001/Accepted 29 August 2001 An early step in the establishment of Salmonella enterica serovar Typhimurium murine infection is the penetration of the intestinal mucosa of the small intestine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11673432" target="_blank">11673432</a>]<br></li>
<li>With hilA expression driven by increasing concentrations of tetracycline in the tetRA -- rtsA strain , loss of Hha still resulted in hilA induction ( Figure 6B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22021388" target="_blank">22021388</a>]<br></li>
<li>In the absence of Hha , the full-length hilA promoter showed about 3.5-fold increase in b-galactosidase levels in both high and low-salt medium ( Figure 2 , line 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16443238" target="_blank">16443238</a>]<br></li>
<li>The absence of Hha had a lesser effect on silencing by the DSR than did H-NS , which is in contrast to the effect of Hha when the USR was also present .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16443238" target="_blank">16443238</a>]<br></li>
<li>As the concentration of Hha in the binding reactions was increased while the concentration of H-NS was held constant , the hilA fragment shifted to a lower mobility protein-DNA complex ( Fig. 7A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23515315" target="_blank">23515315</a>]<br></li>
<li>This enhancement was most noticeable when the concentration of H-NS was kept low ( 300 nM ) and the salt component of the binding reactions was increased to 250 mM in order to lessen the DNA binding capability of H-NS ( i.e. we notice no additional effect of Hha under conditions where H-NS has already fully shifted the promoter fragment ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23515315" target="_blank">23515315</a>]<br></li>
<li>The reduction in hilA-lac expression by overexpression of the RpoA variant with the CTD deleted was seen only in cells grown in the absence of the repressive Hha protein and in the presence of the inducer bile salts .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15126488" target="_blank">15126488</a>]<br></li>
<li>In promoter variants carrying upstream ﬂanking sequences K210/C90 or K242/C90 , the silencing effect of the USR in low-salt medium was essentially lost in the absence of H-NS and was greatly decreased in the absence of Hha .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16443238" target="_blank">16443238</a>]<br></li>
<li>Thus , all the fusions tested were derepressed to different degrees in the absence of H-NS and/or Hha ; however , the greatest difference was seen for the effect on the rtsA-lac fusion .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17675384" target="_blank">17675384</a>]<br></li>
<li>Under low-osmolarity conditions , the up-regulation of rtsA transcription in the absence of Hha and H-NS was evident , similar to the data obtained with the rtsA-lac fusion .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17675384" target="_blank">17675384</a>]<br></li>
<li>The Hha protein enhances the silencing effect of H-NS on hilA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21680637" target="_blank">21680637</a>]<br></li>
<li>Based on these observations , we suggest that Lrp acts as an antagonist of HilD , which induces expression of the hilA gene by derepressing the silencing effect of H-NS and Hha on the hilA promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19074398" target="_blank">19074398</a>]<br></li>
<li>Thus , Hha may modify the architecture of the hilA promoter to negatively regulate its expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15765064" target="_blank">15765064</a>]<br></li>
<li>Based on the evidence presented here that hha is a negative modulator of hilA transcription and invasion , we investigated whether the Hha protein could bind directly to the hilA promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11673432" target="_blank">11673432</a>]<br></li>
<li>To investigate this unique model of regulation by HilD , we determined the effect of a hilD mutation in strains of Salmonella that also contained mutations in the negative regulators Hha , HilE , Pag , and RNase E. Work in our laboratory indicates that Hha regulates the hilA promoter directly , due to its ability to-bind to the hilA promoter upstream of position 39 ( 17 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12511499" target="_blank">12511499</a>]<br></li>
<li>Since neither HilE , Hha , Pag , nor RNase E is solely responsible for repression of hilA : : Tn5lacZY in the absence of HilD , we considered the possibility that these negative regulators function cooperatively to repress hilA expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12511499" target="_blank">12511499</a>]<br></li>
<li>This strain lacks the potent negative factor Hha and exhibits elevated hilA expression ( 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15126488" target="_blank">15126488</a>]<br></li>
<li>HilE and Hha may block this activation by binding to HilD and the hilA promoter , respectively , under extracellular repressive conditions for invasion ( 8 , 26 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15039320" target="_blank">15039320</a>]<br></li>
<li>To date , it is known that the positive regulators HilC and HilD and the negative regulators Hha and Fis can bind directly the hilA promoter ( 8 , 19 , 53 , 59 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14645287" target="_blank">14645287</a>]<br></li>
<li>The presence of 60 nM MBP-Hha also resulted in shift of K242/K55 and C100/C300 fragments , and reduced binding to core promoter fragment ( K55/C90 ) ( line 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16443238" target="_blank">16443238</a>]<br></li>
<li>Four nucleoid-like proteins in enteric bacteria , H-NS , StpA , Hha , and YdgT have a predilection for binding to A+T rich sequences and repress transcription of SPI-2 genes ( rev. in [ 14 ] ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19229334" target="_blank">19229334</a>]<br></li>
<li>Although other nucleoid-associated proteins , such as YdgT , Hha , and StpA , also recognize and selectively silence the expression of foreign DNA , none of them were down-regulated by σE ( Table S2 ) , and therefore were not further investigated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25713562" target="_blank">25713562</a>]<br></li>
<li>Garcıa , J. , Cordeiro , T. N. , Nieto , J. M. , Pons , I. , Juárez , A. , and Pons , M. ( 2005 ) Interaction between the bacterial nucleoid associated proteins Hha and H-NS involves a conformational change of Hha .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23515315" target="_blank">23515315</a>]<br></li>
<li>Forns , N. , Baños , R.C. , Balsalobre , C. , Juarez , A. , and Madrid , C. ( 2005 ) Temperature-dependent conjugative transfer of R27 : role of chromosome - and plasmid-encoded Hha and H-NS proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24024872" target="_blank">24024872</a>]<br></li>
<li>Transcription of hilA is subject to direct or indirect negative control by multiple nucleoid binding proteins , including Hha , H-NS , HU , and Fis ( 10 , 23 ) , and other proteins , including HilE , Pag , and Ams ( 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15126488" target="_blank">15126488</a>]<br></li>
<li>Of the small nucleoid-bind-ing proteins , H-NS and Hha can repress hilA expression , but FIS and HU may activate ( 12 ; citation in references 30 and 47 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12107132" target="_blank">12107132</a>]<br></li>
<li>Our work shows that H-NS and Hha repress hilA under a set of well-defined environmental and physiological conditions and consequently account for SPI1 gene silencing .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21680637" target="_blank">21680637</a>]<br></li>
<li>By controlling expression/activity of HilD , HilC , RtsA , and HilA , many regulators contribute to SPI-1 gene regulation in response to environmental stimuli that include temperature ( H-NS and Hha ) , osmolarity ( H-NS and Hha ) , pH ( OmpR ) , iron availability ( Fur ) , and oxygen levels ( FNR ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27601571" target="_blank">27601571</a>]<br></li>
<li>Its high activity is strongly depressed in low osmolarity conditions by the nucleoid-structuring proteins H-NS and Hha , possibly by formation of a repressive DNA loop .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16443238" target="_blank">16443238</a>]<br></li>
<li>H-NS/Hha negatively regulate the hilA and rtsA genes under low-osmolarity conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17675384" target="_blank">17675384</a>]<br></li>
<li>In addition to the negative control that was exerted by H-NS/Hha under low-osmolarity conditions , the homologous virulence activators HilD , HilC , and RtsA ( Hil activators ) induced rtsA-lac expression in a high-salt medium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17675384" target="_blank">17675384</a>]<br></li>
<li>Thus , cooperative action of H-NS and Hha appears to be responsible for hilA silencing in the low osmolarity conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16443238" target="_blank">16443238</a>]<br></li>
<li>The depletion of NaCl from LB medium lead to a significant decrease in rtsA-lac expression , which is restored for the hns mutant and is further induced for the hns hha double mutant ( Fig. 1 and 8 ) , suggesting that the repression under low-osmolarity conditions is mediated by H-NS/Hha .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17675384" target="_blank">17675384</a>]<br></li>
<li>The requirement for Hil activators to induce the rtsA promoter under high-osmolarity conditions is lost in the absence of Hha and H-NS proteins ( Fig. 8 ) , suggesting that Hil activators antagonize H-NS/Hha-mediated repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17675384" target="_blank">17675384</a>]<br></li>
<li>In the second model , Hha can bind DNA and influence transcription independently of H-NS ( 22 , 23 , 32 , 33 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23515315" target="_blank">23515315</a>]<br></li>
<li>288 , NO. 19 , pp. 13356 -- 13369 , May 10 , 2013 Published in the U.S.A. Structural Insights into the Regulation o Salmonella by the Hha/H-NS Complex * f Foreign Genes in Received for publication , January 21 , 2013 , and in revised form , March 19 , 2013 Published , JBC Papers in Press , March 20 , 2013 , DOI 10.1074 / jbc.M113 .455378 Sabrina S. Ali ‡ , John C. Whitney § ¶ 1 , James Stevenson ‡ , Howard Robinson , P. Lynne Howell § ¶ 2 , and William Wiley Navarre ‡ 3 From the Departments of ‡ Molecular Genetics and § Biochemistry , University of Toronto , Toronto , Ontario M5S 1A8 , Canada , the ¶ Program in Molecular Structure and Function , Research Institute , Hospital for Sick Children , Toronto , Ontario M5G 1X8 , Canada , and the Biology Department , Brookhaven National Laboratory , Upton , New York , 11973-5000 Background : Hha facilitates H-NS-mediated silencing of foreign genes in bacteria .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23515315" target="_blank">23515315</a>]<br></li>
<li>Transcriptional activation by HilC and HilD relieves repression of the hilA promoter by the nucleoid proteins H-NS and Hha ( Olekhnovich and Kadner 2006 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20008574" target="_blank">20008574</a>]<br></li>
<li>Each of the three regulators can independently bind and activate the hilA promoter , their own gene , and the genes of the respective other regulators by counteracting silencing mediated by H-NS and Hha ( YmoA in Yersinia ) ( Olekhnovich and Kadner , 2002 ; Ellermeier et al. , 2005 ; Olekhnovich and Kadner , 2006 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26441883" target="_blank">26441883</a>]<br></li>
<li>HilC and HilD relieve repression of the hilA promoter by the nucleoid proteins H-NS and Hha ( Olekhnovich and Kadner 2006 ) and can activate expression of the invF and sicA/sip transcriptional units independently of HilA ( Rakeman et al. 1999 ; Akbar et al. 2003 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25991823" target="_blank">25991823</a>]<br></li>
<li>Nucleoid proteins Hha and H-NS bind to the promoters of SPI-1 regulator genes hilD , hilC , and rtsA to repress their expression and consequently regulate the expression of SPI-1 apparatus and effector genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23459474" target="_blank">23459474</a>]<br></li>
<li>Nucleoid proteins Hha and H-NS bond with the promoter regions of hilD , hilC , hilA , and rtsA to inhibit the transcription of SPI-1 genes and expression of SPI-1 effectors ( 59 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23459474" target="_blank">23459474</a>]<br></li>
<li>Temperature , osmolarity and growth-phase - dependent silencing of hilA by H-NS and Hha In the Enterobacteriaceae , H-NS and Hha are involved in thermo - and osmoregulation of several genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21680637" target="_blank">21680637</a>]<br></li>
<li>Although the extent of overexpresssion differed , repression of a set of genes by H-NS / Hha at low and high temperature and at low and high osmolarity indicates that the H-NS/Hha complex efficiently silences their expression under a wide range of non-permissive conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19521501" target="_blank">19521501</a>]<br></li>
<li>For insertions surrounding hilA and hilD yet with an opposite orientation to these genes , it is likely that disruption of chromosomal architecture by the TPOP insertions imparted derepression of SPI-1 , a region whose repression relies upon extensive modiﬁcation of DNA topology and interactions with repressive proteins , such as Hha and H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27185788" target="_blank">27185788</a>]<br></li>
<li>The effects of certain environmental conditions , such as growth-phase , oxygen , and osmolarity , may be mediated by small DNA binding proteins , such as Hha and Fis , whose functions can couple cellular physiology to the topology of the bacterial chromosome , altering DNA supercoiling and transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15765064" target="_blank">15765064</a>]<br></li>
<li>The presence of the ﬂanking sequences 40,46 greatly decreases hilA expression by providing sites for binding of silencing proteins , including H-NS and Hha .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16443238" target="_blank">16443238</a>]<br></li>
<li>Taken together , our results show for the first time how Hha and H-NS interact to direct transcriptional repression and reveal that a positively charged surface of Hha enhances the silencing activity of H-NS nucleoprotein filaments .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23515315" target="_blank">23515315</a>]<br></li>
<li>In the absence of Hha and H-NS , however , osmoregulation of the rtsA promoter was lost , and Hil activators were not required for rtsA transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17675384" target="_blank">17675384</a>]<br></li>
<li>Comparing the induced group with the noninduced group , the results ( Fig. 3B ) indicated that temperature change led to downregulation of transcription of hns and hha genes , but no signiﬁcant inﬂuence on transcription of phoP and phoQ was found , suggesting that nucleoid protein complex Hha -- H-NS but not PhoP and PhoQ may be the predominant regulator of SPI-1 effector genes under temperature change .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23459474" target="_blank">23459474</a>]<br></li>
<li>Absence of Hha and H-NS relieved the decreased expression resulting from the presence of either DSR or USR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16443238" target="_blank">16443238</a>]<br></li>
<li>The evidence , as exempliﬁed by H-NS and Hha proteins , that there exists a crosstalk between the plasmid and chromosomal regulatory networks prompted us to assess the global impact of the presence of plasmid R27 on the Salmonella transcriptome .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24024872" target="_blank">24024872</a>]<br></li>
<li>In this report , we investigate the role of nucleoid-associated proteins Hha and H-NS in the expression of virulence activator proteins HilD , HilC , and RtsA with an emphasis on the interplay of positive and negative transcriptional factors in the regulation of the rtsA promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17675384" target="_blank">17675384</a>]<br></li>
<li>Mobility-shift assays revealed the presence of binding sites for the H-NS and Hha proteins , both upstream and downstream of the promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16443238" target="_blank">16443238</a>]<br></li>
<li>Role of Nucleoid-Associated Proteins Hha and H-NS in Expression of Salmonella enterica Activators HilD , HilC , and RtsA Required for Cell Invasion Igor N. Olekhnovich * and Robert J. Kadner † icrobiology , University of Virginia School of Medicine , Charlottesville , Virginia 22908-0734 Received 8 June 2007/Accepted 23 July 2007 ent of M Departm The coordinate expression of Salmonella enterica invasion genes on Salmonella pathogenicity island 1 is under the control of the complex circuits of regulation that involve the AraC/XylS family transcriptional-activators HilD , HilC , and RtsA and nucleoid-associated proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17675384" target="_blank">17675384</a>]<br></li>
<li>[ 15 ] U. Silphaduang , M. Mascarenhas , M. Karmali , and B. K. Coombes , `` Repression of intracellular virulence factors in Salmonella by the Hha and YdgT nucleoid-associated proteins , '' Journal of Bacteriology , vol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22900174" target="_blank">22900174</a>]<br></li>
<li>Repression of Intracellular Virulence Factors in Salmonella by the Hha and YdgT Nucleoid-Associated Proteins Umaporn Silphaduang ,1,2,3 Mariola Mascarenhas ,2 Mohamed Karmali ,2,3 and Brian K. Coombes1 ,2 * Department of Biochemistry and Biomedical Sciences and Antimicrobial Research Centre , McMaster University , Hamilton , ON , Canada1 ; Laboratory for Foodborne Zoonoses , Public Health Agency of Canada , Guelph , ON , Canada2 ; and Department of Molecular and Cellular Biology , University of Guelph , Guelph , ON , Canada3 Received 1 January 2007/Accepted 9 February 2007 The Hha/YmoA family of nucleoid-associated proteins is involved in gene regulation in enterobacteria .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17307861" target="_blank">17307861</a>]<br></li>
<li>This is an Open Access article distributed under the terms of the Creative Commons Attribution License ( http://creativecommons.org/licenses/by/2.0 ) , which permits unrestricted use , distribution , and reproduction in any medium , provided the original work is properly cited Results Flagellar-based motility and surface flagellar expression is abolished in hha ydgT mutants During our characterization of Hha and YdgT-mediated repression of SPI-2 genes , we noted that hha ydgT mutant bacteria settled to the bottom of standing culture tubes whereas wild type cultures remained turbid .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21689395" target="_blank">21689395</a>]<br></li>
<li>The ability of Hha and YdgT to act as positive regulators is manifested only in the presence of both proteins , as single deletions of hha and ydgT had no apparent effect on flagellar biosynthesis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21689395" target="_blank">21689395</a>]<br></li>
				</div></ul><br>
			<label class="collapse_DOM" for="_2"><font size="5px">Hha protein domains</font></label>
				<input id="_2" type="checkbox">
				<div class="DOM"><br><ul><li>Therefore , MBP-Hha was used in all subsequent DNA-binding assays .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17675384" target="_blank">17675384</a>]<br></li>
<li>de Alba et al. ( 52 ) identified residues Glu-25 and Asp-48 from Hha as critical for the FIGURE 6 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23515315" target="_blank">23515315</a>]<br></li>
<li>The Hha residues whose signals were most perturbed included surface-exposed residues as well as hydrophobic residues deeply buried in the interfaces between helices 1 and 3 and helices 2 and 3 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23515315" target="_blank">23515315</a>]<br></li>
<li>Individual structures of both Hha and H-NS N-terminal domains were previously solved by NMR and x-ray crystallography .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23515315" target="_blank">23515315</a>]<br></li>
<li>An electrostatic representation of the Hha solution structure shows an asymmetrical charge distribution that relates to specific functions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23515315" target="_blank">23515315</a>]<br></li>
<li>Taken together , our results support a model whereby Hha mol-ecules coat opposite surfaces of H-NS protein filaments and mediate gene silencing through a positively charged cluster of surface-exposed residues .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23515315" target="_blank">23515315</a>]<br></li>
<li>However , the observation that E. coli YdgT ( 47 ) , Hha ( 22 , 48 ) , and Yersinia YmoA ( 22 ) can bind to the N-terminal oligomerization domain of H-NS suggests that regulatory specificity might be achieved through interaction with accessory proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16301528" target="_blank">16301528</a>]<br></li>
<li>Conclusion : Hha binds the dimerization domain of H-NS and may contact DNA via positively charged surface residues .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23515315" target="_blank">23515315</a>]<br></li>
<li>New roles for key residues in helices H1 and H2 of the Escherichia coli H-NS N-terminal domain : H-NS dimer stabilization and Hha binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18039769" target="_blank">18039769</a>]<br></li>
<li>We consistently observe that mutations in Hha or point mutations in the dimerization domain of H-NS result in substantial up-regulation of hilA transcription but have little effect on H-NS occupancy at the same region in vivo when measured by ChIP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23515315" target="_blank">23515315</a>]<br></li>
<li>Essential residues in the H-NS binding site of Hha , a co-regulator of horizontally acquired genes in Enterobacteria .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23459474" target="_blank">23459474</a>]<br></li>
<li>Hha is known to interact with the N-ter-minal dimerization domain of H-NS ; however , the manner in which this interaction enhances transcriptional silencing is not understood .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23515315" target="_blank">23515315</a>]<br></li>
<li>Although the negatively charged surface of Hha is partially buried upon complex formation with H-NS ( 1 -- 46 ) , a cluster of positively charged residues ( Arg-14 , Arg-16 , Arg-17 , Arg-26 , and Lys-30 ) projects out from the surface of Hha , pointing away from the Hha/H-NS ( 1 -- 46 ) interaction interface ( Fig. 5B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23515315" target="_blank">23515315</a>]<br></li>
<li>Highly conserved , positively charged residues protrude from the surface of Hha when in complex with H-NS ( 1 -- 46 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23515315" target="_blank">23515315</a>]<br></li>
<li>In a previous study carried out by de Alba et al. ( 52 ) , the authors systematically mutated each of the 10 conserved aspartate and glutamate residues of Hha to asparagine and glutamine , respectively , and evaluated the effects of the mutations on the Hha/H-NS interaction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23515315" target="_blank">23515315</a>]<br></li>
<li>Only two of the 10 Hha residues mutated , Glu-25 and Asp-48 , had significant diminishing effects on the Hha/H-NS association , and one of the mutants generated , E53Q , formed insoluble aggregates .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23515315" target="_blank">23515315</a>]<br></li>
<li>Consistent with these findings , the Hha/H-NS ( 1 -- 46 ) structure reveals that the negatively charged surface of Hha , containing residues Glu-25 and Asp-48 , takes part in the interaction interface with the H-NS ( 1 -- 46 ) dimer .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23515315" target="_blank">23515315</a>]<br></li>
<li>Examination of the structure revealed a cluster of conserved basic amino acids that protrude from the surface of Hha on the opposite side of the Hha/H-NS ( 1 -- 46 ) interface .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23515315" target="_blank">23515315</a>]<br></li>
<li>Conserved positively charged residues on the surface of Hha are positioned in the same orientation as the predicted orientation of the DNA-binding domain of H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23515315" target="_blank">23515315</a>]<br></li>
<li>Indeed , the solution structure of the YdgT protein from E. coli ( PDB code 2JQT ) indicates that YdgT lacks the short C-terminal - helix , equivalent to helix 4 in Hha , that interacts with the H-NS resi-due Arg-12 according to our Hha/H-NS ( 1 -- 46 ) complex structure ( 48 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23515315" target="_blank">23515315</a>]<br></li>
<li>Results : Two Hha monomers bind opposing faces of the H-NS N-terminal dimerization domain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23515315" target="_blank">23515315</a>]<br></li>
<li>To avoid the heterogeneous nature of the oligomerization properties of H-NS , multiple C-terminal H-NS truncation constructs were cloned into the vector pCDF1b , each of which contained the N-terminal dimerization domain of H-NS responsible for Hha binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23515315" target="_blank">23515315</a>]<br></li>
<li>We identified a mutation in the N-terminal dimerization domain of H-NS ( I11A ) that abolishes the Hha / H-NS interaction while retaining other properties of H-NS required for normal repression of the proV locus .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23515315" target="_blank">23515315</a>]<br></li>
<li>To address the mechanism of Hha-mediated gene silencing , we have determined the crystal structure of Hha from S. Typhi-murium in complex with the N-terminal dimerization domain of H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23515315" target="_blank">23515315</a>]<br></li>
<li>In this study , we present the 3.2 Å crystal structure of Hha in complex with the N-terminal domain of H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23515315" target="_blank">23515315</a>]<br></li>
<li>To further understand this process , we solved the x-ray crystal structure of Hha in complex with the N-terminal dimerization domain of H-NS ( H-NS ( 1 -- 46 ) ) to 3.2 Å resolution .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23515315" target="_blank">23515315</a>]<br></li>
<li>Structure of Hha in complex with the N-terminal dimerization domain of H-NS ( H-NS ( 1 -- 46 ) ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23515315" target="_blank">23515315</a>]<br></li>
<li>To expand upon these findings , we performed site-directed mutagenesis within the Hha recognition motif of H-NS helix 2 to isolate additional mutations that would abolish the Hha / H-NS interaction but would not otherwise compromise the function of H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23515315" target="_blank">23515315</a>]<br></li>
<li>Garcia et al. ( 30 ) reported that resi-dues within a conserved motif in helix 2 of the H-NS N-termi-nal dimerization domain were perturbed upon titration of Hha and that mutating H-NS residue Arg-12 to His ( H-NSR12H ) disrupted the Hha/H-NS interaction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23515315" target="_blank">23515315</a>]<br></li>
<li>Mapping of the Hha/H-NS interaction interface by two-dimensional heteronuclear single quantum correlation experiments revealed that Hha binds to the N-terminal dimerization domain of H-NS ( 29 , 30 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23515315" target="_blank">23515315</a>]<br></li>
<li>Hha-like proteins mimic the H-NS oligomerization domain ( 23 , 30 ) , and interaction of Hha with H-NS increases the repressive ability of H-NS ( 25 , 31 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18039769" target="_blank">18039769</a>]<br></li>
<li>Overall , Hha is a modestly basic protein but contains 11 acidic residues , 10 of which are highly conserved among the Hha family of proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23515315" target="_blank">23515315</a>]<br></li>
<li>Conserved Positively Charged Residues on the Surface of Hha Are Critical for Gene Silencing -- Sequence alignment of Hha and YdgT proteins from various Enterobactericiae indicates that several charged residues are highly conserved ( Fig. 5A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23515315" target="_blank">23515315</a>]<br></li>
<li>Positively charged residues of the surface of Hha are critical for function .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23515315" target="_blank">23515315</a>]<br></li>
<li>The solution structure of E. coli Hha determined by the Structural Genomics Consortium reveals that the protein consists of four - helices ( 43 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23515315" target="_blank">23515315</a>]<br></li>
<li>An electrostatic surface representation of the Hha solution structure shows that positively charged and negatively charged res-idues cluster on opposing surfaces of the molecule ( Fig. 5B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23515315" target="_blank">23515315</a>]<br></li>
				</div></ul><br>
			<label class="collapse_EVO" for="_3"><font size="5px">Hha evolution</font></label>
				<input id="_3" type="checkbox">
				<div class="EVO"><br><ul><li>A boxshade analysis ( 24 , 52 ) revealed high conservation of amino acid residues ranging from 50 % identical and 64 % similar ( RmoA ) to 99 % identical and 99 % similar ( E. coli Hha ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11673432" target="_blank">11673432</a>]<br></li>
<li>YmoA and Hha belong to an emerging family of transcriptional regulators that negatively modulate bacterial virulence gene expression ( 21 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16301528" target="_blank">16301528</a>]<br></li>
<li>We focused our attention on the small nucleoid-like protein Hha because of its amino acid similarity to YdgT and YmoA ( 7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17307861" target="_blank">17307861</a>]<br></li>
<li>YdgT shows 39 % identity to Yersinia YmoA and 35 % identity to Hha over the entire length of 71 aa ( Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16301528" target="_blank">16301528</a>]<br></li>
<li>In contrast , the R27 genes belonging to the second compartment , which includes mostly horizontally acquired genes , require a helper protein of the Hha family for their complete silencing by chromosomal H-NS , and can also be modulated by plasmidic H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19521501" target="_blank">19521501</a>]<br></li>
<li>The ability to interact with H-NS is conserved among other Hha family members , including YdgT and the Hha ortholog from Yersinia enterolitica , YmoA ( 26 , 28 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23515315" target="_blank">23515315</a>]<br></li>
<li>In this case , Hha synergistically acts with another member of the Hha/YmoA family , YdgT .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17675384" target="_blank">17675384</a>]<br></li>
<li>Overall , Hha is a modestly basic protein but contains 11 acidic residues , 10 of which are highly conserved among the Hha family of proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23515315" target="_blank">23515315</a>]<br></li>
<li>We screened the Salmonella genome using BLAST to identify putative regulatory genes that shared similarity to the YmoA Hha family of proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16301528" target="_blank">16301528</a>]<br></li>
<li>Several lines of evidence have shown that proteins belonging to the Hha family interact with members of the H-NS family to modulate gene expression ( 13 , 16 , 17 , 31 , 32 , 33 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18039769" target="_blank">18039769</a>]<br></li>
<li>The novel Hha / YmoA family of nucleoid-associated proteins : use of structural mimicry to modulate the activity of the H-NS family of proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17675384" target="_blank">17675384</a>]<br></li>
<li>Members of this family display the ability to use the H-NS protein to silence HGT regions with the help of co-repressors of the Hha family of proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19521501" target="_blank">19521501</a>]<br></li>
<li>The Hha protein is a member of a family of small nucleoid-associated histone-like proteins that include H-NS , HU1 , HU2 , and YmoA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11673432" target="_blank">11673432</a>]<br></li>
<li>Evidence for direct protein-protei interaction between members of the enterobacterial Hha / YmoA and H-NS families of proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15790293" target="_blank">15790293</a>]<br></li>
				</div></ul><br>
			<label class="collapse_RP" for="_4"><font size="5px">Processes regulated by Hha</font></label>
				<input id="_4" type="checkbox">
				<div class="RP"><br><ul><li>The Hha regulator also contributes to gene regulation despite playing a less relevant role than H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23554419" target="_blank">23554419</a>]<br></li>
<li>Hong , S. H. , Lee , J. & Wood , T. K. Engineering global regulator Hha of Escherichia coli to control bioﬁlm dispersal .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29057095" target="_blank">29057095</a>]<br></li>
<li>In addition to controlling Salmonella invasion , Hha plays an important role in repressing genes located on SPI2 , which are required for intracellular-growth ( 48 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17675384" target="_blank">17675384</a>]<br></li>
<li>Positive regulators include BarA/SirA , FliZ , EnvZ/OmpR , FadD , CsrB , HilD and some nucleoid binding proteins ( Fis , HU ) , while Pag , HilE , Ams and the nucleoid binding proteins Hha and H-NS are negatively regulating hilA transcription ( Altier et al. 2000a , b Fahlen et al. 2000 ; Lucas et al. 2000 ; Wilson et al. 2001 ; Schechter et al. 2003 ; Baxter et al. 2003 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18563621" target="_blank">18563621</a>]<br></li>
<li>The negative regulators , identiﬁed through transposon mutagenesis , include HilE , Ams , and Pag , whereas Hha , a small nucleoid-associated protein , was identiﬁed by introduction of a serovar Typhimurium chromosomal DNA gene bank into a serovar Typhimurium hilA : Tn5lacZY reporter strain ( 14 , 15 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16391141" target="_blank">16391141</a>]<br></li>
<li>It is also possible that Hha has a repressing effect on transcription of downstream invasion genes independent of hilA transcriptional repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11673432" target="_blank">11673432</a>]<br></li>
<li>Here , we identify Hha as the major repressor that silences SPI-2 virulence genes prior to bacteria sensing an activating environmental cue consisting of low Mg2 , low PO 3 4 , and acidiﬁcation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17307861" target="_blank">17307861</a>]<br></li>
<li>Importantly , the identiﬁcation of Hha as the major repressor of SPI-2 gene-expression makes available facile culture conditions under which to examine the SPI-2 regulon and identify potentially new genes that coregulate with this virulence system .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17307861" target="_blank">17307861</a>]<br></li>
<li>Fahlen et al. have shown that the histonelike protein Hha acts as a repressor of hilA ( 29 ) and have speculated that another repressor also plays an important role in modulating hilA expression in response to growth conditions ( 29 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19074398" target="_blank">19074398</a>]<br></li>
<li>Recent work indicates that the small DNA-binding protein , Hha , has a signiﬁcant role in repressing hilA transcription and the invasive phenotype , particularly in response to osmolarity signals .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12595445" target="_blank">12595445</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_11">
Nieto , J. M. , Madrid , C. , Prenafeta , A. , Miquelay , E. , Balsalobre , C. , Carrascal , M. , and Juárez , A. ( 2000 ) Expression of the hemolysin operon in Escherichia coli is modulated by a nucleoid-protein complex that includes the proteins Hha and H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23515315" target="_blank">23515315</a>]<br></label>
<input id="rp_similarity_11", type="checkbox">
<div class="RP_similar">
<li>Expression of the hemolysin operon in Escherichia coli is modulated by a nucleoid-protein complex that includes the proteins Hha and H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16443238" target="_blank">16443238</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17307861" target="_blank">17307861</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17675384" target="_blank">17675384</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18039769" target="_blank">18039769</a>]<br></li></div>
<li>Hha is a small nucleoid-associated protein ( 8 kDa ) involved in the negative modulation of virulence genes rather than of housekeeping genes in gram-negative bacteria ( 30 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17675384" target="_blank">17675384</a>]<br></li>
<li>Within their role as modulators of gene-expression , Hha and YdgT repress other genes in horizontally acquired regions in Salmonella including the pathogeni-city islands SPI-1 through SPI-5 and genes on the Sal-monella virulence plasmid [ 16 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21689395" target="_blank">21689395</a>]<br></li>
<li>Hha was originally shown to increase the cytoplasmic expression of hemolysin in Escherichia coli ( 18 ) and has also been shown to negatively regulate invasion genes in Salmonella ( 12 ) and virulence genes in the locus of enterocyte effacement in enterohemorrhagic E. coli ( 22 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17307861" target="_blank">17307861</a>]<br></li>
<li>Hha , originally defined as a hemolysin expression modulating protein in Esch-erichia coli ( 27 ) , negatively regulates SPI-1 virulence genes in Salmonella ( 5 ) and virulence genes in the locus of enterocyte effacement in enterohemorrhagic E. coli ( 28 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16301528" target="_blank">16301528</a>]<br></li>
<li>Silphaduang U , Mascarenhas M , Karmali M , Coombes BK : Repression of intracellular virulence factors in Salmonella by the Hha and YdgT nucleoid-associated proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21689395" target="_blank">21689395</a>]<br></li>
<li>A previous report describing the role of Hha in the transcriptional regulation of hilA , a positive activator of virulence genes in SPI-1 ( 12 ) , examined bacterial invasion into the ileal mucosae of anesthetized mice .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17307861" target="_blank">17307861</a>]<br></li>
<li>The main regulatory proteins involved in the expression of Spi-2 include the two-component systems SpiR ( SsrA ) / SsrB , PhoP/PhoQ , OmpR/EnvZ , as well as the DNA-binding proteins HilD , SlyA ( RovA in Yersinia ) , RcsB , the iron regulator Fur , and the nucleoid-associated proteins H-NS , Hha ( YmoA in Yersinia ) , YdgT , IHF , and Fis ( Fass and Groisman , 2009 ; Choi et al. , 2014 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26441883" target="_blank">26441883</a>]<br></li>
<li>In this report , we investigate the role of nucleoid-associated proteins Hha and H-NS in the expression of virulence activator proteins HilD , HilC , and RtsA with an emphasis on the interplay of positive and negative transcriptional factors in the regulation of the rtsA promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17675384" target="_blank">17675384</a>]<br></li>
<li>We now report the identiﬁcation and characterization of the role of the histone-like protein , Hha , in hilA transcription and expression of the Salmonella invasive phenotype .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11673432" target="_blank">11673432</a>]<br></li>
<li>As examples of promoters mapping in HGT DNA , we selected hilA , which controls the expression of the master regulator of the Salmonella pathogenicity island 1 ( SPI1 ) [ 25 ] , sensitive to H-NSR27 modulation in our transcriptomic study ( see Figure 1 ) , and the E. coli hly promoter , which regulates transcription of the operon encoding the toxin a-hemolysin and that has been shown to be modulated by H-NS and Hha [ 5 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19521501" target="_blank">19521501</a>]<br></li>
<li>In addition to the negative control that was exerted by H-NS/Hha under low-osmolarity conditions , the homologous virulence activators HilD , HilC , and RtsA ( Hil activators ) induced rtsA-lac expression in a high-salt medium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17675384" target="_blank">17675384</a>]<br></li>
<li>We noted that flagella distribution on the surface of Δhha ΔydgT ΔpefI-srgD quadruple mutants was less peritrichous and less abundant ( Figure 4C and Figure 4D ) than either wild type or ΔpefI-srgD suggesting that other regulators in addition to PefI-SrgD might be involved in regulating motility through the Hha and YdgT nucleoid-like proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21689395" target="_blank">21689395</a>]<br></li>
<li>that Hha and YdgT activate gene-expression in the wt strain ( see Tables S1 and S2 in the supplemental material ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18039769" target="_blank">18039769</a>]<br></li>
<li>The results reported here show that the Hha and YdgT proteins also repress ssrA and ssrB expression , thereby supporting the hypothesis that the Hha/H-NS complex modulates SPI2 gene-expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18039769" target="_blank">18039769</a>]<br></li>
<li>H-NS , the expression of Hha facilitates acquisition of foreign genes , thus contributing to pathogenic adaptation ( Aznar et al. , 2013 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29034217" target="_blank">29034217</a>]<br></li>
<li>Co-factors such as Hha and YdgT , and other nucleoid associated proteins like Fis contribute to the expression of horizontally acquired genes in Salmonella , including those that regulate virulence ( Schechter et al. , 2003 ; Silphaduang et al. , 2007 ; Vivero et al. , 2008 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29034217" target="_blank">29034217</a>]<br></li>
<li>Hha and YmoA are small basic proteins that have been shown to participate in the modulation of virulence gene regulation in different Gram-negative bacteria ( Madrid et al. , 2006 ; Stoebel et al. , 2008 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26441883" target="_blank">26441883</a>]<br></li>
<li>Essentially identical Hha proteins ( 1-amino-acid difference ) in E. coli and S. enterica serovar Typhimurium are functionally homologous , as S. enterica serovar Typhimurium Hha can repress expression of E. coli hemolysin .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11673432" target="_blank">11673432</a>]<br></li>
<li>Role of Nucleoid-Associated Proteins Hha and H-NS in Expression of Salmonella enterica Activators HilD , HilC , and RtsA Required for Cell Invasion Igor N. Olekhnovich * and Robert J. Kadner † icrobiology , University of Virginia School of Medicine , Charlottesville , Virginia 22908-0734 Received 8 June 2007/Accepted 23 July 2007 ent of M Departm The coordinate expression of Salmonella enterica invasion genes on Salmonella pathogenicity island 1 is under the control of the complex circuits of regulation that involve the AraC/XylS family transcriptional activators HilD , HilC , and RtsA and nucleoid-associated proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17675384" target="_blank">17675384</a>]<br></li>
<li>Repression of Intracellular Virulence Factors in Salmonella by the Hha and YdgT Nucleoid-Associated Proteins Umaporn Silphaduang ,1,2,3 Mariola Mascarenhas ,2 Mohamed Karmali ,2,3 and Brian K. Coombes1 ,2 * Department of Biochemistry and Biomedical Sciences and Antimicrobial Research Centre , McMaster University , Hamilton , ON , Canada1 ; Laboratory for Foodborne Zoonoses , Public Health Agency of Canada , Guelph , ON , Canada2 ; and Department of Molecular and Cellular Biology , University of Guelph , Guelph , ON , Canada3 Received 1 January 2007/Accepted 9 February 2007 The Hha/YmoA family of nucleoid-associated proteins is involved in gene regulation in enterobacteria .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17307861" target="_blank">17307861</a>]<br></li>
<li>Many regulatory proteins inﬂuence invasion gene-expression directly and indirectly , including activators HilA , InvF , HilD , HilC , and RtsA and nucleoid-associated proteins H-NS and Hha ( 2 , 14 , 16 , 23 , 36 , 37 , 44 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17675384" target="_blank">17675384</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_34">
Olekhnovich , I. N. , and Kadner , R. J. ( 2007 ) Role of nucleoid-associated proteins Hha and H-NS in expression of Salmonella enterica activators HilD , HilC , and RtsA required for cell invasion .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23515315" target="_blank">23515315</a>]<br></label>
<input id="rp_similarity_34", type="checkbox">
<div class="RP_similar">
<li>Olekhnovich , I. N. , and R. J. Kadner , 2007 Role of nucleoid-associated proteins Hha and H-NS in expression of Salmonella enterica activators HilD , HilC , and RtsA required for cell invasion .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22021388" target="_blank">22021388</a>]<br></li>
<li>Role of nucleoid-associated proteins Hha and H-NS in expression of Salmonella enterica activators HilD , HilC , and RtsA required for cell invasion .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23554419" target="_blank">23554419</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23459474" target="_blank">23459474</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27185788" target="_blank">27185788</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26441883" target="_blank">26441883</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21075923" target="_blank">21075923</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28660201" target="_blank">28660201</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20889744" target="_blank">20889744</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27601571" target="_blank">27601571</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25135218" target="_blank">25135218</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18441068" target="_blank">18441068</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25547794" target="_blank">25547794</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24488311" target="_blank">24488311</a>]<br></li>
<li>Olekhnovich IN , Kadner RJ : Role of nucleoid-associated proteins Hha and H-NS in expression of Salmonella enterica activators HilD , HilC , and RtsA required for cell invasion .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22017966" target="_blank">22017966</a>]<br></li>
<li>Olekhnovich , I. N. & Kadner , R. J. Role of nucleoid-associated proteins Hha and H-NS in expression of Salmonella enterica activators HilD , HilC , and RtsA required for cell invasion .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27886269" target="_blank">27886269</a>]<br></li>
<li>[ 33 ] I.N. Olekhnovich , R.J. Kadner , Role of nucleoid-associated proteins Hha and H-NS in expression of Salmonella enterica activators HilD , HilC , and RtsA required for cell invasion , J. Bacteriol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23040276" target="_blank">23040276</a>]<br></li></div>
<li>The bacterial nucleoid-associated proteins Hha and H-NS jointly repress horizontally acquired genes in Salmonella , including essential virulence loci encoded within Salmonella pathogenicity islands .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23515315" target="_blank">23515315</a>]<br></li>
<li>Hha regulates , among other genes , expression of Escherichia colihemolysin and esp operons ( 23 , 29 , 31 , 36 ) , the Salmonella enterica serovar Typhimurium hilA modulator of Salmonella pathogenicity island 1 ( SPI1 ) ( 14 , 32 ) , and virulence genes in SPI2 ( 37 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18039769" target="_blank">18039769</a>]<br></li>
<li>The Hha and YdgT proteins repress expression of SPI2 virulence genes ( 37 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18039769" target="_blank">18039769</a>]<br></li>
<li>Under nonpermissive conditions ( i.e. low temperature , low osmolarity and exponential growth phase ) , H-NS plays a crucial role in silencing hilA expression and Hha potentiates H-NS-mediated silencing .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21680637" target="_blank">21680637</a>]<br></li>
<li>These genes include those that encode several two-component regulatory systems ( PhoP/Q , PhoBR , OmpR/EnvZ and SirA/BarA ) , post-transcriptional systems ( CsrAB , RNase E , PNPase and Lon ) , nucleoid proteins ( HU , FIS , H-NS and Hha ) , signaling molecules ( ppGpp and NpnN ) , and other regulatory proteins ( FliZ , FadD and CpxA ) ( 6,7,19,21,22 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17329372" target="_blank">17329372</a>]<br></li>
<li>Transcription of hilA is subject to direct or indirect negative control by multiple nucleoid binding proteins , including Hha , H-NS , HU , and Fis ( 10 , 23 ) , and other proteins , including HilE , Pag , and Ams ( 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15126488" target="_blank">15126488</a>]<br></li>
<li>We assayed expression of the hilA : : lacZ fusion in a mutant strain lacking both chromosomal H-NS and Hha and housing a plasmid also deficient in plasmidic Hha ( strain SV5015 HAV ( R27Dhha ) ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19521501" target="_blank">19521501</a>]<br></li>
<li>The expression of hilA is regulated in response to variety of environmental signals , including pH ( 6 ) , growth phase ( 56 ) , osmolarity ( 36 ) , and oxygen ( 52 ) and by a number of proteins , including BarA/SirA ( 51 ) , CpxA ( 71 ) , CsrB ( RNA ) ( 2 ) , EnvZ/OmpR ( 63 ) , FadD ( 63 ) , Fis ( 94 ) , FliZ ( 63 ) , HilC ( 83 ) , HilD ( 83 ) , Hu ( 84 ) , Mlc ( 58 ) , and RtsA ( 28 ) , which positively regulate expression , and CsrA ( 2 ) , Hha ( 29 ) , HilE ( 7 ) , H-NS ( 84 ) , Lon ( 87 ) , PhoB ( 63 ) , and PhoP/PhoQ ( 9 ) , which negatively regulate expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19074398" target="_blank">19074398</a>]<br></li>
<li>By controlling expression/activity of HilD , HilC , RtsA , and HilA , many regulators contribute to SPI-1 gene regulation in response to environmental stimuli that include temperature ( H-NS and Hha ) , osmolarity ( H-NS and Hha ) , pH ( OmpR ) , iron availability ( Fur ) , and oxygen levels ( FNR ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27601571" target="_blank">27601571</a>]<br></li>
<li>Temperature , osmolarity and growth phase - dependent silencing of hilA by H-NS and Hha In the Enterobacteriaceae , H-NS and Hha are involved in thermo - and osmoregulation of several genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21680637" target="_blank">21680637</a>]<br></li>
<li>Nucleoid proteins Hha and H-NS bind to the promoters of SPI-1 regulator genes hilD , hilC , and rtsA to repress their expression and consequently regulate the expression of SPI-1 apparatus and effector genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23459474" target="_blank">23459474</a>]<br></li>
<li>Apart from H-NS and Hha the f actor for inversion stimulation ( Fis ) , also controls Spi-1 gene-expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26441883" target="_blank">26441883</a>]<br></li>
<li>Similar proteins , including YmoA ( 25 , 26 ) and Hha ( 5 , 28 ) , also perform negative regulatory roles for virulence gene-expression in pathogenic bacteria .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16301528" target="_blank">16301528</a>]<br></li>
<li>The negative regulators of SPI1 gene-expression , HilE and Hha , have been characterized .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15661008" target="_blank">15661008</a>]<br></li>
<li>Several negative regulators of invasion gene-expression have been identiﬁed , including HilE , Hha , and Lon protease .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15039320" target="_blank">15039320</a>]<br></li>
<li>Hha , YbaJ , and OmpA regulate Escherichia coli K12 bioﬁlm formation and conjugation plasmids abolish motility .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24735176" target="_blank">24735176</a>]<br></li>
<li>To avoid this complication , MBP-Hha was puri-ﬁed from E. coli cells lacking H-NS to produce H-NS-free MBP-Hha protein ( see Materials and Methods ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17675384" target="_blank">17675384</a>]<br></li>
<li>It has been proposed that 35 -- 37 Hha -- H-NS complex is required for E. coli haemo-lysin gene regulation in response to thermal and osmotic signals .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16443238" target="_blank">16443238</a>]<br></li>
				</div></ul><br>
			<label class="collapse_SIT" for="_5"><font size="5px">Hha binding site</font></label>
				<input id="_5" type="checkbox">
				<div class="SIT"><br><ul><li>The Hha protein binds to hilA upstream regulatory sequences .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11673432" target="_blank">11673432</a>]<br></li>
<li>The binding sites for HilD ( hatched box ) and H-NS/Hha ( open box , AT tract ) in PhilA region are shown .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19074398" target="_blank">19074398</a>]<br></li>
<li>Complicating this model are reports that Hha can independently bind to specific regulatory sequences within the hilA promoter from S. Typhimurium and the esp operon of enterohemorrhagic E. coli ( 22 , 23 , 32 , 33 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23515315" target="_blank">23515315</a>]<br></li>
<li>Thus , the transcriptional reporter assays suggest that essential sites for H-NS/Hha action are located in sequences K210 to K55 ( USR ) and C90 to C220 ( DSR ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16443238" target="_blank">16443238</a>]<br></li>
<li>Four nucleoid-like proteins in enteric bacteria , H-NS , StpA , Hha , and YdgT have a predilection for binding to A+T rich sequences and repress transcription of SPI-2 genes ( rev. in [ 14 ] ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19229334" target="_blank">19229334</a>]<br></li>
<li>SPI-2 regulation by SsrB and SlyA may be explained in a similar mechanism where both SlyA and SsrB counteract the silencing activity of H-NS/YdgT/Hha by competing for binding to the same target sequences or by altering the structure of the DNA to promote transcription for SPI-2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19229334" target="_blank">19229334</a>]<br></li>
<li>Mobility-shift assays revealed the presence of binding sites for the H-NS and Hha proteins , both upstream and downstream of the promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16443238" target="_blank">16443238</a>]<br></li>
<li>The presence of the ﬂanking sequences 40,46 greatly decreases hilA expression by providing sites for binding of silencing proteins , including H-NS and Hha .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16443238" target="_blank">16443238</a>]<br></li>
				</ul></div><br>
			<label class="collapse_TU" for="_6"><font size="5px">Transcription units regulated by Hha</font></label>
				<input id="_6" type="checkbox">
				<div class="TU"><br><ul><li>Hha may also repress hilA under low-osmolarity conditions ( 11 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12933899" target="_blank">12933899</a>]<br></li>
<li>Overexpression of Hha resulted in reduced hilA transcription ( Fig. 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21680637" target="_blank">21680637</a>]<br></li>
<li>Overexpression of Hha represses hilA transcription in cells growing in LB at 37 6C .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21680637" target="_blank">21680637</a>]<br></li>
<li>Collectively , these data provide evidence that Hha binds speciﬁcally to the hilA promoter , which we believe is the likely mechanism by which it represses hilA transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11673432" target="_blank">11673432</a>]<br></li>
<li>It is also possible that Hha has a repressing effect on transcription of downstream invasion genes independent of hilA transcriptional repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11673432" target="_blank">11673432</a>]<br></li>
<li>Hha is a negative modulator of hilA transcription , the Salmo-nella typhimurium invasion gene transcriptional activator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15765064" target="_blank">15765064</a>]<br></li>
<li>Not surprisingly , the hilD -- lac transcriptional and translational fusions were also regulated by Hha ( Figure 6C ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22021388" target="_blank">22021388</a>]<br></li>
<li>Thus , we conclude that Hha does not require HilD protein , but rather regulates both hilA and hilD ( as well as hilC and rtsA ) transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22021388" target="_blank">22021388</a>]<br></li>
<li>Repression of serovar Typhimurium hilA expression by Hha requires hilA URS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11673432" target="_blank">11673432</a>]<br></li>
<li>Additional repressors of hilA may include HilE , Pag , and Hha , as strains lacking these proteins exhibit increased hilA expression under inducing conditions ( 4 , 10 , 11 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12933899" target="_blank">12933899</a>]<br></li>
<li>To determine whether HilE , Pag , or Hha affects repression , rather than derepression , of hilA , the effect that hilE , pag , and hha mutations have on hilA expression should be examined under repressing environmental conditions in strains lacking HilC and HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12933899" target="_blank">12933899</a>]<br></li>
<li>Other SPI-2 regulators , such as YdgT , Hha and IHF , have been suggested to repress SPI-2 gene expression indirectly , since no binding of these proteins to ssrA and/or ssrB promoters has been demonstrated [ 21,24,25 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29751061" target="_blank">29751061</a>]<br></li>
<li>The negative regulators of SPI1 gene expression , HilE and Hha , have been characterized .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15661008" target="_blank">15661008</a>]<br></li>
<li>The Hha and YdgT proteins repress expression of SPI2 virulence genes ( 37 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18039769" target="_blank">18039769</a>]<br></li>
<li>Coregulation of gene expression by H-NS/Hha proteins was first evidenced for the E. coli hly operon [ 8 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19521501" target="_blank">19521501</a>]<br></li>
<li>This strain lacks the potent negative factor Hha and exhibits elevated hilA expression ( 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15126488" target="_blank">15126488</a>]<br></li>
<li>In addition to the negative control that was exerted by H-NS/Hha under low-osmolarity conditions , the homologous virulence activators HilD , HilC , and RtsA ( Hil activators ) induced rtsA-lac expression in a high-salt medium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17675384" target="_blank">17675384</a>]<br></li>
<li>H-NS/Hha negatively regulate the hilA and rtsA genes under low-osmolarity conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17675384" target="_blank">17675384</a>]<br></li>
<li>The data show that all three genes , hilD , hilC , and rtsA , were repressed by H-NS and/or Hha .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17675384" target="_blank">17675384</a>]<br></li>
<li>Our work shows that H-NS and Hha repress hilA under a set of well-defined environmental and physiological conditions and consequently account for SPI1 gene silencing .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21680637" target="_blank">21680637</a>]<br></li>
<li>For insertions surrounding hilA and hilD yet with an opposite orientation to these genes , it is likely that disruption of chromosomal architecture by the TPOP insertions imparted derepression of SPI-1 , a region whose repression relies upon extensive modiﬁcation of DNA topology and interactions with repressive proteins , such as Hha and H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27185788" target="_blank">27185788</a>]<br></li>
<li>Temperature , osmolarity and growth phase - dependent silencing of hilA by H-NS and Hha In the Enterobacteriaceae , H-NS and Hha are involved in thermo - and osmoregulation of several genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21680637" target="_blank">21680637</a>]<br></li>
<li>Nucleoid proteins Hha and H-NS bind to the promoters of SPI-1 regulator genes hilD , hilC , and rtsA to repress their expression and consequently regulate the expression of SPI-1 apparatus and effector genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23459474" target="_blank">23459474</a>]<br></li>
<li>Salmonella Hha is a small nucleoid-associated histone-like protein , that is 99 % identical to hha from E. coli , and can function interchangeably in E. coli to repress the expression of the hemolysin operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15765064" target="_blank">15765064</a>]<br></li>
<li>The notion that Hha and YdgT act through H-NS to modulate gene expression is supported by experimental evidence that Hha/H-NS complex formation is required for silencing of the hlyCABD operon in E. coli , which encodes the - hemolysin toxin ( 28 , 31 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23515315" target="_blank">23515315</a>]<br></li>
<li>The depletion of NaCl from LB medium lead to a significant decrease in rtsA-lac expression , which is restored for the hns mutant and is further induced for the hns hha double mutant ( Fig. 1 and 8 ) , suggesting that the repression under low-osmolarity conditions is mediated by H-NS/Hha .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17675384" target="_blank">17675384</a>]<br></li>
<li>However , in other systems , such as the E. coli ler operon and the S. enterica hilA promoter , puriﬁed Hha seems to be able to bind DNA independently of H-NS ( 16 , 36 , 45 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17675384" target="_blank">17675384</a>]<br></li>
<li>Thus , it seems that the repressing effect of Hha on hilA is transmitted to transcription of the invF , prgH , and sipC invasion genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11673432" target="_blank">11673432</a>]<br></li>
<li>Based on these observations , we suggest that Lrp acts as an antagonist of HilD , which induces expression of the hilA gene by derepressing the silencing effect of H-NS and Hha on the hilA promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19074398" target="_blank">19074398</a>]<br></li>
<li>As examples of promoters mapping in HGT DNA , we selected hilA , which controls the expression of the master regulator of the Salmonella pathogenicity island 1 ( SPI1 ) [ 25 ] , sensitive to H-NSR27 modulation in our transcriptomic study ( see Figure 1 ) , and the E. coli hly promoter , which regulates transcription of the operon encoding the toxin a-hemolysin and that has been shown to be modulated by H-NS and Hha [ 5 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19521501" target="_blank">19521501</a>]<br></li>
<li>The small nucleoidbinding protein Hha has been reported to bind the URS and repress hilA expression ( 4 , 11 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12933899" target="_blank">12933899</a>]<br></li>
<li>The mRNA levels of 471 genes were 2-fold higher in the hha ydgT mutant , indicating that Hha and YdgT repress gene expression in the wt strain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18039769" target="_blank">18039769</a>]<br></li>
<li>In exponential phase cells , the results obtained provided evidence that H-NS and Hha repress hilA expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21680637" target="_blank">21680637</a>]<br></li>
<li>The results reported here show that the Hha and YdgT proteins also repress ssrA and ssrB expression , thereby supporting the hypothesis that the Hha/H-NS complex modulates SPI2 gene expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18039769" target="_blank">18039769</a>]<br></li>
				</div></ul></div><br>
	</div>
</body>
</html>