<!doctype html>
<style>
@charset "utf-8";
body {
	height: 100%;
	margin: 0;
	padding: 0;
}

.banner {
	position: fixed;
	top: 0;
	width: 100%;
	height: 65px;
	background: gray;
	line-height: 65px;
}
.spacer {
	height: 10px;
}

.main-content {
	padding: 5%;
}

.collapse_ACT {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-style: none solid none none;
	border-color: #B4045F;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_ACT + input{
	display: none;
}
.collapse_ACT + input + div{
	display: none;
}
.collapse_ACT + input:checked + div{
	display: block;
}
.ACT {
	text-align: justify;
	text-justify: inter-word;
}
.ACT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B4045F;
}
.ACT_collapse_similarity:hover{
	background: #e5d3dc;
}
.ACT_collapse_similarity + input {
	display: none;
}
.ACT_collapse_similarity + input + div{
	display: none;
}

.ACT_collapse_similarity + input:checked + div{
	display: block;
}
.ACT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.ACT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B4045F;
	padding-left: 5px;
}
.ACT_collapse_similarity:hover{
	background: #e5d3dc;
}
.ACT_collapse_similarity + input {
	display: none;
}
.ACT_collapse_similarity + input + div{
	display: none;
}
.ACT_collapse_similarity + input:checked + div{
	display: block;
}
.ACT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.ACT_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #B4045F;
}
.ACT_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_DOM {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-style: none solid none none;
	border-color: #B404AE;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_DOM + input{
	display: none;
}
.collapse_DOM + input + div{
	display: none;
}
.collapse_DOM + input:checked + div{
	display: block;
}
.DOM {
	text-align: justify;
	text-justify: inter-word;
}
.DOM_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B404AE;
	padding-left: 5px;
}
.DOM_collapse_similarity:hover{
	background: #e8d7e7;
}
.DOM_collapse_similarity + input {
	display: none;
}
.DOM_collapse_similarity + input + div{
	display: none;
}
.DOM_collapse_similarity + input:checked + div{
	display: block;
}
.DOM_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}.DOM_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #B404AE;
}
.DOM_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_EVO {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #7401DF;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_EVO + input{
	display: none;
}
.collapse_EVO + input + div{
	display: none;
}
.collapse_EVO + input:checked + div{
	display: block;
}
.EVO {
	text-align: justify;
	text-justify: inter-word;
}
.EVO_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #7401DF;
	padding-left: 5px;
}
.EVO_collapse_similarity:hover{
	background: #e2d7ed;
}
.EVO_collapse_similarity + input {
	display: none;
}
.EVO_collapse_similarity + input + div{
	display: none;
}
.EVO_collapse_similarity + input:checked + div{
	display: block;
}
.EVO_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.EVO_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #7401DF;
}
.EVO_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_RP {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #0174DF;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_RP + input{
	display: none;
}
.collapse_RP + input + div{
	display: none;
}
.collapse_RP + input:checked + div{
	display: block;
}
.RP {
	text-align: justify;
	text-justify: inter-word;
}
.RP_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #0174DF;
	padding-left: 5px;
}
.RP_collapse_similarity:hover{
	background: #c9d7e2;
}
.RP_collapse_similarity + input {
	display: none;
}
.RP_collapse_similarity + input + div{
	display: none;
}
.RP_collapse_similarity + input:checked + div{
	display: block;
}
.RP_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.RP_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #0174DF;
}
.RP_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_SIT {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #0489B1;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_SIT + input{
	display: none;
}
.collapse_SIT + input + div{
	display: none;
}
.collapse_SIT + input:checked + div{
	display: block;
}
.SIT {
	text-align: justify;
	text-justify: inter-word;
}
.SIT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #0489B1;
	padding-left: 5px;
}
.SIT_collapse_similarity:hover{
	background: #d0dfe2;
}
.SIT_collapse_similarity + input {
	display: none;
}
.SIT_collapse_similarity + input + div{
	display: none;
}
.SIT_collapse_similarity + input:checked + div{
	display: block;
}
.SIT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.SIT_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #0489B1;
}
.SIT_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_TU {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #04B486;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_TU + input{
	display: none;
}
.collapse_TU + input + div{
	display: none;
}
.collapse_TU + input:checked + div{
	display: block;
}
.TU {
	text-align: justify;
	text-justify: inter-word;
}
.TU_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #04B486;
	padding-left: 5px;
}
.TU_collapse_similarity:hover{
	background: #d0e2de;
}
.TU_collapse_similarity + input {
	display: none;
}
.TU_collapse_similarity + input + div{
	display: none;
}
.TU_collapse_similarity + input:checked + div{
	display: block;
}
.TU_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.TU_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #04B486;
}
.TU_similar > a {
	text-decoration: none;
	color: gray;
}

ul { 
	list-style-type: none;
	padding: 0;
	margin: 0;
}
ul li > a {
	text-decoration: none;
	color: gray;
}
ul li:hover, ul li:focus {
	background: #F0F0F0;
}

ul li:first-child span {
  border-left:none;  
}

</style>
<html>
<head>
<meta charset="utf-8">
<title>TF properties - HilC</title>
<meta name="theme-color" content="#ffffff">
</head>
<body>
	<div class="banner">
		<center><font color="white" size="16px" face="arial">HilC</font></center>
	</div>
	<div align="center" class="main-content">
		<div class="spacer">
	</div><br>			<label class="collapse_ACT" for="_1"><font size="5px">HilC transcriptional activity</font></label>
				<input id="_1" type="checkbox">
				<div class="ACT"><br><ul><li>Due to their weak affinity , neither HilC nor RtsA will occupy this site under physiological conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20686667" target="_blank">20686667</a>]<br></li>
<li>Thus , neither RtsA nor HilC are required for OmpR-mediated regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16045614" target="_blank">16045614</a>]<br></li>
<li>A second , related SPI1 transcriptional-regulator , HilC , was uninvolved in this repression , although hilC has been demonstrated to have only mild effects on Salmonella invasion ( Schechter et al. , 1999 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23289537" target="_blank">23289537</a>]<br></li>
<li>The data in Fig. 2B indicate that HilE speciÔ¨Åcally binds HilD but not HilC or RtsA in vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29378886" target="_blank">29378886</a>]<br></li>
<li>However , it was possible that PhoPQ regulates hilA through HilC or RtsA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22021388" target="_blank">22021388</a>]<br></li>
<li>These results demonstrate that repression is not HilC or RtsA dependent .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20190046" target="_blank">20190046</a>]<br></li>
<li>Arabinose is required for induction of pHilC but was included in all cultures .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20889744" target="_blank">20889744</a>]<br></li>
<li>Again , when hilD was deleted in these strains , Fur regulation of hilD was abrogated , while overproduction of HilC still induced hilD expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
<li>Blackwell Science , LtdOxford , UKMMIMolecular Microbiology 0950-382X Blackwell Publishing Ltd , 200347Original ArticleHilD and HilC directly activate invF expressionS.Akbar , L.M.Schechter , C.P.Lostroh and C.A.Lee Molecular Microbiology ( 2003 ) 47 ( 3 ) , 715 -- 728 AraC/XylS family members , HilD and HilC , directly activate virulence gene expression independently of HilA in Salmonella typhimurium Samina Akbar , ‚Ä† Lisa M. Schechter , ‚Ä° C. Phoebe Lostroh ¬ß and Catherine A. Lee * Department of Microbiology and Molecular Genetics , Harvard Medical School , 200 Longwood Avenue , Boston , MA 02115 , USA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_10">
Akabar , S. , Schechter , L.M. , Lostroh , C.P. and Lee , C.A. ( 2003 ) AraC/XylS family members , HilD and HilC , directly activate virulence gene expression independently of HilA in Salmonella typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17329372" target="_blank">17329372</a>]<br></label>
<input id="act_similarity_10", type="checkbox">
<div class="ACT_similar">
<li>References Akbar , S. , Schechter , L.M. , Lostroh , C.P. , and Lee , C.A. ( 2003 ) AraC/XylS family members , HilD and HilC , directly activate virulence gene expression independently of HilA in Salmonella typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16999831" target="_blank">16999831</a>]<br></li>
<li>Akbar , S. , Schechter , L.M. , Lostroh , C.P. and Lee , C.A. ( 2003 ) AraC/XylS family members , HilD and HilC , directly activate virulence gene expression independently of HilA in Salmonella typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28335027" target="_blank">28335027</a>]<br></li>
<li>Akbar , S. , Schechter , L.M. , Lostroh , C.P. , and Lee , C.A. ( 2003 ) AraC/XylS family members , HilD and HilC , directly activate virulence gene expression independently of HilA in Salmonella typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16045614" target="_blank">16045614</a>]<br></li>
<li>Akbar , S. , L. M. Schechter , C. P. Lostroh , and C. A. Lee , 2003 AraC/XylS family members , HilD and HilC , directly activate virulence gene expression independently of HilA in Sal-monella typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25991823" target="_blank">25991823</a>]<br></li>
<li>References Akbar , S. , Schechter , L. , Lostroh , C. , and Lee , C. ( 2003 ) AraC/XylS family members , HilD and HilC , directly activate virulence gene expression independently of HilA in Salmo-nella typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15661008" target="_blank">15661008</a>]<br></li>
<li>LITERATURE CITED Akbar , S. , L. M. Schechter , C. P. Lostroh and C. A. Lee , 2003 AraC/XylS family members , HilD and HilC , directly activate virulence gene expression independently of HilA in Salmo-nella typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20008574" target="_blank">20008574</a>]<br></li></div>
				<label class="ACT_collapse_similarity" for="act_similarity_17">
AraC/XylS family members , HilD and HilC , directly activate virulence gene expression independently of HilA in Salmonella typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23554419" target="_blank">23554419</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18723621" target="_blank">18723621</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17600486" target="_blank">17600486</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14645287" target="_blank">14645287</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21168230" target="_blank">21168230</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16443238" target="_blank">16443238</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15790293" target="_blank">15790293</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15968047" target="_blank">15968047</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19074398" target="_blank">19074398</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923082" target="_blank">12923082</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14679226" target="_blank">14679226</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15126488" target="_blank">15126488</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17675384" target="_blank">17675384</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15731035" target="_blank">15731035</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25135218" target="_blank">25135218</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15039320" target="_blank">15039320</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18441068" target="_blank">18441068</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15516559" target="_blank">15516559</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18160484" target="_blank">18160484</a>]<br></label>
<input id="act_similarity_17", type="checkbox">
<div class="ACT_similar">
<li>AraC / XylS family members , HilD and HilC , directly activate virulence gene expression independently of HilA in Salmonella typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15256548" target="_blank">15256548</a>]<br></li>
<li>AraC/XylS family members , HilD and HilC , directly activate virulence gene expression independently of HilA in Salmonella typhi-murium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28329249" target="_blank">28329249</a>]<br></li>
<li>AraC/XylS family members , HilD and HilC , directly activate virulence gene expression independently of HilA in Salmonella Typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24375101" target="_blank">24375101</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27601571" target="_blank">27601571</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27185788" target="_blank">27185788</a>]<br></li></div>
<li>The added proteins in each lane are as follows : lane 2 , no added protein ; lane 3 , 150 nM HilC ; lane 4 , 300 nM HilC ; lane 5 , 150 nM HilD ; lane 6 , 300 nM HilD ; lane 7 , 150 nM MBP-RtsA ; lane 8 , 300 nM MBP-RtsA FIG. 8 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17675384" target="_blank">17675384</a>]<br></li>
<li>The proteins added in each lane are as follows : lanes 1 , no added protein ; lanes 2 and 5 , 75 nM HilC ; lanes 3 and 6 , 75 nM HilD ; lanes 4 and 7 , 75 nM MBP-RtsA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17675384" target="_blank">17675384</a>]<br></li>
<li>The added proteins in each lane are as follows : lanes 1 , no added protein ; lanes 2 , 50 nM HilC ; lanes 3 , 100 nM HilC ; lanes 4 , 50 nM HilD , and lanes 5 , 100 nM HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12107132" target="_blank">12107132</a>]<br></li>
<li>It is possible that HilD , but not HilC , activates invF expression from a second promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>In our study , HilC expression was not influenced by E. coli supernatants , while HilD expression was slightly enhanced .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21379575" target="_blank">21379575</a>]<br></li>
<li>This discrepancy makes unlikely that HilC is responsible for transmission of L-arabinose regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25991823" target="_blank">25991823</a>]<br></li>
<li>Furthermore , substitutions at 10 positions allowed increased binding of HilC , but changes at only one position , 83T to G , showed increased binding of HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12107132" target="_blank">12107132</a>]<br></li>
<li>However , when hilD was deleted in these strains , there was no longer a signiÔ¨Åcant effect of Fur overproduction , while HilC overproduction still induced hilD transcription at a level that was approximately the same as the level seen in the hilD background .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
<li>On the other hand , in response to the deletion mutation in hilD , model 3 predicts a 2-fold decrease in HilC expression , which is comparable to the experimental data [ Ellermeier et al. , 2005 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19537165" target="_blank">19537165</a>]<br></li>
<li>The simulation results for model 5 predict a complete loss in HilC levels in response to the hilD knock-out , contradicting the experimental data which shows a 1.5-fold decrease in HilC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19537165" target="_blank">19537165</a>]<br></li>
<li>Thus , this 7-fold effect on HilD seen in this experiment could easily account for the 20-fold effect seen in the wild-type background , despite the limited inhibition of RtsA or HilC activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26884427" target="_blank">26884427</a>]<br></li>
<li>The simulation results for model 6 predict a complete loss in HilC expression in response to the knock-out of hilD as well as the knock-out of sirA , contradicting the experimental data .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19537165" target="_blank">19537165</a>]<br></li>
<li>On the other hand , in response to the knock-out of sirA , it predicts a 3-fold decrease in HilC , which is comparable to the experimental observation of a 2-fold decrease in HilC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19537165" target="_blank">19537165</a>]<br></li>
<li>The 5 end of the in vitro hilC transcript produced in the absence of HilC was mapped to position 71 , relative to the start used in the presence of HilC ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12107132" target="_blank">12107132</a>]<br></li>
<li>Since model 7 was successful in predicting the levels of HilA as well as those of the intermediate components ( HilC and RtsA ) in the network , this was combined with the mechanism proposed by Teplitski et al. , 2003 , in scenario-1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19537165" target="_blank">19537165</a>]<br></li>
<li>Therefore the model predicts that a very strong overexpression of HilC -- or symmetrically RtsA -- should disrupt the bistability of the system , by pushing the entire system in an activated state , either in presence or absence of the repression from HilE .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21107576" target="_blank">21107576</a>]<br></li>
<li>S1L ) strongly supports the idea of direct regulation by HilC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27601571" target="_blank">27601571</a>]<br></li>
<li>Repression of PhilD was strongest at the lower concentration of novobiocin ( 15 mg/ml ) , raising the possibility that a high degree of DNA relaxation reduces repression by OmpR , perhaps through elevated HilC levels brought about by DNA relaxation ( Figure 1A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22457642" target="_blank">22457642</a>]<br></li>
<li>The simulation results for model 1 indicate a complete loss of expression of HilC when hilD is knocked out , which is in contrast to the experimental data where deletion of hilD leads to a 1.5-fold decrease in HilC [ Ellermeier et al. , 2005 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19537165" target="_blank">19537165</a>]<br></li>
<li>It is interesting to observe that for small values of Co ( Co = 0.2 ) the bistability is preserved , while increasing the external production rate of HilC we get only one positive real solution , also in absence of HilE ( T = 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21107576" target="_blank">21107576</a>]<br></li>
<li>It is possible that HilC mediated repression of SPI1 may also contribute to the overall activity of naringenin .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21168230" target="_blank">21168230</a>]<br></li>
<li>It is not known if HilE also controls-the-activity of either RtsA or HilC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16045614" target="_blank">16045614</a>]<br></li>
<li>This is consistent with the feedforward loop model ; when neither RtsA nor HilC are present , regulation of hilA is dampened , even though the primary signal is mediated through HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16045614" target="_blank">16045614</a>]<br></li>
<li>Four other regulated transfrags , three regulated by RtsB and one by HilC , are immediately upstream of genes not signiÔ¨Åcantly regulated according to our standard RNA-seq analysis ( see Table S4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27601571" target="_blank">27601571</a>]<br></li>
<li>Whole-cell extracts were prepared from strains of c3306 , CS2022 , CS2722 , CS2723 , CS2724 , CS2725 , CS2802 and CS2803 grown under the SPI1-activating conditions and then separated in a 10 % SDS-polyacrylamide gel.The separated proteins were transferred to PVDF membranes and then immunostained with antiHilA , anti-SipC , anti-InvF , anti-HilC or anti-HilD serum .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15661008" target="_blank">15661008</a>]<br></li>
<li>The proteins were transferred to PVDF membranes and then immun-ostained with anti-HilA , anti-SipC , anti-InvF , anti-HilC or anti-HilD serum .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15661008" target="_blank">15661008</a>]<br></li>
<li>These data suggest that both HilD and HilC act as simple repressors when bound to this synthetic promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29378886" target="_blank">29378886</a>]<br></li>
<li>Here we provide genetic and biochemical data showing that HilE speciÔ¨Åcally binds to HilD but not to HilC or RtsA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29378886" target="_blank">29378886</a>]<br></li>
<li>It is possible that RtsA , HilC and HilD each respond to an independent signal to activate SPI1 , but what these signals are is not clear .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16045614" target="_blank">16045614</a>]<br></li>
<li>This result suggests that HilC forcefully responds to the environmental stimuli regulating hilA expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15661008" target="_blank">15661008</a>]<br></li>
<li>Still , it is intriguing to consider that under certain conditions HilC and HilD may be able to activate hilA expression and that under other conditions they may be able to increase hilA expression by derepressing hilA transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12933899" target="_blank">12933899</a>]<br></li>
<li>The fact that hilA expression is regulated by L-arabinose might indicate either that L-arabinose directly regulates hilA or that the regulatory input is transmitted to hilA via HilC and/or HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25991823" target="_blank">25991823</a>]<br></li>
<li>To build a construct in which HilD or HilC acts as a simple transcriptional-repressor , we synthesized four pairs of complementary oligonucleotides .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29378886" target="_blank">29378886</a>]<br></li>
<li>Similar observations were made for sipB , which remained under Dam methylation control in HilA , HilC , RtsA , and InvF backgrounds .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20008574" target="_blank">20008574</a>]<br></li>
<li>The data in Fig. 5 suggest that both HilD and HilC homodimerize , as evidenced by an IPTG-dependent decrease in - galactosidase activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29378886" target="_blank">29378886</a>]<br></li>
<li>In this model , HilD or HilC expression and activity would be induced in vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>Anti-HilA , - HilC , - HilD , and - SipC antisera were previously established in our laboratory ( 60 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18245288" target="_blank">18245288</a>]<br></li>
<li>Model 5 In this model , SirA regulates HilD , which in turn regulates HilC , RtsA and HilA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19537165" target="_blank">19537165</a>]<br></li>
<li>, HilD , HilC , RtsA and HilA ) would still be in an active state .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24079299" target="_blank">24079299</a>]<br></li>
<li>Our results conÔ¨Årm these observations and provide additional data for low osmolarity conditions , in which HilD and HilC activators were not essential in mutants lacking H-NS and Hha proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16443238" target="_blank">16443238</a>]<br></li>
<li>In an analogous fashion , Dam-dependent regulation of invF was still observed in HilA , HilC , and RtsA backgrounds , and no information was obtained in a HilD background ( Figure 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20008574" target="_blank">20008574</a>]<br></li>
<li>Various in vitro and in vivo conditions must be tested to determine whether direct activation of invF by HilC and/or HilD occurs in wild-type bacteria and whether this has any relevance in vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10844688" target="_blank">10844688</a>]<br></li>
<li>HilC and RtsA can independently induce HilA expression [ 16 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20686667" target="_blank">20686667</a>]<br></li>
<li>In contrast to invF-lacZ expression from pVV448 , HilD and HilC activate invF-lacZ expression over 45-fold from pSA7 ( Fig. 6 , black and striped bars , pSA7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>HilC and HilD are thought to-bind directly at this site to derepress the hilA promoter under activating conditions ( Schechter and Lee , unpublished results ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11292791" target="_blank">11292791</a>]<br></li>
<li>When present , HilC or HilD binds to this same site and promotes hilA expression ( 63 ; Schechter and Lee , unpublished results ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11292791" target="_blank">11292791</a>]<br></li>
<li>In scenario-3 , SirA is considered to activate HilA and HilC , with HilD as a co-activator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19537165" target="_blank">19537165</a>]<br></li>
<li>In order to address whether additional S. typhimuriumspeciÔ¨Åc genes are required for HilD and HilC to activate invF , we examined both pVV448 and pSA7 reporters in E. coli TOP-10 cells .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>HilC and RtsA are thought to activate themselves ( positive feedback ) as well each other ( cross activation ) in addition to their effect on HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19003447" target="_blank">19003447</a>]<br></li>
<li>HilD , HilC and RtsA are known to form a feed-forward loop for SPI-1 regulation [ 29 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28575106" target="_blank">28575106</a>]<br></li>
<li>In response to different stimuli , HilD activates HilC and RtsA transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21680637" target="_blank">21680637</a>]<br></li>
<li>Namely , to what degree are HilC , RtsA , and HilD interchangeable ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20686667" target="_blank">20686667</a>]<br></li>
<li>RtsA , HilC and HilD all contribute to SPI1 activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16045614" target="_blank">16045614</a>]<br></li>
<li>RtsA , HilC and HilD each contribute to activating hilA in vitro The proposed model predicts that HilD , HilC and RtsA each have a role in activating hilA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16045614" target="_blank">16045614</a>]<br></li>
<li>However , HilD , HilC and RtsA normally act in concert to activate hilA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16045614" target="_blank">16045614</a>]<br></li>
<li>By controlling expression/activity of HilD , HilC , RtsA , and HilA , many regulators contribute to SPI-1 gene regulation in response to environmental stimuli that include temperature ( H-NS and Hha ) , osmolarity ( H-NS and Hha ) , pH ( OmpR ) , iron availability ( Fur ) , and oxygen levels ( FNR ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27601571" target="_blank">27601571</a>]<br></li>
<li>Three AraC-like transcription factors -- HilC , HilD , and RtsA -- positively regulate HilA expression in response to various intracellular and environmental signals ( 5 , 7 , 17 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20190046" target="_blank">20190046</a>]<br></li>
<li>HilD , HilC , and RtsA not only regulate shared target genes but also activate each other 's transcription ( 8 , 10 , 11 , 20 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27601571" target="_blank">27601571</a>]<br></li>
<li>RtsA , HilD , and HilC directly activate dsbA , which encodes a periplasmic disulÔ¨Åde bond oxidoreductase ( 16 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20889744" target="_blank">20889744</a>]<br></li>
<li>HilC and HilD directly bind and activate invF-1 invF-2 P ( Schechter et al. , 1999 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15661008" target="_blank">15661008</a>]<br></li>
<li>In addition , HilD and HilC directly activate invF , sipA and sipC in non-HilA dependent manner ( Akbar et al. , 2003 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21168230" target="_blank">21168230</a>]<br></li>
<li>HilD and HilC positively regulate sprB , most likely cotrans-criptionally with hilC itself , since hilC and sprB are cotranscribed ( see Fig .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27601571" target="_blank">27601571</a>]<br></li>
<li>To this end , we puriÔ¨Åed HilC and HilD proteins that were active for DNA binding and transcriptional regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12107132" target="_blank">12107132</a>]<br></li>
<li>The genes slrP and dsbA are also induced by HilC , HilD and RtsA independently of both HilA and InvF [ 25,28 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17208038" target="_blank">17208038</a>]<br></li>
<li>The levels of induction by HilD , HilC , and RtsA were similar for all genes tested ( c.f. Fig. 2 and Fig. 3B and C ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24375101" target="_blank">24375101</a>]<br></li>
<li>Through a complex feedback and feedforward mechanism , HilC and HilD control their own and each other 's transcription , and together activate transcription of hilA [ 14,15,16 ] ( Figure 1A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22457642" target="_blank">22457642</a>]<br></li>
<li>HilD , HilC , and RtsA all activate transcription of hilA , hilC , hilD , dsbA , invF , slrP , rtsA , and genes in SPI-4 ( 6 , 15 , 32 , 33 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24375101" target="_blank">24375101</a>]<br></li>
<li>In fact , there is evidence that SipA and SipC HilD and HilC directly activate invF expression 723 can be secreted independently of SPI1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>We propose that HilD and HilC can independently activate invF expression and InvF production , which directly and indirectly increases transcription of the sipA and sipC genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>Discussion HilD and HilC directly activate invF expression by binding upstream and downstream of a HilD/C-dependent promoter Previous studies have shown that HilD and HilC can indirectly activate invF expression by derepressing hilA ( Schechter et al. , 1999 ; Schechter and Lee , 2001 ; Olekhnovich and Kadner , 2002 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>These results are consistent with previous reports indicating that HilA , HilC , and HilD regulate SPI1 gene transcription and that HilC and HilD activate hilA transcription equally ( 3 , 7 , 8 , 9 , 10 , 11 , 19 , 33 , 34 , 37 , 40 , 41 , 47 , 49 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18441068" target="_blank">18441068</a>]<br></li>
<li>Under high-osmolarity conditions , HilD , HilC , and RtsA activate SPI1 genes directly and indirectly through hilA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17675384" target="_blank">17675384</a>]<br></li>
<li>HilC and HilD can independently bind and activate the hilA promoter ( Schechter et al. , 1999 ; Schechter and Lee , 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15661008" target="_blank">15661008</a>]<br></li>
<li>The SPI-1 locus encodes effectors and regulators , such as HilA , HilD , HilC , and InvF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20028809" target="_blank">20028809</a>]<br></li>
<li>SPI-1 also encodes transcriptional-activators responsible for its own expression : HilA , HilC , HilD , and InvF ( Lostroh and Lee 2001 ; Ellermeier and Slauch 2007 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25991823" target="_blank">25991823</a>]<br></li>
<li>, HilD , HilC and RtsA [ Schechter and Lee , 2001 ; Olekhnovich and Kadner , 2002 ; Ellermeier et al. , 2005 ] , of which HilD and HilC are encoded on SPI-1 [ Ellermeier and Slauch , 2003 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19537165" target="_blank">19537165</a>]<br></li>
<li>These SPI1 proteins include HilC , HilD , InvF , and RtsA ( Eichelberg and GalaÃÅn , 1999 ; Ellermeier and Slauch , 2004 ; Lostroh and Lee , 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15790293" target="_blank">15790293</a>]<br></li>
<li>In this case , HilD was subjected to positive regulation by four nodes ( SirA-BirA , HilC , RtsA and Fur ) along with self activation [ 11,14,43,44 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24079299" target="_blank">24079299</a>]<br></li>
<li>Thus , our data indicate cross talk between SPI-1 and SPI-2 regulation mediated by HilC and SprB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27601571" target="_blank">27601571</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_100">
As shown in Table 2 , hilA expression was induced 40-fold by RtsA , 20-fold by HilD , and 120-fold by HilC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14679226" target="_blank">14679226</a>]<br></label>
<input id="act_similarity_100", type="checkbox">
<div class="ACT_similar">
<li>Figure 7B shows that expression of hilA was induced 40-fold by RtsA , 20-fold by HilD , and 120-fold by HilC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923082" target="_blank">12923082</a>]<br></li></div>
<li>HilA expression , in turn , is regulated by three AraC-like transcription factors - HilC , HilD , and RtsA -- with homologous DNA binding domains [ 22,31,32 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20686667" target="_blank">20686667</a>]<br></li>
<li>HilA , in turn , is regulated by HilC , HilD , and RtsA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20686667" target="_blank">20686667</a>]<br></li>
<li>The key regulator of this secretion system is HilA , which is in turn regulated by HilD , HilC and RtsA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19537165" target="_blank">19537165</a>]<br></li>
<li>Finally , HilA is regulated by dimers of HilC , HilD and RtsA through an OR operation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19537165" target="_blank">19537165</a>]<br></li>
<li>The master regulator of SPI-1 , HilA , is itself regulated by three homologous transcription factors , HilD , HilC , and RtsA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24375101" target="_blank">24375101</a>]<br></li>
<li>We propose that siiA is regulated by HilD , HilC , RtsA , HilA , and SprB , which would indicate that SPI-1 and SPI-4 gene expression is tightly linked .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24375101" target="_blank">24375101</a>]<br></li>
<li>It is certainly possible that other genes are directly regulated by HilC , HilD and RtsA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17208038" target="_blank">17208038</a>]<br></li>
<li>We propose that DapZ is regulated by both RtsA and HilC but not HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27601571" target="_blank">27601571</a>]<br></li>
<li>The rtsAB operon is transcriptionally activated by HilD , HilC , and RtsA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24488311" target="_blank">24488311</a>]<br></li>
<li>This transfrag is regulated by HilD and HilC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27601571" target="_blank">27601571</a>]<br></li>
<li>Hence , regulation by HilD , HilC , and RtsA must be direct .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24375101" target="_blank">24375101</a>]<br></li>
<li>That means that at least hilA and rtsA are induced by HilD , HilC , and RtsA proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17675384" target="_blank">17675384</a>]<br></li>
<li>hilA expression is induced by three transcriptional-activators , HilC , HilD and RtsA [ 70 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25375226" target="_blank">25375226</a>]<br></li>
<li>SPI-1 expression is triggered by multiple environmental signals and is regulated by the SPI-1-encoded HilC and HilD proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24603618" target="_blank">24603618</a>]<br></li>
<li>SPI1 gene expression is positively regulated by the transcriptional-activators HilA , HilC , and HilD , encoded within SPI1 , and negatively regulated by the regulators HilE and PhoP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18160484" target="_blank">18160484</a>]<br></li>
<li>HilA expression is also regulated by the regulators HilC and HilD [ 29 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21379575" target="_blank">21379575</a>]<br></li>
<li>ssaG and ssaJ are regulated by HilC , and sifB is regulated by SprB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27601571" target="_blank">27601571</a>]<br></li>
<li>In turn , hilA is regulated by HilD and HilC ( Schechter and Lee , 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21168230" target="_blank">21168230</a>]<br></li>
<li>HilC can bind to fragments carrying only the upstream A1 or only the downstream A2 site , but HilD bound well only to the A1 region and not to A2 alone .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12107132" target="_blank">12107132</a>]<br></li>
<li>Thus , HilC and RtsA are also apparently indirectly regulating hilD , contributing to the feedforward loop .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17208038" target="_blank">17208038</a>]<br></li>
<li>Thus , although HilC and HilD proteins can bind to the same DNA target , they have different sequence requirements for DNA recognition .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12107132" target="_blank">12107132</a>]<br></li>
<li>HilC , HilD and RtsA are central activators of hilA expression ( Ellermeier et al. , 2005 ; Lucas & Lee , 2001 ; Schechter et al. , Abbreviations : Cy , Cyanine ; SSC , saline sodium citrate ; SPI1 , Salmonella pathogenicity island 1 ; S. Typhimurium , Salmonella enterica serovar Typhimurium ; TTSS , type III secretion system .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23676436" target="_blank">23676436</a>]<br></li>
<li>Production of HilC induced expression of hilA ~ 120-fold , while RtsA and HilD induced expression of hilA 30 - to 40-fold , similar to previously reported values ( Eichelberg et al. , 1999 ; Rakeman et al. , 1999 ; Schechter et al. , 1999 ; Schechter and Lee , 2001 ; Ellermeier and Slauch , 2003 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16045614" target="_blank">16045614</a>]<br></li>
<li>Furthermore , HilC and HilD can activate inv/sicA transcription in the absence of HilA ( Rakeman et al. 1999 ; Akbar et al. 2003 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20008574" target="_blank">20008574</a>]<br></li>
<li>Transcription of invF is primarily regulated by HilA , but HilD , HilC and RtsA are each capable of partially activating invF ( Eichelberg et al. , 1999 ; Rakeman et al. , 1999 ; Akbar et al. , 2003 ; Ellermeier and Slauch , 2003 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16045614" target="_blank">16045614</a>]<br></li>
<li>Simplistically this suggests that RtsA acts as a better inducer of slrP expression than does either HilC or HilD , at least compared to their ability to induce hilA expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923082" target="_blank">12923082</a>]<br></li>
<li>To test this , HilC and RtsA were placed under IPTG control to determine if expression of hilA-lacZ could be activated by either regulator at 42 ¬∞ C .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26386070" target="_blank">26386070</a>]<br></li>
<li>Lack of HilC reduces hilA expression 2 - to 3-fold ( Lopez-Garrido and Casadesus 2010 ) , while addition of 0.2 % L-arabinose to the culture medium represses hilA .10 - fold .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25991823" target="_blank">25991823</a>]<br></li>
<li>Rakeman et al. ( 1999 ) showed that overexpression of HilC ( SirC ) can activate invF and sipC , but not prgH , independently of hilA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>So , whereas previous results showed that overexpression of HilC induces invF independently of hilA ( Eichelberg et al. , 1999 ; Rakeman et al. , 1999 ) , our results show that overexpression of HilD or HilC can induce invF expression in the absence of hilA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>This implies that arabinose induction of hilC or hilD expression results in abnormally high levels of HilC or HilD , respectively , and may produce artiÔ¨Åcial situations that permit high-level expression of hilA regardless of any existing repression mechanisms ( see Discussion ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11292791" target="_blank">11292791</a>]<br></li>
<li>However , overexpression of either HilC or HilD resulted in constitutive activation of hilA expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12107132" target="_blank">12107132</a>]<br></li>
<li>Inducible expression of either HilC or RtsA protein , but not of Fur , FliZ , or HilD , was sufÔ¨Åcient to activate hilA following growth at 42 ¬∞ C .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26386070" target="_blank">26386070</a>]<br></li>
<li>This effect was speciÔ¨Åc for HilD since overexpression of either RtsA or HilC , the other AraC/XylS type activators of hilA , was sufÔ¨Åcient to activate SPI-1 expression at 42 ¬∞ C ( Fig. 6C ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26386070" target="_blank">26386070</a>]<br></li>
<li>In the presence of an unknown environmental condition , HilC or HilD may activate invF expression , causing induction of invF-dependent genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10844688" target="_blank">10844688</a>]<br></li>
<li>The latter model requires that chromosomal expression of HilD or HilC be high enough to directly activate invF , independently of hilA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>The direct activation of invF expression by HilC or HilD may always accompany derepression of hilA expression such that the two mechanisms act co-operatively rather than independently to induce invF expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10844688" target="_blank">10844688</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_139">
In contrast , neither RtsA nor HilC was required for FliZ induction of hilA expression , although the absolute level of expression was reduced in both mutants , consistent with our model of SPI1 regulation ( Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20889744" target="_blank">20889744</a>]<br></label>
<input id="act_similarity_139", type="checkbox">
<div class="ACT_similar">
<li>In contrast , neither RtsA nor HilC was required for FliZ induction of hilA expression , although the absolute level of expression was reduced in both mutant backgrounds , consistent with our model of SPI1 regulation ( 15 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17951383" target="_blank">17951383</a>]<br></li></div>
<li>In this case overexpression of HilC or HilD may increase hilA expression by overcoming the enhanced repressing mechanism in these mutants .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11755416" target="_blank">11755416</a>]<br></li>
<li>For example , if HilC regulates hilD expression , this might partially account for the effects of HilC on hilA expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11292791" target="_blank">11292791</a>]<br></li>
<li>One specific question then is whether this behavior is intrinsic to these proteins , as assumed in the model , or simply due to these proteins not being expressed at sufficiently high levels ( as HilC and RtsA can independently activate SPI1 gene expression when over expressed ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20686667" target="_blank">20686667</a>]<br></li>
<li>If a condition exists in which hilA is repressed while invF expression is induced through HilC or HilD , it begs the question : Why produce InvF-dependent effectors in the absence of hilA expression when an intact type III secretion apparatus is not thought to be formed ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10844688" target="_blank">10844688</a>]<br></li>
<li>Individual overexpression of HilA , HilC , or HilD in the absence of SPI1 can not activate siiE expression , suggesting that these transcriptional-regulators act in concert or in combination with additional SPI1-encoded regulatory loci to activate SPI4 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18160484" target="_blank">18160484</a>]<br></li>
<li>Another SPI regulator , HilD , which is not associated with Fisbinding , acts in an ordered fashion with HilC to coordinately activate expression of the SPI-1 genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23717649" target="_blank">23717649</a>]<br></li>
<li>For example , HilA , the main regulator directly activating SPI-1 genes , has been reported to get sufficiently expressed in the presence of HilD along with HilC or RtsA [ 11 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24079299" target="_blank">24079299</a>]<br></li>
<li>Consistent with the SPI-1 gene levels , SPI-1 gene transcriptional-regulators HilC , HilD , HilA , rtsA , and invF were also signiÔ¨Åcantly repressed ( Fig. 6b ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27549210" target="_blank">27549210</a>]<br></li>
<li>However , HilA activation is dependent upon a coupled , positive-feedback loop comprising the AraC/XylS family transcription factors HilD , HilC , and RtsA , which drive HilA expression above the SPI-1 activation threshold ( 7 , 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24375101" target="_blank">24375101</a>]<br></li>
<li>In contrast , overproduction of HilC activates transcription of these fusions as expected ; HilC is known to independently activate hilD transcription ( 13 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20889744" target="_blank">20889744</a>]<br></li>
<li>We then examined the effects of deleting fur , overproducing Fur , and overproducing HilC , which is known to independently activate hilD transcription ( 16 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
<li>Transcription of one gene , ytfK , was activated by HilC but not RtsA ( Fig. 3B and C ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24375101" target="_blank">24375101</a>]<br></li>
<li>In contrast , loss of RtsA and HilC did not signiÔ¨Åcantly alter hilD expression , even though RtsA and HilC can clearly activate hilD ( see Discussion ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16045614" target="_blank">16045614</a>]<br></li>
<li>In addition , loss of HilC and RtsA does not signiÔ¨Åcantly decrease hilD transcription , although the two can clearly activate hilD independently .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16045614" target="_blank">16045614</a>]<br></li>
<li>Figure 5A shows that this construct was not affected by either HilC or Fur overproduction and that the presence or absence of hilD had no effect on transcription of this fusion .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
<li>The immunoblotting analyses show that the increased clpPX disruption effect on cellular levels of HilC and HilD disappeared when either hilD or hilC was introduced ( Fig. 4B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18245288" target="_blank">18245288</a>]<br></li>
<li>Therefore , we hypothesized that either the ingly increase when H-NS activity was reduced by expressing HilD-HilC-RtsA regulatory cascade or IHF could play a role H-NSTEPEC only if HilC , HilD or IHF acted as an H-NS Downloaded from www.microbiologyresearch.org by IP : 132.248.34.164 On : Mon , 22 Oct 2018 15:15:0 We next tested the role of IHF in stationary phase-dependent hilA induction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21680637" target="_blank">21680637</a>]<br></li>
<li>As SprB regulates multiple SPI1 promoters , we hypothesized that it likely represses the transcription of HilC , HilD , or RtsA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20190046" target="_blank">20190046</a>]<br></li>
<li>The transcription factors , HilD , HilC and RtsA can function as monomers or homodimers or heterodimers ( in combination ) to bring about activation of PhilA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19003447" target="_blank">19003447</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19003447" target="_blank">19003447</a>]<br></li>
<li>The presence of HilC , HilD , or RtsA resulted in protection of the positions 74 , 70 , 65 , and 34 compared to what was seen for free DNA ( Fig. 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17675384" target="_blank">17675384</a>]<br></li>
<li>However , in contrast to other effectors examined , RtsA induction of slrP is independent of HilA , InvF , and HilD or HilC ( Fig. 7A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923082" target="_blank">12923082</a>]<br></li>
<li>However , transcription of rtsA , hilC and hilD was induced in response to low oxygen independent of RtsA , HilC , or HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16045614" target="_blank">16045614</a>]<br></li>
<li>Still , no one has reported a condition in which wild-type S. typhimurium represses hilA expression while simultaneously activating expression of invF and invF-dependent genes through HilC or HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10844688" target="_blank">10844688</a>]<br></li>
<li>To determine whether HilE , Pag , or Hha affects repression , rather than derepression , of hilA , the effect that hilE , pag , and hha mutations have on hilA expression should be examined under repressing environmental conditions in strains lacking HilC and HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12933899" target="_blank">12933899</a>]<br></li>
<li>However , as a result of the speciÔ¨Åc environmental conditions , hilA expression may be repressed in such a way that HilD and HilC can not derepress it , or HilA activity may be downregulated post-transcriptionally .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>We have three working hypotheses , which are not mutually exclusive : ( i ) RtsA , HilC , and HilD are active under different conditions or induce expression of hilA in response to different environmental cues .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923082" target="_blank">12923082</a>]<br></li>
<li>These results suggest that HilA , HilC , and HilD are each required for expression of siiE , but these regulators are likely to act in concert or in combination with additional SPI1-encoded regulatory loci to activate SPI4 gene expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18160484" target="_blank">18160484</a>]<br></li>
<li>These data suggest that HilC and RtsA may activate hilA expression in response to a set of signals that are not present in laboratory conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15765064" target="_blank">15765064</a>]<br></li>
<li>The in vitro data suggest that HilC , HilD and RtsA act in concert to induce expression of hilA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16045614" target="_blank">16045614</a>]<br></li>
<li>Epistasis analysis indicates that Dam-dependent regulation of SPI-1 requires HilD , while HilA , HilC , and InvF are dispensable .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20008574" target="_blank">20008574</a>]<br></li>
<li>InvF and HilA are transcriptional-regulators of SPI-1 genes that are encoded within SPI-1 ; hilA itself is regulated by several factors , including PhoP-PhoQ , SirA , HilC , and HilD ( 3 -- 5 , 20 , 37 , 44 , 66 , 72 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10633113" target="_blank">10633113</a>]<br></li>
<li>HilD and HilC do not activate invF expression from the HilA-dependent invF promoter Previously , we had shown that HilA activates invF expression by direct interaction with sequences upstream of invF called the HilA box ( Lostroh et al. , 2000 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>Transcription of invF is known to be regulated by HilA , HilC , and HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923082" target="_blank">12923082</a>]<br></li>
<li>This activation is most probably through direct binding of HilD and HilC to sequences upstream and downstream of this alternative HilA-independent promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>These results suggest that HilD and HilC might directly activate invF expression by binding to sequences upstream of invF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>We considered that HilD and HilC might also bind upstream of invF and directly activate transcription from the HilA-activated invF promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>Our studies show that HilD and HilC activate transcription of invF from a promoter that is far upstream of its HilA-dependent promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>HilD and HilC appear to activate transcription of invF from a HilD/C-dependent start site that is 631 nucleotides upstream of the invF ORF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>The rate of change of HilC concentration depends on its basal synthesis , its activation by HilD and SirA , and its own degradation rate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19537165" target="_blank">19537165</a>]<br></li>
<li>As HilA , HilC , and HilD are DNA-binding regulatory proteins required for SPI1/SPI4 expression , we hypothesized that one of them might directly activate siiABCDEF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18160484" target="_blank">18160484</a>]<br></li>
<li>Strong regulation of genes outside SPI-1 was observed , but most of these genes are direct regulatory targets of other SPI-1-associated TFs ( HilD , HilC , RtsA , InvF , and SprB ) , and HilA likely regulates expression of one or more of these TFs .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27601571" target="_blank">27601571</a>]<br></li>
<li>'' Levels of active HilD , HilC , RtsA , and HilA are controlled by many different regulators , indirectly impacting expression of other SPI-1 genes ( 9 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27601571" target="_blank">27601571</a>]<br></li>
<li>Thus , regulation of SPI-4 by HilD , HilC , and RtsA likely coordinates expression of SPI-4 and SPI-1 genes , which are required for attachment to and invasion of epithelial cells , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24375101" target="_blank">24375101</a>]<br></li>
<li>For example , RtsA and HilD induced expression of rtsA approximately four - to Ô¨Åvefold while HilC induced expression of this gene ~ 30-fold .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16045614" target="_blank">16045614</a>]<br></li>
<li>In Salmonella enterica serovar Typhimurium , RtsA , HilD , and HilC directly activate dsbA and deletion of dsbA leads to loss of SPI1 function .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17951383" target="_blank">17951383</a>]<br></li>
<li>HilC , HilD , and RtsA differentially induce expression of dsbA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14679226" target="_blank">14679226</a>]<br></li>
<li>Effect of HilC - and HilD-binding sites on hilC expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12107132" target="_blank">12107132</a>]<br></li>
<li>Even though HilC and RtsA can independently activate HilA expression when constitutively expressed from ectopic promoters , these two regulators are incapable of doing so in the absence of HilD when expressed from their native promoters [ 16 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20686667" target="_blank">20686667</a>]<br></li>
<li>Moreover , HilD and HilC activate regulator InvF independently of HilA ( 31 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22004521" target="_blank">22004521</a>]<br></li>
<li>The expression of HilA is directed by a multi-component feed forward regulatory loop consisting of HilD , HilC , and RtsA , which are AraC-like regulators that can independently activate HilA expression [ 22 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21984608" target="_blank">21984608</a>]<br></li>
<li>RtsA can induce expression of invF independently of HilA , HilC , and HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923082" target="_blank">12923082</a>]<br></li>
<li>Here , we report that HilD and HilC can also activate expression of a subset of SPI1 genes independently of HilA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>Subsequent experiments demonstrated that the Ô¨Çagellumrelated gene product FliZ controls HilD posttranscriptionally , and this in turn activates HilC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18245288" target="_blank">18245288</a>]<br></li>
<li>Therefore , we examined whether the HilC and HilD accumulate in the Lon-depleted cells grown under these conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15661008" target="_blank">15661008</a>]<br></li>
<li>However , epistasis analysis implicated HilD and HilA , but not HilC , in the direct pathway from the salt stimulus to the Sip secretion response .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18441068" target="_blank">18441068</a>]<br></li>
<li>Hence , it is possible that HilC and RtsA could be activated independently of HilD , by other , as yet unknown regulators ( for e.g , oxygen regulation as suggested in Ellermeier et al. 2005 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19003447" target="_blank">19003447</a>]<br></li>
<li>Moreover , it is well known that occurring truncated form of the H-NS protein that acts as an HilD , HilC and RtsA are required for proper hilA induction H-NS antagonist ( Williamson & Free , 2005 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21680637" target="_blank">21680637</a>]<br></li>
<li>In support of this hypothesis , HilC binds to the URS in vitro ( Schechter and Lee , unpublished results ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11292791" target="_blank">11292791</a>]<br></li>
<li>This data , along with in vivo data from Ellermeier et al. [ 19 ] might suggest the presence of mixed HilC , HilD and RtsA dimers .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17208038" target="_blank">17208038</a>]<br></li>
<li>We detected positive regulation of 7/8 genes by both HilC and RtsA ( Fig. 3B and C ; raw data , including for empty vector controls , shown in Table S4 in the supplemental material ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24375101" target="_blank">24375101</a>]<br></li>
<li>The expression of the type III secretion system-related genes involved in invasion and intracellular survival is induced in the presence of transcriptional-regulator proteins such as HilA , HilC , HilD , InvA , InvC , InvF , SirA , and SirB ( 11 , 27 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24780330" target="_blank">24780330</a>]<br></li>
<li>The expression of the genes , those regulate the type III secretion system , are involved in completion of invasion , intracellular survival and multiplication , is induced by the transcriptional-regulator proteins such as HilA , HilC , HilD , InvA , InvC , InvF , SirA , and SirB ( Johnston et al. , 1996 ; Rakeman et al. , 1999 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27565525" target="_blank">27565525</a>]<br></li>
<li>It is interesting to note that several regulators of T3SS1 , such as HilA , HilC , HilD , and InvF , were found in significantly lower levels in the dam mutant .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21984608" target="_blank">21984608</a>]<br></li>
<li>Interestingly , when S. typhimurium is grown under the normally repressing low osmolarity condition , a null mutation in hns results in high hilA expression , even in the absence of HilC and HilD ( L. Schechter and C. Lee , unpublished observations ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11755416" target="_blank">11755416</a>]<br></li>
<li>This is probably due to the additional regulations considered in this model , whereby HilC and RtsA were ampliÔ¨Åed even in the absence of HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19003447" target="_blank">19003447</a>]<br></li>
<li>HilC and HilD are encoded in the SPI1 locus , while RtsA is encoded elsewhere in the chromosome in an operon with RtsB , which negatively regulates expression of Ô¨ÇhDC and therefore the entire Ô¨Çagellar regulon ( 19 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17951383" target="_blank">17951383</a>]<br></li>
<li>However , at this stage , we can not rule out the possible signiÔ¨Åcance of these interactions in controlling the dynamics of HilA since it is possible that both HilC and RtsA are regulated in more complex ways than considered here .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19003447" target="_blank">19003447</a>]<br></li>
<li>This promoter is active in the absence of repressive proteins and is only moderately ( two - to threefold ) stimulated by the HilC or HilD proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15126488" target="_blank">15126488</a>]<br></li>
<li>In particular , HilC is expressed in the absence of HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20686667" target="_blank">20686667</a>]<br></li>
<li>Differential responses of HilC and HilD in SPI1 expression to environmental stimuli We have previously demonstrated that the lon disruption results in the enhanced SPI1 expression in bacterial cells even when they are grown under SPI1-repressing conditions ( Takaya et al. , 2002 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15661008" target="_blank">15661008</a>]<br></li>
<li>110 100 90 ompR + ompR1009 : : Tn10dTc 80 70 60 50 40 30 20 10 0 + ¬¨ + ¬¨ + ¬¨ + ¬¨ hilD hilC + + + ¬¨ ¬¨ ¬¨ + ¬¨ + + + + ¬¨ ¬¨ ¬¨ ¬¨ rtsA Fusion hilA Discussion We propose a model for SPI1 regulation in which the three AraC-like regulators HilD , HilC and RtsA act in a feedforward loop to activate expression of the hilA gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16045614" target="_blank">16045614</a>]<br></li>
<li>HilD acts in a positive-feed-back loop together with HilC and RtsA to activate HilA and as an activator of Ô¨Çagellar class 1 gene expression through activation of the Ô¨ÇhDC P5 promoter , as described in this work .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24488311" target="_blank">24488311</a>]<br></li>
<li>Three AraC/XylS-type regulators , HilC , HilD , and RtsA ( also known as STM4315 ) are important for the activation of SPI-1 in S. Typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21075923" target="_blank">21075923</a>]<br></li>
<li>SPI1 carries all of the genes necessary to create a functional T3SS apparatus , several secreted effectors , and regulators such as HilA , HilC , HilD and InvF ( described in more detail below .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17208038" target="_blank">17208038</a>]<br></li>
<li>Previous studies showed that four SPI1-encoded transcriptional-regulators , HilD , HilC , HilA and InvF , act in an ordered fashion to coordinately activate expression of the SPI1 TTSS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>These regulators , HilD , HilC , HilA , and InvF , act in an ordered fashion to activate coordinated expression of the SPI1 genes ( 2 , 10 , 12 , 13 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18723621" target="_blank">18723621</a>]<br></li>
<li>In lon + cells , little HilC and a trace of HilD were detected even in cells grown under the SPI1-acivating condition , suggesting that these proteins are short-lived and/or exist in low amounts .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15661008" target="_blank">15661008</a>]<br></li>
<li>Immunoblotting for detection of HilC and HilD proteins in wildtype and Lon-depleted cells grown under SPI1-repressing conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15661008" target="_blank">15661008</a>]<br></li>
<li>We previously showed that the SPI1 regulators RtsA , HilC , and HilD activate dsbA from a novel promoter in serovar Typhimurium and that DsbA is required for SPI1 secretion of effector-proteins ( 18 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17951383" target="_blank">17951383</a>]<br></li>
<li>All indications are that , under the conditions that we are examining , dsbA is primarily controlled by RtsA , HilC , and HilD ( 18 ) ; CpxRA , which controls dsbA in E. coli , seems irrelevant to either expression of dsbA or assembly and function of the SPI1 T3SS or Ô¨Çagella in serovar Typhimurium ( Fig. 2C ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17951383" target="_blank">17951383</a>]<br></li>
<li>( B ) SprB can activate the PsiiA promoter in the absence of other SPI1 genes ( SPI1 rtsAB ) ; the other SPI1 regulators -- HilA , HilC , HilD , and RtsA -- can not .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20190046" target="_blank">20190046</a>]<br></li>
<li>HilC and HilD also directly activate the alternative promoter of the invF operon , which is independent of HilA ( 15,16 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17329372" target="_blank">17329372</a>]<br></li>
<li>HilA and HilC activate the invF gene , the product of which activates genes encoding secreted effectors , including sopB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14645287" target="_blank">14645287</a>]<br></li>
<li>For target genes regulated by multiple TFs , the level of regulation shown is for the highest TF in the hierarchy HilD HilC RtsA InvF RtsB SprB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27601571" target="_blank">27601571</a>]<br></li>
<li>Consequently , we propose that the expression of SPI1 is negatively regulated by controlling the turnover of the HilC and HilD transcriptional-regulators by Lon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15661008" target="_blank">15661008</a>]<br></li>
<li>Taking these results together , it is suggested that SPI1 expression is regulated by ClpXP through a mechanism by which FliZ controls the cellular levels of the transcriptional-regulators HilC and HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18245288" target="_blank">18245288</a>]<br></li>
<li>Thus , although HilD is capable of auto-regulation , the results observed could not be due to the loss of this regulatory protein controlling its own expres ¬¨ Pat post-transcriptionally regulates HilD The regulation of hilD is complex ; hilD regulates itself but is also regulated by the transcriptional-regulators HilC and RtsA , which constitute a feed-forward circuit to control the downstream SPI1 regulator HilA ( Ellermeier et al. , 2005 ; Ellermeier and Slauch , 2007 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27341691" target="_blank">27341691</a>]<br></li>
<li>Thus , it is possible that the feed forward loop constituted by HilD , HilC , and RtsA can regulate the expression of ssrAB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25135218" target="_blank">25135218</a>]<br></li>
<li>Rather , HilD , HilC , and RtsA are overcoming the effects of these proteins at the individual promoters and it is the regulation of HilD levels and action that is the key .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22021388" target="_blank">22021388</a>]<br></li>
<li>Concerning SPI-1 regulation , since similar activating effects are seen for HilA , HilD , and HilC , it is possible that IHF activates SPI-1 genes at the level of HilD , which can then inÔ¨Çuence expression of the downstream regulators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23554419" target="_blank">23554419</a>]<br></li>
<li>Expression of the SPI1 T3SS is tightly regulated by the combined action of HilC , HilD , and RtsA , three AraC family members that can independently activate hilA , which encodes the direct regulator of the SPI1 structural genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
<li>In view of these Ô¨Åndings , we suggest that the regulation of SPI1 expression is negatively controlled through deg-radation of the HilC and HilD transcriptional-regulators by Lon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15661008" target="_blank">15661008</a>]<br></li>
<li>On the basis of these results , we propose that Lon negatively regulates the expression of SPI1 genes through degradation of HilC and HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15661008" target="_blank">15661008</a>]<br></li>
<li>In contrast , both HilC and HilD blocked formation of the hilC transcript that was made in their absence and activated the synthesis of a transcript that was ca. 70 nucleotides shorter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12107132" target="_blank">12107132</a>]<br></li>
<li>DNA-binding activity was calculated by measuring the fraction of each labeled DNA fragment that was shifted in the presence of 100 to 300 nM HilC and HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12107132" target="_blank">12107132</a>]<br></li>
<li>A decrease in transcription was observed in the hilC-D background , but this was most likely due to the effect of the loss of HilC and HilD on transcription of hilA and invF ( 2 , 65 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14679226" target="_blank">14679226</a>]<br></li>
<li>As expected , activation of hilA promoter in the absence of the activators HilD and HilC ( lines 5 -- 8 ) was lost or signiÔ¨Åcantly reduced in inducing ( high osmolarity ) conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16443238" target="_blank">16443238</a>]<br></li>
<li>expression from a plasmid in in vitro transcription assays or in the absence of the URS suggest that HilC and HilD can de-repress hilA expression , possibly by antagonizing the repressing effects of proteins bound to the URS ( 21 , 23 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12933899" target="_blank">12933899</a>]<br></li>
<li>This minimal effect upon changes in the C2 protein-binding region was consistent with the in vitro results that expression of the hilC promoter was comparable in the absence or presence of HilC and HilD , even though a different start site was used .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12107132" target="_blank">12107132</a>]<br></li>
<li>A substantial degree of salt inducibility was restored in the K100 to C505 promoter variant carrying the HilC/HilD-binding A2 region ( K85 to K61 ) .20,21 This stimulatory effect by the presence of the A2 region was SPI1-dependent , suggesting that action of HilC or HilD can increase promoter activity that has been silenced by the presence of the DSR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16443238" target="_blank">16443238</a>]<br></li>
<li>Therefore , the increased invasiveness of Dlon mutant could result from the accumulation of HilC and HilD , leading to overexpression of the SPI1 genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15661008" target="_blank">15661008</a>]<br></li>
<li>Expression of the full-length promoter fragment ( 242 to 505 ) was stimulated roughly 24-fold by the presence of SPI-1 , presumably reÔ¨Çecting the action of HilC and HilD ( Table 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15126488" target="_blank">15126488</a>]<br></li>
<li>concentration of HilC , R the concentration of RtsA , A the concentration of HilA , X the concentration of the HilE-HilD complex , and G the concentration of the luciferase reporter for the PhilD promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20686667" target="_blank">20686667</a>]<br></li>
<li>We hypothesized that ClpXP may regulate HilC and HilD through the control of the Ô¨Çagellar regulon , since ClpXP has been shown to regulate its expression negatively by degrading the FlhD/FlhC master regulator , which functions at the apex of the transcription hierarchy of the regulon ( 61 , 62 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18245288" target="_blank">18245288</a>]<br></li>
<li>Our previous Ô¨Ånding that it negatively regulates the expression of the Ô¨Çagellar regulon by controlling the turnover of the master regulator , FlhD/FlhC ( 62 ) , allowed us to hypothesize that it may regulate HilC and HilD by controlling Ô¨Çagellar gene expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18245288" target="_blank">18245288</a>]<br></li>
<li>In addition , the DNA binding sites of the HilD protein have been mapped and include several genes that are coregulated by HilC and RtsA ( 15 ) , suggesting that reduced activation by one of the three activators may inÔ¨Çuence the activation of coregulated genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26386070" target="_blank">26386070</a>]<br></li>
<li>Subsequent experiments showed that the enhancement of cellular levels of HilC and HilD by clpPX disruption seemed to be caused by an increase in the Ô¨Çagellar protein FliZ , encoded by the class 2 Ô¨ÇiAZY operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18245288" target="_blank">18245288</a>]<br></li>
<li>This indicates that the role of the PhoP protein is to activate pmrD transcription and not simply to counteract repression by the PmrA protein on the pmrD promoter as proposed for other regulatory proteins such as HilC and HilD ( 28 , 29 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12676988" target="_blank">12676988</a>]<br></li>
<li>These results taken together suggest that the HilD , HilC , and RtsA proteins induce the transcription of the rtsA promoter by counteracting H-NS/Hha-mediated repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17675384" target="_blank">17675384</a>]<br></li>
<li>Model 6 , which considers the dependencies of the intermediate components HilC and RtsA on HilD and SirA to function as an AND gate , fails in predicting the levels of both these components in response to the sirA knock-out , as well as the hilD knock-out .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19537165" target="_blank">19537165</a>]<br></li>
<li>Fragment A-b , which carries the promoter and transcribed sequences ( 60 to 450 ) , bound HilC well and HilD less well , as shown by the loss of the band corresponding to free DNA , but most of the bound complexes remained in the wells .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12107132" target="_blank">12107132</a>]<br></li>
<li>Model 7 , which considers the dependencies of the intermediate components HilC and RtsA on HilD and SirA to function as an OR gate , successfully predicts the levels of intermediate components under all conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19537165" target="_blank">19537165</a>]<br></li>
<li>Discussion Expression of the SPI1 T3SS is controlled by HilD , HilC , and RtsA , acting in a complex feed-forward loop to activate the hilD , hilC , and rtsA genes , as well as hilA , which encodes the transcriptional-activator of the T3SS structural genes ( Figure 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22021388" target="_blank">22021388</a>]<br></li>
<li>Expression was dependent on HilD , HilC , and RtsA , as evidenced by the lack of activity in the spi1 rtsA deletion strain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26884427" target="_blank">26884427</a>]<br></li>
<li>However , in vitro transcription was strongly decreased by the T-88 substitution , which should impair both HilC and HilD binding and the activity of the upstream extended 10 promoter element .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12107132" target="_blank">12107132</a>]<br></li>
<li>To evaluate the role of the activators HilD and HilC in the regulation of rtsA transcription , the single-copy rtsA-lac transcriptional fusion was tested in the hilD hilC background in the presence and absence of hha and hns mutations ( Fig. 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17675384" target="_blank">17675384</a>]<br></li>
<li>In this report , we investigate the role of nucleoid-associated proteins Hha and H-NS in the expression of virulence activator proteins HilD , HilC , and RtsA with an emphasis on the interplay of positive and negative transcriptional factors in the regulation of the rtsA promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17675384" target="_blank">17675384</a>]<br></li>
<li>As a consequence , in the model proposed by Ellermeier and Slauch , HilD , the most important activator of HilA in vitro , is at the top of the hierarchy and activates the transcription of HilC and RtsA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23554419" target="_blank">23554419</a>]<br></li>
<li>In all cases , increased HilC and HilD levels consequent on the activation of the corresponding genes by IPTG resulted in increased amounts of the SPI1 products , HilA , InvF and SipC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15661008" target="_blank">15661008</a>]<br></li>
<li>The most speculative aspect of the model concerns the mechanism for activation of the SPI1 promoters -- PhilA , PhilC , PhilD , and PrtsA - by HilC , HilD , and RtsA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20686667" target="_blank">20686667</a>]<br></li>
<li>In replicate experiments , the average degree ( standard deviation ) of stimulation by HilC , HilD , or both was 2.77 0.96 , 2.35 0.87 , and 2.75 1.05 , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15126488" target="_blank">15126488</a>]<br></li>
<li>For example , the transcription of many SPI-1 genes can be regulated by HilA , HilC , HilD , HilE , and PhoP/Q [ 8 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21949647" target="_blank">21949647</a>]<br></li>
<li>For example , the transcription of SPI-1 genes can be activated by HilA , HilC , HilD , InvF , and SirA ( 30 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28439039" target="_blank">28439039</a>]<br></li>
<li>In summary , stimulation of hilA transcription by the AraC/XylS family members HilC and HilD does exhibit the expected dependence on residues in - CTD , but activation of transcription involves other - CTD-independent processes as well .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15126488" target="_blank">15126488</a>]<br></li>
<li>As shown in Fig. 8 , the increased effect of Lon depletion on hilA expression was abolished by depletion of both HilC and HilD , suggesting that both proteins are also involved in regulation of SPI1 by Lon under SPI1-repressing conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15661008" target="_blank">15661008</a>]<br></li>
<li>The results of these surveys are shown in Figure 3 and can be summarized as follows : i. Dam-dependent regulation of hilA was not abolished in the absence of HilC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20008574" target="_blank">20008574</a>]<br></li>
<li>HilD , HilC and RtsA then participate in a feed-forward loop that converges on activation of hilA , which in turn activates transcription of structural components of the needle complex ( prg/org ) as well as invF , which is a transcriptional-activator of secreted effector-proteins ( sic/sip and others ) ( Figure 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28335027" target="_blank">28335027</a>]<br></li>
<li>The effect of HilC and HilD proteins on transcription of these promoters indicated that all three promoters are active in the absence of other cellular proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12107132" target="_blank">12107132</a>]<br></li>
<li>3 s 4 Model 2 : Two feedforward loops with OR gate logic for regulation of P by HilD , HilC and RtsA ( monomer hilA activation ) √∞Model 4√û s 5 1/2 6 a3 HilA : √∞Model 2√û e activators are set to 2 , the the activation of PhilA by If Hill coefÔ¨Åcients of all th model generated represents homodimers .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19003447" target="_blank">19003447</a>]<br></li>
<li>In contrast , we detected regulation of nearby genes for all but two FIG 1 Direct regulation of the invF-containing transcript by HilD , HilC , and RtsA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27601571" target="_blank">27601571</a>]<br></li>
<li>In the case of Finally , the rate of change of HilA concentration is dependent on its basal synthesis ( b ; which is a constant ) , 3 activation by the three transactivators ( HilD , HilC and RtsA ) , autorepression as well as its own degradation rate ( a ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19003447" target="_blank">19003447</a>]<br></li>
<li>Thus , if [ HilD ] , [ HilC ] , [ HilA ] , [ RtsA ] and [ SirA ] represent the protein levels of the corresponding transactivators , then the three equations corresponding to the basic model are as follows : dt d [ RtsA ] = b2 + Œ≤2 ¬∑ f ( [ HilD ] ) ‚àí Œ±2 ¬∑ [ RtsA ] dt d [ HilA ] = b3 + Œ≤3 ¬∑ f ( [ HilD ] , [ HilC ] , [ RtsA ] , [ HilA ] , [ SirA ] ) ‚àí Œ±3 ¬∑ [ HilA ] dt In all the above equations , [ HilD ] and [ SirA ] can each be either 1 or 0 , depending on the presence or absence of the signal for induction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19537165" target="_blank">19537165</a>]<br></li>
<li>In this context , slrP is induced by overexpression of HilC , HilD , and RtsA independently of the central SPI1 regulator HilA , with RtsA being the best inducer .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25182488" target="_blank">25182488</a>]<br></li>
<li>Finally , the rate of change of HilA concentration depends on its basal synthesis , activation by the four transactivators ( HilD , HilC , RtsA and SirA ) , autorepression and its own degradation rate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19537165" target="_blank">19537165</a>]<br></li>
<li>This is represented by the following equation : 3 d 1/2 HilA 1/4 b3 √æ b3 f HilD ; HilC ; RtsA ; HilA dt a3 HilA : ‚Ä¢ In all the above equations , [ HilD ] can be either 1 or 0 , depending on the presence or absence of the signal for induction , b1 , b2 and b3 are the basal rates of synthesis of HilC , RtsA and HilA respectively , b1 , b2 and b3 are ampliÔ¨Åcation factors which can be thought of as analogous to translation , a1 , a2 and a3 and are the rates of degradation of HilC , RtsA and HilA respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19003447" target="_blank">19003447</a>]<br></li>
<li>Our data conÔ¨Årm a large degree of overlap between the HilD , HilC , and RtsA regulons and extend this overlap to include the novel HilD targets ; we observed transcription activation of all 8 of the HilD-activated genes by HilC and all but one by RtsA ( Fig. 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24375101" target="_blank">24375101</a>]<br></li>
<li>To test whether the RtsA , HilC , and HilD activators of hilA are also regulated by these proteins , we investigated the transcription of rtsA-lac , hilC-lac , and hilD-lac chromosomal fusions ( 13 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17675384" target="_blank">17675384</a>]<br></li>
<li>Indeed , this is the case for activation of hilD , hilC , rtsA , and hilA by HilD/HilC / RtsA ( 29 , 30 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27601571" target="_blank">27601571</a>]<br></li>
<li>In the case of hilD , post-translational repression by HilE dampens hilD activation , and H-NS repression of hilA counteracts transcriptional activation by HilD , HilC and RtsA ( 66 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28335027" target="_blank">28335027</a>]<br></li>
<li>HilD , HilC and RtsA are thought to form a triangular network system of regulators positively regulating each other , while HilE can regulate this system by acting as a negative regulator of HilD ( Fig. 1 , ‚ä• arrow ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21107576" target="_blank">21107576</a>]<br></li>
<li>FIG 3 - Galactosidase assay showing activation of HilD target genes by HilC and RtsA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24375101" target="_blank">24375101</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_282">
Model 5 : Two feedforward loops with OR gate model of regulation of PhilA by heterodimers of HilC-HilD , HilC-RtsA and HilD-RtsA and additional positive feedback on RtsA and HilC as well as cross activations of RtsA on HilC and vice versa This is of the same form as Model 3 , except that the functions s3 , s4 and s5 are of the type considered in Model 4 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19003447" target="_blank">19003447</a>]<br></label>
<input id="act_similarity_282", type="checkbox">
<div class="ACT_similar">
<li>Model 4 : Two feedforward loops with OR gate model of regulation of PhilA by heterodimers of HilC-HilD , HilC-RtsA and HilD-RtsA In this case functions s3 , s4 and s are m : √∞ 1/2 HilD 1/2 HilC √ûH3 kH3 √æ √∞ 1/2 HilD 1/2 HilC √ûH3 3 odiÔ¨Åed as follows : s 3 s 4 Model 2 : Two feedforward loops with OR gate logic for regulation of P by HilD , HilC and RtsA ( monomer hilA activation ) √∞Model 4√û s 5 In this case , the rate of change of HilA concentration is a additive function of the three input transactivators via the functions s3 , s4 and s5 corresponding to HilD , HilC and RtsA respectively and is represented as : d 1/2 HilA 1/4 1/2 √æ 1/2 √æ √æ b3 b3 s3 s4 s5 s dt Model 3 : Two feedforward loops with OR gate model for regulation of PhilA by monomers of HilD , HilC and RtsA and addition of positive feedback on RtsA and HilC as well as cross activations of RtsA on HilC and vice versa where functions s7 and s9 represent the auto-activation of HilC and RtsA respectively and functions s8 and s10 represent the cross activation of HilC by RtsA and vice versa respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19003447" target="_blank">19003447</a>]<br></li>
<li>Model 4 : Two feedforward loops with OR gate model of regulation of PhilA by heterodimers of HilC-HilD , HilC-RtsA and HilD-RtsA In this case functions s3 , s4 and s are m : √∞ 1/2 HilD 1/2 HilC √ûH3 kH3 √æ √∞ 1/2 HilD 1/2 HilC √ûH3 3 odiÔ¨Åed as follows : s 3 This corresponds to activation via the heterodimeric complex of HilC and HilD with co-operativity H3 and threshold of activation K3 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19003447" target="_blank">19003447</a>]<br></li></div>
<li>1/4 : √∞ 1/2 HilC 1/2 RtsA √ûH4 kH4 √æ √∞ 1/2 HilC 1/2 RtsA √ûH4 4 s 4 This corresponds to activation via the heterodimeric complex of HilC and RtsA with cooperativity H4 and threshold of activation K4 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19003447" target="_blank">19003447</a>]<br></li>
<li>1/4 : √∞ 1/2 HilD 1/2 RtsA √ûH5 kH5 √æ √∞ 1/2 HilD 1/2 RtsA √ûH5 5 Model 2 : Two feedforward loops with OR gate logic for regulation of P by HilD , HilC and RtsA ( monomer hilA activation ) √∞Model 4√û s 5 This corresponds to activation via the heterodimeric complex of HilD and RtsA with cooperativity H5 and threshold of activation K5 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19003447" target="_blank">19003447</a>]<br></li>
<li>We also show that HilD-activated genes are frequently activated by HilC and RtsA , indicating extensive overlap of the HilD , HilC , and RtsA regulons .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24375101" target="_blank">24375101</a>]<br></li>
<li>We conclude that HilD , HilC , and RtsA bind very similar DNA sites , although the failure of RtsA to regulate ytfK ( Fig. 3C ) indicates that there are subtle differences in the sequence speciÔ¨Åcities of HilD , HilC , and RtsA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24375101" target="_blank">24375101</a>]<br></li>
<li>However , this binding of HilD requires a larger target than does binding of HilC , as seen by comparing the binding to fragments C-d and C-f .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12107132" target="_blank">12107132</a>]<br></li>
<li>Changes at any of these positions depressed binding of HilD , and all but the last three positions contributed to-binding of HilC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12107132" target="_blank">12107132</a>]<br></li>
<li>HilD , HilC , and RtsA are homologous AraC-like regulators that bind to very similar DNA sites ; therefore , when these regulators are overexpressed , they are able to self-activate their own expression and to activate the expression of one another , as well as to induce the expression of most genes belonging to the HilD regulon ; however , HilD is considered to be dominant , as deletion of hilD , but not of hilC or rtsA , drastically reduces the expression of the targets of HilD ( 23 , 24 , 26 -- 30 , 61 -- 64 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25135218" target="_blank">25135218</a>]<br></li>
<li>It will be interesting to determine how the binding of HilA upstream of invF affects the ability of HilD and HilC to-bind upstream of and activate invF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>Overproduction of HilD and HilC activate the expression of a chromosomal invF-lacZ fusion in the absence of hilA In order to study the activation of invF by HilD and HilC , we examined the expression of a chromosomal invF-lacZ fusion in a hilA hilD mutant strain transformed with pSA4 , pLS119 or pCH112 , which express HilD , HilC and HilA , respectively , under the control of the arabinose-inducible PBAD promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>In contrast , disruption of the downstream -10 hexamer sequence dramatically reduced the ability of HilD and HilC to activate invF expression ( Fig. 6 , black and striped bars , pSA14 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>To assess the accumulation of SPI1 products in different concentrations of HilC or HilD , these cells were exposed to various concentrations of IPTG for 2.5 h , and HilC , HilD and SPI1 levels were analysed by immunoblotting .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15661008" target="_blank">15661008</a>]<br></li>
<li>The production of HilC and HilD was induced by the addition of 1 mM IPTG for 5 min .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15661008" target="_blank">15661008</a>]<br></li>
<li>That is to say , disruption of Ô¨ÇiZ abolished the increase of HilC and HilD caused by deleting clpPX ( Fig. 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18245288" target="_blank">18245288</a>]<br></li>
<li>Obviously , this central core of regulators ( HilC , HilD , RtsA , HilE ) would integrate multiple signaling inputs , as many sensors have been described and shown to regulate the activation of pathogenicity ; but a noise-driven switch would nevertheless be a simple way of regulation , including complex sensors for population density or physiological state of the host .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21107576" target="_blank">21107576</a>]<br></li>
<li>However , HilD does not work alone under normal conditions : in the absence of HilC and RtsA , loss of HilD has little effect on transcription of hilA or invasion of the intestine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16045614" target="_blank">16045614</a>]<br></li>
<li>However , given that the external regulators of the system , e.g. SirA/BarA and OmpR/EnvZ , seem to function through HilD , loss of HilD could have more profound effect than loss of RtsA or HilC due to the lack of these external activation signals rather than some inherent difference in efÔ¨Åciency of HilD versus HilC or RtsA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16045614" target="_blank">16045614</a>]<br></li>
<li>Studies on regulation of PhilA by different types of O gate architectures involving three activators ( HilD , HilC and RtsA ) R ¬¨ As can be seen from Table 1 , Model 1 ( AND gate ) failed to capture the effect of the different mutants on HilA expression levels since it predicted decrease to basal activity irrespective of which regulator was muta-ted .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19003447" target="_blank">19003447</a>]<br></li>
<li>Earlier work on mathematical modeling of regulation of expression of hilA by HilC , HilD and RtsA was based on feed-forward architecture , considering HilD to be the primary activator of the feed-forward loop [ Maithreye and Mande , 2007 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19537165" target="_blank">19537165</a>]<br></li>
<li>First , Lon is induced to control turnover of HilD and HilC and hence limit the expression of SPI-1 genes , which is of particular relevance after invasion of epithelial cells .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23554419" target="_blank">23554419</a>]<br></li>
<li>Depletion of ClpXP caused signiÔ¨Åcant increases in the amounts of two SPI1-encoded transcriptional-regulators , HilC and HilD , leading to the stimulation of hilA induction and therefore activation of SPI1 expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18245288" target="_blank">18245288</a>]<br></li>
<li>Production of HilD leads to transcriptional activation of HilC , and each activates the expression of both ( 13 , 40 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18245288" target="_blank">18245288</a>]<br></li>
<li>However , this possibility is less likely , because no chromosomal rtsA-lac expression was detected in the absence of HilD , HilC , and RtsA ( 13 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17675384" target="_blank">17675384</a>]<br></li>
<li>Moreover , in the absence of Lon , HilC and HilD accumulate intracellularly ( 317 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23554419" target="_blank">23554419</a>]<br></li>
<li>Comparison of relative expression levels of intermediate components ( HilC and RtsA ) in the network under scenario-2 and scenario-3 ( models 5 , 6 , 7 and 8 ) Analysis of the levels of HilC and RtsA in response to knock-out mutations in sirA and hilD is presented in Table 4 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19537165" target="_blank">19537165</a>]<br></li>
<li>We wanted to determine if HilC , HilD and RtsA could induce expression of hilC , hilD , rtsA and hilA in the absence of the other regulators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16045614" target="_blank">16045614</a>]<br></li>
<li>Therefore , we examined the expression of hilA , rtsA , hilC and hilD transcriptional fusions in the presence and absence of RtsA , HilC , and/or HilD ( Fig. 7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16045614" target="_blank">16045614</a>]<br></li>
<li>The effect of SPI1 inducing and repressing conditions on the expression of rtsA , hilC , hilD and hilA in the absence of in RtsA , HilC , and/or HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16045614" target="_blank">16045614</a>]<br></li>
<li>HilC , HilD and RtsA , in addition to regulation of hilA , are all capable of inducing expression of the inv/spa operon -- and by read-through , the sic/sip operon -- in a fashion independent of HilA [ 15,16,25,27 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17208038" target="_blank">17208038</a>]<br></li>
<li>RtsA induces expression of hilA in the absence of both HilC and HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923082" target="_blank">12923082</a>]<br></li>
<li>The ability of SirA to induce expression of rtsA , hilC , hilD , hilA and invF in the absence of RtsA , HilC , and/or HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16045614" target="_blank">16045614</a>]<br></li>
<li>Under the experimental conditions used , HilC and RtsA were not needed to achieve high hilA expression ; however , HilD was essential since hilA expression was abolished in ŒîhilD mutants regardless of the presence or absence of Gre factors .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28426789" target="_blank">28426789</a>]<br></li>
<li>As shown in Fig. 5B , loss of RtsA or HilC decreased the level of hilA and invF expression but did not signiÔ¨Åcantly affect induction by SirA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16045614" target="_blank">16045614</a>]<br></li>
<li>Expression of hilA-lacZ fusions displays complex regulation and can be activated by the HilC or HilD protein ( 42 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12107132" target="_blank">12107132</a>]<br></li>
<li>Due to the counteracting nature of the repressing URS and derepressing HilC and HilD proteins that directly control hilA expression , the hns-1 : : kan mutation may increase hilA expression under the low-osmolarity conditions ( i ) by decreasing its repression and/or ( ii ) by increasing its derepression by HilC or HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12933899" target="_blank">12933899</a>]<br></li>
<li>But the situation would be entirely different at sites 3 and 4 , in which hilA and the secretion apparatus are not expressed but InvF and certain effectors are produced as a result of direct activation of invF expression by HilC or HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10844688" target="_blank">10844688</a>]<br></li>
<li>Future work can also take into account the differential binding afÔ¨Ånities of HilD and HilC on the hilA promoter as shown by Olekhnovich and Kadner ( 2002 ) and include the regulation of HilC and HilD by CsrA/B as demonstrated by Lawhon et al. ( 2003 ) and Fortune et al. ( 2006 ) to get a better picture of the complex interactions involved in the regulation of PhilA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19003447" target="_blank">19003447</a>]<br></li>
<li>They act by increasing the level of HilD protein to overcome HilE , and controlling HilD activity , or in the case of Fur , lowering the threshold required at the promoter , such that HilD activates its own promoter as well as induces expression of HilC and RtsA , which then act to amplify and accelerate SPI1 expression ( Saini et al. 2010a ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22021388" target="_blank">22021388</a>]<br></li>
<li>) b-galactosidase Activity b-galactosidase Activit SirA induction of hilA and invF requires HilD Previous evidence suggested that SirA required the HilC / HilD/RtsA binding sites in the hilA promoter to induce expression of hilA and thus the entire SPI1 TTSS b-galactosidase Activity b-galactosidase Activit Fig. 5 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16045614" target="_blank">16045614</a>]<br></li>
<li>Recent work has demonstrated that both HilD and HilC bind to sequences upstream of the hilA promoter ( 39 , 44 ) and that the presence of HilD is required for hilA activation even in the absence of several negative regulatory elements ( 9 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15731035" target="_blank">15731035</a>]<br></li>
<li>In the absence of H-NS it is possible that silencing complexes generated by StpA and M4T StpAA77D , although more effective than StpAWT , were unable to impede the combined HilC and HilD-mediated activation of hilA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25375226" target="_blank">25375226</a>]<br></li>
<li>RtsA forms a complex regulatory network with HilC and HilD that ultimately activates expression of hilA , which activates components of SPI-1 ( 9 , 11 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26386070" target="_blank">26386070</a>]<br></li>
<li>LoiA activates expression of hilA gene through activating hilD As the expression of hilA is directly controlled by HilD , HilC and RtsA , we tested whether LoiA regulates HilA through any of these three regulators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28575106" target="_blank">28575106</a>]<br></li>
<li>RtsA increases invasion gene expression by increasing transcription of hilA , which is independent of HilC and HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15765064" target="_blank">15765064</a>]<br></li>
<li>The expression of hilA , the master regulator for SPI1 , is controlled by HilD , HilC , and RtsA , which act in a complex feed-forward loop .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20889744" target="_blank">20889744</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_329">
The expression of hilA , encoding the transcriptional-activator of the T3SS structural genes , is directly controlled by three AraC-like regulators , HilD , HilC , and RtsA , each of which can activate hilD , hilC , rtsA , and hilA genes , forming a complex feed-forward regulatory loop .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26884427" target="_blank">26884427</a>]<br></label>
<input id="act_similarity_329", type="checkbox">
<div class="ACT_similar">
<li>The expression of hilA , encoding the transcriptional-activator of the SPI1 structural genes , is directly controlled by three AraC-like regulators , HilD , HilC , and RtsA , each of which can activate the hilD , hilC , rtsA , and hilA genes , forming a complex feed-forward regulatory loop .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22021388" target="_blank">22021388</a>]<br></li></div>
<li>Transcriptional activation of hilA is dependent upon three AraC-like proteins , HilD , HilC , and RtsA , which bind to the same sites upstream of activated promoters ( 11 , 12 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29378886" target="_blank">29378886</a>]<br></li>
<li>( iii ) RtsA , HilC , and HilD are required for signal ampliÔ¨Åcation such that once one of these genes is turned on , expression of the others is also induced , resulting in greater induction of hilA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923082" target="_blank">12923082</a>]<br></li>
<li>In serovar Typhimurium , dsbA is controlled by the hilA regulators RtsA , HilD , and HilC , and expression of DsbA under SPI1-inducing conditions is independent of CpxRA ( 18 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17951383" target="_blank">17951383</a>]<br></li>
<li>However , induction of hilA is independent of HilC and HilD and is mediated by direct binding of RtsA to the hilA promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923082" target="_blank">12923082</a>]<br></li>
<li>Thus , FliZ-mediated induction of hilA expression is through HilD , while RtsA and HilC act as ampliÔ¨Åers of the signal .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17951383" target="_blank">17951383</a>]<br></li>
<li>In this scheme , HilD and HilC indirectly activate expression of the SPI1 TTS apparatus and its secreted substrates by affecting hilA expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_337">
We demonstrate that HilC , HilD and RtsA are each capable of independently inducing expression of the hilC , hilD and rtsA genes , and that each can independently activate hilA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16045614" target="_blank">16045614</a>]<br></label>
<input id="act_similarity_337", type="checkbox">
<div class="ACT_similar">
<li>We show that HilD , HilC and RtsA can each independently activate expression of the hilD , hilC , rtsAB and hilA genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16045614" target="_blank">16045614</a>]<br></li></div>
<li>We show that HilD and HilC can directly bind and activate invF , which leads to the non-co-ordinate expression of a subset of SPI1 genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>Our current studies demonstrate that HilD and HilC can bind upstream of invF and directly activate its expression in the absence of hilA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>HilC and HilD relieve repression of the hilA promoter by the nucleoid proteins H-NS and Hha ( Olekhnovich and Kadner 2006 ) and can activate expression of the invF and sicA/sip transcriptional units independently of HilA ( Rakeman et al. 1999 ; Akbar et al. 2003 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25991823" target="_blank">25991823</a>]<br></li>
<li>Two additional regulators , HilC and HilD , activate directly the expression of hilA and the invF operon , independently of HilA ( Dieye et al. , 2007 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27404739" target="_blank">27404739</a>]<br></li>
<li>Although HilA is the major regulator , two other regulators encoded within SPI1 , HilC and HilD , can directly increase the expression of both hilA and invF independently of hilA ( 3 , 53 , 56 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14645287" target="_blank">14645287</a>]<br></li>
<li>HilC and HilD bind directly to the upstream sequence of hilA and derepress the transcription of PhilA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15661008" target="_blank">15661008</a>]<br></li>
<li>RtsA belongs to the AraC/XylS family of regulators , and can directly bind the hilA promoter and induce the hilA expression independent of HilC and HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15661008" target="_blank">15661008</a>]<br></li>
<li>Under inducing conditions , activation of BarA/SirA initiates a regulatory cascade involving HilD , HilC , and HilA , which leads both directly and indirectly to the expression of SPI1 and SPI4 genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18160484" target="_blank">18160484</a>]<br></li>
<li>Three homologous proteins , RtsA , HilC and HilD , which belong to the AraC/XylS family of transcriptional-regulators , bind to the DNA immediately upstream of hilA leading to induction of hilA expression ( Schechter and Lee , 2001 ; Olekhnovich and Kadner , 2002 ; Ellermeier and Slauch , 2003 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16045614" target="_blank">16045614</a>]<br></li>
<li>Based on our genetic analyses , and taking into account the published results outlined above , we present a model in which expression of hilA is controlled by the combined action of HilC , HilD and RtsA , each of which can independently bind upstream of hilA to induce its expression ( Schechter and Lee , 2001 ; Olekhnovich and Kadner , 2002 ; Ellermeier and Slauch , 2003 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16045614" target="_blank">16045614</a>]<br></li>
<li>Thus these regulators form a self-reinforcing feed forward regulatory loop and increased expression or activity of one regulator leads to an increase in expression of the other regulators Results HilC , HilD and RtsA can independently induce expression of hilC , hilD and rtsA We have shown that RtsA ( Ellermeier and Slauch , 2003 ) , like HilC and HilD ( Schechter and Lee , 2001 ; Olekhnovich and Kadner , 2002 ) , induces expression of hilA by binding to the HilA promoter region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16045614" target="_blank">16045614</a>]<br></li>
<li>We believe that the inductions of RtsA , HilC , and HilD are self-reinforcing events leading to rapid and fully induced expression of hilA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923082" target="_blank">12923082</a>]<br></li>
<li>Three AraC-like regulators , HilD , HilC , and RtsA , control expression of hilA , and thereby induction of the SPI1 system .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22021388" target="_blank">22021388</a>]<br></li>
<li>Effect of upstream DNA on stimulation of hilA transcription by HilC and HilD proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15126488" target="_blank">15126488</a>]<br></li>
<li>Effect of invF , hilA , and hilC-D mutations on RtsA induction of slrP ( A ) and ability of HilC , HilD , and RtsA to induce expression of hilA ( B ) and slrP ( C ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923082" target="_blank">12923082</a>]<br></li>
<li>HilC , HilD , and RtsA have all been shown to-bind directly upstream of hilA , and deletion of hilC , hilD , or rtsA leads to a lower level of hilA expression ( 16 , 18 , 42 , 47 , 43 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
<li>In addition , these three regulators ( i.e. , HilC , HilD , and RtsA ) activate a regulator within SPI-1 , HilA , which shows homology with the OmpR/ToxR family of DNA-binding proteins and induces transcriptional activation of SPI-1 genes in response to a variety of stimuli ( 1 , 59 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21075923" target="_blank">21075923</a>]<br></li>
<li>Production of HilD leads to transcriptional activation of HilC and RtsA , which each activate expression of themselves and of each other .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17208038" target="_blank">17208038</a>]<br></li>
<li>Role of Nucleoid-Associated Proteins Hha and H-NS in Expression of Salmonella enterica Activators HilD , HilC , and RtsA Required for Cell Invasion Igor N. Olekhnovich * and Robert J. Kadner ‚Ä† icrobiology , University of Virginia School of Medicine , Charlottesville , Virginia 22908-0734 Received 8 June 2007/Accepted 23 July 2007 ent of M Departm The coordinate expression of Salmonella enterica invasion genes on Salmonella pathogenicity island 1 is under the control of the complex circuits of regulation that involve the AraC/XylS family transcriptional-activators HilD , HilC , and RtsA and nucleoid-associated proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17675384" target="_blank">17675384</a>]<br></li>
<li>Three homologous proteins , 8,13,14 HilC , HilD , and RtsA , which belong to the AraC/XylS family of transcription factors can activate expression of hilA and some of its target genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16443238" target="_blank">16443238</a>]<br></li>
<li>In this model , HilD or HilC expression and activity could be induced upon exposure of S. typhimurium to conditions within the intestinal lumen .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>More recently , H-NS has been shown to repress the expression of HilA , HilD , HilC , and RtsA under noninducing conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23554419" target="_blank">23554419</a>]<br></li>
<li>We then examined whether the accumulation of HilC and HilD under SPI1-repressing conditions leads to enhanced expression of SPI1 genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15661008" target="_blank">15661008</a>]<br></li>
<li>HilC and HilD bind directly upstream of the master regulator gene hilA to induce its expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18245288" target="_blank">18245288</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_363">
Both HilC and HilD activate expression of SPI-1 genes by binding upstream of the master regulatory gene hilA to induce its expression [ 48 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22291596" target="_blank">22291596</a>]<br></label>
<input id="act_similarity_363", type="checkbox">
<div class="ACT_similar">
<li>Similar to HilC and HilD , RtsA activates the expression of SPI1 genes by binding upstream of the master regulatory gene , hilA , to induce its expression ( 9,17 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17329372" target="_blank">17329372</a>]<br></li>
<li>Like HilC and HilD , RtsA activates expression of SPI1 genes by binding upstream of the master regulatory gene hilA to induce its expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16045614" target="_blank">16045614</a>]<br></li></div>
<li>In this model , the repressor is always present but can not inhibit hilA expression under activating conditions because HilD ( and , to a lesser extent , HilC ) displaces it from the URS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11292791" target="_blank">11292791</a>]<br></li>
<li>At this point , HilD is able to autoactivate its own promoter , as well as those of HilC and RtsA , leading to robust and rapid induction of the SPI1 T3SS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29378886" target="_blank">29378886</a>]<br></li>
<li>It is likely that Lon may function through its proteolytic activity to regulate the availability of HilC and/or HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11741864" target="_blank">11741864</a>]<br></li>
<li>Our data suggest that HilC , HilD , and RtsA function in similar manners by binding to the same fragment of hilA to induce its expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923082" target="_blank">12923082</a>]<br></li>
<li>Conversely , HilC and HilD act as transcription activators to induce the expression of hilA by binding to the upstream repression sites ( Ellermeier et al. , 2005 ; Lucas and Lee , 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27565525" target="_blank">27565525</a>]<br></li>
<li>HilC and HilD bind to the hilA promoter region , and it is believed that this binding induces the expression of hilA ( 64 , 71 , 72 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923082" target="_blank">12923082</a>]<br></li>
<li>HilC and HilD can each individually bind to the DNA immediately upstream of hilA , and it is believed that this binding leads to hilA expression ( 12 , 52 , 53 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18245288" target="_blank">18245288</a>]<br></li>
<li>By binding to the URS element , HilD and HilC may displace or counteract the repressors , thereby derepressing hilA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>Two AraC-like transcriptional-regulators encoded on Salmonella pathogenicity island 1 ( SPI1 ) , HilC and HilD , bind to the URS to counteract the repression of hilA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11292791" target="_blank">11292791</a>]<br></li>
<li>Under activating conditions , HilC and HilD bind to region 2332 to 239 upstream of the hilA promoter , displacing the repressor from this site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11292791" target="_blank">11292791</a>]<br></li>
<li>These results also explain why our invF-lacZ reporter plasmid , pVV448 , was not activated by HilD or HilC to the same extent as the chromosomal invF-lacZ fusion because pVV448 does not contain the HilD - and HilC-dependent +1 of invF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>Our studies also suggest that the downstream binding site is required for HilD and HilC to activate invF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>These results indicate that HilD and HilC require the downstream binding site to activate invF expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>To see if these sequences are important for HilD and HilC to activate invF , we constructed derivatives of pSA7 in which the upstream TTTGAT was changed to TTTGAG and the downstream TATTCT was changed to TATGCG .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>These results contrast with the ability of HilD and HilC to activate a chromosomal invF fusion 80 - to 170-fold , suggesting that HilD and HilC require different sequences to activate invF and do not use the same promoter as HilA does to activate invF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>HilD and HilC bind to the invF promoter fragment in vitro To examine whether HilD and HilC might directly activate the invF promoter by binding to sequences upstream of invF , we performed gel shift analyses .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>Under these growth conditions , two AraC-like transcriptional-regulators , HilD and HilC , encoded within SPI-1 , activate hilA expression ( Schechter et al. , 1999 ; Schechter and Lee , 2001 , Ellermeier et al. , 2005 ) , the central positive regulator of SPI-1 , by counteracting the repression exerted by the global repressor H-NS on the hilA promoter ( Schechter et al. , 2003 ; Olekhnovich and Kadner , 2006 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21518393" target="_blank">21518393</a>]<br></li>
<li>Indeed , it has been proposed that the molecular role of HilD and HilC is to derepress hilA ( Schechter et al. , 1999 , 2003 ; Schechter and Lee , 2001 ) , likely by counteracting H-NS , although there is debate on this point ( Boddicker et al. , 2003 ; Olekhnovich and Kadner , 2004 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16045614" target="_blank">16045614</a>]<br></li>
<li>To perform this analysis , we varied the degree by which the SPI1 regulators - HilC , HilD , and RtsA - could activate HilD expression by simulating the model at different values for the parameter kD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20686667" target="_blank">20686667</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_385">
The parameter kD specifies the degree by which the SPI1 regulators - HilC , HilD , and RtsA - can activate HilD expression , effectively the strength of positive feedback on HilD expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20686667" target="_blank">20686667</a>]<br></label>
<input id="act_similarity_385", type="checkbox">
<div class="ACT_similar">
<li>The parameter k specifies the degree by D D D which the SPI1 regulators - HilC , HilD , and RtsA - can activate HilD expression , effectively the strength of positive feedback on HilD expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20686667" target="_blank">20686667</a>]<br></li></div>
				<label class="ACT_collapse_similarity" for="act_similarity_387">
Specifically , we varied the degree by which the SPI1 regulators -- HilD , HilC and RtsA - could enhance both HilC and RtsA gene expression , given respectively by the parameters kC and kR in the model .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20686667" target="_blank">20686667</a>]<br></label>
<input id="act_similarity_387", type="checkbox">
<div class="ACT_similar">
<li>The parameters kC and kR specify the degree by which the SPI1 regulators - HilC , HilD , and RtsA - can activate HilC and RtsA expression , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20686667" target="_blank">20686667</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20686667" target="_blank">20686667</a>]<br></li></div>
<li>Since HilA is activated by the three factors HilD , HilC and RtsA ( Ellermeier et al. 2005 ; Ellermeier and Slauch 2007 ) , in analogy with the equations written above , the dynamics of HilA can be described by the following nonlinear system : where again i = 1 corresponds to HilD , i = 2 to HilC , and i = 3 to RtsA , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21107576" target="_blank">21107576</a>]<br></li>
<li>Only when the activating signals are sufficiently strong is HilD expressed at a high enough level to overcome sequestration by HilE and activate the expression of the other SPI1 regulators -- HilC , RtsA , and HilA-and also further induce its own expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20686667" target="_blank">20686667</a>]<br></li>
<li>HilC and HilD also activate the promoter PinvF-2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15661008" target="_blank">15661008</a>]<br></li>
<li>To date , it is known that the positive regulators HilC and HilD and the negative regulators Hha and Fis can bind directly the hilA promoter ( 8 , 19 , 53 , 59 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14645287" target="_blank">14645287</a>]<br></li>
<li>In agreement with genetic data , we show that the puriÔ¨Åed RtsA protein , like HilC and HilD , binds to the hilC , hilD , rtsA , and hilA promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17675384" target="_blank">17675384</a>]<br></li>
<li>We also needed to assume that HilD was necessary for establishing connectivity within the network , where it was again required for HilC - and RtsA-dependent activation of the P , hilC P , and P promoters ( see Materials and Methods for further hilD rtsA details ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20686667" target="_blank">20686667</a>]<br></li>
<li>In addition to the negative control that was exerted by H-NS/Hha under low-osmolarity conditions , the homologous virulence activators HilD , HilC , and RtsA ( Hil activators ) induced rtsA-lac expression in a high-salt medium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17675384" target="_blank">17675384</a>]<br></li>
<li>Our results show that virulence activators HilD , HilC , and RtsA bind to a common site in the rtsA promoter and antagonize H-NS/Hha-mediated repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17675384" target="_blank">17675384</a>]<br></li>
<li>RtsA , HilC , and HilD each activate hilA by binding to the same sites in the promoter region ( 13 , 35 , 36 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20889744" target="_blank">20889744</a>]<br></li>
<li>Similarly , HilC and HilD bound in different manners to the hilA promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12107132" target="_blank">12107132</a>]<br></li>
<li>The expression of the invasion-associated type III secretion system in Salmonella is indeed subject to a remarkably complex regulation involving both speciÔ¨Åc ( InvF , HilA , HilB , and SprA-HilC ) and global ( PhoP-PhoQ , SirA , and RcsB-RcsC ) transcriptional-regulatory proteins ( 1 , 4 , 5 , 9 , 11 , 24 , 27 , 36 , 46 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10768967" target="_blank">10768967</a>]<br></li>
<li>HilA is , in turn , induced by HilD and HilC and regulates InvF , which induces expression of secreted effector-proteins located in SPI-1 ( 26 , 27 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22004521" target="_blank">22004521</a>]<br></li>
<li>For example , the regulatory hierarchy for SPI1 is complex in that SirA activates hilA and hilC , HilC also activates hilA , and HilA and HilC in turn activate invF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17233907" target="_blank">17233907</a>]<br></li>
<li>HilC and HilD are encoded within SPI1 , whereas RtsA is encoded in an operon at 93.9 centisomes that also encodes RtsB , which negatively regulates Ô¨ÇhDC and hence the Ô¨Çagellar regulon ( Ellermeier and Slauch , 2003 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16045614" target="_blank">16045614</a>]<br></li>
<li>To understand how Lon selectively degrades and modulates HilC and HilD , biochemical analysis to examine the direct interaction between both these proteins and Lon in vitro is in progress .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15661008" target="_blank">15661008</a>]<br></li>
<li>The hilD promoter is active in vitro and unaffected by the HilC and HilD proteins or other SPI1 products .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12107132" target="_blank">12107132</a>]<br></li>
<li>In summary , sites A1 and C2 can bind both HilC and HilD in the absence of Ô¨Çanking sequences , whereas sites A2 and D can bind HilC but not HilD , without additional Ô¨Çanking sequences .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12107132" target="_blank">12107132</a>]<br></li>
<li>Furthermore , HilC and HilD are strikingly similar in primary sequence and bind to the same speciÔ¨Åc DNA sequences upstream of PhilA [ 52 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11755416" target="_blank">11755416</a>]<br></li>
<li>Furthermore , HilD and HilC bind the same DNA sites upstream of hilD and hilC ( 38 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24375101" target="_blank">24375101</a>]<br></li>
<li>In contrast , HilD and HilC have been shown to independently bind to the URS and promote hilA expression ( Schechter and Lee , 2001 ; Olekhnovich and Kadner , 2002 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>This is assumed to result from the interaction of HilC and HilD with their own promoters , activating transcriptions ( Lucas and Lee , 2001 ; Olekhnovich and Kadner , 2002 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15661008" target="_blank">15661008</a>]<br></li>
<li>In vitro , HilD and HilC bind the same regions upstream of hilA ( Schechter and Lee , 2001 ; Olekhnovich and Kadner , 2002 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>HilC and HilD respond to different environmental conditions , but either alone is capable of activating hilA expression ( Lucas and Lee , 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21570144" target="_blank">21570144</a>]<br></li>
<li>The hilA activators , HilD and HilC appear to overcome effects of downstream silencing region and disrupt repressive DNA loop .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16443238" target="_blank">16443238</a>]<br></li>
<li>HilD and HilC do not seem to regulate expression of hilD or prgH ( Olekhnovich and Kadner , 2002 ) ( Figs 1 and 2B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>Although HilD and HilC regulate two promoters upstream of hilC , they activate one and repress the other , such that overall HilD and HilC appear to have no effect on hilC expression ( Olekhnovich and Kadner , 2002 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>It is known that HilD can also control gene expression through HilC , SprB , RtsA and SsrA/B4 ,14 , and possibly , as suggested by our results , through SinR and SlyA ; thus , HilD could act on SL1896 through any of these regulators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27886269" target="_blank">27886269</a>]<br></li>
<li>HilA transcriptional expression is autoregulated and tightly modulated by the combined action of three AraC-like transcriptional-activators : HilC , HilD and RtsA [ 13,14 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28426789" target="_blank">28426789</a>]<br></li>
<li>As HilC and HilD activate the transcription from both HilA-dependent and - independent promoters , it is possible that Lon is involved in regulation of SPI1 gene expression by a pathway that depends on HilC and/or HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15661008" target="_blank">15661008</a>]<br></li>
<li>First , HilC and RtsA amplify HilA and HilD expression , in the sense that HilA and HilD expression is reduced the absence of HilC and RtsA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20686667" target="_blank">20686667</a>]<br></li>
<li>In this scheme , activation of invF transcription by HilD and HilC leads to co-transcriptional activation of the downstream sip genes , as has been shown for HilA-activated transcription from the invF promoter ( Darwin and Miller , 1999a ; Eichelberg et al. , 1999 ; Lostroh and Lee , 2000 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>Transcription of hilA is subject to complex control and is stimulated by the SPI-1-encoded HilC and HilD proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15126488" target="_blank">15126488</a>]<br></li>
<li>Induction of SPI1 effector-proteins requires three transcriptional-regulators of the AraC family : HilD , HilC and RtsA ( Schechter et al. , 1999 ; Ellermeier and Slauch , 2003 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27341691" target="_blank">27341691</a>]<br></li>
<li>Transcription from the hilA promoter is in turn mainly regulated by three transcription factors ; HilD , HilC and RtsA , which in a complex arrangement of feedback and feedforward loops bring about maximal induction of HilA ( Altier 2005 ; Jones 2005 ; Ellermeier and Slauch 2007 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19003447" target="_blank">19003447</a>]<br></li>
<li>The regulation of hilA promoter activity is complex , involving the coactivators HilC , HilD and RtsA , as well as other factors which act upstream of these proteins ( Lostroh and Lee , 2001 ; Ellermeier et al. , 2005 ; and references therein ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17163975" target="_blank">17163975</a>]<br></li>
<li>Specifically , HilC , HilD , and RtsA are all capable of individually activating the PhilA PhilC , PhilD , and PrtsA promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20686667" target="_blank">20686667</a>]<br></li>
<li>As HilC and HilD interact with their own promoters and activate their transcription ( Lucas and Lee , 2001 ; Olekh-novich and Kadner , 2002 ) , the increased transcription from hilC and hilD promoters in the Dlon mutant might not result from the accumulation of putative regulators but from the accumulation of HilC and HilD themselves .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15661008" target="_blank">15661008</a>]<br></li>
<li>-- Diagram showing the transcriptional units of Salmonella enterica SPI-1 and the regulatory circuits under the control of transcription factors HilA , HilD , HilC , RtsA , and InvF ( adapted from Lostroh and Lee 2001 ; Ellermeier and Slauch 2003 ; Altier 2005 ; Jones 2005 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20008574" target="_blank">20008574</a>]<br></li>
<li>This work demonstrates the regulation of rtsA , hilC , hilD , and hilA by HilC , HilD and RtsA , and that each regulator has a role in the host during infection .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17208038" target="_blank">17208038</a>]<br></li>
<li>Here we report that RtsA and the related hilA regulators , HilD and HilC , coordinately regulate expression of dsbA and the SPI1 TTSS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14679226" target="_blank">14679226</a>]<br></li>
<li>Acetylation of K297 Regulating HilD DNA-Binding Ability and Stability At the transcriptional level , the expression of hilD is positively autoregulated and modulated by a feed-forward loop involving HilD itself and regulators HilC and RtsA [ 3 , 28 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28329249" target="_blank">28329249</a>]<br></li>
<li>Transcriptional activation by HilC and HilD relieves repression of the hilA promoter by the nucleoid proteins H-NS and Hha ( Olekhnovich and Kadner 2006 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20008574" target="_blank">20008574</a>]<br></li>
<li>The appropriate concentration and activity of HilD is maintained by additional negative and positive regulatory mechanisms , including degradation of HilD mediated by the Lon protease ( Boddicker and Jones , 2004 ; Takaya et al. , 2005 ) , HilD inactivation by direct interaction with HilE ( Baxter et al. , 2003 ) , positive auto-regulation of HilD , as well as by a feed forward positive regulatory loop involving the HilD , HilC and RtsA AraC-like regulators ( Olekhnovich and Kadner , 2002 ; Ellermeier et al. , 2005 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21518393" target="_blank">21518393</a>]<br></li>
<li>Transcription of the hilA gene is also stimulated by the SPI1-encoded AraC/XylS-like regulatory proteins HilC , HilD and RtsA ( Eichelberg and Galan , 1999 ; Lostroh et al. , 2000 ; Rhen and Dorman , 2005 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28631437" target="_blank">28631437</a>]<br></li>
<li>In the SPI1 T3SS , HilD , HilC , and RtsA form three interlocking positive-feedback loops where all three activate each other 's and their own expression ( 23 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20833811" target="_blank">20833811</a>]<br></li>
<li>HilD can be repressed by HilE , and HilC and RstA further amplify the inducing signal .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21984608" target="_blank">21984608</a>]<br></li>
<li>Indeed , under inducing conditions , HilD and HilC , and probably RtsA , counteract the strong HilA and RtsA depression triggered by H-NS ( 292 , 293 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23554419" target="_blank">23554419</a>]<br></li>
				</div></ul><br>
			<label class="collapse_DOM" for="_2"><font size="5px">HilC protein domains</font></label>
				<input id="_2" type="checkbox">
				<div class="DOM"><br><ul><li>Each block shows data for two or three TFs at a single 400-bp genomic region , centered on a ChIP-seq peak for HilC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27601571" target="_blank">27601571</a>]<br></li>
<li>Amino acid sequence alignment between HilC and HilD has revealed that both proteins are highly conserved in a region of 99 C-terminal residues , which forms two potential a-helix -- turn -- a-helix DNA-binding motifs in AraC/XylS-family proteins ( Gallegos et al. , 1997 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15661008" target="_blank">15661008</a>]<br></li>
<li>Both HilC and HilD are predicted to be members of the AraC/XylS-family of transcriptional activators , based on homology within their C-terminal domains that contain a characteristic helix-turn-helix DNA-binding motif ( Eichelberg et al. , 1999 ; Rakeman et al. , 1999 ; Schechter et al. , 1999 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15765064" target="_blank">15765064</a>]<br></li>
<li>As members of the AraC-family , HilC and HilD monomers are expected to bind to a 17-bp region , in which each helix-turn helix motif binds to 3 - to 6-bp stretches on successive DNA helical turns ( 29 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12107132" target="_blank">12107132</a>]<br></li>
<li>Similar to HilC and HilD , RtsA belongs to the AraC/XylS-family of regulators , while RtsB is a helix-turn-helix DNA-binding protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15765064" target="_blank">15765064</a>]<br></li>
<li>In summary , stimulation of hilA transcription by the AraC/XylS-family members HilC and HilD does exhibit the expected dependence on residues in - CTD , but activation of transcription involves other - CTD-independent processes as well .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15126488" target="_blank">15126488</a>]<br></li>
				<label class="DOM_collapse_similarity" for="dom_similarity_7">
Olekhnovich , I.N. , and Kadner , R.J. ( 2004 ) Contribution of the RpoA C-terminal domain to stimulation of the Salmo-nella enterica hilA promoter by HilC and HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16045614" target="_blank">16045614</a>]<br></label>
<input id="dom_similarity_7", type="checkbox">
<div class="DOM_similar">
<li>Contribution of the RpoA C-terminal domain to stimulation of the Salmonella enterica hilA promoter by HilC and HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16443238" target="_blank">16443238</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17675384" target="_blank">17675384</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15968047" target="_blank">15968047</a>]<br></li></div>
<li>1/4 : √∞ 1/2 HilC 1/2 RtsA √ûH4 kH4 √æ √∞ 1/2 HilC 1/2 RtsA √ûH4 4 s 4 This corresponds to activation via the heterodimeric complex of HilC and RtsA with cooperativity H4 and threshold of activation K4 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19003447" target="_blank">19003447</a>]<br></li>
<li>Although the motifs are similar to each other , subtle differences in these motifs likely explain the overlapping but distinct binding proÔ¨Åles of HilC and RtsA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27601571" target="_blank">27601571</a>]<br></li>
<li>Various in vitro and in vivo conditions must be tested to determine whether direct activation of invF by HilC and/or HilD occurs in wild-type bacteria and whether this has any relevance in vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10844688" target="_blank">10844688</a>]<br></li>
<li>To study DNA-binding in vitro , we used Sumo-HilD and Sumo-HilC fusion proteins ( 27 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26884427" target="_blank">26884427</a>]<br></li>
<li>HilA expression , in turn , is regulated by three AraC-like transcription factors - HilC , HilD , and RtsA -- with homologous DNA-binding domains [ 22,31,32 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20686667" target="_blank">20686667</a>]<br></li>
<li>The N-terminal domains of HilD , HilC , and RtsA share approximately 10 % identity , whereas the C-terminal DNA-binding domains share 46 % identity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29378886" target="_blank">29378886</a>]<br></li>
<li>Peptide sequences for the HilD , HilC , and RtsA DNA-binding domains were aligned using ClustalW ( 30 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24375101" target="_blank">24375101</a>]<br></li>
<li>By aligning the sequences of HilD , HilC , and RtsA DNA-binding domains with that of MarA using Pfam ( 36 ) , we predicted the amino acids that make base-speciÔ¨Åc contacts with DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24375101" target="_blank">24375101</a>]<br></li>
<li>The HilD and perhaps the HilC proteins play central roles in controlling hilA transcription and may counteract this repressor activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12107132" target="_blank">12107132</a>]<br></li>
<li>Some members of the AraC/XylS-family have difÔ¨Åcult solubility properties , and the HilC and HilD proteins lost DNA-binding activity upon prolonged storage .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12107132" target="_blank">12107132</a>]<br></li>
<li>HilD and HilC are homologous , particularly in their DNA-binding domains , and thus bind the same sites in the various promoters that they regulate ( 26 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26884427" target="_blank">26884427</a>]<br></li>
<li>HilD and HilC have very similar DNA-binding and transcription-activation domains ( Schechter and Lee , 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>For HilC , no base substitution abolished binding and important positions were located in two short blocks ( Fig. 8A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12107132" target="_blank">12107132</a>]<br></li>
<li>All positions that were important for HilC binding were also important for HilD binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12107132" target="_blank">12107132</a>]<br></li>
<li>Substitutions unimportant for HilC binding generally had little effect on HilD binding , except for the segment from residues 60 to 56 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12107132" target="_blank">12107132</a>]<br></li>
				</div></ul><br>
			<label class="collapse_EVO" for="_3"><font size="5px">HilC evolution</font></label>
				<input id="_3" type="checkbox">
				<div class="EVO"><br><ul><li>HilD remaning ( % Low-osmolarity High-oxygen lon + Dlon lon + Dlon HilC HilD Fig. 7 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15661008" target="_blank">15661008</a>]<br></li>
<li>Blackwell Science , LtdOxford , UKMMIMolecular Microbiology 0950-382X Blackwell Publishing Ltd , 200347Original ArticleHilD and HilC directly activate invF expressionS.Akbar , L.M.Schechter , C.P.Lostroh and C.A.Lee Molecular Microbiology ( 2003 ) 47 ( 3 ) , 715 -- 728 AraC/XylS family members , HilD and HilC , directly activate virulence gene expression independently of HilA in Salmonella typhimurium Samina Akbar , ‚Ä† Lisa M. Schechter , ‚Ä° C. Phoebe Lostroh ¬ß and Catherine A. Lee * Department of Microbiology and Molecular Genetics , Harvard Medical School , 200 Longwood Avenue , Boston , MA 02115 , USA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
				<label class="EVO_collapse_similarity" for="evo_similarity_3">
AraC/XylS family members , HilD and HilC , directly activate virulence gene expression independently of HilA in Salmonella typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23554419" target="_blank">23554419</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18723621" target="_blank">18723621</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17600486" target="_blank">17600486</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14645287" target="_blank">14645287</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21168230" target="_blank">21168230</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16443238" target="_blank">16443238</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15790293" target="_blank">15790293</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15968047" target="_blank">15968047</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19074398" target="_blank">19074398</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923082" target="_blank">12923082</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14679226" target="_blank">14679226</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15126488" target="_blank">15126488</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17675384" target="_blank">17675384</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15731035" target="_blank">15731035</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25135218" target="_blank">25135218</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15039320" target="_blank">15039320</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18441068" target="_blank">18441068</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15516559" target="_blank">15516559</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18160484" target="_blank">18160484</a>]<br></label>
<input id="evo_similarity_3", type="checkbox">
<div class="EVO_similar">
<li>AraC / XylS family members , HilD and HilC , directly activate virulence gene expression independently of HilA in Salmonella typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15256548" target="_blank">15256548</a>]<br></li>
<li>AraC/XylS family members , HilD and HilC , directly activate virulence gene expression independently of HilA in Salmonella typhi-murium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28329249" target="_blank">28329249</a>]<br></li>
<li>AraC/XylS family members , HilD and HilC , directly activate virulence gene expression independently of HilA in Salmonella Typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24375101" target="_blank">24375101</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27601571" target="_blank">27601571</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27185788" target="_blank">27185788</a>]<br></li>
<li>Akabar , S. , Schechter , L.M. , Lostroh , C.P. and Lee , C.A. ( 2003 ) AraC/XylS family members , HilD and HilC , directly activate virulence gene expression independently of HilA in Salmonella typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17329372" target="_blank">17329372</a>]<br></li>
<li>References Akbar , S. , Schechter , L.M. , Lostroh , C.P. , and Lee , C.A. ( 2003 ) AraC/XylS family members , HilD and HilC , directly activate virulence gene expression independently of HilA in Salmonella typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16999831" target="_blank">16999831</a>]<br></li>
<li>Akbar , S. , Schechter , L.M. , Lostroh , C.P. and Lee , C.A. ( 2003 ) AraC/XylS family members , HilD and HilC , directly activate virulence gene expression independently of HilA in Salmonella typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28335027" target="_blank">28335027</a>]<br></li>
<li>Akbar , S. , Schechter , L.M. , Lostroh , C.P. , and Lee , C.A. ( 2003 ) AraC/XylS family members , HilD and HilC , directly activate virulence gene expression independently of HilA in Salmonella typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16045614" target="_blank">16045614</a>]<br></li>
<li>Akbar , S. , L. M. Schechter , C. P. Lostroh , and C. A. Lee , 2003 AraC/XylS family members , HilD and HilC , directly activate virulence gene expression independently of HilA in Sal-monella typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25991823" target="_blank">25991823</a>]<br></li>
<li>References Akbar , S. , Schechter , L. , Lostroh , C. , and Lee , C. ( 2003 ) AraC/XylS family members , HilD and HilC , directly activate virulence gene expression independently of HilA in Salmo-nella typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15661008" target="_blank">15661008</a>]<br></li>
<li>LITERATURE CITED Akbar , S. , L. M. Schechter , C. P. Lostroh and C. A. Lee , 2003 AraC/XylS family members , HilD and HilC , directly activate virulence gene expression independently of HilA in Salmo-nella typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20008574" target="_blank">20008574</a>]<br></li></div>
				<label class="EVO_collapse_similarity" for="evo_similarity_14">
Mol Microbiol 90:1216 -- 1232 Akbar S , Schechter LM , Lostroh CP , Lee CA ( 2003 ) AraC/XylS family members , HilD and HilC , directly activate virulence gene expression independently of HilA in Salmonella typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27549210" target="_blank">27549210</a>]<br></label>
<input id="evo_similarity_14", type="checkbox">
<div class="EVO_similar">
<li>Akbar S. , Schechter L.M. , Lostroh C.P. , Lee C.A. ( 2003 ) AraC/XylS family members , HilD and HilC , directly activate virulence gene expression independently of HilA in Salmonella typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22004521" target="_blank">22004521</a>]<br></li></div>
				<label class="EVO_collapse_similarity" for="evo_similarity_16">
AraC/XylS family members , HilC and HilD , directly bind and derepress the Salmonella typhi-murium hilA promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28329249" target="_blank">28329249</a>]<br></label>
<input id="evo_similarity_16", type="checkbox">
<div class="EVO_similar">
<li>AraC/XylS family members , HilC and HilD , directly bind and derepress the Salmonella Typhimurium hilA promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24375101" target="_blank">24375101</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23504014" target="_blank">23504014</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25547794" target="_blank">25547794</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27601571" target="_blank">27601571</a>]<br></li>
<li>AraC/XylS family members , HilC and HilD , directly bind and derepress the Sal-monella typhimurium hilA promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28426789" target="_blank">28426789</a>]<br></li>
<li>AraC/XylS family members , HilC and HilD , directly bind and derepress the S. typhimurium hilA promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12933899" target="_blank">12933899</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11466291" target="_blank">11466291</a>]<br></li>
<li>AlaC/XylS family members , HilC and HilD , directly bind and derepress the Salmonella typhimurium hilA promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18723621" target="_blank">18723621</a>]<br></li>
<li>SCHECHTER L.M. , LEE C.A. : AraC/XylS family members , HilC and HilD , directly bind and derepress the Salmonella typhimurium hilA promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17575908" target="_blank">17575908</a>]<br></li>
<li>AraC/XylS family members , HilC and HilD , directly bind and derepress the Salmonella typhimurium hilA promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25972862" target="_blank">25972862</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19218381" target="_blank">19218381</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14645287" target="_blank">14645287</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18245288" target="_blank">18245288</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21168230" target="_blank">21168230</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26441883" target="_blank">26441883</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16443238" target="_blank">16443238</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15790293" target="_blank">15790293</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16949866" target="_blank">16949866</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26300871" target="_blank">26300871</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15968047" target="_blank">15968047</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12595445" target="_blank">12595445</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19074398" target="_blank">19074398</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12107132" target="_blank">12107132</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17951383" target="_blank">17951383</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923082" target="_blank">12923082</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14679226" target="_blank">14679226</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15126488" target="_blank">15126488</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17675384" target="_blank">17675384</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29857034" target="_blank">29857034</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15731035" target="_blank">15731035</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25135218" target="_blank">25135218</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15039320" target="_blank">15039320</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12511499" target="_blank">12511499</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21518393" target="_blank">21518393</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18441068" target="_blank">18441068</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16980492" target="_blank">16980492</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24488311" target="_blank">24488311</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18160484" target="_blank">18160484</a>]<br></li>
<li>AraC/XylS family members HilC and HilD , directly bind and derepress the Salmonella typhimurium hilA promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15765064" target="_blank">15765064</a>]<br></li>
<li>Schechter , L. M. , and C. A. Lee , 2001 AraC/XylS family members , HilC and HilD , directly bind and derepress the Salmonella typhi-murium hilA promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25991823" target="_blank">25991823</a>]<br></li>
<li>Schechter , L.M. , and Lee , C.A. ( 2001 ) AraC/XylS family members , HilC and HilD , directly bind and derepress the Salmonella typhimurium hilA promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17971080" target="_blank">17971080</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16045614" target="_blank">16045614</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16999831" target="_blank">16999831</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17163975" target="_blank">17163975</a>]<br></li>
<li>Schechter , L.M. and Lee , C.A. ( 2001 ) AraC/XylS family members , HilC and HilD , directly bind and derepress the Salmonella typhimurium hilA promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17329372" target="_blank">17329372</a>]<br></li>
<li>Schechter , L. , and Lee , C. ( 2001 ) AraC/XylS family members , HilC and HilD , directly bind and derepress the Sal-monella typhimurium hilA promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15661008" target="_blank">15661008</a>]<br></li>
<li>Schechter , L.M. , and Lee , C.A. ( 2001 ) AraC/XylS family members , HilC and HilD , directly bind and derepress the S. typhimurium hilA promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>Mol Microbiol 32 : 629 -- 642 Schechter LM , Lee CA ( 2001 ) AraC/XylS family members , HilC and HilD , directly bind and derepress the Salmonella typhimurium hilA promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22922465" target="_blank">22922465</a>]<br></li>
<li>Schechter LM , Lee CA : AraC/XylS family members , HilC and HilD , directly bind and derepress the Salmonella typhimurium hilA promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23442379" target="_blank">23442379</a>]<br></li>
<li>Schechter LM , Lee CA ( 2001 ) AraC/XylS family members , HilC and HilD , directly bind and derepress the Salmonella typhimurium hilA promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18725932" target="_blank">18725932</a>]<br></li>
<li>Schechter , L. M. & Lee , C. A. AraC/XylS family members , HilC and HilD , directly bind and derepress the Salmonella typhimurium hilA promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27886269" target="_blank">27886269</a>]<br></li>
<li>Schechter L.M. , Lee C.A. ( 2001 ) AraC/XylS family members , HilC and HilD , directly bind and derepress the Salmonella typhimurium hilA promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22004521" target="_blank">22004521</a>]<br></li></div>
<li>The N-terminal domains of HilD , HilC , and RtsA share approximately 10 % identity , whereas the C-terminal DNA binding domains share 46 % identity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29378886" target="_blank">29378886</a>]<br></li>
<li>HilC and RtsA are homologues of HilD with 62 % and 61 % identity with HilD , respectively , in their DNA-binding domains ( Fig. 3A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24375101" target="_blank">24375101</a>]<br></li>
<li>In this region , RtsA carries 56 % identity and 76 % similarity to HilC , and 60 % identity and 75 % similarity to HilD , whereas HilC carries 58 % identity and 72 % similarity to HilD [ 25 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17208038" target="_blank">17208038</a>]<br></li>
<li>This homology is most signiÔ¨Åcant in the car-boxy-terminal third of the proteins , which contains the putative DNA binding domain ( RtsA versus HilC , 56 % identity and 76 % similarity ; RtsA versus HilD , 60 % identity and 75 % similarity ; HilC versus HilD , 58 % identity and 72 % similarity ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923082" target="_blank">12923082</a>]<br></li>
<li>However , the N-terminal domains , which presumably act to sense some environmental parameter , also show limited homology ( RtsA versus HilC , 25 % identity and 42 % similarity ; RtsA versus HilD , 21 % identity and 46 % similarity ; HilC versus HilD , 24 % identity and 50 % similarity ) RtsA , HilC , and HilD apparently function in very similar fashion .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923082" target="_blank">12923082</a>]<br></li>
<li>Amino acid sequence alignment between HilC and HilD has revealed that both proteins are highly conserved in a region of 99 C-terminal residues , which forms two potential a-helix -- turn -- a-helix DNA binding motifs in AraC/XylS family proteins ( Gallegos et al. , 1997 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15661008" target="_blank">15661008</a>]<br></li>
<li>HilA belongs to the OmpR/ToxR family whereas HilD , HilC and InvF belong to the AraC/XylS family of transcriptional regulators ( Kaniga et al. , 1994 ; Bajaj et al. , 1995 ; Schechter et al. , 1999 ; Lostroh et al. , 2000 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>Both HilC and HilD are predicted to be members of the AraC/XylS family of transcriptional activators , based on homology within their C-terminal domains that contain a characteristic helix-turn-helix DNA binding motif ( Eichelberg et al. , 1999 ; Rakeman et al. , 1999 ; Schechter et al. , 1999 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15765064" target="_blank">15765064</a>]<br></li>
<li>In addition , these three regulators ( i.e. , HilC , HilD , and RtsA ) activate a regulator within SPI-1 , HilA , which shows homology with the OmpR/ToxR family of DNA-binding proteins and induces transcriptional activation of SPI-1 genes in response to a variety of stimuli ( 1 , 59 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21075923" target="_blank">21075923</a>]<br></li>
<li>In addition to HilC and HilD , RtsA , which belongs to the AraC/XylS family of transcriptional regulators , has been shown to activate expression of SPI1 genes by binding upstream of hilA to induce its expression ( 13 , 14 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18245288" target="_blank">18245288</a>]<br></li>
<li>As members of the AraC family , HilC and HilD monomers are expected to bind to a 17-bp region , in which each helix-turn helix motif binds to 3 - to 6-bp stretches on successive DNA helical turns ( 29 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12107132" target="_blank">12107132</a>]<br></li>
<li>Similar to HilC and HilD , RtsA belongs to the AraC/XylS family of regulators , while RtsB is a helix-turn-helix DNA binding protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15765064" target="_blank">15765064</a>]<br></li>
<li>However , HilA activation is dependent upon a coupled , positive-feedback loop comprising the AraC/XylS family transcription factors HilD , HilC , and RtsA , which drive HilA expression above the SPI-1 activation threshold ( 7 , 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24375101" target="_blank">24375101</a>]<br></li>
<li>HilD , HilC , and RtsA are homologous AraC family transcription factors .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24375101" target="_blank">24375101</a>]<br></li>
<li>Within RESULTS SPI1 , HilD , a transcriptional regulator of the AraC family , comprises a portion of an auto-regulatory circuit that includes RtsA and HilC ( Ellermeier et al. , 2005 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23289537" target="_blank">23289537</a>]<br></li>
<li>Induction of SPI1 effector proteins requires three transcriptional regulators of the AraC family : HilD , HilC and RtsA ( Schechter et al. , 1999 ; Ellermeier and Slauch , 2003 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27341691" target="_blank">27341691</a>]<br></li>
<li>HilD and HilC may be members of this new family of regulators whose activity is not restricted to promoter-proximal upstream binding sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>The proteins HilD ( 34.3 kDa ) , HilC ( 32.8 kDa ) , and RtsA ( 32.3 kDa ) belong to the AraC/XylS family of transcriptional activators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17675384" target="_blank">17675384</a>]<br></li>
<li>Expression of hilA itself is activated by three AraC family regulators , HilC , HilD , and RtsA ( 25 , 78 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18160484" target="_blank">18160484</a>]<br></li>
<li>In summary , stimulation of hilA transcription by the AraC/XylS family members HilC and HilD does exhibit the expected dependence on residues in - CTD , but activation of transcription involves other - CTD-independent processes as well .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15126488" target="_blank">15126488</a>]<br></li>
<li>Expression of the SPI1 T3SS is tightly regulated by the combined action of HilC , HilD , and RtsA , three AraC family members that can independently activate hilA , which encodes the direct regulator of the SPI1 structural genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
<li>HilC and HilD , which are members of the AraC/XylS family , have been postulated to act as a derepressor of hilA expression by counteracting the action of negative regulatory elements at the hilA promoter ( 12,13 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17329372" target="_blank">17329372</a>]<br></li>
<li>HilC and HilD have recently been shown to bind to the hilA promoter , and it is believed that the binding of one or both of these proteins displaces repressors of hilA expression ( 55 ) , although both proteins are members of the AraC family of transcriptional activators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12595445" target="_blank">12595445</a>]<br></li>
<li>Like HilD , HilC is a member of the AraC/XylS family .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11755416" target="_blank">11755416</a>]<br></li>
<li>Similar to HilD , HilC is an activator of hilA of the AraC/XylS family ( 50 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11083797" target="_blank">11083797</a>]<br></li>
<li>Transcription of hilA is directly activated by HilC and HilD , both members of the AraC/XylS family of transcriptional regulators ( Schechter and Lee 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25991823" target="_blank">25991823</a>]<br></li>
<li>Two SPI1-encoded regulators of hilA expression are the AraC/XylS family members , HilC ( also called SirC or SprA ) and HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12107132" target="_blank">12107132</a>]<br></li>
<li>Two SPI1-encoded proteins , HilC ( SirC or SprA ) and HilD , both members of the AraC/XylS family of transcriptional regulators , induce expression of hilA ( 26 , 67 , 71 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923082" target="_blank">12923082</a>]<br></li>
<li>Two homologous proteins , HilC and HilD , both members of the AraC/XylS family of transcriptional regulators , induce hilA expression ( 12 , 51 , 53 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18245288" target="_blank">18245288</a>]<br></li>
<li>DNA-Binding Activities of the HilC and HilD Virulence Regulatory Proteins of Salmonella enterica Serovar Typhimurium Igor N. Olekhnovich and Robert J. Kadner * ent of M Departm icrobiology , School of Medicine , University of Virginia , Charlottesville , Virginia 22908-0734 Received 17 September 2001/Accepted 3 May 2002 The HilC and HilD proteins of Salmonella enterica serovar Typhimurium are members of the AraC/XylS family of transcription regulators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12107132" target="_blank">12107132</a>]<br></li>
<li>The HilC and HilD proteins are members of the AraC/XylS family of transcription activators , which typically bind to a 34-bp DNA target ( 18 , 36 , 42 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12107132" target="_blank">12107132</a>]<br></li>
<li>Three homologous proteins , 8,13,14 HilC , HilD , and RtsA , which belong to the AraC/XylS family of transcription factors can activate expression of hilA and some of its target genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16443238" target="_blank">16443238</a>]<br></li>
<li>HilD , HilC , and RtsA are members of the AraC family of transcription factors .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24375101" target="_blank">24375101</a>]<br></li>
<li>hilA transcription is elevated in response to two SPI1-encoded AraC/XylS family transcription factors , HilC and HilD ( 8 , 20 , 22 ) , and to the RtsA protein , which is encoded on another SPI ( 9 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15126488" target="_blank">15126488</a>]<br></li>
<li>HilC and HilD are members of the AraC/XylS family and activate transcription from the p and p promoters in an apparently redundant invF sicA manner ( Akbar et al. 2003 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20008574" target="_blank">20008574</a>]<br></li>
<li>Two members of the AraC/XylS family of transcription activators , HilC and HilD , can independently derepress hilA expression by binding directly to the upstream repressing sequence and counteracting its repressing effect in vitro and in vivo ( 24 , 30 , 31 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16391141" target="_blank">16391141</a>]<br></li>
<li>RtsA belongs to the AraC/XylS family of regulators , and can directly bind the hilA promoter and induce the hilA expression independent of HilC and HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15661008" target="_blank">15661008</a>]<br></li>
<li>Some members of the AraC/XylS family have difÔ¨Åcult solubility properties , and the HilC and HilD proteins lost DNA-binding activity upon prolonged storage .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12107132" target="_blank">12107132</a>]<br></li>
<li>Members of the AraC-like protein family , which includes HilC and HilD , have a well-documented ability to displace H-NS [ 45 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22457642" target="_blank">22457642</a>]<br></li>
<li>Like HilC and HilD proteins , InvF belongs to the AraC/XylS family of transcriptional regulators , and its gene lies within SPI-1 ( 28 , 33 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21075923" target="_blank">21075923</a>]<br></li>
<li>Sequence analysis suggests that RtsA is a member of the AraC/XylS family of transcriptional regulators and is related to HilC and HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923082" target="_blank">12923082</a>]<br></li>
<li>Sequence analysis of the regulatory operon revealed that RtsA belongs to the AraC 72 XylS family of transcriptional regulators and is homologous to both HilC ( 37 % identical and 55 % similar overall ) and HilD ( 34 % identical and 55 % similar overall ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923082" target="_blank">12923082</a>]<br></li>
<li>The third and fourth regulators are HilC and HilD , both of which belong to the AraC family .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17233907" target="_blank">17233907</a>]<br></li>
<li>RtsA , HilC , and HilD all belong to the AraC/XylS family of transcriptional regulators and appear to function in similar ways ( 26 , 65 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14679226" target="_blank">14679226</a>]<br></li>
<li>RtsA belongs to the AraC/XylS family of transcriptional regulators and contains similarity to HilC and HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923082" target="_blank">12923082</a>]<br></li>
<li>HilC and HilD , two SPI1-encoded AraC/XylS family members , have been called derepressors and are thought to counteract the repression of hilA by binding to sites within the upstream repressing sequence ( URS ) ( 24 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12933899" target="_blank">12933899</a>]<br></li>
<li>Three homologous proteins , RtsA , HilC and HilD , which belong to the AraC/XylS family of transcriptional regulators , bind to the DNA immediately upstream of hilA leading to induction of hilA expression ( Schechter and Lee , 2001 ; Olekhnovich and Kadner , 2002 ; Ellermeier and Slauch , 2003 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16045614" target="_blank">16045614</a>]<br></li>
<li>Recently , two SPI1-encoded factors , HilC and HilD , which derepress hilA expression , were identiÔ¨Åed and shown to be members of the AraC/XylS family of transcriptional regulators and to independently bind to these upstream repressing sequences to counteract the repression of hilA expression ( 32 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11741864" target="_blank">11741864</a>]<br></li>
<li>HilD and HilC , which are members of the AraC/XylS family of transcriptional regulators , can each individually bind to the DNA immediately upstream of hilA , and it is believed that this binding leads to hilA expression ( 44 , 45 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18723621" target="_blank">18723621</a>]<br></li>
				</div></ul><br>
			<label class="collapse_RP" for="_4"><font size="5px">Processes regulated by HilC</font></label>
				<input id="_4" type="checkbox">
				<div class="RP"><br><ul><li>Regulators , inducing signals Local regulators Global regulatory systems Activating environmental factors SPI2 SsrAB SPI1 InvF , HilA , HilC , HilD PhoPQ , OmpR/EnvZ phosphate starvation , low Mg2 + - or Ca2 + - level , late logarithmic to stationary growth phase high phosphate levels , high Mg2 + - or Ca2 + - level SirA , PhoPQ , ( SPI2 ) low O2-levels , high osmolarity , early logarithmic growth phase , bacterial protein-synthesis high O2-levels , stationary growth phase Repressing environmental factors Figure 3 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11418329" target="_blank">11418329</a>]<br></li>
<li>) e , f , u i EnvZ/OmpR , PhoPQ PhoRB , CsrAB , SirA/BarA , HilA , HilC HilD , InvF b , c , f , g , h , i HilA , SirA/BarA j , , SPI4 ( svr genes ) Unknown SPI5 ( sopB , pips ) PMN migration b , c HilA , SirA/BarA j HilA , SirA/BarA j sopE Invasion , cytokine expression b , c , k , l Iron uptake a Iron limitation a Fur a Macrophage line ( IVET ) m entF Mg uptake 21 n Cation levels n PhoPQ n Macrophages mgtCB Mouse spleen and macrophage line ( IVET ) ; epithelial cell line ( other fusion studies ) m SPI2 Cation levels , Pi levels , osmolarity ( ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10844688" target="_blank">10844688</a>]<br></li>
<li>Reference BioÔ¨Ålm-related function Group Quorum ( Merighi et al. 2009 ) a Membrane protein , Quorum Sensing ( AI-3 ) regulated pmrC sensing Metabolism Metabolism Putrescine transporter potF pps ( Hermans et al. 2011 ) ( White et al. 2010 ) Metabolism ( phosphoenolpyruvate synthase , conversion of pyruvate to PEP ) Needle complex inner membrane protein ( T3SS ) , PhoPQ repressed Purine synthesis , adenylosuccinate synthetase Purine synthesis , amidophosphoribosyltransferase Purine synthesis , bifunctional phosphoribosylaminoimidazolecarboxamide formyltransferase/IMP cyclohydrolase Activation of colanic-acid-biosynthesis , transcriptional co-regulator Activation of colanic-acid-biosynthesis , inhibition Ô¨Çagellar biosynthesis Stationary phase sigma factor , stress response Synthetic RpoS-dependent promoter Regulates SPI-1 expression together with HilD and HilC Hypothetical protein ( similar to E. coli vancomycin sensitivity gene ) Responses to Acyl Homoserine Lactones ( AI-1 ) , inÔ¨Çuences plasmid encoded Ô¨Åmbriae Transcriptional regulator in LysR family ( regulates motility in B. subtilis ) Response regulator of the BarA/SirA two-component system Fur regulated Salmonella iron-transporter Secreted effector protein ( T3SS ) prgH ( Steenackers et al. 2012 ) ( McClelland et al. 2001 ) a ( McClelland et al. 2001 ) a ( McClelland et al. 2001 ) a Virulence purA purF purH Metabolism Metabolism Metabolism rcsA ( Steenackers et al. 2012 ) Matrix rcsC ( Steenackers et al. 2012 ) Matrix rpoS ( Steenackers et al. 2012 ) ( White et al. 2006 ) ( Golubeva et al. 2012 ) ( Hermans et al. 2011 ) Regulator Synthetic Virulence Stress resistance Quorum sensing Putative sig38H4 rtsA sanA sdiA ( Steenackers et al. 2012 ) sinR ( Folkesson et al. 2002 , Marchal et al. 2004 ) ( Steenackers et al. 2012 ) Regulator sirA Metabolism Virulence sitA ( Hermans et al. 2011 ) sopB ( Hernandez et al. 2004 , Jennings et al. 2012 ) ( Steenackers et al. 2012 ) a Quorum sensing Coiled-coil protein , regulated by SdiA srgE ( Hansen et al. 2005 ) b Metabolism Stringent starvation protein , inhibits stationary-phase accumulation of H-NS Chaperon for sth Ô¨Åmbriae synthesis , bioÔ¨Ålm formation on epithelial cells Putative cytoplasmic protein Inner membrane protein Putative ABC transporter periplasmic binding protein Putative cytoplasmic protein Diguanylate cyclase ( c-di-GMP-synthesis ) , activates cellulose production ( matrix ) Putative DNA-binding protein Putative DNA-binding protein Transcriptional regulator ( Ô¨Çagella synthesis and pathogenicity ) Glycosyl transferase for the biosynthesis of colanic acid ( matrix ) Regulator , reduces bioÔ¨Ålm formation and induces motility Hypothetical protein Cell wall-associated hydrolase , contains a CsgD-binding site Regulatory protein Hydroxybutyrate dehydrogenase , O-Ag-capsule biosynthesis Regulator of O-Ag-capsule biosynthesis DNA-binding transcriptional repressor FabR , fatty-acid-biosynthesis Synthetic promoter , sigma-70 responsive ( growth control ) Synthetic promoter , sigma-70 responsive ( growth control ) sspA Fimbriae sthA ( Steenackers et al. 2012 ) Putative Putative Putative Putative Matrix sciJ ( Hermans et al. 2011 ) Hermans et al. unpublished ( Hermans et al. 2011 ) ( Hermans et al. 2011 ) ( Steenackers et al. 2012 ) STM0731 STM1255 STM1851 STM1987 Putative Putative Regulator STM3071 STM4423 tdcA ( Hermans et al. 2011 ) ( Hermans et al. 2011 ) ( Golubeva et al. 2012 ) Matrix wcaA ( Steenackers et al. 2012 ) Regulator Putative Putative ybaJ ybeL ydhO ( Barrios et al. 2006 ) ( Hermans et al. 2011 ) ( Ogasawara et al. 2011 ) Putative Matrix yhiW yihU ( Morgan et al. 2004 ) ( Steenackers et al. 2012 ) Matrix yihVW yijC ( Steenackers et al. 2012 ) ( Feng and Cronan 2009 ) a Metabolism ( Kim and Surette 2006 ) ( Stocki et al. 2007 ) œÉ70-16 œÉ70-7 Synthetic Synthetic Note : References for the bioÔ¨Ålm-related functions in Salmonella are shown .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24735176" target="_blank">24735176</a>]<br></li>
<li>SPI1 also encodes many transcriptional regulators , including HilA , HilD , HilC ( also called SprA/SirC ) , InvF and SprB ( Kaniga et al. , 1994 ; Bajaj et al. , 1995 ; Eichelberg et al. , 1999 ; Rakeman et al. , 1999 ; Schechter et al. , 1999 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>2005573691705Original ArticleModelforSPI1RegulationC.D.Ellermeier , J.R.EllermeierandJ.M.Slauch HilD , HilC and RtsA constitute a feed forward loop that controls expression of the SPI1 type three secretion system regulator hilA in Salmonella enterica serovar Typhimurium Craig D. Ellermeier ,1 ‚Ä† Jeremy R. Ellermeier1 and James M. Slauch1 ,2 * 1Department of Microbiology and 2College of Medicine , University of Illinois , Urbana , IL , USA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16045614" target="_blank">16045614</a>]<br></li>
<li>BioFactors 42:13 -- 40 Elhadad D , Desai P , Rahav G , McClelland M , Gal-Mor O ( 2015 ) Flagellin is required for host-cell-invasion and normal Salmonella pathogenicity island 1 expression by Sal-monella enterica serovar Paratyphi A. Infect Immun 83:3355 -- 3368 Ellermeier CD , Ellermeier JR , Slauch JM ( 2005 ) HilD , HilC and RtsA constitute a feed forward loop that controls expression of the SPI1 type three secretion system regulator hilA in Salmonella enterica serovar Typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27549210" target="_blank">27549210</a>]<br></li>
<li>J Bacteriol 185:5096 -- 5108 Ellermeier CD , Ellermeier JR , Slauch JM ( 2005 ) HilD , HilC and RtsA constitute a feed forward loop that controls expression of the SPI-1 type three secretion system regulator hilA in Salmo-nella enterica serovar Typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19003447" target="_blank">19003447</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_8">
Ellermeier , C.D. , Ellermeier , J.R , and Slauch , J.M. ( 2005 ) HilD , HilC and RtsA constitute a feed forward loop that controls expression of the SPI1 type three secretion system regulator hilA in Salmonella enterica serovar Typhi-murium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27341691" target="_blank">27341691</a>]<br></label>
<input id="rp_similarity_8", type="checkbox">
<div class="RP_similar">
<li>Ellermeier , C. D. , J. R. Ellermeier , and J. M. Slauch , 2005 HilD , HilC and RtsA constitute a feed forward loop that controls expression of the SPI1 type three secretion system regulator hilA in Salmonella enterica serovar Typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25991823" target="_blank">25991823</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22021388" target="_blank">22021388</a>]<br></li>
<li>Ellermeier , C.D. , Ellermeier , J.R. , and Slauch , J.M. ( 2005 ) HilD , HilC and RtsA constitute a feed forward loop that controls expression of the SPI1 type three secretion system regulator hilA in Salmonella enterica serovar Typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17971080" target="_blank">17971080</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16999831" target="_blank">16999831</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17163975" target="_blank">17163975</a>]<br></li>
<li>Ellermeier , C.D. , Ellermeier , J.R. and Slauch , J.M. ( 2005 ) HilD , HilC and RtsA constitute a feed forward loop that controls expression of the SPI1 type three secretion system regulator hilA in Salmonella enterica serovar Typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17329372" target="_blank">17329372</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24682814" target="_blank">24682814</a>]<br></li>
<li>Ellermeier , C. D. , J. R. Ellermeier and J. M. Slauch , 2005 HilD , HilC and RtsA constitute a feed forward loop that controls expression of the SPI1 type three secretion system regulator hilA in Salmonella enterica serovar Typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20008574" target="_blank">20008574</a>]<br></li>
<li>Ellermeier , C.D. , Ellermeier , J.R. , and Slauch , J.M. ( 2005 ) HilD , HilC and RtsA constitute a feed forward loop that controls expression of the SPI1 type three secretion system regulator hilA in Salmonella enterica serovar Typh-imurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22356617" target="_blank">22356617</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24354910" target="_blank">24354910</a>]<br></li>
<li>Mol Microbiol 48 ( 2 ) :385 -- 400 Ellermeier C , Ellermeier J , Slauch J ( 2005 ) HilD , HilC and RtsA constitute a feed forward loop that controls expression of the SPI1 type three secretion system regulator HilA in Salmonella enterica serovar Typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21107576" target="_blank">21107576</a>]<br></li>
<li>[ 19 ] C.D. Ellermeier , J.R. Ellermeier , J.M. Slauch , HilD , HilC and RtsA constitute a feed forward loop that controls expression of the SPI1 type three secretion system regulator hilA in Salmonella enterica serovar Typhimu-rium , Mol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18656407" target="_blank">18656407</a>]<br></li>
<li>Infect Immun 67 : 4099 -- 4105 Ellermeier CD , Ellermeier JR , Slauch JM ( 2005 ) HilD , HilC and RtsA constitute a feed forward loop that controls expression of the SPI1 type three secretion system regulator hilA in Salmonella enterica serovar Typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19756048" target="_blank">19756048</a>]<br></li>
<li>Clin Infect Dis 48:1096 -- 1101 Ellermeier CD , Ellermeier JR , Slauch JM ( 2005 ) HilD , HilC and RtsA constitute a feed forward loop that controls expression of the SPI1 type three secretion system regulator hilA in Salmonella enterica serovar Typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24500522" target="_blank">24500522</a>]<br></li>
<li>Ellermeier , C. D. , Ellermeier , J. R. & Slauch , J. M. HilD , HilC and RtsA constitute a feed forward loop that controls expression of the SPI1 type three secretion system regulator hilA in Salmonella enterica serovar Typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27886269" target="_blank">27886269</a>]<br></li>
<li>Ellermeier CD , Ellermeier JR , Slauch JM : HilD , HilC and RtsA constitute a feed forward loop that controls expression of the SPI1 type three secretion system regulator hilA in Salmonella enterica serovar Typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17208038" target="_blank">17208038</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24079299" target="_blank">24079299</a>]<br></li></div>
<li>Proc natl Acad Sci 108:4554 -- 4561 ellermeier CD , ellermeier Jr , Slauch JM ( 2005 ) HilD , HilC and rtsA constitute a feed forward loop that controls expression of the SPI1 type three secretion system regulator hilA in Salmonella enterica serovar Typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24929817" target="_blank">24929817</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_21">
HilD , HilC and RtsA constitute a feed forward loop that controls expression of the SPI1 type three secretion system regulator hilA in Salmonella enterica serovar Typhi-murium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18723621" target="_blank">18723621</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19218381" target="_blank">19218381</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24375101" target="_blank">24375101</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18245288" target="_blank">18245288</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20190046" target="_blank">20190046</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17951383" target="_blank">17951383</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23230292" target="_blank">23230292</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16988271" target="_blank">16988271</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26386070" target="_blank">26386070</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20889744" target="_blank">20889744</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26884427" target="_blank">26884427</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27601571" target="_blank">27601571</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17675384" target="_blank">17675384</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18469103" target="_blank">18469103</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18441068" target="_blank">18441068</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16980492" target="_blank">16980492</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18160484" target="_blank">18160484</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25547794" target="_blank">25547794</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24488311" target="_blank">24488311</a>]<br></label>
<input id="rp_similarity_21", type="checkbox">
<div class="RP_similar">
<li>HilD , HilC , and RtsA constitute a feed forward loop that controls expression of the SPI1 type three secretion system regulator hilA in Salmonella enterica serovar Typhi-murium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21075923" target="_blank">21075923</a>]<br></li>
<li>HilD , HilC and RtsA constitute a feed forward loop that controls expression of the SPI1 type three secretion system regulator HilA in Salmonella enterica serovar Typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25135218" target="_blank">25135218</a>]<br></li>
<li>HilD , HilC and RtsA constitute a feed forward loop that controls expression of the SPI1 type three secretion system regulator hilA in Salmonella enterica serovar Typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19703107" target="_blank">19703107</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23554419" target="_blank">23554419</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23289537" target="_blank">23289537</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17600486" target="_blank">17600486</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23459474" target="_blank">23459474</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19797361" target="_blank">19797361</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21168230" target="_blank">21168230</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27185788" target="_blank">27185788</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26441883" target="_blank">26441883</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20833811" target="_blank">20833811</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16443238" target="_blank">16443238</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26300871" target="_blank">26300871</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28329249" target="_blank">28329249</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21984608" target="_blank">21984608</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28426789" target="_blank">28426789</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28426789" target="_blank">28426789</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27704705" target="_blank">27704705</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28575106" target="_blank">28575106</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21518393" target="_blank">21518393</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27565525" target="_blank">27565525</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23676436" target="_blank">23676436</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24780330" target="_blank">24780330</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21680637" target="_blank">21680637</a>]<br></li></div>
<li>HilD , HilC , and RtsA not only regulate shared target genes but also activate each other 's transcription ( 8 , 10 , 11 , 20 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27601571" target="_blank">27601571</a>]<br></li>
<li>sion is dependent upon several transcription factors that are important for virulence , including PhoP , RtsA , SirA , HilC , HilD , and Fis ( 9 , 10 , 35 , 61 , 63 , 78 , 90 -- 92 , 97 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15968047" target="_blank">15968047</a>]<br></li>
<li>Arabinose-induced expression of HilC also dramatically increases invF expression ( Fig. 3 , striped bars ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>In contrast to invF-lacZ expression from pVV448 , HilD and HilC activate invF-lacZ expression over 45-fold from pSA7 ( Fig. 6 , black and striped bars , pSA7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>( A ) Immunoblotting analysis of the cellular lysates using anti-SipB , anti-SipC , anti-HilA , anti-HilC , and anti-HilD sera .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18245288" target="_blank">18245288</a>]<br></li>
<li>In these studies , overexpression of HilC ( SirC/SprA ) from a plasmid was found to activate expression of a subset of SPI1 genes in a hilA mutant strain background ( Eichelberg et al. , 1999 ; Rakeman et al. , 1999 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>Environmental signals from two-component systems and global regulators control the expression of the HilC , HilD , and HilA transcription factors , which together form a commitment circuit for the expression of SPI-1 genes ( Bajaj et al , 1996 ; Eichelberg and Galan , 1999 ; Lundberg et al , 1999 ; Kalir et al , 2001 ; Ellermeier et al , 2005 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19756048" target="_blank">19756048</a>]<br></li>
<li>The expression of SPI-1 genes plays an important role in Salmonella invasion of epithelial cells , which requires transcriptional regulators such as HilA , HilC , HilD , InvA , InvC , InvF , and SirA , ( Johnston et al. 1996 ; Ochman et al. 1996 ; Rakeman et al. 1999 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24500522" target="_blank">24500522</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_33">
Olekhnovich , I. N. , and Kadner , R. J. ( 2007 ) Role of nucleoid-associated proteins Hha and H-NS in expression of Salmonella enterica activators HilD , HilC , and RtsA required for cell invasion .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23515315" target="_blank">23515315</a>]<br></label>
<input id="rp_similarity_33", type="checkbox">
<div class="RP_similar">
<li>Olekhnovich , I. N. , and R. J. Kadner , 2007 Role of nucleoid-associated proteins Hha and H-NS in expression of Salmonella enterica activators HilD , HilC , and RtsA required for cell invasion .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22021388" target="_blank">22021388</a>]<br></li>
<li>Role of nucleoid-associated proteins Hha and H-NS in expression of Salmonella enterica activators HilD , HilC , and RtsA required for cell invasion .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23554419" target="_blank">23554419</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23459474" target="_blank">23459474</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27185788" target="_blank">27185788</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26441883" target="_blank">26441883</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21075923" target="_blank">21075923</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28660201" target="_blank">28660201</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20889744" target="_blank">20889744</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27601571" target="_blank">27601571</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25135218" target="_blank">25135218</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18441068" target="_blank">18441068</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25547794" target="_blank">25547794</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24488311" target="_blank">24488311</a>]<br></li>
<li>[ 33 ] I.N. Olekhnovich , R.J. Kadner , Role of nucleoid-associated proteins Hha and H-NS in expression of Salmonella enterica activators HilD , HilC , and RtsA required for cell invasion , J. Bacteriol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23040276" target="_blank">23040276</a>]<br></li>
<li>Olekhnovich , I. N. & Kadner , R. J. Role of nucleoid-associated proteins Hha and H-NS in expression of Salmonella enterica activators HilD , HilC , and RtsA required for cell invasion .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27886269" target="_blank">27886269</a>]<br></li>
<li>Olekhnovich IN , Kadner RJ : Role of nucleoid-associated proteins Hha and H-NS in expression of Salmonella enterica activators HilD , HilC , and RtsA required for cell invasion .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22017966" target="_blank">22017966</a>]<br></li></div>
				<label class="RP_collapse_similarity" for="rp_similarity_39">
J Biol Chem 266 : 815 -- 823 Takaya A , Kubota Y , Isogai E , Yamamoto T ( 2005 ) Degradation of the HilC and HilD regulator proteins by ATP-dependent Lon protease leads to downregulation of Salmonella pathogenicity island 1 gene-expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18200043" target="_blank">18200043</a>]<br></label>
<input id="rp_similarity_39", type="checkbox">
<div class="RP_similar">
<li>Takaya A. , Kubota Y. , Isogai E. , Yamamoto T. ( 2005 ) Degradation of the HilC and HilD regulator proteins by ATP-dependent Lon protease leads to downregulation of Salmonella pathogenicity island 1 gene-expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22004521" target="_blank">22004521</a>]<br></li>
<li>[ PubMed : 12193630 ] Takaya A , Kubota Y , Isogai E , Yamamoto T. Degradation of the HilC and HilD regulator proteins by ATP-dependent Lon protease leads to downregulation of Salmonella pathogenicity island 1 gene-expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21518393" target="_blank">21518393</a>]<br></li>
<li>Takaya A , Kubota Y , Isogai E , Yamamoto T. Degradation of the HilC and HilD regulator proteins by ATP-dependent Lon protease leads to downregulation of Salmonella pathogenicity island 1 gene-expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26810370" target="_blank">26810370</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28329249" target="_blank">28329249</a>]<br></li>
<li>Takaya , A. , Y. Kubota , E. Isogai and T. Yamamoto , 2005 Degradation of the HilC and HilD regulator proteins by ATP-dependent Lon protease leads to downregulation of Salmo-nella pathogenicity island 1 gene-expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20008574" target="_blank">20008574</a>]<br></li>
<li>Takaya , A. , Kubota , Y. , Isogai , E. , and Yamamoto , T. ( 2005 ) Degradation of the HilC and HilD regulator proteins by ATP-dependent Lon protease leads to downregulation of Salmonella pathogenicity island 1 gene-expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16045614" target="_blank">16045614</a>]<br></li></div>
<li>One such regulator , Lon , has been shown to degrade both HilC and HilD , thus down-regulating SPI1 gene-expression ( 58 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18245288" target="_blank">18245288</a>]<br></li>
<li>Recall that HilD , but not HilC , activates expression of invF sevenfold from the pVV448 reporter plasmid , which carries this region ( Fig. 4 , black bars ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>In the presence of an unknown environmental condition , HilC or HilD may activate invF expression , causing induction of invF-dependent genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10844688" target="_blank">10844688</a>]<br></li>
<li>HilA and HilC activate the invF gene , the product of which activates genes encoding secreted effectors , including sopB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14645287" target="_blank">14645287</a>]<br></li>
<li>Another SPI regulator , HilD , which is not associated with Fisbinding , acts in an ordered fashion with HilC to coordinately activate expression of the SPI-1 genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23717649" target="_blank">23717649</a>]<br></li>
<li>The Spi1 genes are highly regulated by a set of DNA-binding proteins , including the AraC-like regulators HilD , HilC , and RtsA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24488311" target="_blank">24488311</a>]<br></li>
<li>DISCUSSION Modulation by Pat on HilD K297 Acetylation The SPI-1 genes are tightly regulated by a set of transcription factors , including HilD , HilC , and RtsA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28329249" target="_blank">28329249</a>]<br></li>
<li>InvF and HilA are transcriptional regulators of SPI-1 genes that are encoded within SPI-1 ; hilA itself is regulated by several factors , including PhoP-PhoQ , SirA , HilC , and HilD ( 3 -- 5 , 20 , 37 , 44 , 66 , 72 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10633113" target="_blank">10633113</a>]<br></li>
<li>In addition , HilD and HilC are important AraC/XylS transcriptional regulators that mediate expression of HilA , which then activates all genes necessary for a functional SPI-1 encoded T3SS ( 29 , 30 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22004521" target="_blank">22004521</a>]<br></li>
<li>Within SPI-1 there are key transcriptional activators which regulate expression of SPI-1 genes : HilC , HilD , HilA , and InvF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22291596" target="_blank">22291596</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22291596" target="_blank">22291596</a>]<br></li>
<li>Transcriptional activators of SPI genes ( HilC , HilD , SsrB , and SlyA ) function in large part through displacement of H-NS from SPI promoters [ 16,44 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22457642" target="_blank">22457642</a>]<br></li>
<li>SPI-1 genes are organized in seven or more transcriptional units , whose expression is under the control of four SPI-encoded transcription factors : HilA , HilC , HilD , and InvF ( Lostroh and Lee 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20008574" target="_blank">20008574</a>]<br></li>
<li>The fundamental transcriptional regulators , HilA , HilC , HilD , InvF , HilE , and RtsA inside or outside SPI-1 play an important role in Ô¨Åne-tuning the expression of SPI-1 [ 34 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23370732" target="_blank">23370732</a>]<br></li>
<li>Dam-dependent regulation of SPI-1 is transmitted via HilD : In an attempt to identify the SPI-1 regulator ( s ) , if any , involved in transmission of Dam-dependent control to SPI-1 , we examined the involvement of the SPI-1 `` general '' transcription factors HilA , HilC , and HilD and the sip-speciÔ¨Åc transcription factor InvF ( Darwin and Miller 1999 ; Eichelberg and Galan 1999 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20008574" target="_blank">20008574</a>]<br></li>
<li>However , these experiments provided evidence that none of the HilA , HilC , RtsA , and InvF transcription factors is involved in Dam-dependent control of SPI-1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20008574" target="_blank">20008574</a>]<br></li>
<li>In addition , these three regulators ( i.e. , HilC , HilD , and RtsA ) activate a regulator within SPI-1 , HilA , which shows homology with the OmpR/ToxR family of DNA-binding proteins and induces transcriptional activation of SPI-1 genes in response to a variety of stimuli ( 1 , 59 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21075923" target="_blank">21075923</a>]<br></li>
<li>Expression of the SPI-1 T3SS and its associated genes ( several of which are encoded outside the SPI-1 pathogenicity island ) is controlled by a hierarchy of regulators ( HilD , HilA , HilC , RtsA , InvF ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29229736" target="_blank">29229736</a>]<br></li>
<li>First , Lon is induced to control turnover of HilD and HilC and hence limit the expression of SPI-1 genes , which is of particular relevance after invasion of epithelial cells .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23554419" target="_blank">23554419</a>]<br></li>
<li>SPI-1 also encodes transcriptional activators responsible for its own expression : HilA , HilC , HilD , and InvF ( Lostroh and Lee 2001 ; Ellermeier and Slauch 2007 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25991823" target="_blank">25991823</a>]<br></li>
<li>The SPI-1 type-III-secretion-system ( TTSS ) structural genes spaM-NOPQRS and invABCEFGH and their regulatory proteins HilA , HilC , HilD [ 31 ] were also identified and annotated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23234298" target="_blank">23234298</a>]<br></li>
<li>) such that HilD ( and also HilC and RtsA ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
<li>Through a complex feedback and feedforward mechanism , HilC and HilD control their own and each other 's transcription , and together activate transcription of hilA [ 14,15,16 ] ( Figure 1A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22457642" target="_blank">22457642</a>]<br></li>
<li>The panel of regulatory proteins ( Table 2 ) included SPI1-encoded transcription factors ( HilA , HilC and HilD ) and transcription factors previously reported to control SPI1 gene-expression ( HilE , BarA/SirA , Fur and FliZ ) [ 17 -- 24 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27564394" target="_blank">27564394</a>]<br></li>
<li>Many regulatory proteins inÔ¨Çuence invasion gene-expression directly and indirectly , including activators HilA , InvF , HilD , HilC , and RtsA and nucleoid-associated proteins H-NS and Hha ( 2 , 14 , 16 , 23 , 36 , 37 , 44 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17675384" target="_blank">17675384</a>]<br></li>
<li>By controlling expression/activity of HilD , HilC , RtsA , and HilA , many regulators contribute to SPI-1 gene regulation in response to environmental stimuli that include temperature ( H-NS and Hha ) , osmolarity ( H-NS and Hha ) , pH ( OmpR ) , iron availability ( Fur ) , and oxygen levels ( FNR ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27601571" target="_blank">27601571</a>]<br></li>
<li>The expression of hilA is regulated in response to variety of environmental signals , including pH ( 6 ) , growth phase ( 56 ) , osmolarity ( 36 ) , and oxygen ( 52 ) and by a number of proteins , including BarA/SirA ( 51 ) , CpxA ( 71 ) , CsrB ( RNA ) ( 2 ) , EnvZ/OmpR ( 63 ) , FadD ( 63 ) , Fis ( 94 ) , FliZ ( 63 ) , HilC ( 83 ) , HilD ( 83 ) , Hu ( 84 ) , Mlc ( 58 ) , and RtsA ( 28 ) , which positively regulate expression , and CsrA ( 2 ) , Hha ( 29 ) , HilE ( 7 ) , H-NS ( 84 ) , Lon ( 87 ) , PhoB ( 63 ) , and PhoP/PhoQ ( 9 ) , which negatively regulate expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19074398" target="_blank">19074398</a>]<br></li>
<li>HilC and HilD also activate the promoter PinvF-2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15661008" target="_blank">15661008</a>]<br></li>
<li>Although many factors and a complex mechanism related to environmental stimuli are involved in the modulation of SPI1 genes , they are thought to converge on the activation of several transcriptional regulators encoded within SPI1 , such as HilA , HilC , HilD and InvF ( 7 -- 9 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17329372" target="_blank">17329372</a>]<br></li>
<li>These regulators , HilD , HilC , HilA , and InvF , act in an ordered fashion to activate coordinated expression of the SPI1 genes ( 2 , 10 , 12 , 13 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18723621" target="_blank">18723621</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_74">
Previous studies showed that four SPI1-encoded transcriptional regulators , HilD , HilC , HilA and InvF , act in an ordered fashion to coordinately activate expression of the SPI1 TTSS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></label>
<input id="rp_similarity_74", type="checkbox">
<div class="RP_similar">
<li>Many studies have shown that the SPI1-encoded transcriptional regulators HilD , HilC , HilA , and InvF act in an ordered fashion to coordinately activate the expression of SPI1 ( for a review , see reference 36 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14977940" target="_blank">14977940</a>]<br></li></div>
<li>These results suggest that HilA , HilC , and HilD are each required for expression of siiE , but these regulators are likely to act in concert or in combination with additional SPI1-encoded regulatory loci to activate SPI4 gene-expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18160484" target="_blank">18160484</a>]<br></li>
<li>Models for invasion gene regulation in vitro In vitro characterization of HilA , HilC , HilD and InvF suggests that these regulators operate in a hierarchy to modulate invasion gene-expression with HilA acting as the central regulator in this cascade ( Fig. 1A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10844688" target="_blank">10844688</a>]<br></li>
<li>We conclude that HilD and HilC have a second role in SPI1 gene regulation that is separate from their role in co-ordinating expression of the SPI1 TTSS through hilA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>These data led us to propose a new model for SPI1 regulation , in which HilC , HilD and RtsA act in a complex feed-forward loop to control expression of hilA ( Figure 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17208038" target="_blank">17208038</a>]<br></li>
<li>The expression of hilA , the master regulator for SPI1 , is controlled by HilD , HilC , and RtsA , which act in a complex feed-forward loop .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20889744" target="_blank">20889744</a>]<br></li>
<li>Thereafter , the regulatory cascade including HilD , HilC , RtsA , ppGpp and other regulatory factors , in response to several other signals , dictates hilA expression levels .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21680637" target="_blank">21680637</a>]<br></li>
<li>HilC and RtsA , on the other hand , appear to play an ancillary role in regulating SPI1 gene-expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20686667" target="_blank">20686667</a>]<br></li>
<li>In addition to the negative control that was exerted by H-NS/Hha under low-osmolarity conditions , the homologous virulence activators HilD , HilC , and RtsA ( Hil activators ) induced rtsA-lac expression in a high-salt medium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17675384" target="_blank">17675384</a>]<br></li>
<li>The expression of the invasion-associated type-III-secretion-system in Salmonella is indeed subject to a remarkably complex regulation involving both speciÔ¨Åc ( InvF , HilA , HilB , and SprA-HilC ) and global ( PhoP-PhoQ , SirA , and RcsB-RcsC ) transcriptional regulatory proteins ( 1 , 4 , 5 , 9 , 11 , 24 , 27 , 36 , 46 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10768967" target="_blank">10768967</a>]<br></li>
<li>The expression of the type-III-secretion-system-related genes involved in invasion and intracellular survival is induced in the presence of transcriptional regulator proteins such as HilA , HilC , HilD , InvA , InvC , InvF , SirA , and SirB ( 11 , 27 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24780330" target="_blank">24780330</a>]<br></li>
<li>The expression of the genes , those regulate the type-III-secretion-system , are involved in completion of invasion , intracellular survival and multiplication , is induced by the transcriptional regulator proteins such as HilA , HilC , HilD , InvA , InvC , InvF , SirA , and SirB ( Johnston et al. , 1996 ; Rakeman et al. , 1999 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27565525" target="_blank">27565525</a>]<br></li>
<li>HilD , HilC and RtsA then participate in a feed-forward loop that converges on activation of hilA , which in turn activates transcription of structural components of the needle complex ( prg/org ) as well as invF , which is a transcriptional activator of secreted effector proteins ( sic/sip and others ) ( Figure 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28335027" target="_blank">28335027</a>]<br></li>
<li>Role of Nucleoid-Associated Proteins Hha and H-NS in Expression of Salmonella enterica Activators HilD , HilC , and RtsA Required for Cell Invasion Igor N. Olekhnovich * and Robert J. Kadner ‚Ä† icrobiology , University of Virginia School of Medicine , Charlottesville , Virginia 22908-0734 Received 8 June 2007/Accepted 23 July 2007 ent of M Departm The coordinate expression of Salmonella enterica invasion genes on Salmonella pathogenicity island 1 is under the control of the complex circuits of regulation that involve the AraC/XylS family transcriptional activators HilD , HilC , and RtsA and nucleoid-associated proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17675384" target="_blank">17675384</a>]<br></li>
<li>In this report , we investigate the role of nucleoid-associated proteins Hha and H-NS in the expression of virulence activator proteins HilD , HilC , and RtsA with an emphasis on the interplay of positive and negative transcriptional factors in the regulation of the rtsA promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17675384" target="_blank">17675384</a>]<br></li>
<li>Within RESULTS SPI1 , HilD , a transcriptional regulator of the AraC family , comprises a portion of an auto-regulatory circuit that includes RtsA and HilC ( Ellermeier et al. , 2005 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23289537" target="_blank">23289537</a>]<br></li>
<li>The regulation of hilA promoter activity is complex , involving the coactivators HilC , HilD and RtsA , as well as other factors which act upstream of these proteins ( Lostroh and Lee , 2001 ; Ellermeier et al. , 2005 ; and references therein ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17163975" target="_blank">17163975</a>]<br></li>
<li>Expression of hilA is under the control of many environmental conditions and three transcriptional activators : HilC , HilD , and RtsA ( Ellermeier et al. , 2005 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27704705" target="_blank">27704705</a>]<br></li>
<li>A - SPI1 B - SPI2 RtsA PhoP SlyA OmpR HilD HilC SirA HilA SsrAB InvF SicA Effectors Effectors RFe l igguraeto1ry cascades activating the expression of SPI1 ( A ) and SPI2 ( B ) related type III effectors Regulatory cascades activating the expression of SPI1 ( A ) and SPI2 ( B ) related type III effectors .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17233907" target="_blank">17233907</a>]<br></li>
<li>In order to clarify the Salmonella response-to-osmotic-stress , we investigated here the osmoregulation of the virulence activator proteins HilD , HilC , and RtsA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17675384" target="_blank">17675384</a>]<br></li>
<li>These include the transcription factors FliZ , HilC , and RtsA , which have each been demonstrated to positively regulate invasion gene-expression by promoting expression of the SPI-1 regulator hilD ( 9 , 10 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27185788" target="_blank">27185788</a>]<br></li>
<li>Discussion Expression of the SPI1 T3SS is controlled by HilD , HilC , and RtsA , acting in a complex feed-forward loop to activate the hilD , hilC , and rtsA genes , as well as hilA , which encodes the transcriptional activator of the T3SS structural genes ( Figure 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22021388" target="_blank">22021388</a>]<br></li>
<li>We have previously shown that three AraC-like regulators , HilD , HilC , and RtsA , act in a complex feed-forward regulatory loop to control the expression of the hilA gene , which encodes the direct regulator of the SPI1 structural genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20889744" target="_blank">20889744</a>]<br></li>
<li>We have recently shown that HilD , HilC , and RtsA act in a complex regulatory loop to control the transcription of the hilA gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17951383" target="_blank">17951383</a>]<br></li>
<li>The HilE CDS and sRNA regulons were distinct from the HilA , HilC and HilD regulons at IEP , and show a novel positive regulatory role for HilE in the control of metabolic gene-expression ( S3 Table ) , in addition to the reported role as a negative regulator of SPI1 genes [ 52 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27564394" target="_blank">27564394</a>]<br></li>
<li>As HilC and HilD activate the transcription from both HilA-dependent and - independent promoters , it is possible that Lon is involved in regulation of SPI1 gene-expression by a pathway that depends on HilC and/or HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15661008" target="_blank">15661008</a>]<br></li>
<li>It has been proposed earlier that the regulation of HilA expression is via a complex feedforward loop involving the transactivators HilD , HilC and RtsA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19003447" target="_blank">19003447</a>]<br></li>
<li>The regulation of the expression of HilA is via a complex feed-forward loop involving three transcriptional activators : HilC , HilD and RtsA , and the negative regulator HilE .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21107576" target="_blank">21107576</a>]<br></li>
<li>This work demonstrates the regulation of rtsA , hilC , hilD , and hilA by HilC , HilD and RtsA , and that each regulator has a role in the host during infection .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17208038" target="_blank">17208038</a>]<br></li>
<li>Here we report that RtsA and the related hilA regulators , HilD and HilC , coordinately regulate expression of dsbA and the SPI1 TTSS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14679226" target="_blank">14679226</a>]<br></li>
<li>Three AraC-like regulators , HilD , HilC , and RtsA , control expression of hilA , and thereby induction of the SPI1 system .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22021388" target="_blank">22021388</a>]<br></li>
<li>As shown in Fig. 8 , the increased effect of Lon depletion on hilA expression was abolished by depletion of both HilC and HilD , suggesting that both proteins are also involved in regulation of SPI1 by Lon under SPI1-repressing conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15661008" target="_blank">15661008</a>]<br></li>
<li>Our previous Ô¨Ånding that it negatively regulates the expression of the Ô¨Çagellar regulon by controlling the turnover of the master regulator , FlhD/FlhC ( 62 ) , allowed us to hypothesize that it may regulate HilC and HilD by controlling Ô¨Çagellar gene-expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18245288" target="_blank">18245288</a>]<br></li>
<li>As expected , disruption of the Ô¨ÇhDC operon , encoding the master regulator for expression of the Ô¨Çagellar regulon , blocked the increase of HilC and HilD levels caused by clpPX disruption .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18245288" target="_blank">18245288</a>]<br></li>
<li>Our evidence shows that ClpXP regulates cellular levels of HilC and HilD via the control of Ô¨Çagellar gene-expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18245288" target="_blank">18245288</a>]<br></li>
<li>Discussion To understand how Lon negatively regulates the expression of SPI1 genes , including the key regulator hilA , we have focused on two activators of hilA , HilC and HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15661008" target="_blank">15661008</a>]<br></li>
<li>On the basis of these results , we propose that Lon negatively regulates the expression of SPI1 genes through degradation of HilC and HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15661008" target="_blank">15661008</a>]<br></li>
<li>These data suggest that most of the 17 novel invasion genes are directly or indirectly involved in invasion and that their regulation TABLE 1 List of regulated noncoding RNAs identiÔ¨Åed in this study TF Fold Strand Start ( bp ) a , b Stop ( bp ) a , b Description regulator ( s ) changec 34013 * 33811 * Antisense to HilC 2.28 d 74905 74971 DapZ RtsA 2.66 868925 868081 Antisense to slrP RtsA 1.25 1343788 1343659 STnc520 SprB 4.08 1418433 1418523 Overlapping / HilC 1.18 1603873 1604092 MgrR HilD 2.12 1603938 1603781 Overlapping HilD , RtsA 3.73 , 2.05 e 3039507 3039347 Antisense to hilA 5 = UTR HilD 4.23 3065151 3066061 InvR HilD , HilC 1.07 , 3.99 e 3483808 3483985 Within dacB RtsA 2.54 a The numbers represent predicted start/stop genome coordinates for a given noncoding RNA ( ncRNA ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27601571" target="_blank">27601571</a>]<br></li>
<li>The key regulator of this secretion system is HilA , which is in turn regulated by HilD , HilC and RtsA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19537165" target="_blank">19537165</a>]<br></li>
<li>RtsA is a close homologue of HilD and HilC but is encoded outside SPI-1 , in an operon that includes another TF-encoding gene , rtsB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27601571" target="_blank">27601571</a>]<br></li>
<li>The transcriptional activators HilD and HilC play an important role in controlling hilA expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25547794" target="_blank">25547794</a>]<br></li>
<li>Effect of HilD , HilC or HilA on chromosomal invF-lacZ fusion expression in a hilA hilD mutant .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>Still , no one has reported a condition in which wild-type S. typhimurium represses hilA expression while simultaneously activating expression of invF and invF-dependent genes through HilC or HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10844688" target="_blank">10844688</a>]<br></li>
<li>Effect of HilD , HilC and HilA on invF expression from pSA7 and mutant derivatives of pSA7 in a Dspi1 mutant .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>( A ) Timecourse simulation of HilD , HilC , RtsA , and HilA expression dynamics in wild-type cells .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20686667" target="_blank">20686667</a>]<br></li>
<li>( C -- D ) Comparison of HilC expression at single-cell resolution in wild type ( C ) and DhilD mutant ( D ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20686667" target="_blank">20686667</a>]<br></li>
<li>We detected positive regulation of 7/8 genes by both HilC and RtsA ( Fig. 3B and C ; raw data , including for empty vector controls , shown in Table S4 in the supplemental material ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24375101" target="_blank">24375101</a>]<br></li>
<li>Under inducing conditions , activation of BarA/SirA initiates a regulatory cascade involving HilD , HilC , and HilA , which leads both directly and indirectly to the expression of SPI1 and SPI4 genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18160484" target="_blank">18160484</a>]<br></li>
<li>For example , four regulators ( HilA , InvF , HilC and HilD ) directly or indirectly modulate expression of genes encoding the SPI1 secretion apparatus and its secreted effectors ( extensively discussed below ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10844688" target="_blank">10844688</a>]<br></li>
<li>HilD also regulates expression of the additional regulators HilC and RtsA , which also control HilA expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29229736" target="_blank">29229736</a>]<br></li>
<li>More recently , H-NS has been shown to repress the expression of HilA , HilD , HilC , and RtsA under noninducing conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23554419" target="_blank">23554419</a>]<br></li>
<li>( D ) Association of ChIP-seq peaks with regulation for HilD , HilC , and RtsA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27601571" target="_blank">27601571</a>]<br></li>
<li>We also show regulation of many HilD target genes by HilC and RtsA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24375101" target="_blank">24375101</a>]<br></li>
<li>HilD , HilC , and RtsA all activate transcription of hilA , hilC , hilD , dsbA , invF , slrP , rtsA , and genes in SPI-4 ( 6 , 15 , 32 , 33 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24375101" target="_blank">24375101</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_129">
HilC , HilD , and RtsA are each capable of activating expression of hilC , hilD , and rtsA , creating a complex regulatory loop that controls hilA expression ( Fig. 1 ) ( 15 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17951383" target="_blank">17951383</a>]<br></label>
<input id="rp_similarity_129", type="checkbox">
<div class="RP_similar">
<li>HilC , HilD , and RtsA are all also capable of activating expression of hilC , hilD , and rtsA independent of each other and comprise a complex feed forward regulatory loop that controls hilA expression ( Fig. 1 ) ( 16 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li></div>
<li>( ii ) RtsA , HilC , and HilD differentially regulate expression of other genes independent of their effects on hilA expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923082" target="_blank">12923082</a>]<br></li>
<li>RtsA increases invasion gene-expression by increasing transcription of hilA , which is independent of HilC and HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15765064" target="_blank">15765064</a>]<br></li>
<li>HilC , HilD , and RtsA differentially induce expression of dsbA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14679226" target="_blank">14679226</a>]<br></li>
<li>Regulation of gene-expression by RtsA , HilC and HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16045614" target="_blank">16045614</a>]<br></li>
<li>Effect of invF , hilA , and hilC-D mutations on RtsA induction of slrP ( A ) and ability of HilC , HilD , and RtsA to induce expression of hilA ( B ) and slrP ( C ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923082" target="_blank">12923082</a>]<br></li>
<li>RtsA , HilC and HilD also induced expression of hilC , hilD and rtsA ( Fig. 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16045614" target="_blank">16045614</a>]<br></li>
				</div></ul><br>
			<label class="collapse_SIT" for="_5"><font size="5px">HilC binding site</font></label>
				<input id="_5" type="checkbox">
				<div class="SIT"><br><ul><li>We selected all HilD or RtsA ChIP-seq peaks that were located within 100 bp of a HilC ChIP-seq peak .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27601571" target="_blank">27601571</a>]<br></li>
<li>The sequences important for HilC binding are TGT starting at 88 , 84A , 82T , and 80T ; AAT starting at 72 ; and TA starting at 67 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12107132" target="_blank">12107132</a>]<br></li>
<li>Consistent with our previous genetic data , HilE is speciÔ¨Åc for HilD and does not bind or affect HilC or RtsA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29378886" target="_blank">29378886</a>]<br></li>
<li>HilD and HilC do not activate invF expression from the HilA-dependent invF promoter Previously , we had shown that HilA activates invF expression by direct interaction with sequences upstream of invF called the HilA box ( Lostroh et al. , 2000 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>DNA fragment A-a , which carries the upstream regulatory sequences from 373 to 42 , formed two major shifted complexes with HilC but only a single complex with HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12107132" target="_blank">12107132</a>]<br></li>
<li>The combined , normalized sequence read count from the ChIP-seq data for HilC is shown in the Ô¨Årst row of each block , with color intensity representing read density .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27601571" target="_blank">27601571</a>]<br></li>
<li>For 400 bp surrounding each of the HilC peaks , we determined the sequence read coverage on each strand for each of the two replicate ChIP-seq datasets for HilC and for HilD/RtsA ( four ChIP-seq peaks were common to all 3 TFs ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27601571" target="_blank">27601571</a>]<br></li>
<li>( C ) Sequence motifs for HilC and RtsA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27601571" target="_blank">27601571</a>]<br></li>
<li>Consistent with this , we were able to infer DNA sequence motifs for HilC ( two similar motifs that are almost completely mutually exclusive in their site composition ) and RtsA from the ChIP-seq data ( Fig. 3C ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27601571" target="_blank">27601571</a>]<br></li>
<li>As for HilC and RtsA , we were able to infer a DNA motif for RtsB binding by searching for enriched sequences within the RtsBbound regions ( Fig. 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27601571" target="_blank">27601571</a>]<br></li>
<li>Much like HilC , RtsA appears to bind to at least two sites within this region as evidenced by multiple bands in the gel shift ( 64 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923082" target="_blank">12923082</a>]<br></li>
<li>HilC binds to at least three sites in the hilC promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12107132" target="_blank">12107132</a>]<br></li>
<li>Here we provide genetic and biochemical data showing that HilE speciÔ¨Åcally binds to HilD but not to HilC or RtsA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29378886" target="_blank">29378886</a>]<br></li>
<li>The consensus sequences overlap but differ so that HilC can bind to both types of sites but HilD only to one .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12107132" target="_blank">12107132</a>]<br></li>
<li>Although the HilC regulatory targets were largely a subset of the HilD targets , HilC binds many more sites across the genome ( 209 sites for HilC versus 18 sites for HilD , not including the HilD sites from our previous study [ 11 ] ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27601571" target="_blank">27601571</a>]<br></li>
<li>Two direct repeats of CNATTNNT were found in the 35-bp consensus tcCtaTTNtTgTAatttTcNg NCcAtTaatNAtA , where capital letters indicate predominant bases ( 4/5 or 5/5 ) , lowercase letters depict 3/5-identical bases i the HilD/HilC binding sites , boldface indicates speciÔ¨Åed bases of the CNATTNNT repeat element , and N is any base .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17675384" target="_blank">17675384</a>]<br></li>
<li>The hilC start site used in the absence of HilC and HilD is associated with the sequence TGTAATTAT , which matches at six of eight positions the extended 10 promoter element that allows transcription initiation in the absence of a 35 element ( 23 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12107132" target="_blank">12107132</a>]<br></li>
<li>These different binding speciÔ¨Åcities of HilC and HilD seen in the gel shift assays were reÔ¨Çected in the sequence dependence for binding to the C2 region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12107132" target="_blank">12107132</a>]<br></li>
<li>Search for repeated sequence motifs in the HilC - and HilD-binding regions showed only the palindromic repeats of the sequence AT-T -- TATT-N9-AATA -- A-AT .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12107132" target="_blank">12107132</a>]<br></li>
<li>Analysis of the DNA sequence upstream of T2 revealed no close matches to consensus sites of the virulence-associated transcription factors : PhoP-PhoQ ( 27 ) , PmrA-PmrB ( 1 ) , HilA ( 14 ) , HilC , or HilD ( 18 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12775687" target="_blank">12775687</a>]<br></li>
<li>( A ) Sequence of the pJG1 HilD/HilC DNA binding reporter plasmid .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29378886" target="_blank">29378886</a>]<br></li>
<li>Even at the C2 site , binding of HilD required a longer stretch of speciÔ¨Åc base sequences than did binding of HilC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12107132" target="_blank">12107132</a>]<br></li>
<li>Studies have shown that HilC , HilD and RtsA have overlapping binding sites on the hilA promoter , but that each has a slightly different DNA recognition sequence [ 24,30 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17208038" target="_blank">17208038</a>]<br></li>
<li>The discrimination between HilC and HilD can not yet distinguish the contributions of speciÔ¨Åc base sequence recognition , A T-rich character , or DNA bending .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12107132" target="_blank">12107132</a>]<br></li>
<li>Although we could not derive a DNA sequence motif for HilD , it is likely that it binds with similar levels of sequence spec-iÔ¨Åcity to HilC and RtsA , given the overlap in target sites ( Fig. 3B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27601571" target="_blank">27601571</a>]<br></li>
<li>Unlike HilC and RtsA , these three regulators are not homologues of HilD ; hence , they are expected to bind DNA with different sequence speciÔ¨Åcities .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24375101" target="_blank">24375101</a>]<br></li>
<li>Thus , although HilC and HilD proteins can bind to the same DNA target , they have different sequence requirements for DNA recognition .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12107132" target="_blank">12107132</a>]<br></li>
<li>Although HilC and HilD bind to the same DNA regions , they have different requirements with respect to the sequence and length of their DNA target ( 18 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15126488" target="_blank">15126488</a>]<br></li>
<li>Although both HilC and - D proteins can bind to the same DNA regions , they showed different dependencies on the sequence and lengths of their DNA targets .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12107132" target="_blank">12107132</a>]<br></li>
<li>This activation is most probably through direct binding of HilD and HilC to sequences upstream and downstream of this alternative HilA-independent promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>These results suggest that HilD and HilC might directly activate invF expression by binding to sequences upstream of invF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>HilD and HilC appear to activate transcription of invF from a HilD/C-dependent start site that is 631 nucleotides upstream of the invF ORF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>HilD and HilC bind to the invF promoter fragment in vitro To examine whether HilD and HilC might directly activate the invF promoter by binding to sequences upstream of invF , we performed gel shift analyses .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>These gel shift assays indicate that HilD and HilC each bind sequences upstream and downstream of the HilD - and HilC-dependent invF promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>To determine whether HilD , HilC , and RtsA bind the same DNA sites upstream of shared target genes , we compared the proÔ¨Åle of ChIP-seq reads across regions with closely positioned HilD/HilC/RtsA ChIP-seq peaks .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27601571" target="_blank">27601571</a>]<br></li>
<li>The results of this analysis clearly indicate that HilD , HilC , and RtsA bind the same DNA sequences ( Fig. 3B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27601571" target="_blank">27601571</a>]<br></li>
<li>We conclude that HilD , HilC , and RtsA bind very similar DNA sites , although the failure of RtsA to regulate ytfK ( Fig. 3C ) indicates that there are subtle differences in the sequence speciÔ¨Åcities of HilD , HilC , and RtsA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24375101" target="_blank">24375101</a>]<br></li>
<li>Comparison of the four known binding site sequences revealed that the C2 region and the A1 region , which bind HilC and HilD , are closely related .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12107132" target="_blank">12107132</a>]<br></li>
<li>Comparison of aligned HilC - and HilD-binding sequences in the hilA , hilC , and hilD promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12107132" target="_blank">12107132</a>]<br></li>
<li>Our data are consistent with an earlier study showing that HilC and HilD bind overlapping sites upstream of hilA , hilD , and hilC but that HilC binding is less sensitive to mutations within the DNA site ( 21 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27601571" target="_blank">27601571</a>]<br></li>
<li>In addition to binding upstream of hilA and invF , HilD and HilC also bind to sites upstream of hilC and within the prgH-hilD intergenic region ( Olekhnovich and Kadner , 2002 ) ( S. Akbar , unpublished results ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>HilC and HilD bind to overlapping sites in the hilA promoter ( Olekhnovich and Kadner , 2002 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16045614" target="_blank">16045614</a>]<br></li>
<li>Furthermore , HilD and HilC bind the same DNA sites upstream of hilD and hilC ( 38 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24375101" target="_blank">24375101</a>]<br></li>
<li>HilC and HilD bind directly to the upstream sequence of hilA and derepress the transcription of PhilA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15661008" target="_blank">15661008</a>]<br></li>
<li>Furthermore , HilC and HilD are strikingly similar in primary sequence and bind to the same speciÔ¨Åc DNA sequences upstream of PhilA [ 52 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11755416" target="_blank">11755416</a>]<br></li>
<li>Each pair consisted of a consensus promoter into which a HilD/HilC binding sequence had been integrated at various positions relative to the 10 and 35 promoter sequences .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29378886" target="_blank">29378886</a>]<br></li>
<li>All three proteins formed single retarded complexes with two rtsA promoter fragments extending from 173 to 1 and from 81 to 29 , indicating the presence of a single and possibly overlapped binding site for HilC , HilD , and RtsA proteins located between 81 and 1 relative to the transcription start site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17675384" target="_blank">17675384</a>]<br></li>
<li>RtsA , HilC , and HilD each activate hilA by binding to the same sites in the promoter region ( 13 , 35 , 36 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20889744" target="_blank">20889744</a>]<br></li>
<li>HilD , HilC , and RtsA bind to a common DNA-binding site at the rtsA promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17675384" target="_blank">17675384</a>]<br></li>
<li>To identify the sequences that contribute to HilD , HilC , and RtsA binding at the rtsA promoter , the DNA fragment extended from 173 to 29 was analyzed by DNase I footprinting ( Fig. 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17675384" target="_blank">17675384</a>]<br></li>
<li>To see if these sequences are important for HilD and HilC to activate invF , we constructed derivatives of pSA7 in which the upstream TTTGAT was changed to TTTGAG and the downstream TATTCT was changed to TATGCG .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>The HilC and HilD proteins bind to their own gene promoters as well as to at least two regions in the hilA promoter , called sites A1 ( positions 242 to 182 , relative to the start of transcription ) and A2 ( positions 85 to 61 ) ( 18 , 24 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15126488" target="_blank">15126488</a>]<br></li>
<li>HilC and HilD are known to independently bind to two regions of the hilA promoter , from approximately 231 to 179 and from 101 to 49 , although they differ in their sequence requirements for recognition of the binding sites ( 64 , 72 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923082" target="_blank">12923082</a>]<br></li>
<li>Two SPI1 gene products , HilC and HilD , have been shown to bind to the upstream repressing sequence of hilA to induce its transcription ( 12 , 52 , 53 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18245288" target="_blank">18245288</a>]<br></li>
<li>HilC and HilD , two SPI1-encoded AraC/XylS family members , have been called derepressors and are thought to counteract the repression of hilA by binding to sites within the upstream repressing sequence ( URS ) ( 24 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12933899" target="_blank">12933899</a>]<br></li>
<li>HilC and HilD have been shown to bind to the upstream repressing sequence ( URS ) of hilA and counteract the repression ( Schechter et al. , 1999 ; Schechter and Lee , 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15661008" target="_blank">15661008</a>]<br></li>
<li>This region comprises both the 're pressing sequences ' that regulate PhilA in cis as well as the binding sites for HilD ( and HilC ) ( L. Schechter and C. Lee , unpublished observations ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11755416" target="_blank">11755416</a>]<br></li>
<li>In summary , sites A1 and C2 can bind both HilC and HilD in the absence of Ô¨Çanking sequences , whereas sites A2 and D can bind HilC but not HilD , without additional Ô¨Çanking sequences .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12107132" target="_blank">12107132</a>]<br></li>
<li>Likewise , the D region resembles the A2 region , and both bind HilC but not HilD without Ô¨Çanking sequences .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12107132" target="_blank">12107132</a>]<br></li>
<li>The hilA and hilC promoters contain multiple binding sites of each type , whereas the hilD promoter contains a site that binds HilC but not HilD without additional binding elements .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12107132" target="_blank">12107132</a>]<br></li>
<li>The previously identiÔ¨Åed binding sites for HilC and HilD are indicated as regions A1 and A2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16443238" target="_blank">16443238</a>]<br></li>
<li>These results show that HilC and HilD use overlapping binding sites and depend on the same nucleotide positions for DNA recognition .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12107132" target="_blank">12107132</a>]<br></li>
<li>Analysis of overlapping HilD , HilC , and RtsA binding sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27601571" target="_blank">27601571</a>]<br></li>
<li>HilC - and HilD-binding sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12107132" target="_blank">12107132</a>]<br></li>
				</ul></div><br>
			<label class="collapse_TU" for="_6"><font size="5px">Transcription units regulated by HilC</font></label>
				<input id="_6" type="checkbox">
				<div class="TU"><br><ul><li>Blackwell Science , LtdOxford , UKMMIMolecular Microbiology 0950-382X Blackwell Publishing Ltd , 200347Original ArticleHilD and HilC directly activate invF expressionS.Akbar , L.M.Schechter , C.P.Lostroh and C.A.Lee Molecular Microbiology ( 2003 ) 47 ( 3 ) , 715 -- 728 AraC/XylS family members , HilD and HilC , directly activate virulence gene expression independently of HilA in Salmonella typhimurium Samina Akbar , ‚Ä† Lisa M. Schechter , ‚Ä° C. Phoebe Lostroh ¬ß and Catherine A. Lee * Department of Microbiology and Molecular Genetics , Harvard Medical School , 200 Longwood Avenue , Boston , MA 02115 , USA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
				<label class="TU_collapse_similarity" for="tu_similarity_2">
AraC/XylS family members , HilD and HilC , directly activate virulence gene expression independently of HilA in Salmonella typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23554419" target="_blank">23554419</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18723621" target="_blank">18723621</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17600486" target="_blank">17600486</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14645287" target="_blank">14645287</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21168230" target="_blank">21168230</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16443238" target="_blank">16443238</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15790293" target="_blank">15790293</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15968047" target="_blank">15968047</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19074398" target="_blank">19074398</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923082" target="_blank">12923082</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14679226" target="_blank">14679226</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15126488" target="_blank">15126488</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17675384" target="_blank">17675384</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15731035" target="_blank">15731035</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25135218" target="_blank">25135218</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15039320" target="_blank">15039320</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18441068" target="_blank">18441068</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15516559" target="_blank">15516559</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18160484" target="_blank">18160484</a>]<br></label>
<input id="tu_similarity_2", type="checkbox">
<div class="TU_similar">
<li>AraC / XylS family members , HilD and HilC , directly activate virulence gene expression independently of HilA in Salmonella typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15256548" target="_blank">15256548</a>]<br></li>
<li>AraC/XylS family members , HilD and HilC , directly activate virulence gene expression independently of HilA in Salmonella typhi-murium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28329249" target="_blank">28329249</a>]<br></li>
<li>AraC/XylS family members , HilD and HilC , directly activate virulence gene expression independently of HilA in Salmonella Typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24375101" target="_blank">24375101</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27601571" target="_blank">27601571</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27185788" target="_blank">27185788</a>]<br></li></div>
				<label class="TU_collapse_similarity" for="tu_similarity_6">
References Akbar , S. , Schechter , L. , Lostroh , C. , and Lee , C. ( 2003 ) AraC/XylS family members , HilD and HilC , directly activate virulence gene expression independently of HilA in Salmo-nella typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15661008" target="_blank">15661008</a>]<br></label>
<input id="tu_similarity_6", type="checkbox">
<div class="TU_similar">
<li>Akabar , S. , Schechter , L.M. , Lostroh , C.P. and Lee , C.A. ( 2003 ) AraC/XylS family members , HilD and HilC , directly activate virulence gene expression independently of HilA in Salmonella typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17329372" target="_blank">17329372</a>]<br></li>
<li>References Akbar , S. , Schechter , L.M. , Lostroh , C.P. , and Lee , C.A. ( 2003 ) AraC/XylS family members , HilD and HilC , directly activate virulence gene expression independently of HilA in Salmonella typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16999831" target="_blank">16999831</a>]<br></li>
<li>Akbar , S. , Schechter , L.M. , Lostroh , C.P. and Lee , C.A. ( 2003 ) AraC/XylS family members , HilD and HilC , directly activate virulence gene expression independently of HilA in Salmonella typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28335027" target="_blank">28335027</a>]<br></li>
<li>Akbar , S. , Schechter , L.M. , Lostroh , C.P. , and Lee , C.A. ( 2003 ) AraC/XylS family members , HilD and HilC , directly activate virulence gene expression independently of HilA in Salmonella typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16045614" target="_blank">16045614</a>]<br></li>
<li>Mol Microbiol 90:1216 -- 1232 Akbar S , Schechter LM , Lostroh CP , Lee CA ( 2003 ) AraC/XylS family members , HilD and HilC , directly activate virulence gene expression independently of HilA in Salmonella typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27549210" target="_blank">27549210</a>]<br></li>
<li>Akbar S. , Schechter L.M. , Lostroh C.P. , Lee C.A. ( 2003 ) AraC/XylS family members , HilD and HilC , directly activate virulence gene expression independently of HilA in Salmonella typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22004521" target="_blank">22004521</a>]<br></li></div>
<li>It is possible that HilD , but not HilC , activates invF expression from a second promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>We found that HilD and HilC induce invF expression from pSA7 in E. coli , comparable to that seen in S. typhimurium ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>We found that neither HilD nor HilC activate invF-lacZ expression from pSA10 ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>Therefore , we wanted to determine if HilC or HilD could also induce expression of slrP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923082" target="_blank">12923082</a>]<br></li>
<li>Arabinose-induced expression of HilC also dramatically increases invF expression ( Fig. 3 , striped bars ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>In contrast , overproduction of HilC activates transcription of these fusions as expected ; HilC is known to independently activate hilD transcription ( 13 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20889744" target="_blank">20889744</a>]<br></li>
<li>Again , when hilD was deleted in these strains , Fur regulation of hilD was abrogated , while overproduction of HilC still induced hilD expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
<li>Figure 5A shows that hilD transcription was induced by both Fur and HilC in the hilD background .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
				<label class="TU_collapse_similarity" for="tu_similarity_21">
For clarity , the genes encoding HilC , RtsA , and HilA are not shown .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22021388" target="_blank">22021388</a>]<br></label>
<input id="tu_similarity_21", type="checkbox">
<div class="TU_similar">
<li>For clarity , the genes encoding HilD , HilC , RtsA and HilA are not shown .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16045614" target="_blank">16045614</a>]<br></li>
<li>For clarity , the genes encoding HilD , HilC , RtsA , and HilA are not shown .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27704705" target="_blank">27704705</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20889744" target="_blank">20889744</a>]<br></li></div>
<li>The genes that encode the HilD and HilC proteins are located on SPI1 , while the gene that encodes RtsA is located on an island integrated at the tRNAPheU ( 12 , 14 , 40 , 43 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17675384" target="_blank">17675384</a>]<br></li>
<li>RtsA is a close homologue of HilD and HilC but is encoded outside SPI-1 , in an operon that includes another TF-encoding gene , rtsB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27601571" target="_blank">27601571</a>]<br></li>
<li>The genes encoding HilC and HilD are located on SPI1 , whereas RtsA is in encoded on a 15 kb Salmonella-speciÔ¨Åc insert located near the gene for tRNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17208038" target="_blank">17208038</a>]<br></li>
<li>HilC and HilD are encoded within the SPI1 locus , while RtsA is encoded in an operon with RtsB located at 93.9 centisomes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
<li>HilC and HilD are encoded within SPI1 , whereas RtsA is encoded in an operon at 93.9 centisomes that also encodes RtsB , which negatively regulates Ô¨ÇhDC and hence the Ô¨Çagellar regulon ( Ellermeier and Slauch , 2003 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16045614" target="_blank">16045614</a>]<br></li>
<li>Transcription of one gene , ytfK , was activated by HilC but not RtsA ( Fig. 3B and C ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24375101" target="_blank">24375101</a>]<br></li>
				<label class="TU_collapse_similarity" for="tu_similarity_30">
As shown in Table 2 , hilA expression was induced 40-fold by RtsA , 20-fold by HilD , and 120-fold by HilC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14679226" target="_blank">14679226</a>]<br></label>
<input id="tu_similarity_30", type="checkbox">
<div class="TU_similar">
<li>Figure 7B shows that expression of hilA was induced 40-fold by RtsA , 20-fold by HilD , and 120-fold by HilC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923082" target="_blank">12923082</a>]<br></li></div>
<li>The hilA gene is silenced by H-NS under low-osmolarity conditions , until H-NS-mediated repression is relieved by HilC and HilD , two other transcription factors encoded within SPI-1 ( Schechter et al. , 2003 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21570144" target="_blank">21570144</a>]<br></li>
<li>ssaG and ssaJ are regulated by HilC , and sifB is regulated by SprB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27601571" target="_blank">27601571</a>]<br></li>
<li>In turn , hilA is regulated by HilD and HilC ( Schechter and Lee , 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21168230" target="_blank">21168230</a>]<br></li>
<li>It is certainly possible that other genes are directly regulated by HilC , HilD and RtsA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17208038" target="_blank">17208038</a>]<br></li>
<li>We propose that siiA is regulated by HilD , HilC , RtsA , HilA , and SprB , which would indicate that SPI-1 and SPI-4 gene expression is tightly linked .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24375101" target="_blank">24375101</a>]<br></li>
<li>The rtsAB operon is transcriptionally activated by HilD , HilC , and RtsA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24488311" target="_blank">24488311</a>]<br></li>
<li>hilA expression is induced by three transcriptional activators , HilC , HilD and RtsA [ 70 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25375226" target="_blank">25375226</a>]<br></li>
<li>However , this condition inhibits hilA expression through a repressor ( R2 ) that can not be counteracted by HilC or HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10844688" target="_blank">10844688</a>]<br></li>
<li>Alternatively , this condition may permit HilC or HilD expression and activity while repressing hilA by a mechanism that can not be counteracted by HilC or HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10844688" target="_blank">10844688</a>]<br></li>
<li>Still , it is intriguing to consider that under certain conditions HilC and HilD may be able to activate hilA expression and that under other conditions they may be able to increase hilA expression by derepressing hilA transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12933899" target="_blank">12933899</a>]<br></li>
<li>Although it has been proposed that SirA directly controls expression of both hilA and hilC ( Teplitski et al. , 2003 ) , it was also determined that HilC was not required for SirA to control hilA expression ( Lucas and Lee , 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16045614" target="_blank">16045614</a>]<br></li>
<li>HilC , HilD and RtsA , when overproduced are clearly able to induce hilD transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17208038" target="_blank">17208038</a>]<br></li>
<li>In this model , HilD or HilC expression and activity would be induced in vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>HilC and RtsA can independently induce HilA expression [ 16 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20686667" target="_blank">20686667</a>]<br></li>
<li>It would also explain why other SPI1 regulators -- namely HilC , HilD , and RtsA -- affect SPI4 gene expression , as they in turn regulate HilA expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20190046" target="_blank">20190046</a>]<br></li>
<li>HilC and RtsA amplify SPI1 gene expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20686667" target="_blank">20686667</a>]<br></li>
<li>Once induced , HilC and RtsA serve to further amplify and also accelerate SPI1 gene expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20686667" target="_blank">20686667</a>]<br></li>
<li>Both HilC and HilD are positive regulators of hilA expression ( Schechter et al. , 1999 ; Lostroh et al. , 2000 ; Lucas and Lee , 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12791144" target="_blank">12791144</a>]<br></li>
<li>Transcription of invF is primarily regulated by HilA , but HilD , HilC and RtsA are each capable of partially activating invF ( Eichelberg et al. , 1999 ; Rakeman et al. , 1999 ; Akbar et al. , 2003 ; Ellermeier and Slauch , 2003 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16045614" target="_blank">16045614</a>]<br></li>
<li>Moreover , hilA also appears to be regulated by other proteins encoded within SPI1 including HilC ( also known as SirC and SprA ) ( Eichelberg et al. , 1999 ; Rakeman et al. , 1999 ; Schechter et al. , 1999 ) and HilD ( Schechter et al. , 1999 ) , further complicating the understanding of how SPI1 gene expression is co-ordinated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10692170" target="_blank">10692170</a>]<br></li>
<li>Under activating conditions , hilA expression is thought to be derepressed by two AraC-like transcriptional regulators encoded on SPI1 : HilD and HilC ( Schechter et al. , 1999 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10844688" target="_blank">10844688</a>]<br></li>
<li>One specific question then is whether this behavior is intrinsic to these proteins , as assumed in the model , or simply due to these proteins not being expressed at sufficiently high levels ( as HilC and RtsA can independently activate SPI1 gene expression when over expressed ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20686667" target="_blank">20686667</a>]<br></li>
<li>To see if these sequences are important for HilD and HilC to activate invF , we constructed derivatives of pSA7 in which the upstream TTTGAT was changed to TTTGAG and the downstream TATTCT was changed to TATGCG .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>The HilC and HilD proteins bind to their own gene promoters as well as to at least two regions in the hilA promoter , called sites A1 ( positions 242 to 182 , relative to the start of transcription ) and A2 ( positions 85 to 61 ) ( 18 , 24 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15126488" target="_blank">15126488</a>]<br></li>
<li>Two SPI1 gene products , HilC and HilD , have been shown to bind to the upstream repressing sequence of hilA to induce its transcription ( 12 , 52 , 53 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18245288" target="_blank">18245288</a>]<br></li>
<li>As detailed below , genetic studies indicate that the downstream -10 sequence , corresponding to the short primer extension product , is essential for HilD and HilC to activ-ate invF expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>HilC has high homology to HilD and also binds to the hilA promoter ; however , it is not required for hilA expression ( 58 , 59 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15039320" target="_blank">15039320</a>]<br></li>
<li>In this model , the repressor is always present but can not inhibit hilA expression under activating conditions because HilD ( and , to a lesser extent , HilC ) displaces it from the URS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11292791" target="_blank">11292791</a>]<br></li>
<li>The strains used were plasmid-containing derivatives of JS273 through JS278 RtsA induction of hilA does not require HilC or HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923082" target="_blank">12923082</a>]<br></li>
<li>As shown in Figure 9 , expression of HilC was not affected by any of the three E. coli supernatants .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21379575" target="_blank">21379575</a>]<br></li>
<li>HilC may be capable of activating invF and invA expression directly , since high-level expression of hilC from a plasmid allows expression of these genes in the absence of hilA ( 30 , 69 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10714991" target="_blank">10714991</a>]<br></li>
<li>Two reports have shown that invF expression , but not prgH expression , can be upregulated by artiÔ¨Åcially expressing high levels of HilC ( SirC , SprA ) even in the absence of hilA ( Ô¨Ågure 1B , blue arrow ) [ 50 , 51 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11755416" target="_blank">11755416</a>]<br></li>
<li>The loss of HilC decreased the overall expression of rtsA but did not affect the ability of SirA to induce expression of the rtsA -- lac , hilC -- lac , or hilD -- lac fusions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16045614" target="_blank">16045614</a>]<br></li>
<li>Thus SirA does not require HilC to regulate expression of the three hilA regulators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16045614" target="_blank">16045614</a>]<br></li>
<li>Consistent with the SPI-1 gene levels , SPI-1 gene transcriptional regulators HilC , HilD , HilA , rtsA , and invF were also signiÔ¨Åcantly repressed ( Fig. 6b ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27549210" target="_blank">27549210</a>]<br></li>
<li>For example , the transcription of SPI-1 genes can be activated by HilA , HilC , HilD , InvF , and SirA ( 30 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28439039" target="_blank">28439039</a>]<br></li>
<li>InvF and HilA are transcriptional regulators of SPI-1 genes that are encoded within SPI-1 ; hilA itself is regulated by several factors , including PhoP-PhoQ , SirA , HilC , and HilD ( 3 -- 5 , 20 , 37 , 44 , 66 , 72 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10633113" target="_blank">10633113</a>]<br></li>
<li>HilD , HilC , and RtsA all positively regulate invF , which is located within SPI-1 and encodes another of the SPI-1-associated TFs ( Fig. 1 and 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27601571" target="_blank">27601571</a>]<br></li>
				<label class="TU_collapse_similarity" for="tu_similarity_70">
SPI-1 genes are organized in seven or more transcriptional units , whose expression is under the control of four SPI-encoded transcription factors : HilA , HilC , HilD , and InvF ( Lostroh and Lee 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20008574" target="_blank">20008574</a>]<br></label>
<input id="tu_similarity_70", type="checkbox">
<div class="TU_similar">
<li>SPI-1 genes are organized in seven or more transcriptional units under the control of the SPI-1-encoded transcription factors HilA , HilC , HilD and InvF ( Lostroh and Lee , 2001 ) and of RtsA , a transcription factor encoded outside SPI-1 ( Ellermeier and Slauch , 2003 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24354910" target="_blank">24354910</a>]<br></li></div>
<li>sinR SL0300 STM0304 HilD SL4247 STM4310 HilD/HilC/RtsA SL4248 STM4312 HilD SL4249 STM4313 HilD nhaA SL0040 STM0039 HilC nhaR SL0041 STM0040 HilC SL1177 STM1239 InvF SL0336 STM0341 SprB SL1770 STM1841 SprB SL3112 STM3138 SprB pckA SL3467 STM3500 SprB yjbJ SL4176 STM4240 SprB NA STnc520 STnc520 STnc520 SprB SL0973 STM1034 HilA SL0975 STM1036 HilA SL0976 STM1037 HilA a The genes listed are direct regulatory targets of SPI-1-associated TFs that are encoded outside SPI-1 whose expression proÔ¨Åles ( from reference 4 ) have an average correlation coefÔ¨Åcient of 0.5 with the proÔ¨Åles for invasion genes within SPI-1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27601571" target="_blank">27601571</a>]<br></li>
<li>In addition , HilD and HilC are important AraC/XylS transcriptional regulators that mediate expression of HilA , which then activates all genes necessary for a functional SPI-1 encoded T3SS ( 29 , 30 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22004521" target="_blank">22004521</a>]<br></li>
<li>Within SPI-1 there are key transcriptional activators which regulate expression of SPI-1 genes : HilC , HilD , HilA , and InvF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22291596" target="_blank">22291596</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22291596" target="_blank">22291596</a>]<br></li>
<li>However , we found that expression of invF on pVV448 is only induced about sevenfold by overexpression of HilD ( Fig. 4 , black bars ) and is not affected at all by overexpression of HilC ( Fig. 4 , striped bars ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>It has also recently been reported that loss of HilD decreases hilC transcription but that loss of HilC does not signiÔ¨Åcantly alter hilD expression , so it is proposed that HilD is the apex of regulation of SPI1 expression ( 13 , 15 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18245288" target="_blank">18245288</a>]<br></li>
<li>Previous studies have shown that HilD , not HilC , is primarily responsible for derepression of hilA expression when Salmonella serovar Typhimurium is grown under our inducing conditions ( 19 , 23 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12933899" target="_blank">12933899</a>]<br></li>
<li>Arabi-nose-induced expression of hilC overcomes repressing conditions and mutations even when hilD is disrupted ( data not shown ) , suggesting that HilC is directly derepressing hilA expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11292791" target="_blank">11292791</a>]<br></li>
<li>It has also been demonstrated that HilC and HilD each induce the expression of the other as well as inducing their own promoters ( 40 , 47 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18245288" target="_blank">18245288</a>]<br></li>
<li>HilC and HilD each activate the expression of the other as well as inducing their own promoters ( 39 , 47 ) , so we then measured transcription from the hilC and hilD promoters on plasmids in hilC - or hilD-disrupted backgrounds on the chromosome .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18245288" target="_blank">18245288</a>]<br></li>
<li>This indicates that the role of the PhoP protein is to activate pmrD transcription and not simply to counteract repression by the PmrA protein on the pmrD promoter as proposed for other regulatory proteins such as HilC and HilD ( 28 , 29 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12676988" target="_blank">12676988</a>]<br></li>
<li>These results taken together suggest that the HilD , HilC , and RtsA proteins induce the transcription of the rtsA promoter by counteracting H-NS/Hha-mediated repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17675384" target="_blank">17675384</a>]<br></li>
<li>Expression of the SPI-1 T3SS and its associated genes ( several of which are encoded outside the SPI-1 pathogenicity island ) is controlled by a hierarchy of regulators ( HilD , HilA , HilC , RtsA , InvF ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29229736" target="_blank">29229736</a>]<br></li>
<li>In this context , slrP is induced by overexpression of HilC , HilD , and RtsA independently of the central SPI1 regulator HilA , with RtsA being the best inducer .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25182488" target="_blank">25182488</a>]<br></li>
<li>We first note that these additional feedback loops , namely the ones regulating the expression of HilC and RtsA , do not add redundancy to the circuit , as the loss of HilD effectively shuts off SPI1 gene expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20686667" target="_blank">20686667</a>]<br></li>
<li>As expected , disruption of the Ô¨ÇhDC operon , encoding the master regulator for expression of the Ô¨Çagellar regulon , blocked the increase of HilC and HilD levels caused by clpPX disruption .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18245288" target="_blank">18245288</a>]<br></li>
<li>While the mechanism by which these genes alter hilA expression is unknown , it is possible that some of these genes encode hilA repressors that may be the targets of the derepressors HilC and HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11673432" target="_blank">11673432</a>]<br></li>
<li>Thus , it is possible that the feed forward loop constituted by HilD , HilC , and RtsA can regulate the expression of ssrAB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25135218" target="_blank">25135218</a>]<br></li>
<li>It has been proposed that the feed-forward loop system of HilC , HilD , and RtsA functions as a switch that controls hilA expression by modulating the threshold of HilD protein required for the ultimate HilA activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26441883" target="_blank">26441883</a>]<br></li>
<li>Thus , although HilD is capable of auto-regulation , the results observed could not be due to the loss of this regulatory protein controlling its own expres ¬¨ Pat post-transcriptionally regulates HilD The regulation of hilD is complex ; hilD regulates itself but is also regulated by the transcriptional regulators HilC and RtsA , which constitute a feed-forward circuit to control the downstream SPI1 regulator HilA ( Ellermeier et al. , 2005 ; Ellermeier and Slauch , 2007 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27341691" target="_blank">27341691</a>]<br></li>
<li>Expression of the SPI1 T3SS is tightly regulated by the combined action of HilC , HilD , and RtsA , three AraC family members that can independently activate hilA , which encodes the direct regulator of the SPI1 structural genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
<li>Discussion Expression of the SPI1 T3SS is controlled by HilD , HilC , and RtsA , acting in a complex feed-forward loop to activate the hilD , hilC , and rtsA genes , as well as hilA , which encodes the transcriptional activator of the T3SS structural genes ( Figure 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22021388" target="_blank">22021388</a>]<br></li>
<li>In contrast , expression was not affected in the ŒîhilA and ŒîhilC mutants that lack the SPI-1 regulators HilA and HilC ( Figs. 1A and B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21518393" target="_blank">21518393</a>]<br></li>
<li>HilC - and HilD-binding sites in the hilC promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12107132" target="_blank">12107132</a>]<br></li>
<li>In addition to the negative control that was exerted by H-NS/Hha under low-osmolarity conditions , the homologous virulence activators HilD , HilC , and RtsA ( Hil activators ) induced rtsA-lac expression in a high-salt medium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17675384" target="_blank">17675384</a>]<br></li>
<li>As expected , most hilA expression from the fulllength promoter was lost in a strain deleted for SPI1 ( DSPI1 ) , which encodes the activators HilC and HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16443238" target="_blank">16443238</a>]<br></li>
<li>The hilD promoter is active in vitro and unaffected by the HilC and HilD proteins or other SPI1 products .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12107132" target="_blank">12107132</a>]<br></li>
<li>HilD can be repressed by HilE , and HilC and RstA further amplify the inducing signal .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21984608" target="_blank">21984608</a>]<br></li>
<li>SPI1 gene expression is positively regulated by the transcriptional activators HilA , HilC , and HilD , encoded within SPI1 , and negatively regulated by the regulators HilE and PhoP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18160484" target="_blank">18160484</a>]<br></li>
<li>HilA expression is also regulated by the regulators HilC and HilD [ 29 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21379575" target="_blank">21379575</a>]<br></li>
<li>Transcription of hilA is subject to complex control and is stimulated by the SPI-1-encoded HilC and HilD proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15126488" target="_blank">15126488</a>]<br></li>
<li>At the transcriptional level , the expression of hilD is positively autoregulated and modulated by a feed-forward regulatory loop involving HilD itself and the AraC-like regulators HilC and RtsA ( Olekhnovich and Kadner , 2002 ; Ellermeier et al. , 2005 ; Golubeva et al. , 2012 ) , while post-transcriptionally it is positively controlled by a regulatory cascade integrated by the SirA/BarA and Csr global regulatory systems ( Mart√≠nez et al. , 2011 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26300871" target="_blank">26300871</a>]<br></li>
<li>Transcription of the hilA gene is also stimulated by the SPI1-encoded AraC/XylS-like regulatory proteins HilC , HilD and RtsA ( Eichelberg and Galan , 1999 ; Lostroh et al. , 2000 ; Rhen and Dorman , 2005 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28631437" target="_blank">28631437</a>]<br></li>
<li>This result indicated that hilC has two promoters and that the upstream promoter is repressed by HilC and HilD while the downstream one is activated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12107132" target="_blank">12107132</a>]<br></li>
<li>Whether HilC and HilD act directly at the hilA promoter is not yet known .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11123690" target="_blank">11123690</a>]<br></li>
<li>The genes slrP and dsbA are also induced by HilC , HilD and RtsA independently of both HilA and InvF [ 25,28 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17208038" target="_blank">17208038</a>]<br></li>
<li>That means that at least hilA and rtsA are induced by HilD , HilC , and RtsA proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17675384" target="_blank">17675384</a>]<br></li>
<li>Under high-osmolarity conditions , HilD , HilC , and RtsA activate SPI1 genes directly and indirectly through hilA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17675384" target="_blank">17675384</a>]<br></li>
<li>HilD and HilC derepress hilA transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14977940" target="_blank">14977940</a>]<br></li>
<li>SPI-1 also encodes transcriptional activators responsible for its own expression : HilA , HilC , HilD , and InvF ( Lostroh and Lee 2001 ; Ellermeier and Slauch 2007 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25991823" target="_blank">25991823</a>]<br></li>
<li>When present , HilC or HilD binds to this same site and promotes hilA expression ( 63 ; Schechter and Lee , unpublished results ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11292791" target="_blank">11292791</a>]<br></li>
<li>In contrast , HilD and HilC have been shown to independently bind to the URS and promote hilA expression ( Schechter and Lee , 2001 ; Olekhnovich and Kadner , 2002 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>Transcription of invF is known to be regulated by HilA , HilC , and HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923082" target="_blank">12923082</a>]<br></li>
<li>Immunoblotting for detection of HilC and HilD proteins in wildtype and Lon-depleted cells grown under SPI1-repressing conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15661008" target="_blank">15661008</a>]<br></li>
<li>However , under low osmolarity conditions , HilD and HilC activators were not essential for hilA expression in hns and hha hns double mutants ( lines 7 and 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16443238" target="_blank">16443238</a>]<br></li>
<li>HilA is , in turn , induced by HilD and HilC and regulates InvF , which induces expression of secreted effector proteins located in SPI-1 ( 26 , 27 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22004521" target="_blank">22004521</a>]<br></li>
<li>It is known that HilD can also control gene expression through HilC , SprB , RtsA and SsrA/B4 ,14 , and possibly , as suggested by our results , through SinR and SlyA ; thus , HilD could act on SL1896 through any of these regulators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27886269" target="_blank">27886269</a>]<br></li>
<li>In fact , there is evidence that SipA and SipC HilD and HilC directly activate invF expression 723 can be secreted independently of SPI1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>In this scheme , HilD and HilC indirectly activate expression of the SPI1 TTS apparatus and its secreted substrates by affecting hilA expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>It has been proposed that HilD and HilC indirectly co-ordinate expression of the SPI1 TTS apparatus and its secreted effector proteins in vivo by directly affecting hilA transcription ( Schechter et al. , 1999 ; Schechter and Lee , 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>As HilC and HilD activate the transcription from both HilA-dependent and - independent promoters , it is possible that Lon is involved in regulation of SPI1 gene expression by a pathway that depends on HilC and/or HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15661008" target="_blank">15661008</a>]<br></li>
<li>First , HilC and RtsA amplify HilA and HilD expression , in the sense that HilA and HilD expression is reduced the absence of HilC and RtsA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20686667" target="_blank">20686667</a>]<br></li>
<li>Under inducing conditions , activation of BarA/SirA initiates a regulatory cascade involving HilD , HilC , and HilA , which leads both directly and indirectly to the expression of SPI1 and SPI4 genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18160484" target="_blank">18160484</a>]<br></li>
<li>We propose that HilD and HilC can independently activate invF expression and InvF production , which directly and indirectly increases transcription of the sipA and sipC genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>HilA , in turn , is subject to multiple controls , with two additional SPI1 regulators , HilC and HilD , inducing hilA expression ( 18 , 32 , 58 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18424519" target="_blank">18424519</a>]<br></li>
<li>The transcriptional activators HilD and HilC play an important role in controlling hilA expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25547794" target="_blank">25547794</a>]<br></li>
<li>These results suggest a model in which HilC and HilD , unlike most other AraC-like transcriptional regulators ( 23 ) , are not activators but instead behave as derepressors of hilA expression ( Fig. 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11292791" target="_blank">11292791</a>]<br></li>
<li>Still , no one has reported a condition in which wild-type S. typhimurium represses hilA expression while simultaneously activating expression of invF and invF-dependent genes through HilC or HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10844688" target="_blank">10844688</a>]<br></li>
<li>We have three working hypotheses , which are not mutually exclusive : ( i ) RtsA , HilC , and HilD are active under different conditions or induce expression of hilA in response to different environmental cues .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923082" target="_blank">12923082</a>]<br></li>
<li>It has also been shown that both HilD and HilC can independently induce expression of a hilA-lac fusion in E. coli ( 72 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923082" target="_blank">12923082</a>]<br></li>
<li>These data suggest that HilC and RtsA may activate hilA expression in response to a set of signals that are not present in laboratory conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15765064" target="_blank">15765064</a>]<br></li>
<li>The in vitro data suggest that HilC , HilD and RtsA act in concert to induce expression of hilA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16045614" target="_blank">16045614</a>]<br></li>
<li>These transcription factors include HilA , HilD , and HilC ( all three are responsible for activation of SPI1-encoded Type 3 Secretion System ( T3SS ) ) [ 67,68 ] ; SprB ( responsible for activation of SPI4 gene expression ) [ 47 ] ; and SsrAB ( two-component system responsible for activation of SPI2-encoded T3SS ) [ 72 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23040276" target="_blank">23040276</a>]<br></li>
<li>Transcription of hilA is directly activated by HilC and HilD , both members of the AraC/XylS family of transcriptional regulators ( Schechter and Lee 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25991823" target="_blank">25991823</a>]<br></li>
<li>Two SPI1-encoded regulators of hilA expression are the AraC/XylS family members , HilC ( also called SirC or SprA ) and HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12107132" target="_blank">12107132</a>]<br></li>
<li>Two SPI1-encoded proteins , HilC ( SirC or SprA ) and HilD , both members of the AraC/XylS family of transcriptional regulators , induce expression of hilA ( 26 , 67 , 71 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923082" target="_blank">12923082</a>]<br></li>
<li>Two homologous proteins , HilC and HilD , both members of the AraC/XylS family of transcriptional regulators , induce hilA expression ( 12 , 51 , 53 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18245288" target="_blank">18245288</a>]<br></li>
<li>In contrast , disruption of the downstream -10 hexamer sequence dramatically reduced the ability of HilD and HilC to activate invF expression ( Fig. 6 , black and striped bars , pSA14 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>Two members of the AraC/XylS family of transcription activators , HilC and HilD , can independently derepress hilA expression by binding directly to the upstream repressing sequence and counteracting its repressing effect in vitro and in vivo ( 24 , 30 , 31 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16391141" target="_blank">16391141</a>]<br></li>
<li>Three homologous proteins , 8,13,14 HilC , HilD , and RtsA , which belong to the AraC/XylS family of transcription factors can activate expression of hilA and some of its target genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16443238" target="_blank">16443238</a>]<br></li>
<li>Our current studies demonstrate that HilD and HilC can bind upstream of invF and directly activate its expression in the absence of hilA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>RtsA belongs to the AraC/XylS family of regulators , and can directly bind the hilA promoter and induce the hilA expression independent of HilC and HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15661008" target="_blank">15661008</a>]<br></li>
<li>Two additional regulators , HilC and HilD , activate directly the expression of hilA and the invF operon , independently of HilA ( Dieye et al. , 2007 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27404739" target="_blank">27404739</a>]<br></li>
<li>Although HilA is the major regulator , two other regulators encoded within SPI1 , HilC and HilD , can directly increase the expression of both hilA and invF independently of hilA ( 3 , 53 , 56 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14645287" target="_blank">14645287</a>]<br></li>
<li>More recently , H-NS has been shown to repress the expression of HilA , HilD , HilC , and RtsA under noninducing conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23554419" target="_blank">23554419</a>]<br></li>
<li>When the individual proteins HilD , HilC , and RtsA are overexpressed from plasmids , they are able to induce expression of the hilA , invF , hilD , hilC , rtsA , slrP , and dsbA genes , which are required for invasion ( 13 -- 15 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17675384" target="_blank">17675384</a>]<br></li>
<li>Thus , if the abilities of these proteins to induce dsbA and hilA are compared , it becomes apparent that RtsA and HilD are better able to induce expression of dsbA than HilC is ( Table 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14679226" target="_blank">14679226</a>]<br></li>
<li>We believe that the inductions of RtsA , HilC , and HilD are self-reinforcing events leading to rapid and fully induced expression of hilA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923082" target="_blank">12923082</a>]<br></li>
<li>These include the transcription factors FliZ , HilC , and RtsA , which have each been demonstrated to positively regulate invasion gene expression by promoting expression of the SPI-1 regulator hilD ( 9 , 10 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27185788" target="_blank">27185788</a>]<br></li>
<li>They act by increasing the level of HilD protein to overcome HilE , and controlling HilD activity , or in the case of Fur , lowering the threshold required at the promoter , such that HilD activates its own promoter as well as induces expression of HilC and RtsA , which then act to amplify and accelerate SPI1 expression ( Saini et al. 2010a ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22021388" target="_blank">22021388</a>]<br></li>
<li>Only when the activating signals are sufficiently strong is HilD expressed at a high enough level to overcome sequestration by HilE and activate the expression of the other SPI1 regulators -- HilC , RtsA , and HilA-and also further induce its own expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20686667" target="_blank">20686667</a>]<br></li>
<li>Due to the counteracting nature of the repressing URS and derepressing HilC and HilD proteins that directly control hilA expression , the hns-1 : : kan mutation may increase hilA expression under the low-osmolarity conditions ( i ) by decreasing its repression and/or ( ii ) by increasing its derepression by HilC or HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12933899" target="_blank">12933899</a>]<br></li>
<li>HilC and HilD bind to the hilA promoter region , and it is believed that this binding induces the expression of hilA ( 64 , 71 , 72 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923082" target="_blank">12923082</a>]<br></li>
<li>HilC and HilD , two SPI1-encoded AraC/XylS family members , have been called derepressors and are thought to counteract the repression of hilA by binding to sites within the upstream repressing sequence ( URS ) ( 24 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12933899" target="_blank">12933899</a>]<br></li>
<li>The AraC/XylS-type regulators ( HilC and HilD ) positively modulate the expression of hilA as transcription activators binding to the upstream repression sites ( 6 , 8,18 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24780330" target="_blank">24780330</a>]<br></li>
<li>Conversely , HilC and HilD act as transcription activators to induce the expression of hilA by binding to the upstream repression sites ( Ellermeier et al. , 2005 ; Lucas and Lee , 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27565525" target="_blank">27565525</a>]<br></li>
<li>Lack of HilC reduces hilA expression 2 - to 3-fold ( Lopez-Garrido and Casadesus 2010 ) , while addition of 0.2 % L-arabinose to the culture medium represses hilA .10 - fold .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25991823" target="_blank">25991823</a>]<br></li>
<li>HilD , HilC , and RtsA are homologous AraC-like regulators that bind to very similar DNA sites ; therefore , when these regulators are overexpressed , they are able to self-activate their own expression and to activate the expression of one another , as well as to induce the expression of most genes belonging to the HilD regulon ; however , HilD is considered to be dominant , as deletion of hilD , but not of hilC or rtsA , drastically reduces the expression of the targets of HilD ( 23 , 24 , 26 -- 30 , 61 -- 64 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25135218" target="_blank">25135218</a>]<br></li>
<li>HilC and HilD , which are members of the AraC/XylS family , have been postulated to act as a derepressor of hilA expression by counteracting the action of negative regulatory elements at the hilA promoter ( 12,13 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17329372" target="_blank">17329372</a>]<br></li>
<li>Our data suggest that HilC , HilD , and RtsA function in similar manners by binding to the same fragment of hilA to induce its expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923082" target="_blank">12923082</a>]<br></li>
<li>Recently , two SPI1-encoded factors , HilC and HilD , which derepress hilA expression , were identiÔ¨Åed and shown to be members of the AraC/XylS family of transcriptional regulators and to independently bind to these upstream repressing sequences to counteract the repression of hilA expression ( 32 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11741864" target="_blank">11741864</a>]<br></li>
<li>In addition to HilC and HilD , RtsA , which belongs to the AraC/XylS family of transcriptional regulators , has been shown to activate expression of SPI1 genes by binding upstream of hilA to induce its expression ( 13 , 14 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18245288" target="_blank">18245288</a>]<br></li>
<li>HilC and HilD bind directly upstream of the master regulator gene hilA to induce its expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18245288" target="_blank">18245288</a>]<br></li>
				<label class="TU_collapse_similarity" for="tu_similarity_164">
Both HilC and HilD activate expression of SPI-1 genes by binding upstream of the master regulatory gene hilA to induce its expression [ 48 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22291596" target="_blank">22291596</a>]<br></label>
<input id="tu_similarity_164", type="checkbox">
<div class="TU_similar">
<li>Similar to HilC and HilD , RtsA activates the expression of SPI1 genes by binding upstream of the master regulatory gene , hilA , to induce its expression ( 9,17 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17329372" target="_blank">17329372</a>]<br></li>
<li>Like HilC and HilD , RtsA activates expression of SPI1 genes by binding upstream of the master regulatory gene hilA to induce its expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16045614" target="_blank">16045614</a>]<br></li></div>
<li>The expression of the type III secretion system-related genes involved in invasion and intracellular survival is induced in the presence of transcriptional regulator proteins such as HilA , HilC , HilD , InvA , InvC , InvF , SirA , and SirB ( 11 , 27 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24780330" target="_blank">24780330</a>]<br></li>
<li>The expression of the genes , those regulate the type III secretion system , are involved in completion of invasion , intracellular survival and multiplication , is induced by the transcriptional regulator proteins such as HilA , HilC , HilD , InvA , InvC , InvF , SirA , and SirB ( Johnston et al. , 1996 ; Rakeman et al. , 1999 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27565525" target="_blank">27565525</a>]<br></li>
<li>Interestingly , when S. typhimurium is grown under the normally repressing low osmolarity condition , a null mutation in hns results in high hilA expression , even in the absence of HilC and HilD ( L. Schechter and C. Lee , unpublished observations ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11755416" target="_blank">11755416</a>]<br></li>
<li>HilC and HilD are encoded in the SPI1 locus , while RtsA is encoded elsewhere in the chromosome in an operon with RtsB , which negatively regulates expression of Ô¨ÇhDC and therefore the entire Ô¨Çagellar regulon ( 19 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17951383" target="_blank">17951383</a>]<br></li>
<li>We found that HilC and HilD , which are positive regulators of hilA expression , accumulate in Lon-depleted cells , and that the enhancement of SPI1 expression that occurs in a lon-disrupted mutant is not observed in the lon hilC hilD triple null mutant .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15661008" target="_blank">15661008</a>]<br></li>
<li>We conclude that HilD and HilC have a second role in SPI1 gene regulation that is separate from their role in co-ordinating expression of the SPI1 TTSS through hilA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>All indications are that , under the conditions that we are examining , dsbA is primarily controlled by RtsA , HilC , and HilD ( 18 ) ; CpxRA , which controls dsbA in E. coli , seems irrelevant to either expression of dsbA or assembly and function of the SPI1 T3SS or Ô¨Çagella in serovar Typhimurium ( Fig. 2C ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17951383" target="_blank">17951383</a>]<br></li>
<li>Activators HilD and HilC are not essential for rtsA expression in the absence of repressors .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17675384" target="_blank">17675384</a>]<br></li>
<li>We speculate that mcpC , sinR , and are regulatory targets of HilD and possibly also of HilC and RtsA , but require additional transcription factors that are not present or not expressed in E. coli under the conditions used in our expression assays .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24375101" target="_blank">24375101</a>]<br></li>
<li>As shown in Fig. 8 , the increased effect of Lon depletion on hilA expression was abolished by depletion of both HilC and HilD , suggesting that both proteins are also involved in regulation of SPI1 by Lon under SPI1-repressing conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15661008" target="_blank">15661008</a>]<br></li>
<li>hilD As HilD is capable of inducing HilA expression in absence of HilC or RtsA , we reasoned that HilC may be able to do the same in the absence of HilD if it is transcribed in a manner similar to hilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20686667" target="_blank">20686667</a>]<br></li>
<li>As shown in Fig. 5B , loss of RtsA or HilC decreased the level of hilA and invF expression but did not signiÔ¨Åcantly affect induction by SirA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16045614" target="_blank">16045614</a>]<br></li>
<li>The data in Fig. 2 demonstrate that RtsA , HilC and HilD are able to induce expression of hilA in the absence of the other regulators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16045614" target="_blank">16045614</a>]<br></li>
<li>Therefore , we examined the expression of hilA , rtsA , hilC and hilD transcriptional fusions in the presence and absence of RtsA , HilC , and/or HilD ( Fig. 7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16045614" target="_blank">16045614</a>]<br></li>
<li>The effect of SPI1 inducing and repressing conditions on the expression of rtsA , hilC , hilD and hilA in the absence of in RtsA , HilC , and/or HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16045614" target="_blank">16045614</a>]<br></li>
<li>We wanted to determine if HilC , HilD and RtsA could induce expression of hilC , hilD , rtsA and hilA in the absence of the other regulators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16045614" target="_blank">16045614</a>]<br></li>
<li>The ability of SirA to induce expression of rtsA , hilC , hilD , hilA and invF in the absence of RtsA , HilC , and/or HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16045614" target="_blank">16045614</a>]<br></li>
<li>Overexpression of RtsA or RtsA/B only partially compensated for the loss of hilA gene expression in the relA spoT strain ( Fig. 5 ) , presumably because of the reduced expression of hilA gene co-activators , HilC and HilD ( 52 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16905537" target="_blank">16905537</a>]<br></li>
<li>HilC , HilD and RtsA , in addition to regulation of hilA , are all capable of inducing expression of the inv/spa operon -- and by read-through , the sic/sip operon -- in a fashion independent of HilA [ 15,16,25,27 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17208038" target="_blank">17208038</a>]<br></li>
<li>RtsA induces expression of hilA in the absence of both HilC and HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923082" target="_blank">12923082</a>]<br></li>
<li>Under the experimental conditions used , HilC and RtsA were not needed to achieve high hilA expression ; however , HilD was essential since hilA expression was abolished in ŒîhilD mutants regardless of the presence or absence of Gre factors .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28426789" target="_blank">28426789</a>]<br></li>
<li>Although overexpression of the HilC or HilD protein results in high and unregulated expression of hilA ( 30 ) , the relative contribution of changes in the level or activity of either protein in control of hilA expression is not clear .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12107132" target="_blank">12107132</a>]<br></li>
<li>As expected , activation of hilA promoter in the absence of the activators HilD and HilC ( lines 5 -- 8 ) was lost or signiÔ¨Åcantly reduced in inducing ( high osmolarity ) conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16443238" target="_blank">16443238</a>]<br></li>
<li>However , transcription of rtsA , hilC and hilD was induced in response to low oxygen independent of RtsA , HilC , or HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16045614" target="_blank">16045614</a>]<br></li>
<li>To test whether the RtsA , HilC , and HilD activators of hilA are also regulated by these proteins , we investigated the transcription of rtsA-lac , hilC-lac , and hilD-lac chromosomal fusions ( 13 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17675384" target="_blank">17675384</a>]<br></li>
<li>Furthermore , HilD and HilC bind the same DNA sites upstream of hilD and hilC ( 38 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24375101" target="_blank">24375101</a>]<br></li>
<li>These results indicate that Lon is involved in the autoregulation of hilC and hilD transcription by modulating amounts of HilC and HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15661008" target="_blank">15661008</a>]<br></li>
<li>The potential role of the HilC - and HilD-binding sites in expression of the hilC and hilD promoters was examined by in vitro transcription and lac fusions driven by promoter variants .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12107132" target="_blank">12107132</a>]<br></li>
<li>In contrast , loss of RtsA and HilC did not signiÔ¨Åcantly alter hilD expression , even though RtsA and HilC can clearly activate hilD ( see Discussion ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16045614" target="_blank">16045614</a>]<br></li>
<li>HilD , HilC , and RtsA all activate transcription of hilA , hilC , hilD , dsbA , invF , slrP , rtsA , and genes in SPI-4 ( 6 , 15 , 32 , 33 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24375101" target="_blank">24375101</a>]<br></li>
<li>RtsA , HilC and HilD also induced expression of hilC , hilD and rtsA ( Fig. 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16045614" target="_blank">16045614</a>]<br></li>
<li>RtsA , HilC and HilD induced expression of hilC approximately threeto fourfold and hilD ~ 10 - to 12-fold .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16045614" target="_blank">16045614</a>]<br></li>
<li>HilD and HilC do not seem to regulate expression of hilD or prgH ( Olekhnovich and Kadner , 2002 ) ( Figs 1 and 2B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>Although HilD and HilC regulate two promoters upstream of hilC , they activate one and repress the other , such that overall HilD and HilC appear to have no effect on hilC expression ( Olekhnovich and Kadner , 2002 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>It has been suggested that HilC and HilD may be capable of inducing expression of one another as well as inducing their own expression ( Lucas and Lee , 2001 ; Olekhnovich and Kadner , 2002 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16045614" target="_blank">16045614</a>]<br></li>
<li>As stated above , it is known that HilC and HilD can activate the expression of hilC and hilD independently of each other as well as inducing their own promoters ( 40 , 47 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18245288" target="_blank">18245288</a>]<br></li>
				<label class="TU_collapse_similarity" for="tu_similarity_203">
We demonstrate that HilC , HilD and RtsA are each capable of independently inducing expression of the hilC , hilD and rtsA genes , and that each can independently activate hilA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16045614" target="_blank">16045614</a>]<br></label>
<input id="tu_similarity_203", type="checkbox">
<div class="TU_similar">
<li>We show that HilD , HilC and RtsA can each independently activate expression of the hilD , hilC , rtsAB and hilA genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16045614" target="_blank">16045614</a>]<br></li></div>
<li>HilC and HilD are also capable of activating hilC and hilD expression independently of each other , and they act in a complex regulatory loop to control hilA expression ( 13 , 50 , 53 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18245288" target="_blank">18245288</a>]<br></li>
<li>Indeed , it appears that RtsA , HilC and HilD are all capable of inducing expression of hilA , hilC , hilD , and invF but that RtsA is preferentially capable of inducing expression of slrP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923082" target="_blank">12923082</a>]<br></li>
				<label class="TU_collapse_similarity" for="tu_similarity_207">
HilC , HilD , and RtsA are all also capable of activating expression of hilC , hilD , and rtsA independently of each other , constituting a complex feed-forward regulatory loop that controls hilA expression ( 13 , 14 , 15 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18245288" target="_blank">18245288</a>]<br></label>
<input id="tu_similarity_207", type="checkbox">
<div class="TU_similar">
<li>HilC , HilD , and RtsA are each capable of activating expression of hilC , hilD , and rtsA , creating a complex regulatory loop that controls hilA expression ( Fig. 1 ) ( 15 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17951383" target="_blank">17951383</a>]<br></li>
<li>HilC , HilD , and RtsA are all also capable of activating expression of hilC , hilD , and rtsA independent of each other and comprise a complex feed forward regulatory loop that controls hilA expression ( Fig. 1 ) ( 16 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li></div>
<li>We demonstrate that HilC , HilD and RtsA are each capable of inducing expression of hilC , hilD and rtsA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16045614" target="_blank">16045614</a>]<br></li>
<li>These data show that HilC , HilD and RtsA are each capable of independently inducing expression of hilC , hilD and rtsA , consistent with our model that HilC , HilD and RtsA constitute a feed forward regulatory loop ( Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16045614" target="_blank">16045614</a>]<br></li>
<li>Interestingly , RtsA induces expression of slrP 45-fold , while HilD and HilC are able to induce expression of slrP only four - and Ô¨Åvefold , respectively ( Fig. 7B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923082" target="_blank">12923082</a>]<br></li>
<li>HilC , HilD , and RtsA differentially induce expression of dsbA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14679226" target="_blank">14679226</a>]<br></li>
<li>Interestingly , RtsA is capable of inducing dsbA 10-fold , while HilC and HilD induce expression of dsbA approximately two - and fourfold , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14679226" target="_blank">14679226</a>]<br></li>
<li>( ii ) RtsA , HilC , and HilD differentially regulate expression of other genes independent of their effects on hilA expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923082" target="_blank">12923082</a>]<br></li>
<li>Effect of invF , hilA , and hilC-D mutations on RtsA induction of slrP ( A ) and ability of HilC , HilD , and RtsA to induce expression of hilA ( B ) and slrP ( C ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923082" target="_blank">12923082</a>]<br></li>
<li>HilD , HilC , and RtsA are capable of self-activat-ing expression in addition to activating expression of one another and of hilA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24375101" target="_blank">24375101</a>]<br></li>
<li>We wanted to determine if HilC and HilD are also capable of inducing expression of dsbA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14679226" target="_blank">14679226</a>]<br></li>
<li>Regulation of HilC and HilD is complex and both are capable of inducing expression of one another as well as their own expression in response to various environmental stimuli ( Ellermeier et al. , 2005 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21168230" target="_blank">21168230</a>]<br></li>
<li>HilC and HilD also induce expression of the inv/spa operon independent of HilA ( 2 , 26 , 67 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923082" target="_blank">12923082</a>]<br></li>
<li>RtsA can induce expression of invF independently of HilA , HilC , and HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923082" target="_blank">12923082</a>]<br></li>
<li>This suggests that RtsA can induce expression of invF independent of HilA , similar to what has been observed for both HilC and HilD ( 2 , 26 , 67 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923082" target="_blank">12923082</a>]<br></li>
<li>Effect of HilC - and HilD-binding sites on hilC expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12107132" target="_blank">12107132</a>]<br></li>
<li>Effect of HilD , HilC and HilA on invF expression from pSA7 and mutant derivatives of pSA7 in a Dspi1 mutant .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>Gre factors affect SPI-1 gene expression by modulating hilD expression The transcriptional expression of hilA is tightly modulated by three AraC activators , HilD , HilC and RtsA , which form an auto-inducing regulatory loop where each protein can activate its own expression , as well as expression of the other two regulators [ 15 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28426789" target="_blank">28426789</a>]<br></li>
				<label class="TU_collapse_similarity" for="tu_similarity_226">
Expression of hilA is controlled by the combined action of three AraC-like transcriptional activators : HilC , HilD , and RtsA ( 14 -- 16 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23504014" target="_blank">23504014</a>]<br></label>
<input id="tu_similarity_226", type="checkbox">
<div class="TU_similar">
<li>Expression of hilA is controlled by the combined action of three AraC-like transcriptional activators : HilC , HilD , and RtsA ( 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25182488" target="_blank">25182488</a>]<br></li></div>
<li>HilA transcriptional expression is autoregulated and tightly modulated by the combined action of three AraC-like transcriptional activators : HilC , HilD and RtsA [ 13,14 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28426789" target="_blank">28426789</a>]<br></li>
<li>Expression of HilA is controlled by the combined action of three AraC-like transcriptional activators : HilC and HilD , both encoded within SPI-1 ( 230 ) , and RtsA , encoded within an independent island ( 231 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23554419" target="_blank">23554419</a>]<br></li>
<li>Expression of hilA is directly controlled by three AraC-like transcriptional regulators , including SPI-1-encoded HilC and HilD , and RtsA encoded outside of SPI-1 , which constitute a feed-forward regulatory loop [ 29 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28575106" target="_blank">28575106</a>]<br></li>
<li>Expression of hilA itself is activated by three AraC family regulators , HilC , HilD , and RtsA ( 25 , 78 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18160484" target="_blank">18160484</a>]<br></li>
<li>Expression of hilA is directly controlled by three AraC-like activators : HilC , HilD , and RtsA ( 18 , 42 , 47 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
<li>HilC and HilD respond to different environmental conditions , but either alone is capable of activating hilA expression ( Lucas and Lee , 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21570144" target="_blank">21570144</a>]<br></li>
<li>HilD , HilC , and RtsA are each capable of activating hilA transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
<li>HilD is a dominant regulator of hilA transcription , while HilC and RtsA amplify hilA gene expression ( 12 , 13 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24488311" target="_blank">24488311</a>]<br></li>
<li>HilD is a dominant regulator of hilA transcription , and HilC and RtsA amplify its expression [ 27 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28329249" target="_blank">28329249</a>]<br></li>
<li>In view of these Ô¨Åndings , we suggest that the regulation of SPI1 expression is negatively controlled through deg-radation of the HilC and HilD transcriptional regulators by Lon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15661008" target="_blank">15661008</a>]<br></li>
<li>First , Lon is induced to control turnover of HilD and HilC and hence limit the expression of SPI-1 genes , which is of particular relevance after invasion of epithelial cells .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23554419" target="_blank">23554419</a>]<br></li>
<li>( iii ) RtsA , HilC , and HilD are required for signal ampliÔ¨Åcation such that once one of these genes is turned on , expression of the others is also induced , resulting in greater induction of hilA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923082" target="_blank">12923082</a>]<br></li>
<li>Thus , FliZ-mediated induction of hilA expression is through HilD , while RtsA and HilC act as ampliÔ¨Åers of the signal .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17951383" target="_blank">17951383</a>]<br></li>
<li>Expression of hilA is under the control of many environmental conditions and three transcriptional activators : HilC , HilD , and RtsA ( Ellermeier et al. , 2005 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27704705" target="_blank">27704705</a>]<br></li>
<li>In serovar Typhimurium , dsbA is controlled by the hilA regulators RtsA , HilD , and HilC , and expression of DsbA under SPI1-inducing conditions is independent of CpxRA ( 18 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17951383" target="_blank">17951383</a>]<br></li>
<li>Acetylation of K297 Regulating HilD DNA-Binding Ability and Stability At the transcriptional level , the expression of hilD is positively autoregulated and modulated by a feed-forward loop involving HilD itself and regulators HilC and RtsA [ 3 , 28 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28329249" target="_blank">28329249</a>]<br></li>
<li>The mechanism by which HilC , HilD and RtsA activate expression of hilA ( and possibly of each other ) apparently involves counteracting silencing by the nucleoid protein Hns [ 30,31 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17208038" target="_blank">17208038</a>]<br></li>
<li>Based on our genetic analyses , and taking into account the published results outlined above , we present a model in which expression of hilA is controlled by the combined action of HilC , HilD and RtsA , each of which can independently bind upstream of hilA to induce its expression ( Schechter and Lee , 2001 ; Olekhnovich and Kadner , 2002 ; Ellermeier and Slauch , 2003 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16045614" target="_blank">16045614</a>]<br></li>
<li>The expression of hilA , the master regulator for SPI1 , is controlled by HilD , HilC , and RtsA , which act in a complex feed-forward loop .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20889744" target="_blank">20889744</a>]<br></li>
<li>Expression of hilA is directly controlled by three AraC-like regulators , HilD , HilC , and RtsA ( 5 , 6 ) , each of which is independently capable of inducing expression of the hilD , hilC , and rtsA genes , as well as hilA , forming a complex feed-forward regulatory loop to control SPI1 expression ( 7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26884427" target="_blank">26884427</a>]<br></li>
				<label class="TU_collapse_similarity" for="tu_similarity_248">
The expression of hilA , encoding the transcriptional activator of the T3SS structural genes , is directly controlled by three AraC-like regulators , HilD , HilC , and RtsA , each of which can activate hilD , hilC , rtsA , and hilA genes , forming a complex feed-forward regulatory loop .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26884427" target="_blank">26884427</a>]<br></label>
<input id="tu_similarity_248", type="checkbox">
<div class="TU_similar">
<li>The expression of hilA , encoding the transcriptional activator of the SPI1 structural genes , is directly controlled by three AraC-like regulators , HilD , HilC , and RtsA , each of which can activate the hilD , hilC , rtsA , and hilA genes , forming a complex feed-forward regulatory loop .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22021388" target="_blank">22021388</a>]<br></li></div>
<li>Effect of upstream DNA on stimulation of hilA transcription by HilC and HilD proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15126488" target="_blank">15126488</a>]<br></li>
<li>Regulation of gene expression by RtsA , HilC and HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16045614" target="_blank">16045614</a>]<br></li>
<li>LoiA activates expression of hilA gene through activating hilD As the expression of hilA is directly controlled by HilD , HilC and RtsA , we tested whether LoiA regulates HilA through any of these three regulators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28575106" target="_blank">28575106</a>]<br></li>
<li>The expression of hilA is controlled by a complex feed-forward regulatory loop consisting of HilD , HilC , and RtsA , each of which can independently activate hilA expression ( 13 ) ( Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20889744" target="_blank">20889744</a>]<br></li>
<li>A second possibility is that RtsA induces expression of hilA independent of HilC and HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923082" target="_blank">12923082</a>]<br></li>
<li>Regulation of hilA expression by RtsA , HilC , and HilD is an example of a feedforward loop .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923082" target="_blank">12923082</a>]<br></li>
<li>RtsA increases invasion gene expression by increasing transcription of hilA , which is independent of HilC and HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15765064" target="_blank">15765064</a>]<br></li>
<li>HilC plays a key role in co-ordinating expression of the SPI-1 genes [ 66 -- 68 ] , and its transcription was found to be decreased 43.82-fold in Dfis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23717649" target="_blank">23717649</a>]<br></li>
<li>If a condition exists in which hilA is repressed while invF expression is induced through HilC or HilD , it begs the question : Why produce InvF-dependent effectors in the absence of hilA expression when an intact type III secretion apparatus is not thought to be formed ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10844688" target="_blank">10844688</a>]<br></li>
<li>In this case overexpression of HilC or HilD may increase hilA expression by overcoming the enhanced repressing mechanism in these mutants .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11755416" target="_blank">11755416</a>]<br></li>
<li>To determine whether HilE , Pag , or Hha affects repression , rather than derepression , of hilA , the effect that hilE , pag , and hha mutations have on hilA expression should be examined under repressing environmental conditions in strains lacking HilC and HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12933899" target="_blank">12933899</a>]<br></li>
<li>However , as a result of the speciÔ¨Åc environmental conditions , hilA expression may be repressed in such a way that HilD and HilC can not derepress it , or HilA activity may be downregulated post-transcriptionally .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>These results also explain why our invF-lacZ reporter plasmid , pVV448 , was not activated by HilD or HilC to the same extent as the chromosomal invF-lacZ fusion because pVV448 does not contain the HilD - and HilC-dependent +1 of invF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>Recall that HilD , but not HilC , activates expression of invF sevenfold from the pVV448 reporter plasmid , which carries this region ( Fig. 4 , black bars ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>These results suggest that HilD and HilC might directly activate invF expression by binding to sequences upstream of invF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>In addition to activating invF expression , HilD and HilC may affect the invH gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>These results indicate that HilD and HilC require the downstream binding site to activate invF expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>Both HilC and HilD also directly activate invF , and it has been suggested that they activate a promoter located upstream of the HilA-dependent promoter PinvF ( 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923082" target="_blank">12923082</a>]<br></li>
<li>Our studies show that HilD and HilC activate transcription of invF from a promoter that is far upstream of its HilA-dependent promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>HilD and HilC do not activate invF expression from the HilA-dependent invF promoter Previously , we had shown that HilA activates invF expression by direct interaction with sequences upstream of invF called the HilA box ( Lostroh et al. , 2000 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>HilC and HilD also directly activate the alternative promoter of the invF operon , which is independent of HilA ( 15,16 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17329372" target="_blank">17329372</a>]<br></li>
<li>HilA and HilC activate the invF gene , the product of which activates genes encoding secreted effectors , including sopB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14645287" target="_blank">14645287</a>]<br></li>
<li>The rationale behind this hilC hilD promoter replacement experiment was to see whether HilC alone could induce HilA expression if expressed from the P promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20686667" target="_blank">20686667</a>]<br></li>
<li>Under the standard growth condition , hilAp activity in the HilC mutant in the exponential phase was equivalent to that in the WT but was not induced at the entry of the stationary phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15161921" target="_blank">15161921</a>]<br></li>
<li>The 5 end of the in vitro hilC transcript produced in the absence of HilC was mapped to position 71 , relative to the start used in the presence of HilC ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12107132" target="_blank">12107132</a>]<br></li>
				</div></ul></div><br>
	</div>
</body>
</html>