<!doctype html>
<style>
@charset "utf-8";
body {
	height: 100%;
	margin: 0;
	padding: 0;
}

.banner {
	position: fixed;
	top: 0;
	width: 100%;
	height: 65px;
	background: gray;
	line-height: 65px;
}
.spacer {
	height: 10px;
}

.main-content {
	padding: 5%;
}

.collapse_ACT {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-style: none solid none none;
	border-color: #B4045F;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_ACT + input{
	display: none;
}
.collapse_ACT + input + div{
	display: none;
}
.collapse_ACT + input:checked + div{
	display: block;
}
.ACT {
	text-align: justify;
	text-justify: inter-word;
}
.ACT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B4045F;
}
.ACT_collapse_similarity:hover{
	background: #e5d3dc;
}
.ACT_collapse_similarity + input {
	display: none;
}
.ACT_collapse_similarity + input + div{
	display: none;
}

.ACT_collapse_similarity + input:checked + div{
	display: block;
}
.ACT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.ACT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B4045F;
	padding-left: 5px;
}
.ACT_collapse_similarity:hover{
	background: #e5d3dc;
}
.ACT_collapse_similarity + input {
	display: none;
}
.ACT_collapse_similarity + input + div{
	display: none;
}
.ACT_collapse_similarity + input:checked + div{
	display: block;
}
.ACT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.ACT_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #B4045F;
}
.ACT_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_DOM {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-style: none solid none none;
	border-color: #B404AE;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_DOM + input{
	display: none;
}
.collapse_DOM + input + div{
	display: none;
}
.collapse_DOM + input:checked + div{
	display: block;
}
.DOM {
	text-align: justify;
	text-justify: inter-word;
}
.DOM_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B404AE;
	padding-left: 5px;
}
.DOM_collapse_similarity:hover{
	background: #e8d7e7;
}
.DOM_collapse_similarity + input {
	display: none;
}
.DOM_collapse_similarity + input + div{
	display: none;
}
.DOM_collapse_similarity + input:checked + div{
	display: block;
}
.DOM_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}.DOM_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #B404AE;
}
.DOM_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_EVO {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #7401DF;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_EVO + input{
	display: none;
}
.collapse_EVO + input + div{
	display: none;
}
.collapse_EVO + input:checked + div{
	display: block;
}
.EVO {
	text-align: justify;
	text-justify: inter-word;
}
.EVO_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #7401DF;
	padding-left: 5px;
}
.EVO_collapse_similarity:hover{
	background: #e2d7ed;
}
.EVO_collapse_similarity + input {
	display: none;
}
.EVO_collapse_similarity + input + div{
	display: none;
}
.EVO_collapse_similarity + input:checked + div{
	display: block;
}
.EVO_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.EVO_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #7401DF;
}
.EVO_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_RP {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #0174DF;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_RP + input{
	display: none;
}
.collapse_RP + input + div{
	display: none;
}
.collapse_RP + input:checked + div{
	display: block;
}
.RP {
	text-align: justify;
	text-justify: inter-word;
}
.RP_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #0174DF;
	padding-left: 5px;
}
.RP_collapse_similarity:hover{
	background: #c9d7e2;
}
.RP_collapse_similarity + input {
	display: none;
}
.RP_collapse_similarity + input + div{
	display: none;
}
.RP_collapse_similarity + input:checked + div{
	display: block;
}
.RP_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.RP_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #0174DF;
}
.RP_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_SIT {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #0489B1;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_SIT + input{
	display: none;
}
.collapse_SIT + input + div{
	display: none;
}
.collapse_SIT + input:checked + div{
	display: block;
}
.SIT {
	text-align: justify;
	text-justify: inter-word;
}
.SIT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #0489B1;
	padding-left: 5px;
}
.SIT_collapse_similarity:hover{
	background: #d0dfe2;
}
.SIT_collapse_similarity + input {
	display: none;
}
.SIT_collapse_similarity + input + div{
	display: none;
}
.SIT_collapse_similarity + input:checked + div{
	display: block;
}
.SIT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.SIT_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #0489B1;
}
.SIT_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_TU {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #04B486;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_TU + input{
	display: none;
}
.collapse_TU + input + div{
	display: none;
}
.collapse_TU + input:checked + div{
	display: block;
}
.TU {
	text-align: justify;
	text-justify: inter-word;
}
.TU_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #04B486;
	padding-left: 5px;
}
.TU_collapse_similarity:hover{
	background: #d0e2de;
}
.TU_collapse_similarity + input {
	display: none;
}
.TU_collapse_similarity + input + div{
	display: none;
}
.TU_collapse_similarity + input:checked + div{
	display: block;
}
.TU_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.TU_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #04B486;
}
.TU_similar > a {
	text-decoration: none;
	color: gray;
}

ul { 
	list-style-type: none;
	padding: 0;
	margin: 0;
}
ul li > a {
	text-decoration: none;
	color: gray;
}
ul li:hover, ul li:focus {
	background: #F0F0F0;
}

ul li:first-child span {
  border-left:none;  
}

</style>
<html>
<head>
<meta charset="utf-8">
<title>TF properties - HilD</title>
<meta name="theme-color" content="#ffffff">
</head>
<body>
	<div class="banner">
		<center><font color="white" size="16px" face="arial">HilD</font></center>
	</div>
	<div align="center" class="main-content">
		<div class="spacer">
	</div><br>			<label class="collapse_ACT" for="_1"><font size="5px">HilD transcriptional activity</font></label>
				<input id="_1" type="checkbox">
				<div class="ACT"><br><ul><li>It was implied that FliZ somehow controls HilD translation , although no direct evidence was given .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20889744" target="_blank">20889744</a>]<br></li>
<li>Moreover , at higher Tet concentrations , HilE-dependent regulation was evident when HilD was present ( Figure 3B ) , although regulation became less dramatic .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22021388" target="_blank">22021388</a>]<br></li>
<li>RcsB represses SPI1 through HilD ( Mouslim et al. , 2004 ; Lin et al. , 2008 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25972862" target="_blank">25972862</a>]<br></li>
<li>doi : 10.3389 / fmicb .2015.00807 Keywords : Salmonella , SPI , CpxR/A , HilD , Lon , regulation , RpoH , virulence Introduction Salmonella enterica groups Gram-negative bacteria comprising around 2500 serotypes , which can infect a wide variety of hosts ranging from humans to birds ( Haraga et al. , 2008 ; SánchezVargas et al. , 2011 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26300871" target="_blank">26300871</a>]<br></li>
<li>HilD inactivation disrupts a positive feedback loop for hilD autogenous activation and causes SPI-1 repression ( Ellermeier et al. , 2005 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24354910" target="_blank">24354910</a>]<br></li>
<li>This may explain why HilD regulates Spi-1 and Spi-2 differentially ( Bustamante et al. , 2008 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26441883" target="_blank">26441883</a>]<br></li>
<li>Positive regulators include BarA/SirA , FliZ , EnvZ/OmpR , FadD , CsrB , HilD and some nucleoid binding proteins ( Fis , HU ) , while Pag , HilE , Ams and the nucleoid binding proteins Hha and H-NS are negatively regulating hilA transcription ( Altier et al. 2000a , b Fahlen et al. 2000 ; Lucas et al. 2000 ; Wilson et al. 2001 ; Schechter et al. 2003 ; Baxter et al. 2003 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18563621" target="_blank">18563621</a>]<br></li>
<li>HilD directly regulates all other SPI-1-associated TFs .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27601571" target="_blank">27601571</a>]<br></li>
<li>Fur regulates HilD in a different manner than it regulates sodB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
<li>In contrast , HilD positively regulates FlhDC ( see Fig .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27601571" target="_blank">27601571</a>]<br></li>
<li>FliZ regulates HilD posttranslationally .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20889744" target="_blank">20889744</a>]<br></li>
<li>HilD ; lysine acetylation ; stability ; DNA-binding ; virulence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28329249" target="_blank">28329249</a>]<br></li>
<li>HilD K297 is essential for its DNA-binding ability .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28329249" target="_blank">28329249</a>]<br></li>
<li>The protein acyltransferase Pat post-transcriptionally controls HilD to repress Salmonella invasion .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28329249" target="_blank">28329249</a>]<br></li>
<li>These seem to function primarily through HilE , which binds directly to HilD , presumably preventing its action ( 5 , 15 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18723621" target="_blank">18723621</a>]<br></li>
<li>( iv ) HilE could regulate HilD function after HilD has bound to DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29378886" target="_blank">29378886</a>]<br></li>
<li>This may be due to HilD countersilencing H-NS repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21075923" target="_blank">21075923</a>]<br></li>
<li>This activation appears to be HilD dependent ( 40 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12595445" target="_blank">12595445</a>]<br></li>
<li>All regulation is through HilD , consistent with our current model for SPI1 regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17951383" target="_blank">17951383</a>]<br></li>
<li>This regulation is clearly mediated through HilD , but FliZ controls neither hilD transcription nor translation initiation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20889744" target="_blank">20889744</a>]<br></li>
<li>Moreover , FliZ can affect regulation when HilD protein is expressed ectopically .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20889744" target="_blank">20889744</a>]<br></li>
<li>It is possible that HilD , but not HilC , activates invF expression from a second promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>It is possible that RtsA , HilC and HilD each respond to an independent signal to activate SPI1 , but what these signals are is not clear .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16045614" target="_blank">16045614</a>]<br></li>
<li>It is suggested that HilD directly activates hilA transcription ( 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18245288" target="_blank">18245288</a>]<br></li>
<li>Surprisingly , our results indicate that functional HilD is required for hilA expression even when known negative regulators are deleted .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12511499" target="_blank">12511499</a>]<br></li>
<li>( B ) Fur overexpression induces hilA expression when HilD is present .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
<li>P as well as P is hilA invF-2 hilA known to be directly activated , although only slightly upregulating , by HilC/HilD ( Akbar et al. , 2003 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15661008" target="_blank">15661008</a>]<br></li>
<li>hilA is itself controlled by HilD , another transcriptional-activator within SPI1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18424519" target="_blank">18424519</a>]<br></li>
<li>The rtsA gene is located outside SPI-1 but is directly activated by HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25991823" target="_blank">25991823</a>]<br></li>
<li>This is important , as HilD is known to induce its own expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27185788" target="_blank">27185788</a>]<br></li>
<li>Nonetheless , this OmpR-mediated regulation is reported to act through HilD , supposedly by posttranscriptional activation ( 222 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23554419" target="_blank">23554419</a>]<br></li>
<li>The added proteins in each lane are as follows : lane 2 , no added protein ; lane 3 , 150 nM HilC ; lane 4 , 300 nM HilC ; lane 5 , 150 nM HilD ; lane 6 , 300 nM HilD ; lane 7 , 150 nM MBP-RtsA ; lane 8 , 300 nM MBP-RtsA FIG. 8 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17675384" target="_blank">17675384</a>]<br></li>
<li>The proteins added in each lane are as follows : lanes 1 , no added protein ; lanes 2 and 5 , 75 nM HilC ; lanes 3 and 6 , 75 nM HilD ; lanes 4 and 7 , 75 nM MBP-RtsA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17675384" target="_blank">17675384</a>]<br></li>
<li>The added proteins in each lane are as follows : lanes 1 , no added protein ; lanes 2 , 50 nM HilC ; lanes 3 , 100 nM HilC ; lanes 4 , 50 nM HilD , and lanes 5 , 100 nM HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12107132" target="_blank">12107132</a>]<br></li>
<li>Another regulator , FliZ , may activate SPI-1 expression by controlling HilD activity ( 13 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24682814" target="_blank">24682814</a>]<br></li>
<li>The evidence that L-arabinose regulates SPI-1 expression through HilD suggested that hilD expression itself might be controlled by L-arabinose .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25991823" target="_blank">25991823</a>]<br></li>
<li>In our study , HilC expression was not influenced by E. coli supernatants , while HilD expression was slightly enhanced .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21379575" target="_blank">21379575</a>]<br></li>
<li>SprB directly activates SPI4 gene expression and weakly represses SPI1 gene expression through HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20190046" target="_blank">20190046</a>]<br></li>
<li>245 SprB transcriptionally represses HilD expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20190046" target="_blank">20190046</a>]<br></li>
<li>Our results show that Pat regulates HilD by reducing its stability .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27341691" target="_blank">27341691</a>]<br></li>
<li>It was initially proposed that OmpR-EnvZ directly controls hilC expression [ 37 ] , but others have favoured a model in which OmpR somehow acts post-transcrip-tionally through HilD protein function [ 2 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22457642" target="_blank">22457642</a>]<br></li>
<li>In E. coli , a hilA -- lacZY fusion was regulated by HilE only when HilD protein was present .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17208038" target="_blank">17208038</a>]<br></li>
<li>HilD was regulated by Pat in a posttranslational modification manner .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26810370" target="_blank">26810370</a>]<br></li>
<li>Coincident with this reduction in HilD protein , hilA = - lacZ expression was potently repressed by bile ( 9.7-fold repression ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27185788" target="_blank">27185788</a>]<br></li>
<li>FliZ overproduction increases hilA transcription in a HilD-dependent manner , suggesting that FliZ activates hilA by posttranscriptionally controlling HilD ( 247 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23554419" target="_blank">23554419</a>]<br></li>
<li>To test changes in hilA expression in wild-type , ΔloiA , ΔhilD , ΔloiAΔhilD , ΔloiAΔhilD + pHilD and ΔloiAΔhilD + pLoiA , strains were also grown under SPI-1-inducing conditions for 4 h to late-exponential phase ( OD600 ~ 0.6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28575106" target="_blank">28575106</a>]<br></li>
<li>Western blot assays To analyse HilA protein levels in wild-type , ΔloiA , ΔhilD , ΔloiAΔhilD , ΔloiAΔhilD + pHilD and ΔloiAΔhilD + pLoiA strains , the corresponding 3 × FLAG-tagged strains were grown in SPI ¬ 1-inducing conditions and then collected .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28575106" target="_blank">28575106</a>]<br></li>
<li>We next examined HilD protein levels in response to FliZ - or HilE-mediated regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20889744" target="_blank">20889744</a>]<br></li>
<li>Alternatively , HilD may be phosphorylated or otherwise modiﬁed such that its activity is modulated in response to particular conditions or mutations .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11292791" target="_blank">11292791</a>]<br></li>
<li>Since hilD transcription appears to be constitutive , one possible mechanism for Lon activity is that it degrades HilD in repressing environmental conditions to regulate hilA expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15765064" target="_blank">15765064</a>]<br></li>
<li>The data in Fig. 2B indicate that HilE speciﬁcally binds HilD but not HilC or RtsA in vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29378886" target="_blank">29378886</a>]<br></li>
<li>The data in Fig. 5 suggest that both HilD and HilC homodimerize , as evidenced by an IPTG-dependent decrease in - galactosidase activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29378886" target="_blank">29378886</a>]<br></li>
<li>S1A in the supplemental material , HilD induction , surprisingly , resulted in about a 50 % increase in rﬂM transcript levels .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24488311" target="_blank">24488311</a>]<br></li>
<li>As shown by the results in Fig. 1B , ﬂhC-lac expression increased approximately 3-fold upon HilD overexpression in an attenuated Salmonella LT2 background under normal LB growth conditions ( Fig. 1B , bar 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24488311" target="_blank">24488311</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_55">
Of the three , HilD has a predominant role , but apparently does not act alone either in vivo or in vitro to sufﬁciently activate SPI1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16045614" target="_blank">16045614</a>]<br></label>
<input id="act_similarity_55", type="checkbox">
<div class="ACT_similar">
<li>Of these three , HilD has a predominant role , but apparently can not act alone either in vitro or in vivo to sufﬁciently activate SPI1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16045614" target="_blank">16045614</a>]<br></li></div>
<li>c Genes conﬁrmed to be regulated by HilD in this or previous studies are indicated in bold text .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24375101" target="_blank">24375101</a>]<br></li>
<li>Unfortunately , no effect on hilA transcription has been studied in order to clarify how it activates SPI-1 ( in general , all 2CRSs are reported to affect HilA through modiﬁcations in HilD ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23554419" target="_blank">23554419</a>]<br></li>
<li>The reduction in protein concentration was speciﬁc to HilD , as loading controls demonstrated equal protein content in all samples ( see Fig .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27185788" target="_blank">27185788</a>]<br></li>
<li>Genes given in gray text were not identiﬁed as being HilD bound in this study .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24375101" target="_blank">24375101</a>]<br></li>
<li>qRT-PCR analysis showed that loiA gene expression was 7.2-fold higher under the in vitro SPI-1-inducing conditions than under non-inducing conditions ( low salt , high O2 ) ( S6 Fig ) , indicating LoiA responds to high osmolarity or low O2 or both to regulate SPI-1 expression via HilD and HilA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28575106" target="_blank">28575106</a>]<br></li>
<li>Still , it is intriguing to consider that under certain conditions HilC and HilD may be able to activate hilA expression and that under other conditions they may be able to increase hilA expression by derepressing hilA transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12933899" target="_blank">12933899</a>]<br></li>
<li>The fact that hilA expression is regulated by L-arabinose might indicate either that L-arabinose directly regulates hilA or that the regulatory input is transmitted to hilA via HilC and/or HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25991823" target="_blank">25991823</a>]<br></li>
<li>Simplistically this suggests that RtsA acts as a better inducer of slrP expression than does either HilC or HilD , at least compared to their ability to induce hilA expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923082" target="_blank">12923082</a>]<br></li>
<li>It was demonstrated that FliZ posttranslationally controls HilD to positively regulate expression of hilA [ 67 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22632036" target="_blank">22632036</a>]<br></li>
<li>HilD is post-translationally activated by FliZ ( Chubiz et al. , 2010 ) , and therefore represents a potential growth-phase specific mechanism by which FliZ could upregulate flhDC transcription coordinately with SPI-1 genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25315056" target="_blank">25315056</a>]<br></li>
<li>On the other hand , HilD activity is positively regulated by FliZ and Fur , through still unknown mechanisms ( Ellermeier and Slauch , 2008 ; Chubiz et al. , 2010 ) , as well as negatively regulated by HilE , through protein-protein interactions ( Baxter et al. , 2003 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26300871" target="_blank">26300871</a>]<br></li>
<li>This is achieved by the ﬂagellar-encoded regulator FliZ , which simultaneously represses type 1 ﬁmbriae and activates Spi-1 via HilD ( Iyoda et al. , 2001 ; Chubiz et al. , 2010 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26441883" target="_blank">26441883</a>]<br></li>
<li>In Salmonella , H-NS silencing of SPI-2 is antagonized by HilD or by SsrB proteins in response to different environmental conditions ( Walthers et al. , 2007 , 2011 ; Bustamante et al. , 2008 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21680637" target="_blank">21680637</a>]<br></li>
<li>pone .0015466 Lahiri et al. , 2008 PMID : 18625332 Humphreys et al. , 2012 PMID : 22341462 → Substrate competition Salmonella arginase II and iNOS of the host → Salmonella up-regulates arginase II activity in RAW264 .7 macrophages → down regulates host iNOS → by this in intestinal lumen beneﬁcial increase of electron acceptor nitrate Arginine has no decarboxylase , but key immune modulator from Salmonella Feature Result Author , weblink THE INTERPLAY OF SALMONELLA PATHOGENICITY ISLANDS AND METABOLISM IacP downstream of sipA for effector-protein within → Facilitates Salmonella invasion to HeLa cells by secretion SPI1 of SPI1 effectors SopA , SopB , and SopD → IacP activated by 4 ′ - phosphopantetheine transferase AcpS Kaniga et al. , 1995 PMCID : PMC177584 Kim et al. , 2011 PMID : 21263021 Viala et al. , 2013 PMID : 23893113 Hung et al. , 2013 PMID : 23289537 Viala et al. , 2013 PMID : 23893113 Lopez et al. , 2012 PMID : 22691391 Rowley et al. , 2012 PMID : 22039967 → Propionyl-CoA represses AcpS , HilD , Salmonella invasion → Low SopB secretion → May be priming of fatty acid metabolism inside the SCV .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25688337" target="_blank">25688337</a>]<br></li>
<li>To verify the importance of HilD in repression of invasion by propionate , we additionally tested its effect on sopB , shown by microarray analysis to be strongly repressed by propionic acid ( Table 1 ) and encoding a TTSS effector located outside SPI1 but controlled by SPI1 regulators ( Ahmer et al. , 1999 , Pfeifer et al. , 1999 , Ehrbar et al. , 2002 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23289537" target="_blank">23289537</a>]<br></li>
<li>ls the expression of SPI-1 and SPI-2 genes by regulating HilD expression tro n o c irA S HilD and OmpR positively regulate ssrAB expression by directly binding to its regulatory region ( Lee et al. , 2000 ; Feng et al. , 2003 , 2004 ; Bustamante et al. , 2008 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21518393" target="_blank">21518393</a>]<br></li>
<li>For example , SPI-2-encoded SsrB downregulates SPI-1 genes by repressing HilA and HilD ( Pérez-Morales et al. , 2017 ) , and reciprocally , SPI-1-encoded HilD upregulates SPI-2 gene expression by directly binding the ssrAB operon ( Bustamante et al. , 2008 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29034217" target="_blank">29034217</a>]<br></li>
<li>Furthermore , HilD regulates directly , or indirectly , through HilA and InvF , the expression of several other genes located outside SPI-1 , including acquired and ancestral genes ( Bustamante et al. , 2008 ; Golubeva et al. , 2012 ; Fàbrega and Vila , 2013 ; Petrone et al. , 2014 ; Singer et al. , 2014 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26300871" target="_blank">26300871</a>]<br></li>
<li>Transcription of invF is primarily regulated by HilA , but HilD , HilC and RtsA are each capable of partially activating invF ( Eichelberg et al. , 1999 ; Rakeman et al. , 1999 ; Akbar et al. , 2003 ; Ellermeier and Slauch , 2003 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16045614" target="_blank">16045614</a>]<br></li>
<li>This data , along with in vivo data from Ellermeier et al. [ 19 ] might suggest the presence of mixed HilC , HilD and RtsA dimers .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17208038" target="_blank">17208038</a>]<br></li>
<li>Furthermore , HilC and HilD can activate inv/sicA transcription in the absence of HilA ( Rakeman et al. 1999 ; Akbar et al. 2003 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20008574" target="_blank">20008574</a>]<br></li>
<li>Production of HilC induced expression of hilA ~ 120-fold , while RtsA and HilD induced expression of hilA 30 - to 40-fold , similar to previously reported values ( Eichelberg et al. , 1999 ; Rakeman et al. , 1999 ; Schechter et al. , 1999 ; Schechter and Lee , 2001 ; Ellermeier and Slauch , 2003 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16045614" target="_blank">16045614</a>]<br></li>
<li>Furthermore , HilD activates transcription of hilC and of hilD itself ( Ellermeier et al. 2005 ) by direct binding to both promoters ( Olekhnovich and Kadner 2002 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25991823" target="_blank">25991823</a>]<br></li>
<li>Indeed , it has been proposed that the molecular role of HilD and HilC is to derepress hilA ( Schechter et al. , 1999 , 2003 ; Schechter and Lee , 2001 ) , likely by counteracting H-NS , although there is debate on this point ( Boddicker et al. , 2003 ; Olekhnovich and Kadner , 2004 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16045614" target="_blank">16045614</a>]<br></li>
<li>Under these growth conditions , two AraC-like transcriptional-regulators , HilD and HilC , encoded within SPI-1 , activate hilA expression ( Schechter et al. , 1999 ; Schechter and Lee , 2001 , Ellermeier et al. , 2005 ) , the central positive regulator of SPI-1 , by counteracting the repression exerted by the global repressor H-NS on the hilA promoter ( Schechter et al. , 2003 ; Olekhnovich and Kadner , 2006 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21518393" target="_blank">21518393</a>]<br></li>
<li>The regulatory cascade diverges when HilD , in a growth-phase-dependent manner , activates expression of hilA and the ssrAB operon , by counteracting the repression exerted by H-NS on their promoters ( Bustamante et al. , 2008 , Schechter et al. , 2003 ; Olekhnovich and Kadner , 2006 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21518393" target="_blank">21518393</a>]<br></li>
<li>The appropriate concentration and activity of HilD is maintained by additional negative and positive regulatory mechanisms , including degradation of HilD mediated by the Lon protease ( Boddicker and Jones , 2004 ; Takaya et al. , 2005 ) , HilD inactivation by direct interaction with HilE ( Baxter et al. , 2003 ) , positive auto-regulation of HilD , as well as by a feed forward positive regulatory loop involving the HilD , HilC and RtsA AraC-like regulators ( Olekhnovich and Kadner , 2002 ; Ellermeier et al. , 2005 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21518393" target="_blank">21518393</a>]<br></li>
<li>Furthermore , since HilE regulates-the-activity of HilD by protein-protein interaction ( Baxter et al. , 2003 ) , 1hilE and 1cpxA 1hilE mutants were also constructed and used as controls .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26300871" target="_blank">26300871</a>]<br></li>
<li>The action of Fur in SPI1 regulation is more complicated ( Troxell et al. 2010 ; Teixido et al. 2011 ) , but also requires HilD ( Ellermeier and Slauch 2008 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22021388" target="_blank">22021388</a>]<br></li>
<li>HilE is a negative regulator of HilD activity that works by direct protein -- protein interaction ( Baxter et al. 2003 ; J. E. Chubiz and J. M. Slauch , unpublished data ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22021388" target="_blank">22021388</a>]<br></li>
<li>One of these , HilE , is a negative regulator of SPI1 that directly binds HilD protein ( Baxter et al. 2003 ; J. E. Chubiz and J. M. Slauch , unpublished results ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22021388" target="_blank">22021388</a>]<br></li>
<li>Mechanistically , HilD indirectly regulates SsrB-regulated genes by antagonizing the H-NS-mediated repression of ssrAB at stationary phase under SPI-1 inducing conditions [ 30 ] ( see also Fig 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27564394" target="_blank">27564394</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_89">
Martínez , L. C. , Banda , M. M. , Fernández-Mora , M. , Santana , F. J. & Bustamante , V. H. HilD induces expression of Salmonella pathogenicity island 2 genes by displacing the global negative regulator H-NS from ssrAB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27886269" target="_blank">27886269</a>]<br></label>
<input id="act_similarity_89", type="checkbox">
<div class="ACT_similar">
<li>HilD induces expression of Salmonella pathogenicity island 2 genes by displacing the global negative regulator H-NS from ssrAB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26300871" target="_blank">26300871</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28329249" target="_blank">28329249</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28660201" target="_blank">28660201</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28426789" target="_blank">28426789</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29447698" target="_blank">29447698</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28426789" target="_blank">28426789</a>]<br></li></div>
				<label class="ACT_collapse_similarity" for="act_similarity_91">
Microbiol Mol Biol Rev 64:694 -- 708 Chubiz JE , Golubeva YA , Lin D , Miller LD , Slauch JM ( 2010 ) FliZ regulates expression of the Salmonella pathogenicity island 1 invasion locus by controlling HilD protein activity in Salmonella enterica serovar typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25972986" target="_blank">25972986</a>]<br></label>
<input id="act_similarity_91", type="checkbox">
<div class="ACT_similar">
<li>Chubiz JEC , Golubeva YA , Lin D , Miller LD , Slauch JM : FliZ regulates expression of the Salmonella pathogenicity island 1 invasion locus by controlling HilD protein activity in Salmonella enterica serovar typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23295139" target="_blank">23295139</a>]<br></li>
<li>Chubiz JE , Golubeva YA , Lin D , Miller LD , Slauch JM : FliZ regulates expression of the Salmonella pathogenicity island 1 invasion locus by controlling HilD protein activity in Salmonella enterica serovar typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22632036" target="_blank">22632036</a>]<br></li>
<li>Chubiz , J. E. , Golubeva , Y. A. , Lin , D. , Miller , L. D. & Slauch , J. M. FliZ regulates expression of the Salmonella pathogenicity island 1 invasion locus by controlling HilD protein activity in Salmonella enterica serovar typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27886269" target="_blank">27886269</a>]<br></li>
<li>[ 72 ] J.E.C. Chubiz , Y.A. Golubeva , D. Lin , L.D. Miller , J.M. Slauch , FliZ regulates expression of the Salmonella pathogenicity island 1 invasion locus by controlling HilD protein activity in Salmonella enterica serovar typhimurium , J. Bacteriol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29369799" target="_blank">29369799</a>]<br></li>
<li>[ 87 ] J.E.C. Chubiz , Y.A. Golubeva , D. Lin , L.D. Miller , J.M. Slauch , FliZ regulates expression of the Salmonella pathogenicity island 1 invasion locus by controlling HilD protein activity in Salmonella enterica serovar typhimurium , J. Bacteriol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23040276" target="_blank">23040276</a>]<br></li>
<li>Chubiz , J. E. , Y. A. Golubeva , D. Lin , L. D. Miller , and J. M. Slauch , 2010 FliZ regulates expression of the SPI1 invasion locus by controlling HilD protein activity in Salmonella enterica serovar Typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22021388" target="_blank">22021388</a>]<br></li>
<li>Chubiz , J.E. , Golubeva , Y.A. , Lin , D. , Miller , L.D. and Slauch , J.M. ( 2010 ) FliZ regulates expression of the Salmonella pathogenicity island 1 invasion locus by controlling HilD protein activity in Salmonella enterica serovar Typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24682814" target="_blank">24682814</a>]<br></li>
<li>Chubiz , J.E. , Golubeva , Y.A. , Lin , D. , Miller , L.D. , and Slauch , J.M. ( 2010 ) FliZ regulates expression of the Salmonella pathogenicity island 1 invasion locus by controlling HilD protein activity in Salmonella enterica serovar Typhimu-rium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22356617" target="_blank">22356617</a>]<br></li>
<li>Chubiz , J.E. , Golubeva , Y.A. , Lin , D. , Miller , L.D. and Slauch , J.M. ( 2010 ) FliZ regulates expression of the Salmonella pathogenicity island 1 invasion locus by controlling HilD protein activity in Salmonella enterica serovar typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28973452" target="_blank">28973452</a>]<br></li>
<li>Chubiz , J.E. , Golubeva , Y.A. , Lin , D. , Miller , L.D. , and Slauch , J.M. ( 2010 ) FliZ regulates expression of the Salmonella pathogenicity island 1 invasion locus by controlling HilD protein activity in Salmonella enterica serovar typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24127899" target="_blank">24127899</a>]<br></li>
<li>Chubiz , J.E.C. , Golubeva , Y.A. , Lin , D. , Miller , L.D. , and Slauch , J.M. ( 2010 ) FliZ regulates expression of the Sal-monella pathogenicity island 1 invasion locus by controlling HilD protein activity in Salmonella enterica serovar Typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27206164" target="_blank">27206164</a>]<br></li>
<li>Chubiz , J.E. , Golubeva , Y.A. , Lin , D. , Miller , L.D. , and Slauch , J. M. ( 2010 ) FliZ regulates expression of the salmonella pathogenicity island 1 invasion locus by controlling HilD protein activity in Salmonella enterica serovar typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27404739" target="_blank">27404739</a>]<br></li>
<li>Chubiz , J.E. , Golubeva , Y.A. , Lin , D. , Miller , L.D , and Slauch , J.M. ( 2010 ) FliZ regulates expression of the sal-monella pathogenicity island 1 invasion locus by controlling HilD protein activity in Salmonella enterica serovar typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27341691" target="_blank">27341691</a>]<br></li></div>
				<label class="ACT_collapse_similarity" for="act_similarity_105">
Eade , C.R. , Hung , C.C. , Bullard , B. , Gonzalez-Escobedo , G. , Gunn , J.S. , Altier , C. , and Payne , S.M. ( 2016 ) Bile acids function synergistically to repress invasion gene expression in Salmonella by destabilizing the invasion regulator HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29349886" target="_blank">29349886</a>]<br></label>
<input id="act_similarity_105", type="checkbox">
<div class="ACT_similar">
<li>Eade CR , Hung CC , Bullard B , Gonzalez-Escobedo G , Gunn JS , Altier C. Bile acids function synergistically to repress invasion gene expression in Salmonella by destabilizing the invasion regulator HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28575106" target="_blank">28575106</a>]<br></li></div>
<li>FIG 2 HilD antagonizes the H-NS-mediated repression exerted on ssrAB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25135218" target="_blank">25135218</a>]<br></li>
<li>We have evidence that HilD modestly activates invF expression independently of its effects on hilA , even in WT S. typhimurium grown under inducing conditions in vitro ( ﬁgure 1B , orange arrow ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11755416" target="_blank">11755416</a>]<br></li>
<li>These results showed that , whereas OmpR-mediated regulation of hilA expression was dampened in a DhilC DrtsA background , regulation was blocked only in strains missing HilD , suggesting that OmpR controls SPI1 expression by regulating hilD expression [ 19 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17208038" target="_blank">17208038</a>]<br></li>
<li>Boddicker et al. found that mutants 46 lacking several of the negative-acting proteins , when grown in inducing conditions ( high osmolarity , low oxygen level ) , show elevated hilA expression , but that they still required HilD function for expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16443238" target="_blank">16443238</a>]<br></li>
<li>To build a construct in which HilD or HilC acts as a simple transcriptional-repressor , we synthesized four pairs of complementary oligonucleotides .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29378886" target="_blank">29378886</a>]<br></li>
<li>Since Fur activation of hilA is clearly dependent upon HilD , we constructed hilD-lacZ fusions in a hilD background .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
<li>Changes in temperature likely affect hilA independently of HilD , with H-NS implicated in this regulation ( Ono et al. 2005 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22021388" target="_blank">22021388</a>]<br></li>
<li>Given that expression of hilA , a gene directly regulated by HilD , signiﬁcantly increases in bile , we investigated if HilD is posttranscriptionally regulated by bile in S. Typhi .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29229736" target="_blank">29229736</a>]<br></li>
<li>If FliZ affects hilA expression solely through HilD , then there should be no ﬂiZ phenotype in a hilD background when the system is activated by other means .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20889744" target="_blank">20889744</a>]<br></li>
<li>Of the three , HilD is dominant , as there is no HilA expression in its absence [ 33 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20686667" target="_blank">20686667</a>]<br></li>
<li>However , even though the PhilC promoter is active in the absence of HilD , HilA is not expressed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20686667" target="_blank">20686667</a>]<br></li>
<li>Of the three , HilD is the most important as HilA is not expressed in its absence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20686667" target="_blank">20686667</a>]<br></li>
<li>However , it is unknown if these changes in stability are responsible for driving changes in SPI‐1 expression in bile , or rather reflects an indirect effect on HilD stability , for example , via alteration of DNA binding ( as DNA‐bound HilD is proposed to be more stable ; Grenz , Cott Chubiz , Thaprawat , & Slauch , 2018 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=30030897" target="_blank">30030897</a>]<br></li>
<li>HilD , a transcriptional-regulator of SPI1 expression , is de-stabilized in cells in which dnaKJ is disrupted .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18723621" target="_blank">18723621</a>]<br></li>
<li>However , we did not observe transcriptional decrease of SPI-2 genes in Δpat by RNA-seq ; one possible reason is that HilD is not required for SPI-2 expression under the in vitro growth conditions [ 41 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26810370" target="_blank">26810370</a>]<br></li>
<li>If a condition exists in which hilA is repressed while invF expression is induced through HilC or HilD , it begs the question : Why produce InvF-dependent effectors in the absence of hilA expression when an intact type III secretion apparatus is not thought to be formed ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10844688" target="_blank">10844688</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_123">
Interestingly , HilD is required for the expression of ssrAB when Salmonella is grown to late stationary phase in LB medium but not when it is grown in minimal media ( 21 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25135218" target="_blank">25135218</a>]<br></label>
<input id="act_similarity_123", type="checkbox">
<div class="ACT_similar">
<li>Interestingly , HilD is required for the expression of ssrAB when Salmonella is grown in LB medium but not when it is grown in minimal media , such as N-MM ( Fig. 3 ) ( 21 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25135218" target="_blank">25135218</a>]<br></li></div>
<li>The invR gene , which we identiﬁed in a global search for new Salmo-nella sRNA genes , is activated by the major SPI-1 transcription factor , HilD , under conditions that favour host cell invasion .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17971080" target="_blank">17971080</a>]<br></li>
<li>Recent work has demonstrated that HilD activates ﬂhDC transcription late in the cell 's growth-phase under non-Spi1 induction conditions ( 50 , 51 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24706743" target="_blank">24706743</a>]<br></li>
<li>In this report , we provide evidence that HilD regulation also occurs during transcription elongation in the 3 ' - UTR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28426789" target="_blank">28426789</a>]<br></li>
<li>g007 In formulating the model , we needed to assume that HilD was dominant in the sense that it was required for activating HilA expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20686667" target="_blank">20686667</a>]<br></li>
<li>The regulation of HilD by BarA/SirA two-component system is believed to be regulated by csrA/B , although it is not known whether the regulation is direct or indirect [ Ellermeier et al. , 2005 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19537165" target="_blank">19537165</a>]<br></li>
<li>Our inquiry into the mechanism by which bile acids inhibit SPI-1 demonstrated that repression is achieved primarily by de-stabilizing the HilD protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27185788" target="_blank">27185788</a>]<br></li>
<li>InvR sRNA of the Salmonella pathogenicity island , SPI-1 , becomes expressed under conditions that favour host cell invasion ; the invR gene is directly controlled by the major virulence gene activator , HilD ( Pfeiffer et al. , 2007 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18399940" target="_blank">18399940</a>]<br></li>
<li>Furthermore , synthesis of the SPI-1 transcription factor HilD is negatively regulated by StdEF , two proteins encoded by the std ﬁmbrial operon [ 71 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29369799" target="_blank">29369799</a>]<br></li>
<li>Although posttranscriptional regulation is dominant for HilD , the cellular concentration of HilD especially is controlled by the Lon protease [ 10 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28329249" target="_blank">28329249</a>]<br></li>
<li>Moreover , the cellular concentration of HilD is controlled by the Lon protease ( Takaya et al. , 2005 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26300871" target="_blank">26300871</a>]<br></li>
<li>It is likely that the effect of HilE is exerted by direct interaction with HilD , thus blocking its activity ( Baxter et al. , 2003 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21680637" target="_blank">21680637</a>]<br></li>
<li>It is possible that the long HilDdependent extension product is not speciﬁc for invF and was primed from another transcript induced by HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>It is unknown whether this mechanism is mediated by direct interaction between HilD and bile acids or if instead the destabilization of HilD is mediated by regulatory intermediaries .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27185788" target="_blank">27185788</a>]<br></li>
<li>Taking these results together , it is suggested that SPI1 expression is regulated by ClpXP through a mechanism by which FliZ controls the cellular levels of the transcriptional-regulators HilC and HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18245288" target="_blank">18245288</a>]<br></li>
<li>The system is complicated by the fact that HilD is autoregulated and hilD transcription is normally induced upon activation of the regulatory circuit ( 13 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20889744" target="_blank">20889744</a>]<br></li>
<li>Cumulatively , the data indicate that invR is a novel member of the SPI1 regulon , and that its promoter is activated directly by HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17971080" target="_blank">17971080</a>]<br></li>
<li>Thus , the system is switched on when HilD activity reaches some threshold sufﬁcient to efﬁciently activate its own expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29378886" target="_blank">29378886</a>]<br></li>
<li>HilD is an AraC/XylS-type transcriptional-regulator that binds to the hilA promoter and is essential for its activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15039320" target="_blank">15039320</a>]<br></li>
<li>SPI-1 repression by L-arabinose is exerted at a single target , HilD , and the mechanism appears to be post-translational .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25991823" target="_blank">25991823</a>]<br></li>
<li>HilA is negatively regulated by HilE which binds to HilD to inhibit the transcription of hilA17 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27886215" target="_blank">27886215</a>]<br></li>
<li>Likewise , SprB , a transcription factor from the LuxR/UhaP family that is positively regulated by HilA , appears to-bind to the HilD promoter and weakly repress its activity [ 65 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20686667" target="_blank">20686667</a>]<br></li>
<li>Because HilD is an AraC-like transcription factor , it is tempting to propose that L-arabinose might regulate HilD activity by directly binding to the protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25991823" target="_blank">25991823</a>]<br></li>
<li>HilD activity is controlled by HilE ; this protein binds HilD and is thought to prevent it from activating the PhilA promoter ( 6 , 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19218381" target="_blank">19218381</a>]<br></li>
<li>HilD can activate the ssrAB operon by speciﬁcally interacting with its regulatory region to upregulate SPI-2 [ 41 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26810370" target="_blank">26810370</a>]<br></li>
<li>Our results indicate that HilE functions not by binding to the hilA promoter but by binding to HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12595445" target="_blank">12595445</a>]<br></li>
<li>Again , this was speciﬁc to HilD , since the downstream acting HilA factor could not restore DapZ levels in these mutant backgrounds .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22922465" target="_blank">22922465</a>]<br></li>
<li>Puriﬁed HilD protein binds to the invR promoter region with high affinity in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17971080" target="_blank">17971080</a>]<br></li>
<li>In turn , the deacetylated K297 enhances the DNA-binding ability but decreases HilD stability .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28329249" target="_blank">28329249</a>]<br></li>
<li>This boost in virulence was suggested to be caused by increased flagellar expression , including the FliZ regulator controlling the SPI1 regulator HilD at post-transcriptional level ( Kage et al. , 2008 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23676436" target="_blank">23676436</a>]<br></li>
<li>Likewise , HilD binds the ﬂhDC promoter to activate the ﬂagellar regulon ( 47 , 48 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29866910" target="_blank">29866910</a>]<br></li>
<li>At this stage , the FliZ ﬂagellar activator has been reported to simultaneously repress the ﬁmbrial genes but activate SPI-1 through HilD ( 15 , 287 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23554419" target="_blank">23554419</a>]<br></li>
<li>SPI1 expression is activated when HilD reaches the threshold required to autoactivate the hilD promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22021388" target="_blank">22021388</a>]<br></li>
<li>Although all three regulators contribute to the induction , the effect is mediated solely through HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16045614" target="_blank">16045614</a>]<br></li>
<li>It is tempting to speculate that the regulation by HilD recruited the PhoH activity as a factor that can contribute to the Salmonella pathogenesis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27886269" target="_blank">27886269</a>]<br></li>
<li>If repression by a particular condition or mutation is mediated by increasing the expression or activity of the repressor , this effect could be counteracted by ﬂooding the system with so much HilD that the repressor is unable to-bind to the URS at all .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11292791" target="_blank">11292791</a>]<br></li>
<li>IHF activation is deduced to be mediated through HilD as for Fis and HU according to the information provided in the text .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23554419" target="_blank">23554419</a>]<br></li>
<li>This threshold is set by HilE , which binds to HilD to prevent the inev-itable minor ﬂuctuations in HilD activity from inappropriately activating the system .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29378886" target="_blank">29378886</a>]<br></li>
<li>These results lend credence to our hypothesis regarding HilD dominance used in formulating the model , namely that HilD is necessary for activating the SPI1 promoter under physiological conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20686667" target="_blank">20686667</a>]<br></li>
<li>Fur regulation is mediated through HilD , but regulation does not appear to be simply at the level of hilD transcription or translation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
<li>Although a molecular interaction remains to be proven , this trans-complementation strongly argues that HilD activates the dapZ promoter directly .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22922465" target="_blank">22922465</a>]<br></li>
<li>HilD could also act directly on the rﬂM promoter region to repress rﬂM transcription or activate transcription of another protein that represses rﬂM .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24488311" target="_blank">24488311</a>]<br></li>
<li>Overall , our data suggest that the role of HilE is to simply bind to HilD , disrupting its ability to-bind to DNA , thereby putting HilD into an inactive or low-activity state .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29378886" target="_blank">29378886</a>]<br></li>
<li>Together , these data suggest that HilE functions to restrict low-level HilD activity , preventing premature activation of SPI1 until positive inputs reach a threshold required to fully induce the system .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29378886" target="_blank">29378886</a>]<br></li>
<li>This finding suggests that HilE functions as a repressor by binding to HilD to prevent its activation of the hilA promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15765064" target="_blank">15765064</a>]<br></li>
<li>Together , our data demonstrate that HilD effectively displaces H-NS bound to those sites that mediate the repression of ssrAB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25135218" target="_blank">25135218</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_170">
Although the exact mechanism of HilE action is not clear , data from bacterial two-hybrid studies suggest that HilE binds directly to HilD preventing its action [ 49 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17208038" target="_blank">17208038</a>]<br></label>
<input id="act_similarity_170", type="checkbox">
<div class="ACT_similar">
<li>Although the mechanism of hilE action is not well understood , the results from bacterial two-hybrid studies suggest that HilE binds directly to HilD , preventing its action ( 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18245288" target="_blank">18245288</a>]<br></li></div>
<li>Under low O2 concentrations , the phosphorylated ArcA after sensing of low O2 signal by ArcB can indirectly activate loiA gene transcription , LoiA can then activate SPI-1 T3SS through directly activating HilD , thus promoting epithelial cell invasion .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28575106" target="_blank">28575106</a>]<br></li>
<li>In all 8 cases , transcription activation required both the inducible HilD plasmid and the inducer , arabinose .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24375101" target="_blank">24375101</a>]<br></li>
<li>The data presented thus far are consistent with a simple linear pathway in which BarA senses the salt/sucrose stimulus and transduces the information to SirA , which informs HilD , which activates hilA transcription , which permits HilA-dependent activation of SPI1 genes , including the T3SS and its Sips .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18441068" target="_blank">18441068</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18441068" target="_blank">18441068</a>]<br></li>
<li>The ﬂiZ gene is transcribed in the ﬂiAZ operon , and FliZ activates the Spi1 transcriptional-activator HilD , which in turn activates Spi1 transcription under high salt ( 200 mM NaCl ) microaerobic-growth conditions ( 48 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24706743" target="_blank">24706743</a>]<br></li>
<li>In results comparable to those for the LT2 strain , ﬂhC transcription upon HilD overexpression also increased in the virulent Salmonella background ST14028s grown under Spi1-in-ducing conditions ( Fig. 1B , bars 5 and 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24488311" target="_blank">24488311</a>]<br></li>
<li>Therefore , we examined whether the HilC and HilD accumulate in the Lon-depleted cells grown under these conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15661008" target="_blank">15661008</a>]<br></li>
<li>However , epistasis analysis implicated HilD and HilA , but not HilC , in the direct pathway from the salt stimulus to the Sip secretion response .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18441068" target="_blank">18441068</a>]<br></li>
<li>Lon protease is known to negatively control HilD ( Chubiz et al. , 2010 , Boddicker & Jones , 2004 ) , and we observed an expected increase in HilD in the lon null mutant , particularly at later time points ( Fig. 9A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23289537" target="_blank">23289537</a>]<br></li>
<li>In contrast , Boddicker et al. ( 5 ) found that the HilD protein was required to activate hilA-lacZ transcription even when several negative regulators , Hha , Ams , HilE , and/or Pag , were deleted in combination .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15126488" target="_blank">15126488</a>]<br></li>
<li>We reasoned that mutations in negative regulators that may bind to the URS should reduce the need for HilD and allow higher hilA expression when HilD is absent .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12511499" target="_blank">12511499</a>]<br></li>
<li>We propose that HilE binds independently to the individual HilD proteins in both homodimers and heterodimers containing HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29378886" target="_blank">29378886</a>]<br></li>
<li>When induced with-tetracycline , there was marked a decrease in - galactosidase activity in both the HilD - and HilC-producing strains ( Fig. 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29378886" target="_blank">29378886</a>]<br></li>
<li>The dashed line between class 2 product FliZ and HilD represents posttranscriptional activation , as described in the text .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24488311" target="_blank">24488311</a>]<br></li>
<li>Subsequent experiments demonstrated that the ﬂagellumrelated gene product FliZ controls HilD posttranscriptionally , and this in turn activates HilC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18245288" target="_blank">18245288</a>]<br></li>
<li>HilD and HilC do not activate invF expression from the HilA-dependent invF promoter Previously , we had shown that HilA activates invF expression by direct interaction with sequences upstream of invF called the HilA box ( Lostroh et al. , 2000 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>These results suggest that HilD and HilC might directly activate invF expression by binding to sequences upstream of invF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>Our studies also suggest that the downstream binding site is required for HilD and HilC to activate invF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>These results indicate that HilD and HilC require the downstream binding site to activate invF expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>Briefly , ArcB responds to low oxygen conditions when bacteria reach the distal ileum and undergoes autophosphorylation , following which the phosphate group is transferred to ArcA ; phosphorylated ArcA ( ArcA-P ) activates loiA gene expression indirectly through unknown regulator ( s ) ; LoiA activates HilD directly through binding the hilD promoter ; HilD then activates hilA and other SPI-1 genes , and thus facilitates the invasion process .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28575106" target="_blank">28575106</a>]<br></li>
<li>For example , OmpR promotes SPI-2 expression by directly binding to both the ssrA and ssrB promoters [ 16 ] ; PhoP activates SPI-2 transcription by directly binding to the ssrB promoter and controls SsrA levels post-transcriptionally [ 15 ] ; SlyA , HilD and Fis proteins can bind to the ssrA promoter to regulate SPI-2 genes [ 19,20,22 ] , and PmrA represses SPI-2 gene transcription through directly binding to the ssrB promoter [ 18 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29751061" target="_blank">29751061</a>]<br></li>
<li>HilD also directly controls the expression of the SPI-1 genes hilD , hilC , and invF , as well as other acquired and ancestral genes located outside SPI-1 , such as rtsA , ﬂhDC , siiA , lpxR , ytfK , , and ( 2 , 25 -- 31 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25135218" target="_blank">25135218</a>]<br></li>
<li>Using mutation and overexpression experiments , real-time PCR analysis , and electrophoretic mobility shift assays , we demonstrated that Fis regulates 63 of the 94 Salmonella pathogenicity island ( SPI ) -1 and SPI-2 genes , by three regulatory modes : i ) binds to SPI regulators in the gene body or in upstream regions ; ii ) binds to SPI genes directly to mediate transcriptional activation of themselves and downstream genes ; iii ) binds to gene encoding OmpR which affects SPI gene expression by controlling SPI regulators SsrA and HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23717649" target="_blank">23717649</a>]<br></li>
<li>In this work , by applying a clustering method to S. Typhimurium SL1344 global expression data from the COLOMBOS database , we show that most of the known genes regulated by HilD , including the flagellar/chemot-axis genes , are indeed co-expressed with SPI-1 ; moreover , nine novel genes that are co-expressed with SPI-1 were identified : gtgE , phoH , sinR , SL1028 , lpxR , SL1896 , SL3812 , SL4247 and SL4433 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27886269" target="_blank">27886269</a>]<br></li>
<li>Another SPI-1 encoded sRNA , InvR was found to be activated by virulence regulatory protein HilD under invasion inducing conditions and repressed the synthesis of a major Salmonella porin OmpD post-transcriptionally [ 21,22 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28302471" target="_blank">28302471</a>]<br></li>
<li>LB LB 0.3 M NaCl L The invR gene belongs to the SPI-1 regulon and is activated by HilD Next , we asked whether invR transcription was under the control of SPI-1-encoded transcription factors .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17971080" target="_blank">17971080</a>]<br></li>
<li>Thus , although HilC and HilD proteins can bind to the same DNA target , they have different sequence requirements for DNA recognition .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12107132" target="_blank">12107132</a>]<br></li>
<li>HilC can bind to fragments carrying only the upstream A1 or only the downstream A2 site , but HilD bound well only to the A1 region and not to A2 alone .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12107132" target="_blank">12107132</a>]<br></li>
<li>According to this model , HilC and RtsA were acquired to compensate for the inability of HilD alone to mediate a robust response .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20686667" target="_blank">20686667</a>]<br></li>
<li>To see if these sequences are important for HilD and HilC to activate invF , we constructed derivatives of pSA7 in which the upstream TTTGAT was changed to TTTGAG and the downstream TATTCT was changed to TATGCG .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>These results contrast with the ability of HilD and HilC to activate a chromosomal invF fusion 80 - to 170-fold , suggesting that HilD and HilC require different sequences to activate invF and do not use the same promoter as HilA does to activate invF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>HilD and HilC bind to the invF promoter fragment in vitro To examine whether HilD and HilC might directly activate the invF promoter by binding to sequences upstream of invF , we performed gel shift analyses .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>In this model , the repressor is always present but can not inhibit hilA expression under activating conditions because HilD ( and , to a lesser extent , HilC ) displaces it from the URS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11292791" target="_blank">11292791</a>]<br></li>
<li>RtsA , HilC , and HilD each activate hilA by binding to the same sites in the promoter region ( 13 , 35 , 36 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20889744" target="_blank">20889744</a>]<br></li>
<li>Similarly , HilC and HilD bound in different manners to the hilA promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12107132" target="_blank">12107132</a>]<br></li>
<li>By binding to the URS element , HilD and HilC may displace or counteract the repressors , thereby derepressing hilA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>Two AraC-like transcriptional-regulators encoded on Salmonella pathogenicity island 1 ( SPI1 ) , HilC and HilD , bind to the URS to counteract the repression of hilA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11292791" target="_blank">11292791</a>]<br></li>
<li>Under activating conditions , HilC and HilD bind to region 2332 to 239 upstream of the hilA promoter , displacing the repressor from this site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11292791" target="_blank">11292791</a>]<br></li>
<li>In the hilA mutant , which expressed HilD , exposure to salt increased hilA , hilD , and sipC transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18441068" target="_blank">18441068</a>]<br></li>
<li>HU or Fis may bind t the hilA promoter and enhance its ability to be derepressed by HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12933899" target="_blank">12933899</a>]<br></li>
<li>HilE and Hha may block this activation by binding to HilD and the hilA promoter , respectively , under extracellular repressive conditions for invasion ( 8 , 26 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15039320" target="_blank">15039320</a>]<br></li>
<li>In this study , we provide further genetic and biochemical data showing that HilE directly binds to HilD and blocks its ability to-bind DNA but does not affect the dimerization or stability of HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29378886" target="_blank">29378886</a>]<br></li>
<li>In this model the regulator either binds to HilD and prevents its function or targets HilD for degradation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
<li>HilE was previously identiﬁed as a negative regulator of the SPI1 T3SS , and two-hybrid data suggested that HilE binds directly to HilD ( 5 , 21 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29378886" target="_blank">29378886</a>]<br></li>
<li>It was shown previously that Lon protease degrades HilD ( 40 ) and that the transcriptional-regulator LeuO directly activates transcription of hilE ( 41 ) , whose product inhibits HilD activity ( 42 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25182488" target="_blank">25182488</a>]<br></li>
<li>HilE represses the activity of HilD by binding to it and preventing it from activating its target promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20686667" target="_blank">20686667</a>]<br></li>
<li>However , bacterial two-hybrid experiments demonstrated that HilD and HilE bind to one another , suggesting that a possible mechanism of action for HilE is to disrupt HilD activation of hilA by disrupting its function .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15731035" target="_blank">15731035</a>]<br></li>
<li>For example , HilD may interact with a ligand that is present under speciﬁc conditions , and the binding of this ligand may affect HilD 's ability to-bind DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11292791" target="_blank">11292791</a>]<br></li>
<li>This shows that the primary effect of HilE is to directly regulate HilD DNA binding activity and that any effect on HilD stability is a secondary effect .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29378886" target="_blank">29378886</a>]<br></li>
<li>Together , these data suggest that the general repression of SPI-1 T3SS is unlikely to be transcriptionally regulated ( i.e. , HilA and HilD independent at least ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28193547" target="_blank">28193547</a>]<br></li>
<li>It is likely that Lon may function through its proteolytic activity to regulate the availability of HilC and/or HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11741864" target="_blank">11741864</a>]<br></li>
<li>These results , taken together , thus indicate both that the amount of HilD present within a bacterium is key to the control of invasion by bile and its components and that bile itself functions to regulate HilD by both lon-dependent and - independent means .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27185788" target="_blank">27185788</a>]<br></li>
<li>Moreover , it is well known that occurring truncated form of the H-NS protein that acts as an HilD , HilC and RtsA are required for proper hilA induction H-NS antagonist ( Williamson & Free , 2005 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21680637" target="_blank">21680637</a>]<br></li>
<li>Our studies show that HilD and HilC activate transcription of invF from a promoter that is far upstream of its HilA-dependent promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>HilD and HilC appear to activate transcription of invF from a HilD/C-dependent start site that is 631 nucleotides upstream of the invF ORF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>These results suggest that HilE binds and directly blocks HilD function and that degradation of HilD is not a signiﬁcant mechanism of regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29378886" target="_blank">29378886</a>]<br></li>
<li>However , our data show that PhoPQ can repress the system independently of HilD and HilE ( reference 6 and unpublished data ) ; thus , this transcriptional induction of hilE is apparently superﬂuous .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29378886" target="_blank">29378886</a>]<br></li>
<li>LeuO activates hilE transcription A tentative model for LeuO-mediated downregulation of SPI-1 via HilE and HilD is that LeuO may activate hilE expression , and that HilE-mediated inhibition of HilD activity ( Baxter et al. , 2003 ) may contribute to SPI-1 downregulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24354910" target="_blank">24354910</a>]<br></li>
<li>Ellermeier and Slauch initially reported that Fur activates hilA transcription in a HilD-dependent manner , and since Fur is thought to be capable of acting only as a repressor , it should repress a repressor that controls HilD ( 322 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23554419" target="_blank">23554419</a>]<br></li>
<li>The rate of change of HilC concentration depends on its basal synthesis , its activation by HilD and SirA , and its own degradation rate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19537165" target="_blank">19537165</a>]<br></li>
<li>Similarly , the rate of change of RtsA concentration depends on its basal synthesis ( b2 ; which is a constant ) as well as on activation by HilD and its own degradation rate ( a2 ) and can be represented as : d 1/2 RtsA 1/4 b þ b f ð 1/2 HilD Þ a 1/2 RtsA : 2 2 dt 2 • Activation of PhilA can be achieved by AND or OR gate logic .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19003447" target="_blank">19003447</a>]<br></li>
<li>Similarly , the rate of change of RtsA concentration depends on its basal synthesis , its activation by HilD and its own degradation rate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19537165" target="_blank">19537165</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_233">
Singer , H.M. , Ku $ hne , C. , Deditius , J.A. , Hughes , K.T. , and Erhardt , M. ( 2014 ) The Salmonella Spi1 virulence regulatory protein HilD directly activates transcription of the flagellar master operon flhDC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27206164" target="_blank">27206164</a>]<br></label>
<input id="act_similarity_233", type="checkbox">
<div class="ACT_similar">
<li>Singer , H.M. , Kühne , C. , Deditius , J.A. , Hughes , K.T. , and Erhardt , M. ( 2014 ) The Salmonella Spi1 virulence regulatory protein HilD directly activates transcription of the flagellar master operon flhDC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27404739" target="_blank">27404739</a>]<br></li></div>
<li>Such examples include counter-effect of HilD on H-NS mediated silencing of ssrA and ssrB [ 5 ] and counter-effect of IHF on H-NS mediated repression of hilA [ 45 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24079299" target="_blank">24079299</a>]<br></li>
<li>The Salmonella Spi1 Virulence Regulatory Protein HilD Directly Activates Transcription of the Flagellar Master Operon ﬂhDC Hanna M. Singer , a Caroline Kühne , b Julia A. Deditius , b Kelly T. Hughes , c Marc Erhardta , b Microbiologie , Département de Médecine , Université de Fribourg , Fribourg , Switzerlanda ; Helmholtz Centre for Infection Research , Braunschweig , Germanyb ; Department of Biology , University of Utah , Salt Lake City , Utah , USAc Infection of intestinal epithelial cells is dependent on the Salmonella enterica serovar Typhimurium pathogenicity island 1 ( Spi1 ) - encoded type III injectisome system and ﬂagellar motility .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24488311" target="_blank">24488311</a>]<br></li>
<li>crossmark Bile Acids Function Synergistically To Repress Invasion Gene Expression in Salmonella by Destabilizing the Invasion Regulator HilD Colleen R. Eade , a Chien-Che Hung , a Brian Bullard , b Geoffrey Gonzalez-Escobedo , b John S. Gunn , b Craig Altiera Department of Population Medicine and Diagnostic Sciences , College of Veterinary Medicine , Cornell University , Ithaca , New York , USAa ; Center for Microbial Interface Biology , Department of Microbial Infection and Immunity , The Ohio State University , Columbus , Ohio , USAb Salmonella spp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27185788" target="_blank">27185788</a>]<br></li>
<li>The Intestinal Fatty Acid Propionate Inhibits Salmonella Invasion through the Post-translational Control of HilD Chien-Che Hung1 , Cherilyn D. Garner1 , James M. Slauch2 ,3 , Zachary W. Dwyer1 , Sara D. Lawhon4 , Jonathan G. Frye5 , Michael McClelland6 , Brian M. M. Ahmer7 ,8 , and Craig Altier1 , * 1Department of Population Medicine and Diagnostic Sciences , College of Veterinary Medicine , Cornell University , Ithaca , NY 14853 , USA 2Department of Microbiology , University of Illinois , Urbana , IL 61801 , USA 3College of Medicine , University of Illinois , Urbana , IL 61801 , USA 4Department of Veterinary Pathobiology , College of Veterinary Medicine and Biomedical Sciences , Texas A&M University , College Station , TX 77843 , USA 5Bacterial Epidemiology and Antimicrobial Resistance Research Unit , U.S. Department of Agriculture , Agriculture Research Service , Richard B. Russell Research Center , Athens , GA 30605 , USA 6Department of Pathology and Laboratory Medicine , University of California , Irvine , Irvine , CA 92617 , USA 7Department of Microbiology , The Ohio State University , Columbus , OH 43210 , USA 8Center for Microbial Interface Biology , The Ohio State University , Columbus , OH 43210 , USA SUMMARY To cause disease , Salmonella must invade the intestinal epithelium employing genes encoded within Salmonella Pathogenicity Island 1 ( SPI1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23289537" target="_blank">23289537</a>]<br></li>
<li>T ranscription of the Salmonella Invasion Gene Activator , hilA Requires HilD Activation in the Absence of Negative Regulators Jennifer D. Boddicker ,1 Boyd M. Knosp ,2 and Bradley D. Jones1 * Department of Microbiology , University of Iowa School of Medicine ,1 and ITS Research Technologies , University of Iowa ,2 Iowa City , Iowa 52242 , Received 25 July 2002/Accepted 24 September 2002 Salmonella enterica serovar Typhimurium causes human gastroenteritis and a systemic typhoid-like infection in mice .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12511499" target="_blank">12511499</a>]<br></li>
<li>HilE Interacts with HilD and Negatively Regulates hilA Transcrip and Expression of the Salmonella enterica Serovar Typhimurium Invasive Phenotype M. Aaron Baxter ,1 Thomas F. Fahlen ,2 Rebecca L. Wilson ,2 and Bradley D. Jones1 ,2 * Genetics Program1 and Department of Microbiology ,2 University of Iowa School of Medicine , Iowa City , Iowa 52242 Received 1 August 2002/Accepted 21 November 2002 tion The ability of Salmonella enterica serovar Typhimurium to traverse the intestinal mucosa of a host is an important step in its ability to initiate gastrointestinal disease .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12595445" target="_blank">12595445</a>]<br></li>
<li>Role of Nucleoid-Associated Proteins Hha and H-NS in Expression of Salmonella enterica Activators HilD , HilC , and RtsA Required for Cell Invasion Igor N. Olekhnovich * and Robert J. Kadner † icrobiology , University of Virginia School of Medicine , Charlottesville , Virginia 22908-0734 Received 8 June 2007/Accepted 23 July 2007 ent of M Departm The coordinate expression of Salmonella enterica invasion genes on Salmonella pathogenicity island 1 is under the control of the complex circuits of regulation that involve the AraC/XylS family transcriptional-activators HilD , HilC , and RtsA and nucleoid-associated proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17675384" target="_blank">17675384</a>]<br></li>
<li>The Journal of Infectious Diseases M A J O R A R T I C L E Acetylation Regulating Protein Stability and DNA-Binding Ability of HilD , thus Modulating Salmonella Typhimurium Virulence Yu Sang ,1 a Jie Ren ,1 a Ran Qin ,4 Shuting Liu ,1 Zhongli Cui ,4 Sen Cheng ,5 Xiaoyun Liu ,5 Jie Lu ,2 Jing Tao ,1 , b and Yu-Feng Yao1 ,3 , b 1Laboratory of Bacterial Pathogenesis , Department of Microbiology and Immunology , Institutes of Medical Sciences and 2Department of Infectious Diseases , Ruijin Hospital , Shanghai Jiao Tong University School of Medicine , and 3Department of Laboratory Medicine , Shanghai East Hospital , Tongji University School of Medicine , 4Key Laboratory of Agricultural Environmental Microbiology , Ministry of Agriculture , College of Life Sciences , Nanjing Agricultural University , and 5Institute of Analytical Chemistry and Synthetic and Functional Biomolecules Center , College of Chemistry and Molecular Engineering , Peking University , Beijing , China HilD , a dominant regulator of Salmonella pathogenicity island 1 , can be acetylated by protein acetyltransferase ( Pat ) in Salmonella Typhimurium , and the acetylation is beneficial to its stability .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28329249" target="_blank">28329249</a>]<br></li>
<li>HilD mediates transcriptional cross talk between SPI-1 and SPI-2 , providing sequential activation of the loci .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20028809" target="_blank">20028809</a>]<br></li>
<li>HilD inhibition causes SPI-1 repression , which results in impaired secretion of SPI-1 effectors and reduced invasion of epithelial cells .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25991823" target="_blank">25991823</a>]<br></li>
<li>As SprB regulates multiple SPI1 promoters , we hypothesized that it likely represses the transcription of HilC , HilD , or RtsA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20190046" target="_blank">20190046</a>]<br></li>
<li>The immunoblotting analyses show that the increased clpPX disruption effect on cellular levels of HilC and HilD disappeared when either hilD or hilC was introduced ( Fig. 4B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18245288" target="_blank">18245288</a>]<br></li>
<li>At this point , HilD is able to autoactivate its own promoter , as well as those of HilC and RtsA , leading to robust and rapid induction of the SPI1 T3SS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29378886" target="_blank">29378886</a>]<br></li>
<li>This activation is most probably through direct binding of HilD and HilC to sequences upstream and downstream of this alternative HilA-independent promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>( D ) Association of ChIP-seq peaks with regulation for HilD , HilC , and RtsA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27601571" target="_blank">27601571</a>]<br></li>
<li>The following procedure was applied for each of HilD , HilC , RtsA , InvF , SprB , and RtsB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27601571" target="_blank">27601571</a>]<br></li>
<li>HilD , HilC and RtsA are thought to form a triangular network system of regulators positively regulating each other , while HilE can regulate this system by acting as a negative regulator of HilD ( Fig. 1 , ⊥ arrow ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21107576" target="_blank">21107576</a>]<br></li>
<li>In the case of hilD , post-translational repression by HilE dampens hilD activation , and H-NS repression of hilA counteracts transcriptional activation by HilD , HilC and RtsA ( 66 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28335027" target="_blank">28335027</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_253">
Model 4 : Two feedforward loops with OR gate model of regulation of PhilA by heterodimers of HilC-HilD , HilC-RtsA and HilD-RtsA In this case functions s3 , s4 and s are m : ð 1/2 HilD 1/2 HilC ÞH3 kH3 þ ð 1/2 HilD 1/2 HilC ÞH3 3 odiﬁed as follows : s 3 s 4 Model 2 : Two feedforward loops with OR gate logic for regulation of P by HilD , HilC and RtsA ( monomer hilA activation ) ðModel 4Þ s 5 In this case , the rate of change of HilA concentration is a additive function of the three input transactivators via the functions s3 , s4 and s5 corresponding to HilD , HilC and RtsA respectively and is represented as : d 1/2 HilA 1/4 1/2 þ 1/2 þ þ b3 b3 s3 s4 s5 s dt Model 3 : Two feedforward loops with OR gate model for regulation of PhilA by monomers of HilD , HilC and RtsA and addition of positive feedback on RtsA and HilC as well as cross activations of RtsA on HilC and vice versa where functions s7 and s9 represent the auto-activation of HilC and RtsA respectively and functions s8 and s10 represent the cross activation of HilC by RtsA and vice versa respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19003447" target="_blank">19003447</a>]<br></label>
<input id="act_similarity_253", type="checkbox">
<div class="ACT_similar">
<li>Model 5 : Two feedforward loops with OR gate model of regulation of PhilA by heterodimers of HilC-HilD , HilC-RtsA and HilD-RtsA and additional positive feedback on RtsA and HilC as well as cross activations of RtsA on HilC and vice versa This is of the same form as Model 3 , except that the functions s3 , s4 and s5 are of the type considered in Model 4 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19003447" target="_blank">19003447</a>]<br></li></div>
<li>Model 4 : Two feedforward loops with OR gate model of regulation of PhilA by heterodimers of HilC-HilD , HilC-RtsA and HilD-RtsA In this case functions s3 , s4 and s are m : ð 1/2 HilD 1/2 HilC ÞH3 kH3 þ ð 1/2 HilD 1/2 HilC ÞH3 3 odiﬁed as follows : s 3 This corresponds to activation via the heterodimeric complex of HilC and HilD with co-operativity H3 and threshold of activation K3 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19003447" target="_blank">19003447</a>]<br></li>
<li>1/4 : ð 1/2 HilD 1/2 RtsA ÞH5 kH5 þ ð 1/2 HilD 1/2 RtsA ÞH5 5 Model 2 : Two feedforward loops with OR gate logic for regulation of P by HilD , HilC and RtsA ( monomer hilA activation ) ðModel 4Þ s 5 This corresponds to activation via the heterodimeric complex of HilD and RtsA with cooperativity H5 and threshold of activation K5 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19003447" target="_blank">19003447</a>]<br></li>
<li>We also show that HilD-activated genes are frequently activated by HilC and RtsA , indicating extensive overlap of the HilD , HilC , and RtsA regulons .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24375101" target="_blank">24375101</a>]<br></li>
<li>FIG 3 - Galactosidase assay showing activation of HilD target genes by HilC and RtsA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24375101" target="_blank">24375101</a>]<br></li>
<li>HilE and FliZ regulate the activity of HilD simultaneously and independently ( 20 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29378886" target="_blank">29378886</a>]<br></li>
<li>Therefore , acetylation regulates both protein stability and DNA-binding ability to maintain the activity and appropriate amount of HilD in Salmonella Typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28329249" target="_blank">28329249</a>]<br></li>
<li>In this work , we combine ChIP-seq and targeted reporter gene assays to map the regulon of HilD in S. Typhimurium and identify regulation upon HilD binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24375101" target="_blank">24375101</a>]<br></li>
<li>The activation of the ﬂagellar system via HilD and simultaneous inactivation via RtsB argue for a dual or even multiprocess interplay between ﬂagellar and Spi1 regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24488311" target="_blank">24488311</a>]<br></li>
<li>We also needed to assume that HilD was necessary for establishing connectivity within the network , where it was again required for HilC - and RtsA-dependent activation of the P , hilC P , and P promoters ( see Materials and Methods for further hilD rtsA details ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20686667" target="_blank">20686667</a>]<br></li>
<li>Transcription from the hilA promoter is in turn mainly regulated by three transcription factors ; HilD , HilC and RtsA , which in a complex arrangement of feedback and feedforward loops bring about maximal induction of HilA ( Altier 2005 ; Jones 2005 ; Ellermeier and Slauch 2007 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19003447" target="_blank">19003447</a>]<br></li>
<li>Fur regulation of HilD is independent of HilE and is also unaffected by RfrA and RfrB , the E. coli RyhB ortholog and paralog found in Salmonella .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
<li>Transcription of hilA is subject to complex control and is stimulated by the SPI-1-encoded HilC and HilD proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15126488" target="_blank">15126488</a>]<br></li>
<li>In serovar Typhimurium , dsbA is controlled by the hilA regulators RtsA , HilD , and HilC , and expression of DsbA under SPI1-inducing conditions is independent of CpxRA ( 18 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17951383" target="_blank">17951383</a>]<br></li>
<li>HilA transcriptional expression is autoregulated and tightly modulated by the combined action of three AraC-like transcriptional-activators : HilC , HilD and RtsA [ 13,14 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28426789" target="_blank">28426789</a>]<br></li>
<li>First , HilC and RtsA amplify HilA and HilD expression , in the sense that HilA and HilD expression is reduced the absence of HilC and RtsA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20686667" target="_blank">20686667</a>]<br></li>
<li>As HilC and HilD activate the transcription from both HilA-dependent and - independent promoters , it is possible that Lon is involved in regulation of SPI1 gene expression by a pathway that depends on HilC and/or HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15661008" target="_blank">15661008</a>]<br></li>
<li>We previously showed that the SPI1 regulators RtsA , HilC , and HilD activate dsbA from a novel promoter in serovar Typhimurium and that DsbA is required for SPI1 secretion of effector-proteins ( 18 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17951383" target="_blank">17951383</a>]<br></li>
<li>Effect of invF , hilA , and hilC-D mutations on RtsA induction of slrP ( A ) and ability of HilC , HilD , and RtsA to induce expression of hilA ( B ) and slrP ( C ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923082" target="_blank">12923082</a>]<br></li>
<li>Effects of 32 overexpression on cellular levels of HilA and HilD proteins and hilA and hilD transcripts .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18723621" target="_blank">18723621</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_274">
We demonstrate that HilC , HilD and RtsA are each capable of independently inducing expression of the hilC , hilD and rtsA genes , and that each can independently activate hilA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16045614" target="_blank">16045614</a>]<br></label>
<input id="act_similarity_274", type="checkbox">
<div class="ACT_similar">
<li>We show that HilD , HilC and RtsA can each independently activate expression of the hilD , hilC , rtsAB and hilA genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16045614" target="_blank">16045614</a>]<br></li></div>
<li>Our current studies demonstrate that HilD and HilC can bind upstream of invF and directly activate its expression in the absence of hilA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>Under inducing conditions , activation of BarA/SirA initiates a regulatory cascade involving HilD , HilC , and HilA , which leads both directly and indirectly to the expression of SPI1 and SPI4 genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18160484" target="_blank">18160484</a>]<br></li>
<li>We show that HilD and HilC can directly bind and activate invF , which leads to the non-co-ordinate expression of a subset of SPI1 genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>Three homologous proteins , RtsA , HilC and HilD , which belong to the AraC/XylS family of transcriptional-regulators , bind to the DNA immediately upstream of hilA leading to induction of hilA expression ( Schechter and Lee , 2001 ; Olekhnovich and Kadner , 2002 ; Ellermeier and Slauch , 2003 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16045614" target="_blank">16045614</a>]<br></li>
<li>Based on our genetic analyses , and taking into account the published results outlined above , we present a model in which expression of hilA is controlled by the combined action of HilC , HilD and RtsA , each of which can independently bind upstream of hilA to induce its expression ( Schechter and Lee , 2001 ; Olekhnovich and Kadner , 2002 ; Ellermeier and Slauch , 2003 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16045614" target="_blank">16045614</a>]<br></li>
<li>Thus these regulators form a self-reinforcing feed forward regulatory loop and increased expression or activity of one regulator leads to an increase in expression of the other regulators Results HilC , HilD and RtsA can independently induce expression of hilC , hilD and rtsA We have shown that RtsA ( Ellermeier and Slauch , 2003 ) , like HilC and HilD ( Schechter and Lee , 2001 ; Olekhnovich and Kadner , 2002 ) , induces expression of hilA by binding to the HilA promoter region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16045614" target="_blank">16045614</a>]<br></li>
<li>Three homologous proteins , 8,13,14 HilC , HilD , and RtsA , which belong to the AraC/XylS family of transcription factors can activate expression of hilA and some of its target genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16443238" target="_blank">16443238</a>]<br></li>
<li>We believe that the inductions of RtsA , HilC , and HilD are self-reinforcing events leading to rapid and fully induced expression of hilA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923082" target="_blank">12923082</a>]<br></li>
<li>Three AraC-like regulators , HilD , HilC , and RtsA , control expression of hilA , and thereby induction of the SPI1 system .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22021388" target="_blank">22021388</a>]<br></li>
<li>Thus , activators HilD and HilC are not essential for rtsA expression in the absence of repressors Hha and H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17675384" target="_blank">17675384</a>]<br></li>
<li>But the situation would be entirely different at sites 3 and 4 , in which hilA and the secretion apparatus are not expressed but InvF and certain effectors are produced as a result of direct activation of invF expression by HilC or HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10844688" target="_blank">10844688</a>]<br></li>
<li>Furthermore , HilC and HilD are strikingly similar in primary sequence and bind to the same speciﬁc DNA sequences upstream of PhilA [ 52 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11755416" target="_blank">11755416</a>]<br></li>
<li>In summary , sites A1 and C2 can bind both HilC and HilD in the absence of ﬂanking sequences , whereas sites A2 and D can bind HilC but not HilD , without additional ﬂanking sequences .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12107132" target="_blank">12107132</a>]<br></li>
<li>The regulation of hilA promoter activity is complex , involving the coactivators HilC , HilD and RtsA , as well as other factors which act upstream of these proteins ( Lostroh and Lee , 2001 ; Ellermeier et al. , 2005 ; and references therein ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17163975" target="_blank">17163975</a>]<br></li>
<li>Induction of SPI1 effector-proteins requires three transcriptional-regulators of the AraC family : HilD , HilC and RtsA ( Schechter et al. , 1999 ; Ellermeier and Slauch , 2003 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27341691" target="_blank">27341691</a>]<br></li>
<li>In this scheme , activation of invF transcription by HilD and HilC leads to co-transcriptional activation of the downstream sip genes , as has been shown for HilA-activated transcription from the invF promoter ( Darwin and Miller , 1999a ; Eichelberg et al. , 1999 ; Lostroh and Lee , 2000 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>Transcriptional activation by HilC and HilD relieves repression of the hilA promoter by the nucleoid proteins H-NS and Hha ( Olekhnovich and Kadner 2006 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20008574" target="_blank">20008574</a>]<br></li>
<li>Transcription of the hilA gene is also stimulated by the SPI1-encoded AraC/XylS-like regulatory proteins HilC , HilD and RtsA ( Eichelberg and Galan , 1999 ; Lostroh et al. , 2000 ; Rhen and Dorman , 2005 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28631437" target="_blank">28631437</a>]<br></li>
<li>HilC and HilD relieve repression of the hilA promoter by the nucleoid proteins H-NS and Hha ( Olekhnovich and Kadner 2006 ) and can activate expression of the invF and sicA/sip transcriptional units independently of HilA ( Rakeman et al. 1999 ; Akbar et al. 2003 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25991823" target="_blank">25991823</a>]<br></li>
<li>As HilC and HilD interact with their own promoters and activate their transcription ( Lucas and Lee , 2001 ; Olekh-novich and Kadner , 2002 ) , the increased transcription from hilC and hilD promoters in the Dlon mutant might not result from the accumulation of putative regulators but from the accumulation of HilC and HilD themselves .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15661008" target="_blank">15661008</a>]<br></li>
<li>-- Diagram showing the transcriptional units of Salmonella enterica SPI-1 and the regulatory circuits under the control of transcription factors HilA , HilD , HilC , RtsA , and InvF ( adapted from Lostroh and Lee 2001 ; Ellermeier and Slauch 2003 ; Altier 2005 ; Jones 2005 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20008574" target="_blank">20008574</a>]<br></li>
<li>This work demonstrates the regulation of rtsA , hilC , hilD , and hilA by HilC , HilD and RtsA , and that each regulator has a role in the host during infection .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17208038" target="_blank">17208038</a>]<br></li>
<li>Here we report that RtsA and the related hilA regulators , HilD and HilC , coordinately regulate expression of dsbA and the SPI1 TTSS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14679226" target="_blank">14679226</a>]<br></li>
<li>Acetylation of K297 Regulating HilD DNA-Binding Ability and Stability At the transcriptional level , the expression of hilD is positively autoregulated and modulated by a feed-forward loop involving HilD itself and regulators HilC and RtsA [ 3 , 28 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28329249" target="_blank">28329249</a>]<br></li>
<li>This is assumed to result from the interaction of HilC and HilD with their own promoters , activating transcriptions ( Lucas and Lee , 2001 ; Olekhnovich and Kadner , 2002 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15661008" target="_blank">15661008</a>]<br></li>
<li>Specifically , HilC , HilD , and RtsA are all capable of individually activating the PhilA PhilC , PhilD , and PrtsA promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20686667" target="_blank">20686667</a>]<br></li>
<li>In vitro , HilD and HilC bind the same regions upstream of hilA ( Schechter and Lee , 2001 ; Olekhnovich and Kadner , 2002 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>Furthermore , HilD and HilC bind the same DNA sites upstream of hilD and hilC ( 38 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24375101" target="_blank">24375101</a>]<br></li>
<li>For example , the transcription of many SPI-1 genes can be regulated by HilA , HilC , HilD , HilE , and PhoP/Q [ 8 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21949647" target="_blank">21949647</a>]<br></li>
<li>For example , the transcription of SPI-1 genes can be activated by HilA , HilC , HilD , InvF , and SirA ( 30 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28439039" target="_blank">28439039</a>]<br></li>
<li>In replicate experiments , the average degree ( standard deviation ) of stimulation by HilC , HilD , or both was 2.77 0.96 , 2.35 0.87 , and 2.75 1.05 , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15126488" target="_blank">15126488</a>]<br></li>
<li>Alternatively , HU or Fis may inﬂuence hilA indirectly by increasing the levels and/or activity of HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12933899" target="_blank">12933899</a>]<br></li>
<li>Expression of hilA-lacZ fusions displays complex regulation and can be activated by the HilC or HilD protein ( 42 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12107132" target="_blank">12107132</a>]<br></li>
<li>The method and parameters that we initially used in this study for clustering the S. Typhimurium SL1344 global gene expression results from the COLOMBOS database , were successful to identify gtgE , phoH , sinR , lpxR , SL1896 and SL4247 , as novel genes regulated by HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27886269" target="_blank">27886269</a>]<br></li>
<li>The regions required for the regulation of ssrAB by HilD and H-NS , deﬁned in this study by expression and binding assays , are shown .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25135218" target="_blank">25135218</a>]<br></li>
<li>In addition to the negative control that was exerted by H-NS/Hha under low-osmolarity conditions , the homologous virulence activators HilD , HilC , and RtsA ( Hil activators ) induced rtsA-lac expression in a high-salt medium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17675384" target="_blank">17675384</a>]<br></li>
<li>Two of the 8 genes , mcpA ( STM3138 ) and mcpC ( STM3216 ) , are predicted to encode Salmo-nella-specific methyl-accepting chemotaxis proteins [ 40 ] and display similar expression patterns to the SPI1-encoded , SPI1-associated and SPI4-encoded genes which are directly or indirectly activated by HilD , and repressed by SsrB , but are not regulated by HilA suggesting they are controlled by the HilD feed-forward loop .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27564394" target="_blank">27564394</a>]<br></li>
<li>Since HilA is activated by the three factors HilD , HilC and RtsA ( Ellermeier et al. 2005 ; Ellermeier and Slauch 2007 ) , in analogy with the equations written above , the dynamics of HilA can be described by the following nonlinear system : where again i = 1 corresponds to HilD , i = 2 to HilC , and i = 3 to RtsA , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21107576" target="_blank">21107576</a>]<br></li>
<li>Conversely , HilC and HilD act as transcription activators to induce the expression of hilA by binding to the upstream repression sites ( Ellermeier et al. , 2005 ; Lucas and Lee , 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27565525" target="_blank">27565525</a>]<br></li>
<li>represses expression of HilD by binding to the hilD leader transcript In E. coli , CsrB and CsrC regulate gene expression by binding to and sequestering the RNA binding protein CsrA , thus counteracting the negative posttranscriptional control exerted by CsrA on its target mRNAs ( Babitzke and Romeo , 2007 ; Lucchetti-Miganeh et al. , 2008 ; Timmermans and Van Melderen , 2010 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21518393" target="_blank">21518393</a>]<br></li>
<li>These results also explain why our invF-lacZ reporter plasmid , pVV448 , was not activated by HilD or HilC to the same extent as the chromosomal invF-lacZ fusion because pVV448 does not contain the HilD - and HilC-dependent +1 of invF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>To perform this analysis , we varied the degree by which the SPI1 regulators - HilC , HilD , and RtsA - could activate HilD expression by simulating the model at different values for the parameter kD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20686667" target="_blank">20686667</a>]<br></li>
<li>Thus , it is possible that the feed forward loop constituted by HilD , HilC , and RtsA can regulate the expression of ssrAB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25135218" target="_blank">25135218</a>]<br></li>
<li>They act by increasing the level of HilD protein to overcome HilE , and controlling HilD activity , or in the case of Fur , lowering the threshold required at the promoter , such that HilD activates its own promoter as well as induces expression of HilC and RtsA , which then act to amplify and accelerate SPI1 expression ( Saini et al. 2010a ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22021388" target="_blank">22021388</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_320">
The parameter kD specifies the degree by which the SPI1 regulators - HilC , HilD , and RtsA - can activate HilD expression , effectively the strength of positive feedback on HilD expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20686667" target="_blank">20686667</a>]<br></label>
<input id="act_similarity_320", type="checkbox">
<div class="ACT_similar">
<li>The parameter k specifies the degree by D D D which the SPI1 regulators - HilC , HilD , and RtsA - can activate HilD expression , effectively the strength of positive feedback on HilD expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20686667" target="_blank">20686667</a>]<br></li></div>
<li>Due to the counteracting nature of the repressing URS and derepressing HilC and HilD proteins that directly control hilA expression , the hns-1 : : kan mutation may increase hilA expression under the low-osmolarity conditions ( i ) by decreasing its repression and/or ( ii ) by increasing its derepression by HilC or HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12933899" target="_blank">12933899</a>]<br></li>
<li>Only when the activating signals are sufficiently strong is HilD expressed at a high enough level to overcome sequestration by HilE and activate the expression of the other SPI1 regulators -- HilC , RtsA , and HilA-and also further induce its own expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20686667" target="_blank">20686667</a>]<br></li>
<li>HilD binds to the regulatory region of ssrAB , thereby counteracting H-NS mediated repression and promoting OmpR binding [ 40 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23295139" target="_blank">23295139</a>]<br></li>
<li>FIG 7 Model for the HilD - and H-NS-mediated regulation of ssrAB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25135218" target="_blank">25135218</a>]<br></li>
<li>As shown in Fig. 8 , the increased effect of Lon depletion on hilA expression was abolished by depletion of both HilC and HilD , suggesting that both proteins are also involved in regulation of SPI1 by Lon under SPI1-repressing conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15661008" target="_blank">15661008</a>]<br></li>
<li>( iii ) RtsA , HilC , and HilD are required for signal ampliﬁcation such that once one of these genes is turned on , expression of the others is also induced , resulting in greater induction of hilA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923082" target="_blank">12923082</a>]<br></li>
<li>A subset of SPI1 genes is induced by HilD overexpression in the absence of hilA Our results suggested that HilA and HilD have independent effects on expression of invF , sipA and sipC in S. typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>HilD , HilC and RtsA then participate in a feed-forward loop that converges on activation of hilA , which in turn activates transcription of structural components of the needle complex ( prg/org ) as well as invF , which is a transcriptional-activator of secreted effector-proteins ( sic/sip and others ) ( Figure 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28335027" target="_blank">28335027</a>]<br></li>
<li>The effect of HilC and HilD proteins on transcription of these promoters indicated that all three promoters are active in the absence of other cellular proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12107132" target="_blank">12107132</a>]<br></li>
<li>RtsA increases invasion gene expression by increasing transcription of hilA , which is independent of HilC and HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15765064" target="_blank">15765064</a>]<br></li>
<li>LoiA activates expression of hilA gene through activating hilD As the expression of hilA is directly controlled by HilD , HilC and RtsA , we tested whether LoiA regulates HilA through any of these three regulators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28575106" target="_blank">28575106</a>]<br></li>
<li>Thus , FliZ-mediated induction of hilA expression is through HilD , while RtsA and HilC act as ampliﬁers of the signal .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17951383" target="_blank">17951383</a>]<br></li>
<li>( C ) In the presence of bile , HilD is more stable , leading to enhanced expression of hilD , hilA , and thus SPI-1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29229736" target="_blank">29229736</a>]<br></li>
<li>First , Lon is induced to control turnover of HilD and HilC and hence limit the expression of SPI-1 genes , which is of particular relevance after invasion of epithelial cells .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23554419" target="_blank">23554419</a>]<br></li>
<li>That is to say , disruption of ﬂiZ abolished the increase of HilC and HilD caused by deleting clpPX ( Fig. 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18245288" target="_blank">18245288</a>]<br></li>
<li>Transcriptional activation of hilA is dependent upon three AraC-like proteins , HilD , HilC , and RtsA , which bind to the same sites upstream of activated promoters ( 11 , 12 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29378886" target="_blank">29378886</a>]<br></li>
<li>However , induction of hilA is independent of HilC and HilD and is mediated by direct binding of RtsA to the hilA promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923082" target="_blank">12923082</a>]<br></li>
<li>Therefore , CpxR and RpoH seem to affect HilD concentration and thus repress the SPI-1 genes differentially ; RpoH by inducing transcription of lon and CpxR by probably affecting the posttranscriptional expression or activity of Lon , or through another protease .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26300871" target="_blank">26300871</a>]<br></li>
<li>Expression and activity of HilD are tightly Funding : This work was supported by the Spanish Ministry of Economy and Competitiveness ( grants BIO2010-15417 and AGL2013-45339-R , CB ) , the Catalonian government ( grant 2014SGR1260 , CB ) , the RecerCaixa program ( grant 2012/ACUP/00048 , CB ) , the Swedish Research Council Natural Sciences and Engineering ( grant D0480901 , UR ) Abstract Gre factors-mediated control of hilD transcription is essential for the invasion of epithelial cells by Salmonella enterica serovar 1 Departament de Genètica , Microbiologia i Estadıstica , Universitat de Barcelona , Barcelona , Spain , 2 Department of Microbiology , Tumor and Cell Biology , Karolinska Institutet , Stockholm , Sweden The invasion of epithelial cells by Salmonella enterica serovar Typhimurium is a very tightly regulated process .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28426789" target="_blank">28426789</a>]<br></li>
<li>ORIGINAL RESEARCH published : 06 August 2015 doi : 10.3389 / fmicb .2015.00807 The two-component system CpxR/A represses the expression of Salmonella virulence genes by affecting the stability of the transcriptional-regulator HilD Miguel A. De la Cruz 1 † , Deyanira Pérez-Morales 2 † , Irene J. Palacios 2 , Marcos Fernández-Mora 2 , Edmundo Calva 2 and Víctor H. Bustamante 2 * Edited by : Dongsheng Zhou , Beijing Institute of Microbiology and Epidemiology , China 1 Unidad de Investigación Médica en Enfermedades Infecciosas y Parasitarias , Centro Médico Nacional Siglo XX1-IMSS , México DF , Mexico , 2 Departamento de Microbiología Molecular , Instituto de Biotecnología , Universidad Nacional Autónoma de México , Cuernavaca , Morelos , Mexico Reviewed by : Weili Liang , National Institute for Communicable Disease Control and Prevention , China Tracy Raivio , University of Alberta , Canada Salmonella enterica can cause intestinal or systemic infections in humans and animals mainly by the presence of pathogenicity islands SPI-1 and SPI-2 , containing 39 and 44 genes , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26300871" target="_blank">26300871</a>]<br></li>
<li>HilD is then responsible for the direct activation of hilA and ssrAB and , indirectly , of the SPI-1 and SPI-2 regulons .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21518393" target="_blank">21518393</a>]<br></li>
<li>All mutants of known hilD regulators tested , both positive ( hilC , hilA , rtsA , rtsB , crsB , csrC , sirA , ﬂiZ , and ompR ) and negative ( hilE , hha , and lon ) , had no effect on the repressive effects of bile and cholate , conﬁrming that bile-mediated repression occurs at the level of HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27185788" target="_blank">27185788</a>]<br></li>
<li>HilD activates transcription of several SPI-1 genes , including components of the T3SS , secreted effector-proteins ( 10 -- 12 ) , and the TFs HilA and InvF ( 7 , 10 , 11 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27601571" target="_blank">27601571</a>]<br></li>
<li>Importantly , these regulators all converge at HilD , which induces SPI-1 genes that encode the secreted effectors and structural components of the TTSS directly by activating their transcription and indirectly through induction of the downstream transcriptional-activator HilA ( 9 , 16 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27185788" target="_blank">27185788</a>]<br></li>
<li>These include HilA that binds and represses the promoter of ssaH [ 24 ] , and HilD that binds and activates the promoter of the ssrAB operon [ 25 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19126220" target="_blank">19126220</a>]<br></li>
<li>Rather , HilD , HilC , and RtsA are overcoming the effects of these proteins at the individual promoters and it is the regulation of HilD levels and action that is the key .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22021388" target="_blank">22021388</a>]<br></li>
<li>Acetylation regulates the protein levels of HilD and HilA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26810370" target="_blank">26810370</a>]<br></li>
<li>SirA activates the expression of SsrB and HilA through HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21518393" target="_blank">21518393</a>]<br></li>
<li>Genome-wide transcriptional and/or binding analyses support that H-NS represses gtgE , sinR , lpxR , SL1896 and SL424773 -- 75 ; however , whether HilD induces the expression of these genes by acting as an anti-repressor of H-NS , and how it positively controls the expression of phoH , remains to be determined .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27886269" target="_blank">27886269</a>]<br></li>
<li>These results show that HilD positively controls the expression of the gtgE , phoH , sinR , lpxR , SL1896 and SL4247 genes , independently of HilA and InvF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27886269" target="_blank">27886269</a>]<br></li>
<li>Thus , HilD positively and directly regulates the expression of the gtgE , phoH , sinR , lpxR and SL4247 genes , and positively but indirectly controls the expression of the SL1896 gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27886269" target="_blank">27886269</a>]<br></li>
<li>In summary , stimulation of hilA transcription by the AraC/XylS family members HilC and HilD does exhibit the expected dependence on residues in - CTD , but activation of transcription involves other - CTD-independent processes as well .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15126488" target="_blank">15126488</a>]<br></li>
<li>In this context , slrP is induced by overexpression of HilC , HilD , and RtsA independently of the central SPI1 regulator HilA , with RtsA being the best inducer .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25182488" target="_blank">25182488</a>]<br></li>
<li>This is represented by the following equation : 3 d 1/2 HilA 1/4 b3 þ b3 f HilD ; HilC ; RtsA ; HilA dt a3 HilA : • In all the above equations , [ HilD ] can be either 1 or 0 , depending on the presence or absence of the signal for induction , b1 , b2 and b3 are the basal rates of synthesis of HilC , RtsA and HilA respectively , b1 , b2 and b3 are ampliﬁcation factors which can be thought of as analogous to translation , a1 , a2 and a3 and are the rates of degradation of HilC , RtsA and HilA respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19003447" target="_blank">19003447</a>]<br></li>
<li>Finally , the rate of change of HilA concentration depends on its basal synthesis , activation by the four transactivators ( HilD , HilC , RtsA and SirA ) , autorepression and its own degradation rate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19537165" target="_blank">19537165</a>]<br></li>
<li>To test whether the RtsA , HilC , and HilD activators of hilA are also regulated by these proteins , we investigated the transcription of rtsA-lac , hilC-lac , and hilD-lac chromosomal fusions ( 13 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17675384" target="_blank">17675384</a>]<br></li>
<li>Future work can also take into account the differential binding afﬁnities of HilD and HilC on the hilA promoter as shown by Olekhnovich and Kadner ( 2002 ) and include the regulation of HilC and HilD by CsrA/B as demonstrated by Lawhon et al. ( 2003 ) and Fortune et al. ( 2006 ) to get a better picture of the complex interactions involved in the regulation of PhilA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19003447" target="_blank">19003447</a>]<br></li>
<li>Our data conﬁrm a large degree of overlap between the HilD , HilC , and RtsA regulons and extend this overlap to include the novel HilD targets ; we observed transcription activation of all 8 of the HilD-activated genes by HilC and all but one by RtsA ( Fig. 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24375101" target="_blank">24375101</a>]<br></li>
<li>We wanted to determine if HilC , HilD and RtsA could induce expression of hilC , hilD , rtsA and hilA in the absence of the other regulators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16045614" target="_blank">16045614</a>]<br></li>
<li>Therefore , we examined the expression of hilA , rtsA , hilC and hilD transcriptional fusions in the presence and absence of RtsA , HilC , and/or HilD ( Fig. 7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16045614" target="_blank">16045614</a>]<br></li>
<li>The effect of SPI1 inducing and repressing conditions on the expression of rtsA , hilC , hilD and hilA in the absence of in RtsA , HilC , and/or HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16045614" target="_blank">16045614</a>]<br></li>
<li>RtsA induces expression of hilA in the absence of both HilC and HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923082" target="_blank">12923082</a>]<br></li>
<li>The ability of SirA to induce expression of rtsA , hilC , hilD , hilA and invF in the absence of RtsA , HilC , and/or HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16045614" target="_blank">16045614</a>]<br></li>
<li>Overproduction of HilD and HilC activate the expression of a chromosomal invF-lacZ fusion in the absence of hilA In order to study the activation of invF by HilD and HilC , we examined the expression of a chromosomal invF-lacZ fusion in a hilA hilD mutant strain transformed with pSA4 , pLS119 or pCH112 , which express HilD , HilC and HilA , respectively , under the control of the arabinose-inducible PBAD promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>HilC , HilD and RtsA , in addition to regulation of hilA , are all capable of inducing expression of the inv/spa operon -- and by read-through , the sic/sip operon -- in a fashion independent of HilA [ 15,16,25,27 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17208038" target="_blank">17208038</a>]<br></li>
<li>Depletion of ClpXP caused signiﬁcant increases in the amounts of two SPI1-encoded transcriptional-regulators , HilC and HilD , leading to the stimulation of hilA induction and therefore activation of SPI1 expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18245288" target="_blank">18245288</a>]<br></li>
<li>Earlier work on mathematical modeling of regulation of expression of hilA by HilC , HilD and RtsA was based on feed-forward architecture , considering HilD to be the primary activator of the feed-forward loop [ Maithreye and Mande , 2007 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19537165" target="_blank">19537165</a>]<br></li>
<li>Production of HilD leads to transcriptional activation of HilC , and each activates the expression of both ( 13 , 40 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18245288" target="_blank">18245288</a>]<br></li>
<li>As controls , the expression of cat transcriptional fusions of the hilA and ssaG genes , which are controlled by HilD directly and indirectly , respectively , were also assessed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27886269" target="_blank">27886269</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_371">
The expression of hilA , encoding the transcriptional-activator of the T3SS structural genes , is directly controlled by three AraC-like regulators , HilD , HilC , and RtsA , each of which can activate hilD , hilC , rtsA , and hilA genes , forming a complex feed-forward regulatory loop .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26884427" target="_blank">26884427</a>]<br></label>
<input id="act_similarity_371", type="checkbox">
<div class="ACT_similar">
<li>The expression of hilA , encoding the transcriptional-activator of the SPI1 structural genes , is directly controlled by three AraC-like regulators , HilD , HilC , and RtsA , each of which can activate the hilD , hilC , rtsA , and hilA genes , forming a complex feed-forward regulatory loop .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22021388" target="_blank">22021388</a>]<br></li></div>
<li>Two additional regulators , HilC and HilD , activate directly the expression of hilA and the invF operon , independently of HilA ( Dieye et al. , 2007 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27404739" target="_blank">27404739</a>]<br></li>
<li>Although HilA is the major regulator , two other regulators encoded within SPI1 , HilC and HilD , can directly increase the expression of both hilA and invF independently of hilA ( 3 , 53 , 56 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14645287" target="_blank">14645287</a>]<br></li>
<li>In contrast , disruption of the downstream -10 hexamer sequence dramatically reduced the ability of HilD and HilC to activate invF expression ( Fig. 6 , black and striped bars , pSA14 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>HilC and HilD bind directly to the upstream sequence of hilA and derepress the transcription of PhilA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15661008" target="_blank">15661008</a>]<br></li>
<li>RtsA belongs to the AraC/XylS family of regulators , and can directly bind the hilA promoter and induce the hilA expression independent of HilC and HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15661008" target="_blank">15661008</a>]<br></li>
<li>In contrast , we detected regulation of nearby genes for all but two FIG 1 Direct regulation of the invF-containing transcript by HilD , HilC , and RtsA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27601571" target="_blank">27601571</a>]<br></li>
<li>3 s 4 Model 2 : Two feedforward loops with OR gate logic for regulation of P by HilD , HilC and RtsA ( monomer hilA activation ) ðModel 4Þ s 5 1/2 6 a3 HilA : ðModel 2Þ e activators are set to 2 , the the activation of PhilA by If Hill coefﬁcients of all th model generated represents homodimers .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19003447" target="_blank">19003447</a>]<br></li>
<li>The production of HilC and HilD was induced by the addition of 1 mM IPTG for 5 min .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15661008" target="_blank">15661008</a>]<br></li>
<li>The acetylation of HilD by Pat maintained HilD stability and was essential for the transcriptional activation of HilA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26810370" target="_blank">26810370</a>]<br></li>
<li>However , this binding of HilD requires a larger target than does binding of HilC , as seen by comparing the binding to fragments C-d and C-f .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12107132" target="_blank">12107132</a>]<br></li>
<li>Changes at any of these positions depressed binding of HilD , and all but the last three positions contributed to-binding of HilC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12107132" target="_blank">12107132</a>]<br></li>
<li>It will be interesting to determine how the binding of HilA upstream of invF affects the ability of HilD and HilC to-bind upstream of and activate invF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>Identiﬁcation of candidate promoter sequences required for HilD - and HilC-dependent activation of invF To better deﬁne the sequences required for HilD - and HilC-dependent activation of invF , we cloned 919 bp from upstream of the invF ORF in front of a lacZ reporter gene in the pRW50 vector ( Lodge et al. , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>Studies on regulation of PhilA by different types of O gate architectures involving three activators ( HilD , HilC and RtsA ) R ¬ As can be seen from Table 1 , Model 1 ( AND gate ) failed to capture the effect of the different mutants on HilA expression levels since it predicted decrease to basal activity irrespective of which regulator was muta-ted .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19003447" target="_blank">19003447</a>]<br></li>
<li>Obviously , this central core of regulators ( HilC , HilD , RtsA , HilE ) would integrate multiple signaling inputs , as many sensors have been described and shown to regulate the activation of pathogenicity ; but a noise-driven switch would nevertheless be a simple way of regulation , including complex sensors for population density or physiological state of the host .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21107576" target="_blank">21107576</a>]<br></li>
<li>However , HilD does not work alone under normal conditions : in the absence of HilC and RtsA , loss of HilD has little effect on transcription of hilA or invasion of the intestine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16045614" target="_blank">16045614</a>]<br></li>
<li>However , given that the external regulators of the system , e.g. SirA/BarA and OmpR/EnvZ , seem to function through HilD , loss of HilD could have more profound effect than loss of RtsA or HilC due to the lack of these external activation signals rather than some inherent difference in efﬁciency of HilD versus HilC or RtsA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16045614" target="_blank">16045614</a>]<br></li>
<li>Upon deletion of ﬂhDC , the activating effect of HilD was neutralized and rﬂM transcription was abolished ( see Fig .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24488311" target="_blank">24488311</a>]<br></li>
<li>The overexpression of pat could increase the acetyl-ation of HilD ( Figure 1A ) , and its absence could decrease this acetylation ( Figure 1D ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28329249" target="_blank">28329249</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_392">
HilD binds to the regulatory regions of gtgE , phoH , sinR , lpxR and SL4247 , but not to that of SL1896 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27886269" target="_blank">27886269</a>]<br></label>
<input id="act_similarity_392", type="checkbox">
<div class="ACT_similar">
<li>HilD binds to the regulatory region of gtgE , phoH , sinR , lpxR and SL4247 , but not to that of SL1896 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27886269" target="_blank">27886269</a>]<br></li></div>
<li>The results of this study allow us to speculate that Lon may degrade the hilA activator protein , HilD , to posttranscriptionally regulate its activity and decrease the level of hilA expression during intracellular growth of Salmonella .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15039320" target="_blank">15039320</a>]<br></li>
<li>The activation of hilE transcription produces HilE protein , which then leads to repression of hilA expression and subsequent expression of the other SPI-1 genes through its interaction with the HilD regulatory protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15731035" target="_blank">15731035</a>]<br></li>
<li>Under the experimental conditions used , HilC and RtsA were not needed to achieve high hilA expression ; however , HilD was essential since hilA expression was abolished in ΔhilD mutants regardless of the presence or absence of Gre factors .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28426789" target="_blank">28426789</a>]<br></li>
<li>Therefore , we propose that acetylation may regulate HilD in a posttranslational manner to regulate the expression of hilA and SPI-1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26810370" target="_blank">26810370</a>]<br></li>
<li>HilD is clearly the most signiﬁcant contributor to the expression of hilA and therefore SPI1 in general , since many systems which control expression of hilA seem to work through HilD in some way ( 6 , 8 , 16 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
<li>Collectively , these results demonstrate that LoiA directly binds to the hilD promoter to activate hilD expression ; and HilD in turn activates hilA expression , leading to the activation of eterminant in loiA deletion reduced the expression of hilA and SPI-1 genes Fig 4 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28575106" target="_blank">28575106</a>]<br></li>
<li>Since SirA activates expression of a hilA : : lacZ transcriptional fusion only in the presence of HilD ( 222 ) , CsrA has been hypothesized to-bind to the hilD mRNA and impair its translation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23554419" target="_blank">23554419</a>]<br></li>
<li>Thus , reduced levels of FliZ are expected to lead to reduced levels of the transcriptional-regulators HilA ( controlled by HilD ) and InvF ( controlled by HilA ) , both of which are known to activate the expression of effector genes located inside and outside ( indicated by a dotted box ) SPI-1 ( 77 -- 79 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29150501" target="_blank">29150501</a>]<br></li>
<li>To determine how HilD , OmpR , and SlyA coordinate to regulate the expression of ssrAB in response to different growth conditions is a matter of our current and future studies .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25135218" target="_blank">25135218</a>]<br></li>
<li>To check if both regulators are important , the regulation of hilC gene expression by HilD and SirA is considered to function as an AND gate , and to check the possibility of one of the regulators being important , this dependency is taken as an OR gate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19537165" target="_blank">19537165</a>]<br></li>
<li>HilD , HilC , and RtsA are homologous AraC-like regulators that bind to very similar DNA sites ; therefore , when these regulators are overexpressed , they are able to self-activate their own expression and to activate the expression of one another , as well as to induce the expression of most genes belonging to the HilD regulon ; however , HilD is considered to be dominant , as deletion of hilD , but not of hilC or rtsA , drastically reduces the expression of the targets of HilD ( 23 , 24 , 26 -- 30 , 61 -- 64 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25135218" target="_blank">25135218</a>]<br></li>
<li>SirA binds to the csrB and csrC regulatory regions , but not to those of hilD , hilA , hilC or ssrAB The results described above are consistent with the notion that SirA acts through HilD to indirectly control the expression of hilA and ssrAB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21518393" target="_blank">21518393</a>]<br></li>
<li>In contrast , the deacetylated HilD tends to be degraded and fails to activate the transcription of HilA and thus downregulates the expression of SPI-1 , leading to an attenuated S. Typhimurium virulence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26810370" target="_blank">26810370</a>]<br></li>
<li>Primary examples of factors that inﬂuence HilD activity are HilE , a major negative regulator of SPI1 T3SS expression , and the ﬂagellar gene product FliZ , which acts indirectly upon HilD to increase activity ( 10 , 16 , 18 -- 20 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29378886" target="_blank">29378886</a>]<br></li>
<li>Thus , while both HilE and FliZ act through HilD , the effects of FliZ are the same in wild-type and hilE backgrounds , indicating that these two factors regulate HilD independently of each other .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20889744" target="_blank">20889744</a>]<br></li>
<li>Since HilE binds to HilD , its action mainly reduces the number of free proteins of HilD per cell and then it has been taken into account in the equation for p1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21107576" target="_blank">21107576</a>]<br></li>
<li>HilE , in turn , is known to-bind HilD and repress the HilD-mediated activation of the PhilA , PhilC , PrtsA , and PhilD promoters ( 6 , 20 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19218381" target="_blank">19218381</a>]<br></li>
<li>110 100 90 ompR + ompR1009 : : Tn10dTc 80 70 60 50 40 30 20 10 0 + ¬ + ¬ + ¬ + ¬ hilD hilC + + + ¬ ¬ ¬ + ¬ + + + + ¬ ¬ ¬ ¬ rtsA Fusion hilA Discussion We propose a model for SPI1 regulation in which the three AraC-like regulators HilD , HilC and RtsA act in a feedforward loop to activate expression of the hilA gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16045614" target="_blank">16045614</a>]<br></li>
<li>However , this possibility is less likely , because no chromosomal rtsA-lac expression was detected in the absence of HilD , HilC , and RtsA ( 13 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17675384" target="_blank">17675384</a>]<br></li>
<li>The expression of hilA , the master regulator for SPI1 , is controlled by HilD , HilC , and RtsA , which act in a complex feed-forward loop .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20889744" target="_blank">20889744</a>]<br></li>
<li>This promoter is active in the absence of repressive proteins and is only moderately ( two - to threefold ) stimulated by the HilC or HilD proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15126488" target="_blank">15126488</a>]<br></li>
<li>It is interesting to note that several regulators of T3SS1 , such as HilA , HilC , HilD , and InvF , were found in significantly lower levels in the dam mutant .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21984608" target="_blank">21984608</a>]<br></li>
<li>This is probably due to the additional regulations considered in this model , whereby HilC and RtsA were ampliﬁed even in the absence of HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19003447" target="_blank">19003447</a>]<br></li>
<li>Affiffinity-purified maltose-binding protein ( MBP ) - HilD , which is active in vivo and specifically binds to HilD-target genes in vitro14 ,29 , and the DNA fragments contained in the respective transcriptional fusion of each gene , were used to perform electrophoretic mobility shift assays ( EMSAs ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27886269" target="_blank">27886269</a>]<br></li>
<li>Therefore , the expression of gtgE is controlled by HilD in SPI-1-inducing conditions and probably by another regulator in SPI-2-inducing conditions , which would coordinate the secretion of GtgE through the T3SS-1 and T3SS-2 , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27886269" target="_blank">27886269</a>]<br></li>
<li>Similarly , the expression of slrP , also encoding an effector-protein secreted through both T3SS-1 and T3SS-2 , is controlled by HilD in SPI-1-inducing conditions and by the response-regulator PhoP in SPI-2-inducing conditions32 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27886269" target="_blank">27886269</a>]<br></li>
<li>Interestingly , when S. typhimurium is grown under the normally repressing low osmolarity condition , a null mutation in hns results in high hilA expression , even in the absence of HilC and HilD ( L. Schechter and C. Lee , unpublished observations ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11755416" target="_blank">11755416</a>]<br></li>
<li>HilC and HilD are encoded in the SPI1 locus , while RtsA is encoded elsewhere in the chromosome in an operon with RtsB , which negatively regulates expression of ﬂhDC and therefore the entire ﬂagellar regulon ( 19 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17951383" target="_blank">17951383</a>]<br></li>
<li>HilD expression and activity are very tightly regulated , highlighting its pivotal role in the biology of S. Typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28426789" target="_blank">28426789</a>]<br></li>
<li>The average HilD half-life for bacteria grown in me-dium alone was 63 min , but the half-life was reduced to 16 min in the presence of bile and 8 min in the presence of cholate , revealing a mechanism in which bile and bile acids repress SPI-1 gene expression and protein levels by posttranslational destabilization of HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27185788" target="_blank">27185788</a>]<br></li>
<li>Briefly , we demonstrated that SirA can not activate expression of SsrB and HilA proteins in the absence of HilD ( Figs. 2 , S1 and S2 ) , in agreement with results from a previous study ( Ellermeier et al. , 2005 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21518393" target="_blank">21518393</a>]<br></li>
<li>The expression of the type III secretion system-related genes involved in invasion and intracellular survival is induced in the presence of transcriptional-regulator proteins such as HilA , HilC , HilD , InvA , InvC , InvF , SirA , and SirB ( 11 , 27 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24780330" target="_blank">24780330</a>]<br></li>
<li>The expression of the genes , those regulate the type III secretion system , are involved in completion of invasion , intracellular survival and multiplication , is induced by the transcriptional-regulator proteins such as HilA , HilC , HilD , InvA , InvC , InvF , SirA , and SirB ( Johnston et al. , 1996 ; Rakeman et al. , 1999 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27565525" target="_blank">27565525</a>]<br></li>
<li>As seen in Fig. 8C , hilD expression was reduced significantly in this mutant , demonstrating that translational control could be detected and therefore that propionate failed to affect hilD transcription or translation , and suggesting post-translational control of HilD as the means of invasion regulation by propionate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23289537" target="_blank">23289537</a>]<br></li>
<li>In lon + cells , little HilC and a trace of HilD were detected even in cells grown under the SPI1-acivating condition , suggesting that these proteins are short-lived and/or exist in low amounts .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15661008" target="_blank">15661008</a>]<br></li>
<li>In contrast , both HilC and HilD blocked formation of the hilC transcript that was made in their absence and activated the synthesis of a transcript that was ca. 70 nucleotides shorter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12107132" target="_blank">12107132</a>]<br></li>
<li>Moreover , in the absence of Lon , HilC and HilD accumulate intracellularly ( 317 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23554419" target="_blank">23554419</a>]<br></li>
<li>All indications are that , under the conditions that we are examining , dsbA is primarily controlled by RtsA , HilC , and HilD ( 18 ) ; CpxRA , which controls dsbA in E. coli , seems irrelevant to either expression of dsbA or assembly and function of the SPI1 T3SS or ﬂagella in serovar Typhimurium ( Fig. 2C ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17951383" target="_blank">17951383</a>]<br></li>
<li>We conclude that HilD , HilC , and RtsA bind very similar DNA sites , although the failure of RtsA to regulate ytfK ( Fig. 3C ) indicates that there are subtle differences in the sequence speciﬁcities of HilD , HilC , and RtsA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24375101" target="_blank">24375101</a>]<br></li>
<li>C , Stabilities of WT HilD , K297Q , K297R and K297A in ΔhilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28329249" target="_blank">28329249</a>]<br></li>
<li>To assess the accumulation of SPI1 products in different concentrations of HilC or HilD , these cells were exposed to various concentrations of IPTG for 2.5 h , and HilC , HilD and SPI1 levels were analysed by immunoblotting .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15661008" target="_blank">15661008</a>]<br></li>
<li>The presence of HilC , HilD , or RtsA resulted in protection of the positions 74 , 70 , 65 , and 34 compared to what was seen for free DNA ( Fig. 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17675384" target="_blank">17675384</a>]<br></li>
<li>However , in contrast to other effectors examined , RtsA induction of slrP is independent of HilA , InvF , and HilD or HilC ( Fig. 7A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923082" target="_blank">12923082</a>]<br></li>
<li>However , transcription of rtsA , hilC and hilD was induced in response to low oxygen independent of RtsA , HilC , or HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16045614" target="_blank">16045614</a>]<br></li>
<li>Even though HilC and RtsA can independently activate HilA expression when constitutively expressed from ectopic promoters , these two regulators are incapable of doing so in the absence of HilD when expressed from their native promoters [ 16 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20686667" target="_blank">20686667</a>]<br></li>
<li>Previous studies showed that four SPI1-encoded transcriptional-regulators , HilD , HilC , HilA and InvF , act in an ordered fashion to coordinately activate expression of the SPI1 TTSS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>These regulators , HilD , HilC , HilA , and InvF , act in an ordered fashion to activate coordinated expression of the SPI1 genes ( 2 , 10 , 12 , 13 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18723621" target="_blank">18723621</a>]<br></li>
<li>SPI1 carries all of the genes necessary to create a functional T3SS apparatus , several secreted effectors , and regulators such as HilA , HilC , HilD and InvF ( described in more detail below .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17208038" target="_blank">17208038</a>]<br></li>
<li>Although many factors and a complex mechanism related to environmental stimuli are involved in the modulation of SPI1 genes , they are thought to converge on the activation of several transcriptional-regulators encoded within SPI1 , such as HilA , HilC , HilD and InvF ( 7 -- 9 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17329372" target="_blank">17329372</a>]<br></li>
<li>In addition , these three regulators ( i.e. , HilC , HilD , and RtsA ) activate a regulator within SPI-1 , HilA , which shows homology with the OmpR/ToxR family of DNA-binding proteins and induces transcriptional activation of SPI-1 genes in response to a variety of stimuli ( 1 , 59 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21075923" target="_blank">21075923</a>]<br></li>
<li>Keywords SirA/BarA ; Csr ; HilD ; two-component system ; Salmonella ; SPI ; virulence gene regulation Introduction Horizontal transfer of DNA fragments , encoding for a variety of virulence factors , has been of paramount importance in the evolution of pathogenic bacteria by providing the ability to survive in and exploit and survive in different host niches and thus to cause different diseases ( Ochman et al. , 2000 , Schmidt and Hensel , 2004 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21518393" target="_blank">21518393</a>]<br></li>
<li>When acetyltransferase Pat was removed , the acetylation level and stability of HilD decreased , which would lead to a reduced amount of HilA and downregulation of Salmonella pathogenicity island 1 ( SPI-1 ) -- encoded T3SS apparatus and effectors , resulting in attenuated S. Typhimurium virulence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26810370" target="_blank">26810370</a>]<br></li>
<li>HilD acts in a positive-feed-back loop together with HilC and RtsA to activate HilA and as an activator of ﬂagellar class 1 gene expression through activation of the ﬂhDC P5 promoter , as described in this work .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24488311" target="_blank">24488311</a>]<br></li>
<li>Three AraC/XylS-type regulators , HilC , HilD , and RtsA ( also known as STM4315 ) are important for the activation of SPI-1 in S. Typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21075923" target="_blank">21075923</a>]<br></li>
<li>To determine whether HilE , Pag , or Hha affects repression , rather than derepression , of hilA , the effect that hilE , pag , and hha mutations have on hilA expression should be examined under repressing environmental conditions in strains lacking HilC and HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12933899" target="_blank">12933899</a>]<br></li>
<li>However , as a result of the speciﬁc environmental conditions , hilA expression may be repressed in such a way that HilD and HilC can not derepress it , or HilA activity may be downregulated post-transcriptionally .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>HilC and HilD bind to the hilA promoter region , and it is believed that this binding induces the expression of hilA ( 64 , 71 , 72 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923082" target="_blank">12923082</a>]<br></li>
<li>HilC and HilD can each individually bind to the DNA immediately upstream of hilA , and it is believed that this binding leads to hilA expression ( 12 , 52 , 53 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18245288" target="_blank">18245288</a>]<br></li>
<li>The simplest model that explains our data is that free LCFAs above some threshold concentration in the cytoplasm bind directly to HilD protein to block its ability to-bind DNA and activate transcription , effectively blocking expression of hilD , hilC , rtsA , and hilA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26884427" target="_blank">26884427</a>]<br></li>
<li>Our data suggest that HilC , HilD , and RtsA function in similar manners by binding to the same fragment of hilA to induce its expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923082" target="_blank">12923082</a>]<br></li>
<li>In this model , HilD or HilC expression and activity could be induced upon exposure of S. typhimurium to conditions within the intestinal lumen .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>More recently , H-NS has been shown to repress the expression of HilA , HilD , HilC , and RtsA under noninducing conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23554419" target="_blank">23554419</a>]<br></li>
<li>We then examined whether the accumulation of HilC and HilD under SPI1-repressing conditions leads to enhanced expression of SPI1 genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15661008" target="_blank">15661008</a>]<br></li>
<li>HilC and HilD bind directly upstream of the master regulator gene hilA to induce its expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18245288" target="_blank">18245288</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_458">
Similar to HilC and HilD , RtsA activates the expression of SPI1 genes by binding upstream of the master regulatory gene , hilA , to induce its expression ( 9,17 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17329372" target="_blank">17329372</a>]<br></label>
<input id="act_similarity_458", type="checkbox">
<div class="ACT_similar">
<li>Both HilC and HilD activate expression of SPI-1 genes by binding upstream of the master regulatory gene hilA to induce its expression [ 48 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22291596" target="_blank">22291596</a>]<br></li>
<li>Like HilC and HilD , RtsA activates expression of SPI1 genes by binding upstream of the master regulatory gene hilA to induce its expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16045614" target="_blank">16045614</a>]<br></li></div>
<li>According to our model , these activating signals , both intracellular cellular and environmental , initiate SPI1 gene expression by inducing the expression and activation of HilD through still unknown mechanisms .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20686667" target="_blank">20686667</a>]<br></li>
<li>In this scheme , HilD and HilC indirectly activate expression of the SPI1 TTS apparatus and its secreted substrates by affecting hilA expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>Interestingly , most of the AraC-like regulators that control virulence gene expression , such as HilD , ToxT , VirF , CfaD , RegA , and Rns , act as H-NS antagonists , and consistently , they bind to AT-rich sites ( 47 , 59 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25135218" target="_blank">25135218</a>]<br></li>
<li>Fur directly binds to the hilD promoter and enhances its own expression ( 15 ) , and HilD functions as an activator of SPI-1 expression ( 17 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24643535" target="_blank">24643535</a>]<br></li>
<li>As HilD has also been reported to positively regulate its own expression ( Olekhnovich and Kadner , 2002 ; Ellermeier et al. , 2005 ) , the expression of hilD was also tested in the hilD mutant .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21518393" target="_blank">21518393</a>]<br></li>
<li>HilD , but not HilA or InvF , regulates the expression of gtgE , phoH , sinR , lpxR , SL1896 and SL4247 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27886269" target="_blank">27886269</a>]<br></li>
<li>HilD induces the expression of gtgE , phoH , sinR , lpxR and SL4247 , but not SL1896 , in E. coli MC4100 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27886269" target="_blank">27886269</a>]<br></li>
<li>HilD activates transcription of genes within and outside SPI-1 , including six other TFs .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27601571" target="_blank">27601571</a>]<br></li>
<li>The results described above , indicating that SirA/BarA and CsrB/C activate expression of hilA and ssrAB through HilD , also suggested that CsrA could directly repress hilD expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21518393" target="_blank">21518393</a>]<br></li>
<li>This suggests that SirA regulates expression of rtsAB and hilC via HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16045614" target="_blank">16045614</a>]<br></li>
<li>HilD leads to transcriptional activation of hilC and rtsA , which activate themselves ( 13 , 34 , 41 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18723621" target="_blank">18723621</a>]<br></li>
<li>HilC , HilD , and RtsA have all been shown to-bind directly upstream of hilA , and deletion of hilC , hilD , or rtsA leads to a lower level of hilA expression ( 16 , 18 , 42 , 47 , 43 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
<li>This allowed us to induce rtsA , with concom-itant induction of hilA ( and hilC ) expression , independently of HilD ( Figure 2B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22021388" target="_blank">22021388</a>]<br></li>
<li>Effect of upstream DNA on stimulation of hilA transcription by HilC and HilD proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15126488" target="_blank">15126488</a>]<br></li>
<li>Interestingly , overexpression of HilD still activates expression of invF from pSA14 and pCL463 by approximately 10-fold .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>For example , RtsA and HilD induced expression of rtsA approximately four - to ﬁvefold while HilC induced expression of this gene ~ 30-fold .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16045614" target="_blank">16045614</a>]<br></li>
<li>HilC , HilD , and RtsA differentially induce expression of dsbA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14679226" target="_blank">14679226</a>]<br></li>
<li>RtsA can induce expression of invF independently of HilA , HilC , and HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923082" target="_blank">12923082</a>]<br></li>
<li>'' Levels of active HilD , HilC , RtsA , and HilA are controlled by many different regulators , indirectly impacting expression of other SPI-1 genes ( 9 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27601571" target="_blank">27601571</a>]<br></li>
<li>Thus , regulation of SPI-4 by HilD , HilC , and RtsA likely coordinates expression of SPI-4 and SPI-1 genes , which are required for attachment to and invasion of epithelial cells , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24375101" target="_blank">24375101</a>]<br></li>
<li>RtsA forms a complex regulatory network with HilC and HilD that ultimately activates expression of hilA , which activates components of SPI-1 ( 9 , 11 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26386070" target="_blank">26386070</a>]<br></li>
<li>Here , we report that HilD and HilC can also activate expression of a subset of SPI1 genes independently of HilA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>Production of HilD leads to transcriptional activation of HilC and RtsA , which each activate expression of themselves and of each other .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17208038" target="_blank">17208038</a>]<br></li>
<li>Strong regulation of genes outside SPI-1 was observed , but most of these genes are direct regulatory targets of other SPI-1-associated TFs ( HilD , HilC , RtsA , InvF , and SprB ) , and HilA likely regulates expression of one or more of these TFs .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27601571" target="_blank">27601571</a>]<br></li>
<li>As HilA , HilC , and HilD are DNA-binding regulatory proteins required for SPI1/SPI4 expression , we hypothesized that one of them might directly activate siiABCDEF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18160484" target="_blank">18160484</a>]<br></li>
<li>The expression of HilA is directed by a multi-component feed forward regulatory loop consisting of HilD , HilC , and RtsA , which are AraC-like regulators that can independently activate HilA expression [ 22 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21984608" target="_blank">21984608</a>]<br></li>
<li>Epistasis analysis indicates that Dam-dependent regulation of SPI-1 requires HilD , while HilA , HilC , and InvF are dispensable .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20008574" target="_blank">20008574</a>]<br></li>
<li>Moreover , HilD and HilC activate regulator InvF independently of HilA ( 31 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22004521" target="_blank">22004521</a>]<br></li>
<li>Nature 401:293 -- 297 Hung CC , Garner CD , Slauch JM , Dwyer ZW , Lawhon SD , Frye JG , McClelland M , Ahmer BM , Altier C ( 2013 ) The intestinal fatty acid propionate inhibits Salmonella invasion through the post-translational control of HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27549210" target="_blank">27549210</a>]<br></li>
<li>Thus , some of these H-NS binding sites may be unproductive for the repression of ssrAB , or another regulator may act along with HilD to counteract a double mechanism of H-NS-mediated repression of ssrAB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25135218" target="_blank">25135218</a>]<br></li>
<li>Relative - galactosidase activity is a percentage of untreated strain value in the presence of 100 ng/ml aTet , inducing production of HilD ( 127.16 units ) or RtsA ( 34.42 units ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26884427" target="_blank">26884427</a>]<br></li>
<li>However , repression of SPI1 independently of HilD is a predominant mechanism of Fnr action on the basis of our data .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22021388" target="_blank">22021388</a>]<br></li>
<li>A concentration of 0.01 pmol of each of the 5 = - biotinylated ﬂhDC promoter fragments was incubated with increasing concentrations of puriﬁed HilD protein ( lane 1 , 0 pmol ; lane 2 , 1.06 pmol ; lane 3 , 4.23 pmol ; lane 4 , 8.45 pmol ; lane 5 , 8.45 pmol ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24488311" target="_blank">24488311</a>]<br></li>
<li>Approximately 20 pM of 5 ′ labelled promoter dsDNA fragments ( PinvR * , PhilA * , PlacZ * ) were incubated with increasing concentrations of puriﬁed HilD protein ( 0 , 1 , 5 , 10 , 20 nM ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17971080" target="_blank">17971080</a>]<br></li>
<li>We found that , although the substitution of these 11 lysines greatly decreased sipC expression , over-expression of Pat still caused a reduction of sipC expression proportional to that by the wild type protein ( Fig. 6B ) , demonstrating that these acetyl-lysines are not important for Pat to control SPI1 , and further suggesting that over-expression of Pat does not affect HilD activity through direct N-lysine acetylation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27341691" target="_blank">27341691</a>]<br></li>
<li>HilD also activates expression of the SPI-1-borne invR gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19007416" target="_blank">19007416</a>]<br></li>
<li>HilD also activates expression of the SPI-1-encoded , non-coding InvR RNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17971080" target="_blank">17971080</a>]<br></li>
<li>HilD not only can induce the expression of hilA to activate SPI-1 , but it also promotes SPI-1 expression independent of HilA [ 24 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28329249" target="_blank">28329249</a>]<br></li>
<li>Constitutive expression of such regulators , HilA and HilD , failed to restore the expression levels of most SPI-1-encoded proteins , implicating a post-transcriptional mechanism for SPI-1 inhibition upon H2O2 treatment .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28193547" target="_blank">28193547</a>]<br></li>
<li>This suggests that SirA induction of hilA expression requires the presence of HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16045614" target="_blank">16045614</a>]<br></li>
<li>In support of our model , we demonstrate that SirA induction of hilC , rtsA , hilA and invF requires the presence of HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16045614" target="_blank">16045614</a>]<br></li>
<li>However , like hilA expression , this transcriptional induction required that HilD be present and is consistent with our model for SPI1 regulation ( Fig. 1 ) ; transcription of hilD , transcription of hilC , transcription of rtsA , and transcription of hilA were all increased upon activation of the system .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
<li>Many other regulators encoded outside of SPI-1 affect SPI-1 genes and/or HilA expression through HilD , including HilE ( repression of hilA by binding to and preventing HilD function ) [ 31 ] , FliZ ( activation of hilA via posttranslational regulation of HilD ) [ 32 ] , DNA adenine methylase ( activation of hilA via posttranslational regulation of HilD ) [ 33 ] , EnvZ-OmpR ( activation of hilA via HilD ) [ 34 ] , CpxA/R ( repression of hilA by decreasing the stability of HilD ) [ 35 ] , BarA/SirA ( activation of hilA via activation of csrB/csrC to block CsrA repression of hilD ) [ 24 ] and Fur ( activation of hilA via an unknown regulation of HilD ) [ 26 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28575106" target="_blank">28575106</a>]<br></li>
<li>Earlier work determined that Fur activation of hilA operate through HilD and that overexpression of HilD resulted in Fur-independent induction of hilA ( 22 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21075923" target="_blank">21075923</a>]<br></li>
<li>So , whereas previous results showed that overexpression of HilC induces invF independently of hilA ( Eichelberg et al. , 1999 ; Rakeman et al. , 1999 ) , our results show that overexpression of HilD or HilC can induce invF expression in the absence of hilA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>However , overexpression of either HilC or HilD resulted in constitutive activation of hilA expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12107132" target="_blank">12107132</a>]<br></li>
<li>Because arabinose-induced expression of hilD results in artiﬁcially high levels of HilD that might compensate for posttranscriptional regulation of hilD , our results can not determine which environmental and regulatory pathways regulate hilA expression by modulating hilD expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11292791" target="_blank">11292791</a>]<br></li>
<li>This implies that arabinose induction of hilC or hilD expression results in abnormally high levels of HilC or HilD , respectively , and may produce artiﬁcial situations that permit high-level expression of hilA regardless of any existing repression mechanisms ( see Discussion ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11292791" target="_blank">11292791</a>]<br></li>
<li>Thus , one might predict that in the absence of hilA , overproduction of HilD would activate expression of these SPI1 genes but would have no effect on expression of prgH , prgK and orgA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>In this work , we demonstrated that Fur regulates expression of hilA via control of HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
<li>First , overproduction of Fur signiﬁcantly increases expression of hilA and hilD when HilD is present .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
<li>De-letion of hilE increases expression of hilA only when HilD is present .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18245288" target="_blank">18245288</a>]<br></li>
<li>Inducible expression of either HilC or RtsA protein , but not of Fur , FliZ , or HilD , was sufﬁcient to activate hilA following growth at 42 ° C .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26386070" target="_blank">26386070</a>]<br></li>
<li>This effect was speciﬁc for HilD since overexpression of either RtsA or HilC , the other AraC/XylS type activators of hilA , was sufﬁcient to activate SPI-1 expression at 42 ° C ( Fig. 6C ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26386070" target="_blank">26386070</a>]<br></li>
<li>In the presence of an unknown environmental condition , HilC or HilD may activate invF expression , causing induction of invF-dependent genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10844688" target="_blank">10844688</a>]<br></li>
<li>The latter model requires that chromosomal expression of HilD or HilC be high enough to directly activate invF , independently of hilA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>The direct activation of invF expression by HilC or HilD may always accompany derepression of hilA expression such that the two mechanisms act co-operatively rather than independently to induce invF expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10844688" target="_blank">10844688</a>]<br></li>
<li>In contrast , the absence of HilE , a regulator which negatively controls HilD activity by protein-protein interactions ( Baxter et al. , 2003 ) , does not affect this repression by CpxR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26300871" target="_blank">26300871</a>]<br></li>
<li>Transcription of hilD undergoes autogenous activation by the HilD protein ( Ellermeier et al. 2005 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25991823" target="_blank">25991823</a>]<br></li>
<li>Moreover , FliZ activates HilD that leads to the stimulation of SPI-1 gene expression ( Chubiz et al. , 2010 ; Hung et al. , 2012 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27404739" target="_blank">27404739</a>]<br></li>
<li>HilD , a member of the AraC family of transcriptional-regulators , induces the expression of HilA ( Schechter et al. , 1999 ; Schechter and Lee , 2001 ; Ellermeier et al. , 2005 ) , an OmpR/ToxRlike transcriptional-regulator , which in turn , activates the expression of InvF ( Lostroh et al. , 2000 ; Lostroh and Lee , 2001 ) , another AraC-like regulator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26300871" target="_blank">26300871</a>]<br></li>
<li>FliZ is needed for eficient induction of the Spi-1 system via post-translational activation of the Spi-1 regulatory protein HilD , which in turn activates ﬂhDC gene transcription ( Iyoda et al. , 2001 ; Chubiz et al. , 2010 ; Mouslim and Hughes , 2014 ; Singer et al. , 2014 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26441883" target="_blank">26441883</a>]<br></li>
<li>Furthermore , CsrA regulates expression of the invasion-associated type III secretion system of S. typhimurium via post-transcriptional regulation of HilD ( Altier et al , 2000 ; Fortune et al , 2006 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24056837" target="_blank">24056837</a>]<br></li>
<li>Fur positively regulates HilA , the master regulator of SPI-1 , via HilD ( Ellermeier & Slauch , 2008 ; Teixidó et al. , 2011 ) or via the repression of the histone-like protein H-NS ( Troxell et al. , 2011b ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23306672" target="_blank">23306672</a>]<br></li>
<li>Posttranscriptional activation of flhDC occurs at the level of the flagellar regulator FliZ via regulation of HilD activity and repression of the antiFlhD4C2 factor YdiV ( Chubiz et al. , 2010 ; Wada et al. , 2011 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27206164" target="_blank">27206164</a>]<br></li>
<li>However , we assayed transcription activation by HilD in Escherichia coli , by fusing regions upstream of candidate genes to the lacZ reporter gene ; failure to observe transcription activation by HilD in such experiments does not necessarily indicate that HilD does not regulate these genes , since factors speciﬁc to Salmo-nella may be required for their activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27601571" target="_blank">27601571</a>]<br></li>
<li>Additionally , a DNA fragment containing the intergenic region upstream of ppk , a gene not regulated by HilD , or sigD , a gene not directly regulated by HilD , was included in the binding reactions as an internal negative control .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27886269" target="_blank">27886269</a>]<br></li>
<li>Therefore , we hypothesized that either the ingly increase when H-NS activity was reduced by expressing HilD-HilC-RtsA regulatory cascade or IHF could play a role H-NSTEPEC only if HilC , HilD or IHF acted as an H-NS Downloaded from www.microbiologyresearch.org by IP : 132.248.34.164 On : Mon , 22 Oct 2018 15:15:0 We next tested the role of IHF in stationary phase-dependent hilA induction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21680637" target="_blank">21680637</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_529">
Boddicker , J.D. , Knosp , B.M , and Jones , B.D. ( 2003 ) Transcription of the salmonella invasion gene activator , hilA , requires HilD activation in the absence of negative regulators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27341691" target="_blank">27341691</a>]<br></label>
<input id="act_similarity_529", type="checkbox">
<div class="ACT_similar">
<li>Boddicker , J.D. , Knosp , B.M. and Jones , B.D. ( 2003 ) Transcription of the Salmonella invasion gene activator , hilA , requires HilD activation in the absence of negative regulators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17329372" target="_blank">17329372</a>]<br></li>
<li>Boddicker , J.D. , Knosp , B.M. , and Jones , B.D. ( 2003 ) Transcription of the Salmonella invasion gene activator , hilA , requires HilD activation in the absence of negative regulators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16045614" target="_blank">16045614</a>]<br></li>
<li>Infect Immun 72:2002 -- 2013 Boddicker JD , Knosp BM , Jones BD ( 2003 ) Transcription of the Salmonella invasion gene activator , hilA , requires HilD activation in the absence of negative regulators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19003447" target="_blank">19003447</a>]<br></li></div>
				<label class="ACT_collapse_similarity" for="act_similarity_533">
Transcription of the Salmo-nella invasion gene activator , hilA , requires HilD activation in the absence of negative regulators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23554419" target="_blank">23554419</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25547794" target="_blank">25547794</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24375101" target="_blank">24375101</a>]<br></label>
<input id="act_similarity_533", type="checkbox">
<div class="ACT_similar">
<li>Transcription of the Salmonella invasion gene activator , hilA , required HilD activation in the absence of negative regulators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16443238" target="_blank">16443238</a>]<br></li>
<li>Transcription of the Salmonella invasion gene activator , hilA , requires HilD activation in the absence of negative regulators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19703107" target="_blank">19703107</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18723621" target="_blank">18723621</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19218381" target="_blank">19218381</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23289537" target="_blank">23289537</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14645287" target="_blank">14645287</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18245288" target="_blank">18245288</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20190046" target="_blank">20190046</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15256548" target="_blank">15256548</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26441883" target="_blank">26441883</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12595445" target="_blank">12595445</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923082" target="_blank">12923082</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15126488" target="_blank">15126488</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17675384" target="_blank">17675384</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15731035" target="_blank">15731035</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15039320" target="_blank">15039320</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12933899" target="_blank">12933899</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18441068" target="_blank">18441068</a>]<br></li></div>
<li>Under noninducing conditions , PhoP/PhoQ activation leads to production of HilE , which antagonizes the HilD activator required for SPI1/SPI4 expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18160484" target="_blank">18160484</a>]<br></li>
<li>HilD also activates ﬂhDC , whose gene products are required for activation of the ﬂagellar class 2 promoters [ 22,23 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29380670" target="_blank">29380670</a>]<br></li>
<li>( B ) SprB can activate the PsiiA promoter in the absence of other SPI1 genes ( SPI1 rtsAB ) ; the other SPI1 regulators -- HilA , HilC , HilD , and RtsA -- can not .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20190046" target="_blank">20190046</a>]<br></li>
<li>We also tested whether HilD could activate invR in the absence of any other SPI1 genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17971080" target="_blank">17971080</a>]<br></li>
<li>This induction is dependent upon the master regulator HilA but is independent of HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923082" target="_blank">12923082</a>]<br></li>
<li>This is consistent with the feedforward loop model ; when neither RtsA nor HilC are present , regulation of hilA is dampened , even though the primary signal is mediated through HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16045614" target="_blank">16045614</a>]<br></li>
<li>Under these speciﬁc conditions , HilD is produced through the actions of post-transcriptional-regulatory mechanisms .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17208038" target="_blank">17208038</a>]<br></li>
<li>While the speciﬁc mechanism is unknown , FliZ-dependent activation of SPI1 gene expression is known to occur through HilD ( 41 , 53 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20833811" target="_blank">20833811</a>]<br></li>
<li>However , this effect apparently is not mediated through Lon , the protease known to affect HilD , as in mutants of lon , sipC remained repressed by Pat ( Fig. 4B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27341691" target="_blank">27341691</a>]<br></li>
<li>An alternative possibility is that the negative regulators modulate hilD expression , whereas another unidentiﬁed negative regulator is able to completely repress the hilA promoter in the absence of HilD , although this seems unlikely since many searches for negative regulators have been conducted .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12511499" target="_blank">12511499</a>]<br></li>
<li>We postulate that during the early stages of epithelial cell infection , Spi1 gene expression is induced and motility is downregulated but ﬂagellar genes are in a state ready to be immediately upregulated at some later time during infection by activation via HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24488311" target="_blank">24488311</a>]<br></li>
<li>In addition to HilD , the expression of lpxR in SPI-1-inducing conditions is positively regulated by SlyA , a MarR-like regulator63 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27886269" target="_blank">27886269</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_548">
Consequently , we suggest that SPI1 expression is controlled by the feedback regulatory loop in which 32 induces Lon to control turnover of HilD , and DnaK , which inhibits 32 function , leading to the modulation of lon expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18723621" target="_blank">18723621</a>]<br></label>
<input id="act_similarity_548", type="checkbox">
<div class="ACT_similar">
<li>Taking these ﬁndings together , it is suggested that SPI1 gene expression is regulated by the feedback regulatory loop in which 32 induces Lon protease to control the turnover of HilD and the DnaKJ chaperone machinery , which can inhibit 32 function , leading to the modulation of lon expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18723621" target="_blank">18723621</a>]<br></li></div>
<li>As depicted in Figure 7 , when HilD is acetylated , it is more stable and promotes the transcription of HilA , leading to the activation of SPI-1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26810370" target="_blank">26810370</a>]<br></li>
<li>As a result , SPI-1 is the key point at which many regulatory inputs are processed at the level of HilA to evaluate whether it is appropriate to proceed with the invasion process ( e.g. , most of the 2CRS regulatory systems interact with HilD posttranscriptionally , which then activates HilA ; in turn , HilA transcription is under the control of other regulators , such as several NAPs ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23554419" target="_blank">23554419</a>]<br></li>
<li>Of note , it seems unlikely that the increased degradation of HilD is mediated by changes in protease expression or activity , as the expression of lon is not increased by bile treatment ( 18 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27185788" target="_blank">27185788</a>]<br></li>
<li>By using this strategy , we previously demonstrated that HilD is required for the expression of SsrB in the presence of H-NS but not when it is inactivated ( 21 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25135218" target="_blank">25135218</a>]<br></li>
<li>Previously , we showed that OmpR ( the response-regulator of the OmpR/EnvZ two-component system ) , in addition to HilD , is required for the expression of SsrA/B when Salmonella is grown in LB medium ; moreover , OmpR is required for the expression of SsrA/B even when H-NS is inactivated ( 21 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25135218" target="_blank">25135218</a>]<br></li>
<li>In particular , HilC is expressed in the absence of HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20686667" target="_blank">20686667</a>]<br></li>
<li>The present report suggests that all of the global regulators control SPI1 expression in a HilD-dependent manner , that is , through posttranscriptional or posttranslational control of HilD , which in turn activates hilC and rtsA ( 15 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18723621" target="_blank">18723621</a>]<br></li>
<li>Most SPI1 genes are negatively regulated by HilE through sequestration of HilD , the major transcriptional-activator of SPI1 , while SopA is a secreted effector-protein that is involved in causing inflammation and diarrhea .59,60 IsrM therefore aids in choreographing the expression of virulence factors .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22546935" target="_blank">22546935</a>]<br></li>
<li>S5 in the supplemental material ) , suggesting that the enhanced deg-radation of HilD is speciﬁc and is mediated by changes in HilD rather than protease activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27185788" target="_blank">27185788</a>]<br></li>
<li>Therefore , it is unlikely that DnaKJ is directly involved in the folding of HilD and its absence accelerates the degradation of HilD by proteases .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18723621" target="_blank">18723621</a>]<br></li>
<li>However , for the sake of simplicity , SirA is considered to directly regulate HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19537165" target="_blank">19537165</a>]<br></li>
<li>It is important to note that while HilD is controlled in a complex fashion , rtsA is regulated at the level of transcription via HilD , consistent with the model for SPI1 regulation ( Fig. 1 ) ( 16 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
<li>Transcription of the hilD gene is activated in response to Fur , but this is also dependent on the HilD protein and is in accordance with our model for SPI1 regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
<li>Because HilE is a negative regulator of hilD ( Baxter et al. , 2003 ) , a tentative interpretation for the occurrence of HilE-dependent SPI-1 repression was that LeuO might increase the HilE level , which in turn might inhibit HilD activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24354910" target="_blank">24354910</a>]<br></li>
<li>At concentrations higher than 1.5 μM , MBP-HilD bound most of the DNA fragments tested , including the negative controls , indicating that it binds non-specifically at these concentrations ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27886269" target="_blank">27886269</a>]<br></li>
<li>The data indicated that the activity of the HilD protein was inhibited during growth at 42 ° C .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26386070" target="_blank">26386070</a>]<br></li>
<li>In previous work , Sang et al [ 14 ] demonstrated that the protein acetylation regulates the stability of HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28329249" target="_blank">28329249</a>]<br></li>
<li>Taking all of our data into account , we hypothesize that FliZ controls-the-activity of HilD by direct interaction , but we can not currently rule out indirect effects .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20889744" target="_blank">20889744</a>]<br></li>
<li>In this work , we further characterize the mechanism of FliZ regulation of HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20889744" target="_blank">20889744</a>]<br></li>
<li>On the basis of this experiment , they implied that FliZ regulates HilD at the level of translation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20889744" target="_blank">20889744</a>]<br></li>
<li>These results , taken together , demonstrate that , although propionate reduces HilD stability , the amount of HilD does not itself dictate the repressive effect of this fatty acid , suggesting instead that propionate functions through a post-translational reduction of HilD activity , with consequent degradation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23289537" target="_blank">23289537</a>]<br></li>
<li>PhilD induction was unaffected by the absence of SsrA/B or HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22457642" target="_blank">22457642</a>]<br></li>
<li>The stability of HilD protein is affected slightly by the presence or absence of FliZ .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20889744" target="_blank">20889744</a>]<br></li>
<li>The results show that disruption of ﬂiZ signiﬁcantly decreased the cellular level of HilD even when the corresponding gene was expressed by the PA1lacO-1 promoter system , suggesting that HilD is controlled posttranscriptionally by FliZ .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18245288" target="_blank">18245288</a>]<br></li>
<li>This work demonstrates that the repression of invasion genes by propionate functions specifically through HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23289537" target="_blank">23289537</a>]<br></li>
<li>This suggests that HilD induces the expression of ssrAB also by counteracting H-NS-me-diated repression of its promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25135218" target="_blank">25135218</a>]<br></li>
<li>As a control , the expression of a cat transcriptional fusion of invF , which is positively regulated by HilD through HilA , was also assessed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27886269" target="_blank">27886269</a>]<br></li>
<li>If repression is mediated by modulating HilD activity , high-level expression of hilD might somehow overcome the effects of such modulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11292791" target="_blank">11292791</a>]<br></li>
<li>Therefore , the regulation of SPI-1 expression is negatively controlled through degradation of HilD by Lon [ 10 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28329249" target="_blank">28329249</a>]<br></li>
<li>We propose that binding of HilD upstream of these genes and regulation by HilD occur in a condition-speciﬁc manner .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24375101" target="_blank">24375101</a>]<br></li>
<li>The existence of a LeuO-HilE-HilD ` pathway ' of SPI-1 repression in S. enterica serovar Typhimurium is supported by several lines of evidence : ( i ) lack of HilE relieves LeuO-mediated repression of SPI-1 ( Fig. 2 ) ; ( ii ) activation of leuO transcription decreases the level of HilD protein ( Fig. 3 ) ; ( iii ) the HilD protein decrease caused by LeuO is suppressed by a hilE mutation ( Fig. 3 ) ; and ( iv ) LeuO activates transcription of hilE ( Fig. 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24354910" target="_blank">24354910</a>]<br></li>
<li>To examine whether HilD is acetylated by Pat in vivo , we coexpressed His-tagged HilD with Pat of S. Typhimurium and determined the acetylation level of HilD by means of Western blot analysis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28329249" target="_blank">28329249</a>]<br></li>
<li>Fis-mediated activation of the Spi-1 genes occurs through different regulatory modes : Fis binds to promoters of Spi-1 encoded genes , or controls conserved upstream regulators such as OmpR to inﬂuence the Spi-1 regulator HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26441883" target="_blank">26441883</a>]<br></li>
<li>Consequently , if Fur acts as a simple repressor of the factor that controls HilD , as it does with ryhB controlling sodB , then the Fur binding site controlling the negative regulator must be far from consensus , leading to weak binding of Fur under normal laboratory conditions ; overproduction of Fur leads to further repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
<li>To investigate this possibility , we performed competitive EMSAs to examine the effect of HilD on H-NS bound to the 302 / 478 region of ssrAB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25135218" target="_blank">25135218</a>]<br></li>
<li>HilD binds to sites overlapping or close to most of the sites bound by H-NS and thus displaces H-NS from most of its binding sites , which disrupts the H-NS nucleoprotein complex , thus leading to the expression of ssrAB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25135218" target="_blank">25135218</a>]<br></li>
<li>Promoters with only the DSR region ( e.g. K55 to C505 ) did not respond to activating factors , as shown by the lack of effect of HilC/HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16443238" target="_blank">16443238</a>]<br></li>
<li>SprB has been shown to activate transcription of siiA ( 58 ) , although the same study showed no regulation of siiA by HilA or HilD , in contrast to our data .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24375101" target="_blank">24375101</a>]<br></li>
<li>The PhoQ-PhoP repressive effect on the prg genes is proposed to occur through repression of HilA by means of posttranscriptional repression of HilD , since no direct effect has been reported in these SPI-1 genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23554419" target="_blank">23554419</a>]<br></li>
<li>To detect whether HilD can be deacetylated by CobB , the HilD coexpressed with Pat was treated with CobB in the presence of nicotinamide adenine dinucleotide ( NAD + ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28329249" target="_blank">28329249</a>]<br></li>
<li>One can envision several ways in which HilE could mechanistically regulate the activity of HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29378886" target="_blank">29378886</a>]<br></li>
<li>However , the effect of HilE upon HilD activity does not correlate with the relative levels of HilD ; overexpression of HilE resulted in a 50-fold decrease in hilA expression , while there was only 10 % less HilD protein at the time that activity was measured ( Fig. 7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29378886" target="_blank">29378886</a>]<br></li>
<li>FliZ positively regulates HilD function at the level of HilD protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20889744" target="_blank">20889744</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_593">
FliZ regulates HilD in the absence of Lon protease , previously shown to degrade HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20889744" target="_blank">20889744</a>]<br></label>
<input id="act_similarity_593", type="checkbox">
<div class="ACT_similar">
<li>FliZ regulates HilD in the absence of Lon protease .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20889744" target="_blank">20889744</a>]<br></li></div>
<li>To distinguish whether FliZ regulates hilA via HilD or independently of HilD , we put rtsA under the control of a tetracycline-induc-ible promoter ( tetRA-rtsA ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20889744" target="_blank">20889744</a>]<br></li>
<li>Thus , to distinguish whether factors regulate hilA via HilD , or independently of HilD , we placed rtsA expression under the control of a tetracycline-inducible promoter ( tetRA -- rtsA ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22021388" target="_blank">22021388</a>]<br></li>
<li>Under the conditions used , ssrAB expression depended strongly on HilD , since there was no expression in a hilD-deficient strain regardless of the presence or absence of Gre factors .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28426789" target="_blank">28426789</a>]<br></li>
<li>These data demonstrate that HilD can activate a subset of SPI1 genes in the absence of hilA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>We reasoned that , if HilD is a derepressor of hilA expression , then deleting negative regulators of the hilA gene would increase hilA expression in the absence of HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12511499" target="_blank">12511499</a>]<br></li>
<li>Since neither HilE , Hha , Pag , nor RNase E is solely responsible for repression of hilA : : Tn5lacZY in the absence of HilD , we considered the possibility that these negative regulators function cooperatively to repress hilA expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12511499" target="_blank">12511499</a>]<br></li>
<li>These results seem inconsistent with the idea that HilD simply negates the function of negative regulators of hilA ( i.e. , derepression ) , since HilD is required to obtain wildtype levels of hilA expression even in the absence of several negative regulators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12511499" target="_blank">12511499</a>]<br></li>
<li>In contrast , we found that in every combination of negative regulator mutants examined , hilA expression was very low in the absence of HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12511499" target="_blank">12511499</a>]<br></li>
<li>However , in all strains tested , the absence of hilD resulted in low-level expression of hilA , suggesting that HilD is required for activation of hilA expression , whether or not negative regulators are present .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12511499" target="_blank">12511499</a>]<br></li>
<li>However , the absence of HilD resulted in a very low level of hilA expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22021388" target="_blank">22021388</a>]<br></li>
<li>But under these conditions , deletion of hilE had no effect on tetRA -- rtsA-driven hilA expression in the absence of HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22021388" target="_blank">22021388</a>]<br></li>
<li>When production of a TetR-controlled HilD protein was induced with anhydrotetracycline , addition of 1.2 mM oleate caused a 7-fold decrease in hilA-lac transcription compared to the untreated control .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26884427" target="_blank">26884427</a>]<br></li>
<li>Our genetic results demonstrate that the effects of propionate center on HilD , and that they are posttranslational , with an alteration of protein stability , as might be expected in the case of protein inactivation secondary to chemical modification .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23289537" target="_blank">23289537</a>]<br></li>
<li>To examine our speculation , we performed acetylation modiﬁcation of HilD in vitro and found that the acetylation level of HilD increased dramatically after its incubation with Pat in the presence of acetyl-CoA ( Figure 6C ) , indicating that HilD is a substrate of Pat .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26810370" target="_blank">26810370</a>]<br></li>
<li>Dynamic Acetylation Regulation Mechanism Maintaining the Amount of HilD The discovery here that acetylation of a specific lysine residue in HilD increases protein stability but decreased its DNA-binding ability provides an example of a direct effect of acetylation on a regulatory process in bacteria ( Figure 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28329249" target="_blank">28329249</a>]<br></li>
<li>Gene expression studies conducted at 42 ° C demonstrated reduced activation of the rtsA gene , which is directly activated by HilD ( 15 ) , suggesting a reduction in the level of either HilD or HilD protein activity in response to growth at 42 ° C .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26386070" target="_blank">26386070</a>]<br></li>
<li>The effects of propionate on SPI1 effectors downstream from HilD were disproportionate to that of HilD stability , with , for example , a 40 % reduction in HilD leading to a 10-fold reduction in the expression of hilA ( Fig. 9 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23289537" target="_blank">23289537</a>]<br></li>
<li>Since rtsA , for example , responds to oxygen in the absence of HilD , there must be additional oxygen regulation that is independent of Fur .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
<li>This could be due to condition-speciﬁc binding of HilD upstream of ssrA , since ssrAB expression is induced by HilD in the late stationary phase , well beyond the time point of our ChIP-seq experiment ( 14 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24375101" target="_blank">24375101</a>]<br></li>
<li>To determine the importance of HilD in repression by propionate , we examined the effects of mutations in genes known to encode regulators of HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23289537" target="_blank">23289537</a>]<br></li>
<li>Taken together , our results suggest that the increased hilA expression seen in the absence of Lon is not due to increased stability of HilD but to increased stability of FliZ .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20889744" target="_blank">20889744</a>]<br></li>
<li>The FliZ regulator is known to enhance activity of the HilD protein ( 41 ) through an unknown posttranslational step ; therefore , the inﬂuence of overexpression of FliZ on hilA-lacZ in response to temperature was tested .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26386070" target="_blank">26386070</a>]<br></li>
<li>The effect of deleting hilE is dampened in these backgrounds , likely due to the lack of HilD autoregulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29378886" target="_blank">29378886</a>]<br></li>
<li>In this case overexpression of HilC or HilD may increase hilA expression by overcoming the enhanced repressing mechanism in these mutants .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11755416" target="_blank">11755416</a>]<br></li>
<li>However , as expression of ssrAB requires HilD when Salmonella grows in LB medium , CsrA-mediated regulation of HilD synthesis appears to be the primary mechanism by which CsrA controls ssrAB expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21518393" target="_blank">21518393</a>]<br></li>
<li>Thus , the regulation of hilA expression by HilD may be mediated in part by the effects of HilD on hilC expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11292791" target="_blank">11292791</a>]<br></li>
<li>The requirement of HilD for hilA expression was further studied by monitoring HilA-FLAG expression in the presence or absence of HilD ( Fig 4C ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28426789" target="_blank">28426789</a>]<br></li>
<li>Interestingly , HilD is not required for expression of the ssrAB operon when Salmonella grows in minimal media containing low concentrations of magnesium and phosphate ( Bustamante et al. , 2008 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21518393" target="_blank">21518393</a>]<br></li>
<li>Given the significant impact of HilD on gene expression in S. Typhimurium , HilD-mediated activation of gene expression only occurs when several environmental and physiological cues provide permissive conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28426789" target="_blank">28426789</a>]<br></li>
<li>K297 was one of the acetylated sites in HilD , and acetylated K297 would increase its stability , which was consistent with the model Sang et al [ 14 ] proposed elsewhere .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28329249" target="_blank">28329249</a>]<br></li>
<li>To account for such multi-factorial effects , the rule for H-NS was defined such that H-NS was turned ` OFF ' only in the presence of all five repressors ( HilD , PhoP , SlyA , IHF and Fur ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24079299" target="_blank">24079299</a>]<br></li>
<li>One plausible means by which post-translational control of HilD might be achieved through the metabolic product propionyl-CoA is through protein modification and inactivation as the result of acylation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23289537" target="_blank">23289537</a>]<br></li>
<li>For one of the clusters ( cluster 2 ) , the four genes , ﬂhC , ﬂgM , ﬂgN , and mcpC , are direct targets of either HilD or RtsB ( ﬂhC is regulated by both ; Fig. 7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27601571" target="_blank">27601571</a>]<br></li>
<li>Our model is ﬂexible and can account for either a completely symmetric regulation or an asymmetric one , and incorporate the effect of an external regulation on the system , such as the repression of HilD by HilE , which has been experimentally shown .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21107576" target="_blank">21107576</a>]<br></li>
<li>Transcription of hilD is under the control of an autogenous , positive feedback loop by the HilD product ( Ellermeier et al. 2005 ; Ellermeier and Slauc Figure 3 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20008574" target="_blank">20008574</a>]<br></li>
<li>Thus , although HilD is capable of auto-regulation , the results observed could not be due to the loss of this regulatory protein controlling its own expres ¬ Pat post-transcriptionally regulates HilD The regulation of hilD is complex ; hilD regulates itself but is also regulated by the transcriptional-regulators HilC and RtsA , which constitute a feed-forward circuit to control the downstream SPI1 regulator HilA ( Ellermeier et al. , 2005 ; Ellermeier and Slauch , 2007 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27341691" target="_blank">27341691</a>]<br></li>
<li>In the case of Finally , the rate of change of HilA concentration is dependent on its basal synthesis ( b ; which is a constant ) , 3 activation by the three transactivators ( HilD , HilC and RtsA ) , autorepression as well as its own degradation rate ( a ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19003447" target="_blank">19003447</a>]<br></li>
<li>In addition , Fur is known to activate SPI1 via the activation of the positive regulators of SPI1 ( i.e. , HilA and HilD ) [ 38,39 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22017966" target="_blank">22017966</a>]<br></li>
<li>HilC and HilD also directly activate the alternative promoter of the invF operon , which is independent of HilA ( 15,16 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17329372" target="_blank">17329372</a>]<br></li>
<li>Expression of the SPI1 T3SS is tightly regulated by the combined action of HilC , HilD , and RtsA , three AraC family members that can independently activate hilA , which encodes the direct regulator of the SPI1 structural genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
<li>These results taken together suggest that the HilD , HilC , and RtsA proteins induce the transcription of the rtsA promoter by counteracting H-NS/Hha-mediated repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17675384" target="_blank">17675384</a>]<br></li>
<li>This indicates that the role of the PhoP protein is to activate pmrD transcription and not simply to counteract repression by the PmrA protein on the pmrD promoter as proposed for other regulatory proteins such as HilC and HilD ( 28 , 29 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12676988" target="_blank">12676988</a>]<br></li>
<li>Together , these data suggest that Fur might activate SPI1 by reducing the H-NS -- mediated silencing of the hilD promoter region and thereby lowering the threshold of HilD required to activate the hilD promoter , as we originally proposed ( Ellermeier and Slauch 2008 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22021388" target="_blank">22021388</a>]<br></li>
<li>Whether the regulation by HilD implies that the gtgE , lpxR and sinR genes also have a role in the Salmonella invasion of host cells and thus in the intestinal infection , as for most other genes regulated by HilD , needs to be investigated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27886269" target="_blank">27886269</a>]<br></li>
<li>Expression was dependent on HilD , HilC , and RtsA , as evidenced by the lack of activity in the spi1 rtsA deletion strain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26884427" target="_blank">26884427</a>]<br></li>
<li>However , in vitro transcription was strongly decreased by the T-88 substitution , which should impair both HilC and HilD binding and the activity of the upstream extended 10 promoter element .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12107132" target="_blank">12107132</a>]<br></li>
<li>In addition , the DNA binding sites of the HilD protein have been mapped and include several genes that are coregulated by HilC and RtsA ( 15 ) , suggesting that reduced activation by one of the three activators may inﬂuence the activation of coregulated genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26386070" target="_blank">26386070</a>]<br></li>
<li>In all cases , increased HilC and HilD levels consequent on the activation of the corresponding genes by IPTG resulted in increased amounts of the SPI1 products , HilA , InvF and SipC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15661008" target="_blank">15661008</a>]<br></li>
<li>The most speculative aspect of the model concerns the mechanism for activation of the SPI1 promoters -- PhilA , PhilC , PhilD , and PrtsA - by HilC , HilD , and RtsA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20686667" target="_blank">20686667</a>]<br></li>
<li>Expression of HilD and SirA from plasmids pBAD-HilD1 and pBAD-SirA1 , respectively , was induced by adding 0.1 % L-arabinose to the medium at the beginning of the bacterial cultures .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21518393" target="_blank">21518393</a>]<br></li>
<li>Anyway , HilD would integrate the regulation of Salmonella virulence genes to the stresses sensed by CpxR/A and RpoH .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26300871" target="_blank">26300871</a>]<br></li>
<li>Thus , if [ HilD ] , [ HilC ] , [ HilA ] , [ RtsA ] and [ SirA ] represent the protein levels of the corresponding transactivators , then the three equations corresponding to the basic model are as follows : dt d [ RtsA ] = b2 + β2 · f ( [ HilD ] ) − α2 · [ RtsA ] dt d [ HilA ] = b3 + β3 · f ( [ HilD ] , [ HilC ] , [ RtsA ] , [ HilA ] , [ SirA ] ) − α3 · [ HilA ] dt In all the above equations , [ HilD ] and [ SirA ] can each be either 1 or 0 , depending on the presence or absence of the signal for induction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19537165" target="_blank">19537165</a>]<br></li>
<li>Model 7 , which considers the dependencies of the intermediate components HilC and RtsA on HilD and SirA to function as an OR gate , successfully predicts the levels of intermediate components under all conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19537165" target="_blank">19537165</a>]<br></li>
<li>Discussion Expression of the SPI1 T3SS is controlled by HilD , HilC , and RtsA , acting in a complex feed-forward loop to activate the hilD , hilC , and rtsA genes , as well as hilA , which encodes the transcriptional-activator of the T3SS structural genes ( Figure 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22021388" target="_blank">22021388</a>]<br></li>
<li>Model 6 , which considers the dependencies of the intermediate components HilC and RtsA on HilD and SirA to function as an AND gate , fails in predicting the levels of both these components in response to the sirA knock-out , as well as the hilD knock-out .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19537165" target="_blank">19537165</a>]<br></li>
<li>Fragment A-b , which carries the promoter and transcribed sequences ( 60 to 450 ) , bound HilC well and HilD less well , as shown by the loss of the band corresponding to free DNA , but most of the bound complexes remained in the wells .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12107132" target="_blank">12107132</a>]<br></li>
<li>Repression of the sulA-lacZY reporter caused by dimerization of the LexA1-87408-HilE fusion protein and the LexA1-87WT fusion protein Mean Fold repression - galactosidase of - galactosidase activity from from sulA-lacZY the sulA-lacZY ( wild type vs reporter SD experimental ) TA Strain SU202 ( sulA-lacZY ) 2,503 2,694 2,789 523 47 1 SU202/pLexA - HilE pDP804 SU202/pLexA-HilD pMS604 SU202/pLexA - HilD pLexA-HilD 102 98 56 0.93 0.90 4.8 FIG. 7 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12595445" target="_blank">12595445</a>]<br></li>
<li>This suggested that the activity of the HilD protein was regulated by growth at 42 ° C and not by the level of the protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26386070" target="_blank">26386070</a>]<br></li>
<li>Subsequent experiments showed that the enhancement of cellular levels of HilC and HilD by clpPX disruption seemed to be caused by an increase in the ﬂagellar protein FliZ , encoded by the class 2 ﬂiAZY operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18245288" target="_blank">18245288</a>]<br></li>
<li>As shown by the results in Fig. 2B , we observed binding of increasing concentrations of puriﬁed HilD protein to DNA fragments comprising the P5 promoter ( fragment 3 and 3 =) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24488311" target="_blank">24488311</a>]<br></li>
<li>DNA-binding activity was calculated by measuring the fraction of each labeled DNA fragment that was shifted in the presence of 100 to 300 nM HilC and HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12107132" target="_blank">12107132</a>]<br></li>
<li>Slauch and colleagues proposed two models to explain Fur activation of Spi1 : either by indirect control of the HilD protein or by affecting the threshold value of HilD concentration required to activate the hilD promoter ( Ellermeier & Slauch , 2008 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27704705" target="_blank">27704705</a>]<br></li>
<li>We demonstrate that the effects of bile on invasion are mediated via the SPI-1 transcriptional-regulator HilD and , similar to SCFAs , bile acids affect HilD by posttranslational destabilization of the HilD protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27185788" target="_blank">27185788</a>]<br></li>
<li>( C ) Third , test the effects of a regulatory factor on hilD -- lac transcriptional and hilD ' - ' lac translational locus fusions , which provide a readout of hilD transcription and translation , respectively , in the absence of HilD autoinduction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22021388" target="_blank">22021388</a>]<br></li>
<li>To determine specifically how propionate might affect hilD , we next examined its effects on transcription in the presence and absence of functional HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23289537" target="_blank">23289537</a>]<br></li>
<li>To experimentally examine the HilD derepression model , we determined the effect of single or multiple mutations in the negative regulator elements hha , hilE , pag , and ams on hilA expression , with or without functional HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12511499" target="_blank">12511499</a>]<br></li>
<li>To further test the requirement for HilD for Fur-mediated regulation of hilA , we examined the effects of Fur overexpression in strains containing the hilA-lac fusion , with and without hilC , hilD , or rtsA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
<li>expression from a plasmid in in vitro transcription assays or in the absence of the URS suggest that HilC and HilD can de-repress hilA expression , possibly by antagonizing the repressing effects of proteins bound to the URS ( 21 , 23 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12933899" target="_blank">12933899</a>]<br></li>
<li>The 2 -- 3-fold decrease in hilD expression caused by the mlc mutation is suficient to account for the nearly 10-fold decrease in hilA and invFA transcription , owing to the nature of the feed-forward regulatory loop , in that moderate effects on HilD production are ampliﬁed ( 9 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17329372" target="_blank">17329372</a>]<br></li>
<li>This minimal effect upon changes in the C2 protein-binding region was consistent with the in vitro results that expression of the hilC promoter was comparable in the absence or presence of HilC and HilD , even though a different start site was used .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12107132" target="_blank">12107132</a>]<br></li>
<li>This strain , whose construction is described in materials and methods , carries a conditional hilD mutation that renders the strain HilD in the absence of tetracycline and HilD1 in the presence of either tetracycline or autoclaved chlortetracycline .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20008574" target="_blank">20008574</a>]<br></li>
<li>This result and the fact that the increase in hilA expression seen in the lon mutant i dependent on FliZ suggest that the primary consequence of losing Lon is increased stability of FliZ , which in turn activates HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20889744" target="_blank">20889744</a>]<br></li>
<li>Western blot analysis revealed that activation of leuO transcription decreased the HilD level , and the decrease was suppressed in a ΔleuO background ( Fig. 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24354910" target="_blank">24354910</a>]<br></li>
<li>We also observed that a hilE null mutation increased the HilD levels in the presence and in the absence of LeuO ( Fig. 3 ) , in accordance with two well known facts : the inhibition of HilD activity by HilE ( Baxter et al. , 2003 ) and the occurrence of autogenous activation of hilD transcription ( Ellermeier et al. , 2005 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24354910" target="_blank">24354910</a>]<br></li>
<li>The ﬁnding that the dnaKJ disruption resulted in an increase in the amount of 32 and therefore stimulated the transcription of lon in S. enterica serovar Typhimurium raises the possibility that the cellular level of HilD may be controlled by the 32-initiated regulatory loop and this regulates SPI1 expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18723621" target="_blank">18723621</a>]<br></li>
<li>Coordinated regulation of the ﬂagellar regulon and SPI-1 in a FliZ-dependent manner was also demonstrated to be regulated by the ClpXP protease , affecting the turnover of the FlhD4C2 complex , FlhD4C2 in turn controls FliZ expression , which regulates the cellular levels of HilD ( 50 , 51 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26056383" target="_blank">26056383</a>]<br></li>
<li>Therefore , the increased transcription of hilC and hilD in the clpPX cells could be explained by ampliﬁcation of the regulatory loop in which FliZ triggered an increase in the amount of HilD via posttranscriptional control .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18245288" target="_blank">18245288</a>]<br></li>
<li>On the other hand , partial loss in activities would suggest the presence of factors other than HilD and SirA , inﬂuencing their induction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19537165" target="_blank">19537165</a>]<br></li>
<li>To evaluate the role of the activators HilD and HilC in the regulation of rtsA transcription , the single-copy rtsA-lac transcriptional fusion was tested in the hilD hilC background in the presence and absence of hha and hns mutations ( Fig. 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17675384" target="_blank">17675384</a>]<br></li>
<li>In this report , we investigate the role of nucleoid-associated proteins Hha and H-NS in the expression of virulence activator proteins HilD , HilC , and RtsA with an emphasis on the interplay of positive and negative transcriptional factors in the regulation of the rtsA promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17675384" target="_blank">17675384</a>]<br></li>
<li>As a consequence , in the model proposed by Ellermeier and Slauch , HilD , the most important activator of HilA in vitro , is at the top of the hierarchy and activates the transcription of HilC and RtsA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23554419" target="_blank">23554419</a>]<br></li>
<li>To distinguish whether HilE regulates hilA via HilD , we induced hilA ( and hilC ) transcription in the presence or absence of HilD by the addition of increasing concentrations of tetracycline in a tetRA -- rtsA background .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22021388" target="_blank">22021388</a>]<br></li>
<li>HilD activates transcription of hilA and other target genes in response to environmental conditions associated with the intestinal microenvironment of the host .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24375101" target="_blank">24375101</a>]<br></li>
<li>However , importantly , production of LeuO decreases hilD transcription in the absence of HilD , decreases hilC and rtsA transcription in the absence of both HilD and HilE , and leads to an invasion defect in the absence of HilE ( 33 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29378886" target="_blank">29378886</a>]<br></li>
<li>A decrease in transcription was observed in the hilC-D background , but this was most likely due to the effect of the loss of HilC and HilD on transcription of hilA and invF ( 2 , 65 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14679226" target="_blank">14679226</a>]<br></li>
<li>HilD induces the expression of gtgE , phoH , sinR , lpxR and SL4247 , but not that of SL1896 , in the absence of other Salmonella-specific regulators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27886269" target="_blank">27886269</a>]<br></li>
<li>To investigate whether HilD induces the expression of SL1896 through FlhDC , a ΔflhDC mutant derivative of S. Typhimurium SL1344 was constructed and the activity of the SL1896-cat fusion was determined in this strain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27886269" target="_blank">27886269</a>]<br></li>
<li>As expected , activation of hilA promoter in the absence of the activators HilD and HilC ( lines 5 -- 8 ) was lost or signiﬁcantly reduced in inducing ( high osmolarity ) conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16443238" target="_blank">16443238</a>]<br></li>
<li>The presence of CsrB restores the expression of hilD , hilA and ssaG in the absence of SirA but not in the absence of HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21518393" target="_blank">21518393</a>]<br></li>
<li>Effect of a hilE mutation on the levels of HilD protein in the presence and in the absence of LeuO .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24354910" target="_blank">24354910</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_685">
( B ) - Galactosidase activity of slrP : : lacZ in the presence and in the absence of LeuO or HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25182488" target="_blank">25182488</a>]<br></label>
<input id="act_similarity_685", type="checkbox">
<div class="ACT_similar">
<li>Effect of HilD expression on the - galactosidase activity of slrP : : lacZ in the presence and in the absence of Lon and LeuO .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25182488" target="_blank">25182488</a>]<br></li>
<li>( B ) Effect of Lon on slrP expression in the presence and in the absence of HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25182488" target="_blank">25182488</a>]<br></li>
<li>( C ) Effect of LeuO on slrP expression in the presence and in the absence of HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25182488" target="_blank">25182488</a>]<br></li></div>
<li>To determine the effect of bile on HilD stability in S. Typhi , S. Typhi Ty2 was transformed with constitutively expressed hemagglutinin ( HA ) - tagged HilD ( from S. Typhi Ty2 ) and subcultured in the presence or absence of bile , and samples were taken at regular intervals following the inhibition of protein synthesis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29229736" target="_blank">29229736</a>]<br></li>
<li>Next we investigated possible posttranslational effects of bile and its components on HilD by assessing the stability of HilD in the presence of bile or cholate in a half-life quantitation assay .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27185788" target="_blank">27185788</a>]<br></li>
<li>HilD is a known activator of hilA transcription ; HilA then activates the transcription of genes encoding the Spi1 type III secretion apparatus and Spi1 effector-proteins through InvF ( 24 , 59 , 60 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24488311" target="_blank">24488311</a>]<br></li>
<li>S2 in the supplemental material , we could rule out an effect of HilA on ﬂagellar gene transcription and thereby conﬁrmed that the observed FIG 1 HilD activates ﬂhDC transcription via P5 of ﬂhDC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24488311" target="_blank">24488311</a>]<br></li>
<li>CpxR could directly repress the transcription of hilD or reduce post-transcriptionally the concentration of HilD and thus affect its positive autoregulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26300871" target="_blank">26300871</a>]<br></li>
<li>HilD could act to either repress rﬂM , the inhibitor of ﬂhDC transcription , or activate ﬂhDC expression and overcome the inhibition of ﬂhDC via RﬂM .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24488311" target="_blank">24488311</a>]<br></li>
<li>Thus , depending on the environmental niche and spatiotemporal stage of infection , the Spi1 regulator HilD could activate ﬂhDC transcription through direct binding to the ﬂhDC promoter or repress ﬂhDC transcription via activation of RtsB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24488311" target="_blank">24488311</a>]<br></li>
<li>Since HilD activates the transcription of hilA ( 14 ) , which in turn can activate HilA-dependent invF expres-A sion ( 10 ) , and directly activates HilC/D-dependent invFD expression ( 16 ) , these results establish that the mlc mutation exerts a negative effect on SPI1 gene expression , mainly by increasing the level of hilE expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17329372" target="_blank">17329372</a>]<br></li>
<li>Furthermore , we demonstrate that HilD is required for the expression of the gtgE , phoH , sinR , lpxR , SL1896 and SL4247 genes , but not for the SL1028 , SL3812 and SL4433 genes , when S. Typhimurium SL1344 is grown in conditions that favor the expression of the SPI-1 genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27886269" target="_blank">27886269</a>]<br></li>
<li>Under this growth condition the SPI-1-encoded regulator HilD activates the sequential expression of the SPI-1 and SPI-2 regulons at the early and late stationary phases of growth , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21518393" target="_blank">21518393</a>]<br></li>
<li>In the absence of Dam methylation , the HilD level is lower , and SPI expression decreases .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20008574" target="_blank">20008574</a>]<br></li>
<li>( A ) HilD is at the top of the SPI-1-regulatory hierarchy , where it regulates its own expression and the expression of HilA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29229736" target="_blank">29229736</a>]<br></li>
<li>SPI-1 expression is repressed by hns and activated by a complex positive feedback loop where the production of the HilD regulatory protein induces the expression of HilA , a transcription factor that directly activates expression of the TTSS and effector-proteins [ 70 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25375226" target="_blank">25375226</a>]<br></li>
<li>Interestingly , FlhDC induces the expression of FliZ that somehow increases activity of HilD and thus the expression of the SPI-1 genes36 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27886269" target="_blank">27886269</a>]<br></li>
<li>However , the effects of Mlc on hilD expression and bacterial invasiveness were not seen in the hilE mutant , and hilE expression was increased in the mlc mutant , which suggests that Mlc exerts positive effects on the expression of SPI1 genes by reducing the expression of HilE , which is known to downregulate the expression of SPI1 genes through direct interaction with HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17329372" target="_blank">17329372</a>]<br></li>
<li>These results suggest that CpxR represses hilA and thus the other SPI-1 genes by affecting the autoregulation of HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26300871" target="_blank">26300871</a>]<br></li>
<li>The BarA/SirA two-component system activates hilA by controlling HilD expression posttranscriptionally via activation of the small RNAs CsrB and CsrC , which antagonize the CsrA RNA binding protein that presumably acts directly at the hilD mRNA ( 2 , 3 , 20 , 28 , 49 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
<li>This reflects the fact that many activating signals , both environmental and intracellular , affect SPI1 gene expression by modifying the activity of HilD protein [ 16,18,19,23,34,35 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20686667" target="_blank">20686667</a>]<br></li>
<li>Consequently , we propose that the expression of SPI1 is negatively regulated by controlling the turnover of the HilC and HilD transcriptional-regulators by Lon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15661008" target="_blank">15661008</a>]<br></li>
<li>Our previous ﬁnding that it negatively regulates the expression of the ﬂagellar regulon by controlling the turnover of the master regulator , FlhD/FlhC ( 62 ) , allowed us to hypothesize that it may regulate HilC and HilD by controlling ﬂagellar gene expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18245288" target="_blank">18245288</a>]<br></li>
<li>Finally , we found that the transcriptional-regulator HilD within SPI-1 was a substrate of Pat and that the stabilization of HilD by Pat is important for the transcriptional activation of HilA and its downstream genes in SPI-1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26810370" target="_blank">26810370</a>]<br></li>
<li>Concerning SPI-1 regulation , since similar activating effects are seen for HilA , HilD , and HilC , it is possible that IHF activates SPI-1 genes at the level of HilD , which can then inﬂuence expression of the downstream regulators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23554419" target="_blank">23554419</a>]<br></li>
<li>In the absence of Lon , HilD protein is not degraded and therefore accumulates irrespective of SCFA treatments .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27185788" target="_blank">27185788</a>]<br></li>
<li>In view of these ﬁndings , we suggest that the regulation of SPI1 expression is negatively controlled through deg-radation of the HilC and HilD transcriptional-regulators by Lon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15661008" target="_blank">15661008</a>]<br></li>
<li>On the basis of these results , we propose that Lon negatively regulates the expression of SPI1 genes through degradation of HilC and HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15661008" target="_blank">15661008</a>]<br></li>
<li>In addition , we found that when the activity of H-NS is inactivated ( Fig. 2 and 4 ) or when the 119 / 336 region is de-leted ( Fig. 3 and 4 ) , full expression of ssrAB becomes independent of HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25135218" target="_blank">25135218</a>]<br></li>
<li>In agreement with these results , we previously showed that full expression of the chromosomally encoded SsrA and SsrB proteins requires HilD in the presence of H-NS but not when it is inactivated ( 21 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25135218" target="_blank">25135218</a>]<br></li>
<li>Additional signals , such as the concentrations of short-chain fatty acids ( i.e. , acetate , propionate , and butyrate ) , as well as the presence of bile salts , impact gene expression ( 9 , 12 , 53 ) , with recent evidence showing that propionyl coenzyme A ( propi-onyl-CoA ) speciﬁcally regulates HilD posttranslationally , possibly by propionylation of the HilD protein ( 54 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25547794" target="_blank">25547794</a>]<br></li>
<li>Moreover , on deletion of the pag gene , located within pagK and pagM , increased hilA expression is observed only in the presence of HilD ( 232 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23554419" target="_blank">23554419</a>]<br></li>
<li>Interestingly , three of the mutations -- rpoA151 , rpoA153 , and rpoA154 -- caused 2-fold increases in hilA expression that were dependent on the presence of HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12511499" target="_blank">12511499</a>]<br></li>
<li>To conﬁrm that CpxR regulates hilA through HilD and not directly , we analyzed the effect of CpxR on the expression of hilA in the presence or not of HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26300871" target="_blank">26300871</a>]<br></li>
<li>Based on these observations , we suggest that Lrp acts as an antagonist of HilD , which induces expression of the hilA gene by derepressing the silencing effect of H-NS and Hha on the hilA promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19074398" target="_blank">19074398</a>]<br></li>
<li>Previous studies indicate that HilD induces the expression of hilA by counteracting the repression exerted by the nucleoid protein H-NS on the promoter of this gene ( Schechter et al. , 1999 ; Schechter and Lee , 2001 ; Olekhnovich and Kadner , 2006 ) ; thus , in the absence of H-NS activity hilA can be expressed independently of HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26300871" target="_blank">26300871</a>]<br></li>
<li>Recent work has demonstrated that both HilD and HilC bind to sequences upstream of the hilA promoter ( 39 , 44 ) and that the presence of HilD is required for hilA activation even in the absence of several negative regulatory elements ( 9 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15731035" target="_blank">15731035</a>]<br></li>
<li>) b-galactosidase Activity b-galactosidase Activit SirA induction of hilA and invF requires HilD Previous evidence suggested that SirA required the HilC / HilD/RtsA binding sites in the hilA promoter to induce expression of hilA and thus the entire SPI1 TTSS b-galactosidase Activity b-galactosidase Activit Fig. 5 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16045614" target="_blank">16045614</a>]<br></li>
<li>SirA , but not HilD , requires the presence of CsrB and CsrC to induce secretion of SPI-1-encoded proteins and expression of the SPI-2 effector-protein SseB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21518393" target="_blank">21518393</a>]<br></li>
<li>Moreover , the demonstration that CsrA binds to the SD sequence and translation initiation codon of hilD mRNA , but not to hilA mRNA ( Fig. 9 ) , strongly correlates with the genetic and biochemical data indicating that regulation of the SPI-1 and SPI-2 regulons require direct activation of csrB and csrC by SirA , which in turn overcomes CsrA-dependent repression of HilD synthesis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21518393" target="_blank">21518393</a>]<br></li>
<li>This reduction in - galactosidase activity is indicative of a proteinprotein interaction between HilE and HilD that leads to dimerization of the LexA DNA-binding proteins and repression of the sulA promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12595445" target="_blank">12595445</a>]<br></li>
<li>Additionally , we show that HilD can induce the expression of gtgE-cat , phoH-cat , sinR-cat , lpxR-cat , SL3812-cat and SL4247-cat , but not SL1896-cat , transcriptional fusions , in the E. coli MC4100 strain , and thus in the absence of other Salmonella-specific regulators or FlhDC ; consistently , HilD bound to the regulatory regions of gtgE , phoH , sinR , lpxR and SL4247 , but not to that of SL1896 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27886269" target="_blank">27886269</a>]<br></li>
<li>Furthermore , CsrB was able to restore the expression of the SPI-1 and SPI-2 genes in the absence of SirA , but not in the absence of HilD ( Fig. 6 ) , indicating that SirA acts through CsrB to regulate these genes , and that HilD is downstream of SirA and CsrB in the regulatory cascade controlling them .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21518393" target="_blank">21518393</a>]<br></li>
<li>These results show that HilD is able to activate the expression of ssrAB and hilA in the absence of SirA , whereas SirA requires the presence of HilD , suggesting that SirA controls HilD expression , which in turn directly regulates hilA and ssrAB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21518393" target="_blank">21518393</a>]<br></li>
<li>In contrast , HilD activated expression of ssrAB and hilA in the absence of SirA , indicating that HilD does not require the presence of SirA to directly activate expression of the SPI-1 and SPI-2 regulatory genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21518393" target="_blank">21518393</a>]<br></li>
<li>Taken together , these results show that CpxR represses the autoregulation of HilD , which in turn affects the expression of hilA and thus the SPI-1 genes , as well as of other virulence genes regulated by HilD , such as ssrB and sseB located in SPI-2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26300871" target="_blank">26300871</a>]<br></li>
<li>In agreement with our results indicating that CpxR represses the HilD-dependent expression of hilD and hilA , both the overexpression of CpxR and the absence of CpxA drastically reduced the production of SsrB-FLAG and SseB proteins ( Figures 1C , 3B ) , which are encoded in SPI-2 and whose expression is also dependent of HilD in the condition tested .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26300871" target="_blank">26300871</a>]<br></li>
<li>moreover , we demonstrate that CpxR-P negatively affects the stability of HilD and thus decreases the expression of HilD-target genes , such as hilD itself and hilA , located in SPI-1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26300871" target="_blank">26300871</a>]<br></li>
<li>Furthermore , we demonstrate that CpxR-P negatively affects the transcription of the SPI-1 genes hilD and hilA , but only in the presence of HilD , which would be expected , since the expression of HilD is autoregulated and HilD directly regulates hilA ( Golubeva et al. , 2012 ; Fàbrega and Vila , 2013 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26300871" target="_blank">26300871</a>]<br></li>
<li>In experiments under conditions resembling the intracellular environment , HilD was not required for activation of the Spi2 regulon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24488311" target="_blank">24488311</a>]<br></li>
<li>The longer fragment carries the promoter for prgH , and it is possible that stable HilD binding to the hilD promoter requires the presence of this divergent upstream promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12107132" target="_blank">12107132</a>]<br></li>
<li>HilD activates HilA , which in turn upregulates the expression of genes encoding the TTSS , such as proteins encoded by the prg-org and the inv-spa operons ( 6 , 17 , 18 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28439039" target="_blank">28439039</a>]<br></li>
<li>Thus , it might be deduced that once inside the SCV , PhoP activates the pag genes , which in turn reduce hilA transcription through HilD and eventually trigger repression of the prg genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23554419" target="_blank">23554419</a>]<br></li>
<li>Other features that need to be taken in account when assessing hilD transcription are the presence of an intact hilD ORF , as the HilD protein positively regulates its own expression [ 15 ] and the presence of the newly described regulatory motif hilD 3 ' - UTR , that modulates hilD mRNA stability by promoting its rapid degradation [ 29 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28426789" target="_blank">28426789</a>]<br></li>
<li>HilD also controls the flagella regulon ( FliZ ) ( 3 ) , implying that the motility decrease in STP22 ~ and STP22 + treated with CEOs can be attributed to the repression of hiID .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24780330" target="_blank">24780330</a>]<br></li>
<li>However , in striking contrast to class I regulators , both the hilD -- lac transcriptional and the hilD ' - ` lac translational fusions were regulated by SirA in the absence of HilD protein ( Figure 4C ) , consistent with the RNA-binding protein CsrA acting at the level of the hilD mRNA to control stability or translatability .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22021388" target="_blank">22021388</a>]<br></li>
<li>An alternative explanation may be that the phenotype conferred by the Δspi2 mutation in not decreasing intestinal colonization results from the absence of SPI1 regulators , such as HilD , that are known to regulate SPI2 genes , including the SsrAB central regulator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19126220" target="_blank">19126220</a>]<br></li>
<li>Although residual SPI-1 repression by LeuO is observed in the absence of HilE , the LeuO-HilE-HilD ` pathway ' appears to be the major mechanism .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24354910" target="_blank">24354910</a>]<br></li>
<li>For target genes regulated by multiple TFs , the level of regulation shown is for the highest TF in the hierarchy HilD HilC RtsA InvF RtsB SprB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27601571" target="_blank">27601571</a>]<br></li>
<li>One could argue that HilD is the most active of the three regulators , but this could be misleading .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16045614" target="_blank">16045614</a>]<br></li>
<li>the stability of the dominant SPI‐1 regulator HilD is increase 2018 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=30030897" target="_blank">30030897</a>]<br></li>
<li>In particular , the free concentration of p1 depends on the concentration e of HilE : the more HilE is present , the more HilD is bound , and therefore removed from the system .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21107576" target="_blank">21107576</a>]<br></li>
<li>) However , the absence of HilE had no effect on the hilD -- lac transcriptional and hilD ' - ` lac translational fusions , showing that the presence of the functional HilD protein is required for regulation ( Figure 3C ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22021388" target="_blank">22021388</a>]<br></li>
<li>Since HilD activates hilA transcription , which in turn activates the transcription of the InvF transcriptional-regulator ( 2 ) , it was surprising that reduced transcription of the central regulator HilA was not detected by microarray analysis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17060472" target="_blank">17060472</a>]<br></li>
<li>To determine how such post-translational control could be achieved , we examined whether the stability of HilD was affected by growth in the presence of propionic acid .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23289537" target="_blank">23289537</a>]<br></li>
<li>As shown in Fig. 9B , the decreased effect of dnaKJ disruption o the cellular level of HilD was abolished by introducing a lon mutation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18723621" target="_blank">18723621</a>]<br></li>
<li>Thus , although the half-life of HilD is slightly affected by changes in FliZ levels , this does not explain the mechanism of regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20889744" target="_blank">20889744</a>]<br></li>
<li>Protein stability assays demonstrated that in S. Typhi the half-life of HilD , the dominant regulator of SPI-1 , is three times longer in the presence of bile ; this increase in stability was independent of the acetyltransferase Pat .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29229736" target="_blank">29229736</a>]<br></li>
<li>Interestingly , despite the fact that Fnr is a negative regulator of hilA expression independent of HilD , the hilD ' - ` lac translational fusion showed a slight decrease in activity in the absence of Fnr .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22021388" target="_blank">22021388</a>]<br></li>
<li>Determining the mechanism of Fur activation of HilD is an important step in understanding the regulation of the SPI1 T3SS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
<li>However , our data suggest that the half-life of HilD is only slightly increased in the absence of Lon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20889744" target="_blank">20889744</a>]<br></li>
<li>Note that , in the absence of HilD , the activation of the PhilC promoter is no longer switch-like but instead continuous .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20686667" target="_blank">20686667</a>]<br></li>
<li>To examine whether FliZ is involved in the posttranscriptional control of HilD , we decided to compare the levels of HilD protein in ﬂiZ and ﬂiZ cells where hilD is expressed under the regulation of the PA1lacO-1 promoter system .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18245288" target="_blank">18245288</a>]<br></li>
<li>To examine this possibility , the cellular level of HilD was determined in 32-overproducing cells in which rpoH , encoding 32 , is induced under the control of the PA1/lacO -1 promoter system .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18723621" target="_blank">18723621</a>]<br></li>
<li>Expression of all three genes was found to decrease in the presence of L-arabinose in a HilD + background ( Figure 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25991823" target="_blank">25991823</a>]<br></li>
<li>The analysis of the other intermediate , namely RtsA , indicates a complete loss in the levels of RtsA in case of model 1 for the hilD knock-out , contradicting the experimental observation of a 4-fold decrease in RtsA in response to mutation in HilD [ Ellermeier et al. , 2005 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19537165" target="_blank">19537165</a>]<br></li>
<li>HilD remains stable in the presence of HilE .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29378886" target="_blank">29378886</a>]<br></li>
<li>Additionally , bacteria grown in the presence of propionate demonstrated a significantly reduced HilD half-life .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23289537" target="_blank">23289537</a>]<br></li>
<li>As seen in the otherwise-wild-type background , Fig. 4A demonstrates a considerably reduced HilD half-life in the presence of bile or cholate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27185788" target="_blank">27185788</a>]<br></li>
<li>To investigate this , we determined the in vivo half-life of HilD in the presence or absence of CpxA or Lon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26300871" target="_blank">26300871</a>]<br></li>
<li>In the case of the PhilD promoter , we observed a weak , step-like increase in activity in the absence of HilD ( Figure 2A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20686667" target="_blank">20686667</a>]<br></li>
<li>In the absence of lon , HilD protein accumulated to high levels in bacteria grown in the presence of cholate ( Fig. 4A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27185788" target="_blank">27185788</a>]<br></li>
<li>In LB the half-life of HilD was 14 min , while in bile the half-life of HilD increased to 40 min , indicating that HilD is approximately three times more stable in the presence than in the absence of bile in S. Typhi ( Fig. 6A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29229736" target="_blank">29229736</a>]<br></li>
<li>To test whether the DnaKJ chaperone machinery is involved in the posttranslational control of HilD , we decided to determine the in vivo half-life of HilD in the presence or absence of FIG. 2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18723621" target="_blank">18723621</a>]<br></li>
<li>DISCUSSION In this study , we sought to better characterize hilA regulation by investigating the model of HilD derepression of the hilA promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12511499" target="_blank">12511499</a>]<br></li>
<li>This ability of the hilD 3 ′ UTR to reduce expression of a heterologous gene supports the view that the 3 ′ UTR can act as an independent module Because hilD mRNA is more abundant in the absence of its native 3 ′ UTR and HilD is a positive regulator of SPI-1 , we wondered whether increased level of hilD mRNA resulted in increased SPI-1 expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24682814" target="_blank">24682814</a>]<br></li>
<li>Consistently , when the activity of H-NS was inactivated , the expression of ssrAB also became independent of HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25135218" target="_blank">25135218</a>]<br></li>
<li>In addition , they show that in the absence of these negative cis elements , or when H-NS is inactivated , the expression of ssrAB becomes independent of HilD , further supporting the role of HilD as an antagonist of H-NS-mediated repression of ssrAB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25135218" target="_blank">25135218</a>]<br></li>
<li>Furthermore , our results revealed that in the absence of the downstream region bound by H-NS or when H-NS is inactivated , the expression of ssrAB becomes independent of HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25135218" target="_blank">25135218</a>]<br></li>
<li>Additionally , growth of Salmonella in the presence of histone deacetylases inhibitors reduced expression of SPI1 by affecting HilD stability , supporting the concept that altering the stability of this regulator is required for Pat to control Salmonella invasion .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27341691" target="_blank">27341691</a>]<br></li>
<li>It was found that LoiA positively regulates the expression of hilA ( the master activator of SPI-1 ) through direct activation of HilD under low O2 conditions , leading to the activation of SPI-1 genes for invasion .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28575106" target="_blank">28575106</a>]<br></li>
<li>Increasing amounts of HilD decreased the amount of free DNA , while the amount of HilDbound DNA increased .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24488311" target="_blank">24488311</a>]<br></li>
<li>Binding of HilD to the A2 region required the presence of ﬂanking sequences .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12107132" target="_blank">12107132</a>]<br></li>
<li>These results thus implicate post-translational regulation of HilD as one of the important means of invasion control , allowing Salmonella to rapidly respond to the environment of the intestine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27341691" target="_blank">27341691</a>]<br></li>
<li>To investigate this unique model of regulation by HilD , we determined the effect of a hilD mutation in strains of Salmonella that also contained mutations in the negative regulators Hha , HilE , Pag , and RNase E. Work in our laboratory indicates that Hha regulates the hilA promoter directly , due to its ability to-bind to the hilA promoter upstream of position 39 ( 17 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12511499" target="_blank">12511499</a>]<br></li>
<li>The overall effect is to control the ability of HilD to activate its own promoter , the critical step in initiating expression of the SPI1 T3SS ( 17 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29378886" target="_blank">29378886</a>]<br></li>
<li>To reveal the involvement of the DnaKJ chaperone machinery in regulating hilA expression , we examined the cellular level of HilD , which has been shown to-bind to the upstream sequence of hilA to activate its transcription directly ( 7 , 13 , 34 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18723621" target="_blank">18723621</a>]<br></li>
<li>Figure 4B shows that while the sirA deletion decreased hilA expression in the presence of HilD , it had no effect on the hilA expression when hilA was being activated by the tetRA -- rtsA construct at 1 mg/mL tetracycline in the absence of HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22021388" target="_blank">22021388</a>]<br></li>
<li>In the tetRA -- rtsA strain , the phoQ24 mutation caused a decrease in hilA expression regardless of the presence or absence of HilD ( Figure 5B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22021388" target="_blank">22021388</a>]<br></li>
<li>Since the postulated function of HilD at the hilA promoter is to counteract the effects of repressors , we examined this model by measuring hilA : : Tn5lacZY expression in strains containing negative regulator mutations in the presence or absence of functional HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12511499" target="_blank">12511499</a>]<br></li>
<li>Thus , the ability of HilD pro-lative β-gal activity tein alone to activate hilA expression is reduced in the presence of oleate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26884427" target="_blank">26884427</a>]<br></li>
<li>Since the expression of OmpR was not affected in the absence of SirA or HilD ( data not shown ) , we hypothesized that SirA could regulate the expression of the ssrAB operon directly , acting along with HilD and OmpR , or indirectly by controlling the expression of HilD in a cascade fashion .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21518393" target="_blank">21518393</a>]<br></li>
<li>As shown in Figure 4B , the expression of the hilD-cat fusion was similarly reduced in these three mutants , indicating that the absence of CpxA or HilD has the same effect on the expression of hilD , and that , when HilD is not present , the absence of CpxA does not longer repress hilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26300871" target="_blank">26300871</a>]<br></li>
<li>The low level of hilD expression in the mlc mutant can be explained by the fact that the increased level of HilE in the mlc mutant can repress the activity of HilD , which is known to act as an activator of its own expression ( 9,36 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17329372" target="_blank">17329372</a>]<br></li>
<li>As shown in Figure 6A , HilD expression increases as the value of the parameter kD increases , equivalent to increasing the strength of the feedback on HilD expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20686667" target="_blank">20686667</a>]<br></li>
<li>This result suggests that , in addition to decreasing the repression of hilA under low-osmolarity conditions ( Fig. 1B ) , the hns mutation increases the derepression of hilA by HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12933899" target="_blank">12933899</a>]<br></li>
<li>To start to deﬁne whether CpxR affects hilA directly or through HilD , which positively regulates hilA , we determined the effect of the overexpression of CpxR on the activity of a hilDcat transcriptional fusion in the WT S. Typhimurium strain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26300871" target="_blank">26300871</a>]<br></li>
<li>As shown in Fig. 6A , increasing amounts of MBP-HilD shifted the DNA -- H-NS complex to a slower-migrating complex similar to that formed by MBP-HilD ; furthermore , the Western blots indicated that increasing amounts of the DNA-HilD complexes correlate with decreasing amounts of H-NS -- FH bound to the DNA fragments .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25135218" target="_blank">25135218</a>]<br></li>
<li>To investigate the importance of HilD level to repression by propionate , we next tested the effect of a lon mutation on HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23289537" target="_blank">23289537</a>]<br></li>
<li>A substantial degree of salt inducibility was restored in the K100 to C505 promoter variant carrying the HilC/HilD-binding A2 region ( K85 to K61 ) .20,21 This stimulatory effect by the presence of the A2 region was SPI1-dependent , suggesting that action of HilC or HilD can increase promoter activity that has been silenced by the presence of the DSR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16443238" target="_blank">16443238</a>]<br></li>
<li>Expression of HilD from pK6-HilD was induced by adding 0.001 % L-arabinose to the medium at the beginning of the bacterial cultures .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27886269" target="_blank">27886269</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27886269" target="_blank">27886269</a>]<br></li>
<li>Fur regulation of HilD does not appear to be simply at the transcriptional or translational level but rather requires the presence of the HilD protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
<li>In the case of HilD , following changes were made to the parameters to reﬂect the fact that HilD has the maximum effect on the regulation of PhilA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19003447" target="_blank">19003447</a>]<br></li>
<li>The second model suggests that Fur ( or something regulated by Fur ) acts at the level of the DNA to affect the threshold of HilD required to activate the hilD promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
<li>It is the amount of HilE that sets this threshold , by blocking the activity of low levels of HilD and thus preventing the inevitable minor ﬂuctuations in HilD activity ( noise ) from inappropriately activating the system .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29378886" target="_blank">29378886</a>]<br></li>
<li>The results ( Fig. 7C ) show that the increased level of 32 does not result in decreased HilD if Lon protease was absent from the cells , suggesting that 32 controls the cellular level of HilD through the induction of Lon protease , which speciﬁcally recognizes and degrades it .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18723621" target="_blank">18723621</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_802">
Furthermore , we found that activation of CpxR decreases the stability of HilD and that , in the absence of the Lon protease , which degrades HilD , the CpxR-mediated repression of the SPI-1 genes is mostly lost .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26300871" target="_blank">26300871</a>]<br></label>
<input id="act_similarity_802", type="checkbox">
<div class="ACT_similar">
<li>CpxR Affects Stability of HilD On the basis of our results indicating that repression of the SPI-1 genes by CpxR is mostly lost in the absence of the Lon protease , which degrades HilD , we hypothesized that CpxR should reduce the stability of HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26300871" target="_blank">26300871</a>]<br></li></div>
<li>MBP-HilD specifically bound the DNA fragments of gtgE , phoH , sinR , lpxR , SL4247 and , as expected , that of hilA , at concentrations of 0.1 to 1.0 μM ; in contrast , at the same concentrations it did not bind the DNA fragment of SL1896 , or those of the negative controls , ppk and sigD ( Fig. 4A -- G ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27886269" target="_blank">27886269</a>]<br></li>
<li>Therefore , the increased invasiveness of Dlon mutant could result from the accumulation of HilC and HilD , leading to overexpression of the SPI1 genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15661008" target="_blank">15661008</a>]<br></li>
<li>Expression of the full-length promoter fragment ( 242 to 505 ) was stimulated roughly 24-fold by the presence of SPI-1 , presumably reﬂecting the action of HilC and HilD ( Table 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15126488" target="_blank">15126488</a>]<br></li>
<li>This regulated induction of hilD eliminates the autoinduction of hilD expression by the HilD protein and thus ensures that the effects observed are not attrib-utable to the control of hilD transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27185788" target="_blank">27185788</a>]<br></li>
<li>Therefore , the expression of ssrAB seems to require the coordinated actions of HilD , which antagonizes repression by H-NS , and OmpR , which probably mediates the interaction of the RNA polymerase with the promoter of this operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25135218" target="_blank">25135218</a>]<br></li>
<li>We hypothesized that ClpXP may regulate HilC and HilD through the control of the ﬂagellar regulon , since ClpXP has been shown to regulate its expression negatively by degrading the FlhD/FlhC master regulator , which functions at the apex of the transcription hierarchy of the regulon ( 61 , 62 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18245288" target="_blank">18245288</a>]<br></li>
<li>The work presented here demonstrates that this repression occurs as a result of the post-translational control of the central SPI1 regulator HilD and requires the high-energy product of propionate metabolism , propionyl-CoA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23289537" target="_blank">23289537</a>]<br></li>
<li>concentration of HilC , R the concentration of RtsA , A the concentration of HilA , X the concentration of the HilE-HilD complex , and G the concentration of the luciferase reporter for the PhilD promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20686667" target="_blank">20686667</a>]<br></li>
<li>In contrast , the expression of the cpxRA-cat fusion was similarly increased in the 1cpxA and 1hilD 1cpxA mutants ( Figure 1D ) , indicating that the absence of CpxA activates the expression of cpxRA independently of HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26300871" target="_blank">26300871</a>]<br></li>
<li>In these assays , the half-life of HilD-Myc in the presence and absence of CpxA was 38 and 20 min , respectively ( Figure 6B ) , supporting the notion that the activation of CpxR by the absence of CpxA decreases the stability of HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26300871" target="_blank">26300871</a>]<br></li>
<li>Therefore , the effect of CpxR-P on the expression of the SPI-1 and SPI-2 genes could be the result of its negative control on the stability of HilD and , as a consequence , on the transcription of hilD , which , in an additive manner , would decrease the concentration of HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26300871" target="_blank">26300871</a>]<br></li>
<li>Although HilD activity was slightly increased in the Δlon background , overproduction of HilE blocks this activity without having any signiﬁcant effect on the steady-state levels of the protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29378886" target="_blank">29378886</a>]<br></li>
<li>Rather , Fur activity requires the presence of the HilD protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
<li>As expected , the presence of HilD also induced the activity of the hilA-cat fusion , but not that of the ssaG-cat fusion ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27886269" target="_blank">27886269</a>]<br></li>
<li>In particular , we showed that IscR represses the synthesis of the Spi1‐carried regulator HilD , decreasing the expression of genes encoding the TTSS apparatus .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27704705" target="_blank">27704705</a>]<br></li>
<li>The distances in nucleotides of the insertions to the cents rst nucleotide of the HilD coding sequence are as follows : TF37 , 936 ; TF38 , 811 ; TF39 , 664 ; BJ606 , 652 , TF40 , 349 ; TF41 , 99 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10767604" target="_blank">10767604</a>]<br></li>
<li>As seen from Fig. 3 , all the models showed a somewhat slow increase to steady state levels of HilA in the presence of the induction signal , HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19003447" target="_blank">19003447</a>]<br></li>
<li>If L-arabinose impaired the ability of HilD to activate gene transcription , the reduction in the hilD mRNA level observed in the presence of L-arabinose might be due to loss of the HilD positive feedback loop .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25991823" target="_blank">25991823</a>]<br></li>
<li>Expression of the sRNA STnc520 decreased 17-fold in the absence of the primary SPI1 regulator , HilD , at ESP ( Fig 7A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27564394" target="_blank">27564394</a>]<br></li>
<li>To further localize the regulatory sequences required for the repression of ssrAB by H-NS , we analyzed the effect of dominant negative H-NSQ92am on the expression of the different ssrAB-cat fusions in the absence of HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25135218" target="_blank">25135218</a>]<br></li>
<li>To check whether 32 modulates the HilD level by controlling the induction of Lon , we examined the effect of 32 overproduction on the levels of HilD in a lon-deﬁcient background .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18723621" target="_blank">18723621</a>]<br></li>
<li>Another SPI regulator , HilD , which is not associated with Fisbinding , acts in an ordered fashion with HilC to coordinately activate expression of the SPI-1 genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23717649" target="_blank">23717649</a>]<br></li>
<li>For example , HilA , the main regulator directly activating SPI-1 genes , has been reported to get sufficiently expressed in the presence of HilD along with HilC or RtsA [ 11 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24079299" target="_blank">24079299</a>]<br></li>
<li>Previously , we showed that when Salmonella grows in LB medium HilD , encoded in SPI-1 , first induces the expression of hilA , located in SPI-1 , and subsequently of the ssrAB operon , located in SPI-2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21518393" target="_blank">21518393</a>]<br></li>
<li>Taken together , these results indicated that SirA , in addition to HilD , OmpR and SsrB , is required for the expression of SPI-2 genes when Salmonella grows in LB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21518393" target="_blank">21518393</a>]<br></li>
<li>Individual overexpression of HilA , HilC , or HilD in the absence of SPI1 can not activate siiE expression , suggesting that these transcriptional-regulators act in concert or in combination with additional SPI1-encoded regulatory loci to activate SPI4 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18160484" target="_blank">18160484</a>]<br></li>
<li>Differential responses of HilC and HilD in SPI1 expression to environmental stimuli We have previously demonstrated that the lon disruption results in the enhanced SPI1 expression in bacterial cells even when they are grown under SPI1-repressing conditions ( Takaya et al. , 2002 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15661008" target="_blank">15661008</a>]<br></li>
<li>This was conﬁrmed by analyzing the - galactosidase activity of transcriptional rﬂM fusions , which showed an increase in rﬂM-lac transcription upon HilD overexpression ( see Fig .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24488311" target="_blank">24488311</a>]<br></li>
<li>Consistent with the notion that FliZ activates T3SS-1 expression through HilD , transcription of hilD was induced in a tviA defective strain ( Fig. 1A , cluster 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19703107" target="_blank">19703107</a>]<br></li>
<li>Interestingly , our results suggest that HilD displaces H-NS from most , but not all , of its binding sites in ssrAB ( Fig. 6 ) , which is consistent with our genetic and binding results indicating the presence of several H-NS binding sites in ssrAB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25135218" target="_blank">25135218</a>]<br></li>
<li>Importantly , these hilD fusion constructs are hilD null so that we can examine regulation in the absence of autoactivation by HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20889744" target="_blank">20889744</a>]<br></li>
<li>Induction of pBAD-HilD also elevated InvR levels in wild-type cells ( threefold ) , suggesting that the invR promoter can be further activated when HilD is highly expressed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17971080" target="_blank">17971080</a>]<br></li>
<li>Nonetheless , further results indicate that increased hilA expression is detected only in the presence of HilD , suggesting that ams may also exert a posttranscriptional effect on HilD ( 232 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23554419" target="_blank">23554419</a>]<br></li>
<li>We propose that the changes in HilD half-life conferred by changing HilE levels are a result of HilD being slightly more unstable when free in solution than its DNA-bound form .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29378886" target="_blank">29378886</a>]<br></li>
<li>They further demonstrated that Pat was involved in regulating HilD stability , but they did not identify lysine res-idues involved in the regulation of HilD stability [ 32 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28329249" target="_blank">28329249</a>]<br></li>
<li>S3 in the supplemental material ) , indicating that Pat-mediated acetylation of HilD is not responsible for the increased stability in bile .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29229736" target="_blank">29229736</a>]<br></li>
<li>As shown in Supplementary Figure S1 , no increase of acetylation signal was observed , suggesting that AcP is not involved in the acetylation of HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28329249" target="_blank">28329249</a>]<br></li>
<li>Thus , this 7-fold effect on HilD seen in this experiment could easily account for the 20-fold effect seen in the wild-type background , despite the limited inhibition of RtsA or HilC activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26884427" target="_blank">26884427</a>]<br></li>
<li>Although HilD stability was decreased in a Δpat Ty2 strain , in line with previous ﬁndings in S. Typhimurium ( 46 , 47 ) , it was still increased in the presence of bile , increasing from 4 min in LB to 13 min in the presence of bile ( see Fig .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29229736" target="_blank">29229736</a>]<br></li>
<li>When either the LexA - HilE plasmid or the LexA-HilD plasmid was present in E. coli SU202 with only a control vector plasmid , there was no reduction in - galactosidase activity , and in fact a slight increase in expression was observed in some of the experiments .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12595445" target="_blank">12595445</a>]<br></li>
<li>However , the data in Fig. 5 show that SirA , even overproduced , can activate neither hilC nor hilA in the absence of HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16045614" target="_blank">16045614</a>]<br></li>
<li>The levels of all four proteins were higher in hilD 3 ′ UTR strains compared with the hilD 3 ′ UTR + background ( Figure 4A ) , suggesting that the increased hilD mRNA level observed in the hilD 3 ′ UTR strain yields a higher amount of HilD protein , which in turn results in increased SPI-1 expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24682814" target="_blank">24682814</a>]<br></li>
<li>The immunoblotting results in Fig. 7 show that HilD was detectable in the absence of IPTG ( isopropyl - - D-thiogalactopy-ranoside ) , a condition in which hilD was expressed by read-through from the PA1lacO-1 promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18245288" target="_blank">18245288</a>]<br></li>
<li>Under low Mg2 + condition , the expression of pagC was similar between both serovars , but it was 5.4-fold lower in the S. Choleraesuis than in S. Typhimurium under normal Mg2 + condition , suggesting S. Choleraesuis was hypersensitive to Mg2 + to cause HilD activation ( Fig. 6B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27886215" target="_blank">27886215</a>]<br></li>
<li>The increase in FimZ leads to the subsequent increase in HilE protein levels , which limits the availability of HilD to activate the hilA promoter due to HilE-HilD binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25547794" target="_blank">25547794</a>]<br></li>
<li>We did note a change in the steady-state levels of HilD in response to HilE .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29378886" target="_blank">29378886</a>]<br></li>
<li>Fur , the primary iron regulatory protein in Salmonella , can activate SPI-1 through HilD in response to the high concentration of Fe in the small intestine [ 26 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28575106" target="_blank">28575106</a>]<br></li>
<li>The decrease in hilA transcription could be due to the marked decrease in the cellular level of HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18723621" target="_blank">18723621</a>]<br></li>
<li>To demonstrate the ability of our assay to detect differences in hilD translation , we also included a csrB csrC double mutant , which decreases HilD translation due to enhanced CsrA-mediated transcript degradation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27185788" target="_blank">27185788</a>]<br></li>
<li>Here we show that Fur regulates HilD posttranslationally , leading to induction of the SPI1 T3SS in high-iron conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
<li>However , our most striking results show that even though HilD appears to be the best hilA activator of the three in vitro , when working alone it can not induce hilA enough to stimulate invasion .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16045614" target="_blank">16045614</a>]<br></li>
<li>Since we have evidence that HilE may regulate invasion gene expression by binding to the HilD protein , we were interested in determining whether hilE could repress invF transcriptional activation in the absence of hilA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12595445" target="_blank">12595445</a>]<br></li>
<li>A cascade of transcriptional activation has been unraveled in which HilD , in response to particular environmental signals , directly affects hilA expression by binding to the upstream repressing sequence , thereby derepressing hilA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16391141" target="_blank">16391141</a>]<br></li>
<li>The level of induction by HilD varied from 2-fold for hilD itself to 146-fold for hilA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24375101" target="_blank">24375101</a>]<br></li>
<li>This regulation mediated by acetylation not only degrades redundant HilD to keep a moderate protein level to facilitate S. Typhimurium growth but also maintains an appropriate DNA-binding activity of HilD to ensure bacterial pathogenicity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28329249" target="_blank">28329249</a>]<br></li>
<li>Significantly , control of HilD by propionate manifests itself post-translationally : this fatty acid demonstrated regulation through HilD even when hilD was expressed from exogenous promoters , and a translational reporter fusion to hilD failed to be repressed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23289537" target="_blank">23289537</a>]<br></li>
<li>The system is triggered when HilD activity crosses a threshold that allows efﬁcient activation of its own promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29378886" target="_blank">29378886</a>]<br></li>
<li>First , our data suggest that HilD is critical for activation of hilA expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12511499" target="_blank">12511499</a>]<br></li>
<li>Given that HilD is required for Fur regulation of hilA , it seemed possible that Fur regulates hilD expression in some manner .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
<li>Taken together , these data show that Fur activates hilA expression ( HilD activity ) in a manner that is mechanistically different from Fur activation of sodB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
<li>Thus , one model to explain Fur regulation of SPI1 is that Fur represses a factor that negatively regulates HilD function .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
<li>Given that HilD is required for FliZ regulation of hilA , it seemed possible that FliZ could regulate hilD transcription or translation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20889744" target="_blank">20889744</a>]<br></li>
<li>One model that would explain FliZ posttranslational regulation of HilD is that FliZ could negatively regulate HilE expression or function .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20889744" target="_blank">20889744</a>]<br></li>
<li>Consistent with its role as a transcriptional-activator , HilD was required for full activation of PhilC ( Figure 1B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22457642" target="_blank">22457642</a>]<br></li>
<li>This interaction suggests that hilA repression , at least by HilE , occurs by inhibition of HilD activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12595445" target="_blank">12595445</a>]<br></li>
<li>Protein analysis techniques showed that SPI-1 repression by bile acids is mediated by posttranslational destabilization of HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27185788" target="_blank">27185788</a>]<br></li>
<li>Figure 6 shows that expression of rtsA was decreased by fur , but only when HilD was present .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
<li>The effect of propionate was more pronounced at lower tetracycline concentrations ( e.g. 0.1 μg ml − 1 produced a greater than 5-fold reduction , while 1 μg ml − 1 had only a 2-fold effect ) , suggesting that larger amounts of HilD can blunt repression by propionate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23289537" target="_blank">23289537</a>]<br></li>
<li>Figure 6 Post-translational regulation of HilD by L-arabinose .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25991823" target="_blank">25991823</a>]<br></li>
<li>The involvement of HilD in LeuO-mediated SPI-1 repression was further investigated by epistasis analysis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24354910" target="_blank">24354910</a>]<br></li>
<li>If HilD was required for SPI-1 repression by LeuO , we reasoned , downregulation of SPI-1 by LeuO should not be observed in the absence of HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24354910" target="_blank">24354910</a>]<br></li>
<li>FliZ regulates HilD independently of Lon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20889744" target="_blank">20889744</a>]<br></li>
<li>FIG 7 HilE regulates HilD activity independently of stability .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29378886" target="_blank">29378886</a>]<br></li>
<li>In particular , HilD directly regulates expression of all 7 TFs , including itself .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27601571" target="_blank">27601571</a>]<br></li>
<li>Fur regulates expression of rtsA via HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
<li>Fur regulates expression of hilA by working through HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
<li>Fur regulates expression of rtsA by controlling HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
<li>Elevated levels of HilD protein activate hilA gene expression ( 41 -- 44 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24488311" target="_blank">24488311</a>]<br></li>
<li>( D ) Effect of overproduced HilD on ﬂagellar gene expression under artiﬁcially induced ﬂhDC expression conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24488311" target="_blank">24488311</a>]<br></li>
<li>Thus , FliZ-me-diated regulation of HilD is posttranslational .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20889744" target="_blank">20889744</a>]<br></li>
<li>Fur regulation of HilD is not mediated through HilE .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
<li>RESULTS FliZ activates hilA through HilD , independent of FlhDC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20889744" target="_blank">20889744</a>]<br></li>
<li>FliZ activation of hilA is dependent on HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20889744" target="_blank">20889744</a>]<br></li>
<li>This leads to decreased production of FliZ , which activates SPI1 ( 33 , 44 ) , also through HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17951383" target="_blank">17951383</a>]<br></li>
<li>Therefore , SlyA might replace HilD to counteract H-NS-mediated repression of ssrAB when Salmonella is grown in minimal media .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25135218" target="_blank">25135218</a>]<br></li>
<li>Over-expression of Pat affects HilD stability , and acetylation of specific lysines is not essential for Pat to regulate HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27341691" target="_blank">27341691</a>]<br></li>
<li>Importantly , for many of these compounds , HilD destabilization appears to be a consequence of prerequisite HilD inactivation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27185788" target="_blank">27185788</a>]<br></li>
<li>However , some such treatments continue to repress HilD activity despite its accumulation , demonstrating a mechanism of HilD inactivation that is independent of HilD degradation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27185788" target="_blank">27185788</a>]<br></li>
<li>In such a model , overproduction of HilD could yield enough active HilD in equilibrium with inactive HilD to derepress hilA expression under repressing conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11292791" target="_blank">11292791</a>]<br></li>
<li>Yet , degradation of HilD is not necessary to repress HilD activity , as SCFAs continue to inhibit SPI-1 expression in a lon mutant ( 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27185788" target="_blank">27185788</a>]<br></li>
<li>Moreover , because of its high affinity , HilD will occupy this site even when expressed at low levels .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20686667" target="_blank">20686667</a>]<br></li>
<li>Furthermore , substitutions at 10 positions allowed increased binding of HilC , but changes at only one position , 83T to G , showed increased binding of HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12107132" target="_blank">12107132</a>]<br></li>
<li>Acetylation of K297 increases HilD stability , but reduces its DNA-binding affiffinity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28329249" target="_blank">28329249</a>]<br></li>
<li>Acetylation of HilD K297 enhances protein stability .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28329249" target="_blank">28329249</a>]<br></li>
<li>Figure S4 HilE negatively regulates HilD expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20686667" target="_blank">20686667</a>]<br></li>
<li>HilD positively regulates gtgE , phoH , sinR , lpxR , SL1896 and SL4247 in S. Typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27886269" target="_blank">27886269</a>]<br></li>
<li>HilC , HilD and RtsA are central activators of hilA expression ( Ellermeier et al. , 2005 ; Lucas & Lee , 2001 ; Schechter et al. , Abbreviations : Cy , Cyanine ; SSC , saline sodium citrate ; SPI1 , Salmonella pathogenicity island 1 ; S. Typhimurium , Salmonella enterica serovar Typhimurium ; TTSS , type III secretion system .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23676436" target="_blank">23676436</a>]<br></li>
<li>HilE interacts with HilD and negatively regulates hilA transcription ( Baxter et al. , 2003 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15661008" target="_blank">15661008</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_902">
HilE interacts with HilD and negatively regulates hilA transcription and expression of Salmonella enterica serovar Typhimurium invasive phenotype .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15790293" target="_blank">15790293</a>]<br></label>
<input id="act_similarity_902", type="checkbox">
<div class="ACT_similar">
<li>Baxter , M.A. , Fahlen , T.F. , Wilson , R.L , and Jones , B.D. ( 2003 ) HilE interacts with HilD and negatively regulates hilA transcription and expression of the salmonella enterica serovar typhimurium invasive phenotype .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27341691" target="_blank">27341691</a>]<br></li>
<li>Baxter , M. , Fahlen , T. , Wilson , R. , and Jones , B. ( 2003 ) HilE interacts with HilD and negatively regulates hilA transcription and expression of the Salmonella enterica serovar Typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15661008" target="_blank">15661008</a>]<br></li>
<li>Baxter , M.A. , Fahlen , T.F. , Wilson , R.L. , and Jones , B.D. ( 2003 ) HilE interacts with HilD and negatively regulates hilA transcription and expression of the Salmonella enterica serovar Typhimurium invasive phenotype .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16045614" target="_blank">16045614</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16999831" target="_blank">16999831</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24354910" target="_blank">24354910</a>]<br></li>
<li>Baxter , M. A. , T. F. Fahlen , R. L. Wilson , and B. D. Jones , 2003 HilE interacts with HilD and negatively regulates hilA transcription and expression of the Salmonella enterica serovar Typhimurium invasive phenotype .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22021388" target="_blank">22021388</a>]<br></li>
<li>Baxter , M.A. , Fahlen , T.F. , Wilson , R.L. and Jones , B.D. ( 2003 ) HilE interacts with HilD and negatively regulates hilA transcription and expression of the Salmonella enterica serovar Typhimurium invasive phenotype .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17329372" target="_blank">17329372</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24682814" target="_blank">24682814</a>]<br></li>
<li>Baxter , M. A. , T. F. Fahlen , R. L. Wilson and B. D. Jones , 2003 HilE interacts with HilD and negatively regulates hilA transcription and expression of the Salmonella enterica serovar Typhimurium invasive phenotype .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20008574" target="_blank">20008574</a>]<br></li>
<li>HilE interacts with HilD and negatively regulates hilA transcription and expression of the Salmonella enterica serovar Typhimu-rium invasive phenotype .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15765064" target="_blank">15765064</a>]<br></li>
<li>HilE interacts with HilD and negatively regulates hilA transcription and expression of the Salmonella enterica serovar Typhimurium invasive phenotype .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19703107" target="_blank">19703107</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23554419" target="_blank">23554419</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19218381" target="_blank">19218381</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23289537" target="_blank">23289537</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14645287" target="_blank">14645287</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25182488" target="_blank">25182488</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27185788" target="_blank">27185788</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26441883" target="_blank">26441883</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16949866" target="_blank">16949866</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26300871" target="_blank">26300871</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15968047" target="_blank">15968047</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19074398" target="_blank">19074398</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923082" target="_blank">12923082</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20889744" target="_blank">20889744</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22546935" target="_blank">22546935</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27601571" target="_blank">27601571</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28575106" target="_blank">28575106</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15731035" target="_blank">15731035</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15039320" target="_blank">15039320</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21518393" target="_blank">21518393</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18441068" target="_blank">18441068</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21680637" target="_blank">21680637</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18160484" target="_blank">18160484</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25547794" target="_blank">25547794</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12933899" target="_blank">12933899</a>]<br></li>
<li>Baxter , M. A. , Fahlen , T. F. , Wilson , R. L. & Jones , B. D. HilE interacts with HilD and negatively regulates hilA transcription and expression of the Salmonella enterica serovar Typhimurium invasive phenotype .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27886215" target="_blank">27886215</a>]<br></li>
<li>Appl Environ Microbiol 76:5300 -- 5304 Baxter MA , Fahlen TF , Wilson RL , Jones BD ( 2003 ) HilE interacts with HilD and negatively regulates hilA transcription and expression of the Salmonella enterica serovar Typhimurium invasive phenotype .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24500522" target="_blank">24500522</a>]<br></li>
<li>doi :10.1046 / j.1365-2958.1996.d01-1718.x Baxter MA , Fahlen TF , Wilson RL , Jones BD ( 2003 ) HilE interacts with HilD and negatively regulates hilA transcription and expression of the Salmonella enterica serovar Typhimurium invasive phenotype .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18563621" target="_blank">18563621</a>]<br></li></div>
<li>The proteins were transferred to PVDF membranes and then immun-ostained with anti-HilA , anti-SipC , anti-InvF , anti-HilC or anti-HilD serum .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15661008" target="_blank">15661008</a>]<br></li>
<li>Interestingly , in S. typhimurium grown under activating , low oxygen and high osmolarity conditions in vitro , HilD is essential for hilA expression whereas loss of hilC has little or no effect on hilA transcription ( Eichelberg et al. , 1999 ; Rakeman et al. , 1999 ; Lucas and Lee , 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>HilD is dominant , and there is no hilA expression in its absence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21680637" target="_blank">21680637</a>]<br></li>
<li>Because HilD is required for hilA expression ( Schechter et al. , 1999 ) , and current models suggest that HilD is also required for invF expression independently of hilA ( Lucas and Lee , 2000 ) , our data are consistent with a model in which hilD is induced by Fis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11123690" target="_blank">11123690</a>]<br></li>
<li>( B ) qRT-PCR analysis of changes in hilA expression in wildtype , ΔloiA , ΔhilD , ΔloiAΔhilD , ΔloiAΔhilD + pHilD and ΔloiAΔhilD + pLoiA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28575106" target="_blank">28575106</a>]<br></li>
<li>As shown above , overexpression of Fur caused an eightfold increase in hilA expression , and this effect was dependent on HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
<li>Still , no one has reported a condition in which wild-type S. typhimurium represses hilA expression while simultaneously activating expression of invF and invF-dependent genes through HilC or HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10844688" target="_blank">10844688</a>]<br></li>
<li>The SPI-encoded regulators HilD and SsrA/B played no detectable role in PssrA induction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22457642" target="_blank">22457642</a>]<br></li>
<li>The transcription factors , HilD , HilC and RtsA can function as monomers or homodimers or heterodimers ( in combination ) to bring about activation of PhilA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19003447" target="_blank">19003447</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19003447" target="_blank">19003447</a>]<br></li>
<li>The genetic evidence to date is consistent with a regulatory cascade of transcriptional activation , in which HilD , HilA and InvF act sequentially to activate SPI1 expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17329372" target="_blank">17329372</a>]<br></li>
<li>Thus far , genetic evidence is most consistent with a cascade of transcriptional activation in which HilD , HilA and InvF act in sequence to activate TTSS-1 gene expression in vitro ( ﬁgure 1B , black bars and arrows ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11755416" target="_blank">11755416</a>]<br></li>
<li>These results suggest that HilA , HilC , and HilD are each required for expression of siiE , but these regulators are likely to act in concert or in combination with additional SPI1-encoded regulatory loci to activate SPI4 gene expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18160484" target="_blank">18160484</a>]<br></li>
<li>We have three working hypotheses , which are not mutually exclusive : ( i ) RtsA , HilC , and HilD are active under different conditions or induce expression of hilA in response to different environmental cues .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923082" target="_blank">12923082</a>]<br></li>
<li>The in vitro data suggest that HilC , HilD and RtsA act in concert to induce expression of hilA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16045614" target="_blank">16045614</a>]<br></li>
<li>Model 5 In this model , SirA regulates HilD , which in turn regulates HilC , RtsA and HilA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19537165" target="_blank">19537165</a>]<br></li>
<li>, HilD , HilC , RtsA and HilA ) would still be in an active state .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24079299" target="_blank">24079299</a>]<br></li>
<li>In this model , HilD or HilC expression and activity would be induced in vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>HilA is , in turn , induced by HilD and HilC and regulates InvF , which induces expression of secreted effector-proteins located in SPI-1 ( 26 , 27 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22004521" target="_blank">22004521</a>]<br></li>
<li>In an analogous fashion , Dam-dependent regulation of invF was still observed in HilA , HilC , and RtsA backgrounds , and no information was obtained in a HilD background ( Figure 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20008574" target="_blank">20008574</a>]<br></li>
<li>Various in vitro and in vivo conditions must be tested to determine whether direct activation of invF by HilC and/or HilD occurs in wild-type bacteria and whether this has any relevance in vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10844688" target="_blank">10844688</a>]<br></li>
<li>Moreover , previous studies indicate that SirA , CsrB/CsrC , CsrA and HilD are required by Salmonella to display phenotypic features involved in intestinal infection ( Jones , 2005 ; Altier , 2005 ; Lucchetti-Miganeh et al. , 2008 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21518393" target="_blank">21518393</a>]<br></li>
<li>To date , in Salmonella , FliZ is known to independently regulate three very different transcriptional-regulators , HilD , RpoS , and FlhD4C2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20889744" target="_blank">20889744</a>]<br></li>
<li>One possibility is that FliZ posttranscriptionally regulates FimZ in a manner similar to FlhD4C2 and HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20833811" target="_blank">20833811</a>]<br></li>
<li>We recently demonstrated that the sRNA chaperone , Hfq , is involved in SPI-1 expression and/or functions pBAD pBAD-HilD pBAD pBAD-HilD - + - + - + - + w L-ara InvR InvR 5S rRNA 5S rRNA B D PinvR - gfp Δ ( sitA-invH ) 50000 pBAD pBAD-HilD pBAD-HilA - + - + - + 40000 L-ara 30000 InvR 20000 10000 5S rRNA 0 wt hfq hilC hilD Fig. 5 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17971080" target="_blank">17971080</a>]<br></li>
<li>Therefore , we propose that acetylation by Pat regulates HilD in a posttranslational way and is beneﬁcial for HilD stability .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26810370" target="_blank">26810370</a>]<br></li>
<li>These result demonstrated that CobB can positively regulate SPI1 through controlling hilD transcription , and this mechanism is different from that of Pat , which regulates Salmonella invasion by reducing HilD Fig. 7 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27341691" target="_blank">27341691</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_940">
Expression analyses and protein-DNA interaction assays showed regulation by HilD for six of these genes : gtgE , phoH , sinR , SL1263 ( lpxR ) and SL4247 were regulated directly , whereas SL1896 was regulated indirectly .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27886269" target="_blank">27886269</a>]<br></label>
<input id="act_similarity_940", type="checkbox">
<div class="ACT_similar">
<li>Expression analyses and protein-DNA interaction assays show that HilD regulates six of these genes : gtgE , phoH , sinR , lpxR and SL4247 were regulated directly , whereas SL1896 was regulated indirectly .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27886269" target="_blank">27886269</a>]<br></li></div>
<li>HilE , a protein of unknown structure encoded outside SPI1 , binds HilD [ 34 ] and prevents it from activating its target promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20686667" target="_blank">20686667</a>]<br></li>
<li>A DNA fragment spanning this region was ﬁrst incubated with H-NS , and then increasing amounts of HilD were added .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25135218" target="_blank">25135218</a>]<br></li>
<li>The ssrAB DNA fragments 1A to 1E were incubated with increasing concentrations of puriﬁed MBP-HilD ( 0 , 0.2 , 0.3 , 0.4 , and 0.5 M ) or H-NS -- FH ( 0 , 0.3 , 0.4 , 0.5 , and 0.6 M ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25135218" target="_blank">25135218</a>]<br></li>
<li>Working model depicting acetylation regulates protein stability and DNA-binding ability of HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28329249" target="_blank">28329249</a>]<br></li>
<li>HilD directly regulates at least 11 other genes ; however , evidence that HilD acts as an antagonist of H-NS exists only for hilA and rtsA ( 23 , 24 , 26 -- 31 , 48 , 49 , 60 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25135218" target="_blank">25135218</a>]<br></li>
<li>The control of HilD by Pat occurs posttranscriptionally , with Pat moderately repressing HilD translation and substantially decreasing HilD stability .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27341691" target="_blank">27341691</a>]<br></li>
<li>Accordingly , Fur represses H-NS , and thereby HilD is no longer repressed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23554419" target="_blank">23554419</a>]<br></li>
<li>FliZ also activates HilD ( 38 , 41 , 53 , 57 , 82 ) and represses FimZ posttranscriptionally ( this study ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20833811" target="_blank">20833811</a>]<br></li>
<li>HilE binds directly and speciﬁcally with HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29378886" target="_blank">29378886</a>]<br></li>
<li>FliZ independently regulates RpoS and HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20889744" target="_blank">20889744</a>]<br></li>
<li>Cholate and bile decrease HilD posttranslational stability .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27185788" target="_blank">27185788</a>]<br></li>
<li>Consistent with the SPI-1 gene levels , SPI-1 gene transcriptional-regulators HilC , HilD , HilA , rtsA , and invF were also signiﬁcantly repressed ( Fig. 6b ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27549210" target="_blank">27549210</a>]<br></li>
<li>However , HilA activation is dependent upon a coupled , positive-feedback loop comprising the AraC/XylS family transcription factors HilD , HilC , and RtsA , which drive HilA expression above the SPI-1 activation threshold ( 7 , 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24375101" target="_blank">24375101</a>]<br></li>
<li>Hence , regulation by HilD , HilC , and RtsA must be direct .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24375101" target="_blank">24375101</a>]<br></li>
<li>hilA expression is induced by three transcriptional-activators , HilC , HilD and RtsA [ 70 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25375226" target="_blank">25375226</a>]<br></li>
<li>That means that at least hilA and rtsA are induced by HilD , HilC , and RtsA proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17675384" target="_blank">17675384</a>]<br></li>
<li>In turn , hilA is regulated by HilD and HilC ( Schechter and Lee , 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21168230" target="_blank">21168230</a>]<br></li>
<li>HilA expression , in turn , is regulated by three AraC-like transcription factors - HilC , HilD , and RtsA -- with homologous DNA binding domains [ 22,31,32 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20686667" target="_blank">20686667</a>]<br></li>
<li>HilA , in turn , is regulated by HilC , HilD , and RtsA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20686667" target="_blank">20686667</a>]<br></li>
<li>We propose that siiA is regulated by HilD , HilC , RtsA , HilA , and SprB , which would indicate that SPI-1 and SPI-4 gene expression is tightly linked .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24375101" target="_blank">24375101</a>]<br></li>
<li>The rtsAB operon is transcriptionally activated by HilD , HilC , and RtsA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24488311" target="_blank">24488311</a>]<br></li>
<li>This transfrag is regulated by HilD and HilC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27601571" target="_blank">27601571</a>]<br></li>
<li>It is certainly possible that other genes are directly regulated by HilC , HilD and RtsA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17208038" target="_blank">17208038</a>]<br></li>
<li>We propose that DapZ is regulated by both RtsA and HilC but not HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27601571" target="_blank">27601571</a>]<br></li>
<li>The regulation of SPI-1 and SPI-2 T3SS is controlled by different mechanisms , in which HilA and HilD are two important proteins that can activate SPI-1 T3SS11 -- 14 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27886215" target="_blank">27886215</a>]<br></li>
<li>InvF and HilA are transcriptional-regulators of SPI-1 genes that are encoded within SPI-1 ; hilA itself is regulated by several factors , including PhoP-PhoQ , SirA , HilC , and HilD ( 3 -- 5 , 20 , 37 , 44 , 66 , 72 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10633113" target="_blank">10633113</a>]<br></li>
<li>Hence , it is possible that HilC and RtsA could be activated independently of HilD , by other , as yet unknown regulators ( for e.g , oxygen regulation as suggested in Ellermeier et al. 2005 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19003447" target="_blank">19003447</a>]<br></li>
<li>The key regulator of this secretion system is HilA , which is in turn regulated by HilD , HilC and RtsA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19537165" target="_blank">19537165</a>]<br></li>
<li>Finally , HilA is regulated by dimers of HilC , HilD and RtsA through an OR operation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19537165" target="_blank">19537165</a>]<br></li>
<li>The master regulator of SPI-1 , HilA , is itself regulated by three homologous transcription factors , HilD , HilC , and RtsA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24375101" target="_blank">24375101</a>]<br></li>
<li>One of these , a target of HilD and RtsA , is antisense to an sRNA , MgrR , which itself is directly regulated by HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27601571" target="_blank">27601571</a>]<br></li>
<li>Transcription of invF is known to be regulated by HilA , HilC , and HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923082" target="_blank">12923082</a>]<br></li>
<li>SPI1 gene expression is negatively regulated by HilE , which binds to HilD and prevents it from activating the SPI1 promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20833811" target="_blank">20833811</a>]<br></li>
<li>SPI-1 expression is triggered by multiple environmental signals and is regulated by the SPI-1-encoded HilC and HilD proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24603618" target="_blank">24603618</a>]<br></li>
<li>SPI1 gene expression is positively regulated by the transcriptional-activators HilA , HilC , and HilD , encoded within SPI1 , and negatively regulated by the regulators HilE and PhoP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18160484" target="_blank">18160484</a>]<br></li>
<li>HilA expression is also regulated by the regulators HilC and HilD [ 29 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21379575" target="_blank">21379575</a>]<br></li>
<li>The dually positive regulation that we describe here for slrP by LeuO , Lon , and HilD under SPI1-inducing conditions and by PhoP under SPI2-inducing conditions partially explains the ability of SlrP to be secreted by both T3SSs under different circumstances .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25182488" target="_blank">25182488</a>]<br></li>
<li>Thus , the hup and ﬁs mutations may decrease hilA expression ( i ) by decreasing its derepression by HilD and/or ( ii ) by increasing its repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12933899" target="_blank">12933899</a>]<br></li>
<li>MS data showed that K23 , K34 , and K132 of HilD could be acetylated partially by Pat , and K297 could be acetylated completely by Pat ( Supplementary Table S3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28329249" target="_blank">28329249</a>]<br></li>
<li>The expression of these sRNAs is controlled by diverse transcription factors , i.e. sE , HilD , Crp and OmpR , and activated by a speciﬁc stress or growth condition .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19007416" target="_blank">19007416</a>]<br></li>
<li>S1 represents activation by HilD and S2 represents activation by SirA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19537165" target="_blank">19537165</a>]<br></li>
<li>HilD expression was slightly enhanced by all three E. coli supernatants and H and SiiE expressions were inhibited by all three E. supernatants .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21379575" target="_blank">21379575</a>]<br></li>
<li>Shown are genes identiﬁed as being directly regulated by HilD and regions identiﬁed as being bound by HilD , based on this and previous studies .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24375101" target="_blank">24375101</a>]<br></li>
<li>As shown in Table 2 , hilA expression was induced 40-fold by RtsA , 20-fold by HilD , and 120-fold by HilC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14679226" target="_blank">14679226</a>]<br></li>
<li>That is , MicC ( regulated by OmpR ) and InvR ( regulated by HilD ) transiently accumulate in early stationary phase ( 52,53,91 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22180532" target="_blank">22180532</a>]<br></li>
<li>In contrast to invF-lacZ expression from pVV448 , HilD and HilC activate invF-lacZ expression over 45-fold from pSA7 ( Fig. 6 , black and striped bars , pSA7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>HilC and HilD directly bind and activate invF-1 invF-2 P ( Schechter et al. , 1999 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15661008" target="_blank">15661008</a>]<br></li>
<li>In addition , HilD and HilC directly activate invF , sipA and sipC in non-HilA dependent manner ( Akbar et al. , 2003 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21168230" target="_blank">21168230</a>]<br></li>
<li>HilD , HilC , and RtsA not only regulate shared target genes but also activate each other 's transcription ( 8 , 10 , 11 , 20 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27601571" target="_blank">27601571</a>]<br></li>
<li>Under high-osmolarity conditions , HilD , HilC , and RtsA activate SPI1 genes directly and indirectly through hilA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17675384" target="_blank">17675384</a>]<br></li>
<li>RtsA , HilD , and HilC directly activate dsbA , which encodes a periplasmic disulﬁde bond oxidoreductase ( 16 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20889744" target="_blank">20889744</a>]<br></li>
<li>In this case , HilD was subjected to positive regulation by four nodes ( SirA-BirA , HilC , RtsA and Fur ) along with self activation [ 11,14,43,44 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24079299" target="_blank">24079299</a>]<br></li>
<li>In scenario-3 , SirA is considered to activate HilA and HilC , with HilD as a co-activator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19537165" target="_blank">19537165</a>]<br></li>
<li>HilD , HilC and RtsA are known to form a feed-forward loop for SPI-1 regulation [ 29 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28575106" target="_blank">28575106</a>]<br></li>
<li>RtsA , HilC and HilD all contribute to SPI1 activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16045614" target="_blank">16045614</a>]<br></li>
<li>HilC , HilD and RtsA , when overproduced are clearly able to induce hilD transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17208038" target="_blank">17208038</a>]<br></li>
<li>In response to different stimuli , HilD activates HilC and RtsA transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21680637" target="_blank">21680637</a>]<br></li>
<li>RtsA , HilC and HilD each contribute to activating hilA in vitro The proposed model predicts that HilD , HilC and RtsA each have a role in activating hilA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16045614" target="_blank">16045614</a>]<br></li>
<li>However , HilD , HilC and RtsA normally act in concert to activate hilA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16045614" target="_blank">16045614</a>]<br></li>
<li>HilC and RtsA are thought to activate themselves ( positive feedback ) as well each other ( cross activation ) in addition to their effect on HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19003447" target="_blank">19003447</a>]<br></li>
<li>In order to address whether additional S. typhimuriumspeciﬁc genes are required for HilD and HilC to activate invF , we examined both pVV448 and pSA7 reporters in E. coli TOP-10 cells .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>Our results conﬁrm these observations and provide additional data for low osmolarity conditions , in which HilD and HilC activators were not essential in mutants lacking H-NS and Hha proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16443238" target="_blank">16443238</a>]<br></li>
<li>Whole-cell extracts were prepared from strains of c3306 , CS2022 , CS2722 , CS2723 , CS2724 , CS2725 , CS2802 and CS2803 grown under the SPI1-activating conditions and then separated in a 10 % SDS-polyacrylamide gel.The separated proteins were transferred to PVDF membranes and then immunostained with antiHilA , anti-SipC , anti-InvF , anti-HilC or anti-HilD serum .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15661008" target="_blank">15661008</a>]<br></li>
<li>Immunoblotting for detection of HilC and HilD proteins in wildtype and Lon-depleted cells grown under SPI1-repressing conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15661008" target="_blank">15661008</a>]<br></li>
<li>Our results show that virulence activators HilD , HilC , and RtsA bind to a common site in the rtsA promoter and antagonize H-NS/Hha-mediated repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17675384" target="_blank">17675384</a>]<br></li>
<li>HilC and HilD are encoded within SPI1 , whereas RtsA is encoded in an operon at 93.9 centisomes that also encodes RtsB , which negatively regulates ﬂhDC and hence the ﬂagellar regulon ( Ellermeier and Slauch , 2003 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16045614" target="_blank">16045614</a>]<br></li>
<li>To understand how Lon selectively degrades and modulates HilC and HilD , biochemical analysis to examine the direct interaction between both these proteins and Lon in vitro is in progress .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15661008" target="_blank">15661008</a>]<br></li>
<li>The hilD promoter is active in vitro and unaffected by the HilC and HilD proteins or other SPI1 products .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12107132" target="_blank">12107132</a>]<br></li>
<li>HilE binds to HilD and prevents it from activating the PhilD , PhilC , PrtsA , and PhilA promoters [ 34 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20686667" target="_blank">20686667</a>]<br></li>
<li>As shown in Fig. 5B , both the MBP-HilD and H-NS -- FH proteins shifted speciﬁcally DNA fragments 1B , 1C , and 1D but did not shift DNA fragments 1A and 1E or the negative control , sigD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25135218" target="_blank">25135218</a>]<br></li>
<li>Consistent with these results , the ΔgreAΔgreB strain efficiently produced and secreted SPI-1 effector-proteins and recovered the ability to invade epithelial cells when HilD expression was ectopically induced ( S5 Fig and Fig 5B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28426789" target="_blank">28426789</a>]<br></li>
<li>Further analysis demonstrated that the 32-mediated system negatively regulates HilD and HilA at the posttranslational and transcriptional levels , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18723621" target="_blank">18723621</a>]<br></li>
<li>Specifically , we varied the degree by which the SPI1 regulators -- HilD , HilC and RtsA - could enhance both HilC and RtsA gene expression , given respectively by the parameters kC and kR in the model .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20686667" target="_blank">20686667</a>]<br></li>
<li>In formulating the model , we focused solely on the interacting SPI1 regulators - HilC , HilD , HilE , and RtsA - and their role in regulating hilA expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20686667" target="_blank">20686667</a>]<br></li>
<li>In agreement with genetic data , we show that the puriﬁed RtsA protein , like HilC and HilD , binds to the hilC , hilD , rtsA , and hilA promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17675384" target="_blank">17675384</a>]<br></li>
<li>To date , it is known that the positive regulators HilC and HilD and the negative regulators Hha and Fis can bind directly the hilA promoter ( 8 , 19 , 53 , 59 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14645287" target="_blank">14645287</a>]<br></li>
<li>HilC and HilD also activate the promoter PinvF-2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15661008" target="_blank">15661008</a>]<br></li>
<li>We considered that HilD and HilC might also bind upstream of invF and directly activate transcription from the HilA-activated invF promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>In the SPI1 T3SS , HilD , HilC , and RtsA form three interlocking positive-feedback loops where all three activate each other 's and their own expression ( 23 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20833811" target="_blank">20833811</a>]<br></li>
<li>The parameters kC and kR specify the degree by which the SPI1 regulators - HilC , HilD , and RtsA - can activate HilC and RtsA expression , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20686667" target="_blank">20686667</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20686667" target="_blank">20686667</a>]<br></li>
<li>HilD can be repressed by HilE , and HilC and RstA further amplify the inducing signal .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21984608" target="_blank">21984608</a>]<br></li>
<li>Indeed , under inducing conditions , HilD and HilC , and probably RtsA , counteract the strong HilA and RtsA depression triggered by H-NS ( 292 , 293 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23554419" target="_blank">23554419</a>]<br></li>
<li>Through genetic and biochemical analyses , FliZ was found to post-translationally control HilD , and as a result , to positively regulate HilA expression [ 14 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21984608" target="_blank">21984608</a>]<br></li>
<li>Thus , SirA activates both csrB and csrC , which prevent CsrA action and preserve HilD activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23554419" target="_blank">23554419</a>]<br></li>
<li>HilE and FliZ act both independently and simultaneously to regulate HilD ( 16 , 20 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29378886" target="_blank">29378886</a>]<br></li>
<li>Hence , LeuO seems to repress SPI-1 by a major ` pathway ' involving HilD and HilE , and also by minor , HilD-and HilE-independent mechanisms .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24354910" target="_blank">24354910</a>]<br></li>
<li>The interaction between FimZ and FimW was found to repress sulA-lacZY expression by 12-fold , indicating a stronger interaction between FimZ and FimW than between HilE and HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12595445" target="_blank">12595445</a>]<br></li>
<li>HilC and HilD can independently bind and activate the hilA promoter ( Schechter et al. , 1999 ; Schechter and Lee , 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15661008" target="_blank">15661008</a>]<br></li>
<li>HilC and HilD are thought to-bind directly at this site to derepress the hilA promoter under activating conditions ( Schechter and Lee , unpublished results ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11292791" target="_blank">11292791</a>]<br></li>
<li>When present , HilC or HilD binds to this same site and promotes hilA expression ( 63 ; Schechter and Lee , unpublished results ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11292791" target="_blank">11292791</a>]<br></li>
<li>In contrast , HilD and HilC have been shown to independently bind to the URS and promote hilA expression ( Schechter and Lee , 2001 ; Olekhnovich and Kadner , 2002 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>METABOLIC ADAPTATION OF SALMONELLA DURING STRESS CONDITIONS Stress factors linking virulence and metabolism When Salmonella enters into an intestinal epithelial cell , environmental factors such as high osmolarity and neutral pH lead to an activation of HilD , which in turn induces HilA and invF gene expression ( Altier , 2005 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25688337" target="_blank">25688337</a>]<br></li>
<li>Three AraC-like transcription factors -- HilC , HilD , and RtsA -- positively regulate HilA expression in response to various intracellular and environmental signals ( 5 , 7 , 17 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20190046" target="_blank">20190046</a>]<br></li>
<li>These results are consistent with previous reports indicating that HilA , HilC , and HilD regulate SPI1 gene transcription and that HilC and HilD activate hilA transcription equally ( 3 , 7 , 8 , 9 , 10 , 11 , 19 , 33 , 34 , 37 , 40 , 41 , 47 , 49 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18441068" target="_blank">18441068</a>]<br></li>
<li>Through a complex feedback and feedforward mechanism , HilC and HilD control their own and each other 's transcription , and together activate transcription of hilA [ 14,15,16 ] ( Figure 1A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22457642" target="_blank">22457642</a>]<br></li>
<li>It is known that HilD can also control gene expression through HilC , SprB , RtsA and SsrA/B4 ,14 , and possibly , as suggested by our results , through SinR and SlyA ; thus , HilD could act on SL1896 through any of these regulators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27886269" target="_blank">27886269</a>]<br></li>
<li>Discussion HilD and HilC directly activate invF expression by binding upstream and downstream of a HilD/C-dependent promoter Previous studies have shown that HilD and HilC can indirectly activate invF expression by derepressing hilA ( Schechter et al. , 1999 ; Schechter and Lee , 2001 ; Olekhnovich and Kadner , 2002 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>In fact , there is evidence that SipA and SipC HilD and HilC directly activate invF expression 723 can be secreted independently of SPI1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>HilD , HilC , and RtsA all activate transcription of hilA , hilC , hilD , dsbA , invF , slrP , rtsA , and genes in SPI-4 ( 6 , 15 , 32 , 33 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24375101" target="_blank">24375101</a>]<br></li>
<li>In Salmonella enterica serovar Typhimurium , RtsA , HilD , and HilC directly activate dsbA and deletion of dsbA leads to loss of SPI1 function .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17951383" target="_blank">17951383</a>]<br></li>
<li>By controlling expression/activity of HilD , HilC , RtsA , and HilA , many regulators contribute to SPI-1 gene regulation in response to environmental stimuli that include temperature ( H-NS and Hha ) , osmolarity ( H-NS and Hha ) , pH ( OmpR ) , iron availability ( Fur ) , and oxygen levels ( FNR ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27601571" target="_blank">27601571</a>]<br></li>
<li>We propose that HilD and HilC can independently activate invF expression and InvF production , which directly and indirectly increases transcription of the sipA and sipC genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>HilD and HilC do not seem to regulate expression of hilD or prgH ( Olekhnovich and Kadner , 2002 ) ( Figs 1 and 2B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>Although HilD and HilC regulate two promoters upstream of hilC , they activate one and repress the other , such that overall HilD and HilC appear to have no effect on hilC expression ( Olekhnovich and Kadner , 2002 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>HilC and HilD respond to different environmental conditions , but either alone is capable of activating hilA expression ( Lucas and Lee , 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21570144" target="_blank">21570144</a>]<br></li>
<li>In brief , HilD binds directly to sites upstream of hilA and invFD and acts as an activator ( 14,16,36 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17329372" target="_blank">17329372</a>]<br></li>
<li>The hilA activators , HilD and HilC appear to overcome effects of downstream silencing region and disrupt repressive DNA loop .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16443238" target="_blank">16443238</a>]<br></li>
<li>Here we describe a fundamental mechanism that activates both SPI-2 and SPI-1 promoters through changes in DNA topology , and this mechanism depends on OmpR and FIS but is independent of SsrB and HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22457642" target="_blank">22457642</a>]<br></li>
<li>The other six genes STM1329 , STM1330 , STM1600 , STM1854 , STM2585 and yneC ( STM4079S ) show a pattern of expression that resembles the SPI2-encoded and SPI2-associated genes , that are positively regulated by both HilD and SsrB under SPI1 and SPI2-inducing conditions , respectively , and are upregulated within macrophages [ 14 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27564394" target="_blank">27564394</a>]<br></li>
<li>The genes slrP and dsbA are also induced by HilC , HilD and RtsA independently of both HilA and InvF [ 25,28 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17208038" target="_blank">17208038</a>]<br></li>
<li>The levels of induction by HilD , HilC , and RtsA were similar for all genes tested ( c.f. Fig. 2 and Fig. 3B and C ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24375101" target="_blank">24375101</a>]<br></li>
<li>To this end , we puriﬁed HilC and HilD proteins that were active for DNA binding and transcriptional regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12107132" target="_blank">12107132</a>]<br></li>
<li>HilD and HilC positively regulate sprB , most likely cotrans-criptionally with hilC itself , since hilC and sprB are cotranscribed ( see Fig .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27601571" target="_blank">27601571</a>]<br></li>
<li>The SPI-1 locus encodes effectors and regulators , such as HilA , HilD , HilC , and InvF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20028809" target="_blank">20028809</a>]<br></li>
<li>SPI-1 also encodes transcriptional-activators responsible for its own expression : HilA , HilC , HilD , and InvF ( Lostroh and Lee 2001 ; Ellermeier and Slauch 2007 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25991823" target="_blank">25991823</a>]<br></li>
<li>, HilD , HilC and RtsA [ Schechter and Lee , 2001 ; Olekhnovich and Kadner , 2002 ; Ellermeier et al. , 2005 ] , of which HilD and HilC are encoded on SPI-1 [ Ellermeier and Slauch , 2003 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19537165" target="_blank">19537165</a>]<br></li>
<li>These SPI1 proteins include HilC , HilD , InvF , and RtsA ( Eichelberg and Galán , 1999 ; Ellermeier and Slauch , 2004 ; Lostroh and Lee , 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15790293" target="_blank">15790293</a>]<br></li>
<li>We thus considered that LeuO might activate hilE transcription , and that HilE might repress SPI-1 via HilD inactivation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24354910" target="_blank">24354910</a>]<br></li>
<li>The model assumes that HilE not only binds and inhibits HilD but also promotes its degradation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20686667" target="_blank">20686667</a>]<br></li>
<li>After repressing loop is disrupted , HilD 39 and its paralogs are still needed to directly activate transcription through DSR under inducing conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16443238" target="_blank">16443238</a>]<br></li>
<li>Our experiments suggest that LeuO and Lon act through HilD under SPI1-inducing conditions , whereas PhoP directly interacts with the slrP promoter to activate transcription under SPI2 inducing conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25182488" target="_blank">25182488</a>]<br></li>
<li>His6-Ulp1 protease , cleaved His6-SUMO , and uncut His6-SUMO-HilD were removed by binding to Ni-NTA agarose .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24488311" target="_blank">24488311</a>]<br></li>
<li>HilE could bind HilD and deliver it to Lon or another protease .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29378886" target="_blank">29378886</a>]<br></li>
<li>Here we provide genetic and biochemical data showing that HilE directly binds to HilD and blocks its ability to-bind DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29378886" target="_blank">29378886</a>]<br></li>
<li>HilE , however , binds to HilD and inhibits its activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20686667" target="_blank">20686667</a>]<br></li>
<li>Here we provide genetic and biochemical data showing that HilE speciﬁcally binds to HilD but not to HilC or RtsA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29378886" target="_blank">29378886</a>]<br></li>
<li>These data suggest that both HilD and HilC act as simple repressors when bound to this synthetic promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29378886" target="_blank">29378886</a>]<br></li>
<li>Moreover , HilD can activate invF expression in a Received 19 July 2016 ; editorial decision 4 February 2017 ; accepted 21 February 2017 ; published online February 24 , 2017 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28329249" target="_blank">28329249</a>]<br></li>
<li>Blackwell Science , LtdOxford , UKMMIMolecular Microbiology 0950-382X Blackwell Publishing Ltd , 200347Original ArticleHilD and HilC directly activate invF expressionS.Akbar , L.M.Schechter , C.P.Lostroh and C.A.Lee Molecular Microbiology ( 2003 ) 47 ( 3 ) , 715 -- 728 AraC/XylS family members , HilD and HilC , directly activate virulence gene expression independently of HilA in Salmonella typhimurium Samina Akbar , † Lisa M. Schechter , ‡ C. Phoebe Lostroh § and Catherine A. Lee * Department of Microbiology and Molecular Genetics , Harvard Medical School , 200 Longwood Avenue , Boston , MA 02115 , USA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_1071">
AraC/XylS family members , HilD and HilC , directly activate virulence gene expression independently of HilA in Salmonella Typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24375101" target="_blank">24375101</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27601571" target="_blank">27601571</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27185788" target="_blank">27185788</a>]<br></label>
<input id="act_similarity_1071", type="checkbox">
<div class="ACT_similar">
<li>AraC/XylS family members , HilD and HilC , directly activate virulence gene expression independently of HilA in Salmonella typhi-murium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28329249" target="_blank">28329249</a>]<br></li>
<li>AraC/XylS family members , HilD and HilC , directly activate virulence gene expression independently of HilA in Salmonella typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23554419" target="_blank">23554419</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18723621" target="_blank">18723621</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17600486" target="_blank">17600486</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14645287" target="_blank">14645287</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21168230" target="_blank">21168230</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16443238" target="_blank">16443238</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15790293" target="_blank">15790293</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15968047" target="_blank">15968047</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19074398" target="_blank">19074398</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923082" target="_blank">12923082</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14679226" target="_blank">14679226</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15126488" target="_blank">15126488</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17675384" target="_blank">17675384</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15731035" target="_blank">15731035</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25135218" target="_blank">25135218</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15039320" target="_blank">15039320</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18441068" target="_blank">18441068</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15516559" target="_blank">15516559</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18160484" target="_blank">18160484</a>]<br></li>
<li>AraC / XylS family members , HilD and HilC , directly activate virulence gene expression independently of HilA in Salmonella typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15256548" target="_blank">15256548</a>]<br></li>
<li>Akabar , S. , Schechter , L.M. , Lostroh , C.P. and Lee , C.A. ( 2003 ) AraC/XylS family members , HilD and HilC , directly activate virulence gene expression independently of HilA in Salmonella typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17329372" target="_blank">17329372</a>]<br></li>
<li>References Akbar , S. , Schechter , L.M. , Lostroh , C.P. , and Lee , C.A. ( 2003 ) AraC/XylS family members , HilD and HilC , directly activate virulence gene expression independently of HilA in Salmonella typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16999831" target="_blank">16999831</a>]<br></li>
<li>Akbar , S. , Schechter , L.M. , Lostroh , C.P. and Lee , C.A. ( 2003 ) AraC/XylS family members , HilD and HilC , directly activate virulence gene expression independently of HilA in Salmonella typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28335027" target="_blank">28335027</a>]<br></li>
<li>Akbar , S. , Schechter , L.M. , Lostroh , C.P. , and Lee , C.A. ( 2003 ) AraC/XylS family members , HilD and HilC , directly activate virulence gene expression independently of HilA in Salmonella typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16045614" target="_blank">16045614</a>]<br></li>
<li>Akbar , S. , L. M. Schechter , C. P. Lostroh , and C. A. Lee , 2003 AraC/XylS family members , HilD and HilC , directly activate virulence gene expression independently of HilA in Sal-monella typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25991823" target="_blank">25991823</a>]<br></li>
<li>References Akbar , S. , Schechter , L. , Lostroh , C. , and Lee , C. ( 2003 ) AraC/XylS family members , HilD and HilC , directly activate virulence gene expression independently of HilA in Salmo-nella typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15661008" target="_blank">15661008</a>]<br></li>
<li>LITERATURE CITED Akbar , S. , L. M. Schechter , C. P. Lostroh and C. A. Lee , 2003 AraC/XylS family members , HilD and HilC , directly activate virulence gene expression independently of HilA in Salmo-nella typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20008574" target="_blank">20008574</a>]<br></li></div>
				</div></ul><br>
			<label class="collapse_DOM" for="_2"><font size="5px">HilD protein domains</font></label>
				<input id="_2" type="checkbox">
				<div class="DOM"><br><ul><li>S1 represents activation by HilD and S2 represents activation by SirA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19537165" target="_blank">19537165</a>]<br></li>
<li>This activation appears to be HilD dependent ( 40 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12595445" target="_blank">12595445</a>]<br></li>
<li>( i ) HilE may simply interact with HilD in such a way that it blocks DNA-binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29378886" target="_blank">29378886</a>]<br></li>
<li>HilE speciﬁcally disrupts HilD DNA-binding activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29378886" target="_blank">29378886</a>]<br></li>
<li>FIG 4 HilE selectively blocks HilD DNA-binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29378886" target="_blank">29378886</a>]<br></li>
<li>All positions that were important for HilC binding were also important for HilD binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12107132" target="_blank">12107132</a>]<br></li>
<li>Substitutions unimportant for HilC binding generally had little effect on HilD binding , except for the segment from residues 60 to 56 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12107132" target="_blank">12107132</a>]<br></li>
<li>The remaining 11 regions have not previously been linked to HilD , and 9 are located outside SPI-1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24375101" target="_blank">24375101</a>]<br></li>
<li>Of the three regulators , HilD apparently has a dominant role .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16045614" target="_blank">16045614</a>]<br></li>
<li>Various in vitro and in vivo conditions must be tested to determine whether direct activation of invF by HilC and/or HilD occurs in wild-type bacteria and whether this has any relevance in vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10844688" target="_blank">10844688</a>]<br></li>
<li>To study DNA-binding in vitro , we used Sumo-HilD and Sumo-HilC fusion proteins ( 27 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26884427" target="_blank">26884427</a>]<br></li>
<li>The N-terminal domains of HilD , HilC , and RtsA share approximately 10 % identity , whereas the C-terminal DNA-binding domains share 46 % identity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29378886" target="_blank">29378886</a>]<br></li>
<li>Peptide sequences for the HilD , HilC , and RtsA DNA-binding domains were aligned using ClustalW ( 30 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24375101" target="_blank">24375101</a>]<br></li>
<li>HilA expression , in turn , is regulated by three AraC-like transcription factors - HilC , HilD , and RtsA -- with homologous DNA-binding domains [ 22,31,32 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20686667" target="_blank">20686667</a>]<br></li>
<li>HilD and HilC have very similar DNA-binding and transcription-activation domains ( Schechter and Lee , 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>Importantly , the HA-tagged HilD used in these studies was functional ( Fig. 5 ) , indicating that the HA tag used does not disrupt HilD structure or activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29229736" target="_blank">29229736</a>]<br></li>
<li>The data in Fig. 2A suggest that ﬁrst , HilD forms homodimers , and second , HilE and HilD indeed interact via protein-protein interaction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29378886" target="_blank">29378886</a>]<br></li>
<li>HilD ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18723621" target="_blank">18723621</a>]<br></li>
<li>Amino acid sequence alignment between HilC and HilD has revealed that both proteins are highly conserved in a region of 99 C-terminal residues , which forms two potential a-helix -- turn -- a-helix DNA-binding motifs in AraC/XylS-family proteins ( Gallegos et al. , 1997 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15661008" target="_blank">15661008</a>]<br></li>
<li>HilD contains a C-terminal helix-turn-helix DNA-binding motif within a conserved 99-amino-acid domain characteristic of AraC/XylS-family members ( 42 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12511499" target="_blank">12511499</a>]<br></li>
<li>Accordingly , DapZ would be a GcvB-like sRNA , that is similar to referring to HilD as an AraC-like transcription factor because it carries the same DNA-binding domain as the ancestral AraC protein ( Schechter et al , 1999 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22922465" target="_blank">22922465</a>]<br></li>
<li>Both HilC and HilD are predicted to be members of the AraC/XylS-family of transcriptional activators , based on homology within their C-terminal domains that contain a characteristic helix-turn-helix DNA-binding motif ( Eichelberg et al. , 1999 ; Rakeman et al. , 1999 ; Schechter et al. , 1999 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15765064" target="_blank">15765064</a>]<br></li>
<li>As members of the AraC-family , HilC and HilD monomers are expected to bind to a 17-bp region , in which each helix-turn helix motif binds to 3 - to 6-bp stretches on successive DNA helical turns ( 29 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12107132" target="_blank">12107132</a>]<br></li>
<li>The Sybyl-Composer model was used to perform homology modeling for panels C and D to build structures for the CTD of HilD , from amino acids 211 to 309 , based on the crystal structure of Rob , an AraC/XylS regulator that is homologous to HilD in its helix-turn-helix DNA-binding domain ( 31 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12511499" target="_blank">12511499</a>]<br></li>
<li>Similar to HilC and HilD , RtsA belongs to the AraC/XylS-family of regulators , while RtsB is a helix-turn-helix DNA-binding protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15765064" target="_blank">15765064</a>]<br></li>
<li>This makes HilD an excellent candidate for contacting the CTD to recruit RNA polymerase to the hilA promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12511499" target="_blank">12511499</a>]<br></li>
<li>In all 8 cases , transcription-activation required both the inducible HilD plasmid and the inducer , arabinose .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24375101" target="_blank">24375101</a>]<br></li>
<li>To further examine the importance of these acetylated lysine residues on HilD function , we replaced all of them with arginines , and expressed this HilD from a plas-11K-R mid ( pwsk129-tetHilD ) in the hilD null mutant strain 11K-R carrying a sipC : : lacZY fusion .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27341691" target="_blank">27341691</a>]<br></li>
<li>The presence of the two regulators ( HilD and SirA ) gives rise to two possibilities , namely , either both the regulators are important or one of them is important .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19537165" target="_blank">19537165</a>]<br></li>
<li>Site-directed mutagenesis of this predicted motif in the 111 / 287 region may help to better deﬁne the binding sites of HilD in ssrAB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25135218" target="_blank">25135218</a>]<br></li>
<li>Rather , Fur activity requires the presence of the HilD protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
<li>In summary , stimulation of hilA transcription by the AraC/XylS-family members HilC and HilD does exhibit the expected dependence on residues in - CTD , but activation of transcription involves other - CTD-independent processes as well .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15126488" target="_blank">15126488</a>]<br></li>
				<label class="DOM_collapse_similarity" for="dom_similarity_33">
Olekhnovich , I.N. , and Kadner , R.J. ( 2004 ) Contribution of the RpoA C-terminal domain to stimulation of the Salmo-nella enterica hilA promoter by HilC and HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16045614" target="_blank">16045614</a>]<br></label>
<input id="dom_similarity_33", type="checkbox">
<div class="DOM_similar">
<li>Contribution of the RpoA C-terminal domain to stimulation of the Salmonella enterica hilA promoter by HilC and HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16443238" target="_blank">16443238</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17675384" target="_blank">17675384</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15968047" target="_blank">15968047</a>]<br></li></div>
<li>These results suggest the importance of proper helical phasing and spacing for the interaction of HilC/HilD bound at the A2 site with the C-terminal domain of the a subunit of RNA polymerase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16443238" target="_blank">16443238</a>]<br></li>
<li>In addition , we found that a single point mutation at leucine 289 in the C-terminal domain of the subunit of RNA polymerase leads to very low levels of hilA : : Tn5lacZY expression , suggesting that HilD activates transcription of hilA by contacting and recruiting RNA polymerase to the hilA promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12511499" target="_blank">12511499</a>]<br></li>
<li>Previous work in our laboratory suggested that HilD activates the expression of hilA by direct contact with the C-terminal domain of RNA polymerase ( 10 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15039320" target="_blank">15039320</a>]<br></li>
<li>Since HilD is a member of the AraC/XylS-family of regulators , we thought it was possible that HilD contacts RNA polymerase at the CTD to activate transcription of hilA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12511499" target="_blank">12511499</a>]<br></li>
<li>A likely explanation for decreased hilA expression in the SL1344 rpoA155 strain is disruption of an interaction between HilD and the CTD of RNA polymerase that is critical for activation of hilA transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12511499" target="_blank">12511499</a>]<br></li>
<li>The HilD and perhaps the HilC proteins play central roles in controlling hilA transcription and may counteract this repressor activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12107132" target="_blank">12107132</a>]<br></li>
<li>Although the exact function of HilE remains to be determined , it clearly affects HilD protein activity ( Ellermeier and Slauch 2008 ; Chubiz et al. 2010 ) and serves as an important example in the following experiments .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22021388" target="_blank">22021388</a>]<br></li>
<li>By aligning the sequences of HilD , HilC , and RtsA DNA-binding domains with that of MarA using Pfam ( 36 ) , we predicted the amino acids that make base-speciﬁc contacts with DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24375101" target="_blank">24375101</a>]<br></li>
<li>HilD also possesses 2 HTH motifs , and K297 is located in the HTH motif close to the C-terminal ; we speculate that acetyl-ation of K297 may affect HilD 's DNA-binding ability .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28329249" target="_blank">28329249</a>]<br></li>
<li>( ii ) Given the results above and our unpublished data indicating that HilD forms homodimers in vivo , it is possible that HilE disrupts HilD dimerization and thereby indirectly blocks HilD DNA-binding and activation of the hilA promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29378886" target="_blank">29378886</a>]<br></li>
<li>This reduction in - galactosidase activity is indicative of a proteinprotein interaction between HilE and HilD that leads to dimerization of the LexA DNA-binding proteins and repression of the sulA promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12595445" target="_blank">12595445</a>]<br></li>
<li>HilD and HilC are homologous , particularly in their DNA-binding domains , and thus bind the same sites in the various promoters that they regulate ( 26 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26884427" target="_blank">26884427</a>]<br></li>
<li>Some members of the AraC/XylS-family have difﬁcult solubility properties , and the HilC and HilD proteins lost DNA-binding activity upon prolonged storage .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12107132" target="_blank">12107132</a>]<br></li>
<li>We hypothesized that loss of Hfq-association with hilD mRNA in Dhfq causes loss of HilD protein synthesis , and thereby one of the strongest hfq phenotypes , i.e. loss of SPI-1 activation and virulence factor ( effector protein ) secretion .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18725932" target="_blank">18725932</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18725932" target="_blank">18725932</a>]<br></li>
<li>Previous work by our research group has shown that HilE interacts with HilD , which prevents the activation of hilA by HilD ( 39 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25547794" target="_blank">25547794</a>]<br></li>
<li>HilE is a Salmonella-speciﬁc protein that may prevent the activation of hilA by protein-protein interactions with HilD ( 8 , 25 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15039320" target="_blank">15039320</a>]<br></li>
<li>One plausible mechanism for reduced HilD stability is the loss of interaction with another protein that contacts HilD and stabilizes it .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27341691" target="_blank">27341691</a>]<br></li>
				</div></ul><br>
			<label class="collapse_EVO" for="_3"><font size="5px">HilD evolution</font></label>
				<input id="_3" type="checkbox">
				<div class="EVO"><br><ul><li>HilD remaning ( % Low-osmolarity High-oxygen lon + Dlon lon + Dlon HilC HilD Fig. 7 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15661008" target="_blank">15661008</a>]<br></li>
<li>HilD is highly conserved between S. Typhi and S. Typhimurium ( 99 % identity ; 2 amino acid changes ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29229736" target="_blank">29229736</a>]<br></li>
<li>Blackwell Science , LtdOxford , UKMMIMolecular Microbiology 0950-382X Blackwell Publishing Ltd , 200347Original ArticleHilD and HilC directly activate invF expressionS.Akbar , L.M.Schechter , C.P.Lostroh and C.A.Lee Molecular Microbiology ( 2003 ) 47 ( 3 ) , 715 -- 728 AraC/XylS family members , HilD and HilC , directly activate virulence gene expression independently of HilA in Salmonella typhimurium Samina Akbar , † Lisa M. Schechter , ‡ C. Phoebe Lostroh § and Catherine A. Lee * Department of Microbiology and Molecular Genetics , Harvard Medical School , 200 Longwood Avenue , Boston , MA 02115 , USA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
				<label class="EVO_collapse_similarity" for="evo_similarity_4">
AraC/XylS family members , HilD and HilC , directly activate virulence gene expression independently of HilA in Salmonella typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23554419" target="_blank">23554419</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18723621" target="_blank">18723621</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17600486" target="_blank">17600486</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14645287" target="_blank">14645287</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21168230" target="_blank">21168230</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16443238" target="_blank">16443238</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15790293" target="_blank">15790293</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15968047" target="_blank">15968047</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19074398" target="_blank">19074398</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923082" target="_blank">12923082</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14679226" target="_blank">14679226</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15126488" target="_blank">15126488</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17675384" target="_blank">17675384</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15731035" target="_blank">15731035</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25135218" target="_blank">25135218</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15039320" target="_blank">15039320</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18441068" target="_blank">18441068</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15516559" target="_blank">15516559</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18160484" target="_blank">18160484</a>]<br></label>
<input id="evo_similarity_4", type="checkbox">
<div class="EVO_similar">
<li>AraC / XylS family members , HilD and HilC , directly activate virulence gene expression independently of HilA in Salmonella typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15256548" target="_blank">15256548</a>]<br></li>
<li>AraC/XylS family members , HilD and HilC , directly activate virulence gene expression independently of HilA in Salmonella typhi-murium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28329249" target="_blank">28329249</a>]<br></li>
<li>AraC/XylS family members , HilD and HilC , directly activate virulence gene expression independently of HilA in Salmonella Typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24375101" target="_blank">24375101</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27601571" target="_blank">27601571</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27185788" target="_blank">27185788</a>]<br></li>
<li>Akabar , S. , Schechter , L.M. , Lostroh , C.P. and Lee , C.A. ( 2003 ) AraC/XylS family members , HilD and HilC , directly activate virulence gene expression independently of HilA in Salmonella typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17329372" target="_blank">17329372</a>]<br></li>
<li>References Akbar , S. , Schechter , L.M. , Lostroh , C.P. , and Lee , C.A. ( 2003 ) AraC/XylS family members , HilD and HilC , directly activate virulence gene expression independently of HilA in Salmonella typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16999831" target="_blank">16999831</a>]<br></li>
<li>Akbar , S. , Schechter , L.M. , Lostroh , C.P. and Lee , C.A. ( 2003 ) AraC/XylS family members , HilD and HilC , directly activate virulence gene expression independently of HilA in Salmonella typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28335027" target="_blank">28335027</a>]<br></li>
<li>Akbar , S. , Schechter , L.M. , Lostroh , C.P. , and Lee , C.A. ( 2003 ) AraC/XylS family members , HilD and HilC , directly activate virulence gene expression independently of HilA in Salmonella typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16045614" target="_blank">16045614</a>]<br></li>
<li>Akbar , S. , L. M. Schechter , C. P. Lostroh , and C. A. Lee , 2003 AraC/XylS family members , HilD and HilC , directly activate virulence gene expression independently of HilA in Sal-monella typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25991823" target="_blank">25991823</a>]<br></li>
<li>References Akbar , S. , Schechter , L. , Lostroh , C. , and Lee , C. ( 2003 ) AraC/XylS family members , HilD and HilC , directly activate virulence gene expression independently of HilA in Salmo-nella typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15661008" target="_blank">15661008</a>]<br></li>
<li>LITERATURE CITED Akbar , S. , L. M. Schechter , C. P. Lostroh and C. A. Lee , 2003 AraC/XylS family members , HilD and HilC , directly activate virulence gene expression independently of HilA in Salmo-nella typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20008574" target="_blank">20008574</a>]<br></li></div>
				<label class="EVO_collapse_similarity" for="evo_similarity_15">
Mol Microbiol 90:1216 -- 1232 Akbar S , Schechter LM , Lostroh CP , Lee CA ( 2003 ) AraC/XylS family members , HilD and HilC , directly activate virulence gene expression independently of HilA in Salmonella typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27549210" target="_blank">27549210</a>]<br></label>
<input id="evo_similarity_15", type="checkbox">
<div class="EVO_similar">
<li>Akbar S. , Schechter L.M. , Lostroh C.P. , Lee C.A. ( 2003 ) AraC/XylS family members , HilD and HilC , directly activate virulence gene expression independently of HilA in Salmonella typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22004521" target="_blank">22004521</a>]<br></li></div>
<li>First , HilD , an AraC/XylS family member , binds directly to several sites within PhilA and de-represses hilA expression ( [ 52 ] ; L. Schechter and C. Lee , unpublished observations ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11755416" target="_blank">11755416</a>]<br></li>
				<label class="EVO_collapse_similarity" for="evo_similarity_18">
AraC/XylS family members , HilC and HilD , directly bind and derepress the Salmonella typhi-murium hilA promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28329249" target="_blank">28329249</a>]<br></label>
<input id="evo_similarity_18", type="checkbox">
<div class="EVO_similar">
<li>AraC/XylS family members , HilC and HilD , directly bind and derepress the Salmonella Typhimurium hilA promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24375101" target="_blank">24375101</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23504014" target="_blank">23504014</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25547794" target="_blank">25547794</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27601571" target="_blank">27601571</a>]<br></li>
<li>AraC/XylS family members , HilC and HilD , directly bind and derepress the Sal-monella typhimurium hilA promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28426789" target="_blank">28426789</a>]<br></li>
<li>AraC/XylS family members , HilC and HilD , directly bind and derepress the S. typhimurium hilA promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12933899" target="_blank">12933899</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11466291" target="_blank">11466291</a>]<br></li>
<li>AlaC/XylS family members , HilC and HilD , directly bind and derepress the Salmonella typhimurium hilA promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18723621" target="_blank">18723621</a>]<br></li>
<li>SCHECHTER L.M. , LEE C.A. : AraC/XylS family members , HilC and HilD , directly bind and derepress the Salmonella typhimurium hilA promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17575908" target="_blank">17575908</a>]<br></li>
<li>AraC/XylS family members , HilC and HilD , directly bind and derepress the Salmonella typhimurium hilA promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25972862" target="_blank">25972862</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19218381" target="_blank">19218381</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14645287" target="_blank">14645287</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18245288" target="_blank">18245288</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21168230" target="_blank">21168230</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26441883" target="_blank">26441883</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16443238" target="_blank">16443238</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15790293" target="_blank">15790293</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16949866" target="_blank">16949866</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26300871" target="_blank">26300871</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15968047" target="_blank">15968047</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12595445" target="_blank">12595445</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19074398" target="_blank">19074398</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12107132" target="_blank">12107132</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17951383" target="_blank">17951383</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923082" target="_blank">12923082</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14679226" target="_blank">14679226</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15126488" target="_blank">15126488</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17675384" target="_blank">17675384</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29857034" target="_blank">29857034</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15731035" target="_blank">15731035</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25135218" target="_blank">25135218</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15039320" target="_blank">15039320</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12511499" target="_blank">12511499</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21518393" target="_blank">21518393</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18441068" target="_blank">18441068</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16980492" target="_blank">16980492</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24488311" target="_blank">24488311</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18160484" target="_blank">18160484</a>]<br></li>
<li>AraC/XylS family members HilC and HilD , directly bind and derepress the Salmonella typhimurium hilA promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15765064" target="_blank">15765064</a>]<br></li>
<li>Schechter , L. M. , and C. A. Lee , 2001 AraC/XylS family members , HilC and HilD , directly bind and derepress the Salmonella typhi-murium hilA promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25991823" target="_blank">25991823</a>]<br></li>
<li>Schechter , L.M. , and Lee , C.A. ( 2001 ) AraC/XylS family members , HilC and HilD , directly bind and derepress the Salmonella typhimurium hilA promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17971080" target="_blank">17971080</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16045614" target="_blank">16045614</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16999831" target="_blank">16999831</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17163975" target="_blank">17163975</a>]<br></li>
<li>Schechter , L.M. and Lee , C.A. ( 2001 ) AraC/XylS family members , HilC and HilD , directly bind and derepress the Salmonella typhimurium hilA promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17329372" target="_blank">17329372</a>]<br></li>
<li>Schechter , L. , and Lee , C. ( 2001 ) AraC/XylS family members , HilC and HilD , directly bind and derepress the Sal-monella typhimurium hilA promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15661008" target="_blank">15661008</a>]<br></li>
<li>Schechter , L.M. , and Lee , C.A. ( 2001 ) AraC/XylS family members , HilC and HilD , directly bind and derepress the S. typhimurium hilA promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>Mol Microbiol 32 : 629 -- 642 Schechter LM , Lee CA ( 2001 ) AraC/XylS family members , HilC and HilD , directly bind and derepress the Salmonella typhimurium hilA promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22922465" target="_blank">22922465</a>]<br></li>
<li>Schechter LM , Lee CA : AraC/XylS family members , HilC and HilD , directly bind and derepress the Salmonella typhimurium hilA promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23442379" target="_blank">23442379</a>]<br></li>
<li>Schechter LM , Lee CA ( 2001 ) AraC/XylS family members , HilC and HilD , directly bind and derepress the Salmonella typhimurium hilA promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18725932" target="_blank">18725932</a>]<br></li>
<li>Schechter , L. M. & Lee , C. A. AraC/XylS family members , HilC and HilD , directly bind and derepress the Salmonella typhimurium hilA promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27886269" target="_blank">27886269</a>]<br></li>
<li>Schechter L.M. , Lee C.A. ( 2001 ) AraC/XylS family members , HilC and HilD , directly bind and derepress the Salmonella typhimurium hilA promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22004521" target="_blank">22004521</a>]<br></li></div>
<li>The master regulator of SPI-1 genes is HilD , an AraC family transcription factor ( TF ) encoded within SPI-1 ( 7 , 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27601571" target="_blank">27601571</a>]<br></li>
<li>Amino acid sequence alignment between HilC and HilD has revealed that both proteins are highly conserved in a region of 99 C-terminal residues , which forms two potential a-helix -- turn -- a-helix DNA binding motifs in AraC/XylS family proteins ( Gallegos et al. , 1997 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15661008" target="_blank">15661008</a>]<br></li>
<li>HilD contains a C-terminal helix-turn-helix DNA-binding motif within a conserved 99-amino-acid domain characteristic of AraC/XylS family members ( 42 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12511499" target="_blank">12511499</a>]<br></li>
<li>The N-terminal domains of HilD , HilC , and RtsA share approximately 10 % identity , whereas the C-terminal DNA binding domains share 46 % identity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29378886" target="_blank">29378886</a>]<br></li>
<li>HilC and RtsA are homologues of HilD with 62 % and 61 % identity with HilD , respectively , in their DNA-binding domains ( Fig. 3A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24375101" target="_blank">24375101</a>]<br></li>
<li>This homology is most signiﬁcant in the car-boxy-terminal third of the proteins , which contains the putative DNA binding domain ( RtsA versus HilC , 56 % identity and 76 % similarity ; RtsA versus HilD , 60 % identity and 75 % similarity ; HilC versus HilD , 58 % identity and 72 % similarity ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923082" target="_blank">12923082</a>]<br></li>
<li>In this region , RtsA carries 56 % identity and 76 % similarity to HilC , and 60 % identity and 75 % similarity to HilD , whereas HilC carries 58 % identity and 72 % similarity to HilD [ 25 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17208038" target="_blank">17208038</a>]<br></li>
<li>However , the N-terminal domains , which presumably act to sense some environmental parameter , also show limited homology ( RtsA versus HilC , 25 % identity and 42 % similarity ; RtsA versus HilD , 21 % identity and 46 % similarity ; HilC versus HilD , 24 % identity and 50 % similarity ) RtsA , HilC , and HilD apparently function in very similar fashion .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923082" target="_blank">12923082</a>]<br></li>
<li>Class III : regulation at the level of the hilA promoter The two-component regulatory system PhoPQ belongs to the class III regulators , which do not require the presence of the functional HilD protein and work at the level of the hilA promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22021388" target="_blank">22021388</a>]<br></li>
<li>In E. coli , cpxR and cpxP belong to the FIGURE 6  CpxR reduces the stability of HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26300871" target="_blank">26300871</a>]<br></li>
<li>The Sybyl-Composer model was used to perform homology modeling for panels C and D to build structures for the CTD of HilD , from amino acids 211 to 309 , based on the crystal structure of Rob , an AraC/XylS regulator that is homologous to HilD in its helix-turn-helix DNA-binding domain ( 31 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12511499" target="_blank">12511499</a>]<br></li>
<li>As members of the AraC family , HilC and HilD monomers are expected to bind to a 17-bp region , in which each helix-turn helix motif binds to 3 - to 6-bp stretches on successive DNA helical turns ( 29 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12107132" target="_blank">12107132</a>]<br></li>
<li>Similar to HilC and HilD , RtsA belongs to the AraC/XylS family of regulators , while RtsB is a helix-turn-helix DNA binding protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15765064" target="_blank">15765064</a>]<br></li>
<li>In addition , these three regulators ( i.e. , HilC , HilD , and RtsA ) activate a regulator within SPI-1 , HilA , which shows homology with the OmpR/ToxR family of DNA-binding proteins and induces transcriptional activation of SPI-1 genes in response to a variety of stimuli ( 1 , 59 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21075923" target="_blank">21075923</a>]<br></li>
<li>In addition to HilC and HilD , RtsA , which belongs to the AraC/XylS family of transcriptional regulators , has been shown to activate expression of SPI1 genes by binding upstream of hilA to induce its expression ( 13 , 14 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18245288" target="_blank">18245288</a>]<br></li>
<li>Most frequently the virulence operons of the PAIs are under direct control of pathogen-speciﬁc transcriptional master regulators ( mostly of the AraC family ) , which are encoded on the PAI ( e.g. , HilA , HilD in Salmonella , LcrF in Yersinia ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26441883" target="_blank">26441883</a>]<br></li>
<li>HilD , a member of the AraC/XylS family of transcriptional regulators , directly induces the expression of HilA21 -- 23 , a regulator with an OmpR/ToxR-like DNA binding 1Programa deGenómicaComputacional , Centro deCienciasGenómicas , Universidad NacionalAutónoma de México , Cuernavaca , Morelos 62210 , México .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27886269" target="_blank">27886269</a>]<br></li>
<li>Since HilD is a member of the AraC/XylS family of regulators , we thought it was possible that HilD contacts RNA polymerase at the CTD to activate transcription of hilA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12511499" target="_blank">12511499</a>]<br></li>
<li>HilD , a member of the AraC/XylS family of transcriptional regulators , derepresses hilA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11083797" target="_blank">11083797</a>]<br></li>
<li>HilD , a member of the AraC/XylS family of transcriptional regulators , induces the expression of hilA , encoding a transcriptional activator of the inv/spa , prg/org , and sic/sip operons [ 3 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28329249" target="_blank">28329249</a>]<br></li>
<li>HilA belongs to the OmpR/ToxR family whereas HilD , HilC and InvF belong to the AraC/XylS family of transcriptional regulators ( Kaniga et al. , 1994 ; Bajaj et al. , 1995 ; Schechter et al. , 1999 ; Lostroh et al. , 2000 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>Both HilC and HilD are predicted to be members of the AraC/XylS family of transcriptional activators , based on homology within their C-terminal domains that contain a characteristic helix-turn-helix DNA binding motif ( Eichelberg et al. , 1999 ; Rakeman et al. , 1999 ; Schechter et al. , 1999 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15765064" target="_blank">15765064</a>]<br></li>
<li>HilD , a member of the AraC family of transcriptional regulators , induces the expression of HilA ( Schechter et al. , 1999 ; Schechter and Lee , 2001 ; Ellermeier et al. , 2005 ) , an OmpR/ToxRlike transcriptional regulator , which in turn , activates the expression of InvF ( Lostroh et al. , 2000 ; Lostroh and Lee , 2001 ) , another AraC-like regulator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26300871" target="_blank">26300871</a>]<br></li>
<li>Within RESULTS SPI1 , HilD , a transcriptional regulator of the AraC family , comprises a portion of an auto-regulatory circuit that includes RtsA and HilC ( Ellermeier et al. , 2005 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23289537" target="_blank">23289537</a>]<br></li>
<li>Induction of SPI1 effector proteins requires three transcriptional regulators of the AraC family : HilD , HilC and RtsA ( Schechter et al. , 1999 ; Ellermeier and Slauch , 2003 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27341691" target="_blank">27341691</a>]<br></li>
<li>Salmonella Typhi and Salmonella Typhimurium both transit the intestinal epithelium in order to mediate disease , and the invasion pathways of these serovars are conserved , as is the HilD protein ( 99.4 % identity ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27185788" target="_blank">27185788</a>]<br></li>
<li>However , HilA activation is dependent upon a coupled , positive-feedback loop comprising the AraC/XylS family transcription factors HilD , HilC , and RtsA , which drive HilA expression above the SPI-1 activation threshold ( 7 , 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24375101" target="_blank">24375101</a>]<br></li>
<li>hilA transcription is elevated in response to two SPI1-encoded AraC/XylS family transcription factors , HilC and HilD ( 8 , 20 , 22 ) , and to the RtsA protein , which is encoded on another SPI ( 9 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15126488" target="_blank">15126488</a>]<br></li>
<li>HilD , HilC , and RtsA are homologous AraC family transcription factors .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24375101" target="_blank">24375101</a>]<br></li>
<li>HilD , an AraC/XylS family member , binds directly to several sites within the hilA promoter and derepresses gene expression , which in turn results in the expression of other associated virulence genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29018419" target="_blank">29018419</a>]<br></li>
<li>HilD and HilC may be members of this new family of regulators whose activity is not restricted to promoter-proximal upstream binding sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>Expression of hilA itself is activated by three AraC family regulators , HilC , HilD , and RtsA ( 25 , 78 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18160484" target="_blank">18160484</a>]<br></li>
<li>HilC and HilD are members of the AraC/XylS family and activate transcription from the p and p promoters in an apparently redundant invF sicA manner ( Akbar et al. 2003 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20008574" target="_blank">20008574</a>]<br></li>
<li>Two members of the AraC/XylS family of transcription activators , HilC and HilD , can independently derepress hilA expression by binding directly to the upstream repressing sequence and counteracting its repressing effect in vitro and in vivo ( 24 , 30 , 31 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16391141" target="_blank">16391141</a>]<br></li>
<li>RtsA belongs to the AraC/XylS family of regulators , and can directly bind the hilA promoter and induce the hilA expression independent of HilC and HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15661008" target="_blank">15661008</a>]<br></li>
<li>Some members of the AraC/XylS family have difﬁcult solubility properties , and the HilC and HilD proteins lost DNA-binding activity upon prolonged storage .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12107132" target="_blank">12107132</a>]<br></li>
<li>The proteins HilD ( 34.3 kDa ) , HilC ( 32.8 kDa ) , and RtsA ( 32.3 kDa ) belong to the AraC/XylS family of transcriptional activators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17675384" target="_blank">17675384</a>]<br></li>
<li>Members of the AraC-like protein family , which includes HilC and HilD , have a well-documented ability to displace H-NS [ 45 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22457642" target="_blank">22457642</a>]<br></li>
<li>Sequence analysis suggests that RtsA is a member of the AraC/XylS family of transcriptional regulators and is related to HilC and HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923082" target="_blank">12923082</a>]<br></li>
<li>Like HilC and HilD proteins , InvF belongs to the AraC/XylS family of transcriptional regulators , and its gene lies within SPI-1 ( 28 , 33 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21075923" target="_blank">21075923</a>]<br></li>
<li>Sequence analysis of the regulatory operon revealed that RtsA belongs to the AraC 72 XylS family of transcriptional regulators and is homologous to both HilC ( 37 % identical and 55 % similar overall ) and HilD ( 34 % identical and 55 % similar overall ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923082" target="_blank">12923082</a>]<br></li>
<li>The third and fourth regulators are HilC and HilD , both of which belong to the AraC family .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17233907" target="_blank">17233907</a>]<br></li>
<li>RtsA , HilC , and HilD all belong to the AraC/XylS family of transcriptional regulators and appear to function in similar ways ( 26 , 65 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14679226" target="_blank">14679226</a>]<br></li>
<li>RtsA belongs to the AraC/XylS family of transcriptional regulators and contains similarity to HilC and HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923082" target="_blank">12923082</a>]<br></li>
<li>HilC and HilD , two SPI1-encoded AraC/XylS family members , have been called derepressors and are thought to counteract the repression of hilA by binding to sites within the upstream repressing sequence ( URS ) ( 24 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12933899" target="_blank">12933899</a>]<br></li>
<li>Three homologous proteins , RtsA , HilC and HilD , which belong to the AraC/XylS family of transcriptional regulators , bind to the DNA immediately upstream of hilA leading to induction of hilA expression ( Schechter and Lee , 2001 ; Olekhnovich and Kadner , 2002 ; Ellermeier and Slauch , 2003 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16045614" target="_blank">16045614</a>]<br></li>
<li>Recently , two SPI1-encoded factors , HilC and HilD , which derepress hilA expression , were identiﬁed and shown to be members of the AraC/XylS family of transcriptional regulators and to independently bind to these upstream repressing sequences to counteract the repression of hilA expression ( 32 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11741864" target="_blank">11741864</a>]<br></li>
<li>HilD and HilC , which are members of the AraC/XylS family of transcriptional regulators , can each individually bind to the DNA immediately upstream of hilA , and it is believed that this binding leads to hilA expression ( 44 , 45 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18723621" target="_blank">18723621</a>]<br></li>
<li>According to this model , HilD would act as a derepressor and not as a typical activator of transcription , which is thought to be the common method of regulation by members of the AraC/XylS family ( 20 , 37 , 42 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12511499" target="_blank">12511499</a>]<br></li>
<li>Expression of the SPI1 T3SS is tightly regulated by the combined action of HilC , HilD , and RtsA , three AraC family members that can independently activate hilA , which encodes the direct regulator of the SPI1 structural genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
<li>HilC and HilD , which are members of the AraC/XylS family , have been postulated to act as a derepressor of hilA expression by counteracting the action of negative regulatory elements at the hilA promoter ( 12,13 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17329372" target="_blank">17329372</a>]<br></li>
<li>HilC and HilD have recently been shown to bind to the hilA promoter , and it is believed that the binding of one or both of these proteins displaces repressors of hilA expression ( 55 ) , although both proteins are members of the AraC family of transcriptional activators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12595445" target="_blank">12595445</a>]<br></li>
<li>In summary , stimulation of hilA transcription by the AraC/XylS family members HilC and HilD does exhibit the expected dependence on residues in - CTD , but activation of transcription involves other - CTD-independent processes as well .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15126488" target="_blank">15126488</a>]<br></li>
<li>Two SPI1-encoded regulators of hilA expression are the AraC/XylS family members , HilC ( also called SirC or SprA ) and HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12107132" target="_blank">12107132</a>]<br></li>
<li>Transcription of hilA is directly activated by HilC and HilD , both members of the AraC/XylS family of transcriptional regulators ( Schechter and Lee 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25991823" target="_blank">25991823</a>]<br></li>
<li>Two SPI1-encoded proteins , HilC ( SirC or SprA ) and HilD , both members of the AraC/XylS family of transcriptional regulators , induce expression of hilA ( 26 , 67 , 71 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923082" target="_blank">12923082</a>]<br></li>
<li>Two homologous proteins , HilC and HilD , both members of the AraC/XylS family of transcriptional regulators , induce hilA expression ( 12 , 51 , 53 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18245288" target="_blank">18245288</a>]<br></li>
<li>Like HilD , HilC is a member of the AraC/XylS family .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11755416" target="_blank">11755416</a>]<br></li>
<li>Similar to HilD , HilC is an activator of hilA of the AraC/XylS family ( 50 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11083797" target="_blank">11083797</a>]<br></li>
<li>Involvement of K297 in HilD 's DNA-Binding Ability HilD , a member of the AraC/XylS family , can bind to the promoters of hilA and invF [ 4 , 19 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28329249" target="_blank">28329249</a>]<br></li>
<li>DNA-Binding Activities of the HilC and HilD Virulence Regulatory Proteins of Salmonella enterica Serovar Typhimurium Igor N. Olekhnovich and Robert J. Kadner * ent of M Departm icrobiology , School of Medicine , University of Virginia , Charlottesville , Virginia 22908-0734 Received 17 September 2001/Accepted 3 May 2002 The HilC and HilD proteins of Salmonella enterica serovar Typhimurium are members of the AraC/XylS family of transcription regulators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12107132" target="_blank">12107132</a>]<br></li>
<li>The HilC and HilD proteins are members of the AraC/XylS family of transcription activators , which typically bind to a 34-bp DNA target ( 18 , 36 , 42 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12107132" target="_blank">12107132</a>]<br></li>
<li>Three homologous proteins , 8,13,14 HilC , HilD , and RtsA , which belong to the AraC/XylS family of transcription factors can activate expression of hilA and some of its target genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16443238" target="_blank">16443238</a>]<br></li>
<li>HilD , HilC , and RtsA are members of the AraC family of transcription factors .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24375101" target="_blank">24375101</a>]<br></li>
				</div></ul><br>
			<label class="collapse_RP" for="_4"><font size="5px">Processes regulated by HilD</font></label>
				<input id="_4" type="checkbox">
				<div class="RP"><br><ul><li>SPI1 also encodes many transcriptional regulators , including HilA , HilD , HilC ( also called SprA/SirC ) , InvF and SprB ( Kaniga et al. , 1994 ; Bajaj et al. , 1995 ; Eichelberg et al. , 1999 ; Rakeman et al. , 1999 ; Schechter et al. , 1999 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>Reference Bioﬁlm-related function Group Quorum ( Merighi et al. 2009 ) a Membrane protein , Quorum Sensing ( AI-3 ) regulated pmrC sensing Metabolism Metabolism Putrescine transporter potF pps ( Hermans et al. 2011 ) ( White et al. 2010 ) Metabolism ( phosphoenolpyruvate synthase , conversion of pyruvate to PEP ) Needle complex inner membrane protein ( T3SS ) , PhoPQ repressed Purine synthesis , adenylosuccinate synthetase Purine synthesis , amidophosphoribosyltransferase Purine synthesis , bifunctional phosphoribosylaminoimidazolecarboxamide formyltransferase/IMP cyclohydrolase Activation of colanic-acid-biosynthesis , transcriptional co-regulator Activation of colanic-acid-biosynthesis , inhibition ﬂagellar biosynthesis Stationary phase sigma factor , stress response Synthetic RpoS-dependent promoter Regulates SPI-1 expression together with HilD and HilC Hypothetical protein ( similar to E. coli vancomycin sensitivity gene ) Responses to Acyl Homoserine Lactones ( AI-1 ) , inﬂuences plasmid encoded ﬁmbriae Transcriptional regulator in LysR family ( regulates motility in B. subtilis ) Response regulator of the BarA/SirA two-component system Fur regulated Salmonella iron-transporter Secreted effector protein ( T3SS ) prgH ( Steenackers et al. 2012 ) ( McClelland et al. 2001 ) a ( McClelland et al. 2001 ) a ( McClelland et al. 2001 ) a Virulence purA purF purH Metabolism Metabolism Metabolism rcsA ( Steenackers et al. 2012 ) Matrix rcsC ( Steenackers et al. 2012 ) Matrix rpoS ( Steenackers et al. 2012 ) ( White et al. 2006 ) ( Golubeva et al. 2012 ) ( Hermans et al. 2011 ) Regulator Synthetic Virulence Stress resistance Quorum sensing Putative sig38H4 rtsA sanA sdiA ( Steenackers et al. 2012 ) sinR ( Folkesson et al. 2002 , Marchal et al. 2004 ) ( Steenackers et al. 2012 ) Regulator sirA Metabolism Virulence sitA ( Hermans et al. 2011 ) sopB ( Hernandez et al. 2004 , Jennings et al. 2012 ) ( Steenackers et al. 2012 ) a Quorum sensing Coiled-coil protein , regulated by SdiA srgE ( Hansen et al. 2005 ) b Metabolism Stringent starvation protein , inhibits stationary-phase accumulation of H-NS Chaperon for sth ﬁmbriae synthesis , bioﬁlm formation on epithelial cells Putative cytoplasmic protein Inner membrane protein Putative ABC transporter periplasmic binding protein Putative cytoplasmic protein Diguanylate cyclase ( c-di-GMP-synthesis ) , activates cellulose production ( matrix ) Putative DNA-binding protein Putative DNA-binding protein Transcriptional regulator ( ﬂagella synthesis and pathogenicity ) Glycosyl transferase for the biosynthesis of colanic acid ( matrix ) Regulator , reduces bioﬁlm formation and induces motility Hypothetical protein Cell wall-associated hydrolase , contains a CsgD-binding site Regulatory protein Hydroxybutyrate dehydrogenase , O-Ag-capsule biosynthesis Regulator of O-Ag-capsule biosynthesis DNA-binding transcriptional repressor FabR , fatty-acid-biosynthesis Synthetic promoter , sigma-70 responsive ( growth control ) Synthetic promoter , sigma-70 responsive ( growth control ) sspA Fimbriae sthA ( Steenackers et al. 2012 ) Putative Putative Putative Putative Matrix sciJ ( Hermans et al. 2011 ) Hermans et al. unpublished ( Hermans et al. 2011 ) ( Hermans et al. 2011 ) ( Steenackers et al. 2012 ) STM0731 STM1255 STM1851 STM1987 Putative Putative Regulator STM3071 STM4423 tdcA ( Hermans et al. 2011 ) ( Hermans et al. 2011 ) ( Golubeva et al. 2012 ) Matrix wcaA ( Steenackers et al. 2012 ) Regulator Putative Putative ybaJ ybeL ydhO ( Barrios et al. 2006 ) ( Hermans et al. 2011 ) ( Ogasawara et al. 2011 ) Putative Matrix yhiW yihU ( Morgan et al. 2004 ) ( Steenackers et al. 2012 ) Matrix yihVW yijC ( Steenackers et al. 2012 ) ( Feng and Cronan 2009 ) a Metabolism ( Kim and Surette 2006 ) ( Stocki et al. 2007 ) σ70-16 σ70-7 Synthetic Synthetic Note : References for the bioﬁlm-related functions in Salmonella are shown .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24735176" target="_blank">24735176</a>]<br></li>
<li>Positive regulators include BarA/SirA , FliZ , EnvZ/OmpR , FadD , CsrB , HilD and some nucleoid binding proteins ( Fis , HU ) , while Pag , HilE , Ams and the nucleoid binding proteins Hha and H-NS are negatively regulating hilA transcription ( Altier et al. 2000a , b Fahlen et al. 2000 ; Lucas et al. 2000 ; Wilson et al. 2001 ; Schechter et al. 2003 ; Baxter et al. 2003 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18563621" target="_blank">18563621</a>]<br></li>
<li>Regulators , inducing signals Local regulators Global regulatory systems Activating environmental factors SPI2 SsrAB SPI1 InvF , HilA , HilC , HilD PhoPQ , OmpR/EnvZ phosphate starvation , low Mg2 + - or Ca2 + - level , late logarithmic to stationary growth phase high phosphate levels , high Mg2 + - or Ca2 + - level SirA , PhoPQ , ( SPI2 ) low O2-levels , high osmolarity , early logarithmic growth phase , bacterial protein-synthesis high O2-levels , stationary growth phase Repressing environmental factors Figure 3 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11418329" target="_blank">11418329</a>]<br></li>
<li>) e , f , u i EnvZ/OmpR , PhoPQ PhoRB , CsrAB , SirA/BarA , HilA , HilC HilD , InvF b , c , f , g , h , i HilA , SirA/BarA j , , SPI4 ( svr genes ) Unknown SPI5 ( sopB , pips ) PMN migration b , c HilA , SirA/BarA j HilA , SirA/BarA j sopE Invasion , cytokine expression b , c , k , l Iron uptake a Iron limitation a Fur a Macrophage line ( IVET ) m entF Mg uptake 21 n Cation levels n PhoPQ n Macrophages mgtCB Mouse spleen and macrophage line ( IVET ) ; epithelial cell line ( other fusion studies ) m SPI2 Cation levels , Pi levels , osmolarity ( ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10844688" target="_blank">10844688</a>]<br></li>
<li>In contrast , HilD positively regulates FlhDC ( see Fig .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27601571" target="_blank">27601571</a>]<br></li>
<li>This regulation is clearly mediated through HilD , but FliZ controls neither hilD transcription nor translation-initiation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20889744" target="_blank">20889744</a>]<br></li>
<li>HilD is not known to control genes outside SPI-1 , except rtsA whose gene product is also involved in SPI-1 transcription ( Ellermeier et al. , 2005 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17971080" target="_blank">17971080</a>]<br></li>
<li>( B ) HilD protein stability in cells in late log phase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20889744" target="_blank">20889744</a>]<br></li>
<li>Therefore , HilD plays a key role in S. Typhimurium pathogenicity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28426789" target="_blank">28426789</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_11">
Ellermeier , J.R. , and Slauch , J.M. ( 2008 ) Fur regulates expression of the Salmonella pathogenicity island 1 type-III-secretion-system through HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19007416" target="_blank">19007416</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24354910" target="_blank">24354910</a>]<br></label>
<input id="rp_similarity_11", type="checkbox">
<div class="RP_similar">
<li>Ellermeier , J. R. , and J. M. Slauch , 2008 Fur regulates expression of the Salmonella pathogenicity island 1 type-III-secretion-system through HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20008574" target="_blank">20008574</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22021388" target="_blank">22021388</a>]<br></li></div>
				<label class="RP_collapse_similarity" for="rp_similarity_13">
Fur regulates expression of the Sal-monella pathogenicity island 1 type-III-secretion-system through HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27704705" target="_blank">27704705</a>]<br></label>
<input id="rp_similarity_13", type="checkbox">
<div class="RP_similar">
<li>Fur regulates expression of the Salmo-nella pathogenicity island 1 type-III-secretion-system through HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23554419" target="_blank">23554419</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24375101" target="_blank">24375101</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26386070" target="_blank">26386070</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27601571" target="_blank">27601571</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25547794" target="_blank">25547794</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24990908" target="_blank">24990908</a>]<br></li></div>
				<label class="RP_collapse_similarity" for="rp_similarity_15">
Ellermeier JR , Slauch JM : Fur regulates expression of the Salmonella pathogenicity island 1 type-III-secretion-system through HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22017966" target="_blank">22017966</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24079299" target="_blank">24079299</a>]<br></label>
<input id="rp_similarity_15", type="checkbox">
<div class="RP_similar">
<li>[ 15 ] J.R. Ellermeier , J.M. Slauch , Fur regulates expression of the Salmonella pathogenicity island 1 type-III-secretion-system through HilD , J. Bacteriol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24937451" target="_blank">24937451</a>]<br></li>
<li>[ 17 ] J.R. Ellermeier , J.M. Slauch , Fur regulates expression of the Salmonella pathogenicity island 1 type-III-secretion-system through HilD , J. Bacteriol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23040276" target="_blank">23040276</a>]<br></li>
<li>Ellermeier , J. R. & Slauch , J. M. Fur regulates expression of the Salmonella pathogenicity island 1 type-III-secretion-system through HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27886269" target="_blank">27886269</a>]<br></li>
<li>[ 26 ] J.R. Ellermeier , J.M. Slauch , Fur regulates expression of the Salmonella pathogenicity island 1 type-III-secretion-system through HilD , J. Bacter-iol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18656407" target="_blank">18656407</a>]<br></li>
<li>Fur regulates expression of the Salmonella pathogenicity island 1 type-III-secretion-system through HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24239962" target="_blank">24239962</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26441883" target="_blank">26441883</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26300871" target="_blank">26300871</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21075923" target="_blank">21075923</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19884332" target="_blank">19884332</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23306672" target="_blank">23306672</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20889744" target="_blank">20889744</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28575106" target="_blank">28575106</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27242152" target="_blank">27242152</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21680637" target="_blank">21680637</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21722794" target="_blank">21722794</a>]<br></li>
<li>Fur regulates expression of the Salmonella patho-genicity island 1 type-III-secretion-system through HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19049822" target="_blank">19049822</a>]<br></li>
<li>Fur regulates expression of the Salmonella pathogeni-city island 1 type-III-secretion-system through HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23831078" target="_blank">23831078</a>]<br></li></div>
<li>2005573691705Original ArticleModelforSPI1RegulationC.D.Ellermeier , J.R.EllermeierandJ.M.Slauch HilD , HilC and RtsA constitute a feed forward loop that controls expression of the SPI1 type three secretion system regulator hilA in Salmonella enterica serovar Typhimurium Craig D. Ellermeier ,1 † Jeremy R. Ellermeier1 and James M. Slauch1 ,2 * 1Department of Microbiology and 2College of Medicine , University of Illinois , Urbana , IL , USA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16045614" target="_blank">16045614</a>]<br></li>
<li>BioFactors 42:13 -- 40 Elhadad D , Desai P , Rahav G , McClelland M , Gal-Mor O ( 2015 ) Flagellin is required for host-cell-invasion and normal Salmonella pathogenicity island 1 expression by Sal-monella enterica serovar Paratyphi A. Infect Immun 83:3355 -- 3368 Ellermeier CD , Ellermeier JR , Slauch JM ( 2005 ) HilD , HilC and RtsA constitute a feed forward loop that controls expression of the SPI1 type three secretion system regulator hilA in Salmonella enterica serovar Typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27549210" target="_blank">27549210</a>]<br></li>
<li>Proc natl Acad Sci 108:4554 -- 4561 ellermeier CD , ellermeier Jr , Slauch JM ( 2005 ) HilD , HilC and rtsA constitute a feed forward loop that controls expression of the SPI1 type three secretion system regulator hilA in Salmonella enterica serovar Typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24929817" target="_blank">24929817</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_26">
HilD , HilC and RtsA constitute a feed forward loop that controls expression of the SPI1 type three secretion system regulator HilA in Salmonella enterica serovar Typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25135218" target="_blank">25135218</a>]<br></label>
<input id="rp_similarity_26", type="checkbox">
<div class="RP_similar">
<li>HilD , HilC and RtsA constitute a feed forward loop that controls expression of the SPI1 type three secretion system regulator hilA in Salmonella enterica serovar Typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19703107" target="_blank">19703107</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23554419" target="_blank">23554419</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23289537" target="_blank">23289537</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17600486" target="_blank">17600486</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23459474" target="_blank">23459474</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19797361" target="_blank">19797361</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21168230" target="_blank">21168230</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27185788" target="_blank">27185788</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26441883" target="_blank">26441883</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20833811" target="_blank">20833811</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16443238" target="_blank">16443238</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26300871" target="_blank">26300871</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28329249" target="_blank">28329249</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21984608" target="_blank">21984608</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28426789" target="_blank">28426789</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28426789" target="_blank">28426789</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27704705" target="_blank">27704705</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28575106" target="_blank">28575106</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21518393" target="_blank">21518393</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27565525" target="_blank">27565525</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23676436" target="_blank">23676436</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24780330" target="_blank">24780330</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21680637" target="_blank">21680637</a>]<br></li>
<li>HilD , HilC and RtsA constitute a feed forward loop that controls expression of the SPI1 type three secretion system regulator hilA in Salmonella enterica serovar Typhi-murium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18723621" target="_blank">18723621</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19218381" target="_blank">19218381</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24375101" target="_blank">24375101</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18245288" target="_blank">18245288</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20190046" target="_blank">20190046</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17951383" target="_blank">17951383</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23230292" target="_blank">23230292</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16988271" target="_blank">16988271</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26386070" target="_blank">26386070</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20889744" target="_blank">20889744</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26884427" target="_blank">26884427</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27601571" target="_blank">27601571</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17675384" target="_blank">17675384</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18469103" target="_blank">18469103</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18441068" target="_blank">18441068</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16980492" target="_blank">16980492</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18160484" target="_blank">18160484</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25547794" target="_blank">25547794</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24488311" target="_blank">24488311</a>]<br></li>
<li>HilD , HilC , and RtsA constitute a feed forward loop that controls expression of the SPI1 type three secretion system regulator hilA in Salmonella enterica serovar Typhi-murium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21075923" target="_blank">21075923</a>]<br></li></div>
<li>J Bacteriol 185:5096 -- 5108 Ellermeier CD , Ellermeier JR , Slauch JM ( 2005 ) HilD , HilC and RtsA constitute a feed forward loop that controls expression of the SPI-1 type three secretion system regulator hilA in Salmo-nella enterica serovar Typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19003447" target="_blank">19003447</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_31">
Ellermeier , C.D. , Ellermeier , J.R , and Slauch , J.M. ( 2005 ) HilD , HilC and RtsA constitute a feed forward loop that controls expression of the SPI1 type three secretion system regulator hilA in Salmonella enterica serovar Typhi-murium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27341691" target="_blank">27341691</a>]<br></label>
<input id="rp_similarity_31", type="checkbox">
<div class="RP_similar">
<li>Ellermeier , C. D. , J. R. Ellermeier , and J. M. Slauch , 2005 HilD , HilC and RtsA constitute a feed forward loop that controls expression of the SPI1 type three secretion system regulator hilA in Salmonella enterica serovar Typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25991823" target="_blank">25991823</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22021388" target="_blank">22021388</a>]<br></li>
<li>Ellermeier , C.D. , Ellermeier , J.R. , and Slauch , J.M. ( 2005 ) HilD , HilC and RtsA constitute a feed forward loop that controls expression of the SPI1 type three secretion system regulator hilA in Salmonella enterica serovar Typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17971080" target="_blank">17971080</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16999831" target="_blank">16999831</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17163975" target="_blank">17163975</a>]<br></li>
<li>Ellermeier , C.D. , Ellermeier , J.R. and Slauch , J.M. ( 2005 ) HilD , HilC and RtsA constitute a feed forward loop that controls expression of the SPI1 type three secretion system regulator hilA in Salmonella enterica serovar Typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17329372" target="_blank">17329372</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24682814" target="_blank">24682814</a>]<br></li>
<li>Ellermeier , C. D. , J. R. Ellermeier and J. M. Slauch , 2005 HilD , HilC and RtsA constitute a feed forward loop that controls expression of the SPI1 type three secretion system regulator hilA in Salmonella enterica serovar Typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20008574" target="_blank">20008574</a>]<br></li>
<li>Ellermeier , C.D. , Ellermeier , J.R. , and Slauch , J.M. ( 2005 ) HilD , HilC and RtsA constitute a feed forward loop that controls expression of the SPI1 type three secretion system regulator hilA in Salmonella enterica serovar Typh-imurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22356617" target="_blank">22356617</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24354910" target="_blank">24354910</a>]<br></li>
<li>Mol Microbiol 48 ( 2 ) :385 -- 400 Ellermeier C , Ellermeier J , Slauch J ( 2005 ) HilD , HilC and RtsA constitute a feed forward loop that controls expression of the SPI1 type three secretion system regulator HilA in Salmonella enterica serovar Typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21107576" target="_blank">21107576</a>]<br></li>
<li>[ 19 ] C.D. Ellermeier , J.R. Ellermeier , J.M. Slauch , HilD , HilC and RtsA constitute a feed forward loop that controls expression of the SPI1 type three secretion system regulator hilA in Salmonella enterica serovar Typhimu-rium , Mol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18656407" target="_blank">18656407</a>]<br></li>
<li>Infect Immun 67 : 4099 -- 4105 Ellermeier CD , Ellermeier JR , Slauch JM ( 2005 ) HilD , HilC and RtsA constitute a feed forward loop that controls expression of the SPI1 type three secretion system regulator hilA in Salmonella enterica serovar Typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19756048" target="_blank">19756048</a>]<br></li>
<li>Clin Infect Dis 48:1096 -- 1101 Ellermeier CD , Ellermeier JR , Slauch JM ( 2005 ) HilD , HilC and RtsA constitute a feed forward loop that controls expression of the SPI1 type three secretion system regulator hilA in Salmonella enterica serovar Typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24500522" target="_blank">24500522</a>]<br></li>
<li>Ellermeier , C. D. , Ellermeier , J. R. & Slauch , J. M. HilD , HilC and RtsA constitute a feed forward loop that controls expression of the SPI1 type three secretion system regulator hilA in Salmonella enterica serovar Typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27886269" target="_blank">27886269</a>]<br></li>
<li>Ellermeier CD , Ellermeier JR , Slauch JM : HilD , HilC and RtsA constitute a feed forward loop that controls expression of the SPI1 type three secretion system regulator hilA in Salmonella enterica serovar Typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17208038" target="_blank">17208038</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24079299" target="_blank">24079299</a>]<br></li></div>
<li>The SPI2 genes also play a role in inducing intestinal inﬂammation and are known to be regulated by HilD , a SPI1 regulator ( 7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20833811" target="_blank">20833811</a>]<br></li>
<li>HilD , HilC , and RtsA not only regulate shared target genes but also activate each other 's transcription ( 8 , 10 , 11 , 20 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27601571" target="_blank">27601571</a>]<br></li>
<li>sion is dependent upon several transcription factors that are important for virulence , including PhoP , RtsA , SirA , HilC , HilD , and Fis ( 9 , 10 , 35 , 61 , 63 , 78 , 90 -- 92 , 97 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15968047" target="_blank">15968047</a>]<br></li>
<li>The ﬂagellar regulon helps control SPI-1 gene-expression through FliZ-mediated posttranslational HilD regulation ( 44 -- 46 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29866910" target="_blank">29866910</a>]<br></li>
<li>We asked if FliZ and RcsCDB also function through HilD to control SPI1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17951383" target="_blank">17951383</a>]<br></li>
<li>Several genetic regulators outside SPI1 have also been shown to control invasion gene-expression through HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27341691" target="_blank">27341691</a>]<br></li>
<li>HilE is a negative regulator that acts upon HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29378886" target="_blank">29378886</a>]<br></li>
<li>We have shown that the 30 UTR-encoded DapZ sRNA is a trans-acting regulator with a GcvB-like seed domain that is transcriptionally activated by the horizontally acquired transcription factor HilD to repress ABC transporter synthesis under conditions that favour host-cell-invasion .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22922465" target="_blank">22922465</a>]<br></li>
<li>The invR gene , which we identiﬁed in a global search for new Salmo-nella sRNA genes , is activated by the major SPI-1 transcription factor , HilD , under conditions that favour host-cell-invasion .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17971080" target="_blank">17971080</a>]<br></li>
<li>Another regulator , FliZ , may activate SPI-1 expression by controlling HilD activity ( 13 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24682814" target="_blank">24682814</a>]<br></li>
<li>Taken together , these results suggest that a protein-acetylation system regulates SPI-1 expression by controlling HilD in a posttranslational manner to mediate S. Typhimurium virulence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26810370" target="_blank">26810370</a>]<br></li>
<li>We recently demonstrated that the sRNA chaperone , Hfq , is involved in SPI-1 expression and/or functions pBAD pBAD-HilD pBAD pBAD-HilD - + - + - + - + w L-ara InvR InvR 5S rRNA 5S rRNA B D PinvR - gfp Δ ( sitA-invH ) 50000 pBAD pBAD-HilD pBAD-HilA - + - + - + 40000 L-ara 30000 InvR 20000 10000 5S rRNA 0 wt hfq hilC hilD Fig. 5 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17971080" target="_blank">17971080</a>]<br></li>
<li>However , a disruption in hilD reduced hilC expression nearly twofold , indicating that HilD regulates hilC expression ( Fig. 3B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11292791" target="_blank">11292791</a>]<br></li>
<li>HilD plays a major role in regulating hilA expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28426789" target="_blank">28426789</a>]<br></li>
<li>pBAD-HilD fully activated invR expression in this mutant background ( Fig. 5D ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17971080" target="_blank">17971080</a>]<br></li>
<li>In contrast , several transcription factors positively or negatively regulate SPI-1 genes by controlling hilD transcription independently of H-NS ( e.g. , Fur ) ( 37 , 38 ) , or by controlling HilD translation ( e.g. , SirA and CsrA ) ( 39 ) or activity ( e.g. , HilE and FliZ ) ( 40 , 41 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27601571" target="_blank">27601571</a>]<br></li>
<li>For example , SPI-2-encoded SsrB downregulates SPI-1 genes by repressing HilA and HilD ( Pérez-Morales et al. , 2017 ) , and reciprocally , SPI-1-encoded HilD upregulates SPI-2 gene-expression by directly binding the ssrAB operon ( Bustamante et al. , 2008 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29034217" target="_blank">29034217</a>]<br></li>
<li>The intestinal fatty acid propionate inhib-its Salmonella invasion through the post-translational control of HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26810370" target="_blank">26810370</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_61">
Boddicker , J.D. , Knosp , B.M. , and Jones , B.D. ( 2003 ) Transcription of the Salmonella invasion gene activator , hilA , requires HilD activation in the absence of negative regulators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16045614" target="_blank">16045614</a>]<br></label>
<input id="rp_similarity_61", type="checkbox">
<div class="RP_similar">
<li>Boddicker , J.D. , Knosp , B.M , and Jones , B.D. ( 2003 ) Transcription of the salmonella invasion gene activator , hilA , requires HilD activation in the absence of negative regulators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27341691" target="_blank">27341691</a>]<br></li>
<li>Transcription of the Salmo-nella invasion gene activator , hilA , requires HilD activation in the absence of negative regulators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23554419" target="_blank">23554419</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25547794" target="_blank">25547794</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24375101" target="_blank">24375101</a>]<br></li></div>
<li>Infect Immun 72:2002 -- 2013 Boddicker JD , Knosp BM , Jones BD ( 2003 ) Transcription of the Salmonella invasion gene activator , hilA , requires HilD activation in the absence of negative regulators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19003447" target="_blank">19003447</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_65">
Transcription of the Salmonella invasion gene activator , hilA , required HilD activation in the absence of negative regulators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16443238" target="_blank">16443238</a>]<br></label>
<input id="rp_similarity_65", type="checkbox">
<div class="RP_similar">
<li>Transcription of the Salmonella invasion gene activator , hilA , requires HilD activation in the absence of negative regulators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19703107" target="_blank">19703107</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18723621" target="_blank">18723621</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19218381" target="_blank">19218381</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23289537" target="_blank">23289537</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14645287" target="_blank">14645287</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18245288" target="_blank">18245288</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20190046" target="_blank">20190046</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15256548" target="_blank">15256548</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26441883" target="_blank">26441883</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12595445" target="_blank">12595445</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923082" target="_blank">12923082</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15126488" target="_blank">15126488</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17675384" target="_blank">17675384</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15731035" target="_blank">15731035</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15039320" target="_blank">15039320</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12933899" target="_blank">12933899</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18441068" target="_blank">18441068</a>]<br></li></div>
<li>We demonstrate transcription activation of 8 genes by HilD ; four of these genes have not been previously described as being regulated by HilD , including lpxR , which encodes a lipid A deacylase important for immune-evasion .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24375101" target="_blank">24375101</a>]<br></li>
<li>Transcription factors of sRNA genes that are associated with membrane perturbation ( OmpR , sE ) are shown in black circles , whereas those of general ( Crp ) or virulence ( HilD ) regulons are shown in green circles .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19007416" target="_blank">19007416</a>]<br></li>
<li>Thus , protein stability may play some role in this control of HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23289537" target="_blank">23289537</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_70">
Eade , C.R. , Hung , C.C. , Bullard , B. , Gonzalez-Escobedo , G. , Gunn , J.S. , Altier , C. , and Payne , S.M. ( 2016 ) Bile acids function synergistically to repress invasion gene-expression in Salmonella by destabilizing the invasion regulator HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29349886" target="_blank">29349886</a>]<br></label>
<input id="rp_similarity_70", type="checkbox">
<div class="RP_similar">
<li>Eade CR , Hung CC , Bullard B , Gonzalez-Escobedo G , Gunn JS , Altier C. Bile acids function synergistically to repress invasion gene-expression in Salmonella by destabilizing the invasion regulator HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28575106" target="_blank">28575106</a>]<br></li>
<li>Bile acids function synergistically to repress invasion gene-expression in Salmonella by destabilizing the invasion regulator HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27886269" target="_blank">27886269</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27185788" target="_blank">27185788</a>]<br></li></div>
				<label class="RP_collapse_similarity" for="rp_similarity_73">
Martínez , L. C. , Banda , M. M. , Fernández-Mora , M. , Santana , F. J. & Bustamante , V. H. HilD induces expression of Salmonella pathogenicity island 2 genes by displacing the global negative regulator H-NS from ssrAB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27886269" target="_blank">27886269</a>]<br></label>
<input id="rp_similarity_73", type="checkbox">
<div class="RP_similar">
<li>HilD induces expression of Salmonella pathogenicity island 2 genes by displacing the global negative regulator H-NS from ssrAB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26300871" target="_blank">26300871</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28329249" target="_blank">28329249</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28660201" target="_blank">28660201</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28426789" target="_blank">28426789</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29447698" target="_blank">29447698</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28426789" target="_blank">28426789</a>]<br></li></div>
				<label class="RP_collapse_similarity" for="rp_similarity_75">
[ PubMed : 12193630 ] Takaya A , Kubota Y , Isogai E , Yamamoto T. Degradation of the HilC and HilD regulator proteins by ATP-dependent Lon protease leads to downregulation of Salmonella pathogenicity island 1 gene-expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21518393" target="_blank">21518393</a>]<br></label>
<input id="rp_similarity_75", type="checkbox">
<div class="RP_similar">
<li>Takaya A , Kubota Y , Isogai E , Yamamoto T. Degradation of the HilC and HilD regulator proteins by ATP-dependent Lon protease leads to downregulation of Salmonella pathogenicity island 1 gene-expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26810370" target="_blank">26810370</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28329249" target="_blank">28329249</a>]<br></li>
<li>J Biol Chem 266 : 815 -- 823 Takaya A , Kubota Y , Isogai E , Yamamoto T ( 2005 ) Degradation of the HilC and HilD regulator proteins by ATP-dependent Lon protease leads to downregulation of Salmonella pathogenicity island 1 gene-expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18200043" target="_blank">18200043</a>]<br></li>
<li>Takaya A. , Kubota Y. , Isogai E. , Yamamoto T. ( 2005 ) Degradation of the HilC and HilD regulator proteins by ATP-dependent Lon protease leads to downregulation of Salmonella pathogenicity island 1 gene-expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22004521" target="_blank">22004521</a>]<br></li>
<li>Takaya , A. , Y. Kubota , E. Isogai and T. Yamamoto , 2005 Degradation of the HilC and HilD regulator proteins by ATP-dependent Lon protease leads to downregulation of Salmo-nella pathogenicity island 1 gene-expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20008574" target="_blank">20008574</a>]<br></li>
<li>Takaya , A. , Kubota , Y. , Isogai , E. , and Yamamoto , T. ( 2005 ) Degradation of the HilC and HilD regulator proteins by ATP-dependent Lon protease leads to downregulation of Salmonella pathogenicity island 1 gene-expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16045614" target="_blank">16045614</a>]<br></li></div>
<li>HilD Induces Expression of Salmonella Pathogenicity Island 2 Genes y Displacing the Global Negative Regulator H-NS from ssrAB b Luary C. Martínez , María M. Banda , Marcos Fernández-Mora , Francisco J. Santana , Víctor H. Bustamante Departamento de Microbiología Molecular , Instituto de Biotecnología , Universidad Nacional Autónoma de México , Cuernavaca , Morelos , Mexico Salmonella pathogenicity islands 1 and 2 ( SPI-1 and SPI-2 ) have essential roles in the pathogenesis of Salmonella enterica .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25135218" target="_blank">25135218</a>]<br></li>
<li>KEYWORDS Salmonella pathogenicity island 1 ; Salmonella invasion ; L-arabinose ; HilD INVASION of epithelial cells by Salmonella enterica requires the expression of genes located in a 40-kb region known as Salmonella pathogenicity island 1 ( SPI-1 ) ( Lostroh and Lee 2001 ; Altier 2005 ; Jones 2005 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25991823" target="_blank">25991823</a>]<br></li>
<li>We show that host intestinal free long-chain-fatty-acids ( LCFAs ) affect Salmonella in - vasion by reducing expression of the SPI1 type-III-secretion-system , acting primarily via the AraC-like activator HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26884427" target="_blank">26884427</a>]<br></li>
<li>Chubiz , J. E. , Y. A. Golubeva , D. Lin , L. D. Miller , and J. M. Slauch , 2010 FliZ regulates expression of the SPI1 invasion locus by controlling HilD protein activity in Salmonella enterica serovar Typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22021388" target="_blank">22021388</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_85">
Microbiol Mol Biol Rev 64:694 -- 708 Chubiz JE , Golubeva YA , Lin D , Miller LD , Slauch JM ( 2010 ) FliZ regulates expression of the Salmonella pathogenicity island 1 invasion locus by controlling HilD protein activity in Salmonella enterica serovar typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25972986" target="_blank">25972986</a>]<br></label>
<input id="rp_similarity_85", type="checkbox">
<div class="RP_similar">
<li>Chubiz JEC , Golubeva YA , Lin D , Miller LD , Slauch JM : FliZ regulates expression of the Salmonella pathogenicity island 1 invasion locus by controlling HilD protein activity in Salmonella enterica serovar typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23295139" target="_blank">23295139</a>]<br></li>
<li>Chubiz JE , Golubeva YA , Lin D , Miller LD , Slauch JM : FliZ regulates expression of the Salmonella pathogenicity island 1 invasion locus by controlling HilD protein activity in Salmonella enterica serovar typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22632036" target="_blank">22632036</a>]<br></li>
<li>Chubiz , J. E. , Golubeva , Y. A. , Lin , D. , Miller , L. D. & Slauch , J. M. FliZ regulates expression of the Salmonella pathogenicity island 1 invasion locus by controlling HilD protein activity in Salmonella enterica serovar typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27886269" target="_blank">27886269</a>]<br></li>
<li>[ 72 ] J.E.C. Chubiz , Y.A. Golubeva , D. Lin , L.D. Miller , J.M. Slauch , FliZ regulates expression of the Salmonella pathogenicity island 1 invasion locus by controlling HilD protein activity in Salmonella enterica serovar typhimurium , J. Bacteriol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29369799" target="_blank">29369799</a>]<br></li></div>
<li>E. coli K-12 lacks around 1,400 genes pres-ent in S. Typhimurium , including several encoding transcriptional regulators , such as HilD , located in the SPIs and other Salmonella islands .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27886269" target="_blank">27886269</a>]<br></li>
<li>Thus , the HilD regulon comprises genes encoding different cellular functions that make to Salmonella an intestinal pathogen .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27886269" target="_blank">27886269</a>]<br></li>
<li>Additionally , HilD also positively regulates several genes necessary for Salmonella replication inside host cells , including the SPI-2 genes4 ,14,29 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27886269" target="_blank">27886269</a>]<br></li>
<li>In silico clustering of Salmonella global gene-expression data reveals novel genes co-regulated with the SPI-1 virulence genes through HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27886269" target="_blank">27886269</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28426789" target="_blank">28426789</a>]<br></li>
<li>Taken together , the results from our clustering analysis show that many genes known to be regulated by HilD , directly or through HilA , InvF or FlhDC , including the SPI-1 genes themselves and genes located in different genomic islands , as well as flagellar/chemotaxis genes4 ,34,40,41 , are indeed co-expressed with SPI-1 ( Fig. 1 ; Table S1 in Supplementary File 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27886269" target="_blank">27886269</a>]<br></li>
<li>In this work , by applying a clustering method to S. Typhimurium SL1344 global expression data from the COLOMBOS database , we show that most of the known genes regulated by HilD , including the flagellar/chemot-axis genes , are indeed co-expressed with SPI-1 ; moreover , nine novel genes that are co-expressed with SPI-1 were identified : gtgE , phoH , sinR , SL1028 , lpxR , SL1896 , SL3812 , SL4247 and SL4433 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27886269" target="_blank">27886269</a>]<br></li>
<li>HilE is a negative regulator of HilD activity that works by direct protein -- protein interaction ( Baxter et al. 2003 ; J. E. Chubiz and J. M. Slauch , unpublished data ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22021388" target="_blank">22021388</a>]<br></li>
<li>One of these , HilE , is a negative regulator of SPI1 that directly binds HilD protein ( Baxter et al. 2003 ; J. E. Chubiz and J. M. Slauch , unpublished results ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22021388" target="_blank">22021388</a>]<br></li>
<li>Furthermore , HilD regulates directly , or indirectly , through HilA and InvF , the expression of several other genes located outside SPI-1 , including acquired and ancestral genes ( Bustamante et al. , 2008 ; Golubeva et al. , 2012 ; Fàbrega and Vila , 2013 ; Petrone et al. , 2014 ; Singer et al. , 2014 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26300871" target="_blank">26300871</a>]<br></li>
<li>The expression of SPI-1 genes plays an important role in Salmonella invasion of epithelial cells , which requires transcriptional regulators such as HilA , HilC , HilD , InvA , InvC , InvF , and SirA , ( Johnston et al. 1996 ; Ochman et al. 1996 ; Rakeman et al. 1999 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24500522" target="_blank">24500522</a>]<br></li>
<li>It has been further implicated in post-transcriptional regulation of the Spi-1-encoded regulator HilD ( Ellermeier et al. , 2005 ; Golubeva et al. , 2012 ) and functions as an activator of Spi-2 virulence genes ( Lee et al. , 2000 ; see also Regulation of Motility , Attachment and Invasion of the Intestinal Epithelium ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26441883" target="_blank">26441883</a>]<br></li>
<li>Based on previous knowledge ( Johnston et al. , 1996 ; Ahmer et al. , 1999 ; Goodier and Ahmer , 2001 ; Altier et al. , 2000a ; 2000b ; Lawhon et al. , 2003 ; Teplitski et al. , 2003 ; Fortune et al. , 2006 ; Teplitski et al. , 2006 ) and new information generated from this study , we present the integration of the regulatory cascade that controls expression of the SPI-1 and SPI-2 virulence regulons via the SirA/BarA and Csr global regulatory systems and the Salmonella-specific regulator HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21518393" target="_blank">21518393</a>]<br></li>
<li>Environmental signals from two-component systems and global regulators control the expression of the HilC , HilD , and HilA transcription factors , which together form a commitment circuit for the expression of SPI-1 genes ( Bajaj et al , 1996 ; Eichelberg and Galan , 1999 ; Lundberg et al , 1999 ; Kalir et al , 2001 ; Ellermeier et al , 2005 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19756048" target="_blank">19756048</a>]<br></li>
<li>This expression proﬁle of virulence genes during planktonic growth is consistent with previous data ( Kage et al. , 2008 ; Lin et al. , 2008 ; Chubiz et al. , 2010 ) showing that FliZ controls the virulence expression through control of HilD at a post-translational level .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22356617" target="_blank">22356617</a>]<br></li>
<li>Furthermore , CsrA regulates expression of the invasion-associated type-III-secretion-system of S. typhimurium via post-transcriptional regulation of HilD ( Altier et al , 2000 ; Fortune et al , 2006 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24056837" target="_blank">24056837</a>]<br></li>
<li>It also demonstrates BarA/SirA and EnvZ/OmpR control of hilA through HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17208038" target="_blank">17208038</a>]<br></li>
<li>Previous studies have shown a regulatory link between the ﬂagellar regulon and SPI-1 via FliZ , which was shown to posttranscriptionally activate HilD , the master regulator that controls SPI-1 expression ( 28 , 29 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26056383" target="_blank">26056383</a>]<br></li>
<li>HilD acts as a regulatory hub for virulence coordinating the expression of several virulence factors encoded inside and outside SPI-1 [ 18,28,40 -- 42 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28426789" target="_blank">28426789</a>]<br></li>
<li>Integration of a complex regulatory cascade involving the SirA / BarA and Csr global regulatory systems that controls expression of the Salmonella SPI-1 and SPI-2 virulence regulons through HilD Luary C. Martínez1 , Helen Yakhnin2 , Martha I. Camacho3 , Dimitris Georgellis3 , Paul Babitzke2 , José L. Puente1 , and Víctor H. Bustamante1 , * 1Departamento de Microbiología Molecular , Instituto de Biotecnología , Universidad Nacional Autónoma de México .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21518393" target="_blank">21518393</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_109">
Martinez , L.C. , Yakhnin , H. , Camacho , M.I. , Georgellis , D. , Babitzke , P. , Puente , J.L. and Bustamante , V.H. ( 2011 ) Integration of a complex regulatory cascade involving the SirA/BarA and Csr global regulatory systems that controls expression of the Salmonella SPI-1 and SPI-2 virulence regulons through HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24682814" target="_blank">24682814</a>]<br></label>
<input id="rp_similarity_109", type="checkbox">
<div class="RP_similar">
<li>Martinez , L.C. , Yakhnin , H. , Camacho , M.I. , Georgellis , D. , Babitzke , P. , Puente , J.L , and Bustamante , V.H. ( 2011 ) Integration of a complex regulatory cascade involving the SirA/BarA and csr global regulatory systems that controls expression of the salmonella SPI-1 and SPI-2 virulence regulons through HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27341691" target="_blank">27341691</a>]<br></li>
<li>PLoS Pathog 7 : e1002253 Martínez LC et al ( 2011 ) Integration of a complex regulatory cascade involving the SirA/BarA and Csr global regulatory systems that controls expression of the Salmonella SPI-1 and SPI-2 virulence regulons through HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24500522" target="_blank">24500522</a>]<br></li>
<li>Martinez , L. C. , H. Yakhnin , M. I. Camacho , D. Georgellis , P. Babitzke et al. , 2011 Integration of a complex regulatory cascade involving the SirA/BarA and Csr global regulatory systems that controls expression of the Salmonella SPI-1 and SPI-2 virulence regulons through HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22021388" target="_blank">22021388</a>]<br></li>
<li>[ 42 ] L.C. Martınez , H. Yakhnin , M.I. Camacho , D. Georgellis , P. Babitzke , J.L. Puente , et al. , Integration of a complex regulatory cascade involving the SirA/BarA and Csr global regulatory systems that controls expression of the Salmonella SPI-1 and SPI-2 virulence regulons through HilD , Mol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23040276" target="_blank">23040276</a>]<br></li>
<li>Integration of a complex regulatory cascade involving the SirA/BarA and Csr global regulatory systems that controls expression of the Salmonella SPI-1 and SPI-2 virulence regulons through HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23289537" target="_blank">23289537</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27185788" target="_blank">27185788</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26441883" target="_blank">26441883</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29473025" target="_blank">29473025</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27920756" target="_blank">27920756</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26300871" target="_blank">26300871</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28426789" target="_blank">28426789</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29751061" target="_blank">29751061</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22546935" target="_blank">22546935</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26884427" target="_blank">26884427</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27601571" target="_blank">27601571</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24187088" target="_blank">24187088</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25135218" target="_blank">25135218</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23676436" target="_blank">23676436</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22843534" target="_blank">22843534</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27886269" target="_blank">27886269</a>]<br></li>
<li>New York : Cold Spring Harbor Laboratory Press Martinez LC , Yakhnin H , Camacho MI , Georgellis D , Babitzke P , Puente JL , Bustamante VH ( 2011 ) Integration of a complex regulatory cascade involving the SirA/BarA and Csr global regulatory systems that controls expression of the Salmonella SPI-1 and SPI-2 virulence regulons through HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24056837" target="_blank">24056837</a>]<br></li>
<li>Martinez LC , Yakhnin H , Camacho MI , Georgellis D , Babitzke P , Puente JL , Bustamante VH : Integration of a complex regulatory cascade involving the SirA/BarA and Csr global regulatory systems that controls expression of the Salmonella SPI-1 and SPI-2 virulence regulons through HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23295139" target="_blank">23295139</a>]<br></li>
<li>Martínez LC , Yakhnin H , Camacho MI , Georgellis D , Babitzke P , Puente JL , Bustamante VH : Integration of a complex regulatory cascade involving the SirA/BarA and Csr global regulatory systems that controls expression of the Salmonella SPI-1 and SPI-2 virulence regulons through HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24079299" target="_blank">24079299</a>]<br></li></div>
<li>This boost in virulence was suggested to be caused by increased flagellar expression , including the FliZ regulator controlling the SPI1 regulator HilD at post-transcriptional level ( Kage et al. , 2008 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23676436" target="_blank">23676436</a>]<br></li>
<li>There is certainly a strong tie between the two ; FliZ is a signiﬁcant regulator of HilD activity and has been shown to play a role in Salmonella virulence only during oral infection , the observed virulence phenotype being largely dependent on SPI1 ( Chubiz et al. 2010 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22021388" target="_blank">22021388</a>]<br></li>
<li>Another SPI regulator , HilD , which is not associated with Fisbinding , acts in an ordered fashion with HilC to coordinately activate expression of the SPI-1 genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23717649" target="_blank">23717649</a>]<br></li>
<li>One such case was observed for HilD , a major transcriptional regulator involved in expression of SPI-1 genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24079299" target="_blank">24079299</a>]<br></li>
<li>A central role of HilD in regulating SPI-1 gene-expression has been previously described since many signaling-pathways that influence SPI-1 expression converge in controlling HilD [ 28 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28426789" target="_blank">28426789</a>]<br></li>
<li>To conﬁrm the involvement of HilD in bile-mediated repression of SPI-1 , we assessed chromosomal sipC : : lacZY expression in a hilD mutant ( Fig. 3B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27185788" target="_blank">27185788</a>]<br></li>
<li>Transcriptional activators of SPI genes ( HilC , HilD , SsrB , and SlyA ) function in large part through displacement of H-NS from SPI promoters [ 16,44 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22457642" target="_blank">22457642</a>]<br></li>
<li>HilD inhibition causes SPI-1 repression , which results in impaired secretion of SPI-1 effectors and reduced invasion of epithelial cells .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25991823" target="_blank">25991823</a>]<br></li>
<li>The novel low-O2 signal-transduction-pathway reported here constitutes both global regulators ( ArcB/ArcA ) and a specifically acquired regulator ( LoiA ) , and direct regulators of SPI-1 ( HilA , HilD ) , indicating the complex mechanisms of SPI-1 regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28575106" target="_blank">28575106</a>]<br></li>
<li>'' Levels of active HilD , HilC , RtsA , and HilA are controlled by many different regulators , indirectly impacting expression of other SPI-1 genes ( 9 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27601571" target="_blank">27601571</a>]<br></li>
<li>The expression of SPI-1 genes is tightly regulated by a number of transcriptional regulators , including HilD , HilA , and InvF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28439039" target="_blank">28439039</a>]<br></li>
<li>Constitutive expression of such regulators , HilA and HilD , failed to restore the expression levels of most SPI-1-encoded proteins , implicating a post-transcriptional mechanism for SPI-1 inhibition upon H2O2 treatment .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28193547" target="_blank">28193547</a>]<br></li>
<li>In addition , these three regulators ( i.e. , HilC , HilD , and RtsA ) activate a regulator within SPI-1 , HilA , which shows homology with the OmpR/ToxR family of DNA-binding proteins and induces transcriptional activation of SPI-1 genes in response to a variety of stimuli ( 1 , 59 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21075923" target="_blank">21075923</a>]<br></li>
<li>Expression of the SPI-1 T3SS and its associated genes ( several of which are encoded outside the SPI-1 pathogenicity island ) is controlled by a hierarchy of regulators ( HilD , HilA , HilC , RtsA , InvF ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29229736" target="_blank">29229736</a>]<br></li>
<li>First , Lon is induced to control turnover of HilD and HilC and hence limit the expression of SPI-1 genes , which is of particular relevance after invasion of epithelial cells .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23554419" target="_blank">23554419</a>]<br></li>
<li>InvF and HilA are transcriptional regulators of SPI-1 genes that are encoded within SPI-1 ; hilA itself is regulated by several factors , including PhoP-PhoQ , SirA , HilC , and HilD ( 3 -- 5 , 20 , 37 , 44 , 66 , 72 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10633113" target="_blank">10633113</a>]<br></li>
<li>SPI-1 genes are organized in seven or more transcriptional units , whose expression is under the control of four SPI-encoded transcription factors : HilA , HilC , HilD , and InvF ( Lostroh and Lee 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20008574" target="_blank">20008574</a>]<br></li>
<li>The fundamental transcriptional regulators , HilA , HilC , HilD , InvF , HilE , and RtsA inside or outside SPI-1 play an important role in ﬁne-tuning the expression of SPI-1 [ 34 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23370732" target="_blank">23370732</a>]<br></li>
<li>In addition , HilD and HilC are important AraC/XylS transcriptional regulators that mediate expression of HilA , which then activates all genes necessary for a functional SPI-1 encoded T3SS ( 29 , 30 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22004521" target="_blank">22004521</a>]<br></li>
<li>Thus , HilD coordinates expression of SPI-1 and non-SPI-1 genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24375101" target="_blank">24375101</a>]<br></li>
<li>Within SPI-1 there are key transcriptional activators which regulate expression of SPI-1 genes : HilC , HilD , HilA , and InvF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22291596" target="_blank">22291596</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22291596" target="_blank">22291596</a>]<br></li>
<li>HilD activates transcription of genes within and outside SPI-1 , including six other TFs .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27601571" target="_blank">27601571</a>]<br></li>
<li>HilD also induces directly , or indirectly through HilA , InvF or other regulators , the expression of several other virulence genes , including genes located in SPI-2 , SPI-4 , SPI-5 and other islands4 ,14,29 -- 34 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27886269" target="_blank">27886269</a>]<br></li>
<li>HilD also directly controls the expression of the SPI-1 genes hilD , hilC , and invF , as well as other acquired and ancestral genes located outside SPI-1 , such as rtsA , ﬂhDC , siiA , lpxR , ytfK , , and ( 2 , 25 -- 31 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25135218" target="_blank">25135218</a>]<br></li>
<li>Given that all other described HilD-activated genes have an established connection to virulence , regulation of , , and ytfK by HilD implicates these genes in virulence-associated processes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24375101" target="_blank">24375101</a>]<br></li>
<li>The Spi1 genes are highly regulated by a set of DNA-binding proteins , including the AraC-like regulators HilD , HilC , and RtsA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24488311" target="_blank">24488311</a>]<br></li>
<li>DISCUSSION Modulation by Pat on HilD K297 Acetylation The SPI-1 genes are tightly regulated by a set of transcription factors , including HilD , HilC , and RtsA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28329249" target="_blank">28329249</a>]<br></li>
<li>We tested possible dominant effects of simultaneous overproduction of both HilD ( acting as activator of P5 transcription ) and RtsB ( acting as repressor of P1 transcription ) on ﬂhDC operon transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24488311" target="_blank">24488311</a>]<br></li>
<li>crossmark Bile Acids Function Synergistically To Repress Invasion Gene Expression in Salmonella by Destabilizing the Invasion Regulator HilD Colleen R. Eade , a Chien-Che Hung , a Brian Bullard , b Geoffrey Gonzalez-Escobedo , b John S. Gunn , b Craig Altiera Department of Population Medicine and Diagnostic Sciences , College of Veterinary Medicine , Cornell University , Ithaca , New York , USAa ; Center for Microbial Interface Biology , Department of Microbial Infection and Immunity , The Ohio State University , Columbus , Ohio , USAb Salmonella spp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27185788" target="_blank">27185788</a>]<br></li>
<li>The Intestinal Fatty Acid Propionate Inhibits Salmonella Invasion through the Post-translational Control of HilD Chien-Che Hung1 , Cherilyn D. Garner1 , James M. Slauch2 ,3 , Zachary W. Dwyer1 , Sara D. Lawhon4 , Jonathan G. Frye5 , Michael McClelland6 , Brian M. M. Ahmer7 ,8 , and Craig Altier1 , * 1Department of Population Medicine and Diagnostic Sciences , College of Veterinary Medicine , Cornell University , Ithaca , NY 14853 , USA 2Department of Microbiology , University of Illinois , Urbana , IL 61801 , USA 3College of Medicine , University of Illinois , Urbana , IL 61801 , USA 4Department of Veterinary Pathobiology , College of Veterinary Medicine and Biomedical Sciences , Texas A&M University , College Station , TX 77843 , USA 5Bacterial Epidemiology and Antimicrobial Resistance Research Unit , U.S. Department of Agriculture , Agriculture Research Service , Richard B. Russell Research Center , Athens , GA 30605 , USA 6Department of Pathology and Laboratory Medicine , University of California , Irvine , Irvine , CA 92617 , USA 7Department of Microbiology , The Ohio State University , Columbus , OH 43210 , USA 8Center for Microbial Interface Biology , The Ohio State University , Columbus , OH 43210 , USA SUMMARY To cause disease , Salmonella must invade the intestinal epithelium employing genes encoded within Salmonella Pathogenicity Island 1 ( SPI1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23289537" target="_blank">23289537</a>]<br></li>
<li>( A ) Immunoblotting analysis of the cellular lysates using anti-SipB , anti-SipC , anti-HilA , anti-HilC , and anti-HilD sera .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18245288" target="_blank">18245288</a>]<br></li>
<li>The method and parameters that we initially used in this study for clustering the S. Typhimurium SL1344 global gene-expression results from the COLOMBOS database , were successful to identify gtgE , phoH , sinR , lpxR , SL1896 and SL4247 , as novel genes regulated by HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27886269" target="_blank">27886269</a>]<br></li>
<li>In addition to the negative control that was exerted by H-NS/Hha under low-osmolarity conditions , the homologous virulence activators HilD , HilC , and RtsA ( Hil activators ) induced rtsA-lac expression in a high-salt medium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17675384" target="_blank">17675384</a>]<br></li>
<li>HilD , but not HilA or InvF , regulates the expression of gtgE , phoH , sinR , lpxR , SL1896 and SL4247 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27886269" target="_blank">27886269</a>]<br></li>
<li>As shown above , overexpression of Fur caused an eightfold increase in hilA expression , and this effect was dependent on HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
<li>As shown in Fig. 8 , the increased effect of Lon depletion on hilA expression was abolished by depletion of both HilC and HilD , suggesting that both proteins are also involved in regulation of SPI1 by Lon under SPI1-repressing conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15661008" target="_blank">15661008</a>]<br></li>
<li>Figure 2B shows that in the hilA hilD mutant , HilD overproduction induces expression of invF , sipA and sipC , whereas expression of prgH , prgK and orgA remains unaffected .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>Effect of overexpression of HilD on SPI1 gene-expression in hilD and hilA hilD mutants .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>Signaling cascades triggered by different environmental and physiological signals converge to control HilD , an AraC regulator that coordinates the expression of several virulence factors .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28426789" target="_blank">28426789</a>]<br></li>
<li>Most SPI1 genes are negatively regulated by HilE through sequestration of HilD , the major transcriptional activator of SPI1 , while SopA is a secreted effector protein that is involved in causing inflammation and diarrhea .59,60 IsrM therefore aids in choreographing the expression of virulence factors .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22546935" target="_blank">22546935</a>]<br></li>
<li>Our current understanding suggests that all of the known regulatory systems involved in SPI1 regulation control expression or activity of HilD at some level .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17208038" target="_blank">17208038</a>]<br></li>
<li>The present report suggests that all of the global regulators control SPI1 expression in a HilD-dependent manner , that is , through posttranscriptional or posttranslational control of HilD , which in turn activates hilC and rtsA ( 15 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18723621" target="_blank">18723621</a>]<br></li>
<li>Primary examples of factors that inﬂuence HilD activity are HilE , a major negative regulator of SPI1 T3SS expression , and the ﬂagellar gene product FliZ , which acts indirectly upon HilD to increase activity ( 10 , 16 , 18 -- 20 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29378886" target="_blank">29378886</a>]<br></li>
<li>HilE is a major negative regulator of HilD and therefore of SPI1 T3SS expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29378886" target="_blank">29378886</a>]<br></li>
<li>Discussion To understand how Lon negatively regulates the expression of SPI1 genes , including the key regulator hilA , we have focused on two activators of hilA , HilC and HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15661008" target="_blank">15661008</a>]<br></li>
<li>This conﬁrmed that HilD controls expression of rtsA and that Fur works exclusively via HilD to control the expression of SPI1 genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
<li>Our evidence shows that ClpXP regulates cellular levels of HilC and HilD via the control of ﬂagellar gene-expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18245288" target="_blank">18245288</a>]<br></li>
<li>Briefly , ArcB responds to low oxygen conditions when bacteria reach the distal ileum and undergoes autophosphorylation , following which the phosphate group is transferred to ArcA ; phosphorylated ArcA ( ArcA-P ) activates loiA gene-expression indirectly through unknown regulator ( s ) ; LoiA activates HilD directly through binding the hilD promoter ; HilD then activates hilA and other SPI-1 genes , and thus facilitates the invasion process .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28575106" target="_blank">28575106</a>]<br></li>
<li>HilD has been shown to regulate its own expression ( Ellermeier et al. , 2005 ) , and thus mutants unable to produce the protein exhibit reduced expression of the gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23289537" target="_blank">23289537</a>]<br></li>
<li>Similarly , the expression of slrP , also encoding an effector protein secreted through both T3SS-1 and T3SS-2 , is controlled by HilD in SPI-1-inducing conditions and by the response regulator PhoP in SPI-2-inducing conditions32 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27886269" target="_blank">27886269</a>]<br></li>
<li>InvF is a transcriptional factor , and given the cross regulation of SPI-1 and SPI-2 by the SPI-1-encoded HilD regulator ( 17 ) , it is possible that in vivo invF is needed to control other genes in Salmonella involved in systemic disease .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20028809" target="_blank">20028809</a>]<br></li>
<li>HilD and SlyA cooperate to directly control the expression of ssrAB in SPI-1-inducing conditions ( our unpublished results ) , which could also apply for lpxR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27886269" target="_blank">27886269</a>]<br></li>
<li>HilD , an AraC/XylS family member , binds directly to several sites within the hilA promoter and derepresses gene-expression , which in turn results in the expression of other associated virulence genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29018419" target="_blank">29018419</a>]<br></li>
<li>Importantly , our combined transcriptomic and coIP data revealed that Hfq exerts a direct role in gene-expression through the control of specific check-points in other well-defined transcriptional regulons , such as HilD in the SPI-1 virulence regulon , and FlhD2C2 in the flagellar gene-expression cascade .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18725932" target="_blank">18725932</a>]<br></li>
<li>The main regulatory proteins involved in the expression of Spi-2 include the two-component systems SpiR ( SsrA ) / SsrB , PhoP/PhoQ , OmpR/EnvZ , as well as the DNA-binding proteins HilD , SlyA ( RovA in Yersinia ) , RcsB , the iron regulator Fur , and the nucleoid-associated proteins H-NS , Hha ( YmoA in Yersinia ) , YdgT , IHF , and Fis ( Fass and Groisman , 2009 ; Choi et al. , 2014 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26441883" target="_blank">26441883</a>]<br></li>
<li>The expression of the type-III-secretion-system-related genes involved in invasion and intracellular survival is induced in the presence of transcriptional regulator proteins such as HilA , HilC , HilD , InvA , InvC , InvF , SirA , and SirB ( 11 , 27 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24780330" target="_blank">24780330</a>]<br></li>
<li>The expression of the genes , those regulate the type-III-secretion-system , are involved in completion of invasion , intracellular survival and multiplication , is induced by the transcriptional regulator proteins such as HilA , HilC , HilD , InvA , InvC , InvF , SirA , and SirB ( Johnston et al. , 1996 ; Rakeman et al. , 1999 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27565525" target="_blank">27565525</a>]<br></li>
<li>We tested the effect of mutations in genes encoding important virulence regulators : HilA , HilD , SsrB , PhoP , RcsB , and Dam .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25972862" target="_blank">25972862</a>]<br></li>
<li>Our results further expand the regulon of HilD and thus identify novel possible Salmonella virulence genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27886269" target="_blank">27886269</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_177">
Therefore , our results further expand the virulence regulon of HilD and identify novel genes possibly involved in the pathogenesis of Salmonella .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27886269" target="_blank">27886269</a>]<br></label>
<input id="rp_similarity_177", type="checkbox">
<div class="RP_similar">
<li>Our findings further expand the virulence regulon of HilD and reveal novel factors possibly involved in the pathogenesis of Salmonella .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27886269" target="_blank">27886269</a>]<br></li></div>
<li>This allows us to monitor transcription and translation of hilD without the complication of HilD autoinduction ( Figure 2C ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22021388" target="_blank">22021388</a>]<br></li>
<li>Although much is known about the control of HilD itself , and a number of regulators have been identified , none was shown to be important for the effects of propionate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23289537" target="_blank">23289537</a>]<br></li>
<li>SirA activates the expression of SsrB and HilA through HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21518393" target="_blank">21518393</a>]<br></li>
<li>controls the expression of HilD through the CsrB and CsrC RNA molecules In other bacteria , SirA/BarA orthologs control gene-expression in a cascade fashion by directly activating the expression of csrB and csrC , encoding the small untranslated RNA molecules CsrB and CsrC , respectively ( Babitzke and Romeo , 2007 ; Lapouge et al. , 2008 ; Lucchetti-Miganeh et al. , 2008 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21518393" target="_blank">21518393</a>]<br></li>
<li>Genome-wide transcriptional and/or binding analyses support that H-NS represses gtgE , sinR , lpxR , SL1896 and SL424773 -- 75 ; however , whether HilD induces the expression of these genes by acting as an anti-repressor of H-NS , and how it positively controls the expression of phoH , remains to be determined .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27886269" target="_blank">27886269</a>]<br></li>
<li>These results show that HilD positively controls the expression of the gtgE , phoH , sinR , lpxR , SL1896 and SL4247 genes , independently of HilA and InvF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27886269" target="_blank">27886269</a>]<br></li>
<li>Thus , HilD positively and directly regulates the expression of the gtgE , phoH , sinR , lpxR and SL4247 genes , and positively but indirectly controls the expression of the SL1896 gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27886269" target="_blank">27886269</a>]<br></li>
<li>In contrast , the deacetylated HilD tends to be degraded and fails to activate the transcription of HilA and thus downregulates the expression of SPI-1 , leading to an attenuated S. Typhimurium virulence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26810370" target="_blank">26810370</a>]<br></li>
<li>HilD activates transcription of several SPI-1 genes , including components of the T3SS , secreted effector proteins ( 10 -- 12 ) , and the TFs HilA and InvF ( 7 , 10 , 11 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27601571" target="_blank">27601571</a>]<br></li>
<li>Dam-dependent regulation of SPI-1 is transmitted via HilD : In an attempt to identify the SPI-1 regulator ( s ) , if any , involved in transmission of Dam-dependent control to SPI-1 , we examined the involvement of the SPI-1 `` general '' transcription factors HilA , HilC , and HilD and the sip-speciﬁc transcription factor InvF ( Darwin and Miller 1999 ; Eichelberg and Galan 1999 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20008574" target="_blank">20008574</a>]<br></li>
<li>Notably , according to its role as a central regulator of the SPI-1 and several other virulence genes , the expression , concentration and activity of HilD is highly controlled .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26300871" target="_blank">26300871</a>]<br></li>
<li>HilD then activates expression of hilA and ssrAB , the genes encoding the central regulators of SPI-1 and SPI-2 , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21518393" target="_blank">21518393</a>]<br></li>
<li>SirA , but not HilD , requires the presence of CsrB and CsrC to induce secretion of SPI-1-encoded proteins and expression of the SPI-2 effector protein SseB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21518393" target="_blank">21518393</a>]<br></li>
<li>Fis affects expression of invF : : Tn5lacZY in the absence of HilA Current models for the regulation of invasion genes suggest that there is a HilA-independent pathway to invF expression that involves HilD ( Lucas and Lee , 2000 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11123690" target="_blank">11123690</a>]<br></li>
<li>As predicted , overproduction of HilD from a PBAD expression plasmid restored SPI-1 effector secretion to almost wild-type levels in the absence of Hfq ( Figure 9A ; compare lanes 1 and 4 ) , and also rescued expression of the PrgI needle protein indicative of a functional SPI-1 secretion apparatus ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18725932" target="_blank">18725932</a>]<br></li>
<li>On the basis of these results , we propose that Lon negatively regulates the expression of SPI1 genes through degradation of HilC and HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15661008" target="_blank">15661008</a>]<br></li>
<li>Moreover , HilD coordinates hilA transcription with the expression of other crucial infection-relevant systems , e.g. , it induces expression of the ﬂagellar operons through activation of ﬂhDC transcription and controls expression of Spi-2 genes ( see also the Sections Regulation of Motility and Regulatory Circuits Controlling Later Stages of Infection and Defense Systems against the Host 's Immune Response ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26441883" target="_blank">26441883</a>]<br></li>
<li>To determine the effect of bile on HilD stability in S. Typhi , S. Typhi Ty2 was transformed with constitutively expressed hemagglutinin ( HA ) - tagged HilD ( from S. Typhi Ty2 ) and subcultured in the presence or absence of bile , and samples were taken at regular intervals following the inhibition of protein-synthesis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29229736" target="_blank">29229736</a>]<br></li>
<li>In this report , we investigate the role of nucleoid-associated proteins Hha and H-NS in the expression of virulence activator proteins HilD , HilC , and RtsA with an emphasis on the interplay of positive and negative transcriptional factors in the regulation of the rtsA promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17675384" target="_blank">17675384</a>]<br></li>
<li>In agreement with our results indicating that CpxR represses the HilD-dependent expression of hilD and hilA , both the overexpression of CpxR and the absence of CpxA drastically reduced the production of SsrB-FLAG and SseB proteins ( Figures 1C , 3B ) , which are encoded in SPI-2 and whose expression is also dependent of HilD in the condition tested .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26300871" target="_blank">26300871</a>]<br></li>
<li>Under these growth conditions , CsrB and CsrC counteract CsrA-mediated translational repression of hilD , thereby allowing the synthesis of HilD , which in turn positively controls the regulatory cascade leading to the expression of the SPI-1 and SPI-2 genes that have essential roles in Salmonella virulence ( 11 -- 15 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24187088" target="_blank">24187088</a>]<br></li>
<li>moreover , we demonstrate that CpxR-P negatively affects the stability of HilD and thus decreases the expression of HilD-target genes , such as hilD itself and hilA , located in SPI-1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26300871" target="_blank">26300871</a>]<br></li>
<li>Taken together , these results show that CpxR represses the autoregulation of HilD , which in turn affects the expression of hilA and thus the SPI-1 genes , as well as of other virulence genes regulated by HilD , such as ssrB and sseB located in SPI-2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26300871" target="_blank">26300871</a>]<br></li>
<li>This study identiﬁes the ﬁrst regulatory RNA of an enterobacterial pathogenicity island , and new roles for Hfq and HilD in SPI-1 gene-expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17971080" target="_blank">17971080</a>]<br></li>
<li>Another example of functional redundancy is the Csr system which modulates carbon metabolism and also regulates SPI1 and SPI2 expression through HilD .38 It comprises two sRNAs , CsrB and CsrC and the RNA chaperone CsrA .39,40 Although the single mutants ΔcsrB or ΔcsrC are not impaired in their ability to infect epithelial cells , a ΔcsrB ΔcsrC mutant shows a significant invasion defect .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22546935" target="_blank">22546935</a>]<br></li>
<li>Keywords SirA/BarA ; Csr ; HilD ; two-component system ; Salmonella ; SPI ; virulence gene regulation Introduction Horizontal transfer of DNA fragments , encoding for a variety of virulence factors , has been of paramount importance in the evolution of pathogenic bacteria by providing the ability to survive in and exploit and survive in different host niches and thus to cause different diseases ( Ochman et al. , 2000 , Schmidt and Hensel , 2004 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21518393" target="_blank">21518393</a>]<br></li>
<li>To determine how HilD , OmpR , and SlyA coordinate to regulate the expression of ssrAB in response to different growth conditions is a matter of our current and future studies .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25135218" target="_blank">25135218</a>]<br></li>
<li>The HilE CDS and sRNA regulons were distinct from the HilA , HilC and HilD regulons at IEP , and show a novel positive regulatory role for HilE in the control of metabolic gene-expression ( S3 Table ) , in addition to the reported role as a negative regulator of SPI1 genes [ 52 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27564394" target="_blank">27564394</a>]<br></li>
<li>They act by increasing the level of HilD protein to overcome HilE , and controlling HilD activity , or in the case of Fur , lowering the threshold required at the promoter , such that HilD activates its own promoter as well as induces expression of HilC and RtsA , which then act to amplify and accelerate SPI1 expression ( Saini et al. 2010a ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22021388" target="_blank">22021388</a>]<br></li>
<li>While the physiological role of the HilD-mediated cross-talk between Spi-1 and Spi-2 remains to be further analyzed in vivo inside host cells , it has been proposed that coordinated regulation of Spi-1 and Spi-2 gene-expression involves also IHF ( as described above ) which inﬂuences HilD expression ( Fass and Groisman , 2009 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26441883" target="_blank">26441883</a>]<br></li>
<li>HilC and HilD also activate the promoter PinvF-2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15661008" target="_blank">15661008</a>]<br></li>
<li>Discussion Expression of the SPI1 T3SS is controlled by HilD , HilC , and RtsA , acting in a complex feed-forward loop to activate the hilD , hilC , and rtsA genes , as well as hilA , which encodes the transcriptional activator of the T3SS structural genes ( Figure 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22021388" target="_blank">22021388</a>]<br></li>
<li>We have previously shown that three AraC-like regulators , HilD , HilC , and RtsA , act in a complex feed-forward regulatory loop to control the expression of the hilA gene , which encodes the direct regulator of the SPI1 structural genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20889744" target="_blank">20889744</a>]<br></li>
<li>We have recently shown that HilD , HilC , and RtsA act in a complex regulatory loop to control the transcription of the hilA gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17951383" target="_blank">17951383</a>]<br></li>
<li>These regulators , HilD , HilC , HilA , and InvF , act in an ordered fashion to activate coordinated expression of the SPI1 genes ( 2 , 10 , 12 , 13 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18723621" target="_blank">18723621</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_214">
Previous studies showed that four SPI1-encoded transcriptional regulators , HilD , HilC , HilA and InvF , act in an ordered fashion to coordinately activate expression of the SPI1 TTSS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></label>
<input id="rp_similarity_214", type="checkbox">
<div class="RP_similar">
<li>Many studies have shown that the SPI1-encoded transcriptional regulators HilD , HilC , HilA , and InvF act in an ordered fashion to coordinately activate the expression of SPI1 ( for a review , see reference 36 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14977940" target="_blank">14977940</a>]<br></li></div>
<li>Although many factors and a complex mechanism related to environmental stimuli are involved in the modulation of SPI1 genes , they are thought to converge on the activation of several transcriptional regulators encoded within SPI1 , such as HilA , HilC , HilD and InvF ( 7 -- 9 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17329372" target="_blank">17329372</a>]<br></li>
<li>RtsA is a transcriptional activator of hilA and HilA is a major activator of SPI1 genes ; HilD is the coordination point for several regulators that operate at the translational and transcriptional levels to control SPI1 and SPI2 expression [ 75 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24664308" target="_blank">24664308</a>]<br></li>
<li>We speculate that the HilD / SsrB-controlled FUN genes could play a role in virulence and to further support this hypothesis , we interrogated published data from transposon-directed insertion site sequencing ( TraDIS ) involving oral infections of chicken , pigs and calves [ 41 ] and data from intraperitoneal infection of BALB/c mice using single gene deletion mutant libraries [ 42 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27564394" target="_blank">27564394</a>]<br></li>
<li>As HilC and HilD activate the transcription from both HilA-dependent and - independent promoters , it is possible that Lon is involved in regulation of SPI1 gene-expression by a pathway that depends on HilC and/or HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15661008" target="_blank">15661008</a>]<br></li>
<li>Under inducing conditions , activation of BarA/SirA initiates a regulatory cascade involving HilD , HilC , and HilA , which leads both directly and indirectly to the expression of SPI1 and SPI4 genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18160484" target="_blank">18160484</a>]<br></li>
<li>Based on the importance of HilA and HilD for Spi-1 gene-expression , it is not surprising that various regulatory factors control their synthesis and activity in response to environmental conditions and ion/nutrient availability ( Figure 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26441883" target="_blank">26441883</a>]<br></li>
<li>Role of Nucleoid-Associated Proteins Hha and H-NS in Expression of Salmonella enterica Activators HilD , HilC , and RtsA Required for Cell Invasion Igor N. Olekhnovich * and Robert J. Kadner † icrobiology , University of Virginia School of Medicine , Charlottesville , Virginia 22908-0734 Received 8 June 2007/Accepted 23 July 2007 ent of M Departm The coordinate expression of Salmonella enterica invasion genes on Salmonella pathogenicity island 1 is under the control of the complex circuits of regulation that involve the AraC/XylS family transcriptional activators HilD , HilC , and RtsA and nucleoid-associated proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17675384" target="_blank">17675384</a>]<br></li>
<li>HilD , HilC and RtsA then participate in a feed-forward loop that converges on activation of hilA , which in turn activates transcription of structural components of the needle complex ( prg/org ) as well as invF , which is a transcriptional activator of secreted effector proteins ( sic/sip and others ) ( Figure 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28335027" target="_blank">28335027</a>]<br></li>
<li>The regulation of hilA promoter activity is complex , involving the coactivators HilC , HilD and RtsA , as well as other factors which act upstream of these proteins ( Lostroh and Lee , 2001 ; Ellermeier et al. , 2005 ; and references therein ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17163975" target="_blank">17163975</a>]<br></li>
<li>Expression of hilA is under the control of many environmental conditions and three transcriptional activators : HilC , HilD , and RtsA ( Ellermeier et al. , 2005 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27704705" target="_blank">27704705</a>]<br></li>
<li>Within RESULTS SPI1 , HilD , a transcriptional regulator of the AraC family , comprises a portion of an auto-regulatory circuit that includes RtsA and HilC ( Ellermeier et al. , 2005 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23289537" target="_blank">23289537</a>]<br></li>
<li>It has been proposed earlier that the regulation of HilA expression is via a complex feedforward loop involving the transactivators HilD , HilC and RtsA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19003447" target="_blank">19003447</a>]<br></li>
<li>The regulation of the expression of HilA is via a complex feed-forward loop involving three transcriptional activators : HilC , HilD and RtsA , and the negative regulator HilE .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21107576" target="_blank">21107576</a>]<br></li>
<li>Three AraC-like regulators , HilD , HilC , and RtsA , control expression of hilA , and thereby induction of the SPI1 system .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22021388" target="_blank">22021388</a>]<br></li>
<li>This work demonstrates the regulation of rtsA , hilC , hilD , and hilA by HilC , HilD and RtsA , and that each regulator has a role in the host during infection .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17208038" target="_blank">17208038</a>]<br></li>
<li>Here we report that RtsA and the related hilA regulators , HilD and HilC , coordinately regulate expression of dsbA and the SPI1 TTSS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14679226" target="_blank">14679226</a>]<br></li>
<li>HilD also regulates expression of the additional regulators HilC and RtsA , which also control HilA expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29229736" target="_blank">29229736</a>]<br></li>
<li>More recently , H-NS has been shown to repress the expression of HilA , HilD , HilC , and RtsA under noninducing conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23554419" target="_blank">23554419</a>]<br></li>
<li>A - SPI1 B - SPI2 RtsA PhoP SlyA OmpR HilD HilC SirA HilA SsrAB InvF SicA Effectors Effectors RFe l igguraeto1ry cascades activating the expression of SPI1 ( A ) and SPI2 ( B ) related type III effectors Regulatory cascades activating the expression of SPI1 ( A ) and SPI2 ( B ) related type III effectors .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17233907" target="_blank">17233907</a>]<br></li>
<li>For example , four regulators ( HilA , InvF , HilC and HilD ) directly or indirectly modulate expression of genes encoding the SPI1 secretion apparatus and its secreted effectors ( extensively discussed below ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10844688" target="_blank">10844688</a>]<br></li>
<li>The master regulator of virulence HilD regulates dapZ DapZ Although their genes partly overlap , the dapB mRNA and DapZ sRNA are clearly differentially expressed ( Figure 2 ; Supplementary Figure S2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22922465" target="_blank">22922465</a>]<br></li>
<li>Most importantly , however , HilD protein also activated the Salmonella dapZ promoter after transfer to E. coli , an organism that lacks SPI-1 and all of its associated transcription factors ( Figure 4C ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22922465" target="_blank">22922465</a>]<br></li>
<li>HilD also activates transcription of genes whose function has not been directly connected to invasion , such as lpxR , which encodes an enzyme that modiﬁes lipid A ( 11 ) , and ﬂhDC , which encodes the master regulator of ﬂagellar motility ( 11 , 18 , 19 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27601571" target="_blank">27601571</a>]<br></li>
<li>Shown is a Northern blot of the wild-type or the hilD strain , each harbouring a control ( pBAD ) or an inducible HilD expression plasmid ( pBAD-HilD ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17971080" target="_blank">17971080</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17971080" target="_blank">17971080</a>]<br></li>
<li>To distinguish whether FliZ regulates hilA via HilD or independently of HilD , we put rtsA under the control of a tetracycline-induc-ible promoter ( tetRA-rtsA ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20889744" target="_blank">20889744</a>]<br></li>
<li>Thus , to distinguish whether factors regulate hilA via HilD , or independently of HilD , we placed rtsA expression under the control of a tetracycline-inducible promoter ( tetRA -- rtsA ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22021388" target="_blank">22021388</a>]<br></li>
<li>In this paper , however , we identiﬁed the Spi1 regulator HilD as a positive regulator of ﬂagellar class 1 gene-expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24488311" target="_blank">24488311</a>]<br></li>
<li>HilD is a known activator of hilA transcription ; HilA then activates the transcription of genes encoding the Spi1 type-III-secretion apparatus and Spi1 effector proteins through InvF ( 24 , 59 , 60 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24488311" target="_blank">24488311</a>]<br></li>
<li>As a control , the expression of a cat transcriptional fusion of invF , which is positively regulated by HilD through HilA , was also assessed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27886269" target="_blank">27886269</a>]<br></li>
<li>In the presence of an unknown environmental condition , HilC or HilD may activate invF expression , causing induction of invF-dependent genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10844688" target="_blank">10844688</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_246">
HilE is a negative global regulator of the expression of many SPI-1 genes by sequestering HilD , a major SPI-1 transcriptional activator [ 8,35 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21949647" target="_blank">21949647</a>]<br></label>
<input id="rp_similarity_246", type="checkbox">
<div class="RP_similar">
<li>HilE encodes a global regulator that negatively modulates the expression of SPI-1 genes by sequestering HilD , which is a major transcriptional activator required for expression of most of the SPI-1 genes [ 35,38 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21949647" target="_blank">21949647</a>]<br></li></div>
<li>HilD also activates expression of the SPI-1-borne invR gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19007416" target="_blank">19007416</a>]<br></li>
<li>HilD also activates expression of the SPI-1-encoded , non-coding InvR RNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17971080" target="_blank">17971080</a>]<br></li>
<li>Salmonella Patogenicity Island 1 ( SPI-1 ) is a chromosomal region containing 39 genes that have crucial virulence roles.The AraC-like transcriptional regulator HilD , encoded in SPI-1 , positively controls the expression of the SPI-1 genes , as well as of several other virulence genes located outside SPI-1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27886269" target="_blank">27886269</a>]<br></li>
<li>The AraC-like regulator HilD positively controls the expression of the SPI-1 genes , as well as many other Salmonella virulence genes including those * Correspondence : Víctor H. Bustamante , Departamento de Microbiología Molecular , Instituto de Biotecnología , Universidad Nacional Autónoma de México , Avenida Universidad 2001 , Cuernavaca , Mor 62210 , Mexico victor@ibt.unam.mx located in SPI-2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26300871" target="_blank">26300871</a>]<br></li>
<li>This sRNA gene is coexpressed with other SPI-1 genes under conditions that favour host-cell-invasion , and directly controlled by HilD , the protein that acts at the top of the SPI-1 transcription factor cascade ( Fig. 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19007416" target="_blank">19007416</a>]<br></li>
<li>In this study , we investigated the possibility that , in addition to HilD , other reported SPI-1 regulators could be involved in controlling the expression of SPI-2 genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21518393" target="_blank">21518393</a>]<br></li>
<li>Our data indicate that CpxR-P decreases the stability of HilD , the regulator that is at the apex of a regulatory cascade controlling the expression of the SPI-1 genes , as well as other virulence genes , such as those located in SPI-2 ( Bustamante et al. , 2008 ; Golubeva et al. , 2012 ; Fàbrega and Vila , 2013 ; Martínez et al. , 2014 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26300871" target="_blank">26300871</a>]<br></li>
<li>For instance , the transcriptional regulator HilD , encoded in SPI-1 , positively controls the expression of the genes within this island , as well as several other genes located outside SPI-1 , which mediate Salmonella invasion-of-host cells4 ,30 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27886269" target="_blank">27886269</a>]<br></li>
<li>The AraC-like transcriptional regulator HilD , encoded within SPI-1 , positively regulates the expression of the SPI-1 genes in a cascade fashion , mainly by directly inducing the expression of hilA , which encodes the master regulator of SPI-1 ( 2 , 5 , 23 -- 26 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25135218" target="_blank">25135218</a>]<br></li>
<li>Of note , HilD-mediated regulation is not restricted to the SPI-1 genes , as HilD also modulates the expression of genes located outside this genetic locus such as sopE , which encodes an effector protein secreted via TTSS-1 , and ssrAB , which encodes the two-component system that acts as the central positive regulator of the SPI-2 genes [ 10,17,18 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28426789" target="_blank">28426789</a>]<br></li>
<li>Our results indicate that CpxR negatively controls the expression of the SPI-1 genes , as well as genes located in SPI-2 , by repressing the autoregulation of HilD , a central positive regulator for the expression of the genes within SPI-1 and SPI-2 and other virulence genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26300871" target="_blank">26300871</a>]<br></li>
<li>In order to clarify the Salmonella response-to-osmotic-stress , we investigated here the osmoregulation of the virulence activator proteins HilD , HilC , and RtsA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17675384" target="_blank">17675384</a>]<br></li>
<li>Similar to the Salmonella v-T3SS regulator HilD , LcrF expression seems also under control of several conserved TCSs : ( i ) the Rcs phosphorelay which is used to adapt their cell-physiology in response to perturbations of the cell envelope ( Li et al. , 2015 ) , ( ii ) the BarA/UvrY ( SirA ) controlling the Csr system in response to carbon sources ( LeGrand et al. , 2015 ) , and ( iii ) the CpxAR system that responds to extra-cytoplasmatic stress ( Liu et al. , 2012 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26441883" target="_blank">26441883</a>]<br></li>
<li>We further demonstrate that , among the many regulators of invasion , the repressive effect of propionate functions solely through the central SPI1 regulator HilD , but that this control is evoked through neither transcription nor translation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23289537" target="_blank">23289537</a>]<br></li>
<li>Whether the regulation by HilD implies that the gtgE , lpxR and sinR genes also have a role in the Salmonella invasion-of-host cells and thus in the intestinal infection , as for most other genes regulated by HilD , needs to be investigated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27886269" target="_blank">27886269</a>]<br></li>
<li>HilD activates HilA , which in turn upregulates the expression of genes encoding the TTSS , such as proteins encoded by the prg-org and the inv-spa operons ( 6 , 17 , 18 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28439039" target="_blank">28439039</a>]<br></li>
<li>2.4  IscR represses the expression of hilD gene and HilD targets Next , to investigate the regulatory role of IscR on hilD , the expression of the hilD fusion was measured and found to be about 1.5 fold up‐reg-ulated in an iscR mutant ( Figure 4a ) , suggesting that IscR acts as a negative regulator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27704705" target="_blank">27704705</a>]<br></li>
<li>Other features that need to be taken in account when assessing hilD transcription are the presence of an intact hilD ORF , as the HilD protein positively regulates its own expression [ 15 ] and the presence of the newly described regulatory motif hilD 3 ' - UTR , that modulates hilD mRNA stability by promoting its rapid degradation [ 29 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28426789" target="_blank">28426789</a>]<br></li>
<li>In addition to expression of the PAI-encoded genes , also loci outside the PAI can be controlled by the PAI master regulators ( e.g. , HilD controls ﬂagellar and Spi-2 gene-expression ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26441883" target="_blank">26441883</a>]<br></li>
<li>Our previous ﬁnding that it negatively regulates the expression of the ﬂagellar regulon by controlling the turnover of the master regulator , FlhD/FlhC ( 62 ) , allowed us to hypothesize that it may regulate HilC and HilD by controlling ﬂagellar gene-expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18245288" target="_blank">18245288</a>]<br></li>
<li>Most frequently the virulence operons of the PAIs are under direct control of pathogen-speciﬁc transcriptional master regulators ( mostly of the AraC family ) , which are encoded on the PAI ( e.g. , HilA , HilD in Salmonella , LcrF in Yersinia ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26441883" target="_blank">26441883</a>]<br></li>
<li>Additional signals , such as the concentrations of short-chain fatty acids ( i.e. , acetate , propionate , and butyrate ) , as well as the presence of bile salts , impact gene-expression ( 9 , 12 , 53 ) , with recent evidence showing that propionyl coenzyme A ( propi-onyl-CoA ) speciﬁcally regulates HilD posttranslationally , possibly by propionylation of the HilD protein ( 54 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25547794" target="_blank">25547794</a>]<br></li>
<li>Additionally , HilD serves as the integration point of various activating environmental signals , including low oxygen , iron limitation , and osmolarity , which together indicate the intestinal microenvironment of the host appropriate for invasion gene-expression ( 12 , 13 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24375101" target="_blank">24375101</a>]<br></li>
<li>The work presented here demonstrates that this repression occurs as a result of the post-translational control of the central SPI1 regulator HilD and requires the high-energy product of propionate-metabolism , propionyl-CoA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23289537" target="_blank">23289537</a>]<br></li>
<li>Furthermore , HilD directly controls the expression of the flhDC operon encoding FlhDC , the master transcriptional complex required for the expression of flagellar and chemotaxis genes30 ,35 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27886269" target="_blank">27886269</a>]<br></li>
<li>As expected , disruption of the ﬂhDC operon , encoding the master regulator for expression of the ﬂagellar regulon , blocked the increase of HilC and HilD levels caused by clpPX disruption .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18245288" target="_blank">18245288</a>]<br></li>
<li>The dissection of the regulatory pathway let us conclude that Gre factors are essential for the expression of HilD , a major regulator of SPI-1 genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28426789" target="_blank">28426789</a>]<br></li>
<li>HilD is a pivotal regulator of S. Typhimurium pathogenicity , as it activates the expression of the genes required for invasion of intestinal epithelium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28426789" target="_blank">28426789</a>]<br></li>
<li>describe that the Gre factors , proteins that prevent backtracking of paused RNA polymer-ase complexes during transcription elongation , are required for the expression of HilD and the subsequent expression of genes involved in the invasion of epithelial cells .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28426789" target="_blank">28426789</a>]<br></li>
<li>The present study has shown that the loss of the DnaKJ chaperone machinery results in the continuous expression of Lon because the negative feedback control is impaired ( Fig. 6 ) , therefore leading to severely diminished expression of SPI1 genes through excess degradation of HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18723621" target="_blank">18723621</a>]<br></li>
<li>Speciﬁcally , HilD is believed to activate transcription of the genes encoding the SpiR ( SsrA ) / SsrB two-component regulatory system that is the master regulator of SPI-2 ( 75 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27601571" target="_blank">27601571</a>]<br></li>
<li>Additionally , HilD not only regulates bacterial entry-into-host-cells , but also coordinates the bacterial response to the intracellular milieu by modulating the expression of ssrAB , which encodes the central positive regulator of the SPI-2 genes [ 17 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28426789" target="_blank">28426789</a>]<br></li>
<li>For example , the ﬂiZ mutation decreases the transcriptional expression of Salmonella pathogenicity island 1 ( SPI1 ) genes ( 14 ) through affecting the activity of their positive regulator , HilD ( 5 , 17 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21804009" target="_blank">21804009</a>]<br></li>
<li>Additionally , we show that in the absence of the Lon protease , which degrades HilD , the CpxR-P-mediated repression of the SPI-1 genes is mostly lost ; Citation : De la Cruz MA , Pérez-Morales D , Palacios IJ , Fernández-Mora M , Calva E and Bustamante VH ( 2015 ) The two-component system CpxR/A represses the expression of Salmonella virulence genes by affecting the stability of the transcriptional regulator HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26300871" target="_blank">26300871</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_282">
The two-com-ponent system CpxR/A represses the expression of Salmonella virulence genes by affecting the stability of the transcriptional regulator HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28575106" target="_blank">28575106</a>]<br></label>
<input id="rp_similarity_282", type="checkbox">
<div class="RP_similar">
<li>The two-component system CpxR/A represses the expression of Salmonella virulence genes by affecting the stability of the transcriptional regulator HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27886269" target="_blank">27886269</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26994203" target="_blank">26994203</a>]<br></li></div>
<li>Based on these data , we propose that a protein-acetylation system is involved in the pathogenesis of S. Typhimurium by regulating the stability of the SPI-1 regulator HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26810370" target="_blank">26810370</a>]<br></li>
<li>In particular , we showed that IscR represses the synthesis of the Spi1‐carried regulator HilD , decreasing the expression of genes encoding the TTSS apparatus .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27704705" target="_blank">27704705</a>]<br></li>
<li>Discussion Here we demonstrate that the acetyltransferase Pat represses the expression of invasion genes in Salmo-nella through the control of the SPI1 central regulator HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27341691" target="_blank">27341691</a>]<br></li>
<li>In addition to being a positive regulator of FlhD4C2 activity ( 50 ) , FliZ posttranslationally activates the regulatory protein HilD , which , in turn , positively regulates the expression of hilA , the gene for a key regulator of the SPI-1 system ( 45 , 51 , 52 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29150501" target="_blank">29150501</a>]<br></li>
<li>It has been previously demonstrated that the deletion of ﬂiZ , encoding a strong positive regulator of HilD , causes a defect in the ability of Salmonella to invade the host intestinal epithelium ( 20 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29378886" target="_blank">29378886</a>]<br></li>
<li>Furthermore , the plasmid pK6-HilD , expressing HilD from an arabinose-inducible promoter , was able to increase the expression of these fusions in the ΔhilD mutant to WT levels or even higher ( Fig. 2A -- F ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27886269" target="_blank">27886269</a>]<br></li>
<li>HilD has been shown to be the major integration point of most signals known to activate the expression of the SPI1 T3SS , acting as a switch to control induction of the system .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29378886" target="_blank">29378886</a>]<br></li>
<li>The overall effect is to control the ability of HilD to activate its own promoter , the critical step in initiating expression of the SPI1 T3SS ( 17 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29378886" target="_blank">29378886</a>]<br></li>
<li>It is important to note that the E. coli MC4100 strain carries a frameshift mutation in the flhDC operon and thus it does not express the flagellar transcriptional regulator FlhDC45 ,46 ; HilD positively regulates the expression of FlhDC in S. Typhimurium30 ,35 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27886269" target="_blank">27886269</a>]<br></li>
<li>These results indicate that the DnaK chaperone machinery is involved in the control of HilD turnover .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18723621" target="_blank">18723621</a>]<br></li>
<li>* p ≤ 0.05 ( Mann -- Whitney U‐test ) of HilD targets To investigate whether IscR acts on Spi1 TTSS gene-expression under its apo‐form or holo‐form , we built a plasmid encoding an IscR variant ( IscR‐3CA ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27704705" target="_blank">27704705</a>]<br></li>
<li>Class II : control of hilD mRNA stability/degradation or translation-initiation Class II regulators function through HilD to control hilA expression , but do so by controlling hilD mRNA stability or translation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22021388" target="_blank">22021388</a>]<br></li>
<li>Figure S4 HilE negatively regulates HilD expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20686667" target="_blank">20686667</a>]<br></li>
<li>Transposon mutagenesis identiﬁed a class of mutants that prevented ﬂhDC autorepression by overexpressing HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24488311" target="_blank">24488311</a>]<br></li>
<li>A variation of this model is that Fur indirectly regulates a factor that positively affects HilD function ; e.g. , Fur represses a repressor of a factor that assists HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
<li>We observed that arabinose-induced expression of HilD activates invF expression > 80-fold ( Fig. 3 , black bars ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>Recall that HilD , but not HilC , activates expression of invF sevenfold from the pVV448 reporter plasmid , which carries this region ( Fig. 4 , black bars ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>( D ) Effect of overproduced HilD on ﬂagellar gene-expression under artiﬁcially induced ﬂhDC expression conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24488311" target="_blank">24488311</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_302">
( A ) Effect of positive feedback on HilD expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20686667" target="_blank">20686667</a>]<br></label>
<input id="rp_similarity_302", type="checkbox">
<div class="RP_similar">
<li>( A ) Effect of HilD positive feedback on HilA expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20686667" target="_blank">20686667</a>]<br></li></div>
<li>Figure 5 Class III , PhoPQ ( PhoQ24 ) represses hilA expression independently of HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22021388" target="_blank">22021388</a>]<br></li>
<li>In particular , HilD directly regulates expression of all 7 TFs , including itself .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27601571" target="_blank">27601571</a>]<br></li>
<li>In this work , we demonstrated that Fur regulates expression of hilA via control of HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_307">
Fur regulates expression of hilA via HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></label>
<input id="rp_similarity_307", type="checkbox">
<div class="RP_similar">
<li>Fur regulates expression of rtsA via HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li></div>
<li>Fur regulates expression of hilA by working through HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
<li>Fur regulates expression of rtsA by controlling HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
<li>Results Rationale and approach We have previously shown that HilE , FliZ , and Fur control hilA expression via HilD ( Ellermeier and Slauch 2008 ; Lin et al. 2008 ; Chubiz et al. 2010 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22021388" target="_blank">22021388</a>]<br></li>
<li>RESULTS HilD links Spi1 and ﬂagellar gene regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24488311" target="_blank">24488311</a>]<br></li>
<li>HilD positively regulates gtgE , phoH , sinR , lpxR , SL1896 and SL4247 in S. Typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27886269" target="_blank">27886269</a>]<br></li>
<li>( B ) qRT-PCR analysis of changes in hilA expression in wildtype , ΔloiA , ΔhilD , ΔloiAΔhilD , ΔloiAΔhilD + pHilD and ΔloiAΔhilD + pLoiA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28575106" target="_blank">28575106</a>]<br></li>
<li>( C ) Western blot analysis of HilA in wild-type , ΔloiA , ΔhilD , ΔloiAΔhilD , ΔloiAΔhilD + pHilD and ΔloiAΔhilD + pLoiA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28575106" target="_blank">28575106</a>]<br></li>
<li>In contrast to invF-lacZ expression from pVV448 , HilD and HilC activate invF-lacZ expression over 45-fold from pSA7 ( Fig. 6 , black and striped bars , pSA7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>SprB directly activates SPI4 gene-expression and weakly represses SPI1 gene-expression through HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20190046" target="_blank">20190046</a>]<br></li>
<li>One such regulator , Lon , has been shown to degrade both HilC and HilD , thus down-regulating SPI1 gene-expression ( 58 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18245288" target="_blank">18245288</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_319">
Olekhnovich , I. N. , and Kadner , R. J. ( 2007 ) Role of nucleoid-associated proteins Hha and H-NS in expression of Salmonella enterica activators HilD , HilC , and RtsA required for cell invasion .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23515315" target="_blank">23515315</a>]<br></label>
<input id="rp_similarity_319", type="checkbox">
<div class="RP_similar">
<li>Olekhnovich , I. N. , and R. J. Kadner , 2007 Role of nucleoid-associated proteins Hha and H-NS in expression of Salmonella enterica activators HilD , HilC , and RtsA required for cell invasion .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22021388" target="_blank">22021388</a>]<br></li>
<li>Role of nucleoid-associated proteins Hha and H-NS in expression of Salmonella enterica activators HilD , HilC , and RtsA required for cell invasion .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23554419" target="_blank">23554419</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23459474" target="_blank">23459474</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27185788" target="_blank">27185788</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26441883" target="_blank">26441883</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21075923" target="_blank">21075923</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28660201" target="_blank">28660201</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20889744" target="_blank">20889744</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27601571" target="_blank">27601571</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25135218" target="_blank">25135218</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18441068" target="_blank">18441068</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25547794" target="_blank">25547794</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24488311" target="_blank">24488311</a>]<br></li>
<li>[ 33 ] I.N. Olekhnovich , R.J. Kadner , Role of nucleoid-associated proteins Hha and H-NS in expression of Salmonella enterica activators HilD , HilC , and RtsA required for cell invasion , J. Bacteriol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23040276" target="_blank">23040276</a>]<br></li>
<li>Olekhnovich , I. N. & Kadner , R. J. Role of nucleoid-associated proteins Hha and H-NS in expression of Salmonella enterica activators HilD , HilC , and RtsA required for cell invasion .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27886269" target="_blank">27886269</a>]<br></li>
<li>Olekhnovich IN , Kadner RJ : Role of nucleoid-associated proteins Hha and H-NS in expression of Salmonella enterica activators HilD , HilC , and RtsA required for cell invasion .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22017966" target="_blank">22017966</a>]<br></li></div>
				<label class="RP_collapse_similarity" for="rp_similarity_325">
Baxter , M. A. , Fahlen , T. F. , Wilson , R. L. & Jones , B. D. HilE interacts with HilD and negatively regulates hilA transcription and expression of the Salmonella enterica serovar Typhimurium invasive phenotype .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27886215" target="_blank">27886215</a>]<br></label>
<input id="rp_similarity_325", type="checkbox">
<div class="RP_similar">
<li>HilE interacts with HilD and negatively regulates hilA transcription and expression of Salmonella enterica serovar Typhimurium invasive phenotype .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15790293" target="_blank">15790293</a>]<br></li></div>
<li>This suggests that SirA regulates expression of rtsAB and hilC via HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16045614" target="_blank">16045614</a>]<br></li>
<li>Interestingly , overexpression of HilD still activates expression of invF from pSA14 and pCL463 by approximately 10-fold .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>Regulation of slrP expression by PhoP , Lon , LeuO , and HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25182488" target="_blank">25182488</a>]<br></li>
<li>( D ) Association of ChIP-seq peaks with regulation for HilD , HilC , and RtsA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27601571" target="_blank">27601571</a>]<br></li>
<li>We also show regulation of many HilD target genes by HilC and RtsA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24375101" target="_blank">24375101</a>]<br></li>
<li>( A ) Timecourse simulation of HilD , HilC , RtsA , and HilA expression dynamics in wild-type cells .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20686667" target="_blank">20686667</a>]<br></li>
<li>Still , no one has reported a condition in which wild-type S. typhimurium represses hilA expression while simultaneously activating expression of invF and invF-dependent genes through HilC or HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10844688" target="_blank">10844688</a>]<br></li>
<li>Effect of HilD , HilC and HilA on invF expression from pSA7 and mutant derivatives of pSA7 in a Dspi1 mutant .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>RtsA , HilC and HilD also induced expression of hilC , hilD and rtsA ( Fig. 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16045614" target="_blank">16045614</a>]<br></li>
<li>Effect of invF , hilA , and hilC-D mutations on RtsA induction of slrP ( A ) and ability of HilC , HilD , and RtsA to induce expression of hilA ( B ) and slrP ( C ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923082" target="_blank">12923082</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_337">
HilC , HilD , and RtsA are each capable of activating expression of hilC , hilD , and rtsA , creating a complex regulatory loop that controls hilA expression ( Fig. 1 ) ( 15 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17951383" target="_blank">17951383</a>]<br></label>
<input id="rp_similarity_337", type="checkbox">
<div class="RP_similar">
<li>HilC , HilD , and RtsA are all also capable of activating expression of hilC , hilD , and rtsA independent of each other and comprise a complex feed forward regulatory loop that controls hilA expression ( Fig. 1 ) ( 16 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li></div>
<li>RtsA increases invasion gene-expression by increasing transcription of hilA , which is independent of HilC and HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15765064" target="_blank">15765064</a>]<br></li>
<li>( ii ) RtsA , HilC , and HilD differentially regulate expression of other genes independent of their effects on hilA expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923082" target="_blank">12923082</a>]<br></li>
<li>HilC , HilD , and RtsA differentially induce expression of dsbA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14679226" target="_blank">14679226</a>]<br></li>
<li>Regulation of gene-expression by RtsA , HilC and HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16045614" target="_blank">16045614</a>]<br></li>
<li>The SPI2-encoded regulators ( SsrA/B ) and regulatory systems that play an important role in intracellular survival and replication of S. Typhimurium ( SlyA , HilD , OmpR/EnvZ , PhoP/Q , PhoB/R ) [ 25 -- 30 ] were investigated during growth in SPI2-inducing conditions [ 31 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27564394" target="_blank">27564394</a>]<br></li>
<li>RtsA is a close homologue of HilD and HilC but is encoded outside SPI-1 , in an operon that includes another TF-encoding gene , rtsB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27601571" target="_blank">27601571</a>]<br></li>
<li>( B ) Transcription levels of two SPI-1 genes under HilD control ( invF and sipB ) in Dam1 ( solid histograms ) and Dam ( open histo-grams ) strains that express hilD from a heterologous , tetracy-cline-dependent promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20008574" target="_blank">20008574</a>]<br></li>
<li>HilD , HilC , and RtsA all activate transcription of hilA , hilC , hilD , dsbA , invF , slrP , rtsA , and genes in SPI-4 ( 6 , 15 , 32 , 33 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24375101" target="_blank">24375101</a>]<br></li>
<li>Hence , LeuO seems to repress SPI-1 by a major ` pathway ' involving HilD and HilE , and also by minor , HilD-and HilE-independent mechanisms .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24354910" target="_blank">24354910</a>]<br></li>
<li>We propose that NaCl impacts Sip secretion in two distinct ways : ( i ) like sucrose , it induces SPI1 transcription via a pathway that includes BarA , SirA , HilD , and HilA ; and ( ii ) unlike sucrose , it enhances secretion of SipB and SipC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18441068" target="_blank">18441068</a>]<br></li>
<li>The SPI-1 type-III-secretion-system ( TTSS ) structural genes spaM-NOPQRS and invABCEFGH and their regulatory proteins HilA , HilC , HilD [ 31 ] were also identified and annotated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23234298" target="_blank">23234298</a>]<br></li>
<li>Consistent with its role as a negative regulator , HilE decreased both HilD and HilA expression in a dose-dependent manner ( Figures 6C and S6C ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20686667" target="_blank">20686667</a>]<br></li>
<li>A genetic screen identiﬁed LeuO and Lon as negative regulators and HilD as a positive regulator under SPI1-inducing conditions ( Fig. 4 and 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25182488" target="_blank">25182488</a>]<br></li>
<li>Flagellar regulation is highly complex and is orchestrated at the levels of transcription , translation , ﬂagellum assembly , and stability , involving a large number of positive and negative regulators , including SirA/BarA ( 38 ) , RtsB ( 39 ) , SlyA and LrhA ( 40 ) , RcsB ( 41 ) , RﬂM ( 42 ) , CsrA ( 43 ) , FimZ ( 44 ) , HilD ( 45 ) , cyclic AMP ( cAMP ) - cAMP receptor protein ( CRP ) complex ( cAMP-CRC ) and H-NS ( 46 ) , and YdiV ( 47 ) , as well as other mechanisms .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26056383" target="_blank">26056383</a>]<br></li>
<li>Through a complex feedback and feedforward mechanism , HilC and HilD control their own and each other 's transcription , and together activate transcription of hilA [ 14,15,16 ] ( Figure 1A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22457642" target="_blank">22457642</a>]<br></li>
<li>The panel of regulatory proteins ( Table 2 ) included SPI1-encoded transcription factors ( HilA , HilC and HilD ) and transcription factors previously reported to control SPI1 gene-expression ( HilE , BarA/SirA , Fur and FliZ ) [ 17 -- 24 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27564394" target="_blank">27564394</a>]<br></li>
<li>Three AraC-like transcription factors -- HilC , HilD , and RtsA -- positively regulate HilA expression in response to various intracellular and environmental signals ( 5 , 7 , 17 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20190046" target="_blank">20190046</a>]<br></li>
<li>METABOLIC ADAPTATION OF SALMONELLA DURING STRESS CONDITIONS Stress factors linking virulence and metabolism When Salmonella enters into an intestinal epithelial cell , environmental factors such as high osmolarity and neutral pH lead to an activation of HilD , which in turn induces HilA and invF gene-expression ( Altier , 2005 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25688337" target="_blank">25688337</a>]<br></li>
<li>The expression of hilA is regulated in response to variety of environmental signals , including pH ( 6 ) , growth phase ( 56 ) , osmolarity ( 36 ) , and oxygen ( 52 ) and by a number of proteins , including BarA/SirA ( 51 ) , CpxA ( 71 ) , CsrB ( RNA ) ( 2 ) , EnvZ/OmpR ( 63 ) , FadD ( 63 ) , Fis ( 94 ) , FliZ ( 63 ) , HilC ( 83 ) , HilD ( 83 ) , Hu ( 84 ) , Mlc ( 58 ) , and RtsA ( 28 ) , which positively regulate expression , and CsrA ( 2 ) , Hha ( 29 ) , HilE ( 7 ) , H-NS ( 84 ) , Lon ( 87 ) , PhoB ( 63 ) , and PhoP/PhoQ ( 9 ) , which negatively regulate expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19074398" target="_blank">19074398</a>]<br></li>
<li>By controlling expression/activity of HilD , HilC , RtsA , and HilA , many regulators contribute to SPI-1 gene regulation in response to environmental stimuli that include temperature ( H-NS and Hha ) , osmolarity ( H-NS and Hha ) , pH ( OmpR ) , iron availability ( Fur ) , and oxygen levels ( FNR ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27601571" target="_blank">27601571</a>]<br></li>
<li>( D and E ) Time-course simulation of HilD ( D ) and HilA ( E ) expression dynamics in wild type and DhilD , DhilC DrtsA , and DhilE mutants at population resolution .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20686667" target="_blank">20686667</a>]<br></li>
<li>) such that HilD ( and also HilC and RtsA ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
<li>Many regulatory proteins inﬂuence invasion gene-expression directly and indirectly , including activators HilA , InvF , HilD , HilC , and RtsA and nucleoid-associated proteins H-NS and Hha ( 2 , 14 , 16 , 23 , 36 , 37 , 44 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17675384" target="_blank">17675384</a>]<br></li>
<li>SPI-1 also encodes transcriptional activators responsible for its own expression : HilA , HilC , HilD , and InvF ( Lostroh and Lee 2001 ; Ellermeier and Slauch 2007 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25991823" target="_blank">25991823</a>]<br></li>
<li>Effect of HilD , HilC or HilA on chromosomal invF-lacZ fusion expression in a hilA hilD mutant .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>HilD , a transcriptional regulator of SPI1 expression , is de-stabilized in cells in which dnaKJ is disrupted .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18723621" target="_blank">18723621</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_365">
HilD , an AraC/XylS type transcriptional regulator , is a particularly important regulator of hilA expression , since a mutation in hilD results in a 14-fold decrease in chromosomal hilA : : Tn5lacZY-080 expression and a 53-fold decrease in invasion of HEp-2 cells ( 42 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12511499" target="_blank">12511499</a>]<br></label>
<input id="rp_similarity_365", type="checkbox">
<div class="RP_similar">
<li>HilD , an AraC/XylS type transcriptional regulator , is of particular importance as a mutation in hilD results in a 14-fold decrease in chromosomal hilA : : Tn5lacZY-080 expression and a 53-fold decrease in invasion of HEp-2 cells .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15765064" target="_blank">15765064</a>]<br></li></div>
<li>The transcriptional activators HilD and HilC play an important role in controlling hilA expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25547794" target="_blank">25547794</a>]<br></li>
<li>These results suggest that HilA , HilC , and HilD are each required for expression of siiE , but these regulators are likely to act in concert or in combination with additional SPI1-encoded regulatory loci to activate SPI4 gene-expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18160484" target="_blank">18160484</a>]<br></li>
<li>Models for invasion gene regulation in vitro In vitro characterization of HilA , HilC , HilD and InvF suggests that these regulators operate in a hierarchy to modulate invasion gene-expression with HilA acting as the central regulator in this cascade ( Fig. 1A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10844688" target="_blank">10844688</a>]<br></li>
<li>The key regulator of this secretion system is HilA , which is in turn regulated by HilD , HilC and RtsA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19537165" target="_blank">19537165</a>]<br></li>
<li>These data suggest that most of the 17 novel invasion genes are directly or indirectly involved in invasion and that their regulation TABLE 1 List of regulated noncoding RNAs identiﬁed in this study TF Fold Strand Start ( bp ) a , b Stop ( bp ) a , b Description regulator ( s ) changec 34013 * 33811 * Antisense to HilC 2.28 d 74905 74971 DapZ RtsA 2.66 868925 868081 Antisense to slrP RtsA 1.25 1343788 1343659 STnc520 SprB 4.08 1418433 1418523 Overlapping / HilC 1.18 1603873 1604092 MgrR HilD 2.12 1603938 1603781 Overlapping HilD , RtsA 3.73 , 2.05 e 3039507 3039347 Antisense to hilA 5 = UTR HilD 4.23 3065151 3066061 InvR HilD , HilC 1.07 , 3.99 e 3483808 3483985 Within dacB RtsA 2.54 a The numbers represent predicted start/stop genome coordinates for a given noncoding RNA ( ncRNA ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27601571" target="_blank">27601571</a>]<br></li>
<li>We conclude that HilD and HilC have a second role in SPI1 gene regulation that is separate from their role in co-ordinating expression of the SPI1 TTSS through hilA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>These data led us to propose a new model for SPI1 regulation , in which HilC , HilD and RtsA act in a complex feed-forward loop to control expression of hilA ( Figure 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17208038" target="_blank">17208038</a>]<br></li>
<li>The expression of hilA , the master regulator for SPI1 , is controlled by HilD , HilC , and RtsA , which act in a complex feed-forward loop .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20889744" target="_blank">20889744</a>]<br></li>
<li>Thereafter , the regulatory cascade including HilD , HilC , RtsA , ppGpp and other regulatory factors , in response to several other signals , dictates hilA expression levels .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21680637" target="_blank">21680637</a>]<br></li>
<li>Through genetic and biochemical analyses , we show that FliZ posttranslationally controls HilD to positively regulate hilA expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20889744" target="_blank">20889744</a>]<br></li>
<li>HilD ; lysine acetylation ; stability ; DNA-binding ; virulence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28329249" target="_blank">28329249</a>]<br></li>
<li>The protein acyltransferase Pat post-transcriptionally controls HilD to repress Salmonella invasion .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28329249" target="_blank">28329249</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_379">
Singer , H.M. , Ku $ hne , C. , Deditius , J.A. , Hughes , K.T. , and Erhardt , M. ( 2014 ) The Salmonella Spi1 virulence regulatory protein HilD directly activates transcription of the flagellar master operon flhDC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27206164" target="_blank">27206164</a>]<br></label>
<input id="rp_similarity_379", type="checkbox">
<div class="RP_similar">
<li>Singer , H.M. , Kühne , C. , Deditius , J.A. , Hughes , K.T. , and Erhardt , M. ( 2014 ) The Salmonella Spi1 virulence regulatory protein HilD directly activates transcription of the flagellar master operon flhDC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27404739" target="_blank">27404739</a>]<br></li></div>
<li>J Appl Microbiol 101:1282 -- 1291 Singer HM , Kuhne C , Deditius JA , Hughes KT , Erhardt M ( 2014a ) The Salmonella Spi1 virulence regulatory protein HilD directly activates transcription of the ﬂagellar master operon ﬂhDC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27549210" target="_blank">27549210</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_382">
The Salmonella Spi1 virulence regulatory protein HilD directly activates transcription of the ﬂagellar master operon ﬂhDC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26441883" target="_blank">26441883</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26300871" target="_blank">26300871</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24706743" target="_blank">24706743</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26884427" target="_blank">26884427</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27601571" target="_blank">27601571</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26056383" target="_blank">26056383</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25135218" target="_blank">25135218</a>]<br></label>
<input id="rp_similarity_382", type="checkbox">
<div class="RP_similar">
<li>The Salmonella Spi1 virulence regulatory protein HilD directly activates transcription of the flagellar master operon flhDC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25315056" target="_blank">25315056</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28329249" target="_blank">28329249</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27886269" target="_blank">27886269</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27564394" target="_blank">27564394</a>]<br></li></div>
				</div></ul><br>
			<label class="collapse_SIT" for="_5"><font size="5px">HilD binding site</font></label>
				<input id="_5" type="checkbox">
				<div class="SIT"><br><ul><li>Importantly , site 1 overlaps with a previously identified HilD binding site ( Figure 3a ; Olekhnovich & Kadner , 2002 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27704705" target="_blank">27704705</a>]<br></li>
<li>We selected all HilD or RtsA ChIP-seq peaks that were located within 100 bp of a HilC ChIP-seq peak .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27601571" target="_blank">27601571</a>]<br></li>
<li>Therefore , HilD binding sequences appear to be important for chromosomal hilA expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12511499" target="_blank">12511499</a>]<br></li>
				<label class="SIT_collapse_similarity" for="sit_similarity_4">
It is believed that HilD directly regulates hilA expression since it has been shown to bind to hilA promoter sequences ( 40 , 43 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12511499" target="_blank">12511499</a>]<br></label>
<input id="sit_similarity_4", type="checkbox">
<div class="SIT_similar">
<li>It is believed that HilD directly regulates hilA expression as it has been shown to bind to hilA promoter sequences .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15765064" target="_blank">15765064</a>]<br></li></div>
<li>The binding sites for HilD ( hatched box ) and H-NS/Hha ( open box , AT tract ) in PhilA region are shown .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19074398" target="_blank">19074398</a>]<br></li>
<li>The ﬂhDC DNA fragment comprising the putative HilD binding site was PCR ampliﬁed from Salmonella Typhimurium LT2 genomic DNA using 5 = digoxigenin ( DIG ) - labeled primers ( 5 = - GATCAT GCTGACACGTACGGATTCTTATGTAAAGAATCGTGGC-3 = and 5 = - DIG-GTCAACACCAAATTCTTTTTTG-3 =) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24488311" target="_blank">24488311</a>]<br></li>
<li>We detected direct regulation of sprB by HilD from a binding site immediately upstream of sprB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27601571" target="_blank">27601571</a>]<br></li>
<li>Even at the C2 site , binding of HilD required a longer stretch of speciﬁc base sequences than did binding of HilC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12107132" target="_blank">12107132</a>]<br></li>
<li>In brief , HilD binds directly to sites upstream of hilA and invFD and acts as an activator ( 14,16,36 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17329372" target="_blank">17329372</a>]<br></li>
<li>HilE binds directly and speciﬁcally with HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29378886" target="_blank">29378886</a>]<br></li>
<li>Binding sites for HilD and H-NS are located close to each other in ssrAB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25135218" target="_blank">25135218</a>]<br></li>
<li>Our results from EMSAs suggested that HilD and H-NS bind to sites located close to each other in ssrAB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25135218" target="_blank">25135218</a>]<br></li>
<li>Here we provide genetic and biochemical data showing that HilE speciﬁcally binds to HilD but not to HilC or RtsA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29378886" target="_blank">29378886</a>]<br></li>
<li>The consensus sequences overlap but differ so that HilC can bind to both types of sites but HilD only to one .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12107132" target="_blank">12107132</a>]<br></li>
<li>DNA fragment A-a , which carries the upstream regulatory sequences from 373 to 42 , formed two major shifted complexes with HilC but only a single complex with HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12107132" target="_blank">12107132</a>]<br></li>
<li>Comparison of known HilD binding sites from the rtsA , hilC , hilD , and hilA promoters ( 64 ) with the identiﬁed P5 PﬂhDC binding site revealed several nucleotides that were conserved between these sites ( Fig. 3C , consensus ; uppercase letters indicate predominant nucleotides [ 80 % conserved ] , and lowercase letters indicate conserved nucleotides [ 60 % conserved ] ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24488311" target="_blank">24488311</a>]<br></li>
<li>Here , we report a novel transcriptional cross talk between the Spi1 and ﬂagellar regulons where HilD transcriptionally activates ﬂhDC gene expression by binding to nucleotides 68 to 24 upstream from the P5 transcriptional start site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24488311" target="_blank">24488311</a>]<br></li>
<li>A cascade of transcriptional activation has been unraveled in which HilD , in response to particular environmental signals , directly affects hilA expression by binding to the upstream repressing sequence , thereby derepressing hilA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16391141" target="_blank">16391141</a>]<br></li>
<li>Our results support a model indicating that HilD displaces H-NS from a region downstream of the promoter of ssrAB by binding to sites overlapping or close to those sites bound by H-NS , which leads to the expression of ssrAB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25135218" target="_blank">25135218</a>]<br></li>
<li>We attempted to identify enriched DNA sequence motifs at the conﬁrmed sites of HilD association using MEME ( default settings ) ( 35 ) , but were unable to detect any signiﬁcant hits , suggesting that HilD binds to a degenerate sequence motif .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24375101" target="_blank">24375101</a>]<br></li>
<li>The previously identiﬁed binding sites for HilC and HilD are indicated as regions A1 and A2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16443238" target="_blank">16443238</a>]<br></li>
<li>Analysis of overlapping HilD , HilC , and RtsA binding sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27601571" target="_blank">27601571</a>]<br></li>
<li>These results show that HilC and HilD use overlapping binding sites and depend on the same nucleotide positions for DNA recognition .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12107132" target="_blank">12107132</a>]<br></li>
<li>Studies have shown that HilC , HilD and RtsA have overlapping binding sites on the hilA promoter , but that each has a slightly different DNA recognition sequence [ 24,30 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17208038" target="_blank">17208038</a>]<br></li>
<li>The hilA and hilC promoters contain multiple binding sites of each type , whereas the hilD promoter contains a site that binds HilC but not HilD without additional binding elements .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12107132" target="_blank">12107132</a>]<br></li>
<li>The discrimination between HilC and HilD can not yet distinguish the contributions of speciﬁc base sequence recognition , A T-rich character , or DNA bending .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12107132" target="_blank">12107132</a>]<br></li>
<li>Analysis of the DNA sequence upstream of T2 revealed no close matches to consensus sites of the virulence-associated transcription factors : PhoP-PhoQ ( 27 ) , PmrA-PmrB ( 1 ) , HilA ( 14 ) , HilC , or HilD ( 18 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12775687" target="_blank">12775687</a>]<br></li>
<li>The distances in nucleotides of the insertions to the cents rst nucleotide of the HilD coding sequence are as follows : TF37 , 936 ; TF38 , 811 ; TF39 , 664 ; BJ606 , 652 , TF40 , 349 ; TF41 , 99 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10767604" target="_blank">10767604</a>]<br></li>
<li>We identiﬁed a region from 68 to 24 nucleotides upstream from the transcriptional start site of the P5 ﬂhDC promoter that was protected from DNase I digestion in the presence of HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24488311" target="_blank">24488311</a>]<br></li>
<li>In addition , the presence of HilD resulted in an enhancement of DNase cleavage at nucleotides 51 , 33 , 28 , and 27 upstream from the P5 transcriptional start , which indicates a potential DNA bend ( Fig. 3A and B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24488311" target="_blank">24488311</a>]<br></li>
<li>To generate plasmids pLoiA , pHilD , pArcA and pArcB , the ORF and the upstream promoter sequence of the corresponding genes were amplified by PCR from genomic DNA of the wild-type strain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28575106" target="_blank">28575106</a>]<br></li>
<li>The hilC start site used in the absence of HilC and HilD is associated with the sequence TGTAATTAT , which matches at six of eight positions the extended 10 promoter element that allows transcription initiation in the absence of a 35 element ( 23 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12107132" target="_blank">12107132</a>]<br></li>
<li>Predominant bases ( 5/5 or 4/5 ) in the consensus sequence are indicated by uppercase letters , bases identical in 3/5 of the HilC/HilD binding sites are indicated by lowercase letters , and degenerate bases are marked by the letter N .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17675384" target="_blank">17675384</a>]<br></li>
<li>The consensus motif logo of the experimentally determined HilD binding sites is shown at the top and was generated using WebLogo ( 71 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24488311" target="_blank">24488311</a>]<br></li>
<li>Although the HilC regulatory targets were largely a subset of the HilD targets , HilC binds many more sites across the genome ( 209 sites for HilC versus 18 sites for HilD , not including the HilD sites from our previous study [ 11 ] ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27601571" target="_blank">27601571</a>]<br></li>
<li>In the model , we assume that all four promoters have the same two binding sites , one highly specific for HilD and the other much less so ( see Assumption 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20686667" target="_blank">20686667</a>]<br></li>
<li>Although we could not derive a DNA sequence motif for HilD , it is likely that it binds with similar levels of sequence spec-iﬁcity to HilC and RtsA , given the overlap in target sites ( Fig. 3B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27601571" target="_blank">27601571</a>]<br></li>
<li>To determine whether HilD , HilC , and RtsA bind the same DNA sites upstream of shared target genes , we compared the proﬁle of ChIP-seq reads across regions with closely positioned HilD/HilC/RtsA ChIP-seq peaks .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27601571" target="_blank">27601571</a>]<br></li>
<li>We conclude that HilD , HilC , and RtsA bind very similar DNA sites , although the failure of RtsA to regulate ytfK ( Fig. 3C ) indicates that there are subtle differences in the sequence speciﬁcities of HilD , HilC , and RtsA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24375101" target="_blank">24375101</a>]<br></li>
<li>Unlike HilC and RtsA , these three regulators are not homologues of HilD ; hence , they are expected to bind DNA with different sequence speciﬁcities .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24375101" target="_blank">24375101</a>]<br></li>
<li>Although HilC and HilD bind to the same DNA regions , they have different requirements with respect to the sequence and length of their DNA target ( 18 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15126488" target="_blank">15126488</a>]<br></li>
<li>Thus , although HilC and HilD proteins can bind to the same DNA target , they have different sequence requirements for DNA recognition .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12107132" target="_blank">12107132</a>]<br></li>
<li>All three proteins formed single retarded complexes with two rtsA promoter fragments extending from 173 to 1 and from 81 to 29 , indicating the presence of a single and possibly overlapped binding site for HilC , HilD , and RtsA proteins located between 81 and 1 relative to the transcription start site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17675384" target="_blank">17675384</a>]<br></li>
<li>RtsA , HilC , and HilD each activate hilA by binding to the same sites in the promoter region ( 13 , 35 , 36 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20889744" target="_blank">20889744</a>]<br></li>
<li>HilD , HilC , and RtsA bind to a common DNA-binding site at the rtsA promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17675384" target="_blank">17675384</a>]<br></li>
<li>To identify the sequences that contribute to HilD , HilC , and RtsA binding at the rtsA promoter , the DNA fragment extended from 173 to 29 was analyzed by DNase I footprinting ( Fig. 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17675384" target="_blank">17675384</a>]<br></li>
<li>To see if these sequences are important for HilD and HilC to activate invF , we constructed derivatives of pSA7 in which the upstream TTTGAT was changed to TTTGAG and the downstream TATTCT was changed to TATGCG .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>Two SPI1 gene products , HilC and HilD , have been shown to bind to the upstream repressing sequence of hilA to induce its transcription ( 12 , 52 , 53 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18245288" target="_blank">18245288</a>]<br></li>
<li>The HilC and HilD proteins bind to their own gene promoters as well as to at least two regions in the hilA promoter , called sites A1 ( positions 242 to 182 , relative to the start of transcription ) and A2 ( positions 85 to 61 ) ( 18 , 24 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15126488" target="_blank">15126488</a>]<br></li>
<li>HilC and HilD are known to independently bind to two regions of the hilA promoter , from approximately 231 to 179 and from 101 to 49 , although they differ in their sequence requirements for recognition of the binding sites ( 64 , 72 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923082" target="_blank">12923082</a>]<br></li>
<li>Our data are consistent with an earlier study showing that HilC and HilD bind overlapping sites upstream of hilA , hilD , and hilC but that HilC binding is less sensitive to mutations within the DNA site ( 21 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27601571" target="_blank">27601571</a>]<br></li>
<li>HilD and HilC bind to the invF promoter fragment in vitro To examine whether HilD and HilC might directly activate the invF promoter by binding to sequences upstream of invF , we performed gel shift analyses .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>These gel shift assays indicate that HilD and HilC each bind sequences upstream and downstream of the HilD - and HilC-dependent invF promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>In addition to binding upstream of hilA and invF , HilD and HilC also bind to sites upstream of hilC and within the prgH-hilD intergenic region ( Olekhnovich and Kadner , 2002 ) ( S. Akbar , unpublished results ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>HilC and HilD bind to overlapping sites in the hilA promoter ( Olekhnovich and Kadner , 2002 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16045614" target="_blank">16045614</a>]<br></li>
<li>HilC and HilD , two SPI1-encoded AraC/XylS family members , have been called derepressors and are thought to counteract the repression of hilA by binding to sites within the upstream repressing sequence ( URS ) ( 24 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12933899" target="_blank">12933899</a>]<br></li>
<li>HilC and HilD have been shown to bind to the upstream repressing sequence ( URS ) of hilA and counteract the repression ( Schechter et al. , 1999 ; Schechter and Lee , 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15661008" target="_blank">15661008</a>]<br></li>
<li>HilC and HilD bind directly to the upstream sequence of hilA and derepress the transcription of PhilA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15661008" target="_blank">15661008</a>]<br></li>
<li>Furthermore , HilC and HilD are strikingly similar in primary sequence and bind to the same speciﬁc DNA sequences upstream of PhilA [ 52 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11755416" target="_blank">11755416</a>]<br></li>
<li>Furthermore , electrophoretic mobility shift assays showed that both HilD and H-NS bind to the ssrAB region containing the repressing sequences .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25135218" target="_blank">25135218</a>]<br></li>
<li>In summary , sites A1 and C2 can bind both HilC and HilD in the absence of ﬂanking sequences , whereas sites A2 and D can bind HilC but not HilD , without additional ﬂanking sequences .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12107132" target="_blank">12107132</a>]<br></li>
<li>Likewise , the D region resembles the A2 region , and both bind HilC but not HilD without ﬂanking sequences .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12107132" target="_blank">12107132</a>]<br></li>
<li>This region comprises both the 're pressing sequences ' that regulate PhilA in cis as well as the binding sites for HilD ( and HilC ) ( L. Schechter and C. Lee , unpublished observations ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11755416" target="_blank">11755416</a>]<br></li>
<li>These different binding speciﬁcities of HilC and HilD seen in the gel shift assays were reﬂected in the sequence dependence for binding to the C2 region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12107132" target="_blank">12107132</a>]<br></li>
<li>Comparison of the four known binding site sequences revealed that the C2 region and the A1 region , which bind HilC and HilD , are closely related .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12107132" target="_blank">12107132</a>]<br></li>
<li>The 657 bp fragment in this reporter plasmid , pSA10 ( -791 to -135 ) , contains the HilD - and HilC-dependent +1 site , but not the HilA-dependent +1 site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>This activation is most probably through direct binding of HilD and HilC to sequences upstream and downstream of this alternative HilA-independent promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>These results suggest that HilD and HilC might directly activate invF expression by binding to sequences upstream of invF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>HilD and HilC do not activate invF expression from the HilA-dependent invF promoter Previously , we had shown that HilA activates invF expression by direct interaction with sequences upstream of invF called the HilA box ( Lostroh et al. , 2000 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>Identiﬁcation of candidate promoter sequences required for HilD - and HilC-dependent activation of invF To better deﬁne the sequences required for HilD - and HilC-dependent activation of invF , we cloned 919 bp from upstream of the invF ORF in front of a lacZ reporter gene in the pRW50 vector ( Lodge et al. , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>Instead , there appears to be a single HilD - and HilC-dependent promoter in this region corresponding to the predicted transcription start site located 631 bp upstream of the invF ORF ; 499 bp upstream of the HilA-dependent +1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>A HilD - and HilC-dependent invF transcription start site lies far upstream of the HilA-dependent transcription start site In order to determine where the HilD - and HilC-dependent promoter of invF is located , we performed primer extension experiments .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>HilD and HilC appear to activate transcription of invF from a HilD/C-dependent start site that is 631 nucleotides upstream of the invF ORF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
				</ul></div><br>
			<label class="collapse_TU" for="_6"><font size="5px">Transcription unit regulated by HilD</font></label>
				<input id="_6" type="checkbox">
				<div class="TU"><br><ul><li>pHilD is pLS118 , and pHilC is pLS119 ( 72 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923082" target="_blank">12923082</a>]<br></li>
<li>HilD is not known to control genes outside SPI-1 , except rtsA whose gene product is also involved in SPI-1 transcription ( Ellermeier et al. , 2005 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17971080" target="_blank">17971080</a>]<br></li>
<li>The rtsA gene is located outside SPI-1 but is directly activated by HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25991823" target="_blank">25991823</a>]<br></li>
<li>The majority of HilD targets are located outside SPI-1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24375101" target="_blank">24375101</a>]<br></li>
<li>Most previously described HilD targets are located within SPI-1 ( 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24375101" target="_blank">24375101</a>]<br></li>
<li>Blackwell Science , LtdOxford , UKMMIMolecular Microbiology 0950-382X Blackwell Publishing Ltd , 200347Original ArticleHilD and HilC directly activate invF expressionS.Akbar , L.M.Schechter , C.P.Lostroh and C.A.Lee Molecular Microbiology ( 2003 ) 47 ( 3 ) , 715 -- 728 AraC/XylS family members , HilD and HilC , directly activate virulence gene expression independently of HilA in Salmonella typhimurium Samina Akbar , † Lisa M. Schechter , ‡ C. Phoebe Lostroh § and Catherine A. Lee * Department of Microbiology and Molecular Genetics , Harvard Medical School , 200 Longwood Avenue , Boston , MA 02115 , USA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
				<label class="TU_collapse_similarity" for="tu_similarity_7">
Akabar , S. , Schechter , L.M. , Lostroh , C.P. and Lee , C.A. ( 2003 ) AraC/XylS family members , HilD and HilC , directly activate virulence gene expression independently of HilA in Salmonella typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17329372" target="_blank">17329372</a>]<br></label>
<input id="tu_similarity_7", type="checkbox">
<div class="TU_similar">
<li>References Akbar , S. , Schechter , L.M. , Lostroh , C.P. , and Lee , C.A. ( 2003 ) AraC/XylS family members , HilD and HilC , directly activate virulence gene expression independently of HilA in Salmonella typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16999831" target="_blank">16999831</a>]<br></li>
<li>Akbar , S. , Schechter , L.M. , Lostroh , C.P. and Lee , C.A. ( 2003 ) AraC/XylS family members , HilD and HilC , directly activate virulence gene expression independently of HilA in Salmonella typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28335027" target="_blank">28335027</a>]<br></li>
<li>Akbar , S. , Schechter , L.M. , Lostroh , C.P. , and Lee , C.A. ( 2003 ) AraC/XylS family members , HilD and HilC , directly activate virulence gene expression independently of HilA in Salmonella typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16045614" target="_blank">16045614</a>]<br></li>
<li>References Akbar , S. , Schechter , L. , Lostroh , C. , and Lee , C. ( 2003 ) AraC/XylS family members , HilD and HilC , directly activate virulence gene expression independently of HilA in Salmo-nella typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15661008" target="_blank">15661008</a>]<br></li>
<li>Akbar , S. , L. M. Schechter , C. P. Lostroh , and C. A. Lee , 2003 AraC/XylS family members , HilD and HilC , directly activate virulence gene expression independently of HilA in Sal-monella typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25991823" target="_blank">25991823</a>]<br></li>
<li>AraC/XylS family members , HilD and HilC , directly activate virulence gene expression independently of HilA in Salmonella typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23554419" target="_blank">23554419</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18723621" target="_blank">18723621</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17600486" target="_blank">17600486</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14645287" target="_blank">14645287</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21168230" target="_blank">21168230</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16443238" target="_blank">16443238</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15790293" target="_blank">15790293</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15968047" target="_blank">15968047</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19074398" target="_blank">19074398</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923082" target="_blank">12923082</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14679226" target="_blank">14679226</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15126488" target="_blank">15126488</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17675384" target="_blank">17675384</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15731035" target="_blank">15731035</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25135218" target="_blank">25135218</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15039320" target="_blank">15039320</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18441068" target="_blank">18441068</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15516559" target="_blank">15516559</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18160484" target="_blank">18160484</a>]<br></li>
<li>AraC / XylS family members , HilD and HilC , directly activate virulence gene expression independently of HilA in Salmonella typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15256548" target="_blank">15256548</a>]<br></li>
<li>AraC/XylS family members , HilD and HilC , directly activate virulence gene expression independently of HilA in Salmonella typhi-murium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28329249" target="_blank">28329249</a>]<br></li>
<li>AraC/XylS family members , HilD and HilC , directly activate virulence gene expression independently of HilA in Salmonella Typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24375101" target="_blank">24375101</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27601571" target="_blank">27601571</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27185788" target="_blank">27185788</a>]<br></li></div>
				<label class="TU_collapse_similarity" for="tu_similarity_17">
Mol Microbiol 90:1216 -- 1232 Akbar S , Schechter LM , Lostroh CP , Lee CA ( 2003 ) AraC/XylS family members , HilD and HilC , directly activate virulence gene expression independently of HilA in Salmonella typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27549210" target="_blank">27549210</a>]<br></label>
<input id="tu_similarity_17", type="checkbox">
<div class="TU_similar">
<li>Akbar S. , Schechter L.M. , Lostroh C.P. , Lee C.A. ( 2003 ) AraC/XylS family members , HilD and HilC , directly activate virulence gene expression independently of HilA in Salmonella typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22004521" target="_blank">22004521</a>]<br></li></div>
<li>Our data show that overexpressing HilD activates invF expression more profoundly than overexpressing HilA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28439039" target="_blank">28439039</a>]<br></li>
<li>We found that HilD and HilC induce invF expression from pSA7 in E. coli , comparable to that seen in S. typhimurium ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>It is possible that HilD , but not HilC , activates invF expression from a second promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>We found that neither HilD nor HilC activate invF-lacZ expression from pSA10 ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>Under PtetA-ﬂhD C conditions , neither HilD nor HilD HTH overexpression showed any effect on ﬂagellar gene expression ( Fig. 1D ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24488311" target="_blank">24488311</a>]<br></li>
<li>Figure S4 HilE negatively regulates HilD expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20686667" target="_blank">20686667</a>]<br></li>
<li>245 SprB transcriptionally represses HilD expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20190046" target="_blank">20190046</a>]<br></li>
<li>However , a disruption in hilD reduced hilC expression nearly twofold , indicating that HilD regulates hilC expression ( Fig. 3B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11292791" target="_blank">11292791</a>]<br></li>
<li>The evidence that L-arabinose regulates SPI-1 expression through HilD suggested that hilD expression itself might be controlled by L-arabinose .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25991823" target="_blank">25991823</a>]<br></li>
<li>It is suggested that HilD directly activates hilA transcription ( 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18245288" target="_blank">18245288</a>]<br></li>
<li>This is important , as HilD is known to induce its own expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27185788" target="_blank">27185788</a>]<br></li>
<li>Yet , degradation of HilD is not necessary to repress HilD activity , as SCFAs continue to inhibit SPI-1 expression in a lon mutant ( 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27185788" target="_blank">27185788</a>]<br></li>
<li>D. Induced HilD expression complements invR expression to a SPI-1 disruption strain ( spi-1 is deleted from sitA to invH ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17971080" target="_blank">17971080</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17971080" target="_blank">17971080</a>]<br></li>
<li>It is believed that HilD directly regulates hilA expression as it has been shown to bind to hilA promoter sequences .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15765064" target="_blank">15765064</a>]<br></li>
<li>Since hilD transcription appears to be constitutive , one possible mechanism for Lon activity is that it degrades HilD in repressing environmental conditions to regulate hilA expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15765064" target="_blank">15765064</a>]<br></li>
<li>( B ) Fur overexpression induces hilA expression when HilD is present .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
<li>Surprisingly , our results indicate that functional HilD is required for hilA expression even when known negative regulators are deleted .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12511499" target="_blank">12511499</a>]<br></li>
<li>These results are consistent with our model that FliZ is functioning through HilD to control hilA expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20889744" target="_blank">20889744</a>]<br></li>
<li>To determine HilD levels in cells in stationary phase , bacteria were grown with shaking in low-salt LB me-dium ( 0.5 % NaCl ) until they reached stationary phase , then subcultured 1:100 in 90 ml of high-salt LB medium ( 1 % NaCl ) containing 100 g/ml ampicillin and 0.4 g/ml tetracycline ( to induce hilD ) , and grown for 18 to 22 h standing at 37 ° C .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20889744" target="_blank">20889744</a>]<br></li>
<li>( B ) qRT-PCR analysis of changes in hilA expression in wildtype , ΔloiA , ΔhilD , ΔloiAΔhilD , ΔloiAΔhilD + pHilD and ΔloiAΔhilD + pLoiA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28575106" target="_blank">28575106</a>]<br></li>
<li>To test changes in hilA expression in wild-type , ΔloiA , ΔhilD , ΔloiAΔhilD , ΔloiAΔhilD + pHilD and ΔloiAΔhilD + pLoiA , strains were also grown under SPI-1-inducing conditions for 4 h to late-exponential phase ( OD600 ~ 0.6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28575106" target="_blank">28575106</a>]<br></li>
<li>After repressing loop is disrupted , HilD 39 and its paralogs are still needed to directly activate transcription through DSR under inducing conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16443238" target="_blank">16443238</a>]<br></li>
				<label class="TU_collapse_similarity" for="tu_similarity_41">
Expression analyses and protein-DNA interaction assays showed regulation by HilD for six of these genes : gtgE , phoH , sinR , SL1263 ( lpxR ) and SL4247 were regulated directly , whereas SL1896 was regulated indirectly .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27886269" target="_blank">27886269</a>]<br></label>
<input id="tu_similarity_41", type="checkbox">
<div class="TU_similar">
<li>Expression analyses and protein-DNA interaction assays show that HilD regulates six of these genes : gtgE , phoH , sinR , lpxR and SL4247 were regulated directly , whereas SL1896 was regulated indirectly .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27886269" target="_blank">27886269</a>]<br></li></div>
<li>The SPI2 genes also play a role in inducing intestinal inﬂammation and are known to be regulated by HilD , a SPI1 regulator ( 7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20833811" target="_blank">20833811</a>]<br></li>
				<label class="TU_collapse_similarity" for="tu_similarity_44">
For clarity , the genes encoding HilD , HilC , RtsA and HilA are not shown .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16045614" target="_blank">16045614</a>]<br></label>
<input id="tu_similarity_44", type="checkbox">
<div class="TU_similar">
<li>For clarity , the genes encoding HilD , HilC , RtsA , and HilA are not shown .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27704705" target="_blank">27704705</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20889744" target="_blank">20889744</a>]<br></li></div>
<li>The genes encoding HilC and HilD are located on SPI1 , whereas RtsA is in encoded on a 15 kb Salmonella-speciﬁc insert located near the gene for tRNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17208038" target="_blank">17208038</a>]<br></li>
<li>HilC and HilD are encoded within the SPI1 locus , while RtsA is encoded in an operon with RtsB located at 93.9 centisomes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
<li>HilC and HilD are encoded within SPI1 , whereas RtsA is encoded in an operon at 93.9 centisomes that also encodes RtsB , which negatively regulates ﬂhDC and hence the ﬂagellar regulon ( Ellermeier and Slauch , 2003 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16045614" target="_blank">16045614</a>]<br></li>
<li>The hilA gene is silenced by H-NS under low-osmolarity conditions , until H-NS-mediated repression is relieved by HilC and HilD , two other transcription factors encoded within SPI-1 ( Schechter et al. , 2003 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21570144" target="_blank">21570144</a>]<br></li>
<li>RtsA is a close homologue of HilD and HilC but is encoded outside SPI-1 , in an operon that includes another TF-encoding gene , rtsB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27601571" target="_blank">27601571</a>]<br></li>
<li>We propose that siiA is regulated by HilD , HilC , RtsA , HilA , and SprB , which would indicate that SPI-1 and SPI-4 gene expression is tightly linked .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24375101" target="_blank">24375101</a>]<br></li>
<li>It is certainly possible that other genes are directly regulated by HilC , HilD and RtsA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17208038" target="_blank">17208038</a>]<br></li>
<li>We propose that DapZ is regulated by both RtsA and HilC but not HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27601571" target="_blank">27601571</a>]<br></li>
<li>In turn , hilA is regulated by HilD and HilC ( Schechter and Lee , 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21168230" target="_blank">21168230</a>]<br></li>
<li>The rtsAB operon is transcriptionally activated by HilD , HilC , and RtsA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24488311" target="_blank">24488311</a>]<br></li>
<li>This transfrag is regulated by HilD and HilC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27601571" target="_blank">27601571</a>]<br></li>
<li>In addition , by reducing HilD synthesis , CsrA would also affect HilD positive autoregulation ( Olekhnovich and Kadner , 2002 ; Ellermeier et al. , 2005 ) , thus reducing hilD transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21518393" target="_blank">21518393</a>]<br></li>
<li>Moreover , hilA also appears to be regulated by other proteins encoded within SPI1 including HilC ( also known as SirC and SprA ) ( Eichelberg et al. , 1999 ; Rakeman et al. , 1999 ; Schechter et al. , 1999 ) and HilD ( Schechter et al. , 1999 ) , further complicating the understanding of how SPI1 gene expression is co-ordinated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10692170" target="_blank">10692170</a>]<br></li>
<li>Both HilC and HilD are positive regulators of hilA expression ( Schechter et al. , 1999 ; Lostroh et al. , 2000 ; Lucas and Lee , 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12791144" target="_blank">12791144</a>]<br></li>
<li>Transcription of invF is primarily regulated by HilA , but HilD , HilC and RtsA are each capable of partially activating invF ( Eichelberg et al. , 1999 ; Rakeman et al. , 1999 ; Akbar et al. , 2003 ; Ellermeier and Slauch , 2003 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16045614" target="_blank">16045614</a>]<br></li>
<li>LoiA regulates hilA and other SPI-1 genes through HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28575106" target="_blank">28575106</a>]<br></li>
<li>HilC , HilD and RtsA , when overproduced are clearly able to induce hilD transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17208038" target="_blank">17208038</a>]<br></li>
<li>That means that at least hilA and rtsA are induced by HilD , HilC , and RtsA proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17675384" target="_blank">17675384</a>]<br></li>
<li>Under high-osmolarity conditions , HilD , HilC , and RtsA activate SPI1 genes directly and indirectly through hilA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17675384" target="_blank">17675384</a>]<br></li>
<li>HilD and HilC derepress hilA transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14977940" target="_blank">14977940</a>]<br></li>
<li>SPI-1 also encodes transcriptional activators responsible for its own expression : HilA , HilC , HilD , and InvF ( Lostroh and Lee 2001 ; Ellermeier and Slauch 2007 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25991823" target="_blank">25991823</a>]<br></li>
<li>However , under low osmolarity conditions , HilD and HilC activators were not essential for hilA expression in hns and hha hns double mutants ( lines 7 and 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16443238" target="_blank">16443238</a>]<br></li>
<li>When present , HilC or HilD binds to this same site and promotes hilA expression ( 63 ; Schechter and Lee , unpublished results ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11292791" target="_blank">11292791</a>]<br></li>
<li>In contrast , HilD and HilC have been shown to independently bind to the URS and promote hilA expression ( Schechter and Lee , 2001 ; Olekhnovich and Kadner , 2002 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>Thus , we conclude that Hha does not require HilD protein , but rather regulates both hilA and hilD ( as well as hilC and rtsA ) transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22021388" target="_blank">22021388</a>]<br></li>
				<label class="TU_collapse_similarity" for="tu_similarity_71">
The two-component regulatory systems , SirA/BarA and OmpR/EnvZ , function through HilD , thus inducing hilC , rtsA and hilA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16045614" target="_blank">16045614</a>]<br></label>
<input id="tu_similarity_71", type="checkbox">
<div class="TU_similar">
<li>The two-component regulatory systems , SirA/BarA and OmpR/EnvZ function through HilD , thus inducing hilC , rtsA , hilA and invF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16045614" target="_blank">16045614</a>]<br></li></div>
<li>Given that expression of hilA , a gene directly regulated by HilD , signiﬁcantly increases in bile , we investigated if HilD is posttranscriptionally regulated by bile in S. Typhi .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29229736" target="_blank">29229736</a>]<br></li>
<li>Coincident with this reduction in HilD protein , hilA = - lacZ expression was potently repressed by bile ( 9.7-fold repression ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27185788" target="_blank">27185788</a>]<br></li>
<li>We have evidence that HilD modestly activates invF expression independently of its effects on hilA , even in WT S. typhimurium grown under inducing conditions in vitro ( ﬁgure 1B , orange arrow ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11755416" target="_blank">11755416</a>]<br></li>
<li>HilD plays a major role in regulating hilA expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28426789" target="_blank">28426789</a>]<br></li>
				<label class="TU_collapse_similarity" for="tu_similarity_77">
As shown in Table 2 , hilA expression was induced 40-fold by RtsA , 20-fold by HilD , and 120-fold by HilC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14679226" target="_blank">14679226</a>]<br></label>
<input id="tu_similarity_77", type="checkbox">
<div class="TU_similar">
<li>Figure 7B shows that expression of hilA was induced 40-fold by RtsA , 20-fold by HilD , and 120-fold by HilC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923082" target="_blank">12923082</a>]<br></li></div>
<li>However , this condition inhibits hilA expression through a repressor ( R2 ) that can not be counteracted by HilC or HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10844688" target="_blank">10844688</a>]<br></li>
<li>Alternatively , this condition may permit HilC or HilD expression and activity while repressing hilA by a mechanism that can not be counteracted by HilC or HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10844688" target="_blank">10844688</a>]<br></li>
<li>SprB directly activates SPI4 gene expression and weakly represses SPI1 gene expression through HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20190046" target="_blank">20190046</a>]<br></li>
<li>It would also explain why other SPI1 regulators -- namely HilC , HilD , and RtsA -- affect SPI4 gene expression , as they in turn regulate HilA expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20190046" target="_blank">20190046</a>]<br></li>
<li>HilD is dominant , and there is no hilA expression in its absence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21680637" target="_blank">21680637</a>]<br></li>
<li>Because HilD is required for hilA expression ( Schechter et al. , 1999 ) , and current models suggest that HilD is also required for invF expression independently of hilA ( Lucas and Lee , 2000 ) , our data are consistent with a model in which hilD is induced by Fis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11123690" target="_blank">11123690</a>]<br></li>
<li>HilA , in turn , is subject to multiple controls , with two additional SPI1 regulators , HilC and HilD , inducing hilA expression ( 18 , 32 , 58 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18424519" target="_blank">18424519</a>]<br></li>
<li>Under activating conditions , hilA expression is thought to be derepressed by two AraC-like transcriptional regulators encoded on SPI1 : HilD and HilC ( Schechter et al. , 1999 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10844688" target="_blank">10844688</a>]<br></li>
<li>BarA/SirA ( 1 ) , HilD ( 14 , 30 ) , HilC/SirC/SprA ( 11 , 12 , 28 , 30 ) , CsrAB ( 1 ) , PhoB , FadD , and FliZ ( 25 ) have been shown to activate hilA expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16391141" target="_blank">16391141</a>]<br></li>
<li>FliZ and RcsBC affect hilA expression through HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17951383" target="_blank">17951383</a>]<br></li>
<li>Still , it is intriguing to consider that under certain conditions HilC and HilD may be able to activate hilA expression and that under other conditions they may be able to increase hilA expression by derepressing hilA transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12933899" target="_blank">12933899</a>]<br></li>
<li>Through genetic and biochemical analyses , we show that FliZ posttranslationally controls HilD to positively regulate hilA expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20889744" target="_blank">20889744</a>]<br></li>
				<label class="TU_collapse_similarity" for="tu_similarity_91">
The Salmonella Spi1 virulence regulatory protein HilD directly activates transcription of the flagellar master operon flhDC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25315056" target="_blank">25315056</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27886269" target="_blank">27886269</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28329249" target="_blank">28329249</a>]<br></label>
<input id="tu_similarity_91", type="checkbox">
<div class="TU_similar">
<li>Singer , H.M. , Ku $ hne , C. , Deditius , J.A. , Hughes , K.T. , and Erhardt , M. ( 2014 ) The Salmonella Spi1 virulence regulatory protein HilD directly activates transcription of the flagellar master operon flhDC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27206164" target="_blank">27206164</a>]<br></li>
<li>Singer , H.M. , Kühne , C. , Deditius , J.A. , Hughes , K.T. , and Erhardt , M. ( 2014 ) The Salmonella Spi1 virulence regulatory protein HilD directly activates transcription of the flagellar master operon flhDC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27404739" target="_blank">27404739</a>]<br></li></div>
<li>These result demonstrated that CobB can positively regulate SPI1 through controlling hilD transcription , and this mechanism is different from that of Pat , which regulates Salmonella invasion by reducing HilD Fig. 7 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27341691" target="_blank">27341691</a>]<br></li>
<li>It is possible that the long HilDdependent extension product is not speciﬁc for invF and was primed from another transcript induced by HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>Furthermore , it has been shown that H-NS directly represses ssrAB and that HilD is required for the expression of ssrAB but only if H-NS is present ( 21 , 42 , 43 , 50 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25135218" target="_blank">25135218</a>]<br></li>
				<label class="TU_collapse_similarity" for="tu_similarity_97">
It is also reported that HilD is able to activate the transcription of invF from a promoter that is far upstream of its HilA-dependent promoter ( 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28439039" target="_blank">28439039</a>]<br></label>
<input id="tu_similarity_97", type="checkbox">
<div class="TU_similar">
<li>Furthermore , it was reported HilD is able to activate the transcription of invF from a promoter that is far upstream of its HilA-dependent promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28439039" target="_blank">28439039</a>]<br></li></div>
<li>The ﬂiZ gene is transcribed in the ﬂiAZ operon , and FliZ activates the Spi1 transcriptional activator HilD , which in turn activates Spi1 transcription under high salt ( 200 mM NaCl ) microaerobic growth conditions ( 48 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24706743" target="_blank">24706743</a>]<br></li>
<li>However , we did not observe transcriptional decrease of SPI-2 genes in Δpat by RNA-seq ; one possible reason is that HilD is not required for SPI-2 expression under the in vitro growth conditions [ 41 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26810370" target="_blank">26810370</a>]<br></li>
				<label class="TU_collapse_similarity" for="tu_similarity_101">
Interestingly , HilD is required for the expression of ssrAB when Salmonella is grown to late stationary phase in LB medium but not when it is grown in minimal media ( 21 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25135218" target="_blank">25135218</a>]<br></label>
<input id="tu_similarity_101", type="checkbox">
<div class="TU_similar">
<li>Interestingly , HilD is required for the expression of ssrAB when Salmonella is grown in LB medium but not when it is grown in minimal media , such as N-MM ( Fig. 3 ) ( 21 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25135218" target="_blank">25135218</a>]<br></li></div>
<li>It is not clear whether SirA is the factor which controls hilC , in addition to HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19537165" target="_blank">19537165</a>]<br></li>
<li>Yet , in the absence of HilD , HilA is not expressed even though hilC is transcribed ( albeit at reduced levels ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20686667" target="_blank">20686667</a>]<br></li>
<li>Of the three , HilD is dominant , as there is no HilA expression in its absence [ 33 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20686667" target="_blank">20686667</a>]<br></li>
<li>Of the three , HilD is the most important as HilA is not expressed in its absence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20686667" target="_blank">20686667</a>]<br></li>
<li>KEYWORDS Salmonella pathogenicity island 1 ; Salmonella invasion ; L-arabinose ; HilD INVASION of epithelial cells by Salmonella enterica requires the expression of genes located in a 40-kb region known as Salmonella pathogenicity island 1 ( SPI-1 ) ( Lostroh and Lee 2001 ; Altier 2005 ; Jones 2005 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25991823" target="_blank">25991823</a>]<br></li>
				<label class="TU_collapse_similarity" for="tu_similarity_108">
Martínez , L. C. , Banda , M. M. , Fernández-Mora , M. , Santana , F. J. & Bustamante , V. H. HilD induces expression of Salmonella pathogenicity island 2 genes by displacing the global negative regulator H-NS from ssrAB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27886269" target="_blank">27886269</a>]<br></label>
<input id="tu_similarity_108", type="checkbox">
<div class="TU_similar">
<li>HilD induces expression of Salmonella pathogenicity island 2 genes by displacing the global negative regulator H-NS from ssrAB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26300871" target="_blank">26300871</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28329249" target="_blank">28329249</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28660201" target="_blank">28660201</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28426789" target="_blank">28426789</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29447698" target="_blank">29447698</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28426789" target="_blank">28426789</a>]<br></li></div>
<li>Mechanistically , HilD indirectly regulates SsrB-regulated genes by antagonizing the H-NS-mediated repression of ssrAB at stationary phase under SPI-1 inducing conditions [ 30 ] ( see also Fig 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27564394" target="_blank">27564394</a>]<br></li>
<li>We demonstrate transcription activation of 8 genes by HilD ; four of these genes have not been previously described as being regulated by HilD , including lpxR , which encodes a lipid A deacylase important for immune evasion .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24375101" target="_blank">24375101</a>]<br></li>
<li>Consistent with the SPI-1 gene levels , SPI-1 gene transcriptional regulators HilC , HilD , HilA , rtsA , and invF were also signiﬁcantly repressed ( Fig. 6b ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27549210" target="_blank">27549210</a>]<br></li>
<li>For example , SPI-2-encoded SsrB downregulates SPI-1 genes by repressing HilA and HilD ( Pérez-Morales et al. , 2017 ) , and reciprocally , SPI-1-encoded HilD upregulates SPI-2 gene expression by directly binding the ssrAB operon ( Bustamante et al. , 2008 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29034217" target="_blank">29034217</a>]<br></li>
<li>Firstly , we demonstrated that SirA positively controls expression of the SPI-1 and SPI-2 genes through HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21518393" target="_blank">21518393</a>]<br></li>
<li>HilD also directly controls the expression of the SPI-1 genes hilD , hilC , and invF , as well as other acquired and ancestral genes located outside SPI-1 , such as rtsA , ﬂhDC , siiA , lpxR , ytfK , , and ( 2 , 25 -- 31 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25135218" target="_blank">25135218</a>]<br></li>
<li>Salmonella Patogenicity Island 1 ( SPI-1 ) is a chromosomal region containing 39 genes that have crucial virulence roles.The AraC-like transcriptional regulator HilD , encoded in SPI-1 , positively controls the expression of the SPI-1 genes , as well as of several other virulence genes located outside SPI-1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27886269" target="_blank">27886269</a>]<br></li>
<li>HilD also induces directly , or indirectly through HilA , InvF or other regulators , the expression of several other virulence genes , including genes located in SPI-2 , SPI-4 , SPI-5 and other islands4 ,14,29 -- 34 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27886269" target="_blank">27886269</a>]<br></li>
<li>The expression of SPI-1 genes is tightly regulated by a number of transcriptional regulators , including HilD , HilA , and InvF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28439039" target="_blank">28439039</a>]<br></li>
<li>For example , the transcription of SPI-1 genes can be activated by HilA , HilC , HilD , InvF , and SirA ( 30 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28439039" target="_blank">28439039</a>]<br></li>
<li>Thus , reduced levels of FliZ are expected to lead to reduced levels of the transcriptional regulators HilA ( controlled by HilD ) and InvF ( controlled by HilA ) , both of which are known to activate the expression of effector genes located inside and outside ( indicated by a dotted box ) SPI-1 ( 77 -- 79 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29150501" target="_blank">29150501</a>]<br></li>
<li>Expression of the SPI-1 genes is controlled through a regulatory cascade formed by the transcriptional regulators HilD , HilA and InvF , encoded within this island4 ,20 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27886269" target="_blank">27886269</a>]<br></li>
<li>Expression of the SPI-1 T3SS and its associated genes ( several of which are encoded outside the SPI-1 pathogenicity island ) is controlled by a hierarchy of regulators ( HilD , HilA , HilC , RtsA , InvF ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29229736" target="_blank">29229736</a>]<br></li>
<li>Expression of SPI-1 genes is coordinately regulated by environmental signals and cellular factors through a regulatory network of SPI-1-encoded transcriptional activators , and one of the main transcription factors is HilD ( 8 -- 10 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24682814" target="_blank">24682814</a>]<br></li>
<li>Expression of hilA is directly controlled by three AraC-like transcriptional regulators , including SPI-1-encoded HilC and HilD , and RtsA encoded outside of SPI-1 , which constitute a feed-forward regulatory loop [ 29 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28575106" target="_blank">28575106</a>]<br></li>
<li>HilA transcriptional expression is autoregulated and tightly modulated by the combined action of three AraC-like transcriptional activators : HilC , HilD and RtsA [ 13,14 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28426789" target="_blank">28426789</a>]<br></li>
<li>Expression of HilA is controlled by the combined action of three AraC-like transcriptional activators : HilC and HilD , both encoded within SPI-1 ( 230 ) , and RtsA , encoded within an independent island ( 231 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23554419" target="_blank">23554419</a>]<br></li>
<li>InvF and HilA are transcriptional regulators of SPI-1 genes that are encoded within SPI-1 ; hilA itself is regulated by several factors , including PhoP-PhoQ , SirA , HilC , and HilD ( 3 -- 5 , 20 , 37 , 44 , 66 , 72 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10633113" target="_blank">10633113</a>]<br></li>
<li>HilD , HilC , and RtsA all positively regulate invF , which is located within SPI-1 and encodes another of the SPI-1-associated TFs ( Fig. 1 and 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27601571" target="_blank">27601571</a>]<br></li>
				<label class="TU_collapse_similarity" for="tu_similarity_129">
SPI-1 genes are organized in seven or more transcriptional units , whose expression is under the control of four SPI-encoded transcription factors : HilA , HilC , HilD , and InvF ( Lostroh and Lee 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20008574" target="_blank">20008574</a>]<br></label>
<input id="tu_similarity_129", type="checkbox">
<div class="TU_similar">
<li>SPI-1 genes are organized in seven or more transcriptional units under the control of the SPI-1-encoded transcription factors HilA , HilC , HilD and InvF ( Lostroh and Lee , 2001 ) and of RtsA , a transcription factor encoded outside SPI-1 ( Ellermeier and Slauch , 2003 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24354910" target="_blank">24354910</a>]<br></li></div>
<li>Additionally , there is crosstalk between SPI-1 and SPI-2 through which HilD induces expression of ssrAB [ 18 ] ( Figure 1A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22457642" target="_blank">22457642</a>]<br></li>
<li>sinR SL0300 STM0304 HilD SL4247 STM4310 HilD/HilC/RtsA SL4248 STM4312 HilD SL4249 STM4313 HilD nhaA SL0040 STM0039 HilC nhaR SL0041 STM0040 HilC SL1177 STM1239 InvF SL0336 STM0341 SprB SL1770 STM1841 SprB SL3112 STM3138 SprB pckA SL3467 STM3500 SprB yjbJ SL4176 STM4240 SprB NA STnc520 STnc520 STnc520 SprB SL0973 STM1034 HilA SL0975 STM1036 HilA SL0976 STM1037 HilA a The genes listed are direct regulatory targets of SPI-1-associated TFs that are encoded outside SPI-1 whose expression proﬁles ( from reference 4 ) have an average correlation coefﬁcient of 0.5 with the proﬁles for invasion genes within SPI-1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27601571" target="_blank">27601571</a>]<br></li>
<li>In addition , HilD and HilC are important AraC/XylS transcriptional regulators that mediate expression of HilA , which then activates all genes necessary for a functional SPI-1 encoded T3SS ( 29 , 30 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22004521" target="_blank">22004521</a>]<br></li>
<li>Thus , HilD coordinates expression of SPI-1 and non-SPI-1 genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24375101" target="_blank">24375101</a>]<br></li>
<li>Within SPI-1 there are key transcriptional activators which regulate expression of SPI-1 genes : HilC , HilD , HilA , and InvF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22291596" target="_blank">22291596</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22291596" target="_blank">22291596</a>]<br></li>
<li>The other six genes STM1329 , STM1330 , STM1600 , STM1854 , STM2585 and yneC ( STM4079S ) show a pattern of expression that resembles the SPI2-encoded and SPI2-associated genes , that are positively regulated by both HilD and SsrB under SPI1 and SPI2-inducing conditions , respectively , and are upregulated within macrophages [ 14 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27564394" target="_blank">27564394</a>]<br></li>
<li>Previously described HilD regulatory targets are hilD itself , hilA , hilC , rtsA , invF , and downstream genes ( the inv-spa and sic-sip operons ) , invR , dsbA , slrP , ssrAB ( 6 ) , and genes in SPI-4 ( 15 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24375101" target="_blank">24375101</a>]<br></li>
<li>The genes that encode the HilD and HilC proteins are located on SPI1 , while the gene that encodes RtsA is located on an island integrated at the tRNAPheU ( 12 , 14 , 40 , 43 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17675384" target="_blank">17675384</a>]<br></li>
<li>Our data also indicate that the two genes downstream of rtsB are likely to be cotranscribed with rtsAB and that the nearby gene is positively regulated by HilD , HilC , and RtsA ( Fig. 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27601571" target="_blank">27601571</a>]<br></li>
<li>This result indicated that hilC has two promoters and that the upstream promoter is repressed by HilC and HilD while the downstream one is activated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12107132" target="_blank">12107132</a>]<br></li>
<li>Whether or not the phoH , SL4247 and SL1896 genes , or even the SL1028 , SL3812 and SL4433 genes that are co-expressed with SPI-1 but not regulated by HilD , have a role in Salmonella virulence is a matter of our current investigation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27886269" target="_blank">27886269</a>]<br></li>
<li>Interestingly , between these genes are SL1265 and SL4248 , located in the S. Typhimurium genomic islands carrying lpxR ( SL1263 ) and SL4247 , respectively , as well as slyA that is positively regulated by HilD ( our unpublished results ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27886269" target="_blank">27886269</a>]<br></li>
<li>Thus , the hup and ﬁs mutations may decrease hilA expression ( i ) by decreasing its derepression by HilD and/or ( ii ) by increasing its repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12933899" target="_blank">12933899</a>]<br></li>
<li>Two of the 8 genes , mcpA ( STM3138 ) and mcpC ( STM3216 ) , are predicted to encode Salmo-nella-specific methyl-accepting chemotaxis proteins [ 40 ] and display similar expression patterns to the SPI1-encoded , SPI1-associated and SPI4-encoded genes which are directly or indirectly activated by HilD , and repressed by SsrB , but are not regulated by HilA suggesting they are controlled by the HilD feed-forward loop .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27564394" target="_blank">27564394</a>]<br></li>
<li>represses expression of HilD by binding to the hilD leader transcript In E. coli , CsrB and CsrC regulate gene expression by binding to and sequestering the RNA binding protein CsrA , thus counteracting the negative posttranscriptional control exerted by CsrA on its target mRNAs ( Babitzke and Romeo , 2007 ; Lucchetti-Miganeh et al. , 2008 ; Timmermans and Van Melderen , 2010 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21518393" target="_blank">21518393</a>]<br></li>
<li>Therefore , CpxR and RpoH seem to affect HilD concentration and thus repress the SPI-1 genes differentially ; RpoH by inducing transcription of lon and CpxR by probably affecting the posttranscriptional expression or activity of Lon , or through another protease .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26300871" target="_blank">26300871</a>]<br></li>
<li>ls the expression of SPI-1 and SPI-2 genes by regulating HilD expression tro n o c irA S HilD and OmpR positively regulate ssrAB expression by directly binding to its regulatory region ( Lee et al. , 2000 ; Feng et al. , 2003 , 2004 ; Bustamante et al. , 2008 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21518393" target="_blank">21518393</a>]<br></li>
<li>The regulatory cascade diverges when HilD , in a growth phase-dependent manner , activates expression of hilA and the ssrAB operon , by counteracting the repression exerted by H-NS on their promoters ( Bustamante et al. , 2008 , Schechter et al. , 2003 ; Olekhnovich and Kadner , 2006 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21518393" target="_blank">21518393</a>]<br></li>
<li>Whether HilC and HilD act directly at the hilA promoter is not yet known .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11123690" target="_blank">11123690</a>]<br></li>
<li>SPI1 gene expression is positively regulated by the transcriptional activators HilA , HilC , and HilD , encoded within SPI1 , and negatively regulated by the regulators HilE and PhoP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18160484" target="_blank">18160484</a>]<br></li>
<li>HilA expression is also regulated by the regulators HilC and HilD [ 29 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21379575" target="_blank">21379575</a>]<br></li>
<li>Transcription of hilA is subject to complex control and is stimulated by the SPI-1-encoded HilC and HilD proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15126488" target="_blank">15126488</a>]<br></li>
<li>At the transcriptional level , the expression of hilD is positively autoregulated and modulated by a feed-forward regulatory loop involving HilD itself and the AraC-like regulators HilC and RtsA ( Olekhnovich and Kadner , 2002 ; Ellermeier et al. , 2005 ; Golubeva et al. , 2012 ) , while post-transcriptionally it is positively controlled by a regulatory cascade integrated by the SirA/BarA and Csr global regulatory systems ( Martínez et al. , 2011 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26300871" target="_blank">26300871</a>]<br></li>
<li>Transcription of the hilA gene is also stimulated by the SPI1-encoded AraC/XylS-like regulatory proteins HilC , HilD and RtsA ( Eichelberg and Galan , 1999 ; Lostroh et al. , 2000 ; Rhen and Dorman , 2005 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28631437" target="_blank">28631437</a>]<br></li>
<li>The hilD promoter is active in vitro and unaffected by the HilC and HilD proteins or other SPI1 products .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12107132" target="_blank">12107132</a>]<br></li>
<li>HilD can be repressed by HilE , and HilC and RstA further amplify the inducing signal .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21984608" target="_blank">21984608</a>]<br></li>
<li>In the hilA mutant , which expressed HilD , exposure to salt increased hilA , hilD , and sipC transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18441068" target="_blank">18441068</a>]<br></li>
<li>Fragments containing the hilA promoter , known to be recognized by HilD ( Schechter and Lee , 2001 ) , served as positive control .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17971080" target="_blank">17971080</a>]<br></li>
<li>SPI1 gene expression is negatively regulated by HilE , which binds to HilD and prevents it from activating the SPI1 promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20833811" target="_blank">20833811</a>]<br></li>
<li>This two-gene operon requires HilD for expression , and its expression is correlated with the other SPI1 genes ( 20 , 22 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20833811" target="_blank">20833811</a>]<br></li>
<li>In addition to the negative control that was exerted by H-NS/Hha under low-osmolarity conditions , the homologous virulence activators HilD , HilC , and RtsA ( Hil activators ) induced rtsA-lac expression in a high-salt medium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17675384" target="_blank">17675384</a>]<br></li>
<li>As expected , most hilA expression from the fulllength promoter was lost in a strain deleted for SPI1 ( DSPI1 ) , which encodes the activators HilC and HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16443238" target="_blank">16443238</a>]<br></li>
<li>HilC has high homology to HilD and also binds to the hilA promoter ; however , it is not required for hilA expression ( 58 , 59 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15039320" target="_blank">15039320</a>]<br></li>
<li>In this model , the repressor is always present but can not inhibit hilA expression under activating conditions because HilD ( and , to a lesser extent , HilC ) displaces it from the URS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11292791" target="_blank">11292791</a>]<br></li>
<li>As detailed below , genetic studies indicate that the downstream -10 sequence , corresponding to the short primer extension product , is essential for HilD and HilC to activ-ate invF expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>To see if these sequences are important for HilD and HilC to activate invF , we constructed derivatives of pSA7 in which the upstream TTTGAT was changed to TTTGAG and the downstream TATTCT was changed to TATGCG .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>The HilC and HilD proteins bind to their own gene promoters as well as to at least two regions in the hilA promoter , called sites A1 ( positions 242 to 182 , relative to the start of transcription ) and A2 ( positions 85 to 61 ) ( 18 , 24 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15126488" target="_blank">15126488</a>]<br></li>
<li>Two SPI1 gene products , HilC and HilD , have been shown to bind to the upstream repressing sequence of hilA to induce its transcription ( 12 , 52 , 53 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18245288" target="_blank">18245288</a>]<br></li>
<li>Expression of HilD-3X-FLAG was induced with tetracycline , an antibiotic cocktail was added to halt transcription and translation , and the half-life of HilD was assessed by Western blotting at time points thereafter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27341691" target="_blank">27341691</a>]<br></li>
<li>However , HilD regulates invF by binding to regions located both upstream and downstream of the HilC/Ddependent invF promoter ( 28 , 60 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25135218" target="_blank">25135218</a>]<br></li>
<li>These results also explain why our invF-lacZ reporter plasmid , pVV448 , was not activated by HilD or HilC to the same extent as the chromosomal invF-lacZ fusion because pVV448 does not contain the HilD - and HilC-dependent +1 of invF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>These results suggest that HilD and HilC do not initiate transcription from the putative upstream promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>HilD and HilC do not activate invF expression from the HilA-dependent invF promoter Previously , we had shown that HilA activates invF expression by direct interaction with sequences upstream of invF called the HilA box ( Lostroh et al. , 2000 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>Recall that HilD , but not HilC , activates expression of invF sevenfold from the pVV448 reporter plasmid , which carries this region ( Fig. 4 , black bars ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>Both HilC and HilD also directly activate invF , and it has been suggested that they activate a promoter located upstream of the HilA-dependent promoter PinvF ( 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923082" target="_blank">12923082</a>]<br></li>
<li>Our studies show that HilD and HilC activate transcription of invF from a promoter that is far upstream of its HilA-dependent promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>These results suggest that HilD and HilC might directly activate invF expression by binding to sequences upstream of invF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>In addition to activating invF expression , HilD and HilC may affect the invH gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>These results indicate that HilD and HilC require the downstream binding site to activate invF expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>HilD proteins with Lon using cells that express hilC and hilD through their own promoters because the transcription of both genes depends on several transcriptional factors .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15661008" target="_blank">15661008</a>]<br></li>
<li>( B ) Transcription levels of two SPI-1 genes under HilD control ( invF and sipB ) in Dam1 ( solid histograms ) and Dam ( open histo-grams ) strains that express hilD from a heterologous , tetracy-cline-dependent promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20008574" target="_blank">20008574</a>]<br></li>
<li>However , several lines of evidence suggest that Dam-dependent regulation of hilD expression is not transcriptional : ( i ) a transcriptional hilDTlac fusion is expressed at similar levels in Dam1 and Dam hosts ( Figure 4 ) ; ( ii ) reduced levels of both hilD mRNA and HilD protein are , however , found in Dam mutants ( Figures 2 and 4 ) ; ( iii ) reduced amounts of hilD mRNA are found in a Dam mutants when the hilD gene is expressed from a heterologous promoter ( Figure 5 ) ; ( iv ) SPI-1 remains under Dam-dependent control when hilD transcription is activated by tetracycline ( Figure 5 ) ; and ( v ) lack of DNA adenine methylation results in hilD mRNA instability ( Figure 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20008574" target="_blank">20008574</a>]<br></li>
<li>It also demonstrates BarA/SirA and EnvZ/OmpR control of hilA through HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17208038" target="_blank">17208038</a>]<br></li>
<li>METABOLIC ADAPTATION OF SALMONELLA DURING STRESS CONDITIONS Stress factors linking virulence and metabolism When Salmonella enters into an intestinal epithelial cell , environmental factors such as high osmolarity and neutral pH lead to an activation of HilD , which in turn induces HilA and invF gene expression ( Altier , 2005 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25688337" target="_blank">25688337</a>]<br></li>
<li>HilA is , in turn , induced by HilD and HilC and regulates InvF , which induces expression of secreted effector proteins located in SPI-1 ( 26 , 27 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22004521" target="_blank">22004521</a>]<br></li>
<li>The transcriptional activators HilD and HilC play an important role in controlling hilA expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25547794" target="_blank">25547794</a>]<br></li>
<li>In this model , HilD or HilC expression and activity would be induced in vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>We have three working hypotheses , which are not mutually exclusive : ( i ) RtsA , HilC , and HilD are active under different conditions or induce expression of hilA in response to different environmental cues .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923082" target="_blank">12923082</a>]<br></li>
<li>Still , no one has reported a condition in which wild-type S. typhimurium represses hilA expression while simultaneously activating expression of invF and invF-dependent genes through HilC or HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10844688" target="_blank">10844688</a>]<br></li>
<li>It has also been shown that both HilD and HilC can independently induce expression of a hilA-lac fusion in E. coli ( 72 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923082" target="_blank">12923082</a>]<br></li>
<li>The in vitro data suggest that HilC , HilD and RtsA act in concert to induce expression of hilA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16045614" target="_blank">16045614</a>]<br></li>
<li>hilA expression is induced by three transcriptional activators , HilC , HilD and RtsA [ 70 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25375226" target="_blank">25375226</a>]<br></li>
<li>Expression of hilA itself is activated by three AraC family regulators , HilC , HilD , and RtsA ( 25 , 78 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18160484" target="_blank">18160484</a>]<br></li>
<li>Expression of hilA is directly controlled by three AraC-like activators : HilC , HilD , and RtsA ( 18 , 42 , 47 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
<li>HilD , HilC , and RtsA are each capable of activating hilA transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
<li>HilC and HilD respond to different environmental conditions , but either alone is capable of activating hilA expression ( Lucas and Lee , 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21570144" target="_blank">21570144</a>]<br></li>
<li>HilD is a dominant regulator of hilA transcription , while HilC and RtsA amplify hilA gene expression ( 12 , 13 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24488311" target="_blank">24488311</a>]<br></li>
<li>HilD is a dominant regulator of hilA transcription , and HilC and RtsA amplify its expression [ 27 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28329249" target="_blank">28329249</a>]<br></li>
<li>Transcription of invF is known to be regulated by HilA , HilC , and HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923082" target="_blank">12923082</a>]<br></li>
<li>The expression of these sRNAs is controlled by diverse transcription factors , i.e. sE , HilD , Crp and OmpR , and activated by a speciﬁc stress or growth condition .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19007416" target="_blank">19007416</a>]<br></li>
<li>The transcription of hilA is regulated by HilD , an important activator that controls cross-talk between SPI1 and SPI2 expression [ 55,58 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26561851" target="_blank">26561851</a>]<br></li>
<li>As above , absence of sipB expression in both HilD Dam1 and HilD Dam hosts prevented any conclusion about Dam methylation dependence ( Figure 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20008574" target="_blank">20008574</a>]<br></li>
<li>The results described above , indicating that SirA/BarA and CsrB/C activate expression of hilA and ssrAB through HilD , also suggested that CsrA could directly repress hilD expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21518393" target="_blank">21518393</a>]<br></li>
<li>Therefore , our model predicts that SirA induces expression of hilA ( and invF ) via HilD ( Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16045614" target="_blank">16045614</a>]<br></li>
<li>This suggests that SirA regulates expression of rtsAB and hilC via HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16045614" target="_blank">16045614</a>]<br></li>
<li>Our data suggest that SirA induces expression of hilC and rtsA via HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16045614" target="_blank">16045614</a>]<br></li>
<li>The genes slrP and dsbA are also induced by HilC , HilD and RtsA independently of both HilA and InvF [ 25,28 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17208038" target="_blank">17208038</a>]<br></li>
<li>HilD , HilC , and RtsA all activate transcription of hilA , hilC , hilD , dsbA , invF , slrP , rtsA , and genes in SPI-4 ( 6 , 15 , 32 , 33 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24375101" target="_blank">24375101</a>]<br></li>
<li>It has been suggested that HilC and HilD may be capable of inducing expression of one another as well as inducing their own expression ( Lucas and Lee , 2001 ; Olekhnovich and Kadner , 2002 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16045614" target="_blank">16045614</a>]<br></li>
<li>As stated above , it is known that HilC and HilD can activate the expression of hilC and hilD independently of each other as well as inducing their own promoters ( 40 , 47 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18245288" target="_blank">18245288</a>]<br></li>
<li>HilC and HilD are also capable of activating hilC and hilD expression independently of each other , and they act in a complex regulatory loop to control hilA expression ( 13 , 50 , 53 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18245288" target="_blank">18245288</a>]<br></li>
				<label class="TU_collapse_similarity" for="tu_similarity_212">
HilC , HilD , and RtsA are each capable of activating expression of hilC , hilD , and rtsA , creating a complex regulatory loop that controls hilA expression ( Fig. 1 ) ( 15 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17951383" target="_blank">17951383</a>]<br></label>
<input id="tu_similarity_212", type="checkbox">
<div class="TU_similar">
<li>HilC , HilD , and RtsA are all also capable of activating expression of hilC , hilD , and rtsA independently of each other , constituting a complex feed-forward regulatory loop that controls hilA expression ( 13 , 14 , 15 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18245288" target="_blank">18245288</a>]<br></li></div>
				<label class="TU_collapse_similarity" for="tu_similarity_214">
We demonstrate that HilC , HilD and RtsA are each capable of independently inducing expression of the hilC , hilD and rtsA genes , and that each can independently activate hilA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16045614" target="_blank">16045614</a>]<br></label>
<input id="tu_similarity_214", type="checkbox">
<div class="TU_similar">
<li>We show that HilD , HilC and RtsA can each independently activate expression of the hilD , hilC , rtsAB and hilA genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16045614" target="_blank">16045614</a>]<br></li></div>
<li>We demonstrate that HilC , HilD and RtsA are each capable of inducing expression of hilC , hilD and rtsA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16045614" target="_blank">16045614</a>]<br></li>
<li>These data show that HilC , HilD and RtsA are each capable of independently inducing expression of hilC , hilD and rtsA , consistent with our model that HilC , HilD and RtsA constitute a feed forward regulatory loop ( Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16045614" target="_blank">16045614</a>]<br></li>
<li>RtsA , HilC and HilD also induced expression of hilC , hilD and rtsA ( Fig. 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16045614" target="_blank">16045614</a>]<br></li>
<li>RtsA , HilC and HilD induced expression of hilC approximately threeto fourfold and hilD ~ 10 - to 12-fold .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16045614" target="_blank">16045614</a>]<br></li>
<li>HilD and HilC do not seem to regulate expression of hilD or prgH ( Olekhnovich and Kadner , 2002 ) ( Figs 1 and 2B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>Although HilD and HilC regulate two promoters upstream of hilC , they activate one and repress the other , such that overall HilD and HilC appear to have no effect on hilC expression ( Olekhnovich and Kadner , 2002 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>It is known that HilD can also control gene expression through HilC , SprB , RtsA and SsrA/B4 ,14 , and possibly , as suggested by our results , through SinR and SlyA ; thus , HilD could act on SL1896 through any of these regulators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27886269" target="_blank">27886269</a>]<br></li>
<li>In fact , there is evidence that SipA and SipC HilD and HilC directly activate invF expression 723 can be secreted independently of SPI1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>Under inducing conditions , activation of BarA/SirA initiates a regulatory cascade involving HilD , HilC , and HilA , which leads both directly and indirectly to the expression of SPI1 and SPI4 genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18160484" target="_blank">18160484</a>]<br></li>
<li>We propose that HilD and HilC can independently activate invF expression and InvF production , which directly and indirectly increases transcription of the sipA and sipC genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>It has been proposed that HilD and HilC indirectly co-ordinate expression of the SPI1 TTS apparatus and its secreted effector proteins in vivo by directly affecting hilA transcription ( Schechter et al. , 1999 ; Schechter and Lee , 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>As HilC and HilD activate the transcription from both HilA-dependent and - independent promoters , it is possible that Lon is involved in regulation of SPI1 gene expression by a pathway that depends on HilC and/or HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15661008" target="_blank">15661008</a>]<br></li>
<li>First , HilC and RtsA amplify HilA and HilD expression , in the sense that HilA and HilD expression is reduced the absence of HilC and RtsA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20686667" target="_blank">20686667</a>]<br></li>
<li>Gre factors affect SPI-1 gene expression by modulating hilD expression The transcriptional expression of hilA is tightly modulated by three AraC activators , HilD , HilC and RtsA , which form an auto-inducing regulatory loop where each protein can activate its own expression , as well as expression of the other two regulators [ 15 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28426789" target="_blank">28426789</a>]<br></li>
<li>HilC and HilD , two SPI1-encoded AraC/XylS family members , have been called derepressors and are thought to counteract the repression of hilA by binding to sites within the upstream repressing sequence ( URS ) ( 24 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12933899" target="_blank">12933899</a>]<br></li>
<li>Only when the activating signals are sufficiently strong is HilD expressed at a high enough level to overcome sequestration by HilE and activate the expression of the other SPI1 regulators -- HilC , RtsA , and HilA-and also further induce its own expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20686667" target="_blank">20686667</a>]<br></li>
<li>The AraC/XylS-type regulators ( HilC and HilD ) positively modulate the expression of hilA as transcription activators binding to the upstream repression sites ( 6 , 8,18 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24780330" target="_blank">24780330</a>]<br></li>
<li>Conversely , HilC and HilD act as transcription activators to induce the expression of hilA by binding to the upstream repression sites ( Ellermeier et al. , 2005 ; Lucas and Lee , 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27565525" target="_blank">27565525</a>]<br></li>
<li>Due to the counteracting nature of the repressing URS and derepressing HilC and HilD proteins that directly control hilA expression , the hns-1 : : kan mutation may increase hilA expression under the low-osmolarity conditions ( i ) by decreasing its repression and/or ( ii ) by increasing its derepression by HilC or HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12933899" target="_blank">12933899</a>]<br></li>
<li>HilC and HilD bind to the hilA promoter region , and it is believed that this binding induces the expression of hilA ( 64 , 71 , 72 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923082" target="_blank">12923082</a>]<br></li>
<li>Fur directly binds to the hilD promoter and enhances its own expression ( 15 ) , and HilD functions as an activator of SPI-1 expression ( 17 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24643535" target="_blank">24643535</a>]<br></li>
<li>According to our model , these activating signals , both intracellular cellular and environmental , initiate SPI1 gene expression by inducing the expression and activation of HilD through still unknown mechanisms .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20686667" target="_blank">20686667</a>]<br></li>
<li>In this scheme , HilD and HilC indirectly activate expression of the SPI1 TTS apparatus and its secreted substrates by affecting hilA expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>More recently , H-NS has been shown to repress the expression of HilA , HilD , HilC , and RtsA under noninducing conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23554419" target="_blank">23554419</a>]<br></li>
<li>Our data suggest that HilC , HilD , and RtsA function in similar manners by binding to the same fragment of hilA to induce its expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923082" target="_blank">12923082</a>]<br></li>
<li>Recently , two SPI1-encoded factors , HilC and HilD , which derepress hilA expression , were identiﬁed and shown to be members of the AraC/XylS family of transcriptional regulators and to independently bind to these upstream repressing sequences to counteract the repression of hilA expression ( 32 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11741864" target="_blank">11741864</a>]<br></li>
<li>In addition to HilC and HilD , RtsA , which belongs to the AraC/XylS family of transcriptional regulators , has been shown to activate expression of SPI1 genes by binding upstream of hilA to induce its expression ( 13 , 14 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18245288" target="_blank">18245288</a>]<br></li>
<li>HilC and HilD bind directly upstream of the master regulator gene hilA to induce its expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18245288" target="_blank">18245288</a>]<br></li>
				<label class="TU_collapse_similarity" for="tu_similarity_244">
Both HilC and HilD activate expression of SPI-1 genes by binding upstream of the master regulatory gene hilA to induce its expression [ 48 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22291596" target="_blank">22291596</a>]<br></label>
<input id="tu_similarity_244", type="checkbox">
<div class="TU_similar">
<li>Similar to HilC and HilD , RtsA activates the expression of SPI1 genes by binding upstream of the master regulatory gene , hilA , to induce its expression ( 9,17 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17329372" target="_blank">17329372</a>]<br></li>
<li>Like HilC and HilD , RtsA activates expression of SPI1 genes by binding upstream of the master regulatory gene hilA to induce its expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16045614" target="_blank">16045614</a>]<br></li></div>
<li>Two SPI1-encoded regulators of hilA expression are the AraC/XylS family members , HilC ( also called SirC or SprA ) and HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12107132" target="_blank">12107132</a>]<br></li>
<li>Two SPI1-encoded proteins , HilC ( SirC or SprA ) and HilD , both members of the AraC/XylS family of transcriptional regulators , induce expression of hilA ( 26 , 67 , 71 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923082" target="_blank">12923082</a>]<br></li>
<li>Two homologous proteins , HilC and HilD , both members of the AraC/XylS family of transcriptional regulators , induce hilA expression ( 12 , 51 , 53 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18245288" target="_blank">18245288</a>]<br></li>
<li>Transcription of hilA is directly activated by HilC and HilD , both members of the AraC/XylS family of transcriptional regulators ( Schechter and Lee 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25991823" target="_blank">25991823</a>]<br></li>
<li>Three homologous proteins , 8,13,14 HilC , HilD , and RtsA , which belong to the AraC/XylS family of transcription factors can activate expression of hilA and some of its target genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16443238" target="_blank">16443238</a>]<br></li>
<li>HilD , HilC , and RtsA are members of the AraC family of transcription factors .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24375101" target="_blank">24375101</a>]<br></li>
<li>SirA activates the expression of SsrB and HilA through HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21518393" target="_blank">21518393</a>]<br></li>
<li>HilC and HilD also induce expression of the inv/spa operon independent of HilA ( 2 , 26 , 67 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923082" target="_blank">12923082</a>]<br></li>
<li>Two members of the AraC/XylS family of transcription activators , HilC and HilD , can independently derepress hilA expression by binding directly to the upstream repressing sequence and counteracting its repressing effect in vitro and in vivo ( 24 , 30 , 31 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16391141" target="_blank">16391141</a>]<br></li>
<li>In contrast , disruption of the downstream -10 hexamer sequence dramatically reduced the ability of HilD and HilC to activate invF expression ( Fig. 6 , black and striped bars , pSA14 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>RtsA belongs to the AraC/XylS family of regulators , and can directly bind the hilA promoter and induce the hilA expression independent of HilC and HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15661008" target="_blank">15661008</a>]<br></li>
<li>Two additional regulators , HilC and HilD , activate directly the expression of hilA and the invF operon , independently of HilA ( Dieye et al. , 2007 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27404739" target="_blank">27404739</a>]<br></li>
<li>Although HilA is the major regulator , two other regulators encoded within SPI1 , HilC and HilD , can directly increase the expression of both hilA and invF independently of hilA ( 3 , 53 , 56 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14645287" target="_blank">14645287</a>]<br></li>
<li>Activators HilD and HilC are not essential for rtsA expression in the absence of repressors .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17675384" target="_blank">17675384</a>]<br></li>
<li>The data in Fig. 2 demonstrate that RtsA , HilC and HilD are able to induce expression of hilA in the absence of the other regulators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16045614" target="_blank">16045614</a>]<br></li>
<li>The ability of SirA to induce expression of rtsA , hilC , hilD , hilA and invF in the absence of RtsA , HilC , and/or HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16045614" target="_blank">16045614</a>]<br></li>
<li>We wanted to determine if HilC , HilD and RtsA could induce expression of hilC , hilD , rtsA and hilA in the absence of the other regulators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16045614" target="_blank">16045614</a>]<br></li>
<li>Therefore , we examined the expression of hilA , rtsA , hilC and hilD transcriptional fusions in the presence and absence of RtsA , HilC , and/or HilD ( Fig. 7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16045614" target="_blank">16045614</a>]<br></li>
<li>The effect of SPI1 inducing and repressing conditions on the expression of rtsA , hilC , hilD and hilA in the absence of in RtsA , HilC , and/or HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16045614" target="_blank">16045614</a>]<br></li>
<li>HilC , HilD and RtsA , in addition to regulation of hilA , are all capable of inducing expression of the inv/spa operon -- and by read-through , the sic/sip operon -- in a fashion independent of HilA [ 15,16,25,27 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17208038" target="_blank">17208038</a>]<br></li>
<li>RtsA induces expression of hilA in the absence of both HilC and HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923082" target="_blank">12923082</a>]<br></li>
<li>A subset of SPI1 genes is induced by HilD overexpression in the absence of hilA Our results suggested that HilA and HilD have independent effects on expression of invF , sipA and sipC in S. typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>If all environmental conditions and mutations affecting hilA and/or invasion gene expression act through HilD , it might favour a model in which HilD is the central regulator of invasion genes instead of HilA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10844688" target="_blank">10844688</a>]<br></li>
<li>Figure 2B shows that in the hilA hilD mutant , HilD overproduction induces expression of invF , sipA and sipC , whereas expression of prgH , prgK and orgA remains unaffected .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>Our current studies demonstrate that HilD and HilC can bind upstream of invF and directly activate its expression in the absence of hilA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>( C ) In the presence of bile , HilD is more stable , leading to enhanced expression of hilD , hilA , and thus SPI-1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29229736" target="_blank">29229736</a>]<br></li>
<li>First , Lon is induced to control turnover of HilD and HilC and hence limit the expression of SPI-1 genes , which is of particular relevance after invasion of epithelial cells .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23554419" target="_blank">23554419</a>]<br></li>
<li>( iii ) RtsA , HilC , and HilD are required for signal ampliﬁcation such that once one of these genes is turned on , expression of the others is also induced , resulting in greater induction of hilA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923082" target="_blank">12923082</a>]<br></li>
<li>LoiA activates expression of hilA gene through activating hilD As the expression of hilA is directly controlled by HilD , HilC and RtsA , we tested whether LoiA regulates HilA through any of these three regulators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28575106" target="_blank">28575106</a>]<br></li>
<li>Thus , FliZ-mediated induction of hilA expression is through HilD , while RtsA and HilC act as ampliﬁers of the signal .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17951383" target="_blank">17951383</a>]<br></li>
<li>The expression of hilA , the master regulator for SPI1 , is controlled by HilD , HilC , and RtsA , which act in a complex feed-forward loop .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20889744" target="_blank">20889744</a>]<br></li>
<li>Expression of hilA is coordinately controlled by the HilD / HilC/RtsA complex , and HilD plays a major role in the regulation of hilA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26810370" target="_blank">26810370</a>]<br></li>
				<label class="TU_collapse_similarity" for="tu_similarity_279">
Expression of hilA is controlled by the combined action of three AraC-like transcriptional activators : HilC , HilD , and RtsA ( 14 -- 16 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23504014" target="_blank">23504014</a>]<br></label>
<input id="tu_similarity_279", type="checkbox">
<div class="TU_similar">
<li>Expression of hilA is controlled by the combined action of three AraC-like transcriptional activators : HilC , HilD , and RtsA ( 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25182488" target="_blank">25182488</a>]<br></li></div>
<li>Expression of hilA is under the control of many environmental conditions and three transcriptional activators : HilC , HilD , and RtsA ( Ellermeier et al. , 2005 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27704705" target="_blank">27704705</a>]<br></li>
<li>In serovar Typhimurium , dsbA is controlled by the hilA regulators RtsA , HilD , and HilC , and expression of DsbA under SPI1-inducing conditions is independent of CpxRA ( 18 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17951383" target="_blank">17951383</a>]<br></li>
<li>Expression of hilA is directly controlled by three AraC-like regulators , HilD , HilC , and RtsA ( 5 , 6 ) , each of which is independently capable of inducing expression of the hilD , hilC , and rtsA genes , as well as hilA , forming a complex feed-forward regulatory loop to control SPI1 expression ( 7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26884427" target="_blank">26884427</a>]<br></li>
				<label class="TU_collapse_similarity" for="tu_similarity_284">
The expression of hilA , encoding the transcriptional activator of the T3SS structural genes , is directly controlled by three AraC-like regulators , HilD , HilC , and RtsA , each of which can activate hilD , hilC , rtsA , and hilA genes , forming a complex feed-forward regulatory loop .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26884427" target="_blank">26884427</a>]<br></label>
<input id="tu_similarity_284", type="checkbox">
<div class="TU_similar">
<li>The expression of hilA , encoding the transcriptional activator of the SPI1 structural genes , is directly controlled by three AraC-like regulators , HilD , HilC , and RtsA , each of which can activate the hilD , hilC , rtsA , and hilA genes , forming a complex feed-forward regulatory loop .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22021388" target="_blank">22021388</a>]<br></li></div>
<li>The mechanism by which HilC , HilD and RtsA activate expression of hilA ( and possibly of each other ) apparently involves counteracting silencing by the nucleoid protein Hns [ 30,31 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17208038" target="_blank">17208038</a>]<br></li>
<li>Acetylation of K297 Regulating HilD DNA-Binding Ability and Stability At the transcriptional level , the expression of hilD is positively autoregulated and modulated by a feed-forward loop involving HilD itself and regulators HilC and RtsA [ 3 , 28 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28329249" target="_blank">28329249</a>]<br></li>
<li>Based on our genetic analyses , and taking into account the published results outlined above , we present a model in which expression of hilA is controlled by the combined action of HilC , HilD and RtsA , each of which can independently bind upstream of hilA to induce its expression ( Schechter and Lee , 2001 ; Olekhnovich and Kadner , 2002 ; Ellermeier and Slauch , 2003 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16045614" target="_blank">16045614</a>]<br></li>
<li>When the individual proteins HilD , HilC , and RtsA are overexpressed from plasmids , they are able to induce expression of the hilA , invF , hilD , hilC , rtsA , slrP , and dsbA genes , which are required for invasion ( 13 -- 15 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17675384" target="_blank">17675384</a>]<br></li>
<li>Thus , if the abilities of these proteins to induce dsbA and hilA are compared , it becomes apparent that RtsA and HilD are better able to induce expression of dsbA than HilC is ( Table 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14679226" target="_blank">14679226</a>]<br></li>
<li>We believe that the inductions of RtsA , HilC , and HilD are self-reinforcing events leading to rapid and fully induced expression of hilA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923082" target="_blank">12923082</a>]<br></li>
<li>Furthermore , HilD and HilC bind the same DNA sites upstream of hilD and hilC ( 38 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24375101" target="_blank">24375101</a>]<br></li>
<li>These results indicate that Lon is involved in the autoregulation of hilC and hilD transcription by modulating amounts of HilC and HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15661008" target="_blank">15661008</a>]<br></li>
<li>To test whether the RtsA , HilC , and HilD activators of hilA are also regulated by these proteins , we investigated the transcription of rtsA-lac , hilC-lac , and hilD-lac chromosomal fusions ( 13 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17675384" target="_blank">17675384</a>]<br></li>
<li>These transcription factors include HilA , HilD , and HilC ( all three are responsible for activation of SPI1-encoded Type 3 Secretion System ( T3SS ) ) [ 67,68 ] ; SprB ( responsible for activation of SPI4 gene expression ) [ 47 ] ; and SsrAB ( two-component system responsible for activation of SPI2-encoded T3SS ) [ 72 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23040276" target="_blank">23040276</a>]<br></li>
<li>However , transcription of rtsA , hilC and hilD was induced in response to low oxygen independent of RtsA , HilC , or HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16045614" target="_blank">16045614</a>]<br></li>
<li>The EnvZ / OmpR and BarA/SirA two-component regulatory systems induce expression of hilA ( 34 , 49 ) , again in a manner dependent on HilD ( 16 ) but independent of HilE ( unpublished data ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
<li>Loss of hns , however , does not relieve repression of hilA by oxygen ( as opposed to osmolarity ) and expression of hilA in an hns mutant still requires HilD under inducing ( high osmolarity , low oxygen ) growth conditions ( L. Schechter and C. Lee , unpublished observations ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11755416" target="_blank">11755416</a>]<br></li>
<li>This allowed us to induce rtsA , with concom-itant induction of hilA ( and hilC ) expression , independently of HilD ( Figure 2B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22021388" target="_blank">22021388</a>]<br></li>
<li>In contrast , lack of HilD reduces hilA expression .100 - fold ( Lopez-Garrido and Casadesus 2010 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25991823" target="_blank">25991823</a>]<br></li>
<li>Effect of upstream DNA on stimulation of hilA transcription by HilC and HilD proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15126488" target="_blank">15126488</a>]<br></li>
<li>Effect of HilD , HilC and HilA on invF expression from pSA7 and mutant derivatives of pSA7 in a Dspi1 mutant .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>( ii ) RtsA , HilC , and HilD differentially regulate expression of other genes independent of their effects on hilA expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923082" target="_blank">12923082</a>]<br></li>
<li>A second possibility is that RtsA induces expression of hilA independent of HilC and HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923082" target="_blank">12923082</a>]<br></li>
<li>Indeed , it appears that RtsA , HilC and HilD are all capable of inducing expression of hilA , hilC , hilD , and invF but that RtsA is preferentially capable of inducing expression of slrP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923082" target="_blank">12923082</a>]<br></li>
<li>The expression of hilA is controlled by a complex feed-forward regulatory loop consisting of HilD , HilC , and RtsA , each of which can independently activate hilA expression ( 13 ) ( Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20889744" target="_blank">20889744</a>]<br></li>
<li>Regulation of hilA expression by RtsA , HilC , and HilD is an example of a feedforward loop .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923082" target="_blank">12923082</a>]<br></li>
<li>RtsA increases invasion gene expression by increasing transcription of hilA , which is independent of HilC and HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15765064" target="_blank">15765064</a>]<br></li>
<li>RtsA can induce expression of invF independently of HilA , HilC , and HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923082" target="_blank">12923082</a>]<br></li>
<li>This suggests that RtsA can induce expression of invF independent of HilA , similar to what has been observed for both HilC and HilD ( 2 , 26 , 67 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923082" target="_blank">12923082</a>]<br></li>
<li>Regulation of HilC and HilD is complex and both are capable of inducing expression of one another as well as their own expression in response to various environmental stimuli ( Ellermeier et al. , 2005 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21168230" target="_blank">21168230</a>]<br></li>
<li>We wanted to determine if HilC and HilD are also capable of inducing expression of dsbA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14679226" target="_blank">14679226</a>]<br></li>
<li>HilD , HilC , and RtsA are capable of self-activat-ing expression in addition to activating expression of one another and of hilA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24375101" target="_blank">24375101</a>]<br></li>
<li>Effect of invF , hilA , and hilC-D mutations on RtsA induction of slrP ( A ) and ability of HilC , HilD , and RtsA to induce expression of hilA ( B ) and slrP ( C ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923082" target="_blank">12923082</a>]<br></li>
<li>Interestingly , RtsA is capable of inducing dsbA 10-fold , while HilC and HilD induce expression of dsbA approximately two - and fourfold , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14679226" target="_blank">14679226</a>]<br></li>
<li>HilC , HilD , and RtsA differentially induce expression of dsbA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14679226" target="_blank">14679226</a>]<br></li>
<li>Interestingly , RtsA induces expression of slrP 45-fold , while HilD and HilC are able to induce expression of slrP only four - and ﬁvefold , respectively ( Fig. 7B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923082" target="_blank">12923082</a>]<br></li>
<li>Interestingly , overexpression of HilD still activates expression of invF from pSA14 and pCL463 by approximately 10-fold .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>Regulation of gene expression by RtsA , HilC and HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16045614" target="_blank">16045614</a>]<br></li>
<li>Regulation of slrP expression by PhoP , Lon , LeuO , and HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25182488" target="_blank">25182488</a>]<br></li>
<li>Ellermeier and Slauch initially reported that Fur activates hilA transcription in a HilD-dependent manner , and since Fur is thought to be capable of acting only as a repressor , it should repress a repressor that controls HilD ( 322 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23554419" target="_blank">23554419</a>]<br></li>
<li>To determine whether HilE , Pag , or Hha affects repression , rather than derepression , of hilA , the effect that hilE , pag , and hha mutations have on hilA expression should be examined under repressing environmental conditions in strains lacking HilC and HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12933899" target="_blank">12933899</a>]<br></li>
<li>However , as a result of the speciﬁc environmental conditions , hilA expression may be repressed in such a way that HilD and HilC can not derepress it , or HilA activity may be downregulated post-transcriptionally .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>We show that , under SPI1-inducing conditions , LeuO and Lon are indirect regulators of the slrP expression that act through HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25182488" target="_blank">25182488</a>]<br></li>
<li>We found that HilC and HilD , which are positive regulators of hilA expression , accumulate in Lon-depleted cells , and that the enhancement of SPI1 expression that occurs in a lon-disrupted mutant is not observed in the lon hilC hilD triple null mutant .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15661008" target="_blank">15661008</a>]<br></li>
<li>We conclude that HilD and HilC have a second role in SPI1 gene regulation that is separate from their role in co-ordinating expression of the SPI1 TTSS through hilA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>All indications are that , under the conditions that we are examining , dsbA is primarily controlled by RtsA , HilC , and HilD ( 18 ) ; CpxRA , which controls dsbA in E. coli , seems irrelevant to either expression of dsbA or assembly and function of the SPI1 T3SS or ﬂagella in serovar Typhimurium ( Fig. 2C ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17951383" target="_blank">17951383</a>]<br></li>
<li>We speculate that mcpC , sinR , and are regulatory targets of HilD and possibly also of HilC and RtsA , but require additional transcription factors that are not present or not expressed in E. coli under the conditions used in our expression assays .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24375101" target="_blank">24375101</a>]<br></li>
<li>HilD can activate the ssrAB operon by speciﬁcally interacting with its regulatory region to upregulate SPI-2 [ 41 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26810370" target="_blank">26810370</a>]<br></li>
<li>For example , OmpR promotes SPI-2 expression by directly binding to both the ssrA and ssrB promoters [ 16 ] ; PhoP activates SPI-2 transcription by directly binding to the ssrB promoter and controls SsrA levels post-transcriptionally [ 15 ] ; SlyA , HilD and Fis proteins can bind to the ssrA promoter to regulate SPI-2 genes [ 19,20,22 ] , and PmrA represses SPI-2 gene transcription through directly binding to the ssrB promoter [ 18 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29751061" target="_blank">29751061</a>]<br></li>
<li>Another SPI-1 encoded sRNA , InvR was found to be activated by virulence regulatory protein HilD under invasion inducing conditions and repressed the synthesis of a major Salmonella porin OmpD post-transcriptionally [ 21,22 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28302471" target="_blank">28302471</a>]<br></li>
<li>LB LB 0.3 M NaCl L The invR gene belongs to the SPI-1 regulon and is activated by HilD Next , we asked whether invR transcription was under the control of SPI-1-encoded transcription factors .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17971080" target="_blank">17971080</a>]<br></li>
<li>Under inducing conditions , HilD also activates a second promoter far upstream of the invF translation start site , termed PinvF-2 ( dotted arrow ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11466291" target="_blank">11466291</a>]<br></li>
				<label class="TU_collapse_similarity" for="tu_similarity_334">
The promoter of the E. coli lacZ gene , which is not recognized by HilD , was included as negative control .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17971080" target="_blank">17971080</a>]<br></label>
<input id="tu_similarity_334", type="checkbox">
<div class="TU_similar">
<li>The promoter of the E. coli lacZ gene ( PlacZ * ) not recognized by HilD served as negative control .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17971080" target="_blank">17971080</a>]<br></li></div>
<li>Additionally , a DNA fragment containing the intergenic region upstream of ppk , a gene not regulated by HilD , or sigD , a gene not directly regulated by HilD , was included in the binding reactions as an internal negative control .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27886269" target="_blank">27886269</a>]<br></li>
				<label class="TU_collapse_similarity" for="tu_similarity_337">
As a negative control in these binding assays , we used a fragment containing the regulatory region of sigD , a gene that is not directly regulated by HilD or H-NS ( 21 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25135218" target="_blank">25135218</a>]<br></label>
<input id="tu_similarity_337", type="checkbox">
<div class="TU_similar">
<li>A fragment containing the regulatory region of sigD , a gene that is not regulated directly by HilD or H-NS , was used as a negative control .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25135218" target="_blank">25135218</a>]<br></li></div>
<li>SPI1 expression is activated when HilD reaches the threshold required to autoactivate the hilD promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22021388" target="_blank">22021388</a>]<br></li>
<li>The system is complicated by the fact that HilD is autoregulated and hilD transcription is normally induced upon activation of the regulatory circuit ( 13 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20889744" target="_blank">20889744</a>]<br></li>
<li>Furthermore , synthesis of the SPI-1 transcription factor HilD is negatively regulated by StdEF , two proteins encoded by the std ﬁmbrial operon [ 71 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29369799" target="_blank">29369799</a>]<br></li>
<li>InvR ( located within SPI1 ) is transcriptionally activated by the primary SPI1 transcription factor , HilD [ 75 ] , confirming the value of correlative analysis for identifying genuine regulatory interactions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27564394" target="_blank">27564394</a>]<br></li>
<li>However , how HilD counteracts the H-NS-mediated repression of ssrAB , or even of other target genes , had not yet been determined .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25135218" target="_blank">25135218</a>]<br></li>
				<label class="TU_collapse_similarity" for="tu_similarity_344">
HilE does not affect HilD dimerization , nor does it control the stability of the HilD protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29378886" target="_blank">29378886</a>]<br></label>
<input id="tu_similarity_344", type="checkbox">
<div class="TU_similar">
<li>HilE does not block dimerization , nor does it signiﬁcantly affect the stability of HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29378886" target="_blank">29378886</a>]<br></li></div>
<li>The immunoblotting results in Fig. 7 show that HilD was detectable in the absence of IPTG ( isopropyl - - D-thiogalactopy-ranoside ) , a condition in which hilD was expressed by read-through from the PA1lacO-1 promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18245288" target="_blank">18245288</a>]<br></li>
<li>hilD expression from the strain complemented with HilD was signiﬁcantly higher than hilD expression from both the reporter strain alone and the reporter carrying the empty vector ( Fig. 5 ) , indicating that in S. Typhi HilD positively regulates its own transcription , either directly or indirectly .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29229736" target="_blank">29229736</a>]<br></li>
<li>Consistent with the notion that FliZ activates T3SS-1 expression through HilD , transcription of hilD was induced in a tviA defective strain ( Fig. 1A , cluster 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19703107" target="_blank">19703107</a>]<br></li>
<li>Double or triple negative regulator mutants that included the pag : : Tn5 mutation expressed 40 Miller units of hilA : : Tn5lacZY expression when HilD was absent ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12511499" target="_blank">12511499</a>]<br></li>
<li>Using two-hybrid analysis , it has been shown that HilE interacts with HilD , which suggests that HilE represses hilA expression by inhibiting the activity of HilD through a protein -- protein interaction ( 19,20 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17329372" target="_blank">17329372</a>]<br></li>
<li>Using two-hybrid analysis it was determined that HilE interacts with HilD suggesting that HilE represses expression of hilA by inhib-iting the activity of HilD ( Baxter et al. , 2003 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16045614" target="_blank">16045614</a>]<br></li>
<li>HilD not only can induce the expression of hilA to activate SPI-1 , but it also promotes SPI-1 expression independent of HilA [ 24 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28329249" target="_blank">28329249</a>]<br></li>
<li>HilD also activates expression of the SPI-1-borne invR gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19007416" target="_blank">19007416</a>]<br></li>
<li>Then , the expression of HilD-Myc from plasmid pBAD-HilD was induced by adding 0.05 % L-arabinose for 45 min .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26300871" target="_blank">26300871</a>]<br></li>
<li>As a control , the expression of a cat transcriptional fusion of invF , which is positively regulated by HilD through HilA , was also assessed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27886269" target="_blank">27886269</a>]<br></li>
<li>This suggests that HilD induces the expression of ssrAB also by counteracting H-NS-me-diated repression of its promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25135218" target="_blank">25135218</a>]<br></li>
<li>As shown in Fig. 8 , the increased effect of Lon depletion on hilA expression was abolished by depletion of both HilC and HilD , suggesting that both proteins are also involved in regulation of SPI1 by Lon under SPI1-repressing conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15661008" target="_blank">15661008</a>]<br></li>
<li>In this case overexpression of HilC or HilD may increase hilA expression by overcoming the enhanced repressing mechanism in these mutants .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11755416" target="_blank">11755416</a>]<br></li>
<li>Under the conditions used , ssrAB expression depended strongly on HilD , since there was no expression in a hilD-deficient strain regardless of the presence or absence of Gre factors .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28426789" target="_blank">28426789</a>]<br></li>
<li>But under these conditions , deletion of hilE had no effect on tetRA -- rtsA-driven hilA expression in the absence of HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22021388" target="_blank">22021388</a>]<br></li>
<li>Nonetheless , since the hup mutations do not trigger impaired hilA transcription in the absence of HilD or affect hilD transcription , it has been deduced that HU transcriptionally activates HilA expression by posttranscriptional modulation of HilD ( 291 , 309 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23554419" target="_blank">23554419</a>]<br></li>
<li>HilE controls an ectopically expressed HilD but does not affect expression of hilD-lacZ transcriptional or translational fusions in the absence of HilD ( 20 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29378886" target="_blank">29378886</a>]<br></li>
<li>The requirement of HilD for hilA expression was further studied by monitoring HilA-FLAG expression in the presence or absence of HilD ( Fig 4C ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28426789" target="_blank">28426789</a>]<br></li>
<li>These data suggest that the L289F mutation does not disrupt interaction with an activator of hilA expression whose loss can be compensated for by overexpression of HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12511499" target="_blank">12511499</a>]<br></li>
<li>We reasoned that , if HilD is a derepressor of hilA expression , then deleting negative regulators of the hilA gene would increase hilA expression in the absence of HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12511499" target="_blank">12511499</a>]<br></li>
<li>Thus , the phoQ24 effect on hilA expression is independent of HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22021388" target="_blank">22021388</a>]<br></li>
<li>We then asked if the inhibition of hilA expression by RcsCDB also works through HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17951383" target="_blank">17951383</a>]<br></li>
<li>These data are consistent with the hypothesis that the RcsCDB effect on hilA expression also works through HilD , at least partially by affecting expression of ﬂiZ .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17951383" target="_blank">17951383</a>]<br></li>
<li>As HilD has also been reported to positively regulate its own expression ( Olekhnovich and Kadner , 2002 ; Ellermeier et al. , 2005 ) , the expression of hilD was also tested in the hilD mutant .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21518393" target="_blank">21518393</a>]<br></li>
<li>HilD has been shown to regulate its own expression ( Ellermeier et al. , 2005 ) , and thus mutants unable to produce the protein exhibit reduced expression of the gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23289537" target="_blank">23289537</a>]<br></li>
<li>The addition of HilE to this system did not markedly alter the ability of the HilD - or HilC-LexADBD proteins to repress lacZ expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29378886" target="_blank">29378886</a>]<br></li>
<li>SprB has been shown to activate transcription of siiA ( 58 ) , although the same study showed no regulation of siiA by HilA or HilD , in contrast to our data .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24375101" target="_blank">24375101</a>]<br></li>
<li>This sRNA gene is coexpressed with other SPI-1 genes under conditions that favour host cell invasion , and directly controlled by HilD , the protein that acts at the top of the SPI-1 transcription factor cascade ( Fig. 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19007416" target="_blank">19007416</a>]<br></li>
<li>Thus , it might be deduced that once inside the SCV , PhoP activates the pag genes , which in turn reduce hilA transcription through HilD and eventually trigger repression of the prg genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23554419" target="_blank">23554419</a>]<br></li>
<li>HilD activates HilA , which in turn upregulates the expression of genes encoding the TTSS , such as proteins encoded by the prg-org and the inv-spa operons ( 6 , 17 , 18 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28439039" target="_blank">28439039</a>]<br></li>
<li>For instance , the H-NS modulator YmoA is preferentially degraded at 37 ◦ C by the ClpP and Lon proteases in Yersinia ( Jackson et al. , 2004 ) , and in Salmonella it has been shown that transcription of the hns gene -- which inﬂuences expression of the Spi-1-encoded invasion genes through the regulator HilD -- is repressed by the iron homeostasis regulator Fur ( Troxell et al. , 2011 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26441883" target="_blank">26441883</a>]<br></li>
<li>It is important to note that the E. coli MC4100 strain carries a frameshift mutation in the flhDC operon and thus it does not express the flagellar transcriptional regulator FlhDC45 ,46 ; HilD positively regulates the expression of FlhDC in S. Typhimurium30 ,35 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27886269" target="_blank">27886269</a>]<br></li>
<li>Furthermore , we demonstrate that HilD is required for the expression of the gtgE , phoH , sinR , lpxR , SL1896 and SL4247 genes , but not for the SL1028 , SL3812 and SL4433 genes , when S. Typhimurium SL1344 is grown in conditions that favor the expression of the SPI-1 genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27886269" target="_blank">27886269</a>]<br></li>
<li>To determine whether the expression of the novel genes found to be co-expressed with the SPI-1 genes is controlled by the major transcriptional regulators encoded in SPI-1 , HilD , HilA and InvF , transcriptional fusions of these genes to the cat gene were constructed in plasmid pKK232 -- 8 , an expression reporter system that we have successfully used in S. Typhimurium14 ,42 -- 44 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27886269" target="_blank">27886269</a>]<br></li>
<li>HilD is one of the transcriptional factors encoded in SPI-1 that positively controls the expression of other genes in the island ( 109 , 110 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25566242" target="_blank">25566242</a>]<br></li>
<li>These results suggest that CpxR represses hilA and thus the other SPI-1 genes by affecting the autoregulation of HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26300871" target="_blank">26300871</a>]<br></li>
<li>Growth phase determines the expression levels of several regulators ( e.g. , Fis is maximally expressed at 1 h of growth , and IHF peaks during the transition into the stationary phase [ 301 , 305 ] ) and inﬂuences its regulatory action ( e.g. , HilD induces SPI-2 genes during the late stationary phase [ 14 ] ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23554419" target="_blank">23554419</a>]<br></li>
<li>However , our results show that HilD does not control the expression of SL1896 through FlhDC , the master regulator of the flagellar/chemotaxis genes , neither through the SPI-1 regulators HilA and InvF ( Table 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27886269" target="_blank">27886269</a>]<br></li>
<li>Taken together , our results support that CpxR-P represses the expression of the SPI-1 and SPI-2 genes mainly by reducing the stability of HilD through the Lon protease .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26300871" target="_blank">26300871</a>]<br></li>
<li>Importantly , these regulators all converge at HilD , which induces SPI-1 genes that encode the secreted effectors and structural components of the TTSS directly by activating their transcription and indirectly through induction of the downstream transcriptional activator HilA ( 9 , 16 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27185788" target="_blank">27185788</a>]<br></li>
<li>Our results elucidate the mechanism by which HilD and H-NS regulate the expression of ssrAB and show for the ﬁrst time that HilD is able to displace H-NS from one of its target genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25135218" target="_blank">25135218</a>]<br></li>
<li>DISCUSSION In this study , we elucidated the mechanism by which HilD and H-NS regulate the expression of ssrAB and thus the SPI-2 genes ( Fig. 7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25135218" target="_blank">25135218</a>]<br></li>
<li>Our data indicate that CpxR-P decreases the stability of HilD , the regulator that is at the apex of a regulatory cascade controlling the expression of the SPI-1 genes , as well as other virulence genes , such as those located in SPI-2 ( Bustamante et al. , 2008 ; Golubeva et al. , 2012 ; Fàbrega and Vila , 2013 ; Martínez et al. , 2014 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26300871" target="_blank">26300871</a>]<br></li>
<li>Of note , HilD-mediated regulation is not restricted to the SPI-1 genes , as HilD also modulates the expression of genes located outside this genetic locus such as sopE , which encodes an effector protein secreted via TTSS-1 , and ssrAB , which encodes the two-component system that acts as the central positive regulator of the SPI-2 genes [ 10,17,18 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28426789" target="_blank">28426789</a>]<br></li>
<li>For instance , the transcriptional regulator HilD , encoded in SPI-1 , positively controls the expression of the genes within this island , as well as several other genes located outside SPI-1 , which mediate Salmonella invasion of host cells4 ,30 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27886269" target="_blank">27886269</a>]<br></li>
<li>Our results indicate that CpxR negatively controls the expression of the SPI-1 genes , as well as genes located in SPI-2 , by repressing the autoregulation of HilD , a central positive regulator for the expression of the genes within SPI-1 and SPI-2 and other virulence genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26300871" target="_blank">26300871</a>]<br></li>
<li>SPI-1 encodes the transcriptional regulators HilD , HilA and InvF , which induce the expression of the genes within this island in a cascade fashion ( Golubeva et al. , 2012 ; Fàbrega and Vila , 2013 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26300871" target="_blank">26300871</a>]<br></li>
				<label class="TU_collapse_similarity" for="tu_similarity_393">
Previously , we dem-onstrated that HilD directly induces the expression of the ssrAB operon , which is located in SPI-2 and codes for the SsrA/B two-component system , the central positive regulator of SPI-2 , thus establishing a transcriptional cross talk between SPI-1 and SPI-2 ( 21 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25135218" target="_blank">25135218</a>]<br></label>
<input id="tu_similarity_393", type="checkbox">
<div class="TU_similar">
<li>Interestingly , when S. Typhimurium is grown to late stationary phase in LB medium , HilD directly induces the expression of the ssrAB operon that is located in SPI-2 and codes for the SsrA/B two-component system , the central positive regulator of the SPI-2 genes , thus establishing a transcriptional cross talk between SPI-1 and SPI-2 ( Bustamante et al. , 2008 ; Martínez et al. , 2014 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26300871" target="_blank">26300871</a>]<br></li></div>
<li>Several studies support the notion that HilD induces the expression of its target genes mainly by counteracting the repression exerted by the histone-like nucleoid structuring protein ( H-NS ) on the respective promoters21 ,22,27,29,71,72 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27886269" target="_blank">27886269</a>]<br></li>
<li>Additionally , HilD not only regulates bacterial entry into host cells , but also coordinates the bacterial response to the intracellular milieu by modulating the expression of ssrAB , which encodes the central positive regulator of the SPI-2 genes [ 17 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28426789" target="_blank">28426789</a>]<br></li>
<li>In addition , HilD can also induce expression of the SPI-2 genes by direct binding to the ssrAB promoter ( the local regulatory system carried within SPI-2 ; see below ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23554419" target="_blank">23554419</a>]<br></li>
<li>The AraC-like transcriptional regulator HilD , encoded within SPI-1 , positively regulates the expression of the SPI-1 genes in a cascade fashion , mainly by directly inducing the expression of hilA , which encodes the master regulator of SPI-1 ( 2 , 5 , 23 -- 26 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25135218" target="_blank">25135218</a>]<br></li>
<li>It was found that LoiA positively regulates the expression of hilA ( the master activator of SPI-1 ) through direct activation of HilD under low O2 conditions , leading to the activation of SPI-1 genes for invasion .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28575106" target="_blank">28575106</a>]<br></li>
				<label class="TU_collapse_similarity" for="tu_similarity_400">
The two-com-ponent system CpxR/A represses the expression of Salmonella virulence genes by affecting the stability of the transcriptional regulator HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28575106" target="_blank">28575106</a>]<br></label>
<input id="tu_similarity_400", type="checkbox">
<div class="TU_similar">
<li>The two-component system CpxR/A represses the expression of Salmonella virulence genes by affecting the stability of the transcriptional regulator HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27886269" target="_blank">27886269</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26994203" target="_blank">26994203</a>]<br></li></div>
<li>Furthermore , our results revealed that in the absence of the downstream region bound by H-NS or when H-NS is inactivated , the expression of ssrAB becomes independent of HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25135218" target="_blank">25135218</a>]<br></li>
<li>In the absence of the 119 / 336 region , the expression of ssrAB was no longer repressed by H-NS ( Fig. 4 ) , whereas deletion of the sequence upstream of the position 55 did not affect the H-NS-mediated repression of ssrAB ( Fig. 3 and FIG 6 HilD displaces H-NS from ssrAB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25135218" target="_blank">25135218</a>]<br></li>
<li>All together , these results indicate that HilD induces the expression of ssrAB by displacing the H-NS nucleoprotein complex from the promoter of this operon ( Fig. 7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25135218" target="_blank">25135218</a>]<br></li>
<li>These results taken together suggest that the HilD , HilC , and RtsA proteins induce the transcription of the rtsA promoter by counteracting H-NS/Hha-mediated repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17675384" target="_blank">17675384</a>]<br></li>
<li>The sizes of the different extension products indicate that the HilD - and HilC-dependent transcripts initiate within the divergently transcribed invH ORF ( Fig. 5A -- C ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>This indicates that the role of the PhoP protein is to activate pmrD transcription and not simply to counteract repression by the PmrA protein on the pmrD promoter as proposed for other regulatory proteins such as HilC and HilD ( 28 , 29 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12676988" target="_blank">12676988</a>]<br></li>
<li>Furthermore , our results indicate that HilD induces the expression of ssrAB by binding to at least two different AT-rich regions : one located on or close to the promoter and the other located far downstream ( Fig. 3 and 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25135218" target="_blank">25135218</a>]<br></li>
<li>HilC and HilD each activate the expression of the other as well as inducing their own promoters ( 39 , 47 ) , so we then measured transcription from the hilC and hilD promoters on plasmids in hilC - or hilD-disrupted backgrounds on the chromosome .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18245288" target="_blank">18245288</a>]<br></li>
<li>It has also been demonstrated that HilC and HilD each induce the expression of the other as well as inducing their own promoters ( 40 , 47 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18245288" target="_blank">18245288</a>]<br></li>
<li>It has been demonstrated that HilC and HilD each signiﬁcantly activate the expression of the other when they are overproduced ( 13 , 40 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18245288" target="_blank">18245288</a>]<br></li>
<li>The FLAG-tagged HilD was immunoprecipitated by antiFLAG antibody , and the acetylation of K297 was decreased by about 20 % ( Figure 4A ) , indicating that the SPI-1 -- inducing condition down-regulates the K297 acetylation level .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28329249" target="_blank">28329249</a>]<br></li>
				<label class="TU_collapse_similarity" for="tu_similarity_413">
Disruption of the hilD gene was checked by PCR and by immu-noblotting analysis of the total proteins with an anti-HilD serum .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15661008" target="_blank">15661008</a>]<br></label>
<input id="tu_similarity_413", type="checkbox">
<div class="TU_similar">
<li>Disruption of the hilD gene was checked by PCR and by immunoblotting analysis of the total proteins with an anti-HilD serum .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15661008" target="_blank">15661008</a>]<br></li></div>
<li>Expression of HilD and SirA from plasmids pBAD-HilD1 and pBAD-SirA1 , respectively , was induced by adding 0.1 % L-arabinose to the medium at the beginning of the bacterial cultures .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21518393" target="_blank">21518393</a>]<br></li>
<li>Expression of HilD from pK6-HilD was induced by adding 0.001 % L-arabinose to the medium at the beginning of the bacterial cultures .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27886269" target="_blank">27886269</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27886269" target="_blank">27886269</a>]<br></li>
<li>Transcription of hilD is under the control of an autogenous , positive feedback loop by the HilD product ( Ellermeier et al. 2005 ; Ellermeier and Slauc Figure 3 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20008574" target="_blank">20008574</a>]<br></li>
<li>Thus , although HilD is capable of auto-regulation , the results observed could not be due to the loss of this regulatory protein controlling its own expres ¬ Pat post-transcriptionally regulates HilD The regulation of hilD is complex ; hilD regulates itself but is also regulated by the transcriptional regulators HilC and RtsA , which constitute a feed-forward circuit to control the downstream SPI1 regulator HilA ( Ellermeier et al. , 2005 ; Ellermeier and Slauch , 2007 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27341691" target="_blank">27341691</a>]<br></li>
<li>CpxR could directly repress the transcription of hilD or reduce post-transcriptionally the concentration of HilD and thus affect its positive autoregulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26300871" target="_blank">26300871</a>]<br></li>
<li>Other features that need to be taken in account when assessing hilD transcription are the presence of an intact hilD ORF , as the HilD protein positively regulates its own expression [ 15 ] and the presence of the newly described regulatory motif hilD 3 ' - UTR , that modulates hilD mRNA stability by promoting its rapid degradation [ 29 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28426789" target="_blank">28426789</a>]<br></li>
<li>This regulated induction of hilD eliminates the autoinduction of hilD expression by the HilD protein and thus ensures that the effects observed are not attrib-utable to the control of hilD transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27185788" target="_blank">27185788</a>]<br></li>
<li>HilD is required for EnvZ/OmpR regulation of hilA It was previously concluded that the EnvZ/OmpR two-component system functioned through hilC to regulate expression of invasion genes ( Lucas and Lee , 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16045614" target="_blank">16045614</a>]<br></li>
<li>Fis affects expression of invF : : Tn5lacZY in the absence of HilA Current models for the regulation of invasion genes suggest that there is a HilA-independent pathway to invF expression that involves HilD ( Lucas and Lee , 2000 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11123690" target="_blank">11123690</a>]<br></li>
<li>Collectively , these data indicated that expression of SPI-1 genes was reduced at 42 ° C through decreased function of the HilD protein , which resulted in reduced expression of rtsA and hilA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26386070" target="_blank">26386070</a>]<br></li>
<li>Moreover , although the expression of SPI1 genes hilA , invF and sipC was increased about 2-fold by over-expression of CobB ( Fig. 8B ) , the half-life of HilD was not affected ( Fig. 8C ) , indicating that CobB controls SPI1 through a mechanism different from that of Pat .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27341691" target="_blank">27341691</a>]<br></li>
<li>Thus , these data strongly support that HilD directly controls the expression of the gtgE , phoH , sinR , lpxR and SL4247 genes , and indirectly , through a regulator found in S. Typhimurium but not in E. coli MC4100 , that of the SL1896 gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27886269" target="_blank">27886269</a>]<br></li>
<li>Previously , by using chromatin immunoprecipitation-sequencing ( ChIP-seq ) and ChIP-qPCR , it was found that HilD binds in vivo to DNA regions associated with the sinR , lpxR and SL4247 ( ) S. Typhimurium 14028s genes ; furthermore , it was shown that HilD can induce the expression of a lpxR-lacZ translational fusion in the E. coli AMD054 strain ( flhDC + ) 31 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27886269" target="_blank">27886269</a>]<br></li>
<li>Our data from this study show that the sequence between positions 106 and 55 has a positive regulatory role for the expression of ssrAB , but this sequence was not required for the HilD - and H-NS-mediated regulation of this operon ( Fig. 3 and 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25135218" target="_blank">25135218</a>]<br></li>
<li>These include HilA that binds and represses the promoter of ssaH [ 24 ] , and HilD that binds and activates the promoter of the ssrAB operon [ 25 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19126220" target="_blank">19126220</a>]<br></li>
<li>We also found CDS genes that were dif-ferentially-expressed ( > 3-fold ) in the absence of both HilD and SsrB , and the majority of these 52 HilD / SsrB-controlled genes were virulence-associated ( Fig 3A and 3B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27564394" target="_blank">27564394</a>]<br></li>
<li>As controls , the expression of cat transcriptional fusions of the hilA and ssaG genes , which are controlled by HilD directly and indirectly , respectively , were also assessed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27886269" target="_blank">27886269</a>]<br></li>
<li>Genome-wide transcriptional and/or binding analyses support that H-NS represses gtgE , sinR , lpxR , SL1896 and SL424773 -- 75 ; however , whether HilD induces the expression of these genes by acting as an anti-repressor of H-NS , and how it positively controls the expression of phoH , remains to be determined .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27886269" target="_blank">27886269</a>]<br></li>
<li>These results show that HilD positively controls the expression of the gtgE , phoH , sinR , lpxR , SL1896 and SL4247 genes , independently of HilA and InvF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27886269" target="_blank">27886269</a>]<br></li>
<li>Thus , HilD positively and directly regulates the expression of the gtgE , phoH , sinR , lpxR and SL4247 genes , and positively but indirectly controls the expression of the SL1896 gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27886269" target="_blank">27886269</a>]<br></li>
<li>Under the experimental conditions used , HilC and RtsA were not needed to achieve high hilA expression ; however , HilD was essential since hilA expression was abolished in ΔhilD mutants regardless of the presence or absence of Gre factors .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28426789" target="_blank">28426789</a>]<br></li>
<li>These results show that HilD is able to activate the expression of ssrAB and hilA in the absence of SirA , whereas SirA requires the presence of HilD , suggesting that SirA controls HilD expression , which in turn directly regulates hilA and ssrAB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21518393" target="_blank">21518393</a>]<br></li>
<li>HilD is clearly the most signiﬁcant contributor to the expression of hilA and therefore SPI1 in general , since many systems which control expression of hilA seem to work through HilD in some way ( 6 , 8 , 16 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
<li>Collectively , these results demonstrate that LoiA directly binds to the hilD promoter to activate hilD expression ; and HilD in turn activates hilA expression , leading to the activation of eterminant in loiA deletion reduced the expression of hilA and SPI-1 genes Fig 4 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28575106" target="_blank">28575106</a>]<br></li>
<li>Since SirA activates expression of a hilA : : lacZ transcriptional fusion only in the presence of HilD ( 222 ) , CsrA has been hypothesized to bind to the hilD mRNA and impair its translation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23554419" target="_blank">23554419</a>]<br></li>
<li>0.1 % L-arabinose was added ( + ) to the medium for inducing expression of HilD and SirA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21518393" target="_blank">21518393</a>]<br></li>
<li>To check if both regulators are important , the regulation of hilC gene expression by HilD and SirA is considered to function as an AND gate , and to check the possibility of one of the regulators being important , this dependency is taken as an OR gate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19537165" target="_blank">19537165</a>]<br></li>
<li>HilD , HilC , and RtsA are homologous AraC-like regulators that bind to very similar DNA sites ; therefore , when these regulators are overexpressed , they are able to self-activate their own expression and to activate the expression of one another , as well as to induce the expression of most genes belonging to the HilD regulon ; however , HilD is considered to be dominant , as deletion of hilD , but not of hilC or rtsA , drastically reduces the expression of the targets of HilD ( 23 , 24 , 26 -- 30 , 61 -- 64 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25135218" target="_blank">25135218</a>]<br></li>
<li>They act by increasing the level of HilD protein to overcome HilE , and controlling HilD activity , or in the case of Fur , lowering the threshold required at the promoter , such that HilD activates its own promoter as well as induces expression of HilC and RtsA , which then act to amplify and accelerate SPI1 expression ( Saini et al. 2010a ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22021388" target="_blank">22021388</a>]<br></li>
<li>Several studies strongly support the idea that HilD induces the expression of hilA and rtsA by counteracting the repression exerted by H-NS on the promoters of these genes ( 23 , 24 , 27 , 48 , 49 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25135218" target="_blank">25135218</a>]<br></li>
<li>Based on these observations , we suggest that Lrp acts as an antagonist of HilD , which induces expression of the hilA gene by derepressing the silencing effect of H-NS and Hha on the hilA promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19074398" target="_blank">19074398</a>]<br></li>
<li>Previous studies indicate that HilD induces the expression of hilA by counteracting the repression exerted by the nucleoid protein H-NS on the promoter of this gene ( Schechter et al. , 1999 ; Schechter and Lee , 2001 ; Olekhnovich and Kadner , 2006 ) ; thus , in the absence of H-NS activity hilA can be expressed independently of HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26300871" target="_blank">26300871</a>]<br></li>
<li>However , there was no strict correlation between the level of HilD protein and the level of hilA expression ; HilD protein levels in the ﬂiZ and pFliZ backgrounds differed by only 2-fold , whereas hilA transcription differed by 10-fold .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20889744" target="_blank">20889744</a>]<br></li>
<li>To assess the HilD activity of tested conditions ( vehicle control and 0.1 mM splitomicin ) , hilA expression was examined by b-galactosidase assays using the bacterial sample at time zero .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27341691" target="_blank">27341691</a>]<br></li>
<li>Expression of the SPI1 T3SS is tightly regulated by the combined action of HilC , HilD , and RtsA , three AraC family members that can independently activate hilA , which encodes the direct regulator of the SPI1 structural genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
<li>Discussion Expression of the SPI1 T3SS is controlled by HilD , HilC , and RtsA , acting in a complex feed-forward loop to activate the hilD , hilC , and rtsA genes , as well as hilA , which encodes the transcriptional activator of the T3SS structural genes ( Figure 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22021388" target="_blank">22021388</a>]<br></li>
<li>In view of these ﬁndings , we suggest that the regulation of SPI1 expression is negatively controlled through deg-radation of the HilC and HilD transcriptional regulators by Lon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15661008" target="_blank">15661008</a>]<br></li>
<li>( B ) In the absence of bile , the turnover of HilD is high and the expression of hilD is at a basal level , and as a result the expression of hilA is low .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29229736" target="_blank">29229736</a>]<br></li>
<li>Since HilD can activate hilA transcription , is at the top of the hierarchical SPI1 regulatory loop , and has a dominant role , the posttranslational control of HilD by Lon is critically important for precise expression of SPI1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18723621" target="_blank">18723621</a>]<br></li>
<li>While the mechanism by which these genes alter hilA expression is unknown , it is possible that some of these genes encode hilA repressors that may be the targets of the derepressors HilC and HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11673432" target="_blank">11673432</a>]<br></li>
<li>Thus , it is possible that the feed forward loop constituted by HilD , HilC , and RtsA can regulate the expression of ssrAB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25135218" target="_blank">25135218</a>]<br></li>
<li>It has been proposed that the feed-forward loop system of HilC , HilD , and RtsA functions as a switch that controls hilA expression by modulating the threshold of HilD protein required for the ultimate HilA activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26441883" target="_blank">26441883</a>]<br></li>
<li>The average HilD half-life for bacteria grown in me-dium alone was 63 min , but the half-life was reduced to 16 min in the presence of bile and 8 min in the presence of cholate , revealing a mechanism in which bile and bile-acids repress SPI-1 gene expression and protein levels by posttranslational destabilization of HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27185788" target="_blank">27185788</a>]<br></li>
<li>Moreover , on deletion of the pag gene , located within pagK and pagM , increased hilA expression is observed only in the presence of HilD ( 232 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23554419" target="_blank">23554419</a>]<br></li>
<li>Overexpression of RtsA or RtsA/B only partially compensated for the loss of hilA gene expression in the relA spoT strain ( Fig. 5 ) , presumably because of the reduced expression of hilA gene co-activators , HilC and HilD ( 52 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16905537" target="_blank">16905537</a>]<br></li>
<li>Although overexpression of the HilC or HilD protein results in high and unregulated expression of hilA ( 30 ) , the relative contribution of changes in the level or activity of either protein in control of hilA expression is not clear .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12107132" target="_blank">12107132</a>]<br></li>
<li>Boddicker , J.D. , Knosp , B.M , and Jones , B.D. ( 2003 ) Transcription of the salmonella invasion gene activator , hilA , requires HilD activation in the absence of negative regulators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27341691" target="_blank">27341691</a>]<br></li>
<li>The presence of CsrB restores the expression of hilD , hilA and ssaG in the absence of SirA but not in the absence of HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21518393" target="_blank">21518393</a>]<br></li>
				<label class="TU_collapse_similarity" for="tu_similarity_463">
( B ) Effect of Lon on slrP expression in the presence and in the absence of HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25182488" target="_blank">25182488</a>]<br></label>
<input id="tu_similarity_463", type="checkbox">
<div class="TU_similar">
<li>( C ) Effect of LeuO on slrP expression in the presence and in the absence of HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25182488" target="_blank">25182488</a>]<br></li></div>
<li>To distinguish whether HilE regulates hilA via HilD , we induced hilA ( and hilC ) transcription in the presence or absence of HilD by the addition of increasing concentrations of tetracycline in a tetRA -- rtsA background .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22021388" target="_blank">22021388</a>]<br></li>
<li>As expected , activation of hilA promoter in the absence of the activators HilD and HilC ( lines 5 -- 8 ) was lost or signiﬁcantly reduced in inducing ( high osmolarity ) conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16443238" target="_blank">16443238</a>]<br></li>
<li>HilD activates transcription of hilA and other target genes in response to environmental conditions associated with the intestinal microenvironment of the host .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24375101" target="_blank">24375101</a>]<br></li>
<li>Schematic representation of the SirA-CsrB/C-CsrA-HilD regulatory cascade controlling the expression of SPI-1 and SPI-2 genes in S. Typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21518393" target="_blank">21518393</a>]<br></li>
<li>Furthermore , CsrB was able to restore the expression of the SPI-1 and SPI-2 genes in the absence of SirA , but not in the absence of HilD ( Fig. 6 ) , indicating that SirA acts through CsrB to regulate these genes , and that HilD is downstream of SirA and CsrB in the regulatory cascade controlling them .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21518393" target="_blank">21518393</a>]<br></li>
<li>Under these growth conditions , CsrB and CsrC counteract CsrA-mediated translational repression of hilD , thereby allowing the synthesis of HilD , which in turn positively controls the regulatory cascade leading to the expression of the SPI-1 and SPI-2 genes that have essential roles in Salmonella virulence ( 11 -- 15 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24187088" target="_blank">24187088</a>]<br></li>
<li>HilD then activates expression of hilA and ssrAB , the genes encoding the central regulators of SPI-1 and SPI-2 , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21518393" target="_blank">21518393</a>]<br></li>
<li>In contrast , HilD activated expression of ssrAB and hilA in the absence of SirA , indicating that HilD does not require the presence of SirA to directly activate expression of the SPI-1 and SPI-2 regulatory genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21518393" target="_blank">21518393</a>]<br></li>
<li>SirA , but not HilD , requires the presence of CsrB and CsrC to induce secretion of SPI-1-encoded proteins and expression of the SPI-2 effector protein SseB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21518393" target="_blank">21518393</a>]<br></li>
<li>Taken together , these results show that CpxR represses the autoregulation of HilD , which in turn affects the expression of hilA and thus the SPI-1 genes , as well as of other virulence genes regulated by HilD , such as ssrB and sseB located in SPI-2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26300871" target="_blank">26300871</a>]<br></li>
<li>In agreement with our results indicating that CpxR represses the HilD-dependent expression of hilD and hilA , both the overexpression of CpxR and the absence of CpxA drastically reduced the production of SsrB-FLAG and SseB proteins ( Figures 1C , 3B ) , which are encoded in SPI-2 and whose expression is also dependent of HilD in the condition tested .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26300871" target="_blank">26300871</a>]<br></li>
<li>moreover , we demonstrate that CpxR-P negatively affects the stability of HilD and thus decreases the expression of HilD-target genes , such as hilD itself and hilA , located in SPI-1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26300871" target="_blank">26300871</a>]<br></li>
<li>Furthermore , we demonstrate that CpxR-P negatively affects the transcription of the SPI-1 genes hilD and hilA , but only in the presence of HilD , which would be expected , since the expression of HilD is autoregulated and HilD directly regulates hilA ( Golubeva et al. , 2012 ; Fàbrega and Vila , 2013 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26300871" target="_blank">26300871</a>]<br></li>
<li>Absence of IgeR did not affect the expression of HilA , HilC/SprA or HilD , which are three speciﬁc regulators located at or near the top of the SPI-1 TTSS regulatory cascade ( Lucas and Lee , 2000 ; Galán , 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17555437" target="_blank">17555437</a>]<br></li>
<li>The activation of hilE transcription produces HilE protein , which then leads to repression of hilA expression and subsequent expression of the other SPI-1 genes through its interaction with the HilD regulatory protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15731035" target="_blank">15731035</a>]<br></li>
<li>For example , hilA expression , rtsA expression , and hilD expression are decreased by deletion of fur at the level of transcription via HilD , supporting the idea that HilD controls the transcription of these genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
<li>HilD acts as a derepressor of PhilA and facilitates the expression of the hilA gene ( 85 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19074398" target="_blank">19074398</a>]<br></li>
<li>Moreover , HilD coordinates hilA transcription with the expression of other crucial infection-relevant systems , e.g. , it induces expression of the ﬂagellar operons through activation of ﬂhDC transcription and controls expression of Spi-2 genes ( see also the Sections Regulation of Motility and Regulatory Circuits Controlling Later Stages of Infection and Defense Systems against the Host 's Immune Response ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26441883" target="_blank">26441883</a>]<br></li>
<li>HilC and HilD , which are members of the AraC/XylS family , have been postulated to act as a derepressor of hilA expression by counteracting the action of negative regulatory elements at the hilA promoter ( 12,13 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17329372" target="_blank">17329372</a>]<br></li>
<li>HilD , a member of the AraC/XylS family of transcriptional regulators , induces the expression of hilA , encoding a transcriptional activator of the inv/spa , prg/org , and sic/sip operons [ 3 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28329249" target="_blank">28329249</a>]<br></li>
<li>This ability of the hilD 3 ′ UTR to reduce expression of a heterologous gene supports the view that the 3 ′ UTR can act as an independent module Because hilD mRNA is more abundant in the absence of its native 3 ′ UTR and HilD is a positive regulator of SPI-1 , we wondered whether increased level of hilD mRNA resulted in increased SPI-1 expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24682814" target="_blank">24682814</a>]<br></li>
<li>Thus , the ability of HilD pro-lative β-gal activity tein alone to activate hilA expression is reduced in the presence of oleate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26884427" target="_blank">26884427</a>]<br></li>
<li>This could be due to condition-speciﬁc binding of HilD upstream of ssrA , since ssrAB expression is induced by HilD in the late stationary phase , well beyond the time point of our ChIP-seq experiment ( 14 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24375101" target="_blank">24375101</a>]<br></li>
<li>MBP-HilD specifically bound the DNA fragments of gtgE , phoH , sinR , lpxR , SL4247 and , as expected , that of hilA , at concentrations of 0.1 to 1.0 μM ; in contrast , at the same concentrations it did not bind the DNA fragment of SL1896 , or those of the negative controls , ppk and sigD ( Fig. 4A -- G ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27886269" target="_blank">27886269</a>]<br></li>
<li>HilD induces the expression of gtgE , phoH , sinR , lpxR and SL4247 , but not that of SL1896 , in the absence of other Salmonella-specific regulators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27886269" target="_blank">27886269</a>]<br></li>
<li>Additionally , these results indicate that the region between positions 119 and 478 contains negative regulatory sequences and that , in the absence of this region , the expression of ssrAB becomes independent of HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25135218" target="_blank">25135218</a>]<br></li>
<li>2.4  IscR represses the expression of hilD gene and HilD targets Next , to investigate the regulatory role of IscR on hilD , the expression of the hilD fusion was measured and found to be about 1.5 fold up‐reg-ulated in an iscR mutant ( Figure 4a ) , suggesting that IscR acts as a negative regulator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27704705" target="_blank">27704705</a>]<br></li>
<li>However , the effects of Mlc on hilD expression and bacterial invasiveness were not seen in the hilE mutant , and hilE expression was increased in the mlc mutant , which suggests that Mlc exerts positive effects on the expression of SPI1 genes by reducing the expression of HilE , which is known to downregulate the expression of SPI1 genes through direct interaction with HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17329372" target="_blank">17329372</a>]<br></li>
<li>We first note that these additional feedback loops , namely the ones regulating the expression of HilC and RtsA , do not add redundancy to the circuit , as the loss of HilD effectively shuts off SPI1 gene expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20686667" target="_blank">20686667</a>]<br></li>
<li>( A ) HilD is at the top of the SPI-1-regulatory hierarchy , where it regulates its own expression and the expression of HilA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29229736" target="_blank">29229736</a>]<br></li>
<li>SPI-1 expression is repressed by hns and activated by a complex positive feedback loop where the production of the HilD regulatory protein induces the expression of HilA , a transcription factor that directly activates expression of the TTSS and effector proteins [ 70 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25375226" target="_blank">25375226</a>]<br></li>
<li>Therefore , the effect of CpxR-P on the expression of the SPI-1 and SPI-2 genes could be the result of its negative control on the stability of HilD and , as a consequence , on the transcription of hilD , which , in an additive manner , would decrease the concentration of HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26300871" target="_blank">26300871</a>]<br></li>
<li>Both HilD and HilA directly promote the expression of the ﬁrst gene in the inv-spa operon , invF ( 41 ) , which is then able to drive the expression of the sic-sip operon and a number of other critical effectors ( 39 , 42 , 43 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29866910" target="_blank">29866910</a>]<br></li>
<li>Furthermore , HilD directly controls the expression of the flhDC operon encoding FlhDC , the master transcriptional complex required for the expression of flagellar and chemotaxis genes30 ,35 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27886269" target="_blank">27886269</a>]<br></li>
<li>As expected , disruption of the ﬂhDC operon , encoding the master regulator for expression of the ﬂagellar regulon , blocked the increase of HilC and HilD levels caused by clpPX disruption .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18245288" target="_blank">18245288</a>]<br></li>
<li>Additionally , the over-expression of HilD from the plasmid pJB3 containing hilD gene under the control of a lac promoter caused the induction of hilA by 2-fold and invF by 6-fold irrespective of hilE or mlc mutation when real-time PCR was used to compare the expression levels ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17329372" target="_blank">17329372</a>]<br></li>
<li>In contrast , the absence of HilD alone blocks the ability of SirA to induce expression of the hilA -- lac or invF -- lac fusions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16045614" target="_blank">16045614</a>]<br></li>
<li>Results in Figure S1 show that the hilD -- lac transcriptional and translational fusions were also not affected by the loss of FliZ , conﬁrming that FliZ controls hilA expression via the post-translational control of HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22021388" target="_blank">22021388</a>]<br></li>
<li>The low level of hilD expression in the mlc mutant can be explained by the fact that the increased level of HilE in the mlc mutant can repress the activity of HilD , which is known to act as an activator of its own expression ( 9,36 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17329372" target="_blank">17329372</a>]<br></li>
<li>As shown in Figure 4B , the expression of the hilD-cat fusion was similarly reduced in these three mutants , indicating that the absence of CpxA or HilD has the same effect on the expression of hilD , and that , when HilD is not present , the absence of CpxA does not longer repress hilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26300871" target="_blank">26300871</a>]<br></li>
<li>In contrast , the expression of the cpxRA-cat fusion was similarly increased in the 1cpxA and 1hilD 1cpxA mutants ( Figure 1D ) , indicating that the absence of CpxA activates the expression of cpxRA independently of HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26300871" target="_blank">26300871</a>]<br></li>
<li>The overexpression of CpxR , from plasmid pK3-CpxR , reduced ﬁve-fold the expression of a hilA-cat transcriptional fusion in the WT strain ( Figure 4C ) , but did not affect the high levels of expression showed by this fusion in the 1hilD 1Cthns mutant ( Figure 4D ) , indicating that CpxR regulates hilA through HilD and not directly .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26300871" target="_blank">26300871</a>]<br></li>
<li>In contrast , the expression of HilD from plasmid pK6-HilD did not increase the activity of the SL1896-cat fusion ( Fig. 3F ) , indicating that HilD induces the expression of SL1896 through a factor not present in E. coli MC4100 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27886269" target="_blank">27886269</a>]<br></li>
<li>Since the expression of OmpR was not affected in the absence of SirA or HilD ( data not shown ) , we hypothesized that SirA could regulate the expression of the ssrAB operon directly , acting along with HilD and OmpR , or indirectly by controlling the expression of HilD in a cascade fashion .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21518393" target="_blank">21518393</a>]<br></li>
<li>HilD is a known activator of hilA transcription ; HilA then activates the transcription of genes encoding the Spi1 type III secretion apparatus and Spi1 effector proteins through InvF ( 24 , 59 , 60 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24488311" target="_blank">24488311</a>]<br></li>
<li>HilC and HilD also directly activate the alternative promoter of the invF operon , which is independent of HilA ( 15,16 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17329372" target="_blank">17329372</a>]<br></li>
<li>In this context , slrP is induced by overexpression of HilC , HilD , and RtsA independently of the central SPI1 regulator HilA , with RtsA being the best inducer .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25182488" target="_blank">25182488</a>]<br></li>
<li>Interestingly , HilD is not required for expression of the ssrAB operon when Salmonella grows in minimal media containing low concentrations of magnesium and phosphate ( Bustamante et al. , 2008 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21518393" target="_blank">21518393</a>]<br></li>
<li>Although this HDI reduced the expression of SPI1 genes ( Supporting Information Figs. 3A and B ) , it did not cause a change in HilD stability ( Supporting Information Fig. 3B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27341691" target="_blank">27341691</a>]<br></li>
<li>HilD , but not HilA or InvF , regulates the expression of gtgE , phoH , sinR , lpxR , SL1896 and SL4247 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27886269" target="_blank">27886269</a>]<br></li>
<li>HilD induces the expression of gtgE , phoH , sinR , lpxR and SL4247 , but not SL1896 , in E. coli MC4100 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27886269" target="_blank">27886269</a>]<br></li>
<li>Interestingly , when S. typhimurium is grown under the normally repressing low osmolarity condition , a null mutation in hns results in high hilA expression , even in the absence of HilC and HilD ( L. Schechter and C. Lee , unpublished observations ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11755416" target="_blank">11755416</a>]<br></li>
<li>HilC and HilD are encoded in the SPI1 locus , while RtsA is encoded elsewhere in the chromosome in an operon with RtsB , which negatively regulates expression of ﬂhDC and therefore the entire ﬂagellar regulon ( 19 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17951383" target="_blank">17951383</a>]<br></li>
<li>Therefore , the expression of gtgE is controlled by HilD in SPI-1-inducing conditions and probably by another regulator in SPI-2-inducing conditions , which would coordinate the secretion of GtgE through the T3SS-1 and T3SS-2 , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27886269" target="_blank">27886269</a>]<br></li>
<li>Similarly , the expression of slrP , also encoding an effector protein secreted through both T3SS-1 and T3SS-2 , is controlled by HilD in SPI-1-inducing conditions and by the response regulator PhoP in SPI-2-inducing conditions32 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27886269" target="_blank">27886269</a>]<br></li>
<li>The expression of the type III secretion system-related genes involved in invasion and intracellular survival is induced in the presence of transcriptional regulator proteins such as HilA , HilC , HilD , InvA , InvC , InvF , SirA , and SirB ( 11 , 27 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24780330" target="_blank">24780330</a>]<br></li>
<li>The expression of the genes , those regulate the type III secretion system , are involved in completion of invasion , intracellular survival and multiplication , is induced by the transcriptional regulator proteins such as HilA , HilC , HilD , InvA , InvC , InvF , SirA , and SirB ( Johnston et al. , 1996 ; Rakeman et al. , 1999 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27565525" target="_blank">27565525</a>]<br></li>
<li>HilD and SlyA cooperate to directly control the expression of ssrAB in SPI-1-inducing conditions ( our unpublished results ) , which could also apply for lpxR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27886269" target="_blank">27886269</a>]<br></li>
<li>Previously , we showed that when Salmonella grows in LB medium HilD , encoded in SPI-1 , first induces the expression of hilA , located in SPI-1 , and subsequently of the ssrAB operon , located in SPI-2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21518393" target="_blank">21518393</a>]<br></li>
<li>However , the ﬁnding that HilC/D proteins are not required for transcription from the hilA promoter is in contrast to the strong requirement for HilD for hilA-lac expression in cells lacking some of the negative factors ( 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15126488" target="_blank">15126488</a>]<br></li>
<li>However , when sequences upstream of the 39 position in the hilA promoter , termed the upstream repressing sequence ( URS ) , were deleted from pLS31 , the hilA reporter plasmid , HilD was no longer required for high-level hilA-lacZY expression ( 42 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12511499" target="_blank">12511499</a>]<br></li>
<li>Figure 4B shows that while the sirA deletion decreased hilA expression in the presence of HilD , it had no effect on the hilA expression when hilA was being activated by the tetRA -- rtsA construct at 1 mg/mL tetracycline in the absence of HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22021388" target="_blank">22021388</a>]<br></li>
<li>Since HilD activates hilA transcription , which in turn activates the transcription of the InvF transcriptional regulator ( 2 ) , it was surprising that reduced transcription of the central regulator HilA was not detected by microarray analysis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17060472" target="_blank">17060472</a>]<br></li>
<li>To examine this possibility , the cellular level of HilD was determined in 32-overproducing cells in which rpoH , encoding 32 , is induced under the control of the PA1/lacO -1 promoter system .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18723621" target="_blank">18723621</a>]<br></li>
<li>However , this effect apparently is not mediated through Lon , the protease known to affect HilD , as in mutants of lon , sipC remained repressed by Pat ( Fig. 4B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27341691" target="_blank">27341691</a>]<br></li>
<li>hilD As HilD is capable of inducing HilA expression in absence of HilC or RtsA , we reasoned that HilC may be able to do the same in the absence of HilD if it is transcribed in a manner similar to hilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20686667" target="_blank">20686667</a>]<br></li>
<li>If a condition exists in which hilA is repressed while invF expression is induced through HilC or HilD , it begs the question : Why produce InvF-dependent effectors in the absence of hilA expression when an intact type III secretion apparatus is not thought to be formed ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10844688" target="_blank">10844688</a>]<br></li>
<li>In other words , by removing the threshold set by HilE , HilD is capable of inducing HilA expression when expressed from the weaker PhilC promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20686667" target="_blank">20686667</a>]<br></li>
<li>As HilD is at the top of the SPI1 transcriptional cascade , HilE is able to repress the expression of all SPI1 genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20686667" target="_blank">20686667</a>]<br></li>
				<label class="TU_collapse_similarity" for="tu_similarity_534">
Consequently , we suggest that SPI1 expression is controlled by the feedback regulatory loop in which 32 induces Lon to control turnover of HilD , and DnaK , which inhibits 32 function , leading to the modulation of lon expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18723621" target="_blank">18723621</a>]<br></label>
<input id="tu_similarity_534", type="checkbox">
<div class="TU_similar">
<li>Taking these ﬁndings together , it is suggested that SPI1 gene expression is regulated by the feedback regulatory loop in which 32 induces Lon protease to control the turnover of HilD and the DnaKJ chaperone machinery , which can inhibit 32 function , leading to the modulation of lon expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18723621" target="_blank">18723621</a>]<br></li></div>
<li>Since experiments by Schechter et al. suggested that HilD is not needed for expression of a PhilA-lacZY reporter plasmid when hilA promoter sequences upstream of position 39 are deleted , it was proposed that HilD is a de-repressor needed to counteract the effects of negative regulators that utilize the URS to repress the hilA promoter ( 42 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12511499" target="_blank">12511499</a>]<br></li>
<li>Of note , it seems unlikely that the increased degradation of HilD is mediated by changes in protease expression or activity , as the expression of lon is not increased by bile treatment ( 18 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27185788" target="_blank">27185788</a>]<br></li>
<li>By using this strategy , we previously demonstrated that HilD is required for the expression of SsrB in the presence of H-NS but not when it is inactivated ( 21 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25135218" target="_blank">25135218</a>]<br></li>
<li>Although it is not the only factor involved in the initiation of hilA expression , HilD is a dominant derepressor of HilA [ 19 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18656407" target="_blank">18656407</a>]<br></li>
<li>If FliZ affects hilA expression solely through HilD , then there should be no ﬂiZ phenotype in a hilD background when the system is activated by other means .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20889744" target="_blank">20889744</a>]<br></li>
<li>We conﬁrmed that FLAG-tagged HilD is fully active by measuring expression from the hilA promoter ( see Fig .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24375101" target="_blank">24375101</a>]<br></li>
<li>Bile acids function synergistically to repress invasion gene expression in Salmonella by destabilizing the invasion regulator HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27886269" target="_blank">27886269</a>]<br></li>
<li>Since we have evidence that HilE may regulate invasion gene expression by binding to the HilD protein , we were interested in determining whether hilE could repress invF transcriptional activation in the absence of hilA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12595445" target="_blank">12595445</a>]<br></li>
<li>A cascade of transcriptional activation has been unraveled in which HilD , in response to particular environmental signals , directly affects hilA expression by binding to the upstream repressing sequence , thereby derepressing hilA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16391141" target="_blank">16391141</a>]<br></li>
<li>When in vitro environmental conditions become favorable for invasion gene expression , HilD de-represses the hilA promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11755416" target="_blank">11755416</a>]<br></li>
<li>Altogether , these results suggest that HilD plays a pivotal role in activating hilA expression under the experimental conditions used .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28426789" target="_blank">28426789</a>]<br></li>
<li>The results of an electrophoretic mobility shift assay ( EMSA ) suggested that reduced hilD expression was not caused by butyrate inhibiting HilD to bind its own promoter ( Figure S2C ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29447698" target="_blank">29447698</a>]<br></li>
<li>Significantly , control of HilD by propionate manifests itself post-translationally : this fatty acid demonstrated regulation through HilD even when hilD was expressed from exogenous promoters , and a translational reporter fusion to hilD failed to be repressed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23289537" target="_blank">23289537</a>]<br></li>
<li>The control of HilD by Pat was through post-transcriptional mechanisms , moderately repressing hilD translation while significantly reducing HilD stability .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27341691" target="_blank">27341691</a>]<br></li>
<li>The strains used were plasmid-containing derivatives of JS273 through JS278 RtsA induction of hilA does not require HilC or HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923082" target="_blank">12923082</a>]<br></li>
<li>RtsA is a better inducer of slrP expression than HilC or HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923082" target="_blank">12923082</a>]<br></li>
<li>Therefore , we wanted to determine if HilC or HilD could also induce expression of slrP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923082" target="_blank">12923082</a>]<br></li>
<li>It was demonstrated that FliZ posttranslationally controls HilD to positively regulate expression of hilA [ 67 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22632036" target="_blank">22632036</a>]<br></li>
<li>Previous studies have shown that HilD , not HilC , is primarily responsible for derepression of hilA expression when Salmonella serovar Typhimurium is grown under our inducing conditions ( 19 , 23 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12933899" target="_blank">12933899</a>]<br></li>
<li>First , our data suggest that HilD is critical for activation of hilA expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12511499" target="_blank">12511499</a>]<br></li>
<li>We previously showed that FliZ-mediated induction of hilA expression is through HilD ( 30 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20889744" target="_blank">20889744</a>]<br></li>
<li>( D ) Effect of overproduced HilD on ﬂagellar gene expression under artiﬁcially induced ﬂhDC expression conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24488311" target="_blank">24488311</a>]<br></li>
<li>We observed that arabinose-induced expression of HilD activates invF expression > 80-fold ( Fig. 3 , black bars ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>Fur regulates expression of hilA by working through HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
<li>Fur regulates expression of hilA via HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
<li>Elevated levels of HilD protein activate hilA gene expression ( 41 -- 44 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24488311" target="_blank">24488311</a>]<br></li>
<li>Figure 5 Class III , PhoPQ ( PhoQ24 ) represses hilA expression independently of HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22021388" target="_blank">22021388</a>]<br></li>
<li>Effect of overexpression of HilD on SPI1 gene expression in hilD and hilA hilD mutants .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>If environmental conditions and regulatory factors modulate hilA expression by affecting hilD transcript stability , high-level expression of HilD from the PBAD promoter may help compensate for hilD transcript instability , allowing expression of hilA under repressing conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11292791" target="_blank">11292791</a>]<br></li>
<li>Because arabinose-induced expression of hilD results in artiﬁcially high levels of HilD that might compensate for posttranscriptional regulation of hilD , our results can not determine which environmental and regulatory pathways regulate hilA expression by modulating hilD expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11292791" target="_blank">11292791</a>]<br></li>
<li>Loss of HilD decreases expression of hilA 10-fold under SPI1-inducing conditions , whereas a mutation of hilC reduces expression of hilA 2-fold ( 26 , 67 , 71 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923082" target="_blank">12923082</a>]<br></li>
<li>First , overproduction of Fur signiﬁcantly increases expression of hilA and hilD when HilD is present .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
<li>De-letion of hilE increases expression of hilA only when HilD is present .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18245288" target="_blank">18245288</a>]<br></li>
<li>However , we found that expression of invF on pVV448 is only induced about sevenfold by overexpression of HilD ( Fig. 4 , black bars ) and is not affected at all by overexpression of HilC ( Fig. 4 , striped bars ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>Figure 2A shows that loss of hilD is complemented by arabinose-induced expression of HilD from pSA4 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535071" target="_blank">12535071</a>]<br></li>
<li>It has also recently been reported that loss of HilD decreases hilC transcription but that loss of HilC does not signiﬁcantly alter hilD expression , so it is proposed that HilD is the apex of regulation of SPI1 expression ( 13 , 15 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18245288" target="_blank">18245288</a>]<br></li>
<li>Figure 6 shows that expression of rtsA was decreased by fur , but only when HilD was present .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
<li>Fur regulates expression of rtsA by controlling HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17993530" target="_blank">17993530</a>]<br></li>
<li>A. HilD stability was reduced by over-expression of Pat .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27341691" target="_blank">27341691</a>]<br></li>
				</div></ul></div><br>
	</div>
</body>
</html>