<!doctype html>
<style>
@charset "utf-8";
body {
	height: 100%;
	margin: 0;
	padding: 0;
}

.banner {
	position: fixed;
	top: 0;
	width: 100%;
	height: 65px;
	background: gray;
	line-height: 65px;
}
.spacer {
	height: 10px;
}

.main-content {
	padding: 5%;
}

.collapse_ACT {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-style: none solid none none;
	border-color: #B4045F;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_ACT + input{
	display: none;
}
.collapse_ACT + input + div{
	display: none;
}
.collapse_ACT + input:checked + div{
	display: block;
}
.ACT {
	text-align: justify;
	text-justify: inter-word;
}
.ACT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B4045F;
}
.ACT_collapse_similarity:hover{
	background: #e5d3dc;
}
.ACT_collapse_similarity + input {
	display: none;
}
.ACT_collapse_similarity + input + div{
	display: none;
}

.ACT_collapse_similarity + input:checked + div{
	display: block;
}
.ACT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.ACT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B4045F;
	padding-left: 5px;
}
.ACT_collapse_similarity:hover{
	background: #e5d3dc;
}
.ACT_collapse_similarity + input {
	display: none;
}
.ACT_collapse_similarity + input + div{
	display: none;
}
.ACT_collapse_similarity + input:checked + div{
	display: block;
}
.ACT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.ACT_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #B4045F;
}
.ACT_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_DOM {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-style: none solid none none;
	border-color: #B404AE;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_DOM + input{
	display: none;
}
.collapse_DOM + input + div{
	display: none;
}
.collapse_DOM + input:checked + div{
	display: block;
}
.DOM {
	text-align: justify;
	text-justify: inter-word;
}
.DOM_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B404AE;
	padding-left: 5px;
}
.DOM_collapse_similarity:hover{
	background: #e8d7e7;
}
.DOM_collapse_similarity + input {
	display: none;
}
.DOM_collapse_similarity + input + div{
	display: none;
}
.DOM_collapse_similarity + input:checked + div{
	display: block;
}
.DOM_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}.DOM_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #B404AE;
}
.DOM_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_EVO {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #7401DF;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_EVO + input{
	display: none;
}
.collapse_EVO + input + div{
	display: none;
}
.collapse_EVO + input:checked + div{
	display: block;
}
.EVO {
	text-align: justify;
	text-justify: inter-word;
}
.EVO_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #7401DF;
	padding-left: 5px;
}
.EVO_collapse_similarity:hover{
	background: #e2d7ed;
}
.EVO_collapse_similarity + input {
	display: none;
}
.EVO_collapse_similarity + input + div{
	display: none;
}
.EVO_collapse_similarity + input:checked + div{
	display: block;
}
.EVO_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.EVO_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #7401DF;
}
.EVO_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_RP {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #0174DF;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_RP + input{
	display: none;
}
.collapse_RP + input + div{
	display: none;
}
.collapse_RP + input:checked + div{
	display: block;
}
.RP {
	text-align: justify;
	text-justify: inter-word;
}
.RP_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #0174DF;
	padding-left: 5px;
}
.RP_collapse_similarity:hover{
	background: #c9d7e2;
}
.RP_collapse_similarity + input {
	display: none;
}
.RP_collapse_similarity + input + div{
	display: none;
}
.RP_collapse_similarity + input:checked + div{
	display: block;
}
.RP_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.RP_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #0174DF;
}
.RP_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_SIT {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #0489B1;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_SIT + input{
	display: none;
}
.collapse_SIT + input + div{
	display: none;
}
.collapse_SIT + input:checked + div{
	display: block;
}
.SIT {
	text-align: justify;
	text-justify: inter-word;
}
.SIT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #0489B1;
	padding-left: 5px;
}
.SIT_collapse_similarity:hover{
	background: #d0dfe2;
}
.SIT_collapse_similarity + input {
	display: none;
}
.SIT_collapse_similarity + input + div{
	display: none;
}
.SIT_collapse_similarity + input:checked + div{
	display: block;
}
.SIT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.SIT_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #0489B1;
}
.SIT_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_TU {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #04B486;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_TU + input{
	display: none;
}
.collapse_TU + input + div{
	display: none;
}
.collapse_TU + input:checked + div{
	display: block;
}
.TU {
	text-align: justify;
	text-justify: inter-word;
}
.TU_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #04B486;
	padding-left: 5px;
}
.TU_collapse_similarity:hover{
	background: #d0e2de;
}
.TU_collapse_similarity + input {
	display: none;
}
.TU_collapse_similarity + input + div{
	display: none;
}
.TU_collapse_similarity + input:checked + div{
	display: block;
}
.TU_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.TU_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #04B486;
}
.TU_similar > a {
	text-decoration: none;
	color: gray;
}

ul { 
	list-style-type: none;
	padding: 0;
	margin: 0;
}
ul li > a {
	text-decoration: none;
	color: gray;
}
ul li:hover, ul li:focus {
	background: #F0F0F0;
}

ul li:first-child span {
  border-left:none;  
}

</style>
<html>
<head>
<meta charset="utf-8">
<title>TF properties - HNS</title>
<meta name="theme-color" content="#ffffff">
</head>
<body>
	<div class="banner">
		<center><font color="white" size="16px" face="arial">HNS</font></center>
	</div>
	<div align="center" class="main-content">
		<div class="spacer">
	</div><br>			<label class="collapse_ACT" for="_1"><font size="5px">HNS transcriptional activity</font></label>
				<input id="_1" type="checkbox">
				<div class="ACT"><br><ul><li>The expression of yciG is highly repressed in the Dhns + pHNS strain , its transcript levels were lower than the detection limit of the Q-PCR cycler and could not be reported with confidence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25375226" target="_blank">25375226</a>]<br></li>
<li>This again suggests a mechanism where Fur binding might relieve HNS-dependent transcription repression , or HNS binding might initiate nucleation of the protein , and repress Fur-dependent activation of gene expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23040276" target="_blank">23040276</a>]<br></li>
<li>expression is heavily regulated by presence of nucleation sites for HNS silencing .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23040276" target="_blank">23040276</a>]<br></li>
<li>Global regulation of gene expression by HNS and Fur Since SPI1 is a foreign fragment of DNA acquired by Salmonella during its evolution , it is understandable that it is subject to a heavy negative regulation [ 89 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23040276" target="_blank">23040276</a>]<br></li>
<li>Part of this negative regulation seems to be played by HNS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23040276" target="_blank">23040276</a>]<br></li>
<li>Plasmid pSR660 , expressing LexADBD , and plasmid pSR658-HNS , expressing LexADBD-H-NS fusion protein whose dimerization capacity has been tested in this system before ( 29 ) , were used as negative and positive controls , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24187088" target="_blank">24187088</a>]<br></li>
<li>Three proteins involved in stress response or global gene regulation again demonstrated increased expression in all three triclosan-resistant mutants ( Table 2 ) including HNS , HtrA ( involved in response to heat-shock ) and CspC ( involved in response to cold-shock ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18388111" target="_blank">18388111</a>]<br></li>
<li>Both are known to regulate virulence factors in Salmonella ( Fur positively , and HNS negatively ) [ 17,22,95 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23040276" target="_blank">23040276</a>]<br></li>
				</div></ul><br>
			<label class="collapse_EVO" for="_3"><font size="5px">HNS evolution</font></label>
				<input id="_3" type="checkbox">
				<div class="EVO"><br><ul><li>HNS belongs to a family of small nucleoid-associated proteins in gram-negative bacteria [ 29,30 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23040276" target="_blank">23040276</a>]<br></li>
				</div></ul><br>
			<label class="collapse_RP" for="_4"><font size="5px">Processes regulated by HNS</font></label>
				<input id="_4" type="checkbox">
				<div class="RP"><br><ul><li>Proteins with signiﬁcantly altered expression in all three triclosan-resistant mutants Expression ratio a Description Protein L700 ( LoT ) L701 ( MeT ) L702 ( HiT ) average 4.4 ( 6.7 ) b 4.1 ( 4.1 ) 4.0 3.4 2.7 ( 6.7 ) 2.3 ( 2.5 ) 2.2 2.1 2.0 ( 15.0 ) 1.9 1.8 ( 0.98 ) 1.6 1.6 ( 1.5 ) 3.0 5.6 3.6 3.0 3.7 2.0 2.0 2.5 2.3 2.1 1.7 1.8 1.7 1.6 7.0 2.2 2.3 3.3 1.5 3.0 2.4 1.6 2.1 1.4 1.3 1.6 1.3 1.4 putative arginine deiminase ( arcA ) STM4467 GcvP HtrA HemL PpS FadB HNS AhpC MaeB HemX GltA CspC GapA TktA 3.3 4.6 6.2 4.0 2.8 2.0 2.3 2.3 1.7 2.3 2.3 1.5 1.9 1.8 glycine cleavage complex protein P , glycine decarboxylase periplasmic serine protease , heat shock protein glutamate-1-semialdehyde aminotransferase phosphoenolpyruvate synthase 3-hydroxyacyl-coA dehydrogenase DNA-binding protein , pleiotropic regulator alkyl hydroperoxide reductase putative transferase uroporphyrinogen methylase citrate synthase cold shock protein , putative regulator glyceraldehyde-3-phosphate dehydrogenase A transketolase 1 isozyme 1.6 1.6 1.8 1.5 1.8 1.5 1.6 1.9 0.5 0.3 0.2 0.4 0.2 1.5 1.7 1.5 1.7 1.6 1.6 1.5 0.6 0.5 0.3 0.6 1.5 1.5 1.4 1.5 1.3 0.5 0.5 0.3 0.5 0.5 0.2 putative pectinesterase glycerol kinase YbhC GlpK MdH 1.6 ( 0.95 ) 1.6 ( 4 ) 1.6 1.5 1.3 0.8 0.4 0.4 0.4 0.3 malate dehydrogenase 30S ribosomal subunit protein S1 protein chain elongation factor regulator of length of O-antigen component of LPS phage lambda receptor protein ; maltose high-afﬁnity receptor putative methyl-accepting chemotaxis protein nucleoside-transport putative carbon starvation protein sensory histidine protein kinase RpsA TufA WzzB LamB STM3152 NupC YjiJ CheA a Expression ratio relative to L696 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18388111" target="_blank">18388111</a>]<br></li>
<li>Selected proteins with differential expression in L700 ( LoT ) compared with L696 Expression ratio a Description Protein Function enoyl - [ acyl-carrier-protein ] reductase ( NADH ) 3-oxoacyl - [ acyl-carrier-protein ] synthase I FabI FabB Fatty acid biosynthesis/metabolism 3.8 2.2 3.0 2.4 2.1 1.6 7.0 2.2 0.3 0.3 0.2 0.2 0.3 0.2 3-hydroxyacyl-coA dehydrogenase DNA-binding protein ; pleiotropic regulator RNA chaperone , negative regulator of cspA transcription cold shock protein , putative regulator putative arginine deiminase ( arcA ) glycine cleavage complex protein P , glycine decarboxylase ﬂagellar biosynthesis ; ﬂagellin , ﬁlament structural protein putative methyl-accepting chemotaxis protein methyl-accepting chemotaxis protein I , serine sensor receptor sensory histidine protein kinase putative methyl-accepting chemotaxis protein methyl-accepting chemotaxis protein II FadB HNS CspE CspC STM4467 GcvP FliC STM3216 Tsr CheA STM3152 CheM Global regulation Cold shock Amino acid metabolism Motility/chemotaxis a Ratio indicates expression in L700 ( LoT ) relative to L696 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18388111" target="_blank">18388111</a>]<br></li>
<li>These predicted Fur or HNS regulated genes include genes associated with virulence as well those not related to virulence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23040276" target="_blank">23040276</a>]<br></li>
<li>In addition , there are additional HNS nucleation sites present in genes which encode for chaperones , effector proteins , and other related invasion proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23040276" target="_blank">23040276</a>]<br></li>
<li>These results indicate a complex interplay between Fur and HNS in regulating cellular global behavior .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23040276" target="_blank">23040276</a>]<br></li>
<li>The results suggest for a complex interplay between Fur and HNS to globally control the cellular behavior .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23040276" target="_blank">23040276</a>]<br></li>
<li>Other regulators implicated in control of ﬂhDC include catabolite gene activator protein ( 48 , 74 ) , LrhA ( 51 ) and HNS , via HdfR ( 47 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12923082" target="_blank">12923082</a>]<br></li>
<li>Three proteins involved in stress response or global gene regulation again demonstrated increased expression in all three triclosan-resistant mutants ( Table 2 ) including HNS , HtrA ( involved in response-to-heat-shock ) and CspC ( involved in response-to-cold shock ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18388111" target="_blank">18388111</a>]<br></li>
<li>Modulation of acid-induced amino acid decarboxylase gene-expression by HNS in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25566242" target="_blank">25566242</a>]<br></li>
<li>HNS controls expression of ﬁmbriae , ﬂagella , SPI1 , SPI2 , and SPI5 loci in S. typhimurium HNS is a nucleoid binding protein that silences gene-expression by shielding DNA from RNAP and transcription factors [ 80,81 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23040276" target="_blank">23040276</a>]<br></li>
<li>In terms of pathogenesis in Salmonella , HNS is known to negatively regulate expression of the genes involved in invasion [ 31 -- 34 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23040276" target="_blank">23040276</a>]<br></li>
<li>Two of the global regulators involved in controlling gene-expression during the infection cycle of the bacterium are Fur and HNS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23040276" target="_blank">23040276</a>]<br></li>
<li>Two of the global regulators involved in control of gene-expression are Fur and HNS [ 17,18 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23040276" target="_blank">23040276</a>]<br></li>
<li>Contents lists available at SciVerse ScienceDirect Computers in Biology and Medicine journal homepage : www.elsevier.com/locate/cbm Interplay between Fur and HNS in controlling virulence gene expression in Salmonella typhimurium Mahendra Kumar Prajapat a , Supreet Saini b , n a Chemical Engineering , Indian Institute of Technology Gandhinagar , VGEC Campus , Chandkheda , Ahmedabad , Gujarat 382424 , India b Chemical Engineering , Indian Institute of Technology Bombay , Powai , Mumbai , Maharashtra 400067 , India abstract article info Salmonella enterica is responsible for a large number of diseases in a wide-range of hosts .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23040276" target="_blank">23040276</a>]<br></li>
<li>Interplay between Fur and HNS in controlling virulence gene-expression in Salmonella typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24643535" target="_blank">24643535</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27185788" target="_blank">27185788</a>]<br></li>
<li>Interplay between Fur and HNS for global regulation of Salmonella gene-expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23040276" target="_blank">23040276</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23040276" target="_blank">23040276</a>]<br></li>
				</div></ul><br>
			<label class="collapse_SIT" for="_5"><font size="5px">HNS binding site</font></label>
				<input id="_5" type="checkbox">
				<div class="SIT"><br><ul><li>HNS box sequence HNS box location S. no .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23040276" target="_blank">23040276</a>]<br></li>
<li>In addition , we also located a number of Fur and HNS binding sites in close proximity to accessory regulators hilC , sprB , and hilA [ 17,19,33,46,47 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23040276" target="_blank">23040276</a>]<br></li>
<li>While HNS binding of the SPI1 region has been previously identiﬁed as a silencing mechanism [ 89 ] , the role of presence of Fur binding sites in the SPI1s ; and HNS binding in the SPI2 , SPI3 , and SPI4 regions has not been previously reported .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23040276" target="_blank">23040276</a>]<br></li>
<li>A Species Dataset Number of Average Total % A+T Source Sequences Size Size ( bp ) ( bp ) Salmonella LeuO + Typhimurium HNS 50.5 This study Escherichia E. coli coli LeuO 59.9 Shimada et al. ( 2011 ) bits bits B 2 n = 50 R = 20.7 E. coli 1 1 2 2 A A T A T 3 3 4 5 4 5 T 6 7 8 9 A ATAT AAAT AATT A T A T AT TTTTATC AT T TA C A G C T A TA C T C GGGCT G 0 CG A A A C G A GT A G A C CC G C CGG 1 C A T 6 7 8 9 10 11 10 11 T N11 A 12 12 2 n = 36 S. Typhimurium 13 13 R = 16.8 14 14 A 15 15 A 16 16 A 1 TTATAT A A T T TAAA AA T A AA AAAGCGTGTTG GTG TA C A C AGAT T 0 GGCG CAG C G C A G A GT CCG C T G T C C TGT A TG A GC CCC A TG GT G C C GG 17 18 19 20 21 17 18 19 20 21 A. Details of the manually curated S. Typhimurium and E. coli LeuO datasets used to derive the LeuO binding motifs .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22804842" target="_blank">22804842</a>]<br></li>
<li>HNS box sequence HNS box location Strand þ þ 3006305y3006313 3010825y3010833 3011464y3011472 3012245y3012253 3013028y3013036 3018129y3018137 3018934y3018942 3019190y3019198 3019673y3019681 3020913y3020921 3023611 ... 3023619 3023757y3023765 3023758y3023766 3023975y3023983 3027358y3027366 3027433y3027441 3031507y3031515 3032259y3032267 3034613y3034621 3036799y3036807 3038496y3038504 3038724y3038732 3038966y3038974 3042887y3042895 3044398y3044406 3045587y3045595 1463152y 1465356y 1473069y 1473917y 1475863y 1479318y 1479477y 1479846y 1480623y 1484296y 1484705y 1485569y 1486355y 1488147y 1489155y 1489392y 1489509y 1491640y 1493907y 1494864y 1495827y 1497681y 1500166y 1500389y 1500827y GCTATATAA AATATATTG GCTATATTA AAAATATCG GCAATATAA GCTATATGA AATATATTG GCTATATTA GCTAAATAA AATACATCG GCAATATAA GCGATATAA GCTATATTA AAAATATCG GCTATATAA GCTAAATAA 1 2 1 2 GCTATAAAA GCGATATAA GCTATGTAA AATATATTG GCTGTATAA AAAATATCG GATATATAA AATTTATCG GCTATAAAA AATATACCG GCTATATAC AATATACCG GCTTTATAA GCTATGTAA AATATAACG ATTATATCG 1463160 1465364 1473077 1473925 1475871 1479326 1479485 1479854 1480631 1484304 1484713 1485577 1486363 1488155 1489163 1489400 þ 3 4 5 6 7 8 9 10 11 12 13 14 15 16 3 4 5 6 7 8 9 10 11 12 13 14 15 16 þ þ þ þ þ þ þ þ þ þ þ þ þ þ AATACATCG GCTATTTAA GCTATACAA GCTATAAAA AATAAATCG AATATAACG GCTTTATAA GTTATATAA AAAATATCG GCTATACAA 17 18 19 20 21 22 23 24 25 26 17 18 19 20 21 22 23 24 25 GCTACATAA GCTTTATAA ATTATATCG GCTATATCA GCTGTATAA GGTATATAA AATATATCC GCTATTTAA AATGTATCG 1489517 1491648 1493915 1494872 1495835 1497689 1500174 1500397 1500835 þ þ þ þ þ þ þ þ Frequency of Binding Sites 2.2 2.0 1.8 1.6 1.4 1.2 1.0 0.8 0.6 0.4 Fur HNS SPI2 SPI1 SPI3 SPI4 0.2 0.0 0 1x106 2x106 3x106 4x106 5x106 Base Number Fig. 2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23040276" target="_blank">23040276</a>]<br></li>
				</ul></div><br>
			<label class="collapse_TU" for="_6"><font size="5px">Transcription units regulated by HNS</font></label>
				<input id="_6" type="checkbox">
				<div class="TU"><br><ul><li>Both are known to regulate virulence factors in Salmonella ( Fur positively , and HNS negatively ) [ 17,22,95 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23040276" target="_blank">23040276</a>]<br></li>
<li>Modulation of acid-induced amino acid decarboxylase gene expression by HNS in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25566242" target="_blank">25566242</a>]<br></li>
<li>Global regulation of gene expression by HNS and Fur Since SPI1 is a foreign fragment of DNA acquired by Salmonella during its evolution , it is understandable that it is subject to a heavy negative regulation [ 89 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23040276" target="_blank">23040276</a>]<br></li>
<li>Global regulation of gene expression by HNS and Fur 4 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23040276" target="_blank">23040276</a>]<br></li>
<li>The effect of vector pSR660 ( negative control ) , expressing the LexA DNA-binding domain ( LexADBD ) , and plasmids pSR658-HNS ( positive control ) or pSR660-SirA , expressing fusion proteins LexADBD-HNS and LexADBD-SirA , respectively , on the expression of sulA : : lacZ fusion was determined in the E. coli SU101 reporter strain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24187088" target="_blank">24187088</a>]<br></li>
<li>The expression of yciG is highly repressed in the Dhns + pHNS strain , its transcript levels were lower than the detection limit of the Q-PCR cycler and could not be reported with confidence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25375226" target="_blank">25375226</a>]<br></li>
				</div></ul></div><br>
	</div>
</body>
</html>