<!doctype html>
<style>
@charset "utf-8";
body {
	height: 100%;
	margin: 0;
	padding: 0;
}

.banner {
	position: fixed;
	top: 0;
	width: 100%;
	height: 65px;
	background: gray;
	line-height: 65px;
}
.spacer {
	height: 10px;
}

.main-content {
	padding: 5%;
}

.collapse_ACT {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-style: none solid none none;
	border-color: #B4045F;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_ACT + input{
	display: none;
}
.collapse_ACT + input + div{
	display: none;
}
.collapse_ACT + input:checked + div{
	display: block;
}
.ACT {
	text-align: justify;
	text-justify: inter-word;
}
.ACT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B4045F;
}
.ACT_collapse_similarity:hover{
	background: #e5d3dc;
}
.ACT_collapse_similarity + input {
	display: none;
}
.ACT_collapse_similarity + input + div{
	display: none;
}

.ACT_collapse_similarity + input:checked + div{
	display: block;
}
.ACT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.ACT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B4045F;
	padding-left: 5px;
}
.ACT_collapse_similarity:hover{
	background: #e5d3dc;
}
.ACT_collapse_similarity + input {
	display: none;
}
.ACT_collapse_similarity + input + div{
	display: none;
}
.ACT_collapse_similarity + input:checked + div{
	display: block;
}
.ACT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.ACT_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #B4045F;
}
.ACT_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_DOM {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-style: none solid none none;
	border-color: #B404AE;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_DOM + input{
	display: none;
}
.collapse_DOM + input + div{
	display: none;
}
.collapse_DOM + input:checked + div{
	display: block;
}
.DOM {
	text-align: justify;
	text-justify: inter-word;
}
.DOM_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B404AE;
	padding-left: 5px;
}
.DOM_collapse_similarity:hover{
	background: #e8d7e7;
}
.DOM_collapse_similarity + input {
	display: none;
}
.DOM_collapse_similarity + input + div{
	display: none;
}
.DOM_collapse_similarity + input:checked + div{
	display: block;
}
.DOM_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}.DOM_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #B404AE;
}
.DOM_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_EVO {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #7401DF;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_EVO + input{
	display: none;
}
.collapse_EVO + input + div{
	display: none;
}
.collapse_EVO + input:checked + div{
	display: block;
}
.EVO {
	text-align: justify;
	text-justify: inter-word;
}
.EVO_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #7401DF;
	padding-left: 5px;
}
.EVO_collapse_similarity:hover{
	background: #e2d7ed;
}
.EVO_collapse_similarity + input {
	display: none;
}
.EVO_collapse_similarity + input + div{
	display: none;
}
.EVO_collapse_similarity + input:checked + div{
	display: block;
}
.EVO_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.EVO_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #7401DF;
}
.EVO_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_RP {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #0174DF;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_RP + input{
	display: none;
}
.collapse_RP + input + div{
	display: none;
}
.collapse_RP + input:checked + div{
	display: block;
}
.RP {
	text-align: justify;
	text-justify: inter-word;
}
.RP_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #0174DF;
	padding-left: 5px;
}
.RP_collapse_similarity:hover{
	background: #c9d7e2;
}
.RP_collapse_similarity + input {
	display: none;
}
.RP_collapse_similarity + input + div{
	display: none;
}
.RP_collapse_similarity + input:checked + div{
	display: block;
}
.RP_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.RP_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #0174DF;
}
.RP_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_SIT {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #0489B1;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_SIT + input{
	display: none;
}
.collapse_SIT + input + div{
	display: none;
}
.collapse_SIT + input:checked + div{
	display: block;
}
.SIT {
	text-align: justify;
	text-justify: inter-word;
}
.SIT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #0489B1;
	padding-left: 5px;
}
.SIT_collapse_similarity:hover{
	background: #d0dfe2;
}
.SIT_collapse_similarity + input {
	display: none;
}
.SIT_collapse_similarity + input + div{
	display: none;
}
.SIT_collapse_similarity + input:checked + div{
	display: block;
}
.SIT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.SIT_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #0489B1;
}
.SIT_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_TU {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #04B486;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_TU + input{
	display: none;
}
.collapse_TU + input + div{
	display: none;
}
.collapse_TU + input:checked + div{
	display: block;
}
.TU {
	text-align: justify;
	text-justify: inter-word;
}
.TU_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #04B486;
	padding-left: 5px;
}
.TU_collapse_similarity:hover{
	background: #d0e2de;
}
.TU_collapse_similarity + input {
	display: none;
}
.TU_collapse_similarity + input + div{
	display: none;
}
.TU_collapse_similarity + input:checked + div{
	display: block;
}
.TU_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.TU_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #04B486;
}
.TU_similar > a {
	text-decoration: none;
	color: gray;
}

ul { 
	list-style-type: none;
	padding: 0;
	margin: 0;
}
ul li > a {
	text-decoration: none;
	color: gray;
}
ul li:hover, ul li:focus {
	background: #F0F0F0;
}

ul li:first-child span {
  border-left:none;  
}

</style>
<html>
<head>
<meta charset="utf-8">
<title>TF properties - IscR</title>
<meta name="theme-color" content="#ffffff">
</head>
<body>
	<div class="banner">
		<center><font color="white" size="16px" face="arial">IscR</font></center>
	</div>
	<div align="center" class="main-content">
		<div class="spacer">
	</div><br>			<label class="collapse_ACT" for="_1"><font size="5px">IscR transcriptional activity</font></label>
				<input id="_1" type="checkbox">
				<div class="ACT"><br><ul><li>The conclusion was that all three regulators mediate Co induction , however clearly with different time-responses as IscR - , followed by Fur - , and OxyR-mediated responses occurred in this order .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21952637" target="_blank">21952637</a>]<br></li>
<li>Our results show that under iron replete conditions , IscR represses Spi1 expression 1.5 fold only ( WT vs. ΔiscR in LB ) , whereas under iron limited conditions , IscR exerts a fivefold repression effect ( WT vs. ΔiscR in LB + dip ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27704705" target="_blank">27704705</a>]<br></li>
<li>Hence , IscR is predicted to be mostly under its holo‐form and at low cellular concentrations , favoring Spi1 TTSS gene expression ( Figure S5 ; Schwartz et al. , 2001 ; Vinella , Loiseau , Ollagnier de Choudens , Fontecave , & Barras , number 13.118 to SM , Application number AR 1A09382857717 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27704705" target="_blank">27704705</a>]<br></li>
<li>In the model , IscR represses iscRSUA transcription when bound to a [ 2Fe-2S ] cluster , while Cra can activate iscRSUA expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19136587" target="_blank">19136587</a>]<br></li>
<li>The combined absence of Cra and IscR derepresses PiscRlacZ expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19136587" target="_blank">19136587</a>]<br></li>
<li>When incubated with increasing concentrations of purified His6‐IscR , the PhilD probe shifted gradually ( Figure 3c ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27704705" target="_blank">27704705</a>]<br></li>
<li>ns , not significant ; * p ≤ 0.05 ( Mann -- Whitney U‐test ) FIGURE 3 IscR binds to the hilD promoter in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27704705" target="_blank">27704705</a>]<br></li>
<li>The iscR mutation led to a 5.5 fold increase in hilD mRNA level compared to the wild‐type strain , confirming that IscR acts as a repressor of hilD ( Figure 4e ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27704705" target="_blank">27704705</a>]<br></li>
<li>Moreover , recent studies have shed new light on the role of IscR by expanding its zone of influence beyond the usual Fe -- S cluster containing proteins ; namely , in enterotoxigenic E. coli , IscR was shown to-bind upstream cfaA and to activate CFA/I fimbriae production in response to iron starvation ( Haines et al. , 2015 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27704705" target="_blank">27704705</a>]<br></li>
<li>IscR regulates synthesis of colonization factor antigen i fimbriae in response to iron starvation in enterotoxigenic Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27704705" target="_blank">27704705</a>]<br></li>
<li>FIGURE 5 Apo‐IscR and holo‐IscR negatively regulates the expression of Spi1 TTSS genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27704705" target="_blank">27704705</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27704705" target="_blank">27704705</a>]<br></li>
<li>So going from iron limited to iron replete conditions will shift the equilibrium from apo‐IscR to holo‐IscR , thereby repressing iscR and inducing Spi1 gene expression by alleviating IscR repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27704705" target="_blank">27704705</a>]<br></li>
<li>Under such environmental conditions -- iron starvation and oxidative-stress -- the apo‐IscR form is predicted to dominate , decreasing Spi1 TTSS gene expression and limiting energetic expenses for the pathogen during macrophage infection .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27704705" target="_blank">27704705</a>]<br></li>
<li>For instance , a D. dadantii iscR mutant is non‐invasive on Saintpaulia ionantha plants because the SUF system can not be activated by IscR , but it presents a wild‐type phenotype on Arabidopsis thaliana presumably because the demand in Fe -- S clusters to be satisfied for the bacte-rium to grow is met by ISC in this case ( Rincon‐Enriquez et al. , 2008 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27704705" target="_blank">27704705</a>]<br></li>
<li>This suggests that in response to H2O2 under aerobic conditions , ArcA may act together with Apo-IscR , up-regulating the nrdEFHI operon Figure 4 Model showing the major differences in the processes regulated by ArcA after H2O2 exposure under aerobic conditions , aerobiosis and anaerobiosis in S. Typhimurium 14028s .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24044554" target="_blank">24044554</a>]<br></li>
<li>A genetic screen led to the identiﬁcation of the transcriptional-regulator IscR , which modulates gene transcription depending on the coordination of its 2Fe -- 2S clusters which can be inﬂuenced by oxidative-stress , oxygen limitation , or iron availability ( Miller et al. , 2014 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26441883" target="_blank">26441883</a>]<br></li>
<li>It is known that the transcriptional-repressor of the isc operon , IscR , needs a functional [ Fe-S ] cluster for its activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14729702" target="_blank">14729702</a>]<br></li>
<li>Indeed , this operon is controlled by 15 IscR , Fur and OxyR , which are the Fe -- S homeostasis - , the iron - and the H2O2-sensing regulators , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21952637" target="_blank">21952637</a>]<br></li>
<li>In iscS and hscA mutants ( 56 ) as well as in iscU and fdx mutants ( Fig. 5 ; Table 3 ) , the assembly of [ Fe-S ] clusters is signiﬁcantly reduced and therefore IscR loses its repressing abilities , resulting in increased expression of the isc operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14729702" target="_blank">14729702</a>]<br></li>
<li>In the hilD iscR mutant , the rtsA , prgH , and invF mRNA levels were 360‐fold , 60‐fold , and 460‐fold lower compared to the wild‐type strain , showing that the expression of rtsA , prgH , and invF is totally abrogated in the absence of both iscR and hilD and suggesting a possible Spi1 gene expression reprogramming process in this back ¬ FIGURE 4 IscR represses hilD gene expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27704705" target="_blank">27704705</a>]<br></li>
<li>Hence , under iron limitation or in the presence of reactive oxygen species ( ROS ) , IscR exists primarily under its apo‐form that activates expression of SUF and no longer represses expression of isc , the amount of Fe -- S clusters produced being at its highest level .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27704705" target="_blank">27704705</a>]<br></li>
<li>Further studies are required to characterize the physiological significance of the TTSS regulation by IscR in both species .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27704705" target="_blank">27704705</a>]<br></li>
<li>In particular , we showed that IscR represses the synthesis of the Spi1‐carried regulator HilD , decreasing the expression of genes encoding the TTSS apparatus .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27704705" target="_blank">27704705</a>]<br></li>
<li>When incubated with increasing concentrations of IscR , the resulting DNA fragment , called PhilD * , displayed a more modest shift compared to PhilD , highlighting the importance of thes conserved nucleotides in the interaction between IscR and the promoter of hilD ( Figure 3d , e ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27704705" target="_blank">27704705</a>]<br></li>
<li>In the transcriptomic analysis , IscR was found to be crucial in informing the cells of the presence of Co as several of the Co up-regulated genes were no longer induced 15 in the absence of iscR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21952637" target="_blank">21952637</a>]<br></li>
<li>This is , however , the ﬁrst evidence of a regulator , other than IscR , involved in the regulation of the iscRSUA operon and the ﬁrst evidence that the assembly of [ Fe-S ] clusters is activated in response to sources of carbon and energy whose catabolism is under the control of Cra .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19136587" target="_blank">19136587</a>]<br></li>
<li>As IscR maturation is catalyzed by the ISC system , a defect in this latter leads to the accumulation of apo‐IscR and activation of the SUF system .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27704705" target="_blank">27704705</a>]<br></li>
<li>However , in response to H2O2 , this Fur repression is abolished and the apoprotein form of IscR upregulates expression of the operon [ 44 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24044554" target="_blank">24044554</a>]<br></li>
				</div></ul><br>
			<label class="collapse_DOM" for="_2"><font size="5px">IscR protein domains</font></label>
				<input id="_2" type="checkbox">
				<div class="DOM"><br><ul><li>Finally , to confirm the direct interaction between IscR and PhilD , DNA‐protein interaction experiments were carried out using Bio‐layer interferometry , a technique previously used to study protein -- protein interactions ( Arlet et al. , 2014 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27704705" target="_blank">27704705</a>]<br></li>
<li>Differentially expressed genes in LT2 and PT4 under the chlorine treatmentsa Fold change Function and gene Annotation 130 ppm , 30 min 390 ppm , 10 min LT2 PT4 LT2 PT4 Fe-S cluster formation and cysteine synthesis/transport protein Chaperone protein HscA Cochaperone HscB 1.59 1.56 2.07 3.45 1.81 1.16 1.06 1.07 1.09 1.15 1.14 1.61 3.31 3.49 3.45 1.47 1.93 2.28 2.09 1.72 2.05 1.71 1.52 2.69 2.72 1.16 1.25 1.84 3.45 1.39 1.06 1.06 1.02 1.02 1.04 1.05 1.34 3.04 2.92 3.45 1.81 1.56 2.90 2.99 1.66 2.06 1.92 1.44 2.31 3.71 1.12 1.23 1.17 1.61 1.19 1.04 1.24 1.25 1.16 1.14 1.24 1.16 1.26 1.29 1.39 1.10 1.35 1.25 1.18 1.28 1.14 1.37 1.23 1.58 1.23 1.09 1.26 1.20 1.39 1.10 1.03 1.06 1.11 1.16 1.02 1.13 1.16 1.57 1.23 1.61 1.11 1.20 1.45 1.44 1.12 1.32 1.38 1.11 1.37 1.42 hscA hscB iscA iscR/yfhP fdx sufE/ynhA sufS sufD sufC sufB sufA yfhJ nifS nifU yfhP sbp cysC cysN cysD cysH cysI cysJ cysA cysP cysK Iron-sulfur cluster assembly protein DNA-binding transcriptional regulator IscR Electron carrier protein Cysteine-desulfuration-protein-SufE Bifunctional cysteine desulfurase/selenocysteine lyase Cysteine-desulfurase activator complex subunit SufD Cysteine-desulfurase ATPase component Cysteine-desulfurase activator complex subunit SufB Iron-sulfur cluster assembly scaffold protein Hypothetical protein Cysteine-desulfurase NifU-like protein Putative iron-sulfur cluster regulatory protein Sulfate transport protein Adenylylsulfate kinase Sulfate adenylyltransferase subunit 1 Sulfate adenylyltransferase subunit 2 Phosphoadenosine phosphosulfate reductase Sulﬁte reductase subunit beta Sulﬁte reductase subunit alpha Sulfate/thiosulfate transporter subunit Thiosulfate transport protein Cysteine-synthase A Phosphotransferase system manX manY manZ ptsH ptsI ptsG Mannose-speciﬁc enzyme IIAB Mannose-speciﬁc enzyme IIC 1.64 1.72 1.31 1.56 1.47 1.52 2.71 2.47 1.76 2.26 1.92 1.74 2.32 2.01 1.49 2.35 2.11 1.52 3.36 2.50 1.83 2.62 2.12 1.72 Mannose-speciﬁc enzyme IID Phosphohistidinoprotein-hexosephosphotransferase PEP-protein phosphotransferaseb Glucose-speciﬁc IIBC component Cell envelope ompF 1.63 1.33 2.52 1.69 1.02 1.50 1.81 1.71 1.77 1.65 1.68 1.53 1.13 1.25 1.65 3.19 1.25 1.07 1.57 1.89 1.87 1.66 1.54 1.47 1.19 1.08 Outer-membrane-protein F precursor 1.32 1.23 3.56 1.10 1.82 1.74 1.86 1.78 1.86 1.52 1.54 1.44 1.02 2.03 1.45 3.21 1.41 1.65 1.81 1.75 1.62 1.88 1.71 1.58 1.48 1.54 Putative hydrogenase membrane component precurosr Putative outer membrane porin precursor Outer-membrane-protein C precursor Flagellin Flagellar basal body rod protein Flagellar basal body rod protein Flagellar basal body rod modiﬁcation protein Flagellar hook protein Cell-proximal portion of basal body rod Flagellar basal body rod protein Flagellar biosynthesis sigma factor FliA Flagellar biosynthesis protein ompA nmpC ompC ﬂjB ﬂgB ﬂgC ﬂgD ﬂgE ﬂgF ﬂgG ﬂiA ﬂiR Nitrosative-stress-sensitive proteins atpA fabB fabF gapA tuf gltA 2.20 1.20 1.91 2.01 1.58 1.26 1.86 1.01 1.52 2.04 1.72 1.27 2.09 1.19 1.93 1.41 1.39 1.49 1.53 2.08 1.24 1.13 1.17 1.55 ATP synthase subunit A 3-Oxoacyl - ( acyl carrier protein ) synthase 3-Oxoacyl - ( acyl carrier protein ) synthase Glyceraldehyde-3-phosphate dehydrogenase Elongation factor Tu Citrate-synthase Metabolic genes atpD 1.36 1.29 1.46 1.51 ATP synthase subunit B ATP synthase subunit B ATP synthase subunit C ATP synthase subunit D 1.92 1.95 2.13 2.38 1.39 1.57 1.63 1.80 1.97 1.76 2.04 2.16 atpF atpG atpH Continued on following pag TABLE 4 -- Continued Fold change Function and gene Annotation 130 ppm , 30 min 390 ppm , 10 min LT2 PT4 LT2 PT4 Maltoporin precursor lamB malE malM malK malP malQ pta 2.31 1.59 1.88 2.04 1.74 1.53 2.28 1.17 1.12 1.16 1.13 1.05 1.05 1.67 2.13 1.51 1.81 1.86 1.65 1.46 1.82 1.00 1.02 Periplasmic maltose-binding protein Periplasmic protein precursor Maltose transport protein/repressor Maltodextrin phosphorylase 4-Alpha-glucanotransferase Phosphate acetyltransferase 1.03 1.04 1.02 1.72 Regulatory genes dksA yﬁA DnaK suppressor protein Ribosome stabilization factor 1.58 2.04 1.21 1.58 1.10 1.14 1.04 1.02 Bioﬁlm formation bssS ycfR Bioﬁlm formation regulatory protein BssS Putative-outer-membrane-protein 1.57 2.76 1.37 2.30 1.07 1.44 1.04 1.54 Biosynthesis and degradation of surface polysaccharides and lipopolysaccharides rfbH rfbF rfbC rfbA rfaJ CDP-6-deoxy-D-xylo-4-hexulose-3-dehydrase Glucose-1-phosphate-cytidylyltransferase dTDP-4 , deoxyrhamnose-3 ,5 - epimerase dTDP-glucose pyrophosphorylase Lipopolysaccharide glucosyltransferase 1.30 1.40 1.18 1.09 1.10 1.45 1.46 1.20 1.13 1.02 1.56 1.73 1.52 1.37 1.26 1.68 1.69 1.55 1.52 1.50 a Differential expression characterized by 1.5-fold expression ( P 0.05 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20562293" target="_blank">20562293</a>]<br></li>
<li>IscR binding motifs are shaded ( 15 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19136587" target="_blank">19136587</a>]<br></li>
<li>The plasmids expressing IscR variant ( piscR‐3CA and pHis6‐iscR‐3CA ) were PCR amplified in two steps using two central oligonucleotides carrying the mutations ( C92A/C98A/C104A ) , the iscR‐3CA fw , and iscR‐3CA‐rv oli-gonucleotides ( Table S4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27704705" target="_blank">27704705</a>]<br></li>
<li>Apo‐IscR binds on type 2 DNA‐binding motifs , and such motifs were found in the promoter region of hilD ( Figure S6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27704705" target="_blank">27704705</a>]<br></li>
<li>Comparison of these two sites with the IscR sites from E. coli hyaA and Y. pseudotuberculosis lcrF shows that they contain five and four of the nine bases in this binding motif , respectively ( Figure 3b ; Giel et al. , 2006 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27704705" target="_blank">27704705</a>]<br></li>
<li>The four substitutions were located in the predicted helix-turn-helix DNA-binding domain of IscR ( 49 ) and were expected to generate inactive proteins and result in the constitutive expression of the isc operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18441067" target="_blank">18441067</a>]<br></li>
<li>Only three amino acids differ between E. coli and Salmonella IscR proteins ( 98 % identity ) , and none of these residues are in the DNA-binding domain , supporting the idea that identical motifs can be recognized by both proteins ( Rajagopalan et al. , 2013 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27704705" target="_blank">27704705</a>]<br></li>
<li>Based on the similarity of NsrR to IscR , which contains an [ Fe -- S ] cluster ( Schwartz et al. , 2001 ) , and other members of the Rrf2-family ( discussed in Rodionov et al. , 2005 ) , it has been suggested ( Bodenmiller & Spiro , 2006 ) that NsrR contains an NO-sensitive [ Fe -- S ] cluster .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17526833" target="_blank">17526833</a>]<br></li>
<li>This IscR variant has all three cluster‐coordinating cysteine residues ( Cys92 , Cys98 , and Cys104 ) converted into alanine , which prevents binding of the [ 2Fe -- 2S ] cluster ( Rajagopalan et al. , 2013 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27704705" target="_blank">27704705</a>]<br></li>
				</div></ul><br>
			<label class="collapse_EVO" for="_3"><font size="5px">IscR evolution</font></label>
				<input id="_3" type="checkbox">
				<div class="EVO"><br><ul><li>It is a member of the Rrf2 family of transcriptional regulators , which also includes the IscR regulator that is involved in regulation of genes involved in [ Fe -- S ] cluster biogenesis ( Schwartz et al. , 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17526833" target="_blank">17526833</a>]<br></li>
<li>Based on the similarity of NsrR to IscR , which contains an [ Fe -- S ] cluster ( Schwartz et al. , 2001 ) , and other members of the Rrf2 family ( discussed in Rodionov et al. , 2005 ) , it has been suggested ( Bodenmiller & Spiro , 2006 ) that NsrR contains an NO-sensitive [ Fe -- S ] cluster .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17526833" target="_blank">17526833</a>]<br></li>
				</div></ul><br>
			<label class="collapse_RP" for="_4"><font size="5px">Processes regulated by IscR</font></label>
				<input id="_4" type="checkbox">
				<div class="RP"><br><ul><li>Differentially expressed genes in LT2 and PT4 under the chlorine treatmentsa Fold change Function and gene Annotation 130 ppm , 30 min 390 ppm , 10 min LT2 PT4 LT2 PT4 Fe-S cluster formation and cysteine-synthesis/transport protein Chaperone protein HscA Cochaperone HscB 1.59 1.56 2.07 3.45 1.81 1.16 1.06 1.07 1.09 1.15 1.14 1.61 3.31 3.49 3.45 1.47 1.93 2.28 2.09 1.72 2.05 1.71 1.52 2.69 2.72 1.16 1.25 1.84 3.45 1.39 1.06 1.06 1.02 1.02 1.04 1.05 1.34 3.04 2.92 3.45 1.81 1.56 2.90 2.99 1.66 2.06 1.92 1.44 2.31 3.71 1.12 1.23 1.17 1.61 1.19 1.04 1.24 1.25 1.16 1.14 1.24 1.16 1.26 1.29 1.39 1.10 1.35 1.25 1.18 1.28 1.14 1.37 1.23 1.58 1.23 1.09 1.26 1.20 1.39 1.10 1.03 1.06 1.11 1.16 1.02 1.13 1.16 1.57 1.23 1.61 1.11 1.20 1.45 1.44 1.12 1.32 1.38 1.11 1.37 1.42 hscA hscB iscA iscR/yfhP fdx sufE/ynhA sufS sufD sufC sufB sufA yfhJ nifS nifU yfhP sbp cysC cysN cysD cysH cysI cysJ cysA cysP cysK Iron-sulfur cluster assembly protein DNA-binding transcriptional regulator IscR Electron carrier protein Cysteine desulfuration protein SufE Bifunctional cysteine desulfurase/selenocysteine lyase Cysteine desulfurase activator complex subunit SufD Cysteine desulfurase ATPase component Cysteine desulfurase activator complex subunit SufB Iron-sulfur cluster assembly scaffold protein Hypothetical protein Cysteine desulfurase NifU-like protein Putative iron-sulfur cluster regulatory protein Sulfate transport protein Adenylylsulfate kinase Sulfate adenylyltransferase subunit 1 Sulfate adenylyltransferase subunit 2 Phosphoadenosine phosphosulfate reductase Sulﬁte reductase subunit beta Sulﬁte reductase subunit alpha Sulfate/thiosulfate-transporter subunit Thiosulfate-transport protein Cysteine synthase A Phosphotransferase system manX manY manZ ptsH ptsI ptsG Mannose-speciﬁc enzyme IIAB Mannose-speciﬁc enzyme IIC 1.64 1.72 1.31 1.56 1.47 1.52 2.71 2.47 1.76 2.26 1.92 1.74 2.32 2.01 1.49 2.35 2.11 1.52 3.36 2.50 1.83 2.62 2.12 1.72 Mannose-speciﬁc enzyme IID Phosphohistidinoprotein-hexosephosphotransferase PEP-protein phosphotransferaseb Glucose-speciﬁc IIBC component Cell envelope ompF 1.63 1.33 2.52 1.69 1.02 1.50 1.81 1.71 1.77 1.65 1.68 1.53 1.13 1.25 1.65 3.19 1.25 1.07 1.57 1.89 1.87 1.66 1.54 1.47 1.19 1.08 Outer membrane protein F precursor 1.32 1.23 3.56 1.10 1.82 1.74 1.86 1.78 1.86 1.52 1.54 1.44 1.02 2.03 1.45 3.21 1.41 1.65 1.81 1.75 1.62 1.88 1.71 1.58 1.48 1.54 Putative hydrogenase membrane component precurosr Putative outer membrane porin precursor Outer membrane protein C precursor Flagellin Flagellar basal body rod protein Flagellar basal body rod protein Flagellar basal body rod modiﬁcation protein Flagellar hook protein Cell-proximal portion of basal body rod Flagellar basal body rod protein Flagellar biosynthesis sigma factor FliA Flagellar biosynthesis protein ompA nmpC ompC ﬂjB ﬂgB ﬂgC ﬂgD ﬂgE ﬂgF ﬂgG ﬂiA ﬂiR Nitrosative-stress-sensitive proteins atpA fabB fabF gapA tuf gltA 2.20 1.20 1.91 2.01 1.58 1.26 1.86 1.01 1.52 2.04 1.72 1.27 2.09 1.19 1.93 1.41 1.39 1.49 1.53 2.08 1.24 1.13 1.17 1.55 ATP synthase subunit A 3-Oxoacyl - ( acyl carrier protein ) synthase 3-Oxoacyl - ( acyl carrier protein ) synthase Glyceraldehyde-3-phosphate dehydrogenase Elongation factor Tu Citrate synthase Metabolic genes atpD 1.36 1.29 1.46 1.51 ATP synthase subunit B ATP synthase subunit B ATP synthase subunit C ATP synthase subunit D 1.92 1.95 2.13 2.38 1.39 1.57 1.63 1.80 1.97 1.76 2.04 2.16 atpF atpG atpH Continued on following pag TABLE 4 -- Continued Fold change Function and gene Annotation 130 ppm , 30 min 390 ppm , 10 min LT2 PT4 LT2 PT4 Maltoporin precursor lamB malE malM malK malP malQ pta 2.31 1.59 1.88 2.04 1.74 1.53 2.28 1.17 1.12 1.16 1.13 1.05 1.05 1.67 2.13 1.51 1.81 1.86 1.65 1.46 1.82 1.00 1.02 Periplasmic maltose-binding protein Periplasmic protein precursor Maltose transport protein/repressor Maltodextrin phosphorylase 4-Alpha-glucanotransferase Phosphate acetyltransferase 1.03 1.04 1.02 1.72 Regulatory genes dksA yﬁA DnaK suppressor protein Ribosome stabilization factor 1.58 2.04 1.21 1.58 1.10 1.14 1.04 1.02 Bioﬁlm formation bssS ycfR Bioﬁlm formation regulatory protein BssS Putative outer membrane protein 1.57 2.76 1.37 2.30 1.07 1.44 1.04 1.54 Biosynthesis and degradation of surface polysaccharides and lipopolysaccharides rfbH rfbF rfbC rfbA rfaJ CDP-6-deoxy-D-xylo-4-hexulose-3-dehydrase Glucose-1-phosphate cytidylyltransferase dTDP-4 , deoxyrhamnose-3 ,5 - epimerase dTDP-glucose pyrophosphorylase Lipopolysaccharide glucosyltransferase 1.30 1.40 1.18 1.09 1.10 1.45 1.46 1.20 1.13 1.02 1.56 1.73 1.52 1.37 1.26 1.68 1.69 1.55 1.52 1.50 a Differential expression characterized by 1.5-fold expression ( P 0.05 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20562293" target="_blank">20562293</a>]<br></li>
<li>Received : 10 December 2015 Revised : 30 September 2016 Accepted : 30 September 2016 DOI 10.1111 / cmi .12680 RESEARCH ARTICLE The iron -- sulfur cluster sensor IscR is a negative regulator of Spi1 type-III-secretion-system in Salmonella enterica Alexandra Vergnes1  Julie P.M. Viala1 †  Rabah Ouadah‐Tsabet1  Bérengère Pocachard1  Laurent Loiseau1  Stéphane Méresse2  Frédéric Barras1  Laurent Aussel1 1 Aix Marseille Université , CNRS , LCB UMR , 7283 , IMM , Marseille , France 2 Aix Marseille Université , CNRS , INSERM , CIML , Marseille , France Correspondence Laurent Aussel , Laboratoire de Chimie Bactérienne , 31 Chemin Joseph Aiguier , 13402 Marseille Cedex 20 , France .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27704705" target="_blank">27704705</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_3">
IscR is essential for yersinia pseudotuberculosis type-III-secretion and virulence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27704705" target="_blank">27704705</a>]<br></label>
<input id="rp_similarity_3", type="checkbox">
<div class="RP_similar">
<li>IscR is essential for Yersinia pseudotuberculosis type-III-secretion and virulence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26441883" target="_blank">26441883</a>]<br></li></div>
				<label class="RP_collapse_similarity" for="rp_similarity_5">
IscR , an Fe-S cluster-containing transcription factor , represses expression of Escherichia coli genes encoding Fe-S cluster assembly proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19136587" target="_blank">19136587</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22831173" target="_blank">22831173</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14729702" target="_blank">14729702</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18441067" target="_blank">18441067</a>]<br></label>
<input id="rp_similarity_5", type="checkbox">
<div class="RP_similar">
<li>IscR , an Fe‐S cluster‐containing transcription factor , represses expression of Escherichia coli genes encoding Fe‐S cluster assembly proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27704705" target="_blank">27704705</a>]<br></li>
<li>IscR , an Fe -- S cluster-containing transcription factor , represses expression of Escherichia coli genes encoding Fe -- S cluster assembly proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17526833" target="_blank">17526833</a>]<br></li></div>
<li>Some Rrf2 family members , including NsrR , IscR , the regulator of iron-sulfur cluster biogenesis , and RirA , a regulator of iron metabolism in Rhizobium leguminosarum , incorporate iron-sulfur ( Fe-S ) clusters ( Johnston et al. , 2007 ; Schwartz et al. , 2001 ; Tucker et al. , 2008 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22831173" target="_blank">22831173</a>]<br></li>
<li>IscR regulates synthesis of colonization factor antigen i fimbriae in response to iron starvation in enterotoxigenic Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27704705" target="_blank">27704705</a>]<br></li>
<li>Regulation of iron -- sulphur cluster homeostasis through transcriptional-control of the Isc pathway by [ 2Fe -- 2S ] -- IscR in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27704705" target="_blank">27704705</a>]<br></li>
<li>IscR dependent gene-expression links iron -- sulphur cluster assembly to the control of O2‐regulated genes in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27704705" target="_blank">27704705</a>]<br></li>
<li>* p ≤ 0.05 ( Mann -- Whitney U‐test ) of HilD targets To investigate whether IscR acts on Spi1 TTSS gene-expression under its apo‐form or holo‐form , we built a plasmid encoding an IscR variant ( IscR‐3CA ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27704705" target="_blank">27704705</a>]<br></li>
<li>FIGURE 5 Apo‐IscR and holo‐IscR negatively regulates the expression of Spi1 TTSS genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27704705" target="_blank">27704705</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27704705" target="_blank">27704705</a>]<br></li>
<li>This supported the view of a primary and key role for IscR , reﬂecting an imbalance in Fe -- S homeostasis ( see below ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21952637" target="_blank">21952637</a>]<br></li>
<li>Conversely , the iscR mutant was shown to be more invasive in HeLa cells and less virulent in mice compared to the wild‐type , consistent with a possible regulation of other genes involved in Salmonella pathogenesis by IscR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27704705" target="_blank">27704705</a>]<br></li>
<li>The holo‐form of IscR acts as a repressor of a large number of genes , including the isc operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27704705" target="_blank">27704705</a>]<br></li>
<li>In E. coli , IscR is a [ 2Fe -- 2S ] containing protein whose maturation The apo‐form of IscR acts as an activator of a few genes , including the suf operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27704705" target="_blank">27704705</a>]<br></li>
<li>Thereby , we wondered if IscR could be involved in the iron regulation of Spi1 genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27704705" target="_blank">27704705</a>]<br></li>
<li>In Yersinia pseudotuberculosis , IscR was found to be involved in the regulation of the type-III-secretion-system ( TTSS ) master regulator LcrF and was demonstrated to be essential for the virulence of Yersinia in a mouse model of infection ( Miller et al. , 2014 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27704705" target="_blank">27704705</a>]<br></li>
<li>Recently , Yersinia IscR was shown to bind upstream of the operon encoding the TTSS master regulator LcrF , which endowed IscR with a positive role in transcription of Yersinia TTSS genes ( Miller et al. , 2014 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27704705" target="_blank">27704705</a>]<br></li>
<li>Hence , under iron limitation or in the presence of reactive-oxygen-species ( ROS ) , IscR exists primarily under its apo‐form that activates expression of SUF and no longer represses expression of isc , the amount of Fe -- S clusters produced being at its highest level .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27704705" target="_blank">27704705</a>]<br></li>
<li>It is a member of the Rrf2 family of transcriptional regulators , which also includes the IscR regulator that is involved in regulation of genes involved in [ Fe -- S ] cluster biogenesis ( Schwartz et al. , 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17526833" target="_blank">17526833</a>]<br></li>
<li>The Fe -- S cluster sensor IscR , a transcriptional regulator matured by the ISC machinery , was shown to bind the promoter of hilD , which encodes the master regulator of Spi1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27704705" target="_blank">27704705</a>]<br></li>
<li>2.3  IscR binds on the promoter of hilD Because the ISC machinery was found to play a role in the regulation of Spi1 TTSS gene-expression , we aimed at identifying a protein matured by ISC and involved in the control of the Spi1 TTSS apparatus .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27704705" target="_blank">27704705</a>]<br></li>
<li>This is , however , the ﬁrst evidence of a regulator , other than IscR , involved in the regulation of the iscRSUA operon and the ﬁrst evidence that the assembly of [ Fe-S ] clusters is activated in response to sources of carbon and energy whose catabolism is under the control of Cra .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19136587" target="_blank">19136587</a>]<br></li>
<li>2.4  IscR represses the expression of hilD gene and HilD targets Next , to investigate the regulatory role of IscR on hilD , the expression of the hilD fusion was measured and found to be about 1.5 fold up‐reg-ulated in an iscR mutant ( Figure 4a ) , suggesting that IscR acts as a negative regulator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27704705" target="_blank">27704705</a>]<br></li>
<li>Indeed , we showed that apo‐IscR , which prevails in an iscU mutant , down‐regulates the expression of the master regulator hilD and thereby of several downstream HilD‐regulated genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27704705" target="_blank">27704705</a>]<br></li>
<li>In particular , we showed that IscR represses the synthesis of the Spi1‐carried regulator HilD , decreasing the expression of genes encoding the TTSS apparatus .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27704705" target="_blank">27704705</a>]<br></li>
<li>Moreover , our results using an Fe -- S cluster‐free variant indicated that the apo‐form of IscR negatively regulates the expression of the Spi1 TTS genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27704705" target="_blank">27704705</a>]<br></li>
<li>IscR acts as an activator in response-to-oxidative-stress for the suf operon encoding Fe-S assembly proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19136587" target="_blank">19136587</a>]<br></li>
<li>IscR plays a role in oxidative stress resistance and pathogenicity of a plant pathogen , Xanthomonas campestris .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27704705" target="_blank">27704705</a>]<br></li>
<li>Biogenesis of Fe / S proteins and pathogenicity : IscR plays a key role in allowing Erwinia chrysanthemi to adapt to hostile conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27704705" target="_blank">27704705</a>]<br></li>
<li>In fact , IscR appears to be important for full virulence of several patho-gens .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27704705" target="_blank">27704705</a>]<br></li>
<li>IscR modulates cat-alase A ( KatA ) activity , peroxide resistance and full virulence of Pseudomonas aeruginosa PA14 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27704705" target="_blank">27704705</a>]<br></li>
				</div></ul><br>
			<label class="collapse_SIT" for="_5"><font size="5px">IscR binding site</font></label>
				<input id="_5" type="checkbox">
				<div class="SIT"><br><ul><li>The two putative IscR binding sites upstream of hilD resemble type 2 motifs that can be bound by both holo‐IscR and apo‐IscR ( Giel , Rodionov , Liu , Blattner , & Kiley , 2006 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27704705" target="_blank">27704705</a>]<br></li>
<li>The IscR type 2 binding sites are represented by the two boxes , ( b ) Sequence alignment of the two putative IscR‐binding sites from Sal-monella enterica hilD with the Escherichia coli hyaA and Yersinia pseudotuberculosis lcrF domains .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27704705" target="_blank">27704705</a>]<br></li>
				</ul></div><br>
			<label class="collapse_TU" for="_6"><font size="5px">Transcription units regulated by IscR</font></label>
				<input id="_6" type="checkbox">
				<div class="TU"><br><ul>				<label class="TU_collapse_similarity" for="tu_similarity_1">
IscR , an Fe-S cluster-containing transcription factor , represses expression of Escherichia coli genes encoding Fe-S cluster assembly proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19136587" target="_blank">19136587</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22831173" target="_blank">22831173</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14729702" target="_blank">14729702</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18441067" target="_blank">18441067</a>]<br></label>
<input id="tu_similarity_1", type="checkbox">
<div class="TU_similar">
<li>IscR , an Fe‐S cluster‐containing transcription factor , represses expression of Escherichia coli genes encoding Fe‐S cluster assembly proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27704705" target="_blank">27704705</a>]<br></li></div>
<li>So going from iron limited to iron replete conditions will shift the equilibrium from apo‐IscR to holo‐IscR , thereby repressing iscR and inducing Spi1 gene expression by alleviating IscR repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27704705" target="_blank">27704705</a>]<br></li>
<li>ns , not significant ; * p ≤ 0.05 ; ** p ≤ 0.01 ( Mann -- Whitney U‐test ) 2.7  IscR reduces the invasion of HeLa cells by Salmonella Because IscR was shown to decrease the expression of Spi1 TTSS genes , we evaluated its contribution to Salmonella virulence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27704705" target="_blank">27704705</a>]<br></li>
<li>Virulence of the iscR mutant was reduced as IscR can reduce HeLa cell invasion by Salmonella , consistent with its negative role in Spi1 TTSS expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27704705" target="_blank">27704705</a>]<br></li>
<li>2.6  IscR mediates iron‐dependent control of Spi1 TTSS genes It was previously shown that iron limitation down‐regulates the expression of Spi1 TTSS genes ( Ellermeier & Slauch , 2008 ; Teixidó , Carrasco , Alonso , Barbé , & Campoy , 2011 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27704705" target="_blank">27704705</a>]<br></li>
<li>FIGURE 5 Apo‐IscR and holo‐IscR negatively regulates the expression of Spi1 TTSS genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27704705" target="_blank">27704705</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27704705" target="_blank">27704705</a>]<br></li>
<li>Noticeably , IscR was not found to shift the PinvF probe in EMSA , supporting the notion that expression of invF is indirectly regulated by IscR ( Figure S3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27704705" target="_blank">27704705</a>]<br></li>
<li>In the transcriptomic analysis , IscR was found to be crucial in informing the cells of the presence of Co as several of the Co up-regulated genes were no longer induced 15 in the absence of iscR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21952637" target="_blank">21952637</a>]<br></li>
<li>These results show that during the first step of the infection , infections confers a central role to the ISC system in the establishmen In a strain lacking the ISC machinery , the expression of iscR increases significantly ( Figure S5 ) and the synthesized IscR is mostly under its apo‐form .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27704705" target="_blank">27704705</a>]<br></li>
<li>Indeed , we showed that apo‐IscR , which prevails in an iscU mutant , down‐regulates the expression of the master regulator hilD and thereby of several downstream HilD‐regulated genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27704705" target="_blank">27704705</a>]<br></li>
<li>2.4  IscR represses the expression of hilD gene and HilD targets Next , to investigate the regulatory role of IscR on hilD , the expression of the hilD fusion was measured and found to be about 1.5 fold up‐reg-ulated in an iscR mutant ( Figure 4a ) , suggesting that IscR acts as a negative regulator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27704705" target="_blank">27704705</a>]<br></li>
<li>In the hilD iscR mutant , the rtsA , prgH , and invF mRNA levels were 360‐fold , 60‐fold , and 460‐fold lower compared to the wild‐type strain , showing that the expression of rtsA , prgH , and invF is totally abrogated in the absence of both iscR and hilD and suggesting a possible Spi1 gene expression reprogramming process in this back ¬ FIGURE 4 IscR represses hilD gene expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27704705" target="_blank">27704705</a>]<br></li>
<li>In iscS and hscA mutants ( 56 ) as well as in iscU and fdx mutants ( Fig. 5 ; Table 3 ) , the assembly of [ Fe-S ] clusters is signiﬁcantly reduced and therefore IscR loses its repressing abilities , resulting in increased expression of the isc operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14729702" target="_blank">14729702</a>]<br></li>
<li>Hence , under iron limitation or in the presence of reactive oxygen species ( ROS ) , IscR exists primarily under its apo‐form that activates expression of SUF and no longer represses expression of isc , the amount of Fe -- S clusters produced being at its highest level .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27704705" target="_blank">27704705</a>]<br></li>
<li>IscR was also demonstrated to repress hilD and subsequently Spi1 gene expression , consistent with the observation that an IscR mutant is hyper invasive in epithelial cells .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27704705" target="_blank">27704705</a>]<br></li>
<li>Similarly , the invF gene was down‐regulated 1.7 fold in an iscR − / piscR strain and 1.3 fold in an iscR − / piscR‐3CA strain , again indicating that apo‐IscR and holo‐IscR modified invF gene expression ( Figure 5a , left ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27704705" target="_blank">27704705</a>]<br></li>
				</div></ul></div><br>
	</div>
</body>
</html>