<!doctype html>
<style>
@charset "utf-8";
body {
	height: 100%;
	margin: 0;
	padding: 0;
}

.banner {
	position: fixed;
	top: 0;
	width: 100%;
	height: 65px;
	background: gray;
	line-height: 65px;
}
.spacer {
	height: 10px;
}

.main-content {
	padding: 5%;
}

.collapse_ACT {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-style: none solid none none;
	border-color: #B4045F;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_ACT + input{
	display: none;
}
.collapse_ACT + input + div{
	display: none;
}
.collapse_ACT + input:checked + div{
	display: block;
}
.ACT {
	text-align: justify;
	text-justify: inter-word;
}
.ACT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B4045F;
}
.ACT_collapse_similarity:hover{
	background: #e5d3dc;
}
.ACT_collapse_similarity + input {
	display: none;
}
.ACT_collapse_similarity + input + div{
	display: none;
}

.ACT_collapse_similarity + input:checked + div{
	display: block;
}
.ACT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.ACT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B4045F;
	padding-left: 5px;
}
.ACT_collapse_similarity:hover{
	background: #e5d3dc;
}
.ACT_collapse_similarity + input {
	display: none;
}
.ACT_collapse_similarity + input + div{
	display: none;
}
.ACT_collapse_similarity + input:checked + div{
	display: block;
}
.ACT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.ACT_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #B4045F;
}
.ACT_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_DOM {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-style: none solid none none;
	border-color: #B404AE;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_DOM + input{
	display: none;
}
.collapse_DOM + input + div{
	display: none;
}
.collapse_DOM + input:checked + div{
	display: block;
}
.DOM {
	text-align: justify;
	text-justify: inter-word;
}
.DOM_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B404AE;
	padding-left: 5px;
}
.DOM_collapse_similarity:hover{
	background: #e8d7e7;
}
.DOM_collapse_similarity + input {
	display: none;
}
.DOM_collapse_similarity + input + div{
	display: none;
}
.DOM_collapse_similarity + input:checked + div{
	display: block;
}
.DOM_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}.DOM_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #B404AE;
}
.DOM_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_EVO {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #7401DF;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_EVO + input{
	display: none;
}
.collapse_EVO + input + div{
	display: none;
}
.collapse_EVO + input:checked + div{
	display: block;
}
.EVO {
	text-align: justify;
	text-justify: inter-word;
}
.EVO_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #7401DF;
	padding-left: 5px;
}
.EVO_collapse_similarity:hover{
	background: #e2d7ed;
}
.EVO_collapse_similarity + input {
	display: none;
}
.EVO_collapse_similarity + input + div{
	display: none;
}
.EVO_collapse_similarity + input:checked + div{
	display: block;
}
.EVO_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.EVO_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #7401DF;
}
.EVO_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_RP {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #0174DF;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_RP + input{
	display: none;
}
.collapse_RP + input + div{
	display: none;
}
.collapse_RP + input:checked + div{
	display: block;
}
.RP {
	text-align: justify;
	text-justify: inter-word;
}
.RP_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #0174DF;
	padding-left: 5px;
}
.RP_collapse_similarity:hover{
	background: #c9d7e2;
}
.RP_collapse_similarity + input {
	display: none;
}
.RP_collapse_similarity + input + div{
	display: none;
}
.RP_collapse_similarity + input:checked + div{
	display: block;
}
.RP_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.RP_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #0174DF;
}
.RP_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_SIT {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #0489B1;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_SIT + input{
	display: none;
}
.collapse_SIT + input + div{
	display: none;
}
.collapse_SIT + input:checked + div{
	display: block;
}
.SIT {
	text-align: justify;
	text-justify: inter-word;
}
.SIT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #0489B1;
	padding-left: 5px;
}
.SIT_collapse_similarity:hover{
	background: #d0dfe2;
}
.SIT_collapse_similarity + input {
	display: none;
}
.SIT_collapse_similarity + input + div{
	display: none;
}
.SIT_collapse_similarity + input:checked + div{
	display: block;
}
.SIT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.SIT_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #0489B1;
}
.SIT_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_TU {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #04B486;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_TU + input{
	display: none;
}
.collapse_TU + input + div{
	display: none;
}
.collapse_TU + input:checked + div{
	display: block;
}
.TU {
	text-align: justify;
	text-justify: inter-word;
}
.TU_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #04B486;
	padding-left: 5px;
}
.TU_collapse_similarity:hover{
	background: #d0e2de;
}
.TU_collapse_similarity + input {
	display: none;
}
.TU_collapse_similarity + input + div{
	display: none;
}
.TU_collapse_similarity + input:checked + div{
	display: block;
}
.TU_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.TU_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #04B486;
}
.TU_similar > a {
	text-decoration: none;
	color: gray;
}

ul { 
	list-style-type: none;
	padding: 0;
	margin: 0;
}
ul li > a {
	text-decoration: none;
	color: gray;
}
ul li:hover, ul li:focus {
	background: #F0F0F0;
}

ul li:first-child span {
  border-left:none;  
}

</style>
<html>
<head>
<meta charset="utf-8">
<title>TF properties - MarR</title>
<meta name="theme-color" content="#ffffff">
</head>
<body>
	<div class="banner">
		<center><font color="white" size="16px" face="arial">MarR</font></center>
	</div>
	<div align="center" class="main-content">
		<div class="spacer">
	</div><br>			<label class="collapse_ACT" for="_1"><font size="5px">MarR transcriptional activity</font></label>
				<input id="_1" type="checkbox">
				<div class="ACT"><br><ul><li>trxB 4.93 J 1.9 ¬ 94 85 73 GTTAACAATATGTGT GTTAACAAAATCGTT GTCAACTAAATGAAT 1.00E-05 5.70E-05 9.50E-05 thioredoxin reductase + ¬ STM1520 marR 1.75 − 1.38 − 1.02 DNA-binding-transcriptional-repressor MarR putative periplasmic protein 5.55 2.75 STM1586 ¬ 1.06 + 171 GTTAAGAAAATGTGC 9.50E-05 STM3216 tsr ¬ 198 1.17 − 1.65 − 1.31 putative methyl-accepting chemotaxis protein short chain dehydrogenase putative glutamic dehyrogenase-like protein glycerol diffusion GTTAACCATTTCTTA 8.10E-06 + − 1.86 − 1.94 STM2445 ucpA − 5.34 − 1.83 STM1795 gluD − 6.28 − 1.34 44 GTTAATGGAGTGTAA GTTAACTATCCGCTA 1.20E-05 9.50E-05 ¬ 121 + STM4087 glpF − 1.63 − 5.7 STM1771 chaA − 1.77 − 5.97 8.77 3.31 217 36 GTTAATGAAATGATT 1.00E-05 ¬ calcium/sodium : proton antiporter major sodium/proline symporter secreted effector-protein transcriptional-regulator GTTAATATTTTGGAA 8.00E-05 234 + − 3.46 STM1125 putP − 8.81 − 3.4 GTTAACACTTTTAAA 9.50E-05 + − 5.89 − 3.51 − 2.31 STM1091 STM2866 STM4405 52 sopB sprB ytfJ − 3.57 − 2.32 − 1.45 − 2.17 − 6.06 − 3.2 GTTAACCCTGTTGAA GTTAATGAAAGGGAA 8.00E-05 8.10E-06 281 67 + GTTAATCATATGTGC ¬ putative transcriptional-regulator putative PTS permease arginine deiminase putative inner membrane protein 3.30E-05 ¬ − 2.76 − 2.07 − 2.92 nanM STM4535 STM4467 STM1130 98 − 1.04 1.02 1.01 − 2.7 − 1.83 − 2.58 GTTAACAGAGGGAAA GTTAATTATTTGTTT GATAACTCCATGTAA 9.50E-05 6.50E-06 8.00E-05 271 115 ¬ + 1.20E-05 GTTAACAACATGCCA anti-RNA polymerase sigma 70 factor putative cytoplasmic protein putative RHS-like protein hypothetical protein 2.75 1.69 STM4165 rsd 67 ¬ 2.74 STM1728 yciG 1.68 1.95 261 ¬ − 2.11 1.50E-05 GTTAATGCATTGTTT ¬ ¬ 6.80E-05 6.80E-05 GTTCATCAAATGTAG GTCAATGATGTGTTA 248 64 2.12 2.41 3.64 3.58 1.18 5.84 2.07 7.09 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24044554" target="_blank">24044554</a>]<br></li>
<li>[ 11 ] Haque MM , Kabir MS , Aini LQ , Hirata H , Tsuyumu S. SlyA , a MarR family transcriptional-regulator , is essential for virulence in Dickeya dadantii 3937 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29857034" target="_blank">29857034</a>]<br></li>
<li>It could be that molecules derived by oxidative-stress caused by NaOCl could interact with MarR , thereby increasing marA expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22752112" target="_blank">22752112</a>]<br></li>
<li>In contrast , DEC binds Rob protein , but does not affect the activity of MarR protein ( 16 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19120970" target="_blank">19120970</a>]<br></li>
<li>All S. Typhimurum ; S. Heidelberg , S. St. Paul ; S. Choleraesuis ; S. Dublin ; S. Weltevreden ; S. Enteritidis ; S. Schwarzengrund ; Escherichia coli O103 : H2 and other E. coli strains ; Shigella boydii es Yes 24.7 Y Yes No No No 26.2 TABLE 2 Effect of STM S. Typhimurium LT2 locus identity STM2239a STM2237 STM2236 STM2235 STM2234 STM2233 STM3881 2239 overexpression on transcription of other genes Gene name Median fol change ( n 6.46 5.09 4.94 4.78 4.49 4.02 3.53 d P value by t test ( n 3 ) 1.63 E 22 1.83 E 15 2.07 E 20 4.71 E 43 3.41 E 23 1.05 E 16 2.49 E 12 3 ) Predicted function Putative phage protein ; homology to antiterminator protein Q of phage P5 Putative inner membrane protein Hypothetical protein Hypothetical protein Putative phage tail ﬁber assembly protein Putative cytoplasmic protein D-Ribose pyranase ; catalyzes the conversion between beta-pyran and beta-furan forms of D-ribose Putative outer membrane protein 2-Oxo-3-deoxygalactonate 6-phosphate aldolase/galactonate dehydratase Ornithine decarboxylase Pyruvate dehydrogenase subunit E1 ; E1 component ; part of pyruvate dehydrogenase ; forms a complex with DlaT and LpdC Dihydrolipoamide acetyltransferase ; E2 component of pyruvate dehydrogenase multienzyme complex 2-Keto-3-deoxygluconate permease ; transports degraded pectin products into the bacterial cell 2-Dehydro-3-deoxyphosphogalactonate aldolase S-Adenosylmethionine synthetase ; catalyzes the formation of S-adenosylmethionine from methionine and ATP Phosphate acetyltransferase ; catalyzes the synthesis of acetylphosphate or propionylphosphate from acetyl-CoA or propionyl-CoA and inorganic phosphate Acetate kinase ; AckA utilizes acetate and can acetylate CheY , which increases signal strength during ﬂagellar rotation ; also involved in conversion of acetate to acetyl-CoA DNA-binding-transcriptional-activator MarA ; transcriptional-activator of genes involved in the multiple antibiotic resistance ( Mar ) phenotype Ribose ABC transport system , ATP-binding protein RbsA ; involved in D-ribose utilization Putative inner membrane protein Multiple antibiotic resistance-protein MarB 2-Oxo-3-deoxygalactonate kinase High-afﬁnity gluconate permease Putative outer membrane protein DNA-binding-transcriptional-repressor MarR ; involved in the activation of both antibiotic resistance and oxidative-stress genes Respiratory NADH dehydrogenase 2 ; cupric reductase ( 2nd module ) Valine tRNA 2A Outer membrane receptor FepA ; outer membrane receptor of ferric enterobactin and colicins B and D ; interacts with the TonB-ExbBD Putative cytoplasmic protein Ribose ABC transporter permease protein ; functions to transport ribose at high afﬁnity ; forms a complex with RbsA2C2B Heat shock chaperone IbpB ; associates with aggregated proteins , together with IbpA Putative amino acid transport protein Enterobactin/ferric enterobactin esterase Putative outer membrane protein Putrescine transporter Valine tRNA 2B Galactonate operon transcriptional-repressor UmuD protein , putative Virulence protein ; similar to virulence protein MsgA ( STM1241 ) ; SsrB-regulated factor O-antigen acetylase ( lipopolysaccharide ) ; O-antigen ﬁve Leucine-rich repeat-containing protein ; induced by the SPI-2 regulator SsrA/B ; secreted by SPI-2 Hypothetical protein Putative tail ﬁber protein of phage ; similar to Escherichia coli putative membrane protein ( AAC74454 .1 ) Homolog of virulence protein MsgA-related protein Putative outer membrane protein Transcriptional regulator NarP ; nitrate/nitrite response-regulator rbsD 3.18 3.17 3.12 3.06 ycfR dgoA speF aceE STM1214 STM3828 STM0701 STM0152 3.53 E 09 2.88 E 39 2.87 E 76 1.88 E 108 2.14 E 04 STM0153 2.90 aceF 1.28 E 37 2.80 STM0161 kdgT 3.47 E 21 5.13 E 36 2.80 2.72 STM3828 .1 N STM3090 metK 1.53 E 63 pta 2.70 STM2338 5.27 E 47 2.66 STM2337 ackA 9.14 E 19 2.61 STM1519.S marA 47 37 08 19 38 13 12 7.30 E 2.36 E 9.44 E 1.42 E 9.62 E 5.16 E 1.15 E 2.59 2.53 2.46 2.42 2.40 2.36 2.35 STM3882 STM0162 STM1518 STM3829 STM3512 STM3361 STM1520 rbsA marB dgoK gntT yhcN marR 36 02 57 1.48 E 1.31 E 3.44 E 2.32 2.31 2.25 STM1211 STM1423 STM0585 ndh valW fepA 1.45 E 06 1.57 E 22 2.25 2.24 STM0587 STM3883 ybdZ rbsC 3.90 E 15 1.95 E 39 7.85 E 31 1.13 E 27 9.27 E 41 2.83 E 06 3.69 E 19 1.31 E 05 0.008566 2.37 E 42 1.17 E 12 2.23 2.15 STM3808.S STM2068 STM0586 STM0509 STM0700 STM1424 STM3830 STM2230 STM2231 STM2232 STM2241 ibpB yeeF fes 2.12 2.10 2.07 2.04 2.03 0.8144 0.20572 0.78253 0.19829 potE valV dgoR oafA sspH2 STM2242 STM2243 0.44144 0.296273 0.076021 0.00941 0.246198 0.603784 0.000117 STM2244 0.114917 STM2245 0.050032 STM2246 0.4088 a Overexpressed from pFLAG-STM2239 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23630960" target="_blank">23630960</a>]<br></li>
<li>The transcriptional-activator MarA , its local repressor MarR , and two proteins with unknown functions , MarB and MarC , are encoded by the mar locus .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28926285" target="_blank">28926285</a>]<br></li>
<li>For example , the Escherichia coli MarR and EmrR proteins regulate genes involved in antibiotic resistance , PecS from Erwinia chrysanthemi controls pectinase and cellulase production , and RovA of Yersinia spp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12775687" target="_blank">12775687</a>]<br></li>
<li>This means that a strong induction of MarA ( and a sequential induction of acrAB ) is expected only when MarR is inactivated as , indeed , occurs with SAL which binds MarR and inactivates it ( 52 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19120970" target="_blank">19120970</a>]<br></li>
<li>marA transcription is repressed by MarR and is de-repressed by compounds such as salicylate , benzoate , dinitrophenol and plumbagin , probably through their binding to MarR ( 12 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19120970" target="_blank">19120970</a>]<br></li>
<li>In this context , marA is regulated negatively by MarR ( Alekshun and Levy 1999 ) , which by binding ligands such as salicylate , lowers its afﬁnity for DNA , allowing derepression of the marRAB operon ( Perera et al. 2009 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22752112" target="_blank">22752112</a>]<br></li>
<li>The binding was sequence-speciﬁc as demonstrated by the ability of unlabelled marO , but not non-speciﬁc DNA , to compete for binding to MarR ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15073288" target="_blank">15073288</a>]<br></li>
<li>Required for bacteriophage lambda integration ; bends DNA and significantly changes transcriptional regulation of many genes Required for the bacterial stringent response that results in reduced transcription in the presence of uncharged t-RNA MarR family transcription regulator i.g. ; i.p. .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21687430" target="_blank">21687430</a>]<br></li>
<li>Here we demonstrate that S. typhimurium marRAB is activated in the presence of bile and that deoxycholate interacts with MarR to prevent DNA binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15073288" target="_blank">15073288</a>]<br></li>
<li>This suggests a plausible model for regulating the ability of SlyA to-bind DNA in which SlyA interacts with DNA in a similar manner to OhrR , with a conserved interaction between the wing region and the DNA minor groove , and sequence specific interactions in the major groove mediated by amino acids in α4 ( including Arg-65 of SlyA ) , but that in the presence of an as yet unidentified ligand Glu-59 ′ might interact with Arg-65 to inhibit SlyA DNA-binding by repositioning the DNA binding lobes in a manner similar to that suggested by the MarR : salicylate complex [ 1,22 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19204896" target="_blank">19204896</a>]<br></li>
<li>Like the Hpr and MarR proteins , SlyA as isolated did not possess , or require , any co-factors to facilitate binding to target DNA suggesting that the intracellular content of SlyA is regulated and/or that a co-effector can interact with SlyA to abolish DNA binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11882648" target="_blank">11882648</a>]<br></li>
<li>MarA regulates its own transcription as well as regulating the expression of the mar regulon , whereas MarR acts by repressing marRAB transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19104017" target="_blank">19104017</a>]<br></li>
<li>MarR negatively regulates the expression of marRAB by binding to the marO operator region .17 On the other hand , proteins encoded by the soxRS locus include the transcriptional-activator , SoxS , and another protein , SoxR , the oxidized form of which can activate soxS expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16201927" target="_blank">16201927</a>]<br></li>
<li>The repressor MarR ( 43 ) , encoded by marR , binds to the marO operator region ( 39 ) to negatively regulate expression of marRAB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11229910" target="_blank">11229910</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_19">
MarR acts as a local repressor by binding to the operator region ( marO ) , which negatively regulates the expression of marA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25128419" target="_blank">25128419</a>]<br></label>
<input id="act_similarity_19", type="checkbox">
<div class="ACT_similar">
<li>MarR negatively regulates the expression of marRAB by binding to the marO operator region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18443112" target="_blank">18443112</a>]<br></li>
<li>MarR negatively regulates expression of marRAB by binding to the marO operator region [ 13,15 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15336432" target="_blank">15336432</a>]<br></li></div>
<li>Incubation of samples in the presence of 1 mM deoxycholate abolished MarR -- marO complexes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15073288" target="_blank">15073288</a>]<br></li>
<li>While normally repressed by MarR , a mutation within the marR or marO genes leads to constitutive transcriptional activation of marRAB , resulting in increased drug resistance that can be overcome by overexpression of wild-type MarR [ 5 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10856650" target="_blank">10856650</a>]<br></li>
<li>In contrast , the inactivation of RamA , MarR , MarA , SoxR , and SoxS did not affect the susceptibilities of the strain [ 17 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21858134" target="_blank">21858134</a>]<br></li>
<li>Therefore , in this study , we investigated the roles of MarR , MarA , SoxR , SoxS , and RamA in the MDR and quinolone resistance of serovar Typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18443112" target="_blank">18443112</a>]<br></li>
<li>However , micF and acrAB are also induced by stress signals in additional ways that are not dependent on the marRAB system .17 Interestingly , antibiotics that serve as substrates of the AcrAB efﬂux pump are very weak inducers of the marRAB operon in E. coli and do not induce these genes in Salmonella .6,8 Based on that , it was suggested that exposure of E. coli or S. enterica to salicylate activates MarA through binding to its repressor , MarR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20237076" target="_blank">20237076</a>]<br></li>
<li>Intracellular levels of MarA are controlled by MarR that binds to marO and negatively modulates expression of the marRAB operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16842216" target="_blank">16842216</a>]<br></li>
<li>The mechanism of induction by phenolic compounds , speciﬁcally salicylate , is by the binding of salicylate to MarR , which inhibits binding of MarR to the marRAB promoter ( Martin & Rosner , 1995 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15073288" target="_blank">15073288</a>]<br></li>
<li>Expression of MarA and MarR is induced by the prevention of MarR binding to the operator ,8 and this may occur by the binding of MarR to salicylate .6,9 Salicylate is , in fact , the most potent known inducer of marRAB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20237076" target="_blank">20237076</a>]<br></li>
<li>Since SAL binds puriﬁed MarR repressor and enhances dissociation of MarR-marO complexes in vitro ( 37 ) , it is thought that , in vivo , repression is relieved and activation of marRAB expression results from the direct interaction of SAL with the repression complex .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9068629" target="_blank">9068629</a>]<br></li>
<li>With the former compound , expression of MarR might block the overexpression and restore the basal transcription rates , whereas direct inactivation of MarR with SAL is expected to inhibit this negative autoregulation mechanism .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19120970" target="_blank">19120970</a>]<br></li>
<li>For instance , in Staphylococcus aureus and Escherichia coli it modulates the transcription of dozens of genes .3,4 Salicylate is also associated with the production of factors involved with bacterial virulence5 and with reduction in the resistance to a variety of antimicrobials .2,3,6 In E. coli and Salmonella enterica , the global regulators MarA and MarR are induced by salicylate .7 MarR binds the mar operator that ﬂanks the promoter of marRAB and represses the transcription of these genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20237076" target="_blank">20237076</a>]<br></li>
<li>In E. coli it is modulated at the lowest level by the local repressor AcrR .14 At a more global level , acrAB expression is modulated by stress conditions and by global regulatory genes , such as marA , soxS , and rob .14,17,18 The acrAB locus is indeed part of the mar , sox , and rob regulons of E. coli , the activation of which confers a low-level resistance to a wide range of antimicrobial agents and organic solvents .17 Previous studies clearly showed that overexpression of marA , which results from marR mutations , would down-regulate the expression of acrR , thus inducing high-level AcrAB expression and antimicrobial resistance .23 Proteins encoded by the mar locus include the transcriptional-activator , MarA , its local repressor , MarR , and MarB and MarC , two proteins with unknown functions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16201927" target="_blank">16201927</a>]<br></li>
<li>Deoxycholate speciﬁcally interacts with MarR Salicylate , an inducer of marRAB activity , has been shown to-bind the repressor MarR , leading to an increase in transcriptional activity of the operon ( Cohen et al. , 1993 ; Martin & Rosner , 1995 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15073288" target="_blank">15073288</a>]<br></li>
<li>Thus induction of marRAB by DEC without inhibition of the repression activity of MarR is expected to be less effective than induction with SAL .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19120970" target="_blank">19120970</a>]<br></li>
<li>[ Int Microbiol 2008 ; 11 ( 4 ) :245 -250 ] Key words : Salmonella · gene regulation · DNA binding · SlyA · MarR · OhrR Introduction Appropriate regulation of gene expression is essential for bacterial pathogens to adapt to a changing host environment .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19204896" target="_blank">19204896</a>]<br></li>
<li>Our results indicate that deoxycholate does interact with MarR to prevent DNA-binding and that this interaction is speciﬁc for deoxycholate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15073288" target="_blank">15073288</a>]<br></li>
				</div></ul><br>
			<label class="collapse_DOM" for="_2"><font size="5px">MarR protein domains</font></label>
				<input id="_2" type="checkbox">
				<div class="DOM"><br><ul><li>trxB 4.93 J 1.9 ¬ 94 85 73 GTTAACAATATGTGT GTTAACAAAATCGTT GTCAACTAAATGAAT 1.00E-05 5.70E-05 9.50E-05 thioredoxin reductase + ¬ STM1520 marR 1.75 − 1.38 − 1.02 DNA-binding transcriptional repressor MarR putative periplasmic protein 5.55 2.75 STM1586 ¬ 1.06 + 171 GTTAAGAAAATGTGC 9.50E-05 STM3216 tsr ¬ 198 1.17 − 1.65 − 1.31 putative methyl-accepting chemotaxis protein short chain dehydrogenase putative glutamic dehyrogenase-like protein glycerol diffusion GTTAACCATTTCTTA 8.10E-06 + − 1.86 − 1.94 STM2445 ucpA − 5.34 − 1.83 STM1795 gluD − 6.28 − 1.34 44 GTTAATGGAGTGTAA GTTAACTATCCGCTA 1.20E-05 9.50E-05 ¬ 121 + STM4087 glpF − 1.63 − 5.7 STM1771 chaA − 1.77 − 5.97 8.77 3.31 217 36 GTTAATGAAATGATT 1.00E-05 ¬ calcium/sodium : proton antiporter major sodium/proline symporter secreted effector protein transcriptional regulator GTTAATATTTTGGAA 8.00E-05 234 + − 3.46 STM1125 putP − 8.81 − 3.4 GTTAACACTTTTAAA 9.50E-05 + − 5.89 − 3.51 − 2.31 STM1091 STM2866 STM4405 52 sopB sprB ytfJ − 3.57 − 2.32 − 1.45 − 2.17 − 6.06 − 3.2 GTTAACCCTGTTGAA GTTAATGAAAGGGAA 8.00E-05 8.10E-06 281 67 + GTTAATCATATGTGC ¬ putative transcriptional regulator putative PTS permease arginine deiminase putative inner membrane protein 3.30E-05 ¬ − 2.76 − 2.07 − 2.92 nanM STM4535 STM4467 STM1130 98 − 1.04 1.02 1.01 − 2.7 − 1.83 − 2.58 GTTAACAGAGGGAAA GTTAATTATTTGTTT GATAACTCCATGTAA 9.50E-05 6.50E-06 8.00E-05 271 115 ¬ + 1.20E-05 GTTAACAACATGCCA anti-RNA-polymerase-sigma 70 factor putative cytoplasmic protein putative RHS-like protein hypothetical protein 2.75 1.69 STM4165 rsd 67 ¬ 2.74 STM1728 yciG 1.68 1.95 261 ¬ − 2.11 1.50E-05 GTTAATGCATTGTTT ¬ ¬ 6.80E-05 6.80E-05 GTTCATCAAATGTAG GTCAATGATGTGTTA 248 64 2.12 2.41 3.64 3.58 1.18 5.84 2.07 7.09 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24044554" target="_blank">24044554</a>]<br></li>
<li>In this respect SlyA resembles the MarR protein , which is thought to adopt a dimeric state to recognize an inverted repeat composed of pentameric subelements ( 23 ) , possibly through interaction with two putative helix-turn-helix motifs ( 24 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11882648" target="_blank">11882648</a>]<br></li>
<li>Here we demonstrate that S. typhimurium marRAB is activated in the presence of bile and that deoxycholate interacts with MarR to prevent DNA-binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15073288" target="_blank">15073288</a>]<br></li>
<li>Our results indicate that deoxycholate does interact with MarR to prevent DNA-binding and that this interaction is speciﬁc for deoxycholate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15073288" target="_blank">15073288</a>]<br></li>
<li>Circled residues in MarR are those that are completely conserved among 14 of 15 members of the MarR-family ( 39a ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9068629" target="_blank">9068629</a>]<br></li>
<li>Using an alignment of the primary structures of MarR and SlyA to delimit the predicted DNA-binding region of SlyA , mutant slyA alleles were created by replacing single amino acids between residues 48 to 94 by Ala. .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19204896" target="_blank">19204896</a>]<br></li>
<li>Based on the MarR structure , in the SlyA dimer , Val13 and Leu126 of one protomer are predicted to be close to the corresponding residues in the other protomer .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19204896" target="_blank">19204896</a>]<br></li>
<li>This suggests a plausible model for regulating the ability of SlyA to bind DNA in which SlyA interacts with DNA in a similar manner to OhrR , with a conserved interaction between the wing region and the DNA minor groove , and sequence specific interactions in the major groove mediated by amino acids in α4 ( including Arg-65 of SlyA ) , but that in the presence of an as yet unidentified ligand Glu-59 ′ might interact with Arg-65 to inhibit SlyA DNA-binding by repositioning the DNA-binding lobes in a manner similar to that suggested by the MarR : salicylate complex [ 1,22 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19204896" target="_blank">19204896</a>]<br></li>
				</div></ul><br>
			<label class="collapse_EVO" for="_3"><font size="5px">MarR evolution</font></label>
				<input id="_3" type="checkbox">
				<div class="EVO"><br><ul><li>[ 11 ] Haque MM , Kabir MS , Aini LQ , Hirata H , Tsuyumu S. SlyA , a MarR family transcriptional regulator , is essential for virulence in Dickeya dadantii 3937 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29857034" target="_blank">29857034</a>]<br></li>
<li>J Bacteriol 177:5140 -- 5147 Perera IC , Lee YH , Wilkinson SP , Grove A ( 2009 ) Mechanism for attenuation of DNA binding by MarR family transcriptional regulators by small molecule ligands .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22752112" target="_blank">22752112</a>]<br></li>
<li>The response regulator OmpR , the MarR homologue SlyA , and the nucleoid protein Fis contrib-ute directly to activate the ssrA-ssrB SPI-2 regulatory locus , which encodes a two-component regulatory system ( 15 -- 18 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21059643" target="_blank">21059643</a>]<br></li>
<li>It is intriguing that each of the segments ( i.e. HK , O and S ) has an ORF at one terminus which has similarity to a transcriptional regulator in the LysR , DeoR and MarR families ( Perez-Rueda & Collado-Vides , 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12177346" target="_blank">12177346</a>]<br></li>
<li>Edited by Emil C. Gotschlich , The Rockefeller University , New York , NY , and approved November 12 , 2008 ( received for review July 22 , 2008 ) he MarR family member SlyA and its homologs are present Tin members of the family Enterobacteriaceae .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19091955" target="_blank">19091955</a>]<br></li>
<li>SlyA has sequence homology with members of a family of transcriptional regulators which include EmrR , MarR , HprR , and PecS proteins , as well as a hypothetical protein ( CY79 ) from Mycobacterium tuberculosis ( 26 , 28 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9284144" target="_blank">9284144</a>]<br></li>
<li>A transcription factor involved in this process is SlyA , a member of the MarR transcription factor family , which recognises inverted repeats in DNA sequences ; TTAN6TAA has been determined to be its consensus sequence [ 6e8 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29857034" target="_blank">29857034</a>]<br></li>
<li>Hong M , Fuangthong M , Helmann JD , Brennan RG ( 2005 ) Structure of an OhrR-ohrA operator complex reveals the DNA binding mechanism of the MarR family .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19204896" target="_blank">19204896</a>]<br></li>
<li>The structure of the MarR family member OhrR bound at its operator DNA shows that the residues of OhrR ( Arg-86 and Arg-94 ) that interact with the minor groove of the DNA target are conserved in SlyA ( Arg-78 and Arg-86 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19204896" target="_blank">19204896</a>]<br></li>
<li>Circled residues in MarR are those that are completely conserved among 14 of 15 members of the MarR family ( 39a ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9068629" target="_blank">9068629</a>]<br></li>
<li>Wilkinson SP , Grove A ( 2006 ) Ligand-responsive transcriptional regulation by members of the MarR family of winged helix proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19204896" target="_blank">19204896</a>]<br></li>
<li>The SlyA protein is a member of the MarR family of transcription factors that includes MarR and EmrR ( E. coli ) , PecS ( Erwinia chrysanthemi ) , HprR ( Bacillus subtilis ) , and RovA ( Yersinia enterocolitica ) ( 20 , 24 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14996792" target="_blank">14996792</a>]<br></li>
<li>DISCUSSION The SlyA protein of S. typhimurium is a member of a family of transcription factors related to MarR and associated with bacterial adaptation to stress ( 1 , 2 , 22 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11882648" target="_blank">11882648</a>]<br></li>
<li>The SlyA protein from Salmonella typhimurium is a member of the MarR family of transcription factors that are required for the adaptation of a variety of bacteria to different environments .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11882648" target="_blank">11882648</a>]<br></li>
				<label class="EVO_collapse_similarity" for="evo_similarity_15">
SlyA is a member of the MarR family of dimeric , winged-helix transcription regulators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19204896" target="_blank">19204896</a>]<br></label>
<input id="evo_similarity_15", type="checkbox">
<div class="EVO_similar">
<li>SlyA is a member of the MarR family of winged-helix transcription factors .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19204896" target="_blank">19204896</a>]<br></li>
<li>SlyA is a member of the MarR family of transcription factors .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12775687" target="_blank">12775687</a>]<br></li></div>
<li>First , MarR is a member of a family of phenolic-sensing regulatory proteins , some of which play a role in virulence ( 54 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9068629" target="_blank">9068629</a>]<br></li>
<li>This protein can also be considered the newest member of the MarR family of phenol-ic-binding regulatory proteins ( 54 ) , of which there are now 16 ( 39a ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9068629" target="_blank">9068629</a>]<br></li>
<li>g001 STM1444 slyA transcriptional regulator for hemolysin ( MarR family ) [ 31 ] STM1753 hnr response regulator in protein turnover [ 96 ] STM2640 rpoE sigma E factor of RNA polymerase , response to periplasmic stress [ 27 ] STM2688 smpB small protein B ; tmRNA-binding protein [ 97 ] STM2826 csrA carbon storage regulator [ 30 ] STM2924 rpoS sigma S factor of RNA polymerase , major sigma factor during stationary phase [ 98 ] catabolite activator protein ( CAP ) , cyclic AMP receptor protein ( CRP family ) STM3466 crp [ 99 ] two-component regulatory system responding to osmolarity and pH STM3501/2 envZ/ompR [ 100 ] host factor I for bacteriophage Q beta replication , a growth-related protein STM4361 hfq [ 33 ] doi :10.1371 / journal.ppat .1000306 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19229334" target="_blank">19229334</a>]<br></li>
<li>Besides phoP/phoQ ( 19 ) and rpoS ( 27 ) , a new regulatory gene slyA was cloned by Libby et al. ( 15 ) from Salmonella serovar Typhimurium as a cytolysin-encoding gene , but later it was proved to be a gene encoding a transcriptional regulatory protein ( 5 ) , which belongs to a family of regulators including EmrR ( 16 ) , MarR ( 28 ) , HprR ( 23 ) , and PecS ( 21 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11939575" target="_blank">11939575</a>]<br></li>
				</div></ul><br>
			<label class="collapse_RP" for="_4"><font size="5px">Processes regulated by MarR</font></label>
				<input id="_4" type="checkbox">
				<div class="RP"><br><ul><li>AE008762 slyA Pathogenicity MarR family transcriptional regulator for hemolysin Table 2 ( continued ) Accession number Probe name Marker group Gene function Control strain Probe sequence Length Melting temp ( 1C ) AF026035 AE008753 D14156 D14156 AE008694 sirA Pathogenicity Invasol SirA : regulator of invasion proteins LT2 Pathogenicity Sensory kinase protein LT2 Pathogenicity UDP-glucose/GDP mannose dehydrogenase in Vi polysaccharide-biosynthesis RKI-Ty1 Pathogenicity Vi polysaccharide export inner-membrane protein RKI-Ty1 Fimbrial Fimbrial usher , bovine colonisation factor LT2 ATGATGACACATATCCGAATGCCAGTAACAATGCCGAAGC 40 ACGCTTGTAAATATTGTCTGGAGTTTGTCGAGATTTCGGC 40 ATCAAGCCACTACGATGCGATCATTGTTGCAGTAGGACAT 40 CATTAGTTATTAAAGAGCGACGTGTTCAGCAAGCCAAGCA 40 GGGAAAGTACAACTCGATATCAATCAACAGCTAGGCGGGT 40 70.3 69.0 70.7 69.3 70.4 phoQ wcdA wzf bcfC lpfD safC sefA stbD stcC AE008868 AE008708 L11008 AE008710 AE008795 Fimbrial Fimbrial Fimbrial Fimbrial Fimbrial Long polar ﬁmbrial operon protein Putative ﬁmbrial usher protein Fimbrial protein Fimbial usher protein Paral putative outer membrane protein LT2 LT2 00-04 LT2 LT2 CTATGCGATGTCCTGTGAATGCCCTGATGATACCTCTCTT 40 TGTAAGTGCTAGTTGGCAGATGACTTCACCATCACACGGT 40 TGGTACTCTCAGCATTACTGCTACTGGTCCACATAACTCA 40 TCGGTTTGCCAACGGTGATTAGCGTCAGTAATAGTGAAAC 40 TCCGCCAGTCAGACTTACGACGAGGATCATAATGAAGATA 40 70.1 71.5 69.3 70.0 69.2 Fimbrial Fimbrial Fimbrial Fimbrial Fimbrial stdB stfE AE008839 AE008703 AE008702 AE008915 AE008916 Putative outer membrane ﬁmbrial usher protein Putative minor ﬁmbrial subunit Putative ﬁmbrial usher Putative ﬁmbrial usher protein Putative ﬁmbrial chaperone protein LT2 LT2 LT2 LT2 LT2 CACAGTCCAACAATAACTACATGCTCAGCCTCAACAAGGT 40 CCTGAGCTGTAACGGCAGAGTGAGCGATTACCTGAAGTTA 40 ATTATCAGTAAGCATCCCGCAACTCTACATCGCCAACAAC 40 ACGGATAATGACAACGACTCTCGCAGTATAATGGCTTCCT 40 AAATTTACGATCAGGCGTGTACGGTTCAGGTGAATGGCTC 40 69.8 71.4 69.9 70.0 71.2 stiC stjB STM4595 Regulator for ﬂjA : DNA-invertase hin Serotyping Serotyping Serotyping Serotyping Serotyping hin ﬂiC AE008826 AE008787 AE008787 X04505 AL627272 LT2 LT2 ATTAGTCGGCTATTAGAGAAAGGCCATCCTCGGCAGCAAC 40 TGTCGCTGTTGACCCAGAATAACCTGAACAAATCCCAGTC 40 GTTGATAAGACGAACGGTGAGGTGACTCTTGCTGGCGGTG 40 ACTACCTTAGGTGGTACTCCTGCTATTACTGGTGATCTGA 40 TTAGCAAGCGACCTTGACAAACATAACTTCAGAACAGGCG 40 71.7 70.9 73.5 68.5 70.3 Filament structural protein , identiﬁes all H antigens Filament structural protein , identiﬁes i antigen Filament structural protein , identiﬁes r antigen Filament structural protein , identiﬁes d antigen LT2 01-2132 03-1949 Filament structural protein , identiﬁes g complex associated antigens Filament structural protein , identiﬁes m , t antigen Filament structural protein , identiﬁes l , w and l , v antigens Filament structural protein identiﬁes e , n , x and e , n , z15 antigens Filament structural protein , identiﬁes 1,2 1,5 , 1,6 , and 1,7 antigens Serotyping Serotyping Serotyping Serotyping Serotyping , m , t , w , n , x ,2 M84980 U06201 AJ292277 AJ292278 AE008826 00-04 GCGATAGCTGGTGCCATTAAAGGTGGTAAGGAAGGAGATA 40 CGGTAGTAACTGACACCACTGCTCCAACTGTTCCTGATAA 40 GTAACCGAAACGCAGCCAAAACCTGTAGCTCTCAGTACAG 40 CTACTGTTACAGGTGATACCGCTGTTACTAAGGTACAGGT 40 ACAATGCCTGCTGGTGCGACAACTAAAACAGAAGTACA 38 70.2 70.1 71.2 68.2 70.1 00-102 01-2132 99-601 LT2 AJ292284 AE008758 AE008792 AL627273 X60665 X60666 AE008792 X61917 Filament structural protein , identiﬁes e , h antigen a - 1-2 polymerase : serogroup B ﬂiC-e , h Serotyping wzy a ( 1-2 ) Serotyping rfbD Serotyping rfbE ( A , D ) Serotyping orf 9.6 Serotyping orf 17.4 Serotyping abe ( B ) Serotyping abe ( C2-C3 ) Serotyping prt ( A , D ) Serotyping wbaV ( B ) Serotyping wbaU ( B , D ) Serotyping wbaA Serotyping wbaV ( D1 ) Serotyping wbaO ( E1 , D2 ) Serotyping wzy a ( 1-6 ) Serotyping 99-929 LT2 LT2 00-04 99-929 99-929 LT2 99-601 GGCAAGTACTATGCTGCAACCTATGACGAAGGTACAGGTA 40 GCTTTCGGGTATGGTGAACTATACGCAGATTTCGGGCTTT 40 TCTGCCTCAATGGGAATTAGGAGTTAAGCGTATGCTGACT 40 GGTGTTCAGGCATTCATCAATGTATGGTGGGAGACAGTTT 40 TGGTTATGTCGGAGTATTCCTGCATGGTTTAATCTTGGGT 40 ATGGTCCGTTCCTGTCTACATTGCATTAGGTTTGCTACTG 40 ACCTTCATATACTGAGTATCAAGTTGGAACTGGTGCTGGG 40 TGTCCTATTACCAACAAGACTGCTTGAGTTAATGCCAGCG 40 70.1 71.1 70.0 69.9 69.1 69.6 68.9 69.8 TDP-dehydrorhamnose synthetase : Serogroup A,B,C2-C3 , D1 , D2 CDP-tyvelose-2-epimerase : serogroup A and D a O-polysaccharide polymerase : serogroup E1 b O-polysaccharide polymerase : serogroup E1 and E4 CDP-abequose synthase : serogroup B CDP-abequose synthase : serogroup C2 -- C3 Paratose synthase : serogroups A and D Abequosyl transferase : serogroup B Manosyl transferase ( a 1-4 linkage ) : serogroups B and D1 O-antigen-polymerase : serogroup C1 Tyvelosyl transferase : serogroup D Manosyl transferase ( b 1-4 linkage ) : serogroups E1 and D2 a - 1-6 polymerase : O27 serovar factor AL627273 X56793 X56793 M84642 M65054 X60665 AF017148 00-04 LT2 CGACATACTGTGATTGGCTTAGCAAGGAAGAGGAACAATG 40 GCAGTGATGATGCTCTTGCGAAAGACTCGTTAGCGATATT 40 CTGATGTTGACGCAATAATCTGTAGCAAGGTACACGCTGA 40 ATTACTTGTGCTTGGTGCCATTCTATCATTGCCTTTGTCA 40 ATCTAATCCACATCGGCGATGGTTAAATGGTGGCAGTAGA 40 GGCTCCCAGACCTTTACTGAAGTTGATCGGCAAGTGTATA 40 ATGGCAAGTCGTGTCTGGAATATCTCGATGGGATTATCAG 40 68.9 70.0 69.8 69.0 70.2 70.2 69.0 LT2 00-102 00-04 99-929 98-3363 03-1949 LT2 LT2 LT2 LT2 O antigen ﬂippase : O6 ,14 serovar factor Fels-1 prophage protein Fels-2 prophage protein Fels-2 prophage protein Gifsy-1 prophage protein Serotyping Phages Phages Phages Phages wzx ( O6 ,14 ) STM0900 STM2740 STM2701 STM2616 AY334017 AE008737 AE008824 AE008823 AE008819 69.9 69.3 70.7 70.0 69.8 TAAAGCGACCTTGAGTATTGGGCTCACTGCTGTAGTAGTT 40 GTTTCATTGAGTACGGATTATCGGGTGACGAAATCGGGTA 40 GAACAGTGGCTCAAAGAGAAGAAACGGACCAGCGATCTTA 40 CTTAAAGCCGGGAAACTGACCATCGATTATGACTACACGC 40 CGGCAGGTTGGTCTAGACATGTTCGTTGGAAAAGTAGAG 39 ACTTACAACCTAACCAGTGATATTGATGCTGCGGCCTATC 40 CAGAACACCAAGATAATCCTTCGCAATTACGCCTCCAACA 40 TAAGAACACTGAGTCTTCTGCAACACACTTTCACCGAGGA 40 CTATGCTCGTCAGACATGCGAAGCCATATTATCAGCCGTG 40 GTTGAAGAGGGTATGCGTGAAGCCAAAGAGATGATGGATG 40 LT2 LT2 Gifsy-2 prophage putative type III secreted protein Gifsy-2 prophage protein Translocated effector protein , encoded by P2-like bacteriophage Secreted outer protein , phage remnant Putative methyl-accepting chemotaxis prophage protein Phages Phages Phages Phages Phages SseI gtgA AE008743 AF254762 AE016848 AF200952 AE008896 69.4 70.0 70.2 71.1 70.1 99-601 LT2 LT2 sopE1 sopE2 STM4210 GCCTCATGTGAACGATAGCAGAACCATATTAGCGAAGGTG 40 CGGAAGTTCTGGCGAACGGTGAACAAAATCTCTACCACAG 40 TTTCAGTAGATGTTTCCGACAATGGTATTTCTGGCGTGGC 40 GAAGACTGGCAAGAAGTTGAGGTTATCGGCTGGCTGTATC 40 SUO1 LT2 LT2 LT2 Putative prophage Cro protein Glycerate kinase II Phages Others Others Others cro glxK AF378725 AE008720 AE008784 AE008911 69.9 71.4 70.1 70 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17029709" target="_blank">17029709</a>]<br></li>
<li>MarR has also been shown to bind other anionic compounds , including 2,4-dinitrophenol , plumbagin and menadione ( Schumacher & Brennan , 2002 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15073288" target="_blank">15073288</a>]<br></li>
<li>[ 11 ] Haque MM , Kabir MS , Aini LQ , Hirata H , Tsuyumu S. SlyA , a MarR family transcriptional regulator , is essential for virulence in Dickeya dadantii 3937 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29857034" target="_blank">29857034</a>]<br></li>
<li>Besides phoP/phoQ ( 19 ) and rpoS ( 27 ) , a new regulatory gene slyA was cloned by Libby et al. ( 15 ) from Salmonella serovar Typhimurium as a cytolysin-encoding gene , but later it was proved to be a gene encoding a transcriptional regulatory protein ( 5 ) , which belongs to a family of regulators including EmrR ( 16 ) , MarR ( 28 ) , HprR ( 23 ) , and PecS ( 21 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11939575" target="_blank">11939575</a>]<br></li>
<li>g001 STM1444 slyA transcriptional regulator for hemolysin ( MarR family ) [ 31 ] STM1753 hnr response regulator in protein turnover [ 96 ] STM2640 rpoE sigma E factor of RNA polymerase , response to periplasmic stress [ 27 ] STM2688 smpB small protein B ; tmRNA-binding protein [ 97 ] STM2826 csrA carbon storage regulator [ 30 ] STM2924 rpoS sigma S factor of RNA polymerase , major sigma factor during stationary phase [ 98 ] catabolite activator protein ( CAP ) , cyclic AMP receptor protein ( CRP family ) STM3466 crp [ 99 ] two-component regulatory system responding to osmolarity and pH STM3501/2 envZ/ompR [ 100 ] host factor I for bacteriophage Q beta replication , a growth-related protein STM4361 hfq [ 33 ] doi :10.1371 / journal.ppat .1000306 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19229334" target="_blank">19229334</a>]<br></li>
<li>All S. Typhimurum ; S. Heidelberg , S. St. Paul ; S. Choleraesuis ; S. Dublin ; S. Weltevreden ; S. Enteritidis ; S. Schwarzengrund ; Escherichia coli O103 : H2 and other E. coli strains ; Shigella boydii es Yes 24.7 Y Yes No No No 26.2 TABLE 2 Effect of STM S. Typhimurium LT2 locus identity STM2239a STM2237 STM2236 STM2235 STM2234 STM2233 STM3881 2239 overexpression on transcription of other genes Gene name Median fol change ( n 6.46 5.09 4.94 4.78 4.49 4.02 3.53 d P value by t test ( n 3 ) 1.63 E 22 1.83 E 15 2.07 E 20 4.71 E 43 3.41 E 23 1.05 E 16 2.49 E 12 3 ) Predicted function Putative phage protein ; homology to antiterminator protein Q of phage P5 Putative inner membrane protein Hypothetical protein Hypothetical protein Putative phage tail ﬁber assembly protein Putative cytoplasmic protein D-Ribose pyranase ; catalyzes the conversion between beta-pyran and beta-furan forms of D-ribose Putative outer membrane protein 2-Oxo-3-deoxygalactonate 6-phosphate aldolase/galactonate dehydratase Ornithine decarboxylase Pyruvate dehydrogenase subunit E1 ; E1 component ; part of pyruvate dehydrogenase ; forms a complex with DlaT and LpdC Dihydrolipoamide acetyltransferase ; E2 component of pyruvate dehydrogenase multienzyme complex 2-Keto-3-deoxygluconate permease ; transports degraded pectin products into the bacterial cell 2-Dehydro-3-deoxyphosphogalactonate aldolase S-Adenosylmethionine synthetase ; catalyzes the formation of S-adenosylmethionine from methionine and ATP Phosphate acetyltransferase ; catalyzes the synthesis of acetylphosphate or propionylphosphate from acetyl-CoA or propionyl-CoA and inorganic phosphate Acetate kinase ; AckA utilizes acetate and can acetylate CheY , which increases signal strength during ﬂagellar rotation ; also involved in conversion of acetate to acetyl-CoA DNA-binding transcriptional activator MarA ; transcriptional activator of genes involved in the multiple antibiotic resistance ( Mar ) phenotype Ribose ABC transport system , ATP-binding protein RbsA ; involved in D-ribose utilization Putative inner membrane protein Multiple antibiotic resistance protein MarB 2-Oxo-3-deoxygalactonate kinase High-afﬁnity gluconate permease Putative outer membrane protein DNA-binding transcriptional repressor MarR ; involved in the activation of both antibiotic resistance and oxidative stress genes Respiratory NADH dehydrogenase 2 ; cupric reductase ( 2nd module ) Valine tRNA 2A Outer membrane receptor FepA ; outer membrane receptor of ferric enterobactin and colicins B and D ; interacts with the TonB-ExbBD Putative cytoplasmic protein Ribose ABC transporter permease protein ; functions to transport ribose at high afﬁnity ; forms a complex with RbsA2C2B Heat shock chaperone IbpB ; associates with aggregated proteins , together with IbpA Putative amino-acid-transport protein Enterobactin/ferric enterobactin esterase Putative outer membrane protein Putrescine transporter Valine tRNA 2B Galactonate operon transcriptional repressor UmuD protein , putative Virulence protein ; similar to virulence protein MsgA ( STM1241 ) ; SsrB-regulated factor O-antigen acetylase ( lipopolysaccharide ) ; O-antigen ﬁve Leucine-rich repeat-containing protein ; induced by the SPI-2 regulator SsrA/B ; secreted by SPI-2 Hypothetical protein Putative tail ﬁber protein of phage ; similar to Escherichia coli putative membrane protein ( AAC74454 .1 ) Homolog of virulence protein MsgA-related protein Putative outer membrane protein Transcriptional regulator NarP ; nitrate/nitrite response regulator rbsD 3.18 3.17 3.12 3.06 ycfR dgoA speF aceE STM1214 STM3828 STM0701 STM0152 3.53 E 09 2.88 E 39 2.87 E 76 1.88 E 108 2.14 E 04 STM0153 2.90 aceF 1.28 E 37 2.80 STM0161 kdgT 3.47 E 21 5.13 E 36 2.80 2.72 STM3828 .1 N STM3090 metK 1.53 E 63 pta 2.70 STM2338 5.27 E 47 2.66 STM2337 ackA 9.14 E 19 2.61 STM1519.S marA 47 37 08 19 38 13 12 7.30 E 2.36 E 9.44 E 1.42 E 9.62 E 5.16 E 1.15 E 2.59 2.53 2.46 2.42 2.40 2.36 2.35 STM3882 STM0162 STM1518 STM3829 STM3512 STM3361 STM1520 rbsA marB dgoK gntT yhcN marR 36 02 57 1.48 E 1.31 E 3.44 E 2.32 2.31 2.25 STM1211 STM1423 STM0585 ndh valW fepA 1.45 E 06 1.57 E 22 2.25 2.24 STM0587 STM3883 ybdZ rbsC 3.90 E 15 1.95 E 39 7.85 E 31 1.13 E 27 9.27 E 41 2.83 E 06 3.69 E 19 1.31 E 05 0.008566 2.37 E 42 1.17 E 12 2.23 2.15 STM3808.S STM2068 STM0586 STM0509 STM0700 STM1424 STM3830 STM2230 STM2231 STM2232 STM2241 ibpB yeeF fes 2.12 2.10 2.07 2.04 2.03 0.8144 0.20572 0.78253 0.19829 potE valV dgoR oafA sspH2 STM2242 STM2243 0.44144 0.296273 0.076021 0.00941 0.246198 0.603784 0.000117 STM2244 0.114917 STM2245 0.050032 STM2246 0.4088 a Overexpressed from pFLAG-STM2239 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23630960" target="_blank">23630960</a>]<br></li>
<li>THE GENES PREDICTED BY MICROARRAY TO BE REGULATED IN SALMONELLA ENTERICA TYPHIMURIUM 14028 EXPOSURED TO 1.6 MG/ML OF NALIDIXIC ACID , SORTED ACCORDING TO FOLD REGULATION ( 50 AND 50 INDICATE FOLD REGULATION ESTIMATED BEYOND 50-FOLD ) ( CONTINUED ) SOS-response regulator Activator protein 2.5 2.5 2.6 lexA pgtA potF transcription transcription-transport 0.001 0.001 0.011 16422795 16420940 16419385 ABC superfamily ( ) , putrescine-transporter Flagellar biosynthesis protein Putative salt-induced outer membrane protein Sigma E ( sigma 24 ) factor of RNA polymerase Rhamnosyl transferase DNA-replication initiator protein Glycosyltransferase Putative permease DNA-damage-inducible protein F Putative transcriptional regulatory protein Phob-dependent ATP-binding pho regulon component Putative invasin Outer membrane lipoprotein Putative fimbrial subunit Multiple antibiotic resistance protein Putative marr-family transcriptional regulator 1,4-alpha-glucan branching enzyme Putative oxidoreductase Putative cytoplasmic protein 2.6 2.6 fliM ydiY flagellum membrane 0.003 0.014 16420511 16419845 2.6 0.002 16421189 rpoE transcription 2.6 2.6 2.7 2.7 2.7 2.7 2.7 0.002 0.000 0.005 0.001 0.002 0.001 0.000 16420617 16422412 16422182 16420524 16422796 16422328 16419642 rfbN dnaA yhjO yedA dinF marT phoH transferase atp binding membrane membrane dna damage transcription atp binding 2.7 2.7 2.8 2.8 2.9 ychP blc stiA marB MarR 0.000 0.000 0.001 0.008 0.002 16420297 16422901 16418679 16420039 16420069 virulence lipoprotein fimbria transcription transcription glgB ydgJ 3.0 3.0 3.0 0.000 0.009 0.004 16422097 16419983 16422797 transferase oxidoreductase cytoplasmic protein metal binding cell-division , dna damage STM4239 FkbR2 ftsW uvrB Putative monoamine oxidase Essential cell-division gene 3.0 3.0 3.1 0.000 0.005 0.006 16421672 16418626 16419304 Uvrb with uvrac is a dna excision repair enzyme DNA primase Transcriptional activator of tdc operon Glycosyl hydrolase Putative outer membrane lipoprotein UDP-N-acetylmuramoylalanine-D-glutamate ligase Positive lysr family transcriptional regulator Fur regulated Salmonella iron-transporter Fur regulated Salmonella iron-transporter Regulator for maltose-metabolism UDP-N-acetylglucosamine : N-acetylmuramyl-transferase Putative outer membrane lipoprotein Component of Sec-independent protein-secretion pathway Iron-sulfur component of fhuf stability protein Rod shape-determining protein MFS superfamily nitrate extrusion protein Cytochrome BD2 subunit I Transcriptional regulator of Aas Suppressor of lon Putative outer membrane or secreted lipoprotein Transcriptional repressor of cel operon Cysteine sulfinate desulfinase Hemolysin expression modulating protein Uptake of enterochelin dnaG tdcA 3.1 3.2 3.2 3.2 3.3 transferase transcription hydrolase lipoprotein atp binding 0.005 0.000 0.000 0.002 0.000 16421767 16421803 16422096 16422811 16418625 glgX STM4253 murD transcription membrane membrane transcription membrane 0.000 0.002 0.000 0.002 0.004 3.3 3.3 3.3 3.4 3.4 LysR sitC sitB SfsA murG 16421564 16421409 16421408 16418690 16418627 lipoprotein membrane 16418577 16422539 0.003 0.001 3.4 3.5 DUF903 tatB metal-binding , 16419892 0.001 3.7 sufD 16421932 16420099 16418862 16420210 16419586 16420083 0.000 0.009 0.000 0.006 0.000 0.000 3.7 3.7 3.8 3.8 3.8 3.9 mreD narU membrane oxidoreductase transcription membrane lipoprotein CydA tyrR sulA DUF1264 transcription transcription virulence membrane celD deaD hha exbB 4.0 4.1 4.3 4.3 16419833 16421839 16418980 1642171 0.004 0.002 0.001 0.000 TABLE 1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17600486" target="_blank">17600486</a>]<br></li>
<li>Proteins encoded by marRAB include MarR , a repressor with sequence identity to known negative regulators [ 4,5 ] ; MarA , a positive transcriptional activator that when overexpressed results in a multidrug-resistance ( mdr ) phenotype [ 6 ] ; and MarB , a protein which is essential for resistance but with an unknown function [ 4 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10856650" target="_blank">10856650</a>]<br></li>
<li>MarR acts as a local repressor by binding to the operator region ( marO ) , which negatively regulates the expression of marA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25128419" target="_blank">25128419</a>]<br></li>
<li>The Salmonella SlyA protein is a transcriptional regulator of the MarR family that was originally identified based on its ability to induce expression of a cryptic hemolysin in E. coli K-12 ( 18 , 19 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15208313" target="_blank">15208313</a>]<br></li>
<li>Required for bacteriophage lambda integration ; bends DNA and significantly changes transcriptional regulation of many genes Required for the bacterial stringent-response that results in reduced transcription in the presence of uncharged t-RNA MarR family transcription regulator i.g. ; i.p. .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21687430" target="_blank">21687430</a>]<br></li>
<li>PhoP-Responsive Expression of the Salmonella enterica Serovar Typhimurium slyA Gene Valia A. Norte , Melanie R. Stapleton , and Jeffrey Green * Department of Molecular Biology and Biotechnology , Krebs Institute for Biomolecular Research , University of Shefﬁeld , Shefﬁeld S10 2TN , United Kingdom Received 11 October 2002/Accepted 22 March 2003 The SlyA protein of Salmonella enterica serovar Typhimurium is a member of the MarR family of transcription regulators and is required for virulence and survival in professional macrophages .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12775687" target="_blank">12775687</a>]<br></li>
<li>Transketolase A , an enzyme in central metabolism , derepresses the marRAB multiple antibiotic resistance operon of Escherichia coli by interaction with MarR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27909434" target="_blank">27909434</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19346309" target="_blank">19346309</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_14">
Sulavik M.C. , Gambino L.F. , Miller P.F. ( 1995 ) The MarR repressor of the multiple antibiotic resistance ( mar ) operon in Escherichia coli : prototypic member of a family of bacterial regulatory proteins involved in sensing phenolic compounds .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19120970" target="_blank">19120970</a>]<br></label>
<input id="rp_similarity_14", type="checkbox">
<div class="RP_similar">
<li>The MarR repressor of the multiple antibiotic resistance ( mar ) operon in Escherichia coli : prototypic member of a family of bacterial regulatory proteins involved in sensing phenolic compounds .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11229910" target="_blank">11229910</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9068629" target="_blank">9068629</a>]<br></li></div>
				<label class="RP_collapse_similarity" for="rp_similarity_16">
Seoane AS & Levy SB ( 1995 ) Characterization of MarR , the repressor of the multiple antibiotic resistance ( mar ) operon in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17156099" target="_blank">17156099</a>]<br></label>
<input id="rp_similarity_16", type="checkbox">
<div class="RP_similar">
<li>Characterization of MarR , the repressor of the multiple antibiotic resistance ( mar ) operon in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10613858" target="_blank">10613858</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20237076" target="_blank">20237076</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15073288" target="_blank">15073288</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9068629" target="_blank">9068629</a>]<br></li></div>
<li>This family of proteins includes MarR and EmrR , which regulate the expression of membrane efﬂux systems that confer resistance to a wide range of toxic compounds ( 26 , 28 ) and also exert some regulatory inﬂuence over genes involved in resistance to oxidative stress .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9284144" target="_blank">9284144</a>]<br></li>
<li>MarR and EmrR regulate a set of genes involved in multiple antibiotic resistance in E. coli , and homologous loci have been identiﬁed in Salmo - nella .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9284144" target="_blank">9284144</a>]<br></li>
<li>The family includes : MarR and EmrR ( Escherichia coli ) , which regulate genes involved in multiple antibiotic resistance ; PecS ( Erwinia chrysanthemi ) , which controls pectinase and cellulase production , facilitating the infection of fruit ; HprR ( Bacillus subtilis ) required for hydrogen peroxide resistance and sporulation regulation ; and RovA ( Yersinia enterocolitica and Y. pseudotuberculosis ) required for the regulation of inva-sin expression ( 1 -- 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11882648" target="_blank">11882648</a>]<br></li>
<li>For example , the Escherichia coli MarR and EmrR proteins regulate genes involved in antibiotic resistance , PecS from Erwinia chrysanthemi controls pectinase and cellulase production , and RovA of Yersinia spp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12775687" target="_blank">12775687</a>]<br></li>
				</div></ul><br>
			<label class="collapse_SIT" for="_5"><font size="5px">MarR binding site</font></label>
				<input id="_5" type="checkbox">
				<div class="SIT"><br><ul><li>In this respect SlyA resembles the MarR protein , which is thought to adopt a dimeric state to recognize an inverted-repeat composed of pentameric subelements ( 23 ) , possibly through interaction with two putative helix-turn-helix motifs ( 24 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11882648" target="_blank">11882648</a>]<br></li>
<li>MarR binding sequences in the marRAB promoters are italicized grey marked and the binding site of Fis is marked in dark grey and underlined .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19120970" target="_blank">19120970</a>]<br></li>
<li>This suggests a plausible model for regulating the ability of SlyA to bind DNA in which SlyA interacts with DNA in a similar manner to OhrR , with a conserved interaction between the wing region and the DNA minor groove , and sequence specific interactions in the major groove mediated by amino acids in α4 ( including Arg-65 of SlyA ) , but that in the presence of an as yet unidentified ligand Glu-59 ′ might interact with Arg-65 to inhibit SlyA DNA-binding by repositioning the DNA binding lobes in a manner similar to that suggested by the MarR : salicylate complex [ 1,22 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19204896" target="_blank">19204896</a>]<br></li>
<li>A transcription factor involved in this process is SlyA , a member of the MarR transcription factor family , which recognises inverted-repeats in DNA sequences ; TTAN6TAA has been determined to be its consensus sequence [ 6e8 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29857034" target="_blank">29857034</a>]<br></li>
				</ul></div><br>
			<label class="collapse_TU" for="_6"><font size="5px">Transcription units regulated by MarR</font></label>
				<input id="_6" type="checkbox">
				<div class="TU"><br><ul><li>MarA regulates its own transcription as well as regulating the expression of the mar regulon , whereas MarR acts by repressing marRAB transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19104017" target="_blank">19104017</a>]<br></li>
<li>The repressor MarR ( 43 ) , encoded by marR , binds to the marO operator region ( 39 ) to negatively regulate expression of marRAB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11229910" target="_blank">11229910</a>]<br></li>
<li>MarR negatively regulates the expression of marRAB by binding to the marO operator region .17 On the other hand , proteins encoded by the soxRS locus include the transcriptional activator , SoxS , and another protein , SoxR , the oxidized form of which can activate soxS expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16201927" target="_blank">16201927</a>]<br></li>
				<label class="TU_collapse_similarity" for="tu_similarity_4">
MarR acts as a local repressor by binding to the operator region ( marO ) , which negatively regulates the expression of marA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25128419" target="_blank">25128419</a>]<br></label>
<input id="tu_similarity_4", type="checkbox">
<div class="TU_similar">
<li>MarR negatively regulates the expression of marRAB by binding to the marO operator region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18443112" target="_blank">18443112</a>]<br></li>
<li>MarR negatively regulates expression of marRAB by binding to the marO operator region [ 13,15 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15336432" target="_blank">15336432</a>]<br></li></div>
<li>However , micF and acrAB are also induced by stress signals in additional ways that are not dependent on the marRAB system .17 Interestingly , antibiotics that serve as substrates of the AcrAB efﬂux pump are very weak inducers of the marRAB operon in E. coli and do not induce these genes in Salmonella .6,8 Based on that , it was suggested that exposure of E. coli or S. enterica to salicylate activates MarA through binding to its repressor , MarR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20237076" target="_blank">20237076</a>]<br></li>
<li>Thus , MarA ( an activator of marRAB ) and MarR ( a repressor of marRAB ) are controlled by the same promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19120970" target="_blank">19120970</a>]<br></li>
<li>Intracellular levels of MarA are controlled by MarR that binds to marO and negatively modulates expression of the marRAB operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16842216" target="_blank">16842216</a>]<br></li>
<li>The similarity to E. coli of nucleotide sequences of the operator/promoter regions , Northern data on inducibility of the operon by SAL , and structural and functional similarities of MarR to its E. coli counterpart all suggests that regulated control of marRAB expression in S. typhimurium is nearly identical to that of E. coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9068629" target="_blank">9068629</a>]<br></li>
<li>In E. coli it is modulated at the lowest level by the local repressor AcrR .14 At a more global level , acrAB expression is modulated by stress conditions and by global regulatory genes , such as marA , soxS , and rob .14,17,18 The acrAB locus is indeed part of the mar , sox , and rob regulons of E. coli , the activation of which confers a low-level resistance to a wide range of antimicrobial agents and organic solvents .17 Previous studies clearly showed that overexpression of marA , which results from marR mutations , would down-regulate the expression of acrR , thus inducing high-level AcrAB expression and antimicrobial resistance .23 Proteins encoded by the mar locus include the transcriptional activator , MarA , its local repressor , MarR , and MarB and MarC , two proteins with unknown functions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16201927" target="_blank">16201927</a>]<br></li>
<li>For instance , in Staphylococcus aureus and Escherichia coli it modulates the transcription of dozens of genes .3,4 Salicylate is also associated with the production of factors involved with bacterial virulence5 and with reduction in the resistance to a variety of antimicrobials .2,3,6 In E. coli and Salmonella enterica , the global regulators MarA and MarR are induced by salicylate .7 MarR binds the mar operator that ﬂanks the promoter of marRAB and represses the transcription of these genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20237076" target="_blank">20237076</a>]<br></li>
<li>Expression of MarA and MarR is induced by the prevention of MarR binding to the operator ,8 and this may occur by the binding of MarR to salicylate .6,9 Salicylate is , in fact , the most potent known inducer of marRAB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20237076" target="_blank">20237076</a>]<br></li>
<li>marA transcription is repressed by MarR and is de-repressed by compounds such as salicylate , benzoate , dinitrophenol and plumbagin , probably through their binding to MarR ( 12 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19120970" target="_blank">19120970</a>]<br></li>
<li>In this context , marA is regulated negatively by MarR ( Alekshun and Levy 1999 ) , which by binding ligands such as salicylate , lowers its afﬁnity for DNA , allowing derepression of the marRAB operon ( Perera et al. 2009 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22752112" target="_blank">22752112</a>]<br></li>
				</div></ul></div><br>
	</div>
</body>
</html>