<!doctype html>
<style>
@charset "utf-8";
body {
	height: 100%;
	margin: 0;
	padding: 0;
}

.banner {
	position: fixed;
	top: 0;
	width: 100%;
	height: 65px;
	background: gray;
	line-height: 65px;
}
.spacer {
	height: 10px;
}

.main-content {
	padding: 5%;
}

.collapse_ACT {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-style: none solid none none;
	border-color: #B4045F;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_ACT + input{
	display: none;
}
.collapse_ACT + input + div{
	display: none;
}
.collapse_ACT + input:checked + div{
	display: block;
}
.ACT {
	text-align: justify;
	text-justify: inter-word;
}
.ACT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B4045F;
}
.ACT_collapse_similarity:hover{
	background: #e5d3dc;
}
.ACT_collapse_similarity + input {
	display: none;
}
.ACT_collapse_similarity + input + div{
	display: none;
}

.ACT_collapse_similarity + input:checked + div{
	display: block;
}
.ACT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.ACT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B4045F;
	padding-left: 5px;
}
.ACT_collapse_similarity:hover{
	background: #e5d3dc;
}
.ACT_collapse_similarity + input {
	display: none;
}
.ACT_collapse_similarity + input + div{
	display: none;
}
.ACT_collapse_similarity + input:checked + div{
	display: block;
}
.ACT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.ACT_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #B4045F;
}
.ACT_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_DOM {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-style: none solid none none;
	border-color: #B404AE;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_DOM + input{
	display: none;
}
.collapse_DOM + input + div{
	display: none;
}
.collapse_DOM + input:checked + div{
	display: block;
}
.DOM {
	text-align: justify;
	text-justify: inter-word;
}
.DOM_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B404AE;
	padding-left: 5px;
}
.DOM_collapse_similarity:hover{
	background: #e8d7e7;
}
.DOM_collapse_similarity + input {
	display: none;
}
.DOM_collapse_similarity + input + div{
	display: none;
}
.DOM_collapse_similarity + input:checked + div{
	display: block;
}
.DOM_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}.DOM_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #B404AE;
}
.DOM_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_EVO {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #7401DF;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_EVO + input{
	display: none;
}
.collapse_EVO + input + div{
	display: none;
}
.collapse_EVO + input:checked + div{
	display: block;
}
.EVO {
	text-align: justify;
	text-justify: inter-word;
}
.EVO_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #7401DF;
	padding-left: 5px;
}
.EVO_collapse_similarity:hover{
	background: #e2d7ed;
}
.EVO_collapse_similarity + input {
	display: none;
}
.EVO_collapse_similarity + input + div{
	display: none;
}
.EVO_collapse_similarity + input:checked + div{
	display: block;
}
.EVO_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.EVO_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #7401DF;
}
.EVO_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_RP {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #0174DF;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_RP + input{
	display: none;
}
.collapse_RP + input + div{
	display: none;
}
.collapse_RP + input:checked + div{
	display: block;
}
.RP {
	text-align: justify;
	text-justify: inter-word;
}
.RP_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #0174DF;
	padding-left: 5px;
}
.RP_collapse_similarity:hover{
	background: #c9d7e2;
}
.RP_collapse_similarity + input {
	display: none;
}
.RP_collapse_similarity + input + div{
	display: none;
}
.RP_collapse_similarity + input:checked + div{
	display: block;
}
.RP_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.RP_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #0174DF;
}
.RP_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_SIT {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #0489B1;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_SIT + input{
	display: none;
}
.collapse_SIT + input + div{
	display: none;
}
.collapse_SIT + input:checked + div{
	display: block;
}
.SIT {
	text-align: justify;
	text-justify: inter-word;
}
.SIT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #0489B1;
	padding-left: 5px;
}
.SIT_collapse_similarity:hover{
	background: #d0dfe2;
}
.SIT_collapse_similarity + input {
	display: none;
}
.SIT_collapse_similarity + input + div{
	display: none;
}
.SIT_collapse_similarity + input:checked + div{
	display: block;
}
.SIT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.SIT_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #0489B1;
}
.SIT_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_TU {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #04B486;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_TU + input{
	display: none;
}
.collapse_TU + input + div{
	display: none;
}
.collapse_TU + input:checked + div{
	display: block;
}
.TU {
	text-align: justify;
	text-justify: inter-word;
}
.TU_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #04B486;
	padding-left: 5px;
}
.TU_collapse_similarity:hover{
	background: #d0e2de;
}
.TU_collapse_similarity + input {
	display: none;
}
.TU_collapse_similarity + input + div{
	display: none;
}
.TU_collapse_similarity + input:checked + div{
	display: block;
}
.TU_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.TU_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #04B486;
}
.TU_similar > a {
	text-decoration: none;
	color: gray;
}

ul { 
	list-style-type: none;
	padding: 0;
	margin: 0;
}
ul li > a {
	text-decoration: none;
	color: gray;
}
ul li:hover, ul li:focus {
	background: #F0F0F0;
}

ul li:first-child span {
  border-left:none;  
}

</style>
<html>
<head>
<meta charset="utf-8">
<title>TF properties - MetR</title>
<meta name="theme-color" content="#ffffff">
</head>
<body>
	<div class="banner">
		<center><font color="white" size="16px" face="arial">MetR</font></center>
	</div>
	<div align="center" class="main-content">
		<div class="spacer">
	</div><br>			<label class="collapse_ACT" for="_1"><font size="5px">MetR transcriptional activity</font></label>
				<input id="_1" type="checkbox">
				<div class="ACT"><br><ul><li>Interestingly , while OxyR , LysR , NodD , IlvY , and MetR all negatively regulate their own genes , they positively regulate a transcript from an overlapping promoter on the complementary strand ( 29 , 36 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2471187" target="_blank">2471187</a>]<br></li>
<li>It is not known how bound MetR protein activates transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1904437" target="_blank">1904437</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_3">
Lane 3 , no MetR protein ; lanes 4 through 8 , 0.5 , 1.0 , 2.0 , 4.0 , and 8.0 ng of MetR protein , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1904437" target="_blank">1904437</a>]<br></label>
<input id="act_similarity_3", type="checkbox">
<div class="ACT_similar">
<li>Lane 3 , no MetR protein ; lanes 4 through 7 , 0.06 , 0.32 , 1.6 , and 8.0 ng of MetR protein , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1904437" target="_blank">1904437</a>]<br></li></div>
				<label class="ACT_collapse_similarity" for="act_similarity_5">
to Lanes 3 , no MetR protein ; lanes 4 through 7 , 0.25 , 0.5 , 1 , and 2 , ug of MetR protein per ml , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8376333" target="_blank">8376333</a>]<br></label>
<input id="act_similarity_5", type="checkbox">
<div class="ACT_similar">
<li>Lane 1 , no MetR protein ; lanes 2 through 4 , 0.25 , 0.5 , and 1 , ug of MetR protein per ml , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8376333" target="_blank">8376333</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8376333" target="_blank">8376333</a>]<br></li></div>
<li>When MetR binds at this site , transcription of metE increases while transcription of metR decreases ( 6 , 51 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1657882" target="_blank">1657882</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_8">
The effect of on MetR homocysteine regulation of metE , metR and metH expression in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1657882" target="_blank">1657882</a>]<br></label>
<input id="act_similarity_8", type="checkbox">
<div class="ACT_similar">
<li>The effect of homocysteine on MetR regulation of metE , metR and metH expression in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1385596" target="_blank">1385596</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1904437" target="_blank">1904437</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1729233" target="_blank">1729233</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7896708" target="_blank">7896708</a>]<br></li></div>
<li>These results suggest that MetR is required for activation of metF-lacZ only under MetJ-mediated repressing conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8376333" target="_blank">8376333</a>]<br></li>
<li>These results suggest that MetR binding is necessary but not sufficient for activation of the metHlacZ fusion .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1904437" target="_blank">1904437</a>]<br></li>
<li>These results suggest that the role of MetR becomes more important as MetJ-mediated repression is increased .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8376333" target="_blank">8376333</a>]<br></li>
<li>This location is probably too far downstream to allow bound MetR protein to directly block transcription initiation by RNAP the at metR Another promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2676984" target="_blank">2676984</a>]<br></li>
<li>Template DNA ( ﬁnal concentration , 2 to 3 nM ) was allowed to-bind to MetR protein in the presence or absence of 2 mM D,L-homocysteine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7896708" target="_blank">7896708</a>]<br></li>
<li>The labeled DNA fragment was shifted to three positions of slower mobility as the MetR concentration was increased from 0.25 to 1 , ug/ml ( Fig. 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8376333" target="_blank">8376333</a>]<br></li>
<li>In addition to MetJ-mediated repression , at least two genes , the metE and metH genes , are activated by MetR , a DNA-binding protein ( 4 , 28 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1729233" target="_blank">1729233</a>]<br></li>
<li>In contrast , for dimeric MetR , the C terminus seems to be dispensable for DNA binding and the protein binds to approximately 25 bp ( reference 24 and references therein ; 38 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7868603" target="_blank">7868603</a>]<br></li>
<li>In Salmonella , for example , NO-Fe-Fur derepresses transcription of hmp ( 94 ) , whereas in E. coli hmp is derepressed by NO-Fe-Fnr and induced in response to intracellular nitrosylated homocysteine levels and MetR binding to the glyA-hmp intergenic region ( 60 , 95 , 96 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26443513" target="_blank">26443513</a>]<br></li>
<li>First , the cell extract containing the overexpressed MetR protein can bind the probe , even when diluted as much as 16,000-fold .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2676984" target="_blank">2676984</a>]<br></li>
<li>JOURNAL OF BACrERIOLOGY , Sept. 1993 , p. / 93/185862 -05 $ 02.00 / 0 Copyright C ) 1993 , American Society for Microbiology Regulation of the Salmonella typhimurium metF Gene by the MetR Protein JOHN M. COWAN , MARK L. URBANOWSKI , MICHAL TALMI , AND GEORGE V. STAUFFER * Department of Microbiology , The University of Iowa , Iowa City , Iowa 52242 Received 22 March 1993/Accepted 7 July 1993 The metF gene in Escherichia coli and Salmonella typhimurium is under negative transcriptional control by the MetJ repressor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8376333" target="_blank">8376333</a>]<br></li>
<li>JOURNAL OF BACTERIOLOGY , June 1991 , p. / 91/113547 -07 $ 02.00 / 0 Copyright © 1991 , American Society for Microbiology The MetR etH Gene : Binding Site in the Salmonella typhimurium m DNA Sequence Constraints on Activation KYNA A. BYERLY , MARK L. URBANOWSKI , AND GEORGE V. STAUFFER * Department of Microbiology , University of Iowa , Iowa City , Iowa 52242 Received 26 December 1990/Accepted 1 April 1991 Transcription of the metH gene in Salmonella typhimurium and Escherichia coli is positively regulated by the metR gene product , a DNA binding protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1904437" target="_blank">1904437</a>]<br></li>
<li>JOURNAL OF BACTERIOLOGY , July 1992 , p. / 92/144833 -05 $ 02.00 / 0 Copyright © 1992 , American Society for Microbiology ole of the MetR Regulatory System in Vitamin B12-Mediated Repression of the Salmonella typhimurium metE Gene R WHI-FIN WU , MARK L. URBANOWSKI , AND GEORGE V. STAUFFER * Department of Microbiology , University of Iowa , Iowa City , Iowa 52242 Received 13 February 1992/Accepted 8 May 1992 The vitamin B12 ( B. 2 ) - mediated repression of the metE gene in Escherichia coli and Salmonella typhimurium requires the B12-dependent transmethylase , the metH gene product .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1385596" target="_blank">1385596</a>]<br></li>
<li>MetR can activate transcription when added at 15 nM but only in the presence of homocysteine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7896708" target="_blank">7896708</a>]<br></li>
<li>In the metJ metR lysogen 748AAlac and the metJ metB metR lysogen 761AAlac , where the MetR protein is inactivated , metA expression is decreased 2.9 - to 3.1-fold ( Table Since metR 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1729233" target="_blank">1729233</a>]<br></li>
<li>Since the MetR protein positively regulates metH expression , we tested whether MetR could bind to the metH control region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1904437" target="_blank">1904437</a>]<br></li>
<li>A gel mobility shift assay was used to determine whether the MetR protein binds to the metA control region and whether homocysteine contributes to the binding of this protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1729233" target="_blank">1729233</a>]<br></li>
<li>MetR also binds to the promoter of metH located 3 ' to the metE gene ( 6 ) and glyA ( 43 ) to regulate transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1657882" target="_blank">1657882</a>]<br></li>
<li>Thus , the ability of HC to regulate metH-lacZ expression does not appear to be strongly dependent either on the efficiency with which MetR binds to the metH-lacZ control region or on the efficiency with which already bound MetR can activate metH-lacZ expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1904437" target="_blank">1904437</a>]<br></li>
<li>The negative regulatory effect of homocysteine on metH gene expression might also be attributed to a conformational change in MetR protein bound at the metH control region , similar to the change proposed to occur when MetR is bound at the metE region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2676984" target="_blank">2676984</a>]<br></li>
<li>However , instead of the facilitative effect proposed to occur for metE expression , this similar conformational change of MetR bound to the metH control region may inhibit transcriptional activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2676984" target="_blank">2676984</a>]<br></li>
<li>Furthermore , repression of the metE-lacZ fusion by B12 is dependent on a functional MetR protein ( Table 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1385596" target="_blank">1385596</a>]<br></li>
<li>However , in the absence of MetJ-mediated repression , metE-lacZ is still regulated over a 100-fold range by MetR ( 28 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8376333" target="_blank">8376333</a>]<br></li>
<li>Mutations -62 G , - SiT , and - SOG , which result in a 4 - to 10-fold reduction in binding affinity for the MetR protein in vitro ( Table 3 ) , still allow a significant response in vivo to the addition of HC ( Table 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1904437" target="_blank">1904437</a>]<br></li>
<li>The sequence protected by the bound MetR protein is indicated by the bracket in Fig. 2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1729233" target="_blank">1729233</a>]<br></li>
<li>However , there was no significant difference in the amount of DNA bound by the MetR protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1904437" target="_blank">1904437</a>]<br></li>
<li>Alternative models in which the MetH-B12 holoenzyme might be proposed to inhibit the activity of the MetR protein ( e.g. , by direct interaction or by methylation of MetR ) are not supported by our data .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1385596" target="_blank">1385596</a>]<br></li>
<li>In a metJ metR double mutant , however , the level of metF-lacZ expression is the same as in a metj mutant , suggesting that MetR antagonizes MetJ-mediated methionine repression of the metF promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8376333" target="_blank">8376333</a>]<br></li>
<li>Thus , residual MetR-mediated activation of the metE-lacZ fusion in this lysogen is probably due to MetR bound exclusively at the wild-type site 2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7896708" target="_blank">7896708</a>]<br></li>
<li>In the second class , activation of the metH-lacZ fusion was reduced , yet there was no appreciable reduction in MetR binding , indicating that the presence of bound MetR is not sufficient for activation of metH-lacZ gene expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1904437" target="_blank">1904437</a>]<br></li>
<li>We tested the effects of the mutations on MetR - and HC-mediated activation of the metH-lacZ gene fusion .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1904437" target="_blank">1904437</a>]<br></li>
<li>These changes severely reduce the affinity of the DNA for the MetR protein , resulting in reduced activation of the metH-lacZ fusion ( Fig. 3 ; Table 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1904437" target="_blank">1904437</a>]<br></li>
<li>Since homocysteine acts as a coactivator for the MetR protein to activate metE expression ( 23 ) , we tested whether the MetR protein is involved with homocysteine in overriding the B12-mediated repression of the metE-lacZ fusion .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1385596" target="_blank">1385596</a>]<br></li>
<li>The activation of the metE gene and the repression of the metR gene by the MetR protein is modulated by homocysteine ( 25 , 27 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2676984" target="_blank">2676984</a>]<br></li>
<li>This fact , in conjunction with earlier findings indicating that the metR gene is negatively autoregulated ( 25 ) , led us to question whether or not a single MetR-mediated event involving binding of the MetR protein to the metE metR control-region DNA affects both the activation of the metE gene and the negative autoregulation of the metR gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2676984" target="_blank">2676984</a>]<br></li>
<li>We propose that the mechanism of repression of the metR gene by the MetR protein may involve a competition by steric hindrance of RNAP bound to the metE promoter and RNAP bound to the metR promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2676984" target="_blank">2676984</a>]<br></li>
<li>It is not clear from these experiments whether the MetR protein activates metE-lacZ expression in the absence of homocysteine and homocysteine modulates this basal activation over only a four - to fivefold range or whether the endogenous pool of homocys teine produced from S-adenosylmethionine via the regenerative pathway ( Fig. 1 ) constitutes a sufficiently high level of the intermediate to potentiate metR activation in the absence of exogenously added homocysteine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2656646" target="_blank">2656646</a>]<br></li>
<li>The addition of 1 mM L-homocysteine did not significantly alter the affinity of binding of the MetR protein to the DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1729233" target="_blank">1729233</a>]<br></li>
<li>Like the CRP-cAMP response , the activity of these regulatory proteins is affected by the presence of a coregulatory molecule , and many LysWMetR members are involved in the feed-back regulation of metabolic pathways .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11370771" target="_blank">11370771</a>]<br></li>
<li>suggested that MetR activation of the metE and metH genes is mediated by one or more of the intermediates in methionine biosynthesis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2656646" target="_blank">2656646</a>]<br></li>
<li>While the functions of YeeY and STM2180 are uncharacterized , MetR is known to be involved in the regulation of methionine biosynthesis ( Maxon et al. , 1989 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22804842" target="_blank">22804842</a>]<br></li>
<li>Maxon , M.E. , Redﬁeld , B. , Cai , X.Y. , Shoeman , R. , Fujita , K. , Fisher , W. , et al. ( 1989 ) Regulation of methionine synthesis in Escherichia coli : effect of the MetR protein on the expression of the metE and metR genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22804842" target="_blank">22804842</a>]<br></li>
<li>Although the level of , B-galactosidase is low in lysogen 244XElac1 [ pGS192 ] because of the absence of MetR , it would have been possible to detect additional repression by B12 in this lysogen if B12 regulates metE by a MetR-independent mechanism .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1385596" target="_blank">1385596</a>]<br></li>
<li>The structure of the metE promoter region proposed here , in which a high-afﬁnity MetR-binding site enhances binding of MetR at a low-afﬁnity promoter-proximal activation site , is similar to the arrangement found in many of the promoters whose transcription is activated by LysR-type regulators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7896708" target="_blank">7896708</a>]<br></li>
<li>The addition of homocysteine reproducibly increased the afﬁnity of MetR for site 2 even in the absence of binding at site 1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7896708" target="_blank">7896708</a>]<br></li>
<li>As shown by the bar graph in Fig. 3A , in the absence of MetR protein , no activation of transcription occurs either with or without homocysteine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7896708" target="_blank">7896708</a>]<br></li>
<li>Symbols are the same as in panel A. DNase I attack ) shows that the afﬁnity of MetR for site 1 is relatively high in both the absence and presence of homocysteine ( 6 and 4 nM , respectively ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7896708" target="_blank">7896708</a>]<br></li>
<li>As the concentration of MetR is raised to 100 nM , however , signiﬁcant transcription occurs even in the absence of homocysteine and parallels the occupancy of site 2 by MetR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7896708" target="_blank">7896708</a>]<br></li>
<li>The in vivo results were supported by results of in vitro transcription runoff experiments in which the a1 ,2 mutant DNA template showed high MetR-dependent transcription in the presence or absence of homocysteine even at a low MetR concentration ( 15 nM ) ( Fig. 4 , lanes 7 and 8 ) ; at this low MetR concentration , homocysteine is required for the transcription of the wild-type template ( lanes 3 and 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7896708" target="_blank">7896708</a>]<br></li>
<li>MetR is another LysR-type regulator in Salmonella which positively regulates metA , metE and metH .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19447191" target="_blank">19447191</a>]<br></li>
<li>This hypothesis would predict that in a metJ mutant , in which all MetJ-mediated repression is abolished , MetR would have no function in metF-lacZ expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8376333" target="_blank">8376333</a>]<br></li>
<li>When DNA fragments from plasmids pGS201-61A , pGS201-62G , pGS201-51T , and pGS201-50G were used , MetR binding ability was decreased by 4 - to 14-fold , suggesting that these mutations changed bases necessary for MetR binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1904437" target="_blank">1904437</a>]<br></li>
<li>Nevertheless , there are some LTTRs , such as MetR , CatR , IlvY , NodD3 , Nac , and IciA , that form dimers in solution , and this seems to be their active form ( 36 -- 41 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24659766" target="_blank">24659766</a>]<br></li>
<li>Other LTTRs , such as MetR , NodD3 , and Nac , form functional dimers that have been shown to interact with DNA and activate or repress transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24659766" target="_blank">24659766</a>]<br></li>
				</div></ul><br>
			<label class="collapse_DOM" for="_2"><font size="5px">MetR protein domains</font></label>
				<input id="_2" type="checkbox">
				<div class="DOM"><br><ul>				<label class="DOM_collapse_similarity" for="dom_similarity_1">
Structurefunction studies on E. coli MetR protein , a putative prokaryotic leucine-zipper protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7730263" target="_blank">7730263</a>]<br></label>
<input id="dom_similarity_1", type="checkbox">
<div class="DOM_similar">
<li>Structurefunction studies on Escherichia coli MetR protein , a putative prokaryotic leucine-zipper protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24659766" target="_blank">24659766</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7868603" target="_blank">7868603</a>]<br></li></div>
<li>SpvR contains an amino-terminal helix-turn-helix motif that has significant homology with corresponding regions of bacterial transcriptional activators of the MetR / LysR-family ( 21 , 34 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1378053" target="_blank">1378053</a>]<br></li>
<li>MetR is a member of a large family of bacterial activator proteins , termed LysR-type transcriptional regulators , which share amino acid sequence homology and domain structure ( 8 , 20 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7896708" target="_blank">7896708</a>]<br></li>
<li>Some members of the LysWMetR family also contain a cationic C-terminus that may be involved in DNA-binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11370771" target="_blank">11370771</a>]<br></li>
<li>To determine which bases in the primary binding site are necessary for MetR recognition and binding , point mutations were constructed by site-directed mutagenesis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1904437" target="_blank">1904437</a>]<br></li>
<li>In contrast , the MetR protein tolerates substantial C-terminal deletions ( 38 ) , suggesting that the C terminus is critical for DNA-binding by only a subclass of LysR-family members .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7868603" target="_blank">7868603</a>]<br></li>
<li>In contrast , for dimeric MetR , the C terminus seems to be dispensable for DNA-binding and the protein binds to approximately 25 bp ( reference 24 and references therein ; 38 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7868603" target="_blank">7868603</a>]<br></li>
<li>These two classes of mutations in the DNA-binding site are grouped spatially , suggesting that the proposed MetR recognition sequence can be divided into two functional domains , one for binding and the other for activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1904437" target="_blank">1904437</a>]<br></li>
<li>The spatial grouping of the mutations suggests that there may be two functional domains to the MetR binding site in the metH control region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1904437" target="_blank">1904437</a>]<br></li>
<li>The binding domain , which consists of the outer ends of the palindrome ( defined by mutations -62 G , -61 A , - SiT , and -50 G ) is necessary for recognition and binding of the MetR protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1904437" target="_blank">1904437</a>]<br></li>
<li>The activation domain ( defined by mutations -53 T , -59 G , and -56 G ) may be necessary for proper orientation or conformation of the MetR protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1904437" target="_blank">1904437</a>]<br></li>
				</div></ul><br>
			<label class="collapse_EVO" for="_3"><font size="5px">MetR evolution</font></label>
				<input id="_3" type="checkbox">
				<div class="EVO"><br><ul><li>Sequence analysis revealed that spvR23 : : TnS interrupts an ORF for a 33K protein with identity to those reported by others ( 27 , 38 , 44 , 48 ) that is homologous to MetR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1657882" target="_blank">1657882</a>]<br></li>
<li>Some members of the LysWMetR family also contain a cationic C-terminus that may be involved in DNA binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11370771" target="_blank">11370771</a>]<br></li>
<li>The LysR/MetR family of gene regulators and intracellular replication genes ... .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15790293" target="_blank">15790293</a>]<br></li>
<li>SpvR contains an amino-terminal helix-turn-helix motif that has significant homology with corresponding regions of bacterial transcriptional activators of the MetR / LysR family ( 21 , 34 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1378053" target="_blank">1378053</a>]<br></li>
<li>The spvR gene product was found to have substantial amino-terminal homology with a large family of bacterial activator transcriptional proteins , including MetR , OxyR , LysR , CysB , and other proteins , at least some of which are involved in amino acid metabolic pathways ( 21 , 34 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1378053" target="_blank">1378053</a>]<br></li>
<li>MetR is a member of a large family of bacterial activator proteins , termed LysR-type transcriptional regulators , which share amino acid sequence homology and domain structure ( 8 , 20 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7896708" target="_blank">7896708</a>]<br></li>
<li>The NahR , TrpI , and CysB members of the LysR family have also been shown to be tetramers in solution ( 14 , 26 , 29 ) , while other LysR proteins , such as MetR , NodD3 , and Nac , appear to be dimers ( 17 , 19 , 25 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7868603" target="_blank">7868603</a>]<br></li>
<li>In contrast , the MetR protein tolerates substantial C-terminal deletions ( 38 ) , suggesting that the C-terminus is critical for DNA binding by only a subclass of LysR family members .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7868603" target="_blank">7868603</a>]<br></li>
<li>SpvR belongs to the LysR/MetR family of prokaryotic regulatory proteins , and SpvB is an actin-speciﬁc mono ( ADP-ribosyl ) - transferase ( Tezcan-Merdol et al. , 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15790293" target="_blank">15790293</a>]<br></li>
<li>are The gene MetR protein belongs to the lysR family of bacterial activator proteins ( 9 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8376333" target="_blank">8376333</a>]<br></li>
				<label class="EVO_collapse_similarity" for="evo_similarity_11">
The MetR protein belongs to the lysR family of DNA binding proteins ( 7 ) and binds to the metE metR control region ( 1 , 24 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1904437" target="_blank">1904437</a>]<br></label>
<input id="evo_similarity_11", type="checkbox">
<div class="EVO_similar">
<li>bound unbound DISCUSSION The MetR protein belongs to the lysR family of DNA binding proteins ( 7 ) , and DNase I protection experiments have shown that MetR binds to the metE metR intergenic promoter region ( 1 , 24 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1904437" target="_blank">1904437</a>]<br></li></div>
<li>While SpvR has been shown to belong to the LysR/MetR family of prokaryotic regulatory proteins ( Taira et al. , 1991a ) , the function ( s ) exerted by the rest of the spv gene products has not been resolved .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10972829" target="_blank">10972829</a>]<br></li>
<li>Analysis of the regulation of spv gene expression was initiated with the finding that the spvR-encoded product was homologous to MetR , a member of the LysR family of positive regulatory proteins ( 30 , 34 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7961444" target="_blank">7961444</a>]<br></li>
<li>The genetic that the metR gene product is a transcriptional activator protein ( 26 , 28 ) is supported by the finding that the MetR protein belongs to a family of DNA-binding proteins ( 9 ) , although none of these previous studies addressed the questions of the identification of a MetR operator sequence or a mechanism of activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2676984" target="_blank">2676984</a>]<br></li>
<li>Second , analysis of the proposed amino acid sequence of the MetR protein indicates that MetR belongs to a family of DNA-binding regulatory proteins ( 9 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2676984" target="_blank">2676984</a>]<br></li>
<li>The homology of SpvR to MetR of the LysR family of positive regulatory proteins suggested that spvR encodes a positive regulatory protein for other plasmid-encoded virulence factors .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1657882" target="_blank">1657882</a>]<br></li>
<li>These characteristics of MetR , including its functions as a positive regulatory protein and as an autorepressor , are also characteristics of other proteins of the same family of proteins as MetR , known as the LysR family of positive regulatory proteins ( 22 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1657882" target="_blank">1657882</a>]<br></li>
<li>spvR encodes a transcriptional activator of the LysR/MetR family of bacterial regulatory proteins and is transcribed separately in the same direction as the spvABCD genes [ 22 -- 25 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12732471" target="_blank">12732471</a>]<br></li>
<li>The LysR/MetR family of gene regulators and intracellular replication genes Like the activity of AraC/XylS regulatory proteins , the regulatory potential of LysR/MetR proteins is modulated through the encounter with small-molecu-lar-weight ligands ( Schell , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15790293" target="_blank">15790293</a>]<br></li>
<li>These include the so-called LysNMetR regulatory protein family , the AraC protein family , the iron response-regulator Fur family , and two-component regulatory systems .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11370771" target="_blank">11370771</a>]<br></li>
<li>Based on protein sequence identity , the SpvR protein associated with intracellular growth of Salmonella clearly belongs to the LysWMetR family ( 19-21 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11370771" target="_blank">11370771</a>]<br></li>
				</div></ul><br>
			<label class="collapse_RP" for="_4"><font size="5px">Processes regulated by MetR</font></label>
				<input id="_4" type="checkbox">
				<div class="RP"><br><ul><li>In Escherichia coli , there are at least four regulators that coordinate sulfur-utilization , namely , CysB , MetR , Cbl , and MetJ .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26386064" target="_blank">26386064</a>]<br></li>
<li>MetR not only positively regulates other genes but also functions as an autorepressor of its own transcription ( 33 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1657882" target="_blank">1657882</a>]<br></li>
<li>MetR is another LysR-type regulator in Salmonella which positively regulates metA , metE and metH .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19447191" target="_blank">19447191</a>]<br></li>
<li>The MetR gene product , a DNA binding protein ( 1 , 24 ) , activates metA ( unpublished data ) , metE , metH , and glyA expression ( 1 , 14 , 26 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1904437" target="_blank">1904437</a>]<br></li>
<li>Alteration in SPI-1 , flagellin and Vi-antigen expression Multiple , ( egpag , pig , SPV , SP12 genes ) Multiple , ( eg invF ; sipB ) Osmolarity RcsB/RcsC Alternative ( T factors RpoS Starvation Multiple , ( eg spv ) Many , induction of stress tolerance , inreased intracellular survival and growth Others H-NS Temperature , ( DNAtopology ) Temperature Iron , pH Many Global gene ( co ) regulator TlpA Fur Unknown flpA Multiple , ( eg sitABCD ) Many , involved in the induction of stress tolerance LysWMetR gene regulators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11370771" target="_blank">11370771</a>]<br></li>
<li>JOURNAL OF BACrERIOLOGY , Sept. 1993 , p. / 93/185862 -05 $ 02.00 / 0 Copyright C ) 1993 , American Society for Microbiology Regulation of the Salmonella typhimurium metF Gene by the MetR Protein JOHN M. COWAN , MARK L. URBANOWSKI , MICHAL TALMI , AND GEORGE V. STAUFFER * Department of Microbiology , The University of Iowa , Iowa City , Iowa 52242 Received 22 March 1993/Accepted 7 July 1993 The metF gene in Escherichia coli and Salmonella typhimurium is under negative transcriptional-control by the MetJ repressor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8376333" target="_blank">8376333</a>]<br></li>
<li>Since homocysteine acts as a coactivator for the MetR protein to activate metE expression ( 23 ) , we tested whether the MetR protein is involved with homocysteine in overriding the B12-mediated repression of the metE-lacZ fusion .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1385596" target="_blank">1385596</a>]<br></li>
<li>Like the CRP-cAMP response , the activity of these regulatory proteins is affected by the presence of a coregulatory molecule , and many LysWMetR members are involved in the feed-back regulation of metabolic pathways .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11370771" target="_blank">11370771</a>]<br></li>
<li>Heni-koff et al. ( 29 ) have recently recognized that these proteins and others [ E. coli IlvY , regulator of ilvC and other genes in the isoleucine-valine biosynthetic pathway ( 30 ) ; E. coli and S. typhimurium CysB , positive regulator of the cysteine regulon ( 31 ) ; S. typhimurium MetR , regulator of the metE and metH genes ( 32 ) ; and the Enterobacter cloacae ampR regulator of the ampC gene , which confers resistance to cepha-losporin ( 33 ) ] comprise a family of bacterial regulatory OxyR - A HT .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2471187" target="_blank">2471187</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_10">
Maxon , M.E. , Redﬁeld , B. , Cai , X.Y. , Shoeman , R. , Fujita , K. , Fisher , W. , et al. ( 1989 ) Regulation of methionine-synthesis in Escherichia coli : effect of the MetR protein on the expression of the metE and metR genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22804842" target="_blank">22804842</a>]<br></label>
<input id="rp_similarity_10", type="checkbox">
<div class="RP_similar">
<li>Regulation of methionine-biosynthesis in Escherichia coli : effect of the MetR protein on the expression of the metE and metR genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1706701" target="_blank">1706701</a>]<br></li>
<li>Regulation of methionine-synthesis in Escherichia coli : effect of the MetR protein on the expression of the metE and metR genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1657882" target="_blank">1657882</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1385596" target="_blank">1385596</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7896708" target="_blank">7896708</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1904437" target="_blank">1904437</a>]<br></li></div>
<li>The spvR gene product was found to have substantial amino-terminal homology with a large family of bacterial activator transcriptional proteins , including MetR , OxyR , LysR , CysB , and other proteins , at least some of which are involved in amino-acid-metabolic pathways ( 21 , 34 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1378053" target="_blank">1378053</a>]<br></li>
<li>The LysR/MetR family of gene regulators and intracellular replication genes ... .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15790293" target="_blank">15790293</a>]<br></li>
				</div></ul><br>
			<label class="collapse_SIT" for="_5"><font size="5px">MetR binding site</font></label>
				<input id="_5" type="checkbox">
				<div class="SIT"><br><ul><li>Amino acid sequence alignment of SpvR with S. typhi-murium MetR ( 42 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1657882" target="_blank">1657882</a>]<br></li>
<li>When MetR binds at this site , transcription of metE increases while transcription of metR decreases ( 6 , 51 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1657882" target="_blank">1657882</a>]<br></li>
<li>Elements of dyad symmetry have been noted in the binding sites for a number of LysR-type proteins , including AmpR ( 26 ) , IlvY ( 49 ) , MetR ( 4 , 47 ) , NahR ( 19 ) , OccR ( 8 ) , and TrpI ( 6 ) , while plausible arrangements of both inverted and direct repeats have been noted for NodD ( 10 , 13 , 48 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7730263" target="_blank">7730263</a>]<br></li>
<li>At a higher concentration ( 58 nM dimer ) , MetR protected an additional region from about positions 224 to 247 upstream of 11 ( site 2 in Fig. 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7896708" target="_blank">7896708</a>]<br></li>
<li>The promoter regions of metR and metE are nearly superimposed , and MetR binds at a single location that includes the regulatory site for both metR and metE ( 6 , 33 , 42 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1657882" target="_blank">1657882</a>]<br></li>
<li>By using a DNase I footprinting assay with purified MetR protein , a 26-bp segment beginning 9 bp upstream from the -35 promoter sequence that is protected from DNase I attack was identified ( Fig. 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1729233" target="_blank">1729233</a>]<br></li>
<li>Site 1 is the region previously shown to be protected from DNase I digestion by bound MetR ( 27 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7896708" target="_blank">7896708</a>]<br></li>
<li>Brackets above the MetR-binding sites indicate sequences protected by MetR from DNase I digestion ( see Fig. 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8376333" target="_blank">8376333</a>]<br></li>
				<label class="SIT_collapse_similarity" for="sit_similarity_9">
The MetR binding site in the Salmonella typhimuium metH gene DNA sequence constraints on activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8376333" target="_blank">8376333</a>]<br></label>
<input id="sit_similarity_9", type="checkbox">
<div class="SIT_similar">
<li>The MetR binding site in the Salmonella typhimurium metH gene : DNA sequence constraints on activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1729233" target="_blank">1729233</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7730263" target="_blank">7730263</a>]<br></li></div>
<li>DNAs isolated from plasmids pGS201-53T , pGS201-59G , and pGS201-56G all showed nearly wild-type levels of binding , suggesting that these mutations do not affect the ability of the sequences to bind MetR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1904437" target="_blank">1904437</a>]<br></li>
<li>The sequence protected by the bound MetR protein is indicated by the bracket in Fig. 2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1729233" target="_blank">1729233</a>]<br></li>
<li>The interrupted palindrome 5 ' - TGAA ----- TTCA-3 ' that may constitute the MetR recognition sequence is lined between the top and bottom sequences .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2676984" target="_blank">2676984</a>]<br></li>
<li>The interrupted palindrome 5 ' - TGAANNTNNCTCA-3 ' ( N = any nucleotide ) that may constitute the MetR recognition sequence is indicated by asterisks .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1729233" target="_blank">1729233</a>]<br></li>
<li>Lysogen 760lElaczRgal-a2 , where site 2 was changed toward the MetR consensus sequence , showed increased b-galactosidase levels in D-methio - TABLE 1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7896708" target="_blank">7896708</a>]<br></li>
<li>Multiple changes were designated as follows : a 7-bp change ( -55 C-56C-57T-58T-64A-65G-66G ) in site 1 away from the MetR consensus binding sequence was designated as allele 1 ( a1 ) , a 2-bp change ( -34 T-36A ) in site 2 toward the MetR consensus sequence was designated as allele 2 ( a2 ) , and a 6-bp change ( -35 C-37G-43G-44A-45T-46C ) in site 2 away from the MetR consensus sequence was designated as allele 3 ( a3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7896708" target="_blank">7896708</a>]<br></li>
<li>Base pair changes in site 1 or site 2 away from the MetR consensus binding sequence resulted in decreased metE-lacZ expression , suggesting that both sites are necessary for expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7896708" target="_blank">7896708</a>]<br></li>
<li>A 2-bp change in site 2 toward the MetR consensus binding sequence resulted in high metE-lacZ expression ; the increased expression was MetR dependent but homocysteine independent .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7896708" target="_blank">7896708</a>]<br></li>
<li>Representative bands in the regions corresponding to MetR-binding sites 1 and 2 were compared with samples with no MetR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7896708" target="_blank">7896708</a>]<br></li>
<li>Included in this protected region is an 8-bp interrupted palindrome , 5 ' - TGAANNNNNTTCA-3 ' , proposed to be the MetR recognition sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1904437" target="_blank">1904437</a>]<br></li>
<li>This sequence shows a match in seven of eight base pairs with an interrupted 8-bp palindrome in the MetR protected region in the metE metR intergenic control region ( 5 ' - TGAANNNNNTTCA-3 ' ) that was shown to be important for appropriate metE and metR expression ( 24 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1904437" target="_blank">1904437</a>]<br></li>
<li>Gel mobility shift assays and DNase I protection experiments showed that the MetR protein binds to a DNA fragment carrying the metA control region and protects a 26-bp region beginning 9 bp upstream of the -35 promoter sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1729233" target="_blank">1729233</a>]<br></li>
<li>In vitro gel mobility shift assays and DNase I protection assays established that the MetR protein binds to and protects a 24-bp sequence in the metH promoter region from DNase I attack .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1904437" target="_blank">1904437</a>]<br></li>
<li>It was shown previously that MetR binds to and protects from DNase I digestion a 24-bp sequence in the metE metR regulatory region from nucleotides 248 to 271 relative to the metE transcription initiation site ( designated as site 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7896708" target="_blank">7896708</a>]<br></li>
<li>In this study , we show that puriﬁed MetR protein also binds to and protects a second 24-bp sequence adjacent to the original site , from nucleotides 224 to 247 relative to the metE transcription initiation site ( designated as site 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7896708" target="_blank">7896708</a>]<br></li>
<li>, ug/ml ) , the MetR protein protected a region from approximately position -68 to -95 upstream of the transcription initiation site ( site 1 in Fig. 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8376333" target="_blank">8376333</a>]<br></li>
				<label class="SIT_collapse_similarity" for="sit_similarity_27">
A DNA footprint analysis showed that MetR binds to a DNA fragment carrying the metF promoter and protects two separate regions from DNase I digestion : a 46-bp region from position -50 to -95 upstream of the transcription initiation site and a 24-bp region from about position +62 to +85 downstream of the transcription initiation site and within the metF structural gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8376333" target="_blank">8376333</a>]<br></label>
<input id="sit_similarity_27", type="checkbox">
<div class="SIT_similar">
<li>In vitro binding assays and DNase I protection experiments verified that MetR binds to metF promoter DNA and protects two separate regions from DNase I attack ( Fig. 1 ) : ( i ) a ca. 46-bp region from position -50 to -95 upstream from the transcription initiation site and encompassing two of the predicted MetR-binding sites ( sites 1 and 2 ) in the metF promoter region and ( ii ) a ca. 24-bp region from approximately position +62 to +85 downstream from the transcriptional initiation site and encompassing the third predicted MetR-binding site located in the metF structural gene ( site 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8376333" target="_blank">8376333</a>]<br></li></div>
<li>DNase I footprint analysis showed that MetR bound at the high-afﬁnity site 1 enhances MetR binding at the low-afﬁnity site 2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7896708" target="_blank">7896708</a>]<br></li>
<li>This palindromic sequence is highly conserved in the ( unpublished data ) and metH metA promoter regions ( 23 ) and has been proposed as the MetR recognition sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1904437" target="_blank">1904437</a>]<br></li>
<li>Single-base-pair changes were introduced into the proposed MetR recognition sequence within the promoter region of a metH-lacZ gene fusion by oligonucleotide-directed mutagenesis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1904437" target="_blank">1904437</a>]<br></li>
<li>The blocking of RNAP movement should be similar , whether transcription initiates from a wild-type metR-galK promoter or from a mutant metR-galK promoter , where the promoter but consensus sequence , the template the binding site , MetR not sequence or has been altered .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2676984" target="_blank">2676984</a>]<br></li>
<li>Because the S. typhimurium metH control region has been cloned and sequenced ( 23 ) , we investigated MetR binding in the metH control region through DNase I protection assays , gel mobility shift assays , and mutational analysis of the proposed MetR binding site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1904437" target="_blank">1904437</a>]<br></li>
<li>These mutations are located within a region that binds the MetR protein , and we propose that these mutations define the MetR binding site for the metE metR control region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2676984" target="_blank">2676984</a>]<br></li>
<li>Mutations that changed each of the three MetR-binding sites individually away from the MetR consensus binding sequence were introduced into the metF control region ( see Materials and Methods and Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8376333" target="_blank">8376333</a>]<br></li>
<li>The mutations proposed to disrupt the MetR binding site ( alleles 1 , 2 , 7 , 8 , and 9 ) are indicated above the sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2676984" target="_blank">2676984</a>]<br></li>
<li>This protected region includes the sequence 5 ' - TGAANNNNNCTCA-3 ' , previously proposed to be a possible MetR recognition sequence for metH on the basis of its homology to the MetR binding site for metE , 5 ' - TGAANNNNNTTCA-3 ' ( 24 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1904437" target="_blank">1904437</a>]<br></li>
<li>The secondary MetR binding site is approximatel 22 to 26 bp long , covering bases 117 to 143 in the metH structural gene ( Fig. 3B ) , and includes the sequence 5 ' - TG AANNNNNTTTC-3 ' , matching six of eight base pairs with the proposed MetR recognition sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1904437" target="_blank">1904437</a>]<br></li>
<li>To locate the specific region on the NheI-BamHI DNA probe fragment to which the MetR protein binds , we used a DNase I protection assay in which the bound MetR protein protects the region of DNA encompassing its binding site from nuclease DNase I attack .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2676984" target="_blank">2676984</a>]<br></li>
<li>The 3 ' end of the proposed MetR binding sequence 5 ' - TGAA ----- TTCA-3 ' is separated from the -35 region of the 22 bp ( Fig. 2 ) ; the analogous distance for metE promoter by the metH gene is 17 bp ( 26 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2676984" target="_blank">2676984</a>]<br></li>
<li>The center of the palindrome for the MetR binding site is located 23 bp upstream from the -35 promoter region of the metA gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1729233" target="_blank">1729233</a>]<br></li>
<li>For these experiments , we assumed that the consensus recognition sequence for MetR binding is 5 ' - TGAANNNNNTTCA-3 ' , the 8-bp interrupted palindrome found within the MetR footprint in the metE promoter region ( 24 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1904437" target="_blank">1904437</a>]<br></li>
<li>Mutation -53 T changed the sequence towards consensus , creating a perfect match with the MetR binding site found in the metE promoter region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1904437" target="_blank">1904437</a>]<br></li>
<li>Isolation of point mutations in the MetR recognition sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1904437" target="_blank">1904437</a>]<br></li>
<li>To determine whether site 2 is functionally important in MetRmediated activation of the metE gene , mutations were constructed in site 2 either toward or away from the MetR consensus sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7896708" target="_blank">7896708</a>]<br></li>
<li>Since site 2 mutations away from the consensus sequence still allow high levels of b-galactosidase in the presence of homocysteine , it is possible that the base pair changes do not eliminate binding of MetR at site 2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7896708" target="_blank">7896708</a>]<br></li>
				</ul></div><br>
			<label class="collapse_TU" for="_6"><font size="5px">Transcription units regulated by MetR</font></label>
				<input id="_6" type="checkbox">
				<div class="TU"><br><ul><li>In E. coli and Salmonella , five transcription factors have been implicated : hmp transcription is activated by MetR ( Membrillo-Herna ́ndez et al. , 1998 ) and RamA ( Hernández - Urzúa et al. , 2007 ) and is repressed by NsrR ( Bodenmiller & Spiro , 2006 ) , FNR ( Cruz-Ramos et al. , 2002 ; Poole et al. , 1996 ) and Fur ( D'Autreaux et al. , 2002 ; Hernández - Urzúa et al. , 2007 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20829289" target="_blank">20829289</a>]<br></li>
<li>Comparison of activation of gene expression and MetR Lysogena l3-Galactosidaseb MetR bindingc 719X201 1.00 1.00 719X201-62G 0.26 ± 0.01 0.24 ± 0.01 719A201-61A 0.07 ± 0.01 0.08 ± 0.02 719X201-59G 0.50 ± 0.01 1.07 ± 0.06 719X201-56G 0.22 ± 0.01 0.83 ± 0.06 719A201-53T 0.69 ± 0.02 1.08 ± 0.15 719X201-51T 0.13 ± 0.01 0.14 ± 0.01 719A201-50G 0.10 ± 0.01 0.09 ± 0.00 a Lysogens of strain GS719 ( metJ metB ) were grown in GM supplemented with D-met .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1904437" target="_blank">1904437</a>]<br></li>
<li>MetR not only positively regulates other genes but also functions as an autorepressor of its own transcription ( 33 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1657882" target="_blank">1657882</a>]<br></li>
<li>It is not known how bound MetR protein activates transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1904437" target="_blank">1904437</a>]<br></li>
<li>The MetR protein has been shown to activate metE gene expression by binding to metE promoter region DNA ( 4 , 28 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1729233" target="_blank">1729233</a>]<br></li>
<li>MetR is a DNA binding protein that binds the promoters of metE and metH to positively regulate transcription ( 33 ) of parts of the methionine operon through modulation by homocystine ( 5 , 7 , 52 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1657882" target="_blank">1657882</a>]<br></li>
<li>MetR also binds to the promoter of metH located 3 ' to the metE gene ( 6 ) and glyA ( 43 ) to regulate transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1657882" target="_blank">1657882</a>]<br></li>
<li>spvR encodes a transcriptional activator of the LysR/MetR family of bacterial regulatory proteins and is transcribed separately in the same direction as the spvABCD genes [ 22 -- 25 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12732471" target="_blank">12732471</a>]<br></li>
<li>Expression of the metE gene requires a functional MetR protein ( 30 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1729233" target="_blank">1729233</a>]<br></li>
<li>This 26-bp region was compared with the DNA sequences of the control regions of the typhimurium ( 28 ) and ( 27 ) which S. metE metH genes , are also activated by MetR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1729233" target="_blank">1729233</a>]<br></li>
<li>These results suggest that homocysteine and the MetR protein control metA-lacZ gene expression at the level of transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1729233" target="_blank">1729233</a>]<br></li>
<li>These results suggest that the MetR protein is a positive activator of the metA gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1729233" target="_blank">1729233</a>]<br></li>
<li>The activation of the metE gene and the repression of the metR gene by the MetR protein is modulated by homocysteine ( 25 , 27 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2676984" target="_blank">2676984</a>]<br></li>
<li>Regulation of the Salmonella typhimurium metA Gene by the MetR Protein and Homocysteine The DNA sequence of the Salmonella typhimurium metA control region is presented .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1729233" target="_blank">1729233</a>]<br></li>
				<label class="TU_collapse_similarity" for="tu_similarity_15">
Regulation of the Salmonella typhimunum metA gene by the MetR protein and homocysteine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8376333" target="_blank">8376333</a>]<br></label>
<input id="tu_similarity_15", type="checkbox">
<div class="TU_similar">
<li>Regulation of the Salmonella typhimurium metA gene by the MetR protein and homocysteine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7896708" target="_blank">7896708</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7868603" target="_blank">7868603</a>]<br></li></div>
<li>Although the MetR protein appears to serve as a positive regulator for the metA gene , its absence does not completely eliminate metA-lacZ expression ( Table 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1729233" target="_blank">1729233</a>]<br></li>
<li>In the metJ metR lysogen 748AAlac and the metJ metB metR lysogen 761AAlac , where the MetR protein is inactivated , metA expression is decreased 2.9 - to 3.1-fold ( Table Since metR 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1729233" target="_blank">1729233</a>]<br></li>
<li>As shown in Table 4 , , - galactosidase levels in lysogen 761XAlac ( pGS69 ) were not changed significantly from those in the untransformed cells , suggesting that the MetR protein is required for full metA-lacZ gene expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1729233" target="_blank">1729233</a>]<br></li>
<li>The mutant lysogen had b-galactosidase levels similar to those in the parental lysogen , suggesting that the residual metE-lacZ expression in the a1 mutant is MetR dependent and that the a1 mutation does not directly affect the metE promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=7896708" target="_blank">7896708</a>]<br></li>
				</div></ul></div><br>
	</div>
</body>
</html>