<!doctype html>
<style>
@charset "utf-8";
body {
	height: 100%;
	margin: 0;
	padding: 0;
}

.banner {
	position: fixed;
	top: 0;
	width: 100%;
	height: 65px;
	background: gray;
	line-height: 65px;
}
.spacer {
	height: 10px;
}

.main-content {
	padding: 5%;
}

.collapse_ACT {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-style: none solid none none;
	border-color: #B4045F;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_ACT + input{
	display: none;
}
.collapse_ACT + input + div{
	display: none;
}
.collapse_ACT + input:checked + div{
	display: block;
}
.ACT {
	text-align: justify;
	text-justify: inter-word;
}
.ACT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B4045F;
}
.ACT_collapse_similarity:hover{
	background: #e5d3dc;
}
.ACT_collapse_similarity + input {
	display: none;
}
.ACT_collapse_similarity + input + div{
	display: none;
}

.ACT_collapse_similarity + input:checked + div{
	display: block;
}
.ACT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.ACT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B4045F;
	padding-left: 5px;
}
.ACT_collapse_similarity:hover{
	background: #e5d3dc;
}
.ACT_collapse_similarity + input {
	display: none;
}
.ACT_collapse_similarity + input + div{
	display: none;
}
.ACT_collapse_similarity + input:checked + div{
	display: block;
}
.ACT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.ACT_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #B4045F;
}
.ACT_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_DOM {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-style: none solid none none;
	border-color: #B404AE;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_DOM + input{
	display: none;
}
.collapse_DOM + input + div{
	display: none;
}
.collapse_DOM + input:checked + div{
	display: block;
}
.DOM {
	text-align: justify;
	text-justify: inter-word;
}
.DOM_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B404AE;
	padding-left: 5px;
}
.DOM_collapse_similarity:hover{
	background: #e8d7e7;
}
.DOM_collapse_similarity + input {
	display: none;
}
.DOM_collapse_similarity + input + div{
	display: none;
}
.DOM_collapse_similarity + input:checked + div{
	display: block;
}
.DOM_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}.DOM_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #B404AE;
}
.DOM_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_EVO {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #7401DF;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_EVO + input{
	display: none;
}
.collapse_EVO + input + div{
	display: none;
}
.collapse_EVO + input:checked + div{
	display: block;
}
.EVO {
	text-align: justify;
	text-justify: inter-word;
}
.EVO_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #7401DF;
	padding-left: 5px;
}
.EVO_collapse_similarity:hover{
	background: #e2d7ed;
}
.EVO_collapse_similarity + input {
	display: none;
}
.EVO_collapse_similarity + input + div{
	display: none;
}
.EVO_collapse_similarity + input:checked + div{
	display: block;
}
.EVO_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.EVO_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #7401DF;
}
.EVO_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_RP {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #0174DF;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_RP + input{
	display: none;
}
.collapse_RP + input + div{
	display: none;
}
.collapse_RP + input:checked + div{
	display: block;
}
.RP {
	text-align: justify;
	text-justify: inter-word;
}
.RP_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #0174DF;
	padding-left: 5px;
}
.RP_collapse_similarity:hover{
	background: #c9d7e2;
}
.RP_collapse_similarity + input {
	display: none;
}
.RP_collapse_similarity + input + div{
	display: none;
}
.RP_collapse_similarity + input:checked + div{
	display: block;
}
.RP_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.RP_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #0174DF;
}
.RP_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_SIT {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #0489B1;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_SIT + input{
	display: none;
}
.collapse_SIT + input + div{
	display: none;
}
.collapse_SIT + input:checked + div{
	display: block;
}
.SIT {
	text-align: justify;
	text-justify: inter-word;
}
.SIT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #0489B1;
	padding-left: 5px;
}
.SIT_collapse_similarity:hover{
	background: #d0dfe2;
}
.SIT_collapse_similarity + input {
	display: none;
}
.SIT_collapse_similarity + input + div{
	display: none;
}
.SIT_collapse_similarity + input:checked + div{
	display: block;
}
.SIT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.SIT_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #0489B1;
}
.SIT_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_TU {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #04B486;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_TU + input{
	display: none;
}
.collapse_TU + input + div{
	display: none;
}
.collapse_TU + input:checked + div{
	display: block;
}
.TU {
	text-align: justify;
	text-justify: inter-word;
}
.TU_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #04B486;
	padding-left: 5px;
}
.TU_collapse_similarity:hover{
	background: #d0e2de;
}
.TU_collapse_similarity + input {
	display: none;
}
.TU_collapse_similarity + input + div{
	display: none;
}
.TU_collapse_similarity + input:checked + div{
	display: block;
}
.TU_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.TU_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #04B486;
}
.TU_similar > a {
	text-decoration: none;
	color: gray;
}

ul { 
	list-style-type: none;
	padding: 0;
	margin: 0;
}
ul li > a {
	text-decoration: none;
	color: gray;
}
ul li:hover, ul li:focus {
	background: #F0F0F0;
}

ul li:first-child span {
  border-left:none;  
}

</style>
<html>
<head>
<meta charset="utf-8">
<title>TF properties - MlrA</title>
<meta name="theme-color" content="#ffffff">
</head>
<body>
	<div class="banner">
		<center><font color="white" size="16px" face="arial">MlrA</font></center>
	</div>
	<div align="center" class="main-content">
		<div class="spacer">
	</div><br>			<label class="collapse_ACT" for="_1"><font size="5px">MlrA transcriptional activity</font></label>
				<input id="_1" type="checkbox">
				<div class="ACT"><br><ul><li>Several regulatory proteins , including OmpR , IHF , H-NS , and MlrA , bind directly to a large intergenic region upstream of the csgD promoter to modulate gene expression in response to changing environmental stimuli ( Gerstel and Romling , 2003 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21570144" target="_blank">21570144</a>]<br></li>
<li>Numerous transcription factors are involved , including OmpR , MlrA , and CpxR and the nonspeciﬁc DNA-binding proteins H-NS and IHF , in addition to the feed-forward loop involving rpoS and the factors that regulate RpoS activity , including IraP and Crl .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25824832" target="_blank">25824832</a>]<br></li>
<li>MlrA represents a newly deﬁned transcriptional-regulator of csgD and agfD that is required for curli production and extracellular matrix formation by wild-type E. coli and S. typhimurium , and it is positively regulated by RpoS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11489123" target="_blank">11489123</a>]<br></li>
<li>Bars 1/4 0.5 mm MlrA expression is positively regulated by RpoS RpoS and MlrA are required for Csg and AgF synthesis and aggregation by wild-type E. coli and S. typhimurium , respectively , but neither of these regulators is necessary for induction of these phenotypes in the constitutive AgFexpressing S. typhimurium x4666 , which contains an altered agfD promoter region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11489123" target="_blank">11489123</a>]<br></li>
<li>agfD is also regulated by the transcriptional-regulators OmpR , MlrA and CpxR-P ( Gerstel and Römling , 2003 ; White and Surette , 2006 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18256702" target="_blank">18256702</a>]<br></li>
<li>Activation of the agfD promoter is regulated by RpoS , OmpR , MlrA , CpxR , H-NS , and IHF ( 23 , 32 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17079680" target="_blank">17079680</a>]<br></li>
<li>As stated before , MlrA is known to control the transcription of the response-regulator CsgD , which in turn regulates the biosynthesis of thin aggregative ﬁmbriae and the transcription of adrA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15458421" target="_blank">15458421</a>]<br></li>
<li>While csgD is positively regulated by RpoS via MlrA , it itself stimulates the production of curli ﬁmbriae through the transcriptional activation of the csgBAC operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16313619" target="_blank">16313619</a>]<br></li>
<li>Adding to complexity , the alternative sigma factor RpoS not only activates and is activated by small RNAs ( see above ) , but also activates the csgD transcriptional-regulator MlrA , while RpoS degradation is indirectly stimulated by another csgD transcriptional-regulator , RtsA [ 86,87,123 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25437188" target="_blank">25437188</a>]<br></li>
<li>In addition , a recently deﬁned transcriptional-regulator , MlrA , controls csgD expression and is positively regulated by RpoS ( Brown et al. , 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15458421" target="_blank">15458421</a>]<br></li>
<li>Moreover , transcriptional expression of csgD , which is tightly regulated in response to different environmental stimuli , is activated by the regulator MlrA ( Brown et al. , 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29163440" target="_blank">29163440</a>]<br></li>
				</div></ul><br>
			<label class="collapse_DOM" for="_2"><font size="5px">MlrA protein domains</font></label>
				<input id="_2" type="checkbox">
				<div class="DOM"><br><ul><li>This suggests that MlrA may interact with an inducer that is not speciﬁc to other known MerR-family members .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11489123" target="_blank">11489123</a>]<br></li>
<li>Our results demonstrated that MlrA , and thus AdrA , is required for cellulose production and bioﬁlm formation in LB complex medium whereas STM1987 ( GGDEF domain containing protein A , gcpA ) is critical for bioﬁlm formation in the nutrient-deﬁcient medium , ATM .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15458421" target="_blank">15458421</a>]<br></li>
<li>In the N-terminal 70 amino acids of MlrA , 17 residues are identical or similar to the 22 conserved residues identiﬁed in different MerR proteins from diverse sources ( Summers , 1992 ) , and many of these residues are also similar in other MerR-related proteins ( Fig. 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11489123" target="_blank">11489123</a>]<br></li>
<li>The 22 conserved residues present in most known MerR proteins ( Summers , 1992 ) are indicated below the alignment , and those that are identical or similar to MlrA are underlined .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11489123" target="_blank">11489123</a>]<br></li>
<li>Comparison of the amino-terminal domains of E. coli MlrA and the most similar proteins in the MerR regulator family .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11489123" target="_blank">11489123</a>]<br></li>
<li>Currently , we are investigating regulation of mlrA in response to a variety of environmental stresses encountered during entry into stationary phase and the natural course of the disease process , what type of ligand may interact with MlrA and whether the cysteine residues play a role in this interaction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11489123" target="_blank">11489123</a>]<br></li>
<li>The N-terminal portion of MlrA contains a putative helix -- turn -- helix domain , residues 6 -- 25 ( Fig. 2 ) , that is associated with DNA-binding by MerR regulators ( Summers , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11489123" target="_blank">11489123</a>]<br></li>
<li>The C-terminal portion of MlrA from E. coli and S. typhimurium contains three conserved cysteine residues .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11489123" target="_blank">11489123</a>]<br></li>
				</div></ul><br>
			<label class="collapse_EVO" for="_3"><font size="5px">MlrA evolution</font></label>
				<input id="_3" type="checkbox">
				<div class="EVO"><br><ul><li>This suggests that MlrA may interact with an inducer that is not speciﬁc to other known MerR family members .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11489123" target="_blank">11489123</a>]<br></li>
<li>Hence , this suggests the possibility that other regulators belonging to the MerR family , such as MlrA , may also interact directly with RNA polymerase in the preinitiation complex to control gene transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11489123" target="_blank">11489123</a>]<br></li>
<li>However , the C-terminal of MlrA only shares signiﬁcant homology to the putative regulators of unknown function , YcgE , ORF242 , and VCA0056 from E. coli K-12 , Salmonella enterica and Vibrio cholerae , respectively , which exhibit an overall similarity to MlrA of 76 % , 70 % and 50 % respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11489123" target="_blank">11489123</a>]<br></li>
<li>Comparison of the amino-terminal domains of E. coli MlrA and the most similar proteins in the MerR regulator family .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11489123" target="_blank">11489123</a>]<br></li>
<li>MlrA exhibits similarity in the N-terminal portion of the molecule with the MerR family of regulatory proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11489123" target="_blank">11489123</a>]<br></li>
				</div></ul><br>
			<label class="collapse_RP" for="_4"><font size="5px">Processes regulated by MlrA</font></label>
				<input id="_4" type="checkbox">
				<div class="RP"><br><ul><li>MltC/E 2672 BarA 1798 RcsD RcsC EnvZ CpxA Motility FliG/M MlrA OmpR CpxR RcsB IHF Crl SirA RpoS FliA RpoD H-NS CsrB/C RprA 3375 csgDEFG csgBAC Curli FlhDC ArcZ expression CsrA Hfq 1697 CsgD P SdsR bcsABZC bcsEFG 1344 MicA BcsE Cellulose synthesis 3611 4451 P CsgD BcsA 1703 adrA yihvW yihU -- yshA 1827 AdrA 1987 bapABCD YihW 2123 4264 3388 DsbA/B O-antigen capsule biosynthesis BapA secretion Figure 2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25437188" target="_blank">25437188</a>]<br></li>
<li>In addition , OmpR , CpxR , H-NS , IHF , MlrA , and RpoS are also involved in biofilm-formation under specific environmental conditions ( 3,9 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22545539" target="_blank">22545539</a>]<br></li>
<li>In addition , a recently deﬁned transcriptional regulator , MlrA , controls csgD expression and is positively regulated by RpoS ( Brown et al. , 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15458421" target="_blank">15458421</a>]<br></li>
<li>Moreover , transcriptional expression of csgD , which is tightly regulated in response to different environmental stimuli , is activated by the regulator MlrA ( Brown et al. , 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29163440" target="_blank">29163440</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_5">
Ogasawara H , Yamamoto K , Ishihama A. Regulatory role of MlrA in transcription activation of csgD , the master regulator of biofilm-formation in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26293911" target="_blank">26293911</a>]<br></label>
<input id="rp_similarity_5", type="checkbox">
<div class="RP_similar">
<li>Ogasawara , H. ; Yamamoto , K. ; Ishihama , A. Regulatory role of MlrA in transcription activation of csgD , the master regulator of biofilm-formation in Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23443158" target="_blank">23443158</a>]<br></li></div>
<li>MlrA transcription is defective in strain SL1344 The mlrA gene has been identiﬁed as a positive regulator of the csgD gene , aggregative morphology and extracellular matrix formation in E. coli and S. typhimurium ( Brown et al. , 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15458421" target="_blank">15458421</a>]<br></li>
<li>STM1987 and MlrA activate cellulose production and bioﬁlm formation in S. typhimurium To identify the genes whose expression bestows SL1344 strain with the capacity for cellulose-biosynthesis and bio-ﬁlm formation , the ﬂanking regions of the inserts carried by plasmids extracted from ﬂuorescent complemented strains were sequenced .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15458421" target="_blank">15458421</a>]<br></li>
<li>As stated before , MlrA is known to control the transcription of the response regulator CsgD , which in turn regulates the biosynthesis of thin aggregative ﬁmbriae and the transcription of adrA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15458421" target="_blank">15458421</a>]<br></li>
<li>Under the inﬂuence of unknown environmental signals , MlrA induces the csgD expression , which is the master regulator of the Salmonella ( and E. coli ) bioﬁlm formation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24735176" target="_blank">24735176</a>]<br></li>
<li>We have identiﬁed a novel regulator , termed MlrA , that is Introduction Bacterial adherence to host tissues and cells is often mediated by non-ﬂagellar ﬁlamentous cell surface appen-dages , known as ﬁmbriae ( or pili ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11489123" target="_blank">11489123</a>]<br></li>
<li>Numerous transcription factors are involved , including OmpR , MlrA , and CpxR and the nonspeciﬁc DNA-binding proteins H-NS and IHF , in addition to the feed-forward loop involving rpoS and the factors that regulate RpoS activity , including IraP and Crl .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25824832" target="_blank">25824832</a>]<br></li>
<li>Most regulators can be classified into following groups : response-to-stress ( BolA , CsiR , GadW , GadX , GadE , OxyR , RpoH , RpoS and YdeO ) , adhesion and biofilm ( BluR , CsgD , FlgM , H-NS , McbR , MlrA , RcdA and RcsB ) , quorum-sensing ( McbR , MqsR and LsrR ) , DNA-repair ( Ada and AidB ) and metabolism ( AppY , FadR and Nac ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26387845" target="_blank">26387845</a>]<br></li>
<li>In this report , we determine that curli mediate MRHA by strain x7122 , and identify that a novel regulator , MlrA , or RpoS ( s ) encoded by the x7122 DNA-containing cosmids s confers curli expression to E. coli HB101 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11489123" target="_blank">11489123</a>]<br></li>
<li>These results indicate that MlrA is a newly deﬁned transcriptional regulator of csgD/agfD that acts as Summary Production of curli ( AgF ) adhesins by Escherichia coli and Salmonella enterica serovar Typhimurium ( S. typhimurium ) is associated with extracellular matrix production and is optimal at low temperature a positive regulator of RpoS-dependent curli and extracellular matrix production by E. coli and S. typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11489123" target="_blank">11489123</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_16">
MlrA , a novel regulator of curli ( AgF ) and extracellular matrix synthesis by Escherichia coli and Salmonella enterica serovar Typhi-murium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17012381" target="_blank">17012381</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19897646" target="_blank">19897646</a>]<br></label>
<input id="rp_similarity_16", type="checkbox">
<div class="RP_similar">
<li>MlrA , a novel regulator of curli ( AgF ) and extracellular matrix synthesis by Escherichia coli and Salmonella enterica Serovar Typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18256702" target="_blank">18256702</a>]<br></li>
<li>MlrA , a novel regulator of curli ( AgF ) and extracellular matrix synthesis by Escherichia coli and Salmonella enterica serovar Typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23376784" target="_blank">23376784</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23443158" target="_blank">23443158</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29163440" target="_blank">29163440</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22545539" target="_blank">22545539</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26293911" target="_blank">26293911</a>]<br></li>
<li>Brown , P.K. , Dozois , C.M. , Nickerson , C.A. , Zuppardo , A. , Terlonge , J. , and Curtiss , R. , 3rd ( 2001 ) MlrA , a novel regulator of curli ( AgF ) and extracellular matrix synthesis by Escherichia coli and Salmonella enterica serovar Typh-imurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15458421" target="_blank">15458421</a>]<br></li>
<li>[ 11 ] Brown PK , Dozois CM , Nickerson CA , Zuppardo A , Terlonge J , Curtiss 3rd R. MlrA , a novel regulator of curli ( AgF ) and extracellular matrix synthesis by Escherichia coli and Salmonella enterica serovar Typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21147214" target="_blank">21147214</a>]<br></li>
<li>Brown , P. K. , Dozois , C. M. , Nickerson , C. A. , Zuppardo , A. , Terlonge , J. , Curtiss , R. , 3rd : MlrA , a novel regulator of curli ( AgF ) and extracellular matrix synthesis by Escherichia coli and Salmonella enterica serovar Typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14503792" target="_blank">14503792</a>]<br></li>
<li>[ 7 ] P.K. Brown , C.M. Dozois , C.A. Nickerson , A. Zuppardo , J. Terlonge , R. Curtiss 3rd , MlrA , a novel regulator of curli ( AgF ) and extracellular matrix synthesis by Escherichia coli and Salmonella enterica serovar Typhimurium , Mol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14643403" target="_blank">14643403</a>]<br></li></div>
				</div></ul><br>
			<label class="collapse_SIT" for="_5"><font size="5px">MlrA binding site</font></label>
				<input id="_5" type="checkbox">
				<div class="SIT"><br><ul><li>In E. coli , MlrA has been demonstrated to directly interact with an inverted-repeat sequence far upstream of the csgD promoter ( Figure 3 ; Table 1 ) [ 88 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25437188" target="_blank">25437188</a>]<br></li>
<li>[ 89 ] BasR/PmrA Escherichia coli K12 TTAANNTT ( N ) TTAANNTT 3 Cra Escherichia coli K12 RSTGAAWCSNTHHW CRP Escherichia coli K12 AAATGTGATCTAGATCACATTT CsgD Escherichia coli K12 CGGGKGAKNKA ‡ Escherichia coli K12 AAAAGNGNNAAAWW ‡ Salmonella enterica serovar Typhimurium ATCC14028 YGGGTGAGTYM § HGGGNDRNBNN § CpxR Escherichia coli K12 GTAA ( N5 -- 7 ) GTAA IHF Salmonella enterica serovar Typhimurium ATCC14028 WATCAA ( N4 ) TTR MlrA Escherichia coli K12 AAAATTGTACA ( 12N ) TGTACAATTTT OmpR Escherichia coli MG1655 TTTRWAYATWRDTTTACA ¶ Salmonella enterica serovar Typhimurium ATCC14028 ACNTTTNGNTACANNTAT RcdA Escherichia coli K12 TTGTNNACA RstA Escherichia coli K12 TACATNTNGTTACA † The consensus sequences are written in International Union of Pure and Applied Chemistry ( IUPAC ) nucleotide code .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25437188" target="_blank">25437188</a>]<br></li>
				</ul></div><br>
			<label class="collapse_TU" for="_6"><font size="5px">Transcription units regulated by MlrA</font></label>
				<input id="_6" type="checkbox">
				<div class="TU"><br><ul><li>It has been described that csgD expression is stimulated by MlrA ( Brown et al. , 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29163440" target="_blank">29163440</a>]<br></li>
<li>Furthermore , s may be required for both expression of s mlrA and direct co-operative interaction with the MlrA protein to initiate csg gene transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11489123" target="_blank">11489123</a>]<br></li>
<li>The fact that strain SL1344 recuperated its capacity to form a bioﬁlm in LB when complemented with the mlrA gene of strain 14028 could suggest either that the mlrA gene of SL1344 is not correctly transcribed or that the MlrA protein produced by strain SL1344 is not functional .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15458421" target="_blank">15458421</a>]<br></li>
<li>While csgD is positively regulated by RpoS via MlrA , it itself stimulates the production of curli ﬁmbriae through the transcriptional activation of the csgBAC operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16313619" target="_blank">16313619</a>]<br></li>
<li>RpoS is also required for the expression of MlrA and , at the same time activates the repressor cpxR [ 7,32 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14643403" target="_blank">14643403</a>]<br></li>
<li>Under the inﬂuence of unknown environmental signals , MlrA induces the csgD expression , which is the master regulator of the Salmonella ( and E. coli ) bioﬁlm formation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24735176" target="_blank">24735176</a>]<br></li>
<li>Furthermore , the csgD expression pattern in MM and CFA in the mlrA mutant is similar to that described in wt , indicating that MlrA is not required for the regulation of csgD by the composition of the culture media .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29163440" target="_blank">29163440</a>]<br></li>
				</div></ul></div><br>
	</div>
</body>
</html>