<!doctype html>
<style>
@charset "utf-8";
body {
	height: 100%;
	margin: 0;
	padding: 0;
}

.banner {
	position: fixed;
	top: 0;
	width: 100%;
	height: 65px;
	background: gray;
	line-height: 65px;
}
.spacer {
	height: 10px;
}

.main-content {
	padding: 5%;
}

.collapse_ACT {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-style: none solid none none;
	border-color: #B4045F;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_ACT + input{
	display: none;
}
.collapse_ACT + input + div{
	display: none;
}
.collapse_ACT + input:checked + div{
	display: block;
}
.ACT {
	text-align: justify;
	text-justify: inter-word;
}
.ACT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B4045F;
}
.ACT_collapse_similarity:hover{
	background: #e5d3dc;
}
.ACT_collapse_similarity + input {
	display: none;
}
.ACT_collapse_similarity + input + div{
	display: none;
}

.ACT_collapse_similarity + input:checked + div{
	display: block;
}
.ACT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.ACT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B4045F;
	padding-left: 5px;
}
.ACT_collapse_similarity:hover{
	background: #e5d3dc;
}
.ACT_collapse_similarity + input {
	display: none;
}
.ACT_collapse_similarity + input + div{
	display: none;
}
.ACT_collapse_similarity + input:checked + div{
	display: block;
}
.ACT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.ACT_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #B4045F;
}
.ACT_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_DOM {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-style: none solid none none;
	border-color: #B404AE;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_DOM + input{
	display: none;
}
.collapse_DOM + input + div{
	display: none;
}
.collapse_DOM + input:checked + div{
	display: block;
}
.DOM {
	text-align: justify;
	text-justify: inter-word;
}
.DOM_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B404AE;
	padding-left: 5px;
}
.DOM_collapse_similarity:hover{
	background: #e8d7e7;
}
.DOM_collapse_similarity + input {
	display: none;
}
.DOM_collapse_similarity + input + div{
	display: none;
}
.DOM_collapse_similarity + input:checked + div{
	display: block;
}
.DOM_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}.DOM_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #B404AE;
}
.DOM_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_EVO {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #7401DF;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_EVO + input{
	display: none;
}
.collapse_EVO + input + div{
	display: none;
}
.collapse_EVO + input:checked + div{
	display: block;
}
.EVO {
	text-align: justify;
	text-justify: inter-word;
}
.EVO_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #7401DF;
	padding-left: 5px;
}
.EVO_collapse_similarity:hover{
	background: #e2d7ed;
}
.EVO_collapse_similarity + input {
	display: none;
}
.EVO_collapse_similarity + input + div{
	display: none;
}
.EVO_collapse_similarity + input:checked + div{
	display: block;
}
.EVO_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.EVO_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #7401DF;
}
.EVO_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_RP {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #0174DF;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_RP + input{
	display: none;
}
.collapse_RP + input + div{
	display: none;
}
.collapse_RP + input:checked + div{
	display: block;
}
.RP {
	text-align: justify;
	text-justify: inter-word;
}
.RP_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #0174DF;
	padding-left: 5px;
}
.RP_collapse_similarity:hover{
	background: #c9d7e2;
}
.RP_collapse_similarity + input {
	display: none;
}
.RP_collapse_similarity + input + div{
	display: none;
}
.RP_collapse_similarity + input:checked + div{
	display: block;
}
.RP_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.RP_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #0174DF;
}
.RP_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_SIT {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #0489B1;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_SIT + input{
	display: none;
}
.collapse_SIT + input + div{
	display: none;
}
.collapse_SIT + input:checked + div{
	display: block;
}
.SIT {
	text-align: justify;
	text-justify: inter-word;
}
.SIT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #0489B1;
	padding-left: 5px;
}
.SIT_collapse_similarity:hover{
	background: #d0dfe2;
}
.SIT_collapse_similarity + input {
	display: none;
}
.SIT_collapse_similarity + input + div{
	display: none;
}
.SIT_collapse_similarity + input:checked + div{
	display: block;
}
.SIT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.SIT_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #0489B1;
}
.SIT_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_TU {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #04B486;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_TU + input{
	display: none;
}
.collapse_TU + input + div{
	display: none;
}
.collapse_TU + input:checked + div{
	display: block;
}
.TU {
	text-align: justify;
	text-justify: inter-word;
}
.TU_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #04B486;
	padding-left: 5px;
}
.TU_collapse_similarity:hover{
	background: #d0e2de;
}
.TU_collapse_similarity + input {
	display: none;
}
.TU_collapse_similarity + input + div{
	display: none;
}
.TU_collapse_similarity + input:checked + div{
	display: block;
}
.TU_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.TU_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #04B486;
}
.TU_similar > a {
	text-decoration: none;
	color: gray;
}

ul { 
	list-style-type: none;
	padding: 0;
	margin: 0;
}
ul li > a {
	text-decoration: none;
	color: gray;
}
ul li:hover, ul li:focus {
	background: #F0F0F0;
}

ul li:first-child span {
  border-left:none;  
}

</style>
<html>
<head>
<meta charset="utf-8">
<title>TF properties - NtrC</title>
<meta name="theme-color" content="#ffffff">
</head>
<body>
	<div class="banner">
		<center><font color="white" size="16px" face="arial">NtrC</font></center>
	</div>
	<div align="center" class="main-content">
		<div class="spacer">
	</div><br>			<label class="collapse_ACT" for="_1"><font size="5px">NtrC transcriptional activity</font></label>
				<input id="_1" type="checkbox">
				<div class="ACT"><br><ul><li>Phosphorylation site of NtrC , a protein phosphatase whose covalent intermediate activates transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10438755" target="_blank">10438755</a>]<br></li>
<li>This higher activity reﬂects the higher concentration of functional NtrC in this strain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10074092" target="_blank">10074092</a>]<br></li>
<li>This somewhat higher level likely reﬂects a higher concentration of functional NtrC as a result of the much longer doubling time ( 300 min ) observed under these conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10074092" target="_blank">10074092</a>]<br></li>
<li>The phosphorylated form of the enhancer-binding protein NtrC has an ATPase activity that is essential for activation of transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27583250" target="_blank">27583250</a>]<br></li>
<li>The role of ArgR in this activation under conditions of carbon limitation could be similar to that proposed above for activation by NtrC under conditions of nitrogen limitation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10074092" target="_blank">10074092</a>]<br></li>
<li>Lilja , A.E. , Jenssen , J.R. and Kahn , J.D. ( 2004 ) Geometric and dynamic requirements for DNA looping , wrapping and unwrapping in the activation of E.coli glnAp2 transcription by NtrC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26687718" target="_blank">26687718</a>]<br></li>
<li>Under nitrogen stress , the bEBP NtrC activates the intragenic 54-dependent promoter to express relA , thereby linking nitrogen stress response to the stringent response ( 57 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28373272" target="_blank">28373272</a>]<br></li>
<li>Because the E. coli study did not utilize a promiscuous , constitutively active bEBP or growth conditions that activate NtrC , which is the cognate bEBP for the argT 54-dependent promoter ( 64 ) , E 54 is likely to remain in stable closed complex at its promoter and block binding of E 70 to the overlapping primary 70-dependent promoter for argT .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28373272" target="_blank">28373272</a>]<br></li>
<li>Under the conditions of our assays , factors that inﬂuence the activity of 54-dependent promoters may be absent or affected , such as the indirect positive effects of DksA and ppGpp ( 65 ) and a promoter sequence-directed speciﬁcity for a particular bEBP even when the bEBP acts from solution , as demonstrated for the Klebsiella pneumoniae nifH promoter for activation by NtrC and NifA ( 66 , 67 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28373272" target="_blank">28373272</a>]<br></li>
<li>Under nitrogen-starvation conditions NtrC multimerizes , binds the enhancer in the cloned promoter region , and hydrolyzes ATP to stimulate transcription by Eσ54 at the glnKamtB promoter ( see Figure 1A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24007446" target="_blank">24007446</a>]<br></li>
<li>To compare the level of expression stimulated by DctD250 to the level that is seen under physiological conditions that activate the promoter-associated bEBP , lacZ expression from the glnKamtB promoter was assayed in the WT strain in nitrogen-limiting medium , which activates NtrC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24007446" target="_blank">24007446</a>]<br></li>
<li>In the presence of activated NtrC , the glnKamtB promoter expresses lacZ at a nearly 10-fold higher level than in the presence of DctD250 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24007446" target="_blank">24007446</a>]<br></li>
<li>Glutamate at the site of phosphorylation of nitrogen-regulatory protein NtrC mimics aspartyl-phos-phate and activates the protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10438755" target="_blank">10438755</a>]<br></li>
<li>Signal Sensing through Two-Component Systems S. Typhimurium has three bEBPs ( NtrC , ZraR , and GlrR ) that are response-regulators of TCSs , in which a sensor kinase protein recognizes the cellular signal , autophosphorylates , and transfers the phosphate to a conserved aspartate residue of the response-regulator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27583250" target="_blank">27583250</a>]<br></li>
<li>Accordingly , an increase in the L-arginine pool would increase the proportion of active ArgR with speciﬁc DNA binding activity , resulting in activation of the catabolic ast operon by NtrC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10074092" target="_blank">10074092</a>]<br></li>
<li>A common switch in activation of the response-regulators NtrC and PhoB : phosphorylation induces dimerization of the receiver modules .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16339942" target="_blank">16339942</a>]<br></li>
<li>However , we favor the first hypothesis , because in the presence of Mg21 phospho-PhoP showed a half-life of approximately 60 min under nondenaturing conditions and the half-life of the regulators that exhibit autophosphatase activity ranges from seconds in CheY to a few minutes in NtrC and PhoB ( 31 -- 33 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10807931" target="_blank">10807931</a>]<br></li>
<li>Cellular processes regulated by σ54-dependent bEBPs in S. Typhimurium include nitrogen metabolism in response to limiting nitrogen conditions [ NtrC ( GlnG , NRI ) , Keener and Kustu , 1988 ; Zimmer et al. , 2000 ] , transport and catabolism of D-glucosaminate ( DgaR , Miller et al. , 2013 ) and glucoselysine/fructoselysine ( GfrR , Miller et al. , 2015 ) , regulation of cytoplasmic pH homeostasis during fermentative growth by the formate-hydrogen lyase system ( FhlA , Hopper and Böck , 1995 ; Lamichhane-Khadka et al. , 2015 ) , response to assaults to the cell envelope ( PspF , Karlinsey et al. , 2010 ; Flores-Kim and Darwin , 2015 and zinc-dependent ZraR , Appia-Ayme et al. , 2012 ) , reduction of nitric oxide under anaerobic conditions ( NorR , Hutchings et al. , 2002 ; Mills et al. , 2005 ) , propionate catabolism ( PrpR , Palacios and Escalante-Semerena , 2000 ) , regulation of amino-sugar synthesis by sRNAs ( GlrR ; Gopel et al. , 2011 ) , and RNA repair/processing ( RtcR , Samuels , 2014 ; Engl et al. , 2016 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27583250" target="_blank">27583250</a>]<br></li>
<li>Unmodiﬁed GlnB inhibits NtrB kinase activity but GlnB-UMP can not interact with NtrB , thus allowing autophosphorylation of NtrB and transfer of the phosphate to NtrC ( Reitzer , 2003 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27583250" target="_blank">27583250</a>]<br></li>
<li>During nitrogen limitation , the level of α-ketoglutarate is high and inhibits GlnB interaction with NtrB , thereby increasing NtrC phosphorylation ( Schumacher et al. , 2013 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27583250" target="_blank">27583250</a>]<br></li>
<li>The E. coli glnH promoter has an IHF binding site ; when bound to this site , IHF serves to facilitate contact between NtrC and s54-holoenzyme and thus to increase glnH transcription ( 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9009317" target="_blank">9009317</a>]<br></li>
<li>The NtrC protein is a s54 transcriptional-activator ; it binds cooperatively to two upstream sites and activates transcription under nitrogen-limiting conditions by directly contacting s54-holoenzyme bound at the P2 promoter ( 48 , 50 , 52 , 60 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9009317" target="_blank">9009317</a>]<br></li>
<li>NtrC must be phosphorylated to activate transcription ; the NtrB protein causes phosphorylated NtrC ( NtrC-P ) to be formed only under nitrogen-limiting conditions ( 24 , 43 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9009317" target="_blank">9009317</a>]<br></li>
<li>NtrC ( GlnG ) The NtrB-NtrC TCS is activated in response to limited nitrogen conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27583250" target="_blank">27583250</a>]<br></li>
				</div></ul><br>
			<label class="collapse_DOM" for="_2"><font size="5px">NtrC protein domains</font></label>
				<input id="_2" type="checkbox">
				<div class="DOM"><br><ul><li>Repressor forms of enhancer-binding protein NtrC : some fail in coupling ATP hydrolysis to open complex formation by s54-holoenzyme .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9009317" target="_blank">9009317</a>]<br></li>
<li>Four of these EBPs have a similar domain organisation to NtrC whilst surprisingly the remaining three resemble LevR in Bacillus subtilis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12125817" target="_blank">12125817</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12581721" target="_blank">12581721</a>]<br></li>
<li>The labeled fragment was digested by Sau3A to generate two end-labeled fragments ; one of them is 210 bp and carries the two putative NtrC binding sites , and the other is 280 bp and carries the putative ArgR binding sites ( Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10074092" target="_blank">10074092</a>]<br></li>
<li>The phosphorylated form of the enhancer-binding protein NtrC has an ATPase activity that is essential for activation of transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27583250" target="_blank">27583250</a>]<br></li>
<li>Constitutive forms of the enhancer-binding protein NtrC : evidence that essential oligomerization determinants lie in the central activation domain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10648513" target="_blank">10648513</a>]<br></li>
<li>Studies with the glnA promoter of S. typhimurium have shown that NtrC bound at the enhancer , located between 2108 and 2140 , interacts directly with s54 holoenzyme by means of DNA loop formation ( 25 , 30 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10074092" target="_blank">10074092</a>]<br></li>
<li>A common switch in activation of the response regulators NtrC and PhoB : phosphorylation induces dimerization of the receiver modules .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16339942" target="_blank">16339942</a>]<br></li>
<li>Members of the NtrC family of EBPs have a modular domain organisation ( Morett and Segovia , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12125817" target="_blank">12125817</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12581721" target="_blank">12581721</a>]<br></li>
<li>Moore , J.B. , Shiau , S.P. , and Reitzer , L.J. ( 1993 ) Alterations of highly conserved residues in the regulatory domain of nitrogen regulator I ( NtrC ) of Escherichia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20545866" target="_blank">20545866</a>]<br></li>
<li>Protein kinase and phosphoprotein phosphatase activities of nitrogen regulatory proteins NtrB and NtrC of enteric bacteria : roles of the conserved amino-terminal domain of NtrC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27583250" target="_blank">27583250</a>]<br></li>
				</div></ul><br>
			<label class="collapse_EVO" for="_3"><font size="5px">NtrC evolution</font></label>
				<input id="_3" type="checkbox">
				<div class="EVO"><br><ul><li>Region of homology arLeth ngd Location Proteins coc re % Identity mpaodmp 33.8 219 4-215 vs 8-228 122 8-129 vs 2-216 vs 5-231 TctD vs OmpR TctD vs NtrC TctD vs ArcA 34.4 27.2 ACKNOWLEDGMENTS This work was supported by a grant to W.W.K. from thie Natural Sciences and Engineering Research Council of Canada .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2666399" target="_blank">2666399</a>]<br></li>
<li>Pseudomonas aeruginosa AlgB , a two-component response regulator of the NtrC family , is required for algD transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8063417" target="_blank">8063417</a>]<br></li>
<li>The C and D domains of PrpR are 56 % similar and 41 % identical to those of NtrC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10648513" target="_blank">10648513</a>]<br></li>
<li>One of these factors , HydG , shows a high degree of homology with the transcriptional activator NtrC from Klebsiella pneumoniae ( 34 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1885554" target="_blank">1885554</a>]<br></li>
<li>Members of the NtrC family of EBPs have a modular domain organisation ( Morett and Segovia , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12125817" target="_blank">12125817</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12581721" target="_blank">12581721</a>]<br></li>
<li>Enhancer-Binding Proteins ( EBPs ) in Bacteria In prokaryotes , one of the best-studied examples of regulatory proteins that use DNA-looping is the NtrC family of enhancer-binding proteins ( EBPs ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12125817" target="_blank">12125817</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12581721" target="_blank">12581721</a>]<br></li>
<li>One of the transcription regulator proteins belonging to the NtrC family from Aquifex aeolicus ( PDB ID : 1ny5B ) , having the best matches with each of these two proteins , was considered as the template for building their models .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24079299" target="_blank">24079299</a>]<br></li>
<li>Previous studies have shown that NtrC , a widely studied member of this family of activators , binds to its ELE and activates transcription through the assembly of large oligomers in which some of the dimers are not in direct contact with the DNA ( Wyman et al. , 1997 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15528672" target="_blank">15528672</a>]<br></li>
<li>PrpR of S. enterica belongs to the NtrC family of transcriptional activators and has been known to regulate the expression of the prpBCDE operon in response to propionate catabolism ( 32 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15805526" target="_blank">15805526</a>]<br></li>
<li>Cloning and nucleotide sequence of the gene encoding the positive regulator ( DmpR ) of the phenol catabolic pathway encoded by pVI150 and identiﬁcation of DmpR as a member of the NtrC family of transcriptional activators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10648513" target="_blank">10648513</a>]<br></li>
				</div></ul><br>
			<label class="collapse_RP" for="_4"><font size="5px">Processes regulated by NtrC</font></label>
				<input id="_4" type="checkbox">
				<div class="RP"><br><ul><li>Under nitrogen stress , the bEBP NtrC activates the intragenic 54-dependent promoter to express relA , thereby linking nitrogen stress response to the stringent-response ( 57 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28373272" target="_blank">28373272</a>]<br></li>
<li>NtrC ( GlnG ) The NtrB-NtrC TCS is activated in response to limited nitrogen conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27583250" target="_blank">27583250</a>]<br></li>
<li>Locus tagb Gene symbol Function bEBPc bEBP enhancer sequenced Activating signal/conditione STM0368-71 prpBCDE Propionate catabolism PrpR NtrC CGTTTCATGAAACG TGCACC ( A/T ) 4TGGTGCA 2-methylcitrate STM0462 glnK amtB Regulator of N metabolism ; NH3 transporter Low intracellular glutamine STM0665-62f gltIJKL Glutamate/aspartate-transporter TGCACC ( A/T ) 4TGGTGCA TGCACC ( A/T ) 4TGGTGCA NtrC NtrC Low intracellular glutamine Low intracellular glutamine Glutamine high-afﬁnity transporter STM0830-28 glnHPQ STM1285-84f yeaGH Serine protein kinase TGCACC ( A/T ) 4TGGTGCA TGCACC ( A/T ) 4TGGTGCA NtrC NtrC Low intracellular glutamine Low intracellular glutamine Arginine/ornithine/glutamine matabolism STM1303-07 astCABDE STM2355 argT Lysine/arginine/ornithine-transport protein TGCACC ( A/T ) 4TGGTGCA NtrC Low intracellular glutamine Glutamine synthetase TGCACC ( A/T ) 4TGGTGCA NK STM4007-05 STM0577-72 glnALG NtrC Low intracellular glutamine NK ---------- Putative PTS STM0571 STM0649-51 Putative hydrolase , NK STM0652 NK 2-keto-3-deoxygluconate permease Phage shock proteins STM1690-86 STM4244f pspABCDE PspF PspF TAGTGTAATTCGCTAACT TAGTGTAATTCGCTAACT Cell envelope stress Cell envelope stress Phage shock protein pspG STM2361 NorR ----- ubiX STM2360-56 STM2840-41 Amino acid transport NK NK Nitric oxide reductase norV ygbD GT ( N ) 7AC Nitric oxide Putative hydrogenase maturation proteins STM2843-42 hydN hypF CATTTCGTACGAAATG FhlA Formate STM2853-44 hycABCDEFGHI - Hydrogenase 3 CATTTCGTACGAAATG CATTTCGTACGAAATG FhlA FhlA Formate Formate STM2854-58 hypABCDE Hydrogenase maturation proteins STM3521-18 --------------- STM3568f NK RtcR Nucleic acid damage rsr yrlBA rtcBA Nucleic acid repair/processing Heat shock sigma factor ( σ32 ) NK NK NK NK rpoH STM3772-66 dgaABCDEF D-glucosaminate utilization -- -- -- -- -- -- -- -- -- DgaR D-glucosaminate STM4172 zraP Zinc-dependent chaperone NK NK ZraR ZraR [ Zinc ] and cell envelope stress [ Zinc ] and cell envelope stress STM4173-74 zraSR Zinc-responsive two component system provide insight into this anti-activator mechanism for regulating bEBP activity ( Flores-Kim and Darwin , 2015 ; Osadnik et al. , 2015 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27583250" target="_blank">27583250</a>]<br></li>
<li>The microarray results for S. Typhi-murium LT2 genes within operons that meet the criteria for up-regulation , or that are known or predicted to be σ54-dependent , are shown in Table 2 and Additional file 1 glmY GlmY sRNA glmZ GlmZ sRNA GlrR NtrC [ 28 ] [ 29 ] STM4007-05 glnALG Glutamine synthetase Predicted σ54-dependent operons : STM0462-63 glnK amtB hypothetical protein PTS ( putative ) NtrC 3.6 67 11 1.8 2.5 2.4 f 17 [ 22 ] [ 18 ] STM0577-72 STM0571 STM0652 NtrC STM0649.S-53 Hydrolase ( putative ) Glutamate/aspartate-transporter [ 18 ] [ 18,30 ] STM0665-62 gltIJKL yeaGH Serine protein kinase ( putative ) STM1285-84 NtrC NtrC [ 18,30 ] [ 31,32 ] [ 5 ] [ 18 ] STM1303-07 Arginine/ornithine/glutamine-metabolism Phage shock proteins Amino acid transport ( putative ) astCABDE STM1690-86 PspF STM2361 pspABCDE ------ ubiX norV ygbD STM2360-56 100 16 13 26 5.6 71 STM2840-41 Nitric oxide reductase NorR FhlA FhlA FhlA [ 18,33 ] [ 34 ] [ 35 ] [ 35 ] STM2843-42 Hydrogenase maturation proteins Hydrogenase 3 Formate-hydrogen lyase system hydN hypF hycABCDEFGHIhypABCDE - rtcBA STM2853-44 STM2854-58 STM3521-18 RtcR RNA-repair system ( putative ) Heat shock sigma factor ( σ32 ) PTS ( putative ) Zinc resistance-associated protein [ 23 ] STM3568 1.7 [ 36,37 ] [ 18 ] [ 3,18 ] rpoH STM3772-66 39 16 3.7 1.4 29 16 STM3773 ZraR ZraR PspF STM4172 zraP STM4173-74 [ 3 ] hydHG pspG Zinc resistance two-component system Phage shock protein Formate dehydrogenase PTS ( putative ) STM4244 [ 38 ] [ 39 ] [ 18 ] STM4285 FhlA fdhF STM4535-40 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24007446" target="_blank">24007446</a>]<br></li>
<li>The control region contains several regulatory consensus sequences , including binding sites for NtrC , cyclic AMP receptor protein ( CRP ) , and ArgR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10074092" target="_blank">10074092</a>]<br></li>
<li>AtoC , acetoacetate-metabolism regulatory protein from E. coli ( SWISS-PROT accession number Q06065 ) ; NtrC , nitrogen regulatory protein from S. typhimurium ( EMBL accession number X85104 ) ; PrpR , propionate-metabolism regulatory protein from S. typhimurium ( GenBank accession number U51879 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9006051" target="_blank">9006051</a>]<br></li>
<li>NtrC , nitrogen regulatory protein from Salmonella serovar Typhimurium ( accession no .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10648513" target="_blank">10648513</a>]<br></li>
<li>Genes Differentially Expressed ( FDR < 5 % ) in 2a Based upon Microarray Comparisons a Accession RE Gene GS AAL20730 AAL21761 AAL20023 AAL20490 AAL20022 AAL21753 AAL21754 AAL21775 AAL20021 AAL21763 AAL22362 AAL19500 AAL21766 AAL19551 AAL19499 AAL19498 AAL20835 AAL21762 AAL20872 AAL20838 AAL23285 AAL21759 AAL21776 AAL21777 AAL21774 AAL20871 AAL21760 AAL19822 AAL20024 AAL20906 AAL19933 AAL23063 AAL19497 AAL21747 AAL23226 AAL21779 AAL21301 AAL21770 AAL21778 AAL19683 AAL21752 AAL20731 AAL23064 AAL20832 AAL21764 AAL20834 AAL19879 AAL21063 AAL21765 AAL21773 AAL20401 AAL21750 AAL23055 AAL22926 AAL21744 AAL21767 AAL19501 AAL23352 AAL21564 AAL21899 AAL20867 AAL21905 Cell division inhibitor Putative acyl carrier protein minD iacP 45.0 45.0 41.0 40.4 40.1 38.2 36.9 35.7 35.7 34.0 32.9 32.7 32.6 32.1 31.5 30.5 28.0 25.0 25.0 25.0 25.0 25.0 25.0 25.0 25.0 25.0 25.0 25.0 25.0 25.0 25.0 22.7 22.6 22.6 21.7 21.6 21.4 21.1 20.0 19.3 19.1 18.6 17.4 13.6 12.1 9.8 9.6 8.7 8.4 8.2 7.9 7.8 7.1 7.1 6.7 6.5 6.3 6.1 5.9 5.9 5.5 5.3 Pathogenicity island encoded protein : SPI5 New outer membrane protein Pathogenicity island encoded protein : SPI5 Cytoplasmic cell invasion protein Cell invasion protein Surface presentation of antigens Pathogenicity island encoded protein : SPI5 Cell invasion protein Phosphoenolpyruvate carboxykinase Outer membrane usher protein Surface presentation of antigens Carbon starvation protein Periplasmic chaperone , required for type 1 ﬁmbriae Fimbrial protein internal segment Methyl accepting chemotaxis protein II Cell invasion protein Filament capping protein Enables ﬂagellar motor rotation Putative carbamate kinase Chaperone related to virulence Invasion protein A Invasion protein E Surface presentation of antigens Flagellar biosynthesis ; ﬂagellin Putative cytoplasmic protein Arginine 3rd transport system Pathogenicity island encoded protein : SPI5 Putative cold-shock protein Outer membrane protein 1a ( ia ; b ; f ) , porin Putative cytoplasmic protein Major type 1 subunit ﬁmbrin ( pilin ) AraC family bacterial regulatory helix-turn-helix protein Putative cytoplasmic protein Invasion protein Cold shock-induced palmitoleoyl transferase Surface presentation of antigens Outer membrane invasion protein Succinyl-coA synthetase , alpha subunit Cytoplasmic cell invasion protein Cell division topological speciﬁcity factor Putative cytoplasmic protein Chemotaxis regulator protein Cell invasion protein Glutamate methyltransferase Similar to cspa but not cold shock induced Putative sensor = kinase in regulatory system Cell invasion protein Surface presentation of antigens Putative membrane transporter of cations and cationic drugs Putative inner membrane protein Phage lambda receptor protein Glycerol kinase Fur regulated Salmonella iron-transporter Surface presentation of antigens Minor ﬁmbrial subunit NtrC family transcriptional regulator 16S rRNA-processing protein Putative cytoplasmic protein Putative regulator of ﬂia Putative ﬁmbrial-like protein sopB nmpC pipC prgI prgH invB STM1089 sipD pckA ﬁmD sicA cstA ﬁmC ﬁmI cheM sipA ﬂiD motB arcC sicP invA invE invC ﬂiC STM2880 artJ orfX cspB ompF STM4239 ﬁmA hilC ytfK invF ddg spaP invG sucD prgJ minE yjbJ cheY sipC cheR cspD yehU sipB invI ydgF orgA lamB glpK sitD spaS ﬁmH pspF rimM yohL ﬂiZ stdA ( continued Table 3 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18407759" target="_blank">18407759</a>]<br></li>
<li>DNA topoiomerase I Similar to MarT and ToxR transcriptional regulators Cold shock protein Multidrug-transporter ClpP protease Putative siderophore-binding protein Putative regulatory protein , possibly involved in gluconate utilisation Purine biosynthesis Putative transcriptional regulator Primosomal protein Putative heavy metal transport/resistance protein PhoPQ-activated gene Putative protein Putative outer membrane efﬂux protein STM3521 yeaG astC yjeB STM2746 topA STM0031 cspB acrD clpP yrdA yieP TGGCACGGAGTTAGCC TGGCATCCCCTTTGCG TGGAACGCCTCCTGCT TGGCAAGGTCTATGCA TGGCACTGTAGTGGCA TGGCACCCTGGGTGCA TGGCACGCGCGTCGCC purH ytfJ dnaT ygjT pagK yfcZ STM2690 B Score Gene name Promoter sequence Gene product/function Associated EBP ( Kullback-Liebler distance ) STM4535 STM3772 STM0577 STM2360 TGGCACGCCGCTTGCT TGGCACAACCTTTGCT TGGCACGCCGTTTGCC TGGCATGCCTTTTGCT Putative PTS EII component Putative PTS EII component Putative PTS EII component Putative diaminopimelate decarboxylase Nitrogen regulatory protein Propionate metabolism Hydrogenase accessory protein Resistance to zinc and lead Putative hydrolase Histidine sensor kinase Histidine transport protein Phage shock response Transcriptional repressor of formate hydrogenlyase Putative ﬂavoprotein Histidine sensor kinase , zinc / lead resistance STM4534 STM3773 STM0571 STM2361 2390 glnK prpB TGGCACATCCTTTGCA TGGCATAGCCTTTGCT TGGCATAAATATTGCT TGGCACGGAAGATGCA TGGCACGCCTTTTGAT TGGCACAATTACTGCA TGGCACGATAGTCGCA TGGCACGCAAATTGTA TGGCATGGAAAATGCT NtrC PrpR hypA zraP STM0649 yfhK hisJ pspA hycA FhlA HydG STM0652 YfhA STM2361 PspF FhlA 2383 2354 YgaA HydG STM2840 hydH TGGCACACTAGCTGCA TGGCATGATCTCTGCT YgaA and YfhA Two of the uncharacterised EBP-encoding genes in Salmonella , ygaA and yfhA , have orthologues in E. coli K12 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12125817" target="_blank">12125817</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12581721" target="_blank">12581721</a>]<br></li>
<li>sN-dependent enhancer binding protein ( EBP ) Proposed functions of genes References regulated by EBP Y. pestis S. typhimurium S. Typhi E. coli NtrC PspF YfhA YgaA NtrC PspF YfhA YgaA NtrC PspF YfhA YgaA Reitzer and Magasanik , 1986 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12125817" target="_blank">12125817</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12581721" target="_blank">12581721</a>]<br></li>
<li>.3504524 + 319067 T 3504469 T T A V ( GTG ) L ( CTG ) T G R ( CGG ) Lipopolysaccharide export system protein , Transport and forms ABC transporter with LptBFG permeability TetR family transcriptional regulator , Drug transport / repressor of ramA metabolism Low-afﬁnity inorganic phosphate Inorganic ion-transport transporter , PiT family and metabolism Putative formate/nitrate-transporter Anaerobic metabolism , transport and permeability Oxaloacetate decarboxylase subunit alpha , Amino acid involved in pyruvate-metabolism , conversion metabolism , energy of oxaloacetate to pyruvate and carbon production dioxide NtrC family of transcriptional regulator , involved in activation of nitrogen metabolism related promoters and sigma 54 ramR 637634 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25136333" target="_blank">25136333</a>]<br></li>
<li>Cellular processes regulated by σ54-dependent bEBPs in S. Typhimurium include nitrogen metabolism in response to limiting nitrogen conditions [ NtrC ( GlnG , NRI ) , Keener and Kustu , 1988 ; Zimmer et al. , 2000 ] , transport and catabolism of D-glucosaminate ( DgaR , Miller et al. , 2013 ) and glucoselysine/fructoselysine ( GfrR , Miller et al. , 2015 ) , regulation of cytoplasmic pH homeostasis during fermentative growth by the formate-hydrogen lyase system ( FhlA , Hopper and Böck , 1995 ; Lamichhane-Khadka et al. , 2015 ) , response to assaults to the cell envelope ( PspF , Karlinsey et al. , 2010 ; Flores-Kim and Darwin , 2015 and zinc-dependent ZraR , Appia-Ayme et al. , 2012 ) , reduction of nitric oxide under anaerobic conditions ( NorR , Hutchings et al. , 2002 ; Mills et al. , 2005 ) , propionate-catabolism ( PrpR , Palacios and Escalante-Semerena , 2000 ) , regulation of amino-sugar synthesis by sRNAs ( GlrR ; Gopel et al. , 2011 ) , and RNA-repair/processing ( RtcR , Samuels , 2014 ; Engl et al. , 2016 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27583250" target="_blank">27583250</a>]<br></li>
<li>Transcription of the glnH promoter is s54 and NtrC dependenta b-Galactosidase activity in strain with mutation glnHp-9lacZb glnAp-9lacZc lacUV5p-9lacZd RESULTS Single mutations in genes involved with glutamine-synthesis do not affect the virulence of S. typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9009317" target="_blank">9009317</a>]<br></li>
<li>Ramakrishnan G , Newton A. FlbD of Caulobacter crescentus is a homologue of NtrC ( NRI ) and activates sigma-54 dependent flagellar gene promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19942438" target="_blank">19942438</a>]<br></li>
<li>Glutamate at the site of phosphorylation of nitrogen-regulatory protein NtrC mimics aspartyl-phos-phate and activates the protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10438755" target="_blank">10438755</a>]<br></li>
<li>Phosphorylation site of NtrC , a protein phosphatase whose covalent intermediate activates transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10438755" target="_blank">10438755</a>]<br></li>
				</div></ul><br>
			<label class="collapse_SIT" for="_5"><font size="5px">NtrC binding site</font></label>
				<input id="_5" type="checkbox">
				<div class="SIT"><br><ul><li>s70-dependent transcription is repressed when NtrC is bound at these two sites , and thus an ntrA strain is a glutamine auxotroph while an ntrC strain is a glutamine prototroph .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9009317" target="_blank">9009317</a>]<br></li>
<li>The control region contains several regulatory consensus sequences , including binding sites for NtrC , cyclic AMP receptor protein ( CRP ) , and ArgR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10074092" target="_blank">10074092</a>]<br></li>
<li>NtrC binds to two sites centered at 2108 and 2140 ( white boxes ) with respect to the s54-dependent start site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9009317" target="_blank">9009317</a>]<br></li>
<li>NtrC binds to two overlapping high-afﬁnity sites in the E. coli sequence ( marked 2 and 3 [ black boxes ] ) at low concentrations and to two additional low-afﬁnity sites ( marked 1 and 4 [ white boxes ] ) at higher concentrations .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9009317" target="_blank">9009317</a>]<br></li>
<li>Potential half-sites for NtrC binding which resemble the consensus GGTGCA ( 50 ) have been underlined in the S. typhimurium sequence ; two half-sites appear to overlap ( within sites 2 and 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9009317" target="_blank">9009317</a>]<br></li>
<li>NtrC binding sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10074092" target="_blank">10074092</a>]<br></li>
<li>Identification of an NtrC protein-binding site and definition of a consensus binding sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24007446" target="_blank">24007446</a>]<br></li>
<li>The 39 end of the proximal NtrC site is about 200 bp upstream of the putative s54 promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10074092" target="_blank">10074092</a>]<br></li>
<li>Our results have extended the characterization of high-afﬁnity glutamine transport in S. typhimurium to show that ( i ) the sequence of the S. typhimurium glnH promoter is homologous to that from E. coli and shares a consensus s54-holoenzyme binding site as well as several apparent NtrC binding sites ; and ( ii ) high-level transcription of the S. typhimurium glnH promoter requires both s54 and NtrC , as measured by transcriptional glnHp-lacZ fusions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9009317" target="_blank">9009317</a>]<br></li>
<li>Computer analysis of the nucleotide sequence of the region upstream of the ast operon in E. coli identiﬁed a putative s54 consensus sequence and two putative NtrC binding sites ; such sequences are consistent with the observed nitrogen regulation of the operon ( 3 , 23 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10074092" target="_blank">10074092</a>]<br></li>
<li>Moreover , the putative s54-holoenzyme ( Es54 ) binding site in the S. typhimurium sequence matched that in the E. coli sequence at 15 of 17 nucleotides , and several reasonably good sequence motifs could be matched to the optimal S. typhimurium NtrC binding site TGCACCN5GGTGCA ( 50 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9009317" target="_blank">9009317</a>]<br></li>
<li>Certain features noted in the upstream sequence in E. coli ( 3 ) -- a putative s54 promoter ( 18 ) , two potential NtrC binding sites ( 18 ) , and a putative integration host factor ( IHF ) binding site ( 4 ) -- are present at the corresponding locations in the S. typhimurium sequence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10074092" target="_blank">10074092</a>]<br></li>
<li>The S. typhi-murium sequence ( Fig. 1 ) contains a sequence downstream of the NtrC sites that appears to be a good candidate for a CRP site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10074092" target="_blank">10074092</a>]<br></li>
<li>Gel retardation experiments showed that ArgR binds speciﬁcally to a DNA fragment carrying the region downstream of the NtrC binding sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10074092" target="_blank">10074092</a>]<br></li>
				</ul></div><br>
			<label class="collapse_TU" for="_6"><font size="5px">Transcription unit regulated by NtrC</font></label>
				<input id="_6" type="checkbox">
				<div class="TU"><br><ul><li>To compare the level of expression stimulated by DctD250 to the level that is seen under physiological conditions that activate the promoter-associated bEBP , lacZ expression from the glnKamtB promoter was assayed in the WT strain in nitrogen-limiting medium , which activates NtrC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24007446" target="_blank">24007446</a>]<br></li>
<li>In Escherichia coli , the major glnH promoter has been shown to be transcribed by s54-holoenzyme and NtrC in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9009317" target="_blank">9009317</a>]<br></li>
<li>NtrC and s54-holoenzyme have further been shown to transcribe the glnH promoter of E. coli in vitro ( 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9009317" target="_blank">9009317</a>]<br></li>
				</div></ul></div><br>
	</div>
</body>
</html>