<!doctype html>
<style>
@charset "utf-8";
body {
	height: 100%;
	margin: 0;
	padding: 0;
}

.banner {
	position: fixed;
	top: 0;
	width: 100%;
	height: 65px;
	background: gray;
	line-height: 65px;
}
.spacer {
	height: 10px;
}

.main-content {
	padding: 5%;
}

.collapse_ACT {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-style: none solid none none;
	border-color: #B4045F;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_ACT + input{
	display: none;
}
.collapse_ACT + input + div{
	display: none;
}
.collapse_ACT + input:checked + div{
	display: block;
}
.ACT {
	text-align: justify;
	text-justify: inter-word;
}
.ACT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B4045F;
}
.ACT_collapse_similarity:hover{
	background: #e5d3dc;
}
.ACT_collapse_similarity + input {
	display: none;
}
.ACT_collapse_similarity + input + div{
	display: none;
}

.ACT_collapse_similarity + input:checked + div{
	display: block;
}
.ACT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.ACT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B4045F;
	padding-left: 5px;
}
.ACT_collapse_similarity:hover{
	background: #e5d3dc;
}
.ACT_collapse_similarity + input {
	display: none;
}
.ACT_collapse_similarity + input + div{
	display: none;
}
.ACT_collapse_similarity + input:checked + div{
	display: block;
}
.ACT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.ACT_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #B4045F;
}
.ACT_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_DOM {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-style: none solid none none;
	border-color: #B404AE;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_DOM + input{
	display: none;
}
.collapse_DOM + input + div{
	display: none;
}
.collapse_DOM + input:checked + div{
	display: block;
}
.DOM {
	text-align: justify;
	text-justify: inter-word;
}
.DOM_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B404AE;
	padding-left: 5px;
}
.DOM_collapse_similarity:hover{
	background: #e8d7e7;
}
.DOM_collapse_similarity + input {
	display: none;
}
.DOM_collapse_similarity + input + div{
	display: none;
}
.DOM_collapse_similarity + input:checked + div{
	display: block;
}
.DOM_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}.DOM_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #B404AE;
}
.DOM_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_EVO {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #7401DF;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_EVO + input{
	display: none;
}
.collapse_EVO + input + div{
	display: none;
}
.collapse_EVO + input:checked + div{
	display: block;
}
.EVO {
	text-align: justify;
	text-justify: inter-word;
}
.EVO_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #7401DF;
	padding-left: 5px;
}
.EVO_collapse_similarity:hover{
	background: #e2d7ed;
}
.EVO_collapse_similarity + input {
	display: none;
}
.EVO_collapse_similarity + input + div{
	display: none;
}
.EVO_collapse_similarity + input:checked + div{
	display: block;
}
.EVO_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.EVO_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #7401DF;
}
.EVO_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_RP {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #0174DF;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_RP + input{
	display: none;
}
.collapse_RP + input + div{
	display: none;
}
.collapse_RP + input:checked + div{
	display: block;
}
.RP {
	text-align: justify;
	text-justify: inter-word;
}
.RP_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #0174DF;
	padding-left: 5px;
}
.RP_collapse_similarity:hover{
	background: #c9d7e2;
}
.RP_collapse_similarity + input {
	display: none;
}
.RP_collapse_similarity + input + div{
	display: none;
}
.RP_collapse_similarity + input:checked + div{
	display: block;
}
.RP_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.RP_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #0174DF;
}
.RP_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_SIT {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #0489B1;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_SIT + input{
	display: none;
}
.collapse_SIT + input + div{
	display: none;
}
.collapse_SIT + input:checked + div{
	display: block;
}
.SIT {
	text-align: justify;
	text-justify: inter-word;
}
.SIT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #0489B1;
	padding-left: 5px;
}
.SIT_collapse_similarity:hover{
	background: #d0dfe2;
}
.SIT_collapse_similarity + input {
	display: none;
}
.SIT_collapse_similarity + input + div{
	display: none;
}
.SIT_collapse_similarity + input:checked + div{
	display: block;
}
.SIT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.SIT_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #0489B1;
}
.SIT_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_TU {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #04B486;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_TU + input{
	display: none;
}
.collapse_TU + input + div{
	display: none;
}
.collapse_TU + input:checked + div{
	display: block;
}
.TU {
	text-align: justify;
	text-justify: inter-word;
}
.TU_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #04B486;
	padding-left: 5px;
}
.TU_collapse_similarity:hover{
	background: #d0e2de;
}
.TU_collapse_similarity + input {
	display: none;
}
.TU_collapse_similarity + input + div{
	display: none;
}
.TU_collapse_similarity + input:checked + div{
	display: block;
}
.TU_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.TU_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #04B486;
}
.TU_similar > a {
	text-decoration: none;
	color: gray;
}

ul { 
	list-style-type: none;
	padding: 0;
	margin: 0;
}
ul li > a {
	text-decoration: none;
	color: gray;
}
ul li:hover, ul li:focus {
	background: #F0F0F0;
}

ul li:first-child span {
  border-left:none;  
}

</style>
<html>
<head>
<meta charset="utf-8">
<title>TF properties - PefI</title>
<meta name="theme-color" content="#ffffff">
</head>
<body>
	<div class="banner">
		<center><font color="white" size="16px" face="arial">PefI</font></center>
	</div>
	<div align="center" class="main-content">
		<div class="spacer">
	</div><br>			<label class="collapse_ACT" for="_1"><font size="5px">PefI transcriptional activity</font></label>
				<input id="_1" type="checkbox">
				<div class="ACT"><br><ul><li>It seems likely that fae and pef share common regulatory features , which place them under negative regulatory control by FaeA and PefI respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10692151" target="_blank">10692151</a>]<br></li>
<li>Thus , H-NS and RpoS , like PefI , appear to regulate Pef production negatively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10692151" target="_blank">10692151</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_3">
PefI appeared to be required for binding of Lrp to GATC II based on methylation protection analysis ( Table 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10692151" target="_blank">10692151</a>]<br></label>
<input id="act_similarity_3", type="checkbox">
<div class="ACT_similar">
<li>Moreover , PefI appeared to be required for binding of Lrp to GATC II at low pH based on methylation protection analysis ( Table 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10692151" target="_blank">10692151</a>]<br></li></div>
<li>RcsB is an activator of capsule synthesis , FimZ upregulates type 1 ﬁmbriae , PefI controls the synthesis of virulence plasmid-encoded ﬁmbriae , and YdiV has weak homology to EAL domain proteins which are known to decrease c-di-GMP levels , enhance swimming , and decrease cell adhesion to surfaces .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19114490" target="_blank">19114490</a>]<br></li>
<li>Solution NMR structure of the plasmid-encoded fimbriae regulatory protein PefI from Salmonella enterica serovar Typhimurium James M. Aramini1 ,2 , * , Paolo Rossi1 ,2 , John R. Cort2 ,3 , Li-Chung Ma1 ,2 , Rong Xiao1 ,2 , Thomas B. Acton1 ,2 , and Gaetano T. Montelione1 ,2,4 , * 1Center for Advanced Biotechnology and Medicine , Department of Molecular Biology and Biochemistry , Rutgers , The State University of New Jersey , Piscataway , New Jersey 08854 2Northeast Structural Genomics Consortium ( NESG ) , Rutgers , The State University of New Jersey , Piscataway , New Jersey 08854 3Biological Sciences Division , Pacific Northwest National Laboratory , Richland , WA 99352 4Department of Biochemistry , Robert Wood Johnson Medical School , University of Medicine and Dentistry of New Jersey , Piscataway , New Jersey 08854 Keywords PefI ; plasmid-encoded fimbriae regulatory protein ; solution NMR structure ; winged helix-turn-helix ; Northeast Structural Genomics Consortium tion c u d tro In The surfaces of many bacteria feature pili or fimbriae , proteinaceous filaments that play an integral role in the adhesion of bacteria to host cells and , hence , in their pathogenicity .1,2 These extracellular structures are key virulence factors and potential targets for antibacterial drug and vaccine development .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20979070" target="_blank">20979070</a>]<br></li>
<li>PefI negatively regulates the production of PefA , the major fimbrial subunit of plas-mid-encoded fimbriae ( Pef ) [ 42 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18652702" target="_blank">18652702</a>]<br></li>
<li>In contrast , PefI appeared to regulate pef negatively , as mutation of pefI caused an increase in Pef fimbriation , and overexpression of PefI blocked pef transcription ( Fig. 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10692151" target="_blank">10692151</a>]<br></li>
<li>Instead , it seems likely that other factor ( s ) , such as the sequences and locations of Lrp binding sites within pef and pap , contribute to the differences in the activities of PefI and PapI in their respective hosts .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10692151" target="_blank">10692151</a>]<br></li>
<li>Similarly , in Salmonella , methylation of pef GATC II triggers Pef fimbrial expression .7 However , PefI acts in tandem with Lrp to block methylation of this site and , hence , represses transcription of pef genes .7 Remarkably , expression of PefI activates pap transcription in E. coli .7 Moreover , the E. coli ortholog FaeA represses K88 fimbrial production by an analogous mechanism .29,30 These findings suggest that the different biological roles of these functional homologs in bacterial fimbrial production are dependent upon factors in addition to their intrinsic biochemical and structural properties .7 Deciphering the exact nature of the interaction between FaeA-like proteins and their cognate Lrp and the mechanism of their regulation of fimbrial transcription awaits further biochemical and structural studies , which will be enabled by protein sample production protocols , NMR chemical shift data , and structural information such as that presented here for stPefI .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20979070" target="_blank">20979070</a>]<br></li>
<li>These results suggest that PefI facilitates binding of Lrp to the GATC II region , in contrast to the role of PapI to move Lrp to the pap GATC I site ( Kaltenbach et al. , 1995 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10692151" target="_blank">10692151</a>]<br></li>
<li>These results are consist-ent with the observation that PefI is a negative regulator of pef transcription , assuming that binding of Lrp to the pef GATC II site inhibits transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10692151" target="_blank">10692151</a>]<br></li>
				</div></ul><br>
			<label class="collapse_DOM" for="_2"><font size="5px">PefI protein domains</font></label>
				<input id="_2" type="checkbox">
				<div class="DOM"><br><ul><li>and Discussion esults R The solution NMR structure of stPefI consists of a 3-stranded winged-helix-turn-helix ( wHTH ) motif , comprised of three α-helices ( α1 , S4-D21 ; α2 , T27-G35 ; α3 , S37-V51 ) and three short antiparallel β-strands ( β1 , V25-K26 ; β2 , L54-V57 ; β3 , L66-L69 ) arranged in an αβααββ topology [ Fig. 1 ( B , C ) ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20979070" target="_blank">20979070</a>]<br></li>
<li>We demonstrate that the structure of stPefI adopts a winged-helix-turn-helix motif ,9,10 consistent with its role as a DNA-binding transcriptional regulator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20979070" target="_blank">20979070</a>]<br></li>
<li>Like stPefI , ecPapI features a highly basic wHTH face opposite a highly acidic face ( not shown ) , and a disordered loop within the wing .11 NMR chemical shift perturbation studies on ecPapI have revealed that residues within the wing and conserved residues in the major-groove binding recognition helix ( α3 ) are sensitive to DNA-binding ; 11 we predict that stPefI behaves in an analogous fashion .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20979070" target="_blank">20979070</a>]<br></li>
<li>References The solution structure of stPefI is highly similar to the structures of a plethora of DNA-binding proteins and transcriptional regulators , including the only other structure to date from the FaeA-like-protein domain family , the solution structure of E. coli PapI ( PDB ID , 2HTJ ; 11 Dali26 Z-score , 7.1 ; Cα RMSD , 2.2 Å ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20979070" target="_blank">20979070</a>]<br></li>
<li>The structure of stPefI exhibits a number of signatures of the classic wHTH motif including , i ) a tri-helical core comprising the helix-turn-helix ( α2-turn-α3 ) followed by a C-terminal beta hairpin wing ( β2-loop-β3 ) , ii ) a ` shs ' sequence pattern ( where ` s ' and ` h ' denote small and hydrophobic residues , respectively ) in the tight turn between α2 and α3 ( G35-L36-S37 ) , and ( iii ) a conserved ` phs ' sequence motif ( where ` p ' is a charged residue ) within α2 ( D29-I30-A31 ) .9,10 Electrostatic surface potential24 [ Fig. 1 ( D ) ] and ConSurf25 [ Fig. 1 ( E ) ] analyses of the stPefI structure reveal a positively charged patch and several conserved residues on the face of the protein comprising the HTH and the wing , consistent with the integral roles of the recognition helix ( α3 ) and the wing in DNA-binding by wHTH proteins .9,10 The opposite face of the protein is less well conserved and features a quite negatively charged surface ( not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20979070" target="_blank">20979070</a>]<br></li>
<li>( E ) ConSurf25 image showing the conserved residues in the wHTH motif of stPefI .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20979070" target="_blank">20979070</a>]<br></li>
				</div></ul><br>
			<label class="collapse_EVO" for="_3"><font size="5px">PefI evolution</font></label>
				<input id="_3" type="checkbox">
				<div class="EVO"><br><ul><li>Identification of trans-acting factors that control pef production The PapI regulatory protein and PefI are 29 % identical at the amino acid level ( Friedrich et al. , 1993 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10692151" target="_blank">10692151</a>]<br></li>
<li>PefI is homologous to the E. coli ﬁmbrial regulatory proteins FaeA and PapI ( 39 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19114490" target="_blank">19114490</a>]<br></li>
<li>References The solution structure of stPefI is highly similar to the structures of a plethora of DNA binding proteins and transcriptional regulators , including the only other structure to date from the FaeA-like protein domain family , the solution structure of E. coli PapI ( PDB ID , 2HTJ ; 11 Dali26 Z-score , 7.1 ; Cα RMSD , 2.2 Å ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20979070" target="_blank">20979070</a>]<br></li>
				</div></ul><br>
			<label class="collapse_RP" for="_4"><font size="5px">Processes regulated by PefI</font></label>
				<input id="_4" type="checkbox">
				<div class="RP"><br><ul><li>Gene , operon or Location Cellular function locus Phenotype affected Molecular mechanism Known regulators ﬂjBA , ﬂiC ﬁm gtr Chromosome Chromosome Motility , adhesion and virulence Adhesion to the intestinal epithelium Lipopolysaccharide O-antigen glycosylation Modiﬁcation of O-antigen chain length Adhesion to the intestinal epithelium Adhesion to the intestinal epithelium Epithelial cell invasion Bioﬁlm formation Motility , adhesion and virulence Flagella Fimbriae LPS Site-speciﬁc recombination Unknown DNA-methylation pattern Hin FimB , FimE OxyR Chromosome and prophage P22 Chromosome Virulence plasmid Chromosome Chromosome Chromosome Chromosome Chromosome Chromosome opvAB pef std SPI-1 csgD ﬂiC lpf iol LPS Fimbriae DNA-methylation pattern DNA-methylation pattern DNA-methylation pattern Unknown Unknown Unknown Unknown Unknown OxyR Lrp , PefI Fimbriae TTSS Extracellular matrix Flagella Fimbriae myo-inositol utilization Dam , HdfR , RosE , SeqA StdE , StdF , FliZ c-di-GMP FliA , ClpX , FliZ , YdiV IolR , ReiD Fig. 1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29369799" target="_blank">29369799</a>]<br></li>
<li>The omega element carries transcriptional terminators and translational stops flanking the chloramphenicol resistance gene , which blocks expression of PefI .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10692151" target="_blank">10692151</a>]<br></li>
<li>RcsB is an activator of capsule synthesis , FimZ upregulates type 1 ﬁmbriae , PefI controls the synthesis of virulence plasmid-encoded ﬁmbriae , and YdiV has weak homology to EAL domain proteins which are known to decrease c-di-GMP levels , enhance swimming , and decrease cell-adhesion to surfaces .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19114490" target="_blank">19114490</a>]<br></li>
<li>In contrast , PefI appeared to regulate pef negatively , as mutation of pefI caused an increase in Pef fimbriation , and overexpression of PefI blocked pef transcription ( Fig. 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10692151" target="_blank">10692151</a>]<br></li>
<li>PefI , a homolog of a regulator of the E. coli pap ﬁmbrial operon , PapI [ 57 ] , represses pef expression , thereby skewing phase variation towards the OFF state .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29369799" target="_blank">29369799</a>]<br></li>
<li>PefI is a negative regulator of this pef operon ( Nicholson and Low , 2000 ) and SrgA , a paralogue of the disulphide oxidoreductase DsbA , is involved in the correct folding of PefA ﬁmbrial subunit ( Bouwman et al. , 2003 ) and SpiA , a component of the T3SS-2 ( Miki et al. , 2004 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25080967" target="_blank">25080967</a>]<br></li>
<li>In certain Salmonella enterobacteria strains , one such class of fimbriae , the virulence plasmid-encoded fimbriae encoded by the pef operon ,3,4 has been shown to be important for adhesion to murine small intestine and fluid accumulation ,5 as well as biofilm-formation on eukaryotic cell surfaces .6 Transcription of the pef operon , induced under acidic conditions , is modulated by the PefI regulatory protein .7 PefI represses plasmid-encoded fimbrial protein production by activating the leucine-responsive regulatory protein ( Lrp ) mediated inhibition of DNA-methylation within the pef promoter region .7 PefI expression also inhibits transcription of flagellar proteins and consequently cell-motility .8 Here we present the solution NMR structure of the 70-residue PefI transcription regulator from Salmonella enterica serovar Typhimurium LT2 [ UniProtKB/TrEMBL ID , ; NESG ID , StR82 ; hereafter referred to as stPefI ] , a member of the FaeA like protein domain family ( Pfam identifier , PF04703 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20979070" target="_blank">20979070</a>]<br></li>
<li>PefI negatively regulates the production of PefA , the major fimbrial subunit of plas-mid-encoded fimbriae ( Pef ) [ 42 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18652702" target="_blank">18652702</a>]<br></li>
				</div></ul><br>
			<label class="collapse_SIT" for="_5"><font size="5px">PefI binding site</font></label>
				<input id="_5" type="checkbox">
				<div class="SIT"><br><ul><li>[ PubMed : 7476191 ] ( A ) Structure-based sequence alignment of stPefI and E. coli PapI ( ecPapI ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20979070" target="_blank">20979070</a>]<br></li>
<li>Pef shares similar GATC box I and II sequences , 102 bp spacing between GATC boxes , a PapI-like homologue , PefI , and sequence similarity within the 2 35 and 2 10 hexamer binding sites for RNA polymerase ( Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10692151" target="_blank">10692151</a>]<br></li>
<li>Instead , it seems likely that other factor ( s ) , such as the sequences and locations of Lrp binding sites within pef and pap , contribute to the differences in the activities of PefI and PapI in their respective hosts .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10692151" target="_blank">10692151</a>]<br></li>
				</ul></div><br>
			<label class="collapse_TU" for="_6"><font size="5px">Transcription units regulated by PefI</font></label>
				<input id="_6" type="checkbox">
				<div class="TU"><br><ul><li>It seems likely that fae and pef share common regulatory features , which place them under negative regulatory control by FaeA and PefI respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10692151" target="_blank">10692151</a>]<br></li>
<li>Complementation analysis with a papBA ± lacZ reporter gene construct showed that pefI and papI are functional homologues , as PefI expressed from a multicopy plasmid activates papBA transcription ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10692151" target="_blank">10692151</a>]<br></li>
<li>Two of these components , PefI and SrgA ( for sdiA-regulated gene ) are related to the pef ( plasmid-encoded fimbriae ) operon located just upstream of pefI .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25080967" target="_blank">25080967</a>]<br></li>
				</div></ul></div><br>
	</div>
</body>
</html>