<!doctype html>
<style>
@charset "utf-8";
body {
	height: 100%;
	margin: 0;
	padding: 0;
}

.banner {
	position: fixed;
	top: 0;
	width: 100%;
	height: 65px;
	background: gray;
	line-height: 65px;
}
.spacer {
	height: 10px;
}

.main-content {
	padding: 5%;
}

.collapse_ACT {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-style: none solid none none;
	border-color: #B4045F;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_ACT + input{
	display: none;
}
.collapse_ACT + input + div{
	display: none;
}
.collapse_ACT + input:checked + div{
	display: block;
}
.ACT {
	text-align: justify;
	text-justify: inter-word;
}
.ACT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B4045F;
}
.ACT_collapse_similarity:hover{
	background: #e5d3dc;
}
.ACT_collapse_similarity + input {
	display: none;
}
.ACT_collapse_similarity + input + div{
	display: none;
}

.ACT_collapse_similarity + input:checked + div{
	display: block;
}
.ACT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.ACT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B4045F;
	padding-left: 5px;
}
.ACT_collapse_similarity:hover{
	background: #e5d3dc;
}
.ACT_collapse_similarity + input {
	display: none;
}
.ACT_collapse_similarity + input + div{
	display: none;
}
.ACT_collapse_similarity + input:checked + div{
	display: block;
}
.ACT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.ACT_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #B4045F;
}
.ACT_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_DOM {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-style: none solid none none;
	border-color: #B404AE;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_DOM + input{
	display: none;
}
.collapse_DOM + input + div{
	display: none;
}
.collapse_DOM + input:checked + div{
	display: block;
}
.DOM {
	text-align: justify;
	text-justify: inter-word;
}
.DOM_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B404AE;
	padding-left: 5px;
}
.DOM_collapse_similarity:hover{
	background: #e8d7e7;
}
.DOM_collapse_similarity + input {
	display: none;
}
.DOM_collapse_similarity + input + div{
	display: none;
}
.DOM_collapse_similarity + input:checked + div{
	display: block;
}
.DOM_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}.DOM_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #B404AE;
}
.DOM_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_EVO {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #7401DF;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_EVO + input{
	display: none;
}
.collapse_EVO + input + div{
	display: none;
}
.collapse_EVO + input:checked + div{
	display: block;
}
.EVO {
	text-align: justify;
	text-justify: inter-word;
}
.EVO_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #7401DF;
	padding-left: 5px;
}
.EVO_collapse_similarity:hover{
	background: #e2d7ed;
}
.EVO_collapse_similarity + input {
	display: none;
}
.EVO_collapse_similarity + input + div{
	display: none;
}
.EVO_collapse_similarity + input:checked + div{
	display: block;
}
.EVO_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.EVO_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #7401DF;
}
.EVO_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_RP {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #0174DF;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_RP + input{
	display: none;
}
.collapse_RP + input + div{
	display: none;
}
.collapse_RP + input:checked + div{
	display: block;
}
.RP {
	text-align: justify;
	text-justify: inter-word;
}
.RP_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #0174DF;
	padding-left: 5px;
}
.RP_collapse_similarity:hover{
	background: #c9d7e2;
}
.RP_collapse_similarity + input {
	display: none;
}
.RP_collapse_similarity + input + div{
	display: none;
}
.RP_collapse_similarity + input:checked + div{
	display: block;
}
.RP_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.RP_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #0174DF;
}
.RP_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_SIT {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #0489B1;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_SIT + input{
	display: none;
}
.collapse_SIT + input + div{
	display: none;
}
.collapse_SIT + input:checked + div{
	display: block;
}
.SIT {
	text-align: justify;
	text-justify: inter-word;
}
.SIT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #0489B1;
	padding-left: 5px;
}
.SIT_collapse_similarity:hover{
	background: #d0dfe2;
}
.SIT_collapse_similarity + input {
	display: none;
}
.SIT_collapse_similarity + input + div{
	display: none;
}
.SIT_collapse_similarity + input:checked + div{
	display: block;
}
.SIT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.SIT_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #0489B1;
}
.SIT_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_TU {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #04B486;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_TU + input{
	display: none;
}
.collapse_TU + input + div{
	display: none;
}
.collapse_TU + input:checked + div{
	display: block;
}
.TU {
	text-align: justify;
	text-justify: inter-word;
}
.TU_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #04B486;
	padding-left: 5px;
}
.TU_collapse_similarity:hover{
	background: #d0e2de;
}
.TU_collapse_similarity + input {
	display: none;
}
.TU_collapse_similarity + input + div{
	display: none;
}
.TU_collapse_similarity + input:checked + div{
	display: block;
}
.TU_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.TU_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #04B486;
}
.TU_similar > a {
	text-decoration: none;
	color: gray;
}

ul { 
	list-style-type: none;
	padding: 0;
	margin: 0;
}
ul li > a {
	text-decoration: none;
	color: gray;
}
ul li:hover, ul li:focus {
	background: #F0F0F0;
}

ul li:first-child span {
  border-left:none;  
}

</style>
<html>
<head>
<meta charset="utf-8">
<title>TF properties - PrpR</title>
<meta name="theme-color" content="#ffffff">
</head>
<body>
	<div class="banner">
		<center><font color="white" size="16px" face="arial">PrpR</font></center>
	</div>
	<div align="center" class="main-content">
		<div class="spacer">
	</div><br>			<label class="collapse_ACT" for="_1"><font size="5px">PrpR transcriptional activity</font></label>
				<input id="_1" type="checkbox">
				<div class="ACT"><br><ul><li>PrpR speciﬁcally binds 2-MC , as shown by its unresponsiveness to citrate , a closely related structural analogue .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15528672" target="_blank">15528672</a>]<br></li>
<li>In this study , we have shown that the 54-dependent prpBCDE promoter is positively regulated by CRP together with PrpR , which implies direct contact between CRP and E 54 or activation via DNA conformational changes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15805526" target="_blank">15805526</a>]<br></li>
<li>Cellular processes regulated by σ54-dependent bEBPs in S. Typhimurium include nitrogen metabolism in response to limiting nitrogen conditions [ NtrC ( GlnG , NRI ) , Keener and Kustu , 1988 ; Zimmer et al. , 2000 ] , transport and catabolism of D-glucosaminate ( DgaR , Miller et al. , 2013 ) and glucoselysine/fructoselysine ( GfrR , Miller et al. , 2015 ) , regulation of cytoplasmic pH homeostasis during fermentative growth by the formate-hydrogen lyase system ( FhlA , Hopper and Böck , 1995 ; Lamichhane-Khadka et al. , 2015 ) , response to assaults to the cell envelope ( PspF , Karlinsey et al. , 2010 ; Flores-Kim and Darwin , 2015 and zinc-dependent ZraR , Appia-Ayme et al. , 2012 ) , reduction of nitric oxide under anaerobic conditions ( NorR , Hutchings et al. , 2002 ; Mills et al. , 2005 ) , propionate catabolism ( PrpR , Palacios and Escalante-Semerena , 2000 ) , regulation of amino-sugar synthesis by sRNAs ( GlrR ; Gopel et al. , 2011 ) , and RNA repair/processing ( RtcR , Samuels , 2014 ; Engl et al. , 2016 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27583250" target="_blank">27583250</a>]<br></li>
<li>This result adds another layer of complexity to a promoter already known to require two other proteins , IHF and PrpR , for transcriptional activation ( Fig. 1B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15805526" target="_blank">15805526</a>]<br></li>
<li>Regulation of prpBCDE expression might be affected by different concentrations of PrpR , which could explain how high concentrations of PrpR would lead to a decrease in promoter activity , but we do not know how this inhibition might occur .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15805526" target="_blank">15805526</a>]<br></li>
<li>Nevertheless , PprpBCDE activity consistently decreased to near background levels in the presence of glucose or glycerol in cells expressing PrpR from Pzt1 ( Fig. 3A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15805526" target="_blank">15805526</a>]<br></li>
<li>Wild-type PrpR protein , encoded by plasmid pPRP56 in strain JE4871 , failed to activate transcription in the presence or absence of propionate in the culture medium ( Table 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10648513" target="_blank">10648513</a>]<br></li>
<li>Signal Sensing through Ligand Binding In S. Typhimurium there are four bEBPs that are known , or predicted , to be regulated by the binding of an effector-molecule to the regulatory domain : NorR , FhlA , PrpR , and RtcR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27583250" target="_blank">27583250</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_9">
It is proposed that this deleterious effect of PrpRc may be due to an uncontrolled ATPase activity of PrpR or to cross-activation of genes whose functions negatively affect cell growth under the conditions tested .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10648513" target="_blank">10648513</a>]<br></label>
<input id="act_similarity_9", type="checkbox">
<div class="ACT_similar">
<li>We speculate that this deleterious effect of PrpRc may be due to an uncontrolled ATPase activity of PrpR or to cross-activation of genes whose functions negatively affect cell growth under the conditions tested .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10648513" target="_blank">10648513</a>]<br></li></div>
<li>Genetic evidence obtained from previous studies suggested that PrpR likely sensed a small-molecule metabolite of propio-nate , not propionate itself , as the signal for activation of 3Present address : Department of Microbiology , University of Iowa , IA 52246 , USA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15528672" target="_blank">15528672</a>]<br></li>
<li>In vitro 2-MC-dependent activation of the prpB promoter by PrpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15528672" target="_blank">15528672</a>]<br></li>
<li>An alternative scenario is that there is no residual transcription of prpBCDE but the GltA ( citrate synthase ) enzyme , which is known to synthesize 2-methylcitrate from Pro-CoA and oxaloacetate ( 20 ) , provides sufﬁcient 2-methylcitrate for the activation of prpBCDE expression by PrpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12700259" target="_blank">12700259</a>]<br></li>
<li>This metabolite may be the coregulator needed by PrpR to activate transcription of prpBCDE .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9851993" target="_blank">9851993</a>]<br></li>
<li>The regulation of the prp operon is complex due to the transcriptional-regulators , such as PrpR , the 54 σ factor RpoN , and IHF , that are required to activate the transcription of prpBCDE ( Simonte et al. , 2017 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29018430" target="_blank">29018430</a>]<br></li>
<li>If this were correct , it would mean that the level of PrpE enzyme synthesized in the absence of 2-methylcitrate is insufﬁcient to raise the concentration of Pro-CoA enough so that residual PrpC levels can generate 2-methylcitrate to activate prpBCDE operon expression by PrpR ( 25 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12700259" target="_blank">12700259</a>]<br></li>
<li>We hypothesize that PrpC is probably involved in synthesizing the coregulator needed by PrpR to activate transcription of the prpBCDE operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9851993" target="_blank">9851993</a>]<br></li>
<li>We suggest that 2-methylcitrate , the proposed product of the reaction catalyzed by PrpC ( 9 , 32 ) or a derivative of it , is the coregulator needed for PrpR to activate transcription of the operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9851993" target="_blank">9851993</a>]<br></li>
<li>The same effect would be observed if the absence of PrpD had the net effect of reducing the amount of 2-meth-ylcitrate made by PrpC , in which case , the observed reduction of prpBCDE operon expression in prpD mutants would be the result of suboptimal concentrations of the signal needed by PrpR to activate transcription of the operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9851993" target="_blank">9851993</a>]<br></li>
<li>This high concentration of PrpC , we argue , is likely to generate sufﬁcient 2-methylcitrate to make PrpR fully functional in the absence of PrpD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9851993" target="_blank">9851993</a>]<br></li>
<li>The PrpR protein is a sensor of 2-methylcitrate ( 2-MC ) , an intermediate of the 2-methylcitric acid cycle used by this bacterium to convert propionate to pyruvate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15528672" target="_blank">15528672</a>]<br></li>
<li>PrpR was unresponsive to citrate ( a close structural analogue of 2-MC ) and to propionate , suggesting that 2-MC , not propionate , is the metabolite that signals the presence of propionate in the environment to S. enterica .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15528672" target="_blank">15528672</a>]<br></li>
<li>The lack of an effect by 2-MC or ATP/ADP on the ability of PrpR to-bind DNA was surprising because of the known stimulatory effect of the co-activator and ATP on DNA binding by other s - dependent transcriptional-activators 54 ( Austin & Dixon , 1992 ; Lee et al. , 1994 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15528672" target="_blank">15528672</a>]<br></li>
<li>atabolite Repression of the Propionate Catabolic Genes in C Escherichia coli and Salmonella enterica : Evidence for Involvement of the Cyclic AMP Receptor Protein Sung Kuk Lee ,1 Jack D. Newman ,1 and Jay D. Keasling1 ,2 * Department of Chemical Engineering , University of California ,1 and Synthetic Biology Department , Physical Biosciences Division , Lawrence Berkeley National Laboratory ,2 Berkeley , California Received 24 September 2004/Accepted 3 January 2005 Previous studies with Salmonella enterica serovar Typhimurium LT2 demonstrated that transcriptional activation of the prpBCDE operon requires the function of transcription factor PrpR , sigma-54 , and IHF .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15805526" target="_blank">15805526</a>]<br></li>
<li>Microbiology ( 2004 ) , 150 , 3877 -- 3887 DOI 10.1099 / mic.0.27299-0 2-Methylcitrate-dependent activation of the propionate catabolic operon ( prpBCDE ) of Salmonella enterica by the PrpR protein Sergio Palacios3 and Jorge C. Escalante-Semerena Department of Bacteriology , University of Wisconsin , 1710 University Avenue , Madison , WI 53726-4087 , USA Correspondence Jorge C. Escalante-Semerena escalante@bact.wisc.edu The function of the PrpR protein of Salmonella enterica serovar Typhimurium LT2 was studied in vitro and in vivo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15528672" target="_blank">15528672</a>]<br></li>
<li>Increasing amounts of 2-MC in the reaction mixture correlated with increased transcription of PrpR ( Fig. 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15528672" target="_blank">15528672</a>]<br></li>
<li>Residue A162 is critical to 2-MC sensing , since the mutant PrpR protein PrpRA162T was as active as the PrpRc protein in the absence of 2-MC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15528672" target="_blank">15528672</a>]<br></li>
<li>To further conﬁrm the involvement of the cAMP-CRP complex in the activation of the prpBCDE promoter , PprpBCDE-rfp expression was measured in crp - and cya-deﬁcient strains expressing PrpR in trans .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15805526" target="_blank">15805526</a>]<br></li>
<li>Taken together , these results demonstrate that the dramatic decrease in prpBCDE promoter activity is most likely the result of decreased PrpR production in the crp and cya mutant strains in combination with the loss of cAMP-CRP activation of PprpBCDE itself .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15805526" target="_blank">15805526</a>]<br></li>
<li>It is possible that PrpR represents a case in which the binding of 2-MC affects other aspects of the activation process .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15528672" target="_blank">15528672</a>]<br></li>
<li>In fact , under the conditions tested , the ATPase activity in the wild-type PrpR protein was not stimulated by addition of 2-MC over a broad range of concentrations ( 0 ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15528672" target="_blank">15528672</a>]<br></li>
<li>It is clear that the ATPase activity of the constitutive protein PrpR is not stimulated by addi-c tion of DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15528672" target="_blank">15528672</a>]<br></li>
				</div></ul><br>
			<label class="collapse_DOM" for="_2"><font size="5px">PrpR protein domains</font></label>
				<input id="_2" type="checkbox">
				<div class="DOM"><br><ul><li>Fractions containing PrpR protein were dialysed against 10 mM Tris/HCl buffer , pH 7 ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15528672" target="_blank">15528672</a>]<br></li>
<li>( iii ) If one assumes that an ELE is needed for the activation of prpBCDE transcription by PrpR , one must conclude that plasmid pPRP25 contains such an element .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10648513" target="_blank">10648513</a>]<br></li>
<li>PrpR has ATPase activity that is not affected by DNA or 2-MC s - dependent activators typically have an ATPase activity 54 that is essential for transcriptional activation ( Austin & Dixon , 1992 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15528672" target="_blank">15528672</a>]<br></li>
<li>The PrpR protein was excised from the GST N-terminal fusion using thrombin protease .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15528672" target="_blank">15528672</a>]<br></li>
<li>A coiled-coil structure was not detected in several other members of this family of activators that sense smallmolecule effectors , including PrpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15528672" target="_blank">15528672</a>]<br></li>
<li>We propose that PrpR and other members of this family of regulators that sense small-molecule co-activators and lack this coiled-coil motif constitute a subclass which uses a different mechanism for signal propagation within the protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15528672" target="_blank">15528672</a>]<br></li>
<li>The C and D domains of PrpR are 56 % similar and 41 % identical to those of NtrC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10648513" target="_blank">10648513</a>]<br></li>
<li>A more detailed analysis of this and other structural differences is needed , and would provide important information to understand the activation mechanism of PrpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15528672" target="_blank">15528672</a>]<br></li>
<li>When the primary amino acid sequence of PrpR was analysed using the COILS coiled-coil structure algorithm ( Lupas et al. , 1991 ) , no coiled-coil structures were predicted in PrpR ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15528672" target="_blank">15528672</a>]<br></li>
<li>5 100 2 prpR JE4766 JE4593 PrpR ( WT ) PrpR ( DA domain ) c 940 ± 45 62 prpR234 prpR235 prpR236 JE6634 JE6635 180 ± 26 260 ± 26 19 28 12 17 F129L E50K prpR237 prpR238 JE6636 JE6637 190 ± 4 980 ± 15 20 13 65 D37N A162T 104 S33R F49L prpR239 prpR240 JE6638 JE6639 580 ± 22 415 ± 21 62 44 38 28 E172G F49S prpR241 prpR242 JE6672 JE6673 65 ± 4 170 ± 9 7 4 18 11 S33N E142K prpR243 prpR244 JE6674 JE6675 500 ± 4 530 ± 77 53 56 33 35 R24S prpR245 prpR246 JE6865 JE6762 490 ± 10 85 ± 12 52 9 32 6 Y180H L47P S21P prpR247 prpR248 JE6763 JE6764 360 ± 22 90 ± 5 38 24 9 6 R24H G104E prpR249 prpR250 JE6765 JE6766 350 ± 47 70 ± 14 37 23 7 5 K81H F178C prpR251 prpR252 JE6767 JE6768 350 ± 12 570 ± 13 37 23 61 36 38 23 340 ± 34 JE6769 D31G prpR253 PrpR is different from other well-studied members of this family of activators , such as XylR and DmpR , in which the region linking the sensing and catalytic domains contains a coiled-coil structure that is important for controlling binding of the effector molecule and activation of the protein ( Garmendia & de Lorenzo , 2000 ; O'Neill et al. , 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15528672" target="_blank">15528672</a>]<br></li>
<li>Signal Sensing through Ligand Binding In S. Typhimurium there are four bEBPs that are known , or predicted , to be regulated by the binding of an effector molecule to the regulatory domain : NorR , FhlA , PrpR , and RtcR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27583250" target="_blank">27583250</a>]<br></li>
<li>It is possible that ATP-binding , hydrolysis , oligomerization and DNA-binding are unregulated events in PrpR , and 2-MC binding may expose the GAFTGA motif , a highly conserved region in the catalytic domain found to be essential for interaction with RNA polymerase / 54 s and for transduction of the ATP hydrolysis signal ( Lee et al. , 2003 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15528672" target="_blank">15528672</a>]<br></li>
<li>The 2-MC-activated PrpR binds to an enhancer-like element located at a distance 5 ′ to the P , contacts the σ54 prpBdependent RNA polymerase by means of DNA loop formation , and activates transcription of prpBCDE from PprpB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16616438" target="_blank">16616438</a>]<br></li>
<li>PrpR-dependent transcription of lacZ suggested that the ELE predicted to be required for PrpR function must be contained within the fragment cloned in plasmid pPRP25 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10648513" target="_blank">10648513</a>]<br></li>
<li>Excision from the tag leaves two amino acid residues from the GST protein tag in the N-terminus of PrpR ( glycine -- serine ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15528672" target="_blank">15528672</a>]<br></li>
<li>The ATPase speciﬁc activity of the constitutive isoform PrpR did not change when DNA c containing the PrpR binding site was added to the reaction mixture ( speciﬁc activity was 237 , 254 , 257 and 253 % liberated PO4 min at 0 , 0 ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15528672" target="_blank">15528672</a>]<br></li>
<li>This suggested that the A domain had a negative effect on the transcriptional-activity of PrpR , and that binding of the co-activator modulated this effect ( Palacios & Escalante-Semerena , 2000 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15528672" target="_blank">15528672</a>]<br></li>
<li>PrpR , however , contains an alanine -- serine substitution in this motif ( GAFTGS ) that does not affect the transcriptional-activity of the protein ( Palacios & Escalante-Semerena , 2000 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15528672" target="_blank">15528672</a>]<br></li>
<li>However , the A domain of PrpR is not homologous to A domains of other members of this family of proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10648513" target="_blank">10648513</a>]<br></li>
<li>It is possible that PrpR represents a case in which the binding of 2-MC affects other aspects of the activation process .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15528672" target="_blank">15528672</a>]<br></li>
<li>Removal of the putative A domain of PrpR resulted in a co-activator-independent , constitutively active ( PrpRc ) protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10648513" target="_blank">10648513</a>]<br></li>
<li>We note that none of the mutations that led to 2-MC-independent PrpR activity were mapped in the linker region between the A and C domains ( Table 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15528672" target="_blank">15528672</a>]<br></li>
<li>Mutational analysis was performed to gain insights into the structural changes of PrpR required to transduce the effect exerted by the binding of 2-MC to other domains of the PrpR protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15528672" target="_blank">15528672</a>]<br></li>
<li>All lesions causing constitutive PrpR activity were mapped to the N-terminal domain of the protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15528672" target="_blank">15528672</a>]<br></li>
<li>All the causal mutations were mapped to the A domain of PrpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15528672" target="_blank">15528672</a>]<br></li>
<li>The signal-sensing ( A ) domain of PrpR regulates its activity Previous studies showed that the removal of the 209 N-terminal residues which compose the signal-sensing Fig. 4 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15528672" target="_blank">15528672</a>]<br></li>
<li>( ii ) The N-terminal A domain of the PrpR protein is most likely the sensing-domain to which 2-methylcitrate , the proposed coactivator of PrpR , binds .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10648513" target="_blank">10648513</a>]<br></li>
<li>Role of 2-MC The signal-sensing-domain of PrpR has a negative effect on the activity of the protein , since removal of the domain yields a constitutively active protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15528672" target="_blank">15528672</a>]<br></li>
<li>One protein ( PrpR ) showed activity similar to that A162T of the PrpR protein , in which the entire A domain was c missing ( Table 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15528672" target="_blank">15528672</a>]<br></li>
<li>It appears that the highest level of ATPase activity in PrpR can only be attained by removal of the entire sensing-domain ( PrpR ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15528672" target="_blank">15528672</a>]<br></li>
				</div></ul><br>
			<label class="collapse_EVO" for="_3"><font size="5px">PrpR evolution</font></label>
				<input id="_3" type="checkbox">
				<div class="EVO"><br><ul><li>A coiled-coil structure was not detected in several other members of this family of activators that sense smallmolecule effectors , including PrpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15528672" target="_blank">15528672</a>]<br></li>
<li>We propose that PrpR and other members of this family of regulators that sense small-molecule co-activators and lack this coiled-coil motif constitute a subclass which uses a different mechanism for signal propagation within the protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15528672" target="_blank">15528672</a>]<br></li>
<li>One protein ( PrpR ) showed activity similar to that A162T of the PrpR protein , in which the entire A domain was c missing ( Table 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15528672" target="_blank">15528672</a>]<br></li>
<li>The C and D domains of PrpR are 56 % similar and 41 % identical to those of NtrC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10648513" target="_blank">10648513</a>]<br></li>
<li>The PrpR protein had striking homology to 50 members of the sigma-54 ( RpoN ) family of transcriptional activators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9006051" target="_blank">9006051</a>]<br></li>
<li>Homology of PrpR to representative members of the sigma-54 family of transcriptional activators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9006051" target="_blank">9006051</a>]<br></li>
<li>This as yet unidentiﬁed signal is thought to serve as a coregulator of the activity of PrpR , the member of the sigma-54 family of transcriptional activators needed for activation of prpBCDE transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9851993" target="_blank">9851993</a>]<br></li>
<li>PrpR of S. enterica belongs to the NtrC family of transcriptional activators and has been known to regulate the expression of the prpBCDE operon in response to propionate catabolism ( 32 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15805526" target="_blank">15805526</a>]<br></li>
<li>Homology of PrpR to the sigma-54 ( RpoN ) family of transcriptional activators and failure of an RpoN-deﬁcient strain to grow on propionate support the idea that PrpR is likely responsible for the transcriptional activation of the prpBCDE operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9006051" target="_blank">9006051</a>]<br></li>
<li>Sequence homology to genes encoding members of the sigma-54 family of transcriptional activators and the identiﬁcation of a consensus sigma-54 promoter 5 * to the prpBCDE operon suggested that PrpR was required to activate expression of this operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10648513" target="_blank">10648513</a>]<br></li>
<li>However , the A domain of PrpR is not homologous to A domains of other members of this family of proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10648513" target="_blank">10648513</a>]<br></li>
<li>Sequence comparison of PrpR with representatives of the family of RpoN-dependent activators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10648513" target="_blank">10648513</a>]<br></li>
				</div></ul><br>
			<label class="collapse_RP" for="_4"><font size="5px">Processes regulated by PrpR</font></label>
				<input id="_4" type="checkbox">
				<div class="RP"><br><ul><li>These redundant systems of propionyl-CoA synthesis are needed because the prpE gene encoding the propionyl-CoA synthetase enzyme is part of the prpBCDE operon under the control of the PrpR regulatory protein , which needs 2-meth-ylcitrate as a coactivator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12700259" target="_blank">12700259</a>]<br></li>
<li>Discussion We developed a propionate-inducible overexpression vector for E. coli comprising the prpBCDE promoter and prpR , which encodes the 2-MC-activated regulatory protein ( PrpR ) of S. enterica .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16616438" target="_blank">16616438</a>]<br></li>
<li>AtoC , acetoacetate-metabolism regulatory protein from E. coli ( SWISS-PROT accession number Q06065 ) ; NtrC , nitrogen regulatory protein from S. typhimurium ( EMBL accession number X85104 ) ; PrpR , propionate-metabolism regulatory protein from S. typhimurium ( GenBank accession number U51879 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9006051" target="_blank">9006051</a>]<br></li>
<li>DNA topoiomerase I Similar to MarT and ToxR transcriptional regulators Cold shock protein Multidrug-transporter ClpP protease Putative siderophore-binding protein Putative regulatory protein , possibly involved in gluconate utilisation Purine biosynthesis Putative transcriptional regulator Primosomal protein Putative heavy metal transport/resistance protein PhoPQ-activated gene Putative protein Putative outer membrane efﬂux protein STM3521 yeaG astC yjeB STM2746 topA STM0031 cspB acrD clpP yrdA yieP TGGCACGGAGTTAGCC TGGCATCCCCTTTGCG TGGAACGCCTCCTGCT TGGCAAGGTCTATGCA TGGCACTGTAGTGGCA TGGCACCCTGGGTGCA TGGCACGCGCGTCGCC purH ytfJ dnaT ygjT pagK yfcZ STM2690 B Score Gene name Promoter sequence Gene product/function Associated EBP ( Kullback-Liebler distance ) STM4535 STM3772 STM0577 STM2360 TGGCACGCCGCTTGCT TGGCACAACCTTTGCT TGGCACGCCGTTTGCC TGGCATGCCTTTTGCT Putative PTS EII component Putative PTS EII component Putative PTS EII component Putative diaminopimelate decarboxylase Nitrogen regulatory protein Propionate metabolism Hydrogenase accessory protein Resistance to zinc and lead Putative hydrolase Histidine sensor kinase Histidine transport protein Phage shock response Transcriptional repressor of formate hydrogenlyase Putative ﬂavoprotein Histidine sensor kinase , zinc / lead resistance STM4534 STM3773 STM0571 STM2361 2390 glnK prpB TGGCACATCCTTTGCA TGGCATAGCCTTTGCT TGGCATAAATATTGCT TGGCACGGAAGATGCA TGGCACGCCTTTTGAT TGGCACAATTACTGCA TGGCACGATAGTCGCA TGGCACGCAAATTGTA TGGCATGGAAAATGCT NtrC PrpR hypA zraP STM0649 yfhK hisJ pspA hycA FhlA HydG STM0652 YfhA STM2361 PspF FhlA 2383 2354 YgaA HydG STM2840 hydH TGGCACACTAGCTGCA TGGCATGATCTCTGCT YgaA and YfhA Two of the uncharacterised EBP-encoding genes in Salmonella , ygaA and yfhA , have orthologues in E. coli K12 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12125817" target="_blank">12125817</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12581721" target="_blank">12581721</a>]<br></li>
<li>Cellular processes regulated by σ54-dependent bEBPs in S. Typhimurium include nitrogen metabolism in response to limiting nitrogen conditions [ NtrC ( GlnG , NRI ) , Keener and Kustu , 1988 ; Zimmer et al. , 2000 ] , transport and catabolism of D-glucosaminate ( DgaR , Miller et al. , 2013 ) and glucoselysine/fructoselysine ( GfrR , Miller et al. , 2015 ) , regulation of cytoplasmic pH homeostasis during fermentative growth by the formate-hydrogen lyase system ( FhlA , Hopper and Böck , 1995 ; Lamichhane-Khadka et al. , 2015 ) , response to assaults to the cell envelope ( PspF , Karlinsey et al. , 2010 ; Flores-Kim and Darwin , 2015 and zinc-dependent ZraR , Appia-Ayme et al. , 2012 ) , reduction of nitric oxide under anaerobic conditions ( NorR , Hutchings et al. , 2002 ; Mills et al. , 2005 ) , propionate-catabolism ( PrpR , Palacios and Escalante-Semerena , 2000 ) , regulation of amino-sugar synthesis by sRNAs ( GlrR ; Gopel et al. , 2011 ) , and RNA-repair/processing ( RtcR , Samuels , 2014 ; Engl et al. , 2016 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27583250" target="_blank">27583250</a>]<br></li>
<li>Locus tagb Gene symbol Function bEBPc bEBP enhancer sequenced Activating signal/conditione STM0368-71 prpBCDE Propionate catabolism PrpR NtrC CGTTTCATGAAACG TGCACC ( A/T ) 4TGGTGCA 2-methylcitrate STM0462 glnK amtB Regulator of N metabolism ; NH3 transporter Low intracellular glutamine STM0665-62f gltIJKL Glutamate/aspartate-transporter TGCACC ( A/T ) 4TGGTGCA TGCACC ( A/T ) 4TGGTGCA NtrC NtrC Low intracellular glutamine Low intracellular glutamine Glutamine high-afﬁnity transporter STM0830-28 glnHPQ STM1285-84f yeaGH Serine protein kinase TGCACC ( A/T ) 4TGGTGCA TGCACC ( A/T ) 4TGGTGCA NtrC NtrC Low intracellular glutamine Low intracellular glutamine Arginine/ornithine/glutamine matabolism STM1303-07 astCABDE STM2355 argT Lysine/arginine/ornithine-transport protein TGCACC ( A/T ) 4TGGTGCA NtrC Low intracellular glutamine Glutamine synthetase TGCACC ( A/T ) 4TGGTGCA NK STM4007-05 STM0577-72 glnALG NtrC Low intracellular glutamine NK ---------- Putative PTS STM0571 STM0649-51 Putative hydrolase , NK STM0652 NK 2-keto-3-deoxygluconate permease Phage shock proteins STM1690-86 STM4244f pspABCDE PspF PspF TAGTGTAATTCGCTAACT TAGTGTAATTCGCTAACT Cell envelope stress Cell envelope stress Phage shock protein pspG STM2361 NorR ----- ubiX STM2360-56 STM2840-41 Amino acid transport NK NK Nitric oxide reductase norV ygbD GT ( N ) 7AC Nitric oxide Putative hydrogenase maturation proteins STM2843-42 hydN hypF CATTTCGTACGAAATG FhlA Formate STM2853-44 hycABCDEFGHI - Hydrogenase 3 CATTTCGTACGAAATG CATTTCGTACGAAATG FhlA FhlA Formate Formate STM2854-58 hypABCDE Hydrogenase maturation proteins STM3521-18 --------------- STM3568f NK RtcR Nucleic acid damage rsr yrlBA rtcBA Nucleic acid repair/processing Heat shock sigma factor ( σ32 ) NK NK NK NK rpoH STM3772-66 dgaABCDEF D-glucosaminate utilization -- -- -- -- -- -- -- -- -- DgaR D-glucosaminate STM4172 zraP Zinc-dependent chaperone NK NK ZraR ZraR [ Zinc ] and cell envelope stress [ Zinc ] and cell envelope stress STM4173-74 zraSR Zinc-responsive two component system provide insight into this anti-activator mechanism for regulating bEBP activity ( Flores-Kim and Darwin , 2015 ; Osadnik et al. , 2015 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27583250" target="_blank">27583250</a>]<br></li>
				</div></ul><br>
			<label class="collapse_SIT" for="_5"><font size="5px">PrpR binding site</font></label>
				<input id="_5" type="checkbox">
				<div class="SIT"><br><ul><li>The proposed ATG start sites for PrpR and PrpB are boxed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15805526" target="_blank">15805526</a>]<br></li>
<li>PrpR binding sites were deﬁned by a region of strong DNase I hypersensitivity located at the 39 end of the symmetric regions ( Fig. 5a ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15528672" target="_blank">15528672</a>]<br></li>
<li>The function of the divergent prpBCDE promoter was completely destroyed even by deleting prpR promoter sequences ( with PrpR supplied in trans ) ( pAPF7 ) , suggesting that the PrpR-binding site for activation of the prpBCDE promoter overlaps the prpR promoter itself ( Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15805526" target="_blank">15805526</a>]<br></li>
<li>DNA footprinting studies identiﬁed the site in the region between prpR and the prpBCDE operon to which the PrpR protein binds .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15528672" target="_blank">15528672</a>]<br></li>
<li>Results from DNase I protection experiments indicated that the PrpR binding site spanned nucleotides 110 to 145 relative to the PprpBCDE promoter ( Fig. 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15528672" target="_blank">15528672</a>]<br></li>
<li>Recently , it has been reported that PrpR bound to two regions in S. enterica that have the consensus sequence 5 - CGTTTCATGAAACG-3 and span bases 55 to 68 and 77 to 90 from the start codon of the prpR gene , as shown in Fig. 1 ( 33 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15805526" target="_blank">15805526</a>]<br></li>
<li>PrpR bound to two regions which have the consensus sequence 59-CGTTTCATGAAACG-39 and which span from 2110 to 2123 and from 2132 to 2145 bases from the 224 element of the prpB promoter ( Fig. 5b ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15528672" target="_blank">15528672</a>]<br></li>
<li>The 2-MC-activated PrpR binds to an enhancer-like element located at a distance 5 ′ to the P , contacts the σ54 prpBdependent RNA polymerase by means of DNA loop formation , and activates transcription of prpBCDE from PprpB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16616438" target="_blank">16616438</a>]<br></li>
<li>DNA footprinting The sequence of the site to which PrpR binds was determined using a DNase I protection assay .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15528672" target="_blank">15528672</a>]<br></li>
<li>Interactions of PrpR with its DNA binding site The DNA sequence recognized by PrpR is composed of two sites , which were termed proximal ( P ) and distal ( D ) to reﬂect their location relative to the prpB promoter ( Fig. 5b ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15528672" target="_blank">15528672</a>]<br></li>
				</ul></div><br>
			<label class="collapse_TU" for="_6"><font size="5px">Transcription units regulated by PrpR</font></label>
				<input id="_6" type="checkbox">
				<div class="TU"><br><ul><li>To determine whether 2-MC was required for the trans-criptional activation of the prpBCDE genes or not , co-activator-dependent transcription of the prpB promoter by PrpR was tested in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15528672" target="_blank">15528672</a>]<br></li>
<li>These data indicated that PrpR , IHF , and transcription sigma factor RpoN were required for the expression of the prpBCDE operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10648513" target="_blank">10648513</a>]<br></li>
<li>We hypothesize that PrpC is probably involved in synthesizing the coregulator needed by PrpR to activate transcription of the prpBCDE operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9851993" target="_blank">9851993</a>]<br></li>
<li>These redundant systems of propionyl-CoA synthesis are needed because the prpE gene encoding the propionyl-CoA synthetase enzyme is part of the prpBCDE operon under the control of the PrpR regulatory protein , which needs 2-meth-ylcitrate as a coactivator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12700259" target="_blank">12700259</a>]<br></li>
<li>The same effect would be observed if the absence of PrpD had the net effect of reducing the amount of 2-meth-ylcitrate made by PrpC , in which case , the observed reduction of prpBCDE operon expression in prpD mutants would be the result of suboptimal concentrations of the signal needed by PrpR to activate transcription of the operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9851993" target="_blank">9851993</a>]<br></li>
<li>If this were correct , it would mean that the level of PrpE enzyme synthesized in the absence of 2-methylcitrate is insufﬁcient to raise the concentration of Pro-CoA enough so that residual PrpC levels can generate 2-methylcitrate to activate prpBCDE operon expression by PrpR ( 25 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12700259" target="_blank">12700259</a>]<br></li>
<li>Sequence homology to genes encoding members of the sigma-54 family of transcriptional activators and the identiﬁcation of a consensus sigma-54 promoter 5 * to the prpBCDE operon suggested that PrpR was required to activate expression of this operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10648513" target="_blank">10648513</a>]<br></li>
				</div></ul></div><br>
	</div>
</body>
</html>