<!doctype html>
<style>
@charset "utf-8";
body {
	height: 100%;
	margin: 0;
	padding: 0;
}

.banner {
	position: fixed;
	top: 0;
	width: 100%;
	height: 65px;
	background: gray;
	line-height: 65px;
}
.spacer {
	height: 10px;
}

.main-content {
	padding: 5%;
}

.collapse_ACT {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-style: none solid none none;
	border-color: #B4045F;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_ACT + input{
	display: none;
}
.collapse_ACT + input + div{
	display: none;
}
.collapse_ACT + input:checked + div{
	display: block;
}
.ACT {
	text-align: justify;
	text-justify: inter-word;
}
.ACT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B4045F;
}
.ACT_collapse_similarity:hover{
	background: #e5d3dc;
}
.ACT_collapse_similarity + input {
	display: none;
}
.ACT_collapse_similarity + input + div{
	display: none;
}

.ACT_collapse_similarity + input:checked + div{
	display: block;
}
.ACT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.ACT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B4045F;
	padding-left: 5px;
}
.ACT_collapse_similarity:hover{
	background: #e5d3dc;
}
.ACT_collapse_similarity + input {
	display: none;
}
.ACT_collapse_similarity + input + div{
	display: none;
}
.ACT_collapse_similarity + input:checked + div{
	display: block;
}
.ACT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.ACT_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #B4045F;
}
.ACT_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_DOM {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-style: none solid none none;
	border-color: #B404AE;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_DOM + input{
	display: none;
}
.collapse_DOM + input + div{
	display: none;
}
.collapse_DOM + input:checked + div{
	display: block;
}
.DOM {
	text-align: justify;
	text-justify: inter-word;
}
.DOM_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B404AE;
	padding-left: 5px;
}
.DOM_collapse_similarity:hover{
	background: #e8d7e7;
}
.DOM_collapse_similarity + input {
	display: none;
}
.DOM_collapse_similarity + input + div{
	display: none;
}
.DOM_collapse_similarity + input:checked + div{
	display: block;
}
.DOM_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}.DOM_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #B404AE;
}
.DOM_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_EVO {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #7401DF;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_EVO + input{
	display: none;
}
.collapse_EVO + input + div{
	display: none;
}
.collapse_EVO + input:checked + div{
	display: block;
}
.EVO {
	text-align: justify;
	text-justify: inter-word;
}
.EVO_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #7401DF;
	padding-left: 5px;
}
.EVO_collapse_similarity:hover{
	background: #e2d7ed;
}
.EVO_collapse_similarity + input {
	display: none;
}
.EVO_collapse_similarity + input + div{
	display: none;
}
.EVO_collapse_similarity + input:checked + div{
	display: block;
}
.EVO_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.EVO_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #7401DF;
}
.EVO_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_RP {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #0174DF;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_RP + input{
	display: none;
}
.collapse_RP + input + div{
	display: none;
}
.collapse_RP + input:checked + div{
	display: block;
}
.RP {
	text-align: justify;
	text-justify: inter-word;
}
.RP_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #0174DF;
	padding-left: 5px;
}
.RP_collapse_similarity:hover{
	background: #c9d7e2;
}
.RP_collapse_similarity + input {
	display: none;
}
.RP_collapse_similarity + input + div{
	display: none;
}
.RP_collapse_similarity + input:checked + div{
	display: block;
}
.RP_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.RP_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #0174DF;
}
.RP_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_SIT {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #0489B1;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_SIT + input{
	display: none;
}
.collapse_SIT + input + div{
	display: none;
}
.collapse_SIT + input:checked + div{
	display: block;
}
.SIT {
	text-align: justify;
	text-justify: inter-word;
}
.SIT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #0489B1;
	padding-left: 5px;
}
.SIT_collapse_similarity:hover{
	background: #d0dfe2;
}
.SIT_collapse_similarity + input {
	display: none;
}
.SIT_collapse_similarity + input + div{
	display: none;
}
.SIT_collapse_similarity + input:checked + div{
	display: block;
}
.SIT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.SIT_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #0489B1;
}
.SIT_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_TU {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #04B486;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_TU + input{
	display: none;
}
.collapse_TU + input + div{
	display: none;
}
.collapse_TU + input:checked + div{
	display: block;
}
.TU {
	text-align: justify;
	text-justify: inter-word;
}
.TU_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #04B486;
	padding-left: 5px;
}
.TU_collapse_similarity:hover{
	background: #d0e2de;
}
.TU_collapse_similarity + input {
	display: none;
}
.TU_collapse_similarity + input + div{
	display: none;
}
.TU_collapse_similarity + input:checked + div{
	display: block;
}
.TU_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.TU_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #04B486;
}
.TU_similar > a {
	text-decoration: none;
	color: gray;
}

ul { 
	list-style-type: none;
	padding: 0;
	margin: 0;
}
ul li > a {
	text-decoration: none;
	color: gray;
}
ul li:hover, ul li:focus {
	background: #F0F0F0;
}

ul li:first-child span {
  border-left:none;  
}

</style>
<html>
<head>
<meta charset="utf-8">
<title>TF properties - PutA</title>
<meta name="theme-color" content="#ffffff">
</head>
<body>
	<div class="banner">
		<center><font color="white" size="16px" face="arial">PutA</font></center>
	</div>
	<div align="center" class="main-content">
		<div class="spacer">
	</div><br>			<label class="collapse_ACT" for="_1"><font size="5px">PutA transcriptional activity</font></label>
				<input id="_1" type="checkbox">
				<div class="ACT"><br><ul><li>The spontaneous PutAp ' putA : : Mud mutants overexpress PutA independent of catabolite repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3539694" target="_blank">3539694</a>]<br></li>
<li>Chemical reduction of PutA also abolished its DNA-binding activity and increased its relative hydrophobicity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8483946" target="_blank">8483946</a>]<br></li>
<li>Effect of membranes on DNA binding by PutA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9098084" target="_blank">9098084</a>]<br></li>
<li>In addition , PutA superrepressor mutants that exhibit increased repression of the put genes show a direct correlation between decreased membrane binding and increased DNA binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9098084" target="_blank">9098084</a>]<br></li>
<li>The increase in putA enzymatic activity and putP permease activity suggests that these mutations decrease putA protein repression of both the PutA and PutP genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3539694" target="_blank">3539694</a>]<br></li>
<li>These mutations increase the efficiency of PutA promoter activity , but are still induced by proline .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3539694" target="_blank">3539694</a>]<br></li>
<li>To test this prediction we studied the effect of proline on PutA binding in vivo and in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8483946" target="_blank">8483946</a>]<br></li>
<li>These two conditions increase the relative hydrophobicity of PutA protein , favoring membrane association and therefore enzymatic activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8483946" target="_blank">8483946</a>]<br></li>
<li>In addition , Brown and Wood have found that reduction of the FAD cofactor the changes conformation of PutA protein ( E. D. Brown and J. M. Wood , personal communication ) , which may promote membrane association .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8483946" target="_blank">8483946</a>]<br></li>
<li>278 PutA superrepressor mutants are defective in membrane association and reduction of the FAD cofactor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9098084" target="_blank">9098084</a>]<br></li>
<li>Conformational change and membrane association of the PutA protein are coincident with reduction of its FAD cofactor by proline .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9098084" target="_blank">9098084</a>]<br></li>
<li>Repression of the put operon requires PutA to-bind to the put control-region DNA in the cytoplasm .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8483946" target="_blank">8483946</a>]<br></li>
<li>Purified PutA protein bound specifically to several nonoverlapping fragments of control region DNA , indicating the presence of multiple binding sites in the control region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1987118" target="_blank">1987118</a>]<br></li>
<li>Evidence for both of these features was obtained : abnormal electrophoretic behavior of put control region DNA and circular permutation data revealed the presence of curved DNA , and the PutA protein was shown to-bind to multiple sites in the put control region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1987118" target="_blank">1987118</a>]<br></li>
<li>To directly determine whether PutA protein binds to the put control region , we analyzed gel retardation of put control region DNA by purified PutA protein in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1987118" target="_blank">1987118</a>]<br></li>
<li>At concentrations of PutA protein that bind to the put control region , PutA protein did not bind to DNA lacking the put control region , including 4X174 DNA , phage lambda DNA , and pBR322 DNA ( Fig. 2C ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1987118" target="_blank">1987118</a>]<br></li>
<li>These results indicate that the PutA protein does not interact simultaneously with membranes and DNA ; that is , binding of the PutA protein to membranes or DNA is mutually exclusive .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9098084" target="_blank">9098084</a>]<br></li>
<li>To distinguish between these alternatives , we directly assayed the binding of puriﬁed PutA protein to DNA and membranes in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9098084" target="_blank">9098084</a>]<br></li>
<li>To answer this question , we developed in vitro assays to determine whether the binding of PutA protein to membranes and DNA occurred simultaneously or was mutually exclusive .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9098084" target="_blank">9098084</a>]<br></li>
<li>Studies with operon fusions demonstrated Genetics 114 : 687-703 November , 198 that the PutA protein negatively regulates transcription of the put genes ( MA-LOY and ROTH 1983 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3539694" target="_blank">3539694</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_21">
Taken together , these genetic results strongly suggest that the PutA protein negatively regulates transcription of the put operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8483946" target="_blank">8483946</a>]<br></label>
<input id="act_similarity_21", type="checkbox">
<div class="ACT_similar">
<li>Taken together , these results strongly suggest that the PutA protein negatively regulates transcription of the put operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1987118" target="_blank">1987118</a>]<br></li></div>
<li>The results indicate that PutA protein responds to a change in its redox state , and this determines whether PutA protein should remain bound to its operator sites in the cytoplasm or associate with the membrane .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8483946" target="_blank">8483946</a>]<br></li>
<li>This regulatory scheme would allow cells to turn off a catabolic gene when it is not needed ( i.e. , in the absence of inducer ) , but it may also provide a clever way to avoid making a membrane-associated protein when the necessary membrane sites are saturated ( e.g. , PutA protein or other flavin dehydrogenases are already bound to all available sites at the membrane ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8483946" target="_blank">8483946</a>]<br></li>
<li>173 , No. 1 Regulation of Proline Utilization in Salmonella typhimurium : a Membrane-Associated Dehydrogenase Binds DNA In Vitro PAULA OSTROVSKY DE SPICER , KATHRYN O'BRIEN , AND STANLEY MALOY Department of Microbiology , University of Illinois , 407 South Goodwin Avenue , Urbana , Illinois 61801 Received 4 May 1990/Accepted 16 October 1990 * The PutA protein is a membrane-associated enzyme that catalyzes the degradation of proline to glutamate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1987118" target="_blank">1987118</a>]<br></li>
<li>JOURNAL OF BACTERIOLOGY , Apr. 1997 , p. 2788 -- / 97 / $ 04.0010 Copyright q 1997 , American Society for Microbiology Regulation of Gene Expression by Repressor Localization : Biochemical Evidence that Membrane and DNA Binding by the PutA Protein Are Mutually Exclusive ALICIA M. MURO-PASTOR , † PAULA OSTROVSKY , ‡ AND STANLEY MALOY * Department of Microbiology , University of Illinois , Urbana , Illinois 61801 Received 14 August 1995/Accepted 12 February 1997 The PutA protein from Salmonella typhimurium is a bifunctional enzyme that catalyzes the oxidation of proline to glutamate , a reaction that is coupled to the transfer of electrons to the electron transport chain in the cytoplasmic membrane .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9098084" target="_blank">9098084</a>]<br></li>
<li>The distribution of PutA protein was determined by assaying its proline dehydrogenase activity in the presence of an artiﬁcial electron acceptor ( 12 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9098084" target="_blank">9098084</a>]<br></li>
<li>The presence or absence of 0.8 mg of wild-type PutA and 250 ng of E. coli EM41 membranes in the samples is indicated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9098084" target="_blank">9098084</a>]<br></li>
<li>PutA was incubated with membranes and DNA in the presence or absence of proline .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9098084" target="_blank">9098084</a>]<br></li>
<li>Seven and one-half micrograms of puriﬁed PutA was incubated with 270 mg of the membranes in the presence or absence of proline in 0.1 M cacodylic buffer , pH 6.8 , containing 10 mM MgCl .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9098084" target="_blank">9098084</a>]<br></li>
<li>When a sufﬁcient concentration of proline is available , PutA binds proline and functionally associates with the electron transport chain in the cytoplasmic membrane where it is enzymatically active .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9098084" target="_blank">9098084</a>]<br></li>
<li>When a high concentration of proline is present , the PutA protein binds proline , which promotes association with the cytoplasmic membrane , where it is enzymatically active .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3539694" target="_blank">3539694</a>]<br></li>
<li>PutA protein , a membrane-associated flavin dehydrogenase , acts as a redox-dependent transcriptional-regulator ABSTRACT The proline utilization ( put ) operon of Sal-monella typhimurium is transcriptionally repressed by PutA protein in the absence of proline .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8483946" target="_blank">8483946</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_34">
Proline dehydrogenase activity of the transcriptional-repressor PutA is required for induction of the put operon by proline .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9882682" target="_blank">9882682</a>]<br></label>
<input id="act_similarity_34", type="checkbox">
<div class="ACT_similar">
<li>Proline dehydrogenase of the transcriptional-repressor PutA is required for induction of the put operon by proline .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9098084" target="_blank">9098084</a>]<br></li></div>
<li>Therefore , the PutA protein must bind proline , flavin adenine dinucleotide , NAD , the membrane , and DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1987118" target="_blank">1987118</a>]<br></li>
<li>As it becomes associated with the membrane , the cytoplasmic concentration of PutA protein drops below that required for repression , allowing expression of the put genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9098084" target="_blank">9098084</a>]<br></li>
<li>These results indicate that the decreased association of PutA protein with cytoplasmic membranes is directly correlated with increased repression of the put genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9098084" target="_blank">9098084</a>]<br></li>
<li>In addition to its catalytic functions , in the absence of proline PutA acts as a transcriptional-repressor that regulates the expression of both put genes by binding to speciﬁc operator sites in the put regulatory region ( 2 , 15 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9098084" target="_blank">9098084</a>]<br></li>
<li>PutA protein binding to the put operator sites can be demonstrated in vivo ( Fig. 3 ) : ( i ) in aputA + strain the PutA binding sites are sensitive to methylation in the presence of proline but protected from methylation in the absence of proline ; ( ii ) in a putA super-repressor mutant ( S. W. Allen and S.M. , unpublished work ) the PutA binding sites are protected from methylation in the presence or absence of proline ; ( iii ) in a putA ( A+C - ) constitutive mutant ( 7 ) the PutA operator sites are sensitive to methylation in the presence or absence of A B -- proline .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8483946" target="_blank">8483946</a>]<br></li>
<li>In the absence of proline , PutA protein may accumulate in the cytoplasm and bind to multiple operator sites in the put control region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1987118" target="_blank">1987118</a>]<br></li>
<li>These results support a model in which the PutA protein shuttles between the membrane ( where it acts as an enzyme but lacks access to DNA-binding sites ) and the cytoplasm ( where it binds DNA and acts as a transcriptional-repressor ) , depending on the availability of proline .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9098084" target="_blank">9098084</a>]<br></li>
<li>In the absence of proline PutA remains in the cytoplasm where it acts as a repressor , and in the presence of proline PutA binds to the cytoplasmic membrane where it acts as a catabolic enzyme .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9098084" target="_blank">9098084</a>]<br></li>
<li>Thus , this model predicts that the derepression of put gene expression is due to the functional compartmentalization of the PutA protein in the membrane where it lacks access to its operator sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9098084" target="_blank">9098084</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_45">
In the absence of proline , PutA protein remains in the cytoplasm , where it binds to the put operators , preventingput gene expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8483946" target="_blank">8483946</a>]<br></label>
<input id="act_similarity_45", type="checkbox">
<div class="ACT_similar">
<li>In the absence of proline PutA remains in the cytoplasm where it binds to the put operators and represses put gene expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9098084" target="_blank">9098084</a>]<br></li></div>
<li>However , when PutA protein binds proline , it associates with the membrane , where it catalyzes the degradation of proline ; the decrease in cytoplasmic PutA protein frees up the operator sites , allowing put gene expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8483946" target="_blank">8483946</a>]<br></li>
<li>In the presence of proline , the PutA protein becomes membrane associated and the put genes are fully expressed ; however , in the absence of proline the PutA protein may remain in the cytoplasm , where it is able to-bind the operator site between the putA and the putP promoters , precluding expression of both genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1987118" target="_blank">1987118</a>]<br></li>
<li>However , when no exogenous proline is available , the PutA protein accumulates in the cytoplasm , where it binds the DNA and represses transcription of the put genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3539694" target="_blank">3539694</a>]<br></li>
<li>The results indicate that induction of the put operon may be due to a change in the conformation of PutA protein following interaction with proline and an electron acceptor , which reduces the protein , increases its relative hydrophobicity , and thus causes it to preferentially associate with the membrane .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8483946" target="_blank">8483946</a>]<br></li>
<li>Reduction of the FAD cofactor in PutA protein is not the signal for PutA membrane association .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9098084" target="_blank">9098084</a>]<br></li>
<li>Catabolism of proline requires PutA to associate with the membrane because reoxidation of the FAD cofactor in PutA needs the presence of an electron acceptor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8483946" target="_blank">8483946</a>]<br></li>
<li>This led to the suggestion that the reduction of the FAD cofactor promotes membrane association of PutA ( 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9098084" target="_blank">9098084</a>]<br></li>
<li>In the presence of proline , wild-type PutA protein appeared exclusively in the membrane fraction .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9098084" target="_blank">9098084</a>]<br></li>
<li>To determine whether the association of the PutA protein with cytoplasmic membranes prevents it from binding to DNA , we assayed the binding of puriﬁed PutA protein to DNA by gel mobility shift assays ( 4 ) in the presence or absence of membranes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9098084" target="_blank">9098084</a>]<br></li>
<li>In the absence of proline the addition of membranes did not affect DNA binding by the PutA protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9098084" target="_blank">9098084</a>]<br></li>
<li>However , when PutA was bound to DNA in the presence of proline , the addition of membranes caused the dissociation of the PutA protein from DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9098084" target="_blank">9098084</a>]<br></li>
<li>Furthermore , the results obtained for the reaction without DNA indicated that the presence of DNA in the samples did not alter the association of PutA with membranes in the presence of proline .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9098084" target="_blank">9098084</a>]<br></li>
<li>However , since the mutations express putP constitutively as well , it seems likely that they increase the activity of the PutA promoter and simultaneously reduce the binding of the putA repressor to the operator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3539694" target="_blank">3539694</a>]<br></li>
<li>The PutA protein is also a transcriptional-repressor that regulates the expression of the put operon in response to the availability of proline .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9098084" target="_blank">9098084</a>]<br></li>
<li>Genetic evidence suggests that in the absence of proline , the PutA protein also represses transcription of the put4 and putP genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1987118" target="_blank">1987118</a>]<br></li>
<li>Increased expression of the PutA protein may lead to an accumulation of the putA protein , which then represses the putP gene even in the presence of proline .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3539694" target="_blank">3539694</a>]<br></li>
<li>The presence of proline , the inducer , is necessary but not sufficient for PutA to discriminate between its roles as an enzyme or as a repressor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8483946" target="_blank">8483946</a>]<br></li>
<li>The PutA protein made by H3-4 must be able to-bind to the operator site ( s ) and also bind proline since it functions in regulation , but it is defective for proline oxidase activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1987118" target="_blank">1987118</a>]<br></li>
<li>Subsequent lanes contained increasing concentrations of PutA protein : lanes 2 , 6 , and 10 contained 56 nM PutA protein ; lanes 3 , 7 , 11 , and 14 contained 112 nM PutA protein ; and lanes 4 , 8 , 12 , and 15 contained 224 nM PutA protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1987118" target="_blank">1987118</a>]<br></li>
<li>The PutA protein is , in fact , bifunctional , possessing both proline oxidase and repressor activities ( 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=2198247" target="_blank">2198247</a>]<br></li>
				</div></ul><br>
			<label class="collapse_DOM" for="_2"><font size="5px">PutA protein domains</font></label>
				<input id="_2" type="checkbox">
				<div class="DOM"><br><ul><li>These results indicate that PutA protein is a DNA-binding protein specific for the put control region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1987118" target="_blank">1987118</a>]<br></li>
<li>Lanes 1 , 5 , 9 , and 13 did not contain any PutA protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1987118" target="_blank">1987118</a>]<br></li>
<li>Despite extensive genetic and biochemical studies of the PutA protein , it was not known if the PutA protein carries out both of these two opposing functions while membrane associated or if instead it carries them out in different cellular compartments .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9098084" target="_blank">9098084</a>]<br></li>
				</div></ul><br>
			<label class="collapse_RP" for="_4"><font size="5px">Processes regulated by PutA</font></label>
				<input id="_4" type="checkbox">
				<div class="RP"><br><ul><li>PutA , a membrane-associated ﬂavin dehydrogenase , acts as a redox-dependent transcriptional regulator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9098084" target="_blank">9098084</a>]<br></li>
<li>The distributions of PutA protein , put control region DNA , and control DNA ( fX174 ) between soluble and precipitable fractions are shown FIG. 3 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9098084" target="_blank">9098084</a>]<br></li>
<li>PutA protein also carries out the enzymatic steps in proline-catabolism .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8483946" target="_blank">8483946</a>]<br></li>
<li>PutA protein , a membrane-associated flavin dehydrogenase , acts as a redox-dependent transcriptional regulator ABSTRACT The proline utilization ( put ) operon of Sal-monella typhimurium is transcriptionally repressed by PutA protein in the absence of proline .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=8483946" target="_blank">8483946</a>]<br></li>
<li>The phenotypes of several types of putA mutants indicate that the PutA protein is also involved in the regulation of the put operon ( 16 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1987118" target="_blank">1987118</a>]<br></li>
<li>The PutA protein is also a transcriptional repressor that regulates the expression of the put operon in response to the availability of proline .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9098084" target="_blank">9098084</a>]<br></li>
<li>Proteins with statistically signiﬁcantly altered expression after treatment with PFD Group Description Protein Ratio Efﬂux RND family , multidrug efﬂux pump multidrug efﬂux pump membrane fusion protein of AcrAB -- TolC outer membrane efﬂux channel OmpF assembly AcrB EmrA AcrA TolC AsmA 2.8 ** 2.7 ** 1.9 ** 1.8 *** 0.5 * ATP-synthesis membrane-bound ATP synthase , F1 sector , b-subunit membrane-bound ATP synthase , F0 sector , subunit b AtpD AtpF 0.9 * 0.8 * Chemotaxis methyl-accepting chemotaxis protein I TsR TcP 0.4 ** 0.3 *** 0.2 *** methyl-accepting transmembrane citrate/phenol chemoreceptor methyl-accepting chemotaxis protein II Hydrogenase/dehydrogenase pyruvate dehydrogenase , dihydrolipoyltransacetylase component pyruvate dehydrogenase , decarboxylase component putative hydrogenase , membrane component 2-oxoglutarate dehydrogenase plasma membrane proline dehydrogenase/pyrroline -5 - carboxylate dehydrogenase CheM AceF AceE OmpA SucA PutA 2.2 ** 2.0 *** 1.3 * 0.5 * 0.3 ** Miscellaneous RpoD AhpC Fbp RplF PykF OsmE HupA SucC HﬂC YbaY ClpB PckA AnsB GlpQ sigma D ( sigma 70 ) factor of RNA polymerase alkyl hydroperoxide reductase 2.5 * 2.2 ** 1.7 * 1.6 * 1.4 * 0.8 * 0.7 * 0.7 *** 0.6 ** 0.6 * 0.6 * 0.5 * 0.5 * 0.5 * fructose-bisphosphatase 50S ribosomal subunit protein L6 pyruvate kinase I transcriptional activator of ntrl gene DNA-binding protein HU-a succinyl-coa synthetase , b-subunit part of modulator for protease speciﬁc for ftsh phage lambda cii repressor glycoprotein/polysaccharide-metabolism ATP-dependent protease , Hsp 100 phosphoenolpyruvate carboxykinase periplasmic L-asparaginase II glycerophosphodiester phosphodiesterase 0.4 * 0.4 * 0.4 * 0.4 ** 0.4 * 0.4 * ManZ mannose-speciﬁc enzyme IID putative outer membrane lipoprotein STM1607 Gor SspA AspA MipA glutathione oxidoreductase stringent starvation protein A , transcriptional regulator aspartate ammonia-lyase ( aspartase ) scaffolding protein for murein-synthesizing holoenzyme Ratio indicates protein expression after disinfectant treatment divided by expression in untreated control .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17897935" target="_blank">17897935</a>]<br></li>
<li>Supporting our initial observations of phoP repression by bile , we found that tran ¬ ORF no.a Common name Gene regulation STM3146 Hydrogenase 2 maturation endopeptidase hybD 10.36 STM3506 Ferrous iron-transport protein B feoB 7.02 STM3150 Hydrogenase 2 small subunit hypO 4.55 STM1907 Copper homeostasis protein CutC cutC 4.46 STM3511 Putative DNA uptake protein yhgI 3.98 STM3465 Hypothetical protein yhfA 3.61 STM3505 Ferrous iron-transport protein A feoA 3.56 STM1324 Putative cytoplasmic protein 3.53 STM0629 Cold shock protein CspE cspE 3.43 STM3360 Arginine repressor argR 3.17 STM1200 Thymidylate kinase tmk 3.16 STM1121 Putative cytoplasmic protein ymdF 3.15 STM2856 Hydrogenase assembly chaperone hypC 2.88 STM3147 Hydrogenase 2 large subunit hybC 2.87 STM3302 Pseudogene , yhhE yhbE 2.84 STM0974 Formate transporter focA 2.71 STM2532 Putative inner membrane lipoprotein 2.55 STM0441 Cytochrome o ubiquinol oxidase subunit III cyoC 10.71 STM0439 Protoheme IX farnesyltransferase cyoE 7.50 STM3630 Dipeptide-transport protein dppA 6.35 ORF01133 Protein PhoH phoH 5.60 STM0544 Fimbrial protein ﬁmI 5.33 STM0154 Dihydrolipoamide dehydrogenase lpdA 5.14 STM0158 Bifunctional aconitate hydratase 2/2 - acnB 4.79 methylisocitrate dehydratase STM2378 3-Oxoacyl - ( acyl carrier protein ) synthase I fabB 4.58 STM4398 d-Alanine/d-serine/glycine permease cycA 4.56 STM0600 Carbon starvation protein cstA 4.54 STM3218 Putrescine-2-oxoglutarate aminotransferase oat 4.43 STM2389 3-ketoacyl-CoAb thiolase fadI 4.20 PSLT052 Plasmid partition protein A parA 4.18 STM1231 DNA-binding transcriptional regulator PhoP phoP 4.06 STM4159 Thiamine-biosynthesis protein ThiH thiH 4.05 STM3557 Glycerol-3-phosphate-transporter periplasmic ugpB 4.03 binding protein STM0734 Succinate dehydrogenase ﬂavoprotein subunit sdhA 3.98 STM0662 Glutamate/aspartate-transporter gltL 3.89 ORF01131 Bifunctional protein PutA putA 3.84 STM1238 Isocitrate dehydrogenase icdA 3.83 STM1712 Aconitate hydratase acnA 3.48 STM0549 Transcriptional regulator FimZ ﬁmZ 3.45 STM2141 Fructose-bisphosphate aldolase fbaB 3.37 STM0735 Succinate dehydrogenase iron-sulfur subunit sdhB 3.25 STM0738 Succinyl-CoA synthetase subunit beta sucC 3.25 STM1744 Oligopeptide-transport protein oppC 3.21 PSLT046 Putative carbonic anhydrase 3.20 STM2190 Galactose-transport protein mglB 3.02 STM2799 DNA binding protein , nucleoid associated stpA 2.97 STM1292 Putative cytoplasmic protein yeaC 2.94 STM0730 Type II citrate synthase gltA 2.90 STM3321 Putative 54 modulation protein yhbH 2.84 STM1742 Oligopeptide-transport protein oppF 2.78 STM3033 Plasmid maintenance protein 2.72 STM4274 Putative inner membrane protein yjcH 2.65 STM4126 Soluble pyridine nucleotide transhydrogenase udhA 2.63 ORF04918 Virulence protein VsdF 2.59 a STM and pSLT designations refer to Salmonella enterica serovar Typhimurium LT2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22366421" target="_blank">22366421</a>]<br></li>
				</div></ul><br>
			<label class="collapse_SIT" for="_5"><font size="5px">PutA binding site</font></label>
				<input id="_5" type="checkbox">
				<div class="SIT"><br><ul><li>Furthermore , PutA protein also bound to these fragments after digestion with 213 PutA PROTEIN BINDS DNA IN VITRO A 8 9 10 11 12 13 14 15 16 17 18 19 20 2 22 23 24 1 2 3 4 5 6 7 ii ' .4 ... t.t. - .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1987118" target="_blank">1987118</a>]<br></li>
<li>sequence showed dyad symmetry , consistent with the fact that PutA protein forms dimers ; and ( iii ) the position of this sequence suggested a simple single operator model in which binding of PutA protein to this site could preclude transcription from both diverging promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1987118" target="_blank">1987118</a>]<br></li>
<li>In addition , PutA protein did not bind to a fragment from within the put control region which lacks putative operator sites even at molar ratios of PutA protein to DNA of 800:1 ( Fig. 2C , WT5 ,8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1987118" target="_blank">1987118</a>]<br></li>
<li>PutA protein binds to multiple sites in the put control region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1987118" target="_blank">1987118</a>]<br></li>
				</ul></div><br>
			<label class="collapse_TU" for="_6"><font size="5px">Transcription units regulated by PutA</font></label>
				<input id="_6" type="checkbox">
				<div class="TU"><br><ul><li>The effect of each mutation on transcription in both directions was determined by measuring lac2 expression from PutA and putP operon fusions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3539694" target="_blank">3539694</a>]<br></li>
<li>MALOYand ROTH ( 1983 ) demonstrated that the PutA and PutP genes are transcribed divergently .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3539694" target="_blank">3539694</a>]<br></li>
<li>Genetic evidence suggests that in the absence of proline , the PutA protein also represses transcription of the put4 and putP genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1987118" target="_blank">1987118</a>]<br></li>
<li>Genetic evidence suggests that putA protein autogenously represses transcription of the PutA and putP genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3539694" target="_blank">3539694</a>]<br></li>
<li>These results suggest that the reduction of FAD by proline is not the inducing signal that determines the subcellular localization of PutA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9098084" target="_blank">9098084</a>]<br></li>
<li>Under inducing conditions , proline oxidation would reduce the tightly associated flavin adenine dinucleotide coenzyme , inducing the PutA protein to become membrane associated and abandon the operator sites , thus permitting expression of the put genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1987118" target="_blank">1987118</a>]<br></li>
<li>In the absence of proline PutA remains in the cytoplasm where it binds to the put operators and represses put gene expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=9098084" target="_blank">9098084</a>]<br></li>
<li>In the presence of proline , the PutA protein becomes membrane associated and the put genes are fully expressed ; however , in the absence of proline the PutA protein may remain in the cytoplasm , where it is able to bind the operator site between the putA and the putP promoters , precluding expression of both genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=1987118" target="_blank">1987118</a>]<br></li>
<li>Increased expression of the PutA protein may lead to an accumulation of the putA protein , which then represses the putP gene even in the presence of proline .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=3539694" target="_blank">3539694</a>]<br></li>
				</div></ul></div><br>
	</div>
</body>
</html>