<!doctype html>
<style>
@charset "utf-8";
body {
	height: 100%;
	margin: 0;
	padding: 0;
}

.banner {
	position: fixed;
	top: 0;
	width: 100%;
	height: 65px;
	background: gray;
	line-height: 65px;
}
.spacer {
	height: 10px;
}

.main-content {
	padding: 5%;
}

.collapse_ACT {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-style: none solid none none;
	border-color: #B4045F;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_ACT + input{
	display: none;
}
.collapse_ACT + input + div{
	display: none;
}
.collapse_ACT + input:checked + div{
	display: block;
}
.ACT {
	text-align: justify;
	text-justify: inter-word;
}
.ACT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B4045F;
}
.ACT_collapse_similarity:hover{
	background: #e5d3dc;
}
.ACT_collapse_similarity + input {
	display: none;
}
.ACT_collapse_similarity + input + div{
	display: none;
}

.ACT_collapse_similarity + input:checked + div{
	display: block;
}
.ACT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.ACT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B4045F;
	padding-left: 5px;
}
.ACT_collapse_similarity:hover{
	background: #e5d3dc;
}
.ACT_collapse_similarity + input {
	display: none;
}
.ACT_collapse_similarity + input + div{
	display: none;
}
.ACT_collapse_similarity + input:checked + div{
	display: block;
}
.ACT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.ACT_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #B4045F;
}
.ACT_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_DOM {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-style: none solid none none;
	border-color: #B404AE;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_DOM + input{
	display: none;
}
.collapse_DOM + input + div{
	display: none;
}
.collapse_DOM + input:checked + div{
	display: block;
}
.DOM {
	text-align: justify;
	text-justify: inter-word;
}
.DOM_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B404AE;
	padding-left: 5px;
}
.DOM_collapse_similarity:hover{
	background: #e8d7e7;
}
.DOM_collapse_similarity + input {
	display: none;
}
.DOM_collapse_similarity + input + div{
	display: none;
}
.DOM_collapse_similarity + input:checked + div{
	display: block;
}
.DOM_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}.DOM_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #B404AE;
}
.DOM_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_EVO {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #7401DF;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_EVO + input{
	display: none;
}
.collapse_EVO + input + div{
	display: none;
}
.collapse_EVO + input:checked + div{
	display: block;
}
.EVO {
	text-align: justify;
	text-justify: inter-word;
}
.EVO_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #7401DF;
	padding-left: 5px;
}
.EVO_collapse_similarity:hover{
	background: #e2d7ed;
}
.EVO_collapse_similarity + input {
	display: none;
}
.EVO_collapse_similarity + input + div{
	display: none;
}
.EVO_collapse_similarity + input:checked + div{
	display: block;
}
.EVO_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.EVO_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #7401DF;
}
.EVO_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_RP {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #0174DF;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_RP + input{
	display: none;
}
.collapse_RP + input + div{
	display: none;
}
.collapse_RP + input:checked + div{
	display: block;
}
.RP {
	text-align: justify;
	text-justify: inter-word;
}
.RP_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #0174DF;
	padding-left: 5px;
}
.RP_collapse_similarity:hover{
	background: #c9d7e2;
}
.RP_collapse_similarity + input {
	display: none;
}
.RP_collapse_similarity + input + div{
	display: none;
}
.RP_collapse_similarity + input:checked + div{
	display: block;
}
.RP_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.RP_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #0174DF;
}
.RP_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_SIT {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #0489B1;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_SIT + input{
	display: none;
}
.collapse_SIT + input + div{
	display: none;
}
.collapse_SIT + input:checked + div{
	display: block;
}
.SIT {
	text-align: justify;
	text-justify: inter-word;
}
.SIT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #0489B1;
	padding-left: 5px;
}
.SIT_collapse_similarity:hover{
	background: #d0dfe2;
}
.SIT_collapse_similarity + input {
	display: none;
}
.SIT_collapse_similarity + input + div{
	display: none;
}
.SIT_collapse_similarity + input:checked + div{
	display: block;
}
.SIT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.SIT_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #0489B1;
}
.SIT_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_TU {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #04B486;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_TU + input{
	display: none;
}
.collapse_TU + input + div{
	display: none;
}
.collapse_TU + input:checked + div{
	display: block;
}
.TU {
	text-align: justify;
	text-justify: inter-word;
}
.TU_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #04B486;
	padding-left: 5px;
}
.TU_collapse_similarity:hover{
	background: #d0e2de;
}
.TU_collapse_similarity + input {
	display: none;
}
.TU_collapse_similarity + input + div{
	display: none;
}
.TU_collapse_similarity + input:checked + div{
	display: block;
}
.TU_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.TU_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #04B486;
}
.TU_similar > a {
	text-decoration: none;
	color: gray;
}

ul { 
	list-style-type: none;
	padding: 0;
	margin: 0;
}
ul li > a {
	text-decoration: none;
	color: gray;
}
ul li:hover, ul li:focus {
	background: #F0F0F0;
}

ul li:first-child span {
  border-left:none;  
}

</style>
<html>
<head>
<meta charset="utf-8">
<title>TF properties - RhaR</title>
<meta name="theme-color" content="#ffffff">
</head>
<body>
	<div class="banner">
		<center><font color="white" size="16px" face="arial">RhaR</font></center>
	</div>
	<div align="center" class="main-content">
		<div class="spacer">
	</div><br>			<label class="collapse_ACT" for="_1"><font size="5px">RhaR transcriptional activity</font></label>
				<input id="_1" type="checkbox">
				<div class="ACT"><br><ul><li>These results conﬁrm that the rha and rhm operons , that are predicted to be controlled by RhaR and RhmR transcription factors , respectively , are transcriptionally induced by L-Rha .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24391637" target="_blank">24391637</a>]<br></li>
<li>Expression and Puriﬁcation of His-Tagged Lrp , Lrp Derivative Mutants , RhaR , , NrdF , FliT , Pat , and CobB For puriﬁcation of Lrp and its derivative mutants , pET22b-lrp , pET22b-lrp K36Q , pET22b-lrp K36R , and pET22b-lrp K25Q were constructed and veriﬁed by sequencing .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27909434" target="_blank">27909434</a>]<br></li>
<li>After Pat treatment at 37 ◦ C for 3 h , the acetylation levels of ( Figure 2A ) , RhaR ( Figure 2B ) , NrdF ( Figure 2C ) , and Lrp ( Figure 3A ) increased 1.24 , 1.77 , 1.56 , and 1.89-fold , respectively , indicating that NrdF , , RhaR and Lrp can be acetylated by Pat in vitro .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27909434" target="_blank">27909434</a>]<br></li>
<li>This may help to explain some of the mutagenic potency and mutational spectra of MX ; however , it does not Gene Nitric oxide synthesis and detoxiﬁcation Carnitine metabolism Molybdenum metabolism Cold shock Indirect activator of L-rhamnose operon Integration host factor Sulfur assimilation via cysteine biosynthesis Response to DNA damage Cyclic AMP receptor protein growth regulation Global anaerobic regulator Regulates response to the redox conditions of growth BA-1 NsrR 2.0E-02 3.4E-02 4.4E-02 4.4E-02 CaiF ModE CspA RhaR 4.3E-02 5.6E-09 BA-4 IHF MX CysB 1.1E-07 6.3E-06 LexA CRP FNR ArcA a of the consequences of these alterations .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19598237" target="_blank">19598237</a>]<br></li>
<li>Rhamnose catabolism in Bacteroides thetaiotaomicron is controlled by the positive transcriptional-regulator RhaR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24391637" target="_blank">24391637</a>]<br></li>
<li>Notably , four proteins including , RhaR , FliT and Lrp are transcriptional-regulators , suggesting that acetylation is an important regulation mechanism for the activity of regulators in bacteria ( Thao et al. , 2010 ; Hu et al. , 2013 ; Ren et al. , 2016 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27909434" target="_blank">27909434</a>]<br></li>
<li>Also , in E. coli CRP requires the presence of RhaR to efficiently activate rhaSR in vivo in response to Lrhamnose ( Wickstrum et al. , 2005 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23393149" target="_blank">23393149</a>]<br></li>
				</div></ul><br>
			<label class="collapse_DOM" for="_2"><font size="5px">RhaR protein domains</font></label>
				<input id="_2" type="checkbox">
				<div class="DOM"><br><ul><li>The predicted binding motifs of DeoR-family RhaR regulators in four distinct taxonomic groups are characterized by unique sequences ; however , each of them has a similar structure that includes two imperfect direct repeats with a periodicity of 10 -- 11 bp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24391637" target="_blank">24391637</a>]<br></li>
<li>Complementary DNA fragments , containing the predicted 38-bp RhaR binding site from the promoter region and ﬂanked on each side by ﬁve guanosine residues ( Table S2 in Supplementary Materials ) were synthesized by Integrated DNA Technologies .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24391637" target="_blank">24391637</a>]<br></li>
<li>The interaction of the predicted RhaR regulator with the ( rhaR ) upstream DNA fragment containing candidate RhaR-binding sites in C. aurantiacus , and the inﬂuence of potential sugar effectors on protein-DNA interaction were assessed in vitro by EMSA ( Figure 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24391637" target="_blank">24391637</a>]<br></li>
<li>The addition of D-glucose and L-Rha had no effect on RhaR-DNA interaction , whereas L-rhamnulose abolished the speciﬁc DNA-binding ability of RhaR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24391637" target="_blank">24391637</a>]<br></li>
				</div></ul><br>
			<label class="collapse_EVO" for="_3"><font size="5px">RhaR evolution</font></label>
				<input id="_3" type="checkbox">
				<div class="EVO"><br><ul><li>The L-Rha catabolic gene cluster in Bacteroides thetaiotaomicron is positively controlled by another AraC-family TF , which is non-orthologous to E. coli RhaR ( 16 % identity ) ( Patel et al. , 2008 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24391637" target="_blank">24391637</a>]<br></li>
<li>The DeoR family includes at least four non-orthologous groups of RhaR regulators that are characterized by distinct DNA motifs with a tandem repeat symmetry .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24391637" target="_blank">24391637</a>]<br></li>
				</div></ul><br>
			<label class="collapse_RP" for="_4"><font size="5px">Processes regulated by RhaR</font></label>
				<input id="_4" type="checkbox">
				<div class="RP"><br><ul><li>In contrast to bacteria , yeast has a common environmental stress response o TABLE V. CMR DNA Repair Genes Altered Transcriptionally by BA-1 Gene Fold symbol change Hypergeometric P-value Gene title DNA ligase 1.4E-06 7.6E-05 8.0E-05 0.002 0.003 0.004 0.005 0.005 0.006 0.006 0.006 0.006 0.010 0.012 0.015 0.015 0.018 0.018 0.023 0.024 0.033 0.033 0.034 0.035 0.036 0.040 0.043 0.043 0.047 0.050 6.38 lig Putative nuclease subunit of the excinuclease complex Adenine DNA glycosylase DNA gyrase , subunit A , type II topoisomerase Putative recombination protein , gap repair Fels-2 prophage : similar to retron in E. coli Uracil-DNA-glycosylase Putative replicative DNA helicase UvrC , which is part of the UvrABC repair enzyme DNA polymerase III , delta prime subunit ATP-dependent DNA helicase DNA polymerase III , chi subunit UvrB , which is part of the UvrABC repair enzyme Replication termination protein Putative DNA helicase Regulator O-6-alkylguanine-DNA / cysteine-protein methyltransferase Putative DNA-repair protein Recombinase , site-speciﬁc Deoxyribodipyrimidine photolyase ( photoreactivation ) Primosomal protein I Putative helicase Primosomal replication protein N Endonuclease III ; DNA glycosylase/apyrimidinic ( AP ) lyase DNA polymerase III , alpha subunit Resembles eukaryotic-type potassium channels Putative ATP-dependent protease , DNA-repair protein Methyl-directed mismatch-repair Putative superfamily I DNA and RNA helicase Putative hydrolase N-terminus 24.55 4.00 22.55 22.50 22.16 27.46 23.12 22.53 22.82 22.27 4.27 5.41 2.28 22.07 21.97 22.50 22.43 25.87 21.88 1.80 22.50 2.88 STM1309 mutY gyrA recR STM2730 ung dnaB uvrC holB recQ holC uvrB tus yoaA oraA ogt radC xerD phrB dnaT ygcB priC nth dnaE 1.72 22.84 22.11 22.46 22.66 1.80 25.05 STM4372 radA mutS STM2767 STM0649 STM2727 2.7E-05 14.8 0.10 1.5 0.13 1.6 dinP 3.4E-05 4.8 0.11 21.6 0.68 21.1 sulA 4E-05 12.6 0.34 1.5 0.70 1.2 umuC 8.0E-06 8.6 0.12 1.8 0.81 21.1 umuD 2 4.4E-05 14.3 0.61 1.2 0.59 21.2 STM1019 4.8E-05 8.9 0.58 21.2 0.35 1.4 dinI 2 0.0002 16.9 0.42 1.3 0.46 1.3 yebG 2 0.0002 13.5 0.20 2.3 0.94 21.1 lexA 0.0003 4.0 0.05 22.3 0.73 1.1 STM1309 0.0010 2.5 0.00 24.5 0.05 21.8 polB 0.0010 2.8 0.46 1.2 0.88 21.0 yigN 0.0012 3.0 0.22 2.2 0.16 21.6 uvrA 0.0013 2.5 0.27 1.3 0.70 1.1 recA 0.0022 7.8 0.95 1.0 0.86 21.1 uvrD 0.0024 2.2 0.46 21.3 0.45 21.4 ruvA 0.0037 2.2 0.47 21.2 0.64 1.2 sbmC 0.0061 7.4 0.21 21.4 0.99 1.0 ydjM 0.0063 2.2 0.08 21.6 0.91 1.0 recN 0.027 16.0 0.23 21.5 0.15 1.7 STM0925 0.9730 1.0 0.55 1.3 0.75 1.1 ybfE 0.8064 0.9 0.11 21.6 0.16 2.1 uvrB 0.4355 1.3 0.01 5.4 0.08 3.1 dinG 0.4182 1.2 0.70 21.1 0.26 21.7 ftsK 0.1837 1.4 0.49 21.9 0.90 21.2 STM1056 0.2094 1.5 0.21 1.5 0.31 21.4 mfd 0.7057 0.9 0.15 1.4 0.50 1.2 minC 0.4689 0.9 0.69 1.1 0.52 1.3 yehR 0.0580 9.6 0.32 21.4 0.37 1.4 ftsY 0.7095 1.1 0.70 1.1 0.91 1.0 recG 0.9149 1.0 0.66 1.4 0.62 22.4 2.0E-02 3.4E-02 4.4E-02 4.4E-02 CaiF ModE CspA RhaR 4.3E-02 5.6E-09 BA-4 IHF MX CysB 1.1E-07 6.3E-06 LexA CRP FNR ArcA a The P-value is from a Bayesian t-test measuring differential expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19598237" target="_blank">19598237</a>]<br></li>
<li>This may help to explain some of the mutagenic potency and mutational spectra of MX ; however , it does not Gene Nitric oxide synthesis and detoxiﬁcation Carnitine metabolism Molybdenum metabolism Cold shock Indirect activator of L-rhamnose operon Integration host factor Sulfur assimilation via cysteine-biosynthesis Response to DNA damage Cyclic AMP receptor protein growth regulation Global anaerobic regulator Regulates response to the redox conditions of growth BA-1 NsrR 2.0E-02 3.4E-02 4.4E-02 4.4E-02 CaiF ModE CspA RhaR 4.3E-02 5.6E-09 BA-4 IHF MX CysB 1.1E-07 6.3E-06 LexA CRP FNR ArcA a of the consequences of these alterations .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19598237" target="_blank">19598237</a>]<br></li>
<li>In the Chloroﬂexales , two unique TFs control LRha metabolism -- the DeoR-family regulator RhaR controls the FIGURE 2  Genomic context of L-rhamnose catabolic genes and regulons in bacteria from seven diverse taxonomic lineages .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24391637" target="_blank">24391637</a>]<br></li>
<li>In the Bacillales , RhaR and RhgR control genes involved in the utilization of L-rhamnose and rhamnogalacturonan , respectively ( Leyn et al. , 2013 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24391637" target="_blank">24391637</a>]<br></li>
<li>In the Thermotogales , the DeoR-family RhaR regulator co-regulates genes involved in the utilization of L-Rha mono - and oligosaccharides ( Rodionov et al. , 2013 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24391637" target="_blank">24391637</a>]<br></li>
<li>In the Actinomycetales , a novel LacI-type regulator ( termed RhaR ) co-regulates genes involved in the uptake and catabolism of L-Rha and L-Rha-containing oligosaccharides .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24391637" target="_blank">24391637</a>]<br></li>
				</div></ul><br>
			<label class="collapse_SIT" for="_5"><font size="5px">RhaR binding site</font></label>
				<input id="_5" type="checkbox">
				<div class="SIT"><br><ul><li>The predicted binding motifs of DeoR-family RhaR regulators in four distinct taxonomic groups are characterized by unique sequences ; however , each of them has a similar structure that includes two imperfect direct repeats with a periodicity of 10 -- 11 bp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24391637" target="_blank">24391637</a>]<br></li>
<li>The synthetic 38-bp DNA region containing a tandem repeat of four individual RhaR sites ( a consensus sequence TCGAAA ) was incubated with increasing concentrations of the puriﬁed recombinant RhaR protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24391637" target="_blank">24391637</a>]<br></li>
<li>( A ) Conservation of predicted RhaR binding sites ( boxed ) identiﬁed in the promoter regions of rha operons in the C. aurantiacus J-10-ﬂ ( Caur ) , C. sp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24391637" target="_blank">24391637</a>]<br></li>
<li>RhaR activates the rhaSR genes via binding to the inverted-repeat of two 17 bp half sites separated by a 17 bp spacer .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24391637" target="_blank">24391637</a>]<br></li>
<li>RhaS activates the rhaBAD and rhaT genes via binding to another inverted-repeat of two sites whose sequence differs from the RhaR consensus binding site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24391637" target="_blank">24391637</a>]<br></li>
				</ul></div><br>
			<label class="collapse_TU" for="_6"><font size="5px">Transcription unit regulated by RhaR</font></label>
				<input id="_6" type="checkbox">
				<div class="TU"><br><ul><li>Expression and Puriﬁcation of His-Tagged Lrp , Lrp Derivative Mutants , RhaR , , NrdF , FliT , Pat , and CobB For puriﬁcation of Lrp and its derivative mutants , pET22b-lrp , pET22b-lrp K36Q , pET22b-lrp K36R , and pET22b-lrp K25Q were constructed and veriﬁed by sequencing .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27909434" target="_blank">27909434</a>]<br></li>
<li>Interestingly , genes encoding L-lactate dehydrogenases ( lldD , lldEFG ) belong to the reconstructed RhaR regulons in certain genomes of the Actinomycetales and Rhodobacterales that encode RhaEW .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24391637" target="_blank">24391637</a>]<br></li>
<li>In C. aurantiacus , the L-Rha utilization genes are organized into a nine-gene rha operon , which is predicted to be transcriptionally controlled by a novel DeoR-family regulator RhaR ( Figure 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24391637" target="_blank">24391637</a>]<br></li>
<li>These results conﬁrm that the rha and rhm operons , that are predicted to be controlled by RhaR and RhmR transcription factors , respectively , are transcriptionally induced by L-Rha .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24391637" target="_blank">24391637</a>]<br></li>
				</div></ul></div><br>
	</div>
</body>
</html>