<!doctype html>
<style>
@charset "utf-8";
body {
	height: 100%;
	margin: 0;
	padding: 0;
}

.banner {
	position: fixed;
	top: 0;
	width: 100%;
	height: 65px;
	background: gray;
	line-height: 65px;
}
.spacer {
	height: 10px;
}

.main-content {
	padding: 5%;
}

.collapse_ACT {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-style: none solid none none;
	border-color: #B4045F;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_ACT + input{
	display: none;
}
.collapse_ACT + input + div{
	display: none;
}
.collapse_ACT + input:checked + div{
	display: block;
}
.ACT {
	text-align: justify;
	text-justify: inter-word;
}
.ACT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B4045F;
}
.ACT_collapse_similarity:hover{
	background: #e5d3dc;
}
.ACT_collapse_similarity + input {
	display: none;
}
.ACT_collapse_similarity + input + div{
	display: none;
}

.ACT_collapse_similarity + input:checked + div{
	display: block;
}
.ACT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.ACT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B4045F;
	padding-left: 5px;
}
.ACT_collapse_similarity:hover{
	background: #e5d3dc;
}
.ACT_collapse_similarity + input {
	display: none;
}
.ACT_collapse_similarity + input + div{
	display: none;
}
.ACT_collapse_similarity + input:checked + div{
	display: block;
}
.ACT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.ACT_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #B4045F;
}
.ACT_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_DOM {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-style: none solid none none;
	border-color: #B404AE;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_DOM + input{
	display: none;
}
.collapse_DOM + input + div{
	display: none;
}
.collapse_DOM + input:checked + div{
	display: block;
}
.DOM {
	text-align: justify;
	text-justify: inter-word;
}
.DOM_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B404AE;
	padding-left: 5px;
}
.DOM_collapse_similarity:hover{
	background: #e8d7e7;
}
.DOM_collapse_similarity + input {
	display: none;
}
.DOM_collapse_similarity + input + div{
	display: none;
}
.DOM_collapse_similarity + input:checked + div{
	display: block;
}
.DOM_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}.DOM_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #B404AE;
}
.DOM_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_EVO {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #7401DF;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_EVO + input{
	display: none;
}
.collapse_EVO + input + div{
	display: none;
}
.collapse_EVO + input:checked + div{
	display: block;
}
.EVO {
	text-align: justify;
	text-justify: inter-word;
}
.EVO_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #7401DF;
	padding-left: 5px;
}
.EVO_collapse_similarity:hover{
	background: #e2d7ed;
}
.EVO_collapse_similarity + input {
	display: none;
}
.EVO_collapse_similarity + input + div{
	display: none;
}
.EVO_collapse_similarity + input:checked + div{
	display: block;
}
.EVO_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.EVO_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #7401DF;
}
.EVO_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_RP {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #0174DF;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_RP + input{
	display: none;
}
.collapse_RP + input + div{
	display: none;
}
.collapse_RP + input:checked + div{
	display: block;
}
.RP {
	text-align: justify;
	text-justify: inter-word;
}
.RP_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #0174DF;
	padding-left: 5px;
}
.RP_collapse_similarity:hover{
	background: #c9d7e2;
}
.RP_collapse_similarity + input {
	display: none;
}
.RP_collapse_similarity + input + div{
	display: none;
}
.RP_collapse_similarity + input:checked + div{
	display: block;
}
.RP_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.RP_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #0174DF;
}
.RP_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_SIT {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #0489B1;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_SIT + input{
	display: none;
}
.collapse_SIT + input + div{
	display: none;
}
.collapse_SIT + input:checked + div{
	display: block;
}
.SIT {
	text-align: justify;
	text-justify: inter-word;
}
.SIT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #0489B1;
	padding-left: 5px;
}
.SIT_collapse_similarity:hover{
	background: #d0dfe2;
}
.SIT_collapse_similarity + input {
	display: none;
}
.SIT_collapse_similarity + input + div{
	display: none;
}
.SIT_collapse_similarity + input:checked + div{
	display: block;
}
.SIT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.SIT_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #0489B1;
}
.SIT_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_TU {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #04B486;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_TU + input{
	display: none;
}
.collapse_TU + input + div{
	display: none;
}
.collapse_TU + input:checked + div{
	display: block;
}
.TU {
	text-align: justify;
	text-justify: inter-word;
}
.TU_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #04B486;
	padding-left: 5px;
}
.TU_collapse_similarity:hover{
	background: #d0e2de;
}
.TU_collapse_similarity + input {
	display: none;
}
.TU_collapse_similarity + input + div{
	display: none;
}
.TU_collapse_similarity + input:checked + div{
	display: block;
}
.TU_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.TU_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #04B486;
}
.TU_similar > a {
	text-decoration: none;
	color: gray;
}

ul { 
	list-style-type: none;
	padding: 0;
	margin: 0;
}
ul li > a {
	text-decoration: none;
	color: gray;
}
ul li:hover, ul li:focus {
	background: #F0F0F0;
}

ul li:first-child span {
  border-left:none;  
}

</style>
<html>
<head>
<meta charset="utf-8">
<title>TF properties - SsrB</title>
<meta name="theme-color" content="#ffffff">
</head>
<body>
	<div class="banner">
		<center><font color="white" size="16px" face="arial">SsrB</font></center>
	</div>
	<div align="center" class="main-content">
		<div class="spacer">
	</div><br>			<label class="collapse_ACT" for="_1"><font size="5px">SsrB transcriptional activity</font></label>
				<input id="_1" type="checkbox">
				<div class="ACT"><br><ul><li>Molecular Microbiology ( 2000 ) 36 ( 3 ) , 749 ± 761 Salmonella SsrB activates a global regulon of horizontally a quired ge c nes Micah J. Worley , * Katherine H. L. Ching and Fred Heffron Department of Microbiology , Oregon Health Sciences University , 3181 SW Sam Jackson Park Rd. , Portland , OR 97201 , USA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10844662" target="_blank">10844662</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_2">
Worley , M.J. , Ching , K.H.L. , and Heffron , F. ( 2000 ) Salmo-nella SsrB activates a global regulon of horizontally acquired genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12864810" target="_blank">12864810</a>]<br></label>
<input id="act_similarity_2", type="checkbox">
<div class="ACT_similar">
<li>Worley , M.J. , Ching , K.H. , and Heffron , F. ( 2000 ) Salmonella SsrB activates a global regulon of horizontally acquired genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12675800" target="_blank">12675800</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15491370" target="_blank">15491370</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12864852" target="_blank">12864852</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630976" target="_blank">17630976</a>]<br></li>
<li>Salmonella SsrB activates a global regulon of horizontally acquired genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17220229" target="_blank">17220229</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17526845" target="_blank">17526845</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24496791" target="_blank">24496791</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19717598" target="_blank">19717598</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12949164" target="_blank">12949164</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19737903" target="_blank">19737903</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17616593" target="_blank">17616593</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15790293" target="_blank">15790293</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23504014" target="_blank">23504014</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16513753" target="_blank">16513753</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11844786" target="_blank">11844786</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19011032" target="_blank">19011032</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26673132" target="_blank">26673132</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11133955" target="_blank">11133955</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19858298" target="_blank">19858298</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15231802" target="_blank">15231802</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23630960" target="_blank">23630960</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17635868" target="_blank">17635868</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16980468" target="_blank">16980468</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17060472" target="_blank">17060472</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15136033" target="_blank">15136033</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20974834" target="_blank">20974834</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26497459" target="_blank">26497459</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15567133" target="_blank">15567133</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16129704" target="_blank">16129704</a>]<br></li></div>
				<label class="ACT_collapse_similarity" for="act_similarity_5">
[ 23 ] Worley MJ , Ching KH , Heffron F. Salmonella SsrB activates a global regulon of horizontally acquired genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21320585" target="_blank">21320585</a>]<br></label>
<input id="act_similarity_5", type="checkbox">
<div class="ACT_similar">
<li>Worley MJ , Ching KH , Heffron F. Salmonella SsrB activates a global regulon of horizontally acquired genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11886586" target="_blank">11886586</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21984608" target="_blank">21984608</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535274" target="_blank">12535274</a>]<br></li></div>
<li>Choi J , Shin D , Yoon H , Kim J , Lee C , Kim M , Seok Y , Ryu S : Salmonella pathogenicity island 2 expression negatively controlled by EIIANtr-SsrB interaction is required for Salmonella virulence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23295139" target="_blank">23295139</a>]<br></li>
<li>Salmonella pathogenicity island 2 expression negatively controlled by EIIANtr-SsrB interaction is required for Salmonella virulence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23006851" target="_blank">23006851</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24643535" target="_blank">24643535</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27628932" target="_blank">27628932</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24835993" target="_blank">24835993</a>]<br></li>
<li>x The response-regulator SsrB activates transcription and binds to a region overlapping OmpR binding sites at Xiuhong Feng , † Don Walthers , Ricardo Oropeza ‡ and Linda J. Kenney * Department of Microbiology and Immunology , University of Illinois at Chicago , 835 S. Wolcott Avenue , M/C 790 , Chicago , IL 60612 , USA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15491370" target="_blank">15491370</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_10">
Feng , X. , Walthers , D. , Oropeza , R. , and Kenney , L.J. ( 2004 ) The response-regulator SsrB activates transcription and binds to a region overlapping OmpR binding sites at Sal-monella pathogenicity island 2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16999831" target="_blank">16999831</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20545866" target="_blank">20545866</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630976" target="_blank">17630976</a>]<br></label>
<input id="act_similarity_10", type="checkbox">
<div class="ACT_similar">
<li>Feng , X. , Walthers , D. , Oropeza , R. , and Kenney , L.J. ( 2004 ) The response-regulator SsrB activates transcription and binds to a region overlapping OmpR binding sites at Salmonella pathogenicity island 2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17784910" target="_blank">17784910</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28631437" target="_blank">28631437</a>]<br></li>
<li>Feng , X. , Walthers , D. , Oropeza , R. , and Kenney , L. J. ( 2004 ) The response-regulator SsrB activates transcription and binds to a region overlapping OmpR binding sites at Salmonella pathogenicity island 2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22418438" target="_blank">22418438</a>]<br></li>
<li>Feng X , Walthers D , Oropeza R , Kenney LJ ( 2004 ) The response-regulator SsrB activates transcription and binds to a region overlapping OmpR binding sites at Salmonella pathogenicity island 2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21059960" target="_blank">21059960</a>]<br></li>
<li>The response-regulator SsrB activates transcription and binds to a region overlapping OmpR binding sites at Salmonella pathogenicity island 2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20807201" target="_blank">20807201</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19737903" target="_blank">19737903</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17158898" target="_blank">17158898</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15972530" target="_blank">15972530</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20202329" target="_blank">20202329</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24643535" target="_blank">24643535</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20656781" target="_blank">20656781</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25135218" target="_blank">25135218</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21518393" target="_blank">21518393</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16129704" target="_blank">16129704</a>]<br></li>
<li>The response regulator SsrB activates transcription and binds to a region overlapping OmpR binding sites at Salmonella pathogenicity island 2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23554419" target="_blank">23554419</a>]<br></li></div>
<li>Discussion The response-regulator SsrB can be phosphorylated by small molecule phosphodonors such as acetyl phosphate and phosphoramidate ( Fig. 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15491370" target="_blank">15491370</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_17">
Walthers , D. , Li , Y. , Liu , Y. , Anand , G. , Yan , J. , and Kenney , L.J. ( 2011 ) Salmonella enterica response-regulator SsrB relieves H-NS silencing by displacing H-NS bound in polymerization mode and directly activates transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22804842" target="_blank">22804842</a>]<br></label>
<input id="act_similarity_17", type="checkbox">
<div class="ACT_similar">
<li>Walthers D , Li Y , Liu Y , Anand G , Yan J , Kenney LJ : Salmonella enterica response-regulator SsrB relieves H-NS silencing by displacing H-NS bound in polymerization mode and directly activates transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23295139" target="_blank">23295139</a>]<br></li>
<li>The Salmonella enterica response-regulator SsrB relieves H-NS silencing by displacing H-NS bound in polymerization mode and directly activates transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21518393" target="_blank">21518393</a>]<br></li>
<li>Salmonella enterica response-regulator SsrB relieves H-NS silencing by displacing H-NS bound in polymerization mode and directly activates transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29034217" target="_blank">29034217</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21680637" target="_blank">21680637</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25713562" target="_blank">25713562</a>]<br></li></div>
<li>x The respo e regulator SsrB activates expression diverse onella pathogenicity island 2 promoters a y the n ociated ns Salm unters H-NS of nd co protein silencing b ucleoid-ass Don Walthers ,1 Ronan K. Carroll ,1 William Wiley Navarre ,2 Stephen J. Libby ,2 Ferric C. Fang2 ,3,4 and Linda J. Kenney1 * 1University of Illinois at Chicago , Department of Microbiology and Immunology , 835 S. Wolcott Ave M/C 790 , Chicago , IL 60612 , USA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630976" target="_blank">17630976</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_22">
The response-regulator SsrB activates expression of diverse Salmo-nella pathogenicity island 2 promoters and counters silencing by the nucleoid-associated protein H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23554419" target="_blank">23554419</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24496791" target="_blank">24496791</a>]<br></label>
<input id="act_similarity_22", type="checkbox">
<div class="ACT_similar">
<li>The response-regulator SsrB activates expression of diverse Salmo-nella pathogenicity island 2 promoters and counters silencing by the nu-cleoid-associated protein H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24643535" target="_blank">24643535</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25085489" target="_blank">25085489</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26497459" target="_blank">26497459</a>]<br></li></div>
<li>[ 45 ] D. Walthers , R.K. Carroll , W.W. Navarre , S.J. Libby , F.C. Fang , L.J. Kenney , The response-regulator SsrB activates expression of diverse Salmonella patho-genicity island 2 promoters and counters silencing by the nucleoid-associated protein H-NS , Mol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23040276" target="_blank">23040276</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_25">
The response-regulator SsrB activates expression of diverse Salmonella pathogenecity island 2 promoters and counters silencing by the nucleoid-associated protein H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20396961" target="_blank">20396961</a>]<br></label>
<input id="act_similarity_25", type="checkbox">
<div class="ACT_similar">
<li>The response-regulator SsrB activates expression of diverse Salmonella pathogenicity island 2 promoters and counters silencing by the nucleoid-associated protein H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29057095" target="_blank">29057095</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20807201" target="_blank">20807201</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26441883" target="_blank">26441883</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29034217" target="_blank">29034217</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17981960" target="_blank">17981960</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19858298" target="_blank">19858298</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28270274" target="_blank">28270274</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24835993" target="_blank">24835993</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18039769" target="_blank">18039769</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25713562" target="_blank">25713562</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21570144" target="_blank">21570144</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21518393" target="_blank">21518393</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18160484" target="_blank">18160484</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21680637" target="_blank">21680637</a>]<br></li>
<li>Walthers , D. , Carroll , R. K. , Navarre , W. W. , Libby , S. J. , Fang , F. C. , and Kenney , L. J. ( 2007 ) The response-regulator SsrB activates expression of diverse Salmonella pathogenicity island 2 promoters and counters silencing by the nucleoid-associated protein H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22418438" target="_blank">22418438</a>]<br></li>
<li>Walthers D , et al. ( 2007 ) The response-regulator SsrB activates expression of diverse Salmonella pathogenicity island 2 promoters and counters silencing by the nucleoid-associated protein H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21059960" target="_blank">21059960</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23690578" target="_blank">23690578</a>]<br></li>
<li>Walthers D , Carroll RK , Navarre WW , Libby SJ , Fang FC , et al. ( 2007 ) The response-regulator SsrB activates expression of diverse Salmonella pathogenicity island 2 promoters and counters silencing by the nucleoid-associated protein H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19229334" target="_blank">19229334</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28192515" target="_blank">28192515</a>]<br></li></div>
<li>Western blots were performed to determine SsrB protein but it also binds to DNA as a heterodimer with RcsA ( 20 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19126546" target="_blank">19126546</a>]<br></li>
<li>Real-time PCR showed a ﬁvefold increase in sciS transcription in an ssrB mutant when compared to wild type at 5 h postinfection , conﬁrming that SsrB negatively regulates sciS ( Fig. 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15972528" target="_blank">15972528</a>]<br></li>
<li>Here , it is shown that SsrB in fact activates a global regulon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10844662" target="_blank">10844662</a>]<br></li>
<li>The results shown in Fig. 4 suggested that PhoP activates srfJ through SsrB , whereas RcsB acts independently .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22661691" target="_blank">22661691</a>]<br></li>
<li>Phosphorylated SsrB activates transcription of target genes ( Deiwick et al. , 1999 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29270156" target="_blank">29270156</a>]<br></li>
<li>Until recently , SsrB was assumed to activate genes exclusively within SPI-2 ( Miao et al. , 1999 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10844662" target="_blank">10844662</a>]<br></li>
<li>This result suggests that , under these conditions , the SsrB regulon , including , SPI2 , would not be expressed and therefore SrfJ , although expressed from PiolE , would not be secreted through T3SS2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29270156" target="_blank">29270156</a>]<br></li>
<li>Because this strain does not carry the ssrB allele , fusions that continued to be expressed differentially on M9-arabinose versus M9-glucose could not be SsrB regulated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10844662" target="_blank">10844662</a>]<br></li>
<li>The response-regulator SsrB , in turn , is autoregulated , activating ssrB , ssrA and spiC ( Feng et al. , 2003 ) , as well as other genes located both inside and outside of © 2004 Blackwell Publishing Lt spiC ssrA ssrB srfH , etc Asp-P SsrB Fig. 1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15491370" target="_blank">15491370</a>]<br></li>
<li>SsrB , in turn , activates ssrB , ssrA and spiC ( ssaB ) as well as additional genes , including srfH .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15491370" target="_blank">15491370</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15491370" target="_blank">15491370</a>]<br></li>
<li>A role for SsrB ~ P was identified by its dual function as a direct transcriptional-activator and as an anti-silencer of H-NS-mediated repression ( Walthers et al. , 2007 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26880544" target="_blank">26880544</a>]<br></li>
<li>Phosphorylation of SsrB by acetyl phosphate and phosphoramidate SsrB protein was phosphorylated using 25 mM acetyl phosphate ( Sigma ) or 25 mM phosphoramidate as described for OmpR ( Kenney et al. , 1995 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15491370" target="_blank">15491370</a>]<br></li>
<li>In Salmonella , H-NS silencing of SPI-2 is antagonized by HilD or by SsrB proteins in response to different environmental conditions ( Walthers et al. , 2007 , 2011 ; Bustamante et al. , 2008 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21680637" target="_blank">21680637</a>]<br></li>
<li>It has been reported that SsrB counters the silencing of H-NS on SPI-2 genes by displacing H-NS bound in its polymerization mode , and subsequently activates SPI-2 transcription ( Walthers et al. , 2007 , 2011 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25713562" target="_blank">25713562</a>]<br></li>
<li>Interestingly , in the absence of H-NS , SsrB is less required for SPI-2 gene expression , and relatively recent work has demonstrated that SsrB can directly displace H-NS polymers along DNA ( Walthers et al. , 2011 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29034217" target="_blank">29034217</a>]<br></li>
<li>Autophosphorylation of the histidine kinase domain initiates a phosphorelay that activates a cognate response-regulator for targeted gene expression , which for the SsrA sensor , is SsrB in S. Typhimurium ( Cirillo et al. , 1998 ; Deiwick et al. , 1999 ; Carroll et al. , 2009 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25442048" target="_blank">25442048</a>]<br></li>
<li>Because the anti-silencing style of gene regulation is indirect and does not rely on specific DNA interactions , searching for SsrB binding sites has not been informative in uncovering this type of regulation ( Tomljenovic-Berube et al. , 2010 ; Worley et al. , 2000 ; Shea et al. , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26880544" target="_blank">26880544</a>]<br></li>
<li>It is possible that SsrB can be phosphorylated from other sources as noted by Walthers et al. [ 52 ] or that the overexpression results in dimerization and self activation similar to what has been observed for PhoP [ 78 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19229334" target="_blank">19229334</a>]<br></li>
<li>Response to external acidiﬁcation is integrated through SsrA Activation of SsrB by SsrA is low under alkaline conditions and high in acidic conditions ( Deiwick et al. , 1999 ; Miao et al. , 2002 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25442048" target="_blank">25442048</a>]<br></li>
<li>Transcription of ssrB is uncoupled from that of ssrA ( Feng et al. , 2003 ) ; therefore , it is likely that SsrB is expressed in the ssrA mutant and it may still be activated by factors other than SsrA , as has been described , for example , for SlyA , PhoP and OmpR ( Bijlsma & Groisman , 2005 ; Feng et al. , 2004 ; Yoon et al. , 2009 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20656781" target="_blank">20656781</a>]<br></li>
<li>Our previous work showed that nitrosative stress represses SsrB - and PhoP-dependent SPI2 transcription in Salmonella ( Bourret et al. , 2009 ; McCollister et al. , 2005 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23217033" target="_blank">23217033</a>]<br></li>
<li>At SPI-2 , SsrB ~ P de-represses H-NS and also activates transcription ( Walthers et al. , 2007 ; 2011 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26880544" target="_blank">26880544</a>]<br></li>
<li>DOI : 10.7554 / eLife .10747.013 SsrB-dependent biofilm formation does not require SPI-2 SsrB is encoded on SPI-2 and regulates SPI-2 genes involved in assembly and function of the type three secretion system encoded by SPI-2 , as well as effectors that are encoded on SPI-2 and outside of SPI-2 ( Feng et al. , 2004 ; Walthers et al. , 2007 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26880544" target="_blank">26880544</a>]<br></li>
<li>For example , SPI-2-encoded SsrB downregulates SPI-1 genes by repressing HilA and HilD ( Pérez-Morales et al. , 2017 ) , and reciprocally , SPI-1-encoded HilD upregulates SPI-2 gene expression by directly binding the ssrAB operon ( Bustamante et al. , 2008 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29034217" target="_blank">29034217</a>]<br></li>
<li>Inside professional phagocytic cells , SPI-2 expression is induced by the two-component systems PhoP/PhoQ , OmpR/EnvZ , and SsrA/SsrB ( Lee et al. , 2000 ; Garmendia et al. , 2003 ; Bijlsma and Groisman , 2005 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25713562" target="_blank">25713562</a>]<br></li>
<li>For example , OmpR and SsrB co-regulate expression of sseI ( srfH ) and ssaB ( Worley et al. , 2000 ; Feng et al. , 2004 ; D. Walthers and L.J. Kenney , unpublished ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630976" target="_blank">17630976</a>]<br></li>
<li>SsrB can directly bind to genes within SPI-2 , and outside of this island , and activate their transcription ( Worley et al. , 2000 ; Walthers et al. , 2007 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29034217" target="_blank">29034217</a>]<br></li>
<li>SsrB sits at a pivotal decision point that determines Salmonella lifestyles When the SsrA kinase is present and activated by acid stress , SsrB is phosphorylated and SsrB ~ P de-represses H-NS and activates transcription at SPI-2 and SPI-2 co-regulated genes , including : sifA ( Walthers et al. , 2011 ) , ssaB , ssaM , sseA and ssaG ( Walthers et al. , 2007 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26880544" target="_blank">26880544</a>]<br></li>
<li>SsrB is autoregulated and activates expression of SPI-2 and SPI-2 co-regulated genes in response to an unknown signal ( s ) ( Feng et al. , 2003 ; 2004 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630976" target="_blank">17630976</a>]<br></li>
<li>Binding occured at low protein concentrations , suggesting high affinity interactions and showed that unphosphorylated SsrB was capable of binding csgD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26880544" target="_blank">26880544</a>]<br></li>
<li>An SsrB mutant , K179A , that was incapable of binding DNA ( Carroll et al. , 2009 ) did not bind csgD even at 300 nM , indicting that binding was specific ( Figure 5E and F ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26880544" target="_blank">26880544</a>]<br></li>
<li>The two-component system of PhoP-PhoQ contrib-utes to post-transcriptional regulation to SsrA-SsrB ( Fass and Groisman , 2009 ; Yoon et al. , 2009 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25442048" target="_blank">25442048</a>]<br></li>
<li>Thus , OmpR appears to function as an activator of ssrA , as reported previously ( Lee et al. , 2000 ) , and SsrB also appears to activate ssrA , although the mechanism of regulation remains to be determined .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12753201" target="_blank">12753201</a>]<br></li>
<li>The DNase I protection footprinting and primer extension assays indicate that SsrB activates transcription by directly binding to DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630976" target="_blank">17630976</a>]<br></li>
<li>SsrB activates transcription of multiple operons within SPI-2 by binding to degenerate DNA targets at diversely organized promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630976" target="_blank">17630976</a>]<br></li>
<li>The isolated C terminus of SsrB ( SsrBC ) binds to DNA and activates transcription ( 15 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21059643" target="_blank">21059643</a>]<br></li>
<li>OmpR ~ P goes on to activate expression of SsrA/B , and SsrB ~ P activates SPI-2 transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25875623" target="_blank">25875623</a>]<br></li>
<li>In turn , an ssaV : : lac transcriptional fusion was expressed under SPI-2-in-ducing conditions , and its expression was dependent on PhoP and SsrB ( Fig. 3B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17616593" target="_blank">17616593</a>]<br></li>
<li>1134 X. Feng , R. Oropeza and L. J. Kenney Fig. 2 suggest that OmpR activates ssrA and , to a lesser extent , SsrB also activates ssrA , whereas both OmpR and SsrB have a signiﬁcant effect on ssrB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12753201" target="_blank">12753201</a>]<br></li>
<li>ssrB srfH , etc Asp-P SsrB SsrB and OmpR activate srfH Fig. 1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15491370" target="_blank">15491370</a>]<br></li>
<li>OmpR and SsrB are hypothesized to be phosphorylated ( as indicated by `` P '' ) in this model .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10633113" target="_blank">10633113</a>]<br></li>
<li>The SsrB protein increases its own levels via positive autoregulation ( 4 ) and the OmpR and PhoP proteins also do so by directly activating ssrB transcription ( 9 , 10 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21059960" target="_blank">21059960</a>]<br></li>
<li>To determine whether regulation by SsrB was direct , we used DNase I protection footprinting to examine the interaction between SsrB and promoter DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21059643" target="_blank">21059643</a>]<br></li>
<li>Wild-type SsrB and SsrB D56A were used as the positive and negative control , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19126546" target="_blank">19126546</a>]<br></li>
<li>e-mail : jt233@mole.bio.cam.ac.uk 32 Garcia-del Portillo , F. and Finlay , B. B. , Targeting of Salmonella 33 Deiwick , J. , Nikolaus , T. , Erdogan , S. and Hensel , M. , 34 Hess , J. , Ladel , C. , Miko , D. and Kaufmann , S. H. , Salmonella 35 Worley , M. J. , Ching , K. H. L. and Heffron , F. , Salmonella SsrB 36 Beuzon , C. R. , Unsworth , K. E. and Holden , D. W. , In vivo 37 Hewitt , E. W. , The MHC class I antigen presentation pathway : strategies for viral immune evasion .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15307188" target="_blank">15307188</a>]<br></li>
<li>Other regulators can activate and/or repress SPI-2 gene expression by acting through SsrA and/or SsrB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29751061" target="_blank">29751061</a>]<br></li>
<li>In contrast , the SPI-2 T3SS and effector gene promoters , ssaB-E , sseA-G , ssaG-L , and ssaM-R , are only very slightly induced by altered DNA topology and the presumed increase in SsrA and SsrB concentrations brought about by novobiocin treatment ( Figure S1A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22457642" target="_blank">22457642</a>]<br></li>
<li>Our results confirm and extend the previously reported regulation of TTSS1 and TTSS2 effectors by InvFSicA and SsrB respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17233907" target="_blank">17233907</a>]<br></li>
<li>DISCUSSION We previously showed that SrfJ can be secreted through the T3SS2 of S. Typhimurium and that its expression is positively regulated by PhoP and SsrB , and negatively regulated by RcsB and IolR ( Cordero-Alba et al. , 2012 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29270156" target="_blank">29270156</a>]<br></li>
<li>The other six genes STM1329 , STM1330 , STM1600 , STM1854 , STM2585 and yneC ( STM4079S ) show a pattern of expression that resembles the SPI2-encoded and SPI2-associated genes , that are positively regulated by both HilD and SsrB under SPI1 and SPI2-inducing conditions , respectively , and are upregulated within macrophages [ 14 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27564394" target="_blank">27564394</a>]<br></li>
<li>Blackwell Science , LtdOxford , UKMMIMolecular Microbiology1365-2958Blackwell Publishing Ltd , 200348411311143Original ArticleSsrA and SsrB regulation are uncoupledX.Feng , R.Oropeza and L.J.Kenney Molecular Microbiology ( 2003 ) 48 ( 4 ) , 1131 -- 1143 Dual regulation by phospho-OmpR of ssrA/B gene expression in Salmonella pathogenicity island 2 Xiuhong Feng , Ricardo Oropeza † and Linda J. Kenney * Department of Molecular Microbiology and Immunology , L-220 , Oregon Health and Science University , 3181 SW Sam Jackson Park Road , Portland , OR 97239-3098 , USA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12753201" target="_blank">12753201</a>]<br></li>
<li>ORIGINAL RESEARCH ARTICLE published : 10 February 2015 doi : 10.3389 / fmicb .2015.00027 involvement of SsrB and H-NS in SPI-2 gene expression Jie Li1 ‡ , Christopher C. Overall 2 ‡ , Ernesto S. Nakayasu 2 † , Afshan S. Kidwai1 , Marcus B. Jones 3 , Rudd C. Johnson1 , Nhu T. Nguyen1 , Jason E. McDermott 2 , Charles Ansong2 , Fred Heffron 1 , Eric D. Cambronne 1 and Joshua N. Adkins2 * 1 Department of Molecular Microbiology and Immunology , Oregon Health and Science University , Portland , OR , USA 2 Biological Sciences Division , Paciﬁc Northwest National Laboratory , Richland , WA , USA 3 Department of Infectious Diseases , J. Craig Venter Institute , Rockville , MD , USA The extracytoplasmic functioning sigma factor σE is known to play an essential role for Salmonella enterica serovar Typhimurium to survive and proliferate in macrophages and mice .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25713562" target="_blank">25713562</a>]<br></li>
<li>Consistently , both mutants failed to activate mTOR suggesting that mTOR activation and attenuation of autophagy are SsrB and SsaV dependent ( Fig 6K and 6L and S6J and S6K Fig ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28192515" target="_blank">28192515</a>]<br></li>
<li>Here we describe a fundamental mechanism that activates both SPI-2 and SPI-1 promoters through changes in DNA topology , and this mechanism depends on OmpR and FIS but is independent of SsrB and HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22457642" target="_blank">22457642</a>]<br></li>
<li>Both effectors are substrates of T3SS1 and T3SS2 , and in both cases there is a direct regulation by PhoP and a minor regulatory role for SsrB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25182488" target="_blank">25182488</a>]<br></li>
<li>Relief of H-NS silencing by SsrB The nucleoid structuring protein H-NS represses horizontally acquired genes of SPI-2 , and we conﬁrmed a role for H-NS at speciﬁc SPI-2 loci , including ssaB , sseA , ssaG and ssaM .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630976" target="_blank">17630976</a>]<br></li>
<li>SsrB binds upstream of the sifA , sifB , and sseJ effector genes and directly regulates transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21059643" target="_blank">21059643</a>]<br></li>
<li>In this study , we demonstrate that SsrB directly stimulates transcription of sifA , sifB , and sseJ by binding upstream of their respective promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21059643" target="_blank">21059643</a>]<br></li>
<li>Thus , SsrB regulates transcription of sifA , sifB , and sseJ by both direct activation and relief of H-NS repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21059643" target="_blank">21059643</a>]<br></li>
<li>Positive regulation by SsrB is a common feature of SPI2 genes and other genes encoding effectors speciﬁcally secreted through T3SS2 ( Xu and Hensel , 2010 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29270156" target="_blank">29270156</a>]<br></li>
<li>Reaction scheme for regulation of ssrA/B-spiC by OmpR and SsrB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15491370" target="_blank">15491370</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15491370" target="_blank">15491370</a>]<br></li>
<li>SsrB binds and bends DNA , resulting in highly curved DNA conformations .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26880544" target="_blank">26880544</a>]<br></li>
<li>There is evidence suggesting that SsrB can be autoregulated and can activate SsrA expression , albeit to a lesser extent ( 237 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23554419" target="_blank">23554419</a>]<br></li>
<li>Although SsrB is a major virulence regulator in Sal-monella , very little is known regarding how it binds DNA and activates transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19126546" target="_blank">19126546</a>]<br></li>
<li>SsrB is a response-regulator that , when phosphorylated , binds to several promoters in SPI2 , and SsrA is its cognate histidine kinase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23504014" target="_blank">23504014</a>]<br></li>
<li>The rRNA methyltransferase gene , rsmC , was used as an internal control , and the SPI-12 gene sspH2 was used as a positive control , as it is regulated by SsrB ( 21 , 32 , 33 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23630960" target="_blank">23630960</a>]<br></li>
<li>[ PubMed : 18198179 ] Husain M , Jones-Carson J , Song M , McCollister BD , Bourret TJ , Vazquez-Torres A. Redox sensor SsrB Cys203 enhances Salmonella fitness against nitric oxide generated in the host immune response to oral infection .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23217033" target="_blank">23217033</a>]<br></li>
<li>Redox sensor SsrB Cys203 enhances Salmonella ﬁtness against nitric oxide generated in the host immune response to oral infection .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26386070" target="_blank">26386070</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24166960" target="_blank">24166960</a>]<br></li>
<li>The requirement for phosphorylation for activation was also observed with the response-regulator SsrB ( see Fig. 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15491370" target="_blank">15491370</a>]<br></li>
<li>RESEARCH ARTICLE The horizontally-acquired response regulator SsrB drives a Salmonella lifestyle switch by relieving biofilm silencing Stuti K Desai1 , Ricksen S Winardhi1 ,2 , Saravanan Periasamy3 , Michal M Dykas4 ,5 , Yan Jie1 ,2 , Linda J Kenney1 ,6,7 * 1Mechanobiology Institute , National University of Singapore , Singapore , Singapore ; 2Department of Physics , National University of Singapore , Singapore , Singapore ; 3Singapore Centre on Environmental Life Sciences Engineering , Nanyang Technological University , Singapore , Singapore ; 4Nanoscience and Nanotechnology Institute , National University of Singapore , Singapore , Singapore ; 5Graduate School for Integrative Sciences and Engineering , National University of Singapore , Singapore , Singapore ; 6Jesse Brown Veterans Affairs Medical Center , University of Illinois-Chicago , Chicago , United States ; 7Department of Microbiology and Immunology , University of Illinois-Chicago , Chicago , United States Abstract A common strategy by which bacterial pathogens reside in humans is by shifting from a virulent lifestyle , ( systemic infection ) , to a dormant carrier state .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26880544" target="_blank">26880544</a>]<br></li>
<li>The ability of unphosphorylated SsrB to regulate biofilm formation was unique , as it was unable to activate sifA-lacZ levels .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26880544" target="_blank">26880544</a>]<br></li>
<li>In contrast , when SsrB was exposed to immobilized EIIAGlc , an enzyme II of the carbohydrate PTS , no interaction was detected between these two proteins ( Fig. 3B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21059960" target="_blank">21059960</a>]<br></li>
<li>However , it appears that this promoter is only active under specific growth conditions since other workers found no evidence of SsrB binding or functional promoter activity directly upstream of ssrB [ 30,63 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24664308" target="_blank">24664308</a>]<br></li>
<li>Therefore , this result indicates that Fur repression of SPI-2 expression is dependent on SsrB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24643535" target="_blank">24643535</a>]<br></li>
<li>Interestingly , this network showed that SlyA is directly upstream of SsrB , but that it also seemed to be directly regulating SPI-2 expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21687430" target="_blank">21687430</a>]<br></li>
<li>Therefore , it makes sense that their expression is not strictly dependent on SsrB since they need to be expressed under conditions that prevent activation of SsrB and SPI2 expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23504014" target="_blank">23504014</a>]<br></li>
<li>SsrB binds upstream of csgD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26880544" target="_blank">26880544</a>]<br></li>
<li>Ntr Fourth , EIIA prevented binding of SsrB to its target promoters ( Fig. 4B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21059960" target="_blank">21059960</a>]<br></li>
<li>For this activity , phosphorylation of SsrB was not required , which is rare amongst response-regulators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26880544" target="_blank">26880544</a>]<br></li>
<li>Phosphorylation of SsrB is absolutely required for srfH activation ( Fig. 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15491370" target="_blank">15491370</a>]<br></li>
<li>DNA-Binding Activity of SsrB Is Selectively Inactivated by NO Congeners .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20660761" target="_blank">20660761</a>]<br></li>
<li>Binding of SsrB to DNA also is inhibited by NO autooxidation products .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20660761" target="_blank">20660761</a>]<br></li>
<li>Thus , in contrast to previous reports ( 40 ) activation of sifA by OmpR is indirect , working through SsrB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21059643" target="_blank">21059643</a>]<br></li>
<li>Thus , one mode of action employed by SsrB to regulate transcription is direct activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21059643" target="_blank">21059643</a>]<br></li>
<li>The expression of sifA was found to be strictly regulated by SsrA/SsrB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11755424" target="_blank">11755424</a>]<br></li>
<li>The speciﬁc signal that is received by sensor SsrA leading to activation of response-regulator SsrB and expression of SPI2 genes is not known to date .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16904940" target="_blank">16904940</a>]<br></li>
<li>The SsrA/SsrB TCS , which is unique to Salmonella , regulates the expression of T3SS-2 effectors .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20656781" target="_blank">20656781</a>]<br></li>
<li>Another possibility is that SsrB or another protein binds to the ssrAB promoter region , with or without OmpR , to activate or repress transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10633113" target="_blank">10633113</a>]<br></li>
<li>The DNA-binding activity of SsrB is preserved after treatment with H2O2 or NO but is inhibited after exposure to ONOO − , oxidative byproducts of NO , or the thiol oxidizer iodoacetate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20660761" target="_blank">20660761</a>]<br></li>
<li>It is tempting to speculate that SsrB activates oafA within macrophages , via the inversion system , to trigger a protective LPS conformational change that is vulnerable to the immune system when extracellular .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10844662" target="_blank">10844662</a>]<br></li>
<li>Collectively , these results demonstrate that SsrB acts to regulate multiple levels of resistance to ROS , to which it is exposed in the SCV .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25249283" target="_blank">25249283</a>]<br></li>
<li>However , SsrB is distinct from NarL in that the isolated Cterminus of NarL is capable of binding to DNA , but it is unable to activate transcription ( Maris et al. , 2002 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15491370" target="_blank">15491370</a>]<br></li>
<li>This system could be contributing to the EnvZ-independent , OmpR-dependent increase in SsrB that we observed at pH 5.7 ( Fig. 10 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15491370" target="_blank">15491370</a>]<br></li>
<li>Single molecule experiments with magnetic tweezers demonstrated that SsrB displaces H-NS from DNA only when it is bound in a polymerization ( stiffening ) mode and not when H-NS is bound to DNA in the bridging mode .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21059643" target="_blank">21059643</a>]<br></li>
<li>In this study , we show that SsrB promotes H-NS release from DNA bound in the polymerization mode , whereas H-NS bound to DNA in the bridging mode is inert to SsrB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21059643" target="_blank">21059643</a>]<br></li>
<li>SsrB , but not its cognate kinase SsrA , is involved in the switch from planktonic growth to a multi-cellular lifestyle , in addition to its role in regulating genes required for pathogenesis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26880544" target="_blank">26880544</a>]<br></li>
<li>N-terminal deletions in SsrB alter its ability to stimulate transcription.The entire SsrB protein is represented schematically in a linear fashion .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15491370" target="_blank">15491370</a>]<br></li>
<li>Most type III effectors were detected in culture supernatants regardless of SsrB overexpression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20974834" target="_blank">20974834</a>]<br></li>
<li>The 17 regulators required for survival in Nramp1 / BALB/c mice are SpvR , FruR , YbdM , HimD , PhoP/PhoQ , SsrA/SsrB , SlyA , Hnr , STM2575 , RpoE , SmpB , CsrA , STM2912 , RpoS , Crp , OmpR/EnvZ , and Hfq ( see Table S1 in the supplemental material ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21041489" target="_blank">21041489</a>]<br></li>
<li>The amount of SsrB protein was similar in NO − 3 2 - and NO − - treated Salmonella ( Fig. 1A , Upper ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20660761" target="_blank">20660761</a>]<br></li>
<li>Soon after uptake of S. enterica serovar Typhimurium in macrophages , SsrB is expressed and induces several proteins that facilitate intracellular replication ( 10 , 36 , 42 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15972528" target="_blank">15972528</a>]<br></li>
<li>Because purified preparations of fulllength SsrB were unstable in solution , we employed a C-ter-minal construct that we previously showed binds to DNA in vitro and activates transcription in vivo ( 15 , 23 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21059643" target="_blank">21059643</a>]<br></li>
<li>SsrBC Stimulates Transcription of sifA , sifB , and sseJ in Vitro in the Absence of Additional Factors -- To determine whether SsrB could directly activate transcription , we used purified SsrBC in an in vitro transcription assay .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21059643" target="_blank">21059643</a>]<br></li>
<li>The transcriptional activity of these reporters was measured in vitro in deﬁned LPM minimal medium , which provides an environment in which SsrB and upstream regulators of virulence genes , PhoPQ and SlyA , are active ( 10 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22493086" target="_blank">22493086</a>]<br></li>
<li>DalS protein analyzed by Western blot in wt and ssrB cells showed that DalS levels were increased in minimal medium compared with LB and that SsrB was required for full expression of DalS in minimal medium ( Fig. 1C ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22418438" target="_blank">22418438</a>]<br></li>
<li>Wild-type and SsrB C203S variants expressed by AV07104 and AV08171 strains , respectively , were grown for 6 h in EG medium , pH 5.5 , in the presence of 750 μM NO − 2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20660761" target="_blank">20660761</a>]<br></li>
<li>As seen in Fig. 4A and B , PhoP and SsrB were necessary for expression of srfJ in LPM .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22661691" target="_blank">22661691</a>]<br></li>
<li>This is in contrast with genes encoding other Salmonella effectors that are speciﬁcally secreted through T3SS2 and whose regulation is strictly dependent on the SsrA/SsrB two-component system ( 45 , 50 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25182488" target="_blank">25182488</a>]<br></li>
<li>Clearly , SsrB has the greater effect , activating spiC expression 60-fold , whereas the stimulation by OmpR is 2.4-fold .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12753201" target="_blank">12753201</a>]<br></li>
<li>To trigger the disease-causing lifestyle , SsrA is activated and adds a phosphate group onto SsrB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26880544" target="_blank">26880544</a>]<br></li>
<li>SsrB is the response-regulator , and membrane resident SpiR is the predicted integral cognate sensor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21984608" target="_blank">21984608</a>]<br></li>
<li>B. PipB expression is induced by SPI-2 growth conditions and is dependent on the SPI-2 transcriptional-activator SsrB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11918798" target="_blank">11918798</a>]<br></li>
<li>The mechanism by which this regulation is achieved is not understood , but results presented here suggest that the response-regulator SsrB relieves H-NS-mediated silencing of SPI-2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630976" target="_blank">17630976</a>]<br></li>
<li>Lee AK , Detweiler CS , Falkow S. OmpR regulates the two-compo-nent system SsrA-SsrB in Salmonella pathogenicity island 2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28270274" target="_blank">28270274</a>]<br></li>
<li>The SsrA sensor kinase phosphorylates the SsrB response-regulator to activate gene expression [ 32 -- 34 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26561851" target="_blank">26561851</a>]<br></li>
<li>SPI-2 gene expression is dependent on the SsrA/SsrB two-component system ; where the SsrB response-regulator activates SPI-2 structural genes encoding a T3SS [ 67,68 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24603618" target="_blank">24603618</a>]<br></li>
<li>When activated , the SsrB response-regulator directly promotes the transcription of multiple genes within Salmonella pathogenicity island 2 ( SPI-2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21059960" target="_blank">21059960</a>]<br></li>
<li>The response-regulator , SsrB , binds directly to the promoters and activates transcription of gene clusters within SPI-2 ( 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21059960" target="_blank">21059960</a>]<br></li>
<li>In the absence of SsrA kinase activity , small inorganic phosphate donors such as acetyl phosphate can phosphorylate SsrB , albeit with a lower efficiency ( Feng et al. , 2004 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26880544" target="_blank">26880544</a>]<br></li>
<li>In Overexpression of wild-type SsrB in the wild-type background contrast , SsrBC L201A was severely impaired in its ability to resulted in a further 2.5-fold increase in expression ( Fig. 10 , form a dimer when cross-linked with BMH ( Fig. 11G ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19126546" target="_blank">19126546</a>]<br></li>
<li>Growth of this strain with glucose as the sole carbon source results in no SsrB being produced , whereas growing this strain with arabinose as the sole carbon source results in ectopic SsrB expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10844662" target="_blank">10844662</a>]<br></li>
<li>In contrast , SsrB stimulated activity ninefold in the wild-type strain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630976" target="_blank">17630976</a>]<br></li>
<li>- Galactosidase assays to deter - positive and negative controls , mine activity from the sseI promoter were performed in strain MJW604 expressing SsrB from an arabinose - respectively ( Fig. 9 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19126546" target="_blank">19126546</a>]<br></li>
<li>Finally , as a negative control , in the ssrB null strain , no SsrB was produced ( lanes 5 and 10 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15491370" target="_blank">15491370</a>]<br></li>
<li>Each strain containing a specific regulator deletion was transformed with pBAD30SsrB or pBAD30 and grown under AMM1 condition in the presence of 0.02 % L-arabinose for 4 hours .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19229334" target="_blank">19229334</a>]<br></li>
<li>These experiments demonstrated a functional role for the conserved palindrome in Sodalis and the requirement for SsrB for transcriptional activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20300643" target="_blank">20300643</a>]<br></li>
<li>So SsrB regulates Salmonella 's defense against the oxidative burst by subverting the burst itself , as well as detoxifying residual ROS from the inhibited respiratory burst .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25249283" target="_blank">25249283</a>]<br></li>
<li>SsrB was found to indirectly regulate this locus by tran - scriptional read-through from the sspH2 ( STM2241 ) promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23630960" target="_blank">23630960</a>]<br></li>
<li>Moreover , phosphorylated EIIANtr could also bind to the SsrB protein in vitro ( Fig .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21059960" target="_blank">21059960</a>]<br></li>
<li>EIIANtr interacted directly with SsrB , which prevented the SsrB protein from binding to its target promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21059960" target="_blank">21059960</a>]<br></li>
<li>On the basis of several observations , we propose that EIIANtr binds directly to the SsrB protein to relieve Ntr SsrB-binding to its target promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21059960" target="_blank">21059960</a>]<br></li>
<li>To survive in the host environment , Salmonella enterica uses a type III secretion system and effector-proteins , which are activated by the SsrA-SsrB two-component system in response to the host environment .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20300643" target="_blank">20300643</a>]<br></li>
<li>When the SsrB concentration was increased to 300 nM , we detected large-scale condensation of the DNA-protein complexes by AFM ( Figure 5C ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26880544" target="_blank">26880544</a>]<br></li>
<li>When the SsrB protein was incubated with 5 ′ end-labeled DNA probes , it bound to the ssaG promoter DNA to form the SsrB -- DNA complex ( Fig. 4A , lane 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21059960" target="_blank">21059960</a>]<br></li>
<li>Conversely , the novel effectors CigR and SssB were restricted to the samples where SsrB was overexpressed ( Table S3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20974834" target="_blank">20974834</a>]<br></li>
<li>Taken together , the data demonstrate that EIIANtr speciﬁcally binds to the SsrB response-regulator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21059960" target="_blank">21059960</a>]<br></li>
<li>As the SsrB protein is a response-regulator , the DNA-binding activity , which is controlled by the phosphorylation state ( 4 ) , the cognate sensor kinase , SsrA , should be activated to phosphorylate the SsrB protein within macrophages .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21059960" target="_blank">21059960</a>]<br></li>
<li>However , unlike the Salmo-nella EIIANtr interaction with the SsrB response-regulator , the E. coli homolog binds to the KdpD sensor protein and stimulates its kinase activity , thereby increasing the levels of phosphorylated Ntr KdpE ( 19 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21059960" target="_blank">21059960</a>]<br></li>
<li>Under similar conditions as shown in Fig. 2B , only 39 % of the SsrB protein was phosphorylated by acetyl phosphate ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15491370" target="_blank">15491370</a>]<br></li>
<li>Furthermore , PAT DE4 was positive for the outer membrane receptor gene instead of but no signal was obtained for the islet genes htrE ( probable porin / ﬁmbrial assembly protein ) , sopD2 ( secreted effector-protein ) , srfJ ( putative virulence factor , activated by transcription factor SsrB ) and sseK2 ( translocated effector-protein SSEK2 by type III secretion system ) while pagK ( PhoPQ-activated protein ) was present only in this array type .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21917347" target="_blank">21917347</a>]<br></li>
<li>This may be in part because , at many promoters , SsrB acts primarily to counter the effects of gene silencing by H-NS , rather than as a direct transcriptional-activator ( 8 , 32 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19126546" target="_blank">19126546</a>]<br></li>
<li>In the absence of the SsrA kinase , SsrB is not phosphorylated , but it can counter H-NS silencing at csgD ( Figure 4A -- D and Figure 6A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26880544" target="_blank">26880544</a>]<br></li>
<li>SsrB regulates the expression of SPI-2 encoded substrate effectors including ssaB , as well as several integrated virulence effectors such as sseL [ 8 ] and srfN [ 9 ] that are encoded elsewhere on the chromosome but that have integrated into the SsrB regulon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19245712" target="_blank">19245712</a>]<br></li>
<li>The SsrB protein is necessary for transcription of the genes specifying the T3SS encoded in the SPI-2 pathogenicity island as well as its secreted effectors ( 19 , 20 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23690578" target="_blank">23690578</a>]<br></li>
<li>Fur represses expression of the ssrB gene encoding the SsrB response-regulator under acidic conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24643535" target="_blank">24643535</a>]<br></li>
<li>Taken together , these results illustrate that the Fur regulator represses production of the SsrB regulator under acidic conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24643535" target="_blank">24643535</a>]<br></li>
<li>SsrA is a sensor kinase activated in the host environment that phosphorylates the SsrB response-regulator to create an active transcription factor needed for spatiotemporal control of virulence genes [ 13,17 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20300643" target="_blank">20300643</a>]<br></li>
<li>SsrA is a sensor kinase that phosphorylates the cognate response-regulator , SsrB , leading to the induction of SPI-2 gene expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17307861" target="_blank">17307861</a>]<br></li>
<li>Although the mechanism of how the SsrA/SsrB two-component system is activated within the phagosome is currently unclear , the SsrB response-regulator promotes expression of SPI-2 genes when Salmonella is grown in minimal medium at acidic pH ( 5 , 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24643535" target="_blank">24643535</a>]<br></li>
<li>These results raised the question : if SsrB expression is uncoupled from SsrA , can SsrB activate SPI-2 gene expression in the absence of the SsrA kinase ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630976" target="_blank">17630976</a>]<br></li>
<li>However , the sseL promoter appears to be exquisitely ﬁne-tuned for regulation by SsrB because the deletion of the SsrB regulator silences this gene under conditions that are otherwise activating .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17158898" target="_blank">17158898</a>]<br></li>
<li>A final interaction score was obtained by taking the log2 value of the ratio between the non-inducing and inducing conditions for each probe to remove SsrB interactions that occur under non-inducing conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20300643" target="_blank">20300643</a>]<br></li>
<li>While our present understanding of SsrB is that it regulates , almost exclusively , direct interactions with the host through SPI2 , the proteomics above indicated that other functions , not involving host interaction , also come under the control of SsrB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25249283" target="_blank">25249283</a>]<br></li>
<li>However , as the SsrB protein should be phosphorylated to function as a transcription factor ( 4 ) , we Ntr can not exclude the possibility that the EIIA -- SsrB interaction may inhibit SsrA-mediated phosphorylation of SsrB or promote dephosphorylation of phosphorylated SsrB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21059960" target="_blank">21059960</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_184">
Because ssrB expression is positively autoregulated ( 7 ) , the levels of SsrB gradually increase as the pH is lowered from pH 7.0 to pH 5.5 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24643535" target="_blank">24643535</a>]<br></label>
<input id="act_similarity_184", type="checkbox">
<div class="ACT_similar">
<li>Because SsrB expression is positively autoregulated ( 7 ) , the levels of SsrB gradually increase as the external pH is lowered from pH 7.0 to pH 5.5 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24643535" target="_blank">24643535</a>]<br></li></div>
<li>We show that under biofilm-inducing conditions , unphosphorylated SsrB is sufficient to activate the expression of csgD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26880544" target="_blank">26880544</a>]<br></li>
<li>The SsrAB phosphorelay is the second level of regulation in the model ; our data suggest that SsrA expression is distinct from phosphorelay activity that leads to transcriptional activation of downstream genes , such as ssaG , indicating that SsrA requires an environmental signal to activate SsrB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15231802" target="_blank">15231802</a>]<br></li>
<li>By using this strategy , we previously demonstrated that HilD is required for the expression of SsrB in the presence of H-NS but not when it is inactivated ( 21 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25135218" target="_blank">25135218</a>]<br></li>
<li>In agreement with these results , we previously showed that full expression of the chromosomally encoded SsrA and SsrB proteins requires HilD in the presence of H-NS but not when it is inactivated ( 21 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25135218" target="_blank">25135218</a>]<br></li>
<li>The idea that the downregulation of SPI2 by the NO-sensing activity of SsrB is key to some aspect of Salmonella pathogenesis is in keeping with previous observations that showed that the repression of SPI2 is as important for Salmonella virulence as its positive regulation ( Coombes et al. , 2005 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21833325" target="_blank">21833325</a>]<br></li>
<li>Synthesis of SPI2-encoded proteins in response to acidic pH or phosphate limitation depends on SsrA and SsrB function Although the SsrAB two-component system has been identiﬁed as the central regulatory system of SPI2 regulation , it is conceivable that this system is bypassed by direct effects of other regulators on SPI2 promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16904940" target="_blank">16904940</a>]<br></li>
<li>SsrB is a SPI-2 encoded transcriptional-regulator that is required for expression of all the SPI-2-encoded genes , as well as for expression of effectors and virulence genes encoded outside of SPI-2 [ 28 -- 31 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26565973" target="_blank">26565973</a>]<br></li>
<li>Similarly , SsrB is thought to directly activate the expression of SPI2 operons as well as other effector genes located outside of SPI2 [ 28,29 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17233907" target="_blank">17233907</a>]<br></li>
<li>Therefore , expression of SPI-2 genes is not induced until the pH is lowered to 5.5 , where levels of SsrB would markedly increase ( Fig. 6A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24643535" target="_blank">24643535</a>]<br></li>
<li>It is likely the effects of σE on SPI-2 transcription is mediated by SsrB because overexpression of SsrB , but not SlyA , can complement the decrease of SPI-2 expression as a result of rpoE deletion ( Yoon et al. , 2009 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25713562" target="_blank">25713562</a>]<br></li>
<li>As expression of the SsrB protein is promoted by several transcription factors , including SsrB itself , the expression of SPI-2 genes can increase to undesirable levels under activating conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21059960" target="_blank">21059960</a>]<br></li>
<li>Excluding fusions that respond to vector sequence or arabinose To exclude fusions from this collection that were regulated by vector sequences or arabinose , in lieu of SsrB , 144 of the original 288 regulated fusions were transduced into MJW136 ( the 800 series ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10844662" target="_blank">10844662</a>]<br></li>
<li>H-NS specifically repressed transcription of sifA , but this repression was antagonized by the presence of SsrB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21059643" target="_blank">21059643</a>]<br></li>
<li>SsrB relieves H-NS repression of SPI-2 genes It was recently shown that the nucleoid protein H-NS protects against foreign DNA by binding to A/T-rich regions of DNA ( Lucchini et al. , 2006 ; Navarre et al. , 2006 ) , including SPI-2 , resulting in the inhibition of transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630976" target="_blank">17630976</a>]<br></li>
<li>Taken together , these results indicate that SsrB can activate biofilm formation in the absence of any phosphorylation signals .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26880544" target="_blank">26880544</a>]<br></li>
<li>These results demonstrated that SsrB is necessary to activate the ssrB promoter even in the absence of PmrA repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23690578" target="_blank">23690578</a>]<br></li>
<li>Furthermore , previous microarray studies disrupted both ssrA and ssrB , which would not uncover a distinct role for SsrB in gene regulation under non SPI-2-inducing conditions in the absence of the SsrA kinase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26880544" target="_blank">26880544</a>]<br></li>
<li>SPI-2 protein production was eliminated in the ssrB ydgT double mutant ( data not shown ) , indicating that SPI-2 activation requires SsrB even in the absence of YdgT .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16301528" target="_blank">16301528</a>]<br></li>
<li>To activate transcription , SsrB must be phosphorylated at Asp56 ( 13 ) , yet surprisingly , overexpression can activate SPI-2 genes in the absence of the SsrA kinase ( 8 , 13 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19126546" target="_blank">19126546</a>]<br></li>
<li>An additional unexpected result was that overexpression of SsrB activated transcription of SPI-2 genes in the absence of its cognate kinase SsrA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630976" target="_blank">17630976</a>]<br></li>
<li>These results indicate that overexpression of SsrB in the absence of the gene for its cognate kinase , ssrA , is suf-ﬁcient to activate SPI-2 transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630976" target="_blank">17630976</a>]<br></li>
<li>Overexpression of SsrB in the absence of its cognate kinase , SsrA , is sufficient to activate SPI-2 transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630976" target="_blank">17630976</a>]<br></li>
<li>SsrB activates SPI-2 in the absence of SsrA When an ssrAB double mutant was complemented with a plasmid containing an inducible ssrB gene , we were surprised to ﬁnd that expression of SsrB in the absence of its cognate kinase is sufficient to activate transcription of SPI-2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630976" target="_blank">17630976</a>]<br></li>
<li>One of the inner membrane components of T6SS , namely SciS , has also been shown to be negatively regulated by SsrB ( transcription regulator involved in SPI-2 activation ) [ 9 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24079299" target="_blank">24079299</a>]<br></li>
<li>Together , these data suggest that Cys203 serves as a redox sensor of RNS , modulating the binding of SsrB to DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20660761" target="_blank">20660761</a>]<br></li>
<li>Taken together , these lines of evidence suggest that the Fur repressor could interfere with binding of the SsrB activator to the ssrB promoter , leading to repression of ssrB transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24643535" target="_blank">24643535</a>]<br></li>
<li>NMR analysis of the L201A mutant indicated only form of SsrB present , it was unable to activate sseI tran - that the substitution leads to unfolding of H4 , which would scription ( see Fig. 9 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19126546" target="_blank">19126546</a>]<br></li>
<li>N O and dinitrosyl -- iron complexes ( DNIC ) , which frequently target 2 3 thiol-containing proteins , are common sources of nitrosative stress.The redox active thiol of Cys203 in the dimerization domain of the SsrB response-regulator that controls SPI2 gene transcription is a bona fide sensor of nitrosative stress in Salmonella .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21833325" target="_blank">21833325</a>]<br></li>
<li>Determination of the specificity of SsrB regulation The specificity of SsrB activity to the intracellular environment was determined .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10844662" target="_blank">10844662</a>]<br></li>
<li>The negatively charged modiﬁcations of ONOO − - treated SsrB may block these interactions and thereby inhibit the formation of dimers necessary for the binding of SsrB to SPI2 promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20660761" target="_blank">20660761</a>]<br></li>
<li>( B ) Acquisition of the transcriptional-regulator SsrB in Salmonella Typhimurium led to cis-regulatory evolution of core genes that allow them to come under the regulatory control of SsrB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29034217" target="_blank">29034217</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29034217" target="_blank">29034217</a>]<br></li>
<li>The redox-sensing capabilities of SsrB appear to be dispensable for virulence of Salmonella in a model of acute systemic infection that is dominated by the antimicrobial activity of the NADPH phagocyte oxidase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20660761" target="_blank">20660761</a>]<br></li>
<li>Mechanism of SsrB transcriptional activation The mechanism of action in the SsrB activation of transcription is similar to the subfamily member NarL , in that phosphorylation appears to relieve inhibition of the C-terminal effector domain ( Fig. 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15491370" target="_blank">15491370</a>]<br></li>
<li>C T A G 1 2 t g a a a a a a t c g a c The isolated C-terminus of SsrB can activate transcription Some response-regulators are prevented from binding to DNA in the absence of phosphorylation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15491370" target="_blank">15491370</a>]<br></li>
<li>SsrB was able to activate transcription of sifA in the C presence ( 1.7-fold ) or absence ( 3-fold ) of CTD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21059643" target="_blank">21059643</a>]<br></li>
<li>SsrB-mediated activation within murine spleen In addition to the cell culture induction experiments , the ability of SsrB to activate genes in vivo at systemic sites of infection was also tested .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10844662" target="_blank">10844662</a>]<br></li>
<li>The ability of SsrB to activate genes in tissue culture media was then compared with the ability of SsrB to activate genes in macrophages .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10844662" target="_blank">10844662</a>]<br></li>
<li>Given the widespread occurrence of the wildtype allele in Salmonella enterica , these ﬁndings indicate that SsrB Cys203 increases Salmonella virulence by serving as a redox sensor of NO resulting from the host immune response to oral infection .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20660761" target="_blank">20660761</a>]<br></li>
<li>Nonetheless , compared with the strain expressing a redox-resistant SsrB C203S variant , wild-type Salmo-nella bearing the NO-responsive allele exhibit increased ﬁtness when exposed to RNS in an NRAMPR , C3H/HeN murine model of acute oral infection .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20660761" target="_blank">20660761</a>]<br></li>
<li>A wild-type SsrB bearing a redox-active thiol in the Cys203 is advantageous to the interactions of Salmo-nella with its host , as demonstrated by the increased ﬁtness of wild-type Salmonella over an isogenic strain expressing the RNS-resistant SsrB C203S variant in a C3H/HeN murine model of acute oral salmonellosis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20660761" target="_blank">20660761</a>]<br></li>
<li>It also appears that the N-terminus of SsrB inhibits-the-activity of its C-terminus , as domain liberation increases its activity compared with the fulllength , unphosphorylated protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15491370" target="_blank">15491370</a>]<br></li>
<li>Is the SsrB expressed in the absence of envZ dependent on OmpR ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15491370" target="_blank">15491370</a>]<br></li>
<li>The activation of the reporter fusion was dependent on the function of SsrB , as no induction was observed in the ssrB background .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=14641178" target="_blank">14641178</a>]<br></li>
<li>Surprisingly , the SsrB response-regulator positively regulated the formation of biofilms by activating csgD expression in the absence of any phos-pho-donors .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26880544" target="_blank">26880544</a>]<br></li>
<li>Thus , the response-regulator SsrB appeared to play a direct role in the activation of srfH , whereas the role of OmpR is less clear .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15491370" target="_blank">15491370</a>]<br></li>
<li>However , high amounts of the sensor SsrA could lead to increased phosphorylation of the response-regulator SsrB , even in the absence of the correct physiological signal .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16904940" target="_blank">16904940</a>]<br></li>
<li>We assume that all genes related to the TTSS are under this regulation , since the reduced transcription observed in the yfgL mutant of S. enterica serovar Enteritidis is the consequence of the transcriptional down-regulation of the central regulators HilA , SsrB , and FlhD2C2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17060472" target="_blank">17060472</a>]<br></li>
<li>Cues from within the host intracellular environment likely play a role in the regulation of the S. Typhimurium T6SS , as it has been determined that transcription of sciS increases approximately 16 h after the invasion of macrophages , and transcript levels are negatively correlated with the activity of the two-component response-regulator of the SPI-2 T3SS , SsrB ( 38 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22493086" target="_blank">22493086</a>]<br></li>
<li>In silico analysis revealed the presence of three cysteines in SsrB , raising the possibility that these residues could be targets of RNS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20660761" target="_blank">20660761</a>]<br></li>
<li>Normalized levels of csgD transcripts were highly up-regulated in the ssrB hns double mutant compared to the ssrB null strain , indicating that the loss of H-NS repression rescued the defect in csgD expression in the absence of SsrB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26880544" target="_blank">26880544</a>]<br></li>
<li>The lack of iNOS abrogated the differences in sifA : : lacZY expression in these two strains , suggesting that the presence of a redox-active Cys203 modulates the transcriptional activity of SsrB in response to RNS produced in the innate response of macrophages .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20660761" target="_blank">20660761</a>]<br></li>
<li>In the absence of SsrB , the transcriptional-activator of the SsrAB regulon , only very low levels of synthesis of fusion proteins were observed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20732994" target="_blank">20732994</a>]<br></li>
<li>Peroxynitrite and products of the autooxidation of NO in the presence of oxygen , but not hydrogen peroxide , inhibit the DNA-binding capacity of SsrB , demonstrating the selectivity of the reaction of Cys203 with reactive nitrogen species ( RNS ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20660761" target="_blank">20660761</a>]<br></li>
<li>In contrast , limitation of inorganic phosphate ( Pi ) in the growth media led to activation of SPI2 genes in the late exponential-growth-phase and was independent of the media pH. Limitation of Pi induced an over-expression of sensor protein SsrB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16904940" target="_blank">16904940</a>]<br></li>
<li>In contrast , in the absence of Fur repression on the ssrB promoter , levels of SsrB protein would gradually increase over the course of environmental acidiﬁcation ( Fig. 6B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24643535" target="_blank">24643535</a>]<br></li>
<li>( B ) In contrast , in the absence of Fur , levels of SsrB gradually increase during environmental acidiﬁcation , which then leads to the gradual induction of SPI-2 expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24643535" target="_blank">24643535</a>]<br></li>
<li>First , EIIA - mediated regulation of SPI-2 genes was dependent on the SsrB protein because ptsN mutation failed to increase SPI-2 expression in the Ntr absence of SsrB ( Fig. 2A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21059960" target="_blank">21059960</a>]<br></li>
<li>We conﬁrmed that the presence of SsrB increased the expression of the selected SPI-2 genes studied here using qRT-PCR , by comparing the expression ratio of these SPI-2 genes in WT vs. ssrB strains ( Figure 4B , black bars ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25713562" target="_blank">25713562</a>]<br></li>
<li>In the absence of H-NS , SPI-2 gene expression is substantially de-repressed , even in the absence of SsrB ( Fig. 10 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630976" target="_blank">17630976</a>]<br></li>
<li>Under inducing conditions , although H-NS repression can still be observed in the absence of SsrB , the effects of H-NS on SPI-2 expression are minimal in the presence of SsrB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630976" target="_blank">17630976</a>]<br></li>
<li>S2 in the supplemental material ) , we hypothesized that SsrB overexpression might increase the probability of identifying novel type III effectors .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20974834" target="_blank">20974834</a>]<br></li>
<li>SsrBC recognition helix mutants K179A and SsrB activity , we expressed the SsrB mutant in a strain contain - M186A cross-linked with BMH to the same extent as the wilding the sseI-lacZ fusion in the presence of the wild-type chro - type protein ( Fig 11 , C and D ) , indicating that substitutions in mosomal ssrB gene .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19126546" target="_blank">19126546</a>]<br></li>
<li>Interestingly , EIIANtr did prevent SsrB from binding to the target DNA : as the concentrations of EIIANtr increased in the reactions , the levels of the SsrB -- DNA complex decreased to generate the free ssaG promoter DNA ( Fig. 4B , lanes 3 -- 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21059960" target="_blank">21059960</a>]<br></li>
<li>When grown under pH 6.0 conditions with the inducer IPTG , the ssrB mutant carrying the SsrB expression plasmid expressed ssaG mRNA at wild-type levels , which increased by 2.5-fold in the absence of the fur gene ( Fig. 5B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24643535" target="_blank">24643535</a>]<br></li>
<li>Under acidic pH conditions , the strain harboring the functional ssrB gene but lacking the ptsN gene overexpressed ssaG , whereas the ptsN mutation failed to increase the levels of ssaG gene transcription in the absence of SsrB ( Fig. 2A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21059960" target="_blank">21059960</a>]<br></li>
<li>However , the up-regulation of SPI1-associated genes in the absence of SsrB suggests an additional layer of transcriptional control for SPI1 genes , via SsrB-mediated repression under SPI2-inducing conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27564394" target="_blank">27564394</a>]<br></li>
<li>Deletion of the genes specifying the nucleoid-associated protein attenuates Salmonella 's virulence in mice ( 48 , 49 ) , whereas preventing PmrA-mediated negative regulation of SsrB increases virulence ( Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23690578" target="_blank">23690578</a>]<br></li>
<li>SseL expression was detected following infection of bone marrow-derived macrophages as early as 30 min postinfection ( Fig. 2A ) , and ﬂuorescence of the fusion protein was dependent on SsrB during infection of mac-rophages , since there was no ﬂuorescence increase in cells infected with the SsrB mutant .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17158898" target="_blank">17158898</a>]<br></li>
<li>Under these conditions , ª 87 % of the SsrB protein was phosphorylated .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15491370" target="_blank">15491370</a>]<br></li>
<li>In the absence of SsrB ( lane 2 ) , no product was observed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15491370" target="_blank">15491370</a>]<br></li>
<li>Promoter activity was determined by measuring b-galactosidase activity in the presence or absence of SsrB overexpression from the low-copy complementing plasmid pKF44 ( Feng et al. , 2004 ) , as shown in Fig. 5 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630976" target="_blank">17630976</a>]<br></li>
<li>In order to determine the transcriptional factors responsible for cas3 negative regulation , its transcriptional activity was assessed in individual mutants lacking the H-NS [ 49 ] , LRP , CRP , CpxR , SsrB , IHF , PhoP , PmrA , FNR or OmpR regulators , particularly since some of them ( H-NS or CRP ) have been previously implicated in the regulation of other CRISPR-Cas systems [ 33 , 50 , 51 ] or in virulence of Salmonella ( CpxR , SsrB , IHF , PhoP , PmrA , FNR or OmpR ) [ 52 -- 58 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28270274" target="_blank">28270274</a>]<br></li>
<li>The role of SsrB in the regulation by PhoP was investigated in a double mutant phoQ24 ssrB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25972862" target="_blank">25972862</a>]<br></li>
<li>SsrB expression resulted in activation of the sseI promoter , whereas expression of SsrB D56A did not activate sseI ( Fig. 9 , columns 1 -- 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19126546" target="_blank">19126546</a>]<br></li>
<li>Thus , we strengthened our prediction of a role for SsrB in the activation of csgD expression by acting as an anti-H-NS molecule .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26880544" target="_blank">26880544</a>]<br></li>
<li>In the absence of H-NS , the requirement for SsrB in activating SPI-2 genes is substantially reduced , suggesting a role for SsrB in countering H-NS silencing .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630976" target="_blank">17630976</a>]<br></li>
<li>Moreover , the SsrB requirement in the absence of H-NS ( see below ) is substantially reduced , suggesting a dual role for SsrB in SPI-2 induction : activation of transcription and countering H-NS-mediated repression ( 238 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23554419" target="_blank">23554419</a>]<br></li>
<li>An examination of transcriptional activation during the course of macrophage infection found that all SsrB relieves H-NS silencing of SPI-2 479 SPI-2 operons are activated during growth in vivo in an SsrB-dependent fashion .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630976" target="_blank">17630976</a>]<br></li>
<li>The greater effect of SsrB on SPI-2 expression in the presence of H-NS suggests that SsrB plays a role in countering H-NS-mediated silencing .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630976" target="_blank">17630976</a>]<br></li>
<li>( ii ) Addition of 600 nM SsrB to the H-NS bound csgD DNA resulted in areas of condensation ( pink arrows ; an ` SsrB signature ' ) along with a few areas where the straight H-NS bound conformation persisted ( yellow line ; an ` H-NS signature ' ) ; Scale bar = 200 nm as in Figure 5A .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26880544" target="_blank">26880544</a>]<br></li>
<li>E , SsrB folds DNA in the presence of H-NS bound in the stiffening mode ( no C MgCl2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21059643" target="_blank">21059643</a>]<br></li>
<li>This behavior has been observed in our single-molecule stretching experiments with H-NS filaments in the presence of SsrB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26880544" target="_blank">26880544</a>]<br></li>
<li>Consistent with the transcription data , SsrB protein levels were 3.5-fold higher in the absence of fur than in its presence under pH 6.0 conditions , but were similar under pH 7.0 conditions regardless of fur background ( Fig. 3C and D ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24643535" target="_blank">24643535</a>]<br></li>
<li>Immunoblots of NeutrAvidin-puriﬁed , biotinylated proteins demonstrated the S-nitrosylation of SsrB in Salmonella cultures exposed to acidiﬁed NO − 3 2 but not in controls grown in NO − ( Fig. 1A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20660761" target="_blank">20660761</a>]<br></li>
<li>The cytoplasmic extracts were tested for the presence of S-nitrosylated SsrB after de-rivatization of S-nitrosylated proteins in a biotin switch assay .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20660761" target="_blank">20660761</a>]<br></li>
<li>Incubation of this DNA fragment with increasing concentrations of purified SsrBc ( 5 ) resulted in a retardation of band migration ( Fig. 1D ) that was inhibited in the presence of unlabeled competitor DNA confirming that SsrB binds to the dalS promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22418438" target="_blank">22418438</a>]<br></li>
<li>Intriguingly , reapplication of force resulted in unfolding of the DNA ( Fig. 4F , blue triangles ) , as seen with H-NS alone ( Fig. 4B ) and despite the presence of SsrB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21059643" target="_blank">21059643</a>]<br></li>
<li>As expected , SseB was expressed only in the presence of SsrB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17158898" target="_blank">17158898</a>]<br></li>
<li>Growth of MJW141 in the presence of arabinose results in expression of SsrB , whereas growth in its absence does not .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10844662" target="_blank">10844662</a>]<br></li>
<li>However , addition of ssrB on an arabinose-inducible plasmid in the presence of arabinose stimulated srfH -- lacZ activity 20-fold , mimicking the effect we observed with SsrB stimulation of srfH in Salmonella-infected macrophages ( Fig. 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15491370" target="_blank">15491370</a>]<br></li>
<li>Yet surprisingly little is known about how it Lys179 and Met186 are in the DNA recognition helix of SsrB and recognizes DNA and relieves H-NS silencing to activate tran-are predicted to be DNA contact residues .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19126546" target="_blank">19126546</a>]<br></li>
<li>This ability of H-NS to re-orient on the DNA without being released would also promote its re-binding and silencing when SsrB or other anti-silencers are released ( Figure 6B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26880544" target="_blank">26880544</a>]<br></li>
<li>Addition of SsrB Displaces H-NS When It Is Bound to DNA C in the Polymerized Form and Not the Bridged Form -- In a previous study , we discovered that H-NS has two modes of interacting with DNA and that Mg2 or other divalent cations can switch its binding mode without H-NS being released from DNA ( 26 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21059643" target="_blank">21059643</a>]<br></li>
<li>In contrast , 10 mM Mg2 promoted H-NS bridging of DNA , and this DNA binding conformation was insensitive to the presence of SsrB ( Fig. 4F ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21059643" target="_blank">21059643</a>]<br></li>
<li>To examine the inﬂuence of SsrA/SsrB in myo-inositol metabolism , the reporter strains were grown in low phosphate - and low magnesium-containing medium ( LPM ) with pH 5.8 known to induce SPI-2 target genes ( Deiwick et al. , 1999 ; Coombes et al. , 2004 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25213016" target="_blank">25213016</a>]<br></li>
<li>In contrast to OmpR ( Ames et al. , 1999 ) , the presence of ssrA DNA in the phosphorylation reaction did not stimulate SsrB phosphorylation ( data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15491370" target="_blank">15491370</a>]<br></li>
<li>However , SsrB does not seem to be involved in regulation of taiA and clyA ( Fig. 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19202096" target="_blank">19202096</a>]<br></li>
<li>Taken together , these results indicated that SirA , in addition to HilD , OmpR and SsrB , is required for the expression of SPI-2 genes when Salmonella grows in LB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21518393" target="_blank">21518393</a>]<br></li>
<li>Because SsrA-based activation of SsrB presumably occurs in the in vitro conditions which we examined ( based on upregulation of ssaG expression ) , we propose that cations and/or pH is likely to be involved in this step .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15231802" target="_blank">15231802</a>]<br></li>
<li>In contrast , the data herein indicate that SsrB appears to regulate a much broader range of functions than reported in previous studies , which had essentially focused on T3SS-2 and its effectors .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25249283" target="_blank">25249283</a>]<br></li>
<li>From the data shown in Fig. 2C , it is apparent that both OmpR and SsrB activate the transcriptional fusions to spiC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12753201" target="_blank">12753201</a>]<br></li>
<li>An evaluation to determine transcriptional factors involved in cas3 regulation , as well as those in cas-CRISPR operon , scse2 and ascas2-1 , led to the finding that the CpxR , SsrB , IHF , PhoP , PmrA , FNR , CRP , LRP and OmpR regulators are not involved in the transcriptional control of these transcriptional units ( Fig .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28270274" target="_blank">28270274</a>]<br></li>
<li>As seen in Fig. 6A and B , addition of 55.5 mM myo-inositol to LB -- 0.3 M NaCl induced expression of srfJ to the same level as that obtained for the iolR mutant in LB -- 0.3 M NaCl in a PhoP - , SsrB - , and RcsB-indepen-dent manner .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22661691" target="_blank">22661691</a>]<br></li>
<li>Expression ratio comparing WT to ssrB indicates that SsrB activates SPI-2 gene expression ; Expression ratio comparing ssrB to ( ssrB , rpoE ) indicates that the up-regulating effects of σE on SPI-2 gene expression are through SsrB in LPM 4 h condition .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25713562" target="_blank">25713562</a>]<br></li>
<li>We reveal that Fur controls SPI-2 genes via repression of SsrB activity and provide evidence that Fur represses SsrB production by binding to the ssrB promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24643535" target="_blank">24643535</a>]<br></li>
<li>A B 4 5 6 7 5 6 7 8 1 2 3 SsrB 1 2 3 4 OmpR 20 39 +111 -122 -95 +181 +185 16 31 100 ATG +209 [ OmpR ] nM [ SsrBc ] nM SsrA is expressed in various backgrounds under SPI-2 activating conditions at pH 5.7 Because it appeared that expression of SsrA and SsrB were uncoupled from one another , we wanted to determine the conditions under which the proteins were expressed .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15491370" target="_blank">15491370</a>]<br></li>
<li>In the present study , we showed that EIIANtr acts on the SsrB response-regulator and allows SPI-2 genes to be expressed at normal levels under inducing conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21059960" target="_blank">21059960</a>]<br></li>
<li>Fur controlled SPI-2 genes via repression of the SsrB response-regulator , a primary activator of SPI-2 expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24643535" target="_blank">24643535</a>]<br></li>
<li>SsrB , the response-regulator of the SsrA/SsrB two-component system , directly activates expression of SPI-2 genes under acidic conditions ( 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24643535" target="_blank">24643535</a>]<br></li>
<li>SsrB can activate expression of all SPI-2-encoded T3SS and its effectors via binding to the promoters of SPI-2 functional genes [ 14 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29751061" target="_blank">29751061</a>]<br></li>
<li>Overexpression of SsrB in the absence of ssrA activates transcription of SPI-2 and SPI-2 co-regulated genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630976" target="_blank">17630976</a>]<br></li>
				<label class="ACT_collapse_similarity" for="act_similarity_297">
Carroll , R.K. , Liao , X. , Morgan , L.K. , Cicirelli , E.M. , Li , Y. , Sheng , W. , et al. ( 2009 ) Structural and functional analysis of the C-terminal DNA binding domain of the Salmonella typhimurium SPI-2 response-regulator SsrB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25442048" target="_blank">25442048</a>]<br></label>
<input id="act_similarity_297", type="checkbox">
<div class="ACT_similar">
<li>Carroll , R.K. , Liao , X. , Morgan , L.K. , Cicirelli , E.M. , Li , Y. , Sheng , W. , Feng , X. , and Kenney , L.J. ( 2009 ) Structural and functional analysis of the C-terminal DNA binding domain of the Salmonella typhimurium SPI-2 response-regulator SsrB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28631437" target="_blank">28631437</a>]<br></li></div>
<li>As predicted , SsrB activates transcription at promoters found upstream of the ﬁrst gene of each functional cluster ( ssaB , sseA and ssaG ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630976" target="_blank">17630976</a>]<br></li>
<li>In the acidic macrophage vacuole , the kinase SsrA phosphorylates SsrB , and SsrB ~ P relieves silencing of virulence genes and activates their transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26880544" target="_blank">26880544</a>]<br></li>
<li>SirA activates the expression of SsrB and HilA through HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21518393" target="_blank">21518393</a>]<br></li>
<li>SsrB activates the expression of SPI-2-related genes , both inside and outside the pathogenicity island .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12949164" target="_blank">12949164</a>]<br></li>
<li>To determine whether the SsrB target of biofilm regulation was dependent on the SPI-2 injectisome or any of its secreted SPI-2 effectors , we examined ssaC and ssaJ null strains for their ability to form biofilms ( Figure 3A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26880544" target="_blank">26880544</a>]<br></li>
<li>Results SsrB supports biofilm formation in the absence of its kinase SsrA The SsrA/B TCRS is activated by environmental stimuli such as pH and osmolality , stimulating the expression of virulence genes essential for intra-cellular growth and survival of Salmonella ( Feng et al. , 2003 ; 2004 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26880544" target="_blank">26880544</a>]<br></li>
<li>Phosporylation of SsrB is not required for biofilm formation Phospho-relay between the sensor kinase , SsrA , and the response-regulator , SsrB , was crucial for activation of SPI-2 virulence genes ( Feng et al. , 2003 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26880544" target="_blank">26880544</a>]<br></li>
<li>In the present work , we focus on events downstream of ssrAB activation and describe the organization of SPI-2 operons and their regulation by the response-regulator SsrB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630976" target="_blank">17630976</a>]<br></li>
<li>When activated within macrophages , the SsrB protein promotes transcription of SPI-2 , possibly by counteracting the H-NS -- and YdgT-mediated repression ( 7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21059960" target="_blank">21059960</a>]<br></li>
<li>The genetic regulation of gogB in Salmonella is inﬂuenced by the transcriptional-activator , SsrB , under SPI-2-inducing conditions , but the modular nature of the gogB gene allows for autonomous expression and type III secretion following horizontal gene transfer into a heterologous pathogen .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15843015" target="_blank">15843015</a>]<br></li>
<li>The expression of SPI-2 invasion genes and effector-proteins is dependent upon the presence of the SsrA/SsrB two-component regulatory system encoded by SPI-2 ( 11 , 18 , 44 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15039320" target="_blank">15039320</a>]<br></li>
<li>SsrB regulation of SPI-2 expression is direct , as evidenced by the ﬁnding that the C-termi-nal DNA-binding domain of SsrB binds to SPI-2 promoters that control genes encoding the TTSS apparatus and effector-proteins ( 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24643535" target="_blank">24643535</a>]<br></li>
<li>SsrB protein binds to the promoters of all SPI-2 functional gene clusters and is essential for expression of the structural , regulatory , and effector components of the SPI-2 locus [ 79 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21984608" target="_blank">21984608</a>]<br></li>
<li>The OmpR protein , apart from regulating expression of porin proteins like OmpC and OmpF in response to osmolarity , regulates the expression of another two-component regulatory system , SsrA-SsrB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18355292" target="_blank">18355292</a>]<br></li>
<li>The response-regulator SsrB controls expression of a type III secretory system in which effectors modify the vacuolar membrane and prevent its degradation via the endocytic pathway .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19126546" target="_blank">19126546</a>]<br></li>
<li>This vicinal configuration and distribution of cystuted with an alanine or a serine , this interaction is likely mainteine residues is unique to SsrB response-regulator proteins in tained , whereas a residue with a negative charge at this position Salmonella serovars typhimurium and typhi .5 The presence of a can destabilize the helix , disrupting the dimer as well as DNA cysteine residue within H4 raised the possibility that it could binding ( see Fig. 9 and also `` Discussion '' ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19126546" target="_blank">19126546</a>]<br></li>
<li>OmpR is at the top of the regulatory hierarchy and binds upstream of the ssrA 35 hexamer ( 16 , 17 ) and directly stimulates transcription .3 The sensor kinase SsrA responds to unknown signals to presumably phosphorylate the response-regulator SsrB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21059643" target="_blank">21059643</a>]<br></li>
<li>When activated upon acidic pH , the SsrB response-regulator binds to the promoters of multiple operons within SPI-2 and promotes their expression ( 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24643535" target="_blank">24643535</a>]<br></li>
<li>In addition , the SsrB regulator also directly binds and autoregulates the ssrA and ssrB promoters to activate their expression ( 7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24643535" target="_blank">24643535</a>]<br></li>
<li>Similar to many other two-component regulatory systems , the expression of SsrA/SsrB is positively autoregulated ; the SsrB protein binds to ssrA and ssrB promoters and activates their transcription ( 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21059960" target="_blank">21059960</a>]<br></li>
<li>Due to these multiple positive feedback loops converging on SsrB expression , without a means to control the levels of activated SsrB protein , SPI-2 genes would be expressed ectopically within phag-osomes where the signals activating these response-regulators exist .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21059960" target="_blank">21059960</a>]<br></li>
<li>The mRNA level of srfN was reduced 8-fold in ssrB cells compared with the wild-type bacteria under SPI-2-inducing conditions and SsrB controls srfN directly through binding to the srfN cis-regulatory element .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28674150" target="_blank">28674150</a>]<br></li>
<li>Strain P8G12 defective in SsrB and unable to activate transcription of SPI2 genes was used as a negative control for comparing the protein patterns .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12112863" target="_blank">12112863</a>]<br></li>
<li>One of the ppGpp-de-pendent components in the signaling pathway leading to SPI2 activation was found to be SsrB , the response-regulator for SPI2 genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16905537" target="_blank">16905537</a>]<br></li>
<li>Our results indicate that , in contrast to many two-component regulatory systems , regulation of the sensor kinase SsrA appears to be uncoupled and distinct from regulation of the response-regulator SsrB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12753201" target="_blank">12753201</a>]<br></li>
<li>establish the extent to which the interaction of SsrB and RNS contributes to Salmonella pathogenesis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20660761" target="_blank">20660761</a>]<br></li>
<li>Similar to the role of EIIANtr for SsrB , the Rap protein Salmonella family members RapC , RapF , and RapH bind to the ComA protein and inhibit its binding to target promoter DNA ( 27 , 29 , 30 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21059960" target="_blank">21059960</a>]<br></li>
<li>This confirms that the absence of SsrB , and not SsrB ~ P , led to a defect in the ability of Salmonella to form surface-attached communities ( Figure 1 -- figure supplement 2 and 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26880544" target="_blank">26880544</a>]<br></li>
<li>Thus , the activation of srfH -- lacZ expression by SsrB , as shown in Figs 3 , 4 and 5 , was a direct effect of SsrB binding to the srfH regulatory region .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15491370" target="_blank">15491370</a>]<br></li>
<li>Here , an unidentified signal causes the sensor kinase SsrA to phosphorylate the DNA binding protein SsrB , which in turn activates transcription of SPI-2 T3SS and effector genes [ 3 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22457642" target="_blank">22457642</a>]<br></li>
<li>OmpR activates SPI-2 genes by binding to the promoter of the ssrAB operon to induce expression of SsrA and SsrB proteins [ 16 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20861532" target="_blank">20861532</a>]<br></li>
<li>Since PhoP regulates expression of ssrA and ssrB ( Bijlsma and Groisman , 2005 ) , it also regulates expression of genes in the SsrB regulon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29270156" target="_blank">29270156</a>]<br></li>
<li>Printed in the U.S.A. Salmonella enterica Response egulator SsrB Relieves R H-NS o M de S Received for publication , July 15 , 2010 , and in revised form , November 4 , 2010 Published , JBC Papers in Press , November 8 , 2010 , DOI 10.1074 / jbc.M110 .164962 Don Walthers ‡ , You Li § ¶ , Yingjie Liu § ¶ , Ganesh Anand , Jie Yan § ¶ , and Linda J. Kenney ‡ ¶ 1 From the ‡ Department of Microbiology and Immunology , University of Illinois , Chicago , Illinois 60612 and the § Department of Physics , Mechanobiology Institute , and Department of Biology , ¶ National University of Singapore , Singapore 119007 The response-regulator SsrB activates expression of genes encoded within and outside of a pathogenicity island ( SPI-2 ) , which is required for systemic infection of Salmonella .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21059643" target="_blank">21059643</a>]<br></li>
<li>A number of directly regulated gene products then regulate additional regulatory cascades , including PmrD , which is able to activate the PmrAB operon independent of iron ; SlyA , which regulates genes important to intra-macrophage survival such as pagC and ugtL ; IraP which prevents MviA-dependent degradation of RpoS leading to RpoS accumulation and its regulation of genes important for stationary phase survival and resistance to oxidative-stress ; HilA , which is an inducer of SPI-1 ( Salmonella pathogenicity island-1 ) , which contains genes involved in invasion of epithelial cells ; and SsrB , which is an inducer of SPI-2 containing genes important in intra-macrophage survival ( adapted from Groisman E. and Mouslim C. Nature Reviews Microbiology ( 2006 ) 4:705 -- 709 ) [ 44,45 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19746165" target="_blank">19746165</a>]<br></li>
<li>Yoon and colleagues also showed that inactivation of SPI-2 T3SS even led to increased protein expression and translocation levels of SrfN during Salmonella infection of macrophage cells ( 25 ) , arguing against its positive regulation by SsrB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28674150" target="_blank">28674150</a>]<br></li>
<li>In HeLa cells ( epithelial ) , where bacteria actively induce their own uptake using the SPI-1 T3SS , substantial induction of expression from PsseA occurred after 1 h in an SsrB - and an OmpR-dependent fashion .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16304611" target="_blank">16304611</a>]<br></li>
<li>Transcriptional reporter experi - ments , quantitative reverse transcription-PCR ( RT-PCR ) , and immunoblotting demonstrated that the virulence regulator SsrB and STM2239 contribute to transcriptional activation of genes in SPI-12 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23630960" target="_blank">23630960</a>]<br></li>
<li>The regulatory proteins PhoP and SlyA are involved in the activation of SsrB ( 3 , 37 ) , and we found that transcriptional activity upstream of sciF also increases in this mutant background ; however , sciG and vrgS transcriptional activity decreased , suggesting additional regulatory inputs to this system between PhoP and SsrB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22493086" target="_blank">22493086</a>]<br></li>
<li>To determine if SirA controls ssaG expression , and thus of other SPI-2 genes , by regulating the ssrAB operon , we analyzed the expression of SsrB in WT and ΔsirA strains , as well as in the ΔhilA , ΔhilD , ΔhilC , and ΔompR mutants , which were tested as controls .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21518393" target="_blank">21518393</a>]<br></li>
<li>CsgD levels were undetected SsrB and H-NS differentially regulate csgD expression Both in E. coli and S. Typhimurium , the expression of csgD is sensitive to various environmental stimuli such as starvation , temperature , pH and osmolality due to the action of upstream global regulators at the intergenic region of the csgDEFG and csgBAC operons .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26880544" target="_blank">26880544</a>]<br></li>
<li>Printed in the U.S.A. nalysis of the C-terminal DNA nella typhimurium SPI-2 Structural and Functional A Binding Domain of the Salmo Response Regulator SsrB * □ S Received for publication , August 13 , 2008 , and in revised form , December 18 , 2008 Published , JBC Papers in Press , January 5 , 2009 , DOI 10.1074 / jbc.M806261200 Ronan K. Carroll ‡ 1 , Xiubei Liao § 1 , Leslie K. Morgan ‡ , Elisha M. Cicirelli ‡ , Yuanhe Li § , Wanyun Sheng § , Xiuhong Feng ¶ , and Linda J. Kenney ‡ 2 From the Departments of ‡ Microbiology and Immunology and § Biochemistry and Molecular Genetics , University of Illinois , Chicago , Illinois 60612 and the ¶ Department of Molecular Microbiology and Immunology , Oregon Health and Science University , Portland , Oregon 97202 In bacterial pathogenesis , virulence gene regulation is controlled by two-component regulatory systems .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19126546" target="_blank">19126546</a>]<br></li>
<li>Based on studies of Salmonella grown in acidic minimal media [ 19 , 26 ] , we conﬁrmed the expected increases in abundance of proteins associated with the SPI-2 T3SS ( SsaC , SseA , and SsaJ ) , the SsrB regulon ( SsrA , SsrB , and SrfN ) , and the PhoP regulon ( PhoP , PhoQ , PagC , MgtA , and MgtB ) during growth in LPM ( Supplemental Table 7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22900174" target="_blank">22900174</a>]<br></li>
<li>Clearly , SsrA and SsrB regulation are uncoupled 1139 an enhanced understanding of the molecular mechanism of OmpR and SsrB activation and the underlying complexity in gene regulation at SPI-2 will probably reveal new insights into transcriptional activation in general .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12753201" target="_blank">12753201</a>]<br></li>
<li>Hybrid arrangements in which classical transcriptional activation and counter-silencing are combined also occur ( Fig. 1c ) , as in the co-regulation of SPI-2 pathogenicity island genes by OmpR and SsrB ( Walthers et al. , 2007 ) , or the co-regulation of the OmpS1 porin by OmpR and Leu regulation of OmpS1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21570144" target="_blank">21570144</a>]<br></li>
<li>Briefly , we demonstrated that SirA can not activate expression of SsrB and HilA proteins in the absence of HilD ( Figs. 2 , S1 and S2 ) , in agreement with results from a previous study ( Ellermeier et al. , 2005 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21518393" target="_blank">21518393</a>]<br></li>
<li>Also in accordance with our previous results ( Bustamante et al. , 2008 ) , expression of ssaG was reduced in both LB and N-MM in the ΔompR and ΔssrB mutants , which lack the SPI-2 regulators OmpR and SsrB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21518393" target="_blank">21518393</a>]<br></li>
<li>SPI-2 expression during growth in rich medium , which roughly mimics conditions in the intestinal lumen , requires OmpR and FIS but is independent of SsrB , SlyA , and PhoP [ 12,18 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22457642" target="_blank">22457642</a>]<br></li>
<li>Lucas and Lee ( 2001 ) , Schechter and Lee ( 2001 ) , Teplitski et al. ( 2003 ) Wilson et al. ( 2001 ) DNA bending protein required for site specific recombination of the flagellar phase variation protein hin ; regulates SPI-1 Transcriptional regulator containing a highly conserved domain of unknown function Two-component regulator that responds to low Mg and defensins SPI-2 encoded two-component regulator required for systemic infection Tunes regulation of SPI-2 more precisely than SsrA/SsrB alone ; controls regulation of many virulence factors Responds to cAMP levels which are determined in part by external glucose concentration Two-component regulator that responds to osmolarity Controls carbon metabolism fis l.p .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21687430" target="_blank">21687430</a>]<br></li>
<li>Coombes et al. ( 15 ) claimed that the expression of SPI2 TTSS and its effector was dependent on SsrB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20202329" target="_blank">20202329</a>]<br></li>
<li>Edited by Roy Curtiss , Arizona State University , Tempe , AZ , and approved Ju We show herein that the Salmonella pathogenicity island 2 ( SPI2 ) response-regulator SsrB undergoes S-nitrosylation upon exposure of Salmonella to acidiﬁed nitrite , a signal encountered by this enter-opathogen in phagosomes of macrophages .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20660761" target="_blank">20660761</a>]<br></li>
<li>Unphosphorylated SsrB activates csgD expression Biofilm formation in Escherichia coli and Salmonella Typhimurium is governed by the master regulator , CsgD , that acts as a transcriptional-activator of genes involved in curli biogenesis and cellulose synthesis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26880544" target="_blank">26880544</a>]<br></li>
<li>Interestingly , sciS transcription is negatively regulated by SsrB , a two-component regulator that activates salmo-nella pathogenicity island 2 ( SPI2 ) genes and additional genes outside of SPI2 that are primarily involved in promoting systemic infection of the host ( 4 , 6 , 13 , 41 , 43 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15972528" target="_blank">15972528</a>]<br></li>
<li>Two of the 8 genes , mcpA ( STM3138 ) and mcpC ( STM3216 ) , are predicted to encode Salmo-nella-specific methyl-accepting chemotaxis proteins [ 40 ] and display similar expression patterns to the SPI1-encoded , SPI1-associated and SPI4-encoded genes which are directly or indirectly activated by HilD , and repressed by SsrB , but are not regulated by HilA suggesting they are controlled by the HilD feed-forward loop .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27564394" target="_blank">27564394</a>]<br></li>
<li>Furthermore , STM2585 and STM1330 are regulated by SlyA and PhoP [ 44 ] , the master regulators of the SsrB regulon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27564394" target="_blank">27564394</a>]<br></li>
<li>It is significant that members of the SsrB regulon are regulated by both OmpR and PhoP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10844662" target="_blank">10844662</a>]<br></li>
<li>Transcription of srfJ is positively regulated by the two-component systems PhoP/PhoQ and SsrA/SsrB and negatively regulated by the RcsC/RcsD/RcsB phosphorelay system .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22661691" target="_blank">22661691</a>]<br></li>
<li>Initially , it was proposed as a putative effector because the gene srfJ is positively regulated by SsrB ( Worley et al. , 2000 ) , the main positive regulator of T3SS2 ( Fass and Groisman , 2009 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29270156" target="_blank">29270156</a>]<br></li>
<li>It is still unknown how the sensing of NO congeners by SsrB Cys203 enhances Salmonella fitness.It is possible that , by decreasing either Salmonella-induced apoptosis or reducing recognition by T and B lymphocytes , the tight control of SPI2 expression exerted by a redox active SsrB may increase intracellular survival or limit the specific immune response to SPI2 effectors .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21833325" target="_blank">21833325</a>]<br></li>
<li>It is interesting to speculate that srfN is initially repressed by H-NS in vitro , whereas during infection of host cells YdcR ( and SsrB ) may come into play and activate the expression of srfN .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28674150" target="_blank">28674150</a>]<br></li>
<li>Surprisingly , we found that , in addition to the regulation by SsrB , srfJ is also negatively regulated by IolR ( Cordero-Alba et al. , 2012 ) , the repressor of the myo-inositol ( MI ) utilization genes in S. Typhimurium ( Kröger and Fuchs , 2009 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29270156" target="_blank">29270156</a>]<br></li>
<li>Analysis of the expression patterns in the different genetic backgrounds ( Figure 2 ) indicates that IolR negatively regulates PiolE , whereas PsrfJ is positively regulated by PhoP and SsrB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29270156" target="_blank">29270156</a>]<br></li>
<li>PhoP positively regulates SPI2 through SsrB ( Bijlsma and Groisman , 2005 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29270156" target="_blank">29270156</a>]<br></li>
<li>Both islands are regulated by PhoP , that positively regulates SPI2 through SsrB and negatively regulates SPI1 through HilA ( Bajaj et al. , 1996 ; Bijlsma and Groisman , 2005 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25972862" target="_blank">25972862</a>]<br></li>
<li>To address this hypothesis , we performed a global screen for genes outside SPI-2 that are activated by SsrB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10844662" target="_blank">10844662</a>]<br></li>
<li>Ten arbitrarily chosen fusions were characterized here ; however , nearly 100 independent , retransduced fusions that are unlinked to SPI-2 were identified as being regulated by plasmid-encoded SsrB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10844662" target="_blank">10844662</a>]<br></li>
<li>SsrB negatively regulates SciS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15972528" target="_blank">15972528</a>]<br></li>
<li>srfH induction by SsrB in murine spleen .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10844662" target="_blank">10844662</a>]<br></li>
<li>In vivo , srfH is activated 166-fold by SsrB in murine spleen .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10844662" target="_blank">10844662</a>]<br></li>
<li>srfH is activated nearly 100-fold by SsrB in both cultured epithelial and macrophage cells .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10844662" target="_blank">10844662</a>]<br></li>
<li>In SsrB , a negatively charged glutamic acid res - These results demonstrate that Lys179 and Met186 are both idue replaces Lys188 of NarL , whereas in RcsB , it is replaced by a important for sseI promoter activation by SsrB , whereas Thr183 serine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19126546" target="_blank">19126546</a>]<br></li>
<li>In contrast , spiC expression is strongly affected by SsrB and less so by OmpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12753201" target="_blank">12753201</a>]<br></li>
<li>sciS is more highly transcribed at 24 h postinfection and is negatively regulated by SsrB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15972528" target="_blank">15972528</a>]<br></li>
<li>A proximal promoter , PsrfJ , responds to intravacuolar signals and is positively regulated by SsrB and PhoP and negatively regulated by RcsB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29270156" target="_blank">29270156</a>]<br></li>
<li>Expression of srfJ is regulated positively by SsrB and PhoP and negatively by RcsB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22661691" target="_blank">22661691</a>]<br></li>
<li>FIG 4 srfJ is regulated by PhoP , SsrB , and RcsB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22661691" target="_blank">22661691</a>]<br></li>
<li>Surprisingly , this regulon is activated by SsrB , a regulator encoded within SPI-2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10844662" target="_blank">10844662</a>]<br></li>
<li>ssaG expression is strongly controlled by SsrB [ 33 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21176126" target="_blank">21176126</a>]<br></li>
<li>DalS is regulated by SsrB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22418438" target="_blank">22418438</a>]<br></li>
<li>SsrB is phosphorylated by phosphoramidate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15491370" target="_blank">15491370</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15491370" target="_blank">15491370</a>]<br></li>
<li>However , it is not known if SsrB directly or indirectly regulates sciS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15972528" target="_blank">15972528</a>]<br></li>
<li>However , it is not known whether SsrB regulates GrvA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15972528" target="_blank">15972528</a>]<br></li>
<li>It is formally possible that SsrB could be activated independently from SsrA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12949164" target="_blank">12949164</a>]<br></li>
<li>In the results presented herein , it is apparent that SsrB has adapted to relieving H-NS-silencing and that phosphorylation is not required for this behavior , nor is it required for DNA bind ¬ ing ( Figure 5B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26880544" target="_blank">26880544</a>]<br></li>
<li>SlrP is another T3SS2 effector that is not regulated by SsrB ( 56 , 57 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23504014" target="_blank">23504014</a>]<br></li>
<li>Because SsrB requires its active site aspartate D56 for activity even when it is overexpressed ( Feng et al. , 2004 ) , these observations suggest that SsrB is phosphorylated by alternate phosphodonors when SsrA is not present or when its kinase function is not stimulated ( see Discussion ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630976" target="_blank">17630976</a>]<br></li>
<li>This result suggests that IolR can repress srfJ when SsrB is inactive but not when SsrB is active ( LB -- 0.3 M NaCl versus LPM ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22661691" target="_blank">22661691</a>]<br></li>
				</div></ul><br>
			<label class="collapse_DOM" for="_2"><font size="5px">SsrB protein domains</font></label>
				<input id="_2" type="checkbox">
				<div class="DOM"><br><ul><li>( A ) Motif consensus logo for SsrB recognition motif was generated using Weblogo software ( http://weblogo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27564394" target="_blank">27564394</a>]<br></li>
<li>Future investigations will be needed to determine whether SsrA , SsrB , EnvZ , OmpR , or SlyA contains redox active residues that impart regulatory functions to these proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19737903" target="_blank">19737903</a>]<br></li>
<li>Next , we investigated EIIANtr -- SsrB interaction in vitro using puriﬁed proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21059960" target="_blank">21059960</a>]<br></li>
<li>SsrBC-DNA Interactions and Contact Residues -- Thus far , we have identified 10 genes that are directly activated by SsrB ; they are AT-rich but do not contain a well conserved consensus site ( 8 , 10 , 12 , 13 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19126546" target="_blank">19126546</a>]<br></li>
<li>Release SsrB srfN ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19245712" target="_blank">19245712</a>]<br></li>
<li>DNA-recognition helix substitutions affect SsrB activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19126546" target="_blank">19126546</a>]<br></li>
<li>t002 Previous attempts by others to identify a conserved SsrB DNA-recognition motif have been unsuccessful .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20300643" target="_blank">20300643</a>]<br></li>
<li>Lanes 1 -- 5 contain cytoplasmic extracts from cultures grown at pH 5.7 ; lanes 6 -- 10 contain cytoplasmic extracts from cultures grown at pH 7.4 and lanes 11 and 12 contain 5 ml of 75 mM puriﬁed SsrB protein as a positive control .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15491370" target="_blank">15491370</a>]<br></li>
<li>Plasmids pKF43 ( full-length SsrB ) and pKF104 ( C-terminal domain ) were used as templates ( 13 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19126546" target="_blank">19126546</a>]<br></li>
<li>This cysteine in SsrB is not conserved in other 203 192 a glutamic acid substitution reduced activity to 34 % .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19126546" target="_blank">19126546</a>]<br></li>
<li>SsrB appears to be similar in this regard .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15491370" target="_blank">15491370</a>]<br></li>
<li>The C-ter-minal DNA-binding domain plus linker region of SsrB ( SsrBc ) ( bases 420 -- 644 ) was cloned in pET3a using primers SsrBc-HISF and SsrBc-HISR as a NdeI/BamHI fragment to generate C-terminal 6 His tag ( SsrBc-6HIS ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22418438" target="_blank">22418438</a>]<br></li>
<li>Because SsrB requires phosphorylation to relieve inhibitory contacts that occlude its DNA-binding domain , additional components must phosphorylate SsrB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630976" target="_blank">17630976</a>]<br></li>
<li>The SsrB binding motif architecture is flexible and conserved in Sodalis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20300643" target="_blank">20300643</a>]<br></li>
<li>The primary structures of SsrB , NarL , and RcsB DNA-binding domains are highly homologous ( Fig. 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19126546" target="_blank">19126546</a>]<br></li>
<li>RcsB and NarL both share one conserved DNA contact residue , as do RcsB and SsrB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19126546" target="_blank">19126546</a>]<br></li>
<li>To determine whether SsrB contains redox-active cysteines , strain AV07104 expressing a Cterminal 3xFLAG-tagged SsrB variant was constructed using the λ Red system .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20660761" target="_blank">20660761</a>]<br></li>
<li>The effect of dimerization helix substitutions on SsrB activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19126546" target="_blank">19126546</a>]<br></li>
<li>Substitution of cysteine at position 203 with alanine had no effect on SsrB activity , indicating that disulfide bond formation is not required for activity ( Fig. 9 , column 7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19126546" target="_blank">19126546</a>]<br></li>
<li>SsrB is unique within this family in that it contains two adjacent cysteine residues within its N-terminal domain , one of which is essential for function ( R.K. Carroll et al. unpublished ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630976" target="_blank">17630976</a>]<br></li>
<li>Thus , SsrB appears to have low sequence speciﬁcity for DNA , perhaps because few residues in the recognition helix make direct base contacts ( R.K. Carroll et al. unpublished ) , which might be important for its role as an antirepressor .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630976" target="_blank">17630976</a>]<br></li>
<li>We examined real-time interaction between EIIANtr and SsrB using the surface plasmon resonance technique .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21059960" target="_blank">21059960</a>]<br></li>
<li>Comparable levels of SsrB protein were interaction surface and residues involved in RcsB-RcsA het - detected with each mutant ( see example Fig. 9 , inset ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19126546" target="_blank">19126546</a>]<br></li>
<li>This cloning strategy resulted in a protein HisSsrB ( 219 amino acids ) that contained a methionine residue followed by six histidine residues at the N-terminus as well as amino acids 2 -- 212 of SsrB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15491370" target="_blank">15491370</a>]<br></li>
<li>The expression of cysteine or tyrosine residues at position 203 of SsrB raises the intriguing possibility that a redox-active moiety at this position provides Salmonella with a selective advantage .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20660761" target="_blank">20660761</a>]<br></li>
<li>SsrB and its mutant deriva-tact residues of SsrB are indicated by arrows .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19126546" target="_blank">19126546</a>]<br></li>
<li>We used the SsrB protein carrying thioredoxin fusion at its N terminus because this fusion facilitated overexpression of SsrB and increased SsrB solubility for protein puriﬁcation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21059960" target="_blank">21059960</a>]<br></li>
<li>Miao and Miller ( 2000 ) found that SifA is regulated by the SPI-2 SsrA-SsrB two component system and that it contains the conserved N-terminal amino acid sequence discussed above .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12864810" target="_blank">12864810</a>]<br></li>
<li>( C ) Aligned promoter regions from the S. Typhimurium SPI-2 island and S. glossinidius SSR-3 island containing the putative SsrB DNA-recognition palindrome .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20300643" target="_blank">20300643</a>]<br></li>
<li>Receiver domain Output domain Activity ( % of wt ) 200 150 100 50 1 SsrB 100 D56A 2 D56A 2 112-212 4 120-212 55 129-212 128 138-212 52 147-212 Amino acid residues important for SsrB activity By comparing the amino acid sequence of SsrB to other response regulators , the highly conserved phosphorylated aspartic acid is most probably D56 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15491370" target="_blank">15491370</a>]<br></li>
<li>This effector is integrated into the intracellular SsrB regulatory cascade , where it plays an important role in virulence in the mouse model of S. enterica serovar Typhimurium infection .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17158898" target="_blank">17158898</a>]<br></li>
<li>SsrB is in the NarL/FixJ subfamily , which possess a helix-turn-helix ( HTH ) motif in the Cterminus ( Baikalov et al. , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26880544" target="_blank">26880544</a>]<br></li>
<li>Furthermore , SsrB appears to be inhibited in the unphosphorylated state via interactions with its alpha helix six in the receiver-domain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15491370" target="_blank">15491370</a>]<br></li>
<li>can dimerize with a wild-type SsrB monomer and that the het - Analytical ultracentrifugation measurements confirmed the erodimer exhibits enhanced wild-type activity ( Fig. 10 , com - presence of a disulfide-linked dimer , which was present at a pare columns 3 and 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19126546" target="_blank">19126546</a>]<br></li>
<li>The DNA-binding domain of SsrB alone can function as a tran-C scription factor in vivo ( 13 ) , but the DNA-binding domain of NarL can not , even though it is capable of DNA-binding ( 17 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19126546" target="_blank">19126546</a>]<br></li>
<li>Based on homology to NarL , SsrB likely binds to DNA as a dimer by inserting a recognition helix from each monomer into subsequent major grooves perpendicular to the axis of the DNA with additional ﬂanking protein -- DNA interactions occurring along a total length of 20 nucleotides .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630976" target="_blank">17630976</a>]<br></li>
<li>N2O3-mediated oxidation of the Cys203 thiol group may have mechanistic consequences for binding of SsrB to DNA , similar to those seen in ONOO − - treated protein or an SsrB Asp203 variant .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20660761" target="_blank">20660761</a>]<br></li>
<li>The CTD of RNAP was dispensable for activation , which suggests a CRP-type class II activation mechanism ( 41 ) and is in agreement with our finding that SsrB binding sites likely overlap the putative 35 hexamers of each promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21059643" target="_blank">21059643</a>]<br></li>
<li>Yet surprisingly little is known about how it Lys179 and Met186 are in the DNA-recognition helix of SsrB and recognizes DNA and relieves H-NS silencing to activate tran-are predicted to be DNA contact residues .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19126546" target="_blank">19126546</a>]<br></li>
<li>In the co-crystal The absolute requirement for Val197 and Leu201 indicates that they structure of the NarLC-DNA complex , two hydrophobic resi-204 208 are important for SsrB activity , and their alignment with dimeriza-dues , Val and Val , reside on the surface of the dimerization helix and engage in intermolecular interactions to form a tion positions in DosR , NarL , and RcsB suggests they are involved homodimer on the DNA-binding site ( 17 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19126546" target="_blank">19126546</a>]<br></li>
				<label class="DOM_collapse_similarity" for="dom_similarity_41">
We identified residues in the DNA rec-ognition helix ( Lys179 , Met186 ) and the dimerization interface ( Val197 , Leu201 ) that are important for SsrB transcriptional activation and DNA-binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19126546" target="_blank">19126546</a>]<br></label>
<input id="dom_similarity_41", type="checkbox">
<div class="DOM_similar">
<li>We identified resi-dues in the DNA-recognition helix and the dimerization interface that are important for SsrB transcriptional activation and DNA-binding .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19126546" target="_blank">19126546</a>]<br></li></div>
<li>The C-terminal 75 amino acid residues of SsrB ( SsrBC ) fold into a four-helix bundle , and the SsrBC dimerization surface is similar to that of DosR , NarL , and RcsB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19126546" target="_blank">19126546</a>]<br></li>
<li>An alignment of the recognition helix ( H3 ) of SsrB and its homologues , NarL and RcsB , is shown in Fig. 5A along with other NarL family members .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19126546" target="_blank">19126546</a>]<br></li>
<li>Sequence alignment of the DNA contact helix and dimerization have shown that sseI is highly dependent on SsrB for transcrip-helix of SsrB with NarL family members .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19126546" target="_blank">19126546</a>]<br></li>
<li>A , primary sequence alignment of tional activation ( 11 , 13 ) , and DNase I footprinting has identi-DNA contact helix ( 9/H3 ) of SsrB with NarL family members for which the fied an SsrB binding site between 122 and 95 upstream of structure is known .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19126546" target="_blank">19126546</a>]<br></li>
<li>H2 and H3 comprise the helix-turn-helix DNA-binding domain , whereas H4 corresponds to the dimerization domain of both NarL and RcsB ( 17 , 20 ) as well as SsrB ( this work ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19126546" target="_blank">19126546</a>]<br></li>
<li>This vicinal configuration and distribution of cystuted with an alanine or a serine , this interaction is likely mainteine residues is unique to SsrB response regulator proteins in tained , whereas a residue with a negative charge at this position Salmonella serovars typhimurium and typhi .5 The presence of a can destabilize the helix , disrupting the dimer as well as DNA cysteine residue within H4 raised the possibility that it could binding ( see Fig. 9 and also `` Discussion '' ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19126546" target="_blank">19126546</a>]<br></li>
<li>Disruption of the dimer-interface by substitution of Cys203 with a negatively charged residue substantially reduces SsrB activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19126546" target="_blank">19126546</a>]<br></li>
<li>We previously reported the genome-wide interaction map of SsrB using chromatin immunoprecipitation and found only two interaction sites within the SPI-6 T6SS , upstream of sciF and sciR ( 51 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22493086" target="_blank">22493086</a>]<br></li>
<li>In place of with a weak effect from a residue predicted to make nonspecific the conserved Val189 of NarL , SsrB has a threonine and RcsB a interactions with the phosphate backbone ( Fig. 7 , column 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19126546" target="_blank">19126546</a>]<br></li>
<li>Site-directed mutagenesis and deletion mutagenesis of ssrB To replace the aspartic acid residue at position 56 of SsrB with alanine , the QuickChange site-directed mutagenesis kit from Stratagene was used , the reaction was performed with the olignonucleotides D56AF and D56AR ( Table 2 ) and pKF43 was used as template .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15491370" target="_blank">15491370</a>]<br></li>
<li>What are the essential features of an SsrB binding site ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15491370" target="_blank">15491370</a>]<br></li>
<li>A Cterminal fragment that contained all of alpha helix ﬁve , alpha helix six and the interdomain linker was not active ( amino acids 112 -- 212 , 2 % activity ) , nor was a construct containing the C-terminal residue of alpha helix ﬁve to the end of SsrB ( amino acids 120 -- 212 , 4 % activity ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15491370" target="_blank">15491370</a>]<br></li>
<li>It is presently not known as to whether activation by SsrB requires the C-terminal domain of the alpha subunit of RNA polymerase ( a CTD ) , as does OmpR ( Igarashi et al. , 1991 ; Slauch et al. , 1991 ) , or whether it makes contact with the sigma subunit of RNA polymerase , as is the case with the OmpR-family member PhoB ( Makino et al. , 1993 ; 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15491370" target="_blank">15491370</a>]<br></li>
<li>SsrA is a tripartite mem-brane-bound histidine sensor kinase that undergoes a series of intra-molecular phosphorylation reactions before it transfers the phosphoryl group to the N-terminal aspartate residue of the response regulator , SsrB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26880544" target="_blank">26880544</a>]<br></li>
<li>These complex domain structures suggest a phosphorelay whereby the phosphoryl group is transferred via intramolecular reactions from the histidine to an aspartate , to a histidine , and then onto the conserved aspartate of the SsrB response regulator ( 8 , 9 , 13 , 15 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19126546" target="_blank">19126546</a>]<br></li>
<li>The location of the four - helices from the SsrB three-dimensional structure are labeled H1 -- H4 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19126546" target="_blank">19126546</a>]<br></li>
<li>Our NMR structures indicate that the residues on the surface of H4 are different from the other SsrB family members .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19126546" target="_blank">19126546</a>]<br></li>
<li>We directed our efforts to the study of the DNA-binding , SsrB C-terminal domain ( SsrBc ) that encompasses amino acids 137 -- 212 , including Cys203 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20660761" target="_blank">20660761</a>]<br></li>
<li>We used atomic force microscopy ( AFM ) to visualize the csgDEFG-csgBAC intergenic region and examined the effect of addition of full-length SsrB , D56A SsrB and SsrBc ( DNA-binding domain alone ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26880544" target="_blank">26880544</a>]<br></li>
<li>We fused the DNA-binding domain of SsrB ( SsrBc ) to the asubunit of RNA polymerase and screened a prey library of ,108 DNA molecules previously counter-selected against self-activation ( Figure 4A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20300643" target="_blank">20300643</a>]<br></li>
<li>The experiment was performed under the recognition helix do not alter the ability of SsrB to dimer-SsrB-inducing conditions so that both forms of the protein ize .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19126546" target="_blank">19126546</a>]<br></li>
<li>Transcription -- To identify which of the predicted DNA contact residues are important for SsrB function , we performed site-directed mutagenesis and determined the ability of the mutant proteins to activate transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19126546" target="_blank">19126546</a>]<br></li>
<li>This result also suggests ture , Ile199 and Asn203 occupy the corresponding positions of that dimerization is required for SsrB activation of the sseI pro-the two valines of NarL .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19126546" target="_blank">19126546</a>]<br></li>
<li>In the present work , we solved the structure of the C-terminal DNA-binding domain of SsrB ( SsrBC ) by NMR and analyzed the effect of amino acid substitutions on function .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19126546" target="_blank">19126546</a>]<br></li>
<li>An alignment of the primary structure of the DNA-binding domains of SsrB and its homo - no conserved residues , raising the logues .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19126546" target="_blank">19126546</a>]<br></li>
<li>The N-terminal receiver-domain of SsrB contains the conserved aspartic acid phosphorylation site and the C-ter-minal effector domain binds DNA ( 13 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19126546" target="_blank">19126546</a>]<br></li>
<li>Carroll RK , et al. ( 2009 ) Structural and functional analysis of the C-terminal DNA-binding domain of the Salmonella typhimurium SPI-2 response regulator SsrB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20660761" target="_blank">20660761</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21059960" target="_blank">21059960</a>]<br></li>
<li>In order to test the effect of alpha helix ﬁve on SsrB function , we made progressive deletions , expressed the various constructs in the Salmonella strain lacking ssrB and measured srfH -- lacZ b-galactosidase activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15491370" target="_blank">15491370</a>]<br></li>
<li>To determine which resi-dues in H4 are important for SsrB dimerization , the potential dimerization residues were substituted independently with alanine and the transcriptional-activity of the mutant proteins determined in the sseI-lacZ fusion strain ( Fig. 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19126546" target="_blank">19126546</a>]<br></li>
<li>All of the critical residues in the dimerization helix and HTH motif required for specific transcriptional-activity by SsrB [ 32 ] are conserved in the Sodalis ortholog ( Figure S3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20300643" target="_blank">20300643</a>]<br></li>
<li>Sequence analysis of SsrB identified 203 NMR structure , the side chain of Cys interacts with Leu two additional cysteines located in the N terminus at positions located in the loop between H3 and H4 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19126546" target="_blank">19126546</a>]<br></li>
<li>Combining our data with previous biochemical work , along with comparative genomic analyses with an orthologous T3SS-encoding genomic island in the tsetse fly endosymbiont , Sodalis glossinidius , we reveal the flexible DNA palindrome that distributes SsrB in the genome to influence transcriptional activation of the SPI-2 T3SS and almost all of its accessory effector proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20300643" target="_blank">20300643</a>]<br></li>
<li>It also appears that the N-terminus of SsrB inhibits the activity of its C-terminus , as domain liberation increases its activity compared with the fulllength , unphosphorylated protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15491370" target="_blank">15491370</a>]<br></li>
<li>The widespread occurrence of a cysteine at this position in SsrB of medically important Salmonella isolates indicates that RNS exert a positive selective pressure in the effector domain of SsrB in both typhoidal and nontyphoidal strains of Salmonella enterica .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20660761" target="_blank">20660761</a>]<br></li>
<li>Role of Cysteine Residues in SsrB Function -- Our analysis of the dimerization helix of SsrBC identified two residues impor-the role of all three cysteine residues in SsrB transcriptional-activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19126546" target="_blank">19126546</a>]<br></li>
<li>In silico analysis revealed the presence of three cysteines in SsrB , raising the possibility that these residues could be targets of RNS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20660761" target="_blank">20660761</a>]<br></li>
<li>N O and dinitrosyl -- iron complexes ( DNIC ) , which frequently target 2 3 thiol-containing proteins , are common sources of nitrosative stress.The redox active thiol of Cys203 in the dimerization domain of the SsrB response regulator that controls SPI2 gene transcription is a bona fide sensor of nitrosative stress in Salmonella .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21833325" target="_blank">21833325</a>]<br></li>
<li>We reasoned that the presence of a redox-sensitive cysteine in the dimerization domain of SsrB may prove advantageous at some point during the association of this enteropathogen with its vertebrate host .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20660761" target="_blank">20660761</a>]<br></li>
<li>Mutational analysis has identiﬁed Cys203 in the C-terminal dimerization domain of SsrB as the redox-active residue responding to nitric oxide ( NO ) conge-ners generated in the acidiﬁcation of nitrite .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20660761" target="_blank">20660761</a>]<br></li>
<li>Mechanism of SsrB transcriptional activation The mechanism of action in the SsrB activation of transcription is similar to the subfamily member NarL , in that phosphorylation appears to relieve inhibition of the C-terminal effector domain ( Fig. 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15491370" target="_blank">15491370</a>]<br></li>
<li>In this respect , Cys203 in the C-terminal H4 helix of the dimerization domain of SsrB serves a function analogous to that of OxyR Cys199 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20660761" target="_blank">20660761</a>]<br></li>
<li>2 2 In conclusion , Cys203 in the C-terminal dimerization domain of the SsrB response regulator is a sensor of RNS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20660761" target="_blank">20660761</a>]<br></li>
<li>We constructed two plasmids in which EIIANtr and SsrB were expressed as forms fused to the T25 and T18 catalytic domains , respectively , of Bordetella pertussis adenylate cyclase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21059960" target="_blank">21059960</a>]<br></li>
<li>We were particularly interested in residues that , when mutated , had no effect on the low-level signaling at pH 7 ( suggesting correct localization of SsrA and its ability to interact with SsrB ) , but which compromised the acidpromoted activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25442048" target="_blank">25442048</a>]<br></li>
<li>- Galactosidase assays were per - function and indicates that neither formed in strain MJW604 expressing SsrB from an arabinose-inducible plasmid to determine the affect of 46 203 cysteine substitutions on SsrB activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19126546" target="_blank">19126546</a>]<br></li>
				</div></ul><br>
			<label class="collapse_EVO" for="_3"><font size="5px">SsrB evolution</font></label>
				<input id="_3" type="checkbox">
				<div class="EVO"><br><ul><li>Additionally , a number of SPI-2 regulators were found to be signiﬁcantly upregulated , which include SsrA/SsrB , EnvZ/OmpR , and SlyA , indicating a similarity to conditions inside the SCV ( 42 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26386064" target="_blank">26386064</a>]<br></li>
<li>SsrB is homologous at the primary sequence level to several other two-component regulators for which structures have been determined , including NarL , RcsB , and DosR ( 16 , 18 -- 21 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19126546" target="_blank">19126546</a>]<br></li>
<li>The primary structures of SsrB , NarL , and RcsB DNA binding domains are highly homologous ( Fig. 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19126546" target="_blank">19126546</a>]<br></li>
<li>SsrB , a member of the NarL protein family , activates transcription by displacing H-NS but does not appear able to break H-NS bridges [ 44 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22457642" target="_blank">22457642</a>]<br></li>
<li>DOI : 10.7554 / eLife .10747.019 Unphosphorylated SsrB binds to the csgD regulatory region SsrB is a NarL family member and X-ray crystallography suggested that phosphorylation was required for DNA binding ( Baikalov et al. , 1996 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26880544" target="_blank">26880544</a>]<br></li>
<li>In the best studied member of this of SsrB strongly induced sseI ( Fig. 7 , column 2 , set to 100 % ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19126546" target="_blank">19126546</a>]<br></li>
<li>An alignment of the recognition helix ( H3 ) of SsrB and its homologues , NarL and RcsB , is shown in Fig. 5A along with other NarL family members .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19126546" target="_blank">19126546</a>]<br></li>
<li>Sequence alignment of the DNA contact helix and dimerization have shown that sseI is highly dependent on SsrB for transcrip-helix of SsrB with NarL family members .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19126546" target="_blank">19126546</a>]<br></li>
<li>A , primary sequence alignment of tional activation ( 11 , 13 ) , and DNase I footprinting has identi-DNA contact helix ( 9/H3 ) of SsrB with NarL family members for which the fied an SsrB binding site between 122 and 95 upstream of structure is known .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19126546" target="_blank">19126546</a>]<br></li>
<li>The expression of the SPI-1 secretion system and the expression of its secreted effectors are known to be coordinately regulated by HilA , a member of the OmpR/ToxR family of transcriptional regulators , in S. Typhimurium ( Lucas and Lee , 2000 ) and the expression of genes encoding the type III secretion system of SPI-2 is dependent on a sensor-kinase system , SsrA-SsrB ( Lee et al. , 2000 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21046347" target="_blank">21046347</a>]<br></li>
<li>Mechanism of SsrB transcriptional activation The mechanism of action in the SsrB activation of transcription is similar to the subfamily member NarL , in that phosphorylation appears to relieve inhibition of the C-terminal effector domain ( Fig. 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15491370" target="_blank">15491370</a>]<br></li>
<li>SsrB is in the NarL subfamily of response regulators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26880544" target="_blank">26880544</a>]<br></li>
				<label class="EVO_collapse_similarity" for="evo_similarity_13">
SsrB is a member of the NarL / FixJ subfamily of DNA-binding response regulators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630976" target="_blank">17630976</a>]<br></label>
<input id="evo_similarity_13", type="checkbox">
<div class="EVO_similar">
<li>SsrB is a member of the NarL/FixJ subfamily of response regulators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630976" target="_blank">17630976</a>]<br></li></div>
<li>Our NMR structures indicate that the residues on the surface of H4 are different from the other SsrB family members .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19126546" target="_blank">19126546</a>]<br></li>
<li>SsrB belongs to the NarL/FixJ family of transcriptional regulators that require phosphorylationdependent dimerization to bind DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26880544" target="_blank">26880544</a>]<br></li>
<li>Similar to the role of EIIANtr for SsrB , the Rap protein Salmonella family members RapC , RapF , and RapH bind to the ComA protein and inhibit its binding to target promoter DNA ( 27 , 29 , 30 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21059960" target="_blank">21059960</a>]<br></li>
<li>Salmonella SsrB and the Sodalis ortholog ( SG1279 ) are 69 % identical and 81 % similar at the amino acid level .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20300643" target="_blank">20300643</a>]<br></li>
<li>These proteins belong to families of well-characterized transcription factors such as the AraC / XylS-like Hil proteins encoded by SPI-1 ( Schechter and Lee , 2001 ) and the two-component regulator pair SsrA and SsrB , encoded by SPI-2 ( Cirillo et al. , 1998 ; Hensel et al. , 1998 ; Garmendia et al. , 2003 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16999831" target="_blank">16999831</a>]<br></li>
				</div></ul><br>
			<label class="collapse_RP" for="_4"><font size="5px">Processes regulated by SsrB</font></label>
				<input id="_4" type="checkbox">
				<div class="RP"><br><ul><li>Cell envelope stress , osmolarity sensor , affects membrane permeability , nutrient availability and flagellar expression Nitrogen assimilation Zinc/lead efflux Potassium transport , osmolarity sensor OmpR EnvZ Bang et al. ( 2000 ) ; Higashitani et al. ( 1993 ) ; Shin & Park ( 1995 ) Zimmer et al. ( 2000 ) Leonhartsberger et al. ( 2001 ) Polarek et al. ( 1992 ) GlnG HydG KdpE NarP NarL GlnL HydH KdpD NarQ NarX Nitrate - and nitrite-responsive , anaerobic-respiration Nitrate - and nitrite-responsive , anaerobic-respiration Phosphoglycerate transport Mg uptake , peptide resistance , macrophage survival 2 + Inorganic phosphate uptake/production Stewart ( 2003 ) Stewart ( 2003 ) PgtA PhoP PhoB PgtB PhoQ PhoR Niu et al. ( 1995 ) Groisman & Mouslim ( 2006 ) Conlin et al. ( 2001 ) ; Wanner & Latterell ( 1980 ) Gunn ( 2008 ) LPS modification , peptide resistance PmrA RcsB PmrB RcsC YojN RstB SsrA Capsule synthesis in response to peptidoglycan stress Capsule synthesis in response to peptidoglycan stress Iron transport Control of T3SS-2 and effectors Laubacher & Ades ( 2008 ) Laubacher & Ades ( 2008 ) Jeon et al. ( 2008 ) RstA SsrB Cirillo et al. ( 1998 ) ; Geddes et al. ( 2007 ) ; Hensel et al. ( 1998 ) ; Shea et al. ( 1996 ) Winnen et al. ( 2003 ) Baraquet et al. ( 2006 ) Hensel et al. ( 1999 ) Tricarboxylate transport TctD TorR TtrR UhpA QseF TctE TorS TtrS UhpB QseE Trimethylamine N-oxide respiration Tetrathionate respiration Glucose 6-phosphate-transport Epinephrine , phosphate and sulphate sensor Island et al. ( 1992 ) Reading et al. ( 2009 ) Rasko et al. ( 2008 ) Yamamoto et al. ( 2009 ) Pontel & Soncini ( 2009 ) Quorum-sensing and host stress hormone sensing Citrate utilization Copper sensing Chemotaxis QseB CitB CusR CheY QseC CitA CusS CheA Analysis of the draft S. Dublin CT02021853 sequence indicated that two predicted SK genes may be pseudogenes based on frameshift mutations when compared with S. Typhimurium LT2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20656781" target="_blank">20656781</a>]<br></li>
<li>14028 wild-type ATCC [ 44 ] Spontaneous nalidixic acid resistant derivative of wild-type 14028 SH100 ΔrelA : : cat ΔspoT : : kan SH100 ΔrelA : : cat ΔspoT : : kan ΔssrB : : tet SH100 Δstm3169 : : kan SH100 stm3169 : : lacZ TM157 stm3169 : : lacZ TH1164 ΔssrB : : tet SH100 TM157 YY2 TH973 TH1162 TH1164 YY3 TM129 this study this study this study this study this study this study SH100 ssaG : : lacZ SH100 ΔssrB : : tet SH100 ΔssaV : : cat SH100 ΔsseF : : kan this stu this stu [ 11 ] this stu dy dy YY1 SH113 TM548 E. coli DH5a SM10lpir Plasmids pGEM-Teasy dy K-12 recA1 endA1 gyrA96 thi-1 hsdR17 supE44 ( lacXYA-argR ) U169 deoR ( 80 dlac ( lacZ ) M15 ) thi-1 thr leu tonA lacY supE recA : : RP4-2-Tc : : Mu lpir Invirtogen [ 45 ] TA cloning vector Promega Nippon Gene pSC101-based low copy number plasmid p15A-based low copy number plasmid , tet template FLAG tag expression vector Integrational plasmid with promoterless lacZ gene Expression vector for His6 fusion protein stm3169 gene in pMW118 stm3169 : : lacZ operon fusion in pLD-lacZΩ pMW118 pACYC184 New England Biolabs Sigma [ 39 ] Invitrogen this study this study pFLAG-CTC pLD-lacZΩ pBAD-HisA pMW-Stm3169 pLD-stm3169Z pLD-ssaGZ ssaG : : lacZ operon fusion in pLD-lacZΩ pBAD-HisA expressing relA gene pFLAG-CTC expressing ssrB gene Red recombinase expression plasmid kan cassette used for gene deletion this stu this stu this stu [ 41 ] [ 41 ] dy dy dy pRelA pSsrB pKD46 pKD4 Table 3 Primers Name Nucleotide sequence ( 5 ' to 3 ' ) a Construction of the deletion mutants CGCCATCCCGCAATGGTTTACATAA TCATTGTTCTGGCCATAACAGC CTTGAAAACCATCATTCGCGCTGAACG TCTGCGGTACGAATGATTGCAGAAACG relA-FW relA-RV spoT-FW spoT-RV stm3169-red-FW stm3169-red-RW ACGTTCATTCACAACATCAGCGGTATTACTGGCCGGCTGTGTGTAGGCTGGAGCTGCTTC ACATATTCTCGATGTATTCCAGATCCTTCGCTGACTGAGCCATATGAATATCCTCCTTAG AACAGAACGAAATATGAAAATTCATATTCCGTCAGCGGCAGTGTAGGCTGGAGCTGCTTC TGTCCATTAATGCAGGTGTAGTAGCAGATTGACAGAGCGCCATATGAATATCCTCCTTAG TTGGTAGCTCAGAGAACCTTCGAAAAACCG TCGCTCGCGTATCGGTGATTCATTCGCTA plementations AGGCTCGAGGTCGCGGTAAGAAGT sseF-red-FW sseF-red-RW pAC-tet-FW pAC-tet-RV Construction of plasmids for the com relA-FW2 relA-RV2 ACAAGCTTACTGTCTGGGGTTTAC ssrB-FW GGGCTCGAGGAATATAAGATCTTATTAGT Table 3 Primers ( Continued ) ssrB-RV stm3169-FW CCCGGATCCATACTCTATTAACCTCATTCT CCGCTCGAGAACACACGTTCATTCACAACATCAG GGAAGATCTATTCTCGATGTATTCCAGATCCTTC stm3169-RV Construcion of the lacZ fusions ssaG-Pro-FW ssaG-Pro-RV stm3169-Pro-FW stm3169-Pro-RV AAAGTCGACCAAATGCTCAGGTAGGAGGGC AAAGGATCCATCATCGATTCTGGGTTGAGC ACGCGTCGACGACGATTTAGCCGGTATGAAAATCA CGGGATCCTTACATATTCTCGATGTATTCCAGA Comfirmation of gene-expression by qRT-PCR gyrA-FW AAGAGCTCCTATCTGGATTATGC gyrA-RV TATTTACCGATTACGTCACCAAC relA-FW ATTGTGCCATTCACCTATCAGTT relA-RV GATATTTTTGTCACGATCCTGCT invF-FW ATCGCTGCTGAATAGTGTAGAAG invF-RV CATTTGTCTGCCAATTGAATAAT stm0209-FW CCTGAACGTAGAAAATTACGAGA GTCAGGTTTTTCACCATGTTACT GTCAATCAGTTGCTCGATATTCTG TTTAATCAGCTTGACGATTTTCTTC TGAACCTGTACGTTATGGATCTC CGCCGTTAATGTTCATTTTATAC GAAGGCAAGATCACCGTATTT CTGATCAGCAGAGATGAAGAGAT ACAAAAGTTGAGGAGCTGAATAAAG stm0209-RV stm0435-FW stm0435-RV stm0748-FW stm0748-RV stm0781-FW stm0781-RV stm1478-FW GCCACTGACGCGTAATATGATAA TTGGTTGTAAAATTGAAGACAAAGG GTCCCCTTCAGGATAAAGGTGTAAT CGAATTATTCCAGAAACTGAAGAAA ATCGCCCTGATTTTTAACCTTATTA TATTCTGACGGTCTGTTTAGCTTTT GTACTGCCCTGAATTTGATACTGTC GTTACCAGAATAATGTCGCAGCTAT stm1478-RV stm1720-FW stm1720-RV stm1746-FW stm1746-RV stm2638-FW stm2638-RV stm3169-FW AATCATCCACATAAAAAGAATCTGG CAAACTCAGCCTTAATCTTATGC ACTTTATCGGCGTTGATCTTAAT ATTAGTATTATCCGAGGCCAGAC CAGTCTTGCAAACTCTACTGCTC ATTGATATTCACAGCAACAAACC AGGTCAGGTTTTTAATACGTTCC CGAACTGACGTTGAATAAAGATGAG stm3169-RV stm3318-FW stm3318-RV stm4319-FW stm4319-RV stm4403-FW stm4403-RV stm4405-FW aUnderlined part indicates P1 or P2 site for pKD4 , or restriction sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21176126" target="_blank">21176126</a>]<br></li>
<li>Interestingly , SsrB also regulates selected genes outside SPI2 , many of which may have been acquired through horizontal transfer ( Worley et al. , 2000 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15790293" target="_blank">15790293</a>]<br></li>
<li>Here , it is shown that SsrB in fact activates a global regulon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10844662" target="_blank">10844662</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_5">
Worley , M.J. , Ching , K.H.L. , and Heffron , F. ( 2000 ) Salmo-nella SsrB activates a global regulon of horizontally acquired genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12864810" target="_blank">12864810</a>]<br></label>
<input id="rp_similarity_5", type="checkbox">
<div class="RP_similar">
<li>Worley , M.J. , Ching , K.H. , and Heffron , F. ( 2000 ) Salmonella SsrB activates a global regulon of horizontally acquired genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12675800" target="_blank">12675800</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15491370" target="_blank">15491370</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12864852" target="_blank">12864852</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630976" target="_blank">17630976</a>]<br></li>
<li>Salmonella SsrB activates a global regulon of horizontally acquired genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17220229" target="_blank">17220229</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17526845" target="_blank">17526845</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24496791" target="_blank">24496791</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25972862" target="_blank">25972862</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19717598" target="_blank">19717598</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12949164" target="_blank">12949164</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22661691" target="_blank">22661691</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15972528" target="_blank">15972528</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19737903" target="_blank">19737903</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17616593" target="_blank">17616593</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29034217" target="_blank">29034217</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15790293" target="_blank">15790293</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23504014" target="_blank">23504014</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16513753" target="_blank">16513753</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11844786" target="_blank">11844786</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19011032" target="_blank">19011032</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26673132" target="_blank">26673132</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11133955" target="_blank">11133955</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17158898" target="_blank">17158898</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19858298" target="_blank">19858298</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15231802" target="_blank">15231802</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23630960" target="_blank">23630960</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15972530" target="_blank">15972530</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17635868" target="_blank">17635868</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16980468" target="_blank">16980468</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17060472" target="_blank">17060472</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15136033" target="_blank">15136033</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20974834" target="_blank">20974834</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26497459" target="_blank">26497459</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25249283" target="_blank">25249283</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16904940" target="_blank">16904940</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16129704" target="_blank">16129704</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29270156" target="_blank">29270156</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15567133" target="_blank">15567133</a>]<br></li></div>
				<label class="RP_collapse_similarity" for="rp_similarity_8">
[ PubMed : 12694612 ] Worley MJ , Ching KH , Heffron F. Salmonella SsrB activates a global regulon of horizontally acquired genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21518393" target="_blank">21518393</a>]<br></label>
<input id="rp_similarity_8", type="checkbox">
<div class="RP_similar">
<li>[ 23 ] Worley MJ , Ching KH , Heffron F. Salmonella SsrB activates a global regulon of horizontally acquired genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21320585" target="_blank">21320585</a>]<br></li>
<li>Worley MJ , Ching KH , Heffron F. Salmonella SsrB activates a global regulon of horizontally acquired genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11886586" target="_blank">11886586</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21984608" target="_blank">21984608</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12535274" target="_blank">12535274</a>]<br></li></div>
<li>SsrB regulon expression characteristics .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10844662" target="_blank">10844662</a>]<br></li>
<li>SsrB negatively regulates SciS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15972528" target="_blank">15972528</a>]<br></li>
<li>Additionally , it possessed no srfJ gene ( encoding a putative virulence factor activated by transcription factor SsrB ) , which was found only for this isolate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21705546" target="_blank">21705546</a>]<br></li>
<li>We used quantitative reverse transcription polymerase chain reaction ( qRT-PCR ) to demonstrate that SsrB has a modest effect in activating transcription in an hns background .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630976" target="_blank">17630976</a>]<br></li>
<li>SseB was used as a positive control as a gene product whose expression is dependent on SsrB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22418438" target="_blank">22418438</a>]<br></li>
<li>By this action on SsrB production , Fur was able to negatively control SPI-2 expression ( Fig. 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24643535" target="_blank">24643535</a>]<br></li>
<li>PhoR TCS TCS TCS TCS Colonization , invasion Colonization Colonization Colonization , invasion , host defense 99 98 72 48 34 92 QseB / YgiX QseC / YgiY RcsB 61 100 RcsB STM2269 RcsD / YojN RcsD 98 47 TCS Colonization , invasion , host defense Colonization Host defense Invasion , host defense Colonization , invasion , host defense Invasion Colonization , invasion RcsA LcrF STM4337 STM4048 STM1185 RﬂM / EcnR RhaS Rnase E/rne pYV0076 Transcription Transcription Translation 93 96 29 22 64 Rnase E 100 RscC RscC STM2271 99 58 TCS NA STM4315 STM4314 STM4314 STM1950 STM1950 RtsA RtsB RtsB SdiA SdiA Transcription Transcription NA 62 56 91 81 NA 41 35 29 28 YpsR YtbR SirA Transcription Transcription TCS Colonization Colonization 99 STM1947 SirA 84 Colonization , invasion , host defense Colonization , invasion , host defense STM1444 75 98 RovA SlyA Transcription 44 98 STM1392 SpiR TCS Host defense LcrH SsrB SscA STM1399 pYV0056 83 27 Translation Host defense SsrB STM1391 96 55 TCS Host defense Metabolic Adaptation by Global Regulatory Systems of the Carbon Metabolism Several global regulators , which are highly conserved between Salmonella and Yersinia , are known that govern multiple cascades of control elements and complex networks in order to manage metabolic adaptation and coordinate this process with pathogenicity mechanisms .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26441883" target="_blank">26441883</a>]<br></li>
<li>Choi J , Shin D , Yoon H , Kim J , Lee C , Kim M , Seok Y , Ryu S : Salmonella pathogenicity island 2 expression negatively controlled by EIIANtr-SsrB interaction is required for Salmonella virulence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23295139" target="_blank">23295139</a>]<br></li>
<li>Salmonella pathogenicity island 2 expression negatively controlled by EIIANtr-SsrB interaction is required for Salmonella virulence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23006851" target="_blank">23006851</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24643535" target="_blank">24643535</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27628932" target="_blank">27628932</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24835993" target="_blank">24835993</a>]<br></li>
<li>Marker group / Accession Gene no.a Gene function gene name number Forward primer Reverse primer Position ( CDS ) Pathogenicity avrA CCTGTATTGTTGAGCGTCTGG AGAAGAGCTTCGTTGAATGTCC 128/549 AF013573 STM2865 SPI 1 encoded protein , inhibits the key proinﬂammatory , antiapoptotic NF-kappa B pathway Gifsy-1 Peyer ´ s patch-speciﬁc virulence factor ACGACTGAGCAGGCTGAG GipA DT104 speciﬁc prophage encoded protein , GTCAATGACCATTGTTCTGTTC lipopolysaccharide assembly-related protein SPI 1 encoded invasion protein GTGAAATTATCGCCACGTTCGGGCAA Involved in macrophage survival ; intracellular AATCGGTGAGTGGATACACG response element to stress , encoded by prophage SPI 3 encoded putative transcriptional regulator , TGACTATCAATGCTCCAGTGAAT required for intramacrophage survival and growth in low Mg2 + TTSS translocated protein , chromosomal coded TGACAGAGGTGCTCCGTTG Fecal shedding factor GATGACACGCCTTCGGATG STM2599 AF246666 gipA TTGGAAATGGTGACGGTAGAC 34/455 AY462995 GTTATGCACGATGATTACACTATC 78/239 STM2896 M90846 AY328029 invA irsA TCATCGCACCGTCAAAGGAACC 287/571 TTCAGCACAGCCTTATAGACC 90/512 STM3764 AF106566 mgtC ATTTACTGGCCGCTATGCTGTTG 38/693 STM2780 STM2513 AY532917 pipB2 shdA CTTCCAGACAGGCGTTCTTG 210/646 GCCAACGGTAACGCTCTG 5029 / 5430 GCTTTGTCCACGCCTTTCATC 3/1245 SPI 4 encodes with SiiC SiiD and SiiF membrane GAATAGAAGACAAAGCGATCATC pore for SiiE , T1SS Leucine-rich repeat protein GTGAAGGACCTCAACCTACAAG STM4260 AJ576316 siiD ( ) STM0800 AF127079 slrP CAGTTCCGCTAAGTGGCTATC 1235 / 1734 GGTGCGCTCATCAGTTGTTC 48/465 Gifsy-2 prophage encoded , copper/zinc CCAGTGGAGCAGGTTTATCG superoxide dismutase Fels-1 putative Cu/Zn superoxide dismutase CAGCCTCAGCAGCAAGTAC precursor Secreted outer protein CCGTGAAGTTGATTGATGATGC STM1044 AF007380 sodC1 STM0924 AF254764 sodCIII GCGGGTTGGGATGATCTTC 47/481 STM2066 AF121227 sopA CAGGATGGATGACAGAACACTC 1904 / 2331 TACCGTCCTCATGCACACTC 195/712 SPI 5 encoded translocated effector protein TCAGAAGRCGTCTAACCACTC ( phosphoinositide phosphatase ) via TTSS Secreted outer protein TGTTACACTCCATCATTCACGG Translocated effector protein , encoded by P2-like CGGGCAGTGTTGACAAATAAAG cryptic bacteriophage Secreted outer protein TGAGGGTAGGGCGGTATTAAC Salmonella plasmid virulence : hydrophilic ACTCCTTGCACAACCAAATGCGGA protein , provides rapid growth and survival within the host Putative virulence factor , activated by TTACAGCAGATTGACGGATTCG transcription factor SsrB SPI 2 encoded secretion system apparatus protein GAATAGCGAATGAAGAGCGTCC Translocated effector protein , regulated by SPI 2 ACAGACAAGTCGCATCTTACAC STM1091 AE008747 sopB STM2945 AE008834 L78932 sopD sopE1 GTAAGTGAGTCCTGCCATTCG 163/623 TGTTGGAATTGCTGTGGAGTC 223/677 STM1855 AF200952 AE006471 sopE2 spvC CTGAAATTGTTGTGGCGTTGG 216/670 TGTCTTCTGCATTTCGCCACC 113/683 STM4426 AE008907 srfJ CGGTATTCCAGCAGGTAGTTG 124/545 STM1418 STM1631 AE008761 AF294582 CATCGTGTTATCCTCTGTCAGC AAATCAATCGCTCGCCGTAG ssaQ sseJ 8/651 674 / TCATAACACCCAACCTTATCCG GGAATCCTGCTCGCTTTGG STM4157 AE008894 Salmonella secreted effektor K1 , SPI-2 TTSS , not located on SPI Salmonella secreted effector K1 , putative SPI-2 TTSS , not located on SPI Leucine reach repeat protein , induced by SPI 2 regulator ssrA/B , located on remnant phage sseK1 ACGCTCAGTCCACCTTCATC STM2137 AE008795 GCAAGACCTCATCTCGTTCAC 100/511 sseK2 ACTGTCGCTGTCGGTCTATC STM2241 AE008800 TTGCTCTTAACGCCTCATCTTC 1272 / 1702 sspH2 Fimbrial lpfD pefA prot6E CCATTGCGAGCGATGTACTG CCTGTGACCTGACCACTTCTG GCCTAAGGTTAGTGTGACTCTC STM3637 AE008868 AB041905 U66901 Long polar ﬁmbrial operon protein TCATTAGCGTTTGGCATTCCC 344/840 TTGTAAGCCACTGCGAAAGATG 104/521 CTAGCAGCCGTTGGTATCC 258/836 Virulence plasmid encoded ﬁmbrial protein Serovar Enteritidis ﬁmbrial biosynthesis protein , located on virulence plasmid Putative ﬁmbriae usher protein CACGGCAGACCATACATCAAC GAAGTGGCAGGCAGCATATAC STM0301 AE008708 safC 651 / 1145 TTGATACTGCTGAACGTAGAAGG TGCGTTCGTGGTTACATGAC TCCAGGTATCTCGGTCCATTG AAGAGGCGAACCATCAGGAC sefA staA stbD stcC 92/493 148/561 177/637 1819 / 2298 671 / 1108 2128 / 90/556 1211 / 1701 1511 / STY4836a Fimbrial protein TTCAGGCAGCGGTTACTATTG STY0207 Fimbrial protein encoded by serovar Typhi CT18 GCGACTGATGGTGACGTATATC STM0337 Fimbial usher protein TTTGCCAACGGTGATTAGCG STM2150 Putative outer membrane protein AATTACGGCGTGAGTGTTAGTC L11008 AL627265 AE008710 AE008795 TGACCGTGCTGTTGTTATGC AE008839 STM3028 Putative outer membrane usher protein CGTATCTGTTCCGTGCCTTG stdB TTGCGTATGTTCCTGGATTAGC STY3086 AL627276 Outer membrane ﬁmbrial usher protein encoded GTGATTGCGGATGTCGGAAG by Serovar Typhi Putative minor ﬁmbrial subunit GGGAAGTAAGACGGTCAACATG Fimbrial protein encoded by serovar Typhi CT18 CCTTGGTGAAGTGGTTGACAG Putative ﬁmbrial usher protein AATACCAGTCCGTCGCCTTC steB GGTAATCCACCACCAACGTG TGAACTGAGAATCTGCGTTAGC CGTCGTCGTTGTCATCATCG STM0198 STY3918 STM0175 AE008703 AL627280 AE008702 stfE stgA stiC GGAAGCCATTATACTGCGAGAG AE008915 STM4572 Putative ﬁmbrial usher protein GTGATACCACCAGCGACTATTC stjB GCCGTGTGTTTCCCGTATTC AL627266 STY0345 Putative ﬁmbrial protein CACCTCGCTGTTCGTCTATTC tcfA Serotypin Table 2 ( continued ) Gene no.a Gene function Forward primer Reverse primer Position ( CDS ) Marker group / Accession gene name number ﬂjA AE008826 STM2770 Repressor of phase-1 ﬂagellin oafA AE008800 STM2232 Acetylation of the O5-antigen GATATTTCCTACGGGCGAGAAG ACGAAGCACTTAGCAAGAACG ATTGCCATTGGATGTCCTCAC 22/480 CAACAGCAACAACAATGAGGAC 368/757 Integron AF261825 715 / 1188 Integrase from Tn4555 , encoded by Salmonella genomic island 1 DNA Integrase1 : Integron associated DNA Integrase2 : Integron associated GCAGTGAGGACAGCAAGTTC AGATGGAGCGATATTCAGAAGC X12870 L10818 intI1 intI2 251/810 33/821 GATGCCTGCTTGTTCTAC GCCTTGCTGTTCTTCTAC GTAGCAAACGAGTGACGAAATG CACGGATATGCGACAAAAAGGT a Gene designation according to McClelland et al. ( 2001 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21093086" target="_blank">21093086</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_21">
Walthers D , et al. ( 2007 ) The response regulator SsrB activates expression of diverse Salmonella pathogenicity island 2 promoters and counters silencing by the nucleoid-associated protein H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21059960" target="_blank">21059960</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23690578" target="_blank">23690578</a>]<br></label>
<input id="rp_similarity_21", type="checkbox">
<div class="RP_similar">
<li>Walthers D , Carroll RK , Navarre WW , Libby SJ , Fang FC , et al. ( 2007 ) The response regulator SsrB activates expression of diverse Salmonella pathogenicity island 2 promoters and counters silencing by the nucleoid-associated protein H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19229334" target="_blank">19229334</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28192515" target="_blank">28192515</a>]<br></li></div>
				<label class="RP_collapse_similarity" for="rp_similarity_23">
The response regulator SsrB activates expression of diverse Salmo-nella pathogenicity island 2 promoters and counters silencing by the nucleoid-associated protein H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23554419" target="_blank">23554419</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24496791" target="_blank">24496791</a>]<br></label>
<input id="rp_similarity_23", type="checkbox">
<div class="RP_similar">
<li>The response regulator SsrB activates expression of diverse Salmo-nella pathogenicity island 2 promoters and counters silencing by the nu-cleoid-associated protein H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24643535" target="_blank">24643535</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25085489" target="_blank">25085489</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26497459" target="_blank">26497459</a>]<br></li>
<li>The response regulator SsrB activates expression of diverse Salmonella pathogenecity island 2 promoters and counters silencing by the nucleoid-associated protein H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20396961" target="_blank">20396961</a>]<br></li>
<li>The response regulator SsrB activates expression of diverse Salmonella pathogenicity island 2 promoters and counters silencing by the nucleoid-associated protein H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29057095" target="_blank">29057095</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20807201" target="_blank">20807201</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26441883" target="_blank">26441883</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29034217" target="_blank">29034217</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17981960" target="_blank">17981960</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19858298" target="_blank">19858298</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28270274" target="_blank">28270274</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24835993" target="_blank">24835993</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18039769" target="_blank">18039769</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25713562" target="_blank">25713562</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21570144" target="_blank">21570144</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21518393" target="_blank">21518393</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18160484" target="_blank">18160484</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21680637" target="_blank">21680637</a>]<br></li>
<li>Walthers , D. , Carroll , R. K. , Navarre , W. W. , Libby , S. J. , Fang , F. C. , and Kenney , L. J. ( 2007 ) The response regulator SsrB activates expression of diverse Salmonella pathogenicity island 2 promoters and counters silencing by the nucleoid-associated protein H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22418438" target="_blank">22418438</a>]<br></li>
<li>[ 45 ] D. Walthers , R.K. Carroll , W.W. Navarre , S.J. Libby , F.C. Fang , L.J. Kenney , The response regulator SsrB activates expression of diverse Salmonella patho-genicity island 2 promoters and counters silencing by the nucleoid-associated protein H-NS , Mol .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23040276" target="_blank">23040276</a>]<br></li></div>
<li>Experiments involving an approach called atomic force microscopy showed that the unmodified SsrB acts to stop a major gene that controls biofilm-formation from being switched off by a so-called repressor protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26880544" target="_blank">26880544</a>]<br></li>
<li>TABLE 2 Level 3 gene ontologies associated with proteins that sign Abundance Signiﬁcant SsrB ChIP score iﬁcantly change in abundance in the ssrB mutant Gene ontology term Pathogenesis Establishment of localization Primary metabolic-process Cellular metabolic-process Cellular localization Macromolecule localization Response to stress Macromolecule metabolic-process Cellular metabolic-process Primary metabolic-process Macromolecule metabolic-process Nitrogen compound-metabolic-process Biosynthetic process Small molecule metabolic-process Oxidation-reduction process Cellular metabolic-process Regulation of biological-process Establishment of localization Response to chemical stimulus Taxis Response to external stimulus Localization of cell Cellular component movement Cell cycle Cell division Cellular response-to-stimulus Cell communication No .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25249283" target="_blank">25249283</a>]<br></li>
<li>To start , we first screened four alternative sigma factor mutants of S. Typhimurium ( rpoS , rpoN , rpoE , rpoH ) for their ability to express a key virulence gene , sseB , that requires SsrB for expression and whose gene product is essential for intrac-ellular pathogenesis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19245712" target="_blank">19245712</a>]<br></li>
<li>The present work describes a novel role for the unphosphorylated response regulator SsrB in de-repressing H-NS ( Figure 6B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26880544" target="_blank">26880544</a>]<br></li>
<li>The ssrA gene encodes a tripartite inner membrane histidine sensor kinase that presumably phosphorylates the response regulator SsrB in response to an unknown signal ( s ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630976" target="_blank">17630976</a>]<br></li>
<li>SsrB protein is also involved in the decision to switch between these states , but how it does so remains a question for future work .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26880544" target="_blank">26880544</a>]<br></li>
<li>Thus , we establish a new role for SsrB , but not the SsrA kinase in positively regulating biofilm-formation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26880544" target="_blank">26880544</a>]<br></li>
<li>SsrB , but not its cognate kinase SsrA , is involved in the switch from planktonic growth to a multi-cellular lifestyle , in addition to its role in regulating genes required for pathogenesis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26880544" target="_blank">26880544</a>]<br></li>
<li>g003 SPI-2 contains four major operons that encode a T3SS , chaperone and effector proteins , as well as the transcriptional regulator SsrB ( Fig 4A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26565973" target="_blank">26565973</a>]<br></li>
<li>These data did not support a critical virulence role of EIIA in gene Glc regulation in contrast to other PTS components ( Salmonella EIIBC / type three Glc secretion system [ TTSS ] -1 [ Lim et al. , 2007 ] ; Salmonella EIIA / SsrB [ Choi et al. , 2010 ] ; Brucella melitensis HPr and Ntr enzyme I/type-IV-secretion [ Dozot et al. , 2010 ] ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24835993" target="_blank">24835993</a>]<br></li>
<li>E COUNTERS THE SILENCING OF H-NS ON SPI-2 GENE EXPRESSION Several regulators ( e.g. , σS , SsrB and SlyA ) have been shown to be capable of relieving H-NS silencing of transcription ( Mujacic and Baneyx , 2006 ; Perez et al. , 2008 ; Walthers et al. , 2011 ; Galagan et al. , 2013 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25713562" target="_blank">25713562</a>]<br></li>
<li>For example , OmpR and SsrB co-regulate expression of sseI ( srfH ) and ssaB ( Worley et al. , 2000 ; Feng et al. , 2004 ; D. Walthers and L.J. Kenney , unpublished ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630976" target="_blank">17630976</a>]<br></li>
<li>DOI : 10.7554 / eLife .10747.013 SsrB-dependent biofilm-formation does not require SPI-2 SsrB is encoded on SPI-2 and regulates SPI-2 genes involved in assembly and function of the type three secretion system encoded by SPI-2 , as well as effectors that are encoded on SPI-2 and outside of SPI-2 ( Feng et al. , 2004 ; Walthers et al. , 2007 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26880544" target="_blank">26880544</a>]<br></li>
<li>For example , SPI-2-encoded SsrB downregulates SPI-1 genes by repressing HilA and HilD ( Pérez-Morales et al. , 2017 ) , and reciprocally , SPI-1-encoded HilD upregulates SPI-2 gene-expression by directly binding the ssrAB operon ( Bustamante et al. , 2008 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29034217" target="_blank">29034217</a>]<br></li>
<li>Autophosphorylation of the histidine kinase domain initiates a phosphorelay that activates a cognate response regulator for targeted gene-expression , which for the SsrA sensor , is SsrB in S. Typhimurium ( Cirillo et al. , 1998 ; Deiwick et al. , 1999 ; Carroll et al. , 2009 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25442048" target="_blank">25442048</a>]<br></li>
<li>These include SpvR , a transcription factor that governs transcription of the spv virulence genes on the 90 kb plasmid ( Grob & Guiney , 1996 ; Grob et al. , 1997 ; Sheehan & Dorman , 1998 ) , the HilA protein that regulates transcription of the SPI-1 island genes ( Akbar et al. , 2003 ; Bajaj et al. , 1996 ; Boddicker et al. , 2003 ) and the SsrA/SsrB two-component system that controls SPI-2 gene-expression ( Cirillo et al. , 1998 ; Deiwick et al. , 1999 ; Lee et al. , 2000 ; Valdivia & Falkow , 1997 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15256548" target="_blank">15256548</a>]<br></li>
<li>Furthermore , PAT DE4 was positive for the outer membrane receptor gene instead of but no signal was obtained for the islet genes htrE ( probable porin / ﬁmbrial assembly protein ) , sopD2 ( secreted effector protein ) , srfJ ( putative virulence factor , activated by transcription factor SsrB ) and sseK2 ( translocated effector protein SSEK2 by type-III-secretion-system ) while pagK ( PhoPQ-activated protein ) was present only in this array type .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21917347" target="_blank">21917347</a>]<br></li>
<li>To eliminate the effects of SsrB on σE -- mediated SPI-2 gene-expression , we constructed an rpoE/ssrB double mutant strain and compared ssaE , sscA , sseI , ssaJ , and pipB expression in ssrB vs. ( rpoE , ssrB ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25713562" target="_blank">25713562</a>]<br></li>
<li>Expression ratio comparing WT to ssrB indicates that SsrB activates SPI-2 gene-expression ; Expression ratio comparing ssrB to ( ssrB , rpoE ) indicates that the up-regulating effects of σE on SPI-2 gene-expression are through SsrB in LPM 4 h condition .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25713562" target="_blank">25713562</a>]<br></li>
<li>This network shows that the primary regulator responsible for SPI-2 virulence expression , SsrB , integrates signals from many regulators and two-component regulatory systems .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21687430" target="_blank">21687430</a>]<br></li>
<li>The Salmonella-speciﬁc sseL gene , controlled by the SPI-2 regulator SsrB , has recently been described as encoding a translocated deubiquitinase that is required for macrophage killing and systemic virulence in mice ( 15 , 72 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17483226" target="_blank">17483226</a>]<br></li>
<li>Summary The two-component regulatory system SsrA-SsrB in Salmonella enterica controls expression of a virulence gene program required for intracellular survival in host cells .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25442048" target="_blank">25442048</a>]<br></li>
<li>As such , SsrB has emerged as a principal regulator of virulence genes in Salmonella enterica .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17158898" target="_blank">17158898</a>]<br></li>
<li>The transcriptional regulator SsrB is involved in a molecular switch controlling virulence lifestyles of Salmonella .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29034217" target="_blank">29034217</a>]<br></li>
<li>Also , the TCS SsrA/SsrB directly controls expression of SPI-2 intramacrophage survival genes ( 5 , 9 ) , while PhoP/PhoQ controls a vast regulon that includes invasion genes , macrophage survival genes , cation-transporters , and genes involved in antimicrobial peptide ( AP ) resistance ( 10 , 30 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16352830" target="_blank">16352830</a>]<br></li>
<li>SsrB inﬂuences transcriptional regulation of SPI-12 bacteriophage genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23630960" target="_blank">23630960</a>]<br></li>
<li>We therefore also report an analysis of SPI2 transcriptional architecture at LSP resulting in localisation of SsrB binding sites and identification of a previously unreported SPI2 TSS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24664308" target="_blank">24664308</a>]<br></li>
<li>( B ) Expression of SseL requires SsrB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17158898" target="_blank">17158898</a>]<br></li>
<li>SPI-2 acquisition was followed by evolution of existing regulatory DNA , creating an expanded SsrB regulon involved in intracellular fitness and host infection .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22418438" target="_blank">22418438</a>]<br></li>
<li>For example , SsrB can replace SlyA , and in some cases , SlyA can replace SsrB for expression of SPI-2-encoded virulence factors ( Yoon et al. , 2009 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20656781" target="_blank">20656781</a>]<br></li>
<li>Many classic transcription factors are known to regulate S. Typhimurium SPI-2 genes by controlling expression of its master regulator ( SsrA and/or SsrB ) at transcriptional or post-transcriptional levels .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29751061" target="_blank">29751061</a>]<br></li>
<li>Relief of H-NS silencing by SsrB The nucleoid structuring protein H-NS represses horizontally acquired genes of SPI-2 , and we conﬁrmed a role for H-NS at speciﬁc SPI-2 loci , including ssaB , sseA , ssaG and ssaM .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630976" target="_blank">17630976</a>]<br></li>
<li>Strains H405Q ack pta , D739A ack pta and H867Q ack pta , were all capable of biofilm-formation ( Figure 2A ) , demonstrating that none of the known phosphate donors for SsrB were required for regulating biofilms .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26880544" target="_blank">26880544</a>]<br></li>
<li>SsrB activates expression of the type-III-secretion-system encoded within SPI-2 , which then allows for replication inside cells and systemic infection in mice .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10633113" target="_blank">10633113</a>]<br></li>
<li>Summary The two-component system SsrA -- SsrB activates expression of a type-III-secretion-system required for replication in macrophages and systemic infection in mice .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630976" target="_blank">17630976</a>]<br></li>
<li>OmpR Regulates the Two-Component System SsrA-SsrB in Salmonella Pathogenicity Island 2 ANTHEA K. LEE , * CORRELLA S. DETWEILER , AND STANLEY FALKOW Department of Microbiology and Immunology , Stanford University School of Medicine , Stanford , California 94305 Received 23 August 1999/Accepted 1 November 1999 Salmonella pathogenicity island 2 ( SPI-2 ) encodes a putative , two-component regulatory system , SsrA-SsrB , which regulates a type-III-secretion-system needed for replication inside macrophages and systemic infection in mice .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10633113" target="_blank">10633113</a>]<br></li>
<li>Little is known about the genetic regulation of SPI-2 except that expression of genes encoding the type-III-secretion-system of SPI-2 is dependent on a sensor-kinase system , SsrA-SsrB ( also called SpiR ) ( 12 , 16 , 36 , 64 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10633113" target="_blank">10633113</a>]<br></li>
<li>Microbiology ( 2003 ) , 149 , 2385 -- 2396 DOI 10.1099 / mic.0.26397-0 The roles of SsrA -- SsrB and Om R -- EnvZ in the regulation of genes encoding the Salmonella typhimurium SPI-2 type-III-secretion-system p Junkal Garmendia , Carmen R. Beuzón ,3 Javier Ruiz-Albert3 and David W. Holden Department of Infectious Diseases , Centre for Molecular Microbiology and Infection , Imperial College London , Armstrong Road , London SW7 2AZ , UK Correspondence David W. Holden d.holden@imperial.ac.uk The type-III-secretion-system ( TTSS ) encoded by Salmonella typhimurium pathogenicity island 2 ( SPI-2 ) is expressed after bacterial entry-into-host-cells .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12949164" target="_blank">12949164</a>]<br></li>
				<label class="RP_collapse_similarity" for="rp_similarity_67">
Garmendia , J. , Beuzon , C.R. , Ruiz-Albert , J. and Holden , D.W. ( 2003 ) The roles of SsrA-SsrB and OmpR-EnvZ in the regulation of genes encoding the Salmonella typhimurium SPI-2 type-III-secretion-system .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29693760" target="_blank">29693760</a>]<br></label>
<input id="rp_similarity_67", type="checkbox">
<div class="RP_similar">
<li>Garmendia , J. , Beuzon , C.R. , Ruiz-Albert , J. , and Holden , D.W. ( 2003 ) The roles of SsrA -- SsrB and OmpR -- EnvZ in the regulation of genes encoding the Salmonella typhimu-rium SPI-2 type-III-secretion-system .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630976" target="_blank">17630976</a>]<br></li>
<li>Garmendia , J. , Beuzon , C.R. , Ruiz-Albert , J. , and Holden , D.W. ( 2003 ) The roles of SsrA-SsrB and OmpR-EnvZ in the regulation of genes encoding the Salmonella typhimu-rium SPI-2 type-III-secretion-system .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16999831" target="_blank">16999831</a>]<br></li>
<li>[ 44 ] J. Garmendia , C.R. Beuzón , J. Ruiz-Albert , D.W. Holden , The roles of SsrA-SsrB and OmpR-EnvZ in the regulation of genes encoding the Salmonella typhi-murium SPI-2 type-III-secretion-system , Microbiology ( Reading , England ) 149 ( 2003 ) 2385 -- 2396 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23040276" target="_blank">23040276</a>]<br></li>
<li>Garmendia J , Beuzon CR , Ruiz-Albert J & Holden DW ( 2003 ) The roles of SsrA-SsrB and OmpR-EnvZ in the regulation of genes encoding the Salmonella typhimurium SPI-2 type-III-secretion-system .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18355292" target="_blank">18355292</a>]<br></li>
<li>Garmendia J , Beuzon CR , Ruiz-Albert J , Holden DW : The roles of SsrA-SsrB and OmpR-EnvZ in the regulation of genes encoding the Salmonella typhimurium SPI-2 type-III-secretion-system .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19245712" target="_blank">19245712</a>]<br></li>
<li>Garmendia J , Beuzón CR , Ruiz-Albert J , Holden DW : The roles of SsrA-SsrB and OmpR-EnvZ in the regulation of genes encoding the Salmonella typhimurium SPI-2 type-III-secretion-system .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23234298" target="_blank">23234298</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24079299" target="_blank">24079299</a>]<br></li>
<li>The roles of SsrA-SsrB and OmpR-EnvZ in the regulation of genes encoding the Salmonella Typhimurium SPI-2 type-III-secretion-system .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25688337" target="_blank">25688337</a>]<br></li>
<li>The roles of SsrA-SsrB and OmpR-EnvZ in the regulation of genes encoding the Salmonella typhimurium SPI-2 type-III-secretion-system .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19717598" target="_blank">19717598</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20807201" target="_blank">20807201</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23554419" target="_blank">23554419</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29214489" target="_blank">29214489</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25128737" target="_blank">25128737</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19737903" target="_blank">19737903</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15699204" target="_blank">15699204</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19074398" target="_blank">19074398</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19858298" target="_blank">19858298</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15231802" target="_blank">15231802</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15972530" target="_blank">15972530</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16904940" target="_blank">16904940</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25713562" target="_blank">25713562</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16129704" target="_blank">16129704</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27886215" target="_blank">27886215</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16352830" target="_blank">16352830</a>]<br></li>
<li>The roles of SsrA -- SsrB and OmpR -- EnvZ in the regulation of genes encoding the Salmonella typhimurium SPI-2 type-III-secretion-system .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15256548" target="_blank">15256548</a>]<br></li>
<li>The roles of SsrA-SsrB and OmpR-EnvZ in the regulation of genes encoding the Sal-monella Typhimurium SPI-2 type-III-secretion-system .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22843534" target="_blank">22843534</a>]<br></li>
<li>The roles of SsrA-SsrB and OmpR-EnvZ in the regulation of genes encoding the Sal-monella typhimurium SPI-2 type-III-secretion-system .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25192993" target="_blank">25192993</a>]<br></li></div>
<li>SPI-2 consists of a 25 kb region encoding the structural components , secreted effector proteins , chaperone proteins and two-component regulatory elements ( SsrA/SsrB ) of a functional type three secretion system ( T3SS ) apparatus .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29751061" target="_blank">29751061</a>]<br></li>
<li>We established here that the OmpR protein of the global two-component regulatory system OmpR-EnvZ regulates the SPI-2 two-com-ponent regulatory system , SsrA-SsrB , which in turn activates the type-III-secretion-system in SPI-2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10633113" target="_blank">10633113</a>]<br></li>
<li>Furthermore , SsrB could complement all three deletions DssrB , DssrA , and DssrA/ssrB ( Figure 7B ) , suggesting that over-expression of SsrB may compensate for differences in phosphorylation that normally play a role in regulation [ 52 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19229334" target="_blank">19229334</a>]<br></li>
<li>The greater effect of SsrB on SPI-2 expression in the presence of H-NS suggests that SsrB plays a role in countering H-NS-mediated silencing .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630976" target="_blank">17630976</a>]<br></li>
<li>The observation that SsrB existed in the absence of EnvZ suggested that additional regulators play a role in activating SsrB expression ( see below ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15491370" target="_blank">15491370</a>]<br></li>
<li>In minimal media , OmpR , PhoP , SsrB , and SlyA have been involved in the expression of ssrAB ( 32 -- 38 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25135218" target="_blank">25135218</a>]<br></li>
<li>Using RIVET , we also conﬁrmed the role of SsrB and OmpR in regulating SPI-2 gene-expression ( Figure 1D ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16304611" target="_blank">16304611</a>]<br></li>
<li>SsrBC mutants K179A , M186A , and V197A SsrB is the major virulence factor in S. enterica , regulating cross-linked similarly to the wild type , suggesting that these expression of genes within and outside of SPI-2 required for amino acids play a different role in SsrB function ( Fig. 11 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19126546" target="_blank">19126546</a>]<br></li>
<li>These data indicate that SsrB plays a role in regulating the expression of STM2234 , STM2235 , STM2236 , and STM2239 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23630960" target="_blank">23630960</a>]<br></li>
<li>Together , these results make clear the existence of a role for SsrB in decreasing ﬂagellar expression following entry-into-host phagocytes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25249283" target="_blank">25249283</a>]<br></li>
<li>This is the first evidence for a role of unphosphorylated SsrB in gene regulation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26880544" target="_blank">26880544</a>]<br></li>
<li>All of these circuits include central regulators encoded in the genetic region involved in the biosynthesis of these TTSS ( HilA for TTSS-1 , SsrB for TTSS-2 , and FlhD2C2 for ﬂagella ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17060472" target="_blank">17060472</a>]<br></li>
<li>SsrB is autoregulated and activates expression of SPI-2 and SPI-2 co-regulated genes in response to an unknown signal ( s ) ( Feng et al. , 2003 ; 2004 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630976" target="_blank">17630976</a>]<br></li>
<li>OmpR ~ P goes on to activate expression of SsrA/B , and SsrB ~ P activates SPI-2 transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25875623" target="_blank">25875623</a>]<br></li>
<li>Transcriptional activators of SPI genes ( HilC , HilD , SsrB , and SlyA ) function in large part through displacement of H-NS from SPI promoters [ 16,44 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22457642" target="_blank">22457642</a>]<br></li>
<li>A number of directly regulated gene products then regulate additional regulatory cascades , including PmrD , which is able to activate the PmrAB operon independent of iron ; SlyA , which regulates genes important to intra-macrophage survival such as pagC and ugtL ; IraP which prevents MviA-dependent degradation of RpoS leading to RpoS accumulation and its regulation of genes important for stationary phase survival and resistance to oxidative stress ; HilA , which is an inducer of SPI-1 ( Salmonella pathogenicity island-1 ) , which contains genes involved in invasion of epithelial cells ; and SsrB , which is an inducer of SPI-2 containing genes important in intra-macrophage survival ( adapted from Groisman E. and Mouslim C. Nature Reviews Microbiology ( 2006 ) 4:705 -- 709 ) [ 44,45 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19746165" target="_blank">19746165</a>]<br></li>
<li>One of the inner membrane components of T6SS , namely SciS , has also been shown to be negatively regulated by SsrB ( transcription regulator involved in SPI-2 activation ) [ 9 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24079299" target="_blank">24079299</a>]<br></li>
<li>Expression of an episomal copy of ssrB resulted in expression of SPI-2 genes in each mutant background , suggesting that SsrB is epistatic to other regulators for SPI-2 transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19229334" target="_blank">19229334</a>]<br></li>
<li>As shown in Fig. 8 , expression of SsrB during growth in Luria broth induced transcription of genes previously shown to be SsrB-dependent .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630976" target="_blank">17630976</a>]<br></li>
<li>Overexpression and purification of proteins The E. coli BL21 ( DE3 ) strain was used as a host for the overproduction of proteins His-SsrBc , His-SsrB , D56A His-SsrB , K179A His-SsrB and His-H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26880544" target="_blank">26880544</a>]<br></li>
<li>srfJ is regulated not only by SsrB but also by PhoP , RcsB , and , surprisingly , by IolR , the main regulator of genes involved in myo-inosi-tol utilization ( 33 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22661691" target="_blank">22661691</a>]<br></li>
<li>Unphosphorylated SsrB activates csgD expression Biofilm formation in Escherichia coli and Salmonella Typhimurium is governed by the master regulator , CsgD , that acts as a transcriptional activator of genes involved in curli biogenesis and cellulose-synthesis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26880544" target="_blank">26880544</a>]<br></li>
<li>Interestingly , sciS transcription is negatively regulated by SsrB , a two-component regulator that activates salmo-nella pathogenicity island 2 ( SPI2 ) genes and additional genes outside of SPI2 that are primarily involved in promoting systemic infection of the host ( 4 , 6 , 13 , 41 , 43 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15972528" target="_blank">15972528</a>]<br></li>
<li>SsrB is a positive , global regulator of SPI-2 and of virulence factors spread throughout the chromosome ( 36 , 65 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20974834" target="_blank">20974834</a>]<br></li>
<li>SsrB regulates the expression of multiple target genes including a number of virulence factors including members of the Salmonella 22,23 pathogenicity islands .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29057095" target="_blank">29057095</a>]<br></li>
<li>In this study , we used genes of Salmonella Pathogenicity Island 2 ( SPI2 ) to construct a novel cancer vaccine , where a TAA , survivin was fused to SseF effector protein and placed under control of SsrB , the central regulator of SPI2 gene-expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19824039" target="_blank">19824039</a>]<br></li>
<li>However , the up-regulation of SPI1-associated genes in the absence of SsrB suggests an additional layer of transcriptional-control for SPI1 genes , via SsrB-mediated repression under SPI2-inducing conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27564394" target="_blank">27564394</a>]<br></li>
<li>( B ) Acquisition of the transcriptional regulator SsrB in Salmonella Typhimurium led to cis-regulatory evolution of core genes that allow them to come under the regulatory control of SsrB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29034217" target="_blank">29034217</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29034217" target="_blank">29034217</a>]<br></li>
<li>SirA activates the expression of SsrB and HilA through HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21518393" target="_blank">21518393</a>]<br></li>
<li>SsrB activates the expression of SPI-2-related genes , both inside and outside the pathogenicity island .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12949164" target="_blank">12949164</a>]<br></li>
<li>Since PhoP regulates expression of ssrA and ssrB ( Bijlsma and Groisman , 2005 ) , it also regulates expression of genes in the SsrB regulon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29270156" target="_blank">29270156</a>]<br></li>
<li>Since PhoP controls expression of the response regulator SsrB at the transcriptional level and of the sensor SsrA at a posttranscriptional level ( Bijlsma and Groisman , 2005 ) , it also indirectly regulates expression of SsrB-regulated genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25972862" target="_blank">25972862</a>]<br></li>
<li>The role of the SsrAB system for expression of SPI2 genes The characterization of mutant strains with transposon insertions in SPI2 ( Shea et al. , 1996 ) indicated an absolute requirement for functional SsrA and SsrB for SPI2 gene-expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10844687" target="_blank">10844687</a>]<br></li>
<li>N O and dinitrosyl -- iron complexes ( DNIC ) , which frequently target 2 3 thiol-containing proteins , are common sources of nitrosative stress.The redox active thiol of Cys203 in the dimerization domain of the SsrB response regulator that controls SPI2 gene transcription is a bona fide sensor of nitrosative stress in Salmonella .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21833325" target="_blank">21833325</a>]<br></li>
<li>Results SsrB supports biofilm-formation in the absence of its kinase SsrA The SsrA/B TCRS is activated by environmental stimuli such as pH and osmolality , stimulating the expression of virulence genes essential for intra-cellular-growth and survival of Salmonella ( Feng et al. , 2003 ; 2004 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26880544" target="_blank">26880544</a>]<br></li>
<li>Printed in the U.S.A. Salmonella enterica Response egulator SsrB Relieves R H-NS o M de S Received for publication , July 15 , 2010 , and in revised form , November 4 , 2010 Published , JBC Papers in Press , November 8 , 2010 , DOI 10.1074 / jbc.M110 .164962 Don Walthers ‡ , You Li § ¶ , Yingjie Liu § ¶ , Ganesh Anand , Jie Yan § ¶ , and Linda J. Kenney ‡ ¶ 1 From the ‡ Department of Microbiology and Immunology , University of Illinois , Chicago , Illinois 60612 and the § Department of Physics , Mechanobiology Institute , and Department of Biology , ¶ National University of Singapore , Singapore 119007 The response regulator SsrB activates expression of genes encoded within and outside of a pathogenicity island ( SPI-2 ) , which is required for systemic infection of Salmonella .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21059643" target="_blank">21059643</a>]<br></li>
<li>The assimilation of newly acquired genes into the regulons of H-NS , PhoP-PhoQ , and SsrA-SsrB , has allowed for the expansion of ﬂexible genetic networks that mediate bacterial pathogenesis and confer pathoadaptive ﬁtness differences .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29034217" target="_blank">29034217</a>]<br></li>
<li>The SsrA/SsrB TCS , which is unique to Salmonella , regulates the expression of T3SS-2 effectors .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20656781" target="_blank">20656781</a>]<br></li>
<li>We recently identiﬁed the regulatory logic controlling SsrB-DNA interactions and showed that the SsrB response regulator controls about 4 % of the Salmonella genome ( 19 ) , including genes in the remnant phage in Salmonella pathogenicity island 12 ( SPI-12 ) ( 19 -- 21 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23630960" target="_blank">23630960</a>]<br></li>
<li>When activated , the SsrB response regulator directly promotes the transcription of multiple genes within Salmonella pathogenicity island 2 ( SPI-2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21059960" target="_blank">21059960</a>]<br></li>
<li>SPI-2 gene-expression is dependent on the SsrA/SsrB two-component system ; where the SsrB response regulator activates SPI-2 structural genes encoding a T3SS [ 67,68 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24603618" target="_blank">24603618</a>]<br></li>
<li>First , EIIA - mediated regulation of SPI-2 genes was dependent on the SsrB protein because ptsN mutation failed to increase SPI-2 expression in the Ntr absence of SsrB ( Fig. 2A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21059960" target="_blank">21059960</a>]<br></li>
<li>As expression of the SsrB protein is promoted by several transcription factors , including SsrB itself , the expression of SPI-2 genes can increase to undesirable levels under activating conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21059960" target="_blank">21059960</a>]<br></li>
<li>Fur controlled SPI-2 genes via repression of the SsrB response regulator , a primary activator of SPI-2 expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24643535" target="_blank">24643535</a>]<br></li>
<li>SsrB , the response regulator of the SsrA/SsrB two-component system , directly activates expression of SPI-2 genes under acidic conditions ( 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24643535" target="_blank">24643535</a>]<br></li>
<li>We hypothesized that the EIIANtr-mediated control of SPI-2 expression could be dependent on SsrB , a response regulator of the SsrA/SsrB two-component system , because this protein is known to be a key regulator that directly activates the transcription of multiple SPI-2 genes ( 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21059960" target="_blank">21059960</a>]<br></li>
<li>We are currently delineating the remainder of the SsrB regulon and determining the roles of these genes in intracellular virulence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10844662" target="_blank">10844662</a>]<br></li>
<li>This phenotype could be related to the transcriptional down-regu-lation of genes encoding proteins involved in the biosynthesis of all Salmonella type-III-secretion-systems , including the central regulators HilA , SsrB , and FlhC , in a yfgL mutant .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17060472" target="_blank">17060472</a>]<br></li>
<li>The expression of the fusions was analyzed in LPM using Salmonella strains with mutations in genes encoding known virulence regulators : PhoP , SsrB , HilA , and RcsB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22661691" target="_blank">22661691</a>]<br></li>
<li>We tested the effect of null mutations in genes encoding relevant regulators : IolR , SsrB , PhoP , and RcsB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29270156" target="_blank">29270156</a>]<br></li>
<li>We tested the effect of mutations in genes encoding important virulence regulators : HilA , HilD , SsrB , PhoP , RcsB , and Dam .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25972862" target="_blank">25972862</a>]<br></li>
<li>Transcriptional reporter experi - ments , quantitative reverse transcription-PCR ( RT-PCR ) , and immunoblotting demonstrated that the virulence regulator SsrB and STM2239 contribute to transcriptional activation of genes in SPI-12 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23630960" target="_blank">23630960</a>]<br></li>
<li>Strain P8G12 defective in SsrB and unable to activate transcription of SPI2 genes was used as a negative control for comparing the protein patterns .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12112863" target="_blank">12112863</a>]<br></li>
<li>Given that SsrB functions as an activator for expression of SPI-2 genes ( 4 ) and that Fur represses ssrB expression ( Fig. 3 ) , it is possible that the Fur regulator controls SPI-2 genes in a fashion dependent on the SsrB regulator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24643535" target="_blank">24643535</a>]<br></li>
<li>SsrB is a SPI-2 encoded transcriptional regulator that is required for expression of all the SPI-2-encoded genes , as well as for expression of effectors and virulence genes encoded outside of SPI-2 [ 28 -- 31 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26565973" target="_blank">26565973</a>]<br></li>
<li>SsrB regulates the expression of SPI-2 encoded substrate effectors including ssaB , as well as several integrated virulence effectors such as sseL [ 8 ] and srfN [ 9 ] that are encoded elsewhere on the chromosome but that have integrated into the SsrB regulon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19245712" target="_blank">19245712</a>]<br></li>
<li>SsrA-SsrB controls expression of the genes of the SPI-2 T3SS , as well as genes encoding SPI-2 effectors located within SPI-2 and elsewhere on the chromosome [ 15,19 e23 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21320585" target="_blank">21320585</a>]<br></li>
<li>Recent works have shown that several genes located outside of SPI-2 are under the control of the SsrA-SsrB regulator as well , and these have been proposed as putative virulence factors ( 10 , 25 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19717598" target="_blank">19717598</a>]<br></li>
<li>In Salmonella enterica serovar typhimurium , the SsrA/SsrB two-component system controls expression of SPI-2 genes as well as several non-SPI-2-encoded virulence genes ( 8 , 11 -- 13 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19126546" target="_blank">19126546</a>]<br></li>
<li>In addition to genes encoding a TTSS , as well as effector proteins , SPI-2 possesses the ssrA and ssrB genes that express the SsrA/SsrB two-component regulatory system .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21059960" target="_blank">21059960</a>]<br></li>
<li>It has been shown recently that transcription of ssrA , a gene encoding a component of SsrA/SsrB , the SPI-2 two-component regulatory system [ 46 -- 48 ] , is modulated by OmpR in infected tissue cultured cells [ 49 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11755424" target="_blank">11755424</a>]<br></li>
<li>Salmonella pathogenicity island 2 ( SPI2 ) is a genomic island containing approximately 30 contiguous genes required to assemble a functional secretion system including the two-component regulatory system called SsrA-SsrB that positively regulates transcription of the secretion apparatus .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17158898" target="_blank">17158898</a>]<br></li>
<li>Additionally , PhoPQ activity is involved in the upregulation of both the PmrA-PmrB and SsrA-SsrB two-component systems , which are both important for intracellular survival ( Gunn and Miller , 1996 ; Deiwick et al. , 1999 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29034217" target="_blank">29034217</a>]<br></li>
<li>The main regulatory proteins involved in the expression of Spi-2 include the two-component systems SpiR ( SsrA ) / SsrB , PhoP/PhoQ , OmpR/EnvZ , as well as the DNA-binding proteins HilD , SlyA ( RovA in Yersinia ) , RcsB , the iron regulator Fur , and the nucleoid-associated proteins H-NS , Hha ( YmoA in Yersinia ) , YdgT , IHF , and Fis ( Fass and Groisman , 2009 ; Choi et al. , 2014 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26441883" target="_blank">26441883</a>]<br></li>
<li>The PhoP/PhoQ system also controls the expression of some other TCSs such as SsrA/SsrB [ 5 ] , PmrA/PmrB [ 6 ] , and RstA/RstB [ 7 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27618760" target="_blank">27618760</a>]<br></li>
<li>To determine if SirA controls ssaG expression , and thus of other SPI-2 genes , by regulating the ssrAB operon , we analyzed the expression of SsrB in WT and ΔsirA strains , as well as in the ΔhilA , ΔhilD , ΔhilC , and ΔompR mutants , which were tested as controls .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21518393" target="_blank">21518393</a>]<br></li>
<li>These data help unite the virulence phenotypes of strains lacking SsrB and RpoE , and highlight new transcriptional regulation that might be essential for appropriate temporal and spatial control of the virulenceassociated type-III-secretion-system during host infection .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19245712" target="_blank">19245712</a>]<br></li>
<li>The response regulator SsrB controls expression of a type III secretory system in which effectors modify the vacuolar membrane and prevent its degradation via the endocytic pathway .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19126546" target="_blank">19126546</a>]<br></li>
<li>The genetic regulation of gogB in Salmonella is inﬂuenced by the transcriptional activator , SsrB , under SPI-2-inducing conditions , but the modular nature of the gogB gene allows for autonomous expression and type-III-secretion following horizontal gene transfer into a heterologous pathogen .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15843015" target="_blank">15843015</a>]<br></li>
<li>The promoters we chose use the SsrB response regulator for expression of the downstream gene or operon , and include both SPI-2-encoded and non-SPI-2-encoded virulence effectors representing structural apparatus genes and effector substrates of the type-III-secretion-system [ 8,30-35 ] Chemiluminescent β-galactosidase Assay Reporter strains were inoculated from an overnight culture into culture medium ( LPM pH 5.8 ) that induces SsrBdependent gene-expression [ 21,36 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19245712" target="_blank">19245712</a>]<br></li>
<li>The OmpR protein , apart from regulating expression of porin proteins like OmpC and OmpF in response to osmolarity , regulates the expression of another two-component regulatory system , SsrA-SsrB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18355292" target="_blank">18355292</a>]<br></li>
<li>Previous studies have shown that three different two-component systems ( SsrA ( SpiR ) / SsrB , EnvZ / OmpR and PhoQ/PhoP ) are involved in the regulation of SPI2 gene-expression ( Fass & Groisman 2009 ) but as a secretion system encoded on SPI2 , can T3SS2 affect the expression of T3SS2 effector genes ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27735192" target="_blank">27735192</a>]<br></li>
<li>The findings reported in this article provide further support to the notion that the PhoP PhoQ two-component system is a central regulator that controls the expression and or activity of other regulatory proteins and systems when cells experience low Mg2 , including the two-component system RstA RstB of E. coli , ( 47 ) , the SpiR SsrB two-component system ( 48 ) , and Salmonella the Salmonella transcription factor SlyA ( 49 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16938894" target="_blank">16938894</a>]<br></li>
<li>SsrB is a response regulator of a two-component system that regulates the majority of the SPI2 encoded virulence factors [ 32 ] and SsaV is a component of the SPI2 type-III-secretion apparatus [ 33,34 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28192515" target="_blank">28192515</a>]<br></li>
<li>These genes encode a two-component system consisting of the sensor kinase ( SsrA ) and cognate DNA binding protein ( SsrB ) that regulates the expression of a type-III-secretion/effector protein system required for intracellular survival and replication of S. Typhimurium inside the acidified vacoular compartment [ 30 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24603618" target="_blank">24603618</a>]<br></li>
<li>Speciﬁcally , HilD is believed to activate transcription of the genes encoding the SpiR ( SsrA ) / SsrB two-component regulatory system that is the master regulator of SPI-2 ( 75 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27601571" target="_blank">27601571</a>]<br></li>
<li>In agreement with this hypothesis , the expression of the SsrB regulatory protein , the partner of SsrA in the SsrA-SsrB two-component regulatory system that controls SPI-2 virulence gene-expression , was also optimal at 0.2 % arabinose ( Fig. 6B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16999831" target="_blank">16999831</a>]<br></li>
<li>SsrA is a sensor kinase that phosphorylates the cognate response regulator , SsrB , leading to the induction of SPI-2 gene-expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17307861" target="_blank">17307861</a>]<br></li>
<li>Additionally , SPI-2 encodes a two component system ( TCS ) , SsrAB , in which SsrA is the sensor kinase and SsrB is the response regulator that directly activates SPI-2 expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28357338" target="_blank">28357338</a>]<br></li>
<li>SsrA is a sensor kinase activated in the host environment that phosphorylates the SsrB response regulator to create an active transcription factor needed for spatiotemporal control of virulence genes [ 13,17 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20300643" target="_blank">20300643</a>]<br></li>
<li>A o o o E S N rp rp rp Δ Δ Δ t w SseB We tested the hypothesis that RpoE is involved in expression of genes that use the SsrB response regulator for activation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19245712" target="_blank">19245712</a>]<br></li>
<li>Previous results have shown that a conservative replacement of the amino acid that is the essential phosphate acceptor eliminated expression of SPI-2 genes suggesting that SsrB requires phosphorylation for activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19229334" target="_blank">19229334</a>]<br></li>
<li>The SsrAB phosphorelay is the second level of regulation in the model ; our data suggest that SsrA expression is distinct from phosphorelay activity that leads to transcriptional activation of downstream genes , such as ssaG , indicating that SsrA requires an environmental signal to activate SsrB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15231802" target="_blank">15231802</a>]<br></li>
<li>These results also strongly advocate for a larger contribution of SsrB in Salmonella pathogenesis owing to its dual regulation of intracellular ( i.e. , the SCV ) as well as mutli-cellular lifestyles ( the carrier state ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26880544" target="_blank">26880544</a>]<br></li>
<li>As a result of these studies , we propose that SsrB , a pathogenicity island-2-encoded response regulator , sits at a pivotal position in governing Salmonella lifestyle fate : to either exist inside the host ( in the SCV ) as a promoter of virulence ; or as a surface-attached multicellular biofilm , maintaining the carrier state .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26880544" target="_blank">26880544</a>]<br></li>
<li>SsrA autophosphorylates in response to the acidic environment of the Salmonella-containing vacuole ( SCV ) within host cells [ 39,40 ] , which initiates a signaling-cascade that promotes SsrB activity as well as ssrAB expression [ 41 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26565973" target="_blank">26565973</a>]<br></li>
<li>SsrB has been shown to bind and activate the promoter of the srfH/sseI gene that encodes a TTSS2 effector [ 27 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17233907" target="_blank">17233907</a>]<br></li>
<li>In the absence of SsrA , unphosphorylated SsrB directs transcription of factors required for biofilm-formation specifically by activating csgD ( agfD ) , the master biofilm regulator by disrupting the silenced , H-NS-bound promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26880544" target="_blank">26880544</a>]<br></li>
<li>We used the srfH -- lacZ transcriptional fusion to further characterize the behaviour of the response regulator SsrB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15491370" target="_blank">15491370</a>]<br></li>
<li>To follow the expression of SsrB , the ssrB chromosomal gene was tagged with the sequence encoding a 3XFLAG epitope , rendering strains expressing a C-terminal 3XFLAG-tagged SsrB protein ( SsrB-FLAG ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21518393" target="_blank">21518393</a>]<br></li>
<li>Fur represses expression of the ssrB gene encoding the SsrB response regulator under acidic conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24643535" target="_blank">24643535</a>]<br></li>
<li>Interestingly , genes encoding the central regulators HilA , SsrB , and FlhC were shown to be repressed in strains unable to express YfgL .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17060472" target="_blank">17060472</a>]<br></li>
<li>An evaluation to determine transcriptional factors involved in cas3 regulation , as well as those in cas-CRISPR operon , scse2 and ascas2-1 , led to the finding that the CpxR , SsrB , IHF , PhoP , PmrA , FNR , CRP , LRP and OmpR regulators are not involved in the transcriptional-control of these transcriptional units ( Fig .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28270274" target="_blank">28270274</a>]<br></li>
<li>In addition to the known regulon of T3SS-related proteins , we show that , through SodCI and bacterioferritin , SsrB controls resistance to reactive-oxygen-species and that SsrB down-regulates ﬂagella and motility .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25249283" target="_blank">25249283</a>]<br></li>
<li>We identify six regions that are transcriptionally active during cell infections and that have regulatory contributions from the regulators of virulence SsrB , PhoP , and SlyA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22493086" target="_blank">22493086</a>]<br></li>
<li>These virulence regulators were varied , including two-component regulators ( PhoP / PhoQ , SsrA/SsrB , and OmpR/EnvZ ) , alternative sigma factors ( RpoE , RpoS , and FliA ) , post-transcriptional/post-translational regulators ( SmpB , CsrA , Hfq , and RseA ) , a response regulator for which the signal sensor is unknown ( Hnr ) , a bending protein essential for some types of recombination ( IHF ) , and an assortment of other transcriptional/putative transcriptional regulators ( SlyA , Crp , FruR , RelA/SpoT , STM1547 , STM3121 , SpvR , and RcsA ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21687430" target="_blank">21687430</a>]<br></li>
<li>Results Rationale for strains used and study design To date , more than 300 genes have been annotated as regulatory genes in Salmonella and of those tested ( 83 ) we showed that 14 regulators including SpvR , FruR , IHF , PhoP/PhoQ , SsrA/SsrB , SlyA , Hnr , RpoE , SmpB , CsrA , RpoS , CRP , OmpR/EnvZ , and Hfq are required for virulence regulation during systemic infection in an acute mouse infection model [ 24 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21711513" target="_blank">21711513</a>]<br></li>
<li>Transcriptional regulators involved in the activation of virulence factors have been extensively studied , with PhoP and SsrB integrating large regulons important for infection ( 17 , 18 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23630960" target="_blank">23630960</a>]<br></li>
<li>We also verified SsrB - and PhoP-dependent transcription of srfN measuring mRNA levels during growth in AMM1 medium ( Additional file 12 , Figure S8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21711513" target="_blank">21711513</a>]<br></li>
<li>( D ) Western blot analysis of SsrB expression in WT and isogenic ΔhilA , ΔhilD , ΔhilC , ΔsirA , and ΔompR strains carrying a chromosomal FLAG-tagged ssrB gene , using monoclonal anti-FLAG antibodies .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21518393" target="_blank">21518393</a>]<br></li>
<li>Expression of SsrB ( A ) and HilA ( B ) in WT , ΔhilD and ΔsirA strains carrying a chromosomal FLAG-tagged ssrB or hilA gene , and containing vector pBADMycHisC , or plasmids pBAD-HilD1 or pBADSirA1 , was analyzed by Western blotting using monoclonal anti-FLAG antibodies .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21518393" target="_blank">21518393</a>]<br></li>
<li>( B ) Summary of gene-expression data and SsrB binding data for pathogenicity islands , prophage islands , S. bongori ROD and genome sequence not in islands .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20300643" target="_blank">20300643</a>]<br></li>
<li>Blackwell Science , LtdOxford , UKMMIMolecular Microbiology1365-2958Blackwell Publishing Ltd , 200348411311143Original ArticleSsrA and SsrB regulation are uncoupledX.Feng , R.Oropeza and L.J.Kenney Molecular Microbiology ( 2003 ) 48 ( 4 ) , 1131 -- 1143 Dual regulation by phospho-OmpR of ssrA/B gene expression in Salmonella pathogenicity island 2 Xiuhong Feng , Ricardo Oropeza † and Linda J. Kenney * Department of Molecular Microbiology and Immunology , L-220 , Oregon Health and Science University , 3181 SW Sam Jackson Park Road , Portland , OR 97239-3098 , USA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12753201" target="_blank">12753201</a>]<br></li>
<li>ORIGINAL RESEARCH ARTICLE published : 10 February 2015 doi : 10.3389 / fmicb .2015.00027 involvement of SsrB and H-NS in SPI-2 gene expression Jie Li1 ‡ , Christopher C. Overall 2 ‡ , Ernesto S. Nakayasu 2 † , Afshan S. Kidwai1 , Marcus B. Jones 3 , Rudd C. Johnson1 , Nhu T. Nguyen1 , Jason E. McDermott 2 , Charles Ansong2 , Fred Heffron 1 , Eric D. Cambronne 1 and Joshua N. Adkins2 * 1 Department of Molecular Microbiology and Immunology , Oregon Health and Science University , Portland , OR , USA 2 Biological Sciences Division , Paciﬁc Northwest National Laboratory , Richland , WA , USA 3 Department of Infectious Diseases , J. Craig Venter Institute , Rockville , MD , USA The extracytoplasmic functioning sigma factor σE is known to play an essential role for Salmonella enterica serovar Typhimurium to survive and proliferate in macrophages and mice .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25713562" target="_blank">25713562</a>]<br></li>
<li>To address this hypothesis , we performed a global screen for genes outside SPI-2 that are activated by SsrB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10844662" target="_blank">10844662</a>]<br></li>
<li>This study reveals that PhoP , OmpR and SsrB control overlapping regulons that include Salmonella-specific genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10844662" target="_blank">10844662</a>]<br></li>
<li>Salmonella acquired SsrB through horizontal gene transfer , and these findings show how this protein now acts as a molecular switch between disease-causing and biofilm-based lifestyles .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26880544" target="_blank">26880544</a>]<br></li>
<li>The response regulator SsrB , in turn , is autoregulated , activating ssrB , ssrA and spiC ( Feng et al. , 2003 ) , as well as other genes located both inside and outside of © 2004 Blackwell Publishing Lt spiC ssrA ssrB srfH , etc Asp-P SsrB Fig. 1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15491370" target="_blank">15491370</a>]<br></li>
<li>SsrB , in turn , activates ssrB , ssrA and spiC ( ssaB ) as well as additional genes , including srfH .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15491370" target="_blank">15491370</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15491370" target="_blank">15491370</a>]<br></li>
				</div></ul><br>
			<label class="collapse_SIT" for="_5"><font size="5px">SsrB binding site</font></label>
				<input id="_5" type="checkbox">
				<div class="SIT"><br><ul><li>( A ) Motif consensus logo for SsrB recognition motif was generated using Weblogo software ( http://weblogo .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27564394" target="_blank">27564394</a>]<br></li>
<li>x The response regulator SsrB activates transcription and binds to a region overlapping OmpR binding sites at Xiuhong Feng , † Don Walthers , Ricardo Oropeza ‡ and Linda J. Kenney * Department of Microbiology and Immunology , University of Illinois at Chicago , 835 S. Wolcott Avenue , M/C 790 , Chicago , IL 60612 , USA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15491370" target="_blank">15491370</a>]<br></li>
				<label class="SIT_collapse_similarity" for="sit_similarity_3">
The response regulator SsrB activates transcription and binds to a region overlapping OmpR binding sites at Salmonella pathogenicity island 2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20807201" target="_blank">20807201</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23554419" target="_blank">23554419</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19737903" target="_blank">19737903</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17158898" target="_blank">17158898</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15972530" target="_blank">15972530</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20202329" target="_blank">20202329</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24643535" target="_blank">24643535</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20656781" target="_blank">20656781</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25135218" target="_blank">25135218</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21518393" target="_blank">21518393</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16129704" target="_blank">16129704</a>]<br></label>
<input id="sit_similarity_3", type="checkbox">
<div class="SIT_similar">
<li>Feng X , Walthers D , Oropeza R , Kenney LJ ( 2004 ) The response regulator SsrB activates transcription and binds to a region overlapping OmpR binding sites at Salmonella pathogenicity island 2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21059960" target="_blank">21059960</a>]<br></li>
<li>Feng , X. , Walthers , D. , Oropeza , R. , and Kenney , L.J. ( 2004 ) The response regulator SsrB activates transcription and binds to a region overlapping OmpR binding sites at Sal-monella pathogenicity island 2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16999831" target="_blank">16999831</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20545866" target="_blank">20545866</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630976" target="_blank">17630976</a>]<br></li>
<li>Feng , X. , Walthers , D. , Oropeza , R. , and Kenney , L.J. ( 2004 ) The response regulator SsrB activates transcription and binds to a region overlapping OmpR binding sites at Salmonella pathogenicity island 2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17784910" target="_blank">17784910</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28631437" target="_blank">28631437</a>]<br></li>
<li>Feng , X. , Walthers , D. , Oropeza , R. , and Kenney , L. J. ( 2004 ) The response regulator SsrB activates transcription and binds to a region overlapping OmpR binding sites at Salmonella pathogenicity island 2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22418438" target="_blank">22418438</a>]<br></li></div>
<li>Our results revealed that SsrB directly regulates transcription at these promoters by binding upstream from , and perhaps overlapping , their respective 35 hexamers ( Fig. 1 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21059643" target="_blank">21059643</a>]<br></li>
<li>SsrB activates transcription of multiple operons within SPI-2 by binding to degenerate DNA targets at diversely organized promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630976" target="_blank">17630976</a>]<br></li>
<li>( XLSX ) 1344 parental and DrelAD ¬ Table S6 Location of SsrB binding sites relative to transcription start sites at SsrB dependent promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24664308" target="_blank">24664308</a>]<br></li>
<li>The SsrB ( 7 ) - and PmrA ( 21 ) - binding sites were boxed with dashed and solid lines , respectively .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24643535" target="_blank">24643535</a>]<br></li>
<li>DNase I protection footprinting presented here and in previous work from our laboratory has identiﬁed SsrB binding sites at nine promoters , including ssrA , ssrB , ssaB , ssaG , ssaM , sseA , sifA , sifB and sseI ( Feng et al. , 2004 ; D. Walthers and L.J. Kenney , unpublished ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630976" target="_blank">17630976</a>]<br></li>
<li>SsrB-dependent promoters are diverse and lack a well-deﬁned consensus binding site To date , we have identiﬁed SsrB binding sites at nine SPI-2 and SPI-2 co-regulated promoters ( this work and Feng et al. , 2004 ; D. Walthers and L.J. Kenney , unpublished ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630976" target="_blank">17630976</a>]<br></li>
<li>Walthers et al. [ 52 ] and Feng et al. [ 81 ] found that there was no consensus SsrB binding site or distance relative to the transcription start sites and Walthers ( ibid ) proposes that different promoters may be activated by distinct SsrB mechanisms .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19229334" target="_blank">19229334</a>]<br></li>
<li>It is noteworthy that a 400 bp divergent promoter region is found upstream of ssaB and contains multiple binding sites for transcription factors including OmpR , SsrB , SlyA and Fis ( Lee et al. , 2000 ; Feng et al. , 2003 ; 2004 ; Kelly et al. , 2004 ; Navarre et al. , 2006 ; D. Walthers and L.J. Kenney , unpublished ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630976" target="_blank">17630976</a>]<br></li>
<li>This phenotype , coupled with activation by SsrB , prompted downstream sequence analysis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10844662" target="_blank">10844662</a>]<br></li>
<li>berkeley.edu ) [ 98 ] using SsrB bound sites determined by ChIP-chip analysis [ 47 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27564394" target="_blank">27564394</a>]<br></li>
<li>Our results demonstrate that SsrB binds directly to SPI-2 promoter DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630976" target="_blank">17630976</a>]<br></li>
<li>Our previous work established that SsrB binds to regions of DNA that are AT-rich , with poor sequence conservation .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19126546" target="_blank">19126546</a>]<br></li>
<li>Genome-wide identification of SsrB palindrome sequences .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20300643" target="_blank">20300643</a>]<br></li>
<li>Highly conserved sites within the promoters were restricted to regions previously footprinted by SsrB [ 25,26 ] , whereas adjacent sequence showed substantial drift ( Figure 4C ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20300643" target="_blank">20300643</a>]<br></li>
<li>The initial motif was generated from sequences from the top ten SsrB interaction peaks and refined using the top 25 peak sequences .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20300643" target="_blank">20300643</a>]<br></li>
<li>( B ) Consensus sequence for SsrB recognition aligned with putative SsrB recognition site within the STnc3020 promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27564394" target="_blank">27564394</a>]<br></li>
<li>SsrB binds upstream of csgD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26880544" target="_blank">26880544</a>]<br></li>
<li>The possibility that SsrB could employ a similarly complex mode of binding would make identiﬁcation of a consensus core binding sequence even more elusive .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630976" target="_blank">17630976</a>]<br></li>
<li>Consistent with this finding , an SsrB binding motif was identified at position 185 to 168 relative to the dalS translational start site ( Fig. 1E ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22418438" target="_blank">22418438</a>]<br></li>
<li>Despite the myriad of binding sites now available , we have been unable to identify a precise consensus sequence , suggesting that SsrB binds to degenerate sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630976" target="_blank">17630976</a>]<br></li>
<li>NarL binds to a well conserved DNA consensus sequence ( 23 ) , whereas the RcsB binding site is more degenerate ( 22 ) , and it is difficult to discern a specific binding motif for SsrB ( 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19126546" target="_blank">19126546</a>]<br></li>
<li>The finding that a flexible palindromic sequence can be selective for SsrB input raises many interesting questions around the nature of regulatory evolution .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20300643" target="_blank">20300643</a>]<br></li>
<li>Further , we used a genetic screen and computational analyses integrating experimental data from S. enterica and sequence data from an orthologous regulatory system in the insect endosymbiont , Sodalis glossinidius , to identify the conserved yet flexible palindrome sequence that defines DNA recognition by SsrB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20300643" target="_blank">20300643</a>]<br></li>
<li>Identification of a functional DNA element for SsrB binding The presence of a conserved palindrome sequence in SPI-2 promoters and in related sequences from the endosymbiont S. glossinidius suggested that regulatory input by SsrB was through a palindrome sequence architecture .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20300643" target="_blank">20300643</a>]<br></li>
<li>g004 SsrB directs transcriptional input using a flexible palindrome architecture Regulatory evolution of the SPI-2 T3SS effector repertoire The above results identified the conserved , yet flexible , palindrome sequence defining DNA recognition by SsrB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20300643" target="_blank">20300643</a>]<br></li>
<li>Surprisingly , at sseA , transcription initiates within the SsrB binding site identiﬁed by DNase I footprinting .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630976" target="_blank">17630976</a>]<br></li>
<li>Our ChIP-on-chip data further suggested that SsrB binding within SPI-2 was specific , with binding peaks overlapping precisely with regions of the DNA footprinted by SsrB [ 25,26 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20300643" target="_blank">20300643</a>]<br></li>
<li>The analysis described above recovered SsrB palindrome sequences at the sseA ' and ssaR locations prompting further validation of these sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20300643" target="_blank">20300643</a>]<br></li>
<li>( A ) Schematic representation of the bacterial one-hybrid screen for functional SsrB binding sequences .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20300643" target="_blank">20300643</a>]<br></li>
<li>Motif analysis of the SsrB recognition site Previously published ChIP-chip analysis of the SsrB regulon identified an 18 bp palindromic sequence with an internal 7-4-7 organisation for SsrB recognition [ 47 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27564394" target="_blank">27564394</a>]<br></li>
<li>A score of 10 was used as the minimum threshold score for an SsrB recognition motif , based on the scores of defined SsrB bound sites scanned with the same PSSM .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27564394" target="_blank">27564394</a>]<br></li>
<li>The ChIP-on-chip data not only provided information on the identities of SsrB-regulated genes but also gave insight as to the identity of the SsrB recognition element specified by the interaction site sequences .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20300643" target="_blank">20300643</a>]<br></li>
<li>To generate plasmids pVarN and pSsrB , the ORF and the upstream promoter sequence of varN or ssrB were ampliﬁed by PCR from genomic DNA of the WT strain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29751061" target="_blank">29751061</a>]<br></li>
<li>A search of the S. enterica genome for this palindromic motif revealed candidates upstream of the previously noted SsrB dependent genes , including two additional SPI-2 sites ; one IGR site upstream of ssaR that until now had been cryptic , and one intragenic palindrome upstream of sseA in the ssaE CDS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20300643" target="_blank">20300643</a>]<br></li>
<li>As a result of our mutational analyses we conclude that so long as the orientation of a single heptamer of the palindrome is conserved with respect to the downstream gene , SsrB is tolerant of degeneracy in the adjacent spacer and heptamer sequences .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20300643" target="_blank">20300643</a>]<br></li>
<li>To generate a consensus set of SsrB regulated genes , we performed an analysis to identify operons in the S. Typhimurium SL1344 genome that encoded at least one gene down-regulated in DssrB cells and that possessed an SsrB binding peak in the upstream regulatory region as defined by our ChIP-on-chip analysis .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20300643" target="_blank">20300643</a>]<br></li>
<li>This activity is mediated by a ﬂexible 18 base pair palindromic sequence in 7-4-7 ′ architecture upstream of SsrB-regulated genes , allowing SsrB to regulate ∼ 5 % of the Salmonella genome ( Tomljenovic-Berube et al. , 2010 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29034217" target="_blank">29034217</a>]<br></li>
<li>We previously identiﬁed an SsrB binding site proximal to the ssrA coding sequence and upstream of ssaB ( Feng et al. , 2004 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630976" target="_blank">17630976</a>]<br></li>
<li>Results The isolated C-terminus of SsrB ( SsrBc ) binds to multiple targets within SPI-2 We previously demonstrated that the isolated C-terminus of SsrB ( SsrBc ) bound to DNA upstream of the ssrA and ssrB coding sequences and to the Gifsy-2 prophage-encoded sseI ( srfH ) promoter ( Feng et al. , 2004 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630976" target="_blank">17630976</a>]<br></li>
<li>This is in contrast to the sseI ( srfH ) promoter where SsrB binds nearly 300 bp upstream of its coding sequence and almost 200 bp upstream of the transcription start site ( Feng et al. , 2004 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630976" target="_blank">17630976</a>]<br></li>
<li>The sequence of the 755 bp csgD regulatory region indicating the H-NS binding region according to Gerstel et al. ( 2003 ) ; and the SsrB binding motif as found by Feng et al. ( 2004 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26880544" target="_blank">26880544</a>]<br></li>
<li>However , due to the large distance between the ssaB coding sequence and the SsrB binding site ( c. 300 bp ) , we re-examined the 418 bp ssaB -- ssrA intergenic region for potential binding sites closer to ssaB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630976" target="_blank">17630976</a>]<br></li>
<li>Interestingly , the disparate location of binding sites with respect to the transcription start site , including sites that are upstream , overlapping or downstream of the start of transcription , suggests that the mechanism of SsrB activation differs from that of a classical transcriptional activator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630976" target="_blank">17630976</a>]<br></li>
<li>This analysis revealed that the majority of SsrB binding sites were located immediately upstream to , or overlapping a region 35 nt upstream of the TSS ( Table S6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24664308" target="_blank">24664308</a>]<br></li>
<li>The transcription start sites of SsrB-dependent genes are positioned at varying distances from the respective SsrB binding site To put the SsrBc binding sites identiﬁed above in the context of a functional promoter , we determined the transcription start site of each gene with SsrBc binding sites as shown in Fig. 2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630976" target="_blank">17630976</a>]<br></li>
<li>A counter-silencing role for SsrB can rationalize the location of certain SsrB DNA binding sites upstream or downstream of transcription start sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630976" target="_blank">17630976</a>]<br></li>
<li>The ssaR TSS has not previously been identified although promoter activity and SsrB binding in the ssaR upstream region has been reported and a consensus SsrB binding site was identified upstream of the ssaR TSS [ 30 ] ( Table S6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24664308" target="_blank">24664308</a>]<br></li>
<li>At promoters where SsrB does not appear to bind close to the 235 region it is also possible that SsrB activates transcription to initiate at alternative TSSs used only under certain growth conditions ; for example , we identified a weak TSS signal 34 nt downstream of a consensus SsrB binding site at the sseI promoter , although the previously reported TSS is located 100 nt upstream of the SsrB binding site ( Table S6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24664308" target="_blank">24664308</a>]<br></li>
<li>A high scoring SsrB binding motif was identified adjacent to the putative -35 site of the STnc3020 promoter ( S8 Fig ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27564394" target="_blank">27564394</a>]<br></li>
<li>Because the putative 19-bp Fur box is located only 5 bp upstream of the SsrB binding site ( 7 , 33 ) on the ssrB promoter ( Fig. 4A ) , and because Fur binds to regions larger than the Fur box ( 31 bp ) ( 11 ) , it is possible that the Fur repressor competes with the SsrB activator for binding to the ssrB promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24643535" target="_blank">24643535</a>]<br></li>
<li>Alignment of the sequences identiﬁed in the present work reveals a common TTAAT ( shown in bold ) in all three of the promoters ( the region protected by SsrBc is underlined ) : ssrA : GAAGAAATTTATCTTCCTAATGATAACACCATCGA TTAATCTTCTGATGAAACTATATGTA ssrB : AAAAACTACACCATTACTTAATATTATCTTAATTTT C srfH : TCTATGCGCCAGTCCTTAATGGCATTATCTGAAT CGTTAAGTAATTTC However , this motif is not present in SsrB binding sites identiﬁed at SPI-2 genes sseA , ssaB , ssaG or ssaM ( D. Walthers and L.J. Kenney , unpubl .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15491370" target="_blank">15491370</a>]<br></li>
<li>( A ) The sequences of the wild-type SsrB binding motif and mutated permutations from the ssaG promoter are shown .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20300643" target="_blank">20300643</a>]<br></li>
<li>24 instances of the 18 bp SsrB binding motif in S. Typhimurium SL1344 have been reported ,23 By performing a random and equal bifurcation of the known binding sites into test and training sets , we were able to achieve recall rates of 75 % in the reconstructed regulatory network for S. Typhimurium SL1344 ( Supplementary Table 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29057095" target="_blank">29057095</a>]<br></li>
<li>SlyA and SsrA/SsrB together activate transcription of the SPI-2 encoded secretion apparatus SlyA is a DNA-binding protein with high affinity for inverted-repeat sequences [ 79 ] and the binding ability of SlyA to the ssrA promoter region was previously reported [ 62 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19229334" target="_blank">19229334</a>]<br></li>
<li>SPI-2 regulation by SsrB and SlyA may be explained in a similar mechanism where both SlyA and SsrB counteract the silencing activity of H-NS/YdgT/Hha by competing for binding to the same target sequences or by altering the structure of the DNA to promote transcription for SPI-2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19229334" target="_blank">19229334</a>]<br></li>
<li>We estimate that on average , binding of SsrB to the csgDEFG-csgBAC intergenic sequence occurred with a bending angle of around 82 ˚ ( Figure 5D and Figure 5 -- figure supplement 2 for naked DNA ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26880544" target="_blank">26880544</a>]<br></li>
<li>A typical 50 ml reaction contained 10 ng of this DNA ( 755 bp of the csgD regulatory region ) mixed with an appropriate amount of SsrB , SsrBc or D56A SsrB and incubated for 15 min at room temperature .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26880544" target="_blank">26880544</a>]<br></li>
<li>Although for some genes it has been noted that SsrB can directly activate transcription by binding immediately upstream of their promoters [ 29 ] , a full analysis of SsrB dependent promoters has been limited by the number of targets for which both SsrB binding site and TSS information was available .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24664308" target="_blank">24664308</a>]<br></li>
<li>A , primary sequence alignment of tional activation ( 11 , 13 ) , and DNase I footprinting has identi-DNA contact helix ( 9/H3 ) of SsrB with NarL family members for which the fied an SsrB binding site between 122 and 95 upstream of structure is known .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19126546" target="_blank">19126546</a>]<br></li>
<li>Applying a stringent threshold to the output allowed us to identify two groups - genes with high-scoring upstream palindromes ( ssaB , ssaG , ssaM , ssaR , sopD2 , sifA , sifB , sseK2 , sseK3 , sseL , sseA9 , steC , and srcA ) and those with mediumscoring palindromes ( 0.7 -- 0.8 threshold ; ssrA , STM1633 , sseI , slrP sspH2 , pipB , sseJ , pipB2 , srfN , sseA and steB ) ( Figure 6A and Dataset S4 ) ( sseA9 denotes the SsrB palindrome sequence upstream of sseA that falls within the ssaE CDS , while sseA refers to the SsrBfootprinted IGR site with only one conserved heptamer defined in Figure 4C ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20300643" target="_blank">20300643</a>]<br></li>
<li>g005 ( meme.nbcr.net/meme/fimo-intro.html ) , we were unable to identify any candidate SsrB binding sites or close matches within the promoters of the candidate and ` validated ' ncRNAs described here and in [ 19,20 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24664308" target="_blank">24664308</a>]<br></li>
<li>Results from DNase I protection assays provide direct evidence that SsrB binds at ssrA and ssrB , although the binding sites lie within the transcribed regions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15491370" target="_blank">15491370</a>]<br></li>
<li>SsrBC-DNA Interactions and Contact Residues -- Thus far , we have identified 10 genes that are directly activated by SsrB ; they are AT-rich but do not contain a well conserved consensus site ( 8 , 10 , 12 , 13 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19126546" target="_blank">19126546</a>]<br></li>
<li>t002 Previous attempts by others to identify a conserved SsrB DNA recognition motif have been unsuccessful .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20300643" target="_blank">20300643</a>]<br></li>
				</ul></div><br>
			<label class="collapse_TU" for="_6"><font size="5px">Transcription units regulated by SsrB</font></label>
				<input id="_6" type="checkbox">
				<div class="TU"><br><ul><li>Salmonella pathogenicity island 2 expression negatively controlled by EIIANtr-SsrB interaction is required for Salmonella virulence .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23006851" target="_blank">23006851</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24643535" target="_blank">24643535</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27628932" target="_blank">27628932</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24835993" target="_blank">24835993</a>]<br></li>
<li>The PhoQ/PhoP two-component system regulates SPI2 genes and some T3SS2-related genes located outside SPI2 through another two-component system , SsrA/SsrB ( 18 , 45 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23504014" target="_blank">23504014</a>]<br></li>
<li>Additionally , it possessed no srfJ gene ( encoding a putative virulence factor activated by transcription factor SsrB ) , which was found only for this isolate .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21705546" target="_blank">21705546</a>]<br></li>
<li>STM2238 was not included because it was not SsrB regulated in our experiments and thus was not prioritized for follow-up .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23630960" target="_blank">23630960</a>]<br></li>
<li>However , it is not known if SsrB directly or indirectly regulates sciS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15972528" target="_blank">15972528</a>]<br></li>
<li>However , it is not known whether SsrB regulates GrvA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15972528" target="_blank">15972528</a>]<br></li>
<li>In the results presented herein , it is apparent that SsrB has adapted to relieving H-NS-silencing and that phosphorylation is not required for this behavior , nor is it required for DNA bind ¬ ing ( Figure 5B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26880544" target="_blank">26880544</a>]<br></li>
<li>This result suggests that IolR can repress srfJ when SsrB is inactive but not when SsrB is active ( LB -- 0.3 M NaCl versus LPM ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22661691" target="_blank">22661691</a>]<br></li>
<li>The rRNA methyltransferase gene , rsmC , was used as an internal control , and the SPI-12 gene sspH2 was used as a positive control , as it is regulated by SsrB ( 21 , 32 , 33 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23630960" target="_blank">23630960</a>]<br></li>
<li>SseB was used as a positive control as a gene product whose expression is dependent on SsrB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22418438" target="_blank">22418438</a>]<br></li>
<li>The rsmC gene was used as an internal control across strains , while sspH2 served as a positive control , as it is known to be SsrB dependent .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23630960" target="_blank">23630960</a>]<br></li>
<li>By analyzing coregulated genes using DNA arrays ( 11 ) , we identiﬁed a gene whose expression under SPI2-induc-ing conditions was strongly SsrB dependent .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17158898" target="_blank">17158898</a>]<br></li>
<li>SsrB D56A and SsrBc condense H-NS-bound csgD DNA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26880544" target="_blank">26880544</a>]<br></li>
<li>In SsrB , a negatively charged glutamic acid res - These results demonstrate that Lys179 and Met186 are both idue replaces Lys188 of NarL , whereas in RcsB , it is replaced by a important for sseI promoter activation by SsrB , whereas Thr183 serine .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19126546" target="_blank">19126546</a>]<br></li>
<li>These data suggest that SsrB is modiﬁed selectively by ONOO − but not H2O2 , both of which are produced by Salmonella-infected macrophages ( 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20660761" target="_blank">20660761</a>]<br></li>
<li>srfH is activated nearly 100-fold by SsrB in both cultured epithelial and macrophage cells .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10844662" target="_blank">10844662</a>]<br></li>
<li>sciS is more highly transcribed at 24 h postinfection and is negatively regulated by SsrB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15972528" target="_blank">15972528</a>]<br></li>
<li>Expression of srfJ is regulated positively by SsrB and PhoP and negatively by RcsB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22661691" target="_blank">22661691</a>]<br></li>
<li>FIG 4 srfJ is regulated by PhoP , SsrB , and RcsB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22661691" target="_blank">22661691</a>]<br></li>
<li>In contrast , spiC expression is strongly affected by SsrB and less so by OmpR .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12753201" target="_blank">12753201</a>]<br></li>
<li>ssaG expression is strongly controlled by SsrB [ 33 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21176126" target="_blank">21176126</a>]<br></li>
<li>The SPI2 operons are all activated by SsrB and yet many SPI2 encoded genes are also controlled either directly or indirectly by multiple ancestral proteins [ 31 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24664308" target="_blank">24664308</a>]<br></li>
<li>Both islands are regulated by PhoP , that positively regulates SPI2 through SsrB and negatively regulates SPI1 through HilA ( Bajaj et al. , 1996 ; Bijlsma and Groisman , 2005 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25972862" target="_blank">25972862</a>]<br></li>
<li>This result suggests that , under these conditions , the SsrB regulon , including , SPI2 , would not be expressed and therefore SrfJ , although expressed from PiolE , would not be secreted through T3SS2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29270156" target="_blank">29270156</a>]<br></li>
<li>SsrB activates transcription of multiple operons within SPI-2 by binding to degenerate DNA targets at diversely organized promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630976" target="_blank">17630976</a>]<br></li>
<li>SsrB regulates SPI-12 phage genes by transcriptional read-through from the sspH2 promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23630960" target="_blank">23630960</a>]<br></li>
<li>This gene is indirectly regulated by PhoP through SsrB but also directly through binding to the sseL promoter ( 40 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23504014" target="_blank">23504014</a>]<br></li>
<li>Expression of SsrB ( A ) and HilA ( B ) in WT , ΔhilD and ΔsirA strains carrying a chromosomal FLAG-tagged ssrB or hilA gene , and containing vector pBADMycHisC , or plasmids pBAD-HilD1 or pBADSirA1 , was analyzed by Western blotting using monoclonal anti-FLAG antibodies .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21518393" target="_blank">21518393</a>]<br></li>
<li>Expression of HilA and SsrB in WT S. Typhimurium carrying chromosomal FLAG-tagged ssrB or hilA genes and containing plasmids pMPM-T3 ( vector ) or plasmid pT3-CsrA expressing CsrA from a constitutive promoter , was analyzed by Western blot ( F ) , as described in figure 2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21518393" target="_blank">21518393</a>]<br></li>
<li>Other regulators can activate and/or repress SPI-2 gene expression by acting through SsrA and/or SsrB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29751061" target="_blank">29751061</a>]<br></li>
<li>Many classic transcription factors are known to regulate S. Typhimurium SPI-2 genes by controlling expression of its master regulator ( SsrA and/or SsrB ) at transcriptional or post-transcriptional levels .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29751061" target="_blank">29751061</a>]<br></li>
<li>Positive regulation by SsrB is a common feature of SPI2 genes and other genes encoding effectors speciﬁcally secreted through T3SS2 ( Xu and Hensel , 2010 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29270156" target="_blank">29270156</a>]<br></li>
<li>The other six genes STM1329 , STM1330 , STM1600 , STM1854 , STM2585 and yneC ( STM4079S ) show a pattern of expression that resembles the SPI2-encoded and SPI2-associated genes , that are positively regulated by both HilD and SsrB under SPI1 and SPI2-inducing conditions , respectively , and are upregulated within macrophages [ 14 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27564394" target="_blank">27564394</a>]<br></li>
<li>In doing so we believe that the nineteen operons containing differentially expressed genes determined by microarray and containing a ChIP-on-chip peak three standard deviations above the mean captured by this analysis represent the genes directly activated by SsrB ( Table 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20300643" target="_blank">20300643</a>]<br></li>
<li>srfJ is regulated not only by SsrB but also by PhoP , RcsB , and , surprisingly , by IolR , the main regulator of genes involved in myo-inosi-tol utilization ( 33 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22661691" target="_blank">22661691</a>]<br></li>
<li>It is significant that members of the SsrB regulon are regulated by both OmpR and PhoP .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10844662" target="_blank">10844662</a>]<br></li>
<li>Transcription of srfJ is positively regulated by the two-component systems PhoP/PhoQ and SsrA/SsrB and negatively regulated by the RcsC/RcsD/RcsB phosphorelay system .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22661691" target="_blank">22661691</a>]<br></li>
<li>The SsrB protein increases its own levels via positive autoregulation ( 4 ) and the OmpR and PhoP proteins also do so by directly activating ssrB transcription ( 9 , 10 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21059960" target="_blank">21059960</a>]<br></li>
<li>In contrast , the SPI-2 T3SS and effector gene promoters , ssaB-E , sseA-G , ssaG-L , and ssaM-R , are only very slightly induced by altered DNA topology and the presumed increase in SsrA and SsrB concentrations brought about by novobiocin treatment ( Figure S1A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22457642" target="_blank">22457642</a>]<br></li>
<li>Two of the 8 genes , mcpA ( STM3138 ) and mcpC ( STM3216 ) , are predicted to encode Salmo-nella-specific methyl-accepting chemotaxis proteins [ 40 ] and display similar expression patterns to the SPI1-encoded , SPI1-associated and SPI4-encoded genes which are directly or indirectly activated by HilD , and repressed by SsrB , but are not regulated by HilA suggesting they are controlled by the HilD feed-forward loop .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27564394" target="_blank">27564394</a>]<br></li>
<li>To construct pRelA and pSsrB , the target genes were amplified by PCR with the following primers : relA-FW2 and relA-RV2 for relA and ssrB-FW and ssrB-RV for ssrB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21176126" target="_blank">21176126</a>]<br></li>
<li>ΔssrA strain was generated that kept the upstream promoter and downstream ssrB gene intact and which did not affect SsrB expression ( Fig. 3A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25442048" target="_blank">25442048</a>]<br></li>
<li>Feng and colleagues [ 28 ] , found ssrA and ssrB expression to be uncoupled , with ssrB having its own promoter which was dependent on both OmpR and SsrB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24664308" target="_blank">24664308</a>]<br></li>
<li>The response regulator SsrB , in turn , is autoregulated , activating ssrB , ssrA and spiC ( Feng et al. , 2003 ) , as well as other genes located both inside and outside of © 2004 Blackwell Publishing Lt spiC ssrA ssrB srfH , etc Asp-P SsrB Fig. 1 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15491370" target="_blank">15491370</a>]<br></li>
<li>Blackwell Science , LtdOxford , UKMMIMolecular Microbiology1365-2958Blackwell Publishing Ltd , 200348411311143Original ArticleSsrA and SsrB regulation are uncoupledX.Feng , R.Oropeza and L.J.Kenney Molecular Microbiology ( 2003 ) 48 ( 4 ) , 1131 -- 1143 Dual regulation by phospho-OmpR of ssrA/B gene expression in Salmonella pathogenicity island 2 Xiuhong Feng , Ricardo Oropeza † and Linda J. Kenney * Department of Molecular Microbiology and Immunology , L-220 , Oregon Health and Science University , 3181 SW Sam Jackson Park Road , Portland , OR 97239-3098 , USA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12753201" target="_blank">12753201</a>]<br></li>
<li>For example , OmpR and SsrB co-regulate expression of sseI ( srfH ) and ssaB ( Worley et al. , 2000 ; Feng et al. , 2004 ; D. Walthers and L.J. Kenney , unpublished ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630976" target="_blank">17630976</a>]<br></li>
<li>The SPI-2-encoded SsrA/SsrB two-component regulatory system ( Cirillo et al. , 1998 , Deiwick et al. , 1999 ; Worley et al. , 2000 ) , controls the expression of the SPI-2 regulon mainly by counteracting the repression exerted by H-NS on genes located within and outside the island ( Walthers et al. , 2007 ; Walthers et al. , 2010 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21518393" target="_blank">21518393</a>]<br></li>
<li>Because SsrA/SsrB is the two-component regulatory system controlling SPI-2 gene expression ( Cirillo et al. , 1998 ; Deiwick et al. , 1999 ) , we next analysed PipB2-2HA expression in an ssrB mutant background .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12864852" target="_blank">12864852</a>]<br></li>
<li>Inside professional phagocytic cells , SPI-2 expression is induced by the two-component systems PhoP/PhoQ , OmpR/EnvZ , and SsrA/SsrB ( Lee et al. , 2000 ; Garmendia et al. , 2003 ; Bijlsma and Groisman , 2005 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25713562" target="_blank">25713562</a>]<br></li>
<li>For example , SPI-2-encoded SsrB downregulates SPI-1 genes by repressing HilA and HilD ( Pérez-Morales et al. , 2017 ) , and reciprocally , SPI-1-encoded HilD upregulates SPI-2 gene expression by directly binding the ssrAB operon ( Bustamante et al. , 2008 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29034217" target="_blank">29034217</a>]<br></li>
<li>SPI-2 genes are expressed in macrophages and dependent on SsrB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630976" target="_blank">17630976</a>]<br></li>
<li>In turn , an ssaV : : lac transcriptional fusion was expressed under SPI-2-in-ducing conditions , and its expression was dependent on PhoP and SsrB ( Fig. 3B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17616593" target="_blank">17616593</a>]<br></li>
<li>( B ) Summary of gene expression data and SsrB binding data for pathogenicity islands , prophage islands , S. bongori ROD and genome sequence not in islands .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20300643" target="_blank">20300643</a>]<br></li>
<li>As seen in Fig. 4A and B , PhoP and SsrB were necessary for expression of srfJ in LPM .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=22661691" target="_blank">22661691</a>]<br></li>
<li>In HeLa cells ( epithelial ) , where bacteria actively induce their own uptake using the SPI-1 T3SS , substantial induction of expression from PsseA occurred after 1 h in an SsrB - and an OmpR-dependent fashion .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16304611" target="_blank">16304611</a>]<br></li>
<li>The intracellular expression of PipB was absolutely dependent on SsrB in both RAW 264.7 macrophagelike cells ( Fig. 2C ) and HeLa epithelial cells ( not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11918798" target="_blank">11918798</a>]<br></li>
<li>At pH 7.4 , expression of SsrB was dependent on OmpR , as in its absence , SsrB levels were considerably reduced ( Fig. 11 , lanes 2 and 7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15491370" target="_blank">15491370</a>]<br></li>
<li>There is evidence suggesting that SsrB can be autoregulated and can activate SsrA expression , albeit to a lesser extent ( 237 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23554419" target="_blank">23554419</a>]<br></li>
<li>The expression of sifA was found to be strictly regulated by SsrA/SsrB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11755424" target="_blank">11755424</a>]<br></li>
<li>Analysis of infected macrophages using ﬂuorescence-activated cell sorting revealed that sseL is induced inside cells and requires SsrB for expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17158898" target="_blank">17158898</a>]<br></li>
<li>The speciﬁc signal that is received by sensor SsrA leading to activation of response regulator SsrB and expression of SPI2 genes is not known to date .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16904940" target="_blank">16904940</a>]<br></li>
<li>Therefore , it makes sense that their expression is not strictly dependent on SsrB since they need to be expressed under conditions that prevent activation of SsrB and SPI2 expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23504014" target="_blank">23504014</a>]<br></li>
<li>sciS was maximally transcribed at 27 h postinfection and is repressed by SsrB , an activator of genes required for promoting intracellular survival .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15972528" target="_blank">15972528</a>]<br></li>
<li>The Salmonella-speciﬁc sseL gene , controlled by the SPI-2 regulator SsrB , has recently been described as encoding a translocated deubiquitinase that is required for macrophage killing and systemic virulence in mice ( 15 , 72 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17483226" target="_blank">17483226</a>]<br></li>
<li>B. PipB expression is induced by SPI-2 growth conditions and is dependent on the SPI-2 transcriptional activator SsrB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11918798" target="_blank">11918798</a>]<br></li>
<li>srfA and srfB , together with srfC , appear to form a 7-kb operon that is part of a putative horizontally acquired DNA segment that was described as regulated by SsrB ( 93 ) , a regulator encoded within SPI-2 that is required for SPI-2-encoded T3SS expression ( 17 , 47 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17616593" target="_blank">17616593</a>]<br></li>
<li>We previously observed that this mutant strain defective in the SPI2 encoded transcriptional activator SsrB did not express SPI2 genes under in vitro conditions inducing the SPI2 system [ 21 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12112863" target="_blank">12112863</a>]<br></li>
<li>Interestingly , genes encoding the central regulators HilA , SsrB , and FlhC were shown to be repressed in strains unable to express YfgL .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17060472" target="_blank">17060472</a>]<br></li>
<li>PipB expression was dramatically reduced in a ssrB : : kan mutant ( Fig. 2B ) , demonstrating that PipB is part of the SsrB regulon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11918798" target="_blank">11918798</a>]<br></li>
<li>Intracellular induction of HA-tagged PipB expression is not detected for an SsrB mutant ( ssrB : : kan ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11918798" target="_blank">11918798</a>]<br></li>
<li>PmrA Represses ssaG Transcription Indirectly , by Hindering Expression of the ssaG Activator SsrB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23690578" target="_blank">23690578</a>]<br></li>
<li>Expression ratio comparing WT to ssrB indicates that SsrB activates SPI-2 gene expression ; Expression ratio comparing ssrB to ( ssrB , rpoE ) indicates that the up-regulating effects of σE on SPI-2 gene expression are through SsrB in LPM 4 h condition .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25713562" target="_blank">25713562</a>]<br></li>
<li>ssrB : : FLAG wt rpoE Δ DnaK SsrB-FLAG pellet Our results indicate that rpoE deletion has no effect on SsrB levels under SPI-2 inducing conditions suggesting that the σE pathway regulates effector expression downstream of ssrAB transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19245712" target="_blank">19245712</a>]<br></li>
<li>The most important and essential regulatory system required for SPI-2 gene expression is the two-component regulatory system ( 2CRS ) SsrA-SsrB , encoded in a single locus located within SPI-2 ( for a deﬁnition of 2CRS see below ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23554419" target="_blank">23554419</a>]<br></li>
<li>It has been shown recently that transcription of ssrA , a gene encoding a component of SsrA/SsrB , the SPI-2 two-component regulatory system [ 46 -- 48 ] , is modulated by OmpR in infected tissue cultured cells [ 49 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11755424" target="_blank">11755424</a>]<br></li>
<li>The expression of the SsrA-SsrB two-component system is regulated by the OmpR-EnvZ two-component system by OmpR binding directly to the ssrAB promoter [ 24 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21320585" target="_blank">21320585</a>]<br></li>
<li>Furthermore , we found that expression of stm3169 was controlled by ppGpp and SsrB , a response regulator of the two-component system located on Salmonella pathogenicity island 2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21176126" target="_blank">21176126</a>]<br></li>
<li>The SsrB protein is necessary for transcription of the genes specifying the T3SS encoded in the SPI-2 pathogenicity island as well as its secreted effectors ( 19 , 20 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23690578" target="_blank">23690578</a>]<br></li>
<li>Recent works have shown that several genes located outside of SPI-2 are under the control of the SsrA-SsrB regulator as well , and these have been proposed as putative virulence factors ( 10 , 25 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19717598" target="_blank">19717598</a>]<br></li>
<li>Use of transcriptional reporter fusions showed that expression of sspH2 , sseI , sseJ , sifA and sifB is dependent on the SPI-2 TTSS regulator SsrA-SsrB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12864810" target="_blank">12864810</a>]<br></li>
<li>Given that SsrB functions as an activator for expression of SPI-2 genes ( 4 ) and that Fur represses ssrB expression ( Fig. 3 ) , it is possible that the Fur regulator controls SPI-2 genes in a fashion dependent on the SsrB regulator .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24643535" target="_blank">24643535</a>]<br></li>
<li>SsrA-SsrB controls expression of the genes of the SPI-2 T3SS , as well as genes encoding SPI-2 effectors located within SPI-2 and elsewhere on the chromosome [ 15,19 e23 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21320585" target="_blank">21320585</a>]<br></li>
<li>SsrB activates the expression of SPI-2-related genes , both inside and outside the pathogenicity island .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12949164" target="_blank">12949164</a>]<br></li>
<li>Among them , the SsrA/SsrB two-compo-nent system , which is encoded by the ssrA and ssrB genes located within SPI-2 , seems to have the most direct effect on SPI-2 expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21059960" target="_blank">21059960</a>]<br></li>
<li>The expression of SPI-2 invasion genes and effector proteins is dependent upon the presence of the SsrA/SsrB two-component regulatory system encoded by SPI-2 ( 11 , 18 , 44 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15039320" target="_blank">15039320</a>]<br></li>
<li>The expression of SPI-2 genes and effector genes located outside the pathogenicity island requires the SPI-2-encoded two-component regulatory system SsrA-SsrB ( 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17360673" target="_blank">17360673</a>]<br></li>
<li>The transcription factor SsrB is required for the expression of the T3SS encoded within SPI2 and for the expression of its substrate effector proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17158898" target="_blank">17158898</a>]<br></li>
<li>Since intracellular growth and suvival of Salmonella in macrophages is dependent upon SPI-2 function , we next confirmed whether expression of stm3169 is regulated by the SsrAB two-component system , which positively controls the expression of SPI-2 genes as well as other genes belonging to the SsrB regulon [ 32 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21176126" target="_blank">21176126</a>]<br></li>
<li>RESULTS Identification of flagellin protein by MALDI-TOF MS The spiC gene is adjacent to the SpiR ( SsrA ) / SsrB two-component regulatory genes , and is the initial gene of operons that encode the structural and secretory components of SPI-2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=18957602" target="_blank">18957602</a>]<br></li>
<li>The expression of the SPI-1 secretion system and the expression of its secreted effectors are known to be coordinately regulated by HilA , a member of the OmpR/ToxR family of transcriptional regulators , in S. Typhimurium ( Lucas and Lee , 2000 ) and the expression of genes encoding the type III secretion system of SPI-2 is dependent on a sensor-kinase system , SsrA-SsrB ( Lee et al. , 2000 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21046347" target="_blank">21046347</a>]<br></li>
<li>OmpR activates SPI-2 genes by binding to the promoter of the ssrAB operon to induce expression of SsrA and SsrB proteins [ 16 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20861532" target="_blank">20861532</a>]<br></li>
<li>SsrB , the response regulator of the SsrA/SsrB two-component system , directly activates expression of SPI-2 genes under acidic conditions ( 5 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24643535" target="_blank">24643535</a>]<br></li>
<li>One of the interesting features of autonomous gogB expression was its co-regulation with other SPI-2 virulence genes by SsrB , the transcriptional activator of the SsrA/SsrB two-component regulatory system .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15843015" target="_blank">15843015</a>]<br></li>
<li>As expression of the SsrB protein is promoted by several transcription factors , including SsrB itself , the expression of SPI-2 genes can increase to undesirable levels under activating conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21059960" target="_blank">21059960</a>]<br></li>
<li>Consistent with these ﬁndings , the H-NS protein represses expression of the SsrB-target genes within SPI-2 , which is relieved by activation of the SsrB protein ( 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21059960" target="_blank">21059960</a>]<br></li>
<li>One of the most interesting genes regulated by SsrB outside SPI-2 is srfJ .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10844662" target="_blank">10844662</a>]<br></li>
<li>Expression of SPI-2 genes is mainly regulated through the SsrA-SsrB two-component system .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20861532" target="_blank">20861532</a>]<br></li>
<li>SPI-2 gene expression is dependent on the SsrA/SsrB two-component system ; where the SsrB response regulator activates SPI-2 structural genes encoding a T3SS [ 67,68 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24603618" target="_blank">24603618</a>]<br></li>
<li>A number of directly regulated gene products then regulate additional regulatory cascades , including PmrD , which is able to activate the PmrAB operon independent of iron ; SlyA , which regulates genes important to intra-macrophage survival such as pagC and ugtL ; IraP which prevents MviA-dependent degradation of RpoS leading to RpoS accumulation and its regulation of genes important for stationary phase survival and resistance to oxidative stress ; HilA , which is an inducer of SPI-1 ( Salmonella pathogenicity island-1 ) , which contains genes involved in invasion of epithelial cells ; and SsrB , which is an inducer of SPI-2 containing genes important in intra-macrophage survival ( adapted from Groisman E. and Mouslim C. Nature Reviews Microbiology ( 2006 ) 4:705 -- 709 ) [ 44,45 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19746165" target="_blank">19746165</a>]<br></li>
<li>SsrB is autoregulated and activates expression of SPI-2 and SPI-2 co-regulated genes in response to an unknown signal ( s ) ( Feng et al. , 2003 ; 2004 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630976" target="_blank">17630976</a>]<br></li>
<li>In addition , the SsrB regulator also directly binds and autoregulates the ssrA and ssrB promoters to activate their expression ( 7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24643535" target="_blank">24643535</a>]<br></li>
<li>Similar to many other two-component regulatory systems , the expression of SsrA/SsrB is positively autoregulated ; the SsrB protein binds to ssrA and ssrB promoters and activates their transcription ( 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21059960" target="_blank">21059960</a>]<br></li>
<li>These genes are located in the SPI2 pathogenicity island and are under the control of the SpiR-SsrB two-component regulatory system : SpiR ( sometimes called SsrA ) is a cytoplasmic membrane located sensor kinase and SsrB is a response regulator ( Fig. 2 ) ( Feng et al. , 2003 ; Fass and Groisman , 2009 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28631437" target="_blank">28631437</a>]<br></li>
<li>SPI-2 encodes for T3SS and the expression of genes is governed by OmpB-OmpR and SsrA-SsrB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20861532" target="_blank">20861532</a>]<br></li>
<li>The expression of SPI2 , and hence T3SS2 , is directly controlled by the SsrA/SsrB two-component system , which is encoded by the ssrA and ssrB genes located within SPI2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23504014" target="_blank">23504014</a>]<br></li>
<li>In contrast , sspH1 and slrP expression was independent of both SsrA-SsrB and the SPI-1 regulator HilA , suggesting that these two genes might be expressed constitutively ( Miao and Miller , 2000 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12864810" target="_blank">12864810</a>]<br></li>
<li>Both SPI1 and SPI2 also encode transcription factors ( HilA and SsrB , respectively ) that are required for expression of the linked T3SS and the effectors they deliver into host cells ( 11 , 12 ) , indicating that these SPIs were acquired as functional units .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25249283" target="_blank">25249283</a>]<br></li>
<li>SsrB binds upstream of the sifA , sifB , and sseJ effector genes and directly regulates transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21059643" target="_blank">21059643</a>]<br></li>
<li>The promoters we chose use the SsrB response regulator for expression of the downstream gene or operon , and include both SPI-2-encoded and non-SPI-2-encoded virulence effectors representing structural apparatus genes and effector substrates of the type III secretion system [ 8,30-35 ] Chemiluminescent β-galactosidase Assay Reporter strains were inoculated from an overnight culture into culture medium ( LPM pH 5.8 ) that induces SsrBdependent gene expression [ 21,36 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19245712" target="_blank">19245712</a>]<br></li>
<li>We also found CDS genes that were dif-ferentially-expressed ( > 3-fold ) in the absence of both HilD and SsrB , and the majority of these 52 HilD / SsrB-controlled genes were virulence-associated ( Fig 3A and 3B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27564394" target="_blank">27564394</a>]<br></li>
<li>Since PhoP regulates expression of ssrA and ssrB ( Bijlsma and Groisman , 2005 ) , it also regulates expression of genes in the SsrB regulon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29270156" target="_blank">29270156</a>]<br></li>
<li>SirA activates the expression of SsrB and HilA through HilD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21518393" target="_blank">21518393</a>]<br></li>
<li>To determine if SirA controls ssaG expression , and thus of other SPI-2 genes , by regulating the ssrAB operon , we analyzed the expression of SsrB in WT and ΔsirA strains , as well as in the ΔhilA , ΔhilD , ΔhilC , and ΔompR mutants , which were tested as controls .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21518393" target="_blank">21518393</a>]<br></li>
<li>Results SsrB supports biofilm formation in the absence of its kinase SsrA The SsrA/B TCRS is activated by environmental stimuli such as pH and osmolality , stimulating the expression of virulence genes essential for intra-cellular growth and survival of Salmonella ( Feng et al. , 2003 ; 2004 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26880544" target="_blank">26880544</a>]<br></li>
<li>As such , S. bongori has evolved in the absence of SsrA-SsrB and its regulatory architecture has not been influenced by it .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20300643" target="_blank">20300643</a>]<br></li>
<li>These data indicate that SsrB plays a role in regulating the expression of STM2234 , STM2235 , STM2236 , and STM2239 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23630960" target="_blank">23630960</a>]<br></li>
<li>Deletion of PsspH2 resulted in loss of STM2239 expression ( Fig. 5C ) , which is consistent with STM2240 and STM2239 transcription by SsrB via transcriptional read-through from the sspH2 promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=23630960" target="_blank">23630960</a>]<br></li>
<li>From the data shown in Fig. 2C , it is apparent that both OmpR and SsrB activate the transcriptional fusions to spiC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12753201" target="_blank">12753201</a>]<br></li>
<li>In contrast , we provide evidence that srfABC is not under the control of SsrB ( Fig. 3 and 4 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17616593" target="_blank">17616593</a>]<br></li>
<li>Similarly , SsrB is thought to directly activate the expression of SPI2 operons as well as other effector genes located outside of SPI2 [ 28,29 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17233907" target="_blank">17233907</a>]<br></li>
<li>Therefore , expression of SPI-2 genes is not induced until the pH is lowered to 5.5 , where levels of SsrB would markedly increase ( Fig. 6A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24643535" target="_blank">24643535</a>]<br></li>
				<label class="TU_collapse_similarity" for="tu_similarity_122">
Because ssrB expression is positively autoregulated ( 7 ) , the levels of SsrB gradually increase as the pH is lowered from pH 7.0 to pH 5.5 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24643535" target="_blank">24643535</a>]<br></label>
<input id="tu_similarity_122", type="checkbox">
<div class="TU_similar">
<li>Because SsrB expression is positively autoregulated ( 7 ) , the levels of SsrB gradually increase as the external pH is lowered from pH 7.0 to pH 5.5 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24643535" target="_blank">24643535</a>]<br></li></div>
<li>We show that under biofilm-inducing conditions , unphosphorylated SsrB is sufficient to activate the expression of csgD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26880544" target="_blank">26880544</a>]<br></li>
<li>By using this strategy , we previously demonstrated that HilD is required for the expression of SsrB in the presence of H-NS but not when it is inactivated ( 21 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25135218" target="_blank">25135218</a>]<br></li>
<li>It is noteworthy that a laterally acquired gene product , SsrB , has evolved the job of regulating the levels of csgD , a transcriptional regulator encoded by the core genome .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26880544" target="_blank">26880544</a>]<br></li>
<li>These data demonstrate that expression of genes in the SsrAB virulon is affected to different extents by the transcriptional activator SsrB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19858298" target="_blank">19858298</a>]<br></li>
<li>As predicted , SsrB activates transcription at promoters found upstream of the ﬁrst gene of each functional cluster ( ssaB , sseA and ssaG ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630976" target="_blank">17630976</a>]<br></li>
<li>An obvious null hypothesis was therefore to test whether SsrB ( in its unphosphorylated state ) , activated the expression of csgD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26880544" target="_blank">26880544</a>]<br></li>
<li>As shown in Fig. 8 , expression of SsrB during growth in Luria broth induced transcription of genes previously shown to be SsrB-dependent .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630976" target="_blank">17630976</a>]<br></li>
<li>Thus , the CTD was not required for SsrB - dependent transcription of C sifA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21059643" target="_blank">21059643</a>]<br></li>
<li>H-NS specifically repressed transcription of sifA , but this repression was antagonized by the presence of SsrB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21059643" target="_blank">21059643</a>]<br></li>
<li>At higher concentrations of arabinose inducer , Fis protein expression reached a maximum ( A ) , SsrB was undetectable ( B ) , the pUC18 reporter plasmid became highly relaxed ( C ) , and ssrA transcription was higher than the repressed control but not as high as the level seen with 0.2 % arabinose ( D ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=16999831" target="_blank">16999831</a>]<br></li>
<li>Consistent with the transcription data , SsrB protein levels were 3.5-fold higher in the absence of fur than in its presence under pH 6.0 conditions , but were similar under pH 7.0 conditions regardless of fur background ( Fig. 3C and D ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24643535" target="_blank">24643535</a>]<br></li>
<li>An estimate of csgD expression by ( C ) Real-time qRT-PCR showed a significant decrease in csgD transcription in the ssrB null , but not in the D56A SsrB and ssrA mutants .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26880544" target="_blank">26880544</a>]<br></li>
<li>Furthermore , previous microarray studies disrupted both ssrA and ssrB , which would not uncover a distinct role for SsrB in gene regulation under non SPI-2-inducing conditions in the absence of the SsrA kinase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26880544" target="_blank">26880544</a>]<br></li>
<li>Complementation strains of the varN mutant ( cVarN and DvarNþpSsrB ) were generated by the expression of the functional varN or ssrB genes on a low-copy-number plasmid , pWSK129 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29751061" target="_blank">29751061</a>]<br></li>
<li>Salmonella expressing the SsrB C203S variant also supported the expression of the sifA : : lacZY , srfJ : : lacZY , and sspH2 : : lacZY transcriptional fusions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20660761" target="_blank">20660761</a>]<br></li>
<li>To follow the expression of SsrB , the ssrB chromosomal gene was tagged with the sequence encoding a 3XFLAG epitope , rendering strains expressing a C-terminal 3XFLAG-tagged SsrB protein ( SsrB-FLAG ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21518393" target="_blank">21518393</a>]<br></li>
<li>Fur represses expression of the ssrB gene encoding the SsrB response regulator under acidic conditions .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24643535" target="_blank">24643535</a>]<br></li>
<li>The intracellular expression of a sifA : : lacZY transcriptional fusion was higher in the strain AV08270 expressing SsrB C203S than in the AV08267 control bearing the wild-type ssrB allele ( Fig. 4B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20660761" target="_blank">20660761</a>]<br></li>
<li>When grown under pH 6.0 conditions with the inducer IPTG , the ssrB mutant carrying the SsrB expression plasmid expressed ssaG mRNA at wild-type levels , which increased by 2.5-fold in the absence of the fur gene ( Fig. 5B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24643535" target="_blank">24643535</a>]<br></li>
<li>In the ssrB deletion strain harboring the pJJ16 plasmid in which the SsrB protein was induced from the lac promoter , transcription of the ssaG gene recovered in acidiﬁed medium containing IPTG ( Fig. 2B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21059960" target="_blank">21059960</a>]<br></li>
<li>Under inducing conditions , although H-NS repression can still be observed in the absence of SsrB , the effects of H-NS on SPI-2 expression are minimal in the presence of SsrB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630976" target="_blank">17630976</a>]<br></li>
<li>Is the SsrB expressed in the absence of envZ dependent on OmpR ?	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15491370" target="_blank">15491370</a>]<br></li>
<li>To verify that expression of SseL was under the control of SsrB , we cloned the native sseL promoter region in front of a cloned sseL gene containing tandem HA epitopes at the C terminus .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17158898" target="_blank">17158898</a>]<br></li>
<li>E UP-REGULATES SPI-2 GENE EXPRESSION BY REPRESSING hns TRANSCRIPTION IN THE CONDITION THAT MIMICS LATE INFECTION In the condition that mimics late infection ( LPM 20 h ) , the expression of ssrB was unaffected by the presence of σE ( Figure 4A ) , suggesting that the general activator SsrB was not causing the up-regulation of SPI-2 genes at this time point .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=25713562" target="_blank">25713562</a>]<br></li>
<li>Surprisingly , the SsrB response regulator positively regulated the formation of biofilms by activating csgD expression in the absence of any phos-pho-donors .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26880544" target="_blank">26880544</a>]<br></li>
<li>In accordance with previous observations ( 32 ) , the expression of the slrP reporter fusion was not affected by the presence or absence of SsrB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19858298" target="_blank">19858298</a>]<br></li>
<li>the absence of SsrB , essentially no transcription from the sseI promoter was detected ( Fig. 7 , column 1 ) , whereas expression residues are underlined .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19126546" target="_blank">19126546</a>]<br></li>
<li>SsrB expression resulted in activation of the sseI promoter , whereas expression of SsrB D56A did not activate sseI ( Fig. 9 , columns 1 -- 3 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19126546" target="_blank">19126546</a>]<br></li>
<li>SsrB was able to activate transcription of sifA in the C presence ( 1.7-fold ) or absence ( 3-fold ) of CTD .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21059643" target="_blank">21059643</a>]<br></li>
<li>Growth of MJW141 in the presence of arabinose results in expression of SsrB , whereas growth in its absence does not .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10844662" target="_blank">10844662</a>]<br></li>
<li>Expression of GogB by Salmonella may require Expression of GogB is reduced in an ssrB mutant background under SPI-2-inducing conditions Using a technique to epitope-tag chromosomal genes in Salmonella , it was reported that GogB expression was under the control of the SPI-2 two-component regulatory system , SsrA/SsrB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=15843015" target="_blank">15843015</a>]<br></li>
<li>Note that expression of the thioredoxin-fused SsrB protein was able to restore ssaG transcription in ssrB mutant , indicating that the thioredoxin fusion did not interfere with regulatory function of SsrB ( Fig .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21059960" target="_blank">21059960</a>]<br></li>
<li>These results indicate that overexpression of SsrB in the absence of the gene for its cognate kinase , ssrA , is suf-ﬁcient to activate SPI-2 transcription .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630976" target="_blank">17630976</a>]<br></li>
<li>SsrB activates SPI-2 in the absence of SsrA When an ssrAB double mutant was complemented with a plasmid containing an inducible ssrB gene , we were surprised to ﬁnd that expression of SsrB in the absence of its cognate kinase is sufficient to activate transcription of SPI-2 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630976" target="_blank">17630976</a>]<br></li>
<li>Furthermore , we found that overexpression of SsrB in the varN mutant restored expression of SPI-2 genes ( ssaV and sifA ) to a level higher than that of the WT strain when bacteria wer grown in RAW264 .7 macrophages ( Supplementary Fig. 2A ) , and also rescued the growth defect of the varN mutant in murine RAW264 .7 macrophages to WT level ( Supplementary Fig. 2B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=29751061" target="_blank">29751061</a>]<br></li>
<li>Similarly , expression of the SsrB mutant ability of SsrBC to dimerize and is the likely cause of its inability C45A in the wild-type background also resulted in a 2-fold to activate sseI transcription ( Fig. 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=19126546" target="_blank">19126546</a>]<br></li>
<li>Overexpression of SsrB in the absence of ssrA activates transcription of SPI-2 and SPI-2 co-regulated genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630976" target="_blank">17630976</a>]<br></li>
<li>In the absence of hns , the requirement for SsrB in activating SPI-2 genes was dramatically reduced , suggesting a role for SsrB in countering H-NS silencing at SPI-2 ( Fig. 10 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=17630976" target="_blank">17630976</a>]<br></li>
<li>First , EIIA - mediated regulation of SPI-2 genes was dependent on the SsrB protein because ptsN mutation failed to increase SPI-2 expression in the Ntr absence of SsrB ( Fig. 2A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21059960" target="_blank">21059960</a>]<br></li>
				</div></ul></div><br>
	</div>
</body>
</html>