<!doctype html>
<style>
@charset "utf-8";
body {
	height: 100%;
	margin: 0;
	padding: 0;
}

.banner {
	position: fixed;
	top: 0;
	width: 100%;
	height: 65px;
	background: gray;
	line-height: 65px;
}
.spacer {
	height: 10px;
}

.main-content {
	padding: 5%;
}

.collapse_ACT {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-style: none solid none none;
	border-color: #B4045F;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_ACT + input{
	display: none;
}
.collapse_ACT + input + div{
	display: none;
}
.collapse_ACT + input:checked + div{
	display: block;
}
.ACT {
	text-align: justify;
	text-justify: inter-word;
}
.ACT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B4045F;
}
.ACT_collapse_similarity:hover{
	background: #e5d3dc;
}
.ACT_collapse_similarity + input {
	display: none;
}
.ACT_collapse_similarity + input + div{
	display: none;
}

.ACT_collapse_similarity + input:checked + div{
	display: block;
}
.ACT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.ACT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B4045F;
	padding-left: 5px;
}
.ACT_collapse_similarity:hover{
	background: #e5d3dc;
}
.ACT_collapse_similarity + input {
	display: none;
}
.ACT_collapse_similarity + input + div{
	display: none;
}
.ACT_collapse_similarity + input:checked + div{
	display: block;
}
.ACT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.ACT_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #B4045F;
}
.ACT_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_DOM {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-style: none solid none none;
	border-color: #B404AE;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_DOM + input{
	display: none;
}
.collapse_DOM + input + div{
	display: none;
}
.collapse_DOM + input:checked + div{
	display: block;
}
.DOM {
	text-align: justify;
	text-justify: inter-word;
}
.DOM_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B404AE;
	padding-left: 5px;
}
.DOM_collapse_similarity:hover{
	background: #e8d7e7;
}
.DOM_collapse_similarity + input {
	display: none;
}
.DOM_collapse_similarity + input + div{
	display: none;
}
.DOM_collapse_similarity + input:checked + div{
	display: block;
}
.DOM_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}.DOM_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #B404AE;
}
.DOM_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_EVO {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #7401DF;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_EVO + input{
	display: none;
}
.collapse_EVO + input + div{
	display: none;
}
.collapse_EVO + input:checked + div{
	display: block;
}
.EVO {
	text-align: justify;
	text-justify: inter-word;
}
.EVO_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #7401DF;
	padding-left: 5px;
}
.EVO_collapse_similarity:hover{
	background: #e2d7ed;
}
.EVO_collapse_similarity + input {
	display: none;
}
.EVO_collapse_similarity + input + div{
	display: none;
}
.EVO_collapse_similarity + input:checked + div{
	display: block;
}
.EVO_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.EVO_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #7401DF;
}
.EVO_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_RP {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #0174DF;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_RP + input{
	display: none;
}
.collapse_RP + input + div{
	display: none;
}
.collapse_RP + input:checked + div{
	display: block;
}
.RP {
	text-align: justify;
	text-justify: inter-word;
}
.RP_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #0174DF;
	padding-left: 5px;
}
.RP_collapse_similarity:hover{
	background: #c9d7e2;
}
.RP_collapse_similarity + input {
	display: none;
}
.RP_collapse_similarity + input + div{
	display: none;
}
.RP_collapse_similarity + input:checked + div{
	display: block;
}
.RP_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.RP_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #0174DF;
}
.RP_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_SIT {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #0489B1;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_SIT + input{
	display: none;
}
.collapse_SIT + input + div{
	display: none;
}
.collapse_SIT + input:checked + div{
	display: block;
}
.SIT {
	text-align: justify;
	text-justify: inter-word;
}
.SIT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #0489B1;
	padding-left: 5px;
}
.SIT_collapse_similarity:hover{
	background: #d0dfe2;
}
.SIT_collapse_similarity + input {
	display: none;
}
.SIT_collapse_similarity + input + div{
	display: none;
}
.SIT_collapse_similarity + input:checked + div{
	display: block;
}
.SIT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.SIT_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #0489B1;
}
.SIT_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_TU {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #04B486;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_TU + input{
	display: none;
}
.collapse_TU + input + div{
	display: none;
}
.collapse_TU + input:checked + div{
	display: block;
}
.TU {
	text-align: justify;
	text-justify: inter-word;
}
.TU_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #04B486;
	padding-left: 5px;
}
.TU_collapse_similarity:hover{
	background: #d0e2de;
}
.TU_collapse_similarity + input {
	display: none;
}
.TU_collapse_similarity + input + div{
	display: none;
}
.TU_collapse_similarity + input:checked + div{
	display: block;
}
.TU_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.TU_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #04B486;
}
.TU_similar > a {
	text-decoration: none;
	color: gray;
}

ul { 
	list-style-type: none;
	padding: 0;
	margin: 0;
}
ul li > a {
	text-decoration: none;
	color: gray;
}
ul li:hover, ul li:focus {
	background: #F0F0F0;
}

ul li:first-child span {
  border-left:none;  
}

</style>
<html>
<head>
<meta charset="utf-8">
<title>TF properties - YfhA</title>
<meta name="theme-color" content="#ffffff">
</head>
<body>
	<div class="banner">
		<center><font color="white" size="16px" face="arial">YfhA</font></center>
	</div>
	<div align="center" class="main-content">
		<div class="spacer">
	</div><br>			<label class="collapse_ACT" for="_1"><font size="5px">YfhA transcriptional activity</font></label>
				<input id="_1" type="checkbox">
				<div class="ACT"><br><ul><li>It may be noted here that Tfsitescan could not identify any binding sites for YfhA , probably due to absence of this information in its backend database .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24079299" target="_blank">24079299</a>]<br></li>
<li>Since EnvZ is a known osmolarity sensor for SPI-2 genes [ 26 ] , it is likely that YfhA regulates T6SS activation ( through RcsB ) depending on the osmolarity sensed by EnvZ .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24079299" target="_blank">24079299</a>]<br></li>
<li>It has also been observed that the osmolarity sensor EnvZ , whose cognate response-regulator is OmpR , can additionally phosphorylate a non-cognate response-regulator YfhA in E. coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24079299" target="_blank">24079299</a>]<br></li>
<li>Considering that the osmolarity sensor EnvZ is not known to have any other non-cognate response-regulator , YfhA was further investigated for its possible role in osmoregulation of T6SS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24079299" target="_blank">24079299</a>]<br></li>
<li>Hence , it is likely that there exists a similar mechanism of phosphorylated YfhA ( YfhA-P ) mediated regulation of sciS and rcsB .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24079299" target="_blank">24079299</a>]<br></li>
<li>YhfA is normally known to be a cognate response-regulator of the YfhK-YfhA two-component system [ 28 ] , which gets activated during stationary phase of growth ( a condition associated with T6SS activation in S. typhimurium ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24079299" target="_blank">24079299</a>]<br></li>
<li>Furthermore , simulation of the Boolean model with a ` mutated ' YfhA did not result in T6SS activation , indicating the probable crucial role of YfhA in the constructed network .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24079299" target="_blank">24079299</a>]<br></li>
<li>Although , validating the effect of this FFL in improving network robustness is outside the scope of this work , the inclusion of YfhA ( and the FFL ) probably makes the T6SS network better equipped to sense and respond to the environmental cues .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24079299" target="_blank">24079299</a>]<br></li>
<li>Sequential activation of SPI-1 , SPI-2 and T6SS through simulation of Boolean models Simulation of the Boolean model was performed in order to study the dynamic behavior of the regulatory network and to verify the fitness of the predicte interactions involving YfhA in the model .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24079299" target="_blank">24079299</a>]<br></li>
<li>Although including this ` mutant ' in the simulation did n't affect the sequential activation and repression of SPI-1 and SPI-2 , the model failed to reach a state with T6SS turned ` ON ' ( Additional file 9 ) , indicating the importance of YfhA in the network .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24079299" target="_blank">24079299</a>]<br></li>
<li>Probable role of YfhA in osmoregulation of T6SS The presence of several binding sites of YfhA ( and its homolog FlrC ) in the upstream regions and ORFs of rcsB and sciS , coupled to the fact that YfhA is also a non-cognate response-regulator of EnvZ , strongly indicates the involvement of YfhA in osmoregulation of T6SS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24079299" target="_blank">24079299</a>]<br></li>
<li>On the other hand , once the response-regulator YfhA gets phosphorylated by EnvZ under high levels of osmolarity , it is likely that it additionally binds to the ORFs of sciS and rcsB , thus limiting the transcription of these two genes .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24079299" target="_blank">24079299</a>]<br></li>
				</div></ul><br>
			<label class="collapse_DOM" for="_2"><font size="5px">YfhA protein domains</font></label>
				<input id="_2" type="checkbox">
				<div class="DOM"><br><ul><li>It may be noted that , the binding sites predicted in this study are based on the motifs recognized by two YfhA homologs from E. coli and V. cholerae .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24079299" target="_blank">24079299</a>]<br></li>
<li>The sequence segments corresponding to these three distinct domains of FlrC and YfhA were identified by searching against the Pfam database [ 33 ] ( Additional file 6 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24079299" target="_blank">24079299</a>]<br></li>
<li>Additional file 6 : Pfam Domains identified in FlrC and YfhA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24079299" target="_blank">24079299</a>]<br></li>
<li>Model structures of REC , AAA , and HTH domains of FlrC and YfhA proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24079299" target="_blank">24079299</a>]<br></li>
<li>Additional file 7 : Homology modeling of three domains of FlrC and YfhA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24079299" target="_blank">24079299</a>]<br></li>
<li>Subsequently , pair-wise structural superposition was performed using the models for each of the domains of FlrC and YfhA ( Additional file 7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24079299" target="_blank">24079299</a>]<br></li>
<li>Details pertaining to homology modeling of the 3 distinct domains from the proteins FlrC and YfhA , and the results obtained through pair-wise superposition of the corresponding domains from the two different proteins .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24079299" target="_blank">24079299</a>]<br></li>
				</div></ul><br>
			<label class="collapse_EVO" for="_3"><font size="5px">YfhA evolution</font></label>
				<input id="_3" type="checkbox">
				<div class="EVO"><br><ul><li>Structural homology between YfhA of S. typhimurium and FlrC of V. cholerae A structural comparison of Vibrio FlrC and Salmonella YfhA ( having 62 % sequence similarity ) was performed to check the extent of their structural homology , which would imply a similar function .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24079299" target="_blank">24079299</a>]<br></li>
<li>Figure 3 Structural homology of FlrC and YfhA .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24079299" target="_blank">24079299</a>]<br></li>
<li>BLAST similarity searches indicated that the protein STM2562 of S. typhimurium is a close homolog of the E.coli YfhA ( 95 % identity , 100 % coverage ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24079299" target="_blank">24079299</a>]<br></li>
<li>A homology search in S. typhimurium indicated the protein YfhA to be the closest homolog of FlrC ( of V. cholerae ) with a sequence similarity of 62 % , strengthening the possibility of YfhA 's involvement in regulation of T6SS in S. typhimurium .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24079299" target="_blank">24079299</a>]<br></li>
				</div></ul><br>
			<label class="collapse_RP" for="_4"><font size="5px">Processes regulated by YfhA</font></label>
				<input id="_4" type="checkbox">
				<div class="RP"><br><ul><li>In addition , the predicted regulatory cascade involving YfhA ( see ` Results and Discussion ' ) , wa also included .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24079299" target="_blank">24079299</a>]<br></li>
<li>YfhA ha also been shown to act as a non-cognate response regulator of the histidine kinase EnvZ [ 27 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24079299" target="_blank">24079299</a>]<br></li>
<li>An interesting outcome of the current study is the identification of YfhA as a potential transcriptional regulator involved in the sub-network corresponding to T6SS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24079299" target="_blank">24079299</a>]<br></li>
<li>Introduction of YfhA in the T6SS sub-network also results in formation of a feed-forward loop involving YfhA , RcsB and SciS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24079299" target="_blank">24079299</a>]<br></li>
<li>sN-dependent enhancer binding protein ( EBP ) Proposed functions of genes References regulated by EBP Y. pestis S. typhimurium S. Typhi E. coli NtrC PspF YfhA YgaA NtrC PspF YfhA YgaA NtrC PspF YfhA YgaA Reitzer and Magasanik , 1986 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12125817" target="_blank">12125817</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12581721" target="_blank">12581721</a>]<br></li>
<li>DNA topoiomerase I Similar to MarT and ToxR transcriptional regulators Cold shock protein Multidrug-transporter ClpP protease Putative siderophore-binding protein Putative regulatory protein , possibly involved in gluconate utilisation Purine biosynthesis Putative transcriptional regulator Primosomal protein Putative heavy metal transport/resistance protein PhoPQ-activated gene Putative protein Putative outer membrane efﬂux protein STM3521 yeaG astC yjeB STM2746 topA STM0031 cspB acrD clpP yrdA yieP TGGCACGGAGTTAGCC TGGCATCCCCTTTGCG TGGAACGCCTCCTGCT TGGCAAGGTCTATGCA TGGCACTGTAGTGGCA TGGCACCCTGGGTGCA TGGCACGCGCGTCGCC purH ytfJ dnaT ygjT pagK yfcZ STM2690 B Score Gene name Promoter sequence Gene product/function Associated EBP ( Kullback-Liebler distance ) STM4535 STM3772 STM0577 STM2360 TGGCACGCCGCTTGCT TGGCACAACCTTTGCT TGGCACGCCGTTTGCC TGGCATGCCTTTTGCT Putative PTS EII component Putative PTS EII component Putative PTS EII component Putative diaminopimelate decarboxylase Nitrogen regulatory protein Propionate metabolism Hydrogenase accessory protein Resistance to zinc and lead Putative hydrolase Histidine sensor kinase Histidine transport protein Phage shock response Transcriptional repressor of formate hydrogenlyase Putative ﬂavoprotein Histidine sensor kinase , zinc / lead resistance STM4534 STM3773 STM0571 STM2361 2390 glnK prpB TGGCACATCCTTTGCA TGGCATAGCCTTTGCT TGGCATAAATATTGCT TGGCACGGAAGATGCA TGGCACGCCTTTTGAT TGGCACAATTACTGCA TGGCACGATAGTCGCA TGGCACGCAAATTGTA TGGCATGGAAAATGCT NtrC PrpR hypA zraP STM0649 yfhK hisJ pspA hycA FhlA HydG STM0652 YfhA STM2361 PspF FhlA 2383 2354 YgaA HydG STM2840 hydH TGGCACACTAGCTGCA TGGCATGATCTCTGCT YgaA and YfhA Two of the uncharacterised EBP-encoding genes in Salmonella , ygaA and yfhA , have orthologues in E. coli K12 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12125817" target="_blank">12125817</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=12581721" target="_blank">12581721</a>]<br></li>
				</div></ul><br>
			<label class="collapse_SIT" for="_5"><font size="5px">YfhA binding site</font></label>
				<input id="_5" type="checkbox">
				<div class="SIT"><br><ul><li>( A ) YfhA binding sites A search for YfhA binding sites with a previously reported 18 base pair ( bp ) binding motif [ 28 ] revealed three probable hits in the upstream of the rcsB gene , when a mismatch of 1 bp was allowed ( Additional file 2 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24079299" target="_blank">24079299</a>]<br></li>
<li>( C ) Structural study of YfhA and FlrC Methods Search for novel regulatory cascades for T6SS activation ( A ) Search for YfhA binding sites A previous study on E. coli had indicated a 18 bp motif recognized by YfhA [ 28 ] .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24079299" target="_blank">24079299</a>]<br></li>
				</ul></div><br>
			<label class="collapse_TU" for="_6"><font size="5px">Transcription unit regulated by YfhA</font></label>
				<input id="_6" type="checkbox">
				<div class="TU"><br><ul><li>In view of this , it may be hypothesized that , under low levels of osmolarity ( typical of an intra-macrophageal environment ) , unphosphorylated YfhA acts as a sigma-54 dependent transcription factor for the genes sciS and rcsB , and is responsible for activating the T6SS expression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24079299" target="_blank">24079299</a>]<br></li>
				</div></ul></div><br>
	</div>
</body>
</html>