<!doctype html>
<style>
@charset "utf-8";
body {
	height: 100%;
	margin: 0;
	padding: 0;
}

.banner {
	position: fixed;
	top: 0;
	width: 100%;
	height: 65px;
	background: gray;
	line-height: 65px;
}
.spacer {
	height: 10px;
}

.main-content {
	padding: 5%;
}

.collapse_ACT {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-style: none solid none none;
	border-color: #B4045F;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_ACT + input{
	display: none;
}
.collapse_ACT + input + div{
	display: none;
}
.collapse_ACT + input:checked + div{
	display: block;
}
.ACT {
	text-align: justify;
	text-justify: inter-word;
}
.ACT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B4045F;
}
.ACT_collapse_similarity:hover{
	background: #e5d3dc;
}
.ACT_collapse_similarity + input {
	display: none;
}
.ACT_collapse_similarity + input + div{
	display: none;
}

.ACT_collapse_similarity + input:checked + div{
	display: block;
}
.ACT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.ACT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B4045F;
	padding-left: 5px;
}
.ACT_collapse_similarity:hover{
	background: #e5d3dc;
}
.ACT_collapse_similarity + input {
	display: none;
}
.ACT_collapse_similarity + input + div{
	display: none;
}
.ACT_collapse_similarity + input:checked + div{
	display: block;
}
.ACT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.ACT_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #B4045F;
}
.ACT_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_DOM {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-style: none solid none none;
	border-color: #B404AE;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_DOM + input{
	display: none;
}
.collapse_DOM + input + div{
	display: none;
}
.collapse_DOM + input:checked + div{
	display: block;
}
.DOM {
	text-align: justify;
	text-justify: inter-word;
}
.DOM_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #B404AE;
	padding-left: 5px;
}
.DOM_collapse_similarity:hover{
	background: #e8d7e7;
}
.DOM_collapse_similarity + input {
	display: none;
}
.DOM_collapse_similarity + input + div{
	display: none;
}
.DOM_collapse_similarity + input:checked + div{
	display: block;
}
.DOM_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}.DOM_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #B404AE;
}
.DOM_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_EVO {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #7401DF;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_EVO + input{
	display: none;
}
.collapse_EVO + input + div{
	display: none;
}
.collapse_EVO + input:checked + div{
	display: block;
}
.EVO {
	text-align: justify;
	text-justify: inter-word;
}
.EVO_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #7401DF;
	padding-left: 5px;
}
.EVO_collapse_similarity:hover{
	background: #e2d7ed;
}
.EVO_collapse_similarity + input {
	display: none;
}
.EVO_collapse_similarity + input + div{
	display: none;
}
.EVO_collapse_similarity + input:checked + div{
	display: block;
}
.EVO_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.EVO_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #7401DF;
}
.EVO_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_RP {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #0174DF;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_RP + input{
	display: none;
}
.collapse_RP + input + div{
	display: none;
}
.collapse_RP + input:checked + div{
	display: block;
}
.RP {
	text-align: justify;
	text-justify: inter-word;
}
.RP_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #0174DF;
	padding-left: 5px;
}
.RP_collapse_similarity:hover{
	background: #c9d7e2;
}
.RP_collapse_similarity + input {
	display: none;
}
.RP_collapse_similarity + input + div{
	display: none;
}
.RP_collapse_similarity + input:checked + div{
	display: block;
}
.RP_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.RP_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #0174DF;
}
.RP_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_SIT {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #0489B1;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_SIT + input{
	display: none;
}
.collapse_SIT + input + div{
	display: none;
}
.collapse_SIT + input:checked + div{
	display: block;
}
.SIT {
	text-align: justify;
	text-justify: inter-word;
}
.SIT_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #0489B1;
	padding-left: 5px;
}
.SIT_collapse_similarity:hover{
	background: #d0dfe2;
}
.SIT_collapse_similarity + input {
	display: none;
}
.SIT_collapse_similarity + input + div{
	display: none;
}
.SIT_collapse_similarity + input:checked + div{
	display: block;
}
.SIT_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.SIT_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #0489B1;
}
.SIT_similar > a {
	text-decoration: none;
	color: gray;
}

.collapse_TU {
	cursor: pointer;
	display: block;
	color: black;
	height: 30px;
	line-height: 30px;
	border-color: #04B486;
	border-style: none solid none none;
	border-width: thick;
	background: #DCDCDC;
}
.collapse_TU + input{
	display: none;
}
.collapse_TU + input + div{
	display: none;
}
.collapse_TU + input:checked + div{
	display: block;
}
.TU {
	text-align: justify;
	text-justify: inter-word;
}
.TU_collapse_similarity {
	cursor: pointer;
	display: block;
	color: black;
	background: white;
	border-style: none none none solid;
	border-color: #04B486;
	padding-left: 5px;
}
.TU_collapse_similarity:hover{
	background: #d0e2de;
}
.TU_collapse_similarity + input {
	display: none;
}
.TU_collapse_similarity + input + div{
	display: none;
}
.TU_collapse_similarity + input:checked + div{
	display: block;
}
.TU_collapse_similarity > a {
	text-decoration: none;
	color: gray;
}
.TU_similar {
	padding-left: 2%;
	text-align: justify;
	text-justify: inter-word;
	border-style: none none none solid;
	border-color: #04B486;
}
.TU_similar > a {
	text-decoration: none;
	color: gray;
}

ul { 
	list-style-type: none;
	padding: 0;
	margin: 0;
}
ul li > a {
	text-decoration: none;
	color: gray;
}
ul li:hover, ul li:focus {
	background: #F0F0F0;
}

ul li:first-child span {
  border-left:none;  
}

</style>
<html>
<head>
<meta charset="utf-8">
<title>TF properties - YncC</title>
<meta name="theme-color" content="#ffffff">
</head>
<body>
	<div class="banner">
		<center><font color="white" size="16px" face="arial">YncC</font></center>
	</div>
	<div align="center" class="main-content">
		<div class="spacer">
	</div><br>			<label class="collapse_ACT" for="_1"><font size="5px">YncC transcriptional activity</font></label>
				<input id="_1" type="checkbox">
				<div class="ACT"><br><ul><li>In the present study , potential protein targets for YncC regulation in Salmonella were revealed by SELDI-TOF , identified and subsequently validated by in vivo and in vitro analyses .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20713450" target="_blank">20713450</a>]<br></li>
<li>However , in contrast to pyciGSTM , pyciGECO was not activated by YncC ( Fig. 7 , lanes 1 and 2 and data not shown ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20713450" target="_blank">20713450</a>]<br></li>
<li>These results are in agreement with the in vitro transcription data ( Fig. 7 ) , and suggest that YncC activation and H-NS repression occur independently at pyciGSTM .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20713450" target="_blank">20713450</a>]<br></li>
<li>H-NS and YncC can bind DNA simultaneously to regulate in vitro transcription at pyciGSTM ( Figs. 7 and 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20713450" target="_blank">20713450</a>]<br></li>
<li>Differential effects of YncC on H-NS-mediated repression .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20713450" target="_blank">20713450</a>]<br></li>
<li>When overproduced , YncC also activated S-dependent transcription at pyciG in E. coli K-12 , but solely by countering the negative effect of H-NS .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20713450" target="_blank">20713450</a>]<br></li>
<li>It is remarkable that in E. coli K-12 , regulation of yciGFE by YncC and by H-NS are intimately linked , whereas in Salmonella , YncC directly activates transcription , and thus , activation by YncC is , at least partly , disconnected from the H-NS network .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20713450" target="_blank">20713450</a>]<br></li>
<li>In vivo however , the magnitude of YncC activation was reduced in the absence of H-NS ( Figs. 2D , E ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20713450" target="_blank">20713450</a>]<br></li>
<li>YncC decreased the magnitude of H-NS-mediated repression ( Fig. 7 , lanes 3 -- 8 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20713450" target="_blank">20713450</a>]<br></li>
<li>The magnitude of H-NS repression was not affected by YncC ( Fig. 7 ) , and the magnitude of YncC activation was not affected by H-NS ( Fig. 7 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20713450" target="_blank">20713450</a>]<br></li>
<li>Consistent with this hypothesis , katN-lacZ transcription activation by His6-YncC was higher in the presence of the rpoSLT2 allele ( ninefold ) ( Fig. 2F lanes 5 and 6 ) than in the presence of the wild-type rpoS allele ( 3.5-fold ) ( Fig. 2F lanes 9 and 10 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20713450" target="_blank">20713450</a>]<br></li>
<li>The location of the bands that are only protected in the presence of both YncC and H-NS is marked with an open arrow .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20713450" target="_blank">20713450</a>]<br></li>
<li>In the wild-type strain , expression of katN-lacZ was induced earlier during growth and its expression level increased when yncC was expressed in trans from pACKyncC , suggesting that YncC might be a limiting factor for katN-lacZ expression under these conditions ( Fig. 2B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20713450" target="_blank">20713450</a>]<br></li>
<li>One hypothesis is that YncC induces yciGFEkatN expression under a specific environmental condition in the exponential phase of growth where S level is low .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20713450" target="_blank">20713450</a>]<br></li>
				</div></ul><br>
			<label class="collapse_DOM" for="_2"><font size="5px">YncC protein domains</font></label>
				<input id="_2" type="checkbox">
				<div class="DOM"><br><ul><li>YncC belongs to the GntR subfamily of FadR ( pfam007729 ) , which binds-as-a-dimer-to-its-operator site via its winged helix domains ( 10 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20713450" target="_blank">20713450</a>]<br></li>
<li>These results suggested that transcription-activation by YncC requires the CTD domain .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20713450" target="_blank">20713450</a>]<br></li>
<li>Compared with the wild-type YncC/McbR proteins , the His6-McbR and His6-YncC proteins contain 18 and 10 additional amino acids ; thus , His6-McbR migrates more slowly on SDS-PAGE than the His6-YncC protein .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20713450" target="_blank">20713450</a>]<br></li>
<li>These experiments might reveal putative cofactors that bind to the C-terminal domain of YncC and modulate its DNA-binding activity .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20713450" target="_blank">20713450</a>]<br></li>
<li>YncC Binds Upstream of the yciGFEkatN Promoter -- Plasmid pyncCHIS encodes a recombinant YncC protein containing six histidine residues at its N terminus ( His6-YncC ) , under the control of the IPTG-inducible promoter of pQE30 .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20713450" target="_blank">20713450</a>]<br></li>
<li>In addition , the amino acid sequences of YncC and McbR diverge in the C-terminal domain ( 46 % identity over amino acids 78 to 221 ) , compared with the N-terminal domain ( 81 % identity over amino acids 1 to 77 ) , which contains the predicted DNA-binding HTH domain ( 8 , 9 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20713450" target="_blank">20713450</a>]<br></li>
<li>YncC was not detected with a RNA polymerase in which the subunit lacks the C-terminal domain ( Fig. 6C ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20713450" target="_blank">20713450</a>]<br></li>
<li>YncC transcriptional activation at pyciGSTM : requirements for S and the C-terminal domain of the subunit of RNA polymerase .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20713450" target="_blank">20713450</a>]<br></li>
<li>However the intrinsic DNA curvature found in many S-regulated promoters ( including pyciGSTM , data not shown ) might facilitate protein-protein contacts between YncC and the CTD of the distal subunit of RNA polymerase , which is used preferentially by E S for activation ( 46 -- 48 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20713450" target="_blank">20713450</a>]<br></li>
				</div></ul><br>
			<label class="collapse_EVO" for="_3"><font size="5px">YncC evolution</font></label>
				<input id="_3" type="checkbox">
				<div class="EVO"><br><ul><li>YncC belongs to the GntR subfamily of FadR ( pfam007729 ) , which binds as a dimer to its operator site via its winged helix domains ( 10 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20713450" target="_blank">20713450</a>]<br></li>
<li>We are grateful to Pierre Dehoux ( PF4 , Institut Pasteur ) for sequence comparisons between YncC and regulators of the GntR family .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20713450" target="_blank">20713450</a>]<br></li>
<li>YgaE ( also known as GabC ) and YncC belong to the GntR family of transcriptional regulators .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=24271167" target="_blank">24271167</a>]<br></li>
<li>In addition , the amino acid sequences of YncC and McbR diverge in the C-terminal domain ( 46 % identity over amino acids 78 to 221 ) , compared with the N-terminal domain ( 81 % identity over amino acids 1 to 77 ) , which contains the predicted DNA binding HTH domain ( 8 , 9 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20713450" target="_blank">20713450</a>]<br></li>
<li>The sequence of the YncC binding region ( in pyciGSTM ) ( Fig. 3B ) is 100 % identical in the genomes of S. enterica subsp .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20713450" target="_blank">20713450</a>]<br></li>
				</div></ul><br>
			<label class="collapse_RP" for="_4"><font size="5px">Processes regulated by YncC</font></label>
				<input id="_4" type="checkbox">
				<div class="RP"><br><ul><li>Accession numbers for the E. coli proteins were P33361 ( YehY ) , P37663 ( YhjY ) , P07003 ( PoxB ) , P76114 ( YncC ) , P27250 ( YjgB ) , P75694 ( YahO ) , P39169 ( YgaU ) , P31677 ( OtsA ) , P77391 ( YeaG ) , P21179 ( KatE ) , P21362 ( YciF ) , and P29013 ( YcgB ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=11004173" target="_blank">11004173</a>]<br></li>
<li>Zhang , X. S. , Garcia-Contreras , R. , and Wood , T. K. ( 2008 ) Escherichia coli transcription factor YncC ( McbR ) regulates colanic acid and biofilm-formation by repressing expression of periplasmic protein YbiM ( McbA ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20713450" target="_blank">20713450</a>]<br></li>
<li>Research A Proteomic Analysis Reveals Differential S - Regulation of the Dependent yciGFE ( katN ) Locus by YncC and H-NS in Salmonella and Escherichia coli K-12 Mélanie Beraud § ¶ , Annie Kolb § ¶ , Véronique Monteil § ¶ , Jacques D'Alayer , and Françoise Norel § ¶ ‡ The stationary phase sigma factor S ( RpoS ) controls a regulon required for general stress resistance of the closely related enterobacteria Salmonella and Esche-richia coli .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20713450" target="_blank">20713450</a>]<br></li>
<li>YncC also Binds to the E. coli K-12 yciGFE Promoter Region -- The ortholog of yncC in E. coli K-12 ( named mcbR ) , regulates colanic acid production by repressing expression of the ybiM gene ( 13 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20713450" target="_blank">20713450</a>]<br></li>
<li>Like the His6-YncC protein , the His6-McbR protein activated expression of the katN-lacZ fusion in the yncC and wild-type strains of Salmonella ( Fig. 2C ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20713450" target="_blank">20713450</a>]<br></li>
				</div></ul><br>
			<label class="collapse_SIT" for="_5"><font size="5px">YncC binding site</font></label>
				<input id="_5" type="checkbox">
				<div class="SIT"><br><ul><li>Based on these data , YncC likely binds DNA as a dimer and we predict that at least two tandem YncC operator sites are present upstream of the pyciGECO promoter .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20713450" target="_blank">20713450</a>]<br></li>
<li>The YncC binding site is AT-rich and contains the inverted-repeat AATATAT .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20713450" target="_blank">20713450</a>]<br></li>
<li>His6-YncC protected a 24 bp sequence centered on the -- 100 region ( Fig. 3B ) relative to the transcription start site ( 14 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20713450" target="_blank">20713450</a>]<br></li>
<li>The first site , located in the upstream YncC box AATaTATCcA ( Fig. 3B ) , matches the consensus over 8 bp , whereas the other , AATaTATttt in the downstream YncC box ( Fig. 3B ) , has 6 bp matches and , therefore , is expected to have lower affinity ( note that the bases matching the H-NS consensus site are indicated in capital letters ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20713450" target="_blank">20713450</a>]<br></li>
<li>As for the pyciGECO , two predicted H-NS binding sites ( agaATAtATT centered at -- 98.5 and AtaTTATCTc centered at -- 93.5 ) in pyciGSTM overlap the single YncC binding site ( Fig. 3B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20713450" target="_blank">20713450</a>]<br></li>
<li>B , Sequences of the YncC binding sites .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20713450" target="_blank">20713450</a>]<br></li>
<li>The protection footprint of about 20 bp at pyciGSTM suggests the presence of a single YncC binding site .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20713450" target="_blank">20713450</a>]<br></li>
<li>The position of the YncC binding site , centered at -- 101 with respect to the transcription start site of yciG , is 10 bp upstream of the most distant transcription activators at simple 70-dependent promoters .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20713450" target="_blank">20713450</a>]<br></li>
				</ul></div><br>
			<label class="collapse_TU" for="_6"><font size="5px">Transcription units regulated by YncC</font></label>
				<input id="_6" type="checkbox">
				<div class="TU"><br><ul><li>Zhang , X. S. , Garcia-Contreras , R. , and Wood , T. K. ( 2008 ) Escherichia coli transcription factor YncC ( McbR ) regulates colanic acid and biofilm formation by repressing expression of periplasmic protein YbiM ( McbA ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20713450" target="_blank">20713450</a>]<br></li>
<li>One hypothesis is that YncC induces yciGFEkatN expression under a specific environmental condition in the exponential phase of growth where S level is low .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20713450" target="_blank">20713450</a>]<br></li>
<li>These proteins are encoded by the S-dependent operon yciG-FEkatN and regulation of their expression by YncC operates at the transcriptional level , as demonstrated by gene fusion analyses and by in vitro transcription and DNase I footprinting experiments with purified YncC .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20713450" target="_blank">20713450</a>]<br></li>
<li>yciE-lacZ expression was induced to higher levels by His-6 McbR than by His6-YncC ( Fig. 4B , lanes 7 and 13 ) , likely because the former is more abundant ( Fig. 5A ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20713450" target="_blank">20713450</a>]<br></li>
<li>YncC also Binds to the E. coli K-12 yciGFE Promoter Region -- The ortholog of yncC in E. coli K-12 ( named mcbR ) , regulates colanic acid production by repressing expression of the ybiM gene ( 13 ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20713450" target="_blank">20713450</a>]<br></li>
<li>Altogether , these results show that YncC and McbR are both able to induce expression of the yciGFE ( katN ) locus in Salmonella and E. coli K-12 , and that this locus is expressed at drastically lower levels in E. coli K-12 than in Salmonella .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20713450" target="_blank">20713450</a>]<br></li>
<li>These results suggested that YncC and S act in concert to induce full expression of the operon .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20713450" target="_blank">20713450</a>]<br></li>
<li>Like the His6-YncC protein , the His6-McbR protein activated expression of the katN-lacZ fusion in the yncC and wild-type strains of Salmonella ( Fig. 2C ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20713450" target="_blank">20713450</a>]<br></li>
<li>YncC Directly Activates E S-dependent In Vitro Transcription at pyciGSTM -- Significant expression of the yciGFEkatN operon was not detected in the rpoS mutant of Salmonella ( 12 ) ( Fig. 2F , lane 2 ) , and katN-lacZ expression was very low in the rpoShns strain of Salmonella ( Figs. 2D , E ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20713450" target="_blank">20713450</a>]<br></li>
<li>In the wild-type strain , expression of katN-lacZ was induced earlier during growth and its expression level increased when yncC was expressed in trans from pACKyncC , suggesting that YncC might be a limiting factor for katN-lacZ expression under these conditions ( Fig. 2B ) .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20713450" target="_blank">20713450</a>]<br></li>
<li>pyncCHIS , but not the pQE30 vector , complemented the yncC strain for katN-lacZ expression and increased katN-lacZ expression in the wild-type strain ( Fig. 2F , lanes 9 and 10 and data not shown ) , indicating that His6-YncC is active .	[<a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=20713450" target="_blank">20713450</a>]<br></li>
				</div></ul></div><br>
	</div>
</body>
</html>